<!DOCTYPE html>
<html>
<head><meta charset="utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>기말프로젝트 (1)</title><script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.1.10/require.min.js"></script>




<style type="text/css">
    pre { line-height: 125%; }
td.linenos .normal { color: inherit; background-color: transparent; padding-left: 5px; padding-right: 5px; }
span.linenos { color: inherit; background-color: transparent; padding-left: 5px; padding-right: 5px; }
td.linenos .special { color: #000000; background-color: #ffffc0; padding-left: 5px; padding-right: 5px; }
span.linenos.special { color: #000000; background-color: #ffffc0; padding-left: 5px; padding-right: 5px; }
.highlight .hll { background-color: var(--jp-cell-editor-active-background) }
.highlight { background: var(--jp-cell-editor-background); color: var(--jp-mirror-editor-variable-color) }
.highlight .c { color: var(--jp-mirror-editor-comment-color); font-style: italic } /* Comment */
.highlight .err { color: var(--jp-mirror-editor-error-color) } /* Error */
.highlight .k { color: var(--jp-mirror-editor-keyword-color); font-weight: bold } /* Keyword */
.highlight .o { color: var(--jp-mirror-editor-operator-color); font-weight: bold } /* Operator */
.highlight .p { color: var(--jp-mirror-editor-punctuation-color) } /* Punctuation */
.highlight .ch { color: var(--jp-mirror-editor-comment-color); font-style: italic } /* Comment.Hashbang */
.highlight .cm { color: var(--jp-mirror-editor-comment-color); font-style: italic } /* Comment.Multiline */
.highlight .cp { color: var(--jp-mirror-editor-comment-color); font-style: italic } /* Comment.Preproc */
.highlight .cpf { color: var(--jp-mirror-editor-comment-color); font-style: italic } /* Comment.PreprocFile */
.highlight .c1 { color: var(--jp-mirror-editor-comment-color); font-style: italic } /* Comment.Single */
.highlight .cs { color: var(--jp-mirror-editor-comment-color); font-style: italic } /* Comment.Special */
.highlight .kc { color: var(--jp-mirror-editor-keyword-color); font-weight: bold } /* Keyword.Constant */
.highlight .kd { color: var(--jp-mirror-editor-keyword-color); font-weight: bold } /* Keyword.Declaration */
.highlight .kn { color: var(--jp-mirror-editor-keyword-color); font-weight: bold } /* Keyword.Namespace */
.highlight .kp { color: var(--jp-mirror-editor-keyword-color); font-weight: bold } /* Keyword.Pseudo */
.highlight .kr { color: var(--jp-mirror-editor-keyword-color); font-weight: bold } /* Keyword.Reserved */
.highlight .kt { color: var(--jp-mirror-editor-keyword-color); font-weight: bold } /* Keyword.Type */
.highlight .m { color: var(--jp-mirror-editor-number-color) } /* Literal.Number */
.highlight .s { color: var(--jp-mirror-editor-string-color) } /* Literal.String */
.highlight .ow { color: var(--jp-mirror-editor-operator-color); font-weight: bold } /* Operator.Word */
.highlight .w { color: var(--jp-mirror-editor-variable-color) } /* Text.Whitespace */
.highlight .mb { color: var(--jp-mirror-editor-number-color) } /* Literal.Number.Bin */
.highlight .mf { color: var(--jp-mirror-editor-number-color) } /* Literal.Number.Float */
.highlight .mh { color: var(--jp-mirror-editor-number-color) } /* Literal.Number.Hex */
.highlight .mi { color: var(--jp-mirror-editor-number-color) } /* Literal.Number.Integer */
.highlight .mo { color: var(--jp-mirror-editor-number-color) } /* Literal.Number.Oct */
.highlight .sa { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Affix */
.highlight .sb { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Backtick */
.highlight .sc { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Char */
.highlight .dl { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Delimiter */
.highlight .sd { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Doc */
.highlight .s2 { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Double */
.highlight .se { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Escape */
.highlight .sh { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Heredoc */
.highlight .si { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Interpol */
.highlight .sx { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Other */
.highlight .sr { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Regex */
.highlight .s1 { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Single */
.highlight .ss { color: var(--jp-mirror-editor-string-color) } /* Literal.String.Symbol */
.highlight .il { color: var(--jp-mirror-editor-number-color) } /* Literal.Number.Integer.Long */
  </style>



<style type="text/css">
/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*
 * Mozilla scrollbar styling
 */

/* use standard opaque scrollbars for most nodes */
[data-jp-theme-scrollbars='true'] {
  scrollbar-color: rgb(var(--jp-scrollbar-thumb-color))
    var(--jp-scrollbar-background-color);
}

/* for code nodes, use a transparent style of scrollbar. These selectors
 * will match lower in the tree, and so will override the above */
[data-jp-theme-scrollbars='true'] .CodeMirror-hscrollbar,
[data-jp-theme-scrollbars='true'] .CodeMirror-vscrollbar {
  scrollbar-color: rgba(var(--jp-scrollbar-thumb-color), 0.5) transparent;
}

/* tiny scrollbar */

.jp-scrollbar-tiny {
  scrollbar-color: rgba(var(--jp-scrollbar-thumb-color), 0.5) transparent;
  scrollbar-width: thin;
}

/*
 * Webkit scrollbar styling
 */

/* use standard opaque scrollbars for most nodes */

[data-jp-theme-scrollbars='true'] ::-webkit-scrollbar,
[data-jp-theme-scrollbars='true'] ::-webkit-scrollbar-corner {
  background: var(--jp-scrollbar-background-color);
}

[data-jp-theme-scrollbars='true'] ::-webkit-scrollbar-thumb {
  background: rgb(var(--jp-scrollbar-thumb-color));
  border: var(--jp-scrollbar-thumb-margin) solid transparent;
  background-clip: content-box;
  border-radius: var(--jp-scrollbar-thumb-radius);
}

[data-jp-theme-scrollbars='true'] ::-webkit-scrollbar-track:horizontal {
  border-left: var(--jp-scrollbar-endpad) solid
    var(--jp-scrollbar-background-color);
  border-right: var(--jp-scrollbar-endpad) solid
    var(--jp-scrollbar-background-color);
}

[data-jp-theme-scrollbars='true'] ::-webkit-scrollbar-track:vertical {
  border-top: var(--jp-scrollbar-endpad) solid
    var(--jp-scrollbar-background-color);
  border-bottom: var(--jp-scrollbar-endpad) solid
    var(--jp-scrollbar-background-color);
}

/* for code nodes, use a transparent style of scrollbar */

[data-jp-theme-scrollbars='true'] .CodeMirror-hscrollbar::-webkit-scrollbar,
[data-jp-theme-scrollbars='true'] .CodeMirror-vscrollbar::-webkit-scrollbar,
[data-jp-theme-scrollbars='true']
  .CodeMirror-hscrollbar::-webkit-scrollbar-corner,
[data-jp-theme-scrollbars='true']
  .CodeMirror-vscrollbar::-webkit-scrollbar-corner {
  background-color: transparent;
}

[data-jp-theme-scrollbars='true']
  .CodeMirror-hscrollbar::-webkit-scrollbar-thumb,
[data-jp-theme-scrollbars='true']
  .CodeMirror-vscrollbar::-webkit-scrollbar-thumb {
  background: rgba(var(--jp-scrollbar-thumb-color), 0.5);
  border: var(--jp-scrollbar-thumb-margin) solid transparent;
  background-clip: content-box;
  border-radius: var(--jp-scrollbar-thumb-radius);
}

[data-jp-theme-scrollbars='true']
  .CodeMirror-hscrollbar::-webkit-scrollbar-track:horizontal {
  border-left: var(--jp-scrollbar-endpad) solid transparent;
  border-right: var(--jp-scrollbar-endpad) solid transparent;
}

[data-jp-theme-scrollbars='true']
  .CodeMirror-vscrollbar::-webkit-scrollbar-track:vertical {
  border-top: var(--jp-scrollbar-endpad) solid transparent;
  border-bottom: var(--jp-scrollbar-endpad) solid transparent;
}

/* tiny scrollbar */

.jp-scrollbar-tiny::-webkit-scrollbar,
.jp-scrollbar-tiny::-webkit-scrollbar-corner {
  background-color: transparent;
  height: 4px;
  width: 4px;
}

.jp-scrollbar-tiny::-webkit-scrollbar-thumb {
  background: rgba(var(--jp-scrollbar-thumb-color), 0.5);
}

.jp-scrollbar-tiny::-webkit-scrollbar-track:horizontal {
  border-left: 0px solid transparent;
  border-right: 0px solid transparent;
}

.jp-scrollbar-tiny::-webkit-scrollbar-track:vertical {
  border-top: 0px solid transparent;
  border-bottom: 0px solid transparent;
}

/*
 * Phosphor
 */

.lm-ScrollBar[data-orientation='horizontal'] {
  min-height: 16px;
  max-height: 16px;
  min-width: 45px;
  border-top: 1px solid #a0a0a0;
}

.lm-ScrollBar[data-orientation='vertical'] {
  min-width: 16px;
  max-width: 16px;
  min-height: 45px;
  border-left: 1px solid #a0a0a0;
}

.lm-ScrollBar-button {
  background-color: #f0f0f0;
  background-position: center center;
  min-height: 15px;
  max-height: 15px;
  min-width: 15px;
  max-width: 15px;
}

.lm-ScrollBar-button:hover {
  background-color: #dadada;
}

.lm-ScrollBar-button.lm-mod-active {
  background-color: #cdcdcd;
}

.lm-ScrollBar-track {
  background: #f0f0f0;
}

.lm-ScrollBar-thumb {
  background: #cdcdcd;
}

.lm-ScrollBar-thumb:hover {
  background: #bababa;
}

.lm-ScrollBar-thumb.lm-mod-active {
  background: #a0a0a0;
}

.lm-ScrollBar[data-orientation='horizontal'] .lm-ScrollBar-thumb {
  height: 100%;
  min-width: 15px;
  border-left: 1px solid #a0a0a0;
  border-right: 1px solid #a0a0a0;
}

.lm-ScrollBar[data-orientation='vertical'] .lm-ScrollBar-thumb {
  width: 100%;
  min-height: 15px;
  border-top: 1px solid #a0a0a0;
  border-bottom: 1px solid #a0a0a0;
}

.lm-ScrollBar[data-orientation='horizontal']
  .lm-ScrollBar-button[data-action='decrement'] {
  background-image: var(--jp-icon-caret-left);
  background-size: 17px;
}

.lm-ScrollBar[data-orientation='horizontal']
  .lm-ScrollBar-button[data-action='increment'] {
  background-image: var(--jp-icon-caret-right);
  background-size: 17px;
}

.lm-ScrollBar[data-orientation='vertical']
  .lm-ScrollBar-button[data-action='decrement'] {
  background-image: var(--jp-icon-caret-up);
  background-size: 17px;
}

.lm-ScrollBar[data-orientation='vertical']
  .lm-ScrollBar-button[data-action='increment'] {
  background-image: var(--jp-icon-caret-down);
  background-size: 17px;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/


/* <DEPRECATED> */ .p-Widget, /* </DEPRECATED> */
.lm-Widget {
  box-sizing: border-box;
  position: relative;
  overflow: hidden;
  cursor: default;
}


/* <DEPRECATED> */ .p-Widget.p-mod-hidden, /* </DEPRECATED> */
.lm-Widget.lm-mod-hidden {
  display: none !important;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/


/* <DEPRECATED> */ .p-CommandPalette, /* </DEPRECATED> */
.lm-CommandPalette {
  display: flex;
  flex-direction: column;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}


/* <DEPRECATED> */ .p-CommandPalette-search, /* </DEPRECATED> */
.lm-CommandPalette-search {
  flex: 0 0 auto;
}


/* <DEPRECATED> */ .p-CommandPalette-content, /* </DEPRECATED> */
.lm-CommandPalette-content {
  flex: 1 1 auto;
  margin: 0;
  padding: 0;
  min-height: 0;
  overflow: auto;
  list-style-type: none;
}


/* <DEPRECATED> */ .p-CommandPalette-header, /* </DEPRECATED> */
.lm-CommandPalette-header {
  overflow: hidden;
  white-space: nowrap;
  text-overflow: ellipsis;
}


/* <DEPRECATED> */ .p-CommandPalette-item, /* </DEPRECATED> */
.lm-CommandPalette-item {
  display: flex;
  flex-direction: row;
}


/* <DEPRECATED> */ .p-CommandPalette-itemIcon, /* </DEPRECATED> */
.lm-CommandPalette-itemIcon {
  flex: 0 0 auto;
}


/* <DEPRECATED> */ .p-CommandPalette-itemContent, /* </DEPRECATED> */
.lm-CommandPalette-itemContent {
  flex: 1 1 auto;
  overflow: hidden;
}


/* <DEPRECATED> */ .p-CommandPalette-itemShortcut, /* </DEPRECATED> */
.lm-CommandPalette-itemShortcut {
  flex: 0 0 auto;
}


/* <DEPRECATED> */ .p-CommandPalette-itemLabel, /* </DEPRECATED> */
.lm-CommandPalette-itemLabel {
  overflow: hidden;
  white-space: nowrap;
  text-overflow: ellipsis;
}

.lm-close-icon {
	border:1px solid transparent;
  background-color: transparent;
  position: absolute;
	z-index:1;
	right:3%;
	top: 0;
	bottom: 0;
	margin: auto;
	padding: 7px 0;
	display: none;
	vertical-align: middle;
  outline: 0;
  cursor: pointer;
}
.lm-close-icon:after {
	content: "X";
	display: block;
	width: 15px;
	height: 15px;
	text-align: center;
	color:#000;
	font-weight: normal;
	font-size: 12px;
	cursor: pointer;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/


/* <DEPRECATED> */ .p-DockPanel, /* </DEPRECATED> */
.lm-DockPanel {
  z-index: 0;
}


/* <DEPRECATED> */ .p-DockPanel-widget, /* </DEPRECATED> */
.lm-DockPanel-widget {
  z-index: 0;
}


/* <DEPRECATED> */ .p-DockPanel-tabBar, /* </DEPRECATED> */
.lm-DockPanel-tabBar {
  z-index: 1;
}


/* <DEPRECATED> */ .p-DockPanel-handle, /* </DEPRECATED> */
.lm-DockPanel-handle {
  z-index: 2;
}


/* <DEPRECATED> */ .p-DockPanel-handle.p-mod-hidden, /* </DEPRECATED> */
.lm-DockPanel-handle.lm-mod-hidden {
  display: none !important;
}


/* <DEPRECATED> */ .p-DockPanel-handle:after, /* </DEPRECATED> */
.lm-DockPanel-handle:after {
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  content: '';
}


/* <DEPRECATED> */
.p-DockPanel-handle[data-orientation='horizontal'],
/* </DEPRECATED> */
.lm-DockPanel-handle[data-orientation='horizontal'] {
  cursor: ew-resize;
}


/* <DEPRECATED> */
.p-DockPanel-handle[data-orientation='vertical'],
/* </DEPRECATED> */
.lm-DockPanel-handle[data-orientation='vertical'] {
  cursor: ns-resize;
}


/* <DEPRECATED> */
.p-DockPanel-handle[data-orientation='horizontal']:after,
/* </DEPRECATED> */
.lm-DockPanel-handle[data-orientation='horizontal']:after {
  left: 50%;
  min-width: 8px;
  transform: translateX(-50%);
}


/* <DEPRECATED> */
.p-DockPanel-handle[data-orientation='vertical']:after,
/* </DEPRECATED> */
.lm-DockPanel-handle[data-orientation='vertical']:after {
  top: 50%;
  min-height: 8px;
  transform: translateY(-50%);
}


/* <DEPRECATED> */ .p-DockPanel-overlay, /* </DEPRECATED> */
.lm-DockPanel-overlay {
  z-index: 3;
  box-sizing: border-box;
  pointer-events: none;
}


/* <DEPRECATED> */ .p-DockPanel-overlay.p-mod-hidden, /* </DEPRECATED> */
.lm-DockPanel-overlay.lm-mod-hidden {
  display: none !important;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/


/* <DEPRECATED> */ .p-Menu, /* </DEPRECATED> */
.lm-Menu {
  z-index: 10000;
  position: absolute;
  white-space: nowrap;
  overflow-x: hidden;
  overflow-y: auto;
  outline: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}


/* <DEPRECATED> */ .p-Menu-content, /* </DEPRECATED> */
.lm-Menu-content {
  margin: 0;
  padding: 0;
  display: table;
  list-style-type: none;
}


/* <DEPRECATED> */ .p-Menu-item, /* </DEPRECATED> */
.lm-Menu-item {
  display: table-row;
}


/* <DEPRECATED> */
.p-Menu-item.p-mod-hidden,
.p-Menu-item.p-mod-collapsed,
/* </DEPRECATED> */
.lm-Menu-item.lm-mod-hidden,
.lm-Menu-item.lm-mod-collapsed {
  display: none !important;
}


/* <DEPRECATED> */
.p-Menu-itemIcon,
.p-Menu-itemSubmenuIcon,
/* </DEPRECATED> */
.lm-Menu-itemIcon,
.lm-Menu-itemSubmenuIcon {
  display: table-cell;
  text-align: center;
}


/* <DEPRECATED> */ .p-Menu-itemLabel, /* </DEPRECATED> */
.lm-Menu-itemLabel {
  display: table-cell;
  text-align: left;
}


/* <DEPRECATED> */ .p-Menu-itemShortcut, /* </DEPRECATED> */
.lm-Menu-itemShortcut {
  display: table-cell;
  text-align: right;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/


/* <DEPRECATED> */ .p-MenuBar, /* </DEPRECATED> */
.lm-MenuBar {
  outline: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}


/* <DEPRECATED> */ .p-MenuBar-content, /* </DEPRECATED> */
.lm-MenuBar-content {
  margin: 0;
  padding: 0;
  display: flex;
  flex-direction: row;
  list-style-type: none;
}


/* <DEPRECATED> */ .p--MenuBar-item, /* </DEPRECATED> */
.lm-MenuBar-item {
  box-sizing: border-box;
}


/* <DEPRECATED> */
.p-MenuBar-itemIcon,
.p-MenuBar-itemLabel,
/* </DEPRECATED> */
.lm-MenuBar-itemIcon,
.lm-MenuBar-itemLabel {
  display: inline-block;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/


/* <DEPRECATED> */ .p-ScrollBar, /* </DEPRECATED> */
.lm-ScrollBar {
  display: flex;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}


/* <DEPRECATED> */
.p-ScrollBar[data-orientation='horizontal'],
/* </DEPRECATED> */
.lm-ScrollBar[data-orientation='horizontal'] {
  flex-direction: row;
}


/* <DEPRECATED> */
.p-ScrollBar[data-orientation='vertical'],
/* </DEPRECATED> */
.lm-ScrollBar[data-orientation='vertical'] {
  flex-direction: column;
}


/* <DEPRECATED> */ .p-ScrollBar-button, /* </DEPRECATED> */
.lm-ScrollBar-button {
  box-sizing: border-box;
  flex: 0 0 auto;
}


/* <DEPRECATED> */ .p-ScrollBar-track, /* </DEPRECATED> */
.lm-ScrollBar-track {
  box-sizing: border-box;
  position: relative;
  overflow: hidden;
  flex: 1 1 auto;
}


/* <DEPRECATED> */ .p-ScrollBar-thumb, /* </DEPRECATED> */
.lm-ScrollBar-thumb {
  box-sizing: border-box;
  position: absolute;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/


/* <DEPRECATED> */ .p-SplitPanel-child, /* </DEPRECATED> */
.lm-SplitPanel-child {
  z-index: 0;
}


/* <DEPRECATED> */ .p-SplitPanel-handle, /* </DEPRECATED> */
.lm-SplitPanel-handle {
  z-index: 1;
}


/* <DEPRECATED> */ .p-SplitPanel-handle.p-mod-hidden, /* </DEPRECATED> */
.lm-SplitPanel-handle.lm-mod-hidden {
  display: none !important;
}


/* <DEPRECATED> */ .p-SplitPanel-handle:after, /* </DEPRECATED> */
.lm-SplitPanel-handle:after {
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  content: '';
}


/* <DEPRECATED> */
.p-SplitPanel[data-orientation='horizontal'] > .p-SplitPanel-handle,
/* </DEPRECATED> */
.lm-SplitPanel[data-orientation='horizontal'] > .lm-SplitPanel-handle {
  cursor: ew-resize;
}


/* <DEPRECATED> */
.p-SplitPanel[data-orientation='vertical'] > .p-SplitPanel-handle,
/* </DEPRECATED> */
.lm-SplitPanel[data-orientation='vertical'] > .lm-SplitPanel-handle {
  cursor: ns-resize;
}


/* <DEPRECATED> */
.p-SplitPanel[data-orientation='horizontal'] > .p-SplitPanel-handle:after,
/* </DEPRECATED> */
.lm-SplitPanel[data-orientation='horizontal'] > .lm-SplitPanel-handle:after {
  left: 50%;
  min-width: 8px;
  transform: translateX(-50%);
}


/* <DEPRECATED> */
.p-SplitPanel[data-orientation='vertical'] > .p-SplitPanel-handle:after,
/* </DEPRECATED> */
.lm-SplitPanel[data-orientation='vertical'] > .lm-SplitPanel-handle:after {
  top: 50%;
  min-height: 8px;
  transform: translateY(-50%);
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/


/* <DEPRECATED> */ .p-TabBar, /* </DEPRECATED> */
.lm-TabBar {
  display: flex;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}


/* <DEPRECATED> */ .p-TabBar[data-orientation='horizontal'], /* </DEPRECATED> */
.lm-TabBar[data-orientation='horizontal'] {
  flex-direction: row;
}


/* <DEPRECATED> */ .p-TabBar[data-orientation='vertical'], /* </DEPRECATED> */
.lm-TabBar[data-orientation='vertical'] {
  flex-direction: column;
}


/* <DEPRECATED> */ .p-TabBar-content, /* </DEPRECATED> */
.lm-TabBar-content {
  margin: 0;
  padding: 0;
  display: flex;
  flex: 1 1 auto;
  list-style-type: none;
}


/* <DEPRECATED> */
.p-TabBar[data-orientation='horizontal'] > .p-TabBar-content,
/* </DEPRECATED> */
.lm-TabBar[data-orientation='horizontal'] > .lm-TabBar-content {
  flex-direction: row;
}


/* <DEPRECATED> */
.p-TabBar[data-orientation='vertical'] > .p-TabBar-content,
/* </DEPRECATED> */
.lm-TabBar[data-orientation='vertical'] > .lm-TabBar-content {
  flex-direction: column;
}


/* <DEPRECATED> */ .p-TabBar-tab, /* </DEPRECATED> */
.lm-TabBar-tab {
  display: flex;
  flex-direction: row;
  box-sizing: border-box;
  overflow: hidden;
}


/* <DEPRECATED> */
.p-TabBar-tabIcon,
.p-TabBar-tabCloseIcon,
/* </DEPRECATED> */
.lm-TabBar-tabIcon,
.lm-TabBar-tabCloseIcon {
  flex: 0 0 auto;
}


/* <DEPRECATED> */ .p-TabBar-tabLabel, /* </DEPRECATED> */
.lm-TabBar-tabLabel {
  flex: 1 1 auto;
  overflow: hidden;
  white-space: nowrap;
}


.lm-TabBar-tabInput {
  user-select: all;
  width: 100%;
  box-sizing : border-box;
}


/* <DEPRECATED> */ .p-TabBar-tab.p-mod-hidden, /* </DEPRECATED> */
.lm-TabBar-tab.lm-mod-hidden {
  display: none !important;
}


/* <DEPRECATED> */ .p-TabBar.p-mod-dragging .p-TabBar-tab, /* </DEPRECATED> */
.lm-TabBar.lm-mod-dragging .lm-TabBar-tab {
  position: relative;
}


/* <DEPRECATED> */
.p-TabBar.p-mod-dragging[data-orientation='horizontal'] .p-TabBar-tab,
/* </DEPRECATED> */
.lm-TabBar.lm-mod-dragging[data-orientation='horizontal'] .lm-TabBar-tab {
  left: 0;
  transition: left 150ms ease;
}


/* <DEPRECATED> */
.p-TabBar.p-mod-dragging[data-orientation='vertical'] .p-TabBar-tab,
/* </DEPRECATED> */
.lm-TabBar.lm-mod-dragging[data-orientation='vertical'] .lm-TabBar-tab {
  top: 0;
  transition: top 150ms ease;
}


/* <DEPRECATED> */
.p-TabBar.p-mod-dragging .p-TabBar-tab.p-mod-dragging,
/* </DEPRECATED> */
.lm-TabBar.lm-mod-dragging .lm-TabBar-tab.lm-mod-dragging {
  transition: none;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/


/* <DEPRECATED> */ .p-TabPanel-tabBar, /* </DEPRECATED> */
.lm-TabPanel-tabBar {
  z-index: 1;
}


/* <DEPRECATED> */ .p-TabPanel-stackedPanel, /* </DEPRECATED> */
.lm-TabPanel-stackedPanel {
  z-index: 0;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/

@charset "UTF-8";
html{
  -webkit-box-sizing:border-box;
          box-sizing:border-box; }

*,
*::before,
*::after{
  -webkit-box-sizing:inherit;
          box-sizing:inherit; }

body{
  font-size:14px;
  font-weight:400;
  letter-spacing:0;
  line-height:1.28581;
  text-transform:none;
  color:#182026;
  font-family:-apple-system, "BlinkMacSystemFont", "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Open Sans", "Helvetica Neue", "Icons16", sans-serif; }

p{
  margin-bottom:10px;
  margin-top:0; }

small{
  font-size:12px; }

strong{
  font-weight:600; }

::-moz-selection{
  background:rgba(125, 188, 255, 0.6); }

::selection{
  background:rgba(125, 188, 255, 0.6); }
.bp3-heading{
  color:#182026;
  font-weight:600;
  margin:0 0 10px;
  padding:0; }
  .bp3-dark .bp3-heading{
    color:#f5f8fa; }

h1.bp3-heading, .bp3-running-text h1{
  font-size:36px;
  line-height:40px; }

h2.bp3-heading, .bp3-running-text h2{
  font-size:28px;
  line-height:32px; }

h3.bp3-heading, .bp3-running-text h3{
  font-size:22px;
  line-height:25px; }

h4.bp3-heading, .bp3-running-text h4{
  font-size:18px;
  line-height:21px; }

h5.bp3-heading, .bp3-running-text h5{
  font-size:16px;
  line-height:19px; }

h6.bp3-heading, .bp3-running-text h6{
  font-size:14px;
  line-height:16px; }
.bp3-ui-text{
  font-size:14px;
  font-weight:400;
  letter-spacing:0;
  line-height:1.28581;
  text-transform:none; }

.bp3-monospace-text{
  font-family:monospace;
  text-transform:none; }

.bp3-text-muted{
  color:#5c7080; }
  .bp3-dark .bp3-text-muted{
    color:#a7b6c2; }

.bp3-text-disabled{
  color:rgba(92, 112, 128, 0.6); }
  .bp3-dark .bp3-text-disabled{
    color:rgba(167, 182, 194, 0.6); }

.bp3-text-overflow-ellipsis{
  overflow:hidden;
  text-overflow:ellipsis;
  white-space:nowrap;
  word-wrap:normal; }
.bp3-running-text{
  font-size:14px;
  line-height:1.5; }
  .bp3-running-text h1{
    color:#182026;
    font-weight:600;
    margin-bottom:20px;
    margin-top:40px; }
    .bp3-dark .bp3-running-text h1{
      color:#f5f8fa; }
  .bp3-running-text h2{
    color:#182026;
    font-weight:600;
    margin-bottom:20px;
    margin-top:40px; }
    .bp3-dark .bp3-running-text h2{
      color:#f5f8fa; }
  .bp3-running-text h3{
    color:#182026;
    font-weight:600;
    margin-bottom:20px;
    margin-top:40px; }
    .bp3-dark .bp3-running-text h3{
      color:#f5f8fa; }
  .bp3-running-text h4{
    color:#182026;
    font-weight:600;
    margin-bottom:20px;
    margin-top:40px; }
    .bp3-dark .bp3-running-text h4{
      color:#f5f8fa; }
  .bp3-running-text h5{
    color:#182026;
    font-weight:600;
    margin-bottom:20px;
    margin-top:40px; }
    .bp3-dark .bp3-running-text h5{
      color:#f5f8fa; }
  .bp3-running-text h6{
    color:#182026;
    font-weight:600;
    margin-bottom:20px;
    margin-top:40px; }
    .bp3-dark .bp3-running-text h6{
      color:#f5f8fa; }
  .bp3-running-text hr{
    border:none;
    border-bottom:1px solid rgba(16, 22, 26, 0.15);
    margin:20px 0; }
    .bp3-dark .bp3-running-text hr{
      border-color:rgba(255, 255, 255, 0.15); }
  .bp3-running-text p{
    margin:0 0 10px;
    padding:0; }

.bp3-text-large{
  font-size:16px; }

.bp3-text-small{
  font-size:12px; }
a{
  color:#106ba3;
  text-decoration:none; }
  a:hover{
    color:#106ba3;
    cursor:pointer;
    text-decoration:underline; }
  a .bp3-icon, a .bp3-icon-standard, a .bp3-icon-large{
    color:inherit; }
  a code,
  .bp3-dark a code{
    color:inherit; }
  .bp3-dark a,
  .bp3-dark a:hover{
    color:#48aff0; }
    .bp3-dark a .bp3-icon, .bp3-dark a .bp3-icon-standard, .bp3-dark a .bp3-icon-large,
    .bp3-dark a:hover .bp3-icon,
    .bp3-dark a:hover .bp3-icon-standard,
    .bp3-dark a:hover .bp3-icon-large{
      color:inherit; }
.bp3-running-text code, .bp3-code{
  font-family:monospace;
  text-transform:none;
  background:rgba(255, 255, 255, 0.7);
  border-radius:3px;
  -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2);
          box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2);
  color:#5c7080;
  font-size:smaller;
  padding:2px 5px; }
  .bp3-dark .bp3-running-text code, .bp3-running-text .bp3-dark code, .bp3-dark .bp3-code{
    background:rgba(16, 22, 26, 0.3);
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4);
    color:#a7b6c2; }
  .bp3-running-text a > code, a > .bp3-code{
    color:#137cbd; }
    .bp3-dark .bp3-running-text a > code, .bp3-running-text .bp3-dark a > code, .bp3-dark a > .bp3-code{
      color:inherit; }

.bp3-running-text pre, .bp3-code-block{
  font-family:monospace;
  text-transform:none;
  background:rgba(255, 255, 255, 0.7);
  border-radius:3px;
  -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.15);
          box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.15);
  color:#182026;
  display:block;
  font-size:13px;
  line-height:1.4;
  margin:10px 0;
  padding:13px 15px 12px;
  word-break:break-all;
  word-wrap:break-word; }
  .bp3-dark .bp3-running-text pre, .bp3-running-text .bp3-dark pre, .bp3-dark .bp3-code-block{
    background:rgba(16, 22, 26, 0.3);
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4);
    color:#f5f8fa; }
  .bp3-running-text pre > code, .bp3-code-block > code{
    background:none;
    -webkit-box-shadow:none;
            box-shadow:none;
    color:inherit;
    font-size:inherit;
    padding:0; }

.bp3-running-text kbd, .bp3-key{
  -webkit-box-align:center;
      -ms-flex-align:center;
          align-items:center;
  background:#ffffff;
  border-radius:3px;
  -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 0 0 rgba(16, 22, 26, 0), 0 1px 1px rgba(16, 22, 26, 0.2);
          box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 0 0 rgba(16, 22, 26, 0), 0 1px 1px rgba(16, 22, 26, 0.2);
  color:#5c7080;
  display:-webkit-inline-box;
  display:-ms-inline-flexbox;
  display:inline-flex;
  font-family:inherit;
  font-size:12px;
  height:24px;
  -webkit-box-pack:center;
      -ms-flex-pack:center;
          justify-content:center;
  line-height:24px;
  min-width:24px;
  padding:3px 6px;
  vertical-align:middle; }
  .bp3-running-text kbd .bp3-icon, .bp3-key .bp3-icon, .bp3-running-text kbd .bp3-icon-standard, .bp3-key .bp3-icon-standard, .bp3-running-text kbd .bp3-icon-large, .bp3-key .bp3-icon-large{
    margin-right:5px; }
  .bp3-dark .bp3-running-text kbd, .bp3-running-text .bp3-dark kbd, .bp3-dark .bp3-key{
    background:#394b59;
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 0 0 rgba(16, 22, 26, 0), 0 1px 1px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 0 0 rgba(16, 22, 26, 0), 0 1px 1px rgba(16, 22, 26, 0.4);
    color:#a7b6c2; }
.bp3-running-text blockquote, .bp3-blockquote{
  border-left:solid 4px rgba(167, 182, 194, 0.5);
  margin:0 0 10px;
  padding:0 20px; }
  .bp3-dark .bp3-running-text blockquote, .bp3-running-text .bp3-dark blockquote, .bp3-dark .bp3-blockquote{
    border-color:rgba(115, 134, 148, 0.5); }
.bp3-running-text ul,
.bp3-running-text ol, .bp3-list{
  margin:10px 0;
  padding-left:30px; }
  .bp3-running-text ul li:not(:last-child), .bp3-running-text ol li:not(:last-child), .bp3-list li:not(:last-child){
    margin-bottom:5px; }
  .bp3-running-text ul ol, .bp3-running-text ol ol, .bp3-list ol,
  .bp3-running-text ul ul,
  .bp3-running-text ol ul,
  .bp3-list ul{
    margin-top:5px; }

.bp3-list-unstyled{
  list-style:none;
  margin:0;
  padding:0; }
  .bp3-list-unstyled li{
    padding:0; }
.bp3-rtl{
  text-align:right; }

.bp3-dark{
  color:#f5f8fa; }

:focus{
  outline:rgba(19, 124, 189, 0.6) auto 2px;
  outline-offset:2px;
  -moz-outline-radius:6px; }

.bp3-focus-disabled :focus{
  outline:none !important; }
  .bp3-focus-disabled :focus ~ .bp3-control-indicator{
    outline:none !important; }

.bp3-alert{
  max-width:400px;
  padding:20px; }

.bp3-alert-body{
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex; }
  .bp3-alert-body .bp3-icon{
    font-size:40px;
    margin-right:20px;
    margin-top:0; }

.bp3-alert-contents{
  word-break:break-word; }

.bp3-alert-footer{
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex;
  -webkit-box-orient:horizontal;
  -webkit-box-direction:reverse;
      -ms-flex-direction:row-reverse;
          flex-direction:row-reverse;
  margin-top:10px; }
  .bp3-alert-footer .bp3-button{
    margin-left:10px; }
.bp3-breadcrumbs{
  -webkit-box-align:center;
      -ms-flex-align:center;
          align-items:center;
  cursor:default;
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex;
  -ms-flex-wrap:wrap;
      flex-wrap:wrap;
  height:30px;
  list-style:none;
  margin:0;
  padding:0; }
  .bp3-breadcrumbs > li{
    -webkit-box-align:center;
        -ms-flex-align:center;
            align-items:center;
    display:-webkit-box;
    display:-ms-flexbox;
    display:flex; }
    .bp3-breadcrumbs > li::after{
      background:url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'%3e%3cpath fill-rule='evenodd' clip-rule='evenodd' d='M10.71 7.29l-4-4a1.003 1.003 0 00-1.42 1.42L8.59 8 5.3 11.29c-.19.18-.3.43-.3.71a1.003 1.003 0 001.71.71l4-4c.18-.18.29-.43.29-.71 0-.28-.11-.53-.29-.71z' fill='%235C7080'/%3e%3c/svg%3e");
      content:"";
      display:block;
      height:16px;
      margin:0 5px;
      width:16px; }
    .bp3-breadcrumbs > li:last-of-type::after{
      display:none; }

.bp3-breadcrumb,
.bp3-breadcrumb-current,
.bp3-breadcrumbs-collapsed{
  -webkit-box-align:center;
      -ms-flex-align:center;
          align-items:center;
  display:-webkit-inline-box;
  display:-ms-inline-flexbox;
  display:inline-flex;
  font-size:16px; }

.bp3-breadcrumb,
.bp3-breadcrumbs-collapsed{
  color:#5c7080; }

.bp3-breadcrumb:hover{
  text-decoration:none; }

.bp3-breadcrumb.bp3-disabled{
  color:rgba(92, 112, 128, 0.6);
  cursor:not-allowed; }

.bp3-breadcrumb .bp3-icon{
  margin-right:5px; }

.bp3-breadcrumb-current{
  color:inherit;
  font-weight:600; }
  .bp3-breadcrumb-current .bp3-input{
    font-size:inherit;
    font-weight:inherit;
    vertical-align:baseline; }

.bp3-breadcrumbs-collapsed{
  background:#ced9e0;
  border:none;
  border-radius:3px;
  cursor:pointer;
  margin-right:2px;
  padding:1px 5px;
  vertical-align:text-bottom; }
  .bp3-breadcrumbs-collapsed::before{
    background:url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'%3e%3cg fill='%235C7080'%3e%3ccircle cx='2' cy='8.03' r='2'/%3e%3ccircle cx='14' cy='8.03' r='2'/%3e%3ccircle cx='8' cy='8.03' r='2'/%3e%3c/g%3e%3c/svg%3e") center no-repeat;
    content:"";
    display:block;
    height:16px;
    width:16px; }
  .bp3-breadcrumbs-collapsed:hover{
    background:#bfccd6;
    color:#182026;
    text-decoration:none; }

.bp3-dark .bp3-breadcrumb,
.bp3-dark .bp3-breadcrumbs-collapsed{
  color:#a7b6c2; }

.bp3-dark .bp3-breadcrumbs > li::after{
  color:#a7b6c2; }

.bp3-dark .bp3-breadcrumb.bp3-disabled{
  color:rgba(167, 182, 194, 0.6); }

.bp3-dark .bp3-breadcrumb-current{
  color:#f5f8fa; }

.bp3-dark .bp3-breadcrumbs-collapsed{
  background:rgba(16, 22, 26, 0.4); }
  .bp3-dark .bp3-breadcrumbs-collapsed:hover{
    background:rgba(16, 22, 26, 0.6);
    color:#f5f8fa; }
.bp3-button{
  display:-webkit-inline-box;
  display:-ms-inline-flexbox;
  display:inline-flex;
  -webkit-box-orient:horizontal;
  -webkit-box-direction:normal;
      -ms-flex-direction:row;
          flex-direction:row;
  -webkit-box-align:center;
      -ms-flex-align:center;
          align-items:center;
  border:none;
  border-radius:3px;
  cursor:pointer;
  font-size:14px;
  -webkit-box-pack:center;
      -ms-flex-pack:center;
          justify-content:center;
  padding:5px 10px;
  text-align:left;
  vertical-align:middle;
  min-height:30px;
  min-width:30px; }
  .bp3-button > *{
    -webkit-box-flex:0;
        -ms-flex-positive:0;
            flex-grow:0;
    -ms-flex-negative:0;
        flex-shrink:0; }
  .bp3-button > .bp3-fill{
    -webkit-box-flex:1;
        -ms-flex-positive:1;
            flex-grow:1;
    -ms-flex-negative:1;
        flex-shrink:1; }
  .bp3-button::before,
  .bp3-button > *{
    margin-right:7px; }
  .bp3-button:empty::before,
  .bp3-button > :last-child{
    margin-right:0; }
  .bp3-button:empty{
    padding:0 !important; }
  .bp3-button:disabled, .bp3-button.bp3-disabled{
    cursor:not-allowed; }
  .bp3-button.bp3-fill{
    display:-webkit-box;
    display:-ms-flexbox;
    display:flex;
    width:100%; }
  .bp3-button.bp3-align-right,
  .bp3-align-right .bp3-button{
    text-align:right; }
  .bp3-button.bp3-align-left,
  .bp3-align-left .bp3-button{
    text-align:left; }
  .bp3-button:not([class*="bp3-intent-"]){
    background-color:#f5f8fa;
    background-image:-webkit-gradient(linear, left top, left bottom, from(rgba(255, 255, 255, 0.8)), to(rgba(255, 255, 255, 0)));
    background-image:linear-gradient(to bottom, rgba(255, 255, 255, 0.8), rgba(255, 255, 255, 0));
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 -1px 0 rgba(16, 22, 26, 0.1);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 -1px 0 rgba(16, 22, 26, 0.1);
    color:#182026; }
    .bp3-button:not([class*="bp3-intent-"]):hover{
      background-clip:padding-box;
      background-color:#ebf1f5;
      -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 -1px 0 rgba(16, 22, 26, 0.1);
              box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 -1px 0 rgba(16, 22, 26, 0.1); }
    .bp3-button:not([class*="bp3-intent-"]):active, .bp3-button:not([class*="bp3-intent-"]).bp3-active{
      background-color:#d8e1e8;
      background-image:none;
      -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 1px 2px rgba(16, 22, 26, 0.2);
              box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 1px 2px rgba(16, 22, 26, 0.2); }
    .bp3-button:not([class*="bp3-intent-"]):disabled, .bp3-button:not([class*="bp3-intent-"]).bp3-disabled{
      background-color:rgba(206, 217, 224, 0.5);
      background-image:none;
      -webkit-box-shadow:none;
              box-shadow:none;
      color:rgba(92, 112, 128, 0.6);
      cursor:not-allowed;
      outline:none; }
      .bp3-button:not([class*="bp3-intent-"]):disabled.bp3-active, .bp3-button:not([class*="bp3-intent-"]):disabled.bp3-active:hover, .bp3-button:not([class*="bp3-intent-"]).bp3-disabled.bp3-active, .bp3-button:not([class*="bp3-intent-"]).bp3-disabled.bp3-active:hover{
        background:rgba(206, 217, 224, 0.7); }
  .bp3-button.bp3-intent-primary{
    background-color:#137cbd;
    background-image:-webkit-gradient(linear, left top, left bottom, from(rgba(255, 255, 255, 0.1)), to(rgba(255, 255, 255, 0)));
    background-image:linear-gradient(to bottom, rgba(255, 255, 255, 0.1), rgba(255, 255, 255, 0));
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 -1px 0 rgba(16, 22, 26, 0.2);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 -1px 0 rgba(16, 22, 26, 0.2);
    color:#ffffff; }
    .bp3-button.bp3-intent-primary:hover, .bp3-button.bp3-intent-primary:active, .bp3-button.bp3-intent-primary.bp3-active{
      color:#ffffff; }
    .bp3-button.bp3-intent-primary:hover{
      background-color:#106ba3;
      -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 -1px 0 rgba(16, 22, 26, 0.2);
              box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 -1px 0 rgba(16, 22, 26, 0.2); }
    .bp3-button.bp3-intent-primary:active, .bp3-button.bp3-intent-primary.bp3-active{
      background-color:#0e5a8a;
      background-image:none;
      -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 1px 2px rgba(16, 22, 26, 0.2);
              box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 1px 2px rgba(16, 22, 26, 0.2); }
    .bp3-button.bp3-intent-primary:disabled, .bp3-button.bp3-intent-primary.bp3-disabled{
      background-color:rgba(19, 124, 189, 0.5);
      background-image:none;
      border-color:transparent;
      -webkit-box-shadow:none;
              box-shadow:none;
      color:rgba(255, 255, 255, 0.6); }
  .bp3-button.bp3-intent-success{
    background-color:#0f9960;
    background-image:-webkit-gradient(linear, left top, left bottom, from(rgba(255, 255, 255, 0.1)), to(rgba(255, 255, 255, 0)));
    background-image:linear-gradient(to bottom, rgba(255, 255, 255, 0.1), rgba(255, 255, 255, 0));
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 -1px 0 rgba(16, 22, 26, 0.2);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 -1px 0 rgba(16, 22, 26, 0.2);
    color:#ffffff; }
    .bp3-button.bp3-intent-success:hover, .bp3-button.bp3-intent-success:active, .bp3-button.bp3-intent-success.bp3-active{
      color:#ffffff; }
    .bp3-button.bp3-intent-success:hover{
      background-color:#0d8050;
      -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 -1px 0 rgba(16, 22, 26, 0.2);
              box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 -1px 0 rgba(16, 22, 26, 0.2); }
    .bp3-button.bp3-intent-success:active, .bp3-button.bp3-intent-success.bp3-active{
      background-color:#0a6640;
      background-image:none;
      -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 1px 2px rgba(16, 22, 26, 0.2);
              box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 1px 2px rgba(16, 22, 26, 0.2); }
    .bp3-button.bp3-intent-success:disabled, .bp3-button.bp3-intent-success.bp3-disabled{
      background-color:rgba(15, 153, 96, 0.5);
      background-image:none;
      border-color:transparent;
      -webkit-box-shadow:none;
              box-shadow:none;
      color:rgba(255, 255, 255, 0.6); }
  .bp3-button.bp3-intent-warning{
    background-color:#d9822b;
    background-image:-webkit-gradient(linear, left top, left bottom, from(rgba(255, 255, 255, 0.1)), to(rgba(255, 255, 255, 0)));
    background-image:linear-gradient(to bottom, rgba(255, 255, 255, 0.1), rgba(255, 255, 255, 0));
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 -1px 0 rgba(16, 22, 26, 0.2);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 -1px 0 rgba(16, 22, 26, 0.2);
    color:#ffffff; }
    .bp3-button.bp3-intent-warning:hover, .bp3-button.bp3-intent-warning:active, .bp3-button.bp3-intent-warning.bp3-active{
      color:#ffffff; }
    .bp3-button.bp3-intent-warning:hover{
      background-color:#bf7326;
      -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 -1px 0 rgba(16, 22, 26, 0.2);
              box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 -1px 0 rgba(16, 22, 26, 0.2); }
    .bp3-button.bp3-intent-warning:active, .bp3-button.bp3-intent-warning.bp3-active{
      background-color:#a66321;
      background-image:none;
      -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 1px 2px rgba(16, 22, 26, 0.2);
              box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 1px 2px rgba(16, 22, 26, 0.2); }
    .bp3-button.bp3-intent-warning:disabled, .bp3-button.bp3-intent-warning.bp3-disabled{
      background-color:rgba(217, 130, 43, 0.5);
      background-image:none;
      border-color:transparent;
      -webkit-box-shadow:none;
              box-shadow:none;
      color:rgba(255, 255, 255, 0.6); }
  .bp3-button.bp3-intent-danger{
    background-color:#db3737;
    background-image:-webkit-gradient(linear, left top, left bottom, from(rgba(255, 255, 255, 0.1)), to(rgba(255, 255, 255, 0)));
    background-image:linear-gradient(to bottom, rgba(255, 255, 255, 0.1), rgba(255, 255, 255, 0));
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 -1px 0 rgba(16, 22, 26, 0.2);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 -1px 0 rgba(16, 22, 26, 0.2);
    color:#ffffff; }
    .bp3-button.bp3-intent-danger:hover, .bp3-button.bp3-intent-danger:active, .bp3-button.bp3-intent-danger.bp3-active{
      color:#ffffff; }
    .bp3-button.bp3-intent-danger:hover{
      background-color:#c23030;
      -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 -1px 0 rgba(16, 22, 26, 0.2);
              box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 -1px 0 rgba(16, 22, 26, 0.2); }
    .bp3-button.bp3-intent-danger:active, .bp3-button.bp3-intent-danger.bp3-active{
      background-color:#a82a2a;
      background-image:none;
      -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 1px 2px rgba(16, 22, 26, 0.2);
              box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 1px 2px rgba(16, 22, 26, 0.2); }
    .bp3-button.bp3-intent-danger:disabled, .bp3-button.bp3-intent-danger.bp3-disabled{
      background-color:rgba(219, 55, 55, 0.5);
      background-image:none;
      border-color:transparent;
      -webkit-box-shadow:none;
              box-shadow:none;
      color:rgba(255, 255, 255, 0.6); }
  .bp3-button[class*="bp3-intent-"] .bp3-button-spinner .bp3-spinner-head{
    stroke:#ffffff; }
  .bp3-button.bp3-large,
  .bp3-large .bp3-button{
    min-height:40px;
    min-width:40px;
    font-size:16px;
    padding:5px 15px; }
    .bp3-button.bp3-large::before,
    .bp3-button.bp3-large > *,
    .bp3-large .bp3-button::before,
    .bp3-large .bp3-button > *{
      margin-right:10px; }
    .bp3-button.bp3-large:empty::before,
    .bp3-button.bp3-large > :last-child,
    .bp3-large .bp3-button:empty::before,
    .bp3-large .bp3-button > :last-child{
      margin-right:0; }
  .bp3-button.bp3-small,
  .bp3-small .bp3-button{
    min-height:24px;
    min-width:24px;
    padding:0 7px; }
  .bp3-button.bp3-loading{
    position:relative; }
    .bp3-button.bp3-loading[class*="bp3-icon-"]::before{
      visibility:hidden; }
    .bp3-button.bp3-loading .bp3-button-spinner{
      margin:0;
      position:absolute; }
    .bp3-button.bp3-loading > :not(.bp3-button-spinner){
      visibility:hidden; }
  .bp3-button[class*="bp3-icon-"]::before{
    font-family:"Icons16", sans-serif;
    font-size:16px;
    font-style:normal;
    font-weight:400;
    line-height:1;
    -moz-osx-font-smoothing:grayscale;
    -webkit-font-smoothing:antialiased;
    color:#5c7080; }
  .bp3-button .bp3-icon, .bp3-button .bp3-icon-standard, .bp3-button .bp3-icon-large{
    color:#5c7080; }
    .bp3-button .bp3-icon.bp3-align-right, .bp3-button .bp3-icon-standard.bp3-align-right, .bp3-button .bp3-icon-large.bp3-align-right{
      margin-left:7px; }
  .bp3-button .bp3-icon:first-child:last-child,
  .bp3-button .bp3-spinner + .bp3-icon:last-child{
    margin:0 -7px; }
  .bp3-dark .bp3-button:not([class*="bp3-intent-"]){
    background-color:#394b59;
    background-image:-webkit-gradient(linear, left top, left bottom, from(rgba(255, 255, 255, 0.05)), to(rgba(255, 255, 255, 0)));
    background-image:linear-gradient(to bottom, rgba(255, 255, 255, 0.05), rgba(255, 255, 255, 0));
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4);
    color:#f5f8fa; }
    .bp3-dark .bp3-button:not([class*="bp3-intent-"]):hover, .bp3-dark .bp3-button:not([class*="bp3-intent-"]):active, .bp3-dark .bp3-button:not([class*="bp3-intent-"]).bp3-active{
      color:#f5f8fa; }
    .bp3-dark .bp3-button:not([class*="bp3-intent-"]):hover{
      background-color:#30404d;
      -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4);
              box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4); }
    .bp3-dark .bp3-button:not([class*="bp3-intent-"]):active, .bp3-dark .bp3-button:not([class*="bp3-intent-"]).bp3-active{
      background-color:#202b33;
      background-image:none;
      -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.6), inset 0 1px 2px rgba(16, 22, 26, 0.2);
              box-shadow:0 0 0 1px rgba(16, 22, 26, 0.6), inset 0 1px 2px rgba(16, 22, 26, 0.2); }
    .bp3-dark .bp3-button:not([class*="bp3-intent-"]):disabled, .bp3-dark .bp3-button:not([class*="bp3-intent-"]).bp3-disabled{
      background-color:rgba(57, 75, 89, 0.5);
      background-image:none;
      -webkit-box-shadow:none;
              box-shadow:none;
      color:rgba(167, 182, 194, 0.6); }
      .bp3-dark .bp3-button:not([class*="bp3-intent-"]):disabled.bp3-active, .bp3-dark .bp3-button:not([class*="bp3-intent-"]).bp3-disabled.bp3-active{
        background:rgba(57, 75, 89, 0.7); }
    .bp3-dark .bp3-button:not([class*="bp3-intent-"]) .bp3-button-spinner .bp3-spinner-head{
      background:rgba(16, 22, 26, 0.5);
      stroke:#8a9ba8; }
    .bp3-dark .bp3-button:not([class*="bp3-intent-"])[class*="bp3-icon-"]::before{
      color:#a7b6c2; }
    .bp3-dark .bp3-button:not([class*="bp3-intent-"]) .bp3-icon, .bp3-dark .bp3-button:not([class*="bp3-intent-"]) .bp3-icon-standard, .bp3-dark .bp3-button:not([class*="bp3-intent-"]) .bp3-icon-large{
      color:#a7b6c2; }
  .bp3-dark .bp3-button[class*="bp3-intent-"]{
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4); }
    .bp3-dark .bp3-button[class*="bp3-intent-"]:hover{
      -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4);
              box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4); }
    .bp3-dark .bp3-button[class*="bp3-intent-"]:active, .bp3-dark .bp3-button[class*="bp3-intent-"].bp3-active{
      -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 1px 2px rgba(16, 22, 26, 0.2);
              box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 1px 2px rgba(16, 22, 26, 0.2); }
    .bp3-dark .bp3-button[class*="bp3-intent-"]:disabled, .bp3-dark .bp3-button[class*="bp3-intent-"].bp3-disabled{
      background-image:none;
      -webkit-box-shadow:none;
              box-shadow:none;
      color:rgba(255, 255, 255, 0.3); }
    .bp3-dark .bp3-button[class*="bp3-intent-"] .bp3-button-spinner .bp3-spinner-head{
      stroke:#8a9ba8; }
  .bp3-button:disabled::before,
  .bp3-button:disabled .bp3-icon, .bp3-button:disabled .bp3-icon-standard, .bp3-button:disabled .bp3-icon-large, .bp3-button.bp3-disabled::before,
  .bp3-button.bp3-disabled .bp3-icon, .bp3-button.bp3-disabled .bp3-icon-standard, .bp3-button.bp3-disabled .bp3-icon-large, .bp3-button[class*="bp3-intent-"]::before,
  .bp3-button[class*="bp3-intent-"] .bp3-icon, .bp3-button[class*="bp3-intent-"] .bp3-icon-standard, .bp3-button[class*="bp3-intent-"] .bp3-icon-large{
    color:inherit !important; }
  .bp3-button.bp3-minimal{
    background:none;
    -webkit-box-shadow:none;
            box-shadow:none; }
    .bp3-button.bp3-minimal:hover{
      background:rgba(167, 182, 194, 0.3);
      -webkit-box-shadow:none;
              box-shadow:none;
      color:#182026;
      text-decoration:none; }
    .bp3-button.bp3-minimal:active, .bp3-button.bp3-minimal.bp3-active{
      background:rgba(115, 134, 148, 0.3);
      -webkit-box-shadow:none;
              box-shadow:none;
      color:#182026; }
    .bp3-button.bp3-minimal:disabled, .bp3-button.bp3-minimal:disabled:hover, .bp3-button.bp3-minimal.bp3-disabled, .bp3-button.bp3-minimal.bp3-disabled:hover{
      background:none;
      color:rgba(92, 112, 128, 0.6);
      cursor:not-allowed; }
      .bp3-button.bp3-minimal:disabled.bp3-active, .bp3-button.bp3-minimal:disabled:hover.bp3-active, .bp3-button.bp3-minimal.bp3-disabled.bp3-active, .bp3-button.bp3-minimal.bp3-disabled:hover.bp3-active{
        background:rgba(115, 134, 148, 0.3); }
    .bp3-dark .bp3-button.bp3-minimal{
      background:none;
      -webkit-box-shadow:none;
              box-shadow:none;
      color:inherit; }
      .bp3-dark .bp3-button.bp3-minimal:hover, .bp3-dark .bp3-button.bp3-minimal:active, .bp3-dark .bp3-button.bp3-minimal.bp3-active{
        background:none;
        -webkit-box-shadow:none;
                box-shadow:none; }
      .bp3-dark .bp3-button.bp3-minimal:hover{
        background:rgba(138, 155, 168, 0.15); }
      .bp3-dark .bp3-button.bp3-minimal:active, .bp3-dark .bp3-button.bp3-minimal.bp3-active{
        background:rgba(138, 155, 168, 0.3);
        color:#f5f8fa; }
      .bp3-dark .bp3-button.bp3-minimal:disabled, .bp3-dark .bp3-button.bp3-minimal:disabled:hover, .bp3-dark .bp3-button.bp3-minimal.bp3-disabled, .bp3-dark .bp3-button.bp3-minimal.bp3-disabled:hover{
        background:none;
        color:rgba(167, 182, 194, 0.6);
        cursor:not-allowed; }
        .bp3-dark .bp3-button.bp3-minimal:disabled.bp3-active, .bp3-dark .bp3-button.bp3-minimal:disabled:hover.bp3-active, .bp3-dark .bp3-button.bp3-minimal.bp3-disabled.bp3-active, .bp3-dark .bp3-button.bp3-minimal.bp3-disabled:hover.bp3-active{
          background:rgba(138, 155, 168, 0.3); }
    .bp3-button.bp3-minimal.bp3-intent-primary{
      color:#106ba3; }
      .bp3-button.bp3-minimal.bp3-intent-primary:hover, .bp3-button.bp3-minimal.bp3-intent-primary:active, .bp3-button.bp3-minimal.bp3-intent-primary.bp3-active{
        background:none;
        -webkit-box-shadow:none;
                box-shadow:none;
        color:#106ba3; }
      .bp3-button.bp3-minimal.bp3-intent-primary:hover{
        background:rgba(19, 124, 189, 0.15);
        color:#106ba3; }
      .bp3-button.bp3-minimal.bp3-intent-primary:active, .bp3-button.bp3-minimal.bp3-intent-primary.bp3-active{
        background:rgba(19, 124, 189, 0.3);
        color:#106ba3; }
      .bp3-button.bp3-minimal.bp3-intent-primary:disabled, .bp3-button.bp3-minimal.bp3-intent-primary.bp3-disabled{
        background:none;
        color:rgba(16, 107, 163, 0.5); }
        .bp3-button.bp3-minimal.bp3-intent-primary:disabled.bp3-active, .bp3-button.bp3-minimal.bp3-intent-primary.bp3-disabled.bp3-active{
          background:rgba(19, 124, 189, 0.3); }
      .bp3-button.bp3-minimal.bp3-intent-primary .bp3-button-spinner .bp3-spinner-head{
        stroke:#106ba3; }
      .bp3-dark .bp3-button.bp3-minimal.bp3-intent-primary{
        color:#48aff0; }
        .bp3-dark .bp3-button.bp3-minimal.bp3-intent-primary:hover{
          background:rgba(19, 124, 189, 0.2);
          color:#48aff0; }
        .bp3-dark .bp3-button.bp3-minimal.bp3-intent-primary:active, .bp3-dark .bp3-button.bp3-minimal.bp3-intent-primary.bp3-active{
          background:rgba(19, 124, 189, 0.3);
          color:#48aff0; }
        .bp3-dark .bp3-button.bp3-minimal.bp3-intent-primary:disabled, .bp3-dark .bp3-button.bp3-minimal.bp3-intent-primary.bp3-disabled{
          background:none;
          color:rgba(72, 175, 240, 0.5); }
          .bp3-dark .bp3-button.bp3-minimal.bp3-intent-primary:disabled.bp3-active, .bp3-dark .bp3-button.bp3-minimal.bp3-intent-primary.bp3-disabled.bp3-active{
            background:rgba(19, 124, 189, 0.3); }
    .bp3-button.bp3-minimal.bp3-intent-success{
      color:#0d8050; }
      .bp3-button.bp3-minimal.bp3-intent-success:hover, .bp3-button.bp3-minimal.bp3-intent-success:active, .bp3-button.bp3-minimal.bp3-intent-success.bp3-active{
        background:none;
        -webkit-box-shadow:none;
                box-shadow:none;
        color:#0d8050; }
      .bp3-button.bp3-minimal.bp3-intent-success:hover{
        background:rgba(15, 153, 96, 0.15);
        color:#0d8050; }
      .bp3-button.bp3-minimal.bp3-intent-success:active, .bp3-button.bp3-minimal.bp3-intent-success.bp3-active{
        background:rgba(15, 153, 96, 0.3);
        color:#0d8050; }
      .bp3-button.bp3-minimal.bp3-intent-success:disabled, .bp3-button.bp3-minimal.bp3-intent-success.bp3-disabled{
        background:none;
        color:rgba(13, 128, 80, 0.5); }
        .bp3-button.bp3-minimal.bp3-intent-success:disabled.bp3-active, .bp3-button.bp3-minimal.bp3-intent-success.bp3-disabled.bp3-active{
          background:rgba(15, 153, 96, 0.3); }
      .bp3-button.bp3-minimal.bp3-intent-success .bp3-button-spinner .bp3-spinner-head{
        stroke:#0d8050; }
      .bp3-dark .bp3-button.bp3-minimal.bp3-intent-success{
        color:#3dcc91; }
        .bp3-dark .bp3-button.bp3-minimal.bp3-intent-success:hover{
          background:rgba(15, 153, 96, 0.2);
          color:#3dcc91; }
        .bp3-dark .bp3-button.bp3-minimal.bp3-intent-success:active, .bp3-dark .bp3-button.bp3-minimal.bp3-intent-success.bp3-active{
          background:rgba(15, 153, 96, 0.3);
          color:#3dcc91; }
        .bp3-dark .bp3-button.bp3-minimal.bp3-intent-success:disabled, .bp3-dark .bp3-button.bp3-minimal.bp3-intent-success.bp3-disabled{
          background:none;
          color:rgba(61, 204, 145, 0.5); }
          .bp3-dark .bp3-button.bp3-minimal.bp3-intent-success:disabled.bp3-active, .bp3-dark .bp3-button.bp3-minimal.bp3-intent-success.bp3-disabled.bp3-active{
            background:rgba(15, 153, 96, 0.3); }
    .bp3-button.bp3-minimal.bp3-intent-warning{
      color:#bf7326; }
      .bp3-button.bp3-minimal.bp3-intent-warning:hover, .bp3-button.bp3-minimal.bp3-intent-warning:active, .bp3-button.bp3-minimal.bp3-intent-warning.bp3-active{
        background:none;
        -webkit-box-shadow:none;
                box-shadow:none;
        color:#bf7326; }
      .bp3-button.bp3-minimal.bp3-intent-warning:hover{
        background:rgba(217, 130, 43, 0.15);
        color:#bf7326; }
      .bp3-button.bp3-minimal.bp3-intent-warning:active, .bp3-button.bp3-minimal.bp3-intent-warning.bp3-active{
        background:rgba(217, 130, 43, 0.3);
        color:#bf7326; }
      .bp3-button.bp3-minimal.bp3-intent-warning:disabled, .bp3-button.bp3-minimal.bp3-intent-warning.bp3-disabled{
        background:none;
        color:rgba(191, 115, 38, 0.5); }
        .bp3-button.bp3-minimal.bp3-intent-warning:disabled.bp3-active, .bp3-button.bp3-minimal.bp3-intent-warning.bp3-disabled.bp3-active{
          background:rgba(217, 130, 43, 0.3); }
      .bp3-button.bp3-minimal.bp3-intent-warning .bp3-button-spinner .bp3-spinner-head{
        stroke:#bf7326; }
      .bp3-dark .bp3-button.bp3-minimal.bp3-intent-warning{
        color:#ffb366; }
        .bp3-dark .bp3-button.bp3-minimal.bp3-intent-warning:hover{
          background:rgba(217, 130, 43, 0.2);
          color:#ffb366; }
        .bp3-dark .bp3-button.bp3-minimal.bp3-intent-warning:active, .bp3-dark .bp3-button.bp3-minimal.bp3-intent-warning.bp3-active{
          background:rgba(217, 130, 43, 0.3);
          color:#ffb366; }
        .bp3-dark .bp3-button.bp3-minimal.bp3-intent-warning:disabled, .bp3-dark .bp3-button.bp3-minimal.bp3-intent-warning.bp3-disabled{
          background:none;
          color:rgba(255, 179, 102, 0.5); }
          .bp3-dark .bp3-button.bp3-minimal.bp3-intent-warning:disabled.bp3-active, .bp3-dark .bp3-button.bp3-minimal.bp3-intent-warning.bp3-disabled.bp3-active{
            background:rgba(217, 130, 43, 0.3); }
    .bp3-button.bp3-minimal.bp3-intent-danger{
      color:#c23030; }
      .bp3-button.bp3-minimal.bp3-intent-danger:hover, .bp3-button.bp3-minimal.bp3-intent-danger:active, .bp3-button.bp3-minimal.bp3-intent-danger.bp3-active{
        background:none;
        -webkit-box-shadow:none;
                box-shadow:none;
        color:#c23030; }
      .bp3-button.bp3-minimal.bp3-intent-danger:hover{
        background:rgba(219, 55, 55, 0.15);
        color:#c23030; }
      .bp3-button.bp3-minimal.bp3-intent-danger:active, .bp3-button.bp3-minimal.bp3-intent-danger.bp3-active{
        background:rgba(219, 55, 55, 0.3);
        color:#c23030; }
      .bp3-button.bp3-minimal.bp3-intent-danger:disabled, .bp3-button.bp3-minimal.bp3-intent-danger.bp3-disabled{
        background:none;
        color:rgba(194, 48, 48, 0.5); }
        .bp3-button.bp3-minimal.bp3-intent-danger:disabled.bp3-active, .bp3-button.bp3-minimal.bp3-intent-danger.bp3-disabled.bp3-active{
          background:rgba(219, 55, 55, 0.3); }
      .bp3-button.bp3-minimal.bp3-intent-danger .bp3-button-spinner .bp3-spinner-head{
        stroke:#c23030; }
      .bp3-dark .bp3-button.bp3-minimal.bp3-intent-danger{
        color:#ff7373; }
        .bp3-dark .bp3-button.bp3-minimal.bp3-intent-danger:hover{
          background:rgba(219, 55, 55, 0.2);
          color:#ff7373; }
        .bp3-dark .bp3-button.bp3-minimal.bp3-intent-danger:active, .bp3-dark .bp3-button.bp3-minimal.bp3-intent-danger.bp3-active{
          background:rgba(219, 55, 55, 0.3);
          color:#ff7373; }
        .bp3-dark .bp3-button.bp3-minimal.bp3-intent-danger:disabled, .bp3-dark .bp3-button.bp3-minimal.bp3-intent-danger.bp3-disabled{
          background:none;
          color:rgba(255, 115, 115, 0.5); }
          .bp3-dark .bp3-button.bp3-minimal.bp3-intent-danger:disabled.bp3-active, .bp3-dark .bp3-button.bp3-minimal.bp3-intent-danger.bp3-disabled.bp3-active{
            background:rgba(219, 55, 55, 0.3); }
  .bp3-button.bp3-outlined{
    background:none;
    -webkit-box-shadow:none;
            box-shadow:none;
    border:1px solid rgba(24, 32, 38, 0.2);
    -webkit-box-sizing:border-box;
            box-sizing:border-box; }
    .bp3-button.bp3-outlined:hover{
      background:rgba(167, 182, 194, 0.3);
      -webkit-box-shadow:none;
              box-shadow:none;
      color:#182026;
      text-decoration:none; }
    .bp3-button.bp3-outlined:active, .bp3-button.bp3-outlined.bp3-active{
      background:rgba(115, 134, 148, 0.3);
      -webkit-box-shadow:none;
              box-shadow:none;
      color:#182026; }
    .bp3-button.bp3-outlined:disabled, .bp3-button.bp3-outlined:disabled:hover, .bp3-button.bp3-outlined.bp3-disabled, .bp3-button.bp3-outlined.bp3-disabled:hover{
      background:none;
      color:rgba(92, 112, 128, 0.6);
      cursor:not-allowed; }
      .bp3-button.bp3-outlined:disabled.bp3-active, .bp3-button.bp3-outlined:disabled:hover.bp3-active, .bp3-button.bp3-outlined.bp3-disabled.bp3-active, .bp3-button.bp3-outlined.bp3-disabled:hover.bp3-active{
        background:rgba(115, 134, 148, 0.3); }
    .bp3-dark .bp3-button.bp3-outlined{
      background:none;
      -webkit-box-shadow:none;
              box-shadow:none;
      color:inherit; }
      .bp3-dark .bp3-button.bp3-outlined:hover, .bp3-dark .bp3-button.bp3-outlined:active, .bp3-dark .bp3-button.bp3-outlined.bp3-active{
        background:none;
        -webkit-box-shadow:none;
                box-shadow:none; }
      .bp3-dark .bp3-button.bp3-outlined:hover{
        background:rgba(138, 155, 168, 0.15); }
      .bp3-dark .bp3-button.bp3-outlined:active, .bp3-dark .bp3-button.bp3-outlined.bp3-active{
        background:rgba(138, 155, 168, 0.3);
        color:#f5f8fa; }
      .bp3-dark .bp3-button.bp3-outlined:disabled, .bp3-dark .bp3-button.bp3-outlined:disabled:hover, .bp3-dark .bp3-button.bp3-outlined.bp3-disabled, .bp3-dark .bp3-button.bp3-outlined.bp3-disabled:hover{
        background:none;
        color:rgba(167, 182, 194, 0.6);
        cursor:not-allowed; }
        .bp3-dark .bp3-button.bp3-outlined:disabled.bp3-active, .bp3-dark .bp3-button.bp3-outlined:disabled:hover.bp3-active, .bp3-dark .bp3-button.bp3-outlined.bp3-disabled.bp3-active, .bp3-dark .bp3-button.bp3-outlined.bp3-disabled:hover.bp3-active{
          background:rgba(138, 155, 168, 0.3); }
    .bp3-button.bp3-outlined.bp3-intent-primary{
      color:#106ba3; }
      .bp3-button.bp3-outlined.bp3-intent-primary:hover, .bp3-button.bp3-outlined.bp3-intent-primary:active, .bp3-button.bp3-outlined.bp3-intent-primary.bp3-active{
        background:none;
        -webkit-box-shadow:none;
                box-shadow:none;
        color:#106ba3; }
      .bp3-button.bp3-outlined.bp3-intent-primary:hover{
        background:rgba(19, 124, 189, 0.15);
        color:#106ba3; }
      .bp3-button.bp3-outlined.bp3-intent-primary:active, .bp3-button.bp3-outlined.bp3-intent-primary.bp3-active{
        background:rgba(19, 124, 189, 0.3);
        color:#106ba3; }
      .bp3-button.bp3-outlined.bp3-intent-primary:disabled, .bp3-button.bp3-outlined.bp3-intent-primary.bp3-disabled{
        background:none;
        color:rgba(16, 107, 163, 0.5); }
        .bp3-button.bp3-outlined.bp3-intent-primary:disabled.bp3-active, .bp3-button.bp3-outlined.bp3-intent-primary.bp3-disabled.bp3-active{
          background:rgba(19, 124, 189, 0.3); }
      .bp3-button.bp3-outlined.bp3-intent-primary .bp3-button-spinner .bp3-spinner-head{
        stroke:#106ba3; }
      .bp3-dark .bp3-button.bp3-outlined.bp3-intent-primary{
        color:#48aff0; }
        .bp3-dark .bp3-button.bp3-outlined.bp3-intent-primary:hover{
          background:rgba(19, 124, 189, 0.2);
          color:#48aff0; }
        .bp3-dark .bp3-button.bp3-outlined.bp3-intent-primary:active, .bp3-dark .bp3-button.bp3-outlined.bp3-intent-primary.bp3-active{
          background:rgba(19, 124, 189, 0.3);
          color:#48aff0; }
        .bp3-dark .bp3-button.bp3-outlined.bp3-intent-primary:disabled, .bp3-dark .bp3-button.bp3-outlined.bp3-intent-primary.bp3-disabled{
          background:none;
          color:rgba(72, 175, 240, 0.5); }
          .bp3-dark .bp3-button.bp3-outlined.bp3-intent-primary:disabled.bp3-active, .bp3-dark .bp3-button.bp3-outlined.bp3-intent-primary.bp3-disabled.bp3-active{
            background:rgba(19, 124, 189, 0.3); }
    .bp3-button.bp3-outlined.bp3-intent-success{
      color:#0d8050; }
      .bp3-button.bp3-outlined.bp3-intent-success:hover, .bp3-button.bp3-outlined.bp3-intent-success:active, .bp3-button.bp3-outlined.bp3-intent-success.bp3-active{
        background:none;
        -webkit-box-shadow:none;
                box-shadow:none;
        color:#0d8050; }
      .bp3-button.bp3-outlined.bp3-intent-success:hover{
        background:rgba(15, 153, 96, 0.15);
        color:#0d8050; }
      .bp3-button.bp3-outlined.bp3-intent-success:active, .bp3-button.bp3-outlined.bp3-intent-success.bp3-active{
        background:rgba(15, 153, 96, 0.3);
        color:#0d8050; }
      .bp3-button.bp3-outlined.bp3-intent-success:disabled, .bp3-button.bp3-outlined.bp3-intent-success.bp3-disabled{
        background:none;
        color:rgba(13, 128, 80, 0.5); }
        .bp3-button.bp3-outlined.bp3-intent-success:disabled.bp3-active, .bp3-button.bp3-outlined.bp3-intent-success.bp3-disabled.bp3-active{
          background:rgba(15, 153, 96, 0.3); }
      .bp3-button.bp3-outlined.bp3-intent-success .bp3-button-spinner .bp3-spinner-head{
        stroke:#0d8050; }
      .bp3-dark .bp3-button.bp3-outlined.bp3-intent-success{
        color:#3dcc91; }
        .bp3-dark .bp3-button.bp3-outlined.bp3-intent-success:hover{
          background:rgba(15, 153, 96, 0.2);
          color:#3dcc91; }
        .bp3-dark .bp3-button.bp3-outlined.bp3-intent-success:active, .bp3-dark .bp3-button.bp3-outlined.bp3-intent-success.bp3-active{
          background:rgba(15, 153, 96, 0.3);
          color:#3dcc91; }
        .bp3-dark .bp3-button.bp3-outlined.bp3-intent-success:disabled, .bp3-dark .bp3-button.bp3-outlined.bp3-intent-success.bp3-disabled{
          background:none;
          color:rgba(61, 204, 145, 0.5); }
          .bp3-dark .bp3-button.bp3-outlined.bp3-intent-success:disabled.bp3-active, .bp3-dark .bp3-button.bp3-outlined.bp3-intent-success.bp3-disabled.bp3-active{
            background:rgba(15, 153, 96, 0.3); }
    .bp3-button.bp3-outlined.bp3-intent-warning{
      color:#bf7326; }
      .bp3-button.bp3-outlined.bp3-intent-warning:hover, .bp3-button.bp3-outlined.bp3-intent-warning:active, .bp3-button.bp3-outlined.bp3-intent-warning.bp3-active{
        background:none;
        -webkit-box-shadow:none;
                box-shadow:none;
        color:#bf7326; }
      .bp3-button.bp3-outlined.bp3-intent-warning:hover{
        background:rgba(217, 130, 43, 0.15);
        color:#bf7326; }
      .bp3-button.bp3-outlined.bp3-intent-warning:active, .bp3-button.bp3-outlined.bp3-intent-warning.bp3-active{
        background:rgba(217, 130, 43, 0.3);
        color:#bf7326; }
      .bp3-button.bp3-outlined.bp3-intent-warning:disabled, .bp3-button.bp3-outlined.bp3-intent-warning.bp3-disabled{
        background:none;
        color:rgba(191, 115, 38, 0.5); }
        .bp3-button.bp3-outlined.bp3-intent-warning:disabled.bp3-active, .bp3-button.bp3-outlined.bp3-intent-warning.bp3-disabled.bp3-active{
          background:rgba(217, 130, 43, 0.3); }
      .bp3-button.bp3-outlined.bp3-intent-warning .bp3-button-spinner .bp3-spinner-head{
        stroke:#bf7326; }
      .bp3-dark .bp3-button.bp3-outlined.bp3-intent-warning{
        color:#ffb366; }
        .bp3-dark .bp3-button.bp3-outlined.bp3-intent-warning:hover{
          background:rgba(217, 130, 43, 0.2);
          color:#ffb366; }
        .bp3-dark .bp3-button.bp3-outlined.bp3-intent-warning:active, .bp3-dark .bp3-button.bp3-outlined.bp3-intent-warning.bp3-active{
          background:rgba(217, 130, 43, 0.3);
          color:#ffb366; }
        .bp3-dark .bp3-button.bp3-outlined.bp3-intent-warning:disabled, .bp3-dark .bp3-button.bp3-outlined.bp3-intent-warning.bp3-disabled{
          background:none;
          color:rgba(255, 179, 102, 0.5); }
          .bp3-dark .bp3-button.bp3-outlined.bp3-intent-warning:disabled.bp3-active, .bp3-dark .bp3-button.bp3-outlined.bp3-intent-warning.bp3-disabled.bp3-active{
            background:rgba(217, 130, 43, 0.3); }
    .bp3-button.bp3-outlined.bp3-intent-danger{
      color:#c23030; }
      .bp3-button.bp3-outlined.bp3-intent-danger:hover, .bp3-button.bp3-outlined.bp3-intent-danger:active, .bp3-button.bp3-outlined.bp3-intent-danger.bp3-active{
        background:none;
        -webkit-box-shadow:none;
                box-shadow:none;
        color:#c23030; }
      .bp3-button.bp3-outlined.bp3-intent-danger:hover{
        background:rgba(219, 55, 55, 0.15);
        color:#c23030; }
      .bp3-button.bp3-outlined.bp3-intent-danger:active, .bp3-button.bp3-outlined.bp3-intent-danger.bp3-active{
        background:rgba(219, 55, 55, 0.3);
        color:#c23030; }
      .bp3-button.bp3-outlined.bp3-intent-danger:disabled, .bp3-button.bp3-outlined.bp3-intent-danger.bp3-disabled{
        background:none;
        color:rgba(194, 48, 48, 0.5); }
        .bp3-button.bp3-outlined.bp3-intent-danger:disabled.bp3-active, .bp3-button.bp3-outlined.bp3-intent-danger.bp3-disabled.bp3-active{
          background:rgba(219, 55, 55, 0.3); }
      .bp3-button.bp3-outlined.bp3-intent-danger .bp3-button-spinner .bp3-spinner-head{
        stroke:#c23030; }
      .bp3-dark .bp3-button.bp3-outlined.bp3-intent-danger{
        color:#ff7373; }
        .bp3-dark .bp3-button.bp3-outlined.bp3-intent-danger:hover{
          background:rgba(219, 55, 55, 0.2);
          color:#ff7373; }
        .bp3-dark .bp3-button.bp3-outlined.bp3-intent-danger:active, .bp3-dark .bp3-button.bp3-outlined.bp3-intent-danger.bp3-active{
          background:rgba(219, 55, 55, 0.3);
          color:#ff7373; }
        .bp3-dark .bp3-button.bp3-outlined.bp3-intent-danger:disabled, .bp3-dark .bp3-button.bp3-outlined.bp3-intent-danger.bp3-disabled{
          background:none;
          color:rgba(255, 115, 115, 0.5); }
          .bp3-dark .bp3-button.bp3-outlined.bp3-intent-danger:disabled.bp3-active, .bp3-dark .bp3-button.bp3-outlined.bp3-intent-danger.bp3-disabled.bp3-active{
            background:rgba(219, 55, 55, 0.3); }
    .bp3-button.bp3-outlined:disabled, .bp3-button.bp3-outlined.bp3-disabled, .bp3-button.bp3-outlined:disabled:hover, .bp3-button.bp3-outlined.bp3-disabled:hover{
      border-color:rgba(92, 112, 128, 0.1); }
    .bp3-dark .bp3-button.bp3-outlined{
      border-color:rgba(255, 255, 255, 0.4); }
      .bp3-dark .bp3-button.bp3-outlined:disabled, .bp3-dark .bp3-button.bp3-outlined:disabled:hover, .bp3-dark .bp3-button.bp3-outlined.bp3-disabled, .bp3-dark .bp3-button.bp3-outlined.bp3-disabled:hover{
        border-color:rgba(255, 255, 255, 0.2); }
    .bp3-button.bp3-outlined.bp3-intent-primary{
      border-color:rgba(16, 107, 163, 0.6); }
      .bp3-button.bp3-outlined.bp3-intent-primary:disabled, .bp3-button.bp3-outlined.bp3-intent-primary.bp3-disabled{
        border-color:rgba(16, 107, 163, 0.2); }
      .bp3-dark .bp3-button.bp3-outlined.bp3-intent-primary{
        border-color:rgba(72, 175, 240, 0.6); }
        .bp3-dark .bp3-button.bp3-outlined.bp3-intent-primary:disabled, .bp3-dark .bp3-button.bp3-outlined.bp3-intent-primary.bp3-disabled{
          border-color:rgba(72, 175, 240, 0.2); }
    .bp3-button.bp3-outlined.bp3-intent-success{
      border-color:rgba(13, 128, 80, 0.6); }
      .bp3-button.bp3-outlined.bp3-intent-success:disabled, .bp3-button.bp3-outlined.bp3-intent-success.bp3-disabled{
        border-color:rgba(13, 128, 80, 0.2); }
      .bp3-dark .bp3-button.bp3-outlined.bp3-intent-success{
        border-color:rgba(61, 204, 145, 0.6); }
        .bp3-dark .bp3-button.bp3-outlined.bp3-intent-success:disabled, .bp3-dark .bp3-button.bp3-outlined.bp3-intent-success.bp3-disabled{
          border-color:rgba(61, 204, 145, 0.2); }
    .bp3-button.bp3-outlined.bp3-intent-warning{
      border-color:rgba(191, 115, 38, 0.6); }
      .bp3-button.bp3-outlined.bp3-intent-warning:disabled, .bp3-button.bp3-outlined.bp3-intent-warning.bp3-disabled{
        border-color:rgba(191, 115, 38, 0.2); }
      .bp3-dark .bp3-button.bp3-outlined.bp3-intent-warning{
        border-color:rgba(255, 179, 102, 0.6); }
        .bp3-dark .bp3-button.bp3-outlined.bp3-intent-warning:disabled, .bp3-dark .bp3-button.bp3-outlined.bp3-intent-warning.bp3-disabled{
          border-color:rgba(255, 179, 102, 0.2); }
    .bp3-button.bp3-outlined.bp3-intent-danger{
      border-color:rgba(194, 48, 48, 0.6); }
      .bp3-button.bp3-outlined.bp3-intent-danger:disabled, .bp3-button.bp3-outlined.bp3-intent-danger.bp3-disabled{
        border-color:rgba(194, 48, 48, 0.2); }
      .bp3-dark .bp3-button.bp3-outlined.bp3-intent-danger{
        border-color:rgba(255, 115, 115, 0.6); }
        .bp3-dark .bp3-button.bp3-outlined.bp3-intent-danger:disabled, .bp3-dark .bp3-button.bp3-outlined.bp3-intent-danger.bp3-disabled{
          border-color:rgba(255, 115, 115, 0.2); }

a.bp3-button{
  text-align:center;
  text-decoration:none;
  -webkit-transition:none;
  transition:none; }
  a.bp3-button, a.bp3-button:hover, a.bp3-button:active{
    color:#182026; }
  a.bp3-button.bp3-disabled{
    color:rgba(92, 112, 128, 0.6); }

.bp3-button-text{
  -webkit-box-flex:0;
      -ms-flex:0 1 auto;
          flex:0 1 auto; }

.bp3-button.bp3-align-left .bp3-button-text, .bp3-button.bp3-align-right .bp3-button-text,
.bp3-button-group.bp3-align-left .bp3-button-text,
.bp3-button-group.bp3-align-right .bp3-button-text{
  -webkit-box-flex:1;
      -ms-flex:1 1 auto;
          flex:1 1 auto; }
.bp3-button-group{
  display:-webkit-inline-box;
  display:-ms-inline-flexbox;
  display:inline-flex; }
  .bp3-button-group .bp3-button{
    -webkit-box-flex:0;
        -ms-flex:0 0 auto;
            flex:0 0 auto;
    position:relative;
    z-index:4; }
    .bp3-button-group .bp3-button:focus{
      z-index:5; }
    .bp3-button-group .bp3-button:hover{
      z-index:6; }
    .bp3-button-group .bp3-button:active, .bp3-button-group .bp3-button.bp3-active{
      z-index:7; }
    .bp3-button-group .bp3-button:disabled, .bp3-button-group .bp3-button.bp3-disabled{
      z-index:3; }
    .bp3-button-group .bp3-button[class*="bp3-intent-"]{
      z-index:9; }
      .bp3-button-group .bp3-button[class*="bp3-intent-"]:focus{
        z-index:10; }
      .bp3-button-group .bp3-button[class*="bp3-intent-"]:hover{
        z-index:11; }
      .bp3-button-group .bp3-button[class*="bp3-intent-"]:active, .bp3-button-group .bp3-button[class*="bp3-intent-"].bp3-active{
        z-index:12; }
      .bp3-button-group .bp3-button[class*="bp3-intent-"]:disabled, .bp3-button-group .bp3-button[class*="bp3-intent-"].bp3-disabled{
        z-index:8; }
  .bp3-button-group:not(.bp3-minimal) > .bp3-popover-wrapper:not(:first-child) .bp3-button,
  .bp3-button-group:not(.bp3-minimal) > .bp3-button:not(:first-child){
    border-bottom-left-radius:0;
    border-top-left-radius:0; }
  .bp3-button-group:not(.bp3-minimal) > .bp3-popover-wrapper:not(:last-child) .bp3-button,
  .bp3-button-group:not(.bp3-minimal) > .bp3-button:not(:last-child){
    border-bottom-right-radius:0;
    border-top-right-radius:0;
    margin-right:-1px; }
  .bp3-button-group.bp3-minimal .bp3-button{
    background:none;
    -webkit-box-shadow:none;
            box-shadow:none; }
    .bp3-button-group.bp3-minimal .bp3-button:hover{
      background:rgba(167, 182, 194, 0.3);
      -webkit-box-shadow:none;
              box-shadow:none;
      color:#182026;
      text-decoration:none; }
    .bp3-button-group.bp3-minimal .bp3-button:active, .bp3-button-group.bp3-minimal .bp3-button.bp3-active{
      background:rgba(115, 134, 148, 0.3);
      -webkit-box-shadow:none;
              box-shadow:none;
      color:#182026; }
    .bp3-button-group.bp3-minimal .bp3-button:disabled, .bp3-button-group.bp3-minimal .bp3-button:disabled:hover, .bp3-button-group.bp3-minimal .bp3-button.bp3-disabled, .bp3-button-group.bp3-minimal .bp3-button.bp3-disabled:hover{
      background:none;
      color:rgba(92, 112, 128, 0.6);
      cursor:not-allowed; }
      .bp3-button-group.bp3-minimal .bp3-button:disabled.bp3-active, .bp3-button-group.bp3-minimal .bp3-button:disabled:hover.bp3-active, .bp3-button-group.bp3-minimal .bp3-button.bp3-disabled.bp3-active, .bp3-button-group.bp3-minimal .bp3-button.bp3-disabled:hover.bp3-active{
        background:rgba(115, 134, 148, 0.3); }
    .bp3-dark .bp3-button-group.bp3-minimal .bp3-button{
      background:none;
      -webkit-box-shadow:none;
              box-shadow:none;
      color:inherit; }
      .bp3-dark .bp3-button-group.bp3-minimal .bp3-button:hover, .bp3-dark .bp3-button-group.bp3-minimal .bp3-button:active, .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-active{
        background:none;
        -webkit-box-shadow:none;
                box-shadow:none; }
      .bp3-dark .bp3-button-group.bp3-minimal .bp3-button:hover{
        background:rgba(138, 155, 168, 0.15); }
      .bp3-dark .bp3-button-group.bp3-minimal .bp3-button:active, .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-active{
        background:rgba(138, 155, 168, 0.3);
        color:#f5f8fa; }
      .bp3-dark .bp3-button-group.bp3-minimal .bp3-button:disabled, .bp3-dark .bp3-button-group.bp3-minimal .bp3-button:disabled:hover, .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-disabled, .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-disabled:hover{
        background:none;
        color:rgba(167, 182, 194, 0.6);
        cursor:not-allowed; }
        .bp3-dark .bp3-button-group.bp3-minimal .bp3-button:disabled.bp3-active, .bp3-dark .bp3-button-group.bp3-minimal .bp3-button:disabled:hover.bp3-active, .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-disabled.bp3-active, .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-disabled:hover.bp3-active{
          background:rgba(138, 155, 168, 0.3); }
    .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-primary{
      color:#106ba3; }
      .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-primary:hover, .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-primary:active, .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-primary.bp3-active{
        background:none;
        -webkit-box-shadow:none;
                box-shadow:none;
        color:#106ba3; }
      .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-primary:hover{
        background:rgba(19, 124, 189, 0.15);
        color:#106ba3; }
      .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-primary:active, .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-primary.bp3-active{
        background:rgba(19, 124, 189, 0.3);
        color:#106ba3; }
      .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-primary:disabled, .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-primary.bp3-disabled{
        background:none;
        color:rgba(16, 107, 163, 0.5); }
        .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-primary:disabled.bp3-active, .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-primary.bp3-disabled.bp3-active{
          background:rgba(19, 124, 189, 0.3); }
      .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-primary .bp3-button-spinner .bp3-spinner-head{
        stroke:#106ba3; }
      .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-primary{
        color:#48aff0; }
        .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-primary:hover{
          background:rgba(19, 124, 189, 0.2);
          color:#48aff0; }
        .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-primary:active, .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-primary.bp3-active{
          background:rgba(19, 124, 189, 0.3);
          color:#48aff0; }
        .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-primary:disabled, .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-primary.bp3-disabled{
          background:none;
          color:rgba(72, 175, 240, 0.5); }
          .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-primary:disabled.bp3-active, .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-primary.bp3-disabled.bp3-active{
            background:rgba(19, 124, 189, 0.3); }
    .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-success{
      color:#0d8050; }
      .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-success:hover, .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-success:active, .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-success.bp3-active{
        background:none;
        -webkit-box-shadow:none;
                box-shadow:none;
        color:#0d8050; }
      .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-success:hover{
        background:rgba(15, 153, 96, 0.15);
        color:#0d8050; }
      .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-success:active, .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-success.bp3-active{
        background:rgba(15, 153, 96, 0.3);
        color:#0d8050; }
      .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-success:disabled, .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-success.bp3-disabled{
        background:none;
        color:rgba(13, 128, 80, 0.5); }
        .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-success:disabled.bp3-active, .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-success.bp3-disabled.bp3-active{
          background:rgba(15, 153, 96, 0.3); }
      .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-success .bp3-button-spinner .bp3-spinner-head{
        stroke:#0d8050; }
      .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-success{
        color:#3dcc91; }
        .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-success:hover{
          background:rgba(15, 153, 96, 0.2);
          color:#3dcc91; }
        .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-success:active, .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-success.bp3-active{
          background:rgba(15, 153, 96, 0.3);
          color:#3dcc91; }
        .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-success:disabled, .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-success.bp3-disabled{
          background:none;
          color:rgba(61, 204, 145, 0.5); }
          .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-success:disabled.bp3-active, .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-success.bp3-disabled.bp3-active{
            background:rgba(15, 153, 96, 0.3); }
    .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-warning{
      color:#bf7326; }
      .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-warning:hover, .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-warning:active, .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-warning.bp3-active{
        background:none;
        -webkit-box-shadow:none;
                box-shadow:none;
        color:#bf7326; }
      .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-warning:hover{
        background:rgba(217, 130, 43, 0.15);
        color:#bf7326; }
      .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-warning:active, .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-warning.bp3-active{
        background:rgba(217, 130, 43, 0.3);
        color:#bf7326; }
      .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-warning:disabled, .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-warning.bp3-disabled{
        background:none;
        color:rgba(191, 115, 38, 0.5); }
        .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-warning:disabled.bp3-active, .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-warning.bp3-disabled.bp3-active{
          background:rgba(217, 130, 43, 0.3); }
      .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-warning .bp3-button-spinner .bp3-spinner-head{
        stroke:#bf7326; }
      .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-warning{
        color:#ffb366; }
        .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-warning:hover{
          background:rgba(217, 130, 43, 0.2);
          color:#ffb366; }
        .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-warning:active, .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-warning.bp3-active{
          background:rgba(217, 130, 43, 0.3);
          color:#ffb366; }
        .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-warning:disabled, .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-warning.bp3-disabled{
          background:none;
          color:rgba(255, 179, 102, 0.5); }
          .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-warning:disabled.bp3-active, .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-warning.bp3-disabled.bp3-active{
            background:rgba(217, 130, 43, 0.3); }
    .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-danger{
      color:#c23030; }
      .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-danger:hover, .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-danger:active, .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-danger.bp3-active{
        background:none;
        -webkit-box-shadow:none;
                box-shadow:none;
        color:#c23030; }
      .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-danger:hover{
        background:rgba(219, 55, 55, 0.15);
        color:#c23030; }
      .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-danger:active, .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-danger.bp3-active{
        background:rgba(219, 55, 55, 0.3);
        color:#c23030; }
      .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-danger:disabled, .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-danger.bp3-disabled{
        background:none;
        color:rgba(194, 48, 48, 0.5); }
        .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-danger:disabled.bp3-active, .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-danger.bp3-disabled.bp3-active{
          background:rgba(219, 55, 55, 0.3); }
      .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-danger .bp3-button-spinner .bp3-spinner-head{
        stroke:#c23030; }
      .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-danger{
        color:#ff7373; }
        .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-danger:hover{
          background:rgba(219, 55, 55, 0.2);
          color:#ff7373; }
        .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-danger:active, .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-danger.bp3-active{
          background:rgba(219, 55, 55, 0.3);
          color:#ff7373; }
        .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-danger:disabled, .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-danger.bp3-disabled{
          background:none;
          color:rgba(255, 115, 115, 0.5); }
          .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-danger:disabled.bp3-active, .bp3-dark .bp3-button-group.bp3-minimal .bp3-button.bp3-intent-danger.bp3-disabled.bp3-active{
            background:rgba(219, 55, 55, 0.3); }
  .bp3-button-group .bp3-popover-wrapper,
  .bp3-button-group .bp3-popover-target{
    display:-webkit-box;
    display:-ms-flexbox;
    display:flex;
    -webkit-box-flex:1;
        -ms-flex:1 1 auto;
            flex:1 1 auto; }
  .bp3-button-group.bp3-fill{
    display:-webkit-box;
    display:-ms-flexbox;
    display:flex;
    width:100%; }
  .bp3-button-group .bp3-button.bp3-fill,
  .bp3-button-group.bp3-fill .bp3-button:not(.bp3-fixed){
    -webkit-box-flex:1;
        -ms-flex:1 1 auto;
            flex:1 1 auto; }
  .bp3-button-group.bp3-vertical{
    -webkit-box-align:stretch;
        -ms-flex-align:stretch;
            align-items:stretch;
    -webkit-box-orient:vertical;
    -webkit-box-direction:normal;
        -ms-flex-direction:column;
            flex-direction:column;
    vertical-align:top; }
    .bp3-button-group.bp3-vertical.bp3-fill{
      height:100%;
      width:unset; }
    .bp3-button-group.bp3-vertical .bp3-button{
      margin-right:0 !important;
      width:100%; }
    .bp3-button-group.bp3-vertical:not(.bp3-minimal) > .bp3-popover-wrapper:first-child .bp3-button,
    .bp3-button-group.bp3-vertical:not(.bp3-minimal) > .bp3-button:first-child{
      border-radius:3px 3px 0 0; }
    .bp3-button-group.bp3-vertical:not(.bp3-minimal) > .bp3-popover-wrapper:last-child .bp3-button,
    .bp3-button-group.bp3-vertical:not(.bp3-minimal) > .bp3-button:last-child{
      border-radius:0 0 3px 3px; }
    .bp3-button-group.bp3-vertical:not(.bp3-minimal) > .bp3-popover-wrapper:not(:last-child) .bp3-button,
    .bp3-button-group.bp3-vertical:not(.bp3-minimal) > .bp3-button:not(:last-child){
      margin-bottom:-1px; }
  .bp3-button-group.bp3-align-left .bp3-button{
    text-align:left; }
  .bp3-dark .bp3-button-group:not(.bp3-minimal) > .bp3-popover-wrapper:not(:last-child) .bp3-button,
  .bp3-dark .bp3-button-group:not(.bp3-minimal) > .bp3-button:not(:last-child){
    margin-right:1px; }
  .bp3-dark .bp3-button-group.bp3-vertical > .bp3-popover-wrapper:not(:last-child) .bp3-button,
  .bp3-dark .bp3-button-group.bp3-vertical > .bp3-button:not(:last-child){
    margin-bottom:1px; }
.bp3-callout{
  font-size:14px;
  line-height:1.5;
  background-color:rgba(138, 155, 168, 0.15);
  border-radius:3px;
  padding:10px 12px 9px;
  position:relative;
  width:100%; }
  .bp3-callout[class*="bp3-icon-"]{
    padding-left:40px; }
    .bp3-callout[class*="bp3-icon-"]::before{
      font-family:"Icons20", sans-serif;
      font-size:20px;
      font-style:normal;
      font-weight:400;
      line-height:1;
      -moz-osx-font-smoothing:grayscale;
      -webkit-font-smoothing:antialiased;
      color:#5c7080;
      left:10px;
      position:absolute;
      top:10px; }
  .bp3-callout.bp3-callout-icon{
    padding-left:40px; }
    .bp3-callout.bp3-callout-icon > .bp3-icon:first-child{
      color:#5c7080;
      left:10px;
      position:absolute;
      top:10px; }
  .bp3-callout .bp3-heading{
    line-height:20px;
    margin-bottom:5px;
    margin-top:0; }
    .bp3-callout .bp3-heading:last-child{
      margin-bottom:0; }
  .bp3-dark .bp3-callout{
    background-color:rgba(138, 155, 168, 0.2); }
    .bp3-dark .bp3-callout[class*="bp3-icon-"]::before{
      color:#a7b6c2; }
  .bp3-callout.bp3-intent-primary{
    background-color:rgba(19, 124, 189, 0.15); }
    .bp3-callout.bp3-intent-primary[class*="bp3-icon-"]::before,
    .bp3-callout.bp3-intent-primary > .bp3-icon:first-child,
    .bp3-callout.bp3-intent-primary .bp3-heading{
      color:#106ba3; }
    .bp3-dark .bp3-callout.bp3-intent-primary{
      background-color:rgba(19, 124, 189, 0.25); }
      .bp3-dark .bp3-callout.bp3-intent-primary[class*="bp3-icon-"]::before,
      .bp3-dark .bp3-callout.bp3-intent-primary > .bp3-icon:first-child,
      .bp3-dark .bp3-callout.bp3-intent-primary .bp3-heading{
        color:#48aff0; }
  .bp3-callout.bp3-intent-success{
    background-color:rgba(15, 153, 96, 0.15); }
    .bp3-callout.bp3-intent-success[class*="bp3-icon-"]::before,
    .bp3-callout.bp3-intent-success > .bp3-icon:first-child,
    .bp3-callout.bp3-intent-success .bp3-heading{
      color:#0d8050; }
    .bp3-dark .bp3-callout.bp3-intent-success{
      background-color:rgba(15, 153, 96, 0.25); }
      .bp3-dark .bp3-callout.bp3-intent-success[class*="bp3-icon-"]::before,
      .bp3-dark .bp3-callout.bp3-intent-success > .bp3-icon:first-child,
      .bp3-dark .bp3-callout.bp3-intent-success .bp3-heading{
        color:#3dcc91; }
  .bp3-callout.bp3-intent-warning{
    background-color:rgba(217, 130, 43, 0.15); }
    .bp3-callout.bp3-intent-warning[class*="bp3-icon-"]::before,
    .bp3-callout.bp3-intent-warning > .bp3-icon:first-child,
    .bp3-callout.bp3-intent-warning .bp3-heading{
      color:#bf7326; }
    .bp3-dark .bp3-callout.bp3-intent-warning{
      background-color:rgba(217, 130, 43, 0.25); }
      .bp3-dark .bp3-callout.bp3-intent-warning[class*="bp3-icon-"]::before,
      .bp3-dark .bp3-callout.bp3-intent-warning > .bp3-icon:first-child,
      .bp3-dark .bp3-callout.bp3-intent-warning .bp3-heading{
        color:#ffb366; }
  .bp3-callout.bp3-intent-danger{
    background-color:rgba(219, 55, 55, 0.15); }
    .bp3-callout.bp3-intent-danger[class*="bp3-icon-"]::before,
    .bp3-callout.bp3-intent-danger > .bp3-icon:first-child,
    .bp3-callout.bp3-intent-danger .bp3-heading{
      color:#c23030; }
    .bp3-dark .bp3-callout.bp3-intent-danger{
      background-color:rgba(219, 55, 55, 0.25); }
      .bp3-dark .bp3-callout.bp3-intent-danger[class*="bp3-icon-"]::before,
      .bp3-dark .bp3-callout.bp3-intent-danger > .bp3-icon:first-child,
      .bp3-dark .bp3-callout.bp3-intent-danger .bp3-heading{
        color:#ff7373; }
  .bp3-running-text .bp3-callout{
    margin:20px 0; }
.bp3-card{
  background-color:#ffffff;
  border-radius:3px;
  -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.15), 0 0 0 rgba(16, 22, 26, 0), 0 0 0 rgba(16, 22, 26, 0);
          box-shadow:0 0 0 1px rgba(16, 22, 26, 0.15), 0 0 0 rgba(16, 22, 26, 0), 0 0 0 rgba(16, 22, 26, 0);
  padding:20px;
  -webkit-transition:-webkit-transform 200ms cubic-bezier(0.4, 1, 0.75, 0.9), -webkit-box-shadow 200ms cubic-bezier(0.4, 1, 0.75, 0.9);
  transition:-webkit-transform 200ms cubic-bezier(0.4, 1, 0.75, 0.9), -webkit-box-shadow 200ms cubic-bezier(0.4, 1, 0.75, 0.9);
  transition:transform 200ms cubic-bezier(0.4, 1, 0.75, 0.9), box-shadow 200ms cubic-bezier(0.4, 1, 0.75, 0.9);
  transition:transform 200ms cubic-bezier(0.4, 1, 0.75, 0.9), box-shadow 200ms cubic-bezier(0.4, 1, 0.75, 0.9), -webkit-transform 200ms cubic-bezier(0.4, 1, 0.75, 0.9), -webkit-box-shadow 200ms cubic-bezier(0.4, 1, 0.75, 0.9); }
  .bp3-card.bp3-dark,
  .bp3-dark .bp3-card{
    background-color:#30404d;
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4), 0 0 0 rgba(16, 22, 26, 0), 0 0 0 rgba(16, 22, 26, 0);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4), 0 0 0 rgba(16, 22, 26, 0), 0 0 0 rgba(16, 22, 26, 0); }

.bp3-elevation-0{
  -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.15), 0 0 0 rgba(16, 22, 26, 0), 0 0 0 rgba(16, 22, 26, 0);
          box-shadow:0 0 0 1px rgba(16, 22, 26, 0.15), 0 0 0 rgba(16, 22, 26, 0), 0 0 0 rgba(16, 22, 26, 0); }
  .bp3-elevation-0.bp3-dark,
  .bp3-dark .bp3-elevation-0{
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4), 0 0 0 rgba(16, 22, 26, 0), 0 0 0 rgba(16, 22, 26, 0);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4), 0 0 0 rgba(16, 22, 26, 0), 0 0 0 rgba(16, 22, 26, 0); }

.bp3-elevation-1{
  -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 0 0 rgba(16, 22, 26, 0), 0 1px 1px rgba(16, 22, 26, 0.2);
          box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 0 0 rgba(16, 22, 26, 0), 0 1px 1px rgba(16, 22, 26, 0.2); }
  .bp3-elevation-1.bp3-dark,
  .bp3-dark .bp3-elevation-1{
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 0 0 rgba(16, 22, 26, 0), 0 1px 1px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 0 0 rgba(16, 22, 26, 0), 0 1px 1px rgba(16, 22, 26, 0.4); }

.bp3-elevation-2{
  -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 1px 1px rgba(16, 22, 26, 0.2), 0 2px 6px rgba(16, 22, 26, 0.2);
          box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 1px 1px rgba(16, 22, 26, 0.2), 0 2px 6px rgba(16, 22, 26, 0.2); }
  .bp3-elevation-2.bp3-dark,
  .bp3-dark .bp3-elevation-2{
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 1px 1px rgba(16, 22, 26, 0.4), 0 2px 6px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 1px 1px rgba(16, 22, 26, 0.4), 0 2px 6px rgba(16, 22, 26, 0.4); }

.bp3-elevation-3{
  -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 2px 4px rgba(16, 22, 26, 0.2), 0 8px 24px rgba(16, 22, 26, 0.2);
          box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 2px 4px rgba(16, 22, 26, 0.2), 0 8px 24px rgba(16, 22, 26, 0.2); }
  .bp3-elevation-3.bp3-dark,
  .bp3-dark .bp3-elevation-3{
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 2px 4px rgba(16, 22, 26, 0.4), 0 8px 24px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 2px 4px rgba(16, 22, 26, 0.4), 0 8px 24px rgba(16, 22, 26, 0.4); }

.bp3-elevation-4{
  -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 4px 8px rgba(16, 22, 26, 0.2), 0 18px 46px 6px rgba(16, 22, 26, 0.2);
          box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 4px 8px rgba(16, 22, 26, 0.2), 0 18px 46px 6px rgba(16, 22, 26, 0.2); }
  .bp3-elevation-4.bp3-dark,
  .bp3-dark .bp3-elevation-4{
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 4px 8px rgba(16, 22, 26, 0.4), 0 18px 46px 6px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 4px 8px rgba(16, 22, 26, 0.4), 0 18px 46px 6px rgba(16, 22, 26, 0.4); }

.bp3-card.bp3-interactive:hover{
  -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 2px 4px rgba(16, 22, 26, 0.2), 0 8px 24px rgba(16, 22, 26, 0.2);
          box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 2px 4px rgba(16, 22, 26, 0.2), 0 8px 24px rgba(16, 22, 26, 0.2);
  cursor:pointer; }
  .bp3-card.bp3-interactive:hover.bp3-dark,
  .bp3-dark .bp3-card.bp3-interactive:hover{
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 2px 4px rgba(16, 22, 26, 0.4), 0 8px 24px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 2px 4px rgba(16, 22, 26, 0.4), 0 8px 24px rgba(16, 22, 26, 0.4); }

.bp3-card.bp3-interactive:active{
  -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 0 0 rgba(16, 22, 26, 0), 0 1px 1px rgba(16, 22, 26, 0.2);
          box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 0 0 rgba(16, 22, 26, 0), 0 1px 1px rgba(16, 22, 26, 0.2);
  opacity:0.9;
  -webkit-transition-duration:0;
          transition-duration:0; }
  .bp3-card.bp3-interactive:active.bp3-dark,
  .bp3-dark .bp3-card.bp3-interactive:active{
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 0 0 rgba(16, 22, 26, 0), 0 1px 1px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 0 0 rgba(16, 22, 26, 0), 0 1px 1px rgba(16, 22, 26, 0.4); }

.bp3-collapse{
  height:0;
  overflow-y:hidden;
  -webkit-transition:height 200ms cubic-bezier(0.4, 1, 0.75, 0.9);
  transition:height 200ms cubic-bezier(0.4, 1, 0.75, 0.9); }
  .bp3-collapse .bp3-collapse-body{
    -webkit-transition:-webkit-transform 200ms cubic-bezier(0.4, 1, 0.75, 0.9);
    transition:-webkit-transform 200ms cubic-bezier(0.4, 1, 0.75, 0.9);
    transition:transform 200ms cubic-bezier(0.4, 1, 0.75, 0.9);
    transition:transform 200ms cubic-bezier(0.4, 1, 0.75, 0.9), -webkit-transform 200ms cubic-bezier(0.4, 1, 0.75, 0.9); }
    .bp3-collapse .bp3-collapse-body[aria-hidden="true"]{
      display:none; }

.bp3-context-menu .bp3-popover-target{
  display:block; }

.bp3-context-menu-popover-target{
  position:fixed; }

.bp3-divider{
  border-bottom:1px solid rgba(16, 22, 26, 0.15);
  border-right:1px solid rgba(16, 22, 26, 0.15);
  margin:5px; }
  .bp3-dark .bp3-divider{
    border-color:rgba(16, 22, 26, 0.4); }
.bp3-dialog-container{
  opacity:1;
  -webkit-transform:scale(1);
          transform:scale(1);
  -webkit-box-align:center;
      -ms-flex-align:center;
          align-items:center;
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex;
  -webkit-box-pack:center;
      -ms-flex-pack:center;
          justify-content:center;
  min-height:100%;
  pointer-events:none;
  -webkit-user-select:none;
     -moz-user-select:none;
      -ms-user-select:none;
          user-select:none;
  width:100%; }
  .bp3-dialog-container.bp3-overlay-enter > .bp3-dialog, .bp3-dialog-container.bp3-overlay-appear > .bp3-dialog{
    opacity:0;
    -webkit-transform:scale(0.5);
            transform:scale(0.5); }
  .bp3-dialog-container.bp3-overlay-enter-active > .bp3-dialog, .bp3-dialog-container.bp3-overlay-appear-active > .bp3-dialog{
    opacity:1;
    -webkit-transform:scale(1);
            transform:scale(1);
    -webkit-transition-delay:0;
            transition-delay:0;
    -webkit-transition-duration:300ms;
            transition-duration:300ms;
    -webkit-transition-property:opacity, -webkit-transform;
    transition-property:opacity, -webkit-transform;
    transition-property:opacity, transform;
    transition-property:opacity, transform, -webkit-transform;
    -webkit-transition-timing-function:cubic-bezier(0.54, 1.12, 0.38, 1.11);
            transition-timing-function:cubic-bezier(0.54, 1.12, 0.38, 1.11); }
  .bp3-dialog-container.bp3-overlay-exit > .bp3-dialog{
    opacity:1;
    -webkit-transform:scale(1);
            transform:scale(1); }
  .bp3-dialog-container.bp3-overlay-exit-active > .bp3-dialog{
    opacity:0;
    -webkit-transform:scale(0.5);
            transform:scale(0.5);
    -webkit-transition-delay:0;
            transition-delay:0;
    -webkit-transition-duration:300ms;
            transition-duration:300ms;
    -webkit-transition-property:opacity, -webkit-transform;
    transition-property:opacity, -webkit-transform;
    transition-property:opacity, transform;
    transition-property:opacity, transform, -webkit-transform;
    -webkit-transition-timing-function:cubic-bezier(0.54, 1.12, 0.38, 1.11);
            transition-timing-function:cubic-bezier(0.54, 1.12, 0.38, 1.11); }

.bp3-dialog{
  background:#ebf1f5;
  border-radius:6px;
  -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 4px 8px rgba(16, 22, 26, 0.2), 0 18px 46px 6px rgba(16, 22, 26, 0.2);
          box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 4px 8px rgba(16, 22, 26, 0.2), 0 18px 46px 6px rgba(16, 22, 26, 0.2);
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex;
  -webkit-box-orient:vertical;
  -webkit-box-direction:normal;
      -ms-flex-direction:column;
          flex-direction:column;
  margin:30px 0;
  padding-bottom:20px;
  pointer-events:all;
  -webkit-user-select:text;
     -moz-user-select:text;
      -ms-user-select:text;
          user-select:text;
  width:500px; }
  .bp3-dialog:focus{
    outline:0; }
  .bp3-dialog.bp3-dark,
  .bp3-dark .bp3-dialog{
    background:#293742;
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 4px 8px rgba(16, 22, 26, 0.4), 0 18px 46px 6px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 4px 8px rgba(16, 22, 26, 0.4), 0 18px 46px 6px rgba(16, 22, 26, 0.4);
    color:#f5f8fa; }

.bp3-dialog-header{
  -webkit-box-align:center;
      -ms-flex-align:center;
          align-items:center;
  background:#ffffff;
  border-radius:6px 6px 0 0;
  -webkit-box-shadow:0 1px 0 rgba(16, 22, 26, 0.15);
          box-shadow:0 1px 0 rgba(16, 22, 26, 0.15);
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex;
  -webkit-box-flex:0;
      -ms-flex:0 0 auto;
          flex:0 0 auto;
  min-height:40px;
  padding-left:20px;
  padding-right:5px; }
  .bp3-dialog-header .bp3-icon-large,
  .bp3-dialog-header .bp3-icon{
    color:#5c7080;
    -webkit-box-flex:0;
        -ms-flex:0 0 auto;
            flex:0 0 auto;
    margin-right:10px; }
  .bp3-dialog-header .bp3-heading{
    overflow:hidden;
    text-overflow:ellipsis;
    white-space:nowrap;
    word-wrap:normal;
    -webkit-box-flex:1;
        -ms-flex:1 1 auto;
            flex:1 1 auto;
    line-height:inherit;
    margin:0; }
    .bp3-dialog-header .bp3-heading:last-child{
      margin-right:20px; }
  .bp3-dark .bp3-dialog-header{
    background:#30404d;
    -webkit-box-shadow:0 1px 0 rgba(16, 22, 26, 0.4);
            box-shadow:0 1px 0 rgba(16, 22, 26, 0.4); }
    .bp3-dark .bp3-dialog-header .bp3-icon-large,
    .bp3-dark .bp3-dialog-header .bp3-icon{
      color:#a7b6c2; }

.bp3-dialog-body{
  -webkit-box-flex:1;
      -ms-flex:1 1 auto;
          flex:1 1 auto;
  line-height:18px;
  margin:20px; }

.bp3-dialog-footer{
  -webkit-box-flex:0;
      -ms-flex:0 0 auto;
          flex:0 0 auto;
  margin:0 20px; }

.bp3-dialog-footer-actions{
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex;
  -webkit-box-pack:end;
      -ms-flex-pack:end;
          justify-content:flex-end; }
  .bp3-dialog-footer-actions .bp3-button{
    margin-left:10px; }
.bp3-drawer{
  background:#ffffff;
  -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 4px 8px rgba(16, 22, 26, 0.2), 0 18px 46px 6px rgba(16, 22, 26, 0.2);
          box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 4px 8px rgba(16, 22, 26, 0.2), 0 18px 46px 6px rgba(16, 22, 26, 0.2);
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex;
  -webkit-box-orient:vertical;
  -webkit-box-direction:normal;
      -ms-flex-direction:column;
          flex-direction:column;
  margin:0;
  padding:0; }
  .bp3-drawer:focus{
    outline:0; }
  .bp3-drawer.bp3-position-top{
    height:50%;
    left:0;
    right:0;
    top:0; }
    .bp3-drawer.bp3-position-top.bp3-overlay-enter, .bp3-drawer.bp3-position-top.bp3-overlay-appear{
      -webkit-transform:translateY(-100%);
              transform:translateY(-100%); }
    .bp3-drawer.bp3-position-top.bp3-overlay-enter-active, .bp3-drawer.bp3-position-top.bp3-overlay-appear-active{
      -webkit-transform:translateY(0);
              transform:translateY(0);
      -webkit-transition-delay:0;
              transition-delay:0;
      -webkit-transition-duration:200ms;
              transition-duration:200ms;
      -webkit-transition-property:-webkit-transform;
      transition-property:-webkit-transform;
      transition-property:transform;
      transition-property:transform, -webkit-transform;
      -webkit-transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9);
              transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9); }
    .bp3-drawer.bp3-position-top.bp3-overlay-exit{
      -webkit-transform:translateY(0);
              transform:translateY(0); }
    .bp3-drawer.bp3-position-top.bp3-overlay-exit-active{
      -webkit-transform:translateY(-100%);
              transform:translateY(-100%);
      -webkit-transition-delay:0;
              transition-delay:0;
      -webkit-transition-duration:100ms;
              transition-duration:100ms;
      -webkit-transition-property:-webkit-transform;
      transition-property:-webkit-transform;
      transition-property:transform;
      transition-property:transform, -webkit-transform;
      -webkit-transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9);
              transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9); }
  .bp3-drawer.bp3-position-bottom{
    bottom:0;
    height:50%;
    left:0;
    right:0; }
    .bp3-drawer.bp3-position-bottom.bp3-overlay-enter, .bp3-drawer.bp3-position-bottom.bp3-overlay-appear{
      -webkit-transform:translateY(100%);
              transform:translateY(100%); }
    .bp3-drawer.bp3-position-bottom.bp3-overlay-enter-active, .bp3-drawer.bp3-position-bottom.bp3-overlay-appear-active{
      -webkit-transform:translateY(0);
              transform:translateY(0);
      -webkit-transition-delay:0;
              transition-delay:0;
      -webkit-transition-duration:200ms;
              transition-duration:200ms;
      -webkit-transition-property:-webkit-transform;
      transition-property:-webkit-transform;
      transition-property:transform;
      transition-property:transform, -webkit-transform;
      -webkit-transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9);
              transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9); }
    .bp3-drawer.bp3-position-bottom.bp3-overlay-exit{
      -webkit-transform:translateY(0);
              transform:translateY(0); }
    .bp3-drawer.bp3-position-bottom.bp3-overlay-exit-active{
      -webkit-transform:translateY(100%);
              transform:translateY(100%);
      -webkit-transition-delay:0;
              transition-delay:0;
      -webkit-transition-duration:100ms;
              transition-duration:100ms;
      -webkit-transition-property:-webkit-transform;
      transition-property:-webkit-transform;
      transition-property:transform;
      transition-property:transform, -webkit-transform;
      -webkit-transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9);
              transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9); }
  .bp3-drawer.bp3-position-left{
    bottom:0;
    left:0;
    top:0;
    width:50%; }
    .bp3-drawer.bp3-position-left.bp3-overlay-enter, .bp3-drawer.bp3-position-left.bp3-overlay-appear{
      -webkit-transform:translateX(-100%);
              transform:translateX(-100%); }
    .bp3-drawer.bp3-position-left.bp3-overlay-enter-active, .bp3-drawer.bp3-position-left.bp3-overlay-appear-active{
      -webkit-transform:translateX(0);
              transform:translateX(0);
      -webkit-transition-delay:0;
              transition-delay:0;
      -webkit-transition-duration:200ms;
              transition-duration:200ms;
      -webkit-transition-property:-webkit-transform;
      transition-property:-webkit-transform;
      transition-property:transform;
      transition-property:transform, -webkit-transform;
      -webkit-transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9);
              transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9); }
    .bp3-drawer.bp3-position-left.bp3-overlay-exit{
      -webkit-transform:translateX(0);
              transform:translateX(0); }
    .bp3-drawer.bp3-position-left.bp3-overlay-exit-active{
      -webkit-transform:translateX(-100%);
              transform:translateX(-100%);
      -webkit-transition-delay:0;
              transition-delay:0;
      -webkit-transition-duration:100ms;
              transition-duration:100ms;
      -webkit-transition-property:-webkit-transform;
      transition-property:-webkit-transform;
      transition-property:transform;
      transition-property:transform, -webkit-transform;
      -webkit-transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9);
              transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9); }
  .bp3-drawer.bp3-position-right{
    bottom:0;
    right:0;
    top:0;
    width:50%; }
    .bp3-drawer.bp3-position-right.bp3-overlay-enter, .bp3-drawer.bp3-position-right.bp3-overlay-appear{
      -webkit-transform:translateX(100%);
              transform:translateX(100%); }
    .bp3-drawer.bp3-position-right.bp3-overlay-enter-active, .bp3-drawer.bp3-position-right.bp3-overlay-appear-active{
      -webkit-transform:translateX(0);
              transform:translateX(0);
      -webkit-transition-delay:0;
              transition-delay:0;
      -webkit-transition-duration:200ms;
              transition-duration:200ms;
      -webkit-transition-property:-webkit-transform;
      transition-property:-webkit-transform;
      transition-property:transform;
      transition-property:transform, -webkit-transform;
      -webkit-transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9);
              transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9); }
    .bp3-drawer.bp3-position-right.bp3-overlay-exit{
      -webkit-transform:translateX(0);
              transform:translateX(0); }
    .bp3-drawer.bp3-position-right.bp3-overlay-exit-active{
      -webkit-transform:translateX(100%);
              transform:translateX(100%);
      -webkit-transition-delay:0;
              transition-delay:0;
      -webkit-transition-duration:100ms;
              transition-duration:100ms;
      -webkit-transition-property:-webkit-transform;
      transition-property:-webkit-transform;
      transition-property:transform;
      transition-property:transform, -webkit-transform;
      -webkit-transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9);
              transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9); }
  .bp3-drawer:not(.bp3-position-top):not(.bp3-position-bottom):not(.bp3-position-left):not(
  .bp3-position-right):not(.bp3-vertical){
    bottom:0;
    right:0;
    top:0;
    width:50%; }
    .bp3-drawer:not(.bp3-position-top):not(.bp3-position-bottom):not(.bp3-position-left):not(
    .bp3-position-right):not(.bp3-vertical).bp3-overlay-enter, .bp3-drawer:not(.bp3-position-top):not(.bp3-position-bottom):not(.bp3-position-left):not(
    .bp3-position-right):not(.bp3-vertical).bp3-overlay-appear{
      -webkit-transform:translateX(100%);
              transform:translateX(100%); }
    .bp3-drawer:not(.bp3-position-top):not(.bp3-position-bottom):not(.bp3-position-left):not(
    .bp3-position-right):not(.bp3-vertical).bp3-overlay-enter-active, .bp3-drawer:not(.bp3-position-top):not(.bp3-position-bottom):not(.bp3-position-left):not(
    .bp3-position-right):not(.bp3-vertical).bp3-overlay-appear-active{
      -webkit-transform:translateX(0);
              transform:translateX(0);
      -webkit-transition-delay:0;
              transition-delay:0;
      -webkit-transition-duration:200ms;
              transition-duration:200ms;
      -webkit-transition-property:-webkit-transform;
      transition-property:-webkit-transform;
      transition-property:transform;
      transition-property:transform, -webkit-transform;
      -webkit-transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9);
              transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9); }
    .bp3-drawer:not(.bp3-position-top):not(.bp3-position-bottom):not(.bp3-position-left):not(
    .bp3-position-right):not(.bp3-vertical).bp3-overlay-exit{
      -webkit-transform:translateX(0);
              transform:translateX(0); }
    .bp3-drawer:not(.bp3-position-top):not(.bp3-position-bottom):not(.bp3-position-left):not(
    .bp3-position-right):not(.bp3-vertical).bp3-overlay-exit-active{
      -webkit-transform:translateX(100%);
              transform:translateX(100%);
      -webkit-transition-delay:0;
              transition-delay:0;
      -webkit-transition-duration:100ms;
              transition-duration:100ms;
      -webkit-transition-property:-webkit-transform;
      transition-property:-webkit-transform;
      transition-property:transform;
      transition-property:transform, -webkit-transform;
      -webkit-transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9);
              transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9); }
  .bp3-drawer:not(.bp3-position-top):not(.bp3-position-bottom):not(.bp3-position-left):not(
  .bp3-position-right).bp3-vertical{
    bottom:0;
    height:50%;
    left:0;
    right:0; }
    .bp3-drawer:not(.bp3-position-top):not(.bp3-position-bottom):not(.bp3-position-left):not(
    .bp3-position-right).bp3-vertical.bp3-overlay-enter, .bp3-drawer:not(.bp3-position-top):not(.bp3-position-bottom):not(.bp3-position-left):not(
    .bp3-position-right).bp3-vertical.bp3-overlay-appear{
      -webkit-transform:translateY(100%);
              transform:translateY(100%); }
    .bp3-drawer:not(.bp3-position-top):not(.bp3-position-bottom):not(.bp3-position-left):not(
    .bp3-position-right).bp3-vertical.bp3-overlay-enter-active, .bp3-drawer:not(.bp3-position-top):not(.bp3-position-bottom):not(.bp3-position-left):not(
    .bp3-position-right).bp3-vertical.bp3-overlay-appear-active{
      -webkit-transform:translateY(0);
              transform:translateY(0);
      -webkit-transition-delay:0;
              transition-delay:0;
      -webkit-transition-duration:200ms;
              transition-duration:200ms;
      -webkit-transition-property:-webkit-transform;
      transition-property:-webkit-transform;
      transition-property:transform;
      transition-property:transform, -webkit-transform;
      -webkit-transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9);
              transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9); }
    .bp3-drawer:not(.bp3-position-top):not(.bp3-position-bottom):not(.bp3-position-left):not(
    .bp3-position-right).bp3-vertical.bp3-overlay-exit{
      -webkit-transform:translateY(0);
              transform:translateY(0); }
    .bp3-drawer:not(.bp3-position-top):not(.bp3-position-bottom):not(.bp3-position-left):not(
    .bp3-position-right).bp3-vertical.bp3-overlay-exit-active{
      -webkit-transform:translateY(100%);
              transform:translateY(100%);
      -webkit-transition-delay:0;
              transition-delay:0;
      -webkit-transition-duration:100ms;
              transition-duration:100ms;
      -webkit-transition-property:-webkit-transform;
      transition-property:-webkit-transform;
      transition-property:transform;
      transition-property:transform, -webkit-transform;
      -webkit-transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9);
              transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9); }
  .bp3-drawer.bp3-dark,
  .bp3-dark .bp3-drawer{
    background:#30404d;
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 4px 8px rgba(16, 22, 26, 0.4), 0 18px 46px 6px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 4px 8px rgba(16, 22, 26, 0.4), 0 18px 46px 6px rgba(16, 22, 26, 0.4);
    color:#f5f8fa; }

.bp3-drawer-header{
  -webkit-box-align:center;
      -ms-flex-align:center;
          align-items:center;
  border-radius:0;
  -webkit-box-shadow:0 1px 0 rgba(16, 22, 26, 0.15);
          box-shadow:0 1px 0 rgba(16, 22, 26, 0.15);
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex;
  -webkit-box-flex:0;
      -ms-flex:0 0 auto;
          flex:0 0 auto;
  min-height:40px;
  padding:5px;
  padding-left:20px;
  position:relative; }
  .bp3-drawer-header .bp3-icon-large,
  .bp3-drawer-header .bp3-icon{
    color:#5c7080;
    -webkit-box-flex:0;
        -ms-flex:0 0 auto;
            flex:0 0 auto;
    margin-right:10px; }
  .bp3-drawer-header .bp3-heading{
    overflow:hidden;
    text-overflow:ellipsis;
    white-space:nowrap;
    word-wrap:normal;
    -webkit-box-flex:1;
        -ms-flex:1 1 auto;
            flex:1 1 auto;
    line-height:inherit;
    margin:0; }
    .bp3-drawer-header .bp3-heading:last-child{
      margin-right:20px; }
  .bp3-dark .bp3-drawer-header{
    -webkit-box-shadow:0 1px 0 rgba(16, 22, 26, 0.4);
            box-shadow:0 1px 0 rgba(16, 22, 26, 0.4); }
    .bp3-dark .bp3-drawer-header .bp3-icon-large,
    .bp3-dark .bp3-drawer-header .bp3-icon{
      color:#a7b6c2; }

.bp3-drawer-body{
  -webkit-box-flex:1;
      -ms-flex:1 1 auto;
          flex:1 1 auto;
  line-height:18px;
  overflow:auto; }

.bp3-drawer-footer{
  -webkit-box-shadow:inset 0 1px 0 rgba(16, 22, 26, 0.15);
          box-shadow:inset 0 1px 0 rgba(16, 22, 26, 0.15);
  -webkit-box-flex:0;
      -ms-flex:0 0 auto;
          flex:0 0 auto;
  padding:10px 20px;
  position:relative; }
  .bp3-dark .bp3-drawer-footer{
    -webkit-box-shadow:inset 0 1px 0 rgba(16, 22, 26, 0.4);
            box-shadow:inset 0 1px 0 rgba(16, 22, 26, 0.4); }
.bp3-editable-text{
  cursor:text;
  display:inline-block;
  max-width:100%;
  position:relative;
  vertical-align:top;
  white-space:nowrap; }
  .bp3-editable-text::before{
    bottom:-3px;
    left:-3px;
    position:absolute;
    right:-3px;
    top:-3px;
    border-radius:3px;
    content:"";
    -webkit-transition:background-color 100ms cubic-bezier(0.4, 1, 0.75, 0.9), -webkit-box-shadow 100ms cubic-bezier(0.4, 1, 0.75, 0.9);
    transition:background-color 100ms cubic-bezier(0.4, 1, 0.75, 0.9), -webkit-box-shadow 100ms cubic-bezier(0.4, 1, 0.75, 0.9);
    transition:background-color 100ms cubic-bezier(0.4, 1, 0.75, 0.9), box-shadow 100ms cubic-bezier(0.4, 1, 0.75, 0.9);
    transition:background-color 100ms cubic-bezier(0.4, 1, 0.75, 0.9), box-shadow 100ms cubic-bezier(0.4, 1, 0.75, 0.9), -webkit-box-shadow 100ms cubic-bezier(0.4, 1, 0.75, 0.9); }
  .bp3-editable-text:hover::before{
    -webkit-box-shadow:0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), inset 0 0 0 1px rgba(16, 22, 26, 0.15);
            box-shadow:0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), inset 0 0 0 1px rgba(16, 22, 26, 0.15); }
  .bp3-editable-text.bp3-editable-text-editing::before{
    background-color:#ffffff;
    -webkit-box-shadow:0 0 0 1px #137cbd, 0 0 0 3px rgba(19, 124, 189, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2);
            box-shadow:0 0 0 1px #137cbd, 0 0 0 3px rgba(19, 124, 189, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
  .bp3-editable-text.bp3-disabled::before{
    -webkit-box-shadow:none;
            box-shadow:none; }
  .bp3-editable-text.bp3-intent-primary .bp3-editable-text-input,
  .bp3-editable-text.bp3-intent-primary .bp3-editable-text-content{
    color:#137cbd; }
  .bp3-editable-text.bp3-intent-primary:hover::before{
    -webkit-box-shadow:0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), inset 0 0 0 1px rgba(19, 124, 189, 0.4);
            box-shadow:0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), inset 0 0 0 1px rgba(19, 124, 189, 0.4); }
  .bp3-editable-text.bp3-intent-primary.bp3-editable-text-editing::before{
    -webkit-box-shadow:0 0 0 1px #137cbd, 0 0 0 3px rgba(19, 124, 189, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2);
            box-shadow:0 0 0 1px #137cbd, 0 0 0 3px rgba(19, 124, 189, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
  .bp3-editable-text.bp3-intent-success .bp3-editable-text-input,
  .bp3-editable-text.bp3-intent-success .bp3-editable-text-content{
    color:#0f9960; }
  .bp3-editable-text.bp3-intent-success:hover::before{
    -webkit-box-shadow:0 0 0 0 rgba(15, 153, 96, 0), 0 0 0 0 rgba(15, 153, 96, 0), inset 0 0 0 1px rgba(15, 153, 96, 0.4);
            box-shadow:0 0 0 0 rgba(15, 153, 96, 0), 0 0 0 0 rgba(15, 153, 96, 0), inset 0 0 0 1px rgba(15, 153, 96, 0.4); }
  .bp3-editable-text.bp3-intent-success.bp3-editable-text-editing::before{
    -webkit-box-shadow:0 0 0 1px #0f9960, 0 0 0 3px rgba(15, 153, 96, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2);
            box-shadow:0 0 0 1px #0f9960, 0 0 0 3px rgba(15, 153, 96, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
  .bp3-editable-text.bp3-intent-warning .bp3-editable-text-input,
  .bp3-editable-text.bp3-intent-warning .bp3-editable-text-content{
    color:#d9822b; }
  .bp3-editable-text.bp3-intent-warning:hover::before{
    -webkit-box-shadow:0 0 0 0 rgba(217, 130, 43, 0), 0 0 0 0 rgba(217, 130, 43, 0), inset 0 0 0 1px rgba(217, 130, 43, 0.4);
            box-shadow:0 0 0 0 rgba(217, 130, 43, 0), 0 0 0 0 rgba(217, 130, 43, 0), inset 0 0 0 1px rgba(217, 130, 43, 0.4); }
  .bp3-editable-text.bp3-intent-warning.bp3-editable-text-editing::before{
    -webkit-box-shadow:0 0 0 1px #d9822b, 0 0 0 3px rgba(217, 130, 43, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2);
            box-shadow:0 0 0 1px #d9822b, 0 0 0 3px rgba(217, 130, 43, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
  .bp3-editable-text.bp3-intent-danger .bp3-editable-text-input,
  .bp3-editable-text.bp3-intent-danger .bp3-editable-text-content{
    color:#db3737; }
  .bp3-editable-text.bp3-intent-danger:hover::before{
    -webkit-box-shadow:0 0 0 0 rgba(219, 55, 55, 0), 0 0 0 0 rgba(219, 55, 55, 0), inset 0 0 0 1px rgba(219, 55, 55, 0.4);
            box-shadow:0 0 0 0 rgba(219, 55, 55, 0), 0 0 0 0 rgba(219, 55, 55, 0), inset 0 0 0 1px rgba(219, 55, 55, 0.4); }
  .bp3-editable-text.bp3-intent-danger.bp3-editable-text-editing::before{
    -webkit-box-shadow:0 0 0 1px #db3737, 0 0 0 3px rgba(219, 55, 55, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2);
            box-shadow:0 0 0 1px #db3737, 0 0 0 3px rgba(219, 55, 55, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
  .bp3-dark .bp3-editable-text:hover::before{
    -webkit-box-shadow:0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), inset 0 0 0 1px rgba(255, 255, 255, 0.15);
            box-shadow:0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), inset 0 0 0 1px rgba(255, 255, 255, 0.15); }
  .bp3-dark .bp3-editable-text.bp3-editable-text-editing::before{
    background-color:rgba(16, 22, 26, 0.3);
    -webkit-box-shadow:0 0 0 1px #137cbd, 0 0 0 3px rgba(19, 124, 189, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px #137cbd, 0 0 0 3px rgba(19, 124, 189, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4); }
  .bp3-dark .bp3-editable-text.bp3-disabled::before{
    -webkit-box-shadow:none;
            box-shadow:none; }
  .bp3-dark .bp3-editable-text.bp3-intent-primary .bp3-editable-text-content{
    color:#48aff0; }
  .bp3-dark .bp3-editable-text.bp3-intent-primary:hover::before{
    -webkit-box-shadow:0 0 0 0 rgba(72, 175, 240, 0), 0 0 0 0 rgba(72, 175, 240, 0), inset 0 0 0 1px rgba(72, 175, 240, 0.4);
            box-shadow:0 0 0 0 rgba(72, 175, 240, 0), 0 0 0 0 rgba(72, 175, 240, 0), inset 0 0 0 1px rgba(72, 175, 240, 0.4); }
  .bp3-dark .bp3-editable-text.bp3-intent-primary.bp3-editable-text-editing::before{
    -webkit-box-shadow:0 0 0 1px #48aff0, 0 0 0 3px rgba(72, 175, 240, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px #48aff0, 0 0 0 3px rgba(72, 175, 240, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4); }
  .bp3-dark .bp3-editable-text.bp3-intent-success .bp3-editable-text-content{
    color:#3dcc91; }
  .bp3-dark .bp3-editable-text.bp3-intent-success:hover::before{
    -webkit-box-shadow:0 0 0 0 rgba(61, 204, 145, 0), 0 0 0 0 rgba(61, 204, 145, 0), inset 0 0 0 1px rgba(61, 204, 145, 0.4);
            box-shadow:0 0 0 0 rgba(61, 204, 145, 0), 0 0 0 0 rgba(61, 204, 145, 0), inset 0 0 0 1px rgba(61, 204, 145, 0.4); }
  .bp3-dark .bp3-editable-text.bp3-intent-success.bp3-editable-text-editing::before{
    -webkit-box-shadow:0 0 0 1px #3dcc91, 0 0 0 3px rgba(61, 204, 145, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px #3dcc91, 0 0 0 3px rgba(61, 204, 145, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4); }
  .bp3-dark .bp3-editable-text.bp3-intent-warning .bp3-editable-text-content{
    color:#ffb366; }
  .bp3-dark .bp3-editable-text.bp3-intent-warning:hover::before{
    -webkit-box-shadow:0 0 0 0 rgba(255, 179, 102, 0), 0 0 0 0 rgba(255, 179, 102, 0), inset 0 0 0 1px rgba(255, 179, 102, 0.4);
            box-shadow:0 0 0 0 rgba(255, 179, 102, 0), 0 0 0 0 rgba(255, 179, 102, 0), inset 0 0 0 1px rgba(255, 179, 102, 0.4); }
  .bp3-dark .bp3-editable-text.bp3-intent-warning.bp3-editable-text-editing::before{
    -webkit-box-shadow:0 0 0 1px #ffb366, 0 0 0 3px rgba(255, 179, 102, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px #ffb366, 0 0 0 3px rgba(255, 179, 102, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4); }
  .bp3-dark .bp3-editable-text.bp3-intent-danger .bp3-editable-text-content{
    color:#ff7373; }
  .bp3-dark .bp3-editable-text.bp3-intent-danger:hover::before{
    -webkit-box-shadow:0 0 0 0 rgba(255, 115, 115, 0), 0 0 0 0 rgba(255, 115, 115, 0), inset 0 0 0 1px rgba(255, 115, 115, 0.4);
            box-shadow:0 0 0 0 rgba(255, 115, 115, 0), 0 0 0 0 rgba(255, 115, 115, 0), inset 0 0 0 1px rgba(255, 115, 115, 0.4); }
  .bp3-dark .bp3-editable-text.bp3-intent-danger.bp3-editable-text-editing::before{
    -webkit-box-shadow:0 0 0 1px #ff7373, 0 0 0 3px rgba(255, 115, 115, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px #ff7373, 0 0 0 3px rgba(255, 115, 115, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4); }

.bp3-editable-text-input,
.bp3-editable-text-content{
  color:inherit;
  display:inherit;
  font:inherit;
  letter-spacing:inherit;
  max-width:inherit;
  min-width:inherit;
  position:relative;
  resize:none;
  text-transform:inherit;
  vertical-align:top; }

.bp3-editable-text-input{
  background:none;
  border:none;
  -webkit-box-shadow:none;
          box-shadow:none;
  padding:0;
  white-space:pre-wrap;
  width:100%; }
  .bp3-editable-text-input::-webkit-input-placeholder{
    color:rgba(92, 112, 128, 0.6);
    opacity:1; }
  .bp3-editable-text-input::-moz-placeholder{
    color:rgba(92, 112, 128, 0.6);
    opacity:1; }
  .bp3-editable-text-input:-ms-input-placeholder{
    color:rgba(92, 112, 128, 0.6);
    opacity:1; }
  .bp3-editable-text-input::-ms-input-placeholder{
    color:rgba(92, 112, 128, 0.6);
    opacity:1; }
  .bp3-editable-text-input::placeholder{
    color:rgba(92, 112, 128, 0.6);
    opacity:1; }
  .bp3-editable-text-input:focus{
    outline:none; }
  .bp3-editable-text-input::-ms-clear{
    display:none; }

.bp3-editable-text-content{
  overflow:hidden;
  padding-right:2px;
  text-overflow:ellipsis;
  white-space:pre; }
  .bp3-editable-text-editing > .bp3-editable-text-content{
    left:0;
    position:absolute;
    visibility:hidden; }
  .bp3-editable-text-placeholder > .bp3-editable-text-content{
    color:rgba(92, 112, 128, 0.6); }
    .bp3-dark .bp3-editable-text-placeholder > .bp3-editable-text-content{
      color:rgba(167, 182, 194, 0.6); }

.bp3-editable-text.bp3-multiline{
  display:block; }
  .bp3-editable-text.bp3-multiline .bp3-editable-text-content{
    overflow:auto;
    white-space:pre-wrap;
    word-wrap:break-word; }
.bp3-divider{
  border-bottom:1px solid rgba(16, 22, 26, 0.15);
  border-right:1px solid rgba(16, 22, 26, 0.15);
  margin:5px; }
  .bp3-dark .bp3-divider{
    border-color:rgba(16, 22, 26, 0.4); }
.bp3-control-group{
  -webkit-transform:translateZ(0);
          transform:translateZ(0);
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex;
  -webkit-box-orient:horizontal;
  -webkit-box-direction:normal;
      -ms-flex-direction:row;
          flex-direction:row;
  -webkit-box-align:stretch;
      -ms-flex-align:stretch;
          align-items:stretch; }
  .bp3-control-group > *{
    -webkit-box-flex:0;
        -ms-flex-positive:0;
            flex-grow:0;
    -ms-flex-negative:0;
        flex-shrink:0; }
  .bp3-control-group > .bp3-fill{
    -webkit-box-flex:1;
        -ms-flex-positive:1;
            flex-grow:1;
    -ms-flex-negative:1;
        flex-shrink:1; }
  .bp3-control-group .bp3-button,
  .bp3-control-group .bp3-html-select,
  .bp3-control-group .bp3-input,
  .bp3-control-group .bp3-select{
    position:relative; }
  .bp3-control-group .bp3-input{
    border-radius:inherit;
    z-index:2; }
    .bp3-control-group .bp3-input:focus{
      border-radius:3px;
      z-index:14; }
    .bp3-control-group .bp3-input[class*="bp3-intent"]{
      z-index:13; }
      .bp3-control-group .bp3-input[class*="bp3-intent"]:focus{
        z-index:15; }
    .bp3-control-group .bp3-input[readonly], .bp3-control-group .bp3-input:disabled, .bp3-control-group .bp3-input.bp3-disabled{
      z-index:1; }
  .bp3-control-group .bp3-input-group[class*="bp3-intent"] .bp3-input{
    z-index:13; }
    .bp3-control-group .bp3-input-group[class*="bp3-intent"] .bp3-input:focus{
      z-index:15; }
  .bp3-control-group .bp3-button,
  .bp3-control-group .bp3-html-select select,
  .bp3-control-group .bp3-select select{
    -webkit-transform:translateZ(0);
            transform:translateZ(0);
    border-radius:inherit;
    z-index:4; }
    .bp3-control-group .bp3-button:focus,
    .bp3-control-group .bp3-html-select select:focus,
    .bp3-control-group .bp3-select select:focus{
      z-index:5; }
    .bp3-control-group .bp3-button:hover,
    .bp3-control-group .bp3-html-select select:hover,
    .bp3-control-group .bp3-select select:hover{
      z-index:6; }
    .bp3-control-group .bp3-button:active,
    .bp3-control-group .bp3-html-select select:active,
    .bp3-control-group .bp3-select select:active{
      z-index:7; }
    .bp3-control-group .bp3-button[readonly], .bp3-control-group .bp3-button:disabled, .bp3-control-group .bp3-button.bp3-disabled,
    .bp3-control-group .bp3-html-select select[readonly],
    .bp3-control-group .bp3-html-select select:disabled,
    .bp3-control-group .bp3-html-select select.bp3-disabled,
    .bp3-control-group .bp3-select select[readonly],
    .bp3-control-group .bp3-select select:disabled,
    .bp3-control-group .bp3-select select.bp3-disabled{
      z-index:3; }
    .bp3-control-group .bp3-button[class*="bp3-intent"],
    .bp3-control-group .bp3-html-select select[class*="bp3-intent"],
    .bp3-control-group .bp3-select select[class*="bp3-intent"]{
      z-index:9; }
      .bp3-control-group .bp3-button[class*="bp3-intent"]:focus,
      .bp3-control-group .bp3-html-select select[class*="bp3-intent"]:focus,
      .bp3-control-group .bp3-select select[class*="bp3-intent"]:focus{
        z-index:10; }
      .bp3-control-group .bp3-button[class*="bp3-intent"]:hover,
      .bp3-control-group .bp3-html-select select[class*="bp3-intent"]:hover,
      .bp3-control-group .bp3-select select[class*="bp3-intent"]:hover{
        z-index:11; }
      .bp3-control-group .bp3-button[class*="bp3-intent"]:active,
      .bp3-control-group .bp3-html-select select[class*="bp3-intent"]:active,
      .bp3-control-group .bp3-select select[class*="bp3-intent"]:active{
        z-index:12; }
      .bp3-control-group .bp3-button[class*="bp3-intent"][readonly], .bp3-control-group .bp3-button[class*="bp3-intent"]:disabled, .bp3-control-group .bp3-button[class*="bp3-intent"].bp3-disabled,
      .bp3-control-group .bp3-html-select select[class*="bp3-intent"][readonly],
      .bp3-control-group .bp3-html-select select[class*="bp3-intent"]:disabled,
      .bp3-control-group .bp3-html-select select[class*="bp3-intent"].bp3-disabled,
      .bp3-control-group .bp3-select select[class*="bp3-intent"][readonly],
      .bp3-control-group .bp3-select select[class*="bp3-intent"]:disabled,
      .bp3-control-group .bp3-select select[class*="bp3-intent"].bp3-disabled{
        z-index:8; }
  .bp3-control-group .bp3-input-group > .bp3-icon,
  .bp3-control-group .bp3-input-group > .bp3-button,
  .bp3-control-group .bp3-input-group > .bp3-input-action{
    z-index:16; }
  .bp3-control-group .bp3-select::after,
  .bp3-control-group .bp3-html-select::after,
  .bp3-control-group .bp3-select > .bp3-icon,
  .bp3-control-group .bp3-html-select > .bp3-icon{
    z-index:17; }
  .bp3-control-group .bp3-select:focus-within{
    z-index:5; }
  .bp3-control-group:not(.bp3-vertical) > *:not(.bp3-divider){
    margin-right:-1px; }
  .bp3-control-group:not(.bp3-vertical) > .bp3-divider:not(:first-child){
    margin-left:6px; }
  .bp3-dark .bp3-control-group:not(.bp3-vertical) > *:not(.bp3-divider){
    margin-right:0; }
  .bp3-dark .bp3-control-group:not(.bp3-vertical) > .bp3-button + .bp3-button{
    margin-left:1px; }
  .bp3-control-group .bp3-popover-wrapper,
  .bp3-control-group .bp3-popover-target{
    border-radius:inherit; }
  .bp3-control-group > :first-child{
    border-radius:3px 0 0 3px; }
  .bp3-control-group > :last-child{
    border-radius:0 3px 3px 0;
    margin-right:0; }
  .bp3-control-group > :only-child{
    border-radius:3px;
    margin-right:0; }
  .bp3-control-group .bp3-input-group .bp3-button{
    border-radius:3px; }
  .bp3-control-group .bp3-numeric-input:not(:first-child) .bp3-input-group{
    border-bottom-left-radius:0;
    border-top-left-radius:0; }
  .bp3-control-group.bp3-fill{
    width:100%; }
  .bp3-control-group > .bp3-fill{
    -webkit-box-flex:1;
        -ms-flex:1 1 auto;
            flex:1 1 auto; }
  .bp3-control-group.bp3-fill > *:not(.bp3-fixed){
    -webkit-box-flex:1;
        -ms-flex:1 1 auto;
            flex:1 1 auto; }
  .bp3-control-group.bp3-vertical{
    -webkit-box-orient:vertical;
    -webkit-box-direction:normal;
        -ms-flex-direction:column;
            flex-direction:column; }
    .bp3-control-group.bp3-vertical > *{
      margin-top:-1px; }
    .bp3-control-group.bp3-vertical > :first-child{
      border-radius:3px 3px 0 0;
      margin-top:0; }
    .bp3-control-group.bp3-vertical > :last-child{
      border-radius:0 0 3px 3px; }
.bp3-control{
  cursor:pointer;
  display:block;
  margin-bottom:10px;
  position:relative;
  text-transform:none; }
  .bp3-control input:checked ~ .bp3-control-indicator{
    background-color:#137cbd;
    background-image:-webkit-gradient(linear, left top, left bottom, from(rgba(255, 255, 255, 0.1)), to(rgba(255, 255, 255, 0)));
    background-image:linear-gradient(to bottom, rgba(255, 255, 255, 0.1), rgba(255, 255, 255, 0));
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 -1px 0 rgba(16, 22, 26, 0.2);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 -1px 0 rgba(16, 22, 26, 0.2);
    color:#ffffff; }
  .bp3-control:hover input:checked ~ .bp3-control-indicator{
    background-color:#106ba3;
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 -1px 0 rgba(16, 22, 26, 0.2);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 -1px 0 rgba(16, 22, 26, 0.2); }
  .bp3-control input:not(:disabled):active:checked ~ .bp3-control-indicator{
    background:#0e5a8a;
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 1px 2px rgba(16, 22, 26, 0.2);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 1px 2px rgba(16, 22, 26, 0.2); }
  .bp3-control input:disabled:checked ~ .bp3-control-indicator{
    background:rgba(19, 124, 189, 0.5);
    -webkit-box-shadow:none;
            box-shadow:none; }
  .bp3-dark .bp3-control input:checked ~ .bp3-control-indicator{
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4); }
  .bp3-dark .bp3-control:hover input:checked ~ .bp3-control-indicator{
    background-color:#106ba3;
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4); }
  .bp3-dark .bp3-control input:not(:disabled):active:checked ~ .bp3-control-indicator{
    background-color:#0e5a8a;
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 1px 2px rgba(16, 22, 26, 0.2);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 1px 2px rgba(16, 22, 26, 0.2); }
  .bp3-dark .bp3-control input:disabled:checked ~ .bp3-control-indicator{
    background:rgba(14, 90, 138, 0.5);
    -webkit-box-shadow:none;
            box-shadow:none; }
  .bp3-control:not(.bp3-align-right){
    padding-left:26px; }
    .bp3-control:not(.bp3-align-right) .bp3-control-indicator{
      margin-left:-26px; }
  .bp3-control.bp3-align-right{
    padding-right:26px; }
    .bp3-control.bp3-align-right .bp3-control-indicator{
      margin-right:-26px; }
  .bp3-control.bp3-disabled{
    color:rgba(92, 112, 128, 0.6);
    cursor:not-allowed; }
  .bp3-control.bp3-inline{
    display:inline-block;
    margin-right:20px; }
  .bp3-control input{
    left:0;
    opacity:0;
    position:absolute;
    top:0;
    z-index:-1; }
  .bp3-control .bp3-control-indicator{
    background-clip:padding-box;
    background-color:#f5f8fa;
    background-image:-webkit-gradient(linear, left top, left bottom, from(rgba(255, 255, 255, 0.8)), to(rgba(255, 255, 255, 0)));
    background-image:linear-gradient(to bottom, rgba(255, 255, 255, 0.8), rgba(255, 255, 255, 0));
    border:none;
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 -1px 0 rgba(16, 22, 26, 0.1);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 -1px 0 rgba(16, 22, 26, 0.1);
    cursor:pointer;
    display:inline-block;
    font-size:16px;
    height:1em;
    margin-right:10px;
    margin-top:-3px;
    position:relative;
    -webkit-user-select:none;
       -moz-user-select:none;
        -ms-user-select:none;
            user-select:none;
    vertical-align:middle;
    width:1em; }
    .bp3-control .bp3-control-indicator::before{
      content:"";
      display:block;
      height:1em;
      width:1em; }
  .bp3-control:hover .bp3-control-indicator{
    background-color:#ebf1f5; }
  .bp3-control input:not(:disabled):active ~ .bp3-control-indicator{
    background:#d8e1e8;
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 1px 2px rgba(16, 22, 26, 0.2);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 1px 2px rgba(16, 22, 26, 0.2); }
  .bp3-control input:disabled ~ .bp3-control-indicator{
    background:rgba(206, 217, 224, 0.5);
    -webkit-box-shadow:none;
            box-shadow:none;
    cursor:not-allowed; }
  .bp3-control input:focus ~ .bp3-control-indicator{
    outline:rgba(19, 124, 189, 0.6) auto 2px;
    outline-offset:2px;
    -moz-outline-radius:6px; }
  .bp3-control.bp3-align-right .bp3-control-indicator{
    float:right;
    margin-left:10px;
    margin-top:1px; }
  .bp3-control.bp3-large{
    font-size:16px; }
    .bp3-control.bp3-large:not(.bp3-align-right){
      padding-left:30px; }
      .bp3-control.bp3-large:not(.bp3-align-right) .bp3-control-indicator{
        margin-left:-30px; }
    .bp3-control.bp3-large.bp3-align-right{
      padding-right:30px; }
      .bp3-control.bp3-large.bp3-align-right .bp3-control-indicator{
        margin-right:-30px; }
    .bp3-control.bp3-large .bp3-control-indicator{
      font-size:20px; }
    .bp3-control.bp3-large.bp3-align-right .bp3-control-indicator{
      margin-top:0; }
  .bp3-control.bp3-checkbox input:indeterminate ~ .bp3-control-indicator{
    background-color:#137cbd;
    background-image:-webkit-gradient(linear, left top, left bottom, from(rgba(255, 255, 255, 0.1)), to(rgba(255, 255, 255, 0)));
    background-image:linear-gradient(to bottom, rgba(255, 255, 255, 0.1), rgba(255, 255, 255, 0));
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 -1px 0 rgba(16, 22, 26, 0.2);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 -1px 0 rgba(16, 22, 26, 0.2);
    color:#ffffff; }
  .bp3-control.bp3-checkbox:hover input:indeterminate ~ .bp3-control-indicator{
    background-color:#106ba3;
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 -1px 0 rgba(16, 22, 26, 0.2);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 -1px 0 rgba(16, 22, 26, 0.2); }
  .bp3-control.bp3-checkbox input:not(:disabled):active:indeterminate ~ .bp3-control-indicator{
    background:#0e5a8a;
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 1px 2px rgba(16, 22, 26, 0.2);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 1px 2px rgba(16, 22, 26, 0.2); }
  .bp3-control.bp3-checkbox input:disabled:indeterminate ~ .bp3-control-indicator{
    background:rgba(19, 124, 189, 0.5);
    -webkit-box-shadow:none;
            box-shadow:none; }
  .bp3-dark .bp3-control.bp3-checkbox input:indeterminate ~ .bp3-control-indicator{
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4); }
  .bp3-dark .bp3-control.bp3-checkbox:hover input:indeterminate ~ .bp3-control-indicator{
    background-color:#106ba3;
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4); }
  .bp3-dark .bp3-control.bp3-checkbox input:not(:disabled):active:indeterminate ~ .bp3-control-indicator{
    background-color:#0e5a8a;
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 1px 2px rgba(16, 22, 26, 0.2);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4), inset 0 1px 2px rgba(16, 22, 26, 0.2); }
  .bp3-dark .bp3-control.bp3-checkbox input:disabled:indeterminate ~ .bp3-control-indicator{
    background:rgba(14, 90, 138, 0.5);
    -webkit-box-shadow:none;
            box-shadow:none; }
  .bp3-control.bp3-checkbox .bp3-control-indicator{
    border-radius:3px; }
  .bp3-control.bp3-checkbox input:checked ~ .bp3-control-indicator::before{
    background-image:url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'%3e%3cpath fill-rule='evenodd' clip-rule='evenodd' d='M12 5c-.28 0-.53.11-.71.29L7 9.59l-2.29-2.3a1.003 1.003 0 00-1.42 1.42l3 3c.18.18.43.29.71.29s.53-.11.71-.29l5-5A1.003 1.003 0 0012 5z' fill='white'/%3e%3c/svg%3e"); }
  .bp3-control.bp3-checkbox input:indeterminate ~ .bp3-control-indicator::before{
    background-image:url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'%3e%3cpath fill-rule='evenodd' clip-rule='evenodd' d='M11 7H5c-.55 0-1 .45-1 1s.45 1 1 1h6c.55 0 1-.45 1-1s-.45-1-1-1z' fill='white'/%3e%3c/svg%3e"); }
  .bp3-control.bp3-radio .bp3-control-indicator{
    border-radius:50%; }
  .bp3-control.bp3-radio input:checked ~ .bp3-control-indicator::before{
    background-image:radial-gradient(#ffffff, #ffffff 28%, transparent 32%); }
  .bp3-control.bp3-radio input:checked:disabled ~ .bp3-control-indicator::before{
    opacity:0.5; }
  .bp3-control.bp3-radio input:focus ~ .bp3-control-indicator{
    -moz-outline-radius:16px; }
  .bp3-control.bp3-switch input ~ .bp3-control-indicator{
    background:rgba(167, 182, 194, 0.5); }
  .bp3-control.bp3-switch:hover input ~ .bp3-control-indicator{
    background:rgba(115, 134, 148, 0.5); }
  .bp3-control.bp3-switch input:not(:disabled):active ~ .bp3-control-indicator{
    background:rgba(92, 112, 128, 0.5); }
  .bp3-control.bp3-switch input:disabled ~ .bp3-control-indicator{
    background:rgba(206, 217, 224, 0.5); }
    .bp3-control.bp3-switch input:disabled ~ .bp3-control-indicator::before{
      background:rgba(255, 255, 255, 0.8); }
  .bp3-control.bp3-switch input:checked ~ .bp3-control-indicator{
    background:#137cbd; }
  .bp3-control.bp3-switch:hover input:checked ~ .bp3-control-indicator{
    background:#106ba3; }
  .bp3-control.bp3-switch input:checked:not(:disabled):active ~ .bp3-control-indicator{
    background:#0e5a8a; }
  .bp3-control.bp3-switch input:checked:disabled ~ .bp3-control-indicator{
    background:rgba(19, 124, 189, 0.5); }
    .bp3-control.bp3-switch input:checked:disabled ~ .bp3-control-indicator::before{
      background:rgba(255, 255, 255, 0.8); }
  .bp3-control.bp3-switch:not(.bp3-align-right){
    padding-left:38px; }
    .bp3-control.bp3-switch:not(.bp3-align-right) .bp3-control-indicator{
      margin-left:-38px; }
  .bp3-control.bp3-switch.bp3-align-right{
    padding-right:38px; }
    .bp3-control.bp3-switch.bp3-align-right .bp3-control-indicator{
      margin-right:-38px; }
  .bp3-control.bp3-switch .bp3-control-indicator{
    border:none;
    border-radius:1.75em;
    -webkit-box-shadow:none !important;
            box-shadow:none !important;
    min-width:1.75em;
    -webkit-transition:background-color 100ms cubic-bezier(0.4, 1, 0.75, 0.9);
    transition:background-color 100ms cubic-bezier(0.4, 1, 0.75, 0.9);
    width:auto; }
    .bp3-control.bp3-switch .bp3-control-indicator::before{
      background:#ffffff;
      border-radius:50%;
      -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 1px 1px rgba(16, 22, 26, 0.2);
              box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 1px 1px rgba(16, 22, 26, 0.2);
      height:calc(1em - 4px);
      left:0;
      margin:2px;
      position:absolute;
      -webkit-transition:left 100ms cubic-bezier(0.4, 1, 0.75, 0.9);
      transition:left 100ms cubic-bezier(0.4, 1, 0.75, 0.9);
      width:calc(1em - 4px); }
  .bp3-control.bp3-switch input:checked ~ .bp3-control-indicator::before{
    left:calc(100% - 1em); }
  .bp3-control.bp3-switch.bp3-large:not(.bp3-align-right){
    padding-left:45px; }
    .bp3-control.bp3-switch.bp3-large:not(.bp3-align-right) .bp3-control-indicator{
      margin-left:-45px; }
  .bp3-control.bp3-switch.bp3-large.bp3-align-right{
    padding-right:45px; }
    .bp3-control.bp3-switch.bp3-large.bp3-align-right .bp3-control-indicator{
      margin-right:-45px; }
  .bp3-dark .bp3-control.bp3-switch input ~ .bp3-control-indicator{
    background:rgba(16, 22, 26, 0.5); }
  .bp3-dark .bp3-control.bp3-switch:hover input ~ .bp3-control-indicator{
    background:rgba(16, 22, 26, 0.7); }
  .bp3-dark .bp3-control.bp3-switch input:not(:disabled):active ~ .bp3-control-indicator{
    background:rgba(16, 22, 26, 0.9); }
  .bp3-dark .bp3-control.bp3-switch input:disabled ~ .bp3-control-indicator{
    background:rgba(57, 75, 89, 0.5); }
    .bp3-dark .bp3-control.bp3-switch input:disabled ~ .bp3-control-indicator::before{
      background:rgba(16, 22, 26, 0.4); }
  .bp3-dark .bp3-control.bp3-switch input:checked ~ .bp3-control-indicator{
    background:#137cbd; }
  .bp3-dark .bp3-control.bp3-switch:hover input:checked ~ .bp3-control-indicator{
    background:#106ba3; }
  .bp3-dark .bp3-control.bp3-switch input:checked:not(:disabled):active ~ .bp3-control-indicator{
    background:#0e5a8a; }
  .bp3-dark .bp3-control.bp3-switch input:checked:disabled ~ .bp3-control-indicator{
    background:rgba(14, 90, 138, 0.5); }
    .bp3-dark .bp3-control.bp3-switch input:checked:disabled ~ .bp3-control-indicator::before{
      background:rgba(16, 22, 26, 0.4); }
  .bp3-dark .bp3-control.bp3-switch .bp3-control-indicator::before{
    background:#394b59;
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4); }
  .bp3-dark .bp3-control.bp3-switch input:checked ~ .bp3-control-indicator::before{
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4); }
  .bp3-control.bp3-switch .bp3-switch-inner-text{
    font-size:0.7em;
    text-align:center; }
  .bp3-control.bp3-switch .bp3-control-indicator-child:first-child{
    line-height:0;
    margin-left:0.5em;
    margin-right:1.2em;
    visibility:hidden; }
  .bp3-control.bp3-switch .bp3-control-indicator-child:last-child{
    line-height:1em;
    margin-left:1.2em;
    margin-right:0.5em;
    visibility:visible; }
  .bp3-control.bp3-switch input:checked ~ .bp3-control-indicator .bp3-control-indicator-child:first-child{
    line-height:1em;
    visibility:visible; }
  .bp3-control.bp3-switch input:checked ~ .bp3-control-indicator .bp3-control-indicator-child:last-child{
    line-height:0;
    visibility:hidden; }
  .bp3-dark .bp3-control{
    color:#f5f8fa; }
    .bp3-dark .bp3-control.bp3-disabled{
      color:rgba(167, 182, 194, 0.6); }
    .bp3-dark .bp3-control .bp3-control-indicator{
      background-color:#394b59;
      background-image:-webkit-gradient(linear, left top, left bottom, from(rgba(255, 255, 255, 0.05)), to(rgba(255, 255, 255, 0)));
      background-image:linear-gradient(to bottom, rgba(255, 255, 255, 0.05), rgba(255, 255, 255, 0));
      -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4);
              box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4); }
    .bp3-dark .bp3-control:hover .bp3-control-indicator{
      background-color:#30404d; }
    .bp3-dark .bp3-control input:not(:disabled):active ~ .bp3-control-indicator{
      background:#202b33;
      -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.6), inset 0 1px 2px rgba(16, 22, 26, 0.2);
              box-shadow:0 0 0 1px rgba(16, 22, 26, 0.6), inset 0 1px 2px rgba(16, 22, 26, 0.2); }
    .bp3-dark .bp3-control input:disabled ~ .bp3-control-indicator{
      background:rgba(57, 75, 89, 0.5);
      -webkit-box-shadow:none;
              box-shadow:none;
      cursor:not-allowed; }
    .bp3-dark .bp3-control.bp3-checkbox input:disabled:checked ~ .bp3-control-indicator, .bp3-dark .bp3-control.bp3-checkbox input:disabled:indeterminate ~ .bp3-control-indicator{
      color:rgba(167, 182, 194, 0.6); }
.bp3-file-input{
  cursor:pointer;
  display:inline-block;
  height:30px;
  position:relative; }
  .bp3-file-input input{
    margin:0;
    min-width:200px;
    opacity:0; }
    .bp3-file-input input:disabled + .bp3-file-upload-input,
    .bp3-file-input input.bp3-disabled + .bp3-file-upload-input{
      background:rgba(206, 217, 224, 0.5);
      -webkit-box-shadow:none;
              box-shadow:none;
      color:rgba(92, 112, 128, 0.6);
      cursor:not-allowed;
      resize:none; }
      .bp3-file-input input:disabled + .bp3-file-upload-input::after,
      .bp3-file-input input.bp3-disabled + .bp3-file-upload-input::after{
        background-color:rgba(206, 217, 224, 0.5);
        background-image:none;
        -webkit-box-shadow:none;
                box-shadow:none;
        color:rgba(92, 112, 128, 0.6);
        cursor:not-allowed;
        outline:none; }
        .bp3-file-input input:disabled + .bp3-file-upload-input::after.bp3-active, .bp3-file-input input:disabled + .bp3-file-upload-input::after.bp3-active:hover,
        .bp3-file-input input.bp3-disabled + .bp3-file-upload-input::after.bp3-active,
        .bp3-file-input input.bp3-disabled + .bp3-file-upload-input::after.bp3-active:hover{
          background:rgba(206, 217, 224, 0.7); }
      .bp3-dark .bp3-file-input input:disabled + .bp3-file-upload-input, .bp3-dark
      .bp3-file-input input.bp3-disabled + .bp3-file-upload-input{
        background:rgba(57, 75, 89, 0.5);
        -webkit-box-shadow:none;
                box-shadow:none;
        color:rgba(167, 182, 194, 0.6); }
        .bp3-dark .bp3-file-input input:disabled + .bp3-file-upload-input::after, .bp3-dark
        .bp3-file-input input.bp3-disabled + .bp3-file-upload-input::after{
          background-color:rgba(57, 75, 89, 0.5);
          background-image:none;
          -webkit-box-shadow:none;
                  box-shadow:none;
          color:rgba(167, 182, 194, 0.6); }
          .bp3-dark .bp3-file-input input:disabled + .bp3-file-upload-input::after.bp3-active, .bp3-dark
          .bp3-file-input input.bp3-disabled + .bp3-file-upload-input::after.bp3-active{
            background:rgba(57, 75, 89, 0.7); }
  .bp3-file-input.bp3-file-input-has-selection .bp3-file-upload-input{
    color:#182026; }
  .bp3-dark .bp3-file-input.bp3-file-input-has-selection .bp3-file-upload-input{
    color:#f5f8fa; }
  .bp3-file-input.bp3-fill{
    width:100%; }
  .bp3-file-input.bp3-large,
  .bp3-large .bp3-file-input{
    height:40px; }
  .bp3-file-input .bp3-file-upload-input-custom-text::after{
    content:attr(bp3-button-text); }

.bp3-file-upload-input{
  -webkit-appearance:none;
     -moz-appearance:none;
          appearance:none;
  background:#ffffff;
  border:none;
  border-radius:3px;
  -webkit-box-shadow:0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), inset 0 0 0 1px rgba(16, 22, 26, 0.15), inset 0 1px 1px rgba(16, 22, 26, 0.2);
          box-shadow:0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), inset 0 0 0 1px rgba(16, 22, 26, 0.15), inset 0 1px 1px rgba(16, 22, 26, 0.2);
  color:#182026;
  font-size:14px;
  font-weight:400;
  height:30px;
  line-height:30px;
  outline:none;
  padding:0 10px;
  -webkit-transition:-webkit-box-shadow 100ms cubic-bezier(0.4, 1, 0.75, 0.9);
  transition:-webkit-box-shadow 100ms cubic-bezier(0.4, 1, 0.75, 0.9);
  transition:box-shadow 100ms cubic-bezier(0.4, 1, 0.75, 0.9);
  transition:box-shadow 100ms cubic-bezier(0.4, 1, 0.75, 0.9), -webkit-box-shadow 100ms cubic-bezier(0.4, 1, 0.75, 0.9);
  vertical-align:middle;
  overflow:hidden;
  text-overflow:ellipsis;
  white-space:nowrap;
  word-wrap:normal;
  color:rgba(92, 112, 128, 0.6);
  left:0;
  padding-right:80px;
  position:absolute;
  right:0;
  top:0;
  -webkit-user-select:none;
     -moz-user-select:none;
      -ms-user-select:none;
          user-select:none; }
  .bp3-file-upload-input::-webkit-input-placeholder{
    color:rgba(92, 112, 128, 0.6);
    opacity:1; }
  .bp3-file-upload-input::-moz-placeholder{
    color:rgba(92, 112, 128, 0.6);
    opacity:1; }
  .bp3-file-upload-input:-ms-input-placeholder{
    color:rgba(92, 112, 128, 0.6);
    opacity:1; }
  .bp3-file-upload-input::-ms-input-placeholder{
    color:rgba(92, 112, 128, 0.6);
    opacity:1; }
  .bp3-file-upload-input::placeholder{
    color:rgba(92, 112, 128, 0.6);
    opacity:1; }
  .bp3-file-upload-input:focus, .bp3-file-upload-input.bp3-active{
    -webkit-box-shadow:0 0 0 1px #137cbd, 0 0 0 3px rgba(19, 124, 189, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2);
            box-shadow:0 0 0 1px #137cbd, 0 0 0 3px rgba(19, 124, 189, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
  .bp3-file-upload-input[type="search"], .bp3-file-upload-input.bp3-round{
    border-radius:30px;
    -webkit-box-sizing:border-box;
            box-sizing:border-box;
    padding-left:10px; }
  .bp3-file-upload-input[readonly]{
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.15);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.15); }
  .bp3-file-upload-input:disabled, .bp3-file-upload-input.bp3-disabled{
    background:rgba(206, 217, 224, 0.5);
    -webkit-box-shadow:none;
            box-shadow:none;
    color:rgba(92, 112, 128, 0.6);
    cursor:not-allowed;
    resize:none; }
  .bp3-file-upload-input::after{
    background-color:#f5f8fa;
    background-image:-webkit-gradient(linear, left top, left bottom, from(rgba(255, 255, 255, 0.8)), to(rgba(255, 255, 255, 0)));
    background-image:linear-gradient(to bottom, rgba(255, 255, 255, 0.8), rgba(255, 255, 255, 0));
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 -1px 0 rgba(16, 22, 26, 0.1);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 -1px 0 rgba(16, 22, 26, 0.1);
    color:#182026;
    min-height:24px;
    min-width:24px;
    overflow:hidden;
    text-overflow:ellipsis;
    white-space:nowrap;
    word-wrap:normal;
    border-radius:3px;
    content:"Browse";
    line-height:24px;
    margin:3px;
    position:absolute;
    right:0;
    text-align:center;
    top:0;
    width:70px; }
    .bp3-file-upload-input::after:hover{
      background-clip:padding-box;
      background-color:#ebf1f5;
      -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 -1px 0 rgba(16, 22, 26, 0.1);
              box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 -1px 0 rgba(16, 22, 26, 0.1); }
    .bp3-file-upload-input::after:active, .bp3-file-upload-input::after.bp3-active{
      background-color:#d8e1e8;
      background-image:none;
      -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 1px 2px rgba(16, 22, 26, 0.2);
              box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 1px 2px rgba(16, 22, 26, 0.2); }
    .bp3-file-upload-input::after:disabled, .bp3-file-upload-input::after.bp3-disabled{
      background-color:rgba(206, 217, 224, 0.5);
      background-image:none;
      -webkit-box-shadow:none;
              box-shadow:none;
      color:rgba(92, 112, 128, 0.6);
      cursor:not-allowed;
      outline:none; }
      .bp3-file-upload-input::after:disabled.bp3-active, .bp3-file-upload-input::after:disabled.bp3-active:hover, .bp3-file-upload-input::after.bp3-disabled.bp3-active, .bp3-file-upload-input::after.bp3-disabled.bp3-active:hover{
        background:rgba(206, 217, 224, 0.7); }
  .bp3-file-upload-input:hover::after{
    background-clip:padding-box;
    background-color:#ebf1f5;
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 -1px 0 rgba(16, 22, 26, 0.1);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 -1px 0 rgba(16, 22, 26, 0.1); }
  .bp3-file-upload-input:active::after{
    background-color:#d8e1e8;
    background-image:none;
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 1px 2px rgba(16, 22, 26, 0.2);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 1px 2px rgba(16, 22, 26, 0.2); }
  .bp3-large .bp3-file-upload-input{
    font-size:16px;
    height:40px;
    line-height:40px;
    padding-right:95px; }
    .bp3-large .bp3-file-upload-input[type="search"], .bp3-large .bp3-file-upload-input.bp3-round{
      padding:0 15px; }
    .bp3-large .bp3-file-upload-input::after{
      min-height:30px;
      min-width:30px;
      line-height:30px;
      margin:5px;
      width:85px; }
  .bp3-dark .bp3-file-upload-input{
    background:rgba(16, 22, 26, 0.3);
    -webkit-box-shadow:0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
    color:#f5f8fa;
    color:rgba(167, 182, 194, 0.6); }
    .bp3-dark .bp3-file-upload-input::-webkit-input-placeholder{
      color:rgba(167, 182, 194, 0.6); }
    .bp3-dark .bp3-file-upload-input::-moz-placeholder{
      color:rgba(167, 182, 194, 0.6); }
    .bp3-dark .bp3-file-upload-input:-ms-input-placeholder{
      color:rgba(167, 182, 194, 0.6); }
    .bp3-dark .bp3-file-upload-input::-ms-input-placeholder{
      color:rgba(167, 182, 194, 0.6); }
    .bp3-dark .bp3-file-upload-input::placeholder{
      color:rgba(167, 182, 194, 0.6); }
    .bp3-dark .bp3-file-upload-input:focus{
      -webkit-box-shadow:0 0 0 1px #137cbd, 0 0 0 1px #137cbd, 0 0 0 3px rgba(19, 124, 189, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
              box-shadow:0 0 0 1px #137cbd, 0 0 0 1px #137cbd, 0 0 0 3px rgba(19, 124, 189, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4); }
    .bp3-dark .bp3-file-upload-input[readonly]{
      -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4);
              box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4); }
    .bp3-dark .bp3-file-upload-input:disabled, .bp3-dark .bp3-file-upload-input.bp3-disabled{
      background:rgba(57, 75, 89, 0.5);
      -webkit-box-shadow:none;
              box-shadow:none;
      color:rgba(167, 182, 194, 0.6); }
    .bp3-dark .bp3-file-upload-input::after{
      background-color:#394b59;
      background-image:-webkit-gradient(linear, left top, left bottom, from(rgba(255, 255, 255, 0.05)), to(rgba(255, 255, 255, 0)));
      background-image:linear-gradient(to bottom, rgba(255, 255, 255, 0.05), rgba(255, 255, 255, 0));
      -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4);
              box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4);
      color:#f5f8fa; }
      .bp3-dark .bp3-file-upload-input::after:hover, .bp3-dark .bp3-file-upload-input::after:active, .bp3-dark .bp3-file-upload-input::after.bp3-active{
        color:#f5f8fa; }
      .bp3-dark .bp3-file-upload-input::after:hover{
        background-color:#30404d;
        -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4);
                box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4); }
      .bp3-dark .bp3-file-upload-input::after:active, .bp3-dark .bp3-file-upload-input::after.bp3-active{
        background-color:#202b33;
        background-image:none;
        -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.6), inset 0 1px 2px rgba(16, 22, 26, 0.2);
                box-shadow:0 0 0 1px rgba(16, 22, 26, 0.6), inset 0 1px 2px rgba(16, 22, 26, 0.2); }
      .bp3-dark .bp3-file-upload-input::after:disabled, .bp3-dark .bp3-file-upload-input::after.bp3-disabled{
        background-color:rgba(57, 75, 89, 0.5);
        background-image:none;
        -webkit-box-shadow:none;
                box-shadow:none;
        color:rgba(167, 182, 194, 0.6); }
        .bp3-dark .bp3-file-upload-input::after:disabled.bp3-active, .bp3-dark .bp3-file-upload-input::after.bp3-disabled.bp3-active{
          background:rgba(57, 75, 89, 0.7); }
      .bp3-dark .bp3-file-upload-input::after .bp3-button-spinner .bp3-spinner-head{
        background:rgba(16, 22, 26, 0.5);
        stroke:#8a9ba8; }
    .bp3-dark .bp3-file-upload-input:hover::after{
      background-color:#30404d;
      -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4);
              box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4); }
    .bp3-dark .bp3-file-upload-input:active::after{
      background-color:#202b33;
      background-image:none;
      -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.6), inset 0 1px 2px rgba(16, 22, 26, 0.2);
              box-shadow:0 0 0 1px rgba(16, 22, 26, 0.6), inset 0 1px 2px rgba(16, 22, 26, 0.2); }
.bp3-file-upload-input::after{
  -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 -1px 0 rgba(16, 22, 26, 0.1);
          box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 -1px 0 rgba(16, 22, 26, 0.1); }
.bp3-form-group{
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex;
  -webkit-box-orient:vertical;
  -webkit-box-direction:normal;
      -ms-flex-direction:column;
          flex-direction:column;
  margin:0 0 15px; }
  .bp3-form-group label.bp3-label{
    margin-bottom:5px; }
  .bp3-form-group .bp3-control{
    margin-top:7px; }
  .bp3-form-group .bp3-form-helper-text{
    color:#5c7080;
    font-size:12px;
    margin-top:5px; }
  .bp3-form-group.bp3-intent-primary .bp3-form-helper-text{
    color:#106ba3; }
  .bp3-form-group.bp3-intent-success .bp3-form-helper-text{
    color:#0d8050; }
  .bp3-form-group.bp3-intent-warning .bp3-form-helper-text{
    color:#bf7326; }
  .bp3-form-group.bp3-intent-danger .bp3-form-helper-text{
    color:#c23030; }
  .bp3-form-group.bp3-inline{
    -webkit-box-align:start;
        -ms-flex-align:start;
            align-items:flex-start;
    -webkit-box-orient:horizontal;
    -webkit-box-direction:normal;
        -ms-flex-direction:row;
            flex-direction:row; }
    .bp3-form-group.bp3-inline.bp3-large label.bp3-label{
      line-height:40px;
      margin:0 10px 0 0; }
    .bp3-form-group.bp3-inline label.bp3-label{
      line-height:30px;
      margin:0 10px 0 0; }
  .bp3-form-group.bp3-disabled .bp3-label,
  .bp3-form-group.bp3-disabled .bp3-text-muted,
  .bp3-form-group.bp3-disabled .bp3-form-helper-text{
    color:rgba(92, 112, 128, 0.6) !important; }
  .bp3-dark .bp3-form-group.bp3-intent-primary .bp3-form-helper-text{
    color:#48aff0; }
  .bp3-dark .bp3-form-group.bp3-intent-success .bp3-form-helper-text{
    color:#3dcc91; }
  .bp3-dark .bp3-form-group.bp3-intent-warning .bp3-form-helper-text{
    color:#ffb366; }
  .bp3-dark .bp3-form-group.bp3-intent-danger .bp3-form-helper-text{
    color:#ff7373; }
  .bp3-dark .bp3-form-group .bp3-form-helper-text{
    color:#a7b6c2; }
  .bp3-dark .bp3-form-group.bp3-disabled .bp3-label,
  .bp3-dark .bp3-form-group.bp3-disabled .bp3-text-muted,
  .bp3-dark .bp3-form-group.bp3-disabled .bp3-form-helper-text{
    color:rgba(167, 182, 194, 0.6) !important; }
.bp3-input-group{
  display:block;
  position:relative; }
  .bp3-input-group .bp3-input{
    position:relative;
    width:100%; }
    .bp3-input-group .bp3-input:not(:first-child){
      padding-left:30px; }
    .bp3-input-group .bp3-input:not(:last-child){
      padding-right:30px; }
  .bp3-input-group .bp3-input-action,
  .bp3-input-group > .bp3-input-left-container,
  .bp3-input-group > .bp3-button,
  .bp3-input-group > .bp3-icon{
    position:absolute;
    top:0; }
    .bp3-input-group .bp3-input-action:first-child,
    .bp3-input-group > .bp3-input-left-container:first-child,
    .bp3-input-group > .bp3-button:first-child,
    .bp3-input-group > .bp3-icon:first-child{
      left:0; }
    .bp3-input-group .bp3-input-action:last-child,
    .bp3-input-group > .bp3-input-left-container:last-child,
    .bp3-input-group > .bp3-button:last-child,
    .bp3-input-group > .bp3-icon:last-child{
      right:0; }
  .bp3-input-group .bp3-button{
    min-height:24px;
    min-width:24px;
    margin:3px;
    padding:0 7px; }
    .bp3-input-group .bp3-button:empty{
      padding:0; }
  .bp3-input-group > .bp3-input-left-container,
  .bp3-input-group > .bp3-icon{
    z-index:1; }
  .bp3-input-group > .bp3-input-left-container > .bp3-icon,
  .bp3-input-group > .bp3-icon{
    color:#5c7080; }
    .bp3-input-group > .bp3-input-left-container > .bp3-icon:empty,
    .bp3-input-group > .bp3-icon:empty{
      font-family:"Icons16", sans-serif;
      font-size:16px;
      font-style:normal;
      font-weight:400;
      line-height:1;
      -moz-osx-font-smoothing:grayscale;
      -webkit-font-smoothing:antialiased; }
  .bp3-input-group > .bp3-input-left-container > .bp3-icon,
  .bp3-input-group > .bp3-icon,
  .bp3-input-group .bp3-input-action > .bp3-spinner{
    margin:7px; }
  .bp3-input-group .bp3-tag{
    margin:5px; }
  .bp3-input-group .bp3-input:not(:focus) + .bp3-button.bp3-minimal:not(:hover):not(:focus),
  .bp3-input-group .bp3-input:not(:focus) + .bp3-input-action .bp3-button.bp3-minimal:not(:hover):not(:focus){
    color:#5c7080; }
    .bp3-dark .bp3-input-group .bp3-input:not(:focus) + .bp3-button.bp3-minimal:not(:hover):not(:focus), .bp3-dark
    .bp3-input-group .bp3-input:not(:focus) + .bp3-input-action .bp3-button.bp3-minimal:not(:hover):not(:focus){
      color:#a7b6c2; }
    .bp3-input-group .bp3-input:not(:focus) + .bp3-button.bp3-minimal:not(:hover):not(:focus) .bp3-icon, .bp3-input-group .bp3-input:not(:focus) + .bp3-button.bp3-minimal:not(:hover):not(:focus) .bp3-icon-standard, .bp3-input-group .bp3-input:not(:focus) + .bp3-button.bp3-minimal:not(:hover):not(:focus) .bp3-icon-large,
    .bp3-input-group .bp3-input:not(:focus) + .bp3-input-action .bp3-button.bp3-minimal:not(:hover):not(:focus) .bp3-icon,
    .bp3-input-group .bp3-input:not(:focus) + .bp3-input-action .bp3-button.bp3-minimal:not(:hover):not(:focus) .bp3-icon-standard,
    .bp3-input-group .bp3-input:not(:focus) + .bp3-input-action .bp3-button.bp3-minimal:not(:hover):not(:focus) .bp3-icon-large{
      color:#5c7080; }
  .bp3-input-group .bp3-input:not(:focus) + .bp3-button.bp3-minimal:disabled,
  .bp3-input-group .bp3-input:not(:focus) + .bp3-input-action .bp3-button.bp3-minimal:disabled{
    color:rgba(92, 112, 128, 0.6) !important; }
    .bp3-input-group .bp3-input:not(:focus) + .bp3-button.bp3-minimal:disabled .bp3-icon, .bp3-input-group .bp3-input:not(:focus) + .bp3-button.bp3-minimal:disabled .bp3-icon-standard, .bp3-input-group .bp3-input:not(:focus) + .bp3-button.bp3-minimal:disabled .bp3-icon-large,
    .bp3-input-group .bp3-input:not(:focus) + .bp3-input-action .bp3-button.bp3-minimal:disabled .bp3-icon,
    .bp3-input-group .bp3-input:not(:focus) + .bp3-input-action .bp3-button.bp3-minimal:disabled .bp3-icon-standard,
    .bp3-input-group .bp3-input:not(:focus) + .bp3-input-action .bp3-button.bp3-minimal:disabled .bp3-icon-large{
      color:rgba(92, 112, 128, 0.6) !important; }
  .bp3-input-group.bp3-disabled{
    cursor:not-allowed; }
    .bp3-input-group.bp3-disabled .bp3-icon{
      color:rgba(92, 112, 128, 0.6); }
  .bp3-input-group.bp3-large .bp3-button{
    min-height:30px;
    min-width:30px;
    margin:5px; }
  .bp3-input-group.bp3-large > .bp3-input-left-container > .bp3-icon,
  .bp3-input-group.bp3-large > .bp3-icon,
  .bp3-input-group.bp3-large .bp3-input-action > .bp3-spinner{
    margin:12px; }
  .bp3-input-group.bp3-large .bp3-input{
    font-size:16px;
    height:40px;
    line-height:40px; }
    .bp3-input-group.bp3-large .bp3-input[type="search"], .bp3-input-group.bp3-large .bp3-input.bp3-round{
      padding:0 15px; }
    .bp3-input-group.bp3-large .bp3-input:not(:first-child){
      padding-left:40px; }
    .bp3-input-group.bp3-large .bp3-input:not(:last-child){
      padding-right:40px; }
  .bp3-input-group.bp3-small .bp3-button{
    min-height:20px;
    min-width:20px;
    margin:2px; }
  .bp3-input-group.bp3-small .bp3-tag{
    min-height:20px;
    min-width:20px;
    margin:2px; }
  .bp3-input-group.bp3-small > .bp3-input-left-container > .bp3-icon,
  .bp3-input-group.bp3-small > .bp3-icon,
  .bp3-input-group.bp3-small .bp3-input-action > .bp3-spinner{
    margin:4px; }
  .bp3-input-group.bp3-small .bp3-input{
    font-size:12px;
    height:24px;
    line-height:24px;
    padding-left:8px;
    padding-right:8px; }
    .bp3-input-group.bp3-small .bp3-input[type="search"], .bp3-input-group.bp3-small .bp3-input.bp3-round{
      padding:0 12px; }
    .bp3-input-group.bp3-small .bp3-input:not(:first-child){
      padding-left:24px; }
    .bp3-input-group.bp3-small .bp3-input:not(:last-child){
      padding-right:24px; }
  .bp3-input-group.bp3-fill{
    -webkit-box-flex:1;
        -ms-flex:1 1 auto;
            flex:1 1 auto;
    width:100%; }
  .bp3-input-group.bp3-round .bp3-button,
  .bp3-input-group.bp3-round .bp3-input,
  .bp3-input-group.bp3-round .bp3-tag{
    border-radius:30px; }
  .bp3-dark .bp3-input-group .bp3-icon{
    color:#a7b6c2; }
  .bp3-dark .bp3-input-group.bp3-disabled .bp3-icon{
    color:rgba(167, 182, 194, 0.6); }
  .bp3-input-group.bp3-intent-primary .bp3-input{
    -webkit-box-shadow:0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), inset 0 0 0 1px #137cbd, inset 0 0 0 1px rgba(16, 22, 26, 0.15), inset 0 1px 1px rgba(16, 22, 26, 0.2);
            box-shadow:0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), inset 0 0 0 1px #137cbd, inset 0 0 0 1px rgba(16, 22, 26, 0.15), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
    .bp3-input-group.bp3-intent-primary .bp3-input:focus{
      -webkit-box-shadow:0 0 0 1px #137cbd, 0 0 0 3px rgba(19, 124, 189, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2);
              box-shadow:0 0 0 1px #137cbd, 0 0 0 3px rgba(19, 124, 189, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
    .bp3-input-group.bp3-intent-primary .bp3-input[readonly]{
      -webkit-box-shadow:inset 0 0 0 1px #137cbd;
              box-shadow:inset 0 0 0 1px #137cbd; }
    .bp3-input-group.bp3-intent-primary .bp3-input:disabled, .bp3-input-group.bp3-intent-primary .bp3-input.bp3-disabled{
      -webkit-box-shadow:none;
              box-shadow:none; }
  .bp3-input-group.bp3-intent-primary > .bp3-icon{
    color:#106ba3; }
    .bp3-dark .bp3-input-group.bp3-intent-primary > .bp3-icon{
      color:#48aff0; }
  .bp3-input-group.bp3-intent-success .bp3-input{
    -webkit-box-shadow:0 0 0 0 rgba(15, 153, 96, 0), 0 0 0 0 rgba(15, 153, 96, 0), inset 0 0 0 1px #0f9960, inset 0 0 0 1px rgba(16, 22, 26, 0.15), inset 0 1px 1px rgba(16, 22, 26, 0.2);
            box-shadow:0 0 0 0 rgba(15, 153, 96, 0), 0 0 0 0 rgba(15, 153, 96, 0), inset 0 0 0 1px #0f9960, inset 0 0 0 1px rgba(16, 22, 26, 0.15), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
    .bp3-input-group.bp3-intent-success .bp3-input:focus{
      -webkit-box-shadow:0 0 0 1px #0f9960, 0 0 0 3px rgba(15, 153, 96, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2);
              box-shadow:0 0 0 1px #0f9960, 0 0 0 3px rgba(15, 153, 96, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
    .bp3-input-group.bp3-intent-success .bp3-input[readonly]{
      -webkit-box-shadow:inset 0 0 0 1px #0f9960;
              box-shadow:inset 0 0 0 1px #0f9960; }
    .bp3-input-group.bp3-intent-success .bp3-input:disabled, .bp3-input-group.bp3-intent-success .bp3-input.bp3-disabled{
      -webkit-box-shadow:none;
              box-shadow:none; }
  .bp3-input-group.bp3-intent-success > .bp3-icon{
    color:#0d8050; }
    .bp3-dark .bp3-input-group.bp3-intent-success > .bp3-icon{
      color:#3dcc91; }
  .bp3-input-group.bp3-intent-warning .bp3-input{
    -webkit-box-shadow:0 0 0 0 rgba(217, 130, 43, 0), 0 0 0 0 rgba(217, 130, 43, 0), inset 0 0 0 1px #d9822b, inset 0 0 0 1px rgba(16, 22, 26, 0.15), inset 0 1px 1px rgba(16, 22, 26, 0.2);
            box-shadow:0 0 0 0 rgba(217, 130, 43, 0), 0 0 0 0 rgba(217, 130, 43, 0), inset 0 0 0 1px #d9822b, inset 0 0 0 1px rgba(16, 22, 26, 0.15), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
    .bp3-input-group.bp3-intent-warning .bp3-input:focus{
      -webkit-box-shadow:0 0 0 1px #d9822b, 0 0 0 3px rgba(217, 130, 43, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2);
              box-shadow:0 0 0 1px #d9822b, 0 0 0 3px rgba(217, 130, 43, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
    .bp3-input-group.bp3-intent-warning .bp3-input[readonly]{
      -webkit-box-shadow:inset 0 0 0 1px #d9822b;
              box-shadow:inset 0 0 0 1px #d9822b; }
    .bp3-input-group.bp3-intent-warning .bp3-input:disabled, .bp3-input-group.bp3-intent-warning .bp3-input.bp3-disabled{
      -webkit-box-shadow:none;
              box-shadow:none; }
  .bp3-input-group.bp3-intent-warning > .bp3-icon{
    color:#bf7326; }
    .bp3-dark .bp3-input-group.bp3-intent-warning > .bp3-icon{
      color:#ffb366; }
  .bp3-input-group.bp3-intent-danger .bp3-input{
    -webkit-box-shadow:0 0 0 0 rgba(219, 55, 55, 0), 0 0 0 0 rgba(219, 55, 55, 0), inset 0 0 0 1px #db3737, inset 0 0 0 1px rgba(16, 22, 26, 0.15), inset 0 1px 1px rgba(16, 22, 26, 0.2);
            box-shadow:0 0 0 0 rgba(219, 55, 55, 0), 0 0 0 0 rgba(219, 55, 55, 0), inset 0 0 0 1px #db3737, inset 0 0 0 1px rgba(16, 22, 26, 0.15), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
    .bp3-input-group.bp3-intent-danger .bp3-input:focus{
      -webkit-box-shadow:0 0 0 1px #db3737, 0 0 0 3px rgba(219, 55, 55, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2);
              box-shadow:0 0 0 1px #db3737, 0 0 0 3px rgba(219, 55, 55, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
    .bp3-input-group.bp3-intent-danger .bp3-input[readonly]{
      -webkit-box-shadow:inset 0 0 0 1px #db3737;
              box-shadow:inset 0 0 0 1px #db3737; }
    .bp3-input-group.bp3-intent-danger .bp3-input:disabled, .bp3-input-group.bp3-intent-danger .bp3-input.bp3-disabled{
      -webkit-box-shadow:none;
              box-shadow:none; }
  .bp3-input-group.bp3-intent-danger > .bp3-icon{
    color:#c23030; }
    .bp3-dark .bp3-input-group.bp3-intent-danger > .bp3-icon{
      color:#ff7373; }
.bp3-input{
  -webkit-appearance:none;
     -moz-appearance:none;
          appearance:none;
  background:#ffffff;
  border:none;
  border-radius:3px;
  -webkit-box-shadow:0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), inset 0 0 0 1px rgba(16, 22, 26, 0.15), inset 0 1px 1px rgba(16, 22, 26, 0.2);
          box-shadow:0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), inset 0 0 0 1px rgba(16, 22, 26, 0.15), inset 0 1px 1px rgba(16, 22, 26, 0.2);
  color:#182026;
  font-size:14px;
  font-weight:400;
  height:30px;
  line-height:30px;
  outline:none;
  padding:0 10px;
  -webkit-transition:-webkit-box-shadow 100ms cubic-bezier(0.4, 1, 0.75, 0.9);
  transition:-webkit-box-shadow 100ms cubic-bezier(0.4, 1, 0.75, 0.9);
  transition:box-shadow 100ms cubic-bezier(0.4, 1, 0.75, 0.9);
  transition:box-shadow 100ms cubic-bezier(0.4, 1, 0.75, 0.9), -webkit-box-shadow 100ms cubic-bezier(0.4, 1, 0.75, 0.9);
  vertical-align:middle; }
  .bp3-input::-webkit-input-placeholder{
    color:rgba(92, 112, 128, 0.6);
    opacity:1; }
  .bp3-input::-moz-placeholder{
    color:rgba(92, 112, 128, 0.6);
    opacity:1; }
  .bp3-input:-ms-input-placeholder{
    color:rgba(92, 112, 128, 0.6);
    opacity:1; }
  .bp3-input::-ms-input-placeholder{
    color:rgba(92, 112, 128, 0.6);
    opacity:1; }
  .bp3-input::placeholder{
    color:rgba(92, 112, 128, 0.6);
    opacity:1; }
  .bp3-input:focus, .bp3-input.bp3-active{
    -webkit-box-shadow:0 0 0 1px #137cbd, 0 0 0 3px rgba(19, 124, 189, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2);
            box-shadow:0 0 0 1px #137cbd, 0 0 0 3px rgba(19, 124, 189, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
  .bp3-input[type="search"], .bp3-input.bp3-round{
    border-radius:30px;
    -webkit-box-sizing:border-box;
            box-sizing:border-box;
    padding-left:10px; }
  .bp3-input[readonly]{
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.15);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.15); }
  .bp3-input:disabled, .bp3-input.bp3-disabled{
    background:rgba(206, 217, 224, 0.5);
    -webkit-box-shadow:none;
            box-shadow:none;
    color:rgba(92, 112, 128, 0.6);
    cursor:not-allowed;
    resize:none; }
  .bp3-input.bp3-large{
    font-size:16px;
    height:40px;
    line-height:40px; }
    .bp3-input.bp3-large[type="search"], .bp3-input.bp3-large.bp3-round{
      padding:0 15px; }
  .bp3-input.bp3-small{
    font-size:12px;
    height:24px;
    line-height:24px;
    padding-left:8px;
    padding-right:8px; }
    .bp3-input.bp3-small[type="search"], .bp3-input.bp3-small.bp3-round{
      padding:0 12px; }
  .bp3-input.bp3-fill{
    -webkit-box-flex:1;
        -ms-flex:1 1 auto;
            flex:1 1 auto;
    width:100%; }
  .bp3-dark .bp3-input{
    background:rgba(16, 22, 26, 0.3);
    -webkit-box-shadow:0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
    color:#f5f8fa; }
    .bp3-dark .bp3-input::-webkit-input-placeholder{
      color:rgba(167, 182, 194, 0.6); }
    .bp3-dark .bp3-input::-moz-placeholder{
      color:rgba(167, 182, 194, 0.6); }
    .bp3-dark .bp3-input:-ms-input-placeholder{
      color:rgba(167, 182, 194, 0.6); }
    .bp3-dark .bp3-input::-ms-input-placeholder{
      color:rgba(167, 182, 194, 0.6); }
    .bp3-dark .bp3-input::placeholder{
      color:rgba(167, 182, 194, 0.6); }
    .bp3-dark .bp3-input:focus{
      -webkit-box-shadow:0 0 0 1px #137cbd, 0 0 0 1px #137cbd, 0 0 0 3px rgba(19, 124, 189, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
              box-shadow:0 0 0 1px #137cbd, 0 0 0 1px #137cbd, 0 0 0 3px rgba(19, 124, 189, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4); }
    .bp3-dark .bp3-input[readonly]{
      -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4);
              box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4); }
    .bp3-dark .bp3-input:disabled, .bp3-dark .bp3-input.bp3-disabled{
      background:rgba(57, 75, 89, 0.5);
      -webkit-box-shadow:none;
              box-shadow:none;
      color:rgba(167, 182, 194, 0.6); }
  .bp3-input.bp3-intent-primary{
    -webkit-box-shadow:0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), inset 0 0 0 1px #137cbd, inset 0 0 0 1px rgba(16, 22, 26, 0.15), inset 0 1px 1px rgba(16, 22, 26, 0.2);
            box-shadow:0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), inset 0 0 0 1px #137cbd, inset 0 0 0 1px rgba(16, 22, 26, 0.15), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
    .bp3-input.bp3-intent-primary:focus{
      -webkit-box-shadow:0 0 0 1px #137cbd, 0 0 0 3px rgba(19, 124, 189, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2);
              box-shadow:0 0 0 1px #137cbd, 0 0 0 3px rgba(19, 124, 189, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
    .bp3-input.bp3-intent-primary[readonly]{
      -webkit-box-shadow:inset 0 0 0 1px #137cbd;
              box-shadow:inset 0 0 0 1px #137cbd; }
    .bp3-input.bp3-intent-primary:disabled, .bp3-input.bp3-intent-primary.bp3-disabled{
      -webkit-box-shadow:none;
              box-shadow:none; }
    .bp3-dark .bp3-input.bp3-intent-primary{
      -webkit-box-shadow:0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), inset 0 0 0 1px #137cbd, inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
              box-shadow:0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), inset 0 0 0 1px #137cbd, inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4); }
      .bp3-dark .bp3-input.bp3-intent-primary:focus{
        -webkit-box-shadow:0 0 0 1px #137cbd, 0 0 0 1px #137cbd, 0 0 0 3px rgba(19, 124, 189, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
                box-shadow:0 0 0 1px #137cbd, 0 0 0 1px #137cbd, 0 0 0 3px rgba(19, 124, 189, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4); }
      .bp3-dark .bp3-input.bp3-intent-primary[readonly]{
        -webkit-box-shadow:inset 0 0 0 1px #137cbd;
                box-shadow:inset 0 0 0 1px #137cbd; }
      .bp3-dark .bp3-input.bp3-intent-primary:disabled, .bp3-dark .bp3-input.bp3-intent-primary.bp3-disabled{
        -webkit-box-shadow:none;
                box-shadow:none; }
  .bp3-input.bp3-intent-success{
    -webkit-box-shadow:0 0 0 0 rgba(15, 153, 96, 0), 0 0 0 0 rgba(15, 153, 96, 0), inset 0 0 0 1px #0f9960, inset 0 0 0 1px rgba(16, 22, 26, 0.15), inset 0 1px 1px rgba(16, 22, 26, 0.2);
            box-shadow:0 0 0 0 rgba(15, 153, 96, 0), 0 0 0 0 rgba(15, 153, 96, 0), inset 0 0 0 1px #0f9960, inset 0 0 0 1px rgba(16, 22, 26, 0.15), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
    .bp3-input.bp3-intent-success:focus{
      -webkit-box-shadow:0 0 0 1px #0f9960, 0 0 0 3px rgba(15, 153, 96, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2);
              box-shadow:0 0 0 1px #0f9960, 0 0 0 3px rgba(15, 153, 96, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
    .bp3-input.bp3-intent-success[readonly]{
      -webkit-box-shadow:inset 0 0 0 1px #0f9960;
              box-shadow:inset 0 0 0 1px #0f9960; }
    .bp3-input.bp3-intent-success:disabled, .bp3-input.bp3-intent-success.bp3-disabled{
      -webkit-box-shadow:none;
              box-shadow:none; }
    .bp3-dark .bp3-input.bp3-intent-success{
      -webkit-box-shadow:0 0 0 0 rgba(15, 153, 96, 0), 0 0 0 0 rgba(15, 153, 96, 0), 0 0 0 0 rgba(15, 153, 96, 0), inset 0 0 0 1px #0f9960, inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
              box-shadow:0 0 0 0 rgba(15, 153, 96, 0), 0 0 0 0 rgba(15, 153, 96, 0), 0 0 0 0 rgba(15, 153, 96, 0), inset 0 0 0 1px #0f9960, inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4); }
      .bp3-dark .bp3-input.bp3-intent-success:focus{
        -webkit-box-shadow:0 0 0 1px #0f9960, 0 0 0 1px #0f9960, 0 0 0 3px rgba(15, 153, 96, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
                box-shadow:0 0 0 1px #0f9960, 0 0 0 1px #0f9960, 0 0 0 3px rgba(15, 153, 96, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4); }
      .bp3-dark .bp3-input.bp3-intent-success[readonly]{
        -webkit-box-shadow:inset 0 0 0 1px #0f9960;
                box-shadow:inset 0 0 0 1px #0f9960; }
      .bp3-dark .bp3-input.bp3-intent-success:disabled, .bp3-dark .bp3-input.bp3-intent-success.bp3-disabled{
        -webkit-box-shadow:none;
                box-shadow:none; }
  .bp3-input.bp3-intent-warning{
    -webkit-box-shadow:0 0 0 0 rgba(217, 130, 43, 0), 0 0 0 0 rgba(217, 130, 43, 0), inset 0 0 0 1px #d9822b, inset 0 0 0 1px rgba(16, 22, 26, 0.15), inset 0 1px 1px rgba(16, 22, 26, 0.2);
            box-shadow:0 0 0 0 rgba(217, 130, 43, 0), 0 0 0 0 rgba(217, 130, 43, 0), inset 0 0 0 1px #d9822b, inset 0 0 0 1px rgba(16, 22, 26, 0.15), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
    .bp3-input.bp3-intent-warning:focus{
      -webkit-box-shadow:0 0 0 1px #d9822b, 0 0 0 3px rgba(217, 130, 43, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2);
              box-shadow:0 0 0 1px #d9822b, 0 0 0 3px rgba(217, 130, 43, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
    .bp3-input.bp3-intent-warning[readonly]{
      -webkit-box-shadow:inset 0 0 0 1px #d9822b;
              box-shadow:inset 0 0 0 1px #d9822b; }
    .bp3-input.bp3-intent-warning:disabled, .bp3-input.bp3-intent-warning.bp3-disabled{
      -webkit-box-shadow:none;
              box-shadow:none; }
    .bp3-dark .bp3-input.bp3-intent-warning{
      -webkit-box-shadow:0 0 0 0 rgba(217, 130, 43, 0), 0 0 0 0 rgba(217, 130, 43, 0), 0 0 0 0 rgba(217, 130, 43, 0), inset 0 0 0 1px #d9822b, inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
              box-shadow:0 0 0 0 rgba(217, 130, 43, 0), 0 0 0 0 rgba(217, 130, 43, 0), 0 0 0 0 rgba(217, 130, 43, 0), inset 0 0 0 1px #d9822b, inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4); }
      .bp3-dark .bp3-input.bp3-intent-warning:focus{
        -webkit-box-shadow:0 0 0 1px #d9822b, 0 0 0 1px #d9822b, 0 0 0 3px rgba(217, 130, 43, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
                box-shadow:0 0 0 1px #d9822b, 0 0 0 1px #d9822b, 0 0 0 3px rgba(217, 130, 43, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4); }
      .bp3-dark .bp3-input.bp3-intent-warning[readonly]{
        -webkit-box-shadow:inset 0 0 0 1px #d9822b;
                box-shadow:inset 0 0 0 1px #d9822b; }
      .bp3-dark .bp3-input.bp3-intent-warning:disabled, .bp3-dark .bp3-input.bp3-intent-warning.bp3-disabled{
        -webkit-box-shadow:none;
                box-shadow:none; }
  .bp3-input.bp3-intent-danger{
    -webkit-box-shadow:0 0 0 0 rgba(219, 55, 55, 0), 0 0 0 0 rgba(219, 55, 55, 0), inset 0 0 0 1px #db3737, inset 0 0 0 1px rgba(16, 22, 26, 0.15), inset 0 1px 1px rgba(16, 22, 26, 0.2);
            box-shadow:0 0 0 0 rgba(219, 55, 55, 0), 0 0 0 0 rgba(219, 55, 55, 0), inset 0 0 0 1px #db3737, inset 0 0 0 1px rgba(16, 22, 26, 0.15), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
    .bp3-input.bp3-intent-danger:focus{
      -webkit-box-shadow:0 0 0 1px #db3737, 0 0 0 3px rgba(219, 55, 55, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2);
              box-shadow:0 0 0 1px #db3737, 0 0 0 3px rgba(219, 55, 55, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
    .bp3-input.bp3-intent-danger[readonly]{
      -webkit-box-shadow:inset 0 0 0 1px #db3737;
              box-shadow:inset 0 0 0 1px #db3737; }
    .bp3-input.bp3-intent-danger:disabled, .bp3-input.bp3-intent-danger.bp3-disabled{
      -webkit-box-shadow:none;
              box-shadow:none; }
    .bp3-dark .bp3-input.bp3-intent-danger{
      -webkit-box-shadow:0 0 0 0 rgba(219, 55, 55, 0), 0 0 0 0 rgba(219, 55, 55, 0), 0 0 0 0 rgba(219, 55, 55, 0), inset 0 0 0 1px #db3737, inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
              box-shadow:0 0 0 0 rgba(219, 55, 55, 0), 0 0 0 0 rgba(219, 55, 55, 0), 0 0 0 0 rgba(219, 55, 55, 0), inset 0 0 0 1px #db3737, inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4); }
      .bp3-dark .bp3-input.bp3-intent-danger:focus{
        -webkit-box-shadow:0 0 0 1px #db3737, 0 0 0 1px #db3737, 0 0 0 3px rgba(219, 55, 55, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
                box-shadow:0 0 0 1px #db3737, 0 0 0 1px #db3737, 0 0 0 3px rgba(219, 55, 55, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4); }
      .bp3-dark .bp3-input.bp3-intent-danger[readonly]{
        -webkit-box-shadow:inset 0 0 0 1px #db3737;
                box-shadow:inset 0 0 0 1px #db3737; }
      .bp3-dark .bp3-input.bp3-intent-danger:disabled, .bp3-dark .bp3-input.bp3-intent-danger.bp3-disabled{
        -webkit-box-shadow:none;
                box-shadow:none; }
  .bp3-input::-ms-clear{
    display:none; }
textarea.bp3-input{
  max-width:100%;
  padding:10px; }
  textarea.bp3-input, textarea.bp3-input.bp3-large, textarea.bp3-input.bp3-small{
    height:auto;
    line-height:inherit; }
  textarea.bp3-input.bp3-small{
    padding:8px; }
  .bp3-dark textarea.bp3-input{
    background:rgba(16, 22, 26, 0.3);
    -webkit-box-shadow:0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
    color:#f5f8fa; }
    .bp3-dark textarea.bp3-input::-webkit-input-placeholder{
      color:rgba(167, 182, 194, 0.6); }
    .bp3-dark textarea.bp3-input::-moz-placeholder{
      color:rgba(167, 182, 194, 0.6); }
    .bp3-dark textarea.bp3-input:-ms-input-placeholder{
      color:rgba(167, 182, 194, 0.6); }
    .bp3-dark textarea.bp3-input::-ms-input-placeholder{
      color:rgba(167, 182, 194, 0.6); }
    .bp3-dark textarea.bp3-input::placeholder{
      color:rgba(167, 182, 194, 0.6); }
    .bp3-dark textarea.bp3-input:focus{
      -webkit-box-shadow:0 0 0 1px #137cbd, 0 0 0 1px #137cbd, 0 0 0 3px rgba(19, 124, 189, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
              box-shadow:0 0 0 1px #137cbd, 0 0 0 1px #137cbd, 0 0 0 3px rgba(19, 124, 189, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4); }
    .bp3-dark textarea.bp3-input[readonly]{
      -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4);
              box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.4); }
    .bp3-dark textarea.bp3-input:disabled, .bp3-dark textarea.bp3-input.bp3-disabled{
      background:rgba(57, 75, 89, 0.5);
      -webkit-box-shadow:none;
              box-shadow:none;
      color:rgba(167, 182, 194, 0.6); }
label.bp3-label{
  display:block;
  margin-bottom:15px;
  margin-top:0; }
  label.bp3-label .bp3-html-select,
  label.bp3-label .bp3-input,
  label.bp3-label .bp3-select,
  label.bp3-label .bp3-slider,
  label.bp3-label .bp3-popover-wrapper{
    display:block;
    margin-top:5px;
    text-transform:none; }
  label.bp3-label .bp3-button-group{
    margin-top:5px; }
  label.bp3-label .bp3-select select,
  label.bp3-label .bp3-html-select select{
    font-weight:400;
    vertical-align:top;
    width:100%; }
  label.bp3-label.bp3-disabled,
  label.bp3-label.bp3-disabled .bp3-text-muted{
    color:rgba(92, 112, 128, 0.6); }
  label.bp3-label.bp3-inline{
    line-height:30px; }
    label.bp3-label.bp3-inline .bp3-html-select,
    label.bp3-label.bp3-inline .bp3-input,
    label.bp3-label.bp3-inline .bp3-input-group,
    label.bp3-label.bp3-inline .bp3-select,
    label.bp3-label.bp3-inline .bp3-popover-wrapper{
      display:inline-block;
      margin:0 0 0 5px;
      vertical-align:top; }
    label.bp3-label.bp3-inline .bp3-button-group{
      margin:0 0 0 5px; }
    label.bp3-label.bp3-inline .bp3-input-group .bp3-input{
      margin-left:0; }
    label.bp3-label.bp3-inline.bp3-large{
      line-height:40px; }
  label.bp3-label:not(.bp3-inline) .bp3-popover-target{
    display:block; }
  .bp3-dark label.bp3-label{
    color:#f5f8fa; }
    .bp3-dark label.bp3-label.bp3-disabled,
    .bp3-dark label.bp3-label.bp3-disabled .bp3-text-muted{
      color:rgba(167, 182, 194, 0.6); }
.bp3-numeric-input .bp3-button-group.bp3-vertical > .bp3-button{
  -webkit-box-flex:1;
      -ms-flex:1 1 14px;
          flex:1 1 14px;
  min-height:0;
  padding:0;
  width:30px; }
  .bp3-numeric-input .bp3-button-group.bp3-vertical > .bp3-button:first-child{
    border-radius:0 3px 0 0; }
  .bp3-numeric-input .bp3-button-group.bp3-vertical > .bp3-button:last-child{
    border-radius:0 0 3px 0; }

.bp3-numeric-input .bp3-button-group.bp3-vertical:first-child > .bp3-button:first-child{
  border-radius:3px 0 0 0; }

.bp3-numeric-input .bp3-button-group.bp3-vertical:first-child > .bp3-button:last-child{
  border-radius:0 0 0 3px; }

.bp3-numeric-input.bp3-large .bp3-button-group.bp3-vertical > .bp3-button{
  width:40px; }

form{
  display:block; }
.bp3-html-select select,
.bp3-select select{
  display:-webkit-inline-box;
  display:-ms-inline-flexbox;
  display:inline-flex;
  -webkit-box-orient:horizontal;
  -webkit-box-direction:normal;
      -ms-flex-direction:row;
          flex-direction:row;
  -webkit-box-align:center;
      -ms-flex-align:center;
          align-items:center;
  border:none;
  border-radius:3px;
  cursor:pointer;
  font-size:14px;
  -webkit-box-pack:center;
      -ms-flex-pack:center;
          justify-content:center;
  padding:5px 10px;
  text-align:left;
  vertical-align:middle;
  background-color:#f5f8fa;
  background-image:-webkit-gradient(linear, left top, left bottom, from(rgba(255, 255, 255, 0.8)), to(rgba(255, 255, 255, 0)));
  background-image:linear-gradient(to bottom, rgba(255, 255, 255, 0.8), rgba(255, 255, 255, 0));
  -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 -1px 0 rgba(16, 22, 26, 0.1);
          box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 -1px 0 rgba(16, 22, 26, 0.1);
  color:#182026;
  -moz-appearance:none;
  -webkit-appearance:none;
  border-radius:3px;
  height:30px;
  padding:0 25px 0 10px;
  width:100%; }
  .bp3-html-select select > *, .bp3-select select > *{
    -webkit-box-flex:0;
        -ms-flex-positive:0;
            flex-grow:0;
    -ms-flex-negative:0;
        flex-shrink:0; }
  .bp3-html-select select > .bp3-fill, .bp3-select select > .bp3-fill{
    -webkit-box-flex:1;
        -ms-flex-positive:1;
            flex-grow:1;
    -ms-flex-negative:1;
        flex-shrink:1; }
  .bp3-html-select select::before,
  .bp3-select select::before, .bp3-html-select select > *, .bp3-select select > *{
    margin-right:7px; }
  .bp3-html-select select:empty::before,
  .bp3-select select:empty::before,
  .bp3-html-select select > :last-child,
  .bp3-select select > :last-child{
    margin-right:0; }
  .bp3-html-select select:hover,
  .bp3-select select:hover{
    background-clip:padding-box;
    background-color:#ebf1f5;
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 -1px 0 rgba(16, 22, 26, 0.1);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 -1px 0 rgba(16, 22, 26, 0.1); }
  .bp3-html-select select:active,
  .bp3-select select:active, .bp3-html-select select.bp3-active,
  .bp3-select select.bp3-active{
    background-color:#d8e1e8;
    background-image:none;
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 1px 2px rgba(16, 22, 26, 0.2);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 1px 2px rgba(16, 22, 26, 0.2); }
  .bp3-html-select select:disabled,
  .bp3-select select:disabled, .bp3-html-select select.bp3-disabled,
  .bp3-select select.bp3-disabled{
    background-color:rgba(206, 217, 224, 0.5);
    background-image:none;
    -webkit-box-shadow:none;
            box-shadow:none;
    color:rgba(92, 112, 128, 0.6);
    cursor:not-allowed;
    outline:none; }
    .bp3-html-select select:disabled.bp3-active,
    .bp3-select select:disabled.bp3-active, .bp3-html-select select:disabled.bp3-active:hover,
    .bp3-select select:disabled.bp3-active:hover, .bp3-html-select select.bp3-disabled.bp3-active,
    .bp3-select select.bp3-disabled.bp3-active, .bp3-html-select select.bp3-disabled.bp3-active:hover,
    .bp3-select select.bp3-disabled.bp3-active:hover{
      background:rgba(206, 217, 224, 0.7); }

.bp3-html-select.bp3-minimal select,
.bp3-select.bp3-minimal select{
  background:none;
  -webkit-box-shadow:none;
          box-shadow:none; }
  .bp3-html-select.bp3-minimal select:hover,
  .bp3-select.bp3-minimal select:hover{
    background:rgba(167, 182, 194, 0.3);
    -webkit-box-shadow:none;
            box-shadow:none;
    color:#182026;
    text-decoration:none; }
  .bp3-html-select.bp3-minimal select:active,
  .bp3-select.bp3-minimal select:active, .bp3-html-select.bp3-minimal select.bp3-active,
  .bp3-select.bp3-minimal select.bp3-active{
    background:rgba(115, 134, 148, 0.3);
    -webkit-box-shadow:none;
            box-shadow:none;
    color:#182026; }
  .bp3-html-select.bp3-minimal select:disabled,
  .bp3-select.bp3-minimal select:disabled, .bp3-html-select.bp3-minimal select:disabled:hover,
  .bp3-select.bp3-minimal select:disabled:hover, .bp3-html-select.bp3-minimal select.bp3-disabled,
  .bp3-select.bp3-minimal select.bp3-disabled, .bp3-html-select.bp3-minimal select.bp3-disabled:hover,
  .bp3-select.bp3-minimal select.bp3-disabled:hover{
    background:none;
    color:rgba(92, 112, 128, 0.6);
    cursor:not-allowed; }
    .bp3-html-select.bp3-minimal select:disabled.bp3-active,
    .bp3-select.bp3-minimal select:disabled.bp3-active, .bp3-html-select.bp3-minimal select:disabled:hover.bp3-active,
    .bp3-select.bp3-minimal select:disabled:hover.bp3-active, .bp3-html-select.bp3-minimal select.bp3-disabled.bp3-active,
    .bp3-select.bp3-minimal select.bp3-disabled.bp3-active, .bp3-html-select.bp3-minimal select.bp3-disabled:hover.bp3-active,
    .bp3-select.bp3-minimal select.bp3-disabled:hover.bp3-active{
      background:rgba(115, 134, 148, 0.3); }
  .bp3-dark .bp3-html-select.bp3-minimal select, .bp3-html-select.bp3-minimal .bp3-dark select,
  .bp3-dark .bp3-select.bp3-minimal select, .bp3-select.bp3-minimal .bp3-dark select{
    background:none;
    -webkit-box-shadow:none;
            box-shadow:none;
    color:inherit; }
    .bp3-dark .bp3-html-select.bp3-minimal select:hover, .bp3-html-select.bp3-minimal .bp3-dark select:hover,
    .bp3-dark .bp3-select.bp3-minimal select:hover, .bp3-select.bp3-minimal .bp3-dark select:hover, .bp3-dark .bp3-html-select.bp3-minimal select:active, .bp3-html-select.bp3-minimal .bp3-dark select:active,
    .bp3-dark .bp3-select.bp3-minimal select:active, .bp3-select.bp3-minimal .bp3-dark select:active, .bp3-dark .bp3-html-select.bp3-minimal select.bp3-active, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-active,
    .bp3-dark .bp3-select.bp3-minimal select.bp3-active, .bp3-select.bp3-minimal .bp3-dark select.bp3-active{
      background:none;
      -webkit-box-shadow:none;
              box-shadow:none; }
    .bp3-dark .bp3-html-select.bp3-minimal select:hover, .bp3-html-select.bp3-minimal .bp3-dark select:hover,
    .bp3-dark .bp3-select.bp3-minimal select:hover, .bp3-select.bp3-minimal .bp3-dark select:hover{
      background:rgba(138, 155, 168, 0.15); }
    .bp3-dark .bp3-html-select.bp3-minimal select:active, .bp3-html-select.bp3-minimal .bp3-dark select:active,
    .bp3-dark .bp3-select.bp3-minimal select:active, .bp3-select.bp3-minimal .bp3-dark select:active, .bp3-dark .bp3-html-select.bp3-minimal select.bp3-active, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-active,
    .bp3-dark .bp3-select.bp3-minimal select.bp3-active, .bp3-select.bp3-minimal .bp3-dark select.bp3-active{
      background:rgba(138, 155, 168, 0.3);
      color:#f5f8fa; }
    .bp3-dark .bp3-html-select.bp3-minimal select:disabled, .bp3-html-select.bp3-minimal .bp3-dark select:disabled,
    .bp3-dark .bp3-select.bp3-minimal select:disabled, .bp3-select.bp3-minimal .bp3-dark select:disabled, .bp3-dark .bp3-html-select.bp3-minimal select:disabled:hover, .bp3-html-select.bp3-minimal .bp3-dark select:disabled:hover,
    .bp3-dark .bp3-select.bp3-minimal select:disabled:hover, .bp3-select.bp3-minimal .bp3-dark select:disabled:hover, .bp3-dark .bp3-html-select.bp3-minimal select.bp3-disabled, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-disabled,
    .bp3-dark .bp3-select.bp3-minimal select.bp3-disabled, .bp3-select.bp3-minimal .bp3-dark select.bp3-disabled, .bp3-dark .bp3-html-select.bp3-minimal select.bp3-disabled:hover, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-disabled:hover,
    .bp3-dark .bp3-select.bp3-minimal select.bp3-disabled:hover, .bp3-select.bp3-minimal .bp3-dark select.bp3-disabled:hover{
      background:none;
      color:rgba(167, 182, 194, 0.6);
      cursor:not-allowed; }
      .bp3-dark .bp3-html-select.bp3-minimal select:disabled.bp3-active, .bp3-html-select.bp3-minimal .bp3-dark select:disabled.bp3-active,
      .bp3-dark .bp3-select.bp3-minimal select:disabled.bp3-active, .bp3-select.bp3-minimal .bp3-dark select:disabled.bp3-active, .bp3-dark .bp3-html-select.bp3-minimal select:disabled:hover.bp3-active, .bp3-html-select.bp3-minimal .bp3-dark select:disabled:hover.bp3-active,
      .bp3-dark .bp3-select.bp3-minimal select:disabled:hover.bp3-active, .bp3-select.bp3-minimal .bp3-dark select:disabled:hover.bp3-active, .bp3-dark .bp3-html-select.bp3-minimal select.bp3-disabled.bp3-active, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-disabled.bp3-active,
      .bp3-dark .bp3-select.bp3-minimal select.bp3-disabled.bp3-active, .bp3-select.bp3-minimal .bp3-dark select.bp3-disabled.bp3-active, .bp3-dark .bp3-html-select.bp3-minimal select.bp3-disabled:hover.bp3-active, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-disabled:hover.bp3-active,
      .bp3-dark .bp3-select.bp3-minimal select.bp3-disabled:hover.bp3-active, .bp3-select.bp3-minimal .bp3-dark select.bp3-disabled:hover.bp3-active{
        background:rgba(138, 155, 168, 0.3); }
  .bp3-html-select.bp3-minimal select.bp3-intent-primary,
  .bp3-select.bp3-minimal select.bp3-intent-primary{
    color:#106ba3; }
    .bp3-html-select.bp3-minimal select.bp3-intent-primary:hover,
    .bp3-select.bp3-minimal select.bp3-intent-primary:hover, .bp3-html-select.bp3-minimal select.bp3-intent-primary:active,
    .bp3-select.bp3-minimal select.bp3-intent-primary:active, .bp3-html-select.bp3-minimal select.bp3-intent-primary.bp3-active,
    .bp3-select.bp3-minimal select.bp3-intent-primary.bp3-active{
      background:none;
      -webkit-box-shadow:none;
              box-shadow:none;
      color:#106ba3; }
    .bp3-html-select.bp3-minimal select.bp3-intent-primary:hover,
    .bp3-select.bp3-minimal select.bp3-intent-primary:hover{
      background:rgba(19, 124, 189, 0.15);
      color:#106ba3; }
    .bp3-html-select.bp3-minimal select.bp3-intent-primary:active,
    .bp3-select.bp3-minimal select.bp3-intent-primary:active, .bp3-html-select.bp3-minimal select.bp3-intent-primary.bp3-active,
    .bp3-select.bp3-minimal select.bp3-intent-primary.bp3-active{
      background:rgba(19, 124, 189, 0.3);
      color:#106ba3; }
    .bp3-html-select.bp3-minimal select.bp3-intent-primary:disabled,
    .bp3-select.bp3-minimal select.bp3-intent-primary:disabled, .bp3-html-select.bp3-minimal select.bp3-intent-primary.bp3-disabled,
    .bp3-select.bp3-minimal select.bp3-intent-primary.bp3-disabled{
      background:none;
      color:rgba(16, 107, 163, 0.5); }
      .bp3-html-select.bp3-minimal select.bp3-intent-primary:disabled.bp3-active,
      .bp3-select.bp3-minimal select.bp3-intent-primary:disabled.bp3-active, .bp3-html-select.bp3-minimal select.bp3-intent-primary.bp3-disabled.bp3-active,
      .bp3-select.bp3-minimal select.bp3-intent-primary.bp3-disabled.bp3-active{
        background:rgba(19, 124, 189, 0.3); }
    .bp3-html-select.bp3-minimal select.bp3-intent-primary .bp3-button-spinner .bp3-spinner-head, .bp3-select.bp3-minimal select.bp3-intent-primary .bp3-button-spinner .bp3-spinner-head{
      stroke:#106ba3; }
    .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-primary, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-primary,
    .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-primary, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-primary{
      color:#48aff0; }
      .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-primary:hover, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-primary:hover,
      .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-primary:hover, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-primary:hover{
        background:rgba(19, 124, 189, 0.2);
        color:#48aff0; }
      .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-primary:active, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-primary:active,
      .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-primary:active, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-primary:active, .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-primary.bp3-active, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-primary.bp3-active,
      .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-primary.bp3-active, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-primary.bp3-active{
        background:rgba(19, 124, 189, 0.3);
        color:#48aff0; }
      .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-primary:disabled, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-primary:disabled,
      .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-primary:disabled, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-primary:disabled, .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-primary.bp3-disabled, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-primary.bp3-disabled,
      .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-primary.bp3-disabled, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-primary.bp3-disabled{
        background:none;
        color:rgba(72, 175, 240, 0.5); }
        .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-primary:disabled.bp3-active, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-primary:disabled.bp3-active,
        .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-primary:disabled.bp3-active, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-primary:disabled.bp3-active, .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-primary.bp3-disabled.bp3-active, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-primary.bp3-disabled.bp3-active,
        .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-primary.bp3-disabled.bp3-active, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-primary.bp3-disabled.bp3-active{
          background:rgba(19, 124, 189, 0.3); }
  .bp3-html-select.bp3-minimal select.bp3-intent-success,
  .bp3-select.bp3-minimal select.bp3-intent-success{
    color:#0d8050; }
    .bp3-html-select.bp3-minimal select.bp3-intent-success:hover,
    .bp3-select.bp3-minimal select.bp3-intent-success:hover, .bp3-html-select.bp3-minimal select.bp3-intent-success:active,
    .bp3-select.bp3-minimal select.bp3-intent-success:active, .bp3-html-select.bp3-minimal select.bp3-intent-success.bp3-active,
    .bp3-select.bp3-minimal select.bp3-intent-success.bp3-active{
      background:none;
      -webkit-box-shadow:none;
              box-shadow:none;
      color:#0d8050; }
    .bp3-html-select.bp3-minimal select.bp3-intent-success:hover,
    .bp3-select.bp3-minimal select.bp3-intent-success:hover{
      background:rgba(15, 153, 96, 0.15);
      color:#0d8050; }
    .bp3-html-select.bp3-minimal select.bp3-intent-success:active,
    .bp3-select.bp3-minimal select.bp3-intent-success:active, .bp3-html-select.bp3-minimal select.bp3-intent-success.bp3-active,
    .bp3-select.bp3-minimal select.bp3-intent-success.bp3-active{
      background:rgba(15, 153, 96, 0.3);
      color:#0d8050; }
    .bp3-html-select.bp3-minimal select.bp3-intent-success:disabled,
    .bp3-select.bp3-minimal select.bp3-intent-success:disabled, .bp3-html-select.bp3-minimal select.bp3-intent-success.bp3-disabled,
    .bp3-select.bp3-minimal select.bp3-intent-success.bp3-disabled{
      background:none;
      color:rgba(13, 128, 80, 0.5); }
      .bp3-html-select.bp3-minimal select.bp3-intent-success:disabled.bp3-active,
      .bp3-select.bp3-minimal select.bp3-intent-success:disabled.bp3-active, .bp3-html-select.bp3-minimal select.bp3-intent-success.bp3-disabled.bp3-active,
      .bp3-select.bp3-minimal select.bp3-intent-success.bp3-disabled.bp3-active{
        background:rgba(15, 153, 96, 0.3); }
    .bp3-html-select.bp3-minimal select.bp3-intent-success .bp3-button-spinner .bp3-spinner-head, .bp3-select.bp3-minimal select.bp3-intent-success .bp3-button-spinner .bp3-spinner-head{
      stroke:#0d8050; }
    .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-success, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-success,
    .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-success, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-success{
      color:#3dcc91; }
      .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-success:hover, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-success:hover,
      .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-success:hover, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-success:hover{
        background:rgba(15, 153, 96, 0.2);
        color:#3dcc91; }
      .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-success:active, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-success:active,
      .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-success:active, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-success:active, .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-success.bp3-active, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-success.bp3-active,
      .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-success.bp3-active, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-success.bp3-active{
        background:rgba(15, 153, 96, 0.3);
        color:#3dcc91; }
      .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-success:disabled, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-success:disabled,
      .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-success:disabled, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-success:disabled, .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-success.bp3-disabled, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-success.bp3-disabled,
      .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-success.bp3-disabled, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-success.bp3-disabled{
        background:none;
        color:rgba(61, 204, 145, 0.5); }
        .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-success:disabled.bp3-active, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-success:disabled.bp3-active,
        .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-success:disabled.bp3-active, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-success:disabled.bp3-active, .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-success.bp3-disabled.bp3-active, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-success.bp3-disabled.bp3-active,
        .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-success.bp3-disabled.bp3-active, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-success.bp3-disabled.bp3-active{
          background:rgba(15, 153, 96, 0.3); }
  .bp3-html-select.bp3-minimal select.bp3-intent-warning,
  .bp3-select.bp3-minimal select.bp3-intent-warning{
    color:#bf7326; }
    .bp3-html-select.bp3-minimal select.bp3-intent-warning:hover,
    .bp3-select.bp3-minimal select.bp3-intent-warning:hover, .bp3-html-select.bp3-minimal select.bp3-intent-warning:active,
    .bp3-select.bp3-minimal select.bp3-intent-warning:active, .bp3-html-select.bp3-minimal select.bp3-intent-warning.bp3-active,
    .bp3-select.bp3-minimal select.bp3-intent-warning.bp3-active{
      background:none;
      -webkit-box-shadow:none;
              box-shadow:none;
      color:#bf7326; }
    .bp3-html-select.bp3-minimal select.bp3-intent-warning:hover,
    .bp3-select.bp3-minimal select.bp3-intent-warning:hover{
      background:rgba(217, 130, 43, 0.15);
      color:#bf7326; }
    .bp3-html-select.bp3-minimal select.bp3-intent-warning:active,
    .bp3-select.bp3-minimal select.bp3-intent-warning:active, .bp3-html-select.bp3-minimal select.bp3-intent-warning.bp3-active,
    .bp3-select.bp3-minimal select.bp3-intent-warning.bp3-active{
      background:rgba(217, 130, 43, 0.3);
      color:#bf7326; }
    .bp3-html-select.bp3-minimal select.bp3-intent-warning:disabled,
    .bp3-select.bp3-minimal select.bp3-intent-warning:disabled, .bp3-html-select.bp3-minimal select.bp3-intent-warning.bp3-disabled,
    .bp3-select.bp3-minimal select.bp3-intent-warning.bp3-disabled{
      background:none;
      color:rgba(191, 115, 38, 0.5); }
      .bp3-html-select.bp3-minimal select.bp3-intent-warning:disabled.bp3-active,
      .bp3-select.bp3-minimal select.bp3-intent-warning:disabled.bp3-active, .bp3-html-select.bp3-minimal select.bp3-intent-warning.bp3-disabled.bp3-active,
      .bp3-select.bp3-minimal select.bp3-intent-warning.bp3-disabled.bp3-active{
        background:rgba(217, 130, 43, 0.3); }
    .bp3-html-select.bp3-minimal select.bp3-intent-warning .bp3-button-spinner .bp3-spinner-head, .bp3-select.bp3-minimal select.bp3-intent-warning .bp3-button-spinner .bp3-spinner-head{
      stroke:#bf7326; }
    .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-warning, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-warning,
    .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-warning, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-warning{
      color:#ffb366; }
      .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-warning:hover, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-warning:hover,
      .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-warning:hover, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-warning:hover{
        background:rgba(217, 130, 43, 0.2);
        color:#ffb366; }
      .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-warning:active, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-warning:active,
      .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-warning:active, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-warning:active, .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-warning.bp3-active, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-warning.bp3-active,
      .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-warning.bp3-active, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-warning.bp3-active{
        background:rgba(217, 130, 43, 0.3);
        color:#ffb366; }
      .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-warning:disabled, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-warning:disabled,
      .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-warning:disabled, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-warning:disabled, .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-warning.bp3-disabled, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-warning.bp3-disabled,
      .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-warning.bp3-disabled, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-warning.bp3-disabled{
        background:none;
        color:rgba(255, 179, 102, 0.5); }
        .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-warning:disabled.bp3-active, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-warning:disabled.bp3-active,
        .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-warning:disabled.bp3-active, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-warning:disabled.bp3-active, .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-warning.bp3-disabled.bp3-active, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-warning.bp3-disabled.bp3-active,
        .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-warning.bp3-disabled.bp3-active, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-warning.bp3-disabled.bp3-active{
          background:rgba(217, 130, 43, 0.3); }
  .bp3-html-select.bp3-minimal select.bp3-intent-danger,
  .bp3-select.bp3-minimal select.bp3-intent-danger{
    color:#c23030; }
    .bp3-html-select.bp3-minimal select.bp3-intent-danger:hover,
    .bp3-select.bp3-minimal select.bp3-intent-danger:hover, .bp3-html-select.bp3-minimal select.bp3-intent-danger:active,
    .bp3-select.bp3-minimal select.bp3-intent-danger:active, .bp3-html-select.bp3-minimal select.bp3-intent-danger.bp3-active,
    .bp3-select.bp3-minimal select.bp3-intent-danger.bp3-active{
      background:none;
      -webkit-box-shadow:none;
              box-shadow:none;
      color:#c23030; }
    .bp3-html-select.bp3-minimal select.bp3-intent-danger:hover,
    .bp3-select.bp3-minimal select.bp3-intent-danger:hover{
      background:rgba(219, 55, 55, 0.15);
      color:#c23030; }
    .bp3-html-select.bp3-minimal select.bp3-intent-danger:active,
    .bp3-select.bp3-minimal select.bp3-intent-danger:active, .bp3-html-select.bp3-minimal select.bp3-intent-danger.bp3-active,
    .bp3-select.bp3-minimal select.bp3-intent-danger.bp3-active{
      background:rgba(219, 55, 55, 0.3);
      color:#c23030; }
    .bp3-html-select.bp3-minimal select.bp3-intent-danger:disabled,
    .bp3-select.bp3-minimal select.bp3-intent-danger:disabled, .bp3-html-select.bp3-minimal select.bp3-intent-danger.bp3-disabled,
    .bp3-select.bp3-minimal select.bp3-intent-danger.bp3-disabled{
      background:none;
      color:rgba(194, 48, 48, 0.5); }
      .bp3-html-select.bp3-minimal select.bp3-intent-danger:disabled.bp3-active,
      .bp3-select.bp3-minimal select.bp3-intent-danger:disabled.bp3-active, .bp3-html-select.bp3-minimal select.bp3-intent-danger.bp3-disabled.bp3-active,
      .bp3-select.bp3-minimal select.bp3-intent-danger.bp3-disabled.bp3-active{
        background:rgba(219, 55, 55, 0.3); }
    .bp3-html-select.bp3-minimal select.bp3-intent-danger .bp3-button-spinner .bp3-spinner-head, .bp3-select.bp3-minimal select.bp3-intent-danger .bp3-button-spinner .bp3-spinner-head{
      stroke:#c23030; }
    .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-danger, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-danger,
    .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-danger, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-danger{
      color:#ff7373; }
      .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-danger:hover, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-danger:hover,
      .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-danger:hover, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-danger:hover{
        background:rgba(219, 55, 55, 0.2);
        color:#ff7373; }
      .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-danger:active, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-danger:active,
      .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-danger:active, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-danger:active, .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-danger.bp3-active, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-danger.bp3-active,
      .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-danger.bp3-active, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-danger.bp3-active{
        background:rgba(219, 55, 55, 0.3);
        color:#ff7373; }
      .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-danger:disabled, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-danger:disabled,
      .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-danger:disabled, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-danger:disabled, .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-danger.bp3-disabled, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-danger.bp3-disabled,
      .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-danger.bp3-disabled, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-danger.bp3-disabled{
        background:none;
        color:rgba(255, 115, 115, 0.5); }
        .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-danger:disabled.bp3-active, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-danger:disabled.bp3-active,
        .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-danger:disabled.bp3-active, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-danger:disabled.bp3-active, .bp3-dark .bp3-html-select.bp3-minimal select.bp3-intent-danger.bp3-disabled.bp3-active, .bp3-html-select.bp3-minimal .bp3-dark select.bp3-intent-danger.bp3-disabled.bp3-active,
        .bp3-dark .bp3-select.bp3-minimal select.bp3-intent-danger.bp3-disabled.bp3-active, .bp3-select.bp3-minimal .bp3-dark select.bp3-intent-danger.bp3-disabled.bp3-active{
          background:rgba(219, 55, 55, 0.3); }

.bp3-html-select.bp3-large select,
.bp3-select.bp3-large select{
  font-size:16px;
  height:40px;
  padding-right:35px; }

.bp3-dark .bp3-html-select select, .bp3-dark .bp3-select select{
  background-color:#394b59;
  background-image:-webkit-gradient(linear, left top, left bottom, from(rgba(255, 255, 255, 0.05)), to(rgba(255, 255, 255, 0)));
  background-image:linear-gradient(to bottom, rgba(255, 255, 255, 0.05), rgba(255, 255, 255, 0));
  -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4);
          box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4);
  color:#f5f8fa; }
  .bp3-dark .bp3-html-select select:hover, .bp3-dark .bp3-select select:hover, .bp3-dark .bp3-html-select select:active, .bp3-dark .bp3-select select:active, .bp3-dark .bp3-html-select select.bp3-active, .bp3-dark .bp3-select select.bp3-active{
    color:#f5f8fa; }
  .bp3-dark .bp3-html-select select:hover, .bp3-dark .bp3-select select:hover{
    background-color:#30404d;
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4); }
  .bp3-dark .bp3-html-select select:active, .bp3-dark .bp3-select select:active, .bp3-dark .bp3-html-select select.bp3-active, .bp3-dark .bp3-select select.bp3-active{
    background-color:#202b33;
    background-image:none;
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.6), inset 0 1px 2px rgba(16, 22, 26, 0.2);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.6), inset 0 1px 2px rgba(16, 22, 26, 0.2); }
  .bp3-dark .bp3-html-select select:disabled, .bp3-dark .bp3-select select:disabled, .bp3-dark .bp3-html-select select.bp3-disabled, .bp3-dark .bp3-select select.bp3-disabled{
    background-color:rgba(57, 75, 89, 0.5);
    background-image:none;
    -webkit-box-shadow:none;
            box-shadow:none;
    color:rgba(167, 182, 194, 0.6); }
    .bp3-dark .bp3-html-select select:disabled.bp3-active, .bp3-dark .bp3-select select:disabled.bp3-active, .bp3-dark .bp3-html-select select.bp3-disabled.bp3-active, .bp3-dark .bp3-select select.bp3-disabled.bp3-active{
      background:rgba(57, 75, 89, 0.7); }
  .bp3-dark .bp3-html-select select .bp3-button-spinner .bp3-spinner-head, .bp3-dark .bp3-select select .bp3-button-spinner .bp3-spinner-head{
    background:rgba(16, 22, 26, 0.5);
    stroke:#8a9ba8; }

.bp3-html-select select:disabled,
.bp3-select select:disabled{
  background-color:rgba(206, 217, 224, 0.5);
  -webkit-box-shadow:none;
          box-shadow:none;
  color:rgba(92, 112, 128, 0.6);
  cursor:not-allowed; }

.bp3-html-select .bp3-icon,
.bp3-select .bp3-icon, .bp3-select::after{
  color:#5c7080;
  pointer-events:none;
  position:absolute;
  right:7px;
  top:7px; }
  .bp3-html-select .bp3-disabled.bp3-icon,
  .bp3-select .bp3-disabled.bp3-icon, .bp3-disabled.bp3-select::after{
    color:rgba(92, 112, 128, 0.6); }
.bp3-html-select,
.bp3-select{
  display:inline-block;
  letter-spacing:normal;
  position:relative;
  vertical-align:middle; }
  .bp3-html-select select::-ms-expand,
  .bp3-select select::-ms-expand{
    display:none; }
  .bp3-html-select .bp3-icon,
  .bp3-select .bp3-icon{
    color:#5c7080; }
    .bp3-html-select .bp3-icon:hover,
    .bp3-select .bp3-icon:hover{
      color:#182026; }
    .bp3-dark .bp3-html-select .bp3-icon, .bp3-dark
    .bp3-select .bp3-icon{
      color:#a7b6c2; }
      .bp3-dark .bp3-html-select .bp3-icon:hover, .bp3-dark
      .bp3-select .bp3-icon:hover{
        color:#f5f8fa; }
  .bp3-html-select.bp3-large::after,
  .bp3-html-select.bp3-large .bp3-icon,
  .bp3-select.bp3-large::after,
  .bp3-select.bp3-large .bp3-icon{
    right:12px;
    top:12px; }
  .bp3-html-select.bp3-fill,
  .bp3-html-select.bp3-fill select,
  .bp3-select.bp3-fill,
  .bp3-select.bp3-fill select{
    width:100%; }
  .bp3-dark .bp3-html-select option, .bp3-dark
  .bp3-select option{
    background-color:#30404d;
    color:#f5f8fa; }
  .bp3-dark .bp3-html-select option:disabled, .bp3-dark
  .bp3-select option:disabled{
    color:rgba(167, 182, 194, 0.6); }
  .bp3-dark .bp3-html-select::after, .bp3-dark
  .bp3-select::after{
    color:#a7b6c2; }

.bp3-select::after{
  font-family:"Icons16", sans-serif;
  font-size:16px;
  font-style:normal;
  font-weight:400;
  line-height:1;
  -moz-osx-font-smoothing:grayscale;
  -webkit-font-smoothing:antialiased;
  content:""; }
.bp3-running-text table, table.bp3-html-table{
  border-spacing:0;
  font-size:14px; }
  .bp3-running-text table th, table.bp3-html-table th,
  .bp3-running-text table td,
  table.bp3-html-table td{
    padding:11px;
    text-align:left;
    vertical-align:top; }
  .bp3-running-text table th, table.bp3-html-table th{
    color:#182026;
    font-weight:600; }
  
  .bp3-running-text table td,
  table.bp3-html-table td{
    color:#182026; }
  .bp3-running-text table tbody tr:first-child th, table.bp3-html-table tbody tr:first-child th,
  .bp3-running-text table tbody tr:first-child td,
  table.bp3-html-table tbody tr:first-child td{
    -webkit-box-shadow:inset 0 1px 0 0 rgba(16, 22, 26, 0.15);
            box-shadow:inset 0 1px 0 0 rgba(16, 22, 26, 0.15); }
  .bp3-dark .bp3-running-text table th, .bp3-running-text .bp3-dark table th, .bp3-dark table.bp3-html-table th{
    color:#f5f8fa; }
  .bp3-dark .bp3-running-text table td, .bp3-running-text .bp3-dark table td, .bp3-dark table.bp3-html-table td{
    color:#f5f8fa; }
  .bp3-dark .bp3-running-text table tbody tr:first-child th, .bp3-running-text .bp3-dark table tbody tr:first-child th, .bp3-dark table.bp3-html-table tbody tr:first-child th,
  .bp3-dark .bp3-running-text table tbody tr:first-child td,
  .bp3-running-text .bp3-dark table tbody tr:first-child td,
  .bp3-dark table.bp3-html-table tbody tr:first-child td{
    -webkit-box-shadow:inset 0 1px 0 0 rgba(255, 255, 255, 0.15);
            box-shadow:inset 0 1px 0 0 rgba(255, 255, 255, 0.15); }

table.bp3-html-table.bp3-html-table-condensed th,
table.bp3-html-table.bp3-html-table-condensed td, table.bp3-html-table.bp3-small th,
table.bp3-html-table.bp3-small td{
  padding-bottom:6px;
  padding-top:6px; }

table.bp3-html-table.bp3-html-table-striped tbody tr:nth-child(odd) td{
  background:rgba(191, 204, 214, 0.15); }

table.bp3-html-table.bp3-html-table-bordered th:not(:first-child){
  -webkit-box-shadow:inset 1px 0 0 0 rgba(16, 22, 26, 0.15);
          box-shadow:inset 1px 0 0 0 rgba(16, 22, 26, 0.15); }

table.bp3-html-table.bp3-html-table-bordered tbody tr td{
  -webkit-box-shadow:inset 0 1px 0 0 rgba(16, 22, 26, 0.15);
          box-shadow:inset 0 1px 0 0 rgba(16, 22, 26, 0.15); }
  table.bp3-html-table.bp3-html-table-bordered tbody tr td:not(:first-child){
    -webkit-box-shadow:inset 1px 1px 0 0 rgba(16, 22, 26, 0.15);
            box-shadow:inset 1px 1px 0 0 rgba(16, 22, 26, 0.15); }

table.bp3-html-table.bp3-html-table-bordered.bp3-html-table-striped tbody tr:not(:first-child) td{
  -webkit-box-shadow:none;
          box-shadow:none; }
  table.bp3-html-table.bp3-html-table-bordered.bp3-html-table-striped tbody tr:not(:first-child) td:not(:first-child){
    -webkit-box-shadow:inset 1px 0 0 0 rgba(16, 22, 26, 0.15);
            box-shadow:inset 1px 0 0 0 rgba(16, 22, 26, 0.15); }

table.bp3-html-table.bp3-interactive tbody tr:hover td{
  background-color:rgba(191, 204, 214, 0.3);
  cursor:pointer; }

table.bp3-html-table.bp3-interactive tbody tr:active td{
  background-color:rgba(191, 204, 214, 0.4); }

.bp3-dark table.bp3-html-table{ }
  .bp3-dark table.bp3-html-table.bp3-html-table-striped tbody tr:nth-child(odd) td{
    background:rgba(92, 112, 128, 0.15); }
  .bp3-dark table.bp3-html-table.bp3-html-table-bordered th:not(:first-child){
    -webkit-box-shadow:inset 1px 0 0 0 rgba(255, 255, 255, 0.15);
            box-shadow:inset 1px 0 0 0 rgba(255, 255, 255, 0.15); }
  .bp3-dark table.bp3-html-table.bp3-html-table-bordered tbody tr td{
    -webkit-box-shadow:inset 0 1px 0 0 rgba(255, 255, 255, 0.15);
            box-shadow:inset 0 1px 0 0 rgba(255, 255, 255, 0.15); }
    .bp3-dark table.bp3-html-table.bp3-html-table-bordered tbody tr td:not(:first-child){
      -webkit-box-shadow:inset 1px 1px 0 0 rgba(255, 255, 255, 0.15);
              box-shadow:inset 1px 1px 0 0 rgba(255, 255, 255, 0.15); }
  .bp3-dark table.bp3-html-table.bp3-html-table-bordered.bp3-html-table-striped tbody tr:not(:first-child) td{
    -webkit-box-shadow:inset 1px 0 0 0 rgba(255, 255, 255, 0.15);
            box-shadow:inset 1px 0 0 0 rgba(255, 255, 255, 0.15); }
    .bp3-dark table.bp3-html-table.bp3-html-table-bordered.bp3-html-table-striped tbody tr:not(:first-child) td:first-child{
      -webkit-box-shadow:none;
              box-shadow:none; }
  .bp3-dark table.bp3-html-table.bp3-interactive tbody tr:hover td{
    background-color:rgba(92, 112, 128, 0.3);
    cursor:pointer; }
  .bp3-dark table.bp3-html-table.bp3-interactive tbody tr:active td{
    background-color:rgba(92, 112, 128, 0.4); }

.bp3-key-combo{
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex;
  -webkit-box-orient:horizontal;
  -webkit-box-direction:normal;
      -ms-flex-direction:row;
          flex-direction:row;
  -webkit-box-align:center;
      -ms-flex-align:center;
          align-items:center; }
  .bp3-key-combo > *{
    -webkit-box-flex:0;
        -ms-flex-positive:0;
            flex-grow:0;
    -ms-flex-negative:0;
        flex-shrink:0; }
  .bp3-key-combo > .bp3-fill{
    -webkit-box-flex:1;
        -ms-flex-positive:1;
            flex-grow:1;
    -ms-flex-negative:1;
        flex-shrink:1; }
  .bp3-key-combo::before,
  .bp3-key-combo > *{
    margin-right:5px; }
  .bp3-key-combo:empty::before,
  .bp3-key-combo > :last-child{
    margin-right:0; }

.bp3-hotkey-dialog{
  padding-bottom:0;
  top:40px; }
  .bp3-hotkey-dialog .bp3-dialog-body{
    margin:0;
    padding:0; }
  .bp3-hotkey-dialog .bp3-hotkey-label{
    -webkit-box-flex:1;
        -ms-flex-positive:1;
            flex-grow:1; }

.bp3-hotkey-column{
  margin:auto;
  max-height:80vh;
  overflow-y:auto;
  padding:30px; }
  .bp3-hotkey-column .bp3-heading{
    margin-bottom:20px; }
    .bp3-hotkey-column .bp3-heading:not(:first-child){
      margin-top:40px; }

.bp3-hotkey{
  -webkit-box-align:center;
      -ms-flex-align:center;
          align-items:center;
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex;
  -webkit-box-pack:justify;
      -ms-flex-pack:justify;
          justify-content:space-between;
  margin-left:0;
  margin-right:0; }
  .bp3-hotkey:not(:last-child){
    margin-bottom:10px; }
.bp3-icon{
  display:inline-block;
  -webkit-box-flex:0;
      -ms-flex:0 0 auto;
          flex:0 0 auto;
  vertical-align:text-bottom; }
  .bp3-icon:not(:empty)::before{
    content:"" !important;
    content:unset !important; }
  .bp3-icon > svg{
    display:block; }
    .bp3-icon > svg:not([fill]){
      fill:currentColor; }

.bp3-icon.bp3-intent-primary, .bp3-icon-standard.bp3-intent-primary, .bp3-icon-large.bp3-intent-primary{
  color:#106ba3; }
  .bp3-dark .bp3-icon.bp3-intent-primary, .bp3-dark .bp3-icon-standard.bp3-intent-primary, .bp3-dark .bp3-icon-large.bp3-intent-primary{
    color:#48aff0; }

.bp3-icon.bp3-intent-success, .bp3-icon-standard.bp3-intent-success, .bp3-icon-large.bp3-intent-success{
  color:#0d8050; }
  .bp3-dark .bp3-icon.bp3-intent-success, .bp3-dark .bp3-icon-standard.bp3-intent-success, .bp3-dark .bp3-icon-large.bp3-intent-success{
    color:#3dcc91; }

.bp3-icon.bp3-intent-warning, .bp3-icon-standard.bp3-intent-warning, .bp3-icon-large.bp3-intent-warning{
  color:#bf7326; }
  .bp3-dark .bp3-icon.bp3-intent-warning, .bp3-dark .bp3-icon-standard.bp3-intent-warning, .bp3-dark .bp3-icon-large.bp3-intent-warning{
    color:#ffb366; }

.bp3-icon.bp3-intent-danger, .bp3-icon-standard.bp3-intent-danger, .bp3-icon-large.bp3-intent-danger{
  color:#c23030; }
  .bp3-dark .bp3-icon.bp3-intent-danger, .bp3-dark .bp3-icon-standard.bp3-intent-danger, .bp3-dark .bp3-icon-large.bp3-intent-danger{
    color:#ff7373; }

span.bp3-icon-standard{
  font-family:"Icons16", sans-serif;
  font-size:16px;
  font-style:normal;
  font-weight:400;
  line-height:1;
  -moz-osx-font-smoothing:grayscale;
  -webkit-font-smoothing:antialiased;
  display:inline-block; }

span.bp3-icon-large{
  font-family:"Icons20", sans-serif;
  font-size:20px;
  font-style:normal;
  font-weight:400;
  line-height:1;
  -moz-osx-font-smoothing:grayscale;
  -webkit-font-smoothing:antialiased;
  display:inline-block; }

span.bp3-icon:empty{
  font-family:"Icons20";
  font-size:inherit;
  font-style:normal;
  font-weight:400;
  line-height:1; }
  span.bp3-icon:empty::before{
    -moz-osx-font-smoothing:grayscale;
    -webkit-font-smoothing:antialiased; }

.bp3-icon-add::before{
  content:""; }

.bp3-icon-add-column-left::before{
  content:""; }

.bp3-icon-add-column-right::before{
  content:""; }

.bp3-icon-add-row-bottom::before{
  content:""; }

.bp3-icon-add-row-top::before{
  content:""; }

.bp3-icon-add-to-artifact::before{
  content:""; }

.bp3-icon-add-to-folder::before{
  content:""; }

.bp3-icon-airplane::before{
  content:""; }

.bp3-icon-align-center::before{
  content:""; }

.bp3-icon-align-justify::before{
  content:""; }

.bp3-icon-align-left::before{
  content:""; }

.bp3-icon-align-right::before{
  content:""; }

.bp3-icon-alignment-bottom::before{
  content:""; }

.bp3-icon-alignment-horizontal-center::before{
  content:""; }

.bp3-icon-alignment-left::before{
  content:""; }

.bp3-icon-alignment-right::before{
  content:""; }

.bp3-icon-alignment-top::before{
  content:""; }

.bp3-icon-alignment-vertical-center::before{
  content:""; }

.bp3-icon-annotation::before{
  content:""; }

.bp3-icon-application::before{
  content:""; }

.bp3-icon-applications::before{
  content:""; }

.bp3-icon-archive::before{
  content:""; }

.bp3-icon-arrow-bottom-left::before{
  content:"↙"; }

.bp3-icon-arrow-bottom-right::before{
  content:"↘"; }

.bp3-icon-arrow-down::before{
  content:"↓"; }

.bp3-icon-arrow-left::before{
  content:"←"; }

.bp3-icon-arrow-right::before{
  content:"→"; }

.bp3-icon-arrow-top-left::before{
  content:"↖"; }

.bp3-icon-arrow-top-right::before{
  content:"↗"; }

.bp3-icon-arrow-up::before{
  content:"↑"; }

.bp3-icon-arrows-horizontal::before{
  content:"↔"; }

.bp3-icon-arrows-vertical::before{
  content:"↕"; }

.bp3-icon-asterisk::before{
  content:"*"; }

.bp3-icon-automatic-updates::before{
  content:""; }

.bp3-icon-badge::before{
  content:""; }

.bp3-icon-ban-circle::before{
  content:""; }

.bp3-icon-bank-account::before{
  content:""; }

.bp3-icon-barcode::before{
  content:""; }

.bp3-icon-blank::before{
  content:""; }

.bp3-icon-blocked-person::before{
  content:""; }

.bp3-icon-bold::before{
  content:""; }

.bp3-icon-book::before{
  content:""; }

.bp3-icon-bookmark::before{
  content:""; }

.bp3-icon-box::before{
  content:""; }

.bp3-icon-briefcase::before{
  content:""; }

.bp3-icon-bring-data::before{
  content:""; }

.bp3-icon-build::before{
  content:""; }

.bp3-icon-calculator::before{
  content:""; }

.bp3-icon-calendar::before{
  content:""; }

.bp3-icon-camera::before{
  content:""; }

.bp3-icon-caret-down::before{
  content:"⌄"; }

.bp3-icon-caret-left::before{
  content:"〈"; }

.bp3-icon-caret-right::before{
  content:"〉"; }

.bp3-icon-caret-up::before{
  content:"⌃"; }

.bp3-icon-cell-tower::before{
  content:""; }

.bp3-icon-changes::before{
  content:""; }

.bp3-icon-chart::before{
  content:""; }

.bp3-icon-chat::before{
  content:""; }

.bp3-icon-chevron-backward::before{
  content:""; }

.bp3-icon-chevron-down::before{
  content:""; }

.bp3-icon-chevron-forward::before{
  content:""; }

.bp3-icon-chevron-left::before{
  content:""; }

.bp3-icon-chevron-right::before{
  content:""; }

.bp3-icon-chevron-up::before{
  content:""; }

.bp3-icon-circle::before{
  content:""; }

.bp3-icon-circle-arrow-down::before{
  content:""; }

.bp3-icon-circle-arrow-left::before{
  content:""; }

.bp3-icon-circle-arrow-right::before{
  content:""; }

.bp3-icon-circle-arrow-up::before{
  content:""; }

.bp3-icon-citation::before{
  content:""; }

.bp3-icon-clean::before{
  content:""; }

.bp3-icon-clipboard::before{
  content:""; }

.bp3-icon-cloud::before{
  content:"☁"; }

.bp3-icon-cloud-download::before{
  content:""; }

.bp3-icon-cloud-upload::before{
  content:""; }

.bp3-icon-code::before{
  content:""; }

.bp3-icon-code-block::before{
  content:""; }

.bp3-icon-cog::before{
  content:""; }

.bp3-icon-collapse-all::before{
  content:""; }

.bp3-icon-column-layout::before{
  content:""; }

.bp3-icon-comment::before{
  content:""; }

.bp3-icon-comparison::before{
  content:""; }

.bp3-icon-compass::before{
  content:""; }

.bp3-icon-compressed::before{
  content:""; }

.bp3-icon-confirm::before{
  content:""; }

.bp3-icon-console::before{
  content:""; }

.bp3-icon-contrast::before{
  content:""; }

.bp3-icon-control::before{
  content:""; }

.bp3-icon-credit-card::before{
  content:""; }

.bp3-icon-cross::before{
  content:"✗"; }

.bp3-icon-crown::before{
  content:""; }

.bp3-icon-cube::before{
  content:""; }

.bp3-icon-cube-add::before{
  content:""; }

.bp3-icon-cube-remove::before{
  content:""; }

.bp3-icon-curved-range-chart::before{
  content:""; }

.bp3-icon-cut::before{
  content:""; }

.bp3-icon-dashboard::before{
  content:""; }

.bp3-icon-data-lineage::before{
  content:""; }

.bp3-icon-database::before{
  content:""; }

.bp3-icon-delete::before{
  content:""; }

.bp3-icon-delta::before{
  content:"Δ"; }

.bp3-icon-derive-column::before{
  content:""; }

.bp3-icon-desktop::before{
  content:""; }

.bp3-icon-diagnosis::before{
  content:""; }

.bp3-icon-diagram-tree::before{
  content:""; }

.bp3-icon-direction-left::before{
  content:""; }

.bp3-icon-direction-right::before{
  content:""; }

.bp3-icon-disable::before{
  content:""; }

.bp3-icon-document::before{
  content:""; }

.bp3-icon-document-open::before{
  content:""; }

.bp3-icon-document-share::before{
  content:""; }

.bp3-icon-dollar::before{
  content:"$"; }

.bp3-icon-dot::before{
  content:"•"; }

.bp3-icon-double-caret-horizontal::before{
  content:""; }

.bp3-icon-double-caret-vertical::before{
  content:""; }

.bp3-icon-double-chevron-down::before{
  content:""; }

.bp3-icon-double-chevron-left::before{
  content:""; }

.bp3-icon-double-chevron-right::before{
  content:""; }

.bp3-icon-double-chevron-up::before{
  content:""; }

.bp3-icon-doughnut-chart::before{
  content:""; }

.bp3-icon-download::before{
  content:""; }

.bp3-icon-drag-handle-horizontal::before{
  content:""; }

.bp3-icon-drag-handle-vertical::before{
  content:""; }

.bp3-icon-draw::before{
  content:""; }

.bp3-icon-drive-time::before{
  content:""; }

.bp3-icon-duplicate::before{
  content:""; }

.bp3-icon-edit::before{
  content:"✎"; }

.bp3-icon-eject::before{
  content:"⏏"; }

.bp3-icon-endorsed::before{
  content:""; }

.bp3-icon-envelope::before{
  content:"✉"; }

.bp3-icon-equals::before{
  content:""; }

.bp3-icon-eraser::before{
  content:""; }

.bp3-icon-error::before{
  content:""; }

.bp3-icon-euro::before{
  content:"€"; }

.bp3-icon-exchange::before{
  content:""; }

.bp3-icon-exclude-row::before{
  content:""; }

.bp3-icon-expand-all::before{
  content:""; }

.bp3-icon-export::before{
  content:""; }

.bp3-icon-eye-off::before{
  content:""; }

.bp3-icon-eye-on::before{
  content:""; }

.bp3-icon-eye-open::before{
  content:""; }

.bp3-icon-fast-backward::before{
  content:""; }

.bp3-icon-fast-forward::before{
  content:""; }

.bp3-icon-feed::before{
  content:""; }

.bp3-icon-feed-subscribed::before{
  content:""; }

.bp3-icon-film::before{
  content:""; }

.bp3-icon-filter::before{
  content:""; }

.bp3-icon-filter-keep::before{
  content:""; }

.bp3-icon-filter-list::before{
  content:""; }

.bp3-icon-filter-open::before{
  content:""; }

.bp3-icon-filter-remove::before{
  content:""; }

.bp3-icon-flag::before{
  content:"⚑"; }

.bp3-icon-flame::before{
  content:""; }

.bp3-icon-flash::before{
  content:""; }

.bp3-icon-floppy-disk::before{
  content:""; }

.bp3-icon-flow-branch::before{
  content:""; }

.bp3-icon-flow-end::before{
  content:""; }

.bp3-icon-flow-linear::before{
  content:""; }

.bp3-icon-flow-review::before{
  content:""; }

.bp3-icon-flow-review-branch::before{
  content:""; }

.bp3-icon-flows::before{
  content:""; }

.bp3-icon-folder-close::before{
  content:""; }

.bp3-icon-folder-new::before{
  content:""; }

.bp3-icon-folder-open::before{
  content:""; }

.bp3-icon-folder-shared::before{
  content:""; }

.bp3-icon-folder-shared-open::before{
  content:""; }

.bp3-icon-follower::before{
  content:""; }

.bp3-icon-following::before{
  content:""; }

.bp3-icon-font::before{
  content:""; }

.bp3-icon-fork::before{
  content:""; }

.bp3-icon-form::before{
  content:""; }

.bp3-icon-full-circle::before{
  content:""; }

.bp3-icon-full-stacked-chart::before{
  content:""; }

.bp3-icon-fullscreen::before{
  content:""; }

.bp3-icon-function::before{
  content:""; }

.bp3-icon-gantt-chart::before{
  content:""; }

.bp3-icon-geolocation::before{
  content:""; }

.bp3-icon-geosearch::before{
  content:""; }

.bp3-icon-git-branch::before{
  content:""; }

.bp3-icon-git-commit::before{
  content:""; }

.bp3-icon-git-merge::before{
  content:""; }

.bp3-icon-git-new-branch::before{
  content:""; }

.bp3-icon-git-pull::before{
  content:""; }

.bp3-icon-git-push::before{
  content:""; }

.bp3-icon-git-repo::before{
  content:""; }

.bp3-icon-glass::before{
  content:""; }

.bp3-icon-globe::before{
  content:""; }

.bp3-icon-globe-network::before{
  content:""; }

.bp3-icon-graph::before{
  content:""; }

.bp3-icon-graph-remove::before{
  content:""; }

.bp3-icon-greater-than::before{
  content:""; }

.bp3-icon-greater-than-or-equal-to::before{
  content:""; }

.bp3-icon-grid::before{
  content:""; }

.bp3-icon-grid-view::before{
  content:""; }

.bp3-icon-group-objects::before{
  content:""; }

.bp3-icon-grouped-bar-chart::before{
  content:""; }

.bp3-icon-hand::before{
  content:""; }

.bp3-icon-hand-down::before{
  content:""; }

.bp3-icon-hand-left::before{
  content:""; }

.bp3-icon-hand-right::before{
  content:""; }

.bp3-icon-hand-up::before{
  content:""; }

.bp3-icon-header::before{
  content:""; }

.bp3-icon-header-one::before{
  content:""; }

.bp3-icon-header-two::before{
  content:""; }

.bp3-icon-headset::before{
  content:""; }

.bp3-icon-heart::before{
  content:"♥"; }

.bp3-icon-heart-broken::before{
  content:""; }

.bp3-icon-heat-grid::before{
  content:""; }

.bp3-icon-heatmap::before{
  content:""; }

.bp3-icon-help::before{
  content:"?"; }

.bp3-icon-helper-management::before{
  content:""; }

.bp3-icon-highlight::before{
  content:""; }

.bp3-icon-history::before{
  content:""; }

.bp3-icon-home::before{
  content:"⌂"; }

.bp3-icon-horizontal-bar-chart::before{
  content:""; }

.bp3-icon-horizontal-bar-chart-asc::before{
  content:""; }

.bp3-icon-horizontal-bar-chart-desc::before{
  content:""; }

.bp3-icon-horizontal-distribution::before{
  content:""; }

.bp3-icon-id-number::before{
  content:""; }

.bp3-icon-image-rotate-left::before{
  content:""; }

.bp3-icon-image-rotate-right::before{
  content:""; }

.bp3-icon-import::before{
  content:""; }

.bp3-icon-inbox::before{
  content:""; }

.bp3-icon-inbox-filtered::before{
  content:""; }

.bp3-icon-inbox-geo::before{
  content:""; }

.bp3-icon-inbox-search::before{
  content:""; }

.bp3-icon-inbox-update::before{
  content:""; }

.bp3-icon-info-sign::before{
  content:"ℹ"; }

.bp3-icon-inheritance::before{
  content:""; }

.bp3-icon-inner-join::before{
  content:""; }

.bp3-icon-insert::before{
  content:""; }

.bp3-icon-intersection::before{
  content:""; }

.bp3-icon-ip-address::before{
  content:""; }

.bp3-icon-issue::before{
  content:""; }

.bp3-icon-issue-closed::before{
  content:""; }

.bp3-icon-issue-new::before{
  content:""; }

.bp3-icon-italic::before{
  content:""; }

.bp3-icon-join-table::before{
  content:""; }

.bp3-icon-key::before{
  content:""; }

.bp3-icon-key-backspace::before{
  content:""; }

.bp3-icon-key-command::before{
  content:""; }

.bp3-icon-key-control::before{
  content:""; }

.bp3-icon-key-delete::before{
  content:""; }

.bp3-icon-key-enter::before{
  content:""; }

.bp3-icon-key-escape::before{
  content:""; }

.bp3-icon-key-option::before{
  content:""; }

.bp3-icon-key-shift::before{
  content:""; }

.bp3-icon-key-tab::before{
  content:""; }

.bp3-icon-known-vehicle::before{
  content:""; }

.bp3-icon-lab-test::before{
  content:""; }

.bp3-icon-label::before{
  content:""; }

.bp3-icon-layer::before{
  content:""; }

.bp3-icon-layers::before{
  content:""; }

.bp3-icon-layout::before{
  content:""; }

.bp3-icon-layout-auto::before{
  content:""; }

.bp3-icon-layout-balloon::before{
  content:""; }

.bp3-icon-layout-circle::before{
  content:""; }

.bp3-icon-layout-grid::before{
  content:""; }

.bp3-icon-layout-group-by::before{
  content:""; }

.bp3-icon-layout-hierarchy::before{
  content:""; }

.bp3-icon-layout-linear::before{
  content:""; }

.bp3-icon-layout-skew-grid::before{
  content:""; }

.bp3-icon-layout-sorted-clusters::before{
  content:""; }

.bp3-icon-learning::before{
  content:""; }

.bp3-icon-left-join::before{
  content:""; }

.bp3-icon-less-than::before{
  content:""; }

.bp3-icon-less-than-or-equal-to::before{
  content:""; }

.bp3-icon-lifesaver::before{
  content:""; }

.bp3-icon-lightbulb::before{
  content:""; }

.bp3-icon-link::before{
  content:""; }

.bp3-icon-list::before{
  content:"☰"; }

.bp3-icon-list-columns::before{
  content:""; }

.bp3-icon-list-detail-view::before{
  content:""; }

.bp3-icon-locate::before{
  content:""; }

.bp3-icon-lock::before{
  content:""; }

.bp3-icon-log-in::before{
  content:""; }

.bp3-icon-log-out::before{
  content:""; }

.bp3-icon-manual::before{
  content:""; }

.bp3-icon-manually-entered-data::before{
  content:""; }

.bp3-icon-map::before{
  content:""; }

.bp3-icon-map-create::before{
  content:""; }

.bp3-icon-map-marker::before{
  content:""; }

.bp3-icon-maximize::before{
  content:""; }

.bp3-icon-media::before{
  content:""; }

.bp3-icon-menu::before{
  content:""; }

.bp3-icon-menu-closed::before{
  content:""; }

.bp3-icon-menu-open::before{
  content:""; }

.bp3-icon-merge-columns::before{
  content:""; }

.bp3-icon-merge-links::before{
  content:""; }

.bp3-icon-minimize::before{
  content:""; }

.bp3-icon-minus::before{
  content:"−"; }

.bp3-icon-mobile-phone::before{
  content:""; }

.bp3-icon-mobile-video::before{
  content:""; }

.bp3-icon-moon::before{
  content:""; }

.bp3-icon-more::before{
  content:""; }

.bp3-icon-mountain::before{
  content:""; }

.bp3-icon-move::before{
  content:""; }

.bp3-icon-mugshot::before{
  content:""; }

.bp3-icon-multi-select::before{
  content:""; }

.bp3-icon-music::before{
  content:""; }

.bp3-icon-new-drawing::before{
  content:""; }

.bp3-icon-new-grid-item::before{
  content:""; }

.bp3-icon-new-layer::before{
  content:""; }

.bp3-icon-new-layers::before{
  content:""; }

.bp3-icon-new-link::before{
  content:""; }

.bp3-icon-new-object::before{
  content:""; }

.bp3-icon-new-person::before{
  content:""; }

.bp3-icon-new-prescription::before{
  content:""; }

.bp3-icon-new-text-box::before{
  content:""; }

.bp3-icon-ninja::before{
  content:""; }

.bp3-icon-not-equal-to::before{
  content:""; }

.bp3-icon-notifications::before{
  content:""; }

.bp3-icon-notifications-updated::before{
  content:""; }

.bp3-icon-numbered-list::before{
  content:""; }

.bp3-icon-numerical::before{
  content:""; }

.bp3-icon-office::before{
  content:""; }

.bp3-icon-offline::before{
  content:""; }

.bp3-icon-oil-field::before{
  content:""; }

.bp3-icon-one-column::before{
  content:""; }

.bp3-icon-outdated::before{
  content:""; }

.bp3-icon-page-layout::before{
  content:""; }

.bp3-icon-panel-stats::before{
  content:""; }

.bp3-icon-panel-table::before{
  content:""; }

.bp3-icon-paperclip::before{
  content:""; }

.bp3-icon-paragraph::before{
  content:""; }

.bp3-icon-path::before{
  content:""; }

.bp3-icon-path-search::before{
  content:""; }

.bp3-icon-pause::before{
  content:""; }

.bp3-icon-people::before{
  content:""; }

.bp3-icon-percentage::before{
  content:""; }

.bp3-icon-person::before{
  content:""; }

.bp3-icon-phone::before{
  content:"☎"; }

.bp3-icon-pie-chart::before{
  content:""; }

.bp3-icon-pin::before{
  content:""; }

.bp3-icon-pivot::before{
  content:""; }

.bp3-icon-pivot-table::before{
  content:""; }

.bp3-icon-play::before{
  content:""; }

.bp3-icon-plus::before{
  content:"+"; }

.bp3-icon-polygon-filter::before{
  content:""; }

.bp3-icon-power::before{
  content:""; }

.bp3-icon-predictive-analysis::before{
  content:""; }

.bp3-icon-prescription::before{
  content:""; }

.bp3-icon-presentation::before{
  content:""; }

.bp3-icon-print::before{
  content:"⎙"; }

.bp3-icon-projects::before{
  content:""; }

.bp3-icon-properties::before{
  content:""; }

.bp3-icon-property::before{
  content:""; }

.bp3-icon-publish-function::before{
  content:""; }

.bp3-icon-pulse::before{
  content:""; }

.bp3-icon-random::before{
  content:""; }

.bp3-icon-record::before{
  content:""; }

.bp3-icon-redo::before{
  content:""; }

.bp3-icon-refresh::before{
  content:""; }

.bp3-icon-regression-chart::before{
  content:""; }

.bp3-icon-remove::before{
  content:""; }

.bp3-icon-remove-column::before{
  content:""; }

.bp3-icon-remove-column-left::before{
  content:""; }

.bp3-icon-remove-column-right::before{
  content:""; }

.bp3-icon-remove-row-bottom::before{
  content:""; }

.bp3-icon-remove-row-top::before{
  content:""; }

.bp3-icon-repeat::before{
  content:""; }

.bp3-icon-reset::before{
  content:""; }

.bp3-icon-resolve::before{
  content:""; }

.bp3-icon-rig::before{
  content:""; }

.bp3-icon-right-join::before{
  content:""; }

.bp3-icon-ring::before{
  content:""; }

.bp3-icon-rotate-document::before{
  content:""; }

.bp3-icon-rotate-page::before{
  content:""; }

.bp3-icon-satellite::before{
  content:""; }

.bp3-icon-saved::before{
  content:""; }

.bp3-icon-scatter-plot::before{
  content:""; }

.bp3-icon-search::before{
  content:""; }

.bp3-icon-search-around::before{
  content:""; }

.bp3-icon-search-template::before{
  content:""; }

.bp3-icon-search-text::before{
  content:""; }

.bp3-icon-segmented-control::before{
  content:""; }

.bp3-icon-select::before{
  content:""; }

.bp3-icon-selection::before{
  content:"⦿"; }

.bp3-icon-send-to::before{
  content:""; }

.bp3-icon-send-to-graph::before{
  content:""; }

.bp3-icon-send-to-map::before{
  content:""; }

.bp3-icon-series-add::before{
  content:""; }

.bp3-icon-series-configuration::before{
  content:""; }

.bp3-icon-series-derived::before{
  content:""; }

.bp3-icon-series-filtered::before{
  content:""; }

.bp3-icon-series-search::before{
  content:""; }

.bp3-icon-settings::before{
  content:""; }

.bp3-icon-share::before{
  content:""; }

.bp3-icon-shield::before{
  content:""; }

.bp3-icon-shop::before{
  content:""; }

.bp3-icon-shopping-cart::before{
  content:""; }

.bp3-icon-signal-search::before{
  content:""; }

.bp3-icon-sim-card::before{
  content:""; }

.bp3-icon-slash::before{
  content:""; }

.bp3-icon-small-cross::before{
  content:""; }

.bp3-icon-small-minus::before{
  content:""; }

.bp3-icon-small-plus::before{
  content:""; }

.bp3-icon-small-tick::before{
  content:""; }

.bp3-icon-snowflake::before{
  content:""; }

.bp3-icon-social-media::before{
  content:""; }

.bp3-icon-sort::before{
  content:""; }

.bp3-icon-sort-alphabetical::before{
  content:""; }

.bp3-icon-sort-alphabetical-desc::before{
  content:""; }

.bp3-icon-sort-asc::before{
  content:""; }

.bp3-icon-sort-desc::before{
  content:""; }

.bp3-icon-sort-numerical::before{
  content:""; }

.bp3-icon-sort-numerical-desc::before{
  content:""; }

.bp3-icon-split-columns::before{
  content:""; }

.bp3-icon-square::before{
  content:""; }

.bp3-icon-stacked-chart::before{
  content:""; }

.bp3-icon-star::before{
  content:"★"; }

.bp3-icon-star-empty::before{
  content:"☆"; }

.bp3-icon-step-backward::before{
  content:""; }

.bp3-icon-step-chart::before{
  content:""; }

.bp3-icon-step-forward::before{
  content:""; }

.bp3-icon-stop::before{
  content:""; }

.bp3-icon-stopwatch::before{
  content:""; }

.bp3-icon-strikethrough::before{
  content:""; }

.bp3-icon-style::before{
  content:""; }

.bp3-icon-swap-horizontal::before{
  content:""; }

.bp3-icon-swap-vertical::before{
  content:""; }

.bp3-icon-symbol-circle::before{
  content:""; }

.bp3-icon-symbol-cross::before{
  content:""; }

.bp3-icon-symbol-diamond::before{
  content:""; }

.bp3-icon-symbol-square::before{
  content:""; }

.bp3-icon-symbol-triangle-down::before{
  content:""; }

.bp3-icon-symbol-triangle-up::before{
  content:""; }

.bp3-icon-tag::before{
  content:""; }

.bp3-icon-take-action::before{
  content:""; }

.bp3-icon-taxi::before{
  content:""; }

.bp3-icon-text-highlight::before{
  content:""; }

.bp3-icon-th::before{
  content:""; }

.bp3-icon-th-derived::before{
  content:""; }

.bp3-icon-th-disconnect::before{
  content:""; }

.bp3-icon-th-filtered::before{
  content:""; }

.bp3-icon-th-list::before{
  content:""; }

.bp3-icon-thumbs-down::before{
  content:""; }

.bp3-icon-thumbs-up::before{
  content:""; }

.bp3-icon-tick::before{
  content:"✓"; }

.bp3-icon-tick-circle::before{
  content:""; }

.bp3-icon-time::before{
  content:"⏲"; }

.bp3-icon-timeline-area-chart::before{
  content:""; }

.bp3-icon-timeline-bar-chart::before{
  content:""; }

.bp3-icon-timeline-events::before{
  content:""; }

.bp3-icon-timeline-line-chart::before{
  content:""; }

.bp3-icon-tint::before{
  content:""; }

.bp3-icon-torch::before{
  content:""; }

.bp3-icon-tractor::before{
  content:""; }

.bp3-icon-train::before{
  content:""; }

.bp3-icon-translate::before{
  content:""; }

.bp3-icon-trash::before{
  content:""; }

.bp3-icon-tree::before{
  content:""; }

.bp3-icon-trending-down::before{
  content:""; }

.bp3-icon-trending-up::before{
  content:""; }

.bp3-icon-truck::before{
  content:""; }

.bp3-icon-two-columns::before{
  content:""; }

.bp3-icon-unarchive::before{
  content:""; }

.bp3-icon-underline::before{
  content:"⎁"; }

.bp3-icon-undo::before{
  content:"⎌"; }

.bp3-icon-ungroup-objects::before{
  content:""; }

.bp3-icon-unknown-vehicle::before{
  content:""; }

.bp3-icon-unlock::before{
  content:""; }

.bp3-icon-unpin::before{
  content:""; }

.bp3-icon-unresolve::before{
  content:""; }

.bp3-icon-updated::before{
  content:""; }

.bp3-icon-upload::before{
  content:""; }

.bp3-icon-user::before{
  content:""; }

.bp3-icon-variable::before{
  content:""; }

.bp3-icon-vertical-bar-chart-asc::before{
  content:""; }

.bp3-icon-vertical-bar-chart-desc::before{
  content:""; }

.bp3-icon-vertical-distribution::before{
  content:""; }

.bp3-icon-video::before{
  content:""; }

.bp3-icon-volume-down::before{
  content:""; }

.bp3-icon-volume-off::before{
  content:""; }

.bp3-icon-volume-up::before{
  content:""; }

.bp3-icon-walk::before{
  content:""; }

.bp3-icon-warning-sign::before{
  content:""; }

.bp3-icon-waterfall-chart::before{
  content:""; }

.bp3-icon-widget::before{
  content:""; }

.bp3-icon-widget-button::before{
  content:""; }

.bp3-icon-widget-footer::before{
  content:""; }

.bp3-icon-widget-header::before{
  content:""; }

.bp3-icon-wrench::before{
  content:""; }

.bp3-icon-zoom-in::before{
  content:""; }

.bp3-icon-zoom-out::before{
  content:""; }

.bp3-icon-zoom-to-fit::before{
  content:""; }
.bp3-submenu > .bp3-popover-wrapper{
  display:block; }

.bp3-submenu .bp3-popover-target{
  display:block; }
  .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-menu-item{ }

.bp3-submenu.bp3-popover{
  -webkit-box-shadow:none;
          box-shadow:none;
  padding:0 5px; }
  .bp3-submenu.bp3-popover > .bp3-popover-content{
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 2px 4px rgba(16, 22, 26, 0.2), 0 8px 24px rgba(16, 22, 26, 0.2);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 2px 4px rgba(16, 22, 26, 0.2), 0 8px 24px rgba(16, 22, 26, 0.2); }
  .bp3-dark .bp3-submenu.bp3-popover, .bp3-submenu.bp3-popover.bp3-dark{
    -webkit-box-shadow:none;
            box-shadow:none; }
    .bp3-dark .bp3-submenu.bp3-popover > .bp3-popover-content, .bp3-submenu.bp3-popover.bp3-dark > .bp3-popover-content{
      -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 2px 4px rgba(16, 22, 26, 0.4), 0 8px 24px rgba(16, 22, 26, 0.4);
              box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 2px 4px rgba(16, 22, 26, 0.4), 0 8px 24px rgba(16, 22, 26, 0.4); }
.bp3-menu{
  background:#ffffff;
  border-radius:3px;
  color:#182026;
  list-style:none;
  margin:0;
  min-width:180px;
  padding:5px;
  text-align:left; }

.bp3-menu-divider{
  border-top:1px solid rgba(16, 22, 26, 0.15);
  display:block;
  margin:5px; }
  .bp3-dark .bp3-menu-divider{
    border-top-color:rgba(255, 255, 255, 0.15); }

.bp3-menu-item{
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex;
  -webkit-box-orient:horizontal;
  -webkit-box-direction:normal;
      -ms-flex-direction:row;
          flex-direction:row;
  -webkit-box-align:start;
      -ms-flex-align:start;
          align-items:flex-start;
  border-radius:2px;
  color:inherit;
  line-height:20px;
  padding:5px 7px;
  text-decoration:none;
  -webkit-user-select:none;
     -moz-user-select:none;
      -ms-user-select:none;
          user-select:none; }
  .bp3-menu-item > *{
    -webkit-box-flex:0;
        -ms-flex-positive:0;
            flex-grow:0;
    -ms-flex-negative:0;
        flex-shrink:0; }
  .bp3-menu-item > .bp3-fill{
    -webkit-box-flex:1;
        -ms-flex-positive:1;
            flex-grow:1;
    -ms-flex-negative:1;
        flex-shrink:1; }
  .bp3-menu-item::before,
  .bp3-menu-item > *{
    margin-right:7px; }
  .bp3-menu-item:empty::before,
  .bp3-menu-item > :last-child{
    margin-right:0; }
  .bp3-menu-item > .bp3-fill{
    word-break:break-word; }
  .bp3-menu-item:hover, .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-menu-item{
    background-color:rgba(167, 182, 194, 0.3);
    cursor:pointer;
    text-decoration:none; }
  .bp3-menu-item.bp3-disabled{
    background-color:inherit;
    color:rgba(92, 112, 128, 0.6);
    cursor:not-allowed; }
  .bp3-dark .bp3-menu-item{
    color:inherit; }
    .bp3-dark .bp3-menu-item:hover, .bp3-dark .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-menu-item, .bp3-submenu .bp3-dark .bp3-popover-target.bp3-popover-open > .bp3-menu-item{
      background-color:rgba(138, 155, 168, 0.15);
      color:inherit; }
    .bp3-dark .bp3-menu-item.bp3-disabled{
      background-color:inherit;
      color:rgba(167, 182, 194, 0.6); }
  .bp3-menu-item.bp3-intent-primary{
    color:#106ba3; }
    .bp3-menu-item.bp3-intent-primary .bp3-icon{
      color:inherit; }
    .bp3-menu-item.bp3-intent-primary::before, .bp3-menu-item.bp3-intent-primary::after,
    .bp3-menu-item.bp3-intent-primary .bp3-menu-item-label{
      color:#106ba3; }
    .bp3-menu-item.bp3-intent-primary:hover, .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-primary.bp3-menu-item, .bp3-menu-item.bp3-intent-primary.bp3-active{
      background-color:#137cbd; }
    .bp3-menu-item.bp3-intent-primary:active{
      background-color:#106ba3; }
    .bp3-menu-item.bp3-intent-primary:hover, .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-primary.bp3-menu-item, .bp3-menu-item.bp3-intent-primary:hover::before, .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-primary.bp3-menu-item::before, .bp3-menu-item.bp3-intent-primary:hover::after, .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-primary.bp3-menu-item::after,
    .bp3-menu-item.bp3-intent-primary:hover .bp3-menu-item-label,
    .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-primary.bp3-menu-item .bp3-menu-item-label, .bp3-menu-item.bp3-intent-primary:active, .bp3-menu-item.bp3-intent-primary:active::before, .bp3-menu-item.bp3-intent-primary:active::after,
    .bp3-menu-item.bp3-intent-primary:active .bp3-menu-item-label, .bp3-menu-item.bp3-intent-primary.bp3-active, .bp3-menu-item.bp3-intent-primary.bp3-active::before, .bp3-menu-item.bp3-intent-primary.bp3-active::after,
    .bp3-menu-item.bp3-intent-primary.bp3-active .bp3-menu-item-label{
      color:#ffffff; }
  .bp3-menu-item.bp3-intent-success{
    color:#0d8050; }
    .bp3-menu-item.bp3-intent-success .bp3-icon{
      color:inherit; }
    .bp3-menu-item.bp3-intent-success::before, .bp3-menu-item.bp3-intent-success::after,
    .bp3-menu-item.bp3-intent-success .bp3-menu-item-label{
      color:#0d8050; }
    .bp3-menu-item.bp3-intent-success:hover, .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-success.bp3-menu-item, .bp3-menu-item.bp3-intent-success.bp3-active{
      background-color:#0f9960; }
    .bp3-menu-item.bp3-intent-success:active{
      background-color:#0d8050; }
    .bp3-menu-item.bp3-intent-success:hover, .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-success.bp3-menu-item, .bp3-menu-item.bp3-intent-success:hover::before, .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-success.bp3-menu-item::before, .bp3-menu-item.bp3-intent-success:hover::after, .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-success.bp3-menu-item::after,
    .bp3-menu-item.bp3-intent-success:hover .bp3-menu-item-label,
    .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-success.bp3-menu-item .bp3-menu-item-label, .bp3-menu-item.bp3-intent-success:active, .bp3-menu-item.bp3-intent-success:active::before, .bp3-menu-item.bp3-intent-success:active::after,
    .bp3-menu-item.bp3-intent-success:active .bp3-menu-item-label, .bp3-menu-item.bp3-intent-success.bp3-active, .bp3-menu-item.bp3-intent-success.bp3-active::before, .bp3-menu-item.bp3-intent-success.bp3-active::after,
    .bp3-menu-item.bp3-intent-success.bp3-active .bp3-menu-item-label{
      color:#ffffff; }
  .bp3-menu-item.bp3-intent-warning{
    color:#bf7326; }
    .bp3-menu-item.bp3-intent-warning .bp3-icon{
      color:inherit; }
    .bp3-menu-item.bp3-intent-warning::before, .bp3-menu-item.bp3-intent-warning::after,
    .bp3-menu-item.bp3-intent-warning .bp3-menu-item-label{
      color:#bf7326; }
    .bp3-menu-item.bp3-intent-warning:hover, .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-warning.bp3-menu-item, .bp3-menu-item.bp3-intent-warning.bp3-active{
      background-color:#d9822b; }
    .bp3-menu-item.bp3-intent-warning:active{
      background-color:#bf7326; }
    .bp3-menu-item.bp3-intent-warning:hover, .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-warning.bp3-menu-item, .bp3-menu-item.bp3-intent-warning:hover::before, .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-warning.bp3-menu-item::before, .bp3-menu-item.bp3-intent-warning:hover::after, .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-warning.bp3-menu-item::after,
    .bp3-menu-item.bp3-intent-warning:hover .bp3-menu-item-label,
    .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-warning.bp3-menu-item .bp3-menu-item-label, .bp3-menu-item.bp3-intent-warning:active, .bp3-menu-item.bp3-intent-warning:active::before, .bp3-menu-item.bp3-intent-warning:active::after,
    .bp3-menu-item.bp3-intent-warning:active .bp3-menu-item-label, .bp3-menu-item.bp3-intent-warning.bp3-active, .bp3-menu-item.bp3-intent-warning.bp3-active::before, .bp3-menu-item.bp3-intent-warning.bp3-active::after,
    .bp3-menu-item.bp3-intent-warning.bp3-active .bp3-menu-item-label{
      color:#ffffff; }
  .bp3-menu-item.bp3-intent-danger{
    color:#c23030; }
    .bp3-menu-item.bp3-intent-danger .bp3-icon{
      color:inherit; }
    .bp3-menu-item.bp3-intent-danger::before, .bp3-menu-item.bp3-intent-danger::after,
    .bp3-menu-item.bp3-intent-danger .bp3-menu-item-label{
      color:#c23030; }
    .bp3-menu-item.bp3-intent-danger:hover, .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-danger.bp3-menu-item, .bp3-menu-item.bp3-intent-danger.bp3-active{
      background-color:#db3737; }
    .bp3-menu-item.bp3-intent-danger:active{
      background-color:#c23030; }
    .bp3-menu-item.bp3-intent-danger:hover, .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-danger.bp3-menu-item, .bp3-menu-item.bp3-intent-danger:hover::before, .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-danger.bp3-menu-item::before, .bp3-menu-item.bp3-intent-danger:hover::after, .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-danger.bp3-menu-item::after,
    .bp3-menu-item.bp3-intent-danger:hover .bp3-menu-item-label,
    .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-danger.bp3-menu-item .bp3-menu-item-label, .bp3-menu-item.bp3-intent-danger:active, .bp3-menu-item.bp3-intent-danger:active::before, .bp3-menu-item.bp3-intent-danger:active::after,
    .bp3-menu-item.bp3-intent-danger:active .bp3-menu-item-label, .bp3-menu-item.bp3-intent-danger.bp3-active, .bp3-menu-item.bp3-intent-danger.bp3-active::before, .bp3-menu-item.bp3-intent-danger.bp3-active::after,
    .bp3-menu-item.bp3-intent-danger.bp3-active .bp3-menu-item-label{
      color:#ffffff; }
  .bp3-menu-item::before{
    font-family:"Icons16", sans-serif;
    font-size:16px;
    font-style:normal;
    font-weight:400;
    line-height:1;
    -moz-osx-font-smoothing:grayscale;
    -webkit-font-smoothing:antialiased;
    margin-right:7px; }
  .bp3-menu-item::before,
  .bp3-menu-item > .bp3-icon{
    color:#5c7080;
    margin-top:2px; }
  .bp3-menu-item .bp3-menu-item-label{
    color:#5c7080; }
  .bp3-menu-item:hover, .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-menu-item{
    color:inherit; }
  .bp3-menu-item.bp3-active, .bp3-menu-item:active{
    background-color:rgba(115, 134, 148, 0.3); }
  .bp3-menu-item.bp3-disabled{
    background-color:inherit !important;
    color:rgba(92, 112, 128, 0.6) !important;
    cursor:not-allowed !important;
    outline:none !important; }
    .bp3-menu-item.bp3-disabled::before,
    .bp3-menu-item.bp3-disabled > .bp3-icon,
    .bp3-menu-item.bp3-disabled .bp3-menu-item-label{
      color:rgba(92, 112, 128, 0.6) !important; }
  .bp3-large .bp3-menu-item{
    font-size:16px;
    line-height:22px;
    padding:9px 7px; }
    .bp3-large .bp3-menu-item .bp3-icon{
      margin-top:3px; }
    .bp3-large .bp3-menu-item::before{
      font-family:"Icons20", sans-serif;
      font-size:20px;
      font-style:normal;
      font-weight:400;
      line-height:1;
      -moz-osx-font-smoothing:grayscale;
      -webkit-font-smoothing:antialiased;
      margin-right:10px;
      margin-top:1px; }

button.bp3-menu-item{
  background:none;
  border:none;
  text-align:left;
  width:100%; }
.bp3-menu-header{
  border-top:1px solid rgba(16, 22, 26, 0.15);
  display:block;
  margin:5px;
  cursor:default;
  padding-left:2px; }
  .bp3-dark .bp3-menu-header{
    border-top-color:rgba(255, 255, 255, 0.15); }
  .bp3-menu-header:first-of-type{
    border-top:none; }
  .bp3-menu-header > h6{
    color:#182026;
    font-weight:600;
    overflow:hidden;
    text-overflow:ellipsis;
    white-space:nowrap;
    word-wrap:normal;
    line-height:17px;
    margin:0;
    padding:10px 7px 0 1px; }
    .bp3-dark .bp3-menu-header > h6{
      color:#f5f8fa; }
  .bp3-menu-header:first-of-type > h6{
    padding-top:0; }
  .bp3-large .bp3-menu-header > h6{
    font-size:18px;
    padding-bottom:5px;
    padding-top:15px; }
  .bp3-large .bp3-menu-header:first-of-type > h6{
    padding-top:0; }

.bp3-dark .bp3-menu{
  background:#30404d;
  color:#f5f8fa; }

.bp3-dark .bp3-menu-item{ }
  .bp3-dark .bp3-menu-item.bp3-intent-primary{
    color:#48aff0; }
    .bp3-dark .bp3-menu-item.bp3-intent-primary .bp3-icon{
      color:inherit; }
    .bp3-dark .bp3-menu-item.bp3-intent-primary::before, .bp3-dark .bp3-menu-item.bp3-intent-primary::after,
    .bp3-dark .bp3-menu-item.bp3-intent-primary .bp3-menu-item-label{
      color:#48aff0; }
    .bp3-dark .bp3-menu-item.bp3-intent-primary:hover, .bp3-dark .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-primary.bp3-menu-item, .bp3-submenu .bp3-dark .bp3-popover-target.bp3-popover-open > .bp3-intent-primary.bp3-menu-item, .bp3-dark .bp3-menu-item.bp3-intent-primary.bp3-active{
      background-color:#137cbd; }
    .bp3-dark .bp3-menu-item.bp3-intent-primary:active{
      background-color:#106ba3; }
    .bp3-dark .bp3-menu-item.bp3-intent-primary:hover, .bp3-dark .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-primary.bp3-menu-item, .bp3-submenu .bp3-dark .bp3-popover-target.bp3-popover-open > .bp3-intent-primary.bp3-menu-item, .bp3-dark .bp3-menu-item.bp3-intent-primary:hover::before, .bp3-dark .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-primary.bp3-menu-item::before, .bp3-submenu .bp3-dark .bp3-popover-target.bp3-popover-open > .bp3-intent-primary.bp3-menu-item::before, .bp3-dark .bp3-menu-item.bp3-intent-primary:hover::after, .bp3-dark .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-primary.bp3-menu-item::after, .bp3-submenu .bp3-dark .bp3-popover-target.bp3-popover-open > .bp3-intent-primary.bp3-menu-item::after,
    .bp3-dark .bp3-menu-item.bp3-intent-primary:hover .bp3-menu-item-label,
    .bp3-dark .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-primary.bp3-menu-item .bp3-menu-item-label,
    .bp3-submenu .bp3-dark .bp3-popover-target.bp3-popover-open > .bp3-intent-primary.bp3-menu-item .bp3-menu-item-label, .bp3-dark .bp3-menu-item.bp3-intent-primary:active, .bp3-dark .bp3-menu-item.bp3-intent-primary:active::before, .bp3-dark .bp3-menu-item.bp3-intent-primary:active::after,
    .bp3-dark .bp3-menu-item.bp3-intent-primary:active .bp3-menu-item-label, .bp3-dark .bp3-menu-item.bp3-intent-primary.bp3-active, .bp3-dark .bp3-menu-item.bp3-intent-primary.bp3-active::before, .bp3-dark .bp3-menu-item.bp3-intent-primary.bp3-active::after,
    .bp3-dark .bp3-menu-item.bp3-intent-primary.bp3-active .bp3-menu-item-label{
      color:#ffffff; }
  .bp3-dark .bp3-menu-item.bp3-intent-success{
    color:#3dcc91; }
    .bp3-dark .bp3-menu-item.bp3-intent-success .bp3-icon{
      color:inherit; }
    .bp3-dark .bp3-menu-item.bp3-intent-success::before, .bp3-dark .bp3-menu-item.bp3-intent-success::after,
    .bp3-dark .bp3-menu-item.bp3-intent-success .bp3-menu-item-label{
      color:#3dcc91; }
    .bp3-dark .bp3-menu-item.bp3-intent-success:hover, .bp3-dark .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-success.bp3-menu-item, .bp3-submenu .bp3-dark .bp3-popover-target.bp3-popover-open > .bp3-intent-success.bp3-menu-item, .bp3-dark .bp3-menu-item.bp3-intent-success.bp3-active{
      background-color:#0f9960; }
    .bp3-dark .bp3-menu-item.bp3-intent-success:active{
      background-color:#0d8050; }
    .bp3-dark .bp3-menu-item.bp3-intent-success:hover, .bp3-dark .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-success.bp3-menu-item, .bp3-submenu .bp3-dark .bp3-popover-target.bp3-popover-open > .bp3-intent-success.bp3-menu-item, .bp3-dark .bp3-menu-item.bp3-intent-success:hover::before, .bp3-dark .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-success.bp3-menu-item::before, .bp3-submenu .bp3-dark .bp3-popover-target.bp3-popover-open > .bp3-intent-success.bp3-menu-item::before, .bp3-dark .bp3-menu-item.bp3-intent-success:hover::after, .bp3-dark .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-success.bp3-menu-item::after, .bp3-submenu .bp3-dark .bp3-popover-target.bp3-popover-open > .bp3-intent-success.bp3-menu-item::after,
    .bp3-dark .bp3-menu-item.bp3-intent-success:hover .bp3-menu-item-label,
    .bp3-dark .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-success.bp3-menu-item .bp3-menu-item-label,
    .bp3-submenu .bp3-dark .bp3-popover-target.bp3-popover-open > .bp3-intent-success.bp3-menu-item .bp3-menu-item-label, .bp3-dark .bp3-menu-item.bp3-intent-success:active, .bp3-dark .bp3-menu-item.bp3-intent-success:active::before, .bp3-dark .bp3-menu-item.bp3-intent-success:active::after,
    .bp3-dark .bp3-menu-item.bp3-intent-success:active .bp3-menu-item-label, .bp3-dark .bp3-menu-item.bp3-intent-success.bp3-active, .bp3-dark .bp3-menu-item.bp3-intent-success.bp3-active::before, .bp3-dark .bp3-menu-item.bp3-intent-success.bp3-active::after,
    .bp3-dark .bp3-menu-item.bp3-intent-success.bp3-active .bp3-menu-item-label{
      color:#ffffff; }
  .bp3-dark .bp3-menu-item.bp3-intent-warning{
    color:#ffb366; }
    .bp3-dark .bp3-menu-item.bp3-intent-warning .bp3-icon{
      color:inherit; }
    .bp3-dark .bp3-menu-item.bp3-intent-warning::before, .bp3-dark .bp3-menu-item.bp3-intent-warning::after,
    .bp3-dark .bp3-menu-item.bp3-intent-warning .bp3-menu-item-label{
      color:#ffb366; }
    .bp3-dark .bp3-menu-item.bp3-intent-warning:hover, .bp3-dark .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-warning.bp3-menu-item, .bp3-submenu .bp3-dark .bp3-popover-target.bp3-popover-open > .bp3-intent-warning.bp3-menu-item, .bp3-dark .bp3-menu-item.bp3-intent-warning.bp3-active{
      background-color:#d9822b; }
    .bp3-dark .bp3-menu-item.bp3-intent-warning:active{
      background-color:#bf7326; }
    .bp3-dark .bp3-menu-item.bp3-intent-warning:hover, .bp3-dark .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-warning.bp3-menu-item, .bp3-submenu .bp3-dark .bp3-popover-target.bp3-popover-open > .bp3-intent-warning.bp3-menu-item, .bp3-dark .bp3-menu-item.bp3-intent-warning:hover::before, .bp3-dark .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-warning.bp3-menu-item::before, .bp3-submenu .bp3-dark .bp3-popover-target.bp3-popover-open > .bp3-intent-warning.bp3-menu-item::before, .bp3-dark .bp3-menu-item.bp3-intent-warning:hover::after, .bp3-dark .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-warning.bp3-menu-item::after, .bp3-submenu .bp3-dark .bp3-popover-target.bp3-popover-open > .bp3-intent-warning.bp3-menu-item::after,
    .bp3-dark .bp3-menu-item.bp3-intent-warning:hover .bp3-menu-item-label,
    .bp3-dark .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-warning.bp3-menu-item .bp3-menu-item-label,
    .bp3-submenu .bp3-dark .bp3-popover-target.bp3-popover-open > .bp3-intent-warning.bp3-menu-item .bp3-menu-item-label, .bp3-dark .bp3-menu-item.bp3-intent-warning:active, .bp3-dark .bp3-menu-item.bp3-intent-warning:active::before, .bp3-dark .bp3-menu-item.bp3-intent-warning:active::after,
    .bp3-dark .bp3-menu-item.bp3-intent-warning:active .bp3-menu-item-label, .bp3-dark .bp3-menu-item.bp3-intent-warning.bp3-active, .bp3-dark .bp3-menu-item.bp3-intent-warning.bp3-active::before, .bp3-dark .bp3-menu-item.bp3-intent-warning.bp3-active::after,
    .bp3-dark .bp3-menu-item.bp3-intent-warning.bp3-active .bp3-menu-item-label{
      color:#ffffff; }
  .bp3-dark .bp3-menu-item.bp3-intent-danger{
    color:#ff7373; }
    .bp3-dark .bp3-menu-item.bp3-intent-danger .bp3-icon{
      color:inherit; }
    .bp3-dark .bp3-menu-item.bp3-intent-danger::before, .bp3-dark .bp3-menu-item.bp3-intent-danger::after,
    .bp3-dark .bp3-menu-item.bp3-intent-danger .bp3-menu-item-label{
      color:#ff7373; }
    .bp3-dark .bp3-menu-item.bp3-intent-danger:hover, .bp3-dark .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-danger.bp3-menu-item, .bp3-submenu .bp3-dark .bp3-popover-target.bp3-popover-open > .bp3-intent-danger.bp3-menu-item, .bp3-dark .bp3-menu-item.bp3-intent-danger.bp3-active{
      background-color:#db3737; }
    .bp3-dark .bp3-menu-item.bp3-intent-danger:active{
      background-color:#c23030; }
    .bp3-dark .bp3-menu-item.bp3-intent-danger:hover, .bp3-dark .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-danger.bp3-menu-item, .bp3-submenu .bp3-dark .bp3-popover-target.bp3-popover-open > .bp3-intent-danger.bp3-menu-item, .bp3-dark .bp3-menu-item.bp3-intent-danger:hover::before, .bp3-dark .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-danger.bp3-menu-item::before, .bp3-submenu .bp3-dark .bp3-popover-target.bp3-popover-open > .bp3-intent-danger.bp3-menu-item::before, .bp3-dark .bp3-menu-item.bp3-intent-danger:hover::after, .bp3-dark .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-danger.bp3-menu-item::after, .bp3-submenu .bp3-dark .bp3-popover-target.bp3-popover-open > .bp3-intent-danger.bp3-menu-item::after,
    .bp3-dark .bp3-menu-item.bp3-intent-danger:hover .bp3-menu-item-label,
    .bp3-dark .bp3-submenu .bp3-popover-target.bp3-popover-open > .bp3-intent-danger.bp3-menu-item .bp3-menu-item-label,
    .bp3-submenu .bp3-dark .bp3-popover-target.bp3-popover-open > .bp3-intent-danger.bp3-menu-item .bp3-menu-item-label, .bp3-dark .bp3-menu-item.bp3-intent-danger:active, .bp3-dark .bp3-menu-item.bp3-intent-danger:active::before, .bp3-dark .bp3-menu-item.bp3-intent-danger:active::after,
    .bp3-dark .bp3-menu-item.bp3-intent-danger:active .bp3-menu-item-label, .bp3-dark .bp3-menu-item.bp3-intent-danger.bp3-active, .bp3-dark .bp3-menu-item.bp3-intent-danger.bp3-active::before, .bp3-dark .bp3-menu-item.bp3-intent-danger.bp3-active::after,
    .bp3-dark .bp3-menu-item.bp3-intent-danger.bp3-active .bp3-menu-item-label{
      color:#ffffff; }
  .bp3-dark .bp3-menu-item::before,
  .bp3-dark .bp3-menu-item > .bp3-icon{
    color:#a7b6c2; }
  .bp3-dark .bp3-menu-item .bp3-menu-item-label{
    color:#a7b6c2; }
  .bp3-dark .bp3-menu-item.bp3-active, .bp3-dark .bp3-menu-item:active{
    background-color:rgba(138, 155, 168, 0.3); }
  .bp3-dark .bp3-menu-item.bp3-disabled{
    color:rgba(167, 182, 194, 0.6) !important; }
    .bp3-dark .bp3-menu-item.bp3-disabled::before,
    .bp3-dark .bp3-menu-item.bp3-disabled > .bp3-icon,
    .bp3-dark .bp3-menu-item.bp3-disabled .bp3-menu-item-label{
      color:rgba(167, 182, 194, 0.6) !important; }

.bp3-dark .bp3-menu-divider,
.bp3-dark .bp3-menu-header{
  border-color:rgba(255, 255, 255, 0.15); }

.bp3-dark .bp3-menu-header > h6{
  color:#f5f8fa; }

.bp3-label .bp3-menu{
  margin-top:5px; }
.bp3-navbar{
  background-color:#ffffff;
  -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 0 0 rgba(16, 22, 26, 0), 0 1px 1px rgba(16, 22, 26, 0.2);
          box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 0 0 rgba(16, 22, 26, 0), 0 1px 1px rgba(16, 22, 26, 0.2);
  height:50px;
  padding:0 15px;
  position:relative;
  width:100%;
  z-index:10; }
  .bp3-navbar.bp3-dark,
  .bp3-dark .bp3-navbar{
    background-color:#394b59; }
  .bp3-navbar.bp3-dark{
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), 0 0 0 rgba(16, 22, 26, 0), 0 1px 1px rgba(16, 22, 26, 0.4);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), 0 0 0 rgba(16, 22, 26, 0), 0 1px 1px rgba(16, 22, 26, 0.4); }
  .bp3-dark .bp3-navbar{
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 0 0 rgba(16, 22, 26, 0), 0 1px 1px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 0 0 rgba(16, 22, 26, 0), 0 1px 1px rgba(16, 22, 26, 0.4); }
  .bp3-navbar.bp3-fixed-top{
    left:0;
    position:fixed;
    right:0;
    top:0; }

.bp3-navbar-heading{
  font-size:16px;
  margin-right:15px; }

.bp3-navbar-group{
  -webkit-box-align:center;
      -ms-flex-align:center;
          align-items:center;
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex;
  height:50px; }
  .bp3-navbar-group.bp3-align-left{
    float:left; }
  .bp3-navbar-group.bp3-align-right{
    float:right; }

.bp3-navbar-divider{
  border-left:1px solid rgba(16, 22, 26, 0.15);
  height:20px;
  margin:0 10px; }
  .bp3-dark .bp3-navbar-divider{
    border-left-color:rgba(255, 255, 255, 0.15); }
.bp3-non-ideal-state{
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex;
  -webkit-box-orient:vertical;
  -webkit-box-direction:normal;
      -ms-flex-direction:column;
          flex-direction:column;
  -webkit-box-align:center;
      -ms-flex-align:center;
          align-items:center;
  height:100%;
  -webkit-box-pack:center;
      -ms-flex-pack:center;
          justify-content:center;
  text-align:center;
  width:100%; }
  .bp3-non-ideal-state > *{
    -webkit-box-flex:0;
        -ms-flex-positive:0;
            flex-grow:0;
    -ms-flex-negative:0;
        flex-shrink:0; }
  .bp3-non-ideal-state > .bp3-fill{
    -webkit-box-flex:1;
        -ms-flex-positive:1;
            flex-grow:1;
    -ms-flex-negative:1;
        flex-shrink:1; }
  .bp3-non-ideal-state::before,
  .bp3-non-ideal-state > *{
    margin-bottom:20px; }
  .bp3-non-ideal-state:empty::before,
  .bp3-non-ideal-state > :last-child{
    margin-bottom:0; }
  .bp3-non-ideal-state > *{
    max-width:400px; }

.bp3-non-ideal-state-visual{
  color:rgba(92, 112, 128, 0.6);
  font-size:60px; }
  .bp3-dark .bp3-non-ideal-state-visual{
    color:rgba(167, 182, 194, 0.6); }

.bp3-overflow-list{
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex;
  -ms-flex-wrap:nowrap;
      flex-wrap:nowrap;
  min-width:0; }

.bp3-overflow-list-spacer{
  -ms-flex-negative:1;
      flex-shrink:1;
  width:1px; }

body.bp3-overlay-open{
  overflow:hidden; }

.bp3-overlay{
  bottom:0;
  left:0;
  position:static;
  right:0;
  top:0;
  z-index:20; }
  .bp3-overlay:not(.bp3-overlay-open){
    pointer-events:none; }
  .bp3-overlay.bp3-overlay-container{
    overflow:hidden;
    position:fixed; }
    .bp3-overlay.bp3-overlay-container.bp3-overlay-inline{
      position:absolute; }
  .bp3-overlay.bp3-overlay-scroll-container{
    overflow:auto;
    position:fixed; }
    .bp3-overlay.bp3-overlay-scroll-container.bp3-overlay-inline{
      position:absolute; }
  .bp3-overlay.bp3-overlay-inline{
    display:inline;
    overflow:visible; }

.bp3-overlay-content{
  position:fixed;
  z-index:20; }
  .bp3-overlay-inline .bp3-overlay-content,
  .bp3-overlay-scroll-container .bp3-overlay-content{
    position:absolute; }

.bp3-overlay-backdrop{
  bottom:0;
  left:0;
  position:fixed;
  right:0;
  top:0;
  opacity:1;
  background-color:rgba(16, 22, 26, 0.7);
  overflow:auto;
  -webkit-user-select:none;
     -moz-user-select:none;
      -ms-user-select:none;
          user-select:none;
  z-index:20; }
  .bp3-overlay-backdrop.bp3-overlay-enter, .bp3-overlay-backdrop.bp3-overlay-appear{
    opacity:0; }
  .bp3-overlay-backdrop.bp3-overlay-enter-active, .bp3-overlay-backdrop.bp3-overlay-appear-active{
    opacity:1;
    -webkit-transition-delay:0;
            transition-delay:0;
    -webkit-transition-duration:200ms;
            transition-duration:200ms;
    -webkit-transition-property:opacity;
    transition-property:opacity;
    -webkit-transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9);
            transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9); }
  .bp3-overlay-backdrop.bp3-overlay-exit{
    opacity:1; }
  .bp3-overlay-backdrop.bp3-overlay-exit-active{
    opacity:0;
    -webkit-transition-delay:0;
            transition-delay:0;
    -webkit-transition-duration:200ms;
            transition-duration:200ms;
    -webkit-transition-property:opacity;
    transition-property:opacity;
    -webkit-transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9);
            transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9); }
  .bp3-overlay-backdrop:focus{
    outline:none; }
  .bp3-overlay-inline .bp3-overlay-backdrop{
    position:absolute; }
.bp3-panel-stack{
  overflow:hidden;
  position:relative; }

.bp3-panel-stack-header{
  -webkit-box-align:center;
      -ms-flex-align:center;
          align-items:center;
  -webkit-box-shadow:0 1px rgba(16, 22, 26, 0.15);
          box-shadow:0 1px rgba(16, 22, 26, 0.15);
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex;
  -ms-flex-negative:0;
      flex-shrink:0;
  height:30px;
  z-index:1; }
  .bp3-dark .bp3-panel-stack-header{
    -webkit-box-shadow:0 1px rgba(255, 255, 255, 0.15);
            box-shadow:0 1px rgba(255, 255, 255, 0.15); }
  .bp3-panel-stack-header > span{
    -webkit-box-align:stretch;
        -ms-flex-align:stretch;
            align-items:stretch;
    display:-webkit-box;
    display:-ms-flexbox;
    display:flex;
    -webkit-box-flex:1;
        -ms-flex:1;
            flex:1; }
  .bp3-panel-stack-header .bp3-heading{
    margin:0 5px; }

.bp3-button.bp3-panel-stack-header-back{
  margin-left:5px;
  padding-left:0;
  white-space:nowrap; }
  .bp3-button.bp3-panel-stack-header-back .bp3-icon{
    margin:0 2px; }

.bp3-panel-stack-view{
  bottom:0;
  left:0;
  position:absolute;
  right:0;
  top:0;
  background-color:#ffffff;
  border-right:1px solid rgba(16, 22, 26, 0.15);
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex;
  -webkit-box-orient:vertical;
  -webkit-box-direction:normal;
      -ms-flex-direction:column;
          flex-direction:column;
  margin-right:-1px;
  overflow-y:auto;
  z-index:1; }
  .bp3-dark .bp3-panel-stack-view{
    background-color:#30404d; }
  .bp3-panel-stack-view:nth-last-child(n + 4){
    display:none; }

.bp3-panel-stack-push .bp3-panel-stack-enter, .bp3-panel-stack-push .bp3-panel-stack-appear{
  -webkit-transform:translateX(100%);
          transform:translateX(100%);
  opacity:0; }

.bp3-panel-stack-push .bp3-panel-stack-enter-active, .bp3-panel-stack-push .bp3-panel-stack-appear-active{
  -webkit-transform:translate(0%);
          transform:translate(0%);
  opacity:1;
  -webkit-transition-delay:0;
          transition-delay:0;
  -webkit-transition-duration:400ms;
          transition-duration:400ms;
  -webkit-transition-property:opacity, -webkit-transform;
  transition-property:opacity, -webkit-transform;
  transition-property:transform, opacity;
  transition-property:transform, opacity, -webkit-transform;
  -webkit-transition-timing-function:ease;
          transition-timing-function:ease; }

.bp3-panel-stack-push .bp3-panel-stack-exit{
  -webkit-transform:translate(0%);
          transform:translate(0%);
  opacity:1; }

.bp3-panel-stack-push .bp3-panel-stack-exit-active{
  -webkit-transform:translateX(-50%);
          transform:translateX(-50%);
  opacity:0;
  -webkit-transition-delay:0;
          transition-delay:0;
  -webkit-transition-duration:400ms;
          transition-duration:400ms;
  -webkit-transition-property:opacity, -webkit-transform;
  transition-property:opacity, -webkit-transform;
  transition-property:transform, opacity;
  transition-property:transform, opacity, -webkit-transform;
  -webkit-transition-timing-function:ease;
          transition-timing-function:ease; }

.bp3-panel-stack-pop .bp3-panel-stack-enter, .bp3-panel-stack-pop .bp3-panel-stack-appear{
  -webkit-transform:translateX(-50%);
          transform:translateX(-50%);
  opacity:0; }

.bp3-panel-stack-pop .bp3-panel-stack-enter-active, .bp3-panel-stack-pop .bp3-panel-stack-appear-active{
  -webkit-transform:translate(0%);
          transform:translate(0%);
  opacity:1;
  -webkit-transition-delay:0;
          transition-delay:0;
  -webkit-transition-duration:400ms;
          transition-duration:400ms;
  -webkit-transition-property:opacity, -webkit-transform;
  transition-property:opacity, -webkit-transform;
  transition-property:transform, opacity;
  transition-property:transform, opacity, -webkit-transform;
  -webkit-transition-timing-function:ease;
          transition-timing-function:ease; }

.bp3-panel-stack-pop .bp3-panel-stack-exit{
  -webkit-transform:translate(0%);
          transform:translate(0%);
  opacity:1; }

.bp3-panel-stack-pop .bp3-panel-stack-exit-active{
  -webkit-transform:translateX(100%);
          transform:translateX(100%);
  opacity:0;
  -webkit-transition-delay:0;
          transition-delay:0;
  -webkit-transition-duration:400ms;
          transition-duration:400ms;
  -webkit-transition-property:opacity, -webkit-transform;
  transition-property:opacity, -webkit-transform;
  transition-property:transform, opacity;
  transition-property:transform, opacity, -webkit-transform;
  -webkit-transition-timing-function:ease;
          transition-timing-function:ease; }
.bp3-popover{
  -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 2px 4px rgba(16, 22, 26, 0.2), 0 8px 24px rgba(16, 22, 26, 0.2);
          box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 2px 4px rgba(16, 22, 26, 0.2), 0 8px 24px rgba(16, 22, 26, 0.2);
  -webkit-transform:scale(1);
          transform:scale(1);
  border-radius:3px;
  display:inline-block;
  z-index:20; }
  .bp3-popover .bp3-popover-arrow{
    height:30px;
    position:absolute;
    width:30px; }
    .bp3-popover .bp3-popover-arrow::before{
      height:20px;
      margin:5px;
      width:20px; }
  .bp3-tether-element-attached-bottom.bp3-tether-target-attached-top > .bp3-popover{
    margin-bottom:17px;
    margin-top:-17px; }
    .bp3-tether-element-attached-bottom.bp3-tether-target-attached-top > .bp3-popover > .bp3-popover-arrow{
      bottom:-11px; }
      .bp3-tether-element-attached-bottom.bp3-tether-target-attached-top > .bp3-popover > .bp3-popover-arrow svg{
        -webkit-transform:rotate(-90deg);
                transform:rotate(-90deg); }
  .bp3-tether-element-attached-left.bp3-tether-target-attached-right > .bp3-popover{
    margin-left:17px; }
    .bp3-tether-element-attached-left.bp3-tether-target-attached-right > .bp3-popover > .bp3-popover-arrow{
      left:-11px; }
      .bp3-tether-element-attached-left.bp3-tether-target-attached-right > .bp3-popover > .bp3-popover-arrow svg{
        -webkit-transform:rotate(0);
                transform:rotate(0); }
  .bp3-tether-element-attached-top.bp3-tether-target-attached-bottom > .bp3-popover{
    margin-top:17px; }
    .bp3-tether-element-attached-top.bp3-tether-target-attached-bottom > .bp3-popover > .bp3-popover-arrow{
      top:-11px; }
      .bp3-tether-element-attached-top.bp3-tether-target-attached-bottom > .bp3-popover > .bp3-popover-arrow svg{
        -webkit-transform:rotate(90deg);
                transform:rotate(90deg); }
  .bp3-tether-element-attached-right.bp3-tether-target-attached-left > .bp3-popover{
    margin-left:-17px;
    margin-right:17px; }
    .bp3-tether-element-attached-right.bp3-tether-target-attached-left > .bp3-popover > .bp3-popover-arrow{
      right:-11px; }
      .bp3-tether-element-attached-right.bp3-tether-target-attached-left > .bp3-popover > .bp3-popover-arrow svg{
        -webkit-transform:rotate(180deg);
                transform:rotate(180deg); }
  .bp3-tether-element-attached-middle > .bp3-popover > .bp3-popover-arrow{
    top:50%;
    -webkit-transform:translateY(-50%);
            transform:translateY(-50%); }
  .bp3-tether-element-attached-center > .bp3-popover > .bp3-popover-arrow{
    right:50%;
    -webkit-transform:translateX(50%);
            transform:translateX(50%); }
  .bp3-tether-element-attached-top.bp3-tether-target-attached-top > .bp3-popover > .bp3-popover-arrow{
    top:-0.3934px; }
  .bp3-tether-element-attached-right.bp3-tether-target-attached-right > .bp3-popover > .bp3-popover-arrow{
    right:-0.3934px; }
  .bp3-tether-element-attached-left.bp3-tether-target-attached-left > .bp3-popover > .bp3-popover-arrow{
    left:-0.3934px; }
  .bp3-tether-element-attached-bottom.bp3-tether-target-attached-bottom > .bp3-popover > .bp3-popover-arrow{
    bottom:-0.3934px; }
  .bp3-tether-element-attached-top.bp3-tether-element-attached-left > .bp3-popover{
    -webkit-transform-origin:top left;
            transform-origin:top left; }
  .bp3-tether-element-attached-top.bp3-tether-element-attached-center > .bp3-popover{
    -webkit-transform-origin:top center;
            transform-origin:top center; }
  .bp3-tether-element-attached-top.bp3-tether-element-attached-right > .bp3-popover{
    -webkit-transform-origin:top right;
            transform-origin:top right; }
  .bp3-tether-element-attached-middle.bp3-tether-element-attached-left > .bp3-popover{
    -webkit-transform-origin:center left;
            transform-origin:center left; }
  .bp3-tether-element-attached-middle.bp3-tether-element-attached-center > .bp3-popover{
    -webkit-transform-origin:center center;
            transform-origin:center center; }
  .bp3-tether-element-attached-middle.bp3-tether-element-attached-right > .bp3-popover{
    -webkit-transform-origin:center right;
            transform-origin:center right; }
  .bp3-tether-element-attached-bottom.bp3-tether-element-attached-left > .bp3-popover{
    -webkit-transform-origin:bottom left;
            transform-origin:bottom left; }
  .bp3-tether-element-attached-bottom.bp3-tether-element-attached-center > .bp3-popover{
    -webkit-transform-origin:bottom center;
            transform-origin:bottom center; }
  .bp3-tether-element-attached-bottom.bp3-tether-element-attached-right > .bp3-popover{
    -webkit-transform-origin:bottom right;
            transform-origin:bottom right; }
  .bp3-popover .bp3-popover-content{
    background:#ffffff;
    color:inherit; }
  .bp3-popover .bp3-popover-arrow::before{
    -webkit-box-shadow:1px 1px 6px rgba(16, 22, 26, 0.2);
            box-shadow:1px 1px 6px rgba(16, 22, 26, 0.2); }
  .bp3-popover .bp3-popover-arrow-border{
    fill:#10161a;
    fill-opacity:0.1; }
  .bp3-popover .bp3-popover-arrow-fill{
    fill:#ffffff; }
  .bp3-popover-enter > .bp3-popover, .bp3-popover-appear > .bp3-popover{
    -webkit-transform:scale(0.3);
            transform:scale(0.3); }
  .bp3-popover-enter-active > .bp3-popover, .bp3-popover-appear-active > .bp3-popover{
    -webkit-transform:scale(1);
            transform:scale(1);
    -webkit-transition-delay:0;
            transition-delay:0;
    -webkit-transition-duration:300ms;
            transition-duration:300ms;
    -webkit-transition-property:-webkit-transform;
    transition-property:-webkit-transform;
    transition-property:transform;
    transition-property:transform, -webkit-transform;
    -webkit-transition-timing-function:cubic-bezier(0.54, 1.12, 0.38, 1.11);
            transition-timing-function:cubic-bezier(0.54, 1.12, 0.38, 1.11); }
  .bp3-popover-exit > .bp3-popover{
    -webkit-transform:scale(1);
            transform:scale(1); }
  .bp3-popover-exit-active > .bp3-popover{
    -webkit-transform:scale(0.3);
            transform:scale(0.3);
    -webkit-transition-delay:0;
            transition-delay:0;
    -webkit-transition-duration:300ms;
            transition-duration:300ms;
    -webkit-transition-property:-webkit-transform;
    transition-property:-webkit-transform;
    transition-property:transform;
    transition-property:transform, -webkit-transform;
    -webkit-transition-timing-function:cubic-bezier(0.54, 1.12, 0.38, 1.11);
            transition-timing-function:cubic-bezier(0.54, 1.12, 0.38, 1.11); }
  .bp3-popover .bp3-popover-content{
    border-radius:3px;
    position:relative; }
  .bp3-popover.bp3-popover-content-sizing .bp3-popover-content{
    max-width:350px;
    padding:20px; }
  .bp3-popover-target + .bp3-overlay .bp3-popover.bp3-popover-content-sizing{
    width:350px; }
  .bp3-popover.bp3-minimal{
    margin:0 !important; }
    .bp3-popover.bp3-minimal .bp3-popover-arrow{
      display:none; }
    .bp3-popover.bp3-minimal.bp3-popover{
      -webkit-transform:scale(1);
              transform:scale(1); }
      .bp3-popover-enter > .bp3-popover.bp3-minimal.bp3-popover, .bp3-popover-appear > .bp3-popover.bp3-minimal.bp3-popover{
        -webkit-transform:scale(1);
                transform:scale(1); }
      .bp3-popover-enter-active > .bp3-popover.bp3-minimal.bp3-popover, .bp3-popover-appear-active > .bp3-popover.bp3-minimal.bp3-popover{
        -webkit-transform:scale(1);
                transform:scale(1);
        -webkit-transition-delay:0;
                transition-delay:0;
        -webkit-transition-duration:100ms;
                transition-duration:100ms;
        -webkit-transition-property:-webkit-transform;
        transition-property:-webkit-transform;
        transition-property:transform;
        transition-property:transform, -webkit-transform;
        -webkit-transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9);
                transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9); }
      .bp3-popover-exit > .bp3-popover.bp3-minimal.bp3-popover{
        -webkit-transform:scale(1);
                transform:scale(1); }
      .bp3-popover-exit-active > .bp3-popover.bp3-minimal.bp3-popover{
        -webkit-transform:scale(1);
                transform:scale(1);
        -webkit-transition-delay:0;
                transition-delay:0;
        -webkit-transition-duration:100ms;
                transition-duration:100ms;
        -webkit-transition-property:-webkit-transform;
        transition-property:-webkit-transform;
        transition-property:transform;
        transition-property:transform, -webkit-transform;
        -webkit-transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9);
                transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9); }
  .bp3-popover.bp3-dark,
  .bp3-dark .bp3-popover{
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 2px 4px rgba(16, 22, 26, 0.4), 0 8px 24px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 2px 4px rgba(16, 22, 26, 0.4), 0 8px 24px rgba(16, 22, 26, 0.4); }
    .bp3-popover.bp3-dark .bp3-popover-content,
    .bp3-dark .bp3-popover .bp3-popover-content{
      background:#30404d;
      color:inherit; }
    .bp3-popover.bp3-dark .bp3-popover-arrow::before,
    .bp3-dark .bp3-popover .bp3-popover-arrow::before{
      -webkit-box-shadow:1px 1px 6px rgba(16, 22, 26, 0.4);
              box-shadow:1px 1px 6px rgba(16, 22, 26, 0.4); }
    .bp3-popover.bp3-dark .bp3-popover-arrow-border,
    .bp3-dark .bp3-popover .bp3-popover-arrow-border{
      fill:#10161a;
      fill-opacity:0.2; }
    .bp3-popover.bp3-dark .bp3-popover-arrow-fill,
    .bp3-dark .bp3-popover .bp3-popover-arrow-fill{
      fill:#30404d; }

.bp3-popover-arrow::before{
  border-radius:2px;
  content:"";
  display:block;
  position:absolute;
  -webkit-transform:rotate(45deg);
          transform:rotate(45deg); }

.bp3-tether-pinned .bp3-popover-arrow{
  display:none; }

.bp3-popover-backdrop{
  background:rgba(255, 255, 255, 0); }

.bp3-transition-container{
  opacity:1;
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex;
  z-index:20; }
  .bp3-transition-container.bp3-popover-enter, .bp3-transition-container.bp3-popover-appear{
    opacity:0; }
  .bp3-transition-container.bp3-popover-enter-active, .bp3-transition-container.bp3-popover-appear-active{
    opacity:1;
    -webkit-transition-delay:0;
            transition-delay:0;
    -webkit-transition-duration:100ms;
            transition-duration:100ms;
    -webkit-transition-property:opacity;
    transition-property:opacity;
    -webkit-transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9);
            transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9); }
  .bp3-transition-container.bp3-popover-exit{
    opacity:1; }
  .bp3-transition-container.bp3-popover-exit-active{
    opacity:0;
    -webkit-transition-delay:0;
            transition-delay:0;
    -webkit-transition-duration:100ms;
            transition-duration:100ms;
    -webkit-transition-property:opacity;
    transition-property:opacity;
    -webkit-transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9);
            transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9); }
  .bp3-transition-container:focus{
    outline:none; }
  .bp3-transition-container.bp3-popover-leave .bp3-popover-content{
    pointer-events:none; }
  .bp3-transition-container[data-x-out-of-boundaries]{
    display:none; }

span.bp3-popover-target{
  display:inline-block; }

.bp3-popover-wrapper.bp3-fill{
  width:100%; }

.bp3-portal{
  left:0;
  position:absolute;
  right:0;
  top:0; }
@-webkit-keyframes linear-progress-bar-stripes{
  from{
    background-position:0 0; }
  to{
    background-position:30px 0; } }
@keyframes linear-progress-bar-stripes{
  from{
    background-position:0 0; }
  to{
    background-position:30px 0; } }

.bp3-progress-bar{
  background:rgba(92, 112, 128, 0.2);
  border-radius:40px;
  display:block;
  height:8px;
  overflow:hidden;
  position:relative;
  width:100%; }
  .bp3-progress-bar .bp3-progress-meter{
    background:linear-gradient(-45deg, rgba(255, 255, 255, 0.2) 25%, transparent 25%, transparent 50%, rgba(255, 255, 255, 0.2) 50%, rgba(255, 255, 255, 0.2) 75%, transparent 75%);
    background-color:rgba(92, 112, 128, 0.8);
    background-size:30px 30px;
    border-radius:40px;
    height:100%;
    position:absolute;
    -webkit-transition:width 200ms cubic-bezier(0.4, 1, 0.75, 0.9);
    transition:width 200ms cubic-bezier(0.4, 1, 0.75, 0.9);
    width:100%; }
  .bp3-progress-bar:not(.bp3-no-animation):not(.bp3-no-stripes) .bp3-progress-meter{
    animation:linear-progress-bar-stripes 300ms linear infinite reverse; }
  .bp3-progress-bar.bp3-no-stripes .bp3-progress-meter{
    background-image:none; }

.bp3-dark .bp3-progress-bar{
  background:rgba(16, 22, 26, 0.5); }
  .bp3-dark .bp3-progress-bar .bp3-progress-meter{
    background-color:#8a9ba8; }

.bp3-progress-bar.bp3-intent-primary .bp3-progress-meter{
  background-color:#137cbd; }

.bp3-progress-bar.bp3-intent-success .bp3-progress-meter{
  background-color:#0f9960; }

.bp3-progress-bar.bp3-intent-warning .bp3-progress-meter{
  background-color:#d9822b; }

.bp3-progress-bar.bp3-intent-danger .bp3-progress-meter{
  background-color:#db3737; }
@-webkit-keyframes skeleton-glow{
  from{
    background:rgba(206, 217, 224, 0.2);
    border-color:rgba(206, 217, 224, 0.2); }
  to{
    background:rgba(92, 112, 128, 0.2);
    border-color:rgba(92, 112, 128, 0.2); } }
@keyframes skeleton-glow{
  from{
    background:rgba(206, 217, 224, 0.2);
    border-color:rgba(206, 217, 224, 0.2); }
  to{
    background:rgba(92, 112, 128, 0.2);
    border-color:rgba(92, 112, 128, 0.2); } }
.bp3-skeleton{
  -webkit-animation:1000ms linear infinite alternate skeleton-glow;
          animation:1000ms linear infinite alternate skeleton-glow;
  background:rgba(206, 217, 224, 0.2);
  background-clip:padding-box !important;
  border-color:rgba(206, 217, 224, 0.2) !important;
  border-radius:2px;
  -webkit-box-shadow:none !important;
          box-shadow:none !important;
  color:transparent !important;
  cursor:default;
  pointer-events:none;
  -webkit-user-select:none;
     -moz-user-select:none;
      -ms-user-select:none;
          user-select:none; }
  .bp3-skeleton::before, .bp3-skeleton::after,
  .bp3-skeleton *{
    visibility:hidden !important; }
.bp3-slider{
  height:40px;
  min-width:150px;
  width:100%;
  cursor:default;
  outline:none;
  position:relative;
  -webkit-user-select:none;
     -moz-user-select:none;
      -ms-user-select:none;
          user-select:none; }
  .bp3-slider:hover{
    cursor:pointer; }
  .bp3-slider:active{
    cursor:-webkit-grabbing;
    cursor:grabbing; }
  .bp3-slider.bp3-disabled{
    cursor:not-allowed;
    opacity:0.5; }
  .bp3-slider.bp3-slider-unlabeled{
    height:16px; }

.bp3-slider-track,
.bp3-slider-progress{
  height:6px;
  left:0;
  right:0;
  top:5px;
  position:absolute; }

.bp3-slider-track{
  border-radius:3px;
  overflow:hidden; }

.bp3-slider-progress{
  background:rgba(92, 112, 128, 0.2); }
  .bp3-dark .bp3-slider-progress{
    background:rgba(16, 22, 26, 0.5); }
  .bp3-slider-progress.bp3-intent-primary{
    background-color:#137cbd; }
  .bp3-slider-progress.bp3-intent-success{
    background-color:#0f9960; }
  .bp3-slider-progress.bp3-intent-warning{
    background-color:#d9822b; }
  .bp3-slider-progress.bp3-intent-danger{
    background-color:#db3737; }

.bp3-slider-handle{
  background-color:#f5f8fa;
  background-image:-webkit-gradient(linear, left top, left bottom, from(rgba(255, 255, 255, 0.8)), to(rgba(255, 255, 255, 0)));
  background-image:linear-gradient(to bottom, rgba(255, 255, 255, 0.8), rgba(255, 255, 255, 0));
  -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 -1px 0 rgba(16, 22, 26, 0.1);
          box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 -1px 0 rgba(16, 22, 26, 0.1);
  color:#182026;
  border-radius:3px;
  -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 1px 1px rgba(16, 22, 26, 0.2);
          box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 1px 1px rgba(16, 22, 26, 0.2);
  cursor:pointer;
  height:16px;
  left:0;
  position:absolute;
  top:0;
  width:16px; }
  .bp3-slider-handle:hover{
    background-clip:padding-box;
    background-color:#ebf1f5;
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 -1px 0 rgba(16, 22, 26, 0.1);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 -1px 0 rgba(16, 22, 26, 0.1); }
  .bp3-slider-handle:active, .bp3-slider-handle.bp3-active{
    background-color:#d8e1e8;
    background-image:none;
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 1px 2px rgba(16, 22, 26, 0.2);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 1px 2px rgba(16, 22, 26, 0.2); }
  .bp3-slider-handle:disabled, .bp3-slider-handle.bp3-disabled{
    background-color:rgba(206, 217, 224, 0.5);
    background-image:none;
    -webkit-box-shadow:none;
            box-shadow:none;
    color:rgba(92, 112, 128, 0.6);
    cursor:not-allowed;
    outline:none; }
    .bp3-slider-handle:disabled.bp3-active, .bp3-slider-handle:disabled.bp3-active:hover, .bp3-slider-handle.bp3-disabled.bp3-active, .bp3-slider-handle.bp3-disabled.bp3-active:hover{
      background:rgba(206, 217, 224, 0.7); }
  .bp3-slider-handle:focus{
    z-index:1; }
  .bp3-slider-handle:hover{
    background-clip:padding-box;
    background-color:#ebf1f5;
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 -1px 0 rgba(16, 22, 26, 0.1);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 -1px 0 rgba(16, 22, 26, 0.1);
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 1px 1px rgba(16, 22, 26, 0.2);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 1px 1px rgba(16, 22, 26, 0.2);
    cursor:-webkit-grab;
    cursor:grab;
    z-index:2; }
  .bp3-slider-handle.bp3-active{
    background-color:#d8e1e8;
    background-image:none;
    -webkit-box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 1px 2px rgba(16, 22, 26, 0.2);
            box-shadow:inset 0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 1px 2px rgba(16, 22, 26, 0.2);
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 1px 1px rgba(16, 22, 26, 0.1);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), inset 0 1px 1px rgba(16, 22, 26, 0.1);
    cursor:-webkit-grabbing;
    cursor:grabbing; }
  .bp3-disabled .bp3-slider-handle{
    background:#bfccd6;
    -webkit-box-shadow:none;
            box-shadow:none;
    pointer-events:none; }
  .bp3-dark .bp3-slider-handle{
    background-color:#394b59;
    background-image:-webkit-gradient(linear, left top, left bottom, from(rgba(255, 255, 255, 0.05)), to(rgba(255, 255, 255, 0)));
    background-image:linear-gradient(to bottom, rgba(255, 255, 255, 0.05), rgba(255, 255, 255, 0));
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4);
    color:#f5f8fa; }
    .bp3-dark .bp3-slider-handle:hover, .bp3-dark .bp3-slider-handle:active, .bp3-dark .bp3-slider-handle.bp3-active{
      color:#f5f8fa; }
    .bp3-dark .bp3-slider-handle:hover{
      background-color:#30404d;
      -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4);
              box-shadow:0 0 0 1px rgba(16, 22, 26, 0.4); }
    .bp3-dark .bp3-slider-handle:active, .bp3-dark .bp3-slider-handle.bp3-active{
      background-color:#202b33;
      background-image:none;
      -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.6), inset 0 1px 2px rgba(16, 22, 26, 0.2);
              box-shadow:0 0 0 1px rgba(16, 22, 26, 0.6), inset 0 1px 2px rgba(16, 22, 26, 0.2); }
    .bp3-dark .bp3-slider-handle:disabled, .bp3-dark .bp3-slider-handle.bp3-disabled{
      background-color:rgba(57, 75, 89, 0.5);
      background-image:none;
      -webkit-box-shadow:none;
              box-shadow:none;
      color:rgba(167, 182, 194, 0.6); }
      .bp3-dark .bp3-slider-handle:disabled.bp3-active, .bp3-dark .bp3-slider-handle.bp3-disabled.bp3-active{
        background:rgba(57, 75, 89, 0.7); }
    .bp3-dark .bp3-slider-handle .bp3-button-spinner .bp3-spinner-head{
      background:rgba(16, 22, 26, 0.5);
      stroke:#8a9ba8; }
    .bp3-dark .bp3-slider-handle, .bp3-dark .bp3-slider-handle:hover{
      background-color:#394b59; }
    .bp3-dark .bp3-slider-handle.bp3-active{
      background-color:#293742; }
  .bp3-dark .bp3-disabled .bp3-slider-handle{
    background:#5c7080;
    border-color:#5c7080;
    -webkit-box-shadow:none;
            box-shadow:none; }
  .bp3-slider-handle .bp3-slider-label{
    background:#394b59;
    border-radius:3px;
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 2px 4px rgba(16, 22, 26, 0.2), 0 8px 24px rgba(16, 22, 26, 0.2);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 2px 4px rgba(16, 22, 26, 0.2), 0 8px 24px rgba(16, 22, 26, 0.2);
    color:#f5f8fa;
    margin-left:8px; }
    .bp3-dark .bp3-slider-handle .bp3-slider-label{
      background:#e1e8ed;
      -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 2px 4px rgba(16, 22, 26, 0.4), 0 8px 24px rgba(16, 22, 26, 0.4);
              box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 2px 4px rgba(16, 22, 26, 0.4), 0 8px 24px rgba(16, 22, 26, 0.4);
      color:#394b59; }
    .bp3-disabled .bp3-slider-handle .bp3-slider-label{
      -webkit-box-shadow:none;
              box-shadow:none; }
  .bp3-slider-handle.bp3-start, .bp3-slider-handle.bp3-end{
    width:8px; }
  .bp3-slider-handle.bp3-start{
    border-bottom-right-radius:0;
    border-top-right-radius:0; }
  .bp3-slider-handle.bp3-end{
    border-bottom-left-radius:0;
    border-top-left-radius:0;
    margin-left:8px; }
    .bp3-slider-handle.bp3-end .bp3-slider-label{
      margin-left:0; }

.bp3-slider-label{
  -webkit-transform:translate(-50%, 20px);
          transform:translate(-50%, 20px);
  display:inline-block;
  font-size:12px;
  line-height:1;
  padding:2px 5px;
  position:absolute;
  vertical-align:top; }

.bp3-slider.bp3-vertical{
  height:150px;
  min-width:40px;
  width:40px; }
  .bp3-slider.bp3-vertical .bp3-slider-track,
  .bp3-slider.bp3-vertical .bp3-slider-progress{
    bottom:0;
    height:auto;
    left:5px;
    top:0;
    width:6px; }
  .bp3-slider.bp3-vertical .bp3-slider-progress{
    top:auto; }
  .bp3-slider.bp3-vertical .bp3-slider-label{
    -webkit-transform:translate(20px, 50%);
            transform:translate(20px, 50%); }
  .bp3-slider.bp3-vertical .bp3-slider-handle{
    top:auto; }
    .bp3-slider.bp3-vertical .bp3-slider-handle .bp3-slider-label{
      margin-left:0;
      margin-top:-8px; }
    .bp3-slider.bp3-vertical .bp3-slider-handle.bp3-end, .bp3-slider.bp3-vertical .bp3-slider-handle.bp3-start{
      height:8px;
      margin-left:0;
      width:16px; }
    .bp3-slider.bp3-vertical .bp3-slider-handle.bp3-start{
      border-bottom-right-radius:3px;
      border-top-left-radius:0; }
      .bp3-slider.bp3-vertical .bp3-slider-handle.bp3-start .bp3-slider-label{
        -webkit-transform:translate(20px);
                transform:translate(20px); }
    .bp3-slider.bp3-vertical .bp3-slider-handle.bp3-end{
      border-bottom-left-radius:0;
      border-bottom-right-radius:0;
      border-top-left-radius:3px;
      margin-bottom:8px; }

@-webkit-keyframes pt-spinner-animation{
  from{
    -webkit-transform:rotate(0deg);
            transform:rotate(0deg); }
  to{
    -webkit-transform:rotate(360deg);
            transform:rotate(360deg); } }

@keyframes pt-spinner-animation{
  from{
    -webkit-transform:rotate(0deg);
            transform:rotate(0deg); }
  to{
    -webkit-transform:rotate(360deg);
            transform:rotate(360deg); } }

.bp3-spinner{
  -webkit-box-align:center;
      -ms-flex-align:center;
          align-items:center;
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex;
  -webkit-box-pack:center;
      -ms-flex-pack:center;
          justify-content:center;
  overflow:visible;
  vertical-align:middle; }
  .bp3-spinner svg{
    display:block; }
  .bp3-spinner path{
    fill-opacity:0; }
  .bp3-spinner .bp3-spinner-head{
    stroke:rgba(92, 112, 128, 0.8);
    stroke-linecap:round;
    -webkit-transform-origin:center;
            transform-origin:center;
    -webkit-transition:stroke-dashoffset 200ms cubic-bezier(0.4, 1, 0.75, 0.9);
    transition:stroke-dashoffset 200ms cubic-bezier(0.4, 1, 0.75, 0.9); }
  .bp3-spinner .bp3-spinner-track{
    stroke:rgba(92, 112, 128, 0.2); }

.bp3-spinner-animation{
  -webkit-animation:pt-spinner-animation 500ms linear infinite;
          animation:pt-spinner-animation 500ms linear infinite; }
  .bp3-no-spin > .bp3-spinner-animation{
    -webkit-animation:none;
            animation:none; }

.bp3-dark .bp3-spinner .bp3-spinner-head{
  stroke:#8a9ba8; }

.bp3-dark .bp3-spinner .bp3-spinner-track{
  stroke:rgba(16, 22, 26, 0.5); }

.bp3-spinner.bp3-intent-primary .bp3-spinner-head{
  stroke:#137cbd; }

.bp3-spinner.bp3-intent-success .bp3-spinner-head{
  stroke:#0f9960; }

.bp3-spinner.bp3-intent-warning .bp3-spinner-head{
  stroke:#d9822b; }

.bp3-spinner.bp3-intent-danger .bp3-spinner-head{
  stroke:#db3737; }
.bp3-tabs.bp3-vertical{
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex; }
  .bp3-tabs.bp3-vertical > .bp3-tab-list{
    -webkit-box-align:start;
        -ms-flex-align:start;
            align-items:flex-start;
    -webkit-box-orient:vertical;
    -webkit-box-direction:normal;
        -ms-flex-direction:column;
            flex-direction:column; }
    .bp3-tabs.bp3-vertical > .bp3-tab-list .bp3-tab{
      border-radius:3px;
      padding:0 10px;
      width:100%; }
      .bp3-tabs.bp3-vertical > .bp3-tab-list .bp3-tab[aria-selected="true"]{
        background-color:rgba(19, 124, 189, 0.2);
        -webkit-box-shadow:none;
                box-shadow:none; }
    .bp3-tabs.bp3-vertical > .bp3-tab-list .bp3-tab-indicator-wrapper .bp3-tab-indicator{
      background-color:rgba(19, 124, 189, 0.2);
      border-radius:3px;
      bottom:0;
      height:auto;
      left:0;
      right:0;
      top:0; }
  .bp3-tabs.bp3-vertical > .bp3-tab-panel{
    margin-top:0;
    padding-left:20px; }

.bp3-tab-list{
  -webkit-box-align:end;
      -ms-flex-align:end;
          align-items:flex-end;
  border:none;
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex;
  -webkit-box-flex:0;
      -ms-flex:0 0 auto;
          flex:0 0 auto;
  list-style:none;
  margin:0;
  padding:0;
  position:relative; }
  .bp3-tab-list > *:not(:last-child){
    margin-right:20px; }

.bp3-tab{
  overflow:hidden;
  text-overflow:ellipsis;
  white-space:nowrap;
  word-wrap:normal;
  color:#182026;
  cursor:pointer;
  -webkit-box-flex:0;
      -ms-flex:0 0 auto;
          flex:0 0 auto;
  font-size:14px;
  line-height:30px;
  max-width:100%;
  position:relative;
  vertical-align:top; }
  .bp3-tab a{
    color:inherit;
    display:block;
    text-decoration:none; }
  .bp3-tab-indicator-wrapper ~ .bp3-tab{
    background-color:transparent !important;
    -webkit-box-shadow:none !important;
            box-shadow:none !important; }
  .bp3-tab[aria-disabled="true"]{
    color:rgba(92, 112, 128, 0.6);
    cursor:not-allowed; }
  .bp3-tab[aria-selected="true"]{
    border-radius:0;
    -webkit-box-shadow:inset 0 -3px 0 #106ba3;
            box-shadow:inset 0 -3px 0 #106ba3; }
  .bp3-tab[aria-selected="true"], .bp3-tab:not([aria-disabled="true"]):hover{
    color:#106ba3; }
  .bp3-tab:focus{
    -moz-outline-radius:0; }
  .bp3-large > .bp3-tab{
    font-size:16px;
    line-height:40px; }

.bp3-tab-panel{
  margin-top:20px; }
  .bp3-tab-panel[aria-hidden="true"]{
    display:none; }

.bp3-tab-indicator-wrapper{
  left:0;
  pointer-events:none;
  position:absolute;
  top:0;
  -webkit-transform:translateX(0), translateY(0);
          transform:translateX(0), translateY(0);
  -webkit-transition:height, width, -webkit-transform;
  transition:height, width, -webkit-transform;
  transition:height, transform, width;
  transition:height, transform, width, -webkit-transform;
  -webkit-transition-duration:200ms;
          transition-duration:200ms;
  -webkit-transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9);
          transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9); }
  .bp3-tab-indicator-wrapper .bp3-tab-indicator{
    background-color:#106ba3;
    bottom:0;
    height:3px;
    left:0;
    position:absolute;
    right:0; }
  .bp3-tab-indicator-wrapper.bp3-no-animation{
    -webkit-transition:none;
    transition:none; }

.bp3-dark .bp3-tab{
  color:#f5f8fa; }
  .bp3-dark .bp3-tab[aria-disabled="true"]{
    color:rgba(167, 182, 194, 0.6); }
  .bp3-dark .bp3-tab[aria-selected="true"]{
    -webkit-box-shadow:inset 0 -3px 0 #48aff0;
            box-shadow:inset 0 -3px 0 #48aff0; }
  .bp3-dark .bp3-tab[aria-selected="true"], .bp3-dark .bp3-tab:not([aria-disabled="true"]):hover{
    color:#48aff0; }

.bp3-dark .bp3-tab-indicator{
  background-color:#48aff0; }

.bp3-flex-expander{
  -webkit-box-flex:1;
      -ms-flex:1 1;
          flex:1 1; }
.bp3-tag{
  display:-webkit-inline-box;
  display:-ms-inline-flexbox;
  display:inline-flex;
  -webkit-box-orient:horizontal;
  -webkit-box-direction:normal;
      -ms-flex-direction:row;
          flex-direction:row;
  -webkit-box-align:center;
      -ms-flex-align:center;
          align-items:center;
  background-color:#5c7080;
  border:none;
  border-radius:3px;
  -webkit-box-shadow:none;
          box-shadow:none;
  color:#f5f8fa;
  font-size:12px;
  line-height:16px;
  max-width:100%;
  min-height:20px;
  min-width:20px;
  padding:2px 6px;
  position:relative; }
  .bp3-tag.bp3-interactive{
    cursor:pointer; }
    .bp3-tag.bp3-interactive:hover{
      background-color:rgba(92, 112, 128, 0.85); }
    .bp3-tag.bp3-interactive.bp3-active, .bp3-tag.bp3-interactive:active{
      background-color:rgba(92, 112, 128, 0.7); }
  .bp3-tag > *{
    -webkit-box-flex:0;
        -ms-flex-positive:0;
            flex-grow:0;
    -ms-flex-negative:0;
        flex-shrink:0; }
  .bp3-tag > .bp3-fill{
    -webkit-box-flex:1;
        -ms-flex-positive:1;
            flex-grow:1;
    -ms-flex-negative:1;
        flex-shrink:1; }
  .bp3-tag::before,
  .bp3-tag > *{
    margin-right:4px; }
  .bp3-tag:empty::before,
  .bp3-tag > :last-child{
    margin-right:0; }
  .bp3-tag:focus{
    outline:rgba(19, 124, 189, 0.6) auto 2px;
    outline-offset:0;
    -moz-outline-radius:6px; }
  .bp3-tag.bp3-round{
    border-radius:30px;
    padding-left:8px;
    padding-right:8px; }
  .bp3-dark .bp3-tag{
    background-color:#bfccd6;
    color:#182026; }
    .bp3-dark .bp3-tag.bp3-interactive{
      cursor:pointer; }
      .bp3-dark .bp3-tag.bp3-interactive:hover{
        background-color:rgba(191, 204, 214, 0.85); }
      .bp3-dark .bp3-tag.bp3-interactive.bp3-active, .bp3-dark .bp3-tag.bp3-interactive:active{
        background-color:rgba(191, 204, 214, 0.7); }
    .bp3-dark .bp3-tag > .bp3-icon, .bp3-dark .bp3-tag .bp3-icon-standard, .bp3-dark .bp3-tag .bp3-icon-large{
      fill:currentColor; }
  .bp3-tag > .bp3-icon, .bp3-tag .bp3-icon-standard, .bp3-tag .bp3-icon-large{
    fill:#ffffff; }
  .bp3-tag.bp3-large,
  .bp3-large .bp3-tag{
    font-size:14px;
    line-height:20px;
    min-height:30px;
    min-width:30px;
    padding:5px 10px; }
    .bp3-tag.bp3-large::before,
    .bp3-tag.bp3-large > *,
    .bp3-large .bp3-tag::before,
    .bp3-large .bp3-tag > *{
      margin-right:7px; }
    .bp3-tag.bp3-large:empty::before,
    .bp3-tag.bp3-large > :last-child,
    .bp3-large .bp3-tag:empty::before,
    .bp3-large .bp3-tag > :last-child{
      margin-right:0; }
    .bp3-tag.bp3-large.bp3-round,
    .bp3-large .bp3-tag.bp3-round{
      padding-left:12px;
      padding-right:12px; }
  .bp3-tag.bp3-intent-primary{
    background:#137cbd;
    color:#ffffff; }
    .bp3-tag.bp3-intent-primary.bp3-interactive{
      cursor:pointer; }
      .bp3-tag.bp3-intent-primary.bp3-interactive:hover{
        background-color:rgba(19, 124, 189, 0.85); }
      .bp3-tag.bp3-intent-primary.bp3-interactive.bp3-active, .bp3-tag.bp3-intent-primary.bp3-interactive:active{
        background-color:rgba(19, 124, 189, 0.7); }
  .bp3-tag.bp3-intent-success{
    background:#0f9960;
    color:#ffffff; }
    .bp3-tag.bp3-intent-success.bp3-interactive{
      cursor:pointer; }
      .bp3-tag.bp3-intent-success.bp3-interactive:hover{
        background-color:rgba(15, 153, 96, 0.85); }
      .bp3-tag.bp3-intent-success.bp3-interactive.bp3-active, .bp3-tag.bp3-intent-success.bp3-interactive:active{
        background-color:rgba(15, 153, 96, 0.7); }
  .bp3-tag.bp3-intent-warning{
    background:#d9822b;
    color:#ffffff; }
    .bp3-tag.bp3-intent-warning.bp3-interactive{
      cursor:pointer; }
      .bp3-tag.bp3-intent-warning.bp3-interactive:hover{
        background-color:rgba(217, 130, 43, 0.85); }
      .bp3-tag.bp3-intent-warning.bp3-interactive.bp3-active, .bp3-tag.bp3-intent-warning.bp3-interactive:active{
        background-color:rgba(217, 130, 43, 0.7); }
  .bp3-tag.bp3-intent-danger{
    background:#db3737;
    color:#ffffff; }
    .bp3-tag.bp3-intent-danger.bp3-interactive{
      cursor:pointer; }
      .bp3-tag.bp3-intent-danger.bp3-interactive:hover{
        background-color:rgba(219, 55, 55, 0.85); }
      .bp3-tag.bp3-intent-danger.bp3-interactive.bp3-active, .bp3-tag.bp3-intent-danger.bp3-interactive:active{
        background-color:rgba(219, 55, 55, 0.7); }
  .bp3-tag.bp3-fill{
    display:-webkit-box;
    display:-ms-flexbox;
    display:flex;
    width:100%; }
  .bp3-tag.bp3-minimal > .bp3-icon, .bp3-tag.bp3-minimal .bp3-icon-standard, .bp3-tag.bp3-minimal .bp3-icon-large{
    fill:#5c7080; }
  .bp3-tag.bp3-minimal:not([class*="bp3-intent-"]){
    background-color:rgba(138, 155, 168, 0.2);
    color:#182026; }
    .bp3-tag.bp3-minimal:not([class*="bp3-intent-"]).bp3-interactive{
      cursor:pointer; }
      .bp3-tag.bp3-minimal:not([class*="bp3-intent-"]).bp3-interactive:hover{
        background-color:rgba(92, 112, 128, 0.3); }
      .bp3-tag.bp3-minimal:not([class*="bp3-intent-"]).bp3-interactive.bp3-active, .bp3-tag.bp3-minimal:not([class*="bp3-intent-"]).bp3-interactive:active{
        background-color:rgba(92, 112, 128, 0.4); }
    .bp3-dark .bp3-tag.bp3-minimal:not([class*="bp3-intent-"]){
      color:#f5f8fa; }
      .bp3-dark .bp3-tag.bp3-minimal:not([class*="bp3-intent-"]).bp3-interactive{
        cursor:pointer; }
        .bp3-dark .bp3-tag.bp3-minimal:not([class*="bp3-intent-"]).bp3-interactive:hover{
          background-color:rgba(191, 204, 214, 0.3); }
        .bp3-dark .bp3-tag.bp3-minimal:not([class*="bp3-intent-"]).bp3-interactive.bp3-active, .bp3-dark .bp3-tag.bp3-minimal:not([class*="bp3-intent-"]).bp3-interactive:active{
          background-color:rgba(191, 204, 214, 0.4); }
      .bp3-dark .bp3-tag.bp3-minimal:not([class*="bp3-intent-"]) > .bp3-icon, .bp3-dark .bp3-tag.bp3-minimal:not([class*="bp3-intent-"]) .bp3-icon-standard, .bp3-dark .bp3-tag.bp3-minimal:not([class*="bp3-intent-"]) .bp3-icon-large{
        fill:#a7b6c2; }
  .bp3-tag.bp3-minimal.bp3-intent-primary{
    background-color:rgba(19, 124, 189, 0.15);
    color:#106ba3; }
    .bp3-tag.bp3-minimal.bp3-intent-primary.bp3-interactive{
      cursor:pointer; }
      .bp3-tag.bp3-minimal.bp3-intent-primary.bp3-interactive:hover{
        background-color:rgba(19, 124, 189, 0.25); }
      .bp3-tag.bp3-minimal.bp3-intent-primary.bp3-interactive.bp3-active, .bp3-tag.bp3-minimal.bp3-intent-primary.bp3-interactive:active{
        background-color:rgba(19, 124, 189, 0.35); }
    .bp3-tag.bp3-minimal.bp3-intent-primary > .bp3-icon, .bp3-tag.bp3-minimal.bp3-intent-primary .bp3-icon-standard, .bp3-tag.bp3-minimal.bp3-intent-primary .bp3-icon-large{
      fill:#137cbd; }
    .bp3-dark .bp3-tag.bp3-minimal.bp3-intent-primary{
      background-color:rgba(19, 124, 189, 0.25);
      color:#48aff0; }
      .bp3-dark .bp3-tag.bp3-minimal.bp3-intent-primary.bp3-interactive{
        cursor:pointer; }
        .bp3-dark .bp3-tag.bp3-minimal.bp3-intent-primary.bp3-interactive:hover{
          background-color:rgba(19, 124, 189, 0.35); }
        .bp3-dark .bp3-tag.bp3-minimal.bp3-intent-primary.bp3-interactive.bp3-active, .bp3-dark .bp3-tag.bp3-minimal.bp3-intent-primary.bp3-interactive:active{
          background-color:rgba(19, 124, 189, 0.45); }
  .bp3-tag.bp3-minimal.bp3-intent-success{
    background-color:rgba(15, 153, 96, 0.15);
    color:#0d8050; }
    .bp3-tag.bp3-minimal.bp3-intent-success.bp3-interactive{
      cursor:pointer; }
      .bp3-tag.bp3-minimal.bp3-intent-success.bp3-interactive:hover{
        background-color:rgba(15, 153, 96, 0.25); }
      .bp3-tag.bp3-minimal.bp3-intent-success.bp3-interactive.bp3-active, .bp3-tag.bp3-minimal.bp3-intent-success.bp3-interactive:active{
        background-color:rgba(15, 153, 96, 0.35); }
    .bp3-tag.bp3-minimal.bp3-intent-success > .bp3-icon, .bp3-tag.bp3-minimal.bp3-intent-success .bp3-icon-standard, .bp3-tag.bp3-minimal.bp3-intent-success .bp3-icon-large{
      fill:#0f9960; }
    .bp3-dark .bp3-tag.bp3-minimal.bp3-intent-success{
      background-color:rgba(15, 153, 96, 0.25);
      color:#3dcc91; }
      .bp3-dark .bp3-tag.bp3-minimal.bp3-intent-success.bp3-interactive{
        cursor:pointer; }
        .bp3-dark .bp3-tag.bp3-minimal.bp3-intent-success.bp3-interactive:hover{
          background-color:rgba(15, 153, 96, 0.35); }
        .bp3-dark .bp3-tag.bp3-minimal.bp3-intent-success.bp3-interactive.bp3-active, .bp3-dark .bp3-tag.bp3-minimal.bp3-intent-success.bp3-interactive:active{
          background-color:rgba(15, 153, 96, 0.45); }
  .bp3-tag.bp3-minimal.bp3-intent-warning{
    background-color:rgba(217, 130, 43, 0.15);
    color:#bf7326; }
    .bp3-tag.bp3-minimal.bp3-intent-warning.bp3-interactive{
      cursor:pointer; }
      .bp3-tag.bp3-minimal.bp3-intent-warning.bp3-interactive:hover{
        background-color:rgba(217, 130, 43, 0.25); }
      .bp3-tag.bp3-minimal.bp3-intent-warning.bp3-interactive.bp3-active, .bp3-tag.bp3-minimal.bp3-intent-warning.bp3-interactive:active{
        background-color:rgba(217, 130, 43, 0.35); }
    .bp3-tag.bp3-minimal.bp3-intent-warning > .bp3-icon, .bp3-tag.bp3-minimal.bp3-intent-warning .bp3-icon-standard, .bp3-tag.bp3-minimal.bp3-intent-warning .bp3-icon-large{
      fill:#d9822b; }
    .bp3-dark .bp3-tag.bp3-minimal.bp3-intent-warning{
      background-color:rgba(217, 130, 43, 0.25);
      color:#ffb366; }
      .bp3-dark .bp3-tag.bp3-minimal.bp3-intent-warning.bp3-interactive{
        cursor:pointer; }
        .bp3-dark .bp3-tag.bp3-minimal.bp3-intent-warning.bp3-interactive:hover{
          background-color:rgba(217, 130, 43, 0.35); }
        .bp3-dark .bp3-tag.bp3-minimal.bp3-intent-warning.bp3-interactive.bp3-active, .bp3-dark .bp3-tag.bp3-minimal.bp3-intent-warning.bp3-interactive:active{
          background-color:rgba(217, 130, 43, 0.45); }
  .bp3-tag.bp3-minimal.bp3-intent-danger{
    background-color:rgba(219, 55, 55, 0.15);
    color:#c23030; }
    .bp3-tag.bp3-minimal.bp3-intent-danger.bp3-interactive{
      cursor:pointer; }
      .bp3-tag.bp3-minimal.bp3-intent-danger.bp3-interactive:hover{
        background-color:rgba(219, 55, 55, 0.25); }
      .bp3-tag.bp3-minimal.bp3-intent-danger.bp3-interactive.bp3-active, .bp3-tag.bp3-minimal.bp3-intent-danger.bp3-interactive:active{
        background-color:rgba(219, 55, 55, 0.35); }
    .bp3-tag.bp3-minimal.bp3-intent-danger > .bp3-icon, .bp3-tag.bp3-minimal.bp3-intent-danger .bp3-icon-standard, .bp3-tag.bp3-minimal.bp3-intent-danger .bp3-icon-large{
      fill:#db3737; }
    .bp3-dark .bp3-tag.bp3-minimal.bp3-intent-danger{
      background-color:rgba(219, 55, 55, 0.25);
      color:#ff7373; }
      .bp3-dark .bp3-tag.bp3-minimal.bp3-intent-danger.bp3-interactive{
        cursor:pointer; }
        .bp3-dark .bp3-tag.bp3-minimal.bp3-intent-danger.bp3-interactive:hover{
          background-color:rgba(219, 55, 55, 0.35); }
        .bp3-dark .bp3-tag.bp3-minimal.bp3-intent-danger.bp3-interactive.bp3-active, .bp3-dark .bp3-tag.bp3-minimal.bp3-intent-danger.bp3-interactive:active{
          background-color:rgba(219, 55, 55, 0.45); }

.bp3-tag-remove{
  background:none;
  border:none;
  color:inherit;
  cursor:pointer;
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex;
  margin-bottom:-2px;
  margin-right:-6px !important;
  margin-top:-2px;
  opacity:0.5;
  padding:2px;
  padding-left:0; }
  .bp3-tag-remove:hover{
    background:none;
    opacity:0.8;
    text-decoration:none; }
  .bp3-tag-remove:active{
    opacity:1; }
  .bp3-tag-remove:empty::before{
    font-family:"Icons16", sans-serif;
    font-size:16px;
    font-style:normal;
    font-weight:400;
    line-height:1;
    -moz-osx-font-smoothing:grayscale;
    -webkit-font-smoothing:antialiased;
    content:""; }
  .bp3-large .bp3-tag-remove{
    margin-right:-10px !important;
    padding:0 5px 0 0; }
    .bp3-large .bp3-tag-remove:empty::before{
      font-family:"Icons20", sans-serif;
      font-size:20px;
      font-style:normal;
      font-weight:400;
      line-height:1; }
.bp3-tag-input{
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex;
  -webkit-box-orient:horizontal;
  -webkit-box-direction:normal;
      -ms-flex-direction:row;
          flex-direction:row;
  -webkit-box-align:start;
      -ms-flex-align:start;
          align-items:flex-start;
  cursor:text;
  height:auto;
  line-height:inherit;
  min-height:30px;
  padding-left:5px;
  padding-right:0; }
  .bp3-tag-input > *{
    -webkit-box-flex:0;
        -ms-flex-positive:0;
            flex-grow:0;
    -ms-flex-negative:0;
        flex-shrink:0; }
  .bp3-tag-input > .bp3-tag-input-values{
    -webkit-box-flex:1;
        -ms-flex-positive:1;
            flex-grow:1;
    -ms-flex-negative:1;
        flex-shrink:1; }
  .bp3-tag-input .bp3-tag-input-icon{
    color:#5c7080;
    margin-left:2px;
    margin-right:7px;
    margin-top:7px; }
  .bp3-tag-input .bp3-tag-input-values{
    display:-webkit-box;
    display:-ms-flexbox;
    display:flex;
    -webkit-box-orient:horizontal;
    -webkit-box-direction:normal;
        -ms-flex-direction:row;
            flex-direction:row;
    -webkit-box-align:center;
        -ms-flex-align:center;
            align-items:center;
    -ms-flex-item-align:stretch;
        align-self:stretch;
    -ms-flex-wrap:wrap;
        flex-wrap:wrap;
    margin-right:7px;
    margin-top:5px;
    min-width:0; }
    .bp3-tag-input .bp3-tag-input-values > *{
      -webkit-box-flex:0;
          -ms-flex-positive:0;
              flex-grow:0;
      -ms-flex-negative:0;
          flex-shrink:0; }
    .bp3-tag-input .bp3-tag-input-values > .bp3-fill{
      -webkit-box-flex:1;
          -ms-flex-positive:1;
              flex-grow:1;
      -ms-flex-negative:1;
          flex-shrink:1; }
    .bp3-tag-input .bp3-tag-input-values::before,
    .bp3-tag-input .bp3-tag-input-values > *{
      margin-right:5px; }
    .bp3-tag-input .bp3-tag-input-values:empty::before,
    .bp3-tag-input .bp3-tag-input-values > :last-child{
      margin-right:0; }
    .bp3-tag-input .bp3-tag-input-values:first-child .bp3-input-ghost:first-child{
      padding-left:5px; }
    .bp3-tag-input .bp3-tag-input-values > *{
      margin-bottom:5px; }
  .bp3-tag-input .bp3-tag{
    overflow-wrap:break-word; }
    .bp3-tag-input .bp3-tag.bp3-active{
      outline:rgba(19, 124, 189, 0.6) auto 2px;
      outline-offset:0;
      -moz-outline-radius:6px; }
  .bp3-tag-input .bp3-input-ghost{
    -webkit-box-flex:1;
        -ms-flex:1 1 auto;
            flex:1 1 auto;
    line-height:20px;
    width:80px; }
    .bp3-tag-input .bp3-input-ghost:disabled, .bp3-tag-input .bp3-input-ghost.bp3-disabled{
      cursor:not-allowed; }
  .bp3-tag-input .bp3-button,
  .bp3-tag-input .bp3-spinner{
    margin:3px;
    margin-left:0; }
  .bp3-tag-input .bp3-button{
    min-height:24px;
    min-width:24px;
    padding:0 7px; }
  .bp3-tag-input.bp3-large{
    height:auto;
    min-height:40px; }
    .bp3-tag-input.bp3-large::before,
    .bp3-tag-input.bp3-large > *{
      margin-right:10px; }
    .bp3-tag-input.bp3-large:empty::before,
    .bp3-tag-input.bp3-large > :last-child{
      margin-right:0; }
    .bp3-tag-input.bp3-large .bp3-tag-input-icon{
      margin-left:5px;
      margin-top:10px; }
    .bp3-tag-input.bp3-large .bp3-input-ghost{
      line-height:30px; }
    .bp3-tag-input.bp3-large .bp3-button{
      min-height:30px;
      min-width:30px;
      padding:5px 10px;
      margin:5px;
      margin-left:0; }
    .bp3-tag-input.bp3-large .bp3-spinner{
      margin:8px;
      margin-left:0; }
  .bp3-tag-input.bp3-active{
    background-color:#ffffff;
    -webkit-box-shadow:0 0 0 1px #137cbd, 0 0 0 3px rgba(19, 124, 189, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2);
            box-shadow:0 0 0 1px #137cbd, 0 0 0 3px rgba(19, 124, 189, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
    .bp3-tag-input.bp3-active.bp3-intent-primary{
      -webkit-box-shadow:0 0 0 1px #106ba3, 0 0 0 3px rgba(16, 107, 163, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2);
              box-shadow:0 0 0 1px #106ba3, 0 0 0 3px rgba(16, 107, 163, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
    .bp3-tag-input.bp3-active.bp3-intent-success{
      -webkit-box-shadow:0 0 0 1px #0d8050, 0 0 0 3px rgba(13, 128, 80, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2);
              box-shadow:0 0 0 1px #0d8050, 0 0 0 3px rgba(13, 128, 80, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
    .bp3-tag-input.bp3-active.bp3-intent-warning{
      -webkit-box-shadow:0 0 0 1px #bf7326, 0 0 0 3px rgba(191, 115, 38, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2);
              box-shadow:0 0 0 1px #bf7326, 0 0 0 3px rgba(191, 115, 38, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
    .bp3-tag-input.bp3-active.bp3-intent-danger{
      -webkit-box-shadow:0 0 0 1px #c23030, 0 0 0 3px rgba(194, 48, 48, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2);
              box-shadow:0 0 0 1px #c23030, 0 0 0 3px rgba(194, 48, 48, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.2); }
  .bp3-dark .bp3-tag-input .bp3-tag-input-icon, .bp3-tag-input.bp3-dark .bp3-tag-input-icon{
    color:#a7b6c2; }
  .bp3-dark .bp3-tag-input .bp3-input-ghost, .bp3-tag-input.bp3-dark .bp3-input-ghost{
    color:#f5f8fa; }
    .bp3-dark .bp3-tag-input .bp3-input-ghost::-webkit-input-placeholder, .bp3-tag-input.bp3-dark .bp3-input-ghost::-webkit-input-placeholder{
      color:rgba(167, 182, 194, 0.6); }
    .bp3-dark .bp3-tag-input .bp3-input-ghost::-moz-placeholder, .bp3-tag-input.bp3-dark .bp3-input-ghost::-moz-placeholder{
      color:rgba(167, 182, 194, 0.6); }
    .bp3-dark .bp3-tag-input .bp3-input-ghost:-ms-input-placeholder, .bp3-tag-input.bp3-dark .bp3-input-ghost:-ms-input-placeholder{
      color:rgba(167, 182, 194, 0.6); }
    .bp3-dark .bp3-tag-input .bp3-input-ghost::-ms-input-placeholder, .bp3-tag-input.bp3-dark .bp3-input-ghost::-ms-input-placeholder{
      color:rgba(167, 182, 194, 0.6); }
    .bp3-dark .bp3-tag-input .bp3-input-ghost::placeholder, .bp3-tag-input.bp3-dark .bp3-input-ghost::placeholder{
      color:rgba(167, 182, 194, 0.6); }
  .bp3-dark .bp3-tag-input.bp3-active, .bp3-tag-input.bp3-dark.bp3-active{
    background-color:rgba(16, 22, 26, 0.3);
    -webkit-box-shadow:0 0 0 1px #137cbd, 0 0 0 1px #137cbd, 0 0 0 3px rgba(19, 124, 189, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px #137cbd, 0 0 0 1px #137cbd, 0 0 0 3px rgba(19, 124, 189, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4); }
    .bp3-dark .bp3-tag-input.bp3-active.bp3-intent-primary, .bp3-tag-input.bp3-dark.bp3-active.bp3-intent-primary{
      -webkit-box-shadow:0 0 0 1px #106ba3, 0 0 0 3px rgba(16, 107, 163, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
              box-shadow:0 0 0 1px #106ba3, 0 0 0 3px rgba(16, 107, 163, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4); }
    .bp3-dark .bp3-tag-input.bp3-active.bp3-intent-success, .bp3-tag-input.bp3-dark.bp3-active.bp3-intent-success{
      -webkit-box-shadow:0 0 0 1px #0d8050, 0 0 0 3px rgba(13, 128, 80, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
              box-shadow:0 0 0 1px #0d8050, 0 0 0 3px rgba(13, 128, 80, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4); }
    .bp3-dark .bp3-tag-input.bp3-active.bp3-intent-warning, .bp3-tag-input.bp3-dark.bp3-active.bp3-intent-warning{
      -webkit-box-shadow:0 0 0 1px #bf7326, 0 0 0 3px rgba(191, 115, 38, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
              box-shadow:0 0 0 1px #bf7326, 0 0 0 3px rgba(191, 115, 38, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4); }
    .bp3-dark .bp3-tag-input.bp3-active.bp3-intent-danger, .bp3-tag-input.bp3-dark.bp3-active.bp3-intent-danger{
      -webkit-box-shadow:0 0 0 1px #c23030, 0 0 0 3px rgba(194, 48, 48, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4);
              box-shadow:0 0 0 1px #c23030, 0 0 0 3px rgba(194, 48, 48, 0.3), inset 0 0 0 1px rgba(16, 22, 26, 0.3), inset 0 1px 1px rgba(16, 22, 26, 0.4); }

.bp3-input-ghost{
  background:none;
  border:none;
  -webkit-box-shadow:none;
          box-shadow:none;
  padding:0; }
  .bp3-input-ghost::-webkit-input-placeholder{
    color:rgba(92, 112, 128, 0.6);
    opacity:1; }
  .bp3-input-ghost::-moz-placeholder{
    color:rgba(92, 112, 128, 0.6);
    opacity:1; }
  .bp3-input-ghost:-ms-input-placeholder{
    color:rgba(92, 112, 128, 0.6);
    opacity:1; }
  .bp3-input-ghost::-ms-input-placeholder{
    color:rgba(92, 112, 128, 0.6);
    opacity:1; }
  .bp3-input-ghost::placeholder{
    color:rgba(92, 112, 128, 0.6);
    opacity:1; }
  .bp3-input-ghost:focus{
    outline:none !important; }
.bp3-toast{
  -webkit-box-align:start;
      -ms-flex-align:start;
          align-items:flex-start;
  background-color:#ffffff;
  border-radius:3px;
  -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 2px 4px rgba(16, 22, 26, 0.2), 0 8px 24px rgba(16, 22, 26, 0.2);
          box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 2px 4px rgba(16, 22, 26, 0.2), 0 8px 24px rgba(16, 22, 26, 0.2);
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex;
  margin:20px 0 0;
  max-width:500px;
  min-width:300px;
  pointer-events:all;
  position:relative !important; }
  .bp3-toast.bp3-toast-enter, .bp3-toast.bp3-toast-appear{
    -webkit-transform:translateY(-40px);
            transform:translateY(-40px); }
  .bp3-toast.bp3-toast-enter-active, .bp3-toast.bp3-toast-appear-active{
    -webkit-transform:translateY(0);
            transform:translateY(0);
    -webkit-transition-delay:0;
            transition-delay:0;
    -webkit-transition-duration:300ms;
            transition-duration:300ms;
    -webkit-transition-property:-webkit-transform;
    transition-property:-webkit-transform;
    transition-property:transform;
    transition-property:transform, -webkit-transform;
    -webkit-transition-timing-function:cubic-bezier(0.54, 1.12, 0.38, 1.11);
            transition-timing-function:cubic-bezier(0.54, 1.12, 0.38, 1.11); }
  .bp3-toast.bp3-toast-enter ~ .bp3-toast, .bp3-toast.bp3-toast-appear ~ .bp3-toast{
    -webkit-transform:translateY(-40px);
            transform:translateY(-40px); }
  .bp3-toast.bp3-toast-enter-active ~ .bp3-toast, .bp3-toast.bp3-toast-appear-active ~ .bp3-toast{
    -webkit-transform:translateY(0);
            transform:translateY(0);
    -webkit-transition-delay:0;
            transition-delay:0;
    -webkit-transition-duration:300ms;
            transition-duration:300ms;
    -webkit-transition-property:-webkit-transform;
    transition-property:-webkit-transform;
    transition-property:transform;
    transition-property:transform, -webkit-transform;
    -webkit-transition-timing-function:cubic-bezier(0.54, 1.12, 0.38, 1.11);
            transition-timing-function:cubic-bezier(0.54, 1.12, 0.38, 1.11); }
  .bp3-toast.bp3-toast-exit{
    opacity:1;
    -webkit-filter:blur(0);
            filter:blur(0); }
  .bp3-toast.bp3-toast-exit-active{
    opacity:0;
    -webkit-filter:blur(10px);
            filter:blur(10px);
    -webkit-transition-delay:0;
            transition-delay:0;
    -webkit-transition-duration:300ms;
            transition-duration:300ms;
    -webkit-transition-property:opacity, -webkit-filter;
    transition-property:opacity, -webkit-filter;
    transition-property:opacity, filter;
    transition-property:opacity, filter, -webkit-filter;
    -webkit-transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9);
            transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9); }
  .bp3-toast.bp3-toast-exit ~ .bp3-toast{
    -webkit-transform:translateY(0);
            transform:translateY(0); }
  .bp3-toast.bp3-toast-exit-active ~ .bp3-toast{
    -webkit-transform:translateY(-40px);
            transform:translateY(-40px);
    -webkit-transition-delay:50ms;
            transition-delay:50ms;
    -webkit-transition-duration:100ms;
            transition-duration:100ms;
    -webkit-transition-property:-webkit-transform;
    transition-property:-webkit-transform;
    transition-property:transform;
    transition-property:transform, -webkit-transform;
    -webkit-transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9);
            transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9); }
  .bp3-toast .bp3-button-group{
    -webkit-box-flex:0;
        -ms-flex:0 0 auto;
            flex:0 0 auto;
    padding:5px;
    padding-left:0; }
  .bp3-toast > .bp3-icon{
    color:#5c7080;
    margin:12px;
    margin-right:0; }
  .bp3-toast.bp3-dark,
  .bp3-dark .bp3-toast{
    background-color:#394b59;
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 2px 4px rgba(16, 22, 26, 0.4), 0 8px 24px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 2px 4px rgba(16, 22, 26, 0.4), 0 8px 24px rgba(16, 22, 26, 0.4); }
    .bp3-toast.bp3-dark > .bp3-icon,
    .bp3-dark .bp3-toast > .bp3-icon{
      color:#a7b6c2; }
  .bp3-toast[class*="bp3-intent-"] a{
    color:rgba(255, 255, 255, 0.7); }
    .bp3-toast[class*="bp3-intent-"] a:hover{
      color:#ffffff; }
  .bp3-toast[class*="bp3-intent-"] > .bp3-icon{
    color:#ffffff; }
  .bp3-toast[class*="bp3-intent-"] .bp3-button, .bp3-toast[class*="bp3-intent-"] .bp3-button::before,
  .bp3-toast[class*="bp3-intent-"] .bp3-button .bp3-icon, .bp3-toast[class*="bp3-intent-"] .bp3-button:active{
    color:rgba(255, 255, 255, 0.7) !important; }
  .bp3-toast[class*="bp3-intent-"] .bp3-button:focus{
    outline-color:rgba(255, 255, 255, 0.5); }
  .bp3-toast[class*="bp3-intent-"] .bp3-button:hover{
    background-color:rgba(255, 255, 255, 0.15) !important;
    color:#ffffff !important; }
  .bp3-toast[class*="bp3-intent-"] .bp3-button:active{
    background-color:rgba(255, 255, 255, 0.3) !important;
    color:#ffffff !important; }
  .bp3-toast[class*="bp3-intent-"] .bp3-button::after{
    background:rgba(255, 255, 255, 0.3) !important; }
  .bp3-toast.bp3-intent-primary{
    background-color:#137cbd;
    color:#ffffff; }
  .bp3-toast.bp3-intent-success{
    background-color:#0f9960;
    color:#ffffff; }
  .bp3-toast.bp3-intent-warning{
    background-color:#d9822b;
    color:#ffffff; }
  .bp3-toast.bp3-intent-danger{
    background-color:#db3737;
    color:#ffffff; }

.bp3-toast-message{
  -webkit-box-flex:1;
      -ms-flex:1 1 auto;
          flex:1 1 auto;
  padding:11px;
  word-break:break-word; }

.bp3-toast-container{
  -webkit-box-align:center;
      -ms-flex-align:center;
          align-items:center;
  display:-webkit-box !important;
  display:-ms-flexbox !important;
  display:flex !important;
  -webkit-box-orient:vertical;
  -webkit-box-direction:normal;
      -ms-flex-direction:column;
          flex-direction:column;
  left:0;
  overflow:hidden;
  padding:0 20px 20px;
  pointer-events:none;
  position:fixed;
  right:0;
  z-index:40; }
  .bp3-toast-container.bp3-toast-container-top{
    top:0; }
  .bp3-toast-container.bp3-toast-container-bottom{
    bottom:0;
    -webkit-box-orient:vertical;
    -webkit-box-direction:reverse;
        -ms-flex-direction:column-reverse;
            flex-direction:column-reverse;
    top:auto; }
  .bp3-toast-container.bp3-toast-container-left{
    -webkit-box-align:start;
        -ms-flex-align:start;
            align-items:flex-start; }
  .bp3-toast-container.bp3-toast-container-right{
    -webkit-box-align:end;
        -ms-flex-align:end;
            align-items:flex-end; }

.bp3-toast-container-bottom .bp3-toast.bp3-toast-enter:not(.bp3-toast-enter-active),
.bp3-toast-container-bottom .bp3-toast.bp3-toast-enter:not(.bp3-toast-enter-active) ~ .bp3-toast, .bp3-toast-container-bottom .bp3-toast.bp3-toast-appear:not(.bp3-toast-appear-active),
.bp3-toast-container-bottom .bp3-toast.bp3-toast-appear:not(.bp3-toast-appear-active) ~ .bp3-toast,
.bp3-toast-container-bottom .bp3-toast.bp3-toast-exit-active ~ .bp3-toast,
.bp3-toast-container-bottom .bp3-toast.bp3-toast-leave-active ~ .bp3-toast{
  -webkit-transform:translateY(60px);
          transform:translateY(60px); }
.bp3-tooltip{
  -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 2px 4px rgba(16, 22, 26, 0.2), 0 8px 24px rgba(16, 22, 26, 0.2);
          box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 2px 4px rgba(16, 22, 26, 0.2), 0 8px 24px rgba(16, 22, 26, 0.2);
  -webkit-transform:scale(1);
          transform:scale(1); }
  .bp3-tooltip .bp3-popover-arrow{
    height:22px;
    position:absolute;
    width:22px; }
    .bp3-tooltip .bp3-popover-arrow::before{
      height:14px;
      margin:4px;
      width:14px; }
  .bp3-tether-element-attached-bottom.bp3-tether-target-attached-top > .bp3-tooltip{
    margin-bottom:11px;
    margin-top:-11px; }
    .bp3-tether-element-attached-bottom.bp3-tether-target-attached-top > .bp3-tooltip > .bp3-popover-arrow{
      bottom:-8px; }
      .bp3-tether-element-attached-bottom.bp3-tether-target-attached-top > .bp3-tooltip > .bp3-popover-arrow svg{
        -webkit-transform:rotate(-90deg);
                transform:rotate(-90deg); }
  .bp3-tether-element-attached-left.bp3-tether-target-attached-right > .bp3-tooltip{
    margin-left:11px; }
    .bp3-tether-element-attached-left.bp3-tether-target-attached-right > .bp3-tooltip > .bp3-popover-arrow{
      left:-8px; }
      .bp3-tether-element-attached-left.bp3-tether-target-attached-right > .bp3-tooltip > .bp3-popover-arrow svg{
        -webkit-transform:rotate(0);
                transform:rotate(0); }
  .bp3-tether-element-attached-top.bp3-tether-target-attached-bottom > .bp3-tooltip{
    margin-top:11px; }
    .bp3-tether-element-attached-top.bp3-tether-target-attached-bottom > .bp3-tooltip > .bp3-popover-arrow{
      top:-8px; }
      .bp3-tether-element-attached-top.bp3-tether-target-attached-bottom > .bp3-tooltip > .bp3-popover-arrow svg{
        -webkit-transform:rotate(90deg);
                transform:rotate(90deg); }
  .bp3-tether-element-attached-right.bp3-tether-target-attached-left > .bp3-tooltip{
    margin-left:-11px;
    margin-right:11px; }
    .bp3-tether-element-attached-right.bp3-tether-target-attached-left > .bp3-tooltip > .bp3-popover-arrow{
      right:-8px; }
      .bp3-tether-element-attached-right.bp3-tether-target-attached-left > .bp3-tooltip > .bp3-popover-arrow svg{
        -webkit-transform:rotate(180deg);
                transform:rotate(180deg); }
  .bp3-tether-element-attached-middle > .bp3-tooltip > .bp3-popover-arrow{
    top:50%;
    -webkit-transform:translateY(-50%);
            transform:translateY(-50%); }
  .bp3-tether-element-attached-center > .bp3-tooltip > .bp3-popover-arrow{
    right:50%;
    -webkit-transform:translateX(50%);
            transform:translateX(50%); }
  .bp3-tether-element-attached-top.bp3-tether-target-attached-top > .bp3-tooltip > .bp3-popover-arrow{
    top:-0.22183px; }
  .bp3-tether-element-attached-right.bp3-tether-target-attached-right > .bp3-tooltip > .bp3-popover-arrow{
    right:-0.22183px; }
  .bp3-tether-element-attached-left.bp3-tether-target-attached-left > .bp3-tooltip > .bp3-popover-arrow{
    left:-0.22183px; }
  .bp3-tether-element-attached-bottom.bp3-tether-target-attached-bottom > .bp3-tooltip > .bp3-popover-arrow{
    bottom:-0.22183px; }
  .bp3-tether-element-attached-top.bp3-tether-element-attached-left > .bp3-tooltip{
    -webkit-transform-origin:top left;
            transform-origin:top left; }
  .bp3-tether-element-attached-top.bp3-tether-element-attached-center > .bp3-tooltip{
    -webkit-transform-origin:top center;
            transform-origin:top center; }
  .bp3-tether-element-attached-top.bp3-tether-element-attached-right > .bp3-tooltip{
    -webkit-transform-origin:top right;
            transform-origin:top right; }
  .bp3-tether-element-attached-middle.bp3-tether-element-attached-left > .bp3-tooltip{
    -webkit-transform-origin:center left;
            transform-origin:center left; }
  .bp3-tether-element-attached-middle.bp3-tether-element-attached-center > .bp3-tooltip{
    -webkit-transform-origin:center center;
            transform-origin:center center; }
  .bp3-tether-element-attached-middle.bp3-tether-element-attached-right > .bp3-tooltip{
    -webkit-transform-origin:center right;
            transform-origin:center right; }
  .bp3-tether-element-attached-bottom.bp3-tether-element-attached-left > .bp3-tooltip{
    -webkit-transform-origin:bottom left;
            transform-origin:bottom left; }
  .bp3-tether-element-attached-bottom.bp3-tether-element-attached-center > .bp3-tooltip{
    -webkit-transform-origin:bottom center;
            transform-origin:bottom center; }
  .bp3-tether-element-attached-bottom.bp3-tether-element-attached-right > .bp3-tooltip{
    -webkit-transform-origin:bottom right;
            transform-origin:bottom right; }
  .bp3-tooltip .bp3-popover-content{
    background:#394b59;
    color:#f5f8fa; }
  .bp3-tooltip .bp3-popover-arrow::before{
    -webkit-box-shadow:1px 1px 6px rgba(16, 22, 26, 0.2);
            box-shadow:1px 1px 6px rgba(16, 22, 26, 0.2); }
  .bp3-tooltip .bp3-popover-arrow-border{
    fill:#10161a;
    fill-opacity:0.1; }
  .bp3-tooltip .bp3-popover-arrow-fill{
    fill:#394b59; }
  .bp3-popover-enter > .bp3-tooltip, .bp3-popover-appear > .bp3-tooltip{
    -webkit-transform:scale(0.8);
            transform:scale(0.8); }
  .bp3-popover-enter-active > .bp3-tooltip, .bp3-popover-appear-active > .bp3-tooltip{
    -webkit-transform:scale(1);
            transform:scale(1);
    -webkit-transition-delay:0;
            transition-delay:0;
    -webkit-transition-duration:100ms;
            transition-duration:100ms;
    -webkit-transition-property:-webkit-transform;
    transition-property:-webkit-transform;
    transition-property:transform;
    transition-property:transform, -webkit-transform;
    -webkit-transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9);
            transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9); }
  .bp3-popover-exit > .bp3-tooltip{
    -webkit-transform:scale(1);
            transform:scale(1); }
  .bp3-popover-exit-active > .bp3-tooltip{
    -webkit-transform:scale(0.8);
            transform:scale(0.8);
    -webkit-transition-delay:0;
            transition-delay:0;
    -webkit-transition-duration:100ms;
            transition-duration:100ms;
    -webkit-transition-property:-webkit-transform;
    transition-property:-webkit-transform;
    transition-property:transform;
    transition-property:transform, -webkit-transform;
    -webkit-transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9);
            transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9); }
  .bp3-tooltip .bp3-popover-content{
    padding:10px 12px; }
  .bp3-tooltip.bp3-dark,
  .bp3-dark .bp3-tooltip{
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 2px 4px rgba(16, 22, 26, 0.4), 0 8px 24px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 2px 4px rgba(16, 22, 26, 0.4), 0 8px 24px rgba(16, 22, 26, 0.4); }
    .bp3-tooltip.bp3-dark .bp3-popover-content,
    .bp3-dark .bp3-tooltip .bp3-popover-content{
      background:#e1e8ed;
      color:#394b59; }
    .bp3-tooltip.bp3-dark .bp3-popover-arrow::before,
    .bp3-dark .bp3-tooltip .bp3-popover-arrow::before{
      -webkit-box-shadow:1px 1px 6px rgba(16, 22, 26, 0.4);
              box-shadow:1px 1px 6px rgba(16, 22, 26, 0.4); }
    .bp3-tooltip.bp3-dark .bp3-popover-arrow-border,
    .bp3-dark .bp3-tooltip .bp3-popover-arrow-border{
      fill:#10161a;
      fill-opacity:0.2; }
    .bp3-tooltip.bp3-dark .bp3-popover-arrow-fill,
    .bp3-dark .bp3-tooltip .bp3-popover-arrow-fill{
      fill:#e1e8ed; }
  .bp3-tooltip.bp3-intent-primary .bp3-popover-content{
    background:#137cbd;
    color:#ffffff; }
  .bp3-tooltip.bp3-intent-primary .bp3-popover-arrow-fill{
    fill:#137cbd; }
  .bp3-tooltip.bp3-intent-success .bp3-popover-content{
    background:#0f9960;
    color:#ffffff; }
  .bp3-tooltip.bp3-intent-success .bp3-popover-arrow-fill{
    fill:#0f9960; }
  .bp3-tooltip.bp3-intent-warning .bp3-popover-content{
    background:#d9822b;
    color:#ffffff; }
  .bp3-tooltip.bp3-intent-warning .bp3-popover-arrow-fill{
    fill:#d9822b; }
  .bp3-tooltip.bp3-intent-danger .bp3-popover-content{
    background:#db3737;
    color:#ffffff; }
  .bp3-tooltip.bp3-intent-danger .bp3-popover-arrow-fill{
    fill:#db3737; }

.bp3-tooltip-indicator{
  border-bottom:dotted 1px;
  cursor:help; }
.bp3-tree .bp3-icon, .bp3-tree .bp3-icon-standard, .bp3-tree .bp3-icon-large{
  color:#5c7080; }
  .bp3-tree .bp3-icon.bp3-intent-primary, .bp3-tree .bp3-icon-standard.bp3-intent-primary, .bp3-tree .bp3-icon-large.bp3-intent-primary{
    color:#137cbd; }
  .bp3-tree .bp3-icon.bp3-intent-success, .bp3-tree .bp3-icon-standard.bp3-intent-success, .bp3-tree .bp3-icon-large.bp3-intent-success{
    color:#0f9960; }
  .bp3-tree .bp3-icon.bp3-intent-warning, .bp3-tree .bp3-icon-standard.bp3-intent-warning, .bp3-tree .bp3-icon-large.bp3-intent-warning{
    color:#d9822b; }
  .bp3-tree .bp3-icon.bp3-intent-danger, .bp3-tree .bp3-icon-standard.bp3-intent-danger, .bp3-tree .bp3-icon-large.bp3-intent-danger{
    color:#db3737; }

.bp3-tree-node-list{
  list-style:none;
  margin:0;
  padding-left:0; }

.bp3-tree-root{
  background-color:transparent;
  cursor:default;
  padding-left:0;
  position:relative; }

.bp3-tree-node-content-0{
  padding-left:0px; }

.bp3-tree-node-content-1{
  padding-left:23px; }

.bp3-tree-node-content-2{
  padding-left:46px; }

.bp3-tree-node-content-3{
  padding-left:69px; }

.bp3-tree-node-content-4{
  padding-left:92px; }

.bp3-tree-node-content-5{
  padding-left:115px; }

.bp3-tree-node-content-6{
  padding-left:138px; }

.bp3-tree-node-content-7{
  padding-left:161px; }

.bp3-tree-node-content-8{
  padding-left:184px; }

.bp3-tree-node-content-9{
  padding-left:207px; }

.bp3-tree-node-content-10{
  padding-left:230px; }

.bp3-tree-node-content-11{
  padding-left:253px; }

.bp3-tree-node-content-12{
  padding-left:276px; }

.bp3-tree-node-content-13{
  padding-left:299px; }

.bp3-tree-node-content-14{
  padding-left:322px; }

.bp3-tree-node-content-15{
  padding-left:345px; }

.bp3-tree-node-content-16{
  padding-left:368px; }

.bp3-tree-node-content-17{
  padding-left:391px; }

.bp3-tree-node-content-18{
  padding-left:414px; }

.bp3-tree-node-content-19{
  padding-left:437px; }

.bp3-tree-node-content-20{
  padding-left:460px; }

.bp3-tree-node-content{
  -webkit-box-align:center;
      -ms-flex-align:center;
          align-items:center;
  display:-webkit-box;
  display:-ms-flexbox;
  display:flex;
  height:30px;
  padding-right:5px;
  width:100%; }
  .bp3-tree-node-content:hover{
    background-color:rgba(191, 204, 214, 0.4); }

.bp3-tree-node-caret,
.bp3-tree-node-caret-none{
  min-width:30px; }

.bp3-tree-node-caret{
  color:#5c7080;
  cursor:pointer;
  padding:7px;
  -webkit-transform:rotate(0deg);
          transform:rotate(0deg);
  -webkit-transition:-webkit-transform 200ms cubic-bezier(0.4, 1, 0.75, 0.9);
  transition:-webkit-transform 200ms cubic-bezier(0.4, 1, 0.75, 0.9);
  transition:transform 200ms cubic-bezier(0.4, 1, 0.75, 0.9);
  transition:transform 200ms cubic-bezier(0.4, 1, 0.75, 0.9), -webkit-transform 200ms cubic-bezier(0.4, 1, 0.75, 0.9); }
  .bp3-tree-node-caret:hover{
    color:#182026; }
  .bp3-dark .bp3-tree-node-caret{
    color:#a7b6c2; }
    .bp3-dark .bp3-tree-node-caret:hover{
      color:#f5f8fa; }
  .bp3-tree-node-caret.bp3-tree-node-caret-open{
    -webkit-transform:rotate(90deg);
            transform:rotate(90deg); }
  .bp3-tree-node-caret.bp3-icon-standard::before{
    content:""; }

.bp3-tree-node-icon{
  margin-right:7px;
  position:relative; }

.bp3-tree-node-label{
  overflow:hidden;
  text-overflow:ellipsis;
  white-space:nowrap;
  word-wrap:normal;
  -webkit-box-flex:1;
      -ms-flex:1 1 auto;
          flex:1 1 auto;
  position:relative;
  -webkit-user-select:none;
     -moz-user-select:none;
      -ms-user-select:none;
          user-select:none; }
  .bp3-tree-node-label span{
    display:inline; }

.bp3-tree-node-secondary-label{
  padding:0 5px;
  -webkit-user-select:none;
     -moz-user-select:none;
      -ms-user-select:none;
          user-select:none; }
  .bp3-tree-node-secondary-label .bp3-popover-wrapper,
  .bp3-tree-node-secondary-label .bp3-popover-target{
    -webkit-box-align:center;
        -ms-flex-align:center;
            align-items:center;
    display:-webkit-box;
    display:-ms-flexbox;
    display:flex; }

.bp3-tree-node.bp3-disabled .bp3-tree-node-content{
  background-color:inherit;
  color:rgba(92, 112, 128, 0.6);
  cursor:not-allowed; }

.bp3-tree-node.bp3-disabled .bp3-tree-node-caret,
.bp3-tree-node.bp3-disabled .bp3-tree-node-icon{
  color:rgba(92, 112, 128, 0.6);
  cursor:not-allowed; }

.bp3-tree-node.bp3-tree-node-selected > .bp3-tree-node-content{
  background-color:#137cbd; }
  .bp3-tree-node.bp3-tree-node-selected > .bp3-tree-node-content,
  .bp3-tree-node.bp3-tree-node-selected > .bp3-tree-node-content .bp3-icon, .bp3-tree-node.bp3-tree-node-selected > .bp3-tree-node-content .bp3-icon-standard, .bp3-tree-node.bp3-tree-node-selected > .bp3-tree-node-content .bp3-icon-large{
    color:#ffffff; }
  .bp3-tree-node.bp3-tree-node-selected > .bp3-tree-node-content .bp3-tree-node-caret::before{
    color:rgba(255, 255, 255, 0.7); }
  .bp3-tree-node.bp3-tree-node-selected > .bp3-tree-node-content .bp3-tree-node-caret:hover::before{
    color:#ffffff; }

.bp3-dark .bp3-tree-node-content:hover{
  background-color:rgba(92, 112, 128, 0.3); }

.bp3-dark .bp3-tree .bp3-icon, .bp3-dark .bp3-tree .bp3-icon-standard, .bp3-dark .bp3-tree .bp3-icon-large{
  color:#a7b6c2; }
  .bp3-dark .bp3-tree .bp3-icon.bp3-intent-primary, .bp3-dark .bp3-tree .bp3-icon-standard.bp3-intent-primary, .bp3-dark .bp3-tree .bp3-icon-large.bp3-intent-primary{
    color:#137cbd; }
  .bp3-dark .bp3-tree .bp3-icon.bp3-intent-success, .bp3-dark .bp3-tree .bp3-icon-standard.bp3-intent-success, .bp3-dark .bp3-tree .bp3-icon-large.bp3-intent-success{
    color:#0f9960; }
  .bp3-dark .bp3-tree .bp3-icon.bp3-intent-warning, .bp3-dark .bp3-tree .bp3-icon-standard.bp3-intent-warning, .bp3-dark .bp3-tree .bp3-icon-large.bp3-intent-warning{
    color:#d9822b; }
  .bp3-dark .bp3-tree .bp3-icon.bp3-intent-danger, .bp3-dark .bp3-tree .bp3-icon-standard.bp3-intent-danger, .bp3-dark .bp3-tree .bp3-icon-large.bp3-intent-danger{
    color:#db3737; }

.bp3-dark .bp3-tree-node.bp3-tree-node-selected > .bp3-tree-node-content{
  background-color:#137cbd; }
.bp3-omnibar{
  -webkit-filter:blur(0);
          filter:blur(0);
  opacity:1;
  background-color:#ffffff;
  border-radius:3px;
  -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 4px 8px rgba(16, 22, 26, 0.2), 0 18px 46px 6px rgba(16, 22, 26, 0.2);
          box-shadow:0 0 0 1px rgba(16, 22, 26, 0.1), 0 4px 8px rgba(16, 22, 26, 0.2), 0 18px 46px 6px rgba(16, 22, 26, 0.2);
  left:calc(50% - 250px);
  top:20vh;
  width:500px;
  z-index:21; }
  .bp3-omnibar.bp3-overlay-enter, .bp3-omnibar.bp3-overlay-appear{
    -webkit-filter:blur(20px);
            filter:blur(20px);
    opacity:0.2; }
  .bp3-omnibar.bp3-overlay-enter-active, .bp3-omnibar.bp3-overlay-appear-active{
    -webkit-filter:blur(0);
            filter:blur(0);
    opacity:1;
    -webkit-transition-delay:0;
            transition-delay:0;
    -webkit-transition-duration:200ms;
            transition-duration:200ms;
    -webkit-transition-property:opacity, -webkit-filter;
    transition-property:opacity, -webkit-filter;
    transition-property:filter, opacity;
    transition-property:filter, opacity, -webkit-filter;
    -webkit-transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9);
            transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9); }
  .bp3-omnibar.bp3-overlay-exit{
    -webkit-filter:blur(0);
            filter:blur(0);
    opacity:1; }
  .bp3-omnibar.bp3-overlay-exit-active{
    -webkit-filter:blur(20px);
            filter:blur(20px);
    opacity:0.2;
    -webkit-transition-delay:0;
            transition-delay:0;
    -webkit-transition-duration:200ms;
            transition-duration:200ms;
    -webkit-transition-property:opacity, -webkit-filter;
    transition-property:opacity, -webkit-filter;
    transition-property:filter, opacity;
    transition-property:filter, opacity, -webkit-filter;
    -webkit-transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9);
            transition-timing-function:cubic-bezier(0.4, 1, 0.75, 0.9); }
  .bp3-omnibar .bp3-input{
    background-color:transparent;
    border-radius:0; }
    .bp3-omnibar .bp3-input, .bp3-omnibar .bp3-input:focus{
      -webkit-box-shadow:none;
              box-shadow:none; }
  .bp3-omnibar .bp3-menu{
    background-color:transparent;
    border-radius:0;
    -webkit-box-shadow:inset 0 1px 0 rgba(16, 22, 26, 0.15);
            box-shadow:inset 0 1px 0 rgba(16, 22, 26, 0.15);
    max-height:calc(60vh - 40px);
    overflow:auto; }
    .bp3-omnibar .bp3-menu:empty{
      display:none; }
  .bp3-dark .bp3-omnibar, .bp3-omnibar.bp3-dark{
    background-color:#30404d;
    -webkit-box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 4px 8px rgba(16, 22, 26, 0.4), 0 18px 46px 6px rgba(16, 22, 26, 0.4);
            box-shadow:0 0 0 1px rgba(16, 22, 26, 0.2), 0 4px 8px rgba(16, 22, 26, 0.4), 0 18px 46px 6px rgba(16, 22, 26, 0.4); }

.bp3-omnibar-overlay .bp3-overlay-backdrop{
  background-color:rgba(16, 22, 26, 0.2); }

.bp3-select-popover .bp3-popover-content{
  padding:5px; }

.bp3-select-popover .bp3-input-group{
  margin-bottom:0; }

.bp3-select-popover .bp3-menu{
  max-height:300px;
  max-width:400px;
  overflow:auto;
  padding:0; }
  .bp3-select-popover .bp3-menu:not(:first-child){
    padding-top:5px; }

.bp3-multi-select{
  min-width:150px; }

.bp3-multi-select-popover .bp3-menu{
  max-height:300px;
  max-width:400px;
  overflow:auto; }

.bp3-select-popover .bp3-popover-content{
  padding:5px; }

.bp3-select-popover .bp3-input-group{
  margin-bottom:0; }

.bp3-select-popover .bp3-menu{
  max-height:300px;
  max-width:400px;
  overflow:auto;
  padding:0; }
  .bp3-select-popover .bp3-menu:not(:first-child){
    padding-top:5px; }
/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/* This file was auto-generated by ensureUiComponents() in @jupyterlab/buildutils */

/**
 * (DEPRECATED) Support for consuming icons as CSS background images
 */

/* Icons urls */

:root {
  --jp-icon-add: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTE5IDEzaC02djZoLTJ2LTZINXYtMmg2VjVoMnY2aDZ2MnoiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-bug: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTIwIDhoLTIuODFjLS40NS0uNzgtMS4wNy0xLjQ1LTEuODItMS45NkwxNyA0LjQxIDE1LjU5IDNsLTIuMTcgMi4xN0MxMi45NiA1LjA2IDEyLjQ5IDUgMTIgNWMtLjQ5IDAtLjk2LjA2LTEuNDEuMTdMOC40MSAzIDcgNC40MWwxLjYyIDEuNjNDNy44OCA2LjU1IDcuMjYgNy4yMiA2LjgxIDhINHYyaDIuMDljLS4wNS4zMy0uMDkuNjYtLjA5IDF2MUg0djJoMnYxYzAgLjM0LjA0LjY3LjA5IDFINHYyaDIuODFjMS4wNCAxLjc5IDIuOTcgMyA1LjE5IDNzNC4xNS0xLjIxIDUuMTktM0gyMHYtMmgtMi4wOWMuMDUtLjMzLjA5LS42Ni4wOS0xdi0xaDJ2LTJoLTJ2LTFjMC0uMzQtLjA0LS42Ny0uMDktMUgyMFY4em0tNiA4aC00di0yaDR2MnptMC00aC00di0yaDR2MnoiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-build: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTYiIHZpZXdCb3g9IjAgMCAyNCAyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTE0LjkgMTcuNDVDMTYuMjUgMTcuNDUgMTcuMzUgMTYuMzUgMTcuMzUgMTVDMTcuMzUgMTMuNjUgMTYuMjUgMTIuNTUgMTQuOSAxMi41NUMxMy41NCAxMi41NSAxMi40NSAxMy42NSAxMi40NSAxNUMxMi40NSAxNi4zNSAxMy41NCAxNy40NSAxNC45IDE3LjQ1Wk0yMC4xIDE1LjY4TDIxLjU4IDE2Ljg0QzIxLjcxIDE2Ljk1IDIxLjc1IDE3LjEzIDIxLjY2IDE3LjI5TDIwLjI2IDE5LjcxQzIwLjE3IDE5Ljg2IDIwIDE5LjkyIDE5LjgzIDE5Ljg2TDE4LjA5IDE5LjE2QzE3LjczIDE5LjQ0IDE3LjMzIDE5LjY3IDE2LjkxIDE5Ljg1TDE2LjY0IDIxLjdDMTYuNjIgMjEuODcgMTYuNDcgMjIgMTYuMyAyMkgxMy41QzEzLjMyIDIyIDEzLjE4IDIxLjg3IDEzLjE1IDIxLjdMMTIuODkgMTkuODVDMTIuNDYgMTkuNjcgMTIuMDcgMTkuNDQgMTEuNzEgMTkuMTZMOS45NjAwMiAxOS44NkM5LjgxMDAyIDE5LjkyIDkuNjIwMDIgMTkuODYgOS41NDAwMiAxOS43MUw4LjE0MDAyIDE3LjI5QzguMDUwMDIgMTcuMTMgOC4wOTAwMiAxNi45NSA4LjIyMDAyIDE2Ljg0TDkuNzAwMDIgMTUuNjhMOS42NTAwMSAxNUw5LjcwMDAyIDE0LjMxTDguMjIwMDIgMTMuMTZDOC4wOTAwMiAxMy4wNSA4LjA1MDAyIDEyLjg2IDguMTQwMDIgMTIuNzFMOS41NDAwMiAxMC4yOUM5LjYyMDAyIDEwLjEzIDkuODEwMDIgMTAuMDcgOS45NjAwMiAxMC4xM0wxMS43MSAxMC44NEMxMi4wNyAxMC41NiAxMi40NiAxMC4zMiAxMi44OSAxMC4xNUwxMy4xNSA4LjI4OTk4QzEzLjE4IDguMTI5OTggMTMuMzIgNy45OTk5OCAxMy41IDcuOTk5OThIMTYuM0MxNi40NyA3Ljk5OTk4IDE2LjYyIDguMTI5OTggMTYuNjQgOC4yODk5OEwxNi45MSAxMC4xNUMxNy4zMyAxMC4zMiAxNy43MyAxMC41NiAxOC4wOSAxMC44NEwxOS44MyAxMC4xM0MyMCAxMC4wNyAyMC4xNyAxMC4xMyAyMC4yNiAxMC4yOUwyMS42NiAxMi43MUMyMS43NSAxMi44NiAyMS43MSAxMy4wNSAyMS41OCAxMy4xNkwyMC4xIDE0LjMxTDIwLjE1IDE1TDIwLjEgMTUuNjhaIi8+CiAgICA8cGF0aCBkPSJNNy4zMjk2NiA3LjQ0NDU0QzguMDgzMSA3LjAwOTU0IDguMzM5MzIgNi4wNTMzMiA3LjkwNDMyIDUuMjk5ODhDNy40NjkzMiA0LjU0NjQzIDYuNTA4MSA0LjI4MTU2IDUuNzU0NjYgNC43MTY1NkM1LjM5MTc2IDQuOTI2MDggNS4xMjY5NSA1LjI3MTE4IDUuMDE4NDkgNS42NzU5NEM0LjkxMDA0IDYuMDgwNzEgNC45NjY4MiA2LjUxMTk4IDUuMTc2MzQgNi44NzQ4OEM1LjYxMTM0IDcuNjI4MzIgNi41NzYyMiA3Ljg3OTU0IDcuMzI5NjYgNy40NDQ1NFpNOS42NTcxOCA0Ljc5NTkzTDEwLjg2NzIgNC45NTE3OUMxMC45NjI4IDQuOTc3NDEgMTEuMDQwMiA1LjA3MTMzIDExLjAzODIgNS4xODc5M0wxMS4wMzg4IDYuOTg4OTNDMTEuMDQ1NSA3LjEwMDU0IDEwLjk2MTYgNy4xOTUxOCAxMC44NTUgNy4yMTA1NEw5LjY2MDAxIDcuMzgwODNMOS4yMzkxNSA4LjEzMTg4TDkuNjY5NjEgOS4yNTc0NUM5LjcwNzI5IDkuMzYyNzEgOS42NjkzNCA5LjQ3Njk5IDkuNTc0MDggOS41MzE5OUw4LjAxNTIzIDEwLjQzMkM3LjkxMTMxIDEwLjQ5MiA3Ljc5MzM3IDEwLjQ2NzcgNy43MjEwNSAxMC4zODI0TDYuOTg3NDggOS40MzE4OEw2LjEwOTMxIDkuNDMwODNMNS4zNDcwNCAxMC4zOTA1QzUuMjg5MDkgMTAuNDcwMiA1LjE3MzgzIDEwLjQ5MDUgNS4wNzE4NyAxMC40MzM5TDMuNTEyNDUgOS41MzI5M0MzLjQxMDQ5IDkuNDc2MzMgMy4zNzY0NyA5LjM1NzQxIDMuNDEwNzUgOS4yNTY3OUwzLjg2MzQ3IDguMTQwOTNMMy42MTc0OSA3Ljc3NDg4TDMuNDIzNDcgNy4zNzg4M0wyLjIzMDc1IDcuMjEyOTdDMi4xMjY0NyA3LjE5MjM1IDIuMDQwNDkgNy4xMDM0MiAyLjA0MjQ1IDYuOTg2ODJMMi4wNDE4NyA1LjE4NTgyQzIuMDQzODMgNS4wNjkyMiAyLjExOTA5IDQuOTc5NTggMi4yMTcwNCA0Ljk2OTIyTDMuNDIwNjUgNC43OTM5M0wzLjg2NzQ5IDQuMDI3ODhMMy40MTEwNSAyLjkxNzMxQzMuMzczMzcgMi44MTIwNCAzLjQxMTMxIDIuNjk3NzYgMy41MTUyMyAyLjYzNzc2TDUuMDc0MDggMS43Mzc3NkM1LjE2OTM0IDEuNjgyNzYgNS4yODcyOSAxLjcwNzA0IDUuMzU5NjEgMS43OTIzMUw2LjExOTE1IDIuNzI3ODhMNi45ODAwMSAyLjczODkzTDcuNzI0OTYgMS43ODkyMkM3Ljc5MTU2IDEuNzA0NTggNy45MTU0OCAxLjY3OTIyIDguMDA4NzkgMS43NDA4Mkw5LjU2ODIxIDIuNjQxODJDOS42NzAxNyAyLjY5ODQyIDkuNzEyODUgMi44MTIzNCA5LjY4NzIzIDIuOTA3OTdMOS4yMTcxOCA0LjAzMzgzTDkuNDYzMTYgNC4zOTk4OEw5LjY1NzE4IDQuNzk1OTNaIi8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-caret-down-empty-thin: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIwIDIwIj4KCTxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSIgc2hhcGUtcmVuZGVyaW5nPSJnZW9tZXRyaWNQcmVjaXNpb24iPgoJCTxwb2x5Z29uIGNsYXNzPSJzdDEiIHBvaW50cz0iOS45LDEzLjYgMy42LDcuNCA0LjQsNi42IDkuOSwxMi4yIDE1LjQsNi43IDE2LjEsNy40ICIvPgoJPC9nPgo8L3N2Zz4K);
  --jp-icon-caret-down-empty: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDE4IDE4Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiIHNoYXBlLXJlbmRlcmluZz0iZ2VvbWV0cmljUHJlY2lzaW9uIj4KICAgIDxwYXRoIGQ9Ik01LjIsNS45TDksOS43bDMuOC0zLjhsMS4yLDEuMmwtNC45LDVsLTQuOS01TDUuMiw1Ljl6Ii8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-caret-down: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDE4IDE4Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiIHNoYXBlLXJlbmRlcmluZz0iZ2VvbWV0cmljUHJlY2lzaW9uIj4KICAgIDxwYXRoIGQ9Ik01LjIsNy41TDksMTEuMmwzLjgtMy44SDUuMnoiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-caret-left: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDE4IDE4Ij4KCTxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSIgc2hhcGUtcmVuZGVyaW5nPSJnZW9tZXRyaWNQcmVjaXNpb24iPgoJCTxwYXRoIGQ9Ik0xMC44LDEyLjhMNy4xLDlsMy44LTMuOGwwLDcuNkgxMC44eiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-caret-right: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDE4IDE4Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiIHNoYXBlLXJlbmRlcmluZz0iZ2VvbWV0cmljUHJlY2lzaW9uIj4KICAgIDxwYXRoIGQ9Ik03LjIsNS4yTDEwLjksOWwtMy44LDMuOFY1LjJINy4yeiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-caret-up-empty-thin: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIwIDIwIj4KCTxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSIgc2hhcGUtcmVuZGVyaW5nPSJnZW9tZXRyaWNQcmVjaXNpb24iPgoJCTxwb2x5Z29uIGNsYXNzPSJzdDEiIHBvaW50cz0iMTUuNCwxMy4zIDkuOSw3LjcgNC40LDEzLjIgMy42LDEyLjUgOS45LDYuMyAxNi4xLDEyLjYgIi8+Cgk8L2c+Cjwvc3ZnPgo=);
  --jp-icon-caret-up: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDE4IDE4Ij4KCTxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSIgc2hhcGUtcmVuZGVyaW5nPSJnZW9tZXRyaWNQcmVjaXNpb24iPgoJCTxwYXRoIGQ9Ik01LjIsMTAuNUw5LDYuOGwzLjgsMy44SDUuMnoiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-case-sensitive: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIwIDIwIj4KICA8ZyBjbGFzcz0ianAtaWNvbjIiIGZpbGw9IiM0MTQxNDEiPgogICAgPHJlY3QgeD0iMiIgeT0iMiIgd2lkdGg9IjE2IiBoZWlnaHQ9IjE2Ii8+CiAgPC9nPgogIDxnIGNsYXNzPSJqcC1pY29uLWFjY2VudDIiIGZpbGw9IiNGRkYiPgogICAgPHBhdGggZD0iTTcuNiw4aDAuOWwzLjUsOGgtMS4xTDEwLDE0SDZsLTAuOSwySDRMNy42LDh6IE04LDkuMUw2LjQsMTNoMy4yTDgsOS4xeiIvPgogICAgPHBhdGggZD0iTTE2LjYsOS44Yy0wLjIsMC4xLTAuNCwwLjEtMC43LDAuMWMtMC4yLDAtMC40LTAuMS0wLjYtMC4yYy0wLjEtMC4xLTAuMi0wLjQtMC4yLTAuNyBjLTAuMywwLjMtMC42LDAuNS0wLjksMC43Yy0wLjMsMC4xLTAuNywwLjItMS4xLDAuMmMtMC4zLDAtMC41LDAtMC43LTAuMWMtMC4yLTAuMS0wLjQtMC4yLTAuNi0wLjNjLTAuMi0wLjEtMC4zLTAuMy0wLjQtMC41IGMtMC4xLTAuMi0wLjEtMC40LTAuMS0wLjdjMC0wLjMsMC4xLTAuNiwwLjItMC44YzAuMS0wLjIsMC4zLTAuNCwwLjQtMC41QzEyLDcsMTIuMiw2LjksMTIuNSw2LjhjMC4yLTAuMSwwLjUtMC4xLDAuNy0wLjIgYzAuMy0wLjEsMC41LTAuMSwwLjctMC4xYzAuMiwwLDAuNC0wLjEsMC42LTAuMWMwLjIsMCwwLjMtMC4xLDAuNC0wLjJjMC4xLTAuMSwwLjItMC4yLDAuMi0wLjRjMC0xLTEuMS0xLTEuMy0xIGMtMC40LDAtMS40LDAtMS40LDEuMmgtMC45YzAtMC40LDAuMS0wLjcsMC4yLTFjMC4xLTAuMiwwLjMtMC40LDAuNS0wLjZjMC4yLTAuMiwwLjUtMC4zLDAuOC0wLjNDMTMuMyw0LDEzLjYsNCwxMy45LDQgYzAuMywwLDAuNSwwLDAuOCwwLjFjMC4zLDAsMC41LDAuMSwwLjcsMC4yYzAuMiwwLjEsMC40LDAuMywwLjUsMC41QzE2LDUsMTYsNS4yLDE2LDUuNnYyLjljMCwwLjIsMCwwLjQsMCwwLjUgYzAsMC4xLDAuMSwwLjIsMC4zLDAuMmMwLjEsMCwwLjIsMCwwLjMsMFY5Ljh6IE0xNS4yLDYuOWMtMS4yLDAuNi0zLjEsMC4yLTMuMSwxLjRjMCwxLjQsMy4xLDEsMy4xLTAuNVY2Ljl6Ii8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-check: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTkgMTYuMTdMNC44MyAxMmwtMS40MiAxLjQxTDkgMTkgMjEgN2wtMS40MS0xLjQxeiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-circle-empty: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTEyIDJDNi40NyAyIDIgNi40NyAyIDEyczQuNDcgMTAgMTAgMTAgMTAtNC40NyAxMC0xMFMxNy41MyAyIDEyIDJ6bTAgMThjLTQuNDEgMC04LTMuNTktOC04czMuNTktOCA4LTggOCAzLjU5IDggOC0zLjU5IDgtOCA4eiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-circle: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMTggMTgiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPGNpcmNsZSBjeD0iOSIgY3k9IjkiIHI9IjgiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-clear: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8bWFzayBpZD0iZG9udXRIb2xlIj4KICAgIDxyZWN0IHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgZmlsbD0id2hpdGUiIC8+CiAgICA8Y2lyY2xlIGN4PSIxMiIgY3k9IjEyIiByPSI4IiBmaWxsPSJibGFjayIvPgogIDwvbWFzaz4KCiAgPGcgY2xhc3M9ImpwLWljb24zIiBmaWxsPSIjNjE2MTYxIj4KICAgIDxyZWN0IGhlaWdodD0iMTgiIHdpZHRoPSIyIiB4PSIxMSIgeT0iMyIgdHJhbnNmb3JtPSJyb3RhdGUoMzE1LCAxMiwgMTIpIi8+CiAgICA8Y2lyY2xlIGN4PSIxMiIgY3k9IjEyIiByPSIxMCIgbWFzaz0idXJsKCNkb251dEhvbGUpIi8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-close: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbi1ub25lIGpwLWljb24tc2VsZWN0YWJsZS1pbnZlcnNlIGpwLWljb24zLWhvdmVyIiBmaWxsPSJub25lIj4KICAgIDxjaXJjbGUgY3g9IjEyIiBjeT0iMTIiIHI9IjExIi8+CiAgPC9nPgoKICA8ZyBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIGpwLWljb24tYWNjZW50Mi1ob3ZlciIgZmlsbD0iIzYxNjE2MSI+CiAgICA8cGF0aCBkPSJNMTkgNi40MUwxNy41OSA1IDEyIDEwLjU5IDYuNDEgNSA1IDYuNDEgMTAuNTkgMTIgNSAxNy41OSA2LjQxIDE5IDEyIDEzLjQxIDE3LjU5IDE5IDE5IDE3LjU5IDEzLjQxIDEyeiIvPgogIDwvZz4KCiAgPGcgY2xhc3M9ImpwLWljb24tbm9uZSBqcC1pY29uLWJ1c3kiIGZpbGw9Im5vbmUiPgogICAgPGNpcmNsZSBjeD0iMTIiIGN5PSIxMiIgcj0iNyIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-code: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjIiIGhlaWdodD0iMjIiIHZpZXdCb3g9IjAgMCAyOCAyOCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KCTxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CgkJPHBhdGggZD0iTTExLjQgMTguNkw2LjggMTRMMTEuNCA5LjRMMTAgOEw0IDE0TDEwIDIwTDExLjQgMTguNlpNMTYuNiAxOC42TDIxLjIgMTRMMTYuNiA5LjRMMTggOEwyNCAxNEwxOCAyMEwxNi42IDE4LjZWMTguNloiLz4KCTwvZz4KPC9zdmc+Cg==);
  --jp-icon-console: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIwMCAyMDAiPgogIDxnIGNsYXNzPSJqcC1pY29uLWJyYW5kMSBqcC1pY29uLXNlbGVjdGFibGUiIGZpbGw9IiMwMjg4RDEiPgogICAgPHBhdGggZD0iTTIwIDE5LjhoMTYwdjE1OS45SDIweiIvPgogIDwvZz4KICA8ZyBjbGFzcz0ianAtaWNvbi1zZWxlY3RhYmxlLWludmVyc2UiIGZpbGw9IiNmZmYiPgogICAgPHBhdGggZD0iTTEwNSAxMjcuM2g0MHYxMi44aC00MHpNNTEuMSA3N0w3NCA5OS45bC0yMy4zIDIzLjMgMTAuNSAxMC41IDIzLjMtMjMuM0w5NSA5OS45IDg0LjUgODkuNCA2MS42IDY2LjV6Ii8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-copy: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMTggMTgiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTExLjksMUgzLjJDMi40LDEsMS43LDEuNywxLjcsMi41djEwLjJoMS41VjIuNWg4LjdWMXogTTE0LjEsMy45aC04Yy0wLjgsMC0xLjUsMC43LTEuNSwxLjV2MTAuMmMwLDAuOCwwLjcsMS41LDEuNSwxLjVoOCBjMC44LDAsMS41LTAuNywxLjUtMS41VjUuNEMxNS41LDQuNiwxNC45LDMuOSwxNC4xLDMuOXogTTE0LjEsMTUuNWgtOFY1LjRoOFYxNS41eiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-cut: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTkuNjQgNy42NGMuMjMtLjUuMzYtMS4wNS4zNi0xLjY0IDAtMi4yMS0xLjc5LTQtNC00UzIgMy43OSAyIDZzMS43OSA0IDQgNGMuNTkgMCAxLjE0LS4xMyAxLjY0LS4zNkwxMCAxMmwtMi4zNiAyLjM2QzcuMTQgMTQuMTMgNi41OSAxNCA2IDE0Yy0yLjIxIDAtNCAxLjc5LTQgNHMxLjc5IDQgNCA0IDQtMS43OSA0LTRjMC0uNTktLjEzLTEuMTQtLjM2LTEuNjRMMTIgMTRsNyA3aDN2LTFMOS42NCA3LjY0ek02IDhjLTEuMSAwLTItLjg5LTItMnMuOS0yIDItMiAyIC44OSAyIDItLjkgMi0yIDJ6bTAgMTJjLTEuMSAwLTItLjg5LTItMnMuOS0yIDItMiAyIC44OSAyIDItLjkgMi0yIDJ6bTYtNy41Yy0uMjggMC0uNS0uMjItLjUtLjVzLjIyLS41LjUtLjUuNS4yMi41LjUtLjIyLjUtLjUuNXpNMTkgM2wtNiA2IDIgMiA3LTdWM3oiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-download: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTE5IDloLTRWM0g5djZINWw3IDcgNy03ek01IDE4djJoMTR2LTJINXoiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-edit: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTMgMTcuMjVWMjFoMy43NUwxNy44MSA5Ljk0bC0zLjc1LTMuNzVMMyAxNy4yNXpNMjAuNzEgNy4wNGMuMzktLjM5LjM5LTEuMDIgMC0xLjQxbC0yLjM0LTIuMzRjLS4zOS0uMzktMS4wMi0uMzktMS40MSAwbC0xLjgzIDEuODMgMy43NSAzLjc1IDEuODMtMS44M3oiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-ellipses: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPGNpcmNsZSBjeD0iNSIgY3k9IjEyIiByPSIyIi8+CiAgICA8Y2lyY2xlIGN4PSIxMiIgY3k9IjEyIiByPSIyIi8+CiAgICA8Y2lyY2xlIGN4PSIxOSIgY3k9IjEyIiByPSIyIi8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-extension: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTIwLjUgMTFIMTlWN2MwLTEuMS0uOS0yLTItMmgtNFYzLjVDMTMgMi4xMiAxMS44OCAxIDEwLjUgMVM4IDIuMTIgOCAzLjVWNUg0Yy0xLjEgMC0xLjk5LjktMS45OSAydjMuOEgzLjVjMS40OSAwIDIuNyAxLjIxIDIuNyAyLjdzLTEuMjEgMi43LTIuNyAyLjdIMlYyMGMwIDEuMS45IDIgMiAyaDMuOHYtMS41YzAtMS40OSAxLjIxLTIuNyAyLjctMi43IDEuNDkgMCAyLjcgMS4yMSAyLjcgMi43VjIySDE3YzEuMSAwIDItLjkgMi0ydi00aDEuNWMxLjM4IDAgMi41LTEuMTIgMi41LTIuNVMyMS44OCAxMSAyMC41IDExeiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-fast-forward: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGggZD0iTTQgMThsOC41LTZMNCA2djEyem05LTEydjEybDguNS02TDEzIDZ6Ii8+CiAgICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-file-upload: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTkgMTZoNnYtNmg0bC03LTctNyA3aDR6bS00IDJoMTR2Mkg1eiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-file: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8cGF0aCBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIiBkPSJNMTkuMyA4LjJsLTUuNS01LjVjLS4zLS4zLS43LS41LTEuMi0uNUgzLjljLS44LjEtMS42LjktMS42IDEuOHYxNC4xYzAgLjkuNyAxLjYgMS42IDEuNmgxNC4yYy45IDAgMS42LS43IDEuNi0xLjZWOS40Yy4xLS41LS4xLS45LS40LTEuMnptLTUuOC0zLjNsMy40IDMuNmgtMy40VjQuOXptMy45IDEyLjdINC43Yy0uMSAwLS4yIDAtLjItLjJWNC43YzAtLjIuMS0uMy4yLS4zaDcuMnY0LjRzMCAuOC4zIDEuMWMuMy4zIDEuMS4zIDEuMS4zaDQuM3Y3LjJzLS4xLjItLjIuMnoiLz4KPC9zdmc+Cg==);
  --jp-icon-filter-list: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTEwIDE4aDR2LTJoLTR2MnpNMyA2djJoMThWNkgzem0zIDdoMTJ2LTJINnYyeiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-folder: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8cGF0aCBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIiBkPSJNMTAgNEg0Yy0xLjEgMC0xLjk5LjktMS45OSAyTDIgMThjMCAxLjEuOSAyIDIgMmgxNmMxLjEgMCAyLS45IDItMlY4YzAtMS4xLS45LTItMi0yaC04bC0yLTJ6Ii8+Cjwvc3ZnPgo=);
  --jp-icon-html5: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDUxMiA1MTIiPgogIDxwYXRoIGNsYXNzPSJqcC1pY29uMCBqcC1pY29uLXNlbGVjdGFibGUiIGZpbGw9IiMwMDAiIGQ9Ik0xMDguNCAwaDIzdjIyLjhoMjEuMlYwaDIzdjY5aC0yM1Y0NmgtMjF2MjNoLTIzLjJNMjA2IDIzaC0yMC4zVjBoNjMuN3YyM0gyMjl2NDZoLTIzbTUzLjUtNjloMjQuMWwxNC44IDI0LjNMMzEzLjIgMGgyNC4xdjY5aC0yM1YzNC44bC0xNi4xIDI0LjgtMTYuMS0yNC44VjY5aC0yMi42bTg5LjItNjloMjN2NDYuMmgzMi42VjY5aC01NS42Ii8+CiAgPHBhdGggY2xhc3M9ImpwLWljb24tc2VsZWN0YWJsZSIgZmlsbD0iI2U0NGQyNiIgZD0iTTEwNy42IDQ3MWwtMzMtMzcwLjRoMzYyLjhsLTMzIDM3MC4yTDI1NS43IDUxMiIvPgogIDxwYXRoIGNsYXNzPSJqcC1pY29uLXNlbGVjdGFibGUiIGZpbGw9IiNmMTY1MjkiIGQ9Ik0yNTYgNDgwLjVWMTMxaDE0OC4zTDM3NiA0NDciLz4KICA8cGF0aCBjbGFzcz0ianAtaWNvbi1zZWxlY3RhYmxlLWludmVyc2UiIGZpbGw9IiNlYmViZWIiIGQ9Ik0xNDIgMTc2LjNoMTE0djQ1LjRoLTY0LjJsNC4yIDQ2LjVoNjB2NDUuM0gxNTQuNG0yIDIyLjhIMjAybDMuMiAzNi4zIDUwLjggMTMuNnY0Ny40bC05My4yLTI2Ii8+CiAgPHBhdGggY2xhc3M9ImpwLWljb24tc2VsZWN0YWJsZS1pbnZlcnNlIiBmaWxsPSIjZmZmIiBkPSJNMzY5LjYgMTc2LjNIMjU1Ljh2NDUuNGgxMDkuNm0tNC4xIDQ2LjVIMjU1Ljh2NDUuNGg1NmwtNS4zIDU5LTUwLjcgMTMuNnY0Ny4ybDkzLTI1LjgiLz4KPC9zdmc+Cg==);
  --jp-icon-image: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8cGF0aCBjbGFzcz0ianAtaWNvbi1icmFuZDQganAtaWNvbi1zZWxlY3RhYmxlLWludmVyc2UiIGZpbGw9IiNGRkYiIGQ9Ik0yLjIgMi4yaDE3LjV2MTcuNUgyLjJ6Ii8+CiAgPHBhdGggY2xhc3M9ImpwLWljb24tYnJhbmQwIGpwLWljb24tc2VsZWN0YWJsZSIgZmlsbD0iIzNGNTFCNSIgZD0iTTIuMiAyLjJ2MTcuNWgxNy41bC4xLTE3LjVIMi4yem0xMi4xIDIuMmMxLjIgMCAyLjIgMSAyLjIgMi4ycy0xIDIuMi0yLjIgMi4yLTIuMi0xLTIuMi0yLjIgMS0yLjIgMi4yLTIuMnpNNC40IDE3LjZsMy4zLTguOCAzLjMgNi42IDIuMi0zLjIgNC40IDUuNEg0LjR6Ii8+Cjwvc3ZnPgo=);
  --jp-icon-inspector: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8cGF0aCBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIiBkPSJNMjAgNEg0Yy0xLjEgMC0xLjk5LjktMS45OSAyTDIgMThjMCAxLjEuOSAyIDIgMmgxNmMxLjEgMCAyLS45IDItMlY2YzAtMS4xLS45LTItMi0yem0tNSAxNEg0di00aDExdjR6bTAtNUg0VjloMTF2NHptNSA1aC00VjloNHY5eiIvPgo8L3N2Zz4K);
  --jp-icon-json: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8ZyBjbGFzcz0ianAtaWNvbi13YXJuMSBqcC1pY29uLXNlbGVjdGFibGUiIGZpbGw9IiNGOUE4MjUiPgogICAgPHBhdGggZD0iTTIwLjIgMTEuOGMtMS42IDAtMS43LjUtMS43IDEgMCAuNC4xLjkuMSAxLjMuMS41LjEuOS4xIDEuMyAwIDEuNy0xLjQgMi4zLTMuNSAyLjNoLS45di0xLjloLjVjMS4xIDAgMS40IDAgMS40LS44IDAtLjMgMC0uNi0uMS0xIDAtLjQtLjEtLjgtLjEtMS4yIDAtMS4zIDAtMS44IDEuMy0yLTEuMy0uMi0xLjMtLjctMS4zLTIgMC0uNC4xLS44LjEtMS4yLjEtLjQuMS0uNy4xLTEgMC0uOC0uNC0uNy0xLjQtLjhoLS41VjQuMWguOWMyLjIgMCAzLjUuNyAzLjUgMi4zIDAgLjQtLjEuOS0uMSAxLjMtLjEuNS0uMS45LS4xIDEuMyAwIC41LjIgMSAxLjcgMXYxLjh6TTEuOCAxMC4xYzEuNiAwIDEuNy0uNSAxLjctMSAwLS40LS4xLS45LS4xLTEuMy0uMS0uNS0uMS0uOS0uMS0xLjMgMC0xLjYgMS40LTIuMyAzLjUtMi4zaC45djEuOWgtLjVjLTEgMC0xLjQgMC0xLjQuOCAwIC4zIDAgLjYuMSAxIDAgLjIuMS42LjEgMSAwIDEuMyAwIDEuOC0xLjMgMkM2IDExLjIgNiAxMS43IDYgMTNjMCAuNC0uMS44LS4xIDEuMi0uMS4zLS4xLjctLjEgMSAwIC44LjMuOCAxLjQuOGguNXYxLjloLS45Yy0yLjEgMC0zLjUtLjYtMy41LTIuMyAwLS40LjEtLjkuMS0xLjMuMS0uNS4xLS45LjEtMS4zIDAtLjUtLjItMS0xLjctMXYtMS45eiIvPgogICAgPGNpcmNsZSBjeD0iMTEiIGN5PSIxMy44IiByPSIyLjEiLz4KICAgIDxjaXJjbGUgY3g9IjExIiBjeT0iOC4yIiByPSIyLjEiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-jupyter-favicon: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTUyIiBoZWlnaHQ9IjE2NSIgdmlld0JveD0iMCAwIDE1MiAxNjUiIHZlcnNpb249IjEuMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbi13YXJuMCIgZmlsbD0iI0YzNzcyNiI+CiAgICA8cGF0aCB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLjA3ODk0NywgMTEwLjU4MjkyNykiIGQ9Ik03NS45NDIyODQyLDI5LjU4MDQ1NjEgQzQzLjMwMjM5NDcsMjkuNTgwNDU2MSAxNC43OTY3ODMyLDE3LjY1MzQ2MzQgMCwwIEM1LjUxMDgzMjExLDE1Ljg0MDY4MjkgMTUuNzgxNTM4OSwyOS41NjY3NzMyIDI5LjM5MDQ5NDcsMzkuMjc4NDE3MSBDNDIuOTk5Nyw0OC45ODk4NTM3IDU5LjI3MzcsNTQuMjA2NzgwNSA3NS45NjA1Nzg5LDU0LjIwNjc4MDUgQzkyLjY0NzQ1NzksNTQuMjA2NzgwNSAxMDguOTIxNDU4LDQ4Ljk4OTg1MzcgMTIyLjUzMDY2MywzOS4yNzg0MTcxIEMxMzYuMTM5NDUzLDI5LjU2Njc3MzIgMTQ2LjQxMDI4NCwxNS44NDA2ODI5IDE1MS45MjExNTgsMCBDMTM3LjA4Nzg2OCwxNy42NTM0NjM0IDEwOC41ODI1ODksMjkuNTgwNDU2MSA3NS45NDIyODQyLDI5LjU4MDQ1NjEgTDc1Ljk0MjI4NDIsMjkuNTgwNDU2MSBaIiAvPgogICAgPHBhdGggdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMC4wMzczNjgsIDAuNzA0ODc4KSIgZD0iTTc1Ljk3ODQ1NzksMjQuNjI2NDA3MyBDMTA4LjYxODc2MywyNC42MjY0MDczIDEzNy4xMjQ0NTgsMzYuNTUzNDQxNSAxNTEuOTIxMTU4LDU0LjIwNjc4MDUgQzE0Ni40MTAyODQsMzguMzY2MjIyIDEzNi4xMzk0NTMsMjQuNjQwMTMxNyAxMjIuNTMwNjYzLDE0LjkyODQ4NzggQzEwOC45MjE0NTgsNS4yMTY4NDM5IDkyLjY0NzQ1NzksMCA3NS45NjA1Nzg5LDAgQzU5LjI3MzcsMCA0Mi45OTk3LDUuMjE2ODQzOSAyOS4zOTA0OTQ3LDE0LjkyODQ4NzggQzE1Ljc4MTUzODksMjQuNjQwMTMxNyA1LjUxMDgzMjExLDM4LjM2NjIyMiAwLDU0LjIwNjc4MDUgQzE0LjgzMzA4MTYsMzYuNTg5OTI5MyA0My4zMzg1Njg0LDI0LjYyNjQwNzMgNzUuOTc4NDU3OSwyNC42MjY0MDczIEw3NS45Nzg0NTc5LDI0LjYyNjQwNzMgWiIgLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-jupyter: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzkiIGhlaWdodD0iNTEiIHZpZXdCb3g9IjAgMCAzOSA1MSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyB0cmFuc2Zvcm09InRyYW5zbGF0ZSgtMTYzOCAtMjI4MSkiPgogICAgPGcgY2xhc3M9ImpwLWljb24td2FybjAiIGZpbGw9IiNGMzc3MjYiPgogICAgICA8cGF0aCB0cmFuc2Zvcm09InRyYW5zbGF0ZSgxNjM5Ljc0IDIzMTEuOTgpIiBkPSJNIDE4LjI2NDYgNy4xMzQxMUMgMTAuNDE0NSA3LjEzNDExIDMuNTU4NzIgNC4yNTc2IDAgMEMgMS4zMjUzOSAzLjgyMDQgMy43OTU1NiA3LjEzMDgxIDcuMDY4NiA5LjQ3MzAzQyAxMC4zNDE3IDExLjgxNTIgMTQuMjU1NyAxMy4wNzM0IDE4LjI2OSAxMy4wNzM0QyAyMi4yODIzIDEzLjA3MzQgMjYuMTk2MyAxMS44MTUyIDI5LjQ2OTQgOS40NzMwM0MgMzIuNzQyNCA3LjEzMDgxIDM1LjIxMjYgMy44MjA0IDM2LjUzOCAwQyAzMi45NzA1IDQuMjU3NiAyNi4xMTQ4IDcuMTM0MTEgMTguMjY0NiA3LjEzNDExWiIvPgogICAgICA8cGF0aCB0cmFuc2Zvcm09InRyYW5zbGF0ZSgxNjM5LjczIDIyODUuNDgpIiBkPSJNIDE4LjI3MzMgNS45MzkzMUMgMjYuMTIzNSA1LjkzOTMxIDMyLjk3OTMgOC44MTU4MyAzNi41MzggMTMuMDczNEMgMzUuMjEyNiA5LjI1MzAzIDMyLjc0MjQgNS45NDI2MiAyOS40Njk0IDMuNjAwNEMgMjYuMTk2MyAxLjI1ODE4IDIyLjI4MjMgMCAxOC4yNjkgMEMgMTQuMjU1NyAwIDEwLjM0MTcgMS4yNTgxOCA3LjA2ODYgMy42MDA0QyAzLjc5NTU2IDUuOTQyNjIgMS4zMjUzOSA5LjI1MzAzIDAgMTMuMDczNEMgMy41Njc0NSA4LjgyNDYzIDEwLjQyMzIgNS45MzkzMSAxOC4yNzMzIDUuOTM5MzFaIi8+CiAgICA8L2c+CiAgICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgICA8cGF0aCB0cmFuc2Zvcm09InRyYW5zbGF0ZSgxNjY5LjMgMjI4MS4zMSkiIGQ9Ik0gNS44OTM1MyAyLjg0NEMgNS45MTg4OSAzLjQzMTY1IDUuNzcwODUgNC4wMTM2NyA1LjQ2ODE1IDQuNTE2NDVDIDUuMTY1NDUgNS4wMTkyMiA0LjcyMTY4IDUuNDIwMTUgNC4xOTI5OSA1LjY2ODUxQyAzLjY2NDMgNS45MTY4OCAzLjA3NDQ0IDYuMDAxNTEgMi40OTgwNSA1LjkxMTcxQyAxLjkyMTY2IDUuODIxOSAxLjM4NDYzIDUuNTYxNyAwLjk1NDg5OCA1LjE2NDAxQyAwLjUyNTE3IDQuNzY2MzMgMC4yMjIwNTYgNC4yNDkwMyAwLjA4MzkwMzcgMy42Nzc1N0MgLTAuMDU0MjQ4MyAzLjEwNjExIC0wLjAyMTIzIDIuNTA2MTcgMC4xNzg3ODEgMS45NTM2NEMgMC4zNzg3OTMgMS40MDExIDAuNzM2ODA5IDAuOTIwODE3IDEuMjA3NTQgMC41NzM1MzhDIDEuNjc4MjYgMC4yMjYyNTkgMi4yNDA1NSAwLjAyNzU5MTkgMi44MjMyNiAwLjAwMjY3MjI5QyAzLjYwMzg5IC0wLjAzMDcxMTUgNC4zNjU3MyAwLjI0OTc4OSA0Ljk0MTQyIDAuNzgyNTUxQyA1LjUxNzExIDEuMzE1MzEgNS44NTk1NiAyLjA1Njc2IDUuODkzNTMgMi44NDRaIi8+CiAgICAgIDxwYXRoIHRyYW5zZm9ybT0idHJhbnNsYXRlKDE2MzkuOCAyMzIzLjgxKSIgZD0iTSA3LjQyNzg5IDMuNTgzMzhDIDcuNDYwMDggNC4zMjQzIDcuMjczNTUgNS4wNTgxOSA2Ljg5MTkzIDUuNjkyMTNDIDYuNTEwMzEgNi4zMjYwNyA1Ljk1MDc1IDYuODMxNTYgNS4yODQxMSA3LjE0NDZDIDQuNjE3NDcgNy40NTc2MyAzLjg3MzcxIDcuNTY0MTQgMy4xNDcwMiA3LjQ1MDYzQyAyLjQyMDMyIDcuMzM3MTIgMS43NDMzNiA3LjAwODcgMS4yMDE4NCA2LjUwNjk1QyAwLjY2MDMyOCA2LjAwNTIgMC4yNzg2MSA1LjM1MjY4IDAuMTA1MDE3IDQuNjMyMDJDIC0wLjA2ODU3NTcgMy45MTEzNSAtMC4wMjYyMzYxIDMuMTU0OTQgMC4yMjY2NzUgMi40NTg1NkMgMC40Nzk1ODcgMS43NjIxNyAwLjkzMTY5NyAxLjE1NzEzIDEuNTI1NzYgMC43MjAwMzNDIDIuMTE5ODMgMC4yODI5MzUgMi44MjkxNCAwLjAzMzQzOTUgMy41NjM4OSAwLjAwMzEzMzQ0QyA0LjU0NjY3IC0wLjAzNzQwMzMgNS41MDUyOSAwLjMxNjcwNiA2LjIyOTYxIDAuOTg3ODM1QyA2Ljk1MzkzIDEuNjU4OTYgNy4zODQ4NCAyLjU5MjM1IDcuNDI3ODkgMy41ODMzOEwgNy40Mjc4OSAzLjU4MzM4WiIvPgogICAgICA8cGF0aCB0cmFuc2Zvcm09InRyYW5zbGF0ZSgxNjM4LjM2IDIyODYuMDYpIiBkPSJNIDIuMjc0NzEgNC4zOTYyOUMgMS44NDM2MyA0LjQxNTA4IDEuNDE2NzEgNC4zMDQ0NSAxLjA0Nzk5IDQuMDc4NDNDIDAuNjc5MjY4IDMuODUyNCAwLjM4NTMyOCAzLjUyMTE0IDAuMjAzMzcxIDMuMTI2NTZDIDAuMDIxNDEzNiAyLjczMTk4IC0wLjA0MDM3OTggMi4yOTE4MyAwLjAyNTgxMTYgMS44NjE4MUMgMC4wOTIwMDMxIDEuNDMxOCAwLjI4MzIwNCAxLjAzMTI2IDAuNTc1MjEzIDAuNzEwODgzQyAwLjg2NzIyMiAwLjM5MDUxIDEuMjQ2OTEgMC4xNjQ3MDggMS42NjYyMiAwLjA2MjA1OTJDIDIuMDg1NTMgLTAuMDQwNTg5NyAyLjUyNTYxIC0wLjAxNTQ3MTQgMi45MzA3NiAwLjEzNDIzNUMgMy4zMzU5MSAwLjI4Mzk0MSAzLjY4NzkyIDAuNTUxNTA1IDMuOTQyMjIgMC45MDMwNkMgNC4xOTY1MiAxLjI1NDYyIDQuMzQxNjkgMS42NzQzNiA0LjM1OTM1IDIuMTA5MTZDIDQuMzgyOTkgMi42OTEwNyA0LjE3Njc4IDMuMjU4NjkgMy43ODU5NyAzLjY4NzQ2QyAzLjM5NTE2IDQuMTE2MjQgMi44NTE2NiA0LjM3MTE2IDIuMjc0NzEgNC4zOTYyOUwgMi4yNzQ3MSA0LjM5NjI5WiIvPgogICAgPC9nPgogIDwvZz4+Cjwvc3ZnPgo=);
  --jp-icon-jupyterlab-wordmark: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyMDAiIHZpZXdCb3g9IjAgMCAxODYwLjggNDc1Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjIiIGZpbGw9IiM0RTRFNEUiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDQ4MC4xMzY0MDEsIDY0LjI3MTQ5MykiPgogICAgPGcgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMC4wMDAwMDAsIDU4Ljg3NTU2NikiPgogICAgICA8ZyB0cmFuc2Zvcm09InRyYW5zbGF0ZSgwLjA4NzYwMywgMC4xNDAyOTQpIj4KICAgICAgICA8cGF0aCBkPSJNLTQyNi45LDE2OS44YzAsNDguNy0zLjcsNjQuNy0xMy42LDc2LjRjLTEwLjgsMTAtMjUsMTUuNS0zOS43LDE1LjVsMy43LDI5IGMyMi44LDAuMyw0NC44LTcuOSw2MS45LTIzLjFjMTcuOC0xOC41LDI0LTQ0LjEsMjQtODMuM1YwSC00Mjd2MTcwLjFMLTQyNi45LDE2OS44TC00MjYuOSwxNjkuOHoiLz4KICAgICAgPC9nPgogICAgPC9nPgogICAgPGcgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMTU1LjA0NTI5NiwgNTYuODM3MTA0KSI+CiAgICAgIDxnIHRyYW5zZm9ybT0idHJhbnNsYXRlKDEuNTYyNDUzLCAxLjc5OTg0MikiPgogICAgICAgIDxwYXRoIGQ9Ik0tMzEyLDE0OGMwLDIxLDAsMzkuNSwxLjcsNTUuNGgtMzEuOGwtMi4xLTMzLjNoLTAuOGMtNi43LDExLjYtMTYuNCwyMS4zLTI4LDI3LjkgYy0xMS42LDYuNi0yNC44LDEwLTM4LjIsOS44Yy0zMS40LDAtNjktMTcuNy02OS04OVYwaDM2LjR2MTEyLjdjMCwzOC43LDExLjYsNjQuNyw0NC42LDY0LjdjMTAuMy0wLjIsMjAuNC0zLjUsMjguOS05LjQgYzguNS01LjksMTUuMS0xNC4zLDE4LjktMjMuOWMyLjItNi4xLDMuMy0xMi41LDMuMy0xOC45VjAuMmgzNi40VjE0OEgtMzEyTC0zMTIsMTQ4eiIvPgogICAgICA8L2c+CiAgICA8L2c+CiAgICA8ZyB0cmFuc2Zvcm09InRyYW5zbGF0ZSgzOTAuMDEzMzIyLCA1My40Nzk2MzgpIj4KICAgICAgPGcgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMS43MDY0NTgsIDAuMjMxNDI1KSI+CiAgICAgICAgPHBhdGggZD0iTS00NzguNiw3MS40YzAtMjYtMC44LTQ3LTEuNy02Ni43aDMyLjdsMS43LDM0LjhoMC44YzcuMS0xMi41LDE3LjUtMjIuOCwzMC4xLTI5LjcgYzEyLjUtNywyNi43LTEwLjMsNDEtOS44YzQ4LjMsMCw4NC43LDQxLjcsODQuNywxMDMuM2MwLDczLjEtNDMuNywxMDkuMi05MSwxMDkuMmMtMTIuMSwwLjUtMjQuMi0yLjItMzUtNy44IGMtMTAuOC01LjYtMTkuOS0xMy45LTI2LjYtMjQuMmgtMC44VjI5MWgtMzZ2LTIyMEwtNDc4LjYsNzEuNEwtNDc4LjYsNzEuNHogTS00NDIuNiwxMjUuNmMwLjEsNS4xLDAuNiwxMC4xLDEuNywxNS4xIGMzLDEyLjMsOS45LDIzLjMsMTkuOCwzMS4xYzkuOSw3LjgsMjIuMSwxMi4xLDM0LjcsMTIuMWMzOC41LDAsNjAuNy0zMS45LDYwLjctNzguNWMwLTQwLjctMjEuMS03NS42LTU5LjUtNzUuNiBjLTEyLjksMC40LTI1LjMsNS4xLTM1LjMsMTMuNGMtOS45LDguMy0xNi45LDE5LjctMTkuNiwzMi40Yy0xLjUsNC45LTIuMywxMC0yLjUsMTUuMVYxMjUuNkwtNDQyLjYsMTI1LjZMLTQ0Mi42LDEyNS42eiIvPgogICAgICA8L2c+CiAgICA8L2c+CiAgICA8ZyB0cmFuc2Zvcm09InRyYW5zbGF0ZSg2MDYuNzQwNzI2LCA1Ni44MzcxMDQpIj4KICAgICAgPGcgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMC43NTEyMjYsIDEuOTg5Mjk5KSI+CiAgICAgICAgPHBhdGggZD0iTS00NDAuOCwwbDQzLjcsMTIwLjFjNC41LDEzLjQsOS41LDI5LjQsMTIuOCw0MS43aDAuOGMzLjctMTIuMiw3LjktMjcuNywxMi44LTQyLjQgbDM5LjctMTE5LjJoMzguNUwtMzQ2LjksMTQ1Yy0yNiw2OS43LTQzLjcsMTA1LjQtNjguNiwxMjcuMmMtMTIuNSwxMS43LTI3LjksMjAtNDQuNiwyMy45bC05LjEtMzEuMSBjMTEuNy0zLjksMjIuNS0xMC4xLDMxLjgtMTguMWMxMy4yLTExLjEsMjMuNy0yNS4yLDMwLjYtNDEuMmMxLjUtMi44LDIuNS01LjcsMi45LTguOGMtMC4zLTMuMy0xLjItNi42LTIuNS05LjdMLTQ4MC4yLDAuMSBoMzkuN0wtNDQwLjgsMEwtNDQwLjgsMHoiLz4KICAgICAgPC9nPgogICAgPC9nPgogICAgPGcgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoODIyLjc0ODEwNCwgMC4wMDAwMDApIj4KICAgICAgPGcgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMS40NjQwNTAsIDAuMzc4OTE0KSI+CiAgICAgICAgPHBhdGggZD0iTS00MTMuNywwdjU4LjNoNTJ2MjguMmgtNTJWMTk2YzAsMjUsNywzOS41LDI3LjMsMzkuNWM3LjEsMC4xLDE0LjItMC43LDIxLjEtMi41IGwxLjcsMjcuN2MtMTAuMywzLjctMjEuMyw1LjQtMzIuMiw1Yy03LjMsMC40LTE0LjYtMC43LTIxLjMtMy40Yy02LjgtMi43LTEyLjktNi44LTE3LjktMTIuMWMtMTAuMy0xMC45LTE0LjEtMjktMTQuMS01Mi45IFY4Ni41aC0zMVY1OC4zaDMxVjkuNkwtNDEzLjcsMEwtNDEzLjcsMHoiLz4KICAgICAgPC9nPgogICAgPC9nPgogICAgPGcgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoOTc0LjQzMzI4NiwgNTMuNDc5NjM4KSI+CiAgICAgIDxnIHRyYW5zZm9ybT0idHJhbnNsYXRlKDAuOTkwMDM0LCAwLjYxMDMzOSkiPgogICAgICAgIDxwYXRoIGQ9Ik0tNDQ1LjgsMTEzYzAuOCw1MCwzMi4yLDcwLjYsNjguNiw3MC42YzE5LDAuNiwzNy45LTMsNTUuMy0xMC41bDYuMiwyNi40IGMtMjAuOSw4LjktNDMuNSwxMy4xLTY2LjIsMTIuNmMtNjEuNSwwLTk4LjMtNDEuMi05OC4zLTEwMi41Qy00ODAuMiw0OC4yLTQ0NC43LDAtMzg2LjUsMGM2NS4yLDAsODIuNyw1OC4zLDgyLjcsOTUuNyBjLTAuMSw1LjgtMC41LDExLjUtMS4yLDE3LjJoLTE0MC42SC00NDUuOEwtNDQ1LjgsMTEzeiBNLTMzOS4yLDg2LjZjMC40LTIzLjUtOS41LTYwLjEtNTAuNC02MC4xIGMtMzYuOCwwLTUyLjgsMzQuNC01NS43LDYwLjFILTMzOS4yTC0zMzkuMiw4Ni42TC0zMzkuMiw4Ni42eiIvPgogICAgICA8L2c+CiAgICA8L2c+CiAgICA8ZyB0cmFuc2Zvcm09InRyYW5zbGF0ZSgxMjAxLjk2MTA1OCwgNTMuNDc5NjM4KSI+CiAgICAgIDxnIHRyYW5zZm9ybT0idHJhbnNsYXRlKDEuMTc5NjQwLCAwLjcwNTA2OCkiPgogICAgICAgIDxwYXRoIGQ9Ik0tNDc4LjYsNjhjMC0yMy45LTAuNC00NC41LTEuNy02My40aDMxLjhsMS4yLDM5LjloMS43YzkuMS0yNy4zLDMxLTQ0LjUsNTUuMy00NC41IGMzLjUtMC4xLDcsMC40LDEwLjMsMS4ydjM0LjhjLTQuMS0wLjktOC4yLTEuMy0xMi40LTEuMmMtMjUuNiwwLTQzLjcsMTkuNy00OC43LDQ3LjRjLTEsNS43LTEuNiwxMS41LTEuNywxNy4ydjEwOC4zaC0zNlY2OCBMLTQ3OC42LDY4eiIvPgogICAgICA8L2c+CiAgICA8L2c+CiAgPC9nPgoKICA8ZyBjbGFzcz0ianAtaWNvbi13YXJuMCIgZmlsbD0iI0YzNzcyNiI+CiAgICA8cGF0aCBkPSJNMTM1Mi4zLDMyNi4yaDM3VjI4aC0zN1YzMjYuMnogTTE2MDQuOCwzMjYuMmMtMi41LTEzLjktMy40LTMxLjEtMy40LTQ4Ljd2LTc2IGMwLTQwLjctMTUuMS04My4xLTc3LjMtODMuMWMtMjUuNiwwLTUwLDcuMS02Ni44LDE4LjFsOC40LDI0LjRjMTQuMy05LjIsMzQtMTUuMSw1My0xNS4xYzQxLjYsMCw0Ni4yLDMwLjIsNDYuMiw0N3Y0LjIgYy03OC42LTAuNC0xMjIuMywyNi41LTEyMi4zLDc1LjZjMCwyOS40LDIxLDU4LjQsNjIuMiw1OC40YzI5LDAsNTAuOS0xNC4zLDYyLjItMzAuMmgxLjNsMi45LDI1LjZIMTYwNC44eiBNMTU2NS43LDI1Ny43IGMwLDMuOC0wLjgsOC0yLjEsMTEuOGMtNS45LDE3LjItMjIuNywzNC00OS4yLDM0Yy0xOC45LDAtMzQuOS0xMS4zLTM0LjktMzUuM2MwLTM5LjUsNDUuOC00Ni42LDg2LjItNDUuOFYyNTcuN3ogTTE2OTguNSwzMjYuMiBsMS43LTMzLjZoMS4zYzE1LjEsMjYuOSwzOC43LDM4LjIsNjguMSwzOC4yYzQ1LjQsMCw5MS4yLTM2LjEsOTEuMi0xMDguOGMwLjQtNjEuNy0zNS4zLTEwMy43LTg1LjctMTAzLjcgYy0zMi44LDAtNTYuMywxNC43LTY5LjMsMzcuNGgtMC44VjI4aC0zNi42djI0NS43YzAsMTguMS0wLjgsMzguNi0xLjcsNTIuNUgxNjk4LjV6IE0xNzA0LjgsMjA4LjJjMC01LjksMS4zLTEwLjksMi4xLTE1LjEgYzcuNi0yOC4xLDMxLjEtNDUuNCw1Ni4zLTQ1LjRjMzkuNSwwLDYwLjUsMzQuOSw2MC41LDc1LjZjMCw0Ni42LTIzLjEsNzguMS02MS44LDc4LjFjLTI2LjksMC00OC4zLTE3LjYtNTUuNS00My4zIGMtMC44LTQuMi0xLjctOC44LTEuNy0xMy40VjIwOC4yeiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-kernel: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICAgIDxwYXRoIGNsYXNzPSJqcC1pY29uMiIgZmlsbD0iIzYxNjE2MSIgZD0iTTE1IDlIOXY2aDZWOXptLTIgNGgtMnYtMmgydjJ6bTgtMlY5aC0yVjdjMC0xLjEtLjktMi0yLTJoLTJWM2gtMnYyaC0yVjNIOXYySDdjLTEuMSAwLTIgLjktMiAydjJIM3YyaDJ2MkgzdjJoMnYyYzAgMS4xLjkgMiAyIDJoMnYyaDJ2LTJoMnYyaDJ2LTJoMmMxLjEgMCAyLS45IDItMnYtMmgydi0yaC0ydi0yaDJ6bS00IDZIN1Y3aDEwdjEweiIvPgo8L3N2Zz4K);
  --jp-icon-keyboard: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8cGF0aCBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIiBkPSJNMjAgNUg0Yy0xLjEgMC0xLjk5LjktMS45OSAyTDIgMTdjMCAxLjEuOSAyIDIgMmgxNmMxLjEgMCAyLS45IDItMlY3YzAtMS4xLS45LTItMi0yem0tOSAzaDJ2MmgtMlY4em0wIDNoMnYyaC0ydi0yek04IDhoMnYySDhWOHptMCAzaDJ2Mkg4di0yem0tMSAySDV2LTJoMnYyem0wLTNINVY4aDJ2MnptOSA3SDh2LTJoOHYyem0wLTRoLTJ2LTJoMnYyem0wLTNoLTJWOGgydjJ6bTMgM2gtMnYtMmgydjJ6bTAtM2gtMlY4aDJ2MnoiLz4KPC9zdmc+Cg==);
  --jp-icon-launcher: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8cGF0aCBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIiBkPSJNMTkgMTlINVY1aDdWM0g1YTIgMiAwIDAwLTIgMnYxNGEyIDIgMCAwMDIgMmgxNGMxLjEgMCAyLS45IDItMnYtN2gtMnY3ek0xNCAzdjJoMy41OWwtOS44MyA5LjgzIDEuNDEgMS40MUwxOSA2LjQxVjEwaDJWM2gtN3oiLz4KPC9zdmc+Cg==);
  --jp-icon-line-form: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICAgIDxwYXRoIGZpbGw9IndoaXRlIiBkPSJNNS44OCA0LjEyTDEzLjc2IDEybC03Ljg4IDcuODhMOCAyMmwxMC0xMEw4IDJ6Ii8+Cjwvc3ZnPgo=);
  --jp-icon-link: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTMuOSAxMmMwLTEuNzEgMS4zOS0zLjEgMy4xLTMuMWg0VjdIN2MtMi43NiAwLTUgMi4yNC01IDVzMi4yNCA1IDUgNWg0di0xLjlIN2MtMS43MSAwLTMuMS0xLjM5LTMuMS0zLjF6TTggMTNoOHYtMkg4djJ6bTktNmgtNHYxLjloNGMxLjcxIDAgMy4xIDEuMzkgMy4xIDMuMXMtMS4zOSAzLjEtMy4xIDMuMWgtNFYxN2g0YzIuNzYgMCA1LTIuMjQgNS01cy0yLjI0LTUtNS01eiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-list: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICAgIDxwYXRoIGNsYXNzPSJqcC1pY29uMiBqcC1pY29uLXNlbGVjdGFibGUiIGZpbGw9IiM2MTYxNjEiIGQ9Ik0xOSA1djE0SDVWNWgxNG0xLjEtMkgzLjljLS41IDAtLjkuNC0uOS45djE2LjJjMCAuNC40LjkuOS45aDE2LjJjLjQgMCAuOS0uNS45LS45VjMuOWMwLS41LS41LS45LS45LS45ek0xMSA3aDZ2MmgtNlY3em0wIDRoNnYyaC02di0yem0wIDRoNnYyaC02ek03IDdoMnYySDd6bTAgNGgydjJIN3ptMCA0aDJ2Mkg3eiIvPgo8L3N2Zz4=);
  --jp-icon-listings-info: url(data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iaXNvLTg4NTktMSI/Pg0KPHN2ZyB2ZXJzaW9uPSIxLjEiIGlkPSJDYXBhXzEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHg9IjBweCIgeT0iMHB4Ig0KCSB2aWV3Qm94PSIwIDAgNTAuOTc4IDUwLjk3OCIgc3R5bGU9ImVuYWJsZS1iYWNrZ3JvdW5kOm5ldyAwIDAgNTAuOTc4IDUwLjk3ODsiIHhtbDpzcGFjZT0icHJlc2VydmUiPg0KPGc+DQoJPGc+DQoJCTxnPg0KCQkJPHBhdGggc3R5bGU9ImZpbGw6IzAxMDAwMjsiIGQ9Ik00My41Miw3LjQ1OEMzOC43MTEsMi42NDgsMzIuMzA3LDAsMjUuNDg5LDBDMTguNjcsMCwxMi4yNjYsMi42NDgsNy40NTgsNy40NTgNCgkJCQljLTkuOTQzLDkuOTQxLTkuOTQzLDI2LjExOSwwLDM2LjA2MmM0LjgwOSw0LjgwOSwxMS4yMTIsNy40NTYsMTguMDMxLDcuNDU4YzAsMCwwLjAwMSwwLDAuMDAyLDANCgkJCQljNi44MTYsMCwxMy4yMjEtMi42NDgsMTguMDI5LTcuNDU4YzQuODA5LTQuODA5LDcuNDU3LTExLjIxMiw3LjQ1Ny0xOC4wM0M1MC45NzcsMTguNjcsNDguMzI4LDEyLjI2Niw0My41Miw3LjQ1OHoNCgkJCQkgTTQyLjEwNiw0Mi4xMDVjLTQuNDMyLDQuNDMxLTEwLjMzMiw2Ljg3Mi0xNi42MTUsNi44NzJoLTAuMDAyYy02LjI4NS0wLjAwMS0xMi4xODctMi40NDEtMTYuNjE3LTYuODcyDQoJCQkJYy05LjE2Mi05LjE2My05LjE2Mi0yNC4wNzEsMC0zMy4yMzNDMTMuMzAzLDQuNDQsMTkuMjA0LDIsMjUuNDg5LDJjNi4yODQsMCwxMi4xODYsMi40NCwxNi42MTcsNi44NzINCgkJCQljNC40MzEsNC40MzEsNi44NzEsMTAuMzMyLDYuODcxLDE2LjYxN0M0OC45NzcsMzEuNzcyLDQ2LjUzNiwzNy42NzUsNDIuMTA2LDQyLjEwNXoiLz4NCgkJPC9nPg0KCQk8Zz4NCgkJCTxwYXRoIHN0eWxlPSJmaWxsOiMwMTAwMDI7IiBkPSJNMjMuNTc4LDMyLjIxOGMtMC4wMjMtMS43MzQsMC4xNDMtMy4wNTksMC40OTYtMy45NzJjMC4zNTMtMC45MTMsMS4xMS0xLjk5NywyLjI3Mi0zLjI1Mw0KCQkJCWMwLjQ2OC0wLjUzNiwwLjkyMy0xLjA2MiwxLjM2Ny0xLjU3NWMwLjYyNi0wLjc1MywxLjEwNC0xLjQ3OCwxLjQzNi0yLjE3NWMwLjMzMS0wLjcwNywwLjQ5NS0xLjU0MSwwLjQ5NS0yLjUNCgkJCQljMC0xLjA5Ni0wLjI2LTIuMDg4LTAuNzc5LTIuOTc5Yy0wLjU2NS0wLjg3OS0xLjUwMS0xLjMzNi0yLjgwNi0xLjM2OWMtMS44MDIsMC4wNTctMi45ODUsMC42NjctMy41NSwxLjgzMg0KCQkJCWMtMC4zMDEsMC41MzUtMC41MDMsMS4xNDEtMC42MDcsMS44MTRjLTAuMTM5LDAuNzA3LTAuMjA3LDEuNDMyLTAuMjA3LDIuMTc0aC0yLjkzN2MtMC4wOTEtMi4yMDgsMC40MDctNC4xMTQsMS40OTMtNS43MTkNCgkJCQljMS4wNjItMS42NCwyLjg1NS0yLjQ4MSw1LjM3OC0yLjUyN2MyLjE2LDAuMDIzLDMuODc0LDAuNjA4LDUuMTQxLDEuNzU4YzEuMjc4LDEuMTYsMS45MjksMi43NjQsMS45NSw0LjgxMQ0KCQkJCWMwLDEuMTQyLTAuMTM3LDIuMTExLTAuNDEsMi45MTFjLTAuMzA5LDAuODQ1LTAuNzMxLDEuNTkzLTEuMjY4LDIuMjQzYy0wLjQ5MiwwLjY1LTEuMDY4LDEuMzE4LTEuNzMsMi4wMDINCgkJCQljLTAuNjUsMC42OTctMS4zMTMsMS40NzktMS45ODcsMi4zNDZjLTAuMjM5LDAuMzc3LTAuNDI5LDAuNzc3LTAuNTY1LDEuMTk5Yy0wLjE2LDAuOTU5LTAuMjE3LDEuOTUxLTAuMTcxLDIuOTc5DQoJCQkJQzI2LjU4OSwzMi4yMTgsMjMuNTc4LDMyLjIxOCwyMy41NzgsMzIuMjE4eiBNMjMuNTc4LDM4LjIydi0zLjQ4NGgzLjA3NnYzLjQ4NEgyMy41Nzh6Ii8+DQoJCTwvZz4NCgk8L2c+DQo8L2c+DQo8Zz4NCjwvZz4NCjxnPg0KPC9nPg0KPGc+DQo8L2c+DQo8Zz4NCjwvZz4NCjxnPg0KPC9nPg0KPGc+DQo8L2c+DQo8Zz4NCjwvZz4NCjxnPg0KPC9nPg0KPGc+DQo8L2c+DQo8Zz4NCjwvZz4NCjxnPg0KPC9nPg0KPGc+DQo8L2c+DQo8Zz4NCjwvZz4NCjxnPg0KPC9nPg0KPGc+DQo8L2c+DQo8L3N2Zz4NCg==);
  --jp-icon-markdown: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8cGF0aCBjbGFzcz0ianAtaWNvbi1jb250cmFzdDAganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjN0IxRkEyIiBkPSJNNSAxNC45aDEybC02LjEgNnptOS40LTYuOGMwLTEuMy0uMS0yLjktLjEtNC41LS40IDEuNC0uOSAyLjktMS4zIDQuM2wtMS4zIDQuM2gtMkw4LjUgNy45Yy0uNC0xLjMtLjctMi45LTEtNC4zLS4xIDEuNi0uMSAzLjItLjIgNC42TDcgMTIuNEg0LjhsLjctMTFoMy4zTDEwIDVjLjQgMS4yLjcgMi43IDEgMy45LjMtMS4yLjctMi42IDEtMy45bDEuMi0zLjdoMy4zbC42IDExaC0yLjRsLS4zLTQuMnoiLz4KPC9zdmc+Cg==);
  --jp-icon-new-folder: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTIwIDZoLThsLTItMkg0Yy0xLjExIDAtMS45OS44OS0xLjk5IDJMMiAxOGMwIDEuMTEuODkgMiAyIDJoMTZjMS4xMSAwIDItLjg5IDItMlY4YzAtMS4xMS0uODktMi0yLTJ6bS0xIDhoLTN2M2gtMnYtM2gtM3YtMmgzVjloMnYzaDN2MnoiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-not-trusted: url(data:image/svg+xml;base64,PHN2ZyBmaWxsPSJub25lIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI1IDI1Ij4KICAgIDxwYXRoIGNsYXNzPSJqcC1pY29uMiIgc3Ryb2tlPSIjMzMzMzMzIiBzdHJva2Utd2lkdGg9IjIiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDMgMykiIGQ9Ik0xLjg2MDk0IDExLjQ0MDlDMC44MjY0NDggOC43NzAyNyAwLjg2Mzc3OSA2LjA1NzY0IDEuMjQ5MDcgNC4xOTkzMkMyLjQ4MjA2IDMuOTMzNDcgNC4wODA2OCAzLjQwMzQ3IDUuNjAxMDIgMi44NDQ5QzcuMjM1NDkgMi4yNDQ0IDguODU2NjYgMS41ODE1IDkuOTg3NiAxLjA5NTM5QzExLjA1OTcgMS41ODM0MSAxMi42MDk0IDIuMjQ0NCAxNC4yMTggMi44NDMzOUMxNS43NTAzIDMuNDEzOTQgMTcuMzk5NSAzLjk1MjU4IDE4Ljc1MzkgNC4yMTM4NUMxOS4xMzY0IDYuMDcxNzcgMTkuMTcwOSA4Ljc3NzIyIDE4LjEzOSAxMS40NDA5QzE3LjAzMDMgMTQuMzAzMiAxNC42NjY4IDE3LjE4NDQgOS45OTk5OSAxOC45MzU0QzUuMzMzMTkgMTcuMTg0NCAyLjk2OTY4IDE0LjMwMzIgMS44NjA5NCAxMS40NDA5WiIvPgogICAgPHBhdGggY2xhc3M9ImpwLWljb24yIiBzdHJva2U9IiMzMzMzMzMiIHN0cm9rZS13aWR0aD0iMiIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoOS4zMTU5MiA5LjMyMDMxKSIgZD0iTTcuMzY4NDIgMEwwIDcuMzY0NzkiLz4KICAgIDxwYXRoIGNsYXNzPSJqcC1pY29uMiIgc3Ryb2tlPSIjMzMzMzMzIiBzdHJva2Utd2lkdGg9IjIiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDkuMzE1OTIgMTYuNjgzNikgc2NhbGUoMSAtMSkiIGQ9Ik03LjM2ODQyIDBMMCA3LjM2NDc5Ii8+Cjwvc3ZnPgo=);
  --jp-icon-notebook: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8ZyBjbGFzcz0ianAtaWNvbi13YXJuMCBqcC1pY29uLXNlbGVjdGFibGUiIGZpbGw9IiNFRjZDMDAiPgogICAgPHBhdGggZD0iTTE4LjcgMy4zdjE1LjRIMy4zVjMuM2gxNS40bTEuNS0xLjVIMS44djE4LjNoMTguM2wuMS0xOC4zeiIvPgogICAgPHBhdGggZD0iTTE2LjUgMTYuNWwtNS40LTQuMy01LjYgNC4zdi0xMWgxMXoiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-numbering: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjIiIGhlaWdodD0iMjIiIHZpZXdCb3g9IjAgMCAyOCAyOCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KCTxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CgkJPHBhdGggZD0iTTQgMTlINlYxOS41SDVWMjAuNUg2VjIxSDRWMjJIN1YxOEg0VjE5Wk01IDEwSDZWNkg0VjdINVYxMFpNNCAxM0g1LjhMNCAxNS4xVjE2SDdWMTVINS4yTDcgMTIuOVYxMkg0VjEzWk05IDdWOUgyM1Y3SDlaTTkgMjFIMjNWMTlIOVYyMVpNOSAxNUgyM1YxM0g5VjE1WiIvPgoJPC9nPgo8L3N2Zz4K);
  --jp-icon-offline-bolt: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjE2Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTEyIDIuMDJjLTUuNTEgMC05Ljk4IDQuNDctOS45OCA5Ljk4czQuNDcgOS45OCA5Ljk4IDkuOTggOS45OC00LjQ3IDkuOTgtOS45OFMxNy41MSAyLjAyIDEyIDIuMDJ6TTExLjQ4IDIwdi02LjI2SDhMMTMgNHY2LjI2aDMuMzVMMTEuNDggMjB6Ii8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-palette: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTE4IDEzVjIwSDRWNkg5LjAyQzkuMDcgNS4yOSA5LjI0IDQuNjIgOS41IDRINEMyLjkgNCAyIDQuOSAyIDZWMjBDMiAyMS4xIDIuOSAyMiA0IDIySDE4QzE5LjEgMjIgMjAgMjEuMSAyMCAyMFYxNUwxOCAxM1pNMTkuMyA4Ljg5QzE5Ljc0IDguMTkgMjAgNy4zOCAyMCA2LjVDMjAgNC4wMSAxNy45OSAyIDE1LjUgMkMxMy4wMSAyIDExIDQuMDEgMTEgNi41QzExIDguOTkgMTMuMDEgMTEgMTUuNDkgMTFDMTYuMzcgMTEgMTcuMTkgMTAuNzQgMTcuODggMTAuM0wyMSAxMy40MkwyMi40MiAxMkwxOS4zIDguODlaTTE1LjUgOUMxNC4xMiA5IDEzIDcuODggMTMgNi41QzEzIDUuMTIgMTQuMTIgNCAxNS41IDRDMTYuODggNCAxOCA1LjEyIDE4IDYuNUMxOCA3Ljg4IDE2Ljg4IDkgMTUuNSA5WiIvPgogICAgPHBhdGggZmlsbC1ydWxlPSJldmVub2RkIiBjbGlwLXJ1bGU9ImV2ZW5vZGQiIGQ9Ik00IDZIOS4wMTg5NEM5LjAwNjM5IDYuMTY1MDIgOSA2LjMzMTc2IDkgNi41QzkgOC44MTU3NyAxMC4yMTEgMTAuODQ4NyAxMi4wMzQzIDEySDlWMTRIMTZWMTIuOTgxMUMxNi41NzAzIDEyLjkzNzcgMTcuMTIgMTIuODIwNyAxNy42Mzk2IDEyLjYzOTZMMTggMTNWMjBINFY2Wk04IDhINlYxMEg4VjhaTTYgMTJIOFYxNEg2VjEyWk04IDE2SDZWMThIOFYxNlpNOSAxNkgxNlYxOEg5VjE2WiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-paste: url(data:image/svg+xml;base64,PHN2ZyBoZWlnaHQ9IjI0IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGggZD0iTTE5IDJoLTQuMThDMTQuNC44NCAxMy4zIDAgMTIgMGMtMS4zIDAtMi40Ljg0LTIuODIgMkg1Yy0xLjEgMC0yIC45LTIgMnYxNmMwIDEuMS45IDIgMiAyaDE0YzEuMSAwIDItLjkgMi0yVjRjMC0xLjEtLjktMi0yLTJ6bS03IDBjLjU1IDAgMSAuNDUgMSAxcy0uNDUgMS0xIDEtMS0uNDUtMS0xIC40NS0xIDEtMXptNyAxOEg1VjRoMnYzaDEwVjRoMnYxNnoiLz4KICAgIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-pdf: url(data:image/svg+xml;base64,PHN2ZwogICB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyMiAyMiIgd2lkdGg9IjE2Ij4KICAgIDxwYXRoIHRyYW5zZm9ybT0icm90YXRlKDQ1KSIgY2xhc3M9ImpwLWljb24tc2VsZWN0YWJsZSIgZmlsbD0iI0ZGMkEyQSIKICAgICAgIGQ9Im0gMjIuMzQ0MzY5LC0zLjAxNjM2NDIgaCA1LjYzODYwNCB2IDEuNTc5MjQzMyBoIC0zLjU0OTIyNyB2IDEuNTA4NjkyOTkgaCAzLjMzNzU3NiBWIDEuNjUwODE1NCBoIC0zLjMzNzU3NiB2IDMuNDM1MjYxMyBoIC0yLjA4OTM3NyB6IG0gLTcuMTM2NDQ0LDEuNTc5MjQzMyB2IDQuOTQzOTU0MyBoIDAuNzQ4OTIgcSAxLjI4MDc2MSwwIDEuOTUzNzAzLC0wLjYzNDk1MzUgMC42NzgzNjksLTAuNjM0OTUzNSAwLjY3ODM2OSwtMS44NDUxNjQxIDAsLTEuMjA0NzgzNTUgLTAuNjcyOTQyLC0xLjgzNDMxMDExIC0wLjY3Mjk0MiwtMC42Mjk1MjY1OSAtMS45NTkxMywtMC42Mjk1MjY1OSB6IG0gLTIuMDg5Mzc3LC0xLjU3OTI0MzMgaCAyLjIwMzM0MyBxIDEuODQ1MTY0LDAgMi43NDYwMzksMC4yNjU5MjA3IDAuOTA2MzAxLDAuMjYwNDkzNyAxLjU1MjEwOCwwLjg5MDAyMDMgMC41Njk4MywwLjU0ODEyMjMgMC44NDY2MDUsMS4yNjQ0ODAwNiAwLjI3Njc3NCwwLjcxNjM1NzgxIDAuMjc2Nzc0LDEuNjIyNjU4OTQgMCwwLjkxNzE1NTEgLTAuMjc2Nzc0LDEuNjM4OTM5OSAtMC4yNzY3NzUsMC43MTYzNTc4IC0wLjg0NjYwNSwxLjI2NDQ4IC0wLjY1MTIzNCwwLjYyOTUyNjYgLTEuNTYyOTYyLDAuODk1NDQ3MyAtMC45MTE3MjgsMC4yNjA0OTM3IC0yLjczNTE4NSwwLjI2MDQ5MzcgaCAtMi4yMDMzNDMgeiBtIC04LjE0NTg1NjUsMCBoIDMuNDY3ODIzIHEgMS41NDY2ODE2LDAgMi4zNzE1Nzg1LDAuNjg5MjIzIDAuODMwMzI0LDAuNjgzNzk2MSAwLjgzMDMyNCwxLjk1MzcwMzE0IDAsMS4yNzUzMzM5NyAtMC44MzAzMjQsMS45NjQ1NTcwNiBRIDkuOTg3MTk2MSwyLjI3NDkxNSA4LjQ0MDUxNDUsMi4yNzQ5MTUgSCA3LjA2MjA2ODQgViA1LjA4NjA3NjcgSCA0Ljk3MjY5MTUgWiBtIDIuMDg5Mzc2OSwxLjUxNDExOTkgdiAyLjI2MzAzOTQzIGggMS4xNTU5NDEgcSAwLjYwNzgxODgsMCAwLjkzODg2MjksLTAuMjkzMDU1NDcgMC4zMzEwNDQxLC0wLjI5ODQ4MjQxIDAuMzMxMDQ0MSwtMC44NDExNzc3MiAwLC0wLjU0MjY5NTMxIC0wLjMzMTA0NDEsLTAuODM1NzUwNzQgLTAuMzMxMDQ0MSwtMC4yOTMwNTU1IC0wLjkzODg2MjksLTAuMjkzMDU1NSB6IgovPgo8L3N2Zz4K);
  --jp-icon-python: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8ZyBjbGFzcz0ianAtaWNvbi1icmFuZDAganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjMEQ0N0ExIj4KICAgIDxwYXRoIGQ9Ik0xMS4xIDYuOVY1LjhINi45YzAtLjUgMC0xLjMuMi0xLjYuNC0uNy44LTEuMSAxLjctMS40IDEuNy0uMyAyLjUtLjMgMy45LS4xIDEgLjEgMS45LjkgMS45IDEuOXY0LjJjMCAuNS0uOSAxLjYtMiAxLjZIOC44Yy0xLjUgMC0yLjQgMS40LTIuNCAyLjh2Mi4ySDQuN0MzLjUgMTUuMSAzIDE0IDMgMTMuMVY5Yy0uMS0xIC42LTIgMS44LTIgMS41LS4xIDYuMy0uMSA2LjMtLjF6Ii8+CiAgICA8cGF0aCBkPSJNMTAuOSAxNS4xdjEuMWg0LjJjMCAuNSAwIDEuMy0uMiAxLjYtLjQuNy0uOCAxLjEtMS43IDEuNC0xLjcuMy0yLjUuMy0zLjkuMS0xLS4xLTEuOS0uOS0xLjktMS45di00LjJjMC0uNS45LTEuNiAyLTEuNmgzLjhjMS41IDAgMi40LTEuNCAyLjQtMi44VjYuNmgxLjdDMTguNSA2LjkgMTkgOCAxOSA4LjlWMTNjMCAxLS43IDIuMS0xLjkgMi4xaC02LjJ6Ii8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-r-kernel: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8cGF0aCBjbGFzcz0ianAtaWNvbi1jb250cmFzdDMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjMjE5NkYzIiBkPSJNNC40IDIuNWMxLjItLjEgMi45LS4zIDQuOS0uMyAyLjUgMCA0LjEuNCA1LjIgMS4zIDEgLjcgMS41IDEuOSAxLjUgMy41IDAgMi0xLjQgMy41LTIuOSA0LjEgMS4yLjQgMS43IDEuNiAyLjIgMyAuNiAxLjkgMSAzLjkgMS4zIDQuNmgtMy44Yy0uMy0uNC0uOC0xLjctMS4yLTMuN3MtMS4yLTIuNi0yLjYtMi42aC0uOXY2LjRINC40VjIuNXptMy43IDYuOWgxLjRjMS45IDAgMi45LS45IDIuOS0yLjNzLTEtMi4zLTIuOC0yLjNjLS43IDAtMS4zIDAtMS42LjJ2NC41aC4xdi0uMXoiLz4KPC9zdmc+Cg==);
  --jp-icon-react: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMTUwIDE1MCA1NDEuOSAyOTUuMyI+CiAgPGcgY2xhc3M9ImpwLWljb24tYnJhbmQyIGpwLWljb24tc2VsZWN0YWJsZSIgZmlsbD0iIzYxREFGQiI+CiAgICA8cGF0aCBkPSJNNjY2LjMgMjk2LjVjMC0zMi41LTQwLjctNjMuMy0xMDMuMS04Mi40IDE0LjQtNjMuNiA4LTExNC4yLTIwLjItMTMwLjQtNi41LTMuOC0xNC4xLTUuNi0yMi40LTUuNnYyMi4zYzQuNiAwIDguMy45IDExLjQgMi42IDEzLjYgNy44IDE5LjUgMzcuNSAxNC45IDc1LjctMS4xIDkuNC0yLjkgMTkuMy01LjEgMjkuNC0xOS42LTQuOC00MS04LjUtNjMuNS0xMC45LTEzLjUtMTguNS0yNy41LTM1LjMtNDEuNi01MCAzMi42LTMwLjMgNjMuMi00Ni45IDg0LTQ2LjlWNzhjLTI3LjUgMC02My41IDE5LjYtOTkuOSA1My42LTM2LjQtMzMuOC03Mi40LTUzLjItOTkuOS01My4ydjIyLjNjMjAuNyAwIDUxLjQgMTYuNSA4NCA0Ni42LTE0IDE0LjctMjggMzEuNC00MS4zIDQ5LjktMjIuNiAyLjQtNDQgNi4xLTYzLjYgMTEtMi4zLTEwLTQtMTkuNy01LjItMjktNC43LTM4LjIgMS4xLTY3LjkgMTQuNi03NS44IDMtMS44IDYuOS0yLjYgMTEuNS0yLjZWNzguNWMtOC40IDAtMTYgMS44LTIyLjYgNS42LTI4LjEgMTYuMi0zNC40IDY2LjctMTkuOSAxMzAuMS02Mi4yIDE5LjItMTAyLjcgNDkuOS0xMDIuNyA4Mi4zIDAgMzIuNSA0MC43IDYzLjMgMTAzLjEgODIuNC0xNC40IDYzLjYtOCAxMTQuMiAyMC4yIDEzMC40IDYuNSAzLjggMTQuMSA1LjYgMjIuNSA1LjYgMjcuNSAwIDYzLjUtMTkuNiA5OS45LTUzLjYgMzYuNCAzMy44IDcyLjQgNTMuMiA5OS45IDUzLjIgOC40IDAgMTYtMS44IDIyLjYtNS42IDI4LjEtMTYuMiAzNC40LTY2LjcgMTkuOS0xMzAuMSA2Mi0xOS4xIDEwMi41LTQ5LjkgMTAyLjUtODIuM3ptLTEzMC4yLTY2LjdjLTMuNyAxMi45LTguMyAyNi4yLTEzLjUgMzkuNS00LjEtOC04LjQtMTYtMTMuMS0yNC00LjYtOC05LjUtMTUuOC0xNC40LTIzLjQgMTQuMiAyLjEgMjcuOSA0LjcgNDEgNy45em0tNDUuOCAxMDYuNWMtNy44IDEzLjUtMTUuOCAyNi4zLTI0LjEgMzguMi0xNC45IDEuMy0zMCAyLTQ1LjIgMi0xNS4xIDAtMzAuMi0uNy00NS0xLjktOC4zLTExLjktMTYuNC0yNC42LTI0LjItMzgtNy42LTEzLjEtMTQuNS0yNi40LTIwLjgtMzkuOCA2LjItMTMuNCAxMy4yLTI2LjggMjAuNy0zOS45IDcuOC0xMy41IDE1LjgtMjYuMyAyNC4xLTM4LjIgMTQuOS0xLjMgMzAtMiA0NS4yLTIgMTUuMSAwIDMwLjIuNyA0NSAxLjkgOC4zIDExLjkgMTYuNCAyNC42IDI0LjIgMzggNy42IDEzLjEgMTQuNSAyNi40IDIwLjggMzkuOC02LjMgMTMuNC0xMy4yIDI2LjgtMjAuNyAzOS45em0zMi4zLTEzYzUuNCAxMy40IDEwIDI2LjggMTMuOCAzOS44LTEzLjEgMy4yLTI2LjkgNS45LTQxLjIgOCA0LjktNy43IDkuOC0xNS42IDE0LjQtMjMuNyA0LjYtOCA4LjktMTYuMSAxMy0yNC4xek00MjEuMiA0MzBjLTkuMy05LjYtMTguNi0yMC4zLTI3LjgtMzIgOSAuNCAxOC4yLjcgMjcuNS43IDkuNCAwIDE4LjctLjIgMjcuOC0uNy05IDExLjctMTguMyAyMi40LTI3LjUgMzJ6bS03NC40LTU4LjljLTE0LjItMi4xLTI3LjktNC43LTQxLTcuOSAzLjctMTIuOSA4LjMtMjYuMiAxMy41LTM5LjUgNC4xIDggOC40IDE2IDEzLjEgMjQgNC43IDggOS41IDE1LjggMTQuNCAyMy40ek00MjAuNyAxNjNjOS4zIDkuNiAxOC42IDIwLjMgMjcuOCAzMi05LS40LTE4LjItLjctMjcuNS0uNy05LjQgMC0xOC43LjItMjcuOC43IDktMTEuNyAxOC4zLTIyLjQgMjcuNS0zMnptLTc0IDU4LjljLTQuOSA3LjctOS44IDE1LjYtMTQuNCAyMy43LTQuNiA4LTguOSAxNi0xMyAyNC01LjQtMTMuNC0xMC0yNi44LTEzLjgtMzkuOCAxMy4xLTMuMSAyNi45LTUuOCA0MS4yLTcuOXptLTkwLjUgMTI1LjJjLTM1LjQtMTUuMS01OC4zLTM0LjktNTguMy01MC42IDAtMTUuNyAyMi45LTM1LjYgNTguMy01MC42IDguNi0zLjcgMTgtNyAyNy43LTEwLjEgNS43IDE5LjYgMTMuMiA0MCAyMi41IDYwLjktOS4yIDIwLjgtMTYuNiA0MS4xLTIyLjIgNjAuNi05LjktMy4xLTE5LjMtNi41LTI4LTEwLjJ6TTMxMCA0OTBjLTEzLjYtNy44LTE5LjUtMzcuNS0xNC45LTc1LjcgMS4xLTkuNCAyLjktMTkuMyA1LjEtMjkuNCAxOS42IDQuOCA0MSA4LjUgNjMuNSAxMC45IDEzLjUgMTguNSAyNy41IDM1LjMgNDEuNiA1MC0zMi42IDMwLjMtNjMuMiA0Ni45LTg0IDQ2LjktNC41LS4xLTguMy0xLTExLjMtMi43em0yMzcuMi03Ni4yYzQuNyAzOC4yLTEuMSA2Ny45LTE0LjYgNzUuOC0zIDEuOC02LjkgMi42LTExLjUgMi42LTIwLjcgMC01MS40LTE2LjUtODQtNDYuNiAxNC0xNC43IDI4LTMxLjQgNDEuMy00OS45IDIyLjYtMi40IDQ0LTYuMSA2My42LTExIDIuMyAxMC4xIDQuMSAxOS44IDUuMiAyOS4xem0zOC41LTY2LjdjLTguNiAzLjctMTggNy0yNy43IDEwLjEtNS43LTE5LjYtMTMuMi00MC0yMi41LTYwLjkgOS4yLTIwLjggMTYuNi00MS4xIDIyLjItNjAuNiA5LjkgMy4xIDE5LjMgNi41IDI4LjEgMTAuMiAzNS40IDE1LjEgNTguMyAzNC45IDU4LjMgNTAuNi0uMSAxNS43LTIzIDM1LjYtNTguNCA1MC42ek0zMjAuOCA3OC40eiIvPgogICAgPGNpcmNsZSBjeD0iNDIwLjkiIGN5PSIyOTYuNSIgcj0iNDUuNyIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-redo: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjE2Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgICA8cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZD0iTTE4LjQgMTAuNkMxNi41NSA4Ljk5IDE0LjE1IDggMTEuNSA4Yy00LjY1IDAtOC41OCAzLjAzLTkuOTYgNy4yMkwzLjkgMTZjMS4wNS0zLjE5IDQuMDUtNS41IDcuNi01LjUgMS45NSAwIDMuNzMuNzIgNS4xMiAxLjg4TDEzIDE2aDlWN2wtMy42IDMuNnoiLz4KICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-refresh: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDE4IDE4Ij4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGggZD0iTTkgMTMuNWMtMi40OSAwLTQuNS0yLjAxLTQuNS00LjVTNi41MSA0LjUgOSA0LjVjMS4yNCAwIDIuMzYuNTIgMy4xNyAxLjMzTDEwIDhoNVYzbC0xLjc2IDEuNzZDMTIuMTUgMy42OCAxMC42NiAzIDkgMyA1LjY5IDMgMy4wMSA1LjY5IDMuMDEgOVM1LjY5IDE1IDkgMTVjMi45NyAwIDUuNDMtMi4xNiA1LjktNWgtMS41MmMtLjQ2IDItMi4yNCAzLjUtNC4zOCAzLjV6Ii8+CiAgICA8L2c+Cjwvc3ZnPgo=);
  --jp-icon-regex: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIwIDIwIj4KICA8ZyBjbGFzcz0ianAtaWNvbjIiIGZpbGw9IiM0MTQxNDEiPgogICAgPHJlY3QgeD0iMiIgeT0iMiIgd2lkdGg9IjE2IiBoZWlnaHQ9IjE2Ii8+CiAgPC9nPgoKICA8ZyBjbGFzcz0ianAtaWNvbi1hY2NlbnQyIiBmaWxsPSIjRkZGIj4KICAgIDxjaXJjbGUgY2xhc3M9InN0MiIgY3g9IjUuNSIgY3k9IjE0LjUiIHI9IjEuNSIvPgogICAgPHJlY3QgeD0iMTIiIHk9IjQiIGNsYXNzPSJzdDIiIHdpZHRoPSIxIiBoZWlnaHQ9IjgiLz4KICAgIDxyZWN0IHg9IjguNSIgeT0iNy41IiB0cmFuc2Zvcm09Im1hdHJpeCgwLjg2NiAtMC41IDAuNSAwLjg2NiAtMi4zMjU1IDcuMzIxOSkiIGNsYXNzPSJzdDIiIHdpZHRoPSI4IiBoZWlnaHQ9IjEiLz4KICAgIDxyZWN0IHg9IjEyIiB5PSI0IiB0cmFuc2Zvcm09Im1hdHJpeCgwLjUgLTAuODY2IDAuODY2IDAuNSAtMC42Nzc5IDE0LjgyNTIpIiBjbGFzcz0ic3QyIiB3aWR0aD0iMSIgaGVpZ2h0PSI4Ii8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-run: url(data:image/svg+xml;base64,PHN2ZyBoZWlnaHQ9IjI0IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGggZD0iTTggNXYxNGwxMS03eiIvPgogICAgPC9nPgo8L3N2Zz4K);
  --jp-icon-running: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDUxMiA1MTIiPgogIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICA8cGF0aCBkPSJNMjU2IDhDMTE5IDggOCAxMTkgOCAyNTZzMTExIDI0OCAyNDggMjQ4IDI0OC0xMTEgMjQ4LTI0OFMzOTMgOCAyNTYgOHptOTYgMzI4YzAgOC44LTcuMiAxNi0xNiAxNkgxNzZjLTguOCAwLTE2LTcuMi0xNi0xNlYxNzZjMC04LjggNy4yLTE2IDE2LTE2aDE2MGM4LjggMCAxNiA3LjIgMTYgMTZ2MTYweiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-save: url(data:image/svg+xml;base64,PHN2ZyBoZWlnaHQ9IjI0IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGggZD0iTTE3IDNINWMtMS4xMSAwLTIgLjktMiAydjE0YzAgMS4xLjg5IDIgMiAyaDE0YzEuMSAwIDItLjkgMi0yVjdsLTQtNHptLTUgMTZjLTEuNjYgMC0zLTEuMzQtMy0zczEuMzQtMyAzLTMgMyAxLjM0IDMgMy0xLjM0IDMtMyAzem0zLTEwSDVWNWgxMHY0eiIvPgogICAgPC9nPgo8L3N2Zz4K);
  --jp-icon-search: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMTggMTgiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTEyLjEsMTAuOWgtMC43bC0wLjItMC4yYzAuOC0wLjksMS4zLTIuMiwxLjMtMy41YzAtMy0yLjQtNS40LTUuNC01LjRTMS44LDQuMiwxLjgsNy4xczIuNCw1LjQsNS40LDUuNCBjMS4zLDAsMi41LTAuNSwzLjUtMS4zbDAuMiwwLjJ2MC43bDQuMSw0LjFsMS4yLTEuMkwxMi4xLDEwLjl6IE03LjEsMTAuOWMtMi4xLDAtMy43LTEuNy0zLjctMy43czEuNy0zLjcsMy43LTMuN3MzLjcsMS43LDMuNywzLjcgUzkuMiwxMC45LDcuMSwxMC45eiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-settings: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8cGF0aCBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIiBkPSJNMTkuNDMgMTIuOThjLjA0LS4zMi4wNy0uNjQuMDctLjk4cy0uMDMtLjY2LS4wNy0uOThsMi4xMS0xLjY1Yy4xOS0uMTUuMjQtLjQyLjEyLS42NGwtMi0zLjQ2Yy0uMTItLjIyLS4zOS0uMy0uNjEtLjIybC0yLjQ5IDFjLS41Mi0uNC0xLjA4LS43My0xLjY5LS45OGwtLjM4LTIuNjVBLjQ4OC40ODggMCAwMDE0IDJoLTRjLS4yNSAwLS40Ni4xOC0uNDkuNDJsLS4zOCAyLjY1Yy0uNjEuMjUtMS4xNy41OS0xLjY5Ljk4bC0yLjQ5LTFjLS4yMy0uMDktLjQ5IDAtLjYxLjIybC0yIDMuNDZjLS4xMy4yMi0uMDcuNDkuMTIuNjRsMi4xMSAxLjY1Yy0uMDQuMzItLjA3LjY1LS4wNy45OHMuMDMuNjYuMDcuOThsLTIuMTEgMS42NWMtLjE5LjE1LS4yNC40Mi0uMTIuNjRsMiAzLjQ2Yy4xMi4yMi4zOS4zLjYxLjIybDIuNDktMWMuNTIuNCAxLjA4LjczIDEuNjkuOThsLjM4IDIuNjVjLjAzLjI0LjI0LjQyLjQ5LjQyaDRjLjI1IDAgLjQ2LS4xOC40OS0uNDJsLjM4LTIuNjVjLjYxLS4yNSAxLjE3LS41OSAxLjY5LS45OGwyLjQ5IDFjLjIzLjA5LjQ5IDAgLjYxLS4yMmwyLTMuNDZjLjEyLS4yMi4wNy0uNDktLjEyLS42NGwtMi4xMS0xLjY1ek0xMiAxNS41Yy0xLjkzIDAtMy41LTEuNTctMy41LTMuNXMxLjU3LTMuNSAzLjUtMy41IDMuNSAxLjU3IDMuNSAzLjUtMS41NyAzLjUtMy41IDMuNXoiLz4KPC9zdmc+Cg==);
  --jp-icon-spreadsheet: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8cGF0aCBjbGFzcz0ianAtaWNvbi1jb250cmFzdDEganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNENBRjUwIiBkPSJNMi4yIDIuMnYxNy42aDE3LjZWMi4ySDIuMnptMTUuNCA3LjdoLTUuNVY0LjRoNS41djUuNXpNOS45IDQuNHY1LjVINC40VjQuNGg1LjV6bS01LjUgNy43aDUuNXY1LjVINC40di01LjV6bTcuNyA1LjV2LTUuNWg1LjV2NS41aC01LjV6Ii8+Cjwvc3ZnPgo=);
  --jp-icon-stop: url(data:image/svg+xml;base64,PHN2ZyBoZWlnaHQ9IjI0IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGggZD0iTTAgMGgyNHYyNEgweiIgZmlsbD0ibm9uZSIvPgogICAgICAgIDxwYXRoIGQ9Ik02IDZoMTJ2MTJINnoiLz4KICAgIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-tab: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTIxIDNIM2MtMS4xIDAtMiAuOS0yIDJ2MTRjMCAxLjEuOSAyIDIgMmgxOGMxLjEgMCAyLS45IDItMlY1YzAtMS4xLS45LTItMi0yem0wIDE2SDNWNWgxMHY0aDh2MTB6Ii8+CiAgPC9nPgo8L3N2Zz4K);
  --jp-icon-table-rows: url(data:image/svg+xml;base64,PHN2ZyBoZWlnaHQ9IjI0IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGggZD0iTTAgMGgyNHYyNEgweiIgZmlsbD0ibm9uZSIvPgogICAgICAgIDxwYXRoIGQ9Ik0yMSw4SDNWNGgxOFY4eiBNMjEsMTBIM3Y0aDE4VjEweiBNMjEsMTZIM3Y0aDE4VjE2eiIvPgogICAgPC9nPgo8L3N2Zz4=);
  --jp-icon-tag: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjgiIGhlaWdodD0iMjgiIHZpZXdCb3g9IjAgMCA0MyAyOCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KCTxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CgkJPHBhdGggZD0iTTI4LjgzMzIgMTIuMzM0TDMyLjk5OTggMTYuNTAwN0wzNy4xNjY1IDEyLjMzNEgyOC44MzMyWiIvPgoJCTxwYXRoIGQ9Ik0xNi4yMDk1IDIxLjYxMDRDMTUuNjg3MyAyMi4xMjk5IDE0Ljg0NDMgMjIuMTI5OSAxNC4zMjQ4IDIxLjYxMDRMNi45ODI5IDE0LjcyNDVDNi41NzI0IDE0LjMzOTQgNi4wODMxMyAxMy42MDk4IDYuMDQ3ODYgMTMuMDQ4MkM1Ljk1MzQ3IDExLjUyODggNi4wMjAwMiA4LjYxOTQ0IDYuMDY2MjEgNy4wNzY5NUM2LjA4MjgxIDYuNTE0NzcgNi41NTU0OCA2LjA0MzQ3IDcuMTE4MDQgNi4wMzA1NUM5LjA4ODYzIDUuOTg0NzMgMTMuMjYzOCA1LjkzNTc5IDEzLjY1MTggNi4zMjQyNUwyMS43MzY5IDEzLjYzOUMyMi4yNTYgMTQuMTU4NSAyMS43ODUxIDE1LjQ3MjQgMjEuMjYyIDE1Ljk5NDZMMTYuMjA5NSAyMS42MTA0Wk05Ljc3NTg1IDguMjY1QzkuMzM1NTEgNy44MjU2NiA4LjYyMzUxIDcuODI1NjYgOC4xODI4IDguMjY1QzcuNzQzNDYgOC43MDU3MSA3Ljc0MzQ2IDkuNDE3MzMgOC4xODI4IDkuODU2NjdDOC42MjM4MiAxMC4yOTY0IDkuMzM1ODIgMTAuMjk2NCA5Ljc3NTg1IDkuODU2NjdDMTAuMjE1NiA5LjQxNzMzIDEwLjIxNTYgOC43MDUzMyA5Ljc3NTg1IDguMjY1WiIvPgoJPC9nPgo8L3N2Zz4K);
  --jp-icon-terminal: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0IiA+CiAgICA8cmVjdCBjbGFzcz0ianAtaWNvbjIganAtaWNvbi1zZWxlY3RhYmxlIiB3aWR0aD0iMjAiIGhlaWdodD0iMjAiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDIgMikiIGZpbGw9IiMzMzMzMzMiLz4KICAgIDxwYXRoIGNsYXNzPSJqcC1pY29uLWFjY2VudDIganAtaWNvbi1zZWxlY3RhYmxlLWludmVyc2UiIGQ9Ik01LjA1NjY0IDguNzYxNzJDNS4wNTY2NCA4LjU5NzY2IDUuMDMxMjUgOC40NTMxMiA0Ljk4MDQ3IDguMzI4MTJDNC45MzM1OSA4LjE5OTIyIDQuODU1NDcgOC4wODIwMyA0Ljc0NjA5IDcuOTc2NTZDNC42NDA2MiA3Ljg3MTA5IDQuNSA3Ljc3NTM5IDQuMzI0MjIgNy42ODk0NUM0LjE1MjM0IDcuNTk5NjEgMy45NDMzNiA3LjUxMTcyIDMuNjk3MjcgNy40MjU3OEMzLjMwMjczIDcuMjg1MTYgMi45NDMzNiA3LjEzNjcyIDIuNjE5MTQgNi45ODA0N0MyLjI5NDkyIDYuODI0MjIgMi4wMTc1OCA2LjY0MjU4IDEuNzg3MTEgNi40MzU1NUMxLjU2MDU1IDYuMjI4NTIgMS4zODQ3NyA1Ljk4ODI4IDEuMjU5NzcgNS43MTQ4NEMxLjEzNDc3IDUuNDM3NSAxLjA3MjI3IDUuMTA5MzggMS4wNzIyNyA0LjczMDQ3QzEuMDcyMjcgNC4zOTg0NCAxLjEyODkxIDQuMDk1NyAxLjI0MjE5IDMuODIyMjdDMS4zNTU0NyAzLjU0NDkyIDEuNTE1NjIgMy4zMDQ2OSAxLjcyMjY2IDMuMTAxNTZDMS45Mjk2OSAyLjg5ODQ0IDIuMTc5NjkgMi43MzQzNyAyLjQ3MjY2IDIuNjA5MzhDMi43NjU2MiAyLjQ4NDM4IDMuMDkxOCAyLjQwNDMgMy40NTExNyAyLjM2OTE0VjEuMTA5MzhINC4zODg2N1YyLjM4MDg2QzQuNzQwMjMgMi40Mjc3MyA1LjA1NjY0IDIuNTIzNDQgNS4zMzc4OSAyLjY2Nzk3QzUuNjE5MTQgMi44MTI1IDUuODU3NDIgMy4wMDE5NSA2LjA1MjczIDMuMjM2MzNDNi4yNTE5NSAzLjQ2NjggNi40MDQzIDMuNzQwMjMgNi41MDk3NyA0LjA1NjY0QzYuNjE5MTQgNC4zNjkxNCA2LjY3MzgzIDQuNzIwNyA2LjY3MzgzIDUuMTExMzNINS4wNDQ5MkM1LjA0NDkyIDQuNjM4NjcgNC45Mzc1IDQuMjgxMjUgNC43MjI2NiA0LjAzOTA2QzQuNTA3ODEgMy43OTI5NyA0LjIxNjggMy42Njk5MiAzLjg0OTYxIDMuNjY5OTJDMy42NTAzOSAzLjY2OTkyIDMuNDc2NTYgMy42OTcyNyAzLjMyODEyIDMuNzUxOTVDMy4xODM1OSAzLjgwMjczIDMuMDY0NDUgMy44NzY5NSAyLjk3MDcgMy45NzQ2MUMyLjg3Njk1IDQuMDY4MzYgMi44MDY2NCA0LjE3OTY5IDIuNzU5NzcgNC4zMDg1OUMyLjcxNjggNC40Mzc1IDIuNjk1MzEgNC41NzgxMiAyLjY5NTMxIDQuNzMwNDdDMi42OTUzMSA0Ljg4MjgxIDIuNzE2OCA1LjAxOTUzIDIuNzU5NzcgNS4xNDA2MkMyLjgwNjY0IDUuMjU3ODEgMi44ODI4MSA1LjM2NzE5IDIuOTg4MjggNS40Njg3NUMzLjA5NzY2IDUuNTcwMzEgMy4yNDAyMyA1LjY2Nzk3IDMuNDE2MDIgNS43NjE3MkMzLjU5MTggNS44NTE1NiAzLjgxMDU1IDUuOTQzMzYgNC4wNzIyNyA2LjAzNzExQzQuNDY2OCA2LjE4NTU1IDQuODI0MjIgNi4zMzk4NCA1LjE0NDUzIDYuNUM1LjQ2NDg0IDYuNjU2MjUgNS43MzgyOCA2LjgzOTg0IDUuOTY0ODQgNy4wNTA3OEM2LjE5NTMxIDcuMjU3ODEgNi4zNzEwOSA3LjUgNi40OTIxOSA3Ljc3NzM0QzYuNjE3MTkgOC4wNTA3OCA2LjY3OTY5IDguMzc1IDYuNjc5NjkgOC43NUM2LjY3OTY5IDkuMDkzNzUgNi42MjMwNSA5LjQwNDMgNi41MDk3NyA5LjY4MTY0QzYuMzk2NDggOS45NTUwOCA2LjIzNDM4IDEwLjE5MTQgNi4wMjM0NCAxMC4zOTA2QzUuODEyNSAxMC41ODk4IDUuNTU4NTkgMTAuNzUgNS4yNjE3MiAxMC44NzExQzQuOTY0ODQgMTAuOTg4MyA0LjYzMjgxIDExLjA2NDUgNC4yNjU2MiAxMS4wOTk2VjEyLjI0OEgzLjMzMzk4VjExLjA5OTZDMy4wMDE5NSAxMS4wNjg0IDIuNjc5NjkgMTAuOTk2MSAyLjM2NzE5IDEwLjg4MjhDMi4wNTQ2OSAxMC43NjU2IDEuNzc3MzQgMTAuNTk3NyAxLjUzNTE2IDEwLjM3ODlDMS4yOTY4OCAxMC4xNjAyIDEuMTA1NDcgOS44ODQ3NyAwLjk2MDkzOCA5LjU1MjczQzAuODE2NDA2IDkuMjE2OCAwLjc0NDE0MSA4LjgxNDQ1IDAuNzQ0MTQxIDguMzQ1N0gyLjM3ODkxQzIuMzc4OTEgOC42MjY5NSAyLjQxOTkyIDguODYzMjggMi41MDE5NSA5LjA1NDY5QzIuNTgzOTggOS4yNDIxOSAyLjY4OTQ1IDkuMzkyNTggMi44MTgzNiA5LjUwNTg2QzIuOTUxMTcgOS42MTUyMyAzLjEwMTU2IDkuNjkzMzYgMy4yNjk1MyA5Ljc0MDIzQzMuNDM3NSA5Ljc4NzExIDMuNjA5MzggOS44MTA1NSAzLjc4NTE2IDkuODEwNTVDNC4yMDMxMiA5LjgxMDU1IDQuNTE5NTMgOS43MTI4OSA0LjczNDM4IDkuNTE3NThDNC45NDkyMiA5LjMyMjI3IDUuMDU2NjQgOS4wNzAzMSA1LjA1NjY0IDguNzYxNzJaTTEzLjQxOCAxMi4yNzE1SDguMDc0MjJWMTFIMTMuNDE4VjEyLjI3MTVaIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgzLjk1MjY0IDYpIiBmaWxsPSJ3aGl0ZSIvPgo8L3N2Zz4K);
  --jp-icon-text-editor: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI0Ij4KICA8cGF0aCBjbGFzcz0ianAtaWNvbjMganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjNjE2MTYxIiBkPSJNMTUgMTVIM3YyaDEydi0yem0wLThIM3YyaDEyVjd6TTMgMTNoMTh2LTJIM3Yyem0wIDhoMTh2LTJIM3Yyek0zIDN2MmgxOFYzSDN6Ii8+Cjwvc3ZnPgo=);
  --jp-icon-toc: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIiB2ZXJzaW9uPSIxLjEiIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0IiBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgoJPHBhdGggZD0iTTcsNUgyMVY3SDdWNU03LDEzVjExSDIxVjEzSDdNNCw0LjVBMS41LDEuNSAwIDAsMSA1LjUsNkExLjUsMS41IDAgMCwxIDQsNy41QTEuNSwxLjUgMCAwLDEgMi41LDZBMS41LDEuNSAwIDAsMSA0LDQuNU00LDEwLjVBMS41LDEuNSAwIDAsMSA1LjUsMTJBMS41LDEuNSAwIDAsMSA0LDEzLjVBMS41LDEuNSAwIDAsMSAyLjUsMTJBMS41LDEuNSAwIDAsMSA0LDEwLjVNNywxOVYxN0gyMVYxOUg3TTQsMTYuNUExLjUsMS41IDAgMCwxIDUuNSwxOEExLjUsMS41IDAgMCwxIDQsMTkuNUExLjUsMS41IDAgMCwxIDIuNSwxOEExLjUsMS41IDAgMCwxIDQsMTYuNVoiIC8+Cjwvc3ZnPgo=);
  --jp-icon-tree-view: url(data:image/svg+xml;base64,PHN2ZyBoZWlnaHQ9IjI0IiB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIyNCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgIDxnIGNsYXNzPSJqcC1pY29uMyIgZmlsbD0iIzYxNjE2MSI+CiAgICAgICAgPHBhdGggZD0iTTAgMGgyNHYyNEgweiIgZmlsbD0ibm9uZSIvPgogICAgICAgIDxwYXRoIGQ9Ik0yMiAxMVYzaC03djNIOVYzSDJ2OGg3VjhoMnYxMGg0djNoN3YtOGgtN3YzaC0yVjhoMnYzeiIvPgogICAgPC9nPgo8L3N2Zz4=);
  --jp-icon-trusted: url(data:image/svg+xml;base64,PHN2ZyBmaWxsPSJub25lIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDI0IDI1Ij4KICAgIDxwYXRoIGNsYXNzPSJqcC1pY29uMiIgc3Ryb2tlPSIjMzMzMzMzIiBzdHJva2Utd2lkdGg9IjIiIHRyYW5zZm9ybT0idHJhbnNsYXRlKDIgMykiIGQ9Ik0xLjg2MDk0IDExLjQ0MDlDMC44MjY0NDggOC43NzAyNyAwLjg2Mzc3OSA2LjA1NzY0IDEuMjQ5MDcgNC4xOTkzMkMyLjQ4MjA2IDMuOTMzNDcgNC4wODA2OCAzLjQwMzQ3IDUuNjAxMDIgMi44NDQ5QzcuMjM1NDkgMi4yNDQ0IDguODU2NjYgMS41ODE1IDkuOTg3NiAxLjA5NTM5QzExLjA1OTcgMS41ODM0MSAxMi42MDk0IDIuMjQ0NCAxNC4yMTggMi44NDMzOUMxNS43NTAzIDMuNDEzOTQgMTcuMzk5NSAzLjk1MjU4IDE4Ljc1MzkgNC4yMTM4NUMxOS4xMzY0IDYuMDcxNzcgMTkuMTcwOSA4Ljc3NzIyIDE4LjEzOSAxMS40NDA5QzE3LjAzMDMgMTQuMzAzMiAxNC42NjY4IDE3LjE4NDQgOS45OTk5OSAxOC45MzU0QzUuMzMzMiAxNy4xODQ0IDIuOTY5NjggMTQuMzAzMiAxLjg2MDk0IDExLjQ0MDlaIi8+CiAgICA8cGF0aCBjbGFzcz0ianAtaWNvbjIiIGZpbGw9IiMzMzMzMzMiIHN0cm9rZT0iIzMzMzMzMyIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoOCA5Ljg2NzE5KSIgZD0iTTIuODYwMTUgNC44NjUzNUwwLjcyNjU0OSAyLjk5OTU5TDAgMy42MzA0NUwyLjg2MDE1IDYuMTMxNTdMOCAwLjYzMDg3Mkw3LjI3ODU3IDBMMi44NjAxNSA0Ljg2NTM1WiIvPgo8L3N2Zz4K);
  --jp-icon-undo: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMjQgMjQiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTEyLjUgOGMtMi42NSAwLTUuMDUuOTktNi45IDIuNkwyIDd2OWg5bC0zLjYyLTMuNjJjMS4zOS0xLjE2IDMuMTYtMS44OCA1LjEyLTEuODggMy41NCAwIDYuNTUgMi4zMSA3LjYgNS41bDIuMzctLjc4QzIxLjA4IDExLjAzIDE3LjE1IDggMTIuNSA4eiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-vega: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8ZyBjbGFzcz0ianAtaWNvbjEganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjMjEyMTIxIj4KICAgIDxwYXRoIGQ9Ik0xMC42IDUuNGwyLjItMy4ySDIuMnY3LjNsNC02LjZ6Ii8+CiAgICA8cGF0aCBkPSJNMTUuOCAyLjJsLTQuNCA2LjZMNyA2LjNsLTQuOCA4djUuNWgxNy42VjIuMmgtNHptLTcgMTUuNEg1LjV2LTQuNGgzLjN2NC40em00LjQgMEg5LjhWOS44aDMuNHY3Ljh6bTQuNCAwaC0zLjRWNi41aDMuNHYxMS4xeiIvPgogIDwvZz4KPC9zdmc+Cg==);
  --jp-icon-yaml: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxNiIgdmlld0JveD0iMCAwIDIyIDIyIj4KICA8ZyBjbGFzcz0ianAtaWNvbi1jb250cmFzdDIganAtaWNvbi1zZWxlY3RhYmxlIiBmaWxsPSIjRDgxQjYwIj4KICAgIDxwYXRoIGQ9Ik03LjIgMTguNnYtNS40TDMgNS42aDMuM2wxLjQgMy4xYy4zLjkuNiAxLjYgMSAyLjUuMy0uOC42LTEuNiAxLTIuNWwxLjQtMy4xaDMuNGwtNC40IDcuNnY1LjVsLTIuOS0uMXoiLz4KICAgIDxjaXJjbGUgY2xhc3M9InN0MCIgY3g9IjE3LjYiIGN5PSIxNi41IiByPSIyLjEiLz4KICAgIDxjaXJjbGUgY2xhc3M9InN0MCIgY3g9IjE3LjYiIGN5PSIxMSIgcj0iMi4xIi8+CiAgPC9nPgo8L3N2Zz4K);
}

/* Icon CSS class declarations */

.jp-AddIcon {
  background-image: var(--jp-icon-add);
}
.jp-BugIcon {
  background-image: var(--jp-icon-bug);
}
.jp-BuildIcon {
  background-image: var(--jp-icon-build);
}
.jp-CaretDownEmptyIcon {
  background-image: var(--jp-icon-caret-down-empty);
}
.jp-CaretDownEmptyThinIcon {
  background-image: var(--jp-icon-caret-down-empty-thin);
}
.jp-CaretDownIcon {
  background-image: var(--jp-icon-caret-down);
}
.jp-CaretLeftIcon {
  background-image: var(--jp-icon-caret-left);
}
.jp-CaretRightIcon {
  background-image: var(--jp-icon-caret-right);
}
.jp-CaretUpEmptyThinIcon {
  background-image: var(--jp-icon-caret-up-empty-thin);
}
.jp-CaretUpIcon {
  background-image: var(--jp-icon-caret-up);
}
.jp-CaseSensitiveIcon {
  background-image: var(--jp-icon-case-sensitive);
}
.jp-CheckIcon {
  background-image: var(--jp-icon-check);
}
.jp-CircleEmptyIcon {
  background-image: var(--jp-icon-circle-empty);
}
.jp-CircleIcon {
  background-image: var(--jp-icon-circle);
}
.jp-ClearIcon {
  background-image: var(--jp-icon-clear);
}
.jp-CloseIcon {
  background-image: var(--jp-icon-close);
}
.jp-CodeIcon {
  background-image: var(--jp-icon-code);
}
.jp-ConsoleIcon {
  background-image: var(--jp-icon-console);
}
.jp-CopyIcon {
  background-image: var(--jp-icon-copy);
}
.jp-CutIcon {
  background-image: var(--jp-icon-cut);
}
.jp-DownloadIcon {
  background-image: var(--jp-icon-download);
}
.jp-EditIcon {
  background-image: var(--jp-icon-edit);
}
.jp-EllipsesIcon {
  background-image: var(--jp-icon-ellipses);
}
.jp-ExtensionIcon {
  background-image: var(--jp-icon-extension);
}
.jp-FastForwardIcon {
  background-image: var(--jp-icon-fast-forward);
}
.jp-FileIcon {
  background-image: var(--jp-icon-file);
}
.jp-FileUploadIcon {
  background-image: var(--jp-icon-file-upload);
}
.jp-FilterListIcon {
  background-image: var(--jp-icon-filter-list);
}
.jp-FolderIcon {
  background-image: var(--jp-icon-folder);
}
.jp-Html5Icon {
  background-image: var(--jp-icon-html5);
}
.jp-ImageIcon {
  background-image: var(--jp-icon-image);
}
.jp-InspectorIcon {
  background-image: var(--jp-icon-inspector);
}
.jp-JsonIcon {
  background-image: var(--jp-icon-json);
}
.jp-JupyterFaviconIcon {
  background-image: var(--jp-icon-jupyter-favicon);
}
.jp-JupyterIcon {
  background-image: var(--jp-icon-jupyter);
}
.jp-JupyterlabWordmarkIcon {
  background-image: var(--jp-icon-jupyterlab-wordmark);
}
.jp-KernelIcon {
  background-image: var(--jp-icon-kernel);
}
.jp-KeyboardIcon {
  background-image: var(--jp-icon-keyboard);
}
.jp-LauncherIcon {
  background-image: var(--jp-icon-launcher);
}
.jp-LineFormIcon {
  background-image: var(--jp-icon-line-form);
}
.jp-LinkIcon {
  background-image: var(--jp-icon-link);
}
.jp-ListIcon {
  background-image: var(--jp-icon-list);
}
.jp-ListingsInfoIcon {
  background-image: var(--jp-icon-listings-info);
}
.jp-MarkdownIcon {
  background-image: var(--jp-icon-markdown);
}
.jp-NewFolderIcon {
  background-image: var(--jp-icon-new-folder);
}
.jp-NotTrustedIcon {
  background-image: var(--jp-icon-not-trusted);
}
.jp-NotebookIcon {
  background-image: var(--jp-icon-notebook);
}
.jp-NumberingIcon {
  background-image: var(--jp-icon-numbering);
}
.jp-OfflineBoltIcon {
  background-image: var(--jp-icon-offline-bolt);
}
.jp-PaletteIcon {
  background-image: var(--jp-icon-palette);
}
.jp-PasteIcon {
  background-image: var(--jp-icon-paste);
}
.jp-PdfIcon {
  background-image: var(--jp-icon-pdf);
}
.jp-PythonIcon {
  background-image: var(--jp-icon-python);
}
.jp-RKernelIcon {
  background-image: var(--jp-icon-r-kernel);
}
.jp-ReactIcon {
  background-image: var(--jp-icon-react);
}
.jp-RedoIcon {
  background-image: var(--jp-icon-redo);
}
.jp-RefreshIcon {
  background-image: var(--jp-icon-refresh);
}
.jp-RegexIcon {
  background-image: var(--jp-icon-regex);
}
.jp-RunIcon {
  background-image: var(--jp-icon-run);
}
.jp-RunningIcon {
  background-image: var(--jp-icon-running);
}
.jp-SaveIcon {
  background-image: var(--jp-icon-save);
}
.jp-SearchIcon {
  background-image: var(--jp-icon-search);
}
.jp-SettingsIcon {
  background-image: var(--jp-icon-settings);
}
.jp-SpreadsheetIcon {
  background-image: var(--jp-icon-spreadsheet);
}
.jp-StopIcon {
  background-image: var(--jp-icon-stop);
}
.jp-TabIcon {
  background-image: var(--jp-icon-tab);
}
.jp-TableRowsIcon {
  background-image: var(--jp-icon-table-rows);
}
.jp-TagIcon {
  background-image: var(--jp-icon-tag);
}
.jp-TerminalIcon {
  background-image: var(--jp-icon-terminal);
}
.jp-TextEditorIcon {
  background-image: var(--jp-icon-text-editor);
}
.jp-TocIcon {
  background-image: var(--jp-icon-toc);
}
.jp-TreeViewIcon {
  background-image: var(--jp-icon-tree-view);
}
.jp-TrustedIcon {
  background-image: var(--jp-icon-trusted);
}
.jp-UndoIcon {
  background-image: var(--jp-icon-undo);
}
.jp-VegaIcon {
  background-image: var(--jp-icon-vega);
}
.jp-YamlIcon {
  background-image: var(--jp-icon-yaml);
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/**
 * (DEPRECATED) Support for consuming icons as CSS background images
 */

:root {
  --jp-icon-search-white: url(data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMTggMTgiIHdpZHRoPSIxNiIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICA8ZyBjbGFzcz0ianAtaWNvbjMiIGZpbGw9IiM2MTYxNjEiPgogICAgPHBhdGggZD0iTTEyLjEsMTAuOWgtMC43bC0wLjItMC4yYzAuOC0wLjksMS4zLTIuMiwxLjMtMy41YzAtMy0yLjQtNS40LTUuNC01LjRTMS44LDQuMiwxLjgsNy4xczIuNCw1LjQsNS40LDUuNCBjMS4zLDAsMi41LTAuNSwzLjUtMS4zbDAuMiwwLjJ2MC43bDQuMSw0LjFsMS4yLTEuMkwxMi4xLDEwLjl6IE03LjEsMTAuOWMtMi4xLDAtMy43LTEuNy0zLjctMy43czEuNy0zLjcsMy43LTMuN3MzLjcsMS43LDMuNywzLjcgUzkuMiwxMC45LDcuMSwxMC45eiIvPgogIDwvZz4KPC9zdmc+Cg==);
}

.jp-Icon,
.jp-MaterialIcon {
  background-position: center;
  background-repeat: no-repeat;
  background-size: 16px;
  min-width: 16px;
  min-height: 16px;
}

.jp-Icon-cover {
  background-position: center;
  background-repeat: no-repeat;
  background-size: cover;
}

/**
 * (DEPRECATED) Support for specific CSS icon sizes
 */

.jp-Icon-16 {
  background-size: 16px;
  min-width: 16px;
  min-height: 16px;
}

.jp-Icon-18 {
  background-size: 18px;
  min-width: 18px;
  min-height: 18px;
}

.jp-Icon-20 {
  background-size: 20px;
  min-width: 20px;
  min-height: 20px;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/**
 * Support for icons as inline SVG HTMLElements
 */

/* recolor the primary elements of an icon */
.jp-icon0[fill] {
  fill: var(--jp-inverse-layout-color0);
}
.jp-icon1[fill] {
  fill: var(--jp-inverse-layout-color1);
}
.jp-icon2[fill] {
  fill: var(--jp-inverse-layout-color2);
}
.jp-icon3[fill] {
  fill: var(--jp-inverse-layout-color3);
}
.jp-icon4[fill] {
  fill: var(--jp-inverse-layout-color4);
}

.jp-icon0[stroke] {
  stroke: var(--jp-inverse-layout-color0);
}
.jp-icon1[stroke] {
  stroke: var(--jp-inverse-layout-color1);
}
.jp-icon2[stroke] {
  stroke: var(--jp-inverse-layout-color2);
}
.jp-icon3[stroke] {
  stroke: var(--jp-inverse-layout-color3);
}
.jp-icon4[stroke] {
  stroke: var(--jp-inverse-layout-color4);
}
/* recolor the accent elements of an icon */
.jp-icon-accent0[fill] {
  fill: var(--jp-layout-color0);
}
.jp-icon-accent1[fill] {
  fill: var(--jp-layout-color1);
}
.jp-icon-accent2[fill] {
  fill: var(--jp-layout-color2);
}
.jp-icon-accent3[fill] {
  fill: var(--jp-layout-color3);
}
.jp-icon-accent4[fill] {
  fill: var(--jp-layout-color4);
}

.jp-icon-accent0[stroke] {
  stroke: var(--jp-layout-color0);
}
.jp-icon-accent1[stroke] {
  stroke: var(--jp-layout-color1);
}
.jp-icon-accent2[stroke] {
  stroke: var(--jp-layout-color2);
}
.jp-icon-accent3[stroke] {
  stroke: var(--jp-layout-color3);
}
.jp-icon-accent4[stroke] {
  stroke: var(--jp-layout-color4);
}
/* set the color of an icon to transparent */
.jp-icon-none[fill] {
  fill: none;
}

.jp-icon-none[stroke] {
  stroke: none;
}
/* brand icon colors. Same for light and dark */
.jp-icon-brand0[fill] {
  fill: var(--jp-brand-color0);
}
.jp-icon-brand1[fill] {
  fill: var(--jp-brand-color1);
}
.jp-icon-brand2[fill] {
  fill: var(--jp-brand-color2);
}
.jp-icon-brand3[fill] {
  fill: var(--jp-brand-color3);
}
.jp-icon-brand4[fill] {
  fill: var(--jp-brand-color4);
}

.jp-icon-brand0[stroke] {
  stroke: var(--jp-brand-color0);
}
.jp-icon-brand1[stroke] {
  stroke: var(--jp-brand-color1);
}
.jp-icon-brand2[stroke] {
  stroke: var(--jp-brand-color2);
}
.jp-icon-brand3[stroke] {
  stroke: var(--jp-brand-color3);
}
.jp-icon-brand4[stroke] {
  stroke: var(--jp-brand-color4);
}
/* warn icon colors. Same for light and dark */
.jp-icon-warn0[fill] {
  fill: var(--jp-warn-color0);
}
.jp-icon-warn1[fill] {
  fill: var(--jp-warn-color1);
}
.jp-icon-warn2[fill] {
  fill: var(--jp-warn-color2);
}
.jp-icon-warn3[fill] {
  fill: var(--jp-warn-color3);
}

.jp-icon-warn0[stroke] {
  stroke: var(--jp-warn-color0);
}
.jp-icon-warn1[stroke] {
  stroke: var(--jp-warn-color1);
}
.jp-icon-warn2[stroke] {
  stroke: var(--jp-warn-color2);
}
.jp-icon-warn3[stroke] {
  stroke: var(--jp-warn-color3);
}
/* icon colors that contrast well with each other and most backgrounds */
.jp-icon-contrast0[fill] {
  fill: var(--jp-icon-contrast-color0);
}
.jp-icon-contrast1[fill] {
  fill: var(--jp-icon-contrast-color1);
}
.jp-icon-contrast2[fill] {
  fill: var(--jp-icon-contrast-color2);
}
.jp-icon-contrast3[fill] {
  fill: var(--jp-icon-contrast-color3);
}

.jp-icon-contrast0[stroke] {
  stroke: var(--jp-icon-contrast-color0);
}
.jp-icon-contrast1[stroke] {
  stroke: var(--jp-icon-contrast-color1);
}
.jp-icon-contrast2[stroke] {
  stroke: var(--jp-icon-contrast-color2);
}
.jp-icon-contrast3[stroke] {
  stroke: var(--jp-icon-contrast-color3);
}

/* CSS for icons in selected items in the settings editor */
#setting-editor .jp-PluginList .jp-mod-selected .jp-icon-selectable[fill] {
  fill: #fff;
}
#setting-editor
  .jp-PluginList
  .jp-mod-selected
  .jp-icon-selectable-inverse[fill] {
  fill: var(--jp-brand-color1);
}

/* CSS for icons in selected filebrowser listing items */
.jp-DirListing-item.jp-mod-selected .jp-icon-selectable[fill] {
  fill: #fff;
}
.jp-DirListing-item.jp-mod-selected .jp-icon-selectable-inverse[fill] {
  fill: var(--jp-brand-color1);
}

/* CSS for icons in selected tabs in the sidebar tab manager */
#tab-manager .lm-TabBar-tab.jp-mod-active .jp-icon-selectable[fill] {
  fill: #fff;
}

#tab-manager .lm-TabBar-tab.jp-mod-active .jp-icon-selectable-inverse[fill] {
  fill: var(--jp-brand-color1);
}
#tab-manager
  .lm-TabBar-tab.jp-mod-active
  .jp-icon-hover
  :hover
  .jp-icon-selectable[fill] {
  fill: var(--jp-brand-color1);
}

#tab-manager
  .lm-TabBar-tab.jp-mod-active
  .jp-icon-hover
  :hover
  .jp-icon-selectable-inverse[fill] {
  fill: #fff;
}

/**
 * TODO: come up with non css-hack solution for showing the busy icon on top
 *  of the close icon
 * CSS for complex behavior of close icon of tabs in the sidebar tab manager
 */
#tab-manager
  .lm-TabBar-tab.jp-mod-dirty
  > .lm-TabBar-tabCloseIcon
  > :not(:hover)
  > .jp-icon3[fill] {
  fill: none;
}
#tab-manager
  .lm-TabBar-tab.jp-mod-dirty
  > .lm-TabBar-tabCloseIcon
  > :not(:hover)
  > .jp-icon-busy[fill] {
  fill: var(--jp-inverse-layout-color3);
}

#tab-manager
  .lm-TabBar-tab.jp-mod-dirty.jp-mod-active
  > .lm-TabBar-tabCloseIcon
  > :not(:hover)
  > .jp-icon-busy[fill] {
  fill: #fff;
}

/**
* TODO: come up with non css-hack solution for showing the busy icon on top
*  of the close icon
* CSS for complex behavior of close icon of tabs in the main area tabbar
*/
.lm-DockPanel-tabBar
  .lm-TabBar-tab.lm-mod-closable.jp-mod-dirty
  > .lm-TabBar-tabCloseIcon
  > :not(:hover)
  > .jp-icon3[fill] {
  fill: none;
}
.lm-DockPanel-tabBar
  .lm-TabBar-tab.lm-mod-closable.jp-mod-dirty
  > .lm-TabBar-tabCloseIcon
  > :not(:hover)
  > .jp-icon-busy[fill] {
  fill: var(--jp-inverse-layout-color3);
}

/* CSS for icons in status bar */
#jp-main-statusbar .jp-mod-selected .jp-icon-selectable[fill] {
  fill: #fff;
}

#jp-main-statusbar .jp-mod-selected .jp-icon-selectable-inverse[fill] {
  fill: var(--jp-brand-color1);
}
/* special handling for splash icon CSS. While the theme CSS reloads during
   splash, the splash icon can loose theming. To prevent that, we set a
   default for its color variable */
:root {
  --jp-warn-color0: var(--md-orange-700);
}

/* not sure what to do with this one, used in filebrowser listing */
.jp-DragIcon {
  margin-right: 4px;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/**
 * Support for alt colors for icons as inline SVG HTMLElements
 */

/* alt recolor the primary elements of an icon */
.jp-icon-alt .jp-icon0[fill] {
  fill: var(--jp-layout-color0);
}
.jp-icon-alt .jp-icon1[fill] {
  fill: var(--jp-layout-color1);
}
.jp-icon-alt .jp-icon2[fill] {
  fill: var(--jp-layout-color2);
}
.jp-icon-alt .jp-icon3[fill] {
  fill: var(--jp-layout-color3);
}
.jp-icon-alt .jp-icon4[fill] {
  fill: var(--jp-layout-color4);
}

.jp-icon-alt .jp-icon0[stroke] {
  stroke: var(--jp-layout-color0);
}
.jp-icon-alt .jp-icon1[stroke] {
  stroke: var(--jp-layout-color1);
}
.jp-icon-alt .jp-icon2[stroke] {
  stroke: var(--jp-layout-color2);
}
.jp-icon-alt .jp-icon3[stroke] {
  stroke: var(--jp-layout-color3);
}
.jp-icon-alt .jp-icon4[stroke] {
  stroke: var(--jp-layout-color4);
}

/* alt recolor the accent elements of an icon */
.jp-icon-alt .jp-icon-accent0[fill] {
  fill: var(--jp-inverse-layout-color0);
}
.jp-icon-alt .jp-icon-accent1[fill] {
  fill: var(--jp-inverse-layout-color1);
}
.jp-icon-alt .jp-icon-accent2[fill] {
  fill: var(--jp-inverse-layout-color2);
}
.jp-icon-alt .jp-icon-accent3[fill] {
  fill: var(--jp-inverse-layout-color3);
}
.jp-icon-alt .jp-icon-accent4[fill] {
  fill: var(--jp-inverse-layout-color4);
}

.jp-icon-alt .jp-icon-accent0[stroke] {
  stroke: var(--jp-inverse-layout-color0);
}
.jp-icon-alt .jp-icon-accent1[stroke] {
  stroke: var(--jp-inverse-layout-color1);
}
.jp-icon-alt .jp-icon-accent2[stroke] {
  stroke: var(--jp-inverse-layout-color2);
}
.jp-icon-alt .jp-icon-accent3[stroke] {
  stroke: var(--jp-inverse-layout-color3);
}
.jp-icon-alt .jp-icon-accent4[stroke] {
  stroke: var(--jp-inverse-layout-color4);
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-icon-hoverShow:not(:hover) svg {
  display: none !important;
}

/**
 * Support for hover colors for icons as inline SVG HTMLElements
 */

/**
 * regular colors
 */

/* recolor the primary elements of an icon */
.jp-icon-hover :hover .jp-icon0-hover[fill] {
  fill: var(--jp-inverse-layout-color0);
}
.jp-icon-hover :hover .jp-icon1-hover[fill] {
  fill: var(--jp-inverse-layout-color1);
}
.jp-icon-hover :hover .jp-icon2-hover[fill] {
  fill: var(--jp-inverse-layout-color2);
}
.jp-icon-hover :hover .jp-icon3-hover[fill] {
  fill: var(--jp-inverse-layout-color3);
}
.jp-icon-hover :hover .jp-icon4-hover[fill] {
  fill: var(--jp-inverse-layout-color4);
}

.jp-icon-hover :hover .jp-icon0-hover[stroke] {
  stroke: var(--jp-inverse-layout-color0);
}
.jp-icon-hover :hover .jp-icon1-hover[stroke] {
  stroke: var(--jp-inverse-layout-color1);
}
.jp-icon-hover :hover .jp-icon2-hover[stroke] {
  stroke: var(--jp-inverse-layout-color2);
}
.jp-icon-hover :hover .jp-icon3-hover[stroke] {
  stroke: var(--jp-inverse-layout-color3);
}
.jp-icon-hover :hover .jp-icon4-hover[stroke] {
  stroke: var(--jp-inverse-layout-color4);
}

/* recolor the accent elements of an icon */
.jp-icon-hover :hover .jp-icon-accent0-hover[fill] {
  fill: var(--jp-layout-color0);
}
.jp-icon-hover :hover .jp-icon-accent1-hover[fill] {
  fill: var(--jp-layout-color1);
}
.jp-icon-hover :hover .jp-icon-accent2-hover[fill] {
  fill: var(--jp-layout-color2);
}
.jp-icon-hover :hover .jp-icon-accent3-hover[fill] {
  fill: var(--jp-layout-color3);
}
.jp-icon-hover :hover .jp-icon-accent4-hover[fill] {
  fill: var(--jp-layout-color4);
}

.jp-icon-hover :hover .jp-icon-accent0-hover[stroke] {
  stroke: var(--jp-layout-color0);
}
.jp-icon-hover :hover .jp-icon-accent1-hover[stroke] {
  stroke: var(--jp-layout-color1);
}
.jp-icon-hover :hover .jp-icon-accent2-hover[stroke] {
  stroke: var(--jp-layout-color2);
}
.jp-icon-hover :hover .jp-icon-accent3-hover[stroke] {
  stroke: var(--jp-layout-color3);
}
.jp-icon-hover :hover .jp-icon-accent4-hover[stroke] {
  stroke: var(--jp-layout-color4);
}

/* set the color of an icon to transparent */
.jp-icon-hover :hover .jp-icon-none-hover[fill] {
  fill: none;
}

.jp-icon-hover :hover .jp-icon-none-hover[stroke] {
  stroke: none;
}

/**
 * inverse colors
 */

/* inverse recolor the primary elements of an icon */
.jp-icon-hover.jp-icon-alt :hover .jp-icon0-hover[fill] {
  fill: var(--jp-layout-color0);
}
.jp-icon-hover.jp-icon-alt :hover .jp-icon1-hover[fill] {
  fill: var(--jp-layout-color1);
}
.jp-icon-hover.jp-icon-alt :hover .jp-icon2-hover[fill] {
  fill: var(--jp-layout-color2);
}
.jp-icon-hover.jp-icon-alt :hover .jp-icon3-hover[fill] {
  fill: var(--jp-layout-color3);
}
.jp-icon-hover.jp-icon-alt :hover .jp-icon4-hover[fill] {
  fill: var(--jp-layout-color4);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon0-hover[stroke] {
  stroke: var(--jp-layout-color0);
}
.jp-icon-hover.jp-icon-alt :hover .jp-icon1-hover[stroke] {
  stroke: var(--jp-layout-color1);
}
.jp-icon-hover.jp-icon-alt :hover .jp-icon2-hover[stroke] {
  stroke: var(--jp-layout-color2);
}
.jp-icon-hover.jp-icon-alt :hover .jp-icon3-hover[stroke] {
  stroke: var(--jp-layout-color3);
}
.jp-icon-hover.jp-icon-alt :hover .jp-icon4-hover[stroke] {
  stroke: var(--jp-layout-color4);
}

/* inverse recolor the accent elements of an icon */
.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent0-hover[fill] {
  fill: var(--jp-inverse-layout-color0);
}
.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent1-hover[fill] {
  fill: var(--jp-inverse-layout-color1);
}
.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent2-hover[fill] {
  fill: var(--jp-inverse-layout-color2);
}
.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent3-hover[fill] {
  fill: var(--jp-inverse-layout-color3);
}
.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent4-hover[fill] {
  fill: var(--jp-inverse-layout-color4);
}

.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent0-hover[stroke] {
  stroke: var(--jp-inverse-layout-color0);
}
.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent1-hover[stroke] {
  stroke: var(--jp-inverse-layout-color1);
}
.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent2-hover[stroke] {
  stroke: var(--jp-inverse-layout-color2);
}
.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent3-hover[stroke] {
  stroke: var(--jp-inverse-layout-color3);
}
.jp-icon-hover.jp-icon-alt :hover .jp-icon-accent4-hover[stroke] {
  stroke: var(--jp-inverse-layout-color4);
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-switch {
  display: flex;
  align-items: center;
  padding-left: 4px;
  padding-right: 4px;
  font-size: var(--jp-ui-font-size1);
  background-color: transparent;
  color: var(--jp-ui-font-color1);
  border: none;
  height: 20px;
}

.jp-switch:hover {
  background-color: var(--jp-layout-color2);
}

.jp-switch-label {
  margin-right: 5px;
}

.jp-switch-track {
  cursor: pointer;
  background-color: var(--jp-border-color1);
  -webkit-transition: 0.4s;
  transition: 0.4s;
  border-radius: 34px;
  height: 16px;
  width: 35px;
  position: relative;
}

.jp-switch-track::before {
  content: '';
  position: absolute;
  height: 10px;
  width: 10px;
  margin: 3px;
  left: 0px;
  background-color: var(--jp-ui-inverse-font-color1);
  -webkit-transition: 0.4s;
  transition: 0.4s;
  border-radius: 50%;
}

.jp-switch[aria-checked='true'] .jp-switch-track {
  background-color: var(--jp-warn-color0);
}

.jp-switch[aria-checked='true'] .jp-switch-track::before {
  /* track width (35) - margins (3 + 3) - thumb width (10) */
  left: 19px;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/* Sibling imports */

/* Override Blueprint's _reset.scss styles */
html {
  box-sizing: unset;
}

*,
*::before,
*::after {
  box-sizing: unset;
}

body {
  color: unset;
  font-family: var(--jp-ui-font-family);
}

p {
  margin-top: unset;
  margin-bottom: unset;
}

small {
  font-size: unset;
}

strong {
  font-weight: unset;
}

/* Override Blueprint's _typography.scss styles */
a {
  text-decoration: unset;
  color: unset;
}
a:hover {
  text-decoration: unset;
  color: unset;
}

/* Override Blueprint's _accessibility.scss styles */
:focus {
  outline: unset;
  outline-offset: unset;
  -moz-outline-radius: unset;
}

/* Styles for ui-components */
.jp-Button {
  border-radius: var(--jp-border-radius);
  padding: 0px 12px;
  font-size: var(--jp-ui-font-size1);
}

/* Use our own theme for hover styles */
button.jp-Button.bp3-button.bp3-minimal:hover {
  background-color: var(--jp-layout-color2);
}
.jp-Button.minimal {
  color: unset !important;
}

.jp-Button.jp-ToolbarButtonComponent {
  text-transform: none;
}

.jp-InputGroup input {
  box-sizing: border-box;
  border-radius: 0;
  background-color: transparent;
  color: var(--jp-ui-font-color0);
  box-shadow: inset 0 0 0 var(--jp-border-width) var(--jp-input-border-color);
}

.jp-InputGroup input:focus {
  box-shadow: inset 0 0 0 var(--jp-border-width)
      var(--jp-input-active-box-shadow-color),
    inset 0 0 0 3px var(--jp-input-active-box-shadow-color);
}

.jp-InputGroup input::placeholder,
input::placeholder {
  color: var(--jp-ui-font-color3);
}

.jp-BPIcon {
  display: inline-block;
  vertical-align: middle;
  margin: auto;
}

/* Stop blueprint futzing with our icon fills */
.bp3-icon.jp-BPIcon > svg:not([fill]) {
  fill: var(--jp-inverse-layout-color3);
}

.jp-InputGroupAction {
  padding: 6px;
}

.jp-HTMLSelect.jp-DefaultStyle select {
  background-color: initial;
  border: none;
  border-radius: 0;
  box-shadow: none;
  color: var(--jp-ui-font-color0);
  display: block;
  font-size: var(--jp-ui-font-size1);
  height: 24px;
  line-height: 14px;
  padding: 0 25px 0 10px;
  text-align: left;
  -moz-appearance: none;
  -webkit-appearance: none;
}

/* Use our own theme for hover and option styles */
.jp-HTMLSelect.jp-DefaultStyle select:hover,
.jp-HTMLSelect.jp-DefaultStyle select > option {
  background-color: var(--jp-layout-color2);
  color: var(--jp-ui-font-color0);
}
select {
  box-sizing: border-box;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-Collapse {
  display: flex;
  flex-direction: column;
  align-items: stretch;
  border-top: 1px solid var(--jp-border-color2);
  border-bottom: 1px solid var(--jp-border-color2);
}

.jp-Collapse-header {
  padding: 1px 12px;
  color: var(--jp-ui-font-color1);
  background-color: var(--jp-layout-color1);
  font-size: var(--jp-ui-font-size2);
}

.jp-Collapse-header:hover {
  background-color: var(--jp-layout-color2);
}

.jp-Collapse-contents {
  padding: 0px 12px 0px 12px;
  background-color: var(--jp-layout-color1);
  color: var(--jp-ui-font-color1);
  overflow: auto;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Variables
|----------------------------------------------------------------------------*/

:root {
  --jp-private-commandpalette-search-height: 28px;
}

/*-----------------------------------------------------------------------------
| Overall styles
|----------------------------------------------------------------------------*/

.lm-CommandPalette {
  padding-bottom: 0px;
  color: var(--jp-ui-font-color1);
  background: var(--jp-layout-color1);
  /* This is needed so that all font sizing of children done in ems is
   * relative to this base size */
  font-size: var(--jp-ui-font-size1);
}

/*-----------------------------------------------------------------------------
| Modal variant
|----------------------------------------------------------------------------*/

.jp-ModalCommandPalette {
  position: absolute;
  z-index: 10000;
  top: 38px;
  left: 30%;
  margin: 0;
  padding: 4px;
  width: 40%;
  box-shadow: var(--jp-elevation-z4);
  border-radius: 4px;
  background: var(--jp-layout-color0);
}

.jp-ModalCommandPalette .lm-CommandPalette {
  max-height: 40vh;
}

.jp-ModalCommandPalette .lm-CommandPalette .lm-close-icon::after {
  display: none;
}

.jp-ModalCommandPalette .lm-CommandPalette .lm-CommandPalette-header {
  display: none;
}

.jp-ModalCommandPalette .lm-CommandPalette .lm-CommandPalette-item {
  margin-left: 4px;
  margin-right: 4px;
}

.jp-ModalCommandPalette
  .lm-CommandPalette
  .lm-CommandPalette-item.lm-mod-disabled {
  display: none;
}

/*-----------------------------------------------------------------------------
| Search
|----------------------------------------------------------------------------*/

.lm-CommandPalette-search {
  padding: 4px;
  background-color: var(--jp-layout-color1);
  z-index: 2;
}

.lm-CommandPalette-wrapper {
  overflow: overlay;
  padding: 0px 9px;
  background-color: var(--jp-input-active-background);
  height: 30px;
  box-shadow: inset 0 0 0 var(--jp-border-width) var(--jp-input-border-color);
}

.lm-CommandPalette.lm-mod-focused .lm-CommandPalette-wrapper {
  box-shadow: inset 0 0 0 1px var(--jp-input-active-box-shadow-color),
    inset 0 0 0 3px var(--jp-input-active-box-shadow-color);
}

.lm-CommandPalette-wrapper::after {
  content: ' ';
  color: white;
  background-color: var(--jp-brand-color1);
  position: absolute;
  top: 4px;
  right: 4px;
  height: 30px;
  width: 10px;
  padding: 0px 10px;
  background-image: var(--jp-icon-search-white);
  background-size: 20px;
  background-repeat: no-repeat;
  background-position: center;
}

.lm-CommandPalette-input {
  background: transparent;
  width: calc(100% - 18px);
  float: left;
  border: none;
  outline: none;
  font-size: var(--jp-ui-font-size1);
  color: var(--jp-ui-font-color0);
  line-height: var(--jp-private-commandpalette-search-height);
}

.lm-CommandPalette-input::-webkit-input-placeholder,
.lm-CommandPalette-input::-moz-placeholder,
.lm-CommandPalette-input:-ms-input-placeholder {
  color: var(--jp-ui-font-color3);
  font-size: var(--jp-ui-font-size1);
}

/*-----------------------------------------------------------------------------
| Results
|----------------------------------------------------------------------------*/

.lm-CommandPalette-header:first-child {
  margin-top: 0px;
}

.lm-CommandPalette-header {
  border-bottom: solid var(--jp-border-width) var(--jp-border-color2);
  color: var(--jp-ui-font-color1);
  cursor: pointer;
  display: flex;
  font-size: var(--jp-ui-font-size0);
  font-weight: 600;
  letter-spacing: 1px;
  margin-top: 8px;
  padding: 8px 0 8px 12px;
  text-transform: uppercase;
}

.lm-CommandPalette-header.lm-mod-active {
  background: var(--jp-layout-color2);
}

.lm-CommandPalette-header > mark {
  background-color: transparent;
  font-weight: bold;
  color: var(--jp-ui-font-color1);
}

.lm-CommandPalette-item {
  padding: 4px 12px 4px 4px;
  color: var(--jp-ui-font-color1);
  font-size: var(--jp-ui-font-size1);
  font-weight: 400;
  display: flex;
}

.lm-CommandPalette-item.lm-mod-disabled {
  color: var(--jp-ui-font-color3);
}

.lm-CommandPalette-item.lm-mod-active {
  background: var(--jp-layout-color3);
}

.lm-CommandPalette-item.lm-mod-active:hover:not(.lm-mod-disabled) {
  background: var(--jp-layout-color4);
}

.lm-CommandPalette-item:hover:not(.lm-mod-active):not(.lm-mod-disabled) {
  background: var(--jp-layout-color2);
}

.lm-CommandPalette-itemContent {
  overflow: hidden;
}

.lm-CommandPalette-itemLabel > mark {
  color: var(--jp-ui-font-color0);
  background-color: transparent;
  font-weight: bold;
}

.lm-CommandPalette-item.lm-mod-disabled mark {
  color: var(--jp-ui-font-color3);
}

.lm-CommandPalette-item .lm-CommandPalette-itemIcon {
  margin: 0 4px 0 0;
  position: relative;
  width: 16px;
  top: 2px;
  flex: 0 0 auto;
}

.lm-CommandPalette-item.lm-mod-disabled .lm-CommandPalette-itemIcon {
  opacity: 0.4;
}

.lm-CommandPalette-item .lm-CommandPalette-itemShortcut {
  flex: 0 0 auto;
}

.lm-CommandPalette-itemCaption {
  display: none;
}

.lm-CommandPalette-content {
  background-color: var(--jp-layout-color1);
}

.lm-CommandPalette-content:empty:after {
  content: 'No results';
  margin: auto;
  margin-top: 20px;
  width: 100px;
  display: block;
  font-size: var(--jp-ui-font-size2);
  font-family: var(--jp-ui-font-family);
  font-weight: lighter;
}

.lm-CommandPalette-emptyMessage {
  text-align: center;
  margin-top: 24px;
  line-height: 1.32;
  padding: 0px 8px;
  color: var(--jp-content-font-color3);
}

/*-----------------------------------------------------------------------------
| Copyright (c) 2014-2017, Jupyter Development Team.
|
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-Dialog {
  position: absolute;
  z-index: 10000;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  top: 0px;
  left: 0px;
  margin: 0;
  padding: 0;
  width: 100%;
  height: 100%;
  background: var(--jp-dialog-background);
}

.jp-Dialog-content {
  display: flex;
  flex-direction: column;
  margin-left: auto;
  margin-right: auto;
  background: var(--jp-layout-color1);
  padding: 24px;
  padding-bottom: 12px;
  min-width: 300px;
  min-height: 150px;
  max-width: 1000px;
  max-height: 500px;
  box-sizing: border-box;
  box-shadow: var(--jp-elevation-z20);
  word-wrap: break-word;
  border-radius: var(--jp-border-radius);
  /* This is needed so that all font sizing of children done in ems is
   * relative to this base size */
  font-size: var(--jp-ui-font-size1);
  color: var(--jp-ui-font-color1);
  resize: both;
}

.jp-Dialog-button {
  overflow: visible;
}

button.jp-Dialog-button:focus {
  outline: 1px solid var(--jp-brand-color1);
  outline-offset: 4px;
  -moz-outline-radius: 0px;
}

button.jp-Dialog-button:focus::-moz-focus-inner {
  border: 0;
}

button.jp-Dialog-close-button {
  padding: 0;
  height: 100%;
  min-width: unset;
  min-height: unset;
}

.jp-Dialog-header {
  display: flex;
  justify-content: space-between;
  flex: 0 0 auto;
  padding-bottom: 12px;
  font-size: var(--jp-ui-font-size3);
  font-weight: 400;
  color: var(--jp-ui-font-color0);
}

.jp-Dialog-body {
  display: flex;
  flex-direction: column;
  flex: 1 1 auto;
  font-size: var(--jp-ui-font-size1);
  background: var(--jp-layout-color1);
  overflow: auto;
}

.jp-Dialog-footer {
  display: flex;
  flex-direction: row;
  justify-content: flex-end;
  flex: 0 0 auto;
  margin-left: -12px;
  margin-right: -12px;
  padding: 12px;
}

.jp-Dialog-title {
  overflow: hidden;
  white-space: nowrap;
  text-overflow: ellipsis;
}

.jp-Dialog-body > .jp-select-wrapper {
  width: 100%;
}

.jp-Dialog-body > button {
  padding: 0px 16px;
}

.jp-Dialog-body > label {
  line-height: 1.4;
  color: var(--jp-ui-font-color0);
}

.jp-Dialog-button.jp-mod-styled:not(:last-child) {
  margin-right: 12px;
}

/*-----------------------------------------------------------------------------
| Copyright (c) 2014-2016, Jupyter Development Team.
|
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-HoverBox {
  position: fixed;
}

.jp-HoverBox.jp-mod-outofview {
  display: none;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-IFrame {
  width: 100%;
  height: 100%;
}

.jp-IFrame > iframe {
  border: none;
}

/*
When drag events occur, `p-mod-override-cursor` is added to the body.
Because iframes steal all cursor events, the following two rules are necessary
to suppress pointer events while resize drags are occurring. There may be a
better solution to this problem.
*/
body.lm-mod-override-cursor .jp-IFrame {
  position: relative;
}

body.lm-mod-override-cursor .jp-IFrame:before {
  content: '';
  position: absolute;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
  background: transparent;
}

/*-----------------------------------------------------------------------------
| Copyright (c) 2014-2016, Jupyter Development Team.
|
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-MainAreaWidget > :focus {
  outline: none;
}

/**
 * google-material-color v1.2.6
 * https://github.com/danlevan/google-material-color
 */
:root {
  --md-red-50: #ffebee;
  --md-red-100: #ffcdd2;
  --md-red-200: #ef9a9a;
  --md-red-300: #e57373;
  --md-red-400: #ef5350;
  --md-red-500: #f44336;
  --md-red-600: #e53935;
  --md-red-700: #d32f2f;
  --md-red-800: #c62828;
  --md-red-900: #b71c1c;
  --md-red-A100: #ff8a80;
  --md-red-A200: #ff5252;
  --md-red-A400: #ff1744;
  --md-red-A700: #d50000;

  --md-pink-50: #fce4ec;
  --md-pink-100: #f8bbd0;
  --md-pink-200: #f48fb1;
  --md-pink-300: #f06292;
  --md-pink-400: #ec407a;
  --md-pink-500: #e91e63;
  --md-pink-600: #d81b60;
  --md-pink-700: #c2185b;
  --md-pink-800: #ad1457;
  --md-pink-900: #880e4f;
  --md-pink-A100: #ff80ab;
  --md-pink-A200: #ff4081;
  --md-pink-A400: #f50057;
  --md-pink-A700: #c51162;

  --md-purple-50: #f3e5f5;
  --md-purple-100: #e1bee7;
  --md-purple-200: #ce93d8;
  --md-purple-300: #ba68c8;
  --md-purple-400: #ab47bc;
  --md-purple-500: #9c27b0;
  --md-purple-600: #8e24aa;
  --md-purple-700: #7b1fa2;
  --md-purple-800: #6a1b9a;
  --md-purple-900: #4a148c;
  --md-purple-A100: #ea80fc;
  --md-purple-A200: #e040fb;
  --md-purple-A400: #d500f9;
  --md-purple-A700: #aa00ff;

  --md-deep-purple-50: #ede7f6;
  --md-deep-purple-100: #d1c4e9;
  --md-deep-purple-200: #b39ddb;
  --md-deep-purple-300: #9575cd;
  --md-deep-purple-400: #7e57c2;
  --md-deep-purple-500: #673ab7;
  --md-deep-purple-600: #5e35b1;
  --md-deep-purple-700: #512da8;
  --md-deep-purple-800: #4527a0;
  --md-deep-purple-900: #311b92;
  --md-deep-purple-A100: #b388ff;
  --md-deep-purple-A200: #7c4dff;
  --md-deep-purple-A400: #651fff;
  --md-deep-purple-A700: #6200ea;

  --md-indigo-50: #e8eaf6;
  --md-indigo-100: #c5cae9;
  --md-indigo-200: #9fa8da;
  --md-indigo-300: #7986cb;
  --md-indigo-400: #5c6bc0;
  --md-indigo-500: #3f51b5;
  --md-indigo-600: #3949ab;
  --md-indigo-700: #303f9f;
  --md-indigo-800: #283593;
  --md-indigo-900: #1a237e;
  --md-indigo-A100: #8c9eff;
  --md-indigo-A200: #536dfe;
  --md-indigo-A400: #3d5afe;
  --md-indigo-A700: #304ffe;

  --md-blue-50: #e3f2fd;
  --md-blue-100: #bbdefb;
  --md-blue-200: #90caf9;
  --md-blue-300: #64b5f6;
  --md-blue-400: #42a5f5;
  --md-blue-500: #2196f3;
  --md-blue-600: #1e88e5;
  --md-blue-700: #1976d2;
  --md-blue-800: #1565c0;
  --md-blue-900: #0d47a1;
  --md-blue-A100: #82b1ff;
  --md-blue-A200: #448aff;
  --md-blue-A400: #2979ff;
  --md-blue-A700: #2962ff;

  --md-light-blue-50: #e1f5fe;
  --md-light-blue-100: #b3e5fc;
  --md-light-blue-200: #81d4fa;
  --md-light-blue-300: #4fc3f7;
  --md-light-blue-400: #29b6f6;
  --md-light-blue-500: #03a9f4;
  --md-light-blue-600: #039be5;
  --md-light-blue-700: #0288d1;
  --md-light-blue-800: #0277bd;
  --md-light-blue-900: #01579b;
  --md-light-blue-A100: #80d8ff;
  --md-light-blue-A200: #40c4ff;
  --md-light-blue-A400: #00b0ff;
  --md-light-blue-A700: #0091ea;

  --md-cyan-50: #e0f7fa;
  --md-cyan-100: #b2ebf2;
  --md-cyan-200: #80deea;
  --md-cyan-300: #4dd0e1;
  --md-cyan-400: #26c6da;
  --md-cyan-500: #00bcd4;
  --md-cyan-600: #00acc1;
  --md-cyan-700: #0097a7;
  --md-cyan-800: #00838f;
  --md-cyan-900: #006064;
  --md-cyan-A100: #84ffff;
  --md-cyan-A200: #18ffff;
  --md-cyan-A400: #00e5ff;
  --md-cyan-A700: #00b8d4;

  --md-teal-50: #e0f2f1;
  --md-teal-100: #b2dfdb;
  --md-teal-200: #80cbc4;
  --md-teal-300: #4db6ac;
  --md-teal-400: #26a69a;
  --md-teal-500: #009688;
  --md-teal-600: #00897b;
  --md-teal-700: #00796b;
  --md-teal-800: #00695c;
  --md-teal-900: #004d40;
  --md-teal-A100: #a7ffeb;
  --md-teal-A200: #64ffda;
  --md-teal-A400: #1de9b6;
  --md-teal-A700: #00bfa5;

  --md-green-50: #e8f5e9;
  --md-green-100: #c8e6c9;
  --md-green-200: #a5d6a7;
  --md-green-300: #81c784;
  --md-green-400: #66bb6a;
  --md-green-500: #4caf50;
  --md-green-600: #43a047;
  --md-green-700: #388e3c;
  --md-green-800: #2e7d32;
  --md-green-900: #1b5e20;
  --md-green-A100: #b9f6ca;
  --md-green-A200: #69f0ae;
  --md-green-A400: #00e676;
  --md-green-A700: #00c853;

  --md-light-green-50: #f1f8e9;
  --md-light-green-100: #dcedc8;
  --md-light-green-200: #c5e1a5;
  --md-light-green-300: #aed581;
  --md-light-green-400: #9ccc65;
  --md-light-green-500: #8bc34a;
  --md-light-green-600: #7cb342;
  --md-light-green-700: #689f38;
  --md-light-green-800: #558b2f;
  --md-light-green-900: #33691e;
  --md-light-green-A100: #ccff90;
  --md-light-green-A200: #b2ff59;
  --md-light-green-A400: #76ff03;
  --md-light-green-A700: #64dd17;

  --md-lime-50: #f9fbe7;
  --md-lime-100: #f0f4c3;
  --md-lime-200: #e6ee9c;
  --md-lime-300: #dce775;
  --md-lime-400: #d4e157;
  --md-lime-500: #cddc39;
  --md-lime-600: #c0ca33;
  --md-lime-700: #afb42b;
  --md-lime-800: #9e9d24;
  --md-lime-900: #827717;
  --md-lime-A100: #f4ff81;
  --md-lime-A200: #eeff41;
  --md-lime-A400: #c6ff00;
  --md-lime-A700: #aeea00;

  --md-yellow-50: #fffde7;
  --md-yellow-100: #fff9c4;
  --md-yellow-200: #fff59d;
  --md-yellow-300: #fff176;
  --md-yellow-400: #ffee58;
  --md-yellow-500: #ffeb3b;
  --md-yellow-600: #fdd835;
  --md-yellow-700: #fbc02d;
  --md-yellow-800: #f9a825;
  --md-yellow-900: #f57f17;
  --md-yellow-A100: #ffff8d;
  --md-yellow-A200: #ffff00;
  --md-yellow-A400: #ffea00;
  --md-yellow-A700: #ffd600;

  --md-amber-50: #fff8e1;
  --md-amber-100: #ffecb3;
  --md-amber-200: #ffe082;
  --md-amber-300: #ffd54f;
  --md-amber-400: #ffca28;
  --md-amber-500: #ffc107;
  --md-amber-600: #ffb300;
  --md-amber-700: #ffa000;
  --md-amber-800: #ff8f00;
  --md-amber-900: #ff6f00;
  --md-amber-A100: #ffe57f;
  --md-amber-A200: #ffd740;
  --md-amber-A400: #ffc400;
  --md-amber-A700: #ffab00;

  --md-orange-50: #fff3e0;
  --md-orange-100: #ffe0b2;
  --md-orange-200: #ffcc80;
  --md-orange-300: #ffb74d;
  --md-orange-400: #ffa726;
  --md-orange-500: #ff9800;
  --md-orange-600: #fb8c00;
  --md-orange-700: #f57c00;
  --md-orange-800: #ef6c00;
  --md-orange-900: #e65100;
  --md-orange-A100: #ffd180;
  --md-orange-A200: #ffab40;
  --md-orange-A400: #ff9100;
  --md-orange-A700: #ff6d00;

  --md-deep-orange-50: #fbe9e7;
  --md-deep-orange-100: #ffccbc;
  --md-deep-orange-200: #ffab91;
  --md-deep-orange-300: #ff8a65;
  --md-deep-orange-400: #ff7043;
  --md-deep-orange-500: #ff5722;
  --md-deep-orange-600: #f4511e;
  --md-deep-orange-700: #e64a19;
  --md-deep-orange-800: #d84315;
  --md-deep-orange-900: #bf360c;
  --md-deep-orange-A100: #ff9e80;
  --md-deep-orange-A200: #ff6e40;
  --md-deep-orange-A400: #ff3d00;
  --md-deep-orange-A700: #dd2c00;

  --md-brown-50: #efebe9;
  --md-brown-100: #d7ccc8;
  --md-brown-200: #bcaaa4;
  --md-brown-300: #a1887f;
  --md-brown-400: #8d6e63;
  --md-brown-500: #795548;
  --md-brown-600: #6d4c41;
  --md-brown-700: #5d4037;
  --md-brown-800: #4e342e;
  --md-brown-900: #3e2723;

  --md-grey-50: #fafafa;
  --md-grey-100: #f5f5f5;
  --md-grey-200: #eeeeee;
  --md-grey-300: #e0e0e0;
  --md-grey-400: #bdbdbd;
  --md-grey-500: #9e9e9e;
  --md-grey-600: #757575;
  --md-grey-700: #616161;
  --md-grey-800: #424242;
  --md-grey-900: #212121;

  --md-blue-grey-50: #eceff1;
  --md-blue-grey-100: #cfd8dc;
  --md-blue-grey-200: #b0bec5;
  --md-blue-grey-300: #90a4ae;
  --md-blue-grey-400: #78909c;
  --md-blue-grey-500: #607d8b;
  --md-blue-grey-600: #546e7a;
  --md-blue-grey-700: #455a64;
  --md-blue-grey-800: #37474f;
  --md-blue-grey-900: #263238;
}

/*-----------------------------------------------------------------------------
| Copyright (c) 2017, Jupyter Development Team.
|
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-Spinner {
  position: absolute;
  display: flex;
  justify-content: center;
  align-items: center;
  z-index: 10;
  left: 0;
  top: 0;
  width: 100%;
  height: 100%;
  background: var(--jp-layout-color0);
  outline: none;
}

.jp-SpinnerContent {
  font-size: 10px;
  margin: 50px auto;
  text-indent: -9999em;
  width: 3em;
  height: 3em;
  border-radius: 50%;
  background: var(--jp-brand-color3);
  background: linear-gradient(
    to right,
    #f37626 10%,
    rgba(255, 255, 255, 0) 42%
  );
  position: relative;
  animation: load3 1s infinite linear, fadeIn 1s;
}

.jp-SpinnerContent:before {
  width: 50%;
  height: 50%;
  background: #f37626;
  border-radius: 100% 0 0 0;
  position: absolute;
  top: 0;
  left: 0;
  content: '';
}

.jp-SpinnerContent:after {
  background: var(--jp-layout-color0);
  width: 75%;
  height: 75%;
  border-radius: 50%;
  content: '';
  margin: auto;
  position: absolute;
  top: 0;
  left: 0;
  bottom: 0;
  right: 0;
}

@keyframes fadeIn {
  0% {
    opacity: 0;
  }
  100% {
    opacity: 1;
  }
}

@keyframes load3 {
  0% {
    transform: rotate(0deg);
  }
  100% {
    transform: rotate(360deg);
  }
}

/*-----------------------------------------------------------------------------
| Copyright (c) 2014-2017, Jupyter Development Team.
|
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

button.jp-mod-styled {
  font-size: var(--jp-ui-font-size1);
  color: var(--jp-ui-font-color0);
  border: none;
  box-sizing: border-box;
  text-align: center;
  line-height: 32px;
  height: 32px;
  padding: 0px 12px;
  letter-spacing: 0.8px;
  outline: none;
  appearance: none;
  -webkit-appearance: none;
  -moz-appearance: none;
}

input.jp-mod-styled {
  background: var(--jp-input-background);
  height: 28px;
  box-sizing: border-box;
  border: var(--jp-border-width) solid var(--jp-border-color1);
  padding-left: 7px;
  padding-right: 7px;
  font-size: var(--jp-ui-font-size2);
  color: var(--jp-ui-font-color0);
  outline: none;
  appearance: none;
  -webkit-appearance: none;
  -moz-appearance: none;
}

input.jp-mod-styled:focus {
  border: var(--jp-border-width) solid var(--md-blue-500);
  box-shadow: inset 0 0 4px var(--md-blue-300);
}

.jp-select-wrapper {
  display: flex;
  position: relative;
  flex-direction: column;
  padding: 1px;
  background-color: var(--jp-layout-color1);
  height: 28px;
  box-sizing: border-box;
  margin-bottom: 12px;
}

.jp-select-wrapper.jp-mod-focused select.jp-mod-styled {
  border: var(--jp-border-width) solid var(--jp-input-active-border-color);
  box-shadow: var(--jp-input-box-shadow);
  background-color: var(--jp-input-active-background);
}

select.jp-mod-styled:hover {
  background-color: var(--jp-layout-color1);
  cursor: pointer;
  color: var(--jp-ui-font-color0);
  background-color: var(--jp-input-hover-background);
  box-shadow: inset 0 0px 1px rgba(0, 0, 0, 0.5);
}

select.jp-mod-styled {
  flex: 1 1 auto;
  height: 32px;
  width: 100%;
  font-size: var(--jp-ui-font-size2);
  background: var(--jp-input-background);
  color: var(--jp-ui-font-color0);
  padding: 0 25px 0 8px;
  border: var(--jp-border-width) solid var(--jp-input-border-color);
  border-radius: 0px;
  outline: none;
  appearance: none;
  -webkit-appearance: none;
  -moz-appearance: none;
}

/*-----------------------------------------------------------------------------
| Copyright (c) 2014-2016, Jupyter Development Team.
|
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

:root {
  --jp-private-toolbar-height: calc(
    28px + var(--jp-border-width)
  ); /* leave 28px for content */
}

.jp-Toolbar {
  color: var(--jp-ui-font-color1);
  flex: 0 0 auto;
  display: flex;
  flex-direction: row;
  border-bottom: var(--jp-border-width) solid var(--jp-toolbar-border-color);
  box-shadow: var(--jp-toolbar-box-shadow);
  background: var(--jp-toolbar-background);
  min-height: var(--jp-toolbar-micro-height);
  padding: 2px;
  z-index: 1;
  overflow-x: hidden;
}

.jp-Toolbar:hover {
  overflow-x: auto;
}

/* Toolbar items */

.jp-Toolbar > .jp-Toolbar-item.jp-Toolbar-spacer {
  flex-grow: 1;
  flex-shrink: 1;
}

.jp-Toolbar-item.jp-Toolbar-kernelStatus {
  display: inline-block;
  width: 32px;
  background-repeat: no-repeat;
  background-position: center;
  background-size: 16px;
}

.jp-Toolbar > .jp-Toolbar-item {
  flex: 0 0 auto;
  display: flex;
  padding-left: 1px;
  padding-right: 1px;
  font-size: var(--jp-ui-font-size1);
  line-height: var(--jp-private-toolbar-height);
  height: 100%;
}

/* Toolbar buttons */

/* This is the div we use to wrap the react component into a Widget */
div.jp-ToolbarButton {
  color: transparent;
  border: none;
  box-sizing: border-box;
  outline: none;
  appearance: none;
  -webkit-appearance: none;
  -moz-appearance: none;
  padding: 0px;
  margin: 0px;
}

button.jp-ToolbarButtonComponent {
  background: var(--jp-layout-color1);
  border: none;
  box-sizing: border-box;
  outline: none;
  appearance: none;
  -webkit-appearance: none;
  -moz-appearance: none;
  padding: 0px 6px;
  margin: 0px;
  height: 24px;
  border-radius: var(--jp-border-radius);
  display: flex;
  align-items: center;
  text-align: center;
  font-size: 14px;
  min-width: unset;
  min-height: unset;
}

button.jp-ToolbarButtonComponent:disabled {
  opacity: 0.4;
}

button.jp-ToolbarButtonComponent span {
  padding: 0px;
  flex: 0 0 auto;
}

button.jp-ToolbarButtonComponent .jp-ToolbarButtonComponent-label {
  font-size: var(--jp-ui-font-size1);
  line-height: 100%;
  padding-left: 2px;
  color: var(--jp-ui-font-color1);
}

#jp-main-dock-panel[data-mode='single-document']
  .jp-MainAreaWidget
  > .jp-Toolbar.jp-Toolbar-micro {
  padding: 0;
  min-height: 0;
}

#jp-main-dock-panel[data-mode='single-document']
  .jp-MainAreaWidget
  > .jp-Toolbar {
  border: none;
  box-shadow: none;
}

/*-----------------------------------------------------------------------------
| Copyright (c) 2014-2017, Jupyter Development Team.
|
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Copyright (c) 2014-2017, PhosphorJS Contributors
|
| Distributed under the terms of the BSD 3-Clause License.
|
| The full license is in the file LICENSE, distributed with this software.
|----------------------------------------------------------------------------*/


/* <DEPRECATED> */ body.p-mod-override-cursor *, /* </DEPRECATED> */
body.lm-mod-override-cursor * {
  cursor: inherit !important;
}

/*-----------------------------------------------------------------------------
| Copyright (c) 2014-2016, Jupyter Development Team.
|
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-JSONEditor {
  display: flex;
  flex-direction: column;
  width: 100%;
}

.jp-JSONEditor-host {
  flex: 1 1 auto;
  border: var(--jp-border-width) solid var(--jp-input-border-color);
  border-radius: 0px;
  background: var(--jp-layout-color0);
  min-height: 50px;
  padding: 1px;
}

.jp-JSONEditor.jp-mod-error .jp-JSONEditor-host {
  border-color: red;
  outline-color: red;
}

.jp-JSONEditor-header {
  display: flex;
  flex: 1 0 auto;
  padding: 0 0 0 12px;
}

.jp-JSONEditor-header label {
  flex: 0 0 auto;
}

.jp-JSONEditor-commitButton {
  height: 16px;
  width: 16px;
  background-size: 18px;
  background-repeat: no-repeat;
  background-position: center;
}

.jp-JSONEditor-host.jp-mod-focused {
  background-color: var(--jp-input-active-background);
  border: 1px solid var(--jp-input-active-border-color);
  box-shadow: var(--jp-input-box-shadow);
}

.jp-Editor.jp-mod-dropTarget {
  border: var(--jp-border-width) solid var(--jp-input-active-border-color);
  box-shadow: var(--jp-input-box-shadow);
}

/* BASICS */

.CodeMirror {
  /* Set height, width, borders, and global font properties here */
  font-family: monospace;
  height: 300px;
  color: black;
  direction: ltr;
}

/* PADDING */

.CodeMirror-lines {
  padding: 4px 0; /* Vertical padding around content */
}
.CodeMirror pre.CodeMirror-line,
.CodeMirror pre.CodeMirror-line-like {
  padding: 0 4px; /* Horizontal padding of content */
}

.CodeMirror-scrollbar-filler, .CodeMirror-gutter-filler {
  background-color: white; /* The little square between H and V scrollbars */
}

/* GUTTER */

.CodeMirror-gutters {
  border-right: 1px solid #ddd;
  background-color: #f7f7f7;
  white-space: nowrap;
}
.CodeMirror-linenumbers {}
.CodeMirror-linenumber {
  padding: 0 3px 0 5px;
  min-width: 20px;
  text-align: right;
  color: #999;
  white-space: nowrap;
}

.CodeMirror-guttermarker { color: black; }
.CodeMirror-guttermarker-subtle { color: #999; }

/* CURSOR */

.CodeMirror-cursor {
  border-left: 1px solid black;
  border-right: none;
  width: 0;
}
/* Shown when moving in bi-directional text */
.CodeMirror div.CodeMirror-secondarycursor {
  border-left: 1px solid silver;
}
.cm-fat-cursor .CodeMirror-cursor {
  width: auto;
  border: 0 !important;
  background: #7e7;
}
.cm-fat-cursor div.CodeMirror-cursors {
  z-index: 1;
}
.cm-fat-cursor-mark {
  background-color: rgba(20, 255, 20, 0.5);
  -webkit-animation: blink 1.06s steps(1) infinite;
  -moz-animation: blink 1.06s steps(1) infinite;
  animation: blink 1.06s steps(1) infinite;
}
.cm-animate-fat-cursor {
  width: auto;
  border: 0;
  -webkit-animation: blink 1.06s steps(1) infinite;
  -moz-animation: blink 1.06s steps(1) infinite;
  animation: blink 1.06s steps(1) infinite;
  background-color: #7e7;
}
@-moz-keyframes blink {
  0% {}
  50% { background-color: transparent; }
  100% {}
}
@-webkit-keyframes blink {
  0% {}
  50% { background-color: transparent; }
  100% {}
}
@keyframes blink {
  0% {}
  50% { background-color: transparent; }
  100% {}
}

/* Can style cursor different in overwrite (non-insert) mode */
.CodeMirror-overwrite .CodeMirror-cursor {}

.cm-tab { display: inline-block; text-decoration: inherit; }

.CodeMirror-rulers {
  position: absolute;
  left: 0; right: 0; top: -50px; bottom: 0;
  overflow: hidden;
}
.CodeMirror-ruler {
  border-left: 1px solid #ccc;
  top: 0; bottom: 0;
  position: absolute;
}

/* DEFAULT THEME */

.cm-s-default .cm-header {color: blue;}
.cm-s-default .cm-quote {color: #090;}
.cm-negative {color: #d44;}
.cm-positive {color: #292;}
.cm-header, .cm-strong {font-weight: bold;}
.cm-em {font-style: italic;}
.cm-link {text-decoration: underline;}
.cm-strikethrough {text-decoration: line-through;}

.cm-s-default .cm-keyword {color: #708;}
.cm-s-default .cm-atom {color: #219;}
.cm-s-default .cm-number {color: #164;}
.cm-s-default .cm-def {color: #00f;}
.cm-s-default .cm-variable,
.cm-s-default .cm-punctuation,
.cm-s-default .cm-property,
.cm-s-default .cm-operator {}
.cm-s-default .cm-variable-2 {color: #05a;}
.cm-s-default .cm-variable-3, .cm-s-default .cm-type {color: #085;}
.cm-s-default .cm-comment {color: #a50;}
.cm-s-default .cm-string {color: #a11;}
.cm-s-default .cm-string-2 {color: #f50;}
.cm-s-default .cm-meta {color: #555;}
.cm-s-default .cm-qualifier {color: #555;}
.cm-s-default .cm-builtin {color: #30a;}
.cm-s-default .cm-bracket {color: #997;}
.cm-s-default .cm-tag {color: #170;}
.cm-s-default .cm-attribute {color: #00c;}
.cm-s-default .cm-hr {color: #999;}
.cm-s-default .cm-link {color: #00c;}

.cm-s-default .cm-error {color: #f00;}
.cm-invalidchar {color: #f00;}

.CodeMirror-composing { border-bottom: 2px solid; }

/* Default styles for common addons */

div.CodeMirror span.CodeMirror-matchingbracket {color: #0b0;}
div.CodeMirror span.CodeMirror-nonmatchingbracket {color: #a22;}
.CodeMirror-matchingtag { background: rgba(255, 150, 0, .3); }
.CodeMirror-activeline-background {background: #e8f2ff;}

/* STOP */

/* The rest of this file contains styles related to the mechanics of
   the editor. You probably shouldn't touch them. */

.CodeMirror {
  position: relative;
  overflow: hidden;
  background: white;
}

.CodeMirror-scroll {
  overflow: scroll !important; /* Things will break if this is overridden */
  /* 50px is the magic margin used to hide the element's real scrollbars */
  /* See overflow: hidden in .CodeMirror */
  margin-bottom: -50px; margin-right: -50px;
  padding-bottom: 50px;
  height: 100%;
  outline: none; /* Prevent dragging from highlighting the element */
  position: relative;
}
.CodeMirror-sizer {
  position: relative;
  border-right: 50px solid transparent;
}

/* The fake, visible scrollbars. Used to force redraw during scrolling
   before actual scrolling happens, thus preventing shaking and
   flickering artifacts. */
.CodeMirror-vscrollbar, .CodeMirror-hscrollbar, .CodeMirror-scrollbar-filler, .CodeMirror-gutter-filler {
  position: absolute;
  z-index: 6;
  display: none;
}
.CodeMirror-vscrollbar {
  right: 0; top: 0;
  overflow-x: hidden;
  overflow-y: scroll;
}
.CodeMirror-hscrollbar {
  bottom: 0; left: 0;
  overflow-y: hidden;
  overflow-x: scroll;
}
.CodeMirror-scrollbar-filler {
  right: 0; bottom: 0;
}
.CodeMirror-gutter-filler {
  left: 0; bottom: 0;
}

.CodeMirror-gutters {
  position: absolute; left: 0; top: 0;
  min-height: 100%;
  z-index: 3;
}
.CodeMirror-gutter {
  white-space: normal;
  height: 100%;
  display: inline-block;
  vertical-align: top;
  margin-bottom: -50px;
}
.CodeMirror-gutter-wrapper {
  position: absolute;
  z-index: 4;
  background: none !important;
  border: none !important;
}
.CodeMirror-gutter-background {
  position: absolute;
  top: 0; bottom: 0;
  z-index: 4;
}
.CodeMirror-gutter-elt {
  position: absolute;
  cursor: default;
  z-index: 4;
}
.CodeMirror-gutter-wrapper ::selection { background-color: transparent }
.CodeMirror-gutter-wrapper ::-moz-selection { background-color: transparent }

.CodeMirror-lines {
  cursor: text;
  min-height: 1px; /* prevents collapsing before first draw */
}
.CodeMirror pre.CodeMirror-line,
.CodeMirror pre.CodeMirror-line-like {
  /* Reset some styles that the rest of the page might have set */
  -moz-border-radius: 0; -webkit-border-radius: 0; border-radius: 0;
  border-width: 0;
  background: transparent;
  font-family: inherit;
  font-size: inherit;
  margin: 0;
  white-space: pre;
  word-wrap: normal;
  line-height: inherit;
  color: inherit;
  z-index: 2;
  position: relative;
  overflow: visible;
  -webkit-tap-highlight-color: transparent;
  -webkit-font-variant-ligatures: contextual;
  font-variant-ligatures: contextual;
}
.CodeMirror-wrap pre.CodeMirror-line,
.CodeMirror-wrap pre.CodeMirror-line-like {
  word-wrap: break-word;
  white-space: pre-wrap;
  word-break: normal;
}

.CodeMirror-linebackground {
  position: absolute;
  left: 0; right: 0; top: 0; bottom: 0;
  z-index: 0;
}

.CodeMirror-linewidget {
  position: relative;
  z-index: 2;
  padding: 0.1px; /* Force widget margins to stay inside of the container */
}

.CodeMirror-widget {}

.CodeMirror-rtl pre { direction: rtl; }

.CodeMirror-code {
  outline: none;
}

/* Force content-box sizing for the elements where we expect it */
.CodeMirror-scroll,
.CodeMirror-sizer,
.CodeMirror-gutter,
.CodeMirror-gutters,
.CodeMirror-linenumber {
  -moz-box-sizing: content-box;
  box-sizing: content-box;
}

.CodeMirror-measure {
  position: absolute;
  width: 100%;
  height: 0;
  overflow: hidden;
  visibility: hidden;
}

.CodeMirror-cursor {
  position: absolute;
  pointer-events: none;
}
.CodeMirror-measure pre { position: static; }

div.CodeMirror-cursors {
  visibility: hidden;
  position: relative;
  z-index: 3;
}
div.CodeMirror-dragcursors {
  visibility: visible;
}

.CodeMirror-focused div.CodeMirror-cursors {
  visibility: visible;
}

.CodeMirror-selected { background: #d9d9d9; }
.CodeMirror-focused .CodeMirror-selected { background: #d7d4f0; }
.CodeMirror-crosshair { cursor: crosshair; }
.CodeMirror-line::selection, .CodeMirror-line > span::selection, .CodeMirror-line > span > span::selection { background: #d7d4f0; }
.CodeMirror-line::-moz-selection, .CodeMirror-line > span::-moz-selection, .CodeMirror-line > span > span::-moz-selection { background: #d7d4f0; }

.cm-searching {
  background-color: #ffa;
  background-color: rgba(255, 255, 0, .4);
}

/* Used to force a border model for a node */
.cm-force-border { padding-right: .1px; }

@media print {
  /* Hide the cursor when printing */
  .CodeMirror div.CodeMirror-cursors {
    visibility: hidden;
  }
}

/* See issue #2901 */
.cm-tab-wrap-hack:after { content: ''; }

/* Help users use markselection to safely style text background */
span.CodeMirror-selectedtext { background: none; }

.CodeMirror-dialog {
  position: absolute;
  left: 0; right: 0;
  background: inherit;
  z-index: 15;
  padding: .1em .8em;
  overflow: hidden;
  color: inherit;
}

.CodeMirror-dialog-top {
  border-bottom: 1px solid #eee;
  top: 0;
}

.CodeMirror-dialog-bottom {
  border-top: 1px solid #eee;
  bottom: 0;
}

.CodeMirror-dialog input {
  border: none;
  outline: none;
  background: transparent;
  width: 20em;
  color: inherit;
  font-family: monospace;
}

.CodeMirror-dialog button {
  font-size: 70%;
}

.CodeMirror-foldmarker {
  color: blue;
  text-shadow: #b9f 1px 1px 2px, #b9f -1px -1px 2px, #b9f 1px -1px 2px, #b9f -1px 1px 2px;
  font-family: arial;
  line-height: .3;
  cursor: pointer;
}
.CodeMirror-foldgutter {
  width: .7em;
}
.CodeMirror-foldgutter-open,
.CodeMirror-foldgutter-folded {
  cursor: pointer;
}
.CodeMirror-foldgutter-open:after {
  content: "\25BE";
}
.CodeMirror-foldgutter-folded:after {
  content: "\25B8";
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.CodeMirror {
  line-height: var(--jp-code-line-height);
  font-size: var(--jp-code-font-size);
  font-family: var(--jp-code-font-family);
  border: 0;
  border-radius: 0;
  height: auto;
  /* Changed to auto to autogrow */
}

.CodeMirror pre {
  padding: 0 var(--jp-code-padding);
}

.jp-CodeMirrorEditor[data-type='inline'] .CodeMirror-dialog {
  background-color: var(--jp-layout-color0);
  color: var(--jp-content-font-color1);
}

/* This causes https://github.com/jupyter/jupyterlab/issues/522 */
/* May not cause it not because we changed it! */
.CodeMirror-lines {
  padding: var(--jp-code-padding) 0;
}

.CodeMirror-linenumber {
  padding: 0 8px;
}

.jp-CodeMirrorEditor {
  cursor: text;
}

.jp-CodeMirrorEditor[data-type='inline'] .CodeMirror-cursor {
  border-left: var(--jp-code-cursor-width0) solid var(--jp-editor-cursor-color);
}

/* When zoomed out 67% and 33% on a screen of 1440 width x 900 height */
@media screen and (min-width: 2138px) and (max-width: 4319px) {
  .jp-CodeMirrorEditor[data-type='inline'] .CodeMirror-cursor {
    border-left: var(--jp-code-cursor-width1) solid
      var(--jp-editor-cursor-color);
  }
}

/* When zoomed out less than 33% */
@media screen and (min-width: 4320px) {
  .jp-CodeMirrorEditor[data-type='inline'] .CodeMirror-cursor {
    border-left: var(--jp-code-cursor-width2) solid
      var(--jp-editor-cursor-color);
  }
}

.CodeMirror.jp-mod-readOnly .CodeMirror-cursor {
  display: none;
}

.CodeMirror-gutters {
  border-right: 1px solid var(--jp-border-color2);
  background-color: var(--jp-layout-color0);
}

.jp-CollaboratorCursor {
  border-left: 5px solid transparent;
  border-right: 5px solid transparent;
  border-top: none;
  border-bottom: 3px solid;
  background-clip: content-box;
  margin-left: -5px;
  margin-right: -5px;
}

.CodeMirror-selectedtext.cm-searching {
  background-color: var(--jp-search-selected-match-background-color) !important;
  color: var(--jp-search-selected-match-color) !important;
}

.cm-searching {
  background-color: var(
    --jp-search-unselected-match-background-color
  ) !important;
  color: var(--jp-search-unselected-match-color) !important;
}

.CodeMirror-focused .CodeMirror-selected {
  background-color: var(--jp-editor-selected-focused-background);
}

.CodeMirror-selected {
  background-color: var(--jp-editor-selected-background);
}

.jp-CollaboratorCursor-hover {
  position: absolute;
  z-index: 1;
  transform: translateX(-50%);
  color: white;
  border-radius: 3px;
  padding-left: 4px;
  padding-right: 4px;
  padding-top: 1px;
  padding-bottom: 1px;
  text-align: center;
  font-size: var(--jp-ui-font-size1);
  white-space: nowrap;
}

.jp-CodeMirror-ruler {
  border-left: 1px dashed var(--jp-border-color2);
}

/**
 * Here is our jupyter theme for CodeMirror syntax highlighting
 * This is used in our marked.js syntax highlighting and CodeMirror itself
 * The string "jupyter" is set in ../codemirror/widget.DEFAULT_CODEMIRROR_THEME
 * This came from the classic notebook, which came form highlight.js/GitHub
 */

/**
 * CodeMirror themes are handling the background/color in this way. This works
 * fine for CodeMirror editors outside the notebook, but the notebook styles
 * these things differently.
 */
.CodeMirror.cm-s-jupyter {
  background: var(--jp-layout-color0);
  color: var(--jp-content-font-color1);
}

/* In the notebook, we want this styling to be handled by its container */
.jp-CodeConsole .CodeMirror.cm-s-jupyter,
.jp-Notebook .CodeMirror.cm-s-jupyter {
  background: transparent;
}

.cm-s-jupyter .CodeMirror-cursor {
  border-left: var(--jp-code-cursor-width0) solid var(--jp-editor-cursor-color);
}
.cm-s-jupyter span.cm-keyword {
  color: var(--jp-mirror-editor-keyword-color);
  font-weight: bold;
}
.cm-s-jupyter span.cm-atom {
  color: var(--jp-mirror-editor-atom-color);
}
.cm-s-jupyter span.cm-number {
  color: var(--jp-mirror-editor-number-color);
}
.cm-s-jupyter span.cm-def {
  color: var(--jp-mirror-editor-def-color);
}
.cm-s-jupyter span.cm-variable {
  color: var(--jp-mirror-editor-variable-color);
}
.cm-s-jupyter span.cm-variable-2 {
  color: var(--jp-mirror-editor-variable-2-color);
}
.cm-s-jupyter span.cm-variable-3 {
  color: var(--jp-mirror-editor-variable-3-color);
}
.cm-s-jupyter span.cm-punctuation {
  color: var(--jp-mirror-editor-punctuation-color);
}
.cm-s-jupyter span.cm-property {
  color: var(--jp-mirror-editor-property-color);
}
.cm-s-jupyter span.cm-operator {
  color: var(--jp-mirror-editor-operator-color);
  font-weight: bold;
}
.cm-s-jupyter span.cm-comment {
  color: var(--jp-mirror-editor-comment-color);
  font-style: italic;
}
.cm-s-jupyter span.cm-string {
  color: var(--jp-mirror-editor-string-color);
}
.cm-s-jupyter span.cm-string-2 {
  color: var(--jp-mirror-editor-string-2-color);
}
.cm-s-jupyter span.cm-meta {
  color: var(--jp-mirror-editor-meta-color);
}
.cm-s-jupyter span.cm-qualifier {
  color: var(--jp-mirror-editor-qualifier-color);
}
.cm-s-jupyter span.cm-builtin {
  color: var(--jp-mirror-editor-builtin-color);
}
.cm-s-jupyter span.cm-bracket {
  color: var(--jp-mirror-editor-bracket-color);
}
.cm-s-jupyter span.cm-tag {
  color: var(--jp-mirror-editor-tag-color);
}
.cm-s-jupyter span.cm-attribute {
  color: var(--jp-mirror-editor-attribute-color);
}
.cm-s-jupyter span.cm-header {
  color: var(--jp-mirror-editor-header-color);
}
.cm-s-jupyter span.cm-quote {
  color: var(--jp-mirror-editor-quote-color);
}
.cm-s-jupyter span.cm-link {
  color: var(--jp-mirror-editor-link-color);
}
.cm-s-jupyter span.cm-error {
  color: var(--jp-mirror-editor-error-color);
}
.cm-s-jupyter span.cm-hr {
  color: #999;
}

.cm-s-jupyter span.cm-tab {
  background: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADAAAAAMCAYAAAAkuj5RAAAAAXNSR0IArs4c6QAAAGFJREFUSMft1LsRQFAQheHPowAKoACx3IgEKtaEHujDjORSgWTH/ZOdnZOcM/sgk/kFFWY0qV8foQwS4MKBCS3qR6ixBJvElOobYAtivseIE120FaowJPN75GMu8j/LfMwNjh4HUpwg4LUAAAAASUVORK5CYII=);
  background-position: right;
  background-repeat: no-repeat;
}

.cm-s-jupyter .CodeMirror-activeline-background,
.cm-s-jupyter .CodeMirror-gutter {
  background-color: var(--jp-layout-color2);
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| RenderedText
|----------------------------------------------------------------------------*/

:root {
  /* This is the padding value to fill the gaps between lines containing spans with background color. */
  --jp-private-code-span-padding: calc(
    (var(--jp-code-line-height) - 1) * var(--jp-code-font-size) / 2
  );
}

.jp-RenderedText {
  text-align: left;
  padding-left: var(--jp-code-padding);
  line-height: var(--jp-code-line-height);
  font-family: var(--jp-code-font-family);
}

.jp-RenderedText pre,
.jp-RenderedJavaScript pre,
.jp-RenderedHTMLCommon pre {
  color: var(--jp-content-font-color1);
  font-size: var(--jp-code-font-size);
  border: none;
  margin: 0px;
  padding: 0px;
}

.jp-RenderedText pre a:link {
  text-decoration: none;
  color: var(--jp-content-link-color);
}
.jp-RenderedText pre a:hover {
  text-decoration: underline;
  color: var(--jp-content-link-color);
}
.jp-RenderedText pre a:visited {
  text-decoration: none;
  color: var(--jp-content-link-color);
}

/* console foregrounds and backgrounds */
.jp-RenderedText pre .ansi-black-fg {
  color: #3e424d;
}
.jp-RenderedText pre .ansi-red-fg {
  color: #e75c58;
}
.jp-RenderedText pre .ansi-green-fg {
  color: #00a250;
}
.jp-RenderedText pre .ansi-yellow-fg {
  color: #ddb62b;
}
.jp-RenderedText pre .ansi-blue-fg {
  color: #208ffb;
}
.jp-RenderedText pre .ansi-magenta-fg {
  color: #d160c4;
}
.jp-RenderedText pre .ansi-cyan-fg {
  color: #60c6c8;
}
.jp-RenderedText pre .ansi-white-fg {
  color: #c5c1b4;
}

.jp-RenderedText pre .ansi-black-bg {
  background-color: #3e424d;
  padding: var(--jp-private-code-span-padding) 0;
}
.jp-RenderedText pre .ansi-red-bg {
  background-color: #e75c58;
  padding: var(--jp-private-code-span-padding) 0;
}
.jp-RenderedText pre .ansi-green-bg {
  background-color: #00a250;
  padding: var(--jp-private-code-span-padding) 0;
}
.jp-RenderedText pre .ansi-yellow-bg {
  background-color: #ddb62b;
  padding: var(--jp-private-code-span-padding) 0;
}
.jp-RenderedText pre .ansi-blue-bg {
  background-color: #208ffb;
  padding: var(--jp-private-code-span-padding) 0;
}
.jp-RenderedText pre .ansi-magenta-bg {
  background-color: #d160c4;
  padding: var(--jp-private-code-span-padding) 0;
}
.jp-RenderedText pre .ansi-cyan-bg {
  background-color: #60c6c8;
  padding: var(--jp-private-code-span-padding) 0;
}
.jp-RenderedText pre .ansi-white-bg {
  background-color: #c5c1b4;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-black-intense-fg {
  color: #282c36;
}
.jp-RenderedText pre .ansi-red-intense-fg {
  color: #b22b31;
}
.jp-RenderedText pre .ansi-green-intense-fg {
  color: #007427;
}
.jp-RenderedText pre .ansi-yellow-intense-fg {
  color: #b27d12;
}
.jp-RenderedText pre .ansi-blue-intense-fg {
  color: #0065ca;
}
.jp-RenderedText pre .ansi-magenta-intense-fg {
  color: #a03196;
}
.jp-RenderedText pre .ansi-cyan-intense-fg {
  color: #258f8f;
}
.jp-RenderedText pre .ansi-white-intense-fg {
  color: #a1a6b2;
}

.jp-RenderedText pre .ansi-black-intense-bg {
  background-color: #282c36;
  padding: var(--jp-private-code-span-padding) 0;
}
.jp-RenderedText pre .ansi-red-intense-bg {
  background-color: #b22b31;
  padding: var(--jp-private-code-span-padding) 0;
}
.jp-RenderedText pre .ansi-green-intense-bg {
  background-color: #007427;
  padding: var(--jp-private-code-span-padding) 0;
}
.jp-RenderedText pre .ansi-yellow-intense-bg {
  background-color: #b27d12;
  padding: var(--jp-private-code-span-padding) 0;
}
.jp-RenderedText pre .ansi-blue-intense-bg {
  background-color: #0065ca;
  padding: var(--jp-private-code-span-padding) 0;
}
.jp-RenderedText pre .ansi-magenta-intense-bg {
  background-color: #a03196;
  padding: var(--jp-private-code-span-padding) 0;
}
.jp-RenderedText pre .ansi-cyan-intense-bg {
  background-color: #258f8f;
  padding: var(--jp-private-code-span-padding) 0;
}
.jp-RenderedText pre .ansi-white-intense-bg {
  background-color: #a1a6b2;
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-default-inverse-fg {
  color: var(--jp-ui-inverse-font-color0);
}
.jp-RenderedText pre .ansi-default-inverse-bg {
  background-color: var(--jp-inverse-layout-color0);
  padding: var(--jp-private-code-span-padding) 0;
}

.jp-RenderedText pre .ansi-bold {
  font-weight: bold;
}
.jp-RenderedText pre .ansi-underline {
  text-decoration: underline;
}

.jp-RenderedText[data-mime-type='application/vnd.jupyter.stderr'] {
  background: var(--jp-rendermime-error-background);
  padding-top: var(--jp-code-padding);
}

/*-----------------------------------------------------------------------------
| RenderedLatex
|----------------------------------------------------------------------------*/

.jp-RenderedLatex {
  color: var(--jp-content-font-color1);
  font-size: var(--jp-content-font-size1);
  line-height: var(--jp-content-line-height);
}

/* Left-justify outputs.*/
.jp-OutputArea-output.jp-RenderedLatex {
  padding: var(--jp-code-padding);
  text-align: left;
}

/*-----------------------------------------------------------------------------
| RenderedHTML
|----------------------------------------------------------------------------*/

.jp-RenderedHTMLCommon {
  color: var(--jp-content-font-color1);
  font-family: var(--jp-content-font-family);
  font-size: var(--jp-content-font-size1);
  line-height: var(--jp-content-line-height);
  /* Give a bit more R padding on Markdown text to keep line lengths reasonable */
  padding-right: 20px;
}

.jp-RenderedHTMLCommon em {
  font-style: italic;
}

.jp-RenderedHTMLCommon strong {
  font-weight: bold;
}

.jp-RenderedHTMLCommon u {
  text-decoration: underline;
}

.jp-RenderedHTMLCommon a:link {
  text-decoration: none;
  color: var(--jp-content-link-color);
}

.jp-RenderedHTMLCommon a:hover {
  text-decoration: underline;
  color: var(--jp-content-link-color);
}

.jp-RenderedHTMLCommon a:visited {
  text-decoration: none;
  color: var(--jp-content-link-color);
}

/* Headings */

.jp-RenderedHTMLCommon h1,
.jp-RenderedHTMLCommon h2,
.jp-RenderedHTMLCommon h3,
.jp-RenderedHTMLCommon h4,
.jp-RenderedHTMLCommon h5,
.jp-RenderedHTMLCommon h6 {
  line-height: var(--jp-content-heading-line-height);
  font-weight: var(--jp-content-heading-font-weight);
  font-style: normal;
  margin: var(--jp-content-heading-margin-top) 0
    var(--jp-content-heading-margin-bottom) 0;
}

.jp-RenderedHTMLCommon h1:first-child,
.jp-RenderedHTMLCommon h2:first-child,
.jp-RenderedHTMLCommon h3:first-child,
.jp-RenderedHTMLCommon h4:first-child,
.jp-RenderedHTMLCommon h5:first-child,
.jp-RenderedHTMLCommon h6:first-child {
  margin-top: calc(0.5 * var(--jp-content-heading-margin-top));
}

.jp-RenderedHTMLCommon h1:last-child,
.jp-RenderedHTMLCommon h2:last-child,
.jp-RenderedHTMLCommon h3:last-child,
.jp-RenderedHTMLCommon h4:last-child,
.jp-RenderedHTMLCommon h5:last-child,
.jp-RenderedHTMLCommon h6:last-child {
  margin-bottom: calc(0.5 * var(--jp-content-heading-margin-bottom));
}

.jp-RenderedHTMLCommon h1 {
  font-size: var(--jp-content-font-size5);
}

.jp-RenderedHTMLCommon h2 {
  font-size: var(--jp-content-font-size4);
}

.jp-RenderedHTMLCommon h3 {
  font-size: var(--jp-content-font-size3);
}

.jp-RenderedHTMLCommon h4 {
  font-size: var(--jp-content-font-size2);
}

.jp-RenderedHTMLCommon h5 {
  font-size: var(--jp-content-font-size1);
}

.jp-RenderedHTMLCommon h6 {
  font-size: var(--jp-content-font-size0);
}

/* Lists */

.jp-RenderedHTMLCommon ul:not(.list-inline),
.jp-RenderedHTMLCommon ol:not(.list-inline) {
  padding-left: 2em;
}

.jp-RenderedHTMLCommon ul {
  list-style: disc;
}

.jp-RenderedHTMLCommon ul ul {
  list-style: square;
}

.jp-RenderedHTMLCommon ul ul ul {
  list-style: circle;
}

.jp-RenderedHTMLCommon ol {
  list-style: decimal;
}

.jp-RenderedHTMLCommon ol ol {
  list-style: upper-alpha;
}

.jp-RenderedHTMLCommon ol ol ol {
  list-style: lower-alpha;
}

.jp-RenderedHTMLCommon ol ol ol ol {
  list-style: lower-roman;
}

.jp-RenderedHTMLCommon ol ol ol ol ol {
  list-style: decimal;
}

.jp-RenderedHTMLCommon ol,
.jp-RenderedHTMLCommon ul {
  margin-bottom: 1em;
}

.jp-RenderedHTMLCommon ul ul,
.jp-RenderedHTMLCommon ul ol,
.jp-RenderedHTMLCommon ol ul,
.jp-RenderedHTMLCommon ol ol {
  margin-bottom: 0em;
}

.jp-RenderedHTMLCommon hr {
  color: var(--jp-border-color2);
  background-color: var(--jp-border-color1);
  margin-top: 1em;
  margin-bottom: 1em;
}

.jp-RenderedHTMLCommon > pre {
  margin: 1.5em 2em;
}

.jp-RenderedHTMLCommon pre,
.jp-RenderedHTMLCommon code {
  border: 0;
  background-color: var(--jp-layout-color0);
  color: var(--jp-content-font-color1);
  font-family: var(--jp-code-font-family);
  font-size: inherit;
  line-height: var(--jp-code-line-height);
  padding: 0;
  white-space: pre-wrap;
}

.jp-RenderedHTMLCommon :not(pre) > code {
  background-color: var(--jp-layout-color2);
  padding: 1px 5px;
}

/* Tables */

.jp-RenderedHTMLCommon table {
  border-collapse: collapse;
  border-spacing: 0;
  border: none;
  color: var(--jp-ui-font-color1);
  font-size: 12px;
  table-layout: fixed;
  margin-left: auto;
  margin-right: auto;
}

.jp-RenderedHTMLCommon thead {
  border-bottom: var(--jp-border-width) solid var(--jp-border-color1);
  vertical-align: bottom;
}

.jp-RenderedHTMLCommon td,
.jp-RenderedHTMLCommon th,
.jp-RenderedHTMLCommon tr {
  vertical-align: middle;
  padding: 0.5em 0.5em;
  line-height: normal;
  white-space: normal;
  max-width: none;
  border: none;
}

.jp-RenderedMarkdown.jp-RenderedHTMLCommon td,
.jp-RenderedMarkdown.jp-RenderedHTMLCommon th {
  max-width: none;
}

:not(.jp-RenderedMarkdown).jp-RenderedHTMLCommon td,
:not(.jp-RenderedMarkdown).jp-RenderedHTMLCommon th,
:not(.jp-RenderedMarkdown).jp-RenderedHTMLCommon tr {
  text-align: right;
}

.jp-RenderedHTMLCommon th {
  font-weight: bold;
}

.jp-RenderedHTMLCommon tbody tr:nth-child(odd) {
  background: var(--jp-layout-color0);
}

.jp-RenderedHTMLCommon tbody tr:nth-child(even) {
  background: var(--jp-rendermime-table-row-background);
}

.jp-RenderedHTMLCommon tbody tr:hover {
  background: var(--jp-rendermime-table-row-hover-background);
}

.jp-RenderedHTMLCommon table {
  margin-bottom: 1em;
}

.jp-RenderedHTMLCommon p {
  text-align: left;
  margin: 0px;
}

.jp-RenderedHTMLCommon p {
  margin-bottom: 1em;
}

.jp-RenderedHTMLCommon img {
  -moz-force-broken-image-icon: 1;
}

/* Restrict to direct children as other images could be nested in other content. */
.jp-RenderedHTMLCommon > img {
  display: block;
  margin-left: 0;
  margin-right: 0;
  margin-bottom: 1em;
}

/* Change color behind transparent images if they need it... */
[data-jp-theme-light='false'] .jp-RenderedImage img.jp-needs-light-background {
  background-color: var(--jp-inverse-layout-color1);
}
[data-jp-theme-light='true'] .jp-RenderedImage img.jp-needs-dark-background {
  background-color: var(--jp-inverse-layout-color1);
}
/* ...or leave it untouched if they don't */
[data-jp-theme-light='false'] .jp-RenderedImage img.jp-needs-dark-background {
}
[data-jp-theme-light='true'] .jp-RenderedImage img.jp-needs-light-background {
}

.jp-RenderedHTMLCommon img,
.jp-RenderedImage img,
.jp-RenderedHTMLCommon svg,
.jp-RenderedSVG svg {
  max-width: 100%;
  height: auto;
}

.jp-RenderedHTMLCommon img.jp-mod-unconfined,
.jp-RenderedImage img.jp-mod-unconfined,
.jp-RenderedHTMLCommon svg.jp-mod-unconfined,
.jp-RenderedSVG svg.jp-mod-unconfined {
  max-width: none;
}

.jp-RenderedHTMLCommon .alert {
  padding: var(--jp-notebook-padding);
  border: var(--jp-border-width) solid transparent;
  border-radius: var(--jp-border-radius);
  margin-bottom: 1em;
}

.jp-RenderedHTMLCommon .alert-info {
  color: var(--jp-info-color0);
  background-color: var(--jp-info-color3);
  border-color: var(--jp-info-color2);
}
.jp-RenderedHTMLCommon .alert-info hr {
  border-color: var(--jp-info-color3);
}
.jp-RenderedHTMLCommon .alert-info > p:last-child,
.jp-RenderedHTMLCommon .alert-info > ul:last-child {
  margin-bottom: 0;
}

.jp-RenderedHTMLCommon .alert-warning {
  color: var(--jp-warn-color0);
  background-color: var(--jp-warn-color3);
  border-color: var(--jp-warn-color2);
}
.jp-RenderedHTMLCommon .alert-warning hr {
  border-color: var(--jp-warn-color3);
}
.jp-RenderedHTMLCommon .alert-warning > p:last-child,
.jp-RenderedHTMLCommon .alert-warning > ul:last-child {
  margin-bottom: 0;
}

.jp-RenderedHTMLCommon .alert-success {
  color: var(--jp-success-color0);
  background-color: var(--jp-success-color3);
  border-color: var(--jp-success-color2);
}
.jp-RenderedHTMLCommon .alert-success hr {
  border-color: var(--jp-success-color3);
}
.jp-RenderedHTMLCommon .alert-success > p:last-child,
.jp-RenderedHTMLCommon .alert-success > ul:last-child {
  margin-bottom: 0;
}

.jp-RenderedHTMLCommon .alert-danger {
  color: var(--jp-error-color0);
  background-color: var(--jp-error-color3);
  border-color: var(--jp-error-color2);
}
.jp-RenderedHTMLCommon .alert-danger hr {
  border-color: var(--jp-error-color3);
}
.jp-RenderedHTMLCommon .alert-danger > p:last-child,
.jp-RenderedHTMLCommon .alert-danger > ul:last-child {
  margin-bottom: 0;
}

.jp-RenderedHTMLCommon blockquote {
  margin: 1em 2em;
  padding: 0 1em;
  border-left: 5px solid var(--jp-border-color2);
}

a.jp-InternalAnchorLink {
  visibility: hidden;
  margin-left: 8px;
  color: var(--md-blue-800);
}

h1:hover .jp-InternalAnchorLink,
h2:hover .jp-InternalAnchorLink,
h3:hover .jp-InternalAnchorLink,
h4:hover .jp-InternalAnchorLink,
h5:hover .jp-InternalAnchorLink,
h6:hover .jp-InternalAnchorLink {
  visibility: visible;
}

.jp-RenderedHTMLCommon kbd {
  background-color: var(--jp-rendermime-table-row-background);
  border: 1px solid var(--jp-border-color0);
  border-bottom-color: var(--jp-border-color2);
  border-radius: 3px;
  box-shadow: inset 0 -1px 0 rgba(0, 0, 0, 0.25);
  display: inline-block;
  font-size: 0.8em;
  line-height: 1em;
  padding: 0.2em 0.5em;
}

/* Most direct children of .jp-RenderedHTMLCommon have a margin-bottom of 1.0.
 * At the bottom of cells this is a bit too much as there is also spacing
 * between cells. Going all the way to 0 gets too tight between markdown and
 * code cells.
 */
.jp-RenderedHTMLCommon > *:last-child {
  margin-bottom: 0.5em;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-MimeDocument {
  outline: none;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Variables
|----------------------------------------------------------------------------*/

:root {
  --jp-private-filebrowser-button-height: 28px;
  --jp-private-filebrowser-button-width: 48px;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-FileBrowser {
  display: flex;
  flex-direction: column;
  color: var(--jp-ui-font-color1);
  background: var(--jp-layout-color1);
  /* This is needed so that all font sizing of children done in ems is
   * relative to this base size */
  font-size: var(--jp-ui-font-size1);
}

.jp-FileBrowser-toolbar.jp-Toolbar {
  border-bottom: none;
  height: auto;
  margin: var(--jp-toolbar-header-margin);
  box-shadow: none;
}

.jp-BreadCrumbs {
  flex: 0 0 auto;
  margin: 8px 12px 8px 12px;
}

.jp-BreadCrumbs-item {
  margin: 0px 2px;
  padding: 0px 2px;
  border-radius: var(--jp-border-radius);
  cursor: pointer;
}

.jp-BreadCrumbs-item:hover {
  background-color: var(--jp-layout-color2);
}

.jp-BreadCrumbs-item:first-child {
  margin-left: 0px;
}

.jp-BreadCrumbs-item.jp-mod-dropTarget {
  background-color: var(--jp-brand-color2);
  opacity: 0.7;
}

/*-----------------------------------------------------------------------------
| Buttons
|----------------------------------------------------------------------------*/

.jp-FileBrowser-toolbar.jp-Toolbar {
  padding: 0px;
  margin: 8px 12px 0px 12px;
}

.jp-FileBrowser-toolbar.jp-Toolbar {
  justify-content: flex-start;
}

.jp-FileBrowser-toolbar.jp-Toolbar .jp-Toolbar-item {
  flex: 0 0 auto;
  padding-left: 0px;
  padding-right: 2px;
}

.jp-FileBrowser-toolbar.jp-Toolbar .jp-ToolbarButtonComponent {
  width: 40px;
}

.jp-FileBrowser-toolbar.jp-Toolbar
  .jp-Toolbar-item:first-child
  .jp-ToolbarButtonComponent {
  width: 72px;
  background: var(--jp-brand-color1);
}

.jp-FileBrowser-toolbar.jp-Toolbar
  .jp-Toolbar-item:first-child
  .jp-ToolbarButtonComponent
  .jp-icon3 {
  fill: white;
}

/*-----------------------------------------------------------------------------
| Other styles
|----------------------------------------------------------------------------*/

.jp-FileDialog.jp-mod-conflict input {
  color: red;
}

.jp-FileDialog .jp-new-name-title {
  margin-top: 12px;
}

.jp-LastModified-hidden {
  display: none;
}

.jp-FileBrowser-filterBox {
  padding: 0px;
  flex: 0 0 auto;
  margin: 8px 12px 0px 12px;
}

/*-----------------------------------------------------------------------------
| DirListing
|----------------------------------------------------------------------------*/

.jp-DirListing {
  flex: 1 1 auto;
  display: flex;
  flex-direction: column;
  outline: 0;
}

.jp-DirListing-header {
  flex: 0 0 auto;
  display: flex;
  flex-direction: row;
  overflow: hidden;
  border-top: var(--jp-border-width) solid var(--jp-border-color2);
  border-bottom: var(--jp-border-width) solid var(--jp-border-color1);
  box-shadow: var(--jp-toolbar-box-shadow);
  z-index: 2;
}

.jp-DirListing-headerItem {
  padding: 4px 12px 2px 12px;
  font-weight: 500;
}

.jp-DirListing-headerItem:hover {
  background: var(--jp-layout-color2);
}

.jp-DirListing-headerItem.jp-id-name {
  flex: 1 0 84px;
}

.jp-DirListing-headerItem.jp-id-modified {
  flex: 0 0 112px;
  border-left: var(--jp-border-width) solid var(--jp-border-color2);
  text-align: right;
}

.jp-id-narrow {
  display: none;
  flex: 0 0 5px;
  padding: 4px 4px;
  border-left: var(--jp-border-width) solid var(--jp-border-color2);
  text-align: right;
  color: var(--jp-border-color2);
}

.jp-DirListing-narrow .jp-id-narrow {
  display: block;
}

.jp-DirListing-narrow .jp-id-modified,
.jp-DirListing-narrow .jp-DirListing-itemModified {
  display: none;
}

.jp-DirListing-headerItem.jp-mod-selected {
  font-weight: 600;
}

/* increase specificity to override bundled default */
.jp-DirListing-content {
  flex: 1 1 auto;
  margin: 0;
  padding: 0;
  list-style-type: none;
  overflow: auto;
  background-color: var(--jp-layout-color1);
}

.jp-DirListing-content mark {
  color: var(--jp-ui-font-color0);
  background-color: transparent;
  font-weight: bold;
}

/* Style the directory listing content when a user drops a file to upload */
.jp-DirListing.jp-mod-native-drop .jp-DirListing-content {
  outline: 5px dashed rgba(128, 128, 128, 0.5);
  outline-offset: -10px;
  cursor: copy;
}

.jp-DirListing-item {
  display: flex;
  flex-direction: row;
  padding: 4px 12px;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.jp-DirListing-item[data-is-dot] {
  opacity: 75%;
}

.jp-DirListing-item.jp-mod-selected {
  color: white;
  background: var(--jp-brand-color1);
}

.jp-DirListing-item.jp-mod-dropTarget {
  background: var(--jp-brand-color3);
}

.jp-DirListing-item:hover:not(.jp-mod-selected) {
  background: var(--jp-layout-color2);
}

.jp-DirListing-itemIcon {
  flex: 0 0 20px;
  margin-right: 4px;
}

.jp-DirListing-itemText {
  flex: 1 0 64px;
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
  user-select: none;
}

.jp-DirListing-itemModified {
  flex: 0 0 125px;
  text-align: right;
}

.jp-DirListing-editor {
  flex: 1 0 64px;
  outline: none;
  border: none;
}

.jp-DirListing-item.jp-mod-running .jp-DirListing-itemIcon:before {
  color: limegreen;
  content: '\25CF';
  font-size: 8px;
  position: absolute;
  left: -8px;
}

.jp-DirListing-item.lm-mod-drag-image,
.jp-DirListing-item.jp-mod-selected.lm-mod-drag-image {
  font-size: var(--jp-ui-font-size1);
  padding-left: 4px;
  margin-left: 4px;
  width: 160px;
  background-color: var(--jp-ui-inverse-font-color2);
  box-shadow: var(--jp-elevation-z2);
  border-radius: 0px;
  color: var(--jp-ui-font-color1);
  transform: translateX(-40%) translateY(-58%);
}

.jp-DirListing-deadSpace {
  flex: 1 1 auto;
  margin: 0;
  padding: 0;
  list-style-type: none;
  overflow: auto;
  background-color: var(--jp-layout-color1);
}

.jp-Document {
  min-width: 120px;
  min-height: 120px;
  outline: none;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Private CSS variables
|----------------------------------------------------------------------------*/

:root {
}

/*-----------------------------------------------------------------------------
| Main OutputArea
| OutputArea has a list of Outputs
|----------------------------------------------------------------------------*/

.jp-OutputArea {
  overflow-y: auto;
}

.jp-OutputArea-child {
  display: flex;
  flex-direction: row;
}

.jp-OutputPrompt {
  flex: 0 0 var(--jp-cell-prompt-width);
  color: var(--jp-cell-outprompt-font-color);
  font-family: var(--jp-cell-prompt-font-family);
  padding: var(--jp-code-padding);
  letter-spacing: var(--jp-cell-prompt-letter-spacing);
  line-height: var(--jp-code-line-height);
  font-size: var(--jp-code-font-size);
  border: var(--jp-border-width) solid transparent;
  opacity: var(--jp-cell-prompt-opacity);
  /* Right align prompt text, don't wrap to handle large prompt numbers */
  text-align: right;
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
  /* Disable text selection */
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

.jp-OutputArea-output {
  height: auto;
  overflow: auto;
  user-select: text;
  -moz-user-select: text;
  -webkit-user-select: text;
  -ms-user-select: text;
}

.jp-OutputArea-child .jp-OutputArea-output {
  flex-grow: 1;
  flex-shrink: 1;
}

/**
 * Isolated output.
 */
.jp-OutputArea-output.jp-mod-isolated {
  width: 100%;
  display: block;
}

/*
When drag events occur, `p-mod-override-cursor` is added to the body.
Because iframes steal all cursor events, the following two rules are necessary
to suppress pointer events while resize drags are occurring. There may be a
better solution to this problem.
*/
body.lm-mod-override-cursor .jp-OutputArea-output.jp-mod-isolated {
  position: relative;
}

body.lm-mod-override-cursor .jp-OutputArea-output.jp-mod-isolated:before {
  content: '';
  position: absolute;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
  background: transparent;
}

/* pre */

.jp-OutputArea-output pre {
  border: none;
  margin: 0px;
  padding: 0px;
  overflow-x: auto;
  overflow-y: auto;
  word-break: break-all;
  word-wrap: break-word;
  white-space: pre-wrap;
}

/* tables */

.jp-OutputArea-output.jp-RenderedHTMLCommon table {
  margin-left: 0;
  margin-right: 0;
}

/* description lists */

.jp-OutputArea-output dl,
.jp-OutputArea-output dt,
.jp-OutputArea-output dd {
  display: block;
}

.jp-OutputArea-output dl {
  width: 100%;
  overflow: hidden;
  padding: 0;
  margin: 0;
}

.jp-OutputArea-output dt {
  font-weight: bold;
  float: left;
  width: 20%;
  padding: 0;
  margin: 0;
}

.jp-OutputArea-output dd {
  float: left;
  width: 80%;
  padding: 0;
  margin: 0;
}

/* Hide the gutter in case of
 *  - nested output areas (e.g. in the case of output widgets)
 *  - mirrored output areas
 */
.jp-OutputArea .jp-OutputArea .jp-OutputArea-prompt {
  display: none;
}

/*-----------------------------------------------------------------------------
| executeResult is added to any Output-result for the display of the object
| returned by a cell
|----------------------------------------------------------------------------*/

.jp-OutputArea-output.jp-OutputArea-executeResult {
  margin-left: 0px;
  flex: 1 1 auto;
}

/* Text output with the Out[] prompt needs a top padding to match the
 * alignment of the Out[] prompt itself.
 */
.jp-OutputArea-executeResult .jp-RenderedText.jp-OutputArea-output {
  padding-top: var(--jp-code-padding);
  border-top: var(--jp-border-width) solid transparent;
}

/*-----------------------------------------------------------------------------
| The Stdin output
|----------------------------------------------------------------------------*/

.jp-OutputArea-stdin {
  line-height: var(--jp-code-line-height);
  padding-top: var(--jp-code-padding);
  display: flex;
}

.jp-Stdin-prompt {
  color: var(--jp-content-font-color0);
  padding-right: var(--jp-code-padding);
  vertical-align: baseline;
  flex: 0 0 auto;
}

.jp-Stdin-input {
  font-family: var(--jp-code-font-family);
  font-size: inherit;
  color: inherit;
  background-color: inherit;
  width: 42%;
  min-width: 200px;
  /* make sure input baseline aligns with prompt */
  vertical-align: baseline;
  /* padding + margin = 0.5em between prompt and cursor */
  padding: 0em 0.25em;
  margin: 0em 0.25em;
  flex: 0 0 70%;
}

.jp-Stdin-input:focus {
  box-shadow: none;
}

/*-----------------------------------------------------------------------------
| Output Area View
|----------------------------------------------------------------------------*/

.jp-LinkedOutputView .jp-OutputArea {
  height: 100%;
  display: block;
}

.jp-LinkedOutputView .jp-OutputArea-output:only-child {
  height: 100%;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

.jp-Collapser {
  flex: 0 0 var(--jp-cell-collapser-width);
  padding: 0px;
  margin: 0px;
  border: none;
  outline: none;
  background: transparent;
  border-radius: var(--jp-border-radius);
  opacity: 1;
}

.jp-Collapser-child {
  display: block;
  width: 100%;
  box-sizing: border-box;
  /* height: 100% doesn't work because the height of its parent is computed from content */
  position: absolute;
  top: 0px;
  bottom: 0px;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Header/Footer
|----------------------------------------------------------------------------*/

/* Hidden by zero height by default */
.jp-CellHeader,
.jp-CellFooter {
  height: 0px;
  width: 100%;
  padding: 0px;
  margin: 0px;
  border: none;
  outline: none;
  background: transparent;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Input
|----------------------------------------------------------------------------*/

/* All input areas */
.jp-InputArea {
  display: flex;
  flex-direction: row;
  overflow: hidden;
}

.jp-InputArea-editor {
  flex: 1 1 auto;
  overflow: hidden;
}

.jp-InputArea-editor {
  /* This is the non-active, default styling */
  border: var(--jp-border-width) solid var(--jp-cell-editor-border-color);
  border-radius: 0px;
  background: var(--jp-cell-editor-background);
}

.jp-InputPrompt {
  flex: 0 0 var(--jp-cell-prompt-width);
  color: var(--jp-cell-inprompt-font-color);
  font-family: var(--jp-cell-prompt-font-family);
  padding: var(--jp-code-padding);
  letter-spacing: var(--jp-cell-prompt-letter-spacing);
  opacity: var(--jp-cell-prompt-opacity);
  line-height: var(--jp-code-line-height);
  font-size: var(--jp-code-font-size);
  border: var(--jp-border-width) solid transparent;
  opacity: var(--jp-cell-prompt-opacity);
  /* Right align prompt text, don't wrap to handle large prompt numbers */
  text-align: right;
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
  /* Disable text selection */
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Placeholder
|----------------------------------------------------------------------------*/

.jp-Placeholder {
  display: flex;
  flex-direction: row;
  flex: 1 1 auto;
}

.jp-Placeholder-prompt {
  box-sizing: border-box;
}

.jp-Placeholder-content {
  flex: 1 1 auto;
  border: none;
  background: transparent;
  height: 20px;
  box-sizing: border-box;
}

.jp-Placeholder-content .jp-MoreHorizIcon {
  width: 32px;
  height: 16px;
  border: 1px solid transparent;
  border-radius: var(--jp-border-radius);
}

.jp-Placeholder-content .jp-MoreHorizIcon:hover {
  border: 1px solid var(--jp-border-color1);
  box-shadow: 0px 0px 2px 0px rgba(0, 0, 0, 0.25);
  background-color: var(--jp-layout-color0);
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Private CSS variables
|----------------------------------------------------------------------------*/

:root {
  --jp-private-cell-scrolling-output-offset: 5px;
}

/*-----------------------------------------------------------------------------
| Cell
|----------------------------------------------------------------------------*/

.jp-Cell {
  padding: var(--jp-cell-padding);
  margin: 0px;
  border: none;
  outline: none;
  background: transparent;
}

/*-----------------------------------------------------------------------------
| Common input/output
|----------------------------------------------------------------------------*/

.jp-Cell-inputWrapper,
.jp-Cell-outputWrapper {
  display: flex;
  flex-direction: row;
  padding: 0px;
  margin: 0px;
  /* Added to reveal the box-shadow on the input and output collapsers. */
  overflow: visible;
}

/* Only input/output areas inside cells */
.jp-Cell-inputArea,
.jp-Cell-outputArea {
  flex: 1 1 auto;
}

/*-----------------------------------------------------------------------------
| Collapser
|----------------------------------------------------------------------------*/

/* Make the output collapser disappear when there is not output, but do so
 * in a manner that leaves it in the layout and preserves its width.
 */
.jp-Cell.jp-mod-noOutputs .jp-Cell-outputCollapser {
  border: none !important;
  background: transparent !important;
}

.jp-Cell:not(.jp-mod-noOutputs) .jp-Cell-outputCollapser {
  min-height: var(--jp-cell-collapser-min-height);
}

/*-----------------------------------------------------------------------------
| Output
|----------------------------------------------------------------------------*/

/* Put a space between input and output when there IS output */
.jp-Cell:not(.jp-mod-noOutputs) .jp-Cell-outputWrapper {
  margin-top: 5px;
}

.jp-CodeCell.jp-mod-outputsScrolled .jp-Cell-outputArea {
  overflow-y: auto;
  max-height: 200px;
  box-shadow: inset 0 0 6px 2px rgba(0, 0, 0, 0.3);
  margin-left: var(--jp-private-cell-scrolling-output-offset);
}

.jp-CodeCell.jp-mod-outputsScrolled .jp-OutputArea-prompt {
  flex: 0 0
    calc(
      var(--jp-cell-prompt-width) -
        var(--jp-private-cell-scrolling-output-offset)
    );
}

/*-----------------------------------------------------------------------------
| CodeCell
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| MarkdownCell
|----------------------------------------------------------------------------*/

.jp-MarkdownOutput {
  flex: 1 1 auto;
  margin-top: 0;
  margin-bottom: 0;
  padding-left: var(--jp-code-padding);
}

.jp-MarkdownOutput.jp-RenderedHTMLCommon {
  overflow: auto;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Variables
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------

/*-----------------------------------------------------------------------------
| Styles
|----------------------------------------------------------------------------*/

.jp-NotebookPanel-toolbar {
  padding: 2px;
}

.jp-Toolbar-item.jp-Notebook-toolbarCellType .jp-select-wrapper.jp-mod-focused {
  border: none;
  box-shadow: none;
}

.jp-Notebook-toolbarCellTypeDropdown select {
  height: 24px;
  font-size: var(--jp-ui-font-size1);
  line-height: 14px;
  border-radius: 0;
  display: block;
}

.jp-Notebook-toolbarCellTypeDropdown span {
  top: 5px !important;
}

/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Private CSS variables
|----------------------------------------------------------------------------*/

:root {
  --jp-private-notebook-dragImage-width: 304px;
  --jp-private-notebook-dragImage-height: 36px;
  --jp-private-notebook-selected-color: var(--md-blue-400);
  --jp-private-notebook-active-color: var(--md-green-400);
}

/*-----------------------------------------------------------------------------
| Imports
|----------------------------------------------------------------------------*/

/*-----------------------------------------------------------------------------
| Notebook
|----------------------------------------------------------------------------*/

.jp-NotebookPanel {
  display: block;
  height: 100%;
}

.jp-NotebookPanel.jp-Document {
  min-width: 240px;
  min-height: 120px;
}

.jp-Notebook {
  padding: var(--jp-notebook-padding);
  outline: none;
  overflow: auto;
  background: var(--jp-layout-color0);
}

.jp-Notebook.jp-mod-scrollPastEnd::after {
  display: block;
  content: '';
  min-height: var(--jp-notebook-scroll-padding);
}

.jp-Notebook .jp-Cell {
  overflow: visible;
}

.jp-Notebook .jp-Cell .jp-InputPrompt {
  cursor: move;
}

/*-----------------------------------------------------------------------------
| Notebook state related styling
|
| The notebook and cells each have states, here are the possibilities:
|
| - Notebook
|   - Command
|   - Edit
| - Cell
|   - None
|   - Active (only one can be active)
|   - Selected (the cells actions are applied to)
|   - Multiselected (when multiple selected, the cursor)
|   - No outputs
|----------------------------------------------------------------------------*/

/* Command or edit modes */

.jp-Notebook .jp-Cell:not(.jp-mod-active) .jp-InputPrompt {
  opacity: var(--jp-cell-prompt-not-active-opacity);
  color: var(--jp-cell-prompt-not-active-font-color);
}

.jp-Notebook .jp-Cell:not(.jp-mod-active) .jp-OutputPrompt {
  opacity: var(--jp-cell-prompt-not-active-opacity);
  color: var(--jp-cell-prompt-not-active-font-color);
}

/* cell is active */
.jp-Notebook .jp-Cell.jp-mod-active .jp-Collapser {
  background: var(--jp-brand-color1);
}

/* collapser is hovered */
.jp-Notebook .jp-Cell .jp-Collapser:hover {
  box-shadow: var(--jp-elevation-z2);
  background: var(--jp-brand-color1);
  opacity: var(--jp-cell-collapser-not-active-hover-opacity);
}

/* cell is active and collapser is hovered */
.jp-Notebook .jp-Cell.jp-mod-active .jp-Collapser:hover {
  background: var(--jp-brand-color0);
  opacity: 1;
}

/* Command mode */

.jp-Notebook.jp-mod-commandMode .jp-Cell.jp-mod-selected {
  background: var(--jp-notebook-multiselected-color);
}

.jp-Notebook.jp-mod-commandMode
  .jp-Cell.jp-mod-active.jp-mod-selected:not(.jp-mod-multiSelected) {
  background: transparent;
}

/* Edit mode */

.jp-Notebook.jp-mod-editMode .jp-Cell.jp-mod-active .jp-InputArea-editor {
  border: var(--jp-border-width) solid var(--jp-cell-editor-active-border-color);
  box-shadow: var(--jp-input-box-shadow);
  background-color: var(--jp-cell-editor-active-background);
}

/*-----------------------------------------------------------------------------
| Notebook drag and drop
|----------------------------------------------------------------------------*/

.jp-Notebook-cell.jp-mod-dropSource {
  opacity: 0.5;
}

.jp-Notebook-cell.jp-mod-dropTarget,
.jp-Notebook.jp-mod-commandMode
  .jp-Notebook-cell.jp-mod-active.jp-mod-selected.jp-mod-dropTarget {
  border-top-color: var(--jp-private-notebook-selected-color);
  border-top-style: solid;
  border-top-width: 2px;
}

.jp-dragImage {
  display: flex;
  flex-direction: row;
  width: var(--jp-private-notebook-dragImage-width);
  height: var(--jp-private-notebook-dragImage-height);
  border: var(--jp-border-width) solid var(--jp-cell-editor-border-color);
  background: var(--jp-cell-editor-background);
  overflow: visible;
}

.jp-dragImage-singlePrompt {
  box-shadow: 2px 2px 4px 0px rgba(0, 0, 0, 0.12);
}

.jp-dragImage .jp-dragImage-content {
  flex: 1 1 auto;
  z-index: 2;
  font-size: var(--jp-code-font-size);
  font-family: var(--jp-code-font-family);
  line-height: var(--jp-code-line-height);
  padding: var(--jp-code-padding);
  border: var(--jp-border-width) solid var(--jp-cell-editor-border-color);
  background: var(--jp-cell-editor-background-color);
  color: var(--jp-content-font-color3);
  text-align: left;
  margin: 4px 4px 4px 0px;
}

.jp-dragImage .jp-dragImage-prompt {
  flex: 0 0 auto;
  min-width: 36px;
  color: var(--jp-cell-inprompt-font-color);
  padding: var(--jp-code-padding);
  padding-left: 12px;
  font-family: var(--jp-cell-prompt-font-family);
  letter-spacing: var(--jp-cell-prompt-letter-spacing);
  line-height: 1.9;
  font-size: var(--jp-code-font-size);
  border: var(--jp-border-width) solid transparent;
}

.jp-dragImage-multipleBack {
  z-index: -1;
  position: absolute;
  height: 32px;
  width: 300px;
  top: 8px;
  left: 8px;
  background: var(--jp-layout-color2);
  border: var(--jp-border-width) solid var(--jp-input-border-color);
  box-shadow: 2px 2px 4px 0px rgba(0, 0, 0, 0.12);
}

/*-----------------------------------------------------------------------------
| Cell toolbar
|----------------------------------------------------------------------------*/

.jp-NotebookTools {
  display: block;
  min-width: var(--jp-sidebar-min-width);
  color: var(--jp-ui-font-color1);
  background: var(--jp-layout-color1);
  /* This is needed so that all font sizing of children done in ems is
    * relative to this base size */
  font-size: var(--jp-ui-font-size1);
  overflow: auto;
}

.jp-NotebookTools-tool {
  padding: 0px 12px 0 12px;
}

.jp-ActiveCellTool {
  padding: 12px;
  background-color: var(--jp-layout-color1);
  border-top: none !important;
}

.jp-ActiveCellTool .jp-InputArea-prompt {
  flex: 0 0 auto;
  padding-left: 0px;
}

.jp-ActiveCellTool .jp-InputArea-editor {
  flex: 1 1 auto;
  background: var(--jp-cell-editor-background);
  border-color: var(--jp-cell-editor-border-color);
}

.jp-ActiveCellTool .jp-InputArea-editor .CodeMirror {
  background: transparent;
}

.jp-MetadataEditorTool {
  flex-direction: column;
  padding: 12px 0px 12px 0px;
}

.jp-RankedPanel > :not(:first-child) {
  margin-top: 12px;
}

.jp-KeySelector select.jp-mod-styled {
  font-size: var(--jp-ui-font-size1);
  color: var(--jp-ui-font-color0);
  border: var(--jp-border-width) solid var(--jp-border-color1);
}

.jp-KeySelector label,
.jp-MetadataEditorTool label {
  line-height: 1.4;
}

.jp-NotebookTools .jp-select-wrapper {
  margin-top: 4px;
  margin-bottom: 0px;
}

.jp-NotebookTools .jp-Collapse {
  margin-top: 16px;
}

/*-----------------------------------------------------------------------------
| Presentation Mode (.jp-mod-presentationMode)
|----------------------------------------------------------------------------*/

.jp-mod-presentationMode .jp-Notebook {
  --jp-content-font-size1: var(--jp-content-presentation-font-size1);
  --jp-code-font-size: var(--jp-code-presentation-font-size);
}

.jp-mod-presentationMode .jp-Notebook .jp-Cell .jp-InputPrompt,
.jp-mod-presentationMode .jp-Notebook .jp-Cell .jp-OutputPrompt {
  flex: 0 0 110px;
}

</style>

    <style type="text/css">
/*-----------------------------------------------------------------------------
| Copyright (c) Jupyter Development Team.
| Distributed under the terms of the Modified BSD License.
|----------------------------------------------------------------------------*/

/*
The following CSS variables define the main, public API for styling JupyterLab.
These variables should be used by all plugins wherever possible. In other
words, plugins should not define custom colors, sizes, etc unless absolutely
necessary. This enables users to change the visual theme of JupyterLab
by changing these variables.

Many variables appear in an ordered sequence (0,1,2,3). These sequences
are designed to work well together, so for example, `--jp-border-color1` should
be used with `--jp-layout-color1`. The numbers have the following meanings:

* 0: super-primary, reserved for special emphasis
* 1: primary, most important under normal situations
* 2: secondary, next most important under normal situations
* 3: tertiary, next most important under normal situations

Throughout JupyterLab, we are mostly following principles from Google's
Material Design when selecting colors. We are not, however, following
all of MD as it is not optimized for dense, information rich UIs.
*/

:root {
  /* Elevation
   *
   * We style box-shadows using Material Design's idea of elevation. These particular numbers are taken from here:
   *
   * https://github.com/material-components/material-components-web
   * https://material-components-web.appspot.com/elevation.html
   */

  --jp-shadow-base-lightness: 0;
  --jp-shadow-umbra-color: rgba(
    var(--jp-shadow-base-lightness),
    var(--jp-shadow-base-lightness),
    var(--jp-shadow-base-lightness),
    0.2
  );
  --jp-shadow-penumbra-color: rgba(
    var(--jp-shadow-base-lightness),
    var(--jp-shadow-base-lightness),
    var(--jp-shadow-base-lightness),
    0.14
  );
  --jp-shadow-ambient-color: rgba(
    var(--jp-shadow-base-lightness),
    var(--jp-shadow-base-lightness),
    var(--jp-shadow-base-lightness),
    0.12
  );
  --jp-elevation-z0: none;
  --jp-elevation-z1: 0px 2px 1px -1px var(--jp-shadow-umbra-color),
    0px 1px 1px 0px var(--jp-shadow-penumbra-color),
    0px 1px 3px 0px var(--jp-shadow-ambient-color);
  --jp-elevation-z2: 0px 3px 1px -2px var(--jp-shadow-umbra-color),
    0px 2px 2px 0px var(--jp-shadow-penumbra-color),
    0px 1px 5px 0px var(--jp-shadow-ambient-color);
  --jp-elevation-z4: 0px 2px 4px -1px var(--jp-shadow-umbra-color),
    0px 4px 5px 0px var(--jp-shadow-penumbra-color),
    0px 1px 10px 0px var(--jp-shadow-ambient-color);
  --jp-elevation-z6: 0px 3px 5px -1px var(--jp-shadow-umbra-color),
    0px 6px 10px 0px var(--jp-shadow-penumbra-color),
    0px 1px 18px 0px var(--jp-shadow-ambient-color);
  --jp-elevation-z8: 0px 5px 5px -3px var(--jp-shadow-umbra-color),
    0px 8px 10px 1px var(--jp-shadow-penumbra-color),
    0px 3px 14px 2px var(--jp-shadow-ambient-color);
  --jp-elevation-z12: 0px 7px 8px -4px var(--jp-shadow-umbra-color),
    0px 12px 17px 2px var(--jp-shadow-penumbra-color),
    0px 5px 22px 4px var(--jp-shadow-ambient-color);
  --jp-elevation-z16: 0px 8px 10px -5px var(--jp-shadow-umbra-color),
    0px 16px 24px 2px var(--jp-shadow-penumbra-color),
    0px 6px 30px 5px var(--jp-shadow-ambient-color);
  --jp-elevation-z20: 0px 10px 13px -6px var(--jp-shadow-umbra-color),
    0px 20px 31px 3px var(--jp-shadow-penumbra-color),
    0px 8px 38px 7px var(--jp-shadow-ambient-color);
  --jp-elevation-z24: 0px 11px 15px -7px var(--jp-shadow-umbra-color),
    0px 24px 38px 3px var(--jp-shadow-penumbra-color),
    0px 9px 46px 8px var(--jp-shadow-ambient-color);

  /* Borders
   *
   * The following variables, specify the visual styling of borders in JupyterLab.
   */

  --jp-border-width: 1px;
  --jp-border-color0: var(--md-grey-400);
  --jp-border-color1: var(--md-grey-400);
  --jp-border-color2: var(--md-grey-300);
  --jp-border-color3: var(--md-grey-200);
  --jp-border-radius: 2px;

  /* UI Fonts
   *
   * The UI font CSS variables are used for the typography all of the JupyterLab
   * user interface elements that are not directly user generated content.
   *
   * The font sizing here is done assuming that the body font size of --jp-ui-font-size1
   * is applied to a parent element. When children elements, such as headings, are sized
   * in em all things will be computed relative to that body size.
   */

  --jp-ui-font-scale-factor: 1.2;
  --jp-ui-font-size0: 0.83333em;
  --jp-ui-font-size1: 13px; /* Base font size */
  --jp-ui-font-size2: 1.2em;
  --jp-ui-font-size3: 1.44em;

  --jp-ui-font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica,
    Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';

  /*
   * Use these font colors against the corresponding main layout colors.
   * In a light theme, these go from dark to light.
   */

  /* Defaults use Material Design specification */
  --jp-ui-font-color0: rgba(0, 0, 0, 1);
  --jp-ui-font-color1: rgba(0, 0, 0, 0.87);
  --jp-ui-font-color2: rgba(0, 0, 0, 0.54);
  --jp-ui-font-color3: rgba(0, 0, 0, 0.38);

  /*
   * Use these against the brand/accent/warn/error colors.
   * These will typically go from light to darker, in both a dark and light theme.
   */

  --jp-ui-inverse-font-color0: rgba(255, 255, 255, 1);
  --jp-ui-inverse-font-color1: rgba(255, 255, 255, 1);
  --jp-ui-inverse-font-color2: rgba(255, 255, 255, 0.7);
  --jp-ui-inverse-font-color3: rgba(255, 255, 255, 0.5);

  /* Content Fonts
   *
   * Content font variables are used for typography of user generated content.
   *
   * The font sizing here is done assuming that the body font size of --jp-content-font-size1
   * is applied to a parent element. When children elements, such as headings, are sized
   * in em all things will be computed relative to that body size.
   */

  --jp-content-line-height: 1.6;
  --jp-content-font-scale-factor: 1.2;
  --jp-content-font-size0: 0.83333em;
  --jp-content-font-size1: 14px; /* Base font size */
  --jp-content-font-size2: 1.2em;
  --jp-content-font-size3: 1.44em;
  --jp-content-font-size4: 1.728em;
  --jp-content-font-size5: 2.0736em;

  /* This gives a magnification of about 125% in presentation mode over normal. */
  --jp-content-presentation-font-size1: 17px;

  --jp-content-heading-line-height: 1;
  --jp-content-heading-margin-top: 1.2em;
  --jp-content-heading-margin-bottom: 0.8em;
  --jp-content-heading-font-weight: 500;

  /* Defaults use Material Design specification */
  --jp-content-font-color0: rgba(0, 0, 0, 1);
  --jp-content-font-color1: rgba(0, 0, 0, 0.87);
  --jp-content-font-color2: rgba(0, 0, 0, 0.54);
  --jp-content-font-color3: rgba(0, 0, 0, 0.38);

  --jp-content-link-color: var(--md-blue-700);

  --jp-content-font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI',
    Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji',
    'Segoe UI Symbol';

  /*
   * Code Fonts
   *
   * Code font variables are used for typography of code and other monospaces content.
   */

  --jp-code-font-size: 13px;
  --jp-code-line-height: 1.3077; /* 17px for 13px base */
  --jp-code-padding: 5px; /* 5px for 13px base, codemirror highlighting needs integer px value */
  --jp-code-font-family-default: Menlo, Consolas, 'DejaVu Sans Mono', monospace;
  --jp-code-font-family: var(--jp-code-font-family-default);

  /* This gives a magnification of about 125% in presentation mode over normal. */
  --jp-code-presentation-font-size: 16px;

  /* may need to tweak cursor width if you change font size */
  --jp-code-cursor-width0: 1.4px;
  --jp-code-cursor-width1: 2px;
  --jp-code-cursor-width2: 4px;

  /* Layout
   *
   * The following are the main layout colors use in JupyterLab. In a light
   * theme these would go from light to dark.
   */

  --jp-layout-color0: white;
  --jp-layout-color1: white;
  --jp-layout-color2: var(--md-grey-200);
  --jp-layout-color3: var(--md-grey-400);
  --jp-layout-color4: var(--md-grey-600);

  /* Inverse Layout
   *
   * The following are the inverse layout colors use in JupyterLab. In a light
   * theme these would go from dark to light.
   */

  --jp-inverse-layout-color0: #111111;
  --jp-inverse-layout-color1: var(--md-grey-900);
  --jp-inverse-layout-color2: var(--md-grey-800);
  --jp-inverse-layout-color3: var(--md-grey-700);
  --jp-inverse-layout-color4: var(--md-grey-600);

  /* Brand/accent */

  --jp-brand-color0: var(--md-blue-700);
  --jp-brand-color1: var(--md-blue-500);
  --jp-brand-color2: var(--md-blue-300);
  --jp-brand-color3: var(--md-blue-100);
  --jp-brand-color4: var(--md-blue-50);

  --jp-accent-color0: var(--md-green-700);
  --jp-accent-color1: var(--md-green-500);
  --jp-accent-color2: var(--md-green-300);
  --jp-accent-color3: var(--md-green-100);

  /* State colors (warn, error, success, info) */

  --jp-warn-color0: var(--md-orange-700);
  --jp-warn-color1: var(--md-orange-500);
  --jp-warn-color2: var(--md-orange-300);
  --jp-warn-color3: var(--md-orange-100);

  --jp-error-color0: var(--md-red-700);
  --jp-error-color1: var(--md-red-500);
  --jp-error-color2: var(--md-red-300);
  --jp-error-color3: var(--md-red-100);

  --jp-success-color0: var(--md-green-700);
  --jp-success-color1: var(--md-green-500);
  --jp-success-color2: var(--md-green-300);
  --jp-success-color3: var(--md-green-100);

  --jp-info-color0: var(--md-cyan-700);
  --jp-info-color1: var(--md-cyan-500);
  --jp-info-color2: var(--md-cyan-300);
  --jp-info-color3: var(--md-cyan-100);

  /* Cell specific styles */

  --jp-cell-padding: 5px;

  --jp-cell-collapser-width: 8px;
  --jp-cell-collapser-min-height: 20px;
  --jp-cell-collapser-not-active-hover-opacity: 0.6;

  --jp-cell-editor-background: var(--md-grey-100);
  --jp-cell-editor-border-color: var(--md-grey-300);
  --jp-cell-editor-box-shadow: inset 0 0 2px var(--md-blue-300);
  --jp-cell-editor-active-background: var(--jp-layout-color0);
  --jp-cell-editor-active-border-color: var(--jp-brand-color1);

  --jp-cell-prompt-width: 64px;
  --jp-cell-prompt-font-family: var(--jp-code-font-family-default);
  --jp-cell-prompt-letter-spacing: 0px;
  --jp-cell-prompt-opacity: 1;
  --jp-cell-prompt-not-active-opacity: 0.5;
  --jp-cell-prompt-not-active-font-color: var(--md-grey-700);
  /* A custom blend of MD grey and blue 600
   * See https://meyerweb.com/eric/tools/color-blend/#546E7A:1E88E5:5:hex */
  --jp-cell-inprompt-font-color: #307fc1;
  /* A custom blend of MD grey and orange 600
   * https://meyerweb.com/eric/tools/color-blend/#546E7A:F4511E:5:hex */
  --jp-cell-outprompt-font-color: #bf5b3d;

  /* Notebook specific styles */

  --jp-notebook-padding: 10px;
  --jp-notebook-select-background: var(--jp-layout-color1);
  --jp-notebook-multiselected-color: var(--md-blue-50);

  /* The scroll padding is calculated to fill enough space at the bottom of the
  notebook to show one single-line cell (with appropriate padding) at the top
  when the notebook is scrolled all the way to the bottom. We also subtract one
  pixel so that no scrollbar appears if we have just one single-line cell in the
  notebook. This padding is to enable a 'scroll past end' feature in a notebook.
  */
  --jp-notebook-scroll-padding: calc(
    100% - var(--jp-code-font-size) * var(--jp-code-line-height) -
      var(--jp-code-padding) - var(--jp-cell-padding) - 1px
  );

  /* Rendermime styles */

  --jp-rendermime-error-background: #fdd;
  --jp-rendermime-table-row-background: var(--md-grey-100);
  --jp-rendermime-table-row-hover-background: var(--md-light-blue-50);

  /* Dialog specific styles */

  --jp-dialog-background: rgba(0, 0, 0, 0.25);

  /* Console specific styles */

  --jp-console-padding: 10px;

  /* Toolbar specific styles */

  --jp-toolbar-border-color: var(--jp-border-color1);
  --jp-toolbar-micro-height: 8px;
  --jp-toolbar-background: var(--jp-layout-color1);
  --jp-toolbar-box-shadow: 0px 0px 2px 0px rgba(0, 0, 0, 0.24);
  --jp-toolbar-header-margin: 4px 4px 0px 4px;
  --jp-toolbar-active-background: var(--md-grey-300);

  /* Input field styles */

  --jp-input-box-shadow: inset 0 0 2px var(--md-blue-300);
  --jp-input-active-background: var(--jp-layout-color1);
  --jp-input-hover-background: var(--jp-layout-color1);
  --jp-input-background: var(--md-grey-100);
  --jp-input-border-color: var(--jp-border-color1);
  --jp-input-active-border-color: var(--jp-brand-color1);
  --jp-input-active-box-shadow-color: rgba(19, 124, 189, 0.3);

  /* General editor styles */

  --jp-editor-selected-background: #d9d9d9;
  --jp-editor-selected-focused-background: #d7d4f0;
  --jp-editor-cursor-color: var(--jp-ui-font-color0);

  /* Code mirror specific styles */

  --jp-mirror-editor-keyword-color: #008000;
  --jp-mirror-editor-atom-color: #88f;
  --jp-mirror-editor-number-color: #080;
  --jp-mirror-editor-def-color: #00f;
  --jp-mirror-editor-variable-color: var(--md-grey-900);
  --jp-mirror-editor-variable-2-color: #05a;
  --jp-mirror-editor-variable-3-color: #085;
  --jp-mirror-editor-punctuation-color: #05a;
  --jp-mirror-editor-property-color: #05a;
  --jp-mirror-editor-operator-color: #aa22ff;
  --jp-mirror-editor-comment-color: #408080;
  --jp-mirror-editor-string-color: #ba2121;
  --jp-mirror-editor-string-2-color: #708;
  --jp-mirror-editor-meta-color: #aa22ff;
  --jp-mirror-editor-qualifier-color: #555;
  --jp-mirror-editor-builtin-color: #008000;
  --jp-mirror-editor-bracket-color: #997;
  --jp-mirror-editor-tag-color: #170;
  --jp-mirror-editor-attribute-color: #00c;
  --jp-mirror-editor-header-color: blue;
  --jp-mirror-editor-quote-color: #090;
  --jp-mirror-editor-link-color: #00c;
  --jp-mirror-editor-error-color: #f00;
  --jp-mirror-editor-hr-color: #999;

  /* Vega extension styles */

  --jp-vega-background: white;

  /* Sidebar-related styles */

  --jp-sidebar-min-width: 250px;

  /* Search-related styles */

  --jp-search-toggle-off-opacity: 0.5;
  --jp-search-toggle-hover-opacity: 0.8;
  --jp-search-toggle-on-opacity: 1;
  --jp-search-selected-match-background-color: rgb(245, 200, 0);
  --jp-search-selected-match-color: black;
  --jp-search-unselected-match-background-color: var(
    --jp-inverse-layout-color0
  );
  --jp-search-unselected-match-color: var(--jp-ui-inverse-font-color0);

  /* Icon colors that work well with light or dark backgrounds */
  --jp-icon-contrast-color0: var(--md-purple-600);
  --jp-icon-contrast-color1: var(--md-green-600);
  --jp-icon-contrast-color2: var(--md-pink-600);
  --jp-icon-contrast-color3: var(--md-blue-600);
}
</style>

<style type="text/css">
a.anchor-link {
   display: none;
}
.highlight  {
    margin: 0.4em;
}

/* Input area styling */
.jp-InputArea {
    overflow: hidden;
}

.jp-InputArea-editor {
    overflow: hidden;
}

@media print {
  body {
    margin: 0;
  }
}
</style>

<!-- Load mathjax -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/latest.js?config=TeX-MML-AM_CHTML-full,Safe"> </script>
    <!-- MathJax configuration -->
    <script type="text/x-mathjax-config">
    init_mathjax = function() {
        if (window.MathJax) {
        // MathJax loaded
            MathJax.Hub.Config({
                TeX: {
                    equationNumbers: {
                    autoNumber: "AMS",
                    useLabelIds: true
                    }
                },
                tex2jax: {
                    inlineMath: [ ['$','$'], ["\\(","\\)"] ],
                    displayMath: [ ['$$','$$'], ["\\[","\\]"] ],
                    processEscapes: true,
                    processEnvironments: true
                },
                displayAlign: 'center',
                CommonHTML: {
                    linebreaks: { 
                    automatic: true 
                    }
                },
                "HTML-CSS": {
                    linebreaks: { 
                    automatic: true 
                    }
                }
            });
        
            MathJax.Hub.Queue(["Typeset", MathJax.Hub]);
        }
    }
    init_mathjax();
    </script>
    <!-- End of mathjax configuration --></head>
<body class="jp-Notebook" data-jp-theme-light="true" data-jp-theme-name="JupyterLab Light">

<div class="jp-Cell-inputWrapper"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput " data-mime-type="text/markdown">
<h1 id="Supervised-Learning-%EA%B8%B0%EB%B0%98-%EB%AC%B8%ED%97%8C-%EB%B6%84%EB%A5%98-(G&amp;I)">Supervised Learning &#44592;&#48152; &#47928;&#54732; &#48516;&#47448; (G&amp;I)<a class="anchor-link" href="#Supervised-Learning-%EA%B8%B0%EB%B0%98-%EB%AC%B8%ED%97%8C-%EB%B6%84%EB%A5%98-(G&amp;I)">&#182;</a></h1>
</div>
</div>
<div class="jp-Cell-inputWrapper"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput " data-mime-type="text/markdown">
<h2 id="1.-Importing-Libraries">1. Importing Libraries<a class="anchor-link" href="#1.-Importing-Libraries">&#182;</a></h2>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell jp-mod-noOutputs  ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[143]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span><span class="kn">import</span> <span class="nn">numpy</span> <span class="k">as</span> <span class="nn">np</span>
<span class="kn">import</span> <span class="nn">pandas</span> <span class="k">as</span> <span class="nn">pd</span>
<span class="kn">import</span> <span class="nn">nltk</span>
<span class="kn">import</span> <span class="nn">re</span>
<span class="kn">import</span> <span class="nn">os</span>
<span class="kn">import</span> <span class="nn">codecs</span>
<span class="kn">from</span> <span class="nn">sklearn</span> <span class="kn">import</span> <span class="n">feature_extraction</span>
<span class="kn">import</span> <span class="nn">mpld3</span>
</pre></div>

     </div>
</div>
</div>
</div>

</div>
<div class="jp-Cell-inputWrapper"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput " data-mime-type="text/markdown">
<h2 id="2.-Importing-the-Dataset">2. Importing the Dataset<a class="anchor-link" href="#2.-Importing-the-Dataset">&#182;</a></h2><p>GNI Corpus 1.0에서 vol.10부터 vol.17까지 301개의 논문지를 이용한다.</p>

</div>
</div>
<div class="jp-Cell-inputWrapper"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput " data-mime-type="text/markdown">
<h3 id="2-1.-sentence_tokenized-%EB%8D%B0%EC%9D%B4%ED%84%B0%EB%A5%BC-PlaintextCorpus%ED%99%94">2-1. sentence_tokenized &#45936;&#51060;&#53552;&#47484; PlaintextCorpus&#54868;<a class="anchor-link" href="#2-1.-sentence_tokenized-%EB%8D%B0%EC%9D%B4%ED%84%B0%EB%A5%BC-PlaintextCorpus%ED%99%94">&#182;</a></h3>
</div>
</div>
<div class="jp-Cell-inputWrapper"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput " data-mime-type="text/markdown">
<p>corpus_root = "C:/Users/s_eoy/AppData/Roaming/nltk_data/corpora/GNI Corpus 1.0/sentence_tokenized"
gniCorpus = nltk.corpus.PlaintextCorpusReader(corpus_root, ".*.txt", encoding = "utf_8")</p>

</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell jp-mod-noOutputs  ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[144]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span><span class="n">corpus_root</span> <span class="o">=</span> <span class="s2">&quot;C:/Users/USER/s_eoy/AppData/Genomics-Informatics-Corpus-master/GNI Corpus 1.0/sentence_tokenized&quot;</span>
<span class="n">gniCorpus</span> <span class="o">=</span> <span class="n">nltk</span><span class="o">.</span><span class="n">corpus</span><span class="o">.</span><span class="n">PlaintextCorpusReader</span><span class="p">(</span><span class="n">corpus_root</span><span class="p">,</span> <span class="s2">&quot;.*\.txt&quot;</span><span class="p">,</span> <span class="n">encoding</span> <span class="o">=</span> <span class="s2">&quot;utf_8&quot;</span><span class="p">)</span>
</pre></div>

     </div>
</div>
</div>
</div>

</div>
<div class="jp-Cell-inputWrapper"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput " data-mime-type="text/markdown">
<h3 id="2-2.-%EB%85%BC%EB%AC%B8%EC%A7%80-%EC%A0%95%EB%B3%B4%EB%A5%BC-list%EB%A1%9C-%EC%A0%80%EC%9E%A5">2-2. &#45436;&#47928;&#51648; &#51221;&#48372;&#47484; list&#47196; &#51200;&#51109;<a class="anchor-link" href="#2-2.-%EB%85%BC%EB%AC%B8%EC%A7%80-%EC%A0%95%EB%B3%B4%EB%A5%BC-list%EB%A1%9C-%EC%A0%80%EC%9E%A5">&#182;</a></h3>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell jp-mod-noOutputs  ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[160]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span><span class="n">stopwords</span> <span class="o">=</span> <span class="n">nltk</span><span class="o">.</span><span class="n">corpus</span><span class="o">.</span><span class="n">stopwords</span><span class="o">.</span><span class="n">words</span><span class="p">(</span><span class="s1">&#39;english&#39;</span><span class="p">)</span>
<span class="kn">from</span> <span class="nn">nltk.stem.wordnet</span> <span class="kn">import</span> <span class="n">WordNetLemmatizer</span>
<span class="n">lemmatizer</span> <span class="o">=</span> <span class="n">WordNetLemmatizer</span><span class="p">()</span>
</pre></div>

     </div>
</div>
</div>
</div>

</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell jp-mod-noOutputs  ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[190]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># 논문지의 제목을 titles에 저장하기</span>
<span class="n">titles</span> <span class="o">=</span> <span class="p">[</span><span class="n">title</span><span class="p">[:]</span> <span class="k">for</span> <span class="n">title</span> <span class="ow">in</span> <span class="n">gniCorpus</span><span class="o">.</span><span class="n">fileids</span><span class="p">()]</span>
<span class="c1"># 논문지의 volume number를 vols에 저장하기</span>
<span class="n">vols</span> <span class="o">=</span> <span class="p">[</span><span class="n">vol</span><span class="p">[</span><span class="mi">4</span><span class="p">:</span><span class="mi">6</span><span class="p">]</span> <span class="k">for</span> <span class="n">vol</span> <span class="ow">in</span> <span class="n">titles</span><span class="p">]</span>
<span class="c1"># 논문지의 발행 연도를 years에 저장하기</span>
<span class="c1"># 각 논문지의 volume number + 2 값이 발행 연도</span>
<span class="n">years</span> <span class="o">=</span> <span class="p">[</span><span class="nb">int</span><span class="p">(</span><span class="n">year</span><span class="p">)</span> <span class="o">+</span> <span class="mi">2</span> <span class="k">for</span> <span class="n">year</span> <span class="ow">in</span> <span class="n">vols</span> <span class="p">]</span>
</pre></div>

     </div>
</div>
</div>
</div>

</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell jp-mod-noOutputs  ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[191]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># K-means 알고리즘 적용하기</span>
</pre></div>

     </div>
</div>
</div>
</div>

</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell   ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[195]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span><span class="n">thesis</span> <span class="o">=</span> <span class="p">{</span><span class="s1">&#39;Title&#39;</span><span class="p">:</span> <span class="n">titles</span><span class="p">,</span> <span class="s1">&#39;Vol.&#39;</span><span class="p">:</span><span class="n">vols</span><span class="p">,</span> <span class="s1">&#39;Year&#39;</span><span class="p">:</span><span class="n">years</span><span class="p">}</span>
<span class="n">frame</span> <span class="o">=</span> <span class="n">pd</span><span class="o">.</span><span class="n">DataFrame</span><span class="p">(</span><span class="n">thesis</span><span class="p">,</span> <span class="n">index</span><span class="o">=</span><span class="p">[</span><span class="n">years</span><span class="p">],</span> <span class="n">columns</span><span class="o">=</span><span class="p">[</span><span class="s1">&#39;Title&#39;</span><span class="p">,</span><span class="s1">&#39;Vol.&#39;</span><span class="p">,</span><span class="s1">&#39;Year&#39;</span><span class="p">])</span>
<span class="n">frame</span><span class="o">.</span><span class="n">head</span><span class="p">(</span><span class="mi">167</span><span class="p">)</span>
</pre></div>

     </div>
</div>
</div>
</div>

<div class="jp-Cell-outputWrapper">


<div class="jp-OutputArea jp-Cell-outputArea">

<div class="jp-OutputArea-child">

    
    <div class="jp-OutputPrompt jp-OutputArea-prompt">Out[195]:</div>



<div class="jp-RenderedHTMLCommon jp-RenderedHTML jp-OutputArea-output jp-OutputArea-executeResult" data-mime-type="text/html">
<div>
<style scoped>
    .dataframe tbody tr th:only-of-type {
        vertical-align: middle;
    }

    .dataframe tbody tr th {
        vertical-align: top;
    }

    .dataframe thead th {
        text-align: right;
    }
</style>
<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>Title</th>
      <th>Vol.</th>
      <th>Year</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>12</th>
      <td>gni-10-1-1.txt</td>
      <td>10</td>
      <td>12</td>
    </tr>
    <tr>
      <th>12</th>
      <td>gni-10-1-16.txt</td>
      <td>10</td>
      <td>12</td>
    </tr>
    <tr>
      <th>12</th>
      <td>gni-10-1-23.txt</td>
      <td>10</td>
      <td>12</td>
    </tr>
    <tr>
      <th>12</th>
      <td>gni-10-1-33.txt</td>
      <td>10</td>
      <td>12</td>
    </tr>
    <tr>
      <th>12</th>
      <td>gni-10-1-40.txt</td>
      <td>10</td>
      <td>12</td>
    </tr>
    <tr>
      <th>...</th>
      <td>...</td>
      <td>...</td>
      <td>...</td>
    </tr>
    <tr>
      <th>16</th>
      <td>gni-14-1-1.txt</td>
      <td>14</td>
      <td>16</td>
    </tr>
    <tr>
      <th>16</th>
      <td>gni-14-1-12.txt</td>
      <td>14</td>
      <td>16</td>
    </tr>
    <tr>
      <th>16</th>
      <td>gni-14-1-2.txt</td>
      <td>14</td>
      <td>16</td>
    </tr>
    <tr>
      <th>16</th>
      <td>gni-14-1-20.txt</td>
      <td>14</td>
      <td>16</td>
    </tr>
    <tr>
      <th>16</th>
      <td>gni-14-1-29.txt</td>
      <td>14</td>
      <td>16</td>
    </tr>
  </tbody>
</table>
<p>167 rows × 3 columns</p>
</div>
</div>

</div>

</div>

</div>

</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell   ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[196]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># 각 연도별로 분류된 논문지 개수</span>
<span class="n">frame</span><span class="p">[</span><span class="s1">&#39;Year&#39;</span><span class="p">]</span><span class="o">.</span><span class="n">value_counts</span><span class="p">()</span>
</pre></div>

     </div>
</div>
</div>
</div>

<div class="jp-Cell-outputWrapper">


<div class="jp-OutputArea jp-Cell-outputArea">

<div class="jp-OutputArea-child">

    
    <div class="jp-OutputPrompt jp-OutputArea-prompt">Out[196]:</div>




<div class="jp-RenderedText jp-OutputArea-output jp-OutputArea-executeResult" data-mime-type="text/plain">
<pre>13    49
14    44
12    41
18    40
16    37
19    34
15    28
17    28
Name: Year, dtype: int64</pre>
</div>

</div>

</div>

</div>

</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell jp-mod-noOutputs  ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[197]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># 각 연도별로 중요한 단어(term)들과 속해있는 논문</span>
</pre></div>

     </div>
</div>
</div>
</div>

</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell jp-mod-noOutputs  ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[198]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span><span class="k">def</span> <span class="nf">tokenize_and_lemmatize</span><span class="p">(</span><span class="n">text</span><span class="p">):</span>
    <span class="n">tokens</span> <span class="o">=</span> <span class="p">[</span><span class="n">word</span><span class="o">.</span><span class="n">lower</span><span class="p">()</span> <span class="k">for</span> <span class="n">sent</span> <span class="ow">in</span> <span class="n">nltk</span><span class="o">.</span><span class="n">sent_tokenize</span><span class="p">(</span><span class="n">text</span><span class="p">)</span> <span class="k">for</span> <span class="n">word</span> <span class="ow">in</span> <span class="n">nltk</span><span class="o">.</span><span class="n">word_tokenize</span><span class="p">(</span><span class="n">sent</span><span class="p">)]</span>
    <span class="n">filtered_tokens</span> <span class="o">=</span> <span class="p">[]</span>
    <span class="k">for</span> <span class="n">token</span> <span class="ow">in</span> <span class="n">tokens</span><span class="p">:</span>
        <span class="k">if</span> <span class="n">re</span><span class="o">.</span><span class="n">search</span><span class="p">(</span><span class="s1">&#39;^[a-zA-Z]+$&#39;</span><span class="p">,</span><span class="n">token</span><span class="p">):</span>
            <span class="n">filtered_tokens</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">token</span><span class="p">)</span>
        <span class="k">if</span> <span class="n">re</span><span class="o">.</span><span class="n">search</span><span class="p">(</span><span class="s1">&#39;^[a-zA-Z]+[0-9]+$&#39;</span><span class="p">,</span><span class="n">token</span><span class="p">):</span>
            <span class="n">filtered_tokens</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">token</span><span class="p">)</span>
        <span class="k">if</span> <span class="n">re</span><span class="o">.</span><span class="n">search</span><span class="p">(</span><span class="s1">&#39;^[0-9]+[a-zA-Z]+$&#39;</span><span class="p">,</span><span class="n">token</span><span class="p">):</span>
            <span class="n">filtered_tokens</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">token</span><span class="p">)</span>
    <span class="n">lemmas</span> <span class="o">=</span> <span class="p">[</span><span class="n">lemmatizer</span><span class="o">.</span><span class="n">lemmatize</span><span class="p">(</span><span class="n">t</span><span class="p">)</span> <span class="k">for</span> <span class="n">t</span> <span class="ow">in</span> <span class="n">filtered_tokens</span><span class="p">]</span>
    <span class="k">return</span> <span class="n">lemmas</span>

<span class="c1"># 정규표현식을 사용해 필요한 단어만 추출, Lemmatize를 적용하기 전의 단어</span>
<span class="k">def</span> <span class="nf">tokenize_only</span><span class="p">(</span><span class="n">text</span><span class="p">):</span>
    <span class="n">tokens</span> <span class="o">=</span> <span class="p">[</span><span class="n">word</span> <span class="k">for</span> <span class="n">sent</span> <span class="ow">in</span> <span class="n">nltk</span><span class="o">.</span><span class="n">sent_tokenize</span><span class="p">(</span><span class="n">text</span><span class="p">)</span> <span class="k">for</span> <span class="n">word</span> <span class="ow">in</span> <span class="n">nltk</span><span class="o">.</span><span class="n">word_tokenize</span><span class="p">(</span><span class="n">sent</span><span class="p">)]</span>
    <span class="n">filtered_tokens</span><span class="o">=</span><span class="p">[]</span>
    <span class="k">for</span> <span class="n">token</span> <span class="ow">in</span> <span class="n">tokens</span><span class="p">:</span>
        <span class="k">if</span> <span class="n">re</span><span class="o">.</span><span class="n">search</span><span class="p">(</span><span class="s1">&#39;^[a-zA-Z]+$&#39;</span><span class="p">,</span><span class="n">token</span><span class="p">):</span>
            <span class="n">filtered_tokens</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">token</span><span class="p">)</span>
        <span class="k">if</span> <span class="n">re</span><span class="o">.</span><span class="n">search</span><span class="p">(</span><span class="s1">&#39;^[a-zA-Z]+[0-9]+$&#39;</span><span class="p">,</span><span class="n">token</span><span class="p">):</span>
            <span class="n">filtered_tokens</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">token</span><span class="p">)</span>
        <span class="k">if</span> <span class="n">re</span><span class="o">.</span><span class="n">search</span><span class="p">(</span><span class="s1">&#39;^[0-9]+[a-zA-Z]+$&#39;</span><span class="p">,</span><span class="n">token</span><span class="p">):</span>
            <span class="n">filtered_tokens</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">token</span><span class="p">)</span>
    <span class="k">return</span> <span class="n">filtered_tokens</span>
</pre></div>

     </div>
</div>
</div>
</div>

</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell   ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[199]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># raw에 함수 적용해 단어 추출하기</span>
<span class="n">totalvocab_lemmatized</span> <span class="o">=</span> <span class="p">[]</span>
<span class="n">totalvocab_tokenized</span> <span class="o">=</span> <span class="p">[]</span>
<span class="k">for</span> <span class="n">i</span> <span class="ow">in</span> <span class="n">raw</span><span class="p">:</span>
    <span class="n">allwords_lemmatized</span> <span class="o">=</span> <span class="n">tokenize_and_lemmatize</span><span class="p">(</span><span class="n">i</span><span class="p">)</span>
    <span class="n">totalvocab_lemmatized</span><span class="o">.</span><span class="n">extend</span><span class="p">(</span><span class="n">allwords_lemmatized</span><span class="p">)</span>
    
    <span class="n">allwords_tokenized</span> <span class="o">=</span> <span class="n">tokenize_only</span><span class="p">(</span><span class="n">i</span><span class="p">)</span>
    <span class="n">totalvocab_tokenized</span><span class="o">.</span><span class="n">extend</span><span class="p">(</span><span class="n">allwords_tokenized</span><span class="p">)</span>
    
<span class="nb">print</span><span class="p">(</span><span class="n">totalvocab_lemmatized</span><span class="p">[:</span><span class="mi">20</span><span class="p">])</span>
<span class="nb">print</span><span class="p">(</span><span class="n">totalvocab_tokenized</span><span class="p">[:</span><span class="mi">20</span><span class="p">])</span>
</pre></div>

     </div>
</div>
</div>
</div>

<div class="jp-Cell-outputWrapper">


<div class="jp-OutputArea jp-Cell-outputArea">

<div class="jp-OutputArea-child">

    
    <div class="jp-OutputPrompt jp-OutputArea-prompt"></div>


<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain">
<pre>[&#39;title&#39;, &#39;survey&#39;, &#39;of&#39;, &#39;the&#39;, &#39;application&#39;, &#39;of&#39;, &#39;ng&#39;, &#39;to&#39;, &#39;sequencing&#39;, &#39;and&#39;, &#39;expression&#39;, &#39;profiling&#39;, &#39;recently&#39;, &#39;the&#39;, &#39;technology&#39;, &#39;of&#39;, &#39;dna&#39;, &#39;sequence&#39;, &#39;variation&#39;, &#39;and&#39;]
[&#39;Title&#39;, &#39;Survey&#39;, &#39;of&#39;, &#39;the&#39;, &#39;Applications&#39;, &#39;of&#39;, &#39;NGS&#39;, &#39;to&#39;, &#39;Sequencing&#39;, &#39;and&#39;, &#39;Expression&#39;, &#39;Profiling&#39;, &#39;Recently&#39;, &#39;the&#39;, &#39;technologies&#39;, &#39;of&#39;, &#39;DNA&#39;, &#39;sequence&#39;, &#39;variation&#39;, &#39;and&#39;]
</pre>
</div>
</div>

</div>

</div>

</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell jp-mod-noOutputs  ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[200]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># Lemmatize가 적용된 단어의 원형을 찾을 수 있게 하는 데이터 생성</span>

<span class="n">vocab_frame</span> <span class="o">=</span> <span class="n">pd</span><span class="o">.</span><span class="n">DataFrame</span><span class="p">({</span><span class="s1">&#39;words&#39;</span><span class="p">:</span><span class="n">totalvocab_tokenized</span><span class="p">},</span> <span class="n">index</span><span class="o">=</span><span class="n">totalvocab_lemmatized</span><span class="p">)</span>

<span class="c1"># 특징(단어사전) - 알파벳 순 정렬</span>

<span class="n">terms</span> <span class="o">=</span> <span class="n">tfidf_vectorizer</span><span class="o">.</span><span class="n">get_feature_names</span><span class="p">()</span>
</pre></div>

     </div>
</div>
</div>
</div>

</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell   ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[201]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span><span class="kn">from</span> <span class="nn">__future__</span> <span class="kn">import</span> <span class="n">print_function</span>

<span class="nb">print</span><span class="p">(</span><span class="s2">&quot;Top terms per Year:&quot;</span><span class="p">)</span>
<span class="nb">print</span><span class="p">()</span>
<span class="n">order_centroids</span> <span class="o">=</span> <span class="n">km</span><span class="o">.</span><span class="n">cluster_centers_</span><span class="o">.</span><span class="n">argsort</span><span class="p">()[:,</span> <span class="p">::</span><span class="o">-</span><span class="mi">1</span><span class="p">]</span>
<span class="k">for</span> <span class="n">i</span> <span class="ow">in</span> <span class="nb">range</span><span class="p">(</span><span class="mi">8</span><span class="p">):</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">&quot;Year </span><span class="si">%d</span><span class="s2"> words:&quot;</span> <span class="o">%</span> <span class="n">i</span><span class="p">,</span> <span class="n">end</span><span class="o">=</span><span class="s1">&#39;&#39;</span><span class="p">)</span>
    <span class="k">for</span> <span class="n">ind</span> <span class="ow">in</span> <span class="n">order_centroids</span><span class="p">[</span><span class="mi">1</span><span class="p">,:</span><span class="mi">7</span><span class="p">]:</span>
        <span class="nb">print</span><span class="p">(</span><span class="s1">&#39; </span><span class="si">%s</span><span class="s1">&#39;</span> <span class="o">%</span> <span class="n">vocab_frame</span><span class="o">.</span><span class="n">loc</span><span class="p">[</span><span class="n">terms</span><span class="p">[</span><span class="n">ind</span><span class="p">]</span><span class="o">.</span><span class="n">split</span><span class="p">(</span><span class="s1">&#39; &#39;</span><span class="p">)]</span><span class="o">.</span><span class="n">values</span><span class="o">.</span><span class="n">tolist</span><span class="p">()[</span><span class="mi">0</span><span class="p">][</span><span class="mi">0</span><span class="p">],</span> <span class="n">end</span><span class="o">=</span><span class="s1">&#39;,&#39;</span><span class="p">)</span>
    <span class="nb">print</span><span class="p">()</span>
    <span class="nb">print</span><span class="p">()</span>
    <span class="nb">print</span><span class="p">(</span><span class="s2">&quot;Year </span><span class="si">%d</span><span class="s2"> titles:&quot;</span> <span class="o">%</span> <span class="n">i</span><span class="p">,</span> <span class="n">end</span><span class="o">=</span><span class="s1">&#39;&#39;</span><span class="p">)</span>
    <span class="k">for</span> <span class="n">title</span> <span class="ow">in</span> <span class="n">frame</span><span class="o">.</span><span class="n">loc</span><span class="p">[</span><span class="n">i</span><span class="p">][</span><span class="s1">&#39;Title&#39;</span><span class="p">]</span><span class="o">.</span><span class="n">values</span><span class="o">.</span><span class="n">tolist</span><span class="p">():</span>
        <span class="nb">print</span><span class="p">(</span><span class="s1">&#39; </span><span class="si">%s</span><span class="s1">.&#39;</span> <span class="o">%</span> <span class="n">title</span><span class="p">,</span> <span class="n">end</span><span class="o">=</span><span class="s1">&#39;&#39;</span><span class="p">)</span>
    <span class="nb">print</span><span class="p">()</span>
    <span class="nb">print</span><span class="p">()</span>
</pre></div>

     </div>
</div>
</div>
</div>

<div class="jp-Cell-outputWrapper">


<div class="jp-OutputArea jp-Cell-outputArea">

<div class="jp-OutputArea-child">

    
    <div class="jp-OutputPrompt jp-OutputArea-prompt"></div>


<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain">
<pre>Top terms per Year:

Year 0 words: cell, Expression, cancers, protein, gene, level, RNAs,

Year 0 titles:</pre>
</div>
</div>

<div class="jp-OutputArea-child">

    
    <div class="jp-OutputPrompt jp-OutputArea-prompt"></div>


<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="application/vnd.jupyter.stderr">
<pre>
<span class="ansi-red-intense-fg ansi-bold">---------------------------------------------------------------------------</span>
<span class="ansi-red-intense-fg ansi-bold">KeyError</span>                                  Traceback (most recent call last)
<span class="ansi-green-intense-fg ansi-bold">~\Documents\python\lib\site-packages\pandas\core\indexes\base.py</span> in <span class="ansi-cyan-fg">get_loc</span><span class="ansi-blue-intense-fg ansi-bold">(self, key, method, tolerance)</span>
<span class="ansi-green-fg">   3360</span>             <span class="ansi-green-intense-fg ansi-bold">try</span><span class="ansi-yellow-intense-fg ansi-bold">:</span>
<span class="ansi-green-intense-fg ansi-bold">-&gt; 3361</span><span class="ansi-yellow-intense-fg ansi-bold">                 </span><span class="ansi-green-intense-fg ansi-bold">return</span> self<span class="ansi-yellow-intense-fg ansi-bold">.</span>_engine<span class="ansi-yellow-intense-fg ansi-bold">.</span>get_loc<span class="ansi-yellow-intense-fg ansi-bold">(</span>casted_key<span class="ansi-yellow-intense-fg ansi-bold">)</span>
<span class="ansi-green-fg">   3362</span>             <span class="ansi-green-intense-fg ansi-bold">except</span> KeyError <span class="ansi-green-intense-fg ansi-bold">as</span> err<span class="ansi-yellow-intense-fg ansi-bold">:</span>

<span class="ansi-green-intense-fg ansi-bold">~\Documents\python\lib\site-packages\pandas\_libs\index.pyx</span> in <span class="ansi-cyan-fg">pandas._libs.index.IndexEngine.get_loc</span><span class="ansi-blue-intense-fg ansi-bold">()</span>

<span class="ansi-green-intense-fg ansi-bold">~\Documents\python\lib\site-packages\pandas\_libs\index.pyx</span> in <span class="ansi-cyan-fg">pandas._libs.index.IndexEngine.get_loc</span><span class="ansi-blue-intense-fg ansi-bold">()</span>

<span class="ansi-green-intense-fg ansi-bold">pandas\_libs\hashtable_class_helper.pxi</span> in <span class="ansi-cyan-fg">pandas._libs.hashtable.Int64HashTable.get_item</span><span class="ansi-blue-intense-fg ansi-bold">()</span>

<span class="ansi-green-intense-fg ansi-bold">pandas\_libs\hashtable_class_helper.pxi</span> in <span class="ansi-cyan-fg">pandas._libs.hashtable.Int64HashTable.get_item</span><span class="ansi-blue-intense-fg ansi-bold">()</span>

<span class="ansi-red-intense-fg ansi-bold">KeyError</span>: 0

The above exception was the direct cause of the following exception:

<span class="ansi-red-intense-fg ansi-bold">KeyError</span>                                  Traceback (most recent call last)
<span class="ansi-green-intense-fg ansi-bold">C:\Users\Public\Documents\ESTsoft\CreatorTemp/ipykernel_1588/160374840.py</span> in <span class="ansi-cyan-fg">&lt;module&gt;</span>
<span class="ansi-green-fg">     11</span>     print<span class="ansi-yellow-intense-fg ansi-bold">(</span><span class="ansi-yellow-intense-fg ansi-bold">)</span>
<span class="ansi-green-fg">     12</span>     print<span class="ansi-yellow-intense-fg ansi-bold">(</span><span class="ansi-blue-intense-fg ansi-bold">&#34;Year %d titles:&#34;</span> <span class="ansi-yellow-intense-fg ansi-bold">%</span> i<span class="ansi-yellow-intense-fg ansi-bold">,</span> end<span class="ansi-yellow-intense-fg ansi-bold">=</span><span class="ansi-blue-intense-fg ansi-bold">&#39;&#39;</span><span class="ansi-yellow-intense-fg ansi-bold">)</span>
<span class="ansi-green-intense-fg ansi-bold">---&gt; 13</span><span class="ansi-yellow-intense-fg ansi-bold">     </span><span class="ansi-green-intense-fg ansi-bold">for</span> title <span class="ansi-green-intense-fg ansi-bold">in</span> frame<span class="ansi-yellow-intense-fg ansi-bold">.</span>loc<span class="ansi-yellow-intense-fg ansi-bold">[</span>i<span class="ansi-yellow-intense-fg ansi-bold">]</span><span class="ansi-yellow-intense-fg ansi-bold">[</span><span class="ansi-blue-intense-fg ansi-bold">&#39;Title&#39;</span><span class="ansi-yellow-intense-fg ansi-bold">]</span><span class="ansi-yellow-intense-fg ansi-bold">.</span>values<span class="ansi-yellow-intense-fg ansi-bold">.</span>tolist<span class="ansi-yellow-intense-fg ansi-bold">(</span><span class="ansi-yellow-intense-fg ansi-bold">)</span><span class="ansi-yellow-intense-fg ansi-bold">:</span>
<span class="ansi-green-fg">     14</span>         print<span class="ansi-yellow-intense-fg ansi-bold">(</span><span class="ansi-blue-intense-fg ansi-bold">&#39; %s.&#39;</span> <span class="ansi-yellow-intense-fg ansi-bold">%</span> title<span class="ansi-yellow-intense-fg ansi-bold">,</span> end<span class="ansi-yellow-intense-fg ansi-bold">=</span><span class="ansi-blue-intense-fg ansi-bold">&#39;&#39;</span><span class="ansi-yellow-intense-fg ansi-bold">)</span>
<span class="ansi-green-fg">     15</span>     print<span class="ansi-yellow-intense-fg ansi-bold">(</span><span class="ansi-yellow-intense-fg ansi-bold">)</span>

<span class="ansi-green-intense-fg ansi-bold">~\Documents\python\lib\site-packages\pandas\core\indexing.py</span> in <span class="ansi-cyan-fg">__getitem__</span><span class="ansi-blue-intense-fg ansi-bold">(self, key)</span>
<span class="ansi-green-fg">    929</span> 
<span class="ansi-green-fg">    930</span>             maybe_callable <span class="ansi-yellow-intense-fg ansi-bold">=</span> com<span class="ansi-yellow-intense-fg ansi-bold">.</span>apply_if_callable<span class="ansi-yellow-intense-fg ansi-bold">(</span>key<span class="ansi-yellow-intense-fg ansi-bold">,</span> self<span class="ansi-yellow-intense-fg ansi-bold">.</span>obj<span class="ansi-yellow-intense-fg ansi-bold">)</span>
<span class="ansi-green-intense-fg ansi-bold">--&gt; 931</span><span class="ansi-yellow-intense-fg ansi-bold">             </span><span class="ansi-green-intense-fg ansi-bold">return</span> self<span class="ansi-yellow-intense-fg ansi-bold">.</span>_getitem_axis<span class="ansi-yellow-intense-fg ansi-bold">(</span>maybe_callable<span class="ansi-yellow-intense-fg ansi-bold">,</span> axis<span class="ansi-yellow-intense-fg ansi-bold">=</span>axis<span class="ansi-yellow-intense-fg ansi-bold">)</span>
<span class="ansi-green-fg">    932</span> 
<span class="ansi-green-fg">    933</span>     <span class="ansi-green-intense-fg ansi-bold">def</span> _is_scalar_access<span class="ansi-yellow-intense-fg ansi-bold">(</span>self<span class="ansi-yellow-intense-fg ansi-bold">,</span> key<span class="ansi-yellow-intense-fg ansi-bold">:</span> tuple<span class="ansi-yellow-intense-fg ansi-bold">)</span><span class="ansi-yellow-intense-fg ansi-bold">:</span>

<span class="ansi-green-intense-fg ansi-bold">~\Documents\python\lib\site-packages\pandas\core\indexing.py</span> in <span class="ansi-cyan-fg">_getitem_axis</span><span class="ansi-blue-intense-fg ansi-bold">(self, key, axis)</span>
<span class="ansi-green-fg">   1162</span>         <span class="ansi-red-intense-fg ansi-bold"># fall thru to straight lookup</span>
<span class="ansi-green-fg">   1163</span>         self<span class="ansi-yellow-intense-fg ansi-bold">.</span>_validate_key<span class="ansi-yellow-intense-fg ansi-bold">(</span>key<span class="ansi-yellow-intense-fg ansi-bold">,</span> axis<span class="ansi-yellow-intense-fg ansi-bold">)</span>
<span class="ansi-green-intense-fg ansi-bold">-&gt; 1164</span><span class="ansi-yellow-intense-fg ansi-bold">         </span><span class="ansi-green-intense-fg ansi-bold">return</span> self<span class="ansi-yellow-intense-fg ansi-bold">.</span>_get_label<span class="ansi-yellow-intense-fg ansi-bold">(</span>key<span class="ansi-yellow-intense-fg ansi-bold">,</span> axis<span class="ansi-yellow-intense-fg ansi-bold">=</span>axis<span class="ansi-yellow-intense-fg ansi-bold">)</span>
<span class="ansi-green-fg">   1165</span> 
<span class="ansi-green-fg">   1166</span>     <span class="ansi-green-intense-fg ansi-bold">def</span> _get_slice_axis<span class="ansi-yellow-intense-fg ansi-bold">(</span>self<span class="ansi-yellow-intense-fg ansi-bold">,</span> slice_obj<span class="ansi-yellow-intense-fg ansi-bold">:</span> slice<span class="ansi-yellow-intense-fg ansi-bold">,</span> axis<span class="ansi-yellow-intense-fg ansi-bold">:</span> int<span class="ansi-yellow-intense-fg ansi-bold">)</span><span class="ansi-yellow-intense-fg ansi-bold">:</span>

<span class="ansi-green-intense-fg ansi-bold">~\Documents\python\lib\site-packages\pandas\core\indexing.py</span> in <span class="ansi-cyan-fg">_get_label</span><span class="ansi-blue-intense-fg ansi-bold">(self, label, axis)</span>
<span class="ansi-green-fg">   1111</span>     <span class="ansi-green-intense-fg ansi-bold">def</span> _get_label<span class="ansi-yellow-intense-fg ansi-bold">(</span>self<span class="ansi-yellow-intense-fg ansi-bold">,</span> label<span class="ansi-yellow-intense-fg ansi-bold">,</span> axis<span class="ansi-yellow-intense-fg ansi-bold">:</span> int<span class="ansi-yellow-intense-fg ansi-bold">)</span><span class="ansi-yellow-intense-fg ansi-bold">:</span>
<span class="ansi-green-fg">   1112</span>         <span class="ansi-red-intense-fg ansi-bold"># GH#5667 this will fail if the label is not present in the axis.</span>
<span class="ansi-green-intense-fg ansi-bold">-&gt; 1113</span><span class="ansi-yellow-intense-fg ansi-bold">         </span><span class="ansi-green-intense-fg ansi-bold">return</span> self<span class="ansi-yellow-intense-fg ansi-bold">.</span>obj<span class="ansi-yellow-intense-fg ansi-bold">.</span>xs<span class="ansi-yellow-intense-fg ansi-bold">(</span>label<span class="ansi-yellow-intense-fg ansi-bold">,</span> axis<span class="ansi-yellow-intense-fg ansi-bold">=</span>axis<span class="ansi-yellow-intense-fg ansi-bold">)</span>
<span class="ansi-green-fg">   1114</span> 
<span class="ansi-green-fg">   1115</span>     <span class="ansi-green-intense-fg ansi-bold">def</span> _handle_lowerdim_multi_index_axis0<span class="ansi-yellow-intense-fg ansi-bold">(</span>self<span class="ansi-yellow-intense-fg ansi-bold">,</span> tup<span class="ansi-yellow-intense-fg ansi-bold">:</span> tuple<span class="ansi-yellow-intense-fg ansi-bold">)</span><span class="ansi-yellow-intense-fg ansi-bold">:</span>

<span class="ansi-green-intense-fg ansi-bold">~\Documents\python\lib\site-packages\pandas\core\generic.py</span> in <span class="ansi-cyan-fg">xs</span><span class="ansi-blue-intense-fg ansi-bold">(self, key, axis, level, drop_level)</span>
<span class="ansi-green-fg">   3768</span>         <span class="ansi-green-intense-fg ansi-bold">if</span> isinstance<span class="ansi-yellow-intense-fg ansi-bold">(</span>index<span class="ansi-yellow-intense-fg ansi-bold">,</span> MultiIndex<span class="ansi-yellow-intense-fg ansi-bold">)</span><span class="ansi-yellow-intense-fg ansi-bold">:</span>
<span class="ansi-green-fg">   3769</span>             <span class="ansi-green-intense-fg ansi-bold">try</span><span class="ansi-yellow-intense-fg ansi-bold">:</span>
<span class="ansi-green-intense-fg ansi-bold">-&gt; 3770</span><span class="ansi-yellow-intense-fg ansi-bold">                 loc, new_index = index._get_loc_level(
</span><span class="ansi-green-fg">   3771</span>                     key<span class="ansi-yellow-intense-fg ansi-bold">,</span> level<span class="ansi-yellow-intense-fg ansi-bold">=</span><span class="ansi-cyan-intense-fg ansi-bold">0</span><span class="ansi-yellow-intense-fg ansi-bold">,</span> drop_level<span class="ansi-yellow-intense-fg ansi-bold">=</span>drop_level
<span class="ansi-green-fg">   3772</span>                 )

<span class="ansi-green-intense-fg ansi-bold">~\Documents\python\lib\site-packages\pandas\core\indexes\multi.py</span> in <span class="ansi-cyan-fg">_get_loc_level</span><span class="ansi-blue-intense-fg ansi-bold">(self, key, level, drop_level)</span>
<span class="ansi-green-fg">   3110</span>                 <span class="ansi-green-intense-fg ansi-bold">return</span> indexer<span class="ansi-yellow-intense-fg ansi-bold">,</span> maybe_mi_droplevels<span class="ansi-yellow-intense-fg ansi-bold">(</span>indexer<span class="ansi-yellow-intense-fg ansi-bold">,</span> ilevels<span class="ansi-yellow-intense-fg ansi-bold">,</span> drop_level<span class="ansi-yellow-intense-fg ansi-bold">)</span>
<span class="ansi-green-fg">   3111</span>         <span class="ansi-green-intense-fg ansi-bold">else</span><span class="ansi-yellow-intense-fg ansi-bold">:</span>
<span class="ansi-green-intense-fg ansi-bold">-&gt; 3112</span><span class="ansi-yellow-intense-fg ansi-bold">             </span>indexer <span class="ansi-yellow-intense-fg ansi-bold">=</span> self<span class="ansi-yellow-intense-fg ansi-bold">.</span>_get_level_indexer<span class="ansi-yellow-intense-fg ansi-bold">(</span>key<span class="ansi-yellow-intense-fg ansi-bold">,</span> level<span class="ansi-yellow-intense-fg ansi-bold">=</span>level<span class="ansi-yellow-intense-fg ansi-bold">)</span>
<span class="ansi-green-fg">   3113</span>             <span class="ansi-green-intense-fg ansi-bold">return</span> indexer<span class="ansi-yellow-intense-fg ansi-bold">,</span> maybe_mi_droplevels<span class="ansi-yellow-intense-fg ansi-bold">(</span>indexer<span class="ansi-yellow-intense-fg ansi-bold">,</span> <span class="ansi-yellow-intense-fg ansi-bold">[</span>level<span class="ansi-yellow-intense-fg ansi-bold">]</span><span class="ansi-yellow-intense-fg ansi-bold">,</span> drop_level<span class="ansi-yellow-intense-fg ansi-bold">)</span>
<span class="ansi-green-fg">   3114</span> 

<span class="ansi-green-intense-fg ansi-bold">~\Documents\python\lib\site-packages\pandas\core\indexes\multi.py</span> in <span class="ansi-cyan-fg">_get_level_indexer</span><span class="ansi-blue-intense-fg ansi-bold">(self, key, level, indexer)</span>
<span class="ansi-green-fg">   3202</span>         <span class="ansi-green-intense-fg ansi-bold">else</span><span class="ansi-yellow-intense-fg ansi-bold">:</span>
<span class="ansi-green-fg">   3203</span> 
<span class="ansi-green-intense-fg ansi-bold">-&gt; 3204</span><span class="ansi-yellow-intense-fg ansi-bold">             </span>idx <span class="ansi-yellow-intense-fg ansi-bold">=</span> self<span class="ansi-yellow-intense-fg ansi-bold">.</span>_get_loc_single_level_index<span class="ansi-yellow-intense-fg ansi-bold">(</span>level_index<span class="ansi-yellow-intense-fg ansi-bold">,</span> key<span class="ansi-yellow-intense-fg ansi-bold">)</span>
<span class="ansi-green-fg">   3205</span> 
<span class="ansi-green-fg">   3206</span>             <span class="ansi-green-intense-fg ansi-bold">if</span> level <span class="ansi-yellow-intense-fg ansi-bold">&gt;</span> <span class="ansi-cyan-intense-fg ansi-bold">0</span> <span class="ansi-green-intense-fg ansi-bold">or</span> self<span class="ansi-yellow-intense-fg ansi-bold">.</span>_lexsort_depth <span class="ansi-yellow-intense-fg ansi-bold">==</span> <span class="ansi-cyan-intense-fg ansi-bold">0</span><span class="ansi-yellow-intense-fg ansi-bold">:</span>

<span class="ansi-green-intense-fg ansi-bold">~\Documents\python\lib\site-packages\pandas\core\indexes\multi.py</span> in <span class="ansi-cyan-fg">_get_loc_single_level_index</span><span class="ansi-blue-intense-fg ansi-bold">(self, level_index, key)</span>
<span class="ansi-green-fg">   2853</span>             <span class="ansi-green-intense-fg ansi-bold">return</span> <span class="ansi-yellow-intense-fg ansi-bold">-</span><span class="ansi-cyan-intense-fg ansi-bold">1</span>
<span class="ansi-green-fg">   2854</span>         <span class="ansi-green-intense-fg ansi-bold">else</span><span class="ansi-yellow-intense-fg ansi-bold">:</span>
<span class="ansi-green-intense-fg ansi-bold">-&gt; 2855</span><span class="ansi-yellow-intense-fg ansi-bold">             </span><span class="ansi-green-intense-fg ansi-bold">return</span> level_index<span class="ansi-yellow-intense-fg ansi-bold">.</span>get_loc<span class="ansi-yellow-intense-fg ansi-bold">(</span>key<span class="ansi-yellow-intense-fg ansi-bold">)</span>
<span class="ansi-green-fg">   2856</span> 
<span class="ansi-green-fg">   2857</span>     <span class="ansi-green-intense-fg ansi-bold">def</span> get_loc<span class="ansi-yellow-intense-fg ansi-bold">(</span>self<span class="ansi-yellow-intense-fg ansi-bold">,</span> key<span class="ansi-yellow-intense-fg ansi-bold">,</span> method<span class="ansi-yellow-intense-fg ansi-bold">=</span><span class="ansi-green-intense-fg ansi-bold">None</span><span class="ansi-yellow-intense-fg ansi-bold">)</span><span class="ansi-yellow-intense-fg ansi-bold">:</span>

<span class="ansi-green-intense-fg ansi-bold">~\Documents\python\lib\site-packages\pandas\core\indexes\base.py</span> in <span class="ansi-cyan-fg">get_loc</span><span class="ansi-blue-intense-fg ansi-bold">(self, key, method, tolerance)</span>
<span class="ansi-green-fg">   3361</span>                 <span class="ansi-green-intense-fg ansi-bold">return</span> self<span class="ansi-yellow-intense-fg ansi-bold">.</span>_engine<span class="ansi-yellow-intense-fg ansi-bold">.</span>get_loc<span class="ansi-yellow-intense-fg ansi-bold">(</span>casted_key<span class="ansi-yellow-intense-fg ansi-bold">)</span>
<span class="ansi-green-fg">   3362</span>             <span class="ansi-green-intense-fg ansi-bold">except</span> KeyError <span class="ansi-green-intense-fg ansi-bold">as</span> err<span class="ansi-yellow-intense-fg ansi-bold">:</span>
<span class="ansi-green-intense-fg ansi-bold">-&gt; 3363</span><span class="ansi-yellow-intense-fg ansi-bold">                 </span><span class="ansi-green-intense-fg ansi-bold">raise</span> KeyError<span class="ansi-yellow-intense-fg ansi-bold">(</span>key<span class="ansi-yellow-intense-fg ansi-bold">)</span> <span class="ansi-green-intense-fg ansi-bold">from</span> err
<span class="ansi-green-fg">   3364</span> 
<span class="ansi-green-fg">   3365</span>         <span class="ansi-green-intense-fg ansi-bold">if</span> is_scalar<span class="ansi-yellow-intense-fg ansi-bold">(</span>key<span class="ansi-yellow-intense-fg ansi-bold">)</span> <span class="ansi-green-intense-fg ansi-bold">and</span> isna<span class="ansi-yellow-intense-fg ansi-bold">(</span>key<span class="ansi-yellow-intense-fg ansi-bold">)</span> <span class="ansi-green-intense-fg ansi-bold">and</span> <span class="ansi-green-intense-fg ansi-bold">not</span> self<span class="ansi-yellow-intense-fg ansi-bold">.</span>hasnans<span class="ansi-yellow-intense-fg ansi-bold">:</span>

<span class="ansi-red-intense-fg ansi-bold">KeyError</span>: 0</pre>
</div>
</div>

</div>

</div>

</div>
<div class="jp-Cell-inputWrapper"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput " data-mime-type="text/markdown">
<h3 id="2-3.-%EB%AA%A8%EB%93%A0-%EB%85%BC%EB%AC%B8%EC%A7%80%EC%9D%98-%EB%82%B4%EC%9A%A9%EC%9D%84-list%EB%A1%9C-%EC%A0%80%EC%9E%A5">2-3. &#47784;&#46304; &#45436;&#47928;&#51648;&#51032; &#45236;&#50857;&#51012; list&#47196; &#51200;&#51109;<a class="anchor-link" href="#2-3.-%EB%AA%A8%EB%93%A0-%EB%85%BC%EB%AC%B8%EC%A7%80%EC%9D%98-%EB%82%B4%EC%9A%A9%EC%9D%84-list%EB%A1%9C-%EC%A0%80%EC%9E%A5">&#182;</a></h3>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell   ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[149]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span><span class="n">raw</span> <span class="o">=</span> <span class="p">[</span> <span class="n">gniCorpus</span><span class="o">.</span><span class="n">raw</span><span class="p">(</span><span class="n">title</span> <span class="o">+</span> <span class="s2">&quot;.txt&quot;</span><span class="p">)</span> <span class="k">for</span> <span class="n">title</span> <span class="ow">in</span> <span class="n">titles</span> <span class="p">]</span>
<span class="n">raw</span><span class="p">[:</span><span class="mi">5</span><span class="p">]</span>
</pre></div>

     </div>
</div>
</div>
</div>

<div class="jp-Cell-outputWrapper">


<div class="jp-OutputArea jp-Cell-outputArea">

<div class="jp-OutputArea-child">

    
    <div class="jp-OutputPrompt jp-OutputArea-prompt">Out[149]:</div>




<div class="jp-RenderedText jp-OutputArea-output jp-OutputArea-executeResult" data-mime-type="text/plain">
<pre>[&#39;Title: Survey of the Applications of NGS to Whole-Genome Sequencing and Expression Profiling\n\nRecently, the technologies of DNA sequence variation and gene expression profiling have been used widely as approaches in the expertise of genome biology and genetics.\n\nThe application to genome study has been particularly developed with the introduction of the next-generation DNA sequencer (NGS) Roche/454 and Illumina/Solexa systems, along with bioinformation analysis technologies of whole-genome de novo assembly, expression profiling, DNA variation discovery, and genotyping.\n\nBoth massive whole-genome shotgun paired-end sequencing and mate paired-end sequencing data are important steps for constructing de novo assembly of novel genome sequencing data.\n\nIt is necessary to have DNA sequence information from a multiplatform NGS with at least 2× and 30× depth sequence of genome coverage using Roche/454 and Illumina/Solexa, respectively, for effective an way of de novo assembly.\n\nMassive short-length reading data from the Illumina/Solexa system is enough to discover DNA variation, resulting in reducing the cost of DNA sequencing.\n\nWhole-genome expression profile data are useful to approach genome system biology with quantification of expressed RNAs from a whole-genome transcriptome, depending on the tissue samples.\n\nThe hybrid mRNA sequences from Rohce/454 and Illumina/Solexa are more powerful to find novel genes through de novo assembly in any whole-genome sequenced species.\n\nThe 20× and 50× coverage of the estimated transcriptome sequences using Roche/454 and Illumina/Solexa, respectively, is effective to create novel expressed reference sequences.\n\nHowever, only an average 30× coverage of a transcriptome with short read sequences of Illumina/Solexa is enough to check expression quantification, compared to the reference expressed sequence tag sequence.\n\nScientists have tried to understand biology through DNA sequence information, since the DNA is verified as the unit of genetic heredity.\n\nAlso, scientists hope to dramatically reduce the cost of reading genomic DNA and to obtain the high-throughput DNA sequence information.\n\nSince the automated DNA sequencers were developed with fluorescent dyes of different colors, laser, and computer technology in the 1980s, the human genome project (HGP) was begun in 1990, and the human genome was completely released in 2003, while further analysis is still being published.\n\nA total of about 3 billion dollars was invested to the project.\n\nIn 1991, the National Human Genome Research Institute (NHGRI) funds were geared toward lowering the cost of DNA sequencing.\n\nSome of technologies invested improved the DNA sequencing.\n\nTo date, the Applied Biosystems, Roche/454, and Illumina/Solexa have successfully developed their technology and applied DNA sequencing in the world during the recent 6 years.\n\nMost of the newest technologies currently in use generate sequences from 36 to 1,000 base pairs, which requires special software for different applications, including whole-genome sequencing, transcriptome analysis, and regulatory gene analysis.\n\nIn particular, in silico method development using bioinformation software for next-generation sequence assembly would be alert for many genome projects and more applications in genome biology.\n\nParticularly, many biologists and geneticists using massive DNA and RNA sequences have used the sequencing applications focused on the variable research fields in medicine, such as improved diagnosis of disease; gene therapy; control systems for drugs, including pharmacogenomics &#34;custom drugs;&#34; evolution; bioenergy and environmental applications of creative energy sources (biofuels); clean up of toxic wastes, including efficient environmental sources against carbon; and agriculture projects, including livestock of healthier, more productive, disease-resistant farm animals and breeding of disease-, insect-, and drought-resistant crops.\n\nIn this paper, we report several ways of genome sequencing and expression profiling in genome biology.\n\nTitle: Roche/454 pyrosequencing technology\n\nThe first high-parallel sequencing system was developed with an emulsion PCR method for DNA amplification and an instrument for sequencing by synthesis using a pyrosequencing protocol optimized on the individual well of a PicoTiterPlate (PTP) [1].\n\nThe DNA sequencing protocol, including sample preparation, is supplied for a user to follow the method, which is developed by 454 Life Science (now part of Roche Diagnostics, Mannheim, Germany).\n\nEven following the manufacturer\&#39;s protocol, to obtain high-quality DNA sequence with maximal total DNA sequence length as possible, one should make a library with both adapters on the sheared DNA fragment and mix the optical ratio of library DNA versus bead for emulsion PCR.\n\nNow, the system is upgraded to through-put a total of an average of 700 Mbp with an average 600 bp/read in one PTP run.\n\nThe sequencer is available for single reads and paired-end read sequencing for the application of eukaryotic and prokaryotic whole-genome sequencing [2, 3], metagenomics and microbial diversity [4, 5], and genetic variation detection for comparative genomics [6].\n\nTitle: Illumina/Solexa technology\n\nThe numerous cost-effective technologies were being developed for human genome resequencing that can be aligned to the reference sequence [7, 8].\n\nThe first successful technology to gain massive DNA sequencing available for resequencing was developed by Solexa (now part of Illumina, San Diego, CA, USA).\n\nThe principle of the system is the method of base-by-base sequencing by synthesis, where the sheared template DNA is amplified on the flat surface slide (flow cell) and detects one base on each template per cycle with four base-specific, fluorescently labeled signals.\n\nSignals for all four fluorescent channels are collected and plotted at each position, enabling quality per scores to be derived using four-color information if desired [8].\n\nNow, a maximum of 300 Gb for reading is available with 101 bp paired-end reading per fragment on a 1-flow cell run in the upgraded HiSeq system.\n\nTitle: Novel whole genome de novo assembly\n\nMore than 11,000 sequencing projects, including targeted projects, were reported on the Genome Online Database (GOLD, http://www.genomesonline.org) in early 2012.\n\nNow, more than 3,000 genome projects have been completed on the diverse genome species, and more than 90% of completed projects were bacterial genome sequencing.\n\nThe greatest bacterial genome sequencing was performed with 454 pyrosequencing because of the available largest long read sequencing, useful for de novo assembly of novel genome sequencing.\n\nThe official depth of the deep sequencing strategy of 454 pyrosequencing technology for whole bacterial genome sequencing for de novo assembly in novel genome sequencing is at least 15-20× in depth of the estimated genome size [9-13].\n\nHowever, Li et al.[3] reported that 6-10× sequencing in qualified runs with 500-bp reads would be enough for de novo assemblies from 1,480 prokaryote genomes with &gt;98% genome coverage, &lt;100 contigs with N50, and size &gt;100 kb.\n\nRecently, prokaryote whole genome sequencing using 101 bp paired-end read data from Illumina/Solexa systems was used for de novo assembly and resequencing.\n\nFor example, a Bacillus subtilis subspecies genome sequence was generated by using the short read sequence from Illumina/Solexa and assembled with the Velvet program [14].\n\nIn this case, the genome assembly was completed, based on the reference genome for ordering the numerous contigs derived from de novo assembly.\n\nEven though numerous contigs assembled with Illumina/Solexa data were produced in the eukaryotic genome, a few drafts for the assembled genome sequence were reported, except for the giant panda genome [15], which was covered with assembled contigs (2.25 Gb), covering approximately 94% of the expected whole genome.\n\nAnother example was the woodland strawberry genome (240 Mb) [16] that was sequenced to 39× depth of the genome, assembled de novo, and anchored to the linkage map of seven pseudochromosomes.\n\nThe genome sequence could be associated with the predicted genes with transcriptome sequence data.\n\nAn ideal method for cost-effective novel genome sequencing using NGS is de novo assembly with diverse shotgun fragment end sequencing data of multiplat systems (Fig.1).\n\nThe first strategy of novel genome DNA sequencing is sequencing the genomic DNA for contig and scaffold construction after randomly sheared shotgun single read-end or paired-end read DNA sequencing using Roche/454 or Illumina/Solexa with information on how to assemble with the NGS data using variable assembly software.\n\nRecently, a catfish genome was sequenced with multiplatform Roche/454 and Illumina/Solexa technology and assembled with an effective combination of low coverage depth of 18× Roche/454 and 70× Illumina/Solexa data using 3 assembly softwares - Newbler software to the 454 reads, Velvet assembler to the Illumina read, and MIRA assembler for final assembly of contigs and singletons derived from initial assembled data - resulting in 193 contigs with an N50 value of 13,123 bp [2].\n\nIn an additional multiplatform data assembly of a 40-Mb eukaryotic genome of the fungus Sordaria macrospra, a combination sequence of 85-fold coverage of Illumina/Solexa and 10-fold coverage by Roche/454 sequencing was assembled to a 40-Mb draft version (N50 of 117 kb) with the Velvet assembler as a reference of a model organism for fungal morphogenesis [17].\n\nIn the recent effective assembly methods reported, combinations of the multiplatform sequence are shown as successful novel genome assembly using variable assembly strategy pipelines.\n\nComparing the pipeline of assembly strategy, we suggest an effective integrated pipeline in which data are filtered to remove low-quality and short-read initial assemblies using variable software and then compared to contigs, hybrid contigs using MIRA assembler, and finally contig orders using SSPACE software (http://www.baseclear.com/dna-sequencing/data-analysis/) [18] for scaffold construction through de novo assembly of novel genome sequencing (Fig.2).\n\nAccording to the comparison of several ways of de novo assembly, we suggest using both DNA sequences from multiplatform NGS with at least 2× and 30× depth sequences of genome coverage using Roche/454 and Illumina/Solexa, respectively, and doing hybrid assembly for cost-effective novel genome sequencing.\n\nTitle: SNP discovery and genotyping with resequencing\n\nResequencing of genomic regions or target genes of interest in a phenotype is the first step in the detection of DNA variations associated with the gene regulation.\n\nThe discovery of single-nucleotide polymorphisms (SNPs) including insertion/deletions (indels), with high-throughput data is useful to study genetic variation, comparative genomics, linkage map, and genomic selection for breeding value with DNA variation.\n\nMany geneticists for biological and genome studies of microbial, plant, animal, and human genomes have effectively used NGS whole-genome resequencing data to use in variable research fields, such as bacterial evolution [19], genomewide analysis of mutagenesis of Escherichia coli strains [20], comparative genomics of Streptococcus suis of swine pathogen [21], genomic variation effects on phenotype and gene regulation in mouse [22], evolution of plant [23], and comparison of genetic variations on the targeted enrichment [24].\n\nThe platforms of resequencing projects have used Illumina/Solexa of short read lengths to align with the reference sequence to discover DNA variations between compared related species\&#39; sequences.\n\nBecause of rare occurrence of SNPs in most species, it is important to identify high-accuracy data to discover DNA variations according to coverage depth using MAQ (http://maq.sourceforge.net/maq-man.shtml) [25] and CLC software (http://www.clcbio.com).\n\nThe public protocol of covering depth to discover SNPs and indels on the heterogeneous genome requires at least 30× of the reference genome, while about 10× depth of coverage is enough for DNA variation study of homogeneous genomes.\n\nOf course, high coverage of depth provides high-quality data in SNP detection on the reference mapping (Fig.3).\n\nHowever, short read lengths of 35 bp or 100 bp show enough to map on the reference sequence using the MAQ software and CLC software in the genome, including short repeated block regions.\n\nBut, geneticists still require long-read sequencing data to distinguish repeated block regions, like paralogous regions derived from gene duplication.\n\nMAQ software provides a consensus sequence of the genotype sequenced of short read lengths with aligned raw reads to the reference sequence.\n\nCLC software checks accuracy by counting reads of DNA variations of each position.\n\nRecently, a novel application of pattern recognition for accurate DNA variations was discovered in the complexity of the genomic region using high-throughput data in a Caucasian population [26].\n\nThey used three independent datasets with Sanger sequencing and Affymetrix and Illumina microarrays to validate SNPs and indels of a clinical target region, FKBP5.\n\nTherefore, it is necessary for multiplatform systems to validate DNA variations in the specific complexity of the genome region.\n\nTitle: Expression profiling\n\nGene expression profiling is a measurement of the regulation of a transcriptome from the whole genome in the field of molecular biology.\n\nA conventional method to measure the relative activity of target genes is DNA microarray technology, which estimates expressed genes with the signals of hybridization of target genes (cDNA from mRNA) on the synthesized oligonucleotides [27].\n\nThe technology is still used for functional genomics in the wide era, including medicine, clinic, plant, and agricultural biotechnology [28-30].\n\nIn addition, microarray technology is also used in the comparative study of proteomics and expression, measuring the level of extracellular matrix protein [30].\n\nSince NGS technology was developed in 2005, the transcriptome of novel whole genomes could be identified with massive parallel mRNA sequencing using Roche/454 and Illumina/Solexa [31-36].\n\nThe Roche/454 system is more useful for gaining novel gene discovery of novel species\&#39; genomes for long read sequencing [37, 38].\n\nOtherwise, Illumina/Solexa is being used to profile the expression of known genes with mapping short read sequences to the known reference genes [39, 40].\n\nIn that case, rare expressed genes and novel genes could be identified with high-throughput expressed sequence tag sequences using Illumina/Solexa.\n\nAlso, it is useful to find significant tissue-specific expression biases with comparison of transcript data [22].\n\nNow, the hybrid mRNA sequence from Rohce/454 and Illumina/Solexa is more powerful for finding novel genes through de novo assembly in any whole-genome species.\n\nThe hybrid sequence data of 20× and 50× coverage of the estimated transcriptome sequence from Roche/454 and Illumina/Solexa, respectively, is effective in creating novel expressed reference sequences, while short-read Illumina/Solexa data are cost-efficient on expression quantification information for comparing exposed samples and natural phenotype samples through mapping to the reference genes (Fig.4).\n\nOnly and average 30× coverage of transcriptome depth of short-read sequences of Illumina/Solexa is enough to check expression quantification, compared to reference expressed sequence tag sequences.\n\nThe expressed information could be different, depending on the software using CAP3, MIRA, Newbler, SeqMan, and CLC.\n\nTherefore, the results should be compared according to variable program options to define robust expression profiling [41].\n\nTo date, a powerful tool of ChIP-on-chip is used for understanding gene transcription regulation.\n\nThus, two-channel microarray technology of a combination of chromatin immunoprecipitation could be used for genomewide mapping of binding sites of DNA-interacting proteins [29].\n\nIn any NGS application, the transcriptome expression information would be more useful than complete genome information research with the lowest sequencing budget for biologists to better understand gene regulation of related genetic phenotypes with the in silico method.\n\nOf in silico methods, conserved miRNA and novel miRNA discovery is available on the massive miRNAnome data in any species.\n\nSpecially, the target genes of miRNA discovered could be robust information to approach genome biology studies.\n\nTranscriptome assembly is smaller than genome assembly and thus should be more computationally tractable but is often harder, as individual contigs can often have highly variable read coverages.\n\nComparing single assemblers, Newbler 2.5 performed the best on our trial dataset, but other assemblers were closely comparable.\n\nCombining different optimal assemblies from different programs, however, gives a more credible final product, and this strategy is recommended [41].\n\nNGS technology provides a cost-effective way of sequencing for novel whole-genome sequencing, resequencing, and expression profiling.\n\nRohce/454 pyrosequencing is recommended to de novo assembly of whole prokaryote novel genomes, while hybrid assembly with Illumina/Solexa sequences would be optimal for whole eukaryote genomes and transcriptome studies of non-model organisms.\n\nAlso, Illumina/Solexa sequencing is useful in detecting DNA variation, mapping the short-read resequence to the reference genome and profiling expressed genes in model organisms.\n\nFurthermore, the high-throughput NGS sequencing enables us to study with an in silico method in variable research application fields of molecular genetics, including population diversity and comparative genomics, in a short time.\n\nStringency of whole-genome DNA shotgun sequencing for novel genome.\n\nWhole-genome shotgun sequencing (left of the Figure) for contig construction: Sequencing of single-end or paired-end fragment of whole-genome DNA shotgun library, which are made with the average range of 200-800-bp fragments for Roche/454 or Illumina/Solexa systems.\n\nIn general, producing a total DNA sequence amount of 15-20× and 60× coverage in depth of a genome depends on using Roche/454 or Illumina/Solexa, respectively.\n\nWhole-genome shotgun mate paired-end sequencing for scaffold construction (right of the Figure): sequencing of the mate paired-end fragments of the whole-genome DNA shotgun library, which are made with the average range of 2-40-Kb fragments for next-generation DNA sequencer.\n\nThe sequencing amount of more than 20× coverage in depth of the genome is effective for scaffold constructions.\n\nIntegrated pipeline for de novo assembly of novel genome sequencing.\n\nThe scheme is filtering data to remove low-quality and shot-read initial assemblies using variable software and compare to contigs, hybrid contigs using MIRA assembler, and contig ordering using SSPACE software to scaffold construction.\n\nView of single nucleotide polymorphism (SNP) discovery through mapping short reads from Illumina/Solexa to reference sequence on MAQ software (A) and CLC software (B).\n\n(A) Short read 35 bp per read of soybean genome shows completely mapped on the soybean reference sequence.\n\nThe MAQ software provides a consensus sequence of the genotype sequenced of short read lengths with aligned raw reads to the reference sequence.\n\n(B) CLC software is useful for counting reads with DNA variations at each position.\n\nA scheme of transcriptome expression analysis through massively parallel signature sequencing (MPSS) technology and bioinformatics: The identification of expressed genes through hybrid de novo assembly with Roche/454 and Illumina/Solexa data (left) and expressed level profiling through mapping the Illumina/Solexa sequence to the expressed sequence tag reference.\n\n&#39;,
 &#34;Title: Differential Expression of PKD2-Associated Genes in Autosomal Dominant Polycystic Kidney Disease\n\nAutosomal dominant polycystic kidney disease (ADPKD) is characterized by formation of multiple fluid-filled cysts that expand over time and destroy renal architecture.\n\nThe proteins encoded by the PKD1 and PKD2 genes, mutations in which account for nearly all cases of ADPKD, may help guard against cystogenesis.\n\nPreviously developed mouse models of PKD1 and PKD2 demonstrated an embryonic lethal phenotype and massive cyst formation in the kidney, indicating that PKD1 and PKD2 probably play important roles during normal renal tubular development.\n\nHowever, their precise role in development and the cellular mechanisms of cyst formation induced by PKD1 and PKD2 mutations are not fully understood.\n\nTo address this question, we presently created Pkd2 knockout and PKD2 transgenic mouse embryo fibroblasts.\n\nWe used a mouse oligonucleotide microarray to identify messenger RNAs whose expression was altered by the overexpression of the PKD2 or knockout of the Pkd2.\n\nThe majority of identified mutations was involved in critical biological processes, such as metabolism, transcription, cell adhesion, cell cycle, and signal transduction.\n\nHerein, we confirmed differential expressions of several genes including aquaporin-1, according to different PKD2 expression levels in ADPKD mouse models, through microarray analysis.\n\nThese data may be helpful in PKD2-related mechanisms of ADPKD pathogenesis.\n\nAutosomal dominant polycystic kidney disease (ADPKD) is a common monogenetic disorder, affecting as many as 1 in 500 people in the general population.\n\nThe disease is characterized by the formation of multiple fluid-filled renal cysts that expand over time and destroy the architecture of the kidney.\n\nFive percent of all cases of chronic renal failure are due to ADPKD, and approximately 50% of ADPKD patients will develop end-stage renal disease by the time they are 60 years of age [1].\n\nIn the early stages of ADPKD, numerous cysts begin to enlarge from many segments of the kidney.\n\nLater, the enlarged regions disassociate from the original nephron to form the actual cysts, which continue to enlarge due to proliferation of epithelial cells and fluid secretion into the cyst lumen.\n\nProgressive renal cyst formation and enlargement result in a loss of renal function and hypertension and culminate in renal failure.\n\nIt is generally believed that cysts enlarge by means of abnormal cell growth, forming what is essentially a fluid-filled tumor that fills by transepithelial fluid secretion.\n\nCyst enlargement may then lead to extracellular matrix remodeling, interstitial fibrosis, a general inflammatory response, and disruption of the normal renal parenchyma, which then interfere with glomerular filtration and vascular blood flow, giving rise to cell death by apoptosis and ultimately renal failure [2].\n\nMany studies have indicated that most cases of ADPKD are accounted for by mutations in the PKD1 and PKD2 genes, encoding the transmembrane protein polycystin-1 (PC1) and polycystin-2 (PC2), respectively.\n\nMice with targeted mutations in the PKD1 or PKD2 genes develop cystic kidneys during embryogenesis, and ADPKD in humans is associated with mutations in the PKD1 or PKD2 genes [3, 4].\n\nIn addition, the product of the PKD1 gene, PC1, has been implicated in a variety of pathways tied to proliferation, including G-protein signaling and the Wnt, activator protein 1 (AP-1), and Janus kinase-signal transducers and activators of transcription cascades [5, 6].\n\nMoreover, depletion of PC1 has been shown to increase cell growth, whereas its overexpression slows cell growth, indicating that PC1 may negatively regulate cell proliferation [7, 8].\n\nPC2, the protein product of PKD2, has also been implicated in cell cycle regulation via its calcium channel activity and stimulation of AP-1 [9, 10]; however, there has been little direct evidence tying PC2 to this process.\n\nHere, we created a number of related cell lines that varied in their expression of PC2.\n\nWe describe studies designed to identify target genes under the control of the PKD2 using Pkd2 knockout (KO) and PKD2 transgenic (TG) mouse embryo fibroblasts (MEFs).\n\nWe used a mouse 30 K whole gene oligonucleotide microarray to identify messenger RNAs whose expression was altered by the overexpression of the PKD2 or KO of the Pkd2 in MEF cells.\n\nTitle: Establishment of MEF derived from wild-type and PKD2 mutation embryos\n\nMEF were acquired from PKD2 TG embryos or Pkd2 KO embryos during development (13.5 days) [11].\n\nHeads and limes were removed from embryos.\n\nThe remaining embryonic tissues were minced and dispersed in 0.05% trypsin prior to incubation at 37℃ for 15 min.\n\nCells were plated in Dulbecco&#39;s Modified Eagle&#39;s Medium of Defined Minimal Essential medium (Welgene Biotech, Taipei, Taiwan) supplemented with 10% fetal bovine serum (Welgene Biotech) and were cultured at 37℃ in an atmosphere of 5% CO2 until confluent growth was achieved.\n\nMEF cells were frozen as stocks at the second passage and were used for the subsequent studies at the third passage.\n\nTitle: Microrray hybridization and data analysis\n\nTotal RNA was prepared using a commercial kit (Qiagen, Valancia, CA, USA) according to the manufacturer&#39;s instructions.\n\nGene expression profiles were obtained using the CodeLink Uniset Mouse Expression Bioarray (Amersham, Buckinghamshire, UK).\n\nThe 30,000 gene probes contained in the bioarray allows the detection of differences in gene expression as low as 1.3-fold with 95% confidence.\n\nTen micrograms of total RNA was amplified, and labeled cDNA was produced.\n\nBiotin-labeled cDNA was hybridized to the array overnight in a shaking incubator at 37℃, and excess target sequences were washed away using a series of saline sodium citrate washes.\n\nThe array was stained by treatment with streptavidin-Alexaflour 647 (Molecular Probes, Eugene, OR, USA), the excess was washed away, and the array was scanned at an excitation wavelength of 632 nm using a GenePix scanner (Molecular Devices, Sunnyvale, CA, USA).\n\nThe resulting image was quantified, and the intensity of each spot was divided by the median spot intensity to provide a scaled and comparable number across multiple arrays.\n\nTitle: DNA microarray scanning and analysis\n\nMicroarrays were scanned using an Arraywox scanner (Applied Precision, Seattle, WA, USA), analyzed using ImaGene version 5.1 software (Biodiscovery, Segundo, CA, USA), and normalized using Genesight version 3.2 software (Biodiscovery).\n\nNormalization was performed by subtracting the means of all genes.\n\nThe data normalized by Genesight were compared using an M/A plot.\n\nGenes differentially expressed were identified by intensity differences, after subtracting the background intensity.\n\nGenes showing expression changes of at least 2-fold were selected, and these selected genes were clustered by the hierarchical method.\n\nTitle: Clustering algorithm\n\nThe normalized log ratio corresponding to each time point was exported to the software for clustering algorithm.\n\nAcuity version 3.1 (Molecular Devices) was used for the &#39;gene shaving&#39; algorithm.\n\nTo estimate the number of clusters in a dataset, the &#39;Gap Statistics&#39; attribute of the Acuity software package was used.\n\nThe cluster number, estimated by gap statistics, was used for an input parameter in gene shaving.\n\nAfter gene shaving, a single linkage was used for hierarchical clustering.\n\nTitle: Semi-quantitative and real-time RT-PCR\n\nTotal RNA was prepared using a commercial kit (Qiagen) according to the manufacturer&#39;s instructions.\n\nSingle-strand cDNA was synthesized by incubating 5 µg total RNA with 200 units AMV, 100 nM oligo(dT)12-18, 1 mM dNTP mixture, and 40 units RNase inhibitor at 42℃ for 1 h in a final volume of 25 µL.\n\nThe reaction was terminated by incubation at 70℃ for 15 min.\n\nThe initial amount of mRNA and reaction conditions were optimized to obtain linearity for mouse 18s rRNA.\n\nFor semi-quantitative reverse transcription (RT)-PCR, the used primers were: human PKD2, forward 5&#39;-CGTGCCCCAGCCCAGTC-3&#39; and reverse 5&#39;-TTCCAGTACAGCCCATCCAATAAG-3&#39;; mouse Pkd2, forward 5&#39;-TGCGAGGGCTGCGAGGTC-3&#39; and reverse 5&#39;-TGTCAGCTTGCGTGTGGTTGC-3&#39;; mouse 18s rRNA, forward 5&#39;-GTAACCCGTTGCACCCCATT-3&#39; and reverse 5&#39;-CCATCCAATCGGTAGTAGCG-3&#39;.\n\nRT-PCR cycling conditions were as follows: 10 min at 95℃, 25 cycles of 50 s at 94℃, 50 s at 57℃, 50 s at 72℃, and 5 min at 72℃.\n\nThe amplified products were separated on a 1% agarose gel.\n\nFor real-time PCR, the used primers were: mouse β-actin, (forward 5&#39;-GACGATGCTCCCCGGGCTGTATTC-3&#39; and reverse 5&#39;-TCTCTTGCTCTGGGCCTCGTCACC-3&#39;) used as a positive control; mouse peroxisome proliferator-activated receptor γ (PPAR-γ), forward 5&#39;-TCTTAACTGCCGGATCCACAAAAA-3&#39; and reverse 5&#39;-ATCTCCGCCAACAGCTTCTCCTTC-3&#39;; mouse protein kinase α (PKCα), forward 5&#39;-GGGCAGCCTCCGTTTGATGGT-3&#39; and reverse 5&#39;-CGCTTGGCAGGGTGTTTGGTC-3&#39;; mouse Wnt4, forward 5&#39;-GCCATCGAGGAGTGCCAATACC-3&#39; and reverse 5&#39;-GGCCACACCTGCTGAAGAGATG-3&#39;; mouse integrin α4 (ITGα4), forward 5&#39;-GTAGCCCCAGTGGAGAGCCTTGTG-3&#39; and reverse 5&#39;-ATGCCAGTGGGGAGTTTGTTATCG-3&#39;; mouse IL6ST, forward 5&#39;-TGAATCGGACCCACTTGAGAGG-3&#39; and reverse 5&#39;-CAGGAGCGGCTTGTTTGAGGTA-3&#39;; mouse aquaporin 1 (AQP1), forward 5&#39;-GGAGGCGCCGAGACTTAGGT-3&#39; and reverse 5&#39;-GCGGGTGAGCACAGCAGAGC-3&#39;; mouse transforming growth factor-β2 (TGF-β2), forward 5&#39;-TCATCCCGAATAAAAGCGAAGAGC-3&#39; and reverse 5&#39;-AGGGCAACAACATTAGCAGGAGAT-3&#39;.\n\nReal-time RT-PCR was performed using the real-time SensiMixPlus SYBR kit as described by the manufacturer&#39;s instructions (Quantance, London, UK).\n\nTitle: Western blot analysis\n\nProteins were extracted using radio-immunoprecipitation assay buffer in MEF cells.\n\nProteins were separated by 12% sodium dodecyl sulfate-polyacryamide gel electrophoresis and were transferred to a polyvinylidene fluoride membrane (Millipore, Bellerica, MA, USA).\n\nMembranes were blocked with 5% non-fat dry milk and were incubated with various antibodies.\n\nAntibodies to PC2 (anti-human and anti-mouse) and AQP1 were obtained from Santa Cruz Biotechnology (Santa Cruz, CA, USA).\n\nAntibodies to β-actin were obtained from Sigma-Aldrich (St. Louis, MO, USA).\n\nAnti-Wnt4 was obtained from R&amp;D Systems (Minneapolis, MN, USA).\n\nEach membrane was washed with phosphate-buffered saline-Tween, and immunocomplexes were detected by enhanced chemiluminescence (Amersham).\n\nTitle: Confirmation of PKD2 expression in Pkd2 KO and PKD2 TG MEFs\n\nMouse Pkd2 mRNA was revealed in KO wild-type (KOWT) MEF cells but not in KO MEF cells.\n\nAlso, Pkd2 mRNA was expressed in TG wild-type (TGWT) MEF cells and PKD2 TG MEF cells.\n\nHuman PKD2 mRNA was only expressed in PKD2 TG MEF cells obtained from PKD2 TG embryos following injection of the human PKD2 transgene (Fig.1A).\n\nConsistently, the expression level of polycystin-2 was lower in KO MEFs than KOWT MEFs as well as higher in TG MEFs than TGWT MEFs (Fig.1B).\n\nTitle: Microarray gene expression analysis\n\nWe used a mouse 30 K whole gene oligonucleotide microarray to identify mRNAs whose expression was altered by the Pkd2 KO and PKD2 TG MEF cells.\n\nThe normalized log ratios corresponding to each time point were exported to the software for clustering algorithm.\n\nWe used Axon&#39;s Acuity 3.1 (Axon Instruments, Union City, CA, USA) for using the &#39;gene shaving&#39; algorithm.\n\nTo estimate the number of clusters in a dataset, we used &#39;Gap Statistics&#39; in Acuity 3.1.\n\nThe cluster number estimated by gap statistics was used for an input parameter in gene shaving.\n\nAfter gene shaving, we used a single linkage hierarchical clustering.\n\nThe single linkage hierarchical clustering method divides 8 &#39;gene shaving&#39; clusters (Fig.2).\n\nThe majority of genes whose expression was appreciably altered encoded proteins involved in critical biological processes, such as metabolism, transcription, cell adhesion, cell cycle, and signal transcription.\n\nForty-five genes whose expression was changed by 2-fold or greater were identified (Table 1).\n\nTitle: Verification of candidate genes by quantitative RT-PCR and immunoblot analysis\n\nTo verify the results obtained by cDNA microarray analysis, expression patterns for selected genes were confirmed at the RNA and protein levels.\n\nWe selected seven genes among the 45 genes (Fig.3A).\n\nAs result of realtime RT-PCR, the expression patterns of PKCα, ITGα4, and AQP1 were identical to microarray analysis in Pkd2 KO MEF cells (Fig.3B).\n\nIn contrast, 6 genes, except for TGF-β2, were identically expressed in PKD2 TG MEF cells (Fig.3C).\n\nEspecially, the AQP1 gene may be significantly regulated by differential Pkd2 gene expression level.\n\nFurthermore, to confirm the AQP1 protein expression level, western blot analysis was performed using MEF cells as well as kidney tissues obtained from Pkd2 KO (or heterozygote) and PKD2 TG mice.\n\nAs result, AQP1 protein level was consistent with mRNA levels in both MEFs and tissues (Fig.4).\n\nAlthough remarkable progress toward understanding the genetics and pathophysiology of ADPKD has been made to date, it is still unclear how mutations in disease-causing genes trigger cystogenesis and what important role other molecules play in the cystic phenotype [12].\n\nOur aim here was to identify genes associated with the cystic phenotype using Pkd2 KO and PKD2 TG MEF cells.\n\nIn a previous study, overexpression of human PKD2 led to anomalies in tubular function, due to abnormalities in tubule morphogenesis [13].\n\nSimilar findings have been made for the PKD1 gene.\n\nFor this gene, gain/loss of function and haploinsufficiency leads to cystogenesis, with the severity of the phenotype being related to the level of imbalance.\n\nMore severe phenotypes are seen for gain and loss of function than for haploinsufficiency [4].\n\nIn the present study, the association between cystogenesis and the loss or gain of PKD2 supports the idea that PC2 functions through Pkd2 mutant MEF cells.\n\nPresently, we found Wnt4 to be up-regulated in Pkd2 KO MEF cells and down-regulated in PKD2 TG MEF cells.\n\nWnt4, a member of the Wnt family of lipid-modified secreted glycoproteins, plays central roles in the initial stages of the tubulogenic program.\n\nWnt4 is expressed in the pretubular aggregate, as one of the first molecular responses to inductive signaling mediated by ureteric epithelium [14].\n\nAQP1 was observed to be up-regulated in both Pkd2 KO and PKD2 TG MEF cells.\n\nAQP1 is a channel-forming integral membrane protein found in erythrocytes and water-transporting tissues, including the kidney, where it is localized to the proximal tubules and thin descending limbs of Henle&#39;s loop.\n\nThe selective expression pattern of AQP1 in human nephrogenesis suggests that proximal tubules and thin descending limbs become permeable to water early in development.\n\nA predominantly apical expression of AQP1 is found during a critical period of the nephrogenesis, during formation of the proximal tubule, as well as in the early stage of ADPKD; in normal tubules, distribution is both apical and basolateral [15].\n\nAnother interesting defect in urine-concentrating ability was described in the Aqp1 knockout mouse.\n\nAlthough these mice were grossly normal in terms of survival and appearance, they were vulnerable to water deprivation and became severely dehydrated.\n\nThese results suggest that AQP1 is required for the formation of concentrated urine [16].\n\nWe have found PKC-α and Bcl-2 associated transcription factor to be down-regulated in Pkd2 KO MEF cells and up-regulated in PKD2 TG MEF cells.\n\nRecent observations in animal models of PKD have implicated apoptosis in its pathogenesis.\n\nPKC- and Erk-dependent pathways are critical components of the cell survival by suppressing the apoptosis [17].\n\nWe also have demonstrated the role for PKD2 in cellular adhesion processes.\n\nIn this study, ITGα4 was up-regulated in Pkd2 KO MEF cells and down-regulated in PKD2 TG MEF cells.\n\nIt suggests that adhesion molecule such as ITGα4 may be associated with PKD2 mutation-related pathogenesis.\n\nIn conclusion, by using the cDNA microarray technique, we have found several candidate genes that may be involved in cystogenesis: PKCα, ITGα4 and AQP1.\n\nThese genes were affected by PKD2 expression and may be related to cyst formation of kidney together with the PKD2 gene.\n\nEstablishment of Pkd2 KO and PKD2 TG MEFs.\n\n(A) Semi-quantitative reverse transcription-PCR analysis of Pkd2 gene in Pkd2 KO and PKD2 TG MEF cells.\n\nHuman- or mouse-specific PKD2 primer sets were used.\n\n18s rRNA was used as a loading control.\n\n(B) Western blot analysis of polycystin 2 (PC2) expression in Pkd2 KO and PKD2 TG MEF cells.\n\nMEFs, mouse embryo fibroblasts; KO, knockout (MEF); TG, transgenic (MEF); KOWT, knockout wild-type (MEF); TGWT, transgenic wild-type (MEF).\n\nClustering of gene expression patterns in Pkd2 mutant MEF cell lines.\n\nWe performed clustering analysis microarray data between KO set ratio data and TG set ratio data using the graphical user interface (GUI) system.\n\n(A) Gene expression profiles of 2,306 probe sets distributed in a dendrogram.\n\n(B) Line plots of each cluster.\n\nKO set, ratio between KO MEF and KOWT MEF; TG set, ratio between TG MEF and TGWT MEF.\n\nMEFs, mouse embryo fibroblasts; KO, knockout (MEF); TG, transgenic (MEF); KOWT, knockout wild-type (MEF); TGWT, transgenic wild-type (MEF).\n\nExpression verification of candidate genes at the mRNA level.\n\n(A) Microarray data for selective genes including PPARγ, PKCα, Wnt4, ITGα4, IL6ST, AQP1, and TGF-β2.\n\n(B, C) Real-time reverse transcription-PCR analysis of gene expression for selective genes in Pkd2 KO and PKD2 TG MEF cells.\n\nβ-actin primer was used as the internal control.\n\nThe experiment was performed in triplicate.\n\nPPAR-γ, peroxisome proliferator-activated receptor γ; PKCα, protein kinase α; ITGα4, integrin α4; AQP1, aquaporin 1; TGF-β2, transforming growth factor-β2; KO, knockout; TG, transgenic; MEF, mouse embryo fibroblasts; KOWT, knockout wild-type (MEF); TGWT, transgenic wild-type (MEF).\n\nExpression verification of AQP1 at the protein level.\n\n(A) Western blot analysis of AQP1 in MEFs and kidney tissues derived from Pkd2 KO (or heterozygote) and PKD2 TG mice.\n\nPKD2 TG and TGWT mice: 18 mo; Pkd2 heterozygote and KOWT mice: 12 mo.\n\nAnti-β-actin was used as the internal control.\n\n(B) For quantification of western blot, Multigauge software (Fuji Film) was used.\n\nAQP1, aquaporin 1; MEF, mouse embryo fibroblasts; KO, knockout; TG, transgenic; KOWT, knockout wild-type (MEF); TGWT, transgenic wild-type (MEF).\n\nTitle: Genes deregualted by differential Pkd2 expression\n\n&#34;,
 &#34;Title: Possibility of the Use of Public Microarray Database for Identifying Significant Genes Associated with Oral Squamous Cell Carcinoma\n\nThere are lots of studies attempting to identify the expression changes in oral squamous cell carcinoma.\n\nMost studies include insufficient samples to apply statistical methods for detecting significant gene sets.\n\nThis study combined two small microarray datasets from a public database and identified significant genes associated with the progress of oral squamous cell carcinoma.\n\nThere were different expression scales between the two datasets, even though these datasets were generated under the same platforms - Affymetrix U133A gene chips.\n\nWe discretized gene expressions of the two datasets by adjusting the differences between the datasets for detecting the more reliable information.\n\nFrom the combination of the two datasets, we detected 51 significant genes that were upregulated in oral squamous cell carcinoma.\n\nMost of them were published in previous studies as cancer-related genes.\n\nFrom these selected genes, significant genetic pathways associated with expression changes were identified.\n\nBy combining several datasets from the public database, sufficient samples can be obtained for detecting reliable information.\n\nMost of the selected genes were known as cancer-related genes, including oral squamous cell carcinoma.\n\nSeveral unknown genes can be biologically evaluated in further studies.\n\nDespite recent advances in surgical, radiation, and chemotherapeutic treatment protocols, the prognosis of oral squamous cell carcinoma (OSCC) remains mournful, with an approximate 50% 5-year mortality rate from disease or associated complications [1].\n\nTherefore, the identification of biological markers is essential to make progress in detecting malignancy at an early stage and developing novel therapies [2].\n\nMicroarray datasets that are created for the same research purposes in different laboratories have accumulated rapidly.\n\nThe results from different datasets are often inconsistent due to the utilization of different platforms, sample preparations, or various technical variations.\n\nIf we could combine such datasets by adjusting for systematic biases that exist among different datasets derived from different experimental conditions, the power of statistical tests would be improved by the increase in sample size [3].\n\nIn OSCC, although lots of microarray-based studies have been conducted to provide insights into gene expression changes, most of these studies have contained insufficient samples for detecting reliable information using statistical analysis [4, 5].\n\nTherefore, this study attempted to combine several datasets in the public database for detecting significant genes.\n\nWe used two small microarray datasets of OSCC for this study, which were based on the same platform but had different expression scales.\n\nThese two datasets were combined after discretization, because a previous study showed that classification could be improved using combined datasets after discretization [3].\n\nAfter combining datasets, we used chi-square test for identifying the significant genes.\n\nChi-square test has been used commonly to detect differentially expressed genes after discretization of expression intensities in the microarray experiment.\n\nIn this study, gene expression ratios of two datasets were transformed with their ranks for each dataset.\n\nNext, the transformed datasets were combined, and a nonparametric statistical method was applied to the combined dataset to detect informative genes.\n\nFinally, we showed that most of the selected genes were known to be involved in various cancers, including OSCC.\n\nTitle: Dataset\n\nTwo microarray datasets were used for this study.\n\nWe acquired these datasets from a public database (Gene Expression Omnibus, GEO).\n\nOne was the expression dataset of 16 tumors and 4 normal tissues from 16 patients, using Affymetrix U133A gene chips (Affymetrix, Santa Clara, CA, USA).\n\nThe other microarray dataset consisted of expression profiles of 22 tumors and 5 normal tissues.\n\nThese two datasets were experimented on under the same platform, Affymetrix U133A.\n\nThe datasets are summarized in Table 1.\n\nTitle: Process for combining datasets\n\nTitle: For combining datasets, gene expression ratios are rearranged in order of expression ratios by each gene in each dataset, and the ranks are matched with the corresponding experimental group.\n\nIf the experimental groups are homogenous, the ranks within the same experimental group would be neighboring.\n\nThe process of discretization of gene expressions is summarized in the following steps [3]:\n\nRank the gene expression ratios within a gene for each dataset.\n\nList in order of the ranks, and assign the order of gene expressions to the corresponding experimental groups.\n\nSummarize the result of (2) in the form of a contingency table for each gene.\n\nCombine the contingency tables that have been summarized for each dataset.\n\nWhen there are three datasets to be combined, the datasets can be added as a single entry, as shown in Table 2, after the transformation of each dataset by rank.\n\nTitle: Identification of significant genes from a combined dataset\n\nTitle: After the summarization of gene expression ratios in the form of a contingency table for each gene, as shown in Table 3, a nonparametric statistical method was applied to the datasets for independence testing between gene expression patterns and experimental groups.\n\nThe test statistics are calculated as follows for each gene:\n\nWhen the sample size is small - generally Ê(nij) less than 5 - Fisher&#39;s exact test is recommended rather than chi-square test.\n\nThe significant genes can be selected by an independence test between the phenotypes and gene expressions using this type of summarized dataset.\n\nci and ri represent the marginal sums of the ith column and row, respectively.\n\nnij is the number of experiments belonging to Ei and Pj, and n represents the total number of experiments.\n\nThe clinical information and expression levels of two datasets are summarized in Table 4 and Fig.1.\n\nSubgroup and sex were similarly distributed in the two datasets.\n\nThe distributions of other factors were not included.The scale of expression levels in the two datasets was different; the expression values of Data 2004 ranged from 0.01 to 740, and those of Data 2005 were from 0.1 to 19,773.\n\nThe expression patterns of the two datasets can be explored in Fig.1.Lots of outliers are shown in Fig.1A in the two datasets containing whole gene sets.\n\nHowever, in subsets of significant genes, the expression ranges got narrow, and the outliers were decreased (Fig.1B).\n\nThe expressions of tumor tissues in Data 2004 were upregulated and varied compared with normal tissues.\n\nIf there was no outlier with a maximum value in the 14th tumor tissue in Data 2004, the expressions of the two different groups would be clearly distinguished.\n\nAny clear differences in expression were not shown between the two groups in Data 2005.\n\nTitle: Upregulated 51 genes in oral squamous cell carcinoma\n\nTo identify differently expressed genes between normal and tumor tissues, we performed chi-square test using a combined microarray dataset.\n\nFifty-one significant genes were selected from a combined dataset with p-value less than 0.005, which were upregulated in OSCC tissues.\n\nThe significance level can be controlled, and more genes can be selected with a lower significance level.\n\nThese selected genes are summarized in Table 5.\n\nMany genes among the selected genes were known as cancer-related genes.\n\nSTAT1 [6], SKP2 [7], IFI16 [8], RHEB [9], FIF44 [10], SOD2 [11, 12], and GREM1 [11] are related to OSCC.\n\nTable 6 [13-56] summarizes the previous studies that have published the relations of selected genes with cancer.\n\nTitle: Expression pattern of the identified genes\n\nTo investigate whether the different experimental groups could be classified with significant genes, an unsupervised hierarchical clustering method was applied to the significant gene set (Fig.2).\n\nThe normal group consisted of 4 tissues and showed significantly lower expression levels when compared with the tumor group.\n\nIn Fig.2, we investigated the classification availability of the identified genes in Data 2004, not in a combined dataset, because the two datasets have different expression scales.\n\nTitle: Network analysis\n\nBased on all identified genes, new and expanded pathway maps and connections and specific gene-gene interactions were inferred, functionally analyzed, and used to build on the existing pathway using the Ingenuity Pathway Analysis (IPA) knowledge base [57].\n\nTo generate networks in this work, the knowledge base was queried for interactions between the identified genes and all other genes stored in the database.\n\nFour networks were found to be significant in OSCC.\n\nThe network with the highest score (Network 1, score = 36) was generated, with 17 identified genes (Table 7, Fig.3).\n\nIn the network diagram, STAT1 and SOD2 neighbored with NMI and AURKA, respectively.\n\nThe expression levels of STAT1 and SOD2 could be expected to be related with those of NMI and SOD2.\n\nActually, the expressions of STAT1 and SOD2 were strongly positively correlated with NMI (r = 0.95) and AURKA (r = 0.87), respectively.\n\nOSCC is associated with substantial mortality and morbidity [58].\n\nTo identify potential biomarkers for early detection of invasive OSCC, microarray experiments have been conducted, and these kinds of microarray datasets have accumulated rapidly in the public database.\n\nHowever, there are many datasets that include insufficient sample sizes for detecting significant genes by statistical analysis.\n\nTherefore, this study attempted to combine several microarray datasets from a public database to identify significant candidates as biomarkers.\n\nIn a microarray data analysis, the information from different datasets obtained under different experimental conditions may be inconsistent even though they are performed with the same research objectives.\n\nMoreover, even when the datasets are generated by the same platform, the data agreement may be affected by technical variations between laboratories.\n\nIn such cases, it could be necessary to use a combined dataset after adjusting for the differences between such datasets for detecting the more reliable information.\n\nCombining datasets is especially useful in OSCC microarray datasets, because there are many datasets with insufficient sample sizes for analysis [4, 5, 59, 60].\n\nFor identifying significant genes classifying tumor and normal groups, we achieved two microarray datasets from a public database, GEO.\n\nThey included 20 and 27 samples, and each sample size was unbalanced between the different groups.\n\nBy combining these two datasets, the sample size was increased, and we had a sufficient sample size for statistical analysis, even though it was still unbalanced.\n\nWhen these datasets were combined, we used the rank of gene expression, because the scale of gene expression was different.\n\nIn this study, we identified 51 significant genes from a combined dataset, and this number could be increased or decreased by the significance level (we used 0.005).\n\nThe selected 51 genes were upregulated in tumor tissues.\n\nMany of the selected genes were proven to be cancer-related genes by previous studies.\n\nSOD2 is associated with lymph node metastasis in OSCC and may provide predictive values for the diagnosis of metastasis [10].\n\nMetastasis is a critical event in OSCC progression.\n\nAn SOD2 variant has also been associated with increased breast cancer and ovarian cancer risk in previous studies [47, 61].\n\nTopBP1 included eight BRCT domains (originally identified in BRCA1), and it was proposed as a breast cancer susceptibility gene [18, 62].\n\nBy semiquantitative reverse transcription PCR analysis, RHEB was shown to be upregulated in OSCC [9].\n\nIn salivary cancer, survival probability rates dropped when Skp2 was overexpressed [7].\n\nOverexpression of Skp2 is associated with the reduction of p27 (KIP1) expression and may have a role in the progression of OSCC [25].\n\nThe expression of RCN2 was linearly related to the tumor mass increase, and its expression was increased in breast cancer [16].\n\nPTPRK was proven as a candidate gene of colorectal cancer [19], and it is a functional tumor suppressor in Hodgkin lymphoma cells [20].\n\nDMTF1 was shown to be amplified in adenocarcinoma of the gastroesophageal junction, residing at 7q21 by aCGH experiments [21].\n\nFEZ1 was involved in ovarian carcinogenesis, and its reduction or loss could be an aid to the clinical management of patients affected by ovarian carcinoma [22].\n\nIt is also a known tumor suppressor gene in breast cancer and gastric cancer [23, 63].\n\nOther ovarian cancer-related genes were NMI [27, 28] and FANCI [44]; breast cancer-related genes were COX11 [42], MELK [33], and FANCI [44] among the selected genes.\n\nMELK was known to be associated with shorter survival in glioblastoma [34].\n\nTTK was associated with progression and metastasis of advanced cervical cancers after radiotherapy [29, 30].\n\nIt might also be a relevant candidate as a new target in cancer therapy, since it plays relevant roles in mitotic progression and the spindle checkpoint [31, 32].\n\nAurora kinase A (AURKA) was associated with skin tumors [36] and colorectal cancer [37, 38].\n\nIn previous studies, OSCC-related genes among the selected genes were STAT1 [14], SKP2 [7, 25], IFI16 [8], RHEB [9], IFI44 [64], SOD2 [10-12], and GREM1 [11].\n\nThe gene set, which has not been proven as OSCC-related genes until now, could be expected to be possibly proven as OSCC-related genes by biological evaluation.\n\nIn this study, we identified significant genes related with OSCC from two microarray datasets in a public database.\n\nFor this, we transformed microarray datasets using ranks of gene expressions with different expression scales, even though they were constructed under the same experimental conditions.\n\nThis method could be useful when using multiple datasets that are created for the same research purpose, By combining these accumulated datasets, we can detect more reliable information due to the increased sample size.\n\nIt saves time and money and avoids repeating experiments.\n\nComparison of expression levels of two datasets.\n\n(A) Whole gene set.\n\n(B) Selected gene set.\n\nExpression patterns of the selected 51 genes.\n\nThese genes were upregulated in oral squamous cell carcinoma tissues, and normal and tumor groups were clearly classified with these genes.\n\nNetwork with the highest score (Network 1).\n\nFunctional relationships between genes based on known interactions in Ingenuity Pathway Analysis (IPA) knowledge are described.\n\nTitle: Summary of two microarray datasets from GEO and the combined dataset\n\nTitle: Combination of contingency tables for three datasets (tij = aij + bij + cij)\n\nTitle: Summary of discretized data using ranks of gene expressions\n\nTitle: Summary of two microarray datasets\n\nTitle: Summary of selected 51 upregulated genes\n\nTitle: Association of the selected genes and cancer\n\nTitle: Four networks generated by upregulated genes in OSCC\n\n&#34;,
 &#34;Title: CaGe: A Web-Based Cancer Gene Annotation System for Cancer Genomics\n\nHigh-throughput genomic technologies (HGTs), including next-generation DNA sequencing (NGS), microarray, and serial analysis of gene expression (SAGE), have become effective experimental tools for cancer genomics to identify cancer-associated somatic genomic alterations and genes.\n\nThe main hurdle in cancer genomics is to identify the real causative mutations or genes out of many candidates from an HGT-based cancer genomic analysis.\n\nOne useful approach is to refer to known cancer genes and associated information.\n\nThe list of known cancer genes can be used to determine candidates of cancer driver mutations, while cancer gene-related information, including gene expression, protein-protein interaction, and pathways, can be useful for scoring novel candidates.\n\nSome cancer gene or mutation databases exist for this purpose, but few specialized tools exist for an automated analysis of a long gene list from an HGT-based cancer genomic analysis.\n\nThis report presents a new web-accessible bioinformatic tool, called CaGe, a cancer genome annotation system for the assessment of candidates of cancer genes from HGT-based cancer genomics.\n\nThe tool provides users with information on cancer-related genes, mutations, pathways, and associated annotations through annotation and browsing functions.\n\nWith this tool, researchers can classify their candidate genes from cancer genome studies into either previously reported or novel categories of cancer genes and gain insight into underlying carcinogenic mechanisms through a pathway analysis.\n\nWe show the usefulness of CaGe by assessing its performance in annotating somatic mutations from a published small cell lung cancer study.\n\nHigh-throughput genomic technologies (HGTs), including next-generation DNA sequencing (NGS), microarray, and serial analysis of gene expression (SAGE), have become effective tools for cancer genomics through various applications.\n\nEspecially, the applications of NGS include whole-genome, exome, and transcriptome approaches for searching for a wide range of cancer-specific genomic alterations, such as point mutations, insertions, and deletions; copy number changes; and rearrangements, leading to the development of cancer [1].\n\nOne of the hurdles in HGT-based cancer genomics is to identify causative mutations or genes out of many candidates from the analysis.\n\nA useful approach is to refer to known cancer genes and associated information [2].\n\nThe list of known cancer genes can be used to determine candidates of cancer driver mutations, while cancer gene-related information, such as gene expression, protein-protein interaction, and pathway information, can be useful for scoring novel candidates.\n\nSome useful cancer genomic databases for this purpose exist, including Cancer Gene Census (CGC) [3] and Catalogue of Somatic Mutations in Cancer (COSMIC) [4], which have been constructed and managed by the Cancer Genome Project of the Welcome Trust Sanger Institute; and Cancer Gene Index (CGI, http://ncicb.nci.nih.gov/NCICB/projects/cgdcp), The Cancer Genome Atlas (TCGA, http://cancergenome.nih.gov), and Cancer Genome Anatomy Project (CGAP, http://cgap.nci.nih.gov), which are maintained by the National Cancer Institute of the U. S. National Institute of Health and National Human Genome Research Institute.\n\nAmong them, COSMIC and CGC are the two most commonly used resources among researchers when checking reported cancer driver mutations.\n\nThe COSMIC database stores information on somatic mutations and associated information extracted from the scientific literature, while the CGC database is a catalog of cancer genes with manually screened somatic mutations extracted from the literature.\n\nThese cancer genes are annotated with information concerning chromosomal location, tumor types in which mutations are found, classes of mutations that contribute to oncogenesis, and other genetic properties.\n\nAlso, CGI is known to be created through an automated linguistic text analysis of millions of MEDLINE abstracts, with manual validation and annotation of the extracted data by expert curators.\n\nIt is a high-quality data resource consisting of genes that have been experimentally associated with human cancer diseases and/or pharmacological compounds, the evidence of these associations, and relevant annotations on the data.\n\nThus, it can also be a valuable resource to annotate mutation data from cancer genomics studies.\n\nCurrently, few specialized tools exist for an automated analysis of a long gene list from HGT-based cancer genomics.\n\nWhile previously described cancer genomic information databases can be useful, they are not effective in processing results from NGS-based cancer genomic studies.\n\nMeanwhile, there are many functional annotation systems to analyze gene lists derived from high-throughput microarray experiments, including DAVID [5], GoMiner [6], GOstat [7], Onto-express [8], and gene set enrichment analysis (GSEA) [9].\n\nDAVID is one of the most popular general purpose functional annotation systems.\n\nIt provides various effective analysis tools and is also applicable in the analysis of gene lists from cancer genomics research.\n\nHowever, DAVID is not optimized for processing results from HGT-based cancer genomics data, especially for NGS-based cancer genomics downstream data; therefore, a more specialized cancer gene annotation system is needed.\n\nHere, we introduce a web-accessible cancer genome annotation system, named CaGe, to provide users with information on cancer genes, mutations, pathways, and associated annotations, based on several cancer gene databases composed of reported cancer-causing genes and associated cancer pathways.\n\nFor a given gene list, CaGe searches cancer gene databases with converted standard gene symbols, analyzes the biological pathways, and provides various cancer gene-related annotations through intuitive web user interfaces.\n\nIt also serves additional functions, including processing various input types and formats, managing jobs for user submitted tasks, and browsing for cancer genes and pathways with various useful hyperlinks between the annotations and the external public annotation databases.\n\nWe hope CaGe will be useful in identifying cancer-causing mutations and genes in HGT-based cancer genomics.\n\nThe goal of this study was to provide researchers with a tool for classifying their candidate genes from HGT-based cancer genome studies into previously reported or novel categories of cancer genes, while providing insight into underlying carcinogenic mechanisms through a pathway analysis.\n\nTo implement the cancer gene annotation function of CaGe, we constructed reported cancer gene and cancer annotation databases from public cancer genomic databases and cancer pathway-gene databases by pathway analysis with reported cancer gene sets and canonical pathways.\n\nWe also constructed a gene ID database to allow various input formats for the input of gene lists and a gene functional annotation database to provide users with functional clues for the annotated candidate genes.\n\nThen, we developed a core retrieval program and web interfaces for the main functions, which include cancer gene annotation, cancer pathway annotation, cancer gene browsing, and cancer pathway browsing.\n\nThe workflow for the database construction and data processing in CaGe is summarized in Fig.1, and the cancer gene annotation page of the CaGe web interface is shown in Fig.2.\n\nTitle: Cancer gene and annotation data\n\nTo construct the reported cancer gene database, we used gene sets from the CGC database (released on Dec, 2010) and CGI database (downloaded on Feb, 2011).\n\nThe cancer pathways for the cancer pathway gene database construction were assigned based on statistical significance from one-tailed Fisher&#39;s exact test for overlapping genes between reported cancer gene sets and canonical pathways from public pathway databases, including KEGG (Release 57.0), BioCarta, and Reactome (downloaded on Feb, 2011).\n\nWe also created a gene ID database to convert various input identifiers into standard gene symbols with HUGO Gene Nomenclature Committee (HGNC) (downloaded on Feb, 2011) data for the standard gene symbols and with Entrez Gene (downloaded on Feb, 2011) and UniProt (Release 2011_03) data for the gene IDs, protein IDs, and functional annotations.\n\nTitle: Input types\n\nAvailable input types include a list of gene symbols, Entrez gene IDs, or UniProt accessions and output files from prediction tools, such as Sorting Intolerant From Tolerant (SIFT) [10] and PolyPhen [11], which evaluate functional effects of mutations on proteins.\n\nCaGe parses the output of those tools to extract the list of genes that have somatic mutations evaluated as functionally damaging, classify them into previously reported and novel candidate cancer genes, and conduct pathway analyses to give insights into underlying carcinogenic mechanisms.\n\nThus, CaGe offers straightforward data processing from HGT-based data without additional data conversion.\n\nTitle: Cancer gene and pathway annotation workflow\n\nAfter acquiring user input, CaGe converts various input gene IDs into standard gene symbols, finds known cancer genes and pathways, links various cancer-related annotations to matched genes, and outputs them in the form of tables or text files through the web interface (Fig.3).\n\nAnother function of CaGe is to identify over-represented cancer-related or other biological pathways from the input gene list by performing one-tailed Fisher&#39;s exact test.\n\nWhen processing more than one annotation job, users can manage their annotation tasks through the CaGe interface, and access to their results is maintained by the internal job database of CaGe until the jobs are deleted by the user or by a scheduled cleaning process.\n\nThe IP addresses of client computers are used for secured job management without a logon process.\n\nCompleted annotation jobs are listed on the job table, and the annotated results can be shown selectively by the user on the annotation result page or can be downloaded as tab-delimited text files for further analyses.\n\nThe annotation on the results page has many useful links to gene and cancer-related information in the CaGe database or external public databases.\n\nIn addition to the cancer gene and pathway annotation function, CaGe provides a function to browse cancer genes and pathways so that users can search cancer-related annotations without an input list.\n\nThe search flows of cancer genes and pathways are connected to each other by crosslinks in the cancer gene information page or pathway information page.\n\nTitle: Database and web interface\n\nBased on the reported cancer-related gene sets from CGC and CGI, a cancer gene database and a cancer pathway gene database were constructed by the described methods.\n\nIn total, 436 CGC-originated and 7181 CGI-originated cancer genes were prepared for the reported cancer gene annotation.\n\nAdditionally, 5,790 CGC-based and 6,744 CGI-based cancer pathway genes were assigned for the annotation of unreported but potential candidate genes (Table 1).\n\nThe gene ID database for gene symbol conversion, based on HGNC, and the functional annotation database, based on Entrez Gene and UniProt database flat files, were constructed.\n\nThe types of annotations provided by CaGe are summarized in Table 2.\n\nThe main window of the CaGe web interface is shown in Fig.2 for the cancer gene annotation process.\n\nMain menus, located in the upper part of the main window, are linked to the four main functions and home page of the system: 1) home page, 2) cancer gene annotation function, 3) cancer pathway annotation function, 4) cancer gene browsing function, and 5) cancer pathway browsing function.\n\nDetailed usages for CaGe are described in the user&#39;s guide at http://mgrc.kribb.re.kr/cage/include/cageUserGuide.pdf.\n\nTitle: System information\n\nCaGe was developed using PHP, R, and python languages; MySQL for database management; and Apache for the web server and is operated on a Linux platform with 8-core Intel Xeon CPUs (2.33 GHz) and 24 GB of main memory.\n\nTitle: Performance evaluation with small cell lung cancer mutation data\n\nTo assess the capability of CaGe, we annotated genes from candidate mutations for a small cell lung cancer genome (SCLC) [12].\n\nWe had 59 genes with predicted functionally damaging mutations after applying 22,910 mutations to the PolyPhen and could annotate 22 previously reported known cancer genes successfully by applying the PolyPhen output for CaGe.\n\nKnown cancer genes included RB1, which was mentioned as an SCLC-related gene in Pleasance&#39;s work; DST and ETS2, which were previously reported as SCLC-related genes but not mentioned in Pleasance&#39;s work; and 3 more genes (PDGFC, IL16, and AGTR2), which are known as lung cancer-related genes (Supplementary Table 1).\n\nThe other 16 genes are known to be related to other cancer types, suggesting that they might be important in the carcinogenesis of SCLC as well.\n\nFrom the pathway analysis, we identified 12 known cancer genes and 11 genes in cancer-related pathways (Supplementary Table 2).\n\nThose 11 genes might also be important in the carcinogenesis of SCLC.\n\nThus, we conclude that CaGe can annotate cancer genes effectively and suggest that CaGe will be useful in the identification of cancer-causing mutations and genes in HGT-based cancer genomics.\n\nIn this paper, we present a new cancer genomic tool, CaGe, for the assessment of candidate cancer genes having somatic mutations from HGT-based cancer genomics.\n\nCaGe provides users with information on cancer genes, mutations, pathways, and associated annotations through cancer gene annotation, cancer pathway annotation, cancer gene browsing, and cancer pathway browsing functions.\n\nIt has a capacity to process SIFT or PolyPhen output files as direct input for usual NGS-based cancer genomics flows.\n\nResearchers can classify their candidate genes from cancer genome studies into previously known or novel categories of cancer genes and gain insight into the underlying carcinogenic mechanisms through a pathway analysis.\n\nWe hope that CaGe will be useful for the identification of cancer-causing mutations and genes in HGT-based cancer genomics.\n\nConstruction and main service flows of CaGe.\n\nSIFT, Sorting Intolerant From Tolerant; NGS, next-generation DNA sequencing; HGNC, HUGO Gene Nomenclature Committee; CGC, Cancer Gene Census; CGI, Cancer Gene Index.\n\nCancer gene annotation page of CaGe web interface.\n\nCancer gene annotation results page of CaGe web interface.\n\nTitle: Cancer gene sets and cancer pathway gene sets in the CaGe database\n\nTitle: Annotations for a cancer gene provided by CaGe\n\n&#34;,
 &#34;Title: Decreases in Casz1 mRNA by an siRNA Complex Do not Alter Blood Pressure in Mice\n\nRecent genomewide association studies of large samples have identified genes that are associated with blood pressure.\n\nThe Global Blood Pressure Genetics (Global BPgen) and Cohorts for Heart and Aging Research in Genome Epidemiology (CHARGE) consortiums identified 14 loci that govern blood pressure on a genomewide significance level, one of which is CASZ1 confirmed in both Europeans and Asians.\n\nCASZ1 is a zinc finger transcription factor that controls apoptosis and cell fate and suppresses neuroblastoma tumor growth by reprogramming gene expression, like a tumor suppressor.\n\nTo validate the function of CASZ1 in blood pressure, we decreased Casz1 mRNA levels in mice by siRNA.\n\nCasz1 siRNA reduced mRNA levels by 59% in a mouse cell line.\n\nA polyethylenimine-mixed siRNA complex was injected into mouse tail veins, reducing Casz1 mRNA expression to 45% in the kidney.\n\nHowever, blood pressure in the treated mice was unaffected, despite a 55% reduction in Casz1 mRNA levels in the kidney on multiple siRNA injections daily.\n\nEven though Casz1 siRNA-treated mice did not experience any significant change in blood pressure, our study demonstrates the value of in vivo siRNA injection in analyzing the function of candidate genes identified by genomewide association studies.\n\nCASZ1 gene is a zinc finger transcription factor that controls cell fate, although little is known about its function [1].\n\nCASZ1 is also known as a survival-related gene that has 2 isoforms.\n\nIt encodes a putative protein of 172 amino acids that localizes predominantly in the perinuclear region, as predicted for transcription factors.\n\nCASZ1 is highly expressed in many human cancer cell lines, although its expression is low in most tissues, implicating it in tumor growth [2].\n\nThese findings indicate that CASZ1 is important for cell proliferation, development, and differentiation, as well as carcinogenesis.\n\nHowever, CASZ1 has recently been reported to be significantly associated with blood pressure and hypertension in genomewide association studies (GWASs) [3].\n\nBlood pressure GWASs by the Global Blood Pressure Genetics (Global BPgen) [4] and the Cohorts for Heart and Aging Research in Genome Epidemiology (CHARGE) consortium [5] have identified 14 independent loci that govern blood pressure or hypertension at a genomewide significance level, 7 of which were replicated in the Japanese population (CASZ1, MTHFR, ITGA9, FGF5, CYP17A1-CNNM2, ATP2B1, and CSK-ULK3) [6].\n\nSingle nucleotide polymorphism (SNP) rs880315, which lies in an intron of CASZ1, was one of the 7 replicate loci (p = 2.2 × 10-8, p = 4.9 × 10-12, and p = 3.0 × 10-7 for systolic blood pressure, diastolic blood pressure, and hypertension, respectively).\n\nCASZ1 was also linked to blood pressure in the Women&#39;s Genome Health Study (rs880315; p = 5.2 × 10-9 for systolic blood pressure) [7].\n\nRecently, the CASZ1 locus was identified in 19,608 subjects of east Asian ancestry by the Asian Genetic Epidemiology Network Blood Pressure (AGEN-BP) consortium [8].\n\nAlthough these blood pressure GWASs have suggested that these loci control blood pressure, their function in blood pressure has not been examined.\n\nTraditionally, knockout or transgenic mice have been used to reveal gene function and can be good approaches for examining candidate genes that lie near the GWAS loci [9].\n\nHowever, this process is expensive and time-consuming [10], requiring technical advances to determine their function in vivo.\n\nSmall interfering RNA (siRNA) is an alternative approach that has the potential to become the preferred tool for that purpose [11].\n\nIn vivo injection of siRNA reduces gene expression by 70% to 80% in certain tissues in mice, demonstrating its efficacy [12].\n\nHowever, specific tissues can not be targeted by in vivo injection of siRNA, and the injected siRNA is unstable in blood serum, limiting their use in delivering siRNA in vivo.\n\nIt also has certain disadvantages, such as effecting inflammatory responses and toxicity [13].\n\nThus, developing siRNA delivery methods in animal models is a challenging but worthy effort to accelerate the discovery of novel genes [11].\n\nWe sought to determine whether murine Casz1 controls blood pressure by administering Casz1 siRNA to mice.\n\nA polyethylenimine-mixed siRNA complex was injected into tail veins, and changes in Casz1 mRNA and blood pressure were measured.\n\nTitle: Animals\n\nBALB/c mice (Japan SLC, Inc., Shizuoka, Japan) were bred in a pathogen-free facility and used at 7 to 9 weeks of age for siRNA injection.\n\nThe mice were maintained on a 12-h light/dark cycle at a constant temperature with free access to food and water.\n\nEvery effort was made to minimize the number of animals that was used and their suffering per the Committee for the Care and Use of Laboratory Animals, College of Pharmacy, Kyung Hee University (KHP-2010-04-06).\n\nTitle: Synthesis of siRNA\n\nCasz1 siRNA oligonucleotides were synthesized by Genolution (Seoul, Korea).\n\nThe target sequence for Casz1 was 5&#39;-GATGTGATCCGACATTACA-3&#39; (siRNA#1), 5&#39;-CTAGAACTGCTCTGTATGT-3&#39; (siRNA#2), and 5&#39;-CAATTCGGCAAGTAATGGA-3&#39; (siRNA #3), and nonspecific scrambled siRNA was used as a negative control.\n\nTo examine the efficacy of siRNAs, siRNAs were transfected into B16F10 cells using G-Fectin (Genolution); 1-2 µL of G-Fectin and 20 nM siRNA were added to 50 µL phosphate buffered saline, and the reaction was incubated for 10 min at room temperature (RT).\n\nB16F10 cells were used to seed 24-well plates at 5 × 104 cells/well and 30% confluence, and the transfection mixture was added to the plate.\n\nThe cells were harvested for RNA extraction after 24 h of incubation.\n\nTotal RNA was prepared using TRIzol (Invitrogen, Carlsbad, CA, USA) per the manufacturer&#39;s instructions.\n\nTitle: Delivery methods\n\nThe transfection agent was in vivo-jetPEI™ (Polyplus, Illkirch-Graffenstaden, France).\n\nPer the manufacturer, 25 µg siRNA and 6.5 µL vivo-jetPEI™ (N/P charge ratio of 6) were diluted with 50 µL 10% glucose solution and 50 µL sterile H2O.\n\nThe solution was vortexed gently and left for 15 min at RT.\n\nThe mixture was injected into the tail veins of 7- to 9-week-old mice, and the treated mice were measured for blood pressure or sacrificed for RNA extraction.\n\nTitle: RNA extraction and real-time PCR\n\nRNA was extracted from the tissues of siRNA-treated mice with TRIzol (Invitrogen).\n\nWe synthesized cDNA from 500 ng of total RNA using the PrimeScript™ RT reagent kit (TaKaRa, Otsu, Japan) per the manufacturer&#39;s protocol.\n\nCasz1 mRNA was analyzed by real-time PCR.\n\nOne-tenth of the cDNA reaction was added to a final volume of 20 µL for each reaction containing SYBR Green I (TaKaRa).\n\nThe primers for Casz1 and glyceraldehye-3-phosphate dehydrogenase (GAPDH) were 5&#39;-GTCTCTTCGGGAACTGCAAG-3&#39;/5&#39;-TGGGACACAGGCACTGTAGA-3&#39; and 5&#39;-GCTCTCTGCTCCTCCTGTTC-3&#39;/5&#39;-CAATACGACCAAATCCGTTG-3&#39;, respectively (forward/reverse sequence).\n\nQuantitative real-time PCR was performed on an ABI Step One Real-time PCR system (Applied Biosystems, Foster, CA, USA) using the following program: 45 cycles of 95℃ for 10 s, 60℃ for 15 s, and 72℃ for 20 s. To normalize the amount of sample cDNA in each reaction, the Ct value of each target gene was subtracted by that of GAPDH to obtain delta Ct [14].\n\nTo calculate the fold-change in expression, the delta Ct value of the case sample was subtracted by that of the control (delta delta Ct = delta Ct case - delta Ct control).\n\nReverse-transcription PCR was performed similarly to quantitative real-time PCR, except that it used 35 cycles for amplification.\n\nThe PCR products were separated by electrophoresis in a 2.0% agarose gel and visualized under UV lamp after ethidium bromide staining.\n\nTitle: Blood pressure measurement\n\nBlood pressure was recorded intraarterially with a computerized data acquisition system (ADInstruments, Bella Vista, Australia).\n\nTo place the intraarterial catheter, mice were anesthetized with avertin.\n\nThe standard dose was 0.014 mL/g avertin, which was made by diluting stock solution (for stock solution, 10 g 2,2,2, tribromoethanol dissolved in 10 mL tertiary amyl alcohol) 40-fold in 0.9% NaCl.\n\nThe intraarterial catheter, a polyethylene tube (0.2 mm I.D., 0.5 mm O.D.)\n\n(Natsume, Tokyo, Japan) that was filled with 0.9% NaCl containing 100 U/mL heparin, was inserted into the right carotid artery and tied in place.\n\nThe blood pressure in the vessel was transmitted along the catheter to the transducer&#39;s diaphragm (MLT0699 Disposable BP Transducer; ADInstruments).\n\nThe diaphragm signal was amplified through a Bridge Amplifier and recorded on a PowerLab system (LabChart 7.2; ADInstruments).\n\nBlood pressure was monitored for 2 h after the injection of the anesthetic and calculated as the average blood pressure between 45 min and 75 min after the injection (1 point per min).\n\nStatistical analysis was performed using PASW Statistics version 18.0 (SPSS Inc., Chicago, IL, USA), and differences in blood pressure between case and control groups were analyzed by independent sample t-test.\n\nTitle: Reduction in Casz1 mRNA by siRNA injection into tail vein\n\nWe examined the efficacies of 3 Casz1 siRNA oligonucleotides in the B16F10 mouse melanoma cell line.\n\nBy real-time PCR, Casz1 mRNA levels fell 0%, 59%, and 54%, and we selected the second siRNA for the in vivo experiments.\n\nPrior to the injection of siRNA, Casz1 expression in heart, lung, liver, kidney, spleen, and brain was measured by reverse-transcription PCR.\n\nCasz1 was expressed in most tissues and at relatively low levels in liver (Fig.1).\n\nCasz1 siRNA was mixed with polyethylenimine, and the complex was injected into the tail veins of mice.\n\nAt 24 h after siRNA injection, various tissues of siRNA-treated mice were analyzed with regard to Casz1 mRNA levels.\n\nCasz1 mRNA expression was normalized to GAPDH, and a scrambled siRNA was used as control.\n\nBy quantitative real-time PCR, Casz1 mRNA levels declined only in kidney (Fig.2A).\n\nCasz1 mRNA levels in the kidney fell by 45%, 55%, and 25% after 1, 2, and 3 injections per day, respectively (n = 3 for each group) (Fig.2B).\n\nTitle: Blood pressure in siRNA-injected mice\n\nBlood pressure was measured 24 h after tail vein injection of the scrambled or Casz1 siRNA (control, n = 4; case, n = 3).\n\nUsing an intraarterial catheter that was connected to a pressure sensor, we recorded blood pressure continuously for 2 h after anesthesia.\n\nBlood pressure did not change after a 1-time injection (p = 0.713).\n\nThus, we injected siRNA twice each day (control, n = 4; case, n = 6) but did not observe any significant difference between the case and control groups (p = 0.545).\n\nAfter 3 siRNA injections daily for 3 days, blood pressure was also unaffected (p = 0.297) (Table 1, Supplementary Table 1).\n\nThis study attempted to validate CASZ1 as a regulator of blood pressure using siRNA in mice.\n\nAlthough Casz1 mRNA levels decreased by 55% in kidney, blood pressure did not change on injection of the siRNA complex into tail veins.\n\nIt is difficult to conclude that CASZ1 does not control blood pressure, because this study has several limitations.\n\nFirst, Casz1 mRNA levels in kidney were reduced up to 55%; if Casz1 expression had been abolished, we might have detected a significant difference in blood pressure between the case and control groups.\n\nSecond, it is possible that CASZ1 functions in other tissues besides the kidney in the regulation of blood pressure.\n\nThe kidney is an essential organ of the urinary system and controls homeostatic functions, such as the regulation of blood pressure, by maintaining salt and water balances.\n\nHowever, the function of CASZ1 in kidney is unknown due to the lack of Casz1 functional studies in kidney.\n\nCasz1 is highly expressed in brain, but siRNA nanoparticles do not access the brain due to the blood-brain barrier [15].\n\nThird, human CASZ1 is 87% and 90% identical to mouse Casz1 in the cDNA and polypeptide sequences, respectively [1].\n\nNevertheless, it is possible that Casz1 function differs between the two species.\n\nBecause most loci that have been identified by GWASs have several genes nearby, it may be necessary to examine many candidates to identify the functional gene.\n\nKnockout and transgenic mice are expensive and time-consuming to generate.\n\nIn contrast, siRNA injection costs less and is much faster to perform, making siRNA injection suitable for functional validation studies of GWAS results.\n\nIn summary, we did not observe any significant change in blood pressure in Casz1 siRNA-treated mice, but our study demonstrates the value of in vivo siRNA injection in the functional analysis of candidate genes from GWASs.\n\nCasz1 mRNA expression in normal mouse tissues.\n\nGAPDH, glyceraldehye-3-phosphate dehydrogenase.\n\nDecreases in Casz1 mRNA by an siRNA complex.\n\n(A) Fold-change in Casz1 mRNA levels in tissues in siRNA-injected mice (n = 2 per tissue).\n\n(B) Reduction in Casz1 mRNA levels in kidney by in vivo siRNA injection.\n\nTitle: Blood pressure in siRNA-injected mice\n\n&#34;]</pre>
</div>

</div>

</div>

</div>

</div>
<div class="jp-Cell-inputWrapper"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput " data-mime-type="text/markdown">
<h2 id="3.-Text-Preprocessing">3. Text Preprocessing<a class="anchor-link" href="#3.-Text-Preprocessing">&#182;</a></h2><p>논문지를 연도별로 분류하여 각 연도마다 가장 자주 나온 주요 단어들을 출력하고자 하므로, 논문지의 내용에서 유의미한 단어만을 추출하는 작업이 필요하다.</p>

</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell jp-mod-noOutputs  ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[57]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span><span class="n">pattern</span> <span class="o">=</span> <span class="s1">&#39;([a-z])[0-9]+([a-z])&#39;</span>
<span class="n">space</span> <span class="o">=</span> <span class="s1">&#39; &#39;</span>
</pre></div>

     </div>
</div>
</div>
</div>

</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell   ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[61]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># 문장 단위 분절</span>

<span class="kn">from</span> <span class="nn">nltk.tokenize</span> <span class="kn">import</span> <span class="n">sent_tokenize</span>
<span class="kn">from</span> <span class="nn">nltk.tokenize</span> <span class="kn">import</span> <span class="n">word_tokenize</span>

<span class="k">for</span> <span class="n">i</span> <span class="ow">in</span> <span class="nb">range</span><span class="p">(</span> <span class="nb">len</span><span class="p">(</span><span class="n">raw</span><span class="p">)</span> <span class="p">)</span> <span class="p">:</span>
    
    <span class="k">if</span> <span class="n">raw</span><span class="p">[</span><span class="n">i</span><span class="p">]</span><span class="o">.</span><span class="n">strip</span><span class="p">()</span> <span class="o">!=</span> <span class="s2">&quot;&quot;</span> <span class="p">:</span>
        <span class="n">raw</span><span class="p">[</span><span class="n">i</span><span class="p">]</span> <span class="o">=</span> <span class="n">re</span><span class="o">.</span><span class="n">sub</span><span class="p">(</span> <span class="sa">r</span><span class="s1">&#39;([a-z])\.([A-Z])&#39;</span><span class="p">,</span> <span class="sa">r</span><span class="s1">&#39;\1. \2&#39;</span><span class="p">,</span> <span class="n">raw</span><span class="p">[</span><span class="n">i</span><span class="p">]</span><span class="o">.</span><span class="n">strip</span><span class="p">()</span> <span class="p">)</span>
        <span class="n">sentences</span> <span class="o">=</span> <span class="n">sent_tokenize</span><span class="p">(</span> <span class="n">raw</span><span class="p">[</span><span class="n">i</span><span class="p">]</span><span class="o">.</span><span class="n">strip</span><span class="p">()</span> <span class="p">)</span>

        <span class="k">for</span> <span class="n">s</span> <span class="ow">in</span> <span class="n">sentences</span> <span class="p">:</span>
          <span class="k">if</span> <span class="n">s</span> <span class="o">!=</span> <span class="s2">&quot;&quot;</span> <span class="p">:</span>
              <span class="n">result</span> <span class="o">=</span> <span class="n">sys</span><span class="o">.</span><span class="n">stdout</span><span class="o">.</span><span class="n">write</span><span class="p">(</span><span class="n">s</span> <span class="o">+</span> <span class="s2">&quot;</span><span class="se">\n</span><span class="s2">&quot;</span><span class="p">)</span>

    <span class="n">result</span>
</pre></div>

     </div>
</div>
</div>
</div>

<div class="jp-Cell-outputWrapper">


<div class="jp-OutputArea jp-Cell-outputArea">

<div class="jp-OutputArea-child">

    
    <div class="jp-OutputPrompt jp-OutputArea-prompt"></div>


<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="text/plain">
<pre>Title: Survey of the Applications of NGS to Whole-Genome Sequencing and Expression Profiling

Recently, the technologies of DNA sequence variation and gene expression profiling have been used widely as approaches in the expertise of genome biology and genetics.
The application to genome study has been particularly developed with the introduction of the next-generation DNA sequencer (NGS) Roche/454 and Illumina/Solexa systems, along with bioinformation analysis technologies of whole-genome de novo assembly, expression profiling, DNA variation discovery, and genotyping.
Both massive whole-genome shotgun paired-end sequencing and mate paired-end sequencing data are important steps for constructing de novo assembly of novel genome sequencing data.
It is necessary to have DNA sequence information from a multiplatform NGS with at least 2× and 30× depth sequence of genome coverage using Roche/454 and Illumina/Solexa, respectively, for effective an way of de novo assembly.
Massive short-length reading data from the Illumina/Solexa system is enough to discover DNA variation, resulting in reducing the cost of DNA sequencing.
Whole-genome expression profile data are useful to approach genome system biology with quantification of expressed RNAs from a whole-genome transcriptome, depending on the tissue samples.
The hybrid mRNA sequences from Rohce/454 and Illumina/Solexa are more powerful to find novel genes through de novo assembly in any whole-genome sequenced species.
The 20× and 50× coverage of the estimated transcriptome sequences using Roche/454 and Illumina/Solexa, respectively, is effective to create novel expressed reference sequences.
However, only an average 30× coverage of a transcriptome with short read sequences of Illumina/Solexa is enough to check expression quantification, compared to the reference expressed sequence tag sequence.
Scientists have tried to understand biology through DNA sequence information, since the DNA is verified as the unit of genetic heredity.
Also, scientists hope to dramatically reduce the cost of reading genomic DNA and to obtain the high-throughput DNA sequence information.
Since the automated DNA sequencers were developed with fluorescent dyes of different colors, laser, and computer technology in the 1980s, the human genome project (HGP) was begun in 1990, and the human genome was completely released in 2003, while further analysis is still being published.
A total of about 3 billion dollars was invested to the project.
In 1991, the National Human Genome Research Institute (NHGRI) funds were geared toward lowering the cost of DNA sequencing.
Some of technologies invested improved the DNA sequencing.
To date, the Applied Biosystems, Roche/454, and Illumina/Solexa have successfully developed their technology and applied DNA sequencing in the world during the recent 6 years.
Most of the newest technologies currently in use generate sequences from 36 to 1,000 base pairs, which requires special software for different applications, including whole-genome sequencing, transcriptome analysis, and regulatory gene analysis.
In particular, in silico method development using bioinformation software for next-generation sequence assembly would be alert for many genome projects and more applications in genome biology.
Particularly, many biologists and geneticists using massive DNA and RNA sequences have used the sequencing applications focused on the variable research fields in medicine, such as improved diagnosis of disease; gene therapy; control systems for drugs, including pharmacogenomics &#34;custom drugs;&#34; evolution; bioenergy and environmental applications of creative energy sources (biofuels); clean up of toxic wastes, including efficient environmental sources against carbon; and agriculture projects, including livestock of healthier, more productive, disease-resistant farm animals and breeding of disease-, insect-, and drought-resistant crops.
In this paper, we report several ways of genome sequencing and expression profiling in genome biology.
Title: Roche/454 pyrosequencing technology

The first high-parallel sequencing system was developed with an emulsion PCR method for DNA amplification and an instrument for sequencing by synthesis using a pyrosequencing protocol optimized on the individual well of a PicoTiterPlate (PTP) [1].
The DNA sequencing protocol, including sample preparation, is supplied for a user to follow the method, which is developed by 454 Life Science (now part of Roche Diagnostics, Mannheim, Germany).
Even following the manufacturer&#39;s protocol, to obtain high-quality DNA sequence with maximal total DNA sequence length as possible, one should make a library with both adapters on the sheared DNA fragment and mix the optical ratio of library DNA versus bead for emulsion PCR.
Now, the system is upgraded to through-put a total of an average of 700 Mbp with an average 600 bp/read in one PTP run.
The sequencer is available for single reads and paired-end read sequencing for the application of eukaryotic and prokaryotic whole-genome sequencing [2, 3], metagenomics and microbial diversity [4, 5], and genetic variation detection for comparative genomics [6].
Title: Illumina/Solexa technology

The numerous cost-effective technologies were being developed for human genome resequencing that can be aligned to the reference sequence [7, 8].
The first successful technology to gain massive DNA sequencing available for resequencing was developed by Solexa (now part of Illumina, San Diego, CA, USA).
The principle of the system is the method of base-by-base sequencing by synthesis, where the sheared template DNA is amplified on the flat surface slide (flow cell) and detects one base on each template per cycle with four base-specific, fluorescently labeled signals.
Signals for all four fluorescent channels are collected and plotted at each position, enabling quality per scores to be derived using four-color information if desired [8].
Now, a maximum of 300 Gb for reading is available with 101 bp paired-end reading per fragment on a 1-flow cell run in the upgraded HiSeq system.
Title: Novel whole genome de novo assembly

More than 11,000 sequencing projects, including targeted projects, were reported on the Genome Online Database (GOLD, http://www.genomesonline.org) in early 2012.
Now, more than 3,000 genome projects have been completed on the diverse genome species, and more than 90% of completed projects were bacterial genome sequencing.
The greatest bacterial genome sequencing was performed with 454 pyrosequencing because of the available largest long read sequencing, useful for de novo assembly of novel genome sequencing.
The official depth of the deep sequencing strategy of 454 pyrosequencing technology for whole bacterial genome sequencing for de novo assembly in novel genome sequencing is at least 15-20× in depth of the estimated genome size [9-13].
However, Li et al.
[3] reported that 6-10× sequencing in qualified runs with 500-bp reads would be enough for de novo assemblies from 1,480 prokaryote genomes with &gt;98% genome coverage, &lt;100 contigs with N50, and size &gt;100 kb.
Recently, prokaryote whole genome sequencing using 101 bp paired-end read data from Illumina/Solexa systems was used for de novo assembly and resequencing.
For example, a Bacillus subtilis subspecies genome sequence was generated by using the short read sequence from Illumina/Solexa and assembled with the Velvet program [14].
In this case, the genome assembly was completed, based on the reference genome for ordering the numerous contigs derived from de novo assembly.
Even though numerous contigs assembled with Illumina/Solexa data were produced in the eukaryotic genome, a few drafts for the assembled genome sequence were reported, except for the giant panda genome [15], which was covered with assembled contigs (2.25 Gb), covering approximately 94% of the expected whole genome.
Another example was the woodland strawberry genome (240 Mb) [16] that was sequenced to 39× depth of the genome, assembled de novo, and anchored to the linkage map of seven pseudochromosomes.
The genome sequence could be associated with the predicted genes with transcriptome sequence data.
An ideal method for cost-effective novel genome sequencing using NGS is de novo assembly with diverse shotgun fragment end sequencing data of multiplat systems (Fig.1).
The first strategy of novel genome DNA sequencing is sequencing the genomic DNA for contig and scaffold construction after randomly sheared shotgun single read-end or paired-end read DNA sequencing using Roche/454 or Illumina/Solexa with information on how to assemble with the NGS data using variable assembly software.
Recently, a catfish genome was sequenced with multiplatform Roche/454 and Illumina/Solexa technology and assembled with an effective combination of low coverage depth of 18× Roche/454 and 70× Illumina/Solexa data using 3 assembly softwares - Newbler software to the 454 reads, Velvet assembler to the Illumina read, and MIRA assembler for final assembly of contigs and singletons derived from initial assembled data - resulting in 193 contigs with an N50 value of 13,123 bp [2].
In an additional multiplatform data assembly of a 40-Mb eukaryotic genome of the fungus Sordaria macrospra, a combination sequence of 85-fold coverage of Illumina/Solexa and 10-fold coverage by Roche/454 sequencing was assembled to a 40-Mb draft version (N50 of 117 kb) with the Velvet assembler as a reference of a model organism for fungal morphogenesis [17].
In the recent effective assembly methods reported, combinations of the multiplatform sequence are shown as successful novel genome assembly using variable assembly strategy pipelines.
Comparing the pipeline of assembly strategy, we suggest an effective integrated pipeline in which data are filtered to remove low-quality and short-read initial assemblies using variable software and then compared to contigs, hybrid contigs using MIRA assembler, and finally contig orders using SSPACE software (http://www.baseclear.com/dna-sequencing/data-analysis/) [18] for scaffold construction through de novo assembly of novel genome sequencing (Fig.2).
According to the comparison of several ways of de novo assembly, we suggest using both DNA sequences from multiplatform NGS with at least 2× and 30× depth sequences of genome coverage using Roche/454 and Illumina/Solexa, respectively, and doing hybrid assembly for cost-effective novel genome sequencing.
Title: SNP discovery and genotyping with resequencing

Resequencing of genomic regions or target genes of interest in a phenotype is the first step in the detection of DNA variations associated with the gene regulation.
The discovery of single-nucleotide polymorphisms (SNPs) including insertion/deletions (indels), with high-throughput data is useful to study genetic variation, comparative genomics, linkage map, and genomic selection for breeding value with DNA variation.
Many geneticists for biological and genome studies of microbial, plant, animal, and human genomes have effectively used NGS whole-genome resequencing data to use in variable research fields, such as bacterial evolution [19], genomewide analysis of mutagenesis of Escherichia coli strains [20], comparative genomics of Streptococcus suis of swine pathogen [21], genomic variation effects on phenotype and gene regulation in mouse [22], evolution of plant [23], and comparison of genetic variations on the targeted enrichment [24].
The platforms of resequencing projects have used Illumina/Solexa of short read lengths to align with the reference sequence to discover DNA variations between compared related species&#39; sequences.
Because of rare occurrence of SNPs in most species, it is important to identify high-accuracy data to discover DNA variations according to coverage depth using MAQ (http://maq.sourceforge.net/maq-man.shtml) [25] and CLC software (http://www.clcbio.com).
The public protocol of covering depth to discover SNPs and indels on the heterogeneous genome requires at least 30× of the reference genome, while about 10× depth of coverage is enough for DNA variation study of homogeneous genomes.
Of course, high coverage of depth provides high-quality data in SNP detection on the reference mapping (Fig.3).
However, short read lengths of 35 bp or 100 bp show enough to map on the reference sequence using the MAQ software and CLC software in the genome, including short repeated block regions.
But, geneticists still require long-read sequencing data to distinguish repeated block regions, like paralogous regions derived from gene duplication.
MAQ software provides a consensus sequence of the genotype sequenced of short read lengths with aligned raw reads to the reference sequence.
CLC software checks accuracy by counting reads of DNA variations of each position.
Recently, a novel application of pattern recognition for accurate DNA variations was discovered in the complexity of the genomic region using high-throughput data in a Caucasian population [26].
They used three independent datasets with Sanger sequencing and Affymetrix and Illumina microarrays to validate SNPs and indels of a clinical target region, FKBP5.
Therefore, it is necessary for multiplatform systems to validate DNA variations in the specific complexity of the genome region.
Title: Expression profiling

Gene expression profiling is a measurement of the regulation of a transcriptome from the whole genome in the field of molecular biology.
A conventional method to measure the relative activity of target genes is DNA microarray technology, which estimates expressed genes with the signals of hybridization of target genes (cDNA from mRNA) on the synthesized oligonucleotides [27].
The technology is still used for functional genomics in the wide era, including medicine, clinic, plant, and agricultural biotechnology [28-30].
In addition, microarray technology is also used in the comparative study of proteomics and expression, measuring the level of extracellular matrix protein [30].
Since NGS technology was developed in 2005, the transcriptome of novel whole genomes could be identified with massive parallel mRNA sequencing using Roche/454 and Illumina/Solexa [31-36].
The Roche/454 system is more useful for gaining novel gene discovery of novel species&#39; genomes for long read sequencing [37, 38].
Otherwise, Illumina/Solexa is being used to profile the expression of known genes with mapping short read sequences to the known reference genes [39, 40].
In that case, rare expressed genes and novel genes could be identified with high-throughput expressed sequence tag sequences using Illumina/Solexa.
Also, it is useful to find significant tissue-specific expression biases with comparison of transcript data [22].
Now, the hybrid mRNA sequence from Rohce/454 and Illumina/Solexa is more powerful for finding novel genes through de novo assembly in any whole-genome species.
The hybrid sequence data of 20× and 50× coverage of the estimated transcriptome sequence from Roche/454 and Illumina/Solexa, respectively, is effective in creating novel expressed reference sequences, while short-read Illumina/Solexa data are cost-efficient on expression quantification information for comparing exposed samples and natural phenotype samples through mapping to the reference genes (Fig.4).
Only and average 30× coverage of transcriptome depth of short-read sequences of Illumina/Solexa is enough to check expression quantification, compared to reference expressed sequence tag sequences.
The expressed information could be different, depending on the software using CAP3, MIRA, Newbler, SeqMan, and CLC.
Therefore, the results should be compared according to variable program options to define robust expression profiling [41].
To date, a powerful tool of ChIP-on-chip is used for understanding gene transcription regulation.
Thus, two-channel microarray technology of a combination of chromatin immunoprecipitation could be used for genomewide mapping of binding sites of DNA-interacting proteins [29].
In any NGS application, the transcriptome expression information would be more useful than complete genome information research with the lowest sequencing budget for biologists to better understand gene regulation of related genetic phenotypes with the in silico method.
Of in silico methods, conserved miRNA and novel miRNA discovery is available on the massive miRNAnome data in any species.
Specially, the target genes of miRNA discovered could be robust information to approach genome biology studies.
Transcriptome assembly is smaller than genome assembly and thus should be more computationally tractable but is often harder, as individual contigs can often have highly variable read coverages.
Comparing single assemblers, Newbler 2.5 performed the best on our trial dataset, but other assemblers were closely comparable.
Combining different optimal assemblies from different programs, however, gives a more credible final product, and this strategy is recommended [41].
NGS technology provides a cost-effective way of sequencing for novel whole-genome sequencing, resequencing, and expression profiling.
Rohce/454 pyrosequencing is recommended to de novo assembly of whole prokaryote novel genomes, while hybrid assembly with Illumina/Solexa sequences would be optimal for whole eukaryote genomes and transcriptome studies of non-model organisms.
Also, Illumina/Solexa sequencing is useful in detecting DNA variation, mapping the short-read resequence to the reference genome and profiling expressed genes in model organisms.
Furthermore, the high-throughput NGS sequencing enables us to study with an in silico method in variable research application fields of molecular genetics, including population diversity and comparative genomics, in a short time.
Stringency of whole-genome DNA shotgun sequencing for novel genome.
Whole-genome shotgun sequencing (left of the Figure) for contig construction: Sequencing of single-end or paired-end fragment of whole-genome DNA shotgun library, which are made with the average range of 200-800-bp fragments for Roche/454 or Illumina/Solexa systems.
In general, producing a total DNA sequence amount of 15-20× and 60× coverage in depth of a genome depends on using Roche/454 or Illumina/Solexa, respectively.
Whole-genome shotgun mate paired-end sequencing for scaffold construction (right of the Figure): sequencing of the mate paired-end fragments of the whole-genome DNA shotgun library, which are made with the average range of 2-40-Kb fragments for next-generation DNA sequencer.
The sequencing amount of more than 20× coverage in depth of the genome is effective for scaffold constructions.
Integrated pipeline for de novo assembly of novel genome sequencing.
The scheme is filtering data to remove low-quality and shot-read initial assemblies using variable software and compare to contigs, hybrid contigs using MIRA assembler, and contig ordering using SSPACE software to scaffold construction.
View of single nucleotide polymorphism (SNP) discovery through mapping short reads from Illumina/Solexa to reference sequence on MAQ software (A) and CLC software (B).
(A) Short read 35 bp per read of soybean genome shows completely mapped on the soybean reference sequence.
The MAQ software provides a consensus sequence of the genotype sequenced of short read lengths with aligned raw reads to the reference sequence.
(B) CLC software is useful for counting reads with DNA variations at each position.
A scheme of transcriptome expression analysis through massively parallel signature sequencing (MPSS) technology and bioinformatics: The identification of expressed genes through hybrid de novo assembly with Roche/454 and Illumina/Solexa data (left) and expressed level profiling through mapping the Illumina/Solexa sequence to the expressed sequence tag reference.
Title: Differential Expression of PKD2-Associated Genes in Autosomal Dominant Polycystic Kidney Disease

Autosomal dominant polycystic kidney disease (ADPKD) is characterized by formation of multiple fluid-filled cysts that expand over time and destroy renal architecture.
The proteins encoded by the PKD1 and PKD2 genes, mutations in which account for nearly all cases of ADPKD, may help guard against cystogenesis.
Previously developed mouse models of PKD1 and PKD2 demonstrated an embryonic lethal phenotype and massive cyst formation in the kidney, indicating that PKD1 and PKD2 probably play important roles during normal renal tubular development.
However, their precise role in development and the cellular mechanisms of cyst formation induced by PKD1 and PKD2 mutations are not fully understood.
To address this question, we presently created Pkd2 knockout and PKD2 transgenic mouse embryo fibroblasts.
We used a mouse oligonucleotide microarray to identify messenger RNAs whose expression was altered by the overexpression of the PKD2 or knockout of the Pkd2.
The majority of identified mutations was involved in critical biological processes, such as metabolism, transcription, cell adhesion, cell cycle, and signal transduction.
Herein, we confirmed differential expressions of several genes including aquaporin-1, according to different PKD2 expression levels in ADPKD mouse models, through microarray analysis.
These data may be helpful in PKD2-related mechanisms of ADPKD pathogenesis.
Autosomal dominant polycystic kidney disease (ADPKD) is a common monogenetic disorder, affecting as many as 1 in 500 people in the general population.
The disease is characterized by the formation of multiple fluid-filled renal cysts that expand over time and destroy the architecture of the kidney.
Five percent of all cases of chronic renal failure are due to ADPKD, and approximately 50% of ADPKD patients will develop end-stage renal disease by the time they are 60 years of age [1].
In the early stages of ADPKD, numerous cysts begin to enlarge from many segments of the kidney.
Later, the enlarged regions disassociate from the original nephron to form the actual cysts, which continue to enlarge due to proliferation of epithelial cells and fluid secretion into the cyst lumen.
Progressive renal cyst formation and enlargement result in a loss of renal function and hypertension and culminate in renal failure.
It is generally believed that cysts enlarge by means of abnormal cell growth, forming what is essentially a fluid-filled tumor that fills by transepithelial fluid secretion.
Cyst enlargement may then lead to extracellular matrix remodeling, interstitial fibrosis, a general inflammatory response, and disruption of the normal renal parenchyma, which then interfere with glomerular filtration and vascular blood flow, giving rise to cell death by apoptosis and ultimately renal failure [2].
Many studies have indicated that most cases of ADPKD are accounted for by mutations in the PKD1 and PKD2 genes, encoding the transmembrane protein polycystin-1 (PC1) and polycystin-2 (PC2), respectively.
Mice with targeted mutations in the PKD1 or PKD2 genes develop cystic kidneys during embryogenesis, and ADPKD in humans is associated with mutations in the PKD1 or PKD2 genes [3, 4].
In addition, the product of the PKD1 gene, PC1, has been implicated in a variety of pathways tied to proliferation, including G-protein signaling and the Wnt, activator protein 1 (AP-1), and Janus kinase-signal transducers and activators of transcription cascades [5, 6].
Moreover, depletion of PC1 has been shown to increase cell growth, whereas its overexpression slows cell growth, indicating that PC1 may negatively regulate cell proliferation [7, 8].
PC2, the protein product of PKD2, has also been implicated in cell cycle regulation via its calcium channel activity and stimulation of AP-1 [9, 10]; however, there has been little direct evidence tying PC2 to this process.
Here, we created a number of related cell lines that varied in their expression of PC2.
We describe studies designed to identify target genes under the control of the PKD2 using Pkd2 knockout (KO) and PKD2 transgenic (TG) mouse embryo fibroblasts (MEFs).
We used a mouse 30 K whole gene oligonucleotide microarray to identify messenger RNAs whose expression was altered by the overexpression of the PKD2 or KO of the Pkd2 in MEF cells.
Title: Establishment of MEF derived from wild-type and PKD2 mutation embryos

MEF were acquired from PKD2 TG embryos or Pkd2 KO embryos during development (13.5 days) [11].
Heads and limes were removed from embryos.
The remaining embryonic tissues were minced and dispersed in 0.05% trypsin prior to incubation at 37℃ for 15 min.
Cells were plated in Dulbecco&#39;s Modified Eagle&#39;s Medium of Defined Minimal Essential medium (Welgene Biotech, Taipei, Taiwan) supplemented with 10% fetal bovine serum (Welgene Biotech) and were cultured at 37℃ in an atmosphere of 5% CO2 until confluent growth was achieved.
MEF cells were frozen as stocks at the second passage and were used for the subsequent studies at the third passage.
Title: Microrray hybridization and data analysis

Total RNA was prepared using a commercial kit (Qiagen, Valancia, CA, USA) according to the manufacturer&#39;s instructions.
Gene expression profiles were obtained using the CodeLink Uniset Mouse Expression Bioarray (Amersham, Buckinghamshire, UK).
The 30,000 gene probes contained in the bioarray allows the detection of differences in gene expression as low as 1.3-fold with 95% confidence.
Ten micrograms of total RNA was amplified, and labeled cDNA was produced.
Biotin-labeled cDNA was hybridized to the array overnight in a shaking incubator at 37℃, and excess target sequences were washed away using a series of saline sodium citrate washes.
The array was stained by treatment with streptavidin-Alexaflour 647 (Molecular Probes, Eugene, OR, USA), the excess was washed away, and the array was scanned at an excitation wavelength of 632 nm using a GenePix scanner (Molecular Devices, Sunnyvale, CA, USA).
The resulting image was quantified, and the intensity of each spot was divided by the median spot intensity to provide a scaled and comparable number across multiple arrays.
Title: DNA microarray scanning and analysis

Microarrays were scanned using an Arraywox scanner (Applied Precision, Seattle, WA, USA), analyzed using ImaGene version 5.1 software (Biodiscovery, Segundo, CA, USA), and normalized using Genesight version 3.2 software (Biodiscovery).
Normalization was performed by subtracting the means of all genes.
The data normalized by Genesight were compared using an M/A plot.
Genes differentially expressed were identified by intensity differences, after subtracting the background intensity.
Genes showing expression changes of at least 2-fold were selected, and these selected genes were clustered by the hierarchical method.
Title: Clustering algorithm

The normalized log ratio corresponding to each time point was exported to the software for clustering algorithm.
Acuity version 3.1 (Molecular Devices) was used for the &#39;gene shaving&#39; algorithm.
To estimate the number of clusters in a dataset, the &#39;Gap Statistics&#39; attribute of the Acuity software package was used.
The cluster number, estimated by gap statistics, was used for an input parameter in gene shaving.
After gene shaving, a single linkage was used for hierarchical clustering.
Title: Semi-quantitative and real-time RT-PCR

Total RNA was prepared using a commercial kit (Qiagen) according to the manufacturer&#39;s instructions.
Single-strand cDNA was synthesized by incubating 5 µg total RNA with 200 units AMV, 100 nM oligo(dT)12-18, 1 mM dNTP mixture, and 40 units RNase inhibitor at 42℃ for 1 h in a final volume of 25 µL.
The reaction was terminated by incubation at 70℃ for 15 min.
The initial amount of mRNA and reaction conditions were optimized to obtain linearity for mouse 18s rRNA.
For semi-quantitative reverse transcription (RT)-PCR, the used primers were: human PKD2, forward 5&#39;-CGTGCCCCAGCCCAGTC-3&#39; and reverse 5&#39;-TTCCAGTACAGCCCATCCAATAAG-3&#39;; mouse Pkd2, forward 5&#39;-TGCGAGGGCTGCGAGGTC-3&#39; and reverse 5&#39;-TGTCAGCTTGCGTGTGGTTGC-3&#39;; mouse 18s rRNA, forward 5&#39;-GTAACCCGTTGCACCCCATT-3&#39; and reverse 5&#39;-CCATCCAATCGGTAGTAGCG-3&#39;.
RT-PCR cycling conditions were as follows: 10 min at 95℃, 25 cycles of 50 s at 94℃, 50 s at 57℃, 50 s at 72℃, and 5 min at 72℃.
The amplified products were separated on a 1% agarose gel.
For real-time PCR, the used primers were: mouse β-actin, (forward 5&#39;-GACGATGCTCCCCGGGCTGTATTC-3&#39; and reverse 5&#39;-TCTCTTGCTCTGGGCCTCGTCACC-3&#39;) used as a positive control; mouse peroxisome proliferator-activated receptor γ (PPAR-γ), forward 5&#39;-TCTTAACTGCCGGATCCACAAAAA-3&#39; and reverse 5&#39;-ATCTCCGCCAACAGCTTCTCCTTC-3&#39;; mouse protein kinase α (PKCα), forward 5&#39;-GGGCAGCCTCCGTTTGATGGT-3&#39; and reverse 5&#39;-CGCTTGGCAGGGTGTTTGGTC-3&#39;; mouse Wnt4, forward 5&#39;-GCCATCGAGGAGTGCCAATACC-3&#39; and reverse 5&#39;-GGCCACACCTGCTGAAGAGATG-3&#39;; mouse integrin α4 (ITGα4), forward 5&#39;-GTAGCCCCAGTGGAGAGCCTTGTG-3&#39; and reverse 5&#39;-ATGCCAGTGGGGAGTTTGTTATCG-3&#39;; mouse IL6ST, forward 5&#39;-TGAATCGGACCCACTTGAGAGG-3&#39; and reverse 5&#39;-CAGGAGCGGCTTGTTTGAGGTA-3&#39;; mouse aquaporin 1 (AQP1), forward 5&#39;-GGAGGCGCCGAGACTTAGGT-3&#39; and reverse 5&#39;-GCGGGTGAGCACAGCAGAGC-3&#39;; mouse transforming growth factor-β2 (TGF-β2), forward 5&#39;-TCATCCCGAATAAAAGCGAAGAGC-3&#39; and reverse 5&#39;-AGGGCAACAACATTAGCAGGAGAT-3&#39;.
Real-time RT-PCR was performed using the real-time SensiMixPlus SYBR kit as described by the manufacturer&#39;s instructions (Quantance, London, UK).
Title: Western blot analysis

Proteins were extracted using radio-immunoprecipitation assay buffer in MEF cells.
Proteins were separated by 12% sodium dodecyl sulfate-polyacryamide gel electrophoresis and were transferred to a polyvinylidene fluoride membrane (Millipore, Bellerica, MA, USA).
Membranes were blocked with 5% non-fat dry milk and were incubated with various antibodies.
Antibodies to PC2 (anti-human and anti-mouse) and AQP1 were obtained from Santa Cruz Biotechnology (Santa Cruz, CA, USA).
Antibodies to β-actin were obtained from Sigma-Aldrich (St. Louis, MO, USA).
Anti-Wnt4 was obtained from R&amp;D Systems (Minneapolis, MN, USA).
Each membrane was washed with phosphate-buffered saline-Tween, and immunocomplexes were detected by enhanced chemiluminescence (Amersham).
Title: Confirmation of PKD2 expression in Pkd2 KO and PKD2 TG MEFs

Mouse Pkd2 mRNA was revealed in KO wild-type (KOWT) MEF cells but not in KO MEF cells.
Also, Pkd2 mRNA was expressed in TG wild-type (TGWT) MEF cells and PKD2 TG MEF cells.
Human PKD2 mRNA was only expressed in PKD2 TG MEF cells obtained from PKD2 TG embryos following injection of the human PKD2 transgene (Fig.1A).
Consistently, the expression level of polycystin-2 was lower in KO MEFs than KOWT MEFs as well as higher in TG MEFs than TGWT MEFs (Fig.1B).
Title: Microarray gene expression analysis

We used a mouse 30 K whole gene oligonucleotide microarray to identify mRNAs whose expression was altered by the Pkd2 KO and PKD2 TG MEF cells.
The normalized log ratios corresponding to each time point were exported to the software for clustering algorithm.
We used Axon&#39;s Acuity 3.1 (Axon Instruments, Union City, CA, USA) for using the &#39;gene shaving&#39; algorithm.
To estimate the number of clusters in a dataset, we used &#39;Gap Statistics&#39; in Acuity 3.1.
The cluster number estimated by gap statistics was used for an input parameter in gene shaving.
After gene shaving, we used a single linkage hierarchical clustering.
The single linkage hierarchical clustering method divides 8 &#39;gene shaving&#39; clusters (Fig.2).
The majority of genes whose expression was appreciably altered encoded proteins involved in critical biological processes, such as metabolism, transcription, cell adhesion, cell cycle, and signal transcription.
Forty-five genes whose expression was changed by 2-fold or greater were identified (Table 1).
Title: Verification of candidate genes by quantitative RT-PCR and immunoblot analysis

To verify the results obtained by cDNA microarray analysis, expression patterns for selected genes were confirmed at the RNA and protein levels.
We selected seven genes among the 45 genes (Fig.3A).
As result of realtime RT-PCR, the expression patterns of PKCα, ITGα4, and AQP1 were identical to microarray analysis in Pkd2 KO MEF cells (Fig.3B).
In contrast, 6 genes, except for TGF-β2, were identically expressed in PKD2 TG MEF cells (Fig.3C).
Especially, the AQP1 gene may be significantly regulated by differential Pkd2 gene expression level.
Furthermore, to confirm the AQP1 protein expression level, western blot analysis was performed using MEF cells as well as kidney tissues obtained from Pkd2 KO (or heterozygote) and PKD2 TG mice.
As result, AQP1 protein level was consistent with mRNA levels in both MEFs and tissues (Fig.4).
Although remarkable progress toward understanding the genetics and pathophysiology of ADPKD has been made to date, it is still unclear how mutations in disease-causing genes trigger cystogenesis and what important role other molecules play in the cystic phenotype [12].
Our aim here was to identify genes associated with the cystic phenotype using Pkd2 KO and PKD2 TG MEF cells.
In a previous study, overexpression of human PKD2 led to anomalies in tubular function, due to abnormalities in tubule morphogenesis [13].
Similar findings have been made for the PKD1 gene.
For this gene, gain/loss of function and haploinsufficiency leads to cystogenesis, with the severity of the phenotype being related to the level of imbalance.
More severe phenotypes are seen for gain and loss of function than for haploinsufficiency [4].
In the present study, the association between cystogenesis and the loss or gain of PKD2 supports the idea that PC2 functions through Pkd2 mutant MEF cells.
Presently, we found Wnt4 to be up-regulated in Pkd2 KO MEF cells and down-regulated in PKD2 TG MEF cells.
Wnt4, a member of the Wnt family of lipid-modified secreted glycoproteins, plays central roles in the initial stages of the tubulogenic program.
Wnt4 is expressed in the pretubular aggregate, as one of the first molecular responses to inductive signaling mediated by ureteric epithelium [14].
AQP1 was observed to be up-regulated in both Pkd2 KO and PKD2 TG MEF cells.
AQP1 is a channel-forming integral membrane protein found in erythrocytes and water-transporting tissues, including the kidney, where it is localized to the proximal tubules and thin descending limbs of Henle&#39;s loop.
The selective expression pattern of AQP1 in human nephrogenesis suggests that proximal tubules and thin descending limbs become permeable to water early in development.
A predominantly apical expression of AQP1 is found during a critical period of the nephrogenesis, during formation of the proximal tubule, as well as in the early stage of ADPKD; in normal tubules, distribution is both apical and basolateral [15].
Another interesting defect in urine-concentrating ability was described in the Aqp1 knockout mouse.
Although these mice were grossly normal in terms of survival and appearance, they were vulnerable to water deprivation and became severely dehydrated.
These results suggest that AQP1 is required for the formation of concentrated urine [16].
We have found PKC-α and Bcl-2 associated transcription factor to be down-regulated in Pkd2 KO MEF cells and up-regulated in PKD2 TG MEF cells.
Recent observations in animal models of PKD have implicated apoptosis in its pathogenesis.
PKC- and Erk-dependent pathways are critical components of the cell survival by suppressing the apoptosis [17].
We also have demonstrated the role for PKD2 in cellular adhesion processes.
In this study, ITGα4 was up-regulated in Pkd2 KO MEF cells and down-regulated in PKD2 TG MEF cells.
It suggests that adhesion molecule such as ITGα4 may be associated with PKD2 mutation-related pathogenesis.
In conclusion, by using the cDNA microarray technique, we have found several candidate genes that may be involved in cystogenesis: PKCα, ITGα4 and AQP1.
These genes were affected by PKD2 expression and may be related to cyst formation of kidney together with the PKD2 gene.
Establishment of Pkd2 KO and PKD2 TG MEFs.
(A) Semi-quantitative reverse transcription-PCR analysis of Pkd2 gene in Pkd2 KO and PKD2 TG MEF cells.
Human- or mouse-specific PKD2 primer sets were used.
18s rRNA was used as a loading control.
(B) Western blot analysis of polycystin 2 (PC2) expression in Pkd2 KO and PKD2 TG MEF cells.
MEFs, mouse embryo fibroblasts; KO, knockout (MEF); TG, transgenic (MEF); KOWT, knockout wild-type (MEF); TGWT, transgenic wild-type (MEF).
Clustering of gene expression patterns in Pkd2 mutant MEF cell lines.
We performed clustering analysis microarray data between KO set ratio data and TG set ratio data using the graphical user interface (GUI) system.
(A) Gene expression profiles of 2,306 probe sets distributed in a dendrogram.
(B) Line plots of each cluster.
KO set, ratio between KO MEF and KOWT MEF; TG set, ratio between TG MEF and TGWT MEF.
MEFs, mouse embryo fibroblasts; KO, knockout (MEF); TG, transgenic (MEF); KOWT, knockout wild-type (MEF); TGWT, transgenic wild-type (MEF).
Expression verification of candidate genes at the mRNA level.
(A) Microarray data for selective genes including PPARγ, PKCα, Wnt4, ITGα4, IL6ST, AQP1, and TGF-β2.
(B, C) Real-time reverse transcription-PCR analysis of gene expression for selective genes in Pkd2 KO and PKD2 TG MEF cells.
β-actin primer was used as the internal control.
The experiment was performed in triplicate.
PPAR-γ, peroxisome proliferator-activated receptor γ; PKCα, protein kinase α; ITGα4, integrin α4; AQP1, aquaporin 1; TGF-β2, transforming growth factor-β2; KO, knockout; TG, transgenic; MEF, mouse embryo fibroblasts; KOWT, knockout wild-type (MEF); TGWT, transgenic wild-type (MEF).
Expression verification of AQP1 at the protein level.
(A) Western blot analysis of AQP1 in MEFs and kidney tissues derived from Pkd2 KO (or heterozygote) and PKD2 TG mice.
PKD2 TG and TGWT mice: 18 mo; Pkd2 heterozygote and KOWT mice: 12 mo.
Anti-β-actin was used as the internal control.
(B) For quantification of western blot, Multigauge software (Fuji Film) was used.
AQP1, aquaporin 1; MEF, mouse embryo fibroblasts; KO, knockout; TG, transgenic; KOWT, knockout wild-type (MEF); TGWT, transgenic wild-type (MEF).
Title: Genes deregualted by differential Pkd2 expression
Title: Possibility of the Use of Public Microarray Database for Identifying Significant Genes Associated with Oral Squamous Cell Carcinoma

There are lots of studies attempting to identify the expression changes in oral squamous cell carcinoma.
Most studies include insufficient samples to apply statistical methods for detecting significant gene sets.
This study combined two small microarray datasets from a public database and identified significant genes associated with the progress of oral squamous cell carcinoma.
There were different expression scales between the two datasets, even though these datasets were generated under the same platforms - Affymetrix U133A gene chips.
We discretized gene expressions of the two datasets by adjusting the differences between the datasets for detecting the more reliable information.
From the combination of the two datasets, we detected 51 significant genes that were upregulated in oral squamous cell carcinoma.
Most of them were published in previous studies as cancer-related genes.
From these selected genes, significant genetic pathways associated with expression changes were identified.
By combining several datasets from the public database, sufficient samples can be obtained for detecting reliable information.
Most of the selected genes were known as cancer-related genes, including oral squamous cell carcinoma.
Several unknown genes can be biologically evaluated in further studies.
Despite recent advances in surgical, radiation, and chemotherapeutic treatment protocols, the prognosis of oral squamous cell carcinoma (OSCC) remains mournful, with an approximate 50% 5-year mortality rate from disease or associated complications [1].
Therefore, the identification of biological markers is essential to make progress in detecting malignancy at an early stage and developing novel therapies [2].
Microarray datasets that are created for the same research purposes in different laboratories have accumulated rapidly.
The results from different datasets are often inconsistent due to the utilization of different platforms, sample preparations, or various technical variations.
If we could combine such datasets by adjusting for systematic biases that exist among different datasets derived from different experimental conditions, the power of statistical tests would be improved by the increase in sample size [3].
In OSCC, although lots of microarray-based studies have been conducted to provide insights into gene expression changes, most of these studies have contained insufficient samples for detecting reliable information using statistical analysis [4, 5].
Therefore, this study attempted to combine several datasets in the public database for detecting significant genes.
We used two small microarray datasets of OSCC for this study, which were based on the same platform but had different expression scales.
These two datasets were combined after discretization, because a previous study showed that classification could be improved using combined datasets after discretization [3].
After combining datasets, we used chi-square test for identifying the significant genes.
Chi-square test has been used commonly to detect differentially expressed genes after discretization of expression intensities in the microarray experiment.
In this study, gene expression ratios of two datasets were transformed with their ranks for each dataset.
Next, the transformed datasets were combined, and a nonparametric statistical method was applied to the combined dataset to detect informative genes.
Finally, we showed that most of the selected genes were known to be involved in various cancers, including OSCC.
Title: Dataset

Two microarray datasets were used for this study.
We acquired these datasets from a public database (Gene Expression Omnibus, GEO).
One was the expression dataset of 16 tumors and 4 normal tissues from 16 patients, using Affymetrix U133A gene chips (Affymetrix, Santa Clara, CA, USA).
The other microarray dataset consisted of expression profiles of 22 tumors and 5 normal tissues.
These two datasets were experimented on under the same platform, Affymetrix U133A.
The datasets are summarized in Table 1.
Title: Process for combining datasets

Title: For combining datasets, gene expression ratios are rearranged in order of expression ratios by each gene in each dataset, and the ranks are matched with the corresponding experimental group.
If the experimental groups are homogenous, the ranks within the same experimental group would be neighboring.
The process of discretization of gene expressions is summarized in the following steps [3]:

Rank the gene expression ratios within a gene for each dataset.
List in order of the ranks, and assign the order of gene expressions to the corresponding experimental groups.
Summarize the result of (2) in the form of a contingency table for each gene.
Combine the contingency tables that have been summarized for each dataset.
When there are three datasets to be combined, the datasets can be added as a single entry, as shown in Table 2, after the transformation of each dataset by rank.
Title: Identification of significant genes from a combined dataset

Title: After the summarization of gene expression ratios in the form of a contingency table for each gene, as shown in Table 3, a nonparametric statistical method was applied to the datasets for independence testing between gene expression patterns and experimental groups.
The test statistics are calculated as follows for each gene:

When the sample size is small - generally Ê(nij) less than 5 - Fisher&#39;s exact test is recommended rather than chi-square test.
The significant genes can be selected by an independence test between the phenotypes and gene expressions using this type of summarized dataset.
ci and ri represent the marginal sums of the ith column and row, respectively.
nij is the number of experiments belonging to Ei and Pj, and n represents the total number of experiments.
The clinical information and expression levels of two datasets are summarized in Table 4 and Fig.1.
Subgroup and sex were similarly distributed in the two datasets.
The distributions of other factors were not included.
The scale of expression levels in the two datasets was different; the expression values of Data 2004 ranged from 0.01 to 740, and those of Data 2005 were from 0.1 to 19,773.
The expression patterns of the two datasets can be explored in Fig.1.Lots of outliers are shown in Fig.1A in the two datasets containing whole gene sets.
However, in subsets of significant genes, the expression ranges got narrow, and the outliers were decreased (Fig.1B).
The expressions of tumor tissues in Data 2004 were upregulated and varied compared with normal tissues.
If there was no outlier with a maximum value in the 14th tumor tissue in Data 2004, the expressions of the two different groups would be clearly distinguished.
Any clear differences in expression were not shown between the two groups in Data 2005.
Title: Upregulated 51 genes in oral squamous cell carcinoma

To identify differently expressed genes between normal and tumor tissues, we performed chi-square test using a combined microarray dataset.
Fifty-one significant genes were selected from a combined dataset with p-value less than 0.005, which were upregulated in OSCC tissues.
The significance level can be controlled, and more genes can be selected with a lower significance level.
These selected genes are summarized in Table 5.
Many genes among the selected genes were known as cancer-related genes.
STAT1 [6], SKP2 [7], IFI16 [8], RHEB [9], FIF44 [10], SOD2 [11, 12], and GREM1 [11] are related to OSCC.
Table 6 [13-56] summarizes the previous studies that have published the relations of selected genes with cancer.
Title: Expression pattern of the identified genes

To investigate whether the different experimental groups could be classified with significant genes, an unsupervised hierarchical clustering method was applied to the significant gene set (Fig.2).
The normal group consisted of 4 tissues and showed significantly lower expression levels when compared with the tumor group.
In Fig.2, we investigated the classification availability of the identified genes in Data 2004, not in a combined dataset, because the two datasets have different expression scales.
Title: Network analysis

Based on all identified genes, new and expanded pathway maps and connections and specific gene-gene interactions were inferred, functionally analyzed, and used to build on the existing pathway using the Ingenuity Pathway Analysis (IPA) knowledge base [57].
To generate networks in this work, the knowledge base was queried for interactions between the identified genes and all other genes stored in the database.
Four networks were found to be significant in OSCC.
The network with the highest score (Network 1, score = 36) was generated, with 17 identified genes (Table 7, Fig.3).
In the network diagram, STAT1 and SOD2 neighbored with NMI and AURKA, respectively.
The expression levels of STAT1 and SOD2 could be expected to be related with those of NMI and SOD2.
Actually, the expressions of STAT1 and SOD2 were strongly positively correlated with NMI (r = 0.95) and AURKA (r = 0.87), respectively.
OSCC is associated with substantial mortality and morbidity [58].
To identify potential biomarkers for early detection of invasive OSCC, microarray experiments have been conducted, and these kinds of microarray datasets have accumulated rapidly in the public database.
However, there are many datasets that include insufficient sample sizes for detecting significant genes by statistical analysis.
Therefore, this study attempted to combine several microarray datasets from a public database to identify significant candidates as biomarkers.
In a microarray data analysis, the information from different datasets obtained under different experimental conditions may be inconsistent even though they are performed with the same research objectives.
Moreover, even when the datasets are generated by the same platform, the data agreement may be affected by technical variations between laboratories.
In such cases, it could be necessary to use a combined dataset after adjusting for the differences between such datasets for detecting the more reliable information.
Combining datasets is especially useful in OSCC microarray datasets, because there are many datasets with insufficient sample sizes for analysis [4, 5, 59, 60].
For identifying significant genes classifying tumor and normal groups, we achieved two microarray datasets from a public database, GEO.
They included 20 and 27 samples, and each sample size was unbalanced between the different groups.
By combining these two datasets, the sample size was increased, and we had a sufficient sample size for statistical analysis, even though it was still unbalanced.
When these datasets were combined, we used the rank of gene expression, because the scale of gene expression was different.
In this study, we identified 51 significant genes from a combined dataset, and this number could be increased or decreased by the significance level (we used 0.005).
The selected 51 genes were upregulated in tumor tissues.
Many of the selected genes were proven to be cancer-related genes by previous studies.
SOD2 is associated with lymph node metastasis in OSCC and may provide predictive values for the diagnosis of metastasis [10].
Metastasis is a critical event in OSCC progression.
An SOD2 variant has also been associated with increased breast cancer and ovarian cancer risk in previous studies [47, 61].
TopBP1 included eight BRCT domains (originally identified in BRCA1), and it was proposed as a breast cancer susceptibility gene [18, 62].
By semiquantitative reverse transcription PCR analysis, RHEB was shown to be upregulated in OSCC [9].
In salivary cancer, survival probability rates dropped when Skp2 was overexpressed [7].
Overexpression of Skp2 is associated with the reduction of p27 (KIP1) expression and may have a role in the progression of OSCC [25].
The expression of RCN2 was linearly related to the tumor mass increase, and its expression was increased in breast cancer [16].
PTPRK was proven as a candidate gene of colorectal cancer [19], and it is a functional tumor suppressor in Hodgkin lymphoma cells [20].
DMTF1 was shown to be amplified in adenocarcinoma of the gastroesophageal junction, residing at 7q21 by aCGH experiments [21].
FEZ1 was involved in ovarian carcinogenesis, and its reduction or loss could be an aid to the clinical management of patients affected by ovarian carcinoma [22].
It is also a known tumor suppressor gene in breast cancer and gastric cancer [23, 63].
Other ovarian cancer-related genes were NMI [27, 28] and FANCI [44]; breast cancer-related genes were COX11 [42], MELK [33], and FANCI [44] among the selected genes.
MELK was known to be associated with shorter survival in glioblastoma [34].
TTK was associated with progression and metastasis of advanced cervical cancers after radiotherapy [29, 30].
It might also be a relevant candidate as a new target in cancer therapy, since it plays relevant roles in mitotic progression and the spindle checkpoint [31, 32].
Aurora kinase A (AURKA) was associated with skin tumors [36] and colorectal cancer [37, 38].
In previous studies, OSCC-related genes among the selected genes were STAT1 [14], SKP2 [7, 25], IFI16 [8], RHEB [9], IFI44 [64], SOD2 [10-12], and GREM1 [11].
The gene set, which has not been proven as OSCC-related genes until now, could be expected to be possibly proven as OSCC-related genes by biological evaluation.
In this study, we identified significant genes related with OSCC from two microarray datasets in a public database.
For this, we transformed microarray datasets using ranks of gene expressions with different expression scales, even though they were constructed under the same experimental conditions.
This method could be useful when using multiple datasets that are created for the same research purpose, By combining these accumulated datasets, we can detect more reliable information due to the increased sample size.
It saves time and money and avoids repeating experiments.
Comparison of expression levels of two datasets.
(A) Whole gene set.
(B) Selected gene set.
Expression patterns of the selected 51 genes.
These genes were upregulated in oral squamous cell carcinoma tissues, and normal and tumor groups were clearly classified with these genes.
Network with the highest score (Network 1).
Functional relationships between genes based on known interactions in Ingenuity Pathway Analysis (IPA) knowledge are described.
Title: Summary of two microarray datasets from GEO and the combined dataset

Title: Combination of contingency tables for three datasets (tij = aij + bij + cij)

Title: Summary of discretized data using ranks of gene expressions

Title: Summary of two microarray datasets

Title: Summary of selected 51 upregulated genes

Title: Association of the selected genes and cancer

Title: Four networks generated by upregulated genes in OSCC
Title: CaGe: A Web-Based Cancer Gene Annotation System for Cancer Genomics

High-throughput genomic technologies (HGTs), including next-generation DNA sequencing (NGS), microarray, and serial analysis of gene expression (SAGE), have become effective experimental tools for cancer genomics to identify cancer-associated somatic genomic alterations and genes.
The main hurdle in cancer genomics is to identify the real causative mutations or genes out of many candidates from an HGT-based cancer genomic analysis.
One useful approach is to refer to known cancer genes and associated information.
The list of known cancer genes can be used to determine candidates of cancer driver mutations, while cancer gene-related information, including gene expression, protein-protein interaction, and pathways, can be useful for scoring novel candidates.
Some cancer gene or mutation databases exist for this purpose, but few specialized tools exist for an automated analysis of a long gene list from an HGT-based cancer genomic analysis.
This report presents a new web-accessible bioinformatic tool, called CaGe, a cancer genome annotation system for the assessment of candidates of cancer genes from HGT-based cancer genomics.
The tool provides users with information on cancer-related genes, mutations, pathways, and associated annotations through annotation and browsing functions.
With this tool, researchers can classify their candidate genes from cancer genome studies into either previously reported or novel categories of cancer genes and gain insight into underlying carcinogenic mechanisms through a pathway analysis.
We show the usefulness of CaGe by assessing its performance in annotating somatic mutations from a published small cell lung cancer study.
High-throughput genomic technologies (HGTs), including next-generation DNA sequencing (NGS), microarray, and serial analysis of gene expression (SAGE), have become effective tools for cancer genomics through various applications.
Especially, the applications of NGS include whole-genome, exome, and transcriptome approaches for searching for a wide range of cancer-specific genomic alterations, such as point mutations, insertions, and deletions; copy number changes; and rearrangements, leading to the development of cancer [1].
One of the hurdles in HGT-based cancer genomics is to identify causative mutations or genes out of many candidates from the analysis.
A useful approach is to refer to known cancer genes and associated information [2].
The list of known cancer genes can be used to determine candidates of cancer driver mutations, while cancer gene-related information, such as gene expression, protein-protein interaction, and pathway information, can be useful for scoring novel candidates.
Some useful cancer genomic databases for this purpose exist, including Cancer Gene Census (CGC) [3] and Catalogue of Somatic Mutations in Cancer (COSMIC) [4], which have been constructed and managed by the Cancer Genome Project of the Welcome Trust Sanger Institute; and Cancer Gene Index (CGI, http://ncicb.nci.nih.gov/NCICB/projects/cgdcp), The Cancer Genome Atlas (TCGA, http://cancergenome.nih.gov), and Cancer Genome Anatomy Project (CGAP, http://cgap.nci.nih.gov), which are maintained by the National Cancer Institute of the U. S. National Institute of Health and National Human Genome Research Institute.
Among them, COSMIC and CGC are the two most commonly used resources among researchers when checking reported cancer driver mutations.
The COSMIC database stores information on somatic mutations and associated information extracted from the scientific literature, while the CGC database is a catalog of cancer genes with manually screened somatic mutations extracted from the literature.
These cancer genes are annotated with information concerning chromosomal location, tumor types in which mutations are found, classes of mutations that contribute to oncogenesis, and other genetic properties.
Also, CGI is known to be created through an automated linguistic text analysis of millions of MEDLINE abstracts, with manual validation and annotation of the extracted data by expert curators.
It is a high-quality data resource consisting of genes that have been experimentally associated with human cancer diseases and/or pharmacological compounds, the evidence of these associations, and relevant annotations on the data.
Thus, it can also be a valuable resource to annotate mutation data from cancer genomics studies.
Currently, few specialized tools exist for an automated analysis of a long gene list from HGT-based cancer genomics.
While previously described cancer genomic information databases can be useful, they are not effective in processing results from NGS-based cancer genomic studies.
Meanwhile, there are many functional annotation systems to analyze gene lists derived from high-throughput microarray experiments, including DAVID [5], GoMiner [6], GOstat [7], Onto-express [8], and gene set enrichment analysis (GSEA) [9].
DAVID is one of the most popular general purpose functional annotation systems.
It provides various effective analysis tools and is also applicable in the analysis of gene lists from cancer genomics research.
However, DAVID is not optimized for processing results from HGT-based cancer genomics data, especially for NGS-based cancer genomics downstream data; therefore, a more specialized cancer gene annotation system is needed.
Here, we introduce a web-accessible cancer genome annotation system, named CaGe, to provide users with information on cancer genes, mutations, pathways, and associated annotations, based on several cancer gene databases composed of reported cancer-causing genes and associated cancer pathways.
For a given gene list, CaGe searches cancer gene databases with converted standard gene symbols, analyzes the biological pathways, and provides various cancer gene-related annotations through intuitive web user interfaces.
It also serves additional functions, including processing various input types and formats, managing jobs for user submitted tasks, and browsing for cancer genes and pathways with various useful hyperlinks between the annotations and the external public annotation databases.
We hope CaGe will be useful in identifying cancer-causing mutations and genes in HGT-based cancer genomics.
The goal of this study was to provide researchers with a tool for classifying their candidate genes from HGT-based cancer genome studies into previously reported or novel categories of cancer genes, while providing insight into underlying carcinogenic mechanisms through a pathway analysis.
To implement the cancer gene annotation function of CaGe, we constructed reported cancer gene and cancer annotation databases from public cancer genomic databases and cancer pathway-gene databases by pathway analysis with reported cancer gene sets and canonical pathways.
We also constructed a gene ID database to allow various input formats for the input of gene lists and a gene functional annotation database to provide users with functional clues for the annotated candidate genes.
Then, we developed a core retrieval program and web interfaces for the main functions, which include cancer gene annotation, cancer pathway annotation, cancer gene browsing, and cancer pathway browsing.
The workflow for the database construction and data processing in CaGe is summarized in Fig.1, and the cancer gene annotation page of the CaGe web interface is shown in Fig.2.
Title: Cancer gene and annotation data

To construct the reported cancer gene database, we used gene sets from the CGC database (released on Dec, 2010) and CGI database (downloaded on Feb, 2011).
The cancer pathways for the cancer pathway gene database construction were assigned based on statistical significance from one-tailed Fisher&#39;s exact test for overlapping genes between reported cancer gene sets and canonical pathways from public pathway databases, including KEGG (Release 57.0), BioCarta, and Reactome (downloaded on Feb, 2011).
We also created a gene ID database to convert various input identifiers into standard gene symbols with HUGO Gene Nomenclature Committee (HGNC) (downloaded on Feb, 2011) data for the standard gene symbols and with Entrez Gene (downloaded on Feb, 2011) and UniProt (Release 2011_03) data for the gene IDs, protein IDs, and functional annotations.
Title: Input types

Available input types include a list of gene symbols, Entrez gene IDs, or UniProt accessions and output files from prediction tools, such as Sorting Intolerant From Tolerant (SIFT) [10] and PolyPhen [11], which evaluate functional effects of mutations on proteins.
CaGe parses the output of those tools to extract the list of genes that have somatic mutations evaluated as functionally damaging, classify them into previously reported and novel candidate cancer genes, and conduct pathway analyses to give insights into underlying carcinogenic mechanisms.
Thus, CaGe offers straightforward data processing from HGT-based data without additional data conversion.
Title: Cancer gene and pathway annotation workflow

After acquiring user input, CaGe converts various input gene IDs into standard gene symbols, finds known cancer genes and pathways, links various cancer-related annotations to matched genes, and outputs them in the form of tables or text files through the web interface (Fig.3).
Another function of CaGe is to identify over-represented cancer-related or other biological pathways from the input gene list by performing one-tailed Fisher&#39;s exact test.
When processing more than one annotation job, users can manage their annotation tasks through the CaGe interface, and access to their results is maintained by the internal job database of CaGe until the jobs are deleted by the user or by a scheduled cleaning process.
The IP addresses of client computers are used for secured job management without a logon process.
Completed annotation jobs are listed on the job table, and the annotated results can be shown selectively by the user on the annotation result page or can be downloaded as tab-delimited text files for further analyses.
The annotation on the results page has many useful links to gene and cancer-related information in the CaGe database or external public databases.
In addition to the cancer gene and pathway annotation function, CaGe provides a function to browse cancer genes and pathways so that users can search cancer-related annotations without an input list.
The search flows of cancer genes and pathways are connected to each other by crosslinks in the cancer gene information page or pathway information page.
Title: Database and web interface

Based on the reported cancer-related gene sets from CGC and CGI, a cancer gene database and a cancer pathway gene database were constructed by the described methods.
In total, 436 CGC-originated and 7181 CGI-originated cancer genes were prepared for the reported cancer gene annotation.
Additionally, 5,790 CGC-based and 6,744 CGI-based cancer pathway genes were assigned for the annotation of unreported but potential candidate genes (Table 1).
The gene ID database for gene symbol conversion, based on HGNC, and the functional annotation database, based on Entrez Gene and UniProt database flat files, were constructed.
The types of annotations provided by CaGe are summarized in Table 2.
The main window of the CaGe web interface is shown in Fig.2 for the cancer gene annotation process.
Main menus, located in the upper part of the main window, are linked to the four main functions and home page of the system: 1) home page, 2) cancer gene annotation function, 3) cancer pathway annotation function, 4) cancer gene browsing function, and 5) cancer pathway browsing function.
Detailed usages for CaGe are described in the user&#39;s guide at http://mgrc.kribb.re.kr/cage/include/cageUserGuide.pdf.
Title: System information

CaGe was developed using PHP, R, and python languages; MySQL for database management; and Apache for the web server and is operated on a Linux platform with 8-core Intel Xeon CPUs (2.33 GHz) and 24 GB of main memory.
Title: Performance evaluation with small cell lung cancer mutation data

To assess the capability of CaGe, we annotated genes from candidate mutations for a small cell lung cancer genome (SCLC) [12].
We had 59 genes with predicted functionally damaging mutations after applying 22,910 mutations to the PolyPhen and could annotate 22 previously reported known cancer genes successfully by applying the PolyPhen output for CaGe.
Known cancer genes included RB1, which was mentioned as an SCLC-related gene in Pleasance&#39;s work; DST and ETS2, which were previously reported as SCLC-related genes but not mentioned in Pleasance&#39;s work; and 3 more genes (PDGFC, IL16, and AGTR2), which are known as lung cancer-related genes (Supplementary Table 1).
The other 16 genes are known to be related to other cancer types, suggesting that they might be important in the carcinogenesis of SCLC as well.
From the pathway analysis, we identified 12 known cancer genes and 11 genes in cancer-related pathways (Supplementary Table 2).
Those 11 genes might also be important in the carcinogenesis of SCLC.
Thus, we conclude that CaGe can annotate cancer genes effectively and suggest that CaGe will be useful in the identification of cancer-causing mutations and genes in HGT-based cancer genomics.
In this paper, we present a new cancer genomic tool, CaGe, for the assessment of candidate cancer genes having somatic mutations from HGT-based cancer genomics.
CaGe provides users with information on cancer genes, mutations, pathways, and associated annotations through cancer gene annotation, cancer pathway annotation, cancer gene browsing, and cancer pathway browsing functions.
It has a capacity to process SIFT or PolyPhen output files as direct input for usual NGS-based cancer genomics flows.
Researchers can classify their candidate genes from cancer genome studies into previously known or novel categories of cancer genes and gain insight into the underlying carcinogenic mechanisms through a pathway analysis.
We hope that CaGe will be useful for the identification of cancer-causing mutations and genes in HGT-based cancer genomics.
Construction and main service flows of CaGe.
SIFT, Sorting Intolerant From Tolerant; NGS, next-generation DNA sequencing; HGNC, HUGO Gene Nomenclature Committee; CGC, Cancer Gene Census; CGI, Cancer Gene Index.
Cancer gene annotation page of CaGe web interface.
Cancer gene annotation results page of CaGe web interface.
Title: Cancer gene sets and cancer pathway gene sets in the CaGe database

Title: Annotations for a cancer gene provided by CaGe
Title: Decreases in Casz1 mRNA by an siRNA Complex Do not Alter Blood Pressure in Mice

Recent genomewide association studies of large samples have identified genes that are associated with blood pressure.
The Global Blood Pressure Genetics (Global BPgen) and Cohorts for Heart and Aging Research in Genome Epidemiology (CHARGE) consortiums identified 14 loci that govern blood pressure on a genomewide significance level, one of which is CASZ1 confirmed in both Europeans and Asians.
CASZ1 is a zinc finger transcription factor that controls apoptosis and cell fate and suppresses neuroblastoma tumor growth by reprogramming gene expression, like a tumor suppressor.
To validate the function of CASZ1 in blood pressure, we decreased Casz1 mRNA levels in mice by siRNA.
Casz1 siRNA reduced mRNA levels by 59% in a mouse cell line.
A polyethylenimine-mixed siRNA complex was injected into mouse tail veins, reducing Casz1 mRNA expression to 45% in the kidney.
However, blood pressure in the treated mice was unaffected, despite a 55% reduction in Casz1 mRNA levels in the kidney on multiple siRNA injections daily.
Even though Casz1 siRNA-treated mice did not experience any significant change in blood pressure, our study demonstrates the value of in vivo siRNA injection in analyzing the function of candidate genes identified by genomewide association studies.
CASZ1 gene is a zinc finger transcription factor that controls cell fate, although little is known about its function [1].
CASZ1 is also known as a survival-related gene that has 2 isoforms.
It encodes a putative protein of 172 amino acids that localizes predominantly in the perinuclear region, as predicted for transcription factors.
CASZ1 is highly expressed in many human cancer cell lines, although its expression is low in most tissues, implicating it in tumor growth [2].
These findings indicate that CASZ1 is important for cell proliferation, development, and differentiation, as well as carcinogenesis.
However, CASZ1 has recently been reported to be significantly associated with blood pressure and hypertension in genomewide association studies (GWASs) [3].
Blood pressure GWASs by the Global Blood Pressure Genetics (Global BPgen) [4] and the Cohorts for Heart and Aging Research in Genome Epidemiology (CHARGE) consortium [5] have identified 14 independent loci that govern blood pressure or hypertension at a genomewide significance level, 7 of which were replicated in the Japanese population (CASZ1, MTHFR, ITGA9, FGF5, CYP17A1-CNNM2, ATP2B1, and CSK-ULK3) [6].
Single nucleotide polymorphism (SNP) rs880315, which lies in an intron of CASZ1, was one of the 7 replicate loci (p = 2.2 × 10-8, p = 4.9 × 10-12, and p = 3.0 × 10-7 for systolic blood pressure, diastolic blood pressure, and hypertension, respectively).
CASZ1 was also linked to blood pressure in the Women&#39;s Genome Health Study (rs880315; p = 5.2 × 10-9 for systolic blood pressure) [7].
Recently, the CASZ1 locus was identified in 19,608 subjects of east Asian ancestry by the Asian Genetic Epidemiology Network Blood Pressure (AGEN-BP) consortium [8].
Although these blood pressure GWASs have suggested that these loci control blood pressure, their function in blood pressure has not been examined.
Traditionally, knockout or transgenic mice have been used to reveal gene function and can be good approaches for examining candidate genes that lie near the GWAS loci [9].
However, this process is expensive and time-consuming [10], requiring technical advances to determine their function in vivo.
Small interfering RNA (siRNA) is an alternative approach that has the potential to become the preferred tool for that purpose [11].
In vivo injection of siRNA reduces gene expression by 70% to 80% in certain tissues in mice, demonstrating its efficacy [12].
However, specific tissues can not be targeted by in vivo injection of siRNA, and the injected siRNA is unstable in blood serum, limiting their use in delivering siRNA in vivo.
It also has certain disadvantages, such as effecting inflammatory responses and toxicity [13].
Thus, developing siRNA delivery methods in animal models is a challenging but worthy effort to accelerate the discovery of novel genes [11].
We sought to determine whether murine Casz1 controls blood pressure by administering Casz1 siRNA to mice.
A polyethylenimine-mixed siRNA complex was injected into tail veins, and changes in Casz1 mRNA and blood pressure were measured.
Title: Animals

BALB/c mice (Japan SLC, Inc., Shizuoka, Japan) were bred in a pathogen-free facility and used at 7 to 9 weeks of age for siRNA injection.
The mice were maintained on a 12-h light/dark cycle at a constant temperature with free access to food and water.
Every effort was made to minimize the number of animals that was used and their suffering per the Committee for the Care and Use of Laboratory Animals, College of Pharmacy, Kyung Hee University (KHP-2010-04-06).
Title: Synthesis of siRNA

Casz1 siRNA oligonucleotides were synthesized by Genolution (Seoul, Korea).
The target sequence for Casz1 was 5&#39;-GATGTGATCCGACATTACA-3&#39; (siRNA#1), 5&#39;-CTAGAACTGCTCTGTATGT-3&#39; (siRNA#2), and 5&#39;-CAATTCGGCAAGTAATGGA-3&#39; (siRNA #3), and nonspecific scrambled siRNA was used as a negative control.
To examine the efficacy of siRNAs, siRNAs were transfected into B16F10 cells using G-Fectin (Genolution); 1-2 µL of G-Fectin and 20 nM siRNA were added to 50 µL phosphate buffered saline, and the reaction was incubated for 10 min at room temperature (RT).
B16F10 cells were used to seed 24-well plates at 5 × 104 cells/well and 30% confluence, and the transfection mixture was added to the plate.
The cells were harvested for RNA extraction after 24 h of incubation.
Total RNA was prepared using TRIzol (Invitrogen, Carlsbad, CA, USA) per the manufacturer&#39;s instructions.
Title: Delivery methods

The transfection agent was in vivo-jetPEI™ (Polyplus, Illkirch-Graffenstaden, France).
Per the manufacturer, 25 µg siRNA and 6.5 µL vivo-jetPEI™ (N/P charge ratio of 6) were diluted with 50 µL 10% glucose solution and 50 µL sterile H2O.
The solution was vortexed gently and left for 15 min at RT.
The mixture was injected into the tail veins of 7- to 9-week-old mice, and the treated mice were measured for blood pressure or sacrificed for RNA extraction.
Title: RNA extraction and real-time PCR

RNA was extracted from the tissues of siRNA-treated mice with TRIzol (Invitrogen).
We synthesized cDNA from 500 ng of total RNA using the PrimeScript™ RT reagent kit (TaKaRa, Otsu, Japan) per the manufacturer&#39;s protocol.
Casz1 mRNA was analyzed by real-time PCR.
One-tenth of the cDNA reaction was added to a final volume of 20 µL for each reaction containing SYBR Green I (TaKaRa).
The primers for Casz1 and glyceraldehye-3-phosphate dehydrogenase (GAPDH) were 5&#39;-GTCTCTTCGGGAACTGCAAG-3&#39;/5&#39;-TGGGACACAGGCACTGTAGA-3&#39; and 5&#39;-GCTCTCTGCTCCTCCTGTTC-3&#39;/5&#39;-CAATACGACCAAATCCGTTG-3&#39;, respectively (forward/reverse sequence).
Quantitative real-time PCR was performed on an ABI Step One Real-time PCR system (Applied Biosystems, Foster, CA, USA) using the following program: 45 cycles of 95℃ for 10 s, 60℃ for 15 s, and 72℃ for 20 s. To normalize the amount of sample cDNA in each reaction, the Ct value of each target gene was subtracted by that of GAPDH to obtain delta Ct [14].
To calculate the fold-change in expression, the delta Ct value of the case sample was subtracted by that of the control (delta delta Ct = delta Ct case - delta Ct control).
Reverse-transcription PCR was performed similarly to quantitative real-time PCR, except that it used 35 cycles for amplification.
The PCR products were separated by electrophoresis in a 2.0% agarose gel and visualized under UV lamp after ethidium bromide staining.
Title: Blood pressure measurement

Blood pressure was recorded intraarterially with a computerized data acquisition system (ADInstruments, Bella Vista, Australia).
To place the intraarterial catheter, mice were anesthetized with avertin.
The standard dose was 0.014 mL/g avertin, which was made by diluting stock solution (for stock solution, 10 g 2,2,2, tribromoethanol dissolved in 10 mL tertiary amyl alcohol) 40-fold in 0.9% NaCl.
The intraarterial catheter, a polyethylene tube (0.2 mm I.D., 0.5 mm O.D.)
(Natsume, Tokyo, Japan) that was filled with 0.9% NaCl containing 100 U/mL heparin, was inserted into the right carotid artery and tied in place.
The blood pressure in the vessel was transmitted along the catheter to the transducer&#39;s diaphragm (MLT0699 Disposable BP Transducer; ADInstruments).
The diaphragm signal was amplified through a Bridge Amplifier and recorded on a PowerLab system (LabChart 7.2; ADInstruments).
Blood pressure was monitored for 2 h after the injection of the anesthetic and calculated as the average blood pressure between 45 min and 75 min after the injection (1 point per min).
Statistical analysis was performed using PASW Statistics version 18.0 (SPSS Inc., Chicago, IL, USA), and differences in blood pressure between case and control groups were analyzed by independent sample t-test.
Title: Reduction in Casz1 mRNA by siRNA injection into tail vein

We examined the efficacies of 3 Casz1 siRNA oligonucleotides in the B16F10 mouse melanoma cell line.
By real-time PCR, Casz1 mRNA levels fell 0%, 59%, and 54%, and we selected the second siRNA for the in vivo experiments.
Prior to the injection of siRNA, Casz1 expression in heart, lung, liver, kidney, spleen, and brain was measured by reverse-transcription PCR.
Casz1 was expressed in most tissues and at relatively low levels in liver (Fig.1).
Casz1 siRNA was mixed with polyethylenimine, and the complex was injected into the tail veins of mice.
At 24 h after siRNA injection, various tissues of siRNA-treated mice were analyzed with regard to Casz1 mRNA levels.
Casz1 mRNA expression was normalized to GAPDH, and a scrambled siRNA was used as control.
By quantitative real-time PCR, Casz1 mRNA levels declined only in kidney (Fig.2A).
Casz1 mRNA levels in the kidney fell by 45%, 55%, and 25% after 1, 2, and 3 injections per day, respectively (n = 3 for each group) (Fig.2B).
Title: Blood pressure in siRNA-injected mice

Blood pressure was measured 24 h after tail vein injection of the scrambled or Casz1 siRNA (control, n = 4; case, n = 3).
Using an intraarterial catheter that was connected to a pressure sensor, we recorded blood pressure continuously for 2 h after anesthesia.
Blood pressure did not change after a 1-time injection (p = 0.713).
Thus, we injected siRNA twice each day (control, n = 4; case, n = 6) but did not observe any significant difference between the case and control groups (p = 0.545).
After 3 siRNA injections daily for 3 days, blood pressure was also unaffected (p = 0.297) (Table 1, Supplementary Table 1).
This study attempted to validate CASZ1 as a regulator of blood pressure using siRNA in mice.
Although Casz1 mRNA levels decreased by 55% in kidney, blood pressure did not change on injection of the siRNA complex into tail veins.
It is difficult to conclude that CASZ1 does not control blood pressure, because this study has several limitations.
First, Casz1 mRNA levels in kidney were reduced up to 55%; if Casz1 expression had been abolished, we might have detected a significant difference in blood pressure between the case and control groups.
Second, it is possible that CASZ1 functions in other tissues besides the kidney in the regulation of blood pressure.
The kidney is an essential organ of the urinary system and controls homeostatic functions, such as the regulation of blood pressure, by maintaining salt and water balances.
However, the function of CASZ1 in kidney is unknown due to the lack of Casz1 functional studies in kidney.
Casz1 is highly expressed in brain, but siRNA nanoparticles do not access the brain due to the blood-brain barrier [15].
Third, human CASZ1 is 87% and 90% identical to mouse Casz1 in the cDNA and polypeptide sequences, respectively [1].
Nevertheless, it is possible that Casz1 function differs between the two species.
Because most loci that have been identified by GWASs have several genes nearby, it may be necessary to examine many candidates to identify the functional gene.
Knockout and transgenic mice are expensive and time-consuming to generate.
In contrast, siRNA injection costs less and is much faster to perform, making siRNA injection suitable for functional validation studies of GWAS results.
In summary, we did not observe any significant change in blood pressure in Casz1 siRNA-treated mice, but our study demonstrates the value of in vivo siRNA injection in the functional analysis of candidate genes from GWASs.
Casz1 mRNA expression in normal mouse tissues.
GAPDH, glyceraldehye-3-phosphate dehydrogenase.
Decreases in Casz1 mRNA by an siRNA complex.
(A) Fold-change in Casz1 mRNA levels in tissues in siRNA-injected mice (n = 2 per tissue).
(B) Reduction in Casz1 mRNA levels in kidney by in vivo siRNA injection.
Title: Blood pressure in siRNA-injected mice
Title: Efficient Mining of Interesting Patterns in Large Biological Sequences

Pattern discovery in biological sequences (e.g., DNA sequences) is one of the most challenging tasks in computational biology and bioinformatics.
So far, in most approaches, the number of occurrences is a major measure of determining whether a pattern is interesting or not.
In computational biology, however, a pattern that is not frequent may still be considered very informative if its actual support frequency exceeds the prior expectation by a large margin.
In this paper, we propose a new interesting measure that can provide meaningful biological information.
We also propose an efficient index-based method for mining such interesting patterns.
Experimental results show that our approach can find interesting patterns within an acceptable computation time.
Biological sequences, such as DNA sequences, have a great number of contiguous patterns consisting of frequent items.
Which patterns are interesting to biologists?
Is a pattern that occurs more frequently more interesting?
So far, in most approaches, the number of occurrences has been a major measure of determining an interesting pattern.
This measure, however, is not enough to discriminate a pattern from the background noise and may induce much time to spend for checking patterns of no biological meaning.
Researchers are more interested in contiguous patterns that are statistically significant than those simply occurring frequently.
The aim of mining interesting patterns is to analyze the important biological functions hidden in the extremely large amounts of genomic sequences.
In this work, we aimed to discover surprising contiguous patterns that occur at a frequency higher than their expected frequencies.
To find such surprising patterns with confidence, we chose to use a more suitable measure, information [1], which is widely studied and used in the field of communication.
In information theory, if a pattern is expected to occur frequently based on some prior knowledge or by chance, then an occurrence of that pattern carries less information.
Thus, we can use information to test the surprise of an occurrence of a pattern.
The information gain is introduced to denote the accumulated information of a pattern in a DNA sequence and is used to exhibit the degree of surprise of the pattern.
Many works for sequential pattern mining take an a priori approach, such as Agrawal and Srikant [2], who used downward closure property to prune infrequent patterns, which says that if a pattern is infrequent, all of its superpatterns must be infrequent.
It suffers from the level-wise difficulty for candidate generation-and-test and needs several database scans for sequential pattern mining.
A typical Apriori-like approach such as Generalized Sequential Patterns (GSP) [3] is a good example of this category.
An efficient algorithm, PrefixSpan [4], has been proposed, representing projection-based sequential pattern mining.
This approach examines only the prefix subsequences and projects only their corresponding postfix subsequences into the databases.
Sequential patterns are grown by exploring length-1 frequent patterns in each projected database.
Using the projected database, however, every expansion of sequential patterns needs a recursive process, which is not effective for DNA sequence mining.
The problem of finding the frequent maximal contiguous pattern from sequences more than two has been introduced in [5-8].
In addition, Yang et al.
[9] have proposed an interesting technique to find periodic patterns in a sequence of events.
Lu et al.
[10] have proposed a pattern discovery algorithm that can identify over-represented patterns inside DNA sequences by introducing a new measurement system.
Another efficient algorithm, MacosVSpan [11], has been proposed, which gets the maximal subsequence of each data item by fixed-length spanning and finally looks for frequent maximal contiguous patterns using a suffix tree.
Although this approach reduces recursive execution for expanding sequence patterns, it also suffers from the problem of producing and using projected databases.
For long data sequences, the projected database grows much faster in comparison with the original database.
Kang et al.
[12] have proposed an approach to improve MacosVSpan using a fixed-length spanning tree, where each node maintains the frequency of subsequence overlapping.
In this approach, all the candidates are produced first, including frequent and nonfrequent patterns; then, each candidate is scanned through the database to see whether it is frequent or not.
This is obviously very time and memory-consuming.
Recently, Zerin et al.
[13] proposed a position-based fast method to find contiguous frequent patterns, which needs to scan the database only once to construct the fixed length spanning tree.
This approach builds the fixed length spanning tree in the same fashion as Kang et al.
[12] but records the sequence identification (ID) and starting position of the fixed length pattern with the frequency in the leaf node of the tree, showing better results than the previous methods.
Rashid et al.
[14] have also proposed an efficient approach to mining significant contiguous frequent patterns from DNA sequences by constructing the fixed length spanning tree and by using a threshold that reduces the number of candidates.
In this paper, we further develop this approach by proposing an index-based method, where the sequence ID and the staring position of each sequence are recorded in the leaf node of the tree as a variable length array.
If a fixed-length pattern occurs multiple times in a sequence, we put the sequence ID as an index and put the start positions of the fixed length patterns in the variable length array.
As a result, this approach significantly reduces memory space more than Zerin et al.[13].
Title: Concepts and definitions

Title: Let Σ = {A, C, G, T} be a set of DNA alphabets, where A, C, G, and T are called DNA characters or bases.
A DNA sequence S is an ordered list of DNA alphabets.
S is denoted by 〈c1, c2, ..., cl〉, where ci ∈ Σ and |S| denotes the length of sequence S. A sequence with length n is called an n-sequence.
A sequence database D is a set of tuples 〈sid, S〉, where sid is a sequence ID.
The sum of the lengths of all sequences in D is denoted as |D|=|S1|+|S2|...|Sn|

Title: Definition 1 (Patter)

A pattern is a contiguous sub-sequence of DNA sequence S drawn from Σ = {A, C, G, T}.
A sequence α = 〈a1, a2, ..., an〉 is called a contiguous sub-sequence of another sequence β = 〈b1, b2, ..., bm〉, and β is a contiguous super-sequence of α, denoted as α⊆β, if there exists integers 1 ≤ j1 ≤ j2 ≤ ... ≤ jn ≤ m and ji+1 = ji + 1 for 1 ≤ i ≤ n-1 such that a1 = bj1, a2 = bj2, ..., an = bjn.
We can also say that α is contained by β.
In our paper, we use the term &#34;(sub)-sequence&#34; to describe &#34;contiguous (sub)-sequence&#34; in brief.
Title: Definition 2 (Support)

Given a pattern P and a sequence S, the number of occurrences of P in S is called the support of pattern P in sequence S, denoted as Sup(P, Si).
For DNA sequence database D, the support of P in D is defined as .
Title: Definition 3 (Confidence)

Given a pattern P = P1, P2, ..., Pq and a DNA sequence database D, the confidence of P1P2 with respect to P1 is defined as Conf(P1P2, P1) = Sup(P1P2,D)/Sup(P1,D).
For example, the character &#34;A&#34; occurs 10 times and &#34;AT&#34; occurs 7 times, and in database D in Table 1, Conf(AT,A) = 7/10 = 0.7.
Title: Definition 4 (Pattern probability)

Given a pattern P = P1, P2, ..., Pq (Pi is a DNA alphabet) and a DNA sequence database D, the pattern probability of P in D is defined as , where Pr(Pi, D) = # of occurrences of an alphabet Pi/|D|.
For example, the pattern probability of pattern &#34;ATCG&#34; in Table 1 is Pr(ATCG,D) = Pr(A,D) × Pr(T,D) × Pr(C,D) × Pr(G,D) = (10/55) × (18/55) × (12/55) × (15/55) = 0.182 × 0.372 × 0.218 × 0.273 = 0.00403.
Title: Definition 5 (Information)

The information carried by a DNA character or base in DNA sequence database D is defined as I(c) = -log|C|Pr(c,D), where |C| is the number of distinct characters in D and Pr(c) is the probability of c occurs in D.
For example, the occurrence probability of character A in Table 1 is Pr(A,D) = # of occurence(A)/|D|.
So, the probability of character A is Pr(A,D) = 10/55 = 0.182 in our example database.
Then, the information of character A in D is, I(A) = -log|C|Pr(A,D) = -log4(0.182) = 1.228.
Title: Definition 6 (Pattern information)

Given a pattern P = P1, P2, ..., Pq and a DNA sequence database D, the pattern information of P in D is defined as I(P) = -log|C|Pr(P,D) = I(P1) + I(P2) +......+ I(Pq).
For example, the pattern information of pattern &#34;ATCG&#34; is I(ATCG,D) = I(A,D) + I(T,D) + I(C,D) + I(G,D) = 1.228 + 0.713 + 1.098 + 0.9365 = 3.9755 in Table 1.
Title: Definition 7 (Information gain)

Given a pattern P = P1, P2, ..., Pq and a DNA sequence database D, the pattern information gain of P in D is defined as IG(P) = I(P) × Support(P).
For example, the information gain of pattern &#34;ATCG&#34; is IG(ATCG,D) = 3.9755 * 5 = 19.8775 in Table 1.
Title: Definition 8 (Finding interesting patterns)

Given a sequence database D and user-specified min_conf and min_in_gain, the problem of finding interesting patterns is to find the complete set of interesting patterns, such that IG(P) and Conf(P) are greater than min_in_gain and min_conf , respectively.
Title: Surprising pattern-mining algorithm

The method for mining surprising contiguous patterns from a DNA sequence database proceeds as follows.
For constructing the fixed length spanning tree, we follow the method suggested by Kang et al.
[12] - that is, read the database sequences and, starting from the first of each sequence, move the position one by one of the fixed length window throughout the sequence (Fig.1).
We put the sequence ID and the starting position in the leaf node of the tree as a variable length array, in addition to the frequency of the pattern moving of the window.
Fig.2 shows our proposed algorithm.
It has two steps.
The first step recursively scans the subsequence with the same length as the fixed-length window from the given sequences to construct a fixed-length spanning tree and put the database sequence ID and starting position of the pattern in the leaf node of tree as a one-side open variable length array.
At this time, it calculates the character probabilities for A, C, G, and T, respectively.
It is straightforward for the calculation of character probabilities for A, C, G, and T. By adding all occurrences of A, C, G, and T and dividing by the total number of characters in the database, it can be obtained easily.
After calculating character probabilities, we can calculate character information, pattern information, and pattern information gain by using definition 5, definition 6, and definition 7.
The second step generates fixed-length patterns from the constructed spanning tree with satisfying the minimum information gain threshold and minimum confidence threshold, and expands sequences by joining candidates and checking the frequency and starting position of the candidates.
At the time of joining the sequences with the same length to generate the candidates of the next length, it checks whether the second candidate starts right to the next position of the first one with the same ID or not.
If so, it increases the frequency counter by one.
Finally, it checks the minimum information gain and minimum confidence threshold.
If the pattern satisfies both thresholds, then it is added to the next-length candidate pattern.
This process is recursively followed for all the candidates with the same length to generate the next-length surprising candidates.
The spanning tree is shown in Fig.3, which is constructed based on the database available in Table 1.
We have constructed a fixed-length spanning tree using the method suggested by Zerin et al.
[13] but put the sequence ID and the staring position in the leaf node of the tree as a variable length array.
Once the tree is constructed like Fig.3, retrieval of the tree can obtain contiguous subsequences with length-4, satisfying the satisfying minimum information gain threshold and minimum confidence threshold.
Then, the obtained length-4 surprising contiguous patterns are 〈ATCG〉, 〈TCGT〉, 〈TGAT〉, 〈CGTG〉, 〈CGTT〉, 〈CATC〉, and 〈GTGA〉, shown in Fig.4.
To generate the length-5 surprising contiguous patterns, we need to join the length-4 surprising patterns; all the items in the first pattern, excluding the first item, should be same as all the items of the second pattern, excluding the last item.
The frequency counter of the next length pattern will increase if both of the patterns are present in the same sequence and the second pattern&#39;s starting positions are the right next to the position of the first pattern in the same sequence.
If the generated patterns satisfy the min_in_gain and min_conf, we consider the new pattern as a next length surprising candidate.
The obtained length-5 surprising patterns are 〈ATCGT〉, 〈TCGTG〉, 〈TCGTT〉, 〈CGTGA〉, 〈CATCG〉, and 〈GTGAT〉.
From Fig.2, we see pattern ATCG starts from the 1st and 11th positions of sequence 10, 2nd and 9th positions of sequence 20, and 2nd position of sequence 30.
The pattern TCGT starts from 3rd and 10th positions of sequence 20, 3rd position of sequence 30, and 1st position of sequence 40.
As a result, the length-5 candidate ATCGT starts from the 2nd and 9th positions of sequence 20 and the 2nd position of sequence 30.
So, the information gain of ATCGT is 14.065, and the confidence of ATCGT with respect to ATCG is 0.6, which satisfy both min_in_gain and min_conf thresholds.
Following the same process, the obtained length-6 surprising patterns are 〈CGTGAT〉, 〈CATCGT〉, and 〈TCGTGA〉, and the obtained length-7 surprising pattern is 〈TCGTGAT〉.
Here, we scan the whole database only once to construct the fixed-length spanning tree and then never scan the database again.
On the other hand, using an index-based method and min_in_gain and min_conf thresholds, our proposed approach consumes less memory and faster execution than Zerin et al.[13].
We compare the performance of the proposed approach with Zerin et al.
[13] for searching contiguous subsequences.
For this purpose, we use a randomly generated DNA sequence database and a real DNA sequence database.
We first generated a DNA sequence database by means of C++ program that randomly generates the variable length datasets.
This database contains 5,000 DNA sequences with lengths from 100 to 1,000.
The real DNA sequence database is downloaded from the bio-mirror portal (http://www.bio-mirror.net), which contains 19,979 DNA sequences, with average sequence length 1,024.
All programs are written in Microsoft Visual C++ 6.0 and run with the Windows XP operating system on a Pentium dual core 2.13 GHz CPU with 1 GB main memory.
In the first experiment, we compare the memory usage of our approach and Zerin et al.
[13] by varying the sequence length.
Fig.5 shows the memory usage according to the sequence length change, where we used our generated database for construction of the spanning tree with a fixed length 7.
Here, we used information gain threshold, min_in_gain = 35(%) and minimum confidence threshold, min_conf = 30(%).
From this experiment, we can see that the proposed approach has efficient improvement over Zerin et al.[13].
When the sequence length becomes longer, it shows better performance in comparison with the existing algorithm.
The memory consumption of our proposed approach and [13] for different values of min_in_gain over a real DNA sequence database is shown in Fig.6.
The x-axis in the graph indicates the change in min_in_gain as a percentage of the data point.
A tree with a fixed length-10 was constructed using the aforementioned real datasets, and min_conf value 0.35 is taken.
Fig.6 indicates that for increasing the value of min_in_gain for both approaches, fewer candidates are generated and less memory is required.
The second experiment examined mining time performance according to change in sequence length.
Fig.7A shows the mining time of the surprising contiguous patterns, starting from length-4 to length-9, in a randomly generated DNA sequence database, where we used information gain threshold min_in_gain = 35% and min_conf = 30%.
On the other hand, we performed the same experiment with real DNA sequence datasets, where we used the value of min_in_gain = 45% and min_conf = 40%, which is shown in Fig.7B.
From Fig.7, we can see that our proposed approach could mine the surprising contiguous patterns within a reasonable computation cost.
The third experiment shows the effect of information gain threshold on mining time to find out the surprising contiguous patterns up to length 8.
In this experiment, we take min_conf 0.3 and 0.4 for the random and real datasets, respectively.
Fig.8 indicates that increasing the information gain threshold decreases mining time for both random and real datasets.
The fourth experiment studied the impacts of varying minimum confidence from 0.2 to 0.5 for random datasets and 0.25 to 0.55 for real datasets.
This time, we take min_in_gain 0.3 and 0.4 for random and real datasets, respectively.
Fig.9 shows our proposed performance with different values for minimum confidence and illustrates a non-linear effect.
The greatest change along the confidence axis is from 0.2 to 0.3 for random datasets and 0.25 to 0.35 real datasets.
In most cases, the characters are evenly distributed.
This means that the A, C, G, and T occur at almost the same ratio in the dataset as the frequency of each character, which is approximately 25%.
So, if the min_conf is set to 0.3, most random patterns will be filtered out, and real patterns occurring more frequently than 25% of the time will survive and be extended.
To summarize, we have developed an index-based method, where we need to scan the database only once to mine the surprising contiguous patterns in biological data sequences, which are considered very important in bioinformatics and computational biology.
Our aim is not to discover the patterns that occur often but rather to find patterns that are surprising by introducing a new threshold information gain.
It has been shown by the experimental results that the proposed approach is very efficient in finding interesting patterns within a reasonable computation cost.
For future work, we will try to optimize the proposed approach by considering a variety of environments with different parameters and also consider promoting new measurement parameters, which is very feasible for describing the sequences in a biological substance.
Fixed length scanning method.
Surprising contiguous pattern mining algorithm.
Index-based fixed-length spanning tree.
Mining length-4, length-5, length-6, and length-7 surprising patterns.
Memory usage w.r.t.
change of sequence length.
Memory usage w.r.t.
change of min_in_gain.
(A, B) Impact of pattern length in mining time.
(A, B) Impact of information gain threshold on mining time.
(A, B) Impact of confidence threshold on mining time.
Title: Example of a DNA sequence database
Title: An Efficient Approach to Mining Maximal Contiguous Frequent Patterns from Large DNA Sequence Databases

Mining interesting patterns from DNA sequences is one of the most challenging tasks in bioinformatics and computational biology.
Maximal contiguous frequent patterns are preferable for expressing the function and structure of DNA sequences and hence can capture the common data characteristics among related sequences.
Biologists are interested in finding frequent orderly arrangements of motifs that are responsible for similar expression of a group of genes.
In order to reduce mining time and complexity, however, most existing sequence mining algorithms either focus on finding short DNA sequences or require explicit specification of sequence lengths in advance.
The challenge is to find longer sequences without specifying sequence lengths in advance.
In this paper, we propose an efficient approach to mining maximal contiguous frequent patterns from large DNA sequence datasets.
The experimental results show that our proposed approach is memory-efficient and mines maximal contiguous frequent patterns within a reasonable time.
Mining patterns from DNA sequences refers to the task of discovering sequences that are similar or identical between different genomic locations or different genomes.
How to efficiently discover long frequent contiguous sequences poses a great challenge for existing sequential pattern mining algorithms.
The problem of finding the maximal contiguous frequent pattern is important in bioinformatics and has been used for predicting biological functions held in genomic sequences [1-4].
In the beginning, the problem of finding common subsequences from sequences of more than 2 was studied [1, 2]; then, many tried to solve more general sequential pattern mining problems.
A typical a priori algorithm, such as Generalized Sequential Pattern algorithm (GSP) [5, 6], adopts a multiple- pass, generation-and-test approach.
In one pass, all single items (1-sequences) are counted.
From the frequent items, a set of candidate 2-sequences are formed, and another pass is made to identify their frequency.
The frequent 2-sequences are used to generate the candidate 3-sequences, and this process is repeated until no more frequent sequences are found.
Normally, a hash tree-based search is employed for efficient support counting.
Finally, non-maximal frequent sequences are removed.
Based on this idea, a more efficient algorithm, called PrefixSpan [7], has been proposed, which examines only the prefix subsequences and projects only their corresponding postfix subsequences into the projected database (PDB).
In each PDB, contiguous sequences are grown by exploring local length-1 frequent sequences.
A memory-based pseudo-projection technique is developed to reduce the cost of projection and speed up processing.
When mining long frequent concatenated sequences, however, this method becomes inefficient in terms of time and space.
Therefore, it is impractical to apply PrefixSpan to mine long contiguous sequences, like biological datasets.
Pan et al.
[8] proposed to use a variable length spanning tree to mine maximal concatenated frequent subsequences and developed algorithms, called MacosFSpan and MacosVSpan, based on the PrefixSpan approach.
MacosVSpan is efficient for mining long concatenated frequent sub-sequences, whereas MacosFSpan has some limitations, because it constructs length-4 fixed length candidate sequences first and recursively mines length-5, length-6 candidate sequences, etc.
This is very time consuming, because in a practical DNA sequence database, a sub-sequence may occur multiple times in the same sequence.
Another problem is that both MacosVSpan and MacosFSpan use the pointer-offset pair to represent the suffixes inside the in-memory pseudo PDB, which is not enough to represent the huge number of suffixes in a physical PDB.
Kang et al.
[9] have claimed that their algorithm is more efficient than the MacosVSpan algorithm, but it was based on MacosFSpan, which needed multiple scans of the database.
Recently, Zerin et al.
[10] have proposed an indexed-based method to find the frequent contiguous sequences with single database scanning.
This approach builds the fixed-length spanning tree in a way similar to Kang et al.
[9], but it records the sequence IDs and starting position of the fixed length patterns with frequency in the leaf node of the tree.
Although this approach shows better results than its predecessors in terms of space, time complexity is still not acceptable.
A practical DNA sequence database contains a large number of sequences, where a sub-sequence may occur multiple times in the same sequence.
Therefore, we envisage using the concept of the suffix tree in terms of the variable length spanning tree of Pan et al.
[8] to have the full advantages of prefix matching.
Another aspect to consider is the size of real DNA sequence databases, which is ever increasing.
For the cases where a DNA sequence database can not fit into the main memory, disk-based mining has been studied, based on partitioning [11-14].
Most of these techniques, however, only consider local frequency counting, although many frequent patterns may look infrequent due to local support pruning.
In this paper, we propose a suffix tree-based approach for maximal contiguous frequent sub-sequence mining from DNA sequence datasets by means of a variable length spanning tree.
We also introduce a combined main memory- and disk-based approach for mining maximal contiguous frequent patterns from extra large DNA sequence databases that can not fit into the main memory.
Title: Concepts and definitions

Let Σ = {A, C, G, T} be a set of DNA alphabets, where A, C, G, and T are called DNA characters or four bases; A is for adenine, C for cytosine, G for guanine, and T for thiamine.
A DNA sequence S is an ordered list of DNA alphabets.
S is denoted by 〈c1, c2... cl〉 where ci ∈ Σ and |S| denotes the length of sequence S. A sequence with length n is called an n-sequence.
DNA sequence database D is a set of tuples 〈Sid, S〉 where Sid is a sequence identifier and S is the corresponding sequence.
A sequence α = 〈a1, a2,..., an〉 is called a contiguous sub-sequence of another sequence β = 〈b1, b2, ..., bm〉, and β is a contiguous super-sequence of α, denoted as α⊆β, if there exists integers 1 ≤ j1 ≤ j2 ≤ ... ≤ jn ≤ m and ji+1 = ji + 1 for 1 ≤ i ≤ n-1 such that a1 = bj1, a2 = bj2, ..., an = bjn.
We can also say that α is contained by β.
In our paper, we use the term &#34;(sub)-sequence&#34; to describe a &#34;contiguous (sub)-sequence&#34; in brief.
A contiguous frequent sub-sequence X is said to be maximal if none of its super-sequences Y is frequent.
Given a DNA sequence database D and a minimum support threshold δ, the problem of maximal contiguous frequent sub-sequence mining is to find the complete set of maximal contiguous frequent patterns from that database.
For example, let our running DNA sequence database be D in Table 1, and the minimum support threshold δ ≥ 2.
Sequence 〈ATCGTGACT〉 is 9-sequence since its length is 9.
Sequence 〈ATCG〉 is a contiguous frequent sub-sequence because it is contained by sequences 10, 20, and 30.
S = 〈CGTGATT〉 is a frequent contiguous sub-sequence of length 7 since both sequences 40 and 50 contain it.
Moreover, it is one of the maximal frequent contiguous sub-sequences, because there is no contiguous frequent super-sequence of 〈CGTGATT〉 with a minimum support of 2.
Title: Definition (Projection)

Let a sequence S = (α + β); then, α is a prefix of S and β is called a projection of S w.r.t.
prefix α.
Title: Definition (Projected database)

Let α be a frequent contiguous sequence in a sequence database S. Then, α-PDB is the collection of postfixes of sequences in S, w.r.t.
prefix α.
For the database in Table 1, for example, 〈C〉-PDB consists of eight postfix sub-sequences: 〈GTGACT〉, 〈T〉, 〈ATCGTT〉, 〈GTT〉, 〈ATCGTGAGA〉, 〈GTGAAG〉, 〈GTGATTG〉, and 〈GTGATT〉.
We have the following lemma regarding the PDB for the mining of maximal contiguous frequent patterns.
Title: Lemma (Projected database)

Let D be a DNA sequence database such that α is prefix(s) of the sequences in the database.
Considering the sub-sequence and super-sequence relationship in terms of contiguity, the size of α-PDB cannot exceed the size of D, which means |Dα| ≤ |D|.
Title: Proof

The lemma is regarding the size of a projected database.
Obviously, the α-PDB can have same suffix sequences as the original database D only if α appears in every sequence in D. Otherwise, if only those suffixes in D that are super-sequences of α will appear in the α-PDB, then α-PDB cannot contain more sequences than D. For every sequence γ in D such that γ is a super-sequence of α, γ appears in the α-PDB in whole only if α is a prefix of γ.
Otherwise, only a sub-sequence of γ appears in the α-PDB.
Therefore, the size of α-PDB cannot exceed the size of D.
For example, if we consider the 〈A〉-PDB from the example database, it contains 9 suffixes: TCGTGACT, CT, TCGATTG, TTG, TCGTGAGA, GA, TTG, TTACT, and CT.
So, can we see that the suffixes CT, TTG, and GA sub-sequences among the suffixes.
So, in terms of contiguity, TCGTGACT, TCGATTG, TCGTGAGA, and TTACT are the super-sequences.
Therefore, 〈A〉-PDB contains only four suffixes.
So, the size of the PDB cannot exceed the size of original database.
This claims that any PDB created from a database that fits into the main memory will certainly be fitted into the main memory.
Title: Maximal contiguous frequent suffix tree algorithm

We describe our algorithm, called Maximal Contiguous Frequent Suffix tree algorithm (MCFS), to discover maximal contiguous frequent patterns from DNA sequence datasets.
Let us assume for now that the DNA sequence database can be fit into the main memory.
The method proceeds as follows: 1) construct four projected databases, namely 〈A〉, 〈T〉, 〈C〉, and 〈G〉, to mine contiguous frequent patterns; 2) insert the suffixes of PDBs into a suffix tree; 3) traverse the whole suffix tree and find the set of frequent contiguous patterns; 4) check the properties of maximality and find the set of maximal contiguous frequent patterns.
In order to efficiently discover frequent sub-sequences, we design a suffix tree structure that stores all the potential sub-sequences and their corresponding supports.
A &#39;potential frequent sub-sequence&#39; is a sub-sequence that has at least two matches with other suffixes inside the PDBs.
Each PDB has its own corresponding sub-tree, defined as follows.
The root of the tree is the entry point.
Each node consists of three fields: item, support, and link to the next node.
Item registers which item this node represents.
Concatenation of all items along the path from the root to this node represents a sub-sequence.
Support registers the support of this suffix sub-sequence from the root node to this node.
This means that if we traverse a particular path from top to bottom, then the path from the root to this node represents a sub-sequence, and support of the last node in the path becomes the support of the sub-sequence.
The height of the suffix tree is not fixed, because the lengths of the suffixes are variable.
Our proposed algorithm (Fig.1) proceeds in three steps.
The first step scans the original database and constructs four PDBs.
While creating PDBs of A, T, C, and G, a register is maintained that checks for duplicate suffixes without sorting the suffixes.
If a suffix appears more than once in a PDB, the suffix is inserted into a temporary buffer with support.
If a suffix X has n replicas in α-PDBs, for example, we move X to the temporary buffer with support n. This kind of suffix uses very little space and thus can be stored in the memory.
Then, we check the suffixes and merge the sub-sequence suffixes with super sequence suffixes according to lemma 1.
In this way, the size of physical PDB does not exceed the size of the original database.
Inside the PDBs, we do not register the value of sequence ID or offset, because we can directly insert suffixes into the suffix tree.
The second step creates and inserts suffixes into suffix tree.
During the insertion process, four situations arise regarding the maximality problem.
First, if the inserting sequence is contained by some suffixes in the PDBs, it will match a branch of the suffix tree; then, we delete it from the result set and increase support up to the matching point to stop inserting.
Second, if the inserted sequence contains some sequence in the tree, denoted as α, one of its suffixes will match the branch representing α; then, we increase the support of α and delete it from the result set and continue to insert other suffixes.
Third, if there exists a suffix totally matching a prefix, say α of a PDB, we continue to find the longest subsequence β following α in S, which matches a root-path in α-subtree.
Fourth, if there exists a suffix α that partly matches a prefix, we merge it with the suffix and increase the corresponding support up to the matching point.
Otherwise, we just insert it into the suffix tree as a new path.
If a suffix has no matches with other suffixes inside a PDB, we just discard the suffix after the traversing.
The third step scans the suffix tree and finds the set of all maximal contiguous frequent patterns.
It first finds the set of contiguous frequent sub-sequences and then finds the set of maximal contiguous frequent patterns by checking the properties of maximality.
Title: An illustrative example

Let us demonstrate the MCFS algorithm using the database of Table 1.
〈A〉-PDB contains only super-sequences of suffixes storing all sub-sequences physically.
According to lemma 1, 〈A〉-PDB contains 4 suffixes: TCGTGACT, TCGATTG, TCGTGAGA, and TTACT.
ACT and ATTG will be moved to &#39;frequent file&#39; with their corresponding supports 2.
The remaining three projected databases can be constructed in a similar manner (Table 2).
Now, the suffixes are inserted into the suffix tree.
Fig.2 shows the suffix sub-tree that can be traversed to find the set of contiguous frequent patterns.
The figure illustrates the mining technique of maximal contiguous frequent patterns from A-subtree.
Mining from T, C, and G-subtrees can be done in a similar way.
The traversal of the suffix sub-tree proceeds as follows.
First, we visit the paths from root to every descending node, and we write the frequent contiguous sub-sequences in the frequent file.
Table 3 shows the content of the frequent file with corresponding supports.
The frequent file previously contained four frequent contiguous sub-sequences with their support, which are ATTG (2), ACT (2), CT (2), and GATTG (2).
After the final traversal on the sub-trees, the contents of the frequent file will be updated.
Now, we scan the &#39;frequent file&#39; presented by Table 3 and check if the maximality criteria are met.
Finally, we have five maximal contiguous frequent patterns written into the output file (Table 4).
Title: MCFS algorithm on extra large DNA sequence database

When a DNA sequence database is too large (e.g., 100 GB or more) to fit into the memory, we have to store it on a disk.
In this case, different mining techniques are needed, and partitioning is one such technique.
Most disk-based partitioning techniques [11-14] find frequent patterns from each partition and check to discover all frequent patterns.
This approach, however, has some drawbacks, because frequent patterns may look infrequent due to local support pruning.
Suppose our database is partitioned into two parts, D1 and D2.
Sequences 10, 20, and 30 are in D1, and 40 and 50 are in D2.
Suffix ACT occurs once in D1, so it is not frequent.
According to them [12, 13], we have to discard it for local support pruning.
Another copy of ACT is in D2 (ID-50), so we discard it from the partition as well.
If we consider globally, then ACT will be a frequent pattern, if the minimum support threshold is 2.
In this way, many frequent patterns can be lost by partitioning.
In order to deal with this problem, we propose a technique using a combined approach of main memory and disk partitioning.
During the first scanning over the database, we partition the original database residing in the disk into smaller parts so that each part can fit into the memory.
In this process, the number of partitions should be minimized by reading as many DNA sequences into the memory as possible to constitute one partition.
The set of frequent patterns in D is obtained by collecting the discovered patterns after running MCFS on these partitions.
The actual maximal contiguous frequent patterns can be identified with only one extra database pass through support counting against all the data sequences in each partition, one at a time.
Therefore, we can employ MCFS to mine databases of any size, with any minimum support, in just two passes of database scanning - one for the original database and one for the portioned databases.
Firstly, store every frequent pattern into a temporary buffer storage, namely &#39;frequent file,&#39; with their corresponding support, and infrequent patterns into temporary buffer storage, namely &#39;output file,&#39; with their corresponding support as well.
After that, we collect all the infrequent patterns from each partition and combine them to count the corresponding support of each infrequent pattern.
The main goal of this approach is to process one partition in the memory at a time to avoid multiple scans over D from the secondary storage.
Fig.3 indicates the technique of mining contiguous frequent patterns from disk-based extra large database.
In the figure, D1, D2, ..., Dn are the partitions of the original database.
Fi is the contiguous frequent patterns, and IFi is the infrequent contiguous patterns from each partition.
CFP is contiguous frequent patterns, and MCFPi is maximal contiguous frequent patterns.
Fi, IFi, CFP, and MCFPi are all stored on the disk.
We compare the performance of our MCFS algorithm with MacosVSpan for mining maximal contiguous frequent sub-sequences.
In this comparison, we only consider the memory-based approach, as done by most existing works.
All programs were written and compiled using Microsoft Visual C++ 6.0, run with Microsoft Windows XP with a Pentium Duel Core 2.13 GHz CPU with 4 GB of main memory and 500 GB hard disk.
As for practical DNA sequence databases, &#39;Human genome&#39; (Homo sapiens GRCh37.64 DNA Chromosome Part 1, 2, 3) and &#39;Bacteria DNA sequence dataset&#39; were downloaded from the NCBI website (http://www.ncbi.nlm.nih.gov/nuccore/).
The human genome database contains 112,000 sequences, with sequence length 60.
The bacteria dataset consists of 20,000 sequences, with sequence length 1,040.
As for validating the combined memory-disk based approach, we aim to show only the run-time efficiency.
We applied our main memory disk-based mining approach to Homo sapiens GRCh37.64 DNA Chromosome Part 1, 2, and 3.
Part 2 has 98,000 sequences and a length of 60, and part 3 has 105,000 sequences with sequence length of 60.
With various values of minimum support, we compared the run-time performance of three approaches: MCFS (our algorithm), MacosVSpan [8], and Latest Approach [9].
Figs.4 and 5 show the retrieval performance with respect to the change of minimum support, indicating that MCFS outperforms the other two.
We also compared the memory usage of the three approaches with various values of minimum support.
The search space was relatively smaller, because we made use of sub-sequence and super-sequence relationships, and whenever reaching to the minimum support threshold, the sub-sequence for contiguous frequent patterns was not searched.
Figs.6 and 7 show the memory usage, indicating that our approach shows relatively low memory usage compared to the other two.
Although both MacosVSpan and our MCFS algorithm process one PDB after another and then produce the maximal contiguous frequent patterns by traversing the suffix tree, the size of the PDB cannot be larger than the original database (according to our proposed lemma); hence, the PDBs can be fit in the memory.
This is why MCFS consumes much less memory compared to MacosVSpan.
The Latest Approach [9] requires slightly larger memory, because it constructs the spanning tree by processing all of them at once.
It does not consider the memory usage while creating and producing the fixed-length spanning tree.
Since it first constructs the fixed-length spanning tree and then expands these candidate item sets to generate longer length candidate item sets, it is not guaranteed to be fit in the memory.
Finally, we validate our combined memory disk-based approach by applying it to Homo sapiens GRCh37.64 DNA Chromosome Parts 1, 2, and 3.
We assume that Parts 1, 2, and 3 are partitioned and stored on the disk.
With various settings of minimum support threshold, we measured the run-time performance (Fig.8).
In this paper, we have proposed an efficient algorithm, called MCFS, for mining maximal contiguous frequent sub-sequences, which requires only one scan of the original DNA sequence database.
The proposed algorithm has the following characteristics.
First, it can accept any value of the minimum support threshold effectively by means of one-time database access and construction of a suffix tree.
Second, it can effectively mine the complete set of maximal contiguous frequent patterns without specifying the sequence lengths in advance.
Third, the proposed method can produce results only by tree search, without expansion for production of a candidate set.
Fourth, from the experimental results, we can see the scalability of our approach.
As a result, it can be applied not only to a DNA sequence with a small number of items (dimension) but also amino acid sequences with a large number of items whose sizes can be very large and other multi-dimensional sequence datasets.
Our experiments show that MCFS outperforms other existing approaches for mining maximal contiguous sub-sequences.
In the future, we intend to extend this work to include gaps and execute it on real biological datasets.
The MCFS algorithm.
MCFS, Maximal Contiguous Frequent Suffix tree algorithm; PDB, projected database.
Complete A-subtree.
Partitioning approach to discover maximal contiguous frequent patterns for extra-large databases.
MCFS, Maximal Contiguous Frequent Suffix tree algorithm; MCFPi, maximal contiguous frequent patterns.
Retrieval performance w.r.t.
change of minimum support (Homo sapiens GRCh37.64 DNA Chromosome Part 1).
MCFS, Maximal Contiguous Frequent Suffix tree algorithm.
Retrieval performance w.r.t.
change of minimum support (bacteria genome dataset).
MCFS, Maximal Contiguous Frequent Suffix tree algorithm.
Memory usage w.r.t.
change of minimum support (Homo sapiens GRCh37.64 DNA Chromosome Part 1).
MCFS, Maximal Contiguous Frequent Suffix tree algorithm.
Memory usage w.r.t.
change of minimum support (bacteria genome sequence).
MCFS, Maximal Contiguous Frequent Suffix tree algorithm.
Performance of MCFS algorithm w.r.t.
increasing minimum support in partitioning approach (on Homo sapiens GRCh37.64 DNA Chromosome Part 1, 2, 3).
MCFS, Maximal Contiguous Frequent Suffix tree algorithm.
Title: A DNA sequence database

Title: Minimized physical projected databases (DBs)

Title: Contents of the frequent file

Title: Contents of the output file
Title: Comparative Evaluation of Intron Prediction Methods and Detection of Plant Genome Annotation Using Intron Length Distributions

Intron prediction is an important problem of the constantly updated genome annotation.
Using two model plant (rice and Arabidopsis) genomes, we compared two well-known intron prediction tools: the Blast-Like Alignment Tool (BLAT) and Sim4cc.
The results showed that each of the tools had its own advantages and disadvantages.
BLAT predicted more than 99% introns of whole genomic introns with a small number of false-positive introns.
Sim4cc was successful at finding the correct introns with a false-negative rate of 1.02% to 4.85%, and it needed a longer run time than BLAT.
Further, we evaluated the intron information of 10 complete plant genomes.
As non-coding sequences, intron lengths are not limited by a triplet codon frame; so, intron lengths have three phases: a multiple of three bases (3n), a multiple of three bases plus one (3n + 1), and a multiple of three bases plus two (3n + 2).
It was widely accepted that the percentages of the 3n, 3n + 1, and 3n + 2 introns were quite similar in genomes.
Our studies showed that 80% (8/10) of species were similar in terms of the number of three phases.
The percentages of 3n introns in Ostreococcus lucimarinus was excessive (47.7%), while in Ostreococcus tauri, it was deficient (29.1%).
This discrepancy could have been the result of errors in intron prediction.
It is suggested that a three-phase evaluation is a fast and effective method of detecting intron annotation problems.
With more and more species&#39; genomes completely sequenced, noncoding sequences have become a focus of researchers&#39; attention, especially for the study of introns.
In order to facilitate further research, a number of intron databases have been developed (Table 1).
The number of plant intron databases is much smaller than that in mammals and only in several model plants (such as Arabidopsis and rice).
Using known genome sequences and coding sequences (expressed sequence tags [ESTs] or cDNA), introns can be detected by aligning coding sequences with genome sequences.
Many tools were developed to detect introns in eukaryotes (Table 2) [1-16].
These tools used different algorithms and computer languages (such as Java, C++, and Python) to predict introns.
Therefore, the question is: there are many intron databases, algorithms, and detection methods for the study of eukaryotes, but which among them are the most suitable for the detection of plant introns?
Among these tools, the Blast-Like Alignment Tool (BLAT) and Sim4cc are the most commonly used tools.
BLAT applies in genomewide alignment [11].
Sim4cc is a tool for aligning cDNA and genomic sequences between species at various evolutionary distances [2].
Rice and Arabidopsis, as monocotyledonous and dicotyledonous model plants, are widespread with regard to in-depth research.
Their genome sequences have been annotated in detail, including their gene sequences, complementary DNA (cDNA) sequences, coding DNA sequence (CDS) sequences, exon sequences, intron sequences, and intergene sequences.
Therefore, it is possible to use this model plant information to test these intron prediction tools.
Genome annotation is a difficult and accurate project-even the best-annotated or most carefully studied genomes are continually re-released; e.g., release 7 of the Rice Genome Annotation Project was available on October 31, 2011 (http://rice.plantbiology.msu.edu/).
But, determining the accuracy and detecting the inherent errors of the genome annotation is a problem.
Since introns are removed from protein-coding transcripts, intron lengths are not expected to respect coding frames across the genome [17].
Using intron length distributions, Roy and Penny [18] point out a rapid and simple method for detecting a variety of possible systematic biases in gene prediction or even problems with genome assemblies.
Roy&#39;s method showed that a good genome annotation is accepted as roughly equal proportions of intron lengths of three phases: a multiple of three bases (3n), one more than a multiple of three bases (3n + 1), and two more (3n + 2).
Skewed predicted intron length distributions thus suggest systematic errors in intron prediction.
But, many plants with sequenced genomes have not been commented on.
In this study, we compared the advantages and disadvantages of BLAT and Sim4cc for model plants&#39; intron predictions, and we attempted to find a better way to predict the intron information of plants.
Based on Roy&#39;s method, we evaluated the intron information of 10 plant genomes and discuss a skew in genome wide intron length distributions that indicates systematic problems with intron predictions.
Title: Genome sequences

Ten plant genome sequences and transcript (EST, CDS, or cDNA) sequences were downloaded and indicated in Table 3 [19-27].
Table 3 contains the name of the 10 plant species, source websites, and genome sequence versions used in this study.
Title: Comparative BLAT and Sim4cc analysis

Using cDNA sequences and gene sequences, we searched rice and Arabidopsis introns by two methods-BLAT and Sim4cc- and then compared the results with annotated information.
The steps of this method are as follows (Fig.1): 1) Using the gene sequences of BLAT with its own cDNA sequences, we found intron information from the BLAT results by Perl script.
2) We sliced gene sequences and cDNA sequences to folders by Perl script.
In these folders, there was one sequence per file, and the gene name was the file name.
Using the same gene name of the gene and cDNA file, we blasted the gene sequences and cDNA sequences using Sim4cc.
Then, we got intron information from the Sim4cc results by Perl script.
3) We compared the results of the two types of software (BLAT and Sim4cc) and then got the annotated intron information.
4) We aligned intron sequences with their own gene sequences to develop detailed intron information, such as the intron position in the gene, intron length, intron number, forward-exon length, and backward-exon length, etc.
5) We compared the results from the two types of software with the annotated information to validate the methods.
Title: Intron length distributions analysis

Using Perl script, we extracted the intron information of the 10 plant genomes from the genome annotation.
Then, we counted the number and percentage of 3n, 3n + 1, and 3n + 2 of these 10 plants&#39; intron length distributions.
Title: A comparison of BLAT and Sim4cc

As a prerequisite, it was assumed that the intron annotated information was correct and complete.
Then, the software&#39;s results were compared with the annotated information.
Three sets of results of intron information were obtained: two sets from the software (BLAT and Sim4cc) and one set from the annotated information (Table 4).
Using BLAT, we found 99.35% and 99.87% of the introns of all rice and Arabidopsis annotated introns, respectively.
These introns were almost all of the introns in the genome - that is, only 0.13% to 0.65% of the introns were not found.
In contrast, by using Sim4cc, 95.15% to 98.98% of the introns were found (1.02% to 4.85% of the introns were lost) of all rice and Arabidopsis annotated introns.
In summary, BLAT got more of the introns in a genome than Sim4cc.
In light of this result, it seems as though that BLAT produces better results than Sim4cc.
We found 30,194 rice genes with at least one intron by BLAT, but the number was 30,177 according to the annotated information.
Because the BLAT results were larger than the annotated results, the BLAT results must have predicted some new and different genes with introns.
In the BLAT results, many short-length introns (less than 50 bp) were predicted, but in fact, these short-length introns were part of transcript sequences and were not real intron sequences.
In contrast, Sim4cc detected 29,875 genes with introns, and all of these genes were contained in the annotation information.
The predicted intron accuracy rate of Sim4cc was 100%.
On accuracy, Sim4cc was better than BLAT.
If Sim4cc is used, the user has to splice a whole genome file to many files: one gene, one file.
The computing process of Sim4cc was more complex than that of BLAT, and each time, Sim4cc only calculated one cDNA sequence to one gene sequence; so, the executing efficiency and speed are not high.
In comparison, BLAT was easier and faster than Sim4cc.
In conclusion, BLAT and Sim4cc can be used to predict introns, but each of them has its advantages and disadvantages.
The comparative results are summarized in Table 5.
Sim4cc was a cross-species spliced alignment program.
In our study, Sim4cc was used to find introns by comparing cDNA sequences and gene sequences.
The correct intron can be obtained by comparing one cDNA sequence with its own gene sequence.
But, a lot of introns were lost by Sim4cc.
In other words, Sim4cc was good at detecting the correct intron but not at predicting the whole number of introns in a genome.
In contrast, BLAT can predict most of the introns - nearly all of the total introns in a genome.
But, there were some false-positive predictions of introns.
However, the proportion of this error was very small.
As a result, BLAT will be proposed to annotate plant genome introns.
Title: Intron length distribution of 10 plants

According to Roy&#39;s method, many predicted introns in the plant genomes had in-frame stop codons, and the predicted introns in these genomes were equally as likely to be a multiple of 3 bp (3n) as to contain a plus one (3n + 1) or two (3n + 2) bp.
Here was an example of three phases from an Arabidopsis thaliana gene, AT1G17600.1 (Fig.2).
By analyzing genome sequence annotations, we got three-phase intron distributions for 10 plant species (Table 6).
If the plant intron annotation is more accurate, the number of three phases should be similar (one-third each).
For 80% (8/10) of species, there were similar numbers of the three phases.
It should be noted that most of these plant species annotations were the best annotations to date, but new annotations will be continually released to correct errors and false-positive results.
Title: Two-species 3n intron skew analysis

For all of the 10 genomes (Table 6), there were very similar numbers of 3n + 1 and 3n + 2 introns, and the percentages of 3n + 1 and 3n + 2 introns were within 0.8%.
In contrast, the number of 3n introns varied much more widely, from 29.1% to 47.7%.
In this study, two species&#39; genome introns showed strongly skewed percentages, in that the 3n intron percentage was much lower or higher than the expected value (one-third).
Such a skew suggests systematic errors in the intron prediction.
The green alga Ostreococcus lucimarinus has one of the highest gene densities known in eukaryotes, with many introns [28].
There was a striking excess of predicted 3n introns (47.7% of all predicted introns, 1,130) compared to 3n + 1 (25.8%, 611) and 3n + 2 (26.5%, 628) introns.
In this case, many predicted 3n introns were not true introns but instead exons.
The unicellular green alga Ostreococcus tauri is the world&#39;s smallest free-living eukaryote known to date [29].
These predicted introns showed a deficit of 3n introns (29.1%, 1,262), much lower than 3n + 1 (35.8%, 1,553) and 3n + 2 (35%, 1,519) introns.
This result is very close to previous studies [18].
In this case, 3n introns may be mistakenly regarded as coding sequences, whereas a 3n + 1 or 3n + 2 intron may be inferred from the disruption of the coding frame.
Title: Concluding remarks

By comparing the advantages and disadvantages of BLAT and Sim4cc in intron prediction, we found that BLAT is faster and can predict more introns than Sim4cc.
Through using intron length distribution to detect introns&#39; annotations, it is a simple and fast method for detecting a variety of possible systematic biases in intron prediction or even for detecting problems with genome assemblies.
Flowchart of a comparison of BLAT and Sim4cc results in predicting introns.
Intron information, including the following information of one intron: gene name, intron number, intron position in the gene, intron length, intron position in the genome, forward-exon length, backward-exon length, and intron sequences.
BLAT, Blast-Like Alignment Tool.
An example of three phases of intron from an Arabidopsis gene, AT1G17600.1.
Upper/lowercase sequence indicates exon/intron sequence.
Asterisks indicate frameshifts introduced by non-3n introns; intronic in-frame stop codons are underlined.
Intron 1 is a 99-bp intron (3n) with one in-frame stop codon.
Intron 2 is a 100-bp intron (3n + 2), which has two in-frame stop codons and thus does not interrupt the open reading frame.
Intron 3 is a 74-bp intron (3n + 1) with three stop codons.
Title: List of intron databases

Title: Tools for detection alternative-splicing/introns

Title: Ten plant species genome sequence sources

Title: Compared BLAT and Sim4cc predicted intron information with annotated intron information

Title: Comparative comparison of BLAT and Sim4cc in intron prediction

Title: Intron three-phase distributions of 10 plant species
Title: Web-Based Database and Viewer of East Asian Copy Number Variations

We have discovered copy number variations (CNVs) in 3,578 Korean individuals with the Affymetrix Genome-Wide SNP array 5.0, and 4,003 copy number variation regions (CNVRs) were defined in a previous study.
To explore the details of the variants easily in related studies, we built a database, cataloging the CNVs and related information.
This system helps researchers browsing these variants with gene and structure variant annotations.
Users can easily find specific regions with search options and verify them from system-integrated genome browsers with annotations.
Copy number variation (CNV) is a common type of structural variation in the human genome.
They have been suggested to be related to disease susceptibility or human phenotype diversity [1-3].
Current genome-wide association studies of CNVs are attempting to find how they are related to disease susceptibility or phenotypic diversity.
Due to the fact that frequencies of CNVs show ethnic differences [4-7], a finding of disease or phenotype association of a specific CNV in one population is hard to generalize to other populations.
However, by comparing the frequencies of target CNVs among different ethnic groups, we could assume the population-specific disease susceptibility or phenotype difference.
In this regard, CNV frequency information of various populations has become a major concern of population or disease association studies.
With increasing interest in CNVs, many CNV projects have been announced recently.
Many of them have already established large databases of CNVs from many different ethnic groups (Database of Genomic Variants [DGV], http://projects.tcag.ca; The Copy Number Variants Projects, http://www.sanger.ac.uk/research/areas/humangenetics/cnv/) [8].
There are also several databases for supporting population-specific studies, including the CNV Control Database for Japanese (http://gwas.lifesciencedb.jp), the Singapore Human Mutation and Polymorphism Database (shmed.bii.a-star.edu.sg) [9], and the Thailand Mutation and Variation Database (www4a.biotec.or.th) [10].
The accumulation of CNV information seems promising, in that we can gradually extend the knowledge of CNVs everywhere in the world.
However, it will take a long time to fill the gap of CNV databases covering the whole population.
As a step to fill this gap, we discovered a Korean CNV with Affymetrix Genome-Wide SNP array 5.0 (Affymetrix, Santa Clara, CA, USA) in a previous study [11].
In this study, we built a Korean database based on the findings of 4,003 copy number variation regions (CNVRs).
To build a tool that is easily accessible via the web for Korean-specific CNV studies, we built a viewer for browsing these CNVs.
Title: System requisites

Java Runtime Environment of Sun Microsystems 1.6.0 (Oracle, Redwood City, CA, USA) or equivalent is required, since the system is written in Java language.
We used MySQL database for storing and retrieving CNV data and GBrowse2 [12] for drawing regional information.
Both are freely available from their distribution websites.
Since the main search pages are written in Java Server Page (JSP) language, Apache Tomcat is needed for the application server and Apache HTTP server is needed for Gbrowse2 viewer pages (Fig.1).
Title: Data

CNVs of our previous study were retrieved from the Affymetrix Genome-Wide Human SNP array 5.0 of 3,578 Korean individuals.
A total of 4,003 CNVRs were defined, and 2,077 CNVRs (51.9%) were potentially novel.
The annotation data for genes were collected based on the Human Mar.
2006 NCBI36/hg18 build, and reference structure variants were retrieved from DGV (hg18.v8.aut.2009).
Title: Database and viewer

Our web-based database viewer can display previously discovered CNVs by their positions (Fig.2).
Users can also filter out CNVs based on the DGV overlapped regions, CNV type (Gain/Loss/Complex), or their frequencies.
Each selected region could be diagnosed in detail by clicking on it.
DGV and OMIM ID columns are linked with corresponding websites, and CNVR position columns are linked with the genome browser.
The genome browser is integrated based on the open source project GBrowse2.
Users can seek or zoom in/out of CNVs across the chromosome by entering positions or clicking zoom buttons.
GBrowse2 can also display interesting areas by dragging the region bar without reloading the entire page.
Gene information of the selected area is also displayed, and details will be given on separate pop-up page by clicking on it.
For a fast-paced research environment, a viewer for searching and observing data in one step is very handy.
We hope our system can help researchers who are interested not only in our target polymorphism study but also in a viewer for polymorphisms for general purposes.
We chose a web-based platform for the tool because of its usability and ease of maintenance.
Since its workload is very small as an input query for a viewer, a web-based platform does not have drawbacks, like other calculation applications with very large data.
System Structures of East Asian Copy Number Variation Database (EACDB).
JSP: PHP (Hypertext Preprocessor).
Database search page and genome browser.
Title: CysQ of Cryptosporidium parvum, a Protozoa, May Have Been Acquired from Bacteria by Horizontal Gene Transfer

Horizontal gene transfer (HGT) is the movement of genetic material between kingdoms and is considered to play a positive role in adaptation.
Cryptosporidium parvum is a parasitic protozoan that causes an infectious disease.
Its genome sequencing reported 14 bacteria-like proteins in the nuclear genome.
Among them, cgd2_1810, which has been annotated as CysQ, a sulfite synthesis pathway protein, is listed as one of the candidates of genes horizontally transferred from bacterial origin.
In this report, we examined this issue using phylogenetic analysis.
Our BLAST search showed that C. parvum CysQ protein had the highest similarity with that of proteobacteria.
Analysis with NCBI&#39;s Conserved Domain Tree showed phylogenetic incongruence, in that C. parvum CysQ protein was located within a branch of proteobacteria in the cd01638 domain, a bacterial member of the inositol monophosphatase family.
According to Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway, the sulfate assimilation pathway, where CysQ plays an important role, is well conserved in most eukaryotes as well as prokaryotes.
However, the Apicomplexa, including C. parvum, largely lack orthologous genes of the pathway, suggesting its loss in those protozoan lineages.
Therefore, we conclude that C. parvum regained cysQ from proteobacteria by HGT, although its functional role is elusive.
Horizontal gene transfer (HGT) is the movement of genetic material between different species of interkingdom, and lateral gene transfer is for intrakingdom movement [1].
The concept for gene transfer was first mentioned for an acquisition of virulence between bacterial strains [2], and then, transfers of multiple drug resistance between Shigellae and Escherichia coli strains were reported in Japan [3].
It is important to note that HGT not only affects correct reconstruction of a phylogenetic tree but also helps to understand reasons of its occurrence.
In bacteria, it is widely known that genetic material of antibiotic resistance is transferred in the gastrointestinal tract [4], and in the case of unicellular eukaryotes, such as Giardia lamblia, Trichomonas vaginalis, and Entamoeba histolytica, the organisms that use fermentation or anaerobic metabolism in the low-oxygen environment overcame the environmental stress by taking genes from bacteria [5].
Moreover, the transferred gene plays a positive role in adaptation to a pathogenic way of life [6].
On the other hand, HGT has been criticized, in that its biological significance is overemphasized.
If a gene has an essential role and participates in many interactions, its transfer might be less likely to occur than that of others or would be detrimental to the recipient.
Thus, some transferred genes are considered nonfunctional [7-9].
Cryptosporidium is a parasitic protozoan of the phylum Apicomplexa.
Among Cryptosporidium, C. parvum causes an infectious disease in humans and animals with diarrhea, called cryptosporidiosis.
Although the disease is prevalent where water quality is poorly managed, there is no satisfactory treatment until now [10, 11].
Genome sequencing of C. parvum was completed in 2004, identifying major metabolic pathways through comparison with other parasites and also reporting those enzymes with high similarities to bacterial and plantal counterparts [12].
In some crucial biosynthesis pathways, C. parvum has enzymes that originated from various organisms, such as bacteria, plants, and algae.
Phylogenomic analyses predicted a set of genes transferred from algae and eubacteria [13] and promising drug targets in nucleotide biosynthesis [14].
However, among 14 bacterial-like enzymes that were reported by Abrahamsen et al., only two enzymes received follow-up attention [15, 16].CysQ, 3&#39;-phosphoadenosine-5&#39;-phosphatase, also known as 3&#39;-phosphoadenosine-5&#39;-phosphosulfate (PAPS) 3&#39;-phosphatase, or 3&#39;(2&#39;), 5&#39;-bisphosphate nucleotidase, was among the 14 bacterial-like enzymes initially reported by the genome analysis.
It has been thought that it is needed during aerobic growth in E. coli to help control the levels of PAPS in cysteine biosynthesis [17].
Recently, CysQ protein has been considered as an important regulator that modulates the sulfate assimilation pathway by affecting levels of intermediates in plants, fungi, and bacteria [18, 19].
Despite its biological importance, no follow-up phylogenetic analysis of CysQ in C. parvum has been reported.
In this study, we assumed that CysQ might have been transferred from bacteria to C. parvum by horizontal gene transfer.
We constructed phylogenetic trees, based on a conserved domain of the protein, and inferred HGT from the phylogenetic incongruence.
Title: Sequence source

Cryptosporidium parvum strain Iowa type II (NCBI taxonomy accession 353152) was chosen for this study.
One of its genes, cgd2_1810, encodes CysQ, a sulfite synthesis pathway protein (accession no.
XP_001388206) [12].
The protein sequence was retrieved from the NCBI Protein Database (http://www.ncbi.nlm.nih.gov/protein).
Title: BLAST search

Sequence similarity searches were performed using BLASTP 2.2.26+ [20] with C. parvum CysQ protein against a nonredundant protein sequence database.
TBLASTN 2.2.26+ [21] was performed in order to search for the orthologs of the sulfur metabolism pathway in the C. parvum genome sequence using E. coli and Mycobacterium tuberculosis protein sequences as queries.
Title: Conserved domain analysis

We obtained information of conserved domains using the NCBI online Conserved Domain-search tool (http://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi) and Conserved Domain Database (CDD) (http://www.ncbi.nlm.nih.gov/Structure/cdd/cdd.shtml) [22, 23].
After the matching CDD model and the corresponding Conserved Domain Tree (CDTree) were identified, the CysQ sequence was added to the matched model, and the corresponding CDTree was then recalculated.
Title: Phylogenetic tree analysis

PhylomeDB is a collection of phylogenetic trees that have been precalculated automatically with a variety of options for a wide range of species [24].
We queried PhylomeDB and downloaded the phylogenetic trees that included CysQ protein.
Title: Pathway and ortholog analysis

CysQ enzyme was found in sulfate assimilation on sulfur metabolism of the Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway.
The list of organisms that harbored this pathway was compiled from KEGG orthology (KO) (http://www.genome.jp/kegg/ko.html) [25].
Title: BLAST for sequence similarity

The BLAST run using the query sequence of Cryptosporidium parvum reported the proteins of the genus Cryptosporidium as the best hits, followed by those of the order Eucoccidiorida, the genus Cryptosporidium belongs to.
The next best hits belonged mostly to Gammaproteobacteria.
Their gene descriptions corresponded to one of the alternative names of CysQ protein or a member protein of the inositol monophosphatase family, except for unclassified proteins and hypothetical proteins.
The best bacterial hit had an identity of 40% and a bit score of 181 bits for the query sequence of 341 amino acids.
From the taxonomy report of the BLAST result, 11 organisms among 110 were eukaryotes, and the other 99 were bacteria.
The bacterial list was composed of 59 Proteobacteria species, including 53 Gammaproteobacteria and 31 Bacteroidetes species.
Title: Phylogenetic analyses

While the BLAST analysis hinted HGT of the cysQ gene from bacteria to C. parvum, the hypothesis should be confirmed by phylogenetic analysis.
A phylogenetic tree for CysQ protein was retrieved from PhylomeDB.
We chose the Phy0018DKQ_ECOL5 tree made by the E. coli protein sequence as a seed and maximum likelihood method with the Jones-Taylor-Thornton (JTT) evolutionary model.
The phylogenetic tree with 170 orthologs comprised three eukaryotes-C. parvum, Arabidopsis thaliana, and Oryza sativa-one Archaea, and 166 Bacteria species.
In the tree, C. parvum was branched with Proteobacteria, while the plantal proteins were the outgroup of prokaryotic proteins.
In OrthoMCL (http://orthomcl.org), CysQ of C. parvum was located within the inositol monophosphatase family of Pfam (entry name OG5_129356) [26].
This ortholog group has only 70 orthologs from 54 different species and paralogs of Viridiplantae or T. vaginalis.
Moreover, it included a larger portion of plants and fungi rather than bacteria, and no metazoan protein orthologs were included.
Unlike PhylomeDB or OrthoMCL, the CDD of NCBI cataloged proteins sharing CysQ or related domains comprehensively.
CysQ protein of C. parvum contains a CysQ domain (accession no.cd01638), which is one of the children of the Fig (FBPase/inositol monophosphatase [IMPase]/glpX-like domain) superfamily.
The Fig superfamily is a metal-dependent phosphatase that organizes two subsets of direct children in the hierarchy of the superfamily: FBPase glpX domain (cd01516) and IMPase-like domain (cd01637).
Cd01637 has 9 children domains: CysQ (cd01638), IMPase (cd01639), bacterial IMPaselike 1 (cd01641), bacterial IMPase-like 2 (cd01643), IPPase (cd10640), FBPase (cd00354), Arch FBPase 1 (cd 01515), Arch FBPase 2 (cd01642), and PAP phosphatase (cd10517).
The whole hierarchy tree of the Fig superfamily comprises a total of 360 cellular organisms: 246 bacteria, 95 eukaryotes, and 19 Archaea (Fig.1A).
Some domains (cd01516, cd01637, cd01638, cd01641, and cd0643) comprise predominantly bacterial proteins in their CDTree, whereas the other domains have a combined composition (cd000354, cd0517, and cd01639) or a high level of Archaea (cd01642 and cd01515).
Domains cd01638, cd01641, and cd01643 are bacterial members of the IMPase family.
All of them show a high proportion of Proteobacteria, at about 65%, 50%, and 43% respectively.
In cd01638, C. parvum CysQ protein is located within the monophyletic gram-negative subtree, ranging from Pseudomonas sringae, Gammaproteobacteria, to Campylobacter jejuni, Epsilonproteobacteria (Fig.1B).
On the other hand, the gram-negative subtree is paraphyletic, in that it has 27 branches of Proteobacteria and Aquificae, Cyanobacteria, and Bacteroidetes, respectively.
Taken together, the phylogenetic analysis strongly supports the hypothesis that the cysQ gene of C. parvum may have been acquired from Proteobacteria by horizontal gene transfer.
Title: Orthologs on sulfate assimilation pathway

CysQ protein participates in sulfate assimilation on sulfur metabolism.
In Fig.2, we show a simplified version of the KEGG pathway, classifying the enzymes into three groups, according to their direction and steps: Class I for EC 2.7.7.4 (CysN) and EC 2.7.7.5 (CysD); Class II for EC 2.7.1.25 (CysC); and Class III for EC 3.1.3.7 (CysQ).
If CysQ of C. parvum is a true CysQ enzyme, playing a role in sulfate assimilation in the parasite, the other components of the pathway should be present in it.
On the contrary, we could not identify such genes in the annotated gene list.
The KEGG pathway did not list C. parvum proteins in the sulfate assimilation pathway.
We looked for the C. parvum proteins by searching the genome sequence using TBLASTN with M. tuberculosis CysN (Rv1286) and CysD (Rv1285) and E. coli CysN (b2751), CysD (b2752), and CysC (b2750) proteins as queries.
Among class I and II proteins, only CysN showed marginal matches to cgd6_3990 (29% and 33% identities, respectively) to M. tuberculosis and E. coli sequences.
Interestingly, this C. parvum protein was reported as elongation factor 1 alpha, not a sulfate adenylyltransferase.
This protein had high similarities to other protozoan or fungal elongation factor 1 alpha proteins.
Thus, we consider this as a false hit.
The class III protein, CysQ, matched to cgd2_1810 (24% and 36% identities, respectively, for M. tuberculosis and E. coli proteins).
This C. parvum gene was annotated &#34;CysQ, sulfite synthesis pathway protein.&#34;
As no other components of the sulfate assimilation pathway, except for CysQ, are found in C. parvum, we may conclude that the pathway does not function in this organism.
We compiled the orthologs of the genes in this pathway using the KO database (Table 1).
Eukaryotic kingdoms, except for protists harbored full ranges of orthologs in all three classes.
Animals and plants showed similar trends in Class I and II, because two classes shared two orthologs (K13811: 3&#39;-phosphoadenosine 5&#39;-phosphosulfate synthase [PAPSS], K00955: bifunctional enzyme CysN/CysC [CysNC]), and even K13811 is specialized in animals and plants.
Fungi also have many orthologs, like animals and plants, in Class I and II, but they have different orthologs (Class I, K00958, sulfate adenylyltransferase [E2.7.7.4C, met3]; Class II, K00860, adenylylsulfatekinase [CysC]).
In prokaryotes, the proportion of Class I genes is higher than Class II.
All Cyanobacteria, two-thirds of Proteobacteria, and Actinobacteria contained one of the orthologs in Class I, whereas Firmicutes, other bacteria, and the Archaea group have a few orthologs in Class I, II, and III.
On the other hand, there were very few orthologs of the sulfate assimilation pathway in protists.
We expanded the protist lineage, cataloging the proteins at the class or species level (Table 2).
Some protists (Choanoflagellates, Entamoeba of Amoebozoa, and Diatoms) had at least one orthologous gene in each of three classes, while most Alveolates, Amoeboflagellate, Euglenozoa, and Diplomonads did not have any orthologs in three classes, many of which are known as parasites causing infectious diseases.
While the sulfate assimilation pathway is generally well conserved in both prokaryotes and eukaryotes, in some protist lineages, the pathway is missing.
Thus, we hypothesize that the pathway may have been lost during the evolution of the lineages.
C. parvum, like other Aveolates, also may have lost it, and cge2_1810 can not function as CysQ properly.
Its function remains elusive, as the sequence similarity to CysQ of M. tuberculosis or E. coli is rather low.
A BLAST search of C. parvum CysQ (cgd2_1810) protein shows the highest similarity with those of proteobacteria.
Although it implies HGT from bacteria to this eukaryote, sequence similarity alone is not enough as its basis for several known reasons [27].
In addition, a phylogenetic analysis should support it.
For this, initially, we relied on the tree built by PhylomeDB.
However, its species coverage was biased or undersampled.
On the other hand, CDD of NCBI is well subdivided into kingdom and function groups.
C. parvum CysQ protein was mapped into the subfamily of IMPase, which is a bacterial CysQ domain, and comprises only bacterial sequences.
Furthermore, C. parvum was located near Gamma- and Alphaproteobacteria in the CDTree.
Hence, it seems that these results demonstrate that gene transfer events occurred from bacteria to C. parvum in the evolutionary process.
On the KEGG pathway, we raise the possibility that Alveolates, Euglenozoa, and Diplomonads of protozoa suffered from the losses of genes in the sulfate assimilation pathway.
But, did C. parvum recover CysQ protein by HGT in the process of evolution?
Sulfate assimilation shows highly conserved orthologs for each taxonomy lineage, and it plays important roles in sulfur metabolism, whereas Alveolates of protozoa, including Cryptosporidium, rarely have orthologous genes.
For the pathogenic bacterium M. tuberculosis, sulfur-containing metabolites are essential to its pathogenesis and persistence in the host [18, 28], and the Database of Essential Genes (DEG) lists that Rv1286 (cysN) and Rv1285 (cysD), not Rv2131c (cysQ), are essential genes in sulfur metabolism of M. tuberculosis [29, 30].
Parasitic protozoa have diverse sulfur-containing amino acid metabolism that are considered to affect virulence and several stress response.
On the other hand, C. parvum and Plasmodium falciparum lack a sulfur assimilation pathway, which is expected to be substituted from host cells [31].
In conclusion, although the sulfate assimilation pathway is missing in some protest lineages, C. parvum has a protein that is predicted as CysQ and has sequence similarity with that of proteobacteria, gram-negative bacteria.
Moreover, the phylogenetic analysis supports the acquisition of cgd2_1810 from proteobacteria through horizontal gene transfer.
Therefore, we can infer that C. parvum lost its genes in the sulfate assimilation pathway, including cysQ, during a parasitic way of life, and it acquired a copy of cysQ from bacteria by horizontal gene transfer.
What is the biological role of this gene product?
As the sole member, without other members, of the pathway, it can not assume the right role of CysQ.
Its function is elusive at the moment.
Conserved Domain Tree (CDTree) involving CysQ (cgd2_1810) protein.
(A) Whole hierarchy of Fig superfamily.
Branch colors indicate Eukaryota (pink), Bacteria (bright blue), and Archaea (orange) domains; Cryptosporidium parvum sequence is added (red branch and circle).
On the right side, color bar marked with domain accession names displays subfamilies in Fig superfamily.
All CDs except for cd01637 are well clustered.
cd01637 is scattered around the tree and highlighted in black ticks.
(B) CDTree of cd01638 domain.
The cd01638 clade that includes only bacteria is classified by taxonomic lineage.
Bacteria are subdivided into two distinct parts, gram-negative and gram-positive bacteria.
Each bacterial phylum is colored as shown in legend.
CysQ protein of C. parvum is shown in red dashed branch with a red circle near Proteobacteria.
A simplified sulfate assimilation pathway.
Inorganic sulfate is converted to adenosine 5&#39;-phosphosulfate (APS) by ATP sulfurylase (EC 2.7.7.4 and/or EC 2.7.7.5, Class I).
APS is phosphorylated by APS kinase (EC 2.7.1.25, Class II) to 3&#39;-phosphoadenosine-5&#39;-phosphosulfate (PAPS).
PAPS is either transferred to 3&#39;-phosphoadenosine-5&#39;-phosphate (PAP) or reconverted to APS by 3&#39;-phosphoadenosine-5&#39;-phosphatase (EC 3.1.3.7, Class III).
Title: Class comparison of ortholog conservation in cellular organismsa

Title: Protist orthologs involved in sulfate assimilation pathway
Title: Replication of the Association of the 6q22.31c Locus near GJA1 with Pulse Rate in the Korean Population

Pulse rate is known to be related to diverse phenotypes, such as cardiovascular diseases, lifespan, arrhythmia, hypertension, lipids, diabetes, and menopause.
We have reported two genomewide significant genetic loci responsible for the variation in pulse rate as a part of the Korea Association Resource (KARE) project, the genomewide association study (GWAS) that was conducted with 352,228 single nucleoride polymorphisms typed in 8,842 subjects in the Korean population.
GJA1 was implied as a functionally causal gene for pulse rate from the KARE study, but lacked evidence of replication.
To re-evaluate the association of a locus near GJA1 with pulse rate, we looked up this signal in another GWAS conducted in a Health Examinee-shared cohort of 3,703 samples.
Not only we were able to confirm two pulse rate loci (1q32.2a near CD46 and 6q22.13c near LOCL644502) identified in the KARE GWAS, we also replicated a locus (6q22.31c) near GJA1 by the lookup in the Health Examinee GWAS.
Considering that the GJA1-encoded protein is a major component of cardiac gap junctions, a functional study might be necessary to validate its genuine molecular biological role in the synchronized contraction of the heart.
Pulse rate, also called heart rate, is the number of heartbeats per unit of time, commonly used as beats per minute (BPM).
Pulse rate is varied in accordance with the body&#39;s need for oxygen changes, such as during exercise or sleep.
Since various health-related phenotypes are closely correlated with pulse rate [1-3], measuring pulse rate is commonly used for diagnosing and accessing medical conditions.
Pulse rate is known to be positively correlated with triglycerides and negatively correlated with high-density lipoprotein cholesterol [3].
Numerous studies have reported an association between heart rate and total and/or cardiovascular mortality [1].
Metabolic syndrome is associated with poor exercise capacity and poor heart rate recovery in patients who have coronary disease [4].
Moreover, high pulse rate is negatively correlated with lifespan in mammals, including humans [2].
Despite the importance of pulse rate as a risk factor for various diseases, however, few genes are known to be associated with pulse rate, and the underlying mechanism for pulse rate remains unraveled.
We previously reported pulse rate-associated loci from the Korea Association Resource (KARE) genomewide association study (GWAS), comprising 8,842 samples in the Korean population [5].
In this study, we extended the previous study and reexamined the relevance of pulse rate-associated single-nucleotide polymorphisms (SNPs) from an additional GWAS of 3,703 Korean samples.
Title: Subjects

In the discovery stage, the association results for pulse rate were obtained from the KARE study.
The information for subjects of the KARE GWAS is described elsewhere [5].
Subjects for the replication stage were recruited from the Health Examinee cohorts, called HEXA-shared cohorts.
The standardized examinations that were applied in this survey included 4,302 participants aged 40 to 70 years.
The characteristics of the cohorts are described in Table 1.
Title: The measurement of pulse rate

Pulse rate is the frequency of the heart beat (BPM).
Pulse rate of KARE was measured 3 times while the participant was lying down, and the average value was used for the analysis (before the first measurement, participants rested for 5 min, and 3 measurements were taken at least 30 s apart).
In HEXA-shared cohorts, the first measurement of the participants, who rested for 5 min, was used.
Title: Genotyping and quality control

The methods for genotyping and quality control for the KARE GWAS have been described elsewhere [5].
For the HEXA GWAS, genomic DNA was extracted from 4,302 participants from an urban cohort and genotyped with Affymetrix Genome-Wide Human SNP array 6.0 (Affymetrix, Inc., Santa Clara, CA, USA).
From 4,302 genotyped samples, we excluded samples with a low call rate (≥ 95%, n = 443), high heterozygosity (n = 25), outliers (n = 26), and gender inconsistencies (n = 8), and those obtained from individuals who had developed any kind of cancer (n = 61) were excluded from subsequent analyses, along with related or identical individuals whose computed average pairwise identity-by-state value was higher than that estimated from cryptic first-degree relatives of samples (n = 33).
SNP markers with a high missing genotype rate (&gt; 5%), minor allele frequency (MAF) (&lt; 0.01), and significant deviation from Hardy-Weinberg equilibrium (p &lt; 1 × 10-6) were excluded, leaving a total of 627,659 markers to be examined in 3,703 individuals.
Title: Statistical analysis

Association analyses were performed using PLINK (http://pngu.mgh.harvared.edu/~purcell/plink/) [6], SAS version 9.2 (SAS institute Inc., Cary, NC, USA), and R statistics package (version 2.7.1) (http://r-project.org/).
The SNP association for pulse rate was tested by linear regression analysis with an additive model, adjusting for age, sex, and recruitment area.
The KARE GWAS and replication GWAS were combined by an inverse-variance meta-analysis method, assuming fixed effects, with Cochran&#39;s Q test used to assess between-study heterogeneity [7].
Meta-analysis calculations were performed using METAL (http://www.sph.umich.edu/csg/abecasis/Metal) in the R program (version 2.7.1).
The discovery GWAS for pulse rate included 352,225 SNPs profiled with Affymetrix Genome-Wide human SNP array 5.0 in 8,842 individuals from the Ansung and Ansan cohorts as described previously [5].
Characteristics of the study, including age, sex, and trait summaries, are presented in Table 1.
The association analysis identified several genomic locations as potentially associated with pulse rate.
Unlike the previous GWAS [5], we did not apply a threshold with a p &lt; 0.01 in both Ansung and Ansan cohort in addition to a combined p &lt; 1 × 10-5 for the SNP selection for the replication study so that we could include more promising SNPs for their association with pulse rate.
We observed 61 significant SNPs by applying only a threshold with a combined p &lt; 1 × 10-5.
After excluding SNPs by cluster plot inspection and removing singletons, 3 independent SNPs (that is, with pairwise linkage disequilibrium statistics r2 &lt; 0.2 and MAF ≥ 0.05 within a 500-kb window of the genomic region) were finally chosen to validate in the independent population - i.e., the HEXA cohort, comprising 3,703 individuals.
In the replication study, all 3 SNPs were confirmed for their association with pulse rate at a p &lt; 0.05 (Table 2).
The first pulse rate signal located to 1q32.2a (rs12731740, MAF = 0.103, poverall = 2.20 × 10-8).
Three genes, such as CD46, Clorf132, and CD34, appear near this locus [5].
The physiological relevance of the 3 genes in relation to pulse rate is not clear [8, 9].
Thus, fine mapping surrounding this locus by targeted resequencing is highly recommended to discover the causal variants for pulse rate.
The second pulse rate SNP is rs12110693 (MAF = 0.494, poverall = 1.10 × 10-8) on 6q22.31c.
The only gene located near this SNP is LOC644502, which has no known function.
The third locus associated with pulse rate is at 6q22.31c (rs2892767, MAF = 0.261, poverall = 2.50 × 10-7).
This locus is near GJA1 (Gap junction protein, alpha 1, 43 kDa), which is a member of the connexin gene family.
The protein encoded by GJA1 is a component of gap junctions, which are intercellular channels that provide a route for the diffusion of low-molecular-weight materials from cell to cell.
In this study, we performed a genomewide scan for pulse rate and identified 3 loci accounting for the variation in pulse rate.
Two of 3 loci were previously reported and replicated once again in an independent HEXA-shared cohort.
The other one (rs2892767) near GJA1 was previously implicated for its association with pulse rate but was not chosen for the replication study, since the SNP did not meet the selection criteria in the discovery stage [5].
In the present study, SNP rs2892767 near GJA1 was taken forward to the replication study using the independent dataset from the HEXA-shared cohort and was nicely confirmed for its association with pulse rate.
Two SNPs, rs2892767 near GJA1 and rs12110693 near LOC644502, are 408 kb apart from each other and are located in a separate LD, indicating that both are independent SNPs.
The independence between both SNPs was further tested by conditional analyses.
We performed multiple linear regression analyses, conditioning the SNPs to each other to find a significantly stronger proxy for pulse rate.
However, no dramatic decrease in statistical significance was observed, and no interaction was found between rs2892767 and rs12110693 (Table 3).
These results further validate that the 2 SNPs are independent of each other.
Gap junctions are thought to have a crucial role in the synchronized contraction of the heart and in embryonic development.
It is also known that gap junctions, clusters of cell-to-cell channels composed of connexins, mediate the orderly spread of electrical excitation throughout the heart [10].
The GJA1-encoding protein has been reported to be involved in the development and function of the heart by allowing direct cell-to-cell exchange of molecules, which mediate multiple signaling events [11].
Mutations in GJA1 cause Mendelian inherited hypoplastic left heart syndrome [12].
Cho et al.
[5] previously reported the same locus near GJA1 to be responsible for pulse rate without evidence of replication.
In the present study, we successfully replicated the locus in an independent replication study.
As indicated from the functional role of GJA1, it is a strong candidate gene for pulse rate.
Thus, further efforts, such as fine mapping or imputation based on large-scale sequencing data near rs2892767 on 6q22.31, would reveal the causal variant of this locus.
In addition, functional studies for GJA1 would be worthwhile to facilitate a full understanding of its biological role in pulse rate.
Our study was relatively underpowered to discover additional novel loci and demonstrated that large-scale genomewide meta-analysis comprising hundreds of thousands samples is required for understanding the missing heritability of pulse rate.
Title: Characteristics of cohorts

Title: Variants that associate with variation in pulse rate

Title: The effect of SNP adjustment on pulse rate
Title: Genetic Analysis of SCN5A in Korean Patients Associated with Atrioventricular Conduction Block

Recent several studies have shown that the genetic variation of SCN5A is related with atrioventricular conduction block (AVB); no study has yet been published in Koreans.
Therefore, to determine the AVB-associated genetic variation in Korean patients, we investigated the genetic variation of SCN5A in Korean patients with AVB and compared with normal control subjects.
We enrolled 113 patients with AVB and 80 normal controls with no cardiac symptoms.
DNA was isolated from the peripheral blood, and all exons (exon 2-exon 28) except the untranslated region and exon-intron boundaries of the SCN5A gene were amplified by multiplex PCR and directly sequenced using an ABI PRISM 3100 Genetic Analyzer.
When a variation was discovered in genomic DNA from AVB patients, we confirmed whether the same variation existed in the control genomic DNA.
In the present study, a total of 7 genetic variations were detected in 113 AVB patients.
Of the 7 variations, 5 (G87A-A29A, intervening sequence 9-3C&gt;A, A1673G-H558R, G3578A-R1193Q, and T5457C-D1819D) have been reported in previous studies, and 2 (C48G-F16L and G3048A-T1016T) were novel variations that have not been reported.
The 2 newly discovered variations were not found in the 80 normal controls.
In addition, G298S, G514C, P1008S, G1406R, and D1595N, identified in other ethnic populations, were not detected in this study.
We found 2 novel genetic variations in the SCN5A gene in Korean patients with AVB.
However, further functional study might be needed.
The SCN5A gene, encoding a voltage-gated Na+ channel, is predominately expressed in the heart, where it has a key role in the generation and propagation of the cardiac impulse [1].
Voltage-gated Na+ channels are transmembrane proteins that produce the fast inward Na+ current responsible for the depolarization phase of the cardiac action potential.
Inherited variations in SCN5A, the gene encoding the pore-forming α-subunit of the cardiac-type Na+ channel (Nav1.5), result in a spectrum of disease entities, termed Na+ channelopathies.
In many previous studies, lots of variations of the SCN5A gene have been observed in various cardiac diseases, such as long-QT syndrome (LQT), Brugada syndrome (Brs), progressive cardiac conduction defect, atrial fibrillation (AF), dilated cardiomyopathy, and overlapping syndromes [2, 3].
Recently, cardiac conduction system disease, manifesting as slower intramyocardial conduction and, in some cases, atrioventricular conduction block (AVB), has been related to SCN5A mutation [4].
Familial AVB, characterized by progressive &#34;degree of block&#34; in association with variable apparent &#34;site of block,&#34; may be transmitted as an autosomal dominant trait.
Two genetically distinct forms of AVB have been identified [5].
Brink et al.
[6] established a genetic link between AVB and a genetic locus at chromosome 19q13, and Schott et al.
[4] mapped AVB to chromosome 3p21.
Such genetic variations in AVB patients have been widely studied in Caucasians, Han Chinese, and Japanese, but no study has yet been published in Koreans as far we know.
Therefore, we carried out a complete sequencing of coding regions of the SCN5A gene, except the untranslated region, in Korean AVB patients to investigate the SCN5A variations associated with AVB and compared them with normal control subjects.
This is the first study of SCN5A genetic variations that examined all coding regions in Korean patients with AVB.
Title: Patient enrollment

We enrolled 113 Korean AVB patients who were diagnosed with electrocardiograms and who had inserted permanent pacemakers and 80 normal controls with no cardiac symptoms.
All patients and control subjects were recruited from four medical centers - Keimyung University, Yeungnam University, Catholic University, and Fatima Hospital (Daegu, Korea) - from August 2009 to September 2011.
This study was approved by the ethical committee at each hospital; consent was obtained from all individuals before enrollment into the study.
Title: DNA extraction

In this study, peripheral blood was collected into ethylenediamine tetraacetic acid-containing tubes, and genomic deoxyribonucleic acid (DNA) was extracted from whole blood samples using the QIAamp DNA blood mini kit (Qiagen, GmbH, Hilden, Germany).
DNA concentration was determined using a NanoDrop ND1000 spectrophotometer (Thermo Fisher Scientific Inc., Wilmington, DE, USA), and purity of the DNA was assessed based on the 260/280 nm absorbance ratio.
Title: Multiplex polymerase chain reaction (PCR) and sequence analysis

The SCN5A gene is located on human chromosome 3p21, and the gene consists of 28 exons (Fig.1) and encodes a protein of 2,016 amino acids with a molecular mass of 227 kDa [7].
Entire coding regions, including exon-intron boundaries of the SCN5A gene, were amplified by multiplex PCR-based direct sequencing.
The primers for PCR amplification were designed based on the GenBank reference sequence; primers are shown in Table 1.
PCR conditions were as follows: an initial denaturation at 95℃ for 15 min; and denaturation at 94℃ for 30 s, annealing at 68-70℃ for 30-60 s, and extension at 72℃ for 60-90 s, repeated for 30 cycles (Table 1).
After multiplex PCR, the reaction mixture was electrophoresed in a 2% agarose gel and stained with ethidium bromide (Fig.2).
Then, amplified PCR products were purified using the QIAquick PCR purification kit (Qiagen) and directly sequenced using the BigDye Terminator ver 3.1 cycle sequencing kit on an ABI PRISM 3100 Genetic Analyzer (Applied Biosystems, Foster City, CA, USA).
Sequencing results were compared with the reference sequences (SCN5A/NM_198056.2/ENSG00000183873/ENST00000333535) using the alignment program BLAST 2.0 of NCBI, and we determined the portion of variation that occurred.
When a variation was discovered in genomic DNA from patients, we confirmed whether the same variation existed in the control genomic DNA.
Title: Statistical analysis

p-values and odds ratios with 95% confidence intervals for each genotype and allele were evaluated by χ2 analysis.
A p-value of &lt;0.05 was considered to indicate a statistically significant difference between AVB patients and normal controls of the study.
In the present study, we sequenced all translated exons and exon-intron boundaries of the SCN5A gene from 113 Korean AVB patients.
As a result, we identified 7 genetic variations (of these, 5 were known variations and 2 were novel variations), consisting of 3 synonymous (G87A-A29A, G3048A-T1016T, and T5457C-D1819D), 3 non-synonymous (C48G-F16L, A1673G-H558R, and G3578A-R1193Q), and 1 splicing donor site of the intron 9-exon 10 boundary (intervening sequence [IVS] 9-3C &gt; A) (Table 2).
The examples of the sequencing result are shown in Fig.3.
Among the 7 genetic variations, 2 single nucleotide polymorphisms (SNPs) have been reported by American and Japanese researchers: non-synonymous SNP in exon 12 (rs1805124, A1673G-H558R) and synonymous SNP in exon 28 (rs1805126, C5457T-D1819D).
A1673G-H558R was identified in 18% of the American populations (n = 71) and 8% of the Japanese populations (n = 100) [8, 9], and C5457T-D1819D was observed in 12.3% of the American populations (n = 130) and 46% of the Japanese populations (n = 100) [9, 10].
We also discovered a synonymous SNP (rs6599230: G87A-A29A, in exon 2) and non-synonymous SNP (rs41261344, G3578A-R1193Q, in exon 20) that are very common polymorphisms in the Han Chinese population.
In addition, we identified 2 novel variations that have not been previously reported.
One of these variations was heterozygous synonymous (G3048A-T1016T), and the other was heterozygous non-synonymous (C48G-F16L).
The genotype and allele frequencies of the identified genetic variations are summarized in Table 2.
There were significant differences in the allele frequencies of A1673G-H558R and G3048A (T1016T) between the two groups (p &lt; 0.05).
There were no statistically significant differences in allele frequencies of the other genetic variations between the two groups (all p &gt; 0.05).
However, G298S [11], G514C [12], P1008S [13], G1406R [14], and D1595N [11], identified in other ethnic populations, were not detected, and A3075T-E1025D and T4847A-F1616Y, discovered in 105 Korean sick sinus syndrome patients in our previous study, were also not detected in this study.
Next, to compare with normal controls, we analyzed normal samples by the same methods whether the same variations existed in their genes.
Just as in the patient group, 4 SNPs (rs6599230, G87A-A29A; rs41312434, IVS9-3C &gt; A; rs1805124, A1673G-H558R; rs1805126, C5457T-D1819D) were detected in the normal control group, but 1 non-synonymous SNP (rs41261344, G3578A-R1193Q) and 2 novel variations (C48G-F16L and G3048A-T1016T) were not detected in the normal control group.
This is the first case report with genetic variations of SCN5A in Korean patients with AVB.
To date, a number of SCN5A variations associated with various cardiac diseases, such as LQT, Brs, progressive cardiac conduction defect, AF, and overlapping syndromes, have been reported.
Although SCN5A genetic variations associated with AVB have been well studied in Caucasians, Han Chinese, and Japanese, no study has yet been published in Koreans as far we know.
In previous studies, it has been shown that in some cases, progressive AVB has been linked to SCN5A gene variations [4].
Furthermore, in our prior study, 2 novel variations were detected in Korean sick sinus syndrome (SSS) patients, which, to our knowledge, has not been previously reported in any ethnic group.
The variations could be significantly associated with functional effects of the SCN5A gene.
Based on these results, we searched for SCN5A genetic variations in Korean AVB and compared them with those of control subjects.
The SCN5A gene consists of 28 exons and encodes a protein of 2016 amino acids with a molecular mass of 227 kDa [7].
In our patients with AVB, there were 7 sites of nucleotide change from exon 2 to exon 28 of the SCN5A gene.
Among of them, 2 sites (G87A-A29A, IVS9-3C &gt; A) were reported as genetic variations in a Western study [7], and T5457C-D1819D has already been reported as a variation or polymorphism without functional effects on the channel in Asian studies [7, 15, 16].
A1673G-H558R is located in the Na+ channel I, II interdomain linker, and previous functional studies have shown that the H558R-encoding minor allele can alter the phenotype of true disease-causing SCN5A mutations [17].
It has been suggested that it modulates Na+ channel functional changes caused by other variations and plays a role in intragenic complementation [18, 19].
Previously, the H558R G allele has been associated with occurrence of early-onset AF and LQT interval in healthy populations.
The H558R G allele frequency has been reported to be 20.4-32% in the white population, 29% in the black population, and 10.4% in the Chinese population.
In our study, the G allele frequency of SCN5A mutation carriers was 8% and 13.1% in the normal control group.
G3578A-R1193Q, which can cause a gain of the sodium ion channel function and cause LQT3 and Brs, was also found.
It has been reported that G3578A-R1193Q confers a risk of LQT3 or Brs in the general population, although it seems to be a common polymorphism in the Han Chinese population.
Electrophysiological studies showed that R1193Q destabilizes inactivation gating and generates a persistent, non-inactivating current, which results in a gain of the sodium channel function, as in LQT3 syndrome [20, 21].
In addition to previously reported variations, we identified 2 novel variations that were not reported in the case of Japanese and Chinese populations.
One of these variations was heterozygous non-synonymous (C48G-F16L), and the other was heterozygous synonymous (G3048A-T1016T).
Since a non-synonymous nucleotide change in the SCN5A gene is likely to induce a functional change in the sodium channel, a non-synonymous substitution was considered a candidate for the variation associated with this disorder.
In this study, a non-synonymous variation, C48G-F16L, was not found in the normal control group but was detected only in the patient group, suggesting that it may effect AVB.
However, G298S [11], G514C [12], P1008S [13], G1406R [14], and D1595N [11], identified in other ethnic populations, were not detected, thought to be due to very low frequency or no variation in the Korean population.
In summary, our data may provide useful information for the identification of novel variations related to AVB, and the novel variants may provide useful bio-markers to study Korean cardiac disease.
In a future study, we will conduct a functional analysis of the novel variation (C48G-F16L) and propose a mechanism for its contribution to the AVB phenotype in patients.
Also, we will try to find out genetic variation sites in other genes that might be responsible for AVB.
Genomic location and exon image of SCN5A gene and voltage-gated Na+ channel α subunit.
(A) The SCN5A gene is located on human chromosome 3p21.
(B) Structure of SCN5A gene: the SCN5A gene consists of 28 exons; among these, exon 1 is an untranslated region.
(C) The voltage-gated Na+ channel α subunit consists of 4 homologous domains (DI-DIV), each containing 6 transmembrane-spanning segments (S1-S6).
Agarose gel electrophoresis of multiplex PCR products from exon 2 to exon 28 of the SCN5A gene.
M, 100 bp DNA size marker; lane 1-6, sample 1-6; lane 7, negative control.
Results of DNA sequencing analysis of SCN5A gene, showing nucleotide and amino acid changes.
DNA sequencing results of SCN5A exon 2 (A, B), exon 12 (C), exon 17 (D), exon 20 (E), and exon 28 (F).
C48G-F16L (A), A1673G-H558R (C), G3048A-T1016T (D), and G3578A-R1193Q (E) are all heterozygous nucleotide changes.
G87A-A29A (B) and T5457C-D1819D (F) are heterozygous or homozygous changes that are causing the synonymous nucleotide change.
Title: Multiplex PCR primers for amplification of the coding regions and flanking intronic sites of SCN5A and PCR conditions for each primer pair

Title: Genetic variations and allele frequencies in the SCN5A gene of Korean complete atrioventricular conduction block patients (n=113) and normal controls (n=80)
Title: Sample Size and Statistical Power Calculation in Genetic Association Studies

A sample size with sufficient statistical power is critical to the success of genetic association studies to detect causal genes of human complex diseases.
Genome-wide association studies require much larger sample sizes to achieve an adequate statistical power.
We estimated the statistical power with increasing numbers of markers analyzed and compared the sample sizes that were required in case-control studies and case-parent studies.
We computed the effective sample size and statistical power using Genetic Power Calculator.
An analysis using a larger number of markers requires a larger sample size.
Testing a single-nucleotide polymorphism (SNP) marker requires 248 cases, while testing 500,000 SNPs and 1 million markers requires 1,206 cases and 1,255 cases, respectively, under the assumption of an odds ratio of 2, 5% disease prevalence, 5% minor allele frequency, complete linkage disequilibrium (LD), 1:1 case/control ratio, and a 5% error rate in an allelic test.
Under a dominant model, a smaller sample size is required to achieve 80% power than other genetic models.
We found that a much lower sample size was required with a strong effect size, common SNP, and increased LD.
In addition, studying a common disease in a case-control study of a 1:4 case-control ratio is one way to achieve higher statistical power.
We also found that case-parent studies require more samples than case-control studies.
Although we have not covered all plausible cases in study design, the estimates of sample size and statistical power computed under various assumptions in this study may be useful to determine the sample size in designing a population-based genetic association study.
In genetic epidemiological research, both case-control and case-parent trio designs have been used widely to evaluate genetic susceptibilities to human complex diseases and markers, such as single-nucleotide polymorphisms (SNPs), to localize disease gene variants [1-5].
The sample size for detecting associations between disease and SNP markers is known to be highly affected by disease prevalence, disease allele frequency, linkage disequilibrium (LD), inheritance models (e.g., additive, dominant, and multiplicative models), and effect size of the genetic variants (e.g., odds ratio, relative risk, etc.)
[4, 6, 7].
Previous studies have shown that a population based case-control design can be more powerful than a family-based study design in identifying genes predisposing human complex traits, both for qualitative traits and for quantitative traits [8-11].
However, some studies reported that the case-parent design is much more powerful than the case-control design in evaluating genetic risk for common complex diseases, because case-control studies are susceptible to bias due to phenotype misclassification or population stratification [12, 13].
Recently, genome-wide association studies (GWASs) using thousands of cases and controls reported many susceptibility SNPs for 237 human traits by the end of June, 2011 (www.genome.gov/GWAStudies).
Since a GWAS evaluates hundreds of thousands of SNP markers, it requires a much larger sample size to achieve an adequate statistical power [14-18].
In genetic association studies, the observed signal for association is referred to be statistically significant if the p-value is less than a preset threshold value (α) of 0.05 to reject a null hypothesis of genetic association.
Testing a large number of SNP markers leads to a large number of multiple comparisons and thus increases false positive rates.
Either the Bonferroni correction or the false discovery rate is generally applied to avoid false positive (type I error) rates.
However, the Bonferroni-corrected p-value, the significance threshold set to 0.05 divided by the total number of SNPs analyzed in a GWAS, is too strict to consider the correlations among SNP markers (e.g., p = 1 × 10-7 for 500,000 (500 K) SNPs, p = 5 × 10-8 for 1 million (M) SNPs) [16, 19].
Therefore, estimating a sufficient sample size to achieve adequate statistical power is critical in the design stage of genetic association [20-24].
Statistical power is the probability to reject a null hypothesis (H0) while the alternative hypothesis (HA) is true.
It is affected by many factors.
For instance, a larger sample size is required to achieve sufficient statistical power.
Although a researcher collects a large number of samples, all samples may not be necessary to be analyzed to detect evidence for association.
A large sample size improves the ability of disease prediction; however, it is not cost-effective that a researcher genotypes more than the effective sample size [25].
Unless researchers estimate sample size and statistical power at the research design stage, it leads to wasted time and resources to collect samples.
An effective sample size can be defined as the minimum number of samples that achieves adequate statistical power (e.g., 80% power).
On the other hand, too small a sample size to detect true evidence for an association increases false negative rates and reduces the reliability of a study.
False negative rates are increased by multiple factors that cause systematic biases, and such biases reduce statistical power [26].
The statistical power of 80% is used widely to avoid false negative associations and to determine a cost-effective sample size in large-scale association studies [7, 22, 23].
However, many researchers tend to overlook the importance of statistical power and sample size calculations.
In this study, we evaluated statistical power with increasing numbers of markers analyzed under various assumptions and compared the sample sizes required in case-control studies and case-parent studies.
We computed the effective sample size and statistical power using a web browser program, Genetic Power Calculator developed by Purcell et al.
[27] (http://pngu.mgh.harvard.edu/~purcell/gpc/), for both case-control and case-parent studies.
We conducted power and sample size calculations under various assumptions about genetic models (i.e., allelic, additive, dominant, recessive, and co-dominant models), minor allele frequencies (MAFs), pair-wise LD, disease prevalence, case-to-control ratio, and number of SNP markers (i.e., single SNP, 500 K SNPs, and 1 M SNPs).
The values tested for heterozygous odds ratio (ORhet) were 1.3, 1.5, 2, and 2.5.
The power and sample sizes were calculated under different ranges of factors, such as MAF of 5%, 10%, 20%, and 30%; LD of 0.4, 0.6, 0.8, and 1; disease prevalence of 0.01%, 0.1%, 5%, and 10%; and case-to-control ratio of 1:1, 1:2, 1:3, and 1:4.
We assumed Hardy-Weinberg equilibrium at the disease-susceptible allele.
The Bonferroni p-value that was specific to the number of SNP makers tested was applied to cover 3 billion base pairs of the human genome (i.e., p = 0.05 for a single SNP marker, p = 1 × 10-7 for 500 K SNP markers, and p = 5 × 10-8 for 1 M SNP markers).
We fixed the proper range of sample sizes from 100 to 2,000 cases, because the power is too low when the sample size is below 100 cases (or trios), and the cost is too high to realistically collect samples when the sample size is above 2,000 [7, 22].
We calculated the sample size to achieve 80% statistical power according to the genetic models and the heterozygous ORs using a single SNP marker in a case-control study under the assumptions of 5% disease prevalence, 5% MAF, complete LD, 1:1 case-to-control ratio, and 5% type I error rate (α) (Table 1).
The dominant model required the smallest sample size to achieve 80% power compared to other genetic models (e.g., 90 cases).
In contrast, the effective sample size to test a single SNP under the recessive model was too large to collect with a limited budget, even if the homozygous OR is greater than 4 (e.g., 1,536 cases).
It reveals difficulty in detecting a disease allele that follows a recessive mode of inheritance with a moderate sample size.
The sample size and statistical power for the allelic test in a case-control study under the different assumptions of ORhet, MAF, disease prevalence, LD, and case-to-control ratio by allowing a 5% type I error rate are shown in Fig.1.
As shown in Fig.1A, a lower sample size was required to test allelic association for a single SNP with a larger MAF at the same risk of disease (OR) under the assumptions of 5% disease prevalence, 5% α, complete LD, and 1:1 case-to-control ratio.
The minimum number of cases decreased from 1,974 cases for a SNP with a MAF of 5% to 545 cases for a SNP with a MAF of 30% under the same assumption.
A high-risk allele showing a high OR requires a smaller sample size to be detected under the same assumption.
While an allele with an OR of 1.3 requires 1,974 cases and 1,974 controls to be significantly detected in a case-control study, a SNP with an OR of 2.5 can be detected in a study of 134 cases and 134 controls under the assumption of a MAF of 5%, disease prevalence of 5%, type I error rate of 5%, and D&#39; of 1 (Fig.1A).
The higher prevalence and the higher LD were associated with increased statistical power: for instance, as the LD increased from 0.4 to 0.6, 0.8, and 1, the statistical power obtained from a study of 1,000 cases and 1,000 controls was obviously increased from 26.5% to 49.2%, 72.8%, and 88.4%, respectively, under the assumption of OR 1.3, 5% MAF, 5% prevalence, and 5% α level (Fig.1B and 1C).
In addition, a 1:4 case-to-control ratio, which is the golden standard ratio for the numbers of cases and controls to be collected in a case-control study, showed the most effective sample size to achieve 80% statistical power.
In many clinical settings, researchers are able to obtain more data from affected individuals than healthy individuals.
On the other hand, there are more healthy participants than participants with a disease in a population-based study.
Therefore, the minimum numbers of cases and controls required to achieve 80% statistical power depend on the study design.
For a SNP with an allelic OR of 2 and 5% MAF, 127 cases and 508 controls are required in the case of a 1:4 case-control ratio, whereas 248 cases and 248 controls are required in the case of a 1:1 ratio to achieve 80% statistical power under the assumption of 5% prevalence, complete LD, and 5% α level.
In Table 2, we compared the number of cases to the number of case-parent trios to perform a case-control study and a study using case-parent trios by increasing the number of SNPs being analyzed.
Genetic association studies with larger numbers of SNP markers require a larger sample size to reduce false positive association due to testing multiple hypotheses.
The sample size required in a case-parent study is generally larger than that of a case-control study design.
For instance, 248 cases and 248 controls (496 individuals) were required to detect a SNP with an ORhet of 2 and 5% MAF in a case-control study, whereas 282 case-parent trios (846 individuals) were required under the assumption of 5% disease prevalence and complete LD by allowing a 5% α level.
However, the sample sizes required in both study designs increase tremendously in a GWAS.
Under the same assumptions as shown above, the number of samples increased from 248 cases for a single SNP analysis to 1,206 cases and 1,255 cases for analyses of 500 K SNPs and 1 M SNPs, respectively, based on the threshold of p-value, calculated using a strict Bonferroni correction for multiple hypotheses comparisons.
The statistical power to test the same number of subjects was higher for the case-control design than for the case-parent trio design (Fig.2).
Both designs of the case-control study and case-parent study are used widely in the field of genetic epidemiology for studying associations between genetic factors and the risk of disease.
Over the past 2 decades, there has been a steep increase in the number of genetic association studies, and these studies have successfully reported a number of gene variants associated with human complex diseases [1, 4, 5, 28].
Recently, GWASs, a new frontier in genetic epidemiology, have identified thousands of new gene variants related to human diseases [29].
The population-based studies with a large sample size have increased statistical power, which leads to smaller variance.
However, it requires too much money and takes too long to collect a sufficient number of samples, and these large-scale studies are more likely to be affected by systematic bias and noise [25, 30].
In the current study, we demonstrated the effective sample sizes that are required to achieve 80% statistical power for a case-control study and case-parent study separately under various assumptions regarding effect size, MAF, disease prevalence, LD, case-to-control ratio, and number of SNPs.
A lower sample size is required under the dominant model in any assumption, while the recessive model requires too many samples under the same assumptions to achieve adequate statistical power.
Further, we confirmed that a lower sample size is required for testing more common SNPs with stronger effect sizes and increased LD between marker allele and disease allele.
A lower sample size is required to study a common disease than a rare disease.
The statistical power increases by increasing the number of controls per case; however, a case-to-control ratio exceeding 1:4 does not yield a significant increase in statistical power.
Among the parameters tested, under the assumption of a high level of LD between a marker and disease variant, a much reduced sample size is needed to detect evidence for association.
Common variants are more informative than rare variants in LD-based indirect association studies on complex diseases.
It means that researchers can reduce the cost by choosing common variants to be genotyped at the design stage of an LD-based association study.
In general, the case-control study design is more powerful than the case-parent study design [8, 10].
Since patients with family histories of the disease are more likely to inherit disease-predisposing alleles than patients without family histories of the disease, researchers can improve the statistical power by sampling patients with affected relatives and by comparing to controls without any family history in case-control association studies [13].
Genome-wide case-control studies have been used to identify genetic variants that predispose to human disease with model assumptions for parameters, such as the inheritance model.
Such studies are powerful in detecting common variants with moderate effect in the occurrence of a disease; however, a study with a large number of SNP markers by using 500 K or 1 million chips requires a large number of samples (e.g., thousands of cases and controls) to achieve adequate statistical power.
A researcher can rarely successfully conduct a large-scale association study without collaboration using a high-throughput microarray chip, in which most embedded SNPs reveal a small effect size between 1.3 and 1.6 [31].
Therefore, researchers who are planning a genetic association study must calculate the effective sample size and the statistical power in the design phase to perform a cost-effective study that reduces false negative and false positive test results.
Although we could not cover all plausible conditions in study design, the estimates of sample size and statistical power that were computed under various assumptions in this study may be useful to determine the sample size in designing a population-based association study.
The statistical power for the allelic test in a case-control study according to (A) minor allele frequency (MAF), (B) disease prevalence, (C) linkage disequilibrium (LD), and (D) case-to-control ratio (M, MAF; P, prevalence; D, LD; R, case-control ratio; A1=1.3, A2=1.5, A3=2, and A4=2.5 for heterozygous odds ratios).
The statistical power according to the number of single-nucleotide polymorphism (SNP) markers for the allelic test in (A) a case-control study and (B) a case-parent study (a1 = 0.05, a2 = 1×10-7, and a3 = 5×10-8 denote the significance thresholds according to the number of markers; A1.3, A1.5, A2, and A2.5 denote the odds ratios of heterozygotes).
MAF, minor allele frequency; D&#39;, linkage disequilibrium.
Title: Number of cases required to achieve 80% power according to the different genetic models in a case-control study

Title: Sample sizes with 80% power by increasing number of SNP markers in case-control and case-parent studies
Title: Performance Comparison of Two Gene Set Analysis Methods for Genome-wide Association Study Results: GSA-SNP vs i-GSEA4GWAS

Gene set analysis (GSA) is useful in interpreting a genome-wide association study (GWAS) result in terms of biological mechanism.
We compared the performance of two different GSA implementations that accept GWAS p-values of single nucleotide polymorphisms (SNPs) or gene-by-gene summaries thereof, GSA-SNP and i-GSEA4GWAS, under the same settings of inputs and parameters.
GSA runs were made with two sets of p-values from a Korean type 2 diabetes mellitus GWAS study: 259,188 and 1,152,947 SNPs of the original and imputed genotype datasets, respectively.
When Gene Ontology terms were used as gene sets, i-GSEA4GWAS produced 283 and 1,070 hits for the unimputed and imputed datasets, respectively.
On the other hand, GSA-SNP reported 94 and 38 hits, respectively, for both datasets.
Similar, but to a lesser degree, trends were observed with Kyoto Encyclopedia of Genes and Genomes (KEGG) gene sets as well.
The huge number of hits by i-GSEA4GWAS for the imputed dataset was probably an artifact due to the scaling step in the algorithm.
The decrease in hits by GSA-SNP for the imputed dataset may be due to the fact that it relies on Z-statistics, which is sensitive to variations in the background level of associations.
Judicious evaluation of the GSA outcomes, perhaps based on multiple programs, is recommended.
Gene set analysis (GSA) is useful in understanding the biological mechanisms underneath a phenotype by assessing the overall evidence of association of variations in an entire set of genes with a disease or a quantitative trait.
GSA has recently been used to investigate many common diseases as an approach for the secondary analysis of a genome-wide association study (GWAS) result [1-3].
Currently, several software tools for GSA are available: GSA-SNP [4], i-GSEA4GWAS [5], GSEA-SNP [6], GeSBAP [7], and so on.
GSA-SNP is useful only when p-values of the single nucleotide polymorphism (SNP) markers are available.
While GSA-SNP has implemented several options for estimating the significance of a gene set, its implementation of Z-statistics may be the most convenient.
Other methods require permuted p-values that are obtained from sample permutation trials; this requires lengthy computation runs.
The Z-statistics method accepts only one set of unpermuted original p-values and compares the score of a gene set against the background distribution made by all the genes; these p-values should be readily available for a typical GWAS.
Similarly, i-GSEA4GWAS also uses only the original set of p-values and thus is as convenient as GSA-SNP.
Instead of sample permutation, it estimates the significance of a gene set via SNP permutation [8].
One of its unique features is a scaling step that emphasizes the gene sets that are enriched with strongly associated genes.
Often, these approaches give different results in terms of the number of gene set hits.
Hence, we compared these two methods using the same dataset while controlling the input parameters as much as possible.
Here, we reanalyzed the type 2 diabetes mellitus (T2DM) GWAS results for the Korean population against the Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway databases.
The GWAS has been done with both the original unimputed and imputed genotypes.
A large discrepancy in the number of significant gene set hits was observed between the two programs as well as the gene set databases.
We also observed that the results were strongly affected by imputation.
Title: Genotyping, imputation, and GWAS

Korean samples used in the East Asian T2DM meta-analysis have been described by Cho et al.[9].
The samples included 1,042 cases with T2DM and 2,943 controls from the Korea Association Resource (KARE) project that were recruited from Ansan and Ansung population-based cohorts, aged 40 to 69.
We used the p-values of both the original unimputed (259,188 SNPs) and imputed (1,152,947 SNPs) genotypes, based on Affymetrix Genome-Wide Human SNP array 5.0 (Affymetrix, Santa Clara, CA, USA), that were made by removing samples and markers that failed a quality control test [10].
SNP imputation has been described by Cho et al.[9].
Briefly, IMPUTE, MACH, and BEAGLE were used, together with haplotype reference panels from the Japanese (JPT) and Han Chinese (CHB) samples that are available in the HapMap database on the basis of HapMap build 36.
Title: GSA-SNP and i-GSEA4GWAS

The GO database was downloaded from the Gene Ontology Consortium (12,902 terms) [11], and the KEGG pathway database was downloaded from the KEGG (311 terms) [12].
For both databases, only the gene sets having 10-200 member genes were used (3534 GO and 211 KEGG terms).
For both GSA-SNP and i-GSEA4GWAS, 20-kb padding was added to both ends of each gene.
Usually, these methods, like i-GSEA4GWAS, pick up the best p-value and assign it to the encompassing gene.
On the contrary, GSA-SNP allows one to choose different schemes for assigning the SNP p-values to each gene: either the best or the second best p-value within the gene boundary.
Choosing the second best p-value has been recommended, as it may reduce the false positive associations with little loss of sensitivity [4].
For GSA-SNP, we downloaded the standalone program (as of Jan. 2011) and executed it locally.
For i-GSEA4GWAS, we used the web server version by uploading the SNP p-values.
GSA-SNP allows several approaches of evaluating gene set significance.
While other approaches require p-values from permutation tests, Z-standardization requires no permuted p-values.
The score of a gene is defined as -log of the p-value assigned to the gene.
For each gene set, the scores of its member genes are averaged, and the Z-statistics of these scores are used to estimate the significance under the assumption of a normal distribution.
The effect of multiple testing is corrected by the false discovery rate (FDR) method [13].
On the other hand, i-GSEA4GWAS compares the distribution of the member gene scores of a gene set to all the genes using K-S statistics and corrects the multiple testing effect using FDR that is based on SNP permutation tests.
Variation in the number of member genes among gene sets is taken care of by multiplying a ratio of &#39;highly significant&#39; genes in a gene set relative to those among all genes.
Here, the &#39;highly significant&#39; genes are defined as the genes that map with at least one of the top 5% of all SNPs in the dataset.
The performance of GSA-SNP and i-GSEA4GWAS looked quite different in terms of the number of hits (Table 1).
GSA-SNP detected 27-94 hits for either the imputed or unimputed genotype dataset with GO gene sets, while 9-20 hits were detected with KEGG gene sets.
Consistently, more gene sets were hit with the scheme of using the second-best p-value of a gene than the best one.
Previously, it has been recommended that one assign the second-best, not the best, p-value to a gene due to a concern that the use of the best p-values of a gene may produce more false positives than the second-best one [4].
We compared the distribution of gene scores that were calculated based on the best and the second-best p-values using a high-volume scatter plot that represented the local density of points by a false color representation (Fig.1).
One may notice the densely populated points along the diagonal axis, meaning that the differences in gene scores were small for the majority of genes.
Nevertheless, there were many genes off-diagonal; for these genes, the gene scores that were calculated with the best p-values were larger than those calculated with the second-best p-values.
While this effect was negligible for the genes having high scores (p &lt; 10-4), many genes having low scores displayed large differences.
The latter were genes that had only one SNP within its boundary plus 20 kb of padding standing out in terms of significance and the rest falling short.
If the best SNP of a gene had been located within a strong linkage disequilibrium (LD) block, the second-best SNP would have been chosen from this block with a p-value close to the best one.
On the other hand, if the best SNP is in weak LD with the second-best one, they would differ from each other considerably.
Considering that Fig.1 was based on the highly densely imputed genotypes, those genes that showed a large difference may have been located within narrow LD blocks or recombination hot spots where imputation may be invalid.
Indeed, the genes that were not located within haplotype blocks showed larger differences in gene scores than those located within haplotype blocks (Supplementary Fig.1).
For those genes located within haplotype blocks, there were inverse relationships between the block size and the difference in gene scores (Supplementary Fig.2).
It is also possible that the apparent associations seen with the best p-values may have been due to random errors; assigning the second-best p-value to a gene may then reduce false associations.
One may argue that the large difference between the best and second-best scores for some genes may be due to the small number of SNPs for those genes.
If this is the case, there should be an inverse relationship between these two quantities.
As shown in Supplementary Fig.3, there is no such evidence.
Here is the rationale for the more hits made by GSA-SNP with the use of the second-best p-values than the best p-values in assigning them to a gene.
GSA-SNP assigns a score to a gene set by averaging gene scores (-logP) of its member genes and calculates Z-statistic by subtracting the mean of all gene scores from the gene set score.
Assigning the second-best p-value to a gene produces lower gene scores than assigning the best one.
While this effect is more pronounced with low-scoring genes, the high-scoring genes suffer little.
Since the mean of all gene scores would also decrease by using the second-best p-values compared to the use of the best p-values, the resulting Z-statistic for a gene set that is composed of high-scoring genes would then increase, yielding more hits.
The GSA-SNP runs with p-values of the imputed genotypes consistently produced fewer hits than with those of the unimputed genotypes.
Imputation guesses the genotype of an untyped marker and fills it in.
The GWAS p-value of an imputed SNP can be either larger or smaller than those of the neighboring genotyped, unimputed ones.
For the former cases, it would not change the best p-value that is assigned to a gene.
On the other hand, for the latter cases, the best p-value that is assigned to a gene can get smaller.
As shown in Fig.2, the imputation increased the significance of many genes.
This has the effect of reducing the number of hits for the GSA-SNP runs with the imputed dataset compared with the unimputed one, similar to the argument given above for the schemes of assigning the best or second-best p-values to a gene.
With the p-values of the imputed SNPs, i-GSEA4GWAS claimed 1,070 GO terms as significant, almost 40-fold more than GSA-SNP, which yielded only 27 terms.
Even with the p-values of the original unimputed SNPs, i-GSEA4GWAS produced about 4.6 times more terms than GSA-SNP (283 vs. 61).
It appears that i-GSEA4GWAS unrealistically produced too many hits (about 1/3 of the input GO terms), hampered by the high proportion of false positives.
This trend was much more pronounced with GO than KEGG, probably due to the more redundant nature of GO than KEGG.
Unlike GSA-SNP, which reported fewer hits with the imputed dataset than the unimputed one, i-GSEA4GWAS produced about 5 times more hits with the former than the latter.
Why did i-GSEA4GWAS perform even more poorly with the imputed dataset that should be more ideal in terms of marker density than the unimputed one?
i-GSEA4GWAS assigns the best p-value to a gene and evaluates the gene set score by comparing the distribution of gene scores (-logP) of a gene set to that of all gene scores.
The significance of the gene set score is estimated by the FDR, which compares the unpermuted distribution of the gene set scores with that of the gene set scores generated from a number of SNPpermuted datasets.
The gene set score is multiplied by a factor, k/K, where k represents the proportion of so-called &#39;significant&#39; genes within the gene set and K represents the proportion of &#39;significant&#39; genes of all genes.
i-GSEA4GWAS defines the &#39;significant&#39; genes as those that are mapped with at least one &#39;top 5%&#39; SNP.
This step is a unique feature of i-GSEA4GWAS, which has the prefix &#39;i&#39; in its name.
Generally, an imputed dataset has much higher marker density than the corresponding unimputed one (4.45-fold difference in our case).
The so-called &#39;top 5%&#39; SNPs will be more with the imputed dataset than the unimputed one (again, a 4.45-fold difference in our case).
This corresponded to p-value cutoff of 0.047 in our case.
The number of so-called &#39;significant&#39; genes does not increase as much as the increase in the marker density: from 2,967 genes for the unimputed dataset to 4,156 genes for the imputed one in our case.
The concept of augmenting gene set scores by the proportion of &#39;significant&#39; genes may be useful, as demonstrated previously in comparison with GSEA [5, 14].
However, the &#39;top 5%&#39; threshold used by i-GSEA4GWAS may be too high, inflating the false positive rates.
This inflation may be more pronounced with an imputed dataset - 1,000 more genes were treated as &#39;significant&#39; with our imputed dataset than with the unimputed one.
Probably, this is one of the main reasons for the particularly poor performance of i-GSEA4GWAS with the imputed dataset.
GSA is useful method in interpreting the result from a GWAS.
A systematic evaluation of its performance is of paramount interest to the GWAS community, as the method is getting popular.
Here, we compared the performance of two such methods using the common datasets and gene set databases.
While GSA-SNP behaved predictably, i-GSEA4GWAS produced too many hits for most of the test settings.
For example, i-GSEA4GWAS reported 3.8- and 6.5-fold more hits, respectively, for GO and KEGG, with the imputed dataset than with the unimputed one.
Imputation is such a useful practice that augments the power of a genotype dataset, and ideally, gene set analyses can benefit from it.
Our study warns that one must be cautious in applying i-GSEA4GWAS to an imputed dataset.
As we pinpointed above concerning the &#39;top 5%&#39; threshold as the potential cause of the high hit rates of i-GSEA4GWAS, it would be interesting to re-evaluate its performance with lower thresholds.
Currently, the threshold is not available for the users to change it.
It would have been better if the user could choose it at will.
For GSA-SNP, we recommend using an imputed dataset if at all possible.
GSA-SNP allows the user to choose k in assigning the k-th best p-value to a gene.
We recommend using k = 2 instead of k = 1, as the latter inflates the scores for some genes, diminishing the power of GSA.
Comparison of the gene scores calculated by two different schemes.
All 15,829 genes that were mapped by at least two single nucleotide polymorphisms (SNPs) (either genotyped or imputed) were included in the high-volume scatter plot that displays the local density of points by a false color representation.
For a given gene, the p-values of SNPs located inside or within 20 kb of the gene boundary were surveyed.
The best (or second-best) of their-log-transformed values was assigned as the gene score.
The X and Y axes represent the second-best and best values, respectively.
Comparison of the gene scores from the unimputed and imputed datasets.
All 15,829 genes that were mapped by at least two single nucleotide polymorphisms (SNPs) (either genotyped or imputed) were included in the high-volume scatter plot that displays the local density of points by a false color representation.
For a given gene, the p-values of SNPs located inside or within 20 kb of the gene boundary were surveyed.
The best of their-log-transformed values was assigned as the gene score.
The X and Y axes represent the gene scores from the unimputed and imputed datasets, respectively.
Title: The number of gene set hits identified by gene set analyses
Title: Characteristics in Molecular Vibrational Frequency Patterns between Agonists and Antagonists of Histamine Receptors

To learn the differences between the structure-activity relationship and molecular vibration-activity relationship in the ligand-receptor interaction of the histamine receptor, 47 ligands of the histamine receptor were analyzed by structural similarity and molecular vibrational frequency patterns.
The radial tree that was produced by clustering analysis of molecular vibrational frequency patterns shows its potential for the functional classification of histamine receptor ligands.
G protein-coupled receptors (GPCRs) are regarded to take up more than one-fourth of marketed human medicines [1, 2].
Drugs targeting GPCRs account for the majority of the best-selling drugs and about 40% of all prescription pharmaceuticals in the marketplace [3].
Histamine receptors belong to one family of rhodopsin-like class A GPCRs, and four subtypes are named in chronological order as H1, H2, H3, and H4.
Histamine exerts its effects through the activation of these four histamine receptors.
Each type of histamine receptor reacts to subtype-specific ligands into an active or inactive form.
GPCRs are integral membrane proteins that consist of 7 transmembrane segments connected by 3 intracellular and 3 extracellular loops of variable length.
The crystal structures of GPCRs with their binding ligands have revealed the features of the ligand binding pockets and extracellular loops [4-6].
A ligand of GPCR activates the receptor by changing the receptor structure to the active form.
The biogenic amine histamine [2-(1H-imidazol-5-yl) ethanamine] (Fig.1) is produced by decarboxylation of L-histidine and acts as a chemical mediator and neurotransmitter in central and peripheral tissues.
It acts as an important pharmacological modulator involved in the processes of allergy, inflammation, neurophysiology, and cancer [7-10].
The molecular understanding of ligand-receptor interactions of GPCR remains unclear and is still the subject of investigations.
There are several theories explaining the ligand-receptor interaction mechanism, such as shape theory, binding theory, and vibration theory [11, 12].
The structures of histamine receptor ligands are so variable that we can not easily classify the pharmacological function of the ligand.
To find any other characteristic in the molecular patterns between agonists and antagonists of histamine receptors, a computational approach to molecular vibration was carried out in an attempt to find a bit of the molecular interaction mechanisms.
Title: Dataset

The simplified molecular-input line-entry system (SMILES) and 3-dimensional structure data format (SDF) files of the dataset were downloaded from the PubChem Compound Database in National Center for Biotechnology Information (NCBI) and used in the further analyses.
All 47 ligand molecules in the dataset, comprising 9 histamine agonists and 38 histamine antagonists, are shown in Table 1.
Title: Structure comparison of histamine receptor ligands

The molecular similarities between histamine and other 46 chemicals were calculated from SMILES of the chemicals.
The structural similarity was calculated and represented as the Tanimoto distance of each molecule from histamine.
The Tanimoto coefficient for pairwise comparison of molecules is the most widely used measure of molecular structural similarity.
This coefficient is defined as Tc = Nab/(Na + Nb - Nab), with Nab being the number of common bits, Na the unique bits in molecule a, and Nb the unique bits in molecule b, using a molecular fingerprint [13].
In this study, the molecular similarity was calculated as the Tanimoto coefficient using the 38-bit set.
Title: Geometry optimization and calculation of molecular vibrational frequency

In order to calculate molecular vibrational frequency, the structure of a chemical must first be geometrically optimized.
Since each provided theoretical 3-D conformer SDF is not at an energy minimum and may not represent the lowest energetic form in a vacuum, solvent, or a binding pocket, each SDF file of a ligand molecule underwent conversion to a single low-energy conformation using the general atomic and molecular electronic structure system (GAMESS) program package [14].
Restricted Hartree-Fock (RHF) calculations using Becke&#39;s exchange and Lee-Yang-Parr&#39;s correlation functionals (BLYP) density functional theory (DFT) method with 6-31G basis set were performed to optimize the geometries of the molecules.
Each result was taken as the representative conformation of the molecule, although the calculation of molecular vibrational frequency has some dependence on conformation.
Each geometry optimization result was subjected to the calculation step for the vibrational frequency with RUNTYP of HESSIAN in the GAMESS program.
Title: Hierarchical clustering of the corralled intensity of molecular vibrational frequency (CIMVF)

For the simplified molecular comparison, the calculated vibrational frequencies of a molecule were then sorted in increasing order and taken into the corrals, the step size of which was 5.
The intensities of each frequency in the same corral were summed up as the representative of the corral in the frequency range of 0-5,000 cm-1.
As a final outcome, this potential molecular descriptor of each molecule was displayed in a 1-dimensional vector containing 1,000 elements.
Finally, the similarity matrix, comprising the descriptors of 47 ligands of histamine receptor, was then subjected to hierarchical clustering in the agglomerative manner.
In this study, the similarity matrix was finally clustered to make an unrooted tree of 47 vertices.
The calculations of CIMVF were performed by in-house scripts, written in Python.
The Tanimoto coefficients between histamine and other ligand molecules are shown in Table 2.
Typically, a Tanimoto coefficient &gt; 0.85 is considered highly similar, and a coefficient &gt; 0.75 is considered similar for the purpose of clustering molecules that may have similar biological activity profiles [15].
The highest Tanimoto coefficient among agonists was 0.45 (imetit), and this is not high enough to be considered a molecule that has potential agonist efficacy.
Moreover, the lowest value of Tanimoto coefficient among agonists was 0.08 (dimaprit and SKF91488) and is also a lower value as an antagonist.
It seems that there is no related pattern between Tanimoto coefficients and the functional types of molecules in the case of histamine receptor agonists or antagonists.
To search for a novel characteristic for the classification of histamine receptor ligands, a kind of molecular calculation using agglomerative hierarchical clustering was adopted in this work.
The result of the hierarchical clustering of the similarity matrix from CIMVF is shown in Fig.2.
As shown in the figure, eight agonists were located nearby (the part in the dotted circle), except impromidine, and all antagonists were clustered close to each other in the radial tree.
We can tell the regional difference between agonists and antagonists in the tree and also find that the information from the molecular vibrational frequency may play a role in the classification of agonists/antagonists for histamine receptor as a possible molecular descriptor.
For these methods, clustering with CIMVF shows the more proper result in the case of histamine receptor ligands.
With a more concentrated study on the relationship between the molecular vibrational frequency and pharmacological function of a ligand, the vibrational spectrum of a molecule may shed light on the field of ligand-receptor interaction mechanisms.
The chemical structure of histidine.
Radial tree of corralled intensity of molecular vibrational frequency (CIMVF) clustering using the complete linkage method.
Antagonists are tagged with &#34;t&#34; to their chemical names as a prefix, whereas agonists are not.
Agonists of histamine receptors except impromidine are located in a cluster (the part in the dotted circle).
Title: List of agonists and antagonists used in the present study

Title: The molecular similarity between histamine and each ligand molecule in Tanimoto coefficient
Title: Perspectives of Integrative Cancer Genomics in Next Generation Sequencing Era

The explosive development of genomics technologies including microarrays and next generation sequencing (NGS) has provided comprehensive maps of cancer genomes, including the expression of mRNAs and microRNAs, DNA copy numbers, sequence variations, and epigenetic changes.
These genome-wide profiles of the genetic aberrations could reveal the candidates for diagnostic and/or prognostic biomarkers as well as mechanistic insights into tumor development and progression.
Recent efforts to establish the huge cancer genome compendium and integrative omics analyses, so-called &#34;integromics&#34;, have extended our understanding on the cancer genome, showing its daunting complexity and heterogeneity.
However, the challenges of the structured integration, sharing, and interpretation of the big omics data still remain to be resolved.
Here, we review several issues raised in cancer omics data analysis, including NGS, focusing particularly on the study design and analysis strategies.
This might be helpful to understand the current trends and strategies of the rapidly evolving cancer genomics research.
In the last decade, numerous genomic studies have addressed the enormous complexity of the cancer genome.
Genomic profiling using microarray technology could stratify the tumors into homogeneous subgroups, providing novel clinical insights for the development of diagnostics and therapeutics as well as systematic views on the underlying mechanisms of tumor progression.
In addition to the microarray technologies, explosive advances on sequencing technologies have been made recently, which is called &#34;next generation sequencing (NGS).&#34;
Compared to the previous DNA sequencing of the Sanger method using dideoxynucleotide termination reaction termed as &#34;first-generation&#34; sequencing, NGS uses massively parallel sequencing method generating hundreds of millions of short (~200 bp) DNA reads, which can sequence a human genome rapidly with extremely lower cost.
The earlier NGS method with the single-end read sequencing inevitably produces the short-read problems, limiting the accuracy of genome alignment.
This could be improved by applying a paired-end sequencing method, allowing substantial advances in identifying not only point mutations but also genomic rearrangements, such as deletions, amplifications, inversions, translocations, and gene-fusions [1, 2].
The NGS technology is now divided into &#34;second generation sequencing&#34; and &#34;third generation sequencing.&#34;
The second generation sequencing refers to the strategies of short-read alignment, while the rapidly being developed technology of the third generation sequencing refers to the single DNA molecule based sequencing.
The third generation method has advantage of less amount of DNA input that allows the emerging field of single cell sequencing [3].
Moreover, there is no step for PCR amplification, therefore, the nucleotide incorporation errors can be handled.
However, all the platforms of NGS technologies still have limitations in accurate base calling and alignment.
The errors are likely to be platform-dependent, which increases the complexity of the data analysis.
Therefore, the cost for bioinformatic analysis, rather than the sequencing itself continues to grow, which is referred to as &#34;the $1,000 genomes, the $100,000 analysis&#34; problem [4].
The application of Exome-Seq and Whole Genome-Seq profiled a mutational spectrum in various cancers [5-8].
The RNA-Seq could profile not only the gene expressions but also new parameters such as allelic expression, alternative splicing [9, 10], RNA-editing [11], and alternative polyadenylation of 3&#39;-untranaslated region (3&#39;-UTR) [12, 13].
Structural variations such as gene-fusions, e.g., TMPRSS2-ERG in prostate cancer [14] and KIF5B-RET in lung cancer [15] have been identified from the analysis of NGS data.
Moreover, the patterns of genome rearrangements can be analyzed systematically.
For example, a novel pattern of genomic rearrangement such as fold-back inversion could be found by simply examining the short read alignment of NGS data [16].
A novel mechanism of cancer genome rearrangement i.e., chromothripsis has been proposed, which represent a catastrophic event of fragmentation and reassembly of a single chromosome [17].
In addition, high-resolution epigenomic profiles of cancer genome could be obtained by applying NGS to chromatin immunoprecipitation-sequencing (ChIP-Seq) and methylated DNA immunoprecipitation-sequencing (MeDIP-Seq).
For example, a recent study of the genome-wide DNA methylation profile showed novel patterns in which the majority of DNA methylation changes occurs at CpG island shores neighboring the regions up to 2 Kb from CpG islands, and revealed the patterns of cancer-specific and tissue-specific DNA methylation [18, 19].
Aberrant histone modification was also found in cancers showing an association with the patient&#39;s prognosis [20].
These studies support the pivotal role of epigenetic regulation in cancer development and progression.
Moving forward, new applications such as genome-wide translocation sequencing (HTGTS) [21] and translocation capture sequencing (TC-Seq) [22] have recently been proposed, which could profile the genome-wide translocation hotspots.
Notably, the advantage of NGS is not restricted to these applications.
It serves a platform for the identification of novel RNAs or DNAs.
For example, the long noncoding RNAs (lncRNAs) that are transcribed from intergenic and intronic regions have been identified in prostate cancers i.e., prostate cancer-associated ncRNA transcripts (PCATs) [23].
A novel class of DNA i.e., extrachromosomal microDNA, has been recently found by NGS technology, which is derived from unique non-repetitive sequence enriched in 5&#39;-UTR, exons, and CpG islands [24].
The role of microDNA in cancer will be the next question.
Title: Challenges in cancer genomics

The primary study goals of cancer genomics are not simple, which include the studies to get either clinical or mechanistic insights from the cancer genome.
To clarify the complicated study designs and strategies of cancer genomics, we have categorized the gene signatures obtained from cancer genome data into four classes prediction, phenotype, function, and molecular targets based on the study goals [25].
The majority of the previous studies have suggested the translational or clinical utility of the genomics data by addressing the candidate biomarkers or the prediction signatures for predicting patients clinical outcomes, such as recurrence, survival, metastasis, or response to therapies.
Notwithstanding the overwhelming identification of candidate biomarkers from the cancer genome, only a handful of candidate biomarkers that have been discovered from genomic analyses can succeed in the validation of the clinical utility [18].
There are several challenges in cancer genomics that preclude clinical utility.
One of them would be data reproducibility.
They might be due in part to the experimental biases as well as sample cohort issues.
The use of different platforms measuring gene expressions and different data processing methods could produce biased observation in each study.
Increasing sample size will be one of the solutions to find proper biomarkers, overcoming the reproducibility problem.
Undoubtedly, large-scale sample collection provides increased statistical power.
However, previous studies, even with large sample sizes, have often failed to reproduce their findings in independent studies [26].
This might be due mostly to the use of different protocols and analysis methods.
Moreover, biased sample collection may also affect the performance of prognostic biomarkers, leading to subsequent failure to validate the biomarker in another patient population [27].
For example, diagnostic biomarkers must be discovered in early-stage tumors; however, the sample collection of early tumors with enough of a sample size might be difficult in the clinical setting [18, 28].
In addition, the cost-effectiveness of the sample size enrolled in a study should be considered.
Simply increasing the sample size might not be the best solution.
The sample sources and qualities are also important factors to be considered in the study design.
For example, circulating DNAs or microRNAs in the plasma or urine can be used to develop &#34;noninvasive&#34; biomarkers in cancer patients, which might bring the technology much closer to the clinic [29, 30].
Attempts to use formalin-fixed paraffin-embedded (FFPE) tissues might also be more applicable to the clinic [31], although the quality and the quantity of the DNAs or RNA extracts from FFPE or plasma are still problematic for genomics studies requesting further elaboration.
Title: Dissecting the tumor heterogeneity

The data complexity comes not only from the heterogeneous or biased sample composition, but also from the innate complexity of tumor biology.
Previously, mounting evidence has shown the enormous heterogeneity of tumors at the molecular level.
The tumor heterogeneity can be explained by two hypothetical models.
One is the clonal segregation model with a multi-step process, and the other is the cancer stem cell theory.
The cancer stem cell model describes the heterogeneous cellular origin of cancers from primitive progenitor cells to mature differentiated cells, which may contribute to tumor heterogeneity.
In this context, genomic profiling studies could define the cancer subpopulation harboring stem-like traits in various cancer types, supporting the cancer stem cell theory [32, 33].
Similarly, we also defined the bilineal trait in a subpopulation of hepatocellular carcinoma (HCC) by comparing the gene expression profiles of HCC and cholangiocarcinoma (CC) [34].
This result showed the continuous liver cancer spectrum between HCC and CC, suggesting that stem-like or de-differentiation traits may give rise to the heterogeneous progression of HCC.
We also suggested that the dysfunction of p53 machinery is associated with the acquisition of the stemness trait in HCC [35].
Recently, this association could be validated by showing p53 knockout mouse model can give rise to bilineal liver cancers [36].
In addition, various host factors may contribute to tumor heterogeneity.
Interactions of the tumor cells with host eco-system, such as innate immune systems, or the reactions of surrounding microenvironment against the tumor may affect the tumor behaviors [37].
Thus, the proper detection of biomarkers might be difficult without considering the effect of host factors.
Furthermore, the intratumoral heterogeneity of cancers have been notified in detail by genome-wide sequencing of multi-loci from the same tumor [38].
The comparison of primary and metastatic tumors by single cell sequencing also revealed the sequential mutation process during cancer progression [39].
Similarly, the comparison of mutations from multiple HCC tumors in the same patient could define the evolutionary lineage among tumors cells [40].
Strikingly, the development of single-cell sequencing technology could provide a more detailed and systematic view on intra-tumoral heterogeneity [41, 42].
Of interest, such attempts enabled the construction of phylogenic trees from mutational heterogeneity, which revealed evolutionary tumor growth opening a new field of &#34;cancer evolution&#34;.
Title: Returning to biology for clinical utility

As discussed above, there are many factors contributing to tumor heterogeneity, which may impede the discovery of new biomarkers.
Thus, we are now urgently in need of developing new strategies for biomarker discovery from cancer genome data.
Considering the huge complexity of the cancer genome and the limitation of current technologies, it would be a reliable strategy to evaluate the functional relevance of the candidate biomarkers rather than simply showing the statistical significance of the association by enrolling larger samples or applying more stringent statistics.
Although the conventional strategies for biomarker discovery do not require the functional significance of the candidates [43], current hurdles in cancer genomics request a functional validation step in the pipeline of the biomarker discovery.
Our limited understanding of the complexity of cancer biology is a significant challenge for translational interpretation of the cancer genome.
Title: Challenges of big data issues and integromics

More recently, systematic structuring and integration of multiple and multi-layered omics data resources, i.e.
integromics, are thought of a state-of-the-art strategy.
The recent establishment of The Cancer Genome Atlas (TCGA) and the International Cancer Genome Consortium (ICGC) could accelerate and facilitate the integration and sharing of cancer genome data [44].
Multi-layered integromics could define tumor heterogeneity, revealing pivotal aberrations of genetic events or signaling pathways [45].
In parallel, genomic repositories for drug activities have been established [46-48].
Linking the profiles of drug sensitivity to the cancer genome could provide a powerful platform to guide rational and personalized cancer therapeutics.
Now, as genomic data are increasing and accumulating enormously, new study designs and analysis strategies for integromics might be required, particularly with the context of tumor heterogeneity and the discovery of the functional biomarkers.
As shown in Fig.1, the first step in cancer integromics is the dissection of tumor heterogeneity.
Then, the next step will be a recapitulation of the relations between clinical/biological phenotypes and molecular genotypes in cancer subpopulations, which can address novel functionalities in particular subpopulations of cancers.
We suggest that the discovery of reliable candidate functional biomarkers as well as functional genetic alterations, so-called &#34;driver events&#34;, can be achieved through performing this step-by-step evaluation of both clinical and functional utilities.
This hybrid study design would open an exciting era for developing new &#34;functional biomarkers&#34; and preventive/therapeutic strategies with the consideration of biological backgrounds of tumor heterogeneity.
We briefly reviewed the current challenges and perspectives of integrative cancer genomics, focusing particularly on the complexity of omics data and cancer biology.
Integromic approaches with functional evaluation should be considered even for clinical as well as mechanistic applications of cancer genome data.
Necessarily, the challenges of big data (particularly the NGS platforms) and integromics should be considered in the study design.
A deep understanding of both cancer biology and omics data characteristics is necessarily required for successful cancer genome analysis.
Moving forward, it is clear that progress will come through large-scale, wide-scope, and multi-disciplinary collaborations and sharing systems, which will accelerate the realization of translational and personalized medicine in the near future.
Hybrid study design for integromics approach.
Title: Alternative Splicing and Its Impact as a Cancer Diagnostic Marker

Most genes are processed by alternative splicing for gene expression, resulting in the complexity of the transcriptome in eukaryotes.
It allows a limited number of genes to encode various proteins with intricate functions.
Alternative splicing is regulated by genetic mutations in cis-regulatory factors and epigenetic events.
Furthermore, splicing events occur differently according to cell type, developmental stage, and various diseases, including cancer.
Genome instability and flexible proteomes by alternative splicing could affect cancer cells to grow and survive, leading to metastasis.
Cancer cells that are transformed by aberrant and uncontrolled mechanisms could produce alternative splicing to maintain and spread them continuously.
Splicing variants in various cancers represent crucial roles for tumorigenesis.
Taken together, the identification of alternative spliced variants as biomarkers to distinguish between normal and cancer cells could cast light on tumorigenesis.
Genes are transcribed into pre-mRNA and pass through splicing as a post-transcriptional modification to generate mature mRNA for translation [1].
Splicing is an essential process for gene expression in eukaryotes, eliminating introns and joining the exons, which occurs in the spliceosome, a complex of small nuclear ribonucleoproteins (snRNPs) and other proteins [2].
This splicing can generate alternative spliced transcripts from pre-mRNA and different exon constitutions, resulting in different proteins.
Alternative splicing can occur in different ways.
The most representative mechanisms are as follows: 1, extending or shortening the exon by alternative donor and acceptor sites; 2, exon skipping; 3, mutual exon exclusion; and 4, intron retention [1, 3, 4].
These mechanisms allow protein isoforms with different biological characters to be produced from single genes [1].
Therefore, complex transcriptomes and proteomes could be derived from a limited number of genes [4].
Namely, this process is an important strategy for the complicated regulation of eukaryotes, and most genes (92-95%) undergo this process [5, 6].
Alternative splicing is regulated according to cell type, developmental stage, and disease states [1, 7, 8].
Biochemical mechanisms for the recognition of splice sites are not understood clearly according to cellular conditions, but some tissue-specific factors participate in alternative splicing [4].
In addition, quantitative gene expression is controlled by nonsense-mediated decay mechanisms that degrade targeting mRNAs, producing nonsense mutations.
Thus, truncated or erroneous proteins with abnormal functions are prevented from being expressed [9].
Alternative transcripts could be related to various diseases, including cancer.
As many as 50% of genetic diseases of humans are related to mutations in splice site sequences and regulatory elements, such enhancers and silencers, resulting in alternative exon constitution [3, 10, 11].
Recently, the SpliceDisease database, providing information for relationships among gene mutation, splicing defects, and diseases, was reported [12].
Especially, aberrent spliced variants are found frequently in cancer, indicating that they could play a role for the survival of cancer cells [8].
Alternative splicing of cancer-related genes could affect cell cycle control, signal transduction pathways, apoptosis, angiogenesis, invasion, and metastasis [8, 13].
Cancer markers allow us to determine the prognosis and therapy for cancer during the remedy of cancer.
Thus, the identification of cancer markers is highlighted in the cancer research field [14].
Cancers result from the accumulation of complex genetic and epigenetic alterations against normal regulation.
Cancerous cells grow irregularly, create malignant tumors, and move to the other parts of the body.
Their alternative spliced transcripts could be detected with cryptic splicing sites.
Accordingly, alternative transcripts produced by splicing events represent good candidates for cancer biomarkers [7, 8, 13].
In the present review, we summarize and discuss the alternative splicing events and their potential as cancer biomarkers.
Alternative splicing is an essential post-transcriptional process for creating various protein isoforms from the same gene.
Approximately 60% of human genes have at least one alternative splice transcript [15].
Alternative splicing is regulated by mutations in genetic regions [3, 7, 14].
Most exons are flanked by the intronic dinucleotides GT (donor site) and AG (acceptor site), which are recognized by the spliceosome (Fig.1A).
Pre-mRNA is reconstructed only by joined exons with the removal of introns.
However, mutations of these splice sites lead to aberrant splicing, producing exon skipping and different joining of exons to truncated or nonfunctional or dysfunctional proteins.
It was reported that mutations of splice sites may play important roles in human disease, and longer proteins also tend to associate to various disease [16, 17].
In the case of the PAX6 gene, a 5&#39; splice site mutation in intron 12 induces exon skipping in relation to autosomal dominant aniridia [18].
In the APC gene, abnormal splicing by a 3&#39; splice site mutation in intron 3 causes exon 4 skipping due to a frameshift in hepatoblastoma [19].
A mutation in intron 25 of the ABCA3 gene creates novel 5&#39; splice sites [20].
The spliceosome is known to recognize cryptic splice sites instead of typical splice sites.
Noncanonical splice sites are GC-AG, GG-AG, GT-TG, GT-CG, AT-AG, GA-AG, GT-AC, and CT-AG (5&#39;-3&#39; splice sites) [21-23].
Mutations at the noncanonical splice site of the SEDL gene cause variants in X-linked spondyloepiphyseal dysplasia tarda [24].
Additionally, less exon skipping and more intron retention by alternative splice sites are observed in cancer tissue compared to normal tissues from an analysis of expression sequence tags (ESTs) [25].
As a mutation of pre-mRNA, A to I RNA editing (conversion adenosine to inosine from deamination) also affects transcriptome diversification.
RNA editing has regulatory roles, such as altering splice sites and sequences necessary for recognition of the spliceosome, resulting in modulation of alternatively spliced transcripts [26].
The last case of alternative splicing is by cis-genomic mutations in regulatory factors, such as branch sites, exonic and intronic splicing enhancers, and silencers [7, 27].
Pre-mRNA has exonic and intronic splicing enhancers (ESEs, ISEs) and silencers (ESSs, ISSs) that promote exon inclusion and exclusion by regulation of splice site recognition, respectively (Fig.1A).
The arginine-serine-rich (SR) protein that binds to ESEs induces splicing via a helping assembly spliceosome by interacting with snRNP.
On the contrary, heterogeneous nuclear ribonucleoprotein is bound to ESEs and ISEs and inhibits splice site recognition by blocking spliceosome assembly [15, 27-29].
Mutations in regulatory factor could disturb the binding of these spliceosome assembly-related proteins.
Mutations at nucleotide positions 57 and 58 of the 174-bp-long exon 7 cause exon 7 skipping as a result of aberrant splicing by interrupted ESE-specific consensus sequences that are recognized by the SC35 and SF2/ASF SR proteins [10, 30].
Additionally, Ron, encoding the tyrosine kinase receptor for macrophage-stimulating protein, has alternative splice transcripts.
It is regulated by overexpression of SF2/ASF binding to ESE and ISE in colon and breast cancer [31].
Alternative splicing is also known to be affected by epigenetic regulation, such as DNA methylation, histone modification, and chromatin structure (Fig.1B).
The relationship between chromatin structure and alternative splicing is still in a maze, but association studies are gradually increasing genomewide [32-34].
Hisone modifications are enriched in exons rather than introns and related to exon expression, especially H3K36me3, H3K79me1, H2BK5me1, H3K27me1, H3K27me2, and H3K27me3 [35, 36].
H3K36me3 marking in exons is found in weakly expressed, alternatively spliced exons, indicating that histone modification has a relation to transcription via splicing-related marking mechanisms [36, 37].
These histone marks could recruit splicing regulators with chromatin binding proteins and affect mRNA splicing [33].
Additionally, hisone acethylation could modulate splicing rates to react quickly to changing conditions with increased RNA polymerase II processivity, and spliceosome rearrangements are affected by histone acetylation [38].
Nevertheless, although there is little evidence, DNA methylation has been reported to have a relationship with splice sites.
CpG dinucleotides are distributed nonrandomly in the genome.
Exon skipping and mutually exclusive exons have significantly lower levels of both CG and mCG in the exonic regions, whereas intron retention has significantly higher levels of CG in both exonic and intronic regions [34].
A DNA-binding protein, CCCTC-binding factor (CTCF), was inhibited by a methylation event of CD45 exon 5 [39].
These epigenetic features are strongly associated with alternative splicing.
Furthermore, these mechanisms are known to be changed according to cell type and disease states.
Especially, in cancer, the epigenetic regulation of chromatin structure effects aberrant gene expression by alternative splicing in cancer [1, 7, 8].
Taken together, gene expression via alternative splicing is altered by complicated and mutual mechanisms, from genetic to epigenetic regulation.
Therefore, it casts light on the understanding of cancer mechanisms by an investigation of alternative splicing patterns.
A number of studies have been reported that alternative splicing is related closely to development, cellular stress, and various diseases, including cancer, as a crucial contributor to transcriptome and proteome diversity [1, 7, 8, 14].
In cancer, with increasing genomic instability, sequence substitution and aberrant alternative splicing occur frequently, leading to erroneous and dysfunctional proteins [14].
Protein isoforms made from this process are developmentally regulated and preferentially re-expressed in cancer and help the differentiation and survival of cancer cells (Fig.2).
The development of genomewide analysis allows large-scale examination of the relationship between alternative splicing and tumorigenesis [40, 41].
Every case of alternative splicing is reported in cancer; among these, the most frequent case is the mutually exclusive exon [14].
For example, overexpression of CD44, involved in cell proliferation, differentiation, migration, and alternative splicing, by different splice sites of CD44 during tumorigenesis indicates that it could play roles in tumor cell invasion and metastasis [42-45].
Tumor suppressor genes, such as p53 and PTEN (Phosphatase and Tensin homolog, deleted on chromosome TEN), have splicing variants associated to cancer [46].
p53 protein isoforms through alternative splicing have critical roles in many biological processes, indicating that dysregulation effects tumorigenesis [47].
Different expression of PTEN and its alternatively spliced transcripts are found to vary in different tissue types.
PTEN regulates p53 stability and in turn regulates its own transcriptional activity.
The PTEN splice variants retained in intron 3 region and intron 5 region have been found in breast cancer [46].
In the case of the APC gene, aberrant splice skipping of exon 4, created by insertion of T in intron 4, is involved in colon cancer [48].
An alternative 5&#39; splice site in BCL-X results in 2 isoforms, long and short (Bcl-x (L), Bcl-x (X)), which have contrasting functions related to apoptosis and are overexpressed in various tumors [49, 50].
Although specific roles of vascular endothelial growth factor (VEGF) isoforms are not known exactly, among the isoforms of VEGF4 involved in formation of new vessels, VEGF165 and VEGF165b, created by an alternative 3&#39; splice site, have different expression in cancer [51, 52].
Over 40 different MDM2 transcripts are identified in normal and cancer by alternative 5&#39; and 3&#39; splice sites.
They mostly lose the p53 binding domain and promote tumor progression and affect prognosis independently [53, 54].
Increasing number of reports have demonstrated the expression of aberrant and abnormal splice variants in cancer cells or tissues.
However, it is not enough evidence for a functional relationship between alternative splicing and cancer.
Hallmarks of the initiation and early growth of cancers during tumor progression using large-scale analyses of splicing variants in the cancer EST database are needed as new RNA prognosis markers.
Cancer is an uncontrolled state and irregularly altered genetically and epigenetically compared to normal regulation (Fig.2).
For this reason, many studies have made an effort to identify specific features and regulation mechanims of various diseases, including cancer.
It is important and necessary to identify cancer markers to be able to distinguish between cancer and normal cells.
Cancer markers could be very helpful in understanding tumorigenesis and developing tumor targets for therapeutic intervention.
However, there still remain unsolved problems in spite of a number of studies [7, 14, 45, 64].
From this point of view, alternatively spliced transcripts by mutation and cryptic splice site altered expression levels in cancer have emerged as strong candidates of cancer biomarker at the mRNA and protein level [7].
As written previously, some alternative transcripts of several genes, CD44, p53, PTEN, BCL-X, VEGF4, and MDM2, have been discovered (see also Table 1) [43, 46, 49, 50, 52, 54].
They are also associated with various cancers directly or indirectly.
Recently, the discovery of biomarkers has improved by genomewide analysis [13, 65, 66].
It could be expected to provide valuable information for the association between alternative splice variants and cancers.
Thus, novel candidate variants could contribute to the development of diagnostic, prognostic, and therapeutic markers.
RNA splicing is a core mechanism to generate mature mRNA for translation, and alternative splicing is an indispensable mechanism, stratgically leading to protein diversity for complicated regulation in eukaryotes.
In cancer, alternative splicing is more flexible, leading various proteins with aberrant functions to promote growth and the spread of cancer cells.
It is important to identify alternative transcripts that function specifically in cancer.
These alternative transcripts could be used not only as diagnostic biomarkers but also prognostic and theraputic biomarkers.
Therefore, studies on genetic and epigenetic regulation in relation to alternative splice variants in cancer could open new windows of research in answering unsolved questions of tumorigenesis.
Genetic and epigenetic regulation of alternative splicing.
(A) Most genes are spliced by typical splice sites (GT-AG) and regulatory sequences in exons (ESS and ESE) and introns (ISS and ISE).
Mutations (*) generate alternative splicing, resulting in various transcripts from a single gene.
(B) H3K36me3 marking in exons is found in weakly expressed alternatively spliced exons (grey thin dotted line), and H3K4me3 marks are rich in spliced exons (black thick dotted line), indicating that histone modification has a relation to transcription via splicing-related marking mechanisms.
Additionally, hisone acethylation could modulate splicing rates to react quickly to changing conditions with increased RNA polymerase II processivity, and spliceosome rearrangements are affected by histone acetylation.
High levels of CG have a tendency not to be spliced (grey line in intron), and exon skipping is in the lower level of both CG and mCG in exonic regions.
Methylation of exon inhibits binding of CCCTC-binding factor (CTCF), leading to exon skipping.
ESE, exonic splicing enhancers; ISE, intronic splicing enhancers; ESS, exonic and intronic splicing silencers; ISS, intronic splicing silencers; SR, arginine-serine-rich; RNP, ribonucleoprotein.
Roles of alternative splicing in cancer.
Dysfunctional proteins are generated by alternative splicing in cancer-related genes (oncogenic and tumor suppressor genes), increasing genomic instability.
These aberrant proteins help cancerous cells to promote their growth, proliferation, and metastasis continuously (tumorigenesis).
Variants of cancer-related genes expressed specifically or differently with their crucial roles for tumorigenesis could be potential biomarkers to distinguish between normal and cancer cells.
Title: Alternative spliced transcript related to cancer
Title: Identifying Copy Number Variants under Selection in Geographically Structured Populations Based on F-statistics

Large-scale copy number variants (CNVs) in the human provide the raw material for delineating population differences, as natural selection may have affected at least some of the CNVs thus far discovered.
Although the examination of relatively large numbers of specific ethnic groups has recently started in regard to inter-ethnic group differences in CNVs, identifying and understanding particular instances of natural selection have not been performed.
The traditional FST measure, obtained from differences in allele frequencies between populations, has been used to identify CNVs loci subject to geographically varying selection.
Here, we review advances and the application of multinomial-Dirichlet likelihood methods of inference for identifying genome regions that have been subject to natural selection with the FST estimates.
The contents of presentation are not new; however, this review clarifies how the application of the methods to CNV data, which remains largely unexplored, is possible.
A hierarchical Bayesian method, which is implemented via Markov Chain Monte Carlo, estimates locus-specific FST and can identify outlying CNVs loci with large values of FST.
By applying this Bayesian method to the publicly available CNV data, we identified the CNV loci that show signals of natural selection, which may elucidate the genetic basis of human disease and diversity.
It has become known that human genomes differ more as a consequence of structural variation than of single-base-pair differences [1-4].
Structural genomic variants, mainly in the form of copy number variants (CNVs), have recently been rediscovered as contributors to evolution and as pathoetiologic elements for complex human diseases [5, 6].
However, so far, there are not many CNV-based genome-wide association studies or population genetics studies compared with the single-nucleotide polymorphism (SNP)-based counterparts.
One notable CNV-based population genetics study was done by Jakobsson et al.[7].
They applied SNP, haplotype, and CNV information to reveal the structures of 29 world-wide populations.
Even though their CNVs were not as effective as SNPs or haplotypes in discriminating those populations, the authors showed the potential to apply this information to human genetic studies.
Other examples of CNV-based disease susceptibility studies are systemic autoimmunity diseases [8, 9], psoriasis [10], and human immunodeficiency virus (HIV) infection and progression to acquired immune deficiency syndrome (AIDS) [11].
Disease could affect the patterns of CNV diversity via natural selection, and higher copy numbers of the immunoregulatory and inflammatory cytokine gene CCL3L1, for example, are associated with lower risks of HIV infection and the progression to AIDS [11]; furthermore, in a genome-wide study, the level of population differentiation at this locus was found to be extraordinary compared to that of other CNVs, suggesting that natural selection may have influenced CCL3L1 copy number in humans [3].
The traditional FST measure is useful for identifying regions of the genome affected by natural selection.
In a recently published review paper, Holsinger and Weir define Wright&#39;s F-statistics (FST in particular) [12] and describe methods-of-moment estimates and how FST estimates should be interpreted [13].
Although the authors also mention the maximum likelihood Bayesian estimates of FST, their description is very limited in identifying genomic regions under selection and do not present an application to the datasets.
They simply compared locus-specific estimates of FST with its genome-wide distribution, and therefore, probabilities are not attached to those with a higher FST.
Here, we outline the steps that constitute testing for outlier loci in population datasets by the maximum likelihood Bayesian method, using BayeScan computer software [14] (Note that the terms of locus and CNV are used interchangeably in this paper).
The logic for determining outlier loci is simple.
If natural selection favors one allele over others at a particular locus in some populations, the FST at that locus will be larger than at loci in which among-population differences are purely a result of genetic drift.
Therefore, genome scans that compare single-locus estimates of FST with the genome-wide background might identify regions of the genome that have been subjected to diversifying selection [15].
We will demonstrate FST estimation by maximum likelihood Bayesian method with the publicly available high-resolution CNV dataset generated by Conrad et al.[4].
We will also show that this method identifies several genomic regions showing signals of natural selection.
Even though we focus mainly on the CNV data here, the detailed steps of analysis are analogous for other types of molecular marker data, such as microsatellites, SNPs, and amplified fragment-length polymorphisms.
Title: F-statistics

Title: Wright [12] introduced F-statistics (FST, FIT, and FIS) as a tool for describing the partitioning of genetic diversity within and among populations that are directly related to the rates of evolutionary processes, such as migration, mutation, and drift.
Specifically, F-statistics can be defined in many different ways: in terms of variances of allele frequencies, correlations between random gametes, and probabilities that two gametes chosen have different alleles.
Depending on the relativity to the subpopulation or to the total population, FST, FIT, and FIS are defined, where subscript IS refers to &#39;individuals within subpopulations,&#39; ST to &#39;subpopulations within the total population,&#39; and IT to &#39;individuals within the total population.&#39;
Following the work of Cockerham [16], F-statistics are defined in terms of the variance components - that is, the total variation in the genetic data is broken down into three components: (a) between subpopulations within the total population (we sometimes say &#39;between populations&#39;); (b) between individuals within subpopulations; and (c) between gametes within individuals.
FST, FIT, and FIS are defined as the expectations under the model of a/(a + b + c), (a + b)/(a + b + c), and b/(b + c) and estimated by the corresponding sample values [17, 18].
Here, it is perhaps pertinent to mention that when Weir and Cockerham [18] presented these definitions, they assumed a model consisting of an ancestral population from which subpopulations have descended in isolation under the same evolutionary processes.
Thus, it is meaningful to have a single measure of population structure; that is, a global FST, which is an average over subpopulations.
However, in identifying candidate loci under natural selection, evidence for locus-specific selection is of interest, and thus the estimators of locus-specific FST will be described in the next section.
Often, readers may be confused with several terms appearing in population genetics.
Wright [12] interprets FST as a measure of the progress of the subpopulation towards the fixation of one allele of each locus in the absence of mutation and hence called a &#39;fixation index.&#39;
FST is also interpreted as a measure of shared ancestry with the subpopulations, relative to that in the population, and is thus called the &#39;coancestry coefficient&#39; [19].
Therefore, if the value of FST is small, it means that the allele frequencies within each subpopulation are similar; if it is large, it means that the allele frequencies of subpopulations are different.
On the other hand, FIS or FIT is defined as the correlation between two gametes that form a zygote relative to the subpopulation or population, and thus, FIS (or FIT) is called the &#39;inbreeding coefficient&#39; [19].
Title: Estimating FST by ANOVA methods

The estimators of F-statistics proposed by Weir [17] and Weir and Cockerham [18] are based on an analysis of variance (ANOVA) of allele frequencies, equivalently called the method-of-moments estimates.
The weighted ANOVA estimates of FST, FIT, and FIS may be expressed in terms of the mean sum of squares for gametes (MSG), individuals (MSI), and populations (we sometimes say &#39;between subpopulations&#39;) (MSP), where the mean squares are estimated by an ANOVA model.
In estimating FST specifically for our analysis of CNV data, we need to consider unbalanced samples (i.e., populations of unequal size).
However, as the formulas are messy, we present here those for balanced samples.
Formulas for unbalanced samples can be found in Rousset (in Appendix A) [20].
Title: The definition of F-statistics used here is

Title: where Q values are probabilities of identity in state: Q1 among the genes (gametes) within individuals, Q2 among genes in different individuals within populations, and Q3 among the populations.
The estimates are expressed in terms of observed frequencies of identical pairs of genes in the sample, with the following relationships:

Title: and

Title: where n is the sample size of each population.
Then, the single locus estimator is given by

Title: (1)

Title: which is found in Weir (1997: 178) [17].
nc will be defined below.
If one needs to obtain the multilocus estimator of , it is usual to compute the estimator as a sum of locus-specific numerators over a sum of locus-specific denominators (see Weir [17] and Weir and Cockerham [18]).
This is the case that map information for SNPs is obtained for each gene, and a weighted-average FST from all SNPs is estimated for each gene [18].
For a set of I loci, the multilocus ANOVA estimators are

Title: (2)

for nc = (S1-S2/S1)/(n-1), where S1 is the total sample size and S2 is the sum of squared sample sizes of populations [21].
For convenience, we denote the estimator by FST.
Rousset [21] explained that the multilocus estimators of Weir [17] and Weir and Cockerham [18] differ slightly, and these two also differ slightly from that proposed by Rousset [21], which assigns more weight to larger samples.
In this paper, the GENEPOP software (version 3.4) (http://wbiomed.curtin.edu.au/genepop/) of Rousset was used for the calculation of FST.
In order to distinguish from those of the method-of-moments estimates of Weir [17] and Weir and Cockerham [18], we will call the estimates of GENEPOP ANOVA estimates.
The estimated values of FST can be negative when levels of differentiation are close to zero and/or sample sizes are small, indicating no population differentiation at these loci [18].
One can assign a value of zero to negative FST estimates.
Title: Identifying selection

Identifying candidate CNVs under natural selection using locus-specific estimates of FST is of great interest, in view of our increased knowledge of the relationships among human populations from genome-wide patterns of variation.
The logic for identifying selection is straightforward.
The pattern of genetic differentiation at a neutral locus is completely determined by the demographic history, migration rates among the populations, and the mutation rates at the loci.
It is reasonable to assume that all autosomal loci have experienced the same demographic history and migration rates among the populations, and the observed population structure can be largely explained by random drift at neutral loci.
However, as the individuals from different populations often vary genetically at a few key sites in their genome, loci showing unusually large amounts of differentiation may indicate regions of the genome that have been subject to positive selection, whereas loci showing unusually small amounts of differentiation may indicate regions of the genome that have been subject to stabilizing (balancing) selection [15, 22-24].
Thus, the outlier method makes it possible to detect divergences in some loci of the genome due to selection.
Title: Bayesian method

Balding and Nichols have proposed the use of population- and locus-specific estimates in the context of a migration-drift equilibrium model [24].
They modeled allele frequencies of biallelic markers using a beta distribution with expectation p and variance p(1-p)/(1+θ) so that FST = 1/(1+θ).
Then, they used a likelihood-based approach to estimate population- and locus-specific FST.
This formulation was extended later to consider multiallelic loci [19].
Falush et al.
[25] considered the nonequilibrium fission model that subpopulations evolve in isolation after splitting from an ancestral population.
We will not distinguish the migration-drift equilibrium model and nonequilibrium fission model, since both lead to the same multinomial-Dirichlet distribution (or beta-binomial for biallelic loci) for the subpopulation allele frequencies [26].
The main difference between the two models resides in the interpretation given to FST: in the case of the migration-drift equilibrium model, FST measures how divergent each subpopulation is from the total population, while in the case of the fission model, it measures the degree of genetic differentiation between each descendant population and the ancestral population.
Title: We consider a collection of J subpopulations and a set of I loci.
Let Ki&gt; be the number of alleles at the i th locus.
The extent of differentiation between subpopulation j and the ancestral population at locus i is measured by .
Let pi = { pik } denote the allele frequencies of the ancestral population at locus i, where pik is the frequency of the allele k at locus i( = 1).
We use p = { pi } to denote the entire set of allele frequencies of the ancestral population and to denote the current allele frequencies at locus i for subpopulation j.
Under the model and the definitions above, the allele frequencies at locus i in subpopulation j follow a Dirichlet distribution with parameters θjpi, a Bayesian prior distribution,

Title: (3)

Title: where θij = 1/-1.
The extent of differentiation at locus i between subpopulation j and the ancestral population is measured by and is the result of its demographic history.
The full prior distribution across loci and populations is given by

Title: (4)

Title: We need a hierarchical model for locus- and population-specific effects to identify candidate loci under natural selection.
It is no doubt that population- and locus-specific estimates of by the moments (or ANOVA) methods are likely to be inaccurate, especially for loci with a small number of different alleles.
As an alternative, Beaumont and Balding [22] proposed a familiar logistic regression model to decompose population- and locus-specific coefficients (bounded between 0 and 1) into a population-specific component, βj, shared by all loci and a locus-specific component, αi, shared by all populations [22]:

Title: (5)

The locus-specific effects express mutation and some forms of selection, and population-specific effects express migration rates, population sizes, and population-specific mating patterns [22].
The advantage of this formulation is that instead of estimating locus- and population-specific I·J coefficients (as in the method of moments or ANOVA), we only have to estimate I the parameters αi and the J parameters βj.
With the estimates of αi and βj, (and equivalently θij = exp(-(αi + βj)) can be estimated.
Above all, the estimate of αi is our objective to detect outlier loci as selection candidates.
Departure from neutrality at a given locus is assumed when αi is significantly different from 0 at that locus.
A positive value of αi suggests diversifying selection, whereas negative values suggest balancing selection.
The posterior probability that a locus is subject to selection, P(αi ≠ 0), is estimated directly from the Markov Chain Monte Carlo method (MCMC) (see below).
Title: Likelihood for allele counts

Title: The data of codominant markers consist of allele counts obtained from samples of size nij.
We use αijk to denote the number of alleles k observed at locus i in the sample from subpopulation j.
Thus, nij = Σkαijk.
The full dataset can be presented as a matrix N = { aij }, where aij = { αij1, αij2, ..., αijKi } is the allele count at locus i for subpopulation j.
The observed allele frequencies, aij, can be considered as sampled from the true alleles frequencies and therefore can be described by the multinomial distribution:

Title: (6)

Title: We adopt the multinomial-Dirichlet distribution as a marginal distribution of aij, the allele frequency counts at a locus within a subpopulation, and the reasons of its adoption are explained in Foll and Gaggiotti [26]:

Title: The joint likelihood for allele counts is obtained by multiplying across all loci and populations:

Title: (7)

Since the allele frequencies in the ancestral population pi = { pik } are unknown, they are estimated by introducing a noninformative Dirichlet prior, pi ~ Dir (1, … , 1), into the Bayesian model [27].
Title: The full Bayesian model is given by

Title: (8)

Following Beaumont and Balding [22], the Gaussian prior distributions are used for the population effects βj and for the locus effects αi.
Their means and variances are chosen to improve convergence and for each to have non-negligible density over almost the whole interval from 0 to 1.
Title: Identifying CNVs subject to selection

As described above, the effect of selection is parameterized by αi in the logistic regression.
Thus, two models in the logistic, one that includes both effects of αi (i.e., αi ≠ 0) and βj (selection model, M2) and another one that does not include the effect of selection (i.e., αi = 0) and only includes the effect of βj (neutral model, M1), are considered.
We can decide in a Bayesian framework whether or not there is selection at all.
This is done by estimating the posterior probabilities for two models for each locus in the logistic regression: neutral (M1) and selection (model 2, M2), on the basis of a dataset N = { aij }.
We use a reversible jump MCMC algorithm [28] to estimate the posterior probability of each one of these models, and this is done separately for each locus i.
We can then have posterior probability that a locus is subject to selection - that is, P(αi ≠0).
This probability is estimated directly from the output of the MCMC by simply counting the number of times αi is included in the model (see Foll and Gaggiotti [27] for a more detailed explanation).
In BayeScan, selection decision (model choice decision), with a specific goal of detecting outlier loci in mind, is actually performed by using posterior odds (PO).
For this, we first need to describe &#39;Bayes factors.&#39;
For the two models of neutral (M1) and selection (M2), the Bayes factor (BF) for model M2 is given by BF = P(N | M2)/P(N | M1), where N = { aij } is a dataset and P indicates probability.
This BF provides a degree of evidence in favor of one model versus another.
In the context of multiple testing - that is, testing a large number of loci simultaneously - we also need to incorporate our skepticism about the chance that each locus is under selection.
This is done by setting the prior odds for the neutral model P(M1)/P(M2).
We make selection decisions by using PO, which is defined as PO = P(M2|N)/P(M1|N) = BF × P(M2)/P(M1).
PO are simply the ratio of posterior probabilities and indicate how much more likely the model with selection (M2) is compared to the neutral model (M1) (GENEPOP software).
PO of 0.5-1.0, 1.0-1.5, 1.5-2.0, and 2-∞ are, respectively, considered as substantial, strong, very strong, and decisive evidence for selection [14].
A big advantage of posterior probabilities is that they directly allow the control of the false discovery rate (FDR), where FDR is defined as the expected proportion of false positives among outlier loci.
Controlling for FDR has a much greater power than controlling for family-wise error rates using Bonferroni correction, for example [29].
In BayeScan, by first setting a target FDR, the PO threshold achieving this FDR is determined, and outliers equal to or above this PO threshold are listed in the output of R that is provided along with BayeScan.
Title: Study populations

We obtained the high-resolution CNV data by Conrad et al.
[4] from the Database of Genomic Variants (http://projects.tcag.ca/variation/).
The dataset consists of 90 Han Chinese (CHB) and Japanese (JPT), 180 Caucasians of European descent (CEU), and 180 Yoruba (YRI) samples.
Among the 450 individuals, we removed 19 individuals with more than 400 missing CNV genotypes.
Each CNV was analyzed as biallelic dominant data, classified as neutral versus non-neutral (losses or gains).
Title: Empirical distribution of FST

To assess population differentiation with an allele frequency spectrum of CNVs, the locus-specific estimates by ANOVA method and by hierarchical Bayesian method were obtained from all autosomes.
The estimated values by the ANOVA method (moments estimates) can be negative when levels of differentiation are close to 0 and/or sample sizes are small, indicating no population differentiation at these loci [18].
In the hierarchical Bayesian method, the locus-specific estimate values are all positive, since they are generated by posterior distributions, which is an important benefit of a Bayesian method.
With the CNV data of this study, the summary statistics of FST was 0.0853 ± 0.1195 (median, 0.0360; range, -0.0071 to 0.8994) for the ANOVA method and 0.2459 ± 0.0115 (median, 0.2462; range, 0.2193 to 0.6166) for the Bayesian method.
The empirical distributions of FST are found in Fig.1.
As can be seen from the Figure, low FST values are prevalent, and a large variability persisted in the ANOVA method, while FST distribution was sharply focused in the Bayesian method.
The comparison of empirical distributions of the ANOVA and Bayesian methods demonstrates that Bayesian estimates perform better than ANOVA in identifying CNVs affected by natural selection.
Title: Bayesian outlier detections

We searched the genome regions that show signals of natural selection by the Bayesian likelihood method implemented via reversible jump MCMC in BayeScan software [27].
For each CNV, BayeScan directly estimates the probability that a CNV is under positive selection, from which the PO are computed [27].
The posterior odds indicate how much more likely the model with selection is compared to the neutral model, and posterior odds of 5.0 was chosen as a threshold in the detection of CNVs under positive selection.
This directly allows us to control the FDR, the expected proportion of false positives among outlier CNVs [29].
In our analysis, we chose FDR to be 0.05.
In the BayeScan application, first following 10 pilot runs of 5,000 iterations and an additional burn-in of 50,000 iterations, we used 50,000 iterations (sample size of 5,000 and a thinning interval of 10).
Title: Outliers detected by BayeScan

BayeScan identified the three outlier CNVs with Conrad et al.
&#39;s [4] CNV data (Table 1, Fig.2).
The most decisive outlier is CNVR2664.1 (chromosome 5), in which most CEU (90%) and all CHB + JPT populations are outlying gain or loss status.
The next decisive outlier is CNVR371.1 (chr.
1), in which only the YRI population has non-neutral CNVs.
In general, the observed population structure observed by hierarchical clustering, multidimensional scaling (MDS) plots, and tree plots can be largely explained by random drift at neutral CNVs.
However, as individuals from different populations often vary genetically at a few key sites in their genome, only the outlier method makes it possible to detect accelerated divergence in some CNV regions of the genome due to local selection.
This local selection is detected relative to the majority of CNVs, in which among-population differences are purely a result of genetic drift.
Successful outlier detection depends on reliable and obtainable estimates of FST and also on sampling variances of FST.
Very large variances that are associated with single locus moment estimates of FST preclude the use of these estimates to detect selection in spite of the fact that sampling variances will decrease with the number of alleles at a locus and with the numbers of populations sampled.
In this respect, the availability of locus- and population-specific Bayesian estimates of FST provides a set of tools for identifying genomic regions or populations with unusual evolutionary histories.
The most important benefits of Bayesian estimates and its selection method are that the Bayesian methods allow probability statements to be made about FST and can be extended to explore the relationship with demographic or environmental covariates in the model [26].
Furthermore, likelihood-based Bayesian methods have the flexibility to accommodate missing data.
However, implementations of Bayesian methods may be computationally demanding.
Selection provides information about the adaptation to a wide range of habitats and climates [30], and thus, interpreting the story of human adaptation is an interesting research area for the studies of evolution and disease processes in the future.
Thus, more studies of outlier detection need to be replicated with large population-based data.
Empirical distributions of copy number variants (CNV) data with the three population groups.
CNV data are from Conrad et al.[4].
Spacial scans for identification of FST outlier copy number variants (CNVs) potentially subject to positive selection by BayeScan.
CNV data are from Conrad et al.[4].
PO, posterior odds.
Title: Identified outlier CNVs potentially subject to positive selection by BayeScan
Title: Interaction Effects of Lipoprotein Lipase Polymorphisms with Lifestyle on Lipid Levels in a Korean Population: A Cross-sectional Study

Lipoprotein lipase (LPL) plays an essential role in the regulation of high-density lipoprotein cholesterol (HDLC) and triglyceride levels, which have been closely associated with cardiovascular diseases.
Genetic studies in European have shown that LPL single-nucleotide polymorphisms (SNPs) are strongly associated with lipid levels.
However, studies about the influence of interactions between LPL SNPs and lifestyle factors have not been sufficiently performed.
Here, we examine if LPL polymorphisms, as well as their interaction with lifestyle factors, influence lipid concentrations in a Korean population.
A two-stage association study was performed using genotype data for SNPs on the LPL gene, including the 3&#39; flanking region from 7,536 (stage 1) and 3,703 (stage 2) individuals.
The association study showed that 15 SNPs and 4 haplotypes were strongly associated with HDLC (lowest p = 2.86 × 10-22) and triglyceride levels (lowest p = 3.0 × 10-15).
Interactions between LPL polymorphisms and lifestyle factors (lowest p = 9.6 × 10-4) were also observed on lipid concentrations.
These findings suggest that there are interaction effects of LPL polymorphisms with lifestyle variables, including energy intake, fat intake, smoking, and alcohol consumption, as well as effects of LPL polymorphisms themselves, on lipid concentrations in a Korean population.
Lipoprotein lipase (LPL) plays an important role in the metabolism and transport of lipids.
It hydrolyzes triglycerides (TGs) in lipoproteins, such as chylomicrons (CMs) and very low-density lipoproteins (VLDLs), into free fatty acids, releasing CM remnants and intermediate-density lipoproteins that act on high-density lipoprotein cholesterol (HDLC) metabolism.
Plasma LPL therefore plays an essential role in elevating HDLC and lowering TG [1-3].
While HDLC plays a role in the removal and transport of cholesterol from peripheral tissues to other tissues where it is required, such as the liver, low-density lipoprotein cholesterol (LDLC) is, conversely, transported from the liver to peripheral tissues.
The concentration of lipids has been tightly associated with the risk of cardiovascular diseases [4-6].
Specifically, LDLC levels have been shown to have a positive log-linear association with the risk of coronary heart disease: a reduction of 1% in the serum concentration of LDLC has been shown to reduce the relative risk of coronary heart disease risk by approximately 1% [7], whereas an increase of 1 mg/dL in serum HDLC levels is associated with a decrease in the risk of coronary heart disease of approximately 2% to 3%, independent of LDLC levels [8].
It was also reported that TG levels are associated with risk of cardiovascular disease, independent of concentrations of other lipids and cardiac risk factors, although there has been controversy [6].
Recently, a report based on the Korea National Health and Nutrition Examination Surveys I, II, III, and IV has indicated that the prevalence of dyslipidemia and hypertriglyceridemia in this country has increased during the last decade [9].
Genetic association studies aimed at identifying the genetic factors associated with variations in lipid concentrations have been performed mostly for Europeans, and they have shown that the polymorphisms in the LPL gene, including the 3&#39; flanking region, are strongly associated with HDLC and TG levels [10-13].
There is a report of a candidate gene approach for lipid concentrations in a Korean population, but the LPL gene was not included in the study [14].
The concentration of lipids is also influenced by environmental factors, such as smoking, alcohol consumption, diet, and physical activity, as well as genetic factors [5, 15].
However, studies about the influence of interactions between LPL polymorphisms and lifestyle factors have not been sufficiently performed.
In this study, we examined the effects of potential interactions between LPL polymorphisms and lifestyle, as well as genetic variations of the LPL gene, including the 3&#39; flanking region, on the lipid profile of a Korean population.
Title: Subjects

Data for this study were obtained from the Korean Genome Epidemiology Study (KoGES), conducted by the Korean National Institute of Health (KNIH).
The study comprised two stages.
In the first stage, a cross-sectional analysis was conducted of samples from urban and rural communities from South Korea, whereas in the second stage, the study was replicated in other urban regions in Korea.
Subjects who were examined in the first stage, described in a previous report [16], were recruited from a rural (Ansung) and an urban (Ansan) community from South Korea, which have been studied since 2001 as a part of KoGES.
In total, 5,018 and 5,020 subjects from the Ansung and Ansan communities, respectively, participated in the study.
The age of the participants ranged from 40 to 69 years.
Samples were scrutinized for quality control purposes, after which 8,842 subjects remained in our sample [16].
We further excluded 1,306 subjects who were on medication for hypertension, diabetes, myocardial infarction, and hyperlipidemia.
Information about the lifestyle of the subjects was based on their responses to questionnaires.
For the second stage of the study, 4,302 samples were obtained from both urban regions, Seoul and Busan, in South Korea that were established from 2006 to 2007 as part of KoGES.
Samples satisfying any of the following criteria were excluded: i) gender inconsistencies, ii) genotype call rate lower than 95%, iii) excessive heterozygosity, iv) cryptic first degree relative, v) outliers on a multidimensional scaling plot, or vi) cancer history.
A total of 3,703 subjects remained in our sample.
These exclusion criteria for samples have been described [17].
Details of the sample characteristics are listed in Table 1.
Title: Genotyping

We obtained genotype data for single-nucleotide polymorphisms (SNPs) located in the LPL gene, including the 3&#39; flanking region, from KoGES, which are accessible to the research community through the Korean Associated Resource project.
The study protocol was approved by the Institutional Review Board of KNIH.
For genotyping of the samples from the Ansung and Ansan cohorts, Affymetrix Genome-Wide Human SNP Array 5.0 (Affymetrix Inc., Santa Clara, CA, USA) was used.
Affymetrix Genome-Wide Human SNP Array 6.0 (Affymetrix Inc.) was used for genotyping samples from the Seoul and Busan cohorts.
The Bayesian Robust Linear Model with the Mahalanobis distance algorithm was used to determine the genotypes at each SNP of Affymetrix 5.0.
The BirdSeed calling genotyping algorithm was used on Affymetrix 6.0.
The SNPs were filtered if any of the following criteria was met: i) a call rate lower than 95%, ii) a minor allele frequency (MAF) lower than 0.05, or iii) significant deviation from the Hardy-Weinberg equilibrium (HWE).
Significant deviation from HWE for 28 successfully genotyped SNPs was determined based on a threshold of p &lt; 0.002 (0.05/28).
SNP imputation was performed using the IMPUTE program (https://mathgen.stats.ox.ac.uk/impute/impute.html).
A reference panel from International HapMap (phase II/release 22/NCBI build 36 and dbSNP build 126) data, comprising SNPs from 90 Japanese (JPT) and Han Chinese (CHB) individuals, was used as a template for inferring untyped genotypes.
Imputed SNPs that were located in the LPL gene, including the 3&#39; flanking region, with low quality were excluded according to the following criteria: low genotype information content (information lower than 0.5), a posterior probability score lower than 0.90, a call rate lower than 0.95, and MAF lower than 0.05.
Title: Biochemical measures

Biochemical data from KoGES were obtained through the Korean Association Resource (KARE) project.
Blood samples were obtained from the participants with at least 8 hours of overnight fasting.
Serum total cholesterol (TCHL), HDLC, and TG concentrations were measured using the Advia 1650 analyzer (Siemens, Tarrytown, NY, USA) for the Ansung and Ansan cohorts and the Hitachi 7600 analyzer (Hitachi, Tokyo, Japan) for the Seoul and Busan study cohorts [16, 17].
LDLC concentration was estimated using the equation by Friedewald et al.
[18] for individuals with TG levels lower than 400 mg/dL.
Title: Statistics

To test the association between LPL SNPs and haplotypes (HTs) and lipid concentration, linear regression analyses, adjusted for age, gender, and body mass index (BMI), were performed.
For stage 1 and imputed data, the residence area of each participant was additionally adjusted.
The influence of the interaction between LPL SNPs and lifestyle variables on lipid levels was analyzed by using linear regression, including main and interaction effects, adjusted for residence area, age, gender, and BMI.
The effect of energy and fat intake, physical activity, smoking, and drinking status on lipid profiles was examined.
Energy intake, fat intake, and physical activity were measured as the total intake of kilocalories per day, daily fat intake, and metabolic equivalent scores per hour, respectively.
Smoking status had two categories, non-smokers and smokers, and alcohol consumption was divided into non-drinkers and drinkers.
These lifestyle factors were available for subjects from stage 1 and imputed data but not from stage 2.
All analyses were performed by log-transforming HDLC, TG, and TCHL values to normalize their distribution.
Untransformed LDLC values were used for all analyses.
To determine the effect of SNPs, untransformed lipid concentrations were used.
Linkage disequilibrium (LD) block partition was performed by Gabriel&#39;s rule, applied in Haploview version 4.1 (Broad Institute of MIT and Harvard University, Boston, MA, USA), and HT inference was performed using the expectation maximization algorithm with PLINK version 1.07 (http://pngu.mgh.harvard.edu/~purcell/plink).
All statistical analyses were performed in an additive model (AA vs. AB vs. BB or +/+ vs. +/- vs. -/-) using PLINK.
Title: Association of LPL SNPs with lipid concentrations

We sequentially discarded 6 SNPs with MAF &lt; 0.05, 2 SNPs with call rate &lt; 95%, and 1 SNP with HWE p-value &lt; 0.002, leaving 19 SNPs for the association study at stage 1.
Our results show that SNPs located in the LPL gene, including the 3&#39; flanking region, were strongly associated with HDLC and TG levels.
The linear regression, adjusted for age, gender, and BMI, using combined data from the two study stages, revealed that 15 SNPs were associated with HDLC and TG levels (Table 2).
The strongest associations were between rs10503669 and both HDLC (combined p = 3.6 × 10-22; 2.2-mg/dL increase per minor T allele) and TG (combined p = 3 × 10-15; 13.6-mg/dL decrease per minor T allele) in additive models.
However, no strong association was detected between the SNP and either LDLC or TCHL levels (Supplementary Fig.1).
For LPL SNPs, we tested 2 intronic SNPs and 2 SNPs located in the 3&#39; untranslated region (UTR) from stage 1.
Of these, two SNPs in the 3&#39; UTR (rs11570892 and rs3200218) were not significantly associated with any lipid concentration (Supplementary Table 1).
Two intronic SNPs, rs263 and rs271, were significantly associated to higher HDLC levels (p = 1.2 × 10-10 and p = 2.5 × 10-11; 1.2- and 1.3-mg/dL increase per minor A and T allele, respectively) and lower TG levels (p = 1.7 × 10-6 and p = 7.2 × 10-7; 8.8- and 9-mg/dL decrease per minor A and T allele, respectively) (Table 2).
In stage 2, 7 intronic SNPs and 2 SNPs that were located in the 3&#39; UTR were tested.
It was found that rs263 and rs271 were also strongly associated with increased HDLC (p = 5.5 × 10-6 and p = 5.1 × 10-6, respectively; both showed a 1.5-mg/dL increase per minor A and T allele) (Table 2) and decreased TG levels (p = 3.3 × 10-3 and p = 2.2 × 10-3; 8.6- and 8.7-mg/dL decrease per minor A and T allele, respectively).
Additionally, 4 intronic SNPs (rs253, rs326, rs327, and rs12679834) were significantly associated with increased HDLC (p = 4.2 × 10-3, p = 1.5 × 10-9, p = 1.6 × 10-9 and p = 3.1 × 10-14; increase of 0.8, 2.1, 2.1, and 3.3 mg/dL per minor allele, respectively) and decreased TG levels (p = 3.6 × 10-3, p = 9.1 × 10-7, p = 5.9 × 10-7 and p = 1.3 × 10-6; increase of 7.4, 12.2, 12.3, and 13.7 mg/dL per minor allele, respectively) (Supplementary Table 2).
The analysis using combined data from the two stages showed that rs263 and rs271 were also significantly associated with HDLC (p = 1.7 × 10-11 and p = 5.7 × 10-12, respectively; increase in both cases of 1.2 mg/dL per minor A and T allele) and TG levels (p = 6.8 × 10-7 and p = 2.7 × 10-7; decrease of 8.1, 8.3 mg/dL per minor A and T allele, respectively) (Table 2).
Title: Association of LPL HTs with lipid concentration

Two blocks, including SNPs located in the LPL gene, were identified (Supplementary Fig.2), and the association of HT with lipid profiles for the two blocks was examined.
The analysis using combined data from the two stages showed that block 2, which includes SNPs located in the LPL gene, including the 3&#39; flanking region, was the region most strongly associated with lipid concentrations (Table 3).
HT3 in block 2 was strongly associated with increased HDLC levels (p = 2.86 × 10-22; 2.25-mg/dL increase per HT3) and reduced TG levels (p = 9.09 × 10-15; 13.7-mg/dL decrease per HT3), whereas HT1 was strongly associated with reduced HDLC levels (p = 1.1 × 10-14; 1.2-mg/dL decrease per HT1) and higher TG levels (p = 3.05 × 10-12; 8.6-mg/dL increase per HT1).
There was no significant association between HT and either LDLC or TCHL.
The analysis using imputed data also showed a strong association between HTs and lipid levels in the 2 blocks, including LPL SNPs (Supplementary Table 3).
HT1 in block 2 was significantly associated with a reduced HDLC level (p = 4.2 × 10-15; 1.3-mg/dL decrease per HT1) and an increased level of TG (p = 1.9 × 10-8; 8-mg/dL increase per HT1), whereas HT3 was significantly associated with increased HDLC (p = 7.8 × 10-16; 2-mg/dL increase per HT3) and LDLC levels (p = 5.1 × 10-3; 2.3-mg/dL increases per HT3) and decreased TG levels (p = 5.6 × 10-13; 15.1-mg/dL decrease per HT3).
Title: Effect of the interaction between polymorphisms located in the LPL gene and lifestyle on lipid concentration

We tested LPL SNPs and HTs from stage 1.
All significant associations were found in the results of the dominant model for the minor allele.
The association of minor alleles for 3 SNPs (rs263, rs271, rs328) with higher HDLC levels also depended on energy intake (Supplementary Fig.3).
Of these, the minor alleles of 2 SNPs (rs263, rs271) were associated with lower TG levels, also depending on energy intake.
The association of the minor alleles of two SNPs (rs263, rs271) with higher HDLC levels also depended on fat intake (Fig.1).
The minor alleles of 2 SNPs (rs263, rs271) were also associated with higher HDLC levels and lower TG levels through an interaction with cigarette smoking and with lower TG levels through an interaction with alcohol consumption (Fig.2, Supplementary Fig.4).
Interaction effects of HT1 and 2 in block 1, which were formed by two SNP (rs271, rs263) in strong LD (r2 = 0.99), with lifestyle factors on HDLC and TG levels were found to be almost same with those of rs271 and rs263 (data not shown).
In block 2, HT3 was associated with higher HDLC levels and lower TG levels and depended on an interaction with energy intake (Fig.3, Supplementary Fig.3).
HT4 in block 2 was associated with lower TG levels through an interaction with cigarette smoking (Supplementary Fig.4).
For the interaction between LPL polymorphisms and physical activity, there was no significant association with any lipid levels.
The human LPL gene is located on chromosome 8p22 and includes 10 exons encoding 475 amino acids, of which 448 amino acids are part of the mature protein and 27 of the signal peptide [19, 20].
LPL has an essential role in the metabolism of lipids, as it hydrolyzes a number of substrates, such as long- and short-chain glycerides and phospholipids in vitro, as well as major substrates, such as CMs and VLDL in vivo [21].
Dysfunction of LPL has been reported to be associated with chylomicronemia, atherosclerosis, obesity, Alzheimer&#39;s disease, and dyslipidemia [22].
Genetic association studies have shown that some SNPs located in the LPL gene, including the 3&#39; flanking region, are strongly associated with HDLC and TG levels, mostly in Europeans.
Specifically, by performing meta-analyses of genomewide association studies, Aulchenko et al.
[13] found that rs2083637 and rs10096633 were associated with HDLC and TG levels, and it was also demonstrated that rs325 and rs10105606, reported by Waterworth et al.
[11], and rs12678919, reported by Kathiresan et al.
[23], were associated with HDLC and TG levels.
Studies using the candidate gene approach, which examines the effects of genetic variants of a putative gene in an association, have reported that some LPL SNPs are associated with HDLC and TG levels.
Tang et al.
[24] recently revealed that rs326 and rs328 (which convert serine to a stop codon) and rs13702 are associated with HDLC and TG levels in African-Americans and European-American populations.
Here, rs263 and rs271, located, respectively, in introns 5 and 6, were strongly associated with increased HDLC and decreased TG levels (Table 2).
The two significant SNPs formed an HT block with strong LD (r2 = 0.99) in stage 1, and rs3200218, located in the 3&#39; UTR, formed another HT block with 15 SNPs located in the 3&#39; flanking region.
At stage 2, nine SNPs in the LPL gene remained after SNP filtering, out of which six were significantly associated with HDLC levels (Supplementary Table 2).
SNPs rs343, rs253, rs263, and rs271, located, respectively, in introns 3, 4, 5, and 6, formed an HT block with moderate LD (r2 &gt; 0.3).
SNPs rs326, rs327, rs12679834, rs11570892n, and rs3200218 formed another HT block with 31 SNPs located in the 3&#39; flanking region.
In the two blocks at stages 1 and 2, some HTs were found to be strongly associated with HDLC and TG levels (Supplementary Tables 4 and 5).
The analysis using the combined data from both stages showed that HTs were also significantly associated with HDLC and TG levels (Table 3).
Using the imputation data, we found that more LPL SNPs were included in the two blocks.
HTs that were significantly associated with lipid profiles were also identified (Supplementary Table 3).
The strongest association of HT with HDLC and TG levels was observed in block 2 (from both stages).
A previous study reported that one HT was associated with increased LPL activity [25].
The HT was found to be consistent with HT3 in block 2, found using stage 1 and imputed data, which was associated here with increased HDLC and decreased TG levels.
In block 2, we found that one SNP, rs328, which converts serine to a stop codon, was included using imputed data.
Interestingly, the minor allele G of rs328 was found only in HT3.
It has been reported that this minor allele is associated with increased HDLC and decreased TG concentrations in Japanese [26] and European populations [10, 27], and our results support this finding.
This truncated variant was also demonstrated to have higher catalytic activity [28-30].
SNP rs328 has been reported to be in strong LD with rs17482753 in European (r2 = 1) and African (r2 = 0.852) populations [31].
rs17482753 was genotyped in stage 1 and 2 of this study, and the SNP was found to be in strong LD with rs328, according to the imputed data (r2 = 0.993).
We additionally identified strong LD (r2 = 0.93) between rs328 and rs17482753 by using genotype data of 830 Korean participants obtained from a previous study [32], in which samples in stage 1 of this study were included.
Similar to the results of rs328 obtained using the imputed data in this study, the rs17482753 minor allele was found to be strongly associated with raised HDLC and reduced TG levels in a dominant manner at both stages.
In a previous study, LPL SNPs located in the 3&#39; UTR region were considered possible targets affecting miRNA binding sites, and 5 SNPs were predicted to be putative miRNA binding sites by polymiRTS [33].
The 3&#39; UTR region of the LPL gene was included in block 2 in this study, and two of the predicted 5 SNPs were tested (Supplementary Table 1).
Rs1059611 major allele T from the imputed data was significantly associated with reduced levels of HDLC and LDLC and with increased TG levels in an additive manner.
In this SNP, the major T allele was predicted to create a new miRNA (miR-136) binding site, causing abnormal repression of this gene.
The major allele of another SNP, rs11570892, was predicted to disrupt a conserved miRNA (miR-592) binding site, and this could cause abnormal control of normal repression.
However, this SNP showed inconsistent association with LDLC levels in stages 1 and 2 of this study.
Some environmental factors have been suggested to be associated with lipid profiles.
Physical activity was reported to be associated with higher HDLC and lower TG and TCHL concentrations [34], whereas smoking was reported to be associated with higher LDLC and TG and lower HDLC levels [35, 36].
Alcohol consumption was suggested to be associated with increased HDLC levels [37, 38], whereas excessive alcohol consumption was associated with a higher TG level [39].
A low fat intake, in turn, was associated with reduced TCHL, LDLC, and HDLC and increased TG levels [40].
Some LPL SNPs have also been shown to be associated with lipid profiles through their interaction with smoking, alcohol consumption, and physical activity.
Specifically, rs320 has been suggested to interact with smoking [41, 42] and physical activity [42], and rs1801177 interacts with physical activity [43].
The SNP rs328, which is a nonsense SNP included in our imputed data, has been associated with HDL through its interaction with smoking and alcohol consumption in females but not in males [44].
In a recent study, however, the interaction of several LPL SNPs, including rs328, with alcohol consumption was suggested to have no influence on HDLC levels [45].
In this study, we show that the effects of some LPL polymorphisms on lipid concentrations were regulated by several environmental factors.
Carriers of each minor allele of 3 SNPs (rs263, rs271, and rs328) were associated with slightly higher HDLC levels, depending on an interaction with energy intake.
This effect, however, was not observed among non-carriers (Supplementary Fig.3).
In two of these SNPs (rs263 and rs271), which were in strong LD (r2 = 0.99 in stages 1 and 2), each major genotype carrier led to higher levels of TG, whereas the TG levels of each minor allele carrier were not affected by a higher energy intake.
Carriers of each minor allele, but not non-carriers, led to a higher HDLC level through an interaction with a higher intake of fat (Fig.1).
Each major genotype carrier of two SNPs (rs263 and rs271) showed lower HDLC levels and higher TG levels than minor allele carriers through an interaction with smoking (Fig.2).
In the major genotype carrier of these two SNPs, higher TG levels were also found by interacting with alcohol consumption (Supplementary Fig.4), whereas each minor allele carrier was less affected than non-carriers by smoking or alcohol consumption.
For interactions of HTs, blocks 1 and 2 were shown to interact with inconsistent lifestyle variables.
Interaction effects of HT1 and 2 in block 1 with lifestyle factors, which were formed by the two SNPs, on lipid levels were almost same with those of rs263 and rs271 (data not shown).
In the LPL gene, 61 missense mutations have been identified, and interestingly, 42 of them are located in exons 5 and 6, which may be contained in block 1 [46].
Carrier of HT3 in block 2, which was reported to be associated with increased LPL activity [25], showed higher HDLC levels and lower TG levels, whereas non-carriers were not affected by a higher energy intake (Fig.3, Supplementary Fig.3).
In the non-carrier of HT4 in block 2, higher TG levels were found by interacting with cigarette smoking (Supplementary Fig.4).
Smoking and moderate alcohol consumption were reported to be related with reduced and increased LPL activity, respectively [47, 48], but the precise mechanisms by which the interaction of LPL polymorphisms with environmental factors affects lipid concentrations have not been demonstrated yet.
In conclusion, we found that HTs are strongly associated with HDLC and TG levels in the 2 blocks in which LPL SNPs are included and that SNPs in these blocks are also strongly associated with HDLC and TG levels in stage 1.
Interestingly, these results were replicated in stage 2, which consists of independent samples, and the combined analyses, including the samples of stages 1 and 2, showed the most significant results.
Moreover, by analyzing the effect of the interaction of LPL polymorphisms with several lifestyle factors on plasma lipid levels, we have shown that the effects of some LPL SNPs and HTs on HDLC and TG concentrations are regulated by energy and fat intake, smoking, and alcohol consumption.
Since these results have not been reported in any previous studies, they are novel findings.
These interaction effects suggest that the appropriate lifestyle of each individual could be recommended according to the type of LPL polymorphism.
These results should be confirmed in other populations and in a longitudinal study.
Effect of the interaction of rs271, which was in strong linkage disequilibrium with rs263 (r2 = 0.99), with daily fat intake on high-density lipoprotein cholesterol (HDLC) levels (p = 9.6 × 10-4).
Subjects were divided into daily fat intake ≤ 20 g (TT or TC, 774; CC, 1,314), 21-30 g (TT or TC, 695; CC, 1,183), 31-40 g (TT or TC, 547; CC, 934), and ≥ 41 g (TT or TC, 702; CC, 1,144).
Interaction p-value was calculated by linear regression analysis, which includes main and interaction effects, adjusted for residence area, age, gender, and body mass index.
Effect of the interaction of rs271, which was in strong linkage disequilibrium with rs263 (r2 = 0.99), with cigarette smoking on high-density lipoprotein cholesterol (HDLC) (A) and triglyceride (TG) (B) levels (p = 5.7 × 10-3 and p = 8.60 × 10-3, respectively).
Subjects were divided into non-smokers (TT or TC, 1,629; CC, 2,689) and smokers (TT or TC, 700; CC, 1,289).
Interaction p-values were calculated by linear regression analyses, which includes main and interaction effects, adjusted for residence area, age, gender, and body mass index.
Effect of the interaction of haplotype 3 in block 2 (TGTGTAGAGCTCCTCG) with daily energy intake on high-density lipoprotein cholesterol (HDLC) level (p = 5.6 × 10-3).
Subjects were divided into daily energy intake ≤ 1,500 kcal (HT3/HT3 or HT3/-, 387; -/-, 1,353), 1,501-2,000 kcal (HT3/HT3 or HT3/-, 646; -/-, 2,297), 2001-2,500 kcal (HT3/HT3 or HT3/-, 358; -/-, 1,216), and ≥ 2,501 kcal (HT3/HT3 or HT3/-, 235; -/-, 801).
Interaction p-values were calculated by linear regression analyses, which includes main and interaction effects, adjusted for residence area, age, gender, and body mass index.
HT, haplotype.
Title: Description of subjects used in stages 1 and 2 of the study

Title: SNPs associated with HDLC and TG concentrations at each stage of the study

Title: Association between haplotypes, including LPL SNPs and lipid concentrations, using combined data from stages 1 and 2
Title: Effect of Genetic Predisposition on Blood Lipid Traits Using Cumulative Risk Assessment in the Korean Population

Dyslipidemia, mainly characterized by high triglyceride (TG) and low high-density lipoprotein cholesterol (HDL-C) levels, is an important etiological factor in the development of cardiovascular disease (CVD).
Considering the relationship between childhood obesity and CVD risk, it would be worthwhile to evaluate whether previously identified lipid-related variants in adult subjects are associated with lipid variations in a childhood obesity study (n = 482).
In an association analysis for 16 genome-wide association study (GWAS)-based candidate loci, we confirmed significant associations of a genetic predisposition to lipoprotein concentrations in a childhood obesity study.
Having two loci (rs10503669 at LPL and rs16940212 at LIPC) that showed the strongest association with blood levels of TG and HDL-C, we calculated a genetic risk score (GRS), representing the sum of the risk alleles.
It has been observed that increasing GRS is significantly associated with decreased HDL-C (effect size, -1.13 ± 0.07) compared to single nucleotide polymorphism combinations without two risk variants.
In addition, a positive correlation was observed between allelic dosage score and risk allele (rs10503669 at LPL) on high TG levels (effect size, 10.89 ± 0.84).
These two loci yielded consistent associations in our previous meta-analysis.
Taken together, our findings demonstrate that the genetic architecture of circulating lipid levels (TG and HDL-C) overlap to a large extent in childhood as well as in adulthood.
Post-GWAS functional characterization of these variants is further required to elucidate their pathophysiological roles and biological mechanisms.
Dyslipidemia is an important etiological factor in the development of cardiovascular disease (CVD), which is the leading cause of deaths for adults.
Dyslipidemia is mainly characterized by elevated levels of triglycerides (TGs), increased low-density lipoprotein cholesterol (LDL-C), and reduced high-density lipoprotein cholesterol (HDL-C) [1].
In the past few years, genome-wide association studies (GWASs) and meta-analyses have identified common genetic variants that contribute to blood lipid phenotypes in Caucasian-based populations [2-8].
In addition, we have convincingly mapped 16 loci for major lipoprotein concentrations (TG, HDL-C, and LDL-C) in a two-stage association study in individuals of East Asian ancestry [9].
Given the growing global epidemic of metabolic syndrome, including obesity and type 2 diabetes (T2D), lipoprotein metabolism disorders have become an important health problem in adulthood but also have emerged as an increasingly prevalent risk factor in childhood [10].
Although many GWASs and meta-analyses have provided susceptibility genes or loci for lipid-related traits from multiple large population-based cohorts of adults, it is of great importance to evaluate genetic predisposition at an early age.
However, little is known about their genetic effects in a younger population.
Moreover, most identified variants have shown modest effects and have explained only a small proportion of heritable components for disease risk [11].
Nevertheless, when multiple genetic markers are considered together, literature-based genetic risk scores might be useful in improving the identification of the genetic contribution in those at high risk of disease [12-14].
Therefore, the aim of this study was to consolidate the associations of GWAS-based candidate variants with blood lipid concentrations in a childhood obesity study and to evaluate genetic risk scores (GRS) for lipid-related risk phenotypes.
Title: Study subjects

Korean Association Resource (KARE) study was collected through the Korean Genome Epidemiologic Study project - in total, 10,038 people, 40-69 years old, living in Ansung and Ansan, from 2001-2002.
For more information, the study has already been reported [9, 15].
The Health Examinee cohort (HEXA) shared project is a community-based epidemiological cohort of 1,200,000 subjects (aged, 40-69) who were recruited for baseline examination from 2004 to 2009.
To share the controls, 4,302 subjects were randomly selected to participate between 2006 and 2007.
The BioBank Japan study was approved by the ethical committees in the Institute of Medical Science, the University of Tokyo, and the Center for Genomic Medicine, RIKEN.
Subjects for the Health2 cohorts for the replication study were selected from another community-based cohort sample provided by the Health2 study, as described previously [15].
We examined 7,861 selected from the 8,500 participants (aged, 40-69).
The students aged 9 and 12 years were recruited from Gwacheon City and Gyeonggi Province between April and June 2010.
This study was conducted as part of the Korean Children-Adolescents Study (KoCAS), which has been monitored yearly since their entry into elementary school at age 7 in Gwacheon City or fourth grade at age 10 in Seoul and Gyeonggi Province, Korea.
Subjects who were enrolled in a specific diet program or were taking any medications known to affect appetite were excluded from the study.
The study protocol was approved by the institutional review board of Seoul-Paik Hospital, Inje University, and the Korea Center for Disease Control and Prevention.
Informed consent was obtained from the children&#39;s parents.
Title: Genotyping

KARE samples were genotyped using Affymetrix Genome-Wide Human SNP array 5.0 (Affymetrix, Santa Clara, CA, USA) and processed by Bayesian Robust Linear Modeling using the Mahalanobis Distance (BRLMM) genotyping algorithm [16].
The HEXA shared sample was genotyped using Affymetrix Genome-Wide Human SNP array 6.0 and processed with the Birdseed Genotyping Algorithm [17].
The BioBank Japan study was genotyped using the Illumina Human610-Quad BeadChip (Illumina Inc., San Diego, CA, USA).
A childhood obesity study was genotyped using the Illumina Omni1-Quad BeadChip.
Individuals were excluded as follows: genotyping call rate, sex inconsistency, heterozygosity, identity-by-state value, and any kinds of tumor.
For more information, the study has been reported [15].
For the replication study in Health2 (7,861), we performed a genotype assay using the TaqMan reaction for four single nucleotide polymorphisms (SNPs) (rs2074356, rs16940212, rs12708980, rs599839) and the GoldenGate assay (Illumina Inc.) for nine SNPs (rs780092, rs10503669, rs2001945, rs603446, rs12686004, rs11216126, rs12229654, rs519113, rs12654264, rs2738446).
To analyze the quality control, we conducted duplicate genotyping using 1-2.5% of samples.
For further association analyses, concordance rates were satisfied with duplicates of over 99% and a genotype success rate of over 98%.
Title: SNP imputation

In each GWAS data point, imputation analysis was performed using the IMPUTE reference HapMap Asian (Japanese [JPT]/Han Chinese [CHB]) population panel, based on NCBI build 36 and dbSNP build 129.
Of these, in each cohort, we dropped SNPs with a posterior probability score &lt; 0.90, high genotype information content (info &lt; 0.5), Hardy-Weinberg equilibrium (HWE) (p &lt; 1 × 10-7), SNP missing rate &gt; 0.1, and minor allele frequency (MAF) &lt; 0.01.
In the former analysis, only imputed SNPs that satisfied the genomewide SNP quality control criteria were considered.
Title: Association analysis

In performing the association tests, SNPs were analyzed with PLINK (http://pngu.mgh.harvard.edu/~purcell/plink/) and SAS version 9.1 (SAS institute Inc., Cary, NC, USA).
Individuals who were on lipid-lowering therapy and hypertensive medications were excluded for dyslipidemia-related phenotypes, such as TG, HDL-C, and LDL-C. LDL-C concentrations were calculated with Friedewald&#39;s formula [18].
Missing values were assigned for individuals with TG over 400 mg/dL.
The dyslipidemia-related phenotypes were tested by multivariate linear regression analysis in an additive genetic model (1-d.f.
), including age, sex, and recruitment area as covariates.
TG and HDL-C were transformed prior to analysis using natural log transformation to remove skewing.
LDL-C was normally distributed, and no transformations were required.
A GRS was examined using multivariate linear regression analysis, adjusting for age, sex, and recruitment area.
The clinical characteristics and statistics of the variables for each study sample are described in Table 1.
Our previous large-scale GWAS identified 9 novel and 24 known genetic loci influencing metabolic traits in East Asians [9].
In an attempt to determine whether well-established lipid-related variants contribute to significant and reproducible genetic factors for susceptibility to blood lipid concentrations in a childhood obesity study, we first examined the associations between individual genetic variants and lipid-related phenotypes using linear regression analysis.
Of these 16 SNPs for blood lipid traits, rs10503669 in LPL (genotyped SNP in a childhood obesity study) was significantly associated with TG and HDL-C, respectively.
In addition, rs16940212 in LIPC (linkage disequilibrium-based proxy SNP) was associated with HDL-C (Table 2).
The direction of these associations was consistent with our previous GWAS [9].
Having two loci (rs10503669 at LPL and rs16940212 at LIPC) showing the strongest association with TG and HDL-C, we next investigated whether the cumulative allelic dosage of risk alleles contributes to the quantitative variation of lipoprotein concentrations in the KARE study using a method of calculating a GRS.
We evaluated the joint effects of the best-associated SNPs at the two loci showing evidence of association with HDL-C (rs10503669 at LPL and rs16940212 at LIPC).
We calculated a GRS, representing the sum of the risk alleles, and observed that increasing GRS was significantly associated with decreased HDL-C (effect size, -1.13 ± 0.07) compared to SNP combinations without two risk variants (Fig.1).
In addition, a positive correlation was observed between allelic dosage score and risk allele (rs10503669 at LPL) for high TG levels (effect size, 10.89 ± 0.84) (Fig.2).
Blood lipid levels are an important etiological contributor to CVD.
CVD is the leading cause of death among individuals with T2D, in which lipid abnormalities are characterized by hypertriglyceridemia, reduced levels of HDL-C, and elevated levels of LDL-C.
These lipid-related traits, including ~14-54% for TG, ~34-42% for HDL-C, and ~28-50% for LDL-C, are highly heritable in familial T2D or diabetic patients [19, 20].
These estimates indicate that genetic variants play important roles in explaining inter-individual variations in blood lipid levels.
Although systematic reviews have reported relationships between childhood obesity and adult CVD risk [21, 22], there is not a shred of evidence that blood lipid status in adults is an independent or dependent risk factor in a childhood obesity group.
Early childhood obesity might be the best time for dyslipidemia intervention, including screening, early detection, and management.
The aim of this study was to consolidate the associations of 16 GWAS-based candidate variants with blood lipid concentrations across the lifespan.
In GWAS-based candidate approaches, the well-known LPL gene was significantly associated with low TG and high HDL-C levels in a childhood obesity study, respectively.
This finding is well matched to the negative correlation between blood TG and HDL-C concentrations [23].
In addition, the LIPC gene was further observed in decreased HDL-C.
In analyses of the joint effects of these variants, we found that GRSs on the two risk alleles (rs10503669 at LPL and rs16940212 at LIPC) were associated with a cumulative effect of TG and HDL-C levels.
However, these facts indicate that currently identified risk variants might have low discriminatory ability and modest genetic contribution to disease prediction [24].
Lipoprotein lipase (LPL) plays an important role in lipid metabolism by hydrolyzing TGs of circulating chylomicrons and very low-density lipoproteins.
In addition, LPL is expressed in the brain regions that are functionally relevant to learning, memory, and other cognitive functions.
Many GWASs and meta-analyses have shown that variations in the well-known LPL gene for multiple lipid-related traits are implicated in multiethnic populations [2, 3, 9, 25-28].
An increasing number of studies have suggested an association of LPL gene variants with the risk of cerebrovascular as well as CVD [29].
Recent genome-wide investigations identified schizophrenia-associated loci in the chromosome 8p22 region, including LPL [30-32].
It was suggested that LPL is an attractive candidate gene that might be involved in the potential role of lipid metabolism in schizophrenia.
Stages of early childhood development can be taken into account when considering a diagnosis of mental illness [33].
Hepatic lipase (LIPC) gene encodes hepatic triglyceride lipase, which is expressed in liver.
LIPC has a dual function as a triglyceride hydrolase and ligand/bridging factor for receptor-mediated lipoprotein uptake.
A GWAS has identified a possible role of LIPC in advanced age-related macular degeneration (AMD) [34].
AMD is an eye condition affecting the central part of the retina.
Although a condition that is commonly associated with the elderly, macular degeneration affects younger people through a rare inherited condition [35].
Based on a lack of direct relationships between HDL-C and AMD risk, the LIPC association may not be the result of an effect on HDL-C levels, but it could represent a pleiotropic effect as a functional component, such as a lipoprotein transporter, underlying the biological mechanisms involving the cholesterol pathway.
Taken together, our findings demonstrate that the genetic architecture of circulating lipid levels (TG and HDL-C) overlap to a large extent in childhood as well as in adulthood.
A genetic risk assessment by incorporating two risk alleles suggests encouraging evidence that identified genetic risk variants for lipid phenotypes have cumulative effects on lipid concentrations.
Post-GWAS functional characterization of these variants is further required to elucidate the pathophysiological role and biological mechanisms.
Combined effects of risk alleles on high-density lipoprotein cholesterol (HDL-C).
Allelic dosage scores are plotted on the X axis.
The histogram (Y axis on left, grey bars) represents the percentage of individuals in each risk allele category.
The histogram (Y axis on right, black bars) represents mean HDL-C concentrations.
(A) Eight single nucleotide polymorphisms (SNPs) including the risk alleles (rs10503669 at LPL and rs16940212 at LIPC).
(B) Six SNPs without the risk alleles (rs10503669 at LPL and rs16940212 at LIPC).
Combined effects of risk alleles on triglyceride (TG).
Allelic dosage scores are plotted on the X axis.
The histogram (Y axis on left, grey bars) represents the percentage of individuals in each risk allele category.
The histogram (Y axis on right, black bars) represents mean TG concentrations.
(A) Five single nucleotide polymorphisms (SNPs) including the risk allele (rs10503669 at LPL).
(B) Four SNPs without the risk allele (rs10503669 at LPL).
Title: Descriptive statistics of variables for each study cohort

Title: Association between SNPs and lipid-related traits in the population with discovery sets and replication study
Title: Human Population Admixture in Asia

Genetic admixture in human, the result of inter-marriage among people from different well-differentiated populations, has been extensively studied in the New World, where European colonization brought contact between peoples of Europe, Africa, and Asia and the Amerindian populations.
In Asia, genetic admixing has been also prevalent among previously separated human populations.
However, studies on admixed populations in Asia have been largely underrepresented in similar efforts in the New World.
Here, I will provide an overview of population genomic studies that have been published to date on human admixture in Asia, focusing on population structure and population history.
Before I review the recent research progress on Asian admixed populations, I would like to first give some overview of the background of admixture studies, especially in the context of population genomics, since admixture analysis is only feasible when individual genomic data are available.
Human migration resulted in population differentiation, and many populations, especially those living in different continents, have been isolated for quite a long time.
However, subsequent migrations that have occurred over the past millennia have resulted in gene flow between previously separated human subpopulations.
This has been a common phenomenon throughout the history of modern humans, as previously isolated populations often come into contact through colonization and migration.
As a result, admixed populations came into being when previously mutually isolated populations met and inter-married.
It is important to conduct a full analysis of genetic structure and characterize the genetic make-up of admixed populations.
On the one hand, this will shed light on human genetic history; on the other hand, increased population admixture influences genome heterozygosity, which in turn will affect phenotypes relevant to health; thus, genetic admixture has many implications in medical research.
The estimation of genetic admixture in human populations has been used for a variety of purposes, ranging from the confirmation of historical events, such as population of the effects of other biological parameters, but the emphasis in early studies was on gene mapping of continuous traits.
Admixture mapping is a method for localizing disease-causing genetic variants that differ in frequency across populations.
This approach was proposed by McKeigue in 1998 [1]; it is based on the association of local chromosomal ancestry with the disease, to test for the linkage of the disease or trait with parental ancestry at each locus, defined as 0, 1, or 2 allele copies inherited from the ancestral populations.
However, the idea of using genetic architecture in admixed genomes to locate disease-associated genes was actually first proposed by Rife in 1954 [2], although the original idea was to use the admixture linkage disequilibrium (ALD) generated in recently admixed human populations to assign the traits to linkage groups.
Admixture of populations often leads to extended linkage disequilibrium (LD), which could greatly facilitate the mapping of human disease genes [3-6].
Gene mapping by admixture linkage disequilibrium (MALD) has been shown to have special value theoretically [4, 7-14] and empirically [15-29].
MALD has received much attention recently [3, 5].
The initial attraction of MALD was its significantly increased extension of LD (or extended LD) in admixed populations, which requires far fewer markers in mapping disease genes.
Previous studies predicted that MALD typically requires only 2,000-3,000 ancestry informative markers (AIMs) for a genomewide search [1, 16].
The utility of MALD depends upon how far LD extends over a chromosomal interval, which, in turn, dictates the spacing and number of markers required for a genomewide scan.
Therefore, it is imperative to characterize the magnitude and extent of ALD in the admixed population in which MALD will be performed.
Stephens et al.
[10] showed that ALD in African-Americans (AfAs) could exist up to 10 cM, even after 9 generations by simulation.
Parra et al.
[30] reported strong LD between the FY-null and AT3 loci, which are 22 cM apart on chromosome 1q22.
Substantially extended LD in AfA populations were reported in several regions or chromosomes [31-33] using primarily short tandem repeat markers.
Patterson et al.
[12] showed that strong ALD extends to 17 cM on average in AfAs.
Contrarily, it was also reported that the LD in AfA populations is shorter than or similar to those seen in non-admixed populations using single-nucleotide polymorphisms (SNPs)-detectable to about 44 kb or larger [34].
The contradicting observations might be due to the selection of the markers based on their allele frequency difference between the parental populations, as shown theoretically by Chakraborty and Weiss [7].
It is therefore important to examine empirically the relationship of LD in the admixed population and allele frequency difference between parental populations.
In fact, there have been many studies that have investigated LD patterns in recent admixed populations, such as the AfAs, Mexicans, and Hispanics.
In Asia, the only study published so far is on the Uyghurs in Xinjiang, China [35].
In practice, the principle of admixture mapping, proposed by McKeigue [1], is a little different from that of MALD, because it is based on the association of local chromosomal ancestry with the disease rather than on LD between the marker and phenotype, but the information and essence are the same.
It is most advantageous to apply this approach to populations that have descended from a recent mix of two ancestral groups that have been geographically isolated for many tens of thousands of years, providing new insights into population history and genotype-phenotype relationships.
With the efforts of many scientists in this field, statistical methods, panels of AIMs for admixture typing, and software programs have been rapidly developed; in addition, recent advances in genotyping and sequencing technologies have facilitated genomewide investigation of human genetic variations.
Admixture mapping has been applied to a wide range of traits and diseases, for which it is hypothesized that the differences in disease rates across populations are due to population-specific frequency differences of the causal variant(s).
To apply admixture mapping, a set of genetic markers called AIMs, provide information at each locus of the ancestral origin of the allele.
An AIM is a polymorphic locus that exhibits substantially different allele frequencies between the ancestral populations, which are often from different geographical regions.
For example, Fig.1 shows an allele frequency distribution of 9,781 AIMs screened from genomewide SNPs from European (EUR) and East Asian (EAS) populations.
With a set of AIMs, one can estimate the geographical origins of the ancestors of an individual and ascertain what proportion of the ancestry is derived from each geographical region.
The primary utility of estimates of continental ancestry using AIMs is to control for confounding from population stratification in genetic association studies among unrelated individuals from admixed populations.
Population admixture has the potential to lead to confounding results from population stratification in genetic association studies, leading to both false positive (type I error) and false negative (type II error) results [36-39].
Now, to facilitate the application of admixture mapping, a number of high-density mapping panels have been constructed, Such panels are already available for disease gene discovery in AfAs [16, 21, 40, 41], Hispanics/Latinos [23, 26, 40], Mexican-Americans (the largest subgroup of Hispanic/Latinos) [28, 41], and Uyghurs [35].
Identifying targets of positive selection in humans has been a central topic in human evolutionary and population genetics.
Traditional studies have been relying on the analysis of individual candidate genes.
Since the genomics era, the availability of high-density SNPs has provided essential resources for systematically interrogating the entire genome for detecting signatures of natural selection.
As a matter of fact, many genomewide scans for recent or ongoing positive selection have been recently performed in humans, while most of the studies to date have been performed in non-admixed populations.
In the situation of genetic admixture, when previously isolated populations meet and mix, the resulting admixed population can benefit from several genetic advantages, including increased genetic variation, the creation of novel genotypes, and the masking of deleterious mutations, and these admixture benefits are thought to play an important role in biological adaptation to local environment.
As a result, this has led to the use of admixture estimates to detect the action of natural selection in human populations.
Such techniques have been regarded as powerful methods for the detection of selection effects, especially when an appreciable number of generations have elapsed since the original admixture.
Several studies based on genomic data have been conducted in recently admixed populations in the New World [42-44].
Recently, we developed a new approach that can be used to detect natural selection both before and after population admixture, and we applied this method in AfA data.
It is particularly meaningful to investigate natural selection in AfAs due to the high mortality their African ancestry has experienced in history.
In this study, we examined 491,526 autosomal SNPs genotyped in 5,210 individuals and conducted a genomewide search for selection signals in 1,890 AfAs.
Several genomic regions showing excess African or EUR ancestry, which were thought of as the footprints of selection since population admixture, were detected based on a commonly used approach.
However, we also developed a new strategy to detect natural selection both pre- and post-admixture by reconstructing an ancestral African population (AAF) from inferred African components of ancestry in AfAs and comparing it with indigenous African populations (IAF).
Interestingly, many selection candidate genes identified by the new approach were associated with AfA-specific high-risk diseases, such as prostate cancer and hypertension, suggesting an important role that these disease-related genes might have played in adapting to a new environment.
CD36 and HBB, whose mutations confer a degree of protection against malaria, were also located in the highly differentiated regions between AAF and IAF.
Further analysis showed that the frequencies of alleles protecting against malaria in AAF were lower than that in IAF, which is consistent with the relaxed selection pressure of malaria in the New World.
There is no overlap between the top candidate genes detected by the 2 approaches, indicating the different environmental pressures AfAs experienced pre- and post-population admixture.
We suggest that the new approach (see Fig.2 for a schematic framework of this approach) is reasonably powerful and can also be applied to other admixed populations, such as Latinos and Uyghurs.
I will not go further on this topic in this review paper, since so far there is no study that has been performed in Asian populations using admixture analysis.
However, we believe that the discoveries have greatly enriched our understanding of human origins and history and hold large potential for identifying genes with important biological functions; this, in turn, will elucidate the genetic basis of some human diseases [45, 46].
These studies together provide many new insights into the natural selection process and mechanisms, which will ultimately improve the modern evolution theory.
Human diasporas over the past millennia have resulted in even more frequent intercontinental marriages and population admixtures.
Many recently admixed populations, such as AfAs and Mestizos, have received much attention due to their potential advantages in the discovery of disease-associated genes.
Particularly, a disease gene mapping strategy, named admixture mapping or MALD, as aforementioned, has been developed [1, 7, 11, 47].
The statistical power of admixture mapping or MALD relies on the fact that population admixture creates extended and elevated LD between loci with different allele frequencies among the parental source populations [7, 10, 48], while the historical population admixture processes determine the LD pattern in an admixed population.
Therefore, as shown in several theoretical and simulation studies, population admixture dynamics have a strong effect on the power of admixture mapping [48-51].
In fact, accurate understanding of population admixture dynamics is important not only to admixture mapping but also to other applications, such as elucidating population history [52] and detecting natural selection signatures in admixed populations [49, 53].
However, the fine admixture dynamics of recently admixed populations has not been well established, even though some studies have examined the simulated data [54, 55] or experimental data with sparse markers [48, 51].
In addition, admixture dynamics of relatively ancient admixed populations has rarely been studied.
Recently, the availability of genomewide high-density SNPs data has facilitated the study on detailed genetic structures of admixed populations [35, 44, 56-61].
However, most of these studies relied on simplified models that do not take into account the inherent complexity of the admixture processes.
Moreover, the haplotype and chromosomal segment patterns shaped by recombination within each individual have been deliberately ignored in most studies due to many inherent challenges [62].
For individuals from admixed populations that have existed for a long time, their chromosomes resemble a mosaic of chromosomal segments of distinct ancestry (CSDA).
The CSDA in the admixed population would have been reshaped and rearranged by recombination in each generation, which should provide some information about the population history.
In other words, the CSDA will be spliced into smaller pieces as the number of generations since admixture increases, while the chromosomes from recently admixed individuals contain much longer CSDAs.
Information regarding the average CSDA length has been used to infer the number of generations since admixture in various studies [57, 63-66].
However, the distribution of CSDA length may contain more valuable information concerning population admixture history and admixture dynamics, which has not yet been explored.
In a recent study [67], we proposed 2 approaches to explore population admixture dynamics and, by analyzing genomewide empirical and simulated data, demonstrated that the approach based on the distribution of CSDA was more powerful than that based on the distribution of individual ancestry proportions.
As a result, analysis of 1,890 AfAs showed that a continuous gene flow (CGF) model (Fig.3), in which AfAs continuously received gene flow from EUR populations over about 14 generations, best explained the admixture dynamics of AfAs among several putative models.
Interestingly, we observed that some AfAs had much more EUR ancestry than the simulated samples, indicating substructures of local ancestries in AfAs that could have been caused by individuals from some particular lineages having continuously inter-married with people of EUR ancestry.
On the contrary, the admixture dynamics of Mexicans was more likely to be explained by a gradual admixture (GA) model (Fig.3), in which Mexicans continuously received gene flow from both EUR and Amerindian populations for about 24 generations.
The higher proportion of long CSDAs observed in Mexican-Americans relative to Mexican-Mestizos suggested recent gene flow from EUR-Americans to Mexican-Americans.
The genetic components of sub-Saharan Africans in Middle Eastern populations, such as Mozabite, Bedouin, and Palestinians, could be explained by one early admixture followed by some recent gene flow from Africans.
In summary, this study not only provides new approaches to explore population admixture dynamics but also advances our understanding on historical admixture processes of AfAs, Mexicans, and Middle Eastern populations.
Again, since admixed populations in Asia have not been studied, I will not go further into details.
This approach can be also applied in admixed populations in Asia to explore the admixture dynamics of admixed populations in Asia.
In East Asia, most of the populations that have been studied are thought to be relatively homogeneous.
Most people identify themselves with one particular ethnic group, but in fact, we are highly mixed.
A report from the HUGO PanAsia SNP Consortium, which was published in Science in 2009 [68], revealed a picture of prevalent gene flow among Asian populations.
As a collaborative effort of 93 scientists from 10 countries, this study conducted the first large-scale and genomewide study on the genetic diversity of 73 Asian populations, representing a broad geographic sample of the major ethnic groups and linguistic families in Asia.
Considerable gene flow among Asian populations was observed amongst sub-populations in these clusters, including those groups believed to practice endogamy based on linguistic, cultural, and ethnic information.
In fact, most populations studied show evidence of admixture in population structure analyses.
For example, the Han Chinese have grown to become the largest ethnic group today, in a demographic expansion that has occurred mostly within historical times.
The analysis in this study reveals that the 6 Han Chinese populations show varying degrees of admixture (Fig.4) between a northern &#39;Altaic&#39; cluster and a &#39;Sino-Tibetan/Tai-Kadai&#39; cluster, which is most frequent in the ethnic groups sampled from southern China and northern Thailand.
Finally, most of the Indian populations showed evidence of shared ancestry with EUR populations, which is consistent with our understanding of the expansion of Indo-EUR speaking populations (Fig.4).
Many human populations that settled in Central Asia demonstrate an array of mixed anthropological features of East Eurasians (EEA) and West Eurasians (WEA), indicating a possible scenario of biological admixture between already differentiated EEA and WEA populations.
However, their complex biological origin and genomic make-up, as well as their genetic interaction with surrounding populations, are not well understood.
Despite Central Asia being a vast territory that has been crucial in human history due to its strategic location [69], populations in this region are not on the list of the 2 large-scale international collaborative efforts, such as the International HapMap Project [70] and the 1000 Genome Project [71], and population genomic studies on Central Asian populations have been largely underrepresented in similar efforts worldwide [68, 72-76].
In an attempt to decipher the genetic structure and population history in Central Asia, we conducted, to our knowledge, the first genomewide studies of the Uyghur residing in Xinjiang, which is geographically located in Central Asia.
Uyghur that settled in Xinjiang of China (UIG) is a population presenting a typical admixture of eastern and western anthropometric traits; we dissected its genomic structure at the population level, individual level, and chromosome level using 20,177 SNPs spanning over almost the entire chromosome 21.
Our results showed that UIG was formed by two-way admixture, with 60% of ancestry from EUR and 40% of ancestry from EAS.
Overall LD in UIG was similar to that in its parental populations, represented by EAS and EUR for common alleles, and UIG manifested elevation of LD only within 500 kb and at a level of 0.1 &lt; r2 &lt; 0.8 when AIMs were used.
The size of chromosomal segments that were derived from EASs and EURs was on average 2.4 cM and 4.1 cM, respectively.
Both the magnitude of LD and fragmentary ancestral chromosome segments indicated a long period of history of Uyghur.
Under the assumption of a hybrid isolation (HI) model, the admixture event of UIG was estimated to have taken place about 126 (107-146) generations, or 2,520 (2,140-2,920) years ago, assuming 20 years per generation.
In spite of the long history and short LD of Uyghur compared with recent admixture populations, such as African-Americans, we suggested that MALD is still applicable in Uyghur, except that about 10-fold AIMs are necessary for a whole-genome scan.
Later in the same year, following up on our previous study, we conducted a genomewide analysis of admixture for 2 Uyghur population samples (HGDP-UG and PanAsia-UG) collected from northern and southern Xinjiang of China, respectively.
Both HGDP-UG and PanAsia-UG showed substantial admixture of EAS and EUR ancestries, with an empirical estimation of ancestry contribution of 53:47 (EAS:EUR) and 48:52 for HGDP-UG and PanAsia-UG, respectively.
The effective admixture time under a model with a single pulse of admixture was estimated to be 110 generations and 129 generations; i.e., admixture events occurred about 2,200 years and 2,580 years ago for HGDP-UG and PanAsia-UG, respectively, assuming an average of 20 years per generation.
Despite their earlier history versus other admixture populations, such as AfAs and Latinos, admixture mapping, an economical and powerful approach for localizing disease genes, holds equal promise for Uyghurs because of their mixture of ancestry from different continents as well as their large population size.
We screened multiple databases and identified a genomewide SNP panel that can distinguish EAS and EUR ancestry of chromosomal segments in Uyghurs.
The panel contains 8,150 AIMs showing large frequency differences between EAS and EUR (FST &gt; 0.25, mean FST = 0.43) but little frequency difference (7,999 AIMs validated) within both EAS and EUR populations (FST &lt; 0.05, mean FST &lt; 0.01).
We evaluated the effectiveness of this admixture map for localizing disease genes in 2 Uyghur populations.
Our map constitutes the first practical resource for admixture mapping in Uyghurs, and it will enable studies of diseases showing different genetic risks in European and EAS populations.
Southeast Asia, together with the Island Pacific region, is a cultural melting pot of migrating Neolithic farmers and indigenous Mesolithic hunter-gatherer communities.
On the basis of increased cultural, linguistic, and genetic diversity, the origins of Southeast Asian populations are thought to be more complex than those to their north.
Two major prehistoric movements of people had great influence on the linguistic, cultural, and genetic diversity of the region.
The first colonization of modern humans in the region was around 45,000 years before the present [77], which could have brought the ancestors of modern Papuans and Australians into the region [78].
Notably, archaeological evidence has confirmed human habitation in Eastern Indonesia as far back as 32,000 years ago [79].
The second major human migration event, which is often termed the Austronesian expansion, was relatively recent, coinciding with the first agricultural settlements in Island Southeast Asia (ISEA) [80]; it is often putatively linked to demic dispersal from mainland China during the Mid-Holocene [81], and based on linguistic studies, the Austronesian expansion is proposed to have originated in Taiwan around 5,000-6,000 years ago and spread to Southeast Asia, Near and Remote Oceania, and Madagascar [82, 83].
However, our knowledge about the historical origin and spread of Austronesian-speaking peoples has been overwhelmingly from linguistic, archeological, and anthropological studies, while contributions from genetic studies are limited, and especially, genetic dating of the expansion time of Austronesians is almost absent [84].
Undoubtedly, the Austronesian expansion has had a major cultural influence on Asia-Pacific pre-history.
The biological contribution of Austronesian-speaking peoples to the pre-existing local populations in ISEA was first noted by Wallace in 1869 [85].
As Wallace observed, people in the region between mainland Asia and New Guinea in the west are similar to their neighbors in mainland Southeast Asia, while the eastern groups near New Guinea are similar to Melanesians, leaving intervening populations intermediate in appearance [85].
This admixture pattern was revealed by mtDNA and non-recombining region of the Y chromosome (NRY) studies [86, 87] as well as by a recent study [81] based on multiple autosomal and X-chromosomal SNPs.
The genetic admixture with local populations during the Austronesian expansion could provide a unique opportunity to shed light on the migration history of Austronesian-speaking people.
However, previous studies based on haploid genomes could not study admixture at the individual level.
Studies of multiple loci of the nuclear genome are more powerful [88] in studying admixture at the individual level, but the sparse low-density loci examined in previous genomewide studies do not allow estimation of the admixture time, which relies on recombination information extracted from high-density genomewide data.
We analyzed dense genomewide SNP data from 2 studies [68, 89].
The first consists of about 50,000 SNPs analyzed in 288 individuals from 13 Austronesian-speaking populations and 2 Papuan-speaking populations across Indonesia and Papua New Guinea, while the second consists of about 680,000 SNPs analyzed in 36 individuals from 7 populations from Indonesia and 25 individuals from New Guinea.
Apart from a general description of population structure and its relationship with geography, language, and demographic history, we focus on estimating the amount and time of admixture involving Asian and Papuan ancestry across E. Indonesia to test the various scenarios that might explain the admixture history.
Because genetic recombination breaks down parental genomes into segments of different sizes, the genome of a descendant of an admixture event is composed of different combinations of these ancestral segments, or &#39;blocks&#39; [90].
Admixture time can be estimated from the information based on distribution of ancestral segments and the recombination breakpoints in an admixed genome [34, 57, 63, 66, 90].
In this study, we assume that a cline of admixture times that decreases from west to east across E. Indonesia would support a scenario of incoming Asians admixing with indigenous Papuan groups, while a cline in the opposite direction would indicate incoming Papuans admixing with resident Asians.
Moreover, the actual admixture times would indicate when the migration occurred.
Our approach is thus to estimate population admixture time by taking advantage of available genomewide data of both E. Indonesian populations showing admixture and their ancestry reference populations.
In summary, the admixture time analysis of 2 different datasets provides compelling evidence that people of Asian ancestry began moving through E. Indonesia about 4,000 years ago, from west to east, and admixed with resident groups of Papuan ancestry.
Furthermore, this scenario is in excellent agreement with linguistic and archeological evidence for a pre-Austronesian presence of Papuans in E. Indonesia [91, 92] and for an eastward spread of Austronesian-speaking farmers across E. Indonesia [82, 83].
Our analyses thus refute suggestions that the Asian ancestry observed in Indonesia largely predates the Austronesian expansion [93, 94].
Instead, our analyses of genomewide data indicate that there was a strong and significant genetic impact associated with the Austronesian expansion in Indonesia, just as similar analyses have pointed to a strong and significant genetic impact associated with the Austronesian expansion through Near and Remote Oceania [87, 95, 96].
Given that admixture among human populations (and between modern and archaic humans) is increasingly being recognized as a significant aspect of modern human biology [97], estimates of the time of admixture should provide important new insights into the history of our species.
All the published results in this area so far have been focused on populations in India.
There were recently 2 studies on the Siddis, which are a group of people showing evident genetic admixture in India.
The Siddis (Afro-Indians) are a tribal population whose members live in coastal Karnataka, Gujarat, and some parts of Andhra Pradesh.
Historical records indicate that the Portuguese brought the Siddis to India from Africa about 300-500 years ago; members of this population are believed to be descendants of the Bantu-speaking population of Africa.
However, there is little information about their more precise ancestral origins.
One Indian group, led by Dr. Mukerji, studied this population [98] based on genomewide SNP data generated using Affymetrix 50K chips.
The authors carried out this study by using a set of 18,534 autosomal markers common between Indian, CEPH-HGDP, and HapMap populations.
Principal components analysis clearly revealed that the African-Indian population derived its ancestry from Bantu-speaking west African as well as Indo-European-speaking north and northwest Indian population(s).
STRUCTURE and ADMIXTURE analyses show that overall, the OG-W-IPs derived 58.7% of their genomic ancestry from their African past and have very little inter-individual ancestry variation (8.4%).
The extent of LD also reveals that the admixture event has been recent.
Functional annotation of genes encompassing the AIMs that are closer in allele frequency to the Indian ancestral population revealed significant enrichment of biological processes, such as ion channel activity, and cadherins.
They briefly examined the implications of determining the genetic diversity of this population, which could provide opportunities for studies involving admixture mapping.
The second group, led by Dr. Thangaraj, performed a genomewide survey to understand the population history of the Siddis [99].
Using hundreds of thousands of autosomal markers, they showed that the Siddis inherited ancestry from Africans, Indians, and possibly Europeans (Portuguese).
Additionally, analyses of the uniparental (Y-chromosomal and mitochondrial DNA) markers indicate that the Siddis trace their ancestry to Bantu speakers from sub-Saharan Africa.
They estimated that the admixture between the African ancestors of the Siddis and neighboring South Asian groups probably occurred in the past 8 generations (-200 years ago), consistent with historical records.
Recent advances in genotyping and sequencing technologies have facilitated genomewide investigation of human genetic variations and provided new insights into population structure and admixture history.
Admixed populations are attracting more and more attention from both evolutionary and medical studies as well as from the other fields.
Without understanding the genetic structure and history of admixed populations well, our knowledge about human genetics would not be complete.
Unfortunately, studies on admixed populations in Asia are so far very limited, although gene flow among Asian populations is prevalent [68].
The studies published and mentioned in this review have been focused on investigating the genetic structure and genetic history of a limited number of well-known admixed populations in Asia.
Further efforts should be made to reveal local adaptation signatures and also apply admixture mapping in those well-known admixed populations; in the future, more powerful methods are expected to be developed and applied to many more populations with longer histories and more complex admixture scenarios.
European and East Asian allele frequencies for the 9,781 ancestry informative markers.
Blue dots indicate allele frequency of European and East Asian populations.
Red dots indicate allele frequency of an admixed population from Xinjiang.
A schematic framework for detecting natural selection in African-American populations based on admixture analysis.
Typical admixture models.
HI model and CGF model were adapted from Long [49], GA model was adapted from Ewens and Spielman [50].
In each model, the genetic contributions of Pop1 and Pop2 are m1 and m2, respectively.
The admixed population experienced Gi generations, which range from 1 to t generations.
Estimated population structure.
Each individual is represented by a thin vertical line, which is partitioned by STRUCTURE into K colored segments representing estimated membership fractions in each K cluster.
Black lines separate individuals of different populations.
Populations are labeled below the figure.
All population IDs except the 4 HapMap samples (YRI, CEU, CHB, and JPT) are denoted by 4 characters.
The first 2 letters indicate the country where the samples were collected or (in the case of Affymetrix) genotyped according to the following convention: AX, Affymetrix; CN, China; ID, Indonesia; IN, India; JP, Japan; KR, Korea; MY, Malaysia; PI, Philippines; SG, Singapore; TH, Thailand; TW, Taiwan.
The last 2 letters are unique IDs for the population.
Populations from the same linguistic group or neighboring geographic locations tend to share the same cluster.
At K = 14, each language family can be specified by a cluster (color), although Sino-Tibetan-speaking populations tend to cluster with both Altaic- and Tai-Kadai-speaking populations.
The figure shown for a given K is based on the highest probability run of 10 STRUCTURE runs at that K.
Title: Characterization of Chromatin Structure-associated Histone Modifications in Breast Cancer Cells

Chromatin structure and dynamics that are influenced by epigenetic marks, such as histone modification and DNA methylation, play a crucial role in modulating gene transcription.
To understand the relationship between histone modifications and regulatory elements in breast cancer cells, we compared our chromatin immunoprecipitation sequencing (ChIP-Seq) histone modification patterns for histone H3K4me1, H3K4me3, H3K9/16ac, and H3K27me3 in MCF-7 cells with publicly available formaldehyde-assisted isolation of regulatory elements (FAIRE)-chip signals in human chromosomes 8, 11, and 12, identified by a method called FAIRE.
Active regulatory elements defined by FAIRE were highly associated with active histone modifications, like H3K4me3 and H3K9/16ac, especially near transcription start sites.
The H3K9/16ac-enriched genes that overlapped with FAIRE signals (FAIRE-H3K9/14ac) were moderately correlated with gene expression levels.
We also identified functional sequence motifs at H3K4me1-enriched FAIRE sites upstream of putative promoters, suggesting that regulatory elements could be associated with H3K4me1 to be regarded as distal regulatory elements.
Our results might provide an insight into epigenetic regulatory mechanisms explaining the association of histone modifications with open chromatin structure in breast cancer cells.
The epigenetic regulation by DNA methylation and post-translational modifications of histones without altering DNA sequences is tightly linked to the gene expression program in eukaryotic genomes.
The N-terminal tails of histones are subject to various types of modifications, such as acetylation, methylation, phosphorylation, ubiquitination, glycosylation, and sumoylation [1].
Dynamic changes by histone-modifying enzymes affect the chromatin accessibility to transcriptional machinery and thus modulate gene activation or silencing in diverse biological processes, such as transcription, DNA repair, development and differentiation, and genome stability [2-4].
Moreover, nucleosome positioning can also be controlled by specific histone modifications, leading to alteration of cross-talk among chromatin structure, exon-intron architecture, and RNA polymerase II binding [5, 6].
Genomewide profiles of histone modifications with chromatin immunoprecipitation sequencing (ChIP-Seq) have revealed the characteristic genomic distribution and the association of gene functions and activities in eukaryotic genomes [7-12].
ChIP-Seq is a popularly used method to find genomewide protein binding sites with a high resolution by chromatin-immunoprecipitation by massive DNA sequencing using high-throughput sequencing technology [11].
For example, trimethylated histone H3 lysine 4 (H3K4me3), which is catalyzed by trithorax complex, as well as histone acetylation (e.g., H3K9/16ac) are usually enriched at promoter regions or transcription start sites (TSSs) with open chromatin structure and positively correlated with gene transcriptional activation level.
H3K4me1 is also found to be enriched at enhancer-associated regions.
These modifications recruit the transcriptional machinery at target sites.
Trimethylation of H3K27, catalyzed by polycomb group protein complex 2 (PRC2), contributes to gene silencing by promoting chromatin condensation and chromatin stabilization and is likely to spread over larger regions around TSSs of silent genes.
The signals of H3K9me3 are high in silent genes or repressed chromatin domains, such as heterochromatin.
The comprehensive and comparative analysis of many histone modifications in the human genome demonstrates that the combination of several histone modifications shows a modular pattern and regulates transcriptional activation in a cooperative manner [13].
Interestingly, many loci of H3K27me3 are often colocalized with those of H3K4me3 around TSSs of lowly expressed genes in stem-like cells, many of which are positioned at developmental regulators, including transcription factors and signaling proteins [14].
This suggests that the formation of a bivalent chromatin domain can be a signature of epigenetic memory and that programmed gene expression during differentiation is dependent on chromatin modifications [14].
The interactions of variations in the genome, epigenome, and transcriptome with sensing of environmental stimuli can determine phenotypic plasticity.
Traditionally, phenotypic variation has been explained primarily through genetic variation with sequence changes during evolution.
However, epigenetic variations that are potentially sensitive to environmental inputs can alter transcriptional activity, which in turn contributes to diversity of complex traits [15].
Such observations have been demonstrated in human cancers and plant development, such as floral symmetry and vernalization response (reviewed in [15, 16]).
For example, silencing of tumor suppressor genes, including p16, VHL, MLH1, APC, and E-cadherin, is associated with DNA hypermethylation in the gene promoters [17].
In relation to histone modifications, overexpression of EZH2, a histone H3K27 methyltransferase, has been observed and positively correlated with the progression of multiple malignancies, including prostate cancer, breast cancer, lymphoma, myeloma, colorectal cancer, endometrial cancer, bladder cancer, and melanoma [18].
In addition, generalized loss of H4K16ac and H4K20me3 is found in lymphoma and colorectal cancer, leading to transcriptional silencing [16].
Organization of genomic DNA into higher-order chromatin structures is involved in transcriptional regulation in eukaryotes [19, 20].
In particular, cis-regulatory elements complexes located in open chromatin regions, depleted of nucleosomes, are activated by the recruitment of transcriptional machinery [19, 20].
Such active regulatory elements have been identified by formaldehyde-assisted isolation of regulatory elements (FAIRE) technique, a simple high-throughput screening method to isolate and map active regulatory elements depleted of nucleosomes in eukaryotes, especially in clinical samples, by forming crosslinks between histones and DNA and subsequent utilization of hybridization of tiling microarrays or next-generation sequencing [21-23].
FAIRE was first demonstrated in Saccharomyces cerevisiae, where formaldehyde-crosslinked chromatin immediately upstream of genes was preferentially segregated into the aqueous phase in a manner that was strongly negatively correlated with nucleosome occupancy [21].
Results from both yeast and human samples showed that enrichment of regions upstream of genes was positively correlated with transcription of downstream genes [21-23].
Breast cancer is a heterogeneous and progressive disease, known as the most common cancer among women.
For early diagnosis and prognosis, the development of tumor makers in breast cancer is important, but the markers for its early detection are rare [24, 25].
The most clinically useful tissue-based marker genes in breast cancer are steroid receptors, estrogen receptors, progesterone receptors, and HER-2 [24, 25].
The mutations of BRCA1 and BRCA2 are strong indicators of breast cancer development, but their mutation is not common [25].
Except for genetic variation-based diagnosis, a comprehensive understanding of the gene expression program as well as epigenetic mechanisms is definitely required for the development of diagnosis markers and epigenetic therapies.
We recently reported epigenetic regulatory mechanisms of gene expression with 3 different histone modifications (H3K4me1, H3K4me3, and H3K9/14ac) in normal (MCF-10A) and breast cancer (MCF-7) cells [26].
In particular, we demonstrated the change of transcriptional activity, delineating differential enrichment of histone modifications in both MCF-10A and MCF-7, which defines the functional regulatory elements in the genome with a cell type-specific chromatin environment [26].
To address the relationships between histone modifications and FAIRE and between FAIRE-related chromatin structure and transcriptional activity, we analyzed FAIRE-chip [23], histone modification-ChIP-Seq [26], and gene expression microarrays derived from MCF-7.
Our result provides an understanding of epigenetic regulatory mechanisms with open chromatin in breast cancer cells.
Title: Subjects

The FAIRE-chip data, covering human chromosomes 8, 11, and 12, derived from MCF-7 cell lines, were downloaded from the NCBI Gene Expression Omnibus (GEO) database (GSE11579) [23].
The ChIP-Seq data (SRA045635) of H3K4me1, H3K4me3, H3K9/14ac, and input DNA (mononucleosome digested with micrococcal nuclease) in MCF-7 were obtained from the Sequence Read Archive (SRA) at the National Center for Biotechnical Information (NCBI) [26].
The H3K27me3 ChIP-Seq data were generated with Genome Analyzer IIx (Illumina Inc., San Diego, CA, USA) according to the method described by Choe et al.[26].
The gene expression data using the Human Genome U133Plus 2.0 array platform (Affymetrix Inc., Santa Clara, CA, USA) were collected from the GEO database (GSM276046-GSM276048 for MCF-7) [27].
Title: Peak identification and statistical analysis

The FAIRE-chip data were processed by using CisGenome [28] for normalization, signal detection, and identification of significant FAIRE regions.
The 26-bp ChIP-Seq reads for H3K4me1, H3K4me3, H3K9/14ac, H3K27me3, and input DNA were aligned to a human reference sequence (hg18) using the CASABA 1.6 program (Illumina Inc.), and the resulting mapped tag counts were normalized for the comparative analysis.
To identify peaks enriched with a specific histone modification, the Hypergeometric Optimization of Motif EnRichment (HOMER) package version 3.2 [29] was used with the following options: approximate fragment length, 150 bp; peak size, 150 bp; minimum distance between peaks, 370 bp (equivalent to peak size × 2.5); Poisson p-value threshold relative to local tag count, 0.0001; default false discovery rate threshold, 0.001; and center switch for centering peaks on maximum ChIP fragment overlap and calculating focus ratios.
The FAIRE sites coinciding with histone-modified peaks were defined when the distance between the center positions of FAIRE sites and histone-modified peaks was shorter than 100 base pairs.
The co-existing sites of FAIRE and histone modifications were plotted, centered at the genes&#39; TSSs using seqMINER with k-mean clustering method [30].
Expression levels of genes associated with the FAIRE-histone-modified regions were examined, and gene ontology (GO) enrichment and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway analysis were performed by using the DAVID Functional Annotation Tool [31].
In addition, functional, enriched motifs in FAIRE-histone-modified regions were also found by using MEME suite [32] and the TOMTOM motif database [33].
Title: Chromatin structure defined by FAIRE and histone modifications

The chromatin structure was analyzed by comparing genomic regions defined by FAIRE with histone modification sites detected by ChIP-Seq in a breast cancer cell line, MCF-7.
Due to the limited information of FAIRE-chip data, we analyzed regions enriched in human chromosomes 8, 11, and 12.
The overall positions of regulatory elements and histone modifications relative to TSS are depicted in Fig.1A.
The k-clustered pattern, depending on their enrichment level, showed that most H3K4me3 and H3K9/14ac modifications, known as active chromatin marks, were enriched near TSS, whereas H3K27me3, a repressive mark, was not.
Many regulatory elements detected by FAIRE-chip were also located near TSS, reflecting that promoters are one of the major nucleosome-free sites.
The relative positions of 3 histone modification enrichments (H3K4me1, H3K4me3, and H3K9/14ac) were aligned to the center of FAIRE signals and overlapped within ± 1-kb regions from TSSs (Fig.1B).
The H3K4me1 had a broader spectrum than the other 2 modifications in ± 0.5-1.5-kb regions.
These results implied that regulatory elements were highly correlated with active histone modifications and associated with open chromatin structure.
A total of 2,804 regulatory elements from FAIRE-chip data were identified by CisGenome analysis, and the number of ChIP-Seq peaks was calculated using HOMER program: 18,938 for HK4me1, 4,516 for H3K4me3, 5,763 for H3K9/14ac, and 3,324 for H3K27me3 (Table 1).
The FAIRE sites were located in especially functional element-related regions: promoters (32.2% of FAIRE sites analyzed), 4 kb upstream of promoters (3.2%), gene bodies (39.1%), and intergenic regions (25.4%).
In particular, the highest enrichment of FAIREs in promoters could be identified by the normalization in quantitative comparison of FAIRE profiles (i.e., the total number of peaks/total length of each of genomic feature).
We selected the overlapping regulatory elements of FAIRE signals with ChIP-Seq peaks and looked at their co-occurrence; the regulatory elements with H3K4me1 (FAIRE-H3K4me1) were 1,006; FAIRE-H3K4me3, 1,000; and FAIRE-H3K9/14ac, 1,264.
Among them, 334 elements showed enrichment of both H3K4me3 and H3K9/14ac (Fig.1C).
This relationship was further confirmed by the distribution of FAIRE-histone modifications, shown in Fig.1D.
The highest population of FAIRE-H3K4me3 and FAIRE-H3K9/14ac was observed immediately downstream of TSS (Fig.1D).
A weak enrichment of FAIRE-H3K4me1 elements was detected upstream of TSS, where a shoulder peak of FAIRE-H3K4me1 and H3K4me3 was positioned.
For example, 4 FAIRE regulatory elements of cyclin D1 (CCND1), involved in tumorigenesis as a cell cycle regulator, were located at the promoter and overlapped with peaks of H3K4me3 and H3K9/14ac but not with H3K4me1 or H3K27me3 (Fig.1E).
Instead, the H3K4me1 peaks in the CCND1 gene locus were expanded along the gene body and far upstream of TSS.
Title: Gene expression and FAIRE-histone modifications

The gene expression program is tightly controlled by a dynamic chromatin environment, which epigenetic factors, like histone modifications and DNA methylation, play a crucial role in determining.
As shown in Fig.1, we found the linkage of FAIRE regulatory elements with histone modification profiles.
To further assess the FAIRE-histone modifications, the gene expression profiles for MCF-7 cells were integrated (Fig.2).
From the overlapped regulatory elements determined by comparison of FARE-H3K4me1 (1,006), FAIRE-H3K4me3 (1,000), and FAIRE-H3K9/14ac (1,264), we selected 229 genes associated with at least 2 of 3 FAIRE-histone modification combinations.
Scatter plots were produced to see how the expression level agreed with degree of histone modification (Fig.2A-2C).
The Pearson correlation coefficients between histone modifications and the expression level of genes with FAIRE elements were generally low.
The highest coefficient was r = 0.50 for the pair of gene expression and FAIRE-H3K9/14ac (Fig.2C), and the next was r = 0.4 for gene expression and FAIRE-H3K4me3 (Fig.2B).
However, FAIRE-H3K4me1 showed almost no correlation with gene expression level (r = -0.03) (Fig.2A).
To examine whether breast cancer-related genes were up-regulated in MCF-7 and appeared to have a high level of H3K9/14ac, as found in our previous study [26], we selected 68 genes, the expression levels of which ranked in the top 30% among FAIRE-H3K9/14ac-associated genes, and performed DAVID Functional Annotation analysis.
We could isolate 29 functionally significant genes with p &lt; 0.05: ATM, BTG1, CCND1, CDK4, CDKN1B, CRADD, CSDA, CTR9, DDB2, DUSP6, ERBB3, ESPL1, FADD, H2AFX, KRT18, KRT8, MADD, MDM2, MYC, NR4A1, POLA2, RIPK2, RRM2B, SART3, SMARCC2, TSG101, UBE2N, XPOT, and YWHAZ.
These genes were associated with the following GO categories: regulation of apoptosis, programmed cell death, nuclear lumen, protein ubiquitination, DNA damage checkpoint, mitotic cell cycle, and small conjugating protein ligase activity (Fig.2D).
Moreover, the KEGG pathway analysis displayed their involvement in the cell cycle, p53 signaling pathway, mitogen-activated protein kinase signaling pathway, and pathways in cancer (Fig.2E).
Title: Sequence motif analysis for FAIREs marked by active histone modifications

As many FAIRE regulatory elements were linked with active histone modifications, we explored the possible existence of functional sequence motifs for known transcription factors in FAIRE-H3K4me3 and FAIRE-H3K9/14ac sites.
The binding motifs, such as CTCF, MYB, GFY-staf, ETS, and NRF1, were common in both FAIRE-H3K4me3 and FAIRE-H3K9/14ac sites (Fig.3A).
However, NFY and RUNX motifs existed only in FAIRE-H3K4me3 sites, and the GATA3 motif was specifically detected in FAIRE-H3K9/14ac sites.
For FAIRE-H3K4me1 sites, AP-1/2, NF1, CTCF, AP-2, FOXA1, USF1, and MAFA motifs were identified (Fig.3B).
Interestingly, the CTCF motif was commonly found in FAIRE-H3K4me1, FAIRE-H3K4me3, and FAIRE-H3K9/14ac.
The genomewide positioning of regulatory elements marked by histone modifications is illustrated in a Venn diagram (Fig.3C-3E).
More than 60% of the FAIRE-H3K4me3 and FAIRE-H3K9/14ac regions carrying binding motifs were distributed at gene promoters; over 20% in the gene body; and at small portions far upstream of promoters (Fig.3D and 3E).
In contrast, the population of FAIRE-H3K4me1 was highly enriched in gene body regions (66.4%) as well as upstream of promoters (31.3%) but almost negligible at promoters (2.3%) (Fig.3C).
FAIRE has been known to enrich functional DNAs located in DNase I hypersensitive sites, active promoters, and transcriptional start sites [22].
The enrichment of such regulatory regions in the aqueous phase might result in easy identification of genomic function without bias.
Eeckhoute et al.
[23] demonstrated that FOXA1-bound enhancers defined by FAIRE in human chromosomes 8, 11, and 12 were closely related with cell-type specific chromatin remodeling.
In addition, it was reported that FAIREs are highly associated with DNase I hypersensitivity sites, RNA polymerase II, and TAF1 binding sites [22].
In combination with the information on these FAIRE regulatory elements in MCF-7 cells, we analyzed our genomewide histone modification data (H3K4me1, H3K4me3, and H3K9/14ac) generated by ChIP-Seq [26].
It was shown that the H3K4me1 distribution, covering the entire human genome, which was different from the pattern of H3K4me3 and H3K9/14ac, was 53% of the gene body, 42% of the intergenic region, and 5% of the promoter.
The promoter regions covered 54% and 52% of total sequence reads in H3K4me3 and H3K9/14ac, respectively.
The pair-wise colocalization analysis between 2 histone modifications gave poor correlation coefficients in the H3K4me1-H3K4me3 pair (r = 0.14) and H3K4me1-H3K9/14ac pair (r = 0.19) but good coincidence in the H3K4me3-H3K9/14ac pair (r = 0.86) when the Pearson&#39;s correlation coefficients were calculated with normalized tag counts detected within 1 kb upstream and downstream of TSSs [26].
The co-occupancy of FAIRE elements with histone modifications is examined in Fig.1, where it is clear that the promoters were the most abundant regulatory elements.
Two active histone modifications, H3K4me3 and H3K9/14ac, were highly enriched within ± 1 kb from TSSs (Fig.1A), and a comparative analysis showed that the FAIRE regulatory elements associated with histone modifications were positioned at promoters (Fig.1D).
Such epigenetic relationship between FAIRE and active histone modification marks has also been demonstrated in various cell types [22, 23, 34-36].
As exemplified in Fig.1E, H3K4me1 was differentially positioned, and the distribution of FAIRE-H3K4me1 sites was away from promoter regions, meaning that FAIRE-H3K4me1 sites might be distal regulatory elements, such as enhancers [37].
The level of gene expression is also modulated, depending on the degree and position of various kinds of histone modifications.
As shown in Fig.2A-2C, genes with FAIRE sites carrying H3K4me3 and H3K9/14ac had relatively high expression levels compared to those with FAIRE-H3K4me1.
Some genes related with breast cancer were up-regulated and showed high levels of H3K9/14ac in our previous study [26].
We therefore examined 68 genes with FAIRE-H3K9/14ac sites with the DAVID GO analysis tool.
These genes were significantly related with cell cycle, apoptosis, DNA damage, and signaling pathways, reflecting that their in vivo functions are essential for cell survival and proliferation.
Many of the regulatory sites associated with histone modifications contained transcription factor binding motifs (Fig.3A and 3B).
CTCF binding sites, commonly found in FAIRE-H3K4me1, FAIRE-H3K4me3, and FAIRE-H3K9/14ac, are related to a function of insulators and involved in high-order chromatin structure [38], and DNA demethylation is also known to be coincident with FAIRE-related open chromatin [36, 39].
Using computational motif analysis coupled with ChIP assay, Waki et al.
[36] demonstrated that enrichment of a binding motif for nuclear family I (NFI) transcription factors was highly associated with adipocyte-specific FAIRE signals as well as active histone modifications, like H3K4me3 and H3K27ac, providing a global view of cell type-specific regulatory elements in the genome and an identification of transcriptional regulators of adipocyte differentiation [35].
Such selective activities of regulatory elements were also reported by monitoring the chromatin structure at FOXA1-bound enhancers defined by FAIRE [26].
This evidence supports the possibility that open chromatin structures are subject to be bound by many transcription factors and that histone modifications function as markers for these factors to be recruited.
In conclusion, our results suggest that genomic regions defined by FAIRE in breast cancer cells should be highly associated with active histone modifications, such as H3K4me1, H3K4me3, and H3K9/14ac, and play a crucial role in controlling gene expression programs.
This analysis will provide an understanding of epigenetic regulatory mechanisms with open chromatin in breast cancer cells.
Chromatin structure in genomic regions defined by formaldehyde-assisted isolation of regulatory elements (FAIRE) in MCF7.
(A) Profile of FAIRE and histone modifications (H3K4me1, H3K4me3, H3K9/14ac, and H3K27me3) on a chromosome scale (chromosome 8, 11, and 12).
The peaks denoted by spots were plotted by k-clustering using seqMiner program.
(B) Histone modification peak profiles were analyzed, centered on FAIRE signals.
The Y axis represents the normalized sum of peak counts, multiplied by 10,000.
(C) The overlapping of each FAIRE-histone modification was estimated.
(D) FAIRE-histone modifications ranging from -5 kb to 5 kb of transcription start sites (TSSs) were plotted.
(E) Chromatin state at CCND1 locus.
The box indicates ±3 kb region from TSS.
Formaldehyde-assisted isolation of regulatory elements (FAIRE)-histone modifications and gene expression profile.
(A) Gene expression level and histone modification enrichment.
Inserts are Pearson correlation coefficients (r).
(B) Gene expression level and histone H3K4me3 enrichment.
(C) Gene expression level and histone H3K9/14ac enrichment.
(D) Gene ontology (GO) analysis for top 30% ranked expressed genes in H3K9/14ac enrichment.
(E) Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway analysis for top 30% ranked expressed genes in H3K9/14ac enrichment.
MAPK, mitogen-activated protein kinase.
Motif analysis for formaldehyde-assisted isolation of regulatory elements (FAIRE)-related genomic regions marked by H3K4me1, H3K4me3, and H3K9/14ac.
(A) Motif analysis for FAIRE marked by H3K4me3 and H3K9/14ac.
(B) Motif analysis for FAIRE marked by H3K4me1.
(C) Genomic distribution of motifs identified from FAIRE-H3K4me1.
(D) Genomic distribution of motifs identified from FAIRE-H3K4me3.
(E) Genomic distribution of motifs identified from FAIRE-H3K9/14ac.
N.D., not determined.
aUpstream regions from transcription start sites.
Title: The number of peaks of formaldehyde-assisted isolation of regulatory elements (FAIRE) and histone modifications analyzed in this study
Title: Systems Biological Approaches Reveal Non-additive Responses and Multiple Crosstalk Mechanisms between TLR and GPCR Signaling

A variety of ligands differ in their capacity to bind the receptor, elicit gene expression, and modulate physiological responses.
Such receptors include Toll-like receptors (TLRs), which recognize various patterns of pathogens and lead to primary innate immune activation against invaders, and G-protein coupled receptors (GPCRs), whose interaction with their cognate ligands activates heterotrimeric G proteins and regulates specific downstream effectors, including immuno-stimulating molecules.
Once TLRs are activated, they lead to the expression of hundreds of genes together and bridge the arm of innate and adaptive immune responses.
We characterized the gene expression profile of Toll-like receptor 4 (TLR4) in RAW 264.7 cells when it bound with its ligand, 2-keto-3-deoxyoctonate (KDO), the active part of lipopolysaccharide.
In addition, to determine the network communications among the TLR, Janus kinase (JAK)/signal transducer and activator of transcription (STAT), and GPCR, we tested RAW 264.7 cells with KDO, interferon-β, or cAMP analog 8-Br.
The ligands were also administered as a pair of double and triple combinations.
Cells perceive changes in the extracellular environment and create appropriate responses through a complex network of interwoven signaling cascades.
For example, lipopolysaccharide (LPS) causes severe inflammatory responses by binding to and signaling through Toll-like receptor 4 (TLR4) on the cell surface, and events downstream of TLR stimulation are extremely dynamic [1, 2].
The inner core regions of all LPS molecules contain at least one 2-keto-3-deoxyoctonic acid (KDO) residue, which possesses endotoxin toxicity that is nearly equivalent to LPS itself [3, 4].
We studied how KDO-mediated signal transduction through TLR4 could be modulated by the presence of interferon-β (IFN-β) and/or 8-bromoadenosine-3&#39;,5&#39;-cyclic monophosphate (8-Br).
IFN-β uses the Janus kinase (JAK)/signal transducer and activator of transcription (STAT) pathway, and 8-Br is a cell-permeable analog of cAMP that is regulated by Gαs- or Gαi-coupled G protein-coupled receptor (GPCR).
In response to LPS, TLR4 activates the MyD88-independent pathway to produce tumor necrosis factor α (TNFα) [5], whereas 8-Br activates cyclic AMP response element-binding protein (CREB), which binds to cAMP-responsive element (CRE) sequences, 5&#39;-TGACGTCA-3&#39;, and is also a substrate for various cellular kinases [6-10].
The rationale behind the present investigation was to understand the pathway interactions among KDO, IFN-β, and 8-Br and how each of these pathways modulates the signaling from others.
If two separate pathways triggered by two distinct ligands regulate the transcription of the same gene, then concomitant stimulation with both of the ligands should result in an additive response equivalent to the sum of the separate responses [11].
In contrast, if the pathways intersect or modulate each other, concomitant addition of the ligands would yield a non-additive response that is either greater than or less than the expected additive response.
Title: Cells, reagents, and RNA preparation

RAW 264.7 macrophages were stimulated with KDO (100 ng/mL), IFN-β (300 pM), and/or 8-Br (100 µM).
These ligands were applied individually or in combinations containing KDO for 60 min and 120 min.
Total RNA was extracted using TriPure (Roche, Indianapolis, IN, USA) following the manufacturer&#39;s protocol.
Each experiment was conducted in duplicate.
Title: Oligonucleotide array fabrication and annotation

Mouse oligonucleotide arrays were fabricated with 15,631 oligomers that were 65 bp or 70 bp long.
The oligomers were purchased from Operon (Huntsville, AL, USA) and Sigma-Genosys (St. Louis, MO, USA) and were inkjet-printed onto glass slides by Agilent Technologies (Santa Clara, CA, USA).
The list of genes is available through the Gene Expression Omnibus (GEO, http://www.ncbi.nlm.nih.gov/geo) under accession number GPL254.
Title: Gene expression analysis

Hybridization and the resulting analyses were performed as previously described [11-13].
Cy5-labeled cRNA (from ligand-treated cells) and Cy3-labeled cRNA (from time-matched controls) were hybridized in the array.
Dye-swap labeling was performed for each pair of samples.
The arrays were scanned with an Agilent Scanner G2505A (Agilent Technologies), and image files were extracted with background subtraction and dye normalization using Agilent G2566AA Extraction Software version A.6.1.1 (Agilent Technologies).
The microarray data used in this study were deposited into GEO (http://www.ncbi.nlm.nih.gov/geo) under the series accession number GSE11449 and platform ID GPL254.
Title: Quantitative real-time PCR (QRT-PCR)

Selected genes were subjected to real-time PCR analysis in duplicate using the 7500 Real-Time PCR System (Applied Biosystems, Foster City, CA, USA) following the protocol provided by the manufacturer.
The measurement was normalized to a β actin RNA control.
The primers of genes are as follows: TNF, F 5&#39;-TCA GCG AGG ACA GCA AGG-3&#39;, R 5&#39;-GTG AGT GAA AGG GAC AGA ACC-3&#39;; β-actin, F 5&#39;-CTT TGC AGC TCC TTC GTT GC-3&#39;, R 5&#39;-ACG ATG GAG GGG AAT ACAGC-3&#39;.
The remaining primer sequences are available upon request.
Title: Cytokine analysis

The Bio-Plex cytokine assay (Bio-Rad Laboratory, Hercules, CA, USA) was used for quantification of cytokines in tissue culture supernatants.
In the kit, an antibody directed against each cytokine is covalently coupled to a different color-coded polystyrene bead.
The conjugated beads are reacted with a sample containing a known (standard) or unknown amount of cytokines.
After unbound cytokines are removed, biotinylated detection antibodies are added to the reaction.
The complexes are detected by streptavidin-phycoerythrin, which has fluorescence characteristics.
Quantification is carried out by a specialized microtiter plate reader.
Title: Protein phosphorylation

Sodium dodecyl sulfate-polyacrylamide gel electrophoresis was performed to resolve a mixture of proteins by size.
The resolved proteins were transferred from the gel to a nitrocellulose membrane (Invitrogen, Carlsbad, CA, USA) via electroelution.
The blot was processed for detection of specific phosphoproteins with a mixture of antibodies.
All of the incubations and washes were conducted at room temperature with gentle shaking using an oscillating platform shaker.
A Molecular Dynamics Storm 860 gel and a blot analysis system (Amersham Pharmacia Biotech, Piscataway, NJ, USA) were used.
Title: Data selection, analysis, and visualization

Features that were differentially expressed in response to ligand stimulation were identified with Linear Models for Microarray Data (LIMMA, http://bioinf.wehi.edu.au/limma).
The Benjamini-Hochberg method was used to control the false discovery rate [14].
Four independent experiments were used to identify differentially expressed genes, and p &lt; 0.05 by ANOVA was deemed statistically significant.
The average log2-(treated/control) and the average difference (avg.
DIF) were hierarchically clustered to visualize the patterns of gene expression changes and the nonadditivity, respectively [15].
Only features with significant expression changes and significant nonadditive responses at one or more time points were included, respectively.
Clustering was done one way across the features with experimental conditions aligned in ligand, followed by time course orders.
Euclidean correlation coefficient and complete linkage were used as similarity metrics [16].
The hierarchical clustering program that was used was implemented in the Multiple Experiment Viewer (MeV, www.tigr.org/software/tm4/mev.html).
Title: Ingenuity pathway analysis (IPA)

Gene accession numbers, fold-changes of each treatment, and p-value were loaded into the IPA5.0 software (http://www.ingenuity.com).
The genes were grouped with their molecular functions and depicted as a network picture, with their direct and indirect relationships represented with the gene ontology by the canonical pathway analysis.
All these annotations used Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway maps, and the most significant genes were summarized in the network and as a graph.
Title: General patterns of gene signature in single-ligand studies

We first characterized gene expression profiles after treating RAW 264.7 macrophages with the individual ligands in a kinetic study (60 and 120 min).
We found that each of the three ligands, when applied singly, exhibited maximum transcriptional activation after 120 min of stimulation (as analyzed by Limma) (Fig.1A).
We further explored the expression data using k-means/medians clustering (KMC) in MeV, in which we divided the gene expression profiles into clusters in order to identify groups of genes that might participate in similar pathways.
The clusters that exhibited changes greater than 2-fold are shown in Fig.1B.
Upon single-ligand treatment, KDO and IFN-β mediated robust transcriptional upregulation, whereas 8-Br stimulation resulted in both up- and downregulation of gene expression.
The top-most functions of the genes by all ligand treatment and major networks are given in Fig.2.
Title: KDO regulated activators and repressors of immune signaling

After classifying the genes by KMC, we subcategorized them according to their functions.
KDO-regulated genes included transcription factors and genes involved in inflammation, immune responses, signal transduction, cell cycle regulation, cell proliferation, differentiation, metabolism, transport, cell adhesion, and apoptosis (Supplementary Table 1).
As expected, KDO-treated cells exhibited a gene regulation pattern similar to that in LPS-treated cells, which included upregulation of Tnf, Il-1α, Il-1β, Cxcl4, and c-Myc.
Furthermore, we also observed that KDO regulated several immediate early response genes, such as Egr1, Egr3, Gadd45α, Dusp1, and Dusp2, all of which are reportedly upregulated by LPS [11].
Our results indicated that KDO elicits transcriptional responses that are similar in gene identity and potency to LPS responses.
During the KDO-mediated transcriptional response in RAW 264.7 cells, many of the genes induced at 120 min were candidate feedback repressors of signal transduction [17].
Such repressors were specific to nuclear factor-κB (NF-κB) and Il-1 and include nuclear factor of kappa light polypeptide gene enhancer in B-cells inhibitor epsilon (Nfκbie, also known as IκBε), nuclear factor of kappa light polypeptide gene enhancer in B-cells inhibitor zeta (Nfκbiζ, also known as Iκbζ), and interleukin 1 receptor antagonist (Il-1ra).
In addition, KDO upregulated Tnfsf9, Tnfaip3, Traf1, and Fas6, indicating that KDO has the potential to elicit inflammation.
Its upregulation of Irf1 suggested that KDO also activated interferon-stimulated response elements through the MyD88-independent pathway.
Other KDO-induced genes included IFN-β, CD40, Socs5, Cish, Fosl1, Rgs16, CCR9, Ifit3, and cardiotropin 1 (Ctf1).
Among these genes, Socs5 and Cish are feedback inhibitors of the JAK/STAT pathway; both were upregulated after 120 min of stimulation.
KDO also promoted Il-10 expression after 120 min of stimulation; this gene contains 2 Creb binding sites in its promoter; so, KDO may have upregulated this gene through enhanced phosphoinositide 3-kinase (PI3K)-v-akt murine thymoma viral oncogene (AKT)-glycogen syntase kinase (GSK) pathway signaling and Creb phosphorylation.
Title: IFN-β induced gene expression of inflammatory/immune responses

Binding of IFN-β to the Type I-IFN receptor activates the JAK/STAT signaling pathway.
Phosphorylated Stat proteins regulate gene transcription by binding to conserved sequences within the promoters of IFN-induced genes.
IFN-stimulated Stat activation mediates the expression of a second family of transcription factors, the Irf proteins [18].
Irf proteins also mediate transcriptional responses to IFN stimulation.
In our study, IFN-β stimulation was associated with Stat2 and Irf1 production, and recent findings indicate that Irf proteins play critical roles in interferon regulation [19].
IFN-β induced a number of genes involved in inflammatory/immune responses, transcription, signaling, and apoptosis.
The most highly responsive genes induced by IFN-β were interferon-inducible GTPase 1 (Iigp1), Ifit1, myxovirus (influenza virus) resistance 2 (Mx2), Ifit2, Cxcl9, Cxcl10, Cxcl11, ISG15 ubiquitin-like modifier (ISG15), Socs1, CD40, and TLR3.
Secondary responsive genes included Gbp1, Gbp2, Gbp5, Il-13, and Fas.
The fold-changes in the expression of these genes are provided in Supplementary Table 2.
Title: 8-Br differentially regulated key inflammatory genes

The 8-Br-regulated genes were involved in transcription, signaling, inflammation, apoptosis, metabolism, proteolysis, transport, and cell cycle regulation (Supplementary Table 3).
Treating cells with 8-Br upregulated several immediate early transcriptional targets, including nuclear receptor subfamily 4 group A member 2 (Nr4a2), Gadd45 beta, Dusp1, Pde4b, Crem, Cebpb, Vegf, Bag3, and v-maf musculoaponeurotic fibrosarcoma oncogene family protein F (Maff).
Surprisingly, we observed that 8-Br treatment downregulated transcription of Rgs16, Tnf, Cxcl4, Ptger4, Tnfsf9, c-Myc, and Map3k8 by more than 2-fold (Fig.3A).
8-Br can influence the mitogen-activated protein kianse (MAPK) pathway by downregulating Mapk3k8.
Map3k8 activates the kinases p38 and JNK and the transcription factors NF-κB and NF-AT, although its specialized role is to activate the extracellular signal-regulated protein kinase (ERK) pathway downstream of most TLR signals [20-23].
We observed 8-Br-mediated downregulation of 2 proto-oncogenes, c-Myc and Map3k8, and a cell cycle gene, Gadd45α (Supplementary Table 3), which emphasizes that 8-Br cAMP may also have important inhibitory roles in cancer cell proliferation [24, 25].
We next hypothesized that the co-downregulated genes may have common transcription factor binding sites; therefore, we examined their promoter regions using TOUCAN (http://homes.esat.kuleuven.be/~saerts/software/toucan.php) and analyzed the genes downregulated by 8-Br using MotifMogul (http://xerad.systemsbiology.net/MotifMogulServer).
We found AP1 and/or NF-κB subunit binding sites in the upstream regions of all of the downregulated genes, which adds to evidence from previous studies [26, 27].
Among these genes were several classical targets of NF-κB (for example, Tnf) and AP-1 (for example, c-Myc and Gadd45α); these genes are highlighted with arrows in Fig.3A and shown in greater detail in Fig.3B.
Since protein kinase A (PKA) activation by cAMP suppresses the MAPK cascade through phosphorylation of Raf [28], we can not exclude the possibility that elevated cAMP may, in part, reduce the transcription of the targeted genes by modulating protein binding to AP-1 sites, by inhibiting the ERK pathway through Raf, and/or by specifically affecting NF-κB subunits.
Importantly, Il-1β was up-regulated quickly, whereas Il-1α and Il-10 were not affected prior to 120 min of stimulation.
This differential mRNA regulation of genes that are all involved in inflammation suggested that cAMP utilized more than one distinct mechanism to alter the expression of specific sets of genes.
Title: Double-ligand studies: non-additive responses

To understand how the presence of one or more ligands modified the transcriptional responses to the other ligands, we evaluated features that exhibited non-additive responses during the double-ligand studies and grouped them into three categories: significant responses 1) to both ligands; 2) to either ligand A or ligand B; 3) to neither ligand (Table 1).
A total of 433 features showed non-additive responses above threshold levels in at least one of the double-ligand studies.
In all of the double-ligand studies involving KDO, we found differences in gene transcription that were not present when the ligands were applied individually.
Applying KDO together with IFN-β augmented IFN-β-mediated transcriptional responses.
In contrast, when KDO was combined with 8-Br, each ligand abrogated the single-ligand response of the other.
Title: KDO enhances IFN-β signaling

KDO enhanced IFN-β-dependent signaling pathways in a time-dependent manner.
Since TLR4 activation can also produce IFN-β, the KDO-mediated enhancement suggested cooperation between exogenously applied and endogenously produced IFN-β (Supplementary Table 4).
Several IFN-β and KDO primary responsive genes were upregulated, including genes that activate the immune system.
In some cases, the IFN-β gene signature dominated over the KDO gene signature.
For example, IFN-β inhibited two KDO primary responsive genes, Il-1ra and Nfκbζ, but KDO did not change the expression of any of the interferon-responsive primary genes.
The expression levels of select genes were validated by using QRT-PCR (Fig.4).
Cytokine assays, conducted following a combination of LPS and IFN-β treatment, also demonstrated non-additive enhancement of Il-6, Il-10, RANTES, and TNF protein levels (Fig.5A).
This increase could be due to synergistic production of transcription factors in response to these ligands, apart from their classical transcription factors.
It is likely that the rapid, greater-than-2-fold increase in Socs1 and Cish production may have contributed to the attenuated expression of the KDO secondary responsive genes [29, 30].
Il-10 is an anti-inflammatory cytokine regulated through the PI3K-AKT-GSK-CREB pathway [31, 32], and we noted its expression after 120 min of stimulation, suggesting that the PI3K pathway may have been activated after 60 min of stimulation.
KDO- and IFN-β-treated cells exhibited upregulation of many interferon-regulatory factors (IRFs), including Irf1, Irf4, and Irf8.
Irf4 inhibits the expression of MyD88-dependent IRF genes by competing with IRF5 for binding to the same region of MyD88 [33].
Combined KDO and IFN treatment also upregulated genes with IRF3 binding sites in their promoters; i.e., Ifit1, Ifit2, Ifit3, Isg15, and Gbp1.
Our gene profiling results suggested that IRF3 is sufficient to induce antiviral responses at the initial activation time point, consistent with an earlier report that demonstrated a role for IRF3-dependent genes in establishing antiviral responses in Jurkat cells [34].
Title: KDO and IFN-β regulate gamma activated sequence (GAS) genes together

Interestingly, two of the genes that exhibited non-additive responses are produced only by IFN-γ GAS elements: Irf8, previously known as interferon consensus sequence-binding protein 1 (ICSBP), and transporter associated with antigen processing-1 (Tap1).
Upregulation of GAS-driven genes requires the Stat1 homodimer exclusively but not NF-κB; they are both produced by IFN-γ.
Interestingly, in our study, we did not observe elevated expression of IFN-γ or STAT1 mRNA, but we still observed the production of GAS-driven genes.
Two phosphorylation sites, Tyr-701 and Ser-727, differentially regulate Stat1, and phosphorylation of both sites is required for maximal transcriptional activity [35, 36].
A previous study demonstrated that Stat1 is phosphorylated on both Tyr-701 and Ser-727 in LPS-treated cells after 2 h [37].
We surmise that this enhanced phosphorylation of STAT1 may be mediated by KDO and IFN-β.
However, when we conducted STAT1 phosphoprotein measurements with KDO and IFN-β in combination, we observed a 3-fold upregulation in phosphorylated STAT1 after 30 min (Fig.5B).
This increase in phosphorylated STAT1 during double-ligand stimulation was lower than the increase in phosphorylation of STAT1 induced by IFN-β alone (10-fold).
Our previous studies with LPS- and IFN-γ-treated RAW 264.7 cells also showed non-additive increases in Irf8 and Tap1 gene expression [11].
This elevated GAS gene expression definitively indicated that KDO and IFN-β activate GAS promoters, either directly or indirectly.
Title: KDO abrogated the anti-inflammatory properties of 8-Br

Supplementary Table 5 shows the genes regulated upon double-ligand treatment with KDO and 8-Br.
Importantly, combined KDO and 8-Br application increased the transcription of Tnf but did not affect Ccl4, Map3k8, or c-Myc (Fig.3A).
Since 8-Br downregulated these genes during the single-ligand application studies, these results suggested that the KDO-induced pathway attenuated these 8-Br-induced changes.
We classified the direct targets of CREB (major TF induced by cAMP analogs) and NF-κB (direct target of KDO) to see which of the major transcription factors are affected (Table 2).
For CREB, we searched in the website developed by Marc Montminy, which takes advantage of CHIP-on-chip data (http://natural.salk.edu/CREB).
For potent NF-κB sites, we looked in the website http://www.bu.edu/nf-kb.
Surprisingly, both of these transcription factors are not affected by double ligand treatment of KDO and 8-Br, reinsisting that KDO directly inhibits 8-Br-induced changes.
CREB can be induced to a certain extent through the ERK pathway by KDO; however, such a claim remains to be proven in the future studies.
This approach further raises a question on by which mechanism, if not CREB and NF-κB, KDO attenuates 8-Br induced changes, which also remains to be tested.
One possible explanation for this phenomenon is that KDO stimulation rapidly downregulated the 8-Br-induced increase in intracellular cAMP through the action of phosphodiesterases (PDEs) that transform cAMP into inactive AMP [38].
In support of this idea, LPS stimulation of mouse peripheral leukocytes induced Pde4b mRNA accumulation and increased PDE4 activity [39].
This increase in PDE4 activity leads to a decrease in cAMP, thus removing the cAMP constraint and allowing for full induction of Tnf mRNA.
PKA phosphorylation inhibits Raf1 directly [40, 41], and this inhibition occurs downstream of Ras [42, 43] to inhibit Ras-dependent signaling to ERK [44].
Hence, Raf can be the crosstalk point between the KDO and 8-Br signal transduction pathways.
Title: KDO and IFN-β suppressed the effects of 8-Br in triple-ligand studies

Supplementary Table 6 shows the transcriptional responses in RAW 264.7 cells upon triple-ligand stimulation.
We observed the following effects: 1) upregulation of the primary and secondary KDO- and IFN-β-responsive genes; 2) production of various inflammatory cytokines and their receptors (Il-1a, Il-1b, Il-6, Il-20, Il-17rb, Il-15, Il-18, Il-13, Il-4ra, Il-10ra, and Tnf) and the anti-inflammatory cytokine Il-10; 3) induction of apoptosis-inducing transcription factors (Fosl-1, Atf3, Egr1, Egr2, Fos-b, Jun dimerization protein [Jdp], and Irf14); and 4) up-regulation of multiple G-protein-coupled receptors, like leukotriene B4 receptor1 (Ltb4r1), complement component 5a receptor 1 (C5ar1), chemokine (c-c) receptor 1 (Ccr1), Gpr132, Gpr65, and calcitonin receptor-like (Calcr1).
In these triple-ligand studies, the transcription factors exerting the greatest regulatory control over gene expression were NF-κB, Ap1, Nfat, Stat2, and Stat3.
Our data demonstrated that concomitant stimulation with all three ligands induced KDO and IFN-β primary and secondary responsive genes, but it repressed 8-Br-induced transcriptional changes.
Title: Triple-ligand stimulation revealed Il-6 gene induction does not require de novo synthesis of NF-κBζ

We observed an interesting transcriptional change in gene clusters (Fig.6).
NF-κBζ is weakly expressed in cells but induced to a marked extent upon LPS stimulation and regulates a set of gene expression, including Il-6 [45].
However, a recent study demonstrated that Il-6 promoter can also be activated by unknown molecules acting far from the transcriptional active site by cis-regulatory molecules [46].
We observed the same phenomenon when we stimulated RAW 264.7 cells with KDO.
NF-κBζ was upregulated by KDO at 60 min, and Il-6 was not induced by KDO till 120 min, which is in line with a previous observation [45].
This observation is very novel to be explained.
Triple-ligand studies showed upregulation of Il-6 as early as 60 min (about 6-fold).
This indicates that though KDO-induced pathways need de novo synthesis of NF-κBζ for Il-6 production, it does not require de novo synthesis of NF-κBζ for the induction of Il-6 when combined with IFN-β and 8-Br.
We then looked for the possible ligand that could have mediated this and found that KDO and 8-Br never produced Il-6 on single-ligand stimulation.
In that case, the only remaining candidate is IFN-β.
IFN-β upregulated Il-6 in our experiment at 120 min (TLR3 can produce Il-6 due to the late phase of NF-κB activation).
IFN-β also increased mRNAs of Atf3, Ccl4, and Map3k8 at 120 min.
It is of interest to know what pathway regulates these genes in future studies, since most of them were reported to be regulated by a MyD88-dependent pathway during TLR signaling.
Either the presence of a putative STAT1 site is sufficient to induce the MyD88-dependent genes or IFN-β might have crosstalk with the MAPK pathway.
In RAW cells and primary macrophages, LPS-induced signaling is mainly passed through the activation of the NF-κB/IκB kinase (IKK) pathways.
The p50 and RelA complex mediates the majority of genes induced by LPS.
In our studies, many induced genes appeared to be activated as a primary consequence of initial NF-κB activation.
Although the induction of the genes was through the NF-κB pathway, few genes were differentially expressed when we combined the ligands.
We have restricted our studies to 2 h; hence, we can not emphasize on the secondary sequences of NF-κB, but we observed that most of the induced genes have putative or known AP-1 or IRF-3 sites.
The gene expression in dual treatment of IFN-β and KDO was specifically mediated by STAT proteins and NF-κB.
Also, gene transcription during macrophage activation through TLR4 and the elevation of IFN-β simultaneously resulted in alterations in the chromatin structure of gene promoters that further influenced transcription initiation events.
Combining IFN-β and KDO induced the IFN-β secondary responsive genes Gbp1, Gbp2, and Gbp5 by more than 4-fold, but it attenuated several KDO secondary responsive genes, including Socs5, Cish, Fosl, Rgs16, Cxcr9, Ifit3, and Ctf1.
In contrast, dual KDO and IFN-β treatment enhanced the expression of several transcription factors after 60 and 120 min of stimulation, most notably Atf3.
Atf3 is a stress-inducible member of the Atf/Creb transcription factor family [47], and according to a recent systems biological approach, Atf3 negatively regulates TLR4-induced Il-6 and Il-12 transcription [48].
As evident from our QRT-PCR results, IFN-β never regulated MyD88-independent genes except Il-10.
However, IFN-β treatment during TLR4 signaling increased Rantes, Il-10, and Tnf levels.
It is interesting to note that that there should be a competition for the common signaling repressors and activators of the inflammatory ligand TNF and anti-inflammatory ligand Il-10 at approximately 4 h of signaling.
It is also worthwhile to note that if TNF pathways dominate, it should lead to increased Il-10-repressed genes and direct the cells into the apoptosis pathway.
In addition to this effect, we observed upregulation but not enhanced levels of NF-κB p105, implicating this subunit in KDO and IFN-β responses not only through IκB-NF-κB interaction but also through transcriptional regulation.
The elevated KDO and IFN-β primary and secondary responsive genes were not unexpected, considering that we observed higher NF-κB signaling and endogenous production of IFN-β under double-ligand stimulation.
This suggests that there exists some crosstalk and pathway modulation in a positive way that increased the levels of gene expression.
Macrophages, when infected with pathogenic Gram-negative bacteria, lead to the induction of about 1,000 genes, and almost all of these genes were reported to be induced only though the TLR4 signaling cascade.
Excessive production of cytokines or inflammatory agents is always associated with sepsis and shock.
The data presented in our microarray also show that KDO and LPS induce identical gene expression profiles.
This suggests that the contribution of other family members of TLR is minor, and TLR4 on the whole contributes to KDO-induced gene expression changes.
Further, most of these gene responses were mediated by the MyD88-dependent pathway, which is essential for the optimal induction of many well-established cytokines, like Il-1, TNF, Il-10, Il-12, and Il-6.
On the other hand, in our microarray studies, we observed that Tnf and other inflammatory genes were downregulated by 8-Br, whereas upregulated Il-10 levels.
However, in double-ligand treatment, KDO abrogated all the gene responses of 8-Br.
cAMP-dependent and -independent signaling mechanisms may be involved in mediating the gene regulation during double-ligand stimulation with KDO and 8-Br.
Another observation was the differential regulation of mRNAs of Il-1β, Il-1α, and Tnf by KDO and 8-Br.
Ccl3, Ccl4, Il-1β, Tnf-α, and Gadd45β are the genes whose promoters have been proven to be regulated by RelA [49].
In our studies, all these genes were found to be upregulated at 60 min of KDO treatment, which is consistent with a previous report [50].
However, 8-Br increased Il-1β levels, but it did not increase Tnf levels.
Both genes are regulated through the MyD88-dependent signaling.
On the other hand, there was no change in Il-1α levels.
In double-ligand stimulation, all these changes were reversed.
We tried to find answers through the promoter and transcription factor search analysis.
Transcription factors evolved to regulate the gene expression, and they have the ability to modify the expression of a given gene.
Since Tnf is downregulated by 8-Br at 60 min itself, it should be happening either at the promoter level or at the primary pathway level.
Although Tnf has AP1 binding sites, it is not sufficient to induce Tnf mRNA, whereas NF-κB is necessary for its upregulation.
8-Br should have affected the translocation of p65/p50 subunit to the nucleus.
Il-1β is highly expressed in activated macrophages and monocytes, and LPS is a well-established stimulus for its induction.
Il-1β has one or more CREB binding sites in the promoter.
Since 8-Br activates CREB, the presence of one or more CREB sites seems to be sufficient for its induction.
On the other hand, Il-1α mRNA level was not changed.
With the information available on its regulation, it seems that the AP1 site is necessary for its induction.
8-Br-induced pathways may have either directly or indirectly affected AP1 translocation to the nucleus.
In this context, to support our notion, we observed the downregulation of some AP1 genes.
8-Br-induced pathways modulated the MAPK pathway, which must have happened as a dual channel.
First, 8-Br may have inhibited the pathway upstream of AP1.
Second, it should have inhibited the translocation of the specific subunits of AP1 into the nucleus.
Gene regulation studies of 8-Br suggest the following mechanism of the pathway activation/repression: 1) 8-Br affected NF-κB activity directly; 2) 8-Br-induced CREB is sufficient to induce Il-1β expression; and 3) 8-Br affected the different subunits of AP1 specifically, and at the same time, it inhibited the pathway of MAPK upstream of AP1.
PDEs are enzymes that catalyze the hydrolysis of cAMP and cGMP, inactivating these second messengers [51].
PDE consists of 11 families, and amongst them four, including PDE4, PDE7, and PDE8, are cAMP-specific [52].
PDE4 isoforms are major inactivators of cAMP signaling [53].
In our studies, stimulation of TLR caused a major upregulation of PDE4B but not the paralogs PDE4A or PDE4D.
Ablation of PDE4B impacted LPS signaling, and TNF-α mRNA and protein were decreased by &gt;50% in PDE4B-/- but not in PDE4A-/- or PDE4D-/- macrophages [54].
This indicates the powerful role of PDE4B in macrophages, by inhibiting cAMP and its critical role in LPS signaling.
Another possible explanation for this pathway modulation is the interaction between NF-κB and PKA.
Since the catalytic subunit of PKA works as a complex with NF-κB/IκB in the inactive state [55], stimuli that degrade IκB also activate the associated PKA catalytic subunit, which promotes the phosphorylation of p65 at Ser 276 and increases the transcriptional activity of NF-κB.
Two GPCR ligands, isoproterenol and prostaglandin E2, induce cAMP through the activation of Gαs and adenylate cyclase.
cAMP production inhibits pro-inflammatory mediators, including TNF and nitric oxide in RAW 264.7 cells [56, 57].
Apart from this, multiple signaling pathways or mechanisms are likely to be involved in mediating gene responses to KDO and 8-Br.
In some cases, an increase in cAMP level by pituitary adenylate cyclase blocks IκB degradation, p65/Rel translocation, and/or the rearrangement of CREB/Jun/CREB-binding protein (CBP) complexes [58].
In other cases, an increase in cAMP is directly proportional to the decrease in MAPK/ERK kinase (MEK) kinase pathways [59], and transactivating properties of PKA are reported to be involved in NF-κB activation [60].
After 60 min, triple-ligand treatment upregulated many transcription factors, such as Nr4a2, Crem, Atf3, Irf8, Nfkbp100, Egr2, Egr1, Cish, Stat3, Stat2, Irf1, and FosB.
After 120 min of stimulation, we observed upregulated zinc finger and BTB domain-containing 32 (Zbtb32), Fosl1, RelB, Irf7, LPS-induced TNF-α factor (Litaf), and Jundm2.
Although the regulatory mechanisms of these genes are complex, they all culminate in the production of various pro-inflammatory cytokines.
NIH3T3 cells, when stimulated with the cAMP analog forskolin, activate p38 by PKA, which in turn phosphorylates CREB through RSK2.
However, this process is delayed in comparison with the direct phosphorylation of CREB by PKA [61].
This study shows that there exist crosstalk mechanisms between PKA and p38, thus allowing the cells to prolong the incoming signal.
Both LPS and the cAMP analog 8-Br activate p38, like forskolin, forming the crosstalk points between GPCR and Toll-like receptor.
This mechanism would have partly contributed to the observed dominant gene signature of KDO in triple-ligand studies.
We also noted that there exist substantial differences in the gene expression profiles of RAW 264.7 cells and primary macrophages.
For example, TNF alpha is produced highly in RAW 264.7 cells than in macrophages, and there are quantitative and temporal differences between these cells.
The KDO-induced gene signature dominated over IFN-β and the 8-Br-induced gene signature.
In addition, we also explained the genes that have the ability to act as positive (Tnf, Il-1) [62, 63] and negative feedback (Il-1ra) [64] regulators of KDO signaling.
Excessive activation of cytokines is associated with septic shock and clinical consequences of inflammatory disorders.
It is also of interest that many genes induced by 8-Br have the potential to act as anti-inflammatory actions.
Thus, our studies prove that elevating intracellular cAMP levels during the times of immunological stress might ameliorate the shock-induced mortality.
Taken together, our study identified several target genes that are activated/repressed by KDO, IFN-β, and/or 8-Br in RAW 264.7 cells.
Based on these data, we also proposed regulatory signaling networks that could be responsible for the crosstalk between Toll-like receptors and non-Toll-like receptors, such as GPCRs.
These findings not only highlight the decisive regulation of various genes by three important ligands but also add depth to our broad understanding of biological signaling connections between complex systems of functionally interacting macromolecules.
Gene responses in macrophages to LPS vary between different methylation patterns [65], sub-clones of RAW 264.7 cells [66], chromatin structure of gene promoters, and chromatin remodeling events [67].
To define the action of a given ligand, it is necessary to rule out these options and particularly emphasize on the true effects of a ligand.
Though our studies form the basis for the novel regulatory pathways of gene induction during macrophage action, future studies are warranted to construct a signaling map.
The transcriptional signature of RAW 264.7 cells after stimulation with 2-keto-3-deoxyoctonate (KDO), interferon-β (IFN-β), and 8-bromoadenosine-3&#39;,5&#39;-cyclic monophosphate (8-Br).
(A) The bars in the diagram represent the numbers of differentially expressed genes at 60 min and 120 min of stimulation, as assessed with Limma.
(B) Hierarchical clustering of responses to 60 min and 120 min of stimulation was conducted using K-means clustering and visualized by MeV software.
The cluster shown represents those macrophage genes that were significantly regulated by KDO, IFN-β, or 8-Br treatment.
Red squares represent genes that were upregulated, black squares indicate no change, and green squares indicate downregulated genes.
Top functions and tumor necrosis factor (TNF) networks.
(A) The top functions for all treatments.
As noted, various treatments had various effects on their respective pathways and functions, with 2-keto-3-deoxyoctonate (KDO) the most active ligand, followed by interferon-β (IFN-β) and 8-bromoadenosine-3&#39;,5&#39;-cyclic monophosphate (8-Br).
(B) TNF networks.
Ingenuity pathway analysis (IPA) pictures show the down- and upregulation of the ligand-induced gene expression changes.
Dual-ligand analysis of 2-keto-3-deoxyoctonate (KDO) and 8-bromoadenosine-3&#39;,5&#39;-cyclic monophosphate (8-Br).
(A) Clustering of KDO and 8-Br-induced gene expression changes.
(B) MotifMogul picture showing the promoter analysis of Tnf, c-Myc, and Gadd45α.
Quantitative real-time PCR validation of genes regulated by 2-keto-3-deoxyoctonate (KDO), interferon-β (IFN-β), or KDO + IFN-β.
KDO enhanced the expression of RANTES and interleukin 10 (IL-10) together with IFN-β.
G-CSF, granulocyte-colony stimulating factor; TNF, tumor necrosis factor.
Dual-ligand analysis of 2-keto-3-deoxyoctonate (KDO) and interferon-β (IFN-β).
(A) Cytokine analysis of non-additive genes in KDO and IFN-β stimulation.
(B) Stat1 phosphoprotein analysis upon KDO and IFN-β stimulation.
(C) KDO and IFN-β signaling mechanisms.
GAS, gamma activated sequence; G-CSF, granulocyte-colony stimulating factor; IL, interleukin; IRF3, interferon-regulatory factor 3; ISRE, interferon-stimulated response element; JAK/TYK, Janus kinase/tyrosine kinase; STAT, signal transducer and activator of transcription; TLR4, Toll-like receptor 4; TNF, tumor necrosis factor.
Regulation of NF-κBζ and interleukin 6 (Il-6) by all ligands.
(A) NF-κBζ, which is highlighted in black arrow, was regulated by 2-keto-3-deoxyoctonate (KDO) as well as by triple-ligand stimulation in RAW 264.7 cells.
(B) Il-6 was regulated by interferon-β (IFN-β) at 120 min, but triple ligands upregulated Il-6 robustly as early as 60 min itself.
8-Br, 8-bromoadenosine-3&#39;,5&#39;-cyclic monophosphate.
Title: Number of features showing non-additive responses in double-ligand studies with KDO, IFN-β, and 8-Br

Title: The potent CREB and NF-κB sites were searched, and their presence or absence is listed
Title: Discovery and Evaluation of Polymorphisms in the AKT2 and AKT3 Promoter Regions for Risk of Korean Lung Cancer

AKT is a signal transduction protein that plays a central role in the tumorigenesis.
There are 3 mammalian isoforms of this serine/threonine protein kinase-AKT1, AKT2, and AKT3-showing a broad tissue distribution.
We first discovered 2 novel polymorphisms (AKT2 -9826 C/G and AKT3 -811 A/G), and we confirmed 6 known polymorphisms (AKT2 -9473 C/T, AKT2 -9151 C/T, AKT2 -9025 C/T, AKT2 -8618G/A, AKT3 -675 A/-, and AKT3 -244 C/T) of the AKT2 and AKT3 promoter region in 24 blood samples of Korean lung cancer patients using direct sequencing.
To evaluate the role of AKT2 and AKT3 polymorphisms in the risk of Korean lung cancer, genotypes of the AKT2 and AKT3 polymorphisms (AKT2 -9826 C/G, AKT2 -9473 C/T, AKT2 -9151 C/T, AKT2 -9025 C/T, AKT2 -8618G/A, and AKT3 -675 A/-) were determined in 360 lung cancer patients and 360 normal controls.
Statistical analyses revealed that the genotypes and haplotypes in the AKT2 and AKT3 promoter regions were not significantly associated with the risk of lung cancer in the Korean population.
These results suggest that polymorphisms of the AKT2 and AKT3 promoter regions do not contribute to the genetic susceptibility to lung cancer in the Korean population.
Lung cancer has been the leading cause of cancer-related deaths in Korea, and its incidence continues to rise [1].
Despite therapeutic advances, the overall 5-year survival remains only 15% [2].
Lung cancer is a multicellular and multistage process that involves a number of genetic changes in oncogenes and tumor suppressor genes.
Nevertheless, the prognosis of lung cancer has remained poor despite innovations in diagnostic testing and surgical technique and the development of new chemotherapeutic agents.
Recently introduced targeted agents show different responses according to histological subtype, and the efficiency of treatment modalities for lung cancer depends on the time of diagnosis [3].
Therefore, there is a great need for rapid and efficient early detection methods.
For developing improved molecular biomarkers for the early detection and prediction of response to chemotherapy, it is important to identify genetic alterations specific to each subtype of lung cancer.
Single-nucleotide polymorphisms (SNPs) represent an important class of genetic variations and affect an individual&#39;s susceptibility to disease in certain circumstances [4-6].
Activation of the intracellular survival signal transduction protein AKT, known as protein kinase B, has been proposed as a central signaling event in carcinogenesis and has been shown in experimental models to confer resistance to chemotherapy and radiation [7, 8].
AKT is activated by phosphatidylinositol 3 phosphates, the products of phosphoinositide 3-kinase (PI3K) activity [9].
Additionally, AKT activity is commonly dysregulated in a variety of human tumors because of frequent inactivation of the PTEN tumor suppressor gene, which negatively regulates phosphatidylinositol 3 phosphate levels, and alterations of this gene have been identified in various cancers, including lung cancer [10, 11].
To date, 3 human isoforms of AKT have been identified, AKT1, AKT2, and AKT3, which are expressed in lung, breast, colon, and prostate tumor tissues [12].
Somatic mutations of AKTs have been reported in endometrial carcinomas, hypoglycemia, and breast cancer [13, 14].
However, somatic mutations in AKT families are rare in lung cancer [15, 16].
Previously, it was reported that an AKT1 polymorphism was not associated with schizophrenia [17, 18].
AKT1 and AKT2 polymorphisms have been reported to be related to survival of esophageal cancer [19], and it was reported that an AKT1 polymorphism was associated with survival in surgically resected non-small cell lung cancer [20].
However, the association between AKT2 and AKT3 polymorphisms and the risk of lung cancer has not yet been clarified.
This study discovered polymorphisms of the AKT2 and AKT3 promoter regions and their relation with the risk of lung cancer in a Korean population.
Title: Study subjects

Between August 2001 and November 2010, blood samples were collected from 720 subjects, including 360 lung cancer patients and 360 normal controls without cancer.
Lung cancer patients were recruited from the patient pool at the Genomic Research Center for Lung and Breast/Ovarian Cancer, and control subjects were randomly selected from a pool of healthy volunteers who had visited the Cardiovascular Genome Center and Genomic Research Center for Allergy and Respiratory Diseases.
Detailed information on diet, smoking status, drinking status, lifestyle, and medical history were collected by trained interviewers using a structured questionnaire.
Out of 360 cases, information was available for 352 on smoking status, 317 on stage, and 343 on cell type, while information was available for 272 on smoking status of 360 controls.
All study subjects provided written consent and were ethnic Koreans, and all participating Institutional Review Boards approved the study protocol.
Title: Preparation of genomic DNA and direct sequencing

Genomic DNA was prepared from peripheral blood samples using a Puregene blood DNA kit (Gentra, Minneapolis, MN, USA), following the manufacturer&#39;s protocol.
To identify polymorphisms in the promoter of the AKT2 and AKT3 genes, human genomic DNA was isolated from the whole blood of 24 lung cancer patients for direct sequencing, and the promoter region of the AKT2 and AKT3 genes was amplified.
Polymerase chain reaction (PCR) amplifications were performed on a PTC-225 Peltier Thermal cycler (MJ Research Inc., Waltham, MA, USA) using Ampli-TagGold (Roche, Branchburg, NJ, USA).
All amplifications were performed using 35 cycles of 30 s at 95℃, 1 min at 64℃, and 1 min at 72℃, followed by a single 10-min extension at 72℃.
PCR products were purified using the Montage PCR96 Cleanup kit (Millipore, Bedford, MA, USA) and eluted in 20 µL of nuclease-free H2O.
DNA cycle sequencing was carried out using the BigDye Terminator V 3.1 Cycle Sequencing kit (Perkin Elmer, Foster City, CA, USA).
Multiscreen SEQ 384-well filter plates were used for dye removal, and sequences were then analyzed on an Applied Biosystems 3700 (ABI, Foster City, CA, USA).
All polymorphisms and sequence alignments were analyzed using Polyphred.
Title: Genotyping

After direct sequencing of the AKT2 and AKT3 genes, we performed genotyping for the 8 polymorphisms of the AKT2 and AKT3 genes.
The genotypes of the AKT2 and AKT3 polymorphisms were screened by TaqMan fluorogenic 5&#39; nuclease assay (ABI).
The final volume of the PCR was 5 µL, containing 10 ng of genomic DNA, and 2.5 µL TaqMan Universal PCR Master Mix, with 0.13 µL of 40 × Assay Mix.
Thermal cycle conditions were as follows: 50℃ for 2 min to activate uracil N-glycosylase and prevent carryover contamination, 95℃ for 10 min to activate the DNA polymerase, and 45 cycles of 95℃ for 15 s and 60℃ for 1 min.
All PCRs were performed using 384-well plates on a Dual 384-Well GeneAmp PCR System 9700 (ABI), and the endpoint fluorescent readings were performed on an ABI PRISM 7900 HT Sequence Detection System (ABI).
Duplicate samples and negative controls were included to ensure accuracy of genotyping.
Title: Statistical analysis

Allele frequencies, genotype frequencies, and departures of genotype distributions from Hardy-Weinberg equilibrium (HWE) for each SNP were analyzed using chi-square test or Fisher&#39;s exact test.
A p-value of &lt;0.05 was considered statistically significant.
Linkage disequilibrium (LD) was tested on pairwise combinations of polymorphisms using the absolute value of the standardized measure of LD, D&#39;, calculated by Haploview version 3.2.
The haplotypes and their frequencies were estimated by Haploview version 3.2.
Genotype-specific risks were estimated as odds ratios with associated 95% confidence intervals by unconditional logistic regression (SAS Institute Inc., Cary, NC, USA) and adjusted for age, gender, and smoking status.
Table 1 shows the clinicopathological features of cases and controls.
All the clinical characteristics, except for smoking status, were not significantly different between the cases and controls.
This difference of smoking status was controlled in the multiple logistic regression analysis, adjusted for age, gender, and smoking status.
By direct sequencing of the AKT2 promoter region in 24 lung cancer patient samples, we identified 1 novel polymorphism (-9826 C/G) and 4 known polymorphisms (-9473 C/T, -9151 C/T, -9025 C/T, and -8618 G/A).
Also, we identified 1 novel polymorphism (-811 A/G) and 2 known polymorphisms (-675 A/- and -244 C/T) by direct sequencing in the AKT3 promoter region (Table 2).
The genotype distributions of the polymorphisms were in HWE.
Also, LD and haplotypes in the polymorphisms of AKT2 were calculated.
However, the LD and haplotype block in AKT3 polymorphisms were unidentified because of the low frequency of the AKT3-811 G allele and AKT3-244 T allele (2.1% and 2.2%, respectively) (Fig.1).
Accordingly, we focused on the -675 A/- polymorphism in the AKT3 promoter region.
Further analyses were then performed on the samples from 460 lung cancer patients and 460 controls.
Association of lung cancer risk with AKT polymorphisms was then analyzed, revealing no association of the polymorphisms with the risks of lung cancer (Table 3).
The association of the polymorphisms with the risk of lung cancer was further examined after stratifying the subjects according to gender and smoking status.
However, the subsequent analysis revealed no significant association.
Furthermore, the haplotypes of the AKT2 polymorphisms were not associated with the risks of lung cancer in 3 alternative models (data not shown).
V-akt murine thymoma viral oncogene homolog (AKT) is a serine/threonine protein kinase that is activated by PI3K-α and mediates PI3L signaling [21].
A major recurrent mutation (Glu17Lys) in the AKT1 gene, which encodes protein kinase B, has been identified in several solid tumors, including breast, colon, and ovarian cancers [22].
Overall, the frequency of AKT1 mutations in non-small cell lung carcinoma (NSCLC) is about 1%, and they have only been identified in squamous cell carcinoma [23].
Also, mutations in the AKT2 and AKT3 genes are rare in several tumors, including lung cancer [15].
Recently, several polymorphisms in the AKT1 gene have been reported in relation to human disease [24-26].
Kim et al.
[20] first reported that the AKT1 polymorphisms could be used as prognostic markers for patients with early-stage NSCLC.
However, there has been no study on the association between lung cancer risk and polymorphisms of AKT2 and AKT3.
In this study, we investigated the association between AKT2 and AKT3 polymorphisms and the risk of lung cancer in the Korean population.
The genotypes and haplotypes of the AKT2 and AKT3 genes were determined in 720 Koreans to evaluate the role of the AKT2 and AKT3 polymorphisms in the risk of lung cancer; however, the results revealed no significant difference between lung cancer patients and normal controls.
Also, by TFSEARCH (version 1.3) analysis of the AKT2 and AKT3 polymorphism regions, the location of these variants did not have sequences for transcription factor binding sites.
This result suggests that polymorphisms of the AKT2 and AKT3 promoter region are unlikely to play an important role in the susceptibility to lung cancer.
Polymorphisms often show ethnic variation [6, 27].
In the present study, we observed the frequencies of 5 polymorphisms in the AKT2 promoter region, and we observed frequencies of 3 polymorphisms in the AKT3 promoter region in 24 Korean populations using direct sequencing.
In the direct sequencing result, we detected no frequencies in another known polymorphisms (AKT2: rs11666690, rs11665880, rs35814223, rs34408497, rs35408040, rs10418387; AKT3: rs11588441, rs6679386, rs41267521).
The frequencies of AKT2 promoter polymorphisms showed similar frequencies in the Asian ethnic groups (Han Chinese and Japanese) of the dbSNP database (www.ncbi.nlm.nih.gov/SNP).
The frequency of the minor allele of rs35835614 in the AKT3 gene was 0.275; however, the frequency of this polymorphism has not been reported in the dbSNP database (www.ncbi.nlm.nih.gov/SNP).
A series of our previous studies showed that polymorphisms of oncogenes, tumor suppressor genes, and receptor tyrosine kinases are related to lung cancer risk [6, 28, 29].
Recently, we reported that STK11 and PIK3CA polymorphisms are not related to genetic susceptibility to lung cancer [30, 31].
The genotypes of the AKT2 and AKT3 promoter polymorphisms, determined in 720 Koreans in the present study, revealed no significant difference between lung cancer patients and normal controls.
To our best knowledge, this is the first study on the association of AKT2 and AKT3 promoter polymorphisms and the risk of lung cancer in a Korean population.
Polymorphism location in promoter region and linkage disequilibrium (LD).
Coding exons are marked by black blocks, and untranslated regions are marked by white blocks.
The first base of the translational site is denoted as nucleotide +1.
(A) AKT2 location of promoter polymorphisms and LD block.
(B) AKT3 location of promoter polymorphisms.
The LD of AKT3 polymorphisms was not identified.
aFurther analyses polymorphism sites.
Title: Characteristics of the study subjects

Title: Polymorphism confirmation of the AKT2 and AKT3 promoter region by direct sequencing

Title: Association between AKT2 and AKT3 promoter polymorphisms and the risk of lung cancer
Title: Prediction of Colorectal Cancer Risk Using a Genetic Risk Score: The Korean Cancer Prevention Study-II (KCPS-II)

Colorectal cancer (CRC) is among the leading causes of cancer deaths and can be caused by environmental factors as well as genetic factors.
Therefore, we developed a prediction model of CRC using genetic risk scores (GRS) and evaluated the effects of conventional risk factors, including family history of CRC, in combination with GRS on the risk of CRC in Koreans.
This study included 187 cases (men, 133; women, 54) and 976 controls (men, 554; women, 422).
GRS were calculated with most significantly associated single-nucleotide polymorphism with CRC through a genomewide association study.
The area under the curve (AUC) increased by 0.5% to 5.2% when either counted or weighted GRS was added to a prediction model consisting of age alone (AUC 0.687 for men, 0.598 for women) or age and family history of CRC (AUC 0.692 for men, 0.603 for women) for both men and women.
Furthermore, the risk of CRC significantly increased for individuals with a family history of CRC in the highest quartile of GRS when compared to subjects without a family history of CRC in the lowest quartile of GRS (counted GRS odds ratio [OR], 47.9; 95% confidence interval [CI], 4.9 to 471.8 for men; OR, 22.3; 95% CI, 1.4 to 344.2 for women) (weighted GRS OR, 35.9; 95% CI, 5.9 to 218.2 for men; OR, 18.1, 95% CI, 3.7 to 88.1 for women).
Our findings suggest that in Koreans, especially in Korean men, GRS improve the prediction of CRC when considered in conjunction with age and family history of CRC.
Colorectal cancer (CRC), also called colon cancer or large bowel cancer, includes cancerous growths in the colon, rectum, and appendix [1].
With 655,000 deaths worldwide per year, it is the fourth most common form of cancer in the United States (US) and the third leading cause of cancer-related deaths in the western world [1, 2].
In Korea, CRC is one of the most commonly diagnosed cancers, and its incidence is now dramatically increasing with the westernization of lifestyles [3].
According to statistics for Korea, the incidence of CRC was 9.8 per 100,000 men and 10.4 per 100,000 women from 1999-2001 [4].
These incidence rates for CRC increased to 18.2 per 100,000 men and 13.7 per 100,000 women in 2003 [5].
Given the high incidence of CRC and its significant cost to society, the ability to accurately predict the possibility of developing the disease using identifiable risk factors may help both physicians and patients prevent its occurrence [6].
Numerous studies have identified risk factors related to CRC, such as age, sex, family history of CRC, smoking, physical activity, aspirin/nonsteroidal anti-inflammatory drug (NSAID) use, vegetable intake, body mass index (BMI), alcohol consumption, and hormone replacement therapy by women [7-10].
Recently, there have been a number of studies that developed a risk score or a prediction model of certain diseases, such as coronary heart disease (CHD) and cancers, using these identified risk factors [11, 12].
However, these risk scores or prediction models have excluded genetic risk factors.
Genetic polymorphisms contributing to certain disease incidences, such as CHD, could be one type of emerging risk factor under investigation in studies generally focused on a priori selected candidate genes [13, 14].
Advances in genome technologies have made it possible to genotype and evaluate many single-nucleotide polymorphisms (SNPs) throughout the human genome to identify novel disease susceptibility genes [15].
A CRC prediction model has been developed in a previous study that estimates the probability of developing CRC, given a specific age, risk factor profile, and time period in white men and women aged 50 years and older [16].
However, genetic polymorphisms have not been included in the study.
Another prediction model, developed among middle-aged Japanese men, included conventional risk factors without genetic risk factors [17].
A recent study developed a prediction of CHD risk, aggregating information from multiple SNPs into a single genetic risk score (GRS) and determined an improvement in the prediction of incident CHD in the Atherosclerosis Risk in Communities (ARIC) study [18].
From recent studies, several SNPs that may play an important role in triggering CRC have been introduced by a genome-wide association study (GWAS) among whites, Japanese, and Chinese but not Koreans [19-22].
Moreover, prediction models that were developed recently were performed among whites and the Japanese population [16, 17].
Previous studies have shown that combining multiple loci with modest effects into a global GRS might improve the identification of persons who are at risk for common complex diseases [23-25].
Therefore, in this study, we intended to describe a GRS by aggregation of multiple SNPs contributing to CRC through a GWAS.
Furthermore, we aimed to develop a prediction model consisting of conventional risk factors as well as a genetic risk factor, such as GRS, in Koreans in the Korean Cancer Prevention Study-II (KCPS-II).
Title: Study population

The initial study population included 200,595 individuals, 20-77 years of age, who visited 16 health promotion centers nationwide from April 2004 to December 2007 in the KCPS-II.
Of these, there were 325 confirmed cases of CRC [26], but 132 cases ≥55 years of CRC onset age were excluded to obtain early-onset CRC cases.
For controls, they were recruited from the Korean Metabolic Syndrome Research Initiative study, a part of KCPS-II, in Seoul, initiated in December 2005.
A total of 9,128 individuals were recruited in 2006, and an additional 17,569 individuals were recruited in 2007.
Therefore, the total Seoul cohort included 26,697 volunteers.
Volunteers from the first round had routine health examinations at the Health Promotion Center in university hospitals between January 2006 and December 2007.
From this total, 1,004 individuals were genotyped using Affymetrix Genomewide Human SNP Array 5.0 (Affymetrix, Santa Clara, CA, USA).
However, 10 of 1,004 individuals were removed because of low genotyping call rates (&lt;95%), and 4 individuals were shown to have biological relatives; so, one member of each pair was excluded.
Eleven and 2 individuals were also excluded as a result of gender mismatches [27].
An additional 6 cases and 1 control were excluded due to missing anthropometric measurements (height, weight, BMI, waist circumstance [WC], and blood pressure [BP]) and self-reported questionnaire information (smoking status and alcohol consumption).
A detailed description of the KCPS-II study design and methods of selection of controls in this study are published elsewhere [27].
Therefore, a total of 1,163 participants (men, 687; women, 476) were included in this study: 187 cases (men, 133; women, 54) and 976 controls (men, 554; women, 422).
A written consent form was signed by all study participants, and the Institutional Review Board of Yonsei University approved the study protocol.
Title: Genotyping

DNA samples were isolated from the peripheral blood of participants and genotyped using Affymetrix Genomewide Human SNP Array 5.0 (Affymetrix Inc.) at DNA Link Inc. (Seoul, Korea).
Internal quality control (QC) measures were employed to ensure accuracy of the data.
The QC call rate (dynamic model algorithm) was ≥95%, and heterozygosity of X chromosome markers identified the gender for each sample.
Genotype calling was performed by Birdseed (v2) algorithm.
Chromosome Y was not analyzed.
A total of 1,163 individuals were genotyped via this platform in the analysis.
PLINK (v1.07) was used to estimate identity by state (IBS) over all SNPs [28].
A default set of 426,019 SNPs was used for further analysis, as recommended by Affymetrix.
In the quality assurance screening, we flagged SNPs with genotype call rates &lt; 95%, minor allele frequencies &lt; 0.01, and SNPs showing deviation from Hardy-Weinberg equilibrium (HWE) at p &lt; 0.0000001.
The final set of acceptable markers included 312,506 autosomal SNPs.
Accuracy of the genotyping was calculated by Bayesian robust linear modeling using the Mahalanobis distance (BRLMN) algorithm [29].
Title: Chemistry and anthropometric measurements

Serum, separated from peripheral venous blood, was obtained from each participant after a 12-h fast and then stored at -70℃ until analyzed.
For anthropometric measurements, WC was measured on exposed waists midway between the lower rib and the iliac crest using a measuring tape.
For difficult cases, WC was measured at 3 cm above the navel.
Weight and height were measured while participants were wearing light clothing.
BMI was calculated as weight (kg) divided by height squared (m2).
Both systolic and diastolic BP was measured after a 15-min rest.
In addition, each participant was interviewed using a structured questionnaire to collect information on smoking and alcohol consumption as well as demographic characteristics, such as age, gender, and family and past history of clinical diseases.
Cigarette smoking was classified into never smokers, ex-smokers, and current smokers.
Alcohol consumption was divided into nondrinkers and current drinkers.
Regular physical activity was tracked as either &#34;yes&#34; or &#34;no&#34;.
Title: SNP selection and GRS calculation

In the association of SNPs with CRC, the SNPs with p &lt; 10-5 in Korean men were: rs17391002 (CXCL12), rs9549448 (SOX1), rs254833 (MYO10), rs2553614 (TMEM71), rs13153032 (NSUN2), rs2288073 (FLJ30851), rs9604214 (SOX1), rs9865670 (OPA1), rs17186320 (KIAA1009), rs1509497 (RFX8), rs235428 (PHF20L1), rs9845920 (OPA1), rs9846212 (OPA1), rs6763744 (OPA1), rs4128317 (ALK), rs7646304 (OPA1), rs17047306 (SPATA17), rs1490338 (SPATA17), rs902351 (SPATA17), and rs2543662 (ITSN2) (Supplementary Table 1).
The SNPs with p &lt; 10-5 in Korean women in the association between SNPs and CRC were: rs10083736 (GOT2), rs16987827 (DHX35), rs8046516 (GOT2), rs9926182 (GOT2), rs17523778 (FAM174B), rs4974411 (TPRA1), rs1834902 (H2AFY), rs16895308 (MAST4), rs8032832 (FAM174B), rs6901560 (PD6), rs11025480 (PRMT3), rs3814110 (BNC2), rs16895307 (MAST4), rs7089063 (MARCH8), rs16893688 (IBTK), rs6861487 (MAST4), rs9613463 (MN1), rs11242237 (H2AFY), rs11150094 (WWOX), and rs9625253 (MN1) (Supplementary Table 2).
Each SNP in this study was assumed to be associated with risk according to an additive genetic model, which performs well, even when the true genetic model may not be known or may be incorrectly specified [30].
A GRS was calculated on the basis of reproducible tagging of SNP-associated loci reaching genomewide levels of significance.
In this study, the GRS was calculated with the 3 SNPs in Korean men and 5 SNPs in Korean women showing the strongest association with CRC (p &lt; 10-6).
The GRS was created by two methods: a simple count method (count GRS) and a weighted method (weighted GRS) [31, 32].
Both methods anticipated each SNP to be independently associated with risk.
We assumed an additive genetic model for each SNP, applying a linear weighting of 0, 1, or 2 to genotypes containing 0, 1, or 2 risk alleles, respectively.
This model is known to perform well, even when the true genetic model is unknown or wrongly specified [30].
The count model assumes that each SNP in the panel contributes equally to the risk for CRC and was calculated by summing the values for each of the SNPs.
The weighted GRS was calculated by multiplying each beta-coefficient by the number of corresponding risk alleles (0, 1, 2).
Title: Outcome classification

The principle outcome variables were prevalence (n = 165) and incidence rates (n = 22), based on national cancer registry and hospitalization records.
Although Korea has a national cancer registry, reporting was not complete during the time of follow-up, and consequently, hospital admission files were used to identify first admission events for CRC.
An incident of CRC was coded as occurring, based on either a positive report from the national cancer registry or upon hospital admission for a cancer diagnosis [33].
According to the International Classification of Diseases, Tenth Revision (ICD-10), CRC was coded as C18-C20 [34].
Title: Statistical analysis

All analyses were conducted using PLINK version 1.06 (Free Software Foundation, Inc., Boston, MA, USA) and SAS statistical software version 9.0 (SAS Institute Inc., Cary, NC, USA).
All statistical tests were two-sided, and statistical significance was determined as p &lt; 0.05.
To evaluate general characteristics of the study population, means and standard deviations (SD) were calculated, and frequency of cigarette smoking, alcohol consumption, and physical activity was determined.
Paired t-tests were performed to indicate the differences between case participants and control participants for both men and women.
A X2 goodness-of-fit test was used to assess whether SNPs were in HWE and to determine differences in genotype frequencies between CRC cases and controls.
The GRS was categorized into quartiles.
The CRC risk associated with genotype was estimated as s ORs and 95% confidence interval (CI), computed using logistic regression with an additive genetic model.
We also used receiver operating characteristic (ROC) curve analysis and calculated the area under the curve (AUC; also known as the C statistic) to evaluate the discrimination power of the model.
In addition, internal validity of each model was checked using bootstrap [35], while 10-fold crossvalidation was used for the external validity of each model (Supplementary Tables 3 and 4) [36].
Our analysis included 687 men (cases, 133; controls, 554) and 476 women (cases, 54; controls, 422), 20-77 years of age.
The number of prevalent cases of CRC identified was 166.
An additional 22 incident cases of CRC occurred during follow-up.
Table 1 details the general characteristics of case participants and control participants at baseline.
The mean age (SD) was 42.9 (± 8.7) years for the study population-43.5 (± 8.9) for men and 42.0 (± 8.4) for women, respectively.
No significant differences were observed in BMI, WC, systolic BP, or diastolic BP among case participants and control participants for both men and women (p &gt; 0.05); however, case participants were older and had a family history of CRC more than control participants in both men and women (p &lt; 0.001).
Table 2 indicates the area under the ROC curves for models that included both conventional risk factors and a genetic risk factor, GRS, compared with the AUC for the model that included only conventional risk factors.
In the prediction model of CRC, we included age and family history of CRC as conventional risk factors and counted GRS or weighted GRS as genetic risk factors.
For both men and women, including the GRS in the model increased the AUC over that observed when the AUC was based on only age or age and family history of CRC.
For men, the AUC (95% CI) was 0.729 (range, 0.682 to 0.767) for conventional risk factors plus counted GRS (p &lt; 0.001) and 0.719 (range, 0.677 to 0.761) for conventional risk factors plus weighted GRS (p &lt; 0.001).
The AUC (95% CI) was 0.692 (range, 0.647 to 0.732) for conventional risk factors alone.
The increase in the AUC for the model with counted GRS was 0.042 and 0.032 for the model with weighted GRS.
For women, the AUC (95% CI) was 0.650 (range, 0.615 to 0.680) for conventional risk factors plus counted GRS (p &lt; 0.001) and 0.646 (range, 0.612 to 0.674) for conventional risk factors plus weighted GRS (p &lt; 0.001).
The AUC (95% CI) was 0.603 (range, 0.569 to 0.637) for conventional risk factors alone.
The increase in the AUC for the model with counted GRS was 0.052 and 0.048 for the model with weighted GRS.
We further examined the association between GRS and CRC risk for both men and women in the KCPS-II, with stratification by family history of CRC (Fig.1).
The interaction between counted or weighted GRS and family history of CRC was significant in men, indicating a stronger genetic effect among participants with a positive family history of CRC than in those without it (p for interaction &lt; 0.05) (ROC, 0.834 for counted GRS; ROC, 0.822 for weighted GRS).
Women with a positive family history of CRC in the highest quartile of weighted GRS had an OR of 22.3 (95% CI, 1.4 to 344.2) and 18.1 (95% CI, 3.7 to 88.1) compared to those without a family history of CRC and counted or weighted GRS in the lowest quartile, respectively (ROC = 0.826 for counted GRS; ROC = 0.818 for weighted GRS).
However, they were not statistically significant (p for interaction &gt; 0.05).
In addition, smoking, alcohol consumption, BMI, and WC did not interact significantly with GRS (data not shown).
This study evaluated the ability of the GRS, which aggregates information from multiple genetic variants, to improve the prediction of CRC risk beyond the prediction risk afforded using conventional risk factors.
For both men and women, inclusion of counted GRS or weighted GRS increased the AUC by 0.5% to 4.2% beyond the AUC provided by conventional risk factors, such as age and family history of CRC.
Men with a positive family history of CRC and GRS in the highest quartile were determined to have a statistically significant increased risk of CRC than those without a family history of CRC and GRS in the lowest quartile.
However, women with a positive history of CRC and GRS in the highest quartile were determined to have an increased risk of CRC than those without a family history of CRC and GRS in the lowest quartile, but this result was not statistically significant.
CRC is a multifactorial disease involving a variety of elements, leading to the development of clinical manifestations [37].
This recognition had led to the development of risk assessment tools that attempt to synthesize the values of numerous variables into a single statement regarding the risk of developing cancer [38].
In this study, 20 SNPs were respectively genotyped in Korean men and women.
Among these SNPs, 3 SNPs in Korean men and 5 SNPs in Korean women showing the strongest association with CRC were used for the calculation of GRS.
The GRS was calculated using a linear weighting of 0, 1, or 2 for genotypes containing 0, 1, or 2 risk alleles, respectively.
The weighted GRS was computed by multiplying each beta-coefficient by the number of corresponding risk alleles.
However, when multiplying each beta-coefficient by the number of corresponding risk alleles, negative values of beta-coefficients may be obtained in some genotypes of the SNPs.
Therefore, it may affect the OR values for CRC when compared to the OR values for CRC determined using counted GRS.
Still, the calculation results of both counted GRS and weighted GRS were similar to each other.
Cornelis et al.
[31] and Ripatti et al.
[32] used methods similar to the GRS created for our study.
Several other studies have reported different ways of calculating risk scores for the prediction of diseases [39-41].
Horne et al.
[39] introduced a regression method for calculating risk scores that incorporated 3 genetic polymorphisms and other risk factors and found that the frequency of coronary heart disease was different at different regression score levels.
Ortlepp et al.
[40] concluded that multiple SNPs are better than single SNPs and that as many as 200 SNPs may be necessary for &#34;reasonable&#34; genetic discrimination.
Aston et al.
[41] suggested that a score based on 90 SNPs in 78 genes can predict the risk of breast cancer, but the identity of the SNPs and the algorithm for calculating the score remain proprietary.
An alternative way to calculate GRS using machine approaches, such as support vector machines (SVMs), could be introduced, as SVMs have already been used to deal with many biological problems, such as DNA expression profiles [42].
Still, further studies are needed to use machine learning approaches, such as SVMs, for the calculation of GRS.
To our knowledge, there have been no studies evaluating a GRS using SNPs contributing to CRC for the prediction of the disease in the Korean population.
The present study evaluated a prediction model using counted GRS or weighted GRS together with conventional risk factors, such as age and family history of CRC among Koreans.
The risk of CRC is said to increase in individuals with a family history of CRC, in particular those &gt;50 years of age [43, 44].
From a recent study, a CRC prediction model was developed with known major risk factors of age, BMI, alcohol consumption, smoking status, and physical activity level for middle-aged Japanese men [17].
Another recent study on the prediction model of CRC included an individual&#39;s age, sex, history of CRC, sigmoidoscopy/colonoscopy, polyps, family history of CRC, smoking, physical activity, aspirin/NSAID use, vegetable intake, BMI, and hormone replacement in women [16].
In our study, the prediction model of CRC was comprised of conventional risk factors, such as age and family history of CRC, together with the GRS.
As determined, inclusion of counted GRS or weighted GRS revealed improved estimates of CRC prediction beyond that provided by conventional risk factors, such as age and family history of CRC.
For example, when counted GRS were added to the prediction model of CRC consisting of age and family history of CRC, the AUC increased by 4.2% in men and 5.2% in women, whereas the AUC increased by 3.2% in men and 4.8% in women when weighted GRS were added to the same model.
Studies showing significant relationships of GRS in conjunction with coronary heart disease, type 2 diabetes, and breast cancer have reported that considering the contribution of multiple SNPs may improve the predictive value of GRS for such diseases [18, 31, 45].
In other words, combining multiple loci with modest effects into a global GRS might improve identification of persons who are at risk for such diseases [23-25].
For example, in the ARIC study, the contribution of multiple SNPs into a single GRS was responsible for an improvement in the prediction of incident CHD [18].
In a study that used counted GRS or weighted GRS to determine the risk for type 2 diabetes in US men and women, individuals in the highest quintile of GRS had a significantly increased risk of type 2 diabetes compared to those in the lowest quintile; however, the addition of GRS increased the AUC by only 1%.
In this instance, the GRS was determined to be useful when combined with the joint effects of BMI and counted GRS or family history of diabetes and counted GRS [30].
In our study, individuals in the highest quartile of GRS had increased risk of CRC compared to those in the lowest quartile of GRS for both men and women.
In addition, in strata of family history of CRC and GRS, this increase was even higher in individuals with a family history of CRC in the highest quartile of GRS compared to those without a family history of CRC in the highest quartile of GRS in both men and women.
Still, there were statistically significant interactions in men but not in women.
In this study, the most commonly used conventional risk factors, such as smoking and alcohol consumption, were also not included in the prediction model of CRC, as smoking, alcohol consumption, BMI, and WC did not significantly interact with the GRS.
Therefore, further studies are needed to verify these results.
A family history of CRC is commonly used as a surrogate marker for determining genetic susceptibility to CRC and remains one of the strongest risk factors for the disease [10, 31, 46].
Approximately 25% of all CRC cases occur in individuals with a family history of the disease and no genetic disorders [47].
In addition, some retrospective studies have suggested that a history of CRC in a first-degree relative (a parent or sibling) elevates a person&#39;s lifetime risk of CRC from 1.8-fold to 8.0-fold [10, 47].
This family history risk factor may encompass both genetic and shared environmental components [31].
In our study, after controlling for age and GRS, the strong relationship between family history of CRC and risk of CRC persisted.
These findings suggest that other risk loci remain to be discovered or that family history has a much larger shared environmental component than previously thought [31].
Our study was not without limitations.
The cross-sectional design precluded the determination of causality, and a prevalent case bias may exist due to the higher number of prevalent cases (n = 165) of CRC included compared to the number of incident cases (n = 22).
Combining prevalence and incidence cases could introduce survival biases.
Still, the 5-year survival rate for CRC in Koreans was 71.3% in 2009 while that in Americans, Europeans, and Japanese was 65.0%, 56.2%, and 65.2%, respectively [43].
It could be said that Koreans have higher survival rates for CRC compared to other ethnic populations.
Additionally, this study is also a case-cohort study.
Blood samples of prevalent cases used in this study were from baseline, and incidence cases during the follow-up period, suggested as prevalent cases in this study, might have been missed, as other blood samples were not taken.
Therefore, those prevalent cases at baseline might have become incident cases or mortality cases during the follow-up period.
It is hard to say if this study was performed among survivors.
Another limitation included a self-reported family history of CRC, thus precluding the definitive exclusion of potential misclassifications.
The statistical power of the current study might be too low, as genotyping was performed separately for men and women.
In addition, performing multiple tests separately in both men and women may increase error rates.
Although CRC affects men and women equally, gender differences in CRC may exist.
For example, regarding colorectal polyps and tumors, men had a greater risk of polyps (OR, 1.52; 95% CI, 1.41 to 1.64) and tumors (OR, 1.43; 95% CI, 1.22 to 1.68) than women.
In addition, women had greater number of purely right-sided polyps and tumor development [48].
Therefore, detection of genetic effects separately in men and women may be needed.
In addition, age differences in case and control participants may also increase error rates, as control participants may become CRC patients when they reach the case age.
This study also involved the lack of validation and replication of the current study results.
Therefore, it is hard to say that there may have been a true association between GRS and CRC in Korean men and women.
However, bootstrapping and 10-fold cross validation were used for internal and external validity of this current study.
Furthermore, although sigmoidoscopy/colonoscopy history was the strongest risk factor in the previous study, this current study did not include it as one of the conventional risk factors of CRC.
Cases included in this study were also relatively small.
This study also excluded cases ≥ 55 years of CRC onset age to obtain early-onset CRC cases.
Therefore, estimate effects of cases ≥ 55 years of CRC onset age were hard to be seen in this study.
Finally, most SNPs found to be associated with CRC among the study population were not similar to those SNPs found in relation to CRC among other populations.
It also could be due to differences in ethnic population and the ages of case participants included in this study (cases &lt; 55 years).
Nevertheless, this relatively large-scale study demonstrated the effectiveness of the prediction model of CRC using the GRS consisting of only SNPs that associated significantly with CRC and evaluated the effects on the risk for CRC in combination with conventional risk factors, such as family history of CRC, with the GRS.
Moreover, the present study included the Korean population, whereas previous studies involving CRC prediction models using conventional risk factors or the relationship of genetic risk factors to CRC were limited to white and Japanese populations [17, 20-22].
In conclusion, our findings suggest that the prediction model of CRC revealed improved prediction estimates when age, family history of CRC, and the GRS in the Korean population were included.
Furthermore, when compared to those in the lowest quartile of GRS in the presence or absence of a family history of CRC, the risk of CRC was found to be significantly increased in individuals with a family history of CRC in the highest quartile of GRS.
However, it was statistically significant in men but not in women.
Findings in this current study might provide a small piece of evidence in prediction of CRC for reducing its prevalence and incidence rates.
The prediction model developed in this study needs to be validated or replicated in an independent population.
Therefore, further studies are needed to be applied to the general population.
Effects of family history of CRC and GRS on the risk of CRC in Koreans who participated in the KCPS-II (adjusted for age, BMI and smoking status) (A) counted GRS in men and women (B) weighted GRS in men and women.
CRC, colorectal cancer; GRS, genetic risk scores; KCPS-II, Korean Cancer Prevention Study-II; BMI, body mass index; OR, odds ratio.
aMeans statistically significant.
Title: General characteristics of case participants and control participants in KCPS-II

Title: Odds ratios and 95% confidence intervals for colorectal cancer using counted and weighted genetic risk score in Korean men and women in KCPS-II
Title: Association Study between Folate Pathway Gene Single Nucleotide Polymorphisms and Gastric Cancer in Koreans

Gastric cancer is ranked as the most common cancer in Koreans.
A recent molecular biological study about the folate pathway gene revealed the correlation with a couple of cancer types.
In the folate pathway, several genes are involved, including methylenetetrahydrofolate reductase (MTHFR), methyltetrahydrofolate-homocysteine methyltransferase reductase (MTRR), and methyltetrahydrofolate-homocysteine methyltransferase (MTR).
The MTHFR gene has been reported several times for the correlation with gastric cancer risk.
However, the association of the MTRR or MTR gene has not been reported to date.
In this study, we investigated the association between the single nucleotide polymorphisms (SNPs) of the MTHFR, MTRR, and MTR genes and the risk of gastric cancer in Koreans.
To identify the genetic association with gastric cancer, we selected 17 SNPs sites in folate pathway-associated genes of MTHFR, MTR, and MTRR and tested in 1,261 gastric cancer patients and 375 healthy controls.
By genotype analysis, estimating odds ratios and 95% confidence intervals (CI), rs1801394 in the MTRR gene showed increased risk for gastric cacner, with statistical significance both in the codominant model (odds ratio [OR], 1.39; 95% CI, 1.04 to 1.85) and dominant model (OR, 1.34; 95% CI, 1.02 to 1.75).
Especially, in the obese group (body mass index ≥ 25 kg/m2), the codominant (OR, 9.08; 95% CI, 1.01 to 94.59) and recessive model (OR, 3.72; 95% CI, 0.92 to 16.59) showed dramatically increased risk (p &lt; 0.05).
In conclusion, rs1801394 in the MTRR gene is associated with gastric cancer risk, and its functional significance need to be validated.
According to the Korea Central Cancer Registry data, gastric cancer is ranked as the most common cancer in Korean and men and accounted for about 20.1% of all cancers in Koreans.
In Korea, 29,727 cases of gastric cancer were newly diagnosed, and the crude incidence rate of gastric cancer in 2009 was 59.9 per 100,000.
In males, the number of newly diagnosed cases and incidence rates of stomach cancer in 2009 were 19,953 cases and 80.2 per 100,000, respectively.
The 5-year relative survival rates of gastric cancers were increased by 22.5% from 42.8% in 1993-1995 to 65.3% in 2005-2009 [1].
Evidence from pathology and epidemiology studies has provided a human model of gastric carcinogenesis with the following sequential stages: chronic gastritis; atrophic gastritis; intestinal metaplasia; and dysplasia [2].
Also, an environmental element of gastric cancer occurrence exists plentifully.
The best well-known risk factors for gastric cancer are Helicobacter pylori infection, by far the strongest established risk factor for gastric cancer; a family history; and smoking.
Several factors related to nutrition and food preservation, such as high intake of salt-preserved foods and dietary nitrite or low intake of fruit and vegetables, are likely to increase the risk of gastric cancer [3].
Cancerogenesis, which is the loss of cellular differentiation that leads the digestive tract to cancer, is inhibited by nutrition factors, such as retinoid, vitamins B-complex (including folate), vitamin C, D3, and E, polyphenol, fiber, calcium, selenium, and polyunsaturated fatty acids (e.g., Omega-3) [4].
Especially, the environmental factors in cancer and a high intake of vitamin B-complex play important roles in DNA synthesis, repair, and methylation [5].
There have been many epidemiological studies that gastric cancer is associated with high-risk dietary profiles (low folate, vitamin B6 intake, and high alcohol), smoking, and low blood folate concentration [6-8].
Folate is the water-soluble form of vitamin B9 in foods [9].
Leafy vegetables, such as spinach, turnip greens, lettuces, dried beans, peas, fortified cereal products, sunflower seeds, and certain fruits, are rich sources of folate.
The recommended dietary allowance for adults is 400 µg of food folate a day, which is equivalent to about 240 µg synthetic folic acid in supplements or fortified food.
Women of child-bearing age planning a pregnancy should take 400 µg synthetic folic acid daily in addition to their normal dietary intake [10, 11].
There is now substantial data to support an important role for folate in the prevention of neural tube defects (NTDs), Down syndrome, vascular disease, various cancers, Alzheimer&#39;s disease, cognitive function, and affective disorders [12].
Cumulative evidences suggest that food containing folate decreases the risk of colorectal, pancreatic, and esophageal cancers [13].
Also, a western lifestyle, which is associated with high total caloric or fat intake (include red meat), and inactive life pattern, has been considered one of the main reasons for increasing trends of cancer in Koreans [14].
Variations in levels of serum total homocysteine (tHcy) can result from genetic or nutrient-related disturbances in the folate pathway.
In this mechanism, fasting levels of tHcy mainly reflect the remethylation pathway.
In the remethylation pathway, the primary methyl donor for the vitamin B12-dependent conversion of Hcy to methionine is 5-methlytetrahydrofolate, which in turn forms 5, 10-methlytetrahydrofolate by means of the enzyme methylenetetrahydrofolate reductase (MTHFR).
Recent reports suggested that individual genetic variation or single nucleotide polymorphisms (SNPs) in various genes involved in cellular folate metabolism or transport may also be implicated in cancer risk.
In the folate metabolism pathway, cellular folate act as donors and receptors of methyl groups in the biosynthesis of nucleotide precursors used for DNA synthesis and provide methyl groups for methylation of DNA, RNA, and proteins [15].
MTHFR, methyltetrahydrofolate-homocysteine methyltransferase reductase (MTRR), and methyltetrahydrofolate-homocysteine methyltransferase (MTR) genes are well known for their association with the folate pathway [16,17].
MTHFR maps to chromosome 1p36.3 and is 2.2-kb long and contains 11 exons.
The gene product is a 77-kD protein, although a smaller isoform of approximately 70 kD has been observed in some tissues, such as liver [18].
SNPs in MTHFR, such as C677T (rs1801133), A1298C (rs1801131), and G1793A (rs2274976), have been suggested to be associated with several cancers (colon, gastric, and breast), cardiovascular disease, NTDs, and pregnancy complications [6, 9, 19-21].
In cancer patients, higher sensitivity to 5-fluorouracil (5-FU) among patients carrying the MTHFR 677TT and 1298AA genotypes compared to the others was reported, demonstrating a strong predictive ability of these polymorphisms in response to 5-FU-based chemotherapy in gastric cancer [22, 23].
The MTRR gene was mapped to human chromosome 5p15.3-p15.2, is 34 kb long, comprises 15 exons, has 22 codon SNP (cSNP) variation sites, and is thought to produce cytosolic and mitochondrial mRNA isoforms [24, 25].
Polymorphisms in MTRR-rs1801394 (A66G), rs1532268 (S175L), and rs10380 (H595Y)-have been associated with the risk of cancers (breast, colon, prostate, pancreatic, and acute lymphoblastic leukemia), Down syndrome, and Alzheimer disease [26-31].
The MTR gene maps to human chromosome 1q42, is 105 kb, comprises 33 exons, and has 116 SNP variation sites.
One polymorphism in MTR (rs1805087) has been associated with colorectal cancer and non-Hodgkin lymphoma risk [8, 32].
For gastric cancer, a genetic variation in MTHFR has been recently reported [33, 34].
Also, case-control studies with specific results on folate intake (or blood concentration) and gastric cancer risk suggest a protective role in a couple of reports [35, 36].
The MTR or MTRR gene variations were associated with colorectal and pancreatic cancer risk [8, 37], but there has been no report on gastric cancer.
In this study, we investigated the association between polymorphisms in MTHFR, MTRR, and MTR and the risk of gastric cancer in Koreans.
Title: Clinical samples

Buffy coat samples of 1,261 patients (69.4%) who had undergone surgery at the Gastric Cancer Center, National Cancer Center (NCC) of Korea, between September 2001 and December 2005 were included as cases.
Most of the selected gastric cancer patients possessed distal stomach tumors.
Archival 375 (30.6%) normal buffy coat samples who had joined cancer screening examinees from the NCC of Korea between August 2002 and December 2005 were also included as controls in this study.
Specification of group is listed in Table 1.
This study was approved by the Institutional Review Board (IRB) of the NCC of Korea (NCCNSH03-024).
Title: DNA extraction and sample preparation

DNA was extracted from 350 µL of whole blood using the MagAttract DNA Blood Midi M48 Kit (Qiagen, Valencia, CA, USA) using a Qiagen BioRobot M48 workstation, according to the manufacturer&#39;s protocols automatically.
The purity and concentration of isolated DNA were determined by a Nanodrop ND-1000 spectrophotometer (Nanodrop Technologies, Wilmington, DE, USA).
We needed more detailed quantity of each sample for genotyping reaction; so, we measured the quantity of DNA using the Quant-iT PicoGreen dsDNA assay kit (Invitrogen, Inc., Carlsbad, CA, USA) and made a dry plate for genotyping reactions with 10 ng per well of 384 plates.
Title: Primer selection and assay design

Seventeen SNPs in MTHFR, MTRR, and MTR were selected, covering previously studied SNPs, such as C677T (rs1801133) and A1298C (rs1801131) in MTHFR [29] and A66G (rs1801394) in MTR [27].
The SNP information, including nucleic acid sequences, was collected from dbSNP (http://www.ncbi.nlm.nih.gov/projects/SNP/; Build 136).
SNPs in coding regions and promoters were chosen at first.
Regulatory SNPs with validated allele frequency and nonsynonymous cSNPs were finally included.
Multiplexed PCR primers were designed for the best PCR reaction and designed for evading the biophysical hurdle - secondary structure, self-ligation, competition of primers, etc.
- using MassARRAY Assay Designer version 3.0 (Sequenom, Inc., San Diego, CA, USA) (Table 2).
Title: PCR amplification

PCR reactions were performed in a total volume of 5 µL with 10 ng of genomic DNA, 1.625 mM MgCl2, 0.1 units of HotStarTaq polymerase (Qiagen), 0.5 mM dNTPs (Invitrogen, Inc.), and 100 nM primers.
The PCR reactions started at 94℃ for 15 min, followed by 45 cycles of 94℃ for 20 s, 50℃ for 30 s, and 72℃ for 1 min, with a final extension of 72℃ for 3 min.
Title: Genotyping

Genotyping was carried out using the iPLEX assay on the MassARRAY Platform (Sequenom, Inc.).
The iPLEX extension was performed in a total volume of 9 µL with 50 µM dNTP/dideoxynucleotide phosphate (ddNTP) each, 0.063 unit/µL Thermo Sequenase (Sequenom, Inc.), and 625 nM to 1.25 µM extension primers.
iPLEX extension was performed using 2-step 200 short-cycle programs.
The sample was denatured at 94℃ for 5 s, and strands were annealed at 52℃ for 5 s and extended at 80℃ for 5 s. The annealing and extension cycle was repeated 4 more times for a total of 5 cycles, looped back to a 94℃ denaturing step for 5 s, and then entered the 5-cycle annealing and extension loop again.
The 5 annealing and extension steps with the single denaturing step were repeated an additional 39 times for a total of 40 cycles.
A final extension was done at 72℃ for 3 min.
iPLEX extension products were desalted by adding 6 mg resin (SpectroCLEAN; Sequenom, Inc.) and 16 µL water.
After full rotation in room temperature, the reaction mixture was centrifuged at 3,500 g, 5 min.
After desalting, products were transferred to SpectroCHIP using a Nanodispenser (SpectroPOINT; Sequenom, Inc.) and then read through matrix-assisted laser desorption/ionization time-of-flight (SpectroReader; Sequenom, Inc.).
The resulting genotype data were collected by MassArray Typer software version 4.0 (Sequenom, Inc.).
Title: Statistical analysis

The chi-square test for association was used to test differences of genotype frequencies between normal and gastric cancer patients.
Odds ratios (OR) and their 95% confidence intervals (CI) in relation to MTHFR, MTTRR, and MTR genotypes were calculated.
Also, after adjustment for sex, family history, smoking, drinking, and H. pylori infection, global chi-square test was also employed to calculate OR and their 95% CIs for individuals.
Statistical analyses were performed using SAS version 9.0 (SAS Institute Inc., Cary, NC, USA).
Title: Association between folate pathway genes and gastric cancer risk

One thousand two hundred sixty-one gastric cancer patients and 375 control groups were included in the present study.
From 17 SNPs, rs1476413, rs2066470, rs2274976, rs3737964, rs4846048, rs7533315, rs1805087, rs1532268, rs2303080, rs162036, rs2287780, rs16879334, and rs10380 had a minor allele frequency less than 5%.
For each SNP, the p-value of χ2-test and OR were calculated (Table 3).
Significant associations between genotypes of folate pathway SNPs and the risk of gastric cancer were only observed for rs1801394 (p &lt; 0.05).
In rs1801394, the frequencies of the AG heterozygote genotype were 0.411 and 0.366 in patients and control groups, respectively.
The risk of gastric cancer in patients with the risk allele was increased as OR, 1.39 (codominant model; 95% CI, 1.04 to 1.85) or OR, 1.34 (dominant model; 95% CI, 1.02 to 1.75) with statistical significance (p &lt; 0.05).
Title: Association between folate pathway genes and obese gastric cancer patients

Further, the association between the 17 SNPs of folate pathway genes and gastric cancer risk was analyzed and stratified by obesity categories (BMI, &lt;25 vs. ≥25) (Table 4).
Interestingly, the risk of rs1801394 was dramatically increased for the codominant model (OR, 9.08; 95% CI, 1.01 to 94.59; p &lt; 0.05) only among obese subjects.
For the dominant model, the OR was also increased with statistical significance (OR, 1.98; 95% CI, 1.06 to 3.49; p &lt; 0.05).
Also, the recessive model showed significantly higher risk (OR, 3.72; 95% CI, 0.92 to 16.59).
Additionally, we analyzed the correlation among family history, H. pylori infection, and folate gene SNPs but did not find any significant association (data not shown).
For early detection and diagnosis of cancer, the discovery of new biomarkers is very important, and the interest of researchers is growing rapidly.
Also, genetic factors, including polymorphisms of genes involved in tumorigenesis, may partly explain the difference in individual susceptibility to cancer [38].
In the present study, we studied the impact of folate pathway gene polymorphisms on the risk of gastric cancer in a Korean population.
Since folic acid was a critical cofactor in one-carbon metabolism involving in the biological methylation and nucleotide synthesis pathways, our study may find clues for the possible effect to dietary effects on gastric cancer.
In the report of previous studies, the frequency of genotypes for MTHFR was similar with our results-less than 5% for many SNPs [39, 40].
For MTHFR, one study found the association of rs1801133 with gastric cancer risk but no association of rs1801131 [34].
However, different results have been reported in other studies, showing no association for either SNP with gastric cancer [33].
We also found no association of either SNP with gastric cancer risk, which is concordant with the results of Kim&#39;s group [33].
For MTR, although a couple of reports have shown a significant association between the polymorphism of MTR and the risk of certain cancers [32], we could not find any association in our study population of gastric cancer.
In our study, a significant association of rs1801394 (A66G) in MTRR was found, especially with high OR among the obese gastric cancer group.
MTRR regenerates a functional methionine synthase via reductive methylation so that methionine synthase can catalyze methionine synthesis, which is an essential amino acid required for protein synthesis and one-carbon metabolism.
The A66G polymorphism is reported to be functional, so that the variant enzyme has a lower affinity for MTR [41].
A large number of studies have been conducted to evaluate the role of rs1801394 in different kinds of cancers; the results are still plausible.
One meta-analysis reported that the A66G polymorphism should contribute to tumor susceptibility, showing significantly increased risk among Asians with the G allele, which is also in concordance with our result [42].
Our result is biologically plausible, since the polymorphisms or gene-environment interactions, rather than folate intake alone, would have an impact on the risk for digestive track cancer, because functional SNPs in folate-related genes were known to contribute to the alteration of folate metabolism [30].
The SNPs in folate pathway genes (such as MTHFR) were reported to influence to the decrease of the activity of the enzyme, leading to hyperhomocysteinemia, particularly in folate-deficient states [43].
Homocysteine was related to cancer formation, like tumor necrosis factor, obesity, and the folate pathway, and is known to be one of the main risk factors for distal gastric cancer, including H. pylori infection and dietary factors [44].
Although obesity (BMI &gt; 25) was more prevalent in patients with cardia cancer compared to patients with gastric distal cancer in Koreans [45], a previous study reported that obesity is a major risk factor for several types of cancer, including gastric cancer [46].
Also, many epidemiological studies have shown that obesity is a risk factor for breast cancer, colon and kidney cancer, and malignant adenomas of the esophagus.
Obesity subjects have an approximately 1.5-3.5-fold increased risk of developing these cancers compared with normal-weight subjects [47].
Here, we report the association of genetic variations in MTRR with the risk of gastric cancer for the first time.
In further studies, we need to validate our finding in a larger population, considering detailed clinical information, and study the functional relevance of polymorphisms with cancer development more.
Also, we need to consider other genes in the folate pathway and investigate gastric cancer susceptibility with epidemiological and environmental factors (e.g., nutrition intake, 5-FU drug interference, blood folate concentration, etc.).
Title: Description of the case and control group

Title: Primers used for the genotyping of polymorphisms in folate metabolic pathway genes

Title: Association between the folate pathway polymorphisms and gastric cancer patient risk, ORa value (95% confidence intervals)

Title: Association between the folate pathway polymorphisms for gastric cancer risk by obesity, ORa value (95% confidence intervals)
Title: Effect of Combining Multiple CNV Defining Algorithms on the Reliability of CNV Calls from SNP Genotyping Data

In addition to single-nucleotide polymorphisms (SNP), copy number variation (CNV) is a major component of human genetic diversity.
Among many whole-genome analysis platforms, SNP arrays have been commonly used for genomewide CNV discovery.
Recently, a number of CNV defining algorithms from SNP genotyping data have been developed; however, due to the fundamental limitation of SNP genotyping data for the measurement of signal intensity, there are still concerns regarding the possibility of false discovery or low sensitivity for detecting CNVs.
In this study, we aimed to verify the effect of combining multiple CNV calling algorithms and set up the most reliable pipeline for CNV calling with Affymetrix Genomewide SNP 5.0 data.
For this purpose, we selected the 3 most commonly used algorithms for CNV segmentation from SNP genotyping data, PennCNV, QuantiSNP; and BirdSuite.
After defining the CNV loci using the 3 different algorithms, we assessed how many of them overlapped with each other, and we also validated the CNVs by genomic quantitative PCR.
Through this analysis, we proposed that for reliable CNV-based genomewide association study using SNP array data, CNV calls must be performed with at least 3 different algorithms and that the CNVs consistently called from more than 2 algorithms must be used for association analysis, because they are more reliable than the CNVs called from a single algorithm.
Our result will be helpful to set up the CNV analysis protocols for Affymetrix Genomewide SNP 5.0 genotyping data.
Human genome variation has facilitated the understanding of inter-individual phenotypic differences [1, 2].
In addition to single-nucleotide polymorphisms (SNPs), it is widely accepted that large-scale DNA structural variation, termed copy number variation (CNV), is a major component of human genetic diversity [2, 3].
Genomewide SNP genotyping data can be used for CNV calling [4]; therefore, if we can get reliable CNV calls from the SNP genotyping data, a CNV-based genomewide association study (GWAS) can be realized.
Among many whole-genome CNV analysis platforms, SNP arrays have been suggested as a resource for CNV discovery due to their ubiquitous genome coverage and relatively advantageous resolution.
However, despite the importance of CNV-disease association analysis, CNV calling from SNP genotyping data has not been well established.
Affymetrix Genomewide SNP 5.0 is one of the commonly used SNP array platforms for SNP-GWAS as well as CNV analysis [5].
We previously validated the accuracy and reproducibility of CNVs called from Affymetrix SNP array 5.0 data by comparing the CNV calls from 3 different array platforms using NEXUS software: Affymetrix SNP array 5.0, Agilent 2X244K CNV array, and NimbleGen 2.1M CNV array [6].
Recently, a number of CNV defining algorithms have been developed, which have facilitated the CNV-based GWAS [7-14].
However, due to the fundamental limitation of SNP genotyping data for the measurement of signal intensity, there are still concerns regarding the possibility of false discovery or low sensitivity for detecting CNVs [15, 16].
Indeed, CNV calling is dependent on the types of array platforms and analytic tools.
Each platform and calling algorithm has its own advantages and disadvantages; so, one single algorithm or array platform is not always best for determination of CNVs [17-19].
Recently Pinto et al.
[20] showed that different analytic tools applied to the same raw data typically yielded CNV calls with &lt;50% concordance and, using multiple algorithms, minimize the number of false discoveries.
To remedy the potential limitations of SNP array for CNV detection, more than one way of CNV calling by using several different segmentation algorithms are performed, and overlapped calls are used for GWAS analysis [21-24].
In this study, we tried to verify the effect of adopting multiple CNV calling algorithms and set up the most reliable pipeline for CNV calling with Affymetrix Genomewide SNP 5.0 data.
We selected the 3 most commonly used algorithms for CNV segmentation from SNP genotyping data, PennCNV, QuantiSNP, and BirdSuite.
After defining the CNV loci using the 3 different algorithms, we assessed how many of them overlapped with each other, and we also validated the CNVs by genomic quantitative PCR (qPCR).
Finally we concluded that CNVs that were consistently called from more than 2 different calling algorithms are more reliable than the CNVs called from a single algorithm.
Our result will be helpful to set up the CNV analysis protocols for Affymetrix Genomewide SNP 5.0 genotyping data.
Title: Study materials

We used Affymetrix Genomewide SNP 5.0 genotyping data provided by the Korea Association Resource (KARE) consortium, Korean Genome Epidemiology Study (KoGES).
As a control, we purchased a HapMap cell line, GM10851, from Coriell Institute for Medical Research (Camden, NJ, USA) and extracted genomic DNA using the DNeasy Blood &amp; Tissue Kit (Qiagen, Hilden, Germany).
Title: Pre-processing SNP array data

Before CNV calling procedures, all required pre-processing procedures, including allele correction, summarization, and background correction, were performed as described previously by using the software provided by Affymetrix [5].
In brief, background-corrected data were normalized using quantile normalization and summarized by median polish.
Title: CNV calling

For this study, Affymetrix 5.0 SNP array (Affymetrix, Santa Clara, CA, USA) data of 10 subjects were randomly selected from the KARE dataset for CNV calling.
Affymetrix 5.0 data of NA10851 was used as a control.
For defining CNVs, we choose 3 different segmentation algorithms, PennCNV [25], QuantiSNP [26] and BirdSuite [27].
PennCNV implements a hidden Markov model (HMM) and considers SNP allelic ratio distribution in addition to signal intensity.
We only used CNVs from PennCNV with samples that had a standard deviation of log R ratio (LRR) smaller than 0.2, drift values of b-allele frequency (BAF_drift) smaller than 0.01, and waviness factor between -0.05 and 0.05.
For QuantiSNP, we did not apply any criteria for sample filtering, since QuantiSNP uses an Objective Bayes Hidden-Markov Model (OB-HMM) for calibrating the model for fixing the false positive error rate and maximum marginal likelihood to set other hyperparameters.
It was originally developed for detecting CNVs from BeadArray SNP genotyping data of Illumina, but Affymetrix data also could be processed as long as they have probe signal intensities and B-allele frequencies.
Birdsuite uses exclusive copy number analysis routine (Canary) for CNP locus and HMM for finding rare CNVs.
It generates a logarithm of the odds ratio (LOD) value for each CNV that describes the likelihood of a CNV relative to no CNV over a given interval, including flanking sequences.
Of the BirdSuite calls, only the CNVs that have an LOD value smaller than 2 were used for further analysis.
All 3 methods were used with default values, and no other option was added or changed.
Title: Validation

To estimate the false discovery rate of our CNV-calling algorithm, CNVs were validated by genomic real-time qPCR.
For this purpose, we randomly selected 25 CNV loci and performed genomic qPCR using DNAs from study subjects who showed corresponding CNVs on that loci.
The PCR primers used in this study were designed using the PrimerQuest program (http://www.idtdna.com/Scitools/Applications/Primerquest).
To verify the specificity of the PCR reactions under the unified denaturation temperature (60℃), we performed PCR and agarose gel electrophoresis for each primer set.
We also screened the University of California Santa Cruz (UCSC) database (http://genome.ucsc.edu/) to confirm the unique sequence without any repeat sequences in the primers.
Sequence information of the primers for genomic qPCR validation is listed in Table 1.
Ten microliters of the reaction mixture contained 10 ng of genomic DNA, SYBR Premix Ex Taq TM II (TaKaRaBio, Shiga, Japan), 1×ROX (Toyobo, Osaka, Japan), and 10 pmol of primers.
Thermal cycling conditions consisted of 1 cycle of 10 s at 95℃, followed by 40 cycles of 5 s at 95℃, 10 s at 61℃, and 20 s at 72℃.
All PCR experiments were repeated twice, and amplification efficiencies for both target and reference genes were evaluated using a standard curve over 1:5 serial dilutions.
The copy number of each target was defined as 2-ΔΔCT, where ΔCt is the difference in threshold cycles for the sample in question normalized against the reference gene (RNaseP) and expressed relative to the value obtained by calibrator DNA (NA10851 and Promega DNA), as described elsewhere [28].
The general characteristics of the CNV calls from the 3 algorithms are shown in Table 2.
The total number of CNV calls from the 3 algorithms was similar-range 814 to 1,362.
The median size of the CNVs called by PennCNV (12.9 kb) were bigger than the other 2 algorithms (7.5 kb and 7.9 kb).
The size of the CNVs consistently detected by 2 or more algorithms was generally bigger than that of the CNVs detected by a single method.
For example, the average sizes of the CNVs that were detected from all 3 algorithms, 2 algorithms, and 1 method were 107 kb, 73.2 kb, and 15.9 kb, respectively (Table 3).
Fig.1 shows the examples of different sizes of CNVs according to the LRR distribution of gain- and loss-type CNV regions, depending on how many algorithms detected the CNV in that region.
These data suggest that larger-sized CNVs are generally more prominent; so, they can be relatively easily detected by any segmentation method, regardless of the algorithm.
But, detection of the smaller-sized CNVs can be affected more easily by the characteristics of the CNV defining algorithm; so, they can be detected by 1 particular algorithm.
A recent report examining the impact of inaccuracy of CNV detection from SNP genotyping data revealed that CNVs, defined as the copy number changes of &gt;7 consecutive probes, were fairly reliable in case of deletion type CNVs [29].
In our previous study exploring the CNV profiles of Koreans using an Affymetrix array, we suggested the filtering condition of CNV call as &gt;6 consecutive probes to be reliable [5].
Therefore, to make more reliable conditions in this study, we filtered the CNV calls as &gt;7 consecutive probes.
Under this criterion, the number of CNV calls from the 3 algorithms was reduced down from 295 to 385, while the median size became bigger-50 kb by PennCNV and QuantiSNP and 23.5 kb by BirdSuite (Table 2).
In this condition, same as above, the size of the CNVs consistently detected by 2 or more algorithms was generally bigger than that of the CNVs detected by a single method (Table 3).
In terms of the number of CNVs consistently identified by different algorithms, of the CNV calls using the PennCNV algorithm, 48.6% (396/814 CNVs) was detected by 2 or more algorithms, and only 4.9% (40/814 CNVs) was detected by all 3 algorithms.
Similarly, of the CNV calls using QuantiSNP, 42.9% (466/1,087 CNVs) was detected by 2 or more algorithms and only 3.9% (42/1,087 CNVs) was detected by all 3 algorithms.
However, in the case of the BirdSuite algorithm, although the number of CNV calls was the biggest, only 9.6% (131/1,362 CNVs) was detected by 2 or more algorithms and 2.9% (40/1,362 CNVs) was detected by all 3 algorithms.
Taken together, only 17% of the total CNV calls (465/2,735 CNVs) were defined by more than 2 algorithms, and only 1.5% (40/2,735 CNVs) was defined by all 3 algorithms (Table 3).
In the filtering condition of &gt;7 consecutive probes, the number of CNVs consistently identified by different algorithms was improved.
For example, of the CNV calls using the PennCNV algorithm, 76.6% (226/295 CNVs) was detected by 2 or more algorithms and only 23.4% (69/295 CNVs) was detected uniquely by PennCNV.
QuantiSNP calls showed a similar trend.
However, even under this condition, 92.5% (356/385 CNVs) of BirdSuite calls were unique (Table 3).
All the CNVs consistently detected by 2 algorithms were from PennCNV and QuantiSNP, while no CNVs that were detected by the BirdSuite algorithm were consistently detected by 2 algorithms, with only 1 exception.
These data suggest that CNV calls from SNP genotyping depend substantially on the characteristics of calling algorithms, and only part of the CNV calls from each algorithm seems to be reliable.
Therefore, CNVs identified from SNP genotyping data must be validated as completely as possible, and especially in the case of using a single calling algorithm, the CNVs must be validated more carefully.
Filtering the CNV calls by the number of consecutive probes can improve the reliability of CNV calls.
To validate the CNVs identified from 3 different algorithms in this study, we randomly selected 25 CNV regions across the whole chromosomes.
The genomic qPCR validation results are listed in Table 4.
In case of the CNVs identified by all 3 algorithms, 71.4% (5/7) of the consistency was observed between the CNV call and genomic qPCR result.
Of the CNVs identified by more than 2 algorithms, 57.6% (19/33) was consistent.
Of the unique CNV calls defined by just a single algorithm, only 38.3% (18/47) was consistent, but the other 61.7% (28/46) was not consistent.
In the case of the CNVs defined as &gt;7 consecutive probes, the consistency was generally improved-47.8% (unique CNVs), 62.2% (CNVs identified by more than 2 algorithms), and 83.3% (CNVs identified by all 3 algorithms).
These results indicate that CNV calls from 2 or more algorithms are more reliable than those from single algorithms.
CNV calls from all 3 algorithms are, of course, the most reliable, but this is too stringent; so, the number of CNVs is not applicable for GWAS analysis.
In this study, we aimed to verify the effect of adopting multiple CNV calling algorithms and set up the most reliable pipeline for CNV calling with Affymetrix Genomewide SNP 5.0 data.
We found that CNVs defined by a single CNV calling algorithm may not reliable enough for further GWAS study, regardless of the types of algorithms.
Based on our findings, we propose that for reliable CNV-based GWAS using SNP array data, CNV calls must be performed with at least 3 different algorithms, and the CNVs consistently called from more than 2 methods must be used for association analysis.
Signal intensity (Log R ratio) plots of the copy number variation (CNV) regions.
(A, B) CNVs detected from all 3 algorithms.
(C, D) CNVs detected from 2 of the 3 algorithms.
(E, F) CNVs uniquely detected by a single algorithm.
Green bars, copy number-loss CNV regions; red bars, copy number-gained CNV regions.
Title: Sequence information of the primers for genomic qPCR validation

Title: General characteristics of the copy number variation (CNV) calls from 3 algorithms

Title: Number of copy number variations (CNVs) consistently defined from 3 different algorithms

Title: Genomic qPCR validation of the CNV calls from SNP genotyping data
Title: Comparison of SNP Variation and Distribution in Indigenous Ethiopian and Korean Cattle (Hanwoo) Populations

Although a large number of single nucleotide polymorphisms (SNPs) have been identified from the bovine genome-sequencing project, few of these have been validated at large in Bos indicus breeds.
We have genotyped 192 animals, representing 5 cattle populations of Ethiopia, with the Illumina Bovine 8K SNP BeadChip.
These include 1 Sanga (Danakil), 3 zebu (Borana, Arsi and Ambo), and 1 zebu × Sanga intermediate (Horro) breeds.
The Hanwoo (Bos taurus) was included for comparison purposes.
Analysis of 7,045 SNP markers revealed that the mean minor allele frequency (MAF) was 0.23, 0.22, 0.21, 0.21, 0.23, and 0.29 for Ambo, Arsi, Borana, Danakil, Horro, and Hanwoo, respectively.
Significant differences of MAF were observed between the indigenous Ethiopian cattle populations and Hanwoo breed (p &lt; 0.001).
Across the Ethiopian cattle populations, a common variant MAF (≥0.10 and ≤0.5) accounted for an overall estimated 73.79% of the 7,045 SNPs.
The Hanwoo displayed a higher proportion of common variant SNPs (90%).
Investigation within Ethiopian cattle populations showed that on average, 16.64% of the markers were monomorphic, but in the Hanwoo breed, only 6% of the markers were monomorphic.
Across the sampled Ethiopian cattle populations, the mean observed and expected heterozygosities were 0.314 and 0.313, respectively.
The level of SNP variation identified in this particular study highlights that these markers can be potentially used for genetic studies in African cattle breeds.
Ethioipian cattle breeds are the core to sustain the livelihoods of most of the farming and pastoral communities.
Indigenous breeds are known for their better adaptation for low input systems, heat tolerance, disease resistance, local preferences, and cultural and aesthetic values.
The unique genetic background of Ethiopian cattle and their adaptation to the low input system support their conservation and breed development.
However, common for developing countries, the genetic diversity and the genetic merits of most Ethiopian indigenous cattle populations are not yet well understood and exploited.
Consequently, some of the indigenous cattle populations are already extinct and endangered, while the risk status of many of them is unknown.
A recent microsatellite based study revealed that most Ethiopian cattle breeds are under threat from uncontrolled breeding practices and are at high risk of becoming genetically homogenous [1].
Studies on genetic diversity of Ethiopian cattle have focused on limited cattle breeds, and all of these studies employed low-density microsatellite, mitochondrial, or Y-chromosome markers [1, 2].
A wide range of molecular markers has been developed and used for molecular characterization of populations.
Among these, single nucleotides polymorphisms (SNPs) are the most abundant molecular markers in the genome.
They are widely dispersed throughout genomes, with a variable distribution among species; genotyping results are easy to reproduce in different laboratories and are simple to score.
The usefulness of SNPs in population diversity and structure studies has been demonstrated in several studies in livestock species [3-6].
Although a large number of SNPs have been identified from the bovine genome-sequencing project, few of these have been validated at large in Bos indicus breeds.
Yet, the level of polymorphism in Ethiopian cattle population has not been known and analyzed.
Prior to this study, no SNPs have been described for these populations.
Hence, we report the first preliminary findings on SNP variation in Ethiopian indigenous cattle populations and Hanwoo breed.
This study is part of the collaborative project between the Korean Government, the Ethiopian Institute of Agricultural Research, and the International Livestock Research Institute, designed with the objectives to estimate genetic diversity between and within populations using SNP markers.
We also evaluated the level of SNP polymorphisms in selected Ethiopian cattle populations and compared to the Hanwoo cattle breed.
Title: Sampling and DNA sample collection

Nasal samples were collected using Performagene LIVESTOCK&#39;s nasal swab DNA collection kit (DNA Genotek Inc., Kanata, ON, Canada) from 5 cattle populations-Ambo, Borana, Arsi, Horro, and Danakil-inhabiting different agro-ecologies.
Sampling was carried out by considering potential geo-environmental gradients (highland, low-lands), production systems (mixed-crop livestock, pastoral/agro-pastoral), and ethnic groups raising predominantly the populations (Table 1, Fig.1).
Representative sites were selected based upon secondary data obtained from livestock departments and expertise.
Herdsmen and owners of the animals were contacted to ensure that the animals were not genetically related.
Animals were randomly sampled from multiple herds.
Genomic DNA was extracted from nasal samples based on laboratory protocols.
In addition, DNA samples from Hanwoo cattle were included for comparison purposes.
Title: Breeds/populations descriptions

The Danakil cattle breed is grouped under Sanga and originated in East and Northeast Africa, where Sanga cattle evolved.
The Horro breed was formed as a result of interbreeding between zebu-Sanga and Sanga-zebu admixture populations.
The large-sized Borana breed descended from secondary cattle domestication in the arid areas of the &#39;Fertile Crescent&#39; about 5000 BP [7].
The breed is predominantly distributed in arid and semi-arid agro-ecologies of the Borana plateau.
The small-sized Arsi cattle probably developed from a group of small short-horned Abyssinian zebu by the highland Oromo people.
The Ambo cattle is grouped under Small East African zebu and is believed to have descended from the recent introductions of zebu into Africa [8].
Pictures of sampled animals are depicted in Fig.2.
Title: Genotyping and quality control

In total, 192 animals representing 5 cattle populations of Ethiopia were genotyped for 8,773 SNPs with the Illumina Bovine 8K SNP BeadChip (Illumina, San Diego, CA, USA) according to Illumina protocols [9].
Over the loci, the average GenTrain score, 10% GenCall (10% GC) score, and 50% GenCall (50% GC) score were 0.86, 0.87, and 0.83, respectively.
In this experiment, about 99.91% of the markers identified had a GenTrain score greater than the minimum acceptable value (0.25) (http://www.illumina.com).
All of the samples had at least a 0.20 or greater 10% GC score, while the 50% GC score ranged from 0.75 to 0.91, with the average call rate of 0.913.
For Hanwoo cattle, about 51,162 SNPs were genotyped using the Illumina Bovine 50K SNP BeadChip [10].
8K and 50K genotypes were merged using Golden Helix SNP Variation Suite software version 7 (Golden Helix, Inc., Bozeman, MT, USA).
To ensure the highest quality of data from the 8K and 50K SNP BeadChips, duplicate marker genotypes were identified and dropped.
Finally about 7,045 SNPs that were common for both Bead-Chips were subjected to analysis.
Markers selected for diversity analysis were all located on autosomal chromosomesand had at least an 80% call rate.
Title: Data analysis

Frequency of minor allele frequency (MAF), distribution, and deviation from Hardy-Weinberg equilibrium (HWE) were estimated using Golden Helix SNP Variation Suite version 7 (Golden Helix, Inc.).
The SAS GLM package version 9.1 (SAS Institute Inc., Cary, NC, USA) procedure was used to test the effect of breed on MAF variation.
Observed and expected heterozygosities and hierarchical F-statistics were estimated using GENALEX software [11].
Title: MAF variation

The MAF for each SNP was calculated from the genotypic data and is presented in Table 2.
Analysis of 7,045 markers revealed that the mean MAF was 0.23, 0.22, 0.21, 0.21, 0.23, and 0.29 for Ambo, Arsi, Borana, Danakil, Horro, and Hanwoo cattle, respectively.
A significant difference was observed between the Ethiopian indigenous cattle populations and Hanwoo (p &lt; 0.001).
The average value for Hanwoo was 32% higher than the value for Ethiopian cattle populations.
The overall MAF in the present finding for indigenous Ethiopian cattle breeds is higher than an earlier report for indicine breeds but lower than the average value reported for most taurine breeds [3].
The value found for Hanwoo cattle was comparable with the previous results for taurine breeds but higher than the reported average for the same breed [10, 12].
This could be attributed to the different marker density used, and a higher density marker (Illumina Bovine 50K SNP BeadChip) was used in the previous study.
The higher values for taurine breeds can be explained by the fact that SNP loci used in this study were detected in taurine breeds, and their average minor allele frequency was much lower in indicine breeds.
Across the Ethiopian cattle populations, common variant MAFs (≥0.10 and ≤0.5) accounted for 74% of the total SNPs (Table 3).
Hanwoo cattle displayed a higher proportion of common variant SNPs (90%).
Rare variant MAFs were observed in about 9% and 4% of markers in the Ethiopian and Hanwoo breeds, respectively.
The relatively higher proportion of fixed alleles in Ethiopia cattle populations could be attributed to the inbreeding as a result of uncontrolled breeding practices.
A common variant MAF proportion of 83% was reported in sheep breeds [5].
The distribution of MAFs revealed that nearly 34.08% of the markers in Ethiopian cattle populations and 53% in Hanwoo cattle displayed a high degree of polymorphisms (0.3 ≤ MAF ≤ 0.5).
Yet, this value was higher than an earlier report both for Hanwoo (25%) and Holstein (28%) [12].
Much higher MAFs were detected in Bos taurus breeds than in Bos indicus breeds [3].
The proportion of fixed markers varied among the sampled populations.
Among the Ethiopian cattle populations, the highest (8.96%) and lowest (6.30%) were obtained in the Borana and Ambo populations, respectively.
The Hanwoo breed displayed a lower level of fixed SNPs (1.10%).
On average, 83.36% of the markers displayed polymorphisms (MAF &gt; 0.05) within Ethiopian cattle populations, and the remaining 16.64% of the SNPs considered as monomorphic.
The Hanwoo breed showed the highest proportion of polymorphisms (94%) (Fig.3).
The level of polymorphic SNPs in the present study was higher than what was previously reported for taurine breeds [10].
But, it was in close agreement with the value reported for sheep breeds using genome-wide SNPs [5].
Similarly, Gautier et al.
[13] reported a lower rate of polymorphisms using 696 SNPs localized in BTA03 from European and West African cattle breeds (94.6% and 81.8%), respectively.
The higher observed polymorphisms in Hanwoo cattle could explained by the fact that most of the bovine sequence data available in the database are from Bos taurus breeds.
Title: Genetic diversity among Ethiopian cattle populations

An examination of the variability within breeds was carried out to compare the level of hererozygosity between the populations, and the results are presented in Table 4.
Across the sampled Ethiopian cattle populations, the mean observed and expected heterozygosities were 0.314 ± 0.001 and 0.313 ± 0.001, respectively.
The mean values were lower than the values reported from a microsatellite markers analysis [1].
However, they were in concurrence with the values reported using SNPs for the Angus breed (0.332) [14] and for sheep breeds [5].
The overall estimated F-statistics were FIS = -0.003 (within-population inbreeding estimates) and FIT = 0.016 (total inbreeding).
Among the Ethiopian cattle populations, genetic differentiation (FST) was estimated at 0.019.
Genetic characterization of Ethiopian cattle breeds using microsatellites reported within-population inbreeding of 0.071, total inbreeding of 0.083, and genetic differentiation of 0.013 [1].
The FST for Ethiopian cattle populations was low, which could be due to their common ancestral origin.
There were loci significantly deviating from HWE.
Overall, significant deviation from Hardy-Weinberg proportions (p &lt; 0.05) was observed for 405 markers in Ethiopian cattle populations (Ambo, 441; Arsi, 475; Borana, 314; Danakil, 314; and Horro, 481) and in 549 of the markers in Hanwoo.
The highest number of SNPs that departed from HWE was observed in Hanwoo cattle, whereas within the Ethiopia cattle populations, Borana displayed lower proportions of markers (SNPs) deviating from HWE.
The variation in proportion of SNPs displaying deviation from HWE among the sampled populations could be explained by selection pressure and population structure (stratification, admixture).
In conclusion, a significant difference was detected for the rate of polymorphisms and MAFs between the Hanwoo and the Ethiopian cattle populations.
The level of variation identified in this particular study highlights that these markers can be potentially used for genetic studies in African cattle populations.
The high within-breed diversity observed in Ethiopian indigenous cattle populations could be an opportunity and allow improvements through within-population selection schemes.
Map of Ethiopia showing location of sampled populations.
Pictures of sampled Ethiopian cattle populations.
(A) Danakil adult female.
(B) Horro adult female.
(C) Borana adult female.
(D) Arsi adult female.
(E) Ambo adult female.
(F) Horro adult male.
(G) Borana adult males.
(H) Arsi adult male.
(I) Ambo adult male.
Polymorphic and monomorphic dynamics of single nucleotide polymorphism (SNPs).
Title: Summary of sampled populations and their characteristics with respect to breed group, distribution over ecological zones, and affinity to ethnic communities

Title: Mean MAF values in indigenous Ethiopian cattle populations and in Korea Hanwoo cattle

Title: Minor allele frequency distribution of 8K BeadChip SNP in Bos taurus and in Ethiopian cattle breeds

Title: Observed (HO) and expected heterozygosities (HE), mean number of different alleles (Na), mean number of effective alleles (Ne), Shannon&#39;s information index (I) and fixation index (F) in indigenous Ethiopian and Korean cattle populations
Title: DNA Barcoding of Fish, Insects, and Shellfish in Korea

DNA barcoding has been widely used in species identification and biodiversity research.
A short fragment of the mitochondrial cytochrome c oxidase subunit I (COI) sequence serves as a DNA bio-barcode.
We collected DNA barcodes, based on COI sequences from 156 species (529 sequences) of fish, insects, and shellfish.
We present results on phylogenetic relationships to assess biodiversity the in the Korean peninsula.
Average GC% contents of the 68 fish species (46.9%), the 59 shellfish species (38.0%), and the 29 insect species (33.2%) are reported.
Using the Kimura 2 parameter in all possible pairwise comparisons, the average interspecific distances were compared with the average intraspecific distances in fish (3.22 vs. 0.41), insects (2.06 vs. 0.25), and shellfish (3.58 vs. 0.14).
Our results confirm that distance-based DNA barcoding provides sufficient information to identify and delineate fish, insect, and shellfish species by means of all possible pairwise comparisons.
These results also confirm that the development of an effective molecular barcode identification system is possible.
All DNA barcode sequences collected from our study will be useful for the interpretation of species-level identification and community-level patterns in fish, insects, and shellfish in Korea, although at the species level, the rate of correct identification in a diversified environment might be low.
DNA barcoding is a simple and useful step toward understanding the ecosystem.
It also serves to further our interests in biodiversity research [1].
A short standardized sequence (400-800 bp) of DNA can be used to distinguish individuals of a species.
This approach was taken, because genetic diversity between species is markedly greater than that within species [2].
Numerous computational analysis methods and systems have been introduced for this purpose [3-5].
The use of this system can provide rapid, accurate, cost-effective, and automatable process for species identification.
The success rate of each barcoding application varies significantly among groups.
Moreover, global datasets that represent extensive ecosystems are expected to be subjected to particular difficulties, especially in groups in which recent speciation rates are high and effective population sizes are large and reasonably stationary [6].
Several studies of species-level identification have covered many groups of organisms, including birds, fishes, and various arthropods [4, 6-8].
In order to use the barcoding system for species identification, cytochrome c oxidase subunit I (COI) sequences were obtained in this study from 529 sequences, representing 156 species from fish, insects, and shellfish in the Korean peninsula.
The first community-level barcoding studies were conducted in the most diverse terrestrial and marine ecosystems in an inland and coastal area of South Korea (include reference).
We collected samples to obtain an overview of the variation patterns for 529 COI sequences among 68 fish species, 29 insect species, and 59 shellfish species.
Multiple specimens were collected for most of the species.
Fish and shellfish were collected from Yeosu in Jeollanam-do; shellfish were collected from Taean; and insects were collected from Chungcheongnam-do, Gangwon-do, Gyeongsangbuk-do, and Jeollabuk-do in South Korea.
Samples were collected using different, technically appropriate methods (Fig.1, Supplementary Table 1) [9].
If possible, the samples were obtained from widely distributed places in South Korea.
Genomic DNA was isolated from samples using the Qiagen DNeasy 96 blood and tissue kit (Qiagen, Valencia, CA, USA) according to the instructions.
DNA fragments of target genes were amplified by polymerase chain reaction (PCR) with primers for the COI gene (primer sequences: LCO1490 GGTCAACAAATCATAAAGATATTGG and HCO 2198 TAAACTTCAGGGTGACCAAAAAATCA) [10].
PCR amplification was performed using Top-Taq PreMix (2×; CoreBio, Seoul, Korea) under the following conditions: denaturation (1 min at 94℃), annealing at 51℃ for amplification of the COI gene, and extension (2 min at 72℃).
PCR products were purified with the Core-One PCR purification kit (CoreBio), and TA cloning was performed using the pGEM-T Easy Vector system (Promega, Madison, WI, USA) by Macrogen Inc.
The clones for each marker were sequenced with forward (SP6) and reverse (T7) primers using an ABI 3730XL sequencer (Applied Biosystems, Foster City, CA, USA).
The sequences reported in this paper have been deposited in GenBank under accession numbers HM180413-HM180941.
To obtain the species information for each operational taxonomic unit (OTU) in a phylogenetic tree, a BLAST search was performed using the BLASTN program from NCBI [11].
A cutoff value for the BLAST result was established as follows: query coverage &gt; 90% and identity &gt; 75% for COI.
The levels of sequence divergence within and between the selected species were investigated using the pairwise Kimura 2 parameter (K2P) distance model [12].
The neighbor-joining tree, with gap positions ignored on a pairwise basis, was constructed using the neighbor-joining (NJ) method with K2P distances in MEGA4 [13].
These distances were hierarchically arranged in accordance with intraspecific and interspecific species differences within each genus.
When the sequence dataset consisted of only 2 genera from the same family, an intergeneric comparison within the family was not performed.
After BLASTN annotation analyses were conducted, K2P distances were compared at different taxonomic levels, revealing distinct features in the sequences both within and between species.
With respect to the COI sequences of the 156 species represented, the interspecific K2P distances for the COI sequences from the 68 fish species, the 59 shellfish species, and the 29 insect species ranged from 0% to 45.25% (fish, 0% to 40.99%; insects, 0% to 10.34%; shellfish, 0% to 45.25%) (Fig.2A), whereas the intraspecific K2P distances with ≥3 sequences ranged from 0% to 0.985% (fish, 0% to 0.985%; insects, 0.005% to 0.635%; shellfish, 0% to 0.817%) (Fig.2B).
The average interspecific distances and average intraspecific distances were, respectively, 3.58 and 0.14 in shellfish, 3.22 and 0.41 in fish, and 2.06 and 0.25 in insects (Table 1).
In shellfish, the greatest interspecific K2P differences were 25.57-fold higher than the intraspecific values.
The overall base composition in each species of fish, insect, and shellfish was as follows: T (thymine) ranged from 27.4% to 33.7% (highly abundant); G (guanine) ranged from 16.8% to 21.5% (not highly abundant) (Table 1).
These findings for fish were consistent with previous studies showing that T occurred more frequently and G occurred less frequently than A (adenine) and C (cytosine) [8].
In our polytypic species analysis with more than 3 individuals in each species, the average intraspecific difference was approximately 0.5%, and the maximum intraspecific divergence was only 1.86% (Table 2).
The highest overall GC% content was found in the 18 species of fish.
Lower values were found in the 2 species of insects and in the 6 species of shellfish (Table 2).
The fish Chelidonichthys spinosus had a high GC% content of 50.9%.
The mean GC% content of the 18 barcoded fish species was higher than that of the 6 shellfish species (46.9 ± 2.2% vs. 38.0 ± 4.9%) (see also Table 2).
Sixteen of the 21 species with GC% content ≥45% were fish, whereas only 1 shellfish species exhibited GC% content ≥45%.
The GC% content can be used in a new approach to evaluate animal evolutionary relationships, although the relationship between GC% content and the evolutionary branching date is not very accurate [14].
Moreover, the average divergence of congeneric species pairs was greater than that found for intraspecific differences, but 10 species in 5 genera had interspecific distances below 0.1% (Table 3).
These species included Hexagrammos agrammus/H.
otakii, Ampedus humeralis/A.
subcostatus, Anomala luculenta/A.
mongolica, Chlorostoma argyrostoma turbinatum/C.
turbinate, and Omphalius rusticus rusticus/O.
pfeifferi carpenteri.
In addition, the NJ tree exhibited shallow interspecific divergence except at the first deep divergence (Fig.3).
In fish, several clades had a high level of bootstrap support (≥97%) (Fig.3A).
These clades included Thrysa chefuensis and T. adelae, Hexagrammos otakii and H. agrammus.
In insects, the clades that had a high level of bootstrap support (≥95%) included Fusinus forceps, F. longicaudus, Mytilus galloprovincialis, and M. edulis.
In shellfish, 2 clades separated out with a high level of bootstrap support (≥99%) (Fig.3B).
These clades included Anomala mongolica and A. luculenta, Ampedus humeralis and A. subcostatus (Fig.3C).
In conclusion, we obtained DNA barcodes using COI sequences from fish, insects, and shellfish.
The aims of this research were species identification and contribution to biodiversity research.
At the species level, the rate of correct identifications might be low in a diversified environment.
However, DNA barcoded sequences can be used for the interpretation of species-level identification and community-level patterns in fish, insects, and shellfish.
Map showing the locations of the cruises and the materials collected in this study.
Each circle represents one sampling locality, and circle size is proportional to the number of samples in our study.
Google Map was used (http://maps.google.co.kr) [9].
Distribution of interspecific Kimura 2 parameter (K2P) distances for cytochrome c oxidase subunit I (COI) sequences from the 68 fish species, the 59 shellfish species, and the 29 insect species.
Vertical lines show the mean pairwise distance at each level.
The X- and Y-axes represent K2P distance values and the percentage of individuals, respectively.
(A) Interspecific K2P distances.
(B) Intraspecific K2P distances.
The neighbor-joining tree of fish, insects, and shellfish based on cytochrome c oxidase subunit I (COI) sequences.
(A) Fish.
(B) Insects.
(C) Shellfish.
Title: Mean percentage base composition, comparing COI sequences and K2P distance among fish, insects, and shellfish

Title: Maximum intraspecific distance and GC% content among fish, insects, and shellfish (threshold &gt; 0.5%)

Title: Maximum Kimura 2 parameter (K2P) distances with congeneric species pairs
Title: Editor&#39;s Introduction to This Issue

This issue features a number of review articles that cover many topics, from the fundamental aspect of our genome and transcriptome to the application of genomics for understanding diseases.
Profs.
Goh and Choi at Yale University, USA, review an emerging technology called exome sequencing and its application in identifying disease-causing variants.
While it focuses mainly on inherited diseases, complex diseases are also touched upon, and the common challenges ahead are discussed.
Prof. Ikegawa at RIKEN, Japan, gives a historical overview of genome-wide association studies (GWASs) and discusses their future prospects in a narrative tone.
Prof. Kyong Soo Park&#39;s group at Seoul National University, Korea, gives a more specific account of the application of GWASs in assessing genetic risk factors of gestational diabetes.
My group at Soongsil University, Korea, also contributes a review on expression quantitative trait loci and their role in predisposing disease susceptibility.
Recently, transposable elements (TEs) have been attracting attention, as they are not &#39;junk&#39; anymore, and play important roles in gene regulation and genomic evolution.
Prof. Kyudong Han&#39;s group at Dankook University, Korea, focuses on a few types of TEs that are specific to primate genomes.
Besides these review articles, we have several interesting original articles and application notes, as well as a preliminary research communication.
For example, Prof. Taesung Park&#39;s group at Seoul National University, Korea, reports a novel analysis of gene-gene interactions in GWAS data using a network graph method.
While calculating the gene-gene interactions is a difficult task, interpreting such results is an even more daunting task.
Their new method facilitates such a process, as demonstrated with success in its application to the study of body mass index.
Genomics &amp; Informatics is now available in PubMed Central (PMC), a full-text service, and PubMed, PMC&#39;s sister abstract service.
We expect more broad international visibility of Genomics &amp; Informatics through these services.
Our next immediate goal is the coverage by SCOPUS, one of the renowned citation databases.
We plan to apply for inclusion into SCOPUS in the middle of 2013.I hope you enjoy this year&#39;s final issue of Genomics &amp; Informatics your continued support is greatly appreciated.
Title: Application of Whole Exome Sequencing to Identify Disease-Causing Variants in Inherited Human Diseases

The recent advent of next-generation sequencing technologies has dramatically changed the nature of biomedical research.
Human genetics is no exception-it has never been easier to interrogate human patient genomes at the nucleotide level to identify disease-associated variants.
To further facilitate the efficiency of this approach, whole exome sequencing (WES) was first developed in 2009.
Over the past three years, multiple groups have demonstrated the power of WES through robust disease-associated variant discoveries across a diverse spectrum of human diseases.
Here, we review the application of WES to different types of inherited human diseases and discuss analytical challenges and possible solutions, with the aim of providing a practical guide for the effective use of this technology.
Whole exome sequencing (WES) is a technique to selectively capture and sequence the coding regions of all annotated protein-coding genes.
Coupled with next-generation sequencing (NGS) platforms, it enables the analysis of functional regions of the human genome with unprecedented efficiency.
Since its first reported application [1, 2], WES has emerged as a powerful and popular tool for researchers elucidating genetic variants underlying human diseases (Fig.1) despite certain limitations (see below).
WES pipelines generally follow a similar process, regardless of the capture method and NGS platform used, as summarized in Fig.2.
The experimental pipeline can be divided into two parts: 1) preparing genomic DNA libraries and hybridizing them to capture arrays and 2) NGS of the eluted target fragments.
There are a number of commercially available capture arrays, and their strengths and weaknesses have been well described elsewhere [3].
Once short sequencing reads have been generated, they are mapped to the reference human genome, and variant calling is carried out.
Subsequent annotation of these variants is necessary to further evaluate their potential biological effect; publicly available software and databases can be used for this purpose.
WES is a robust technology that is extremely practical for investigating coding variation at the genome-wide level.
Despite the plummeting cost of sequencing in the past few years, WES at mean coverage depth of 100× still costs five times less than whole genome sequencing (WGS) at mean coverage depth 30×.
In addition, the size of WES data per patient is approximately a sixth of WGS data, resulting in reduced processing time and imposing less of a burden in terms of data storage.
However, it is important to understand the limitations of WES technology.
Certain protein-coding regions might not be covered due to incomplete annotation of the human genome.
Further, WES does not cover potentially functional non-coding elements, including untranslated regions, enhancers, and long-noncoding RNAs, although these are, in themselves, not clearly defined.
Another drawback to WES is the limited ability to detect structural variations, such as copy-number variations, translocations, and inversions.
In spite of these limitations, WES is still the tool of choice for many researchers-its practical advantages allow large numbers of patients to be screened in a robust fashion, a crucial aspect of mutation discovery in human genetics research.
WES has been used in identifying genetic variants associated with a variety of diseases.
Here, we discuss different categories of inherited human disease and the methodology employed to examine each case.
Diseases inherited in a recessive pattern have traditionally been highly amenable to genetic analysis, due to the fact that homozygous variants are easily detectable.
Previously, if a large family with many affected members was available for pedigree analysis, one would perform linkage analysis by genotyping family members in order to identify relatively short genomic intervals that presumably contained the disease-associated variant.
Direct interrogation of entire genomes for homozygous variants is now possible with NGS technologies, and public datasets can be used to exclude common variants that are less probable to be disease-causing.
Identifying recessive variants is especially straightforward if the proband is a product of a consanguineous marriage-sequencing the entire exome of the patient would likely yield a manageable number of rare homozygous variants, and further studies could be performed to investigate the functional relevance of these variants.
In one of the earliest reports of the application of WES technology, Bilgüvar et al.
[4] identified recessive mutations in WDR62 in multiple Turkish consanguineous patients with severe developmental brain defects.
Very little was known about WDR62 at that time, but subsequent genetic and functional studies have since validated the importance of this gene in proper brain development.
More recently, a large cohort of patients of similar genetic status and clinical manifestation were analyzed using homozygosity mapping from WES data [5].
The authors uncovered 22 genes not previously identified as disease-causing, further demonstrating the power of WES as an ideal tool for gene discovery.
Mendelian diseases with dominant modes of inheritance pose greater technical challenges for genetic analysis.
Heterozygous variants are generally more difficult to detect and analyze, not only due to their sheer number as compared to homozygous variants but also because they are subjected to higher false positive and false negative errors.
Nevertheless, one could still design one of the following experiments to elucidate the genetic architecture underlying these diseases, depending on the availability of patient cohorts and the effect of the disease on reproductive fitness.
Title: Familial cohort

If large families with the disease of interest are available, WES should be performed on at least two affected family members.
In analyzing the results, any shared variants could potentially be disease-causing; using linkage data to focus on the variants located in linked intervals would greatly reduce the number of variants to be considered.
A recent example of such an approach is a WES study on familial amyotrophic lateral sclerosis (FALS) [6].
The authors recruited two large families with FALS and selected two affected members with the greatest genetic distance from each family for WES.
Among the rare functional variants that were identified, they tested the shared variants for Mendelian segregation among affected family members, eventually narrowing their search down to a single common gene, PFN1.
Title: Non-familial cohort

When large families with the disease are not readily available, one could perform WES on a number of unrelated patients with similar clinical manifestations and select genes that are commonly mutated in the patient cohort.
In carrying out such an experiment, it is important to consider the background mutation burden of the entire human gene set; it is possible that the longest genes might contain the highest number of rare functional variants and therefore appear to be the most interesting.
To circumvent such errors in analysis, the variant burden of each gene should be normalized using data from a control population.
In a WES study of pseudohypoaldosteronism type II (PHAII), the variant burden of every gene from 11 unrelated patients was compared against that of 699 controls [7].
A single gene that was specifically enriched for mutations in the patient set, KLHL3, was identified (5 variants from 11 patients compared to 2 from 699 controls, with p-value of 1.1 × 10-8).
Title: Non-inheritable diseases

If the disease phenotype is severe enough such that it affects reproductive fitness, one could hypothesize that de novo variants-variants that emerged during meiosis of the germ cells that are unique to the offspring-are directly associated with the disease.
These can be detected by performing trio-based WES on the unaffected parents and the proband, and variants that are present in the offspring only and not in the parents will be called de novo variants.
The rate of de novo mutations has been shown to be relatively stable and affected primarily by paternal age [8].
WES of family quartets can also be carried out, where unaffected siblings are recruited simultaneously and their de novo mutations are compared against those of the proband.
A series of studies involving autism trios and quartets recently demonstrated that rare de novo mutations were associated with the risk of autism and identified multiple de novo mutations in SCN2A, KATNAL2, and CHD8, implicating these genes in the genetic etiology underlying autism [9-11].
The majority of the disease burden that modern societies endure can be attributed to common complex diseases.
These diseases typically have both genetic and environmental causes, and they also possess significant genetic heterogeneity, making the identification of disease-causing genes very challenging.
Traditionally, the use of linkage analysis of families with extreme disease phenotypes has been somewhat successful in mutation discovery [12].
However, these mutations are particular to patients with extreme phenotypes and do not explain the pathophysiology of disease for patients in the general population.
Large genome-wide association studies of common variants have also been employed with limited success; the findings in many studies are not robust and only explain a small fraction of the disease risk [13].
As a result, greater attention has been drawn to profiling rare variants with allele frequencies of less than 1% in complex diseases.
Taking advantage of WES technology, several large-scale projects have been launched.
One such example is a project led by National Heart, Lung, and Blood Institute (NHLBI) to discover novel genes underlying cardiovascular disorders.
As a result of sequencing the exomes of 2,440 individuals, the study reported an excess of rare functional variants and concluded that large numbers of subjects will be required to attain sufficient power to discover variants that are significantly associated with the disease traits [14].
WES yields extensive lists of genomic variation, and there are many important caveats to bear in mind when carrying out analysis of such data.
In this section, we address the potential challenges that the investigators need to consider when designing or performing the experiments.
Recent improvements of the WES technique in both experimental and bioinformatics pipelines have reduced the occurrence of false positive (false variant that is called true) and false negative (true variant that is failed to be called) variant calls substantially.
Currently, we can achieve ~98% sensitivity and 99.8% specificity from exome sequencing data, minimizing the chance of missing any true variants as compared to the analysis of SNP array data (data not shown).
However, as mentioned previously, it is technically challenging to detect and evaluate rare heterozygous variants because heterozygous variant calling is more susceptible to the technical errors and require higher read depth.
For example, at a coverage depth of 4× (i.e., if a base is covered by 4 independent reads) and assuming a 1% per-base error rate, a homozygous variant will be called if 0 and 4 reads are observed for the reference and nonreference alleles respectively, with a false positive rate of 2 × 10-4.
However, if both reference and nonreference alleles have 2× coverage, a heterozygous variant will be called with a false positive rate of 0.34.
This presents a particular problem when one considers the uneven coverage in WES resulting from differential capture efficiencies across the exome.
To achieve reliable heterozygous calling, it is therefore necessary to achieve sufficient coverage depth across the entire exome so as to minimize the number of bases with low coverage.
Another challenge to analysis is read misalignments; the typical length of NGS reads is near or less than 100 bp, and even paired reads are subject to being improperly aligned, because there are large portions of the human genome where the DNA sequence is highly repetitive and duplicated.
In fact, segmental duplicated regions, defined as intervals larger than 1 kb having a homology &gt;90% with other parts of the genome, encompass about 5% of the human genome [15].
Hence, imposing more rigorous filtering criteria to remove such reads, including Phred score and mapping quality score cutoffs, is essential.
The recent, dramatic acceleration in human population growth over the last several millennia has resulted in an excess of rare variants in the human genome, and these variants have not had enough time to be subjected to natural selection [14].
Since the identification of disease-associated variants consists largely of assessing rare variants, one must bear in mind the possibility that a variant of interest could be population-specific and not necessarily disease-causing.
This is especially problematic if the patient cohort is not of European or, more specifically, northwestern European descent.
Recent public sequencing projects, such as the 1000 Genomes Project and NHLBI Go Exome Sequencing Project, have covered a greater variety of non-European ethnic groups than before, and they report that these groups harbor a greater burden of rare variants as compared to Europeans [14, 16].
For example, indigenous African individuals may carry 2-3 times more rare variants, and East Asian individuals may carry 1.5 times more variants when compared to individuals of European descent.
Rare variants specific to the population of study might be easily mistaken as patient-enriched variants; it is thus necessary to screen the presumed variant with carefully matched healthy individuals.
Locus heterogeneity describes the phenomenon where a single disease can be caused by multiple different loci across different patients.
Over the last few decades, studies performing linkage analysis followed by positional cloning have identified many genes as being responsible for various inherited diseases; the Online Mendelian Inheritance in Man (OMIM) database currently records 3,000 genes as being disease-causing (http://www.ncbi.nlm.nih.gov/omim).
This also suggests that the majority of the remaining diseases without known associations with genes have not been amenable to classical approaches and will likely display extensive genetic heterogeneity.
Increasing the size of the patient cohort will increase the power to discover variants, as shown in Fig.3.
Pathway or gene ontology analyses can also be performed to link the mutated genes into certain functional categories-there are several publicly available tools for this purpose.
A good example of this method is the aforementioned study on PHAII-the authors initially identified KLHL3 from WES of 11 patients and subsequently considered KLHL3&#39;s presumed functional partner, CUL3, as a potential gene candidate.
Extending the screening to 52 patients, KLHL3 was found to be mutated in 24 patients (46.2%) and CUL3 in another 17 patients (32.7%), explaining a total of 78.8% of the entire patient set.
Another possible approach is to re-evaluate the clinical diagnosis of patients in the cohort by correlating subtle differences of clinical measurements and genetic variants in order to focus the analysis on a more clinically homogeneous set of patients as a separate cohort.
There is no doubt that NGS has provided researchers with unprecedented power to resolve the genetic etiology of various human diseases.
When WES was first introduced, its utility was highly debated due to its apparent limitations, such as incomplete coverage of functional elements and low sensitivity for structural variant detection.
However, the practical advantages of the technology have made it a favored tool for researchers, and WES will likely continue to be widely used for the foreseeable future.
Furthermore, as library capture methods and data analysis pipelines improve, increasing amounts of genomic information, aside from single-nucleotide changes and short indels, can be extracted.
Examples of this include homozygosity interval mapping, common SNP genotyping information for various population-level analyses, and detection of structural variants [17, 18].
Finally, the strengths of WES-short turnaround times, low cost, and relatively easy data interpretation-make it an optimal tool for clinical diagnosis.
The increased use of WES in the clinic will surely spur the development of personalized medicine and reinvent treatment practices in the near future [1, 19].
Increasing number of whole exome sequencing papers over time.
Overview of whole exome sequencing pipeline.
SNV, single nucleotide variant.
Increased sensitivity to detect causal gene as patient cohort size increases.
Percentages shown denote ratio of patients that carry rare functional variants.
Simulation was performed assuming that 0.1% of healthy controls carry rare functional variants, and was iterated 105 times.
Each iteration was evaluated and called &#39;detected&#39; if the p-value exceeded the genome-wide significance.
Title: A Short History of the Genome-Wide Association Study: Where We Were and Where We Are Going

Recent rapid advances in genetic research are ushering us into the genome sequence era, where an individual&#39;s genome information is utilized for clinical practice.
The most spectacular results of the human genome study have been provided by genome-wide association studies (GWASs).
This is a review of the history of GWASs as related to my work.
Further efforts are necessary to make full use of its potential power to medicine.
We are in the genome sequence era.
Now, we can all read our genome sequences if we have a bit of time and money.
We can know our own program of life given by our fathers and mothers, which consists of 3,000,000,000 bp × 2 of DNA sequence.
This knowledge (knowing ourselves) is revolutionizing medicine.
The information gives us endless possibilities and a huge number of applications in research and clinical practice of medicine.
Preventive medicine, personalized medicine, pharmacogenomics, and pharmacogenetics-these were just dreams on the desk or in the brain of medical scientists and doctors a few years ago but are all achievable at the bedside today.
The best application of the information obtained by genome sequences is an attempt to conquer common diseases or lifestyle-associated diseases.
Common diseases are our common problem.
It is a serious medical, social, and economical problem all over the world.
It&#39;s a problem of the present and the future world.
The study of our genome has given us a tool to tackle it-the genome-wide association study (GWAS), which is now in full bloom.
As we all know, GWASs have succeeded in the identification of susceptibility genes of many common diseases [1].
For more than 15 years, I have been working on genomic studies and committed to the GWAS from its dawn.
In this paper, I briefly review a history of human genome research by focusing on GWASs related to my work.
Title: Where were you at the start of the new century?
What were you doing at that time?
I was in Japan, in Tokyo, in the SNP Research Center (SRC).
SRC was founded as a part of the Japanese Millennium Project to study single-nucleotide polymorphisms (SNPs).
The Millennium Project is a broad effort to get industry, academia, and government to cooperate on a wide range of R&amp;D projects to revitalize the competitiveness of the industry and to energize Japan&#39;s economy.
The project focused on developing technologies for three main target areas of vital importance to Japan: informatics, ecology, and an aging society.
A variety of large-scale human genome studies were done as projects to revolutionize medicine and cope with the aging society.
In April 2000, SRC started collaboration with the Human Genome Center (HGC), Institute of Medical Science (IMS), University of Tokyo, and the Japan Science and Technology Agency (JST).
Its mission was to identify up to 150,000 SNPs throughout the human genome within two years, to make the information available to the public, and to develop analytical tools for polymorphisms (note: SRC was renamed Center for Genomic Medicine [CGM] in 2008 [http://www.src.riken.jp/]).
SNPs are the most common form of DNA sequence variation.
They are the most useful and convenient polymorphic markers to map genes that modify disease susceptibility or those related to drug responsiveness.
If we had a dense catalog of SNPs, we could narrow down the loci relating common disease susceptibilities to a small region containing one gene or so by linkage disequilibrium mapping.
To get such a catalog, the discovery of a large number of SNPs was first necessary [2].
The draft sequence was published in 2001 [3].
It was a collaborative work of the public sector via the international Human Genome Project (HGP) and the private sector via Celera Genomics.
HGP refers to the international 13-year effort, formally begun in October 1990, to discover all the human genes and make them accessible for further biological study.
Another project goal was to determine the complete sequence of the human genome.
Actually, the latter group led by Dr. Craig Venter, was the critical player for the success of finishing the sequence, which invented the strategy of the shotgun sequence, followed by computational assembly [4].
It was a revolution in genomic research.
With the help of the draft sequence, Japanese Single-Nucleotide Polymorphisms (JSNP), the first population-specific SNP database (http://snp.ims.u-tokyo.ac.jp/), was established as a collaborative work of HGC-IMS and JST [5].
A total of 190,562 genetic variations (mainly SNP) in 24 subjects were discovered and confirmed and have been made available through this website [6].
Based on information in the JSNP database, Tanaka&#39;s group in SRC succeeded in identifying a susceptibility gene for myocardial infarction [7].
This study was the first GWAS in the world [8, 9].
By a two-stage association study-i.e., a GWAS followed by replication of a limited number of the SNPs with significant p-values-they found a functional SNP in the lymphotoxin-α (LTA) gene associated with myocardial infarction.
Their GWAS was quite primitive by current standards.
They used only ~100,000 SNPs from the JSNP database, because in those days, the number of genes was considered to be ~100,000, and basically, one gene is expected to be covered by one SNP.
Both of these assumptions have been subsequently disproven by the HGP [10] and the HapMap Project [11].
The coverage of the genome in the GWAS was low, because the success rate for this genotyping was only 71%.
The estimated power (β value) for detection of a locus was also low, because the number of subjects was very small.
However, the basic idea for the current GWAS, including linkage disequilibrium mapping for identifying the most associated SNPs and a pooled control strategy, were already present in the frontier study.
Their study clearly showed that the GWAS was a reality, not the dream of statistical geneticists, and that a denser map of common SNPs was indispensable for realizing the dream.
Completion of human genome sequencing was announced on April 2003.
Reading of 99% of the gene-containing part of the human sequence was finished to 99.99% accuracy [10].
However, in a true sense, the human genome sequence had (and has) not been finished.
There are still many gaps in the &#34;reference&#34; sequence-for example, in the centromeres-and given the variation of the human genome sequence, many must wonder what the completion of sequencing the human genome is in the first place.
After completion of the sequencing of the human genome, our center, SRC, joined The International HapMap Project from its start.
The project was launched in October 2002 to create a public, genome-wide database of common human sequence variations, providing information needed as a guide to genetic studies of clinical phenotypes [2].
The HapMap Project is a natural extension of the HGP.
While the reference sequence of the genome constructed by the HGP is informative about the vast majority of bases that are invariant across individuals, HapMap focuses on DNA sequence differences among individuals.
The main genome researchers in the world got together and made an infrastructure of the association study-i.e., a database of 1 million genetic variations for 4 representative populations: Africans, European Caucasian, Chinese, and Japanese.
The Africans were the Yoruba people in Ibadan, Nigeria.
The European Caucasians were a Utah, USA population with Northern and Western European ancestry collected in 1980 by the Centre d&#39;Etude du Polymorphisme Humain (CEPH).
The Chinese consisted of unrelated Han Chinese in Beijing, China.
The Japanese were unrelated self-identified Japanese living in Tokyo, Japan.
In the HapMap Project, we are proud to say that SRC made the World No.
1 contribution as a research group by genotyping ~1/4 of the genome (Table 1).
HapMap data document the generality of hotspots of recombination, a block-like structure of linkage disequilibrium and low haplotype diversity, leading to substantial correlations of SNPs with many of their neighbors [11].
There is the extensive redundancy among nearby SNPs, providing the potential to extract extensive information about genomic variation without complete re-sequencing, as well as efficiencies through selection of tag SNPs and optimized association analyses.
After HapMap, GWAS became as easy as expected and intended; to perform a GWAS, we just collect DNA from patients, genotype it by using commercial arrays, and do a bit of statistics for the results (actually, in most cases, commercial companies do it for us as a business) (Fig.1).
Genotyping has become work, not research; commercial genotyping systems have spread all over the world.
By sharing the controls, collecting controls is no longer necessary, except for diseases with extraordinary high prevalence (&gt;10%).
A huge number of susceptibility genes are mapped by this &#39;easy&#39; method.
As a consequence, after HapMap, as commercial genotyping systems (Illumina, etc.)
have spread, the association study has spread all over the world.
As a result, a gold rush has come [1].
The study of genetic polymorphisms (SNP, copy number variation, etc.)
with its application to disease studies was selected for the No.
1 scientific event in the year 2007 [12].
Naturally, the application refers to the identification of susceptibility genes for common diseases by GWAS.
You must have seen (and are still seeing) so many GWAS papers in good journals, which have reported susceptibility genes for so many common diseases, including obesity, hypertension, diabetes, and osteoporosis.
For example, Nature Genetics is one of the top journals in the field of genetics, with an impact factor in 2010 of 36.377, and in that year, it published a total of 181 papers, of which 56% was GWAS-related.
So, for the journal, the genetic study is almost equal to the association study.
There came such a big wave of medical science, and of course, we got on the wave.
We did so many GWASs that were the first for diseases, including GWASs for osteoarthritis [13], rheumatoid arthritis [14], and diabetic nephropathy [15].
The big wave has also spread to East Asia.
Many good GWAS papers have been published from Korea and China as fruit of this international collaboration [16-18].
Good days are ending, however.
Today, the initial enthusiasm for GWASs is decreasing, and the GWAS society is facing severe criticisms.
A frequent complaint is that GWAS results mean little to patients due to the small effect of variants on disease risk and their relatively small contribution to common disease etiology.
This claim might be partially correct.
In the first place, &#34;significantly associated&#34; does not mean that the association identified by the study is significant in biology, medicine, or actual life.
For example, a paper reported the identification of a SNP that is associated with height, but its estimated additive effect is only 0.44 cm on height [19].
Another example of dubious significance is a GWAS study claiming that cartilage thickness of the hip joint measured on X-ray films is decreased in subjects with a susceptibility allele; they have a 0.07-mm (not cm!)
narrower joint space than those who do not have the allele [20].
However, the low risk (low odds ratio) is not always due to the GWAS itself.
In many studies, it is the problem of particular studies themselves, not of the GWAS.
For examples, using pooled controls of general populations as controls of highly prevalent diseases, like diabetes mellitus, hypertension, and obesity, must decrease the odds ratio of the susceptibility alleles because of contamination of the patients in the controls.
Also, ambiguous definition of populations, like &#34;UK Caucasians&#34; or &#34;Western Europeans,&#34; would compromise the result by admixture of different ethnic subgroups.
Thus, it is necessary to detect population stratification prior to association analysis.
Moreover, a GWAS is not a method to conclude something.
It&#39;s a method to map the gene: an associated variant is not necessarily a true causal variant.
GWASs present candidates of the causal variant.
To identify and prove the causality, we need to find the best associated variations by fine mapping around the smoking gun GWAS signal, followed by functional studies for them.
Statistics must lead to biology.
Biology must lead to medicine.
It is true that after all, no treatment, no benefit, to the patients has been given for common diseases from GWAS results so far.
My comment is not intended to look down on GWASs.
It is a method to start something; it gives us wonderful start points.
You can start the work based on the reality of the disease-clinical evidence (patient, clinical record) from the human evidence in front of you-to the patients at your bedside.
Another problem is that not infrequently, the interpretation of the &#39;mapping&#39; results is wrong.
In many studies, authors just make a story for causality by relating to known genes near the marker SNPs, even if they are far apart on the genome.
The association does not tell which gene is implicated; it just tells which sequence variation is wrong.
Some of the readers might be thinking that all genes have already been discovered by the international genome project.
This is totally wrong.
There are still many more genes that have not been identified.
For example, we have found a novel susceptibility gene, DVWA, that is associated with osteoarthritis [21].
DVWA was in a gene desert region when we first found it in 2004.
While we were working on cloning it [21, 22], a computer-predicted gene was mapped to the region; however, the prediction was a mess.
The number of coding exons was predicted to be 19, while it was actually 3.
Only 1 in 7 experimentally determined exons of DVWA had been correctly predicted regarding the exon-intron boundary and coding region (Fig.2).
We also found a novel susceptibility gene for osteoporosis, FONG [23], which was not predicted correctly in the public database.
GWASs can be a starting point to find a hidden gene in the genome, and we should remember that now, the concept of the &#34;gene&#34; itself is changing.
We have already known that there are so many non-protein-coding genes: the genes encoding functional RNAs [24].
They are quite compatible to the original concept of the gene: information in the genome that determines phenotype or trait.
To code proteins is just one of the many ways to determine phenotype.
GWASs have so many false positives.
p &lt; 10-2 (1% significance level) means a 1 in 100 coincidence.
If you try 1,000 times, you can hit 10 targets and 100,000 times, 1,000 targets-all quite by chance.
Therefore, association needs replication, but repeating in the same population is not a good idea; it might just introduce the same error in the first study, which is sometimes difficult to detect.
Studies in different populations, preferably in populations from different ethnic groups, are desirable.
Because one population is one experiment of nature, one ethnic group is one experiment given from heaven.
Every experiment is precious.
Multiple replications in large samples provide the most straightforward path for identifying robust and broadly relevant associations.
International collaboration is necessary.
For international replication, East Asia has a definite advantage when we consider its relative genetic similarity among the subpopulations, large population size, and high medical standard of the countries in the area.
Currently, most people look at GWASs as a method to identify a marker for genetic diagnosis and risk assessment.
Yes, by knowing the genetic risk, we can re-evaluate and modify our lifestyle and prevent unhappy problems, like side effects of drugs.
However, GWASs could give us far sweeter fruit.
Through functional analysis of a gene and its associated variants, we can approach the mechanism of the disease.
By studying the meaning of allelic differences of causal variants, we can clarify the molecular pathogenesis of the disease.
With the knowledge of the pathogenesis, we can walk directly to develop innovative treatments and discover new drugs (Fig.3).
Therefore, identification of the significantly associated SNP is just &#39;end of the beginning&#39; to get to the final goal: treatment of patients with common diseases.
However, at present, after identification of &#39;marker&#39; SNPs by GWASs, we have no golden road to find &#39;causal&#39; SNPs (variants).
We have no customized method to identify functional variants.
There is still a long and winding road before us.
It&#39;s just the beginning.
Structure of genome-wide association study.
It consists of 3 steps: phenotyping (recruitment of cases and controls), genotyping (typing of single-nucleotide polymorphisms [SNPs] of individual subjects), and mapping (statistical analysis to determine linkage disequilibrium block).
Gene structure of DVWA and its computer prediction.
LOC344875: a predicted gene in the public database.
DVWA has two major alternative splicing forms: long (L-) and short (S-) DVWA.
White box: untranslated region, black box: coding region.
Dotted line indicates matching of the position.
Real value of genome-wide association study (GWAS).
A GWAS is useful for diagnosis and treatment.
Title: Structure of International HapMap Project
Title: Transposable Elements: No More &#39;Junk DNA&#39;

Since the advent of whole-genome sequencing, transposable elements (TEs), just thought to be &#39;junk&#39; DNA, have been noticed because of their numerous copies in various eukaryotic genomes.
Many studies about TEs have been conducted to discover their functions in their host genomes.
Based on the results of those studies, it has been generally accepted that they have a function to cause genomic and genetic variations.
However, their infinite functions are not fully elucidated.
Through various mechanisms, including de novo TE insertions, TE insertion-mediated deletions, and recombination events, they manipulate their host genomes.
In this review, we focus on Alu, L1, human endogenous retrovirus, and short interspersed element/variable number of tandem repeats/Alu (SVA) elements and discuss how they have affected primate genomes, especially the human and chimpanzee genomes, since their divergence.
Transposable elements (TEs), mobile segments of genetic material, were first discovered by McClintock [1].
Since then, they have been identified in a variety of eukaryotes [2].
Recent genome sequencing projects have consistently shown that TEs make up ~50% of primate genomes, while coding DNA occupies only ~2% of the genomes [3-5].
TEs are generally divided into two categories, DNA transposons and retrotransposons (Fig.1), based on their manner of mobilization.
DNA transposons move using a cut-and-paste mechanism [6].
In contrast, retrotransposons move in a copy-and-paste fashion by duplicating the element into a new genomic location via an RNA intermediate [7].
Thus, retrotransposons increase their copy number more rapidly than DNA transposons.
Retrotranspons include short interspersed element (SINE), long interspersed element (LINE), and human endogenous retrovirus (HERV).
Alu and short interspersed element/variable number of tandem repeats/Alu (SVA) elements are primate-specific retrotransposons, and their full-length is 300 bp and 2 kb, respectively.
The Alu element is the most successful SINE in terms of its copy number; ~1.2 million Alu copies exist in the human genome.
LINE is ~6 kb in length, and thus, it is much longer than the SINEs.
This element has two open reading frames encoding enzymatic machineries essential for the propagation of the three elements; the Alu element depends on reverse transcriptase of LINE for making their dispersed copies in the host genome [8].
In contrast to SINE and LINE, which do not have long terminal repeats (LTRs), the full-length HERV (~10 kb) has two LTRs, and three genes-gag, pol, and env-are located between them [8, 9].
Studies on active TEs have suggested that the elements could alter gene expression by providing cis-regulatory elements, such as promoters, enhancers, and transcription factor binding sites [10].
Through these mechanisms, altered transcriptional activity could lead to dysfunctional and abnormal proteins.
Through de novo TE insertion within a gene, TEs could alter a gene product, which could be either harmful or beneficial to its host genome [11, 12].
In cases where an inserted TE causes a harmful effect on its host genome, the TE is likely to go to inactivation and fossilization by evolutionary accumulation of mutations and silencing effects [13].
Since the divergence of the human and chimpanzee, ~6 million years ago, many TEs have propagated in each genome.
Among them are the Alu, L1, SVA, and HERV-K (HML-2) elements.
During the past 6 million years, 5,530 Alu, 1,835 L1, 864 SVA, and 113 HERV-K (HML-2) elements are estimated to have been newly inserted in the human genome (Table 1) [8, 14, 15].
These elements could act as an agent causing human-specific genomic rearrangements via de novo TE insertions, TE insertion-mediated deletions, and homologous recombination events [12, 16-19].
Furthermore, some of the recent integrated TEs are capable of producing new copies in the human genome.
These de novo TE insertions have the potential to cause a genomic difference among human populations and even human individuals, which could be related to human phenotypes and diseases [20].
In this review, we describe species-specific TEs and discuss how they affect their host genomes, focusing on illustrating the mechanisms that they utilize with examples.
Taken together, we suggest that TEs, often called &#39;junk&#39; DNA, in fact have many functions and play a significant and dynamic role in primate genomic evolution.
Comparative genomics allows us to investigate species-specific TEs.
Through a combinational method of computational data mining and experimental verification, species-specific TEs have been well studied in various primate genomes, including human, chimpanzee, and rhesus macaque.
During the past 6 million years, more than 10,000 human-specific TEs were newly inserted in the human [21].
The majority of the identified elements belong to Alu, L1, and SVA retrotransposons.
These elements have contributed to the genomic difference between human and non-human primates through insertion and post-insertion recombination between the elements [8, 22].
Alu elements are most ubiquitous in the human genome.
A master gene model has been generally accepted to explain the amplification of Alu elements.
In the model, only limited numbers of hyperactive &#34;source&#34; or &#34;master&#34; genes are able to produce a high number of Alu copies over evolutionary time.
One of the previous studies about the AluYb subfamily, one of the most active Alu subfamilies in the human genome, found that the copy number of AluYb elements is very different between human and non-human primates; a high copy number of AluYb elements exists in the human, but a very small number of the elements exists in non-human primates [23].
The oldest AluYb element resides at its orthologous position in all hominid primate genomes, demonstrating that the AluYb subfamily emerged 18 to 25 million years ago.
Then, after approximately 20 million years of retrotranspositional quiescence, a major expansion of the subfamily occurred only in the human genome within the past several million years.
To explain their successful proliferation in the human genome, a new model, &#34;stealth driver,&#34; was introduced.
In this model, a high copy of the Alu elements could be driven at least in part by &#34;stealth driver&#34; elements, which maintain low retrotransposition activity over extended periods of time.
Although this element has low retrotransposition activity, they have the potential to produce short-lived hyperactive copies responsible for the remarkable expansion of AluYb elements within the human genome [24].
L1 is another successful element, occupying ~17% of the human genome.
The copy number of Alu elements is much higher than that of L1 (Alu, ~1.2 million copies; L1, ~520,000 copies).
Nonetheless, Alu elements are responsible for ~11% of the human genome, because the Alu element is much shorter than L1 (Alu average length, 300 bp; full-length of LINE, 6 kb) [25].
Through characterization of sequence diversity of chimpanzee-specific L1 subfamilies as compared to their human-specific counterparts, it was concluded that L1s experienced different evolutionary fates between humans and chimpanzees within the past ~6 million years.
Although the species-specific L1 copy numbers were on the same order in the two species (1,800 human-specific L1s vs. 1,200 chimpanzee-specific L1s), the number of retrotransposition-competent elements was much higher in the human genome than in the chimpanzee genome.
The species-specific L1s were grouped into several L1 subfamilies.
All human L1 subfamilies belonged to a single lineage, but two distinct L1 lineages were identified in the chimpanzee genome [15].
SVA is shared only in human and apes.
Fewer than 1,000 copy numbers were detected in orangutan (~15 million years ago), and no SVA detected in Old World monkeys indicating SVA is a hominid-specific element.
Like Alu elements, SVA elements retrotranspose to another locus in trans by using reverse-transcriptase encoded by L1s [8].
Due to the limited mobilization and short evolutionary time, the copy number of SVA is very small compared to Alu and L1 elements [15, 26, 27].
There are six SVA subfamilies (SVA_A to SVA_F) in the human genome.
The older, SVA_A to SVA_D, evolved in a single lineage, whereas human-specific SVA_E and SVA_F were derived independently from their ancestral sequences.
The copy number of SVA was estimated throughout the primate genomes of human, chimpanzee, and gorilla, and there was no significant difference among them.
The two elements, Alu and L1, showed a huge expansion at a specific evolutionary time along the primate lineage, but SVA still did not show any burst in its copy number [26, 27].
The comparison of human and chimpanzee genomic sequences showed that the two genomes have a much higher sequence identity than we expected [3, 4].
In spite of the sequence similarity, TEs have remarkably generated genomic differences between the two species since their divergence [22].
Many studies have suggested that a number of TEs are still active to retrotranspose and have the potential to cause genomic variations as a major driver [12, 16-19, 28].
In reality, TEs have rearranged human and non-human primate genomes through various mechanisms, such as de novo TE insertion, TE insertion-mediated deletion, and homologous recombination between them (Fig.2) [29].
These genomic changes caused by TEs have increased the genomic difference between human and non-human primates, and some of the human-specific genomic rearrangements caused human diseases [28, 34].
Advanced sequencing technology, including next-generation sequencing, and combined computational analyses have accelerated the studies on the dynamics of TE mobilization [35].
In reality, human-specific TEs have been continuously investigated, and the majority of them are Alu, L1, and SVA elements [8, 14].
The relationship between human brain evolution and Alu elements was studied.
Since the divergence of the human and chimpanzee lineages, the human brain has rapidly changed in terms of mass [36].
It is not an exaggeration to say that Alu elements are in part responsible for the human brain mass.
Interestingly, de novo Alu insertions have been identified in many human brain genes that are related to neuronal functions and neurological disorders [37].
The inserted Alu elements belong to AluYa5, AluYb8, and AluYc1, which are human-specific Alu subfamilies [37].
Approximately 1,800 human-specific L1s were identified in the human genome [15].
They belonged to two different subfamilies, pre-Ta and Ta; Ta is subdivided into Ta-0 and Ta-1 by diagnostic nucleotides [38].
Among hominid-specific SVA subfamilies, SVA_E and SVA_F are only detected in the human genome, but the other four subfamilies, SVA_A, SVA_B, SVA_C, and SVA_D, are shared in human and other apes, including chimpanzee and gorilla [26].
HERV appeared in the primate genome through germ-line infection [30].
There are approximately 98,000 HERVs in the human genome.
Full-length HERVs are ~10 kb in length, but most of the HERVS existing in the human are defective due to truncation and accumulation of mutations during primate evolution [39].
Among various HERV subfamilies, HERV-K (HML2) is the youngest element in the human genome [8, 40]; 113 human-specific HERV-Ks were identified in the human genome, and among them, there were 15 and 98 full-length HERV-Ks and solitary LTRs, respectively [39].
These de novo TE insertions showed polymorphisms among human populations and even human individuals [31, 39, 41, 42].
Therefore, they have the potential to be used as a genetic marker for racial identification [31, 41, 42].
De novo TE insertions contribute to the genome expansion.
Actually, some of them somewhat decreased their host genomes involving the insertion-mediated deletion of host genome sequences [32, 33].
Through comparative genomic analyses, 50 L1 insertion-mediated deletion events were found in the human and chimpanzee genomes [18].
The sizes of the deleted sequences were variable, and in sum, ~18 kb and ~15 kb of sequences were removed from the human and chimpanzee genomes, respectively.
Based on the result, it was estimated that L1 insertions may have deleted up to 7.5 Mb of target genomic sequences during the primate radiation.
Alu insertions were also involved in the genomic deletions at its insertion target regions through Alu retrotransposition-mediated deletion.
A total of 33 deletion events responsible for a ~9,000-bp deletion in human and chimpanzee genomes were identified.
It was suggested that Alu retrotransposition may have contributed to over 3,000 deletion events, leading to a ~900-kb deletion during primate evolution [17].
Additionally, 13 SVA insertion-mediated deletions (SIMDs) were also identified in the human genome, and they deleted 30,785 bp of the human genome compared with the chimpanzee genome (Table 1).
Among the 13 SIMDs, 9 were associated with the SVA_D subfamily, occupying the largest portion of SVAs, which suggests that SIMD frequency is directly correlated to the copy number of SVA elements.
Furthermore, one of the deletion events occurred in the tMDC II gene associated with sperm-egg binding prior to fertilization [43, 44].
After TE insertions into the host genome, they could generate genomic variations through unequal homologous recombination events between them [17, 19, 43].
The copy number of TEs is closely related to the frequency of the recombination between them.
Thus, compared to other TEs, Alu and L1 elements have a high probability of generating genomic structural variations due to their ubiquity.
In reality, 492 Alu recombination-mediated deletions (ARMDs) were identified in the human genome, and they deleted ~400 kb of human genomic sequences (Table 1).
About 60% of the deletion events were related to known or predicted genes, including three that deleted functional exons.
Thus, the ARMD process has produced a considerable portion of the genomic and phenotypic variations between humans and chimpanzees since the divergence of the two species [16].
The recombination between L1 elements has also deleted human genome sequences.
Seventy-three L1 recombination-associated deletions (L1RADs) were identified in the human genome [45].
The sizes of the deletion events range from 56 to 64,113 bp, and ~450 kb of human genomic sequence was deleted through this L1RAD process (Table 1).
Thus, the L1RAD event has deleted 25 times as much human genomic sequence as the L1 insertion-mediated deletion event [18, 45].
The TEs inserted in intra- and inter-genic regions could alter cellular gene expression, increasing genomic instability [9].
About a decade ago, gene regulation by TEs was studied only in specific genes through experimental validation.
However, genome-wide analyses of gene regulation by TEs were recently conducted due to the developed high-throughput technologies.
The findings showed that TEs have many regulatory sequences, such as promoters, enhancers, polyadenylation signals, and cryptic splicing donor (5&#39;) and acceptor (3&#39;) sites, by which the transcript architecture of nearby genes can be altered [10, 46, 47].
When TEs insert into the intronic region of genes, they could create a new exon by offering splicing sites, and this process is called &#34;exonization&#34; [48].
This mechanism is related to exon variations, such as cassette exons and intron retention in exons, increasing mRNA instability [49].
The TEs residing upstream of any gene could act as an alternative promoter, leading to new alternative transcripts with a new transcriptional start site [10].
Some TEs carry bidirectional promoters with transcription factor binding sites.
For example, LTR and L1 have sense promoters initiating their transcription and antisense promoters having the potential to initiate the transcription of other genes in the opposite direction [50, 51].
There is a microRNA gene cluster in human chromosome 19 (C19MC), over 100 kb in length.
This cluster consists of the duplication of a core cassette, including a minus-strand Alu element.
The cluster grew successfully during primate evolution, and the Alu element promotes microRNA expression by RNA polymerase III [52].
TEs not only initiate transcription but also terminate it by offering a polyadenylation signal.
A full-length L1 contains 19 polyadenylation signals that could cause premature mRNA truncation [53].
The genes that contain TEs in their genic region have a tendency to produce various transcript forms, causing transcriptome diversity [10].
The orientation of TEs could be a factor affecting gene expression, which is well described by a &#34;head-on collision&#34; hypothesis.
During DNA replication, DNA polymerase collides with RNA polymerase transcription complexes moving in the opposite direction to the movement of the DNA polymerase.
It was observed that the collision slows down the DNA replication [54].
In cases where active TE exists in the opposite direction to its nearby genes, an RNA polymerase transcription complex transcribing the TE could encounter any of the RNA polymerase transcription complexes transcribing nearby genes, which could reduce the expression of the gene [54, 55].
In reality, transcription of the E-globin gene is repressed by an Alu element that has been inserted in the opposite direction to the gene [56].
On the other hand, TEs inserting in the same direction to its nearby genes show no effect on the expression of the genes [55].
Histone modification plays an important role in gene transcriptional regulation, and through this process, the host genome could regulate the activation of TEs [57, 58].
In reality, most TEs are accompanied by repressive histone modifications (e.g., H3K9me2 and H3K27me3), which cause the formation of heterochromatin.
Conversely, TEs could affect the expression of host genes through histone modifications [59, 60].
The level of histone modifications was calculated in all families of human TEs, and older TE families carried more histone modifications than younger families.
Interestingly, TEs proximal to genes carry more histone modifications than the ones that are distal to genes, which suggests that some epigenetic modifications of TEs may serve to regulate the expression of host genes [61].
DNA methylation is a strict silencing mechanism, and the host genome could use this process to repress the activation of TEs [61].
In general, DNA methylation occurs in a CpG dinucleotide.
Because Alu and SVA elements have a high degree of CpG dinucleotides, they are vulnerable to methylation [26, 62].
It was observed that TEs regain their activity to mobilize and regulate the expression of host genes when the silencing effect becomes slackened with increasing genomic instability.
In addition, the demethylation of TEs is associated with human diseases, commonly in cancer [63-65].
miRNAs, one of the most active factors regulating gene expression, could be derived from TEs.
Fifty-five genes derived from TEs were identified in the human genome, and their characterization showed that TE-derived miRNAs could potentially regulate the complex and dynamics of human genes [66].
TEs have shown a variety of impacts on their host genomes.
In this review, we describe HERV, Alu, L1, and SVA elements, which are thought to still be active in the human genome.
A number of research studies related to TEs have shed new light on their amplification mechanisms and their function in primate genomes.
Furthermore, recent research of TEs in the rhesus macaque genome provides a glimpse into their diversity and strong influence on the overall differences in genomic architecture between the Old World monkey (e.g., rhesus macaque) and hominid (e.g., human and ape) lineages [67].
The occurrence of de novo TE insertions, TE insertion-mediated deletions, and post-insertion recombination between TEs within the human and chimpanzee lineages has caused genetic alteration, lineage-specific genomic rearrangements, and phenotypic variations, further contributing to the divergence of humans and chimpanzees.
As a whole, this review calls into question whether TEs should be considered &#34;junk&#34; DNA at all.
Rather, TEs represent a potent evolutionary force associated with genomic fluidity in their host genomes.
Structures of transposable elements.
These elements could be categorized into retrotransposons (Alu, long interspersed element [LINE], and human endogenous retrovirus [HERV]) (A) and DNA transposons (e.g., MARINER) (B) based on their manner of mobilization.
In addition, autonomous elements (e.g., HERV and LINE) have coding genes responsible for their own mobilization but also other nonautonomous elements (e.g., Alu and short interspersed element/variable number of tandem repeats/Alu [SVA]).
Alu consists of two monomers separated by an A-rich connector, one of which, the left monomer, includes internal RNA polymerase III promoter (A and B boxes).
Full-length of LINE is ~6 kb and has open reading frames (ORFs) encoding RNA-binding protein, endonuclease, and reverse-transcriptase, which are flanked by untraslated regions (UTRs).
ORF1 and ORF2 are separated by an ~60-bp-long intergenic spacer (IS).
SVA contains a (CCCTCT)n hexamer, Alu-like sequences, variable number of tandem repeat (VNTR), and short interspersed element-R (SINE-R).
An arrow on Alu-like sequences indicates the direction of Alu.
HERV has gag, prt, pol, and env genes flanked by a long terminal repeat (LTR), which encodes capsid protein, protease, polymerase, and envelop protein, respectively, used in viral infection.
As an example of DNA transposon, mariner has a gene encoding transposase with a DNA-binding domain and catalytic domain flanked by an inverted repeat (IR).
All elements are flanked by target site duplication (TSD) through integration.
DDE, the conserved DDE sequence of the mariner transposase; NLS, nuclear localization signal.
Schematic representation of genomic rearrangement and gene expression alteration by transposable elements (TEs) in host genome.
(A) Classical TE insertion by recognizing 5&#39;-TTAAA-3&#39;, (B) non-classical TE insertion, (C) nonallelic homologous recombination (NAHR)-mediated deletion, (D) nonhomologous end-joining (NHEJ)-mediated deletion, (E) mechanism of gene expression alteration by TEs integrated into the host gene.
Depending on location of insertion in the host gene, TEs could generate alternative transcripts or disrupt the expression.
ORF, open reading frame; Grey and pink arrow boxes, target site duplication; black line, flanking region; grey line, intervening region; dotted circles, homologous recombination regions; pink boxes, microhomology region.
Title: Genomic rearrangement associated with active transposable elements (TEs) in human genome
Title: Understanding Disease Susceptibility through Population Genomics

Genetic epidemiology studies have established that the natural variation of gene expression profiles is heritable and has genetic bases.
A number of proximal and remote DNA variations, known as expression quantitative trait loci (eQTLs), that are associated with the expression phenotypes have been identified, first in Epstein-Barr virus-transformed lymphoblastoid cell lines and later expanded to other cell and tissue types.
Integration of the eQTL information and the network analysis of transcription modules may lead to a better understanding of gene expression regulation.
As these network modules have relevance to biological or disease pathways, these findings may be useful in predicting disease susceptibility.
Genetic epidemiology is the study of the role of genetic factors in determining health and disease in families and in populations and the interplay of such genetic factors with environmental factors [1].
Depending on the number of genetic factors that contribute to pathogenicity, genetic diseases are categorized into two classes.
A so-called monogenic disorder is usually caused by a single gene mutation and has been the subject of intense gene hunting studies using familial linkages, totaling to more than 4,000 diseases being linked to causal mutations [2].
Completion of the Human Genome Project expedited the studies of not only monogenic diseases but also complex diseases.
The latter, also called polygenic or multifactorial diseases, are contributed by a number of genetic factors of relatively smaller effects.
Unlike monogenic disorders that often affect only a small portion of a population and are consequently rarely observed, polygenic diseases are usually frequently observed in a population and are also termed common diseases.
Chronic diseases, such as cancer, diabetes, hypertension, and so on, belong to this category.
The Human Genome Project paved the way to cataloging common variants in the human genome.
The International HapMap Project is a global collaboration to map single-nucleotide polymorphisms (SNPs) in the human genome and to construct haplotype maps that show which allele of a locus co-occur with which allele of another locus.
Using three representative continental populations-that is, Africans, Europeans, and Asians-the international consortium cataloged roughly 1 SNP for every 300 bp of the human genome.
The information of position, frequency, and correlation structure, called linkage disequilibrium of these SNPs, has enabled us to design high-density SNP chips that can measure the genotypes of fractions of those SNPs all at once and infer the genetic make-up of the major portion of the whole genome.
Using these chips, genotypes of families or unrelated populations are measured, and the loci displaying statistically significantly different frequency patterns between disease-affected and unaffected groups are sought.
Individuals harboring such alleles may have a higher or lower disease susceptibility than control groups.
Scanning genome-wide associations of genotype-phenotype relationships is called a genome-wide association study (GWAS) and has become extremely popular, reporting more than 1,000 papers within the last 5-6 years [3], probably due to the hypothesis-free nature of its study design.
While GWASs have been applied to studies of disease gene hunting, they are also well suited to interrogating genes affecting heritable phenotypes, such as height, obesity, plasma glucose concentration, blood pressure, bone density, and so on.
Unlike disease studies where samples are segregated into case and control groups, these phenotypes display a continuous spectrum of values in normal populations.
For s uch a quantitative trait, its statistical correlation with the genotypes within a population is sought, instead of dichotomizing the samples into case and control groups.
The genomic loci that show correlations with such a trait are called quantitative trait loci (QTLs).
According to the current estimate, the human genome harbors around 22,000 genes and 200,000 mRNA transcripts [4].
One way that variations in the genomic sequence can exert phenotypic differences is through the alteration of structure and function of the gene products.
Genetic variation may also result in changes in the expression levels of products and, thereby, their activities.
Since most SNPs identified through GWASs do change the amino acid sequence of a protein, large portions of the genetic variation may be regulatory in nature.
Differences in gene expression profiles between individuals then become molecular and intermediate phenotypes that in turn induce changes in higher-order disease traits [5].
In fact, Cheung et al.
[6] observed some genes showing markedly different expression levels between individuals and familial aggregation of expression phenotypes.
In their follow-up study using 14 large families, they also identified approximately 1,000 expression phenotypes that showed significant evidence of linkage to specific chromosomal regions [7].
Dixon et al.
[8] also observed heritable expression patterns in about 30% of the genes and transcripts of their probes and identified the genomic regions where the genotypes were associated with those expression phenotypes.
These are termed expression QTL (eQTL) and can be proximal (cis) or distal (trans) to the gene of expression variation (Fig.1).
Other species also show eQTL.
For example, eQTLs have been observed in recombinant inbred mouse strains [9-11], yeast [12, 13], and plants [14].
The B-lymphoblastoid cell lines established by the International HapMap Project provide invaluable resources for the study of eQTL: for these samples, matched genotypes and expression profiles are publicly available.
It is well known that genome-wide genotype profiles have the power to discriminate continental populations.
Principal component analysis of the genotypes of the HapMap populations clearly clusters the individuals according to their population origins [15].
On the other hand, it appears that the population difference in expression levels is negligible [16] or very small [17].
For example, the expression levels averaged over each population are highly correlated between populations (Fig.2).
Apparently, the within-group expression variance is much larger than the between-group variance.
This is perhaps due to the fact that unlike genotypes that can drift somewhat freely without functional alteration, gene expression levels are tightly regulated by complex genetic programs.
Instead of focusing on population differences in expression levels, intense effort has been made to cataloging eQTLs and classifying them into cis and trans.
For example, using the HapMap samples of European and African ancestry, Duan et al.
[18] found about 5,000 expression quantitative nucleotides (eQTNs) in each population.
They used 4 Mb of distance between the gene and SNP as the criteria for classifying local and distant eQTNs and observed that genes harboring distant eQTNs were enriched with functions, like transmembrane signal transduction, implying roles in gene expression regulation [18].
Stranger et al.
[17] used all 270 HapMap samples to identify at least 1,348 and 180 genes showing cis and trans association signals, respectively.
Their result is in sharp contrast to the previous estimate that counts most eQTLs as trans rather than cis [7].
Veyrieras et al.
[19] reanalyzed the HapMap data using a sophisticated Bayesian hierarchical model to resolve the eQTLs at a fine scale: most eQTLs lay either within genes or close to genes (&lt;20 kb); eQTLs were enriched around transcription start sites and 250 bp upstream of transcription end sites, and exons were more likely to be eQTLs than introns.
Their finding of enriched eQTLs upstream of transcription end sites is interesting, in that this site is related to post-transcriptional control and mRNA stability.
This underscores the important role of mRNA stability besides transcription initiation in determining steady-state mRNA levels [19].
The presence of cis-acting eQTLs for a gene implies that these alleles have different transcription levels or allelespecific expression.
The typical microarray chips designed for expression measurement do not usually distinguish minute sequence differences of the SNP alleles residing within a transcript.
One can infer the allele-specific expression using statistical tests that model the expression levels of a gene with the number of minor alleles at each locus.
On the other hand, by hybridizing the transcripts onto SNP chips or directly sequencing the transcripts, one can validate the allele-specific expression.
Ge et al.
[20] used Illumina Human 1M Beadchips to measure allele-specific expression in 53 lymphoblastoid cell lines derived from European descendants.
With the advent of next-generation sequencing technologies, RNA-seq has become popular and has been applied to detecting allele-specific expression.
Pickrell et al.
[21] sequenced RNA from 69 lymphoblastoid cell lines derived from unrelated African individuals who participated in the International HapMap Project, discovering a number of eQTLs involved in allele-specific expression or allele-specific splicing.
So far, most eQTL studies have examined expression-genotype relations in lymphoblastoid cell lines.
Since many genes display tissue-specific expression patterns, one may argue that the expression-genotype relations observed in Epstein-Barr virus-transformed lymphoblastoid cell lines may not be preserved in other tissues or cell types.
Gerrits et al.
[22] observed that eQTLs were quite different, depending on the differentiation status of the cells, suggesting uncommon functional regulatory networks among different cell types.
Nica et al.
[23] compared cis-regulatory eQTLs from three different tissues of co-twin samples from the MuTHER project (http://www.muther.ac.uk/), observing 30% shared eQTLs, 29% exclusively tissue-specific eQTLs, and substantial differences in expression levels in different tissues.
In addition to the similar proportion of exclusively tissue-specific cis-regulatory eQTLs from five different tissues, Fu et al.
[24] also observed many examples of alternative regulation, where a gene is cis-associated with different SNPs, depending on the tissue.
If one were to use eQTL information in predicting disease susceptibility, it is thus imperative to understand expression-genotype relations in the tissue of interest, not just in lymphoblastoid cell lines [25].
When transcript expression levels were measured in adipose tissues as well as blood in large population cohorts, unlike expression profiles in blood, a marked correlation between expression levels in adipose tissues and obesity-related traits was observed [26].
Schadt et al.
[5] observed over 6,000 associations between genotypes and expression in 400 human liver samples to understand the genetic architecture of gene expression in the human liver, a metabolically important tissue involved in various common diseases, such as obesity, cancer, diabetes, and atherosclerosis.
We have reviewed above that natural variation in gene expression is substantial and heritable.
Genetic analyses of expression phenotypes have identified cis- and trans-associated loci, culminating in our understanding of gene expression regulation.
One should understand that not all genes showing natural variation are regulated by either cis- or trans-eQTLs.
For example, genes downstream of eQTL-regulated genes in a regulatory network may also show natural variation.
If they are regulated primarily by a regulator, their expression profile may resemble that of the regulator.
On the other hand, if multiple regulators are involved, the downstream target genes may show complex expression profiles that may look independent of those of the regulators.
Integrating the bioinformatic learning of regulatory programs and genetic analysis of cis- and trans-acting DNA variants may lead to a better understanding of gene expression regulation [27].
Nayak et al.
[28] constructed co-expression networks based on the HapMap expression profiles of African, European, and Asian ancestries, discovering that the subnetwork structures are not random but relevant to biological pathways or disease susceptibility.
This implies that genes are regulated by a common set of regulatory modules and that the variable activity of the modules due to genetic variation can result in clustered expression profiles of the target genes.
Li et al.
[9], using 32 recombinant inbred mouse strains, reported 29 transcription modules and their associations with 18 classical physiological and behavioral traits, possibly through eQTLs.
Individual variation in gene expression levels may be influenced by environmental and genetic factors.
A substantial proportion of the variation is heritable and thus has a genetic basis.
Based on studies on population-level genotype-expression data, it is now well established that DNA variations affect the expression levels of nearby or remote genes.
Since the genotype data used in these studies were mostly measured on commercial SNP chip platforms of common variants, the causal eQTLs must be probed SNPs or located within the linkage disequilibrium blocks tagged by the probed SNPs.
Whatever the case is, one may wonder whether those eQTLs evolved neutrally without any selection pressure or whether they were the result of selection over human population history.
The co-expression network study by Nayak et al.
[28] indicated that the subnetworks are modular and coherent in biological functions.
It is very unlikely that such network structures were derived as the result of random drift in genomic variations.
It is also interesting that evolutionary selection operated on a network or pathway level.
Although they must have occurred gene by gene, the whole sum of the variations has biological themes.
If the activity of the pathways is variable between normal individuals, it has an implication in predicting disease susceptibility.
As the subjects probed by gene expression profiling are normal individuals, the variations of the pathway activity may not be large enough to trigger diseases.
Nevertheless, they may predispose the genetic background of an individual to be susceptible or resistant to certain diseases.
For example, individuals having slightly diminished immune pathway activities may have a high tendency of contracting infectious diseases or other immune-related diseases.
The next challenge would then be the experimental validation of this hypothesis.
The concept of expression quantitative trait loci (eQTLs).
If transcript abundance depends on the genotype of a genomic locus, such a region is called an eQTL.
If an eQTL is found inside or near the boundary of the target gene, it is classified as cis-regulatory (A); otherwise, it is trans-regulatory (B).
In the latter case, the expression level of a target gene is not regulated by its cis-eQTL but through a regulator, such as a transcription factor (TF), whose expression is regulated by its cis-element.
Comparison of average expression levels of all genes between two ethnic groups.
The expression profile datasets of the International HapMap populations (European [CEU] and African [YRI]) measured in lymphoblastoid cell lines were downloaded from Gene Expression Omnibus (GEO accession no.
GSE6536).
For each of the 19,723 genes, the log-transformed expression levels of both 60 YRI and 60 CEU individuals were averaged and plotted on the x- and y-axis, respectively.
Title: Finding Genetic Risk Factors of Gestational Diabetes

Gestational diabetes mellitus (GDM) is a complex metabolic disorder of pregnancy that is suspected to have a strong genetic predisposition.
It is associated with poor perinatal outcome, and both GDM women and their offspring are at increased risk of future development of type 2 diabetes mellitus (T2DM).
During the past several years, there has been progress in finding the genetic risk factors of GDM in relation to T2DM.
Some of the genetic variants that were proven to be significantly associated with T2DM are also genetic risk factors of GDM.
Recently, a genome-wide association study of GDM was performed and reported that genetic variants in CDKAL1 and MTNR1B were associated with GDM at a genome-wide significance level.
Current investigations using next-generation sequencing will improve our insight into the pathophysiology of GDM.
It would be important to know whether genetic information revealed from these studies could improve our prediction of GDM and the future development of T2DM.
We hope further research on the genetics of GDM would ultimately lead us to personalized genomic medicine and improved patient care.
Gestational diabetes mellitus (GDM) is defined as abnormal glucose tolerance that is first identified or diagnosed during pregnancy [1].
It is estimated that approximately 2% to 5% of all pregnancies in Korean women are complicated by GDM [2].
The clinical significance of GDM is that it increases the risk of adverse pregnancy outcomes [3].
In a recent large-scale multinational prospective study, increased maternal glucose concentration during pregnancy was significantly associated with increased neonatal birth weight, primary cesarean delivery, neonatal hypoglycemia, and increased placental C-peptide levels [4].
After parturition, about 15% of GDM women had persistent diabetes at an early postpartum period of 2 months [5].
GDM is also reported to recur at a frequency of 45% in subsequent pregnancies [6].
Women with a previous history of GDM are at increased risk of future development of type 2 diabetes mellitus (T2DM).
The risk of T2DM is 3.5 times greater in women with a history of GDM compared to the general population in Koreans [7].
In addition, offspring of GDM women are also at risk of developing obesity and T2DM [8].
The incidence of GDM is expected to rise, as it will parallel the increasing rate of obesity and T2DM.
Furthermore, the recent recommendation from the International Association of Diabetes and Pregnancy Study Group has lowered the diagnostic threshold of GDM and is expected to increase the incidence of GDM [9].
GDM is thought to have strong heritability.
GDM women have an increased positive family history of T2DM.
Compared to pregnant women with normal glucose tolerance, GDM women have a significantly greater parental history of T2DM (13.2% vs. 30.1%, p &lt; 0.001) [10].
In addition, both GDM women and their offspring are at increased risk of future development of T2DM.
The heritability estimate of T2DM was reported to be quite high (h2 = 0.69) in a recent study performed in Europeans [11].
These findings are indirect evidence that GDM has familial tendency.
However, there is no study that has specifically evaluated the heritability of GDM using familial clustering or twins [12].
It would be important to estimate the heritability of GDM and compare it with that of T2DM in women.
During normal pregnancy, women experience increased adiposity and weight gain, which begin near mid-pregnancy and progress throughout the third trimester [13].
In this period, insulin resistance ensues as a consequence of multiple factors, including increased production of placental growth hormone, estrogen, and tumor necrosis factor α [14, 15].
Pregnant women with normal glucose tolerance can increase their β-cell insulin secretion in response to this increased insulin resistance during pregnancy.
Although the mechanism of increased β-cell insulin secretion during pregnancy is not fully understood, it is reported that prolactin, which increases during pregnancy, can repress islet menin levels and stimulate β-cell proliferation in mice [16].
In addition, recent reports suggest that β-cell serotonin signaling is also a major determinant of β-cell mass during pregnancy [17].
It has been suggested from several clinical studies that GDM women have limited insulin secretion capacity that cannot compensate for the increased insulin resistance [13].
Similar to GDM, T2DM is also characterized by relative deficiency in insulin secretion in the face of increased insulin resistance.
Various factors, including increased age, obesity, high-fat diet, and sedentary lifestyle, can induce insulin resistance, and those who do not have sufficient β-cell insulin secretory capacity are more likely to develop T2DM [13].
Therefore, it is said that a large proportion of GDM women are experiencing future T2DM in advance.
Based on the findings that GDM women are at high risk of T2DM and both GDM and T2DM share similar pathophysiologies, it is reasonable to assume that they might also share similar genetic risk factors.
Soon after the initial reports of T2DM genome-wide association (GWA) studies [18-21], several studies investigated whether genetic variants that were identified through GWA studies of T2DM were also associated with the risk of GDM [22, 23].
Cho et al.
[22] genotyped variants in or near CDKAL1, CDKN2A/2B, FTO, HHEX, IGF2BP2, SLC30A8, TCF7L2, KCNJ11, and PPARG in 869 GDM women and 632 carefully selected nondiabetic control subjects.
They found that genetic variants in CDKAL1 and CDKN2A/2B were highly associated with the risk of GDM (p &lt; 1 × 10-6).
In addition, variants in HHEX, IGF2BP2, SLC30A8, and TCF7L2 were all nominally associated with GDM (p &lt; 0.05).
Among a total of 18 genetic variants studied, 9 reached a nominal significance level (p &lt; 0.05).
In Fig.1, the risk allele frequency as well as the odds ratios of known T2DM variants are compared among a control group (n = 632, men : women = 287 : 345), T2DM group (n = 761, men : women = 354 : 407), and GDM group (n = 869) in Koreans [22, 24].
For most of the variants, there was an increasing trend of risk allele frequencies from control to T2DM and from T2DM to GDM.
Lauenborg et al.
[23] also found that variants in TCF7L2, CDKAL1, and TCF2 were significantly associated with the risk of GDM in Europeans, consisting of 283 GDM women and 2,446 glucose-tolerant control women.
Variants in KCNQ1, which were first identified in an East Asian T2DM GWA study, were also significantly associated with the risk of GDM in Koreans (p &lt; 0.05) [25].
Recently, it was reported that TPH1 and HTR2B play a crucial role in regulating pancreatic β-cell mass during pregnancy in a mouse model and that their genetic alterations could result in GDM [17].
A total 6 genetic variants in HTR2B and 11 variants in TPH1 were identified and genotyped in Korean GDM women and control subjects [26].
Although there were no significant associations of these variants with the risk of GDM, they were associated with measures of obesity and weight gain during pregnancy [26].
Following these candidate approach studies was a two-staged GWA study that was performed in Korean GDM women [27].
A total of 468 GDM women and 1,242 nondiabetic control women were genotyped using Affymetrix Genome-Wide Human SNP Array 5.0.
Variants that passed the prespecified p-value threshold in the stage 1 genome scan were further genotyped in 931 GDM women and 783 nondiabetic control women.
Two variants, one located in an intron of CDKAL1 (rs7754840) and one near MTNR1B (rs10830962), were associated with a risk of GDM at a genome-wide significance level (p = 6.65 × 10-16 and p = 2.49 × 10-13, respectively).
This was the first GWA study investigating the genetic risk factors of GDM.
Although variants in MTNR1B have been previously reported to be associated with increased fasting glucose levels, the study was the first to report that MTNR1B variants are associated with GDM at a genome-wide significance level.
One of the limitations was that it was not sufficiently powered to find truly novel genetic variants that were only specific in GDM.
It should be noted that current GWA and GWA meta-analysis performed in T2DM recruits more than 100,000 cases and controls [28].
One of the interesting findings was that genetic variants of T2DM were enriched in GDM subjects.
Among the 34 confirmed T2DM genetic variants, 8 were associated with a risk of GDM [27].
In addition, when the β-coefficients (or odds ratio) of the variants derived from the logistic regression were compared between GDM and T2DM, there was a significant positive correlation of β-coefficients between the two [27].
These findings suggest that GDM and T2DM might share similar genetic backgrounds, at least in part.
GWA studies have opened a new era in diabetes research.
Our knowledge on the genetic predisposition of GDM as well as T2DM is expected to increase even faster as next-generation sequencing technology is applied to this field.
There should be even larger GWA studies on GDM, and GWA meta-analyses should be available.
In this way, we could find variants that have smaller effect sizes but are more specific to GDM than T2DM.
In order to understand the genetic determinants of glucose and insulin concentration during pregnancy, genetic associations regarding these quantitative traits should be investigated.
A fast way is to see whether genetic variants that are known to affect glucose or insulin concentration in the normal population also affect glucose or insulin concentration during pregnancy [29].
In addition, a GWA study on glucose or insulin concentration during pregnancy should also be helpful.
In this way, we would be able to better understand the pathophysiology of GDM.
By definition, GDM encompasses women with pre-exiting T2DM, maturity onset diabetes of the young (MODY), or even type 1 diabetes patients that were not diagnosed previously.
It would be important to know the proportion of MODY patients that constitute GDM.
In particular, about 15% of GDM patients remain diabetic at early postpartum periods, and a significant portion of these subjects might fall into the category of MODY.
The contribution of genetic variants known to cause MODY, such as GCK and HNF1A, in GDM has been reviewed in recent literature [30].
However, whole-exome sequencing will provide us with better bird&#39;s eye view on the contribution of MODY genes in GDM.
In addition, it might be able to find novel MODY genes in those who have persistent diabetes after GDM pregnancy and also have a strong family history of diabetes.
One of the first steps in translating the genetic information into clinical practice would be to predict the future development of T2DM in GDM women.
GDM women are at particularly high risk of developing T2DM and require preventive measures and early screening of T2DM.
Genetic information might improve our prediction of T2DM in women with a history of GDM.
This is an area of active research [31, 32], and we are looking forward to studies that use genotype risk scores in predicting T2DM in GDM women.
A similar approach could also be applied in predicting GDM, as more genetic variants associated with GDM are expected to be revealed.
The functional consequences of the current common genetic variants identified through GWA studies of GDM are not well understood yet.
It is not known whether they are markers in linkage disequilibrium with nearby causal variants or whether they have unknown but relevant functional roles.
Next-generation sequencing might give answers to these questions, but a huge number of samples and much effort will be required.
GDM women are at increased risk of developing T2DM and have familial clustering of T2DM.
A common pathophysiology that is shared by GDM and T2DM is impaired compensatory increase in insulin secretion to overcome the increased insulin resistance.
The GWA study of GDM enabled us to investigate the common genetic risk factors of GDM, and it revealed that GDM and T2DM share similar genetic backgrounds, at least in part (Fig.2).
Although this information has significantly improved our insight in the pathogenesis of GDM, there are more unanswered questions remaining that should be explicitly expressed.
Using the technology of next-generation sequencing, it is expected that more causal, rare variants will be identified.
This genetic information would ultimately improve our prediction of GDM and future T2DM.
In addition, we hope that personalized genomic medicine could be available using the advances in the genetics of GDM and T2DM.
Comparison of risk allele frequencies of known type 2 diabetes mellitus (T2DM) genetic variants among a control group (n = 632, men : women = 287 : 345), T2DM group (n = 761, men : women = 354 : 407), and gestational diabetes mellitus (GDM) group (n = 869) in Koreans.
There is an increasing trend of risk allele frequencies from controls to T2DM and from T2DM to GDM.
The odds ratio (OR) and 95% confidence interval of the risk variants for risk of T2DM and GDM are shown above the bars.
Gestational diabetes mellitus (GDM) and type 2 diabetes mellitus (T2DM) have similar pathophysiologies of impaired compensation of insulin secretion in the face of increased insulin resistance.
GDM women have increased familial clustering of T2DM and are at high risk of developing T2DM.
Using genetic association studies, it has been shown that GDM and T2DM share similar genetic backgrounds, at least in part.
Title: Association Analysis of Reactive Oxygen Species-Hypertension Genes Discovered by Literature Mining

Oxidative stress, which results in an excessive product of reactive oxygen species (ROS), is one of the fundamental mechanisms of the development of hypertension.
In the vascular system, ROS have physical and pathophysiological roles in vascular remodeling and endothelial dysfunction.
In this study, ROS-hypertension-related genes were collected by the biological literature-mining tools, such as SciMiner and gene2pubmed, in order to identify the genes that would cause hypertension through ROS.
Further, single nucleotide polymorphisms (SNPs) located within these gene regions were examined statistically for their association with hypertension in 6,419 Korean individuals, and pathway enrichment analysis using the associated genes was performed.
The 2,945 SNPs of 237 ROS-hypertension genes were analyzed, and 68 genes were significantly associated with hypertension (p &lt; 0.05).
The most significant SNP was rs2889611 within MAPK8 (p = 2.70 × 10-5; odds ratio, 0.82; confidence interval, 0.75 to 0.90).
This study demonstrates that a text mining approach combined with association analysis may be useful to identify the candidate genes that cause hypertension through ROS or oxidative stress.
Hypertension is defined as blood pressure measurement consistently higher than 140 mm Hg systolic blood pressure (SBP) and/or 90 mm Hg diastolic blood pressure (DBP) [1].
It is a complex syndrome determined by genetic and environmental factors and affected by multiple genetic factors to 30% to 50% of blood pressure variability in human hypertension [2].
Although hypertension is a leading cause of cardiovascular disease, ischemic heart disease, and stroke, the exact cause of hypertension is unclear [3].
Oxidative stress, which results in an excessive product of reactive oxygen species (ROS), is one of the fundamental mechanisms of the development of hypertension.
In the vascular system, ROS has physical and pathophysiological roles that are important in vascular remodeling and endothelial dysfunction associated with hypertension [4].
Since 1960, when the association between free radicals and hypertension was reported [5], plenty of data supporting a role of oxidative stress in hypertension have been published.
However, the evidence of whether oxidative stress causes hypertension is weak, and a few clinical studies have shown the relationship between blood pressure and ROS.
Nonetheless, oxidative stress has an important role in vascular biology and a potential role in hypertension.
In this study, ROS-hypertension-related genes were collected by the biological literature-mining tools, such as SciMiner and gene2pubmed, in order to identify the genes that would cause hypertension through ROS.
Further, single nucleotide polymorphisms (SNPs) located within these gene regions were examined statistically for their association with hypertension in 6,419 Korean individuals, and pathway enrichment analysis using the associated genes was performed.
Title: Study participants and genotyping

The Korea Association Resource (KARE) study recruited 10,038 participants aged 40 years to 69 years from the rural Ansung and urban Ansan cohorts and has been previously described in detail [6]; 1,196 subjects were excluded due to poor genotyping data, and we also excluded subjects with prehypertensive status (120 mm Hg &lt; SBP &lt; 140 mm Hg and/or 80 mm Hg &lt; DBP &lt; 90 mm Hg).
In total, 6,420 participants-1,968 hypertensive cases with hypertensive therapy or SBP ≥ 140 mm Hg or DBP ≥ 90 mm Hg and 4,452 controls with SBP ≤ 120 mm Hg and DBP ≤ 80 mm Hg-were examined for a hypertension case control study.
The Affymetrix Genome-Wide Human SNP array 5.0 (Affymetrix, Inc., Santa Clara, CA, USA) was used to genotype KARE study individuals.
The accuracy of the genotyping was examined by Bayesian Robust Linear Modeling using the Mahalanobis distance (BRLMM) genotyping algorithm [7].
The sample and SNP quality control criteria have been described [6].
In brief, samples with accuracies that were lower than 98%, high missing genotype call rates (≥4%), high heterozygosity (&gt;30%), or gender biases were excluded.
SNPs were excluded according to filter criteria as follows: SNP call rate &gt; 5%, minor allele frequency &lt; 0.01, and Hardy-Weinberg equilibrium p &lt; 1 × 10-6.
After quality control, 8,842 individuals and 352,228 markers remained.
Title: Ascertaining ROS- and hypertension-related genes

The SciMiner [8] web-based literature mining tool was used to obtain gene sets associated with ROS and hypertension.
SciMiner was run on a query of &#34;Reactive Oxygen Species&#34; [MeSH] AND &#34;Hypertension&#34; [MeSH], identifying ROS-hypertension articles and genes as of April 24, 2012.
We also retrieved genes for these ROS-hypertension articles from NCBI gene2pubmed (ftp://ftp.ncbi.nlm.nih.gov/gene/DATA) data.
The newly found genes from gene2pubmed were added to the ROS-hypertension gene set.
The positions of genes in the human genome were downloaded from the Ensembl Biomart database (NCBI build 36).
Some gene symbols were different from the results of SciMiner and Biomart, such as NOS2A → NOS2 and STN → EEF1A2.
The functional analysis tools, such as SciMiner, WebGestalt [9, 10], and DAVID [11, 12], were used for enrichment analysis to find the pathway with ROS-hypertension-associated genes, and the statistical significance of biological functions was calculated with Benjamini and Hochberg-adjusted p &lt; 0.05 as the cutoff.
Title: Statistical analyses

PLINK version v1.07 (http://pngu.mgh.harvard.edu/~purcell/plink) was used to perform the association analysis, and the hypertension case control study was tested by logistic regression analysis.
The association tests were based on an additive genetic model and adjusted for recruitment area, age, sex, and body mass index.
Title: Ascertaining ROS and hypertension candidate genes

With the results of SciMiner, queried with &#34;&#39;Reactive Oxygen Species&#39; [MeSH] AND &#39;Hypertension&#39; [MeSH]&#34;, 574 genes were obtained from 903 ROS-hypertension-related articles; 49 genes were found through the NCBI gene2pubmed data with these 903 papers, and only 2 genes out of 49 genes were new to the 574 SciMiner genes.
Three hundred seventeen genes (55%) among the 576 ROS-hypertension genes were referenced in only 1 article (Table 1) and were excluded for further analysis, with 259 genes remaining.
Using Ensembl Biomart (NCBI Build 36), we then extracted the position information of 259 genes, and the genes located on chromosomes X, Y, and MT were also excluded.
Finally, 237 genes that included SNPs genotype information from KARE data within the gene boundary (±20 kb upstream and downstream of the gene) were selected, and 2,945 SNPs were tested for hypertension association analysis.
The frequently mentioned genes (number of papers &gt; 40) in the ROS-hypertension papers are shown in Table 2.
Title: Association analysis of hypertension

We examined 2,945 SNPs of 237 genes for a hypertension case control study by logistic regression analysis; 68 genes were significantly associated with hypertension (p &lt; 0.05) (Table 3).
The most significant SNP was rs2889611 within mitogen-activated protein kinase 8 (MAPK8; p = 2.70 × 10-5; odds ratio [OR], 0.82; confidence interval [CI], 0.75 to 0.90), and rs1356415 from PROM1 and rs4536994 from KDR were strongly associated with hypertension (p = 3.45 × 10-4; OR, 1.18; CI, 1.08 to 1.29 and p = 3.73 × 10-4; OR, 1.19; CI, 1.08 to 1.31, respectively).
Title: Functional analysis of ROS-hypertension gene set

The 68 targets that were significantly associated with ROS and hypertension were tested for functional enrichment analysis.
Three functional analysis tools, SciMiner, Web-Gestalt, and DAVID, identified 34 significantly over-represented biological functions in the Kyoto Encyclopedia of Genes and Genomes pathway [13, 14].
The most significant biological pathway from the 3 functional analysis tools was focal adhesion, involved in the cell communication pathway group (Table 4).
The most frequent pathway group was cancer pathways (n = 9), such as glioma and pancreatic cancer.
Eight other pathways related to signal transduction and 11 organismal system pathways (level 1) related to the immune system, endocrine system, and nervous system were significantly identified.
Oxidative stress due to excess production of ROS is one of the reasons for the development of hypertension [4].
To identify genetic risk factors that induce hypertension through ROS, this study extracted ROS-hypertension-related genes using text-mining tools and investigated the association of genes with hypertension in 6,419 unrelated Koreans.
MAPK8, PROM1, and KDR had strong association signals with hypertension (p &lt; 4 × 10-4).
Especially, MAPK8 was published 13 times in ROS-hypertension articles, while most genes strongly associated with hypertension (p &lt; 0.01) were published an average of 6.29 times.
MAPK8, known as JNK1, included 19 SNPs in the KARE genotype data, and 14 SNPs among the 19 SNPs were significantly associated with hypertension, ranging in p-value from 2.7 × 10-5 to 1.3 × 10-3-moderate in comparison with a genome-wide association study (GWAS)-significant p-value (5.0 × 10-8); thus MAPK8 was not considered as the candidate gene of hypertension in previous GWAS studies [15, 16].
MAPK8 plays a key role in T cell proliferation, apoptosis, and differentiation through the studies of Jnk1-deficient mice [17, 18].
MAPK8 was included on HumanCVD Beadchip, a customized cardiovascular disease (CVD) SNP chip containing more than 2,100 CVD candidate genes [19].
However, previous cardiovascular disease GWASs regarding high-density lipoprotein particle features, lipids, and apolipoproteins did not report the association of the MAPK8 gene [20, 21].
Therefore, it needs replication to make it sure whether MAPK8 is indeed involved in the development of hypertension through ROS.
Using the text-mining tool, we found 237 ROS-hypertension-related genes.
The most frequent gene was AGT (angiotensinogen [serpin peptidase inhibitor, clade A, member 8]), which was reported on 320 ROS-hypertension articles, but it was not associated with hypertension in this study or our previous report [22].
Most of the genes that were published in more than 40 articles were not associated with hypertension or showed weak associations; 6 of 17 genes were significant, and the lowest p-value was 0.014 (nitric oxide synthase 1 [neuronal], NOS1).
The average number of articles for genes with strong signals (p &lt; 0.01) was 6.29 articles, and that for those with moderate signals (0.01 ≤ p &lt; 0.05) was 18.90 articles.
Two large GWASs, the International Consortium for Blood Pressure Genome-Wide Association Studies (IC-BPGWAS) [23] and Asian Genetic Epidemiology Network Blood Pressure (AGEN-BP) [16], reported 33 blood pressure candidate loci in 2011.
Among 66 genes within the 33 blood pressure candidate loci, 6 genes were included in the ROS-hypertension gene set as follows: NPPA, NPPB, PTPN11, CYP1A1, GNAS, and EDN3.
We examined their association with hypertension by case control study, and NPPA, NPPB, and CYP1A1 were associated with hypertension with p &lt; 0.05.
The weakly associated SNP rs1023252 (p = 0.047) overlapped with NPPA and NPPB, and rs2472299 within the CYP1A1 locus was previously mentioned for the oxidative stress pathway from WikiPathway (http://www.wikipathways.org).
In conclusion, we listed ROS-hypertension genes that were extracted by a text-mining approach and tested their association with hypertension in Korean population.
Several genes, including the MAPK8 gene, were identified as potential genes causing hypertension through ROS.
This study demonstrates that a text-mining approach combined with association analysis may be useful to identify candidate genes that cause hypertension through ROS or oxidative stress.
Title: Distribution of number of genes according to number of papers

Title: Frequent reactive oxygen species (ROS)-hypertension genes (number of papers &gt; 40)

Title: Associated genes of hypertension (p &lt; 0.01)

Title: Top 10 most significant pathways through pathway enrichment analysis with 68 genes
Title: A Fosmid Cloning Strategy for Detecting the Widest Possible Spectrum of Microbes from the International Space Station Drinking Water System

In this study, fosmid cloning strategies were used to assess the microbial populations in water from the International Space Station (ISS) drinking water system (henceforth referred to as Prebiocide and Tank A water samples).
The goals of this study were: to compare the sensitivity of the fosmid cloning strategy with that of traditional culture-based and 16S rRNA-based approaches and to detect the widest possible spectrum of microbial populations during the water purification process.
Initially, microbes could not be cultivated, and conventional PCR failed to amplify 16S rDNA fragments from these low biomass samples.
Therefore, randomly primed rolling-circle amplification was used to amplify any DNA that might be present in the samples, followed by size selection by using pulsed-field gel electrophoresis.
The amplified high-molecular-weight DNA from both samples was cloned into fosmid vectors.
Several hundred clones were randomly selected for sequencing, followed by Blastn/Blastx searches.
Sequences encoding specific genes from Burkholderia, a species abundant in the soil and groundwater, were found in both samples.
Bradyrhizobium and Mesorhizobium, which belong to rhizobia, a large community of nitrogen fixers often found in association with plant roots, were present in the Prebiocide samples.
Ralstonia, which is prevalent in soils with a high heavy metal content, was detected in the Tank A samples.
The detection of many unidentified sequences suggests the presence of potentially novel microbial fingerprints.
The bacterial diversity detected in this pilot study using a fosmid vector approach was higher than that detected by conventional 16S rRNA gene sequencing.
Rationing and water recycling are an essential part of life at International Space Stations (ISSs) [1].
Water purification machines on the ISS partly mimic these processes, but they do not rely on microbes or any other living organisms.
These machines cleanse wastewater in a 3-step process.
The first step involves a filter that removes particles and debris.
The water then passes through multifiltration beds that contain substances that remove organic and inorganic impurities.
Finally, the catalytic oxidation reactor removes volatile organic compounds and kills bacteria and viruses.
Once the water is purified, astronauts try to ensure that the water is used with maximum possible efficiency.
Even with intense conservation and recycling efforts, the space station will gradually lose water because of inefficiencies in the life support system.
Water that passes through animals&#39; bodies is refreshed by natural processes [2].
Microbes in the soil break down urea and convert it to a form that plants can absorb and use to build new plant tissue.
Water excreted by animals also evaporates into the atmosphere and rains back down to the earth as freshwater-a natural form of distillation.
ISSs always need to be resupplied with water, because no water reprocessing technology is currently available for use in space.
National Aeronautics and Space Administration (NASA) scientists continue to look for ways to improve the life-support systems of space stations, which can thereby reduce water losses and aid in finding ways to reuse other waste products.
Bacteria and viruses are extraordinarily abundant in water, soil, and sediment microbial ecosystems [3].
In drinking water, soil, and sediment, viral abundance exceeds that of co-existing host populations by 10- to 100-fold [4].
Microscopy and molecular genetic tools have been critical in demonstrating that bacteria and viruses are dynamic components of microbial ecosystems capable of significantly influencing the productivity and population biology of their host communities.
Moreover, these approaches have begun to describe the immense genetic diversity of viral communities.
A critical first step in the application of many cultivation-independent approaches to virus ecology is obtaining a viral concentrate from an environmental sample.
Culture-dependent methods also rely on viruses being present at sufficiently high levels to be detected [3].
In this study, the methodology for the isolation and concentration of viruses from water, soil, and aquatic sediment samples has been discussed in detail.
Microorganisms constitute two-thirds of the earth&#39;s biological diversity [5, 6].
As many as 99% of the microbes present in certain environments can not be detected by regular techniques [7].
Protocols have been developed to assess unexplored microbial diversity in order to overcome the existing barriers that prevent the estimation of biodiversity [8].
New screening methods have been designed to construct specific fosmid libraries that can help distinguish novel microbial populations in drinking water samples [9].
To screen the complete populations, cloning vectors, like fosmids and bacterial artificial chromosomes (BACs), are being developed [10-12].
This article describes the various methodologies and tools that have been developed to understand and identify the widest spectrum of microbial communities, including bacteria, archaea, and viruses, through fosmid cloning analysis.
Bacteriophage phi29 (Φ29) DNA polymerase, the 66-kD product of viral gene 2, has been shown to be capable of efficient DNA amplification both in vitro and in vivo [13-15].
This enzyme is very reliable with respect to the amplification of genomic DNA for constructing libraries, especially when the starting genomic material is limited.
This enzyme requires 2 proteins for its activity: viral DNA polymerase P2 and Φ29 terminal protein p3 [16].
The enzyme has high processivity (&gt;70 kb) and a remarkable strand-displacement ability with single-stranded DNA and double-stranded DNA.
Based on this enzymatic characteristic, the replication of both DNA strands can be processed from each terminal-priming event without the synthesis of RNA-primed Okazaki fragments.
The enzyme also has a 3&#39;-5&#39; proofreading exonuclease function [17].
Bacteriophage ϕDNA polymerase amplifies circular DNA in a rolling-circle amplification (RCA) mechanism.
This characteristic has been used to amplify and clone the complete circular DNA genome of bacteria and viruses.
The amplified DNA could be used for direct sequencing and was cloned after digestion with a single cutting restriction endonuclease.
The use of this enzyme simplified the cloning steps and increased the cloning efficiency of the complete genome of circular DNA viruses [18].
The Φ29 DNA polymerase was ideal for our project because of the limited amount of starting material available.
In addition, the microbes in our sample could not be cultivated using traditional approaches.
Using exonuclease-resistant random primers, Dean and colleagues have achieved up to a 10,000-fold amplification from 1 ng of M13 template and have also improved the average product length to greater than 40 kb [19].
The method is based on multiple displacement amplification technology and provides high yields of approximately 40 µg of DNA from only a few genome copies.
In a previous study on subsurface flow-constructed wetlands, PCR-based methods were developed and used for the detection of pathogenic microbial populations, such as Campylobacter spp.and Yersinia enterocolitica, in treated wastewater [20, 21].
In this study, we constructed fosmid libraries to assess the microbial diversity in drinking groundwater samples from Kennedy Space Center&#39;s ground service equipment tank and the space shuttle water tank (iodine-treated water) and examined the samples for the presence of many different bacterial populations during the water purification processes.
The goals of this study were: 1) to compare the sensitivity of the fosmid cloning strategy with that of traditional culture-based and 16S rRNA-based approaches and 2) to detect the widest possible spectrum of microbial populations during the water purification process.
Title: Collection of water samples

Two drinking water samples from the Kennedy Space Center, Orlando, Florida, USA-namely, water samples from the ground service equipment tank (Prebiocide) and the space shuttle water tank (iodine-treated water; Tank A)-were collected for microbial diversity assessment.
Title: Isolation of genomic DNA

Approximately 500 mL each of the Prebiocide and Tank A water samples was concentrated using several Amicon Ultra-15 filters (50K NMWL; Millipore, Billerica, MA, USA).
The filtered concentrates were combined together and further concentrated to 100 µL by using a vacuum evaporator (HT-4 II Evaporation system; Genevac Inc., Valley Cottage, NY, USA) at 28℃ for 2 h. DNA isolation was conducted using Trizol reagent, following the instructions provided by the manufacturer (Invitrogen, Carlsbad, CA, USA).
During the last step, the DNA pellets were dissolved in 10 µL of 8 mM NaOH and then incubated at 60℃ for 10 min.
Title: Amplification of genomic DNA

The purified DNA was subjected to randomly primed RCA to amplify any DNA present in the water samples, according to the procedure detailed in the GenomiPhi DNA Amplification Kit manual (Amersham Biosciences, Piscataway, NJ, USA).
This kit consists of 3 components: the sample buffer; the reaction buffer; and an enzyme mix containing random primers, dNTPs, and Φ29 DNA polymerase [22].
One microliter of the concentrated water sample from either the Prebiocide or Tank A was denatured with 9 µL of sample buffer at 95℃ for 5 min and cooled rapidly on ice.
The amplification reaction was performed by adding 9 µL of reaction buffer and 1 µL of enzyme mix to the denatured sample.
The reaction mixtures were incubated at 30℃ for 16 h. After amplification, the tubes were incubated at 65℃ for 10 min to inactivate the Φ29 DNA polymerase enzyme.
The amplified genomic DNA was precipitated by ethanol and then dissolved in 10 mM Tris-HCl (pH 7.5) and 1 mM ethylenediaminetetraacetic acid.
Most of the random primers and dNTPs were removed after precipitation with ethanol.
Title: Construction of the fosmid library

The amplified high-molecular-weight DNA from the Prebiocide or Tank A water samples was cloned with fosmid vectors.
In order to select DNA fragments longer than 20 kb, the amplified DNA was visualized by pulsed-field gel electrophoresis (PFGE) in 1% low-melting-point agarose at 4.5 V/cm in 1 × TBE buffer for 13 h. DNA fragments larger than 20 kb were cut out, extracted from the gel, and re-run for 5 h under the same conditions.
These resolved DNA fragments were used to construct a fosmid library by following the procedure detailed by the manufacturer.
Using the CopyControl Fosmid Library Construction Kit (Epicentre, Madison, WI, USA), the sheared DNA was end-repaired to generate blunt 5&#39;-phosphorylated ends and then ligated directly into the cloning-ready CopyControl pCC1FOS vector (Fig.1) at a 10:1 vector:insert ratio.
The ligated DNA was then packaged using ultrahigh-efficiency MaxPlax Lambda Packaging Extract (Epicentre) and was plated on phage T1-resistant Escherichia coli cells to produce the library.
Packaging a single 10-µL ligation reaction generated more than 100,000 fosmid clones from the DNA isolated from the Prebiocide and Tank A water samples.
DNA from several randomly chosen clones was purified using the Direct Lysis Fosmid96 Kit (Epicentre) for high-throughput fosmid sequencing.
Title: Restriction enzyme analysis

One microliter of the RCA product from the Prebiocide or the Tank A samples was incubated with 10 U of HindIII at 37℃ overnight.
Restriction digestion products were separated by 1% agarose gel electrophoresis and visualized by ethidium bromide staining.
In addition, among the 9,740 clones obtained from the fosmid library constructs, subsets of randomly selected clones from the Prebiocide or Tank A samples were digested with NotI restriction enzyme for measuring insert DNA sizes.
Title: DNA sequencing

Each colony from the different libraries was picked and then inoculated in Luria Broth media containing chloramphenicol and glycerol (10%, final concentration) in a 384-well plate.
The cells were cultured at 37℃ for 1 day and stored at -80℃.
DNA was isolated using the Qiagen Miniprep Kit (Qiagen, Valencia, CA, USA) according to the manufacturer&#39;s instructions.
Initially, 100 clones from each library were selected for the sequencing reaction (Macrogen, Seoul, Korea).
BLAST sequence homology searches were conducted against the GenBank database after subtracting the vector sequence.
Subsequently, 1,260 colonies from the Prebiocide library and 2,600 colonies from the Tank A library were sequenced.
The major sequences encoding specific genes were identified from both Prebiocide and Tank A samples on the basis of protein sequences (with more than 75 amino acid residues) from GenBank.
Title: PCR detection of different microbial groups in drinking water samples

DNA was extracted from the Prebiocide and Tank A water samples by using the Qiagen Miniprep Kit.
Several microbial groups were analyzed by PCR amplification of gene fragments of several phylogenetic groups representing different microbial communities [23].
These groups included general bacteria, Bacillus species, gram-positive bacteria (high G + C content), Streptomyces species, eukaryotes (fungi, protists, and green algae), and general fungi.
The PCR primer pairs used in the analysis and the groups they represent are listed in Table 1.
Title: Genomic DNA amplification using the Phi29 DNA polymerase

The DNA plug did not show any detectable amount of DNA when analyzed using PFGE.
The GenomiPhi DNA Amplification Kit (Amersham Biosciences) is claimed to have very high sensitivity for DNA detection; the smallest trace of DNA should be amplified if present.
In the protocol provided with the amplification kit, it is recommended that the amplification reaction be started with 10 ng of DNA.
Although the results initially showed that the concentrated samples did not contain any detectable amount of DNA, the amplification procedure was continued with these samples.
The control (DNase free water) and the DNA isolated from the Prebiocide and Tank A samples showed a similarly positive pattern after DNA amplification, which seemed to be nonspecific due to the contamination of the Φ29 DNA polymerase enzyme with exogenous DNA (Fig.2A).
Similarly, the positive controls, such as lambda DNA, showed significant amplification of up to 20-30 µg in a 20-µL reaction.
The distribution of the amplified DNA ranged from 20 to 40 kb.
Repeated trials of less than 3 h did not result in any amplification; however, a DNA product was noted after 6 h of incubation.
The size of the amplified DNA (between 20 kb and 40 kb) was determined by PFGE.
The GenomiPhi DNA Amplification Kit manual explains that nonspecific amplification usually occurs, even in the absence of template DNA, because of the presence of random primers and the high amplification performance capability of the enzyme.
In order to confirm whether the amplified DNA had been specifically amplified from the genomic DNA template, 1 µg of each of the amplified DNA samples was digested with HindIII (Fig.2B).
Several fragmented DNA bands with expected sizes were observed in the digestion reaction of the amplified DNA from the lambda (positive control), Prebiocide, Tank A, and control samples, showing that the amplification was specific.
Smeared bands of indigestible DNA were also obtained because of partial and/or nonspecific amplification of the genomic DNA.
Title: Construction of fosmid vector libraries

Before constructing the fosmid vector libraries, several domain or species-specific PCR reactions were performed.
Both the Prebiocide and Tank A samples and the control (DNase-free water) showed amplification with different primer sets (Fig.3).
Primer pairs used for species-specific PCR amplification to detect the microbial diversity are listed in Table 1 [24-26].
After constructing the fosmid vector libraries, 4,000 clones from the Prebiocide sample, 2,290 clones from the Tank A sample, and 3,450 clones from the control were picked.
As expected, NotI digestion of the fosmid DNA released and produced inserts that were more than 40 kb in size (Fig.4).
Subsets digested with the NotI enzyme were sequenced.
Only DNA sequences that exhibited more than 80% similarity with an existing GenBank sequence were included in the analysis.
Title: DNA sequence analysis

BLAST sequence homology searches were also performed against the GenBank database after subtracting the vector sequence.
Subsequently, 1,260 single colonies from the Prebiocide library and 2,600 single colonies from the Tank A library were picked.
The major sequences encoding specific genes were then identified from the Prebiocide (26 clones), Tank A (27 clones), and control samples (21 clones) on the basis of the GenBank protein sequences.
Supplementary Table 1 shows representative BLAST search results from GenBank for the Prebiocide, Tank A, and control sample clones.
Title: Identification of microbial populations and diversity

The microbial populations, identified on the basis of sequences for both the Prebiocide and Tank A drinking water samples, are listed in Table 2 and Supplementary Table 1.
Bradyrhizobium japonicum, Mesorhizobium loti, and Bradyrhizobium species were present in the Prebiocide library.
Bradyrhizobium and Mesorhizobium belong to rhizobia, a large community of nitrogen fixers often found in association with plant roots [27].
Ralstonia metallidurans and R. eutropha were detected in Tank A.
Strains of R. metallidurans (renamed as Cupriavidus metallidurans strain CH34) are prevalent in soils with high heavy metal content [28].
Sequences encoding specific genes from Burkholderia fungorum, a species abundant in soil and groundwater, were found in both the Prebiocide and Tank A samples [23, 29].
Protein sequences homologous to those of several genera of opportunistic pathogens were retrieved by RCA.
The genomic DNA amplification by PCR detected different microbial groups in drinking water samples (Table 1, Fig.3).
In the current study, a fosmid cloning strategy was used to assess the microbial populations in the ISS drinking water system-namely, samples from the Prebiocide and Tank A.
Initially, no microbes could be cultivated, and conventional PCR reactions failed to amplify 16S rDNA fragments from low biomass samples.
Therefore, randomly primed RCA was used to amplify any DNA present in the samples, followed by size selection using PFGE.
Genomic DNA amplification was performed using the DNA isolated from concentrated DNA samples obtained from Prebiocide and Tank A drinking water samples in the ISS.
We used the GenomiPhi DNA Amplification Kit from Amersham; the DNA amplification experiments using a Φ29 DNA polymerase proved to be ideal for the scarce amount of genomic starting material available.
However, nonspecific amplification can occur either in the presence or in the absence of the DNA template in the reaction.
For the lambda DNA sample, the presence of amplified DNA can be confirmed if the Φ29 DNA polymerase is specific to the template DNA by digestion with restriction enzymes.
Lambda DNA (48.5 kb) has many HindIII restriction sites, producing a unique distribution of sizes.
However, even in the case of lambda DNA, we confirmed that some amount of indigestible amplified DNA still exists, which appears to be due to nonspecific amplification and/or partial amplification (Fig.2B).
According to the Amersham kit user manual, the enzyme synthesizes single-stranded DNA or double-stranded DNA.
An alternative explanation is that there were many single-stranded DNA fragments that could not be cut by restriction enzymes.
To test this theory, the amplified DNA was denatured at 95℃ for 3 min and reannealed at room temperature for 30 min to decrease the amount of single-stranded DNA.
However, restriction digestion still resulted in a large amount of indigestible DNA (data not shown).
Using the concentrated water samples also proved that such samples had sufficient DNA, which could be used to amplify genomic DNA by using commercially available kits, even when only a few microbes were present.
Nevertheless, amplification of genomic DNA may require some additional DNA preparation steps, because boiling cells for a short time may not effectively release the genomic DNA for use as template.
We decreased the incidence of nonspecific amplification by reducing the incubation time to less than 12 h when it was possible to use the amplified DNA as a template.
The amplified high-molecular-weight DNA obtained from the Prebiocide, Tank A, and control samples was cloned into fosmid vectors.
Several hundred clones were randomly selected for sequencing, which was followed by Blastn/Blastx searches (Table 2, Supplementary Table 1).
Sequences encoding specific genes from Burkholderia, a species abundant in soil and groundwater, were obtained for both the Prebiocide and Tank A samples [23, 29].
Bradyrhizobium and Mesorhizobium, belonging to rhizobia, a large community of nitrogen fixers often found in association with plant roots, were present in the Prebiocide library [27].
Ralstonia, which is prevalent in soils with a high heavy metal content, was detected in the Tank A library [28].
Giardia lamblia is a common cause of diarrhea in humans, and the presence of homologous protein sequences with the protein encoded by G. lamblia in Tank A samples warrants further study [30].
The current NASA technologies used for treating drinking water in the ISS are effective in removing viable cultivable microbes.
Conventional PCR failed to detect any amplifiable rRNA genes, which suggests that the purification of this drinking water was satisfactory.
However, the amplification of naked DNA from these samples enabled elucidation of the microbial diversity in the samples.
When drinking water was not treated with a biocide, protein sequences from diverse genera could be retrieved.
The detection of many unidentified sequences (data not shown) also suggests the presence of potentially novel microbial fingerprints.
The bacterial diversity observed in this pilot study with the fosmid approach was greater than that observed with conventional 16S rRNA gene sequencing.
CopyControl pCC1FOS Fosmid vector map.
The pCC1FOS cloning vector was linearized at its Eco72 I (blunt) site, dephosphorylated, and used for cloning blunt-end genomic DNA.
CMR, chloramphenicol resistance.
DNA gel pictures.
(A) Genomic DNA amplification by rolling-circle amplification for Prebiocide and Tank A drinking water samples.
Lane M, 5 kb DNA ladder; lane 1, Prebiocide DNA product; lane 2, negative control (no Φ29 DNA polymerase); lane 3, Tank A DNA product; lane 4, positive control (lambda DNA 10 ng); lane 5, control (DNase-free water).
Amplification conditions: 6 h, 30℃.
The amplified DNA was analyzed by 1% agarose gel electrophoresis in 0.5× TBE buffer at 150 V. (B) Restriction digestion of the amplified genomic DNAs by using HindIII enzyme.
Lanes are same as in (A).
Lane M2, 1 kb DNA ladder.
The digested samples were analyzed by performing routine 0.8% agarose gel electrophoresis in 1× TBE buffer at 150 V.
Genomic DNA amplification and detection of different microbial groups by PCR in drinking water samples.
Lane M, DNA ladder; lane 1, amplified Prebiocide DNA product; lane 2, amplified Tank A DNA product; lane 3, amplified control DNA product; lane 4, no DNA input for PCR.
The amplified DNA product was analyzed by 1% agarose gel electrophoresis in 0.5× TBE buffer at 150 V. The arrowhead shows the amplification of genomic DNA and the presence of corresponding microbial DNA.
NotI restriction enzyme digestion.
A combination of NotI restriction enzyme digestion of fosmid DNA isolated from randomly chosen clones from the Prebiocide, Tank A, and control (DNase free water) libraries and ethidium bromide-stained agarose gel electrophoresis verifies the presence of a &gt;40-kb insert.
Lane M, DNA ladder; lanes 1-10 show NotI-digested fosmid DNA selected from the Prebiocide, Tank A, and control libraries.
Pulsed-field gel electrophoresis conditions: 4.5 V/cm, 1-7 s, 13 h, 1× TBE buffer at 14℃.
Title: Primer pairs used for PCR amplification to detect the microbial diversity in the Prebiocide and Tank A drinking water samples

Title: List of microbial populations identified on the basis of sequences in both the Prebiocide and Tank A drinking water samples
Title: Network Graph Analysis of Gene-Gene Interactions in Genome-Wide Association Study Data

Most common complex traits, such as obesity, hypertension, diabetes, and cancers, are known to be associated with multiple genes, environmental factors, and their epistasis.
Recently, the development of advanced genotyping technologies has allowed us to perform genome-wide association studies (GWASs).
For detecting the effects of multiple genes on complex traits, many approaches have been proposed for GWASs.
Multifactor dimensionality reduction (MDR) is one of the powerful and efficient methods for detecting high-order gene-gene (GxG) interactions.
However, the biological interpretation of GxG interactions identified by MDR analysis is not easy.
In order to aid the interpretation of MDR results, we propose a network graph analysis to elucidate the meaning of identified GxG interactions.
The proposed network graph analysis consists of three steps.
The first step is for performing GxG interaction analysis using MDR analysis.
The second step is to draw the network graph using the MDR result.
The third step is to provide biological evidence of the identified GxG interaction using external biological databases.
The proposed method was applied to Korean Association Resource (KARE) data, containing 8838 individuals with 327,632 single-nucleotide polymorphisms, in order to perform GxG interaction analysis of body mass index (BMI).
Our network graph analysis successfully showed that many identified GxG interactions have known biological evidence related to BMI.
We expect that our network graph analysis will be helpful to interpret the biological meaning of GxG interactions.
In recent years, a significant advance of technology has been achieved in both genotyping and computing fields, which has enabled us to perform large-scale genome-wide association studies (GWASs) [1, 2].
GWASs have provided enormous potential in identifying genetic variants for common complex diseases.
By GWASs, researchers have discovered many genetic variants that affect common complex traits, such as height, type 2 diabetes, and blood pressure [3-5].
In particular, the nature of obesity is related with many chronic diseases, such as type 2 diabetes, cardiovascular disease, and cancers [6-8].
In addition, it has been found that obesity may be caused by multiple genes, environmental factors, and those interactions [9-11].
In order to identify the genes associated with obesity, body mass index (BMI) has been commonly used as an obesity-related phenotype or trait in GWASs.
Many studies have reported that the genetic variants in FTO are associated with BMI [12, 13].
Most of these studies have used the single single-nucleotide polymorphism (SNP) approach, in which each single susceptible locus is searched in association with obesity.
The single SNP approach has long been used in detecting genetic traits, and it is especially optimal for Mendelian traits.
However, complex traits are well known to be caused by multiple genetic factors as well as several environmental factors and their interactions.
Thus, the single SNP approach is not appropriate for detecting multiple genetic factors affecting common complex polygenic traits.
For the identification of multiple and joint genetic factors associated with complex traits, the multifactor dimensionality reduction (MDR) method has been proposed by Ritchie et al.
[14] and applied in detecting gene-gene (GxG) interactions [15, 16].
However, their approach can be applied only to binary traits, and the covariates that may affect the phenotype can not be adjusted.
In order to overcome these drawbacks, generalized MDR (GMDR) analysis was proposed by Lou et al.
[17], in which both binary and quantitative traits can be considered phenotypes while adjusting for covariates.
Although MDR and GMDR have been widely used to detect GxG interactions in genetic association studies, their applications have been mainly involved in dealing with candidate genes or a small set of genes.
Since enormous computational complexity and high-speed computing resources are required in GWASs, neither MDR nor GMDR has been successfully applied to GWASs.
In order to overcome these computational limitations, we previously proposed an efficient stepwise approach to identify GxG interactions for GWAS data [18] using cuGWAM, a graphic processing units (GPU)-based MDR program [19].
However, it still remains a challenge to have a biological interpretation of the identified GxG interactions.
In order to elucidate the identified GxG interactions, we propose a network graph analysis.
Our network analysis method focuses not only on the biological meaning but also on the strength of interaction.
We applied our network graph approach to Korean Association Resource (KARE) data for GxG interactions and its interpretation.
We first performed GMDR analysis for BMI.
Using our network graph analysis, we then identified several GxG interactions having reasonable biological interpretation from the network graph using public biological resources.
Title: The KARE data

The KARE project, established as part of the Korean Genome Epidemiology Study (KoGES) in 2001, has started to undertake a large-scale genome-wide association analysis in Korean population-based cohort.
These cohorts consist of 10,038 participants in the urban Ansan (n = 5,020) and rural Ansung (n = 5,018) communities.
The objective of the KARE project is to identify genetic factors of several clinical traits and lifestyle-related diseases by GWAS, based on population-based cohorts.
In this analysis, we focused on BMI, one of more than 260 lifestyle-related traits.
DNA samples were isolated from the peripheral blood of all participants and were genotyped with Affymetrix Genome-Wide Human SNP array 5.0 (Affymetrix, Rockville, MD, USA).
Genotypes were called using Bayesian Robust Linear Modeling using the Mahalanobis distance (BRLMM) algorithm [20].
We performed a quality control process for the sample and genotype as previously described in Cho et al.[2].
In order to increase the coverage of common variants and capture additional association signals, we performed SNP imputation with PLINK using the Japanese/Chinese (JPT/CHB) component of HapMap as the reference.
After sample and genotype quality controls, 327,872 SNPs for 8,842 individuals were available in the KARE data.
We applied our proposed regularization method to Korean genome-wide data.
Title: Network graph analysis

Title: GMDR analysis step

GWAS datasets usually have enormous number of markers, which causes a severe computational burden.
In order to deal with this computation problem, the single SNP association test with covariate adjustment is performed to all SNPs in a dataset.
This step can be achieved via several toolkits that support regression analysis, such as PLINK.
Filtering by specific criteria, such as p-value, is subsequently performed in order to reduce the number of SNPs.
With the selected SNPs from the single SNP association analysis, exhaustive two-way GMDR analysis is then applied.
However, general toolkits supporting GMDR analysis can only handle hundreds of SNPs.
Although the number of SNPs in this step can be reduced by giving more stringent criteria, it may ignore the SNPs having marginally less significant effects.
In addition, a scheme to reduce overfitting, such as cross-validation, requires increased linear computation time.
In order to solve this problem, faster toolkits are recommended, such as GPU-based GMDR analysis toolkits like cuGWAM [19].
Since the number of combinations in GMDR analysis is usually very large, all results of GMDR can not be stored.
Thus, a criterion is needed to store the specific interactions satisfying the criterion, such as balanced accuracy (BA).
Title: Network graph construction

Network graph is often used to visualize multiple sets of interactions.
It consists of nodes and edges connecting two nodes.
For simplicity of representation, we focus on two-way interactions.
Since the two-way interaction is for the relationship between two variables, it can be easily represented as a graph that contains two nodes representing two variables of interaction and an edge corresponding to the interaction.
According to this scheme, all interactions identified by GMDR analysis can be visualized in the network graph.
However, this scheme can cause too complex a graph structure when the number of interactions is large.
Thus, it is recommended to select a certain number of possible interactions in the network graph to reduce the complexity of the network graph.
Alternatively, the complexity of the network graph can be reduced by gene mapping information.
Since many SNPs are placed in a gene or adjacent to a gene, it is possible to annotate multiple SNPs into a single node of a gene according to their position.
However, this approach requires a delicate mapping process.
In addition, the network graph can represent the strength of the interaction by the thickness of the edge-that is, the higher the number of interactions between two nodes, the thicker the edge becomes.
Title: Network graph analysis

It is important to derive a reasonable biological interpretation of identified GxG interactions.
Here, we propose a network graph analysis step to aid the biological interpretation of the identified GxG interaction by using existing biological resources.
In other words, the goal of this step is to find known biological evidence that supports the identified GxG interactions.
Resources used in this step consist of either a pair of a marker and target or two related markers, where a marker corresponds to a gene or SNP and a target represents a biologically proven target related with a marker.
From a resource with marker and target information, such as HuGENet [21], miRBase [22], and is-rSNP [23], we simply assume that there is biological evidence if two SNPs within an interaction share the same biological target defined in the biological resource.
For resources, such as COXPRESdb [24], having information about the correlation among genes, we assume that all correlated genes in these resources have biological interactions.
Although this is a naïve assumption, the use of these resources is mainly to support the interactions identified by GMDR analysis.
Note that many known biological resources have their own unique data structures that differ from each other.
Consequently, the integration of multiple resources is recommended in order to provide further biological evidence for a given GxG interaction.
In our GMDR analysis using KARE data, we first performed the single SNP association test via PLINK (http://pngu.mgh.harvard.edu/~purcell/plink/) using a linear regression model with covariate adjustment of age, area, and sex.
We used the p-value from linear regression as a criterion of screening and used a p-value threshold of 3 × 10-1 for the screening process; 101,837 SNPs were carried out after the screening process, as a result of the single SNP association test.
The top 10 SNPs with the smallest p-values are listed in Table 1.
The GMDR analysis was then performed using our GPU-based software, cuGWAM [19], in order to evaluate all possible two-way interactions and used a 10-fold cross-validation scheme, with covariate adjustment of age, area, and sex.
An exhaustive search for two-way interactions from the selected SNPs took 19 h on the GPU system with three GTX285 graphic cards.
In this step, a total of 51,853,363,660 possible two-way interaction models were evaluated.
In GMDR analysis, 10-fold cross-validation was used.
For each cross-validation set, the top 10,000 interactions with the highest test BAs were selected, where BA is defined as the arithmetic mean of sensitivity and specificity [25].
We then calculated cross-validation consistency (CVC), which represents how many times the same two-way interaction is selected out of 10 cross-validation sets.
Finally, we performed a screening step for every pair of interactions achieved from GMDR analysis that satisfied the two criteria, CVC ≥ 9 and test BA ≥ 0.5.
Finally, we found 524 two-way interactions under these screening criteria.
Among these interactions, 127 are from genes that are known to be associated with obesity and are summarized in Table 2.
Among these interactions, five genes-FTO, CBARA1, CHUK, SLIT3, and NRXN3-and 59 SNPs have been reported in previous studies [15, 16, 26-29].
Visualization of these 524 two-way interactions into a network graph was then made after annotating SNPs to the genes.
We used Gephi (https://gephi.org) as a visualization tool.
Gene annotation was performed according to the hg18 human genome reference and dbSNP 129.
These 524 twoway interactions are displayed in the network graph (Fig.1).
Each node represents either an SNP or gene.
If a SNP is annotated to a known gene, then it is denoted by the gene name.
The number of interactions between nodes was represented as the thickness of the edge.
For example, node FTO has a very thick edge with node NT5C2.
Conversely, some nodes have a large number of nodes that are connected with it.
We call these nodes hub nodes or hub SNPs.
Hub nodes are represented by a gray background color.
We investigated which SNPs that were included in the 524 two-way interactions are related with obesity using DAVID [30], a comprehensive set of functional annotation tools.
As a result, we detected six genes-NDUFA8, NDUFB8, NDUFC2, NDUFS2, COX5A, and ATP6V1B2.
The role of these genes in the metabolism pathway is related to oxidative phosphorylation, which plays an important role of hepatic mitochondrial function in the development of obesity [31].
Among the 524 two-way interactions, some SNPs might have weak main effects but have strong interactions with other SNPs.
Table 3 shows the SNPs from hub genes that show significant two-way interactions with other SNPs but have weak marginal effects in the single SNP analysis.
For the network graph analysis, four public databases (HuGENet [21], COXPRESdb [24], miRBase [22], and isrSNP [23]) were collected and used to investigate the biological relationship between two SNPs with an interaction.
A short summary of each database is given in Table 4.
All databases used in this study were converted from the original bulk to a database table with automated script for the integrated investigation.
The script for this conversion can be provided upon request.
In order to improve the biological relevance of the interpretation, we excluded all interactions having linkage disequilibrium (LD) between two SNPs.
One of the results of the network graph analysis is given in Fig.2.
We identified at least one line of biological evidence among 26% of the 524 GxG interactions.
In order to investigate the relationship between CVC and the rate of biological evidence, we performed an additional network analysis using 1,838 interactions (CVC ≥ 7) and found out that there was a strong relationship between CVC of the GMDR analysis and the presence of known biological evidence.
As shown in Table 5, the proportion of interactions having a known biological interaction was significantly higher when CVC = 10 than when 7 ≤ CVC &lt; 10 (39% vs. 17% on average).
In addition, from the network analysis of 179 interactions having CVC = 10, 69 interactions showed biological evidence and 30 (43% of 69) shared a known relationship against BMI-related diseases, such as cardiovascular disease, body weight, hypertension.
The main purpose of a GWAS is to detect genetic variants represented by a single SNP or a set of SNPs that are associated with common complex polygenic traits of interest [2, 4, 5, 16, 17].
However, it is commonly known that multiple genetic factors as well as several environmental factors and their interactions are involved in causing complex diseases.
Thus, the single SNP approach is not appropriate for detecting multiple genetic factors for common complex polygenic traits.
Recently, many researchers have tried GxG interaction analysis in GWASs.
However, they focused on rather restricted regions with a small number of SNPs due to computational complexity [32, 33].
In this paper, we have successfully performed two-way GxG interaction analysis of 8,838 individuals with the selected 101,837 SNPs, which cover most genomic regions.
It became possible with the help of a GPU-based system with three GPU graphic cards.
It took only 19 h to perform 51,853,363,660 possible two-way interaction analyses with 10-fold cross-validation.
Our results showed that the FTO, CBARA1, CHUK, SLIT3, and NRXN3 genes are strongly associated with obesity.
In addition, our approach provided more information about possible GxG interactions.
Note that some SNPs having weak main effects showed strong interactions with other SNPs (Table 3).
It would have been very difficult to detect these SNPs through single SNP analysis, which demonstrates the usefulness of our approach in detecting GxG interactions.
In addition, our analysis provided some novel interactions that have not been reported in previous studies.
For example, 9 SNPs from the FTO gene were found to have interactions with SNPs from CNNM2, AS3MT, and NT5C2.
Similar interactions were found between LOC301738 and PDZRN4, as shown in Fig.1.
Our network graph summarizes two-way interactions between SNPs very efficiently.
We expect that this visualization tool could be helpful to researchers.
It is also quite efficient to identify specific patterns and hub genes.
Through our network graph analysis, we also identified several pieces of evidence that supported the identified interactions with known biological information.
From the investigation with four published databases, we found that 39% of a total of 179 interactions with CVC = 10 are known to have biological interactions from the four databases we collected.
In addition, our network graph analysis showed a higher identification rate of shared biological relevance when the CVC was large than when the CVC was low, which suggests that interactions having a high value of CVC tend to share more known biological evidence that are actually related the phenotype.
In summary, our network graph analysis from GxG interaction analysis successfully identified the relationship between interactions from GMDR analysis and actual biological relevance.
In addition, our strategy interpreted GxG interactions very effectively in finding known biological evidence using multiple public databases, especially in finding evidence that is actually related with the phenotype.
Visualized result of significant interactions that have their cross-validation consistency ≥9.
Arranged for readability: gray background indicates hub node, and red, white, blue, and yellow names indicate that they are identified for their relation with obesity, single-nucleotide polymorphism, gene, and unidentified gene locus, respectively.
A visualization of gene-gene interaction interpretation with biological knowledge.
Two red circles denote two single-nucleotide polymorphisms (SNPs) within a two-way interaction, and purple circles denote corresponding genes against two SNPs.
Gray circles denote diseases that are known to be related with both of the genes.
Yellow and orange circles denote a disrupted transcriptional factor by both of the two SNPs and gene sets including genes from both SNPs, respectively.
Title: Top 10 SNPs from linear regression analysis

Title: Result of two-way interaction test

Title: Single nucleotide polymorphisms (SNPs) with weak marginal effect and strong interaction

Title: Short summary of biological knowledge used in this study

Title: Proportion of known biological interactions by cross-validation consistency (CVC)
Title: QCanvas: An Advanced Tool for Data Clustering and Visualization of Genomics Data

We developed a user-friendly, interactive program to simultaneously cluster and visualize omics data, such as DNA and protein array profiles.
This program provides diverse algorithms for the hierarchical clustering of two-dimensional data.
The clustering results can be interactively visualized and optimized on a heatmap.
The present tool does not require any prior knowledge of scripting languages to carry out the data clustering and visualization.
Furthermore, the heatmaps allow the selective display of data points satisfying user-defined criteria.
For example, a clustered heatmap of experimental values can be differentially visualized based on statistical values, such as p-values.
Including diverse menu-based display options, QCanvas provides a convenient graphical user interface for pattern analysis and visualization with high-quality graphics.
Genomics and proteomics data are typically analyzed by hierarchical clustering, followed by visualization with heatmaps [1-3].
Various algorithms have been implemented in the data clustering procedure [4].
The visualization of clustered data includes tree-based hierarchical clustering patterns and heatmaps of experimental values [5].
Simultaneously carrying out clustering and visualization in a single platform provides a convenient tool for choosing an appropriate clustering algorithm and finding patterns in the resulting heatmaps.
Previously, bioinformaticists used programmable tools, such as R and Matlab, and commercial data-mining packages to analyze their data.
A simple and integrated program will allow experimental scientists to intuitively identify meaningful patterns from a large dataset without requiring knowledge of scripting computer languages or statistical theory.
Herein, we introduce a user-friendly tool, QCanvas, which integrates diverse clustering algorithms and an interactive heatmap display interface (Fig.1).
This program directly imports raw experimental data in a matrix format and displays these data in a heatmap.
Various clustering methods can be applied to two-dimensional data, with the real-time generation of clustered heatmaps.
Furthermore, subsets of heatmap data can be selectively displayed, based on user-defined filters.
QCanvas is an easy-to-use and powerful tool for fast data analysis and interpretation by bench scientists.
Without any knowledge of scripting languages and without any graphics-editing software, one can generate and customize tree-clustered heatmaps with high-quality graphics.
Title: Data clustering

QCanvas provides a total of eight popular measures for generating the similarity matrix-i.e., Correlation uncenter, Correlation center, Absolute corr-uncenter, Absolute corrcenter, Spearman rank, Kendall&#39;s tau, Euclidean distance, and City-block distance.
All of these measures have typically been included among the data clustering methods of previous tools [4].
In QCanvas, the calculation of the similarity matrix is selectively applied to the data for the x-axis and the y-axis independently.
Hierarchical clustering is simultaneously carried out based on the established similarity matrices.
QCanvas provides diverse algorithms for hierarchical clustering, such as the average method, centroid method, single method, and complete method.
QCanvas uses a standard window-based graphical user interface (GUI), providing multiple windows to comparatively visualize patterns of various combinations of similarity matrices and hierarchical clustering methods.
This program provides quantitative trees for displaying clustering patterns and similarity measures together.
Title: Heatmap optimization for pattern recognition

QCanvas software recognizes text-based data in a matrix format.
For demonstration purposes, a small microarray gene expression dataset is included in the software package and can be downloaded from the website (http://compbio.sookmyung.ac.kr/~qcanvas).
Once the input data are imported into the QCanvas window, a heatmap of the non-clustered data is displayed (Fig.2A).
The user can easily test various data-clustering and tree-building methods on the raw data and interactively select appropriate heatmaps with tree structures (Fig.2B).
The GUI provides various menu-based options to optimize the display of heatmaps, trees, and annotations.
The colors, locations, and sizes of the trees and the annotations can be customized in a flexible manner.
The scale and color scheme of the heatmaps can also be adjusted in an interactive window.
The node colors can be customized for positive, negative, missing, or zero values.
The color contrast between nodes can also be interactively adjusted.
The overall vertical or horizontal size of a component of a figure can be customized and saved in postscript format for a high-image quality.
Title: Data filtering for the selection of major markers

Heatmaps that are based on data clustering display the overall profiles of the experimental values for the given samples.
QCanvas provides a data-filtering option to selectively display data nodes satisfying a given threshold.
In the example shown in Fig.2C, data points with a 2-fold change (increase or decrease) in gene expression are selectively displayed.
In many cases, a dataset includes experimental values and statistical confidence levels together.
The option for data filtering in QCanvas is useful for analyzing patterns in the experimental data that are statistically significant.
One can filter the heatmap profiles using statistical confidence data that are included in a separate file.
In the example shown in Fig.2D, the gene expression data are filtered based on the p-values for the fold-change.
QCanvas can import two separate files together for simultaneous data clustering and filtering.
The GUI menu for data filtering enables the pattern analysis to be performed easily, without the need for manual data processing or the use of scripting languages.
This report introduces QCanvas, a program that provides a convenient and powerful interface for the pattern analysis of large-scale omics data.
This program enables the user to conduct data clustering, data filtering, and graphics editing simultaneously on an integrated platform.
These steps are typically performed on omics data, such as DNA (or protein) microarray data.
All essential functionalities were integrated into the user-friendly interface of QCanvas.
The simple and intuitive nature of this tool meets the practical needs of research scientists working on omics data who do not have expertise in bioinformatics approaches.
The program is freely available with demo data and a step-by-step tutorial through the website (http://compbio.sookmyung.ac.kr/~qcanvas).
Graphical user interface of QCanvas.
Data retrieving, processing, and visualization can be carried out through interactive, user-friendly menus.
QCanvas provides additional tools for the optimization of size, color, and shape of trees and heatmaps.
Illustration of data clustering and filtering by QCanvas.
The microarray data for 20 selected genes in five cancer samples were used for this demonstration.
Red represents up-regulated gene expression, and green represents down-regulated gene expression.
The scale is log2 (fold-change).
(A) Input matrix data are visualized using a heatmap.
(B) The clustering of both cancer samples and genes is interactively carried out using the graphical user interface.
(C) The selective display of genes with high or low expression in cancer samples (2-fold changes).
(D) The selective display of genes with significant changes (p &lt; 0.01).
Title: Developing JSequitur to Study the Hierarchical Structure of Biological Sequences in a Grammatical Inference Framework of String Compression Algorithms

Grammatical inference methods are expected to find grammatical structures hidden in biological sequences.
One hopes that studies of grammar serve as an appropriate tool for theory formation.
Thus, we have developed JSequitur for automatically generating the grammatical structure of biological sequences in an inference framework of string compression algorithms.
Our original motivation was to find any grammatical traits of several cancer genes that can be detected by string compression algorithms.
Through this research, we could not find any meaningful unique traits of the cancer genes yet, but we could observe some interesting traits in regards to the relationship among gene length, similarity of sequences, the patterns of the generated grammar, and compression rate.
In formal language theory a language is simply a set of strings of characters drawn from some alphabet, where the alphabet (terminal) is a set of symbols.
When we consider biological sequences simply as a language in the context of formal language theory (treating DNA, RNA, or protein sequences just as strings of alphabets of four nucleotides or 20 amino acids), a grammatical inference method based on formal language theory can be applied [1-3].
Nevill-Manning and Witten [4] pioneered the attempt to produce the context-free grammarof biological sequences in an automatic way.
This task can be formalized as the problem of finding the smallest context-free grammar by recursively replacing the repeats by a new symbol.
The algorithm builds a hierarchy of phrases by forming a new rule out of existing pairs of symbols, including non-terminal symbols.
Title: For example, if we consider the string &#34;atattattatt,&#34; the simplest way to represent the string by context-free grammar is the following:

Title: &lt;Grammar 0&gt;

Title: S → atattattatt

Title: The most frequently occurring sequence in the string is &#34;at,&#34; which occurs four times.
Thus, introducing a new nonterminal symbol, &#39;A,&#39; and creating a new rule for this yields the following modified grammar:

Title: &lt;Grammar 1&gt;

Title: S → AAtAtAt

Title: A → at,

where the grammar consists of a start symbol (i.e., S), two terminal symbols (i.e., a, t) represented by lowercase letters, two non-terminal symbols (i.e., S, A) represented by uppercase letters, and two production rules (i.e., S → AAtAtAt, A → at) with a left- and a right-hand side consisting of a sequence of these symbols.
Title: Repeatedly replacing the frequently occurring patterns &#34;At,&#34; again to a new nonterminal symbol, B, gives the following modified grammar:

Title: &lt;Grammar 2&gt;

Title: S → ABBB

Title: A → at

Title: B→ At,

where the grammar consists of a start symbol (i.e., S), two terminal symbols (i.e., a, t), three non-terminal symbols (i.e., S, A, B), and three production rules (i.e., S → ABBB, A → at, B → At).
By applying the three production rules by replacing an occurrence of the nonterminals on the left-hand side of the production rule with those that appear on the right-hand side, the string &#34;atattattatt&#34; can be derived from the non-terminal S by constantly applying a series of derivations: S → ABBB → atBBB → atAtBB → atattBB → atattAtB → atattattB → atattattAt → atattattatt.
Based on this concept, grammar-based compression algorithms have shown some success for various applications [4-7].
Especially, grammar can capture distant repetitions occurring far apart, which was a limitation of sliding window approaches.
However, grammar-based compression algorithms at this moment do not show the best performance for compression itself [6].
Thus, our motivation of this study is not to develop a new algorithm or find the most efficient way to compress biological sequences for storage purposes.
Our sole purpose of developing a new tool is to investigate any grammatical traits of biological sequences, based on formal language theory.
We have developed a slightly modified version of Sequitur [4] called JSequitur for automatically creating hierarchical structures of sequences [8], as in Fig.1.
Our main contribution is to improve Sequitur to work better in a graphic user interface (GUI) environment, as our main interest was in studying the generated grammar, rather than enhancing the compression rate itself.
JSequitur is implemented in Java and organized into six classes, as in Fig.2: Sequitur, Symbol, Guard, Terminal, Nonterminal, and Rule.
Sequitur class is called first and connects with all of the other classes.
Symbol class is the connecter class, which streams sequences of input to the system.
Rule class accesses Terminal and NonTerminal classes in order to create rules.
Guard class, which is based on digram uniqueness, is responsible for rule confirmation.
Thus, our string compression algorithm operates by reading in a new symbol and processing it by appending it to the top-level string and then examining the last symbols of that string; it then applies zero or more of the following transformations until none applies anywhere in the grammar; it then repeats the cycle by reading in a new symbol.
Title: The following production rules, which have been created automatically, are an exemplary output of applying JSequitur to the partial sequence of the TERT gene (175 bp, &#34;gcccccgggtgtccctgtcacgtgcagggtgagtgaggcgcggtccccgggtgtc cctgtcacgtgcagggtgagtgaggcgcggtccccgggtgtccctgtcacgtgcag ggtgagtgaggcgcggtcccc&#34;):

Title: R0 → g R1 R2 R3 R4 R5 R6 R7 R6 R8 R5 R9 R1 R3 R4 R10 R11 g a R4 a R12 R13 R14 R7 R15 R8 R16 R10 R14 c

Title: R1 → c c

Title: R2 → R1 c

Title: R3 → R11 R17 R2 R18

Title: R4 → R17 g

Title: R5 → R16 R13 R19

Title: R6 → t R2

Title: R7 → R19 R4

Title: R8 → R18 R4

Title: R9 → t R1

Title: R10 → a a

Title: R11 → R12 R17

Title: R12 → g g

Title: R13 → c g

Title: R14 → R19 R15

Title: R15 → R9 c

Title: R16 → R10 R11 g a R4 a R12

Title: R17 → g t

Title: R18 → t R17 R10 c

Title: R19 → R13 g,

where the grammar consists of a start symbol (i.e., R0), four terminal symbols (i.e., a, t, g, c), 20 non-terminal symbols (i.e., R0-R19), and 20 production rules for each nonterminal.
In summary, the partial sequence of 175 bp of the TERT gene could be compressed to 37 symbols with 20 rules.
For testing purposes, 104 cancerous genes from 6 cancer types (bladder, breast, endometrial, leukemia, lung, and melanoma) were initially chosen, and JSequitur was applied.
Table 1 shows the result of applying one of the string compression algorithms of JSequitur to these genes.
The rule column in Table 1 shows the number of generated rules from the context-free grammar, while the ratio column shows the ratios of the compressed sequences vs. the original sequences.
Fig.3 is a sorted diagram in the order of the length of the original sequence.
In this specific case, it shows that the length of the original sequence influences the compression rate of the target sequence, even though there are many other factors that influence compression rate.
For example, compression rate can also be influenced by the algorithm itself, depending on whether we replace the longest pattern first or the most frequently occurring pattern first.
We also compared some mouse genes to find any homologous traits in regards to compression rate and hierarchical structure of the grammar.
For example, we compared human MUC1 (Homo sapiens, 5,279 bp) with mouse MUC1 (Mus musculus, 5,614 bp), and the compression rates for these two sequences were 0.180 and 0.195, respectively.
For the ARHA genes, the compression rate for human ARHA (68,833 bp) was 0.140, whereas that for mouse ARHA (41,255 bp) was 0.157.
Thus, the distance on the evolutionary tree can be measured by compression algorithms, to a certain extent.
We have developed a GUI-based JSequitur, based on string compression algorithms, to examine grammatical traits of biological sequences.
On top of compression capacity, a string compression algorithm is appealing for studying biological sequences, because it can give insights into the structure of these sequences.
Precisely constructed models for linguistic structure can play a vital role in the process of discovery itself.
We also applied JSequitur to analyze 104 cancer genes for testing purposes only.
Even though there are some interesting results in regards to the relationship among gene length, similarity of sequences, the patterns of the generated grammar, and compression rate, our test samples were too small to conclude anything.
Thus, our result should be regarded as preliminary for future research.
We should consider various factors other than grammatical structures and compression rates.
As our main purpose of developing the tool was to examine any grammatical traits of biological sequences, the graphical user interface was important for a semiautomatic screening process.
However, we still need to implement various features to compare gene structures to summarize statistics in regards to grammatical structures and to combine evolutionary trees.
Hopefully, these features will be implemented in the next version of JSequitur.
We also hope to enhance the algorithm more elaborately to handle reversal, translocation, and shuffling.
User interface of JSequitur program.
JSequitur class diagram.
Compression rates in relation to gene length.
Title: One hundred four genes and their compression rates
Title: Preliminary Study of Bioinformatics Patents and Their Classifications Registered in the KIPRIS Database

Whereas a vast amount of new information on bioinformatics is made available to the public through patents, only a small set of patents are cited in academic papers.
A detailed analysis of registered bioinformatics patents, using the existing patent search system, can provide valuable information links between science and technology.
However, it is extremely difficult to select keywords to capture bioinformatics patents, reflecting the convergence of several underlying technologies.
No single word or even several words are sufficient to identify such patents.
The analysis of patent subclasses can provide valuable information.
In this paper, I did a preliminary study of the current status of bioinformatics patents and their International Patent Classification (IPC) groups registered in the Korea Intellectual Property Rights Information Service (KIPRIS) database.
There are many ways that research gaps between science and technology can be identified.
Bibliographic citation of both academic literature and patent documents is one of the solutions.
Unfortunately, patent documents are considerably less accessible [1], compared to academic papers.
Whereas vast amount of new information is made available to the public through patents, only a small set of patents are cited in academic papers [2].
Thus, the essential problem arising from citing patents related to bioinformatics is to devise a classification methodology that identifies the relevant patent and only those patents.
The suitability of the methodology depends on the purpose of the analysis.
However, no single methodology is ideal for identifying patents for a convergence technology, such as bioinformatics.
By its nature, it is at the intersection of a number of patent classes.
Accordingly, this paper tentatively employs several methodologies simultaneously to analyze the current status of bioinformatics and to find relevant subclasses, using the Korea Intellectual Property Rights Information Service (KIPRIS, http://www.kipris.or.kr/) patent search system.
Initially, a multiple-word search is undertaken of these patents&#39; titles and abstracts.
The selected lists are then tabulated, based on International Patent Classification (IPC) subclasses together with some known bioinformatics subclasses.
These were then reviewed manually to establish a revised list of bioinformatics patents.
These steps have been repeated several times to establish the list of relevant IPC subclasses that appear most frequently in KIPRIS bioinformatics patents.
Title: KIPRIS is an internet-based patent document search service that is available to the public free of charge.
Korea Institute of Patent Information has been providing KIPRIS since 1996 on behalf of Korean Intellectual Property Office (KIPO).
It covers publications of Korean Intellectual Property Rights (IPR) applications, legal status information, and trial information.
Using the KIPRIS system, the following simple query (documents, containing and the keyword DNA, RNA, and protein, and containing the keyword &#39;algorithm, database, visualization&#39; in the abstract),

(DNA + RNA + protein + genome) * AB = [algorithm + database + visualization], would retrieve 468 registered patents from the KIPRIS database, as in Fig.1.
The number of patents can be further reduced by manual screening.
Likewise, a common methodology is to undertake keyword searches of the patent title, abstract, or claim, as it is not practical to review the complete claims, specifications, and drawings of related patents.
However, this approach of a pure keyword search has the inherent problem of both including irrelevant patents along with bioinformatics patents and of potentially missing relevant bioinformatics patents.
For example, the number of patent documents that have the keyword &#39;bioinformatics&#39; in the KIPRIS database is 446, as of November 18, 2012.
However, the majority of them can not be classified as bioinformatics patents.
On the other hand, there is no warranty that actual bioinformatics patents would at least have the keyword &#39;bioinformatics&#39; in their documents.
Nonetheless, this approach illustrates the converging nature of the technologies comprising bioinformatics and represents a good starting point for examining the cross-classification of bioinformatics patents.
An alternative approach to expand a patent search beyond pure keywords is to include classification information and to use a definition of bioinformatics patents based on several patent subclasses in which bioinformatics patents are expected to be clustered.
A good starting point to choose relevant IPC (http://www.wipo.int/classifications/ipc/) bioinformatics classes would be to reference preselected bioinformatics classes.
For example, the US Patent Classification (USPC) bioinformatics patent subclasses (382/125, 702/19, 702/20, 702/21, 703/11, and 703/12) suggested in [3, 4] could be used, and they have been mapped into IPC groups G06K 9/00, G01N 33/48, G01N 33/50, G01N 31/00, and G06G 7/58, as in Table 1, by using the USPC-to-IPC reverse concordance system (http://www.uspto.gov/patents/resources/classification/index.jsp).
Then, queries using various keywords combined with the groups in Table 1 were initially performed, using the Patent Information Analysis System (PIAS, http://www.kipris.or.kr/kor/add_service/service/service_pias_1.jsp) system, as in Fig.2.
New subclasses or groups linked to bioinformatics can be identified by using various combinations of query terms, as is listed in the left window of Fig.2.
These were then reviewed manually to establish a revised list of bioinformatics patents.
These steps were repeated several times to extend the list of relevant IPC groups that appear most frequently in the KIPRIS database.
The query terms can be dependent on the characteristics of the intellectual property associated with patents, such as algorithms, data contents, network analysis, and user interfaces.
The most frequently occurring groups based on the query term containing the keyword &#39;bioinformatics&#39; in addition to the combination of biological sequence keywords in the abstract was C07K 14/47 (peptides having more than 20 amino acids from mammals), followed by C07K 16/08, C12N 15/11, C12N 15/31, C12Q 1/68, C12N 15/10, G01N 33/68, G06F 17/30, and G06F 19/00.
The most frequently occurring groups containing informatics keywords such as visualization, algorithm, software, and database, together with some biological sequences was C12N 15/00 (mutation or genetic engineering; DNA or RNA concerning genetic engineering), followed by C12N 15/63, C12Q 1/68, G01N 33/68, G06F 17/30, and G06F 19/00.
The most frequently occurring groups related to database contents or networks was G06F 17/30 (digital computing or data processing equipment or methods, specially adapted for specific functions; information retrieval, database structures therefor).
Combining the results of various queries, I finally selected 213 patents that are most related to bioinformatics.
The main groups with the highest frequency were C12Q 001 (measuring or testing processes involving enzymes or micro-organisms) and G06F 017 (digital computing or data processing equipment or methods, specially adapted for specific functions), followed by G01N 033, C12N 015, C07K 014, G06F 019, G06Q 050, H04L 012, G06F 015, and G10L 015.
Fig.3 shows the yearly distribution of these groups.
By identifying the relevant subclasses of bioinformatics patents, the research gaps between science and technology can be identified.
In this paper, I did a preliminary study of the current status of bioinformatics patents registered in the KIPRIS database.
I identified some of the subclasses and groups related to bioinformatics patents, but the work presented here should be regarded as preliminary, and a more detailed analysis using text-mining techniques will follow in the near future.
Query result of (DNA + RNA + protein + genome) * AB = [algorithm + database + visualization].
Combining the results of various queries, using Patent Information Analysis System (PIAS, http://www.kipris.or.kr/kor/add_service/service/service_pias_1.jsp): combining biological keywords (dna?
+ rna?
+ &#34;nucleic acid&#34; + protein?
+ polynucleotid?
+ nucleotid?
+ deoxyribonuc?
+ ribonucleo?
+ &#34;amino acid&#34; + genom?
+ proteom?
+ SNP + &#34;Single Nucleotide Polymorphism&#34; + (metabolic ^2 pathway?)
+ (signal?
^2 pathway?)
+ gene + oligonucleotid?
+ microarray + (DNA ^2 chip?))
with informatics keywords (computer?
+ software?
+ program?
+ algorithm?
+ data?
+ protocol?
+ internet?
+ interfac?
+ proteomics + bioinformatics + ((text?
+ data?)
^2 (Mining?
+ retrieval?))
+ mining?
+ ontology?
+ clustering?
+ statistic?
+ automat?
+ visualiz?)
in various ways.
Yearly distribution of patent groups in Fig.2.
Title: Mapping US Patent Subclasses Described in Rasmussen [4] to IPC subclasses and groups
Title: Editor&#39;s Introduction to This Issue

The theme of this issue&#39;s review articles is epigenomics-one of the hot topics in genome sciences.
Recent publications by the Encyclopedia of DNA Elements (ENCODE) Consortium have sparked interest in this field.
Dr. Jung Kyoon Choi, KAIST, Korea, reviews the results of the ENCODE project on the chromatin environment in regulatory elements.
Dr. Jae-Bum Bae, National Institute of Health, Korea, reviews the status quo of several international collaborative projects on human epigenomics.
We plan to invite more reviews on epigenomics in upcoming issues as well.
One of the original articles by Dr. Kyu Bum Kwack&#39;s group, Cha University, Korea, reports a diplotype analysis on the association between polymorphisms and clearance of hepatitis B virus.
This new type of analysis has the potential to detect novel signals.
In addition, we have four special articles from four groups that report different analysis viewpoints of the same exome dataset of Korean leukemia patients.
In the application notes section of this issue, one may find useful resources, such as a phylogenomic database and an siRNA processing tool.
Dr. Yong-Hwan Lee&#39;s group, Seoul National University, Korea, reports a comprehensive database on the evolutionary relationships among a ubiquitous molecular chaperone family.
Dr. Hyun Seok Kim&#39;s group, University of Texas Southwestern Medical Center, USA, describes a simple and yet extremely convenient and powerful tool for processing cell-based assay data.
As of last fall, Genomics &amp; Informatics has been available in PubMed Central (PMC) and PubMed.
Since then, we are now observing a dramatic increase in access to our articles around the world.
However, it is premature to assess the increase in impact factor.
I hope you enjoy this year&#39;s first issue of Genomics &amp; Informatics.
Your continued support is greatly appreciated.
Title: Association between Interferon-Inducible Protein 6 (IFI6) Polymorphisms and Hepatitis B Virus Clearance

CD8+ T cells are key factors mediating hepatitis B virus (HBV) clearance.
However, these cells are killed through HBV-induced apoptosis during the antigen-presenting period in HBV-induced chronic liver disease (CLD) patients.
Interferon-inducible protein 6 (IFI6) delays type I interferon-induced apoptosis in cells.
We hypothesized that single nucleotide polymorphisms (SNPs) in the IFI6 could affect the chronicity of CLD.
The present study included a discovery stage, in which 195 CLD patients, including chronic hepatitis B (HEP) and cirrhosis patients and 107 spontaneous recovery (SR) controls, were analyzed.
The genotype distributions of rs2808426 (C &gt; T) and rs10902662 (C &gt; T) were significantly different between the SR and HEP groups (odds ratio [OR], 6.60; 95% confidence interval [CI], 1.64 to 26.52, p = 0.008 for both SNPs) and between the SR and CLD groups (OR, 4.38; 95% CI, 1.25 to 15.26; p = 0.021 and OR, 4.12; 95% CI, 1.18 to 14.44; p = 0.027, respectively).
The distribution of diplotypes that contained these SNPs was significantly different between the SR and HEP groups (OR, 6.58; 95% CI, 1.63 to 25.59; p = 0.008 and OR, 0.15; 95% CI, 0.04 to 0.61; p = 0.008, respectively) and between the SR and CLD groups (OR, 4.38; 95% CI, 1.25 to 15.26; p = 0.021 and OR, 4.12; 95% CI, 1.18 to 14.44; p = 0.027, respectively).
We were unable to replicate the association shown by secondary enrolled samples.
A large-scale validation study should be performed to confirm the association between IFI6 and HBV clearance.
Between 350 and 400 million people worldwide are chronically infected with the hepatitis B virus (HBV) [1, 2].
In most HBV-infected patients, spontaneous recovery (SR) by the host immune system is common.
However, 5% to 10% of patients fail to recover and remain as HBV-induced chronic liver disease (CLD) patients [3].
CLD, including HBV-induced chronic hepatitis B (HEP) and HBV-induced cirrhosis (CIR), is a major cause of hepatocellular carcinoma, which can lead to liver-related death [4].
The high mortality of CLD is a major problem in HBV-endemic countries [5].
In Korea, which is an HBV endemic area, more than 70% of CLD patients are infected by HBV [6, 7].
CD8+ T cells are key factors involved in the chronicity of CLD.
The major roles of CD8+ T cells in HBV clearance are the production of interferon (IFN)-γ, which inhibits HBV gene expression and the assembly of HBV RNA-containing capsids, and the induction of apoptosis of virus-infected hepatocytes, which requires physical contact with CD8+ T cells [8-11].
However, the CD8+ T cells of CLD patients undergo activation-induced apoptosis instead of proliferation in the presence of antigen-presenting cells [12, 13].
Apoptosis of antigen-specific CD8+ T cells in CLD patients and lymphocytic choriomeningitis virus (LCMV)-infected type I IFN receptor-null mice is mediated by B-cell lymphoma (Bcl)-2 [12, 14-16], indicating that type I IFN is critical to the survival of antigen-specific CD8+ T cells during the transition from acute to chronic HBV infection.
Kolumam et al.
[16] reported that type I IFN acts directly on CD8+ T cells to allow clonal expansion and memory formation in response to LCMV infection.
Type I IFN receptor-null CD8+ T cells neither produce antiviral molecules, including IFN-γ, granzyme B, and tumor necrosis factor (TNF)-α nor show reduced survival after antigen-induced stimulation [16].
Type I IFN on CD8+ T cells is critical for survival, proliferation, and antiviral functions [16].
IFNs are a well-known family of cytokines with antiviral effects [17, 18].
IFNs modulate cellular proliferation and stimulate immune responses through several IFN-stimulated genes (ISGs) [19].
IFN-α-inducible protein 6 (IFI6) is a type I ISG [20-22] that maps to chromosome 1p35 [23] and is regulated by the Janus tyrosine kinase signal transducer and activator of transcription signaling pathway [24].
IFI6 is a mitochondria-targeted protein; it inhibits the release of cytochrome c from mitochondria and delays the apoptotic process initiated and transduced by the TNF-related apoptosis-inducing ligand/caspase 8 pathway [25].
The role of IFI6 is strongly associated with the immune system, but its antiviral effects are not well known [26].
In the present study, we hypothesized that IFI6 may be a survival-promoting factor for CD8+ T cells and therefore a determinant of the chronicity of HEP.
The frequencies of IFI6 polymorphisms in CLD patients and SR controls were compared using logistic regression.
Title: Subjects for the case-control study

A discovery stage included 305 blood samples obtained from the outpatient clinic of the Gastroenterology Department and from the Center for Health Promotion of Ajou University Hospital (Suwon, Korea) without gender or age restrictions between March 2002 and February 2006.
Samples were derived from genetically unrelated Korean patients.
The experimental protocol was approved by the institutional review board.
Samples were divided into SR control (n = 107), HEP (n = 111), and CIR (n = 87) groups, according to serological markers and biopsy results.
Three samples in the HEP group were not genotype-replicated and were excluded from the analysis.
Finally, 107 SR control, 108 HEP, and 87 CIR patients were analyzed.
In the replication stage, 736 blood samples were collected from Ajou University Hospital and Keiymung University (Daegu, Korea) between February 2006 and September 2012.
Samples were derived from genetically unrelated Korean patients.
The experimental protocol was approved by the institutional review board.
Samples were divided into two 205 SR controls, 437 HEP patients, and 94 CIR patients according to serological markers and biopsy results.
All samples were infected with HBV and classified into one of the three groups, according to their HBV infection status, clinical data, and serological profile, by a pathologist.
Every 6 months for &gt;12 months, the 218 patients were subjected to serological tests for serum levels of hepatitis B core antibody (Anti-HBc II Reagent Kit; Abbott Laboratories, South Pasadena, CA, USA), hepatitis B surface antigen (HBsAg) (Anti-HBs; Abbott Laboratories), and hepatitis B surface antibody (HBsAb) (HBsAg; Abbott Laboratories).
Liver function was evaluated by measuring aspartate aminotransferase (AST), alanine aminotransferase (ALT), albumin, and bilirubin levels using commercially available assays.
All samples showed elevated ALT at least once during the follow-up period and were positive for HBV DNA, irrespective of hepatitis B e antigen (HBeAg) positivity.
Patients in the SR group were HBsAg-negative, HBeAg-negative, anti-HBs-positive, and anti-HBc-positive and had recovered from HBV infection.
Patients in the CLD group, including those in the HEP and CIR groups, were HBsAg-positive for more than 6 months with elevated ALT and AST (≥2 times the normal upper limit).
Samples that were positive for anti-hepatitis C virus (Genedia HCV ELISA 3.0; GreenCross, Yoingin, Korea) or anti-immunodeficiency virus antibodies (HIV Ag/Ab combo; Abbott Laboratories) were excluded.
Title: Sample preparation

All blood samples were stored at -80℃ for the handling of human genomic DNA.
Genomic DNA was purified using G-DEX blood genomic DNA (gDNA) purification kits (Intron Biotechnology Inc., Seongnam, Korea).
The gDNA for the discovery analysis was quantified using the picogreen dsDNA quantification reagent following a standard protocol (Molecular Probes, Eugene, OR, USA).
The plates were read using a VICTOR3 1420 Multilabel counter (excitation 480 nm, emission 520 nm; PerkinElmer Inc., Waltham, MA, USA), and a standard curve for gDNA concentration was generated using known concentrations of lambda DNA.
The quality of the gDNA analyzed in the replication stage was determined using a NanoDrop ND-1000 UV-Vis Spectrophotometer (Thermo, Eugene, OR, USA).
Genomic DNA was diluted to a concentration of 10 ng/µL in 96-well PCR plates.
Title: Single nucleotide polymorphism (SNP) selection and genotyping

In the discovery stage, six SNPs were selected from a public SNP database (http://www.ncbi.nlm.nih.gov/snp/) for the genotyping assay: 1) polymorphic in Chinese and Japanese; 2) tag SNPs in Asian; 3) might have functionality in protein or expression level.
The selected SNPs were 1) one SNP in the 5&#39; flanking region (rs2808426); 2) three intronic SNPs (rs10902662, rs1316896, and rs4908351); 3) one SNP in the untranslated region (rs1141747); and 4) one SNP in the 3&#39; flanking region (rs2808430).
The genotyping was performed using the GoldenGate kit according to a standard protocol (Illumina Inc., San Diego, CA, USA).
Oligos were amplified by allele-specific primer extension.
After hybridization to a sentrix array matrix, signal intensities were read by BeadArray Reader (Illumina Inc.).
Genotyping analysis was performed using GenomeStudio software (version 1.5.16; Illumina Inc.).
In the replication stage, rs2808426, which was identified in the discovery stage, was genotyped using Taqman technology.
The probes were labeled with FAM or VIC dye at the 5&#39; end and a minor groove binder and nonfluorescent quencher at the 3&#39; end.
All reactions were performed following the supplier&#39;s protocol.
SNP genotyping reactions were performed on the ABI PRISM 7900HT real-time PCR system (Applied Biosystems, Foster City, CA, USA).
After the PCR amplification, allelic discrimination was performed on the ABI PRISM 7900HT.
Allele calls were made with SDS v2.4 software (Applied Biosystems).
Title: Statistical analysis

The genetic models for the association test were divided according to additive (AA vs. Aa vs. aa), dominant (AA vs. Aa plus aa), and recessive (AA plus Aa vs. aa) models.
The χ2 test was used to assess the Hardy-Weinberg equilibrium (HWE) in the SR, HEP, CIR, and CLD groups.
The difference between groups was determined by the odds ratio (OR).
ORs were presented with 95% confidence intervals (95% CIs) and adjusted for age and sex.
Each individual haplotype was inferred from the EM algorithm using the SAS haplotype procedure (version 9.1; SAS Institute Inc., Cary, NC, USA).
Linkage disequilibrium (LD) blocks were checked by the Gabriel method using Haploview software (version 4.2; Broad Institute, Cambridge, MA, USA).
All statistical tests were performed using SAS software, and the significance level was set at p &lt; 0.05.
The probability values obtained were corrected for multiple testing by using Bonferroni&#39;s correction and permutation test.
Bonferroni&#39;s p-value for reaching significance was 0.025 (0.05/2).
The Plink program was used to confirm the results and permutation test (n = 100,000; http://pngu.mgh.harvard.edu/~purcell/plink/).
The fate of the patients infected with HBV was determined by several factors, including host immune reactions.
Type I IFNs play a key role in the defense against HBV infection and therefore in the prevention of chronic hepatitis.
IFI6 is induced by type I IFN.
To test the effect of IFI6 polymorphisms on the chronicity of HEP, samples were collected from SR controls (HBsAg-), who recovered from HBV infection without any treatment, and CLD patients, including HEP and CIR groups (HBsAg+), who were at risk of HBV infection.
To analyze first whether variations in the IFI6 gene were associated with the susceptibility to HEP in the Korean population, 107 controls in the SR group, 108 patients in the HEP group, and 95 patients in the CIR group were analyzed for six SNPs of IFI6 (n = 302).
The characteristics of the study subjects are summarized in Table 1.
In the first phase or discovery stage, four out of six SNPs (rs1316896, rs4908351, rs1141747, and rs2808430) were monomorphic.
Genetic variants of rs2808426 and rs10902662 did not show evidence of departure from minor allele frequency and HWE in either of the groups (p &gt; 0.05).
Two genotypes had minor allele frequencies greater than 1% (Table 2).
The results of the genotype analysis showed that the CC genotype was the most common in the rs2808426 and rs10902662 polymorphisms in all groups.
To analyze the genetic association between IFI6 polymorphisms and clearance from CLD, HEP, and CIR, multiple logistic regression analysis with adjustment for gender and age was performed.
In the comparison between SR and CLD patients, rs2808426 was associated with CLD in a recessive model (OR, 4.38; 95% CI, 1.25 to 15.26; p = 0.021).
In addition, rs10902662 showed significant differences between the SR and CLD groups in a recessive model (OR, 4.12; 95% CI, 1.18 to 14.44; p = 0.027).
After the permutation test, the rs2808426 and rs10902662 SNPs still had significant correlations (p &lt; 0.026).
After Bonferroni&#39;s correction, only rs2808426 had significant correlations (p = 0.042) (Table 2).
Comparison between the SR and HEP groups showed that the IFI6 SNPs rs2808426 and rs10902662 in the promoter region were associated with a higher risk that correlated with the homozygous variant TT genotype in a recessive model (OR, 6.60; 95% CI, 1.64 to 26.52; p = 0.008).
After the permutation test, the rs2808426 and rs10902662 SNPs still had significant correlations (p = 0.001 in both genotype analyses), which were maintained after Bonferroni&#39;s correction (p = 0.016 in both genotype analyses) (Table 2).
The results of the multiple logistic regression analysis comparing the SR and CIR groups showed that the rs2808426 and rs10902662 SNPs were not associated in all genetic models (Table 2).
The possible genetic linkage between the rs2808426 and rs10902662 polymorphisms in the protection against chronic HBV infection was examined.
LD blocks were constructed by the Gabriel method using Haploview software.
The complete LD block consisted of rs2808426 and rs10902662 and showed a pairwise |D&#39;| = 1 and r2 = 0.942, which reflect strong LD.
The variants across IFI6 consisted of a single LD block structure composed of two haplotypes (HTs).
The diplotype consisted of HT1 C-C (C allele of rs2808426; C allele of rs10902662) and HT2 T-T (T allele of rs2808426; T allele of rs10902662).
The results of the HT estimation showed that the CC and TT haplotypes accounted for over 99% distribution in all groups.
According to three genetic models, estimated HTs were used for diplotype analysis by logistic regression, adjusting for age and sex.
In the recessive model, HT1 frequency was significantly different between the SR and the CLD (OR, 0.021; 95% CI, 1.25 to 15.26; p = 0.021) and HEP (OR, 6.67; 95% CI, 1.64 to 26.52; p = 0.008) groups.
Analysis of the HT2 diplotype showed a significant difference between the SR and HEP (OR, 0.15; 95% CI, 1.64 to 26.52; p = 0.008) and CLD (OR, 0.24; 95% CI, 1.18 to 14.44; p = 0.027) groups in the dominant model (Table 3).
All diplotype p-values remained significant after the permutation test (p &lt; 0.022), and with the exception of HT2 in the SR and CLD groups, almost all of the diplotype p-values remained significant after Bonferroni&#39;s correction (p &lt; 0.042).
To replicate the significant associations of the SNP rs2808426, 736 samples, consisting of 205 SR, 437 HEP, and 94 CIR patients, were collected.
The clinical information of the patients included in the analysis is summarized in Table 1.
The second-stage genotyping was performed using the Taqman assay.
The association of rs2808426 with CLD was assessed using the three genetic models, and multiple logistic regression with adjustment for gender and age was used as the first-stage analysis.
The results of the genotype analysis of the second set of samples in association with CLD are summarized in Table 4.
The significance of the results of the first genotype analysis was not maintained in the second genotype analysis.
Furthermore, no significant associations were detected in a meta-analysis of the first-stage and second-stage samples (Table 4).
The rs2808426 and rs10902662 SNPs are located in the 5&#39; flanking region and the first intron of the IFI6 gene, respectively.
These SNPs by themselves are known to regulate gene expression by causing alternative splicing or by changing the binding to a transcription factor or microRNA [21].
The presence of the rs2808426 SNP in the promoter region of IFI6 led us to screen for transcription factors with binding sites near or on rs2808426 (C &gt; T).
The binding of several transcription factors, including isoforms of the glucocorticoid receptor α, STAT4, v-ets erythroblastosis virus E26 oncogene homolog 1 (ETS1), and ETS2, to the protective allele (C) was predicted by ALLGEN PROMO (version 3.0.2; http://alggen.lsi.upc.es/cgi-bin/promo_v3/promo/promoinit.cgi?dirDB=TF_8.3) [27].
Interestingly, the binding of ETS1 to the region containing rs2808426 T was not predicted.
Differential binding of ETS1 according to the genotype of rs2808426 may affect the expression of IFI6.
IFI6 expression by type I IFNs triggers the formation of IFN-stimulated gene factor 3 (ISGF3) complexes containing activated STAT1/STAT2 and IFN regulatory factor 9 and their translocation into the nucleus, where they bind to the tandem IFN-stimulated regulatory element (ISRE) in the promoter of IFI6 [21, 28-31].
Tandem binding of ISGF3 to the ISRE is required for maximum expression of IFI6 [32], and the promoter region, including rs2808426, enhances IFI6 expression more than the ISRE region alone [21].
The ISGF3-binding site for the ISRE is separated from the ETS1-binding site by about 1.35 kb.
The transcription factor ETS1 may regulate the expression of intracellular adhesion molecule-1 by protein-protein interaction with STAT1, which is a component of ISGF3 [33].
Overexpression of ETS1 in the MCF-7 breast cancer cell line enhances the expression of IFI6 up to 18.4-fold [34].
These data led us to speculate that the interaction between ETS1 and STAT1 in the ISGF3 complex may increase the expression of IFI6.
The present study investigated the association between the rs2808426 and rs10902662 polymorphisms of the IFI6 gene and the clearance of HBV in the Korean population by multistage comparison between the SR and CLD groups, including the HEP and CIR groups.
In the first stage of the analysis, significant associations between the rs2808426 and rs10902662 polymorphism genotypes and diplotypes were detected.
A risk that was associated with the TT genotype in rs2808426 and rs10902662 was detected in the comparison between the SR and the CLD and HEP groups.
Strong LD was found between the SNPs rs2808426 and rs10902662, containing most of the promoter region.
In addition, diplotype analysis showed that the C-C HT was associated with a higher chance of SR than the T-T/T-T diplotype and that the C-C HT had a protective effect.
The results of the first-stage analysis suggested that rs2808426 and rs10902662 may serve as candidate genetic screening markers for HBV clearance or that causative variants that are responsible for HBV clearance may be present in this LD block.
The association between IFI6 polymorphisms and HBV-induced chronic disease suggest that these polymorphisms might change the expression level of IFI6 according to transcription factor binding.
Therefore, an increase in IFI6 expression that is associated with polymorphisms of the gene could inhibit the release of cytochrome c from mitochondria and block the transmission of the apoptosis signals through Bim in HBV-specific CD8+ T cells.
HBV-specific CD8+ T cells would thus escape from antigen-induced apoptosis, proliferate, and then differentiate into activated CD8+ T cells to eliminate HBV from the host.
The results of the first-stage analysis suggested that IFI6 polymorphisms play a significant role.
In previous studies, CD8+ T cell-related gene polymorphisms, such as those of secreted phosphoprotein 1, interleukin-18, and cyclin D2, were reported to affect the natural course of chronic HBV infections in the Korean population, but the effect of their genetic association is minor (OR, 0.69 to 1.44) [35, 36].
Furthermore, genomewide association studies of human leukocyte antigen (HLA) region polymorphisms, including HLA-DPA1, HLA-DPB1, and HLA-DQ, demonstrated their association with the chronicity of HBV [37-43].
In our first-stage analysis, the protective effect of the rs2808426 and rs10902662 polymorphisms was stronger than that reported previously in studies addressing the association with HBV (OR, 6.60).
The genotype and diplotype distribution in both groups remained significant after multiple testing by Bonferroni&#39;s correction and permutation test.
These results might support that genetic variation in IFI6 affects the clearance of HBV.
A second set of samples was used to replicate the results of the first-stage analysis.
However, in the second association analysis, the comparison of the SR and the HEP and CIR groups did not yield significant results, even when merging the first- and second-stage samples in a meta-analysis.
This could have been due to variation in the sampling cohort, environmental interactions, inadequate statistical power, or gene interactions [1, 44-49].
Furthermore, information on factors important for the progression of liver disease was lacking in the samples analyzed, such as data on alcohol consumption [50].
Although our data could not be reproduced, the results showing an association between IFI6 polymorphisms and HBV chronicity are significant.
Our study is the first study to investigate the association between IFI6 polymorphisms and HBV clearance as an ISG.
In addition, SR patients were used as controls instead of normal healthy subjects to show the effect of genomic background on the chronicity of HBV infection.
Normal controls that never contracted HBV are not suitable to show the genetic effects.
Future studies should include a larger sample size and additional information in the replication study to validate the significance of the results through epistasis and environmental interactions.
In addition, IFI6 promoter variations should be characterized using next-generation sequencing techniques, causal variants should be identified, and mechanisms underlying the effect of IFI6 on HBV clearance that is mediated by HBV antigen-specific CD8+ T cell survival need to be investigated.
In the present study, an initial discovery stage showed that the rs2808426 and rs10902662 genotypes and the corresponding diplotype were associated with a higher probability of HBV clearance in a Korean population.
However, the results could not be replicated in a second stage with a different patient sample.
Further studies should be aimed at showing how IFI6 affects HBV clearance by promoting HBV antigen-specific CD8+ T cell survival.
Moreover, identification of causal variants in the IFI6 by including a large number of samples may help clarify the role of IFI6 on HBV clearance.
Title: Clinical characteristics of study subjects

Title: Genotype frequencies and associations between SR and CLD in IFI6 SNPs

Title: Diplotype frequencies and associations between SR and CLD in IFI6 SNPs

Title: Multistage genotype analysis of rs2808426 (C &gt; T) in IFI6 gene
Title: ENCODE: A Sourcebook of Epigenomes and Chromatin Language

Until recently, since the Human Genome Project, the general view has been that the majority of the human genome is composed of junk DNA and has little or no selective advantage to the organism.
Now we know that this conclusion is an oversimplification.
In April 2003, the National Human Genome Research Institute (NHGRI) launched an international research consortium called Encyclopedia of DNA Elements (ENCODE) to uncover non-coding functional elements in the human genome.
The result of this project has identified a set of new DNA regulatory elements, based on novel relationships among chromatin accessibility, histone modifications, nucleosome positioning, DNA methylation, transcription, and the occupancy of sequence-specific factors.
The project gives us new insights into the organization and regulation of the human genome and epigenome.
Here, we sought to summarize particular aspects of the ENCODE project and highlight the features and data that have recently been released.
At the end of this review, we have summarized a case study we conducted using the ENCODE epigenome data.
Title: The Human Genome Project was started in 1990 and completed by the International Human Genome Sequencing Consortium in April 2003 [1].
Even though the human whole genome sequencing was one of the biggest achievements in human genetics, still more remains to be done.
In September 2003, the National Human Genome Research Institute (NHGRI) launched an international research group called Encyclopedia of DNA Elements (ENCODE) to identify all functional regulatory regions across the entire human genome [2].
For this project, an international group of 442 members from 32 institutions studied 147 different types of cells with 24 types of different experiments (http://www.genome.gov/ENCODE) [3, 4].
The ENCODE project consists of a pilot and a technology development phase [5].
Methods used for the first phase were mainly chromatin immunoprecipitation (ChIP) and quantitative PCR.
A set of regions, corresponding to 30,000 kilobases (~1% of whole human genome), have been targeted [6].
The studies highlighted the success on characterizing functional elements in the human genome and were published in June 2007 in Nature [6] and Genome Research [7].
In the second phase of this project, several new advanced technologies were employed to gain higher accuracy and increase coverage.
The key technology was so-called next-generation sequencing.
Many software tools were developed within the ENCODE project, such as Clustered Aggregation Tool (CAGT) [8].
This tool has been applied to the datasets of chromatin marks and transcription factors to generate a comprehensive archive of histone modifications and nucleosome positioning around transcription factor binding sites.
On September 5, 2012, the results of the ENCODE studies were published in 30 papers in Nature, Genome Research, and Genome Biology to express 5 years of progress toward the goal of the project [9].
The subjects of the ENCODE project to date include:

Title: - Transcription factor motifs

Title: - Chromatin patterns at transcription factor binding sites

Title: - Characterization of intergenic regions and gene definition

Title: - RNA and chromatin modification patterns around promoters

Title: - Epigenetic regulation of RNA processing

Title: - Noncoding RNA characterization

Title: - DNA methylation patterns

Title: - Enhancer discovery and characterization

Title: - Three-dimensional connections across the genome

Title: - Characterization of network topology

Title: - Machine learning approaches to genomics

Title: - Impact of functional information on understanding variation

Title: - Impact of evolutionary selection on functional regions

All ENCODE information and data are freely available for immediate use.
Title: Accessible (open) chromatin

Until recently, it was generally believed that 80% of human genome was noncoding or junk DNA [10] and regulatory regions located in close proximity of genes to be expressed.
The ENCODE project estimated that 2.94% of the whole human genome was protein coding, while 80.4% of sequences governed how those genes are regulated [11].
Open chromatin regions (nucleosome-depleted regions that are sensitive to cleavage by the DNase I enzyme) have been widely used to identify active DNA regulatory elements, including promoters, enhancers, silencers, insulators, and locus control regions [12, 13].
Open chromatin has been mapped by two approaches.
A total of 2.89 million DNase hypersensitive sites (DHSs) mapped by DNase I digestion, followed by next-generation sequencing in 125 cell types and 4.8 million sites across 25 cell types that exhibited reduced nucleosomal crosslinking, were mapped based on formaldehyde-assisted isolation of regulatory elements assay [11].
On average, approximately 200,000 DHSs were found to be active in any one cell [14].
Open chromatin regions have different proportions, based on their genomic location, and they cover approximately 2% of the genome [15].
Open chromatin regions are enriched for regulatory information and located at or near transcription start sites [6].
Title: DNA methylation and histone modification

The methylation status for the ENCODE cell lines has been obtained by three techniques: 1) sodium bisulfite conversion, 2) reduced representation bisulfite sequencing, and 3) methylation-sensitive restriction enzyme (methyl-seq).
They profiled the DNA methylation status of over than one million CpG in all ENCODE cell lines and have provided quantitative determination of the proportion of CpG methylation at each site [5, 11, 16-18].
The result has added a lot to our present knowledge of cell-type selective patterns of methylation associated with genomic occupancy of transcriptions factors [17].
The ENCODE team has identified the specific patterns of post-translational histone modifications (e.g., H3K36me3, H3K27ac, and H3K27me3) by ChIP sequencing technique [6].
Then, histone modifications have been connected to the regulatory regions, such as promoters, enhancers, transcribe domains, and silenced regions [6, 11].
The maps of various histone modifications have been constructed genomewide in different cell lines.
Title: Nucleosome positioning

In contrast to histone modifications, which are identified by ChIP-seq, nucleosome positioning data were generated without immunoprecipitation by using micrococcal nuclease digestion followed by sequencing [19-21].
The method that distinguishes nucleosome positioning is based on the ability of nucleosomes to protect related DNA from digestion by enzyme [22].
Protected fragments are sequenced to produce genomewide maps of nucleosome localization.
By using a new tool, the ENCODE project has identified the differences in patterns of nucleosome positioning around the majority of transcription factor binding sites [8].
These data have tremendous value for analyzing the relationship between transcription factor binding, histone modifications, and gene expression regulation.
Title: Transcription factors

Transcription factors can bind to DNA and alter the expression of genes [23].
The ENCODE Consortium has detected 45 million different events where transcription factors occupy DNA across 41 diverse cell types [16].
For this, high-throughput sequencing (ChIP-seq) technology was applied to create occupancy maps for binding sites for a variety of DNA binding factors [5].
The architecture of the network of human transcription factors was found to be quite complex [24].
Title: Annotation of the outputs of genomewide association studies

A large number of polymorphisms associated with increased risks of certain diseases or particular quantitative traits have been discovered by genomewide association studies.
The identified genetic markers or risk variants, mostly composed of single nucleotide polymorphisms, were enriched within the genomic regions that were previously thought to be &#39;junk DNA&#39; [11, 16, 17, 24-28].
The results of the ENCODE project have demonstrated that variants that fall in regulatory regions could indirectly influence coding genes linked to diseases or clinical traits.
With the DHS map in hand, a research group studied more than 5,500 genome-wide association study (GWAS) single nucleotide polymorphisms (SNPs).
The result of their research revealed that approximately 75% of GWAS diseases- and trait-associated single nucleotide polymorphisms (SNPs).
are concentrated in or near DNase I hypersensitive sites [29].
Together, they give new insights to the researchers who are carrying out GWASs.
Title: A case epigenomic study based on the ENCODE data

All ENCODE data are freely available for download and analysis.
The ENCODE website at the University of California Santa Cruz (UCSC) genome browser (http://encodeproject.org or http://genome.ucsc.edu/ENCODE) provides information about how to access the data.
More than 100 researchers who were not part of the ENCODE project have used the ENCODE data for their studies.
We had a question about the conflicts between the openness of transcription factor binding regions and the existence of histones for various modifications.
Whereas regulatory DNA elements underlying open chromatin are made accessible by nucleosome-depleted states, the presence of nucleosomes is required for specific histone modifications that mediate the specific regulation of open chromatin regions [30].
To handle this question, we downloaded the data for chromatin accessibility (DHS), in vivo nucleosome occupancy, 15 chromatin states, histone modifications, and transcription factor binding sites for the GM12878 lymphoblastoid cell line from the ENCODE tracks of the UCSC genome browser.
We discovered the presence of boundary nucleosomes just inside of open chromatin (black curve in Fig.1A).
In vivo nucleosomes that were reconstituted purely based on naked DNA [21, 31] also peaked right inside of open chromatin (gray shade in Fig.1A).
The corresponding DNA sequences displayed an increase in the C/G dinucleotide frequency (red dots in Fig.1B) and a significant decrease in the A/T dinucleotide frequency (blue dots in Fig.1B), exhibiting nucleosome-favoring features.
However, there was a difference in the peak position between the in vivo and in vitro nucleosomes, indicating cellular reprogramming of sequence-encoded nucleosome positioning.
The similar patterns were found in yeast (data not shown here).
This evolutionarily conserved feature provides a platform for histone modifications associated with promoters and enhancers.
We examined histone modification levels for the regulatory areas whose borders were occupied by DNA-encoded nucleosomes and those that were free of nucleosome sequences.
Among 15 different chromatin states [30], active promoters, poised promoters, and strong enhancers tended to contain nucleosome-encoding sequences.
Highest enrichment was found for poised promoters, and the lowest percentage was found with heterochromatin states.
All these data indicate intrinsic nucleosome positioning present in the regulatory regions that are frequently accessed or poised for later activation but are not supposed to be silent.
As with other ENCODE data users, the main goal of our study was to add a few more new directions to the ENCODE epigenetics road map.
The ENCODE project results have dramatically increased our knowledge about chromatin and its regulation.
The results have forced us to think better about genetics.
The ENCODE consortium has developed multiple technologies and approaches to discover the functional elements encoded in the human genome.
This project generated more than 15 trillion bytes of raw data, mapping diverse chromatin properties in several cell types.
Data generated by the ENCODE project are freely available and have been used by many researchers.
By using the ENCODE data, we discovered the presence of boundary nucleosomes for specific histone modifications in open chromatin.
Although the ENCODE project has provided a unique and informative window through which to view evolutionary change, it has looked only at 147 types of cells, and the human body has 1,000 cell types.
The project is just the start of a long journey to understanding the mysteries of our genome.
Sequenced-directed positioning of boundary nucleosome in open chromatin.
(A) In vivo (black curve) and in vitro (gray shade) nucleosome occupancy across open chromatin in human GM12878 lymphoblastoid cell line.
(B) Normalized frequencies of C/G and A/T dinucleotides across the boundaries of open chromatin.
Title: Bioinformatics Interpretation of Exome Sequencing: Blood Cancer

We had analyzed 10 exome sequencing data and single nucleotide polymorphism chips for blood cancer provided by the PGM21 (The National Project for Personalized Genomic Medicine) Award program.
We had removed sample G06 because the pair is not correct and G10 because of possible contamination.
In-house software somatic copy-number and heterozygosity alteration estimation (SCHALE) was used to detect one loss of heterozygosity region in G05.
We had discovered 27 functionally important mutations.
Network and pathway analyses gave us clues that NPM1, GATA2, and CEBPA were major driver genes.
By comparing with previous somatic mutation profiles, we had concluded that the provided data originated from acute myeloid leukemia.
Protein structure modeling showed that somatic mutations in IDH2, RASGEF1B, and MSH4 can affect protein structures.
Thanks to the next-generation sequencing (NGS) technology, genomics research area has been developed dramatically in recent years.
Since 2008, many efforts have been made to find somatic mutations in cancer genomes; hundreds of samples by exome sequencing have been used to develop diagnosis and prognosis markers in various cancers in recent researches.
Cheaper price has enabled us to get more exome sequencing data from hundreds of samples, even thousands of genomes.
By applying exome sequencing to cancer genomes, we can discover somatic mutations and somatic copy-number alterations (SCNAs) on a massive and genome-wide scale.
Traditionally, array-based comparative genomic hybridization has been used to discover SCNA, but exome sequencing has many advantages, in that one can derive various results, such as somatic mutations, loss of heterozygosity (LOH), and SCNA.
It is still under debate because bioinformatics methods could give different results depending on their parameters and software, but major driver genes of somatic mutations show a similar consensus.
We had used an in-house bioinformatics pipeline and databases for this research, especially &#34;somatic copy-number and heterozygosity alteration estimation&#34; (SCHALE) to find LOH and SCNA.
Furthermore, driver genes were also inferred from network and pathway analyses.
Protein structure prediction was used to predict putative structural changes by somatic mutations and also possibly their functional changes.
Title: Dataset

Whole-exome sequencing data of tumor/normal paired samples from 10 individuals with blood cancer were obtained in FASTQ format from PGM21 (The National Project for Personalized Genomic Medicine).
For 10 of the samples, the genotype information from single nucleotide poymorphism (SNP) chip was additionally obtained (but not the raw data).
Title: Data processing and quality control (QC)

The QC of sequencing reads was performed with NGS QC Toolkit, and the high-quality reads were aligned to the human reference genome (hg19, http://genome.ucsc.edu) using BWA (http://bio-bwa.sourceforge.net/) [1] with default parameteres.
Picard (http://picard.sourceforge.net) was used to remove PCR duplicates.
Base quality recalibration and local realignment around indels were performed using Genome Analysis Toolkit (GATK, http://www.broadinstitute.org/gatk/).
Because our previous study showed that the discordance of indel alignments had potential to generate false-positive somatic mutations, we performed the local realignment after the merging of alignments from same patients.
Template lengths were calculated using the processed alignments, and the distributions for each samples were plotted to examine the qualities of sequencing libraries.
Variant genotyping for each sample was performed with UnifiedGenotyper of GATK, and the variants were filtered as follows: mapping quality zero (MQ0) ≥ 4 &amp;&amp; MQ0/depth of coverage (DP) &gt; 0.1, variation quality (QUAL) &lt; 200, quality by depth (QD) &lt; 5, Fisher strand (FS) &gt; 60, MQ &lt; 40, DP &lt; 10 (5 if no variant allele), genotype quality (GQ) &lt; 13.
The genomic annotation of variants was performed with custom-made scripts and refSeq information.
Title: Identification of somatic mutations

We had developed a bioinformatics pipeline for this award program (Fig.1).
VarScan [2] was used to identify somatic mutations.
The output of VarScan was filtered as follows: somatic p-value ≤ 0.05, normal allele depth ≤ 1, tumor allele depth ≥ 5, tumor total depth ≥ 10.
The mutation set was annotated using a custom-made script with refSeq, SIFT (http://sift.jcvi.org/), MutationAssessor, phyloP46way (vertebrate), dbSNP 137 [3], in-house Korean variation database, etc.
We selected the driver mutations that were not in dbSNP 137, excluding clinically associated variants (Flagged track in University of California Santa Cruz [UCSC] Genome Browser) and an in-house Korean variation database, and were deleterious missense, nonsense, read-through, splicesite, or coding indel mutations (NS/SS/I).
The deleterious missense mutations were determined if at least 2 out of the following 3 were true: PhyloP score &gt; 1.5, SIFT prediction = deleterious, MutationAssessor Func.
Impact = high or medium.
The total numbers of candidates for each filtering step are summarized in Fig.1.
We found somatic mutations in samples G10, G08, and G09, but they were not NS/SS/I.
Title: Comparison with genotypes from SNP chip

To compare with genotypes from SNP chip, the genotyping was performed using UnifiedGenotyper with the intervals, including all the coordinates of SNP chip and the option-output_mode EMIT_ALL_SITES.
For each sample with genotype information from SNP chip, overall genotype concordance, non-reference sensitivity, and non-reference discrepancy rate were calculated according to the guide of GATK VariantEval using the genotypes with DP ≥ 5.
Title: Genotype comparison of samples

The multi-sample SNP genotyping was performed with UnifiedGenotyper to compare the genotypes on the variant positions where all samples were able to be genotyped.
The filtration scheme was the same as above.
The distance between two samples was calculated as the sum of genotype difference values (0 if two genotypes are same, 1 if a genotype is 0/1 and another is 0/0 or 1/1, 2 if a genotype is 0/0 and another is 1/1).
The distances were divided by the number of the variant positions.
Heatmap and similarity matrix were used to visualize the distances.
Title: Genome annotation

We used in-house genome annotation pipelines and various databaes, as summarized below.
Title: Genomic annotation: refGene

Title: Human Genome Variation Society (HGVS) nomenclature: nucleotide (non-coding, coding), protein

Title: SIFT - tolerant amino acid substitutions

Title: MutationAssessor

Title: phyloP46way_vertebrate

Title: GERP; GERP_element: rsScore

Title: RepeatMasker

Title: Tandem repeat definition

Title: dbSNP 137; Common, Flagged, Mult

Title: Korean variation database (54 Personal Genomes)

Title: OMIM morbid genes

Title: Oncogenes and tumor suppressors from Ariadne Pathway Studio

Title: Detection of LOH and SCNA by SCHALE

Title: Detection LOH and SCNA was performed with custom-made software SCHALE (Fig.2).
For the positions where the genotypes were heterozygous in normal samples, the values of LOH and SCNA were calculated.
The values of LOH were calculated by the equation:

Title: LOH = | RT - 0.5 | - | RN - 0.5 |, where RN is the count of reads supporting a non-reference allele divided by the total depth in the normal sample and RT is in tumor sample.
The values of SCNA were calculated by the equation:

SCNA = log2(tumor depth/normal depth), where the depths were normalized using R library tweeDEseq.
The values of LOH and SCNA were segmented together using R library DNAcopy with the parameters alpha = 0.01 and undo.
SD = 2.
The segments and values were plotted for each chromosome with the idiograms in R library SNPchip.
Title: Network analysis

Ingenuity Pathway Analysis (IPA) was used to perform the network study of the identified somatically mutated genes and the genes in the LOH region.
Title: Protein structure modeling

Before the structural modeling, the protein features related to the somatic mutations were found by BLAST on UniProt (10 most similar proteins, expectation value = 1e-30) and residue mapping.
Homology modeling was performed using MODELLER (http://salilab.org/modeller) with the templates 1LWD for IDH2, 3KSY for RASGEF1B, and 2O8B for MSH4.
Molecular dynamics simulation was performed with GROMACS (http://pubs.acs.org/doi/abs/10.1021/ct700301q).
The structural images were drawn using PyMOL (http://www.pymol.org/).
Title: QC and results

We had used several methods to check the qualities of sequence reads.
First, we used NGS QC Tools [4] to confirm sequence qualities (Supplementary Table 1).
Overall, they had a very good quality, according to NGS QC Tools results.
Template length (library size) was also evenly distributed for all samples (Supplementary Fig.1).
Title: Reference genome mapping

About 98% of reads were mapped onto the human reference genome (UCSC Genome Browser, hg19), summarized in Supplementary Table 2.
After a duplicate removal step, 77.85% reads were retained; each sample had up to 25% duplicates.
Eighty-nine percent of target areas covered at least 10 sequence reads, which is sufficient for discovering somatic mutations.
We did not get target enrichment kit information from PGM21, so we had compared coverage information from several kits and concluded that SureSelect All Exon 50Mb was used for exome sequencing.
Title: Genotyping by GATK and comparison with SNP chip

In order to check the genotyping parameters we had used, 10 SNP chip results were compared.
Because SNP chip data were genotypes, we could not infer copy number changes; we could only calculate concordance between exome sequencing and SNP chip.
We had used a separate bioinformatics pipeline for checking concordance, and GATK [5] was used mainly for genotyping of germline samples.
We had discovered 18,500-19,200 coding SNPs.
The number of SNPs from the 5&#39; untranslated region (UTR) was close to that of the 3&#39; UTR, so the target enrichment kit was not the latest 3&#39; UTR extension version (Supplementary Table 3).
In general, the 3&#39; UTR is much larger than the 5&#39; UTR; a new 3&#39; UTR extension version of the exome kit will give at least a few times more SNPs than 5&#39; UTR.
Two hundred fifty coding SNPs were retained after removal of dbSNP137 [6] and an in-house Korean database comprised of 54 normal individuals.
Among them, we can focus on about 160 variants of NS/SS/I, which is believed to be most functionally important (Supplementary Table 4).
Overall concordance was at least 99.6%.
In recent personal genome papers, they reported more than 99.95% concordance between whole-genome and SNP chips.
Usually, one can achieve higher concordance if he uses strict parameters for their genotyping.
Our purpose is to discover somatic mutations, not genotyping of normal germline, so we did not put much effort to get higher concordance values.
Especially, G10.N SNP chip data were totally different from the exome, because only 77% of SNPs were shared (Supplementary Table 5).
As one can see in the table, 77% means they are different individuals (false positive for Award program).
Distance and similarity matrix were calculated to see the differences and similarity (Supplementary Table 6, Supplementary Fig.2).
We had concluded that sample G06 has a different sample pair from different individuals.
G10 also has only 92% similarity and showed a little similarity between G03 and G04.
For such reasons, G10 was possibly contaminated and therefore rejected from our bioinformatics analysis.
In conclusion, we had used only 8 sample pairs for discovery of somatic mutations, LOH, and SCNA.
Title: A list of somatic mutations

After removal of SNPs and indels from dbSNP137 [6] and an in-house Korean database, 27 somatic mutations (NS/SS/I) were discovered from 8 samples.
Among them, NPM1 showed a TCTG internal tandem insertion.
NPM1-ITD is known to be major driver gene in acute myeloid leukemia (AML) (Table 1, Supplementary Table 7).
NPM1-ITD and GATA2 were recurrently found in two samples.
We had removed all variants from dbSNP and an in-house Korean database that showed polymorphisms or normal variants, because they can be putatively functionally neutral, even if they are discovered as somatic mutations.
We had downloaded the COSMIC (http://www.sanger.ac.uk/genetics/CGP/cosmic/) database and compared with our somatic mutations.
We found the same highly recurrent somatic mutation variants in four genes: NPM1, IDH2, CEBPA, and GATA2.
In 26 genes, we can easily see the NPM1, IDH2, GATA2, CEBPA genes-known to be highly mutated in AML [3].
Based on the recent somatic mutation profiles from exome sequencing, one can see the following trends of them.
In solid tumors, somatic mutation in TP53 is high.
But in melanoma, BRAF mutation is high.
In hepatocellular carcinoma, CTNNB1 mutation is high.
It means that somatic mutations can play a different role, depending on cancer.
In AML, other than the well-known TP53 mutation, genes, such as FLT3, TET2, and NPM1, show somatic mutations and are the driver genes [7].
Title: Discovery of LOH and SCAN by SCHALE

We had used SCHALE to discover LOH and SCNA.
It appears that sample G06 has many LOHs and SCNAs, but an incorrect sample pair can give those results (Supplementary Fig.3).
We also discovered high LOH on chrX in G10 (Fig.3).
We could not find any somatic mutations in G10.
Previously, we had described that it could be contamination, but another possibility is that they are exomes from identical twins with different genders (we do not have any gender information from any samples).
Abnormal regions on chr1 in G05 showed LOH in the germline, but it is quite unnatural (Fig.4).
We are not describing much for this area until experimental validation is performed.
We had found one LOH region on chr13 in G05 (Fig.5).
INTS6 and KIAA1704 are located at both ends and potentially could make a fusion gene (Fig.6).
Experimental validation is also required for this assumption.
INTS6 is expressed in blood; many expressed sequence tags (ESTs) were found in INTS6 in UniGene (http://www.ncbi.nlm.nih.gov/UniGene).
But, KIAA1704 has a small number of ESTs, so we could not infer expression levels from UniGene.
There are 61 genes on chr13:45,578,409-52,035,344, and we had an additional set of network and pathway analyses, including 6 genes (MAST2, L1TD1, MSH4, GATA2, VDAC2, and PHF6) with somatic mutations.
The top network is &#34;Cancer, Hematological Disease, Immunological Disease,&#34; comprised of GATA2 and MAST2.
Many genes in LOH regions are included in this network.
We can infer that those genes with somatic mutations and LOH can interact with each other (Fig.7).
Title: Network and pathway analyses

Using 26 genes, we had used IPA (http://www.ingenuity.com/) [8] for network and pathway analyses.
It can be summarized in two major networks (Table 2).
The top network is &#34;Cancer, Hematologicial Disease, Cell Cycle,&#34; and it has NPM1, GATA2, and CEBPA genes (Figs.8 and 9).
These three genes showed a highly connected network and well-known major driver genes in leukemia.
The second network is related to cell death and survival (Fig.10).
IDH2 and PHF6 genes were located in this network.
But overall, somatic mutation frequency in the top network is much higher than the second network above.
IPA analysis reports are summarized in Supplementary Table 8.
Title: Protein structure modeling

We have examined protein features related to residues substituted by the identified 27 somatic mutations and found the features of 19 mutations (Table 3).
The mutation of CEBPA generates a duplication of an amino acid that affects the original helical secondary structure.
The residue corresponding to the mutation of IDH2 has a function to bind with its substrate, so the substitutions of the residue decrease the binding affinity with the substrate.
The protein region frame-shifted by the mutation of NPM1 is related to the function of nuclear localization, important for maintaining the structure.
The structural modeling of RASGEF1B shows that the mutated residue Arg261 generates a new hydrogen bond with Asn303, affecting the stretch of neighbor helical structures.
The protein structure of MSH4 shows that the salt bridge between Glu415 and Arg474 is essential for the folding and stability of the protein structure but is broken by the mutation (Table 4, Fig.11).
We have analyzed 10 exome sequencing samples and SNP chip data using a variety of bioinformatics methods.
According to the concordance results, comparing with SNP chip data, we concluded that the G10.N exome sample is not the correct pair.
Comparing SNPs between normal and tumor pairs revealed that the normal-tumor pair of sample G06 was not matched, and the G10 exome showed abnormal LOH in chrX but had no somatic mutation.
So, we had removed G06 and G10 for discovering somatic mutations.
After removing G06 and G10, with 8 paired-end samples, we discovered 27 functionally important somatic mutations with significance.
They included well-known genes in AML, such as NPM1, IDH2, GATA2, and CEBPA.
Especially, NPM1 and GATA2 were found in 2 out of 8 samples.
In sample G05, we found one abnormal region at chr1 and an LOH region at chr13.
We could conclude that the LOH region could potentially make a fusion gene by INTS6 and KIAA1704, and we did further network analysis on that.
The results of IPA analysis gave &#34;Cell Death and Survival, Cell Cycle&#34; network as the top network, and it includes GATA2 and MAST2 (somatic mutation).
We had done IPA analysis with 26 genes, giving the &#34;Cancer, Hematological Disease, Cell Cycle Network&#34; as the top network, and it includes NPM1, GATA2, and CEBPA, which are well-known driver genes in AML [3].
Since those 3 genes have many edges in the result, we could conclude that those genes are indirect evidence of driver genes.
Title: The second network is related with &#34;Cell Death and Survival,&#34; and it includes IDH2 and PHF6 genes, the same as G05&#39;s top network.
So far, our conclusion is that 8 samples are related with the &#34;Cancer, Hematological Disease, Cell Cycle Network&#34; but sample G05 is related with the &#34;Cell Death and Survival Network.&#34;
We had searched previous works of genome-wide somatic mutation discovery and concluded that 8 exome samples might be AML.
Somatic mutation profiles of blood cancer are summarized in Table 5 [3, 9, 10].
But, we could not find any somatic mutations in the FLT3 or RAS gene.
Chronic lymphocytic leukemia and Blast Crisis chronic myeloid leukemia did not have somatic mutations in NPM1 or CEBPA.
Seven out of 8 samples showed neither LOH nor SCNA, so we could conclude that those patients had a normal karyotype with no translocation or inversion.
If possible, additional RNA-Seq experiments or another technique can provide enough evidence of karyotypes of those samples.
In CEBPA, IDH2, and NPM1, we saw somatic mutations located in helix regions.
By using protein structure modeling, we found that IDH2, RASGEF1B, and MSH4 could have structural variations, affecting protein functions.
Somatic mutations in IDH2 could break hydrogen bonds.
Bioinformatics pipeline for exome data and numbers of somatic mutation candidates for each filtering step.
SNP, single nucleotide polymorphism; LOH, loss of heterozygosity; SCNA, somatic copy-number alterations; SCHALE, somatic copy-number and heterozygosity alteration estimation; NS/SS/I, nonsense, splice-site, or coding indel mutations.
Somatic copy-number and heterozygosity alteration estimation (SCHALE), developed for detection of loss of heterozygosity (LOH) and somatic copy-number alterations (SCNA).
TMM, trimmed mean of M values; CBS, circular binary segmentation.
Changes in loss of heterozygosity and somatic copy-number alteration in sample G10.
Abnormal region on chr1 in G05.
LOH, loss of heterozygosity; SNP, single nucleotide poymorphism; CNA, copy-number alteration.
Loss of heterozygosity on chr13 in G05.
Genes in a loss of heterozygosity region.
Image adopted from University of California Santa Cruz (UCSC) Genome Browser, chr5:45,578,409-52,035,344.
Ingenity Pathway Analysis (IPA) from genes with loss of heterozygosity and somatic mutation (sample G05).
Data were analyzed through the use of IPA (Ingenuity® Systems, http://www.ingenuity.com) [8].
&#34;Cancer, Hematological Disease, Cell Cycle&#34; Network.
Data were analyzed through the use of Ingenity Pathway Analysis (IPA) (Ingenuity® Systems, http://www.ingenuity.com) [8].
Major interaction in top network.
Data were analyzed through the use of Ingenity Pathway Analysis (IPA) (Ingenuity® Systems, http://www.ingenuity.com) [8].
&#34;Cell Death and Survival, Cell Cycle, Cell-to-Cell Signaling and Interaction, Cellular Movement&#34; Network.
Data were analyzed through the use of Ingenity Pathway Analysis (IPA) (Ingenuity® Systems, http://www.ingenuity.com) [8].
Location of somatic mutation in protein structure.
(A) IDH2.
(B) RASGEF1B.
(C) MSH4.
Title: A list of functionally important somatic mutations in 8 samples

Title: Two top networks inferred from Ingenuity Pathway Analysis

Title: Protein residues in which somatic mutation is located

Title: Protein structure modeling

Title: Somatic mutation profile of three leukemia studies
Title: Effects of Somatic Mutations Are Associated with SNP in the Progression of Individual Acute Myeloid Leukemia Patient: The Two-Hit Theory Explains Inherited Predisposition to Pathogenesis

This study evaluated the effects of somatic mutations and single nucleotide polymorphisms (SNPs) on disease progression and tried to verify the two-hit theory in cancer pathogenesis.
To address this issue, SNP analysis was performed using the UCSC hg19 program in 10 acute myeloid leukemia patients (samples, G1 to G10), and somatic mutations were identified in the same tumor sample using SomaticSniper and VarScan2.
SNPs in KRAS were detected in 4 out of 10 different individuals, and those of DNMT3A were detected in 5 of the same patient cohort.
In 2 patients, both KRAS and DNMT3A were detected simultaneously.
A somatic mutation in IDH2 was detected in these 2 patients.
One of the patients had an additional mutation in FLT3, while the other patient had an NPM1 mutation.
The patient with an FLT3 mutation relapsed shortly after attaining remission, while the other patient with the NPM1 mutation did not suffer a relapse.
Our results indicate that SNPs with additional somatic mutations affect the prognosis of AML.
Acute myeloid leukemia (AML) is the most common type of leukemia among hematologic malignancies.
This disease is a clonal hematopoietic disease that is caused by either inherited or acquired genetic mutations.
AML is known to be a molecularly and clinically heterogeneous disease [1].
In order to cure this disease, cytosine-arabinoside (Ara-C)-based chemotherapy is widely used.
Two thirds of AML patients who receive Ara-C based chemotherapy regimen, will achieve complete remission (CR).
However, more than 50% of patients suffer a relapse after attaining CR [2].
Eventually, a great majority of AML patients die from progressive disease after relapse by various genetic changes that will endow tumor cells with resistance against most commonly used anticancer agents.
Therefore, it is prerequisite to identify prognostic markers that will help in deciding the suitable treatment modalities.
A lot of studies have focused on finding somatic mutations in cancer cells by whole-exome sequencing (WES).
However, these studies did not fully explain the mechanisms of how disease relapses.
In the initial development of cancer in humans, Knudson&#39;s two-hit hypothesis has provided important insights into the pathogenesis of tumors in autosomal dominant tumor predisposition syndromes [3].
With an analogy, we focused on the additional genetic changes that will predispose susceptible individuals with a pre-existing genetic abnormality to develop leukemia.
To address this issue, germline single nucleotide polymorphism (SNP) analysis was performed using the UCSC hg19 program in 10 AML patients (samples, G1 to G10), and somatic mutations were identified in the same tumor sample using SomaticSniper and VarScan2.
Our results suggest that SNPs with additional somatic mutations may directly affect the prognosis of AML and emphasize the need to validate in a larger patient cohort.
Title: DNA sample preparation (normal and tumor samples)

Genomic DNA samples were prepared for WES.
They were freshly obtained from AML patients&#39; bone marrow at initial diagnosis.
Subsequent bone marrow samples were aspirated if necessary according the disease status of the patients.
AML blast tumor cells were isolated from bone marrow of each patient using Ficoll gradient separation.
Matching patients&#39; saliva samples were collected as germline controls.
Each tumor and control DNA was extracted via standard protocols of DNA purification.
DNA samples for validation were obtained from our frozen stock as needed.
Title: Whole-exome sequencing

We prepared and sequenced exomes using the Solexa sequencing technology platform (HiSeq2000; Illumina, San Diego, CA, USA) following the manufacturer&#39;s instructions.
We randomly sheared 3 µg of genomic DNA using the Covaris System to generate 150-bp inserts.
The fragmented DNA was end-repaired using T4 DNA polymerase and Klenow polymerase, and Illumina paired-end adaptor oligonucleotides were ligated to the sticky ends.
We analyzed the ligation mixture by electrophoresis on an agarose gel and sliced and purified fragments of 200.250 bp.
The purified DNA library was hybridized with the SureSelect Human All Exons probes set (Agilent, Santa Clara, CA, USA) to capture 50 Mb of targeted exons following the manufacturer&#39;s instruction.
We prepared the HiSeq2000 paired-end flowcell with the manufacturer&#39;s protocol using the captured exome library.
Clusters of PCR colonies were then sequenced on the HiSeq2000 platform using recommended protocols from the manufacturer.
Title: Read mapping and alignment and variant analysis

We recently developed an in-house analysis pipeline for cancer genome sequencing data.
This pipeline essentially includes 1) mapping and alignment, 2) discovery of sequence nucleotide variations (SNVs) and insertions and deletions (indels), and 3) filtering and annotation of SNVs and indels.
Briefly, once the raw sequence data were created, the output short reads were aligned to a reference genome (NCBI human genome assembly build 36) using UCSC hg19.
Each alignment was assigned a mapping quality score by UCSC hg19, which is the Phred-scaled probability that the alignment is incorrect.
The PCR duplicates were detected and removed by SAMTOOLS (http://samtools.sourceforge.net/).
After alignment, we used GATK v1.4 to call SNVs for each chromosomal position.
Filtering and annotation of SNVs and indels were performed using ANNOVAR (http://www.openbioinformatics.org/annovar/).
We defined somatic mutation SNVs as those detected by both SomaticSniper and VarScan2 with a somatic p-value &lt;0.05.
We also set a minimum coverage of 5 in both normal and tumor samples.
Conclusively, final somatic mutations were selected using the in-house filter.
Germline SNP analysis was done to detect germline heterozygous mutations that are carried by AML patients.
To address this issue, germline SNP analysis was performed using the UCSC hg19 program in 10 AML patients (samples G1 to G10) (Table 1).
We chose 18 major genes (TET2, ASXL1, DNMT3A, CEBPA, PHF6, WT1, TP53, EZH2, RUNX1, PTEN, FLT3, NPM1, HRAS, KRAS, NRAS, KIT, IDH1, and IDH2) to be analyzed in 10 samples (Table 2).
Among these 18 genes, KRAS and DNMT3A seemed to play a critical role in the progression of AML, associated with the chemotherapy response.
KRAS is a protein product of the normal KRAS gene, which performs an essential function in normal tissue signaling, and the mutation of a KRAS gene is an essential step in the development of many cancers [4].
DNMT3A is a key component of the epigenetic regulation of genes, as it catalyzes the addition of methyl groups to the cytosine residue of CpG dinucleotides [5].
However, variations only in KRAS and DNMT3A are not enough to explain the mechanism of AML progression, since additional somatic changes exist in AML patients.
SNPs of KRAS were detected in 4 different individuals (G2, G6, G7, and G8), as shown in Table 1, while those of DNMT3A were detected in 5 individuals (G3, G4, G6, G8, and G10) among 10 AML patients (Table 1).
Patients G6 and G8 showed SNPs in both KRAS and DNMT3A.
We then searched for somatic mutations of each individual in order to prove the &#39;two-hit&#39; theory.
Table 3 shows the results of the somatic mutation analysis.
SNPs of KRAS and DNMT3A were detected in both samples G6 and G8 (Table 3).
However, they had different clinical courses after the first induction.
They both attained a CR after the first induction; however, G6 relapsed within 390 days after attaining remission, while G8 did not relapse for 402 days.
We suggested that an additional somatic mutation may have affected the clinical courses above.
A somatic mutation in IDH2 was identified in both G6 and G8, while that of FLT3 and NPM1 was detected in G6 and G8, respectively.
FLT3 mutation is known as a bad prognostic factor in AML [6].
NPM1 mutation, on the other hand, is usually known as a good prognostic factor in AML [7].
Therefore, we could speculate that SNPs (KRAS and DNMT3A) with additional somatic mutations (FLT3 and NPM1) affect the prognosis of AML differentially.
In addition, G2 and G7 shared the same SNPs in KRAS but their clinical courses showed a striking difference (Table 4).
G7 had an additional germline SNP in KIT.
This patient never attained CR despite intensive chemotherapy.
As we searched further for somatic mutations in G7, a somatic mutation in TET2 was detected as a second hit.
On the other hand, an NPM1 mutation was observed in G2, showing a divergent outcome compared with G7.
G2 attained CR after the first induction of chemotherapy and did not relapse.
We observed a divergent outcome in patients having the same SNPs but with different somatic mutations as a second hit.
Somatic mutations that were differentially detected at the time of diagnosis strongly suggest clinical implications and warrant future study in a larger cohort to enhance our understanding of the mechanisms of disease progression in AML.
Title: Germline SNPs (KRAS and DNMT3A) in 10 AML patients (samples, G1 to G10)

Title: Distribution of germline SNPs (18 top genes)a in 10 AML patients (samples, G1 to G10)

Title: Somatic mutation calling results in 10 AML patients (samples, G1 to G10)

Title: Clinical course of patients
Title: Annotation of Genes Having Candidate Somatic Mutations in Acute Myeloid Leukemia with Whole-Exome Sequencing Using Concept Lattice Analysis

In cancer genome studies, the annotation of newly detected oncogene/tumor suppressor gene candidates is a challenging process.
We propose using concept lattice analysis for the annotation and interpretation of genes having candidate somatic mutations in whole-exome sequencing in acute myeloid leukemia (AML).
We selected 45 highly mutated genes with whole-exome sequencing in 10 normal matched samples of the AML-M2 subtype.
To evaluate these genes, we performed concept lattice analysis and annotated these genes with existing knowledge databases.
Acute myeloid leukemia (AML) is one of the most wellstudied diseases in the genomic research area [1, 2].
AML occurs usually in middle-aged people and is diagnosed by increasing leukemic myeloblasts in blood over 30% [3].
AML is a genetically heterogeneous disease, since 1/3 of AML patients have chromosomal rearrangements, like t(8;21) and t(15;17), but other AML patients have normal karyotypes [4].
With recent advances of high-throughput genomic technology, a favorable prognosis has been observed with some genetic changes in cytogenetically normal AML [5].
These results were reflected by the World Health Organization (WHO) diagnostic criteria; the NMP1 and CEBPA mutations were included in the 2008 revision of these criteria [6].
The molecular change of AML is considered to be the accumulation of somatic mutations in hematopoietic progenitor cells [7].
Next-generation sequencing technology gave us new insights into the clonal heterogeneity of leukemic mutations so that we can make an explanation why some of these mutations are highly reproducible but others are very rare [8].
Still, in 30% of cytogenetically normal AML, the genetic causality origin or strongly associated genetic changes have not yet been discovered [9, 10].
With advances of high-throughput technology, discovery of disease-associated genes is growing [11].
As a consequence, the genetic knowledge databases are growing rapidly.
Accordingly, the annotation of candidate causal genes in genetic studies is a very challenging process for researchers.
We propose a workflow of the detection of somatic mutation candidates in 10 normal matched AML samples and introduce concept lattice analysis for clustering the samples that have highly mutated genes in common.
Title: Primacy sequence analysis

We received the fastq files of whole-exome sequencing results of tumor and matched normal sample data of 10 AML patients from the Korea Genome Organization in December 2012.
There were no patient-related medical or characteristic data.
We aligned the sequencing reads to the human reference genome (hg 19, GRCh37) from USCC by BWA 0.6.2 [12] (Figs.1 and 2).
To filter the known single nucleotide polymorphisms (SNPs), we used dbSNP bulid 137.
We removed PCR duplicates and filtering low-quality SNPs by Samtools 0.1.18 [13], Picard 1.68 [14], and GATK 2.3.4 [15].
After the filtering process, the SAM file was converted to VCF file by VCF Tools 0.1.10 [16].
For detecting somatic mutation candidates, we obtained the difference in VCF files between tumor and normal samples.
For annotation of these somatic mutation candidates, we used the ANNOVAR tool [17].
Title: Formal concept analysis

We used formal concept analysis (FCA) for the construction of hierarchical relationships among samples sharing highly mutated genes [18].
FCA is a useful method in conceptual clustering of objects, attributes, and their binary relationship.
In FCA, the sets of formal objects and formal attributes together with their relation to each other form a &#34;formal context,&#34; which can be represented by a crosstable.
In our case, the objects are 10 AML samples, and the attributes are 45 highly mutated genes.
We defined the formal context as K = (G, M, I), where G is a set of objects (i.e., samples), M is a set of attributes (i.e., mutated genes), and I ⊆ G × M is the incidence relations where (g, m) ⊆ I if object g has attribute m. For A ⊆ G and B ⊆ M, we define the operators A&#39; = {m ∈ M|gIm for all g ∈ A} (i.e., the set of attributes common to the objects in A) and B&#39; = {g ∈ G|gIm for all m ∈ B} (i.e., the set of objects common to the attributes in B).
A pair of (A, B) is a formal concept of k(G, M, I) if and only if A ⊆ G, B ⊆ M, A&#39; = B, and A = B&#39;.
A is called the extent and B is the intent of the concept (A, B).
The extent consists of all objects belonging to the concept while the intent contains all attributes shared by the objects.
The concept of a given context is naturally ordered by the subconcept-superconcept relation, defined by (A1, B1) ≤ (A2, B2): &lt;=&gt; A1 ⊆ A2 (&lt;=&gt; B2 ⊆ B1).
The ordered set of all concepts of the context (G, M, I) is denoted by C(G, M, I) and is called the concept lattice of (G, M, I).
We represent the structure of this concept lattice with a Hasse diagram, in which nodes are the concepts and the edges correspond to the neighborhood relationship among the concepts.
All concepts above an object label (below the attribute label) include that object (attribute).
The top element of a lattice is a unit concept, representing a concept that contains all objects.
The bottom element is a zero concept having no object.
Title: Overview of mutations

We have identified 12,908 somatic mutation candidates in 10 AML sequenced exomes, including 1,281 point mutations, 625 insertion/deletions (Indels) (Table 1, Fig.3).
The point mutations include 7,483 synonymous single nucleotide variations (SNVs), 4,297 nonsynonymous SNVs, 282 stopgain SNVs, 14 stoploss SNVs, and 5 frameshift substitutions, and the Indels include 247 frame shift insertions, 310 frameshift deletions, and 68 nonframe shifts (Fig.4).
For each patient, the average nonsynonymous mutation count was 429.7 (SD, 97.16).
About 342 to 665 genes have nonsynonymous somatic mutation candidates at least once in each AML sample (Table 2).
Recurrent mutated genes were observed in all samples.
Title: Mutation analysis

The most frequently mutated genes across all samples were USP9Y and MUC5B; these genes were mutated in 5 samples.
These genes were also highly mutated in each sample; for USP9Y genes, it had 6 nonsynonymous mutations in sample 3.
We have selected 45 highly mutated genes (1.5%) from 2981 mutated genes.
We defined highly mutated genes as genes having 3 or more nonsynonymous mutations in each sample (Table 3).
In a comparison of mutations with the COSMIC database [19], among 45 highly mutated genes, 21 genes matched to hematopoietic and lymphoid tissue malignancy terms, and 21 genes matched to other cancer types.
In 3 genes, there was no matched term in COSMIC (Table 4).
We used the concept lattice to construct the hierarchical relationship between the samples that had 45 highly mutated genes.
Concept Biolattice analysis is a mathematical framework based on concept lattice analysis for better biological interpretation of genomic data.
The top element of a lattice is a unit concept, representing a concept that contains all objects.
The bottom element is a zero concept having no object [20, 21].
For comparing with the Concept lattice (Fig.5), we also performed hierarchical clustering analysis by Ward method.
In hierarchical clustering, cluster 1 has 5 samples (nos.
1, 2, 5, 9, and 10), cluster 2 has 2 samples (nos.
4 and 7), and others have 1 sample each (Fig.6).
We divided the samples into 4 subgroups by interpretation of the concept lattice result (Fig.7).
Lattice subgroup 1 shared SYNE1 gene mutation, and samples 3, 4, and 7 were included in this subgroup.
Subgroup 2 was comprised of 5 samples (nos.
1, 2, 5, 6, and 9) that had MUC5 gene mutations in common.
Samples 10 and 8 could be isolated by the uniqueness of their mutated gene sharing pattern.
In this study, we proposed a workflow of matched normal AML exome sequencing analysis and the framework for defining sample subgroups.
We observed every sample having a nonsynonymous mutation associated with hematological and lymphoid malignancy genes, but the candidate oncogenes showed diverse characters.
We selected 45 genes that had 3 or more nonsynonymous mutations and performed hierarchical clustering analysis of the samples by these genes.
In classic hierarchical clustering analysis by Ward&#39;s method, we could not identify the genetic relationship of those clusters.
On the other hand, the result of concept lattice analysis gave us insight into the mutational pattern of each sample.
In subgroup 1, samples 3, 4, and 7 shared SYNE1 gene mutations.
SYNE1 gene encodes a spectrin repeat-containing protein expressed in skeletal and smooth muscle and peripheral blood lymphocytes that localizes to the nuclear membrane [21].
This gene is not a well-known leukemic gene but is observed in some hematological malignancies and other cancer types [22].
In glioblastoma, SYNE1 mutation is significantly correlated with the overexpression of several known glioblastoma survival genes [23].
In the case of sample 3, the ITGAX gene, encoding ankyrin repeat domain 18A, was mutated.
This gene is well known by the association with leukemia [24] and lung cancer [25].
For sample 4, the possible oncogene is LAMC3.
LAMC3 gene encodes laminins, which are the major non-collagenous constituent of basement membrane.
LAMC3 mutations are associated with several cancers, including colon cancer, lung cancer, and melanoma, and candidate tumor suppressor genes in bladder transitional cell carcinoma [26].
LAMC3 is involved in the phosphatidylinositol 3-kinase.
Akt signaling pathway, since it has a role in cell adhesion.
The ANKRD18A gene is the oncogene candidate for sample 7 and is a novel epigenetic regulation gene in lung cancer [25].
Therefore, it is possible that the pair relationship of those genes (ITGAX-SYNE1, LAMC3- SYNE1, and ANKRD18A-SYNE1) could contribute together to evolve the leukemic cell transformation.
The major limitation of our study is that we could not validate the mutation results by Sanger method or deep sequencing.
We selected highly mutated genes having 3 mutations or more, but this definition is arbitrary, so we might have lost candidate oncogenes in some patients.
In this study, we suggest the concept of clustering samples that have diverse mutated genes.
AML is very heterogeneous genetic disease.
Despite the small number of samples we have studied, the genetic variation patterns were not common for all samples.
It could have been better to evaluate more sample data for analysis by clustering analysis.
Primary sequence analysis pipeline.
Workflow of detection of somatic mutation candidate from exome sequencing of normal matched samples from 10 acute myeloid leukemia.
Distribution of somatic mutation candidates in all samples.
NS SNV, nonsynonymous single nucleotide variation.
Distribution of nonsynonymous somatic mutations in 10 acute myeloid leukemia samples.
NS SNPs, nonsynonymous single nucleotide polymorphisms.
Concept lattice of 45 genes and 10 acute myeloid leukemia patients having 3 or more nonsynonymous mutations, annotated by COSMIC database.
Red rectangles represent annotated hematopoietic and lymphoid tissue malignancy; yellow rectangles represent other cancer type annotated in the COSMIC database.
Hierarchical clustering of samples by binarized score of 45 highly mutated gene states.
Subgroup analysis by concept lattice.
(A) Subgroup 1 shares SYNE1 mutation in samples 3, 4 and 7.
(B) Subgroup 2 shares MUC5B mutation in samples 1, 2, 5, 6, and 9.
(C) Subgroup 3 sample 8 only has mutated genes, such as MUC2 and HELZ2.
(D) Subgroup 4 has sample 10, having only mutated genes, like MUC6, CDH23, BRD2, OR6V1, and DNAH17.
Title: The distribution of somatic mutation candidates in 10 AML samples

Title: Classification of genes according to the count of mutations in each sample

Title: List of 3 more mutated genes in 10 AML samples

Title: Comparison of list of 3 more mutated genes with COMIC database
Title: Genetic Risk Prediction for Normal-Karyotype Acute Myeloid Leukemia Using Whole-Exome Sequencing

Normal-karyotype acute myeloid leukemia (NK-AML) is a highly malignant and cytogenetically heterogeneous hematologic cancer.
We searched for somatic mutations from 10 pairs of tumor and normal cells by using a highly efficient and reliable analysis workflow for whole-exome sequencing data and performed association tests between the NK-AML and somatic mutations.
We identified 21 nonsynonymous single nucleotide variants (SNVs) located in a coding region of 18 genes.
Among them, the SNVs of three leukemia-related genes (MUC4, CNTNAP2, and GNAS) reported in previous studies were replicated in this study.
We conducted stepwise genetic risk score (GRS) models composed of the NK-AML susceptible variants and evaluated the prediction accuracy of each GRS model by computing the area under the receiver operating characteristic curve (AUC).
The GRS model that was composed of five SNVs (rs75156964, rs56213454, rs6604516, rs10888338, and rs2443878) showed 100% prediction accuracy, and the combined effect of the three reported genes was validated in the current study (AUC, 0.98; 95% confidence interval, 0.92 to 1.00).
Further study with large sample sizes is warranted to validate the combined effect of these somatic point mutations, and the discovery of novel markers may provide an opportunity to develop novel diagnostic and therapeutic targets for NK-AML.
Acute myeloid leukemia (AML), which is characterized by the rapid growth of abnormal myeloid cells in the marrow, is a highly malignant and cytogenetically heterogeneous type of cancer [1].
AML is categorized into three risk groups: favorable, intermediate, and unfavorable.
AML with a normal karyotype (NK-AML) accounts for approximately 40% to 50% of adult patients and 25% of pediatric patients with AML and is composed of a heterogeneous group with an intermediate prognosis [2].
In the past few years, next-generation sequencing technologies have been developed rapidly in the field of cancer genomics [3, 4] and enabled a direct genome-wide association test for NK-AML at single-base resolution in an unbiased manner [5].
A comprehensive understanding of the genetic lesions is an important basis for developing personalized therapies for the treatment of NK-AML [6].
Ley et al.
[5] carried out the first whole-genome association study for AML by using single-end whole-genome sequencing data and found 181 single nucleotide variations (SNVs), including 28 indels, nonsynonymous, and splicing sites (NPM1 and FLT3, respectively).
Mardis et al.
[7] performed paired-end whole-genome sequencing for cytogenetically normal AML, found seven nonsynonymous SNVs (nsSNVs), one splice site SNV, two indels in coding regions, and 52 somatic point mutations in conserved or regulatory genomic regions, and observed a 10% recurrence rate in the IDH1 mutation in 188 additional AML samples.
In the following year, Ley et al.
[5] carried out a case-control study of relapsed AML among patients who were reported in 2008.
They found a one-base-pair deletion in the DNMT3A gene, which showed a recurrence risk of 22% in AML [8].
De Weer et al.
[9] detected deletions within a chromosomal region of 7q35-q36, which contains the CNTNAP2 gene, in both Kasumi-3 and MUTZ-3 cell lines using high-resolution array comparative genomic hybridization (CGH).
Particularly, Bejar et al.
[10] identified novel associations between somatic mutations in two genes, ETV6 and GNAS, and both myelodysplastic syndrome (previously known as preleukemia) and AML using high-throughput genotyping technology.
In the same year, Link et al.
[11] identified a novel TP53 susceptibility mutation in AML patients through the use of whole-genome sequencing.
Recent exome-sequencing studies found more than 60 susceptible variants, including indels and splicing site mutations, in AML patients [12].
However, to the best of our knowledge, none of the previous studies attempted to predict the risk of NK-AML by analyzing whole-exome sequencing data.
This study aimed to evaluate the effect of nsSNVs in 10 pairs of tumor and normal cells of NK-AML patients with whole-exome sequencing data.
Finally, we developed genetic risk prediction models for NK-AML by creating genetic risk scores (GRS).
Title: Whole-exome sequencing data analysis

A total of 20 DNA samples were obtained from 10 pairs of bone marrow (tumor) and germline (normal) cells collected from 10 NK-AML patients.
We illustrated the workflow for whole-exome sequencing data analysis in Fig.1.
The raw reads in FASTQ file format were mapped to the human reference, which was developed by the 1,000 genome project (human_g1k_v 37.fasta), by using the Burrows-Wheeler aligner (BWA v 0.6.1), which generates SAM format files [13].
The SAM files were converted into binary format files (BAM) by samtools v0.2.6, which reduces the file size and improves computing efficiency [14].
The read group information of the sequencing machine was added to the head of the BAM files.
The aligned sequences were sorted in the order of chromosomes by Picard v1.79 (http://picard.sourceforge.net) and went through a PCR duplicate marking process, which enables the Genome Analysis Toolkit (GATK) to ignore duplicates in subsequent processing [15].
Finally, the BAM files were indexed by bamtools v2.2.0 [16].
We performed a local realignment prior to recalibration, which gives the most accurate quality scores for each sample.
Local realignment with known indel sites (Mills_and_1000G_gold_standard.indels.b37.vcf, 1000G_phase1.indels.b37.vcf) for each individual does not require multiple sample realignments, which demand extreme computational power.
However, we carried out a local realignment with the matched tumor and normal samples together to prevent misalignment due to the differences between these two tissue types.
Recalibration was performed with multiple known sites (dbSNV_137.b37, Mills_and_1000G_gold_standard.indels.b37.vcf, and 1000G_phase1.indels.b37.vcf), which may increase recalibration accuracy.
We reduced the BAM file size to about 1/100 of the original file size by using the GATK tool, which saved variant calling time without losing any essential information.
We used the UnifiedGenotyper of GATK for variant calling, followed by variant recalibration with known sites (hapmap_3.3.b37.vcf, 1000G_omni2.5.b37.vcf, dbsnp_137.b37.vcf, and Mills_and_1000G_gold_standard.indels.b37.vcf), and annotated by them using snpEff v2.0.57 [17].
Title: Statistical analysis

We performed logistic regression analyses between the somatic mutations and NK-AML using PLINK/SEQ v0.08 (http://atgu.atgu.mgh.harvard.edu/plinkseq), which provides powerful utilities in variant call format (vcf) for analyzing whole-exome and -genome data.
Further, we verified the odd ratios and p-values estimated from PLINK/SEQ using Stata, v11.2 (Stata Corp., College Station, TX, USA).
We selected the somatic nsSNVs with complete call rates and evaluated the GRS models composed of the variants associated with NK-AML.
The GRS was calculated for each individual by accumulating the number of risk alleles (0, 1, or 2) of the SNVs.
We created stepwise GRS models, comprised of the selected SNVs, according to their significance level; if the significance level was equal between two or more SNVs, we selected the SNVs in the order of their chromosomal position.
In addition, we evaluated a GRS model that consisted of gene variants reported in previous leukemia studies [9, 10, 18, 19].
We compared the area under the receiver operating characteristic curve (AUC) of each GRS model using the &#34;roctab&#34; and &#34;roccomp&#34; commands in Stata.
The improvement in concordance between the empirical and reported quality scores of SNVs is shown in Fig.2A.
A significant improvement in the accuracy of quality scores after recalibration is shown in Fig.2B.
The association test for NK-AML yielded 42 SNVs that had a p-value of less than 0.05 and passed the quality threshold of PLINK/SEQ (data not shown).
Among 42 SNVs, we excluded 11 SNVs with no rs ID, four SNVs with call rates less than 1, and five synonymous SNVs for subsequent analysis.
A total of 21 nsSNVs located in 18 candidate genes were selected and included in the GRS models after considering pairwise linkage disequilibrium (Table 1).
This study replicated three somatic mutations of the MUC4, CNTNAP2, and GNAS genes, which were reported in previous studies on leukemia [9, 10, 18, 19].
In addition, we identified novel point mutations in 15 other genes.
The SNVs rs75156964 (ATP13A3; odds ratio [OR], 9.33; p = 0.013) and rs56213454 (GNAS; OR, 9.33; p = 0.013) showed the most significant evidence for association, and rs6604516 (PPIAL4G; OR, 21.00; p = 0.016) showed the strongest effect size among the 21 SNVs.
Other SNVs also showed strong effects in the risk of NK-AML (OR, 9.33 to 13.5).
We compared the prediction accuracy of 21 stepwise GRS models.
The predictive power for NK-AML reached 100% in model 5, which consisted of five nsSNVs: rs75156964 (ATP13A3, 3q29), rs56213454 (GNAS, 20q13.3), rs6604516 (PPIAL4G, 1q21.1), rs10888338 (OR2T33, 1q44), and rs2443878 (ANKRD36, 2q11.2) (Table 2, Fig.3A).
The five SNVs (rs75459784, rs55824312, rs200239604, rs56213454, and rs56371919) in three previously reported genes-MUC4, CNTNAP2, and GNAS-also showed a high AUC value (AUC, 0.98; 95% confidence interval [CI], 0.92 to 1.00).
We evaluated the effects of 21 nsSNVs of 18 candidate genes for NK-AML in this study.
Among 18 genes, we replicated previously reported associations of two genes (CNTNAP2 and GNAS) with AML [9, 10].
The mutation in the MUC4 gene has been reported to be associated with acute lymphoblastic leukemia [18, 19].
The MUC4 gene encodes a mucin protein and a high-molecular-weight glycoprotein in humans.
This integral membrane glycoprotein, which is observed on the cell surface, plays various roles in tumor progression [20].
Particularly, MUC4, complexed with v-erb-b2 erythroblastic leukemia viral oncogene homolog 2 (ERBB2), results in the repression of apoptosis and the stimulation of proliferation; in addition, the overexpression of MUC4 in carcinoma cells promotes loss of cellular polarity by ERBB2-mediated disruption [21].
The CNTNAP2 gene is one of the largest genes in the entire human genome and occupies approximately 1.5% of chromosome 7.
This gene encodes a neurexin family member that functions in the vertebrate nervous system.
The GNAS gene has a highly complex imprinted expression pattern.
This gene provides instructions for making one component, the stimulatory alpha subunit, of a protein complex called a guanine nucleotide-binding protein (G-protein).
G-protein alpha subunit regulates the cyclic AMP (cAMP) pathway.
GNAS gene mutations are known to be associated with high cAMP signaling [22].
The novel associations between 15 somatic mutations (PPIAL4G, LRRN2, OR2T33, ANKRD36, ATP13A3, TRIML1, PABPC1, MUC5B, TAS2R43, ATF7IP, KIAA1033, ATP8A2, L2HGDH, LYPD5, and SUSD2) and NK-AML need to be replicated, and their functional mechanisms in the AML should be investigated in future studies.
The GRS models that were comprised of the somatic nsSNVs showed extremely high predictive accuracy for the risk of NK-AML (86% to 100%).
The combination of five nsSNVs of previously reported genes (MUC4, CNTNAP2, and GNAS) had a predictive ability of 98% for the risk of NK-AML in this study.
One of the limitations in this study was the small number of study subjects (n = 10), which resulted in substantial variations in the 95% CIs of the effects (ORs) of each SNV.
In conclusion, we have highlighted 21 susceptibility nsSNVs located in 18 genes.
The GRS model that comprises five candidate SNVs is highly informative in predicting the risk for NK-AML.
The discovery of novel markers may provide an opportunity to develop novel diagnostic and therapeutic targets for NK-AML.
Further study with a larger sample size is necessary to validate the AML-related gene mutations and will provide an opportunity to develop a powerful genetic risk prediction model for NK-AML.
Next-generation sequencing data analysis workflow.
Base quality score recalibration.
Raw reads (pink) and recalibrated (blue) base quality scores for a patient.
Panel (A) compares the reported base quality scores to the empirical estimates.
Panel (B) shows the difference between the average reported and empirical quality scores for each machine cycle.
Receiver operating characteristic analysis to measure the discriminatory power of each genetic risk score model in predicting normal-karyotype acute myeloid leukemia.
(A) Model 1 (rs75156964), model 3 (rs75156964, rs56213454, and rs6604516), and model 5 (rs75156964, rs56213454, rs6604516, rs10888338, and rs2443878).
(B) A model comprising five candidate single nucleotide variations (rs75459784, rs55824312, rs200239604, rs56213454, and rs56371919) in MUC4, CNTNAP2, and GNAS, reported in previous studies (area under the receiver operating characteristic curve, 0.98; 95% confidence interval, 0.92 to 1.00).
GRS, genetic risk score.
Title: Associations between 21 SNVs and NK-AML in 10 pairs of tumor and normal samples

Title: The area under the ROC curves for genetic risk score models consisting of SNVs
Title: Eukaryotic DNAJ/K Database: A Comprehensive Phylogenomic Analysis Platform for the DNAJ/K Family

Proteins in DNAJ/K families are ubiquitous, from prokaryotes to eukaryotes, and function as molecular chaperones.
For systematic phylogenomics of the DNAJ/K families, we developed the Eukaryotic DNAJ/K Database (EDD).
A total of 12,908 DNAJs and 4,886 DNAKs were identified from 339 eukaryotic genomes in the EDD.
Kingdom-wide comparison of DNAJ/K families provides new insights on the evolutionary relationship within these families.
Empowered by &#39;class&#39;, &#39;cluster&#39;, and &#39;taxonomy&#39; browsers and the &#39;favorite&#39; function, the EDD provides a versatile platform for comparative genomic analyses of DNAJ/K families.
Ubiquitous heat shock proteins (HSPs) play essential roles as molecular chaperones that protect cells through protein folding as a stress response under heat shock [1].
The HSPs are well conserved in all eukaryotes and prokaryotes [2].
The HSPs are classified on the basis of their approximate molecular mass (e.g., the 70-kDa species is the Hsp70 family).
DNAJ and DNAK are members of the Hsp40 and Hsp70 families, respectively [3].
These proteins are known to cooperate in many cellular processes, such as DNA replication, protein folding, protein export, and stress response in an ATP hydrolysis-dependent manner [4-6].
The cooperation of DNAJ and DNAK is dependent on the interactions between the J domain of DNAJ and the ATPase domain of DNAK [7].
DNAJ proteins were classified using the J, zinc finger, and carboxy-terminal domains [8].
Type I DNAJs contain all three domains, while type II proteins have a J domain linked directly to a carboxy-terminal domain without a zinc finger domain.
The type III proteins include only the J domain.
However, such domain architectures of DNAJ were not deliberated on most eukaryotic genome annotations, which might cause overestimation of DNAJ entries [9, 10].
Likewise, many studies of DNAKs have been used without any unified nomenclature or notation [11, 12], although ATPase and peptide-binding domains had been defined [13].
Recent classifications and identifications based on functional domains of the DNAJ/K families revealed 22, 89, and 41 DNAJs [9, 10, 14], and 14, 18, and 17 DNAKs [11, 12, 15] in the yeast, Arabidopsis, and human genomes, respectively.
However, there is no informative platform that archives the previously identified DNAJ/Ks.
Furthermore, only a few genomes have systematically identified DNAJ/K families, although hundreds of genomic sequences are publicly available.
This problematic circumstance has demanded the development of a comprehensive platform that archives sequence information, classifies them according to their domain structures, and analyzes them using bioinformatics tools.
The platform should provide user-friendly interfaces and easy access.
Therefore, we developed a web-based phylogenomic platform called the Eukaryotic DNAJ/K Database (EDD).
Title: Identification and classification of DNAJ/K family from 339 eukaryotic genomes

An automated pipeline was programmed to identify and classify the DNAJ/K families (Fig.1).
All protein sequences of the 339 genomes deposited in the Comparative Fungal Genomics Platform (CFGP; http://cfgp.snu.ac.kr) [16] were filtered according to the following description.
Six InterPro terms (release 12.0) for DNAJ (IPR012724, IPR008971, IPR003095, IPR001305, IPR002939, and IPR001623) and three for DNAK (IPR013126, IPR001023, and IPR012725) were used to retrieve corresponding sequences.
When the J domain or any of the DNAK domains was at least 50 amino acids (aa), it was defined as &#39;putative&#39;, and the others were tagged as &#39;candidate DNAJ/Ks&#39;.
The HPD tripeptide motif and glycine-rich region were identified from the &#39;putative DNAJ&#39; sequences.
The DNAK family was clustered by sequence similarity (BLAST E-value cutoff e-5) using the Tribe Markov clustering (MCL) algorithm [17], because no class has been made in this family.
Finally, 12,908 DNAJs and 4,886 DNAKs were identified and deposited into EDD.
In the DNAJ family, 895, 3,172, and 8,700 proteins were classified as type I, II, and III, respectively.
Twenty-one clusters were determined in DNAK, and 4,853 proteins (99.1%) belonged to the first four clusters.
Title: Web-based interfaces of the EDD

First, all protein and nucleic acid sequences identified as DNAJ/Ks can be accessed according to class, species, and cluster through user-friendly interfaces, such as the &#39;class&#39; and &#39;cluster&#39; browsers on the EDD website.
For example, statistics pages of each class were displayed using tables and diagrams in the &#39;class&#39; browser.
The statistics pages are automatically updated when new data are added to the EDD.
Secondly, the EDD allows users to browse the detailed information on DNAJ/Ks and to analyze it via &#39;favorite&#39;.
It is a customized cyber-workbench where users can create folders, store favored items, and perform 10 bioinformatics analyses, such as BLAST search, BLASTMatrix, ClustalW, DNAJ/K statistics, a DNAJ class viewer, a DNAJ/K domain viewer, and a glycine ratio viewer.
Especially, &#39;favorite&#39;, implemented in the EDD, is specialized for the functional analysis of DNAJ/K proteins (e.g., DNAJ/K domain view).
The data saved in &#39;favorite&#39; can be shared with CFGP 2.0 for further comparative analysis [16].
Finally, a highlighted function of the EDD is a &#39;taxonomy&#39; browser powered by the &#39;Species-driven User Interface&#39;, which provides an interactive interface for displaying the taxonomical hierarchy for DNAJ/Ks.
Users can drag and drop any taxon or taxa and perform sequence download, statistics, class and cluster analyses, ClustalW alignment, and BLAST search.
The comprehensive information on DNAJ/Ks conserved in 339 genomes over seven phyla is a useful resource for molecular chaperone studies in eukaryotes.
Phylogenetic analysis between phyla or kingdoms as well as genomic analysis within a species can be performed in the EDD.
The user-friendly web interfaces and various analysis tools implemented in this database will accelerate the management of large-scale data.
The EDD will be a novel platform for phylogenomic studies and a model for comprehensive genomic analyses.
System architecture and data processing pipeline in the Eukaryotic DNAJ/K Database.
MCL, Markov clustering.
Title: siMacro: A Fast and Easy Data Processing Tool for Cell-Based Genomewide siRNA Screens

Growing numbers of studies employ cell line-based systematic short interfering RNA (siRNA) screens to study gene functions and to identify drug targets.
As multiple sources of variations that are unique to siRNA screens exist, there is a growing demand for a computational tool that generates normalized values and standardized scores.
However, only a few tools have been available so far with limited usability.
Here, we present siMacro, a fast and easy-to-use Microsoft Office Excel-based tool with a graphic user interface, designed to process single-condition or two-condition synthetic screen datasets.
siMacro normalizes position and batch effects, censors outlier samples, and calculates Z-scores and robust Z-scores, with a spreadsheet output of &gt;120,000 samples in under 1 minute.
Human tissue-derived cell lines have served as an effective platform for understanding the molecular biology of diseases and increasingly for drug discovery [1, 2].
For wholesale analysis of gene sets or chemical compound files, cell line-based high-throughput screening is typically performed on a well-to-well basis in high-density microtiter plates.
From this, significant changes in cellular phenotype by each perturbation in a given library compared to mock controls or other samples are identified.
Examples of perturbations include synthetic small molecules, natural products, cDNA expression libraries, and, of primary focus here, genomewide collections of short interfering RNAs (siRNAs).
It is a truism that cell-based screens are highly vulnerable to environmental variations.
In particular, siRNA screens involve multiple processing steps employing automatic or semiautomatic liquid-handling equipment carried out over many days.
Each step introduces position-specific variables and distinct operational failures.
This, combined with unavoidable batch effects from cell cultures and reagent efficacy, can introduce asymmetric phenotypic variation that is independent of the perturbations of interest.
Accounting for and correcting this variation is well recognized as an essential component of successful high-throughput screening.
Statistical methods to process whole-genome siRNA screen data have been reported by others [3, 4], but limited open-source tools are currently available [3, 5, 6].
Moreover, as most available open-source tools are written in R/bioconductor [3, 6], modifications that require screen-specific command lines are challenging without prior experience in script programming.
Very recently, a web-based tool (web cellHTS2) and its standalone package [5] have been published.
However, they either run on a remote server or require other specialized preinstalled software.
Here, we present siMacro, a GUI-based simple tool for processing cell-based high-throughput screening datasets.
siMacro has been implemented in Visual Basic for Applications (VBA) and packaged as a Microsoft Office Excel add-in.
It allows one-step, fast, and easy suppression of outlier values; normalization; and standardization of a complete raw dataset from a genomewide siRNA screen in an intuitive spreadsheet format.
The tool processes the data points associated with a 2-condition genomewide screen with biological triplicates on a standard laptop computer in less than 1 minute.
Title: Prerequisite

We assume the screen was done in 96- or 384-well plates and passed the standard quality control metric (by Z or Z&#39; factor, for example).
siMacro is robust against sporadic bad wells from triplicate experiments but will not censor an entire plate compromised by massive failure, such as broad contamination.
We also assume siRNAs in the library plate are randomly distributed, which is generally true for most commercially available genomewide siRNA libraries.
siMacro currently supports 1- or 2-condition screens.
Title: Input data

Title: siMacro requires all the individual plate readouts be put into an Excel spreadsheet with the field headers: day or cell batch, plate name, well name, and raw data columns per siRNA or a pool (Fig.1C).
siMacro allows only numerical data field values except for manually censored &#34;bad&#34; wells masked by &#39;NA.&#39;
Title: Normalization

Most popular normalization protocols employ either on-plate control-based or sample-based methods.
Although there is no golden rule for this, we prefer the latter for most siRNA screens, as 1) cell-based siRNA screening is vulnerable to within-plate variation, such as an edge effect and column/row effects that are not corrected by on-board controls [4], and 2) because siRNA hits are often rare within a given test plate, the majority of samples proxies the mock control.
Therefore, siMacro currently only supports normalization by samples at the resolution of a plate, row, and column.
Depending on the cellular phenotype and experimental design, the optimal choice of resolution varies and can therefore be specified by the screener.
If row or column effects are trivial or there are more than 10-30 hits per 96-well plate, a user can choose plate median normalization (default).
As many liquid dispensers move row-wise, a row effect is very common in the siRNA screens.
If the hits/plate is less than 10 per 96-well plate and accompanied by a significant row effect, row-median normalization is a better choice.
Title: Censoring bad wells

Sporadic bad wells are often manually censored one by one, which is inefficient and error-prone in large-scale screens.
If the experiment is done in triplicate or more, bad wells can be detected automatically by the inflated coefficient of variation (CV) among the replicates.
siMacro identifies bad wells by applying a user-defined cutoff to the CV among normalized values from a replicate.
The default is 1%, meaning 1% of the total genes with the highest CV will have a masked outlier well.
This significantly reduces false positives but can also overcorrect.
Therefore, siMacro reports flags for all genes with censored wells to aid user decisions with regard to exclusion from downstream analysis.
Title: Standardization and hit scoring

A unified scoring scheme is employed that accounts for batch effects from multiday experiments.
Under the assumption of normality, the Z-score, which indicates how many standard deviations an observation is away from the mean, is an intuitive scoring metric.
However, it is sensitive to biological outliers (hits) in the data pool, resulting in deflated scores.
An alternative is to use the robust Z-score, which is a measure of the median absolute deviation from the median.
siMacro calculates the Z-score and the robust Z-score from the user-provided unit of experiment: e.g., the day, cell batch, or plate.
siMacro takes the mean of log2-transformed normalized values from replicates for the calculation.
For a 2-condition synthetic phenotype screen, the log2 ratio between the 2 conditions is used per siRNA to calculate the Z and the robust Z-scores for a synthetic effect.
siMacro returns output, including individual and mean normalized values, Z-score, robust Z-score, and flags, for the censored bad wells directly onto the Excel data sheet as additional columns (Fig.1D).
The original source written in R is implemented in VBA and packaged as an Excel add-in.
Since ease of use is the main objective of the plug-in, it provides simple GUI and depends only on Excel.
As Excel is widely utilized and familiar to most biologists, siMacro provides an immediate option for dataset processing together with Excel-based data visualization tools.
siMacro runs on Microsoft Office Excel 2007 or later for the Windows OS or Excel 2011 for the Mac OS.
On a standard laptop computer with a Pentium dual-core 1.73 GHz processor and 1 GB memory, the operating time is under 60 seconds to process a triplicate 2-condition genomewide data set.
As a test set, the direct lethality dataset from 21,125 sets of siRNA oligos in a non-small cell lung cancer line, H1155 [7], was used.
From raw data for cell viability (Fig.1A), it was evident that there was a row effect and day-to-day variation in transfection efficiency as well as sporadic technical outliers due to contamination (e.g., T1Raw of siCATSPER in Fig.1C).
With the default 1% highest CV filter and row-median normalization, siMacro censored, normalized, and standardized the dataset (Fig.1B and 1D).
siMacro has been developed for but is not limited to cell-based genomewide siRNA screens.
Data processing example of siMacro.
One of 3 measures of cell viability against siRNAs targeting 21,115 genes in a non-small cell lung cancer line, H1155 [7], is plotted before data processing (A) and after processing by siMacro (B).
Screen capture of the part of an input spread sheet (C) and siMacro ouput (D) are presented.
Title: ERRATUM: Figure Legend Correction.
Characteristics in Molecular Vibrational Frequency Patterns between Agonists and Antagonists of Histamine Receptors

Title: http://dx.doi.org/10.5808/GI.2012.10.2.128

Title: Genomics Inform 2012;10(2):128-132

The legend of Figure 1 was given incorrectly.
Histidine should be corrected as histamine.
Title: Perspectives of International Human Epigenome Consortium

As the International Human Epigenome Consortium (IHEC) launched officially at the 2010 Washington meeting, a giant step toward the conquest of unexplored regions of the human genome has begun.
IHEC aims at the production of 1,000 reference epigenomes to the international scientific community for next 7-10 years.
Seven member institutions, including South Korea, Korea National Institute of Health (KNIH), will produce 25-200 reference epigenomes individually, and the produced data will be publically available by using a data center.
Epigenome data will cover from whole genome bisulfite sequencing, histone modification, and chromatin access information to miRNA-seq.
The final goal of IHEC is the production of reference maps of human epigenomes for key cellular status relevant to health and disease.
Epigenome is the word describing the thing that is not included in genetics.
&#34;Epi&#34; means upon or outside; so, a direct translation is &#34;the thing that is not related with genetics or above genetics.&#34;
Practically, it means gene regulation phenomena, including by means of DNA methylation, histone modification, and miRNA.
To explore epigenomic regulation, International Human Epigenome Consortium (IHEC) was initiated to profile the epigenome of all human cell types.
To understand the current IHEC, it is required to turn back into the initial steps for fostering this consortium.
It has passed more than a decade after completion of the human genome project, and several efforts were conceived to accelerate finding disease-causing genetic variations, such as genome-wide association study (2003), Encyclopedia of DNA Element (ENCODE, 2003), The Cancer Genome Atlas (TCGA, 2006), International Cancer Genome Consortium (ICGC, 2008), and National Institutes of Health (NIH) Roadmap Epigenomics (2008) (Table 1).
IHEC was born with ground making effort of these several international consortiums.
Although the Human Genome Project, which defined the exact sequence of 3 billion bases in DNA, is one of the big milestones in genome research, cataloging DNA sequence information has a limited effect on defining whole varieties of genome variations that are expected to be crucial cause of disease.
Also, the sequence itself does not provide how the genome is packaged in chromosomes and chromatin, which is essential for organismal development and differentiation.
So, epigenetic regulation, in addition to transcription factor binding profile, is emerging as a powerful tool to identify disease correlation with genome information [1].
In 2004, a National Cancer Institute (NCI)-sponsored workshop, Epigenetic Mechanisms in Cancer, had initiating discussions of necessity of the Human Epigenome Project, which will be in parallel with the Human Genome Project.
In 2005, there was a special workshop on the BLUEPRINT for a Human Epigenome Project, the American Association of Cancer Research (AACR) Human Epigenome Workshop, on defining human epigenomes at high resolution.
The consensus of workshop participants was that there were compelling reasons from both scientific and public health perspectives to initiate a human epigenome project that could take full advantage of advances in several existing US and European initiatives.
In 2006, TCGA began as a 3-year pilot with an investment of $50 million each from the NCI and National Human Genome Research Institute (NHGRI) to analyze the genome and epigenome of 20 types of cancer tissue.
In 2008, amending the TCGA, international efforts were assembled to analyze cancer genome/epigenome variation.
As a result, the ICGC was launched in 2008 to coordinate large-scale cancer (epi)genome studies in tumors from 50 cancer types and/or subtypes, with participation of 15 countries.
ICGC is focused on obtaining a comprehensive description of genomic, transcriptomic, and epigenomic changes in 50 different tumor types.
Also, the international Alliance for Human Epigenetics and Disease (AHEAD) [2] was conceived among participating countries.
As a result, the Roadmap Epigenomics Program was fostered as AHEAD&#39;s suggestion.
Finally the NIH Roadmap Epigenomics Program was initiated with the funding of $200 million for 5 years from NIH [3].
Roadmap Epigenomics aimed to produce reference epigenome data from a variety of human cells.
As NIH and European epigenome mapping activity was commenced as a locally scattered project, there were efforts to assemble reference epigenome mapping as an international collaboration.
From 2009, several workshops were held in NIH or Europe, and finally at the 2010 Washington meeting, IHEC was officially born.
Title: The governance of IHEC

Through February 2013, 7 member organizations/countries joined IHEC, including NIH Roadmap Epigenomics, EU BLUEPRINT, Germany Deutsches Epigenom Programm (DEEP), Canadian Institutes for Health Research (CIHR), Japan Science and Technology Agency (JST), Korea KNIH, and Italy.
Several other programs, such as ENCODE, England, France, and Singapore, expressed their intents to join IHEC.
IHEC consists of an Executive Committee (EXEC, association of funding agency) and International Scientific Steering Committee (ISSC, representatives of the member research institution).
Title: The goal of IHEC

IHEC aims at cataloging 1,000 human reference epigenomes for the next 7 to 10 years.
The 1,000 epigenome includes 250 types of human cells, and 10% of the epigenome can be mapped on a model organism.
A long-term objective of IHEC is to determine the extent to which the epigenome has shaped human populations over generations and in response to the environment and to produce human reference epigenome maps for key cellular status relevant to health and disease.
IHEC will focus on key cellular status, such as stemness, immortality, proliferation, differentiation, senescence, and stress, thereby generating new knowledge that will catalyze progress in health research and regenerative medicine.
IHEC will coordinate rapid distribution of the data to the entire research community with minimal restrictions to accelerate translation of this new knowledge into health and diseases.
IHEC also will coordinate the development of common bioinformatics standards, data models, and analytical tools to organize, integrate, and display whole epigenomic data generated from this important international effort.
IHEC will set up an efficient structure that will coordinate this international effort so that the interest and priorities of individual participants, self-organizing consortia, funding agencies, and nations are addressed.
IHEC will encourage the minimal amount of redundancy between the different epigenetics efforts around the world.
IHEC will also interact and coordinate its efforts with other international projects, such as ICGC and ENCODE.
The secondary goal of IHEC is to catalyze the development of new and robust technologies that will facilitate the characterization and functional analysis of the epigenome in health and diseases, thereby driving down the costs of epigenome mapping substantially, and to support the dissemination of knowledge and standards related to new technologies, software, and methods to facilitate data integration and sharing between epigenetics researchers around the globe.
By the production of 1,000 epigenomes, IHEC expects that we can understand the key cellular aspects of 250 types of human cells more deeply.
Also, having ~250 types of reference epigenomes, we can have the power to compare disease epigenomes to normal references, which will help explain the missing heritability of common complex diseases.
Now, the production of &gt;600 epigenomes has been planned, and ~60 epigenomes have been finished [3, 4].
Title: The role and project of IHEC member institutions

Title: NIH Roadmap Epigenomics Program

The NIH Roadmap Epigenomics Program began in 2008 and has a budget of $200 million.
Under the cover of this program, the NIH Common Fund and NIH Institutes and Centers have supported a total of 68 grants in the areas of epigenetic technology development, identification of novel epigenetic marks, reference epigenome mapping, and disease epigenomics investigations.
The Reference Epigenomic Mapping Consortium, funded through the Common Fund&#39;s Roadmap Epigenomics Program, has been generating genome-wide epigenomic maps for a variety of cell and tissue types.
The majority of reference epigenomes that have been generated contains information on epigenetic modifications, including a core set of histone marks, DNA methylation, chromatin accessibility, and gene expression information.
A subset of reference epigenomes also contains an expanded set of 20 additional histone modifications [5].
Data for 52 complete and partial epigenome datasets for varieties of normal human cells and tissues are available.
Epigenomes were mapped on embryonic stem (ES) cells, ES cell derivatives, induced pluripotent stem cells, multiple fetal tissues, several varieties of blood and immune cells, breast cell types, placenta, and solid tissues (e.g., adipose, gastrointestinal tract, skin, and brain).
Roadmap Epigenomics plans to complete ~100 additional epigenomes by the end of the program [3, 4].
Title: European Union: BLUEPRINT

The BLUEPRINT consortium, consisting of 41 leading European universities, research institutes, and industry entrepreneurs, was selected for funding in early 2011.
The project officially started on October 1, 2011 and will run for 4-5 years.
BLUEPRINT has an overall budget of 39.4 million €, for which it receives 30 million € in funding from the European Commission.
BLUEPRINT aims to generate at least 100 reference epigenomes on blood lineage (hematopoietic) cells from healthy individuals and on their malignant leukemic counterparts.
The BLUEPRINT reference epigenome will cover the IHEC epigenome set and further expand up to 19 &#39;marks,&#39; called a &#39;BLUEPRINT set&#39; (including H3K79me3, H3K9/14ac, H4K20me3, H4pan/H4K16ac, H2AZ-ac, H2AZ, p300, and RNA PolII) [4, 6].
Title: Korea National Institute of Health (KNIH) Metabolic Epigenome Project

The project proposal from KNIH, the Center for Genome Sciences, Epigenome Mapping Center (EMC), 5 chronic disease-related hospital, were finally selected for funding during the period of 2012-2013.
The project officially started on January 2012 and will run for &gt;5 years.
KNIH has an overall budget of $10.9 million for 5 years.
Participating clinical institutions were chosen from Samsung Medical Center (heart tissue and cardiomyopathy), Hanyang University Hospital (blood, synoviocyte cell, and rheumatoid arthritis), Asan Medical Center (pancreatic tissue, adipose tissue, T2D, and obesity), and Seoul National University Hospital (kidney tissue and chronic kidney disease).
KNIH aims to generate at least 50 reference epigenomes and study them to advance and exploit knowledge of the underlying biological processes and mechanisms in health and disease.
KNIH will focus on 50 distinct types of homogenous cells of tissues related to 5 chronic diseases (heart failure, autoimmune diseases, diabetes, obesity, and chronic kidney disease) from normal and disease tissue, mainly obtained by tissue removal or transplantation.
The Korea reference epigenome will entail the analysis of 12 &#39;marks&#39;; an IHEC CORE set: DNAme, H3K4me3, H3K4me1, H3K9me3, H3K27me3, H3K27ac, H3K36me3, RNA-seq, and RNA-chromatin immunoprecipitation (RNA-ChIP), DNaseI hypersensitivity, formaldehyde-assisted isolation of regulatory elements (FAIRE)-seq, chromatin interaction analysis using paired end tags (CHIA-PET), and miRNA-seq (Tables 2 and 3) [4].
Title: Japan Science and Technology Agency

JST&#39;s leading program is the Core Research for Evolutional Science and Technology (CREST)&#39;s disease epigenome project.
In 2011, JST started a research area of &#34;Development of fundamental technologies for diagnosis and therapy based upon epigenome analysis (disease epigenome).&#34;
This research area focuses on the IHEC projects and various epigenome analyses related to diseases and stem cells.
In 2011-2012, 10 projects were awarded, including 3 IHEC projects.
JST&#39; proposal focused the epithelial cells of the digestive system (stomach, colon, and liver), the vascular endothelial cells, and the epigenome in germ cells and reproductive tissue (placental cell, cytotrophoblast, syncytiotrophoblast, endometrial cells, basal layer cells, proliferative functional layer cells, and secretary functional layer cells) (Tables 2 and 3) [4].
Title: The Italian epigenome program

The Italian epigenome program involves two initiatives: the Milan Genomic Initiative (MGI) and the Italian Flagship Project EPIGEN.
The MGI was started in 2011 as a joint operation among Istituto Europeo di Oncologia (IEO), Istituto Italiano di Technologia (IIT), and Istituto FIRC di Oncologia Molecolare (IFOM) and is located at the IFOM-IEO campus in Milan, a multi-institutional biomedical research area hosting over 500 scientists.
The MGI includes 16 independent research-groups in the field of epigenetics.
The three institutions are committed to 5-year support of the &#34;Center of Genomic Science&#34; for a total of ~10 million €.
EPIGEN was launched by the Italian Ministry of Research and started in January 2012.
It includes 25 independent groups.
EPIGEN is supported by funding of 30 million € for 4 years.
MGI has been a member of IHEC since 2011, and EPIGEN is currently following the required procedures with the Italian Ministry of Research for formal application to IHEC.
In parallel, both initiatives are working on a coordination plan with the goal of joint participation to IHEC.
MGI expects to provide 50 reference epigenomes, completed to IHEC standards, over the next 5 years, including cells from hematopoietic, mammary, adipose, liver, and neural tissues.
Epigen focuses on research projects, such as the role of chromatin structure, DNA methylation, and nuclear architecture in epigenetic regulation and epigenetic control through the noncoding component of the genome: structure and dynamics, epigenetic control of cellular identity and human pathologies, epidrugs for therapeutic use, and plant epigenomics [4].
Title: Germany epigenome project: DEEP

DEEP&#39;s funding is approx.
16 million € by the German Federal Ministry of Education and Research (BMBF), from 2012 to 2017.
DEEP creates an interdisciplinary research platform of 17 subprojects, linking 21 epigenomic mapping and functional analysis groups across Germany.
As the German contribution to IHEC, DEEP plans to produce and functionally interpret 70 reference epigenomes of selected human (and some murine) cells and tissues.
DEEP will focus on cell types connected to metabolic diseases, such as steatosis (hepatocytes and Kupffer cells) and obesity (adipocytes [small/large, visceral/subcutaneous], monocytes, and macrophages), as well as inflammatory diseases, such as Crohn disease (mucosa, macrophages, and T-memory/effector cells) and rheumatoid arthritis (fibroblasts, macrophages, and T-memory/effector cells) (Tables 2 and 3) [6].
Title: Canada: The Canadian Epigenetics, Environment and Health Research Consortium (CEEHRC)

In 2011, the CIHR established a national collaborative research funding strategy to coordinate epigenetics research in Canada, CEEHRC.
The budget of the CEEHRC is $50 million over 7 years.
CEEHRC is supporting two EMCs and two Epigenomic Data Coordination Centers (EDCCs), for a total investment of $15 million over 5 years.
The CEEHRC-funded EMCs are the Center for Epigenomic Mapping Technologies (CEMT), at the University of British Columbia/BC Cancer Agency, and the Multidimensional Epigenomics Mapping Center at McGill (MEMCM), at McGill University.
The funded EDCCs are at Simon Fraser University/BC Cancer Agency and at McGill University.
The CEMT group is focusing on a number of cancer and stem cell types, including studies on peripheral and lymphoid B-cells, T-lymphocytes, primitive hematopoietic cells, and patient-derived induced pluoripotent stem cell, as well as thyroid cells and mammary, endometrial, fallopian, and colonic epithelium.
The MEMCM group has three initial focus areas: autoimmune and inflammatory disease (using various blood cell types), cardio-metabolic disease (including cardiac and skeletal muscle, liver, adipose tissue, and pancreatic islets), and neuropsychiatric disease (including targeted brain areas in post mortem suicide human tissue, and animal models of hypothalamus/pituitary/adrenal axis dysfunction).
In addition, MEMCM specifically addresses population variations in epigenomes within these focus areas.
The EMCs together expect to provide 200 reference epigenomes, completed to IHEC standards, over the next 5 years (Tables 2 and 3) [6].
Title: The standard and definition of IHEC epigenome project

Title: Tissue and cell standard

The normal phenotype of a sample should be macroscopically and microscopically confirmed by at least two pathologists.
To monitor the cell quality, RNA quality should be assessed by RNA integrity number (RIN).
The value is expected to exceed 8 (cell lines) or 4 (cells isolated from primary tissues) and should be described as the sample information.
The purity of cells whose epigenome is analyzed must be assessed by immunocytochemistry, cell sorting, or other methods.
The cell purity is expected to exceed 90%.
However, if this is practically difficult, the purity of a sample should be described as the sample information.
For optimal ChIP results, it is recommended to use more than 1 × 106 cells per chromatin immunoprecipitation.
Title: Definition of IHEC epigenome set

The IHEC epigenome set consists of methylome (Bisulfite-seq [required], methylated DNA immunoprecipitation-seq [MeDIP-seq], methylation-sensitive restriction enzyme-seq [MRE-seq], reduced representation bisulphite-seq [RRBS], methylation capturing-seq [methylCap-seq] [any of 4 required on Roadmap]), RNA (RNA-seq, RNA-ChIP [required to Roadmap], smRNA-seq [optional]), ChIP-seq [input, H3K27me3, H3K36me3, H3K4me1, H3K4me3, H3K27ac, H3K9me3], and chromatin accessibility (DNaseI hypersensitivity, digital genomic footprinting, FAIRE-seq [any 3 optional]) (Table 3) [6].
Title: Definition of IHEC metadata

Metadata standards are the data-related information required for data integration produced from individual IHEC institutions.
It consists of 3 components: data standards, assay standards, and programmatic interoperability and programming interfaces, such as Representational State Transfer Application Programing Interfaces (REST APIs), allowing programs (apps) to pull data and metadata together from various sources across the web.
Metadata standards allow computer programs and end users to access essential information about performed experiments, such as the information about assays, samples assayed, and algorithms used to produce interpretable data.
In the preparatory stage, the project was to complete a case study in data and software integration using the Genboree system [6].
Several standards have yet to be defined, particularly controlled vocabularies and ontologies for important data elements, such as sample types, disease conditions, and phenotypes [6].
Title: Definition of IHEC data ecosystem standard

The IHEC data ecosystem project is to provide the infrastructure of data repositories and resources that enable data sharing by working together for the collection, processing, storage, and distribution of data from IHEC.
Data are normally submitted to the central data repository site by individual data producers.
Processed IHEC data, such as peak calls and epigenome maps, may not be deposited in a central data repository and may be only made available directly by the data distribution component (local) of the data ecosystem.
There is value in distributing both the raw and processed data, and both will be provided.
A virtual data integration center will be conceived in developing an IHEC data sharing system, and a joint project with a metadata standard project will be done in the near future [6, 7].
Although the international consortium is now in good shape, it still has several limitations.
As the IHEC aims at producing 1,000 epigenomes from ~250 human cell types, there is a limitation on tissue or cell availability.
Epigenome production from extremely hard-to-get cells or tissues seems impossible to accomplish and looks even unnecessary.
To overcome this limitation, there are some efforts to perform ChIP-seq with fewer than ~1,000 cells and to perform single-cell-level epigenome analysis [14].
Also, there is an individual variation issue: as epigenomes differ by individual, age, cell type, and developmental stage, it will be very hard to determine the real reference data among numerous epigenomic states.
It is required to determine the exact reference epigenome more precisely.
Moreover, overlapping target tissues among member institutions is a limitation.
Cell types, such as blood and adipocyte, that easy to get seem to be popular targets in every IHEC member and should be coordinated in a central way to foster synergy from each mapping activity.
Another issue is a limitation of international collaboration.
As all funds are controlled by an individual institution&#39;s decision, drawing a concrete conclusion on a specific point is expected to be a time-consuming process.
There is newly emerging technology on targeting several novel epigenome sets, such as carboxyl methylation [15], m6A RNA methylation [16], hydroxyl methylation [8], and 3D chromosomal interaction [17].
Also, collaboration with the ENCODE project on transcription factor binding site profiling should be actively considered in near future (Fig.1).
Furthermore, in the next phase of IHEC, disease epigenome should be considered.
Actually, all 7 member countries are focusing on disease relatedness actively, such as the metabolic epigenome, hematopoietic epigenome, and digestive organ disorders, but more active engagement and control should be anticipated in each project&#39;s detailed goals.
Extension to epigenome wide association study should also be considered to increase the clinical importance of IHEC&#39;s reference mapping efforts [9].
Perspectives of epigenome research.
MeDIP, methylated DNA immunoprecipitation-sequencing; MIRAseq, methylated CpG island recovery assay-sequencing; iPS, induced pluoripetent stem cell; mC-seq, methylated cytosine-sequencing; CHIA-PET, chromatin interaction analysis using paired end tags; T2D, type 2 diabetes; GWAS, genome-wide association study; ENCODE, Encyclopedia of DNA Element; EWAS, epigenomewide association study; hmC-seq, hydroxyl methyl cytosine-sequencing; IDH, isocitrate dehydrogenase; IHEC, International Human Epigenome Consortium.
Title: Major international efforts after the Human Genome Project (HGP) completion in 2003

Title: 1,000 Epigenome production scheme

Title: Definition of International Human Epigenome Consortium (IHEC) epigenome set
Title: Editor&#39;s Introduction to This Issue

As promised in last issue&#39;s editorial, we continue with the theme of epigenomics.
Last issue featured Dr. Jung Kyoon Choi&#39;s review on the Encyclopedia of DNA Elements (ENCODE) project, with a special emphasis on the chromatin environment in regulatory elements.
In this issue, Dr. Li Shen, Mt.
Sinai School of Medicine, USA, and colleagues complement it by reviewing the relationship between the transcriptome and chromatin modifications from the perspective of ENCODE.
We also feature a review on bacterial transcriptomics by Prof. Byung-Kwan Cho&#39;s group, KAIST, Korea.
They indicate that a large portion of a given bacterial genome is under transcriptional control.
One of the original articles by Drs.
Jong-Il Kim and Jeong-Sun Seo&#39;s group, Seoul National University, Korea, reports heritability indices of various facial features in Korean families, discovering strong genetic influences on facial anthropometric traits.
As the international visibility of Genomics &amp; Informatics has increased through its participation with PubMed Central (PMC) and PubMed, a broader readership naturally accompanies more critical post-publication reviews.
A North American reader spotted an uneven writing style in one of the articles published in the December 2012 issue.
Upon notification, we confirmed that its introduction section was indeed literally copied and pasted from earlier publications without quotations.
We asked the corresponding authors to clarify this issue, and they admitted to plagiarism in their article.
Since then, we immediately posted a retraction on our journal&#39;s website.
This issue, the first issue after this incident, publishes the retraction.
Genomics &amp; Informatics adheres to the guidelines per the Committee on Publication Ethics (http://publicationethics.org/).
We regret this misconduct.
Nevertheless, we appreciate your continued support.
Title: Human Transcriptome and Chromatin Modifications: An ENCODE Perspective

A decade-long project, led by several international research groups, called the Encyclopedia of DNA Elements (ENCODE), recently released an unprecedented amount of data.
The ambitious project covers transcriptome, cistrome, epigenome, and interactome data from more than 1,600 sets of experiments in human.
To make use of this valuable resource, it is important to understand the information it represents and the techniques that were used to generate these data.
In this review, we introduce the data that ENCODE generated, summarize the observations from the data analysis, and revisit a computational approach that ENCODE used to predict gene expression, with a focus on the human transcriptome and its association with chromatin modifications.
In September 2012, 30 research papers, including 6 in Nature, were published online (http://www.nature.com/encode/) about genomescale data from a decade-long project, the Encyclopedia of DNA Elements (ENCODE) [1].
Aiming to delineate all functional elements encoded in the human genome [1-4], the ENCODE project examined 1% of the human genome in its pilot phase and scaled up to the whole genome in its second phase.
It released data from more than 1,600 sets of experiments from 147 types of tissues [4].
These data include a catalog of human protein-coding and noncoding RNAs as well as protein-DNA interactions, chromatin and DNA accessibility, histone modifications, DNA methylation, and long-range chromosomal interactions.
The ENCODE consortium reported that 80.4% of the human genome serves some type of known biochemical function [4].
The unprecedented volume and span of this study will make it an excellent resource for biological analyses.
On the other hand, it can be overwhelming for a researcher to access and interpret the data.
ENCODE also developed novel (or refined existing) techniques for data generation and computational analyses.
These techniques are by no means restricted to ENCODE and can be employed in a wide array of applications.
Because of the vast span of ENCODE, we will limit our scope to the human transcriptome and its association with chromatin modifications in this review.
We aim to introduce the data that ENCODE generated and the techniques (both experimental and computational) that were used to generate them.
We summarize the observations that ENCODE found.
We also provide a detailed discussion of a machine learning method that was used to predict gene expression from chromatin modifications with higher accuracy than its predecessors.
Title: ENCODE releases a reference gene set (GENCODE) and RNA expression catalogs

The ENCODE project has produced a reference gene set, referred to as GENCODE (http://www.gencodegenes.org) [18].
GENCODE (version 7) identified a comprehensive set of 20,687 protein-coding and 9,640 manually curated long noncoding RNA (lncRNA) loci (representing 15,512 noncoding transcripts/isoforms).
Currently, in version 15, it has 20,447 coding genes and 13,249 lncRNA loci (representing 22,531 noncoding transcripts/isoforms).
For this reference gene set, GENCODE used manual gene annotation from the Human and Vertebrate Analysis and Annotation (HAVANA) group (http://www.sanger.ac.uk/research/projects/vertebrategenome/havana/) and automatic gene annotation from Ensembl [19].
To construct a comprehensive RNA expression catalog, ENCODE sequenced RNA (using RNA-seq) from 15 different cell lines in multiple subcellular fractions [18].
The expression catalog in multiple cell types provides the transcriptome of coding and noncoding genes, short (&lt;200 bp) and long (&gt;200 bp) in length, polyadenylated or nonpolyadenylated, as well as the compartment in cells where RNAs in each type are populated.
Title: Long noncoding RNAs

LncRNAs are nonprotein-coding transcripts longer than 200 nucleotides.
LncRNAs have been known to have similar characteristics as coding RNAs.
They are polyadenylated, associated with chromatin signatures, and have multiexonic structure [20, 21].
Some lncRNAs use identical or almost identical transcription initiation complexes [22] and sometimes overlap with protein-coding genes and can be transcribed from either strand [22, 23].
GENECODE interrogated the properties of 15,512 lncRNAs.
Table 2 summarizes the comparison between lncRNAs and coding RNAs.
Expressed lncRNAs have an activating histone modification profile similar to that of protein-coding genes, with slightly excess levels of both silencing (H3K27me3) and activating (H3K36me3) marks in lncRNAs [24].
Although many lncRNAs are polyadenylated, they are significantly enriched in nonpolyadenylated transcripts compared with coding RNAs [24].
As expected, the lncRNAs have significantly lower protein potentials compared with mRNAs.
Interestingly, they are biased toward two-exon transcripts and predominately localized in the chromatin and nucleus, while coding RNAs are predominantly observed in the cytosol; only 6% of them have a 2-exon structure.
In order to identify the function and targets of lncRNAs, Derrien et al.
[24] investigated the correlation between lncRNAs and coding RNAs; both positive and negative correlations were found.
Overall, the number of positive correlations was larger than the negative correlations.
Compared with trans-acting lncRNAs, cis-acting lncRNAs showed more positive correlations.
Interestingly, lncRNAs that intersected protein-coding exons in the antisense orientation showed a strong pattern of coexpression [24].
They also checked sequence conservation by BLASTing human lncRNAs against all available mammalian genomes.
Around 30% (n = 4,546) of lncRNAs appeared to have arisen in the primate lineage, and 0.7% (n = 101) of them appeared to be human-specific [24].
Title: RNA splicing

Recent studies suggest that many pre-mRNA processing events are cotranscriptional [25-27].
RNA imaging found that splicing can follow the completion of transcription [28].
Tilgner et al.
[29] investigated cotranscriptional splicing by interrogating RNA fractions from several cellular compartments in K562 cells.
They found that only a tiny fraction of exons were found to be surrounded completely by an unspliced intron in chromatin-associated RNA.
This suggests that splicing is already occurring during transcription [18, 29].
The strong enrichment of spliceosomal small nuclear RNAs (snRNAs) in the chromatin-associated fraction compared with other fractions also supports cotranscriptional splicing.
These observations confirm the idea that chromatin structure could play a role in splicing [7, 30-34].
However, for alternative exons and lncRNAs, splicing tends to occur later and might remain unspliced in some cases [29].
Title: RNA editing

Li et al.
[35] reported 10,210 exonic sites in the human genome where an RNA sequence did not match with the DNA sequence, suggestive of RNA editing.
However, there have been debates about whether their observations occurred by sequencing error, gene duplication, mapping error, or read-end misalignment [36-38].
Park et al.
[39] developed a pipeline to filter sequencing artifacts in identifying RNA editing.
They found that the majority of non-A-to-G variants came from incorrect read mapping across splice junctions.
Most of the edits they found were A-to-G(I) variants, which corresponds with recent observations [40] but differs from Li et al.
&#39;s report [35] of a substantial number of noncanonical single nucleotide variant edits in the RNA of human lymphoblastoid cells.
Most A-to-G(I) edits were located in introns and untranslated regions, with only a fraction of sites reproducibly edited across multiple cell lines.
Title: Pseudogenes

Pseudogenes have been considered nonfunctional sequences of genomic DNA that lost their coding potential due to disruptive mutations, such as frameshifts and premature stop codons [41-44].
Recent studies have shown that pseudogenes can regulate their parent genes [45-49].
Using manual annotation, with the assistance of computational pipelines, GENCODE created a database called Pseudogene Decoration Resource (psiDR).
psiDR provides a variety of information on 11,216 peudogenes, including transcription activity, chromatin features, functional genomics, and evolutionary constraints [50].
Transcribed pseudogenes show enhanced chromatin accessibility and enrichment with histone marks, although they are lower than those of coding genes.
The majority of pseudogenes contains no or very few transcription factor binding sites (TFBSs), but the differences between the number of TFBSs associated with transcribed and nontranscribed pseudogenes are significant.
Title: Small RNAs

GENCODE also categorized 7,054 small RNAs into 2,756 micro RNAs (miRNAs), snRNAs, small nucleolar RNAs (snoRNAs), and transfer RNAs (tRNAs).
They found that miRNAs and tRNAs were abundant in cytosol, snoRNAs were in the nucleus, and snRNAs were in both the nucleus and cytosol.
snRNAs were found abundantly in the chromatin-associated RNA fractions, which further supports predominant splicing during transcription.
Title: Enhancer RNAs

RNAs at enhancers (eRNAs) were first characterized by observing transcription activities at the promoter-distal CREB binding protein-binding sites in mouse cortical neurons [51].
The bidirectional property of eRNAs and their association with gene expression were further studied using nascent RNAs from global run-on sequencing (GROseq) data [52, 53].
ENCODE used RNA assays to detect transcription activity.
Besides the bidirectional property, ENCODE identified transcriptional initiation using cap analysis gene expression (CAGE) [54] signals.
Interestingly, they found polyadenylated eRNAs, although most eRNAs were prevalent in the nonpolyadenylated form.
They also observed that histone marks associated with eRNAs were the factors for transcriptional initiation and elongation: H3K27ac, H3K79me2, and RNA polymerase.
These lines of evidence suggest regulatory functions of eRNAs.
Enabled by the unprecedented volume of data generated by the ENCODE project, Dong et al.
[57] performed a very interesting study that attempted to predict gene expression from chromatin features using machine learning techniques.
As usually practiced, a machine learning study is composed of a series of procedures that typically involve data collection, data representation, model building, and testing.
Using machine learning terminology, the response variables here are gene expression patterns that are predicted/modeled, while the predictors or features are various chromatin measures.
Chromatin data were collected from the ENCODE project with 11 chromatin modifications, one histone variant, and DNase I hypersensitivity, all mapped in seven cell lines (Table 3).
Gene expression data were from different cellular compartments, using two different RNA isolation approaches and sequenced with different technologies (Table 3).
With such diversity of RNA sources, Dong et al.
[57] answered not only the general question of whether gene expression can be predicted with satisfactory accuracy but also the questions of whether different RNA sources that are sequenced by different technologies can be predicted differently using the same chromatin features.
The gene expression data are relatively easy to represent.
They were separated into two classes: transcriptional start site (TSS)-based and transcript-based (Tx-based).
TSS-based expression data are read counts within a 101-bp window centered on the TSS, which measures transcriptional initiation.
Tx-based expression data are summarized read counts from the whole transcript, which measures transcriptional elongation.
However, the representation of chromatin data seems to be tricky and requires further research.
Dong et al.
[57] used a strategy called &#34;bestbin,&#34; which considers the chromatin signals across the entire gene body, including 2-kb flanking regions.
It basically segregates each genic region into equal bins of 100 bp and summarizes the chromatin signals within each.
A training dataset was used to identify the bin that correlates most with gene expression, and the learned parameter values were applied to testing data.
Other strategies [58, 59] are possible, but the &#34;bestbin&#34; strategy was found to be superior [57].
RNA sequencing data are known to contain very little or no background noise, with a large portion of the genes having 0 read counts.
Therefore, the response variables become a mixture of a 0 component and a positive counting component.
Neither a classifier nor regression method seems to be able to capture the variability of both.
To deal with the challenge, a two-step approach was used by [57], so that a classifier first categorized genes as &#34;expressed&#34; or &#34;unexpressed,&#34; Then, a regression method was used to predict the expression levels of the expressed genes.
The final prediction is the product of the classifier and the regression method.
To test the performance of their approach, each dataset was separated into a training set and a testing set.
On the training set, the &#34;best bin&#34; and a few other parameters were determined.
After that, a 10-fold crossvalidation was performed on the testing set to evaluate the model.
AUC [60] was used to represent the accuracy of the classifier.
Two criteria were used to represent the accuracy of the regression method.
Pearson correlation coefficient (PCC) was used to measure the similarity between the predicted value and experimental value.
Root mean square error was used to measure the disparity between the predicted value and experimental value.
Overall, the two-step model achieved very satisfactory performance, with a PCC &gt; 0.9 for a number of datasets and a PCC &gt; 0.8 for 71% of the whole data.
Looking at the two steps separately, the AUC can be as high as 0.95 for the classifier, and the PCC can be as high as 0.77 for the regression method when predicting CAGE-measured polyA+ cytosolic RNA expression in K562 cells.
Similar performance was achieved in other datasets.
It was also found that H3K9ac and H3K4me3 are the most important predictors for the classifier, strengthening their roles as activation marks at TSSs.
In contrast, H3K79me2 and H3K36me3 are the most important predictors for the regression methods, strengthening their roles as elongation marks at gene bodies.
These findings show that the two-step model not only improved the accuracy of prediction but also enabled the identification of the chromatin features that are associated with different transcriptional roles.
During the past decade, the ENCODE project has evolved into a genomewide scale, and the dataset it generated has expanded in quantity as well as in scope.
The ENCODE project has provided a global view about the human transcriptome and most noticeably found that the transcribed region of the human genome is more abundant than we previously thought.
This finding significantly reduced the so-called intergenic regions, as defined in the traditional sense.
The quantitative measurement of RNA species in several cellular compartments as well as their polyadenylation provided a comprehensive view of RNA generation.
In this review, we have revisited the characteristics of both coding and noncoding transcripts in association with their structures and locations in cells.
Besides the human transcriptome and the associated chromatin modification data that we discussed, the ENCODE consortium also mapped transcription factor binding sites and their associated DNA motifs, as well as DNA methylation and long-range chromosomal interactions [4].
In parallel, the Roadmap Epigenomics Project (http://www.roadmapepigenomics.org) and International Human Epigenome Consortium (http://www.ihec-epigenomes.org) have been accumulating data of a similar scale to understand the human genome in other tissues and conditions.
It is remarkable that the ENCODE data altogether have associated more than 80% of the human genome with some type of biochemical function so far, and the coverage will continue to increase as we map additional protein-DNA interactions in the near future.
It has now become very clear that so-called &#34;junk DNA&#34; is not evolutionarily vestigial but has specific structural or biochemical functions.
While data generation has been a major goal of ENCODE, the need to integrate the current datasets is becoming more and more important.
Computational approaches have been developed to exploit the ENCODE data at to a fuller potential.
For instance, chromatin features were used to model gene expression [57, 61]; integrative methods were developed to annotate genomes [62-64]; visualization tools were developed to investigate epigenomic regulation at a global scale (also see ngs.plot at https://code.google.com/p/ngsplot/) [65, 66]; and large regulatory networks were reconstructed, based on TFs and DNaseI footprinting [67, 68].
The network-based approach as well as the chromosomal interactions [69] provided novel angles in studying gene regulation at higher levels.
New approaches to integrate the large amount of data to provide new biological insights are on the horizon.
Title: Quality metrics of chromatin modification ChIP-seq data employed by ENCODE

Title: Comparison of lncRNAs and coding RNAs

Title: Summary of data collected for modeling gene expression using chromatin features
Title: Recent Progress of Genome Study for Anaplastic Thyroid Cancer

Anaplastic thyroid cancer (ATC) belongs to the most malignant and rapidly progressive human thyroid cancers and its prognosis is very poor.
Also, it shows high resistance to cancer treatments, so that effective treatment for ATC has not been found to date, and virtually all patients terminate their life rapidly after diagnosis.
Although targeted treatment of genetic alterations has emerged as an extremely promising approach to human cancers, such as BRAF in metastatic melanoma, it remains unclear that how commonly genomic alterations are influenced in ATC tumorigenesis.
In recent years, genome wide approaches have been exploited to find genetic alterations associated with complex diseases, including cancer.
Here, we reviewed the comprehensive genetic alterations in ATC and recent approaches in the context of identifying genomic alterations associated with ATC.
Since surprisingly few reports have been published on the genome wide study of ATC, this review puts emphasis on the urgent needs of genomic research for the prevention and treatment of ATC.
Thyroid cancer is the fastest growing cancer worldwide [1].
Especially, in Korea, the average annual incidence rates of thyroid cancer in 2010 was 17.8% recording the highest increasing rate among all cancers (National Cancer Information Center; http://www.cancer.go.kr/).
Thyroid carcinomas can be sub-classified according to their histological and clinical features such as cellular origins or prognosis [2].
The most common subtype of thyroid cancers are papillary thyroid cancer (PTC; 80% to 85%) and follicular thyroid cancer (FTC; 10% to 25%), which are differentiated tumors derived from follicular epithelial cells.
Poorly differentiated thyroid cancer (PDTC; 5% to 10%) and anaplastic thyroid cancer (ATC; 2% to 3%) are also derived from follicular cells.
Parafollicular C cell-derived medullary thyroid cancer (MTC) comprises 2-3% of all thyroid cancers [2, 3].
The overall survival rates for PTC (98%), FTC (92%), and MTC (80%) are high [4], and these types have a more favorable prognosis by multimodality treatment.
However, ATC has an almost typically rapid and lethal progression, and the survival rate for ATC is only 13% [4] or 1-17% [2] with a mean survival of 6 months after diagnosis [5].
Although ATC supposed to be a follicular-cell-derived cancer, and represents a terminal stage in the dedifferentiation of PTC or FTC [6], the molecular mechanisms underlying ATC are not clear.
Because ATC is rare disease, it has been difficult to obtain sufficient samples to study the mechanism of the tumorigenesis associated with ATC which should have great impact to survival and treatment of patients [7].
Here, we reviewed the recent progress of the identification of genetic and epigenetic changes in ATC, focusing especially on genomic study.
By this effort, we may clarify the further substances of genomic research to conquer the most aggressive type of thyroid carcinomas.
To date, a considerable number of genetic mutations have been identified that play roles in the tumorigenesis of ATC, mainly by traditional approaches (Table 1) [8-13].
The most well-known mutation of thyroid cancer is BRAFV600E, occurring in approximately 26% of ATCs and 45% of PTCs [8, 14].
The BRAFV600E mutant protein is produced by the point mutation of T1799A coding sequence, and as consequence the BRAF kinase shows constitutive activity [15-18].
Its oncogenic and transforming capability has been well studies [19] and interestingly, it is strongly associated with poor prognosis of PTC, including aggressive pathological characteristics, increased recurrence rates, and failure of treatment [20, 21].
The tumor suppressor gene p53 is commonly mutated in ATC [8] but uncommon in well-differentiated PTC and FTC.
For example, Donghi et al.
[22] reported that p53 was mutated in approximately 71% of undifferentiated thyroid cancers (UDTC).
Also Fagin et al.
[23] reported 83% mutation rate of p53 in ATCs, and Quiros et al.
[24] found 88% of p53 mutation in ATC patients using immunohistochemical analysis.
RAS mutations were detected rarely in PTC, but more prevalently in FTC and ATC [25], as if there are relationship between RAS mutations and the aggressiveness of tumor behavior and/or poor prognosis [26, 27].
From three members of RAS family (HRAS, KRAS, and NRAS), codon 12/13 of KRAS mutation is known to be associated with poor survival for more aggressive subtype of thyroid tumors, including ATC [26].
RAS is known as an important regulator of cell growth, differentiation and survival of thyroid cancer [28].
In tumorigenesis, RAS may mediate multiple transduction signaling pathways such as Raf-MAPK and phosphatidylinositol 3-kinase (PI3K)/Akt pathways [29].
Mutations of PIK3CA are also common in ATC [8].
PIK3CA encodes the p110α catalytic subunit of PI3K and PI3K, in turns, involves in the PTEN/PI3K/AKT pathway, which regulates several cellular processes, including cell cycle progression, adhesion and motility.
Especially, this pathway linked to the cell survival, angiogenesis and glucose homeostasis which are important for tumorigenesis [30].
Garcia-Rostan et al.
[30] showed that the missense mutations of PIK3CA are higher in ATC than in others such as well-differentiated thyroid cancer.
Additionally, ATC patients who had mutated PIK3CA showed also Ras or BRAF alterations.
Garcia-Rostan et al.
[31, 32] demonstrated the significant association of mutations in β-catenin and the change of its expression with poor prognosis of ATC.
They found that activating mutations of CTNNB1, β-catenin encoding gene, often cause the activation of WNT-β-catenin signaling.
In general, as an ubiquitously expressed cytoplasmic protein, β-catenin is a downstream signaling molecule in Wnt pathway, and known to have a crucial role in cell-cell adhesion mediated by E-cadherin.
Interestingly, when it is upregulated, the nuclear translocation is occurred so that the transcription of various tumor-promoting genes could be triggered.
Axin 1 is a scaffold protein and located in chromosome 16p13.3.
It is supposed to be a tumor suppressor and play a role also in the Wnt pathway.
Kurihara et al.
[33] found the mutation of Axin 1 in 81.8% of 22 Japanese ATC samples.
Together, the genetic alterations of genes in the Wnt signaling pathway are found frequently with ATC suggesting the important role of Wnt pathway for ATC tumorigenesis.
Although the mutation prevalence is not high, mutations in the gene for isocitrate dehydrogenase 1 (IDH1) have been recently identified in ATC [9, 34].
The overall prevalence of mutations of IDH1 was reported as 11% in 18 ATC samples by Murugan et al.[9].
Murugan and Xing [10] reported two novel point mutations in the anaplastic lymphoma kinase (ALK) gene, C3592T and G3602A, with a prevalence of 25% in 8 ATC tumor samples.
ALK, one of the receptor tyrosine kinases, is a member of insulin receptor subfamily and these transverse point mutations caused the amino acid changes L1198F and G1201E, followed by increase of tyrosine kinase activities.
Initially, ALK was identified as an oncogeneic fusion gene NPM1-ALK in anaplastic large-cell non-Hodgkin lymphoma [35].
Further, fusion gene EML4-ALK was found in non-small-cell lung cancer [36] and TMP3/4-ALK and RANBP2-ALK in inflammatory myofibroblastic tumors [37].
They were aberrantly activated and promoted cell proliferation and survival [38, 39].
Although these mutations have been reported to date, it is not clear what of those are the driver mutations of ATC development or if there is any.
Also, many other mutations and genes might be associated with ATC tumorigenesis, which may contribute a lot to the diagnosis and treatment of ATC.
Therefore, it is a natural consequence that scientists try to adapt genomic approaches to analyze the mechanism of ATC in very recent years.
Hemmer et al.
[12] studied DNA copy number changes by comparative genomic hybridization (CGH) in 69 primary thyroid carcinomas including 13 ATC patients.
From those 11 ATC cases, showed chromosomal changes (10 gains, 3 deletions, both in 2 cases).
Regions of frequent gains were mainly located in 7p22-pter, 8q22-qter, and 9q34-qter regions in 31%, 23% and 23% of ATC cases, respectively.
The known proto-oncogenes such as PDGR-α, myc and ABL1, and VAVZ are located, in these regions but still no functional evidences were found supporting the association of copy number changes of these regions and the aggressiveness of ATC.
Also, Rodrigues et al.
[40] examined 39 thyroid cancer patients including 7 ATCs.
All cases of ATC presented chromosomal imbalances, especially, they found gains at chromosome 20p and 20q, and losses of Xp in 6 cases of ATC, and gains at 3q and 5p in 5 cases ATC.
Some important genes included PI3K, BCL6, and SST at 3q; invasion and metastases genes (MMP9 and MMP24), a cell cycle and mitotic checkpoint gene (STK15) at 20q.
The loss of tumor suppressor gene related 7q31 was detected.
The chromosomal instability of ATC revealed as significantly higher than well-differentiated thyroid cancer (WDTC) or PDTC when compared the CGH-detectable abnormalities as reported by couple of studies [12, 40].
Kitamura et al.
[41] detected allelic losses using 21 ATC patient samples in 1q (40%), 9p (58%), 11p (33%), 11q (33%), 17p (44%), 17q (43%), 19p (36%), and 22q (38%).
Wreesmann et al.
[13] reported the unique gains at chromosome 3p13-14 and 11q13 (where the cyclin D1 and FOSL1 located) and losses at 5q11-31 in ATC patients.
Liu et al.
[11] found frequent copy number gains in many receptor tyrosine kinase (RTK) genes containing the EGFR, PDGFRα, PDGFRβ, VEGFR1, VEGFR2, KIT, MET, PIK3Ca, PIK3Cb, and PDK1 genes.
Many of the genes with copy number gains are known to be proto-oncogenes so that they may be play a important role in ATC tumorigenesis by increasing the protein level followed by aberrant activation of the oncogenic pathway.
The prevalence of copy number gains was generally higher in ATC than differentiated thyroid cancer [42], also as mentioned in other studies.
It is supposed that the copy number variations, which could be occurred by chromosomal amplification or instability, should be important and involved to the progression and aggressiveness of ATC [43].
Chromosomal alteration could cause the genomic rearrangements and fusion genes.
In PTC, rearrangement causing RET/PTC fusion gene was observed often, but it rarely occur in benign follicular thyroid adenoma and follicular variant PTC [44].
Liu et al.
[11] reported the same RET/PTC rearrangement in 3 cases of ATC tissues, but because some ATC tissues contained PTC tissues, they were suspicious to their result.
Therefore, if chromosomal rearrangement and fusion genes, including RET/PTC fusion, involve to the ATC progression, should be investigated further.
Although not many investigations for genome wide expression analysis were performed, several studies examined altered expression of genes in ATC cell lines or patient samples using microarray (Table 2) [45-50].
The upand downregulated genes in ATC identified as players in the regulation of important cellular functions which may influencing to the tumorigenesis.
Onda et al.
[45] found 31 over- and 56 under-expressed genes using 11 ATC cell lines compared with normal thyroid gland; then they validated these data with 10 ATC patient samples, among them, DSTN, HSPA8, Stathmin, LDH-A, ATP5A1, PSM B6, B23, HDP-1, and LDH-B were significantly upregulated in primary ATC patient samples.
On the other hand, TG, PBP, and FES were downregulated in primary ATC tissues compared with normal thyroid glands.
In other study of ATC cell lines, Akaishi et al.
[50] identified transmembrane protein 34 (TMEM34) as specifically downregulated gene.
Although it is not a microarray analysis, Passon et al.
[46] measured mRNA levels for CREM, PAX8, and NIS in 6 normal thyroid tissues, 22 PTCs, 12 FTCs, and 4 ATCs using quantitative polymerase chain reaction analysis.
Compared to normal tissues, NIS and PAX8 mRNA levels were significantly reduced in all types of thyroid cancer, especially in ATCs.
Conversely, CREM mRNA levels were increased in all types of thyroid cancer including ATC.
To find molecular profile related to poor prognosis of thyroid cancer, Montero-Conde et al.
[47] examined 7 ATC, 6 PDTC, 24 PTC and 7 FTC tissues by microarray.
They comprised 13 dedifferentiated thyroid cancer (ATC and PDTC) vs. 31 WDTC (PTC and FTC) tissues.
In this data, they found that 1,031 genes showed more than 2-fold changes of its expression in absolute values.
Interestingly, pathways which are known to associated with tumorigenesis were preferentially altered in ATC and PDTC tissues; the MAP kinase signaling pathway, the transforming growth factor-β signaling pathway, focal adhesion and cell motility, activation of actin polymerization, and cell cycle.
UBE2C, S100A4, and STK5 were significantly overexpressed in ATC and PDTC than WDTC.
CDH1, TG, and PAX8 were significantly underexpressed in ATC and PDTC than WDTC, of those the down regulation of TG and PAX8 were also reported by Onda et al.
[45] and Passon et al.[46].
In other study, Pallante et al.
[48], reported downregulation of CBX7 expression using microarray analysis in various thyroid carcinoma-derived cell lines.
Then, they validated this finding in various thyroid tissue samples.
They show that CBX7 expression decreased with malignancy grade cancer tissues than in normal tissues.
Interestingly, in same study, CBX7 loss of heterozygosity occurred in 36.8% (7 of 19 cases) of PTCs and in 68.7% (11 of 16 cases) of ATCs.
mRNA for CDKN2A/p16 which is the molecular target of CBX7 and known to be a tumor suppressor, was also highly expressed.
As direct study of patient samples, Salvatore et al.
[49] did a microarray analysis using 5 ATC compared with 10 PTC and 4 normal thyroid tissue and validated their data with further 22 ATC, 39 PTC and 19 normal thyroid samples.
914 genes distinguished ATC from normal tissue: and only 114 genes were found to discriminated normal thyroid from both PTC and ATC.
Especially, 54 genes were upregulated in ATC and they are largely involved in cell cycle progression or chromosome instability.
Among them, TOP2A, CDC20, CENPA, NEK2, PRC1, CCNB1, CDCA8, TTK, CENPF, and PLK1 expression was ATC-specifically up-regulated.
Taken together, the previous studies of genomic expression analysis suggested some of interesting genes and molecular changes using microarray, they could not find clearly the major driver genes or pathways for ATC tumorigenesis.
This may be due to the hardness of sample collection or ATC itself could have a very complex heterogeneity.
But the technical limitation of microarray itself may contribute also largely.
The recently applicable next generation sequencing technology could help to solve this issue so that the molecular contribution of gene expression changes for ATC could be resolved.
Since the epigenetic changes of tumor-associated genes are known to play important role and have physiological function in the development of various human cancers and its progression [51, 52], they may also influence to the tumorigenesis of the most malignant form of thyroid cancer, ATC.
In PTC cells, genome-wide hypermethylation was found by a recent study using DNA methylation microarray which is revealed to be driven by BRAFV600E signaling [53].
Other couple of papers focused more to the change of methylation pattern in ATC.
Schagdarsurengin et al.
[54] analyzed the methylation pattern of 17 gene promoters in ATC, included those involved in DNA damage avoidance and repair (MGMT, GSTP1), apoptosis (DAPK, UCHL1), cell cycle control (p16, RB), tumor invasion and metastasis (TIMP3, CD26), hormone response (RARβ, ERα, ERβ), signal transduction (PTEN, RASSF1A), regulation of thyrocyte function and growth (TSHR), and other tumor-associated genes (DLC1, SLC5A8).
Among these genes, they found that TSHR, MGMT, UCHL1, p16, and DAPK were frequently methylated in ATC.
PTEN, as a well-established tumor suppressor, is involved in the PI3K/AKT pathway.
Hou et al.
[55] found that PTEN methylation is necessary to progress the malignancy from benign thyroid adenoma to FTC and aggressive ATC.
The methylation of a gene is usually associated with its silencing and the increased methylation of PTEN in ATC is in concordance with decreased or even lack of PTEN expression in previous reports [56, 57].
They also found that PTEN methylation in thyroid tumors was closely associated with the activating genetic alterations in the PI3K/Akt pathway, including mutations in Ras, PIK3CA, PTEN gene and PIK3CA amplification [55].
Four microRNAs (miRs) (-30d, -125b, -26a, and 30a-5p) were reported by Visone et al.
[58] as decreased in ATC compared with normal thyroid tissue.
From these, miR-26a and -125b may be targeting HMGA1 and HMGA2 which are known to be involved in thyroid cell transformation [59].
In other report by Mitomo et al.
[60], found the upregulation of miRs-21, -146b, -221, and -222 and downregulation of -26a, -138, -219, and -345 in thyroid cancer.
From these, miR-21 targets E2F (involved in cell cycle and apoptosis) and inhibits PTEN.
Interestingly, miR-138 which was found to be downregulated in both study of Visonet&#39;s and Mitomo&#39;s group [58, 60], targets hTERT which is also found to be reduced.
Takakura et al.
[61] identified ATC specifically deregulated miRNAs in ATC cell line.
They found that overexpression of miR-17-92 cluster of 7 miRNAs (miR-17-3p, -17-5p, -18a, -19a, -19b, -20a, and -92-1), which has a crucial role in cell growth and survival.
Additionally, miR-17-3p and -17-5p 5p were overexpressed in 3 out of the 6 ATC patient samples.
Although ATC is known to be one of the most aggressive tumor types, there has been little progress in the diagnosis and treatment, largely due to the shortage of genetic information associated with the molecular mechanism of ATC tumorigenesis.
In this review, we discussed genetic alterations, structural variations, and epigenetic changes of ATC performed by many scientists to date.
As shown, only limited information is available, and few studies were performed using genome-based approaches.
Generally, the quantities and qualities of samples for genomic study are very high, so that it is not easy to fulfill the requirements.
Therefore, collecting a sufficient number of samples for ATC matching these criteria is a kind of challenge for many scientists.
However, current next-generation sequencing techniques require smaller amounts of samples and are applicable for less-qualified samples but provide more accurate results for researchers to evaluate diverse genetic alterations.
Recently, the technical advances of genome study developed explosively so that it could provide more comprehensive information on cancer genomics, especially for gene expression, sequence variations, copy number changes, and epigenetic changes [62].
Using such technology such as next generation sequencing and big data, rapid progress has been realized in many parts of cancer molecular biology, which in turn contributes to the development of early diagnosis tools and/or targeted molecular therapies.
Thus, less-detailed cancer types have been intensively targeted by current genomic studies.
For example, many studies have analyzed the genomic alterations associated with lung cancer and reported detailed molecular events that could be used directly in the clinic, such as selection of the most effective drug for each patient.
For thyroid cancer, The Cancer Genome Atlas (TCGA) has deposited the analysis results of -500 thyroid cancer genomes in their public website, which may be published very soon.
These analyses will provide a more detailed, multidimensional genomic characterization of thyroid cancer genomes.
However, most of the 500 thyroid cancer genomes in TCGA were only from PTC patients, and none was from an ATC patient.
It is clear that genomic study of ATC is urgently needed so that the molecular mechanisms of ATC and the genetic and epigenetic changes in ATC could be more preciously understand.
Studies on ATC-associated genetic alterations, such as mutations and aberrant structural and epigenetic changes, based on genomic approaches should help to elucidate the molecular consequences of the activation of oncogenes and the loss of tumor suppressors to improve our understanding of tumor biology in ATC.
Especially, using recently developmed NGS technology, we will get a deep understanding of ATC-specific molecular mechanisms for successful diagnosis and treatment of this aggressive cancerand further facilitate the search for prognostic and predictive markers and new therapeutic targets for personalized medicine.
Title: High prevalence of genomic alterations in ATC

Title: Known expression changes in ATC
Title: Current Challenges in Bacterial Transcriptomics

Over the past decade or so, dramatic developments in our ability to experimentally determine the content and function of genomes have taken place.
In particular, next-generation sequencing technologies are now inspiring a new understanding of bacterial transcriptomes on a global scale.
In bacterial cells, whole-transcriptome studies have not received attention, owing to the general view that bacterial genomes are simple.
However, several recent RNA sequencing results are revealing unexpected levels of complexity in bacterial transcriptomes, indicating that the transcribed regions of genomes are much larger and complex than previously anticipated.
In particular, these data show a wide array of small RNAs, antisense RNAs, and alternative transcripts.
Here, we review how current transcriptomics are now revolutionizing our understanding of the complexity and regulation of bacterial transcriptomes.
An operon, which is an independent unit of biochemical function in the bacterial genome, consists of functionally related genes under the control of a single promoter [1].
Thus, the cluster of genes in an operon is transcribed into a single mRNA molecule, referred to as a polycistronic system.
This foundational concept of bacterial genome organization has been leading to the discovery of fundamental regulatory mechanisms of bacterial transcription.
Bacterial transcription is a biological process that is initiated from a promoter by the RNA polymerase (RNAP) holoenzyme complex, consisting of a single RNAP core unit and a σ-factor subunit.
Association with different σ-factor subunits provides promoter specificity to the RNAP core unit.
Seven species of σ-factor subunits, for example, have been identified in Escherichia coli, and each participates in the transcription of a specific set of genes in response to growth conditions [2].
In addition to the σ-factor association, transcription is either activated or repressed by a wide range of transcription factors, such as the Lac repressor of the lac operon.
It was estimated that E. coli encodes roughly 300 regulatory DNA-binding proteins, of which 35% are activators, 43% are repressors, and 22% are dual regulators [3].
Bacterial gene expression can be regulated by other regulatory factors, in addition to regulatory DNA-binding proteins, such as small regulatory RNAs and internal promoters within operons, which increase the transcriptome complexity [4, 5].
Over the last decade or so, many of these examples have been discovered by quantitative analysis of bacterial transcriptomes, which are defined as the complete set of cellular RNA transcripts.
Although DNA microarrays have provided comprehensive information on the transcriptome&#39;s complexity in bacterial cells [6, 7], the advent of next-generation sequencing (NGS) has dramatically accelerated our analytical capacity via high-throughput transcriptome sequencing (RNA-seq) in combination with mRNA enrichment methods [8].
This large amount of data has revealed many unexpected features in the bacterial transcriptome, including gene structures, multiple promoters, and RNA-based regulation [7, 9, 10].
In some cases, small RNAs (sRNAs) account for 10% to 20% of bacterial RNA products, which have important regulatory roles [8].
These new findings suggest that bacterial transcription is much more complicated and subtle than previously thought.
In this review, we discuss advances in bacterial transcriptomics, based on RNA-seq, and the current understanding of the complexity and regulation of bacterial transcriptomes.
Like eukaryotes, bacterial transcriptome analyses have been initiated by the development of microarrays [11].
In particular, high-density tiling arrays, which consist of hundreds of thousands of DNA oligonucleotide probes representing both strands of a given genome, have been massively used to study bacterial transcriptomes.
The major advantage of tiling arrays is to interrogate the boundaries of the entire set of transcripts in a cell without taking account of genome annotation [12].
Consequently, this technical advance discovered many new RNA molecules, such as regulatory sRNAs.
Based on this experimental approach, comprehensive transcriptomics have recently been published for Bacillus subtilis [13], Halobacterium salinarum [14], Mycoplasma pneumonia [6], Caulobacter crescentus [15], Listeria monocytogenes [16], and E. coli [10].
One of these studies, for example, focused on the analysis of transcriptionally active regions across an entire genome of Listeria monocytogenes, revealing the complete Listeria operon map as well as various types of RNAs, including at least two of which are involved in virulence in mice [16].
Although the high-density tiling arrays were able to interrogate the bacterial transcriptomes without genome annotation, the array-based approach is limited by a high rate of noise due to cross-hybridization, the low dynamic range of detection due to signal saturation, and the inability to detect low copy number of transcripts [17].
High-throughput sequencing of RNAs using NGS methods has overcome some drawbacks of the array-based approach.
Currently, commercially available NGS systems include the Roche 454 system, based on the pyrosequencing; the Illumina sequencing system, based on sequencing by synthesis; the SOLiD system, based on sequencing by oligonucleotide ligation and detection; and the Ion torrent system, based on the use of a semiconductor-based sequencing technique [18, 19].
Also, third-generation sequencing technologies have been actively developed to determine the sequence directly from a single DNA molecule without DNA library amplification [18].
Over the conventional Sanger sequencing method, the primary advantage offered by NGS methods is the inexpensive production of large volumes of sequence data, which can be used to identify and quantify rare transcripts without prior knowledge of a particular gene [19, 20].
Unlike a hybridization-based array approach, RNA-seq allows unambiguous mapping of transcripts to unique regions of the genome with single-base resolution; hence, there is essentially lower background noise [21].
In addition to the accurate quantification of a transcriptome consisting of known genes, RNA-seq allows researchers to determine the correct gene annotation, expressed single-nucleotide polymorphisms, novel genes, and RNAs with high levels of reproducibility for both technical and biological replicates.
However, bacterial mRNA can exist as little as 1% to 5% of total RNA; so, mRNA enrichment is a challenging step to obtain sufficient transcript coverage [8].
Currently, several methods are being used to remove the rRNA and tRNA fraction from the total RNA pool.
Among those methods, terminator 5&#39;-phosphate-dependent exonuclease treatment has been successfully applied to enrich primary transcriptome by reducing processed or degraded RNAs with a 5&#39;-monophosphate end (e.g., rRNAs and tRNAs) [9, 10].
To remove the rRNA fraction from the total RNA pool, Ribo-Zero rRNA removal kits, using biotinylated probes that selectively bind rRNA, have been used successfully for a wide range of organisms, from bacteria to human (Epicentre).
The requirements of the ideal RNA-seq method are strand-specificity and quantitation across a wide dynamic range [22, 23].
However, a common strategy of an RNA-seq library preparation is to convert single-stranded RNA molecules into double-stranded cDNA fragments of certain sizes, flanked by platform-specific adapter sequences.
This means that double-strand cDNA is synthesized from randomly primed hexamers, which cannot provide information on the transcribed strands.
A major shortcoming of conventional RNA-seq methods is therefore the lack of RNA polarity information.
The polarity of RNA transcripts is very useful in facilitating subsequent computational analyses, including the correction of novel gene annotations and the detection of overlapping genetic features encoded in opposite orientations.
In particular, polarity information is important for understanding small bacterial genomes, in which genes are densely coded, with overlapping untranslated regions (UTRs) or open reading frames (ORFs) [23].
However, the recently developed strand-specific RNA-seq analysis enhances the value of bacterial transcriptomics.
In terms of how to mark the transcribed strand, several methods can be categorized largely into two approaches.
One is marking a strand by orientation-dependent adaptor ligation to the 5&#39; and 3&#39; ends of the RNA transcript, such as RNA ligation [24], SMART [25], NNSR priming [26], and SMART-RNA ligation [23].
The other is marking one strand by chemical modification, either on the RNA itself by bisulfite treatment [27] or on the second strand cDNA by dUTP incorporation [28].
Among several strand-specific RNA-seq methods, the dUTP method was identified as the leading protocol because of its wide range of coverage and availability of paired-end sequencing (Fig.1A) [23].
Briefly, the dUTP method starts with converting the initial mRNA into cDNA.
This first cDNA strand is used as a template for second cDNA strand synthesis, which incorporates dUTP instead of dTTP.
After marking both ends of the cDNA by ligation of adaptor sequences, the second strand is subsequently degraded by uracil-N-glycosylase treatment, which selectively removes the uracil site on the DNA.
Finally, the remaining first-strand cDNA, with directional adaptor sequences, can be sequenced on the Illumina Platform using both single-read and paired-end sequencing [28].
With these technological breakthroughs, several bacterial RNA-seq studies showed that bacterial transcription is not as simple as previously thought [10, 29-32].
For example, in the transcriptomic analyses of Salmonella typhimurium, E. coli, Vibrio cholera, and Xanthomonas campestris pv.
vesicatoria, it was revealed that noncoding RNAs play an important role in diverse regulatory mechanisms [29-32].
In addition, the complexity of the E. coli transcription unit indicates that bacterial transcription is regulated by the differential use of diverse transcription start sites (TSSs), with corresponding transcription factors under given environmental conditions [10].
In bacteria, cellular RNA comprises primary transcripts that are marked by a 5&#39;-triphosphate (5&#39;PPP) group and processed or degraded RNAs that carry either a 5&#39;-monophosphate (5&#39;P) or 5&#39;-hydroxyl group (5&#39;OH) [7, 9].
In particular, primary transcriptome analysis is beneficial for improving the annotation of any sequenced bacterial genome.
Due to the absence of a poly(A) tail at the 3&#39;-end and the instability of mRNAs with very short half-lives, however, bacterial primary transcriptome analysis has been challenging [8].
Also, cellular RNA was considered to consist mainly of rRNA and tRNA in prokaryotic cells; therefore, mRNA enrichment is essential to determine the primary transcriptome from RNA-seq.
Differential RNA-seq (dRNA-seq) enriches RNA samples for primary transcripts by use of a 5&#39;P-dependent terminator exonuclease (TEX) that specifically degrades RNAs carrying a 5&#39;P, leading to relative enrichment of primary transcripts and depletion of processed RNAs.
Then, the 5&#39;-end positions of the primary transcripts can be identified by comparison of cDNA libraries generated with and without TEX treatment (Fig.1B) [9, 31, 33-36].
The 5&#39;-end of RNA transcripts, corresponding to the TSS, is mostly related with the regulation of transcribed genes, including promoters, ribosome binding sites, transcription factor binding sites, antisense RNAs (asRNAs), and other regulatory noncoding RNAs (ncRNAs).
Bacterial TSSs have been identified mainly through primer extension and S1 nuclease protection mapping assay, but these methods are labor-intensive (Fig.2A) [37].
High-density tiling arrays have also been used for large-scale TSS mapping in E. coli and Caulobacter crescentus, with limited mapping resolution [15, 38].
In addition, indirect TSS identification in the E. coli transcriptome has been made by RNAP chromatin immunoprecipitation, coupled with microarray (ChIP-chip) [39].
More recently, dRNA-seq has been developed to map TSSs in Helicobacter pylori [9].
It revealed 1907 TSSs, which were found in upstream regions of annotated mRNAs within annotated genes on the same DNA strand or on the opposite DNA strand and in regions without any annotated genes in proximity.
In addition to the 337 primary operons defined by the TSSs, it obtained 126 additional suboperons and 66 monocistrons overlapping the 3&#39; part of polycistrons in H. pylori.
Alternative TSSs for the determination of suboperons (or transcription units) have been determined through the integration of massive-scale sequencing and 5&#39;-rapid amplification of cDNA ends (5&#39;-RACE) method in E. coli (Fig.2B) [10].
The consecutive iterations of experiments identified a total of 4,133 TSSs.
Interestingly, it was found that ~35% of promoters contained multiple TSSs, defining alternative transcription units in the E. coli genome.
For example, the transcription factor Lrp activates stpA, encoding an H-NS-like DNA-binding protein from a dominant TSS position (2,796,558).
However, two other TSSs are used less in the exponential growth phase.
These results show that the bacterial transcriptome is actively regulated by means of alternative TSSs in response to differential environmental conditions.
dRNA-seq method has been applied to detect genomewide TSS maps in other bacterial species.
Interestingly, the cyanobacterium Synechocystis contains 3,537 TSSs, 64% of which were assigned to asRNAs or ncRNAs [40].
In conclusion, the dRNA-seq approach is a strong tool for annotation of operon structures, transcription units (or suboperons), and detection of TSSs, providing important insights into the understanding of a comprehensive transcription regulatory network.
In bacteria, regulatory ncRNAs have emerged as key players, acting by various mechanisms to modulate metabolic, physiological, and pathogenic processes, as well as bacterial adaptive immunity [41].
The largest group of regulatory ncRNAs consists of regulatory sRNAs, which affect the transcription, translation, and stability of target mRNAs through base pairing with the targets (Fig.2C).
These sRNAs can work in cis by overlapping their target genes encoded on the opposite DNA strand of the same genomic locus or in trans by targeting genes located elsewhere on the chromosome [41, 42].
Therefore, the discovery and characterization of bacterial sRNAs are critical for the understanding of the complexity of bacterial transcriptomes.
Primary transcriptome analysis allows one to detect the presence of sRNAs at their abundance in a genome-scale by measuring nonprotein-coding RNA transcripts.
For instance, an unexpected 60 sRNAs have been discovered in the primary transcriptome analysis of H. pylori, indicating that the complexity of gene expression in the small H. pylori genome is increased by genomewide antisense transcription [9].
In this way, many sRNAs were recently discovered from various bacterial species, including the opportunistic pathogen Burkholderia cenocepacia (13 sRNAs), Salmonella Typhi Ty2 (55 sRNAs), and L. monocytogenes (50 sRNAs).
The sRNAs from B. cenocepacia and L. monocytogenes were induced during niche switching and involved in virulence, respectively [16, 43, 44].
Furthermore, comparing of the sRNAs with global gene expression could be facilitated in the analysis of a set of genes that are mainly involved in the pathogenic features of B. cenocepacia.
In addition, immunoprecipitation of specific RNA-binding proteins can lead to selectively enriched sRNAs [8].
The first attempt at sRNA analysis, based on the NGS technique (RIP-seq), was carried out in Salmonella [29].
Using the RNA-binding property of Hfq proteins, Hfq-associated RNAs could be enriched and analyzed by immunoprecipitation of Hfq-RNA complexes, followed by deep sequencing.
Through this study, more than twice as many new sRNAs were discovered over previously known sRNAs [29].
These results indicate that sRNAs are abundant in the bacterial transcriptome, representing its regulatory complexity.
RNA-seq technology has uncovered that 5% to 25% of all protein-coding genes encoded on the sense strand interact with the asRNAs that are transcribed from the antisense strand, thereby regulating their transcription, translation, or degradation (Fig.2C) [8].
These transcripts are named asRNAs, which overlapped partially with the 5&#39;-UTR or 3&#39;-UTR of their target protein-coding genes [16].
The abundance of these transcripts has been revealed in several bacterial genomes, including Sallmonella Typhimurium [29], B. Subtilis [13], E. coli [45], Staphylococcus aureus [46], and M. pneumonia [6].
Using RNaseIII, which specifically degrades double-stranded short RNAs, for example, stable sRNA molecules that map symmetrically to both strands were enriched from Staphylococcus aureus.
RNA-seq of the enriched RNA molecules resulted in the detection of 1387 ORFs, which covers 50% of the antisense strand [46].
Another study identified ~1,000 novel asRNAs in E. coli using high-throughput sequencing [45].
Briefly, the unique 5&#39; end of the primary transcriptome was analyzed by treatment with tobacco acid pyrophosphatase to convert 5&#39;PPP groups into 5&#39;P, followed by ligation of an RNA oligonucleotide and reverse-transcription with a random primer.
Further studies of several asRNAs suggested that they are involved in a wide range of cellular mechanisms, such as the translational regulation of the quorum-sensing gene rpaR in Rhodopseudomonas palustris [47].
Based on the abundance of transcripts, it has been suggested that the number of asRNAs correlates negatively with genome size [7].
Large asRNAs could be the consequence of genome reduction with transcriptional regulation, mediated at the RNA level.
Taken together, RNA-seq approaches are revolutionizing our understanding of the abundance of asRNAs in bacteria [42].
With emerging advances in NGS, RNA-seq has uncovered the complexity of the bacterial transcriptome.
Although transcriptional regulation by asRNAs, novel ncRNAs, and alternative splicing events have been well known in eukaryotes, similar regulatory events in prokaryotes have been newly elucidated in recent studies.
In accordance with technological developments, several approaches for computational pipeline and bioinformatics analysis have been also progressed.
Here, we review sequencing methods, their advantages, and several key results for bacterial transcriptome analysis.
Undoubtedly, RNA-seq is having a breathtaking impact on our understanding of biological systems and will provide entirely new insights into the biological parts in living genomes.
Furthermore, the concept of artificial genetic circuits has been advanced in synthetic biology, due to their simplicity in manipulating the cell [48].
For example, promoters, sRNAs, ncRNAs, and ribosome-binding sites are considered as biological parts in a cell, which can be used to assemble new genetic circuits, such as switches or oscillators.
A wide range of biological parts can be screened and listed up from transcriptome analysis by NGS.
Nevertheless, many efforts to develop diverse experimental approaches using NGS methods will reveal the unexpected complexity of bacterial transcriptomes.
Methods for strand-specific RNA-seq.
(A) RNA-seq method using dUTP incorporation and uracil-N-glycosylase treatment.
(B) dRNA-seq method using 5&#39;-monophosphate (5&#39;P)-dependent terminator exonuclease (TEX) and terminal tobacco acid pyrophosphatase (TAP).
TSS, transcription start site.
Complexity of bacterial transcriptome.
(A) Operon structure of bacterial genome.
(B) Definition of transcription units using alternative transcription start sites.
(C) Various noncoding RNAs in regulation.
Title: Heritabilities of Facial Measurements and Their Latent Factors in Korean Families

Genetic studies on facial morphology targeting healthy populations are fundamental in understanding the specific genetic influences involved; yet, most studies to date, if not all, have been focused on congenital diseases accompanied by facial anomalies.
To study the specific genetic cues determining facial morphology, we estimated familial correlations and heritabilities of 14 facial measurements and 3 latent factors inferred from a factor analysis in a subset of the Korean population.
The study included a total of 229 individuals from 38 families.
We evaluated a total of 14 facial measurements using 2D digital photographs.
We performed factor analysis to infer common latent variables.
The heritabilities of 13 facial measurements were statistically significant (p &lt; 0.05) and ranged from 0.25 to 0.61.
Of these, the heritability of intercanthal width in the orbital region was found to be the highest (h2 = 0.61, SE = 0.14).
Three factors (lower face portion, orbital region, and vertical length) were obtained through factor analysis, where the heritability values ranged from 0.45 to 0.55.
The heritability values for each factor were higher than the mean heritability value of individual original measurements.
We have confirmed the genetic influence on facial anthropometric traits and suggest a potential way to categorize and analyze the facial portions into different groups.
In the context of bone morphology, the craniofacial region represents the most complex and special part of the body, consisting of 22 separate bones in addition to 22 deciduous and 32 permanent teeth [1].
Most skull bones are developed from neural crest cells, where the skull vault, mandible, and maxilla are formed from intramembranous ossification in the mesenchyme without cartilage.
On the other hand, bones from other parts of body are originated from mesoderm and are formed from endochondral ossification.
The human face is made from the fusion of 5 prominences (the fronto-nasal prominence and two pairs of maxillary and mandibular prominences), which occur from the 4th to 10th weeks of development, where defects in the process result in many kinds of facial clefts [2].
Studies on human facial morphology can impact various areas, including plastic and dental surgery, facial anatomy, reconstruction, and constitutional diagnosis [3, 4], as well as studies on anthropology and genetics.
Of these, the implication of genetic factors on facial morphology has been considered important in understanding hereditary disorders as well as tracing the anthropological origin [5, 6].
Genetic effects on facial measurements have been investigated considerably in the past.
Several twin studies evaluated the genetic and non-genetic influence on facial traits [7-10].
One classical twin study showed significant genetic variance of alveolar bone height (h2 = 0.36) using intraclass correlation between twins (MZr = 0.70 and DZr = 0.52) [7].
Another twin study found significant sex differences of genetic variance in anterior face height and no genetic influence on angular measurements [8].
On the other hand, several angle measurements, such as soft-tissue facial angle and Holdaway angle, were shown to have genetic effects [11] in a Turkish Anatolian siblings study.
Family studies, in addition to the twin studies, have reported several key findings regarding the genetic contribution of facial traits [12-17].
In 1991, Kohn [13] found an important role of genetic influence on craniofacial growth.
A family study based on ethnically homogeneous Chuvasha pedigrees has found that genetic factors substantially contribute to head shape, where the unadjusted estimates of heritability were found to range from 0.35 to 0.71 [14].
In a recent study using the Hallstatt skull, it was shown that the additive genetic variation accounted for approximately 30% of the phenotypic variation in total facial dimensions (maximum facial breadth, length, and height) [16].
Familial correlation can also be used as an important indicator to grasp genetic and environmental proportions.
A morphological characteristics study based on 125 Belgian families has found evidence of genetic determination in longitudinal body measurements and, to a lesser extent, in soft tissues and in nose or mouth regions [12].
There exist a variety of complex indicators for the measurement of facial size and length.
A common characterization of facial measurements by reducing the dimension using factor analysis can be very helpful in understanding the genetics behind facial morphology.
In recent years, most studies based on this concept assessed 2 common factors: horizontal components (HOCs) and vertical components (VECs).
Several investigations found that the genetic effect was estimated differently between 2 factors [8, 10, 14, 18, 19].
In these studies, however, the extraction of common latent variables in facial complex measurements was only limited to HOCs and VECs.
In understanding the implication of genetic influence of facial morphology, other characterization methods in addition to the horizontal and vertical component could also be useful.
Although many researchers have studied the genetic effects on facial morphology, most of them have focused on congenital diseases accompanied by facial anomalies.
Genetic studies on facial morphology targeting healthy population of various ethnicities are therefore still lagging behind.
The aims of this study were to infer common latent factors by analyzing the pattern found in a factor analysis of 14 facial measurements from the Korean population and investigate the existence of genetic effects for the factors through estimation of familial correlations and heritability.
Title: Subjects

The study was approved and conducted by the Korean Institute of Oriental Medicine (KIOM) from November 1, 2006 to July 31, 2008.
Three local clinical centers and 10 hospitals were recruited for this work.
Photographs for facial analysis were taken by trained investigators according to a standard protocol.
To ensure the quality of the photographs, we took 2 facial pictures from both the front and the lateral side of each subject and selected the better representative pictures for the analysis.
During the first and second phase of this survey, 114 and 164 individuals were collected, respectively.
Subjects with poor-quality photographs were excluded from analysis.
The study included a total of 229 individuals from 38 families, and of these, one family had a large extended pedigree.
All samples were of Korean origin.
Anyone who had a facial anomaly or surgery was excluded in this study.
The mean and standard deviation (SD) of parents&#39; age were 55.2 and 13.9, and those of the offspring&#39;s age were 36.0 and 17.4, respectively.
Title: Facial measurements

Digital photographs used for facial measurements have the advantages of obtaining quick and permanent data [20].
To minimize the potential variability in the measurements among different centers, a single digital camera, with a standard protocol, was employed throughout the collection of data.
To correct for distance variable, a standard sliding caliper was held by the subjects while taking photographs.
The resulting images were sent to KIOM from each center.
Twenty-five landmarks were selected and produced as described [21] using Image J program, an image processing program developed by the National Institutes of Health (NIH) (Fig.1).
Each of these landmarks was recorded as coordinates in this program.
A single measurement can be generated, representing the shortest distance between any of the 2 landmark coordinates of choice.
Fourteen of such measurements described below were selected, which best represented the facial characteristic [21].
A total of 14 facial measurements in 5 regions (head and face, orbital, nose, mouth, and ears) were evaluated.
For the head and face region, 6 measurements (physiognomical height of the face [tr-gn], height of the forehead I [tr-g], special upper face height [g-sn], height of the lower face [sn-gn], width of the face [zy-zy], and width of the mandible [go-go]) were recorded.
Three measurements were collected from the orbital region: intercanthal width (en-en), binocular width (ex-ex), and length of the eye fissure (en-ex).
The nose region had 3 measurements: height of the nose (n-sn), width of the nose (al-al), and nasal protrusion (sn-prn).
One measurement each was taken from the mouth and ear regions: width of the mouth (ch-ch) and length of the auricle (sa-sba) (Table 1, Fig.1).
To minimize intermeasurement bias, all the measurements were taken by a single, well-trained personnel.
To verify the robustness and reliability of the measurement method employed herein, we also recorded the intraobserver precision of all 14 measurements on the 20 subsamples, of which age and sex were taken into consideration.
The resulting mean intracorrelation coefficient was 0.93 (0.80-0.99), showing high reliability.
Title: Statistical analysis

For the calculation of the mean and SD of 14 craniofacial measurements, only individuals with age ≥ 20 were used to control growth effects.
In addition, all genetic analyses for estimating familial correlation and heritability included all age groups.
To estimate the common latent variable in facial measurements, we implemented factor analysis using SAS version 9.1 (SAS Inc., Cary, NC, USA).
The eigenvalues for 3 factors were greater than 1 (i.e., the cut-off of the eigenvalue), and the cumulative proportion was 79%.
Orthogonal Varimax rotation and principal method were employed.
The total number of factors used in this study was 3.
The cut-off value of factor patterns was above 0.6 [22, 23].
Before estimating the genetic effects, we calculated 3 factor scores for each individual and evaluated the normal distribution of facial measurements, including factor scores, by Quantile-Quantile (Q-Q) plot and Kolmogorov-Smirnov&#39;s value.
Most Q-Q plot lines were straight, and the measurements tended to follow a normal distribution (Kolmogorov-Smirnov&#39;s p &gt; 0.05).
PEDINFO option in S.A.G.E.
(2009; Statistical Analysis for Genetic Epidemiology, Release 6.0.1; http://darwin.cwru.edu/) was used to understand family information, such as offspring N and sibling N.
Because facial measurements depend on age, sex, and body mass index (BMI) of an individual, we used covariate variables, such as age, sex, age*sex, age2, age2*sex, and BMI.
In this modeling, the significant covariate variables were adjusted by screen option of the Sequential Oligogenic Linkage Analysis Routines (SOLAR) package [24].
As a result, the residual variation, which is the remainder of the variation among phenotypes, excluding the portion pertaining to be important covariates, was obtained and applied to approximate the familial correlation and heritability.
We used the FCOR option in S.A.G.E.
to estimate familial correlations.
Familial correlations were calculated by a set of two random variables and arbitrary weights {wi} [25].
Narrow-sense heritability (h2) (proportion of phenotype variance attributable to additive genetic variance) was also estimated by maximum likelihood procedure using SOLAR.
SOLAR employs a variance-components method, which has a long history in classical genetic study.
Additionally, a bivariate model was used to test the genetic correlation between pairs of facial measurements within each factor.
We estimated two parameters, based on maximum likelihood ratio method: additive genetic correlation (ρG) and environmental correlations (ρE).
Table 2 presents the sample and pedigree information.
The number of total samples used in this study is 229.
The portion of females (58.95%) was higher than males (41.05%).
In the pedigree information, the size of each pedigree ranged from 2 to 94.
One large extended family (n = 94) provided great statistical power to estimate the inheritance of facial measurements.
The number of parent-offspring and sibling pairs was 412 and 191, respectively.
Among sibling pairs, the number of brother-sister pairs was the largest (93 pairs).
The number of sister-sister and brother-brother pairs was 54 and 44, respectively.
Basic statistics for the 14 original measurements in males and females are summarized in Table 3.
Only those results with age ≥ 20 were considered, to account for the variable introduced by age.
The mean value for males (n = 79) was significantly higher than females (n = 120) in all measurements (p &lt; 0.05).
Familial correlations and heritabilities of residual variation, adjusted by significant covariates, are presented in Table 4.
The mean contribution of significant covariates was 0.27 and ranged from 0.10 to 0.51.
As shown, the spousal correlations (p ≥ 0.05), which represent the impacts of a common environment and partially assortative mating, were lower than the other correlations in most measurements.
Except for height of the lower face (sn-gn), width of the mandible (go-go), length of the eye fissure (en-ex), width of the mouth (ch-ch), and length of the auricle (sa-sba), the correlations between parent and offspring were statistically significant (p &lt; 0.05).
The correlations between siblings were statistically significant for width of the mandible (go-go), nasal protrusion (sn-prn), and length of the auricle (sa-sba).
The heritabilities of 13 facial measurements, except for width of the mouth (ch-ch), were statistically significant (p &lt; 0.05) and ranged from 0.25 to 0.61.
The heritability of intercanthal width (en-en) in the orbital region was found to be the highest (h2 = 0.61, SE = 0.14), whereas in the nose region, width of the nose (al-al) showed the highest heritability (h2 = 0.53, SE = 0.13).
Table 5 shows the pattern of rotated factors by factor analysis with 14 facial measurements.
We obtained 3 factors whose eigenvalue was greater than 1.
Factors were denoted according to loading scores of rotated factors above 0.6 [22, 23].
Factor 1 included 4 measurements: height of the lower face (sn-gn), width of the mandible (go-go), width of the nose (al-al), and width of the mouth (ch-ch).
Factor 2 included intercanthal width (en-en), biocular width (ex-ex), and length of the eye fissure (en-ex), which were related to the orbital region.
Factor 3 included physiognomical height of the face (tr-gn), special upper face height (g-sn), and height of the nose (n-sn).
The familial correlations and heritabilities of 3 factors are shown in Table 6.
In Factor 1, the correlation of spouse (p ≥ 0.05) was lower than that of parent-offspring and sibling pairs, whereas meaningful familial correlations were not found in Factor 2 or Factor 3.
The heritabilities of 3 factors were statistically significant (p &lt; 0.05) and ranged from 0.45 to 0.55.
Factor 1, which was related to the lower face, had the highest value for heritability (h2 = 0.55, SE = 0.12).
Using a bivariate model, we examined genetic correlations for all pairs within each factor (Table 7).
There were several pairs with a statistically significant genetic correlation (p ≤ 0.05): the pair of height of the lower face (sn-gn) and width of the nose (al-al) (ρG = 0.79, SE = 0.23) for Factor 1, the pair of intercanthal width (en-en) and biocular width (ex-ex) (ρG = 0.82, SE = 0.08) and the pair of biocular width (ex-ex) and length of the eye fissure (en-ex) (ρG = 0.81, SE = 0.09) for Factor 2, and the pair of physiognomical height of the face (tr-gn) and special upper face height (g-sn) (ρG = 0.74, SE = 0.17) and the pair of special upper face height (g-sn) and height of the nose (n-sn) (ρG = 0.98, SE = 0.06) for Factor 3.
There are several distinctive features of facial morphology found in Koreans compared to western populations.
First, the nose of Koreans tends to be relatively shorter and broader than that found in Italians [26], Caucasians [27], and Latvian [28] populations.
Koreans also have a relatively narrower face (zy-zy) and wider mandible (go-go), and eyes (en-ex) when compared to Italians and wider intercanthal width (en-en), narrower width of the mouth (ch-ch), and longer special upper face height (g-sn) and length of the auricle (sa-sba) than Caucasians.
Compared to Latvians, Koreans have broader (zy-zy and go-go) and longer (tr-gn) faces, narrower binocular width (ex-ex) and length of the eye fissure (en-ex), and wider intercanthal width (en-en).
The heritabilities that were significant for 13 phenotypes had a wide range of values (h2 = 0.25-0.61).
Previous papers have reported estimated heritabilities of craniofacial anthropometric traits, and the results have been in agreement with ours.
In one study based on an Indian population, the heritabilities of nasal height and breadth were 0.42 and 0.50; in addition, that of facial height and bizygomatic breadth were 0.41 and 0.61 [29].
Another study enrolling a Chuvasha pedigree reported that the heritabilities of nasal height, physiognomic facial height, bizygomatic breadth, and bigonial breadth were 0.42, 0.38, 0.58, and 0.50, respectively [14].
The other paper targeting orbital traits of Brazilians reported that the heritabilities of intercanthal width, biocular width, and length of the eye fissure were 0.39, 0.34, and 0.51 [30].
Recently, we reported the heritabilities and suggestive genetic loci associated with ocular and nasal traits in an isolated Mongolian population [31].
The heritabilities in Mongolians (h2 = 0.48-0.90) were slightly higher than those of Koreans (h2 = 0.25-0.61).
The difference between the two populations can be explained by the low level of genetic heterogeneity and environmental effects in the isolated population model.
Several groups have reported that vertical facial variables displayed greater genetic effects than horizontal facial variables [8, 9, 32].
However, another group has challenged this by presenting an opposite result, although it was found not to be statistically significant [14].
The results obtained in our study support for a greater genetic effect of the horizontal craniofacial variables, where vertical variables (tr-gn, tr-g, g-sn, and sn-gn) displayed less heritability than the horizontal variables (zy-zy and go-go).
A similar pattern was also observed for the nose region.
We were able to identify 3 latent factors related to facial measurements through factor analysis (Table 5).
Factor 2 showed relation to the orbital region: intercanthal width (en-en), biocular width (ex-ex), and length of the eye fissure (en-ex).
When we consider the early development of the face, there are very unique processes for making eyes and orbits.
Factor 2 may be related with one of the common regulators controlling these processes.
Factor 3 included vertical length values in the upper face region (special upper face height [g-sn] and height of the nose [n-sn], and physiognomical height of the face [tr-gn]).
Because all three values are related with vertical length of the nose (n-sn), this factor may be related with the early developmental process of the nose from the lateral and/or medial nasal prominence.
Factor 1 was mainly related to the lower portion of the face-height of the lower face (sn-gn), width of the mandible (go-go), width of the nose (al-al), and width of the mouth (ch-ch) and this part was made from the maxillary and mandibular prominences of the first pharyngeal arch.
However, there remains the possibility that Factor 1 is related with a general factor rather than a specific regulator, because 7 of the 14 measurement values had loading scores above or very close to 0.6.
Such findings are indicative of a necessity to categorize each facial portion into different classes to improve our understanding of facial genetics.
In previous studies, however, the concept of factor analysis in the genetic study of facial morphology was mainly confined to horizontal and vertical measures.
Hence, this new concept of facial regions in the current report may provide significant insight into the genetic study of facial morphology.
Factors that are explained by a common facial variable can have a higher genetic variance than each facial measurement [33].
Factor 1, related to the lower face, also indicated the highest genetic variance (h2 = 0.55, SE = 0.12), which was significantly higher than the mean of each trait (mean h2 = 0.36).
The estimated heritabilities for Factor 2 and Factor 3 also showed a similar trend (h2 = 0.52, SE = 0.15; mean h2 = 0.44 and h2 = 0.45, SE = 0.13; mean h2 = 0.34, respectively) (Table 6).
They suggest that the factors reflect the genetic effect well than when compared to each original measurement in the regions.
Additionally, this study employed a bivariate model to investigate shared genetic effects for all pairs of facial measurements within each factor.
We estimated the genetic and environment correlations and found 5 pairs with a statistically significant genetic correlation: 1 for Factor 1 and 2 each for Factor 2 and Factor 3.
The significant genetic correlation ranged from 0.74 to 1.00, which suggested that there exists a large genetic portion simultaneously affecting facial measurements in each factor.
To date, various study designs have been employed for identifying the genetic effects of a specific trait where, in general, family, twin, or sibling designs were preferentially chosen.
Amongst these three, the family design has many advantages.
Firstly, familial correlation and relative risk can be estimated [34].
Secondly, a family study is likely to increase statistical power for finding causative genes and inheritance modes [35] where, especially, a large expended pedigree provides sufficient evidence of linkage as well as maximum statistical power [36].
A variety of methods for measuring facial length or size have been introduced by many researchers [37].
Simple and direct measurement using calipers takes a lot of time and effort and has the probability of incorporating errors between different observers.
Another method of facial measurement is radiographic cephalometry, which has the advantage of observing hard tissue, such as bones, but this method is not suited for facial morphology, since it requires high cost and is time-consuming and is thus not suited for large-scale epidemiology.
An alternative method used in this study was 2D digital photography.
Although this method is difficult to locate bony structures under the skin, which is necessary to determine facial landmarks as well as measure curved surfaces, this has several advantages.
The pictures are quickly obtainable and the measurements can be permanently stored.
Observers can be trained easily for precise and unbiased measurement.
Radiographic cephalometry is selected for clinical patient care, whereas photographic method is better for investigating facial morphology in a large epidemiological study [20], thereby suggesting that each method for facial measurement should be chosen to fit the purpose of the study; we found our choice of photographic method to be reliable and useful in our study.
Our study has several new aspects and advantages: we provide the first reliable data for facial morphology for the Korean population; we tried a new approach to study faces using factor analysis; and the family data containing a large extended pedigree used in this study provided high statistical power for estimating heritability.
Yet, this study is not without caveats.
The most obvious limitation in this study is the small sample size.
Since the number of subjects was not large enough, the familial correlations between first-degree relatives, especially most sibling pairs, might not have been statistically significant.
In our results, the familial correlations of parent-offspring pairs were relatively more significant than those of sibling pairs (Table 4).
This is probably due to a relatively small number of sibling pairs than parent-offspring pairs.
For the heritability estimation of the 14 phenotypes, the value for width of the mouth (ch-ch) was found to be the smallest and not statistically significant.
This may be due to the change in facial expression while taking photographs (emotion or tension), causing variability in width of the mouth (ch-ch).
Facial tension created during unusual situations, such as while taking photographs, can influence the measurements, especially for the width of mouth, because of the likelihood for this region to be affected by psychological tension compared to other measurement parts.
In addition, we were not able to provide enough data for each age in children, which is an important basis for the diagnosis of congenital facial anomalies.
In conclusion, we have verified the genetic effect on facial anthropometric traits and suggested a potential way to categorize and analyze the facial portions into different groups.
Despite many reports on congenital diseases with accompanying facial anomalies, the understanding of common genetic mechanisms, including specific genes that determine the facial morphology in general healthy populations, is still obscure.
Our report is a step towards understanding this.
Validation with the results of other studies involving varying populations, in addition to studies on genetic linkage association or biologic studies, is warranted for furthering our understanding of this subject.
Twenty five facial landmarks used in this study.
al, alare; ch, cheilion; en, endocanthion; ex, exocanthion; g, glabella; gn, gnathion; go, gonion; n, nasion; prn, pronasale; sa, superaurale; sba, subaurale; sn, subna; tr, trichion; zy, zygion.
Title: Facial landmarks and measurements in each region

Title: Sample and pedigree information used in the study

Title: Mean and SD of craniofacial measurements of adults age ≥ 20

Title: Familial correlations and heritabilities of facial measurements

Title: Factor analysis of 14 facial measurements

Title: Familial correlations and heritabilities of factor variables

Title: Genetic and environmental correlations for pairs of facial-related measurements
Title: Allelic Frequencies of 20 Visible Phenotype Variants in the Korean Population

The prediction of externally visible characteristics from DNA has been studied for forensic genetics over the last few years.
Externally visible characteristics include hair, skin, and eye color, height, and facial morphology, which have high heritability.
Recent studies using genome-wide association analysis have identified genes and variations that correlate with human visible phenotypes and developed phenotype prediction programs.
However, most prediction models were constructed and validated based on genotype and phenotype information on Europeans.
Therefore, we need to validate prediction models in diverse ethnic populations.
In this study, we selected potentially useful variations for forensic science that are associated with hair and eye color, iris pattern, and facial morphology, based on previous studies, and analyzed their frequencies in 1,920 Koreans.
Among 20 single nucleotide polymorphisms (SNPs), 10 SNPs were polymorphic, 6 SNPs were very rare (minor allele frequency &lt; 0.005), and 4 SNPs were monomorphic in the Korean population.
Even though the usability of these SNPs should be verified by an association study in Koreans, this study provides 10 potential SNP markers for forensic science for externally visible characteristics in the Korean population.
DNA genotyping has been used in forensic science for over 30 years for human identification by using short tandem repeat (STR) markers, which are highly polymorphic variants [1].
Recently, single nucleotide polymorphisms (SNPs) have also been used for forensic marker, because they may provide better profiling from degraded DNA samples than STRs and do not involve repetitive sequences that may make stutter artifacts, which complicates STR profiling interpretation [2, 3].
SNPs for forensic analyses can be used for human identification, kinship analysis, inferring biogeographic ancestry, and estimating appearance traits [4].
In general, forensic DNA profiling is used for identifying persons compared with profiles of known suspects or missing persons included in a DNA database.
However, in the case of absent information on felons or missing persons in the DNA profile database, it is helpful to predict unknown persons with DNA markers [5].
The prediction of externally visible characteristics (EVCs) from DNA has been studied for forensic genetics over the last few years [5-7].
The EVCs include hair, skin and eye color, height, and facial morphology, which have high heritability [8-10].
The genome-wide association study (GWAS), a linkage analysis and candidate gene study, has been used to identify genetic variants influencing such EVCs.
Variations in the MC1R gene have been associated with red hair [11].
The red hair prediction method, based on a combination of non-synonymous SNPs in MC1R, was already developed for forensic science more than 10 years ago [12], and its accuracy was 84% in the prediction of red-haired individuals.
The SNPs in the OCA2 locus and the adjacent HERC2 gene in Europeans showed a strong correlation with blue and brown eye color [13, 14].
Other genes were also identified as contributing to hair, eye, and skin color variations, such as IRF4, KITLG, SLC24A4, SLC45A2, TYRP1, TYP, and ASIP [15].
Recent studies using genome-wide association analysis have identified genes and variations correlating with human visible phenotypes and developed a phenotype prediction program [16-19].
HIrisPlex is capable of predicting hair and eye color from DNA with 24 SNP variations [19].
Six of the 24 SNPs used for eye color prediction in the HIrisPlex program are rs12913832 (HERC2), rs1800407 (OCA2), rs12896399 (SLC24A4), rs16891982 (SLC45A2 [MATP]), rs1393350 (TYR), and rs12203592 (IRF4), and 4 of these SNPs within HERC2, OCA2, SLC45A2, and IRF4 are also used for hair color prediction.
To predict hair color and hair color shade, HIrisPlex uses 22 SNPs as follows: N29insA, rs11547464, rs885479, rs1805008, rs1805005, rs1805006, rs1805007, rs1805009, Y152OCH, rs2228479, and rs1110400 from the MC1R gene, rs28777 (SLC45A2 [MATP]), rs16891982 (SLC45A2 [MATP]), rs12821256 (KITLG), rs4959270 (EXOC2), rs12203592 (IRF4), rs1042602 (TYR), rs1800407 (OCA2), rs2402130 (SLC24A4), rs12913832 (HERC2), rs2378249 (PIGU/ASIP), and rs683 (TYRP1).
The results of these 24 SNPs when their minor alleles are input into HIrisPlex are used to predict the color of an individual with the highest probability of the 3 eye color categories (brown, blue, or intermediate) and 4 hair color categories (blond, brown, red, and black).
On average, their prediction accuracy was 69.5% for blonde hair, 78.5% for brown, 80% for red, 87.5% for black hair, and over 90% for blue and brown eyes.
Few GWASs on color-related phenotype have been carried out in Asian populations [20, 21].
Therefore, most prediction models for EVCs have been constructed and validated based on genotype and phenotype information on Europeans; these prediction models must be validated in diverse ethnic populations [16-19].
In this study, we selected potentially useful variations for forensic science that are associated with hair and eye color, iris pattern, and facial morphology, based on previous studies, and analyzed their frequencies in the Korean population.
Title: Population and DNA extraction

The subjects in this study were collected from the Korea Association Resource (KARE) study, which has been described in detail previously [22].
Briefly, KARE consists of 2 community-based cohorts, Ansan (urban community) and Ansung (rural community) in Korea, and includes 10,038 participants aged 40 to 69 years.
Genomic DNA of 1,920 male subjects randomly selected from the KARE study was extracted from cell lines immortalized with Epstein Bar virus.
Title: SNP selection

Using the NHGRI GWAS catalog (http://www.genome.gov/gwastudies) [23], 9 articles were identified with the query terms &#34;hair,&#34; &#34;eye,&#34; &#34;iris,&#34; and &#34;facial morphology,&#34; and 26 SNPs with association p-values &lt; 5 × 10-8 (GWAS p-value) were selected from the articles [10, 24-31].
The most significant SNPs were selected from the SNPs that were under pairwise linkage disequilibrium (LD) (r2 &gt; 0.8).
Title: Genotyping and quality control

Genotyping was performed at a multiplex level using the Illumina Golden Gate genotyping system [32].
The genotype quality score for retaining data was set to 0.1.
SNPs that could not satisfy the following criteria were excluded: 1) a minimum call rate of 90%, 2) no duplication error, and 3) Hardy-Weinberg equilibrium greater than p &gt; 0.001.
All 20 SNPs were successfully genotyped.
Using the GWAS catalog, we extracted 26 candidate SNPs that might be useful for forensic science in terms of EVCs.
Two SNPs in MC1R and HERC2 were already genotyped in subjects of the KARE study.
Among 26 SNPs, 2 pairs of SNPs, rs1847134 and rs1393350 within TYR and rs4900109 and rs12896399 within SLC24A4, respectively, were under pairwise LD (r2 &gt; 0.8), and 1 SNP in each pair was selected.
Two SNPs showing low probe design ability were excluded from the probe selection process.
Finally, 20 out of 26 SNPs were selected for genotyping in Koreans.
One SNP, rs7559271 (PAX3), was associated with facial morphology [31]; 2 SNPs, rs3739070 (TRAF3IP1) and rs10235789 (SEMA3A), were identified for the association with iris patterns, such as furrows and crypts, respectively [29]; and the remaining 17 SNPs were associated with hair and/or eye color [10, 24-28, 30].
Among these variants, 8 SNPs were included in the HIrisPlex program the determine hair and iris color.
Allele frequencies and other genetic parameters for the 20 SNPs are provided in Table 1.
All analyzed SNPs were in Hardy-Weinberg equilibrium in the Korean population (p &gt; 0.001).
Three SNPs that were associated with facial morphology or iris characteristics in Europeans [29, 31] were polymorphic in the Korean population as follows: rs7559271 (PAX3), rs3739070 (TRAF3IP1), and rs10235789 (SEMA3A).
Their minor allele frequencies (MAFs) were 0.303, 0.029, and 0.103, respectively.
Among 17 SNPs related to hair or eye color, 4 SNPs were monomorphic and 6 SNPs were very rare (MAF &lt; 0.005) in Koreans.
The 4 monomorphic SNPs in Koreans were rs12203592 (IRF4), rs12821256 (KITLG), rs12913832 (HERC2), and rs8033165 (intergenic).
Their allele frequencies in Europeans (CEU) in the HapMap database were 0.167, 0.146, 0.792, and 0.447, respectively.
The 6 rare SNPs in Koreans were rs16891982 (SLC45A2), rs1408799 (TYRP1), rs35264875 (TPCN2), rs1393350 (TYR), and rs1805007 (MC1R), and their allele frequencies in Europeans were 0.983, 0.695, 0.175, 0.226, and 0.121, respectively.
The HapMap database did not include the allele frequency of rs12931267 (MC1R).
Since frequency information alone does not explain phenotype variability, association analysis is required to confirm the genetic effect of the SNPs between appearance phenotype and SNPs.
Stokowski et al.
[20] reported that SNPs in TYR, SLC45A2, and SLC24A5 were associated with skin pigmentation and largely accounted for differences between those with the darkest and lightest skin in a South Asian sample.
A previous study analyzed skin and hair color in Koreans, categorizing hair color into 3 types and skin color into 4 types [33].
The 7 polymorphic SNPs in the Korean population are likely to contribute to color variation within the Korean population.
In addition, the 4 monomorphic SNPs could be used for distinguishing East Asians from Europeans.
In this study, we provided the allele frequency of 20 EVCs SNPs by genotyping a large number of Koreans (1,920 individuals) in comparison to the 45 Chinese and 45 Japanese recruited for the International HapMap Project.
Even though the usability of these SNPs should be verified by association study in Koreans, this study might provide 10 potential SNP markers for forensic science for EVCs in the Korean population.
Title: Allele frequencies of 20 visible phenotype SNPs
Title: Retraction: A Fosmid Cloning Strategy for Detecting the Widest Possible Spectrum of Microbes from the International Space Station Drinking Water System

Title: http://dx.doi.org/10.5808/GI.2012.10.4.249

Title: Genomics Inform 2012;10(4):249-255

The editorial committee of Genomics &amp; Informatics has concluded that a substantial portion of the above article [1] was &#39;copied and pasted&#39; from earlier publications without appropriate attribution.
Although the plagiarized texts are detected only in the Introduction section [2], such a misconduct is unacceptable in a scientific writing.
The authors have acknowledged plagiarism in the above article and requested retraction.
Title: ERRATUM: Acknowledgments Correction.
Bioinformatics Interpretation of Exome Sequencing: Blood Cancer

Title: http://dx.doi.org/10.5808/GI.2013.11.1.24

Title: Genomics Inform 2013;11(1):24-33

Title: The funding acknowledgment in this article was partially omitted as published.
Additional acknowledgment is as follows:

This work was supported by a National Research Foundation of Korea (NRF) grant funded by the Korea government (MEST) (nos.
20110030770 and 20120009215), a grant from the Next-Generation BioGreen 21 Program (nos.
PJ008019 and PJ008068), Rural Development Administration, a grant from KRIBB Research Initiative Program, Republic of Korea, and a grant of the National Project for Personalized Genomic Medicine, Ministry for Health &amp; Welfare, Republic of Korea (A111218).
Title: ERRATUM: Acknowledgments Correction.
Annotation of Genes Having Candidate Somatic Mutations in Acute Myeloid Leukemia with Whole-Exome Sequencing Using Concept Lattice Analysis

Title: http://dx.doi.org/10.5808/GI.2013.11.1.38

Title: Genomics Inform 2013;11(1):38-45

Title: The funding acknowledgment in this article was partially omitted as published.
Additional acknowledgment is as follows:

This work was supported by the basic science research program through the National Research Foundation of Korea (NRF), funded by the Ministry of Education, Science and Technology (2012-0000994).
This material is based upon work supported by a grant of the National Project for Personalized Genomic Medicine, Ministry for Health &amp; Welfare, Republic of Korea (A111218).
Title: Editor&#39;s Introduction to This Issue

This is a special issue featuring several review articles in the area of metagenomics.
The advancement of next-generation sequencing technology expedites metagenomics, which profiles the population of species in an environment.
Prof. Jihyun Kim at Yonsei Univerisity invited three renowned metagenomics researchers to write review articles.
In the first one, a group headed by Prof. Hana Yi at Korea University College of Medicine, Seoul, Korea surveys databases and analysis tools in metagenomics.
The second review article by Prof. Seon-Woo Lee&#39;s team at Dong-A University, Busan, Korea focuses on soil-borne microorganisms with particular emphasis on their potential as sources of novel biological activities.
The last review by Prof. Jin-Woo Bae and colleagues at Kyung Hee University, Seoul, Korea discusses viral metagenomes in a variety of environmental samples.
In addition, we feature four more articles that were winners of outstanding prizes for this year&#39;s KOGO Young Researchers Best Paper Award.
Three of them deal with genome-wide association studies using Korea Association Resource (KARE) data.
For example, Dr. Kyung-Won Hong at Korea National Institute of Health, Cheongwon, Korea reports risk variants associated with orthostatic hypotension and their functional implications.
The last awardee is Ms. Yun-Jeong Kwon from Prof. Heui-Soo Kim&#39; lab at Pusan National University, Busan, Korea.
She reports the SINE-VNTR-Alu elements, a retrotransposon, in relation to associated functional genes.
We hope you enjoy this issue and appreciate your continued support.
Title: Analytical Tools and Databases for Metagenomics in the Next-Generation Sequencing Era

Metagenomics has become one of the indispensable tools in microbial ecology for the last few decades, and a new revolution in metagenomic studies is now about to begin, with the help of recent advances of sequencing techniques.
The massive data production and substantial cost reduction in next-generation sequencing have led to the rapid growth of metagenomic research both quantitatively and qualitatively.
It is evident that metagenomics will be a standard tool for studying the diversity and function of microbes in the near future, as fingerprinting methods did previously.
As the speed of data accumulation is accelerating, bioinformatic tools and associated databases for handling those datasets have become more urgent and necessary.
To facilitate the bioinformatics analysis of metagenomic data, we review some recent tools and databases that are used widely in this field and give insights into the current challenges and future of metagenomics from a bioinformatics perspective.
Metagenomics is defined as the study of the metagenome, which is total genomic DNA from environmental samples.
Metagenomics has long been one of the major research tools in microbial ecology since the term was first used in 1998 [1].
Metagenomic information allows for a more in-depth understanding of the ecological role, metabolism, and evolutionary history of microbes in a given ecosystem by analyzing environmental DNA directly without prior cultivation.
Metagenomics has made unprecedented contributions to microbial ecology; among them, one of the most outstanding discoveries of metagenomics is the first description of proteorhodopsin in marine bacteria [2].
Moreover, many hypotheses and questions in ecology and evolutionary biology have been tested and answered through metagenomic research.
For example, Fuhrman et al.
[3] tested a latitudinal gradient of biodiversity, which is one of the most widely recognized patterns in macroscopic taxa, on marine bacteria, and more recently, the taxonomic and functional distinction of bacteria in desert compared to other nondesert biomes was investigated through cross biome comparison using shotgun metagenomics [4].
Varied definitions converge into two main categories, targeted metagenomics and shotgun metagenomics, depending on their purpose and target materials.
This review offers an overview of the tools and databases that are widely used in both targeted and shotgun metagenomics, in particular emphasizing on metagenome sequences generated by recent next-generation sequencing (NGS) technologies.
The recent development of sequencing technologies has enabled us to assess much deeper layers of microbial communities by generating tons of nucleotide sequences at lower costs.
NGS technologies have revolutionized the field of microbial ecology, as they have allowed researchers to reach the true level of diversity more closely through more in-depth sequencing.
There are various applications using these NGS platforms, ranging from single-gene targeted sequencing to whole-genome sequencing and shotgun metagenome sequencing.
Details about the principles and applications of each NGS technique have already been reviewed elsewhere [5].
The first NGS platform, pyrosequencer (GS20), generated just 20 Mb per run with a read length of 100 bp on average.
A recent version of pyrosequencer (GS-FLX Titanium) has an advantage over other NGS techniques, especially 16S rRNA gene-based surveys, as it produces around ~500 Mb per run with a longer read length (400-500 bp), which is sufficient to cover partial hypervariable regions of 16S rRNA.
Illumina sequencing produces many more reads with cheaper price that are more accurate (accuracy of &gt;99%) than 454 platforms (accuracy of 98.93%) [6] but is somewhat limited in certain fields due to the relatively shorter read length (&lt;100 bp) of early versions.
However, Illumina became more popular, since it gradually improved readable lengths (i.e., 2 × 250 bp in MiSeq).
More recently, Pacific Biosciences has released a new sequencing technology, PacBio RS, and Oxford Nanopore Technologies introduced GridION/MinION devices, both of which allow single-molecule sequencing with a much longer read length.
However, further improvements are necessary for use in practice due to the high intrinsic errors (10% to 15% in PacBio and around 4% in GridION) [7].
Since the first launch of the NGS platform in 2006, novel sequencing technologies have been developed continuously and rapidly.
As a result, massive sequence data produced by NGS technologies are accumulating at an unflagging rate.
However, the computing power and development of algorithms needed to deal with the huge datasets efficiently are not keeping up with the speed of data production.
For example, access to sequence data is still hampered by unsuitable data storage systems, such as short read archive (SRA), and many early papers were not accompanied by data deposition into public databases.
Repositories should be big enough to be ready to allow the increasing volume of upcoming sequence data, and all data must be deposited in a standardized manner.
In addition to the shortage of data storage space and confounding submission formats, the characteristics of the produced sequence data pose another problem in further processing.
For instance, shorter read length, which is an inevitable limit of high-throughput sequencing, is a barrier for sequence assembly, and the relatively higher rates of sequencing errors compared with previous Sanger methods also make it hard to recover genuine sequences.
Targeted metagenomics refers to a metagenomic approach that surveys genes or genomic regions of particular interest by targeted methods (activity- and sequence-driven studies) [8].
Sequence-driven targeted metagenomics is normally conducted by PCR-based directed sequencing of environmental genomic DNA.
Ribosomal RNA genes (e.g., small subunit [SSU] and large subunit [LSU]) have been used as taxonomic marker genes for identifying microbial species.
The characteristics of having both highly conserved and variable regions have allowed for accurate taxonomic identification and made it easier to design primers targeting the whole taxa of interest.
Although taxonomic resolution of the 16S rRNA gene is not sufficient for delineating taxa at the species or strain level in some cases [9], the SSU of ribosomal RNA (16S rRNA) has been used widely as a standard marker gene in prokaryotes.
For the last few decades, molecular methods to assess microbial diversity have focused on rRNA genes using probe-based approaches, such as fluorescent in situ hybridization and microarray, fingerprinting methods, and molecular cloning.
Some of them are still in use, but in this review, we focus mainly on sequence data generated by recent NGS techniques.
Microbial community profiling using taxonomic marker genes (e.g., 16S rRNA gene) commonly uses an operational taxonomic unit (OTU)-based approach, as the sequence-based species definition in microbes is still vague and current public databases still do not reach the full extent of microbial diversity, despite the massive sequencing efforts.
This OTU-based approach is now generally accepted in most microbial community studies based on environmental samples.
Over the last few years, 454 pyrosequencing has been a major source of generating amplicon metagenomics data among NGS platforms due to its capability of producing a relatively longer read length.
Therefore, bioinformatic analysis tools dealing with sequence data have been developed and tailored for pyrosequencing results.
More detailed information about the algorithms and processes at each step can be found in several other reviews [7, 10].
In this part, we introduce recent tools and databases and provide brief explanations about how they work in the course of the analysis workflow (Table 1) [11-29].
The first part of an analysis of NGS-generated data starts from filtering out &#39;noise&#39; sequences.
Most metagenomic studies based on single- or multiple-gene amplicons have used 454 pyrosequencing due to its advantage of producing longer read lengths, and currently available denoising algorithms have also been developed for that purpose.
The denoising process per se does not remove actual sequences but keeps abundant information on erroneous sequences by retaining representative reads.
Several denoising algorithms have been suggested so far.
PyroNoise [11] implements a flowgram clustering method, and other denoising tools, such as Denoiser [12], DADA [13], and Acacia [14], use sequence abundance information on the denoising process.
Similarly, single-linkage preclustering can be used before performing the formal OTU clustering to reduce &#39;noise&#39; sequences generated by PCR and sequencing errors [30].
It first ranks sequences in order of decreasing abundance, and rarer sequences within a certain threshold are merged into the original abundant sequences.
Once denoising and additional quality control processes are completed, chimeric sequences should be removed from the dataset.
Chimeras are artificial recombinants between two or more parental sequences, and they are normally formed when prematurely terminated fragments reanneal to other template DNA during PCR amplification [31].
These artificial molecules make it difficult to differentiate the original sequence from recombinants, resulting in overestimation of the level of microbial diversity in environmental samples [32].
Once chimeras are generated and sequenced, they need to be identified and removed from the dataset using bioinformatics tools.
However, detecting chimeras is still challenging, as breakpoints can take place at any position more than once, and NGS platforms generate shorter lengths of sequences, making them hard to differentiate the source of parents with insufficient taxonomic information.
Several elegant algorithms and tools have been suggested for preferentially identifying chimeric sequences in high-throughput datasets.
These tools include UCHIME [15], ChimeraSlayer [16], Perseus [11], and Decipher [17].
All of these tools, except for ChimeraSlayer, use sequence frequency information to detect chimeras, assuming that chimeric sequences are less frequently represented in a given dataset than normally amplified sequences.
There is no algorithm to detect chimeras perfectly, but to date, it has been known that UCHIME outperforms other algorithms, at least for short NGS reads [15].
Although there are still several limitations in detecting chimeric sequences that are formed by hybridization between closely related organisms, the tools listed above work well on SSUs and LSUs, but further validations are necessary for internal transcribed spacers and other functional genes.
The next step following chimera detection is OTU clustering, which is an essential process in community analysis, as it sorts sequences with the closest matches and then gives a taxonomic meaning to the clustered group.
Sequencing errors, chimeras, and clustering algorithms have great influence on the quality of OTUs [33].
There are generally two ways that have been suggested for generating OTUs.
One is alignment-based clustering, and the other is the alignment-free clustering method.
Sequence alignment is done by either aligning query sequences against pre-aligned reference sequences [34] or using pairwise and multiple sequence alignments [35].
It is known that alignment quality has a significant impact on OTU clustering results [36].
Alignment quality varies, depending on gene loci and the parameters used, but sequence alignment incorporating secondary structure information generally improves OTU assignment, at least for 16S rRNA gene sequences [37].
The nearest alignment space termination (NAST) algorithm [34] has been used successfully in microbial ecology as a profile-based alignment tool, and SINA aligner [38], based on partial order alignment, and infernal [39], using consensus RNA secondary structure profiles, were recently introduced.
However, there is currently no method or algorithm that does automatic alignment almost perfectly or up to the quality that can be achieved manually.
Alternatively, alignment-free methods are more broadly used in picking OTUs.
Commonly used alignment-free tools are UCLUST [18], CD-HIT [19], and ESPRIT-Tree [20].
UCLUST and CD-HIT implemented their own sorting processes in OTU clustering by sorting sequences in order of decreasing abundance (i.e., UCLUST) or decreasing length (i.e., CD-HIT), and ESPRIT-Tree uses an average linkage-based hierarchical clustering algorithm.
There are some other tools that implement a &#39;homopolymer collapse option&#39; for minimizing homopolymer errors in 454 reads: CLOTU [40], SCATA (http://scata.mykopat.slu.se), CrunchCluster [41], and TBC [21].
However, these tools are currently applicable only to 454 reads, and there is a fundamental limitation of not being able to detect &#39;real&#39; sequence differences appearing in homopolymer regions.
SCATA has suggested another OTU picking approach by using both reference sequences and query sequences together in the clustering process, and then the OTU is assigned to a taxonomic identity of the reference sequence if the reference sequence belongs to the cluster.
A taxonomy-dependent approach was recently proposed as an alternative to OTU clustering approaches [42].
This method has two major advantages over OTU clustering.
First, direct taxonomic assignment to each query sequence is more tolerant to sequencing errors than the OTU picking process, as the assignment process is less affected by mismatches or insertion/deletion errors, while sequencing errors are known to generate many spurious OTUs.
It also helps prevent a massive loss of erroneous but still taxonomically meaningful sequences.
Second, it enables researchers to perform a more standardized community analysis, based on a single assignment rule.
There is no truly &#39;universal&#39; primer set, and the difference in primer sets makes it difficult to compare datasets, as the different primer sets amplify differing variable regions within 16S rRNA genes and catch only a partial body of the whole &#39;true&#39; community.
On the other hand, a taxonomy-supervised method can circumvent the problem by directly assigning sequences to the closest relatives using full-length reference sequences.
This approach also facilitates cross-comparison and meta-analysis across versatile microbial community studies.
A caveat is the low taxa coverage of current marker gene databases, which offsets the advantage of this approach and hinders the application in practice by failing to find the closest matches to known members, due primarily to its lack of close relatives in a given database.
Another limitation is the relatively lower assignment accuracy and precision of short NGS reads compared to those of full-length sequences.
It is generally known that even longer 454 Titanium reads are able to be identifiable with high confidence scores at best to the genus level [43], even genus-level classification is doubted [44].
However, this problem will be soon overcome as more high-quality sequences are added to the databases.
For example, EzTaxon-e benefits from using artificially defined species names, which are defined based on combinatorial evaluation using both phylogeny and pairwise similarity between unclassified sequences on all taxonomic levels.
It endows taxonomically more meaningful information to those &#39;unclassified&#39; sequences rather than remaining unknown or environmental sequences.
Moreover, recent updates have slightly improved the database coverage by adding error-free 454 reads that are selected using abundance information (http://eztaxon-e.ezbiocloud.net/).
Identifying each individual sequence is of great importance in microbial community analysis, as taxonomic information gives us access to basic information of its traits, such as physiology, epidemiology, and evolutionary history, and it allows for indirect inference of their ecological roles in a given environment.
There are several methods that have been suggested for assigning microbial taxonomy with high-throughput sequence data.
BLAST [45] is one of the most widely used algorithms when classifying sequences.
The top BLAST hits by searching against reference databases are generally used for taxonomic identification of query sequences.
There are several well-curated public SSU databases, such as RDP [22], SILVA [23], Greengenes [24], and EzTaxon-e [25].
However, the top BLAST hit does not always give the correct taxonomy result, especially in shorter reads.
Alternatively, Greengenes and EzTaxon-e use BLAST in conjunction with global alignment, and the post processing of a BLAST-aligned output has been suggested to improve assignment accuracy using the lowest common ancestor (LCA) algorithm in MEGAN [46] and the optimal consensus method (F-measure) in TANGO [47].
The probabilistic approach also gives a similar level of accuracy but is much faster than a BLAST search.
The naïve Bayesian algorithm [48], implemented in RDP, uses 8-mer word-matching for training datasets and provides assignment results with bootstrapped confidence estimates.
Phylogenetic placement method is a recently suggested approach that places query sequences into a phylogenetic guide tree on the basis of various evolutionary models.
This approach is particularly useful in a case where there are no close relatives to the query sequence in databases, due primarily to the low DB coverage, which is the case with microbial eukaryotes.
Tools using phylogenetic methods in taxonomic assignment include the evolutionary placement algorithm (EPA) [49], pplacer [50], and SEPP [51], and these algorithms were recently incorporated into QIIME [28] and AMPHORA2 [52].
Until the arrival of NGS technologies in this field in 2006, shotgun libraries had been constructed using circular vectors, each of which had an insert derived from a metagenomic DNA fragment [53].
Sequencing strategies designed either to completely assemble the sequence of each long-insert vector (tens to thousands of kilobases) or to generate a paired-end read from small inserts (around 1-2 kilobases) were applied to those vector libraries.
The former strategy has a unique strength, in that it is relatively easy to generate contigs orders of magnitude longer than the read length when assembling the reads from homogeneous templates [2].
However, such a strategy is not scalable to analyze a large number of clones, as the tremendous amount of reads per clone needs to be screened.
The latter strategy, a genuine random shotgun approach, took essentially the same approach with the NGS-based metagenome assembly, which enables analysis of a large number of clones but requires high coverage in order to be successful [54, 55].
Regardless of the strategies taken, vector library construction and subsequent sequencing are time-consuming and cost-ineffective, and only a few organizations can lead such efforts.
NGS technologies have proven their utility in shotgun metagenomics since their earliest application appeared in 2006 with studies using 454 pyrosequencing data.
After all, technologies like 454 and Illumina sequencing have become routinely applied for shotgun metagenomics, accompanying changes in the trends of the field as well as the properties of the data themselves.
As sequencing cost continues to decrease, more researchers outside the big sequencing centers have started to participate in metagenomic studies, resulting in higher individuality and diversity of metagenome data.
Bioinformatics for metagenome analysis has progressed, along with the explosion of data, both by extension from the tools and databases previously developed in genomics and the adoption of methodologies used by ecologists.
Less than a decade has passed since metagenomic shotgun sequencing took its place as general practice for biologists, and it is fair to say that bioinformatics for shotgun metagenomics is at a premature stage.
In this section, we aim to summarize the methods that seem to be the &#39;current&#39; standard, the most widely used, in the analysis of shotgun metagenomic data (Fig.1).
Assembly is the process of merging overlapped short reads into larger contiguous sequences (contigs).
Current sequencing technologies normally break genomic DNA into pieces, and original genomic sequences need to be recovered from fragmented reads by an assembly process.
In metagenomics for retrieving coding regions, it is necessary to make it easier to do functional annotations, as a more accurate classification/annotation is possible in longer sequence information.
Most assemblers have been developed for the single genome or clonal populations with high coverage.
There are currently few metagenomic assemblers, as metagenomic reads are more complex, owing to nonclonal heterogeneous reads resulting from multiple strains differing only by partial regions or rearrangements, lower or uneven coverage across genomes, sharing of repetitive sequences among closely related species, and lateral gene transfer sharing similar entities between distantly related organisms.
At an early stage of metagenomic studies, reference mapping approaches (e.g., Newbler, AMOS, MIRA) were applied to the metagenome assembly process.
However, with the low coverage and complexity of megenomic reads, de novo assemblers were preferred to reference mapping, based on the advantage of feasibility for dealing with the aforementioned problems.
Starting from initial assemblers, such as SOAPdenovo [56], there are now several improved assemblers available: are Genovo [57], Meta-IDBA [58], Meta-Velvet [59], MAP [60], and Ray Meta [61].
Metagenome-specialized assemblers have to distinguish reads from different species.
MetaVelvet and Meta-IDBA resolve this issue by partitioning the de brujin graph based on k-mer coverage and separately assemble each subgraph.
Ray Meta does not decompose the de brujin graph; instead, it uses a heuristics-guided graph traversal approach to find the optimal assembly.
Meta-IDBA, MetaVelvet, and Ray Meta were developed to perform well on short reads (e.g., Illumina sequencing).
On the other hand, other tools, like Genovo and MAP, were developed for longer reads (e.g., 454 sequencing).
The outputs from various assemblers can be used to generate scaffolds using Bambus2 by avoiding misjoins between distantly related organisms by detecting repeats and genomic variants [62].
Taxonomic classification of reads, which is called binning, is the most basic step in the characterization of microbial communities by metagenome sequencing.
However, taxonomic binning of metagenome shotgun sequences is a challenging task for researchers, especially when working with short reads derived from NGS.
There are several reasons that make binning a nontrivial job.
One of the reasons is that NGS technologies generally produce short reads.
When applying shotgun sequencing to complex microbial communities found in soil, seawater, freshwater, and the gastrointestinal tract, the sequencing coverage does not reach the level required to make the assembly practically useful, even by current deep sequencing methods.
The majority of reads remains unassembled.
As a result, the majority of reads is taxonomically classified by the information in their short length.
Short length presents a number of drawbacks in binning.
Homology search-based methods suffer from a lack of alignment confidence and difficulties in predicting protein sequences from partial genes.
Phylogenetic methods suffer from a lack of resolution due to insufficient phylogenetic information.
Another challenge is the size of data that have to be processed during the binning process.
Data size here refers to both the reads obtained from metagenomes and the sequences in the reference database.
The novelty of microbes in the environmental samples also hampers binning, as they are not represented by any sequence in the reference database.
One last challenging thing for researchers is the diversity of binning tools.
There are dozens of choices, differing in both logical and practical aspects.
A straightforward approach for taxonomic classification of metagenome shotgun reads is searching for a similar sequence in a collection of known sequences that carries the taxonomic identity of each sequence.
Similarity-based methods vary in at least three dimensions.
1) Choice of reference database: the search space could be restricted to a particular marker gene (e.g., SSU rRNA for prokaryotes) or a small number of selected protein families or could be as general as the entire NCBI nr database.
2) Search algorithm: from BLASTN (in case of using rRNA gene marker), BLASTX (protein database), and BLASTP (protein database after gene prediction) to a hidden Markov model (HMM)-based homology search (searching for protein families).
3) Taxonomic assignment from the hit information: from simply transferring the identity of the best hit to the use of LCA, modified LCA, or more complicated phylogenetic inferences.
Among the popularly used tools, MEGAN uses the NCBI nr database for searching and LCA algorithms for assignment [29].
MTR and SOrt-ITEMS modified MEGAN&#39;s LCA algorithm by use of taxonomic information shared by hits (MTR) or by performing a reciprocal BLAST hit to reduce false positive hits (SOrt-ITEMS) [63, 64].
MG-RAST exploits both rRNA gene sequences and protein-coding sequences [65].
For the rRNA gene sequences, MG-RAST follows the RDP pipeline [22], while for protein-coding genes, it first predicts the protein coding sequences and then performs a BLASTX search against a number of databases, including SEED [66], and extracts taxonomic information from SEED hits.
CARMA uses an algorithm similar to LCA and offers two ways for searching the database of known protein sequences: the NCBI nr database searched by BLASTX and the Pfam database searched by HMMER3 [67].
MetaPhyler is a power-up version of AMPHORA and uses 31 marker genes to reduce the search space [68, 69].
Composition-based classification of DNA sequences has been proven to be very useful, although it does not appear as intuitive as similarity-based methods.
Composition-based methods exploit the uniqueness of base composition (from single to oligonucleotide levels) found across the genomes of different taxonomic entities.
In many cases, these methods implement machine-learning algorithms.
Ultimately, all of these methods use the taxonomic information of the reference database to assign a taxonomic identity to the reads.
However, they can be divided into supervised and unsupervised methods, based on their dependence on the reference training set during the initial learning procedure.
The most popular tools using supervised learning are PhyloPythia, NBC, and Phymm.
Starting from the publicly available microbial genome sequences, NBC trains a naïve Bayes classifier based on the N-mer frequency profiles of each genome [70], Phymm builds an interpolated Markov Model using variable-length oligonucleotides typically found in taxa [71], and PhyloPythia trains a support vector machine classifier based on variable-length oligonucleotide composition [72].
Among the above, NBC and Phymm are suitable for classifying short reads generated from NGS sequencers.
The popular tools employing unsupervised learning are TACOA and TaxSOM.
TACOA introduced a kernelized k-nearest neighbor approach to cluster the reads [73], while TaxSOM uses batch-learning self-organizing maps (BLSOMs) and growing SOMs (GSOMs) to generate the clusters of related reads [74].
Both TaxSOM and TACOA are not suitable for unassembled short NGS reads.
There are tools that combine composition-based approaches and similarity-based approaches together to gain accuracy and discard fewer reads.
PhymmBL linearly combines the BLASTN score and Phymm score and thereby gains more power to discriminate between similar BLAST hits [71].
RITA also combines a BLAST search with a composition-based NBC; however, unlike PhymmBL, RITA puts more weight on the BLAST result [75].
Both approaches are reported to perform well, even with short reads, but as they involve a BLAST search, they consume much time.
There is a third type of taxonomic binning method recently implemented in the tool MetaPhlAn, which uses clade-specific marker genes [76].
From the database of microbial genomes, MetaPhlAn precalculated 400,141 genes that are most representative of each taxonomic unit.
In theory, detection of the reads that match these markers can classify the members at the species level.
MetaPhlAn uses BLASTN search to compare the reads against the set of marker genes.
As the search space is markedly reduced from general sequence databases used in other approaches, MetaPhlAn exhibits unusually high speed.
Functional assignments and pathway reconstructions are ultimate steps of shotgun metagenomics that allow the characterization of the functional potential of uncultivated microbes or microbial communities under investigation.
Connecting the sequences of metagenomic data to specific functions in the environments can be performed by using web-based workflows offered by useful portals without access to high-performance computers.
These online metagenome annotation services, like IMG/M [77], METAREP [78], CAMERA [79], and MG-RAST [80], provide platforms for gene prediction, assignment of functional categories, protein families and gene ontologies, and inference of protein interactions and metabolic pathways represented in the metagenomic data (Table 2).
While an installable workflow like MEGAN4 [29] also exists, MEGAN4 does not provide gene predictions or homology-based or profile-based annotation analyses.
Instead, the pipeline uses ready-made annotation data derived from BLASTX or BLASTP against the NCBI nr database.
Gene finding or gene prediction is a fundamental step for annotation.
Classical gene finders that have been developed for a single genome are unsuitable for metagenomic analysis, because metagenome data are made up of a mixture of sequences from different organisms and often comprise mainly short assemblies and unassembled reads.
Moreover, the high error rate of NGS can lead to frameshifts and make gene prediction more difficult.
For this reason, several dedicated gene prediction programs have been developed, like Meta-Gene [81], MetaGeneAnnotator [82], Orphelia [83], FragGeneScan [84], Glimmer-MG [85], and MetaGenemark [86].
These programs incorporate different models for gene prediction, such as machine learning algorithms [87], HMMs [88], and di-codon usages [89].
MetaGeneMark (GeneMark) uses codon usage-incorporated HMM; Prodigal incorporates a machine learning algorithm [90]; MetaGene incorporates di-codon usage [91]; FragGeneScan uses a sequencing error model and codon usage-incorporated HMM; MetaGeneAnnotator uses a machine learning algorithm and di-codon usage information; Glimmer-MG uses an interpolated Markov model [92]; and Orphelia uses a codon usage-incorporated machine learning algorithm.
Functional analysis of metagenome generally uses a homology-based approach and involves a BLAST [45] search against a database, integrating several individual databases curated for specific analysis [93].
For functional category annotation, COGs [94] and eggnogs [95] databases are used.
COGs, one of the widely used gene categories, was constructed from 66 genomes.
Because COGs has a relatively small size of functional categories appropriate for determining well-known genes and because the database has not been updated for a long time, COGs shows low sensitivity for recently discovered genes.
eggNOGs, created in 2011 and used by the portal service MG-RAST, shows higher sensitivity than COGs, because it is constructed based on the pre-annotation of orthologous groups from 1,133 genomes using COGs and KOGs.
A BLAST-based approach is still widely used in functional category annotation, but it is hampered by its computational complexity and lack of homologous sequences in reference databases [93].
For a protein family analysis, the resources in the Pfam [96] and TIGRfam [97] databases are applied through the use of a HMM-based algorithm, profile (HMM profile), and search tool (HMMER; [98]).
Similarity search results are transferred to annotations, based on the best hit&#39;s information.
The Pfam and TIGRfam databases are resources consisting of curated multiple alignments and HMMs generated from the Sanger Institute and J. Craig Venter Institute, respectively.
A previous study reported that more protein family predictions were available using Pfam, because the Pfam database contains a higher number of protein families (14,831 protein families in Pfam 27.0 release) than TIGRfam database (4,284 protein families in TIGRfam 13.0 release) [93].
While IGM/M, METAREP, and CAMERA use HMM-based databases, such as Pfam and TIGRfam, MG-RAST uses an HMM-independent database, namely FIGfams [99].
FIGfams are sets of protein sequences and alignments that are similar along their full length and are believed to implement the same function generated from National Microbial Pathogen Data Resource (NMPDR).
Gene ontology is assigned using the gene ontology (GO) database in most publically available pipelines.
GO analysis was available in version 3 of MEGAN, but it was replaced by two functional methods, based on the SEED classification and Kyoto Encyclopedia of Genes and Genomes (KEGG) in MEGAN4.
In addition to protein family analysis, MG-RAST offers information on putative protein-protein interactions by applying EMBL&#39;s Search Tool for the Retrieval of Interacting Genes/Proteins (STRING) tool [100].
Metabolic pathway reconstruction is generally performed using the KEGG database [101].
While IMG/M, CAMERA, and MG-RAST use the KEGG database and KEGG graphs, METAREP uses PRIAM [102].
PRIAM is a method for automated enzyme detection, based on the available ENZYME database [103], and provides KEGG graphs for visualization.
Other pathway databases and tools, like MetaCyc [104] and MetaPath [105], are also used to annotate functional roles for metagenomic data but have not yet been integrated into online portal services for metagenomic analysis.
An alternative to the methods mentioned above for inferring metabolic pathways is &#39;subsystem&#39; in SEED [66].
Subsystems represent the collection of functional roles that constitute similar or related forms of complex structural and metabolic pathways, such as glyoxylate bypass, sulfur oxidation, and ribosomal protein paralogs.
The subsystem annotation based on SEED is currently implemented in IMG/M, MG-RAST, and MEGAN4.
Early metagenomic studies focused mainly on investigating less complex ecosystems and specific targets in a given environment due to high sequencing costs, ecosystem complexity, and lack of high-performance computing and bioinformatics tools.
Much work has been done on testing new molecular techniques and tools and discovering novel enzymes and taxonomic lineages rather than testing many scientific hypotheses.
This is why many previous and even current studies are not sufficiently replicated in experimental design.
Of course, it is possible to compare the difference between metagenomic studies, which are not replicated [106].
However, it enables us to obtain statistically meaningful and biologically more relevant conclusions if larger sample sizes with more replicates are provided at a deeper layer of the sequencing regimen.
For inferring statistically meaningful differences between metagenome samples and exploring metabolic/taxonomic diversity in a given sample, metadata should be provided and standardized according to a proper rule.
More standardized formats for describing marker genes, genomes, and metagenome datasets were recently suggested in minimum information about any (x) sequence checklists (MIxS) [107].
This standardized format makes it possible to do many statistical analyses and meta-analyses across metagenomic studies by providing more standardized contextual information about the environment sampled as well as experimental and sequencing information.
Centralizing contextual information will become more common in future metagenomic studies.
Aside from experimental design and contextual data, metagenomic data have inherent limitations that must be overcome in the future.
Metagenomic reads commonly show a relatively low genomic coverage compared to that of a single genome, and the short length of sequencing reads makes only fragmented information by the incomplete assembly and annotation processes accessible.
Initiatives are already under way for filling the gap between metagenomic reads by doing co-assembly with single-cell genomics [108] and joint analysis between multiple metagenomes simultaneously [109], on the assumption that the same species must exist in different samples and that the co-occurrence helps extract shared information.
The ultimate goal of metagenomics is a comprehensive understanding of our ecosystem.
In the near future, metagenomics will be one of the essential parts of viewing our ecosystem through integration with other &#39;-omics&#39; approaches such as metatranscriptomics and metaproteomics.
Over the last few years, metagenomics has accelerated the understanding of microbial ecology and evolution, thanks to the technical advances of sequencing platforms.
Metagenomics itself is not a panacea that is able to answer all unresolved questions and uncover the &#39;dark matter&#39; of our knowledge in microbial diversity and function studies, but it is undoubtedly an excellent tool, giving insight into completing the jigsaw puzzles of our understanding of the surrounding biosphere by exploring both &#39;who is out there&#39; and &#39;what they do&#39; in nature.
It is true that studying metagenomics is still recognized by ordinary microbial ecologists as a tool that is difficult to tackle due to its complexity and the enormous amount of sequence data that needs to be dealt with.
However, it is not surprising that we will witness &#39;doing metagenomics&#39; in the future as if we routinely do PCR and gel electrophoresis in our laboratory.
Current development of tools and databases for metagenomic studies is in its infancy, and there are still many challenges to overcome.
It will not take long to see an explosive growth of metagenomics with a revolution of metagenomic bioinformatics.
Overall workflow and bioinformatics tools for shotgun metagenomic analysis.
HMM, hidden Markov model; KEGG, Kyoto encyclopedia of genes and genomes; STRING, Search Tool for the Retrieval of Interacting Genes/Proteins.
Title: Bioinformatic resources for studying targeted metagenomics

Title: Comparison of five major resources for metagenomic functional annotation
Title: Bioprospecting Potential of the Soil Metagenome: Novel Enzymes and Bioactivities

The microbial diversity in soil ecosystems is higher than in any other microbial ecosystem.
The majority of soil microorganisms has not been characterized, because the dominant members have not been readily culturable on standard cultivation media; therefore, the soil ecosystem is a great reservoir for the discovery of novel microbial enzymes and bioactivities.
The soil metagenome, the collective microbial genome, could be cloned and sequenced directly from soils to search for novel microbial resources.
This review summarizes the microbial diversity in soils and the efforts to search for microbial resources from the soil metagenome, with more emphasis on the potential of bioprospecting metagenomics and recent discoveries.
In 1898, a microbiologist, Heinrich Winterberg, first described that there was a discrepancy in the number of microorganisms between culturable bacteria on nutrient media and the total bacteria in nature counted by microscopy.
Since then, microbial unculturability, the so-called &#39;great plate count anomaly,&#39; has long been recognized in microbiology [1].
While microbial unculturability is not fully understood so far, microbial diversity has been analyzed extensively in various environments.
This was mostly due to the advance of a molecular microbial ecology.
The development of culture-independent analysis adopted amplification and DNA sequencing of microbial signature rRNA sequences from nature without cultivation of entire microbial species [2].
Therefore, analysis of the microbial phylogenetic marker genes, such as the 16S rRNA gene, revealed that microorganisms are the true dominant organism in nature [3].
The number of prokaryotic cells was estimated to be 4-6 × 1030 cells on Earth, and the prokaryotes represent the largest pool of carbon, nitrogen, and phosphorus among living organisms [4].
Microbial diversity has also been recognized from various soils, and only a small fraction of total microbial species in soil (i.e., less than 1%) has been characterized by cultivation-based methods [5, 6].
Since the majority of unculturable bacteria in soil has not been cultured [7], it has not been functionally characterized.
The microbial diversity in soil is probably highest compared to other microbial communities.
Culture-independent analysis of microbial diversity in soil revealed that most bacterial members abundant in soils are members of the phyla Proteobacteria, Acidobacteria, Actinobacterium, Firmicutes, and some Verrucomicrobia [8, 9], showing that a large number of the 16S rRNA gene clones originated from uncultured bacterial species.
The abundance of the members was somewhat variable in different types of soil.
For example, the forest soils retained higher members of the phylum Acidobacteria but fewer β-proteobacterial members [10].
On the other hand, agricultural soils harbor higher numbers of members in β-proteobacteria but less Acidobacteria.
The interesting result recently reported by Mendes et al.
[11] indicated that the relative abundance of bacterial phyla, not their profiles, could be important for microbial community function in a specific soil, such as disease-suppressive soil.
The metagenome is the total microbial genome isolated directly from microorganisms in nature.
The first proposal of the term &#39;metagenome&#39; was by Handelsman et al.
[12], while the first approach using environmental DNA was reported in 1995 to search for genes encoding cellulases [13].
The soil metagenome can be isolated directly from various types of soil either directly or indirectly without bacterial cultivation [14-16].
While the microbial diversities of various microbial habitats are being actively investigated by taking advantage of next-generation sequencing technology to analyze a large number of 16S rRNA gene amplicon sequences derived from a variety of soils [8], the advances in bioprospecting metagenomics are relatively slow.
As a bioprospecting purpose, the purified soil metagenomic DNA would be cloned into several different plasmid vectors.
In general, high-molecular-weight DNAs cloned into fosmid, cosmid, or bacterial artificial chromosome (BAC) vectors are introduced into a surrogate host bacterium, such as Escherichia coli [17].
Especially, the fosmid-based preparation of a metagenomic library is the most frequently used strategy, due to their high cloning efficiency, the improved stability in E. coli, and the feasibility to construct a medium-sized (40 kb)-insert DNA library [18].
Considering the low frequency of finding genes for novel bioactivities from a soil metagenomic library [19], the cloning efficiency is a crucial factor to construct a large clone member library.
One of the most critical steps for successful library construction is the isolation of pure high-molecular-weight DNA, which is suitable for cloning into the vectors [8, 20].
However, high contents of soil-derived humic acids are co-purified during the high-molecular-weight DNA isolation process, and humic acids interfere with the cloning process of high-molecular-weight DNA into a vector.
Various methods to remove humic acids during soil metagenomic DNA purification usually end up with the recovery of relatively low-molecular-weight DNA, which is not suitable to clone into a fosmid or BAC.
Several commercial kits would be plausible to purify soil metagenomic DNA for the amplification purpose of 16S rRNA genes, which will be subsequently used for microbial diversity analysis.
However, the sizes of DNA fragments isolated by most of the commercial kits are rather small to clone into vectors, such as fosmids and BACs.
Therefore, optimal methods for the direct isolation of high-molecular-weight DNA from soil have been developed and used for bioprospecting metagenomics [14, 16, 20].
Although obtaining a large number of metagenomic clones from soil is a prerequisite to search for novel microbial genes from soil, the technologies for metagenomic library construction were generalized using fosmids and cosmids.
Various novel microbial resources could be expected from soil metagenomic libraries, because soils bear the highest microbial diversity compared to any other microbial community.
Title: Tapping soil metagenome

Since functional metagenomics (or bioprospecting metagenomics) was attempted from a soil ecosystem [17], a number of microbial genes encoding novel enzymes or bioactive compounds have been identified and characterized [21, 22].
Here, we summarize several technical points to be considered for bioprospecting metagenomics and the major results of novel enzymes and bioactivities from the soil metagenome.
Title: Screening strategies and high-throughput screening (HTS)

Metagenomic libraries with either large-insert DNA or small-insert DNA were used to search for novel microbial genes.
The library with the small-insert DNA could be feasible to identify novel enzymes, while that with the large-insert DNA is also frequently used [23].
Screening of both types of metagenomic libraries has been conducted and successfully identified many novel enzymes and bioactivities [21].
Two different approaches for screening the metagenomic libraries are frequently used, such as function-based (expression-dependent) screening and sequence-based (homology-dependent) screening.
Function-based screening is dependent on the expression of metagenomic genes in a surrogate host, such as E. coli, and subsequently detects metabolic activities in the heterologous host.
Therefore, the proper expression of metagenomic genes is essential to detect the phenotypic characteristics of the desired activity by the function-based screening approach.
It is arguable that the majority of genes from uncultured soil microorganisms would not be expressed properly in a host bacterium.
There has been no extensive analysis to assess the expression ratio of metagenomic genes in defined culturable bacteria.
In spite of the expression barrier of cloned metagenomic genes in E. coli, the function-driven approach has proven to be feasible to search for novel genes and gene products [22].
Recently, function-based screening approaches have been advanced and modified to search for novel microbial genes compared with the original approach-i.e., the direct detection of metabolic function [17].
Host strains or mutants were used to identify genes for functional complementation in trans by metagenomic clones [24], and another advance in function-based screening was combined with HTS to detect the induced gene expression in a host bacterium with substrates, autoinducers, and metabolic products provided exogenously [25-27].
One of the approaches to overcome the host expression barrier is to develop a broad-host range vector to express a metagenomic library in a variety of host bacteria.
This approach, in fact, is promising and nourishing the function-based screening of metagenomics to obtain diverse microbial resources for enzymes and biomolecules [28].
The sequence-based approach is to screen for marker genes using DNA probe or PCR primers designed from DNA sequences of already known genes.
Since this approach would detect gene variants with the conserved motif from the metagenome, it is arguable if this approach would really screen true novel genes from microbial diversity.
Nevertheless, there are a number of examples with some successful identification of novel microbial enzymes [22, 29].
A recent approach with sequence-based screening is taking advantage of the advances in next-generation sequencing.
Direct sequencing of metagenomic DNA by next-generation sequencing is now used increasingly to study gene inventories in the metagenomes of specific microbial communities [30].
Direct sequencing of a metagenome generates a large dataset, which should be analyzed by support of the appropriate informatics.
Analysis of the soil metagenome by the sequencing-based approach is still a huge challenge, since the soil microbial community is the most complicated one with the highest microbial diversity compared with any other microbial community.
A combination of appropriate bioinformatics, such as comparative sequence analysis, with the large set of soil metagenome sequences will be a new opportunity to search for microbial enzymes and novel bioactivities in the future [31].
HTS is another issue for bioprospecting metagenomics with the direct sequencing of the metagenome.
If both function-based and sequence-based screening is combined with HTS, novel microbial enzymes and bioactivities would be detected at higher frequency compared with conventional screening.
This is mainly because the large numbers of metagenomic clones in libraries should be searched for novel enzymes and biomolecules to tap the great soil microbial diversity.
Some of the recently adopted HTS methods for drug discovery may enable the discovery of novel enzymes and biomolecules from the soil metagenome.
Multidisciplinary efforts, combined with bioinformatics, analytical chemistry, and high-throughput technology, is required to search for novel enzymes and bioactivities from soil metagenomes.
Title: Novel enzymes from soil metagenome

Many novel enzymes were identified from various metagenomic studies, and we summarized the examples of enzyme recovery from soil metagenome studies in Table 1 [10, 32-58].
One of the most prevailing novel enzymes found from the soil metagenome is esterase/lipase.
Lipolytic enzymes, such as esterases and lipases, are important biocatalysts for biotechnological applications.
The interesting features of lipolytic enzymes include no requirement for cofactors, remarkable stability in organic solvents, broad substrate specificity, stereoselectivity, and positional selectivity [23].
The features of lipolytic enzymes are especially attractive for organic synthesis if the enzymes with specific chiral resolution could be retrieved from the soil metagenome.
Therefore, the discovery of novel lipolytic enzymes from the soil metagenome has received much attention compared to other enzymes.
Considering that lipolytic enzymes are the most well characterized and highly studied enzymes among biocatalysts, the finding of several novel families of esterase/lipase enzymes is quite surprising.
One of our previous results with EstD2 revealed that EstD2 was not similar to any previously described enzymes, while the enzyme clearly displayed esterase activity [39].
A similar protein found in the GenBank database also exhibited the same activity, while it was initially annotated as a hypothetical protein.
Therefore, function-driven selection of EstD2 constituted a novel family of esterases and re-annotated many proteins as esterase/lipases that were previously known as hypothetical proteins.
Similar studies with enzymes found in the soil metagenome suggested that soil microbial diversity truly bears the great extent of novel microbial enzymes.
Cellulolytic enzymes and cell wall-degrading enzymes (CWDEs) of plant cells are also enzymes with biotechnological interest, mostly due to biomass degradation, with the purpose of bioenergy production.
In contrast to the enzymatic interest, the identification of novel cellulolytic enzymes or other CWDEs is not well documented.
The reason for the rare recovery of cellulases and CWDEs is poor secretion of these enzymes in E. coli, the major host bacteria, and incompatible detection methods of enzymatic activity for a large member of metagenomic libraries from the soil metagenome.
For the successful detection of cellulases and CWDEs, technical elaboration with HTS would be necessary to enhance the detection of proper enzyme activity.
Bioinformatics, such as comparative sequence analysis of the metagenome, could be another choice to identify cellulolytic enzymes and CWDEs from massive datasets derived from direct sequencing of the soil metagenome.
A number of enzymes for other carbohydrate metabolism and a few lactonases were also detected from the soil metagenome, and they are listed in Table 1.
However, many enzymes are not feasible to detection by simply screening for the bacterial phenotypic changes, while the lipolytic enzymes were simply identified by tributyrin hydrolysis on culture medium.
Title: Bioactivities from soil metagenome

In addition to the novel enzymes, soil metagenomes are rich sources of a variety of small molecules with bioactivities, such as antibiotics and other pharmaceutically applicable activities.
The search for novel bioactivities is primarily based on the phenotypic detection of bacterial traits of the host bacteria bearing metagenomic libraries.
Antagonistic activity and colony color changes of the host bacterium are typical examples to detect metagenomic clones, potentially associated with novel secondary metabolite production.
However, the conventional approach exhibits a rarer chance to identify novel gene clusters and bioactivities [19].
Therefore, it is necessary to develop new detection strategies for novel bioactivities from the soil metagenome.
Nonetheless, the simple detection of antibiosis or color changes of the host bacterial colonies with metagenomic clones brought some interesting results, suggesting the great potential of novel bioactivities from the soil metagenome.
We summarize the identified bioactivities and their gene clusters from the soil metagenome in Table 2.
These bioactivities include antibacterial turbomycins, glycopeptides, cyanobactins, type II polyketides, trans-acyltransferase polyketides, and the anticancer agent ET-743 [21].
Some of the bioactivities from the soil metagenome may also be discovered by homology-based screening.
Whole metagenome sequences from soil, which will be accomplished in the future, would be a rich source to be probed to identify gene clusters for bioactive compound production.
The soil metagenome, a rich microbial source, is still unexplored, since major microbial species have not been cultured and characterized from various soils.
Both function-driven metagenomics and homology-dependent metagenomics have started out in soils and brought some successful discovery of novel microbial enzymes and secondary metabolites.
Further analysis of soil microbial communities and the appropriate enrichment of target resources from soils will increase the chance to discover novel microbial enzymes and bioactivities from the soil metagenome.
In addition, the adoption of HTS, the advances in sequencing technology, and the proper bioinformatics would be necessary to improve the efficacy of bioprospecting metagenomics of soils for the discovery of novel enzymes and bioactivities.
Title: Microbial enzymes identified from bioprospecting metagenomics from various soils

Title: Microbial bioactivities identified from bioprospecting metagenomics from various soils
Title: Comparative Viral Metagenomics of Environmental Samples from Korea

The introduction of metagenomics into the field of virology has facilitated the exploration of viral communities in various natural habitats.
Understanding the viral ecology of a variety of sample types throughout the biosphere is important per se, but it also has potential applications in clinical and diagnostic virology.
However, the procedures used by viral metagenomics may produce technical errors, such as amplification bias, while public viral databases are very limited, which may hamper the determination of the viral diversity in samples.
This review considers the current state of viral metagenomics, based on examples from Korean viral metagenomic studies-i.e., rice paddy soil, fermented foods, human gut, seawater, and the near-surface atmosphere.
Viral metagenomics has become widespread due to various methodological developments, and much attention has been focused on studies that consider the intrinsic role of viruses that interact with their hosts.
Viruses are the most abundant and diverse biological entities in the biosphere, and their global numbers have been estimated at 1.2 × 1030 and 2.6 × 1030 in the ocean and soil, respectively [1].
Thus, viruses are key elements that contribute to the life cycles of cellular organisms [2].
However, the study of viral ecology in natural habitats has been limited due to the difficulties of viral culture [3].
The lack of a ubiquitous marker gene, such as the 16S rRNA gene shared by all bacteria and archaea, also hampered our understanding of the genetic diversity of viruses, prior to the introduction of metagenomics into the field of viral ecology [1, 4].
Recently, viral metagenomics has enabled researchers to explore the community structure and diversity of viruses in various natural ecosystems [5].
This methodology depends on a priori knowledge of the viral types that may be present [6, 7].
The first viral metagenome of uncultured marine viral communities was published in 2002 [8], and there have been many subsequent advances in the methodologies (e.g., methods for amplifying the initial viral genomes) and tools used for bioinformatics analysis in viral metagenomics.
These technical developments have facilitated explorations of the abundance and diversity of viruses from a wide range of natural habitats in Korea.
In Korea, viral metagenomics was applied for the first time to unique samples, including fermented foods and atmospheric samples, as well as habitats where viruses are expected to have significant roles, such as rice paddy soil, seawater, and the human gut.
This review describes recent advances in viral metagenomics and provides summaries of studies that have been conducted to characterize Korean viral metagenomes.
In addition, the advantages and disadvantages of the most widely used viral DNA amplification methods are discussed, based on empirical knowledge.
Further directions for the study of virus-host interactions are also highlighted.
Viral metagenomics is the study of viral metagenomes (known as viromes), which are obtained directly from environmental samples using viral particle purification and shotgun sequencing.
Viral metagenomic studies have increased gradually since the expansion of metagenomic approaches to viral ecology; i.e., over 200 (61 reviews) investigations of the viral communities in environmental samples have been published (Fig.1) [8-13].
In 2002, Breitbart et al.
[8] used shotgun library sequencing and reported that the majority of the sequences in marine viral metagenomes shared no similarity with any genes in public databases, which suggested that most environmental viruses remained uncharacterized.
This was the first study using viral metagenomic approaches in the field.
Subsequently, many studies have surveyed the viral diversity of unexplored habitats using viral metagenomics.
Cultivation in a host is usually necessary to obtain a virus from the environment that is being investigated.
However, the application of metagenomics techniques based on genetic information can circumvent this obstacle [14].
Based on the physical characteristics of virions, viral particles can be isolated from environmental samples, which are enriched using a combination of size filtration (e.g., &lt;0.22 µm) and density gradient centrifugation (e.g., 1.35-1.5 g/mL cesium chloride) [14], so that the virome can be obtained from the purified viral particles.
There is a lack of evolutionarily conserved genes, such as the prokaryotic 16S ribosomal RNA gene, in viral genomes.
Therefore, the fragmented viral metagenomic sequences obtained by whole-genome shotgun DNA sequencing are used to analyze the viral ecology instead.
Viral genomes are small (on average, they comprise a few to several dozen kilobases); so, valuable genome coverage can be achieved easily by DNA sequencing.
The advent of high-throughput sequencing techniques, such as 454 pyrosequencing and Illumina sequencing, makes it easier to achieve suitable genome coverage at low cost.
Indeed, over 90 of either nearly complete or complete novel viral genomes were assembled using these methods in recent studies [9, 10, 15-20].
However, it is inevitable that an amplification step is necessary for small viral genomes prior to DNA sequencing.
Linker amplified shotgun library (LASL) and multiple displacement amplification (MDA) are the main methods used to amplify viromes in viral metagenomics.
The LASL method, developed by Breitbart et al.
[8], PCR-amplifies a virome after adapter attachment to randomly fragmented viral DNAs.
The application of adapter attachment is restricted to double-stranded DNA sequences; so, only double-stranded DNA (dsDNA) viruses can be detected using the LASL method.
Thus, the dominance of dsDNA tailed bacteriophages was reported initially in uncultured marine viral assemblages [8], as well as other environmental samples, such as kimchi [11], aquatic water [8], and feces [21].
The MDA technique [22] amplifies DNA isothermally using the phi29 polymerase and random hexaprimers and has been used before in microbiology [22].
The high amplification efficiency of this method means that it is suitable for amplification during viral metagenomics applications.
Particularly, the phi29 polymerase of the MDA technique selectively amplifies circular genomes (estimated at 100 times) [10], and it has facilitated the discovery of abundant single-stranded DNA and RNA viruses in environmental samples [23, 24] (described in detail below).
The analysis of viral metagenome data using bioinformatics is one of the most challenging aspects of viral metagenomics.
One million to 1 billion reads of viral metagenomic sequences are typically generated by high-throughput sequencing platforms (with an average read length of 350-400 bp using 454 GS-FLX Titanium and 2 × 150 bp using Illumina HiSeq 2500).
After removing any low-quality, redundant, and chimeric sequences, the viral sequences are compared to sequences in public databases (e.g., the GenBank non-redundant nucleotide database, MG-RAST, and CAMERA) using BLAST [25] or USEARCH [26].
The identification of viral sequences based on significant amino acid similarity (E-value of &lt;10-3) was first described by Breitbart et al.
[8], and it has since been extended to the exploration of environmental viromes, although the E-score applied to viral metagenomic studies appears to be regarded as &#34;a loose standard&#34; [27].
Most of the environmental viromes detected by viral metagenomics are defined as orphan (unassigned) sequences.
The majority of viral sequences shares no amino acid similarity with previously observed genes (average 40% to 50%, occasionally up to 90%, of sequences); so, they are characterized as &#34;unknown&#34; [5].
Comparisons of viral sequences with the data in public databases have demonstrated that little is known about environmental viruses.
Thus, the majority of the unassigned sequences in viral metagenomes is often regarded as &#34;junk sequences&#34; due to a lack of suitable bioinformatics tools and viral databases for their characterization [1, 28].
At present, the viral databases are biased toward animal and plant viruses, although viruses that infect prokaryotes (bacteriophages) are sparsely represented.
Most of the latter are restricted to phages that infect bacteria belonging to the phyla Proteobacteria, Firmicutes, and Actinobacteria [29, 30].
Moreover, even &#34;known&#34; viral sequences share low amino acid similarities (&lt;50%) with viral protein sequences [9, 11, 13, 31]; so, the majority of environmental viruses representing novel viral species and their viral diversity is much greater than considered previously.
The observation of a high percentage of ORFans (open reading frames with no homologs in known genes in the databases) in viral genomes [32] also supports the novelty of environmental viromes.
Thus, researchers could discover novel viruses in orphan sequence pools that currently remain &#34;untapped resources.&#34;
These findings indicate our current lack of knowledge about viral genetic information and emphasize the need for physiological studies of viruses to understand viral ecology based on genomic data.
Using viral metagenomic approaches, viral diversity and abundance have been investigated in various natural ecosystems in Korea, including rice paddy soil, fermented foods, the human gut, seawater, and the near-surface atmosphere (Table 1).
The morphologies of environmental viruses have been imaged using transmission electron microscopy (Fig.2).
Sipho- and podo-like tailed viruses were found in fermented foods (Fig.2A-2C), while non-tailed small viruses were detected in the near-surface atmosphere (Fig.2G-2I).
Various types of tailed, non-tailed, circular, and long linear viruses were found in the human gut (Fig.2D-2F).
In general, the virions ranged in size from 30 to 60 nm.
In agreement with the results of previous viral metagenomic studies [23, 31], over half of the sequences in the Korean environmental viromes were described as orphan sequences, based on comparisons with viral proteins in public databases (Table 1).
Most of the sequences identified were assigned to the Siphoviridae, Podoviridae, and Myoviridae families of dsDNA viruses and the Circoviridae, Germiniviridae, Nanoviridae, and Microviridae families of ssDNA viruses (Fig.3).
The first viral metagenomic study in Korea surveyed uncultured viral assemblages in rice paddy soil in 2008 [10], where MDA was used with phi29 DNA poly merase and random hexaprimers to amplify viral DNA and to construct clone libraries for metagenome sequencing.
The soil was found to contain a rich pool of unknown ssDNA viruses and dsDNA viruses.
This study also focused on the effect of MDA amplification on different types of genomic DNA and showed that MDA preferentially amplified circular DNA genomes.
This was also demonstrated using an environmental sample from surface seawater [12], where dsDNA viruses alone were retrieved in the LASL library, whereas ssDNA viruses were overwhelmingly represented in the MDA library.
Thus, the amplification methods used in viral metagenomics can affect the ratios of viral sequences greatly and lead to inaccurate estimates of viral diversity.
Next, Park et al.
[11] investigated the abundance and diversity of uncultured viral assemblages in fermented shrimp, kimchi, and sauerkraut-fermented foods that have been consumed for a long time around the world.
In contrast to the soil virome, dsDNA bacteriophages from the families Myoviridae, Podoviridae, and Siphoviridae dominated the fermented foods, and they contained a low complexity of viral assemblages compared with other environmental habitats, such as seawater, human feces, marine sediment, and soil.
However, it is possible that the viral diversity of the viromes detected in fermented foods may have been constrained to dsDNA viruses by the LASL method.
A large number of unknown microbes, such as bacteria, archaea, microbial eukarya, and viruses, constitute up to 1012 bacteria per gram of feces in the human gastrointestinal tract [33], and it is expected that gut viruses will affect the relationships among viruses, bacteria, and gut epithelial cells [34].
Kim et al.
[13] investigated the abundance and diversity of DNA viruses in fecal samples from five healthy Koreans, particularly ssDNA viruses.
Using epifluorescence microscopy with SYBR Gold staining [35], the viral abundance ranged from 108 to 109 per gram of feces, which was 10-fold less than the bacterial abundance in many other environments that harbored 10-100-fold more viruses (e.g., aquatic environments).
Moreover, the diversity of gut viral assemblages was lower than that of gut bacteria.
These results support Reyes et al.
[29], who found that viral-microbial interactions in the human intestine could not be described as a predator-prey relationship, and instead, it was referred to as &#34;kill the winner,&#34; which was driven by a lytic life cycle.
Airborne viruses are now regarded as major environmental risk factors for complex disease pathogenesis [36-38].
However, the atmosphere remains &#34;one of the last frontiers of biological exploration on Earth&#34; [9].
Using viral metagenomics with an advanced airborne particle sampling system, Whon et al.
[9] conducted the first study of the diversity and community composition of airborne viruses in the near-surface atmosphere.
The viral abundance in the atmosphere exhibited seasonal changes (increasing from autumn to winter before decreasing until spring) in the range of 106 to 107 viruses per m3, and the temporal variations in viral abundance were inversely correlated with seasonal changes in temperature and absolute humidity.
Plant-associated ssDNA geminivirus-related viruses and animal-infecting circoviruses dominated the viral assemblages, with low numbers of nanoviruses and microphages in air viromes, which suggests that airborne viral assemblages are affected greatly by terrestrial plants and animal activities.
The compositions of viral assemblages are determined by how the virome is amplified.
Thus, the compositions of the viral assemblages detected in fermented foods and marine samples are biased toward dsDNA viruses, such as sipho-, podo-, and myophages, when the LASL method is used, whereas the viral assemblages detected in rice paddy soil, human gut, marine, and near-surface atmosphere samples contain high proportions of ssDNA viral sequences, due to the use of MDA, as shown in Fig.3.
In addition, the compositions of the viral assemblages characterized in Korean environments tend to depend on their specific microbial features.
The human gastrointestinal tract and fermented foods are exposed to massive numbers of gut bacteria and lactic acid bacteria, respectively [39-41].
By contrast, the atmosphere contains far less cellular metabolism and reproductive activity than other environments, such as the soil, seawater, fermented foods, and the human gut [42].
On this basis, the lowest abundance of eukaryotic viruses was observed in the viral assemblages in the human gut and fermented foods, whereas a high abundance of eukaryotic viruses was detected in the near-surface atmosphere.
High levels of prokaryote and eukaryote cells are present in rice paddy soil [43] and seawater [44], and so, comparable amounts of bacteriophages and eukaryotic viruses were detected in their viromes.
The development of viral metagenomics has facilitated the discovery of novel, previously undescribed viral species.
An artifact of the MDA method is that it selectively amplifies the circular genomes of ssDNA viruses, so that a large number of ssDNA viral sequences have been identified in environmental viromes.
Thus, there is great interest in the distribution and host range of ssDNA viruses.
In particular, microphages from the family Microviridae have been identified in a wide range of environments [13, 45, 46].
In contrast to the ecology of marine dsDNA phages, marine ssDNA phages in the family Microviridae have distinct spatial and temporal distributions [45].
ssDNA microphages were abundant in the healthy human gut and their genotypes were much more diverse than those reported previously [13].
Moreover, prophage-like elements in the genomes of gut microbes, such as Bacteroides and Prevotella spp., were characterized as a novel subgroup in the family Microviridae [13, 47], while viral sequences from the human gut were clustered with prophage-like elements from Bacteroides and Prevotella spp.[13].
This suggests that Bacteroides and Prevotella spp.are included within the host range of the ssDNA microphages in the human gut [47].
Eukaryotic ssDNA viruses that infect plants and mammals have been identified in many environmental viromes.
Circoviruses that are known to infect birds and pigs have also been identified in the viromes of invertebrates and a fish [48, 49], while geminiviruses that cause plant diseases have been detected in whiteflies, which act as insect vectors of plant viruses [50].
A recent study by Whon et al.
[9] investigated airborne DNA viral assemblages in near-surface atmosphere samples and showed that a high number of viruses (log 6 to 7 viruses per m3) were present in the air, which were dominated by geminivirus-related viruses.
These viruses were identified as plant fungal pathogen-infecting mycoviruses-i.e., Sclerotinia sclerotiorum hypovirulence-associated DNA virus 1-which indicates that the airborne viral assemblages in the near-surface atmosphere may have strong interactions with plants.
These results highlight the extensive distribution of ssDNA viruses in a wide range of environments, and their host ranges may be wider than previously recognized.
Thus, the discovery of novel genomes of ssDNA viral families in metagenomic studies could revolutionize our knowledge of the ecology and evolution of ssDNA viruses.
In the last decade, viral ecologists have focused on community-level analyses of viruses to understand their abundance and genetic diversity in specific environments.
The ecological effects of viruses, particularly bacteriophages, are known to control host populations via the &#34;kill the winner system,&#34; while they drive mortality and evolutionary change in microorganisms via lateral gene transfer by infecting their host bacteria, although the basic issue of &#34;who infects whom&#34; is poorly understood [2, 29, 51, 52].
In the ocean, for example, viruses regulate the microbial abundance, release dissolved organic matter, and affect global biogeochemical cycles by killing up to 40% of host bacteria per day [53, 54].
In contrast, symbiotic functions of viruses, such as host survival, competition, and protection from pathogenic infections, are beginning to be understood [55], and evidence for a beneficial interaction in phage-host interactions was found in the mammalian gut ecosystem [29, 56].
When host survival is threatened, a variety of environmental factors can trigger prophage induction, and the liberated prophages may become completely virulent [57].
Overall, these studies suggest that prophage induction may responsible for triggering dysbiosis and changes in the microbial population by altering host phenotypes, thereby leading to a new environmental niche.
Traditionally, host culture-dependent techniques, such as plaque assays, have been widely used for the identification of phage and host bacteria interactions.
However, plaque assays require isolated host bacteria; so, they are low-throughput methods.
This method is also difficult to apply to environmental samples where lysogenic infections are prevalent, because the method relies on observations of visible plaque formations, which are often absent from lysogenic infections [3, 58, 59].
Recently, Deng et al.
[52] demonstrated a new technique, known as &#34;viral tagging,&#34; for identifying the interactions between cultivated host bacteria and their phages, which used the nucleic acid stain SYBR Gold to generate fluorescently labeled phages, so that the host cells fluoresced with viral tagging, thereby allowing the sorting of virus-tagged cells by flow cytometry [52, 60].
This emerging technique is undoubtedly helpful for not only exploring virus-host interactions in their natural habitats when the method is combined with other experimental tools, such as single viral genomics [61] and phageFISH [62], but also identifying viral receptors in macro-organisms (e.g., the mammalian gut) if the method is combined with a fluorescently labeled receptor protein during histological examinations.
The emergence of viral metagenomics has facilitated advances in virology and allowed us to understand novel aspects of viral ecology.
At present, viral metagenomics is a powerful and sensitive technique for detecting viruses that cannot be identified by traditional culture- and sequence-based approaches.
Most importantly, viral metagenomics suggests that novel viruses interact constantly with the human population.
Thus, viral metagenomics can facilitate the improved surveillance of viral pathogens in the fields of public health and food security.
This technique can be used to understand viral ecology by exploring the environmental viromes that are generated by viral metagenomics.
Overview of viral metagenomic studies conducted between 2002 and 2013.
The cumulative number of publications determined by PubMed searches using the keywords &#39;(virus OR viral OR virome) AND (metagenome OR metagenomics)&#39; is shown on the y-axis.
The first viral metagenome and Korean metagenomes (highlighted by arrows), as well as the technical development of high-throughput sequencing platforms (below the highlighted arrow), are shown with the year of publication and/or public release.
Transmission electron micrographs showing the different morphologies of virus particles found in viral samples from Korea.
(A-C) Fermented foods.
(D-F) Human feces.
(G-I) Atmosphere.
Comparison of the compositions of viral assemblages identified in Korean environments.
The percentages of viral families in each virome indicate the assignments of the metagenomic sequences to viral families.
The amplification methods used in each study are designated as follows: M, multiple displacement amplification; L, linker-amplified shotgun library.
Title: Published Korean viral metagenomics
Title: Genome-Wide Association Study of Orthostatic Hypotension and Supine-Standing Blood Pressure Changes in Two Korean Populations

Orthostatic hypotension (OH) is defined by a 20-mm Hg difference of systolic blood pressure (dtSBP) and/or a 10-mm Hg difference of diastolic blood pressure (dtDBP) between supine and standing, and OH is associated with a failure of the cardiovascular reflex to maintain blood pressure on standing from a supine position.
To understand the underlying genetic factors for OH traits (OH, dtSBP, and dtDBP), genome-wide association studies (GWASs) using 333,651 single nucleotide polymorphisms (SNPs) were conducted separately for two population-based cohorts, Ansung (n = 3,173) and Ansan (n = 3,255).
We identified 8 SNPs (5 SNPs for dtSBP and 3 SNPs for dtDBP) that were repeatedly associated in both the Ansung and Ansan cohorts and had p-values of &lt;1 × 10-5 in the meta-analysis.
Unfortunately, the SNPs of the OH case control GWAS did not pass our p-value criteria.
Four of 8 SNPs were located in the intergenic region of chromosome 2, and the nearest gene (CTNNA2) was located at 1 Mb of distance.
CTNNA2 is a linker between cadherin adhesion receptors and the actin cytoskeleton and is essential for stabilizing dendritic spines in rodent hippocampal neurons.
Although there is no report about the function in blood pressure regulation, hippocampal neurons interact primarily with the autonomic nervous system and might be related to OH.
The remaining SNPs, rs7098785 of dtSBP trait and rs6892553, rs16887217, and rs4959677 of dtDBP trait were located in the PIK3AP1 intron, ACTBL2-3&#39; flanking, STAR intron, and intergenic region, respectively, but there was no clear functional link to blood pressure regulation.
Orthostasis causes a gravitational shift in circulating blood from the intra-thoracic space to lower extremities and reduces venous return to the heart.
This transient increase in sympathetic tone and decrease in parasympathetic tone and these normal cardiovascular reflexes cause an increase in heart rate and vascular resistance, restoring normal blood pressure (BP) and cardiac output [1].
Orthostatic hypotension (OH) is the failure of the cardiovascular reflexes to maintain BP on standing from a supine position [2].
The Malmö Preventive Project reported that individuals with OH had significantly increased all-cause mortality (hazard ratio, 1.4 to 1.6) [3].
Title: OH has been defined by the international consensus as a decrease in systolic BP ≥ 20 mm Hg and/or a decrease in diastolic BP ≥ 10 mm Hg within 3 mintues of standing [4].
The prevalence of OH varies from 5% to 30%, depending greatly on the population being studied [5-7].
Aging is associated with a definite increase in the prevalence of OH and was reviewed in Benvenuto and Krakoff [2].
In addition to the aging effects, OH is more prevalent in hypertensive [8, 9], neurological [10], and diabetic patients [11, 12].
The all-cause mortality rate was higher in patients with OH than in those without OH [6, 7]

We have been conducting an epidemiology study of OH as part of the Korean Genome and Epidemiology Study (KoGES) and previously reported that middle-aged adults (40-70 years), enrolled from 2001 to 2002, had a 12.3% overall prevalence of OH and that the OH frequency increased significantly with age, from 6.4% in those aged 40-44 years to 23.1% in those aged 65-69 (p &lt; 0.001) [9].
In addition, the study confirmed the correlations between OH and hypertension, body mass index (BMI), and type 2 diabetes [9].
The genetic contribution to OH can be hypothesized that the frequency of OH in families with a history of essential hypertension is higher than in the general population [4].
A genetic association study was conducted by Luo et al.
[13], and they identified a functional variant of NEDD4L (epithelial sodium channel E3-ubiquitin ligase).
To identify genetic factors associated with OH trait in Koreans, we conducted genome-wide association studies (GWASs) for OH case controls using genome-wide single nucleotide polymorphism (SNP) genotypes and also tested supine and standing BPs as quantitative traits.
Title: Study subjects

The study subjects consisted of two population-based cohorts, Ansung and Ansan, which have been conducted as part of the KoGES.
The phenotype of the cohort population was previously described [14].
Briefly, the subjects came from the Ansung and Ansan cities, located in Gyeonggi-do province near Seoul, Korea.
Written informed consent was obtained from all participants, and this research project was approved by the institutional review board in the Korea National Institute of Health (KNIH).
Subjects with genotype accuracies below 98% and high missing genotype call rates (≥4%), high heterozygosity (&gt;30%), or inconsistency in sex were excluded from subsequent analyses.
Individuals who had a tumor were excluded, as were related individuals whose estimated identity-by-state values were high (&gt;0.80).
After these quality control steps, 8,842 samples were selected.
Title: Clinical characteristics

BP was measured 3 times in the supine position.
Before the first measurement, participants rested for 5 minutes, and the 3 measurements were taken in one arm showing the higher BP at least 3 minutes apart.
After the supine position measurement, the BP was measured soon after standing.
Two of 8,842 subjects did not undergo BP measurement in the standing posture.
Also, 1,291 individuals who had been treated with anti-hypertensive drug therapy and 1,114 self-reported diabetic patients or diagnosed diabetic patients by KoGES were excluded.
Ultimately, 3,255 individuals from Ansan and 3,173 individuals from Ansung were used for the GWASs.
Other cardiovascular risk factors, such as cholesterol level and fasting glucose level, were measured from blood samples after overnight fasting.
Title: Study genotypes

The genotyping of the cohort population was previously described for the Korea Associated Resource (KARE) study [14].
Most DNA samples were isolated from the peripheral blood of participants and genotyped using the Affymetrix Genomewide Human SNP array 5.0 (Affymetrix, Inc., Santa Clara, CA, USA).
The quality control steps of genotypes have been described elsewhere [14].
Briefly, the accuracy of the genotyping was determined by Bayesian Robust Linear Modeling using the Mahalanobis Distance (BRLMM) genotyping algorithm [15].
Consequently, 333,651 SNPs had a missing genotype call rate below 0.1, a minor allele frequency (MAF) greater than 0.01, and no deviation from Hardy-Weinberg equilibrium (HWE) (p &gt; 1 × 10-6).
To examine the population stratification, multidimensional scaling analysis and principal component analysis were performed using 44,724 pruned SNP markers [14].
Title: Statistical analysis

The orthostatic BP changes were calculated by deltaBP = mean supine position BP - standing position BP.
Delta systolic blood pressures (dtSBP), delta diastolic blood pressure (dtDBP), and OH cases (dtSBP ≥ 20 and/or dtDBP ≥ 10 [16]) were analyzed by linear or logistic regression, controlling for covariates, such as age, sex, BMI, and systolic BP.
Statistical analyses were performed using PLINK version 1.07, using default options [17] and SPSS version 15.0 (SPSS Inc., Chicago, IL, USA).
The GWAS were conducted for Ansan and Ansung separately.
The Ansan GWAS and Ansung replication study results were combined by inverse-variance meta-analysis method, assuming fixed effects, with Cochran&#39;s Q test used to assess between-study heterogeneity [18].
The selected SNPs had a common MAF (&gt;0.05) and the population heterogeneity (p &gt; 0.05).
The asymptotic HWE tests were conducted using PLINK, and all the reported p-values were two-sided.
Title: Clinical characteristics

Table 1 describes the clinical characteristics of OH case and control subjects in the Ansan and Ansung cohorts; 311 participants (7.6%) and 496 participants (14.3%) were found to have OH in Ansan and Ansung, respectively.
The gender distributions of OH differed significantly in both cohorts, and more male OH in Ansung and less male OH in Ansan were observed.
The age of OH in Ansung (58.8 ± 7.8 years) was older than that in Ansan (51.5 ± 8.3 years), reflecting the higher prevalence of OH in Ansung.
The BMI was significantly smaller in Ansung OH cases than controls, but it was not significant in Ansan.
While the mean supine BP was significantly higher in OH than controls, the standing BP was significantly lower in OH than controls.
Title: Genome-wide association studies

GWAS were conducted for each cohort, and the target phenotypes were dtSBP, dtDBP, and OH.
The GWAS results of each cohort were combined by meta-analyses, and quantile-quantile plots and Manhattan plots are depicted in Fig.1.
All detected SNPs are described in Table 2.
Five SNPs for dtSBP trait and three SNPs for dtDBP trait passed our criteria (replicated in both cohorts [Ansung and Ansan] and p &lt; 1 × 10-5), but unfortunately, the SNPs of the OH case control GWAS did not pass our p-value criteria.
Among the significant SNPs, four SNPs (rs6736587, rs6756959, rs6715049, and rs17020502) were in the high linkage disequilibrium status (r2 &gt; 0.9) and were including the most significant SNP, rs6736587 (meta-analysis p-value, 5.3 × 10-8).
The region of four SNPs was located in the intergenic region of chromosome 2, and the nearest gene (CTNNA2) was located at 1 Mb of distance.
The remaining SNPs revealed a suggestive association (p &lt; 1 × 10-5) with dtSBP at rs7098785 (PIK3AP1 intron) and with dtDBP at rs6892553, rs16887217, and rs4959677, which were located in the ACTBL2 3&#39; flanking, steroidogenic acute regulatory protein (STAR) intron, and intergenic regions (chromosome 6p 25.2), respectively.
These normal baro-reflex responses cause an increase in heart rate and vascular resistance, restoring normal BP and cardiac output [1].
OH is the failure of the cardiovascular reflexes to maintain BP on standing from a supine position [2].
Also, the frequency of OH in families with a history of essential hypertension may be higher than in the general population [4].
Therefore, the identification of genetic factor for orthostasis will help us understand OH mechanisms.
The signal plot for the most significant SNP (rs6736587) is depicted in Fig.2; the region was intergenic and a functional gene was located 1 Mbp upstream.
Interestingly, this region is known as the chromosome 2p12-11.2 deletion syndrome region, and several cases report mental or psychomotor retardation [19, 20].
There was only one gene, CTNNA2 (or known as alpha-N-catenin), around ±1 Mbp.
CTNNA2 is a linker between cadherin adhesion receptors and the actin cytoskeleton and is essential for stabilizing dendritic spines in rodent hippocampal neurons [21].
Although there is no report about function in BP regulation, hippocampal neurons interact primarily with the autonomic nervous system.
Therefore, the CTNNA2 gene may function in orthostatic BP regulation.
Among the other SNPs, rs7098785 was located in the phosphoinositide-3-kinase adaptor protein 1 (PIK3AP1) gene.
PIK3AP1 is involved in B-cell development [22].
Interestingly, in silico analyses by the UCSC genome browser (http://genome.ucsc.edu/) indicated that the SNP was located in a DNase I hypersensitive region and in the binding site of transcription factors (MEF2C, BATF, MEF2A).
Therefore, the rs7098785 SNP might have a regulatory function of PIK3AP1 expression.
The remaining three SNPs of dtDBP were located in the ACTBL2 (beta-actin like protein 2) 3&#39; flanking region, STAR intron, and intergenic region (chromosome 6p 25.2).
The ACTBL2 protein is involved in various types of cell motility [23], and STAR is involved in the regulation of steroid hormone synthesis [24].
Our study was the first trial to identify genetic variants for orthostatic BP traits in Koreans.
We hope that our finding could help us understand the underlying mechanism of orthostatic BP changes and related diseases.
Quantile-quantile plots and Manhattan plots of genomewide association studies for delta systolic blood pressure (A, D), delta diastolic blood pressure (B, E), and orthostatic hypotension (C, F).
Signal plot of the most significant association with delta systolic blood pressure.
SNP, single nucleotide polymorphism.
Title: Clinical characteristics of orthostatic hypotension case controls in Ansan and Ansung cohorts phenotype

Title: Significant SNPs of genomewide association study and replication results
Title: Gene Set Analyses of Genome-Wide Association Studies on 49 Quantitative Traits Measured in a Single Genetic Epidemiology Dataset

Gene set analysis is a powerful tool for interpreting a genome-wide association study result and is gaining popularity these days.
Comparison of the gene sets obtained for a variety of traits measured from a single genetic epidemiology dataset may give insights into the biological mechanisms underlying these traits.
Based on the previously published single nucleotide polymorphism (SNP) genotype data on 8,842 individuals enrolled in the Korea Association Resource project, we performed a series of systematic genome-wide association analyses for 49 quantitative traits of basic epidemiological, anthropometric, or blood chemistry parameters.
Each analysis result was subjected to subsequent gene set analyses based on Gene Ontology (GO) terms using gene set analysis software, GSA-SNP, identifying a set of GO terms significantly associated to each trait (pcorr &lt; 0.05).
Pairwise comparison of the traits in terms of the semantic similarity in their GO sets revealed surprising cases where phenotypically uncorrelated traits showed high similarity in terms of biological pathways.
For example, the pH level was related to 7 other traits that showed low phenotypic correlations with it.
A literature survey implies that these traits may be regulated partly by common pathways that involve neuronal or nerve systems.
Genome-wide association (GWA) studies are now well established for discovering hypothesis-free genetic loci whose variations are associated with a phenotype [1-4].
Downstream analysis of the study result in terms of gene sets is gaining popularity these days, as it facilitates biological interpretation of a GWA result [5].
The Korea Association Resource (KARE) project has collected epidemiological and genotype data from the regional cohorts in Ansung and Ansan, Korea [6].
A number of GWA studies based on this KARE data have been published so far [6-8].
We pursued re-analyses of these studies in terms of gene sets in a systematic way to warrant cross-comparisons of the traits.
We mimicked the analysis conditions of the original published GWA works as much as possible, based on the de facto standard program PLINK [9].
For the gene set analysis, we used GSA-SNP software [10], an efficient Java-based application that accepts a list of single nucleotide polymorphisms (SNPs) and their association p-values and outputs the gene set p-values after multiple testing correction.
We applied this procedure to 49 baseline quantitative traits that were of epidemiological, anthropometric, or blood chemistry parameters.
As expected, those traits that are known to share common biological mechanisms and to overlap in their population incidences indeed showed similar profiles of the significantly associated gene sets.
To our surprise, some of the traits that were phenotypically uncorrelated in our study population and were seemingly unrelated to each other showed similar profiles of the significantly associated gene sets.
Our results may demonstrate a useful strategy of discovering pleiotropy, which refers to a phenomenon of common pathways involved in distinct phenotypes.
Title: Phenotype and genotype data

Details of the KARE study design have been reported [6].
Briefly, the genotype data were measured for a total of 10,038 residents in both Ansung and Ansan provinces, Korea, using Affymetrix Genome-wide Human SNP Array 5.0 chips (Affymetrix Inc., Santa Clara, CA, USA).
After quality control, 352k SNP genotypes for 8,842 samples were used in the subsequent GWA analyses.
The epidemiological trait data for these individuals were also received from the KARE project.
Among a total of 49 quantitative traits, we took a logarithm of 5 trait values to balance the distribution.
Title: GWA analyses

We used the imputed genotype data that comprised 1.8 million SNP markers [6].
GWA analyses of these 49 quantitative traits were performed by linear regression under the additive genetic model, as implemented in PLINK.
Title: Gene set analyses

We performed GSA using the Gene Ontology (GO) databases, where only gene sets having 10-200 members (2,476 biological process GO terms) were used.
We applied the Z-statistic method, as implemented in GSA-SNP [10], with the default options.
Briefly, those SNPs residing inside or within 20 kb of the boundary of each gene were compiled, and the second best p-value was assigned to the gene.
See Kwon et al.
[11] for the rationale of using the second best p-value instead of the best p-value.
The gene score was defined as the -log of the p-value assigned to the gene.
The Z-statistic was then calculated for each gene set [10].
The p-values for each gene set were computed under the assumption of a normal distribution of the Z-statistic, followed by multiple testing correction using the false discovery rate method.
When the member genes of a gene set overlapped in their genomic loci or were located in tandem within a short block of strong linkage disequilibrium, the p-values assigned to them might have been highly correlated.
In such cases, only one of them was included to calculate the gene set score.
Title: Semantic similarity between GO terms

The R package GOSemSim was developed to compute semantic similarity among GO terms, sets of GO terms, gene products, and gene clusters.
This package contains functions to estimate the semantic similarity of GO terms based on Resnik&#39;s, Lin&#39;s, Jiang and Conrath&#39;s, Rel&#39;s, and Wang&#39;s methods.
Here, we used Wang&#39;s method for our analysis.
This method determines the semantic similarity of two GO terms based on both the locations of these terms in the GO graph and their relationships with their ancestor terms [12].
The similarity index between the two GO terms was between 0 (the least similar) and 1 (identical).
The similarity between two sets of GO terms was calculated as follows.
First, the highest value against all members of the other set was calculated for a given GO term in one of the sets.
Then, all of these values were collected and averaged to provide the similarity between the two sets.
The significance of the similarity index was inferred, based on the distribution of the indices from 106 random samplings.
Parallel and systematic GWA studies of a number of traits using a single genetic epidemiology dataset give a unique opportunity to explore common biological pathways regulating multiple traits that do not display phenotypic correlations.
In this report, we focused on the 49 baseline quantitative trait data that were demographic, anthropometric, or blood chemistry parameters.
We surveyed the literature for the published GWA studies, based on the KARE data, and extracted the information on the analysis conditions, such as exclusion of samples and covariates of association regression.
For the GWA analyses of all 49 traits, we included area, age, and sex as covariates, and for some of the traits, additional covariates were added, based on the literature information (Table 1).
The phenotypic relations were also learned from principal component analysis (PCA).
For example, the directions pointed by waist and body mass index (BMI) were similar in the PCA plot (Fig.1); using BMI as one of the covariates for the analysis of waist.
On the contrary, we used a literature guide for the analysis of BMI [13].
GWA analyses for all 49 traits were performed using linear regression, as implemented in PLINK, and the resulting SNP p-values were fed into GSA-SNP for gene set analysis.
For each of the 49 traits, we identified biological process GO terms that showed p &lt; 0.05 after multiple testing correction (the lists are available upon request).
As GO terms are hierarchically arranged, verbatim matches are not desirable for the comparison of a pair of gene sets.
Instead, one should take into consideration the number of nodes in the GO tree that separate a pair of terms.
For this, we used so-called semantic similarity, as implemented in the R package GOSemSim.
For each pair of traits, we calculated the semantic similarity of the gene sets and plotted it against the correlation coefficient between the trait values (Fig.2).
The pairs showing semantic similarity greater than 0.8 corresponded to traits that had high correlations in the trait values.
We also noted many pairs having high semantic similarity, despite low phenotypic correlation.
We selected 46 trait pairs that showed a semantic similarity in gene sets greater than an arbitrary cutoff of 0.75 and depicted the interaction network using Cytoscape (Fig.3).
The traits related to blood lipid levels, such as total cholesterol, high density lipoprotein cholesterol, low density lipoprotein cholesterol, and triglycerides (TGs), formed a small network by themselves.
As cholesterol and TGs are components of lipoproteins, it is not surprising to observe such a network.
Both hip and waist-hip ratio are connected to suprailiac skinfold; they are all related to body fat.
The most striking feature is a large network involving 20 traits.
In this network, the traits related to blood glucose levels, such as GLU0, GLU60, GLU120, HbA1c, and HOMA, form a subnetwork.
In addition, the traits related to either blood pressure or liver damage also formed respective subnetworks, as expected from their high phenotypic correlations.
We also noted some hub nodes having many first neighbors.
Interestingly, the pH level was connected to 7 other traits, none of which showed high phenotypic correlations with it.
We examined the gene sets that were commonly shared between these traits (Table 2).
Among them, 11 were related to neuron development and function.
We surveyed the literature for putative involvement of neuronal or nerve systems in the regulation of those 8 traits (Table 3).
Although it is presumptive, this may imply that these traits are regulated partly by common pathways that involve the neuronal system.
In conclusion, our work may be a useful approach for discovering pleiotropy.
Principal component analysis of the phenotype values of 49 quantitative traits used in this study.
See Table 1 for the trait abbreviations.
Scatter plot of correlation coefficients versus biological process Gene Ontology semantic similarities (GOSemSim) between trait pairs.
The red horizontal line marks the GOSemSim value at 0.75.
Interaction network between traits based on biological process Gene Ontology semantic similarity.
Pairs of traits having the semantic similarity greater than 0.75 are connected.
On the other hand, the edge color represents the direction of phenotypic correlations (red and green for positive and negative correlations, respectively), and the brightness of the line represents the level of correlation (the brighter the higher correlation, and the darker the lower correlation).
Title: List of the traits used in this work

Title: The biological Gene Ontology (GO) terms commonly associated with pH and its first neighbor traits in Fig.3 GO category

Title: Literature evidence of neuronal systems regulating pH and its first neighbor traits in Fig.3
Title: Structure and Expression Analyses of SVA Elements in Relation to Functional Genes

SINE-VNTR-Alu (SVA) elements are present in hominoid primates and are divided into 6 subfamilies (SVA-A to SVA-F) and active in the human population.
Using a bioinformatic tool, 22 SVA element-associated genes are identified in the human genome.
In an analysis of genomic structure, SVA elements are detected in the 5&#39; untranslated region (UTR) of HGSNAT (SVA-B), MRGPRX3 (SVA-D), HYAL1 (SVA-F), TCHH (SVA-F), and ATXN2L (SVA-F) genes, while some elements are observed in the 3&#39;UTR of SPICE1 (SVA-B), TDRKH (SVA-C), GOSR1 (SVA-D), BBS5 (SVA-D), NEK5 (SVA-D), ABHD2 (SVA-F), C1QTNF7 (SVA-F), ORC6L (SVA-F), TMEM69 (SVA-F), and CCDC137 (SVA-F) genes.
They could contribute to exon extension or supplying poly A signals.
LEPR (SVA-C), ALOX5 (SVA-D), PDS5B (SVA-D), and ABCA10 (SVA-F) genes also showed alternative transcripts by SVA exonization events.
Dominant expression of HYAL1_SVA appeared in lung tissues, while HYAL1_noSVA showed ubiquitous expression in various human tissues.
Expression of both transcripts (TDRKH_SVA and TDRKH_noSVA) of the TDRKH gene appeared to be ubiquitous.
Taken together, these data suggest that SVA elements cause transcript isoforms that contribute to modulation of gene regulation in various human tissues.
The human genome contains actively moving retrotransposons (human endogenous retrovirus [HERV], long interspersed nucleotide element [LINE], and short interspersed nucleotide element [SINE]).
Approximately 36% of the human genome has been generated through retrotransposition of LINE and other RNA species by the LINE reverse transcriptase [1, 2].
Retrotransposition is progressing in the human population by L1, Alu, and SINE-VNTR-Alu (SVA) insertions [3].
L1 is an autonomous retrotransposon that contains an internal RNA polymerase II promoter and a reverse transcriptase, whereas Alu and SVA lack activities for independent mobilization [4, 5].
Therefore, Alu and SVA are assumed to use the L1 protein machinery for their own mobilization [3], and retrotransposition events of marked SVA elements occur indeed by L1 elements in human cultured cells [5, 6].
The SVA elements were originally named from the SINE-R retroposon, derived from an endogenous retrovirus, the HERV-K LTR element.
SINE-R11, 14, and 19 have been isolated by colony blot hybridization using the LTR element as probe [7].
SINE-R.C2 has been found in the third intron of the C2 gene on the short arm of human chromosome 6, which was a human-specific element [8].
Within the Xq21.3 block, two SINE-R retroposons (HS307 and HS408) were identified [9].
Multiple copy numbers of retroposons have been successively detected in hominoid primates and humans [9-16].
Other similar sequences have been associated with Alu-like sequences within the HLA-RP1 (STK19) gene [17].
These composite retroposons with the entire structure are named SVA (SINE-R, VNTR, and Alu).
The SVA families are evolutionarily young and hominoid-specific retroelements.
They have ability to influence a genomic locus in which they reside and cause various human diseases as insertional mutagens.
In fact, Fukuyama muscular dystrophy (FCMD) is derived from the polymorphic SVA-E insertion in the 3&#39; untranslated region (UTR) of the FUKUTIN gene [18].
SVA elements could also drive transcription of functional genes.
In the 5&#39; upstream region of TBPL2, the SVA element serves as a promoter, while the SVA-D element in the WDR66 gene promotes the transcription of a human-specific transcript variant [19].
Here, we analyzed structure variants of functional genes mediated by SVA subfamilies and examined their expression patterns in various human tissues.
Title: Bioinformatic analysis

To identify SVA consensus sequences in the human genome, we obtained SVA sequences from the Giri database (http://www.girinst.org).
The SVA subfamilies were aligned using the BioEdit program [20].
Then, we identified SVAs in each region.
RepeatMasker (http://www.repeatmasker.org) and the UCSC genome site (http://genome.ucsc.edu) were employed to analyze isoform structures of functional genes.
The human expressed sequence tag and RefSeq mRNA were also used to identify alternatively spliced transcripts.
The expression pattern of SVA fusion genes in normal human tissues was analyzed using GeneCard (http://www.genecards.org).
We obtained microarray data from the BioGPS database, and then we generated a heatmap according to microarray values.
High expression levels were indicated by brighter color, and low expression levels were indicated by darker color.
Title: Human RNA samples

A human 20-RNA tissue master panel (1, adrenal gland; 2, bone marrow; 3, cerebellum; 4, whole brain; 5, fetal brain; 6, fetal liver; 7, heart; 8, kidney; 9, liver; 10, lung; 11, placenta; 12, prostate; 13, salivary gland; 14, skeletal muscle; 15, spinal cord; 16, testis; 17, thymus; 18, thyroid; 19, trachea; 20, uterus) was purchased from Clontech (Mountain View, CA, USA).
Title: Reverse-transcription (RT) and reverse transcription polymerase chain reaction (RT-PCR) amplification

To eliminate possible DNA contamination of purchased RNA samples, Turbo DNA-free (Ambion, Austin, TX, USA) was used according to the manufacturer&#39;s instructions.
A no-RT control was also amplified to double-check the absence of DNA contamination.
Quantity of RNA samples was measured using a ND-1000 UV-Vis spectrophotometer (NanoDrop, Wilmington, DE, USA).
Moloney-Murine-Leukemia-Virus reverse transcriptase with an annealing temperature of 42℃ was used for the RT reaction with RNase inhibitor (Promega, Madison, WI, USA).
To develop the specific primers for individual alternative transcripts, primer pairs were designed with the aid of Primer3 (http://frodo.wi.mit.edu/) (Table 1).
In each run, 1 µL of cDNA was used as template for amplification per reaction.
RT-PCR was performed using reactions containing a mixed cDNA template, representing a combination of different tissues examined.
RT-PCR amplification for functional genes and a housekeeping gene was carried out for 30 cycles of 94℃ for 3 minutes, 56-60℃ for 1 minute, and 72℃ for 3 minutes.
As a standard control, G3PDH was amplified through RT-PCR in human tissues.
PCR products were loaded on 1-2% agarose gels and stained with ethidium bromide.
SVAs are composite elements consisting of multiple domains: a CCCTCT repeat, Alu-like domain, a GC-rich variable number of tandem repeat (VNTR), and SINE-R derived from the HERV-K LTR element [7-9, 21, 22].
They are flanked by target site duplications and terminate in a poly(A) tail (Fig.1).
In genomic sequence analysis, SVA elements are present in G + C-rich regions; however, they do not have any preferences for inter- or intragenic regions.
SVA families are separated into six subfamilies (SVA-A to SVA-F), based upon point mutation and insertion and deletion events within the SINE-R [22].
Among them, four subfamilies (SVA-A, SVA-B, SVA-C, SVA-D) are present in gibbons and orangutans, while two subfamilies (SVA-E and SVA-F) are restricted to the human lineage [22].
SVA elements residing in genes are potentially disruptive in either orientation.
Approximately 1/3 of all SVA elements in the human genome reside in genic regions, with 20% of those SVA elements being in the same orientation as a gene [23].
As shown in Fig.2, we analyzed the genomic structure of SVA fusion genes using bioinformatic tools (Repeat-Masker program and University of California, Santa Cruz [UCSC] genome browser).
SVA elements are detected in the 5&#39;UTR of the HGSNAT (SVA-B, AK_057293), MRGPRX3 (SVA-D, NM_054031.2), HYAL1 (SVA-F, NM_153281), TCHH (SVA-F, AK_307946), and ATXN2L (SVA-F, AY_188334-8) genes, while some elements are observed in the 3&#39;UTR of the SPICE1 (SVA-B, NM_144718), TDRKH (SVA-C, AK_225160), GOSR1 (SVA-D, NM_001007024), BBS5 (SVA-D, NM_1523842), NEK5 (SVA-D, NM_199289; AK_126330), ABHD2 (SVA-F, NM_007011), C1QTNF7 (SVA-F, NM_031911.3), ORC6L (SVA-F, AK_024077.1), TMEM69 (SVA-F, NM_016486.2), and CCDC137 (SVA-F, NM_199287.2) genes.
Within the genic region, SVA elements seem to prefer the 3&#39;UTR compared to another regions.
Recently, a novel promoter derived from the SVA-D element was identified in the 5&#39;UTR of the TBPL2 gene (DB089735) [19].
In order to understand whether the SVA element could act as a transcriptional regulator or not, the functional activity of the PARK7 SVA element was performed, suggesting that the SVA with the SINE-R region deleted showed significant activity to enhance reporter gene activity in SK-N-AS and MCF-7 cells [24].
Likewise, we have also demonstrated the activity of an alternative promoter by the HERV-H LTR element in the human GSDML gene [25].
The transcripts of the LTR-derived promoter were widely distributed in various tissues, whereas the transcripts of the original promoter were found in stomach tissues, compared to those of various human tissues [25].
A human-specific solitary LTR element (L47334) was previously shown to have enhancer activity in Tera-1 human testicular carcinoma cells [26].
The varying genetic structure of the alternative promoter or enhancer results in different functional effects [27].
Those events could regulate alternative transcript variants, resulting in the production of different protein isoforms [28].
In the case of leptin receptor isoform 219.1, SVA sequences form the C-terminal coding exon.
Using the SVA-specific probe, Southern blot analysis in hominoids and Old World monkeys was performed, indicating that it was found to be a human-specific SVA element [29].
Furthermore, new exons are generated from intronic retrotransposons or the elements inserts into the exon, leading to a new transcript [30].
In the present study, the LEPR (SVA-C, NM_001003680), ALOX5 (SVA-D, AB_208946), PDS5B (SVA-D, AK_128502.2), and ABCA10 (SVA-F, AL_832004) genes showed alternative transcripts by the SVA exonization event.
To investigate the supply of alternative splicing sites at SVA elements, we analyzed related splicing sites using human genomic sequences (http://genome.ucsc.edu/) (Fig.3).
In the case of ABCA10 (AL832004) genes, the SVA element provided both a splicing donor and acceptor site for the cassette exon.
In the case of the ALOX5 (AB208946) and PDS5B (AK128502) genes, the SVA element provided a splicing donor site for alternative 5&#39; splicing.
Conversely, the LEPR (NM001003680) gene was provided a splicing acceptor site for alternative 3&#39; splicing.
These phenomena conformed to the canonical splicing site.
Also, the PDS5B (AK128502) and ABCA10 (AL832004) genes induced exonization by providing alternative splicing sites in the VNTR region of SVA elements.
Exonization of the LEPR (NM001003680) and ALOX5 (AB208946) genes occurred by the SINE-R region of SVA elements.
This integration event of SVA elements into the exon or intron causes the promotion of transcriptional variants of functional genes.
In WDR66 (NM_144668), the SVA-F element, inserted in intron 19, could produce human-specific transcripts that spliced to the last three exons [19].
Those transcripts, derived from SVA elements, could have important biological function in humans, therefore deserving further investigation in various tissues of hominoid primates.
SVA elements are capable of generating individual variation in gene expression at loci in which they are present.
RT-PCR amplification could be a good indicator to detect alternative variants derived from SVA elements.
As shown in Fig.4, dominant expression of HYAL1_SVA appeared in lung tissues, while HYAL1_noSVA showed ubiquitous expression in various human tissues.
Expression of both transcripts (TDRKH_SVA and TDRKH_noSVA) of the TDRKH gene appeared to be ubiquitous.
We also examined in silico expression of SVA fusion genes using microarray data obtained by the BioGPS database.
Transcripts of the TCHH and HYAL1 genes appeared dominant in placenta and liver, respectively.
The ALOX5 gene showed dominant expression in whole blood and lung.
Also, dominant expression of the ABHD2 gene appeared in testis and prostate tissues, whereas the BBS5, NEK5, MRGPRX3, PDS5B, OR6W1P, ABCA10, TMEM69, C1QTNF7, and ATXN2L genes indicated ubiquitous expression (Fig.5).
SVA elements are nonautonomous retrotransposons that cause diseases in humans, and they are mobilized in trans by active L1 elements.
The quantitative real-time polymerase chain reaction analysis of fukutin mRNA in lymphoblasts from FCMD patients indicated that the disease results from SVA-E insertion in the 3&#39;UTR of the FUKUTIN gene.
Sequence data demonstrated an abnormal splicing event by the integration of an SVA-E element [18].
In the case of the PMS2 gene, SVA-F has inserted in intron 7, and causes Lynch syndrome.
Sequence analysis of the RT-PCR product revealed a 71-bp SVA-F element between PMS2 exon 7 and 8 in the aberrant transcripts [31].
Exon 3 of the PNPLA2 gene was interrupted by a 1.8-kb SVA-F insertion, which causes lipid storage disease with subclinical myopathy [32], while an SVA-F element inserted in intron 1 of the ARH gene causes hypercholesterolemia [33].
Taken together, alternative promoter, enhancer, polyadenylation, and exonization events by SVA elements cause various transcript isoforms and evolutionary dynamics that contribute to flexibility in the regulation of gene expression and hominoid radiation, including human disease.
Nucleotide sequence alignments of SINE-VNTR-Alu (SVA) families.
Dots indicate no change to the SVA-A sequences, and dashes indicate gaps.
The consensus sequences of SVA-A, -B, -C, -D, -E, and -F were taken from Giri DB (http://www.girinst.org).
Main domains (Alu, VNTR, and SINE-R) are indicated.
Alternative splicing patterns of functional genes by SINE-VNTR-Alu (SVA) families.
SVA element is indicated by the arrow.
Exonization and splice site analysis by SINE-VNTR-Alu (SVA) element in functional genes.
UTR, untranslated region; ORF, open reading frame.
Reverse transcription polymerase chain reaction analysis of transcript variants of functional genes in various human tissues.
1, adrenal gland; 2, bone marrow; 3, cerebellum; 4, whole brain; 5, fetal brain; 6, fetal liver; 7, heart; 8, kidney; 9, liver; 10, lung; 11, placenta; 12, prostate; 13, salivary gland; 14, skeletal muscle; 15, spinal cord; 16, testis; 17, thymus; 18, thyroid; 19, trachea; 20, uterus.
G3PDH indicates the positive control.
Heatmap of SINE-VNTR-Alu (SVA) fusion gene expression profiles.
Horizontal lines separate the 6 clusters via biological systems.
SVA fusion genes are ordered according to SVA subfamily type at vertical lines.
Values increase from dark to bright.
Title: List of RT-PCR primer sets for expression analysis of SVA fusion genes
Title: Genome-Wide Association Study of Liver Enzymes in Korean Children

Liver enzyme elevations, as an indicator of liver function, are widely associated with metabolic diseases.
Genome-wide population-based association studies have identified a genetic susceptibility to liver enzyme elevations and their related traits; however, the genetic architecture in childhood remains largely unknown.
We performed a genome-wide association study to identify new genetic loci for liver enzyme levels in a Korean childhood cohort (n = 484).
We observed three novel loci (rs4949718, rs80311637, and rs596406) that were multiply associated with elevated levels of alanine transaminase and aspartate transaminase.
Although there are some limitations, including genetic power, additional replication and functional characterization will support the clarity on the genetic contribution that the ST6GALNAC3, ADAMTS9, and CELF2 genes have in childhood liver function.
Plasma liver enzyme tests are broadly applied to predict biochemical liver functions and to diagnose liver diseases.
Alanine transaminase (ALT) and aspartate transaminase (AST) are closely associated with hepatocyte damage and steatosis, including non-alcoholic fatty liver disease (NAFLD) [1].
Elevated liver enzyme levels, as prospective risk factors, have been reported to be involved in type 2 diabetes and coronary heart disease [2, 3].
However, alkaline phosphatase and gamma-glutamyltranspeptidase are related with biliary, cholestatic, and alcoholic diseases [4].
Overweightedness in childhood or adolescence is persistent in adulthood obesity and is closely associated with adverse health effects, including metabolic diseases [5].
A recent cohort study in Korean adolescents has shown strong associations of liver enzyme levels with a cluster of metabolic syndrome components [6].
Thus, finding genetic markers of liver enzyme levels in childhood would be important for early prediction of liver function and its related disease in adulthood.
Liver enzyme levels are affected by environmental and genetic factors.
The heritabilities of liver enzyme levels range from approximately 35% to 61% [7, 8].
To date, two genome-wide association studies (GWASs) in European ancestry populations have identified genetic variants located in multiple genes associated with levels of ALT (HSD17B13-MAPK10, TRIB1, CHUK, and SAMM50), AST (NBPF3-ALPL), and both of them (CPN1 and PNPLA3) [9, 10].
In addition, a meta-analysis identified pleiotropic effects between NAFLD and metabolic traits, such as low-density lipoprotein, high-density lipoprotein, triglyceride, fasting glucose, and insulin resistance [11].
Despite the identification of novel genes or loci associated with liver enzymes in cohort populations, the common variants in childhood are still largely unknown.
Here, we present findings from a GWAS in childhood subjects to identify new susceptibility loci for liver enzyme levels.
Title: Study subjects

Students, aged between 8 and 13 years, were recruited from Gwacheon City and Kyunggi Province from April to June 2010.
This study was performed as a part of the Korean Children-Adolescents Study (KoCAS), which has been monitored yearly since their entry into elementary school at age 7 in Gwacheon City or fourth grade at age 10 in Seoul and Gyeonggi Province, Korea.
Subjects who were enrolled in a specific diet program or were taking any medications known to affect appetite were excluded from the study.
AST and ALT levels were measured in the serum of each subject.
The study protocol was approved by the institutional review board of Seoul Paik Hospital, Inje University, and the Korea Center for Disease Control and Prevention.
Informed consent was obtained from the childrens&#39; parents.
Title: Genotyping and quality control

A total of 532 samples were genotyped using the Illumina HumanOmni1-Quad BeadChip (Illumina Inc., San Diego, CA, USA).
Samples that revealed the following properties were excluded: genotyping call &lt; 98%, heterozygosity &gt; 30%, sex inconsistency, cryptic first-degree relative, or duplication.
A total of 48 subjects were excluded from the sample QC process.
Markers with a high missing call rate (&gt; 5%), minor allele frequency &lt; 0.01, and significant deviation from Hardy-Weinberg equilibrium (p &lt; 1 × 10-6) were also excluded.
The remaining 747,076 single nucleotide polymorphisms (SNPs) were used in subsequent analyses for association.
Title: Association analyses

Genome-wide association was analyzed using the PLINK software (http://pngu.mgh.harvard.edu/~purcell/plink).
The association between genotypes and concentration of liver enzymes in plasma was evaluated with linear regression analysis, controlling for age and body mass index (BMI) as covariates.
The Manhattan plots of -log10 (p-value) were made using Haploview v. 4.2.
The quantile-quantile plots were generated using R statistics package to assess the overall significance of the genome-wide associations.
A total of 747,076 SNP genotype assays were tested on the DNA samples of 484 Korean childhood subjects.
ALT and AST levels of the subjects according to age and sex are presented in Supplementary Table 1.
Linear regression analyses were performed to estimate the p-values for the association between genotype distributions and levels of ALT and AST.
Manhattan plots of association analyses of all SNPs with respect to ALT and AST are presented with the chromosomal positions (X-axis) and negative logarithm of p-values (Y-axis) of each SNP (Fig.1).
Quantile-quantile plots comparing the association p-values with those expected for a null distribution are also presented (Fig.2).
The highest-ranked SNPs showing the lowest p-values (p &lt; 1.0 × 10-5) of association with ALT and AST levels are listed in Table 1.
In addition, genotype clusters of the SNPs are presented in Supplementary Fig.1.
Although there was no SNP satisfying the criteria of genome-wide significance (p &lt; 1.0 × 10-8), we identified three loci (rs4949718, rs80311637, and rs596406 in ST6GALNAC3, ADAMTS9, and CELF2, respectively) that were suggestively associated with levels of ALT and AST.
Of these SNPs, rs4949718 was found to be more strongly associated with ALT levels (p = 1.87 × 10-7) but less in AST levels (p = 1.49 × 10-6).
The missense rs80311637 (V1597M) and intronic rs596406 SNPs were more strongly associated with an increased risk of AST than ALT.
The levels of ALT and AST, known as transaminases, are important indicators for determining hepatocyte injury or hepatocellular diseases.
However, ALT is known to be found in the highest level in the liver; thus, ALT levels are more liver-specific indicators [4].
Moreover, it has been reported that ALT levels are more significantly associated with a cluster of metabolic syndrome than AST levels in a Korean adolescent cohort study [6].
In this study, we identified three novel genes (ST6GALNAC3, ADAMTS9, and CELF2) that were multiply associated with elevated levels of ALT and AST.
Of the three loci, rs4949718 in the intron region of ST6GALNAC3 yielded the lowest p-value for the association with ALT, compared to AST levels (Table 1).
In addition, rs80311637 (ADAMTS9) and rs596406 (CELF2) were shown to be more strongly associated with AST levels, compared to ALT.
These results indicate that the liver-specific feature of ALT was not driven as the secondary association for AST.
In recent, a previous GWAS in an Amish population identified two genetic variants (rs17209512 and deletion) in glutamic-oxaloacetic transaminase 1 (GOT1) that were associated with serum AST.
However, the rs17209512 SNP, which is in incomplete LD with rs76850691, was not replicated in our study (Table 1).
For the three multiply associated loci, we performed conditional analyses to test whether the association with the other trait was simply secondary to the primary association (Supplementary Table 2).
In these analyses, the association signals on rs4949718 for AST and rs80311637 and rs596406 for ALT were abolished by adjustment of the ALT and AST, respectively.
It is assumed that the multiple effects of these loci on ALT and AST resulted from the secondary effects by physiologically correlated features between ALT and AST [12].
In addition, when comparing with BMI-unadjusted data, the association signals in ALT were relatively weaker than those of the BMI-adjusted data.
However, there was no significant difference in p-values before and after adjusting for BMI in AST (data not shown).
ST6GALNAC3 encodes a type of sialyltransferase that plays a role in the transfer of sialic acids from cytidine 5&#39; monophospho-N-acetylneuraminic acid (CMP-NeuAc) to the carbohydrate termini of glycoproteins and glycolipids [13].
Previous GWASs have revealed the associations of ST6GALNAC3 SNPs with human electroencephalogram and nicotine dependence [14, 15]; however, there has been no report on an association with liver enzyme levels.
Although the association with liver enzymes is still unknown, ST6GALNAC3 has been shown to be associated with subscapular skin-fold thickness, a type of obesity-related trait, in a Korean adult population [16].
Obesity affects elevated liver enzyme levels by inducing inflammatory responses in the liver, accompanied by increases in C-reactive protein [17].
The involvements between obesity and liver enzyme levels have been found in cohort studies [18-20].
Therefore, the association of ST6GALNAC3 with obesity may influence liver enzyme levels indirectly.
In addition, ST6GALNAC3 protein interacts with α2-Heremans-Schmid glycoprotein (AHSG), which is also known as fetuin-A, by enzyme-substrate specificity [13].
AHSG is abundantly distributed in serum protein and is produced mainly in the liver, and elevated levels of this protein are associated with the risk of NAFLD [21].
Considering that liver enzyme levels are closely involved indicators in NAFLD [22, 23], the interaction with AHSG might induce an association between ST6GALNAC3 and liver enzyme levels.
ADAMTS9 is a metalloproteinase that plays a role in cleavage of proteoglycans and is expressed in skeletal muscle and pancreas [24].
ADAMTS9 has been reported as a type 2 diabetes susceptibility region in a GWAS [24].
This gene has also shown an association with insulin resistance and impaired beta-cell function in the pancreas [25].
Taken together, the association of ADAMTS9 with liver enzymes might be due to its susceptibility to diabetes and insulin resistance.
CELF2 (also known as CUGBP2) induces the skipping of exon 11 of insulin receptor (IR).
The mRNA expression level of exon-skipped IR is significantly lower in liver than in other organs in mouse models [26].
It has been reported that a decrease in the number of IRs on target cells can cause insulin resistance [27].
Therefore, it is assumed that CELF2 affects liver enzyme levels by lowering expression of IR in liver that induces insulin resistance.
In summary, our data demonstrate new multiple associations with ALT and AST levels in a Korean childhood cohort.
Further replication and functional studies remain to be done to clarify the genetic factors of liver dysfunction.
Manhattan plots of association results for alanine transaminase (ALT) and aspartate transaminase (AST) levels.
Manhattan plots of the genome-wide analyses of association with ALT (A) and AST (B) are based on a total of 484 Korean children.
Color scale of X-axis represents chromosomes, and negative logarithms of p-values are shown on the Y-axis.
Quantile-quantile plots of association results for alanine transaminase (ALT) and aspartate transaminase (AST) levels in Korean children for 747,076 single nucleotide polymorphisms (SNPs).
Quantile-quantile plots for ALT (A) and AST (B) AST are composed of X-axis and Y-axis indicate the negative log-scale of expected p-values of each SNP and the negative log-scale of the observed p-values, respectively.
A straight line indicates the expected results under Hardy-Weinberg equilibrium.
Title: Genetic loci associated with levels of ALT and AST at p &lt; 1 × 10-5 in the GWAS of Korean children (n = 484)
Title: Protein Backbone Torsion Angle-Based Structure Comparison and Secondary Structure Database Web Server

Structural information has been a major concern for biological and pharmaceutical studies for its intimate relationship to the function of a protein.
Three-dimensional representation of the positions of protein atoms is utilized among many structural information repositories that have been published.
The reliability of the torsional system, which represents the native processes of structural change in the structural analysis, was partially proven with previous structural alignment studies.
Here, a web server providing structural information and analysis based on the backbone torsional representation of a protein structure is newly introduced.
The web server offers functions of secondary structure database search, secondary structure calculation, and pair-wise protein structure comparison, based on a backbone torsion angle representation system.
Application of the implementation in pair-wise structural alignment showed highly accurate results.
The information derived from this web server might be further utilized in the field of ab initio protein structure modeling or protein homology-related analyses.
Protein is a primary component of living organisms that has structural and chemical roles.
Among the many characters of proteins, the structural information has been a major concern for biological and pharmaceutical studies for its intimate relationship to the function of the protein.
Most of the databases of protein structure utilize structure analysis, based on the 3D representation of the positions of protein atoms.
The space of possible positions of atoms of proteins, however, could be described using the torsional representation of atoms along the axis of covalent single bonds.
Most of the movements of atoms are actually limited to the rotations along the single bond axes, considering the rotational property of covalent single bonds and the infrequency of the change of the length and angles of the covalent bonds for high-energy barriers.
The reliability of this torsional system, which represents the native processes of structural change in the structural analysis, was partially proven by previous research of structure alignment [1].
In most typical representation systems of protein structures, the 3D structure of proteins is represented by Cα atom positions on the Cartesian system.
Better homology delineation was achievable by applying the torsional representation of backbone structures than a typical Cartesian system in the structure alignment.
The structures of two proteins were aligned using the information of the torsion angles of Cα atoms of the backbones to compare the structural similarity.
The strings of the torsion angles of the two protein backbones were compared using a sequence alignment algorithm.
The conducted backbone torsion angle-based structure alignment has advantages over typical structure alignment methods from the comparison of backbone topology, rather than the general globular 3D shape of proteins [2].
The better accuracy of the torsion angle system in the structural alignment analysis was utilized for the construction of a relevant information repository.
Thus, a structural information repository and analysis tool based on better structural representation of a torsional system might be appreciable.
Here, the secondary structural information- and structure comparison function-providing web server, based on the backbone torsional representation of a protein structure, is newly introduced.
The structure alignment function with torsion angles of amino acid backbones was supplied in a developed web server.
Functions of secondary structure database search, secondary structure calculation, and protein pair-wise structural comparison are supplied.
The secondary structure of protein data bank (PDB) and structural classification of proteins (SCOP) entries are supplied using MySQL infrastructure.
Protein secondary structure calculation and pair-wise comparison functions are supplied, based on JAVA/JSP infrastructure.
The newly developed web server serves information of the constructed secondary structure database from prebuilt 3D structure databases using a classification scheme, driven from backbone torsion angles.
The utility of protein structure comparison, based on the backbone torsional representation, is also provided by the server.
Details of the classification scheme of secondary structures, the comparison method, and the similarity measurements are as follows.
Title: Secondary structure determination

Three categories of &#34;helix,&#34; &#34;extended,&#34; and &#34;others&#34; were used for the classification of the secondary structures of each residue.
Secondary structure determination for each residue referred to the backbone torsion angle values.
Amino acid residue was classified into &#34;helix&#34; if the backbone torsion angles belonged to the range of (φ, ψ) = (-155° to -47°, -62° to -52°), (-104° to -47°, -52° to -37°), and (-117° to -104°, -52° to -37°).
A residue was classified into the &#34;extended&#34; secondary structure if the backbone torsion angles belonged to the range of (φ, ψ) = (-155° to -138°, 90° to 155°), (-140° to -64°, 90° to 180°), (-64° to -53°, 90° to 100°, or 110° to 168°).
Residues with backbone torsion angles belonging to other ranges were classified as &#34;others.&#34;
The range was determined based on typical Ramachandran plots.
Title: Pair-wise structure alignment

Title: Two measurements (RamRMSD and logPr) were used to determine the structural similarity of two aligned proteins, as described in the previous structure alignment study [1].
RamRMSD was named for the value representing the distance of the two points of the residues on the Ramachandran plots of the two proteins.
While RamRMSD is the root mean square deviation of the Euclidean distance of the φ and ψ angles of matched residues, which is similar to the measurement of Δt used by Karpen et al.
[3], logPr is the statistical quantity that describes the probability that a closer alignment than observed is possible.
In this study, both the geometric distance of RamRMSD and statistical quantity of logPr were utilized for the assessment of the structural similarity [2].
The Euclidean distance can be defined as follows:

Title: D = (Δφ2 + Δψ2)1/2,

Title: where D is the distance and

Title: Δφ2 = (φ1 - φ2)2, if (φ1 - φ2)2 ≤ 1802

Title: (360 - |φ1 - φ2|)2, if (φ1 - φ2)2 &gt; 1802

Title: Δψ2 = (ψ1 - ψ2)2, if (ψ1 - ψ2)2 ≤ 1802

Title: (360 - |ψ1 - ψ2|)2, if (ψ1 - ψ2)2 &gt; 1802,

Title: where φ1 and φ2 are φ angles from each residue and ψ1 and ψ2 are ψ angles from each residue.
Conditional terms are added to find the smallest distance between any two angles with our -180° to +180° notation; i.e., for example, not to consider the distance of two angles, +180° and -180°, as 360° apart rather than 0° apart.
The RamRMSD would be as follows:

where n is the total number of residues to be compared and Dk is the distance of points of kth residues of each protein on each Ramachandran plot as defined above.
RMSD is weak to small number of local deviations [4].
logPr circumvents this problem of the RMSD and is defined as the logarithm of the mean probability of finding a closer angular similarity than the observed similarity in a random environment between each torsion angle pair of compared chains.
Title: If the difference of the φ and ψ angles is defined as a vector Ω (ωφ1, ωψ1, ωφ2, ωψ2, … , ωφn, ωψn), where ωφk is the difference of two φ angles of the kth amino acid of each n-residue-long string and ωψk is the difference of two ψ angles of the kth amino acid of each n-residue-long string, the constant probability density function ρ (ω) and the Pr-value in a random environment could be mathematically written as follows:

Title: where ω is the angular difference, and

where n is the number of total residues being compared and every angular difference is presumed to be statistically independent.
Because multiplied values range from 0 to 1, the Pr-value is more strongly dependent for small values than for large values.
We used the logPr-value to circumvent a computational overflow problem and used log base 10 for easy comprehension of the order of magnitude of the probability, Pr.
Also, Bonferroni correction was applied for proper comparison of the similarities from protein pairs of different length.
Thus, the logPr value could be defined as follows.
Global alignment with no gaps was performed using these two measurements.
The comparison frame was shifted by a single residue for each frame with boundary conditions for the most similar alignment.
Title: Parameter settings for alignments and clustering

Global alignment with a gap open penalty of 13, extension penalty of 3, and free end gap penalty was conducted for sequences of 30 proteases and 30 kinases.
A UPGMA algorithm with bootstrapping of 100 replicates was used for tree construction from a sequence of proteases.
CLC bioinformatics workbench was used for alignment and tree calculation, and the Geneious workbench was used for graphical representation.
(8 + logPr), RamRMSD, and (1-TM-score) were used for distance, and a Fitch-Margoliash algorithm was employed for building trees from protein structures.
TM-score was normalized by the size of the target protein of the comparison pair.
An appropriate integer (8) was added to logPr to make distances positive.
Trees were generated from a distance matrix using the FITCH program of the PHYLIP package.
The Geneious workbench was used for graphical representation of trees.
Title: Receiver operating characteristic (ROC) curve analysis

Title: ROC curve analysis illustrates the accuracy of a binary classifier with graphical representation of the specificity and sensitivity with varying threshold for the discrimination of true and false pairs on a plot.
After setting a threshold for the delineation of positive and negative classes, true positive (TP), true negative (TN), false positive (FP), and false negative (FN) are defined.
The sensitivity, or true positive rate (TPR), and specificity, or true negative rate (TNR), are defined as follows:

,
Title: while numerical ROC values are also defined as follows:

where Ti signifies the number of true positives ranked ahead of the ith false positive.
An ROC curve was drawn following these calculations.
To draw the curve, the thresholds are varied from the ones for the most sensitive discrimination, where all predictions are positive, to the one for the most specific discrimination, where all predictions are negative.
A good classifier would show high sensitivity and high specificity concurrently.
The more accurate classifier displays an earlier and steeper increase of the curve, while a less accurate classifier displays a less steep increase in the lower left region.
Thus, the area under the curve could possibly signify the accuracy of the classifier.
Area under the ROC curve (AUROC) was calculated using approximations of the curve, with a grid width of 0.01.
The secondary structure database was constructed from 92,998 PDB chains and 64,799 SCOP entries based on the simple classification scheme according to the backbone torsion angles (http://lcbb.snu.ac.kr/TorsWebDB).
SCOP entry 3D structure files were obtained from the ASTRAL PDB style database [5-7].
The function of the calculation of new secondary structures is additionally supplied.
The backbone torsion angle-based structure alignment tool is also supplied by the newly developed web server.
The platform of the web application was built in the machine with an AMD phenome II quad-core CPU (3.0 GHz) and openSUSE Linux 11.2 operating system.
JAVA/JSP infrastructure was utilized for the construction of the server platform.
The web application introduced here offers functions of secondary structure database search, secondary structure calculation, and pair-wise protein structure comparison.
The index page of the server web application supplies hyperlinks to the three functions, along with a general introduction and the explanation of the cases of limitations in the supply of data (Fig.1).
The query for each function is input, and the result is retrieved through JSP pages in coordination with MySQL database engines in the case of the secondary structure database search.
The results of the secondary structure database search query display information of entries of the searched database that satisfy the criteria (Fig.2).
Protein structure comparison supports pair-wise comparison of proteins from both uploaded files and specified IDs of SCOP entries, and the result of the comparison displays logPr and RamRMSD values as similarity measurements and CPU times that were spent for the calculation of each value (Fig.3).
The robustness of the backbone torsion angle method in structure alignment was partly validated before by the clustering analysis of 4 types of proteases [1].
More distant groups of proteins were used for the assessment of the accuracy of the backbone torsion angle method here.
A mixed set of 30 kinases and 30 proteases were clustered, and the accuracy of delineation was analyzed with the resulting 1,770 pairs (Fig.4).
Trees of clustering from logPr (Fig.4A) and RamRMSD (Fig.4B) measurements showed an apparent delineation of kinases from proteases.
The distances between these two groups were very long, so that one could easily recognize the perfect separation of each from the other.
The distances among kinases and proteases each were rather comparable in the case of logPr measurements, while RamRMSD showed much shorter distances of proteases than those of the kinases.
The difference between the distances of the two measurements might have originated from the weighting of the smaller distance in the case of logPr, which might have caused the distances among kinases to shrink.
The logPr and RamRMSD tree showed a smaller distance between internodes than the distance between terminal leaves and the last internode, indicating that structural information is quite homogeneous within the group of identical functional homology.
The clear discrimination of two homology group members indicates that the structural classification is strongly robust with the backbone torsion angle method, especially in the case of a lucid functional difference.
The clustering tree from the TM-align RMSD measurement did not show clear delineations between kinases and proteases, while it showed some aggregations of each as subclusters (Fig.4C).
Though 22 kinases were posed close to each other, members of this major cluster of kinases were in very close proximity to members of the protease group, making them hard to be clearly separated from proteases, referring the pair-wise distances.
Furthermore, 8 kinases were posed within the cluster of proteases.
The distribution of pair-wise distances among 60 proteins was also rather even.
This incomplete separation markedly shows the better ability of our backbone torsion angle method than the conventional 3D alignment algorithm, like the TM-align method.
The inferior performance of TM-align might be due to the inconsideration of connectional information among the matched Cα atoms.
Neglect of this information might have made the algorithm crucially vulnerable to the similar ostensible shape, with different topology of the connecting backbones.
According to our previous finding [2], the proteins generally adopt strong globular structures, and the most significant difference among proteins is the topology of the backbone structure.
Thus, there is a possibility that TM-align might have been misled by the general globular positions of Cα atoms.
This partly indicates the robustness of backbone torsion angle alignments and the possible problems of 3D methods that neglect the connectional information of matched reference points.
The numerical measurement of accuracy of both of our new backbone torsion angle-based method and the typical 3D method of TM-align was performed with ROC curve analysis (Fig.4D).
Fortunately, our new method, with both logPr and RamRMSD measurements, showed perfect accuracy, with an AUROC of 1.0, which means that every pair-wise distance might be correctly classified as true or false pairs.
This clear discrimination reflects the long distance between the two groups in the trees of logPr and RamRMSD measurements (Fig.4A and 4B).
The TM-align method with RMSD and TM-score measurements, however, showed no marked improvements in this more clearly distinctive set than the set of protease subtypes of a previous study [1], possibly indicating less robustness of 3D methods in more distinctive cases of functional differences.
This is also shown by the similarity of the AUROCs of the RMSD (0.6846) and TM-score (0.6319) to the previous ones of 0.5965 and 0.5494 each.
The TM-score result showed worse performance than that of the RMSD result in the TM-align method assessment.
In this study, torsion angle-based representation of the protein backbone structure was utilized for the construction of a secondary structure information repository and analysis tool of pair-wise structure comparison.
This web application provides a search interface for deposited secondary structures of PDB and SCOP entries and a secondary structure calculation utility for the user&#39;s own structures.
It also supports a pair-wise protein structure comparison utility that is more accurate than the typical 3D alignment method in both cases of clear functional differences, as shown here, and very similar functional homologies, as shown previously [1].
The web server tries to explain in as much detail as possible the changes and occurrence of exceptions along the computational processes.
A secondary structure search through a query string using sequence alignment algorithms might be possible for further improvements.
Future protein structure research might be aided more through a secondary and tertiary structure homology search utility.
A tertiary and quaternary structural information repository might be possible to be newly built by automatically utilizing the possibility of 1D representations of protein structures using backbone torsion angles and better accuracy of structural homology delineations of the comparison tool that was provided here.
The information derived from this web application might be further utilized in the field of ab initio protein structure modeling or protein homology-related analyses.
The first page of the web application of the server (http://lcbb.snu.ac.kr/TorsWebDB).
Three functions-secondary structure database search, secondary structure calculation, and protein pair-wise structure alignmentsupplied with hyperlinks.
Sample query result of secondary structure database search.
The query was for the entries of the structural classification of proteins (SCOP) database with a helix content of 10.0%.
The sample result of a protein pair-wise structure comparison analysis.
The similiarity is informed through RamRMSD and logPr.
Clustering trees and accuracy analysis from intermingled kinases and proteases.
The clustering trees of 30 kinases (dark dots) and 30 proteases (pale dots) built with logPr (A), RamRMSD (B), and TM-align RMSD (C) are displayed.
Backbone torsion angle methods showed perfect delineation of kinases and proteases, while TM-align showed rather promiscuous clustering, with partially correct clusters of kinases.
Only the result of the RMSD of TM-align was displayed, which showed better accuracy than the TM-score result.
Receiver operating characteristic (ROC) curve (D) shows perfect performance of the backbone torsion angle alignment with area under the ROC curve (AUROC) of 1.0, while the TM-align method showed similar performance with an AUROC of 0.6846 (RMSD) and 0.6319 (TM-score) to that of a previous set of little functional difference [1].
Title: ERRATUM: Author&#39;s Affiliation Correction.
Heritabilities of Facial Measurements and Their Latent Factors in Korean Families

Title: http://dx.doi.org/10.5808/GI.2013.11.2.83

Title: Genomics Inform 2013;11(2):83-92

This erratum is being published to correct the author&#39;s affiliation.
Title: 1Genomic Medicine Institute, Medical Research Center, Seoul National University, Seoul 110-799, Korea,

Title: 2Department of Biomedical Sciences, Seoul National University Graduate School, Seoul 110-799, Korea,

Title: 3Department of Biochemistry and Molecular Biology, Seoul National University College of Medicine, Seoul 110-799, Korea,

Title: 4Division of Constitutional Medicine Research, Korea Institute of Oriental Medicine, Daejeon 305-811, Korea,

Title: 5Psoma Therapeutics, Seoul 153-781, Korea,

Title: 6Department of Epidemiology, Graduate School of Public Health and Institute of Health and Environment, Seoul National University, Seoul 151-742, Korea

Title: 1Genomic Medicine Institute, Medical Research Center, Seoul National University, Seoul 110-799, Korea,

Title: 2Department of Biomedical Sciences, Seoul National University Graduate School, Seoul 110-799, Korea,

Title: 3Department of Biochemistry and Molecular Biology, Seoul National University College of Medicine, Seoul 110-799, Korea,

Title: 4Psoma Therapeutics, Seoul 153-781, Korea,

Title: 5Department of Epidemiology, Graduate School of Public Health and Institute of Health and Environment, Seoul National University, Seoul 151-742, Korea,

Title: 6Division of Constitutional Medicine Research, Korea Institute of Oriental Medicine, Daejeon 305-811, Korea
Title: Editor&#39;s Introduction to This Issue

Earlier issues in this year have focused on the theme of epigenomics.
This issue starts with a review on the interplay between epigenetics and cancer genetics, which is followed by two review articles on cancer genomics.
One of the primary goals of cancer genomics is to translate its discovery to clinical practice.
Subsequent review articles provide perspectives on clinical informatics and methods on how to prioritize disease-related genes from next-generation sequencing data.
Genome-wide data should be interpreted in the context of systems biology, and the following review articles offers comprehensive information on network biology and its applications.
The next two review articles handle often overlooked but essential components of genomics, i.e., biobanks and ethical standards.
We have two additional reviews, totaling 10.
A couple of original research articles also address cancer genomics.
The remaining five articles and an application note cover a wide range of genomics and informatics.
In this issue, we have five articles in total that have been contributed from outside of Korea.
We hope that Genomics &amp; Informatics will advance next year to become a truly international journal that is authored and read worldwide.
Title: Interplay between Epigenetics and Genetics in Cancer

Genomic instability, which occurs through both genetic mechanisms (underlying inheritable phenotypic variations caused by DNA sequence-dependent alterations, such as mutation, deletion, insertion, inversion, translocation, and chromosomal aneuploidy) and epigenomic aberrations (underlying inheritable phenotypic variations caused by DNA sequence-independent alterations caused by a change of chromatin structure, such as DNA methylation and histone modifications), is known to promote tumorigenesis and tumor progression.
Mechanisms involve both genomic instability and epigenomic aberrations that lose or gain the function of genes that impinge on tumor suppression/prevention or oncogenesis.
Growing evidence points to an epigenome-wide disruption that involves large-scale DNA hypomethylation but specific hypermethylation of tumor suppressor genes, large blocks of aberrant histone modifications, and abnormal miRNA expression profile.
Emerging molecular details regarding the modulation of these epigenetic events in cancer are used to illustrate the alterations of epigenetic molecules, and their consequent malfunctions could contribute to cancer biology.
More recently, intriguing evidence supporting that genetic and epigenetic mechanisms are not separate events in cancer has been emerging; they intertwine and take advantage of each other during tumorigenesis.
In addition, we discuss the collusion between epigenetics and genetics mediated by heterochromatin protein 1, a major component of heterochromatin, in order to maintain genome integrity.
Cancers have traditionally been accepted as diseases that are induced by the accumulation of a set of genetic mutations that have been considered the major causes of neoplasia [1].
But, is genomic instability a consequence of cancer progression or indeed a cause that drives cancer development?
From about 1902 up until now, the involvement of genomic instability in tumorigenesis has long been the subjet of debate.
In the beginning, Theodor Boveri&#39;s work with sea urchins showed that all chromosomes are required for normal development, known as the Boveri-Sutton chromosome theory.
He had also suggested in 1902 that malignant tumors might be the result of a certain abnormal condition of the chromosomes, which may arise from multipolar mitosis.
In the last century, remarkable findings and progresses in cancer research have added genomic instability to the hallmark list of cancer.
Moreover, genomic instability generates the genetic and phenotypic heterogenicity and diversity that expedite tumorigenesis and the acquisition of cancer hallmarks.
Our genomes are constantly threatened by a number of challenges that are originating from extracellular and intracellular stimuli.
Effective cellular responses to these enormous genotoxic stresses are acutely required to maintain genomic integrity.
These stresses can induce a variety of problems in the DNA, such as double-strand breaks, single-strand breaks, oxidative lesions, and pyrimidine dimers.
Eukaryotes have evolved elaborate DNA damage response (DDR) mechanisms to properly respond to the genotoxic stresses of these lesions.
DNA damage induces several cellular responses, including checkpoint activation, DNA repair, and the triggering of apoptotic pathways for irreparable DNA damage (Fig.1).
Underlying these DDRs are functions of many tumor suppressors, which are frequently lost in cancers.
To keep the genome intact, DNA replication must be carried out without error, and if there is a mistake, it should be repaired correctly.
Also, to give time for cells to repair any defect, cells must have intact checkpoint responses.
Replicated genomes must be passed to daughter cells without error; therefore, either miscopying or missegregation of chromosomes should be avoided.
If these self-guarding machineries are not effective, cells that have problems should die.
So, DNA replication, DNA repair, transcription, cell cycle machinery, chromosome segregation, and apoptosis are six domains to keep genomic integrity.
Also, these DDRs are coordinately regulated and work together, via a series of signaling pathways that eventually activate surveillance and checkpoints for genomic integrity.
In other words, cancer cells have problems in these machineries, leading to genomic instability.
The word &#34;epigenetics&#34; was termed in the early 1940s to describe the events that could not be wholly explained by traditional genetics [2].
Conrad Waddington (1905-1975) defined epigenetics as &#34;the branch of biology which studies the causal interactions between genes and their products which bring the phenotype into being&#34; [3].
The epigenetic field now actively uncovers the molecular mechanisms underlying these phenomena, and epigenetics has been defined today as &#34;the study of changes in gene function that are mitotically and/or meiotically heritable and that do not entail a change in DNA sequence&#34; [4].
In other words, epigenetics is the study of changes in gene expression or phenotype, caused by mechanisms other than DNA sequences, and some of these epigenetic changes have even been shown to be heritable.
Our genetic information is stored in the form of DNA as genetic code, but it has to be copied into RNA as an intermediate messenger or non-coding functional component.
This copying process, transcription, is regulated at several steps, including initiation, elongation, and termination, by a variety of mechanisms.
Moreover, eukaryotic transcription has to deal with chromatin templates, such that histone modifications and chromatin remodeling vastly affect gene expression.
Several types of cancers and genetic disorders are often associated with abnormal transcription, highlighting the importance of regulated transcription in the pathology and prevention of cancers.
Part of the genome has no transcribed information, which, however, has functional roles by various mechanisms.
For example, cis elements (DNA sequences) can act as regulatory elements for transcription and replication.
Also, centromeric DNA is required for mitotic and meiotic chromatin structure, and telomeric DNA plays a role in sheltering the ends of chromosomes and eventually guarding the genome.
Indeed, centromeric and telomeric heterochromatin structures consist of specific repetitive sequences, which play a role in maintaining genomic integrity, such that heterochromatin structure is required for accurate chromosome segregation (centromere) and inhibiting DNA breakages and nonhomologous end-joining (telomere).
Aneuploidy and chromosomal rearrangement, which are frequently observed in many cancers, are in part caused by aberrant chromatin structure.
Generally, genomic instability is often associated with cancer and can be indicative of a poor prognosis for some types of cancer.
Lately, this concept has now been expanded to the field of epigenetics, and disorganization of epigenetic regulatory mechanisms is prevalent in cancer [5, 6].
Particular gene expression is determined not only by the genetic status of gene itself but also by epigenetic features, such as promoter regions, enhancers, and insulators, in chromatin and chromatin-modifying enzymes.
Mutations in tumor suppressors and/or oncogenes induce a loss or gain of function, respectively, and abnormal expression of these genes are likely to be linked to cancer initiation.
Feinberg proposed a hypothesis that epigenetic alterations can induce genetic changes and contribute to tumor progression and also initiation [7].
Epigenetic changes occur without a change in the DNA sequence, and they can be induced by various factors, such as chromatin structure, including DNA methylation, histone variants, and modifications; nucleosome remodeling; and small non-coding regulatory RNAs (Fig.2) [8].
In addition to histone proteins, DNA is associated with several nonhistone chromosomal proteins, including high-mobility group and heterochromatin protein 1 (HP1), which can also play an important regulatory role in DNA-dependent processes, such as transcription, replication, recombination, and DNA repair, in a similar manner to histones.
Title: Histone modifications and cancer

Nucleosomes are the basic units of chromatin that are composed of DNA wrapped around 8 histone core proteins [9].
One of best understood epigenetic mechanisms is histone modification.
The histone N-terminal tails protrude from nucleosome cores and thus are subject to numerous modifications, such as acetylation, methylation, phosphorylation, ubiquitination, ADP-ribosylation, and citrullination, creating a massively complex bar code.
It is thought that activating or repressive complexes are recruited to DNA and reshape chromatin into relaxed or a tightly packed structure, based on the histone code-instructions through a combination of histone modifications [10].
Like a writer, histone-modifying enzymes write the histone code via histone modifications and mark specific sites of the genome with histone modifications-for example, active promoter regions that are marked with augmented trimethylation in H3 at lysine 4 (H3K4me3) inactive promoters that are marked with augmented trimethylated H3 at lysine 27 (H3K27me3) or trimethylated H3 at lysine 9 (H3K9me3), and enhancers that are marked with monomethylated H3 at lysine 4 (H3K4me1) and/or acetylated H3 at lysine 27 (H3K27ac) [11-13].
These histone modification patterns are regulated by a variety of enzymes, including histone acetyltransferases and deacetylases, which introduce and remove acetyl groups, respectively.
Similarly, histone methyltransferases (HMTs) and demethylases introduce and remove methyl groups.
The combination of histone modifications largely affects gene expression by the recruitment of transcriptional regulatory proteins, which bind to specific modified histone markings.
Thus, a combination of histone modifications acts as a histone code-high-level information residing above the genetic code, which is genetic information stored in the form of DNA.
Aberrant patterns of histone modification marks are often found in cancer.
Advances in high-throughput sequencing have allowed for genome-wide mapping of chromatin changes that occur during tumor initiation and progression [8].
A recent report suggested that cancer cells often experience losses of histone acetylation and methylation, occurring predominantly at the acetylated Lys16 and trimethylated Lys20 residues in histone H4.
These losses are also associated with the hypomethylation of repetitive DNA sequences, which is a well-known characteristic of cancer cells [14].
Title: DNA methylation and cancer

One of the best studied epigenetic signals is DNA methylation.
In contrast to a dazzling variety of histone modifications, DNA methylation is a simple covalent chemical modification, resulting in the attachment of a methyl (CH3) group at the 5&#39; carbon position of the cytosine ring.
When methyl groups are attached to the DNA in genes, transcription of these genes is usually turned off, and then these genes are silenced.
When a cell divides, its DNA is copied via replication and divided equally into two daughter cells during mitosis.
During this process, the pattern of DNA methylation can also be copied onto the new daughter DNA, allowing the information that determines whether a gene is &#34;on&#34; or &#34;off&#34; to be inherited to the two daughter cells [15].
DNA methylation in mammals occurs mostly at CpG dinucleotides, and methylation of CpG islands explains a stable gene silencing mechanism [15].
In normal somatic cells, most (over 50%) CpG islands are unmethylated.
DNA methylation is important for the regulation of non-CpG islands, CpG island promoters, and repetitive sequences to maintain genome stability [15, 16].
Furthermore, DNA methylation plays important roles in X chromosome inactivation, imprinting, embryonic development, silencing of repetitive elements and germ cell-specific genes, differentiation, and maintenance of pluripotency [16-18].
DNA methylation is controlled by a family of DNA methyltransferases (DNMTs) that catalyze the transfer of methyl groups from S-adenosyl-L-methionine to the 5&#39; position of cytosine bases in the CpG dinucleotide.
Methyl-binding domain (MBD) proteins, such as MeCP2, MBD1, MBD2, and MBD4, bind to methylated CpG sites and are involved in transcriptional repression [19].
In cancer, hypomethylation usually occurs at repeated DNA sequences, such as long interspersed nuclear elements, whereas hypermethylation predominantly involves CpG islands [20].
DNA hypermethylation has been shown to result in abnormal silencing of several tumor suppressor genes in most types of cancer [21].
Thus, DNA methylation is grossly reduced, but DNA hypermethylation is specifically and locally augmented at CpG islands at tumor suppressor genes (Table 1); yet, its involvement and role in tumorigenesis remain controversial.
Yet, experimental progress has added an emerging epigenetic aberration, &#34;aberrant DNA methylation,&#34; to an established cancer hallmark, &#34;genomic instability.&#34;
DNA hypermethylation at tumor suppressors or genes having tumor suppressive functions is more frequent than their mutations in cancer cells.
Hundreds to thousands of genes are hypermethylated in cancer cells, while only tens of genes are mutated.
Epigenetic silencing via DNA methylation results in gene inactivation and promotes carcinogenesis, indicating that DNA methylation impinges on carcinogenesis.
Title: MicroRNA and cancer

MicroRNAs (miRNAs) constitute a class of small noncoding RNAs that have epigenetic functions in posttranscriptional regulation of gene expression via mRNA degradation or sequestering and translational repression and have physiologically essential roles in most cellular processes.
Therefore, their dysfunction is implicated in human diseases, including cancer, as well as inherited diseases and developmental defects.
The human genome may encode over 1,400 miRNAs, which frequently target many genes related to cancer development or prevention [22].
In light of the linkage between miRNA and cancer, they have been categorized into three groups: oncogenic, tumor-suppressive, or context-dependent miRNAs [23].
In tumorigenesis, the loss of tumor-suppressive miRNAs promotes the expression of target oncogenes, while increased expression of oncogenic miRNAs can repress target tumor suppressor genes [23].
Mutations in a miRNA can disrupt its recognition of binding targets and further result in oncogene activation and/or tumor suppressor repression.
Indeed, oncogenic miRNAs, such as miR-155, miR-21, and miR-17 to -92, whose expression triggers cancer development, appear to be amplified, and tumor suppressive miRNAs, such as miR-146 and miR-15 to -16, whose loss triggers cancer development, appear to be downregulated in cancers [23].
Moreover, mutations of miRNAs, including miR-101 and miR-29, targeting epigenetically modifying enzymes, such as EZH2 [24, 25] and DNMT3 [26], respectively, are often found in human cancers.
Aberrant expression or mutation of such miRNAs is likely to further reinforce extensive and heavy epigenetic alterations [23, 27] and might have effects on histone or DNA methylation at promoters of other miRNAs that target oncogenes and tumor suppressors.
Underexpression of miR-127, which targets BCL6, is correlated with abnormal methylation at its promoter in cancer [28].
In the course of remarkable progress in the study of cancer epigenetics, new observations linking aberrant miRNA expression and tumorigenesis imply that miR-mimic molecules might serve as effective therapeutics by replacing tumor suppressor miRs or targeting oncogenic miRs.
Many types of human cancers and disorders/diseases are often induced by misregulation of gene expression.
Mechasnims for this misregulation involve both genetic mutations (genetic instability) and epigenetic modifications that disrupt the function of genes, including tumor suppressor and oncogenes, as well as other genes related to cancer, via altered activity or expression of the products of the genes.
Given the importance of epigenetic dysregulation in tumor initiation and development, differentiating causative &#34;drivers&#34; (causes) and resulting &#34;passengers&#34; (outcomes) is becoming an important priority for cancer biology as well as epigenetics.
Driver genes have to be essential for cancer causation, whereas passenger genes are not necessary [29].
By the development and improvement of technology, it may eventually be possible to accurately and minutely distinguish epigenetic aberrations of driver genes [30, 31].
Many current clinical investigations and trials of epigenetics-based cancer therapy, highlighting the reciprocal regulation of epigenetically modifying enzymes, miRNAs, and genetic defects in cancer, have come from scientific discoveries of how epigenetic aberrations in cancer manage to play a key role at every stage of tumorigenesis and have such a profound impact on the underlying mechanisms of tumorigenesis.
Nevertheless, cooperation between epigenetics and genetics in tumorigenesis (Fig.3) is the subject of debate.
The interconnection between these two processes becomes increasingly apparent, in large part because it is realized that mutations of various epigenetic modifier genes are associated with human cancers [32].
The mutation of epigenetic modifiers presumably fosters profound epigenetic anomalies in terms of DNA methylation, histone modifications, miRNA, and nucleosome positioning and occupancy, which expedites global dysregulation of gene expression.
Underlying this dysregulated epigenome is genomic instability, which generates mutations in epigenetic modifiers that induce abnormal gene expression, which may predispose one to cancer [32], although this remains to be demonstrated.
Insight into its underlying mechanisms helps establish the epigenetic involvement in tumorigenesis.
Because epigenetic processes involve a number of overlapping and interconnected mechanisms and pathways, it is difficult to definitely identify the cellular and molecular identities and the mechanism(s) underlying the epigenetic outcomes and phenotype.
Based on studies of cell lines that have originated from tumor tissues and tissue samples collected from cancer patients, aberrant epigenetic marks can result in inappropriate expression of genes that are associated with cancer (tumor suppressors and/or oncogenes).
It is now generally accepted that human cancer cells harbor global epigenetic abnormalities and that epigenetic alterations may be the key to tumor initiation and development [5, 6, 8].
The cancer epigenome is characterized by substantial changes in various epigenetic regulatory layers; herein, we introduce some important examples of epigenetic disruptions that cause mutations of key genes and/or alterations of signaling pathways in cancer development.
However, this idea has now been expanded to be a scientific norm by consolidating and incorporating the altered epigenetic regulatory mechanisms that are common and prevalent in cancer [5, 6].
Both genetics and epigenetics ultimately involve abnormal gene expression-i.e., phenotype.
The expression level of a particular gene is determined by its context at the genetic level, which consists of several cis elements, including promoters, regulatory regions, enhancers, and insulators, each of which recruits specific transcriptional proteins for activation, repression, and constitutive or facultative transcription by various mechanisms.
In other words, cis elements can act as a primary platform for recruiting epigenome-modifying enzymes and/or chromatin remodeling factors.
The modified epigenome follows immediately and acts as a secondary platform for transcriptional effector machinery.
Finally, transcription is turned on or off, and its level is determined by the overall epigenomic context.
The genetic road to cancer is relatively straightforward: mutation of tumor suppressors and/or oncogenes causes either a loss or gain of function and abnormal expression-namely the very relationship between altered genotypealtered phenotype.
On the contrary, there are no genetic changes but a sick phenotype in the epigenetic road to cancer.
The epigenetic pathway to cancer is much more complicated and roundabout and is determined by integrating numerous epigenetic variations, including DNA methylation, histone variants and modifications, and nucleosome remodeling, as well as small non-coding regulatory RNAs (Fig.2) [8].
During tumor initiation and progression, the epigenome undergoes multiple alterations, including a genome-wide loss of DNA methylation (hypomethylation), frequent increases in promoter methylation of CpG islands, and aberrant changes in nucleosome occupancy and histone modification profile, and these epigenetic aberrations drive tumor cell heterogeneity, as well as tumorigenesis.
Epigenetic aberration is a major feature of cancer, but its etiological role in tumor development remains controversial.
This contrasts with genomic instability, which is known to produce cancer-causing mutations.
Recent evidence demonstrates that a dysregulated epigenome can promote tumorigenesis via altered gene expression; however, the question of whether a dysregulated epigenome can predispose one to DNA damage and generate genomic instability remains clearly unaddressed.
Only few possible mechanisms have been proposed; one possible mechanism by which chromosome segregation errors promote tumorigenesis is by inhibiting chromosome condensation for mitotic entry or destabilizing heterochromatin at centromeres, which facilitates chromosomal misalignment and accelerates the formation of unattached chromosomes or malfunctioning spindles.
A second mechanism is chromosomal breakage and non-homologous end-joining, which result in chromosomal rearrangement and extensive structural alterations in chromosomes.
Third, a dysregulated epigenome at telomeres promotes its elongation or chromosomal rearrangement.
Intriguingly, histone deacetylase inhibitors or knockdown/knockout of epigenome modifiers can induce DNA damage, but whether this DNA damage can indeed predispose one to cancer remains unestablished.
Title: Heterochromatin protein 1

HP1 was originally identified in Drosophila, functioning in heterochromatin-mediated gene silencing.
Three different paralogs of HP1 are found in Drosophila melanogaster: HP1a, HP1b, and HP1c.
Subsequently, orthologs of HP1 were also discovered in Schizosaccharomyces pombe (Swi6), xenopus (Xhp1α and Xhp1γ), chicken (CHCB1, CHCB2, and CHCB3), and mammals (HP1α, HP1β, and HP1γ) (Fig.4) [33, 34].
HP1 binds directly to the methylated K9 residue of histone H3 (H3K9me), a surrogate marker for transcriptionally repressive heterochromatin, and is critical for its maintenance [35-37].
Therefore, its canonical functions include maintaining heterochromatin integrity as a fundamental unit of heterochromatin, silencing by heterochromatin formation, and gene repression by heterochromatization of euchromatin.
HP1 proteins are characterized by two conserved domains: the chromodomain (CD) in its N-terminus and the chromo shadow domain (CSD) in the C-terminus (Fig.4).
The CD has been shown to directly bind H3K9me, while the CSD is implicated in interacting with a partner protein and its homo- and hetero-dimerization.
Two CD and CSD domains are separated by a hinge domain that is involved in DNA and RNA binding (Figs.4 and 5) [38, 39].
HP1 interacts with numerous epigenomic modifiers with different cellular functions in different organisms (Fig.5).
Some of these HP1-interacting partners are histone methyltransferse, DNMT, and methyl CpG-binding protein MeCP2 (Fig.5), supporting its role in epigenomic modification.
Title: HP1 and chromatin structure

HP1 proteins are mostly enriched at heterochromatin centromeres and pericentromeric regions, telomeres and subtelomeric regions, and transcriptionally repressive genes.
However, HP1 is also found at euchromatic sites [40, 41], though whether euchromatic HP1 has a disparate function and which HP1 paralog is located at euchromatin remain unclarified.
A structure-based study revealed that a hydrophobic pocket of the HP1 CD interacts with histone H3K9me [42].
This epigenetic mark is generated by a conserved family of HMTs, named after the Drosophila member SU (VAR)3-9, discovered as a suppressor factor involved in position-effect variegation [43, 44].
Both HP1 and SU (VAR)3-9 function in heterochromatin structure formation.
Loss of SU(VAR)3-9 results in displacement of HP1 from heterochromatic regions and alteration in gene repression [45].
Mechanisms by which HP1 localizes to euchromatin sites appear to involve more than the recognition of H3K9me, which is poorly understood.
An alternative mechanism of localization might be mediated via interactions between its CSD and other factors.
HP1 CSD homodimerizes through an alpha-helical region and generates a platform that can interact with the PxVxL motif in its interacting partner proteins, such as DNMT1/3, SU(VAR) 3-9, and the p150 subunit of CAF-1 (Fig.5) [46].
A second alternative mechanism of localization on chromatin involves interactions with RNA through a hinge domain, such that association of HP1 with specific loci in Drosophila and centric regions in mouse makes them susceptible to RNase treatment [39, 47].
For example, it has been demonstrated that an HP1 subcode with sumoylation plays a role in heterochromatin architecture, via its association with microsatellite RNAs [48].
A third mechanism for chromatin localization is found at the distal ends of Drosophila telomeres, where HP1 is thought to bind DNA directly [49].
However, in mammals, localization of HP1 proteins to telomeric regions occurs via H3K9 association [50-53].
Together, HP1 localizes to chromatin using both genetic (DNA or RNA sequence) and epigenetic (histone modification) information.
Title: HP1 subcode: above the histone code

Recently, the &#34;HP1 subcode&#34; hypothesis is emerging in addition to the genetic code and histone code: a hypothesis that the transcription of genetic information encoded in DNA-i.e., genetic code-is regulated in part by histone variants, modifications, and nucleosome occupancy.
The histone code hypothesis states that the type and combination of histone modifications are interpreted by the cell as instructions for organizing gene expression and constitutes the most advanced conceptual framework for deciphering the genetic information cascades that regulate gene expression.
Further, a combination of HP1 paralogs and modifications may be defined as a second regulatory layer of the code-that is, a subcode-involved in transcription, especially gene silencing, within the general context of the histone code.
The HP1 subcode can be secondary-level information residing above the genetic and histone codes and regulate HP1 interactions with other key proteins, further modulating gene silencing and possibly replication, recombination, and DNA repair.
In this review, we will discuss the interplay between genetics and epigenetics in cancer through HP1.
Title: Pleiotropic functions of HP1

Besides heterochromatin formation and the maintenance functions of HP1, this protein also recruits the cohesin complex to pericentromeric heterochromatin for centromeric sister chromatid cohesion.
Recent studies have shown that HP1 does not always act in the context of heterochromatin and functions in gene activation and telomere maintenance [38].
Furthermore, HP1 interacts with many different proteins, including transcription factors, chromatin regulators, and DNA replication and repair factors, as well as components of the nuclear envelope [54], and functions via collaboration with its interacting proteins in diverse cellular processes, including mitosis progression [55, 56], DNA damage repair [57-59], transcription [60-62], and chromatin remodeling (Figs.4 and 5) [63, 64].
Title: HP1-mediated transcriptional regulation in response to DDR

Recently, we proposed a mechanism in which HP1 could recover transcription in response to DNA damage signals [60].
HP1 accumulates at the promoter before DNA damage, but BRCA1 is recruited to the promoter after the damage while promoter-resident HP1 is disassembled.
Importantly, HP1 assembly is recovered post release from the damage in a BRCA1-HP1 interaction-dependent manner and simultaneously targets SUV39H1 to chromatin.
HP1/SUV39H1 restoration at the promoter results in BRCA1 disassembly and histone methylation.
In the aftermath, transcriptional repression resumes.
This report provides a partial explanation for the targeting of HP1 to a variety of chromatin structures and its functions in constructing de novo heterochromatin from euchromatin or facultative heterochromatin, in addition to maintaining previously existing heterochromatin.
In addition, a role in the DDR for HP1, previously implicated in heterochromatin and silencing, reveals new connections between tumor-suppressive processes that maintain genome integrity.
Title: HP1 depletion activates DDRs

Recent studies have revealed the possible link of HP1 proteins to the DDR [59, 65, 66].
However, the spatial and temporal regulation of the association and dissociation of HP1 with chromatin in response to DNA damage remains unclear and controversial.
A couple of studies have reported that DNA damage induces the transient dissociation of inhibitory HP1 proteins from DNA damage sites in order to help DNA repair proteins/complexes gain access to chromatin [65, 66].
In contrast, another study has indicated that HP1 is recruited and associates at DNA damage sites, such as UV-induced DNA lesions and chromosomal breaks upon DNA damage, which helps the DDR via its dynamic dissociation and association with DNA damage sites [59].
Recently, Lee and colleagues [67] have shown that HP1 promotes homologous recombination repair via recruitment of BRCA1 to damaged chromatin.
Together, chromatin protein HP1 plays some unelucidated roles in keeping genomic as well as epigenomic integrity.
Title: Whether and how HP1 is related to cancer

An aberrant epigenome in cancer cells results from an altered chromatin structure resulting from alterations in chromatin-modifying enzymes and/or chromatin proteins.
Examples of HP1γ-interacting proteins are the histone proteins, epigenetic modifiers, transcription factor family, DNA repair factors, and replication proteins (Fig.5) [48, 54-57, 60, 62, 63, 67-80].
The diversity of HP1γ-associated proteins has increased with the recent discovery that HP1γ interacts with BRCA1, a tumor suppressor that guards the genome.
In addition, HP1γ is involved in the recovery of BRCA1-mediated transcription post-DDR.
There is no direct evidence that HP1γ is a component of BRCA1-mediated tumor suppression.
However, a couple of tantalizing observations support this possibility: HP1γ expression is elevated in prostate cancer [81], and in addition, depletion of HP1 paralogs activates the formation of DNA damage repair foci.
HP1 paralogsare recruited to repair foci and promote DNA repair, following DNA damage.
This raises the question of whether increased or decreased HP1γ can foster or suppress tumor initiation and development.
In light of the HP1γ-BRCA1 association, it would be intriguing to test whether the genes encoding the HP1 family are also overexpressed in breast cancers.
It would also be interesting to test whether the HP1 family impinges on genomic stability.
On the road to cancer: genomic instability.
Failure of DNA damage responses that conduct surveillance and checkpoints to guard genome causes genomic instability, the most common characteristic of human cancers.
It has therefore been proposed that genomic instability contributes to and drives tumor initiation and development.
Defects in the DNA damage responses generate genomic instability and facilitate tumorigenesis.
Epigenetic mechanisms.
Variations in chromatin structure but not DNA sequences modulate the use of the genome by 1) DNA methylation, 2) histone modifications (methylation, phosphorylation, and acetylation), 3) histone variant composition (dark), 4) chromatin remodeling (sparse or dense nucleosome occupancy), and noncoding RNAs.
Epigenetic silencing is more frequent that mutations in cancer cells.
In cancer cells, hundreds to thousands of genes are hypermethylated (DNA methylation), histone codes at more than thousands of genes are aberrantly modified via histone modifications or histone variant compositions, and miRNA expression is dysregulated, while tens of genes are affected by mutations.
Both epigenetic silencing and inactivating mutations or deletions result in gene inactivation.
(A) Heterochromatin protein 1 (HP1) paralogs in human.
Amino acid sequence alignment of HP1α, β, and γ.
(B) A schematic diagram of the HP1γ polypeptide.
The HP1γ polypeptide has an N-terminal chromodomain, a hinge domain, and C-terminal chromoshadow domain.
Interactions of heterochromatin protein 1 (HP1) with a diversity of proteins and its possible roles (references in parentheses).
The putative cellular functions of protein-protein interactions of HP1 are shown in circles.
Some of their biological significance is not yet clarified.
Only the proteins that have been shown to interact with HP1 in vivo are listed here.
Title: Genes are epigenetically regulated in cancer
Title: Application of Cancer Genomics to Solve Unmet Clinical Needs

The large amount of data on cancer genome research has contributed to our understanding of cancer biology.
Indeed, the genomics approach has a strong advantage for analyzing multi-factorial and complicated problems, such as cancer.
It is time to think about the actual usage of cancer genomics in the clinical field.
The clinical cancer field has lots of unmet needs in the management of cancer patients, which has been defined in the pre-genomic era.
Unmet clinical needs are not well known to bioinformaticians and even non-clinician cancer scientists.
A personalized approach in the clinical field will bring potential additional challenges to cancer genomics, because most data to now have been population-based rather than individual-based.
We can maximize the use of cancer genomics in the clinical field if cancer scientists, bioinformaticians, and clinicians think and work together in solving unmet clinical needs.
In this review, we present one imaginary case of a cancer patient, with which we can think about unmet clinical needs to solve with cancer genomics in the diagnosis, prediction of prognosis, monitoring the status of cancer, and personalized treatment decision.
Despite the explosive increase in cancer genomics data, the actual application of genomics data and approach to clinics offers big challenges [1, 2].
The main reason for the delay in the application to clinics is non-connectivity between bioinformaticians, non-clinical scientists, and clinicians.
Clinicians are not confident in how much additional information might be given through genomics technology compared to conventional tools, and non-clinical scientists and bioinformaticians are not aware of what clinical problems should be solved with priority.
Cancer clinicians expect novel diagnostic, prognostic, and therapeutic values of this new technology [1].
Now, the clinical field is aware that multiple assays using next-generation sequencing (NGS) has some advantage over single genetic tests [1, 3].
A recent cancer guideline from the National Comprehensive Cancer Network (NCCN) pointed out that epidermal growth factor receptor (EGFR) and/or anaplastic lymphoma kinase (ALK) testing should be conducted as part of multiplex/NGS in non-small-cell lung cancer (NSCLC) [4].
The feasibility of an NGS diagnostic platform at cancer clinics was validated by cancer centers in the United States [5] and Europe [6].
One platform showed that potential hurdles, time, cost, purity, and ethical problems could be overcome in implementing NGS using low-coverage whole-genome sequencing, whole-exome sequencing (WES), and whole-transcriptome sequencing [5].
Another platform sequenced clinical samples, including formalin-fixed, paraffin-embedded ones, and showed that this approach is practical and can give some benefits to the patients with lung cancer [6].
The WES approach in Mendelian disorders showed that ~25% of sequencing can give a specific diagnosis based on the germline genetic make-up from the data of 250 probands, mostly consisting of neurological phenotypes in real practice [7].
The clinical application of cancer genomics should be considered in the context of unmet clinical needs to maximize the value of this novel paradigm.
Here, we dealt with the potential clinical application of cancer genomics in the aspect of practical management of cancer patients, along with a clinical scenario covering the whole course of cancer management.
We overviewed the potential of cancer genomics in the context of real cancer practice through this approach.
A 64-year-old male presented with a cough since 2 months ago.
He quit smoking 10 years ago and had smoked an average of one pack per day for 30 years.
He had regular check-ups every year and had the last check-up with no abnormality 6 months ago.
His chest roentgenogram showed a mass shadow in the lung.
He got a percutaneous needle aspiration (PCNA) and biopsy but did not have any tumor cells.
He repeated the PCNA and biopsy but developed pneumothorax (air leakage from lung) related to the diagnostic procedure.
His second biopsy showed some malignant cells, suggestive of adenocarcinoma.
After a staging work-up with a computed tomography (CT) scan, positron-emission tomography (PET) scan, brain magnetic resonance imaging scan, and bronchoscopy, his tumor was revealed as T2N0 clinical stage.
He got a surgery based on the result of the pathology and staging work-up.
In this situation, we face unmet clinical needs of early cancer diagnosis.
He was diagnosed with symptoms related to the cancer, despite his regular check-up.
If we detect his malignancy before the symptoms, we can get a better result compared with the symptomatic case [8].
This imaginary patient was diagnosed with operable cancer, but as many as 60% to 70% of lung cancer patients are diagnosed with inoperable, advanced disease [9].
To diagnose the cancer sooner, screening procedures have been studied extensively in the clinical field.
Lots of trials failed to show an actual benefit of screening strategy in lung cancer [10, 11], but recently, a screening trial using low-dose chest CT of lung cancer was proven to be successful [8].
Low-dose chest CT reduced the mortality from lung cancer through early detection in people with a history of at least 30 pack-years of smoking.
Despite this good approach to detect early lung cancer, the reduction of mortality is 20% [8].
This advantage should be counterbalanced with the potential harmful effects of radiation exposure or false positivity [12].
One important usage of genomics should be early detection.
Many biomarkers were investigated for the early detection of lung cancer-for example, circulating tumor DNA and RNA [13].
But, false positivity and false negativity of single tests prevent these tests from clinical usage.
If we use multiple biomarkers using cancer genomics for the detection of early cancer, the false positives and negatives could be reduced to the level of clinical usage.
Another effective approach of early diagnosis of cancer is to select a high-risk population for screening tests.
Many genome-wide association studies identified some germline single-nucleotide polymorphism (SNPs) that are associated with the risk of cancers [14-16], and these findings suggested important potential germline markers for the definition of high-risk populations.
Dr. Mardis told the case of Mike Snyder as an example of an actual application of a germline genome test [17].
He sequenced his own genome and found a higher risk of type 2 diabetes; since that time, he checked his glucose level regularly, and he detected high glucose levels at the very beginning.
After that, he exercised regularly and returned to normal glucose levels.
The concept of early diagnosis, like the story of Mike Snyder, can be applied to the early detection of cancer, too.
If we define a high-risk population with germline genetic tests, we could focus on the use of screening tests in this population.
This approach will also save money on screening tests for the whole population.
This patient had a bad experience of pneumothorax.
This complication gives pain and discomfort of the sensation of dyspnea, prolongs the period of admission, and increases the cost of cancer management.
This problem originated from the invasive nature of the diagnostic procedure-PCNA and biopsy using a needle, moving back and forth through the patient&#39;s fragile lung tissue.
New diagnostic tools using cancer genomics-for example, identification of cancer-specific genetic alterations from circulating tumor DNA-can avoid this type of complication from invasive procedures.
One pioneering study showed that circulating tumor DNA could be combined with genomic technology to diagnose and monitor the status of cancer very safely [18].
He underwent surgery performed by thoracic surgeons.
His final pathological diagnosis was adenocarcinoma and revealed the involvement of some mediastinal lymph nodes; therefore, the final pathological stage was T2N2, IIIA.
This stage required adjuvant chemotherapy, and he was referred to the department of medical oncology.
A medical oncologist prescribed adjuvant chemotherapy-a navelbine and cisplatin regimen (NP) for 4 cycles.
NP 4 cycles has been prescribed to patients with high-risk clinical stage IB, IIA, IIB, IIIA, and IIIB, regardless of other clinicopathological factors.
The actual benefit from this treatment is only a 4% long-term survival advantage, with some hematological and non-hematological toxicities [19, 20].
The way to improve the risk-to-benefit ratio of adjuvant treatment is further selection of a patient population that could benefit from this treatment.
Stage-based risk calculation-the current practice-cannot predict an accurate risk of relapse and attenuates the value of adjuvant treatment.
Single patient-based risk calculation should be done to identify the adequate patients for adjuvant treatment.
Cancer genomics will provide useful tools for a single patient-based risk calculation model.
This personalized adjuvant model will definitely increase the proportion of patients who benefit from it.
There are also clinical needs of prognosis prediction even before surgery.
Surgical management of stage IIIA is controversial.
If he had been diagnosed as stage IIIA preoperatively, he might have chosen a no-surgery option, such as concurrent chemoradiation, because no definite benefit of surgery was demonstrated until now, although this is controversial [21, 22].
We have relatively sensitive diagnostic methods, such as CT, PET, and endobronchial ultrasonography, but the sensitivity and specificity are still not perfect.
One focus of genomics technology should be a more accurate prognosis preoperatively, in addition to postoperatively.
Prediction issues in cancer patients are not limited to survival, recurrence, and response to specific treatment.
The goal of cancer care is improvement of survival and, more importantly, quality of life.
Predictive markers of bone metastasis or bone-related events, such as fracture or paralysis, are very important to improve the quality of life in cancer patients.
We have no clinically validated markers to predict these events [23].
An unbiased approach is needed to discover good biomarker for these events, because we have very limited clues to dig into these problems.
The most important advantage of genomics approach is that is &#34;unbiased&#34; with regard to any hypotheses or theories.
That is why clinical problems with low-depth knowledge are a good candidate for the application of cancer genomics.
The connection between clinicians and genomic scientists can facilitate the development of clinical problems that are solved with a genomic approach.
He got 4 cycles of adjuvant chemotherapy and then followed up regularly with a physical examination, blood test, CT scan, and/or PET scan.
At 6 months since the completion of adjuvant chemotherapy, he presented with right-side chest pain, and his CT scan showed tumor relapse in the right lung and pleura.
He planned systemic anticancer treatment for life prolongation and palliation of symptom, not curative.
The role of CT scan and/or PET scan is not defined [24], but many clinicians use this high-cost image for monitoring relapse.
The reason for the popular use of this high-cost image is that there is no other alternative of monitoring cancer status.
Recent studies showed the possibility of circulating tumor cell and cell-free DNA as a tool for monitoring tumor status [18, 25].
Cancer genomics enables circulating tumor cell or cell-free DNA to detect relapse earlier than image findings or tumor markers.
The delicate delineation of circulating DNA with NGS technologies is an essential part of this success story.
No tool is available with a definite clinical benefit for the monitoring of cancer; therefore, we absolutely need new, non-invasive tools for it.
Because it takes at least several years to prove the benefit of monitoring, we should start a well-designed prospective study using cancer genomics to prove it as soon as possible.
The medical oncologist who was in charge of him tested for EGFR mutation and ALK by fluorescence in situ hybridization to see rearrangements of ALK.
Tests showed no EGFR mutation and no ALK translocation.
He chose cytotoxic chemotherapy with pemetrexed and cisplatin according to the double-negative results of the EGFR and ALK tests.
The ultimate goal of the clinical application of cancer genomics should be therapeutics.
There are increasing gene lists for NSCLC that have matching targeted agents [26, 27], but many patients do not have genetic alterations that can be a target.
Even for the same type of cancer-for example, lung adenocarcinoma-the genetic make-up is very heterogeneous.
The accurate definition of the genetic make-up of each patient will be the first step for personalized medicine.
Many target panels have been developed by many vendors and hospitals for commercial or non-commercial use [3], but the usage is still very limited.
As more cancer genome data are available [28, 29], the list of target panels will increase and be specified to tumor types.
Now, we have two targeted agents to prolong the survival of lung cancer patients: the EGFR tyrosine kinase inhibitor (TKI) erlotinib or gefitinib for EGFR mutations and the ALK TKI crizotinib for ALK translocations.
For this case, there was no targeted agent, because he did not have EGFR or ALK alterations.
Based on the histology of his tumor, he was treated with pemetrexed and cisplatin [30].
Despite the substantial toxicity of cytotoxic chemotherapy, many patients are prescribed these drugs with the individualized possibility of this agent not known.
This decision relies on population data from large-scale prospective trials.
Because clinical trials targeting specific genetic alterations are increasing, the clinical need for more genetic information is also increasing [31].
If the patient was diagnosed as stage IV initially, the amount of tissue for genetic testing is very limited.
The amount of tissue is enough for 2 or 3 genetic tests.
For these patients, multiplexing or NGS technologies have important advantages to overcome the limitation of tissue amounts in the clinic.
This patient got adjuvant chemotherapy with NP for 4 cycles but progressed 6 months after the completion.
We guess that the patient did not benefit from NP for 4 cycles.
The usual speculation is whether there is a better way to choose adjuvant treatment.
The way to increase the benefit of adjuvant treatment could be the application of variable adjuvant treatment that is selected from individualized factors.
Personalized cancer genomic analysis can show different and private profiles of individual cancers, and accumulation of these genomic data enables different adjuvant treatment strategies for each patient.
Despite the theoretical promise of personalized cancer genomic analysis for the selection of the right drug, there are few examples of this direct application of cancer genomics.
One of important example is the story of Lukas Wartman, a doctor at Washington University in St. Louis [32].
He suffered from a second relapse of acute lymphoblastic leukemia and sequenced the tumor DNA and RNA.
He found very high expression of FLT3 and then figured out that sunitinib, a kidney cancer drug, could block tumor survival signals.
Sunitinib induced complete remission in him, and thereafter, he received allogeneic stem cell transplantation and is now in remission.
Another story from the Fox Chase Cancer Center was reported on KIT mutations from a pancreatic neuroendocrine tumor [33].
The patient had an advanced pancreatic neuroendocrine tumor and sequenced his tumor using a cancer panel.
He had a KIT mutation and was treated with imatinib, a well-known KIT inhibitor that is not approved for his tumor, for more than 2 years.
If he had been in in routine practice without cancer panel sequencing, he would have died without imatinib treatment.
A pharmacogenomic approach could be used to minimize toxicity to cancer treatment, too.
In this case, we usually see germline SNP data.
Some germline SNPs are already known to predict erratic side effects to specific drugs.
One example is the UGT1A1 germline polymorphism for the metabolism and toxicity of irinotecan.
If the patient has a homozygous polymorphism in the UGT1A1 gene-the *28 variant-he should be prescribed with a dose reduction to avoid severe, sometimes fatal, toxicity from irinotecan.
A genomic approach is required to properly assess pharmacogenomics in association with drug toxicity.
A commercial chip, the Affymetric DMET chip, covers 25-32% of genes of pharmacogenomics, and it is an important question whether more coverage with WES will be more beneficial.
He was treated with pemetrexed and cisplatin for 4 cycles, and his tumor shrank with the treatment.
But, his tumor rebounded, with a complaint of dyspnea and cough, 4 months after the completion of treatment.
After 4 cycles of second-line chemotherapy with docetaxel, he had stable disease.
Even in the case of a dramatic response to the first-line treatment, nearly all cancers rebound with various relapse-free durations [30, 34].
The choice of second-line treatment depends on clinical guidelines, clinical parameters, and clinicians&#39; experiences.
Any of the parameters may not be best if a specific patient&#39;s situation is not considered.
We do not know the exact mechanism of resistance to the previous treatment and do not have any good way to assess the mechanism of resistance.
If cancer genomics is adopted in this situation, we can profile the tumor tissue that was resistant to the previous treatment.
The profile can help clinicians decide on a second-line treatment option and improve the individual&#39;s outcome.
The resistance mechanism could vary, depending on multidimensional factors, such as the patient, tumor biology, and previous treatment.
Especially, this situation should be approached individually with the concept of true personalized medicine.
His disease began to grow again and he refused further cytotoxic chemotherapy.
He was in hospice care and died 1 year after the initial diagnosis of relapse.
After his death, his tumor was sequenced by whole-transcriptome analysis, and his tumor showed a ROS1 translocation.
Now, a ROS1 inhibitor is available for this type of cancer.
The median overall survival of metastatic lung cancer patients is 1 year if there is no genetic alteration for which targeted agents are available [30, 34], as in this case.
If the patient has an EGFR-sensitive mutation or ALK translocation, the median overall survival time reaches 2 years [35, 36].
A ROS1 targeting agent, another example of successful targeting of oncogene addiction, is now available after his death [27].
If the ROS1 translocation was known for his tumor and he was treated with a ROS1 inhibitor, he could have a chance to live an additional one year or more.
Tumors have very heterogeneous profiles [37, 38], and resistant tumors have even more heterogeneous ones [39].
This is why tumors should ultimately be approached on an individual basis, rather than a population basis.
Now is the right time to think about how we can use cancer genomics for individual patients.
There are still lots of technological hurdles-e.g., functional annotation of sequencing results, the way to integrate multi-dimensional data, and the exploration of the clinical meaning of sequencing variants for the routine clinical usage of cancer genomics.
Defining unmet clinical needs is as important as overcoming technological hurdles to maximize the benefit from cancer genomics.
Early detection, non-invasive molecular diagnosis, risk prediction of relapse and tumor-related events, monitoring of disease status, matching good drugs, and finding out personal grade resistance mechanisms are good examples of unmet clinical needs that should be solved with cancer genomics.
It would speed up and optimize the application of genomics to cancer clinics if cancer biologists, bioinformaticians, and cancer clinicians cooperate towards one goal of achieving it.
Title: Genomic Profiling of Liver Cancer

Development of liver cancers is driven largely by genomic alterations that deregulate signaling pathways, influencing growth and survival of cancer cells.
Because of the hundreds or thousands of genomic/epigenomic alterations that have accumulated in the cancer genome, it is very challenging to find and test candidate genes driving tumor development and progression.
Systematic studies of the liver cancer genome have become available in recent years.
These studies have uncovered new potential driver genes, including those not previously known to be involved in the development of liver cancer.
Novel approaches combining multiple datasets from patient tissues have created an unparalleled opportunity to uncover potential new therapeutic targets and prognostic/predictive biomarkers for personalized therapy that can improve clinical outcomes of the patients with liver cancer.
Primary liver cancer is one of the most common cancers in the world, accounting for an estimated 600,000 deaths annually [1].
In Korea, liver cancer is the second leading cause of cancer-related deaths (10,946 death in 2010) [2].
In the United States, primary liver cancer has gained major interest, because the incidence of liver cancer has increased over the past 25 years, and the incidence and mortality rate of liver cancer are expected to double over the next 10 to 20 years [3-5].
Therefore, prevention and treatment of liver cancer are of great concern.
While primary liver cancer is one of the few cancers with well-defined major risk factors, such as hepatitis virus infection, alcohol consumption, and obesity, the molecular pathogenesis of liver cancer is not well understood [6-8].
Patients with liver cancer have a highly variable clinical course [7, 9], indicating that liver cancer comprises several biologically distinct subgroups.
Therefore, it will be necessary to understand liver cancer at a genomic and molecular level for improved stratification of patients, identification of novel druggable targets, and development of personalized treatment, based on the biological characteristics of tumors in each patient.
Due to the complex nature of cancers that are initiated by activation of many different oncogenes or inactivation of different tumor suppressor genes and progress by additional genetic or epigenetic alterations, conventional gene-by-gene approaches are likely to deliver only a limited understanding of the biological and pathological characteristics of cancer cells.
However, the recent development of technologies for gene expression profiling, genome-wide copy number analysis, and whole-genome sequencing has enabled the comprehensive characterization of entire cancer genomes.
This information promises to improve our understanding of the genetic and epigenetic alterations driving the development of liver cancer, as well as ultimately provide guidance on personalized treatment of patients.
Although these large-scale genomic data promise improvement in the treatment of liver cancer, challenges remain for the management and integration of these data for a better understanding of how these alterations give rise to the development of liver cancer and, most importantly, the translation of these findings into personalized patient treatment.
Title: Comparative genomic hybridization (CGH)

Since the discovery of frequent genetic rearrangements and copy number aberrations in the cancer genome [10], cytogenetic methods have been used widely to find the genetic changes in cancer genomes.
The CGH method was the first tool to provide a genome-wide investigation of copy-number alterations in cancer [11].
Increased or decreased regions of copy number are believed to have either oncogenes or tumor suppressor genes.
Although the resolution of early CGH mapping technology is limited by 2 Mbp (high-copy-number amplifications) to 10 Mbp (low-copy-number amplification or deletion), this method has identified many loci for oncogenes and tumor suppressor genes in liver cancer.
Multiple studies have reported amplification of the 8q24 loci, in which MYC is located [12-15].
Other studies also identified prevalent amplification of the 1q, 6p, and 17q regions and frequent deletion of the 8p, 16q, 4q, and 17p regions [12, 15].
Title: Microarray-based technologies

Microarray technology has been used extensively to gather global-scale data from cell lines or tissues of interest.
This technology was first used to make a quantitative assessment of mRNAs or non-coding RNAs.
Gene expression profiling studies based on microarray technologies have identified conserved gene expression signatures that are significantly related with clinical outcomes or pathological phenotypes and have uncovered many subgroups of cancer that were not recognized with conventional approaches in a variety of cancers [16-18].
These approaches have been used successfully to identify therapeutic targets for various cancers and to predict overall survival or recurrence-free survival rates of cancer patients [19-21].
Use of microarray technology is not restricted to genome-wide gene expression profiling of cells or tissues but expands to uncovering single-nucleotide polymorphisms associated with cancer risk, methylation status of gene promoters, DNA copy number alterations, protein expression profiling, and re-sequencing of cancer genomes.
Because microarray-based CGH can provide copy-number analyses with much higher resolution than conventional CGH, it has quickly replaced conventional CGH [22, 23].
Microarray technologies were also adopted by the proteomics community.
Protein microarray was quickly established by rapid improvements to miniaturizing western blotting and tissue lysate dotting technologies onto solid surfaces.
There are two different experimental approaches for protein arrays: forward-phase protein array (FPPA) and reverse-phase protein array (RPPA).
In FPPA, antibodies or polypeptides are printed on the slide surface, and each slide?is incubated with labeled tissue or cell lysate.
This approach allows one to simultaneously measure multiple protein features, such as expression, interaction with peptide ligands, and protein phosphorylation, from the samples.
In RPPA, an individual tissue lysate in printed on the slide surface, and each RPPA slide consists of hundreds or thousands of tissue samples.
Each slide is then probed with one peptide ligand or antibody at a time, and a protein feature of interest is assessed and compared across many tissues or cell lines.
Because RPPA needs only several thousands of cells to collect high-quality data, it is better suited for cancer research.
In contrast, FPPA is not suitable for tissue-based studies, because it needs a large quantity of tissue or cell lysate for detection [24-27].
Title: Next-generation sequencing

Due to the emergence of second-generation sequencing technologies that allow massive parallel collection of sequence information, the cost per base for sequencing the entire genome has reduced dramatically [28].
These new technologies provided unique opportunities to investigate all sequences of entire cancer genomes to uncover the genetic changes that arise during cancer development.
Many different technologies have been developed by different companies: Roche Applied Science (454 Genome Sequencer FLX System; Indianapolis, IN, USA), Life Technologies (Sequencing by Oligonucleotide Ligation and Detection or SOLiD; Carlsbad, CA, USA), Illumina (Genome Analyzer II; San Diego, CA, USA), Helicos BioSciences (HeliScope Single Molecule Sequencer; Cambridge, MA, USA), and Ion Torrent Systems (now owned by Life Technologies, Ion Proton System).
These new technologies have been used to identify potential driver mutations in the development of liver cancer through whole-genome and -exome sequencing.
The first sequencing of the primary liver cancer genome revealed a total of 11,731 somatic mutations [29].
Following studies showed that the CTNNB1, TP53, and EGFR genes were frequently mutated in liver cancer [30-33].
Additionally, the ARID family, including ARID1A, ARID1B, and ARID2, was mutated in about half of all tumors, suggesting a potential role of them in the development of liver cancer.
In previous studies [18, 34-37], gene expression data from human liver cancer have identified several subgroups that are significantly associated with clinical outcomes, such as overall survival and recurrence-free survival.
These findings strongly indicated that gene expression patterns faithfully reflect biological and pathological heterogeneity among patients with liver cancer and would be highly valuable in predicting the clinical outcome of patients.
The current challenge for clinical use is to identify those who will not benefit from conventional treatments and to offer alternative therapies.
If master drivers, such as genes or pathways determining the biological characteristics of the tumors, can be identified, we can explore their therapeutic potentials.
However, gene expression profiling approaches are also limited, because they only provide information related to transcriptional regulation.
Title: Integromics: Integration of multiple-omics data

Previous studies undoubtedly showed that gene expression patterns (or signatures) are useful markers that can stratify patients and offer useful prognostic information [18, 37-42].
The current research focus has moved to uncovering genetic/epigenetic factors that are key regulators of specific signaling pathways altered in malignant cancer cells, potentially guiding the discovery of new therapeutic (and druggable) targets [8, 43-45].
But, the selection of such candidates for future validations from long gene lists produced by genome-wide screening is a significant challenge because of potential confounding factors embedded in many genome-wide data from human cancers.
Furthermore, because the gene expression data from tumor tissues only provide &#34;snapshot&#34; information of genetic/epigenetic alterations and lack in-depth information on interactive and time-dependent alterations during tumor progression, it is not easy to distinguish the driver genes from passenger genes whose alterations simply echo changes in cell proliferation or organ physiology.
As discussed earlier, CGH analyses have identified many recurrent candidate loci of DNA copy number changes in liver cancer.
Some of these genomic regions have well-recognized tumor suppressor genes and oncogenes.
However, because of the large number of candidate genes in the identified genomic loci, functional validation of all candidates would be impractical.
Thus, there is an urgent demand for development of a new approach that would quickly prioritize candidates according to the likelihood of their direct involvement in tumor development.
In a previous study, systematic integration of gene copy number data and gene expression from the same patients has been proposed to identify potential driver genes [46].
This study demonstrated that DNA copy number alteration data provide additional prognostic significance.
Integrative analysis of copy number alteration data and gene expression further identified 50 driver candidates whose activation was triggered by copy number amplifications of genes and also significantly associated with aggressive tumor phenotypes in liver cancer.
Alterations in DNA copy number and expression patterns of thousands of genes are important characteristics of many cancer cells.
Because the use of high-throughput microarray-based technologies, mass spectrometry, and second-generation sequencers for the analysis of cancer genomes inevitably produces many false-positive results, it is not easy to select practical numbers of candidate genes for future evaluation of them as therapeutic targets and/or prognostic/predictive biomarkers.
Therefore, it is important to combine two or more genomic scale datasets (e.g., array-based CGH data, promoter methylation, and expression of coding or non-coding genes), collected independently from the same tissues (Fig.1).
RPPA is a newly established proteomic technology [24-27] that provides quantitative assessment of the expression and modification of many proteins.
For example, the phosphorylation level of particular proteins can be estimated easily by using antibodies specific to certain phosphorylation sites of proteins.
Use of the phosphorylation antibodies allows us to assess the signaling pathways by looking at many kinase substrates simultaneously through multiplexed phosphorylation-specific antibody analysis.
However, assessment of signaling pathways by RPPA is restricted by the availability of antibodies with very high specificity.
This limitation can be overcome by integration of multiple datasets for systematic analysis.
Integration of proteomic data with genomic data would significantly help us have better insights into tumor development and progression.
Furthermore, contributor genes (or drivers) will be quickly identified by integration of proteomic data with one or more genomic datasets.
Title: The Cancer Genome Atlas (TCGA) Project: new opportunity for cancer genomics

TCGA is a landmark research program supported by the National Human Genome Research Institute and National Cancer Institute at the National Institutes of Health in the United States that was started as a pilot project in 2006 and expanded later in 2009.
The goal of TCGA is to collect comprehensive genomic and proteomic information on all major human cancers, including liver cancer.
Initial efforts focused on glioblastoma, lung squamous cell cancer, and ovarian cancer as pilot projects [47-50].
By using various different platforms, TCGA currently gathers many different genome-wide data, including mRNA expression, microRNA expression, somatic mutations, copy number alterations, and promoter methylation.
In addition, it also generates proteomic data by using RPPA technology.
The project plans to collect genomic and proteomic data from more than 500 tissues per cancer type and release the data to the public without any restriction in use of the data (Table 1).
This ambitious project has identified novel driver genes and biomarkers on the basis of genomic, transcriptomic, proteomic, and epigenomic alterations.
Some findings are clinically relevant and unexpected.
For example, we have now learned that non-hypermutated adenocarcinomas of the colon and rectum are not distinguishable at the genomic level [51].
In lung squamous cell cancer, while KRAS and EGFR mutations, the most commonly activated oncogenes in lung adenocarcinoma, are extremely rare, alterations in the FGFR kinase family are common [49].
Thus, massive data from a large number of tissues have created an unprecedented opportunity for taking an integrated approach toward a systems-level understanding of disruptions in cellular and molecular pathways in cancer.
Genomics has become a dominant tool for understanding cancer biology, revealing unexpected surprises and providing cancer classification according to biological differences.
The amount of liver cancer-related genomic data that have generated during the past few years is really incredible.
Analysis approaches of these data have been advanced to efficiently process this information and allow us to perform integrated genome-wide analysis on a scale previously unthinkable.
These approaches are also beginning to investigate the dense network of molecular pathways driving the development of liver cancer.
The anticipated benefits of these data will be more a faithful assessment of prognosis than the current stage system, improved prediction of treatment response, and discovery of new therapeutic targets.
Future challenges lie in translating this discovery into personalized treatment for the patient with liver cancer.
Integromics approaches.
Genomics data provide long lists of candidate driver genes or proteins.
Integration of these independently generated data from the same specimens will greatly improve the chance of identifying real driver genes or therapeutic targets.
Title: Cancer types investigated by the TCGA project
Title: Perspectives on Clinical Informatics: Integrating Large-Scale Clinical, Genomic, and Health Information for Clinical Care

The advances in electronic medical records (EMRs) and bioinformatics (BI) represent two significant trends in healthcare.
The widespread adoption of EMR systems and the completion of the Human Genome Project developed the technologies for data acquisition, analysis, and visualization in two different domains.
The massive amount of data from both clinical and biology domains is expected to provide personalized, preventive, and predictive healthcare services in the near future.
The integrated use of EMR and BI data needs to consider four key informatics areas: data modeling, analytics, standardization, and privacy.
Bioclinical data warehouses integrating heterogeneous patient-related clinical or omics data should be considered.
The representative standardization effort by the Clinical Bioinformatics Ontology (CBO) aims to provide uniquely identified concepts to include molecular pathology terminologies.
Since individual genome data are easily used to predict current and future health status, different safeguards to ensure confidentiality should be considered.
In this paper, we focused on the informatics aspects of integrating the EMR community and BI community by identifying opportunities, challenges, and approaches to provide the best possible care service for our patients and the population.
The development of electronic medical record (EMR) systems began as a means to document clinical activities for in-patients and out-patients [1].
They have evolved as the primary front-line patient care clinical tool for medical professionals.
The completion of the Human Genome Project opened the era of research in genomics and proteomics.
Genome research provides keys to understanding the mechanisms of disease.
In clinical informatics, the widespread adoption of the EMR system has generated large amounts of heterogeneous clinical data-some structured and others unstructured.
In addition, the explosive health-related contents from online communities, mobile applications, and electronic personal health records increased the availability of non-traditional data on individual activities and life style [2].
In genetics, since the completion of the Human Genome Project in 2003, the acquisition, analysis, and presentation of whole-genomic data has become faster, cheaper, and more reliable day by day [3].
Such dramatic technological advances affect the development of new prevention, diagnosis, and treatment patterns for routine clinical care.
The massive amount of heterogeneous data from two different domains is expected to provide personalized, preventive, and predictive healthcare services in the near future [4].
Integrated use of EMR and bioinformatics is beginning to influence the changes in the research paradigm-that is, rapid introduction of new concepts into the point of care.
Dr. Want used clinical bioinformatics (CBI) with the definition of &#34;the clinical application of bioinformatics-associated sciences and technologies to understand molecular mechanisms and potential therapies for human disease&#34; [5].
CBI aims to deal with the challenge of integrating genomic and clinical data to accelerate the translation of knowledge into effective treatment plan development and personalized prescription.
It is to assist clinicians in various ways, including new biomarker discovery, identification of genotype and phenotype correlations, and pharmacogenomics at the point of care.
Biomedical informatics is a more popular term.
Biomedical informatics is defined as an emerging, multi-disciplinary field, and it is the integration of the computational methods and diverse technologies used in life science research, such as genomics, proteomics, systems biology, computer sciences, and healthcare applications, such as electronic health records (EHRs) [6].
The adoption of EMRs enables one to conduct comprehensive phenotypic-genotypic association studies using the genotypes obtained from whole genome sequencing of a given cohort in combination with the phenome data of the same population, as available in the EMR database, as shown in Fig.1.
The major challenge in enabling such convergent research is to provide easy storage, user-friendly visualization, speedy analysis, knowledge generation, and presentation of clinically relevant information at the point of care.
Relevant information should be extracted and linked with medical records in a clinically applicable manner.
The convergence of discovery research for clinical implementation can only be accomplished through stringent data management, analysis, interpretation, and quantification in a multidisciplinary research environment.
In addition, since genetic information can be easily identifiable, ethical issues, such as informed consent and stewardship over this database, also should be considered as the data grow.
The communities of EMRs and bioinformatics (BI) have different histories.
While the EMR community focused on clinical activities and clinical workflows, the BI community originated from the biological research community, which included physicists, computer scientists, statisticians, and clinical researchers.
What are the optimal ways to integrate the tremendous advances in BI into routine clinical work?
In this paper, we will primarily focus on the informatics aspects of this large question by identifying opportunities, challenges, and approaches for the ultimate goal of providing the best care possible for our patients and the population.
A number of research projects using large-scale health record datasets have been conducted in various communities around the globe.
For example, the NSF BIGDATA program solicitation (http://www.nsf.gov), which is partially funded by the National Institutes of Health (NIH), includes large-scale data collection and analysis.
This program aims to advance the core scientific and technological means of managing, analyzing, visualizing, and extracting useful information from large, diverse, distributed, and heterogeneous datasets.
INBIOMEDvision (http://www.inbiomedvision.eu) is a two-year initiative funded by the European Commission 7th Framework Program of Information and Communication Technologies (ICT), with the aim of bridging the communities of bioinformatics and medical informatics.
It is a coordination support action (FP7) that has the aim of promoting biomedical informatics by means of permanent monitoring of the scientific state-of-the-art and existing activities in the theme, execution of prospective analyses on the emerging challenges and opportunities, and dissemination of knowledge in the field [7].
The Electronic Medical Records and Genomics (eMERGE) Network is a national consortium organized by National Human Genome Research Institute (NHGRI) to develop, disseminate, and apply approaches to research.
It combines DNA biorepositories with EMR systems for large-scale, high-throughput genetic research with the ultimate goal of returning genomic testing results to patients in a clinical care setting.
The network is currently exploring more than a dozen phenotypes (with 13 additional electronic algorithms having already been published).
Various models of returning clinical results have been implemented or planned for pilot at sites across the network.
Themes of bioinformatics, genomic medicine, privacy, and community engagement are of particular relevance to eMERGE [8].
In addition to nationwide collaboration projects, there is a research project using large-scale EHRs.
Hanauer et al.
[9] used large-scale, longitudinal EHR data to conduct research on associations of medical diagnoses and to explore patterns of specific disease progression.
Lin et al.
[10] proposed a symptom-disease treatment (SDT) association rule, mining a comprehensive EHR of approximately 2.1 million records from a major hospital.
Based on selected International Classification of Disease (ICD-9) codes, they were able to identify clinically relevant and accurate SDT associations from patient records in seven distinct diseases, ranging from cancers to chronic and infectious diseases.
Research combining biology and clinical information must address &#34;the storage, retrieval, analysis, and dissemination of molecular information in a clinical setting,&#34; as suggested by the America Medical Informatics Association (AMIA)-initiated genomics working group [11].
The AMIA proposed specific areas as follows: 1) development of a database structure to unify clinical and genomic data, 2) connecting biology information with patient health records, 3) development of a genome-enabled EMR system, 4) linking clinical trial and drug discovery information, 5) supporting the development benchmark clinical/molecular datasets, 6) developing clinical decision-support tools utilizing molecular information, and 7) visualizing and modeling the molecular basis of disease.
The major challenge is how to integrate the heterogeneous data into one database system.
Should there be a single database or should one consider a federated model?
Furthermore, one should consider various cases by various users, which would determine the overall system architecture.
It is not possible to move large amounts of genomic data.
Then, how and where should the high-intensity computation be managed?
The expected raw sequencing data for one person is approximately 4 terabytes.
The integrated database can have a potential impact on the prevention, diagnosis, and treatment of disease.
To make this desire come true, it is important to connect genomics data with clinical information.
Genetic test results are already used to assess the risk of breast cancer patients, determine the potential adverse drug reactions on individual patient metabolism, and identify treatment plans for cancers.
Genetic test results have suggested a diagnosis for patients with neuropathy, inflammatory bowel disease, and Proteus syndrome and have guided therapeutic care for patients with arterial calcifications, movement disorders, and Miller syndrome [12-17].
The number of applications of genomics in diagnosing diseases and guiding treatment procedures in the clinic will continue to increase.
The integrated use of EMR and BI data needs to consider four key informatics areas: data modeling, analytics, standardization, and privacy.
Title: Data modeling and data warehouse

Data modeling and warehouse are two key concepts within proper systems architecture for medicine and bioinformatics.
The first generation of clinical data warehouses (CDWs) is mostly stored in commercial relational database management systems and collects structured contents, and the healthcare community has developed a large clinical database, which is called the CDW [18].
The extraction, transformation, and load is essential for converting and integrating distributed healthcare data.
Ad hoc reporting tools using online analytical processing technology are used to gain intuitive and simple analysis results in clinical informatics.
Bioclinical data warehouses that integrate heterogeneous patient-related clinical or omics data should be considered.
Applications analyzing bioclinical data warehouses will include genetic epidemiology and evaluation of decision-support systems before production systems.
In less than a decade, the Human Genome Project has been established to generate a large amount of biological data to practice diagnostics, prognostics, and therapeutics [19].
The central products using standardized data model are GenBank [20], SWISS-PROT [21], Exon-Intron [22], and IMGT [23].
However, standardized data models for integrated bioclinical data warehouses have not been developed yet.
This area will be important to allow researchers to rapidly spread information throughout the world and inspire thousands of research projects.
The experience in designing storages for digital radiological imaging, also known as picture archiving and communication system (PACS), may provide some guidance in bioinformatics data warehouses.
In PACS, one has multiple image storages that are not necessarily part of EMR; however, the diagnostic reports are part of the EMR.
The PACS images are then made available to physicians as needed.
Title: Analytics

The analytics technology that is commonly used for these systems is mainly traditional data mining techniques developed in the 1980s.
The popular term of analysis technology in the business and computer science communities in the 1990s was business intelligence.
Recently, technologies that require advanced and unique data storage, management, analysis, and visualization became important in applications that are very large and complex databases.
Title: Standardization

The lack of standardized vocabularies for clinical informatics has hampered the development of automated clinical decision support systems.
Finding the laboratory term representing the same meaning of serum sodium for different database systems is troublesome in multi-center data integration.
The National Library of Medicine has developed the Unified Medical Language System (UMLS) to enable these different vocabularies to be interoperable by developing a vocabulary at least at a basic level [24].
The same problems are known in bioinformatics.
DNA sequences have different names and are joined in some databases only with varying levels of confidence.
Codification of molecular diagnostic or cytogenetic results using existing medical vocabularies will have difficulties due to a lack of sufficient terms for molecular findings.
For example, the Systematic Nomenclature of Medicine (SNOMED) has minimal codes related to the description of molecular diagnostic findings.
The Logical Observation Identifiers Names and Codes (LOINC) vocabulary has recently added a significant number of molecular pathology terms; however, it lacks the rich context-defining relationships provided by ontology [24].
The Clinical Bioinformatics Ontology (CBO) addresses this gap by providing uniquely identified concepts related to clinically significant molecular findings.
The CBO consists of nearly 7,000 concepts, each of which is associated with a global unique identifier, and is associated with more than 15,500 relationships [25-27].
As the efforts to integrate two different domains are increased, vocabulary issues will be essential.
Title: Privacy

Individual genomic data are easily identifiable and can be used to predict current and future health status.
Thus, extracting knowledge from large health data employs a significant risk of privacy information breach; thus, researchers need to consider Health Insurance Portability and Accountability Act (HIPAA) and Institutional Review Board (IRB) requirements for building a privacy-preserving and trustworthy database infrastructure and conducting research [28].
When personal genetics data can be incorporated into EMR systems, different safeguards to ensure confidentiality will be required.
A de-identified bio-data warehouse combining traditional clinical and genomic information will be essential to conduct translational research.
One major difference between the EMR community and BI community is the degree of multidisciplinary open collaboration.
In EMRs, multidisciplinary collaborative efforts have been limited.
Most of the EMR systems in use today are based on proprietary software that hampers data exchange with other systems.
In the BI community, global collaboration, aided by wide use of open source software and development methodologies, has been a key success factor.
The BI community has been global in scope from the beginning, and information sharing and free exchange of software tools have ushered in the era of open source software in health informatics.
There is a great need for the EMR community to adopt more open source software methodologies that would allow rapid global collaboration (http://www.osehra.org).
Genomic technologies hold the potential to improve the diagnosis and treatment of inherited and complex diseases-including cancer- and facilitate the move towards personalized predictive medicine.
The higher throughput and rapidly falling costs of next-generation sequencing have resulted in voluminous genomic data and downstream computational challenges.
Thus, the shift from this powerful discovery research to clinical implementation can only be accomplished with careful integration with EMRs, a frontline patient care tool.
The most prominent reason to integrate clinical information and biology information under the same system is to provide opportunities for bi-directional exchange of data, technology, and knowledge between two disciplines with different histories and cultures.
Additionally, open global cooperation will provide opportunities to make rapid progress in understanding, treating, and preventing human diseases.
Electronic medical record (EMR) databases and PheWAS.
(A) Dataflow shows how EMRs of enrolled patients are curated and incorporated into a database.
(B) In silico cohort is generated from an EMR database with respect to various categories (e.g., billing diagnoses or disease-related genotypes).
The cohort is tested for the association with various phenotypes, as available in EMR databases (PheWAS).
R/O, rule out; BMI, body mass index.
Title: A Primer for Disease Gene Prioritization Using Next-Generation Sequencing Data

High-throughput next-generation sequencing (NGS) technology produces a tremendous amount of raw sequence data.
The challenges for researchers are to process the raw data, to map the sequences to genome, to discover variants that are different from the reference genome, and to prioritize/rank the variants for the question of interest.
The recent development of many computational algorithms and programs has vastly improved the ability to translate sequence data into valuable information for disease gene identification.
However, the NGS data analysis is complex and could be overwhelming for researchers who are not familiar with the process.
Here, we outline the analysis pipeline and describe some of the most commonly used principles and tools for analyzing NGS data for disease gene identification.
The breakthrough in next-generation sequencing (NGS) techniques has enabled rapid sequencing of whole genomes at low cost and has brought tremendous opportunities as well as challenges to biomedical research [1-5].
The cost of sequencing is projected to continue to drop, with whole-genome sequencing expected to be as low as $1000 [4] and whole-exome sequencing (i.e., sequencing approximately 1% of the coding regions of the genome [6]) to be $500 [1].
Such low costs enable a small research lab to generate large amounts of sequence data in a short period of time on a relatively small budget.
The low cost also allows sequencing-based clinical genetic tests to be routinely performed to provide guidance for health professionals for disease etiology and management.
The challenge now is how to reliably synthesize and interpret this large amount of raw data from sequencing platforms.
One goal of genomic research is to determine which genes are causal for the disease of interest.
With the NGS data, this is not a trivial task.
Large-scale whole-genome and whole-exome sequencing analyses identify large amounts of genomic variants, most of which are not related to disease risk.
Thus, disease gene prioritization principles and tools are needed [2] to identify and prioritize variants of interest from the large pool of candidates.
Currently, identifying candidate causal mutations/genes is often a complex practice that involves several dozens of steps and/or tools to accomplish.
These tools work jointly in a series of steps and thus functionally form a &#34;workflow&#34; or a &#34;pipeline.&#34;
Ideally, the process of disease gene prioritization generates an ordered list of genes that puts high-risk candidates on the top of the stack and filters out &#34;benign&#34; or neutral variants.
In reality, this process usually involves multiple filtering steps, some of which have underlying assumptions that can affect how variants are filtered.
The focus of this review is to provide a broad overview of the workflow and present some of the bioinformatics software tools that are currently available.
More in-depth comparisons of the bioinformatics algorithms and tools are described elsewhere [3, 5].
Typically, the output from NGS platforms is in the standard FASTQ format.
The FASTQ file is a text file containing a list of short DNA fragments, generally less than 150 base pairs (bp) long, and a list of quality scores associated with the DNA fragments.
For whole-genome or whole-exome sequencing, a FASTQ file contains millions of records.
For example, the human exome is ~50 million base pairs (Mbp), and a typical whole-exome sequencing dataset has at least 30X depth of coverage (i.e., each bp is sequenced 30 times on average).
Such a FASTQ file will contain ~15 million raw 100-bp sequences.
To identify the causal genes from this raw data, a variant gene prioritization pipeline usually involves three stages or phases of data processing: 1) raw sequence processing and mapping, 2) variant discovery and genotyping, and 3) disease gene filtering and/or prioritization.
Title: Phase 1: Raw sequence processing and mapping

Title: Overview

The goal of Phase 1 is to preprocess the FASTQ file and evaluate the quality of the raw reads and then to map or align the sequences to the reference genome-that is, to find the best location on the reference genome where each sequence might have originated.
Successfully mapped sequences are annotated with information that includes the location on a specific chromosome and how well the sequence matches the reference genome.
After initial mapping, there are several subsequent recalibration steps to improve the mapping results.
Title: Preprocessing

The raw FASTQ file contains all of the raw sequences, some of which are of low quality.
The overall quality should be evaluated, and the lower-quality reads should be removed before mapping to the reference genome to improve mapping efficiency.
FASTQC (http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) is a popular tool that produces quality analysis reports on FASTQ files.
FASTQC provides graphical reports on several useful statistics, such as &#34;Per base sequence quality,&#34; &#34;Sequence duplication levels,&#34; and &#34;Overrepresented sequences,&#34; etc.
Specifically, &#34;Per base sequence quality&#34; determines if trimming low-quality sequences is needed before the mapping/alignment step; &#34;Sequence duplication levels&#34; evaluates the library enrichment and complexity; and &#34;Overrepresented sequences&#34; evaluates potential adaptor contamination.
Low-quality bases and adaptor contaminations can cause an otherwise mappable sequence not to map and therefore should be removed (e.g., trim the 3&#39; end of the read.
which tends to have low quality).
FASTX-Toolkit (http://hannonlab.cshl.edu/fastx_toolkit/) provides a set of command line tools for manipulating FASTQ files, such as trimming input sequences down to a fixed length or trimming low-quality bases.
Cutadapt (https://github.com/marcelm/cutadapt) can be used for trimming short reads to remove potential adapter contamination.
Title: Mapping

Once high-quality sequence data are obtained, a number of bioinformatics programs, including Mosaik [7], MAQ (http://maq.sourceforge.net/), mrFAST [8], BWA [9], Bowtie2 [10], SOAP2 [11], and Subread [12], can be used to perform short-read sequence alignment to the reference genome.
Several recent studies provided a detailed description and comparison of the alignment programs [5, 13], although it could be difficult to choose among aligners, because benchmark results vary across studies [10, 12, 14].
BWA and Bowtie2 are two popular programs for alignment.
Both programs employ the Burrows-Wheeler transform (BWT) algorithm and have been shown to yield very good overall performance [13].
Bowtie2 is often used for mapping ChIP-seq and RNA-seq data, and BWA is often used for mapping whole-genome/exome data.
We highlight the BWA program here, because it is accurate, fast, thoroughly tested, and well supported.
Additionally, BWA has been utilized in multiple large-scale projects and well-defined workflows, including the Genome Analysis Toolkit (GATK) by Broad Institute [15], the 1000 Genome Project [16], the NHLBI GO Exome Sequencing Project (ESP) (http://evs.gs.washington.edu/EVS/), and the HugeSeq workflow [17].
Various commercial companies, such as Seven Bridges Genomics (https://www.sbgenomics.com/) and Geospiza GeneSifter (http://www.geospiza.com/), also utilize BWA in their standard workflows.
After mapping the reads, the final output file from most software is typically in the sequence alignment/map (SAM) format [18].
In some cases, the SAM file is converted to its binary format (BAM) using SAMtools [18] to reduce the file size and optimize computation performance.
Title: Recalibration

After the initial mapping procedure, subsequent recalibration steps are typically employed to improve the mapping results, such as performing local realignment or de novo assembly for regions that contain insertions/deletions (indels), removing PCR duplicates, and recalibrating the base quality scores.
Indel realignment is a highly recommended post-BAM processing procedure.
Due to the trade-off between speed and accuracy, indels are not well aligned by most general purpose aligners.
This could lead to false variant calls in downstream analyses.
Regions with high probability of potential indels can be realigned locally using IndelRealigner, part of the GATK toolkit.
Another commonly used recalibration procedure is removing PCR duplicates.
If a DNA fragment is amplified many times by PCR during the sequencing library construction, these artificially duplicated sequences can be considered as support of a variant by downstream variant discovery programs.
Some BAM processing programs, such as Picard (http://picard.sourceforge.net/) and Samtools [18], can identify these artificially duplicated sequences and remove them.
Base recalibration is also a recommended step, because the sequencer may have assigned a biased quality score upon reading a base (e.g., the score of a second &#34;A&#34; base after a first &#34;A&#34; base may always receive a biased quality score from a sequence machine [19]).
Tools, such as Base-Recalibrator in the GATK toolkit, can calibrate the quality score to more accurately reflect the probability of a base mismatching the reference genome.
One additional optional step, recommended by GATK, is data compression and reads reduction, especially for high-coverage data.
For example, if a large chunk of sequences matches the reference exactly, it is not necessary to keep all the data, as they do not carry useful information for downstream analyses (assuming we are only interested in the sites that are different from the reference genome).
In such a scenario, keeping one copy of each of the consensus sequences may be sufficient, and the redundancies can be removed to reduce file size and enable faster downstream computing.
However, keeping a copy of the original file is highly recommended after data compression.
Title: Phase 2: Variant discovery and genotyping

Title: Overview

In many scenarios, only the sites that differ from the reference genome are of interest, because sites that are identical to the reference genome are not expected to be related to pathological conditions.
Once raw sequences are properly mapped to the reference genome, the next step is to find all positions in an individual&#39;s genome that differed from the reference.
This phase is referred to as variant discovery, or variant calling.
Similar to the mapping phase, variant calling also contains an initial discovery step, followed by several filtering processes to remove sequencing errors and other types of false discoveries, and finally, the individual genotypes are inferred (i.e., if a locus is heterozygous, homozygous, or hemizygous for the variant).
The output of variant calling contains all the variants and related information.
Sites that are identical to the reference genome (i.e., invariant sites) are usually not included in the output variant file.
Title: Variant discovery and genotyping

A number of variant calling software packages can be used to identify variants and call individual genotypes.
Some of the commonly used software programs are SAMtools [18], freebayes (http://github.com/ekg/freebayes), SNPtools [20], GATK UnifiedGenotyper, and GATK HaplotypeCaller.
Some of the tools, including SAMtools, SNPtools, and the GATK UnifiedGenotyper, use a mapping-based approach.
Other tools, such as freebayes and the GATK Haplotype-Caller, use a local assembly approach.
A more detailed survey and comparison of the tools have been previously described [5, 21].
These procedures typically take the BAM files from the &#34;raw sequence processing and mapping&#34; phase, together with the reference genome sequence in a FASTA file.
The output file from this step is usually in the standard variant call format (VCF).
Some of the tools allow the user to specify known variants to optimize the variant discovery.
For example, GATK UnifiedGenotyper uses known variants (e.g., from the dbSNP database) as high-confidence datasets to assist model training.
After variant and genotype calling, the probability scores of variants are recalibrated, and artifacts are identified.
The artifacts can be removed subsequently either by filtering the data using a series of criteria or by building and applying an error model.
One example of the variant discovery tool is the GATK UnifiedGenotyper, which calls variants by examining the reference genome base by base, without correcting misalignment issues.
This procedure could lead to false positive calls, particularly in regions containing indels.
To increase the accuracy and sensitivity, calling multiple samples simultaneously is recommended.
By taking into account the information of a variant in a population, the accuracy and sensitivity of variant discovery are greatly improved, especially for common variants [16, 22].
The GATK Haplotype-Caller, as compared to the UnifiedGenotyper, calls all variants (single nucleotide polymorphisms [SNPs], indels, structural variations, etc.)
simultaneously by performing local de novo assembly of haplotypes and emits more accurate call sets, with the drawback of being slower.
In general, structural variations (SVs) and copy number variations (CNVs) are more difficult to detect than SNPs and indels because of their heterogeneous nature.
For SVs and CNVs, it is generally recommended to apply a combination of several tools and take the overlapping variant sites for high-confidence calls [17].
Title: Genotype phasing and refinement

After the initial variant discovery and recalibration, variant genotyping and/or phasing are performed to provide critical information for downstream medical and population genetic studies that require accurate haplotype structures.
For example, Mendelian diseases may be caused by a compound heterozygote event where heterogeneous mutations are found on different chromosomes.
In such a scenario, knowledge about the paternal or maternal origin of a variant can provide valuable information.
The GATK workflow recommends three steps of genotype phasing: transmission based on pedigree, haplotype based on reads, and statistical imputation.
GATK PhaseByTransmission incorporates pedigree data to assist genotype calling, while GATK Read-BackedPhasing performs physical phasing of variant calling based on sequence reads.
Statistical imputation of genotypes can be processed by several programs, such as Beagle [23,24], IMPUTE2 [25], and MACH [26].
Title: Phase 3: Disease gene filtering and/or prioritization

Title: Overview

A typical variant call pipeline will produce approximately 3 million variants from one whole-genome sequencing data set and more than 20 thousand variants from one whole-exome sequencing data set [27].
Because disease causing mutations might just be several &#34;needles&#34; in this tremendous &#34;haystack&#34; of variants, rigorous algorithms and high-quality databases are essential to accurately locate the candidate genes.
This process is done either by filtering out variants that are not likely to carry a disease risk or by ranking all variants based on biological/statistical models, such that high-risk candidates rank at the top of the list [1, 2, 4].
Unlike Phase 1 and Phase 2, which have largely converged to well-formed computation workflows and platform-free file formats (e.g., FASTQ, BAM, and VCF formats), Phase 3 has more variations in the tools and workflows to suit the different needs of various research or clinical projects.
Title: Annotation

Although many different strategies can be used to search for disease-causing mutations, some general procedures are shared.
The first step, variant annotation, typically investigates the potential pathogenic impact of a variant, before any subsequent filtering or prioritizing.
The goal is to identify and report the effect of each variant with respect to predefined &#34;features,&#34; such as genes.
For example, if a variant (SNP or indel) is present within the coding region, the potential effects include change to amino acid (non-synonymous), no change to amino acid (synonymous), introduction of stop codon (stop gain), removal of stop codon (stop loss), addition/removal of amino-acid(s) (inframe indel), and change to the open reading frame (frame-shifting indel), etc.
Other than variant call files, the annotation process requires a feature file that defines the location of genomic features (e.g., transcript, exon, intron, etc.
), which is usually in browser extensible data (BED) format [28], or generic feature format version 3 (GFF3) format maintained by the Sequence Ontology project [29].
In many situations, the reference genome in FASTA format is also provided as part of the input, and the annotated variants are likely to be platform-specific [30, 31].
Annotation is part of the workflow for almost all variant analysis tools, such as the open source software package VAAST [31], ANNOVAR [27], and many commercial service providers (e.g., Ingenuity, Golden Helix, Geospiza GeneSifter, Omicia, Seven Bridges Genomics, Biobase, etc.).
Once the variants are annotated, a number of methods can be used to predict the severity of a variant, based either on the protein sequence conservation information (e.g., BLOSOM62, SIFT) or the DNA sequence conservation information (e.g., PhastCons).
The SIFT [32] score system assumes that protein &#34;motifs&#34; or &#34;sites&#34; are functionally important if they are highly conserved across species.
For example, coding regions, active sites of enzymes, many splice sites, and promoters are usually conserved across species [32].
Technically, SIFT uses protein homology to calculate position-specific scores, which are then used to evaluate if an amino acid substitution at a specific location is damaging or tolerated.
The accuracy of the SIFT prediction depends largely on the availability and the diversity of the homologous sequences across species.
Therefore, the application and accuracy of SIFT are limited if there is limited homolog information or if the diversity is low.
The BLOSUM62 matrix [33] is a more general-purpose score system that is based on the alignment of protein homologs with a maximum of 62% identity.
The BLOSUM62 matrix is then calculated based on the count of the observed frequency of specific amino acid substitutions.
The PhastCons score [34] is calculated based on DNA sequence conservation in a multiple alignment.
It uses a statistical model of sequence evolution and considers the phylogeny relationship of multiple species.
Instead of measuring the statistical similarity and diversity in percent identity, PhastCons uses a phylogenetic Hidden Markov Model and provides base-by-base conservation scores.
Because the PhastCons score is calculated for each base, it can be applied to regions beyond the coding sequence to provide valuable information for prioritizing non-coding variants.
In addition to the conservation information, many software packages also incorporate protein structure information, such as the positions of active sites and secondary structures, into the prediction algorithm (e.g., Polyphen2, SNAP).
Polyphen2 [35] combines eight sequencing features (e.g., congruency of the variant allele to the multiple alignment) and three structural features (e.g., changes in hydrophobic propensity) to score an amino acid substitution.
The homology search and model training of Polyphen2 are based on the UniProt database.
It worth noting that Polyphen2 provides two different prediction models that use different datasets for training, with HumVar tuned to score Mendelian diseases and HumDiv tuned to evaluate rare alleles at loci that are potentially involved in complex traits.
SNAP [36] is another tool that uses various protein information, such as the secondary structure and conservation, to predict the functional effect of a variant.
Title: Filtering

After variant annotation, two types of disease-gene identification strategies are commonly employed.
A filter-based approach uses a series of criteria to sort out variants or genes that are unlikely to be causative, given a disease model.
For example, in a typical workflow of ANNOVAR, variants that overlap with the 1000 Genome Project are removed from the list.
Since the 1000 Genome Project aimed to identify common SNPs with a minor allele frequency (MAF) greater than 1%, these variants are thought to be &#34;benign&#34; in a rare Mendelian disease, under the assumption that the causing mutation of a rare disease should have a very low MAF.
The HapMap [37, 38] database, which intended to identify haplotype-defining variants in different populations, can also be used for variant selection.
In some cases, high frequency of a haplotype can be related to a specific disease, and can help researchers focus on a subset of variants within a specific region.
Another commonly applied filter is to exclude non-coding variants under the assumption that a disease-causing mutation should occur in the coding regions and affect protein sequences.
Several other filtering criteria are also commonly used, such as filtering variants with little predicted functional impact (e.g., synonymous variants) or having low-quality scores (e.g., low read-depth or low genotype quality score).
A disease model can also help filter variants.
For example, a disease under a recessive mode of inheritance requires more than one deleterious variant in a gene.
The filtering strategy is intuitive and easy to perform and has been successful in early whole-genome/exome studies on rare, single-gene Mendelian diseases.
Numerous tools have been developed to apply various filtering strategies (reviewed in [1, 2, 4, 5]).
However, caution should be taken when studying common, complex diseases, because applying hard filters could remove real casual variants.
For instance, common diseases could be caused by common variants that have incomplete penetrance (i.e., not all individuals carrying the variant will have the disease).
Title: Ranking

To overcome the difficulties associated with hard filtering, a prioritizing approach ranks each gene or feature based on the cumulative disease risk of potential deleterious variants.
As an example, the VAAST tool kit performs the composite likelihood ratio test to determine the risk of a gene or a feature [30, 31].
Information of each variant within the gene, such as the predicted biological impact, allele frequency, and the conservation score of the variant allele, is considered simultaneously under a likelihood framework.
One advantage is that it reduces the risk of filtering out potential risky coding variants, since there are no hard filtering steps.
Another advantage is that it can score all variants, including non-coding variants that would have otherwise been filtered out.
The statistical framework used by VAAST considers two sets of information: 1) the likelihood of observing the MAF of a variant under a disease model versus a non-disease model; and 2) the likelihood that a variant is deleterious versus non-deleterious.
A combination of the observed frequency and predicted functional impact of variants is used to assess the disease risk of a gene.
A practical consideration is that most variants in the sequencing study are rare and may be difficult to achieve sufficient power for a statistical model [39].
To overcome this, a common solution is to assess the cumulative effects of multiple variants in a defined genomic region, such as a gene or an exon [39-41].
This approach evaluates the overall genetic burden of multiple rare variants and is referred to as the burden test.
Two general methods are commonly used to collapse multiple rare variants [40], known as the cohort allelic sum test (CAST) and combined multivariate and collapsing (CMC) method.
The CAST method measures the differences (between case group and control group) in the number of individuals who carry one or more rare variants [39, 42], while the CMC method treats all rare variants as a single count for analysis with common variants [39].
Since both methods implicitly assume that all variants influence the phenotype in the same direction [39, 40], it could introduce substantial noise if a lot of the rare variants are neutral [40].
Alternatives to these two basic methods are also available [40].
For example, by default, the VAAST program combines all variants that have less than three copies in a gene into one pseudo-site for scoring.
Title: Pathway analysis

Pathway analysis is an alternative way to search for deleterious genes.
The assumption is that a single mutation in a single gene may not be very harmful but that a combination of mutations in several genes causes the disease.
In this type of analysis, variants are put in a context of biological processes, pathways, and networks to gain global perspective on the data.
A large number of knowledge bases and tools have been developed for this task (reviewed in [43, 44]).
In general, pathway systems contain a curation process that is either manual or automatic, followed by database assembly and refinement [43].
Many public knowledge base and analysis tools are available, such as Kyoto Encyclopedia of Genes and Genomes (KEGG) (http://www.kegg.jp/); BioCyc (http://www.biocyc.org/), provided by SRI international; PANTHER [45]; Reactome [46]; and DAVID [47].
Optionally, commercial systems, such as Ingenuity Pathway Analysis (http://www.ingenuity.com/products/ipa), Pathway Studio (http://www.elsevier.com/online-tools/pathway-studio), and GeneGo (http://portal.genego.com/), are also available.
Ingenuity Variant Analysis (http://www.ingenuity.com/products/variant-analysis) is also backed up with Ingenuity Knowledge Base and can consider the upstream and downstream genes of a variant in a pathway and evaluate the potential role of a gene by incorporating the related genes.
To help readers who have limited experience with NGS data to understand the workflow, in this section we will show a real example of a pipeline that was used to process whole-exome sequencing data from raw sequences to a ranked candidate gene list (Fig.1).
The pipeline is constructed, based on the three-phase organization, as described in the previous sections.
We start from raw sequence FASTQ files and assume no quality control has been performed or reported.
Please note that each tool is likely to require dependent files, such as the reference genome, the 1000 Genome variants, dbSNP variants, and the genome feature file in GFF3 format, etc.
Title: Sequence mapping

- Raw sequence quality control.
FastQC can be used to perform graphical quality checks on the raw sequence.
FastX can be used to filter the reads by specifying quality score ranges.
- Initial mapping.
BWA is recommended for mapping Illumina reads.
The first step of BWA is to generate suffix array coordinates for mapping the reads; then, one performs the actual alignment and outputs SAM format files.
The SAM file is converted to BAM format, sorted, and indexed using Samtools for faster downstream processing (sorting is often required by many downstream tools).
- Alignment recalibration.
The GATK RealignerTarget-Creator tool is used to create target regions for realignment, followed by the IndelRealigner tool to perform the actual realignment and output new BAMs.
The Picard MarkDuplicates tool is then used to mark and remove duplicated sequences, followed by the Picard BuildBamIndex tool to rebuild the index.
Next, GATK&#39;s BaseRecalibrator tool is used to recalibrate the base quality scores, and PrintReads is used to output the final, analysis-ready reads in BAM format.
Title: Variant discovery

- Initial variant discovery.
The GATK UnifiedGenotyper tool is used to perform initial raw variant calling and output VCF files.
- Variant recalibration and filtering.
SNPs and indels are processed separately but in a similar manner; so, we will only describe SNPs as an example.
The GATK Select-Variants tool is used to select SNPs within the exome regions from raw VCF files.
The VariantRecalibrator tool is then used to build Gaussian models for recalibration, followed by the ApplyRecalibration tool to apply models for SNP recalibration.
Low-quality SNPs are then removed by imposing a filtering step, based on a user-defined variant quality score recalibration (VQSR) score threshold.
After SNPs and indels are reprocessed, the GATK CombineVariants tool is used to combine the recalibrated SNPs and indels.
Title: Disease gene prioritization

- Variant annotation.
VCF files are converted to genome variation format (GVF) format using the VAAST vaast_converter tool.
The variants are then annotated using the VAAST VAT tool.
- Variant selection.
The annotated variants are selected and condensed to condenser (CDR) format using the VAAST VST tool for cases and controls, respectively.
- Gene prioritization.
VAAST is used to perform disease gene ranking and output a text file containing a list of ranked candidate genes.
High-throughput data processing and analysis are becoming reliable and efficient, thanks to numerous high-quality tools that are mostly open source, with long-term active support and frequent updates to incorporate new advances in the field.
A typical workflow covering all three phases outlined above involves many different tools and/or steps, databases, score systems, and file re-formatting steps.
Therefore, it is not a trivial task to perform a data processing task, starting from the raw FASTQ files to the identification of candidate genes.
There are many tools and dependencies of tools that need to be installed and many steps and many parameters that need to be specified.
This could be a big hurdle for researchers that have limited experience with NGS data pipelines.
It would be a good idea for the community to converge on several standardized, well-configured, project-optimized, and automated workflows that can be used widely with minimum user interface.
HugeSeq [17] by Stanford is a comprehensive, automated workflow that integrates around two dozen bioinformatics tools and can process FASTQ/FASTA files, perform various quality control and clean-ups, and output variant call files.
Commercial services, such as Seven Bridges Genomics and Geospiza, also have preconfigured, comprehensive, automated pipelines for their customers.
Flexibility of pipelines, on the other hand, may be important for researchers who want to have complete control on the pipeline design to fit their projects.
In such cases, it is important to have plasticity for users to customize pipelines.
For example, a flexible pipeline system should allow users to integrate specific tools that are not part of a general purpose workflow and allow users to supply user-defined control files (e.g., providing a BED file to restrict the variant call within specific regions).
There are a number of such pipelines that are well defined and easy to use, such as the GATK pipeline, gkno system (http://gkno.me/), Galaxy [48-50], and VAAST [30, 31].
Highly efficient and accurate bioinformatics tools are available for most of the steps for analyzing sequence data, from processing the raw NGS data to generating the final report of potential risk genes.
Users can choose automated workflow versus customized pipelines to suit their research projects.
Many bioinformatics tools are optimized, such that a small server (e.g., 48 cores, 250 GB RAM, 2 TB storage) can perform the whole analysis workflow within a reasonable time (e.g., 250 hours in nonparallel mode and 25 hours in parallel mode for one exome at 30× coverage, as benchmarked by HugeSeq).
Furthermore, the scientific community is converging towards standard workflows and standard file formats, crossing different research institutes, companies, and platforms.
Several file formats are becoming standard, such as FASTQ/FASTA, SAM, BAM, VCF, and GFF3.
Large publically funded projects, such as the 1000 Genome Project, the HapMap Project, and the NHLBI GO Exome Sequencing Project, have played important roles in the unification process.
Nevertheless, many challenges remain, and we would like to highlight a few: 1) data standards still need wider agreement.
For example, a number of variant discovery tools for SV and CNV use different file formats and need to be converted to the more standard format for downstream analysis [17]; 2) more standardized, automated workflows need to be developed to accommodate different data and projects and minimize the user interaction, such that different research groups can perform independent data analysis and the results can be easily compared; and 3) large, high-quality, and unified databases are essential for disease gene identification/prioritization, and continuous efforts from the scientific community are needed.
Title: In addition to the challenges, a number of improvements that are urgently needed for current analysis pipeline are being actively developed and implemented:

- Sequence mapping.
Apply haplotype-based mapping and de novo assembly to reduce mismatches and increasing specificity.
- Variant discovery and genotyping.
Improve the sequencing technology (e.g., longer read length) and analytical methods (e.g., de novo assembly-based variant discovery) to identify and infer genotypes of complex variations, such as SVs, CNVs, large indels, and transposons.
- Candidate gene prioritization.
Develop complex prioritization strategies for large pedigrees, combine linkage analysis with association studies, and integrate pathway analysis to eliminate false positive genes.
- Non-coding variant annotation.
Understand the functional impact of non-coding DNA elements, as the Encyclopedia of DNA Elements (ENCODE) Project intended to do, to greatly broaden our view of disease-causing mutations.
- Cloud computing.
High-throughput sequencing projects generate large amounts of data, create huge computational challenges, and require numerous tools and libraries for comprehensive data analysis.
Develop cloud-based computing resources, such as the 1000 Genome Project (http://aws.amazon.com/1000genomes/), to create a more efficient way of managing, processing, and sharing data.
High-throughput sequencing will continue to transform biomedical sciences in almost every aspect [51] and provide new insights for our understanding of human diseases.
For example, three groups recently reported discoveries of several new autism genes and suggested a much more complex disease mechanism, based on large-scale sequencing data of nearly 600 trios and 935 additional cases [52-54].
Along with the revolutionary discoveries based on NGS data, new tools and techniques will be developed to facilitate fast and accurate analysis.
Example workflow for disease gene identification using next-generation sequencing data.
SAM, sequence alignment/map; BAM, binary format; SNP, single nucleotide polymorphism; VCF, variant call format.
Title: Review of Biological Network Data and Its Applications

Studying biological networks, such as protein-protein interactions, is key to understanding complex biological activities.
Various types of large-scale biological datasets have been collected and analyzed with high-throughput technologies, including DNA microarray, next-generation sequencing, and the two-hybrid screening system, for this purpose.
In this review, we focus on network-based approaches that help in understanding biological systems and identifying biological functions.
Accordingly, this paper covers two major topics in network biology: reconstruction of gene regulatory networks and network-based applications, including protein function prediction, disease gene prioritization, and network-based genome-wide association study.
Network representations have been used to describe interactions between entities of interest in various areas.
In particular, network representations are useful to analyze and visualize complex biological activities.
Global patterns in a large-scale complex system can be shown by representing the entities and their interactions with nodes and edges, respectively.
For instance, Schwikowski et al.
[1] created protein-protein interaction (PPI) networks to predict novel protein functions in yeast Saccharomyces cerevisiae.
By using the network representations, it was found that 2,358 among 2,709 total interactions compose a single large network.
Also, biological pathway databases, such as Kyoto Encyclopedia of Genes and Genomes (KEGG) [2] and Reactome [3], provide numerous pathways that are represented by networks with nodes of molecules and directed edges of their actions.
In addition, various mathematical properties and models of a network have been developed in graph theory.
Several reviews [4-6] have illustrated the mathematical properties and topological characteristics of a network with biological systems.
The advance of high-throughput technologies, including DNA microarray [7], next-generation sequencing [8], and the two-hybrid screening system [9], has led to the large-scale datasets in genomics and proteomics, which are referred to as &#39;omics&#39; data.
These omics data have been collected and organized to identify biological functions.
This paper focuses on biological network analysis related to omics data, such as gene expression levels and PPIs.
We first report several major public interaction databases for the omics data and then introduce two major topics in network biology: reconstruction of gene regulatory networks (GRNs) and network-based applications, including protein function prediction, disease gene prioritization, and network-based genome-wide association study.
Experimental results from high-throughput technologies, such as the two-hybrid screening system for detecting interactions between biological molecules, have formed various types of network datasets that are released and updated in public databases on the web.
These databases commonly enable web-based searches and provide raw datasets of pairs of molecules.
In this review, we report 11 useful public databases for PPIs and transcriptional regulatory interactions (Table 1) [10-20].
For PPIs, BioGRID [13], MIPS [11], and STRING [16] are the most frequently used to predict protein functions for PPIs.
BioGRID provides 496,761 non-redundant PPIs for various species, ranging from yeast S. cerevisiae to human.
On the other hand, MIPS not only provides mammalian PPIs but also offers the functional catalogs that contain descriptions of protein functions.
Unlike BioGRID and MIPS, STRING contains identified and predicted functional interactions of proteins with functional similarity scores (i.e., STRING offers weighted networks).
For transcriptional regulatory interactions, transcriptional regulatory element database (TRED) [19] offers GRNs and transcriptional factors for three species; human, mouse, and rat.
On the other hand, RegulonDB [20] contains both experimental datasets and computational prediction results of transcriptional regulatory interactions for the Escherichia coli K-12 organism.
From the advances of high-throughput technologies, large-scale networks have been identified and are available from various public databases, as summarized in Table 1.
In this section, we focus on network-based applications, especially how to use previously identified network resources in order to obtain meaningful biological interpretations.
We first introduce three basic concepts in graph theory [53] to give an overview of the basic concepts of the network-based methods.
First, a neighborhood of a node is a set of nodes connected to the node.
Second, the distance between two nodes is defined as a length of the shortest path between them if the path exists; otherwise, it is set to infinity.
Finally, an adjacency matrix is a binary and symmetric matrix such that its ij-th element is equal to 1 if there is an edge from a node i to a node j; otherwise, it is 0.
In some cases, a weighted adjacency matrix can be used to represent the strengths of edges that usually fall between 0 and 1.
With these basic concepts, we introduce three network-based applications: protein function prediction, disease gene prioritization, and genome-wide association study.
Title: Protein function prediction

With the results of sequencing genomes, the efforts to predict protein functions have been focused on the functionalities of genomic annotations.
In the initial stage, the prediction of protein function begins with the sequence homologies to annotated proteins [54-57].
These methods have been successful, but 70% of proteins still remain unannotated [58].
Accordingly, various types of methods have been developed to characterize unannotated proteins.
In this review, we introduce four approaches in protein function prediction, based on direct interactions in the network: neighborhood-based, graph-based, Bayesian, and Kernel-based approaches.
The methods are summarized in Table 3 [10, 11, 13-16, 59-83].
In neighborhood-based approach, the proposed methods [1, 62, 63, 65] commonly consider the number of edges connected to annotated proteins in a neighborhood.
The neighborhood counting method [1] only takes into account the frequencies of annotated proteins in the neighborhood and chooses the top three functions, with the calculated frequencies ranked in descending order for each protein.
Other methods also have derived their own scores, such as χ2 statistics [62] and functional similarities [63, 65], based on the annotated protein information in the neighborhood.
These methods choose a function with the highest score as a predicted function for each protein.
The graph-based approach is similar to the neighborhood-based approach, but the graph-based approach focuses more on how to label the unannotated proteins with graph theoretical properties, such as the distance and the adjacency matrix.
The label assignment models have been proposed with the adjacency matrix [66] and the weighted adjacency matrix [67], respectively.
Since these assignment problems are computationally intractable, the heuristic methods, such as the simple threshold rule or simulated annealing [84], have been applied.
To avoid these computational problems, several propagation-flavored methods have been developed.
First, the label propagation methods [69, 70] obtain the optimal assignments and the optimal combination of the weight matrices that reflects different types of networks.
Second, the functional flow method [68] iteratively spreads the flow from the annotated protein to the unannotated proteins by connected edges.
The functional flow score, defined as an amount of the flow, is the criterion of the prediction.
Third, the sequential linear neighborhood propagation method [71] sequentially updates unlabeled proteins according to their shortest path distance to the set of labeled proteins.
Finally, the neighbor relativity coefficient (NRC) method [72] derives the NRC score by integrating various graph topological properties, such as the shortest path distance, path connectivity, and common neighbors.
The Bayesian approach takes into account the posterior probabilities of binary label random variables to obtain the prediction from the observed network and annotated proteins.
Markov random field (MRF)-based [85] methods [74, 75] have been proposed and modified to Bayesian MRF [77] recently.
To predict protein functions, these methods commonly derive the marginal posterior probability of the binary label variable given other variables and then estimate the posterior probability by Gibbs sampling.
In addition to MRF-based models, other probabilistic models [78-80] have been developed under hierarchical structures from gene ontology [64]-which provides gene product annotation data that are characterized into three categories: biological processes, cellular components, and molecular functions-with some models, such as the hierarchical binomial neighborhood model [79].
In particular, Jiang et al.
[80] considered the auto-probit model with a weighted network information from STRING [16], and their auto-probit model also reflects the uncertainty of the annotation [64].
The Kernel-based approach considers the protein function prediction problem as a classification problem.
To reflect network information into the classification state, the network information is converted into a kernel matrix.
Lanckriet et al.
[81] proposed the kernel-based support vector machine (SVM) method, which incorporates heterogeneous types of data, such as amino acid sequence, gene expression data, and PPI network data, by converting these data into kernel matrices.
The SVM method can be reformulated as semi-definite programming (SDP) [86] with kernels.
Although the SVM method performs well, this method becomes slow when the dimension increases, caused by the computational complexity of the SDP.
To remedy this problem, Lee et al.
[82] proposed the kernel-based logistic regression (KLR) method by combing MRF-based methods [74, 75] with the diffusion kernel [87].
The KLR can contain multiple functions and various types of datasets at once.
It has been shown that the KLR method is faster than the SVM and is comparable to the SVM in prediction accuracy [82].
Recently, Wang et al.
[83] proposed the function-function correlated multi-label method, which treats all function categories in the prediction at once, while other methods only consider one function at a time, except for KLR.
Title: Disease gene prioritization

Disease-gene association studies have focused on identifying relationships between disease phenotypes and genes to reveal and understand human disease mechanisms.
Although traditional approaches, including linkage analysis and association studies, have been successful, the specified genomic intervals often contain tens or even hundreds of genes.
It may not be possible by experiments to identify the correct causative genes of all the genes that lie on the specified intervals.
To reduce experimental costs and efforts, the prioritization of candidate genes becomes one of the major tasks in disease-gene association studies.
Taking into consideration that genes corresponding to similar disease phenotypes tend to be neighbors in either a PPI network [88, 89] or a pathway [90], several network-based disease gene prioritization methods have been proposed recently.
The network-based disease gene prioritization methods [91-94] define different similarity scores between the disease and genes, based on either functional annotations or PPI networks, to rank candidate genes.
The random walk method [91] considers the random walk on graphs and uses a diffusion kernel matrix [87] derived from a PPI network as the steady-state transition probability matrix.
The random walk method then defines the similarity score as the sum of elements of the diffusion kernel corresponding to known disease genes for each candidate gene.
CIPHER [92] adopts a regression model for similarities between phenotypes and considers the Gaussian kernel for closeness between genes.
The Pearson correlation coefficient between the similarity of phenotypes and the closeness between genes is used as a similarity score.
On the other hand, PRINCE [93] and MINProp [94] are based on the iterative label propagation algorithm [95].
PRINCE defines the prioritization function, which combines functional similarities from the network information with prior information for disease phenotypes as the similarity score.
The prioritization function is obtained by iteratively applying a label propagation algorithm.
MINProp first defines two heterogeneous networks, such as a gene network and a disease network, and then additionally defines bipartite networks between two heterogeneous networks from known disease-gene associations.
After initializing the labels of the disease genes, MINProp iteratively propagates label information through three networks until convergence occurs.
Finally, the converged label scores are used as the similarity scores.
From the comparison study in [93, 94], PRINCE and MINProp outperform the random walk and CIPHER methods, respectively.
There is no comparison study between PRINCE and MINProp, but MINProp is more general than PRINCE, since MINProp can be applied to three or more heterogeneous networks.
We report these disease gene prioritization methods with their data resources in Table 4 [12, 13, 15-18, 91-94, 96-99].
Title: Genome-wide association study (GWAS)

The GWAS measures DNA sequence variations in the human genome to identify associations between genetic variants and diseases (or phenotypes).
To measure genetic variations, the single- nucleotide polymorphism (SNP), which represents a single base-pair change in the DNA sequence, is generally used as a marker of a genomic region in the GWAS.
Generally, the GWAS conducts a comparison of the SNPs between case and control groups (i.e., disease and non-disease groups) by statistical methods, such as ANOVA and logistic regression, to identify significant SNPs related to the disease.
Genetic risk factors revealed by a GWAS can be used as preventive markers or for therapeutic targets in curing the disease.
There have been more than 11,000 SNPs discovered as candidate bio-markers in common diseases [100].
The large number of SNPs detectable in the human genome can, however, lead to multiple testing problems.
To control the false positive errors (i.e., type I errors in the context of statistical testing procedures), the Bonferroni correction and false discovery rate [101] methods are commonly adopted.
Although these multiple testing rules have been successful in the identification of significant single SNPs, these test procedures often fail to detect genomic regions that are weakly associated with the disease and still ignore the combined effects caused by the interactions between genes.
The network-based GWAS methods [102-104], summarized in Table 5 [12, 13, 17, 102-114], take into account both interactions between genes or proteins with association information available from a GWAS.
The HumanNet method [115] combines functional gene networks derived from multiple network sources, such as the PPI network, and mRNA co-expression with the log odds ratio from GWAS data to prioritize disease genes.
By combining functional gene networks and the information from GWAS data, HumanNet has higher power to detect disease-related genes.
Unlike HumanNet, NIMMI [116] and dmGWAS [117] focus on identifying groups of genes associated with diseases, based on GWAS data and PPI network data.
NIMMI constructs a PPI network with weights of genes by using a modified Google PageRank algorithm [118].
The weights of genes are then combined with the gene-based association p-values calculated from GWAS data.
The subnetworks of genes are identified by the DAVID method [119, 120] with the network structure and weights of genes.
On the other hand, dmGWAS only considers gene-based association p-values as the node weights and proposes a modified version of the dense module searching method [121] to prioritize candidate groups of genes.
We have reviewed a number of methods related to two topics in network data analysis: network reconstruction and network-based application.
Network reconstruction can be thought of as a reverse-engineering problem whose goal is to rebuild true structures or relationships from observations.
In particular, we focused on statistical methods for building GRNs, including the Gaussian graphical model, correlation network, Bayesian network, and information theory-based methods.
Most methods that we have reviewed consider the sparsity on the graph structure to select a small number of significant dependencies between nodes.
This sparsity condition is adequate for the network in biological systems, since it reflects a scale-free feature, where several nodes have many edges but the majority of nodes only has three or four edges [5].
Although most methods in Gaussian graphical models are well studied in their theoretical properties, these methods have limitations when applied to biological data.
Since these methods basically assume a Gaussian distribution, they are only applicable for continuous-type datasets, such as gene expression levels.
To construct networks from other types of data, such as binary or counts, the Bayesian network and information theory-based methods are more attractive than correlation-based methods.
Second, we introduced various methods in three network-based applications.
Most methods consider similarity measures between nodes and then use these measures to predict biological functions or prioritize candidate genes associated with diseases.
As technologies in the experiments progress, the network-based methods can be improved and widely extended.
For instance, even though the neighborhood counting method [1] in protein function prediction only considers the count of annotated functions in the neighborhood from a PPI network, the recently developed methods [72, 83] can contain not only neighborhood information but also functional similarities from multiple networks.
In addition, module-assisted methods that focus on identifying a functional group of proteins are also available and well summarized in [122].
Furthermore, the network-based tumor stratification (NBS) method has been developed recently [123].
This NBS method combines mutation profiles from The Cancer Genome Atlas (TCGA) projects [116, 117, 124, 125] and network information to obtain informative strata (e.g., subtypes of cancer).
Although these network-based methods have been improved, these methods still lack accuracy compared with other methods [115].
Since the high-throughput data may contain many false positives [126], the network-based methods are affected in their accuracy.
Although their performance depends on the quality of data, their effects are expected to decrease in the future as improvements are made in measurement accuracy.
Title: Public network resources

Title: Reconstruction methods for gene regulatory network

Title: Network-based protein function prediction methods

Title: Network-based disease gene prioritization methods

Title: Network-based genome-wide association study methods
Title: Current Status, Challenges, Policies, and Bioethics of Biobanks

Many biobanks were established as biorepositories for biomedical research, and a number of biobanks were founded in the 1990s.
The main aim of the biobank is to store and to maintain biomaterials for studying chronic disease, identifying risk factors of specific diseases, and applying personalized drug therapies.
This report provides a review of biobanks, including Korean biobanks and an analysis of sample volumes, regulations, policies, and ethical issues of the biobank.
Until now, the top 6 countries according to the number of large-scale biobanks are the United Kingdom, United States, Sweden, France, the Netherlands, and Italy, and there is one major National Biobank of Korea (NBK) and 17 regional biobanks in Korea.
Many countries have regulations and guidelines for the biobanks, and the importance of good management of biobanks is increasing.
Meanwhile, according to a first survey of 456 biobank managers in the United States, biobankers are concerned with the underuse of the samples in their repositories, which need to be advertised for researchers.
Korea Biobank Network (KBN) project phase II (2013-2015) was also planned for the promotion to use biospecimens in the KBN.
The KBN is continuously introducing for researchers to use biospecimens in the biobank.
An accreditation process can also be introduced for biobanks to harmonize collections and encourage use of biospecimens in the biobanks.
KBN is preparing an on-line application system for the distribution of biospecimens and a biobank accreditation program and is trying to harmonize the biobanks.
Biobanks provide important materials for many research realms, such as biomarker development for the early diagnoses of specific diseases, including cancer and genetic diseases, and for applying personalized drug therapies.
There is no consensus definition for a biobank, but it is considered a biorepository that stores biospecimens for use for diagnostic or research purposes.
Also, a biobank is an institution that collects, stores, processes, and uses biological materials, genetic data, and associated epidemiological data from human beings or distributes them to researchers [1].
The term &#34;research biobank&#34; means a collection of human biological material and data obtained directly by the analysis of this material, which is used or is to be used for research purposes [2].
A repository is defined as an organization, place, room, or physical entity that may receive, process, store, maintain, and/or distribute specimens, their derivatives, and their associated data.
In this paper, we provide a review of biobanks, including Korean biobanks and an analysis of sample volumes, regulations, policies, and ethical issues of the biobank.
The oldest biobank for the Framingham Heart Study (FHS), funded by the National Institute of Health-National Heart, Lung, and Blood Institute (NIH-NHLBI), collects blood samples and data and was established in 1948.
The purpose of the Framingham program was the development of case-finding procedures.
A total of 5,209 persons between the age of 30 and 62 from Framingham, Massachusetts participated in this study, and three generations of participants (for a total of almost 15,000 participants) were recruited; the researchers began clinical examinations and lifestyle-related interviews to seek risk factors related to cardiovascular disease (CVD) development.
FHS has revealed the significant identification of the main CVD risk factors, such as high blood pressure, high blood cholesterol, smoking, and obesity, as well as immense information on factors, such as blood triglyceride and high-density lipoprotein cholesterol levels, age, and so on [3].
By the late 1990s, scientists realized that some diseases originated from a single defective gene, but most genetic diseases are caused by multiple genetic factors on multiple genes [4].
For an understanding of whole-genome information of humans, the Human Genome Project (HGP) was begun in 1990, and the human genome was completely released in 2003.
Owing to the increased demands on qualified biospecimens for research, the number of biobanks has increased significantly between 1980 and 1999 worldwide [5].
Many countries, including the European countries, Japan, and Korea, have established large-scale biobanks to collect a large quantity of patient specimens from over 200,000 people, based on their population and/or on their disease-oriented groups (Table 1).
Until now, the top 6 countries, according to the number of large-scale biobanks, have been the United Kingdom (n = 15), United States (n = 14), Sweden (n = 12), France (n = 9), the Netherlands (n = 8), and Italy (n = 8).
Seventy percent of the world&#39;s biobanks are located in Europe.
Sixty percent of sponsors for biobanks are governmental or national institutes, and 16% to 17% of biobanks are sponsored by non-profit public service corporations, universities, and hospitals.
The number of biobanks is increasing recently, and particularly, 43 biobanks were launched in the 2000s.
Most biobanks (60%) recruited less than 100,000 people and thirty percents of the biobanks recruited 100,000 to 1,000,000 people.
The number of biobanks collecting blood is 35; 35 collect other biofluids, 43 collect biomaterials, and 16 secure tissue [6].
In 2005, the UK Biobank started collecting DNA samples and personal information from 500,000 volunteers aged between 45 and 69 years.
The main purpose of the UK Biobank is to look for the relationship between disease, lifestyle, and genes, as well as to identify risk factors that affect individual response to specific drug treatment.
The UK Biobank has collected biospecimens from 500,000 people between 2006 and 2010, and they provided their specimens to researchers starting from March 2012 [7, 8].
In 2008, United States researchers stored 270 million specimens in biobanks, and the rate of new sample collection is 20 million per year [9].
In 2009, Time magazine chose biobanking as one of the 10 ideas &#34;changing the world right now&#34; [10].
Elger and Caplan [11] demonstrated that the challenge produced by biobanks is immense: after more than 50 years of classical health research ethics, regulatory agencies have begun to question fundamental ethical milestones.
A new report by Visiongain (http://www.visiongain.com), a business information provider based in London, UK, predicts the world market for biobanking on human medicine will generate $24.4 bn in 2017 and expand strongly to 2023.
In 1995, the Ministry of Education and Science Technology established the Korea National Research Resource Centers for collecting various bioresources.
However, the scale and variety of the collections were lacking, because collections were based on an individual research project.
The Ministry of Environment established biobanks, but their samples were limited to specimens needed for the research of environmentally induced diseases.
Therefore, there was a necessity to establish a biobank that was centered on hospitals for collecting human biospecimens [12].
In 2008, the Ministry of Health and Welfare started the Korea Biobank Project (KBP) and tried to support establishing biobanks within university-affiliated general hospitals as well as creating a network among these biobanks, including National Biobank of Korea (NBK), which is the largest national biorepository in Korea, located in Osong, the central region in South Korea [13, 14].
NBK provides essential biospecimens to scientific researchers by collecting, maintaining, and distributing DNA and serum, plasma, and lymphoblastoid cell line (LCL).
A new building for NBK, the largest in Asia, was constructed in April, 2012 at Osong, Chungcheongbuk-do, Korea.
The three-story building, including storage rooms that can contain 26 million vials of biospecimens, was constructed.
The structure also has laboratories, an auditorium, and so on.
The budget for the project was secured from the Korean central government in 2008, and the construction was started in 2010 and finished in 2012 [14].
Over the years, many biobanks, including 17 regional biobanks, have been established and are moving from a phase of sample collection to open their samples for researches.
There are many guidelines for the management of biobanks, such as the International Society for Biological and Environmental Repositories (ISBER) guideline and Organization for Economic Cooperation and Development (OECD) [15] guideline [16].
ISBER is the largest international forum that addresses the technical, legal, ethical, and managerial issues relevant to repositories of biological and environmental specimens.
ISBER has the &#34;Best Practices for Repositories/Collection, Storage and Retrieval of Human Biological Materials for Research.&#34;
Also, ISBER has introduced standard PREanalytical Coding (SPREC) and can be used in the biobank [17].
In October 2009, the OECD council adapted a recommendation for Human Biobanks and Genetic Research Databases (HBGRDs).
The purpose of this recommendation suggests guidelines for the establishment, management, governance, operation, access, use, and discontinuation of HBGRDs, which are structured resources that can be used for the purpose of genetic research.
The OECD Best Practice Guidelines for Biological Resource Centers set out further complementary quality assurance and technical aspects for the acquisition, maintenance, and provision of high-quality biological materials in a secure manner.
The standard operating protocols (SOPs), regulations, and management for human biospecimens are important.
DNA stability and DNA purity in clinical samples are essential for correct performance and interpretation for diagnosis and research.
Generally, quality assurance of human biospecimens consists of a DNA stability test, DNA purity test, microorganism contamination test, and cross-contamination test.
All biospecimens collected from the Korea Genome and Epidemiology Study (KoGES) in the case of the NBK underwent a DNA degradation test and purity test.
Ten percent of biospecimens stored in NBK were randomly chosen and tested for the quality assurance test to ensure the high quality of DNA extracted from the biospecimens and to check contamination by microorganisms.
The quality of DNA is important for genetic research, such as genome-wide association studies (GWASs) and whole-genome sequencing (WGS).
In general, pure preparations of DNA and RNA have an OD260/OD280 of 1.8 and 2.0 respectively.
LCL quality assurance was also conducted, and 10% of the stocked LCL was randomly selected to undergo survival rate investigation and cross-contamination test in the NBK [13].
Informed consent is getting a patient to sign a written consent form and is the process of communication between a patient and a medical doctor.
In medical diagnosis and treatment, including invasive procedures, a doctor or a researcher must give sufficient information to the patient.
Informed consent can be given based on a clear understanding of the process, implications, and consequences of an action.
The consent process should include sufficient explanation about how individual information would be handled [18].
The consent process includes the impact of research results on the lives of participants, their families, and their communities [19].
Explanations should be on confidentiality, participants&#39; rights of withdrawal, and specifications [20].
Due to the difficulties of securing specific informed consent, broad consent is a more general form of consent in which individuals agree to have their biosamples and personal information collected and stored in the biobank and for future, unspecified research [21].
Further issues have been discussed recently, such as privacy, confidentiality and data protection, controlling data access, accessibility to biospecimens, benefit sharing, commercialization, intellectual property rights, and genetic discrimination [22].
In order to ensure bioethics and bioethical safety in the life sciences and biotechnologies, each of the institutions should set up its own Institutional Review Board (IRB).
When a genetic testing or research institution obtains written informed consent from a test subject concerning the use of specimens for research purposes, it may provide the specimens to either a person conducting research on genes or an institution licensed to open a biobank in Korea [1].
In Korea, the Bioethics and Safety Act was completely revised to reinforce research ethics, including biobanks, and the law took effect in 2013.
The number of biobanks within university-affiliated hospitals will increase.
This law encourages biobanks to strengthen ethical use of biospecimens, and the law assesses biobanks&#39; responsibility to obtain informed consent from donors and is supervised by the IRB.
The law also encourages individual researchers to deposit and use biospecimens at any biobank instead of collecting their own sample collection and distribution processes [1, 14].
Many biobanks have policies for the distribution of biospecimens and data.
For example, there is a guideline to distribute biospecimens in the National Health and Nutrition Examination Survey (NHANES), which is a program of studies designed to assess the health and nutritional status of adults and children in the United States.
NHANES&#39;s biospecimens are representative samples of Americans and should not be used for the independent case-control study, and 1individual vial must be stored permanently.
All proposals for the use of NHANES samples are evaluated by a technical panel for scientific merit and by the NHANES Ethics Review Board (ERB).
To determine if this limited resource should be used in the proposed projects, a technical panel will evaluate the public health significance and scientific merit of the proposed research.
Distribution of a 0.5-mL sample is available once, and after finishing the research, the remaining samples are abandoned, or remnants more than 0.3 mL should be returned to NHANES; third-party biospecimen transfer is not permitted.
Researchers who received specimens must submit the result of the analysis, such as quality control and analytical methods with documents, to the National Center for Health Statistics (NCHS).
Surplus data without DNA analysis results would be opened in the website and given a period of 3 months for analysis.
A researcher&#39;s intellectual property rights are not permitted because of research for public health (http://www/cdc/gov/nchs/nhanes.htm).
The Prostate, Lung, Colorectal, and Ovarian Cancer Screening Trial (PLCO) biorepository in the United States has 2.9 million biological samples, including blood, and researchers can use samples and data through the process of peer review.
Researchers can submit summarized proposals to the PLCO, and the chosen research is recommended for full research proposals and submissions are available twice in June/July or December/January annually.
The review process takes 4 or 6 months.
Researchers with approved proposals should submit IRB approval documents and contract the Material Transfer Agreement (MTA) mutually, and biospecimens are distributed to researchers when approved; researchers should report experimental results to Information Management Systems (IMS) (http://www.plcostars.com).
At present, distribution of biospecimens is available to Korean researchers irrespective of research funds, and the distribution for data is accessible to Korean researchers without financial sponsors in Korean biobanks.
All proposals for distribution of data and biospecimens should be reviewed and be approved by the IRB belonging to an institute prior to submission.
Opened data and biospecimens for distribution in the NBK are shown in Tables 2 and 3.
First of all, researchers who want to receive the data and biospecimens have to submit the application document to the NBK.
The submitted documents will be reviewed, and a board meeting for distribution is held monthly; distribution of work for the approved application will be performed approximately within one or two months from submission of the application documents for distribution.
NBK is planning a website for the distribution process.
Most of the distributed biospecimens are DNA samples in the NBK, and a few body fluid samples, including sera and plasma, were distributed.
According to the advancement of the technology, the minimum quantity for clinical examination and for the analysis of metabolites is diminishing.
However, sera and plasma are very limited resources, and political decisions and long-term planning for the use of body fluid samples will be needed [12].
In 2000, following the HGP, biological research moved into the so-called genomic era.
Diseases have been studied by identifying genes and their specific function and understanding the role played by genetics in the beginning and prognosis of diseases, and an advanced world of medicine, known as &#34;personalized medicine,&#34; has been initiated [5].
Accessibility to human specimens and data for the purposes of research is important for realms, such as genomics, proteomics, metabolomics, molecular imaging, and nanotechnology [23].
There are genetic factors that affect susceptibility to common diseases [24].
These diseases may have several genetic risk factors that influence each other, and interact with the environment, and this has spurred the development of large-scale biobanking projects to identify susceptible genes [25].
Despite intensive research over the last decade, most of the genetic basis of common human diseases remains unknown.
The identification of meaningful genetic variants, genes, and pathways involved in special diseases offers a potential route to new therapies, improved diagnosis, and better disease prevention.
For some time, it has been hoped that the advent of GWASs would provide a successful new tool for solving the genetic basis of many of these common causes of human morbidity and mortality.
Advances in sequencing technology enabled us to sequence the whole human genome [26, 27].
The use of WGS of large patient cohorts is a much-needed approach in researching complex traits; it is still being ruled out to detect low-frequency genetic variants, due to its high costs [28, 29].
Among the new and emerging fields of research is proteomics, the study of the full set of proteins encoded by the genome.
Also, proteomics is strongly associated with the discovery phase, the first step in the process chain to create diagnostic content [30].
Due to the increased demands on qualified biospecimens for research, the number of biobanks has significantly increased recently.
By virtue of the advancements in bioinformatics and biotechnology, storing biospecimens and data on a large scale requires that biobanks harmonize biobank processes and regulations [5].
Most biobanks did their best to secure more biospecimens at the inception of the foundation and comparably distribution rate for the researchers is low.
According to a first survey of 456 biobank managers in the United States, nearly 70% of those questioned were concerned with underuse of the samples in their repositories [31].
Therefore, many biobankers are trying to advertise their biospecimens to researchers in regional academic societies.
Like the United States, the NBK has faced a similar situation for the underuse of biospecimens stored in the biobank.
In the first phase of the KBP (2008-2012), biobanks concentrated on securing facilities, such as storage equipment, as well as securing manpower and funding for biobanks.
However, in the second phase (2013-2015), more R&amp;D projects are needed that support clinicians, so that it will be possible to gather resources with more information.
For maximizing the value of biospecimens, consistency in the methods by which they are retrieved, processed, stored, and transported is important.
Ideally, this should involve the use of agreed SOPs in general [32].
The other issue in the next big step is automated biobanking, and today&#39;s biobanks demand and move toward automated sample management systems that meet the requirements for reliability, sample integrity, high capacity, and high throughput.
Horn, the former director of the Genetic Alliance, demonstrates that biobanks have to cooperate together to achieve their mission, and standardization is going to be important for researchers to get samples from different collections; we need catalogs that say where these samples exist [33].
Accreditation processes can also be introduced to biobanks for harmonizing collections and encouraging use of biospecimens in the biobanks, including the KBN.
The NBK is also preparing an online distribution portal for the convenient application of distribution to researchers and preparing a biobank accreditation program; these will harmonize the biobanks in Korea.
Title: Biobanks with sample sizes ≥ 200,000 (2012)

Title: Data for distribution ofthe National Biobank of Korea (as of June 2013)

Title: Biospecimens for distribution of the National Biobank of Korea (as of June 2013)
Title: Introduction to International Ethical Standards Related to Genetics and Genomics

The rapid advances in genetic knowledge and technology raise various, sometimes unprecedented, ethical dilemmas in the scientific community as well as the public realm.
To deal with these dilemmas, the international community has prepared and issued ethical standards in various formats.
In this review, seven international standards regarding genetics and genomics will be briefly introduced in chronological order.
Critical reflections on them will not be provided in this review, and naturally, they have their own problems and shortcomings.
However, a common set of the principles expressed in them will be highlighted here, because they are still relevant, and many of them will be more relevant in the future.
Some of the interesting contents will be selected and described.
After that, the morality of one recent event related to whole-genome sequencing and person-identifiable genetic data will be explored based on those international standards.
Title: Recently, there was a very unique bio-art exhibition by New York-based artist Heather Dewey-Hagborg, titled &#34;Stranger Visions&#34; [1].
She collected samples from discarded chewing gum, strands of hair, and cigarette butts around New York City.
She took these items to Genspace, a do-it-yourself community biology laboratory, where DNA was extracted from each of the objects.
By sequencing specific genomic regions and cross-referencing the results with published data, she was able to create 3-dimensional (3D) sculpted portraits of her anonymous donors using a 3D printer.
She said that this work is designed to stimulate a cultural dialog about genetic surveillance.
However, this provocative art project surely raises a question: Is it ethical?
To find an answer to this question, as a starting point, it would be helpful to review some major conventions, resolutions, declarations, or guidelines issued by international public organizations, such as the United Nations (UN); the United Nations Educational, Scientific and Cultural Organization (UNESCO); and the Council of Europe.
These norms have been providing the principles and recommendations that guide the member states in their legislation and policy.
They are worth reviewing, because, as Marie Curie once wrote, &#34;science is essentially international.&#34;
Knowledge, technologies, and money-capital and research fund-in the bioscience field cross borders freely and very quickly.
In this review, seven international standards relevant to genetics and genomics will be introduced briefly in chronological order with some background information, and the possible answer to the questions with regard to the bio-art project will be provided after that.
In July 1990, just a couple of months before the Human Genome Project (HGP) officially began, the Council for International Organizations of Medical Sciences (CIOMS) held the conference, titled &#34;Genetics, Ethics, and Human Values: Human Genome Mapping, Genetic Screening, and Therapy,&#34; in Tokyo and in Inuyama City, Japan.
The Conference was co-sponsored by the World Health Organization (WHO) and UNESCO.
Here, the Conference produced the Declaration consisting of eight statements.
The following information is the main content of the Declaration [2].
The Conference agreed that the HGP would present no inherent ethical problems but was concerned about genetic testing, which would be available through the project, since the identification and sequencing of new genes can expand the scope for diagnostic tests.
The Declaration emphasizes the importance of the welfare of the person tested and the confidentiality of the test results.
For this, adequate genetic counseling is recommended.
Also, the Declaration requires that the needs of developing countries should obtain their due share of the benefits that ensue from the HGP.
Title: Regarding gene therapy, the Declaration states that it should be limited to conditions that cause significant disability and not employed simply to enhance or suppress cosmetic, behavioral, or cognitive traits that are not related to any recognized human disease.
Regarding germ-line gene therapy, the Declaration states that &#39;continued discussion of both its technical and ethical aspects is essential and that, before germ-line therapy is undertaken, its safety must be well established, for changes in germ cells would affect the descendants of patients.&#39;
The Declaration acknowledges public concern about the growth of genetic knowledge, which is partly caused by a misconception: the genetic knowledge reveals an essential aspect of humanness (i.e., genetic essentialism); it may reduce human beings to mere base pairs of DNA (i.e., genetic reductionism).
To correct this misconception, the Declaration recommends education of the public and open discussion.
To ensure the setting and observance of ethical standards, continuous multidisciplinary and transcultural dialog and ethical sensitivity in policy-making are recommended.
The Declaration, consisting of 6 sections and 25 articles, was adopted unanimously on November 11, 1997 and endorsed by the UN General Assembly on December 9, 1998.
Its Guidelines for the implementation were also endorsed on November 16, 1999.
The aims of the Declaration are to promote and develop ethical studies and the actions arising out of them on the consequences of scientific and technological progress in the fields of biology and genetics, within the framework of respect for human rights and fundamental freedoms.
UNESCO recognizes that research on the human genome and the resulting applications open up vast prospects for progress in improving the health of individuals and of humankind as a whole.
However, it emphasizes that such research should fully respect human dignity, freedom, and human rights, as well as the prohibition of all forms of discrimination based on genetic characteristics.
The following information is the main content of the Declaration [5].
The Declaration covers the issues of human dignity, the human genome, research on the human genome, conditions for the exercise of scientific activity, solidarity, and international cooperation, etc.
Article 1 states that the human genome underlies the fundamental unity of all members of the human family, as well as the recognition of their inherent dignity and diversity.
It even declares that the human genome is the heritage of humanity in a symbolic sense.
Article 2 prohibits genetic discrimination and states that human dignity makes it imperative not to reduce individuals to their genetic characteristics and to respect their uniqueness and diversity.
Article 3 acknowledges that the human genome can be expressed differently according to each individual&#39;s natural and social environment, including the individual&#39;s state of health, living conditions, nutrition, and education, rejecting genetic determinism.
Article 4 declares that the human genome in its natural state shall not give rise to financial gains.
Article 10 affirms that no research or research applications concerning the human genome should prevail over respect for the human rights, fundamental freedoms, and human dignity of individuals or of groups of people.
Article 11 prohibits practices that are contrary to human dignity, such as reproductive cloning of human beings, but there is no mention of therapeutic cloning.
The Declaration consists of five sections and 27 articles and seems to be more relevant to researchers in the fields of genetics and genomics than the other six standards.
The aim of this Declaration is to ensure the respect of human dignity and protection of human rights and fundamental freedoms in the collection, processing, use, and storage of human genetic data, human proteomic data, and of the biological samples from which they are derived, in keeping with the requirements of equality, justice, and solidarity, while giving due consideration to freedom of thought and expression, including freedom of research.
It is also very relevant today, since the importance of human genetic data for economic and commercial purposes increases and the growing amount of personal data collected makes genuine irretrievability increasingly difficult.
The following is the content of the Declaration [6].
This Declaration covers various practical issues: withdrawal of consent; the right to know or right not to know of research results; access to his or her own data; privacy and confidentiality; circulation and international cooperation; sharing of benefits; storage and destruction; and cross-matching.
Article 4 describes that human genetic data have a special status on account of their sensitive nature: they can be predictive of genetic predispositions concerning individuals, and the power of predictability can be stronger than assessed at the time of deriving the data; they may have a significant impact on the family, including offspring, extending over generations, and in some instances on the whole group; they may contain information, the significance of which is not known at the time of the collection of biological samples; and they may have cultural significance for persons or groups.
Article 7 requires that every effort should be made to ensure that human genetic data and human proteomic data are not used in a way that leads to discrimination or stigmatization.
In this regard, appropriate attention should be paid to the findings of population-based genetic studies and behavioral genetic studies and their interpretations.
Article 19 emphasizes the need to share the benefits resulting from the use of human genetic and proteomic data or biological samples collected for medical and scientific research.
It describes the various forms of benefits: special assistance to the persons and groups that have taken part in the research; access to medical care; provision of new diagnostics, facilities for new treatments, or drugs stemming from the research; support for health services; capacity-building facilities for research purposes; and development and strengthening of the capacity of developing countries to collect and process human genetic data.
It was the first time in the history of bioethics that member states committed themselves and the international community to respect and apply the fundamental principles of bioethics set forth within a single text consisting of 28 articles.
This Declaration addresses ethical issues related to medicine, life sciences, and associated technologies as applied to human beings, taking into account their social, legal, and environmental dimensions.
The aims of this Declaration include: to recognize the importance of freedom of research and the benefits derived from scientific and technological developments, while stressing the need for such research and developments to occur within the framework of ethical principles; to respect human dignity, human rights, and fundamental freedoms; to safeguard and promote the interests of the present and future generations; and to underline the importance of biodiversity and its conservation as a common concern of humankind.
The following information is the content of the Declaration that is relevant to genetics and genomics [7].
Article 16 emphasizes the impact of life sciences on future generations, including their genetic constitution.
To protect the environment, the biosphere, and biodiversity, Article 17 encourages to pay due regard to the interconnection between human beings and other forms of life, to the importance of appropriate access and utilization of biological and genetic resources, to respect for traditional knowledge, and to the role of human beings.
Article 21 requires that, in transnational practices, States should take appropriate measures, both at the national and international levels, to combat bioterrorism and illicit traffic in organs, tissues, samples, genetic resources, and genetic-related materials.
However, some experts criticize the process of drafting the Declaration, which excluded a necessary group of stakeholders, and claim that the Declaration therefore fails to provide the guidance that can be readily applied in different settings [8].
Although there are more international norms besides the seven standards introduced in this article and will be more, a set of currently recognized, common ethical principles regarding genetics and genomics can be drawn from these standards (Table 1).
There are hardly any clashes over the main principles or attitudes towards the human genome among these norms.
Small differences, which are mostly found in details, have resulted from adding new principles and details, rather than changing or negating the previously declared principles.
Keeping these principles in mind, let us go back to Dewey-Hagborg&#39;s project.
The action concerned is the genetic testing of abandoned DNA without the donor&#39;s consent.
Did the artist violate their human rights, especially fundamental freedom, by testing their genomes without consent?
Since the artist does not know their identity, she could not obtain their consent.
One protocol provides ideas for cases when it is not possible to contact a person for a genetic test.
It says that the law may allow the test to be carried out when the testing is for the benefit of family member(s) on his or her biological material previously sampled for another purpose and when the expected benefit cannot be otherwise obtained and where the test cannot be deferred [9].
Therefore, Dewey-Hagborg&#39;s action seems unlikely to be justified.
In addition, from a legal perspective, New York&#39;s law outlaws most DNA testing without written consent, and the United Kingdom also outlawed non-consensual DNA testing in 2006.
However, the legal status of DNA retrieved in the environment is uncertain in many jurisdictions.
Another important question is whether the testing infringed their privacy.
Since the individuals concerned are unlikely to be recognizable from the sculptures, the artist argues that her project does not invade privacy.
Then, what if it is technically possible to identify the donor or is possible in the near future?
Does it change the nature of the action?
To all these questions, there may not be one correct answer at all or there can be many alternative answers to each question.
For most ethical dilemmas, the first step to find good answers to them is to have ethical sensitivity-the ability to perceive ethical problems as ethical problems.
After that, continuous, open, multidisciplinary, transcultural dialog will greatly help our journey, and our colleagues will be very reliable companions.
Title: Principles and suggestions common to the seven instruments
Title: Receptor for Advanced Glycation Endproducts (RAGE), Its Ligands, and Soluble RAGE: Potential Biomarkers for Diagnosis and Therapeutic Targets for Human Renal Diseases

Receptor for advanced glycation endproducts (RAGE) is a multi-ligand receptor that is able to bind several different ligands, including advanced glycation endproducts, high-mobility group protein (B)1 (HMGB1), S-100 calcium-binding protein, amyloid-β-protein, Mac-1, and phosphatidylserine.
Its interaction is engaged in critical cellular processes, such as inflammation, proliferation, apoptosis, autophagy, and migration, and dysregulation of RAGE and its ligands leads to the development of numerous human diseases.
In this review, we summarize the signaling pathways regulated by RAGE and its ligands identified up to date and demonstrate the effects of hyper-activation of RAGE signals on human diseases, focused mainly on renal disorders.
Finally, we propose that RAGE and its ligands are the potential targets for the diagnosis, monitoring, and treatment of numerous renal diseases.
Receptor for advanced glycation endproducts (RAGE) is a transmembrane protein that belongs to the immunoglobulin superfamily [1].
As its name implicates, it can bind to advanced glycation endproducts, the resulting product of non-enzymatic glycosylation [2], and it also has the ability to interact with multiple ligands having common motifs as a so-called multi-ligand receptor.
The ligands include high-mobility group protein (B)1 (HMGB1), S-100 calcium-binding protein, amyloid-β-protein, Mac-1, and phosphatidylserine [3].
Interaction between RAGE and its ligands activates various cellular processes, including inflammation, proliferation, apoptosis, autophagy, and migration [4].
Expression of RAGE is different, depending on the organ, developmental stage, and cellular condition [5].
It is highly expressed during the embryonic stage, whereas it is mostly kept at low levels in adults [5, 6].
Mostly, it is stimulated by cellular stresses, such as inflammation, and is therefore easily found to be abnormally over-expressed in many human chronic diseases [7].
Accumulation of advanced glycation endproducts (AGEs) is highly associated with the pathogenesis of diabetes, which causes serious consequences on metabolic systems [8].
In addition, expression of S100 proteins is reported to be involved in many cancers, including breast, lung, kidney, and thyroid cancers.
Mostly, their up-regulation promotes tumorigenesis by stimulating metastasis through the activation of relevant transcription factors or by acting as chemoattractants [9, 10].
In addition to cancers, RAGE and its ligands are abnormally regulated in various types of chronic immune diseases, such as atherosclerosis, Alzheimer disease, and arthritis [7].
Increasing evidence demonstrates the critical role of RAGE signaling in many other human diseases and suggests its stimulating factors as potential biomarkers for diagnosis or therapy.
In this review, we are going to summarize the RAGE signaling pathway and its contribution to the pathogenesis of human diseases, focused specifically on kidney diseases.
Further, we propose the possibility of using RAGE and its ligands as therapeutic targets.
Human RAGE is encoded by a gene located on major histocompatibility complex (MHC) class III region on chromosome 6 [11].
The mature RAGE has three main parts, consisting of extracellular, transmembrane, and cytosolic regions.
The extracellular region is composed of one V-type and two C-type domains, and the V-type domain is responsible for interaction with multiple RAGE ligands.
The transmembrane domain anchors RAGE to the cellular membrane, and signals transduce into the cell via the cytosolic domain [1, 12].
Besides, RAGE variants, represented by three forms, called N-truncated, dominant-negative, and soluble RAGE, can be generated either by natural alternative splicing or by the action of membrane-associated proteases (Fig.1) [1].
The N-truncated form of RAGE lacks a V-domain so that it can not interact with ligands, whereas the cytosolic domain is missing in dominant-negative RAGE, which results in no signal transduction, though it can bind to ligands.
Without the transmembrane domain, soluble RAGE is formed and is able to circulate out of the cell and act as decoys by preventing ligands from binding to RAGE [13].
Therefore, soluble RAGE can neutralize the effect of RAGE ligands, and many studies have suggested that this function might be applied to inhibition of RAGE signals in human diseases [14].
RAGE and its ligands are highly expressed in developmental stages and enhance survival of neurons during development [15].
After embryonic development, these genes are down-regulated and kept a low expression levels during normal life, except for certain conditions, such as injury and aging [15].
Interestingly, unlike other receptors, expression of RAGE is positively regulated by its ligand stimulation, which means that increasing concentration of ligands leads to up-regulation of RAGE.
So, RAGE signals are accelerated more and more by the accumulation of signal stimulants [16].
RAGE is found on the cell surface of various immune cells, and most of its ligands are mainly secreted by immune cells, including macrophages and dendritic cells; therefore one of the major roles of RAGE is involved in inflammation [7, 14].
Stimulation of RAGE by its ligands activates the proinflammatory transcription factor nuclear factor kappa B (NF-κB), and it then translocates into the nucleus and activates its target genes, including several kinds of cytokines responsible for innate or adaptive immunity [17, 18].
Also, NF-κB can bind to glyoxalase (Glo1) and suppress its inhibitory activity on AGE production [19].
Target genes of NF-κB include anti-apoptotic genes, such as Bcl proteins; therefore, cell survival is also under the control of NF-κB [16].
In addition, the mitogen-activated protein kinase signaling pathway itself can be activated by RAGE stimulation.
Specifically, activated RAGE is able to stimulate ERK, p38, and JNK, and it leads to induction of cell proliferation [20].
Another process involved in RAGE signaling is cell migration via transforming growth factor beta (TGF-β) signal cascades.
TGF-β is activated by RAGE ligand stimulation and enhances its downstream signals.
They include the Ras homolog family and Rho-associated kinase, and activation of these genes leads to stable actin filament formation and cell migration [21].
Due to RAGE signals being concerned with multiple cellular signaling pathways, in particular with immune responses and cell survival, the development of numerous human diseases is associated with it.
They include diabetes, inflammatory diseases, cancers, and cardiovascular diseases, and blockade of RAGE is increasingly proposed as a therapeutic target [4].
Expression of RAGE and its ligands is kept at a low level in a wide range of cell types, including endothelium, smooth muscle cells, mononuclear phagocytes, neurons, and cardiac myocytes [22].
However, when RAGE ligands are accumulated by pathological development of relevant diseases, it alters the structure of tissues as well as compromise protein function in them [23, 24].
Also, increased circulating RAGE ligands induce the expression of their receptor, RAGE; therefore, RAGE ligand signals are stimulated as the ligands increase [16].
In kidneys, accumulation of RAGE ligands is well studied to contribute to both diabetic and non-diabetic nephropathy (DN).
Firstly, AGEs have been found primarily to be up-regulated in renal basement membranes in diabetic nephropathies and in some other renal diseases, including glomerulosclerosis and arteriosclerosis, which are not related to diabetes [25].
In diabetes, AGEs are highly accumulated by increasing mitochondrial production of reactive oxygen species, followed by high intracellular glucose levels [26].
Abnormally high concentrations of AGEs impair proximal tubular cells, which are responsible for regulation of AGE metabolism, resulting in damage to those cells and their function [27, 28].
Also, kidneys play key roles in the clearance of AGEs, such that the impaired structure of the kidneys by AGEs fails to degrade them [29].
In turn, more and more AGEs are accumulated as the kidney is damaged by the action of AGEs, and finally, it results in DN [27, 28, 30].
Recently, increasing evidence has shown that RAGE ligands other than AGEs, including HMGB1 and S-100 proteins, are also up-regulated in renal diseases.
HMGB1 is a nuclear protein essential for regulating the physical interaction of DNA with certain transcription factors, such as p53 [29]; however, when it is released extracellularly, it is able to mediate inflammatory reactions and contribute to the pathogenesis of several diseases, including glomerulonephritis, lupus nephritis, and renal cell carcinoma (RCC) [31, 32].
In addition, cell proliferation, migration, and invasion as well as inflammation are also activated in RCC via ERK1/2 activation [32].
Interaction between RAGE and its ligands changes renal cellular processes, including inflammation, proliferation, and migration, and they are highly significant processes to keep the normal function of kidneys; therefore, dysregulation of RAGE or its ligand concentrations might give fatal effects, leading to the development and pathogenesis of renal diseases.
Title: Diabetic nephropathy

DN is one of the diseases triggered by diabetes and finally leads to renal failure and elevation of arterial blood pressure [33].
Cellular mechanisms involved in the development and pathogenesis of DN include glomerular hypertrophy, production of oxidative stresses, fibrosis, and inflammation, followed by hyperglycemia.
As mediators, renin, angiotensin, aldosterone, protein kinases, and NADPH are existing, and hyper-activation of them results in up-regulation of fibrotic and inflammatory factors, including TGF-β [33-36].
Also, hyperglycemia stimulates glycation and oxidation of macromolecules, which result in the formation of AGEs as endproducts [37].
Accumulated AGEs join in the up-regulation of inflammatory factors and oxidative stress and accelerate the pathogenesis of the disease [38].
Title: Cardio-renal syndrome

Cardio-renal syndrome is a combined disorder having both heart and kidney dysfunction.
The primary failed organ can be either the heart or kidney, and in both cases, the one that is diseased earlier impairs the other one, and finally, they negatively affect each other [37].
One of the suggested mechanisms making these two organs negatively interact with each other is mediated by AGEs [39].
Accumulation of AGEs is commonly observed in patients with both heart failure and renal disorders [40, 41].
With heart failure, increase of AGEs leads to inflammation, fibrosis, and delayed calcium uptake, which subsequently result in diastolic dysfunction.
In addition, accumulation of AGEs in renal dysfunction is negatively affected by highly activated inflammation and fibrosis, and further damage of kidney structure impairs its own function of AGE clearance [39].
Symptoms of the disease might be enhanced by this vicious cycle, as well as with other multifactorial signaling pathways involved in the pathogenesis.
Title: Autosomal dominant polycystic kidney disease

Autosomal dominant polycystic kidney disease is one of the most highly prevalent inherited disorders and is characterized by the development of thousands of epithelial-lined, fluid-filled cysts in the kidney [42].
Cell proliferation, inflammation, and fibrosis are the major processes of the development and pathogenesis of the disease, and various factors, including chemokines and growth factors, are engaged in these signaling pathways [43-45].
Recently, several studies have reported that expression of HMGB1 and certain types of S-100 (S-100A8, 9) is increased around the cystic region and proposed them to be disease-stimulating agents [46-48].
So, inflammatory and growth factors secreted by cyst-lined epithelial cells and increased circulating AGEs finally result in compromised renal function at the end stage of the disease.
Title: Renal cell carcinomas

RAGE and its ligands are reported to be abnormally expressed in various cancers, including breast, lung, kidney, thyroid, prostate, and oral cancers.
S100 protein, one of the representative ligands of RAGE, which has at least over 20 subtypes, has been well reviewed to play major roles in tumor-related processes, including proliferation, apoptosis, metastasis, and invasion [10].
In renal cancer, S100A1 has been observed to be highly expressed in papillary RCC and oncocytoma, which are relatively common types of renal cancers, whereas it is absent in chromophobe RCC [49].
Likewise, a broad range of S100 proteins has been detected differently, depending on the subtype of renal carcinoma, which means that it should be a useful biomarker for discriminating the subtype of renal cancers if it is correctly examined [50].
In addition, HMGB1, another ligand of RAGE, has appeared to be up-regulated and interacts with its receptor for promoting cell proliferation, migration, and invasion via the ERK pathway in RCC [32].
From this evidence, RAGE and its multiple ligands are increasingly suggested as biomarkers and therapeutic targets for different types of renal carcinomas.
In various types of renal diseases, including both diabetic- and non-diabetic disorders, RAGE and its ligands are commonly observed to be dysregulated.
The major problem of these phenomena in renal diseases might be the vicious cycle that impairs kidney by increased AGEs or when other RAGE ligands fail to remove them, causing these ligands to accumulate and the relevant signals to accelerate.
Therefore, numerous studies suggest RAGE and its ligands as new potential targets for the diagnosis, monitoring, and treatment of renal diseases that have abnormal regulation of them.
Firstly, ligands expression or secretion level can be used as a diagnostic or staging marker.
S100B is already used in evaluations after treatment of melanoma in a clinical setting [51].
Secondly, more trials targeting RAGE ligands either with inhibitors of them or the soluble form of RAGE have been reported.
Soluble RAGE treatment significantly alleviates the symptoms of nephritis in vivo [52], and in cancer studies, nude mice injected with soluble RAGE-over-expressing cells have shown remarkable inhibition of tumor formation [53].
With inhibitors of AGEs, several agents, including aminoguanidine and pyridoxamine, have been tried in humans; some of them have shown enhanced renal function in diabetic patients, but these trials were terminated early because of safety concerns [54, 55].
Finally, more recently, epigenetic regulation by microRNAs of molecules involved in RAGE signaling pathways is also being studied [56].
Structures of receptor for advanced glycation end products and its three main isoforms.
Full-length receptor for advanced glycation endproducts (RAGE) has three different extracellular domains (V, C, C&#39;) and one cytosolic domain.
Multi-ligands bind to the V-type domain and transduce signals through the intracellular domain.
The N-truncated isoform lacks the V-type domain, which fails to have receptor-ligand interactions.
The dominant-negative form has no cytosolic domain and is not able to transduce signals into the cell.
Soluble RAGE is formed either by alternative splicing or protease activity and is secreted and prevents ligands from binding to RAGE.
V, variable domain; C, constant domain.
Title: Study of Modern Human Evolution via Comparative Analysis with the Neanderthal Genome

Many other human species appeared in evolution in the last 6 million years that have not been able to survive to modern times and are broadly known as archaic humans, as opposed to the extant modern humans.
It has always been considered fascinating to compare the modern human genome with that of archaic humans to identify modern human-specific sequence variants and figure out those that made modern humans different from their predecessors or cousin species.
Neanderthals are the latest humans to become extinct, and many factors made them the best representatives of archaic humans.
Even though a number of comparisons have been made sporadically between Neanderthals and modern humans, mostly following a candidate gene approach, the major breakthrough took place with the sequencing of the Neanderthal genome.
The initial genome-wide comparison, based on the first draft of the Neanderthal genome, has generated some interesting inferences regarding variations in functional elements that are not shared by the two species and the debated admixture question.
However, there are certain other genetic elements that were not included or included at a smaller scale in those studies, and they should be compared comprehensively to better understand the molecular make-up of modern humans and their phenotypic characteristics.
Besides briefly discussing the important outcomes of the comparative analyses made so far between modern humans and Neanderthals, we propose that future comparative studies may include retrotransposons, pseudogenes, and conserved non-coding regions, all of which might have played significant roles during the evolution of modern humans.
Modern humans are indeed a very young species compared to their cousins, evolving just about 200,000 years ago (ya), which is a fraction of the 6 million years since the divergence of the human and chimpanzee lineages [1].
Fossils suggest that modern humans first emerged in East Africa and spread fairly quickly all over the world in the next 185,000 years or so (reviewed in Lu et al.[2]).
After the divergence of humans and chimps, the major landmark in human history is the emergence of bipedals about 4 million years ago (mya), which enabled them to use their two feet as hands.
Many species evolved afterwards until the evolution of Homo erectus, who, for the first time, migrated out of Southern Africa and initiated the spread of humans all around the globe.
The migrated population of Homo erectus in East Africa eventually gave rise to modern humans about 200,000 ya and to Homo neanderthalensis, or Neanderthals, about 400,000 ya [3, 4].
Neanderthals survived until 28,000 ya, while modern humans are still surviving [5].
During the latter part of their existence, Neanderthals lived in Europe, as well as in Western Asia and the Middle East [6, 7].
Various lines of evidence suggest that modern humans started to migrate from East Africa to Europe and other parts of the world 100,000 ya, and the fossil evidence of humans and Neanderthals indicated that these species might have come into contact as early as 80,000 ya and co-habited for up to 10,000 years at certain geographic locations [6].
In the field of evolutionary biology, one of the most sought after questions has been what made modern humans superior than other related species-i.e., the genomic features that are unique to humans.
The whole-genome sequencing of chimps, rhesus macaque, and other primates has given considerable boosts in this field, as the sequences of these primates opened up the possibility to conduct comprehensive comparative studies to the single-nucleotide level [8, 9].
Many attempts have been taken to identify the genetic reasons why modern humans developed such complex biological features than other primates, including the larger brain-to-body ratio, bipedalism, morphological changes, and significant development of communication skills and cognitive behavior.
Recent studies have used various statistical methods to compare the sequence of these primates with humans in an attempt to find human-specific genes and gene regulatory sequences, eventually showing unexpectedly rapid evolution in the human lineage after the divergence from the ancestral primates [10-15].
The results from these analyses exhibit a good overview of the human-specific genomic elements, but these results are unable to distinguish which of these human-specific elements are specific to modern humans only.
Since there has been no complete genome sequence of any archaic humans until recently, such sequence comparisons have been made only between the modern human genome and other primates, bypassing archaic humans, resulting in an overwhelming number of differences and the inability to identify which sequences changes are unique to modern humans and which are shared by all Homo species.
Therefore, the comparative analysis between modern humans and archaic humans is expected to be more interesting and valuable by being more effective in identifying the critical genes and/or regulatory elements that may be fully or partially responsible for the evolution of the modern humans over other humans.
In order to identify such sequence changes, the modern human genome sequence must be compared with that of archaic humans.
Neanderthals have always been a desired target for this purpose for multiple reasons.
They are the closest cousins of modern humans, both anatomically and based on intelligence.
Their slightly larger brain and wider body structure are the primary anatomical differences from the modern humans [16].
Fossil evidence suggests that Neanderthals used stone tools, they were hunter gatherers, and they had a social life, indicating that they had similar intelligence as modern humans until about 50,000 ya, when a &#34;Creative explosion&#34; occurred in modern humans [17, 18].
Another critical reason to target Neanderthals is that they are the latest archaic humans to go extinct, and their remains have been found in sufficiently good condition for analyzing down to the molecular level [19-21].
To make a comprehensive genomic comparison between modern and archaic humans, the whole genome of archaic humans has to be sequenced.
The availability and widespread use of massively parallel high-throughput sequencing have now made it possible to sequence archaic genomes, which seemed impossible even a decade ago.
The idea of sequencing an ancient genome was first implemented on cave bear [22] and, after its success, the mammoth genome [23].
The initial sequencing attempts on Neanderthals included sequencing of the mitochondrial genome, which was successful to a certain extent [24-26].
These successes eventually led paleogeneticists to attempt sequencing Neanderthal nuclear genomes.
Almost all of the attempts have been made by amplifying the genome fragments via PCR and parallel sequencing, while some involve the use of a metagenomic approach.
The early success of such attempts eventually led to the establishment of the Neanderthal Genome Project in 2005, which first announced the complete genome sequence of Neanderthals in 2010 [27] and recently released a cleaner and higher-coverage version (http://www.eva.mpg.de/neandertal/).
Comprehensive comparisons have been made by the group on certain genetic entities with some very interesting inferences, based on the initial low-coverage sequencing data, but improvements in many aspects can be made by utilizing the recent, better version of the sequence and considering more types of genetic variances in the study.
Here, we will discuss the inferences made from the comparison and what can be done next to answer some interesting questions regarding the evolution of modern humans.
Many lines of archaeological evidence indicate that humans and Neanderthals may have coexisted in certain geographic locations.
This gives rise to the most debatable question regarding the recent history of humans: did modern humans and Neanderthals interbreed?
If they did, was it to an extent where meaningful exchange of genetic information may have occurred?
Do we still carry any genetic elements of Neanderthals?
Comparing the genome sequences is probably the best way to answer all these questions.
All other features, including the level of intelligence in Neanderthals, have been speculated from bones, settlements, or artifacts found, and there is no way to be certain about the practicality or validity of the inferences made from these remains.
The hypothesis that Neanderthals were able to practice complex behavior has already been disputed [28-30].
There has also been a significant amount of debate about the admixture of humans and Neanderthals.
Morphological analyses have provided strong arguments both for and against the genetic exchange between these two species [31, 32], as have the comparative analyses of DNA sequences of these species [33-35].
The genome sequence itself can not validate many of these inferences, either, but it can answer the question of admixture and articulate the genetic elements that are unique to Neanderthals or to humans or those that are shared by both.
Studies on the expression or protein functions in association with these unique elements, despite being plausible only for those found in the human genome, can eventually facilitate analysis of complex biological phenomena, such as reasoning, language, or other qualitative or behavioral traits, at the molecular level.
In the past, a candidate gene approach has been successfully implemented to identify the presence or variance of certain genes that were believed to be modern human-specific.
Using this approach, a number of speculations about Neanderthals could be made, including their skin and hair color.
It was also discovered that Neanderthals had the same FOXP2 gene as modern humans, which was previously linked to language ability [36].
This approach of fishing for particular genes proved efficient, but there is a lot more than known genes in the whole genome that may play a critical regulatory role in gene expression and thus development.
For instance, almost half of the human genome consists of transposable elements; these elements can affect gene expression by activating or deactivation functional genetic elements and by altering the protein coding by creating alternative splicing or creating new chimeric genes ([37-39], reviewed in [40, 41]).
Transposable elements are also polymorphic among different populations of modern humans, and their association with phenotypic traits, including diseases, has been extensively studied, albeit with a lot more to be learned [42-44].
The whole-genome sequence is necessary for identifying transposable element insertions that may have taken place only in modern humans and subsequently assessing their functional impact.
Title: Comparative analysis

The only part of the Neanderthal genome that was sequenced completely from multiple specimens until recently was the mitochondrial DNA (mtDNA).
The major analysis that was made with the mtDNA sequence was to seek an answer to the question of interbreeding.
The Neanderthal mtDNA sequence consistently falls outside the spectrum of variations observed in the modern human mtDNA sequence, indicating no interbreeding between the species [24-26, 45, 46].
Previously, besides detecting specific mutations in MCPH1 and FOXP2, the candidate gene approach also detected the presence of fragments of the MC1R gene that may indicate the red hair and pale skin of Neanderthals [47], segments of the ABO blood group locus [48], and a taste receptor gene [49].
Despite many critical technical challenges in sequencing ancient genomes ([26]; reviewed in [50]), the whole genome of Neanderthals has been sequenced recently by Green et al.
[27], and it allowed this research group to perform a genome-wide sequence comparison with the modern human genome sequence, and the significant results from the initial analyses are discussed briefly below.
Title: Substitutions and indels

Green et al.
[27] inferred that 10,535,445 substitutions and 479,863 indels have occurred in the modern human genome after the divergence from chimps.
The vast majority of these occurred before Neanderthals and modern humans diverged.
However, 78 non-synonymous nucleotide substitutions that are fixed for a derived state in modern humans are different in the Neanderthal counterpart, as Neanderthals carry the ancestral state of these polymorphic nucleotides.
Only five genes were identified that have more than one fixed substitution in their coding regions, and one of them has an altered start or stop codon.
It is particularly interesting that three of these genes are expressed in skin, including RPTN, which encodes the protein repetin, and TRPMI, which encodes melastatin.
This might be indicative of a change in selection of skin physiology in the modern human lineage.
When looking into differences in regulatory elements, a total of 132 substitutions and 36 indels were identified in the untranslated regions.
One microRNA of unknown function, hsa-mir-1304, which was identified by parallel sequencing of human embryonic stem cells [51], was found to have one fixed substitution and one single nucleotide insertion.
Since the substitution occurs in the seed region, it is not unlikely that this microRNA has different targets between these two species.
Title: Selective sweeps in modern humans

There are 212 positively selected regions identified in modern humans that occurred early during the history in conjunction with or shortly after their divergence from Neanderthals.
The largest of the positively selected regions contains the gene THADA, single-nucleotide polymorphisms (SNPs) in the vicinity of which have been linked with type II diabetes [52].
This change may have affected the energy metabolism of early modern humans.
A number of other genes that lie on these selectively swept regions have been associated with other genetic disorders, such as autism, schizophrenia, and Down syndrome.
Since both autism and schizophrenia are related to cognitive development, it could be assumed that multiple genes involved in cognitive development in humans were positively selected early in the history of modern humans.
Title: Admixture

Even though previous studies with Neanderthal mtDNA [26, 33] and initial sequencing of nuclear DNA showed no evidence of interbreeding between Neanderthals and humans [21, 53], the most striking revelation after the comparative analyses between the whole-genome sequence of Neanderthals and multiple modern human individual genomic sequences was the demonstration of admixture between these two species.
Green et al.
[27] compared the Neanderthal genome with eight modern-day human genomes of European Americans, East Asians, and West African ancestry.
Surprisingly, the Neanderthal genome appeared more similar to all non-African genomes than to African ones.
They share significantly more derived alleles (alleles that are different from in chimp) with non-African populations than with the African, and when compared with European and Asian individual genomes, Neanderthals are found equally close to both populations.
This and some other analyses made by the group only indicate an exchange of DNA between Neanderthals and the non-African population.
With further comparative analysis, the same group also identified that gene flow occurred unidirectionally from Neanderthals to the modern non-African human population.
A couple of other findings from different experiments, along with a genome-wide comparison, provide strong evidence for exchange of genetic information.
The first finding involves the MAPT locus in chromosome 17, which has two distinct haplotypes, H1 and H2.
H1 is abundant in almost all populations of modern humans, while H2 is found only among Europeans and found to have entered into the Homo lineage approximately only 10,000 to 30,000 ya.
However, a comparison between H1 and H2 in chimp suggests that the common founder of H1 and H2 is far older than 30,000 years.
Even though the Neanderthal genome sequence has been found to carry the H1 haplotype, coinhabitation of the H2 chromosome carriers during the time period when modern humans coexisted with archaic humans can not be ruled out because of the scarcity of archaic genome data.
Thus, one can still argue that the H2 haplotype found in modern humans could possibly be a result of horizontal gene transfer between modern humans and Neanderthals and remained in modern humans under selective pressure, possibly because the H1 haplotype has a role in neurodegenerative diseases [54].
In a similar scenario, haplotype D of the microcephalin gene is found to have originated 1.1 mya in a lineage other than modern humans but integrated into the modern human genome only about 37,000 ya.
It has thus been speculated that this haplotype was horizontally transferred into modern humans from archaic humans, most likely Neanderthals [34].
However, in a more recent study, the microcephalin locus from a Neanderthal individual in Italy was sequenced and found to be homozygous for the ancestral non-D haplotype [55].
The whole-genome study by Green et al.
[27] does not support these speculations, either, since the Neanderthals they analyzed do not carry these D-haplotypes.
One striking revelation from the whole-genome comparison by Green et al.
[27] is the equal level of similarity of the Neanderthal genome with Papuan and Chinese and French, although fossil records show the existence of Neanderthals only in Europe and western Asia.
The group explained this anomaly by arguing that the interbreeding between the species occurred earlier than previously expected, before the divergence of Europeans, East Asians, and Papuans.
Archaeological evidence suggests that modern humans appeared in the Middle East before 100,000 ya, where Neanderthals were already present, and probably remained until 50,000 ya [56]; this makes the prediction by Green et al.
[27] probable.
Title: Increasing coverage, sequencing more Neanderthals, and the Y chromosome

For a more comprehensive comparison of whole-genome sequences between Neanderthals and modern humans, the sequence coverage of Neanderthals has to be increased.
The three approaches made so far to sequence the Neanderthal genome have resulted in the sequencing of only 65,000 bases [21], 1 million bases [19], and finally, the draft genome sequenced recently [27], consisting of only two-thirds of the whole genome with a mere 1.3× coverage.
With such low coverage, it is hard to form meaningful contigs, and a number of important genetic entities will remain unnoticed.
Even though it was beyond imagination to sequence a Pleistocene specimen a decade ago, the progress that has been made in the last 5 years is good enough to expect that more such specimens be sequenced in coming years.
The more specimens from various geographic locations that are sequenced, the more likely it will be to construct a reference genome sequence for Neanderthals.
As the human genome sequence varies considerably among different populations, it is expected that Neanderthals also have variation in their genomic sequences among different populations from different locations.
Such variations can only be identified by sequencing a wide range of specimens, and these variations may again change insights into the Neanderthal-modern human relationship.
The sequencing of Neanderthals first started with its mitochondrial DNA in 1997 [25].
Comparisons between Neanderthal and modern human mitochondrial DNA have been made extensively, but these comparisons only reveal the maternally inherited difference between the species, as mitochondria DNA is transmitted maternally.
The complete genomic sequence of Neanderthals published recently is also from a female specimen.
Thus, the Y-chromosome of Neanderthals or paternal inheritance has yet to be examined.
Comparisons of the Y chromosome sequence of Neanderthals with currently established Y-haplogroups for modern humans should provide some insights into the admixture hypothesis.
With respect to the recent finding of admixture of Neanderthals with non-African populations, the Neanderthal Y chromosome should not match the Y haplogroups A or B, as these haplogroups are the oldest of the clades and almost restricted to Africans and their descendants [57, 58].
Since haplogroup E is found in Africa, the Middle East, Southern Europe, and Asia [59-61], the Neanderthal Y chromosome may match this haplogroup, but it should not match the haplogroups E1b1a*, E2b1, or B2a1a, as they are specifically treated as Bantu expansion markers, while Neanderthals interbred only with non-Africans [62, 63].
Title: Retrotransposon insertion polymorphism

Almost half of the human genome comprises retrotransposons.
Although they were overlooked for a significant period of time in our genetic study, their importance in chromosome structure, gene regulation, and disease predisposition has now been well established.
Retroelements are widely divided into two categories-one with long terminal repeats (LTRs) and another without the LTRs.
Short and long interspersed repeat elements (SINEs and LINEs, respectively) are two of the major classes of non-LTR retroelements, while SINEs are the more abundant class.
Among SINEs, Alu is the predominant type of retroelement.
Among all Alus found in the entire human genome, only about 0.5% is found to be present in the human genome but absent in orthologous regions of other primates and are thus identified as human-specific.
This &#39;young&#39; group of Alus is composed of only about 5,000 Alu elements that are believed to have integrated in the human genome after the divergence of humans and great apes [64-68].
Studying the retrotransposon insertion loci in Neanderthals will identify truly modern human-specific retrotransposon insertion polymorphisms.
A similar comparative analysis would reveal other transposable elements, such as L1, SVAs, and HERVs, that are specific to modern humans only, as well as those that are specific to Neanderthals.
Retroelements are particularly important in population genetics.
It is extremely rare that a newly inserted transposable element is completely excised; thus, they act as a genetic fossil that is homoplasy-free.
This identical-by-descent nature of retroelements makes them better markers for population and evolutionary studies than SNPs, in the sense that SNPs can, though rarely, be mutated back to the previous state.
SNPs are also very hard to detect while handling ancient genomes due to transformation and deamination [69], while retrotransposon insertion polymorphisms (RIPs) refer to the presence or absence of a retrotransposon.
Once a retrotransposon is inserted at a new location in an individual, it is subject to genetic drift.
Over a short period, it starts spreading in to the population.
Depending on when a retroelement has integrated at a certain locus, it will be shared by different species or, if recently enough, by different populations of the same species.
Thus, RIPs occurring before the divergence of chimps and humans are shared by humans and chimps, but those occurring after are present only in humans.
RIPs that are even more recent are specific to certain human populations only [70, 71].
For instance, some RIPs are found only in Africans, some in Han Chinese, and so on.
The detailed information about all polymorphic retroelements and their frequency in different populations is extensively cataloged in the dbRIP database [43].
The identical-by-descent and homoplasy-free nature of RIPs makes them useful genetic markers in population and evolutionary genetics.
The specificity of RIPs can play a significant role in answering the question of admixture of Neanderthals and modern humans.
Finding RIPs that are shared between Neanderthals and non-African populations but not present in African populations can be considered solid support for the proposed admixture between Neanderthals and non-African populations.
In an ongoing study in our laboratory, over 500 RIPs were identified to be present in Khoisan and Bantu individuals, who represent the oldest lineage of modern humans from Southern Africa, but not in the reference human genome (unpublished).
These oldest African lineage-specific RIPs theoretically should also be absent from Neanderthals.
Title: Human-specific unprocessed pseudogenes

Pseudogenization has always been an interesting topic that has not been explored much.
Many pseudogenes have been identified lately that lost their functional capacity in human lineage after the divergence of humans and chimps, particularly related to immunological functions [72].
The &#34;less is more&#34; hypothesis states that gene loss may direct evolutionary changes, as these pseudogenized genes have impacts on adaptation of the species through evolution [73].
Even though pseudogenization does not initiate under selective pressure, the gene loss is retained and subsequently allows adaptation.
Pseudogenes are found particularly important in humans, and some of them have even been identified as being responsible for certain human-specific phenotypes.
For example, the sarcomeric myosin gene was lost at the time of emergence of the genus Homo and is thought to be responsible for marked reduction of hominin masticatory muscles, leading to expansion of brain size [74].
Analyzing pseudogenes in Neanderthals would be beneficial to identify genes that were lost after the divergence of Neanderthals and modern humans and determine what biological impact they may have.
Title: Human accelerated conserved noncoding sequences (HACNSs)

HACNSs are sequences in the human genome that were conserved throughout vertebrate history but changed significantly after the divergence of humans and chimps.
Human genomes have a number of such sequences that obtained surprisingly more mutations after the emergence of humans than before.
These regions are typically rich in cis-regulatory transcriptional enhancers that render specific expression pattern of genes involved during development [75-77].
Identification of these cis-regulatory elements in human or other large genomes is done mostly by cross-species sequence comparison, primarily because the functional cis-regulatory elements are generally unique to the genome, which makes paralogy-based identification of such sequences nearly impossible [78].
Since it has long been proposed that phenotypic variation between humans and chimps is mostly brought about by regulatory elements than coding sequences [79], it would be particularly interesting to make an elaborated comparison between differences in conserved regions in modern humans and Neanderthals.
Such a comparison between the human genome and initial Neanderthal draft genome sequence [27], involving a total of 2,613 human accelerated regions, revealed that the Neanderthal sequence carried 3,259 human-specific changes in these regions.
The comparison revealed that 51 positions in 45 regions were different between these two species; Neanderthals carried the ancestral form while all modern humans carry the human-specific variant.
In a recent study, it has been found that the Neanderthal genome retains the ancestral state of a polymorphic site in a conserved noncoding microRNA, which is involved in regulating two genes that are important for teeth formation [80].
This may explain the dental differences between modern humans and Neanderthals.
These findings are interesting enough to initiate further studies to analyze the probable impact of the variations in conserved noncoding sequences (CNSs) that exist between these two species.
Furthermore, previously in three different studies, the number of HACNSs was found to be between 202 and 1175 [10, 11, 15].
All these studies used different methods to identify CNSs and included more species during the comparison.
Their data can also be included in the future for the comparison of CNSs between modern humans and Neanderthals.
The most rapidly evolving HACNS identified so far, named HACNS1, has accumulated 16 human-specific changes out of its 546 bases since the divergence of humans and chimps [12].
HACNS1 functions in multiple structures early in the developing stages of the mouse embryo as a transcriptional enhancer, including the developing anterior limb [12].
But, this function is missing for the orthologous enhancers in chimpanzee and rhesus, suggesting that HACNS1 has a different function in humans.
When the chimpanzee enhancers are humanized by introducing 13 of the 16 human-specific substitutions, gene expression was observed in the limb, indicating that the substitutions that were identified by comparative analysis were directly responsible for the functional modification.
In a recent study, 16 human-specific mutations in HACNS1 were also found in Neanderthals, suggesting that the phenotypic function that is related to this region was also expressed in Neanderthals [81].
The comparison can be extended to other human accelerated CNS regions to identify any substitutions that occurred after the divergence of humans from Neanderthals.
Conserved regions have also been presumed to play important roles in developing certain functions in humans-for instance, noncoding regions of 150 presynaptic genes in humans are highly conserved and may have critical regulatory roles in the expression of these genes [82].
Comparative analyses of these regions between different populations of modern humans and Neanderthals may give directions to a better understanding of neurodevelopmental and psychiatric disorders.
Involvement of noncoding sequences in the developing brain was also revealed by another experiment involving 49 human accelerated regions (HARs); the most accelerated region in the human genome, HAR1, is part of a noncoding RNA sequence expressed in the developing brain [83].
A study by Burbano et al.
[83] identified that 8.3% of HAR substitutions are not shared between modern humans and Neanderthals, but the study did not focus on HAR1.
A complete Neanderthal genomic sequence should provide an understanding of the evolutionary origin of these regulatory RNAs.
Despite many technical challenges, the whole-genome sequence for a Homo species other than modern human is now there for the first time in just five years of the establishment of the project.
It is no longer a question of the possibility of sequencing more genomes from archaic remains, and more such achievements will only strengthen our understanding of human evolution.
All of our knowledge on human species-specific genetic elements thus far is based on comparison with non-human primates, which gives us information only on how humans are different from other primates, such as chimpanzee-not how modern humans excelled from archaic humans.
Modern humans are considerably superior to their predecessors, and the availability of high-quality Neanderthal sequences can shed some light on the genetic basis of this phenomenon.
An initial comparison between the genome sequences of these two species has given us some valuable insight, and more studies are being conducted to identify further variations between Neanderthal and humans.
However, a lot remains to be done, which has been discussed in this article, albeit briefly.
Even though the sequence comparison itself can not provide much confidence on the biological differences between the Homo species, the application of reverse genetics can be initiated from this point to speculate and validate the possible biological effects of such sequence differences.
Once the molecular functions of such differences are identified in an in vitro analysis, they can be administered into mouse models to observe the phenotypic results of these changes.
One such experiment has already been implemented by administering a human version of FOXP2 in the mouse genome to observe its effects [84].
Many behavioral and qualitative traits of humans can be understood at the molecular level using similar approaches.
There has also been hype by the media over resurrecting Neanderthals by modifying the chimpanzee genome to be more Neanderthal-like [85].
But, it is still an impossible task to regenerate a species just from the genomic sequence, even if the ethical issues can be resolved [86].
However, Neanderthals do not have to be reincarnated to provide us with an enormous opportunity in the field of human genetics and evolution, as their genome is already offering a lot to better our understanding of ourselves.
Title: Somatic Mutaome Profile in Human Cancer Tissues

Somatic mutation is a major cause of cancer progression and varied responses of tumors against anticancer agents.
Thus, we must obtain and characterize genome-wide mutational profiles in individual cancer subtypes.
The Cancer Genome Atlas database includes large amounts of sequencing and omics data generated from diverse human cancer tissues.
In the present study, we integrated and analyzed the exome sequencing data from ~3,000 tissue samples and summarized the major mutant genes in each of the diverse cancer subtypes and stages.
Mutations were observed in most human genes (~23,000 genes) with low frequency from an analysis of 11 major cancer subtypes.
The majority of tissue samples harbored 20-80 different mutant genes, on average.
Lung cancer samples showed a greater number of mutations in diverse genes than other cancer subtypes.
Only a few genes were mutated with over 5% frequency in tissue samples.
Interestingly, mutation frequency was generally similar between non-metastatic and metastastic samples in most cancer subtypes.
Among the 12 major mutations, the TP53, USH2A, TTN, and MUC16 genes were found to be frequent in most cancer types, while BRAF, FRG1B, PBRM1, and VHL showed lineage-specific mutation patterns.
The present study provides a useful resource to understand the broad spectrum of mutation frequencies in various cancer types.
Recent progress in high-throughput sequencing technology has contributed to the generation of genome-wide somatic mutation profiles in diverse cancer samples.
The Cancer Genome Atlas (TCGA) is one of largest collaborative efforts to generate multi-level omics data on human cancer tissue samples.
Particularly, information on genome-wide somatic mutations has been collectively profiled from exome sequencing data from thousands of patients&#39; tumor samples.
Somatic mutation is a main driving force for cancer development and progression.
Thus, many researchers have tried to complete the catalog of somatic mutations in cancer cell lines [1, 2].
Somatic mutation is also known to be involved in key mechanisms for cellular sensitivity or resistance against chemotherapy [3-5].
In our previous study using cancer cell line data, we reported that somatic mutation was a more significant classifier than cancer lineage in predicting the anticancer drug response [6].
Thus, we identified many unknown association patterns between cancer drug response and mutational genotypes in cancer cell lines-e.g., MYC-amp mutation-specific sensitivity of insulin-like growth factor 1 receptor inhibitors.
In addition, mutation information provided important clues for us to better interpret the biological relevance of molecular signatures identified from the transcriptome and proteome data of diverse cancer cell lines.
The next step should be to find out the clinical application of mutation-specific drug responses or molecular signatures obtained from cell line-based analysis.
Thus, it is important to systematically analyze the mutational genotype (mutaome) of various human tissue samples and identify mutations significantly associated with specific types of tumors.
&#39;Mutaome&#39; means the cancer mutational landscape, including mutations in oncogenes and tumor suppressors.
In the present study, we organized all sequence-based mutation information into gene-based frequency data.
Then, we comparatively determined the major genes of somatic mutations in diverse cancer subtypes and cancer stages (i.e., non-metastatic and metastatic samples).
This work will provide practical information for directing in vitro cell line-based mutation-specific phenotypes to clinical applications in cancer drug discovery and mechanism studies.
Title: Data acquisition

Somatic mutation data for tumor tissue samples from 2938 patients, harboring 11 cancer types, were obtained from the data portal of TCGA, which were freely available.
These data (level 2) for 10 cancer types, except ovarian serous cy_stadenocarcinoma (OV), provide genome-wide somatic mutations on each sample experimented with the Illumina Genome Analyzer DNA Sequencing platform (Illumina, San Diego, CA, USA).
The somatic mutation data for OV were organized, combined with data produced from Illumina and the ABI SOLiD DNA System Sequencing platforms (Applied Biosystems, Foster City, CA, USA).
The details, including the full name of each cancer type and updated dates of mutation data, are provided in Table 1.
Together with somatic mutation data, clinical data for each patient were downloaded from the data portal of TCGA.
These data were applied to categorize samples into metastasis and non-metastasis.
The annotation of &#39;pathologic_M&#39; was available to indicate the stage of metastasis in the patient&#39;s tumor samples.
The stage of M0&#39; means that there was no evidence of distant metastasis, and &#39;M1&#39; means that a pathological distant metastasis was found.
In this study, the samples from patients annotated as &#39;M0&#39; were classified as non-metastatic, and those with &#39;M1&#39; were classified as metastatic.
Title: Analysis of somatic mutations

The number of detected mutations in each cancer type ranged from tens to hundreds of thousands.
We organized the detected point mutations into 23,050 human genes.
The number of mutant genes per sample was counted for ~3,000 samples.
In addition, the mutation frequency and its percentage were calculated for all samples and each cancer type.
The major mutant genes that ranked within the top 3 in each cancer type were selected based on the observed frequency in the overall, non-metastatic, and metastatic samples.
The patterns of frequency for the selected major genes were analyzed through hierarchical clustering method.
The clustering and its visualization on a heatmap were performed using the software QCanvas [7].
QCanvas can be down-loaded freely from the website http://compbio.sookmyung.ac.kr/~qcanvas.
Title: Mutation frequency in patients&#39; tumor samples

From the TCGA database, thousands of patients&#39; tumor samples were analyzed to detect variants in the whole genome.
To quantify the genome-wide mutation profile in diverse tumors, we organized the whole mutant genes into several cancer types and stages, using the annotation information obtained from TCGA.
Overall, a total of 871,684 mutations were virtually found in 23,050 human genes from 2,938 patient tumor samples (Table 1).
Samples covered 11 diverse cancer types (Table 1).
These data were continuously updated and produced by analyzing additional samples.
Together with the extended production of multi-level omics data using the same patients&#39; tissue samples, TCGA provides a useful resource for understanding the role of mutations in cancer progression.
The distribution of mutant genes on each sample showed that a variety of genes are mutated in individual tumor samples.
Most samples contained on average 20-80 mutations (Fig.1A).
This means that each single tumor sample has mutations in multiple genes.
Various mutations in a cancer sample were already well-characterized and constructed as open source data [8], and the significance of multiple mutations in a single tumor has been constantly suggested [9, 10].
It is appropriate that a tumor may also consist of a heterogeneous collection of cells with different types of mutations.
Furthermore, in the aspect of lineage dependency, the amount of mutations in individual samples varied, depending on cancer type (Fig.1C).
Lung adenocarcinoma samples have a wide range of mutation frequencies in individual samples and harbor more mutations than other lineages.
The broad range of mutation frequencies in lung cancer was also referred to in Lawrence et al.[11].
In contrast, thyroid carcinoma has relatively few mutations in individual samples.
Further studies are required to understand the association with the amounts of mutations and major biological factors in each cancer type.
This can be analyzed by comparing with the patient&#39;s clinical information, including smoking.
On the other hand, the mutation of each gene was observed at a very low frequency in all samples (Fig.1B).
Most genes contained mutations in less than 1% of samples, and only a few genes contained mutations in over 5% of all samples.
Generally, the mutation of a gene showed low frequency in all cancer types, except in uterine corpus endometrioid carcinoma (UCEC) (Fig.1D).
Mutant genes were found in 2.4% of UCEC samples (&gt;2-fold greater than other lineages), and sometimes, a gene showed a mutation in &gt;8% of UCEC tumors.
In conclusion, there are only a few genes in which mutations are frequently (i.e., &gt;1-2%) found in tumor.
Genes with relatively frequent mutations in tumors may have a significant role in cancer progression.
Title: Comparative analysis of mutations between non-metastatic and metastatic samples

TCGA provides annotations for the stage of metastasis in each patient sample from the clinical data.
According to these data, thousands of samples for 8 cancer types, except for OV, glioblastoma multiforme (GBM), and UCEC, were classified into 89 metastatic and 1341 non-metastatic samples in order to compare the mutation frequency between them.
The annotation for metastasis was not provided for the excluded cancer types-OV, GBM, and UCEC.
Interestingly, metastatic samples had similar mutation frequencies as non-metastatic samples for most cancer types (Fig.2).
Kidney renal clear cell carcinoma (KIRC) had significant (p &lt; 0.01) difference in frequency between metastatic and non-metastatic samples.
This result implied that there is no differential occurrence of mutations between metastatic and non-metastatic samples, except in minor case.
Title: Identification of major mutant genes

In this study, the mutation frequency was analyzed separately in overall, non-metastatic, and metastatic samples.
We focused on mutant genes exhibiting high frequency in each sample group.
A total of 12, 10, and 15 genes were ranked within the top 3 mutations in at least one of cancer type in the overall, non-metastatic, and metastatic samples, respectively (Table 2).
Especially, TP53 and TTN showed dominant frequencies for over 1,000 of all samples.
In addition, the pattern of mutation frequency for the selected major mutant genes was analyzed in diverse cancer categories (Fig.3).
Regardless of sample group, MUC16, TTN, and TP53 were found to be frequent in most cancer types.
TP53 is a well-known mutant gene, playing an important role in cancer progression [12, 13].
It was reported that TP53 mutation is frequently represented in major cancer lineages [6].
Mutation TTN and MUC16 has not been reported to be critical in cancers.
This analysis shows that they may have potential, specific roles in cancer development or progression.
Among 12 major mutant genes selected from the overall samples, BRAF, FRG1B, PBRM1, and VHL had cancer subtype-specific mutation patterns (Fig.3A).
Especially, PBRM1 and VHL showed strong specificity in KIRC tumors.
As previously reported [14], the alterations of VHL (a tumor suppressor gene) are clearly dominant in renal cell carcinoma.
Together with VHL, PBRM1 was identified as a major gene, frequently mutated in renal carcinoma [15].
An association with the loss of its expression and renal cell carcinoma progression was suggested in previous studies [16].
Title: Mutant genes dependent on metastasis

We found that there was no difference in the overall frequency of metastatic and non-metastatic samples (Fig.2).
The nine mutant genes found in non-metastatic samples were all included in the major mutant genes in the overall samples (Fig.3B).
The lineage-dependent frequency in non-metastatic samples was also similar with the overall pattern in Fig.3A.
However, half of the 15 major mutant genes from metastatic samples were different from the mutant genes in non-metastatic or overall samples (Fig.3C).
The mutations of FAM182B, LOC653544, PLEC, STAG2, PTPRT, CSMD3, and LRP1B represented unique frequencies in metastatic samples.
Especially, PTPRT is a member of the protein tyrosine phosphatase (PTP) family.
The deletion of PTPRD, included in the same PTP family, is frequently seen in metastatic cutaneous squamous cell carcinoma [17].
Further studies are required for other major metastasis-associated mutant genes.
In conclusion, the diversity of major mutant genes in metastatic samples is quite different from those in non-metastatic tumors, although the overall mutation frequency is similar between metastatic and non-metastatic tumors.
The present study provides a useful resource for understanding the varied frequency of diverse mutations in patients&#39; tumor samples.
Overall mutation frequency in tissue samples.
(A) The distribution of 2,938 patient tissue samples for the number of mutant genes per sample.
(B) The distribution of 23,050 mutant genes for % mutation frequency.
(C) Box plot of the number of mutant genes per sample for each caner subtype.
(D) Box plot of % sample frequency of individual mutation for each caner subtype.
Number in the box represents median value.
LUAD, lung adenocarcinoma; LUSC, lung squamous cell carcinoma; OV, ovarian serous cy_stadenocarcinoma; THCA, thyroid carcinoma; HNSC, head and neck squamous cell carcinoma; GBM, glioblastoma multiforme; UCEC, uterine corpus endometrioid carcinoma; KIRC, kidney renal clear cell carcinoma; COAD, colon adenocarcinoma; BLCA, bladder urothelial carcinoma; READ, rectum adenocarcinoma.
Comparison of mutation frequency between metastatic and non-metastatic cancer samples.
Three cancer types-OV, GBM, and UCEC-were excluded, because the information on metastasis was not provided.
The lines on the bar represent the standard deviation for the number of mutant genes per sample.
The numbers on the dotted lines represent the value of the standard deviation in each sample.
The significant difference between metastatic and nonmetastatic samples was considered based on the probability of t-test.
**p &lt; 0.01 and *p &lt; 0.1.
The numbers under the x-axis show the number of metastatic and non-metastatic samples in each cancer type.
LUAD, lung adenocarcinoma; LUSC, lung squamous cell carcinoma; OV, ovarian serous cy_stadenocarcinoma; THCA, thyroid carcinoma; HNSC, head and neck squamous cell carcinoma; KIRC, kidney renal clear cell carcinoma; COAD, colon adenocarcinoma; BLCA, bladder urothelial carcinoma; READ, rectum adenocarcinoma.
Lineage-dependent frequency of major mutant genes.
(A) Heatmap of the 12 major mutant genes in overall samples.
Twelve mutant genes are ranked in the top 3 in at least one cancer type based on the % frequency in the overall samples.
(B) Heatmap of the 17 major mutant genes in non-metastatic samples, ranked in the top 3 in at least one cancer type based on the % frequency in metastatic and non-metastatic samples.
Two genes colored in blue are derived from non-metastatic samples, and 7 genes colored in red are derived from metastatic samples.
Others colored in black are commonly selected in metastatic and non-metastatic samples.
(C) Heatmap of the 17 major mutant genes in metastatic samples.
The list of genes is the same as in Fig.3B.
Red represents high % frequency and yellow represents low % frequency.
In the heatmap for metastatic and non-metastatic samples, three cancer types-OV, GBM, and UCEC-were excluded, because the information on metastasis was not provided.
THCA, thyroid carcinoma; OV, ovarian serous cy_stadenocarcinoma; LUAD, lung adenocarcinoma; LUSC, lung squamous cell carcinoma; HNSC, head and neck squamous cell carcinoma; BLCA, bladder urothelial carcinoma; COAD, colon adenocarcinoma; READ, rectum adenocarcinoma; GBM, glioblastoma multiforme; UCEC, uterine corpus endometrioid carcinoma; KIRC, kidney renal clear cell carcinoma.
Title: Survey of genome-wide somatic mutations in patient tumor samples

Title: List of mutant genes showing high mutation frequency in each cancer lineage
Title: Characterization of H460R, a Radioresistant Human Lung Cancer Cell Line, and Involvement of Syntrophin Beta 2 (SNTB2) in Radioresistance

A radioresistant cell line was established by fractionated ionizing radiation (IR) and assessed by a clonogenic assay, flow cytometry, and Western blot analysis, as well as zymography and a wound healing assay.
Microarray was performed to profile global expression and to search for differentially expressed genes (DEGs) in response to IR.
H460R cells demonstrated increased cell scattering and acidic vesicular organelles compared with parental cells.
Concomitantly, H460R cells showed characteristics of increased migration and matrix metalloproteinase activity.
In addition, H460R cells were resistant to IR, exhibiting reduced expression levels of ionizing responsive proteins (p-p53 and γ-H2AX); apoptosis-related molecules, such as cleaved poly(ADP ribose) polymerase; and endoplasmic reticulum stress-related molecules, such as glucose-regulated protein (GRP78) and C/EBP-homologous protein compared with parental cells, whereas the expression of anti-apoptotic X-linked inhibitor of apoptosis protein was increased.
Among DEGs, syntrophin beta 2 (SNTB2) significantly increased in H460R cells in response to IR.
Knockdown of SNTB2 by siRNA was more sensitive than the control after IR exposure in H460, H460R, and H1299 cells.
Our study suggests that H460R cells have differential properties, including cell morphology, potential for metastasis, and resistance to IR, compared with parental cells.
In addition, SNTB2 may play an important role in radioresistance.
H460R cells could be helpful in in vitro systems for elucidating the molecular mechanisms of and discovering drugs to overcome radioresistance in lung cancer therapy.
Lung cancer, the most common cause of cancer-related death in men and women, is responsible for 1.38 million deaths annually, as of 2008 [1].
The main types of lung cancer are small-cell lung cancer (approximately 19%) and non-small-cell lung cancer (NSCLC; around 80%).
Although NSCLC is sometimes treated with surgery, advanced and metastatic NSCLC usually responds better to chemotherapy and radiotherapy (RT) [2].
RT, either alone or in combination with surgery or chemotherapy, plays a major part in NSCLC management.
Tumor radioresistance may greatly reduce the efficiency of RT for NSCLC [3].
RT of a primary tumor accelerates metastatic growth in mice [4].
These invasive and metastatic mechanisms are associated with the recruitment of complex and numerous classes of proteins, including cadherins.
Specifically, matrix metalloproteinases (MMPs) play a key role in the degradation of extracellular matrix (ECM) proteins.
Augmented MMP expression and its activity are associated with advanced tumor stage and poor survival.
Elevated plasma levels of MMP-9 have been found in lung, breast, and liver cancers during RT [5, 6].
In addition, MMP-9 from sublethally irradiated tumors promotes Lewis lung carcinoma cell invasiveness and pulmonary metastasis [7].
Recently, MMP inhibitors were shown to block radiation-induced invasiveness of human pancreatic cancer cells [8].
In addition, expression of β-catenin plays a pivotal role in maintaining the integrity of cellular functions [9].
Ionizing radiation (IR)-induced double-strand DNA breaks ead to activation of ataxia telangiectasia mutated kinase (ATM), which responds through autophosphorylation and activation of DNA repair pathways, including phosphorylation of histone H2AX (γ-H2AX), a signal for recruitment of molecular DNA repair complexes.
In addition to a role in DNA damage repair, H2AX is required for DNA fragmentation during apoptosis and is phosphorylated by various kinases in response to apoptotic signals.
ATM stimulates IR-induced cycle arrest through activation of checkpoint kinases; p53; and cyclin-dependent kinase inhibitors, such as p21 [10].
Moreover, poly(ADP ribose) polymerase (PARP), which is involved in DNA repair in response to environmental stress [11], helps cells maintain their viability, and cleavage of PARP serves as a marker of cells undergoing apoptosis [12].
IR also induces endoplasmic reticulum (ER) stress via upregulation of GRP78 and GRP94 in rat intestinal epithelial cells [13].
The level of C/EBP-homologous protein (CHOP) expression is elevated during ER stress, and CHOP-deficient cells have enhanced survival when challenged with toxins that induce ER stress, suggesting that CHOP functions to mediate programmed cell death [14].
Syntrophins are intracellular peripheral membrane proteins of 58-60 kD, originally identified as proteins enriched at the postsynaptic apparatus [15].
The three syntrophin isoforms, α1, β1, and β2, are encoded by different genes but have similar domain organizations [16].
Syntrophins consist of a C-terminal syntrophin unique domain and an N-terminal pleckstrin homology domain, split by insertion of a PDZ domain.
Although syntrophins have been recognized as an important component of many signaling events, such as insulin secretion, synapse signaling, and migration regulation [17-21], little is known about the involvement of syntrophin beta 2 (SNTB2) in radioresistance.
In the present study, we established radioresistant NSCLC H460R cells.
We tried to analyze these cells in response to IR to identify SNTB2 among upregulated genes in radioresistant H460R cells and to provide new insights into the involvement of SNTB2 in radioresistance.
Title: Cell culture

Human NSCLC cell lines (ATCC, Bethesda, MD) were grown in Dulbecco&#39;s modified Eagle&#39;s medium (Welgene, Daegu, Korea), supplemented with 10% fetal bovine serum (FBS; Invitrogen, Carlsbad, CA, USA), 100 U/mL penicillin, and 100 µg/mL streptomycin (Invitrogen), at 37℃ under an atmosphere containing air and 5% CO2.
Cell morphology was observed under an inverted light microscope.
Title: Establishment of radiation-resistant H460R cells

H460 cells were seeded in 100-mm culture dishes and irradiated with 2 Gy γ-rays using a Gammacell 3000 (Atomic Energy of Canada Ltd., Chalk River, ON, Canada) at a dose rate of 3.2 Gy/min when cells approached 60% confluence.
When the cells reached about 90% confluence, they were subcultured into new culture dishes.
The cells were treated again with 2 Gy γ-rays at about 60% confluence.
These procedures were repeated over a period of 24 months.
The parental cells were cultured under the same conditions without ionizing irradiation.
Title: Acridine orange staining

Acidic vesicular organelles were quantified by vital staining with Acridine Orange (Sigma, St. Louis, MO, USA).
Both the cytoplasm and nucleolus fluoresced bright green and dim red, whereas acidic compartments fluoresced bright red.
The intensity of red fluorescence is proportional to the degree of acidity and/or the volume of the cellular acidic compartment [22].
Cells were stained with acridine orange and analyzed by flow cytometry and Cell Quest software (Becton Dickinson, Franklin Lakes, NJ, USA).
Title: MTT assay and Trypan Blue staining

For the MTT assay, cells were seeded at a density of 1,000 cells per well on 96-well plates, and 3-(4,5-dimethyl-2-thiazolyl)-2,5-diphenyl-2H-tetrazolium bromide reagent (Sigma) was added to each well.
After incubation for 4 h at 37℃ in an incubator under 5% CO2, dimethyl sulfoxide was added, and absorption was measured at a wavelength of 595 nm.
Viable cells were identified based on their ability to exclude the dye via Trypan Blue (Invitrogen) staining.
Title: Cell cycle analysis and reactive oxygen species (ROS) measurement

Cells were fixed with 75% cold ethanol and stained with a solution containing propidium iodide and RNase A (Sigma).
For ROS measurement, cells were stained with 5-(and-6)-carboxy-2&#39;,7&#39;-dichlorodihydrofluorescein diacetate (DCFDA; Molecular Probes, Eugene, OR, USA), and the fluorescence of 10,000 cells was measured by flow cytometry.
Title: Clonogenic assay

Colonies were fixed in methanol and stained with 1% crystal violet.
Colonies 1.0 mm in diameter were counted, and average colony numbers were multiplied by the dilution factor of the cell number.
Each value at every dose was divided by the control value for the survival fraction.
Title: Gelatin zymography

Conditioned media were separated with the use of 10% sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) containing 0.2% gelatin.
The gel was renatured with 2.5% Triton X-100 buffer for 1 h and incubated with developing solution (50 mM Tris, pH 7.5, 10 mM CaCl2) for 18-20 h at 37℃.
The gel was then stained with 0.5% Coomassie Brilliant Blue (Sigma) in 30% methanol and 10% glacial acetic acid and destained with the same solution without dye.
Title: Wound healing assay

Cells were seeded on a 24-well plate, and a wound was generated using a 200-µL tip; medium containing 10% FBS was added.
Wound healing was observed under a microscope after 0, 16, 24, and 48 h. Images were captured and analyzed using ImageJ software (National Institutes of Health, Bethesda, MD, USA).
Title: Western blot analysis

Cells were dissolved in RIPA buffer (50 mM Tris, 0.1% SDS, 1% sodium deoxycholate, 1% Triton X-100, 2 mM ethylenediaminetetraacetic acid, 120 mM NaCl) containing phosphatase and protease inhibitors.
After quantification with Bradford assay reagent (Bio-Rad, Hercules, CA, USA), proteins were separated by 8% SDS-PAGE, transferred to a polyvinylidene difluoride membrane (Millipore, Billerica, MA, USA), and probed with specific antibodies overnight at 4℃.
Membranes were developed using a peroxidase-conjugated secondary antibody and visualized by chemiluminescence with enhanced chemiluminescence substrate (Amersham Biosciences, Piscataway, NJ, USA).
Title: Microarray analysis

Parental and radioresistant cells were irradiated with 4 Gy γ-rays.
Cells were harvested after 48 h, and microarray was performed using Affymetrix GeneChip human genome U133 plus 2.0 arrays (ITSTech, Seoul, Korea).
Data were normalized via global scaling (GenPlex), and genes over the cutoff of 2 were selected based on values of log2 (fold-change): upregulated genes with &#39;p &lt; 0.05&#39; for detection value were selected in the H460R array, and downregulated genes with &#39;p &lt; 0.05&#39; for detection value were selected in the H460 array (ITSTech and Macrogen, Seoul, Korea).
Analysis of significant genes was performed by Welch&#39;s t-test.
Differentially expressed genes (DEGs) were analyzed between parental and radioresistant cells.
Fold-change values in descending order of 200 genes were selected from upregulated genes and downregulated genes.
False discovery rate and Bonferroni-corrected p-values were calculated (Macrogen).
Title: SNTB2 knockdown by siRNA

Cells were plated in 6-well plates at a density of 2 × 105 cells per well.
After overnight culture, each well was transfected with 20-40 nM siRNA using Lipofectamine 2000 transfection reagent (Invitrogen).
After 24 h of transfection, the cells were treated with or without IR.
The SNTB2 siRNA and negative control siRNA were as follows: SNTB2 (sense: 5&#39;-GUCUCUCUAGGCUGCAUGUdTdT-3&#39;, antisense: 5&#39;-ACAUGCAGCCUAGAGAGACdTdT-3&#39;) and negative control (sense: 5&#39;-GAUCAUACGUGCGAUCAGAdTdT-3&#39; and antisense: 5&#39;-UCUGAUCGCACGUAUGAUCdTdT-3&#39;).
Title: Statistical analyses

Three independent experiments were performed, and the results are presented as the mean ± standard deviation.
Statistical analyses of experimental data were determined using two-sided student&#39;s t-test.
p-values &lt; 0.05 were considered significant.
Title: Establishment and cellular morphology of H460R cells

Radioresistance means the relative resistance of cells, tissues, organs, or organisms to the injurious effects of IR.
During the treatment period, recovery from radiation damage between fractions has been shown to increase tumor resistance against fractionated RT [3].
Radioresistance in vitro may be induced by exposure to small doses of IR [23].
To generate radiation-resistant cells, H460 cells were seeded in 100-mm culture dishes and irradiated with 2 Gy γ-rays at a dose rate of 3.2 Gy/min when cells approached 60% confluence.
When the cells reached about 90% confluence, they were subcultured into new culture dishes.
The cells were treated again with 2 Gy γ-rays at about 60% confluence.
These procedures were repeated over a period of 24 mo.
The parental cells were cultured under the same conditions without ionizing irradiation.
These cells were designated H460R and utilized in this study.
The parental H460 cells typically appeared cobblestone-like and showed tight cell-cell junctions, which is characteristic of the epithelial phenotype.
In contrast, H460R cells exhibited a spindle-like morphology and demonstrated a loss of cell-to-cell contact and an increase in cell scattering, as well as acidic vesicular organelles (Figs.1A and 2A).
Since a little difference in proliferative potential, such as cell cycle, has been reported between parental H460 and radioresistant H460R cells [23], we also examined whether established H460R cells are different from the parental H460 cells in cell proliferation, cell cycle, and ROS levels under physiologically normal conditions.
Consistent with Lee et al.
&#39;s findings [23], no significant differences in cell proliferation, cell cycle, or ROS levels were observed between H460 and H460R cells under physiologically normal conditions (Fig.1B, 1C, and 1D).
Title: Increased migration and MMP2 and MMP9 activity in H460R cells

Since spindle-like morphology and an increase in cell scattering were found in H460R cells (Fig.1A), a small number of cells were seeded and cultured for 8 days; colonies were stained with crystal violet.
We observed scattered morphology of colonies in radioresistant H460R cells compared with parental cells.
To test the possibility of increased metastasis in H460R cells, we measured migration and representative MMP activities, including MMP2 and MMP9, using a wound healing assay (Fig.2B and 2C) and gelatin zymography (Fig.2D), respectively.
We confirmed significantly increased migration as well as MMP2 and MMP9 activities in H460R cells.
Since E-cadherin and N-cadherin are known as important marker proteins for the epithelial-to-mesenchymal transition, expression levels of these proteins were determined via Western blot analysis.
E-Cadherin was not detected (data not shown), and no difference was observed in the expression of N-cadherin protein between H460 and H460R cells.
However, β-catenin protein levels increased in radioresistant H460R cells (Fig.2D).
Title: Differential response of H460R cells to γ-irradiation

It has been known that radioresistant cells have some characteristics, such as DNA repair and anti-apoptotic properties, in response to IR as well as other anticancer drugs [23].
Next, we checked the response to IR between parental H460 cells and resistant H460R cells.
A colony formation assay indicated that H460R cells were more resistant to cell death than parental cells after 4 and 6 Gy irradiation (Fig.3A and 3B).
In addition, cell cycle analysis showed that H460 cells were subjected to early apoptosis, whereas H460R cells were arrested at the G2/M phase.
Subsequently, the subG1 fraction in H460 cells was increased 2-fold over H460R cells after 24 h with IR (Fig.3C).
From these results, we considered H460R cells to be radioresistant.
To evaluate the cellular response to IR, levels of proteins with respect to DNA damage and cell cycle were analyzed.
IR led to enhanced phosphorylation of p53 in the parental cells compared with H460R cells.
While γ-H2AX expression appeared after 8 h and peaked 32 h after IR in parental cells, it was only slightly induced in radioresistant H460R cells.
Moreover, p21, p27, and cyclin B1 remained increased in response to IR in H460R cells.
We also observed that the cleaved form of PARP increased in H460 cells after 8 h and peaked after 32 h, whereas a slight increase of cleaved PARP was observed in H460R cells (Fig.3D).
X-linked inhibitor of apoptosis protein (XIAP) expression levels increased in H460R cells but decreased in parental cells after IR exposure.
To test whether IR induced upregulation of ER stress in the two cell lines, we performed Western blot analysis for representative marker proteins, such as glucose-regulated protein 78 (GRP78), glucose-regulated protein 94 (GRP94), and CHOP.
In parental cells, GRP78 and CHOP increased in a time-dependent manner in response to IR up to 24 h and decreased after 32 h of IR, while expression of GRP94 decreased in a time-dependent manner following IR.
Moreover, IR decreased GRP78 but increased GRP94 and CHOP slightly, which were rarely detected, regardless of IR in radioresistant cells (Fig.3E).
Title: Global expression analysis and involvement of SNTB2 in radioresistance

A cDNA microarray was used to analyze DEGs in H460R cells in response to IR.
Four hundred genes that were either up-regulated or down-regulated, demonstrating more than 2-fold changes in H460R cells, are categorized in Table 1, which is shown to summarize the enriched gene ontology terms of the selected gene set.
Among up-regulated genes in response to IR, we focused on genes listed in the &#39;plasma membrane&#39; category among the cellular components categories.
In addition to being a membrane adapter protein, SNTB2 was modulated by interaction with MRP2 [24], a member of the ATP-binding cassette (ABC) protein superfamily, functioning as an ATP-dependent export pump for anionic conjugates in the apical membranes of epithelial cells.
Since ABC proteins have been implicated in resistance [25, 26], SNTB2 was selected for further analysis to evaluate whether it is involved in radioresistance.
Consistent with the microarray data, protein levels of SNTB2 in H460 cells were highly increased in response to IR compared to the parental cells (Fig.4A).
Knockdown of SNTB2 by siRNA sensitized H460R cells as well as the parental cells to IR (Fig.4B and 4C), although the effect of si-SNTB2 on H460R was less compared with parental cells.
SNTB2 expression levels were examined in other NSCLC cell lines to determine whether A549 and H1299 cells expressed high levels of SNTB2 compared with H460 cells and H1299, which are more resistant to IR exposure (Fig.5A).
We verified that knockdown of SNTB2 by siRNA made H1299 cells more sensitive to IR exposure (Fig.5B and 5C), suggesting that SNTB2 plays a role in radioresistance.
Cancer cells often demonstrate resistance to IR, fail to arrest the cell cycle to repair DNA damage, and continue to proliferate under genotoxic stress [10].
Therefore, it is very important to understand radioresistance as well as enhanced invasiveness for RT.
In the present study, we established radioresistant NSCLC H460 cells by fractionated low-dose IR and evaluated their response to IR.
Identification of DEGs related to DNA damage, cell growth, and ECM compared with parental cells in radioresistant lung cancer cells that have been developed by fractionated IR has been reported [23, 27].
IR induces autophagy and the formation of acidic vesicular organelles in cancer cells [22].
We also found that H460R cells had a distinct morphology, with enhanced cell scattering and an increase in acidic vesicular organelles.
Low-dose IR has been shown to induce changes in the cellular microenvironment, leading to mesenchymal changes, and to significantly enhance the ability of migration and invasion in breast cancer cells [28].
Differentiation pathways have been involved in radioresistance as well as chemoresistance [29].
For example, Wnt/beta-catenin mediates radiation resistance [3, 30].
Some of these genes may be responsible for the emergence of resistance to therapy-for example, MMPs [31].
We also observed that radioresistant H460R cells exhibited increased activities of migration and representative MMPs, including MMP2 and MMP9, compared with parental cells.
Our findings are consistent with recent studies showing that MMPs are able to control metastasis [2, 15, 32].
In addition, expression of β-catenin appeared in radioresistant cells but not in parental cells.
Based on our observation of rare expression of E-cadherin and N-cadherin, β-catenin might play an important role in radioresistant H460R cells.
Our findings suggest that H460R cells might have the potential for invasion and metastasis, which will be elucidated in a future study.
H460 cells with wild-type p53 responded with a shorter G2/M phase, leading to increased apoptosis compared with radioresistant H460R cells.
Results from flow cytometry correlated with a series of Western blot data.
Expression levels of XIAP decreased in the parental cells but increased in H460R cells.
Expression patterns of GRP78 and GRP94 in H460R cells were different from those of parental cells.
Strongly increased CHOP protein levels in H460 cells after 8 h by IR, but not in H460R cells, were consistent with a previous study showing that CHOP-deficient cells have enhanced survival when challenged with toxins that induce ER stress [14].
These data suggest that H460R cells are altered in response to IR-induced ER stress as well as DNA damage.
In this study, SNTB2 increased after IR exposure, and knockdown of SNTB2 led to an increase in sensitivity in both H460 and H460R cells in response to IR.
However, the effect of si-SNTB2 on H460R was less compared with the parental cells.
H460R cells can be considered to be genetically more heterogeneous in response to repeated IR.
In fact, we observed that si-SNTB2 did not sufficiently knock down SNTB2 levels in H460R cells compared with the parental cells, based on data from the Western blot analysis.
Hence, we alternatively utilized IR-resistant H1299 cells to confirm that SNTB2 is involved in radioresistance.
H1299 cells, which express endogenously high SNTB2 levels and are radioresistant, were more sensitive to IR exposure when SNTB2 was down-regulated by siRNA.
This suggests that overexpression of SNTB2 is linked to radioresistance in H1299 as well as H460R cells, implying that SNTB2 plays a critical role in the radioresistance of cancer cells.
We are planning to explore the gain of function by SNTB2 using its ectopic overexpression in a further study and to clarify how SNTB2 plays a role in radioresistance.
In summary, we established H460R-radioresistant cells by fractionated low-dose IR and showed that these cells exhibit not only morphological changes but also a differential response to IR at the molecular level compared with parental cells.
Among DEGs, SNTB2 played a role in radioresistance.
Therefore, H460R cells will be helpful as an in vitro cellular model of radioresistance to evaluate its mechanism and to develop drugs to overcome radioresistance occurring during RT.
Analysis of radioresistant H460R cells under physiologically normal conditions.
(A) Morphology changes and increased vacuoles in H460R cells.
Arrow points to a vesicular organelle (left).
Flow cytometry after acridine orange staining (right).
(B) MTT activity.
(C) Cell cycle and (D) reactive oxygen species levels between H460 and H460R cells from flow cytometric analysis.
Not significant (ns); p &gt; 0.05 vs. H460 (n = 3).
MMP, matrix metalloproteinase.
Migration and matrix metalloproteinase (MMP) 2 and MMP9 activities increased in H460R cells.
(A) Comparison of colony morphology between H460 and H460R cells.
(B, C) Wound healing assay.
(D) Zymography (upper) and expression of representative proteins (lower) related to the epithelial-to-mesenchymal transition between H460 and H460R cells.
**p &lt; 0.01 vs. H460 (n = 3).
Differential response of H460R cells to γ-irradiation.
(A, B) Clonogenic assay.
(C) Cell cycle analysis.
(D) Western blot of DNA damage response.
(E) Analysis of representative proteins related to endoplasmic reticulum stress.
*p &lt; 0.05 vs. H460 (n = 3).
CTL, control; XIAP, X-linked inhibitor of apoptosis; PARP, poly(ADP ribose) polymerase; CHOP, C/EBP-homologous protein.
Effects of si-syntrophin beta 2 (SNTB2) on ionizing radiation (IR)-induced apoptosis in H460 and H460R cells.
(A) SNTB2 expression, relative ratio of SNTB2 proteins, and poly(ADP ribose) polymerase (PARP) cleavage in response to IR (4 Gy).
After irradiation, Western blot analysis was performed.
(B) Effect of SNTB2 knockdown on IR-induced cell viability.
After transfection using si-mock or si-SNTB2 siRNA, cells were analyzed by Western blot (upper), and after IR to siRNA-treated cells, cell viability was measured via Trypan Blue exclusion assay (lower).
*p &lt; 0.05 vs. si-mock (n = 3).
(C) Colony formation assay after IR to siRNA-treated H460 and H460R cells.
Effects of si-syntrophin beta 2 (SNTB2) on ionizing radiation (IR)-induced apoptosis in H2199 cells.
(A) Expression level of SNTB2 in non-small-cell lung cancer cell lines (upper) and colony formation assay after IR in H460 and H1299 cells (lower).
CTL, control.
Effect of SNTB2 knockdown on IR-induced cell viability (B) and IR-induced apoptosis (C) in H1299 cells.
After transfection with si-mock or si-SNTB2 into H1299 cells, cells were irradiated, and cell viability was measured by a Trypan Blue exclusion assay; lysates were analyzed by Western blot using cleaved caspase-9/-3, poly(ADP ribose) polymerase (PARP), Bcl-2, and survivin antibodies.
Title: Enriched gene ontology of upregulated or downregulated genes in radioresistant H460R cells
Title: Genetic Variations of ABCC2 Gene Associated with Adverse Drug Reactions to Valproic Acid in Korean Epileptic Patients

The multidrug resistance protein 2 (MRP2, ABCC2) gene may determine individual susceptibility to adverse drug reactions (ADRs) in the central nervous system (CNS) by limiting brain access of antiepileptic drugs, especially valproic acid (VPA).
Our objective was to investigate the effect of ABCC2 polymorphisms on ADRs caused by VPA in Korean epileptic patients.
We examined the association of ABCC2 single-nucleotide polymorphisms and haplotype frequencies with VPA related to adverse reactions.
In addition, the association of the polymorphisms with the risk of VPA related to adverse reactions was estimated by logistic regression analysis.
A total of 41 (24.4%) patients had shown VPA-related adverse reactions in CNS, and the most frequent symptom was tremor (78.0%).
The patients with CNS ADRs were more likely to have the G allele (79.3% vs. 62.7%, p = 0.0057) and the GG genotype (61.0% vs. 39.7%, p = 0.019) at the g.-1774delG locus.
The frequency of the haplotype containing g.-1774Gdel was significantly lower in the patients with CNS ADRs than without CNS ADRs (15.8% vs. 32.3%, p = 0.0039).
Lastly, in the multivariate logistic regression analysis, the presence of the GG genotype at the g.-1774delG locus was identified as a stronger risk factor for VPA related to ADRs (odds ratio, 8.53; 95% confidence interval, 1.04 to 70.17).
We demonstrated that ABCC2 polymorphisms may influence VPA-related ADRs.
The results above suggest the possible usefulness of ABCC2 gene polymorphisms as a marker for predicting response to VPA-related ADRs.
Various forms of epilepsy are among the most common serious brain disorders and present through convulsion, autonomic movement, impaired consciousness, and more [1].
These neurological disorders affect an estimated 42 million people of all ages worldwide [2].
The blood-brain barrier (BBB) has challenged the treatment of epilepsy, since many antiepileptic drugs (AEDs) are not effectively distributed to the vascular endothelium or, accordingly, to their targets in brain [3, 4].
The BBB, thought to be the first-line barrier to the disposition of drug from the blood to the brain [5], consists of the interplay of three major microvascular components.
Tight junctions between the endothelial cells constitute the major permeability barrier.
The overall biology of the barrier is shaped by the interactions between the endothelium and the pericytes/smooth muscle cells and the astrocyte foot processes that cover most of the abluminal surface of the microvasculature [6].
Drug pharmacokinetics is controlled by drug transporters, which regulate drug absorption, distribution, and excretion.
Drug transporters are considered a second-line barrier for limiting drug disposition from the blood to the brain [7].
In brain capillary endothelial cells, multidrug resistance protein 1 (MDR1); multidrug resistance-associated protein 1 (MRP1), MRP2, MRP4, and MRP5; and breast cancer resistance protein (BCRP) are among the many drug transporters that might be expressed at the luminal side of the membrane [5, 8, 9].
MRP2, in particular, is over-expressed (about 225% greater) in drug-refractory temporal lobe epileptic patients with hippocampal sclerosis (HS) [5, 10-12].
Drug reactions can be different between individuals with the same AED blood concentrations and therapeutic levels, because drug distribution is influenced by variable levels of drug transporters.
MRP2 is thought to play a more important role in the distribution of AEDs than other transporters in the epileptic brain [13].
MRP2 has been demonstrated to transport many AEDs, including valproate (VPA), carbamazepine (CBZ), phenytoin (PHT), and more [9, 14-17].
Valproic acid has become established as one of the most widely used AEDs in the treatment of both generalized and partial seizures in adults and children and is considered to be a substrate of MRP2 [9, 14-17].
Probenecid, an inhibitor of MRPs, has been shown to increase the concentration of valproic acid in the brain [18].
In animal models, TR- rats (MRP2 knockout) have lower VPA and VPA-glucuronide in their bile than control Wistar rats (which intrinsically express MRP2) [19].
Functional polymorphisms in genes encoding drug transporters can alter AED uptake, cerebral distribution, and efflux, resulting in individual differences in AED concentration and effectiveness and/or the occurrence of adverse drug reactions (ADRs).
Many functional single-nucleotide polymorphisms (SNPs) have been reported.
For example, mRNA expression of the c.1446CG MRP2 genotype was recently revealed to be higher than the c.1446CC genotype in the liver [20].
In addition, the c.2302C &gt; T (exon 18, Arg768Trp) mutation is responsible for Dubin-Johnson syndrome [21, 22].
The c.2302C &gt; T and c.4348G &gt; A genotypes correlate with significantly lower MRP2 protein expression levels compared to wild-type and V417I [21].
The c.1249G &gt; A mutation significantly reduces the amount of MRP2 mRNA in human preterm placentas [23].
The g.-1774delG polymorphism has been linked with toxic hepatitis by our group [24].
In the present study, we investigated the association between the g.-1774delG MRP2 genotype and ADRs of the central nervous system (CNS) in VPA treatment groups.
Title: Subjects

This retrospective study included 168 epileptic Korean patients who received VPA at Sinchon and Gangnam Severance Hospitals.
Forty-one patients demonstrated VPA dose-related ADRs in the central nervous system (dizziness, headache, somnolence, diplopia, dysarthria, tremor, etc.
), while the remainder (n = 127) did not.
Patients who were diagnosed with chronic active epilepsy, West syndrome, Lennox-Gastaut syndrome, progressive myoclonic epilepsy, tuberous sclerosis, Sturge-Weber syndrome, hamartoma, or brain tumors were excluded.
There were no statistical differences in age, sex, response/non-response, and sclerosis between the two groups.
Demographic characteristics of the epileptic patients are presented in Table 1.
DNA from control subjects (n = 110) was randomly selected from the DNA bank of the Korea Pharmacogenomics Research Network at Seoul National University.
Blood samples were collected from each subject, and DNA was extracted using a QIAamp DNA blood mini kit (Qiagen GmbH, Hilden, Germany).
Title: Genetic analysis

Polymorphisms of the MDR1, MRP, and BCRP genes in the Korean population were discovered by denaturing gradient gel electrophoresis, two-dimensional gene scanning, and direct PCR using methods similar to those described in a previous paper [24].
Genotype screening of each locus in control and epileptic patients was performed by the SNaPshot or SNaPIT method (Applied Biosystems, Foster City, CA, USA), according to the protocols supplied by the manufacturer.
Title: Statistical analysis

Haploview software (version 3.2) was used to design MRP2 haplotype constructs and analyze major or minor haplotypes based on a standard expectation-maximization algorithm.
Allele and genotype frequencies of transporter polymorphisms were assessed using chi-square tests (version 11.5 for Windows; SPSS Inc., Chicago, IL, USA).
Title: Logistic regression

The strength of the association between dose-related CNS ADR patients and the presence of the G allele at the g.-1774 region was evaluated as the odds ratio (OR) obtained with logistic regression analysis (SPSS version 11.5).
ORs were adjusted for gender, age, HS, use of AEDs (larmotrigene, CBZ, PHT, and topiramate), and the presence of the G allele at the g.-1774 promoter region.
Title: Cell culture

SH-SY5Y (ATCC, Manassas, VA, USA), a human brain neuronal cell line originated from a neuroblastoma, was a gift from Dr.
In Suk Kim, Yonsei University College of Medicine, Seoul, Korea.
SH-SY5Y cells were maintained with Dulbecco&#39;s modified Eagle medium supplemented with 10% fetal bovine serum (Invitrogen, Carlsbad, CA, USA) and 1% PS (100 units penicillin, 100 µg streptomycin, Gibco, Grand Island, NY, YSA) and were cultured under preconfluent monolayer conditions in 100-mm-diameter polyd-lysine-coated culture dishes at 37℃ with 5% humidified CO2.
Title: Plasmid preparation of MRP2 promoter

A pGL3 basic vector containing the human MRP2 promoter region (about 2.3 kb, -2314 to +348 relative to the translation initiation site) was constructed by our group in a previous study from homozygotic persons with MRP2 haplotypes 1, 2, and 3 [24].
The g.-1774delG polymorphism is located in haplotype 1, and the g.-24C &gt; T polymorphism is located in haplotype 3.
The reporter vector of a minor haplotype variant containing only the g.-1549G &gt; A variation was constructed using mutagenic primers, introducing a -24 T → C change to the plasmid of haplotype 3 containing both g.-1549G &gt; A and g.-24C &gt; T variations.
Haplotype 2 includes no polymorphism (wild-type).
In short, a total of five clones were prepared: pGL3 basic; haplotypes 1, 2, and 3; and g.-1549A alone.
Title: Dual luciferase assay

The pGL3 basic vector containing WT MRP2 promoter variants was transfected into SH-SY5Y cells to measure the activity of the MRP2 promoter with and without VPA (P4543; Sigma Aldrich, St. Louis, MO, USA).
The day before transfection, 0.2 × 106 SH-SY5Y cells were seeded in a 12-well plate in order to reach 90-95% confluence at the time of transfection; 1.6 µg of pGL3-MRP2, including the variants and mock, was added to wells with Lipofectamine 2000 (Invitrogen).
Vector-containing Renilla luciferase was cotransfected in order to confirm the transfection efficiency of the pGL3-MRP2 vector.
After a 3- to 5-h incubation, cells were washed with phosphate-buffered saline (PBS; Invitrogen) and incubated at 37℃ with 5% humidified CO2 for 24 h. Reporter activity was measured using the Dual Luciferase Reporter Assay system (Promega Corporation, Madison, WI, USA).
After removing the growth medium, cells were washed with PBS, and the plates were placed on a rocking platform shaker with 250 µL of 1 × passive lysis buffer per well for 15 min.
Lysates were transferred to an e-tube, and luciferase binding was performed with 50 µL LARII and 10-50 µL cell lysate in a 96-well assay plate (Costar; Corning Incorp., Corning, NY, USA).
After 10 s, light emission was measured using a luminometer (Berthold Technologies, Wildbad, Germany), and Stop &amp; Glo was added to each well to stop firefly luciferase activity and evoke Renilla luciferase activity.
Title: SNP association with drug response or resistance

Drug resistance was defined as the occurrence of at least four seizures over the year before recruitment, with trials of more than three appropriate AED, at maximal tolerated doses, based on the occurrence of clinical side effects at supra-maximal doses [25].
The patients were classified as either responsive or resistant to drug therapy.
We did not find any significant associations between the selected drug transporter SNPs and either group.
Title: Clinical characteristics of epilepsy patients

A total of 41 (24%) epileptic patients had ADRs of the CNS, and 127 (76%) epileptic patients did not (non-ADR).
The demographic characteristics are listed in Table 1.
The gender frequency, drug response, and the average age of drug use between the two groups were similar.
(p &gt; 0.05) Subjects without ADRs used fewer drugs than those with ADRs (p = 0.039).
The most frequent symptom out of 14 types of ADRs to VPA use was tremors (78.0%).
Based on logistic regression, ADRs were more likely to occur in patients with g.-1774delG (presence of G allele), which had the highest OR (9.10) among the considered factors (p &lt; 0.05).
Title: Genetic polymorphisms of MRP2 in the Korean population

MRP polymorphisms covering all MRP2 exons, exonintron junctions, and the promoter region up to -2.3 Kb from the translation initiation site have been described in the Korean population [24].
We investigated 11 of the polymorphisms from previous studies in 110 healthy control people and 168 epileptic patients.
We found four promoter, four exonic, and three intronic polymorphisms among them.
Title: Relationship between genotypes of MRP2 and CNS ADRs to VPA

Allele frequencies of the MRP2 polymorphisms are shown in Table 2.
Non-ADR patients were more likely to have the deletion allele instead of the G allele at g.-1774 when compared with patients with CNS ADRs (p = 0.0057; ADR, 20.7%; non-ADR, 37.3%).
The frequency of the T allele at g.-24 was higher in patients with CNS ADRs than those without (p = 0.0274; ADR, 34.1%; non-ADR, 22.0%).
The difference between the groups based on g.-24C &gt; T frequencies was not significant after Bonferroni&#39;s correction, while the difference based on g.-1774delG frequencies remained significant.
The genotype frequencies of each polymorphism are shown in Table 3.
Genotype frequency distributions were consistent with Hardy-Weinberg equilibrium (each p &gt; 0.5).
The ADR group patients were more likely to have a GG genotype than a deldel genotype at g.-1774delG compared to the non-ADR group (p = 0.023, homo GG and deldel p = 0.0146).
The TT and CT genotype frequencies at g.-24C &gt; T were higher in patients with CNS ADRs than non-ADR patients (p = 0.035, hetero CT → TT p = 0.019).
The frequencies of haplotypes present in at least 10% of the population are presented in Table 4.
Haplotype frequencies of the MRP2 polymorphisms at all site.
The frequency of haplotype 1 was significantly higher in patients without CNS ADRs (32.3%) than those with ADRs (15.8%) (p = 0.0039).
The frequency of haplotype 3 was higher in patients with CNS ADRs than those without (p = 0.0139), but this difference was no longer significant after Bonferroni&#39;s correction was applied.
Title: Measurement of MRP2 promoter activity

VPA stimulated MRP2 promoter activity about 10-fold.
The epileptic patients with ADRs of the CNS had less haplotype 1 (containing g.-1774del) than the epileptic patients with no ADRs.
By inference, the g.-1774G allele, a major allele of the MRP2 promoter sequence, was associated with ADRs of the CNS in patients treated with VPA.
Epileptic patients with g.-1774del instead of g.-1774G were more likely to be resistant to VPA-related ADRs, probably because the g.-1774del allele produced higher MRP2 promoter activity than g.-1774G, leading to more MRP2 proteins in the brain tissue after VPA treatment.
The presence of more MRP2 affected the distribution of AEDs (especially VPA) from the blood to the brain, resulting in the appearance of VPA-related ADRs in patients with g.-1774G rather than g.-1774del.
Our luciferase assay data supported this hypothesis.
There was no difference in luciferase activity between g.-1774del and the others (g.-1774G, g.-24C&gt;T, g.-1549G &gt; A) with non-VPA treatment.
Luciferase activity was different, however, following 3 mM VPA treatment; g.-1774del promoter (haplotype 1) activity was 134% higher than g.-1774G promoter (haplotype 2) activity (n = 5; p &lt; 0.05 by t-test).
Further, the g.-1774del MRP2 promoter increased 1,014% with 3 mM VPA treatment compared to non-VPA treatment.
However, the g.-1774G MRP2 promoter also increased 816% with 3 mM VPA treatment compared to non-VPA treatment.
The change of g.-1774G → del increased the expression of MRP2 in neuronal SH-SY5Y cells treated with 3 mM VPA (Fig.1).
Title: Factors influencing ADRs to VPA

Many factors, including gender; age; treatment with CBZ, lamotrigine, PHT, or topiramate; and HS, did not influence ADRs.
The presence of the genotype G/G at the g.-1774 region of the MRP2 promoter influenced ADRs to VPA in epileptic patients.
Logistic regression showed that the G/G correlated independently with ADRs to VPA (p &lt; 0.05; OR, 9.102; 95% confidence interval, 1.111 to 73.378).
The del/del allele was the reference category (Table 5).
The G/G genotype at g.-1774 might influence lower expression of MRP2 than del/del in the brain, leading to ADRs.
The selective cellular barrier controlling drug delivery into the brain has been identified by the over-expression of MRP2 in capillary endothelia, astrocytes, and neuronal cells in previous studies [5, 26, 27].
In this study, we found that the g.-1774del genotype in the promoter region up-regulated the expression of MRP2.
Patients with g.-1774del showed resistance to ADRs by increasing the expression of MRP2, followed by a decrease of drug concentration in the brain.
Our data supported this hypothesis and verified an association between g.-1774delG and ADR.
The difference of luciferase activity between g.-1774G and g.-1774del in neuronal SH-SY5Y cells was not large.
However, they showed extensive differences by statistical analysis (p &lt; 0.05 by t-test).
Because the expression level of MRP2 is lower in brain neuronal tissue than in hepatic tissue, even a little change is thought to be important.
The luciferase assay of MRP2 promoter activity has not been conducted in brain capillary endothelial cells yet.
In this cell line, SNP function is thought to be more effective, since MRP2 is more highly expressed in endothelial cells than in neuronal cells [28].
In reality, if MRP2 and other transporters are overexpressed in the brain, most patients should be not only non-ADR but also resistant to drug therapy.
However, allele and genotype frequencies of g.-1774delG were not different between drug-resistant and drug-responsive patients (p &gt; 0.05).
The absence of a strict definition of drug-resistant epilepsy may have weakened the statistical analysis.
The selected grouping was only a foundation for retrospective chart review.
If the definition was stricter-for example, at least one seizure per year or non-seizure over the year-the different frequencies of g.-1774delG within each group would have been more significant.
An accurate study design and definitions are a key part of effective pharmacogenomic studies [2].
Standards for association studies in pharmacogenomics have been presented in previous studies [2].
In spite of the retrospective study design, an association with g.-1774delG was noticeably significant.
This SNP was reported previously in our group in relation to toxic hepatitis [24].
In that study, the g.-1774del genotype caused lower expression of MRP2 than g.-1774G in a human hepatocyte-originated cell line (HepG2).
This finding is contrary to the results of the present study.
There are different known transcription factors in hepatocytes and neuronal and endothelial cells.
The promoter activity in the cells transfected with a g.-1774G (haplotype 2) clone was about 100 times stronger than those transfected with the control vector in hepatic cells.
A strong transcriptional activator is present in hepatocytes.
However, the difference in promoter activity between g.-1774G and mock control was lower in neuronal SH-SY5Y cell lines.
Neuronal transcription factors are expected to be weaker than those in hepatocytes.
The binding sequence is &#34;AAAAACAACAAGATAA&#34; (the underlined C is the anti-sense sequence at g.-1774G).
Transcription factors that are known to bind this sequence are GATA-1, Evi-1, FOXO, HNF3, and more (Genomatix software GmbH).
Each tissue-specific transcription factor binds this sequence and induces different levels of MRP2 expression.
For example, HNF3, a common activator in hepatocytes, was thought to activate MRP2 expression through binding with g.-1774G rather than g.-1774del.
Consequently, the expression of MRP2 of g.-1774del would be lower than g.-1774G.
In contrast, if neuronal cell transcriptional activators were more likely to bind g.-1774del over g.-1774G, neuronal expression of MRP2 of g.-1774del would be higher than g.-1774G.
The factors that bind to this region have yet to be determined.
Previous studies have referred to the different binding factors of this region in human coronary artery smooth muscle cells without specific identification.
In the present study, a luciferase assay demonstrated that VPA stimulated the activity of the MRP2 promoter in SH-SY5Y cells.
VPA is a well-known histone deacetylase (HDAC) inhibitor that induces apoptosis in cancer cells [29, 30].
In normal cells, however, VPA activates chromosomal transcriptional ability by inhibiting HDAC.
Chromosomal DNA forms a compact nucleosome with histone proteins and needs to be unpacked in order to start transcription.
HDAC removes the acetyl group in acetyl-lysine residues and biologically inhibits the activity of transcription [31].
In neuronal SH-SY5Y cells, VPA could activate the MRP2 promoter by inhibiting HDAC.
MRP2 has farnesoid X receptor, retinoid X receptor (RXR), and peroxisome proliferator-activated receptor/RXR binding regions.
VPA interacts with these proteins and regulates the expression level of MRP2 protein [32, 33].
P-gp polymorphisms have repeatedly been associated with drug response in epilepsy [25, 34-37].
Up to now, however, the only functional studies of MRP2 have been conducted in a knockout animal model [38].
This report is the first investigation to show an association between MRP2 polymorphisms and ADRs in epilepsy.
These results may be useful in tailoring AED medication.
The G allele of the MRP2 promoter was associated with ADRs to VPA in Korean epileptic patients.
The del allele of MRP2 protected the brain against adverse effects of VPA by over-expressing MRP2 in brain neuronal cells.
These data will help design therapy to minimize ADRs in epileptic patients and aid in the development of new AEDs.
However, further independent replication studies are needed to unequivocally confirm the association.
Luciferase activity of MRP2 promoter variants.
Valproic acid (VPA) stimulated MRP2 promoter activity.
The g.-1774del (-GCG) variant was more activated than g.-1774G (GGCG) in cells treated with 3 mM VPA.
*p &lt; 0.05.
Title: Demographic characteristics of epilepsy patients treated with VPA

Title: Allele frequencies of the MRP2 polymorphisms

Title: Genotype distribution of MRP2 variants in control and epilepsy patients

Title: Frequency of MRP2 haplotypes in control and epilepsy patients

Title: Multiple logistic regression analysis for the determinants of CNS ADRs caused by VPA
Title: TT Mutant Homozygote of Kruppel-like Factor 5 Is a Key Factor for Increasing Basal Metabolic Rate and Resting Metabolic Rate in Korean Elementary School Children

We investigated the contribution of genetic variations of KLF5 to basal metabolic rate (BMR) and resting metabolic rate (RMR) and the inhibition of obesity in Korean children.
A variation of KLF5 (rs3782933) was genotyped in 62 Korean children.
Using multiple linear regression analysis, we developed a model to predict BMR in children.
We divided them into several groups; normal versus overweight by body mass index (BMI) and low BMR versus high BMR by BMR.
There were no differences in the distributions of alleles and genotypes between each group.
The genetic variation of KLF5 gene showed a significant correlation with several clinical factors, such as BMR, muscle, low-density lipoprotein cholesterol, and insulin.
Children with the TT had significantly higher BMR than those with CC (p = 0.030).
The highest muscle was observed in the children with TT compared with CC (p = 0.032).
The insulin and C-peptide values were higher in children with TT than those with CC (p= 0.029 vs. p = 0.004, respectively).
In linear regression analysis, BMI and muscle mass were correlated with BMR, whereas insulin and C-peptide were not associated with BMR.
In the high-BMR group, we observed that higher muscle, fat mass, and C-peptide affect the increase of BMR in children with TT (p &lt; 0.001, p &lt; 0.001, and p = 0.018, respectively), while Rohrer&#39;s index could explain the usual decrease in BMR (adjust r2 = 1.000, p &lt; 0.001, respectively).
We identified a novel association between TT of KLF5 rs3782933 and BMR in Korean children.
We could make better use of the variation within KLF5 in a future clinical intervention study of obesity.
Kruppel-like factors (KLFs) are zinc-finger transcription factors considered to be crucial regulators of cell differentiation, development, and cell cycle progression [1].
Each KLF has its own function and acts as a transcriptional activator or repressor, depending on the promoters [2].
Several features distinguish KLFs from the highly conserved classical zinc-finger transcription factors [2, 3].
Zinc-fingers are located at the carboxyl terminus of the protein and permit KLFs to bind to related GC- and CACCC-boxes of DNA [4].
In contrast to the zinc-finger regions, the non-DNA-binding regions are highly controversial, regulate activation and repression, and often mediate interaction of protein-protein [3].
Recently, a potential role for KLFs in adipogenesis has been suggested [5].
For example, KLF5 and KLF2 are expressed in 3T3-L1 preadipocytes and bind directly to the peroxisome proliferator-activated receptor γ2 (PPARγ2) promoter or inhibit its activity [6].
KLF5 (also known as BTEB2 or IKLF) was first identified as a transcription factor for SMemb/non-muscle myosin heavy chain-B (NMHC-B), a marker of smooth muscle cells in vascular disease [7].
KLF5 upregulates in activated smooth muscle cells or fibroblasts of vascular lesions and downregulates in adult vessels [7, 8] KLF5 binds directly to the PPARγ promoter, and PPARγ activity can be enhanced by either C/EBPβ or C/EBPδ [4].
Therefore, several studies suggested KLF5 is downstream of C/EBPβ or C/EBPδ and upstream of PPARγ during adipocyte differentiation [4, 5].
Just like KLF2, KLF5 might contribute to adipogenesis transcriptional regulation via PPARγ.
KLF15 induces differentiation of 3T3-L1 in the late stage and expression of the glucose transporter GLUT4 [9].
KLF7 is expressed in adipocytes, as well as several human tissues, such as pancreas, skeletal muscle, and liver [10].
However, the functions of these KLFs in adipocyte differentiation in vivo remain unclear.
The prevalence of overweight and obesity is increasing worldwide.
The imbalance between energy expenditures and energy intakes causes excessive fat accumulation, leading to an increase of the number and/or size of fat cells [4].
Basal metabolic rate (BMR) is the minimum energy required to retain the physiological functions of the body while awake.
BMR accounts for approximately 45% to 70% of total energy expenditure in most healthy adults and is affected by age, gender, body surface area, body composition, genetic composition, pregnancy, and hormonal status directly [11, 12].
Thus, estimating BMR is important for establishing strategies of obesity prevention programs [11].
For several years, numerous predictive equations for BMR have been developed in different populations.
Although many studies have shown the influence of ethnicity on BMR, their results have been inconsistent [11, 12].
On the other hand, resting metabolic rate (RMR) is also the energy requirement to support vital functions while resting in the supine position.
Just like the BMR, RMR is quantitatively the largest component of daily energy expenditure (60-75%).
In order to evaluate RMR, direct and indirect calorimeters, such as whole-room calorimeters, open-circuit Douglas bag method, metabolic carts, and ventilated hood systems, are required, and these methods are not helpful nor cost-effective for large-scale epidemiological studies.
Therefore, regression equations have been used to estimate resting energy requirements in large populations [13].
So far, several studies have suggested that uncoupling protein 2 (UCP2) is involved in the regulation of energy expenditure metabolism, body weight, and obesity [14].
In other studies, the polymorphism of the uncoupling protein 1 (UCP1) gene did not have an independent effect on BMR; however, its synergistic effect with the beta3-AR gene polymorphism induced a lower BMR than the polymorphism of beta3-AR gene alone [15].
Recently, to investigate the clinical relevance of KLF5 in complex diseases, such as cardiovascular disease, several single-nucleotide polymorphisms (SNPs) within KLF5 have been analyzed; however, an association of KLF5 SNPs with BMR of children has not been identified yet.
The purpose of this study is 1) to identify the contribution of novel genetic variations of KLF5 to increased BMR and the suppression of obesity in Korean children and 2) to make better use of the alterations of KLF5 in future clinical intervention studies.
Title: Study population

A total of 62 eligible subjects were recruited from 3rd grade to 6th grade students of elementary schools in 2005.
Ethical permission for the study was obtained from the Institutional Review Boards, and all subjects provided written informed consent.
Demographics of age, gender, weight, height, body mass index (BMI), fat mass, muscle, waist-hip ratio (WHR), and BMR, as well as skinfold thickness, lipid profile, and hematological data, were also recorded.
All variables were measured for developing the predictive equation model of BMR.
Title: Anthropometric measurements

All measurements were made using standard techniques.
The standing height was measured with a stadiometer (Jenix) attached to a wall.
Weight, height, fat mass, muscle mass, and WHR were measured by bioelectrical impedance analysis (Inbody 720; Biospace, Seoul, Korea).
BMI was calculated by dividing weight (kg) by the square of height (m).
Subjects were divided into two groups by the 85 percentile of BMI values.
Children with normal weight were the lower BMI group (BMI &lt; 85th), and the higher BMI group was overweight (BMI ≥ 85th).
Also, Rohrer index (RI) was measured by height and weight for obesity judgment.
Skinfold thickness of the triceps and subscapular and suprailiac regions were measured by caliper (Cambridge Scientific Industries, Cambridge, MD, USA).
Title: RMR and BMR measurements

RMR was measured using a MedGem metabolic analyzer (Microlife USA, Dunedin, FL, USA) [16].
This device is a handheld indirect calorimeter used for its accuracy and portability in clinical settings and is becoming popular in research studies as a screening tool and a criterion measure of RMR.
This device calculates RMR using florescent quenching of ruthenium in the presence of oxygen and the modified Weir equation, with an assumed respiration quotient of 0.85 to estimate carbon dioxide production [17].
Prior to each measurement, the MedGem was autocalibrated on zero airflow according to the manufacturer&#39;s instructions.
Participants rested in the supine position on a dormitory bed with a pillow for 15 min in a climate-controlled laboratory, where lights were dimmed and the setting was quiet.
Following this rest period, participants were asked to breathe through a mouthpiece in a relaxed fashion for approximately 7-10 min, depending on the stability of the VO2 measures.
The participants were allowed to hold the mouthpiece in their mouth during the measurement if they felt it necessary.
Once the participant had reached steady-state oxygen consumption, as measured by the MedGem, data collection was completed.
If the children were unable to reach steady-state oxygen consumption in the allotted time of approximately 10 min, the test was terminated at 10 min by the device, and RMR was calculated as a mathematical average from the final 8 min of data collection.
The MedGem then calculated steady-state RMR for a 24-h period (kcal/day) [13].
BMR was measured using indirect calorimetry.
All measurements were repeated three times.
Title: Blood biochemistry

Blood samples were collected after 12-h fasting and were immediately placed in 4℃.
For genotyping, whole blood was collected in ethylenediaminetetraacetic acid-treated tubes.
For other blood profiles, blood samples were collected in serum separator tubes and centrifuged at 3,000 rpm for 20 min at 4℃.
Serum samples were frozen at -80℃ until used.
Levels of total cholesterol (TC), triglyceride (TG), high-density lipoprotein cholesterol (HDL-C), apolipoprotein B, apolipoprotein A1, and C-reactive protein were measured by an auto analyzer (Boeringer Mannheim, Mannheim, Germany).
Low-density lipoprotein cholesterol (LDL-C) was calculated as described by the Friedewald and Laucer equation [18, 19]: LDL-C = TC - HDL-C-(TG/5).
Levels of fasting glucose were measured by using an autoanalyzer (Ekachem DTSC module; Johnson &amp; Johnson, New Brunswick, NJ, USA).
Fasting insulin was measured by glucotrend (Roche, Mannheim, Germany), and homeostasis model assessment of insulin resistance (HOMA-IR) was calculated as described by Matthews equation [20]: HOMA-IR = fasting glucose (mmol/L)/22.5 × insulin (µL/mL), fasting glucose (mmol/L) = fasting glucose (mg/mL) × 0.0555.
Title: Analysis of the KLF5 SNP

The polymorphisms for the KLF5 (rs3782933) gene were analyzed (http://www.ncbi.nlm.gov/).
For each sample, the genomic DNA was isolated from whole blood using the QIAamp DNA blood mini kit (Qiagen, Valencia, CA, USA) according to the supplier&#39;s instructions.
PCR was performed using the Takara DNA Polymerase Kit (Takara Bio Inc, Shiga, Japan).
All the primers for PCR amplification and DNA sequencing were designed using the Primer3 software (http://Frodo.wi.mit.edu/cgi-bin/primer3/primer3); the sequences are available upon request.
PCR reactions were carried out in a final volume of 25 µL containing 10× buffer, 1.5 mmol/L MgCl2, 20 µmol/L dNTP, 0.5 µmol/L of each primer, 10 ng of genomic DNA as template, and 0.5 U polymerase.
Each PCR product was purified and subjected to DNA sequencing by using the QIAquick PCR purification Kit (Qiagen) and the ABI Prism genetic analyzer (Applied Biosystems, Foster City, CA, USA) after confirming the purity and mobility of each PCR product by agarose gel electrophoresis.
Each sample was sequenced for both strands to confirm results.
Title: Statistical analysis

The results are presented as mean ± standard error range.
p-values for each genotype and allele were evaluated by χ2 analysis and Fisher&#39;s exact test.
A p-value of &lt;0.05 was considered to indicate a statistically significant difference between obese subjects and normal-weight subjects of the study.
Associations of clinical variables, such as age, BMI, height, weight, and BMR, were analyzed by Pearson correlation test.
Multiple linear regression analysis was performed to develop a BMR equation.
All analyses were performed using SPSS version 18.0 (SPSS Inc., Chicago, IL, USA).
Title: Distribution of genotypes and alleles of KLF5 gene

The recruited 62 children were aged 11.51 ± 0.97 y, and girls made up more than half of the children recruited (n = 33, 53.2%).
Mean BMI estimated from height and weight was 20.26 ± 3.41 kg/m2.
The average BMR was 1,018.42 ± 113.86 kcal/day, and muscle averaged 28.26 ± 4.99 kg (data not shown).
Children with the C allele were nearly twice those with the T allele in each group.
In the overweight group, allele frequencies were 67.6% for C allele and 32.4% for T allele.
However, we found no statistical significance between the frequencies of alleles and genotypes in two groups (Table 1).
On the other hand, four children had TT genotype in the high-BMR group and only one child had TT genotype in the low-BMR group.
Thus, four in five children with the TT genotype belong to the high-BMR group.
However, there were no differences of alleles and genotypes in the low- and high-BMR groups (Table 1).
Title: Clinical characteristics of 62 children according to genotype of rs3782933

The p-values for Hardy-Weinberg equilibrium and minor allele frequency of KLF rs3782933 were 0.8286 and 0.3333, respectively.
The SNP of KLF5 showed significant differences with several clinical factors, such as BMR, muscle, LDL-C, insulin, and C-peptide (Table 2).
Children with the TT genotype had significantly higher BMR than those with the wild-type (CC genotype) (p = 0.030).
The highly correlated factors with BMR, such as height and muscle composition, were also higher in children with the TT genotype than those with the CC genotype (p = 0.032).
The diabetic factors, such as insulin and C-peptide, were also higher in the children with the TT genotype than those with the CC genotype (p =0.029 vs. p = 0.004, respectively) (Table 2).
On the other hand, the alleles of the KLF5 gene showed a divergent trend when compared with the genotype.
For example, the children with the T allele had significantly lower TC and LDL-C than those with the C allele, regardless of BMR (Table 3).
Title: Risk factors related to BMR

The parametric Pearson correlation analysis of the data revealed that BMR was significantly positively correlated with height (r = 0.888, p &lt; 0.001), weight (r = 0.873, p &lt; 0.001), BMI (r = 0.669, p &lt; 0.001), fat mass (r = 0.596, p &lt; 0.001), muscle (r = 1.000, p &lt; 0.001), insulin (r = 0.377, p = 0.002), and C-peptide (r = 0.618, p &lt; 0.001) (data not shown).
Because the value of BMR and RMR was similar to each other, we analyzed the regression equation of BMR with the contributors.
Linear regression analysis was performed to estimate the relative contributions of BMI, muscle, fat mass, insulin, and cholesterol to the inter-individual variations of BMR.
In the linear regression analysis, BMI and muscle mass were correlated with BMR in every genotype of KLF5, whereas insulin and C-peptide were not associated with BMR in children with the CT and TT compared to CC genotype.
It was interesting that TC was negatively correlated with BMR in only the TT genotype in contrast to the CC or CT genotype.
(Figs.1 and 2).
Multiple linear regression analysis was performed to estimate the relative contribution of age, muscle, RI, tricep, glucose, and insulin to BMR in children.
We identified that age, muscle, RI, tricep, WHR, and fat mass could explain 100% of BMR (adjusted r2 = 1.000, p &lt; 0.001) in children (Table 4).
In the high-BMR group, we observed that higher muscle, fat mass, and C-peptide affected the increase of BMR in children with TT genotype (p &lt; 0.001, p &lt; 0.001, and p = 0.018, respectively), while RI could explain the usual decrease in BMR (adjusted r2 = 1.000, p &lt; 0.001).
Our prediction equation of BMR including age, muscle, RI, tricep, WHR, and fat mass specific for children from 10 to 13 years old is as follows: BMR = 369.634+(1.038 × age) + (22.452 × muscle) - (0.087 × RI)+ (0.090 × tricep) + (11.147 × WHR) + (0.532 × fat mass).
This study was the first report of a KLF5 SNP (rs3782933) involved in the susceptibility to BMR in Korean elementary school children.
Children with the TT genotype had significantly higher BMR, muscle, HLD-C, insulin, and C-peptide and lower LDL-C than those with the wild-type CC.
However, we found that BMI, muscle, and fat mass could explain the usual increase in BMR in every genotype in KLF5, whereas insulin and C-peptide were not associated with the increase of BMR, except wild-type, in the linear regression analysis.
Using multiple linear regression analysis, we estimated the predicting equation of BMR in children.
In the high-BMR group, TT genotype increased BMR according to muscle, fat mass, and C-peptide, and there was an inverse correlation between BMR and RI (adjust r2 = 1.000, p &lt; 0.001).
Our results suggested an association between the TT mutant homozygotes of KLF5 rs3782933 and BMR in Korean children.
BMR is affected by many factors, such as age, gender, body composition, and genetic variation.
BMR measurement in humans was primarily used for the diagnosis of hypo- and hyperthyroidism.
Recently, BMR was used to estimate human energy and food requirements as well as the diagnosis of thyroid disease, diabetes, and leukemia [21].
Up to the present, alleles of specific genes, such as UCP1, LPL, and beta3-AR, were considered to play a major role in the development of metabolic complications in the obese population [22].
Moreover, some reports suggested that the variation in the beta3-adrenergic receptor gene influences BMR in obese subjects and that the synergistic effect of polymorphisms in UCP1 and beta3-adrenergic receptor genes leads to a lower BMR [15, 23].
Recently, Suchanek et al.
[24] investigated the association between an alpha2 Heremans-Schmid glycoprotein (AHSG) gene polymorphism and obesity.
There was a significant correlation between homozygotes of AHSG and increased BMR [24].
However, the association of the variation in KLF5 and BMR has not been confirmed yet in Korean children.
Although we identified the novel genetic variation in a noncoding region of KLF5, this SNP might affect complex diseases, such as obesity and diabetes.
Many disease-associated mutations are predicted to be SNPs.
Some of the SNPs associated with complex diseases might be located in the coding and/or noncoding region and lead to alterations in the function of the protein [25].
Recently, Oishi et al.
[1] showed that a novel signaling and transcriptional network involving MEF2A and KLF5 -1,282 bp plays a crucial role in the pathogenesis of hypertension and that the KLF5 promoter allele (-1282A) is associated with a higher prevalence of hypertension [1].
KLF5 has also been involved in vascular smooth muscle cell differentiation, growth, and gene expression [3].
In this population, we divided them into two groups by BMI: normal weight and overweight.
Children who are overweight have the potential to be obese adults when they grow up.
We could infer from the relationship between the KLF5 polymorphism and BMR that the variation of KLF is associated with obesity.
Several studies of diverse ethnic groups, including Pima Indians, Caucasians, and Japanese, have demonstrated that obese individuals with type 2 diabetes have a higher BMR compared to the obese non-diabetic populations [26].
In our result, 58% of children in the high-BMR group were overweight, and the children with the TT mutant homozygote of KLF5 came under the high-BMR group.
Recently, the predicting equations for BMR have been verified in many populations [12, 27-29].
To cross-validate three predicting equations for BMR established by the Schofield database, Henry, and Cole, Ramirez-Zea M used mean values for age, weight, height, and BMR.
These findings suggested that the equation seemed to be appropriate to estimate BMR in men only, not women [28].
Another study indicated that a new equation may be more appropriate for calculating BMR using age, height, weight, and fat in obese children [27].
In our equation, we determined a BMR equation for children between the ages of 10 and 13 years old based on age, muscle, RI, tricep, WHR, and fat mass.
Our predicting equation in children was similar to the previous studies.
In the normal-weight group, we observed that the variables of age, weight, muscle, RI, glucose, and insulin were significant predictors of BMR, which accounted for 100% of the variation in BMR.
However, the final model of our equation for BMR in the overweight group included both weight and muscle (data not shown).
On the other hand, we divided them into 2 groups: low- and high-BMR group.
In the high-BMR group, we established the predictive equation using muscle, fat mass, RI, and C-peptide.
We observed an inverse correlation between BMR and RI in the high-BMR group, and the children with the TT genotype belonged to the high-BMR group.
These results indicated that the children with the TT genotype have a higher BMR compared with the children with the CC genotype.
Although the mean insulin value was higher in the children with the TT genotype than those with the CC genotype, there was no significant association between HOMA-IR and TT genotype in our subjects.
We postulated that the relation of insulin and HOMA-IR may contribute to insulin resistance (IR) in our children.
IR was calculated according to HOMA using fasting insulin and glucose concentrations [30].
Children who were overweight have a higher morbidity rate of obesity than children with normal weight.
Obese adults have more IR than normal, which may predispose them to diabetes [30].
Obesity-induced IR is the major contributor of metabolic syndrome, and both obesity and IR are components of metabolic syndrome [31, 32].
However, we could not observe IR.
Also, the insulin value and HOMA-IR were not involved in the equation of the overweight group.
In the high-BMR group only, we observed that the higher C-peptide affected the increase of BMR in children with the TT genotype.
Therefore, obese children with the TT genotype may have no potential for becoming obese adults with metabolic syndrome.
There are several limitations in this study.
First of all, a sample size of this study was too small, because 1) it was necessary to use a direct calorimeter for measuring RMR, but the device and its expendables, such as mouthpieces, were too expensive and not cost-effective for our study.
If the children were unable to reach steady-state oxygen consumption in the allotted time of approximately 10 min, the children repeated numerous times after resting, and it was not easy for younger participants.
2) Parent&#39;s signed permission for these children was mandatory at the first visit, but several parents and children rejected informed consent.
3) It was not sufficient to obtain the blood samples of all children for analyzing the genetic polymorphism of KLF5, and there was a genotyping error sometimes.
Second, we could not find out the citations to explain the association between the KLF5 polymorphism and BMR.
There is abundant satisfactory research showing the predictive equation for BMR in several populations.
Although we discovered the studies to suggest the genes involved in the regulation of energy expenditure metabolism, including UCP1, there was not enough research to demonstrate the association of KLF5 and BMR in this study.
In this study, we found that the TT homozygote of KLF5 plays a crucial role for increasing BMR and fat mass in Korean children.
We could make better use of the novel variations within KLF5 in a future clinical intervention study of obesity.
Correlation of basal metabolic rate (BMR) and clinical variables according to genotype distribution of rs3782933.
Resting metabolic rate (RMR) and VO2 were positively correlated with BMR in only the CC genotype in contrast to the CT or TT genotypes (A and B); meanwhile, body mass index (BMI) and muscle were related with BMR in CC, CT, and TT genotypes (C and D).
Correlation of basal metabolic rate (BMR) and clinical variables according to genotype distribution of rs3782933.
Insulin and C-peptide were correlated with BMR in only the CC genotype in contrast to the CT or TT genotype (A and B); meanwhile, cholesterol was not associated with BMR regardless of genotypes (C).
Title: Genotype and allele distribution of KLF5 rs3782933 SNP in children (n = 62)

Title: Anthropometric and clinical characteristics of elementary school children according to allele and genotype of rs3782933

Title: Anthropometric and clinical characteristics of elementary school children according to allele and genotype of rs3782933

Title: Stepwise multiple regression models for predicting BMR in children
Title: Phylogeny of Flavobacteria Group Isolated from Freshwater Using Multilocus Sequencing Analysis

Sequence analysis of the 16S rRNA gene has been widely used for the classification of microorganisms.
However, we have been unable to clearly identify five Flavobacterium species isolated from a freshwater by using the gene as a single marker, because the evolutionary history is incomplete and the pace of DNA substitutions is relatively rapid in the bacteria.
In this study, we tried to classify Flavobacterium species through multilocus sequence analysis (MLSA), which is a practical and reliable technique for the identification or classification of bacteria.
The five Flavobacterium species isolated from freshwater and 37 other strains were classified based on six housekeeping genes: gyrB, dnaK, tuf, murG, atpA, and glyA.
The genes were amplified by PCR and subjected to DNA sequencing.
Based on the combined DNA sequence (4,412 bp) of the six housekeeping genes, we analyzed the phylogenetic relationship among the Flavobacterium species.
The results indicated that MLSA, based on the six housekeeping genes, is a trustworthy method for the identification of closely related Flavobacterium species.
Flavobacterium is a genus of gram-negative bacteria belonging to the family Flavobacteriaceae (order: Flavobacteriales, class: Flavobacteria, phylum: Bacteroidetes) [1].
The term Flavobacterium was first introduced by Bergey et al.in 1923 [1].
Since then, a number of Flavobacterium species have been identified, and the classification of the genus has been continuously amended.
At present, it is widely accepted that the genus Flavobacterium contains bacteria that are yellow-pigmented, aerobic, and generally rod-shaped [2], and some Flavobacterium species mobilize through a gliding motility mechanism [3].
Flavobacterium strains could be isolated from various habitats, including freshwater, soil, plants, and food [3, 4].
The 16S rRNA gene has been widely used in many studies concerned with the identification or classification of microorganisms.
However, using only one housekeeping gene as a single marker is not enough to distinguish some bacterial species, especially when they are closely related.
In addition, gene-specific changes caused by horizontal gene transfer or recombination could be an obstacle to the phylogenetic studies that are based on a single gene marker [5, 6].
To overcome the limitation, multilocus sequence analysis (MLSA) has been proposed as an alternative method in many studies.
In reality, MLSA is a microbial genotyping method used to characterize a variety of microorganisms, including the genera of Pantoea [7], Bifidobacterium [8], Borrelia [9], Bradyrhizobium [10], and Bacteroides [11].
In MLSA, approximately seven housekeeping genes are required, but a standard for the given number and size of genes has not been established.
This concise approach is based on concatenated sequences of multiple housekeeping genes, uses the concatenated sequences to identify the phylogenetic aspects of desired species, and investigates the patterns of genotypes between each species cluster [12].
This study was designed to classify the genetic diversity of collected Flavobacterium species.
In addition, we also investigated their relationship using phylogenetic analysis and compared the results for phylogenetic markers.
Title: Bacterial isolates

Seven Flavobacterium isolates were collected.
The two Flavobacterium reference strains, F. aquatile ATCC 11947 and F. frigidimaris F3-JCM12218, were purchased from the Japan Collection of Microorganisms (JCM), and five Flavobacterium isolates-F. koreense ARSA-15, F. koreense ARSA-19, F. koreense ARSA-42, F. chungnamense ARSA-103, and F. chungnamense ARSA-108-were previously isolated from the Cheon-ho reservoir in the Korea [13, 14].
Title: Bacterial growth and genomic DNA isolation

This seven strains were cultured on R2A medium (Difco, Detroit, MI, USA) for 5 days at 25℃ and produced creamyellow, semi-transparent colonies.
We extracted genomic DNA from the seven Flavobacterium strains using the Wizard genomic DNA purification kit (Promega, Madison, WI, USA), according to the manufacturer&#39;s instruction.
Title: Polymerase chain reaction

Through a PCR assay, we amplified seven housekeeping genes from the bacteria: 16S rRNA (16S ribosomal RNA) gene, atpA (ATP synthase alpha subunit), dnaK (molecular chaperone protein), glyA (serine hydroxy-methyltransferase), gyrB (DNA gyrase, subunit B), murG (N-acetylglucosaminyl transferase), and tuf (elongation factor Tu).
For the PCR amplification of the genes, we used primers designed in a previous study [15].
However, we failed to amplify the dnaK and atpA genes through PCR with the previously designed primers.
Thus, we manually designed new PCR primers to amplify these two genes.
The primers are listed in Table 1.
PCR amplification of each gene was performed in a 25-µL reaction volume using 10 µL of a DNA polymerase kit (TaKaRa EmeraldAmp GT PCR Master Mix; TaKaRa, Ohtsu, Japan), 200 nM of each oligonucleotide primer, and 50 ng of template DNA.
The PCR conditions were an initial denaturation at 95℃ for 5 min; followed by 35 cycles of 30 s of denaturation at 95℃, 30 s of annealing at 55℃, and 2 min of extension at 72℃; and followed by 5 min of a final extension at 72℃.
Two microliters of the resulting PCR products was loaded on a 1% agarose gel, stained with ethidium bromide, and visualized using UV fluorescence.
The remaining PCR products were purified with the Wizard SV gel and PCR clean-up kit (Promega, Madison, USA) according to the manufacturer&#39;s instruction and sequenced on an ABI 3130XL automated DNA sequencer (Applied Biosystems, Foster City, CA, USA).
Title: DNA sequences analysis

For each of the seven genes, we aligned the DNA sequences obtained from the seven Flavobacterium isolates using the software BioEdit, version 7.0 (http://www.mbio.ncsu.edu/Bioedit/bioedit.html/).
After that, the housekeeping gene sequences of the 14 added reference strains were collected using BLAST (http://blast.ncbi.nlm.nih.gov/) and aligned in the same way.
Title: Phylogenetic analysis

To construct the phylogenetic relationship of the 21 Flavobacterium strains, we used the molecular evolutionary genetics analysis (MEGA) software package, version 5.05 [16], with the Kimura two-parameter model and bootstrap analyses, based on 1,000 replications.
In addition, we compared the genetic similarity of the six housekeeping genes and the 16S rRNA gene between the genus of Flavobacterium using the pair distance of ClustalW (weighted), which was supported by DNASTAR&#39;s MegAlign program in Lasergene, version 5.0 (http://www.dnastar.com).
The MEGA program was also used to determine the pairwise distances, G+C contents, and substitution positions.
Title: Nucleotide sequence accession numbers

The DNA sequences of the seven amplified Flavobacterium isolates have been deposited in GenBank under accession numbers JX256829-JX256870 (Supplementary Table 1).
In this study, to determine the phylogenetic relationship of the five closely related Flavobacterium species that were previously isolated from a freshwater reservoir in Chonan, Korea [13, 14], we constructed a maximum-liklihood phylogenetic tree, but the tree showed a very low bootstrap value (&lt;25%) between them.
Previously, Nicolas et al.
[15] recommended several housekeeping genes to analyze the phylogenetic relationship among fish-pathogenic F. psychrophilum strains.
This approach is called MLSA, which is very useful for distinguishing closely related species and showing population structure.
We have chosen seven housekeeping genes as phylogenetic markers-atpA, dnaK, glyA, gyrB, murG, tuf (elongation factor Tu), and trpB (tryptophan synthase, subunit B)-because these loci contain the largest amounts of polymorphisms among the 50 F. psychrophilum strains [15].
This study is the first to apply Flavobacterium species isolated from a freshwater reservoir.
First, we tried to amplify seven housekeeping genes from five Flavobacterium isolates [13, 14], which are closely related to each other based on the 16S rRNA gene.
However, we could not amplify the trpB gene from our samples from a freshwater reservoir.
Therefore, we collected the sequences of six housekeeping genes and the 16S rRNA gene from five Flavobacterium isolates [13, 14], isolated from a freshwater reservoir in Cheonan, Korea, and two Flavobacterium reference strains, F. aquatile ATCC 11947 and F. frigidimaris F3-JCM122, as interspecies out-groups.
In addition, we searched for more Flavobacterium species that were available for the six housekeeping genes from the GenBank database.
We found that 52 Flavobacterium species were available for all six housekeeping gene sequences in the database.
Among them, 50 Flavobacterium species were F. psychrophilum that were identified and studied by Nicolas et al.[15].
The 50 F. psychrophila were assigned to 33 sequence types.
Consequently, to evaluate the resolution and efficiency of MLSA, we aligned and trimmed each housekeeping gene sequence (555 bp of gyrB, 680 bp of dnaK, 833 bp of tuf, 725 bp of murG, 796 bp of atpA, and 823 bp of glyA genes) from the five Flavobacterium isolates, 33 distinct F. psychrophila, two other Flavobacterium species (F. branchiophilum FL-15 and F. indicum GPTSA100-9), and two reference strains.
As shown in Fig.1, we constructed two different phylogenetic trees of the Flavobacterium species, based on only 16S rRNA gene sequences (Fig.1A) and based on the concatenated sequences of the six housekeeping genes (Fig.1B).
The bootstrap values of the tree branches clearly improved in Fig.1B compared with those in Fig.1A.
Therefore, we could state that the phylogenetic tree of the genus Flavobacterium, produced by the MLSA approach, is more reliable than the tree based on a single locus (e.g., 16S rRNA gene).
To investigate the nucleotide diversity of each gene among the 10 Flavobacterium species, we analyzed the pairwise distances determined by MEGA5.05 (Table 2).
Among the six housekeeping genes and the 16S rRNA gene, the highest average pairwise distance was 0.303 in murG, while the lowest average pairwise distance was 0.043 in the 16S rRNA gene.
We found that the average pairwise distance within the gyrB, dnaK, tuf, atpA, and glyA genes was 0.218, 0.141, 0.115, 0.150, and 0.215, respectively, among Flavobacterium species.
In addition, we estimated the pairwise distance of the six concatenated housekeeping genes, and thus, the pairwise distance was 0.164 on average.
The pairwise distance of the six concatenated housekeeping genes was significantly greater than the pairwise distance of the 16S rRNA gene (0.043).
Next, we analyzed G + C contents and variable nucleotide sites of six housekeeping genes in our MLSA data and 16S rRNA gene (Table 2).
The G + C contents of the genes ranged from 35.4% (murG) to 48.9% (16S rRNA) (Table 2).
The G + C content among housekeeping genes was found to be variable.
As a result, this study has demonstrated that MLSA, based on multiloci, produces more reliable phylogenetic trees among Flavobacterium species compared with the tree based on a single locus (e.g., 16S rRNA).
When a high number of sequential similarities exist in genetic markers, it might be difficult to identify the phylogenetic relationship between microorganisms of the same genus.
Therefore, MLSA will be a good approach to classify the genetic relationships among microorganisms.
Maximum-likelihood phylogenetic tree showing the phylogenetic relationship of 16S rRNA gene sequences (A) and the concatenated sequences of gyrB-dnaK-tuf-murG-atpA-glyA genes obtained from 42 Flavobacterium species strains and Escherichia coli (B).
Numbers on the branches indicate bootstrap values (shown as percentages of 1,000 replications).
Scale bar, 0.05 substitutions per nucleotide position.
Purple box indicates discordant cluster between two phylogenetic trees.
Title: Primers used for multilocus sequence analysis of the Flavobacterim species in this study

Title: Average pairwise distances, calculated using Kimura two-parameter model, and average G + C content of the seven housekeeping genes in this study
Title: Forensic Body Fluid Identification by Analysis of Multiple RNA Markers Using NanoString Technology

RNA analysis has become a reliable method of body fluid identification for forensic use.
Previously, we developed a combination of four multiplex quantitative PCR (qRT-PCR) probes to discriminate four different body fluids (blood, semen, saliva, and vaginal secretion).
While those makers successfully identified most body fluid samples, there were some cases of false positive and negative identification.
To improve the accuracy of the identification further, we tried to use multiple markers per body fluid and adopted the NanoString nCounter system instead of a multiplex qRT-PCR system.
After measuring tens of RNA markers, we evaluated the accuracy of each marker for body fluid identification.
For body fluids, such as blood and semen, each body fluid-specific marker was accurate enough for perfect identification.
However, for saliva and vaginal secretion, no single marker was perfect.
Thus, we designed a logistic regression model with multiple markers for saliva and vaginal secretion and achieved almost perfect identification.
In conclusion, the NanoString nCounter is an efficient platform for measuring multiple RNA markers per body fluid and will be useful for forensic RNA analysis.
RNA analysis has emerged as a reliable method of body fluid identification for forensic use [1-7].
Conventionally, immunological, enzymatic, and chemical detection of specific protein markers is used [8-10].
For example, prostate-specific antigen has been used for semen and hemoglobin for blood identification [9, 10].
These assays provide important information for crime scenes, but they can provide wrong information because of cross-reactions.
Recently, several body fluid-specific mRNA markers have been discovered [1-7]: β-spectrin (SPTB), porphobilinogen deaminase (PBGD), and hemoglobin alpha locus 1 (HBA1) for blood; matrix metalloproteinase 7 and 11 (MMP7 and MMP11) for menstrual blood; statherin (STATH) and histatin 3 (HTN3) for saliva; kallikrein 3 (KLK3) and protamine 1 and 2 (PRM1 and PRM2) for semen; and human beta-defensin 1 (HBD-1) and mucin 4 (MUC4) for vaginal secretion [11].
After DNA/RNA co-extraction methods were applied to the forensic field, RNA analysis has become routine forensic analysis [12, 13].
RNA expression level is generally measured using reverse-transcription and standard end-point PCR (RT-PCR) or quantitative PCR (qRT-PCR) in a forensic lab.
Recently, using microarray platforms, body fluid-specific mRNA markers were identified at a genome-wide level [14-16], and multiplex qRT-PCR probes of these markers were developed for one-step identification of body fluid type [7, 16].
Currently, most of the multiplex qRT-PCR assays use a single marker for each body fluid.
However, using a single marker for each body fluid may lead to false positive/negative identification, as some markers are expressed in more than two types of body fluid [4-6].
Recently, Roeder and Haas [17] suggested a novel approach of using a minimum of five mRNA markers for each body fluid and a scoring method based on multiple markers.
In this regard, the multiplex qRT-PCR system is not useful, because the number of fluorescent dyes in one reaction is limited.
The NanoString nCounter (NanoString Technologies, Seattle, WA, USA) is a recent platform that can quantify the expression of hundreds of mRNAs in a single reaction using color-coded molecular barcodes [18, 19].
NanoString nCounter is also known as a sufficiently robust method to measure expression in degraded RNA samples, such as formalin-fixed paraffin-embedded tissues and crude tissue lysates [18, 20].
Therefore, applying NanoString nCounter to forensic identification of body fluids enables the digital quantification of multiplexed markers for each body fluid.
In this study, we adopted the NanoString nCounter and designed a panel of NanoString probes for multiple markers for each body fluid.
We tested the multiple markers in a large number of body fluid samples.
As a result, we found that the new multiplexed method could improve the specificity and sensitivity of identification.
We suggest that using multiple mRNA markers for each body fluid could improve the accuracy of body fluid identification.
Title: Sample collection and RNA preparation

Twelve samples for each body fluid (blood, saliva, semen, and vaginal secretion) were collected from healthy Korean volunteers with informed consent from the participants.
The study protocol was approved by the Institutional Review Boards of Chungnam National University Hospital.
To prepare total RNA, we employed the Qiagen RNeasy Mini kit (Qiagen, Hilden, Germany) according to the manufacturer&#39;s protocol.
Extracted total RNA was analyzed using Experion RNA StdSens (Bio-Rad, Hercules, CA, USA) to check its quality and quantity.
Title: NanoString experiment

To validate the mRNA candidates as body fluid specific markers, NanoString technology was employed.
Color-coded barcodes that represented a single target mRNA were synthesized, targeting 18 body fluid-specific mRNA markers, and 2 endogenous controls for mRNA.
Briefly, 100 ng of total RNA was hybridized to the barcode, and then probe-mRNA complexes were immobilized on a streptavidin-coated cartridge according to the manufacturer&#39;s protocol.
Subsequently, the cartridges were placed in the digital analyzer, and barcodes were counted.
All the counts were normalized by the count of GAPDH barcodes.
Title: Statistical analysis

We applied student&#39;s t-test to evaluate the significance of gene expression differences between the tissue of interest and the other three tissues among the four body fluids.
We applied receive operating characteristic (ROC) analysis to estimate the sensitivity and specificity of each marker, using the ROCR package [21] of R software (version 2.6.1).
Again, the four body fluids were divided into two groups: a tissue of interest and the remaining three tissues among the four body fluids.
Results with a p-value of &lt; 0.05 were considered significant.
Previously, we found a dozen mRNA markers and developed multiplex qRT-PCR probes for body fluid identification [16].
But, most of the markers, except pro-platelet basic protein (PPBP), were not perfect in terms of area under the curve (AUC) value (AUC, 1).
To improve the accuracy of identification, we considered a multiplexed assay using multiple probes for each body fluid.
While the multiplex qRT-PCR system is limited by the number of available dyes in a reaction, the NanoString nCounter can measure the expression of hundreds of RNAs at once [18].
So, we adopted the NanoString nCounter as the multiplexed identification method using multiple RNA markers for each body fluid.
After designing NanoString probes for 18 mRNA markers (four to five for each body fluid and two for controls), we performed NanoString nCounter assays with a total of 12 RNA samples for each body fluid.
As a result, we got a dozen good probes (Fig.1) according to the following criteria: 1) p &lt; 0.005 and 2) body fluid-specific expression pattern.
Results of the other probes are shown in Supplementary Fig.1.
To test the value of each marker as a body fluid-specific marker, we performed ROC analysis (Fig.2).
For blood- or semen-specific markers, the identification accuracy was perfect (AUC, 1).
But, for saliva and vaginal secretion, no single marker showed perfect accuracy (Fig.2).
To improve the accuracy of saliva and vaginal secretion identification, we tried a combination of markers using logistic regression analysis [22].
We evaluated the sensitivity and specificity of each marker and then evaluated the performance of the combined markers (Fig.3).
For saliva and vaginal identification, using two markers significantly improved the identification accuracy (Table 1).
Therefore, we could identify each body fluid with multiple makers for each body fluid using the NanoString nCounter quite accurately.
Previously, we developed multiplex qRT-PCR probes that used one specific mRNA marker for each body fluid [16].
In that work, blood was perfectly identified using a single marker, PPBP.
However, for other body fluids, such as vaginal secretion and saliva, it was difficult to find out markers that perfectly discriminated them from other body fluids.
For example, as saliva and vaginal secretion showed similar expression patterns, many selected body fluid-specific markers were expressed in both body fluids.
This problem led us to consider a combination of multiple markers for perfect identification.
A limit in the number of markers that can be assessed in a reaction was the biggest problem when we considered the use of multiple markers for each body fluid.
As the number of different fluorescent dyes available in a qRT-PCR reaction is limited (i.e., five), we considered a new method that can measure at least 10 RNAs at once.
Fortunately, the recently developed NanoString nCounter allowed us to measure hundreds of probes in a single reaction; so, we adopted the NanoString platform in our body fluid identification project.
Measuring the expression of multiple makers per body fluid has a big advantage.
As we have shown, by using two markers for each body fluid, we could identify saliva and vaginal secretions that were not identified perfectly by a single marker.
As we could identify blood and semen perfectly using only one marker per body fluid, we conclude that a total of 10 markers is enough to identify four different body fluids in one reaction.
In this regard, the NanoString nCounter is a promising platform that allows us to identify each body fluid in one reaction with high accuracy.
Expression patterns of selected mRNA markers in our NanoString nCounter assay dataset.
Each count was normalized to GAPDH mRNA.
(A) Blood.
(B) Saliva.
(C) Semen.
(D) Vaginal.
Receive operating characteristic analysis of selected mRNA markers in our NanoString nCounter assay dataset.
AUC, area under the curve.
(A) Blood.
(B) Saliva.
(C) Semen.
(D) Vaginal.
Workflow for the selection of multiple mRNA markers.
AUC, area under the curve.
Title: Performance of body fluid identification using multiple markers
Title: Molecular Vibration-Activity Relationship in the Agonism of Adenosine Receptors

The molecular vibration-activity relationship in the receptor-ligand interaction of adenosine receptors was investigated by structure similarity, molecular vibration, and hierarchical clustering in a dataset of 46 ligands of adenosine receptors.
The resulting dendrogram was compared with those of another kind of fingerprint or descriptor.
The dendrogram result produced by corralled intensity of molecular vibrational frequency outperformed four other analyses in the current study of adenosine receptor agonism and antagonism.
The tree that was produced by clustering analysis of molecular vibration patterns showed its potential for the functional classification of adenosine receptor ligands.
Membrane proteins play crucial roles in organismal transduction of information.
Molecular recognition in biological membrane systems is the initial process triggering external factors to be passed into the inner part of a cell.
G protein-coupled receptors (GPCRs), which consist of 7 transmembrane segments, play pivotal roles for signal transduction in higher eukaryotes and organize the largest families of proteins in the human genome [1].
The fact that drugs targeting GPCRs constitute over one-third of pharmaceuticals shows the importance of GPCRs as drug targets [2].
Adenosine receptors (AdoRs) belong to a family of rhodopsin-like class A GPCRs, which constitute the largest portion of GPCRs in humans, including olfactory receptors, and there are four known subtypes of AdoRs-namely, AdoR1, AdoRA2A, AdoRA2B, and AdoR3 [3, 4].
Although these AdoR subtypes have distinct amino acid sequences and tissue-specific distribution, their endogenous agonist and ligand-binding pockets are highly conserved.
The biogenic nucleoside adenosine (Fig.1) exerts its effects by interacting with AdoRs that are involved in various diseases, such as cardiac ischemia, arrhythmia, neurodegeneration, diabetes, glaucoma, and inflammation [5].
The physiological effects of AdoRs can be used for diagnoses and planning of the surgical strategies, such as in coronary artery diseases [6].
Olfaction, the sense of smell, is also mediated by class A GPCRs-namely, olfactory receptors.
The fundamental mechanism of olfaction is now under some controversy [7-9].
Various attempts were introduced to describe the molecular mechanism of receptor-ligand recognition in olfaction, such as the binding theory and vibration theory [10-12].
The former, where the specificity of ligand is explained by molecular shape, has been developed into a concept of a pharmacophore and is generally accepted by researchers.
Nevertheless, this is not sufficient to give the account of ligand variety and agonism complexity of GPCRs.
In recent decades, there have been many efforts to find an efficient way to seek or make appropriate ligands working on the target receptors through a chemogenomic approach.
Application of molecular descriptors can make an efficient way to discriminate ligand binding modes in receptor-ligand relationships in such cases as in silico chemogenomic screening [13].
Molecular descriptors encoding information about the molecular structure can be classified by the dimensionality of their molecular representation, and they are almost on the structural, topological, and geometrical bases [14].
They are also including information on dipole moment, electric polarizability, and electrostatic potential in their data types.
Lately, a computational approach was carried out to search for a molecular vibration-activity relationship in the agonism of histamine receptors, and the author suggested that the molecular vibrational frequency pattern may play a role as a possible molecular descriptor for histamine receptor ligands [15].
The EigenVAlue (EVA) descriptor is also based on infrared range molecular vibrations among the various molecular descriptors and is a unique one, based on eigenvalues corresponding to individual calculated normal modes among various molecular descriptors [16, 17].
In the present paper, we tested the potential of the corralled intensity of molecular vibrational frequency (CIMVF) [15] as a molecular descriptor, compared to the EVA descriptor and other descriptors for the classification of AdoR ligands.
Title: Dataset

All 46 ligand molecules in the dataset, comprising 23 AdoR agonists and 23 antagonists, are shown in Table 1, with the Tanimoto distance of each ligand from adenosine.
The Tanimoto coefficient is a widely used measure of molecular structural similarity.
The coefficient is defined as Tc = Nab/(Na + Nb - Nab), with Nab being the number of common bits, Na the unique bits in molecule a, and Nb the unique bits in molecule b, using a molecular fingerprint [18].
In this study, the molecular similarity to adenosine was calculated as the Tanimoto coefficient using the 38-bit set.
The simplified molecular-input line-entry system (SMILES) and 3-dimensional structure data format (SDF) files of the dataset were downloaded from the PubChem Compound Database in National Center for Biotechnology Information (NCBI) and used in further analyses.
Title: Molecular descriptors

Title: Molecular fingerprints: MACCS keys, PubChem, and Klekota-Roth fingerprint

Molecular fingerprints are binary bit string representations that capture diverse aspects of molecular structure and properties and are popular tools for virtual screening [19].
In this study, three fingerprints-MACCS keys [20], PubChem fingerprint [21], and Klekota-Roth fingerprint [22]-were tested to be compared with the Tanimoto coefficient and other descriptors.
The descriptor numbers of the MACCS, PubChem, and Klekota-Roth fingerprints are 320, 881, and 4860, respectively.
The MACCS keys of each ligand were generated by a MACCS key generator, and the other two fingerprints of the ligands were calculated by PaDEL-Descriptor [23].
Title: EVA descriptor

An ideal molecular descriptor should encode all the features of a molecule in numerical form.
EVA, based on the normal coordinate eigenvalues, is derived from calculated infrared range vibrational frequencies and consequently has the characteristic feature of molecular-specific vibration.
The resulting vibrational frequencies were then convolved using a sum of Gaussian functions to generate a pseudospectrum of 3N-6 overlapping kernels: , where N is the number of atoms, σ is a fixed standard deviation of the Gaussian function, and fi is the i-th frequency of the molecule.
We fixed the value of σ to 5 cm-1 in this study.
Title: CIMVF descriptor

For a simplified comparison of the molecular vibration patterns, the calculated molecular vibrations of a ligand were sorted in increasing order and taken into each corral with a fixed step (e.g., 5 cm-1) size.
The intensities of each molecular vibrational frequency in the same corral were summed up as the representative of the corral.
As a molecular descriptor of a ligand, the intensity of each corral is displayed in a 1-dimensional vector containing 800 elements from the vibrational frequency range of 0-4,000 cm-1.
The calculations of CIMVF were performed by in-house scripts written in Python.
EVA and CIMVF descriptors were produced from PubChem SDF data of ligands through geometry optimization and calculations of molecular vibrations with the GAMESS program package [24].
The similarity matrices obtained from these five cases of molecular representations were subjected to hierarchical clustering using complete linkage method.
Title: Calculation of molecular vibration

Since the EVA and CIMVF descriptors require geometry optimization of a given molecule, each provided theoretical 3-D conformer SDF underwent single low-energy conformation using the GAMESS program package [24].
Restricted Hartree-Fock calculations using the BLYP DFT method with the 6-31G basis set were performed to optimize the geometries of the molecules.
Each result was taken as the representative conformation of the molecule, although the calculation of molecular vibrational frequency has some dependence on conformation.
The results of geometry optimization were subjected to the calculation step for the molecular vibrational frequency with RUNTYP of HESSIAN in the GAMESS program.
Title: Hierarchical clustering of molecular descriptors and dendrogram structure comparison

To test the availability of CIMVF as a molecular descriptor for the classification of AdoR ligands, a kind of molecular calculation using agglomerative hierarchical clustering was adopted in this work.
Each fingerprint or descriptor of ligands was gathered into a matrix of cognate fingerprint or descriptor.
Finally, the similarity matrix, comprising descriptors of 46 ligands of AdoRs, was then subjected to hierarchical clustering in the agglomerative manner.
In this study, each similarity matrix was finally clustered to make a dendrogram of 46 vertices.
To compare the structures of the resulting dendrogram, we used a program for pairwise comparison of phylogenies, which shows the topological difference between two dendrograms [25].
The multiple comparison of the 5 dendrograms was also carried out using a meta-tree-generating program for comparing multiple alternative dendrograms [26].
As shown in Fig.1, adenosine and caffeine, a typical agonist and antagonist of AdoRs, respectively, share the structure of purine.
However, the Tanimoto coefficient between the two ligands is not high.
The Tanimoto coefficient of some agonists to adenosine was lower than 0.3, though they did not take a large share.
Moreover, the lowest Tanimoto coefficient of the agonists was 0.12766 (capadenoson) and is a lower value as an antagonist.
The results of hierarchical clustering of the similarity matrices from the MACCS keys, PubChem, and Klekota-Roth fingerprints; EVA; and CIMVF are shown in Fig.2.
The dendrograms obtained from MACCS keys and PubChem fingerprint were shown to be similar to the Tanimoto coefficient pattern.
The result of the Klekota-Roth fingerprint analysis showed a somewhat different pattern to the fingerprint-based one, as only two antagonists were located in the cluster of agonists.
However, the three fingerprint-based results did not match the known facts of AdoR agonism.
In the case of EVA, Takane and Mitchell [27] reported a structure-odor relationship using the EVA descriptor and hierarchical clustering.
The authors reported that the dendrograms that were produced by the EVA method outperformed those from UNITY 2D in a classification of odorant molecules.
However, the EVA analysis in the current study was positive but not perfect for the molecular property-activity relationship in AdoR agonism.
The larger σ got, the more the clusters in the dendrogram got entangled (data not shown).
It might come from the Gaussian function that smeared out the vibrational frequencies, such that vibrations of similar frequency overlapped together.
The dendrogram obtained from the EVA descriptor showed an eligible tree for the agonism of AdoRs on the whole (Fig.2D).
One exception in which an antagonist (ZM241385) was clustered in a subtree of agonists might have resulted from smoothing by the Gaussian convolution of molecular vibration data.
All antagonists were clustered into a subtree in the dendrogram obtained from CIMVF analysis, as shown in Fig.2E.
We can tell the regional difference between agonists and antagonists in the dendrogram and also find that the information from the CIMVF analysis can play a role in the discrimination of agonists from antagonists in AdoR agonism.
The discreteness of CIMVF data seems to facilitate the performance of the binary classification of AdoR ligands.
To find the topological difference between dendrograms, similarity scores between every two dendrograms were shown in Table 2.
The similarity between two dendrograms was represented as a percentage.
For example, the score of topological similarity between the two dendrograms of CIMVF and EVA analyses is 59.6%.
The value has the highest similarity among all comparisons.
A distance meta-tree build from five dendrograms of 46 AdoR ligands was illustrated in Fig.3.
As shown in the tree, the resulting dendrogram produced from the CIMVF data analysis has the nearest position to that produced from the EVA data analysis and is located the farthest from the dendrogram produced from the analysis of MACCS keys.
It is generally known that structurally similar molecules have similar properties or functions; however, a small change in molecular structure can fairly affect its vibrational frequencies.
The structural variety of AdoR ligands was shown in Fig.4.
The Tanimoto coefficients of LUF5835, BAY60-6583, and capadenoson to adenosine were smaller than 0.18, and the ligands share a hydroxyphenyl pyridine dicarbonitrile backbone (Fig.4A).
They seem to be dissimilar to other agonists that share the structure of purine.
Several antagonists (Fig.4B) do not have a similar structure to caffeine; however, they act as antagonists of AdoRs.
As mentioned above, the current experiment does provide circumstantial evidence for the molecular vibrational information to AdoR agonism, at least.
With a more concentrated study on the relationship between the molecular vibrational frequency and pharmacological function of a ligand, the vibrational spectrum of a ligand molecule may explicitly propose a novel path to the field of receptor-ligand interaction mechanisms.
The chemical structures of adenosine and caffeine as an agonist and an antagonist of adenosine receptors, respectively.
(A) Adenosine, an endogenous agonist of adenosine receptors.
(B) Caffeine, a typical antagonist of adenosine receptors.
Dendrograms of clustering using fingerprints, EigenVAlue (EVA), and corralled intensity of molecular vibrational frequency (CIMVF) by complete linkage method.
Antagonists were tagged with &#34;t_&#34; to their chemical names as a prefix.
(A) MACCS keys.
Agonists with low Tanimoto coefficient to adenosine were clustered to a subtree of antagonists.
(B) PubChem fingerprint.
The clustering result of the PubChem fingerprint was similar to that of MACCS clustering.
(C) Klekota-Roth fingerprint.
Some antagonists were located in a subtree of agonists.
(D) EVA.
About half of the antagonists were located in the cluster of agonists.
An antagonist, ZM241385, was clustered in the agonist subtree.
(E) CIMVF.
All agonists and antagonists were categorized to each cluster.
Meta-tree build from multiple comparison of 5 dendrograms.
CIMVF, corralled intensity of molecular vibrational frequency; EVA, EigenVAlue.
Structural variety of adenosine receptors (AdoR) ligands (Tani: Tanimoto coefficient to adenosine).
(A) AdoR agonists.
(B) AdoR antagonists.
Title: Dataset of adenosine receptor ligands used in the present study and their molecular similarity in the Tanimoto coefficient to adenosine

Title: Topological difference between dendrograms in Fig.2
Title: hpvPDB: An Online Proteome Reserve for Human Papillomavirus

Human papillomavirus (HPV) infection is the leading cause of cancer mortality among women worldwide.
The molecular understanding of HPV proteins has significant connotation for understanding their intrusion in the host and designing novel protein vaccines and anti-viral agents, etc.
Genomic, proteomic, structural, and disease-related information on HPV is available on the web; yet, with trivial annotations and more so, it is not well customized for data analysis, host-pathogen interaction, strain-disease association, drug designing, and sequence analysis, etc.
We attempted to design an online reserve with comprehensive information on HPV for the end users desiring the same.
The Human Papillomavirus Proteome Database (hpvPDB) domiciles proteomic and genomic information on 150 HPV strains sequenced to date.
Simultaneous easy expandability and retrieval of the strain-specific data, with a provision for sequence analysis and exploration potential of predicted structures, and easy access for curation and annotation through a range of search options at one platform are a few of its important features.
Affluent information in this reserve could be of help for researchers involved in structural virology, cancer research, drug discovery, and vaccine design.
Human papillomavirus (HPV), a virus from the papillomavirus family, is capable of infecting humans.
About 200 different strains of HPV identified, based on DNA homology, have been found to be etiologically linked to cervical, vaginal, vulvar, penile, anal, oral, and plantar infectious lesions and cancers, as well [1, 2].
The HPV genome, a double-stranded DNA molecule, consists of 8 kilobase pairs (kbp) of nucleotides, which comprises 3 regions: 6 early open reading frames (ORFs)-E1, E2, E4, E5, E6, and E7; 2 late ORFs-L1 and L2; and an upstream regulatory region [3].
A considerable volume of HPV specific information pertaining to its genome, proteome, structure, and disease association is available scattered on the web with trivial annotations; however, it is not customized to explore for data analysis, host-pathogen interaction, strain-disease association, drug designing, and sequence analysis, etc.
Therefore, we proposed to develop a comprehensive reserve on HPV with maximum possible inputs and outputs for the end users.
Title: Data retrieval and curation

Amongst the existing 200 strains of HPV, 150 have been sequenced as of now, and their data available at the National Center for Biotechnology Information (NCBI).
Genome and proteome information of those viral strains was retrieved from NCBI.
Besides PubMed, various other online resources and published literature were also explored for validating genomic, proteomic, as well as strain and disease-associated information on HPV strains.
HPV strain-specific information, such as strain name, sequencing status, sequencing centre, NCBI accession ID, associated disease information with references, genome statistics (GC%, AT%, A, T, G, C count, genes, and proteins), etc., were curated from various online resources, and protein parameters (length, molecular weight, isoelectric point) were calculated using ExPASy ProtParam [4].
Title: Protein structure prediction and validation

MODELLER9v10 [5] and the SWISS-MODEL [6] server were used for protein structure prediction.
The stereochemistry of each protein was evaluated through PROCHECK [7] analysis, available at the RCSB validation server (http://deposit.rcsb.org/validate/), and validated using ProSA-web [8] (http://prosa.services.came.sbg.ac.at/prosa.php).
Title: Reserve architecture and design

Human Papillomavirus Proteome Database (hpvPDB), the relational reserve, was developed using Microsoft SQL Server 2005 as the back end.
The website is powered by XAMPP (Windows Version 1.7.3).
HTML, JavaScript, and CGI-PERL-based web interfaces were employed to execute SQL queries.
The curated data and related information were stored in tables.
The application layer, the web interface, and the backend relational tables were integrated using CGI-PERL.
The overall architecture of hpvPDB is shown in Fig.1.
Title: Reserve features

hpvPDB interfaces are made to help the users for easy navigation and information retrieval.
Home, About, Tools, Search, and Advanced Search interfaces can be explored to obtain strain- and protein-specific information.
User can access the meta information about different strains using a search box.
Reserve comprises the strain-specific detailed informationon its name, sequencing status, submission details, date of submission, NCBI IDs, disease types and subtypes, type of DNA, genome length, molecular weight, nucleotide composition (A, T, G, C, AT, GC content), number of genes and proteins, and protein list.
A genome map of each strain obtained by Geneious 5.4.4 software (available from http://www.geneious.com/) is also integrated in this page.
Users, through an advanced search option, can precisely access the Genome and Proteome information separately by selecting HPV genome or HPV proteome.
Each protein entry comprises protein overview (name, locus, function, etc.
), protein sequence information (amino acid sequences with NCBI accession number with provision for direct protein BLAST [9] against NCBI nr database), protein parameters (length, molecular weight, theoretical isoelectric point [pI], amino acid composition, etc.
), protein structure (predicted 3D structure by homology modeling viewed by Jmol (available from http://www.jmol.org/) [10] with the JAVA platform, Ramachandran plot obtained by PROCHECK and Z-score and Energy plot obtained by ProSA-Web.
hpvPDB platform also provides a phylogeny analysis tool to perform multiple sequence alignment and phylogenetic tree construction of selected HPV proteins using the Phylogeny.
fr web service [11].
The original Human Papillomaviruses Database was developed and hosted by the Los Alamos National Laboratory (LANL) between 1994 and 1999 with funding from the National Institute of Allergy and Infectious Diseases (NIAID) [12].
&#39;Human Papillomaviruses: A Compilation and Analysis of Nucleic Acid and Amino Acid Sequences&#39; contains four annual data books of papillomavirus information published in both paper and electronic form (1994, 1995, 1996, and 1997) but has not been updated since 1997 [12].
This contains nucleotide sequences of few HPV strains and other papillomaviruses, amino acid and nucleotide sequence alignments, analysis, related host sequences, and database communication.
We did not find any structural information in that database.
In hpvPDB, along with updated protein sequence information, genome and protein structure information is also provided.
hpvPDB brings together comprehensive information on a total of 1,036 protein sequences and 743 predicted structures.
The outcome of this study might provide a platform for simultaneous structural comparative analysis of these proteins and help in finding out variations in their structures to explore why different strains of HPV have causative associations with different types of cancers.
Further, this might also help in designing specific drugs or vaccines against specific strains of HPV.
This reserve provides a resource to help virologists identify potential roles for viral protein.
Currently the hpvPDB is updated manually through online resources and available scientific publication review; however, to sustain the quality, these data are analyzed and checked before incorporation into this reserve.
Meanwhile, to provide regular updates, our team is committed to searching for newly sequenced HPV strains, updating this reserve, and serving the users.
System architecture of Human Papillomavirus Proteome Database (hpvPDB) showing data collection, analysis, union, and user interface.
HPV, human papillomavirus; CDS, coding sequence.
Title: Editor&#39;s Introduction to This Issue

This issue contains diverse research topics in genomics and bioinformatics.
Although the information about genetic variations has increased, we still have a limited understanding about the control mechanisms of how the different tissues and cells within an individual have tissue or cell type-specific gene expression.
Regarding this issue, Drs.
Shrutii Sarda and Sridhar Hannenhalli, University of Maryland, review the biological implications of the epigenome and the impact of NGS technology on epigenomics research.
Drs.
Semi Kim and Jung Weon Lee review the epithelial-mesenchymal transition, one of the important mechanisms for understanding the invasion and metastasis of cancer, by focusing on TMPRSS4 and TM4SF5.
Dr. Quang Vinh Nguyen&#39;s group, University of Western Sydney, reports an interesting and useful tool to support interactive visualization, with which we can extract knowledge from complex genetic and clinical data.
This tool supports patient-to-patient analysis from an overview of the patient population in the similarity space to detailed views of genes.
Both Dr. Byeong-Chul Kang&#39;s group, Insilicogen, Inc., and Dr. Byungwook Lee&#39;s group, Korean BioInformation Center, report genetic variation-based population genetics studies.
The Insilicogen group reports a semantic model for ethnic disparity-associated SNPs using HapMap data, and the KOBIC group reports ethnic variant SNPs and copy number polymorphisms in Asians.
To get better accessibility, we changed Genomics &amp; Informatics to an e-Journal.
At this time, three outstanding scientists from Canada, France, and Egypt have joined as new editorial board members.
From this issue, 50 editorial board members from 6 countries across the world will lead this journal.
Genomics &amp; Informatics is already available in PubMed Central and PubMed.
Based on these efforts together, we expect broader international visibility of Genomics &amp; Informatics.
Title: Membrane Proteins Involved in Epithelial-Mesenchymal Transition and Tumor Invasion: Studies on TMPRSS4 and TM4SF5

The epithelial-mesenchymal transition (EMT) is one mechanism by which cells with mesenchymal features can be generated and is a fundamental event in morphogenesis.
Recently, invasion and metastasis of cancer cells from the primary tumor are now thought to be initiated by the developmental process termed the EMT, whereby epithelial cells lose cell polarity and cell-cell interactions, and gain mesenchymal phenotypes with increased migratory and invasive properties.
The EMT is believed to be an important step in metastasis and is implicated in cancer progression, although the influence of the EMT in clinical specimens has been debated.
This review presents the recent results of two cell surface proteins, the functions and underlying mechanisms of which have recently begun to be demonstrated, as novel regulators of the molecular networks that induce the EMT and cancer progression.
Metastasis is the leading cause of cancer-related deaths in most cancer types.
As an initial step in cancer metastasis, epithelial tumor cells in general disseminate from a primary solid tumor mass and invade into the surrounding stromal tissues.
Invasion is enhanced by tumor cell activation of the epithelial-mesenchymal transition (EMT) [1, 2, 3, 4].
The EMT is characterized by the loss of epithelial apicobasal polarity and cell-cell contacts, modulation of cell-matrix adhesion, enhanced proteolytic activity, cytoskeletal remodeling, and acquisition of the ability to migrate and invade the extracellular matrix (ECM) [1, 3].
During the EMT, epithelial cells undergo molecular changes; epithelial cells gradually lose their epithelial markers, such as E-cadherin, ZO-1, and cytokeratins, and concomitantly acquire mesenchymal markers, such as vimentin, fibronectin, N-cadherin, and alpha smooth muscle actin [1, 3].
The EMT plays a critical role in the formation of various tissues and organs, such as the mesoderm, neural crest, heart, secondary palate, and peripheral nervous systems, during embryonic development and wound healing in adult organism [2, 4].
Furthermore, the EMT is implicated in pathological processes, such as tumor cell invasion and metastasis and organ fibrosis [2].
One of the hallmarks of the EMT is the functional loss of E-cadherin, which is currently thought to be a metastasis suppressor [5].
Downregulation of E-cadherin is usually mediated by E-cadherin transcriptional repressors/EMT-inducing transcription factors, including the Snail superfamily of zinc-finger factors (Snail and Slug), the ZEB family (ZEB1 and ZEB2), and basic helix-loop-helix factors (Twist1 and E47), which have been associated with tumor invasion and metastasis [4, 5].
These factors repress transcription of E-cadherin by interacting with proximal E-box elements in the E-cadherin promoter [5].
In addition, these E-cadherin repressors may be directly or indirectly involved in the upregulation of certain mesenchymal genes [5], although the precise mechanism of this regulation is largely unknown.
The EMT is triggered by soluble growth factors, such as members of the transforming growth factor-β (TGFβ) and fibroblast growth factor families, epidermal growth factor, and hepatocyte growth factor (HGF) [3, 4].
Subsequent activation of receptor-mediated signaling triggers the activation of intracellular effector molecules, such as members of the small GTPase family, leading to changes in cytoskeletal organization, and also results in the activation of EMT-inducing transcription factors [3, 4].
In addition, components of the ECM, such as collagen, and activation of integrin co-receptors are also involved in the EMT process [3].
Certain proteases are sufficient to induce the EMT [2]; for example, matrix metalloproteinase-3 triggers the EMT by increasing the cellular levels of reactive oxygen species, which in turn induces Snail expression [6].
Recently, microRNAs (miRs) have been identified as a novel class of EMT regulators; miRs that negatively regulate the EMT include miR-153, -155, -194, -25, -212, and -200, and miRs that positively regulate the EMT include miR-29a, -103/107, -150, and -221/22 [7].
miRs regulate invasion and metastasis by targeting the transcripts of various genes involved in the EMT event, including EMT-inducing transcription factors.
For example, members of the miR-200 family are negative regulators of the EMT and are essential for maintenance of the epithelial status through the downregulation of ZEB1 and ZEB2.
In turn, miR-200 members are transcriptionally repressed by ZEB1 and ZEB2, thus establishing a double-negative feedback loop [8].
The EMT was recently shown to be linked to stemness and self-renewal capacity [9, 10].
In cases of breast cancer stems, the linkage among EMT phenotype, stemness, and drug resistance has been well studied [11].
Further, epithelialmesenchymal plasticity (consisting of EMT and MET) is also described in circulating tumor cells (CTCs) [12, 13, 14].
CTCs with various degrees of EMT phenotypes are found during breast cancer metastasis [15].
Therefore, CTCs may involve self-renewal capacity, which is linked to the EMT, during cancer metastasis [16].
Title: Introduction to type II transmembrane serine proteases

Dysregulation of proteases is a hallmark of cancer progression; thus, proteases in general have been the subject of numerous cancer studies.
Extracellular proteolytic enzymes, including matrix metalloproteinases (MMPs) and serine proteases, contribute to tumor cell invasion and metastasis through both direct proteolytic activity and the regulation of cellular signaling and functions [17, 18, 19].
Most members of the serine protease family are either secreted or sequestered in cytoplasmic organelles awaiting signal-regulated release.
Recently, type II transmembrane serine proteases (TTSPs) have been recognized as a new subfamily of serine proteases that have in common an extracellular proteolytic domain, a single-pass transmembrane domain, a short intracellular domain, and a variable-length stem region containing modular structural domains [20, 21, 22, 23, 24].
Enteropeptidase (also known as enterokinase), identified over a century ago due to its pivotal role in food digestion, is the first TTSP, which was revealed by molecular cloning of the enteropeptidase cDNA two decades ago [25].
TMPRSS2, human airway trypsin-like protease (HAT), corin, and matriptase have been subsequently identified as cell surface-associated proteases [23, 24].
To date, 20 TTSPs have been identified in mouse and humans due to the analysis of sequence data from mammalian genome projects [23].
Analysis of the tissue distribution of TTSPs and gene targeting in mice of certain TTSPs suggested that a significant number of TTSPs may have important functions in embryonic development and homeostasis of mammalian tissues, such as heart, skin, inner ear, placenta, and digestive tract [23, 24].
Most TTSPs are overexpressed in a variety of tumors compared to normal tissues, implicating their potential as novel markers of tumor development and progression and possible molecular targets for anti-cancer therapeutics [23, 26].
Recently, a number of works have focused on the evaluation of the expression of individual TTSPs during tumor progression and on the investigation of the potential roles of these proteases in tumor cell proliferation, migration, and invasion [23, 27].
Title: TMPRSS4 in cancer

TMPRSS4 (gene ID, 56649; chromosomal location, 11q23.3), initially referred to as TMPRSS3, was originally identified as a gene expressed in most pancreatic cancer tissues but not in the normal pancreas or chronic pancreatitis [28].
To date, 7 isoforms have been reported.
The deduced sequence of 437 amino acids of the longest isoform (isoform 1) contains a serine protease domain with putative trypsin-like activity and a transmembrane domain [28].
In human, TMPRSS4 mRNA was detected in bladder, esophagus, stomach, small intestine, colon, and kidney [28], although the physiological roles of TMPRSS4 remain unknown.
Furthermore, TMPRSS4 expression was upregulated in malignant compared to benign thyroid neoplasm and was suggested as both a diagnostic and prognostic marker [29, 30].
TMPRSS4 was associated with poor prognosis in non-small-cell lung cancer (NSCLC) with squamous cell histology [31], triple-negative breast cancer [32], cervical cancer [33], and gastric cancer patients [34].
Kim et al.
[35] reported that TMPRSS4 mRNA levels were upregulated in colorectal cancer tissues versus adjacent normal mucosa.
The authors also reported that TMPRSS4 protein expression was significantly higher in human colorectal cancer tissues from advanced stages (52.5% and 50.0% of stages III and IV, respectively) than in that of early stage (6.3% in stage I), suggesting that TMPRSS4 may play a role in the progression of non-invasive tumors to invasive malignancies [35].
Jia et al.
[36] showed that the inhibitory tripeptide tyroserleutide led to downregulation of TMPRSS4 in hepatocellular carcinoma (HCC), thereby reducing the invasion and metastasis of HCC induced by irradiation.
Taken together, TMPRSS4 may be a novel biomarker for the prognosis of certain types of cancers and could be employed for diagnostics and therapeutics.
On the other hand, the mechanism by which TMPRSS4 expression is modulated has not been well characterized.
Recently, Nguyen et al.
[37] reported that TMPRSS4 was increased in NSCLC cells under hypoxic conditions, suggesting that hypoxia within the tumor microenvironment may upregulate TMPRSS4 expression.
Title: Function of TMPRSS4 in the regulation of EMT and invasion

In colon cancer cells, TMPRSS4 induces downregulation of E-cadherin and leads to EMT events, accompanying morphological changes and actin reorganization [38].
Suppression of TMPRSS4 by siRNA reduces cell invasion in colon and lung cancer cells, while overexpression TMPRSS4 induces migration, invasion, and metastasis [38].
Attachment and spreading of cells on the ECM, with concomitant formation of stress fibers and focal adhesions, is a prerequisite for cell migration.
TMPRSS4 also modulates cell-matrix adhesion and cell spreading mainly through modulation of integrins, such as α5β1, which has been centrally implicated in the EMT and cell motility [39, 40], which probably contributes to enhanced motility and invasiveness.
One of the molecular mechanisms by which TMPRSS4 mediates the EMT and invasiveness in tumor cells is that TMPRSS4 mediates focal adhesion kinase (FAK) signaling pathway activation and extracellular signal-regulated kinase (ERK) activation, mainly through integrin α5 upregulation, leading to the EMT and invasiveness.
Furthermore, TMPRSS4 overexpression in human colorectal cancer tissues positively correlates with enhanced expression of integrin α5 and inversely correlates with E-cadherin expression, confirming that TMPRSS4 modulates expression of EMT markers.
Recently, Larzabal et al.
[41] reported that miR-205 is involved in TMPRSS4-induced integrin α5 expression in NSCLC cells.
To further implicate TMPRSS4 in the EMT, Cheng et al.
[42] suggested that interactions between HGF activator inhibitor (HAI-1) and TMPRSS4 contribute to EMT events, including E-cadherin reduction and morphological changes in pancreatic cancer cells.
In addition, TMPRSS4-induced E-cadherin reduction and EMT play a critical role in radiation-induced long-term metastasis of residual HCC in nude mice [43].
Interaction of TMPRSS4 and integrin α5, based on the observation that TMPRSS4 partially interacts with integrin α5 under certain coimmunoprecipitation conditions in a cell line-dependent manner [35] (Kim S, unpublished observation), suggests the possibility that TMPRSS4 may modulate or participate in the interaction of integrin and other cell surface proteins (for example, tetraspanin, receptor tyrosine kinases, etc.
), leading to subsequent signaling transduction activation.
In fact, TMPRSS4 can interact with urokinase plasminogen activator receptor (uPAR; CD87) [44], which can induce the EMT in hypoxic breast cancer cells [45], although it is not clear whether TMPRSS4 interacts with uPAR directly or via integrin(s).
Loss or reduction of E-cadherin expression is a well-known hallmark of the EMT and correlates positively with tumor cell invasion and metastasis [3].
TMPRSS4 appears to modulate SIP1/ZEB2 expression, based on the observation that SIP1 mRNA is upregulated in TMPRSS4-overexpressing colon cancer cells, although induction of SIP1 at the protein level remains to be determined.
Therefore, it is possible that SIP1 mediates TMPRSS4-induced EMT events, including E-cadherin reduction.
Several studies have shown that suppression of high endogenous E-cadherin expression renders non-invasive cells partially invasive [46], whereas reconstitution of E-cadherin results in tumor cell reversion from an invasive mesenchymal phenotype to a benign epithelial phenotype [46, 47].
In contrast, other studies have shown that ectopic expression of E-cadherin could not reverse EMT phenotypes induced by the transcription factor Twist1 [10].
On the other hand, downregulation of E-cadherin was required for TMPRSS4-mediated EMT and invasion in colon cancer cells but was not sufficient for induction of these phenotypes [35], suggesting that downregulation of E-cadherin is not the sole contributor to TMPRSS4-mediated phenotypes.
In this respect, upregulation of specific mesenchymal markers, such as integrin α5, besides the downregulation of E-cadherin by TMPRSS4, may be required for full invasiveness during colon cancer progression (Fig.1).
Title: Molecular mechanisms and signals regulated by TMPRSS4

Numerous studies have focused on the investigation of in vivo substrates of TTSPs.
However, few studies have conclusively addressed the in vivo molecular targets and function of TTSPs during tumor progression.
In vitro, several TTSPs, including matriptase, were shown to activate prourokinase plasminogen activator, pro-macrophage stimulating protein-1, and pro-HGF, which are implicated in the proliferation, migration and invasion of various cancer cell types [23].
Like most of the members of the TTSP family, TMPRSS4 can activate epithelial sodium channel (ENaC) in vitro through its proteolytic activity, possibly regulating sodium and water flux across high-resistance epithelia [48, 49].
TMPRSS4 induces cancer cell invasion in a manner that is dependent on serine proteolytic activity [38], and inhibitory compounds against TMPRSS4 serine protease activity were reported to reduce colon cancer cell invasion [50].
However, it remains unknown which precursor substrates are cleaved by TMPRSS4 to contribute to tumor progression.
On the other hand, it has recently been reported that TMPRSS4 induces urokinase plasminogen activator (uPA) gene expression through activation of the transcription factors AP-1, Sp1, and Sp3 in mainly a JNK-dependent manner in prostate and lung cancer cells but not in colon cancer cells [44].
uPA is a well-known serine protease involved in invasion and metastasis and correlates with poor prognosis in breast, lung, stomach, bladder, colon, prostate, and ovarian cancers [51], and TMPRSS4 expression significantly correlates with uPA expression in human lung and prostate adenocarcinomas [44].
In addition, TMPRSS4-mediated uPA expression contributes to prostate cancer cell invasion (Fig.1) [44].
It is intriguing that TMPRSS4 activates JNK signaling pathways, possibly through its association with uPAR, leading to uPA expression.
uPAR can induce the EMT and stem cell-like properties in breast cancer cells by activating diverse cell signaling pathways, including ERK, phosphoinositide-3-kinase-Akt, and Rac1 [45, 52].
Therefore, the association of TMPRSS4 and uPAR and subsequent cell signaling modulation may be a novel mechanism for the control of invasion and the EMT.
The observations that TMPRSS4 modulates cell signaling and subsequently activates both AP-1 and Sp1/3 transcriptional activities [44], which have been reported to be involved in the transcriptional regulation of the EMT and invasion [53], suggest that TMPRSS4 could modulate the expression of various genes, which may be associated with invasion and metastasis.
Title: The tetraspanins

Tetraspanins (TM4SFs) have four transmembrane protein domains with two extracellular loops and one intracellular loop (ICL) and the N- and C-terminal tails [54].
They are expressed on the cell surface and/or intracellular vesicles and contain 33 members in mammals [55].
Tetraspanins, or TM4SFs, are suggested to be located at tetraspanin-enriched microdomains (TERMs) [56], where they form protein-protein complexes in a homophilic or heterophilic manner with other TM4SFs, integrins, or growth factor receptors [57, 58].
The protein complexes are known to regulate the dynamics of the complex components on the cell surface with regard to diffusion, trafficking, retention, and stability, in addition to influencing intracellular signal transductions [56, 59, 60].
Title: TM4SF5 in cancer

TM4SF5 (gene ID, 9032) maps to chromosome 17 at 17p13.3 according to Entrez Gene.
In AceView, it covers 11.34 kb, from 4621928 to 4633262 (NCBI 36, March 2006), on the direct strand containing 4 different gt-ag introns.
Its transcription produces 2 alternatively spliced mRNAs via alternative polyadenylation sites, which putatively encode 2 different isoforms (197 and 132 amino acids), containing an L6 membrane domain (http://www.ncbi.nlm.nih.gov/IEB/Research/Acembly/av.cgi?db=35g&amp;c=Gene&amp;l=TM4SF5).
TM4SF5 (20,823 Da) is a transmembrane glycoprotein; as a family group, it is related to the tetraspanin family (transmembrane 4 L six family), including TM4SF1 (L6, L6-Ag), TM4SF4 (IL-TIMP), TM4SF518 (L6D), and TM4SF20 [61, 62].
TM4SF5 is highly expressed in diverse types of cancers, including liver, pancreatic, gastric, colon, adrenocorticotropic hormone (corticotropin)-negative bronchial carcinoid tumors, soft-tissue sarcoma, nonendocrine lung, and papilla vateri carcinoma [63, 64, 65, 66].
Similar to tetraspanins (i.e., transmembrane 4 superfamily, TM4SFs), TM4SF5 has four transmembrane domains (TM1-TM4), short cytoplasmic domains at their N- and C-termini, an ICL between TM2 and TM4, two extracellular loops, a smaller extracellular loop between TM1 and TM2, and a larger extracellular loop between TM3 and TM4 [61, 62].
Recent clinical studies separately report that TM4SF5 is highly expressed in tumors from deceased breast cancer patients, compared to those from 10-year breast cancer survivors [67], and that postoperative 5-year overall survival of esophageal cancer patients negatively correlates with TM4SF5 expression [68].
These reports suggest that TM4SF5 overexpression correlates with poor prognosis of cancer patients.
Title: TM4SF5-mediated regulation of signaling molecules

TM4SF5 can appear to form TERMs on the cell surface, via formation of large protein-protein complexes with tetraspanins, integrins, and growth factor receptors [61, 69].
Therefore, by virtue of the protein complex formation, overexpressed TM4SF5 in cancer cells can influence or activate diverse intracellular signaling pathways for cell adhesion, proliferation, the EMT, migration, and invasion for tumor progression and maintenance.
TM4SF5 is shown to associate with integrins α2, β1 [70, 71], α5 [72], and epidermal growth factor receptor (EGFR) [73, 74] during cell migration [70, 71], angiogenesis [72], drug resistance [74], and fibrosis [73].
With association and retention of integrin α5 on the cell surface, TM4SF5 can activate intracellular signaling for FAK/c-Src activation, leading to STAT3 activity for vascular endothelial growth factor (VEGF) induction [72].
In addition, TM4SF5 directly interacts with FAK or c-Src to regulate migration [75] and invasive ECM degradation [76].
In addition, TM4SF5 expression causes AKT activation, which in turn causes phosphorylation of p27Kip1 Ser10 for its cytosolic translocation, where it can regulate RhoA activity for morphological change and migratory function [74].
Title: TM4SF5-mediated EMT in tumor progression

TM4SF5 expression in hepatocytes or NSCLC leads to EMT phenotypes, which in turn cause loss of contact inhibition [74], enhance migration and invasion for metastasis [77], and render gefitinib resistance [78].
TM4SF5 expression causes morphological changes through abnormal regulation of RhoA and Rac1 in hepatocytes, together with the loss of E-cadherin expression leading to EMT induction [74] via induction of Slug [79].
Inhibition of TM4SF5-mediated signaling events of the cytosolic enrichment of p27Kip1 abolishes abnormal multilayer cell growth [74] and retards the G1 to S phase progression [80].
Further, inhibition of TM4SF5-mediated EMT by suppression of cytosolic p27Kip1 expression leads gefitinib-resistant NSCLC cells to become gefitinib-sensitive [78].
TM4SF5 is involved in activation of hepatic stellate cells by causing EMT processes, leading to a correlation with the development of liver fibrosis in CCl4-treated mouse models [81].
TM4SF5 expression is achieved by TGFβ1-mediated Smad actions on EGFR activation [73], such that the important roles of the multifunctional cytokine TGFβ1 in the activation of hepatic stellate cells and the EMT are confirmed in a development of murine liver fibrosis.
Since liver fibrosis can eventually lead to hepatocarcinoma at a high rate of over 70% [82], the roles of TM4SF5 in the development of both fibrosis and tumorigenesis in the liver can be reasonable.
Meanwhile, TM4SF5 expression enhances directional migration and invasion of hepatocytes.
TM4SF5 in hepatocytes causes directional migration at an enhanced speed and the formation of more invadosome-like structures enriched with cortactin, actin, and actin-regulatory proteins, like Arp2 and WASP [77].
TM4SF5-mediated directional migration involves direct interaction and activation of FAK via the ICL domain of TM4SF5 and the F1 lobe of the FAK FERM domain [75].
Further, TM4SF5-mediated invasive ECM degradation requires direct interaction between the COOH-terminus of TM4SF5 and c-Src, which is linked to Tyr845 phosphorylation of EGFR to form more invasive protrusions [76].
TM4SF5-mediated multilayer growth [74], FAK activity, migration, and invasion [75] are abolished by an anti-TM4SF5 reagent, TSAHC (a synthetic compound), which appears to affect its N-glycosylation and at the same time block induction of the TM4SF5-dependent EMT phenotype and multilayer growth [83].
Therefore, TM4SF5 also plays important roles in tumor initiation and progression, possibly being supported by an EMT process.
Title: Other TM4SF5-mediated EMT-related biological processes

The EMT is well known to be related also to the development [84] and stemness of self-renewal [9].
We also observed that TM4SF5 can play roles in other EMT-mediated biological processes, like development of muscles and self-renewal of cancer cells.
In zebrafish, suppression of tm4sf5 results in abnormal development, with an aberrant trunk and morphology of muscle fibers, presumably via alterations in the expression and localization of integrin α5, which is necessary for somite boundary maintenance (Choi YJ and Lee JW, in revision).
In addition to liver fibrosis and tumorigenesis, therefore, TM4SF5 expression is importantly involved in the development of zebrafish muscles, which might bemediated by EMT.
Presumably, these diverse cellular effects by TM4SF5 expression might be possible due to the characteristics of TM4SF5-similar to tetraspanins-which forms large protein networks via heterophilic or homophilic interactions between tetraspanins, integrins, and growth factor receptors.
TM4SF5 is shown to bind integrin α2, β1 [70, 71], α5 [72], EGFR [73], and interleukin 6 receptor (IL6R) (Ryu J and Lee JW, in revision).
Although its ligand has not been identified, interaction(s) with (an)other membrane protein or receptor can recapitulate the ligand binding-based activation.
Therefore, TM4SF5 can transduce signaling activities for diverse cellular functions, including the EMT and different EMT-mediated phenotypes.
Although diverse miRs are known to regulate the EMT [7], miRs targeting TM4SF5 are being studied.
Title: TM4SF5-mediated gene regulation

Comparison of protein expression patterns between TM4SF5-null and -expressing cells shows a negative correlation between TM4SF5 and cell-cell adhesion-related molecules of epithelial markers, including E-cadherin [74], and a positive correlation between TM4SF5 and mesenchymal markers, including Slug [79], supporting TM4SF5-mediated EMT.
Reverse transcription polymerase chain reaction analyses of show that TM4SF5-mediated regulation of their expressions occurs at the transcriptional level (Lee JW, unpublished observation).
However, the signaling pathways underlying this regulation have not been determined yet.
In addition, TM4SF5 expression also correlates with cytosolic p27Kip1 [74].
Although p27Kip1 in the nucleus is inhibitory to cyclin-dependent kinases, suppressing the cell cycle and proliferation, its localization in the cytosol can lead to tumorigenic functions [85].
Cytosolic p27Kip1 has been reported in different clinical reports, where different cancer types show enriched cytosolic localization of p27Kip1 [86, 87, 88], suggesting that cytosolic p27Kip1 can be tumorigenic [89].
p27Kip1 can be phosphorylated by Akt, KIS, or JNK [90, 91, 92], resulting in translocalization and stabilization in the cytosol, where it binds to and inactivates RhoA GTPase, leading to alterations in actin organization and motility regulation [93].
TM4SF5 expression also causes overexpression of p27Kip1, although how it occurs is unknown; TM4SF5 causes Akt-mediated Ser10 phosphorylation of p27Kip1, leading to its stabilization, RhoA activity changes, and eventually morphological elongation for the EMT and contact inhibition loss [74].
JNK-mediated p27Kip1 phosphorylation in a TM4SF5-dependent manner also results in localization of p27Kip1 at cell-cell contacts [91], possibly leading to altered actin organization at cell-cell contacts.
In addition, inhibition of the proteasome in terms of activity and subunit expression also depends on TM4SF5 expression, resulting in morphological changes and the EMT, suggesting another novel mechanism for TM4SF5-mediated EMT [79].
Meanwhile, TM4SF5 causes activation of the FAK/c-Src signaling pathways, leading to STAT3 phosphorylation at Tyr705 for the induction and secretion of VEGF, which can stimulate neighboring endothelial cells for enhanced (tumor) angiogenesis [72].
During modeling of the tumor microenvironment, cancer cells overexpressing TM4SF5 appear to negatively regulate expression of the cytokine IL6, and exogenous IL6 treatment leads to less STAT3 signaling activation in TM4SF5-positive cancer cells (Ryu J and Lee JW, in revision); thus, TM4SF5-dependent suppression of IL6 can be a strategy for TM4SF5-positive tumor cells to avoid pro-immunological actions by IL6 secreted by neighboring immune cells.
As for invasion, TM4SF5 expression also increases the mRNA and protein levels of MMP2, in addition to its activity [77].
Therefore, TM4SF5 expression correlates with or plays important roles in tumorigenesis in different mechanisms, including induction of the EMT and gene regulation (Fig.2).
Considering that such membrane proteins as TMPRSS4 or TM4SF5 may be important upstream regulators of the EMT and the invasiveness of cancer cells and because their expression differs substantially in normal and cancer tissues, targeting them could be a novel therapeutic strategy for the treatment of cancer metastasis.
In the future, the functional involvement of TMPRSS4 and/or TM4SF5 in the initiation and progression of tumors needs to be evaluated using mouse models.
Cancer-associated mutations and singlenucleotide polymorphisms within TMPRSS4 or TM4SF5 also need to be analyzed in association with cancer risk.
Cellular functions of TMPRSS4.
ENaC, epithelial sodium channel; uPA, urokinase plasminogen activator; EMT, epithelialmesenchymal transition.
TM4SF5-mediated epithelial-mesenchymal transition (EMT) is involved in diverse cellular functions, leading to liver tumorigenesis and maintenance in addition to developmental processes.
Title: Next-Generation Sequencing and Epigenomics Research: A Hammer in Search of Nails

After the initial enthusiasm of the human genome project, it became clear that without additional data pertaining to the epigenome, i.e., how the genome is marked at specific developmental periods, in different tissues, as well as across individuals and species-the promise of the genome sequencing project in understanding biology cannot be fulfilled.
This realization prompted several large-scale efforts to map the epigenome, most notably the Encyclopedia of DNA Elements (ENCODE) project.
While there is essentially a single genome in an individual, there are hundreds of epigenomes, corresponding to various types of epigenomic marks at different developmental times and in multiple tissue types.
Unprecedented advances in next-generation sequencing (NGS) technologies, by virtue of low cost and high speeds that continue to improve at a rate beyond what is anticipated by Moore&#39;s law for computer hardware technologies, have revolutionized molecular biology and genetics research, and have in turn prompted innovative ways to reduce the problem of measuring cellular events involving DNA or RNA into a sequencing problem.
In this article, we provide a brief overview of the epigenome, the various types of epigenomic data afforded by NGS, and some of the novel discoveries yielded by the epigenomics projects.
We also provide ample references for the reader to get in-depth information on these topics.
Next-generation sequencing (NGS) has resulted in an exponential growth in data in the past decade, primarily fueled by the general interest of the scientific community in functionally annotating the human genome.
The power of high-throughput parallel sequencing was deployed to characterize human genetic variation, in an international effort launched in 2008, called the 1000 Genomes Project, with the hope that such a large-scale endeavor would help in identifying all the underlying genetic differences that lead to disease resistance/susceptibility [1].
In the meantime, based on numerous genome-wide association studies (GWAS), it was realized that 93% of the trait-associated polymorphisms within the human population were located in non-coding regions, potentially within cis-regulatory elements [2].
In fact, in the mid-1970s, King and Wilson [3] pointed out the importance of mapping regulatory events and hinted that a small difference in the time/level of activation of a single gene could influence overall systemic development.
They estimated the genetic distance between human and chimpanzee to be much smaller than the corresponding genetic distance between the sibling species of Drosophila, thereby underscoring the need to explore regulatory/expression differences at different times as a basis to explain the evolutionary differences at the organismal level [3].
A mechanistic understanding of spatio-temporal gene expression in eukaryotes is far from complete.
While genetic differences might be expected to explain expression divergence across species and expression variability across individuals of a population, it cannot explain how almost-identical genomes in more than 200 different cell types within an individual organism can drive such varied expression profiles specific to each cell type [4].
DNA is packaged in a nucleoprotein complex structure called chromatin that is highly dynamic, in that its &#34;states&#34; vary from one cell type to another.
Therefore, a possible solution to this conundrum is to analyze these &#34;states&#34; at the genome level, i.e., characterize the landscape of the epigenome, which further qualifies and modulates the functional effect of genetic information [5].
Epigenome, meaning &#34;on or above genomes,&#34; refers to such sequence-independent heritable properties of the genome that can modulate the functional output of the genome.
While, as described initially, heritability is an important qualifying criterion of an epigenomic mark (most notably, DNA methylation) [6], currently, the term epigenome is loosely used to encompass a myriad of chemical changes to DNA or histone proteins, chromatin accessibility, small and long (non-coding) RNA localization, and higher-order DNA organization (including nucleosome occupancy and positioning, and 3D chromatin interactions) (Fig.1) [7,8,9].
The ultimate combined effect of the epigenome is to determine the transcriptome (the set of all transcripts) of a cell precisely.
Epigenomic features influence the regulatory program of each gene&#39;s expression in several ways: they define the local environment of specific processes by regulating the chromatin architecture, determine access of transcription factors to DNA, as well as serve to keep a &#34;memory&#34; of cell type-specific features facilitating heritability of epigenetic characteristics [10, 11].
The will of the global scientific community to sequence the human genome spurred advances in sequencing technologies that led to what is now known as NGS.
The improvement in cost and speed of sequencing in the last decade has significantly surpassed the analogous improvement in computer technologies as predicted by Moore&#39;s law [12].
Well beyond the initial goal of sequencing complete genomes of several species, NGS is now routinely used as a tool to investigate the diverse array of cellular events involving DNA or RNA, such as the identification of genomic loci bound by a specific protein, the detection of pairs of spatially proximal or interacting chromosomal loci, etc.
Broad availability of NGS technologies has led to a paradigm shift in molecular biology research; from probing singular cellular events to an unbiased genome-scale mapping of such events, and has enabled genome-wide elucidation of the epigenomic landscapes in hundreds of cell types, across developmental times, in human, as well as other species [13].As mentioned earlier, it is the locus-specific &#34;epigenetic code&#34; (a specific signature or combination of several epigenetic modifications) that helps define a cell&#39;s expression program and identity [14], thereby distinguishing it from other cell types.
Unlike the largely static genome of an individual, the epigenome is highly variable and dynamic; yet, many of these marks can be passed down a cell&#39;s lineage, or from one generation to the next.
In this section, we describe several well-studied epigenomic marks and outline what is known about their role in gene expression modulation.
We will also outline various NGS-based assays that are used in comprehensively mapping each of the epigenetic marks (Table 1) [15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34].
Title: DNA methylation

One of the more stable and heritable epigenetic marks is DNA methylation.
The human genome is highly methylated; approximately 80% of cytosines in CpG dinucleotides are chemically modified at their fifth carbon atom with a methyl group [35].
Historically, DNA methylation was associated with transcriptional silencing (as evidenced by many promoter-based studies) [36].
Although genome-scale profiling of this epigenetic mark revealed that in some instances DNA methylation is correlated with transcriptional activation, it is enriched in the gene bodies of active genes [37].
Thus, the associations between an epigenomic mark and functional output may be location-specific, thereby complicating their functional interpretation.
The available assays for identifying methylated CpG dinucleotides in a genome vary in terms of resolution and cost.
Restriction enzyme-based assays (e.g., methylation-sensitive restriction enzyme sequencing) involve the digestion of genomic DNA with several methylation-sensitive enzymes [15], followed by sequencing of digested fragments.
Affinity-based enrichment assays, such as methylated DNA immunoprecipitation sequencing and methyl-CpG-binding domain protein sequencing [16, 17], selectively target methylated fragments (usually up to 100 bp) using an antibody, followed by sequencing of those fragments.
Both these methods are low-resolution and essentially qualitative.
Chemical modification methods, such as bisulfite sequencing (BS), have higher resolution and provide quantitative estimates of methylation.
They exploit the fact that when treated with sodium bisulfite, unmethylated cytosines convert to uracil while methylated cytosines do not.
Thus, mapping of sequenced fragments reveals the quantitative estimate of methylation at each CpG locus.
The two variations to BS include reduced representation BS and whole-genome bisulfite sequencing, or methylC-seq (whole-genome BS); the former employs restriction enzyme digestion to target certain regions of interest prior to sodium bisulfite treatment [18], whereas the latter couples the chemical treatment with whole-genome sequencing [19].
Upon the discovery that 5-hydroxymethylcytosine (5-hmC: an intermediate produced during active DNA demethylation) and 5-methylcytosine (5-mC) are both resistant to bisulfite treatment, it became critical to distinguish between the different methyl variants, including 5-formylcytosine and 5-carboxylcytosine.
To this end, oxidative bisulfite sequencing (oxBS-seq) was invented as an improvement for methylC-seq to measure single base-pair resolution methylation levels of 5-mC [20].
This is accomplished by oxidizing 5-hmCs (using potassium perruthenate) before sodium bisulfite treatment, such that they are now sensitive to the subjected chemical conversion to uracil, thus ensuring that the remaining cytosines are all 5-mCs.
Title: Histone modifications

Chromosomal DNA is wrapped around histone octamers, essentially composed of 4 kinds of subunits; viz., H2A, H2B, H3, and H4.
These proteins are subject to chemical modifications at specific residues of histone tails; some well-studied modifications include phosphorylation, methylation, and ubiquitination [38].
These modifications are involved in setting the stage for directed transcriptional activation and repression by controlling DNA accessibility or recruitment of other protein complexes.
This idea has been extended into the &#34;histone code&#34; hypothesis [39, 40]-that complex combinations of distinct histone modifications, like H3K27me3 (a mark of repressed regions), H3K4me3 (a mark of gene promoters), and H3K27ac (a mark of transcriptionally active regions), etc.
[41], underlie specific transcriptional programs.
This notion has been further extended in more recent works into an &#39;epigenomic code&#39; to include epigenomic marks other than histone modifications [42].
Chromatin immunoprecipitation (ChIP) of DNA-histone complexes is achieved by cross-linking histones with the DNA, digesting the cross-linked DNA, and then using an antibody specific to the N-tail modification of interest (to any of the &gt;100 post-translational modifications to histone tails and globules [43]), followed by sequencing of the purified DNA fragments.
This method is called chromatin immunoprecipitation sequencing (ChIP-seq) and was primarily used to map the genomic locations of DNA-associated proteins.
Due to the relatively large fragment size, the mapped sequence reads thus obtained from ChIP-seq [21] provide a low-resolution localization map of modified histones in the genome.
An alternative strategy, chromatin immunoprecipitation-exonuclease (ChIP-exo), uses exonuclease to digest the precipitated DNA fragment from either end to better resolve the modified loci at base-pair resolution [22].
Title: Nucleosome positioning and occupancy

The packaging of DNA coiled around histones produces distinct structures called nucleosomes that form repeating units with approximately 147 bp wrapped around each unit, separated by varying lengths of linker DNA.
The occupancy and periodic positioning of nucleosomes can control the accessibility of DNA to transcription factors [23] and DNases, as well as the transcription rate of active gene bodies [44], and are thus considered an epigenetic mark.
Biochemically active regulatory regions are generally depleted of nucleosomes [45], whereas inactive repeat regions (heterochromatin) have higher affinity to form nucleosome structures [46].
A map of nucleosome occupancy is generated by using micrococcal nuclease (MNase) for digestion of the chromatin [23], followed by high-throughput sequencing-a technique called MNase-seq.
MNase digests all linker DNA but preserves the DNA wound around the nucleosome; when the latter is sequenced and mapped to the reference, a map of the original positioning of nucleosomes is obtained.
Due to some inherent biases in MNase affinity (preference to AT-rich regions) and the lack of single base-pair resolution data, attempts were made to develop more precise technologies [24].
An MNase-independent technique that involves chemically modifying engineered histones to bring about cleavage of DNA wound around histones allows direct mapping of nucleosome centers.
Title: Chromatin accessibility

Epigenetic mechanisms, such as chromatin accessibility, impact transcription factor binding to DNA, transcriptional specificity, and hence, transcriptional regulation.
Open and easily accessible regions of DNA within the chromatin are indicative of local territories of transcriptional activity.
Measuring &#34;openness&#34; of DNA at different regions genome-wide, based on the DNase hypersensitivity assay, has helped discover several classes of functional elements, like promoters and enhancers [47].
It has also aided in identifying cell-type specific behaviors by comparison of accessibility profiles [47].
Regions of accessible chromatin are reflective of active regulatory sites.
The enzyme DNase I is capable of digesting DNA in nucleosome-depleted regions (i.e., free unwound DNA).
Post-digestion sequencing reveals large blocks of DNase hypersensitive sites (DHS) in chromatin (DNase-seq) [25, 48], which, upon further deep sequencing, can reveal up to 40-bp footprints of protected regions (potentially bound by transcription factors).
These smaller regions are called digital genomic footprints (DGF) [26].
Formaldehyde-assisted isolation of regulatory elements sequencing is another technique that, by deep-sequencing random fragments of the genome post-crosslinking, measures the frequency of shearing at different loci, thus quantifying accessibility at high resolution, because free DNA that is unbound by factors is more susceptible to shearing [27].
Title: 3D chromatin architecture

The array of nucleosomes organized in 30-nm fibers, called chromatin, has diverse functions well beyond mere compaction.
Coordinated activity of distal elements is orchestrated by short- and long-range DNA interactions, which is determined by the 3D chromatin structure as well as the local environment of individual genes.
For instance, chromatin conformation/looping mediates a promoter&#39;s access to its enhancers, thereby determining the transcriptional fate of a gene [49].
Spatial proximity can be determined by measuring the interaction frequencies of linearly distal fragments by a suite of chromosome conformation capture (3C)-based assays.
Chromosomes assume a specific tertiary structure with profound implications for cellular function and fate [49].
While the overall chromatin structure cannot be directly measured, measuring spatial distances between pairs of genomic loci has become increasingly more efficient with the arrival of 3C technologies [28].
Treatment with formaldehyde crosslinks spatially proximal DNA loci in cells, and then post-digestion, the resulting fragments are allowed to ligate in a weak solution (cross-linked fragments ligate with higher frequency than random fragments).
This frequency is quantitatively assessed by quantitative polymerase chain reaction.
Known loci are selected for by using sequence-specific probes that allow assaying specific pairs of regions in so-called one-versus-one mapping.
Circular chromosome conformation capture, or 4C, allows whole-genome mapping of all spatial interactions of one specific region of interest (one-vs-all mapping) [29].
Chromosome conformation capture carbon copy, or 5C, deploys several anchors and primers to ensure targeting of several regions at once to attain higher coverage (many-vs-many) [30].
Finally, the most recent derivative of 3C, namely Hi-C, permits surveying of the whole genome [31] in a relatively unbiased fashion for frequencies of spatially interacting pairs of loci in the entire genome without restricting the exploration to selected loci.
This was done with reasonably high resolution, where the human genome was partitioned into 100-kb blocks and the spatial interaction between every pair of intra-chromosomal regions was assessed, while interchromosomal interactions have been mapped at 1-Mb resolution [31].
A variation of Hi-C, called chromatin interaction analysis by paired-end tag sequencing (ChIA-PET), involves a ChIP reaction to isolate chromatin interactions involving regions of DNA that bind to a protein of interest [32].
For example, an antibody targeting RNA polymerase II can be used to precipitate interactions involving a promoter and another interacting region, such as a distal enhancer or other promoters [50].
Title: Non-coding RNA localization

Specific classes of non-coding RNA-short RNAs (viz.
micro RNA [miRNA], short interfering RNA [siRNA], and piwi-interacting RNA [piRNA]) and long non-coding RNAs (lncRNA)-regulate gene expression through epigenetic mechanisms, influencing several cellular processes, like X chromosome inactivation [51], genomic imprinting [52], and some cellular processes [53], and cancer [54].
LncRNAs tether epigenetic complexes capable of methylating DNA and modifying histones to the chromatin, enabling allele- and locus-specific regulation.
For example, lncRNA Xist covers the inactive X chromosome and recruits PCR2, a Polycomb complex responsible for trimethylation of H3K27 [51].
Furthermore, lncRNAs, by virtue of their length, are suited for orchestrating specific regulatory events particular to a target locus [55].
Additionally, miRNA (~22 nucleotides long) have shown to be linked with cancer and can act as either oncogenes or tumor-suppressor genes.
They can be involved in establishing DNA methylation and regulating histone modification by interacting with their target mRNA to alter chromatin of the corresponding DNA template [56].
siRNA, and its interference machinery has also been implicated in the formation of heterochromatin [57].
A catalog of small RNAs has been generated by deep sequencing of total RNA from whole transcriptomes using smRNA-seq technology [34].
Selection of smRNA can be carried out by sequential ligation of adapters to its unique 5&#39; mono/triphosphate ends, produced as a result of processing the smRNA population in the cell [58].
On the other hand, lncRNA are poly-adenylated, and much of the challenge in identifying them from whole-genome mRNA-seq data [33] lies in the computational and analytical domains [59].
Spurred by unprecedented advances in sequencing technologies as well as other assays, and a recognition of the potential of NGS in determining the epigenomic landscape, several research consortia have formed to generate genome-wide maps of human epigenomic marks by sharing resources and publishing standard operating procedures, thereby ensuring best practices and high-quality datasets [60].
One of the first concerted efforts in this direction was the Encyclopedia of DNA Elements (ENCODE) Project, started in 2003 and funded by the National Human Genome Research Institute at the National Institute of Health (NIH-NHGRI), which involved hundreds of researchers in dozens of labs across the globe.
The ENCODE consortium has performed 1,650 epigenomic profiling on 147 cell lines assessing the transcriptome, transcription factor binding for dozens of transcription factors, chromatin topology, histone modifications, DNA methylation, and more (http://www.encodeproject.org) [13].
Five years after launching ENCODE, the NIH funded a second large-scale mapping project called the Roadmap Epigenomics Program.
While ENCODE focused on generating data in cell lines, the Roadmap program focuses on mapping epigenomes of high-priority normal human primary tissue and human embryonic stem cell (ESC)-derived cell types.
As of May 2012, the Roadmap includes 61 &#34;complete&#34; epigenomes (of a variety of cell types) (http://www.roadmapepigenomics.org) [61].
The data repository can be accessed interactively at Human Epigenome Atlas (http://www.genboree.org/epigenomeatlas/multiGridViewerPublic.rhtml), via the Washington University Epigenome Browser (http://www.epigenomegateway.wustl.edu/browser), as well as the UCSC Genome Browser (http://www.genome.ucsc.edu).
Further, the NIH Roadmap is one of the members of an even larger consortium called the International Human Epigenome Consortium (IHEC; http://www.ihec-epigenomes.org), formed by institutions in 7 countries.
To name a few, BLUEPRINT: a European initiative to generate about 100 reference epigenomic maps; the CREST/IHEC team in Japan: to produce reference epigenomes specializing in gastrointestinal epithelial cells, vascular endothelial cells, and cells of reproductive organs; and the Epigenomic Platform Program: a Canadian program to map epigenomic variations in disease cells.
Together, these efforts are expected to generate over 1,000 reference human epigenomes.
Other epigenome mapping efforts focused on specific disease areas have also been undertaken.
These include the Cancer Genome Atlas (TCGA), funded by the NIH in 2006 [62], which is a part of the International Cancer Genome Consortium (ICGC; http://www.icgc.org) that was officially put together in 2008.
By providing a rich functional annotation of the genome, the numerous epigenomic datasets have significantly improved our ability to probe the mechanisms of gene regulation as well as shed light on disease processes mediated by perturbation of normal regulatory processes.
A collection of 30 scientific reports published with the initial release of the ENCODE data provides a detailed report of the initial integrative analysis of the epigenomics data [47, 63, 64, 65, 66].
At a basic level, the data have revealed epigenomic signatures of a variety of functional elements, thus significantly enriching their annotation.
For instance, many non-coding RNAs have been successfully annotated as a result of deciphering an epigenomic code specific to their promoters and gene bodies [67, 68].
High-throughput profiling of the histone modification HeK4me1 has revealed that it is a characteristic mark of enhancer elements; this, and other correlative marks have enabled the discovery of tens of thousands of cell-specific enhancers [69, 70] in the human genome, drastically improving our ability to probe the mechanisms of transcriptional regulation.
In fact, chromatin signatures specific to promoters, enhancers, and repressed regions, etc., have all been modeled by computational methods like ChromHMM [42].
Furthermore, by isolating footprints of protected DNA present inside blocks of hypersensitive or nucleosome-depleted DNA, a comprehensive list of potential cis-regulatory binding sites/sequences has been identified, despite the lack of specific antibodies or the knowledge of cell-specific regulators [2].
Availability of epigenomic data has revived predictive modeling of condition-specific gene expression.
For example, Dong et al.
[71] generated a 2-step quantitative model that includes genome localization data for 11 histone modifications and 1 histone variant in 7 human cell lines and gene expression data quantified using RNA-seq and cap analysis gene expression (CAGE) in the nuclear and cytosolic compartments of the cell.
The Pearson&#39;s correlation coefficient between their predicted and measured expression levels in about 78 experiments ranged between 0.6 and 0.9 (median r = 0.83).
Natarajan et al.
[72] trained a classifier that associates cell-specific TF sequence motif matches in DHS regions with different expression patterns in 19 diverse cell types, thus predicting cell-type specific expression directly from regulatory elements in open chromatin.
Prior to the availability of NGS, details of the spatial structure of chromatin and its correlates, as well as its functional impacts were largely uncharted.
Characterization of the spatial proximity of distal genomic elements with 3C-derivative assays has revealed a high rate of widespread promiscuous interactions among enhancers and promoters.
For instance, more than 40% of promoters interact with multiple distal sites, and this is likely true for enhancers as well [73].
The spatial chromatin interaction data have also helped challenge some previously held beliefs that were nevertheless unsupported.
For instance, historically, the DNA-binding protein CTCF was known to function, among other things, as an insulator that inhibited the spatial interaction between an enhancer and its target promoter when flanking a CTCF-bound locus.
However, recent 5C data showed that almost 60% of all enhancer-promoter interactions happen, despite an intervening CTCF-bound locus [74].
In fact, there is increasing evidence that CTCF, along with cohesin is involved in coordinating long-range interactions between regulatory elements, and that these events are reflective of cell-type specific transcriptional programs [75].
With the availability of putative genome-wide enhancers along with their cell type-specific activity profile across numerous cell types, it is now possible to investigate an alternate layer of transcriptional regulation represented by a network of distal enhancers that exhibit correlated activities across cell types [76, 77] and jointly underlie correlated expression of functionally linked genes.
Such analysis brings forth an alternative view of transcriptional regulation, where instead of a single gene regulated by one or more regulatory elements, one ought to consider the collective of enhancers and genes, co-localized in nuclear space to achieve coexpression of functionally linked genes [78, 79].
Epigenomic profiling along the developmental time course from ESCs to differentiated cells has shed light on transcriptional regulatory mechanisms during early development.
For instance, comparison of the &#34;histone code&#34; between the two developmental timeframes indicates that regions marked by repressive histone marks are generally larger in differentiated cells and that these regions span developmental genes.
It was also noted that the same regions are more accessible in ESCs [80, 81], thus suggesting an epigenome-mediated regulation of developmental gene transcription.
Precisely how this is accomplished is yet to be clarified.
The epigenomic data and the discovery of significant patterns of epigenomic marks associated with specific functional regions have also helped clarify genotype-phenotype association data.
Numerous GWAS have revealed several polymorphic loci associated with various diseases [82].
However, a mechanistic interpretation of these signals has been hampered by the fact that most of the disease-associated genetic variants reside in non-coding regions with no obvious functional interpretation.
By combining epigenomic maps and genetic variants, it was found that these disease-associated variants are enriched within cell type-specific accessible (DHS regions) and active regulatory regions [2].
Also, an epigenomic model of cardiac enhancers was shown to provide a better causal interpretation of GWAS signals associated with heart-related traits [83].
The slew of epigenetic data has revolutionized our perception of the human genome.
Although not all &#34;epigenetically marked&#34; regions of the genome are likely to be functional, it is very likely that a large portion is functional-much more than the 5-10% of the genome that is deemed to evolve under purifying selection [84, 85].
Analysis of transcriptional and epigenetic data has revealed that almost half of the genome is involved in carrying out specific biochemical functions [13].
Where these biochemical functions fit in the big picture, and how they are regulated are still open questions.
Another fundamental problem is that the causal relationships among various epigenomic events, such as DNA methylation, histone modifications, TF binding, and gene expression, etc., are currently not well understood.
Although correlations between these events have been well documented in different contexts, we do not know if one is necessary and/or sufficient to observe another.
Before the full potential of NGS in mapping the epigenome can be attained, several technical challenges need to be overcome.
For instance, techniques like ChIP-seq, used for profiling transcription factor binding and histone modifications require a large amount of starting material, ranging up to 5 million cells.
Collection of such a large population of primary differentiated cells, progenitors, and those from specific developmental stages is inherently hard.
Heterogeneity of a cell population is another issue that needs to be resolved.
It is challenging to obtain a homogeneous population of cells of a singular type from primary tissues that often contain a heterogeneous population of cell types, and it is this lack of homogeneity that can adversely impact the interpretability and generalizability of the results obtained from analyzing the epigenomic data.
Yet, many solutions to tackle the problems above are underway.
First, there is the emergence of techniques-like ChIP-nano (nano-scale ChIP), which is a highly sensitive small-scale ChIP-seq protocol with a requirement of &lt;50,000 cells as starting material [86], and single-molecule real-time sequencing [87], which aims at alleviating cell-population effects during mapping specific features.
Second, methods that are aimed at the targeted knock-out of epigenetic modifiers, like DNA methyltransferases and histone modification enzymes, at predetermined DNA sequences (using transcription activator-like effector nucleases [TALENs]) [88], can help to better characterize the functional relationships between gene expression and epigenetics, thus providing proof of concept for expression modulation (also called epigenetic editing methods) [89].
Also, powerful statistical approaches are being developed to integrate data [42, 90] to resolve some of these fundamental issues.
With more epigenomic maps being profiled, and more effective computational approaches being developed, we are inching towards a holistic mechanistic understanding of cellular and organismal processes.
A schematic depicting the context of several epigenetic marks on chromatin; ranging from DNA methylation, nucleosome positioning patterns affecting the size and distribution of DNase I hypersensitive sites, transcriptional activity at non-coding sites leading to the production of small and long RNA, chemical modifications of histone moieties (e.g., mono-, di- and tri-methylation of the 4th lysine of the H3 subunit), and chromatin folding to form localized structures in 3D nuclear space (anticlockwise).
Adapted from Fig.1 in Telese et al.
[9], Copyright©2013, with permission from Elsevier.
Title: A summary of epigenetic marks, their types (wherever applicable), and the NGS-based assays used to map their location and distribution
Title: Interactive Visualization for Patient-to-Patient Comparison

A visual analysis approach and the developed supporting technology provide a comprehensive solution for analyzing large and complex integrated genomic and biomedical data.
This paper presents a methodology that is implemented as an interactive visual analysis technology for extracting knowledge from complex genetic and clinical data and then visualizing it in a meaningful and interpretable way.
By synergizing the domain knowledge into development and analysis processes, we have developed a comprehensive tool that supports a seamless patient-to-patient analysis, from an overview of the patient population in the similarity space to the detailed views of genes.
The system consists of multiple components enabling the complete analysis process, including data mining, interactive visualization, analytical views, and gene comparison.
We demonstrate our approach with medical scientists on a case study of childhood cancer patients on how they use the tool to confirm existing hypotheses and to discover new scientific insights.
Among cancers, acute lymphoblastic leukemia (ALL) is the most common cancer occurring in children and is the driving paradigm for the work presented in this paper.
Despite presenting with similar clinical features, ALL patients do not always respond in a similar manner to the same treatment strategies.
The underlying complexities of the disease are not always clearly reflected by the clinical presentation or pathological results.
Genomic variation, typified by single-nucleotide polymorphisms (SNPs), have considerable influence on how humans develop diseases and respond to pathogens, chemicals, drugs, vaccines, and other agents [1].
Similarly, gene expression patterns in diagnostic bone marrow can predict the sub-type of pediatric ALL as well as indicate relapse potential [2, 3].
The co-regulated activity of genes leads to metabolic actions that are driven by small changes in the expression of a large number of genes rather than large changes in only a few [2].
As a result, understanding genomic data and biomedical data (e.g., annotated clinical attributes, treatment details, domain ontologies, and patient&#39;s background) is crucial to improving the diagnosis and treatment of ALL.
As a rule, the knowledge discovery process involves the dialog between experts from different domains, including laboratory scientists, biostatisticians, computational biologists, and clinicians.
In reality, most techniques were developed based on statistical analysis with simple visualization [4] in an attempt to create techniques that would be applicable in several domains.
Basic statistics and visualizations without effective interaction and capabilities to control the visual data mining process are often insufficient for the analysis and exploration processes.
Intelligent visualization of complex genomic data will therefore bring the insight of information as well as the discovery of relationships, non-trivial structures, and irregularities that may pertain to the disease course of the patient.
Genomic datasets are complex data structures with measurements for millions of SNPs affecting the activity of several thousand genes.
These genomic &#39;attributes&#39; are built into models that inform us about individual patients and allow cross-referencing and determining the interconnections between patients.
It is important then to provide adequate tools in order to visualize these models and interactions and to simultaneously examine many attributes in the context of relevant clinical properties that can be measured in real time for the purpose of assessing a patient&#39;s clinical course and directing his clinical management.
Fundamentally, a large number of attributes need to be reduced to a manageable size so that data visualization techniques can be effectively applied.
Typical dimensionality reduction methods applied to microarray data include principle component analysis [5], non-negative matrix factorization [6], multidimensional scaling [7], and local linear embedding [8].
The choice of method to apply depends to some extent on the nature of the information expected to be encoded in the dataset, since the quality of the resulting reduced matrices can vary across different datasets.
Remarkably, most of the visualization techniques use matrix heatmaps, genomic coordinates, or networks (or pathways metaphor) to represent multidimensional genomic data [9,10,11].
- Heatmaps are common graphical representations that are used to present genomics values as matrices of colors.
Matrix heatmaps are typically represented as rectangles (such as cBio Cancer Genomics Portal [12] and Caleydo StratomeX [13]) and occasionally as circles (such as CircleMap [14]).
- Genomic Coordinates represent oncogenomics data by showing alterations tied to their genomic loci.
This approach is only suitable for analyzing the genomic topography of alternations or for inspecting particular genomic loci [9].
Typical techniques in this approach are Genomics Viewer [15], UCSC Cancer Denomics Browser [16], and Savant Genome Browser [17].
- Networks (or graphs) can be used to show changes in state for 1) interpreting and exploring large biological networks and 2) assembly and curation of pathways (such as Cytoscape [18], VANTED [19], VisANT [20], NAViGaTOR [21], and Cerebral [22]).
Visualization techniques for gene expression techniques that include reasonably advanced visualizations are presented in [23,24,25].
Other good visualization tools show the interdependencies of genes by bringing gene expression into context with pathways [26] or using dimension reduction methods to provide multi-dimensional data visualizations [27].
Web-based visual analysis integrates multiple visualization components, such as scatterplot, phylogenetic tree, and genome-wide circus viewer, to enable the analysis [28].
Although these tools provide somewhat effective ways for the analysis of data, they do not effectively present the data such that medical analysts can interactively explore and manipulate the information.
Our approach seeks to provide a capability for viewing patients in the similarity space based on detailed genomic and gene of interest analysis, representing the underlying biological basis for the disease.
This paper presents a novel visual analysis approach and a tool where medical scientists can interrogate large and complex genomic and biological data.
We focus on patient-to-patient comparisons through the biological data, including background and treatment, and high-dimensional genomic data, including Affymetrix expression microarrays and Illumina SNP microarrays.
The innovation lies in its capability of providing seamless and multiple views of data, from overview of the entire patient population with the similarity space to overviews, detailed views, and genes of interest (GOI) views of selected items, for patient-to-patient comparison.
The technology provides various ways to display the multi-dimensional data in cooperation with the automated analysis.
In addition, by integrating domain knowledge into the development, we have provided meaningful visualization for medical data analysis as well as a quality assurance tool to verify the effectiveness of the automated analysis.
Title: Biological background and data analysis

The expression and genomic SNP profiles of 100 pediatric B-cell ALL patients treated at The Children&#39;s Hospital at Westmead were generated using Affymetrix expression microarrays (U133A, U133A 2.0, and U133 Plus 2.0) and Illumina NS12 SNP microarrays, respectively.
Each Affymetrix expression microarray has 22,277 attributes, while each Illumina SNP microarray has 13,917 attributes.
Each attribute was mapped to a probe of DNA (or a gene), and the value for each attribute corresponded to the expression levels or genotype for the gene (each DNA spot contains picomoles (1012 moles) of a specific DNA sequence, known as probes or reporters).
The annotations for each gene were mapped on separate files.
Expression microarrays were hybridized with diagnostic bone marrow samples and genomic microarrays were hybridized with remission peripheral blood samples.
The patients were treated following the Berlin-Frankfurt-Munich 95 (BFM95) protocol or the complimentary Australian and New Zealand Children&#39;s Hematology and Oncology Group Study VIII (Study 8).
Using these datasets, we aimed to develop a predictive model of treatment outcome by identifying genes capable of differentiating patients that survived and those that did not.
To achieve this, we applied an attribute deletion approach: identifying and removing genes that were almost certainly not involved in a biological phenomenon and patient comparison.
We used the attribute importance ranking that was implemented in &#39;Random forest&#39; [29] to identify these genes.
The remaining informative genes were then used to build a similarity space whereby the distances between patients in this space were indicative of genetic similarity.
The expression values were z-score-normalized within each platform, concatenated, and z-score-normalized to minimize inter- and intra-platform biases.
For the genomic SNP profiles, the theta scores were used to represent genotypes.
The two datasets were then concatenated, and 30% of all the patients were withheld from the model-building process as a validation set.
Eight bootstraps of the validation set&#39;s patients were generated, and for each bootstrap, a random forest was performed.
Each random forest produced an attribute ranking list, and we combined these eight lists to create a global list of an attribute&#39;s importance.
The expression and SNP values for the top 250 genes in this global list were used to create a 3D similarity space using singular value decomposition-a matrix decomposition technique [30].
Title: The visual analytics model

Our visual analytics model reflects the importance of the domain knowledge in the visual analytics process (see Fig.1).
Earlier, we mentioned the importance of validating a method by the outcomes in the discipline that the method is used.
To do this, visual analytics must adapt to the respective discipline and conform to the language and norms expected of that discipline.
For example, the expression of identified significant genes is required to be validated by the domain experts when the analysis is undertaken in independent laboratories.
An appropriate visualization helps the analyst to gain an understanding of the data and construct knowledge through our powerful human visual perception and reasoning capabilities.
We provide three interactive views for patient-to-patient comparison, including 1) overviews of the entire patient population in the similarity space, 2) detailed views of selected patients, and 3) GOI.
We describe these visualizations in the following sections.
Domain analysts should be able to interact and explore through different views to make further discovery of, and insights into, the data.
They can obtain a better understanding of the data and their structures and contribute their domain expertise to the knowledge discovery process.
From the discovery of new knowledge, the analyst can evaluate, refine, go beyond, and ultimately confirm hypotheses built from previous iterations and automated analysis.
The challenge presented to the construction of appropriate visualization is to cover the range of visualizing all of the data but making only all of the relevant data available to the analyst when it is relevant to do so.
The presented work was developed with strong involvement from cancer researchers at the Children&#39;s Hospital in Westmead, Sydney, Australia, who are experts in ALL research.
The philosophy behind the work is that with this new way of looking at genetic and clinical details from the perspectives of both domain knowledge and technologies, visual analytics can provide a medium for the discovery of genetic and clinical problems, potentially leading to improved ALL treatment strategies.
Title: Interactive visualization

The microarray automated data analysis produced a similarity space of the patients based on their genetic properties.
A mapping table of 250 GOI is also created during this process.
In order to present these results in a form suitable for visual analysis by medical experts, we have created interactive visualizations that operate with the treated data.
The interactive visualization is applied to 1) present a flexible, changeable, and meaningful display of the patient cohort; 2) filter, explore, and manipulate the information; and 3) interactively provide the details of both the original and processed data on demand.
Details of the interactive visualization components are presented as follows.
Title: Overview of patients

The prototype version of the new interactive visualization system provides the set of views addressing the needs described above.
The visualization provides a global overview of the entire cohort structure and the relative patient-to-patient comparisons but then allows the analyst to drill into the details of the relationships.
Displaying the entire visual structure at once allows analysts to move rapidly to any location in the space.
This makes less effort to navigate across the large structures and explore specific parts.
We utilized and extended our former works [31, 32] to provide a flexible, simple, yet powerful interface of the patient population in the similarity space.
The visualization is capable of displaying both 2D and 3D visual projections of the same structure and switching between them from the same &#34;area&#34; in the dataset (see Figs.2 and 3).
Although each user might have a different preference, the 2D visualization is normally a better choice, thanks to its simplicity and familiarization to the analyst, its flexibility in interaction and attributed mapping, no obscuration, and perspective correction.
The following sections are limited to the 2D version that was extended from TabuVis [31].
It is a robust and powerful visual analysis system that provides flexible, customizable, and effective visualization for multidimensional data.
Our innovation lies in the ability to provide an easy-to-use yet effective way to view multidimensional data at different angles by mapping various attributes to different visual properties.
Key aspects for visually analyzing data with multiple attributes are quality and appropriateness of the analytical platform whose presentation can be adjusted via domain experts.
The extended system consists of multiple components enabling the analysis process, including data processing, automatic marks, customizable visualization via interaction, control attributes, filtering, statistical display, save/open works, and many other figures.
The overview visualization supports several data types, including Integer, Float, Text (string), Boolean (true/false, yes/no, or 0/1), Date (with various format), Date and Time, and Percentage.
The data types are categorized into two groups: Categorical (text and boolean) and Quantitative (integer, float, date, date and time, and percentage).
We also map each data type to a color for better identification, particularly text → dark blue, boolean → blue, integer → red, float (and percentage) → orange, and date (and time) → purple.
The color scheme can be easily modified via a property file and/or menus when running (see Fig.2).
Title: Patient&#39;s positions

The patient&#39;s positions are mapped into the 2D space according to his genetic property.
This property reflects the genetic similarity of the patients.
Two patients are close together if their genes are similar, and conversely, they are located far from each other if their genetic properties are different.
For example, Fig.2 clearly illustrates two distinctive groups of patients, marked by the dash-line ovals.
The small group contains mostly deceased patients, while the other contains patients that responded well to the treatment and survived the disease.
The fact that the deceased patients are located close to each other in the gene space may support the hypothesis that genetic properties are essential to determine whether a patient is likely to respond well in the context of the clinical treatment.
Title: Interactive exploration

Title: Mapping property

Rich graphical attributes are employed to provide clinical and background properties of the patients.
The attributes are carefully selected based on the feedback from the medical analysts.
The mapping property and items can be easily adjusted, filtered (or hidden), and/or re-mapped to different attributes (e.g., patient background and biomedical information) via interactive menus to suit the preferences and new requirements.
The mapping options are label, axes, colors, size, shapes, and visual bars.
Title: Filtering

From the visualization, users can interactively filter out uninteresting patients to enhance the view of the associated data.
Filtering can be applied to all categorical and quantitative attributes through interactive menus.
The filtering allows the researcher/clinician (domain expert) to extract, picture, and interrogate specific features for the patients selected within the similarity space.
Fig.4 illustrates the view at a navigational stage showing all medium-risk patients only.
Patient identifications are also displayed in this example.
The figure highlights three patients with similar clinical presentations, such as they were all stratified as medium risk (MR) (blue color), but having diverse outcomes.
These patients were managed on the two treatment protocols BFM95 (larger size, ALL48) and Study 8 (smaller size, ALL134 and ALL323).
With the patients placed into a similarity space based on genetic data, we have applied further computational methods to extract, visualize, and compare data from the chosen patients.
By looking at the figure, we can quickly see that ALL323 is located at the top-left, while the other two patients are positioned closely together at a distance from the first patient.
This observation indicates that the two surviving patients ALL48 and ALL134 are genetically similar and that they are genetically different from the deceased patient, ALL323.
Title: Probe set analysis

The system also provides a capability to show and compare the property of a particular probe set across the patient&#39;s population.
This property allows the analysts to have a comparative view of how a probe set in a gene behaves across the population.
Fig.5 shows an example of the visualization of the population that is colored by the value of the probe set &#34;212022_s_at&#34; at the gene &#34;MKI67.&#34;
The figure clearly indicates the significant difference of patients in the bottom-left cluster, which are mostly painted with &#34;greenish&#34; colors, corresponding to the high values, in comparison to reddish colors, corresponding to low values.
It raises a new hypothesis that the probe set &#34;212022_s_at&#34; (or the gene) might play an important role in specifying the similarity space.
Title: Clustering

We provide a clustering method to group the patients in the similarity space.
The clustering is used to enhance the visibility of patients in the space as the groups who are potentially sharing similar genomic properties.
In addition, by using a filtering method, the analysts can quickly filter out unwanted clusters for clearer views of the population.
We applied the k-means++ clustering algorithm [33] in the implementation.
The algorithm was chosen because of its ability of choosing the initial values (or &#34;seeds&#34;) for the popular k-means clustering algorithm and thus avoiding the sometimes poor clustering outcomes found by the standard k-means algorithm.
The number of clusters can be specified via a menu.
Fig.6 shows an example of the visualization of the entire patient population who are painted with 4 colors corresponding to 4 clusters.
Title: Patient-to-patient visual analysis

From the overview of all patients, users can select one or more patients as well as groups of patients to analyze further.
At first glance, the analytical view provides full information of both genomic and biological data.
The layout of the panels is as follows: 1) the top-left panel displays information of the Affymetrix expression microarrays, 2) the top-right panel displays information of the Illumina SNPs, and 3) the middle table displays all background information of the patients, and the bottom table displays the treatment details (see Fig.7).
Each probe set in the gene is represented as a point on the x-y coordinate system.
All genes are distributed orderly from top to bottom along the y-coordinate while their values are represented horizontally at the x-coordinate.
This visual projection follows the biological rule-the order in which genes are displayed is sorted according to chromosomal order.
A quick sort process is performed in prior to the visualization.
By providing an overview of the entire genetic and biomedical information, the analysts can identify patterns and abnormalities before exploring further.
Fig.7 shows an example of the analytical window of the genetic and clinical data corresponding to patient ALL323.
We can see that the Affymetrix expression microarray values of this person vary from -2 to 4, in which the majority of the genes&#39; values are [-1, 0].
It is interesting to see that there is a region of loss of heterozygosity in ALL323 (highlighted by the red dash-line rectangle).
Following our revised model of the visual analytics process, the analytical exploration is an essential process to discover insight and knowledge.
We provide a variety of functions to support the visual exploration and analysis, including highlighting the importance or difference, zooming, filtering, detailing on demand, and comparing multiple patients or groups of patients.
A user can interactively highlight GOI and significantly different genes as well as deemphasize the others.
This feature allows one to gain a better view of those GOI in comparison with the entire gene structure.
The user can also select the option of displaying those GOI only.
A semantic zooming technique is applied to zoom in (or zoom out) on any particular area of focus (see Fig.8), which allows the analyst to explore and view in detail the genetic information.
The level of details is updated dynamically depending on the amount of focused genes and the size of the windows.
This property ensures the best fit of information on the available space.
In addition, detailed information of a particular gene is shown when the mouse is over the gene (see Fig.8E and 8F).
Fig.8 shows an example of the visualization of patient ALL323 (same as Fig.7) when various options and navigations are applied.
In particular, the figure presents the views when a) the GOI are emphasized from the structure, b) the view shows the GOI only, c) the significantly different genes are highlighted, d) the significantly different genes are highlighted within the GOI, e) zoom in is applied to SNP data (at the right panel), and f) the user zooms in on the gene expression data (at the left panel) so that the detailed information of probe sets in the genes is displayed.
We also provide a mechanism to compare multiple patients by aligning the analytical windows together.
This figure is particularly helpful for identifying the similarities, differences, regularities, and irregularities of the patients.
Fig.9 presents the view of genetic and clinical information of the 3 chosen patients (highlighted in Fig.4).
First, the complete data of each patient can be directly compared.
By way of example, the figure indicates a significant difference of patient ALL323 when compared to the other two patients ALL48 and ALL134-in particular, a region of loss of heterozygosity in ALL323, a region that may contain the genetic differences with ALL134 and ALL48, which may explain the differences in treatment outcome despite being considered similar according to the clinical presentation.
We now describe the technical details for visualization of the complete set of SNPs and probe sets for the genes represented by the 250 developed features.
Title: Visual analysis of GOI

The traditional methods usually use heatmaps to show all probe sets or genes in the datasets.
Although this approach can show the overview, its visualizations are very dense.
It makes it really difficult to analyze those important genes in detail from the crowded population.
Our system innovatively focuses the interactive visualization of the GOI resulting from the automated analysis.
By reducing the overhead of analyzing a large quantity of genes, the analysts can easily compare the GOI through a large population of patients.
The Gene Comparison Interactive Visualization is designed to allow an analyst to drill down further into the genetics and treatment data of patients identified as significant by the processes described in the previous sections.
This visualization implements several mechanisms whereby the similarity and differences between patients and groups of patients can be examined in greater detail.
An important feature of this interactive visualization is the use of active regions to indicate what the analyst is currently focused on.
Knowing this makes it possible to infer what extra information the analyst might find useful and to make sure that it is available.
This interactive visualization consists of 5 separate visual components: A) the primary patient probe set heatmap component, B) the gene zoom component, C) the color gradient and Gaussian curve component, D) the gene ontology and probe set annotation component, and E) the patient biomedical data component (see Fig.10).
These 5 components are designed to interact with each other as well as other components, including the &#34;Patient-to-Patient Visual Analysis&#34; (described in section 6) and the main visualization (described in section 5) in several different ways to assist the analyst to examine the similarities and differences between significant patients and significant groups of patients.
Fig.11 presents an overview of the visualization of the entire Genes of Interest Visual Analysis module when the 3 patients highlighted in the similarity space are added.
Please note that the figure shows both the probe set heatmap component and the patient information component.
The prototype has been used for analyzing the biomedical and genomic datasets of 100 ALL cancer patients, from the perspectives of domain knowledge and technologies.
In spite of the size limitation of the dataset due to the expensive data collection, the discovery is quite encouraging in the case of ALL patients.
This pilot result will place a foundation, enabling further enhancement and discovery using more comprehensive datasets and/or clinical trials.
On the basis that patients who cluster together will have genotypically similar tumors, it is then anticipated that newly diagnosed and as yet untreated patients will be compared directly to local &#39;biologically&#39; similar neighbors.
By comparing the management and treatment outcomes for patients in this manner, the analyst (i.e., clinician) can glean specific information from the local neighbors that will assist with their clinical decision making for the individual patient, moving towards the &#39;personalization&#39; of medicine.
The analysis has partially confirmed this accuracy of the similarity space.
To illustrate this, Fig.11A highlights three patients with the red-dash ellipses, where the deceased patient ALL123 is at the top left and the surviving patients ALL26 and ALL302 are located close to each other at the near-bottom right.
These patients have similar clinical presentations, such as all being stratified as high-risk, but diverse outcomes.
Fig.11B shows the detailed views of the biomedical and genomic data of the three selected patients, showing the significant difference in gene expression values of patient ALL123 in comparison with ALL26 and ALL302.
This property may explain the differences in treatment outcome of the three patients despite the similar clinical presentation and treatment regimen.
The interactive heatmap visualization of the expanded set of SNPs and gene expression probe sets used in the array platforms allows the analyst to examine each specific genetic attribute represented by the 250 features (Fig.11C) that further confirm the hypothesis.
We also carried out an analysis for different groups of patients.
The experiments also confirm the effectiveness of the similarity space.
For example, the genetic analysis of four clusters of patients (see Fig.5) has unveiled that all probe sets of the gene MKI67 of the patients in cluster 0 (at the bottom-left and painted with a light-pink color) are significantly different from the others (see Fig.12).
Further visualization with the probe set mapping across the patient&#39;s population has confirmed this property (see Fig.5).
Details of the other findings (and the confirmation of the prognoses) are described further as follows.
There is little coherence between the genomic property and the biomedical property, such as risk, dead, or survival rate.
Similar patients in terms of background and clinical information might have significant differences in their genetic properties.
For example, Fig.13 illustrates the details of several genes of interest for the four patients: ALL92, ALL129, ALL323, and ALL321.
The patients&#39; clinical properties are similar, including MR, male, deceased, and Study 8 protocol.
However, the Affymetrix expression microarrays and Illumina SNPs values are quite varied among these patients.
Early treatments are more effective than late treatments (see Fig.14).
Particularly, there was only one death out of 19 cases (95% chance of survival from the disease) if the treatment was started within 5 years after birth.
The dead case also happened to a very high-risk patient who was unlikely to survive because of the disease.
If the treatment was started after 5 years but within 10 years from birth, the survival rate is 86% (3 deaths out of 22 cases), and if the treatment was used after 10 years from birth, the survival rate reduces significantly to 69% (4 deaths out of 13 cases).
Among the very high-risk patients ALL123, ALL143, ALL144, and ALL302 (top-left), patient ALL302 (near bottom-right) was the only survivor whose genetic property was significantly different from the others (at the top-left).
Further examination using the 250 GOIs illustrated in Fig.15 confirms this hypothesis for the expression values, especially for the genes ABHD10, LEPREL1, CNOT4, CHD9, EPOR, RASAL2, ZNF141, and SLC25A31.
The variation at a particular gene or a combination of these genes might have contributed to the survival rate or an improvement in treatment.
This finding can be verified by further analysis using larger and more comprehensive datasets and, ultimately, the actual proof of the clinical experiments.
Patients who were born in later years have a better chance of survival.
For example, the rate of survival for those patients born after 1995 was approximately 94% while the rate for those patients born before 1995 was approximately 83%.
This property reflects the improvement in treatment technology, treatment methodology, and living conditions.
Female patients tend to fare better than male patients overall (see Fig.16).
There is a 92.5% chance of survival in females (3 dead cases out of 40) versus an 88.3% chance of survival in males (7 dead cases out of 60).
The Study 8 protocol is dominant.
The BFM 95 protocol is mostly applied to standard- to medium-risk patients who were born before 2000.
The Study 8 protocol is a much more popular method, and it was used for patients regardless of risk strategies and age.
Fig.17 shows the protocols used in ALL treatment, including Study 8 (green), BFM 95 (blue), Interfant 99 (orange), and others (red).
Chemotherapy is more effective in treatment.
It is interesting that nearly one-half of all patients did not have any treatments (or there were no treatment records) on record.
The untreated patients mostly have standard to MRs.
The bone marrow transplant treatment method has little effect on the survival rate (only 1 out of 5 cases survived).
The most commonly chosen treatment method is chemotherapy, the survival rate for which is over 90% (see Fig.18).
The visual analysis loop.
Visual analytics model, reflecting the engagement of the domain knowledge in the visual analytics process.
GOI, genes of interest.
2D visualization of the entire patient sample.
The entire 100 patients in the 2D similarity space with mapping attributes of 1) color → risk stratification (red, very high risk; orange, high risk; blue, medium risk; green, normal; and purple, unknown), 2) shape → gender (O, female; X, male), and 3) bar → status (top-bar, deceived; no-bar, survived).
It shows that most of the deceased patients are located in the top-left area.
3D visualization of the entire patient sample.
The entire 100 patients using the orange-purple color-blind friendly scheme using the same dataset as in Fig.2.
1) Main colors (spheres) represent year of birth, 2) outer colors (signs) represent risk stratifications (red, very high risk; orange, high risk; yellow, medium risk; green, normal; and grey, unknown), and 3) status (bright outer-bound, deceived; no-out-bound, survived).
Visualization at an exploration stage.
Patients with medium risk who were treated with the Study 8 and BFM 95 protocols.
Visualization showing the mapping by a probset.
Probeset &#34;212022_s_at&#34; is applied to specify the colors across the population; green color corresponds to high values and red color corresponds to low values.
It indicates clearly the significant difference of patients in the bottom-left cluster, who are mostly painted with &#34;greenish&#34; colors corresponding to the high values, in comparison to reddish colors corresponding to the low values.
Visualization with clustering.
Four groups of patients in the cohort corresponding to four colors (red, orange, green, and blue).
The first group (red color) contains mostly deceased patients (with a bar on top).
Detailed view of the genomic and biological data of a patient.
Detailed view for patient ALL323.
The top-left panel displays information of the Affymetrix expression microarrays; the top-right panel displays information of the Illumina SNPs; the middle table displays all background information of the patient; and the bottom table displays the treatment details.
The view also highlights the pattern of the gene expression data and a gap in the single nucleotide polymorphism data.
Visualization of a patient at various navigational stages and options.
Views of patient ALL323 when the genes of interest are emphasized from the structure (A), the view shows the genes of interest only (B), the significantly different genes are highlighted (C), the significantly different genes are highlighted within the genes of interest (D), zoom in is applied to single-nucleotide polymorphism data (at the right-panel) (E), and further zoom is applied to gene expression data (at the left-panel) (F) so that detailed information of the probe sets in genes is displayed.
Comparison windows for patients.
Comparison windows for three patients, namely ALL323, ALL48, and ALL134, respectively.
The difference is highlighted in the yellow boxes in the single nucleotide polymorphism panel and the corresponding enlarged red boxes.
Genes of interests visual analysis for patients.
There are 5 separate visual components the primary patient probeset heatmap component (component A), the gene zoom component (component B), the color gradient and Gaussian curve component (component C), the gene ontology and probeset annotation component (component D), and the patient biomedical data component (component E).
A seamless analysis from overview to detailed view to genes of interest.
Transition view from overview of three patients (ALL123, ALL26, and ALL302) in the similarity space (A) to patient-to-patient comparison view (B) and finally to the genes of interest view (C).
Genetic variation in the clusters of patients.
All probe sets in gene MKI67 (highlighted) in group 0 (or cluster 0) are significantly different from the other groups.
Genetic variation in patients despite the similarity in biomedical data.
Details of several genes of interest for the four patients: ALL92, ALL129, ALL321, and ALL323.
The patients&#39; clinical properties are similar, including medium risk, male, deceased, and Study 8 protocol.
However, the Affymetrix expression microarrays and Illumina SNP values are quite varied among these patients.
The survival rate by year of treatment: treatment within 5 years from birth (A), treatment after within 5 to 10 years from birth (B), and treatment after 10 years from birth (C), respectively.
Early treatments are more effective than late treatments.
The genetic property of high-risk patients.
For the very high-risk patients, the genetic property of patient ALL302 (the only survivor at the bottom-right) is significantly different from the others (at the top-left).
Further examination using the 250 genes of interest confirms this hypothesis for the expression values, especially for the genes ABHD10, LEPREL1, CNOT4, CHD9, EPOR, RASAL2, ZNF141, and SLC25A31.
Views of patients who are females (A) and males (B), respectively.
Female patients tend to fare better than male patients overall.
Views of patients with different protocols, including Study 8, BFM 95, Interfant 99, and others.
The larger items represent deceased patients, and the colors represent the protocols, including Study 8 (green), BFM 95 (blue), Interfant 99 (orange), and others (red).
Views of patients with different treatment protocols, including chemotherapy and bone marrow transplant.
Chemotherapy is more effective in treatment (green color).
It is interesting that nearly one-half of all patients did not have any treatments (blue color).
The untreated patients mostly had standard to medium risks.
Title: Semantic Modeling for SNPs Associated with Ethnic Disparities in HapMap Samples

Single-nucleotide polymorphisms (SNPs) have been emerging out of the efforts to research human diseases and ethnic disparities.
A semantic network is needed for in-depth understanding of the impacts of SNPs, because phenotypes are modulated by complex networks, including biochemical and physiological pathways.
We identified ethnicity-specific SNPs by eliminating overlapped SNPs from HapMap samples, and the ethnicity-specific SNPs were mapped to the UCSC RefGene lists.
Ethnicity-specific genes were identified as follows: 22 genes in the USA (CEU) individuals, 25 genes in the Japanese (JPT) individuals, and 332 genes in the African (YRI) individuals.
To analyze the biologically functional implications for ethnicity-specific SNPs, we focused on constructing a semantic network model.
Entities for the network represented by &#34;Gene,&#34; &#34;Pathway,&#34; &#34;Disease,&#34; &#34;Chemical,&#34; &#34;Drug,&#34; &#34;ClinicalTrials,&#34; &#34;SNP,&#34; and relationships between entity-entity were obtained through curation.
Our semantic modeling for ethnicity-specific SNPs showed interesting results in the three categories, including three diseases (&#34;AIDS-associated nephropathy,&#34; &#34;Hypertension,&#34; and &#34;Pelvic infection&#34;), one drug (&#34;Methylphenidate&#34;), and five pathways (&#34;Hemostasis,&#34; &#34;Systemic lupus erythematosus,&#34; &#34;Prostate cancer,&#34; &#34;Hepatitis C virus,&#34; and &#34;Rheumatoid arthritis&#34;).
We found ethnicity-specific genes using the semantic modeling, and the majority of our findings was consistent with the previous studies - that an understanding of genetic variability explained ethnicity-specific disparities.
Tremendous efforts have been made to identify ethnicity-specific single-nucleotide polymorphisms (SNPs) in human diseases [1, 2].
Dosage effect or disease susceptibility is not equal among ethnic groups.
Type 2 diabetes prevalence is 12.8%, 8.4%, and 6.6% in non-Hispanic blacks, in Mexican-Americans, and in non-Hispanic whites aged 20 years or older [3].
Ethnic differences were causes of environmental exposure and experiences with health care, and these ethnic disparities likely influence different outcomes in people with certain diseases [4].
Genetic differences also cannot ignore its important role in determining ethnic disparities.
Despite recent advances in the genetic studies for ethnic disparities, little is known about how ethnic genetic differences contribute to inter-ethnic disparities in disease risk or disease-related complex traits [5].
Recently, semantic modeling has gained attention as a powerful tool for organizing and integrating biological big data [6].
Semantic technology is needed to provide the knowledge to gain an adequate interpretation of integrated biological systems [7].
Recent advances in ontology development, like semantic modeling, are considered to contribute to the next-generation approach by enabling the researcher to actually ask scientific questions instead of constructing complicated databases for scientific questions and answers [8].
This combination of data integration and visualization could provide important insights into heterogeneous data on millions of genes, chemical compounds, diseases, and pathways [9, 10].
To model a semantic network, the BioXM software is a customizable knowledge management program for large scientific data, and the latest solution is designed to provide meaningful interactions through graphical browsing [11].
Through an advanced query builder, the knowledge consisting of many different and connected queries is flexibly examined.
In this way, models for a research project can be constructed and extended effectively.
Many data modeling studies and software developments have been advanced, but there are relatively few studies using semantic modeling for ethnicity-specific SNPs.
To evaluate the ethnic disparities, in the current study, we identified ethnicity-specific SNPs for three populations, including North Americans with European ancestry (CEU), Japanese from Tokyo (JPT), and Yorubans from Ibadan (YRI) from HapMap, and constructed a semantic model for ethnicity-specific SNPs.
We expect that semantic model-based research will provide valuable information on ethnicity-specific gene-based SNPs and strongly affect useful knowledge in ethnic disparities.
Title: Study subjects

We downloaded the SNPs data from Haplotype Map (HapMap) phase 3 (http://www.hapmap.org) for CEU (Utah residents with Northern and Western European ancestry), JPT (Japanese in Tokyo, Japan), and YRI (Yoruba in Ibadan, Nigeria).
We focused on the gene-based SNP associations in the three ethnicities, because ethnicity is a highly heritable polygenic quantitative trait of biomedical importance.
Ethnicity-specific SNPs were obtained by eliminating common SNPs.
Title: Enrichment analysis for SNP-based gene set

Ethnicity-specific HapMap SNPs were mapped to genes from UCSC RefGene (http://genome.ucsc.edu; ver.hg18) [12].
For the mapped genes, gene set enrichment analysis (GSEA) was performed using the Database for Annotation, Visualization, and Integrated Discovery (DAVID ver.6.7) [13] with Gene Ontology (GO) terms, including biological process (BP), cellular component (CC), and molecular function (MF).
The p-values were calculated for the probability of getting a set of genes within a given GO group.
Title: Semantic modeling

To look for diverse interactions of ethnicity-specific SNPs, we constructed a semantic model using BioXM [7], which efficiently manages knowledge, such as complex scientific research data.
The model provides semantic networks with useful relationship information between participating entities.
Our semantic model consists of seven entities, including &#34;Gene [14],&#34; &#34;Pathway [14],&#34; &#34;Disease [14],&#34; &#34;Chemical [14],&#34; &#34;Drug [15],&#34; &#34;SNP [12],&#34; and &#34;ClinicalTrials (http://www.clinicaltrials.gov)&#34;, and 10 relations, including &#34;Pathway-Gene,&#34; &#34;Disease-Pathway,&#34; &#34;Disease-Chemical,&#34; &#34;Gene-Disease,&#34; &#34;Gene-Chemical,&#34; &#34;SNP-Gene,&#34; &#34;Chemical-Pathway,&#34; &#34;Chemical-Drug,&#34; &#34;ClinicalTrials-Disease,&#34; and &#34;Drug-ClinicalTrials.&#34;
Conversion of all data to entity input format was parsed using Python.
Title: Discovery of ethnicity-specific SNPs

We identified ethnicity-specific SNPs by eliminating common SNPs from HapMap samples and mapped the SNP positions to the UCSC RefGene lists.
22, 25, three 332 genes were identified in the CEU, in the JPT individuals, and in the YRI individuals, respectively (Fig.1).
Comparison of the three sets showed that YRI individuals had a biased order of SNP-based genes.
This result was a consensus among previous evolutionary findings.
CEU and JPT belong to the same cluster, together with Amerindians and Australopapuan, while YRI belongs to a separate cluster showing the first split between Africans and non-Africans [16, 17].
African populations subdivided from other sub-Saharan African populations, and a small subset of this population migrated out of Africa in the past 100,000 years.
African and non-African populations divided in the past 40,000 years.
Phylogenetic analysis of Y chromosomal haplotypes, mtDNA, and autosomes are indicative of the longest history of population subdivision in Africa.
Africans are the most ancestral population in human and have fewer sites in linkage disequilibrium, compared with non-African populations [18].
To explore the meaningful biological information of structural variations, we performed GSEA for the SNP-based genes using GO categories (BP, CC, and MF) in the DAVID tool.
The significantly categorized functions (p &lt; 0.01) of SNP-based genes for YRI are shown as pie charts in Fig.2, but none was significantly enriched for CEU and JPT.
Six groups of BP and four groups of MF had significant enrichment score ranges of 1.67-4.85 and 7.16E-04-0.002, respectively.
The top pie chart in BP presents G-protein-coupled receptor protein signaling pathway, including chemotaxis, and defense response to bacterium (Fig.2A).
In the enriched region, 8% of BP was chemotaxis (GO: 0006935) with an enrichment score of 3.88.
Chemotaxis contributes to enhancement of disease aggressiveness in African-Americans [19].
The MFs that were significantly enriched were G-protein-coupled receptor activity and binding, olfactory receptor activity, and transmembrane receptor activity (Fig.2B).
Enriched functions in cellular components were keratin filament (GO:0045095) with an enrichment score of 5.86, which contained the KRTAP gene family (KRTAP12-3, KRTAP4-11, KRT14, KRTAP4-4, KRTAP9-8, KRTAP10-7, KRTAP10-8).
KRTAP genes are up-regulated in white hair than in black hair by a microarray analysis.
Immunoreactivity for KRTP genes in white hair follicles was increased compared with black hair.
Therefore Choi et al.
[20] suggested that greying hair, a sign of aging, is associated with hair growth rate.
Title: Semantic modeling for ethnicity-specific SNPs

Semantic modeling is an emerging method for comprehensively understanding complicated BPs and spacious networks [7].
The continuous production of increasingly large-scale data in biology needs better visualization of complex and huge biological data.
We constructed a semantic network model in order to analyze biologically functional implications for ethnicity-specific SNPs.
Overall, network entities were used, such as &#34;Gene&#34; (records, 46,354), &#34;Pathway&#34; (records, 362), &#34;Disease&#34; (records, 9,647), &#34;Chemical&#34; (records, 153,021), &#34;Drug&#34; (records, 6,712), &#34;ClinicalTrials&#34; (records, 1,273), and &#34;SNP&#34; (records, 379), and pairwise relationships between entity-entity were curated as &#34;Gene-Pathway&#34; (records, 46,354), &#34;Gene-Disease&#34; (records, 18,391,755), &#34;Gene-Chemical&#34; (records, 308,405), &#34;Disease-Chemical&#34; (records, 401,145), &#34;Disease-Pathway&#34; (records, 43,139), &#34;Chemical-Pathway&#34; (records, 196,073), &#34;Chemical-Drug&#34; (records, 1,702), &#34;SNP-Gene&#34; (records, 379), &#34;ClinicalTrials-Drug&#34; (records, 1,419), and &#34;ClinicalTrials-Disease&#34; (records, 1,210).
Entities, including &#34;Pathway,&#34; &#34;Chemical,&#34; and &#34;Disease,&#34; were collected from the Comparative Toxicogenomics Database (CTD) [14, 21], which is a public database to promote the understanding of the interaction of genes, chemical compounds, and disease networks in human health.
Drugs were mapped from DrugBank [15, 22], which provides detailed drug action information.
We linked a novel relationship for &#34;Chemical-Drug&#34; and &#34;Gene-SNP&#34; by curating the relationship of entities using Python ver.2.6, and the remaining relations were collected from the CTD.
Fig.3 shows that semantic modeling of ethnicity-specific SNPs is dynamic and flexible.
Hierarchy structure is where the parent can have one child, while in Directed Acyclic Graph (DAG) networks, like BioXM, the parent can have more than one child.
For example, Gene A is associated with Chemical B or Pathway C. Also, Gene A is associated with Drug C, because Gene A is a curated interaction with Disease B, and Disease B is a curated association with Drug C.
Title: Ethnicity-specific SNPs reveals association with 3 diseases and 1 drug

Diseases and drugs are very clinically important for understanding ethnic disparities.
Many diseases and drugs have been reported to be involved in ethnic disparities, disease susceptibility, drug response, and disposition [23, 24, 25].
We curated &#34;SNP-Gene-Disease-Chemical-Drug&#34; interactions in the semantic networks for ethnicity-specific SNPs.
Using these semantic &#34;Gene-Disease&#34; networks, we analyzed the functional implications of ethnic variants.
There were 123 diseases associated with ethnicity-specific SNPs in common populations, 3 CEU-specific, and 46 YRI-specific, but JPT had no specified disparity between different ethnic populations (Supplementary Fig.1A).
Three diseases associated with CEU-specific SNPs were shown as phantom limb (MESH:D010591), trochlear nerve diseases (MESH: D020432), and vulvitis (MESH:D014847), while diseases associated with YRI-specific SNPs were observed, such as acquired immune deficiency syndrome (AIDS)-associated nephropathy (AIDSAN), hypertension, primary amyloidosis, and pelvic infection.
AIDSAN (MESH:D016263) rates are higher in African-Americans than whites.
Although the mortality and morbidity from AIDS infection are reduced, AIDSAN remains a major complication of AIDS infection (http://statgen.ncsu.edu/).
Hypertension (MESH:C537095) is a disease threatening the public health in sub-Saharan Africa.
In some areas, blacks exhibit higher rates of hypertension than whites.
Increased salt intake and obesity are the leading causes of the prevalence of hypertension in Africa [26].
Pelvic infection (MESH:D034161) is a kind of inflammatory disease that blacks are more prone to take than other ethnic groups [27].
By applying the &#34;SNP-Gene-Disease-Chemical-Drug&#34; model, 2 and 14 drugs were revealed with CEU-specific and YRI-specific groups, but JPT-specific drugs had no results (Fig.4, Supplementary Fig.1B).
One drug (methylphenidate, DB00422) was reported to have ethnic disparities in previously drug studies.
The mean dose of methylphenidate is about 1.5 times higher in African-Americans than whites [28], and its use is steadily increasing in South Africa [29].
Title: Ethnicity-specific associations with 5 pathways

Analysis using the semantic model for ethnicity-specific SNPs identified 5, 7, and 100 CEU-specific, JPT-specific, and YRI-specific biochemical pathways, respectively.
In hemostasis (REACT:604), associated with cardiovascular diseases, plasminogen activator inhibitor-1 activity levels of Africans are lower compared to Caucasians.
These negative effects can be seen already at a young age.
If addressed in early life, it is possibly adjustable through behavior and optimal dietary changes [30].
Systemic lupus activity measure (SLAM; KEGG:05322) scores were higher in African-Americans (mean = 12.6) and Hispanics (11.0) than in Caucasians (8.5).
It caused lack of health insurance, onset of abrupt disease, presence of anti-Ro (SSA) antibody, absence of HLA-DRB, high levels of helplessness, and abnormal illness behaviors.
Caucasians lived under less crowded conditions, had less abnormal illness behaviors, and had more education.
The results of the regression analyses were showed significant association between higher SLAM scores and higher helplessness, absence of HLA-DRB1*0301, and presence of HLA-DRB*0201 (p &lt; 0.01) [31].
Prostate cancer (KEGG:05215) is a diagnosed male reproductive system cancer.
Incidence of prostate cancer in African-American men is higher than in European men (1.6 times).
Amundadottir et al.
[32] identified that the chromosomal 8q24 region is most frequently gained in prostate cancers, and this gained region has been correlated with aggressive tumors [33].
Estimated population attributable risk is greater in Africans than in European populations.
Hepatitis C virus (HCV; KEGG:05160) is a major cause of chronic liver disease in humans.
Rates of HCV prevalence in sub-Saharan Africa are the highest in central Africa (3.0%) compared with the median (2.2%).
Conjeevaram et al.
[34] showed that African-Americans with chronic HCV have lower response to interferon-based antiviral therapy than Caucasian Americans [35].
Rheumatoid arthritis (RA; KEGG:05323) is an autoimmune disease and may affect many organs.
The RA prevalence in urban South Africans is similar in Caucasians [36].
In the current study, the pathways shared between all populations were signal transduction (REACT:111102), olfactory transduction (KEGG:04740), and metabolic pathways (KEGG:01100).
These pathways were common disease-pathway interactions in previous research.
Although ethnicity-specific genes are identified in each population, it is generally observed that genes that are associated with a trait or disease can converge to the same pathway [37].
Those genes are also supposed to converge to common pathways shared between all populations.
Therefore, a pathway-based approach allows us to systematically evaluate multiple polymorphic genes from different populations with respect to pathways as a biological unit [38].
Moreover, the pathway-based approach has more capability to detect rare genetic variants with a small effect that do not survive at the stringent significance level [39].
We identified ethnicity-specific SNPs from HapMap data and constructed a semantic network model for the HapMap SNP dataset.
Functional studies were analyzed with genebased ethnicity-specific SNPs.
Our semantic network model showed robust interactions between ethnic-specific SNPs and public data.
However, this model is still in the early stage, and greater data connection and development of more flexible algorithms are required.
We expect that our semantic network model is useful for ethnicity-specific SNPs, and our findings will provide prioritization of ethnicity-specific gene-based SNP candidates.
Title: Identification of Ethnically Specific Genetic Variations in Pan-Asian Ethnos

Asian populations contain a variety of ethnic groups that have ethnically specific genetic differences.
Ethnic variants may be highly relevant in disease and human differentiation studies.
Here, we identified ethnically specific variants and then investigated their distribution across Asian ethnic groups.
We obtained 58,960 Pan-Asian single nucleotide polymorphisms of 1,953 individuals from 72 ethnic groups of 11 Asian countries.
We selected 9,306 ethnic variant single nucleotide polymorphisms (ESNPs) and 5,167 ethnic variant copy number polymorphisms (ECNPs) using the nearest shrunken centroid method.
We analyzed ESNPs and ECNPs in 3 hierarchical levels: superpopulation, subpopulation, and ethnic population.
We also identified ESNP- and ECNP-related genes and their features.
This study represents the first attempt to identify Asian ESNP and ECNP markers, which can be used to identify genetic differences and predict disease susceptibility and drug effectiveness in Asian ethnic populations.
There has been an explosion of data describing genetic variants in humans.
Structural genetic variations, such as single nucleotide polymorphisms (SNPs) and copy number variations (CNVs), have given rise to myriad differences in human populations [1].
The study of human genetic variations has both evolutionary significance and medical applications and can help scientists understand ancient human population migrations as well as how different human groups are biologically related to one another [2, 3].
SNPs represent the most frequent type of human DNA variation [4].
The main goals of SNP research include understanding the genetics of the human phenotype variation and, especially, the genetic basis of human diseases [5, 6].
Genome-wide linkage and association studies have been made possible by highly accurate methods for typing SNPs [7].
CNV is a form of genomic structural variation that results in the cell having an abnormal number of copies of one or more sections of DNA [8].
CNVs can be limited to a single gene or include a contiguous set of genes.
CNVs can result in having either too many or too few dosage-sensitive genes, which may be responsible for a substantial amount of human phenotypic variability, complex behavioral, traits and disease susceptibility [9].
A copy number polymorphism (CNP) is a CNV that occurs in more than 1% of the population [10].
SNP platforms can also be used for typing CNVs.
This allows for generalized genotyping of both SNPs and CNVs simultaneously on a common sample set, with advantages in terms of cost and unified analysis.
Although CNP detection from SNP genotyping data is a difficult task and has the limitation of false positive results, it will provide a more comprehensive view of genomic variations and complement genome-wide association studies in identifying disease susceptibility loci.
SNPs are being used in studies of human migration and evolution, as well as those of human health.
The Human Genome Organization (HUGO) Pan-Asian SNP Consortium reported a large-scale survey of autosomal variations from a broad geographic sample of 72 Asian human populations [11].
The study indicated that most populations show relatedness within ethnic or linguistic groups, despite significant gene flow among groups.
This relatedness may have important implications for our understanding of genetics and disease.
Data on ethnic populations in the Pan-Asia region can be valuable to show the spectrum of genetic diversity and networks of ethnic groups.
Thus, notwithstanding the population size problem in some ethnic groups, we investigated the ethnic specificity based on SNP and CNP information.
Here, we identified genetic variations-ethnically specific SNPs (ESNPs) and ethnically specific CNPs (ECNPs)-of Asian populations using SNP genotypic profiling.
These ESNP and ECNP markers can be used to identify genetic differences and to predict disease susceptibility and drug effectiveness in Asian populations.
Title: Dataset

We obtained a genome-wide 58,960-SNP dataset (Affymetrix GeneChip Human Mapping 50K Xba chip; Affymetrix Inc., Santa Clara, CA, USA) from the HUGO Pan-Asian SNP Consortium website (http://www4a.biotec.or.th/PASNP) [12].
The SNP data were obtained from 1,833 individual DNA samples collected from blood and buccal samples of 72 Asian and non-Asian ethnic groups.
Filtering SNPs is an important step in genome-wide studies to minimize potential false findings.
We filtered SNPs at 2 levels: individual and SNP.
In the individual-level filtering step, we removed individual SNPs that had a &lt;90% genotyping call rate and that were not in Hardy-Weinberg equilibrium (p &lt; 0.001).
We filtered out 89 of the initial 1,833 individuals and used SNPs from the 1,744 individuals for further study.
In the SNP-level filtering step, 1,692 of the initial 58,960 SNPs were filtered out, because 389 SNPs had an SNP call rate below 90%, 451 SNPs were physically unmapped, 103 SNPs could not be mapped to dbSNP, and 749 SNPs were monomorphic.
Finally, we obtained 56,025 SNP markers from 1,744 genotyped individuals, representing 72 ethnic groups in the HUGO Pan-Asian SNP Consortium.
For comparison with other ethnic variants, we obtained SNPs of 209 HapMap individuals representing 4 populations (China Han [CHB]; Japan Japanese [JYP]; USA European [CEU]; Africa Yoruba [YRI]) (http://www.hapmap.org/).
The 243 SNPs in the HUGO Pan-Asian with the HapMap dataset were nonshared SNP loci and excluded.
Title: Haplotype phasing by fastPHASE

Phasing is needed to determine which variants are inherited by an individual at each locus and to more accurately determine the relationships between unrelated individuals in large population datasets.
Using phasing population data, we can identify haplotypes, which are essentially segments of DNA that are common to a particular ethnic group.
The fastPHASE program version 1.1.4 [13] was used to estimate missing genotypes and reconstruct haplotypes from unphased SNP genotype data of unrelated individuals in order to identify ethnicity-specific SNPs.
Title: Analysis of CNPs

SNP genotyping arrays recently have been used for CNP detection and analysis, because the arrays can serve dual roles for SNP- and CNV-based association studies.
To detect CNP markers from the SNPs, we used the Affymetrix Genotyping Analysis Software and Copy Number Analysis Tool (CNAT version 3.0), which was downloaded from the Affymetrix website (http://www.affymetrix.com/products/software).
We calculated the distribution of genomic smoothing copy number signal intensity (upper-boundary = 2.78, lower-boundary = 1.51) to detect the CNPs (Supplementary Fig.1).
We regarded markers with values beyond the upper- and lower-boundary values as CNVs.
Then, we selected the CNPs that presented variation in &gt;1% of the individuals as CNVs that occurred in more than 1% of the population [14].
We downloaded the detection tools from the website (http://www.affymetrix.com/products/software) and used these programs for our Pan-Asian SNP data.
We regarded markers with values outside of the upper- and lower-boundary values as CNVs.
Then, we selected CNPs which presented variation in &gt;1% of the 1,196 individuals.
Title: Selection of ethnicity-specific genetic variations: ESNPs and ECNPs

To investigate the distribution of ESNP and ECNP markers in Pan-Asian ethnic groups, we performed the following steps, as shown in Supplementary Fig.2.
First, we classified the ethnic groups into 3 hierarchical categories by population estimation of the phylogenic analysis (in Supplementary Fig.2): superpopulation (clustering size = 4), subpopulation (ESNP clustering size = 12, ECNP clustering size = 11), and ethnic population (ESNP clustering size = 72, ECNP clustering size = 60).
The clustering size in parentheses refers to how many distinct populations were used in the ESNP and ECNP analysis.
The superpopulations consisted of African, Caucasian, American Indian, Asian, and outlier (Indian and Uyghur) populations.
The subpopulations comprised population groups based on linguistic similarities and population structures and 2 additional groups (Mongoloid features [IN-NI]/Mongoloid features [IN-TB] and China Uyghur [CN-UG]), and the ethnic populations represented each ethnic group.
Second, we identified ethnicity-specific variations based on SNPs and CNPs markers using the nearest shrunken centroid method (NSCM), which is not affected by the minor-allele frequency [15, 16].
The NSCM of the R package pamr carried out multiple tests for genetic heterogeneity and frequency spectrum on genes having ethnicity-specific variants.
It shrinks each of the class centroids toward the overall centroid for all classes by a threshold.
This shrinkage makes the classifier more accurate by eliminating the effect of noise.
This method is a suitable attempt to solve the classification problem when there are a large number of features from which classes and a relatively small number of cases are predicted, and it is a significant approach to identify which features contribute most to the classification [15].
Third, we identified specific ethnic groups that correlated with each SNP or CNP.
Finally, to investigate the characteristics of ethnicity-specific variations, we obtained the top 10% of ESNPs and ECNPs markers.
Title: Functional analysis of ESNP and ECNP markers

To investigate the characteristics of the ECNP- and ESNP-related genes, we selected the ECNP and ESNP markers ranked in the top 1% of the ethnically specific variations.
We mapped ESNP and ECNP markers to gene structure, based on Entrez gene-centered information at NCBI [16].
We identified the relationships among ESNP- and ECNP-genes using the Ingenuity Systems Pathway Analysis tool (http://www.ingenuity.com), which is a web-based tool for pathway and network analysis of genes.
Title: Genotype features

We obtained 58,960 Pan-Asian SNPs from 72 Pan-Asian populations listed in the supplementary documents (Supplementary Table 1).
The SNP dataset shows the spectrum of genetic and phenotypic diversity in Asian populations and allows us determine the ethnic specificity of Asian populations based on ESNP and CNP makers.
We devised an approach to identify ethnic differences from the genotype profiles of the populations.
Pan-Asian SNPs were filtered using several steps, as described in Materials and Methods.
With the filtered SNPs, we examined the inter-SNP distances, the allele frequency, and the heterozygosity distribution to identify the genotypic features of each ethnic group.
The average inter-SNP distance was 52 kb.
More than 41% of SNPs fell below 10 kb, and 14% was over 100 kb (Supplementary Fig.3).
The inter-SNP distance distribution within 10 kb of one another was similar to the distribution observed in the HapMap data.
We investigated minor allele frequency and heterozygosity across the ethnic groups (Supplementary Fig.4).
We compared SNP genotypes in the ethnic groups with those identified by the HapMap [17] consortium.
The proportion of Pan-Asian SNPs in each minor allele frequency and heterozygosity range was similar to those of the 4 HapMap ethnic groups.
In particular, the SNP proportions for CN-UG, Proto-Austroloids (IN-DR), and Singapore Indian (SG-ID) were almost the same as those of the 4 HapMap ethnic groups.
We found that 5 ethnic groups (America Melanesians [AX-ME], Thailand Mlabri [TH-MA], Indonesia Mentawai [ID-MT], CN-UG, and SG-ID) had minor allele frequency and heterozygosity distributions that were significantly different from those of other ethnic groups.
Approximately 40% of the SNPs in the AX-ME, TH-MA, and ID-MT populations fell into the &lt;0.05 range.
The proportion of SNPs with heterozygosity &lt;0.05 was lowest in the CN-UG and SG-ID groups, whereas the proportion of SNPs with heterozygosity &lt;0.10 was highest in the TH-MA and AX-ME groups.
Title: Distribution of ESNPs and ECNPs

We confirmed the high proportion of African (YRI)-specific SNPs from 3 races: Asian, Caucasian, and African (Supplementary Fig.5).
Then, to search ESNPs from Pan-Asian ethnic groups, we identified 9,306 ESNPs from the Pan-Asian SNP markers and examined the distribution of the ESNPs at the 3 population levels: super-, sub-, and ethnic population (Fig.1A).
We found that 67.82% of ESNPs in the superpopulation level were others, including IN-NI, IN-TB, and CN-UG.
It could show that isolated ethnic groups keep their original features and that these features are also presented in other population levels.
In the subpopulations, Austronesian, IN-NI, and IN-TB had 95.31% of ESNPs.
Additionally, we identified 28,282 CNPs based on the SNP chip profiles and then selected 5,167 ECNPs.
We then examined the ECNPs in the 3 population levels (Fig.1B).
In the superpopulations, 36.57% of the ECNPs occurred in Caucasoids.
Of the subpopulation ECNPs, 56.26% had Mongoloid features and 36.95% occurred in Uyghurs.
Across the ethnic groups, 26.35% of the ECNPs were from Caucasoids (IN-NL).
Most ECNPs occurred in Caucasoid populations (IN-NL, IN-IL, IN-DR, IN-SP, IN-EL, IN-WI, IN-WL, and SG-ID) and in ethnic groups that have Mongoloid features (IN-NI, IN-TB, and CN-UG).
The ethnic group having Mongoloid features had more ECNPs than other groups, which shows that Pan-Asian ECNPs could serve as the criteria for ethnic specificity in Pan-Asian populations.
Title: The features of genes associated with ESNPs and ECNPs

We obtained genes associated with ESNPs and ECNPs based on the NCBI Gene database [18].
We identified 156 ESNP-related genes and 52 ECNP-related genes (Supplementary Table 2).
Most of the ESNP-containing genes were associated with known molecular functions encoded for cellular assembly and maintenance, cellular movement, cell death and survival, and lipid metabolism (Table 1).
These ESNP-related gene sets (CEP192, GRK5, TUBGCP3, TUBGCP6, ABCA1, CHGA, ITGB3, VAV2, MYLK3, ANK2, ITGB3, ELN, LTBP4, FAM107B, DIDO1, HOXD3, PRPX2, KCNMA1, RASGRF1, NF1, and UBR2) showed association with diseases, such as cardiovascular disease, dermatological diseases and conditions, developmental disorders, and connective tissue disorders.
Defects in latent transforming growth factor beta binding protein 4 (LTBP4) may be a cause of cutis laxa and severe pulmonary, gastrointestinal, and urinary abnormalities [19].
Centrosomal protein 192 kDa (CEP192) is part of a large multisubunit complex required for microtubule nucleation at the centrosome.
This gene is regulated by hepatitis B virus and has roles in cells, such as duplication, replication, assembly, and nucleation.
The ECNP-related genes are associated with known molecular functions encoded for cell-to-cell signaling and interaction, molecular transport, and drug metabolism (Table 1).
These ECNP-containing genes (ARID1B, CAPN7, DDR2, SLC22A1, SLC22A3, ASTN2, GABRA5, GABRG3, RBFOX1, TLR4, MAG, TLR4, SNX9, and SYNJ2) are associated with cardiovascular disease, neurological disease, psychological disorders, and developmental disorders.
Interestingly, the APOBEC gene, encoding the C→U editing enzyme family, such as apolipoprotein B mRNA editing enzyme and catalytic polypeptide-like 2, is associated with ECNPs.
This editing gene could be affected by genetic diversity and admixtures in Pan-Asian populations.
In this ethnically specific genetic variation study, we found the features of gene sets having ethnically specific variations.
The AT-rich interactive domain 1B (SWI1-like) (ARID1B) gene encodes an AT-rich DNA interaction domain-containing protein.
This protein is a component of the SWI/SNF chromatin remodeling complex and may play a role in cell-cycle activation.
It is associated with chromatin-mediated maintenance of transcription and nervous system development.
This gene is related to autosomal dominant mental retardation type 12.
Discoidin domain receptor tyrosine kinase 2 (DDR2) is associated with colorectal neoplasm, metastatic colorectal cancer, hepatocellular carcinoma, gastrointestinal stromal tumor, and so on.
These ethnically specific genetic variations represent relationships with cell cycling, metabolism, and the developmental and immune systems.
These genetic variations can be affected by admixture and evolution.
These ethnically specific variants may be forced to adapt in order to survive in new environments and play a significant role in development and functional systems associated with diseases [20].
Title: Conclusion

Our analysis was able to identify ethnically variable SNPs associated with phenotypic changes.
We selected 9,306 ESNPs and 5,167 ECNPs in 72 Pan-Asian populations.
We found that representative ethnic groups with specific ESNPs are recently branched-out subpopulations, whereas representative ethnic groups with ethnic specific CNPs are early fixed subpopulations, as shown in Supplementary Fig.6.
This likely occurred due to accelerated and accumulated genetic drifts or selective pressure.
This shows that ESNPs may participate in the ongoing creation of genetic variation through selective pressure or selective sweep.
These ethnically specific variations are associated with 156 ESNP-related genes and 52 ECNP-related genes.
Ethnically specific genetic variations may affect phenotype variation and disease susceptibility.
Although it is not enough information to compare ECNP distribution and ESNP distribution directly, significant gene sets having ESNPs or ECNPs can be useful to study disease risk and drug susceptibility.
Distribution of ethnic variant single nucleotide polymorphisms (A) and ethnic variant copy number polymorphisms (B) across ethnic groups.
PG8 consists of Indo-European and Dravidian Southwest Asians.
IN-NI, Mongoloid features; IN-TB, Mongoloid features; TH-MA, Mlabri; MY-MN, Malay; ID-ML, Malay; ID-LE, Lembata; PI-UI, Filipino; ID-DY, Dayak; PI-UN, Filipino; ID-SB, Kambera; IN-NL, Caucasoids; CN-UG, Uyghur; IN-EL, Caucasoids; IN-IL, Caucasoids; IN-SP, Caucasoids; JP-RK, Ryukyuan.
Title: ESNP- and ECNP-related gene set summary
Title: Editor&#39;s Introduction to This Issue

Recently, next-generation sequencing (NGS) technologies have become the most important part of genetic testing to understand life phenomena and disease mechanisms.
As one of the important moves towards personalized medicine, NGS technology is being rapidly introduced to the clinical field.
However, considering the cost and time for the analysis after sequencing per se, whole-genome or whole-exome sequencing might not be the most efficient genetic testing methods yet.
Regarding this issue, Dr. Byung Chul Kim&#39;s group, Theragen Bio Institute, reports a fast and cost-effective NGS method to identify somatic mutations in lung cancer.
They chose 30 genes potentially associated with lung cancer development, performed target gene capture, and analyzed the findings by NGS.
This approach could be useful for detecting well-known key mutations as well as novel variants in lung cancer.
Dr. Hyung Doo Shin&#39;s group, Sogang University, hypothesized that polymorphisms in the tyrosine-protein kinase Tec, which is involved in various immune responses, might be involved in aspirin-exacerbated respiratory disease pathogenesis.
Although they did not observe a positive association, this study can help us understand the genetic component of this condition.
Dr. Dorairaj Sudarsanam&#39;s group, Loyola College, India, reported the results of their exploration of the Gm18 and m1G37 modification positions in tRNA sequences.
Dr. Satish Kumar&#39;s group, Mahatma Gandhi Institute of Medical Sciences, India, suggested the molecular interaction of natural inhibitors with the human papillomavirus-16 E6 oncoprotein.
In the application note section, Dr. Dong Su Yu&#39;s group, KOBIC, reports a simple and fast protein function annotation system, named SFannotation.
Genomics &amp; Informatics is already available in PubMed Central and PubMed.
Taking these efforts together, we expect broader international visibility of Genomics &amp; Informatics.
Title: New Lung Cancer Panel for High-Throughput Targeted Resequencing

We present a new next-generation sequencing-based method to identify somatic mutations of lung cancer.
It is a comprehensive mutation profiling protocol to detect somatic mutations in 30 genes found frequently in lung adenocarcinoma.
The total length of the target regions is 107 kb, and a capture assay was designed to cover 99% of it.
This method exhibited about 97% mean coverage at 30× sequencing depth and 42% average specificity when sequencing of more than 3.25 Gb was carried out for the normal sample.
We discovered 513 variations from targeted exome sequencing of lung cancer cells, which is 3.9-fold higher than in the normal sample.
The variations in cancer cells included previously reported somatic mutations in the COSMIC database, such as variations in TP53, KRAS, and STK11 of sample H-23 and in EGFR of sample H-1650, especially with more than 1,000× coverage.
Among the somatic mutations, up to 91% of single nucleotide polymorphisms from the two cancer samples were validated by DNA microarray-based genotyping.
Our results demonstrated the feasibility of high-throughput mutation profiling with lung adenocarcinoma samples, and the profiling method can be used as a robust and effective protocol for somatic variant screening.
Non-small-cell lung cancer is an increasingly common and lethal disease, accounting for 25% of all cancer deaths.
Sequencing of the lung cancer genome is of particular interest for identifying driver mutations and their pathways involved in cancer growth and development [1].
Somatic mutational profiles are crucial for cancer diagnosis and classification, which lead to tailoring the best therapeutic strategy to individual patients [2].
Previous studies using the Sanger sequencing method have identified several key mutations associated with lung cancer.
Massive PCR amplification and Sanger sequencing of 623 candidate cancer genes in 188 lung adenocarcinomas discovered 26 mutational target genes [3].
Although the study provided highly valuable results, it is very time-consuming and costly-so much so that a single laboratory can hardly perform this kind of large-scale sequencing projects.
A mass spectrometric-based mutation detection technology, named OncoMap, has been effective in identifying somatic mutations in cancer genomes [4].
Currently, it can detect more than 1,000 mutations in 112 commonly mutated genes that were previously identified as oncogenes and tumor suppressors [5].
Although OncoMap is a high-throughput method for mutational profiling with both fresh frozen and paraffin-embedded tissue samples, the mutation detection is limited to previously identified mutations, and it cannot discover novel mutations.
Recent advancements of next-generation sequencing technology have made breakthroughs in identifying unknown somatic mutations [6].
Combined with sequencing technology, targeted enrichment techniques have been developed to reduce sequencing cost and time [7].
Several recent studies have reported targeted resequencing of cancer samples using next-generation sequencing technologies [8, 9].
Here, we present a fast and cost-effective method to identify somatic mutations in lung cancer.
We surveyed the literature and chose 30 mutational target genes that were associated strongly with lung adenocarcinoma development.
Target gene capture was performed using selector probes, which involved circularization and amplification of specific restriction fragments covering the target genes using rolling circle amplification [10].
The captured target DNAs were analyzed by next-generation sequencing to find somatic variations.
This method could be useful to detect previously known recurrent mutations as well as novel variations.
Title: Preparation of genomic DNA from cancer cell lines

We ordered normal genomic DNA sample from the Coriell Institute Cell Repository (CCR ID NA17022; Camden, NJ, USA), which originated from a normal Caucasian male of European descent.
For cancer genomic DNAs, H-1650 and H-23 cancer cells were cultured and harvested for DNA preparation.
Genomic DNA (gDNA) was extracted with the QIAamp DNA blood kit according to the manufacturer&#39;s instructions (Qiagen, Hilden, Germany).
The DNA quality and quantity were assessed with the use of a Nanodrop spectrophotometer (Thermo Fisher Scientific, Wilmington, DE, USA).
Title: Design and oligonucleotides

The list of the entire exons for the 30 genes was obtained from a consensus coding sequence database, CCDS (build 36.3), showing a total number of 701 exons covering 102 kb, according to hg18 (March, 2006 assembly).
Targeted restriction fragments were selected using Disperse software [11].
Templates for circularization of each chosen targeted fragment (selector probes) were designed using ProbeMaker software [12].
Each selector probe consisted of two sequences of 20-25 nucleotides complementary to the ends of its targeted restriction fragment.
The 3&#39;-biotin-labeled oligonucleotides (Integrated DNA Technologies, Coralville, IA, USA) were prepared by incubating the oligonucleotides with 1× Tdt buffer (NEB), 1× CoCl2 (NEB), 0.1 mM dUTP-biotin (Roche Diagnostics, Mannheim, Germany), and 0.2 units/µL terminal transferase (NEB) in a final volume of 50 µL.
The reaction was incubated at 37℃ for 1 h and followed by enzyme inactivation at 75℃ for 20 min.
Title: Target enrichment

Eight different restriction reactions were used to digest genomic DNAs from each sample, including SfcI and Hpy188I in NEB buffer 4; DdeI and AluI in NEB buffer 2; MseI and Bsu36I in NEB buffer 3; MslI and BfaI in NEB buffer 4; HpyCH4III and Bsp1286 in NEB buffer 4; SfcI and NlaIII in NEB buffer 4; MseI and HpyCH4III in NEB buffer 4; and HpyCH4V and EcoO109I in NEB buffer 4 (New England Biolabs, Ipswich, MA, USA).
The restriction reactions contained 1 unit each of two restriction enzymes and their corresponding compatible NEB buffer in 1× concentration and 0.85 µg/µL bovine serum albumin (BSA) in a total volume of 10 µL.
The reactions were incubated at 37℃ for 60 min, followed by enzyme inactivation at 80℃ for 20 min.
A total of 80 µL of pooled digested sample was mixed with 10 pM biotinylated selector probes, 1 M NaCl, 10 mM Tris-HCl (pH 7.5), 5 mM EDTA, and 0.1% Tween-20 in a total volume of 160 µL.
The mixture was incubated and hybridized at 95℃ for 10 min, 75℃ for 30 min, 68℃ for 30 min, 55℃ for 30 min, and 46℃ for 10 h. The hybridized solution was mixed with 10 µL M-280 streptavidin-coated magnetic beads (3.35 × 107 beads/mL; Invitrogen, Carlsbad, CA, USA) in 1 M NaCl, 10 mM Tris-HCl (pH 7.5), 1 mM EDTA, and 0.1% Tween-20 in a final volume of 200 µL and incubated at room temperature for 10 min.
After incubation, the beads were collected using a ring magnet and washed in 1 M NaCl, 10 mM Tris-HCl (pH 7.5), 5 mM EDTA, and 0.1% Tween-20 in a total volume of 200 µL at 46℃ for 30 min with rotation.
Title: Multiple displacement amplification

Genomic fragments were circularized by incubating the beads with 1× Ampligase reaction buffer, 0.25 U/µL Ampligase (Epicentre, Madison, WI, USA), and 0.1 µg/µL BSA in a total volume of 50 µL at 55℃ for 10 min.
The circularized molecules were separated from the beads by incubation with 5 µL sample buffer at 95℃ for 10 min and collected with a ring magnet rack.
The supernatant was incubated with 5 µL reaction buffer and 0.2 µL enzyme mix (Templiphi; GE Life Sciences, Piscataway, NJ, USA) at 30℃ for 4 h, followed by inactivation at 65℃ for 10 min.
Title: Real-time quantitative PCR analysis

The enriched samples were analyzed with real-time quantitative PCR (qPCR) and DNA quantification to evaluate enrichment bias and specificity.
PCR primers were placed randomly in the targeted regions and quality controlled using standard genomic DNA.
The qPCR results were used to estimate how much target DNA was present in the amplification products, and then, the specificity (proportion target material) was estimated by measuring the amount of DNA in the reactions.
The enriched and pre-enriched control DNAs were diluted (final dilution 1:3,600) in a PCR mix containing 1× PCR buffer, 2 mM MgCl2, 1 unit Platinum Taq, 0.2 mM dNTP (10297-018; Invitrogen), 1× SYBR Green I, 10% DMSO, and 0.16 µM of either on-target primer or off-target primer to a total volume of 30 µL.
The genomic reference DNA was 10 ng of template in the same PCR mixture as described above.
The qPCR was performed using an LC480 Real-Time PCR system (Roche), and the conditions were as follows: 95℃ for 5 min followed by 40 cycles of (95℃ 15 s, 56℃ 30 s), with end-point measurement of the fluorescence after each completed cycle.
Title: Library preparation and massive parallel sequencing

Libraries were prepared according to the manufacturer&#39;s instructions (Illumina, San Diego, CA, USA).
Briefly, 5 µg of gDNA in 200 µL nuclease-free water was fragmented by a Bioruptor (Diagenode, Liege, Belgium) at high power for 30 min (30 s ON and 30 s OFF).
Overhangs of fragmented gDNA were converted to blunt ends using T4 DNA ligase and Klenow enzyme.
Subsequently, an &#39;A&#39; base was added to the ends of double-stranded DNA using Klenow exo- (3&#39; to 5&#39; exo minus).
The paired-end adaptor (Illumina) with a single &#39;T&#39; base overhang at the 3&#39; end was ligated to the products above.
The PE adaptor-ligated products were separated on a 2% agarose gel and excised from the gel from approximately 400 bp to 500 bp.
The sequencing libraries were bar-coded to allow sequencing of 6 samples in one lane of a flow cell.
Size-selected DNA fragments were enriched by PCR with PE primers 1.1 and 2.1 (Illumina).
The concentration of the libraries was measured on both a Nanodrop (Thermo Fisher Scientific) and Qubit IT (Invitrogen).
Finally, the libraries were validated by a Bioanalyzer (Agilent Technologies, Santa Clara, CA, USA).
The gDNA library was sequenced using the Illumina genome analyzer GAIIx according to the manufacturer&#39;s instructions.
Title: Short read alignment and variation calling

A total of 76-bp paired-end or single-end sequence reads with ~200-bp insert size were aligned to the human reference genome (NCBI build 37, hg19) with BWA algorithm1 ver.0.5.8c and default parameters [13].
Two mismatches were permitted in a 45-bp seed sequence.
Putative single nucleotide polymorphisms (SNPs) and small INDELs were called by the pileup vcf function of Samtools (ver.0.1.9) [14].
The options used in filtering variations were a minimum of 6 for mapping depth and quality threshold of 50 for single nucleotide variations (SNVs) and 20 for small INDELs.
Predicted SNVs were compared with NCBI dbSNP version 131 (http://www.ncbi.nlm.nih.gov/projects/SNP/) to remove known SNP information.
Non-synonymous SNV information was extracted by comparing UCSC (http://genome.ucsc.edu/) reference gene information with the somatic mutation list in the COSMIC cancer information database (http://www.sanger.ac.uk/genetics/CGP/cosmic/).
PolyPhen (polymorphism phenotyping) was utilized to predict possible changes in protein structure and function resulting from a non-synonymous amino acid change.
Title: Genome-wide SNP analysis

SNP genotyping was performed using the Axiom genotyping solution, including an Axiom Genome-Wide ASI 1 Array Plate and reagent kit, according to the manufacturer&#39;s protocol (Affymetrix, Santa Clara, CA, USA).
Briefly, total genomic DNA (200 ng) was treated with 20 µL of denaturation buffer and 40 µL neutralization buffer, followed by amplification for 23 h using 320 µL of Axiom amplification mix.
Amplified DNA was randomly fragmented into 25 to 125 base pair (bp) sizes with 57 µL of Axiom fragmentation mix at 37℃ for 30 min, followed by DNA precipitation for DNA clean-up and recovery.
DNA pellets were dried and resuspended with 80 µL of hybridization master mix; 3 µL of suspended sample was kept for sample qualification.
A hybridization-ready sample was denatured using a PCR machine at 95℃ for 20 min and 48℃ for 3 min.
Denatured DNA was transferred to a hybridization tray and loaded onto a GeneTitan MC with an Axiom ASI array plate (Affymetrix).
Hybridization continued on the GeneTitan for 24 h, followed by loading of ligation, staining, and stabilization reagent trays into the instrument.
After chip scanning, the cel intensity file was normalized, and genotype calling was done using Genotyping Console 4.1 with Axiom GT1 algorithms according to the manufacturer&#39;s manual.
The cut-off values for data quality control were DISHQC ≥ 0.82 for hybridization and QC call rate ≥ 97%.
Title: Capture design

The 30 genes known to be mutated in lung cancer were chosen for targeted resequencing (Table 1).
The 701 coding regions in 30 genes covered 102 kb according to the consensus coding sequence (CCDS) database.
Capture sequences to achieve redundant coverage over the coding regions were chosen based on length (100-1,000 bp) and GC content (20-65%) and to avoid repetitive genomic elements in the ends.
After analysis of in silico-digested restriction fragments of the target regions, the best combinations of restriction enzymes were selected to provide over 99% coverage of targeted bases.
Title: Target enrichment analysis

A normal sample (NA17022) and two lung adenocarcinoma cell lines (H-1650 and H-23) were examined in this study.
It is reported in the COSMIC database that the H-1650 cell line has a deletion mutation in the EGFR gene, while the H-23 cell line has mutations in the KRAS, STK11, and TP53 genes.
After target gene enrichment, the enriched DNAs were analyzed using qPCR with primers targeting the regions of interest, along with qPCR primers targeting irrelevant, non-amplified loci outside the target regions.
The correlation between the replicates was high (average r2 = 0.97), and the majority of primer pairs clustered within a range of 3 Cts (threshold cycle in qPCR).
Therefore, the enrichment was highly reproducible, and the enrichment bias was minimal.
The enriched DNAs showed an average Ct of 17 with the target primers, while primer pairs targeting loci outside of the target regions had average Cts of 33.
This indicated that the enrichment was target-specific.
The average specificity in the normal samples was 28% when calculated from the Ct and amount of enriched product (Table 2).
Title: Sequencing analysis of enriched DNAs

To analyze the enriched samples, we sequenced the DNAs using a GAIIx next-generation sequencing instrument (Illumina) and evaluated important metrics to consider actual coverage, specificity, and reproducibility across the targeted loci.
On average, 4.3 gigabases (Gb) was produced per sample, and they were mapped to the reference genome (NCBI build 37, hg19) at a 73-90% mapping rate (Table 3).
The 26-41% of the uniquely mapped reads were found in the region of interest, demonstrating moderate specificity of this approach.
The normal sample showed the lowest mapping rate (73.03%) but the highest specificity (41.58%), indicating that cancer genomes are less efficient for exome sequencing due to genomic changes.
In addition, about 97% of the targeted bases were covered at more than 30× (Fig.1).
This high depth coverage could allow us to examine low-purity cancer samples, which are not normally analyzed by Sanger sequencing or genotyping tools.
The actual coverage of the normal sample differed, depending on the gene.
The coverage of most target genes was more than 95% at 30×, but two genes, CDKN2A and LTK, showed low coverage of 75% and 84%, respectively (Table 1).
Using the final mapped reads, we constructed a genomic profile database for detecting SNVs (Supplementary Table 1) and short insertions and deletions (INDELs) (Supplementary Table 2).
In each sample, we identified 81-179 SNVs and 50-90 small INDELs in target gene regions (Table 4).
Cancer samples (H-1650 and H-23) showed about twice as many SNVs than the normal sample (NA17022).
By subtracting SNVs found in the SNP databases, we identified cancer-specific somatic mutation candidates, and identical somatic mutations of the cancer cell lines in the COSMIC database were found.
All previously known somatic mutations in the TP53, KRAS, and STK11 genes of sample H-23 and in the EGFR gene of sample H-1650 were identified in this study (Table 4).
The validity of the data was also examined with a genome-wide SNP microarray, which has 37 SNPs in the target region (Axiom Array; Affymetrix).
The genotyping data showed 80-91% concordance without any bias (Table 5).
The disconcordant variations were not biased to any sample, coverage, or genotype.
Our targeted resequencing method for somatic mutation profiling in lung cancer from 30 cancer-related genes produced unbiased target DNAs repeatedly.
Analysis of the enriched DNAs by next-generation sequencing identified previously known mutations in the samples.
Further analysis of more samples by targeted resequencing will reveal many novel variant candidates.
Target enrichment was performed using Selector technology (Halo Genomics, Uppsala, Sweden), which showed improved coverage and compatibility with next-generation sequencing library construction by employing rolling-circle amplification [10].
This method exhibited about 97% mean coverage at 30× depth, average 42% specificity, and high reproducibility (r2 = 0.98) of target enrichment in the normal sample (Tables 1, 2, 3), indicating that this is applicable to targeted resequencing of clinical samples.
Although the enrichment specificity was moderate, this was overcome by increasing sequencing depth.
As the total DNA bases of the target regions was 107 kb, 97% coverage at 30× depth was achieved by 3.25 Gb of sequencing, which does not create any cost issues by using next-generation sequencing technologies.
Exome resequencing has proven to be robust and effective for somatic variant detection in coding regions [31, 32].
Comparisons of sequencing data from normal and cancer tissues from individual patients have unveiled individual somatic mutation profiles.
However, the cancer cell lines that we used had no normal cell pairs.
To solve this problem, we tried to remove previously known normal variations as much as possible.
Predicted SNVs from the sequencing data were further filtered using common variation information from the most updated dbSNP database.
As a result, we found many somatic mutations, which included previously reported mutations in the COSMIC database, such as variations in TP53, KRAS, and STK11 of sample H-23 and in EGFR of sample H-1650 (Table 4).
Especially, the number of reads that covered the four variations was more than 1,000.
This provides many advantages when compared to whole-genome sequencing or whole-exome sequencing.
Sequencing of cancer samples has raised several issues, such as sample purity and cancer heterogeneity.
These shortcomings can only be overcome by in-depth sequencing of target regions.
Therefore, our protocol is also useful for cost-effective somatic mutation screening of admixed clinical cancer samples.
Compared to whole-genome sequencing, targeted sequencing has an issue with uneven coverage of targeted genes.
In Table 3, cancer samples showed fewer sequencing reads at target regions.
This low target capture efficiency in cancer samples could be explained by genetic variations in cancer genomes that inhibit the hybridization between cancer DNA fragments and the designed capture oligonucleotides.
This may be overcome by trial-and-error screening in selecting more efficient oligonucleotides.
There are two kinds of target capture technologies: hybridization and PCR.
Hybridization-based target capture has been widely used and is able to cover more target regions but is time-consuming and hard to handle with many samples.
In contrast, the PCR-based method is faster and allows us to handle more samples.
Our protocol is a multiple displacement amplification-based method that is as efficient as the PCR-based method.
For example, hybridization-based target capture methods, such as the Agilent SureSelect target enrichment kit and NimbleGen SeqCap EZ kit, normally handle 1-8 samples at the same time, while our protocol with a liquid handler could process 96 samples in parallel.
This costs 12 times less money.
Therefore, our target enrichment is scalable and easy to handle with multiple samples.
This can result in remarkable reduction of total cost when combined with multiplexed next-generation sequencing.
Therefore, our target resequencing protocol provides a scalable sample-handling tool for a genetic variation study of lung adenocarcinoma.
Target coverage in cancer and normal samples.
The cumulative coverage of targeted bases (i.e., the fraction of all sequenced bases in the target regions that share more than a particular read depth) were plotted after sequencing 4.34 Gb of H-1650 (black), 6.03 Gb of H-23 (blue), and 3.25 Gb of NA17022 (red).
The sequencing yield in the three samples resulted in 30× coverage of 92% (H-1650), 95% (H-23), and 97% (NA17022) of all target regions.
Title: 30× coverage of individual target genes after deep sequencing

Title: Target capture specificity analyzed by qPCR

Title: Mapping statistics of next-generation sequencing experiments

Title: Somatic variation candidates from the target gene regions

Title: Comparison of SNV calls with DNA microarray genotyping results
Title: Association Analysis of TEC Polymorphisms with Aspirin-Exacerbated Respiratory Disease in a Korean Population

The tyrosine-protein kinase Tec (TEC) is a member of non-receptor tyrosine kinases and has critical roles in cell signaling transmission, calcium mobilization, gene expression, and transformation.
TEC is also involved in various immune responses, such as mast cell activation.
Therefore, we hypothesized that TEC polymorphisms might be involved in aspirin-exacerbated respiratory disease (AERD) pathogenesis.
We genotyped 38 TEC single nucleotide polymorphisms in a total of 592 subjects, which comprised 163 AERD cases and 429 aspirin-tolerant asthma controls.
Logistic regression analysis was performed to examine the associations between TEC polymorphisms and the risk of AERD in a Korean population.
The results revealed that TEC polymorphisms and major haplotypes were not associated with the risk of AERD.
In another regression analysis for the fall rate of forced expiratory volume in 1 second (FEV1) by aspirin provocation, two variations (rs7664091 and rs12500534) and one haplotype (TEC_BL2_ht4) showed nominal associations with FEV1 decline (p = 0.03-0.04).
However, the association signals were not retained after performing corrections for multiple testing.
Despite TEC playing an important role in immune responses, the results from the present study suggest that TEC polymorphisms do not affect AERD susceptibility.
Findings from the present study might contribute to the genetic etiology of AERD pathogenesis.
Asthma is a disease that is caused by inflammation in the lung and bronchus and is affected by genetic and environmental influences.
Aspirin-exacerbated respiratory disease (AERD), which was first reported in 1922, is a type of asthma.
AERD is characterized by the following three symptoms: bronchial asthma, aspirin sensitivity, and nasal polyposis [1, 2, 3].
It has been reported that 10-20% of asthma patients have aspirin sensitivity, whereas 1-2% of the non-asthma population shows aspirin sensitivity [4, 5].
Although the mechanisms of AERD pathogenesis are still not fully understood, inflammatory responses by overproduction of leukotrienes are regarded as the main pathogenesis of AERD.
Aspirin is a commonly used medication, which belongs to the non-steroidal anti-inflammatory drugs.
Despite being a widely used medication, aspirin intake also causes various side effects, including manifested gastrointestinal ulcer, stomach bleeding, and tinnitus, especially in higher doses.
Although the side effects of aspirin are not common, these effects have been reported in about 10% of adult asthmatics.
In AERD pathogenesis, aspirin inhibits the activation of cyclooxygenase-1 enzyme, leading to block of production of prostaglandin and thromboxane.
This mechanism causes overproduction of leukotrienes, such as leukotriene B4, leukotriene C4, leukotriene D4, and leukotriene E4 [6, 7].
The tyrosine-protein kinase Tec (TEC) is a member of the non-receptor tyrosine kinases and has critical roles in cell signaling transmission, calcium mobilization, gene expression, and transformation [8].
It has been known that TEC family kinases are associated with various intracellular signaling mechanisms, such as cytokine receptors and lymphocyte surface antigens [9, 10].
In particular, it was demonstrated that the TEC family proteins are involved in regulation of leukotriene secretion via the mast cell signaling pathway [11].
Therefore, we hypothesized that TEC polymorphisms might be involved in AERD pathogenesis.
In the present study, 38 TEC single nucleotide polymorphisms (SNPs) were genotyped in a total of 592 subjects, which comprised 163 AERD cases and 429 aspirin-tolerant asthma (ATA) controls, to examine the associations between TEC polymorphisms and AERD susceptibility.
Title: Study subjects

Subjects in this study were recruited from the Asthma Genome Research Center, comprising the hospitals of Soonchunhyang, Chunnam, Chungbuk, Seoul National, and Chung-Ang Universities in Korea.
All subjects provided written informed consents, and the study protocols were approved by the institutional review board of each hospital.
Diagnosis of AERD was performed according to a modified method as previously described [12].
We also performed aspirin challenge in subjects with a history of aspirin hypersensitivity, presence of urticaria, nasal polyp, and sinusitis.
The AERD case group included patients with 20% or greater decreases in forced expiratory volume in 1 second (FEV1) or 15% to 19% decreases in FEV1 with naso-ocular or cutaneous reactions, whereas subjects showing a rate of FEV1 decline less than 15% without extrabronchial nasal or skin symptoms were included in the ATA group.
The clinical diagnostic factors for the present study are summarized in Table 1.
Title: SNP selection and genotyping

To investigate the associations between TEC polymorphisms and the risk of AERD, we selected candidate SNPs based on allele frequencies in the Asian population, linkage disequilibrium (LD) status, and National Center for Biotechnology information.
The data for selection were obtained from the International HapMap database (http://hapmap.ncbi.nlm.nih.gov/).
Genotyping of 38 TEC polymorphisms was performed in a total of 592 subjects, including 163 AERD cases and 429 ATA controls.
Genotyping was carried out with 20 ng of genomic DNA by TaqMan assay using ABI prism 7900HT sequence detection system software version 2.3 (Applied Biosystems, Foster City, CA, USA) in all subjects.
Assay IDs of all SNPs used in TaqMan assay are listed in Supplementary Table 1.
Title: Statistics

We applied a widely used measure of linkage disequilibrium to all pairs of biallelic loci: Lewontin&#39;s D&#39; (|D&#39;|) [13] and r2.
Haplotypes of each individual were inferred using the PHASE algorithm (ver.
2.0), developed by Stephens et al.[14].
Linear regression analysis was performed to examine the differences in the rates of decline in FEV1 following aspirin challenge among the genotypes and major haplotypes.
The data were managed and analyzed using Statistical Analysis System, version 9.2 (SAS Inc., Cary, NC, USA).
Associations for AERD under the logistic model were adjusted by smoking status, atopy, body mass index (BMI), age, and sex (male = 0, female = 1).
Significant associations are shown in boldface (p &lt; 0.05).
We recruited a total of 592 subjects, which consisted of 163 AERD cases and 429 ATA controls, for the present study.
According to the results, four clinical characteristics showed significant differences between the case and control groups (Table 1).
The fall rate by aspirin provocation was significantly higher in AERD subjects than ATA controls (24.63 ± 16.11 vs. 3.54 ± 4.85; p = 0.0001).
Percentage of predicted FEV1 in the AERD subjects showed decreased lung function than in ATA subjects (87.58 ± 16.94 vs. 91.66 ± 16.87; p = 0.009).
Also, age of first medical examination and BMI were lower in AERD cases than ATA controls.
The other diagnostic factors showed no significant differences between the case and control group.
In the present study, 38 TEC polymorphisms and 12 major haplotypes were used for the association analyses.
Locations of the polymorphisms are shown in a genetic map of TEC with their LD status (Fig.1A).
Minor allele frequencies (MAFs) of the SNPs in Korean subjects are displayed in Supplementary Table 2 with their allele change, position, and p-value for Hardy-Weinberg equilibrium.
All SNPs were in Hardy-Weinberg equilibrium.
The LD blocks were obtained by using HaploView software, and the haplotypes of each LD block were calculated by PHASE software (Fig.1B and 1C).
We used major haplotypes that had frequencies higher than 0.05 for further analyses.
The minor haplotypes that had frequencies lower than 0.05 were merged and presented as &#39;Others.&#39;
To compare genetic differences among ethnicities, we obtained MAFs of Caucasians, Han Chinese, Japanese, and Africans from the NCBI database (dbSNP) (Supplementary Table 2) and calculated LD blocks using genotype data from the International HapMap database (Supplementary Fig.1).
As a result, Asian populations showed similar MAFs, whereas the other populations showed distinct differences.
However, we did not find any correlation in LD status in any ethnicity.
Logistic analyses were performed to investigate the associations between TEC polymorphisms and the risk of AERD.
The result from analyses revealed that all SNPs and major haplotypes were not associated with the risk of AERD (Table 2).
On the other hand, two SNPs (rs7664091, p = 0.04 and rs12500534, p = 0.03) and one major haplotype in LD block 2 (TEC_BL2_ht4, p = 0.03) showed marginal associations in the regression analysis with the decline rate of FEV1 (Supplementary Table 3).
However, the association signals of these polymorphisms disappeared after performing corrections for multiple testing (data not shown).
Taken together, these results indicate that the TEC polymorphisms are not associated with AERD susceptibility, at least in the Korean population.
The lack of associations in this study suggests that although TEC plays an important role in immune responses, TEC variants do not directly affect decreased pulmonary function by aspirin uptake.
However, considering the fact that the difference in frequency of polymorphisms showed different effects in various populations, replication studies in AERD subjects in other ethnicities are recommended.
The present study still had several limitations.
First, an average statistical power of 66.41% indicated an insufficient sample size.
However, the rare condition of AERD made it difficult to recruit subjects in the Korean asthma cohort.
Second, SNPs in coding sequence were not selected, due to very low frequencies, although non-synonymous SNPs (nsSNP) in exonic regions could affect risk of the disease.
Further studies are required to examine the molecular role of nsSNPs in TEC.
In conclusion, we hypothesized that TEC might impact on AERD susceptibility.
However, our results showed that 38 TEC polymorphisms and 12 major haplotypes were not associated with the risk of AERD.
Although further studies are required to investigate the exact role of the TEC SNPs in immune responses, the preliminary results of the present study may provide useful information for AERD pathogenesis.
Schematic physical map, haplotypes and linkage disequilibrium (LD) plot of TEC.
(A) Polymorphisms identified in TEC.
Coding exons are marked by shaded blocks and untranslated region by white blocks.
The LD coefficients (r2) are based on the genotypes of Korean samples.
(B) Haplotypes are calculated using genotypes of TEC polymorphisms in a Korean population.
Only those with frequencies over 0.05 are shown in tables.
Haplotypes with frequencies lower than 0.05 are merged into &#34;Others.&#34;
(C) LD coefficients (|D&#39;|) among the selected SNPs based on the genotypes of whole study subjects in this study.
Title: Clinical profile of subjects in study with TEC polymorphisms

Title: Association analysis of TEC polymorphisms and haplotypes with the risk of AERD in a Korean population
Title: Elucidating Molecular Interactions of Natural Inhibitors with HPV-16 E6 Oncoprotein through Docking Analysis

Human papillomavirus (HPV) infection is the leading cause of cancer mortality among women worldwide.
The life-threatening infection caused by HPV demands the need for designing anticancerous drugs.
In the recent years, different compounds from natural origins, such as carrageenan, curcumin, epigallocatechin gallate, indole-3-carbinol, jaceosidin, and withaferin, have been used as a hopeful source of anticancer therapy.
These compounds have been shown to suppress HPV infection by different researchers.
In the present study, we explored these natural inhibitors against E6 oncoprotein of high-risk HPV-16, which is known to inactivate the p53 tumor suppressor protein.
A robust homology model of HPV-16 E6 was built to anticipate the interaction mechanism of E6 oncoprotein with natural inhibitory molecules using a structure-based drug designing approach.
Docking analysis showed the interaction of these natural compounds with the p53-binding site of E6 protein residues 113-122 (CQKPLCPEEK) and helped the restoration of p53 functioning.
Docking analysis, besides helping in silico validation of natural compounds, also helps understand molecular mechanisms of protein-ligand interactions.
Human papillomavirus (HPV), a virus from the papillomavirus family, accounts for 5.2% of all cancers worldwide and is also the causative agent of a subset of anogenital, head and neck cancers, and cervical cancer [1, 2].
Amongst these, cervical cancer contributes the most to cancer mortality in women worldwide, with an estimated 49,300 diagnoses and 274,000 deaths annually [3].
Although there are more than 200 types of HPV identified, the most commonly associated with cervical cancer are &#39;high-risk&#39; HPV-16 and HPV-18, responsible for about 62.6% and 15.7% of cervical cancers, respectively [4].
Besides, among all the HPV-associated cancers, the proportion attributable to HPV types 16 and 18 is 63-80% of penile cancers, 80-86% of vulva/vaginal cancers, 93% of anal cancers, and 89-95% of oropharyngeal cancers [5].
HPV-16 and HPV-18 remain the primary targets for anticancer drug development.
The E6 and E7 onco-proteins have been shown to interact specifically with the p53 and pRb tumor suppressor proteins, respectively [6].
Both pRb and p53 negatively regulate the cell cycle and also appear to inhibit G0-G1 and G1-S phase transitions.
These interactions apparently play important roles in the induction of cell immortality.
The importance of p53-mediated apoptosis has been recognized in terms of the maintenance of homeostasis, as well as the prevention of neoplastic transformation.
E6 forms a ternary complex with p53 and E6AP (E6 associated protein), resulting in the degradation of p53 via the ubiquitination pathway [7].
HPV has been known as a causative agent for cervical cancer for 3 decades, but effective treatment against HPV infection is still unavailable [8].
Natural products have been identified as promising sources of drugs for treatment and prevention of cancer in recent years [9].
Carrageenan is known to prevent the binding of HPV virions to cells.
It can also block HPV infection through a second, post attachment heparan sulfate-independent effect [10].
Bava et al.
[11] demonstrated that curcumin sensitizes cervical cancer cells to the therapeutic effect of taxol, acting in the down-regulation of both nuclear factor κB (NF-κB) and the threonine/serine kinase AKT pathway.
Curcumin is cytotoxic to cervical cancer cells in a concentration-dependent and time-dependent manner, and the cytotoxicity is higher in HPV-infected cells.
The authors confirmed the inhibitory action of curcumin on NF-κB activation, through the degradation of IκB, and on AP-1 binding, which results in the downregulation of cyclooxygenase 2 (COX-2) and increased tumor cell apoptosis [11].
Moreover, it was shown that curcumin downregulates the expression of E6 and E7 proteins of HPV-16, resulting in loss of the transforming phenotype and the cessation of cellular growth.
Singh and Singh [12] explored other molecular mechanisms exerted by curcumin in cervical cancer cells, showing that it inhibits telomerase activity, the RAS and ERK signaling pathway, cyclin D1, COX-2, and inducible nitric oxide synthase activity.
Withaferin A (WA) is the major active component of the Indian medicinal plant Withania somnifera, with antitumor, antiangiogenic, and radiosensitizing activity [13, 14].
An in vitro and in vivo study conducted by Munagala et al.
[15] showed the inhibitory effect of WA against the proliferation of cervical cancer cells.
WA was shown to downregulate the expression of HPV E6 oncoprotein and restore the p53 pathway, resulting in apoptosis of cervical cancer cells [15].
(-)-Epigallocatechin-3-gallate (EGCG), the most abundant and active tea catechin, has antiviral and antitumor properties.
According to a clinical study, EGCG was effective in patients with HPV-infected cervical lesions when delivered in the form of ointment or capsule [16].
A study conducted by Qiao et al.
[17] showed the inhibition effect of EGCG on the growth of CaSki (HPV-16 positive) and HeLa (HPV-18 positive) cells in a time- and concentration-dependent manner.
Their results also observed that EGCG could inhibit the expression of HPV E6/E7.
Lee et al.
[18] isolated jaceosidin from the methanol (MeOH) extract of Artemisia argyi and reported its inhibitory effects on binding between oncoproteins and tumor suppressor p53.
Indole-3-carbinol (I3C), found in cruciferous vegetables, such as cauliflower, broccoli, cabbage, and brussel sprouts,was demonstrated to have anti-estrogenic activities in cervical cells [19].
The purpose of this in silico study was to observe the interaction between active compounds obtained from natural products used for the treatment against HPV.
In this study, a three-dimensional structure of E6 protein of HPV type 16 was modeled using the Phyre 2 server, and structural refinement and energy minimization were performed by the YASARA Energy Minimization Server.
In order to reveal the interaction between E6 oncoprotein with natural ligands, docking analysis was performed using the AutoDock tool.
Title: Hardware and software

The study was carried out on a Dell Workstation with a 2.26 GHz processor, 6 GB RAM, and 500 GB hard drive running in the Windows operating system.
Bioinformatics software, such as AutoDock4.0 and online resources, were used in this study.
Title: E6 of HPV-16

E6 protein of HPV-16 was selected as the drug target.
The HPV-16 E6 (GenBank ID: AAD33252.1) protein sequence was retrieved from NCBI (http://www.ncbi.nlm.nih.gov/).
Title: Protein structure prediction and validation of drug target

For modeling of the three-dimensional structure of the E6 protein, Phyre 2 server [20] was used.
Structural refinement and energy minimization of the predicted model were carried out using the YASARA Energy Minimization Server [21].
The refined model reliability was assessed through Procheck [22], ProSA-web [23], ProQ [24], and Verify-3D [25].
The refined model was further verified by the ERRAT server [26].
Title: Ligand preparation

Chemical structures of natural compounds (carrageenan, curcumin, EGCG, I3C, jaceosidin, WA), along with CAS (Chemical Abstracts Service) registry numbers, reported in the literature were retrieved from the PubChem database (Fig.1) [27].
Title: Protein-ligand docking

Protein-ligand docking studies were performed using the AutoDock 4.2 program [28].
It is one of the most widely used methods for protein-ligand docking.
All the pre-processing steps for ligand and protein files were performed using AutoDock Tools 1.5.4 (ADT), which has been released as an extension suite to the Python Molecular Viewer [28].
ADT was used to prepare the receptor molecule (HPV-16 E6) by adding all hydrogen atoms into carbon atoms of the receptor, and Kollman charges were also assigned.
For docked ligands, non-polar hydrogens were also added.
Gasteiger charges were assigned and torsions degrees of freedom were allocated by ADT.
The Lamarckian genetic algorithm (LGA) was applied to model the interaction pattern between the E6 oncogenic protein and selected inhibitors.
The grid maps representing the receptor proteins in the docking process were calculated using AutoGrid (part of the AutoDock package).
A grid of 50, 50, and 50 points in the x, y, and z directions was centered on the p53-binding sites of E6 proteins.
For all docking procedures, 10 independent genetic algorithm runs with a population size of 150 were considered for each molecule under study.
A maximum number of 25 × 105 energy evaluations, 27,000 maximum generations, a gene mutation rate of 0.02, and a crossover rate of 0.8 were used for the LGA.
Autodock was run in order to prepare corresponding docking log file (DLG) files for further analysis.
Title: Visualization

The visualization of structure files was done using the graphical interface of the ADT tool and the PyMol molecular graphics system (http://www.pymol.org).
Title: Tertiary structure modeling and validation

E6 protein of HPV type 16 has 158 amino acids in its protein sequence.
Since the three-dimensional structure of E6 was not available, its structure model was predicted using the Phyre 2 server by taking chain C of the crystal structure of full-length HPV oncoprotein e62 in complex with the lxxll peptide of ubiquitin ligase e6ap (PDB ID: 4GIZ) as a structural template.
The query coverage of the target-template alignment was 90% with 96% identity.
The predicted structure was then subjected to the Yet Another Scientific Artificial Reality Application (YASARA) Energy Minimization Server for structural refinement.
The total energy for the refined structure obtained from the YASARA Energy Minimization Server was -48,054.9 kcal/mol (score, -2.55), whereas prior to energy minimization, it was 81,463,636,897 kcal/mol (score, -4.21).
The stereochemistry of the refined model (Procheck analysis) (Fig.2A) revealed that 99.2% of residues were situated in the most favorable region and additional allowed regions, and 0.8% (Gln106) of residues were in the generously allowed region, whereas none of the residues fell in the disallowed region of the Ramachandran plot (Fig.2B).
ProSA-web evaluation revealed a compatible Z score (Fig.2C) value of -4.34, which is well within the range of native conformations of crystal structures [23].
The overall residue energies of the E6 3D model were largely negative except for one peak (Fig.2D).
The 3D model of HPV-16 E6 protein showed an LG score of 2.531 by Protein Quality Predictor (ProQ) tool, implying a high accuracy level of the predicted structure.
A ProQ LG score &gt;2.5 is necessary for suggesting that a model is of very good quality [24].
Similar assumptions were achieved using the ERRAT plot (Fig.2E), where the overall quality factor was 91%.
All of these outcomes recommended the reliability of the proposed model.
Title: Docking analysis of HPV-16 E6 with natural ligands

As all natural ligands (inhibitors) were found to be docked in various conformations and with varying binding energies, the lowest energy conformation was selected.
Upon docking, the high-ranked binding energies of modeled structures of E6 oncoprotein with natural ligands were obtained (Table 1).
HPV E6 degrades the p53 tumor suppressor protein.
To accomplish the process of degradation, E6 proteins bind and modify the target specificity of the ubiquitin ligase E6AP.
Based on the resolved structure of HPV-16 E6 by Zanier et al.
[29], Cherry et al.
[30] revealed in their docking analysis study that few flavonoid compounds bind in the hydrophobic pocket of the E6 and E6AP interface and mimic the leucines in the conserved α-helical motif of E6AP.
Another study also showed that the biological activities of E6 are mediated by the interaction of E6 with the LXXLL motif of E6AP [31].
In our study, the E6 protein of HPV-16 showed one LXXLL motif (103LCDLL107).
Out of six natural compounds included in our study, three (carrageenan, EGCG, and I3C) were found to interact with the LXXLL motif of HPV-16 E6 by forming a hydrogen bond with Leu106.
According to Crook et al.
[32], the p53-binding site is around amino acid residues 113-122 (CQKPLCPEEK) of HPV-16 E6 protein.
The active site of the model was then analyzed based on the docking interaction of E6 with the p53-binding site with all 6 natural ligands.
Fig.3 shows a docked pose of six ligand molecules with E6 protein, which showed almost similar docking results.
The detailed interactions are given in Table 1.
Each ligand interacts with the receptor at the p53-binding site.
Among the 6 different ligands, carrageenan showed the lowest binding energy (-13.95 kcal/mol) and inhibition constant (0.059 µM) for the protein-ligand complex.
Carrageenan was found to interact with the E6 residues Ala53, Phe54, Leu106, Ile108, Leu117, and Lys122 by forming hydrogen bonds.
As mentioned in the introduction, carrageenan blocks HPV infection [10]; from our docking study, it might be revealed that the block of HPV infection might be due the high binding affinity of carrageenan towards the p53-binding site of E6 protein.
WA showed the second lowest binding energy of -7.58 kcal/mol and an inhibition constant of 2.77 µM.
During docking with the receptor hydrogen bond, interactions were found with three residues (Ala53, Leu117, and Lys122) of E6.The other three compounds-EGCG, I3C and jaceosidin-were also found to interact with E6 protein with binding energies of -4.13, -4.06, and -4.01 kcal/mol, respectively.
Leu106, Ile108, and Leu117 were the interacting residues of the receptor proteins with EGCG, whereas I3C interacted with Leu106, Pro116, and Lys122; Jaceosidin interacted with Phe54, Pro116, Leu117, and Lys122.
Curcumin was found to interact with E6 protein with less affinity (binding energy of -3.09 kcal/mol).
All natural ligands were reported to inhibit HPV infection; our docking study also revealed the in silico validation of the inhibition.
HPV-16 is known for causing high-risk cervical cancer.
All ligands were found to interact with E6 oncoprotein of HPV-16 with significant binding energy and with amino acid residues known for p53 binding.
This interaction might prevent E6 protein from interacting with host p53 protein, which may correlate with why these natural compounds are used to treat HPV infections.
Title: Conclusion

Different natural products have been identified and used as promising drug sources against cancer caused by HPV.
Advancement in bioinformatics and computational biology are proficient invalidating those natural drugs through in silico approaches.
E6 proteins of high-risk HPV 16 inactivate p53 by inducing its degradation, a process that requires a ternary complex between E6, E6AP, and p53.
Low-risk HPVs, such as HPV-6 and HPV-11, do not have the ability to degrade p53.
Thus, E6 of HPV-16 is of considerable interest for designing novel inhibitors to overcome the challenges of cervical cancer.
A high-quality 3D model of E6 obtained through a computational approach and docking analysis employing AutoDock 4.2 in this study provided high-throughput validation.
This in silico approach may be of interest in designing new drugs from natural sources against cervical cancer.
Natural compounds reported to use against human papillomavirus infection.
(A) 3D structure of predicted human papillomavirus 16 (HPV-16) E6 model.
(B) Ramachandran plot of predicted E6 model (the red, dark yellow, and light yellow regions represent the most favored, allowed, and generously allowed regions).
(C) ProSA-web Z-scores of all protein chains in PDB determined by X-ray crystallography (light blue) and nuclear magnetic resonance spectroscopy with respect to their length.
The Z-score of E6 was present in the range represented in black dot (D) Energy plot for the predicted E6 of HPV-16.
(E) ERRAT plot for residue-wise analysis of homology model.
Interaction profile of E6 with natural ligands carrageenan (A), withaferin A (B), epigallocatechin gallate (C), indole-3-carbinol (D), jaceosidin (E), and curcumin (F) showing interaction of ligands with active site residues of E6 by forming hydrogen bonds, shown in dotted lines.
Title: Polar contact information from docking calculations between ligands and protein
Title: Examining the Gm18 and m1G Modification Positions in tRNA Sequences

The tRNA structure contains conserved modifications that are responsible for its stability and are involved in the initiation and accuracy of the translation process.
tRNA modification enzymes are prevalent in bacteria, archaea, and eukaryotes.
tRNA Gm18 methyltransferase (TrmH) and tRNA m1G37 methyltransferase (TrmD) are prevalent and essential enzymes in bacterial populations.
TrmH involves itself in methylation process at the 2&#39;-OH group of ribose at the 18th position of guanosine (G) in tRNAs.
TrmD methylates the G residue next to the anticodon in selected tRNA subsets.
Initially, m1G37 modification was reported to take place on three conserved tRNA subsets (tRNAArg, tRNALeu, tRNAPro); later on, few archaea and eukaryotes organisms revealed that other tRNAs also have the m1G37 modification.
The present study reveals Gm18, m1G37 modification, and positions of m1G that take place next to the anticodon in tRNA sequences.
We selected extremophile organisms and attempted to retrieve the m1G and Gm18 modification bases in tRNA sequences.
Results showed that the Gm18 modification G residue occurs in all tRNA subsets except three tRNAs (tRNAMet, tRNAPro, tRNAVal).
Whereas the m1G37 modification base G is formed only on tRNAArg, tRNALeu, tRNAPro, and tRNAHis, the rest of the tRNAs contain adenine (A) next to the anticodon.
Thus, we hypothesize that Gm18 modification and m1G modification occur irrespective of a G residue in tRNAs.
Marinobacter species is one of the most ubiquitous classes formed in the marine ecosystem.
It occurs throughout the water column in deep oceans, and it exerts a significant impact on various biogeochemical cycles.
Marinobacter aquaeolei is involved in redox reactions that utilize oxygen and nitrate as terminal electron acceptors [1].
It exhibits hydrocarbon-degrading mechanisms and is capable of diverse extremophilic attributes (psychrophily, oligotrophy, and halotolerance) [2].
Marinomonas MWYL1 is able to grow on the betaine molecule and use dimethylsulfoniopropionate (DMSP) as their sole carbon source.
Dimethyl sulphide is released as a by-product of metabolism of DMSP and reaches the atmosphere, which subsequently forms cloud condensation nuclei during oxidation [3].
The key gene (dddD) in M. MWYL1 encodes for acetyl-CoA transferase, which adds CoA to DMSP, prior to its subsequent cleavage [4].
Thus, M. MWYL1 is involved in the cycling of the climate-changing gas DMS, and their genomic potential still needs to be unraveled.
Nitrobacter hamburgensis, a gram-negative bacterium, forms a biofilm, which is considered a suitable alternative for the treatment of wastewater that otherwise requires large and expensive reactors for efficient bioremediation [5].
Nitrobacter winogradskyi inhabits many soil types, natural stones, as well as both fresh and salt water.
It derives its energy at once through nitrite oxidation and carbon dioxide fixation, thus acting as a chemolithoautotroph.
The gammaproteobacterium class of Nitrosococcus oceani is an obligate chemolithoautotroph capable of extracting energy and reducing power from the oxidation of ammonia to nitrite.
N. oceani&#39;s impact on the ecosystem is the massive release of nitrogen oxide and nitrogen gases into the atmosphere, thus completing the nitrification cycle.
It also contributes to the maintenance of concentrations of nitrate in the nitrogen pool of the deep ocean at 40 µM [6].
Title: Insight into tRNA modifications

tRNA is a small chain of nucleotides that decipher the genetic code to protein synthesis on codon and anticodon interactions and has a potential role in DNA repair mechanism tRNA methyltransferase brings changes in the nucleosides in the tRNA structure.
Recently, some of these nucleoside modifications have been reported to be involved in the cell response to environmental stress and in the repair of DNA damage as well.
In many cases, the posttran-scriptional modification of tRNA has facilitated translation of deviant sense codons [7], and these modifications are essential for tRNA folding function.
The amino acyl tRNA synthetase recognizes the cognate tRNA through their structure and chemistry, contributed by modified nucleosides at the anticodon loop [8].
Thus, the modification of stem and loop structures of tRNAs regulates gene expression by ordering conformation and dynamics for recognition of the codon and maintenance of the reading frame.
Similarly, mRNAs are programmed to shift their frame when there is no modification in tRNA [8].
In an actively dividing cell, tRNA gene content determines the relative abundance of tRNA isoacceptor, which, in turn, determines codon translation efficiency [9].
The relative abundance of a particular tRNA isoacceptor is kingdom-specific, and enzymes present in the organism increase the translational efficiency of codons through modification in the anticodon wobble base.
It implies that the modified enzymes play a major role in the evolution of genomic composition [9].
Methylation reactions account for the majority of posttranscriptional modifications in tRNA [10].
tRNA methyltransferase (Trm) catalyzes the modification along the length of tRNA, including at the anticodon site.
Indeed, Trm has a potential and crucial role in enhancing the synthesis of proteins that participate in the damage response [11].
Title: tRNA Gm18 methyltransferase (TrmH)

To date, 150 modified nucleosides have been identified in RNA, and in tRNA alone 92 such modifications have been recorded [12].
Prominent among those are 2&#39;-OH ribose methylated nucleoside, which is commonly found in tRNAs.
The posttranscriptional modification in RNA is mediated through specific methyltransferase enzymes [13].
These modifications can occur in the noncoding region of the RNA sequences.
In fact, tRNA contains abundant modified nucleosides, which stabilize the L-shaped tRNA structure and improves their molecular recognition [14].
A conserved guanosine is present at position 18 in the D loop of t-RNA modified to 2&#39;-OH methyl guanosine.
This modification stabilizes the L-shaped structure of t-RNA by interacting with pseudouridine 55 [15].
The enzyme TrmH catalyzes the reaction through transfer of a methyl group from S-adenosyl-methionine (AdoMet) to the 2&#39;-OH group of the G18 ribose in tRNA D-loops [16].
The TrmH gene in Escherichia coli [17], Thermus thermophilus [18], and Aquifex aeolicus [19] has been identified to have similar catalytic motifs in them.
TrmH is subdivided into two classes based on their substrate tRNA specificity.
The type I enzymes are involved in the modification of all tRNA species, whereas type II modifies only a subset of tRNA species [20].
The Gm18 modification contributes towards defense reactions during pathogen encounters and has been observed in high frequency both in eukaryotes and prokaryotes, including viral RNA [21].
Therefore, a Gm18 modification study gains a significant factor of an infectious microbe, as it acts as a toll-like receptor 7 antagonist [21], and it can be utilized as an anti-inflammatory drug as well [20].
Title: tRNA m1G37 methyltransferase (TrmD)

Lots of modified nucleosides are present at the region of anticodons, specifically at positions 34 (the wobble position) and 37 (3&#39; and adjacent to the anticodon).
One of them is the 1-methyl guanosine (m1G) modification occurring at position 37 in tRNA [22].
This modification is catalyzed by TrmD, which catalyzes the addition of a methyl group from AdoMet to G at position 37, adjacent to the anticodon [23].
The specificity of this enzyme is determined by V, T, and D side loops and the presence of a G36pG37 sequence [24].
TrmD is a vital enzyme for maintaining the correct reading frame during translation [25].
Lack of this modification at the 37th position (i.e., next to the anticodon) leads to a +1 frameshift and hinders the translation efficiency [26].
The nucleus-encoded tRNA is methylated at G before being transported to the cytoplasm, whereas in mitochondria, the tRNA is methylated at G37 by a nucleus-encoded enzyme and transported into organelles [27].
Both TrmH and TrmD enzymes are under the class of the SpoU family and share similar catalytic reactions [28].
Hence, the study confirms that tRNA subsets contain G bases at the 18th position (D-loop) and next to the anticodon (anticodon loop).
FASTA format of tRNA gene sequences of M. aquaeolei, M. MWYL1, N. hamburgensis, N. winogradskyi, and N. oceani were retrieved from a tRNA database (http://gtrnadb.ucsc.edu/) [29] and used for multiple sequence alignment by BioEdit software (http://www.mbio.ncsu.edu/bioedit/bioedit.html) [30].
The presence of Gm18 modification bases were analyzed in total tRNAs of M. aquaeolei, M. MWYL1, N. hamburgensis, N. winogradskyi, and N. oceani.
These organisms&#39; tRNA contains a G residue at the 18th position adjacent to the anticodon loop, except tRNAMet, tRNAPro, and tRNAVal (Table 1).
Furthermore, the 18th position of tRNAs in M. aquaeolei (51 tRNAs) and M. MWYL1 (83 tRNAs) contain thymine (T), cytosine (C), or adenine (A) instead of G residue.
M. aquaeolei&#39;s tRNAMetCAT (two copies), tRNAProCGG, tRNAProCGG, and tRNAProTGG have T instead of G residues.
M. MWYL1 contains 9 copies of tRNAMetCAT; of these, six tRNAs have a T residue at the 18th position and three contain a G residue at the 18th position.
Only three proline-specific tRNAs are formed in M. MWYL1, and all of them show a T base at the 18th position instead of G. The organism N. hamburgensis comprises 50 tRNA subsets; among these, eight tRNAs showed no G residue at the 18th position, and the rest of the 42 had a G residue.
Thus, the tRNAs without G residues are tRNAMetCAT, tRNAProCGG (three copies), tRNAProGGG, tRNAProTGG, and tRNAValGAC.
Therefore, eight tRNAs at the 18th position show T, C, or A.
Besides tRNAMetCAT, tRNAProCGG and tRNAProGGG show a C residue, and tRNAProCGG and tRNAProTGG hold a T residue at the 18th position.
Nevertheless, tRNAValGAC confirmed an A residue at the 18th position.
Among six proline-specific tRNAs (tRNAProCGG), only one showed a G residue at the 18th position.
Furthermore, N. winogradskyi and N. oceani revealed 47 and 45 tRNAs, respectively.
These two organisms hold similar properties-i.e., tRNA subsets, like tRNAMetCAT, tRNAProCGG, tRNAProGGG, and tRNAProTGG, containing a T residue at the 18th position as a substitute for the G residue.
Title: Locating guanosine next to the anticodon in tRNAs

M. aquaeolei, M. MWYL, N. hamburgensis, N. winogradskyi, and N. oceani have common conserved tRNA subsets (tRNAArg, tRNALeu, and tRNAPro) that contain a G residue at the 38th position (G37pG38) next to the anticodon.
Variations of the G residue in archaea Methanococcus jannaschii tRNAs, such as tRNACys (G35pG36); tRNAHis, and tRNAArg (G36pG37); tRNAGln and tRNATry (G37pG38); tRNAArg, tRNAPro, and tRNASer (G37pG39); tRNALeu, tRNAGlu, tRNATrp, tRNASer, and tRNAPro (G39pG40) [31] are already revealed.
M. aquaeolei, M. MWYL1, and N. hamburgensis share similar tRNA subsets (tRNAArg, tRNALeu, and tRNAPro) that contain a G residue at the 38th position (G37pG38) next to the anticodon (Table 1).
N. winogradskyi loses the G residue in tRNAArg, but it has presented it at the 38th position in tRNAHisGTG, tRNALeuCAG, tRNALeuGAG, tRNALeuTAG, tRNAProCGG, tRNAProGGG, and tRNAProTGG.
N. oceani contains conserved tRNAs (tRNAArgCCG, tRNALeuCAG, tRNALeuGAG, tRNALeuTAG, tRNAProCGG, tRNAProGGG, tRNAProTGG) in which the G residue (G37pG38) is presented next to the anticodon (Table 1).
An interesting finding is that tRNAMet and tRNAPro are the only two tRNA copies that do not have a G residue at 18th position in all of these organisms (Fig.1).
Additionally, the tRNAArg, tRNALeu, and tRNAHis share common properties, except tRNAPro.
But, tRNAPro has a G residue at the 18th and 38th positions in these five organisms (Fig.2).
It is proposed that G18 modification base occurrence and non-occurrence in tRNA subsets in Marinobacter is attributed to the inhabitably extreme environments (M. aquaeolei, M. MWYL1), including chemolithotropic N. hamburgensis, N. winogradskyi, and N. oceani.
Thus, a notable finding is that only methionine- and proline-specific tRNAs do not have a G residue at the 18th position, and even its absence at the 18th position does not hamper the possibility to methylate the 2&#39;-OH group of ribose in tRNAs.
It might be a modified Gm18 base (D loop) that interacts with a ψ55 tertiary base (T loop) for conformational rigidity of the tRNA structure [32].
Another finding is the m1G modification site in tRNAs, in which it was previously reported that only three types of tRNAs (tRNAArg, tRNALeu, and tRNAPro) held a G base next to the anticodon [22, 33].
Later, the m1G modification was reported in additional tRNAs, such as tRNAGln in archaea and tRNAAsp in eukaryotes [34].
A study reported tRNAGln with m1G in Mycobacterium tuberculosis H37Rv, tRNAPhe with m1G in Staphylococcus aureus MRSA252, and Streptococcus pneumoniae D39 [31].
The Gm18 modification base G is conserved in these five organisms.
For instance, tRNAMet and tRNAPro sequences do not have a G residue at the 18th position.
Furthermore, the N. hamburgensis organism does not have a G residue in tRNAValGAC.
The m1G and its location are highly conserved and located at the 38th position in tRNAs.
Mostly the purine bases G or A are placed next to the anticodon in all tRNAs.
This modification may influence the codon-anticodon interaction in the translation process [35].
These findings suggest that either these two modifications are not necessary to occur in all tRNAs or might occur with/without G residues at the respective positions.
Hence, we conclude that the Gm18 modifications and m1G37 tRNAs are essential in all three domains and have revealed a conserved tRNAs with m1G37 in a few marine and chemolithoautotrophs.
Further study is called for to include more marine and extremophiles to gain insights into tRNA modification bases and their role in biological processes.
tRNAs sequences are marked at 18th positions, in all organism where tRNAMet and tRNAPro sequences hold A, T or C instead of G at 18th position.
tRNA sequences with G residue occur at 38th position next to the anticodon (G37pG38 sequence).
Title: Amino acid-specific tRNAs that lack a G residue at the 18th and 38th positions
Title: SFannotation: A Simple and Fast Protein Function Annotation System

Owing to the generation of vast amounts of sequencing data by using cost-effective, high-throughput sequencing technologies with improved computational approaches, many putative proteins have been discovered after assembly and structural annotation.
Putative proteins are typically annotated using a functional annotation system that uses extant databases, but the expansive size of these databases often causes a bottleneck for rapid functional annotation.
We developed SFannotation, a simple and fast functional annotation system that rapidly annotates putative proteins against four extant databases, Swiss-Prot, TIGRFAMs, Pfam, and the non-redundant sequence database, by using a best-hit approach with BLASTP and HMMSEARCH.
Functional annotation of putative proteins is a fundamental and essential practice in the postgenomics era [1]; it allows us to analyze genomic and genetic features, such as physiological activity and metabolism, as well as to discover medically and industrially relevant enzymes.
Since large numbers of putative proteins were discovered from a vast amount of sequencing data generated using high-throughput sequencing technologies, including those of the next and third generation, many automated functional annotation systems have contributed greatly to the annotation of them with minimal manual effort [2].
However, their runtime performance of functional annotation against large extant databases often causes a bottleneck, and especially, standalone tools, such as AutoFACT [3] and BLANNOTOR [4], demand high-performance hardware resources for fast annotation from users.
From the user&#39;s perspective, a web-based annotation server system would be a useful tool to bypass the demands of high-performance computer resources, and besides, they offer user-friendly interfaces.
The RAST server system is particularly popular and can be used to rapidly annotate many microbial proteins against a specially curated subsystem database [5].
Web server systems, however, may be undesirable because of critical obstacles, such as the limitation of usable server resources, a long waiting time by many queries, a low-bandwidth network or unstable traffic flow associated with the upload of query data and download of outputs, and data security problems.
Thus, some users prefer standalone systems to web-based systems in spite of the demand for high-performance resources.
Although standalone and web-based systems have good and bad points, slow runtime performance in themselves cannot be avoided because of the exponential increase in database sizes, without controlling some aspect of the annotation workflow.
We developed SFannotation, which rapidly annotates putative proteins by using single or bidirectional best-hit approach with sequence-based methods-BLASTP [6] and HMMSEARCH [7]-against big extant databases: Swiss-Prot [8], TIGRFAMs [9], Pfam [10], and the non-redundant sequence database (NR) of NCBI [11].
As best-hit approaches, especially bidirectional best-hit [12], have been widely utilized in searching reliable homologous protein sequences, such as orthologs, as well as functional annotation systems [13,14,15,16], SFannotation can reliably annotate putative proteins.
Remarkably, SFannotation can rapidly annotate proteins against large extant databases by our hierarchical workflow.
Before annotating putative proteins against Swiss-Prot, TIGRFAMs, Pfam, and the NR database, SFannotation filters out all proteins described in the databases by terms, such as &#34;unknown,&#34; &#34;hypothetical,&#34; &#34;unclassified,&#34; &#34;uncharacterized,&#34; &#34;putative,&#34; &#34;predicted,&#34; and &#34;conserved&#34; (Fig.1A), because some putative proteins may be misannotated by their inclusion.
Then, using BLASTP and HMMSEARCH, SFannotation searches homologous proteins and domains in each refined database using a default threshold (≤10-5 E-value) and selects the highest-scoring homolog to annotate putative proteins as the best-hit approach, such as single best hit and bidirectional best hit [12, 16].
Putative proteins are hierarchically annotated using the following database priority: Swiss-Prot → TIGRFAMs → Pfam → NR, which is ordered according to their reliability (Fig.1B).
Once annotated, the putative proteins are no longer queried using homology searches against the other databases.
For example, if a putative protein is annotated against Swiss-Prot, it is excluded from annotation against the other databases, while the remaining unannotated putative proteins continue to be annotated against the other databases.
Therefore, the runtime performance can be reduced, because the number of unannotated putative proteins gradually decreases (Fig.2).
SFannotation is written in Perl and bash shell and is implemented on a Linux/Unix system on which BLASTP and HMMSEARCH are able to function.
SFannotation automatically annotates putative proteins with downloading of all four databases, as well as BLASTP and HMMSEARCH.
SFannotation is implemented by a command line on the Linux/Unix system: &#34;perl SFannotation --download --fasta &lt;input fasta file&gt; --speedup&#34; (Supplementary Fig.1).
Database filtration (A) and workflow of the SFannotation annotation system (B).
Black arrows represent putative proteins that are annotated by the best-hit approach, and red arrows represent the conversion of unannotated proteins to query putative proteins to search homologs against other databases.
Runtime of the SFannotation system (red) and a best-hit approach without the hierarchical SFannotation workflow (black).
Randomly selected proteins from Escherichia coli MG 1655 (GenBank accession number: U00096) were tested using a 64-bit Linux system (Ubuntu) possessing 20 CPU threads.
Title: Regulation and Function of the Peg3 Imprinted Domain

A subset of mammalian genes differ functionally between two alleles due to genomic imprinting, and seven such genes (Peg3, Usp29, APeg3, Zfp264, Zim1, Zim2, Zim3) are localized within the 500-kb genomic interval of the human and mouse genomes, constituting the Peg3 imprinted domain.
This Peg3 domain shares several features with the other imprinted domains, including an evolutionarily conserved domain structure, along with transcriptional co-regulation through shared cis regulatory elements, as well as functional roles in controlling fetal growth rates and maternal-caring behaviors.
The Peg3 domain also displays some unique features, including YY1-mediated regulation of transcription and imprinting; conversion and adaptation of several protein-coding members as ncRNA genes during evolution; and its close connection to human cancers through the potential tumor suppressor functions of Peg3 and Usp29.
In this review, we summarize and discuss these features of the Peg3 domain.
In mammals, a small subset of autosomal genes are functionally different between two alleles due to an epigenetic mechanism, termed genomic imprinting [1, 2].
The number of imprinted genes is estimated to be 100-200 per mammalian genome.
These genes are usually clustered in specific chromosomal regions; thus, several genes from a given genomic interval are imprinted together as a group.
Each imprinted domain, harboring 5 to 10 members, spans a relatively large genomic interval, ranging from a few hundred kilobases to megabases in length [3].
The genomic structure of a given domain, such as gene content, order, and distance, is well conserved between different mammalian lineages.
The evolutionary constraint maintaining this domain structure is likely related to the fact that the imprinted genes within one domain are usually co-regulated through shared cis regulatory elements.
These regulatory regions are termed imprinting control regions (ICRs) [4].
Several ICRs have been identified from individual domains by a series of human and mouse genetic studies in the past 20 years.
Genetic and molecular studies investigating these ICRs, particularly the ICR of the H19/Igf2 domain, have been very insightful with regard to our understanding of epigenetics, as well as long-distance transcriptional control mechanisms in humans and mice [2].
Paternally expressed gene 3 (Peg3) is one of the imprinted genes that were discovered 20 years ago through subtractive pooling schemes of mRNA between paternal and maternal disomic embryos [5].
An independent study also discovered Peg3 as an important gene for muscle development; it was thus named Pw1 [6].
Shortly after these simultaneous discoveries of mouse Peg3, the human homolog was also identified as one of the zinc finger genes located in human chromosome 19q13.4 [7].
Since then, six additional imprinted genes have been identified from the 500-kb genomic interval surrounding the Peg3 locus, confirming the presence of an imprinted domain in the human and mouse genomes; it was thus named the Peg3 imprinted domain.
A series of follow-up investigations have been conducted to characterize the genomic structures and regulatory mechanisms, as well as the functions of individual genes located within the Peg3 imprinted domain.
In this short review, we intend to 1) summarize the information that we have learned so far regarding the Peg3 domain and 2) discuss the similarities and differences between the Peg3 domain and other imprinted domains.
The 500-kb genomic interval of the Peg3 domain harbors seven imprinted genes: paternally expressed Peg3; Usp29 (ubiquitin-specific hydrolase 29); APeg3 (antisense Peg3); Zfp264 (zinc finger protein 264); and maternally expressed Zim1 (zinc finger gene, imprinted 1), Zim2, and Zim3 (Fig.1) [8,9,10,11,12].
All of these imprinted genes, except Zim1, are conserved among individual mammalian species.
Zim1, however, is found only in mammals with large litter sizes, such as rodents and canines [13].
The order, spacing, and orientation of these imprinted genes are also well conserved between different mammalian species.
This is quite unusual, since the Peg3 domain itself is part of a large zinc finger gene family cluster, which is known to go through dynamic genomic changes during evolution.
This is even more evident in the genomic regions immediately surrounding the Peg3 domain, which are occupied by two gene families: olfactory receptor and vomeronasal organ receptor.
These two gene families show many more frequent lineage-specific expansions and deletions during mammalian evolution, resulting in quite different structures between different species [14].
The changes observed from the surrounding genomic regions are in stark contrast to the structural conservation observed in the Peg3 domain.
Within the 500-kb region of the Peg3 domain, the seven imprinted genes are unevenly distributed: three genes are clustered together on one side, and the remaining four genes are on the other side (Fig.1).
By contrast, the middle 250-kb region is more or less empty, lacking any obvious open reading frame (ORF).
Yet, this overall genomic layout is well conserved among all mammals [8].
According to the results from a sequence comparison of several mammalian species, this middle 250-kb region is filled with 20 small genomic fragments, ranging from 100-300 bp in length, that show high levels of sequence conservation between species.
These regions, termed evolutionarily conserved regions (ECRs), are likely cis regulatory elements that may be involved in controlling the transcription and imprinting of the Peg3 domain [15].
This prediction has been recently confirmed through the epigenomic results in which the majority of these ECRs is associated with two histone modifications, H3K4me1 and H3K27ac (monomethylation of lysine 4 and acetylation of lysine 27, respectively, of histone 3) [16], which are closely associated with enhancers [17].
The relative positions, spacing, and orientations of these potential cis regulatory elements are also well conserved among all mammals.
This large number of potential cis regulatory elements within the 250-kb region hints at one possibility that unknown, but complex, regulatory mechanisms exist to control transcription and imprinting of the Peg3 domain.
Although the 500-kb domain has been structurally well preserved during mammalian evolution, the protein-coding capacity of each imprinted gene has been quite often lost independently in the different lineages of mammals (Fig.1).
For example, three zinc finger genes, Zim2, Zim3, and Zfp264, all have lost their ORFs in the rodent lineage [9, 10].
The current sequences of these genes in rodents have diverged substantially from their original coding sequences, so that they are barely recognizable as zinc finger genes.
Thus, these genes may have lost their ORFs quite a long time ago, at the early stages of rodent evolution.
Nevertheless, they are still transcriptionally active, indicating that these genes might have functionally adapted as non-coding RNA genes.
In fact, one of these genes, Zim3, has become an antisense gene to Usp29: the transcribed regions overlap quite extensively between maternally expressed Zim3 and paternally expressed Usp29.
Usp29 also has taken a similar evolutionary path as seen in the three zinc finger genes.
In the bovine lineage, the overall sequence structure of Usp29 is still intact, without any major deletions and insertions, but it contains several base substitutions causing STOP codons, indicating that these substitutions likely occurred in recent evolutionary times in the bovine lineage [13].
Interestingly, a similar situation has also been observed in one human population: one substitution causing a STOP codon has been found within the coding region of USP29 (http://browser.1000genomes.org/index.html).
So far, Peg3 is the only gene that has maintained its ORF in all lineages of mammals, while the other imprinted genes have lost ORFs in the different lineages of mammals.
This suggests that the functions of Peg3 might be the most critical for the survival of mammals as a protein-coding gene.
On the other hand, the other members of the Peg3 domain might be less critical for the survival of mammals, which subsequently allows them to have adapted as ncRNA genes during mammalian evolution.
The transcription of the Peg3 domain is regulated through at least five promoters: two bidirectional promoters for Peg3/Usp29 and Zim3/Zfp264 and three individual promoters for APeg3, Zim1, and Zim2.
Among these promoter regions, three regions are differentially methylated between two alleles; thus they are named differentially methylated regions (DMRs).
These include the DMRs of Peg3, Zim2, and Zim3 (Fig.1).
According to previous studies, the PEG3-DMR is a gametic DMR, such that its allele-specific methylation is established during gametogenesis in parents-in this case, the maternal-specific methylation during oogenesis [18].
In contrast, the DMRs of Zim2 and Zim3 are established after fertilization and are thus categorized as somatic DMRs [19].
The promoter of Zim1, on the other hand, is not methylated at all on either allele, whereas the methylation status of the APeg3 promoter is currently unknown.
Among the three DMRs, the Peg3-DMR is the most critical for transcription and imprinting of the Peg3 domain.
Deletion of part of this DMR causes changes in the transcriptional levels of all genes in this domain [20].
The same mutation also affects the monoallelic expression (imprinting) pattern of two genes, Zfp264 and Zim2: a switch from paternal- to maternal-specific expression for Zfp264 and a change from maternal to biallelic expression for Zim2.
These results demonstrate that the Peg3-DMR is an ICR for this domain.
It is, however, currently unknown how the Peg3-DMR obtains DNA methylation during oogenesis and how the Peg3-DMR serves as a major cis regulatory region governing the overall transcription of the Peg3 domain.
The 4.0-kb Peg3-DMR can be divided into two small regions: the first 1.5-kb region harbors the bidirectional promoter and the first exon for both Peg3 and Usp29, and the second 2.5-kb region covers the first intron of Peg3.
The 2.5-kb intron region displays an unusual tandem repeat sequence structure, and one small motif (GGCGCCATCTT) within these repeats turns out to be a DNA-binding site for the transcription factor YY1 [21, 22].
Since this YY1-binding motif is part of the tandem repeats, multiple YY1 binding sites, 10 sites on average, are easily found within the 2.5-kb region of the Peg3-DMR for any given mammal [23].
A series of studies have been conducted to understand the potential roles for YY1 in the function of the Peg3-DMR.
First, according to the results, conditional knockdown of YY1 during oogenesis results in hypomethylation of the Peg3-DMR, suggesting that the protein level of YY1 is very critical for the establishment of DNA methylation in the Peg3-DMR [24].
A similar outcome has also been observed in somatic cells using in vivo and in vitro systems [25, 26].
Thus, YY1 is likely to be involved in setting DNA methylation and also in the maintenance of the Peg3-DMR.
Second, the expression of Peg3 is usually up-regulated in response to the reduced levels of YY1 protein.
This suggests that YY1 may function as a repressor of the transcription of Peg3 in somatic cells.
This outcome is usually accompanied by global changes in the expression levels of other imprinted genes in the Peg3 domain, confirming a major role played by YY1 in the entire Peg3 domain [26].
Collectively, the results obtained so far confirm that YY1 is a major regulator of the Peg3 domain, possibly through the Peg3-DMR.
Along with the five promoters, the 20 ECRs located in the middle 250-kb region are also predicted to be involved in the transcriptional control of the Peg3 domain.
To further follow up on this prediction, a series of 3C (chromosome conformation capture) experiments were conducted using various adult tissues harvested from mouse [15].
The results revealed that one of these ECRs interacts with the bidirectional promoter of Peg3/Usp29.
This particular one, ECR18, is located 200 kb upstream of the bidirectional promoter, which is actually positioned closer to the bidirectional promoter of Zim3/Zfp264.
The interaction is the most prominent in the brain but is also detected in other tissues, such as testis and liver.
As described earlier, ECR18 is associated with two histone modifications, H3K4me1 and H3K27ac, which mark active enhancers.
Subsequent in vitro promoter assays indeed demonstrated that ECR18 functions as an enhancer for the promoter of Peg3/Usp29 in several cell lines tested.
The 3C experiments further indicated that ECR18 also interacts with the other promoters of the Peg3 domain.
In particular, the bidirectional promoter of Zim3/Zfp264 and the promoter of Zim2 were shown to interact with ECR18 in the testis.
Thus, ECR18 might be a shared enhancer for several genes in the Peg3 domain, not just for Peg3 and Usp29.
The transcription and imprinting of the Peg3 domain is likely to be co-regulated through the Peg3-DMR, given the observations described above, such as global changes in the expression levels of the Peg3 domain either by targeting of the Peg3-DMR or by targeting of the transcription factor YY1.
This predicted mechanism might be explained in the following manner, which involves ECR18 as a shared enhancer for the Peg3 domain (Fig.2).
On the paternal allele, the Peg3-DMR is unmethylated, and functions as a dominant promoter for both Peg3 and Usp29, thus outcompeting the promoters of the remaining genes, Zim1 and Zim2, for usage of the shared enhancer ECR18.
On the maternal allele, the Peg3-DMR is methylated and silent, so that the promoters of Zim1 and Zim2 can access ECR18 for their transcription.
This results in the paternal expression of Peg3 and Usp29 and the maternal expression of Zim1 and Zim2.
This is particularly true in the brain, as strict imprinting and transcription of these four genes are usually observed in the neonatal brain.
On the other hand, the imprinting and transcription of the remaining genes are either bi-allelic or not detectable in the brain; thus, the proposed model is more applicable for explaining the imprinting of the four genes in the brain.
This model could explain some of the effects observed from the deletion mutant of the Peg3-DMR, showing bi-allelic expression of Zim2 in carriers inheriting the mutation paternally [20].
Deletion of part of the Peg3-DMR, the 2.5-kb YY1-binding sites, is likely to be responsible for the down-regulation of Peg3 expression and subsequently allows the access of Zim2 to the shared enhancer ECR18 on the paternal allele, resulting in the bi-allelic expression of Zim2.
Although reasonable, this model needs to be revised with additional data in the near future in order to explain many aspects of the imprinting and transcription of the Peg3 domain that cannot be explained at the moment.
At the organismal level, mammalian imprinted genes usually have similar functions, controlling fetal growth rates and nurturing behaviors [27, 28].
This is likely the case for the imprinted genes located within the Peg3 domain, based on the following lines of evidence.
First, most genes in the Peg3 domain are expressed in early embryogenesis and the brain.
The expression levels of Peg3 and Zim1 are very high in embryos and placenta [5, 11].
In brain, Peg3, Zim1, Zim2, and APeg3 are all expressed in the hypothalamus [29, 30], which is known to be involved in controlling maternal-caring behaviors and milk provision [31].
Second, mutations in the Peg3 locus tend to have two consistent phenotypic effects [29, 32].
The pups with mutant alleles have defects in milk-suckling behaviors.
As a consequence, the carriers become progressively weaker than their littermates due to the insufficient uptake of milk, which quite often results in perinatal lethality.
The adult mice of both genders also have various reproductive and behavioral problems [32,33,34].
Mutant females produce a relatively small number of mature eggs, resulting in reduced litter sizes.
Also, pregnant females have problems in milk letdown and in maternal-caring behaviors, thus causing the perinatal lethality observed among wild-type pups.
It is interesting to point out that Peg3 mutations have effects on a set of paired behaviors between the mothers and their pups for milk provision, milk suckling, and letdown.
This has been a basis for the &#39;co-adaptation&#39; theory of mammalian genomic imprinting [35].
In the case of bovine Usp29, a mutation deleting a 100-kb interval within the middle 250-kb region has been shown to be responsible for the stillbirth of calves [36].
The exact mechanism by which this deletion causes stillbirth is currently unknown, but this mutational effect is overall consistent with the phenotypes that have been observed from the Peg3 domain, controlling the growth and development of the fetus during the gestational period.
At the cellular level, the predicted functions of Peg3 are worthwhile to be discussed.
Peg3 is a potential tumor suppressor, given the observation that many human patients with ovarian and breast cancers lose the expression of PEG3 due to DNA hypermethylation on the PEG3-DMR [37,38,39].
This was further demonstrated by in vitro studies in which reintroduction of human PEG3 into primary cell lines derived from ovarian cancer patients resulted in the inhibition of cell growth [39].
According to recent data from in vitro studies, Peg3 also appears to be involved in autophagy in endothelial cells [40].
Soluble forms of several structural proteins, including decorin, are shown to trigger autophagy in endothelial cells as part of the apoptotic pathways that are designed to remove stressed and damaged cells.
Interestingly, PEG3 has been found to play a major role in conveying this external signal through vesicular endothelial growth factor receptor 2 (VEGFR2) to induce the transcription of two core components, BECLIN1 and LC3, for autophagosomes.
The potential roles of Peg3 in autophagy are particularly intriguing, since Peg3 is predicted to play important roles in the placenta, where autophagy is critical for the constant turnover of protein and other nutritional materials [41].
It is also relevant to note that nutritional starvation of pregnant females causes changes in the expression levels of a subset of mammalian genes.
Interestingly, the majority of these genes is also affected by mutations on Peg3, suggesting that these genes are likely either direct or indirect downstream genes of Peg3 [41].
Thus, Peg3 may be a major player in controlling in nutritional supply between females and pups during pregnancy.
Since autophagy is regarded as a part of apoptotic pathways, the potential roles for Peg3 in autophagy are also consistent with previous observations hinting at the functions of Peg3 in apoptosis.
These include up-regulation of Peg3 during p53-mediated apoptosis, muscle cachexia (or wasting) in response to tumor load, and induction of Peg3 in response to hypoxic conditions [42,43,44].
Taken together, these results support the idea that Peg3 is a tumor suppressor with a potential connection to autophagy.
At the molecular level, the imprinted genes in the Peg3 domain code for two different types of proteins: a ubiquitin-specific protease for Usp29 and zinc finger proteins for Peg3, Zim1, Zim2, Zim3, and Zfp264.
According to recent studies, USP29 might also be a tumor suppressor, given the observation that USP29 is required for the stability of Claspin, a key protein for the DNA damage checkpoint [45].
This is quite intriguing, since DNA hypermethylation of the PEG3-DMR has always been regarded as an epigenetic abnormality affecting the expression of PEG3 but not USP29, although the PEG3-DMR controls both genes.
Thus, it would be interesting to test whether the outcome of this DNA hypermethylation manifests as down-regulation of PEG3 only or PEG3 and USP29 together.
Among the four zinc finger proteins within the Peg3 domain, Peg3 is the only gene that has been studied so far at the protein level.
According to recent studies, PEG3 binds to many genomic loci as a DNA-binding protein with transcriptional repression activity [46].
It is well known that many zinc finger proteins, including those in the Peg3 domain, repress their downstream genes through interaction with KRAB-associated protein 1 (KAP-1) and SET domain-containing histone methylase at H3K9 (SETDB1) [47, 48].
Thus, we predict that PEG3 may also repress its downstream genes through SETDB1/KAP-1-driven H3K9me3.
According to recent studies, several placenta-specific gene families are de-repressed in mouse embryos with the Peg3-null mutant allele; yet, these gene families are usually repressed through H3K9me3-mediated mechanisms [49, 50].
Thus, this result again supports the idea that PEG3 controls its downstream genes through H3K9me3-mediated mechanisms.
At the molecular level, several genes within the Peg3 domain are also predicted to function as ncRNA genes (Fig.1).
Among these genes, APeg3 is quite unique as an RNA gene due to the following reasons.
First, it is well conserved among all mammals [51].
Second, it is an antisense transcript gene localized in the 3&#39;-UTR of another proteincoding gene-in this case, Peg3 [12].
Third, APeg3 is expressed only in the hypothalamus but highly up-regulated in response to osmotic challenge [30].
Given these special circumstances, it has been predicted that APeg3 may be designed to control the expression of Peg3.
Consistent with this prediction, in vitro studies demonstrated that APeg3 has the potential to down-regulate the mRNA and protein levels of Peg3 [51].
Although the mechanistic basis for the observed down-regulation needs to be investigated further, the in vivo functions of APeg3 are most likely connected to those of Peg3.
The Peg3 domain displays unique features and also shares several features with the other imprinted domains.
First, the domain structure has been well preserved during mammalian evolution, as seen in the other domains.
This makes sense with the prediction that the imprinting and transcription of the Peg3 domain are likely controlled, as a group, through a shared cis regulatory element-in this case, the Peg3-DMR.
As part of the conserved genomic interval, many ECRs are found to be localized within the middle 250-kb region and are also predicted to be enhancers for individual imprinted genes within the Peg3 domain (Fig.1).
Coordinating these potential enhancers with individual promoters may be needed, since some of these ECRs are likely to be shared among individual imprinted genes.
As an ICR, the Peg3-DMR might be responsible for this coordinating task.
According to a situation observed in the H19/Igf2-imprinted domain, its ICR functions as an enhancer blocker through another transcription factor, CTCF, controlling the accessibility of a shared enhancer between H19 and Igf2 [1, 2].
Interestingly, the ICR of the H19/Igf2 domain also has an unusual cluster of CTCF binding sites, which is very similar to the localization of a tandem array of multiple YY1 binding sites within the Peg3-DMR [52].
Although both ICRs are similar in many ways, their roles as ICRs are likely different from each other.
This prediction is based on the result that YY1 and YY1 binding sites do not display enhancer-blocking activity [53].
One plausible role for YY1 might be &#39;linking&#39; or &#39;tethering&#39; long-distance enhancers to the corresponding target promoters [54].
As described earlier, ECR18 interacts with the bidirectional promoter of Peg3/Usp29.
This long-distance interaction, or linking, between ECR18 and the bidirectional promoter may be mediated through the YY1/YY1-binding sites within the Peg3-DMR (Fig.2).
A similar scenario is also possible for the other ECRs found within the Peg3 domain, which warrants a series of systematic and careful investigations in the near future.
Overall, as seen in other domains, the conserved genomic structure of the Peg3 domain makes sense with the presence of co-regulation through the Peg3-DMR, but the mechanisms governing the Peg3 domain by this ICR are likely to be unique, based on the involvement of the transcription factor YY1.
Second, the overall functional outcomes of the Peg3 domain are similar to those observed from other domains, such as controlling fetal growth rates and maternal-caring behaviors [27, 28, 35].
In particular, this has been consistently demonstrated through functional studies of the Peg3 locus [29, 32].
Despite these similar outcomes at the organismal level, the potential function of each imprinted gene is likely to be unique at the molecular and cellular levels.
For instance, Peg3 codes for a DNA-binding protein with repression activity at the molecular level and inhibits cell growth as a tumor suppressor at the cellular level.
Yet, Peg3 is regarded as a growth stimulator at the organismal level, since the down-regulation of Peg3 usually results in reduction in body size.
The functional outcomes of Peg3 seem to be somewhat contradictory between the cellular and organismal levels.
This is quite unique compared to other imprinted genes encoding proteins with different biochemical properties, such as paternally expressed insulin-like growth factor 2 (Igf2) and maternally expressed insulin-like growth factor 2 receptor (Igf2r).
The phenotypic outcomes of this pair of ligand and receptor are overall similar at the cellular and organismal levels [27, 28].
Igf2 stimulates cell growth, and thus, higher levels of Igf2 result in bigger body size.
Igf2r derives exactly the opposite phenotypes to those of Igf2, inhibiting cell growth, and thus, more Igf2r results in smaller body size.
As proposed earlier, the conflicting phenotypes of Peg3 at the cellular and organismal levels could be explained in the following manner [55].
Peg3 might be involved in programmed cell death during the development of neuron cells-specifically, a set of neuron cells producing growth suppressors.
The absence of Peg3 could result in the overgrowth of these neuron cells with an excessive amount of growth suppressors at the cellular level, causing smaller body size at the organismal level.
Although this remains to be tested, it is feasible, since many imprinted genes are highly expressed in the neuroendocrine system, such as the hypothalamus, which is responsible for controlling body size through growth hormones [31].
In that regard, it is interesting to note that another paternally expressed DNA-binding gene, Zac1 (zinc finger protein regulating apoptosis and cell cycle arrest), also shows similar conflicting outcomes-it is a growth suppressor at the cellular level but a growth stimulator at the organismal level [56, 57].
Since both Peg3 and Zac1 are growth suppressors at the cellular level, these genes have been quite often identified as tumor suppressors.
Although the overall functional contributions by the Peg3 domain are similar to those of other domains at the organismal level, the detailed function of each imprinted gene is predicted to be unique at the cellular and molecular levels.
Third, the Peg3 domain also contains several ncRNA genes, as seen in other imprinted domains.
These ncRNA genes include Zim2, Zim3, and Zfp264 in the rodent lineage (Fig.1).
These ncRNA genes are unique due to the following reasons.
First, these ncRNA genes are of relatively recent evolutionary origin, so that their ancestral genes can be identified easily through sequence similarity to their corresponding protein-coding sequences.
It is currently unknown whether these ncRNA genes are in the process of becoming pseudogenes or have already adapted as ncRNA genes with some functions.
Regardless, they have maintained their transcriptional activity for quite a long time, given the accumulation of mutations disrupting their ORFs, suggesting that their gene products, ncRNA, or transcription itself is now part of the Peg3 domain.
If some of them have already adapted as ncRNA genes, examining their sequences should provide some clues regarding how protein-coding sequences become ncRNA genes during evolution.
Thus, the ncRNA genes of the Peg3 domain might provide a very unique opportunity to understand the evolutionary paths of mammalian genes.
In that regard, it is relevant to note that mammalian X chromosome inactivation-specific transcript (Xist) represents another well-known case where a proteincoding gene has become an ncRNA gene [58].
Second, some functions of these ncRNA genes should be lineage-specific to rodents, since the original genes of these ncRNA genes are still protein-coding genes in all other mammals.
This level of lineage-specific change has not been observed from other imprinted domains.
This might be related to the fact that the Peg3 domain has been part of a large zinc finger gene family, which is known to be vulnerable to genomic rearrangements, including deletion and duplication [14].
Also, the functional redundancy afforded by the zinc finger gene family might have allowed its members to drift away from their proteincoding sequences without severe phenotypic consequences.
Overall, the structurally conserved genomic interval of the Peg3 domain might have allowed several protein-coding members to adapt into ncRNA genes, which seems to be quite different from the evolutionary patterns observed from other imprinted domains.
The Peg3 domain shows many similar features as seen in other imprinted domains, which include its conserved genomic structure, allowing long-distance transcriptional regulation through ICRs, as well as allele-specific DNA methylation and subsequent monoallelic expression.
In the past two decades, these unusual features have been studied extensively using other imprinted domains as model loci.
At the same time, the Peg3 domain displays several unique features that have not been studied in other imprinted domains.
These unique features are thus worthwhile to study in the future in increase our understanding of genomic imprinting and imprinting-related human disorders.
First, YY1/YY1-binding sites appear to be a major player in the Peg3 domain.
In particular, a tandem array of YY1-binding sites has been selected for the function of the Peg3-DMR; yet, the molecular basis for this function is still unknown.
One possibility would be that these multiple YY1-binding sites might be involved in setting DNA methylation during oogenesis.
The other possibility would be that these YY1-binding sites might be required for the transcriptional control of Peg3 and adjacent imprinted genes.
Targeting these YY1-binding sites through mutagenesis experiments should provide a clear and direct answer regarding the in vivo role of multiple YY1-binding sites.
Second, Peg3 has been recognized as a potential tumor suppressor for some time, without any progress in our understanding.
According to recent studies, Peg3 may play a major role in regulating autophagy of endothelial cells, which is very intriguing, due to the following reasons: 1) Peg3 is highly expressed in the organs that are regarded as endothelial cell-rich tissues, such as placenta, and 2) autophagy is also regarded as part of apoptotic pathways.
Thus, characterizing the potential roles of Peg3 in autophagy should be a worthy project in the immediate future.
Third, the Peg3 domain appears to have evolved very rapidly during mammalian evolution.
As a consequence, several members of this 500-kb domain have changed from protein-coding to ncRNA genes.
It is well known that the ancestral state of a given genomic interval cannot be easily reconstructed, due to the many changes that have accumulated during long evolutionary times.
In the case of the Peg3 domain, its 500-kb genomic structure has been so well preserved that its ancestral structure can be readily reconstructed by comparing the Peg3 domains of several mammals.
Thus, careful sequence analyses should provide immediate insights regarding the evolutionary paths of mammalian genes, especially those that have undergone the transformation from protein-coding to non-coding RNA genes.
Overall, these directions are worthwhile to pursue in the near future, which should most likely provide additional exciting information regarding how the Peg3 domain is regulated and what functional roles the Peg3 domain plays in mammalian biology.
Evolutionary conservation of the Peg3 imprinted domain.
The 500-kb genomic interval of the Peg3 domain is shown using the mouse genomic region as a representative locus, along with its relative position in mouse chromosome 7 (Mmu7).
The paternally expressed and maternally expressed genes are indicated with blue and red arrows, respectively.
The three differentially methylated regions (DMRs) are indicated with grey boxes.
The 20 evolutionarily conserved regions (ECRs) are shown with vertical blue lines, except ECR18 in red.
The graph in the middle represents the sequence conservation levels of the Peg3 domain among placental mammals.
The table below summarizes the evolutionary conservation of gene content and their open reading frames (ORFs) with different colors: black for conserved and green for non-conserved.
The first two exons of mouse Usp29 were previously named ossification center-associated transcript (Ocat), while those in humans and cow are named MER1 repeat-containing imprinted transcript 1 (MIMT1).
aThe ORF indicates the loss of the USP29 ORF in one human population.
Transcription and imprinting control of the Peg3 domain.
The schematic representation illustrates a potential mode of the Peg3-DMR and ECR18 in the co-regulation and mono-allelic expression (imprinting) of four genes in the mouse brain: the paternally expressed Peg3 and Usp29 and the maternally expressed Zim1 and Zim2.
On the paternal allele, the Peg3-DMR becomes a dominant user for the shared enhancer ECR18 and out-competes the remaining two genes, Zim1 and Zim2.
Thus, Peg3 and Usp29 are expressed from the paternal allele, while Zim1 and Zim2 become silent on the paternal allele.
On the maternal allele, the Peg3-DMR is methylated, thus causing the repression of both Peg3 and Usp29.
As a consequence, the two genes Zim1 and Zim2 can access ECR18 and be expressed from the maternal allele.
However, the current model cannot explain the imprinting of the three remaining genes, APeg3, Zim3 and Zfp264, since some of these genes are expressed mainly in testis but not in brain.
Title: A Follow-up Association Study of Genetic Variants for Bone Mineral Density in a Korean Population

Bone mineral density (BMD) is one of the quantitative traits that are genetically inherited and affected by various factors.
Over the past years, genome-wide association studies (GWASs) have searched for many genetic loci that influence BMD.
A recent meta-analysis of 17 GWASs for BMD of the femoral neck and lumbar spine is the largest GWAS for BMD to date and offers 64 single-nucleotide polymorphisms (SNPs) in 56 associated loci.
We investigated these BMD loci in a Korean population called Korea Association REsource (KARE) to identify their validity in an independent study.
The KARE population contains genotypes from 8,842 individuals, and their BMD levels were measured at the distal radius (BMD-RT) and midshaft tibia (BMD-TT).
Thirteen genomic loci among 56 loci were significantly associated with BMD variations, and 3 loci were involved in known biological pathways related to BMD.
In order to find putative functional variants, nearby SNPs in relation to linkage equilibrium were annotated, and their possible functional effects were predicted.
These findings reveal that tens of variants, not a single factor, may contribute to the genetic architecture of BMD; have an important role regardless of ethnic group; and may highlight the importance of a replication study in GWASs to validate genuine loci for BMD variation.
In an aging society, osteoporosis is a major health problem.
It is characterized by systemic impairment of bone mass, strength, and microarchitecture, leading to increased risk of fragility fractures, particularly hip fracture [1].
In the United States, the cost of incident fractures due to osteoporosis and low bone mass was $17 billion in 2006.
It is expected to rise to around $25.3 billion by the year 2025 [2].
Likewise, the burden for them is prevalent in Asia.
The annual incidence of osteoporosis-related fractures increased to 1,661, 1,646, 1,623, and 1,614 per 100,000 persons from 2005 to 2008 in Korea [3].
Osteoporosis is defined by the measurement of bone mineral density (BMD), which is so far the best single predictor [4, 5].
BMD is a genetic component, with estimates of heritability ranging from 0.46 to 0.84 at all skeletal sites [6].
In recent years, genome-wide association studies (GWASs) have enhanced the understanding of the genetic architecture underlying quantitative traits and common, complex diseases [7].
These studies provide key insights into the variation of quantitative traits and prospects for designing specifically effective strategies for diseases [8].
There have been many GWASs to identify genetic loci that influence BMD, and dozens of candidates have been presented [9,10,11,12,13].
However, only several loci have been consistently identified in a few replication studies [14,15,16], suggesting that BMD is a complex trait affected by multiple components [17].
Recently, a meta-analysis of 17 studies for BMD of the femoral neck and lumbar spine was performed for the first time in order to identify consensus BMD-associated single-nucleotide polymorphisms (SNPs) throughout races [18].
It included 32,951 individuals across North America, Europe, East Asia, and Australia.
The meta-analysis offered 56 associated loci that were highly significant in both the discovery stage and replication stage.
This also proposed a model of BMD loci and successfully validated it with an independent population.
Here, we investigate those 56 BMD loci in a Korean population called Korea Association REsource (KARE) [19].
The analysis showed that BMD variation is a polygenic and complex trait, and it could be even different, depending on sex and the skeletal site.
The finding that some crucial loci are still significant in KARE supports the validity of BMD loci.
This study may contribute to the importance of a replication study in GWASs to understand the common mechanism underlying BMD.
Title: Study subjects

Subjects from the KARE GWAS have been described [19].
Since it launched in 2001, the total of 10,038 participants have been recruited from the Anseong and Ansan cohorts in the Republic of Korea.
Both cohorts were designed to view a variety of quantitative traits from a long-term perspective.
Besides, clinical surveys have been performed through epidemiological surveys and physical examinations.
Participants have been examined every 2 years, with a third follow-up study in 2008.
The analysis of KARE data has been approved by the Korea National Institute for Bioethics Policy (KONIBP), Republic of Korea (P01-201311-RS-02-01).
Title: Genotyping, quality control, and imputation

A total of 10,004 DNA samples were genotyped with the Affymetrix Genome-Wide Human SNP array 5.0 (Affymetrix, Santa Clara, CA, USA).
Genotypes were called by Bayesian Robust Linear Modeling using the Mahalanobis Distance (BRLMM) algorithm [20].
The quality score criteria and procedures have been described in a previous GWAS [19].
After each step of the quality control, genotypes from 8,842 individuals with a total of 352,228 markers were left and used for the association analysis.
In this study, we additionally excluded 257 individuals taking medication that influenced BMD levels.
SNP imputation was conducted using the IMPUTE2 program [21].
The 1000 Genomes Phase I integrated haplotypes (released in Dec 2013), produced using SHAPEIT2 in NCBI build 37 coordinates, were used as the reference panel [22].
All 1,092 individuals with 36.8 million SNPs were initially utilized, and haplotypes with singleton sites were removed.
Since there is no X chromosome in the current version of the reference panel, a previous version (released in Mar 2012) was used as the reference panel for the X chromosome.
Because the previous GWAS was based on NCBI build 36, SNP positions should be converted to build 37 coordinates by using the University of California, Santa Cruz (UCSC) liftOver tool [23].
SNP orientations were aligned to the forward strand using PLINK [24].
In order to improve the performance of the SNP imputation, SNPs were pre-phased by SHAPEIT2 [22, 25].
Then, SNPs were imputed directly onto the haplotype estimates by IMPUTE2.
Imputation was carried out for the regions within 500 kb on either side of 64 SNPs reported in the previous GWAS [18].
SNPs with a minor allele frequency &lt; 0.01 and imputation quality score (INFO) &lt; 0.4 were filtered out.
Then, we combined 51 imputed SNPs with 9 directly typed KARE SNPs for association analyses.
Title: Statistical analysis

The association analyses were done using PLINK version 1.07 [24] for both genotyped and imputed SNPs.
A test for BMD was done by linear regression with an additive model (1 degree of freedom [d.f.])
after adjustments for age, sex, and recruitment area.
Statistical significance was determined by p &lt; 0.05 in a two-tailed test.
p-values were not adjusted for multiple testing of SNPs.
Proximal SNPs and pairwise linkage disequilibrium (LD) were computed by PLINK.
A threshold for correlation was set as r2 &gt; 0.8.
Annotation of SNPs was done by SNPnexus [26].
The effect of SNPs on protein function was predicted using SIFT [27] and PolyPhen [28].
Transcription factor binding sites were inferred from the TRANSFAC database [29].
The KARE study population included GWA genotypes from 8,842 individuals [19].
We removed 257 individuals currently taking medication to remove any potential influence on BMD.
BMD was measured by speed of sound (SOS) at the distal radius (BMD-RT) and midshaft tibia (BMD-TT), but the value was transformed to T-score for analysis.
The mean T-score of the radius and tibia of the 8,586 subjects was 0.038 ± 0.016 (SEM) and -0.472 ± 0.017, respectively.
The meta-analysis of 17 GWASs identified 64 SNPs in 56 loci associated with BMD with genomewide significance.
The analysis also proposed a model of BMD-decreasing alleles, and their effects were validated with an independent population: a prospective study in postmenopausal Danish women aged 55-86 years.
The alleles were scored with weights based on their individual effects on BMD.
Even though the previous GWAS was based on lumbar spine and femoral neck BMD, we assumed that the effects of BMD loci would be available to BMD at other sites, such as the distal radius and midshaft tibia.
Thus, the model was applied to the KARE population to prove its validity.
Among 64 SNPs, 9 SNPs were genotyped in the chip, but 55 other SNPs were imputed from genotypes in the 1000 Genomes (1KG) Project (Phase I).
After filtering, 60 SNPs were used for the analysis, and individuals were scored according to the scoring system of the previous study.
Then, individuals were grouped into five bins based on their scores, and the mean of BMD of each group was measured (Fig.1).
While BMD-RT did not follow the model proposed in the previous study, BMD-TT showed the pattern as expected.
In other words, BMD at the midshaft tibia decreased as the score increased.
The analysis on the basis of 60 SNPs explained 3.2% of the total genetic variance in BMD-TT.
In order to identify variants significantly influential in the Korean population, an association analysis of 60 SNPs was done with KARE data.
Single-marker tests of association with BMD were carried out by trend test (1 d.f.)
after adjustments for age, sex, and recruitment area (Table 1).
Twelve SNPs showed a directionally consistent association with the previous GWAS (p &lt; 0.05).
Six loci were associated with BMD-RT and six loci were associated with BMD-TT, but there were no loci associated with both.
One of these loci is on the X chromosome.
The same analysis was carried out by gender strarification (Table 2).
Three loci were prioritized in the female sample set, and one of them (rs13204965) was newly found in the analysis.
Therefore, a total of 13 independent SNPs were selected for further analysis (Fig.2).
As mentioned in the previous GWAS, some of these 13 SNPs were seen as members of biological pathways related to BMD: the RANK-RANKL-OPG pathway (TNFRSF11B), mesenchymal stem cell differentiation (SP7), and Wnt signaling (WNT4).
Interestingly, SNPs in TNFRSF11B and SP7 were associated with BMD-RT, whereas the SNP in WNT4 was associated with BMD-TT (Tables 1 and 2).
For the purpose of identifying putative functional SNPs, markers in the surrounding 100-Kb region were examined using LD (Table 3).
Likewise, they were imputed from 1KG Phase I, centered on 13 loci.
Eight BMD-associated SNPs were highly correlated (r2 &gt; 0.8) with functional SNPs annotated in the 1KG reference.
rs13245690 was related to 5 putative functional variants, which correlated with each other by LD (r2 &gt; 0.8).
Two SNPs (rs1524498 and rs41281692) were located within coding regions in CPED1.
rs1524498 leads to synonymous coding, but rs41281692 leads to a conservative change of an amino acid.
The remaining 3 SNPs (rs10228519, rs10274486, and rs1357756) were located in the intronic regions in CPED1 and are predicted to alter alleles in transcription factor binding sites.
Each allele was anticipated to affect the binding of CUTL1, POU2F1, and GATA-1, respectively.
rs2016266 in SP7 was correlated with two SNPs (rs1318648 and rs17125266) in ESPL1.
Both were associated with missense mutations.
rs17125266 was predicted to be conservative, but rs1318648 was predicted to be deleterious.
rs2062377 was related to a conservative missense variant in TNFRSF11B.
On the contrary, rs12821008 was related to the sense variant.
Two other SNPs were associated with the 3&#39; untranslated region: rs1346004 in GALNT3 and rs163879 in DCDC5 probably affect splicing regulation.
In addition, rs7932354 was predicted to be related to a splice site variant in F2.
In this report, we identified that the model proposed in a previous GWAS [18] is available to the Korean population.
Furthermore, we found 13 genomic loci significantly associated with BMD variations in the population.
Our results reveal that dozens of variants with small effects may contribute to the genetic architecture of BMD and have important roles, regardless of ethnic groups.
Our study supports the important role of several biological pathways related to BMD.
The Wnt factor (WNT4) has been known to be associated with BMD in many studies [30,31,32].
A pathway involved in mesenchymal stem cell differentiation (SP7) was also previously known [33, 34].
The RANK-RANKL-OPG pathway (TNFRSF11B) is another pathway influencing BMD [35,36,37].
Interestingly, SNPs in TNFRSF11B and SP7 were associated with BMD-RT, whereas an SNP in WNT4 was associated with BMD-TT.
It may suggest differential genetic influences between skeletal sites.
In line with this phenomenon, our results reveal that there is sex- and site-specificity underlying BMD variation.
rs13204965 was only significant in women and showed significant sex heterogeneity.
Also, in the same manner, the variant was only associated with females in a previous GWAS [18].
In a recent GWAS focusing on women of a specific age (postmenopausal women, age 55-85 years), rs13204965 was associated with BMD at the femoral neck and lumbar spine [13].
In case of rs13245690, a sex-stratified analysis in women showed a more significant result than in pooled samples.
These results reflect the sexual dimorphism of bone.
Based on the different composition at different skeletal sites, all 12 loci that were significant in pooled samples showed site specificity in BMD.
Interestingly, even though the extent of effect size was different from skeletal sites, the direction was consistent between sites.
This indicates that BMD-associated loci commonly affect BMD determination across skeletal sites, but the different sensitivity of each site may lead to differential genetic impact on BMD variation.
This could account for the decreased effect size of BMD loci in the Korean population.
While the meta-analysis looked at the BMD at the femoral neck and lumbar spine, the KARE focused on BMD at the distal radius and midshaft tibia.
Besides, the former were measured by dual-energy X-ray absorptiometry, but the latter were estimated as SOS by quantitative ultrasound.
It has been previously shown that BMD of the radius, assessed by ultrasound, poorly coincides with radiologically assessed BMD at the lumbar spine [38].
These represent the limitations of this study- that direct replication at specific sites cannot be performed.
Nevertheless, some loci found in the previous GWAS still showed a directionally consistent association in this study.
This suggests that a common mechanism underlies BMD throughout different skeletal sites.
In order to identify putative functional variants, SNP-nexus was chosen to annotate proxy SNPs near the BMD-associated loci.
In coding regions, some SNPs were expected to present single amino acid substitutions that could be damaging or tolerated.
On the other hand, the other SNPs were expected to be synonymous.
However, this type of mutation sometimes leads to unsuspected effects.
Other annotated SNPs had possible functional effects on the regulation of splicing or the binding activity of transcription factors.
To understand how genetic variants affect the phenotype exactly, multi-layered experimental approaches should be carried out in the future.
In conclusion, these findings highlight the polygenic and complex variation of BMD, which could differ even by sex and skeletal site, but provide evidence of the validity in BMD-associated loci between independent populations.
This study may contribute to the importance of replication studies in GWASs in understanding the common mechanism underlying BMD.
Combined effect of bone mineral density (BMD)-decreasing alleles in Korea Association REsource (KARE) data.
The genetic score of each individual was computed on the basis of the weighted model proposed in a previous study [18].
Effects are shown for BMD levels measured at the distal radius (BMD-RT) (A) and BMD levels measured at the midshaft tibia (BMD-TT) (B).
Histograms represent the numbers of individuals with each genetic score area (left y-axis).
Round spots (right y-axis) show mean BMD-RT standardized values in A and mean BMD-TT standardized values in panel (B).
Vertical lines represent 95% confidence limits.
Effects of significant BMD loci on BMD at different skeletal sites.
Genetic effects are expressed as standardized values and shown for BMD levels measured at the distal radius (BMD-RT; filled circles) and BMD levels measured at the midshaft tibia (BMD-TT; open circles).
Horizontal lines represent 95% confidence limits.
ars13204965 was seen as significant in women only.
Title: Estimated effects of genomewide-significant SNPs on BMD-RT and BMD-TT

Title: Gender-stratified association analysis of SNPs on BMD-RT and BMD-TT

Title: Predicted functional SNPs in LD with targeted SNP in each locus
Title: Genome-Wide Association Study of Medication Adherence in Chronic Diseases in the Korean Population

Medication adherence is generally defined as the extent of voluntary cooperation of a patient in taking medicine as prescribed.
Adherence to long-term treatment with chronic disease is essential for reducing disease comorbidity and mortality.
However, medication non-adherence in chronic disease averages 50%.
This study was conducted a genome-wide association study to identify the genetic basis of medication adherence.
A total of 235 medication non-adherents and 1,067 medication adherents with hypertension or diabetes were used from the Korean Association Resource project data according to the self-reported treatment status of each chronic disease, respectively.
We identified four single nucleotide polymorphisms with suggestive genome-wide association.
The most significant single nucleotide polymorphism was rs6978712 (chromosome 7, p = 4.87 × 10-7), which is located proximal to the GCC1 gene, which was previously implicated in decision-making capability in drug abusers.
Two suggestive single nucleotide polymorphisms were in strong linkage disequilibrium (r2 &gt; 0.8) with rs6978712.
Thus, in the aspect of decision-making in adherence behavior, the association between medication adherence and three loci proximal to the GCC1 gene seems worthy of further research.
However, to overcome a few limitations in this study, defining the standardized phenotype criteria for self-reported adherence should be performed before replicating association studies.
Medication adherence is generally defined as the extent to which patient take medications to coincide with the healthcare provider&#39;s recommendation [1].
Adherence to long-term treatment in chronic disease, such as hypertension and diabetes, is essential for reducing the risks of disease comorbidity and mortality [2].
Poor adherence to medication can cause worsening of disease, death, and increased financial burden for society [1].
Most cases of hypertension and diabetes are inextirpable.
The main goal of treatment for these chronic diseases is to delay disease progression and to prevent developing complications.
Thus, medication adherence is one of the most important factors for success in lifelong treatment [3].
However, medication non-adherence in chronic disease averages 50% [4].
The World Health Organization reported that non-adherence is a leading cause of preventable death and cost.
Predictors of poor adherence are a useful indicator for physicians to identify patients who are most in need of interventions to improve adherence [1].
Illness perceptions and self-efficacy are powerful predictors of adherence [2].
These health-related cognitions have associations with personality, and personality traits can be conveniently summarized by a five-factor model of personality (conscientiousness, neuroticism, agreeableness, extraversion, and openness) [5].
Several studies have explored the associations between personality and adherence.
Among the 12 previous publications exploring associations between personality traits and medication adherence, 7 studies reported that medication adherence has been positively related to conscientiousness, 5 studies reported negative relationships with neuroticism, 2 studies reported about agreeableness with positive associations, and 1 study reported a positive relationship with extraversion [6, 7, 8].
A genetic basis of personality traits has been identified from behavior genetic studies.
Twin, adoption, and family studies show that heritability of personality is about 50% [9].
Several studies have attempted to identify candidate genes with five dimensions of personality, but the results were inconclusive.
Genetic linkage studies have suggested only a few genomic regions for personality traits.
The genetic influences on personality traits are likely attributable to many genes with small effects.
Few genome-wide association studies (GWASs) have identified genetic variants associated with personality traits.
However, most of the associations with personality traits could not be replicated, except in one study [8].
Unlike the attempts to elucidate the genomic associations of personality, there is no research on the genetic aspect of medication adherence until now.
We assumed that medication adherence has a genetic background, as the personality traits that affect medication adherence have a genetic basis for heritability.
Therefore, we conducted a GWAS to identify the association between medication adherence and genotype.
Title: Study subjects

A nested case-control study of medication adherence in chronic diseases (hypertension or diabetes) was conducted within the Korean Association Resource (KARE) project data (which included genotypes from 8,842 participants from the rural Anseong and urban Ansan cohorts as part of the Korean Genome Epidemiology Study) [10].
Two groups were extracted from the KARE data, one of medication non-adherents with chronic disease (cases) and the other of medication adherents with chronic disease (controls), based on the following procedure (Fig.1).
There were 1,387 and 613 individuals who were diagnosed with hypertension and diabetes, respectively, of a total of 8,842; 1,540 individuals were defined as patients with chronic disease after excluding 230 patients with comorbid hypertension and diabetes.
Among 1,540 patients with chronic disease, a patient who was undergoing treatment for hypertension or diabetes or who was taking hypertensive or diabetes medicines (including insulin) were classified as the control group (n = 1,072).
Patients who was not receiving treatment or taking medicines were classified as the case group (n = 239).
Individuals whose information about treatment was not provided were removed (n = 229).
Consequently, 235 cases and 1,067 controls were used for the GWAS after excluding 9 patients with comorbid psychotic disorders or cognitive disorders, such as dementia [1].
Title: Genotyping

The genotyping and quality control procedures for the KARE data have been described previously [10].
Single nucleotide polymorphisms (SNPs) with a high missing gene call rate (&gt;5%), low minor allele frequency (&lt;0.05), and significant deviation from Hardy-Weinberg equilibrium (p ≤ 0.001) were excluded, leaving a total of 311,493 SNPs to be examined.
Title: Statistical analyses

The case-control GWAS was conducted using PLINK software version 1.07 [11].
A quantile-quantile (QQ) plot and a Manhattan plot were produced using the R statistics package [12].
A regional plot was generated using LocusZoom 1.1 [13].
Student&#39;s t-test and Fisher&#39;s exact test were used to test differences in basic and clinical characteristics between case and control groups.
Title: Characteristics of study subjects

The general and clinical characteristics of the case and control groups are described in Table 1.
The case group (non-adherents) consisted of 80 subjects with diabetes and 155 individuals with hypertension, and the control group (adherents) consisted of 240 and 827, respectively.
Individuals with diabetes tended to adhere less to medication than hypertensive patients [14].
The mean age of the case group was 54.29 ± 8.10 and 57.66 ± 7.90 years in the control group.
Adherents were significantly older than non-adherents.
Males were less adherent than females.
All results above are consistent with previous studies of factors affecting medication adherence [3].
Among the clinical characteristics, diastolic blood pressure in hypertensive individuals was higher in the non-adherence group.
However, systolic blood pressure in hypertensive patients and fasting plasma glucose in diabetic person were similar between adherence and non-adherence groups.
The rates of family history of hypertension, diabetes, and their life-threatening complications, such as cerebrovascular attack and coronary artery disease, did not differ between case and control groups.
Title: SNPs associated with medication adherence

The GWAS for medication adherence in 1,067 medication adherents and 235 medication non-adherents showed multiple SNPs with a suggestive association (Fig.2).
The QQ plot showed that most of the observed associations lay close to the expected distribution, conforming to the null hypothesis of no association.
The genomic inflation factor was 1.00, indicating no evidence of inflation in the test statistics due to population stratification or other sources.
SNPs with a suggestive genome-wide association (p &lt; 1 × 10-5) are shown in Table 2.
Although no SNP met the criteria of genome-wide significance (p &lt; 5 × 10-8), four loci (rs6978712, rs10487488, rs7690819, rs11973012) with suggestive significance were identified.
The most significant SNP was rs6978712 (p = 4.87 × 10-7), which is located proximal to the GCC1 gene, which was previously implicated in decision-making capability in drug abusers [15].
Three SNPs on chromosome 7 were in close proximity, and Fig.3 shows a regional association for these SNPs proximal to the GCC1 gene.
Medication adherence averages only 50% among patients receiving treatment for chronic diseases.
Poor medication adherence contributes to worsening disease, death, and increased health care costs.
Moreover, if medication adherence is not guaranteed, new therapeutic options, such as personalized medicine that aims to customize healthcare to individual patients based on their genetic background, are merely useless [16].
However, in spite of its importance in personal and social aspects, the genetic background for medication adherence has never been studied.
Since personality traits that have a genetic background affect behaviors, including medication non-adherence, this study aimed to explore the genetic basis of medication adherence in chronic diseases for the first time.
Among the hypertensive or diabetic patients from the KARE cohort study, 235 medication non-adherent individuals (case) and 1,067 medication adherent individuals (control) were extracted according to the self-reported treatment status for the disease from the cohort questionnaire.
Patient self-reports are considered a simple and effective method to measure adherence, since a gold standard does not exist [1].
Patients with cognitive impairments that can affect adherence were excluded.
Defining rigorous phenotype criteria is paramount to the success of a case-control GWAS [17].
Thus, the method of classifying individuals as either case or control in this study was validated by comparing the general characteristics of the case-control subjects.
The differences in demographic factors (age and gender) and disease factors for adherence between the two groups were consistent with previous reports (Table 1) [3].
Nevertheless, the classification criteria in this study have limitations.
First, there is the possibility of misclassification of treatment-adherent individuals as medication non-adherent patients, because the criteria were based exclusively on current medication status for hypertension or diabetes.
For example, patients with mild hypertension or diabetes who follow instructions from the physician to modify their diet or lifestyle as an initial therapy can be classified as medication non-adherent individuals.
Second, the different ratio of the number of hypertensive and diabetic patients between the case and control groups can affect the GWAS results [18].
Additionally, the risk of volunteer bias is present, in which individuals who participated in the KARE cohort possibly tended to adhere more to health care services.
To overcome these limitations, standardized phenotype criteria for self-reported adherence, such as the five-item Medication Adherence Report Scale-MARS-5, which has already been used in various settings [19], and the eight-item Morisky Medication Adherence Scale-MMAS-8, which has been validated in over 1,000 individuals with hypertension [20], can be applied in the study design.
These criteria enable reliable and replicable classification of medication adherence [17].
This study identified four loci that were suggestively associated with medication adherence.
The most significantly associated SNP was rs6978712 (p = 4.87 × 10-7), which is located nearby the GCC1 gene on chromosome 7.
Two suggestive SNPs were in strong linkage disequilibrium (r2 &gt; 0.8) with rs6978712 (Fig.3).
This might be due to the close physical proximity of those three SNPs.
GCC1 encodes a peripheral membrane protein that may have a role in the organization of the trans-Golgi network subcompartment that is associated with membrane transport [21].
The trans-Golgi network is a major secretory pathway that sorts and directs newly synthesized proteins to subcellular destinations, including neuronal synapses.
Increased transcription of GCC1 in the brain cortex of drug abusers has been reported in one study [15].
This study indicates that GCC1 is involved in decision-making capabilities in drug abusers.
Patients with chronic diseases weigh the benefits from adherence with treatment against the constraints on their daily lives and the perceived risks of treatment, such as side effects, time, money, and effort involved, either consciously or subconsciously [3].
Thus, in the aspect of decision-making in adherence behavior, the association between medication adherence and the three SNPs proximal to the GCC1 gene identified in this study seems worthy of further research.
The personality trait-associated SNPs were not associated with medication adherence in this study.
Because of the multivariate association between personality traits and medication adherence, it seems difficult to apply the personality trait-associated SNPs to the medication adherence-associated SNPs directly.
This study suggests potential evidence for a genetic basis of medication adherence in chronic disease patients.
A complete identification of the genetic influences for medication adherence appears feasible, because medication adherence is complexly influenced by not only personality traits but also many factors, including socio-economic factors, healthcare-related factors, and demographic factors [3].
Nevertheless, identifying genetic associations for poor medication adherence in chronic disease patients requires further research.
It would be helpful for physicians to initiate personalized medication therapy by modifying treatment strategies, such as changing the route of administration, the complexity of treatment, and the degree of behavioral change required, for chronic disease patients with expected poor medication adherence from their genetic background.
Summary of the sampling and quality control procedure.
KARE, Korean Association Resource project; HTN, hypertensive patients; DM, diabetes patients; SNP, single nucleotide polymorphism; GWA, genome-wide association; MAF, minor allelic frequency; HWE, Hardy-Weinberg equilibrium.
Quantile-quantile (QQ) plot and Manhattan plot of genome-wide association study (GWAS) for medication adherence.
(A) QQ plot shows the differences between the observed and expected -log10 (p-value) of the study results.
The red line indicates the expected distributions under the null hypothesis.
The gray shaded region represents the 95% confidence interval.
(B) Manhattan plot of single nucleotide polymorphisms for medication adherence.
X-axis represents chromosomes.
Y-axis represents -log10 (p-values) obtained by genome-wide association analysis.
The red line indicates the suggestive threshold for genome-wide significance (p &lt; 1 × 10-5).
Regional association plot of the most significant loci associated with medication adherence.
Purple circle indicates the strongest association (rs6978712) detected in the genome-wide analysis.
Other circles indicate SNPs genotyped in this study, and their color reflects the linkage disequilibrium (r2) with rs6978712.
The blue line represents the recombination rates in the 1000 Genome Mar 2012 ASN population.
SNPs, single nucleotide polymorphisms; ASN, super population of the east Asian, which includes Han Chinese in Beijing, China; Japanese in Tokyo, Japan; Southern Han Chinese; Chinese Dai in Xishuangbanna, China; and Kinh in Ho Chi Minh City, Vietnam.
Title: Characteristics of the study subjects

Title: SNPs with p &lt; 1 × 10-5 in the GWAS of 235 medication non-adherent cases and 1,067 medication adherent controls from the KARE cohort
Title: Epidemiological and Genome-Wide Association Study of Gastritis or Gastric Ulcer in Korean Populations

Gastritis is a major disease that has the potential to grow as gastric cancer.
Gastric cancer is a very common cancer, and it is related to a very high mortality rate in Korea.
This disease is known to have various reasons, including infection with Helicobacter pylori, dietary habits, tobacco, and alcohol.
The incidence rate of gastritis has reported to differ between age, population, and gender.
However, unlike other factors, there has been no analysis based on gender.
So, we examined the high risk factors of gastritis in each gender in the Korean population by focusing on sex.
We performed an analysis of 120 clinical characteristics and genome-wide association studies (GWAS) using 349,184 single-nucleotide polymorphisms from the results of Anseong and Ansan cohort study in the Korea Association Resource (KARE) project.
As the result, we could not prove a strong relation with these factors and gastritis or gastric ulcer in the GWAS.
However, we confirmed several already-known risk factors and also found some differences of clinical characteristics in each gender using logistic regression.
As a result of the logistic regression, a relation with hyperlipidemia, coronary artery disease, myocardial infarction, hyperlipidemia therapy, hypotensive or antihypotensive drug, diastolic blood pressure, and gastritis was seen in males; the results of this study suggest that vascular disease has a potential association with gastritis in males.
Gastric cancer is a common cancer type.
In the world, this cancer type has the second highest incidence among males and third highest among females [1].
In Korea, the incidence rate of gastric cancer was second highest, and the mortality rate of gastric cancer was third highest among all cancer types in a 2010 Korean cancer statistics study [2].
The incidence rate of gastric cancer differs between patient age, location, and even sex [3], because gastric cancer has various subtypes [3, 4] and a lot of risk factors [5].
Helicobacter pylori is known for having a relation with gastric cancer [6].
Almost all gastric cancer patients are infected with H. pylori [3].
As known before, dietary habits also have the potential to affect gastric cancer [1].
The incidence risk factors have differences in gastric cancer or peptic ulcers by blood type [7] and sex [8].
Gastric cancer type also depends on patient populations or race [5].
So, an analysis is needed to find known and unknown risk factors in diverse phenotypes [3].
Gastritis is related with gastric cancer.
Chronic gastric inflammation has the potential to grow into gastric cancer [9].
Many risk factors are related with gastritis, as already reported.
However, even though gender is known to be associated with gastritis infraction, there is not much information about the effects on gastritis depending on gender.
So, we examined Korean risk factors of gastritis and gastric ulcer using genotypes and clinical characteristics of patients who were diagnosed with gastritis or gastric ulcer in each gender.
For revealing the correlation with gastritis and known and unknown risk factors, we analyzed single-nucleotide polymorphisms (SNPs) and epidemiological data of the Korea Association Resource (KARE) project, which comprised the results of the Anseong and Ansan cohorts study.
Title: Clinical characteristics and study genotypes

This study analyzed cohort data that comprised the Anseong and Ansan population study in the KARE projects.
Anseong is a rural area, and Ansan is a city.
Both areas are in Gyeonggi-do.
Citizen of these two cities have different lifestyles, and they are exposed to different environment.
Detailed information of the KARE data was reported [10].
The KARE data included 8,842 individuals, 352,228 SNPs, and 277 phenotypes.
Among 8,842 total individuals, we divided patients and normal subjects for a control and case study using a positive diagnosis experience of gastritis.
There were 1,885 patients and 6,957 normal subjects.
First, for selecting the case, we eliminated 104 patients who were diagnosed under age 20 or had unknown age.
Then, 1,781 patients remained.
Of these, 804 patients were men, and 977 patients were women.
Of 6,957 normal subjects, no one was aged under 20; 3,335 were men, and 3,622 were women.
Among 352,228 SNPs, we excluded 3,044 SNPs based on the Hardy-Weinberg equilibrium test for quality control.
After frequency and genotyping pruning, 349,184 SNPs remained.
Among 277 total phenotypes, we filtered missing phenotypes and low genotyping rates.
Then, 120 clinical characteristics remained.
We also eliminated gastritis phenotype variables and unknown drug information variables; 101 clinical characteristics remained.
Title: Statistical analysis

For data filtering and finding significant SNPs, we used PLINK version 1.07, that is a tool made for analyzing whole-genome association using computational methods [11].
We used the default options of PLINK [11], and we analyzed phenotypes by logistic regression test for classifying patients and normal subjects and estimating factors.
We also assessed the result factors of the logistic regression by Student&#39;s t-test for revealing meaningful differences between patients and normal subjects using R version 3.0.2 for finding gastritis-associated factors.
Then, we used the receiver operating characteristic (ROC) curve and area under the curve (AUC) scores to confirm the prediction ability of the factors.
Title: Clinical characteristics

Tables 1 and 2 explain the results of the logistic regression test among total clinical characteristics in each gender.
There were differences in gender-specific clinical characteristics.
Among 1,781 total patients, 977 patients and 804 patients were male and female, respectively.
Patients had several disease-association factors in both genders: area, positive diagnosis of hyperlipidemia, positive diagnosis of mental disease, continual use of one or more drugs, waist, and height.
Through previous studies, how these factors are related with gastritis was verified.
The incidence of gastritis is affected by population, geographic variation, or lifestyle [1, 5].
In this study, we used cohort data that comprised country and city populations.
As shown Tables 1 and 2, in both genders, the population of local A had a higher gastritis incidence rate.
This means that differences in patient lifestyle and environment affect the disease incidence rate.
It is well known that stress has an influence on carcinogenesis.
A study reported that gastritis is closely connected with mental illness [12] and differs in degree according to drug use-taking medicine affects the stomach.
Nonsteroidal anti-inflammatory drugs can cause gastrointestinal damage [13].
Losing appetite is the one of phenotypes of gastritis that are connected with waist size or hip size as gastritisassociated factors.
However, definite evidence is lacking [14].
As shown in Tables 1 and 2, we can explain the associative relation that waist size or hip size is smaller despite patients being taller than normal.
Also, height is considered to be a gastritis risk factor.
As shown in a previous study, higher height tends to increase the gastric cancer incidence rate [15].
In males and females, hyperlipidemia and gastritis are related with gastritis.
Only male cases are associated with vascular disease.
Coronary artery has the potential to grow into myocardial infarction [16], and myocardial infarction is related with blood pressure [17].
As previously reported, approximately 30% of patients with coronary heart disease suffer from hyperlipidemia at the same time [18].
So, hyperlipidemia, coronary artery, and myocardial infarction are all related to each other.
As shown in Table 1, in males, by logistic regression, there is a relation with hyperlipidemia, coronary artery, myocardial infarction, hyperlipidemia therapy, hypotensive or antihypotensive drug use, and diastolic blood pressure.
This relation is more remarkable in males than in females.
Repeatedly, it can be considered that vascular disease is connected with gastritis.
In fact, in this study, including males and females, 16 patients had gastritis and myocardial infarction (20%), 77 patients had gastritis and hyperlipidemia (36%), and 13 patients had gastritis and coronary artery (25.5%).
Female patients with gastritis have 4 more gastritis-associated factors by logistic regression test compared with males.
In the case of taking osteoporosis medicine, the incidence rate is 1.7 times higher compared with normal.
Taking anticonvulsants is a meaningful factor by logistic regression test, but few patients were taking this medicine [4]; so, it is hard to prove that taking anticonvulsants has a positive relation with gastritis.
Patients who were diagnosed with tuberculosis and patients who have taken tuberculosis drugs suffer gastritis more than normal subjects.
This gives information on the relation between gastritis and tuberculosis in females.
ROC curves can show how gastritis-associated factors of Tables 1 and 2 affect gastritis patients and normal subjects.
Whether these factors are conclusive should be confirmed by AUC values.
Figs.1 and 2 are ROC curves using the result of the logistic regression test of Tables 1 and 2, respectively.
Each AUC value is 0.697 and 0.687.
This result means that these factors are not useful as gastric-specific markers.
Title: Genome-wide association studies

We selected the top 20 ranked SNPs by association test using chi-square test and p-values among 349,184 SNPs in males and in females, respectively.
The results of SNPs association analysis described in Tables 3 and 4.
Tables 3 and 4 are top 20 ranked SNPs of genomewide association analysis in males and females, respectively.
They are sorted by P-value.
But there was no considerable SNP associated with gastritis.
Astonishingly, there was no common SNP between males and females.
This means that associated-SNPs are different, depending on the patient&#39;s gender.
Table 5 describes the important SNPs associated with gastritis by logistic regression analysis among the top 20 ranked SNPs.
The p-values of these SNPs were all &lt;0.001.
Figs.3 and 4 show the ROC curve by using these SNPs in males and in females, respectively.
The AUC scores are 0.675 and 0.658 in males and in females, respectively.
The AUC score was too low to use as a specific factor for diagnosis.
In this study, we confirmed that vascular disease and gastritis have considerable association in Korean males.
We also verified that gastritis is affected by other various drugs.
However, we could not find gastritis-specific biomarkers for diagnosis.
Gastritis has numerous causes and is distributed as variable subtypes by phenotype.
So, correctly diagnosing it is very important for therapy.
Gastritis has non-specific phenotypes that can obscure finding the disease causes.
So, more information is also needed to analyze the complex factors associated with disease rather than each factor associated with the disease.
For example, confirmation of H. pylori infection, tobacco, smoking period, and alcohol intake are considered.
This needs further study.
Receiver operating characteristic (ROC) curve of gastritis-associated factors in males.
This shows how well the 22 gastritis-associated clinical characteristic factors (Table 1) classify patients and normal subjects in males.
As much as the ROC curve, these factors can classify gastritis patients and normal subjects in males.
Receiver operating characteristic (ROC) curve of gastritis-associated factors in females.
This shows how well the 26 gastritis-associated clinical characteristic factors (Table 2) classify patients and normal subjects in females.
As much as the ROC curve, these factors can classify gastritis patients and normal subjects in females.
Receiver operating characteristic (ROC) curve of gastritis-associated single-nucleotide polymorphisms (SNPs) in males.
This shows how well 20 ranked gastritis-associated SNPs (Table 3) classify patients and normal subjects in males.
As much as the ROC curve, these factors can classify gastritis patients and normal subjects in males.
Receiver operating characteristic (ROC) curve of gastritis-associated single-nucleotide polymorphisms (SNPs) in females.
This shows how well 20 ranked gastritis-associated SNPs (Table 4) classify patients and normal subjects in females.
As much as the ROC curve, these factors can classify gastritis patients and normal subjects in females.
Title: Clinical characteristics of gastritis variables in male

Title: Clinical characteristics of gastritis variables in female

Title: Top 20 ranked SNPs of genomewide association analysis in males

Title: Top 20 ranked SNPs of genomewide association analysis in females

Title: Gastritis-associated single-nucleotide polymorphisms
Title: Editor&#39;s Introduction to This Issue

Transposable elements (TEs) are genetic elements that have the potential to move from one site to another in the genome.
Therefore, TEs mediate diverse recombination events and eventually can contribute to genomic evolution.
In addition, TEs contribute to genomic instability and diverse diseases, including cancer.
However, the mechanisms and extent of disease contribution of TEs are not well known yet.
Regarding these areas, three review papers are reported in this issue.
Dr. Byung-Dong Kim of Seoul National University reviewed foldback intercoil (FBI) DNA and the mechanism of DNA transposition mediated by FBI.
In this review, he introduced the FBI DNA element and the mechanistic features that are relevant to the application of FBI DNA to the mechanisms of DNA transposition.
Dr. Nam-Soo Kim&#39;s group of Kangwon National University reviewed genome size variations in plants in relation to TEs.
Of all eukaryotes, TEs are especially prevalent in plants.
The extreme genomic size variation of plants has posited a conundrum in evolutionary genetics.
The authors introduced genomic size variations in relation to TEs using some examples, such as the Liliaceae family, known to have large genomes in angiosperms.
Dr. Heui-Soo Kim&#39;s group of Pusan National University reviewed the biological implications of TEs in genomic evolution, genetic instability, and diverse human diseases.
They gave an overview of the importance of TEs in studying genomic evolution and genetic instability and suggested that further in-depth studies on the mechanisms related to these phenomena will be useful for clinical applications.
There is another interesting review regarding epigenetic regulation in this issue.
Dr. Joomyeong Kim&#39;s group of Louisiana State University introduced the regulation and function of the Peg3 imprinted domain.
The authors summarized the information that we have learned so far regarding the Peg3 domain and discussed the similarities and differences between the Peg3 and other imprinted domains.
The other three articles are the Korean Association Resource (KARE) data genomic analysis award winners.
The analysis of KARE data from young scientists&#39; creative points of view has generated new interpretations and interesting evidence, which we should pay attention to in future studies.
Genomics &amp; Informatics is already available in PubMed Central and PubMed.
Recently, Genomics &amp; Informatics applied to be listed in SCOPUS.
Title: Foldback Intercoil DNA and the Mechanism of DNA Transposition

Foldback intercoil (FBI) DNA is formed by the folding back at one point of a non-helical parallel track of double-stranded DNA at as sharp as 180° and the intertwining of two double helixes within each other&#39;s major groove to form an intercoil with a diameter of 2.2 nm.
FBI DNA has been suggested to mediate intra-molecular homologous recombination of a deletion and inversion.
Inter-molecular homologous recombination, known as site-specific insertion, on the other hand, is mediated by the direct perpendicular approach of the FBI DNA tip, as the attP site, onto the target DNA, as the attB site.
Transposition of DNA transposons involves the pairing of terminal inverted repeats and 5-7-bp tandem target duplication.
FBI DNA configuration effectively explains simple as well as replicative transposition, along with the involvement of an enhancer element.
The majority of diverse retrotransposable elements that employ a target site duplication mechanism is also suggested to follow the FBI DNA-mediated perpendicular insertion of the paired intercoil ends by non-homologous end-joining, together with gap filling.
A genome-wide perspective of transposable elements in light of FBI DNA is discussed.
The transposable element (TE) was first described by McClintock [1] as a controlling element that jumps from one position to another in the maize chromosome in the mid-1940s, long before the discovery by Watson and Crick of the double helix structure of DNA as a genetic element [2].
It was a time when, after the rediscovery of Mendel&#39;s Law of Heredity in 1900, the cytogenetic study of chromosomes was at the forefront of genetics and when genes were thought to be beads on a string located on the chromosome.
It was in the late 1970s and early 1980s when insertion sequence (IS) elements and transposons (Tn) were discovered and found to be similar to McClintock&#39;s controlling elements, and her AC/DS elements were confirmed at the DNA sequence level as a TE [3].
Salient features of DNA transposable elements (dTEs), including terminal inverted repeats (TIRs), target site duplication (TSD), the transposase gene, and simple and replicative transposition mechanisms have been well established by extensive molecular biology and biochemistry studies [4].
Then, retrotransposable elements (rTEs), including long terminal repeat (LTR)-retrotransposons, non-LTR-retrotransposons, and other retro-elements, were being added to the repertoire since the 1980s to make the picture diverse and complicated [5].
Many new families of TEs are being added, especially from eukaryotes, even by computational screening in the post-genome era, which has necessitated a new classification based on their structures and mechanisms of transposition [6].
The presence of foldback intercoil (FBI) DNA was first reported by Kim in 1985 [7] and shown by a space filling model to mediate intra-molecular homologous recombination of inversions and deletions.
It was further shown in 1987 that FBI DNA can mediate inter-molecular DNA rearrangements, such as site-specific insertions, at the foldback tip and DNA transpositional integration at the intercoil end of the detached dTE [8].
It will be examined in this review how FBI DNA mediation of transposition can be extended to different classes and families of rTEs.
Readers are suggested to refer to many good reviews available on the classification, structures, genetic contents, and function of the gene products and on the mechanisms of transposition with excellent diagrammatic illustrations [5, 9, 10, 11, 12].
This review will minimize duplicate descriptions and focus on the mechanistic features that are relevant to the application of FBI DNA to the mechanisms of DNA transposition.
To build the basis for comparison and understand transposition mechanisms, it is necessary to consider TE structures based on the presence or absence of the following factors: 1) RNA intermediates before transposition, 2) TIRs or LTRs, 3) transposase or reverse transcriptase (RT) and other proteins related to autonomous or non-autonomous transposition, 4) TSD, and 5) other features, like DNA replication modes, hairpin structure, and enhancers.
Transposition of dTEs involves direct movement of the dTE intermediate either by a simple cut-and-paste mechanism, as in Tn5 [13], Tn7 [14], and Tn10 [15], or by replicative transposition, in which case one copy stays in the original site and a daughter dTE copy appears in a distant location [16, 17, 18].
Step-wise description of the transposition events includes the requirement and pairing of both dTE ends [13, 19], binding of the transposase proteins to the terminal sequences, double-strand breaks (DSBs) of the dTE terminals [14] and target ends, ligation of the 3&#39;-OH donor end and the 5&#39;-P target end, and gap repair of the single-strand gaps on the target that were generated by DSB [20].
In the case of the Mu bacteriophage, the gap can be repaired while the entire transposon gets replicated in a cointegrate mechanism [21, 22].
Transposition of rTEs involves an RNA intermediate that is reverse-transcribed by RT into a cDNA.
This double-stranded linear DNA functions as the direct precursor for integration by an integrase protein into a new location in the host chromosome in a &#39;copy and paste&#39; manner, as in the &#39;cut-and-paste&#39; transposition reaction in dTEs [20, 23].
TIRs are a hallmark of dTEs, as LTRs have been that of rTEs until they were joined by non-LTR retrotransposons, which have a 5&#39;-untranslated region (5&#39; UTR) and 3&#39; UTR, as typified by LINE-1 (L1), or which has neither of the terminal elements, as typified by the Alu element.
It becomes clear that the presence of terminal repeats in the TEs is no longer a common requirement for the transposition reaction.
The description of the protein factors above will suffice for this review, except for the long interspersed nuclear element (LINE) open reading frames (ORFs).
The retrotransposition and integration of LINE have been viewed as a coupled process, called target-primed reverse-transcription [5, 24].
However, this model has recently been enhanced by the finding that L1 reverse-transcription does not require base pairing between the primer and template [25].
Both the RT and endonuclease domains in L1 are encoded by the same ORF.
Therefore, the cDNA that was reverse-transcribed from the mRNA expressed from the genomic copy of LINE is inserted into the host genome [9].
TSDs are generated in almost all classes and families of TEs, except for the Helitron dTE superfamily [6, 9, 10].
The size of the TSD varies from 2 to 11, but the sequences are not conserved.
The size is unique to superfamilies and can be used as a diagnostic feature.
The ubiquitous presence of TSDs, which are generated by gap fillings (GFs) after TE integration on the target [26], reflects the unity in transposition reactions, regardless of dTE or rTE, and the presence or absence of terminal repeats [22].
Yet, the meaning of the small oligomer size of the duplicated target is seldom discussed in terms of DNA structure.
This aspect will be dealt with below with the FBI DNA mechanism.
Helitron, a eukaryotic dTE, is considered, together with Polintons, a third class of TEs for many unique aspects [9].
Helitron does not have TIRs but has 5&#39;-TC and CCTR-3&#39; termini instead.
They contain 16-20-bp hairpins 10-12 nucleotides inside of the 3&#39;-end and transpose precisely between the 5&#39;-A and T-3&#39; with no TSD generation.
It is known to adopt replicative rolling circle transposition [27].
Helitrons indiscriminately capture and mobilize gene sequences and may impact hybrid vigor or heterosis in maize [28].
The hairpin is presumed to serve as the terminator of rolling circle replication [9].
Since the hairpin sequence corresponds to the FBI DNA motif, it will be interesting to consider whether it can not only serve as a terminator or rolling circle replication but also serve as a replication origin for its own replication and for frequent gene capture activities, which the Helitron is capable of.
Polinton was discovered and characterized based on bioinformatics studies [27].
Polinton is 15-20 kb long, with a 6-bp TSD and 100-1,000-bp TIR at both ends.
Polintons code up to 10 proteins, including a family of B DNA polymerases (POLBs), a retroviral-like integrase, an A transposase, and an adenoviral-like cysteine protease.
Polintons, however, with their structural characteristics and presence of TSD, should follow non-homologous end-joining (NHEJ) with the GF transposition mechanism, as ordinary dTEs.
What makes it unique is the presence of a self-encoded POLB and a short 1-3-bp terminal tandem repeat that supports a protein-primed self-synthesis mechanism for Polinton propagation.
This mechanism calls for an extrachromosomal single-stranded Polinton intermediate that forms a racketlike structure and follows POLB-mediated replication to become double-stranded before being integrated into the host genome [9].
It has been proposed that eukaryotic dTEs follow three types of transposition mechanisms according to different replication modes: cut-and-paste for most dTEs, rolling circle replication for Helitrons, and synthetic replication for Polintons [9].
In fact, there seems to be many different cases of transposon-related replication that need to be clarified in the future.
As mentioned earlier, continuation of DNA replication of an inserted TE molecule, like in Mu co-integration, may also fill the gap to generate TSDs.
It should be emphasized that DNA transposition has been grouped in two: simple, conservative, or &#39;cut-and-paste&#39; mechanism and replicative transposition [16, 17, 29].
In Tn5, at the outer ends, ori-C-like 9-bp sequences are found, which may serve to direct host replication functions to the ends of the element during transposition [17].
Transposition of Tn5 may involve replication proceeding from both ends in a symmetrical manner.
This type of mechanism may differ from the polarized replication of other elements [17].
A synthetic mechanism is proposed for Polinton propagation to explain protein-primed replication of a racket-like structure of a single-stranded Polinton [9].
If it gets to be understood that the heteroduplex structure that is derived from FBI DNA may serve as the template of DNA replication, many of the diverse cases of DNA replication may be clarified (see below).
FBI DNA is formed by the folding back at one point of a non-helical parallel track of double-stranded DNA at as sharp as 180° and the intertwining of the bent helixes within each other&#39;s major groove to form an intercoil with a diameter of 2.2 nm (Fig.1A).
FBI DNA could be considered a double-stranded version of a hairpin or stem-and-loop structure of RNA and single-stranded DNA.
But, such sharp bending and formation of a four-stranded intercoil are not considered possible with a double helical B-DNA with a diameter of 2.0 nm [8].
In an extended definition, FBI DNA may contain a loop and a much longer stem under biological conditions.
Intercoil DNA is designated to distinguish itself from supercoil DNA, which is thicker than 2.2 nm.
Intercoil DNA is also termed to avoid the use of four-stranded DNA, which has the connotation of four-stranded base pairing among homologous repeats.
It needs to be emphasized that intercoil DNA is formed by two intertwining duplexes of any sequence, but when homology is met at a certain region, homologous recombination may proceed via heteroduplex formation with the help of the proper enzymes involved.
FBI DNA was first formulated based upon a rare but unique electron microscopy (EM) configuration of plant mitochondrial DNA (mtDNA) molecules and space-filling models [7, 8].
Once the FBI DNA structure was realized with the space filling model, it was possible to perform homologous recombination with direct contact of the repeat sequences.
The intra-molecular homologous recombination of a deletion and inversion occurs via 1) synapsis of the repeats in the intercoil, 2) heteroduplex formation by 90° base flipping, and 3) nick-and-close resolution of the sugar phosphate backbone crossing over [7].
Inter-molecular homologous recombination, known as site-specific insertion, on the other hand, is mediated by direct contact between the attP site at the tip of the FBI DNA and the attB site on the target DNA [8].
Perpendicular approach of the foldback tip onto the target site effectively resolves the year-long topological puzzle encountered in four-stranded base pair winding of two circular DNA molecules and explains that 6- or 7-bp attP and attB sites correspond to the diameter of the approaching FBI DNA tip and the target width.
DNA transposition is explainable by the perpendicular approach of the detached transposon ends in intercoil form to the host target, as described in detail below.
Transposition of a dTE molecule in FBI DNA configuration is tested by a space filling model, as presented in Fig.1 and as a detailed diagram in Fig.2 [30].
The TE molecule, detached from the host by a DSB in a blunt end cut, approaches, as shown in the diagram, at a perpendicular angle to the target site.
Both 3&#39;-OH ends of the TE ligate with the 5&#39;-P ends of the staggered-cut target, leaving the overhang target ends single-stranded, which would be repaired by gap filling.
As shown in the diagram (Fig.2), when the TE ends approach the target on the minor groove side, the target size would be 5 bp on the minor groove side and 7 bp on the major groove side.
This explains the meaning of the average size of the 5-7-bp target duplication, as it matches the diameter of the incoming intercoil DNA, 2.2 nm.
In reality, the target sizes vary between 2 to even over 10 bp in a TE superfamily-specific manner.
This TSD size variation could be explained by several factors-namely, local squeeze and stretch of the DNA double helix; unequal cutting positions of the TE ends, as exemplified in Tn7 by the 3-nt overhang by staggered DSB [14]; the characteristics of the endonucleases involved; and even temporary loosening of the intercoil end during a very elaborate and concerted coordination of the transpososome structure [8].
In earlier days, when IS and Tn, and even the LTR retrotransposons, were the major players of transposition, it was the presence of terminal repeats that attracted attention for explaining the transpositional mechanism, as in homologous recombination.
Therefore, it was somewhat puzzling to explain the cases of non-LTR retrotransposons with asymmetric termini sequences.
But, it is now TSD that attracts attention as a common factor for the transposition mechanism of all TEs, except for Helitrons.
It is the structural nature of the intercoil, which is formed by intertwining duplexes, regardless of the homology between the two termini.
When both ends of a TE synapse in the form of an intercoil, are cut by a DSB off the host duplex strand, and approach the target in perpendicular direction, their 3&#39;-OH ends ligate to the staggered 5&#39;-P ends of the broken target.
The resulting overhang single-strand DNA gaps on both flanking ends of the target are repaired to generate the accompanying TSD.
The significance of the terminal repeats is then the homologous recombination that is required for further transactions of recombination, replication, and transcription.
The enhancer is a cis element of DNA that affects, as a third factor, DNA transactions, such as transcription, homologous recombination, and transposition [31, 32, 33, 34].
It has remained somewhat of an enigma, because the presence of an enhancer element and its binding proteins at a distance in a sequence and orientation non-specific manner enhances the efficiency of the reaction by 1,000 times.
Rather unique configurations of rigid rod-like DNA with a big protein complex at one end [35, 36 Fig.4-2] may shed light on the enhancer concept and give us an idea of what actually happens under cellular conditions (Fig.3).
Folding back of a DNA duplex at a distant point would bring an enhancer site into juxtaposition with the base of a TE, which has also branched out as FBI DNA (Figs.4 and 5).
Proteins binding to the enhancer site and to both ends of the TE and any other additional factors together would form a tight DNA-protein complex, called a transpososome.
This transpososome would provide a tight and stable surgical platform for a series of highly concerted precision cut-and- paste reactions: namely, DSB of the TE termini, sealing of the host duplexes by NHEJ, staggered cuts on the target site, NHEJ of the TE 3&#39;-OH ends and target 5&#39;-P ends, and finally, gap-filling of the 5-7-nucleotide single-strand portions to flank the transposed TE.
In replicative transposition (Fig.5), there are two possibilities [21].
First, the gap-filling reaction during TSD may continue on to the TE molecule by displacement or polarized replication, even though the direction of replication may be reversible [20].
Second, as shown in the diagram, TIRs in the FBI DNA configuration may transform into heteroduplexes and serve as primers for bidirectional replication of the internal TE, thus keeping one copy at the original site and depositing the other copy to the target site in juxtaposition.
Inversion of the replicated TE may result from homologous recombination, as predicted [7, 16].
TSD serves as a common landmark of unity on transposition mechanisms for both dTEs and rTEs.
The only exception so far is the Helitron superfamily, which lacks TSD and employs a rolling circle replication model.
As long as a TSD is generated during TE integration into the host chromosome, it indicates that the immediate intermediate of integration is in a double-stranded DNA form, regardless of RNA or single-stranded DNA being the initial intermediate.
From the EM pictures of FBI and rigid rod configurations of native DNA (Fig.3), an ideal configuration of a transpososome may be envisioned.
It becomes apparent that vertical insertion of the DSB ends of the TE in an intercoil alignment is probably the best-fitting mechanism of DNA transposition.
Small sizes of the target, around 6-7 bp, encountered in TSD as well as in attP and attB in site-specific insertion all point to the fact that they correspond to the diameter of FBI DNA 2.2 nm.
The transpososome, made of FBI DNA configurations of the TE and the enhancer element, may effectively abolish lingering mechanistic puzzles about keeping four broken strands of the TE and the host from flying apart and answer how a TE molecule jumps onto a new target site at a distance.
Replicative transposition is explained likewise, even utilizing the terminal homology as a possible origin of replication.
It would be, however, difficult, if not impossible, to reproduce such a transpososome complex in vitro by FBI DNA configuration, which is critically needed for EM observations and crystallography analysis.
There are four main modes of transactions by which the FBI DNA motif may be utilized in DNA functions in the cell: alpha deletion, omega inversion, needle point site-specific insertion, and NHEJ-GF integration of TE [8].
With such an abundant presence in strategically important positions in the genome and very dynamic functional roles, it is hoped that FBI DNA receives the proper recognition and interpretation for a better understanding of the life process.
(A-C) Space filling model of foldback intercoil DNA for DNA transposition (Adopted from Kim.
KAST Rev Mod Sci Technol 2007;3:65-79 [8], with permission of Korean Academy of Science and Technology).
Diagrammatic illustration of a perpendicular insertion of a transposon in the foldback intercoil DNA configuration (DNA double helix modified from Klug.
J Mol Biol 2004;335:3-26 [30], with permission of Elsevier Ltd.).
Transmission electron microscopy pictures of rigid rod-like DNA that includes unique foldback intercoil DNA configurations (Adopted from Kim.
KAST Rev Mod Sci Technol 2007;3:65-79 [8], with permission of Korean Academy of Science and Technology).
Diagrammatic illustration of a simple conservative or cut-and-paste transposition in terms of foldback intercoil DNA and an enhancer concept (Adopted from Kim.
Foldback Intercoil DNA 2008 [36], with permission of Free Academy Press).
Diagrammatic illustration of a replicative transposition in terms of foldback intercoil DNA and an enhancer concept (Adopted from Kim.
Foldback Intercoil DNA 2008 [36], with permission of Free Academy Press).
Title: Transposable Elements and Genome Size Variations in Plants

Although the number of protein-coding genes is not highly variable between plant taxa, the DNA content in their genomes is highly variable, by as much as 2,056-fold from a 1C amount of 0.0648 pg to 132.5 pg.
The mean 1C-value in plants is 2.4 pg, and genome size expansion/contraction is lineage-specific in plant taxonomy.
Transposable element fractions in plant genomes are also variable, as low as ~3% in small genomes and as high as ~85% in large genomes, indicating that genome size is a linear function of transposable element content.
Of the 2 classes of transposable elements, the dynamics of class 1 long terminal repeat (LTR) retrotransposons is a major contributor to the 1C value differences among plants.
The activity of LTR retrotransposons is under the control of epigenetic suppressing mechanisms.
Also, genome-purging mechanisms have been adopted to counter-balance the genome size amplification.
With a wealth of information on whole-genome sequences in plant genomes, it was revealed that several genome-purging mechanisms have been employed, depending on plant taxa.
Two genera, Lilium and Fritillaria, are known to have large genomes in angiosperms.
There were twice times of concerted genome size evolutions in the family Liliaceae during the divergence of the current genera in Liliaceae.
In addition to the LTR retrotransposons, non-LTR retrotransposons and satellite DNAs contributed to the huge genomes in the two genera by possible failure of genome counter-balancing mechanisms.
Although some plants exist in aquatic environments, plants are mainly denizens of land.
Since the first brave aquatic algae landed on plants in the Silurian period about 450 million years ago, plants had spread from the tundra to tropic and aquatic lake or ocean to alpines.
The physical mass and size are also enormously variable, from only a few millimeters long in bryophytes to over 90 meters tall in eucalyptus.
The sessile nature and lack of obvious immune systems of green plants may have adopted different life strategies compared to species in the animal kingdom.
The genome, the genetic material of an organism, has been adopted for ambient environments.
As seen in the wide range of ecological niches and structural complexity, the cellular genome is also highly variable in plant species by as much as over 2,000-fold [1, 2].
Cellular DNA content has been measured in over 6,000 plant species [1, 2], and highquality DNA sequence information is available from about 50 plant species [3].
Prior to the genomic era, the general idea was that plant genomes constitute a large amount of &#39;junk&#39; or &#39;parasitic&#39; or &#39;selfish&#39; DNAs [4, 5], which are mostly repetitive DNAs and transposable elements.
However, these DNAs have been revisited with a plethora of genomic information and considered genomic gold in the evolution of eukaryotic taxa [6, 7, 8].
The extreme genome size variation of eukaryotes, especially in plant taxa, has posited a conundrum in evolutionary genetics.
The recently updated information of whole-genome sequences is shedding light from diverse angles to understand genome size variations with evolutionary and biological significance in many organisms.
This review excerpts topics on genome size variation in relation with transposable elements.
In the last session, the transposable elements and genome evolution in the Liliaceae family, known to have large genomes in angiosperms, will be discussed.
The redundant nature of eukaryotic genomes was analyzed well before the dawn of modern genomic analysis.
Waring and Britten [9] demonstrated that not all DNA coded for genes in eukaryotes by reassociation kinetic analysis of mouse DNA.
The DNA reassociation kinetic analysis, Cot analysis, allows one to dissect the genomic composition in several species, in which the genomes of complex organisms can be fractionated into several classes depending on the repetitiveness, such as unique, slightly repetitive, moderately repetitive, and highly repetitive sequences [10].
The Cot analysis was very useful in the analysis of the genome complexity in species with large genomes, such as major crop plants [11, 12, 13].
When the eukaryotic chromosomes are dissected at the molecular level, sequence composition is not even along the chromosomes.
While the low-copy protein coding sequences are clustered in discrete blocks between various repetitive DNA sequences, the repeated DNA sequences are located at a few defined sites or widely dispersed, depending on the classes of repetitive DNA [14, 15].
The chromosomal landmarks of repeated sequences include centromeres, telomeres, nucleolus organizing region (NOR), and heterochromatin.
While centromeres house highly repeated satellite DNA sequences and Ty3/gypsy long terminal repeat (LTR) retrotransposons, Ty1/copia LTR retrotransposons and microsatellites or simple sequence repeats are dispersed and present in clusters [15, 16].
The 18S-5.8S-25S rRNA gene repeats are on the NORs, which appear as secondary constrictions in cytological preparations [14, 15].
Another repeat of the 5S rRNA gene is also clustered in one or more sites in chromosomes [14, 15, 16].
The chromosomal tips, telomeres, also contain many thousands of the telomere repeat TTAGGG [14, 17].
Besides these repeat motifs, many other types of intercalary tandem repeats or dispersed repeats are also scattered along the chromosomes.
Eukaryotic nuclear DNA is packaged into chromatin structure.
During interphase, the local structure of chromatin depends on the genes present in the DNA.
The low-copy protein-coding genes are loosely packaged into euchromatin, whereas sequences of protein-noncoding genes are tightly packaged into the gene desert heterochromatin.
The local chromatin structure can be altered by epigenetic chemical modification of the DNA and chromatin-associated proteins.
The chromatin structure dynamic is a critical process in the regulation of DNA replication, gene expression control, and cell division [18].
Some points of the epigenetic control of gene expression and DNA replication are beyond the scope of this review; so, epigenetic chromatin modifications will be dealt here in detail only in relation with genome size and transposable elements.
Since the discovery of transposable elements in maize in 1940s by McClintock [19, 20, 21, 22], TEs have been found to be ubiquitously present in eukaryotes [23].
TEs are capable of moving their locations within the genome so that they can generate genomic plasticity by inducing various chromosomal mutations and allelic diversity [24, 25, 26].
TEs are conventionally classified into two classes based on the transposition mode: class 1 TEs and class 2 TEs [27].
Class 1 TEs are the retrotransposons that retrotranspose via an mRNA intermediate in a &#34;copy-and-paste&#34; process that can lead to extremely high copy numbers in the genome.
Class 2 TEs comprise transposons that transpose directly via DNA by a &#34;cut-and-paste&#34; process that allows them to attain moderate copies in the genome.
With the plethora of information of whole-genome sequences in a growing number of species [3, 28], more and more families of TEs are being mined, which necessitates the adoption of a unified classification system for TEs.
This systematic classification groups the TEs into 2 major classes, 9 orders, and 29 superfamilies by the mode of chromosomal integration, gene structure, and sequence similarity [23].
Class 1 TEs can be subdivided into several subclasses but generally into two subclasses: LTR retrotransposons and non-LTR retrotransposons.
Although both LTR and non-LTR retrotransposons are in plant genomes, LTR retrotransposons are more prevalent than non-LTR retrotransposons in plant species.
LTRs range from a few hundred base pairs to several kb (kilobases), and LTRs start with nucleotides of 5&#39;-TG-3&#39; and end with 5&#39;-CA-3&#39;.
Because the LTR retrotransposons integrate into 4- to 6-bp staggered cut sites, they produce 4- to 6-bp target site duplication (TSD) upon integration.
The LTR retrotransposons encode two open reading frames, GAG and POL [29, 30, 31].
GAG encodes a protein for replication, and POL encodes protease (PR), integrase (INT), reverse transcriptase (RT), and RNase H (RH).
The LTR retrotransposons are divided into two superfamilies, Ty1-copia (Pseudovirideae) and Ty3-gypsy (Metavirideae), based on the order of genes they encode [23].
The overall sequences are similar, but the gene order is PR-INT-RT-RH in Ty1-copia and PR-RT-INT-RH in Ty3-gypsy retrotransposons, respectively.
Phylogenetic analyses of the RT domain revealed a monophyletic origin of the Ty1-copia and Ty3-gypsy subfamilies [32, 33].
Moreover, the high sequence similarity and gene structure between Ty3-gypsy elements and retroviruses indicate that retroviruses evolved from Ty3-gypsy elements by acquisition of the env gene [31, 34].
Although non-LTR retrotransposons are less prevalent than LTR retrotransposons in plant taxa, four main orders of class 1 non-LTR retrotransposons, Dictyostelium intermediate repeat sequence, Penelope-like element), long interspersed nuclear elements and short interspersed nuclear element), are found in plants [23, 31].
DIRSs carry unusual split direct repeats or an inverted repeat at both ends and do not produce TSD upon integration.
Termini of PLE elements are LTR-like, but their orientation can be either a direct or an inverted orientation.
They have an RT and endonuclease domain, and the RT domain of PLE is more similar to telomerase than the RT of LTR retrotransposons.
Both long interspersed nuclear elements (LINEs) and short interspersed nuclear elements (SINEs) lack LTRs at their termini.
While LINEs can reach several kb, SINEs are short (80-500 bp).
LINEs are predominant elements in animal genomes, whereas they are not prevalent in plant genomes except the Del-2 element, which is highly abundant in the genomes of species in the genus Lilium.
The Del-2 elements will be discussed in more detail in the following session.
Of the five major superfamilies (R2, L1, RTE, I, and Jokey) in the LINE order, only L1 and I are present in plant genomes [23, 31].
SINEs are also highly abundant in animal genomes, especially in mammalian genomes; they are relatively less dominant in plant genomes.
The best known SINE is the Alu element, which is present in up to 500,000 copies in the human genome [35].
The origin of SINEs is obscure and is presumed to have originated from various polymerase III transcripts.
Their AT-rich 3&#39;-region carries 35-bp tandem repeats or contains a poly(T) tail, the pol III termination signal [35].
Class II DNA transposons can be subdivided into subclass 1 of terminal inverted repeat (TIR) transposons and subclass 2 of non-TIR transposons [23, 31].
The subclass 1 TIR DNA transposons are flanked by various lengths of TIRs and produce various lengths of small TSDs upon integration.
The TIR transposons encode a transposase, which excises the element from the genome and integrates into a new site in the genome, because it has both endonuclease and ligase activity [36, 37, 38].
The TIRs form a secondary structure where transposases bind.
Most of the known transposases carry a DDE catalytic motif, which comes in close proximity to each other and forms a catalytic center to enable the enzyme to excise the transposon DNA from the original site and reinsert it into the target site [39].
The subclass 1 TIR DNA transposons are classified into 10 superfamilies based on the sequence similarities.
Of the 10 superfamilies, plant genomes harbor 6 superfamilies: Tc-Mariner, hAT, Mutator, P, PIF-Harbinger, and CACTA.
In the subclass 2 non-TIR DNA transposons, two non-TIR DNA transposon superfamilies, Helitrons and Maverick, are present.
Helitrons are widespread, diverse, and abundant in some plants, but their identification is difficult due to their lack of a terminal repeat structure [40].
The Helitrons replicate via a rolling-circle mechanism and do not produce TSDs upon integration.
Helitrons can attain very high copy numbers and create genomic diversity.
Indeed, they are present in high copies, and captured gene fragments of pseudogenes contribute to maize genome diversity [41, 42, 43].
Depending on the movability both class 1 and class 2 TEs can be further classified into autonomous and non-autonomous elements.
While the autonomous elements equip all the machinery in mobilizing themselves, the non-autonomous elements lack one or all of the components of this machinery [44].
The non-autonomous elements are decayed autonomous elements by deletion or other mutations, but they are highly abundant in plant genomes [23].
Transposition of class 1 elements is a multi-step process.
That is, they must be transcribed and translated to produce proteins to reverse-transcribe the mRNA and package it into virus-like particles.
The non-autonomous class 1 retrotransposons lack self-encoded proteins, so that the multi-step is blocked.
However, they can be mobilized if the missing proteins are provided in trans [45].
The BARE1 and BARE2 elements illustrate an example of the autonomy and non-autonomy of the class 1 retrotransposons.
BARE1 (Barley RetroElement1) is a copia-like retrotransposon and highly abundant in the barley genome [46].
It is an autonomous element, and the protein machinery encoded is produced.
The BARE2 is a non-autonomous element, having most of its protein-coding domains intact, except for the defective Gag gene.
The defect of BARE2 was complemented by the BARE1 machinery to retrotranspose in the barley genome [47].
In class 2 DNA transposons, the non-autonomous elements cannot express transposase to catalyze their own transposition.
Because they were derived from autonomous elements, they form a binary system in terms of mobilization, which was recognized in the Ac/Ds system by the early pioneering observation in maize by McClintock [22].
miniature inverted repeat transposable elements) are worthwhile mentioning here among the non-autonomous class 1 transposons [48, 49, 50].
Because MITEs are small and carry only the minimal feature of TIR, they have been elusive in the identification of autonomous elements to mobilize them to attain very high copies in plant genomes.
By an extensive bioinformatics search, it was revealed that Stowaway MITEs were derived from the Tc1/mariner superfamily, and Tourist MITEs were derived from the PIF/Harbinger superfamily [51].
Thus, it was deduced that MITEs can burst by autonomous elements in these superfamilies.
C-value or genome size refers to the total amount of DNA in an unreplicated haploid or gametic nucleus of an organism [52].
C-value varies tremendously by as much as 660,000-fold among eukaryotes, from 2.3 Mb in Encephalitozoon intestinalis, a parasitic microsporidian, to 148,880 Mb in Paris japonica, a monocotyledonous plant in Liliales [53, 54, 55].
The general trend of C-value is that simple organisms have small genomes and complex higher organisms have large genomes.
However, the C-value variation is even present among closely related species, and genomic DNA content is often poorly correlated with organismal complexity, which projects the C-value paradox [56].
Plant taxa show more C-value variation than other taxa (Table 1) [3, 57].
Among land plants, the genome size variation is as much as a 2,348-fold difference from the smallest, Genlisea margaretae (1C = 63.4 Mbp), to the largest, Paris japonica (1C = 148,880 Mb).
Large size differences exist even in a single genus at the same ploidy level as in the genus Eleocharis, which comprises more than 250 species, in which E. acicularis (2n = 20, C = 0.25 pg) is 20 times smaller than E. palustris (2n = 16, C = 5.5 pg) [58].
Using Feulgen microdensitometry or flow cytometry, C-value has been measured in over 6,000 plant species and categorized in the Kew Plant DNA C-value Databases (http://data.kew.org/cvalues/) [1, 2].
The average genome size (1C) is 5.809 Gbp (1 pg = 980 Mbp) in angiosperms and 18.157 Gbp in gymnosperms, respectively.
The median 1C-value is 2.401 Gbp for angiosperms and 17.506 Gbp for gymnosperms, respectively; indicating that genome size in angiosperms is not evenly distributed but skewed, favoring a small size.
With the rapid development of high-throughput sequencing technologies, the list of complete genomes that have been sequenced is growing sharply to provide deep insights into genomic evolution by comparing whole-genome information [28, 59, 60].
In plants, genome sequences are available in more than 50 species, including 36 dicots, 16 monocots, 1 gymnosperm, and 1 each of a lycophyte and bryophyte [28].
The genome size in the list ranges from the smallest genome of the carnivorous Utricularia gibba (bladderwort) at 82 Mb [61] to the largest genome of a gymnosperm species, Picea abies (Norway spruce), at 19,800 Mbp [62].
The most frequent genome size of the sequenced plant genomes is about 500 Mbp, which is similar to the average genome size in the Kew Plant DNA C-value Database (http://data.kew.org/cvalues).
Repeated DNA sequences are predominant in the plant genome.
The repeat sequences range from 3% of the small genome of U. gibba [61] to 85% of the large genome of maize [63].
Transposable elements constitute most of the repeated DNAs.
Of the two classes of transposable elements, class 1 retrotransposons constitute significant proportions in plant genomes.
Of the first 50 plant genomes sequenced, the number of full-length LTR retrotransposons is available in 21 species, in which the genome size increment correlated with the abundance of repeated sequences (r2 = 0.584) and LTR retrotransposons (r2 = 0.68) [28].
The &#34;cut-and-paste&#34; class 2 DNA transposons multiply their copies when the transpositions occurs from a recently replicated genomic region into unreplicated genomic regions or when the excision site is repaired by gene conversion using the intact sister chromatid as a template [64].
Thus, the class 2 DNA transposons do not usually attain high copy numbers, and their impact on genome size is minimal.
Among class 2 elements, MITEs are an exception in copy number, because they are present at extremely high copy numbers in some genomes [65, 66].
However, the small size of MITEs, 200-500 bp, has a limitation on their impact on genomic obesity [67].
The semiconservative &#34;copy-and-paste&#34; transposition can increase copy numbers of the class 1 retrotransposons continuously.
Indeed, the magnitude of C-value variation in plant genomes cannot be explained solely by whole genome duplication (WGD), and genome contents are highly correlated with the amount of class 1 retrotransposons among the sequenced genomes [28, 67].
Unless there are mechanisms of removing these class 1 retrotransposons or other repeated sequences, plants may end up with a &#39;one-way ticket&#39; to genomic obesity [68].
Does a large genome actually cause any problem in the adaptation of host species?
In a sense, the dramatic increase of noncoding or repeated sequences must give some burden to the cell for replication and metabolism, which was postulated by the &#39;large genome constraint hypothesis&#39; [69].
That is, the nutritional requirements of the extra phosphorus (P), a major constituent of DNA, and subsequent longer replication times may result in longer generation times and a reduction in fecundity, leading to selective disadvantage [70].
Actually, plants having a 1C-value of less than 3 pg could be annual, biennial, or perennial, whereas plants having a 1C-value of larger than 25 pg cannot grow quickly enough to be annual and are to be obligate perennials [71].
Genome sizes of angiosperms are skewed, favoring a small size, except for a few extreme large genomes [1, 67].
The fact that lower unicellular organisms usually have relatively small genomes implies that genomic obesity by redundant sequences must have posed a negative impact on their survival.
Nevertheless, some plant genomes have survived with so many burdensome large fractions of repeated DNAs, implying that the negative effect of redundant DNAs may have been negated by some compensation.
Of the 50 sequenced plant genomes, the median C-value was about 500 Mbp.
The species with around 500 Mbp genomes (rice, sorghum, etc.)
have genomes that are about 30% transposable elements; so, it was argued that the average transposable element fraction is around 30% of the plant genomes [28].
Large genome information with high accuracy and effective bioinformatics tools have unveiled the biological roles of repeated DNAs, for which several excellent reviews are available on this aspect [6, 72, 73, 74].
The ever increasing class 1 retrotransposons can be counterbalanced by genome-purging mechanisms that include illegitimate or unequal recombination between LTRs and other types of deletions [75].
Intrastrand homologous recombination between directly repeated LTRs deletes the sequences between LTRs, leaving solo LTRs [76].
Analysis of solo LTRs and comparison of internally deleted retrotransposons showed that illegitimate intrastrand homologous recombination may be the driving force to maintain slim genomes in Arabidopsis [77] and rice [78].
Recently inserted TEs are often removed from the genome, leading to rapid genomic turnover.
For example, the rice genome experienced several bursts of LTR retrotransposons during the last 5 million years but ultimately removed over half of the inserted LTR retrotransposon DNA [79].
The cotton genus, Gossypium, provided another example genome expansion and contraction.
The diploid Gossypium species showed 3-fold differences in genome size, from 88 Mbp in the D-genome of New World cotton to 2,572 Mbp in the K-genome of Australian cotton [80].
Gorge3 is a gypsy-like retrotransposon in the Gossypium species.
The expansion and contraction of Gorge 3 were lineage-specific phylogenetically among Gossypium species.
The smaller genomes showed not only slow gain of Gorge3 compared to the larger genomes but also effective removal of Gorge3 elements, which exceeded the rate of the gain [76].
Non-homologous end joining (NHEJ) after double-strand break (DSB) can also lead to massive genomic restructuring by purging LTR retrotransposons in a small genome [81].
The genome of Oryza brachyantha is 60% smaller than its close relative cultivating species, Oryza sativa, in which 50% of the size difference was due to the amplification and deletion of recent LTR retrotransposons.
Of the 32,038 protein-coding genes of O. brachyantha, only 70% of them were in collinear positions in comparison with the rice genome.
So, the authors argued that the low LTR retrotransposon activity and massive internal deletions of the LTRs by NHEJ after DSB shaped the current O. brachyantha genome.
Similarly, NHEJ after DSB was found to be a common pathway in the genomic reduction of A. thaliana [82].
The genome of Norway spruce (Picea abies) sheds another interesting view on genomic evolution in gymnosperms.
It is the only genome sequenced in gymnosperms [62] and the largest genome among the first 50 sequenced plant genomes [28].
The sequence data showed no evidence of WGD since the divergence with angiosperms 350 million years ago.
Surprisingly, LTR retrotransposons of the Norway spruce genome are mostly low-copy LTRs with 80% singletons.
The large genome seems to have resulted from a slow but steady accumulation of diverse LTR retrotransposons, which was possible due to the lack of efficient elimination mechanisms, as evidenced by the fact that other pine species also have low-copy LTRs [80].
The 24-nucleotide small RNAs are involved in TE silencing [83], but expression of these small RNAs in Norway spruce is also low.
The average gymnosperm genomes are relatively larger than those of angiosperms.
But, the gymnosperms have not gone awry to obese genomes, as seen in some angiosperm species.
More gymnosperm genome sequences will give some answers for this modest modulation of LTR retrotransposon families and genomic expansion in gymnosperms.
Arabidopsis lyrata revealed several interesting phenomena on genomic contraction and expansion.
It is an out-crossing relative to the self-pollinating model species A. thaliana.
The A. lyrata genome is 1.6 times larger than that of A. thaliana [84].
Comparison of the two genomes revealed that there were large-scale genome rearrangements, but most differences in genome size were derived from hundreds of thousands of small deletions in A. thaliana, mostly in non-coding areas and retrotransposons [84].
A subsequent comparative study with other plant genomes showed that A. thaliana lost introns 6 times faster than A. lyrata [77].
In a genome-wide survey of synonymous substitution rates, A. thaliana had a higher mutation rate than A. lyrata, which might relate to the higher rate of intron loss and rapid genome reduction [85].
In the mutational hazard hypothesis [86], non-coding DNA is prone to accumulate deleterious mutations and be purged.
So, with the high mutation rate in A. thaliana and intron loss, the DNA loss process may still be ongoing in A. thaliana genomes, suggesting pervasive selection for smaller genomes [84].
Transposon insertions and copy number increases are potentially detrimental to the host genome.
As a consequence, eukaryotic genomes employ epigenetic surveillance systems that allow TEs to be under-controlled.
Ironically, however, Nina Fedoroff proposed a somewhat different view on plant genome size variations with transposable elements in her seminal AAAS presidential address: &#34;I argue that transposable elements accumulate in eukaryotic genomes because of, not despite, epigenetic silencing mechanisms&#34; [26].
Nevertheless, epigenetic regulations of the transposable elements are the first line of defense against uncontrolled transposable element proliferation [75].
Epigenetic mechanisms regulate gene expression and do not involve changes in DNA sequence.
They include DNA methylation [83, 87, 88], chromatin remodeling [89], and siRNAs [88, 89].
By genomic, epigenomic, and population genetic analyses with A. thaliana, Hollister and Gaut [83] presented three results: i) gene expression is negatively correlated with the density of methylated TEs, ii) methylated TEs near genes are under purifying selection, but unmethylated TEs or TEs far from genes are not under purifying selection, and iii) TE insertions are distributed by age and methylation status, so that older, methylated TEs are farther from genes.
Thus, host silencing of TEs near genes has deleterious effects on neighboring gene expression, resulting in the preferential loss of methylated TEs from gene-rich euchromatic regions.
The young LTRs near genes are targeted for DNA methylation by guidance of 24-nt small interfering RNA, which results in the silencing of LTRs as well as silencing of expression of nearby genes [83, 88].
The comparison of A. thaliana with its LTR-laden and larger relative A. lyrata can back up this view, because LTRs in A. thaliana are further from genes by more effective purging than those of A. lyrata [84, 88].
So, it is an evolutionary trade-off in which beneficial TE silencing casts a fitness cost by the deleterious effects on the expression of nearby genes [83].
In maize, the mop1 mutant has a deficiency of RNA-dependent RNA polymerase, which is a component of the RNA-directed DNA methylation pathway [89].
In a global gene expression analysis of the mop1 mutant, most of the class 2 DNA transposons (78%) showed increased expression, but 68% of the analyzed class 1 retrotransposons showed decreased expression [90].
Several histone deacetylases, enzymes involved in heterochromatin formation, increased their expression in the mop1 mutant, implying increased heterochromatin formation in the mutant.
Thus, there are several pathways that lead to TE silencing, and these pathways interact with each other, so that some families are rigorously suppressed but expression of some families is relaxed, which permits a rapid burst of some transposable families in a short period of time [84, 91].
The genomes of the species in Liliaceae are highly variable in 1C-value, ranging from 3.4 pg in Prosartes smithii to 127.4 pg in Fritillaria assyriaca [92].
Especially, the species in the subclasses Lilieae and Tulipeae in the Lilioideae subfamily harbor large genomes in the Liliaceae family.
By comparative analysis of the relationship of genome size with phylogenetic among 78 species in Liliaceae, Leitch et al.
[92] demonstrated that the ancestral genome size of the Liliaceae species was 6.73 pg, and punctuated genome size evolution occurred twice in Liliaceae.
The first genome size evolution was at the base of the polytomy divergence in Liliaceae, which was associated with a shift from the smaller genomes to the larger genomes of the Lilioideae (median C value, 14.2-50.9 pg) and Medeoloideae (median C value, 14.2-18.9 pg) subfamilies.
Table 2 shows the genomic features of two genera, Lilium and Fritillaria, in the Lilioideae subfamily.
The second genomic evolution was at the divergence of Lilioideae from Medeoloideae, which was about 12 million years ago [93].
Punctuated genomic evolution that led to speciation was also observed at the molecular level in other species [94].
The subfamily Medeoloideae is a sister subfamily to Lilioideae in the Liliaceae family.
While the species in Medeoloideae bear rhizomes, inconspicuous flowers, fleshy animal-dispersed fruits, and broad reticulate-veined leaves, the species in Lilioideae show bulbs, showy flowers, winddispersed seeds, and narrow parallel-veined leaves in open habitats [93, 95].
Because these ecological shifts and the adaptive morphological features coincided with the appearance of a large genome size, an adaptive role of genome size in Lilioideae is suggested-i.e., if the large genome has any selective advantage.
However, as argued by Leitch et al.
[92], there was no tendency toward increasing genome size from the ancestral species, having about 6.73 pg.
Their proposal was that the large genomes of Lilioideae were a necessary or random consequence, simply due to the large diversification at the base of the clade.
Oliver et al.
[96] proposed a theory that genomic size evolution is proportional to genome size, with the fastest rates occurring in the larger genomes, among 20 major eukaryotic clades.
So, the larger genomes of Lilium and Fritillaria in Lilioideae might have drifted away to even larger genomes with an increased rate of genome size evolution [92].
Although species in the genus Lilium have been known to carry large genomes, studies of transposable elements have not been carried out extensively.
The genome of the species in the genus Lilium spans from 32.8 to 47.9 pg, with a mean value of 39.6 pg [92].
A Ty3/gypsy retrotransposon, del-1, was isolated from Lilium henryi [86].
Del-1 Ty3/gypsy was present in approximately 13,000 copies in the L. henryi genome, which accounts for 1% of the genome of L. henryi.
By DNA reassociation analysis, Joseph et al.
[97] showed that Cot1 repeated DNAs constitute 2.2% of the L. henryi genome, whereas they constitute 9.7% in L. longiflorum.
In a Southern blot analysis with 14 Lilium species, del-1 Ty3/gypsy retrotransposons showed high copy variation, at least a 100-fold difference, among Lilium species, implying that the del-1 Ty3/gypsy element was highly active after divergence of the genus Lilium.
The integration time of del-1 Ty3/gypsy was estimated to be about 0.6 million years ago (mya) in L. henryi and 1.6 mya in L. longiflorum [16].
Del-2 is a LINE-like element found in Lilium speciosum [98, 99].
Del-2 is 4.5 kb in size and is present in 240,000 copies, constituting about 4% of the L. speciosum genome [99].
Del-2-related sequences were also highly abundant in Tulip and bluebells (Scilla nonsripta, both in Liliaceae), Hyacinthaceae, Amarylidaceae, and Alliaceae plants but less abundant in other monocots, such as Poaceae, implying that they are specific to the Liliaceae family.
LINEs are highly abundant in mammals, but they are scant in most plant species [23, 31].
However, the del-2 LINE-like elements are highly abundant in species of the Liliaceae family.
LIREs (lily retroelement) are Ty1/copia retrotransposons isolated from L. longiflorum [16].
LIREs showed approximately the same abundance as the del-1 Ty3/gypsy retrotransposon in species of the genus Lilium.
However, unlike the del2 LINEs, LIRE Ty1/copia retrotransposons are absent in other species, such as tulips, Narcissus, Ipheion, Hosta longgips, and O. sativa.
We mined eight Ty1/copia elements from Lilium lancifolium (unpulished).
Characterization of these retrotransposons may provide more information on the Ty1/copia retrotransposons in the genus Lilium.
The genus Fritillaria is a sister genus in the Lilieae subclass in the Lilioideae subfamily.
The Fritillaria species harbor an even larger genome (min.
30.8 pg, max.
89.2 pg, mean 50.9 pg) than Lilium species [92].
In F. affinis (1C = 45.6 pg) and F. imperialis (1C = 43.0 pg), chromovirus-like sequences and Tat lineage Ty3/gypsy retrotransposons are predominant in both species [100].
Not only do retrotransposons contribute to the gigantic Fritillaria genome, satellite DNAs also FriSAT1 repeats also contribute highly to the Fritillaria genome.
The FriSAT1 repeats constitute approximately 26% of the F. affinis genome [100].
With these results, they argued that there are no predominant repeats that track the increasing/decreasing trends of genome size in Fritillaria.
Instead, many diverse families of transposable elements and repeats constitute the gigantic Fritillaria genomes.
Thus, as evidenced with full-genome sequencing of the large genome of Norway spruce (P. abies) [80, 82], the highly obese Fritillaria genome may have had happened partly by the failure of the repeats to be purged effectively to counterbalance the amplification of retrotransposons.
Large fractions of eukaryotic genomes contain repeated DNA sequences that are mostly transposable elements.
The large quantity of accurate genomic information in various plant taxa and the development of computational biology have revolutionized our conventional view on transposable elements, which were once dishonorably christened &#39;selfish DNAs&#39; and have now been switched to &#39;genome treasure.&#39;
Genome size variation is enormously variable in plant taxa, and most of this variation is due to the variable amounts of class 1 LTR retrotransposons in the host genomes.
The &#39;copy-and-paste&#39; retrotransposition of the class 1 LTR retrotransposons left the plant genomes to get into genomic obesity.
The uncontrollable amplification of class 1 LTR retrotransposons could have caused a negative effect on host genomes not only by the various mutational effects but also by the large genomic constraint in replication.
Thus, epigenetic surveillance systems control the activity of retrotransposons and counterbalance genome size amplification by genome-purging mechanisms, such as illegitimate or unequal recombination between LTRs and NHEJ.
The epigenetic surveillance systems, however, might have been weakened in certain times, and some retrotransposons in certain lineages had escaped the epigenetic radar to have an amplification burst in a relatively short period of time during plant evolution and diversification.
However, this specific amplification might have not happened in all plant taxa, as seen in the Norway spruce, which has a relatively larger genome than average angiosperm genomes.
Although we need more gymnosperm sequence information, the steady accumulation of diverse LTR retrotransposons with weakly efficient elimination mechanisms can also account for the genome sizes in gymnosperms.
In addition to LTR retrotransposon amplification, as seen in Lilium and Fritillaria, non-LTR retrotransposons and simple satellite DNAs also contributed to the large genomes that happened partly by the failure of effective purging to counterbalance the growth of genome size.
Biological (hybridization, alien chromatin introgression, environmental stresses, or pathogens, etc) and nonbiological (temperature shift, UV irradiation, etc.)
stresses might have caused the weakening of epigenetic surveillance to allow genome bursts in certain species.
So, the current genomes may be the outcome of a long battle of repeat sequence amplification and genome-purging systems in ambient environments during plant evolution.
Title: Minimum, maximum, and mean 1C amounts in several major groups of plants and animals

Title: Genomic features of the species in the genera Lilium and Fritillaria
Title: The Impact of Transposable Elements in Genome Evolution and Genetic Instability and Their Implications in Various Diseases

Approximately 45% of the human genome is comprised of transposable elements (TEs).
Results from the Human Genome Project have emphasized the biological importance of TEs.
Many studies have revealed that TEs are not simply &#34;junk&#34; DNA, but rather, they play various roles in processes, including genome evolution, gene expression regulation, genetic instability, and cancer disposition.
The effects of TE insertion in the genome varies from negligible to disease conditions.
For the past two decades, many studies have shown that TEs are the causative factors of various genetic disorders and cancer.
TEs are a subject of interest worldwide, not only in terms of their clinical aspects but also in basic research, such as evolutionary tracking.
Although active TEs contribute to genetic instability and disease states, non-long terminal repeat transposons are well studied, and their roles in these processes have been confirmed.
In this review, we will give an overview of the importance of TEs in studying genome evolution and genetic instability, and we suggest that further in-depth studies on the mechanisms related to these phenomena will be useful for both evolutionary tracking and clinical diagnostics.
Transposable elements (TEs) are DNA sequences that are capable of integrating into the genome at a new site within the cell of its origin.
Sometimes, the change in their positions creates or reverses mutations, thereby altering the cell&#39;s genotype.
Barbara McClintock&#39;s discovery of these &#34;jumping genes&#34; earned her a Nobel Prize in 1983 [1].
TEs are prevalent in all plants and animals.
In mammals, TEs and their remnants make up almost half of the genome, and in some plants, they constitute up to 90% of the genome [2].
TEs consist of two major classes: DNA transposons and retrotransposons.
DNA transposons are capable of moving and inserting into new genomic sites [3].
Although they are currently not mobile in the human genome, they were active during early primate evolution until ~37 million years ago (Mya) [4].
Retrotransposons replicate by forming RNA intermediates, which are then reverse-transcribed to make DNA sequences and inserted into new genomic locations [4].
Based on the presence of long terminal repeats (LTRs), retrotransposons are further classified into two groups: LTR and non-LTR transposons.
In humans, LTR elements are called human endogenous retroviruses (HERVs).
It is estimated that HERVs inserted into the human genome &gt;25 Mya [5, 6].
Non-LTR retrotransposons include long interspersed element 1 (LINE-1 or L1), Alu, and SVA elements.
Studies have revealed that these are the only TEs that are currently active in humans [7, 8, 9, 10, 11].
TEs have driven genome evolution in multiple ways.
Retrotransposons comprise a large proportion of the genome, especially in plants and mammals.
The effect of the increase in retrotransposons has been tolerated during evolution.
Accumulating literature has proven that mobile elements are useful tools for studying genome evolution and gene function [12].
A comparative analysis has shown that the human genome makes 655 perfect full-length matches with vertebrate TEs.
TE insertions have been shown to have many effects, such as regulation of gene expression, increased recombination rate, and unequal crossover.
TE insertions have caused many effective changes in the human genome, and the selected changes have been responsible for the evolution of the human lineage [13].
The human genome contains many recently inserted active TEs, such as AluYa5, AluYb8, and AluYc1.
Alu elements are a family of primate-specific short interspersed DNA elements.
Various studies have proposed that Alu element insertions have created many variants that can potentially be used as DNA markers in human population studies, as well as in forensic analyses.
Kass et al.
[14] have identified an Alu-based polymorphism that consists of four alleles, from which the evolutionary order can be predicted.
The effects of TEs on genetic instability and human diseases have not been thoroughly studied.
The mobile property of TEs is the reason for their mutational potential.
TE insertions may create a broad range of effects on humans, ranging from silent mutations to alternative splicing.
Both insertions and excisions of TEs can cause genomic instability, thus causing many human diseases, including genetic disorders, psychiatric problems, and cancer [15, 16, 17, 18, 19, 20, 21].
Furthermore, TE insertions may result in insertional mutations, non-allelic homologous recombination (NAHR), creation of novel regulatory sequences, and epigenetic changes [22].
A large number of human diseases that are associated with NAHR between Alu elements have been reported [22].
Technical advances have helped to detect TE-associated diseases and develop novel biomarkers for clinical diagnostics.
Computational tools have been developed to study the dynamics of transposition at a population level, thus providing critical insights into the mechanisms behind genome evolution.
Finally, combining the genomic materials from diverse individuals followed by high-throughput sequencing can enhance the significance of characterizing genomic polymorphisms in a population [23].
In this review, we will provide updates on our current understanding of the roles of TEs in genome evolution and genetic instability.
Further, we focus on how their activity affects gene expression and causes disease states in human beings.
Brain evolution is an important process that accelerated the evolution of humans.
This occurred due to natural selection and genomic variation, a major source of which has been TE insertions.
TE insertions contributed markedly to variation and increased the speed of evolution [24, 25, 26].
Furthermore, they increase the recombination rate, in addition to affecting genes and their expression [24, 26].
From an evolutionary perspective, humans are unique in the speed of evolution and the number and activity of TEs.
It is proposed that the high frequency of TE insertions is responsible at least in part for the rapid evolution of humans.
It is speculated that some sets of genes might have been activated or suppressed due to variations caused by TE insertions, which in turn increased the chances for evolution [13].
There has been a prominent impact of retrotransposons on human evolution at the genomic level.
Retrotransposons have shaped human evolution at the RNA level through various mechanisms, such as modulation of gene expression, RNA editing, and epigenetic regulation [27].
Title: LTR retrotransposons

In humans, LTR retrotransposons are called HERVs (Fig.1), which constitute 5% of the genome.
The human genome shows 99% similarity with chimpanzees and bonobos.
Hence, the differences between these species are likely to be in regulatory sequences: promoters, enhancers, polyadenylation signals, and transcription factor (TF) binding sites.
The LTRs of HERVs help in regulating the expression of nearby genes.
The active human-specific LTRs that have been identified belong to the HERV-K family.
It is proposed that some of these endogenous retroviruses may have integrated into regulatory regions of the human genome and that they eventually contributed to human evolution [28].
Khodosevich et al.
[28] suggested that regulatory sequences found in retroviral LTRs may alter the expression of (or even inactivate) adjacent genes.
On the other hand, HERV insertion may benefit the host, for example, by reverting harmful mutation [28].
Title: Non-LTR retrotransposon

Human non-LTR retroposons include both active (L1, Alu, and SVA) and inactive elements (L2 and mammalian-wide interspersed repeat).
Although more than 500,000 copies of L1 elements are found in the human genome, only 100 copies are known to be intact [29].
An intact L1 element is approximately 6 kilobases (kb) in length, with a 5&#39; untranslated region (UTR) containing an internal RNA polymerase II promoter, two open reading frames (ORF1 and ORF2), and a 3&#39; UTR (Fig.1) [29, 30].
L1 elements are the only autonomous TEs in the human genome because of their retrotransposition property, called target-primed reverse transcription [27].
Alu elements are often called &#34;a parasite&#39;s parasite,&#34; because they do not code for a polymerase and, hence, are non-autonomous in nature.
Alu elements depend on L1 elements for retrotransposition machinery [31, 32, 33].
However, they are considered the most successful TEs in the human genome in terms of copy number [27].
An intact SVA element is approximately 2 kb in length, which includes a hexamer repeat, an Alu-like region, a variable number of tandem repeats, and a HERV-K10-like sequence.
SVA elements are also non-autonomous and most likely depend on L1 retrotransposition machinery [34, 35].
Two key features of non-LTR retrotransposons that control retrotransposition activity are high copy number and continued activity over millions of years [5, 27].
From an evolutionary perspective, the observed uniqueness of non-LTR retrotransposons is due to their vertical transfer in both primates and mammals [4, 5, 36].
Amplification rates among non-LTR retrotransposons are not uniform (Alu, 40 Mya; L1, 12-40 Mya; SVA, 6 Mya) [27].
Among non-LTR retrotransposons, Alu elements are the most thoroughly studied in relation to evolution.
Alu repeats may cause genomic diversity in various ways.
Their amplification has enabled them to become the largest family of mobile elements in the human genome.
It is estimated that thousands of Alu elements have integrated into the human genome since the divergence of humans and African apes [37, 38, 39, 40].
Although some Alu insertions have caused harmful mutations, most have contributed to genetic diversity [8].
Moreover, Alu repeats have also influenced the accumulation of single-nucleotide polymorphisms in the genome [30, 40, 41].
From previous reports, it is evident that most of the recent Alu insertions are the source of genetic variations, which have been useful for studying both the relationships between populations and the evolution and organization of tandem-arrayed gene families [37, 38, 39, 40].
Batzer and Deninger [42] suggested that the Alu insertion, in relation to genetic variation, may also be useful in generating species-specific genetic markers.
TEs can cause genomic instability either by insertions or by rearrangements in the genome.
Notably, structural variations in the human genome are the primary cause of inter-individual variability.
Structural variations include insertion, deletion, inversion, duplication, and translocation.
The characteristics of TEs, such as abundance in the genome, high sequence identity, and ability to move, make them major contributors to genomic instability [5, 27, 43].
Recent studies have revealed the implications of TEs in genomic instability and human genome evolution [44].
Mutations associated with TE insertions are well studied, and approximately 0.3% of all mutations are caused by retrotransposon insertions [27].
Such insertions can be deleterious by disrupting the regulatory sequences of a gene.
When a TE inserts within an exon, it may change the ORF, such that it codes for an aberrant peptide, or it may even cause missense or nonsense mutations.
On the other hand, if it is inserted into an intronic region, it may cause an alternative splicing event by introducing novel splice sites, disrupting the canonical splice site, or introducing a polyadenylation signal [8, 9, 10, 11, 42, 43].
In some instances, TE insertion into intronic regions can cause mRNA destabilization, thereby reducing gene expression [45].
Similarly, some studies have suggested that TE insertion into the 5&#39; or 3&#39; region of a gene may alter its expression [46, 47, 48].
Thus, such a change in gene expression may, in turn, change the equilibrium of regulatory networks and result in disease conditions (reviewed in Konkel and Batzer [43]).
The currently active non-LTR transposons, L1, SVA, and Alu, are reported to be the causative factors of many genetic disorders, such as hemophilia, Apert syndrome, familial hypercholesterolemia, and colon and breast cancer (Table 1) [8, 10, 11, 27].
Among the reported TE-mediated genetic disorders, X-linked diseases are more abundant than autosomal diseases [11, 27, 45], most of which are caused by L1 insertions.
However, the phenomenon behind L1 and X-linked genetic disorders has not yet been revealed.
The breast cancer 2 (BRCA2) gene, associated with breast and ovarian cancers, has been reported to be disrupted by multiple non-LTR TE insertions [9, 18, 49].
There are some reports that the same location of a gene may undergo multiple insertions (e.g., Alu and L1 insertions in the adenomatous polyposis coli gene) (Table 1).
It has also been proposed that inverted repeats are likely to be hotspots of genomic instability [50].
Closely occurring Alu repeats form hairpin structures that are prone to double-strand breaks (DSBs) and excision [50, 51].
In addition, de novo Alu insertions may create new inverted repeats that result in rearrangements in future generations [43].
Due to the abundance of TEs in the human genome, the probability of TE-mediated NAHR translocations is high.
Kolomietz et al.
[52] reported that Alu elements are often found in and around the breakage points of translocations and result in diseases [53].
Some studies have analyzed the human genome using the chimpanzee reference genome and found that deletions that are caused by Alu-mediated NAHRs are 9 times more frequent than L1-mediated NAHRs in the human genome [54, 55].
Alu-mediated NAHRs are known to be associated with various genetic disorders and cancer (Table 1) [8, 10].
L1 endonuclease creates abundant DSBs that are required for retrotransposition in mammalian cells and eventually contribute to genomic instability [56].
However, it has been difficult to define this in an experimental condition, as physiological conditions cannot be simulated in vitro.
Moreover, it is also difficult to distinguish DSBs caused by L1 from DSBs caused by other mechanisms in vivo [44, 56].
From recent studies, it is apparent that the methylation state of DNA is associated with cancer [57].
TEs-in particular, the promoters of L1 elements-are reported to be demethylated in cancer cells [58, 59].
On the other hand, the methylation of retrotransposons is supposed to be a defense mechanism against retrotransposition in somatic cells [60].
It has also been reported by some studies that the L1 transcription rate is increased in hypomethylated cancer cells [58, 61].
Demethylation of TE promoters may result in their activation, which in turn could modify the TF level in the cell.
It is possible that such changes in TF levels lead to alterations in global gene expression [57].
In addition, demethylation may result in the activation of the L1 antisense promoter, which may eventually produce cancerassociated chimeric transcripts [62, 63].
Taken together, the mechanisms discussed above have demonstrated the considerable impact of TEs on human genome evolution, genetic instability, and disease occurrence.
There has been a recent increase in studies demonstrating the roles of TEs in multiple molecular processes.
Importantly, several studies have found an association between TEs and cancer conditions.
Technological developments have led to promising techniques (e.g., next-generation sequencing) that will assist researchers in studying, understanding, and confirming the role of TEs in genetic instability and diseases.
Such progress may lead to the development of novel therapeutic strategies in the near future, such as personalized gene therapy for the treatment of genetic disorders.
The clinical community has already realized the importance of personalized cancer treatments and is moving toward excellence in such treatment strategies.
Therefore, we believe that in-depth studies on the role(s) of TEs in evolution, the epigenetic control of gene expression, and clinical aspects will be of paramount importance in uncovering novel mechanisms that can be targeted for therapeutic intervention.
Classification and structure of transposable elements.
DRs, direct repeats; ITRs, inverted terminal repeats; TSD, tandem site duplication; LTRs, long terminal repeats; UTRs, untranslated region; ORF, open reading frame; L, left; R, right; SINE, short interspersed nuclear element.
Title: Disease conditions associated with transposable element (TE) insertions
Title: Editor&#39;s Introduction to This Issue

This is a special issue, consisting over 10 selected dissertations that were awarded in 3 genome analysis competitions (KOGO Young Scientist Best Paper Award, KNIH KARE Data Analysis Award, and KOBIC Bioinformatics Software Award), hosted by KOGO (Korea Genome Organization) in 2014.
A dissertation (In-Pyo Baek et al.
)on developing software that defines and visualizes phenotype-related single-nucleotide variants (SNVs) among whole SNVs through next-generation sequencing (NGS) won the grand prize for the KOBIC award, and the KNIH KARE award was given to the elucidation of epigenetic interactions of genes regulated by non-coding elements in complex diseases by Min Kyung Sung et al.
The other 8 consist of dissertations regarding GAWS, genomewide classification of miRNA, and short-read phasing for constructing haplotypes in certain genomic loci that are associated with BMI.
In addition, there are 4 review papers on diverse genome research issues.
Dr. Feng Zhang&#39;s group of Fudan University reviewed the genomic architecture, synthesis mechanism, and biological roles of copy number variants.
Dr. Hyun-Seok Park&#39;s group of Ewha Womans University reviewed the design of hidden Markov models and their application in solving various computational problems in epigenetics.
Drs.
Seon-Hee Yim and Yeun-Jun Chung reviewed current issues in direct-to consumer genetic testing.
Dr. Perumal Elumalai reviewed the chemo-preventive potential of nimbolide for cancer treatment.
Several genetic association studies in hyperuricemia, maturity-onset diabetes, BMI, and insulin levels will be interesting for readers.
In silico characterization of a drug target candidate enzyme in legionellosis, rifampicin resistance mechanisms in tuberculosis treatment, and a study of phytochemicals for the treatment of type 1 diabetes will be useful information for researchers in this field.
Genomics &amp; Informatics is already available in PubMed Central and PubMed.
Recently, Genomics &amp; Informatics applied for listing in SCOPUS.
Title: Genome Architecture and Its Roles in Human Copy Number Variation

Besides single-nucleotide variants in the human genome, large-scale genomic variants, such as copy number variations (CNVs), are being increasingly discovered as a genetic source of human diversity and the pathogenic factors of diseases.
Recent experimental findings have shed light on the links between different genome architectures and CNV mutagenesis.
In this review, we summarize various genomic features and discuss their contributions to CNV formation.
Genomic repeats, including both low-copy and high-copy repeats, play important roles in CNV instability, which was initially known as DNA recombination events.
Furthermore, it has been found that human genomic repeats can also induce DNA replication errors and consequently result in CNV mutations.
Some recent studies showed that DNA replication timing, which reflects the high-order information of genomic organization, is involved in human CNV mutations.
Our review highlights that genome architecture, from DNA sequence to high-order genomic organization, is an important molecular factor in CNV mutagenesis and human genomic instability.
Genetic mutations have been known as one of the key factors in the pathogenesis of human diseases.
Besides the well-known single-nucleotide variants, it has been shown that the large-scale genomic variants also make a great contribution to human health.
&#39;Genomic disorders&#39; are human diseases caused by relatively large genomic rearrangements [1].
Such large-scale genomic variants (named copy number variation [CNV]) can also be frequent in human populations [2, 3].
CNV involves DNA segments larger than 1 kb and exhibits variable copy numbers among individuals, comprising deletions and duplications/insertions [4, 5].
In the past 10 years, CNV has been found to play an important role in both sporadic Mendelian disorders and complex diseases.
Previous studies have reported that CNV can be mediated by multiple molecular mechanisms involving various genomic features.
Here, we focus on CNV mutagenesis and review the involvement of human genome architecture in CNV instability and the underlying molecular mechanisms.
Title: Genomic disorders and low-copy repeats

Large-scale genomic changes in the human genome can be associated with human diseases.
Such clinical conditions resulting from human genome architecture are termed &#39;genomic disorders&#39; [1].
The structural features, such as genomic repeats, can provide substrates for homologous recombination and induce genomic rearrangements and genomic disorders.
Stankiewicz and Lupski [6] defined region-specific lowcopy repeats (LCRs) as paralogous genomic segments spanning 10-400 kb of genomic DNA and sharing ≥95%-97% sequence identity.
The non-allelic homologous recombination (NAHR) between directly oriented LCRs can generate microdeletions and microduplications of megabases in size, which are frequently associated with genomic disorders (Fig.1).
For example, the 22q11.2 deletion syndrome is a well-investigated disorder caused by microdeletions between the paired LCRs in human 22q11.2, which deletes one copy of TBX1, CRKL, MAPK1, and several additional genes [7, 8, 9, 10].
In addition to microdeletions, microduplications can also manifest as genomic disorders.
The 1.4-Mb microduplication involving the PMP22 gene in human 17p12 can lead to CMT1A, which is a classical model for disease resulting from gene dosage effects [11, 12].
Title: Segmental duplication and NAHR

Genomic repeats play a significant role in human evolution and have a strong association with genomic CNVs [6, 13, 14, 15].
In 2001, Eichler [16] initially conducted a systematically bioinformatic analysis for such low-copy genomic repeats and defined them as segmental duplications (SDs), which have a high degree of sequence identity (&gt;90%-95%) and large genomic sizes (1-100 kb).
After that, Bailey et al.
[17] further performed a whole-genome assembly comparison to detect SDs with pair-wise alignments ≥ 90% and ≥ 1 kb in the human genome.
In addition to human SDs, the subsequent analyses also identified the SD architecture in the genomes of other primates, including chimpanzee, gorilla, and orangutan, and even in the mouse genome [17, 18, 19, 20], all of which have been archived in the online database of the University of California Santa Cruz (UCSC) Genome Browser (http://genome.ucsc.edu/).
LCR/SD is a very important category of DNA architecture in the human genome.
It has been found that they are associated with duplicated genes and pseudogenes [21], co-localize and overlap with Alu elements and CNV [22, 23], and play an important role in genome evolution [24, 25, 26].
LCR/SD pairs, acting as substrates, are thought to be a key factor in triggering NAHR events and causing CNV mutations [4, 27, 28, 29, 30, 31, 32].
Generally, reversely oriented SDs can align and subsequently crossover with each other via NAHR, resulting in copy-neutral inversions of the flanking DNA fragments (Fig.1A).
Similarly, NAHR events between direct SD pairs can cause CNVs.
Based on the different positions of SD pairs, different types of CNVs occur.
Duplications or deletions take place in NAHR events between different chromatids (interchromatid) (Fig.1B), while only deletions occur in NAHR on the same chromatid (intra-chromatid) (Fig.1C).
Based on the observations in specific pathogenic loci, SD properties (including homology length, distance, and sequence similarity) were shown to affect the incidence of NAHR [33].
In a recent study on common CNVs in human populations, it was found that SD length and inter-SD distance were the major SD properties involved in NAHR frequency [34].
A model of chromosomal compression/extension/looping has also been proposed for homology mispairing in NAHR [34].
Title: High-copy repeats in CNV instability

The genomic repeats representing DNA primary structures can be divided into LCR/SD and high-copy repeats.
Compared to LCR/SD, high-copy repeats constitute a great portion of the human genome.
Interspersed repeats are the most common type of high-copy repeats, which cover over 44% of the human genome [35].
One of the major classes of interspersed repeats is the retrotransposon, including short interspersed elements (SINEs), long interspersed elements (LINEs), and endogenous retroviruses (ERVs).
SINEs are short DNA sequences (100-400 bp in length) with an internal (mobile) polymerase III promoter [15], making up about 11% of the human genome [36].
The most common SINEs are Alu elements, which burst out in the evolution of primates [37].
Moreover, Alu elements play an important role in disease, such as breast cancer, Ewing&#39;s sarcoma, familial hypercholesterolemia, and so on [38].
In 2008, Kim et al.
[22] found a strong association between Alu elements and old SDs.
By means of NAHR, Alu elements contribute to the formation of CNVs, especially deletions [39, 40, 41, 42, 43].
The most classic repeats in LINEs are the LINE-1 (L1) elements, which are 6-8 kb in length [44].
Covering about 17% of the human genome, L1 elements can elevate genomic instability, provide resources for NAHR [39, 40, 41, 45, 46], and cause human diseases [47, 48, 49, 50].
In the human genome, there exist at least 50,000 copies of ERVs [51], which are defined as human endogenous retroviruses (HERVs), covering about 4.9% of human DNA sequences [21].
Via the NAHR mechanism, HERVs were found to induce large deletions and cause hypotonia and motor, language, and cognitive delays [52, 53].
Intriguingly, a series of studies show that there is a strong association between HERVs and CNVs in the region of AZFa, a well-known locus related to male infertility [54, 55, 56, 57].
As discussed above, repeat-mediated NAHR is one of the major mutational mechanisms for CNV formations.
In these recombination-based events, paired repeats in direct orientation contribute to CNV instability and disease traits.
However, is it the only way for the genomic repeats to induce CNV mutations?
The recent investigation of DNA replication-based mechanisms provides novel insights into repeat-mediated CNV instability.
Title: Inverted repeats involved in CNV instability

Genomic repeats, especially inverted repeats (IRs), can cause DNA replication error and induce CNV formation.
IRs, sharing high sequence similarity in adjacent loci, are found to align or crossover with each other and form specific DNA secondary structures, such as cruciform structures, during DNA replication [58].
Formation of such secondary structures can cause DNA replication fork stalling, and later, jumping into the wrong locus to continue replicating (Fig.2).
This mechanism of triggering replication errors subsequently results in genomic rearrangements and CNV mutations [24].
Title: IRs induce complex CNVs by replication errors

During DNA replication, IRs could form DNA secondary structures, which will induce replication fork stalling.
Template switching and replication resumption further result in CNV mutations (Fig.2).
Notably, replication-associated events usually lead to complex CNVs, which include the combined segments of deletions and duplications.
As was reported, Chen et al.
[59] identified three complex CNVs that could be explained by a model of serial replication slippage (SRS).
In this model, IRs have the potential to induce SRS and cause CNV mutations.
IRs can induce complex CNVs, as observed in the MECP2 locus in chromosome Xq28 and the PLP1 locus in chromosome Xq22 [60].
To elucidate the mechanisms of complex CNVs in the PLP1 locus, Hastings et al.
[61] found both microhomology and IRs at the breakpoints.
They proposed that both breakage of replication forks and the IR-mediated aberrant repair process can result in complex CNVs.
This model was termed &#39;microhomology-mediated break-induced replication,&#39; which was used to explain the formation of the complex CNVs involving individual genes or even single exons [62].
Title: Self-chains in CNV formation

The aforementioned SDs are long (&gt;1 kb) and LCRs in the human genome.
Besides SDs, self-chains (SCs) are another type of short LCRs, which were previously analyzed and mapped via self-alignment in the human genome utilizing BLASTZ [63, 64].
SCs are short in length (91% of which range from 150 bp to 1 kb in size) [14].
Furthermore, SCs have a limited number of matched alignments in the human genome.
Thus, SCs represent a distinct category of human short LCRs.
In 2013, Chen et al.
[65] studied deletion CNVs in the NRXN1 gene and its flanking regions.
After mapping and analyzing the breakpoints of 32 deletions, they found a significant bias that minus SCs (i.e., paired SCs in the inverted orientation) were overrepresented in the vicinity of deletion breakpoints in the NRXN1 region.
Furthermore, they claimed that the SCs can increase genomic instability and cause deletions via DNA replication errors.
Their work contributes to the exploration on SC-mediated CNVs.
To perform a genomewide analysis on the contribution of SCs to human CNV instability, Zhou et al.
[14] plotted the numbers of SC regions with different orientations in the entire human genome.
After masking the SDs and gaps in the human genome, utilizing the germline CNVs in human populations and the somatic CNVs in various cancer genomes, they observed a significant biased distribution of CNV breakpoints to SC regions, which indicated that SC-mediated secondary structures may induce DNA replication errors and potentially generate different types of CNVs, such as deletions and duplications.
In this case, SCs represent a new genomic architecture for the underlying regional susceptibility to genomic instability, further giving rise to CNVs.
Title: DNA repair and nonhomologous end-joining

While DNA double-strand breaks (DSBs) occur, nonhomologous end-joining (NHEJ) is one of the molecular mechanisms for repairing DSBs and maintaining genome integrity.
Once a DSB is detected, the broken DNA ends are bridged and modified by the enzyme machinery.
After that, the final ligation is needed for DNA repair.
Unlike NAHR, NHEJ can take place without any homology as the substrate.
Notably, deletions or insertions of several base pairs are usually brought to the joint point.
More mechanistic details of NHEJ are provided in some previous works [66, 67, 68].
In addition to the aforementioned genomic features, some high-order genome organizations might contribute to genome instability.
New observations in the human genome showed that the DNA mutation rate is associated with DNA replication timing.
Stamatoyannopoulos et al.
[69] found that the human point mutation rate is markedly increased in genomic regions of late replication.
This correlation indicates that DNA replication timing, as an important feature of replication dynamics, is involved in genomic instability and enlightens the investigation on the relationship of replication timing and CNV instability.
Title: Replication timing as a high-order genomic feature

DNA replication takes place at replication forks following a fixed way [70].
In the human genome, the segments of chromosomes replicate in a temporal order [71], and the whole genome is spatially segregated by replication zones of different organizations.
With some replicons in one spatial compartmentalization of chromatin fired synchronously, this chromosomal unit shares the same replication timing, termed the &#39;replication domain.&#39;
Therefore, the genome consists of several replication domains with different replication timing and the timing transition regions.
Replication timing can be measured by two distinct methods, based on current genome technologies [72, 73, 74].
One method is to label the newly replicated DNA with chemically tagged nucleotides.
Then, the DNA will be isolated from cells at various times during S phase by immune-precipitation or density fractionation.
In the other method, since DNA segments that replicate earlier accumulate more copies than those that replicate late in most cells-the DNA content of a region simply reflects the replication timing.
After being classified by florescence-activated cell sorting, the DNAs extracted from S phase and G1 phase cells, respectively, are compared by next-generation sequencing or microarray technologies.
By either way, a replication timing profile can be generated (Fig.3) [75].
Based on the timing profiles, a lot of progress has been made on understanding the replication program and its relationship with other genome architectures.
Recent findings indicate significant links between replication timing and the features of primary genomic structures [76].
The genomic regions where DNA replicates earlier usually have more genes, fewer LINEs, and higher GC content [77, 78].
Moreover, it is noticed that DNA replication timing correlates with transcription [79, 80, 81].
Expressed genes replicate earlier, while repressed genes replicate late.
Although this correlation shows a discrepancy between multicellular and single-celled organisms, it is worth noting that such works indeed reveal the striking association of replication timing and transcriptional activity in humans [77, 82, 83].
Moreover, recent findings show that replication timing strongly correlates with three-dimensional chromatin structures [84].
In Hi-C data, it has been observed that chromatin is organized into two separate compartments.
Remarkably, DNA that resides in close spatial structures replicates in near time, and chromatin that interacts between two compartments is exactly at the timing transition regions.
This observation suggests replication timing as an independent advanced genomic feature.
Title: Replication timing and CNV instability in human populations and cancers

The relationship of DNA replication timing and genomic instability, which is involved in genomic mutation and human disease, is what people are most concerned about.
As mentioned above, human mutation rates, based on evolutionary divergence and single-nucleotide polymorphism frequency, are increased in late-replicated regions [69].
Koren et al.
[75] generated a high-resolution timing profile of the human genome and investigated the relationship between DNA replication timing and point mutations.
In accordance with the previous discovery, this association was also observed and proved to be much stronger.
How is CNV related to replication timing?
Recent studies showed some distinct but multi-dimensional relationships between CNVs and replication timing.
Based on the duplication hotspots conserved between two species of Drosophila, Cardoso-Moreira et al.
[85] explored the roles of replication timing in genomic instability.
They found that Drosophila duplication hotspots were enriched in late-replicated regions, unlike the aforementioned sequences of high sequence identity in the human genome.
However, in spite of the association observed in Drosophila, the situation seems to be more complicated in mammalian genomes.
In the study of Koren et al.
[75], the relationship between early/late replication timing and CNV mutation was also investigated.
The CNVs, mediated by different mechanisms, showed divergent patterns, suggesting a multi-dimensional interaction between CNVs and replication timing.
In addition to the observations in human populations, recent findings have also discovered the relationship of genomic reorganization and the subsequently generated genetic variation during cell fate changes.
Lu et al.
[86] have investigated the impact of altered replication timing on the CNV landscape during reprogramming.
Approximately 40% of the human genome changes with regard to replication timing between human induced pluripotent stem cells (iPSCs) and their parent fibroblasts.
Intriguingly, the CNV distribution shows a correlation with the changed timing profile.
In particular, CNV gains tend to be located in the genomic regions that switch to replicate earlier.
This correlation is conserved among different reprogramming methods.
Compared with cell fate changes, replication timing is disrupted in many disease states, including cancer [87].
It has been noticed that numerous alterations to the replication program take place during carcinogenesis.
One of the changes is the aberrant asynchronous replication of loci that replicate synchronously in normal cells.
This phenomenon exists in not only cancer but also noncancerous cells [88, 89, 90].
This abnormal replication program apparently has a notable impact on genomic stability and thus increases the frequency of chromosomal rearrangements and CNVs.
Recent findings have indicated that aberrant DNA replication timing is involved in changes in gene expression, epigenetic modifications, and an increased CNV mutation frequency [91, 92].
An analysis of 331,724 somatic copy number alterations (SCNAs) has shown that SCNAs increase in late-replicating regions among cells of different cancer types.
Like the findings in iPSCs, the SCNA distribution is related to replication timing in tumor cells.
In particular, amplification boundaries tend to be located in early-replicated regions, whereas deletion boundaries are more likely to reside in late-replicated regions [93].
Title: Integrated replication dynamic related with CNV instability

In the study of Koren et al.
[75], point mutations and CNVs showed different patterns in their correlations with replication timing.
These observations may reflect the distinct mutational mechanisms between these two types of genomic variants and suggest complex effects of DNA replication on CNV instability.
We hypothesized that integrated replication dynamics, which are not just early/late replication timing, contribute to CNV mutation.
It has been reported that dividing the genome into early/late replication timing alone does not give the entire characteristics of DNA replication fork dynamics [72].
Actually, the timing transitional regions represent the interactions of two spatially dependent chromatin compartments, which are DNA segments with low rates of replication fork progression.
Notably, slower fork speed and increased fork stalling have been found to be associated with cancer cells and result in CNV mutations [94].
Actually, Chen et al.
[95] have conducted a statistical method, estimating replication dynamics, and observed its significant association with CNV instability.
Replication dynamics may be used as a measure of the progress of genome replication and regional replication stress and provides novel insights into the roles of DNA replication in CNV mutagenesis.
Human genomic repeats play an important role in CNV mutation, genomic disorders, and genome evolution.
Both low-copy genomic repeats (including LCRs and SDs) and high-copy repeats (including Alu, LINEs, and HERVs) can induce CNV formation via classical DNA recombination-based mechanisms, such as NAHR.
Furthermore, paired repeats (especially those in the inverted orientation) are even more crucial as substrates to form DNA secondary structures and cause DNA replication fork stalling and replication stress.
This will induce DNA replication errors and subsequently generate CNV mutations.
Besides the primary structural features (e.g., the organization of repeat sequences in the human genome) and repeat-mediated secondary DNA structures, higher-order genomic architecture (such as replication timing) is also involved in CNV instability.
Further investigation of the role of DNA replication dynamics in CNV mutagenesis will reveal more mutational mechanisms underlying genomic disorders and genome evolution.
The non-allelic homologous recombination (NAHR) events between paired low-copy repeats (LCRs)/segmental duplications (SDs) [1].
Paired LCRs/SDs are depicted as bold arrows (red and blue) with the orientation indicated by arrowheads.
Capital letters near the LCRs/SDs refer to the flanking unique sequences, while the same letter on different lines indicates the homologues on the other strand.
Dashed crossed lines represent a homologous recombination event.
(A) The NAHR event between reversely oriented LCRs/SDs can cause inversion, a copy-neutral structural variation.
(B) The inter-chromatid NAHR events between directly oriented LCRs/SDs result in deletions and duplications.
(C) The intra-chromatid NAHR events between directly oriented LCRs/SDs can generate deletions and ring-shaped DNA segments that will be lost in subsequent cell divisions.
Repeat-induced DNA replication errors and copy number variation (CNV) formation.
The straight lines depict single DNA strands, and the solid arrows (red and blue) represent genomic repeats.
The dashed lines indicate newly synthesized DNA strands.
During DNA replication, adjacent repeats could form DNA secondary structures (such as hairpin) that consequently result in replication fork stalling.
Then, CNVs are generated via DNA template switching.
For example, (1) jumping over the secondary structures and restarting DNA replication lead to deletions and (2) switching to a new template (shown in green lines) and switching back result in duplications of the green DNA segment.
The DNA replication timing profile of human lymphoblastoid cells.
The data of human chromosome 4, which were obtained from Koren et al.
[75], are shown.
The blue lines show the replication timing, high values of which indicate that DNA replicates early in these regions, and vice versa.
Title: A Review of Three Different Studies on Hidden Markov Models for Epigenetic Problems: A Computational Perspective

Recent technical advances, such as chromatin immunoprecipitation combined with DNA microarrays (ChIp-chip) and chromatin immunoprecipitation-sequencing (ChIP-seq), have generated large quantities of high-throughput data.
Considering that epigenomic datasets are arranged over chromosomes, their analysis must account for spatial or temporal characteristics.
In that sense, simple clustering or classification methodologies are inadequate for the analysis of multi-track ChIP-chip or ChIP-seq data.
Approaches that are based on hidden Markov models (HMMs) can integrate dependencies between directly adjacent measurements in the genome.
Here, we review three HMM-based studies that have contributed to epigenetic research, from a computational perspective.
We also give a brief tutorial on HMM modelling-targeted at bioinformaticians who are new to the field.
Many researchers have shown that formal language theory is an appropriate tool in analyzing various biological sequences [1, 2].
The hidden Markov model (HMM) is most closely related to regular grammars, because an n-gram is a subsequence of n items from a given sequence, and language models that are built from n-grams are actually (n-1)-order Markov models.
However, the research of modeling biological sequences has usually focused on nucleotide or amino acid sequences that encode RNA or interact with proteins [3, 4, 5, 6, 7, 8, 9, 10, 11, 12].
On the other hand, noncoding DNA regions, which occupy approximately 98% of human DNA, have not been considered for HMM-based analysis.
The reason is partially due to the fact that a large proportion of noncoding DNA has been believed to have no known biological functions.
However, recent technical advances, such as chromatin immunoprecipitation sequencing (ChIP-seq), DNase I hypersensitive sites sequencing (DNase-seq), formaldehyde-assisted isolation of regulatory elements (FAIRE) [13, 14], and computational epigenetics, have started to convert unannotated noncoding DNA into highly annotated functional areas [15, 16].
The work is analogous to dissecting the region that constitutes the noncoding DNA and understanding the type of meaning each element contains.
For this reason, the field of epigenetics has received a boost of attention and is currently among the fastest moving areas in molecular biology.
However, epigenetic mechanisms are highly interwoven in a complex network of interactions.
Disentangling this network is an important goal of epigenetic research.
Thus, various bioinformatic challenges arise from the analysis of epigenetic data, and HMMs have played a significant role in solving important epigenetic problems, as HMMs are well suited to the task of discovering unobserved &#39;hidden&#39; states from &#39;observed&#39; sequences in their spatial genomic context.
In this paper, we give a tutorial review of the design of HMMs and their applications to solve various computational epigenetic problems.
We selected three representative works to compare different designs of HMMs for various computational epigenetic problems: the Li et al.
[17] two-hidden-state HMM to determine transcription factor binding sites, the Xu et al.
[18] three-hidden-state HMM to compare histone modification sites, and the Ernst and Kellis [19] multi-state multivariate HMM to analyze systematic state dynamics of human cells.
We want to clarify the fact that this review is by no means exhaustive and that there exist many other types of HMMs for computational epigenetic problems.
An HMM is a statistical model that can be used to describe observable events that depend on hidden factors.
An HMM consists of two stochastic processes: an invisible process of hidden states based on a Markov chain and a visible process of observable symbols.
A first-order HMM can be defined formally as a quintuple (S, π, ∑, a, e), where S = {1, 2, .

.

.

, n} is a finite set of hidden states; π is vector of size n defining the starting probability distribution; ∑ = 1, 2, .

.

.

, m is a finite set of output symbols; aij is a two-dimensional matrix of transition probabilities of moving from state i to state j; and ei(x) is an n × m matrix of emission probabilities of generating symbol x in state i.
The key property of a Markov chain is that the probability of each symbol xi depends only on the value of the preceding symbol xi-1 [i.e., P(xi ∣xi-1)], not on the entire previous sequence [i.e., P(xi ∣xi -1, .

.

.

, x1)].
In the bioinformatics context, a nucleic one for genes, genomes, amino acids, or RNA is a sequence.
And sequences can represent functional regions in the genome.
Whereas previous studies of coding DNAs and promoters usually modeled their HMMs using nucleotide or amino acid sequences as their output symbols, recent HMM studies that are related to epigenomics tend to model their HMMs using chromatin marks in bins of equal length as output symbols, replacing the traditional nucleotide or amino acid sequences.
To explain the difference, let us consider a simple example.
Suppose that adjacent regions of genomic sequences are divided into multiple 10-bp bins (though unrealistic), as in Fig.1, in which some kinds of chromatin marks or methylation profiles are annotated.
Suppose also that we define two imaginary methylated states, &#39;M&#39; (in green color) and &#39;U&#39; (in orange color), based on some kinds of epigenetic profiles.
Let us consider a toy HMM for Fig.1.
Given random training data, we try to determine five parameters of the HMM.
An HMM is usually visualized as a directed graph with vertices corresponding to the states and the edges representing pairs of states with transition probability aij and emission probability ei(j), as in Fig.2.
The graph defines the topology of the model, while the emission and the transition probabilities define the parameters of the model.
The given HMM tries to capture the statistical differences in the two hidden states of &#39;M&#39; and &#39;U.&#39;
The transition probability represents the change of the methylation state in the underlying Markov chain.
According to Fig.2A, there is a 20% chance of moving from state &#39;M&#39; to state &#39;U&#39; (aMU), an 80% chance of staying in state &#39;M&#39; (aMM), a 10% chance of moving from state &#39;U&#39; to state &#39;M&#39; (aUM), and a 90% chance of staying in &#39;U&#39; (aUU).
The probability of starting from M and U is 60% and 40%, respectively.
M and U use different sets of emission probabilities to reflect the symbol statistics.
In epigenetic studies, emission symbols are not nucleotide or amino acid sequences.
Rather, emission symbols are usually defined as a value or even a vector of chromatin marks.
Fig.2A is an ergodic HMM.
An ergodic HMM is one for which the underlying Markov chain is irreducible and admits a unique stationary distribution.
In contrast, a left-to-right version of an instantiated HMM in Fig.2B is an HMM with a Markov chain that starts in a particular initial state, traverses intermediate states, and terminates in a final state.
Each circle shape represents a hidden state.
The random variable xt is the hidden state at time t. The variables &#39;M&#39; and &#39;U&#39; are the observations at time t (with St ∈ {M, U}).
The arrows in the diagram denote conditional dependencies.
Unlike an ergodic HMM, the chain may not go backwards while traversing the trellis.
Genome-wide mapping of epigenetic information follows a basic three-stage design process [13]: conversion of the epigenetic information into genetic information, application of standard DNA techniques, and computational analysis to infer the epigenetic information.
All experimental methods for epigenomic annotation generate large amounts of data and require efficient ways of processing the data.
In the remainder of this review, we show how HMMs have contributed to answering important epigenetic questions by summarizing HMM-based bioinformatic works, from a computational perspective.
Table 1 summarizes the HMM parameters of the three studies that will be reviewed in this paper, in the order of the hidden state complexity.
Li et al.
[17] developed a method to determine transcription factor binding sites from chromatin immunoprecipitation combined with DNA microarrays (ChIP-chip) experiments on an Affymetrix tiling array of chromosomes 21 and 22.
Owing to the Affymetrix array characteristics and genome sequence similarity, probes with the same 25-mer sequence tend to be spotted at different locations on the chips.
Two-stage procedures were employed to filter out short-range repetitive probe measurements prior to downstream analysis.
Then, tiling array data from various experiments were gathered to normalize and model the behavior of each individual probe, where the behavior of each probe was modeled as a normal distribution.
Finally, a two-hidden-state (ChIP-enriched state and nonenriched state) HMM was built to estimate the probability of enrichment at each probe location.
Given J potential binding sites along chromosomes covered by K total probes, the initial probabilities and transition probabilities of HMM were characterized, as in Fig.3.
Initial probabilities were set to J/K for the ChIP-enriched state and to 1 - J/K for the nonenriched state.
Transition probabilities were set to J/K for transition to a different state and 1 - J/K for staying in the same state.
The emission probability distribution of probe in a single dataset was set to N(µi + 2σi, (1.5σi)2) for the ChIP-enriched state and N(µi, σi2) for the nonenriched state (where µi and σi are the mean and standard deviation, respectively, of probe i).
The parameters were based on previous results with the Affymetrix SNP arrays [20].
Other methods that are based on two-state HMMs for the identification of methylated regions were proposed [22, 23, 24, 25].
A common characteristic of these studies is the modeling of two different populations of measurements to differentiate nonenriched regions from enriched ones.
Ji and Wong [22] suggested TileMap, which is an effective statistical tool to identify genomic loci that show transcriptional or protein binding patterns of interest.
Martin-Magniette et al.
[23] proposed ChIP-mix, which is a statistical method based on the mixture of regression to classify probes in ChIP-chip experiments.
Johannes et al.
[24] presented a parametric classification method for genomewide comparisons of chromatin profiles between multiple ChIP samples.
In Moghaddam et al.
[25], they showed that H3K4me2 and H3K27me3 distribution patterns were similar overall in various Arabidopsis thaliana accessions and remained largely unchanged in their F1 progeny [25].
One general limitation of the methods in the previous section is that they only enable binary classification.
For this reason, a three-state HMM has been proposed for the analysis of so-called differential histone modification sites (DHMSs) of H3K27me3 by Xu et al.[18].
To capture the histone modifications on the whole-genome scale, Xu et al.
[18] proposed a qualitative analysis for the genome-wide comparison of histone modification sites by computationally comparing two ChIP-seq libraries generated from different cell types or experimental conditions.
To do that, the whole genome was partitioned into 1-kb bins, and the number of centers of ChIP fragments was counted and normalized in each bin, generating a profile of ChIP fragment counts.
Based on the observations of ChIP fragment counts, Fig.4 shows an HMM to infer the states of histone modification changes at each genomic location.
The HMM is characterized by three features: the prior probability of the start state, the emission probability, and the transmission probability.
A DHMS was defined as a bin in which the ratio of intensities between two profiles (L1 and L2) is larger than τ (L1-enriched DHMSs) or smaller than 1/τ (L2-enriched DHMSs), where τ is a predetermined threshold [19].
Based on the definition of a DHMS, the state Si takes one of the following three values: α0 if 1/τ ≤ p(1,i)/p(2,i) ≤ τ; α1 if p(1,i)/p(2,i) &gt; τ; and α2 if p(1,i)/p(2,i) &lt; 1/τ.
(For a region of k bins, the notation x1,i, x2,i was used for the ChIP fragment counts in L1 and L2, and the notation p1,i, p2,i was used for the intensity in L1 and L2, respectively, at the ith bin in that region.)
The initial state is fixed to take the value of α0, since the region is assumed to start from the genomic locations where the histone modification is depleted in both libraries.
The emission probability P(x1,i, x2,i ∣si) was calculated as in Fig.4 by integrating p1,i and p2,i over all possible values constrained by si.
The transmission probability table was trained using the Baum-Welch algorithm [21].
Other methods of three-state HMMs for the identification of methylated regions have been proposed.
Seifert et al.
[26] also proposed a three-state HMM specifically designed for the analysis of Arabidopsis MeDIP-chip data to differentiate between unmethylated (&#39;U&#39;), methylated (&#39;M&#39;), and highly methylated regions (&#39;I&#39;).
They utilized a three-state HMM with state-specific multivariate Gaussian emission densities to analyze methylation levels of chromosomal regions in methylation profiles.
Arand et al.
[27] suggested a four-state HMM (fully methylated, not methylated on both sides, methylated only on the lower strand, and methylated only on the upper strand) to determine the methylation status of both DNA strands.
In this paper, we reviewed three different types of HMMs and their applications in order of the complexity of the hidden states, from a purely computational perspective.
HMMs provide a sound mathematical framework for modeling and analyzing epigenetic data.
The appropriate method for the analysis of DNA methylation depends upon the goals of the study.
Researchers can design the most appropriate HMMs for their specific research needs and continued improvements in analysis methods make the study of DNA methylation more accessible.
A sample sequence, divided in 10-bp bins, annotated with two hidden states: M and U.
A Toy hidden Markov model (HMM) and its 5 parameters.
(A) An ergodic model of a toy HMM.
(B) An equivalent left-to-right version of the HMM in Fig.2A (picture adapted from the cs262 class slides by S. Batzoglou with permission).
A Two-state hidden Markov model (HMM) [17].
A three-state hidden Markov model (HMM) [18].
A multivariate, multi-state hidden Markov model (HMM) [19].
(A) Genomic profiling.
To apply the model, the genome was divided into 200-base-pair nonoverlapping intervals, within which each of the count of 41 marks that mapped to the interval was annotated.
(B) Binarization.
For each 200-bp interval, the input ChIP-Seq sequence tag count is processed into a binary presence/absence call.
(C) Learning.
Each model was scored based on the log-likelihood of the model minus a penalization on the model complexity, determined by the Bayesian Information Criterion (BIC).
(D) Annotation and analysis.
The vector of 41 numerical values was assigned, each representing the result of a different biochemical assay, and each of the 200-base-pair intervals was assigned to its most likely state under the model (picture adapted from the cs262 class slides by S. Batzoglou with permission).
Title: Five parameters of two-, three-, and multi-state hidden Markov models (HMMs) [17, 18, 19]
Title: Reflections on the US FDA&#39;s Warning on Direct-to-Consumer Genetic Testing

In November 2013, the US Food and Drug Administration (FDA) sent a warning letter to 23andMe, Inc. and ordered the company to discontinue marketing of the 23andMe Personal Genome Service (PGS) until it receives FDA marketing authorization for the device.
The FDA considers the PGS as an unclassified medical device, which requires premarket approval or de novo classification.
Opponents of the FDA&#39;s action expressed their concerns, saying that the FDA is overcautious and paternalistic, which violates consumers&#39; rights and might stifle the consumer genomics field itself, and insisted that the agency should not restrict direct-to-consumer (DTC) genomic testing without empirical evidence of harm.
Proponents support the agency&#39;s action as protection of consumers from potentially invalid and almost useless information.
This action was also significant, since it reflected the FDA&#39;s attitude towards medical application of next-generation sequencing techniques.
In this review, we followed up on the FDA-23andMe incident and evaluated the problems and prospects for DTC genetic testing.
On November 22, 2013, the US Food and Drug Administration (FDA) sent a warning letter to Ann Wojcicki, co-founder and CEO of 23andMe, Inc. and ordered the company to &#34;immediately discontinue marketing of the 23andMe Saliva Collection Kit and Personal Genome Service (PGS) until such time as it receives FDA marketing authorization for the device&#34; [1].
Since 23andMe is one of the largest direct-to-consumer (DTC) genetic testing companies leading the field, the FDA&#39;s action could have big implications for the present and future.
This FDA letter was even called a &#34;bombshell&#34; in one Nature Editorial [2].
Various responses and speculation have followed.
Some people support the FDA&#39;s action as protection of consumers from potentially invalid information, while others insist that the FDA&#39;s action violates the rights of individuals to receive information and of &#39;commercial speech&#39; and undermines democratization of health care and patient empowerment [3].
One of those responses is noteworthy: 7 months after the FDA&#39;s action, in June 2014, Lee and Lee [4] of the Korea Health Industry Development Institute (KHIDI) submitted a report to the Public-Private Joint Regulatory Advancement Initiative (PPJRAI), titled &#34;Market Analysis and Strategies for Regulatory Reform in Medical Industry,&#34; which expressed a stance somehow different from that of the US FDA.
To facilitate the discussion in our scientific community, we examined the development of the 23andMe event and reflected on the future of genetic-genome testing, including whole-genome sequencing as a medical device.
DTC genetic testing, also known as at-home genetic testing, refers to genetic tests that are sold directly to consumers via the internet, television, or print advertisement without necessarily involving health care providers.
Su [5] grouped the reasons for using DTC genetic tests into three categories; identity (such as ancestry, paternity, or ethnicity)-seeking; disease risk-testing to complement health care; and curiosity-driven testing.
DTC genetic testing has its pros and cons.
It is more accessible and affordable than traditional genetic tests, which usually target one specific gene at a time.
DTC testing companies and many personalized medicine advocates argue that knowing the genetic risks for potential diseases will help people to improve their health and wellness by changing what they can and managing what they can not, as claimed in 23andMe&#39;s TV commercial [6].
23andMe, one of the largest DTC genetic testing companies, also made the same claims and has offered the PGS to &gt;475,000 customers since 2007 [7].
23andMe&#39;s PGS genotypes the DNA sample on their custom single-nucleotide polymorphism (SNP) chip and provides health reports on 254 diseases and traits, including carrier status and drug response, which, the company claims, enables customers to personalize their health care.
Additionally, some proponents suggest, as another argument, that DTC genetic testing can contribute to the empowerment of consumers by increasing knowledge and to subsequent democratization of health care [5].
However, many geneticists and physicians have rejected the claims.
They believe that since DTC genetic testing is not reliable enough to be used clinically, it will just end up increasing health care costs unnecessarily [8].
Apart from its scientific weaknesses and insufficient clinical utility, there are other concerns raised by critics, including psychological and financial damage to consumers, especially by unreliable test results, health care inequity, and genetic privacy.
Since we believe that scientific validity is the foremost foundation of DTC genetic testing and should presuppose its clinical usage, we will focus on it in this review.
There are quite a few scientific limitations to genome-based health risk estimation services in general, not just 23andMe&#39;s PGS, but they may not be apparent, even to medical professionals, who do not have enough knowledge in genetics and genomics, as well as lay customers.
First of all, for monogenic disorders, DTC genetic testing panels include a small fraction of known disease-causing mutations or sometimes just include mutations present only in specific populations.
This incompleteness of testing results in misinterpretations, which makes its clinical value doubtful.
Also, for polygenicmultifactorial disorders and conditions, DTC testing panels include only a small percentage of the markers among many known associated markers, not to mention unknown ones, which also makes the risk prediction incomplete and, as a result, inaccurate.
We also should not forget that gene-gene and gene-environment interactions are not taken into consideration for risk estimation at all.
In addition, it is hard to say that their algorithms for estimating total combined risks are very well validated.
In summary, genome-based DTC tests are incomplete, and the interpretation of their results is hardly valid.
The biology is much more complicated than some health care entrepreneurs believe, and genotypes and sequences are not information but are just basic tools to use for understanding biology.
Even in the case of valid associations that have been replicated by many studies, the effect sizes of most SNPs on specific diseases seem to be small to modest [9].
Secondly, even though their customers are provided detailed background information on the method used for genotyping and the statistical interpretation, which supposedly helps their understanding, there is still ample possibility for misconceptions.
Some professional societies have issued statements on DTC genetic testing.
The American College of Medicine Genetics said, in their 2003 statement, that due to the complexities of testing and counseling, patients&#39; use of DTC genetic testing kits may pose harms, which include the following: test utilization for inappropriate purposes, misinterpretation of test results, and lack of necessary follow-up [10].
In 2007, the American Society of Human Genetics also expressed their concerns in a statement on DTC genetic testing, saying that consumers are at risk of harm if the quality of tests is not high enough-i.e., the analytical or clinical validity and clinical utility of tests are not appropriate [11].
Analytical validity shows how accurately and reliably the test detects a specific genetic variant.
Clinical validity shows how well the genetic variant is related to the existence or risk of a specific disease or condition.
Clinical utility shows how likely the test can improve patient outcomes [12].
Consumers are also at risk of harm if claims made about tests are misleading or simply false and if information and counseling are not properly provided to enable the consumer to make an informed decision about whether to take the test or not and about what actions to take after the genetic test [11].
Due to these criticisms and repeated interventions from regulators, including the US FDA, especially in the United States, the DTC market does not look very promising at this moment.
Even one Nature Editorial said that DTC genetic testing is not yet a viable business model, and most firms in the market have stopped providing services involved with personal disease risk estimation or have even totally shut down [2].
On November 22, 2013, the US FDA sent a warning letter to 23andMe, Inc. and ordered the company to immediately cease marketing of their Saliva Collection Kit and PGS until it received proper marketing authorization for the device [1].
This letter was sent after the firm failed to provide additional information on the analytical and clinical validity of the PGS the FDA requested.
It said that although the FDA has been diligently working to help the company comply with regulatory requirements since July 2009, it stopped communicating with the FDA since May.
Apparently making matters worse, the company initiated new marketing campaigns, including TV commercials.
This made the FDA conclude that the company intends to expand the PGS&#39;s uses and consumer base without obtaining marketing authorization from the agency [2].
The FDA clearly stated that their PGS requires marketing approval, because it is intended for use &#34;in the diagnosis of disease or other conditions or in the cure, mitigation, treatment, or prevention of disease, or is intended to affect the structure or function of the body.&#34;
In other words, the FDA considers the PGS an unclassified medical device, which requires premarket approval or de novo classification [1].
After receiving the letter, on November 26, 2014, Wojcicki acknowledged in a blog post that they were behind schedule with their responses to the FDA and that the agency needs to be convinced of the quality of their PGS data.
On December 5, 2013, the company announced that it would suspend its health-related genetic tests to comply with the US FDA&#39;s directive during the regulatory review process [13].
Based on the letter, the FDA does not seem to think that the analytical validity of 23andMe&#39;s SNP genotyping is a problem per se as far as its results are correct but feels that the clinical validity of risk estimation has yet to be validated, which may be practically impossible.
After that, 23andMe announced that it has been working with the FDA, and the agency seemed to consider the firm&#39;s response to be adequate [14].
However, the firm is not able to provide any timeline for resuming their PGS.
The US FDA&#39;s action evoked mixed reactions.
Some expressed their concerns, saying that the FDA is overcautious and paternalistic and hinders medical advances.
According to Green and Farahany [3], the FDA&#39;s action violates the rights of individuals to receive information and of &#39;commercial speech&#39; and undermines the democratization of health care and patient empowerment.
Also, they were concerned that this kind of overcautious regulation might stifle the consumer genomics field itself and insisted that the agency should not restrict DTC genomic testing without empirical evidence of harm.
However, rather ironically, they ended up emphasizing the PGS&#39;s poor clinical utility by saying that health-related interpretations of the PGS, particularly about common genetic variants, relate only indirectly to preventing or diagnosing disease [4].
If their comment is true, what is the point of taking the test?
It is simply not ethical to entice people to pay for useless products.
One non-profit technology policy think tank, called TechFreedom, launched an online petition asking the FDA to reverse the ban and collected a total of 10,073 signatures on January 18, 2014 [15].
Many petitioners said that their genetic codes belong to them, so they have the &#34;right to know&#34; what is written there, and that the assumption that they should be &#34;protected from knowing&#34; because they are not smart enough to understand is simply insulting.
All of these concerns are understandable, but we insist that genetic testing with scientific flaws itself is a great harm to consumers and that those flaws need to be addressed before discussing the protection of anyone&#39;s right to know or not to know.
Of course, there are many people who support the FDA&#39;s action as protection of consumers from potentially invalid and almost useless information.
Among the professional proponents, Baudhuin emphatically wrote in her opinion paper that if it is too burdensome to prove the validity of the information that the DTC testing companies offer, they should not sell the information in the first place [2].
Some consumers are already initiating procedures for class-action lawsuits against 23andMe, claiming that it is selling misleading information [2].
On November 27, 2013, just 5 days after the FDA&#39;s warning letter was sent to 23andMe, the first class-action lawsuit was already filed in the Southern District Court of California [16].
However, even among the supporters, some people saw the FDA&#39;s action as rather unusual, because the decision was largely based on its theoretical harm rather than actual harm, which the FDA has usually been waiting for before acting.
In Korea, Lee and Lee [4] of the KHIDI submitted a report in June 2014 to the PPJRAI, titled &#34;Market Analysis and Strategies for Regulatory Reform in Medical Industry,&#34; which expressed a stance somehow different from that of the US FDA [4].
In this report, they suggested that it will help to expand the PGS market if genetic testing agencies are exempted from providing at least two scientific reports to support or prove the clinical validity of their tests when they make a notification of their testing items for disease prevention to the Korea Center for Disease Control and Prevention [4].
This recommendation seems to be in opposition to the US FDA&#39;s.
Also, they recommended an amendment to the Bioethics and Safety Act, which will allow non-medical institutions, such as biotechnology companies, to perform health-related PGSs, at least for disease prediction and prevention (not diagnosis) [4].
However, they added that this predictive, preventive genetic testing should be performed under the supervision of physicians, like the US FDA insists.
Since these recommendations are supposed be delivered to the ministries involved in regulating genetic testing, we may guess which direction our governmental regulation system will take in the near future.
23andMe is now working to get FDA approval for its health tests; the company submitted seven health-related tests to the FDA for review in July 2014; it plans to submit 100 additional tests before the end of the year [17].
However, it is very likely that this process will take years.
However, they have been trying different strategies, including finding new markets.
From October 2014, 23andMe has begun selling its health-related genetic tests in Canada, despite still being blocked by the US FDA from offering its health risk-related service [18].
Downing and Ross wrote that sales of DTC genetic tests, including 23andMe&#39;s PGS, could be resumed only if reliable data have been accumulated that can demonstrate that the tests have positive clinical validity and can be used safely without professional supervision [19].
As we mentioned above, this task is easier said than done-it can be practically impossible.
Even the same authors actually emphasized the importance of the physician&#39;s role in the same article, which can &#34;contextualize&#34; test results for maximizing clinical benefit, preventing adverse effects, and genuinely empowering patients [19].
Zettler et al.
[20] wrote that the FDA&#39;s warning to 23andMe may effectively mark the end of the DTC genetic testing field, at least in the United States, if 23andMe&#39;s PGS cannot meet the FDA&#39;s standards for clinical validity.
DTC genetic testing companies, such as Navigenics and deCODEme, were already sold to other firms and stopped their health-related services in 2012 [20].
The problem is that no one knows, at the moment, how strict the FDA&#39;s standards for clinical validity will be.
Title: Another question will definitely be this: what will happen to whole-genome and exome sequencing, which is already replacing single-gene tests or SNP chips, in terms of clinical application?
Interestingly, just 3 days before warning 23andMe, the FDA authorized Illumina to market four &#34;next-generation&#34; gene sequencing devices: a sequencing platform, two assays-the MiSeqDx Cystic Fibrosis 139-Variant Assay and MiSeqDx Cystic Fibrosis Clinical Sequencing Assay-and a &#34;universal kit&#34; that allows laboratories that purchase the platform to develop and validate sequencing of any part of a patient&#39;s genome and to find any differences between the patient and the reference [21].
What made the difference in the fate of these two companies?
We have not made an inquiry about this directly to the US FDA but can get some clues from the FDA&#39;s press release about this authorization.
First of all, it said that these assays will check &#34;specific points in the patient&#39;s CFTR gene sequence&#34; to detect known and unknown variants.
Their reports will include the existence of genetic variants in the gene sequence that are known to be associated with one specific disease but will not provide medical interpretations of the findings.
Second, it did not mention non-patient customers at all.
Finally, the press release also emphasized the clinician&#39;s involvement.
It said that &#34;this information-rich technology is becoming more accessible for use by physicians in the care of their patients&#34; and that &#34;the new technology also gives physicians the ability to take a broader look at their patients&#39; genetic makeup and can help in diagnosing disease or identifying the cause of symptoms&#34; [21].
We believe that the US FDA&#39;s authorization of the MiSeqDX for cystic fibrosis reflects the agency&#39;s attitude towards any tests involving the human genome.
Therefore, from this, we can guess the answer to the question we posed above: any genetic/genome/exome tests will be required to prove their analytical and clinical validity and clinical utility for specific diseases and conditions if they are intended for medical applications.
Until then, we suggest that their application should be restricted to research applications and family studies for diagnosing genetic disorders of unknown etiology.
When a human being is involved, we believe that the most important ethics principle is &#34;Do no harm.&#34;
Title: Review on Molecular and Chemopreventive Potential of Nimbolide in Cancer

Cancer is the most dreaded disease in human and also major health problem worldwide.
Despite its high occurrence, the exact molecular mechanisms of the development and progression are not fully understood.
The existing cancer therapy based on allopathic medicine is expensive, exhibits side effects; and may also alter the normal functioning of genes.
Thus, a non-toxic and effective mode of treatment is needed to control cancer development and progression.
Some medicinal plants offer a safe, effective and affordable remedy to control the cancer progression.
Nimbolide, a limnoid derived from the neem (Azadirachta indica) leaves and flowers of neem, is widely used in traditional medical practices for treating various human diseases.
Nimbolide exhibits several pharmacological effects among which its anticancer activity is the most promising.
The previous studies carried out over the decades have shown that nimbolide inhibits cell proliferation and metastasis of cancer cells.
This review highlights the current knowledge on the molecular targets that contribute to the observed anticancer activity of nimbolide related to induction of apoptosis and cell cycle arrest; and inhibition of signaling pathways related to cancer progression.
Cancer is the most dangerous disease of human and a major health problem worldwide [1].
Preclinical studies have shown that cancer is a large group of diseases characterized by the uncontrolled cell growth and spread of abnormal cells.
Over the past quarter of a century, an outstanding progress in understanding of the proteins involved in cancer progression has grown, providing chances for identifying new targets for anti-cancer therapy [2, 3].
Present modes of treatment based on synthetic drugs have limited potential, because they are toxic and expensive and also alter cell signaling pathways.
Natural drugs that are safe, affordable, and effective are needed to control cancer development and progression.
Natural products have been used for thousands of years in the management of several diseases including various types of cancer [4].
Nimbolide was first derived from the leaves and flowers of neem.
Neem (Azadirachta indica), is a traditional medicinal plant of the Meliaceae family widely distributed in Asia, Africa and other tropical parts of the world.
All parts of the neem tree offer amazing potential for medicinal, agricultural and industrial exploitation and have been evaluated for antiinflammatory, antipyretic, antihistamine, antifungal, antitubercular, antiprotozoal, vasodilatory, antimalarial, diuretic, spermicidal, antiarthritic, insect repellent, antifeedant, and antihormonal activities [5].
Studies of extracts from all major parts of neem plant including the leaves, flowers, fruits, and seeds, have shown promising chemopreventive and therapeutic effects in pre-clinical research [6].
Extracts of neem leaf have been reported to be non-toxic and non-mutagenic and are found to possess immunomodulatory as well as anti-inflammatory and anticarcinogenic properties [7].
There are many studies showing the ethanolic extract of neem leaves to possess anticancer activity.
Ethanolic neem leaf extract (ENLE) exhibited anticancer activity against N-methyl-N&#39;-nitro-N-nitrosoguanidine-induced oxidative stress and gastric carcinogenesis [8].
ENLE induces apoptosis in a prostate cancer cell line (PC-3) by up-regulating the pro-apoptotic protein Bax and decreasing the level of Bcl-2 protein resulting in DNA fragmentation in prostate cancer cells [9, 10].
Many bioactive compounds are isolated from this plant among which, nimbolide belongs to the limonoid group.
It is the major component of the leaves of A. indica.
Nimbolide (5,7,4&#39;-trihydroxy-3&#39;,5&#39;-diprenylflavanone), is a tetranortriterpenoid with a, β-unsaturated system and δ-lactonic ring (Fig.1) [11].
It has been shown to exhibit numerous types of biological activities, including, anti-malarial [12], anti-bacterial activity [13], anti-feedent, [14] and anticancer activities [15, 16, 17, 18, 19].
Literature evidence reveals that α, β-unsaturated ketone structural element is responsible for the anticancer activity of nimbolide [20, 21].
Cancers are abnormal cell growth caused by genomic modification.
So, any agent that has anti-cancer activity either protects genetic material from alterations or kills the genetically altered cancer cells.
The active component nimbolide from neem acts on cancer cells and kill them by altering the several molecular pathways.
The cytotoxicity of nimbolide has been widely studied over the last several years in a large variety of cancer cell lines [22].
It was reported recently that nimbolide inhibited cancer progression by influencing multiple mechanisms, including prevention of procarcinogen activation and oxidative DNA damage, upregulation of antioxidant and induction of apoptosis, inhibition of tumor cell proliferation, invasion, angiogenesis, and metastasis [19, 23, 24, 25, 26].
The potential mechanisms of cancer prevention by nimbolide are described below and summarized in Fig.2.
Tumorigenesis and cancer progression are thought to be the result of some changes in different types of genetic pathways [27, 28].
Nimbolide, a chief constituent of neem, shows a vital role in cancer prevention and treatment through the modulation of various biological activities, including molecular cascades.
However, understanding the mechanism of action of nimbolide in the activation or inactivation of genetic pathways will provide significant information to develop therapeutic approaches to manage various types of cancers.
Nimbolide induces in vitro cytotoxic activity against human cancer cell lines [20].
Nimbolide decreased cell viability, with an IC50 ranging from 4 to 10 µM and averaging 6 µM for the neuroblastoma (NE-115) and osteosarcoma (143B) cell lines [15].
Sastry et al.
[21] have tested the in vitro cytotoxicity of nimbolide against a panel of human cancer cell lines.
Treatment of cells with 0.5-5.0 µM nimbolide resulted in growth inhibition of the U937, HL-60, THP1, and B16 cell lines.
It has been shown to interfere with the expression of cell survival proteins (Bcl-2, Bcl-xL, IAP-1, and IAP-2), proliferation (cyclin D1), invasion (matrix metalloproteinase [MMP]-9), and angiogenesis (vascular endothelial growth factor [VEGF]) by inhibiting inhibitor of κb (Iκb) kinase (IKK)/nuclear factor kappa B (NF-κB) [16].
Nimbolide treatment to human colon carcinoma (HT-29) cells at 2.5-10 µM resulted in moderate to very strong growth inhibition [29].
Nimbolide significantly inhibited cell viability, with IC50 values of 4.0 µM and 2.7 µM for 24 and 48 h, respectively, in MCF-7 cells and 6.0 µM and 3.2 µM for 24 and 48 h, respectively, in MDA-MB-231 cells [30].
The insulin-like growth factor (IGF) system is involved in the proliferation, survival, and migration of tumor cells.
IGF-I receptor (IGF-IR) is a receptor tyrosine kinase that is overexpressed in about 70% of breast cancers [31, 32].
It is established that the consequences of IGF-IR activation by its ligands result in the recruitment of major adapter signaling proteins, such as src/collagen homology proteins, which lead to interaction with Grb2/Sos [33].
Recruitment of these molecules activates two distinct signal transduction pathways.
One pathway activates Ras, Raf, and mitogen-activated protein kinase (MAPK), resulting in the transcription of genes that drive proliferation, and the other pathway involves phosphoinositide 3-kinase (PI3K)/Akt, which are responsible for cell survival and anti-apoptotic signal transduction [34].
Downregulation of these molecules will give rise to inhibition of cell survival, invasion, and induction of apoptosis.
Aberrant activation of PI3K/Akt signaling has been associated with the development and progression of many cancers [35], making this pathway an attractive target for therapeutic strategies.
The increase in p-Akt level in breast tumor samples correlates with a poor prognosis [36, 37] and predicts a worse outcome among endocrine-treated patients [38].
The major inhibitor of Akt is the dual-specificity protein and lipid phosphatase, phosphatase tensin homolog deleted on chromosome 10 (PTEN) that counteracts the PI3K-dependent Akt activation by dephosphorylating the D3 position of PIP3 [39].
PTEN activity is frequently lost by mutations, deletions or promoter methylation silencing in many primary and metastatic human cancers [40, 41].
The inhibition of PI3K/Akt and activation of the PTEN pathway constitute a good strategy in the prevention of cancer.
The recent findings of Karkare et al.
[42] proved that nimbolide suppresses glioblastoma viability and detains tumor growth by inhibiting CDK4/6 activity, leading to retinoblastoma hypophosphorylation and cell cycle arrest and by inhibiting growth factor pathways that are hyperactivated in glioblastoma, including the PI3K-Akt, MAPK, and JAK-STAT pathways [42].
It has been reported that decreased IGF-IRβ expression and increased IGFBP-3 expression are observed in nimbolide-treated breast cancer cells.
The protein expression of the IGF signaling molecules IRS-2, PI3K, pAkt, Ras, Raf, MEK, and MAPK was significantly decreased in nimbolide-treated cells.
This is further supported by a significant increase in the protein expression of PTEN upon nimbolide treatment in breast cancer cell lines (Fig.3).
Thereby, nimbolide inhibits the cell survival and proliferation of breast cancer and prostate cancer cells [19, 43].
NF-κB comprises a family of transcription factors involved in the regulation of a wide variety of biological responses.
NF-κB has a well-known function in the regulation of immune responses and inflammation, but growing evidence supports a major role in oncogenesis.
NF-κB regulates the expression of genes involved in many processes that play a key role in the development and progression of cancer, such as proliferation, migration, and apoptosis [44, 45].
This transcription factor is localized in the cytosol and is blocked by Iκb.
Activation of NF-κB may result from different signaling pathways triggered by a variety of cytokines, growth factors, and tyrosine kinases [44].
Recently, it was studied that nimbolide exerts potent anticancer effects in HepG2 cells by inhibiting NF-κB activation and its downstream events, such as activation of the Wnt/β-catenin pathway and apoptosis evasion.
Inhibitors of NF-κB which can block several signalling pathway, have developed as a successful candidates for novel anti-cancer regimens.
Thus, nimbolide, by targeting multiple components of the NF-κB signaling pathway to inhibit tumor progression, is a promising agent for cancer prevention and therapy [17].
Gupta et al.
[16] found that suppression of NF-κB activation by nimbolide was caused by inhibition of IKK, which led to suppression of IκB phosphorylation and degradation, nuclear translocation, DNA binding, and gene transcription in myeloid and leukemic cells.
Nimbolide significantly decreased the protein expression of IKKα, IKKβ, and NF-κB in breast cancer cell lines [46].
Metastasis is the process by which a cancer cell leaves the primary tumor, travels to a distant site via the circulatory system, and begins a secondary tumor.
In order to metastasize, cancer cells must invade through the basement membrane and the extracellular matrix (ECM).
Proteolysis of the ECM is an important step in metastasis, and the process is associated with the upregulated production and activity of several ECM-degrading proteases.
The matrix metalloproteinases (MMPs) constitute a family of structurally related, zinc-dependent endopeptidases that are capable of degrading the protein components of the ECM and basement membrane and release growth factors from ECM stores [47, 48].
MMP activity is regulated by specific inhibitors, the tissue inhibitors of MMP (TIMPs).
Activation of chemokines and the urokinase plasminogen activator (uPA)/uPA receptor (uPAR) system is also a central mediator of tumor-cell migration and invasion, and IGF-I signaling can influence the uPAR pathway.
Chemokines are chemotactic cytokines that cause the directed migration of leukocytes and are induced by inflammatory cytokines, growth factors, and pathogenic stimuli.
Chemokine signaling results in the transcription of target genes that are involved in cell invasion, motility, and survival.
The upregulation of chemokine molecules in tumor biology as compared with &#34;normal&#34; cells confers chemokines as &#34;a magic bullet,&#34; as targeting them can potentially hit cancer cells and their metastasis, leaving non-affected cells unharmed [49].
The expression of the chemokines CCL2, CXCL12, and CXCL8 and their receptors CCR2, CXCR1, CXCR2, and CXCR4 is significantly decreased in nimbolide-treated breast cancer cell lines [46].
uPA is a serine protease that is involved in cancer progression, mainly invasion and metastasis [50].
uPA can be observed as a multifunctional protein that is involved in both proteolysis and signal transduction [51].
Recently, nimbolide has been shown to interfere with the expression of NF-κB-regulated proteins, like Bcl-2, cyclooxygenase 2, MMP-9, and VEGF, by inhibiting IKK.
Therefore, it is likely that nimbolide will interfere with cell migration invasion and angiogenesis [29].
The neem limonoids azadirachtin and nimbolide inhibit hamster cheek pouch carcinogenesis by prevention of procarcinogen activation and oxidative DNA damage, upregulation of antioxidants and carcinogen detoxification enzymes, and inhibition of tumor invasion and angiogenesis [24].
The protein expression of MMP-2 and -9 is significantly decreased, and TIMP-2 expression is increased in nimbolide-treated breast cancer cells [46].
Other studies also reported that the protein expression and activity of MMP-2 and -9 were significantly decreased in myeloid leukemia cell lines [16] and colon cancer cells [29].
Limonoid-treated xenografts exhibited significant down-regulation in the expression of proteins involved in tumor cell survival (Bcl-2, Bcl-xL, c-IAP-1, survivin, and Mcl-1), proliferation (c-Myc and cyclin D1), invasion (MMP-9, intercellular adhesion molecule 1), metastasis (CXCR4), and angiogenesis (VEGF) [26].
The migration and invasive potential of the MCF-7 and MDA-MB-231 cell lines were considerably suppressed upon nimbolide treatment.
The results showed that nimbolide downregulated the expression of uPA, uPAR, chemokines, phopho-epidermal growth factor receptor, vascular endothelial growth factor, NF-κB, IKKα, IKKβ, MMP-2, and MMP-9 and upregulated the expression of TIMP-2, suggesting that nimbolide inhibits angiogenesis and metastasis of breast cancer [46].
The loss of the ability to regulate the cell-cycle is characteristic of cancer cells and results in uncontrollable proliferation.
Processing cells through the first gap (G) phase of the cell cycle is a step that is frequently disordered in cancer [52].
Nimbolide has been investigated in different studies for its ability to mediate cell cycle arrest.
In many in vitro and in vivo studies, nimbolide has shown cell cycle-regulatory effects [18, 25, 53, 54].
It has been reported that nimbolide inhibits cell proliferation by interfering with cell cycle kinetics by inducing G0/G1 and S phase arrest, primarily caused through the repression of cyclin A/cyclin D1 [16, 54, 55].
In another interesting molecular study, a Japanese group has proved that nimbolide, a triterpenoid present in certain edible parts of A. indica, arrested HT-29 cells in the G2/M and G0/G1 stages apparently through upregulation of p21, which is a well-known downstream effector of the p53.
p53 is a very important anticancer protein that regulates a large number of genes that are involved in cancer progression.
Nimbolide has also been shown to upregulate cyclin D2 and CDK2 and to suppress the expression of cyclin A, cyclin E, CDK2, and Rad17 at the same time [53].
Flow cytometric analysis of U937 cells showed that nimbolide treatment (1-2.5 µM) resulted in cell cycle disruption by decreasing the number of cells in G0/G1 phase, with initial increases in S and G2/M phases.
It is shown that nimbolide can affect cell cycle progression and induce apoptosis in colon cancer, oral carcinoma, and cervical cancer [18, 25, 55].
Nimbolide significantly suppressed the viability of HeLa cells in a dose-dependent manner by inducing cell cycle arrest at G0/G1 phase, accompanied by p21 accumulation and down-regulation of the cell cycle regulatory proteins cyclin B, cyclin D1, and proliferating cell nuclear antigen (PCNA) [55].
Nimbolide treatment results in the accumulation of cells in G0/G1 phase and decreased in S-phase by up regulating p21 and downregulating the cell cycle-regulatory proteins cyclins and PCNA.
Cyclin D1 is known as a proto-oncogene whose gene amplification and protein overexpression of which are frequently observed in tumor cells.
The activated cyclin D1/CDK4 and cyclin D1/CDK6 complex phosphorylates the retinoblastoma protein to induce the expression of target genes essential for S phase entry, facilitating the progression from G1 to S phase [56].
Cyclin B1 is a G2/mitotic-specific protein that plays a role in the initiation of mitosis and tumorigenesis [57].
It was reported that cyclin B1 depletion inhibits proliferation and induces apoptosis in human tumor cells [57].
Nimbolide treatment decreases cyclin (A1, B1, C, D1, and E1) expression in breast cancer cells.
p21Waf1/Cip1, originally identified as an inhibitor of the cyclin/CDK complexes, has also been shown to have a role as an adaptor protein that assembles and promotes the kinase activity of cyclin D/CDK4 complexes [58].
The level of p21 was significantly increased in nimbolide-treated breast cancer cell lines.
PCNA, a cofactor for DNA polymerase δ, plays a central role in cell cycle progression [59].
PCNA is involved in a wide range of cellular functions, including DNA replication, repair, and epigenetic maintenance, and is often used as a diagnostic and prognostic marker.
The protein expression of PCNA is decreased in nimbolide-treated breast cancer cells [19].
Nimbolide directly inhibited CDK4/CDK6 kinase activity, leading to hypophosphorylation of the retinoblastoma protein, cell cycle arrest at G1-S, and cell death [42].
In animal tumor models, nimbolide (100 µg/kg) has been shown to exhibit chemopreventive activity against 7,12-dimethylbenzanthracene (DMBA) 3-induced hamster buccal pouch carcinogenesis by downregulating proteins involved in cell cycle progression and transduce apoptosis by both the intrinsic and extrinsic pathways [25].
Apoptosis, or programmed cell death, is essential for the maintenance of development and homeostasis of multi-cellular organisms by eliminating superfluous or unwanted cells.
Any alteration or change in the normal process of apoptosis may increase cell survival and support tumor development and progression [60].
The extrinsic and intrinsic pathways represent the two major well-studied apoptotic processes.
Inefficient apoptosis is considered one of the hallmarks of tumorigenicity [61].
Moreover, induction of apoptosis is an important target for cancer therapy [62].
The extrinsic pathway is initiated by cell surface-expressed death receptors of the tumor necrosis factor superfamily.
One of the central pathways of apoptosis is initiated by cytokines, such as tumor necrosis factor-α, Fas ligand (FasL), and tumor necrosis factor-α-related apoptosis-inducing ligand (TRAIL) [63].
The intrinsic pathway is initiated by anticancer drugs, growth factor withdrawal, or hypoxia or via induction of oncogenes.
These stimuli induce permeabilization of the outer mitochondrial membrane and activate the mitochondrial pathway [64].
Caspases are a family of evolutionarily conserved cysteine proteases that play a essential role in the majority of apoptotic pathways.
Death signals activate the proteolytic cascade of caspases through two main pathways: an extrinsic and intrinsic pathway [65].
Both pathways converge to the activation of caspase-3, the closer homolog of Caenorhabditis elegans CED-3 [66].
Caspase-3 activates downstream enzymes of the caspase family and contributes with them to generate the typical apoptotic cell death phenotype [67].
Harish Kumar et al.
[25] reported that nimbolide transduces apoptosis by both the intrinsic and extrinsic pathways in DMBA-induced hamster buccal pouch carcinogenesis.
Working on the choriocarcinoma (BeWo) cells, have shown that nimbolide, induces apoptosis through engagement of the mitochondrial pathway.
The involvement of this pathway is based on the observation that nimbolide mediates the upregulation of Apaf1 and a caspase-3 and decrease in the Bcl2/Bax ratio [54].
Nimbolide sensitizes human colon cancer cell lines (HCT-116 and HT-29) to TRAIL through reactive oxygen species- and Erk-dependent up-regulation of death receptors, p53, and Bax.
A normal breast cell line (MCF-10A) and breast cancer cell lines (MCF-7) were treated with nimbolide (1-5 µM) for 6 h, followed by TRAIL for 24 h. The results indicated that whereas nimbolide and TRAIL alone were minimally effective in inducing apoptosis in MCF-7 cells, the combination of both enhanced the number of apoptotic cells to 42%.
Conversely, the combination of nimbolide and TRAIL was unable to evoke apoptosis in the normal breast cell line.
The results showed a lack of DR5 and DR4 induction in MCF-10A cells by nimbolide, whereas a dose-dependent induction of these receptors was observed in MCF-7 cells [68].
The decreased expression of Bcl-2 and increased expression of bax, cytochrome c, Smac, and caspases, together with changes in nuclear morphology and mitochondrial transmembrane potential, seen in nimbolide treatment and inhibition of NF-κB activation by nimbolide, stimulate the intrinsic apoptotic pathway in HepG2 cells [17].
In another study, nimbolide repressed cell proliferation by inhibiting the IGF1/IGF-IR-PI3K/Akt pathway and induced apoptosis through the activation of both the extrinsic and intrinsic apoptotic pathways in prostate cancer cells [43].
Nimbolide also induced apoptosis in breast cancer cells by upregulating pro-apoptotic protein (Bad, Bax, FasL, FADD, TRAIL, and cytochrome c) expression and down-regulating anti-apoptotic proteins (Bcl-2, Bcl-xL, Mcl-1, and XIAP-1).
Nimbolide activates caspase-3, -8, and -9, which favors the cleavage of poly(ADP) ribose polymerase into 115- and 85-kDa peptides, thus inducing apoptosis in breast cancer cell lines.
Further, the nimbolide-induced apoptotic cells were detected using DAPI and AO/EtBr dual staining.
After treatment with nimbolide, the cells exhibited the typical morphological changes associated with apoptosis: cell shrinkage, nuclear condensation, and membrane blebbing [30].
Activation of caspase-3, -8, and -9, suggests that nimbolide potentiated both the extrinsic and intrinsic pathways of apoptosis in human breast and colon cancer cells [68, 30].
The overview of all of the signaling molecules modulated by nimbolide is shown in Fig.4.
Cancer has continually been the leading cause of death worldwide for decades.
Thus, researchers have actively devoted themselves to studying cancer therapeutics.
The present mode of treatment based on chemotherapy and radiotherapy is very expensive and also exhibits serious side effects in human beings.
Keeping in view the significance of herbs, this review is written to show the role of nimbolide in the prevention of various types of cancer through the activation or inactivation of various signaling pathways.
These reported features, combined with the absence of side effects and being inexpensive and easy to access, neem and its constituent nimbolide may be proved very effective therapeutics in the management of cancers.
Nimbolide is a potential chemopreventive agent that is able to suppress multiple signaling pathways involved in carcinogenesis and hence is an attractive candidate for further research.
Currently, nimbolide is undergoing extensive research to determine the dose efficacy and administration form that could enhance its bioavailability and chemopreventive properties.
How nimbolide mediates all of these effects is not completely understood.
However, most of the anticancer activities assigned to nimbolide have been based on in vitro studies.
While in vitro studies predominantly established the anti-cancer effects of nimbolide on cancer cells, its efficiency in vivo has still to be proven.
Structure of nimbolide.
Molecular formula, C27H30O7; molecular weight, 466.5; appearance, white to off-white solid.
Potential mechanisms of cancer prevention by nimbolide.
Detailed scheme of the growth factor signalling pathways targeted by nimbolide in cancer cells.
Nimbolide strongly inhibits IGF-IR and also affects downstream signalling via the MAPK (ERK) and PI3K/Akt.
Transcription factors like NF-κB, c-Myc, and β-catenin consequently cannot be activated, thus modulating target gene expression.
As most of the target genes are implicated in cell cycle regulation and proliferation, the cancer cells are finally blocked in cell cycle progression.
ERK, extracellularsignal-regulated kinase; IGF-IR, insulin-like growth factor 1 receptor; MAPK, mitogen-activated protein kinase; NF-κB, nuclear factor kappa B; PI3K, phosphoinositide 3-kinase; PTEN, phosphatase tensin homolog deleted on chromosome 10.
Nimbolide shows an important role in cancer prevention via inhibition of cell survival, cell cycle progression, invasion, migration and induction of apoptosis.
The diverse molecular targets influenced by nimbolide include the growth factors and their receptor, signaling molecules, transcription factors, protein kinases, enzymes and genes regulating cell proliferation and apoptosis.
Red down arrow indicates that these molecules are down-regulated by nimbolide, whereas, green up arrow indicate that these molecules are up-regulated by nimbolide.
Bad, Bcl-2-associated death promoter; Bax, Bcl-2-associated X protein; Bcl-2, B cell lymphoma 2; Bcl-xL, B-cell lymphoma-extra-large; DR-5, death receptor-5; EGFR, epidermal growth factor receptor; ERK, extracellular-signal-regulated kinase; FADD, Fas-associated death domain; FasL, Fatty acid synthase ligand; ICAM, Intercellular adhesion molecule; IGF-1, insulin-like growth factor 1; IGFBP-3, IGF-binding protein 3; IKK, IkB kinase; IRS, insulin receptor substrate; MCL-1, myeloid cell leukemia 1; MMPs, matrix metalloproteinases; NF-κB, nuclear factor kappa B; PARP, poly(ADP) ribose polymerase; PCNA, proliferating cell nuclear antigen; PI3K, phosphoinositide 3-kinase; PTEN, phosphatase tensin homolog deleted on chromosome 10; Rb, retinoblastoma; TIMP, tissue inhibitor of matrix metalloproteinase; TRAIL, tumor necrosis factor apoptosis inducing ligand; VEGF, vascular endothelial growth factor; VEGFR, vascular endothelial growth factor receptor; XIAP, X-linked inhibitor of apoptosis protein; uPA, urokinase plasminogen antigen; uPAR, urokinase plasminogen antigen receptor.
Title: Short Reads Phasing to Construct Haplotypes in Genomic Regions That Are Associated with Body Mass Index in Korean Individuals

Genome-wide association (GWA) studies have found many important genetic variants that affect various traits.
Since these studies are useful to investigate untyped but causal variants using linkage disequilibrium (LD), it would be useful to explore the haplotypes of single-nucleotide polymorphisms (SNPs) within the same LD block of significant associations based on high-density variants from population references.
Here, we tried to make a haplotype catalog affecting body mass index (BMI) through an integrative analysis of previously published whole-genome next-generation sequencing (NGS) data of 7 representative Korean individuals and previously known Korean GWA signals.
We selected 435 SNPs that were significantly associated with BMI from the GWA analysis and searched 53 LD ranges nearby those SNPs.
With the NGS data, the haplotypes were phased within the LDs.
A total of 44 possible haplotype blocks for Korean BMI were cataloged.
Although the current result constitutes little data, this study provides new insights that may help to identify important haplotypes for traits and low variants nearby significant SNPs.
Furthermore, we can build a more comprehensive catalog as a larger dataset becomes available.
Genome-wide association studies (GWASs) have been a useful tool to identify genetic variants that affect various traits [1].
Numerous novel important genetic variants associated with disease susceptibility have been identified through GWASs [2].
However, genome-wide association (GWA) results for complex diseases can generally explain only a small part of the genetic variants for complex diseases [3].
Generally, a GWA study identifying an association between a trait and a genetic variation may be limited in understanding complex diseases involving multiple functional loci [4].
Recently, many approaches to detect an association between a trait and one or multiple genetic variants have been proposed to study numerous data from GWASs [5].
Inference about linkage disequilibrium (LD), known as non-independent association of alleles at different loci, provides information for the association of genetic variants affecting complex traits [6].
Using LD analysis, it is possible to characterize multiple genomic variants associated with phenotypes in terms of haplotypes.
A haplotype defines a combination of phased alleles in a chromosomal region [4].
Thus, haplotype analysis is useful to understand multiloci genetic associations and to identify susceptibility loci for diseases.
In spite of the advantages of haplotype analysis, performing an analysis on a genome-wide scale is not simple, due to the uncertainty and complexity of haplotypes [7].
Whole-genome sequencing (WGS) of a reference population may provide insights into the potential causal variants hidden within the LD block of interest.
While the 1000 Genomes Project aims to provide such information in worldwide populations or ethnic groups [8], it would be more informative to focus on a reference population on a national scale than on a global scale.
Recently, several Korean reference WGS datasets have been published based on next-generation sequencing (NGS) platforms [9].
Here, we tried to construct haplotype blocks affecting body mass index (BMI) in Korean individuals through integrative analysis of NGS and GWA data.
We analyzed haplotypes using NGS data only in ranges that were nearby significant single-nucleotide polymorphisms (SNPs) that were identified by GWA results.
The analysis may have several advantages in haplotype analysis.
The complexity and the time can be reduced by analyzing smaller regions that are known to harbor significant GWA signals.
We cataloged all possible 44 haplotype blocks within the LD blocks of significant SNPs for BMI.
Title: Overall pipeline of the analysis

The NGS raw data were mapped to reference sequences using an alignment tool.
The significant SNPs for BMI were selected from the GWA result data.
The LD blocks that encompassed the significant SNPs were searched with Haploview [10], and those NGS reads that were mapped to the LD ranges were selected with Samtools.
For each LD block, the heterozygous variants from the short reads were phased with Samtools.
Finally, the phased haplotypes harboring the significant SNPs were cataloged (Fig.1).
Title: Genotyping, imputation, and GWA analysis

The Korean samples and genotype data used in this study have been described by Cho et al.[11].
Briefly, through the Korea Association Resource (KARE) project, a total of 10,038 participants were recruited from Ansan and Anseong population-based cohorts, aged 40 to 69.
The genotypes that had been measured on Affymetrix Genome-Wide Human SNP array 5.0 (Affymetrix, Santa Clara, CA, USA) were filtered for quality control, resulting in 8,842 samples and 352,228 markers.
The genotype data were expanded to a total of 1,827,004 SNPs through imputation with IMPUTE [12] using the Japanese in Tokyo, Japan (JPT)/Han Chinese in Beijing, China (CHB) component of HapMap as the reference.
BMI traits were tested for association by linear regression analysis with an additive model after adjustments for recruitment region, age, and sex as covariates using PLINK.
The threshold of genome-wide significance was set at p &lt; 5 × 10-4.
The significant SNPs were clustered into LD blocks (r2 &gt; 0.9) using HaploView [13].
Title: NGS data analysis

We downloaded previously published whole-genome NGS data from the TIARA database, sequenced at Seoul National University College of Medicine (TIARA; http://tiara.gmi.ac.kr) [14].
The short reads from seven samples (AK3, AK4, AK5, AK6, AK7, AK14, and AK20) were mapped to the human reference genome (hg19) using the alignment algorithm in Genomic Short-read Nucleotide Alignment Program (GSNAP) [15].
From the resulting BAM file, we selected the mapped reads within the LD blocks that encompassed the significant SNPs from the BMI GWA result using the Samtools view function [16], creating a separate BAM file for each LD block.
Title: Heterozygous SNP calling and phasing

The heterozygous SNPs were phased using the phase function in Samtools, version 0.1.19-44428cd [17].
The Samtools phase algorithm calls heterozygous SNPs automatically and phases those that segregate on the same DNA fragment as inferred from paired-end read information in the BAM file.
We used the default options for the Samtools phase function.
We confirmed the heterozygous SNPs through independent calling using the Samtools mpileup function [17].
Whenever an LD block split into multiple phased haplotype blocks, we kept those that encompassed the significant SNPs from the GWAS.
Title: GWA analysis

The genotype data of 8,842 samples in a total of 10,038 participants in the KARE were analyzed to find significant SNPs associated with BMI in Korean individuals.
Among the total of 1,827,004 loci, 435 loci were identified using a p-value cutoff (p &lt; 5 × 10-4) from the GWAS.
The default algorithm in the HaploView program for identifying LD blocks was taken from confidence intervals [13].
We set a ±20-kb window option and r2 &gt; 0.9 to capture LD blocks of 435 significant SNPs based on the KARE genotype data.
For 435 loci that were significant in the association analysis (p &lt; 5 × 10-4), 53 LD blocks were identified using HaploView (Table 1).
These 53 LD blocks contained all 435 significant loci.
Thus, a typical LD block included multiple significant loci.
Title: Phasing short reads

We downloaded whole-genome paired-end NGS data of seven samples from the TIARA database to discover haplotypes with the SNPs associated with BMI within the 53 LD blocks from the GWA analysis.
We mapped the short reads to the human reference genome (hg19), and those reads with bad mapping quality (mapping quality &lt;5) were filtered out.
The average mapping rate was about 93%.
We separated the mapped reads into the 53 LD blocks for each of seven individuals (Table 1).
Each group of separated reads was called for variants, and the heterozygous SNPs were phased using the phase function in Samtools.
Not all of the SNPs in an LD block were always phased into a contiguous haplotype block.
Whenever multiple haplotype blocks were found for a given LD block, we kept only those that encompassed the significant SNPs identified from the GWAS.
Title: Construction of the haplotype blocks

After phasing heterozygous SNPs using Samtools, we searched for phased haplotypes that harbored significant risk alleles for BMI from the GWA analysis results.
We found 44 haplotype blocks in 23 of 53 LD blocks.
We could not find haplotypes with significant risk alleles in the rest of the LD blocks.
We ordered 44 haplotypes and established a catalog for possible haplotype blocks associated with BMI variations in the Korean population (Table 2).
For example, AK4 and AK7 share the TCTGAGCC haplotype, which comprises the variants at bps 246142250, 246142279, 246144436, 246146137, 246146178, 246146444, 246148791, and 246149166 in chromosome 1.
Bp 57895600 in chromosome 18 has been previously known as rs8089366 (G/T) in the dbSNP database and was a significant SNP for BMI in KARE.
On the other hand, bp 57900630, which participated in the same haplotype, has not been registered in the dbSNP database.
In Table 2, 16 LD blocks were shared by several individuals, while the rest of the LD blocks were carried by single individuals.
Title: Distribution of haplotype blocks

We calculated the average space between alleles in a haplotype block from our results.
For example, AK4, AK5, and AK7 share CGCC at bps 182065282, 182065314, 182065341, and 182065351 in chromosome 4 (Table 2).
The haplotype block ranges from bp 182065282 through bp 182065351; the length of the haplotype is 69 bp.
Since there are four SNPs, the average space is about 17.
We plotted the distribution of the average spaces between SNPs within a haplotype (Fig.2).
The number of alleles found for a haplotype is also plotted in Fig.2.
The GWAS is a powerful method to identify genomic variants affecting traits.
Many studies have found important SNPs associated with diseases.
However, the study is not well suited to identify rare variants or small effects of several SNPs [18].
Here, we tried to find haplotype blocks in 7 Korea individuals that harbor SNPs that are significantly associated with BMI in a Korean population.
The significant SNPs for BMI were detected through GWASs, and the haplotypes were discovered by analyzing the NGS data.
We selected short reads within the LD ranges that encompassed the significant SNPs from the GWAS results, and the heterozygous variants that were called from those reads were phased, based on the paired-end information.
As a result, we detected 44 haplotype blocks harboring SNPs that are significantly associated with BMI in Koreans.
The haplotypes may affect BMI in Korea individuals.
However, it is not statically powerful, due to the small number of samples (7 individuals) used in this study.
Actually, the alleles of a haplotype need to be compared with those of the haplotype of the same range in other individuals to have high accuracy.
We could not follow this strategy.
For example, most of the haplotypes found by this analysis were discovered in a single individual.
In our small dataset, a significant SNP from a GWAS is not likely to be shared by several individuals, because many SNPs have a minor allele frequency of around 10% in a population.
This problem may be solved if a large dataset is used.
In addition, read depth also reduces accuracy.
In Table 2, several haplotypes were identified in a single GWA LD block.
These haplotypes could not be joined, as there were not enough short reads that could bridge them using the phase function in Samtools.
In addition, AK7 has the GATX haplotype at bps 57903745, 57904011 (rs590654), 57904088 (rs590215), and 57907787 in chromosome 18, and AK4 has XAXC in the same locus.
An X represents a missing allele, probably due to low coverage.
We were able to recover them, as Samtools was still able to phase them.
We filled in these alleles manually to build the haplotype.
In Table 2, the green letters represent such alleles that have been filtered by read depth and recovered manually.
Although the accuracy is not high, this integrative method has several advantages.
First, rare variants can be detected.
For example, AK4 has the TT risk allele haplotype at bps 57895600 and 57900630 in chromosome 18.
The former was associated with BMI and was registered in the dbSNP database, while the latter was not registered in the dbSNP database.
Bp 57900630 is possibly a low-frequency risk allele, which is difficult to be detected by GWAS.
Second, we can explain a trait systematically.
rs559623 is an SNP (C/A: forward strand) that is associated with BMI.
It may act with bp 57844375 in chromosome 18 as CC according to our results (Table 2).
Traits can be explained properly by multiple variants.
Finally, this method is useful for studying phasing.
Many SNPs from an array or NGS reads are unphased genotypes.
However, the bulk of SNP information needs to be phased for identifying co-located alleles.
While GATK HaplotypeCaller, Samtools Phase, and Beagle are used to phase variants, there are several problems, such as the long execution time and the need for large system memory.
In fact, it takes too long to phase entire chromosomes or a single chromosome.
However, with this method, it is possible to analyze efficiently with regard to time and memory, as it does not consider reference and genotype data of the entire chromosome.
This method focuses only on the data within regions that are nearby important variants from the GWA analysis results.
We constructed a crucial haplotype catalog for BMI traits in Korean individuals by integratively analyzing NGS data and GWA analysis data.
Pipeline for this study.
First, next-generation sequencing raw data (fastq) are aligned using Genomic Short-read Nucleotide Alignment Program (GSNAP), and then, the reads without bad mapping quality are selected.
In addition, significant single-nucleotide polymorphisms (SNPs) are learned from the genome-wide association study results, and linkage disequilibrium (LD) ranges with the SNPs are searched by HaploView.
Using Samtools, reads within the LD ranges are selected and phased.
As a result, a haplotype catalog can be constructed, including haplotypes with significant SNPs.
Boxplots of average space between alleles (A) and the number of alleles in haplotype blocks (B).
Title: Representative LD blocks from the GWA resultsa

Title: Haplotypes with significant SNPsa
Title: In silico Identification of SFRP1 as a Hypermethylated Gene in Colorectal Cancers

Aberrant DNA methylation, as an epigenetic marker of cancer, influences tumor development and progression.
We downloaded publicly available DNA methylation and gene expression datasets of matched cancer and normal pairs from the Cancer Genome Atlas Data Portal and performed a systematic computational analysis.
This study has three aims to screen genes that show hypermethylation and downregulated patterns in colorectal cancers, to identify differentially methylated regions in one of these genes, SFRP1, and to test whether the SFRP genes affect survival or not.
Our results show that 31 hypermethylated genes had a negative correlation with gene expression.
Among them, SFRP1 had a differentially methylated pattern at each methylation site.
We also show that SFRP1 may be a potential biomarker for colorectal cancer survival.
The Encyclopedia of DNA Elements (ENCODE) project, launched in 2003 by the US National Human Genome Research Institute (NHGRI), has accelerated the study of various aspects of epigenetics [1], among which DNA methylation has been one of the most actively studied areas [2].
DNA methylation refers to the chemical conversion of a cytosine nucleotide in DNA to 5-methylcytosine by DNA methyltransferase.
This phenomenon usually takes place at the genomic region where a cytosine is immediately followed by a guanine and where the dinucleotide is enriched more than expected from the G + C content of the genome.
Such CpG islands (CGIs) are typically found in the promoter region of a gene, regulating its expression [3].
Tumor suppressor genes (TSGs), including DNA repair genes that repair DNA mismatch during DNA replication, maintain the normal activities of cells, preventing cancer development.
On the other hand, their hypermethylation may repress their expression, influencing the development and growth of cancer cells.
Moreover, TSG expression may cause gene silencing through the hypermethylation of CGIs, and such a paradigm opens up a new avenue of cancer diagnosis and treatment based on DNA methylation profiles [4].
In cancer cells, genes that play important roles in cell growth and differentiation, such as TSG, DNA repair genes, and apoptosis-related genes, show downregulation in expression due to hypermethylation in their promoter regions.
For example, the expression of p16, a well-known TSG, is downregulated due to its promoter hypermethylation [5], and DNA repair genes, such as BRCA1 and hMLH1, are also hypermethylated, repressing their gene expression [6].
It appears that hypermethylation profiles in CGIs may be useful targets for cancer treatment.
Some DNA methylation inhibitors have been used as anticancer agents, paving a new avenue for the development of anticancer therapeutics [7].
Gene expression profiles have been used in cancer diagnosis and prognosis, but mRNA levels usually fluctuate temporally or are greatly influenced by environmental cues, resulting in unstable diagnostic sensitivity.
On the other hand, DNA hypermethylation can show high specificity and sensitivity in cancer diagnosis [8].
Furthermore, its molecular diagnosis is gaining more and more acceptance due to its availability through real-time and quantitative blood testing [9].
In this study, we surveyed colon cancer data to look for genes that show hypermethylation, and investigated their effects on patient survival.
Title: Data

The Cancer Genome Atlas (TCGA), launched in 2009, aimed to analyze the genomic features through sequencing of about 30 different types of cancers.
We downloaded the microarray datasets on gene expression and DNA methylation of colorectal cancer and normal samples from the following URL: https://tcga-data.nci.nih.gov/tcga/dataAccessMatrix.htm (the &#34;Data Type&#34; field was selected as &#34;Clinical,&#34; &#34;DNA Methylation,&#34; and &#34;Expression-Genes&#34;).
There were 12 matched cancer/normal pairs of both data types, and these were defined as the discovery set.
The microarray platforms were UNC_AgilentG4502A_07_3 and Illumina Infinium Human DNA Methylation 27 (Illumina, San Diego, CA, USA) for gene expression and DNA methylation, respectively.
For the methylation data, the so-called M-value was calculated as the log2 ratio of the intensities of the methylated probe versus unmethylated one, and the beta-value was the ratio of the methylated probe intensity and the overall intensity, either methylated or unmethylated [9].
For the clinical outcome analysis, we downloaded an expanded dataset from the URL, totaling 524 samples (202 samples on the Illumina Infinium Human DNA Methylation27 platform and 322 samples on the Illumina Infinium Human DNA Methylation450 platform), and 419 samples with clinical information were used in the survival analysis.
For these samples, we downloaded a text file on tumor stage, last contact days, and vital status.
Title: Identification of hypermethylated and downregulated genes

The discovery dataset was used for gene selection.
Illumina Infinium Human DNA Methylation27 level 2 data were screened for differentially methylated regions (DMRs), where the median M of the tumor samples was greater than +1 and that of the matched normal samples was less than -1 [9].
For each gene, the mean mRNA expression level in tumor samples was compared to that in normal samples.
Genes showing at least 4-fold down-regulation were selected.
DAVID (http://david.abcc.ncifcrf.gov/) was used for the functional annotation of the gene list.
Title: Survival analysis

Survival analysis was undertaken with the survival package in R (version 2.13.0).
Survival time was represented by the days since cancer diagnosis in the hospital until the last contact date.
The survival time of the patients who were still alive at the last contact date were treated as censored.
Kaplan-Meier survival plot and Cox regression analysis were also performed with the survival package.
In order to partition patients into two groups according to methylation level, the cutoff point was determined by the maximally selected rank statistics method, available in the R maxstat package.
Title: Hypermethylated and downregulated genes in colorectal cancer

We used the discovery set (12 pairs of colorectal tumor and matched normal samples) to survey genes showing correlating differential DNA methylation and differential gene expression.
All methylation probes (n = 27,578) were annotated by the name of the neighboring gene and were matched to the gene expression matrix.
In order to look for the probes that were specifically hypermethylated only in tumors, we used the M-value of the methylation data; the median M of tumors (n = 12) was greater than 1, while the matched normal samples (n = 12) had a median M of less than -1.
There were 634 such probes.
We also identified 707 genes showing more than 4-fold downregulation in tumor samples compared to their matched normal samples.
There were 31 genes that were shared by those two lists (Table 1).
Fig.1 shows the scatterplot of the mean differential methylation versus mean differential expression of these 31 genes (red points) on the background of 27,578 methylation probes (grey points).
At the sample level, these 31 genes showed a negative correlation between methylation and expression (Fig.2).
Title: Gene ontology and pathway analysis

Gene ontology analysis using DAVID revealed that the 31 genes were enriched in the Wnt receptor signaling pathway (Fisher exact = 9.6E-5); 4 genes-RSPO2, RSPO3, SFRP1, and SFRP2-belonged to this pathway (Table 2).
Site of Frizzled proteins (SFRP) genes interact directly with Wnt, the ligand of the Wnt signaling pathway Frizzled (Fz) receptor, inhibiting the binding of Wnt to Fz receptor [10].
The Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway diagram (http://www.genome.jp/kegg/) also confirmed that Wnt and SFRP regulate the Wnt signaling pathway at the upstream level (Fig.3).
Title: Wnt signaling pathway-related genes in colorectal cancer

Both the SFRP1 and SFRP2 genes are known to show DNA hypermethylation in colorectal cancers; their relatively low expression causes upregulation of Wnt signaling and thus tumor cell proliferation [11].
Wnt2, a member of the Wnt gene family, and SFRP1 showed drastically opposite gene expression patterns in the discovery dataset (Fig.4).
Given the premise that downregulation of SFRP1 and SFRP2 due to DNA hypermethylation deteriorates the inhibition of the Wnt ligand signaling pathway, we surveyed the gene expression profile of the pathway genes, including SFRP.
The significance of the differential expression of each gene was measured by t test using the discovery dataset.
The p-value distributions were compared between the genes involved in Wnt signaling (n = 151) and those that were not involved in Wnt signaling (n = 17,658) using Wilcoxon rank-sum test.
The genes involved in Wnt signaling showed more significant differential expression than the others (p = 0.01658).
SFRP1 and WNT2 were among the top ranked genes of the pathway (Fig.5).
Title: DMR of SFRP1 gene in colorectal cancer

Illumina Human DNA Methylation27 and 450 are microarray platforms that can measure the methylation values of promoter regions, with 27,578 and 331,182 probes, respectively.
On the former platform, there are four probes in the promoter region of SFRP1, and their methylation status varies probe to probe, while the latter has 33 additional probes for SFRP1.
We examined which probes showed hypermethylation and correlated with expression change using the expanded dataset, for which both data types were available (n = 203).
Among the four SFRP1 probes that were shared between the two platforms, one of them (cg06166767, chr8:41287005) did not show hypermethylation, while the other three showed similar hypermethylation (Fig.6).
Table 3 shows the correlation between differential methylation and differential expression, which was measured using linear regression, revealing a significant correlation of cg15839448 (chr8:41285687) (p = 0.1E-9).
R/Bioconductor package methy Analysis was used to depict the chromosomal location in relation to CGI, along with the methylation beta-value, as a heat map (Fig.7), confirming the variation among probes.
Aberrant methylation patterns of SFRP have been reported not only in colorectal cancers but also in gastric cancers [12], breast cancers [10], and pancreatic cancers [13].
From TCGA, we downloaded 12 matched pairs for each cancer of the following organs: bladder, breast, esophageal, kidney, liver, lung, pancreas, prostate, stomach, and uterine corpus endometrium.
Except for liver and endometrial cancers, the DNA methylation levels of both SFRP1 and SFRP2 were consistently higher in cancers than in the normal counterparts (Fig.8).
It should be noted that in colorectal cancers, the mean methylation M-value of the tumor samples was greater than 0.
While stomach cancers (STAD) also showed a mean M of greater than 0, the difference between the tumor and normal sample was not as large as in colorectal cancers.
In all other cancers, the mean M was below 0, implying marginal hypermethylation at most.
Title: Survival analysis: SFRP methylation in association with clinical factors

It is well known that the survival rate of colorectal cancer is negatively correlated with the tumor stage: the higher the stage, the poorer the survival.
We confirmed this trend in the expanded dataset (n = 419).
Stages IIA and IIB were merged into stage II, and similarly, stages IIIA, IIIB, and IIIC were merged into stage III.
The Kaplan-Meier survival curves showed, as expected, lower survival for higher stages (Fig.9).
The Cox regression analyses also indicated a significant association of stage IV with survival rate (mean hazard ratio 6.083) (95% confidence interval [CI], 1.778 to 20.812; p = 0.004) (Table 4).
Upon confirming the anticipated survival trends by stage in the expanded dataset, we tested whether the methylation level of SFRP1 correlated with the survival rate.
Among the four methylation probes, cg15839448 was chosen, as its methylation level showed the most consistent correlation with the expression level of SFRP1.
In order to separate the samples into two groups that showed a survival difference according to SFRP1 methylation beta-values, we used maximally selected rank statistics, implemented in R (the maxstat package), yielding the cutoff point (beta-value = 0.598) (Fig.10A).
Similarly, cg23207990 was chosen for SFRP2, yielding the cutoff point (beta-value = 0.713) (Fig.10B).
The group with highly methylated SFRP1 showed poorer survival than the other group (p = 0.0181) (Fig.11A).
On the other hand, SFRP2 methylation did not exhibit a significant relationship (p = 0.107) (Fig.11B).
The corresponding hazard ratios from the Cox regression analysis were 1.93 (95% CI, 1.119 to 3.329) and 1.703 (95% CI, 0.892 to 3.25) for SFRP1 and SFRP2, respectively (Table 4).
As higher tumor stage shows a higher risk rate in the cumulative hazard curve (Fig.12A), higher SFRP1 methylation also increases the risk rate (Fig.12B).
In conclusion, hypermethylation of SFRP genes has been linked to the downregulation of their expression in colorectal cancers [11].
Here, we confirm it using independent TCGA datasets.
The evidence was stronger for SFRP1 but only marginal for SFRP2.
We also demonstrate that promoter hypermethylation of SFRP1 is linked to poor survival of colorectal cancer patients.
Scatterplot of differential methylation versus differential expression.
A total of 27,578 methylation probes are plotted in grey, while those probes showing downregulated expression by 4-fold or more (Δ expression &lt; -2) and methylation value (M value) change of +1 or more are plotted as red points.
Negative correlation between gene expression and DNA methylation in 31 selected genes.
Each circle represents a sample.
Wnt signaling pathway from Kyoto Encyclopedia of Genes and Genomes (KEGG) (SFRP genes are in red box).
Heat map of WNT2 and SFRP1 in the discovery dataset.
The 12 samples on the left are normal, while the 12 samples on the right are tumors.
Wilcoxon rank-sum test with Wnt signaling-related genes and the others.
Three Wnt signaling pathway genes, SFRP1 (-17.76), SFRP2 (-5.98), and WNT2 (-13.84), are marked by red ticks in the rug plot.
Differential methylation in four SFRP1 probes (tumor, 166; normal, 37).
Methylation heat map of four SFRP1 probes.
SFRP1 is marked by a vertical red line in the ideogram of chromosome 8, while tracks of CpG islands and exon-intron structures are shown.
The heat map in methylation profile track shows the methylation beta-values of the four probes (cg06166767, cg13398291, cg15839448, and cg22418909 from left to right).
Each row of the heat map represents one of the samples, which are ordered as shown by the The Cancer Genome Atlas (TCGA) sample codes.
The last two digits of the TCGA sample codes represent sample types: &#34;01&#34; for tumor sample and &#34;11&#34; for matched normal.
Comparison of SFRP1 (left panel) and SFRP2 (right panel) methylation among 11 cancer types.
Each cancer type is shown by the boxplot of normal samples, followed by that of tumor samples.
Colorectal cancers are enclosed by red boxes.
Survival curves by colorectal cancer stage.
SFRP gene cutoff points by maximally selected rank statistic.
The cutoff points are marked by vertical dotted lines at 0.598 and 0.713 for SFRP1 (A) and SFRP2 (B), respectively.
Survival curves of SFRP1 (A) and SFRP2 (B) by methylation beta-values.
X-axis represents survival time in days.
Cumulative hazard curve of colorectal cancer by stage (A) and SFRP1 methylation (B) values.
X-axis represents survival time in days.
Title: Differentially hypermethylated and downregulated genes

Title: Differentially hypermethylated and downregulated genes

Title: Differentially hypermethylated and downexpressed genes

Title: Hazard ratio of colorectal patients by tumor stage and SFRP methylation status
Title: Understanding Epistatic Interactions between Genes Targeted by Non-coding Regulatory Elements in Complex Diseases

Genome-wide association studies have proven the highly polygenic architecture of complex diseases or traits; therefore, single-locus-based methods are usually unable to detect all involved loci, especially when individual loci exert small effects.
Moreover, the majority of associated single-nucleotide polymorphisms resides in non-coding regions, making it difficult to understand their phenotypic contribution.
In this work, we studied epistatic interactions associated with three common diseases using Korea Association Resource (KARE) data: type 2 diabetes mellitus (DM), hypertension (HT), and coronary artery disease (CAD).
We showed that epistatic single-nucleotide polymorphisms (SNPs) were enriched in enhancers, as well as in DNase I footprints (the Encyclopedia of DNA Elements [ENCODE] Project Consortium 2012), which suggested that the disruption of the regulatory regions where transcription factors bind may be involved in the disease mechanism.
Accordingly, to identify the genes affected by the SNPs, we employed whole-genome multiple-cell-type enhancer data which discovered using DNase I profiles and Cap Analysis Gene Expression (CAGE).
Assigned genes were significantly enriched in known disease associated gene sets, which were explored based on the literature, suggesting that this approach is useful for detecting relevant affected genes.
In our knowledge-based epistatic network, the three diseases share many associated genes and are also closely related with each other through many epistatic interactions.
These findings elucidate the genetic basis of the close relationship between DM, HT, and CAD.
Recent data have shown the inextricable relationship between diabetes mellitus (DM), hypertension (HT), and coronary artery disease (CAD).
For instance, approximately 70% of patients with DM reported being affected by HT, which is about twice as common in patients with DM as in those without it [1].
Furthermore, patients with both DM and HT were reported to have double the risk for CAD, which is the most prevalent cause of morbidity in type 1 or type 2 DM [2].
However, the underlying genetic contributions driving the increased prevalence of HT and CAD in diabetic patients are poorly understood.
There have been many studies seeking to understand the mechanisms of complex traits using a single-locus-based approach, but they have not been capable of explaining their complicated genetic effects.
Instead, it is necessary to consider the joint genetic effects produced through the simultaneous perturbation of epistatically interacting variants.
Consequently, epistasis is now increasingly assumed to be an important factor in demonstrating complex disease, and there are many studies in which evidence of epistasis has been found [3].
Unfortunately, in spite of such efforts, most such studies usually have been conducted in a European descent cohort [4] and have not included any Asian patients.
In this work, we performed a genomewide epistasis analysis using the East Asian cohort from Korea Association Resource (KARE) data.
This is meaningful work, because obvious differences exist between ethnically diverse populations, such as susceptibility genes, allele frequency, and linkage disequilibrium (LD) structure.
Most genetic variants associated with complex disease are usually located in non-coding genomic regions, and many studies have suggested that such variants might be involved in a transcriptional regulatory role [5].
In this regard, when building an appropriate model, it can be very limiting to assign affected genes using only the nearest ones.
For systematic annotation, therefore, we have suggested extensive non-coding variant annotation using regulatory elements, including promoters and distal enhancers.
In this work, we illustrate how this annotation method can be used to establish a reasonable genetic model in complex diseases.
Title: KARE data

We used 352,228 single-nucleotide polymorphisms (SNPs) genotyped using Affymetrix Genomewide Human SNP Array 5.0 (genomic coordinates hg18; Affymetrix Inc., Santa Clara, CA, USA) for 8,842 Korean individuals from the KARE project.
We converted hg18 to hg19 using the liftOver tool from UCSC (http://genome.ucsc.edu/cgi-bin/hgLiftOver) and converted Affymetrix ID to dbSNP reference cluster ID using the annotation file offered by Affymetrix (http://affymetrix.com).
In this study, we focused on DM, HT, and CAD (myocardial infarction and CAD) and considered people who did not have any disease as the controls (Table 1).
Title: Epistasis detection using GPU-based Boolean Operation-based Screening and Testing

We applied GPU-based Boolean Operation-based Screening and Testing (GBOOST) to analyze epistatic interaction effects from the genomewide SNP data [6].
Stage I: screening-in the screening stage, BOOST evaluated all pairwise interactions using the Kirkwood superposition approximation.
Stage 2: testing-pairs of epistatic loci with PBOOST ≤ 4.89e-06 were further evaluated by a logistic regression model to compute interaction effects [7].
After that, GBOOST conducted a χ2 test with four degrees of freedom (df = 4) and calculated p-values to determine whether the interaction effect was significant.
The numbers of interactions tested were 226,507, 251,344, and 151,090 in DM, HT, and CAD, respectively; thus, the genome-wide significance levels were 4.41e-08, 3.98e-08, and 6.62e-08 using Bonferroni correction.
Title: Enrichment of epistatic SNPs in regulatory elements

To test for the enrichment of epistatic SNPs (Table 2) in regulatory elements, we overlapped SNPs with enhancers (see below) and footprints data.
We obtained genomic DNase I footprints across 41 cell types from the Encyclopedia of DNA Elements (ENCODE) Project (The ENCODE Project Consortium 2012; http://genome.ucsc.edu/ENCODDE/downloads.html).
Nucleotide resolution analysis of DNase I cleavage patterns allowed us to discover the footprints of where the transcription factor bound [8].
To determine whether the observed count was significantly greater than expected by chance, we randomly selected 7,967 variants (the same number of epistatic SNPs) from dbSNPs (59,180,134 autosomal SNPs) and repeated the overlap analysis 1,000 times.
The level of significance (p-value) was computed as the proportion of counts that were equal to or larger than the observed count from the null distribution.
Title: Target gene mapping to SNPs using regulatory elements

We used all transcript annotations from GENCODE.v19, which contains 20,242 protein-coding genes.
The promoter was defined as the region 1.5 kb upstream of the transcription start site of a gene.
When a SNP was in the promoter region of a gene, we assigned the gene to the SNP.
In the case of SNPs that were not assigned using the promoter, we assigned genes using the enhancer that was connected with the promoter of the target gene.
We obtained genomewide multiple-cell-type enhancer data from previous studies [9, 10].
Maurano et al.
[9] identified distal enhancer-to-promoter connections using DNase I profiles across 349 diverse cell types, and Andersson et al.
[10] identified extensive active enhancers and their target promoters by using Cap Analysis Gene Expression (CAGE) data based on the FANTOM5 panel of ~400 distinct cell types.
We collected a total of 663,583 pairs of enhancers and target genes.
If an SNP was not assigned, we looked for the closest SNP in LD based on CHB/JPT populations (Han Chinese in Beijing, China/Japanese in Tokyo, Japan; Asian ancestry) with an r2 &gt; 0.8 that was within 500 kb using the SNAP proxy tool (http://www.broadinstitute.org/mpg/snap/) [11].
The summary of the number of assigned genes is shown in Table 3.
Title: Enrichment of the affected genes in the known gene set

We downloaded DM-, HT-, and CAD-related gene information from Phenopedia (HuGE Navigator, version 2.0) [12].
The HuGE Navigator is an integrated knowledge-based database of genetic associations that have been extensively explored, based on the literature, and is continuously updated from PubMed (http://www.ncbi.nlm.nih.gov/pubmed/).
To test whether the genes we discovered were statistically enriched within known genes involving the disease, we calculated the p-value using the hyper-geometric test: where N represents the total number of genes (N = 20,242), n is the number of genes we found, and M and k are the number of genes in the known DB (Table 3) and that overlap, respectively.
In the KARE dataset, there were substantial overlaps between the cases of DM, HT, and CAD (Fig.1).
We found that the occurrence of one disease was largely dependent on another.
For example, the incidence of HT was significantly associated with CAD (odds ratio, 2.39; 95% confidence interval [CI], 1.64 to 3.48; p = 4.29e-06).
Similarly, the odds ratio of incident DM for HT versus normal was 2.15 (95% CI, 1.79 to 2.56; p = 2.20e-16), and the odds ratio of CAD for DM was 1.68 (95% CI, 1.02 to 2.68; p = 2.82e-02) (Fig.2).
We analyzed all pairwise interaction effects from the KARE data using GBOOST.
As the result, we detected 1,962, 2,023, and 1,709 epistatic interactions in DM, HT, and CAD, respectively (Table 3).
For a full list, see the Supplementary Tables 1,2,3.
We overlapped these epistatic SNPs within genomic regions, including the promoter, gene distal region, intron, and exon.
(Fig.2).
As a result, the majority of detected epistatic SNPs was found to reside in non-coding regions, especially within introns and gene distal regions, which are intergenic regions, excluding promoters.
Furthermore, we tested whether the epistatic SNPs were enriched in regulatory element, such as enhancers and DNase I footprints (Fig.3).
We identified statistically significant enrichment of the SNPs in enhancer regions (p = 0.001), as well as DNase I footprint regions (p &lt; 0.001).
This suggests that the disruption of regulatory factor binding might be involved in the disease mechanism.
For that reason, we annotated epistatic SNPs using regulatory element information, including promoters, as well as enhancers (see Methods).
We also proved that the assigned genes were enriched in each known disease-associated gene set (Table 3).
We built a knowledge-based epistatic network.
We discovered that the diseases share many associated genes and also have a number of epistatic interactions within and between genes involved in each disease (Fig.4).
For a detailed list, see the Supplementary Table 4.
For example, we detected an epistatic interaction effect between atypical protein kinase C isotype-specific interacting protein (ASIP) and hexokinase II (HK2) in DM.
Previous studies have reported that overexpression of ASIP inhibits insulin-induced glucose uptake [13], and HK2 plays a central role in glucose metabolism [14].
Our result implies that gene pairs, such as ASIP and HK2, could boost the incidence of DM through epistatic interactions.
We also detected several significant epistatic interaction effects in CAD and HT between several DM-associated gene pairs.
This finding could provide a genetic basis for the increased prevalence of CAD and HT in diabetic patients.
In this paper, we carried out a genomewide epistasis analysis to understand the comprehensive genetic relationships between DM, HT, and CAD in a Korean cohort.
For systematic annotation, we employed whole-genome multiple-cell-type enhancer data, because the majority of detected epistatic SNPs was located in non-coding regions.
Our enrichment analysis in regulatory DNA regions identified substantial variants that might affect transcriptional regulatory function through disruptions of promoter or enhancer elements where regulatory factors bind.
We confirmed that the genes that were discovered using the non-coding element are statistically enriched in known genes involved in the diseases, which suggests that this annotation method is considerably relevant.
We found that the diseases shared many associated genes, implying their inseparable relationship.
A large number of epistatic interactions were also intricately connected between genes involved in DM, HT, and CAD.
This integrated map, based on epistatic interactions, could provide a genetic basis for the underlying mechanisms of the increased prevalence of the diseases&#39; dependence on one another.
In conclusion, our results illustrate how extensive non-coding variant annotation based on chromatin interactions of distal enhancers can be used to dissect complex genetic interactions in complex diseases.
The summary of the incidence of diabetes mellitus (DM), hypertension (HT), and coronary artery disease (CAD).
(A) Venn diagram illustrating the overlaps among diseases.
(B) The odds ratio (OR) for the disease incidence.
Pie chart representing the genomic regions of epistatic single nucleotide polymorphisms.
CAD, coronary artery disease; DM, diabetes mellitus; HT, hypertension.
Enrichment of epistatic single-nucleotide polymorphisms (SNPs) in enhancer and DNase I footprints.
The distribution of expected overlaps was obtained from 1,000 random sets.
A red line indicates the observed count of SNPs in enhancer and DNase I footprints.
The knowledge-based epistatic network between coronary artery disease (CAD), diabetes mellitus (DM), and hypertension (HT).
Uncolored circles connected by gray lines with colored circles indicate genes known to be involved with DM (green), HT (blue) and CAD (red), respectively.
The colored lines indicate the epistatic interactions detected in DM (green), HT (blue) and CAD (red) patients.
Title: Summary of KARE data used

Title: Summary of detected epistatic interactions

Title: Target gene assignment and enrichment analysis using known DB across diseases
Title: Genome-Wide Association Study of Metabolic Syndrome in Koreans

Metabolic syndrome (METS) is a disorder of energy utilization and storage and increases the risk of developing cardiovascular disease and diabetes.
To identify the genetic risk factors of METS, we carried out a genome-wide association study (GWAS) for 2,657 cases and 5,917 controls in Korean populations.
As a result, we could identify 2 single nucleotide polymorphisms (SNPs) with genome-wide significance level p-values (&lt;5 × 10-8), 8 SNPs with genome-wide suggestive p-values (5 × 10-8 ≤ p &lt; 1 × 10-5), and 2 SNPs of more functional variants with borderline p-values (5 × 10-5 ≤ p &lt; 1 × 10-4).
On the other hand, the multiple correction criteria of conventional GWASs exclude false-positive loci, but simultaneously, they discard many true-positive loci.
To reconsider the discarded true-positive loci, we attempted to include the functional variants (nonsynonymous SNPs [nsSNPs] and expression quantitative trait loci [eQTL]) among the top 5,000 SNPs based on the proportion of phenotypic variance explained by genotypic variance.
In total, 159 eQTLs and 18 nsSNPs were presented in the top 5,000 SNPs.
Although they should be replicated in other independent populations, 6 eQTLs and 2 nsSNP loci were located in the molecular pathways of LPL, APOA5, and CHRM2, which were the significant or suggestive loci in the METS GWAS.
Conclusively, our approach using the conventional GWAS, reconsidering functional variants and pathway-based interpretation, suggests a useful method to understand the GWAS results of complex traits and can be expanded in other genomewide association studies.
Metabolic syndrome (METS) is a disorder of energy utilization and storage and increases the risk of developing cardiovascular disease and diabetes.
METS includes multiple clinical traits, as follows: increased plasma glucose, abdominal obesity, dyslipidemia, and high blood pressure [1].
METS is a great concern in developing countries, because the prevalence of METS is gradually increasing, especially in countries where it follows obesity trends, sedentary lifestyle, and high consumption of calories [2, 3, 4].
A Korean twin study showed that the METS has 51%-60% heritability, indicating a significant role of genetic factors in the development of METS [5].
Therefore, understanding the genetic factors underlying the syndrome and their correlation is clinically important.
Recent advances in high-throughput genomics technologies have allowed massive testing of genetic variants in minimal time [6, 7].
The reductions in cost and time have made it feasible to conduct large-scale genome-wide association studies (GWASs) that genotype many thousands of single nucleotide polymorphisms (SNPs) in thousands of individuals.
So far, approximately 1,900 reports with 13,000 SNPs (GWAS catalog; Apr 14, 2014) have been published to identify the gene-disease or -non-disease trait associations.
The quantitative traits related to METS have already been studied by the conventional GWAS in the Korean population [8], but there is no Korean GWAS report about METS cases and controls.
On the other hand, relatively little trait heritability can be explained by the conventional GWAS [9].
These phenomena, called missing heritability problems, are hard to solve by conventional GWAS, in part because of the extensive multiple testing correction in GWAS analysis, low effect size of common variants, and the difficulty of detecting low-frequency or rare variants in conventional GWAS.
Multiple testing correction is necessary to exclude false-positive loci, but simultaneously, it discards many true-positive loci [10].
Also, most SNPs in these GWASs lie in intergenic and intron regions and do not appear to affect protein sequence.
Thus, these SNPs are likely functionally neutral or just proxies of causal variants located in the same linkage disequilibrium (LD).
To understand the amount of true-positive signals in the discarded association results, we computed the proportion [V(G)/V(P)] of the phenotypic variance [V(P)] that is explained by the genotype variance [V(G)] using the significant and discarded SNP results [11].
Recently, Fransen et al.
[10] reported a GWAS using expression quantitative trait loci (eQTL) information.
They selected eQTL among the GWAS results for Crohn&#39;s disease and conducted follow-up replication studies [10].
They showed that eQTL-based pre-selection for follow-up is a useful approach for identifying risk loci from a moderately sized GWAS.
Based on previous knowledge, we applied an alternative analysis strategy to understand the genetic components of METS.
First of all, we conducted a conventional GWAS for METS cases and healthy controls to discover the top significant signals.
Thereafter, we tried to uncover the functional variants, such as nonsynonymous SNPs (nsSNPs) and eQTLs, among the SNPs to be discarded using the stringent criteria of the conventional GWAS.
Finally, we drew a pathway of the significantly associated GWAS SNPs and the remaining less significantly associated functional SNPs.
The overall study design is schematically described in Supplementary Fig.1.
Title: Study subjects

The study subjects were originally derived from a part of the Korean Genome and Epidemiology Study (KoGES) project, which was the national project to establish genome epidemiology cohorts of Korean dwellers or immigrants/emigrants [12].
Among the KoGES cohorts, the Korean Association Resource Consortium (KARE) has established a public GWAS dataset by using the Ansan-Anseong cohort, which is an ongoing biennially followed-up cohort in the KoGES [13].
The KARE dataset consists of the individual SNP chip genotypes and the epidemiological/clinical phenotypes for studying the genetic components of Korean public health.
Written informed consent was obtained from all participants at the KoGES, and this research project was approved by the Institutional Review Board of Korea National Institute of Health (KNIH).
The obtained KARE dataset passed the quality control criteria and was reported in previous GWAS reports [8, 13].
Briefly, the subjects with genotype accuracies below 98% and high missing genotype call rates (≥5%), high heterozygosity (&gt;30%), or inconsistency in sex were excluded from subsequent analysis.
Individuals who had a tumor were excluded, as were related individuals whose estimated identity-by-state values were high (&gt;0.80).
Based on these criteria, 8,842 samples were selected; these quality control steps have been described in a previous GWAS [13].
Title: Study phenotypes and covariates

We used the general information on resident areas (Anseong or Ansan), sex, and age as the covariates and past disease history of diabetes, hypertension, and lipidemia as exclusion criteria for non-METS healthy controls.
The height and body weight were used to calculate the body mass index (BMI) as another covariate, and waist circumference (WC), systolic and diastolic blood pressures (SBP and DBP), fasting plasma glucose levels (GLU0), high-density lipoprotein (HDL) cholesterol, and triglyceride (TG) were used to diagnose METS.
METS was defined by the presence of three or more of the following five components according to the NCEP-ATPIII criteria using WC for Asians [14, 15]: WC (≥90 cm for men and ≥80 cm for women), HDL (&lt;40 mg/dL for men, &lt;50 mg/dL for women), TG (≥150 mg/dL), SBP (≥130 mm Hg) and/or DBP (≥85 mm Hg), and GLU0 (≥100 mg/dL).
Title: Study genotypes

The genotyping of the cohort population was previously described for the KARE study [16].
Most DNA samples were isolated from the peripheral blood of participants and genotyped using Affymetrix Genomewide Human SNP array 5.0 (Affymetrix, Inc., Santa Clara, CA, USA).
The quality control steps of the genotypes have been described elsewhere [13].
Briefly, the calling of the genotyping was determined by Bayesian Robust Linear Modeling using the Mahalanobis Distance genotyping algorithm [17].
Consequently, 352,227 SNPs had a missing genotype call rate below 0.1, a minor allele frequency greater than 0.01, and no deviation from Hardy-Weinberg equilibrium (p &gt; 1 × 10-6).
Additionally, the previous GWAS reported no population stratification between the Anseong and Ansan cohorts [13].
Title: Statistical analysis

The GWAS for METS cases and controls was conducted by logistic regression analysis, adjusting for residential area, sex, age, and BMI as covariates, implemented in PLINK version 1.07 [18].
The significant associations were defined by genomewide significance level p-values (&lt;5 × 10-8) and genome-wide suggestive p-values (5 × 10-8 ≤ p &lt; 1 × 10-5).
The LD between the previously reported GWAS SNPs and the SNPs of the current GWAS was investigated with SNAP web-based software (http://www.broadinstitute.org/mpg/snap) and GWAS catalog (http://www.genome.gov/gwastudies/).
For example, we entered our top significant SNPs in the SNAP input panel and found high LD SNPs with r2 &gt; 0.9 and D&#39; = 1 around 1 Mbp.
The high-LD SNPs were investigated in the GWAS catalog (http://www.genome.gov/gwasstudies) as to whether they were previously reported or not.
To maximize the candidate risk factors of METS, we selected additional functional SNPs in the eQTLs or nsSNP loci (5 × 10-5 ≤ p &lt; 1 × 10-4).
Among the Affymetrix 5.0 SNPs, we investigated the eQTL SNPs from regulomeDB (http://regulomedb.org) and the nsSNPs from BioMart (http://www.biomart.org).
The genetic variances of the top association SNPs were estimated by GCTA v1.24 [16], which is a tool for estimating the proportion [V(G)/V(P)] of phenotypic variance [V(P)] explained by SNPs [V(G)] for complex traits.
We selected the SNP sets based on the GWAS p-values from 100 to 1,000 SNPs with 10-SNP intervals and from 1,000 to 5,000 SNPs with 1,000-SNP intervals.
We decided the number of SNPs in the maximum set based on the genetic variance approximated the METS heritability reported from the Korean twin study [5].
The pair-wise genetic relationships were estimated using the make-grm option, and the proportion of the phenotypic variance explained by the associated SNPs was estimated by the grm-test option with the restricted maximum likelihood [11].
Title: In silico analysis

The functional relevance of the associated SNP sites was analyzed by overlapping the gene-coding sequence or the Encyclopedia of DNA Element (ENCODE) regulatory element positions in the University of California Santa Cruz (UCSC) genome browser (http://genome.ucsc.edu).
Thereafter, regulomeDB (http://regulome.stanford.edu/) was utilized to extract eQTL information.
In addition, Pathway Studio version 9.0 software (Ariadne Genomics, Rockville, MD, USA) was utilized to analyze the functional interactions and possible pathways among genes/proteins in our data.
It provides an interpretation of the biological implications from gene/protein expression data, the establishment of molecular pathways, and an identification of protein interaction maps and their association to cellular process [19].
Title: Genome-wide association study

Table 1 describes the clinical characteristics of Ansung and Ansan regarding the METS criteria: BMI, WC, SBP, DBP, GLU0, HDL, and TG.
Based on the NCEP-ATPIII METS criteria for Asians [14], 2,657 KARE subjects were included in the METS cases.
The SNPs showing strong and moderate evidence of association (p &lt; 1 × 10-5) are indicated in the Manhattan plot of the GWAS (Fig.1).
In addition to these SNPs, we identified several functional SNPs with suggestive evidence of association (5 × 10-8 ≤ p).
In this study, we selected 12 SNPs, of which 2 had genome-wide significant associations (p &lt; 5 × 10-8), 8 had suggestive associations (5 × 10-8 ≤ p &lt; 1 × 10-5), and 2 had functional variants (1 × 10-5 ≤ p &lt; 1 × 10-4) (Table 2) [8, 20, 21, 22].
The top SNP (rs11216126) and 3 suggestive SNPs (rs6589566, rs174 82753, and rs10503669) were previously reported as being associated with METS-related traits, such as serum cholesterol levels or TG levels [8, 20, 21].
LD analysis using 10 SNPs was conducted with the previously reported GWAS SNPs.
As a result, 5 SNPs had strong LD with the 15 highly linked GWAS SNPs (Table 3) [23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34].
Among the 6 remaining SNPs associated with METS in our GWAS results, 2 SNPs (rs180349 and rs17410962) showed high LD with the previously reported SNPs (r2 &gt; 0.9 and D&#39; = 1) even though the 2 SNPs have not been reported regarding metabolic traits.
Therefore, we discovered 10 significant or suggestive associated SNPs in the METS GWAS, but 6 of them were already reported or linked to the reported SNPs.
The remaining 4 suggestive signals and 2 functional variants have been first reported in the current study, and a replication study should be performed in other independent populations.
Title: In silico annotation of the linked genes and functional relevance

The 10 associated SNPs and the LD SNPs were located in six functional gene regions, and one SNP was located in the intergenic region.
The top signals were located downstream of a functional spliceosome-associated protein, named BUD13, a homolog of yeast (BUD13) gene chromosome 11 and near the BUD13 gene.
BUD13 has been reported to be associated with lipid, metabolic syndrome X [23], TG [32], and metabolic traits in East Asians [8], demonstrating that it is putatively functionally associated with METS in the Korean population (Supplementary Table 1).
The second significant SNP was rs6589566, which has 6 high-LD SNPs.
Notably, in silico annotation of the SNP&#39;s function showed that rs651821 was located in the 5&#39; untranslated region (UTR) of the APOA5 gene, and also, the SNP was reported as an eQTL of the transgelin (TAGLN) gene (Table 4).
The results indicate that the remaining SNPs are surrogate markers of rs6589566.
Among them, rs651821 and rs964184 are associated with TG level, as one component of METS evaluation, in Chinese populations [26] and in Mexicans [35].
Both SNPs exhibit eQTL of the TAGLN gene.
TAGLN has been documented as a repressive regulator of matrix metalloproteinase 9 (MMP9) gene expression [36] and is considered a putative tumor suppressor due to suppression of MMP-9, which harbors tumor metastatic properties [37].
However, MMP-9 is also involved in the progression of METS via chymase activity [38] and has been suggested to be used a diagnostic marker of METS [39].
Thus, it can be explained that the functional eQTL TAGLN of rs651821 and rs964184 could be a novel marker for the evaluation of METS in terms of strong regulation of MMP-9.
The third most significant signals were located in the lipoprotein lipase (LPL) gene region.
Three SNPs (rs10503669, rs17482753, and rs17410962) located in chr 8 were eQTL that contributed to LPL expression in monocytes.
LPL is a critical protein of lipid metabolism and is significantly associated with METS in Asian Indians [40], indicating that the functional eQTL-SNP of LPL expression could be a marker for the evaluation of METS in Korean populations.
Those three SNPs were in strong LD with rs328, which is a stop-gain mutation of the LPL coding sequence [41] (Supplementary Table 1).
Although the remaining 14 SNPs were non-eQTL-SNPs, those SNPs have been reported as being in association with HDL cholesterol, low-density lipoprotein cholesterol, TG, and obesity, indicating that they are putative candidate markers for the evaluation of METS.
Although the remaining 6 SNPs and their nearest genes have not been functionally studied regarding METS-associated traits, further studies are required to elucidate for their role in METS.
Title: Pathway network analysis

The results of the V(G)/V(P) for 100 to 5,000 SNPs were plotted in Fig.2.
When we used 5,000 SNPs, V(G)/V(P) approximated 50%, and we extracted functional SNPs, such as eQTLs and nsSNPs, from the 5,000 SNPs.
We could extract 159 eQTLs and 18 nsSNPs among the 5,000 SNPs (Supplementary Table 2).
Notably, 6 eQTL genes and 2 nsSNP genes consisted of LPL and the apolipoprotein A-V (APOA5) pathway through the interaction of a number of mediated genes (Table 4).
Among them, muscle creatinine kinase (CKM) has been documented to regulate LPL activity [42], demonstrating that it is putatively associated with METS.
Those additionally identified genes might be candidate targets of METS for further study (Fig.3).
eQTLs and nsSNPs provide insights into the regulation of transcription and aid in the interpretation of GWASs [22].
Most of the eQTL resources are available in online databases, such as RegulomeDB (http://regulome.stanford.edu/), including several published resources in various cell types, such as monocytes [43], human brain [44], lymphoblastoid cell lines [45, 46], and human liver [47].
Probably, RegulomeDB is one of the most useful eQTL databases, because it contains rich information about the products of the ENCODE project, such as transcription factor binding sites, chromatin structure, histone modification, and eQTLs.
Our pathway results suggest an internal mechanism of LPL, APOA5, and muscarinic acetylcholine receptor M2 (CHRM2) functions in METS.
Therefore, we suggest that 6 eQTLs and 2 nsSNP loci might be additional targets for further association studies and functional analysis.
Conclusively, our approach using the conventional GWAS, reconsidering functional variants and the pathway-based interpretation, suggests a useful method to understand the GWAS results of complex traits and can be expanded in other GWASs.
Manhattan plot of metabolic syndrome genome-wide association study -log10(p-values).
All black and grey circles indicate the individual single nucleotide polymorphisms (SNPs).
The red horizontal line is the genome-wide significance level (p = 5 × 10-8), and the blue horizontal line is the genome-wide suggestive level (5 × 10-8 ≤ p &lt; 1 × 10-5).
The top significant SNPs are depicted on the right site of the SNP.
The proportion of phenotypic variance [V(P)] explained by the genotypic variance [V(G)].
The horizontal axis denotes the number of single nucleotide polymorphisms (SNPs).
Approximately 50% of phenotypic variance could be explained by the top 5,000 SNPs.
Illustration of molecular pathway for significantly associated single nucleotide polymorphism (SNP) loci in the metabolic syndrome genome-wide association study (GWAS).
The molecules depicted by the significant GWAS loci (reds), functional SNPs (expression quantitative trait loci or nonsynonymous SNPs) loci (green), and the other intermediate molecules (yellow) are illustrated on the cell organelles.
Title: Clinical characteristics of metabolic syndrome-related traits

Title: Genome-wide association results for METs case-control study in the Korean population

Title: Reported GWAS SNPs that show LD with our study SNPs

Title: GWAS results of eQTLs and nonsynonymous SNPs consisting of the LPL and APOA5 pathways
Title: Pathway Analysis of Metabolic Syndrome Using a Genome-Wide Association Study of Korea Associated Resource (KARE) Cohorts

Metabolic syndrome (MetS) is a complex disorder related to insulin resistance, obesity, and inflammation.
Genetic and environmental factors also contribute to the development of MetS, and through genome-wide association studies (GWASs), important susceptibility loci have been identified.
However, GWASs focus more on individual single-nucleotide polymorphisms (SNPs), explaining only a small portion of genetic heritability.
To overcome this limitation, pathway analyses are being applied to GWAS datasets.
The aim of this study is to elucidate the biological pathways involved in the pathogenesis of MetS through pathway analysis.
Cohort data from the Korea Associated Resource (KARE) was used for analysis, which include 8,842 individuals (age, 52.2 ± 8.9 years; body mass index, 24.6 ± 3.2 kg/m2).
A total of 312,121 autosomal SNPs were obtained after quality control.
Pathway analysis was conducted using Meta-analysis Gene-Set Enrichment of Variant Associations (MAGENTA) to discover the biological pathways associated with MetS.
In the discovery phase, SNPs from chromosome 12, including rs11066280, rs2074356, and rs12229654, were associated with MetS (p &lt; 5 × 10-6), and rs11066280 satisfied the Bonferroni-corrected cutoff (unadjusted p &lt; 1.38 × 10-7, Bonferroni-adjusted p &lt; 0.05).
Through pathway analysis, biological pathways, including electron carrier activity, signaling by platelet-derived growth factor (PDGF), the mitogen-activated protein kinase kinase kinase cascade, PDGF binding, peroxisome proliferator-activated receptor (PPAR) signaling, and DNA repair, were associated with MetS.
Through pathway analysis of MetS, pathways related with PDGF, mitogen-activated protein kinase, and PPAR signaling, as well as nucleic acid binding, protein secretion, and DNA repair, were identified.
Further studies will be needed to clarify the genetic pathogenesis leading to MetS.
Metabolic syndrome (MetS) is a complex disorder related to type 2 diabetes mellitus (T2DM) and cardiovascular diseases, and its prevalence is continuously increasing worldwide [1, 2].
Insulin resistance, obesity, and inflammation are major factors leading to MetS; however, the effect of genetic and environmental factors cannot be ignored [3].
Sedentary lifestyle, decreased physical activity, high caloric intake, and westernized food habits are environmental factors leading to obesity and MetS [4, 5].
Parental and maternal obesity in early pregnancy is related to increased risk of childhood obesity, which could later lead to obesity in young adulthood [6, 7].
In addition, family history of obesity, insulin resistance, and T2DM can increase the risk of MetS, implying the importance of genetic contribution.
Candidate gene studies in MetS identified genes involved in glucose and insulin signaling, such as insulin receptor substrate 1 (IRS1), peroxisome proliferator-activated receptor γ (PPARG), insulin-like growth factor 1 (IGF1), and genes involved in lipid metabolism, such as adiponectin (ADIPOQ), apolipoprotein A5 (APOA5), and low-density lipoprotein receptor (LDLR) [3, 8, 9, 10].
Through genome-wide association studies (GWASs), a larger number of candidate genes could be further analyzed, and important susceptibility loci were discovered, including fat mass and obesity associated protein (FTO) and the melanocortin 4 receptor gene (MC4R), which were associated with body mass index (BMI) [11, 12].
In another GWAS of MetS, the lipid locus at rs964184 was associated with high-density lipoprotein (HDL)-cholesterol and very low-density lipoprotein-cholesterol [13].
In a meta-analysis in Korea that used the GWAS results from the Korea Associated Resource (KARE) cohort, susceptibility loci in 12q24.11 and 12q24.13 were associated with HDL-cholesterol levels, and genetic factors associated with osteoporosis and metabolic traits, such as T2DM, dyslipidemia, and obesity, could also be identified [14, 15, 16].
GWASs have their strengths in screening susceptible genes associated with complex diseases [17, 18].
However, GWASs focus more on individual single-nucleotide polymorphisms (SNPs) that meet a stringent statistical significance, rather than explaining the interaction of genes, and they can only explain a small portion of genetic heritability [19, 20, 21].
In addition, due to its small effect size, certain SNPs in a GWAS that have been identified to be associated with a disease might not show up in another study of the same disease.
This can be seen in two published studies of T2DM and Crohn&#39;s disease, which could not find most of the proven susceptibility loci through GWASs and succeeded in achieving moderate significance after replication studies or meta-analysis [22, 23].
To overcome this limitation, pathway-based approaches have been introduced to improve the interpretability of the GWAS.
Pathway-based analysis integrates GWAS data with genes in the selected biological pathways or gene sets from predefined human databases [19, 24].
The strength of pathway analysis is its large effect size and higher power to detect genes that might have been missed through a GWAS [24, 25, 26].
Pathway analysis, such as Meta-analysis Gene-Set Enrichment of Variant Associations (MAGENTA), only requires the SNP p-values and chromosome positions, simplifying the analysis of GWASs [27].
MAGENTA analyzes the statistical power of GWASs through integration of variant association p-values into gene scores, correcting for confounding factors, such as gene size, SNP density, and linkage disequilibrium properties [27].
Through MAGENTA analysis, biological pathways associated with triglyceride, HDL-cholesterol, T2DM, and BMI have been identified [27, 28, 29].
Pathway analysis is a supplementary way to further analyze the results of GWASs.
However, there are few studies that have used this approach to identify biological pathways associated with MetS in Asians.
The aim of this study was to further elucidate the genomic data of the KARE cohort and to identify the biological pathways related with MetS through a pathway-based approach.
Title: Subjects

The cohort data from the KARE were used for the analysis.
The KARE project, initiated in 2007, is a large cohort study that recruited two population studies from the rural Anseong and urban Ansan cohorts.
We analyzed the data of 8,842 individuals (age, 52.2 ± 8.9 years; BMI, 24.6 ± 3.2 kg/m2).
Anthropometric measurements, including weight, height, and waist circumference, were measured in all subjects, and BMI was calculated (kg/m2).
Systolic and diastolic blood pressures (BP) were examined in all subjects.
Fasting plasma glucose and lipid profiles, including serum total cholesterol, HDL-cholesterol, and triglyceride levels, were measured after an overnight fast.
Detailed information on the study protocol has been previously described by Cho et al.[16].
MetS was defined according to the modified Third Report of the National Cholesterol Education Program (NCEP-ATPIII) diagnostic criteria, which require the presence of three out of the five following factors: 1) abdominal obesity, defined through waist circumference, using the cut-off values for Asians (≥90 cm in men and ≥80 cm in women), 2) triglycerides ≥ 150 mg/dL or being on lipid-lowering treatment, 3) low HDL-cholesterol (men &lt; 40 mg/dL, women &lt; 50 mg/dL) or being on lipid-lowering treatment, 4) systolic/diastolic BP ≥ 130/85 mm Hg or being on anti-hypertensive treatment, and 5) fasting plasma glucose ≥ 100 mg/dL or previous diagnosis of T2DM or anti-diabetic treatment [30, 31].
Title: Genome-wide association dataset analyses

Genotyping was done using Affymetrix Genome-wide Human SNP Array 5.0 (Affymetrix Inc., Santa Clara, CA, USA).
Samples with gender inconsistencies and low call rates (&lt;96%) were excluded.
Quality control (QC) procedures were performed using PLINK version 1.07 [32].
Samples were excluded if there was a high missing call rate (&gt;5%), low minor allele frequency (&lt;0.05), or significant deviation from Hardy-Weinberg equilibrium (p &lt; 1 × 10-6).
The total genotyping rate of the remaining individuals was 99.58%.
A total of 312,121 autosomal SNPs were obtained after QC, representing 8,842 individuals (4,183 males and 4,659 females).
An additive model was used for the analysis.
Detailed information on the quality control procedure of the genotypes is described elsewhere by Cho et al.[16].
Title: Pathway-based analysis

Pathway analysis was conducted using MAGENTA (http://broadinstitute.org/mpg/magenta) to discover biological pathways or gene sets associated with MetS.
Detailed information on this analysis is described by Segre et al.[27].
Briefly, the steps of MAGENTA analysis were as follows: 1) SNP association p-values and chromosome positions from the GWAS are used as input; 2) each gene located at a predetermined boundary is mapped to a single SNP; 3) based on the regional SNP p-values, gene scores are ranked, and the best SNP p-values are determined; 4) gene scores are corrected for confounding factors, such as gene size and linkage disequilibrium-related properties; and 5) gene set enrichment p-values are determined by analyzing the gene sets enriched with highly ranked gene scores and the selected biological pathway or gene sets [27].
False discovery rate (FDR) was also identified through multiple test correction.
Additional information, including 95th and 75th percentile cutoffs and the number of observed and expected genes within each pathway, were also calculated.
Since 75th percentile cutoffs have greater power in interpreting complex diseases that are highly polygenic, this cutoff value was used for our interpretation [27, 29, 33].
A total of 8,842 subjects (4,183 men and 4,659 women) were involved in the study.
Of them, 3,253 (36.8%) had MetS.
Clinical characteristics of subjects with and without MetS are shown in Table 1.
Individuals with MetS were older, with higher BMI, systolic BP, triglycerides, and fasting plasma glucose and lower HDL-cholesterol levels compared to subjects without MetS.
In the discovery set of the GWAS, three SNPs associated with MetS were identified, demonstrated in Table 2.
SNPs from chromosome 12, including rs11066280, rs2074356, and rs12229654, had a p &lt; 5 × 10-6.
From these SNPs, only rs11066280 satisfied the Bonferroni-corrected cutoff (unadjusted p &lt; 1.38 × 10-7, Bonferroni-adjusted p &lt; 0.05).
The top 10 significant biological pathways or gene sets associated with MetS at the 75th percentile cutoff are shown in Table 3.
The pathways were as follows: electron carrier activity (gene ontology [GO] term), signaling by platelet-derived growth factor (PDGF) (Reactome), mitogen-activated protein kinase kinase kinase (MAPKKK) cascade (GO term), PDGF binding (GO term), nucleic acid binding (protein analysis through evolutionary relationships [PANTHER] molecular function), PPAR signaling (Ingenuity), negative regulation of gene-specific transcription from RNA polymerase II promoter (GO term), non-motor microtubule-binding protein (PANTHER molecular function), protein secretion (GO term), and DNA repair (GO term).
At the FDR level, although all pathways were above 0.05, the pathway of electron carrier activity had an FDR value &lt; 0.1, and the pathways of signaling by PDGF and PPAR signaling showed an FDR &lt; 0.2.
In this study, through pathway analysis of MetS, important pathways, including electron carrier activity, signaling by PDGF, MAPKKK cascade, PDGF binding, nucleic acid binding, PPAR signaling, negative regulation of genespecific transcription from RNA polymerase II promoter, non-motor microtubule binding protein, protein secretion, and DNA repair, were identified.
In the previous GWAS using KARE cohorts, rs11066280 and rs2074356 in chromosome 12q24.13, near the chromosome 12 open reading frame, human C12orf51 (C12orf51), and rs12229654 in chromosome 12q24.11, near myosin, light chain 2 (MYL2) were identified to be associated with HDL-cholesterol, hypertension, T2DM, and dyslipidemia [14, 34].
Drinking behavior was also associated with rs11066280 (C12orf51) in Korean men and Han Chinese [34, 35].
In other published GWASs on BMI, important variants on loci near/in FTO, MC4R, and transmembrane protein 18 (TMEM18) were associated with BMI, the latter also having a strong association with BMI in children [36].
Genetic variants in zinc finger protein 259 (ZNF259), lipoprotein lipase (LPL), and APOA5 were also associated with MetS [37].
In the GWAS of European Americans and Finnish cohorts, APOC1 was related with dyslipidemia and central obesity, and the gene cluster region in SNP rs964184, near/in gene APOA1/C3/A4/A5, was associated with MetS [3, 38].
Through MAGENTA analysis, pathways related with electron carrier activity and PDGF signaling and binding, as well as PPAR signaling, were identified as some of the top ranking pathways associated with MetS.
Electron carrier activity may be related with electron transport activity in the mitochondria.
Abnormal regulation of mitochondrial function is associated with factors, such as reduced electron transport chain, which can lead to insulin resistance and MetS [39, 40].
In obese and diabetic patients, fewer and diminished mitochondrial electron transport enzymes, especially complex I, were observed in the skeletal muscle [41, 42, 43].
Defects in the electron transport chain can impair carbohydrate metabolism, affecting the tricarboxylic acid cycle and limiting ATP activity, which could result in lactic acidosis [44].
In addition, mitochondria respiratory chains are major sites of reactive oxygen species (ROS) production, and excess electrons can increase ROS, stimulating proinflammatory processes and mutagenesis, contributing to mitochondrial dysfunction [40, 45].
Pathways related to PDGF binding and signaling and the MAPKKK cascade were also associated with MetS in this study.
The PDGF signaling pathway has been identified to be associated with BMI [29].
PDGF is an important activator of cell proliferation and migration, mediated by the mitogen-activated protein kinase (MAPK) family, and PDGF signaling regulates angiogenesis [46, 47].
In animal studies, PDGF-mediated pathways played a crucial role in healing myocardial infarction, myocardial fibrosis, and defects in the pathway lead to prolongation of inflammation [48, 49].
In a human study, serum PDGF isoform b levels were lower in individuals with MetS, while increased PDGF expression with elevated urinary PDGF-BB was seen in patients with diabetic nephropathy [50, 51].
MAPK pathways are involved in adipogenesis and metabolic homeostasis, and defects in these pathways due to factors, such as oxidative stress, can lead to abnormal adipose regulation, insulin resistance, and obesity [52, 53].
In addition, increased MAPK signaling had a detrimental effect on β-cell function and insulin homeostasis, which could contribute to the development of MetS [54].
The PPAR isotypes PPAR-α, -δ, and -γ play an important role in lipid and glucose metabolism [55].
PPAR-α is expressed in tissues, including skeletal muscle and liver, regulating lipid metabolism and inflammatory processes, whereas PPAR-δ and PPAR-γ are involved in adipocyte differentiation [56].
Genetic variations in PPAR can affect glucose uptake, fasting glucose levels, and BMI [57, 58, 59].
In a GWAS of T2DM in a Finnish population, variants near PPAR-γ were associated with T2DM [60].
In a Korean study, polymorphisms in PPAR-δ were related with BMI and fasting glucose in non-diabetics [61].
Other pathways related with nucleic acids, such as RNA and DNA, as well as pathways of protein secretion were also associated with MetS.
MicroRNAs regulate the action and secretion of insulin, as well as lipid metabolism, playing an important role in the pathogenesis of diabetes, obesity, and cancer [62, 63, 64].
Abnormal expression of microRNAs in pancreatic beta-cells affects beta-cell function and insulin secretion [65].
MicroRNA expression is also related to appetite control in the brain; neural signaling in the muscle, pancreas, and liver; and biological processes of lipid metabolism, which are linked to obesity [65, 66].
In addition, microRNA-33 is an important regulator of lipid metabolism, regulating insulin signaling and fatty acid regulation, and may be a therapeutic target for treating MetS [67].
Mitochondrial dysfunction is an important cause, leading to diabetes [68].
DNA damage in mitochondria and vascular cells can have a detrimental effect on mitochondrial function, increasing ROS production and promoting atherosclerosis [68, 69].
Variations in mitochondrial DNA can also lead to MetS, hyperinsulinemia, and T2DM [70, 71, 72].
In addition, insulin also regulates DNA repair, and a chronic hyperglycemic state can damage DNA, contributing to genomic mutation, which can be associated with cancer [73].
Abnormal protein secretion can affect metabolic traits, proven through studies demonstrating increased secretion of fatty acid-binding protein 4 (FABP4), and frizzled-related protein 4 (SFRP4), associated with obesity, insulin resistance, and abnormal insulin sensitivity [74, 75, 76].
Retinol-binding protein 4 (RBP4), expressed in adipocytes and liver, showed a positive correlation with MetS in a Chinese population, associated with insulin resistance and dyslipidemia [77, 78].
The exact association between the pathway of non-motor microtubule binding and MetS can not be explained.
However, studies have shown non-motor microtubule binding sites to have an important role in mitosis and to be essential in the embryonic development of Drosophila [79].
More studies will be needed to elucidate the association between non-motor microtubule binding and MetS.
One of the strengths of this study is the usage of a pathway-based approach to further analyze the KARE GWAS datasets.
Pathway-based approaches of MetS in Asians are relatively scarce.
Therefore, this study might help further elucidate the pathophysiology of MetS.
Although the pathways identified in our study did not show an FDR value &lt; 0.05, important pathways related with BMI, lipid and glucose metabolism, including signaling by PDGF and PPAR signaling, had an FDR &lt; 0.2.
Other pathway-based approaches will be needed to further validate the identified pathways.
Pathway-based analysis has its strengths in improving the interpretability of the GWAS.
However, current pathway analysis tools are limited in finding a well-defined pathway, and their isolated characteristics make it hard to combine them with other analyses [80].
In addition, the limited knowledge base and imprecision of gene annotations restricts their usage and integration with other analysis methods [81].
Therefore, these limitations will need to be improved to generalize this approach and increase its applicability.
In conclusion, through pathway analysis of MetS, significant biological pathways associated with lipid and glucose metabolism could be identified, and these results might contribute to the understanding of MetS.
Title: Clinical and biochemical characteristics of subjects with metabolic syndrome and controls

Title: Significant SNPs associated with metabolic syndrome in the discovery phase of GWAS

Title: Top ten significant GSEA associated with metabolic syndrome at the 75th percentile cutoff
Title: Genome-Wide Association Study of Hepatitis in Korean Populations

Hepatitis is a common and serious disease for the Korean population.
It is caused by a virus, the A and B types of which are plentiful in Koreans.
In this study, we tried to find genetic factors for hepatitis through genome-wide association studies.
We took 368 cases and 1,500 controls from Anseong and Ansan cohort data.
About 300,000 single-nucleotide polymorphisms and 20 epidemiological variables were analyzed.
We did not find any meaningful significant single nucleotide polymorphisms, but we confirmed the influence of major epidemiological variables on hepatitis.
Hepatitis is inflammation of the liver, most commonly caused by a viral infection [1].
Five main hepatitis viruses are known, referred to as types A, B, C, D, and E. We are concerned with these main types because of the burden of illness and death; they also have the potential for outbreaks and epidemic spread.
In particular, types B and C lead to chronic disease in hundreds of millions of people and together constitute the most common cause of liver cirrhosis and cancer [1].
The Korean Health Insurance Review and Assessment Service [2] reported on C type hepatitis patients in Korea, described in Table 1.
Table 1 shows that the number of patients has been increased steadily.
The prevalence rate of hepatitis type C in Koreans is 1%-1.5%.
From the prevalence rate, 500,000-600,000 patients are estimated.
Only 10% of them are treated.
If we find risk factors for hepatitis, it is helpful for the prevention and treatment of hepatitis.
In this study, we performed a genome-wide association study of hepatitis in Korean populations.
We tried to find significant single-nucleotide polymorphisms (SNPs) and epidemiological traits related to hepatitis.
Title: Phenotype and genotype data

The study subjects are based on the Anseong and Ansan cohort data, part of the Korea Association Resource (KARE) projects.
The genotypes and phenotypes of the cohort population are described in Cho et al.[3].
Subjects with genotype accuracies below 98%, high missing genotype call rates (≥4%), high heterozygosity (&gt;30%), or inconsistency in sex were excluded from subsequent analyses.
Individuals who had a tumor were excluded, as were related individuals whose estimated identity-by-state values were high (&gt;0.80).
After these quality control steps, 352,000 SNP genotypes for 8,842 individuals were selected [4].
The epidemiological trait data for these individuals were also from the KARE project.
Among the total of 8,842 individual cases, 368 had hepatitis with age over 30, and 1,500 controls were randomly selected from non-hepatitis individuals with age over 30.
Table 2 summarizes the clinical characteristics of the phenotypes in this study.
The chosen dataset was imbalanced; the number of cases was smaller than controls.
A dataset is imbalanced if it contains many more samples from one class than from the rest of the classes [5].
In this case, the classification analysis showed good accuracy in the majority class but very poor accuracy in the minority class.
Therefore, we needed to transform the imbalanced dataset to a balanced dataset.
We applied an &#39;oversampling&#39; scheme [5] to overcome the imbalance problem.
The final dataset contained 1,500 controls and 1,500 cases.
Title: Statistical analysis

To find significant SNPs, we used PLINK, version 1.07 [6].
Other statistical analyses were performed using R, version 3.1.
We used logistic regression to find major factors related with hepatitis.
Receiver operating characteristic (ROC)/area under the curve (AUC) analysis was performed to confirm the prediction power of the major factors that were found.
Title: Genome-wide association studies

Table 3 summarizes 20 SNPs that were top-ranked by p-value in the genome-wide association analysis.
Unfortunately, there were no significant SNPs that met p &lt; 5 × 10-8.
We performed logistic regression test on the top-ranked SNPs, and we took 10 SNPs in Table 4.
Logistic regression measures the relationship between a categorical dependent variable (phenotype) and one or more independent variables (SNPs).
Fig.1 shows the ROC plot for the classification test using the 10 SNPs.
The AUC value from the ROC plot is 0.700; it is not enough as a biomarker.
Title: Epidemiological studies

Using the traits in Table 2, we performed a logistic regression test.
Table 5 summarizes the results.
As we can see, diabetes, gastritis, allergy, external head injury, taking arthritis drug, and degree of albumin were highly correlated with hepatitis.
In the case of diabetes, the probability that a diabetes patient had hepatitis was 4 times higher than a diabetes-free person.
In general, the hepatitis C virus is often associated with diabetes, and some diabetics may even develop chronic hepatitis [7].
Gastritis is influenced by hepatitis.
If we have hepatitis, the probability of getting gastritis is increased 1.5 times.
Especially, chronic gastritis develops by chronic hepatitis [8].
Sometimes, allergy cahepatitis.
It increases the probability of hepatitis by 1.8 times.
Hepatitis virus can cause arthritis [9]; it increases the probability of arthritis by 1.9 times.
The degree of albumin is inversely proportional to hepatitis (odds ratio [OR], 0.8), because the liver makes albumin, and hepatitis enervates the process.
The OR between external head injury and hepatitis is very high (OR, 1.31).
We cannot explain the medical relationship between them.
It needs more analysis.
Table 6 shows AUC values from the variables in Table 5.
As we can see, four variables can explain 64.7% of the cause of hepatitis, and 8 variables can explain 69.3%.
Fig.2 shows the ROC plot for eight variables in Table 3.
From the epidemiological analysis, we found relevant variables with hepatitis.
We confirmed that hepatitis has a wide relation with other diseases.
If we make a disease network in which the node is a disease and the edge is a correlation coefficient between two nodes, we can understand the relationship among diseases more clearly.
Current known disease networks [10, 11] do not show detailed relationships between hepatitis and other diseases.
This is a future research topic.
KARE data are the result of a cohort study.
It contains a small number of samples for specific diseases, whereas the whole population is very big.
It induces an imbalanced dataset for statistical analysis.
Our study implies a basic limitation, even though we tried to complement the problem.
We also did not find any significant SNPs related with hepatitis.
If we combine the knowledge of other biological databases, we may get a more meaningful interpretation for the results of our experiment.
Receiver operating characteristic plot for 4 single nucleotide polymorphisms derived from logistic regression (area under the curve, 0.700).
Receiver operating characteristic plot for 8 epidemiological variables derived from logistic regression (area under the curve, 0.693).
Title: Statistics of hepatitis type C

Title: Clinical characteristics of variables in this study

Title: Top-ranked SNPs of genome-wide association analysis

Title: Logistic regression test for SNP data

Title: Logistic regression test for epidemiological data

Title: Area under the curve (AUC) values
Title: Effective Population Size of Korean Populations

Recently, new methods have been developed for estimating the current and recent changes in effective population sizes.
Based on the methods, the effective population sizes of Korean populations were estimated using data from the Korean Association Resource (KARE) project.
The overall changes in the population sizes of the total populations were similar to CHB (Han Chinese in Beijing, China) and JPT (Japanese in Tokyo, Japan) of the HapMap project.
There were no differences in past changes in population sizes with a comparison between an urban area and a rural area.
Age-dependent current and recent effective population sizes represent the modern history of Korean populations, including the effects of World War II, the Korean War, and urbanization.
The oldest age group showed that the population growth of Koreans had already been substantial at least since the end of the 19th century.
Effective population size (Ne) is the theoretically useful population size of an ideal population, in which the influence of random genetic drift is identical to that of the actual population [1].
Therefore, the effective population size is a good representative of the genetic potential of the actual population; yet, it is easier to handle population genetic problems.
There are several kinds of effective population size, depending on the focus of measurement [2, 3, 4, 5], and the most representative ones are the inbreeding effective population size, the variance effective population size, and the eigenvalue effective population size.
The variance effective population is easy conceptually, and it can be estimated based on the gene (or allele) frequencies [6, 7] or heterozygote excess [8].
In addition, the estimation can be based on linkage disequilibrium (LD) [9], and advanced related methods based on LD have been applied to human genome data to infer the past population history of human populations [10, 11, 12].
Recently, new methods have been developed to estimate the current Ne and recent changes in Ne [13, 14].
In studies, the effective population size of the current generation was estimated based on deviations from Hardy-Weinberg equilibrium (HWE), and an effective population size of recent generations was estimated using the linkage equilibrium (LE) between variants in different chromosomes [13].
The past population changes can be inferred from the LD decay, depending on recombination rate [14].
Different from previous methods using regressive methods [10, 11, 12], the relationship between the effective population size of the previous generation and that of the current generation [14] was presented by improving a recurrence formula of previous studies [15, 16].
The applications of these studies to human genome data showed excellent agreement with the actual recent changes of various human populations [13, 14].
HapMap Phase III data were used for studies to infer the past population histories of various human populations.
There were similar data for Korean populations (the Korean HapMap data) [17, 18]; however, for the continuity of the results, the data were not included for the analyses in previous studies [13, 14].
In 2007, right after the Korean HapMap data, the Korea Association Resource (KARE) was initiated to enable genomewide analyses for 8,842 individuals from the population-based cohorts [19].
The effort was successful in revealing new and population-specific genetic factors of various traits [19, 20, 21, 22].
The KARE data were also provided to many other researchers, which yielded many novel findings [23, 24, 25, 26, 27, 28, 29, 30].
In the current study, the current and recent effective population sizes of Koreans were estimated, and the past changes in effective population sizes were studied using the KARE data.
Different from the HapMap data, the KARE data contain many more individuals with their age information.
In addition to the analyses of the total population, region-specific and age-specific analyses were conducted in the current study.
Title: Estimating Ne based on HWE deviations, LE deviations, and LD decay

Based on the formula for testing HWE [31], estimating the effective population sizes of current populations utilizes the deviations of HWE, as shown in Eq.
(1) from a previous study [13].
(1) where n is the sample size, and the expectation indicates the mean value of HWE deviations.
AA is the observed genotype frequency of a variant, and AA is the observed allele frequency of a variant.
The same as the HapMap data, the KARE data excluded variants that deviated from HWE for a significance level of 0.001.
Therefore, similar to the previous study, the mean of the HWE deviations should be adjusted by dividing the mean by the correction term of 0.9873 [13].
A variant in a chromosome is in LE with a variant in another chromosome; however, the random genetic drift induces deviation from LE.
The deviations depend on the effective population size of recent generations, and the recent effective population size can be estimated using the deviations.
Assuming a constant population size, the effective population size that reflects recent changes in population size (the recent Ne) can be estimated based on Eq.
(2), which was derived in a previous study [13].
The estimate is different from the inbreeding Ne estimate using unlinked loci [9, 32].
(2) where n indicates the sample size.
As indicated previously [13], the correction due to sampling involves 1/2Ne, in addition to 1/n, because the haplotype frequencies were estimated using maximum likelihood estimation [9, 33, 34].
If the haplotypes are derived directly from the genotyping or sequencing, the correction term due to sampling should be 1/n.
Title: The decay of LD between linked loci is different, depending on the recombination rate.
Based on this information, the past population history can be inferred.
The LD estimates at a certain recombination rate contain the influences of all Ne values from the current to a certain number of past generations [14].
Therefore, the previous study showed that the changes of Ne estimates, depending on recombination rate, just represent an overall picture of past population history by comparing the actual estimates to the estimates of various past population histories.
Similar to the previous study, the effective population sizes were estimated using the linked variants, depending on the recombination rates, between the variants (Eqs.
1 and 3-5).
Following the previous assumptions [13, 34], constant population sizes and LD at equilibrium were assumed.
(3)

Title: Eq.
(3) presents the relationship between LD estimates at equilibrium (r2eq), Ne, and recombination rate (C).
The LD estimates in Eq.
(3) indicate the estimates of the original population, excluding the sampling bias.
Sampling causes more complicated relationships in the LD estimates of the linked variants than the LD estimates of unlinked variants, due to the maximum likelihood estimation of haplotype frequencies [14].
Therefore, as shown in Eq.
(4), the LD estimates of sampled populations (r2s) include the influence of the Ne of the original population instead of a simpler factor, 1/(2ns), in addition to the influence of sample size (ns) and the LD estimates of the original population (r2o).
(4)

Title: To eliminate sampling bias, the Ne of the original population should be known.
It is possible to combine and solve Eqs.
(3) and (4) for a given ns and r2s.
A better way is to use the Ne estimate of the current generation (Eq.
1) in the recurrent formula in a previous study [14].
By doing so, Eq.
(5) was derived.
(5)

In Eq.
(5), Ne indicates the effective population size at equilibrium for recombination rate (C); Nec indicates the effective population size of the current generation from Eq.
(1); r2o indicates the LD estimates of the original population; and r2eq indicates the LD estimates at equilibrium from the generation when the LD estimates reached equilibrium to the previous generation right before the current generation -i.e., the parent generation.
Therefore, to estimate Ne at equilibrium for a certain recombination rate, Nec should be estimated first.
The estimated Nec is incorporated into Eq.
(4) to derive r2o.
In Eq.
(5), substituting r2eq with the expression in Eq.
(3) provides a cubic equation of Ne in which Ne can be solvable.
The Ne estimates could be examined, depending on various recombination rates, to infer past population histories in which the impact of current Ne was excluded.
Title: Data

The KARE data were generated and kindly provided through a grant program by the Korea Centers for Disease Control and Prevention.
The data consisted of a total of 8,842 individuals: 4,637 from Ansan (an urban area) and 4,205 from Anseong (a rural area) in Gyeonggi province.
There were 32 groups depending on ages from 39 to 70, and the numbers of individuals in each of the 32 groups were: 21, 421, 469, 489, 485, 473, 426, 400, 368, 291, 325, 268, 215, 207, 241, 242, 259, 197, 225, 238, 231, 255, 239, 230, 232, 254, 270, 210, 254, 200, 194, and 13, respectively.
Because there were small numbers of individuals for the ages of 39 and 70, they were combined with the age groups of 40 and 69, respectively.
Therefore, the total number of age groups was 30.
It is neither necessary nor efficient to use all single-nucleotide polymorphisms (SNPs), because estimating Ne does not require many variants.
Only 10,000 SNPs or 1,000 pairs of SNPs were enough to estimate the correct Ne in the simulations of previous studies [13, 14].
In the simulation studies, the variances were least biased for the most frequent variants.
Therefore, the same as in previous studies [13, 14], SNPs with minor allele frequencies higher than 0.4 and without missing genotypes were used for the analyses.
The estimates of LD were the r2 estimates-i.e., the squared correlation coefficient between two variants.
The current effective population size was 100,778 using the total population.
The estimate was much larger than the estimates using the HapMap data.
The largest Ne in the populations of the HapMap data was 10,437 using CEUp (the parents of Utah, USA residents with ancestry from northern and western Europe).
The estimate of the Korean population was approximately 10 times larger than the largest estimates using the HapMap data.
From the definition, the effective population size is influenced by many factors, such as mating structure, migration, and admixture [1].
No big difference is expected in the mating structure between KARE and HapMap.
Because Korean populations were relatively rarely admixed, the inflation of Ne due to admixture could be excluded.
The effects of migration on Ne estimates were studied previously [35, 36].
The Ne, based on LD, which is more robust than the joint estimation of Ne and migration rate, could be overestimated to be close to the global (metapopulation) Ne when the migration rate is high [35].
After the Korean War, the Republic of Korea (South Korea) experienced severe urbanization and extreme population concentration in a metropolitan area [37].
The percentage of the population in an urban area was more than 70% of the total population in the late 1980s [37].
A high population density due to migration introduces good opportunities to meet a diverse partner within the population, which increases Ne.
The population density of the Republic of Korea is among the highest among nations, ranking 20th among a total of 265 countries or areas (https://data.un.org/Data.aspx?d=PopDiv&amp;f=variableID%3A14).
Due to the Korean War and urbanization, the Gyeonggi province, located in the middle of the Korean peninsula, has experienced severe migration.
Because the capital, Seoul, is located in the middle of Gyeonggi province, the region has also experienced a severe concentration of the population.
These factors might explain the large effective population size.
Fig.1 shows the LD status of the total population, the population of the urban area, and the population of the rural area, depending on recombination rates.
In the previous study, the r2 decay plots of chromosome 14, depending on recombination rates, were prominently higher than the plots of other chromosomes [14]; however, in the current study, none of the Korean populations showed increases in the r2 estimates of chromosome 14 (Fig.1 and Supplementary Fig.1 for the age groups).
However, when the region from 65,700 kb to 6,800 kb, which is responsible for the increases in the r2 estimates in the previous study, was examined, similar allele frequency spectra were seen (Fig.2A).
The frequency spectra show significant concentrations of specific minor allele frequencies (higher than 0.4), indicating selection pressures in the region.
Other populations in the HapMap data showed similar trends of selection pressures, but the concentrated frequencies usually were less than 0.4 [14].
The most similar spectra among the HapMap data were JPT (Japanese in Tokyo, Japan) and CHB (Han Chinese in Beijing, China), and the frequency spectrum of KARE looked like a mixture of the spectra of JPT and CHB.
In a previous study [14], CHD (Chinese in metropolitan Denver, CO, USA) was almost the same as CHB.
Although the distributions were more dispersed, ASW (African ancestry in southwest USA) and MEX (Mexican ancestry in Los Angeles, CA, USA) were similar to CHB.
The genotype data were generated using Affymetrix Genome-Wide Human SNP array 5.0 [19]; however, the genotype data of the HapMap project were generated using two platforms of the Illumina Human 1M and the Affymetrix SNP 6.0 [38].
Because of the difference in the platforms used, the current study showed a smaller number of SNPs in the region (Fig.2A).
If the same platforms were used for genotyping, similar increases in the r2 estimates of chromosome 14 to JPT and CHB might be observed in Fig.1 [14].
The region contains the complete gephyrin gene (GPHN).
In Fig.2B, a huge LD block of the region was observed using Haploview [39], similar to a previous study [14].
The past population history of the total population is shown in Fig.1.
Excluding the influence of current effective population size, the past population history of Koreans was similar to the CHB of the HapMap data.
The results looked reasonable, considering the similarity of the modern histories between Korea and China, which include the resistance to Western civilizations and the short period of colonization.
However, there is a huge difference between the current Ne of Korean populations and the current Ne of CHB.
The current Ne of CHB was 2,926 [14].
The Chinese government has enforced a one-child policy since 1978; however, the Korean government encouraged a two-child policy without any forcible action [40].
These differences in policy might have resulted in significant differences in the current Ne sizes of the Korean and Chinese populations.
The populations of both the urban and rural areas showed similar past population histories as that of the total population (Fig.1).
Fig.3 shows the estimates of the current and recent effective population sizes of various age groups.
Because the initial sample collection was conducted from 2001 to 2003, the year of birth for individuals aged 69 to 40 would be 1933 to 1962, respectively, if the baseline year of collection were assumed to be 2002.
Therefore, there could be inaccuracies of 1 or 2 years in assigning the year of birth.
The recent Ne involves recent changes in the effective population sizes, usually within a few generations.
Therefore, the recent effective population sizes were larger than the current effective population sizes in Fig.3, because the current Ne of individuals born in a certain year merely represents the effective population size of the population born in a certain year.
However, the recent Ne represents the effective population sizes of several previous generations.
The current effective population sizes were mostly between 5,000 and 30,000, and the recent effective population sizes were usually higher than the current Ne and increased as the year of birth increased.
The current and recent Ne of samples with age 69 or older (born in 1933) was 9,457 and 20,165, respectively.
For comparison, the maximum estimate of the current Ne of the HapMap data was 10,437.
The current Ne of Koreans born in 1933 was already large, probably because of the high population density in Korea.
The recent Ne, which is double the current Ne, indicates that the population growth of Korea lasted long before the colonial domination by Japan in 1910.
There were two age groups of 58 and 63, for which the current Ne estimates were less than 5,000.
The corresponding years are 1944 and 1939, respectively.
World War II began in 1939 and ended in 1945, and comparatively fewer births would be expected during the period.
However, there was a large increase of the current Ne in 1943, presenting a nettlesome question.
There were several large increases of the current Ne in 1943, 1947, 1952, and 1958.
The increases in 1947 and 1958 could be explained by the baby boom right after the wars; however, 1943 and 1952 were the 2 years and 1 year, respectively, before the end of the wars, which lasted for 6 and 3 years, respectively.
The current and recent Ne estimates were compared to the number of the total population and to the rates of population growth recorded in the Korean Statistical Almanac of the Korean Statistical Information Service (http://kosis.kr).
The rates of population growth fit with the increased current Ne in 1943 if the 1-year inaccuracies in assigning the year of birth are considered.
The increments of the current Ne in 1943 and 1952 could be due to the decreased crude death rate and increased migration rates [37].
Using the KARE data, the current and recent effective population sizes of Korea were estimated, based on HWE and LE deviations, and the past changes in population sizes were derived, based on the pattern of LD decay.
The results showed clear exponential growth of the Korean population, similar to CHB of the HapMap data.
The population growth of China has been positive continuously since 1400, except for the period from 1683 to 1700 [41].
The censuses in the Joseon Dynasty period of Korea showed low rates of population growth in the 1800s [37].
The large recent Ne of the 69-year-old age group (born in 1933) indicates that Korea might have experienced continuous population growth at least in 1873, assuming a generation time of 20 years for the 3 most recent generations.
A simulation result of rapid growth (Ne changes: 1000, 2000, 4000, 10,000) showed half (5,212) the current Ne [13]; however, the recent Ne of the 69-year-old age group was double the current Ne.
Therefore, it is likely that the population growth started much earlier than 1873 [13, 14].
In the metropolitan area, people from all over the country might have increased the Ne.
The migration effects and the discrepancy between the census and the results need more explanations.
The age-dependent Ne estimates showed good concordance with the actual modern history of Korea, although several of them require more explanation.
Because of the inaccuracies in assigning the year of birth, a sample of the 58-year-old age group might be mixed with a sample of the age groups of 57 and 59.
Although it is a rare occasion, individuals having genomic potential to increase current Ne in the 57 and 59 age groups might have been assigned to the 58-year-old age group accidently during the sample collection.
In any case, the population growth in war time needs more explanation.
In Fig.3, the trend of recent Ne followed the trend of current Ne in many cases.
Especially, the increased current Ne in 1943 contributed to the rapid increase in the recent Ne in the late 1950s as the individuals born in 1943 could contribute to reproduction.
The recent Ne decreased rapidly in the 42-year-old age group (born in 1960), which needs explanation.
The baby-boom individuals born in 1958 were 2 years old and could not contribute to giving birth.
The recent Ne increased again in 1961 and 1962, possibly due to the contributions of individuals who were born in the previous baby-boom period and reached reproductive age.
In the previous studies, it was not necessary to analyze a large number of variants.
However, the current study showed that the number of genotypes, as well as the number of individuals, might be important for the quality of the results to infer the past population history.
In Supplementary Fig.2, the estimates of many age groups were less consistent, even with larger numbers of samples than the sample sizes of the HapMap data.
The smallest sample size for the former was 197, and the largest sample size for the latter was 143.
However, the number of genotypes would not have had much influence much on the estimations of the current and recent Ne, as indicated in previous simulation studies [13].
Therefore, the possibility of lower genotype quality could not be excluded, either.
It is also possible that inconsistent estimates were seen more in the CHB than in other populations, which showed the most rapid population growth.
Because Korean populations showed more rapid population growth than CHB, the rapid growth might be the reason for the inconsistent estimates.
With more genotypes, the increases in r2 estimates in chromosome 14 could also be seen clearly in Fig.1, similar to CHB and JPT of the HapMap project.
The recent Ne of the total population was 2,518,501, which was incredibly large.
Considering that the current Ne of the total population was 100,778, the KARE data showed that the population is under serious deduction of effective population size.
The ratio of the current to recent Ne was 0.04.
This small ratio might be due to migration and extreme population concentration to a metropolitan area.
Migration might have had more influence on the recent Ne rather than the current Ne, because the recent Ne reflects the population that moved from all over the country.
In addition, the division of the Korean peninsula after the Korean War might have resulted in the extreme ratio.
The population moved to the metropolitan area from all over the country, but they could not move back to the northern part of the Korean peninsula.
The ratio was not comparable to any of the HapMap data, in which 0.64 of JPT was the lowest, and all populations were concordant with their recent population changes.
In Fig.3, the tendency started in 1933 for the eldest ones in the data, and it became more severe as the age of the sampled individuals decreased.
The total sample contains more relatively young individuals than elders; however, the ratio of 0.04 was too small.
If the ratios were correct, as shown in the HapMap data, Fig.3 shows that the significant reduction in effective population size began in 1953, right after the Korean War, probably due to migration and the division of the Korean peninsula.
As indicated previously [13], the Ne estimates do not represent the actual population size exactly.
Further studies for the effects of migrations and confinements with refined samples and data could be helpful.
The estimates of the current effective population size were 91,433 in the rural area and 76,097 in the urban area.
The result was surprising, because the urban area is more populated than the rural area.
The age distributions of each region differed significantly (Supplementary Fig.3A).
The sample of the urban area contains a smaller number of elders and a larger number of relatively young individuals; however, the sample of the rural area shows relatively even distribution, with a slightly larger number of elders (Supplementary Fig.3A).
If the current effective population sizes, depending on age, were examined (Supplementary Fig.3B), the current effective population sizes of the rural area were relatively even compared to the current effective population sizes of the urban area.
Some of the estimates were negative, which generally indicate large Ne values and sampling failures, considering the results of previous studies [13].
For the current Ne estimates of the urban area, the estimates were usually small, except for several estimates of relatively younger ages.
The overall small uneven estimates and the uneven age distribution of the urban area might have resulted in the smaller current Ne estimate than the estimate of the rural area.
The recent Ne of the rural area was 495,871, but the estimate of the recent Ne of the urban area was a negative value.
As mentioned previously, the negative value might have come from a large Ne and sampling failure.
Therefore, the negative recent Ne estimate of the urban area probably indicates an extremely large recent Ne.
More studies are necessary to determine the effects of uneven age distributions, rapid population fluctuations, migrations, and population confinements on the Ne estimates.
Linkage disequilibrium decay and Ne estimates of the total population, the population of the urban region, and the population of the rural region, depending on recombination rate.
Region from 65,700 kb to 6,800 kb of chromosome 14.
(A) Distribution of minor allele frequencies.
(B) Linkage disequilibrium block.
CHB, Han Chinese in Beijing, China; JPT, Japanese in Tokyo, Japan; KARE, Korean Association Resource.
Current and recent effective population sizes, depending on age groups, which were converted to years of birth (total population: the census at the end of the year; rate of population growth: the population growth per 1,000 from the census at the end of the year).
Title: Relevance Epistasis Network of Gastritis for Intra-chromosomes in the Korea Associated Resource (KARE) Cohort Study

Gastritis is a common but a serious disease with a potential risk of developing carcinoma.
Helicobacter pylori infection is reported as the most common cause of gastritis, but other genetic and genomic factors exist, especially single-nucleotide polymorphisms (SNPs).
Association studies between SNPs and gastritis disease are important, but results on epistatic interactions from multiple SNPs are rarely found in previous genome-wide association (GWA) studies.
In this study, we performed computational GWA case-control studies for gastritis in Korea Associated Resource (KARE) data.
By transforming the resulting SNP epistasis network into a gene-gene epistasis network, we also identified potential gene-gene interaction factors that affect the susceptibility to gastritis.
Gastritis is a common disease in Korea.
There is a report that 9.9% of Koreans have gastritis [1].
Gastritis is associated with the potential risk of developing gastric cancer or peptic ulcer disease [2].
A common cause of gastritis is Helicobacter pylori infection [3].
However, in addition to the cause, other genetic and genomic factors have been examined for gastritis.
Human leukocyte antigen class II allele is an important risk factor for chronic atrophic gastritis and gastric carcinoma in Koreans [4].
Moreover, many studies have reported that single-nucleotide polymorphisms (SNPs), including rs1143627 in interleukin-1 beta (IL1B), rs4073 in interleukin 8 (IL8), rs4986790 in Toll-like receptor 4 (TLR4), and rs16260 in cadherin-1 (CDH1), are associated with susceptibility to gastric diseases, especially gastric cancer [5, 6, 7, 8].
The results from SNP studies show the importance of research for the association between SNP and disease, based on the association and functional relationship between gastritis and gastric carcinoma [9].
In other words, a casual SNP in one disease may be a genetic factor for another disease.
However, previous studies only targeted the effect from each single variation; results have been rarely obtained for high-order genetic interactions in gastritis from genome-wide association (GWA) studies to date.
In this study, we performed a GWA case-control study for gastritis using SNP genotype data of the Korea Associated Resource (KARE) project [10].
Our study was focused on detecting significant epistatic SNP-SNP interactions and the resulting gene-gene interactions that are putative causal factors in gastritis.
A validation and functional analysis of the result was performed on the obtained relevance gene-gene epistasis network.
Our network construction method is based on the previous relevance network approach [11].
To measure the strength of the association between a pair of SNPs and gastritis, we used the mutual information that has shown to be effective in detecting epistasis in case-control GWA studies [12, 13].
An efficient permutation scheme was adopted to extract significant interaction pairs, and we also approximated the p-values by exploiting the relationship between the mutual information value and the χ2 value [14].
We further transformed this SNP network into a relevant gene-gene epistasis network to validate the biological significance of our findings.
A functional analysis was performed on the gene-gene network, and the topological properties of the network were also investigated.
Title: Pre-processing for KARE data

The KARE genotype data we used in the study initially consisted of 352,228 SNPs and 8,842 samples after the screening by genotype calling and quality control performed [10].
We performed the following additional stages of pre-processing.
First, we selected case-control samples based on the survey of the disease history of the patients, which was carried out in the KARE project.
In the first stage, we collected 1,885 patients who self-reported that they had gastritis in the past as cases.
We also found 4,117 individuals who self-reported that they had no history of having gastritis or any other diseases.
Among those patients, we randomly selected 1,885 individuals as controls to avoid bias in the study.
After the selection of case-control patients, we filtered out SNPs that corresponded to the following conditions: minor allele frequency &lt;0.01 in each group [15], pairwise linkage disequilibrium r2 &gt; 0.8 [16], and SNPs in the X chromosome.
PLINK [17] was used for the calculation of these values.
The resulting dataset consisted of 185,426 SNPs and 3,770 samples for the case-control study.
Title: Mutual information

We used mutual information measures to assess the strength of the association between a pair of SNPs and the disease status of gastritis.
Mutual information has been widely used to measure dependence or independence between two random variables [11, 12, 18, 19].
It is a non-parametric measure and is able to detect both linear and non-linear associations [14].
This measure is based on Shannon&#39;s entropy, H(X) = Σx∈X - p(x)log(p(x)), which shows the uncertainty of the random variable X.
Mutual information I(X;Y) between random variables X and Y is defined by the composition of entropy as follows: I(X;Y) = H(X) + H(Y) - H(X,Y).
H(X), H(Y) denote entropies for the random variables X, Y, and H(X,Y) denotes the joint entropy of the two random variables as follows: H(X,Y)=Σx∈XΣy∈Y - p(x,y)log(p(x,y)).
A high mutual information value indicates a strong association between two random variables.
The measure can also be extended to assess the strength of association between a pair of SNPs and a phenotype.
The extended version of mutual information is as follows: I(X1,X2;Y) = H(X1,X2) + H(Y) - H(X1,X2,Y), where X1 and X2 are random variables for two SNPs, and Y denotes random variables for the disease.
In our recent work [12], we have shown that this measure could identify high-order epistatic interactions both with and without marginal effects by using simulation models, such as in Culverhouse et al.
&#39;s [20] and Velez et al.
&#39;s studies [21].
Title: As the genotype and disease status are represented as discrete values, it is more convenient to consider the random variables as a partition of the combination of the genotypes and disease status.
Then, the entropy of a random variable X can be represented in terms of the partition as follows:

Title: where X = {A1, A2,…, An} is a partition on the set of samples S = {A1∪A2∪…∪An}, and no intersections exist between elements in the partition.
The joint entropy of two random variables for the partition of S, X = {A1, A2,…, An} and Y = {B1, B2,…, Bm} is defined as follows:

The entropy also can be extended to the joint entropy of multiple random variables (e.g., 3 or 4 SNPs) naturally.
Title: Extraction of statistically significant epistatic interactions

As mentioned above, mutual information is a non-parametric measure, and the distribution of values from the population is unknown; therefore, it is difficult to show the statistical significance of values calculated by this measure.
Rather than doing the computationally too-expensive permutation tests for every pair of SNPs, which causes severe multiple testing issues when applied to GWA studies, we adopted the alternative permutation scheme proposed [11].
First, we replicated 30 permutations for disease status in case-control samples.
For every possible pair of given SNPs, we calculated mutual information of SNPs with the permutated disease status labels and obtained the average value from 30 replications.
θ denotes the maximum value of the averages.
If a mutual information value between SNPs and the phenotype from real data is higher than θ, then we consider that the pair of SNPs shows a more significant association with gastritis than a random association.
To further assess the statistical significance of the identified SNP pairs, we approximated the p-values of the SNP pairs by exploiting the following relationship between χ2 value and mutual information [14]: χ2 ~ 2Nln2·I(X1,X2;Y), where N denotes the number of patients in the study.
Title: Relevance network construction and assessment of significance

Network analysis is a powerful tool to understand biological systems.
Given the significant epistatic interactions between SNP pairs and disease status detected by the permutation scheme, we constructed SNP-SNP epistatic interaction networks, where SNP represents the node and the significant interaction between SNP pairs represents the edge.
However, it is difficult to assess the biological significance of the interaction networks directly because of the lack of interaction databases for SNPs.
To overcome this limitation, we directly mapped these networks into a gene-gene relevance network.
Fig.1 represents a brief scheme of how to map the network.
Suppose that some SNPs map directly to genes A and B, and there are at least two SNPs that have significant interactions for disease status.
If one SNP maps to gene A and another SNP maps to gene B, then we consider that genes A and B have a unidirectional edge.
The weight of the edge is defined by number of the interacting SNP pairs.
For example, in Fig.1, there are four SNP interactions between gene A and gene B.
Thus, we consider that gene A and gene B have an edge, the weight of which is 4 in the gene-gene network.
Finally, the top 5% edges having the largest edge weights are used in the biological validation and topological investigation.
We constructed a gene-gene network for each chromosome showing intra-chromosome interactions.
We measured the network topologies of each network using Cytoscape [22] and also ran a gene ontology (GO) enrichment analysis for sets of genes in the network.
We used DAVID [23] to validate the biological significance of the networks.
Fig.2 illustrates the overall analysis scheme used in this study.
Title: Statistically significant epistatic interactions in each chromosome

We ran the permutation method for each individual chromosome separately.
As a result, we obtained SNP pairs that were non-randomly associated with the status of gastritis for the given patients.
Fig.3 shows the number of such SNP pairs for each chromosome.
We found that there were approximately 2%-4% associated pairs among all possible pairs in the chromosomes.
We then calculated the p-value for each SNP pair by using the approximation scheme [14] and under Bonferroni correction [24].
Table 1 shows the number of statistically significant SNP pairs within each chromosome.
Significant SNP pairs and p-values are listed in Supplementary Table 1.
In total, 293 SNP pairs showed statistical significance in the study.
Chromosomes 9 and 15 had many significant pairs, but many of those pairs included SNPs with marginal effects (rs169730 in chromosome 9, rs493971 in chromosome 15).
There were few significant SNP pairs within the chromosomes, and these pairs were not found in previous studies of gastric disease; therefore, further investigation of the pairs is needed to assess their biological significance.
Title: Network topologies

Table 2 summarizes the results of the network topology measures (number of nodes and edges, clustering coefficient, and network centralization) for the gene-gene interaction network for each chromosome.
The table shows that all chromosomes showed similar properties: the number of edges is 10-30-fold higher than the number of nodes, and the measurements reveal high clustering coefficients and network centralization.
Additionally, every network from each chromosome consisted of only one component.
Figs.4 and 5 present the structure of networks showing the highest values for the measures, especially the clustering coefficient and network centralization.
In the real world, many biological networks show high average clustering coefficients [25].
The network in chromosome 18 showed the highest clustering coefficient, and we found some hub genes in the network that were reported to be associated with gastric disease.
For example, Uchino et al.
[26] reported frequent loss of heterozygosity at the deleted in colorectal carcinoma (DCC) locus in gastric cancer.
Higher network centralization values show that hub genes highly affect the network [27].
In other words, if the hubs are removed, then the network may divide into several components.
The network in chromosome 20 showed the highest value for the centralization measure.
In the network receptor-type tyrosine-protein phosphatase T (PTPRT), one of the hub genes was reported to be one of 15 genes in CpG islands showing significant differential methylation in gastric carcinoma [28].
From the results, we determined that there were many close gene-gene interactions derived from SNP-SNP interactions for gastritis and that if a gene had a high degree in the network, then this gene was connected to other genes with which it had frequent SNP-SNP interactions.
We also found this tendency in most chromosomes.
Title: Enrichment analysis

We ran an enrichment analysis of the gene-gene interaction network for GO using DAVID [23].
Table 3 summarizes the significantly enriched terms from DAVID.
Every term that was annotated in the table was significant under a false discovery rate (FDR)-based correction (p &lt; 0.05 after adjustment).
Chromosome 19 had the most enriched terms among chromosomes, and these terms were related to regulation (biological process) and binding (molecular function).
Fig.6 presents the internal structure of the sub-network for transcription (GO:0006350) in chromosome 19.
This network has many zinc finger (ZNF) family genes.
Taniuchi et al.
[29] reported that zinc-binding protein-89 (ZBP-89), which is related to the ZNF gene family and is a Krüppel-type zinc finger protein, is overexpressed in gastric cancer patients.
Additionally, interleukin-1 receptor activity (GO:0004908) was significantly enriched in chromosome 2.
Fig.7 presents the internal structure of the gene-gene relevance network for the GO terms.
As mentioned in the introduction, interleukin family genes and intra-SNPs were reported [5, 7]; therefore, these genes and SNPs are associated with the susceptibility to gastric cancer.
Genes that were enriched in the GO terms were not connected in the relevance gene-gene network; still, these genes had few SNP-SNP interactions with every other gene (weight equal or less than 11).
We also ran an enrichment analysis for the Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway [30], but no terms were significant under FDR-based adjustment.
In this study, we applied a simple but powerful method to detect epistatic interactions that distinguish gastritis patients from controls.
We found several putative SNP-SNP interactions using the mutual information measure.
Additionally, relevance gene-gene interaction networks that are derived from epistatic interactions show high clustering coefficients and centralization.
We found that some sub-networks in the recovered network had biological significance for gastric disease from the GO enrichment analysis.
However, we did not find these results to have direct associations with gastritis, because gastritis has been less extensively studied than other gastric diseases, such as gastric cancer or ulcer.
We expect that future studies on gastritis will be necessary to validate the results from our study.
In this study, only intra-chromosome interactions were considered.
We expect that if we extend the method to inter-chromosomal analyses, this could increase the chances of finding novel sub-networks that are enriched in pathways for gastritis.
However, this extension will impose a computational burden and increase the type I error; therefore, the development of a more flexible framework is needed to resolve this issue.
Scheme for mapping the SNP-SNP interaction network onto the gene-gene epistasis network.
SNP, single-nucleotide polymorphism.
Illustration of the network construction method.
GO, gene ontology; M.I., mutual information; SNP, single-nucleotide polymorphism.
Number of significant single-nucleotide polymorphism pairs for each chromosome.
Graphical visualization of the gene-gene interaction network in chromosome 18, which shows the highest clustering coefficient (0.848).
Red edges represents frequent SNPSNP interactions between two genes (&gt;100).
SNP, single-nucleotide polymorphism.
Graphical visualization of gene-gene interaction network in chromosome 20, which shows the highest network centralization (0.953).
Red edges represent frequent SNPSNP interactions between two genes (&gt;100).
SNP, single-nucleotide polymorphism.
Graphical visualization of the internal structure of the sub-network, which consists of genes related to transcription (GO:0006350).
Red edges represent frequent SNP-SNP interactions between two genes (&gt;100).
GO, gene ontology; SNP, single-nucleotide polymorphism.
Graphical visualization of the internal structure of the sub-network, which consists of genes related to interleukin-1 receptor activity (GO:0004908).
Red edges represent frequent SNP-SNP interactions between two genes (&gt;10).
GO, gene ontology; SNP, single-nucleotide polymorphism.
Title: Number of significant SNP pairs in each chromosome

Title: Network topologies for gene-gene interaction networks for each chromosome

Title: Significantly enriched GO terms for each chromosome
Title: Genome-Wide Association Study Identifies Candidate Loci Associated with Platelet Count in Koreans

Platelets are derived from the fragments that are formed from the cytoplasm of bone marrow megakaryocytes-small irregularly shaped anuclear cells.
Platelets respond to vascular damage, contracts blood vessels, and attaches to the damaged region, thereby stopping bleeding, together with the action of blood coagulation factors.
Platelet activation is known to affect genes associated with vascular risk factors, as well as with arteriosclerosis and myocardial infarction.
Here, we performed a genome-wide association study with 352,228 single-nucleotide polymorphisms typed in 8,842 subjects of the Korea Association Resource (KARE) project and replicated the results in 7,861 subjects from an independent population.
We identified genetic associations between platelet count and common variants nearby chromosome 4p16.1 (p = 1.46 × 10-10, in the KIAA0232 gene), 6p21 (p = 1.36 × 10-7, in the BAK1 gene), and 12q24.12 (p = 1.11 × 10-15, in the SH2B3 gene).
Our results illustrate the value of large-scale discovery and a focus for several novel research avenues.
Blood circulates through the body, affecting the tissues by delivering oxygen and nutrients responsible for tissue viability.
The size, number, and concentration of a cell in the blood vary among populations, and these have been regarded as factors influencing various disorders, such as erythrocytosis, anemia, hypertension, and cardiovascular diseases.
During the last 2 decades, genetic studies have revealed that there exists a genetic effect on several hematologic variables.
Moreover, previous studies reported that genetic factors strongly influence the variation in the counts and size of blood cells [1, 2].
Although a few genes are known to affect hematologic traits, the physiological mechanisms underlying the traits are largely unrevealed.
Platelets are anuclear cytoplasmic fragments that play a key role in maintaining primary adhesion, aggregation, and secretion and providing procoagulant surface and clot retraction.
The primary functions of a platelet count are to assist in the diagnosis of bleeding disorders and to monitor patients who are being treated for any disease involving bone marrow failure.
Low platelet counts or abnormally shaped platelets are associated with bleeding disorders, whereas high platelet counts sometimes indicate disorders of the bone marrow.
Platelet activation is known to affect genes associated with vascular risk factors, as well as with arteriosclerosis and myocardial infarction.
Platelet count is a readily available laboratory test and has been associated with different clinical and epidemiologic factors and are tightly regulated and inversely correlated in the healthy population [3].
Furthermore, because platelet count differs between inter-ethnic groups, gender and ethnicity should be important considerations for a platelet count study [4].
Recently, genome-wide association studies (GWASs) for hematological traits have been reported [5, 6, 7].
Particularly, 68 single-nucleotide polymorphisms (SNPs) associated with platelet count were identified by a GWAS [8].
However, although the heritability of variation in platelet count ranged from 54% to more than 80% [9, 10, 11], the genetic variants reported to date explain only a small fraction of the heritability in platelet count [12].
Therefore, new studies have opportunities to unveil additional genetic variants for explaining missing heritability [12].
In this study, we conducted a GWAS to find a contribution of the genetic basis on hematologic variables, such as platelet count, in Koreans.
Title: Subjects

We conducted a GWAS using individuals sampled from a part of the Korean Genome Epidemiology Study (KoGES).
The study subjects were recruited from population-based cohorts in the regions of Anseong and Ansan.
The standardized examinations applied in this survey included 10,038 participants aged 40 to 69 years.
Subjects for the replication study were recruited from the Health2 cohorts, another population-based cohort comprising the Wonju, Pyeongchang, Gangneung, Geumsan, and Naju regions in Korea.
Among a total of 8,500 participants within the Health2 cohort, 7,861 subjects were selected for the replication analysis based on their age and information about concomitant disease and medication [13].
Title: Platelet counts and genotyping

Peripheral blood was isolated by using the G-DEX TM IIb DNA Extraction Kit (iNtRON Biotechnology, Seongnam, Korea).
Platelet count was measured using an ADVIA 120 (Bayer, Tarrytown, NY, USA).
Genomic DNA, genotyped on the Affymetrix Genome-Wide Human SNP array 5.0 (Affymetrix, Inc., Santa Clara, CA, USA), was isolated from peripheral blood drawn from Anseong and Ansan cohort participants.
Of 9,603 genotyped samples, we excluded samples with high heterozygosity (&gt;30%, n = 11) and gender inconsistencies (n = 41).
Also, individuals who had developed any kind of cancer (n = 101) were excluded from subsequent analyses.
To examine population stratification overall, related or identical individuals with higher values than first-degree relatives of Korean sib-pair samples were also excluded according to average pair-wise identity-by-state (IBS) values (&gt;0.80, n = 608).
The methods to estimate heterozygosity and IBS have been described elsewhere [13].
SNP markers with a high missing genotype rate (&gt;5%), low minor allele frequency (MAF, &lt;0.01), and significant deviation from Hardy-Weinberg equilibrium (p &lt; 1 × 10-6) were excluded.
Consequently, a total of 352,228 markers were used for the GWAS.
For the replication analysis, 19 SNPs of the Health2 population, comprising 7,861 participants, were genotyped by GoldenGate assay (Illumina Inc., San Diego, CA, USA).
Title: Imputation

Imputation was carried out by using the IMPUTE program version 1 (http://mathgen.stats.ox.ac.uk/impute/impute.html) [14, 15].
On the basis of NCBI, build 36 and dbSNP, build 126, we initially used 90 individuals from Japanese in Tokyo, Japan (JPT) and Han Chinese in Beijing, China (CHB) founders in HapMap as a reference panel, comprising 3.99 million SNPs (release 22).
After removing SNPs with an MAF &lt; 0.01 and an SNP missing rate &gt; 0.05, we combined the remaining 1.8 million imputed SNPs with the directly typed Korea Association Resource (KARE) SNPs for the association analyses.
Association analyses for imputed SNPs were carried out by the SNPTEST program (http://www.stats.ox.ac.uk/~marchini/software/gwas/snptest.html).
Title: Statistical analysis

Association analysis of platelet trait with genotypes was performed using a linear regression model, adjusting for age, sex, and recruitment area.
Analyses were performed with the software PLINK (http://pngu.mgh.harvard.edu/~purcell/plink), SAS version 9.1 (SAS Inc., Cary, NC, USA), and R statistics package version 2.7.1.
The KARE GWAS and replication study were combined by an inverse-variance meta-analysis method, assuming fixed effects, with Cochran&#39;s Q test to assess between-study heterogeneity [16].
Regional plots were generated using LocusZoom [17].
Characteristics of the study participants, including age, sex, and trait summaries, are presented in Table 1.
Quantile-quantile analysis of 2-d.f.
logistic regression statistics for the comparison of genotype frequencies in the cohorts confirmed the genetic homogeneity of these two components of the KARE study population (Fig.1).
We observed significant SNPs by applying a Bonferroni adjustment cut-off with a combined p-value &lt; 1.0 × 10-7.
The GWAS identified several genomic locations as potentially associated with platelet count (Fig.2).
In a follow-up examination of the Health2 cohort, we examined 7,861 selected from 8,500 participants (aged, 40 to 69 years).
We confirmed 3 replicated signals, of which 3 signals, replicating previously documented reports, and one novel locus were discovered.
We observed 3 independent signals with p &lt; 10-5 for platelet counts.
Table 2 and Fig.3 show results for three regions that had combined genomewide significant evidence for association with platelet count in the KARE GWAS and replication samples.
For platelet counts, the strongest evidence for association was at 12q24 (rs739496, MAF = 0.11, combined p = 1.15 × 10-15) (Table 2, Fig.3B).
The associated signal is concordant with a previously published result in the Japanese population [6].
This SNP is located in the 3&#39; untranslated region (UTR) of SH2B adaptor protein 3 (also known as LNK).
SH2B3 is a member of the APS family of adaptor proteins, which play a pivotal role as broad inhibitors of growth factors and cytokine signaling pathways.
The second SNP, rs3733606 (MAF = 0.50, combined p = 1.46 × 10-10) (Table 2, Fig.3A) in 4p16, is in the 3&#39; UTR of the KIAA0232 gene, which translates the functionally unknown hypothetical protein LOC9778.
The third locus associated with platelet is at 6p21 (rs9296095, MAF = 0.23, combined p = 1.67 × 10-7) (Table 2, Fig.3C).
We performed a GWAS of platelet count using 352,225 SNPs profiled with the Affymetrix Genome-Wide human SNP array 5.0 in 8,842 individuals from the Anseong and Ansan cohorts as described previously [13].
In a two-stage design (8,842 discovery and 7,861 replication samples), we confirmed three loci associated with platelet count at a genomewide significance level (&lt; 1.0 × 10-7).
Besides an unknown functional gene, we found two candidate genes, SH2B3 and BAK1, responsible for the variation in platelet counts.
These genes are potential candidates for affecting platelet count.
Lymphocyte adapter protein (SH2B3, also known as LNK) is expressed in hematopoietic precursor cells and in endothelial cells and is known to be involved in inflammation [18].
LNK-deficient mice show a marked difference of hematopoiesis, accompanying increased numbers of various types of cells, such as megakaryocytes, B lymphoid, erythroid progenitor, and hematopoietic stem cells [19].
Several SNPs within the SH2B3 region are well known as variants associated with blood pressure, myocardial infarction, type 1 diabetes, and celiac disease [20].
BAK1, a multidomain pro-apoptotic family member, has shown to be involved in apoptotic cell death, playing a role as an essential mediator [21, 22, 23, 24].
Kamatani et al.
[6] reported that BAK1 is a putative strong candidate gene accounting for numbers of platelets.
This SNP is located in the 4th intron in BAK1 (Bcl2-antagonist/killer1), which encodes a protein acting as a strong proapoptotic effector that is known to control platelet lifespan [25].
The intrinsic machinery for apoptosis regulates the life span of anucleate platelets [25].
KIAA0232 has no known biological function and no clue for a related biological pathway.
Given the weak linkage disequilibrium block around KIAA0232, further investigation is required for the biological function.
Some studies have discovered genetic factors associated with platelet count through genome-wide associated studies across diverse ethnic groups [12, 26].
Qayyum et al.
[12] reported that a candidate gene, BAK1, was significant in African-Americans.
Also, Shameer et al.
[26] reported that SH2B3 and KIAA0232 in European ancestry were also significant in platelet count and mean platelet volume, respectively.
Numerous previous studies have demonstrated the association between platelet counts and various phenotypes in human and mice [27].
The Atherosclerosis Risk in Community (ARIC) study has shown that platelet counts are positively correlated with leukocytes [28].
Turakhia et al.
[29] also reported the association between higher platelet counts and residual thrombus after fibrinolytic therapy, which is in agreement with the ARIC study.
The evidence of a relationship between platelet count and insulin resistance in non-obese type 2 diabetic patients was reported from a study on Japanese [30].
The number of platelets is also a possible predictor of the risk of death and cardiovascular disease [31].
In conclusion, we identified and validated common variants at 1 novel locus, BAK1, and 2 known variants, SH2B and KIAA0232, responsible for the variation of platelet counts in population-based cohorts.
Our research demonstrates the results from a meta-analysis and follow-up genotyping to retrieve positive evidence for the association of 3 loci with platelet counts.
In addition, fine mapping and functional studies on the discovered loci will help us understand the hidden physiological mechanisms underlying platelet count.
Quantile-quantile plot for platelet count.
The observed p-values (y axis) were compared with the expected p-values under the null distribution (x axis) for each trait.
The shaded region represents the 95% concentration band.
Chromosome plot for platelet counts.
Genome-wide association study for log-transformed platelet count on a population-based sample of 8,842 individuals from the Korea Association Resource (KARE) study.
The x axis represents the genomic position (in Gb) of 352,225 single nucleotide polymorphisms; they show -log10(p-value).
Single nucleotide polymorphisms with a p-value &lt; 1 × 10-4 are highlighted in red.
Regional plot of three discovered variants.
(A-C) p-value plots showing the association signals in the region of KIAA0232 on chromosome 4 (A), SH2B3 on chromosome 12 (B), and BAK1 on chromosome 6 (C).
In the top panel, the association signals scaled by -log10(p-value) (typed or imputed SNPs) at each locus are distributed in a genomic region 500 kb to either side of the lead association signal (typed).
Each SNP is plotted as a circle along the chromosomal position, and linkage disequilibrium between the lead SNP and the other SNPs is colored as a scale from low (blue) to high (red) or is colored gray if linkage disequilibrium information was not available in the 1,000 genomes June 2010 CHB+JPT samples.
The lead SNP is colored purple diamond, and the overall meta-analysis result is shown with a purple circle.
The recombination rate estimated from HapMap phase 2 is plotted in blue.
The bottom panel illustrates the locations of known genes.
Genetic information is based on NCBI build 36 and dbSNP build 130.
SNP, single-nucleotide polymorphism; CHB, Han Chinese in Beijing, China; JPT, Japanese in Tokyo, Japan.
Title: Samples used in this study

Title: Variants that associate with variation in platelet counts
Title: The ABCG2 Polymorphism rs2725220 Is Associated with Hyperuricemia in the Korean Population

Elevated serum uric acid levels are associated with a variety of adverse health outcomes, including gout, hypertension, diabetes mellitus, metabolic syndrome, and cardiovascular diseases.
Several genome-wide association studies on uric acid levels have implicated the ATP-binding cassette, subfamily G, member 2 (ABCG2) gene as being possibly causal.
We investigated an association between the single-nucleotide polymorphism (SNP) rs2725220 in the ABCG2 gene and uric acid levels in the Korean population.
A total of 991 subjects in Seoul City were used for a replication study with ABCG2 SNP rs2725220.
The rs2725220 SNP in the ABCG2 gene was associated with mean uric acid levels (effect per allele 0.25 mg/dL, p &lt; 0.0001).
Subjects with the GC/CC genotype had a 1.78-fold (range, 1.22- to 2.62-fold) higher risk of having abnormal uric acid levels (≥7.0 mg/dL) than subjects with the GG genotype.
When analyzed by gender, the association with ABCG2 was stronger in men than in women.
The association with ABCG2 was much stronger in male subjects with body mass index (BMI) ≥ 26.4 (odds ratio, 5.09; 95% confidence interval, 2.41 to 10.8) than in male subjects with BMI &lt; 26.4.
This study clearly demonstrates that genetic variations in ABCG2 influence uric acid levels in Korean adults.
Elevated serum uric acid levels are associated with a variety of adverse health outcomes, including gout, hypertension, diabetes mellitus, metabolic syndrome, and cardiovascular diseases [1].
Several genome-wide association studies (GWASs) have identified genes that may be causally associated with uric acid levels, including the ATP-binding cassette, subfamily G, member 2 (ABCG2) gene (MIM 603756) [2, 3, 4, 5, 6, 7].
Replication studies of ABCG2 variants have been performed in Western and Asian countries [8, 9].
Okada et al.
[4] identified a significant association between the rs2725220 single-nucleotide polymorphism (SNP) on chromosome 4q22 and uric acid levels.
The ABCG2 protein is a high-capacity transporter for uric acid excretion in the kidney, liver, and gut [10, 11].
In this study, we analyzed the relationship between uric acid levels and the ABCG2 SNP rs2725220 using a group of volunteers from the Korean Metabolic Syndrome Research Initiative Study in Seoul.
We also analyzed the association of the ABCG2 SNP with body mass index and waist circumference levels.
Title: Study population

Subjects for the GWAS were recruited from the Korean Metabolic Syndrome Research Initiative Study in Seoul City, which was initiated in December 2005.
A total of 9,128 individuals were recruited in 2006, and an additional 17,569 individuals were recruited in 2007 [12, 13, 14].
Therefore, the total Seoul City cohort included 26,697 volunteers.
Volunteers from the first round underwent routine health examinations at the health promotion centers in university hospitals between January 2006 and December 2007.
From this total, 6,563 individuals were randomly selected for measurement of adiponectin levels.
The subject characteristics were described in a previous study [14].
In brief, of the 6,563 individuals whose adiponectin was measured, 1,004 individuals were genotyped.
The 1,004 subjects were all healthy individuals and were not undergoing any treatment for hyperuricemia.
The Institutional Review Board of Human Research of Yonsei University approved the protocols of this study, and written informed consent was obtained from all subjects prior to enrollment.
Title: Data collection

Participants were interviewed using a structured questionnaire to collect a personal history of cigarette smoking (never smoked, ex-smoker, or current smoker) and demographic characteristics (age, gender, etc.).
Waist circumference was measured midway between the lower rib and iliac crest.
For measurements of weight and height, light clothing was worn.
Body mass indices were calculated as the subject&#39;s weight (kg) divided by the square of the subject&#39;s height (m2).
For clinical chemistry assays, serum was separated from peripheral venous blood samples obtained from each participant after a 12-hour fast and stored at -70℃.
Biomarkers of metabolic syndrome, including fasting blood glucose, total cholesterol, triglycerides, and high-density lipoprotein cholesterol, and uric acid levels were measured.
Quality control of the data was conducted in accordance with the procedures recommended by the Korean Association of Laboratory Quality Control.
Title: Genotyping assays

Samples from the Seoul City cohort were genotyped on the Affymetrix Genome-Wide Human SNP Array 5.0 (Affymetrix Inc., Santa Clara, CA, USA) at DNALink.
For the data obtained from this chip, internal quality control (QC) measures were used: the QC call rate (Dynamic Model algorithm) always exceeded 86%.
Genotype calling was carried out using the Birdseed (v2) algorithm.
A total of 1,004 individuals were genotyped using this platform.
Ten of the 1,004 individuals were removed due to low genotyping call rates (&lt;95%).
Three individuals were excluded due to failed genotyping of rs2725220, leaving a total of 991 individuals for the study.
Title: Statistical analysis

Data were expressed as means ± standard deviation (SD).
Most statistical analyses were performed using PLINK and SAS (version 9.2; SAS Institute, Cary, NC, USA).
The ABCG2 gene SNP rs2725220 was selected using the Affymetrix Genomewide Human SNP Array 5.0.
Each SNP was tested for possible effects on uric acid levels under an additive model.
The multivariate linear regression models used in the study incorporated covariates (age and sex).
Multiple logistic regression analysis was also performed.
Odds ratios (ORs) with 95% confidence intervals (CIs) were calculated to examine the association of the ABCG2 SNP with abnormal uric acid levels (≥7 mg/dL for men and ≥6 mg/dL for women).
All statistical tests were two-sided, and the statistical significance was determined as p &lt; 0.05.
The majority of individuals in this study was middle-aged (Table 1).
This sample of Korean volunteers had low uric acid levels on average.
Mean levels of uric acid in the dataset were higher in males (6.23 mg/dL) than in females (4.21 mg/dL).
The percentage of individuals with abnormal uric acid levels (≥7.0 mg/dL) was 14.7% for the Seoul City sample.
The percentage of current smokers in the Seoul City dataset was 46.4% among male subjects and 3.9% among female subjects.
Table 2 shows the p-values from a linear regression model for uric acid levels in the cohort sample, with age and sex included as covariates in the model.
The rs2725220 SNP in the ABCG2 gene was found to be associated with mean uric acid levels (effect per allele 0.25 mg/dL, p &lt; 0.0001), which was significant after Bonferroni correction (p = 0.00025).
The minor allele frequency of rs2725220 was 0.222, and Hardy-Weinberg equilibrium was satisfied (p = 0.7079).
We also analyzed the association of rs2725220 with abnormal uric acid levels (Table 3).
Subjects with the GC/CC genotype had a 1.78-fold (range, 1.22- to 2.62-fold) higher risk of having abnormal uric acid levels (≥7.0 mg/dL) than subjects with the GG genotype.
When analyzed by gender, the association with ABCG2 was stronger in men than in women.
In Table 4, we analyzed the association of ABCG2 by body mass index (BMI) quartile levels and waist circumference quartile levels in male subjects.
The association with ABCG2 was much stronger in male subjects with BMI ≥ 26.4 (OR, 5.09; 95% CI, 2.41 to 10.8) than in male subjects with BMI &lt; 26.4.
The association with ABCG2 was also a little bit stronger in male subjects with waist circumference (WC) ≥ 91 (OR, 2.10; 95% CI, 1.08 to 4.10) than in male subjects with WC &lt; 91.
In a cohort study of 991 subjects, an ABCG2 gene SNP was associated with increased uric acid, consistent with previous studies.
Several GWASs have reported that ABCG2 exerts a major influence on uric acid levels [2, 3, 6, 7].
Dehghan et al.
[6] conducted a genomewide study in which an SNP in the ABCG2 gene, rs2231142, displayed strong evidence of an association with uric acid levels (p &lt; 10-60).
Another GWAS also reported that rs2231142 showed a strong association with uric acid levels (p = 5.1 × 10-6) [2].
More recently, a GWAS in European-American obesity cases and controls reported significant genome-wide association of two SNPs with uric acid levels: rs2622605 and rs1481017 [3].
Another recent GWAS also reported a strong association between rs2231142 and uric acid levels (p = 3.34.1 × 10-42) [7].
In the present study, rs2725220 was found to have a strong association with uric acid levels.
In agreement with these findings, a recent study reported that rs2725220 was strongly associated with uric acid levels (effect per allele -0.135 mg/dL, p = 4.2 × 10-30) [4].
Several recent studies reported that the association between serum uric acid and the allelic effects of SNPs in ABCG2 was sex-specific [8, 9].
In a recent study of 4 US populations, the association between rs2231142 and serum uric acid was significantly stronger in men and postmenopausal women compared to their premenopausal counterparts [9].
In the present study, the association of the ABCG2 gene SNP with uric acid was stronger in men than in women.
Associations between genotypes and phenotypes can be modified by gene-environment interactions.
A modification of the effect of the associations between ABCG2 SNPs and serum uric acid concentrations by interactions with obesity has been reported [3, 9].
In a recent study of 4 US populations, the association between rs2231142 and serum uric acid was stronger in normal-weight subjects than in obese subjects [9].
In the present study, we also examined the association of the ABCG2 SNP with uric acid levels according to BMI level and waist circumference.
The association was stronger in subjects with a high BMI and high WC levels than in subjects with a low BMI and low WC levels.
The ABCG2 gene encodes a membrane transporter belonging to the ATP-binding cassette superfamily of membrane transporters, a group involved in the trafficking of biological molecules across cell membranes.
The ABCG2 gene contains 16 exons and spans over 66 kb [15].
Allikmets et al.
[16] mapped the ABCG2 gene to human chromosome 4q22 using radiation hybrid analysis.
Candidate gene analysis suffers from population substructure.
We estimated the identity by state over all SNPs, and only four individuals were shown to be biological relatives.
Genetic studies in Asian populations may not identify the same set of genes as those in European populations [17].
Nevertheless, the Seoul City cohort study in Korean populations indicates that the ABCG2 gene on chromosome 4 is associated with serum uric acid levels, as is the case in several other populations.
Title: General characteristics of the study population

Title: Association of ABCG2 gene SNP rs2725220 with uric acid levels and lipid levels based on a linear regression model

Title: Odds ratio (OR) of polymorphic genotypes of ABCG2 gene SNP rs2725220 on uric acid levelsa in Seoul City data (n = 991)

Title: Odds ratio (OR) of polymorphic genotypes of ABCG2 gene SNP rs2725220 on uric acid levelsa in Korean men (n = 558)
Title: Replication of Interactions between Genome-Wide Genetic Variants and Body Mass Index in Fasting Glucose and Insulin Levels

The genetic regulation of glucose and insulin levels might be modified by adiposity.
With regard to the genetic factors that are altered by adiposity, a large meta-analysis on the interactions between genetic variants and body mass index with regard to fasting glucose and insulin levels was reported by the Meta-Analyses of Glucose- and Insulin-related trait Consortium (MAGIC), based on European ancestry.
Because no replication study has been performed in other ethnic groups, we first examined the link between reported single-nucleotide polymorphisms (SNPs) and fasting glucose and insulin levels in a large Korean cohort (Korean Genome and Epidemiology Study cohort [KoGES], n = 5,814).
The MAGIC study reported 7 novel SNPs for fasting glucose levels and 6 novel SNPs for fasting insulin levels.
In this study, we attempted to replicate the association of 5 SNPs with fasting glucose levels and 5 SNPs with fasting insulin levels.
One SNP (rs2293941) in PDX1 was identified as a significant obesity-modifiable factor in Koreans.
Our results indicate that the novel loci that were identified by MAGIC are poorly replicated in other ethnic groups, although we do not know why.
The rapid socioeconomic developments in developing countries have led to a shift in infrastructure, technology, and food supply, promoting overnutrition and sedentary lifestyles [1].
Consequently, the prevalence of metabolic diseases, such as obesity and type 2 diabetes, has risen dramatically [2, 3].
In particular, Asian countries have experienced a notable increase in metabolic disease; thus, prevention and control of metabolic diseases should be the top public health priorities in Asian populations [3].
Glucose and insulin levels after overnight fasting are important indicators of glucose tolerance, insulin resistance, and type 2 diabetes [4].
Adiposity is believed modulate the genetic determinants of fasting glucose and insulin levels [5].
To identify genetic determinants that could be modified by body mass index (BMI), the interactions between genome-wide single-nucleotide polymorphisms (SNPs) and BMI with regard to fasting glucose and insulin levels were analyzed by the Meta-Analyses of Glucose- and Insulin-related traits Consortium (MAGIC) [6].
MAGIC identified several novel genetic variants: 7 SNPs (ARAP1, FOXA2, DPYSL5, PCSK1, PDX1, PPP1R3B, and OR4S1) for fasting glucose level and 6 SNPs (COBLL1-GRB14, IRS1, PPP1R3B, PDGFC, UHRF1BP1, and LYPLAL1) for fasting insulin level [6].
The MAGIC study comprised approximately 70,000 nondiabetic individuals of European ancestry, and its results should be confirmed in other ethnic groups [7].
Because no replication study has been performed in other ethnicities, we examined these SNPs and their association with fasting glucose and insulin levels and determined whether they are modified by obesity in the Korean population.
The study design can be found in Fig.1.
Title: Study subjects and genotypes

The subjects and their genotypes in the original study have been reported [7].
The initial subjects were obtained from a subset of the Korean Genome and Epidemiology Study cohort (KoGES), known as Korean Association Resource (KARE, n = 8,842) [7].
Written informed consent was obtained from all participants, and this project was approved by the institutional review board at Korea National Institute of Health (KNIH).
Title: Phenotypes and quantitative traits

Past diabetes history was obtained through structured questionnaires by interviewers.
Anthropometric measurements, including weight and height, were also obtained, and BMI was calculated as weight (kg)/height (m2) [2].
Biochemical assessments were performed, including fasting serum glucose, 2-h 75 g oral glucose tolerance test, and fasting serum insulin (FSI).
A total of 3,028 subjects with prevalent hyperglycemia (fasting glucose ≥ 110, 2-h glucose level after 75 g oral glucose challenge ≥ 140), or self-reported diabetes history were excluded.
Of the 8,842 individuals who were examined in the previous genomewide association study [7], 5,814 nondiabetic individuals were selected.
Title: SNP genotypes

The genotypes were obtained using the Affymetrix Genome-Wide Human SNP array 5.0 (Affymetrix Inc., Santa Clara, CA, USA).
The genotype quality control criteria have been reported in a previous genome-wide association study [7].
Briefly, every 500 ng of genomic DNA was genotyped on the Affy 5.0, and the Bayesian robust linear genotyping algorithm using Mahalanobis distance was used for genotype calling [8].
Samples with a high missing genotype call rate (≥4%), high heterozygosity (&gt;30%), inconsistency in sex, and any tumor were excluded from subsequent analyses.
Title: Genotype imputation

Of the 13 SNPs that were reported by the MAGIC consortium, 4 SNPs were genotyped in the original KARE genotype data using the Affy 5.0.
To maximize the validation SNPs in Koreans, the Chinese-Japanese HapMap dataset was imputed to obtain 6 additional SNPs.
Thus, 10 SNPs were analyzed in this study.
The detailed imputation procedure has been published.
Briefly, the KARE dataset, comprising 351,677 SNPs in 8,842 individuals, was merged with that of the international HapMap Phase II Japanese (JPT) + Chinese (HCB) panel 2 [9].
The genotypes of the KARE individuals were imputed using IMPUTE [10].
Using the following filtering scheme-INFO ≥ 0.5 and posterior probability ≥ 0.9-1,827,004 SNPs were imputed, and 6 SNPs that matched among the 13 SNPs that were reported by MAGIC were selected for study.
Title: Statistical analysis

Glucose and insulin levels were natural log-transformed.
To examine the effects of SNP-BMI interactions on them, we conducted 4 statistical tests for each SNP.
In all analyses, residential area, age, sex, and BMI were the covariates.
The first 2 tests were linear regression analyses of lean (BMI &lt; 25) and overweight subjects (BMI ≥ 25).
The overall interactions between SNPs and BMI were analyzed using continuous BMI values and dichotomized BMI (lean subjects were coded with 1, and overweight subjects were coded with 2).
Continuous BMI-SNP interaction analyses were performed using QUICKTEST (http://toby.freeshell.org/software/quicktest.shtml), and the dichotomized BMI-SNP interactions were analyzed using PLINK, version 1.07 [11].
The fasting serum glucose and insulin levels of our subjects are shown in Table 1.
We divided subjects into 4 groups, based on BMI; most Korean subjects lay in the 20 ≤ BMI &lt; 25 and 25 ≤ BMI &lt; 30 categories.
The MAGIC study had BMI cutoffs of &lt; 25 and ≥ 30, and 237 of our subjects (4%) had BMI ≥ 30.
Asians have lower rates of overweightedness and obesity than Western populations, per conventional definitions (BMI ≥ 25 for overweightedness and ≥ 30 for obesity) [3].
Despite having lower BMIs, Asian countries have a similar or greater prevalence of diabetes versus Western countries [12].
Thus, we used BMI &lt; 25 and BMI ≥ 25 to analyze the effects of SNPs and SNP-dichotomized BMI interactions by stratum.
Of the 6 SNPs that were associated with FSI levels in the MAGIC studies, 5 SNPs could be tested in the current study, and no SNP was replicated in all 3 statistical tests (stratified BMI, continuous BMI interaction p-values, and dichotomized BMI interaction p-values) (see Table 2).
Of the 7 reported SNPs that were linked to fasting plasma glucose levels, 5 SNPs could be tested in the current study, and 1 SNP (rs2293941) in pancreatic and duodenal homeobox 1 (PDX1) was significantly associated in lean subjects (beta ± SE, 0.007 ± 0.002; p = 0.001) but not in overweight subjects.
The SNP-continuous BMI interaction was not significant, but the SNP-dichotomized BMI interaction was marginal (p = 0.04).
Our results indicate that the novel loci that were identified by MAGIC are poorly replicated in other ethnic groups, although we do not know why.
We have identified an important genetic factor of glucose metabolism, PDX1, which regulates pancreatic beta-cell function and survival by governing expression of the insulin-encoding INS gene [13].
This study was limited by its exclusion of 3 other SNP-BMI interactions from MAGIC.
These SNPs could not be extracted from the imputation data using the original Affy 5.0 in the KARE study.
Another limitation of this study was the relatively weak significance of its association results compared with those of MAGIC.
We do not know whether this lack of significance is due to the relatively smaller sample size of this study compared with MAGIC, the inaccuracy of imputation genotypes, or ethnic differences in the SNP-BMI interactions.
These issues should be addressed in other Asian populations.
Although the replication of gene-environment interactions in other ethnic groups is difficult, interethnic conservation of the interaction results might help us determine the underlying mechanisms of the target phenotypes.
Further, the rs2293941 SNP might help us understand individual differences in beta-cell function.
Schematic of the study design.
SNP, single-nucleotide polymorphism; BMI, body mass index; MAGIC, Meta-Analysis of Glucose and Insulin Consortium; KARE, Korean Association Resource consortium.
Title: Clinical characteristics

Title: Replication results of MAGIC-reported SNPs for fasting insulin levels and fasting glucose levels
Title: Identification and Functional Characterization of P159L Mutation in HNF1B in a Family with Maturity-Onset Diabetes of the Young 5 (MODY5)

Mutation in HNF1B, the hepatocyte nuclear factor-1β (HNF-1β) gene, results in maturity-onset diabetes of the young (MODY) 5, which is characterized by gradual impairment of insulin secretion.
However, the functional role of HNF-1β in insulin secretion and glucose metabolism is not fully understood.
We identified a family with early-onset diabetes that fulfilled the criteria of MODY.
Sanger sequencing revealed that a heterozygous P159L (CCT to CTT in codon 159 in the DNA-binding domain) mutation in HNF1B was segregated according to the affected status.
To investigate the functional consequences of this HNF1B mutation, we generated a P159L HNF1B construct.
The wild-type and mutant HNF1B constructs were transfected into COS-7 cells in the presence of the promoter sequence of human glucose transporter type 2 (GLUT2).
The luciferase reporter assay revealed that P159L HNF1B had decreased transcriptional activity compared to wild-type (p &lt; 0.05).
Electrophoretic mobility shift assay showed reduced DNA binding activity of P159L HNF1B.
In the MIN6 pancreatic β-cell line, overexpression of the P159L mutant was significantly associated with decreased mRNA levels of GLUT2 compared to wild-type (p &lt; 0.05).
However, INS expression was not different between the wild-type and mutant HNF1B constructs.
These findings suggests that the impaired insulin secretion in this family with the P159L HNF1B mutation may be related to altered GLUT2 expression in β-cells rather than decreased insulin gene expression.
In conclusion, we have identified a Korean family with an HNF1B mutation and characterized its effect on the pathogenesis of diabetes.
Maturity-onset diabetes of the young (MODY) is a monogenic form of diabetes characterized by an early onset, usually before the age of 25 years; an autosomal dominant inheritance; and a defect in pancreatic β-cell function [1].
Depending on the specific gene mutation, the synthesis and secretion of insulin are altered at different stages.
MODY is classified according to its affected genes, which include those encoding enzymes, transcription factors, and other types of proteins [1, 2].
The transcription factors associated with MODY act in the nucleus of β-cells and regulate the transcription of INS or other genes encoding enzymes involved in glucose transport and metabolism [1].
Hepatocyte nuclear factor-1β (HNF-1β) is a homeodomain-containing transcription factor that forms a homodimer or heterodimer with structurally related HNF-1α [3].
HNF1B has 9 exons and encodes a 557-amino-acid peptide.
Its structure is characterized by a highly conserved DNA-binding domain composed of an atypical POU-specific (POUS) and POU-homeo (POUH) domain, but the molecular properties of HNF-1β have not been studied much.
HNF-1 β is known for playing a role in tissue-specific gene expression in organs, including the liver, kidney, and pancreatic islets [4], and is involved in the β-cell transcription factor network [5].
Heterozygous mutations of HNF1B, which encodes HNF-1β, result in maturity-onset diabetes of the young 5 (MODY5), which is characterized by early-onset diabetes and various abnormalities, such as renal cysts, renal impairment, genital malformation, or hepatobiliary involvement [6, 7, 8, 9].
Patients with mutations in HNF1B have impaired insulin secretory responses to glucose and insulin secretagogues [7, 10, 11] and show progressive loss in basal insulin secretion.
Mutation in HNF1B was first described by Horikawa et al.in 1997 [12].
Different mutation types, including missense, nonsense, and frameshift mutations, have been found in different domains [13, 14, 15].
Recently, mutations in exon 2 and in the DNA-binding domain have been reported [16, 17, 18, 19, 20].
Barbacci et al.
[21] characterized eight naturally occurring mutations in different domains.
Truncated mutations showed defective nuclear localization and weak dominant-negative activity, whereas a frameshift mutation within the QSP-rich domain had partially reduced transcriptional activity.
Missense mutations in POUS and POUH exhibited severe decreases in transcription.
A certain mutation showed a gain-of-function phenotype [22, 23].
In vitro studies suggested that clinical phenotypes may be related to loss of function and/or dominant-negative mechanisms [8, 24].
In this study, we have identified a family with MODY5 harboring a heterozygous P159L HNF1B mutation.
We evaluated the functional consequences of this HNF1B mutation on glucose metabolism.
Title: Sequencing of HNF1B

The HNF1B of the patient was sequenced by Sanger method in peripheral blood DNA.
Her father, mother, and younger brother were also screened by the same method.
Written informed consent for the genetic study was obtained from the patient and her family members before sequencing.
Title: Wild-type and mutant plasmid constructs

Human wild-type HNF1B, cloned in pCMV6b, was provided by Dr. J. Takeda, Gifu University, Japan.
A point mutation was generated by PCR-based site-directed mutagenesis using the QuickChange Mutagenesis Kit (Stratagene, La Jolla, CA, USA).
The promoter of human glucose transporter type 2 (GLUT2) (-1296 to +312), cloned in the pGL3-Basic vector (Promega, Madison, WI, USA), was provided by Dr. M.S.
Lee, Sungkyunkwan University, Korea.
The sequences of all genes for the experiment were confirmed by direct nucleotide sequencing.
Title: Cell culture

COS-7 cells were maintained in Dulbecco&#39;s modified Eagle&#39;s medium (DMEM) with 10% fetal bovine serum (Invitrogen, Carlsbad, CA, USA), supplemented with 100 U/mL of penicillin and 100 µg/mL of streptomycin, before transient transfection.
MIN6 cells were cultured in RPMI 1640 with 11.1 mM D-glucose supplemented with 10% fetal bovine serum (Invitrogen) and the same antibiotics as for COS-7.
Cells were cultured at 37℃ in 5% CO2:95% air.
Title: Transient transfection

COS-7 cells were subcultured in 12-well plates the day before transfection; 0.2 µg of pGL3-GLUT2 promoter, 0.02 µg of HNF1B expression plasmids, and 0.05 µg of pCMV-β-galactosidase were combined with LipofectAMINE PLUS agent (Invitrogen) according to the manufacturer&#39;s protocol.
The cells, in 400 µL of serum-free DMEM, were treated with the complex for 3 h, and the medium was changed to DMEM with 10% fetal bovine serum.
MIN6 cells were subcultured in 6-well plates the day before transfection.
As it is difficult to transfect genes into MIN6 cells, the transfection efficiency was evaluated with the green fluorescent protein gene (GFP) first.
After the efficiency was confirmed, 0.5 µg of expression vector for HNF1B per well was administered.
Serum-containing medium was added up to the normal volume after 3 h of incubation and replaced with fresh complete RPMI 1640 after 8 h. MIN6 cells were harvested for RNA extraction and protein quantitation after 30 h after the start of transfection.
Title: Luciferase reporter assays

Twenty-four hours after the start of transfection, Cos7 cells were lysed with reporter lysis buffer and harvested, and the transcriptional activity was measured using the Luciferase assay system (Promega) and Lumet LB9507 (Berthold, Bad Wildbad, Germany) according to the manufacturer&#39;s instructions.
To normalize transfection efficiency, β-galactosidase activity was measured.
Ten microliters of cell lysate was mixed with 3 µL of 100 × MgCl2 (0.1 M MgCl2 and 4.5 M β-mercaptoethanol) and 66 µL 1 × o-nitrophenyl-β-D-galactoside.
The activity of β-galactosidase was measured by VersaMax spectrophotometry at 420 nm (Molecular devices, Sunnyvale, CA, USA).
Title: Electrophoretic mobility shift assay

Double-stranded oligonucleotides composed of the following sequences were used for human GLUT2 promoter probes: 5&#39;-AAG ACC TCA GTA AAG ATT AAC CAT CAT TA-3&#39;; 1 µg of up and down sequences of single-stranded oligomer were hybridized within PE buffer (20 mM Tris, 10 mM MgCl2, 50 nM NaCl, 1 mM dithiothreitol [DTT]) by incubating them at 72℃ for 10 min and then at room temperature for 15 min.
The probe was labeled with [α-32P] dATP by Klenow DNA polymerase (Ambion, Austin, TX, USA), and the labeled probe (25,000 cpm) was incubated with nuclear extracts of COS-7 cells that were transfected with pcDNA, wild-type HNF1B, and H153N and P159L HNF1B in 10 mM HEPES (pH 7.9, Sigma-Aldrich, St. Louis, MO, USA) containing 50 mM KCl, 0.1 mM EDTA, 0.25 mM DTT, 0.1 mg/mL poly(dIdC), 0.01% Nonidet P-40, and 10% glycerol at room temperature.
DNA-protein complexes were separated on 6% polyacrylamide gels and detected by autoradiography.
Title: RNA isolation and real-time PCR

Mutant or wild-type HNF1B was overexpressed in the mouse pancreatic β-cell line MIN6.
Total RNA was isolated using TRIzol (Invitrogen) according to the manufacturer&#39;s instructions.
To prepare cDNA, 6 µL of reaction buffer (Invitrogen), 3 µL of 100 mM DTT, 1.5 µL of 10 mM dNTP, 0.6 µL of random primer, 0.3 µL of RNase inhibitor, 1.0 µL of RTase, and 1 µg of RNA were mixed, and RNase-free water was added up to 30 µL.
The mixture was incubated 37℃ for 1 h and 72℃ for 10 min using a PCR system.
Expression levels of genes were determined by using SYBR Master Mix (Takara, Otsu, Japan) and an AB 7500 Real-time PCR system (Applied Biosystems, Foster City, CA, USA).
The primer sequences for the PCR were as follows: 5&#39;-GGC TAA TTT CAG GAC TGG TT-3&#39; and 5&#39;-TTT CTT TGC CCT GAC TTC CT-3&#39; for GLUT2 and 5&#39;-CTC GGC TCA CCG TTT CCT T-3&#39; and 5&#39;-CGC GCT TGC TAA TAG TGC AG-3&#39; for INS.
Samples were prepared repetitively, and each sample was analyzed in duplicate.
Expression levels of each gene were measured relative to 18S and normalized to the expression levels in total RNA with the ΔΔCT method.
Title: Western blot analysis

Cells were lysed in a buffer containing 20 mM Tris-HCl, pH 7.4, 5 mM Na4P2O7, 100 nM NaF, 2 mM Na3VO4, 1% NP-40, and protease inhibitor (1 µg/µL of aprotinin, 1 µg/µL of leupeptin, and 1 mM of PMSF; Sigma-Aldrich).
After determination of protein concentrations, equal amounts of cell lysates were separated by sodium dodecyl sulfate polyacrylamide gel electrophoresis.
Separated proteins were transferred onto a nitrocellulose membrane (Whatman, Dassel, Germany).
The membrane was blocked with 5% skim milk in Tris-buffered saline-Tween-20 (TBS-T) for 1 h at room temperature and incubated with rabbit polyclonal anti-HNF-1β antibody (Santa Cruz Biotech, San Diego, CA, USA) at 4℃.
After 8 h, the membranes were probed with secondary antibody, and bands were visualized by enhanced chemi-luminescence (Pierce, Rockford, IL, USA).
Title: Statistical analysis

Data were expressed as mean ± standard error (SE).
The non-parametric t-test and one-way ANOVA were used to compare the mean values between the groups with GraphPad Prism version 5.0 (GraphPad Software, Inc., La Jolla, CA, USA).
p &lt; 0.05 was considered statistically significant.
All individual experiments were performed at least in duplicate and were repeated five times on separate occasions.
A 27-year-old woman with inadequately controlled diabetes mellitus was admitted for glycemic control.
She was diagnosed as having diabetes mellitus at an age of 14 (fasting plasma glucose 137 mg/dL, postprandial 2-h glucose 239 mg/dL), with normal body mass index (21.9 kg/m2) and 24-h urine C-peptide level (83.3 µg/day; reference range, 50 to 100µg/day).
Although autoantibodies for pancreatic β-cell were not detected, she required insulin therapy for glycemic control.
Her serum C-peptide level decreased from 1.8 ng/mL to 0.9 ng/mL during the past year.
The total daily insulin requirement was 0.68 IU/kg.
The patient&#39;s father (diagnosed in his early 30s) and grandmother (the age of diagnosis was unknown) also had diabetes.
She did not have urogenital anomalies.
Sequencing of HNF1B revealed a heterozygous P159L point mutation (CCT to CTT in codon 159) in the patient.
Among the three other family members, her father was found to have the identical mutation.
The pedigree of diabetes mellitus and the result of Sanger sequencing are shown in Fig.1.
This nonsynonymous variant co-segregated according to the affected state of diabetes within the family.
Two point mutations, P159L and H153N (for comparison as functionally proven mutations; i.e., positive control), were generated by in vitro mutagenesis (Fig.2).
Western blot analysis confirmed that the mutant protein was apparently expressed in the transfected COS-7 cells (Fig.3A).
The region, including nucleotide -1030 of human GLUT2 promoter, which is known to be the cis-element for HNF-1β binding [24], was inserted into the luciferase vector (Fig.3B).
Luciferase activity of the human GLUT2 promoter was increased with overexpression of wild-type HNF-1β compared to the control (p &lt; 0.05 vs. control) (Fig.3C).
When P159L HNF-1β and H153N HNF-1β were expressed, a significant reduction of transcription activity was observed (p &lt; 0.05 vs. wild-type, respectively).
In the electrophoretic mobility shift assay (EMSA), the proteins of the expected sizes were translated (Fig.4A).
The EMSA showed that wild-type HNF-1β protein bound well to the GLUT2 promoter, but the mutant proteins had decreased binding activity (Fig.4B).
The mRNA levels of GLUT2 and INS in MIN6 cells after HNF1B transfection were also investigated.
The transfection into MIN6 was confirmed with an efficiency of 60% to 65% (Fig.5A).
Western blot analysis showed that the protein expression was well preserved after transfection (Fig.5B).
The quantitated mRNA levels were normalized to that of 18S.
When the wild-type HNF-1β was overexpressed, the relative mRNA level of GLUT2 was increased by 2.3-fold (p &lt; 0.05 vs. control) (Fig.5C).
Compared to the wild-type HNF-1β, the relative mRNA levels of GLUT2 for the H153N and P159L mutants were significantly decreased (p &lt; 0.05 vs. wild-type, respectively).
There was no significant difference in the mRNA expression level of INS among the four groups (Fig.5D).
In this study, we identified a MODY5 family with a nonsynonymous mutation in the HNF1B gene.
The patient was suspected to have MODY, especially MODY5, because of the lack of response to sulfonylureas, early (within 3 years) requirement of exogenous insulin, and features of insulin resistance [25].
Although the patient did not have specific clinical features of HNF1B mutation, such as cystic kidney disease [6], she had fulfilled the clinical diagnostic criteria of MODY.
The clinical features of MODY5 vary as either diabetes or urinary tract malformation, or both could be the presenting symptoms [14, 20, 26].
According to a systematic review performed by Chen et al.
[27], renal structural anomalies are relatively less frequent in missense mutations than in other types of mutation.
On the contrary, the percentage of diabetes mellitus and the frequency of insulin treatment tended to be higher.
The mutation of P159L was heterozygous c.476C&gt;T in exon 2 of HNF1B, which is in the DNA-binding domain.
Chen et al.
[27] found that mutations in HNF1B were more likely to be located in the DNA-binding domain, and the hotspots were in exons 2 and 4.
Yamagata [28] reviewed that N228K is a relatively common polymorphism in Pima Indians, and the A241T, G492S, and S465R mutations might be rare polymorphisms.
In HNF1B-deficient mouse embryos, a dorsal bud of the pancreas is formed transiently and becomes absent, whereas the ventral bud is not affected [29].
In two 8-week human fetuses with HNF1B mutations, severe pancreas hypoplasia was observed [30].
With these consistent findings, the abnormality in the morphogenesis of the pancreas is thought to be one of the causes of diabetes in MODY5 patients.
Another suggested molecular mechanism is related to altered GLUT2 expression.
Wild-type HNF-1β is known to bind to the cis-element of GLUT2 and efficiently increase its transcription [24].
In this study, overexpression of wild-type HNF-1β resulted in an increase in transcriptional activity of the GLUT2 promoter, whereas overexpression of the P159L mutant resulted in decreased GLUT2 expression.
Previous studies showed similar results in various types of cells, such as COS-7, MIN6, and HepG2 [20, 24].
GLUT2 is a low-affinity, high-capacity transporter expressed mainly in hepatocytes, renal proximal tubule cells, and pancreatic β-cells [31, 32].
GLUT2 plays a role in β-cell insulin secretion, as it senses external glucose and transports it into pancreatic β-cells [33].
In vivo studies showed that GLUT2-deficient mice were hyperglycemic, resulting in diabetes with impaired glucose sensing by pancreatic β-cells [34], and GLUT2 was undetectable on β-cells of the two HNF1B mutant fetuses [29].
So, decreased expression of GLUT2 in β-cells, combined with a gradual reduction in insulin secretion, could be involved in the pathogenesis of diabetes mellitus in MODY5 patients.
In addition, GLUT2 may also affect β-cell differentiation in the development of the pancreas [34, 35].
EMSA was done to examine whether the reduced transcription activity was due to the decreased DNA-binding capacity.
Both mutant HNF-1β proteins had decreased DNA-binding activity compared to the wild-type.
It was previously reported that the H153N HNF-1β mutation had diminished DNA-binding activity [20].
Since both P159L and H153N mutants have intact dimerization domains, it is possible that they could dimerize with wild-type HNF-1β [9].
However, the mutant/wild-type dimer seems to have a defect in DNA binding and limited ability to increase the transcription of GLUT2.
The mRNA expression of GLUT2 was increased with wild-type HNF-1β but significantly suppressed, almost to the level observed in the empty control vector, in both mutants.
The findings of the real-time PCR analysis of GLUT2 were consistent with the reporter gene assay and DNA binding assay.
On the other hand, there has been limited knowledge regarding the mRNA expression of INS with mutant HNF-1β.
The mRNA levels of INS were not statistically different between the wild-type- and mutant-expressing MIN6 cells.
In one study with HNF-1α, neither the wild-type nor the mutant activated GLUT2 or the insulin promoter in MIN6 cells [36].
The authors concluded that endogenous HNF-1α, GLUT2, and insulin proteins in MIN6 cells could interfere in the transfected reporter assay.
The mRNA expression of INS is also considered to be regulated by other transcription factors, and it might have been compensated for by complex mechanisms against the mutation of HNF1B.
Overall, the impaired insulin secretion in MODY5 patients might be explained by pancreatic hypoplasia or β-cell dysgenesis rather than decreased mRNA expression of INS.
As described above, decreased mRNA expression of GLUT2 might also result in the inhibition of insulin secretion.
This study has some limitations.
First, we could not sequence HNF1B of the patient&#39;s grandmother, who was also a patient with diabetes.
If she had the same P159L mutation of HNF1B, it would have revealed an autosomal dominant inheritance more clearly.
Second, alterations in insulin secretion with the mutation in the pancreatic β-cell line were not evaluated.
Further studies of functional changes in the P159L mutation are needed.
In conclusion, we have identified a P159L mutation of HNF1B in a family with MODY5 and have characterized the functional consequences of this nonsynonymous mutation in the pathogenesis of diabetes.
This mutation revealed decreased DNA-binding activity and decreased transcriptional activity of GLUT2.
The mRNA expression of GLUT2 in pancreatic β-cell lines was also significantly decreased with this mutation, whereas that of INS was not altered.
Impaired insulin secretion in the patient with the P159L mutation in HNF1B might be associated with the alteration of GLUT2-related signaling pathways.
The pedigree of diabetes mellitus in the case family and the result of HNF1B sequencing.
(A) The patient&#39;s father and grandmother were affected by diabetes mellitus.
(B) Sanger sequencing of HNF1B was done with blood samples of the patient and her family members (parents and a younger brother), which revealed the same heterozygous P159L (CCT to CTT) mutation in the patient and her affected father.
The sites of mutation and the oligonucleotides used for in vitro mutagenesis of HNF1B.
(A) Upper bar shows the structure of wild-type human HNF-1β.
It consists of a dimerization domain (a.a. 1-32), DNA-binding domain (a.a. 106-310), and transactivation domain (a.a. 310-557).
The P159L (c.476C&gt;T) mutation and the H153N mutation (c.457C&gt;A) were generated as lower bars.
The H153N mutant was a functionally proven mutation.
(B) The oligonucleotides used for mutagenesis are shown.
WT, wild-type; POUS, POU-specific; POUH, POU-homeo; HNF-1β, hepatocyte nuclear factor-1β.
Transcriptional activity of the wild-type and mutant HNF-1β.
The WT and mutant (H153N, P159L) HNF1B expression plasmids (0.02 µg) were transfected into COS-7 cells with 0.2 µg of pGL3-hGLUT2 promoter.
β-Galactosidase activity was used for normalization of transfection efficiency.
(A) HNF-1β proteins expressed by Western blot.
(B) Human GLUT2 promoter regions inserted into the luciferase vector.
(C) The relative luciferase activity was measured.
Fold-induction refers to the activity with vehicle.
Result is represented as means ± SE of five independent experiments.
veh, vehicle; WT, wild-type; HNF-1β, hepatocyte nuclear factor-1β; GLUT2, glucose transporter type 2.
*p &lt; 0.05.
DNA-binding activity of the wild-type and HNF-1β mutants.
Electrophoretic mobility shift assay (EMSA) was done with oligonucleotide probes of the human GLUT2 promoter: 5&#39;-AAG ACC TCA GTA AAG ATT AAC CAT CAT TA-3&#39;.
The probe was incubated with nuclear extracts of COS-7 cells transfected with pcDNA, wild-type, and two HNF1B mutants.
(A) The proteins of HNF-1β were detected by Western blot.
(B) EMSA shows that wild-type HNF-1β is binding to the GLUT2 promoter, but mutants are not.
WT, wild-type; FP, fluorescence polarization; GAPDH, glyceraldehyde 3-phosphate dehydrogenase; HNF-1β, hepatocyte nuclear factor-1β; GLUT2, glucose transporter type 2.
Effects of mutant HNF-1β on the mRNA levels of GLUT2 and INS in MIN6.
MIN6 cells were transfected with wild-type or mutant HNF-1β.
Thirty hours after the transfection, cells were harvested and prepared for quantitative PCR analysis.
Isolated RNAs were quantitated by real-time PCR using specific primers for GLUT2 and INS.
The mRNA level of each gene was normalized by 18S.
(A) The transfection efficiency was about 60%-65%, measured by GFP transfection in MIN6 cells.
(B) The wild-type and mutant HNF-1β proteins were expressed well by Western blot.
GLUT2 (C) and INS mRNA (D) levels are represented as means ± SE of five independent experiments.
veh, vehicle; WT, wild-type; HNF-1β, hepatocyte nuclear factor-1β; GLUT2, glucose transporter type 2; GFP, green fluorescent protein.
*p &lt; 0.05.
Title: Analysis of Gene Expression in Cyclooxygenase-2-Overexpressed Human Osteosarcoma Cell Lines

Osteosarcoma is the most common primary bone tumor, generally affecting young people.
While the etiology of osteosarcoma has been largely unknown, recent studies have suggested that cyclooxygenase-2 (COX-2) plays a critical role in the proliferation, migration, and invasion of osteosarcoma cells.
To understand the mechanism of action of COX-2 in the pathogenesis of osteosarcoma, we compared gene expression patterns between three stable COX-2-overexpressing cell lines and three control cell lines derived from U2OS human osteosarcoma cells.
The data showed that 56 genes were upregulated, whereas 20 genes were downregulated, in COX-2-overexpressed cell lines, with an average fold-change &gt; 1.5.
Among the upregulated genes, COL1A1, COL5A2, FBN1, HOXD10, RUNX2, and TRAPPC2are involved in bone and skeletal system development, while DDR2, RAC2, RUNX2, and TSPAN31are involved in the positive regulation of cell proliferation.
Among the downregulated genes, HIST1H1D, HIST1H2AI, HIST1H3H, and HIST1H4C are involved in nucleosome assembly and DNA packaging.
These results may provide useful information to elucidate the molecular mechanism of the COX-2-mediated malignant phenotype in osteosarcoma.
Osteosarcoma is the most common primary bone tumor, representing over 56% of all bone tumors.
It generally affects young people with the age of 15 to 19 and usually occurs in long bones near the metaphyseal growth plates.
It has a high tendency to metastasize, and about 10%-20% of patients have macroscopic evidence of metastasis at the time of diagnosis, while 80%-90% of patients are assumed to have micrometastasis.
Although it has been postulated that rapid bone growth, exposure to radiation, and genetic predispositions, such as RB or p53 mutations, are thought to be risk factors for developing osteosarcoma, the etiology has not been fully understood [1, 2, 3].
Prostaglandin endoperoxide synthase 2 (PTGS2), also called as cyclooxygenase-2 (COX-2), catalyzes the convertsion of arachidonic acid to prostaglandin H2, from which various prostanoids, including prostaglandin E2, are produced [4].
Accumulating evidence indicates that COX-2 is involved in osteosarcoma development and progression.
Several studies have reported that high levels of COX-2 expression is associated with advanced clinical stage and metastasis [5, 6], as well as with lower overall survival rates and disease-free survival rates [7, 8, 9].
In addition, COX-2 inhibition by using RNAi or antisense oligonucleotide inhibits cell proliferation and invasion in human osteosarcoma cells [10, 11].
Also, selective COX-2 inhibitors reduce not only osteosarcoma cell proliferation and invasion in vitro but also tumor growth and metastasis in vivo [12, 13].
Moreover, we have previously reported that COX-2 overexpression promotes cell proliferation, migration, and invasion in U2OS human osteosarcoma cells [14].
These studies strongly suggest that COX-2 might be a causal factor for the development and progression of osteosarcoma.
However, the exact mechanisms of action of COX-2 in osteosarcoma are largely unknown.
In an attempt to figure out the mechanism of action of COX-2 in osteosarcoma, we analyzed the gene expression profiles in three COX-2-overexpressed U2OS stable cell lines and three control stable cell lines.
Title: Establishment and maintenance of stable cell lines

Human COX-2 cDNA was subcloned into the pcDNA3 vector containing neor.
U2OS cells were transfected with COX-2 or pcDNA3 DNA using Lipofectamine2000 (Life Technologies, Grand Island, NY, USA).
Transfectants were selected in the presence of geneticin, and individual clones were maintained in Dulbecco&#39;s modified Eagle&#39;s medium, containing fetal bovine serum (10%), penicillin (100 units/mL), streptomycin (100 units/mL), and geneticin (700 µg/mL), as reported previously [14].
Title: RNA isolation

Total RNA was extracted from cells with Trizol (Life Technologies), purified with the addition of chloroform, and precipitated with the addition of isopropanol.
The RNA concentration was determined by a spectrophotometer, and the quality of RNA was evaluated by the OD 260/280 ratio and gel electrophoresis.
Title: Hybridization to expression arrays

The following procedures were carried out by Macrogen Co. (Seoul, Korea).
First, total RNA was amplified and purified using the Ambion Illumina RNA amplification kit to yield biotinylated cRNA (Ambion, Austin, TX, USA).
Briefly, 550 ng of total RNA was reverse-transcribed to cDNA using a T7 oligo(dT) primer.
Second-strand cDNA was synthesized, in vitro-transcribed, and labeled with biotin-NTP.
After purification, 750 ng of labeled cRNA was hybridized to the humanHT-12 expression v.4 bead array (Illumina, San Diego, CA, USA) for 16-18 h at 58℃.
The array signal was detected using Amersham fluorolink streptavidin-Cy3 (GE Healthcare Bio-Sciences, Little Chalfont, UK).
Arrays were scanned with an Illumina bead array reader/confocal scanner.
Array data were filtered by a detection p-value &lt; 0.05 (similar to signal to noise).
Selected gene signal values were log-transformed and normalized by the quantile method.
Title: Statistical analysis

Basic statistical analyses were performed using Microsoft Excel.
Hierarchical cluster analysis was conducted with normalized log2-gene expression values using Cluster 3.0, and the results were visualized using Java Treeview [15, 16].
An unrooted tree was drawn with R package.
Biological function analysis was performed with official gene names using DAVID (http://david.abcc.ncifcrf.gov/).
Title: Stable cell lines

We have previously established stable cell lines over-expressing human COX-2 in U2OS human osteosarcoma cells.
To avoid clonal variations, we established three stable COX-2-overexpressing cell lines (U2OS-COX-2 #1, #2, and #3) and three control cell lines (U2OS-pcDNA3 #1, #2, and #3).
High levels of COX-2 expression were observed by western blot analysis in U2OS-COX-2 cells, whereas the expression was barely detectable in U2OS-pcDNA3 cells.
When we cultured the stable cell lines, we observed that cell proliferation, migration, and invasion rates were increased significantly in U2OS-COX-2 cells as compared with U2OS-pcDNA3 cells [14] (Table 1).
Title: Gene expression

We first performed unsupervised cluster analysis comparing COX-2-overexpressing cell lines and control cell lines.
Among 24,371 probe sets, we selected and analyzed 567 probe sets with a standard deviation &gt; 0.5.
The group of U2OS-COX-2 (U2OS-COX-2 #1, #2, and #3) and the group of U2OS-pcDNA3 (U2OS-pcDNA3 #1, #2, and #3) were well segregated in the unsupervised cluster analysis, confirming that our experiment was validated (Fig.1).
To identify significantly differentially expressed genes (DEGs) in COX-2-overexpressing osteosarcoma cells, we first performed the Wilcoxon rank sum test between the group of U2OS-COX-2 and the group of U2OS-pcDNA3.
The 2,144 probe sets passed the criteria.
Then, to identify DEGs, we selected 76 probe sets with an average fold-change &gt; 1.5.
Among the DEGs, 56 genes were upregulated while 20 genes were downregulated in the group of U2OS-COX-2 as compared to the group of U2OS-pcDNA3 (Tables 2 and 3).
The heat map of DEGs is provided in Fig.2.
The gene encoding COX-2, PTGS2, was upregulated by 10.41-fold in the COX-2-overexpressing cell lines as compared to the control cell lines, confirming again that our experiment is validated (Table 2).
Title: Functional analysis

Title: The DEGs were concentrated in several important biological functions

The functions that were most significantly upregulated by COX-2 overexpression in U2OS cells were skeletal system development and morphogenesis, bone development and morphogenesis, ossification, and positive regulation of cell proliferation (p &lt; 0.05) (Table 4).
The genes of COL1A1, COL5A2, FBN1, HOXD10, RUNX2, and TRAPPC2 are involved in skeletal system development and morphogenesis.
The genes of COL1A1, COL5A2, and RUNX2 are involved in bone development, morphogenesis and ossification.
The genes of DDR2, RAC2, RUNX2, and TSPAN31 are involved in the positive regulation of cell proliferation (Table 4).
The functions that were most significantly downregulated by COX-2 overexpression in U2OS cells were nucleosome assembly, chromatin assembly, and DNA packaging (p &lt; 0.05).
The genes of HIST1H1D, HIST1H2AI, HIST1H3H, and HIST1H4C are involved in these processes (Table 5).
COX-2 has been reported to promote cell proliferation, migration, and invasion in osteosarcoma cells.
COX-2 gene knockdown by RNAi significantly reduced the cell proliferation, migration, and invasion in SaOS2 cells [11], whereas stable expression of COX-2 increased the ability in U2OS cells [14] (Table 1).
These studies demonstrated that abnormal COX-2 overexpression is an important determinant for the malignant phenotype of osteosarcoma cells.
By using bioinformatics tools, we here provided useful information to understand the underlying mechanism by which COX-2 promotes the malignant phenotype in osteosarcoma cells.
According to our data, the genes encoding extracellular matrix proteins are highly upregulated by COX-2 (Tables 2 and 4).
The genes COL1A1, COL5A2, and FBN1 encode α1 chain of type I collagen, α2 chain of type V collagen, and fibrillin 1, respectively.
It has been known that the type I and type V collagens are major extracellular matrix proteins in bone [17].
Although there is a report that type I collagen promotes cell proliferation and migration in gastric cancer cells [18], the causal relationship between the overexpression of those genes and osteosarcoma has not been investigated well.
Therefore, further studies are necessary to elucidate whether overexpression of these genes might be involved in the COX-2-mediated malignant phenotype in osteosarcoma.
Extracellular matrix proteins not only are structural components of the connective tissue in the body but also regulate cell fates through their receptors on the cell surface.
DDR2 is one of the receptors for collagen, the expression of which was highly upregulated by COX-2 (Tables 2 and 4).
DDR2 is a receptor tyrosine kinase, and its overexpression promotes the migration and invasion of cancer cells derived from breast, head and neck, and prostate [19, 20, 21].
Therefore, COX-2 might promote cell proliferation and motility through upregulation of DDR2.
Further studies are necessary to elucidate this possibility.
RAC2, a highly upregulated gene by COX-2, is a member of the Rho family of GTPases (Tables 2 and 4).
Although there is no report on Rac2 function in osteosarcoma, there is a report that Rac2 GTPase is required for B cell proliferation and survival [22].
Therefore, COX-2 might promote cell proliferation through upregulation of Rac2 function.
Among the highly upregulated genes, RUNX2 encodes a transcription factor governing osteoblast differentiation and skeletal morphogenesis.
Recent studies on Runx2 function in cancer are controversial.
Runx2 overexpression reduces cell proliferation in osteosarcoma cell lines [23] but promotes cell motility or invasion in MCF-7 breast cancer cells and PC prostate cancer cells [24, 25].
Therefore, further studies are required to elucidate the function of Runx2 in osteosarcoma and the relation between COX-2 and Runx2.
Intriguingly, the genes encoding histone proteins are highly downregulated by COX-2 (Tables 3 and 5).
The genes HIST1H1D, HIST1H2AI, HIST1H3H, and HIST1H4C encode histone H1H1, histone H2A1, histone H3A1, and histone H41, respectively.
As is well known, the nucleosome is the basic unit of DNA packaging, consisting of a core particle and linker DNA.
A core particle comprises 147 base pairs of DNA wrapped around a histone octamer (2 copies each of histone H2A, H2B, H3, and H4).
The core particles are connected by 10-90 bp of linker DNA, which could be naked or bound to histone H1 [26].
The nucleosome itself has been known to be a general gene repressor [27, 28].
In addition, RNAi-mediated knockdown of H2A.Z reduced cell proliferation in bladder cancer cells [29].
These results are compatible with our finding that COX-2 downregulates the expression of histone genes.
Therefore, it is worth studying the mechanism of action of histones in osteosarcoma and the mechanism by which COX-2 downregulates histone genes.
Osteosarcoma is a highly malignant tumor, and the 5-year survival rate is only 20%-30% in patients with macroscopic evidence of metastasis at diagnosis [3].
While the etiology of osteosarcoma has been largely unknown, recent studies have suggested that COX-2 plays a critical role in osteosarcoma development and progression.
In this regard, we believe that our study will provide useful information to understand the mechanism of action of COX-2 in osteosarcoma.
Segregation between the group of U2OS-COX-2 and the group of U2OS-pcDNA3.
Unsupervised cluster analysis was done between six stable cell lines using 567 probe sets with a standard deviation &gt; 0.5.
Normalized log2-gene expression values were used for hierarchical clustering, and an unrooted tree was drawn by R package.
COX-2, cyclooxygenase-2.
Gene expression patterns of 76 differentially expressed genes (DEGs).
Hierarchical cluster analysis was done between six stable cell lines using 76 DEGs.
Normalized log2-gene expression values were analyzed by Cluster 3.0, and the results were visualized using Java Treeview, with red being high and green indicating low.
Title: Summary of stable cell line characteristics

Title: Upregulated DEGs in COX-2-overexpressed cell lines

Title: Downregulated DEGs in COX-2-overexpressed cell lines

Title: Biological functions of upregulated DEGs

Title: Biological functions of downregulated DEGs
Title: The Usage of an SNP-SNP Relationship Matrix for Best Linear Unbiased Prediction (BLUP) Analysis Using a Community-Based Cohort Study

Best linear unbiased prediction (BLUP) has been used to estimate the fixed effects and random effects of complex traits.
Traditionally, genomic relationship matrix-based (GRM) and random marker-based BLUP analyses are prevalent to estimate the genetic values of complex traits.
We used three methods: GRM-based prediction (G-BLUP), random marker-based prediction using an identity matrix (so-called single-nucleotide polymorphism [SNP]-BLUP), and SNP-SNP variance-covariance matrix (so-called SNP-GBLUP).
We used 35,675 SNPs and R package &#34;rrBLUP&#34; for the BLUP analysis.
The SNP-SNP relationship matrix was calculated using the GRM and Sherman-Morrison-Woodbury lemma.
The SNP-GBLUP result was very similar to G-BLUP in the prediction of genetic values.
However, there were many discrepancies between SNP-BLUP and the other two BLUPs.
SNP-GBLUP has the merit to be able to predict genetic values through SNP effects.
Many important human traits are moderately to highly heritable [1].
Yang et al.
[2] found that roughly 45% of the genetic variance of human height can be explained by the regression on common single-nucleotide polymorphisms (SNPs).
So-called dark matter of the genome in human height complex traits, especially, can be explained by the existence of gene-by-gene or gene-by-environment interactions [3,4,5].
Although the causal variants are not in complete linkage disequilibrium (LD) with the SNPs and although SNP-SNP interaction effects do exist, the traditional independent and identically distributed (IID) assumption has been used for so-called SNP-best linear unbiased predictor (SNP-BLUP).
We calculated an SNP-SNP relationship matrix from a genomic relationship matrix (GRM) for best linear unbiased prediction (BLUP) analysis.
This usage is called SNP-genomic linear unbiased prediction (SNP-GBLUP).
The prediction of SNP effects is the best feature of this BLUP.
This fact can help BLUP scientists predict the genetic values of other sample groups.
Also, through the SNP-SNP relationship matrix, scientists can view the SNP-SNP interaction via the covariance terms of that matrix.
We used the BLUP to estimate the numerical values of genetic factors that reside in an individual&#39;s DNA information.
The expression of diverse and complicated genetic factors can lead to phenotypic quantitative values, and the genetic factors can translate into numerical values.
SNPs can be the representatives of genetic factors.
Therefore, the invisible but real SNP effects can be translated into numerical values.
Title: BLUP, including best linear unbiased estimation (BLUE), is a standard method for predicting the random effects and fixed effects of a mixed model [6].
In general, random effects can be SNP effects, and fixed effects can be factors, such as sex.
This method was originally designed in the field of animal breeding for estimating breeding values but is now applicable to many areas of research [6].
In particular, the advent of DNA sequencing technology has made it possible for the application of the BLUP-BLUE mixed model to predict SNP effects.
The BLUP method, originally developed by Henderson, which is now applicable to animals, plants, and humans, uses statistical regression and exploits fixed effects as well as random effects.
It is a general model that is able to find polygenic effects.
In fact, polygenic effects are the replacement of quantitative trait loci (QTL) effects or randomly distributed genomewide effects.
The mixed linear model, including BLUP and BLUE, is in the form: y = Xb + Zu + e, (1) where y is the vector of phenotypic values of given quantitative traits and b and u are vectors of fixed and random effects, respectively.
X and Z are the design matrices.
The random effects and residual error effects are assumed to have a normal distribution as u~MVN(0,Gu) and e~MVN(0,R), where MVN indicates a multivariate normal distribution and E(y) = Xb, cov(u, e&#39;) = 0.
The general solution using the maximum likelihood (ML) method, which shows the effects of b and u, is in the form of: (2)

Here, Gu-1 is the inverse matrix of the variance-covariance matrix of random effects, which indicates the SNP-SNP variance-covariance matrix.
The GRM, G, can be calculated using R package &#34;rrBLUP.&#34;
For the feasible and precise prediction of the breeding value of organisms, genome-wide studies are quickly becoming mainstream.
SNP information can show us the precise and broad application of BLUP, because it is widely applicable, and hundreds of thousands of genetic markers (SNPs) can be tested for association with a phenotype [7].
We used three methods: GRM-based BLUP (G-BLUP), SNP-BLUP, and SNP-GBLUP.
G-BLUP is a GRM-based method.
SNP-BLUP is assumed to be IID between SNPs [8, 9].
SNP-GBLUP exploits the SNP-SNP relationship matrix.
We found out the calculation method of the SNP-SNP variance covariance matrix using the GRM G [9, 10, 11].
The tabular method, to calculate the Gu matrix, developed by Fernando and Grossman [12], uses the IBD concept, which is complicated and vulnerable to incorrect calculations in organisms.
There is no R package available in Gu matrix calculation (http://www.r-project.org).
However, the G matrix can be calculated from R codes [12, 13].
We proposed a simple method for calculating the Gu matrix from the G matrix.
The SNP-SNP relationship matrix means the variance-covariance matrix between SNPs.
Height in humans is a classical quantitative trait, is easy to measure, and has high heritability.
The heritability of height has been estimated to be ~0.8 [14, 15, 16, 17, 18].
We analyzed human height using the BLUP method.
There were 997 individuals and 35,675 total SNPs in the analysis.
Despite the use of many SNPs, we did not achieve satisfactory heritability results (about 30% of 0.8 total heritability).
However, it was consistent with the results, which showed that 45% of height variance can be explained by considering all SNPs simultaneously.
Title: Data preparation

We used the Ansan-Anseong cohort dataset in Korea.
This dataset was established for a Korean chronic diseases study, with Ansan and Anseong representing urban and rural areas of Korea, respectively.
The subjects were men between 40-69 years of age who had been residents of the region for at least 6 months.
The basic survey was conducted from 2000 to 2001, and our study was based on the 3rd Ansan-Anseong cohort dataset version 2.1.
We chose height as the phenotypic data and sex as a fixed effect.
The SNPs of the cohort dataset were implemented using Affymetrix Genome-wide Human SNP Array 5.0 (Affymetrix, Santa Clara, CA, USA).
The mean call rate was 99.01%, and the genetic analysis result, proved by SNPstream UHT 12 plex, was 99.934%.
The total number of genotyped SNPs was 352,228, and they were filtered using the conditions of minor allele frequency (MAF; &lt;0.01), Hardy-Weinberg equilibrium (&lt;0.0001), and missing genotyping (missing &gt;0.2).
The SNPs were pruned using PLINK (http://pngu.mgh.harvard.edu/~purcell/plink), leaving 35,675 SNPs for the analysis of human height; 997 individuals were involved in the analysis.
The GRM was calculated by using R package &#34;rrBLUP&#34; with the option &#34;Expectation-Maximization (EM) algorithm.&#34;
Then, using the restricted maximum likelihood method (REML) of the same packages, we calculated the SNP effects, genetic values, error variance, and genetic value variance.
The EM imputation algorithm was used for the GRM, because we dealt with high-density SNPs.
The REML method is used in a rather small size sample instead of the ML method.
Title: The method of statistical SNP-SNP variance-covariance matrix calculation

Title: From the regression Eq.
(1), Henderson [19] used the ML method to derive the BLUP equation, which is in the form: f(y, u) = g(y|u)h(u) = g(Xb + Zu + e|u)h(u) = g(e)m(u).
(3)

Title: Then, we used Zu as the independent variable instead of u, which denotes the breeding value or genetic value Zu for the derivation of BLUP.
We assumed the normality condition of the genetic value Zu and random effect u.
Based on these assumptions, the variance-covariance matrix of genetic values, Zu, is the GRM, G. f(y, Zu) = g(y|Zu)h(Zu) = g(e)m(Zu) = g(e)k(u)(4)

Title: From Eqs.
(3) and (4), (5)

Title: (6)

Eqs.
(5) and (6) represent the probability density function (pdf) of the BLUP model (Eq.
1).
Eqs.
(5) and (6) are equivalent, and we can easily know that Gu-1 = ZTG-1Z.
This important relationship links the SNP-SNP variance covariance matrix Gu to the GRM G.
Title: The generalized least squares for the BLUP

Title: We used the &#34;rrBLUP&#34; package for calculating the G matrix and predicting the genetic values of men&#39;s height complex traits.
&#34;rrBLUP&#34; uses a practically generalized least squares (GLS) solution for the BLUP.
For the G-BLUP, we used the G matrix, which was calculated from the R &#34;rrBLUP&#34; package using information on 35,675 SNPs.
For the random effects estimation, such as SNP effects, we used two methods.
One was SNP-BLUP, which is assumed to be IID between SNPs, and the other was SNP-GBLUP, which uses the statistical SNP-SNP variance covariance matrix.
We will call this the SNP-SNP relationship matrix.
(7) (8)

Title: The Sherman-Morrison-Woodbury lemma

Title: From the relationship Gu-1 = ZTG-1Z, we should calculate the inverse matrix of the Gu-1 matrix.
Because a lot of time was needed for calculating the inverse matrix directly, we used the Sherman-Morrison-Woodbury (SMW) lemma [20, 21, 22, 23].
The formula was as follows: (A + YGZ*)-1 = A-1 - A-1Y(G-1 + Z*A-1Y)-1Z*A-1,(9) where A and G are both invertible, and A + YGZ* are invertible if and only if G-1 + Z*A-1 Y are invertible.
Practically, to reduce the error, we used A as the identity matrix (I matrix), and the practically used formula was as follows: (10)

Calculating the SNP-SNP relationship matrix Gu using the lemma was faster than the direct calculation, because we used larger SNPs than the sample size.
Title: Genetic value prediction and SNP-GBLUP

The result of the genetic value prediction is shown in Fig.1 (Fig.1A, G-BLUP; Fig.1B, SNP-BLUP; Fig.1C, SNP-GBLUP).
These results show the genetic value variances are nearly identical between G-BLUP and SNP-GBLUP.
However, SNP-BLUP is very disparate from the other two BLUPs.
Fig.2 shows the histogram of SNP random effects (Fig.2A, SNP-BLUP; Fig.2B, SNP-GBLUP).
Both cases indicate that the SNP effects in both BLUPs are distributed normally and that they suit the normal assumption of SNP effects.
However, the result of the genetic values was very different in both cases, as shown in Fig.3 and Supplementary Table 1.
Supplementary Table 1 shows the height phenotype and the genetic value prediction of the three BLUPs.
G-BLUP and SNP-GBLUP showed identical results.
The estimated residual error variance was 21.39 in the three BLUP cases.
The fixed effect was 166.6 (men) and 153.3 (women) in both G-BLUP and SNP-GBLUP and 167.4 (men) and 154.1 (women) in SNP-BLUP.
Title: SNP-SNP relationship matrix

Supplementary Table 2 shows the calculated 1-8th SNP components of the Gu square matrix obtained using the SMW lemma.
The variance components (diagonal parts) were the values close to 1.
It suggests that the Gu matrix can be interpreted as the SNP-SNP relationship matrix.
The covariance components can be interpreted as the SNP-SNP relationships or, in other words, SNP-SNP interaction terms, because the variance components represent the SNPs&#39; relationships themselves.
Title: Estimated heritability

We estimated the narrow-sense heritability (h2) using simple regression between the genetic values and the height phenotypic values.
The heritability was 0.24 in G-BLUP and SNP-GBLUP and 0.20 in SNP-BLUP.
According to Yang et al.
&#39;s article [2], only 45% of variance can be explained by total SNPs.
Therefore, our estimated heritability was not poor, and of the generally accepted narrow-sense heritability of 0.8, we explained 33% in G-BLUP and SNP-GBLUP.
We explained 53% of the heritability when using total SNPs.
Also, we found the method that is widely applicable, SNP-GBLUP, because it can predict the SNP effects.
It is impossible in G-BLUP, and Gu matrix components can be used widely, such as in genome-wide association studies (GWASs), because they can be interpreted as representing the interaction terms between SNPs.
Title: SNP-GBLUP and its applicability

The G-BLUP uses the GRM.
This matrix contains the information between individuals.
However, SNP-BLUP and SNP-GBLUP use the direct information between SNPs.
SNP-BLUP is assumed to be IID between SNPs.
This IID assumption is good but not accurate, because it ignores the interaction terms between SNPs.
Instead, SNP-GBLUP uses the covariance structures directly.
This BLUP is more proximal to actuality than SNP-BLUP.
The covariance terms of the SNP-SNP relationship matrix can be used as interaction terms in phenotype-related analysis.
It can provide the scientist with clues for SNP-SNP interactions.
The covariance terms of it are independent of the phenotypic values, because they came from the Z matrix and GRM.
However, after proper constant multiplication, as in BLUP, they may be interpreted as interaction terms between SNPs.
Probably, the SNP effects through SNP-GBLUP rather than SNP-BLUP can be used in Bayesian BLUP.
Bayesian BLUP excludes the low or zero effects of SNPs.
The accurately calculated SNP effects through SNP-GBLUP can be classified into low-effect SNPs and high-effect SNPs as absolute values.
Low-effect SNPs can be excluded, and Bayesian BLUP can exploit only high-effect SNPs [24, 25, 26].
Title: Heritability and the GLS approach in BLUP

Narrow-sense heritability (h2) reflects the additive effects of QTL.
The estimated heritability from our data was smaller than the generally accepted heritability.
This is because causal variants were not in complete LD with the SNPs that were genotyped.
Incomplete LD might occur if causal variants have a lower MAF than genotyped.
The effects of the SNPs are treated statistically as random, and the SNPs have a small effect on the trait [2].
However, we achieved better heritability than the predicted value in common GWASs-i.e., only ~5% of the phenotypic variance in human height [27].
The present studies aim at predicting genetic values using the covariance structures between SNPs via GRM [28].
The numerator relationship matrix can be replaced by a genome-based GRM [29].
This G-BLUP, using GRM, uses information of SNPs [30].
We used three methods-particularly, the method of normal distribution-based SNP-SNP covariance structures.
These structures were constructed using GRM and the Z design matrix.
Algorithmically, the calculation of GRM uses the frequency of genotyped SNPs and the Z matrix.
Also, in the previous study, the SNP-SNP covariance structures assumed that the identity matrix was multiplied by proper constants.
However, we calculated the SNP-SNP relationship matrix using GRM and the Z matrix.
This square matrix contains the information of relations between SNPs and can make it possible to estimate the SNP effects.
Also, adequate use of the SMW lemma could make it plausible to estimate the SNP effects and genetic values simultaneously with faster computation time [21].
We expect that SNP-SNP covariance structures can make it feasible for calculating the genetic values and SNP effects that are linked to the QTL more accurately.
Genetic value estimation combines the performance and kinship information, which is based on a known pedigree [31].
The accuracy of genomic prediction depends on many factors, such as the genetic architecture of the trait and the population, the methodology of estimating the SNP effects, the distribution of the markers, and the degree of LD [32].
These can be important for accurate BLUP analysis, and there can be two approaches for solving BLUP.
The GLS approach and Henderson&#39;s ML-based approach lead to identical solutions.
The &#34;rrBLUP&#34; package uses the GLS approach.
In that package, when the GRM is calculated, the imputation uses the mean method or EM algorithm.
Especially, the EM algorithm can be used for haplotype inferences with SNPs.
The EM algorithm is the most popular statistical algorithm because of its interpretability and stability.
This algorithm is a deterministic procedure, consumes less computation time, and is easier for convergence checks [33].
The REML method is used in BLUP analysis with a small sample size.
The use of REML for variance component estimation avoids the small sample bias associated with fixed effects and seems to avoid selection bias.
Henderson [34] asserted the use of REML in BLUP models [35, 36].
Thus, we used the EM algorithm and REML method schemes.
The histogram of genetic value variances of G-BLUP (A), SNP-BLUP (B), and SNP-GBLUP (C).
The shapes of the histograms of G-BLUP and SNP-GBLUP were nearly identical.
However, the shape of the histogram of SNP-BLUP was disparate from the other two BLUPs.
G-BLUP, genomic relationship matrix-based best linear unbiased prediction; SNP-BLUP, single nuleotide polymorphism (SNP)-best linear unbiased predictor; SNP-GBLUP, SNP-genomic linear unbiased prediction.
The histogram of the SNP effects of SNP-BLUP (A) and SNP-GBLUP (B).
They were approximately distributed normally.
However, the predicted genetic values were very dissimilar.
SNP-BLUP, single nuleotide polymorphism (SNP)-best linear unbiased predictor; SNP-GBLUP, SNP-genomic linear unbiased prediction.
The histogram of the genetic values of G-BLUP (A), SNP-BLUP (B), and SNP-GBLUP (C).
The genetic value distribution was similar to the normal distribution.
G-BLUP, genomic relationship matrix-based best linear unbiased prediction; SNP-BLUP, single nuleotide polymorphism (SNP)-best linear unbiased predictor; SNP-GBLUP, SNP-genomic linear unbiased prediction.
Title: Genome-Wide Identification and Classification of MicroRNAs Derived from Repetitive Elements

MicroRNAs (miRNAs) are known for their role in mRNA silencing via interference pathways.
Repetitive elements (REs) share several characteristics with endogenous precursor miRNAs.
In this study, 406 previously identified and 1,494 novel RE-derived miRNAs were sorted from the GENCODE v.19 database using the RepeatMasker program.
They were divided into six major types, based on their genomic structure.
More novel RE-derived miRNAs were confirmed than identified as RE-derived miRNAs.
In conclusion, many miRNAs have not yet been identified, most of which are derived from REs.
MicroRNAs (miRNAs) are small non-coding RNAs of ~22 nucleotides (nt) in length and are single-stranded in their mature form.
Primary miRNAs are expressed from genomic regions and processed to generate precursor miRNAs by Drosha [1].
Precursor miRNAs have a hairpin structure; therefore, their sources (or, the genomic loci from which they originate) have a palindromic structure [2].
The precursor miRNAs are exported from the nucleus to the cytoplasm and processed into a duplex by Dicer, one of which is preferentially loaded into Argonaute (AGO) [1, 3].
Mature miRNAs function via the RNA-induced silencing complex and AGO protein-mediated binding to the target mRNA by complementary base pairing to the 3&#39; untranslated region [4].
Repetitive elements (REs) are interspersed throughout the genome, and they increase genomic instability through various mechanisms.
REs consist of transposable elements (TEs) and tandem repeats (e.g., satellite DNA, simple repeat DNA).
REs can directly impact coding sequences or other functional sequences in the host genome as follows.
They can affect transcription by acting as alternative promoters [5, 6, 7], forming structural isoforms through alternative exons, and by providing polyadenylation signal sites important in transcriptional termination [8].
REs are also important for inhibiting gene expression at the post-transcriptional level by producing miRNA sequences [2, 9].
REs comprise paralogous miRNA gene families and speciesspecific miRNA gene families [10].
Some miRNAs originate from unique genomic sequences, and others originate from REs.
Recently, the association of REs with miRNAs was established in several studies that demonstrated connections between miRNAs and TEs [9, 11, 12].
These studies suggest that REs are important for miRNA origin, expression, and regulatory network formation [2, 9, 12, 13, 14, 15, 16].
Especially, some REs have a palindrome structure, and these sequences have great potential to make a precursor miRNA form.
In previous studies, miniature inverted TE (MITE)-derived miRNAs were identified in the human genome [2].
As one of the REs, medium reiterated sequences (MERs) have a palindrome structure in the mammalian genome [17].
MERs were also predicted to make miRNAs [10], and MER-derived miRNAs were confirmed in experiments in human cell lines [11].
REs are ubiquitous and scattered throughout the host genome in abundant numbers; therefore, these RE families have the possibility of making paralogous miRNAs.
A MITE-derived miRNA, miR-548, has many homologous gene families [18], and a MER-derived miRNA, miR-1302, also has many homologous miRNAs in the human genome [10].
Likewise, the long interspersed elements (LINE) element also makes an miRNA precursor form by &#34;tail to tail&#34; method [12].
In the case of hsa-miR-28, two LINE elements are oppositely oriented and then make one miRNA precursor form [15].
Based on these results, we separated miRNAs in the case of two REs making one miRNA and of a palindrome structure RE making one miRNA.
REs prefer rapid evolution compared to other genomic sequences; RE-derived miRNAs have a tendency to make phylogeny-specific miRNAs [9].
In this respect, primate-specific Alu-derived miRNAs are primate-specific, and MITE-derived miR-548 was mainly discovered in primates [18].
Genomic duplication events, such as segmental duplications or tandem duplications, also create REs and RE-derived miRNAs in animals [19, 20].
Therefore, many RE-derived miRNAs were identified in the human, rhesus, and mouse genomes [20].
In the plant genome, TE insertions can make both siRNAs and miRNAs, and MITEs have an important role in the creation and evolution of novel miRNAs [13].
In this respect, to analyze TE-overlapping patterns and abundant overlapping TEs with miRNAs in the human genome can provide evolutionary clues in further studies.
In the miRBase database, 55 experimentally validated human miRNA genes derived from TEs are described, and 85 novel miRNAs are predicted from the potential conserved secondary structures of 587 human TEs [9].
However, these studies concentrated exclusively on the identification of miRNAs containing REs and did not analyze the patterns of overlap between REs and miRNAs.
Moreover, newly identified miRNAs and small transcripts with the potential to form miRNAs have not been considered.
Therefore, we analyzed TEs that overlapped with both previously identified and novel miRNAs and examined six patterns of overlap that occur.
Our results suggest that REs contribute to the production of human miRNA genes by a number of mechanisms.
Title: Computational analysis

We used miRNAs annotated as small non-coding RNA genes defined by the GENCODE database, v.19 (http://www.gencodegenes.org) [21].
RepeatMasker outputs (hg19 assembly, RM v.330, repbase libraries 20120124) were obtained from the University of California, Santa Cruz (UCSC) genome browser (http://genome.ucsc.edu/, hg19).
To analyze the intersection between miRNA and REs, we used the intersectBed command (with options -wa and -wb) in BEDTools [22].
Title: Classification of miRNAs into six types

The RE-matched precursor miRNA sequences obtained were divided into six major types.
The six types were as follows: miRNAs that overlapped with two or more REs (type 1), miRNAs related to the TcMar-Mariner family (type 2) and the MER family (type 3), and miRNAs sorted according to the matching scheme between REs and miRNAs (type 4 to type 6).
The classification scheme is described in a flow chart (Supplementary Fig.1).
miRNA sorting and deletion of duplications were performed using Microsoft Excel (Supplementary Table 1).
Title: MiRNAs originating from two or more REs in the human genome

In total, 1,900 miRNAs were confirmed as RE-originated, including 406 previously identified miRNAs and 1,494 novel miRNAs (Table 1).
We identified 452 type 1 miRNAs, which have two or more RE-derived precursor miRNAs (23.79%) (Fig.1A).
Only 72 previously identified miRNAs (or 15.93%) and 380 novel miRNAs were classified as type 1 (Fig.2).
Most identified miRNAs overlapped with two REs, and three (miR-325, miR-649, and miR-5692b) overlapped with three REs.
Interestingly, miR-649 consisted of three different RE families, including a LINE, short interspersed element (SINE), and DNA transposon.
Those regions with three or more RE-derived miRNAs were more likely to be novel miRNAs.
Three novel miRNAs (AC079412.1, AL158077.1, and AL356865.1) overlapped with five REs.
Title: Palindromic structure of RE-derived miRNAs

Precursor miRNAs form palindromic structures.
Therefore, TE families with a palindromic sequence structure, including MITEs and MERs, have the potential to form mature miRNAs [2, 10, 11].
Both of these RE families may be able to form miRNA sequences themselves.
Therefore, we assigned MITE-derived miRNAs and MER-derived miRNAs into the type 2 and type 3 categories, respectively.
Type 2 precursor miRNAs were distinguished by the presence of MITEs, specifically MADE1, which consists of two 37-bp terminal inverted repeats flanked by 6 bps of the internal sequences; 390 regions were included in this category (20.53%) (Fig.1B).
For most type 2 miRNAs, the MADE1 sequences constituted more than 90% of the total miRNA sequences.
These miRNA sequences may or may not have RE-derived sequences on both terminal sides, and miRNA precursors containing RE-derived sequences were classified as type 1.
The palindromic sequence structure of MADE1 has the potential to form precursor miRNAs, and several studies identified mature MADE1-derived miRNAs.
In previous studies, MADE1-derived miRNAs were identified as a part of the miR-548 gene family [2].
Seed shifting events in the miR-548 gene family were detected by evolutionary analysis [18].
According to our criteria, most genes in the miR-548 family were type 2 miRNAs, including seven miRNAs that were previously identified [2].
However, miR-548a-2 and miR-548a-3 were classified as type 1, because MADE1 sequences were inserted into RE sequences, and together, these RE sequences produce miRNA sequences.
Interestingly, MADE1-derived miRNAs are inserted into the specific (hot-spot) sequence TA-TAT or repetitive sequences, such as LINEs, long terminal repeat (LTR) elements, and other DNA elements.
Some MADE1-derived miRNAs harbored hot-spot sequences (TA-TAT) in their miRNA gene sequences (Fig.3).
Therefore, MADE1-derived miRNAs likely formed the miR-548 family, known to be primate-specific [2, 18].
Type 3 precursor miRNAs were identified by the presence of MER sequences (14.05%, n = 267) (Fig.1C).
Most MER-derived miRNA precursor sequences overlapped with MER sequences, because MER palindrome sequences are similar to miRNA precursor sequences and may be able to form miRNA sequences themselves [11].
Notably, miR-1302-5 was classified as a type 1 precursor miRNA, because it combined two RE families (MER53 and AluSx).
Title: Patterns of RE-overlap with miRNAs

Type 4 precursor miRNAs harbor one RE sequence (2.53%, n = 48) (Fig.1D).
Precursor miRNAs are approximately 60-80 nt [23]; so, it is unlikely for REs to occur in precursor miRNA sequences.
Most type 4 miRNAs contained Low_complexity and Simple_repeat, because they tend to be relatively short.
In the Simple_repeat family, a short repeat sequence helps to form miRNA precursor sequences by binding regions of complementary short repeat sequences.
For example, miR-574 contains repeat (TG)n in its precursor regions (Supplementary Fig.2A), and (TG)n sequences contain miR-574-5p sequences (Supplementary Fig.2B).
Type 5 precursor miRNAs were those formed from flanking sequences and one RE (9.63%, n = 183) (Fig.1E).
This category had the highest ratio of identified miRNAs to total miRNAs.
The novel miRNA nomenclature process requires cloning or expression evidence.
Then, this information is described in a manuscript accepted for publication [24, 25].
The identified miRNAs have a tendency to be abundantly and ubiquitously expressed in the host.
Intriguingly, two RE families, SINE/mammalian-wide interspersed repeat (MIR) and LINE/L2, were commonly detected in type 5 miRNA precursor sequences.
These two families were abundant in conserved segments and were commonly detected in murine intergenic regions of human orthologs [26].
These results indicate that the L2 and MIR TE families were highly conserved and that these RE-derived miRNAs have important functional roles in the host.
Taken together, type 5 miRNAs are relatively abundant and evolutionarily conserved.
By contrast, type 6 precursor miRNAs, or those formed from a single RE, represented 29.47% of the sample (n = 560) (Fig.1F).
The REs contained in type 6 precursor miRNAs have the potential to produce miRNA sequences themselves.
In type 6 miRNAs, SINE/Alu elements are the most common.
In this study, we classified miRNAs based on overlap patterns in identified miRNAs and novel miRNAs.
In some cases, two or more REs were approached by &#34;tail to tail&#34; method and then making one miRNA precursor form.
We classified these cases as type 1, and LINE/L1-derived miRNAs were abundantly discovered (Table 2).
A previous study showed that miR-558 is derived only from MLT1C in the LTR family [9], but we found two repeat families (LTRs and simple repeats) in the locus.
We also determined that miR-619 and miR-1302-5 were derived from the combination of two adjacent repeat families.
Recently, the updated human genome assembly was open to the public (GRCh38/hg38), and novel REs have also been identified.
Our classification of RE-overlapping miRNAs can provide the criterion to explain the origins, evolution, and family expansion of miRNAs in another human genome assembly or in other species.
In type 1 precursor miRNAs, some cases overlapped with the same families of TEs, which can make it possible to form an miRNA precursor form in a &#34;tail to tail&#34; scheme.
In a previous study, two LINE elements were predicted to form in an oppositely oriented method, like hsa-miR-28 [12], and LINE was also the most abundant TE family in the type 1 miRNAs in our study.
This result can help identify new miRNAs derived from two or more REs.
MADE1s are broadly distributed among eukaryotes and function as regulatory RNAs in many genomes [27].
They are expressed with the gene sequences in which they are inserted.
This phenomenon provides active opportunities for MADE1 hairpins to function through an RNA interference enzymatic mechanism involved in functional gene regulation [28].
We identified miRNAs that consisted of more than 90% MADE1 sequences and determined the specific mechanisms underlying the formation of MADE1-derived miRNA palindrome sequences.
Previous reports found 103 orthologs of the miR-1302 family in placental mammals.
Moreover, the family has undergone multiple duplication events, and some of the duplicated genes have diverged functionally (e.g., RNA-based TE defense mechanisms), whereas others have become pseudogenes or have been eliminated from the genome [10].
Therefore, it has been suggested that the miRNA gene family evolved according to a birth-and-death model [10, 29, 30, 31].
Alu elements and miRNAs are related.
Alu elements and those resulting from duplication events are induced to make new miRNAs and, specifically, an miRNA cluster on chromosome 19 (C19MC) [32].
C19MC presents primate-specific imprinted patterns in the placenta and may be an example of co-evolution between Alu elements and miRNAs [33, 34].
Alu elements are abundant in human chromosome 19; hence, miRNAs are also abundantly detected, including C19MC [32].
Most SINE elements, such as Alu, tRNAs, and 5s-rRNAs, are expressed by polymerase III (pol III), and several miRNAs are expressed by pol III.
Some miRNAs are expressed together with Alu elements by pol III [35].
These data also demonstrate the strong relationship between miRNA and Alu elements.
In an evolutionary aspect, primate-specific miRNAs were discovered to contain an Alu element in their precursor form.
Other small REs, such as tandem duplications, occupy the middle region of miRNA precursor forms.
Therefore, type 4 miRNAs have abundant Simple_repeat families (Table 2).
These data demonstrate that miRNAs have been made by genomic evolutionary events.
In conclusion, we determined that 1,900 RE-derived miRNAs can be divided into six major types.
Of them, 406 identified miRNAs and 1,494 novel miRNAs were confirmed using the GENCODE database, and their RE patterns were sorted using the RepeatMasker program.
The results suggest that RE sequences were interspersed throughout the genome and form miRNA precursor sequences that play important roles in the host genome.
These regions may contribute to the evolution of biological complexity.
Schematic of the precursor miRNAs derived from repetitive elements (REs) located in the genome.
REs and precursor miRNAs are shown as horizontal bold bars and upper thin bars, respectively.
The type 1 precursor miRNAs are formed by two adjacent REs (A).
Type 2 precursor miRNAs are formed by MADE1 (B), whereas type 3 precursor miRNAs are formed by medium reiterated sequence (MER) (C).
The type 4 precursor miRNAs contain RE sequences (D), and the type 5 precursor miRNAs are located between the flanking sequences and REs (E).
The type 6 precursor miRNAs are retained within the RE sequences (F).
The ratio of identified miRNAs to total miRNAs.
Hot-spot sequences at both ends of MADE1 insertion sequences.
MADE1 is inserted into TA-TAT sequences.
Title: The number of REs-derived miRNAs and their rates

Title: Distribution of RE-derived miRNAs by type
Title: Molecular Characterization of Legionellosis Drug Target Candidate Enzyme Phosphoglucosamine Mutase from Legionella pneumophila (strain Paris): An In Silico Approach

The harshness of legionellosis differs from mild Pontiac fever to potentially fatal Legionnaire&#39;s disease.
The increasing development of drug resistance against legionellosis has led to explore new novel drug targets.
It has been found that phosphoglucosamine mutase, phosphomannomutase, and phosphoglyceromutase enzymes can be used as the most probable therapeutic drug targets through extensive data mining.
Phosphoglucosamine mutase is involved in amino sugar and nucleotide sugar metabolism.
The purpose of this study was to predict the potential target of that specific drug.
For this, the 3D structure of phosphoglucosamine mutase of Legionella pneumophila (strain Paris) was determined by means of homology modeling through Phyre2 and refined by ModRefiner.
Then, the designed model was evaluated with a structure validation program, for instance, PROCHECK, ERRAT, Verify3D, and QMEAN, for further structural analysis.
Secondary structural features were determined through self-optimized prediction method with alignment (SOPMA) and interacting networks by STRING.
Consequently, we performed molecular docking studies.
The analytical result of PROCHECK showed that 95.0% of the residues are in the most favored region, 4.50% are in the additional allowed region and 0.50% are in the generously allowed region of the Ramachandran plot.
Verify3D graph value indicates a score of 0.71 and 89.791, 1.11 for ERRAT and QMEAN respectively.
Arg419, Thr414, Ser412, and Thr9 were found to dock the substrate for the most favorable binding of S-mercaptocysteine.
However, these findings from this current study will pave the way for further extensive investigation of this enzyme in wet lab experiments and in that way assist drug design against legionellosis.
Legionella pneumophila is a gram-negative intracellular facultative pathogen that is mainly responsible behind hospital and community-acquired legionellosis and about 90% cases of legionellosis are caused by this species [1].
Legionellosis patients predominantly have pneumonia, chills, fever even their cough likely to be dry or phlegm nature.
L. pneumophila isolation by comparing clinical and environmental L. pneumophila isolates precludes different sources whether it is contagious or not through a number of typing methods.
Such methods of typing make it&#39;s easier like pulsed-field gel electrophoresis (PFGE), is usually considered to be extremely biased [2, 3, 4, 5, 6].
PFGE can recognize unique strains of L. pneumophila with a precise profile that is considered sporadic.
The origin of L. pneumophila sg 1 clone was identified at 1997.
In Paris, legionellosis was caused by a single L. pneumophila sero group 1 strain [7].
An ample of enzymes is required in bacterial metabolism.
Drugs resistance against L. pneumophila considered to the search for most novel drugs of designing.
At present, computational analysis was taken place in order to discover novel drug targets that are non-homologous to human.
All enzymes involved in metabolic pathway of those certain bacteria are precursor to design such kind of drugs.
Phosphoglucosaminemutase and phosphomannomutase, these two typically have the potential target sites.
Phosphomannomutase is processed with glycosylation, adding the groups of sugar molecules (oligosaccharides) to proteins.
The enzyme phosphoglucosamine mutase catalyzes the chemical reaction alpha-D-glucosamine 1-phosphate to D-glucosamine 6-phosphate, which converts alpha-D-glucosamine 1-phosphate to D-glucosamine 6-phosphate.
This enzyme is phenomenally the same as phosphomannomutase, which transfers a phosphate group within a molecule.
The systematic name of phosphoglucosamine mutase is alpha-D-glucosamine 1,6-phosphomutase.
It participates in metabolism of amino sugars.
Phosphoglucosamine mutase (GlmM) catalyzes the formation of glucosamine-1-phosphate from glucosamine-6-phosphate, an essential step in the pathway for UDPN-acetylglucosamine biosynthesis in bacteria.
This enzyme must be phosphorylated to be active and acts according to a ping-pong mechanism involving glucosamine-1,6-diphosphate as an intermediate [8].
The phosphoglucosamine mutase auto-phosphorylates in vitro in the presence of ATP.
The same is pragmatic with phosphoglucosamine mutases from other bacterial species, yeast N-acetylglucosamine-phosphate mutase, and rabbit muscle phosphoglucomutase.
Labeling of GlmM enzyme with ATP requires divalent cation.
The label can be lost if it is incubated more vigorously with of its substrates.
At glycosylation, the phosphomannomutase enzyme converts mannose-6-phosphate to mannose-1-phosphate [9].
Mannose-1-phosphate is converted into GDP-mannose which transfers mannose to the growing oligosaccharides chain.
Congenital disorder type Iais is initiated by mutations in the PMM2 gene.
Mutations alter the formation of phosphomannomutase enzyme that lead to the reduced enzyme activity and shortage of GDP mannose within cells.
As there have no enough activated mannose, incorrect oligosaccharides are produced.
Abnormal glycosylated proteins in organs and tissues regulate the signs and symptoms in CDG-Ia [10].
In addition, it participates in the metabolism of both fructose and mannose.
So, homology modeling will predict the desired function and possible disease treatment if needed because of its importance on cell metabolism systems.
The present study is aimed to predict the three-dimensional (3D) structure of phosphoglucosamine mutase by means of homology modeling.
Consequently, to depict its structural features and to comprehend the molecular function, the structural model for the desired protein was constructed.
Title: Sequence retrieval

The amino acid sequences of the enzyme phosphoglucosamine mutase in L. pneumophilia (strain Paris) were retrieved from the UniProt Knowledge Base (UniProtKB) database, which is the foremost hub for the compilation of well-designed information on proteins, with consistent, accurate, and rich annotation.
The accession ID of phosphoglucosamine mutase is Q5X1A3, and it contains 455 amino acids.
Title: Analysis of physico-chemical properties

ProtParam (http://web.expasy.org/protparam/), a tool of Expasy was used for the analysis of the physiological and chemical properties from the protein sequence.
This tool can predict different physico-chemical properties, like the molecular weight, isoelectric pH, aliphatic index, grand average of hydropathicity (GRAVY), and extinction coefficients.
Title: Secondary structure prediction

Secondary structure was predicted by using the self-optimized prediction method with alignment (SOPMA) [11].
The protein&#39;s secondary structure includes an α helix, 310 helix, pi helix, beta bridge, extended strand, beta turns, bend region, random coil, ambiguous states, and other states.
SOPMA predicts these properties by using homologous protein identification, sequence alignment, and conformational score determination method.
Prediction accuracy was confirmed by correlation coefficient value.
Plain text format data were inputted, and default parameters were set.
Title: Protein-protein interaction networking

Protein cooperates with other proteins to perform accurate functions.
To identify protein-protein interactions, STRING was used.
STRING is a biological database that is used to construct protein-protein interaction (PPI) networks for different known and predicted interactions.
At present, the database covers up to 5,214,234 proteins from 1,133 organisms [12].
Title: Model building

3D structure determination of a protein is the key step of structural genomics initiative [13].
To predict the 3D homology model of phosphoglucosamine mutase, Phyre2 (Protein Homology/Analogy Recognition Engine) [14], the most popular online protein fold identification server, was used.
Phyre2 uses a dataset of known proteins taken from different reliable databases, such as Structural Classification of Proteins (SCOP) database and Protein Data Bank (PDB).
Through sequential steps, such as profile construction, similarity analysis, and structural properties, Phyre2 selects the best suited template and generates a protein model.
To get an accurate model, intensive mode of protein modeling was selected.
The input data of this enzyme were in FASTA format.
In this respect, the intensive mode of protein modeling was selected in order to get an accurate model.
After model building, it is necessary to further refine in quest of the best model generation.
Title: Model refinement

Homology-based modeling often contains significant local distortions that render the structure models less useful for high-resolution functional analysis.
To refine the predicted protein model, ModRefiner [15], an algorithm for atomic-level, high-resolution protein structure refinement, was used.
Protein sequences were given in the FASTA format, and refinement was done for several times to get the most accurate structure.
Title: Evaluation and validation of the model

The accuracy and stereo chemical quality of the predicted model were evaluated with PROCHECK [16, 17] by Ramachandran plot [18] analysis, which was done through &#34;Protein structure and model assessment tools&#34; of Swissmodel workspace; 2.5 Å resolution was selected for PROCHECK analysis.
The best model was selected on the basis of overall G-factor, number of residues in the core, and allowed, generously allowed, and disallowed regions.
ERRAT [19], Verify3D [20], and Qualitative Model Energy Analysis (QMEAN) [21] were used for further analysis of the selected model.
The verified structure was visualized by Swiss-PDB Viewer [22].
Title: Active site analysis

The active site is the specific region of the target protein responsible for its activity and is constructed of different amino acids.
To identify the active site with the determined model, Computed Atlas of Surface Topography of proteins (CASTp) [23] server was used.
This provides an online resource for locating, delineating, and measuring concave surface regions on three-dimensional structures of proteins, including pockets located on protein surfaces and voids buried in the interior of proteins.
Title: Docking simulation study

Molecular docking is a computer simulation procedure to calculate the conformation of a receptor-ligand complex.
It is used to identify the binding affinity and interaction energy of the molecules with the target protein.
Docking analysis was performed by AutoDock Vina [24], which is an automated procedure for predicting the interaction of ligands with bio-macromolecular targets.
Before initiating the docking stimulations, phosphoglucosamine mutase protein was modified by adding polar hydrogen, removing all the water molecules, and was also set with the grid box for its binding site, whereas all the torsional bonds of ligands were set free by the ligand module.
To evaluate the binding energy on the macromolecule coordinate, a three-dimensional grid box (box size, 76 × 76 × 76 Å; box center, 11 × 90.5 × 57.5 for x, y, and z, respectively) was created using Auto Grid, which calculates the grid map.
The combined binding with target protein phosphoglucosamine mutase and ligand, s-mercaptocysteine, was obtained by using PyMOL (The PyMOL Molecular Graphics System, version 1.5.0.4, Schrödinger, LLC) [25].
The UniProt Knowledge Base (UniProtKB) delivers an authoritative resource for protein sequences and functional information.
Sequences of phosphoglucosamine mutase of Legionella pneumophilia (strain Paris) were obtained from UniProtKB.
Manual annotation is the landmark of the SwissProt section of UniProtKB [26, 27].
The protein sequence was analyzed using the ProtParam server [28], which can predict the physical and chemical parameters for the protein.
The parameters of this server are helpful for experimental handling of the protein, like biological analysis or extraction.
ProtParam results reveal that the protein has 23,295 extinction coefficients, 27.68 instability index, 108.00 aliphatic index, and 0.059 grand average of hydrophobicity, with more positively charged residues than negatively charged amino acids.
The physico-chemical properties of phosphoglucosamine mutase are tabulated in Table 1.
Secondary structure analysis is increasing day by day to predict protein function and structure.
The secondary structure of phosphoglucosamine mutase was predicted by SOPMA with standard parameters.
Secondary structure parameters of phosphoglucosamine mutase are presented in a tabulated form in Table 2, which shows it contains 42.64% alpha helix, 18.80% extended strand, 9.89% beta turn, and 29.69% random coil.
The graphical secondary structure of phosphoglucosamine mutase is shown in Fig.1.
PPI network generation has become very important tool of the modern biomedical research arena for the understanding of complex molecular mechanisms and the detection of novel modulators of disease processes.
These types of work have been shown to be very important in the study of a wide range of human diseases, as well as their signaling pathways [29, 30, 31].
PPI of phosphoglucosamine mutase was generated through STRING, presented in Fig.2.
STRING forecasts a confidence score and 3D structures of protein and protein domains.
Confidence scores were generated on the basis of different parameters, like neighborhood, co-occurrence, coexpression, and homology.
STRING utilizes references from UniProt and predicts functions of different interacting proteins.
PPI network demonstrates that phosphoglucosamine mutase interacts with 10 other proteins, such as mur E is hypothetical protein with a confidence score of 0.687; glmU is bifunctional N-acetylglucosamine-1-phosphate (Glc-N-1-P) uridyltransferase/glucosamine-1-phosphate N-acetyltransferase (UDP-GlcNAc), which catalyzes the last two sequential reactions in the de novo biosynthetic pathway for UDP-GlcNAc.
It is also responsible for the acetylation of Glc-N-1-P to give GlcNAc-1-P and for the uridyl transfer from UTP to GlcNAc-1-P, which produces UDP-GlcNAc.
This protein is closely related to glmM with the highest confidence score (0.998).
The second highest confidence protein is glmS, glucosamine-fructose-6-phosphate aminotransferase, which catalyzes the first step in hexosamine metabolism, converting fructose-6P into glucosamine-6P using glutamine as a nitrogen source.
Another important protein, murC (confidence score, 0.667), UDP-N-acetylmuramate-L-alanine ligase, works in cell wall formation.
Homology modeling of the unique and essential metabolic protein was done by using Phyre2 in order to obtain the 3D structure of them.
3D protein structures give important insights about the molecular basis of protein function and thereby allow an effective design of experiments [32].
That is why, in the understanding and manipulation of biochemical and cellular functions of proteins, the high-resolution 3D structure of a protein is the main key [19].
Phyre2 generated the best suited result, showing that the predicted structure had a 100% confidence level and uses the template c3pdkB.
Secondary structure and disorder prediction leads to a conclusion that phosphoglucosamine mutase has disordered region of 4%.
To gain a more accurate model, refinement through ModRefiner was done.
After derivation by the ModRefiner, the refined model (Fig.3) of the desired enzyme was analyzed for further advancement.
In the initial model of phosphoglucosamine mutase, the percent of residues in the favored region in the Ramachandran plot was 84.0% versus 95.0% in the final model.
The red, brown, and yellow colored regions are the symbol of the favored, allowed, and generously allowed regions, respectively, the same as defined by PROCHECK (Fig.4).
Parameters, such as residue in the favored, allowed, and generously allowed region and G-factor, are the determinants of a good model [33, 34, 35].
Subsequent to that, PROCHECK was used to measure the accuracy of protein models.
Parameter comparisons of these proteins were made with well-refined structures that have similar resolution.
Through PROCHECK analysis, specific information about the protein chains and their stereochemical quality, like Ramachandran plot quality, peptide bond planarity, bad non-bonded interactions, main chain hydrogen bond energy, C alpha chirality, and overall G factor, and the side chain parameters like standard deviations of chi1 gauche minus, can be obtained [36].
Ramachandran plot statistics of phosphoglucosamine mutase revealed that most of the amino acid residues (above 90% of amino acid residues) were present in most favored regions (Table 3).
Thus, the protein model was very good, seeing that all of the residues were within the limits of the Ramachandran plot.
Verification was also done by ERRAT, Verify3D, and QMEAN server.
ERRAT uses a quadratic error function to characterize and differentiate between correctly and incorrectly determined regions of protein structures based on characteristic atomic interaction [37].
The overall quality of the model by ERRAT analysis was 89.791.
The Verify3D graph value of the model is 0.71, which indicates that the environmental profile of phosphoglucosamine mutase is quite good [38, 39, 40].
On the basis of a linear combination of six structural descriptions, the QMEAN scoring function estimates the global quality of the models.
The local geometry model analysis is done by a torsion angel potential over three consecutive amino acids, and the quality of the model can be compared to a reference structure of high resolution obtained from X-ray crystallography analysis through Z score.
QMEAN Z-score provides an estimation of the &#34;degree of nativeness&#34; of the structural features observed in a model and indicates that the model is of comparable quality to experimental structures [41].
The assessing of long-range interactions is carried through secondary structure specific distance-dependent pairwise residue level potential.
A solvation potential describes the burial status of the residues.
Secondary structure element and accessibility agreement ensures the quality assessment between the predicted and calculated secondary structure and solvent accessibility [21].
The respective values of Z-scores of C_β interaction energy, solvation energy, torsion angle energy, secondary structure, and solvent accessibility are 0.28, -1.03, -1.20, 1.43, and 1.12 in the case of phosphoglucosamine mutase, as shown in Table 4.
The overall QMEAN score for phosphoglucosamine mutase is 1.11.
QMEAN-generated results confer phosphoglucosamine mutase as a qualified model for further drug target scopes.
The active site of phosphoglucosamine mutase was predicted by using CASTp server.
Further, in this study, we have also reported the best active site area of the experimental enzyme, as well as the number of amino acids involved in it.
Fig.5 shows the interacting residues Arg419, Thr414, Ser412, and Thr9 with protein-ligand from the docking that had been suggested by CASTp which was found in the active site of phosphoglucosamine mutase.
In order to understand docking studies with phosphoglucosamine mutase and s-mercaptocysteine, analysis of lowest docked energy value, calculated Root mean square deviation value, involvement of H bonds, and interacting residues was considered (Table 5).
Least docked energy postulates a better docking result.
Receptor-ligand analysis of our predicted protein shows the lowest energy of -4.0 kcal/mol, as well as a root mean square distance of 0.000 Å, and it contains four hydrogen bonds.
Thr414, Val306, Asp308, Gly307, Thr9, Asp10, and Ser412 are the interacting molecules where ligand interacts with the protein receptor.
S-mercaptocysteine (3-disulfanyl-L-alanine (2R)-2-amino-3-disulfanyl-propanoic acid), which has a molecular weight of 153.22 g/mol (Table 6), was found to bind at the active site of phosphoglucosamine mutase with the lowest binding energy (Fig.6).
It has been clear that s-mercaptocysteine formed similar hydrogen bond interactions with phosphoglucosamine mutase.
From the active site analysis, 41 amino acid residues were found in the potent active site.
The interaction between the active site residues and the ligand found in our present study is useful for perceiving the potential mechanism of residues and the drug binding.
The hydrogen bonds play a significant role for the structure and function of biological molecules, and we found significant results.
Among the 41 residues, Thr9, Arg246, Val306, Gly307, Asp308, Ser412, and Thr414 interacted with the ligand; the others did not.
Docking analysis with ligand identified specific residues-viz.
Thr414, Ser412, and Thr9 (Fig.7)-within the phosphoglucosamine mutase binding pocket to play an important role in ligand binding affinity, which further itself inhibits its function and exposes studies about new drug discovery.
The putative drug targets phosphoglucosamine mutase, phosphoglyceromutase, and phosphomannomutase for legionellosis have been reported as potential in the literature.
That is why in our study, the 3D structure of phosphoglucosamine mutase from L. pneumophila (strain Paris) was predicted and validated by a variety of bioinformatics tools and software.
Analyzing the results, it could be concluded that future characterization of phosphoglucosamine mutase from L. pneumophila (strain Paris) will be noteworthy for the regulation of legionellosis.
The modeled 3D structure will provide a good-quality foundation for experimental development of the crystal structure, and the molecular docking study will assist efficient inhibitor design against legionellosis in the future.
Predicted secondary structure of phosphoglucosamine mutase.
Here, helix is indicated by blue, while extended strands and beta turns are indicated by red and green, respectively.
Protein-protein interaction network of phosphoglucosamine mutase.
Refined model of the protein phosphoglucosamine mutase.
Ramachandran plot analysis of predicted protein through Swiss model Workshop.
Active site determination of the phosphoglucosamine mutase protein.
(A) The green region indicates the most potent active site.
(B) The amino acid residues in the active site.
The overall binding between the phosphoglucosamine mutase and S-mercaptocysteine.
Graphical representation of docking study between S-mercaptocysteine and phosphoglucosamine mutase (yellow dashed lines indicate hydrogen bonds).
(A) Visualization of S-mercaptocysteine-phosphoglucosamine mutase.
(B) Hydrogen bond detection through PyMOL.
Title: Physicochemical properties of phosphoglucosamine mutase from Legionella pneumophila (strain Paris)

Title: Calculated secondary structure elements of phosphoglucosamine mutase of Legionella pneumophila by SOPMA

Title: Ramachandran plot statistics of phosphoglucosamine mutase from Legionella pneumophila (strain Paris)

Title: Z-scores of QMEAN for individual component

Title: Comparative docking study of the ligand to the target

Title: Description of the ligand molecule
Title: Understanding Rifampicin Resistance in Tuberculosis through a Computational Approach

The disease tuberculosis, caused by Mycobacterium tuberculosis (MTB), remains a major cause of morbidity and mortality in developing countries.
The evolution of drug-resistant tuberculosis causes a foremost threat to global health.
Most drug-resistant MTB clinical strains are showing resistance to isoniazid and rifampicin (RIF), the frontline anti-tuberculosis drugs.
Mutation in rpoB, the beta subunit of DNA-directed RNA polymerase of MTB, is reported to be a major cause of RIF resistance.
Amongst mutations in the well-defined 81-base-pair central region of the rpoB gene, mutation at codon 450 (S450L) and 445 (H445Y) is mainly associated with RIF resistance.
In this study, we modeled two resistant mutants of rpoB (S450L and H445Y) using Modeller9v10 and performed a docking analysis with RIF using AutoDock4.2 and compared the docking results of these mutants with the wild-type rpoB.
The docking results revealed that RIF more effectively inhibited the wild-type rpoB with low binding energy than rpoB mutants.
The rpoB mutants interacted with RIF with positive binding energy, revealing the incapableness of RIF inhibition and thus showing resistance.
Subsequently, this was verified by molecular dynamics simulations.
This in silico evidence may help us understand RIF resistance in rpoB mutant strains.
Decades after the discovery of the Mycobacterium tuberculosis (MTB) organism, tuberculosis (TB) remains a major cause of morbidity and mortality in several developing countries.
Nearly one-third of the world&#39;s population is considered to be infected with MTB infection, with 8.6 million new patients and 1.3 million deaths in the year 2012, including 3,20,000 deaths among human immunodeficiency virus-positive individuals, with around 2.0-2.4 million infected cases of TB in India alone-i.e., 26% of the total burden [1].
Multidrug-resistant strains of this pathogen, emerging in association with human immunodeficiency virus, have added a frightening dimension to the problem [2].
Outbreaks of extensively drug-resistant tuberculosis have also been an increasing threat in certain regions around the world [3].
Most drug-resistant MTB clinical strains are resistant to the frontline anti-TB drugs isoniazid (isonicotinic acid hydrazine) and rifampicin (RIF) [4].
RIF belongs to the rifamycin group and is a bactericidal antibiotic drug introduced in 1963 [5].
It inhibits bacterial DNA-dependent RNA synthesis by inhibiting bacterial DNA-dependent RNA polymerase [6, 7].
It has a complex structure, having a molecular weight 822.95 g/mol and containing an aromatic nucleus (Fig.1) linked on both sides by an aliphatic bridge [8].
Due to the lipophilic profile of the MTB cell membrane, RIF diffuses easily across it [9].
It inhibits bacterial RNA synthesis by binding to the β subunit of DNA-dependent RNA polymerase, thus blocking RNA transcription [6] by inhibiting RNA chain initiation and elongation [10].
Even though the molecular target of RIF has been well distinguished, the specific mechanism by which it interacts with rpoB to lead to mycobacterial killing remains unclear [8].
RIF resistance is mainly due to mutations in a well-defined, 81-base-pair central region of the rpoB gene [11].
More than 96% of RIF-resistant strains contain a mutation in this 81-bp region of rpoB [12, 13].
The mutation that is most frequently found in clinical isolates of MTB is rpoB S531L.
In MTB from countries around the world, rpoB S450L (S531L in the case of Escherichia coli) was identified in 40%-93% of all RIF resistance isolates [14,15,16].The single amino acid mutation at codon 445 (i.e., His to Tyr [CAC/TAC]) of the rpoB gene is also reported to be the most widespread mutation, associated with RIF resistance [17].
The rapid advances in molecular biology and the accessibility of new information generated after whole-genome sequencing of MTB will be useful in understanding the mechanism of RIF resistance.
In this study, we explored the interaction of rpoB (both wild and mutant proteins) with RIF through molecular docking analysis.
Title: Hardware and software

The study was carried out on a Dell Workstation with a 2.26 GHz processor, 6 GB RAM, and 500 GB HDD running in the Windows operating system.
Bioinformatics software, such as AutoDock4.2, and online resources were used.
Title: MTB rpoB protein

DNA-directed RNA polymerase beta chain (rpoB), encoded by Rv0667 of MTB, is reported to be directly involved in RIF resistance.
The rpoB protein sequence was retrieved from NCBI (http://www.ncbi.nlm.nih.gov/).
Title: Protein structure prediction and validation of rpoB

Protein Homology/Analogy Recognition Engine (Phyre2) server [18] was used for modeling the three-dimensional structure of rpoB.
Phyre2 is an automatic fold recognition server that predicts the three-dimensional structure of a protein sequence using the principles and techniques of homology modeling.
The Phyre is widely used by the biological community, with more than 150 submissions per day [18].
The Phyre2 server was publicly released in February 2011 as a replacement for the original Phyre server and provides extra functionality over Phyre, a more advanced interface, and a fully updated fold library and uses the HHpred/HHsearch package for homology detection.
Two mutants of rpoB (S450L and H445Y) were generated using Modeller9v10 [19].
Structural refinement and energy minimization of the predicted models were carried out using the Yet Another Scientific Artificial Reality Application (YASARA) Energy Minimization Server [20].
The refined model reliability was assessed through Procheck [21], ProSA-web [22], and ProQ [23].
Title: Ligand preparation

The ligand RIF used in this study against rpoB was retrieved from the PubChem database [24] in mol2 format, and the PyMol molecular graphics system (http://www.pymol.org) was used to convert it into PDB format.
Title: Protein-ligand docking

Protein-ligand docking studies were performed using the AutoDock 4.2 program [25].
It is one of the most widely used methods for protein-ligand docking.
All pre-processing steps for the ligand and protein files were performed using the AutoDock Tools 1.5.4 program (ADT), which has been released as an extension suite to Python Molecular Viewer [25].
The ADT program was used to prepare the receptor molecule (rpoB) by adding all hydrogen atoms to the receptor, and Kollman charges were also assigned.
For docked ligands, non-polar hydrogens were also added.
Gasteiger charges were assigned, and torsional degrees of freedom were allocated by the ADT program.
The Lamarckian genetic algorithm (LGA) was applied to model the interaction pattern between the receptor and ligand.
The grid maps representing the receptor proteins in the docking process were calculated using AutoGrid (part of the AutoDock package).
A grid of 50, 50, and 50 points in the x, y, and z directions was centered on the known active site residues of rpoB protein.
For all docking procedures, 10 independent genetic algorithm runs with a population size of 150 were considered for each molecule under study.
A maximum number of 25 × 105 energy evaluations; 27,000 maximum generations; a gene mutation rate of 0.02, and a crossover rate of 0.8 were used for the LGA.
The AutoDock program was run in order to prepare the corresponding docking log (DLG) file for further analysis.
Title: Molecular dynamic simulations of the rpoB-RIF complex

The docking complexes of RIF with wild and mutant (S450L) rpoB were subjected to molecular dynamic (MD) simulation using the GROMACS 4.6.3 [26] simulation packages, employing the gromos54A7 all-atom force field, as this particular mutant has more impact on drug resistance than H445Y.
The protein was protonated at default pH, and the ligand topology was generated by the PRODRG server [27].
Then, the complexes were solvated by explicit SPC/E (extended simple point charge) water model [28] in cubic boxes with a minimum edge distance of 7 Å from the box.
Both systems were neutralized electrically by adding 0.10 mM NaCl.
Both systems had been relaxed through steepest descent energy minimization, followed by temperature (NVT-constant number of particles, volume and temperature) and pressure (NPT-constant number of particles, pressure and temperature) equilibrium for 100 ps each.
The NVT was performed at 300 K with position restraints applied to all of the backbone atoms, and during NPT, all of these restraints were removed.
All bond lengths were constrained using the linear constraint solver (LINCS) algorithm [29].
For the temperature and pressure coupling, the velocity rescale thermostat [30] and isotropic Parrinello-Rahman barostat [31] were applied, respectively.
For the calculation of long-range electrostatic interactions, Particle Mesh Ewald (PME) was used with a fourth-order spline interpolation and 0.15-nm Fourier grid spacing [32].
The short-range non-bonded van der Waals and Coulombic interactions were considered between particles within 12 Å of cutoff distance.
During the production run, all of these protocols were followed along with the temperature coupling at 323 K using the velocity rescale thermostat with a coupling time constant of (τT) of 0.8 ps and pressure coupling by the Parrinello-Rahman barostat at 1 bar, via a coupling constant of τP = 2 ps with compressibility at 0.000045/bar.
Snapshots of the trajectory were taken every 2 ps, and the Microsoft Excel program was used for preparation of the graph.
Title: Visualization

The visualization of the structure files was done using the graphical interface of the ADT tool and the PyMol molecular graphics system (http://www.pymol.org).
RpoB of MTB, encoded by Rv0667, has 1172 amino acids in its protein sequence, while E. coli rpoB has 1342.
The S531L and H526Y mutations are based on the protein sequence of E. coli rpoB protein.
Upon sequence comparison using ClustalW [33] of rpoB protein in both species, it was found that in MTB, the mutation occurred at codon positions 450 (S450L) and 445 (H445Y) as compared to E. coli (Fig.2).
Title: Validation of model

Since the three-dimensional structure of rpoB of MTB was still not available, its structure model was predicted using the Phyre2 server by taking chain C of DNA-directed RNA polymerase subunit beta from E. coli (PDB ID, 3LU0) [34] as the structural template.
The query coverage of the target-template alignment was 91% with 100.0% confidence by the single highest-scoring template.
The predicted structure was then subjected to the YASARA Energy Minimization Server for structural refinement.
The total energy for the refined structure obtained from the YASARA Energy Minimization Server was -130,601.7 kcal/mol (score, -2.60),whereas prior to energy minimization, it was 359,868,668,000.6 kcal/mol (score, -4.20).
The stereochemistry of the refined model (Procheck analysis) revealed that 87.1% of residues were situated in the most favorable region, and 12.3% was in additional and generously allowed regions, whereas only 0.5% of residues fell in the disallowed region of the Ramachandran plot (Fig.3A).
The ProSA-web evaluation revealed a compatible Z score (Fig.3D) value of -12.3, which is well within the range of native conformations of crystal structures [22].
The overall residue energies of the rpoB 3D model were largely negative, except for a few peaks.
The 3D model of rpoB showed a Levitt-Gerstein (LG) score of 4.065 by the Protein Quality Predictor (ProQ) tool, implying a high accuracy level of the predicted structure.
A ProQ LG score &gt; 4.0 is necessary for suggesting that a model has extremely good quality [23].
The single amino acid mutation at codon 450 (i.e., Ser to Leu [TCG to TTG]) and at codon 445 (i.e., His to Tyr [CAC/TAC]) of the rpoB gene is reported to be the most widespread mutation, associated with RIF resistance [17].
Thus, mutant models of rpoB (S450L and H445Y) were generated and subjected to validation.
The stereochemistry of the rpoB (S450L) model (Procheck analysis) revealed that 89.3% of residues were situated in the most favorable region, and 9.9% was in additional allowed and generously allowed regions, whereas only 0.8% of residues fell in the disallowed region of the Ramachandran plot (Fig.3B).
In the case of the rpoB (H445Y) model, 87.5% of residues were found in the most favorable region, and 11.6% was in additional and generously allowed regions; only 0.9% of residues fell in the disallowed region (Fig.3C).
The ProSA-web evaluation revealed a compatible Z score value of around -12.1 for both rpoB mutant models (i.e., S450L and H445Y) (Fig.3E and 3F), which is well within the range of native conformations of crystal structures [22].
The overall residue energies of the rpoB 3D model were largely negative, expect for a few peaks.
The 3D model of both mutants showed an LG score of more than 4.1 by the ProQ tool, implying a high accuracy level of the predicted structure.
A ProQ LG score &gt; 4 is necessary for suggesting that a model has extremely good quality [23].
Title: Docking analysis of rpoB and RIF

On the basis of the modeled structure of rpoB (Fig.4A), three-dimensional structures of the two mutants were obtained.
The protein (rpoBs)-ligand (RIF) analysis was performed using AutoDock4.2 software [25].
Out of the 10 poses obtained, the best ligand pose was selected based on the lowest binding energy confirmation.
As the amino acid (serine at 450 and histidine at 445) position of rpoB was reported to undergo mutation in most of the RIF resistance strains, the docking of RIF was performed around these amino acid positions.
Upon docking, RIF interacts with the wild-type rpoB protein (Fig.4B) with a binding energy of -4.96 kcal/mol and inhibition constant of 231.21 µM, whereas in the case of the rpoB (S450L) and rpoB (H445Y) mutants, the binding energies were found to be 8.38 kcal/mol and 3.91 kcal/mol, respectively, with no inhibition constants predicted, which revealed RIF resistance in the mutant proteins.
Title: MD simulation studies of the rpoB and RIF complex

A ligand binding interaction always influences the stability of the receptor protein.
It was shown in Fig.5A that the Root Mean Square Deviation (RMSD) graph of wild-type rpoB lies a bit down than the S450L mutant during most of the time scale.
The same was also observed in the Root Mean Square Fluctuation (RMSF) plot (Fig.5D), where most interacting amino acids of wild-type rpoB that participate in the interaction with RIF fluctuate less comparatively.
This justifies that the sustainability of the rpoB-RIF interaction and wild-type rpoB is more acceptable over the mutant.
We also calculated the average number of hydrogen bonds and their occurrence between the receptor and ligand in both cases and found that in the case of wild-type rpoB, the protein formed more consistent numbers of H-bonds with the ligand (Fig.5B) as a consequence of a good interaction.
Most importantly, these bindings were also described via interaction energies (IEs).
The sum of short-range Coulombic and van der Waals interaction energies, taken as the total interaction energy between the protein and ligand, was plotted in Fig.5C.
This plot clearly indicates a big difference in IEs between the two complexes, and the wild-type rpoB possesses comparatively less interaction energy and hence shows more affinity towards RIF.
Brandis and Hughes [15] created the rpoB S531L mutation in Salmonella which is also present in other location of MTB and demonstrated that the minimum inhibitory concentration value of RIF for rpoB S531L mutant was 3,000 mg/L, whereas for the wild-type, it was only 12 mg/L [15].
This correlates with our in silico docking and MD simulation study-that mutant proteins with positive binding energy bind ineffectively with RIF compared to wild-type protein and thus may require high concentrations of RIF for inhibition.
We have employed a computational approach to study the interaction between RIF and rpoB and its mutant models.
Our in silico docking study revealed that mutation in rpoB at amino acid positions 450 (S450L) and 445 (H445Y) might be involved in drug resistance.
In wild-type rpoB, the RIF binds more effectively with rpoB with low binding energy and thus inhibits rpoB protein.
Other mutations in rpoB need to be explored further for understanding the resistance mechanism.
Chemical structure of rifampicin (formula, C43H58N4O12; molecular weight, 822.95 g/mol).
Sequence alignment of rpoB protein of Escherichia coli and Mycobacterium tuberculosis (MTB) with the respective codon position and reported amino acid substitution.
(A) Ramachandran plot of predicted rpoB model.
(B) Ramachandran plot of predicted rpoB (S450L).
(C) Ramachandran plot of predicted rpoB (H445Y) mutant models.
(D) Zplot of rpoB.
(E) Zplot of rpoB (S450L) model.
(F) Zplot of rpoB (H445Y).
ProSA-web Z-scores of all protein chains in PDB, determined by X-ray crystallography and nuclear magnetic resonance (NMR) spectroscopy, with respect to their length.
The Z-score of rpoB and its mutants were present in that range, represented as black dots.
(A) 3D structure of predicted rpoB model.
(B) Docking interaction of rifampicin with rpoB enzyme (with binding energy of -4.96 kcal/mol), showing hydrogen bonds as dotted lines.
(A) Root Mean Square Deviation plot of rpoB wild-type and S450L mutant.
(B) Occurrence of hydrogen bonds between rpoB and rifampicin (RIF) throughout the simulation with standard deviation.
(C) Interaction energy profile between rpoB and RIF.
(D) Root Mean Square Fluctuation plot comparison of wild-type and S450L mutant near the binding site region.
Red and blue are used for the wild-type and S450L mutant, respectively.
Title: Elucidation of the Inhibitory Effect of Phytochemicals with Kir6.2 Wild-Type and Mutant Models Associated in Type-1 Diabetes through Molecular Docking Approach

Among all serious diseases globally, diabetes (type 1 and type 2) still poses a major challenge to the world population.
Several target proteins have been identified, and the etiology causing diabetes has been reasonably well studied.
But, there is still a gap in deciding on the choice of a drug, especially when the target is mutated.
Mutations in the KCNJ11 gene, encoding the kir6.2 channel, are reported to be associated with congenital hyperinsulinism, having a major impact in causing type 1 diabetes, and due to the lack of its 3D structure, an attempt has been made to predict the structure of kir6.2, applying fold recognition methods.
The current work is intended to investigate the affinity of four phytochemicals namely, curcumin (Curcuma longa), genistein (Genista tinctoria), piperine (Piper nigrum), and pterostilbene (Vitis vinifera) in a normal as well as in a mutant kir6.2 model by adopting a molecular docking methodology.
The phytochemicals were docked in both wild and mutated kir6.2 models in two rounds: blind docking followed by ATP-binding pocket-specific docking.
From the binding pockets, the common interacting amino acid residues participating strongly within the binding pocket were identified and compared.
From the study, we conclude that these phytochemicals have strong affinity in both the normal and mutant kir6.2 model.
This work would be helpful for further study of the phytochemicals above for the treatment of type 1 diabetes by targeting the kir6.2 channel.
Type 1 and type 2 diabetes mellitus is a major health problem worldwide.
In case of type 1 diabetes, the body is unable to produce insulin due to the autoimmune destruction of the beta-cells in the pancreas [1], and insulin injection is the only known preventive measure in this case [2].
Most of the anti-diabetic drugs of synthetic origin have serious adverse effects; therefore, phytochemicals and plant extracts with anti-diabetic properties have been tested both in vivo and in vitro as alternatives for diabetic treatments [3].
Phytochemicals, such as curcumin (Curcuma longa), genistein (Genista tinctoria), piperine (Piper nigrum), and pterostilbene (Vitis vinifera), are reported to have potent anti-diabetic properties.
Curcumin, when tested in diabetic animals, exhibited a good sign for the prevention and treatment of diabetic encephalopathy [4].
Genistein also plays important roles in the regulation of glucose homeostasis in type 1 diabetes by down-regulating G6Pase, PEPCK, fatty acid β-oxidation, and carnitine palmitoyl transferase activities while up-regulating malic enzyme and glucose-6-phosphate-dehydrogenase activities in the liver, with preservation of pancreatic β-cells.
The supplementation of genistein is helpful for preventing insulin-dependent diabetes mellitus onset [5] and piperine, an alkaloid, has also been reported to possess potential anti-diabetic effects [6].
Experimental results suggested the antiglycemic effects of pterostilbene in an induced rat model of hyperglycemia.
Therefore, the antioxidant and antihyperglycemic activities of pterostilbene may confer a protective effect in preventing diabetes [7].
Several receptors (insulin-like growth factor receptor, glucose transporter, and kir6.2) and their associated signaling pathways have been elucidated and are involved in glucose regulation and diabetes.
But, a significant gap still remains as to making the choice of the drug against the target receptor in the disease condition.
Kir6.2, a major subunit of the ATP-sensitive K+ channel, an inward-rectifying potassium ion channel, is an integral membrane protein that allows K+ to flow from the outside of the cell to the inside, which is controlled by G-proteins associated with sulfonylurea receptor (SUR), to constitute the ATP-sensitive K+ channel.
During glycolysis, an increase in the ATP/ADP ratio blocks KATP channels, causing membrane depolarization, and helps in opening the voltage-dependent calcium channel, which facilitates the influx of calcium, triggering the exocytosis of insulin.
Mutations in the two subunits of SUR1 and kir6.2 result in the opening of the pancreatic KATP channel and permanent closing of the calcium channel, thus blocking insulin exocytosis.
Mutations in KCNJ11, the gene encoding the channel, are reported to be associated with congenital hyperinsulinism [8, 9].
Ten possible mutations affecting the regular mechanism of kir6.2 [8, 10, 11] have been identified as probable causes of type 1 diabetes.
Due to the unavailability of the crystal structure of kir6.2 protein, an attempt was made here to predict both the secondary and tertiary structures using in silico approach.
The objective of the current investigation is to describe atomic interactions and the inhibitory effect between both wild-type and mutant models of kir6.2 with phytochemicals, such as curcumin, genistein, piperine, and pterostilbene, computationally.
Title: Structure prediction and model validation of kir6.2 (wild, mutant)

The primary sequence of kir6.2 (entry name, KCNJ11_HUMAN) was retrieved from SWISS-PROT/UniProt KB (ID, Q14654).
The secondary structure of both wild-type and mutant kir6.2 was predicted using Discovery Studio 3.5.
The tertiary structure prediction for kir6.2 models (both wild type and mutant) were performed using the protein fold recognition server Phyre2 (http://www.sbg.bio.ic.ac.uk/phyre2/) with 100% confidence.
The predicted models were energetically minimized in ModRefiner (http://zhanglab.ccmb.umich.edu/ModRefiner/) to draw the initial starting models closer to their native state, in terms of hydrogen bonds, backbone topology, and side chain positioning.
The coordinates of the predicted model of kir6.2 were submitted in Protein Model DataBase (http://bioinformatics.cineca.it/PMDB/).
Structural validation was done in Procheck and Errat2 of the Saves web server (http://nihserver.mbi.ucla.edu/SAVES_3/).
Protein model quality was checked in the protein structure analysis tool ProSA (http://prosa.services.came.sbg.ac.at/prosa.php).
Energetically minimized structures of both kir6.2 models were visualized and superimposed in Discovery Studio 3.5.
Title: Molecular docking of kir6.2 model with phytochemicals

Molecular docking between kir6.2 models and phytochemicals was accomplished using the AutoDock-4.2 algorithm (http://autodock.scripps.edu/).
Kollman united atom charges, solvation parameters, and polar hydrogens were added into the kir6.2 PDB files for the preparation of the protein in the docking simulation.
Chemical structures of phytochemicals having anti-diabetic properties were extracted from the Pubchem (http://ncbi.nlm.nih.gov/pccompound) database of the National Center of Biotechnology Information (NCBI) web server (Supplementary Fig.1) in SDF format and converted to three-dimensional structures in PDB format using Discovery Studio 3.5.
Ligand molecules were prepared by choosing the root and restricting the number of torsion within a minimum range.
AutoDock requires precalculated grid maps, one for each atom type present in the flexible molecules being docked, and it stores the potential energy arising from the interaction with rigid macromolecules.
One hundred runs of blind dockings were performed for all inhibitors with both the wild-type and mutant protein model of kir6.2.
Then next round of docking was performed by setting a grid of 98, 100, and 62 points in the x, y, and z directions, respectively, in the ATP-binding site of kir6.2 [12] for both models by applying the Lamarckian genetic algorithm.
The best poses of docking during both rounds were compared and reported with the binding energy reported for each case.
The visualization and analysis were performed using Discover Studio 3.5 package.
Title: Structure prediction and validation of kir6.2 structural models

The primary sequence of kir6.2 (UniProt ID, Q14654) is a transmembrane protein with 390 amino acid residues.
Ten mutations [10] were introduced in the primary sequence of kir6.2 for mutant model prediction purposes.
Two tertiary structural models were predicted for both wild (template PDB ID, 3JYC) and mutant-type kir6.2 (template PDB ID, 3SYA) separately using Phyre2 [13] with 100% confidence, followed by energy minimization using ModRefiner [14] web server.
The structural model of kir6.2 is publically available in the Protein Model DataBase (PMID ID, PM0079770).
The quality of the model was compared with three existing models of kir6.2 available in the Protein Model Portal (http://www.proteinmodelportal.org/query/uniprot/Q14654).
The first model available in the Protein Model Portal was done for the amino acid region of 179-351, taking template PDB ID 1U4F, chain C with a 53% identity score.
However, this model is not complete in comparison to the present predicted model of kir6.2 (PMID, PM0079770), which is modeled from amino acid region 33-358.
The second and third models available are almost complete and modeled for the same amino acid region as in the present model of wild kir6.2.
While comparing the quality with all experimentally determined protein chains in the current PDB using the ProSA web server [15] by means of a significant statistical score (z-score) in terms of folding energy, the second and third models had a z-score computed in the ProSA web server of -5.16 (Supplementary Fig.2B) and -5.6 (Supplementary Fig.2C), respectively, while in comparison, the current model had a good folding energy score-i.e., z-score of -5.96 (Supplementary Fig.2A).
The overall quality factors of these two models were also compared with the current predicted model of kir6.2 using Errat2.
The overall quality factor for the second and third models deposited in the Protein Model Portal was computed to be 69.132 (Supplementary Fig.3B) and 67.508 (Supplementary Fig.3C), respectively, which is quite poor in comparison to the quality score of 78.778 (Supplementary Fig.3A) computed for the current model of kir6.2 (PM0079770).
Again, superimposition of the second model (blue color) available in Protein Model Portal and the present model of kir6.2 (red color) also gives a clear-cut idea of the quality of the model presented here (Supplementary Fig.4).
The quality of the backbone folding pattern of the present model (Supplementary Fig.5A) in comparison to the third model available in Protein Model Portal (Supplementary Fig.5B) is also good.
From the above, it is found that the model predicted here for kir6.2 is good in comparison to all three existing models available in Protein Model Portal.
Title: Structural comparison of wild and mutant models of kir6.2

Similarly, by comparison of the wild and mutant models of kir6.2, the quality is confirmed as good for both predicted models of kir6.2, because the values of the computed z-score of the computed models of kir6.2 are within the range of z-scores of the groups of structures from different sources, like X-ray and nuclear magnetic resonance (Supplementary Fig.6).
The Ramachandran plot statistics (Supplementary Fig.7) show 93.8% of residues in the most favored region in the case of the wild kir6.2 model, whereas in the mutant model, 92.8% of residues are found in the same region.
At the same time, no residues were found in the disallowed region in both the wild and mutant models of kir6.2, which signified the good quality of the backbone folding pattern in both models.
The non-bonded interactions between different atoms were plotted residue-wise against error function to calculate the overall quality factor of both models in Errat2, which justified the improvement in quality of kir6.2 mutant model (Supplementary Fig.8).
The comparison of secondary structural elements of both wild-type and mutant kir6.2 (Supplementary Table 1) reveals that there is an increment in disordered regions in the mutant model, but there is no major structural deviation due to the mutation introduced at the sequence level (Fig.1).
This is also supported by the root mean square deviation of the main chain at the tertiary structural level, which is computed as 0.854 by superimposition of the structural models of both wild and mutant kir6.2 (Fig.2).
Title: Molecular docking analysis of kir6.2 model with phytochemicals

The results of molecular docking between the wild-type kir6.2 model and all four phytochemicals (Supplementary Fig.1) separately for 100 runs (blind docking) and 10 runs (ATP-binding site docking) revealed that phytochemicals bind favorably in the same pocket of the wild kir6.2 model.
The same observation was found in the case of the docking of the mutant kir6.2 model with the same phytochemicals (Supplementary Fig.1), which supports the prediction of the pocket as an active binding region for phytochemicals.
The best docked poses found in energetically favorable binding conditions with phytochemicals are reported (Supplementary Table 2).
The comparative data of interacting residues within 4 Å in both the wild and mutant kir6.2 predicted models are reported in Table 1.
The hydrophobic amino acids Ala-178, Leu-181, Phe-183, and Ala-300 are found within the pocket, providing stability towards the binding of phytochemicals with the kir6.2 models.
The interacting residues of the mutant kir6.2 model are deciphered in Fig.3, along with hydrogen bond interactions within 3.5 Å of distance (Table 2).
As per the information in Table 2, the amino acids Phe-183 and Arg-301 have common involvement in hydrogen bonding in the mutant model of the kir6.2 protein with curcumin, genistein, piperine, and pterostilbene, suggesting strong binding affinity in the predicted binding pocket.
It is a well-established fact that the main cause of type 1 diabetes is the autoimmune destruction of β-cells in the pancreas [1].
Reports demonstrate that kir6.2 is an ATP/ADP ratio-sensitive protein that plays a vital role in operating the K+ channel and simultaneously controls Ca+2 ion channel function, which in turn triggers the exocytosis of insulin.
But, due to mutations [10] in certain amino acids (H46Y, R50Q, Q52R, G53D, V59M, L164P, C166T, K170T, R201H, and Y330S), the whole mechanism is perturbed, blocking the exocytosis of insulin, which might be a cause of type-1 diabetes.
Again, due to the unavailability of the crystal structure of kir6.2, molecular targeting with drugs is difficult.
But, a rational design of in silico structure prediction approaches comes in handy under such circumstances to study this further.
From the structural inspection of both wild-type and mutant kir6.2 models, it is concluded that mutation has no major structural changes or deviation in the structure quality of the model.
In recent years, due to the adverse effect of synthetic drugs, phytochemicals have drawn substantial attention as alternative medications.
The literature reports curcumin, genistein, piperine, and pterostilbene to have strong inhibitory effects against type 1 diabetes.
Hence, all four of these phytochemicals were docked with predicted models of kir6.2 in the ATP-binding pocket [12] to check the inhibitory effect after mutation.
The results of docking suggested that all phytochemicals bind at high affinity with both models, and the common interacting residues Ala-178, Leu-181, Phe-183, and Ala-300 were found in the same pocket, even after mutation.
Amino acid residues, like Phe-183 and Arg-301, have also been observed as participating in hydrogen bonding within the binding pocket of mutant kir6.2 in the case of docking with all four inhibitors, supporting the fact that mutation might not affect the binding affinity of kir6.2 protein with phytochemicals.
The current investigation concluded that phytochemicals, like curcumin, genistein, piperine, and pterostilbene, have strong inhibitory effects on kir6.2 protein.
In addition, the study reports that these four phytochemicals namely, curcumin, genistein, piperine, and pterostilbene are effective in both normal and mutant conditions of kir6.2, suggesting future implications for type 1 diabetes mellitus treatment.
Comparison of predicted secondary structures in wild-type (kir6.2) and mutant-type (mu_kir6.2) model.
Alignment done for amino acids 33-358 of kir6.2 models.
Secondary structural element (DSC) helix is represented by red cylinder, and beta sheets are represented by blue arrows in the picture.
Superimposition of predicted tertiary models of kir6.2 wild-type (blue color) and mutant (red color).
The mutation position, along with amino acid residues, is mentioned in the picture.
Atomic interaction shown between phytochemicals curcumin (A), genistein (B), piperine (C), and pterostilbene (D) with mutant kir6.2 model within the binding pocket.
Hydrogen bond interactions within 3.5 Å are presented in dotted lines in the pictures.
Title: Hydrophobic amino acid residues of kir6.2 models (wild-type and mutant) participating in binding interaction within 4 Å within the binding pocket with phytochemicals

Title: Amino acid residues participating in hydrogen bond interaction in mutant-type kir6.2 model with all four phytochemicals
Title: MAP: Mutation Arranger for Defining Phenotype-Related Single-Nucleotide Variant

Next-generation sequencing (NGS) is widely used to identify the causative mutations underlying diverse human diseases, including cancers, which can be useful for discovering the diagnostic and therapeutic targets.
Currently, a number of single-nucleotide variant (SNV)-calling algorithms are available; however, there is no tool for visualizing the recurrent and phenotype-specific mutations for general researchers.
In this study, in order to support defining the recurrent mutations or phenotype-specific mutations from NGS data of a group of cancers with diverse phenotypes, we aimed to develop a user-friendly tool, named mutation arranger for defining phenotype-related SNV (MAP).
MAP is a user-friendly program with multiple functions that supports the determination of recurrent or phenotype-specific mutations and provides graphic illustration images to the users.
Its operation environment, the Microsoft Windows environment, enables more researchers who cannot operate Linux to define clinically meaningful mutations with NGS data from cancer cohorts.
It is widely accepted that mutation is the main causal factor for diverse tumorigenesis [1, 2].
The recent development of next-generation sequencing (NGS) technologies has revolutionized the speed and throughput of DNA sequencing, which facilitates the discovery of new driver mutations [3].
For example, the ARID1A gene is mutated in 83% of gastric cancers with microsatellite instability and the SF3B1 gene is somatically mutated in 9.7% of chronic lymphocytic leukemia patients [4, 5].
In colorectal cancers, although the majority of recurrent mutations have been previously known, some novel mutations, such as the mutations in the SOX9 and FAM123B genes, were also identified by NGS technology, which may have implications for colorectal tumorigenesis [6].
In melanoma, approximately 40% was found to have causal mutations affecting B-Raf function [7].
Along with the well-known driver mutations, the novel mutations identified by NGS will be useful for predicting the prognosis and molecular characterization of cancers [8, 9].
Recently, a number of mutation calling tools, such as VarScan [10], Genome Analysis ToolKit [11], and MuTect [12], have been developed to detect single-nucleotide variants (SNVs) or indels from NGS data.
However, SNV calling from a single cancer case is not the final step for defining clinically meaningful mutations.
To define the pathogenic SNVs in human cancers, a commonly used approach is to identify the phenotype-associated recurrent mutations in the study group.
Due to the data size burden of NGS output, profiling the mutations associated with a certain phenotype or prognosis in a study group by a researcher who does not have a bioinformatics background or facilities is very difficult.
Currently, there is no user-friendly tool supporting methods for the detection of recurrent or phenotype-specific mutations.
In this study, in order to support the definition of recurrent mutations or phenotype-specific mutations from NGS data of a group of cancers with diverse phenotypes, we aimed to develop a user-friendly tool, named mutation arranger for defining phenotype-related SNV (MAP).
MAP software was designed as a standalone program, compatible in Microsoft Windows environments with a user-friendly graphic interface, and compiled codes of MAP can be easily installed.
Both Variant Call Format [13] and its annotation files, such as ANNOVAR output [14], can be used as input files for MAP.
When a user loads the input file, called Sample Descriptor file (GSF), MAP software displays the sample set information and filter options (Fig.1).
Once data are loaded, the user can filter the data using the &#39;Analysis Options&#39; tab in MAP software based on annotation information.
By using the sample set and analysis options, MAP software generates the summary metrics for multiple samples.
Summary metrics include the mutation status for each sample, total mutation frequency, p-values for differences between phenotype groups based on clinical information, genomic position, reference/observed sequence, and additional information, such as read depth by default.
MAP software also displays more detailed information in summary metrics based on the user-selected &#39;Analysis Options.&#39;
In the &#39;Analysis Result&#39; tab, MAP software provides the frequency- or p-value-based filter function.
This functionality supports detection of the recurrent mutations in study subjects and identification of the phenotype-specific mutation that occurs significantly in a certain phenotypic subclass.
Then, MAP software represents the user-selected mutations graphically in the &#39;Visualization&#39; tab.
To sort the variants by genomic position, p-value, or frequency, users can use the sort function in the &#39;Analysis Result&#39; tab.
Details of the program download, installation, and analysis procedures are available in the user manual.
The major steps of MAP are as follows.
Title: - Generating mutation summary metrics in study subjects

Title: - Determination of recurrent mutations by frequency

Title: - Determination of phenotype-specific mutations by association test

Title: - Graphical illustration of user-selected mutations

Mutation calling files from Genome Analysis ToolKit [11] can be used directly as input files for &#39;Sample Descriptor.&#39;
Alternatively, manually prepared standard Variant Call Format [13] is also available.
Annotation information files, such as ANNOVAR outputs, can be used as input files for Sample Descriptor.
GSF is a comma-separated values text file that describes the input files, such as GATK output files and ANNOVAR output files, and phenotype information.
MAP imports this Sample Descriptor file as the input file.
Details are available in the user manual.
When users load the input file, MAP software displays the sample set information and filter options in the &#39;Source&#39; tab.
If any file is not prepared properly, MAP software informs the user in the &#39;Sample Set Files&#39; tab.
After data are loaded, the user can filter the data using the &#39;Analysis Option&#39; tab based on annotation information.
MAP software supports eight filter options for exonic function.
By using the mutation calls of each sample, MAP generates the summary metrics (Fig.2A).
Based on user-selected filters in the &#39;Analysis Options&#39; tab, summary metrics represent additional annotation information, such as gene name, function, known variant, and amino acid change.
The additional annotations can be hidden using the SNP filter option in the &#39;Analysis Option&#39; tab.
These mutation metrics are exported into a comma-separated values text file for further investigation.
In the &#39;Analysis Result&#39; tab, MAP software defines the recurrent mutations in study subjects by frequency.
The frequency threshold can be determined arbitrarily by the user in the &#39;Frequency Filter&#39; function.
MAP also defines the phenotype-specific mutations using the mutation metrics and phenotype information.
Phenotype-specific mutations that occur more frequently in one of two clinical conditions (e.g., drug-responder vs. non-responder) are determined by chi-square test under the user-defined significance level.
The graphic user interface is implemented in the software for users to easily handle large-scale data or access the analysis result (Fig.2B and 2C).
The recurrent or phenotype-specific mutations can also be demonstrated visually in heat-map style.
Furthermore, MAP software can support the visualization for additional information at a time.
For example, the left and right of the heat-map represent the sample name and mutational spectrum, respectively.
The bottom of the heat-map represents the gene name by color for mutational spectrum.
Frequency or p-value plots are represented on top of the heat-map (Fig.2B and 2C).
This graphical result can be exported into an image file for further investigation.
MAP is a user-friendly program with multiple functions that supports the determination of recurrent or phenotype-specific mutations and provides graphic illustration images to the users.
Its operation environment, Microsoft Windows, enables more researchers who cannot operate Linux to define clinically meaningful mutations with NGS data from cancer cohorts.
Screenshot of source tab in MAP software.
Once data are loaded, the Sample Set Files panel and Analysis Options panel are displayed in the Source tab.
In the Sample Set Files tab, MAP software displays the input file status and phenotype information.
MAP software supports the filter function in the Analysis Options tab based on annotation information.
Examples of the Sample Set Files tab and Analysis Options tab are shown in red and blue boxes, respectively.
MAP, mutation arranger for defining phenotype-related single-nucleotide variant.
Examples of MAP software output.
MAP software generates the summary metrics and graphical illustration of user-selected mutations.
(A) Example of summary metrics.
Summary metrics include mutation status for each sample, total mutation frequency, p-values for differences between phenotype groups based on clinical information, genomic position, reference/observed sequence, and additional information, such as read depth by default.
(B, C) Examples of frequency (B)- and p-value (C)-based visualization, respectively.
The left and right of the heat-map represent the sample name and mutational spectrum, respectively.
The bottom of the heat-map represents the gene name by color for the mutational spectrum.
Frequency or p-value plots are represented on top of the heat-map.
MAP, mutation arranger for defining phenotype-related single-nucleotide variant.
Title: Editor&#39;s Introduction to This Issue

Understanding a pathogen&#39;s genome is important for controlling infectious disease, in addition to determining its scientific impact.
Acinetobacter baumannii has emerged as an important opportunistic pathogen that causes nosocomial infections, such as pneumonia, bacteremia, and sepsis.
Given that iron is one of the essential nutrients for living organisms, including pathogens, the survival mechanism of Acinetobacter baumannii in iron-limited environments is important.
Regarding this issue, Dr. Man Hwan Oh&#39;s group of Dankook University reviewed the gene clusters involved in the biosynthesis and transport of acinetobactin, an iron-chelating molecule involved in competition with host cells for iron, in Acinetobacter baumannii.
There are two articles on genome data management and in silico modeling.
One study presents an integrated geographic information system (GIS)-ontology application for handling microbial genome data, and the other one validates the biological closeness of the predicted in silico model to infer structure-function relationships and the molecular dynamics of substrate binding in human fatty acid.
Title: Comparative Modeling and Molecular Dynamics Simulation of Substrate Binding in Human Fatty Acid Synthase: Enoyl Reductase and β-Ketoacyl Reductase Catalytic Domains

Fatty acid synthase (FASN, EC 2.3.1.85), is a multi-enzyme dimer complex that plays a critical role in lipogenesis.
This lipogenic enzyme has gained importance beyond its physiological role due to its implications in several clinical conditions-cancers, obesity, and diabetes.
This has made FASN an attractive pharmacological target.
Here, we have attempted to predict the theoretical models for the human enoyl reductase (ER) and β-ketoacyl reductase (KR) domains based on the porcine FASN crystal structure, which was the structurally closest template available at the time of this study.
Comparative modeling methods were used for studying the structure-function relationships.
Different validation studies revealed the predicted structures to be highly plausible.
The respective substrates of ER and KR domains-namely, trans-butenoyl and β-ketobutyryl-were computationally docked into active sites using Glide in order to understand the probable binding mode.
The molecular dynamics simulations of the apo and holo states of ER and KR showed stable backbone root mean square deviation trajectories with minimal deviation.
Ramachandran plot analysis showed 96.0% of residues in the most favorable region for ER and 90.3% for the KR domain, respectively.
Thus, the predicted models yielded significant insights into the substrate binding modes of the ER and KR catalytic domains and will aid in identifying novel chemical inhibitors of human FASN that target these domains.
Fatty acid synthase (FASN, E.C.
2.3.1.85) is a multi-enzyme complex that synthesizes endogenous fatty acids.
Seven cycles of FASN-catalyzed reactions result in the conversion of acetyl-CoA and malonyl-CoA into 16-carbon palmitate [1, 2] by the successive addition of 2-carbon units per synthesis cycle.
Acyl carrier protein (ACP) holds the growing chain covalently and transports it sequentially to the active sites of β-ketoacyl synthase (KS), acetyl transacylase, malonyl transacylase, β-hydroxy acyl dehydratase, enoyl reductase (ER), β-ketoacyl reductase (KR), and finally thioesterase (TE), from which palmitate is cleaved by catalytic actions.
Two major forms of FASN are known: FASN type I (FASNI) and type II.
FASNI is a multimeric multi-enzyme complex involved in the synthesis of palmitate in an integrative manner on a single polypeptide chain (α) or on two different polypeptides (α and β).
FASNI is further subdivided into two subtypes.
Animal FASN, which is a homodimeric protein (α2), is arranged head to tail [3], and the microbial subgroup FASN, which is an oligomer of higher order, consists of certain bacterial FASN type I α6 hexamers and of fungal α6β6 dodecomers [4].
Type II FASN consists of independent proteins that catalyze individual reactions and is present in bacteria, plants, and eukaryotic mitochondria [5].
Besides the important biochemical role that FASN plays in lipid biogenesis, this complex enzyme has been implicated in several pathological conditions.
In comparison with normal cells, the overexpression of FASN has been reported in several types of cancers, including prostrate, breast, ovarian, and colon cancers [6, 7, 8], which strongly suggests the involvement of FASN in cancer progression.
Similarly, FASN overexpression in ocular cancer and retinoblastoma and its potential as an anti-cancer target using enzyme inhibitors have been reported from our lab [9].
FASN is also indicated to have a potential role in the clinical management of microbial infections, obesity, and diabetes.
This enzyme is a potential target to control bacterial infections by using agents, such as cerulenin and triclosan [10], and an anti-tumor target against different cancers [11] and is also targeted for the treatment of obesity [12, 13] and diabetes [14, 15].
It is therefore important to understand the structure of this multi-enzyme protein in order to suitably target its various catalytic domains.
The resulting ligands could be potential therapeutic leads.
In this study, we have attempted to predict the 3D structure of the enoyl reductase (ER) and KR domains of human FASN in unbound form by comparative modeling.
At the time of this model development, the structurally closest template was the reported crystal structure of porcine FASN (PDB ID: 2VZ8, 3.2 Å).
This mammalian FASN template revealed a complex architecture, covering five catalytic domains and also the inter-connecting linkers [16].
The crystal structures of some of the other individual domains of human FASN were available in ligand-bound forms-MAT (PDB ID: 2JFK), ACP (PDB ID: 2CG5), and TE (PDB ID: 2PX6, 3TJM) and in apo forms-KS (PDB ID: 3HHD) and MAT (PDB ID: 2JFD).
Most recently, crystal structures of human FASN domains-ligand-bound ER (PDB ID: 4W9N) and apo ER (PDB ID: 4W82)-and KR (PDB ID: 4PIV) have been reported.
In order to validate the biological closeness of the predicted in silico model and also to infer the structure-function relationships, the predicted structures here were computationally docked with their respective physiological substrates.
Further, these structures were also subjected to molecular dynamics (MD) simulation in apo and holo conditions to understand the conformational dynamics and stability.
The amino acid sequences of the ER and KR domains of human FASN, were retrieved from Uniprot Kb/Swissprot server (uniprot ID: P49327).
The sequences of the ER and KR domains were 229 and 255 amino acids in length, respectively.
The retrieved sequences were BLAST-analyzed against PDB towards identifying suitable templates for homology modeling.
From the available crystal structures deposited in PDB during the time of this study (early 2014), the crystal structure of mammalian (porcine, Sus scrofa) fatty acid synthase (chain A, PDB ID: 2VZ8) showed the closest homology and was therefore used as the template for modeling the unbound ER and KR domains.
Title: Homology modeling of ER and KR domains

The pairwise sequence alignments between the template and the targets (human ER and KR) were built using MODELLER 9v7 [17, 18].
Further, the aligned target-template file was used to generate homology models.
A total of 100 initial models were generated for both the ER and KR domains.
The qualities of these models were ranked based on discrete optimized protein energy (DOPE) score generated by MODELER 9v7 [17].
The models showing the lowest DOPE score was chosen as the best model.
Further, these best models of the ER and KR domains were processed using the &#39;Protein Preparation Wizard&#39; module of Schrodinger Suite (Schrödinger, LLC, New York, NY, USA) to correct the structural defects, to add and optimize hydrogen atoms, to assign bond orders, and also to selectively assign tautomerization and ionization states.
Title: Model validation

The validations of structural geometric properties, like backbone conformation, and of the compatibility of residue interactions were performed using the Structural Analysis and Verification Server (SAVES; http://nihserver.mbi.ucla.edu/SAVES).
The Ramachandran plot statistics determined by the PROCHECK program (analyzing the phi [Φ] and psi [ψ] torsion angles) were used to verify the backbone conformation and overall stereochemical quality of the protein structures.
The non-bonded atomic interactions of refined structures were analyzed using the ERRAT program [19].
Based on the inferences from the Ramachandran plot and ERRAT predictions, further refinement of side chain and backbone atoms was performed using MODREFINER [20] and was subsequently loop-refined using MODELLER 9v7 scripts.
Additionally, these models were again refined using the WHATIF program [21] to remove atomic clashes (bumps) by rotating the side chain torsion angles (chi1, chi 2, and chi 3).
Finally, the quality and plausibility of the predicted 3D structures were validated using ProQ, a model quality assessment program developed by Wallner and Elofsson [22].
Title: MD simulation of homology models

MD simulations for the modeled proteins were carried out using the Desmond program, an explicit solvent MD package (version 3.1, Desmond Molecular Dynamics System; D. E. Shaw Research, New York, NY, USA and version 3.1, Maestro-Desmond Interoperability Tools; Schrödinger) with inbuilt optimized potentials for liquid simulation (OPLS 2005) force field [23].
The correctness of the chemical structures provided to Desmond was ensured using Protein Preparation Wizard (macro models), Ligprep (chemical molecules), and Epik (ligand protonation states).
The system was set up for simulation using a predefined water model (simple point charge, SPC) as solvent in a cubic box with periodic boundary conditions specifying the shape and size of box as 10 Å × 10 Å × 10 Å distance.
The desirable electrically neutral system for simulation was built with 0.15 M NaCl (physiological concentration of monovalent ions) in 10 Å buffer using the system-built option.
The relaxation of the system was achieved by implementing Steepest Descent and the limited-memory Broyden-Fletcher-Goldfarb-Shanno algorithms in a hybrid manner.
The simulation was performed under NPT ensemble for 5 ns implementing the Berendsen thermostat and barostat methods.
A constant temperature of 300 K was maintained throughout the simulation using the Nose-Hoover thermostat algorithm [24] and Martyna-Tobias-Klein Barostat algorithm [25] to maintain 1 atm of pressure, respectively.
The short-range coulombic interactions were analyzed using a cut-off value of 9.0 Å using the short-range method.
The smooth particle mesh ewald method [26] was used for handling long-range coulombic interactions.
Tolerance value usually affects the accuracy of long-range interactions; therefore, a smaller value of 1e-9 is set for accurate computations implemented by the SHAKE algorithm.
The final production run was carried out for 5 ns, and the trajectory sampling was done at an interval of 1.0 ps [23].
Title: Binding pocket prediction

Site Map 2.6 (Schrödinger, LLC) was used to identify the active sites on the predicted models.
Various countermaps were also generated to distinguish the hydrophobic and hydrophilic regions on the active site regions.
Finally, the best binding pockets were ranked based on the Sitemap score.
Title: Substrate docking and postdocking simulation

The substrates for the ER and KR domains-namely, the trans butenoyl and β-ketobutyryl groups-were sketched using the 2D draw tool (version 9.3, Maestro; Schrödinger, LLC).
The optimal 3D structures of the sketched substrates were generated using Ligprep Module, which fixes ionization states, tautomers, and other stereochemical errors.
The correct chiral forms of the optimized substrates were used for docking with the corresponding FASN domains.
Receptor grid files for glide docking were generated to cover the volume of the predicted active sites.
Here, van der Waals radius is scaled to 1.0 with a partial cut-off of 0.25 to soften the potential for non-polar parts of a receptor, where other atoms are free of scaling.
The receptor grid file and the prepared substrate were docked using Glide standard precision, where the ligand sampling was set to be flexible, ensuring the sample ring conformation and nitrogen inversions.
Ligands were set to select only less than 300 atoms and less than 50 rotatable bonds with a van der Waals scaling factor of 0.8 with a partial cut-off of 0.15.
Out of the 1,000 poses generated per docking run, 10 energetically favorable poses per ligand were selected.
Glide score, an empirical docking scoring function that implements the OPLS 2005 force field, was used to infer the affinity and binding mode of the substrate.
The best docked substrate conformation for the ER and KR (holo) domains was subjected to MD simulation for 5 ns, similar to that of the methods discussed for MD simulations of the apo form, except that maximum iterations of 2,000 steps were applied on solute heavy atoms alone with a convergence threshold as 1.0 kcal/mol/Å.
Title: Target-template alignment

The amino acid sequences of the human FASN ER and KR domains were retrieved from Uniprot (P49327).
Comparative modeling method was implemented to predict the structures of the ER and KR domains.
The structural templates for modeling the query sequences were searched against PDB using BLASTP.
Many of the hits obtained were similar to the query sequences, mainly representing the reductase families.
The best templates were selected in accordance with the optimal pairwise alignment, sequence coverage, and sequence conservation.
Accordingly, mammalian (porcine, Sus scrofa) fatty acid synthase (chain A) (PDB ID: 2VZ8, 3.2 Å resolution, region 1635 to 1860, 82% similarity) was chosen for ER.
Similarly, the template chosen for the KR domain (porcine, Sus scrofa) showed significant homology (83% of sequence identity matching to region 1866 to 2117 of 2VZ8) and was used for modeling the KR domain.
Title: Homology modeling

We had attempted to model the protein structure of human FASN ER domain earlier for the purpose of understanding its bonding interactions with the known inhibitor triclosan [27].
There was a close correlation between the inhibition constant (Ki) obtained from the in silico model and the experimentally obtained biochemical IC50.
However, in the present study, we implemented advanced algorithms and MD simulations to provide a more refined and highly plausible structure, with more emphasis on the catalytic active site.
As discussed above, the crystal structure of Sus scrofa FASN (PDB ID 2VZ8) was used as the template to model the 3D structures of the human ER and KR domains.
The pairwise sequence alignment between the template and targets were built using MODELLER 9v7.
The best models for the human ER and KR domains were chosen based on the lowest DOPE score.
Further, these models were found to have similar secondary structures (folds and loops) when compared with that of the template.
The backbone alignment score was generated using maestro (version 9.3, Maestro, Schrödinger, LLC), superimposing the respective templates.
The alignment scores were found to be 0.27 for modeled human ER and 0.01 for modeled KR domain.
In the postmodel development, we also compared our predicted structures with the very recently reported crystal structures of the human ER (4W82) and KR domains (4PIV) [28, 29], wherein the alignment scores were found to be 0.28 and 0.048 for modeled ER and KR, respectively.
Generally, the lower alignment score (less than 0.7-0.8) is indicative of a good alignment and higher structurally identity, according to Yang and Honig [30].
Hence, taken together, all of these findings imply the predictive accuracy of the in silico methods implemented here.
The predicted structure of the human ER domain showed a Rossmann fold at region 1651 to 1794, which harbors a nucleotide binding cavity (1671 to 1688) and also overlapping substrate binding cavities at 1650-1653 and 1795-1863, as proposed by Maier et al.[16].
Similarly, the predicted structure of the KR domain showed a binding cavity at the region covering 1864 to 2117.
The active cavity residue Lys1995 and the nucleotide binding residues 1886-1901 [16] were found to harbor the predicted binding cavity.
The overall topology and secondary structure information of the ER and KR domains are shown in Figs.1 and 2.
Title: Secondary structure analysis

PDBsum server, which uses Gail Hutchinson&#39;s PROMOTIF program to compute the secondary structure motif information of the models generated (http://www.ebi.ac.uk/pdbsum/), was used to predict the topology of the predicted structures.
The overall secondary structure elements of the ER domain include 18.8% of residues forming strands, 44.5% forming alpha helices, 0.9% forming 3-10 helices, and 35.5% of residues forming other structures.
Similarly, secondary structure elements of the KR domain include 16.1% of residues forming strands, 35.7% forming alpha helices, 0.0% forming 3-10 helices, and 48.2% of residues forming other structures.
The ER model has 2 beta-sheets, 5 beta-alpha-beta motifs, 1 beta bulge, 8 strands, 12 helices, 12 helix-helix interactions, and 10 beta turns.
Similarly, the KR domain has 2 beta-sheets, 4 beta-alpha-beta motifs, 1 beta bulge, 9 strands, 7 helices, 5 helix-helix interactions, 19 beta turns, and 2 gamma turns.
Title: Model validation

The initial models were subjected to different structure evaluation tools for understanding the geometries, backbone configuration, dihedrals, and residue-residue interactions.
The Ramachandran plot for the ER domain showed 0.5% residues in a disallowed region.
In the case of the KR domain, 0.0% of the residues were in disallowed regions.
Hence, these models were further refined using the WHAT IF program to remove the atomic bumps and subsequently loop-refined using MODELLER 9v7 scripts and the MODREFINER algorithm.
The final refined structures of both domains showed no residues in the disallowed region of the Ramachandran plot (Figs.3 and 4).
The overall stereochemical parameters for the modeled proteins were measured using G-factor and ERRAT calculations by PROCHECK.
G-factor is a measure of the proper dihedrals and covalent bond orders, and it is expressed overall as log-odd score.
G-factor scores for optimal structures range from 0.0 to 0.1 with increasing order of confidence.
A negative G-factor score indicates improper conformation of the residues, while higher positive scores indicate proper conformation.
In the case of the predicted model of the ER domain, the overall log-odd score was found to be 0.30, which indicates a high plausibility of the structure with negligible improper conformations.
Similarly, the KR model also showed an overall G-factor score of 0.22, suggestive of a high plausibility of the structure.
Moreover, the non-bonded atomic interactions of the models were analyzed using the ERRAT tool, wherein the overall quality score for the predicted structures of the ER and KR domains were found to be 95.92 and 80.85, respectively.
The ERRAT scores for both models were above 50 and are considered a standard for good models.
To validate this further, the modeled structures were also assessed for their quality using the ProQ server.
ProQ assessment for the ER model predicted the LGscore to be 4.362 (&gt;4.0, extremely good model) and the MaxSub score to be 0.416 (&gt;0.1, fairly good model).
Similarly, for the KR domain, the LGscore was 3.509 (&gt;2.5, very good model) and the MaxSub score was 0.246 (&gt;0.1, fairly good model), suggesting higher plausibility of the models.
Title: MD simulation

MD simulation was carried out for the predicted models to understand the stability and conformational changes of the modeled proteins in holo and apo forms.
The simulation was carried out in a water (SPC-molecule)-solvated system with optimal physiological conditions, wherein the temperature and pressure were maintained at 300 K and 1 atm for both the ER and KR domains.
The computed total energy and potential energy remained stable from start of the dynamics simulation until the course of the the 5-ns simulation in the case of both proteins.
The ER domain had a total energy of -82,181.468 kJ/mol and a potential energy of -101,788.978 kJ/mol.
Similarly, KR also had stable distribution with a total energy of -141,285.655 kJ/mol and potential energy of -174,168.487 kJ/mol.
The event trajectory was observed to remain stable throughout the simulation process.
Further, the stability of the modeled proteins was verified by plotting the root mean square deviation (RMSD) graph for backbone atoms during the production run.
The RMSD for the ER domain remained stable for the 5-ns time frame with minimal deviation until the end of 5 ns (Fig.5A).
Similarly, the RMSD plot for the KR model also suggested a stable trajectory during the production run (Fig.6A).
Moreover, the radius of gyration was also calculated for each model during the simulation process and was found to be within the range of 1 Å, indicating the compactness of the models predicted (Figs.5B and 6B).
Hence, these validation studies summarize the higher plausibility of the predicted models.
Title: Active site prediction

Site Map 2.6 (Schrödinger, LLC) was used to predict the active sites in the modeled proteins.
The presently modeled human FASN ER domain comprises two subdomains forming the Rossmann fold and substrate binding fold.
According to Maier et al.
[16], on the porcine ER structure, NADP+ is likely to be found in the junction of the subdomains, and Lys1771 and Asp1797 may be donor residues for substrate protonation.
This orientation favors the entry of substrate through a tunnel to the nicotinamide ring and its exit through an opening at the posterior part of the ER domain [16].
This fold orientation was observed in our predicted model of ER (Fig.7).
In the case of the KR domain, the predicted active site had Lys1995 as a central key substrate-interacting residue surrounded by Asn2038, Tyr2034, and Ser2021residues, which also coincides with the observations of Maier et al.
[16] on porcine KR.
Title: Docking and post-docking simulation

Normally, in FASN-catalyzed lipid biosynthesis, seven cycles of two carbon additions are essential for synthesizing the 16-carbon palmitate.
The respective initial physiological substrates for the KR and ER domains-namely, β-ketobutyryl and trans butenoyl groups-were built using 2D draw.
Further, these structures were optimized using LigPrep 2.6 (Schrödinger, LLC) and docked to the respective catalytic active sites using Glide 5.8 (Schrödinger, LLC) (Figs.8 and 9).
Reports suggest that in these β-carbon processing enzymes, KR and ER have been studied using derivatives of S-β-ketobutyryl and S-crotonyl (S-acyl-N-acetyl cysteamine thioester derivatives) as model substrates [3, 31].
The Glide docking score for the ER trans butenoyl complex was found to be -2.283 kcal/mol with a Glide energy of -11.638 kcal/mol.
Similarly, the Glide docking score for the KR domain was found to be -5.25738 kcal/mol with a Glide energy of -16.827 kcal/mol.
Finally, both docked complexes were validated for the stability of complex formation, implementing a MD simulation using DESMOND.
The simulations for both holoenzyme complexes were carried out for 5 ns.
The RMSD plot for all atoms of each holoenzyme during the production run was analyzed.
The plots showed no significant increase in deviation until the completion of 5 ns (Figs.5A and 6A).
Similarly, the radius of gyration was also calculated for the complex structures, which inferred the compactness of the docked complexes, as it was within the range of 1 Å in the case of both complexes.
Hence, all of these findings strongly suggest the highly stable complex formation of the ER and KR domains with their respective substrates (Figs.5B and 6B).
The root mean square fluctuation plot of all residues on pre-substrate binding and post-substrate binding (Fig.10A and 10B) clearly indicates the decrease in overall flexibility of the residues in the holo condition of both the ER and KR domains.
Based on the in silico approaches employed in this study, the plausible stable homology models of the ER and KR domains of FASN protein are suggested, which are also found to be in substantial agreement with the reported crystal structures.
Further, the favorable outcomes of substrate binding with the ER and KR domains validate the predicted structures and their potential for screening anti-FASN ligands.
The insights from this in silico study would be very helpful in pharmacologic lead generation, as FASN inhibition is favorably implicated in the treatment of cancers, diabetes, obesity, and infections.
(A) Overall topology of the enoyl reductase (ER) domain from N to C terminus.
(B) The 3D structure of the ER domain; helices are shown in red, beta sheets in cyan, and turns in green.
(A) Overall topology of the β-ketoacyl reductase (KR) domain from N to C terminus.
(B) The 3D structure of the KR domain, with helices shown in red, beta sheets in cyan, and turns in green.
Ramachandran plots for the modeled enoyl reductase domain (A, before refinement; B, after refinement).
Ramachandran plots for the modeled β-ketoacyl reductase domain (A, before refinement; B, after refinement).
Time-dependent root mean square deviation (RMSD) (Å) and radius of gyration (Å) for the enoyl reductase model during 5-ns molecular dynamics simulation.
(A) The RMSD trajectories of the backbone atoms for the apo-protein are shown in blue and the holo-protein in red.
(B) The radius of gyration trajectories for the apo-protein are shown in blue and the holo-protein in red.
Time-dependent root mean square deviation (RMSD) (Å) and radius of gyration (Å) for the β-ketoacyl reductase model during 5 ns of molecular dynamics simulation.
(A) The RMSD trajectories of the backbone atoms of the apo-protein are shown in blue and the holo-protein in red.
(B) The radius of gyration trajectories for the apo-protein are shown in blue and the holo-protein in red.
Molecular surface area of the enoyl reductase domain.
(A) Red region shows the Rossmann fold; yellow region shows the nucleotide binding residues, and the blue region shows the two donor residues for protonation of the substrates (Lys1771 and Asp1797).
(B) Depicts the tunnel through which the substrate moves into the next domain.
Molecular docking of enoyl reductase (ER) with its substrate.
(A) Substrate trans butenoyl is docked to the ER domain.
The substrate is shown in ball and stick form, the hydrogen bond is a dotted yellow line, and weak interactions are dotted orange lines.
(B) 2D interaction map of trans butenoyl substrate with the ER domain.
Amino acids are mentioned with 3 letter codes, and arrow indicates hydrogen bonding with Val1680.
Molecular docking of β-ketoacyl reductase (KR) with its substrate.
(A) Substrate β-ketobutyryl is docked to the KR domain.
The substrate is shown in ball and stick form, the hydrogen bond is a dotted yellow line, and weak interactions are dotted orange lines.
(B) 2D interaction map of β-ketobutyryl substrate to the KR domain.
Amino acids are mentioned with 3 letter codes, and arrow indicates hydrogen bonding with Ser2021 and Tyr2034.
Time-dependent root mean square fluctuation (RMSF) plots (Angstrom).
(A) RMSF trajectory plot of enoyl reductase domain showing residue-wise fluctuation in apo (blue) and holo (red) conditions.
(B) RMSF trajectory plot for β-ketoacyl reductase (KR) domain showing residue-wise fluctuation of KR domain in apo (blue) and holo (red) conditions.
Title: Genes Involved in the Biosynthesis and Transport of Acinetobactin in Acinetobacter baumannii

Pathogenic bacteria survive in iron-limited host environments by using several iron acquisition mechanisms.
Acinetobacter baumannii, causing serious infections in compromised patients, produces an iron-chelating molecule, called acinetobactin, which is composed of equimolar quantities of 2,3-dihydroxybenzoic acid (DHBA), L-threonine, and N-hydroxyhistamine, to compete with host cells for iron.
Genes that are involved in the production and transport of acinetobactin are clustered within the genome of A. baumannii.
A recent study showed that entA, located outside of the acinetobactin gene cluster, plays important roles in the biosynthesis of the acinetobactin precursor DHBA and in bacterial pathogenesis.
Therefore, understanding the genes that are associated with the biosynthesis and transport of acinetobactin in the bacterial genome is required.
This review is intended to provide a general overview of the genes in the genome of A. baumannii that are required for acinetobactin biosynthesis and transport.
Iron is one of the essential nutrients for host and pathogenic bacteria, and both require the metal as a cofactor or as a prosthetic group for biologically important proteins that are involved in many basic cellular functions [1].
Most iron in the mammalian host is sequestered and bound to high-affinity iron-binding proteins, such as transferrin, lactoferrin, and ferritin, or complexed to the heme of hemoproteins [2].
Hence, iron is rarely found as free iron in the mammalian host and is one of the major limiting factors of pathogenic bacteria.
Therefore, pathogenic bacteria have evolved sophisticated mechanisms for the acquisition of iron from host tissues because of the limited availability of iron to the bacteria and the inevitable competition with the host [3, 4, 5, 6, 7, 8].
Siderophores are high-affinity iron-chelating compounds secreted by pathogenic bacteria [3].
Siderophores are classified into three major groups-catecholates, hydroxamates, and carboxylates-by the ligands that are used to chelate the ferric iron [7].
The diversity of siderophores may have resulted from evolutionary pressure, causing the bacteria to produce structurally different siderophores that can not be transported by specific transport systems of other bacteria.
For example, Escherichia coli produces a prototype catechol siderophore, which is a cyclic trimer of 2,3-dihydroxybenzoyl-L-serine [5].
In comparison, vibriobactin, produced by Vibrio cholera, is a non-cyclic derivative of 2,3-dihydroxybenzoic acid (DHBA) [9].
Genes that are involved in the biosynthesis and uptake of siderophores are often clustered within the bacterial genome, and their products are closely linked to the virulence of pathogenic bacteria.
Like the pathogenesis of many other bacteria, the pathogenesis of Acinetobacter baumannii, a major opportunistic pathogen that causes serious diseases in compromised patients, including pneumonia, bacteremia, urinary tract infection, sepsis, and meningitis [10, 11], depends primarily on the organism&#39;s ability to uptake and utilize iron [12, 13, 14].
Numerous studies have investigated the response of various A. baumannii isolates in iron-limited conditions and identified several gene clusters that are required for the synthesis and transport of siderophores [15].
Especially, the acinetobactin-mediated iron acquisition system has been the most extensively characterized system in A. baumannii [16, 17].
Acinetobactin, a catechol-hydroxamate siderophore, which is a non-cyclic derivative of DHBA that is linked to threonine and N-hydroxyhistamine, is produced and utilized by three putative systems encoded within the acinetobactin gene cluster of A. baumannii.
Moreover, A. baumannii strains have significant genetic diversity in the numbers and types of these iron uptake and utilization systems [15].
Therefore, these results indicate that genetic diversity in acinetobactin gene clusters could be a major factor in determining the virulence among A. baumannii strains.
In this review, we describe a gene cluster in the genome of A. baumannii that is involved in the biosynthesis and transport of acinetobactin.
Furthermore, we present the genetic diversity of the gene cluster that is responsible for acinetobactin biosynthesis and transport in different A. baumannii strains.
A. baumannii express high-affinity iron acquisition systems, such as siderophores, to survive under iron-limited host environments.
The iron acquisition system that is mediated by the siderophore acinetobactin has been extensively characterized in A. baumannii.
Especially, the gene cluster involved in the biosynthesis and transport of acinetobactin has been found in various clinical isolates of A. baumannii, except A. baumannii SDF [15].
Acinetobactin is synthesized from a DHBA, threonine, and hydroxyhistamine via proteins encoded by genes within the gene cluster.
The mixed-type siderophore, which consists of catecholate and hydroxamate groups, exhibits high affinity for iron [16, 17].
Particularly, it has been found that 18 coding regions in the chromosome of A. baumannii ATCC 19606 are required for acinetobactin biosynthesis and the uptake of ferric-acinetobactin complexes [15, 18].
The acinetobactin gene cluster of A. baumannii ATCC 19606 is illustrated in Fig.1.
The acinetobactin-mediated iron acquisition system is composed of three putative systems encoded within the acinetobactin cluster.
First, the biosynthesis of acinetobactin is accomplished by the proteins encoded by basA, basB, basC, basD, basF, basG, basH, basI, and basJ [14, 15].
Second, the acinetobactin that is produced by A. baumannii is secreted via a siderophore efflux system of the ABC superfamily, consisting of the proteins encoded by barA and barB [18].
Finally, ferric-acinetobactin complexes move into bacterial cells via a receptor for ferric-acinetobactin complexes, consisting of the proteins encoded by bauA, bauB, bauC, bauD, bauE, and bauF [14].
Especially, bauB, bauC, bauD, and bauE play important roles in the translocation of ferric-acinetobactin complexes into bacterial cells [15].
In addition, putative binding sites for ferric uptake regulator, which controls bacterial iron homeostasis, exist in the upstream regions of basJ, entE/basD, basA/bauF, and bauD/basD.
All of these genes that are involved in the biosynthesis and transport of acinetobactin are significantly up-regulated under iron-limited conditions [15].
Particularly, a functional analysis of basD, bauA, and bauD mutants under iron-limited conditions showed that iron acquisition that is mediated by the biosynthesis and transport of acinetobactin is essential for the survival of A. baumannii under iron-limited conditions [13].
Moreover, a recent study by Gaddy et al.
[13] indicated that BasD and BauA play important roles in the persistence of infection by A. baumannii within epithelial cells and their apoptotic death.
The results obtained from a mouse sepsis model also showed that the acinetobactin-mediated iron acquisition system is a major factor in the pathogenesis of A. baumannii, such as the establishment of bacterial infection and cell damage.
Especially, impairment of BasD production resulted in an intermediate virulence phenotype in ex vivo and in vivo infection models in comparison to those of the parental strain and bauA mutant.
These results represent that both acinetobactin synthesis and uptake of iron-acinetobactin complexes are absolutely required for the pathogenesis of A. baumannii.
Penwell et al.
[18] described that the gene cluster that is involved in acinetobactin production and its utilization exists in the chromosomes of different A. baumannii strains.
In addition, the gene cluster is found in A. baumannii AYE, which is a natural entA mutant incapable of producing acinetobactin [18, 19].
However, the cognate clusters in the A. baumannii AB0057 and ACICU strains contain the additional putative genes AB57_2807 and AB57_2818 [20], and ACICU_02575 and ACICU_02586 [21], respectively (Fig.1).
In the case of A. baumannii AB307-294, three additional predicted genes, ABBFA_001054, ABBFA_001053, and ABBFA_001065, are also added in the bas-bau-bar gene cluster (Fig.1) [20].
Although the functions of these additional genes in siderophore production and its utilization are not characterized, these results indicate the possibility that genetic diversity in acinetobactin gene clusters could be closely linked to the virulence of A. baumannii.
Enterobactin that is produced by E. coli is a high-affinity siderophore for iron [5].
In the enterobactin biosynthesis, chorismic acid is converted to DHBA by the products of entA, entB, and entC.
An amide linkage of DHBA to L-serine (Ser) is then catalyzed by the proteins encoded by entD, entE, entF, and entB.
The molecules of the DHBA-Ser that is formed undergo intermolecular cyclization, yielding enterobactin.
A. baumannii also employs basF and basJ within the acinetobactin gene cluster, which are homologs of entC and entB, respectively, to produce mature acinetobactin [18].
However, entA has not been found in the acinetobactin gene cluster.
Recently, it was found that a single functional copy of an entA ortholog is located in anupstream region of the modE-modA-modB-modC gene cluster that is required for the molybdenum transport system in A. baumannii ATCC 19606 (Fig.2) [18].
In particular, a functional analysis based on gene cloning and biochemical tests showed that entA is absolutely required for the production of DHBA, which is used by A. baumannii as a precursor for acinetobactin biosynthesis.
The study also demonstrated that acinetobactin intermediates and DHBA play important roles in the virulence of A. baumannii in ex vivo and in vivo infection models.
Even though entA is found outside of the acinetobactin cluster in A. baumannii strains, the genetic contents surrounding the entA gene are variable among different clinical isolates [18].
For example, the entA gene of A. baumannii ATCC 19606 is next to the coding regions of the mod gene cluster (Fig.2).
However, the entA gene of A. baumannii ATCC 17978 is situated within a large gene cluster that encodes an uncharacterized siderophore-mediated system (Fig.2).
Especially, this cluster, in which mobile DNA elements exist, is flanked by inverted repeats, a feature that can explain its transfer by lateral processes.
Therefore, these results represent that genetic components that are required for iron acquisition via the acinetobactin-mediated system give rise to horizontal gene transfer with complex chromosomal rearrangement processes.
Consequently, the newly added siderophore gene cluster with the entA gene via horizontal gene transfer may cause these two strains, ATCC 19606 and ATCC 17978, to acquire iron via different siderophore-mediated systems from different sources.
The possibility was proven by a genomic analysis of fully sequenced and annotated A. baumannii isolates [15, 19].
Four siderophore gene clusters are newly predicted to be involved in siderophore production and the uptake of iron-siderophore complexes (Table 1).
Cluster 1 is indicated in the genomes of all strains analyzed, with the exception of A. baumannii 8399.
Cluster 2 and cluster 4 are found only in A. baumannii ATCC 17978 and isolate 8399, respectively.
Cluster 5 is present in all genomes analyzed except for that of A. baumannii ATCC 17978.
The possibility is also supported by the result that the iron uptake phenotype of the A. baumannii ATCC 17978 basD mutant was not different from that of the parental 17978 strain when compared under iron-limited conditions.
A. baumannii employs specialized systems that are required for iron acquisition in iron-limited host environments [16].
In this review, we described the genes of A. baumannii that are involved in acinetobactin biosynthesis and transport.
A number of studies have identified the genes that are essential for iron acquisition [13, 15, 16, 17].
Especially, the data presented in this review indicate that A. baumannii strains have genetic diversity in the acinetobacter gene cluster.
Even though the roles of the additional genes in acinetobactin cluster in different.
A. baumannii strains are not characterized, we suggest that the genes could be associated with A. baumannii virulence.
Therefore, functional analysis of the genes in the bacteria is required.
Furthermore, we showed both a single entA functional ortholog and variability in the bacterial genome.
Therefore, we suggest that A. baumannii has continuously evolved the iron acquisition mechanism, which is associated with acinetobactin from different sources by horizontal gene transfer.
Physical map of siderophore gene clusters in Acinetobacter baumannii.
Siderophore gene clusters in different A. baumannii strains.
The gene clusters are based on the database of A. baumannii ATCC 19606, AB0057, ACICU, and AB307-294, which were retrieved separately from GenBank (accession Nos.
ACQB00000000, CP001182-3, CP000863, and CP001172).
Each color indicates the biological functions in the biosynthesis and transport of acinetobactin.
Red arrows represent the genes that were added in the acinetobactin gene cluster.
Genetic map of the gene cluster containing the entA ortholog.
The arrows represent the coding regions of the genes identified in the genomes of Acinetobacter baumannii ATCC 19606 and ATCC 17978.
The names of the genes are shown above each coding region.
Title: Siderophore gene clusters in sequenced Acinetobacter baumannii isolates
Title: An Ontology-Based GIS for Genomic Data Management of Rumen Microbes

During recent years, there has been exponential growth in biological information.
With the emergence of large datasets in biology, life scientists are encountering bottlenecks in handling the biological data.
This study presents an integrated geographic information system (GIS)-ontology application for handling microbial genome data.
The application uses a linear referencing technique as one of the GIS functionalities to represent genes as linear events on the genome layer, where users can define/change the attributes of genes in an event table and interactively see the gene events on a genome layer.
Our application adopted ontology to portray and store genomic data in a semantic framework, which facilitates data-sharing among biology domains, applications, and experts.
The application was developed in two steps.
In the first step, the genome annotated data were prepared and stored in a MySQL database.
The second step involved the connection of the database to both ArcGIS and Protégé as the GIS engine and ontology platform, respectively.
We have designed this application specifically to manage the genome-annotated data of rumen microbial populations.
Such a GIS-ontology application offers powerful capabilities for visualizing, managing, reusing, sharing, and querying genome-related data.
The advances in genomics technologies and other molecular research, together with recent developments in information technologies, have generated a great quantity of data in the molecular biology domain [1].
For example, GenBank, as a comprehensive and well-known database of National Center for Biotechnology Information (NCBI, http://www.ncbi.nlm.nih.gov) online resources, holds nucleotide sequences for more than 280,000 species that are available to the public [2].
Another example of large biological datasets is the 2013 edition of the annual Nucleic Acids Research database issue, which currently lists 1,512 databases, organized into 14 categories and 41 subcategories, available at http://www.oxfordjournals.org/nar/database/a/ [3].
Among organisms, rumen microbes have not been an exception to these debates.
But, much less attention has been paid to integrate and manage the genomic data related to these microorganisms.
Molecular research on ruminal microbes has gained attention as more genomic sequences and related genomic data are becoming available to scientific communities.
The primary focus of this study was to develop an application that allows users to interactively manage the genome-annotated data of rumen microbial consortia.
The rumen environment is the main part of the digestive system and consists of a complex microbial community that makes the rumen microbial ecosystem [4].
This ecosystem is a natural habitat that houses a wide spectrum of bacteria, protozoa, anaerobic fungi, and bacteriophages [5].
By having a symbiotic mutualistic relationship with ruminants, the microbes are capable of converting feed components into a source of energy for the animals (i.e., volatile fatty acids).
Ruminants are able to utilize a variety of feeds owing to an extremely diversified rumen microbial ecosystem [4].
According to previous molecular studies, it has been reported that the population of rumen bacteria accounts for at least 300 to 400 phylotypes [6,7,8].
There is no doubt that progress in biological research could not be achieved without equivalent efforts in informatics research.
Since data management and analysis are facing new challenges in parallel with the age of big data, many research activities have been conducted for handling the biological data.
There are several special issues that reflect the ever increasing importance of data management approaches in biology [9,10,11,12,13].
Geographic information system (GIS) is a computer-based system designed for displaying, storing, manipulating, and analyzing geographical data [14].
As an analytical system, GIS can serve biological systems by establishing biological databases, with the use of powerful representation and display tools, as well as providing query and analysis tools.
An example of a GIS query is finding all hospitals that are equipped with an advanced technology, located in the city center, and with more than 500 beds.
A number of GIS-based approaches have been used to handle biological information.
A typical example in the biology domain is discussed by Kozak et al.
[15], in which they applied GIS technologies and concepts for evolutionary biology.
Their study uses the GIS to track macroevolutionary changes among species and the evolutionary process of geographic variation within species.
In previous studies of GIS applications in genomics, a major effort was made by Dolan et al.[16].
They presented a GIS application, called Genome Spatial Information System (GenoSIS), to manage and represent genomic information.
Ontology is about terminology (domain vocabulary), all essential concepts in a domain, their classification, taxonomy, relations (including all important hierarchies and constraints), and domain axioms [17].
It represents the particular meanings of concepts by specifying the concepts and their relationships.
The use of ontology enables one to structure biological data (genome/gene data) in a machine-understandable form by defining biological concepts, relationships, and instances in a semantic structure.
This paper presents an integrated GIS-ontology application for the management of microbial genome data.
The main rationale behind integrating GIS and ontology is that these two distinct areas of research can complement each other for biological applications.
While the GIS is commonly recognized as a powerful and integrated tool with unique capabilities for storing, manipulating, analyzing, and visualizing geographical information, ontology provides a rich collection of procedures and algorithms to meaningfully share and exchange data.
Our application is implemented by using ArcGIS and Protégé to couple GIS-based biological applications with ontology.
The genome-annotated data of rumen microbial flora were adopted to evaluate the performance of the application.
The system can be extended and employed for handling other types of genomic data.
More specifically, this study adopts the functionality of linear referencing (LR) for representing genome-annotated data in a GIS environment.
Our application uses ontology to semantically organize and reason the data in a semantic and machine-understandable framework.
In this application, we defined a measurement unit for genomes, so that one meter was assigned per nucleotide.
The users have the flexibility to interact with the genome as a layer or with different genomes as a collection of layers.
The ArcGIS platform of the application supports users with interactive genomic maps and implementation of special queries.
Title: Linear referencing

LR is one of the GIS tools representing geographic locations of linear events based on relative positions on linear features [18], where the beginning and end of the events are recorded as measurement values in event tables.
Using LR, attributes are linearly referenced and linked dynamically to layers (e.g., the genome).
This technique can be effectively used to represent genomes and genes as layers and linear events, respectively.
By applying this technique, the genomic data initially stored in event tables can be visualized, displayed, queried, and analyzed on the genome network.
The LR method provides flexibility to split a linear genome into new sets of gene segments (i.e., line events), without having to draw a new line layer for gene events.
As presented in Fig.1, the LR process includes the fields of location and attribute(s) associated with gene events.
As shown, the line gene events are related to the genome layers through the values of the field &#34;Genome-ID.&#34;
The location of line events (genes) on a given genome layer is indicated in the fields &#34;Start&#34; and &#34;End.&#34;
In other words, a line event uses both &#34;Start&#34; and &#34;End&#34; measure values to describe a portion (i.e., a gene) of a genome.
Title: Ontology

From a computer science perspective, ontology is defined as the structure representing a shared, formal, and explicit specification of knowledge in a particular domain (e.g., biology).
Hendler [19] describes ontology as a set of knowledge terms, containing the terminology (domain vocabulary), all essential concepts in the domain, the semantic interconnections, and simple rules of logic and inference for some particular topics.
A typical example of an ontology approach in biology is using standard concepts to classify microorganisms.
There are a large number of identified microbes in nature, so that this amount has long stimulated interest to establish an anthological classification for them.
Gene Ontology (GO) is another example of an ontology application in biology.
GO is intended to formulize biological knowledge in orthogonal hierarchies [20].
It includes controlled vocabularies that are described for gene products with respect to their biological processes, cellular components, and molecular functions [21].
Ontology specification languages are knowledge representation languages used to encode ontologies in a formal (machine-readable and -processable) and standard format.
The most common and popular ontology language is OWL from the World Wide Web Consortium (W3C).
OWL concepts/classes (e.g., genome, gene, etc.)
are created using formal specifications that an individual object/instance (e.g., genome and gene instances) needs to satisfy to belong to a particular class.
The individuals that share similar characteristics would be members of the same concept.
Title: Implementation

As depicted in Fig.2, the ontology-based GIS application was implemented using a combination of the two platforms of ArcGIS [22] and Protégé [23] and MySQL as a database management system.
ArcGIS is a commercial software package containing a set of integrated applications, such as ArcMap and ArcCatalog.
ArcMap is the main mapping application allowing users to generate maps, query attributes, analyze spatial relationships, and lay out final projects.
The ArcCatalog application allows users to manage geographic data [24].
Several programs have been developed to edit and view ontologies, such as Ontolingua and Protégé.
Both Ontolingua and Protégé are capable of generating a separate frame for every knowledge item that covers the information, such as definitions, links, and relationships [20].
In the proposed application, we applied the Protégé software as an ontology-developing environment.
Protégé is a free and open-source platform that provides a set of tools for constructing domain ontologies and knowledge-based applications.
We constructed the rumen microbial genome ontology to organize genome-related data, including classes (e.g., microbial genomes found in rumen and their associated genes) and their relationships/properties (e.g., gene name, product, catalyzed reaction, and accession number of a gene).
In order to implement the system, we applied the genome-annotated data of several ruminal microbes deposited in NCBI.
In the first phase of implementation, the genome-annotated data were stored in a MySQL database using Navicat software (Fig.3).
Navicat, as a powerful solution for MySQL administration, offers a graphical interface for managing databases (http://navicat.com/).
The second phase involved the connection of the database to both ArcGIS and Protégé as the GIS engine and ontology platform, respectively.
The database was connected to the ArcCatalog application of ArcGIS.
The genome layers were created as shapefiles via ArcCatalog and displayed in an ArcMap environment (Fig.4).
As shown in Fig.5, the DataMaster plugin of Protégé was integrated with a MySQL database through a JDBC driver.
DataMaster is a plugin for Protégé editor that allows the conversion of a relational database into an RDF model (resource descriptive framework) that can be processed and understood by machines [25].
After integrating DataMaster with the MySQL database, each of the tables from the database was imported into Protégé as an OWL class, along with its instances and properties.
One of the advantages of a GIS application in the genomic area is that users can easily query and analyze genome-related data.
In addition, the users are able to select locations on a genome map and display their records in the attribute table.
GIS has a special functionality to support users with simple and complex queries for the data represented in each layer.
As shown in Fig.6, complicated queries were made based on the attributes of genomic data and a given gene location.
Fig.6 presents two query examples: one is for a special gene product (phosphoglucosamine mutase) in the annotated genome of Streptococcus bovis, and another is a query based on a specific accession number belonging to the gene (genes).
The results of both query examples are presented on the genome, such that they are highlighted with a red rectangle on the genome layer.
ArcGIS allows one to use &#34;Identify tool.&#34;
Applying this tool allows users to be able to have information about a gene displayed in ArcMap.
It has been used as the fastest way to find out the characteristics of a location on a map.
On the gene event layer, users are able to identify genes by clicking on a particular location (Fig.7).
This will present the attribute data of the selected location.
The genome ontology of ruminal microbes was developed using OWL/RDF language in the Protégé environment.
Fig.8 shows a screenshot of Protégé&#39;s class tab, indicated by a red rectangle.
Via this tab, classes, along with their subclasses for the ontology, are displayed in the blue rectangle, and all instances and their attributes are shown in yellow and green rectangles, respectively.
Additional tabs for object properties and data type properties are available in the editor, which has the same view as the class tab.
In conclusion, the proposed application provides users with the ability to manage genomic data related to the microbial community of the rumen.
This application will be particularly helpful to researchers and experts who are working in fields, such as biology, animal sciences, and genomics.
Two central aspects of designing and using such an interactive application are data querying and data sharing.
The application allows users (1) to access, visualize, store, query, and analyze the genomic data and (2) to share such data among biology users and applications by providing a semantic structure of data.
The data stored in the ontology is machine-processable and can be shared and reused by biology applications.
With the emergence of large datasets in biology, life scientists are encountering bottlenecks in handling biological data.
The increase of genomic data caused by next-generation sequencing technologies is leading to a continuous decrease in the effectiveness of conventional genomic browsers [26].
Most genomic browsers are principally used for visualizing the genomic features and a user&#39;s genomic data [26].
In order to visualize genomic features, different genomic browsers are available on the web, such as Ensembl [27] and University of California, Santa Cruz (UCSC) [28].
Some genomic browsers represent gene annotations as horizontal data tracks.
For example, the UCSC genomic browser represents annotations as a series of horizontal tracks over the genomic sequence.
In this browser, each track presents a specific type of annotation [28].
In comparison to the paradigm of horizontal data tracks applied by genomic browsers, the GIS application uses the concepts of a LR technique to present genomes in a dynamic manner, where users can define/change attributes of genes in a table and interactively see the gene events on genome layers.
A previous study of a GIS application in genomic data management employed the concepts and tools that were defined for displaying the geographic data to develop GenoSIS.
GenoSIS was proposed to spatially represent genomes and interact with genome-related attribute data [16].
However, this system is based on a fixed linear data model, which presents the genome as a linear feature with associated attributes.
The system is fixed in the sense that defining new or changing gene attributes requires users to graphically draw new lines and change the GIS file (i.e., genome shapefile and geodatabase file).
Moreover, this system lacks the ability to semantically represent genome-related information, where information can meaningfully be interpreted by software agents and can be shared between different biological applications.
One of the most interesting capabilities of GIS compared to existing genomic browsers is its ability to make complex attribute and spatial queries.
A GIS query is a user-defined request that examines genome/gene layer or tabular attributes based on user-selected criteria and displays only those features or records that satisfy the criteria.
The proposed application has the potential to store, represent, visualize, and analyze genome-annotated data in a dynamic manner.
In addition, our application adopted ontology to organize genomic data in a semantic framework, which facilitates data sharing among biology domains, applications, and experts.
During recent years, there have been many web-based developments that use ontology as their knowledge base, such as GO [21], Plant Ontology [29], and Microbial Ontology [30], etc.
Ontologies define domain concepts and the relationships between them and therefore provide a domain language that is meaningful to both machines and humans [30].
In the present study, we used Protégé, which contains a query interface known as the SPARQL query panel, through which users are able to query particular information from the ontology [30].
However, the proposed application in this study is subject to some limitations.
One of the limitations is that the application is not able to represent all genomic data (e.g., sequences) and can not make a query on several genome event layers simultaneously.
Another limitation is related to the lack of powerful inference engines to reason genomic data for the purpose of data sharing.
Effective inference engines are required to semantically interpret data and exchange them with their intended meanings.
One source of limitation is also related to the fact that the application is available on desktop; future research is required to develop an online version of the application and a data server consisting of microbial genome databases for browsing genome-annotated data.
Our work continues toward future plans, including the extension of this application to develop an ontology-based GIS interactive database for rumen microbial communities.
Effort is already under way for developing such a comprehensive database to contain a wide spectrum of rumen microbial genomes.
A schematic diagram outlining the linear referencing procedure for mapping gene events on a genome linear feature.
The route identifier filed &#34;GenomeID&#34; is presented with a red box.
The ontology-based GIS architecture for managing genomic data.
The representation of genome-annotated data in a MySQL database using Navicat.
Microbial genomes displayed in ArcMap as different map layers.
Diagram of how a MySQL database connects to the Protégé platform.
The DataMaster plugin is shown by a red small rectangle.
Querying gene attributes using ArcMap.
The query window is presented in the figure.
Identifying features with the identify tool.
The gene attributes are demonstrated in the identify window.
Representation of a Protégé screen, as an ontology-developing environment.
Title: Editor&#39;s Introduction to This Issue

Since the advent of next-generation sequencing, we have realized that a substantial amount of the human genome, not just the 2% to 5% of the human genome that is occupied by coding genes, is transcribed, which means that there is a substantial amount of non-protein-coding RNAs in the human genome.
Non-coding RNAs are RNA transcripts that are not translated into proteins; some of them play a role in the regulation of gene expression and will be eventually implicated in on diverse diseases, including cancer.
In this issue, Dr. Yong Sun Lee (University of Texas) reviewed the implications of a novel type of non-coding RNA, nc886, which was recently discovered by his group.
nc886 is known to have putative tumor suppressor and tumor-sensing functions.
In this review, the characteristics and roles of this ncRNA in tumorigenesis are introduced.
Dr. Namshin Kim&#39;s group (KOBIC) reported the effect of exome sequencing depth on the discovery of sequence variants for clinical use.
They suggested the optimal sequencing depth for defining most diverse function-related variants, which would be useful for next-generation sequencing based biomedical research and clinical translation.
Dr. Jung Kyoon Choi&#39;s group (KAIST) investigated how transcription and epigenetic processes are genetically regulated and associated with each other.
For this purpose, they examined the changes in chromatin accessibility and gene expression in response to genetic variation by means of quantitative trait loci mapping.
In addition, there are three articles on in silico modeling.
Title: A Novel Type of Non-coding RNA, nc886, Implicated in Tumor Sensing and Suppression

nc886 (=vtRNA2-1, pre-miR-886, or CBL3) is a newly identified non-coding RNA (ncRNA) that represses the activity of protein kinase R (PKR).
nc886 is transcribed by RNA polymerase III (Pol III) and is intriguingly the first case of a Pol III gene whose expression is silenced by CpG DNA hypermethylation in several types of cancer.
PKR is a sensor protein that recognizes evading viruses and induces apoptosis to eliminate infected cells.
Like viral infection, nc886 silencing activates PKR and induces apoptosis.
Thus, the significance of the nc886:PKR pathway in cancer is to sense and eliminate pre-malignant cells, which is analogous to PKR&#39;s role in cellular innate immunity.
Beyond this tumor sensing role, nc886 plays a putative tumor suppressor role as supported by experimental evidence.
Collectively, nc886 provides a novel example how epigenetic silencing of a ncRNA contributes to tumorigenesis by controlling the activity of its protein ligand.
Non-coding RNAs (ncRNAs) are defined as RNA transcripts that are not translated into proteins.
Classically, only a limited number of RNAs, including ribosomal RNAs, transfer RNAs, small nuclear RNAs, and small nucleolar RNAs, were perceived as ncRNAs.
However, this view changed completely after in-depth analysis of mammalian transcriptomes (reviewed in Mattick and Makunin [1]).
A decade ago, the advent of next generation sequencing and high-resolution genome tiling arrays revealed that up to 70% of the human genome is transcribed but only about 2% is occupied by protein-coding regions.
Moreover, some ncRNAs were found to play regulatory roles for gene expression, as compared to classical ncRNAs that play housekeeping roles.
So, ncRNAs are much more dominant in quantity and more diverse in function, than they were thought before.
These regulatory ncRNAs have been a topic of intense study during the last decade.
So far, most regulatory ncRNAs bind to target genes via nucleic acid interaction to serve as a specificity factor to recruit effector proteins (Fig.1).
For example, nuclear ncRNAs recruit chromatin remodeling factors and microRNAs (miRNAs) recruit the mRNA-slicing complex (reviewed in Fatica and Bozzoni [2] and Filipowicz et al.[3]).
Our laboratory has recently discovered a ncRNA that functions in a completely different way and we named it nc886 [45].
nc886&#39;s function is to bind to a protein and control its activity, rather than to recognize target DNA or RNA.
Here, a summary of its characteristics and roles in cancer will be reviewed.
nc886 has been given several names, including vtRNA2-1, CBL3, and pre-miR-886.
vtRNA2-1 is currently its official name and stands for vault RNA 2-1 [6].
The vault RNAs (vtRNAs) are the RNA components in the vault complex, a large ribonucleoprotein particle whose precise function has not been elucidated yet (reviewed in Berger et al.[7]).
In the human genome, there are three canonical vtRNA genes, vtRNA1-1, 1-2, and 1-3. nc886 was thought to be a vtRNA paralog because of sequence homology and therefore was named vtRNA2-1 [6].
However, nc886 is less similar in sequence to canonical vtRNAs than they are similar to each other [5].
Furthermore, nc886 is neither in physical association with the vault complex nor necessary for the integrity of the complex [5].
nc886 is functionally distinct from vtRNA1-1 [458910].
Another alias for nc886, CBL3, stands for cord blood lymphocyte (CBL)-derived ncRNA 3.
This name originated from the fact that nc886 was one of the most induced ncRNAs when CBL was infected with the Epstein-Barr virus (EBV) [11].
Interestingly, canonical vtRNAs were also induced by EBV, which also contributed to the early misconception of nc886 as a vtRNA [12].
In addition, nc886 was once classified as a miRNA precursor and so was registered in the miRNA database (miRBase: http://www.mirbase.org/) [13].
This was due to the high sensitivity of the next-generation sequencing technologies, together with the high intracellular expression level of nc886 but its short half-life [5].
Thus, its degradation products are continuously produced at a quantity high enough to be captured by small RNA sequencing.
According to the deepBase (http://deepbase.sysu.edu.cn/) where small RNA sequencing data were collected and displayed [14], the nc886 sequence was captured at significantly higher frequency than most other genomic regions.
In addition, 20-30 nucleotides at the 5&#39;- and 3&#39;-end of nc886 were captured more prominently than its middle portion.
This sequencing pattern misinterpreted as each end corresponding to mature miR-886-5p and -3p respectively.
However, our extensive analysis proved that it is not a miRNA precursor.
First and foremost, nc886 appears as a single band at 101 nucleotides (nts) in Northern hybridization [458910].
Second, the biogenesis of nc886 is dissimilar to that of miRNAs; nc886 is neither processed from Drosha cleavage nor diced into mature miRNAs by Dicer [5].
Third, there is no inverse correlation in expression between nc886 and the predicted target genes of miR-886-5p or -3p, which is against our expectation if nc886 would to produce functional mature miRNAs [5].
Since nc886 does not produce a miRNA precursor nor is in the vault complex, its aliases pre-miR-886 and vtRNA2-1 are misleading.
Therefore, our research group renamed it non-coding 886 (nc886) [4] and this name will be used throughout the review.
Whereas many other regulatory ncRNAs are transcribed by RNA polymerase II (Pol II) (Fig.1), nc886 is transcribed by RNA polymerase III (Pol III) as evidenced by its sequence and experimental data.
nc886 contains the well-conserved Pol III promoter elements Box A and B followed by an oligo-dT sequence, which is a Pol III termination signal (red colors in Fig.2).
The actual occupation of Pol III on the nc886 genomic region was consistently shown by chromatin immunoprecipitation followed by ultra-high-throughput sequencing (ChIP-seq data) from several research groups [151617].
It is generally thought that Pol III activity is elevated in cancer [18].
In agreement with this idea, the nc886 expression level is high in many cancer cells relative to normal tissues (YSL, unpublished data) [5].
On the contrary, nc886 expression is silenced by CpG DNA hypermethylation in many malignancies [891920].
Methylation at CpG dinucleotides is the most common type of epigenetic modification of the mammalian genomic DNA (reviewed in Klose and Bird [21]).
Some genes have CpG islands, which are blocks of sequence where CpG dinucleotides are enriched and located mostly in the promoter region.
Hypermethylation in CpG islands represses gene transcription and this is one of the most frequent ways that tumor suppressor genes are silenced in cancer.
Noticeably, the genomic region of nc886 has a CpG island (blue bar in Fig.2).
Many studies, done mostly on protein-coding genes transcribed by Pol II, indicate that chromatin remodeling is the prevalent mechanism by which CpG methylation silences gene expression.
However, CpG methylation has barely been examined in ncRNA genes transcribed by Pol III.
To the best of our knowledge, nc886 is the first and so far only case of a Pol III transcript that is silenced by CpG hypermethylation in cancer.
So, nc886 should serve as an ideal model gene for studying a mechanism for epigenetic regulation of Pol III transcription.
nc886 is encoded on human chromosome 5q31 and its locus is flanked by transforming growth factor β-induced (TGFBI) and SMAD5, both of which are implicated in the transforming growth factor β (TGF-β) signaling pathway (Fig.2).
Although this suggests a possible co-regulation of nc886 with these two genes, it is interesting to consider the CCCTC-binding factor (CTCF), a barrier protein blocking the spread of heterochromatin (reviewed in Ong and Corces [22]).
ChIP-seq data from the Encyclopedia of DNA Elements (ENCODE) indicate that there are two CTCF binding sites at each side of the nc886 region (brown bars in Fig.2), suggesting that the chromatin status of nc886 is being insulated by CTCF.
According to our unpublished data, nc886 seemed to be co-regulated with its flanking genes in some cancer cells but not in others.
Given the established role of the TGF-β signaling in cancer, it will be interesting to investigate the involvement of TGF-β in the regulation of nc886/TGFBI/SMAD5 expression.
As aforementioned, nc886 is a molecular switch for a protein to modulate its activity.
That protein is protein kinase R (PKR).
PKR is an interferon-inducible, double-stranded RNA (dsRNA)-dependent protein kinase that senses viral infection in the cellular cytoplasm (reviewed in Garcia et al.[23]).
The PKR signaling pathway is very complicated: it is activated by several other stresses besides viral infection and its activation affects many downstream pathways.
Elaboration on the whole PKR pathway is beyond the scope of this review, but here a very brief introduction only about relevant parts will be provided.
Upon viral infection, the classical PKR pathway begins with its binding to virus-derived dsRNA that activates PKR and subsequently its downstream events, including (1) phosphorylation of eukaryotic initiation factor 2α subunit (eIF2α) to block global protein synthesis and (2) activation of nuclear factor κ-light-chain-enhancer of activated B cells (nuclear factor κB [NF-κB]).
The PKR:eIF2α branch leads to apoptosis and this explains how mammalian organisms eliminate virally-infected cells.
In contrast, the PKR:NF-κB branch is usually considered to be pro-survival.
Besides the innate immune response, numerous papers have also documented PKR&#39;s role in cancer in the absence of viral infection.
Along with its two opposite functions through eIF2α and NF-κB, both tumor suppressive and oncogenic roles for PKR have been reported in cancer (reviewed in Marchal et al.[24]).
nc886 binds to PKR with an affinity comparable to dsRNA and prevents PKR from being activated, in contrast to the PKR-activating ligand dsRNA [458910].
One reason for these contradictory outcomes between nc886 and dsRNA is their difference in length.
dsRNA is long enough to accommodate PKR dimerization which is a necessary step for its autophosphorylation and subsequent activation (Fig.3).
nc886 at 101 nts long is probably too short for two PKR molecules to bind.
When nc886 is ectopically expressed, it attenuates dsRNA-mediated PKR activation [10].
Notably, knockdown of nc886 is sufficient for PKR activation in the absence of dsRNA [4589], indicating that human PKR has an intrinsic tendency to self-activate and so needs a repressor to avoid its aberrant activation.
In summary, nc886 is a PKR inhibitor.
Both knockdown of nc886 and introduction of dsRNA activate PKR (Fig.3).
nc886 silencing does occur during tumorigenesis and dsRNA is generated during viral infection.
The significance of PKR in viral infection is to sense viruses and remove them by inducing apoptosis through eIF2α phosphorylation.
We implanted this scenario in cancer and proposed a tumor surveillance model.
In this model, nc886 silencing in pre-malignant cells is sensed by PKR, which eliminates them through eIF2α phosphorylation.
Another important point in this model is the escape of viruses or cancer cells from the PKR:eIF2α cell death pathway.
Viruses have various strategies to bypass or disarm PKR.
Likewise, some potential cancer cells can survive the PKR surveillance through genetic or epigenetic alterations.
For an elaborate description on the tumor surveillance model, please read this review [25].
In addition to the nc886:PKR pathway in the tumor surveillance, does nc886 silencing play an active role in tumor etiology or progression?
Two recent papers have provided several pieces of evidence for nc886 being a putative tumor suppressor [89].
First, nc886 expression is suppressed in clinical tumor samples compared to normal tissues, and more importantly, the low expression of nc886 correlates with poor survival of patients.
This was seen in paired normal/tumor specimens from 88 gastric cancer (GC) and 84 esophageal squamous cell carcinoma (ESCC) patients [89].
nc886 expression is also silenced in a number of cancer cell lines of diverse tissue origins, such as esophagus [8], stomach [9], breast (YSL, unpublished data), ovary (YSL, unpublished data), bile duct [4], head and neck, lung, cervix [5], etc.
Besides our data, several other research groups have reported that the suppressed expression of nc886 in many different cancer types including lung cancer [192627], thyroid cancer [28], and breast cancer [29].
Although these papers claimed that they detected mature miRNAs, miR-886-5p and -3p, nc886 is most probably what they detected, because their detection methods could not efficiently distinguish the mature miRNAs from nc886.
Second, the silencing of nc886 RNA expression is caused by CpG hypermethylation, which is one of the most common mechanisms to inactivate tumor suppressor genes in cancer.
The significance of nc886 CpG methylation in the prognosis of cancer patients has been proven by our ESCC and GC data [89] as well as other groups&#39; data in lung cancer [19] and acute myeloid leukemia [20].
Third, acute depletion of nc886 activates oncogenes.
These oncogenes include renowned ones, such as FOS, MYC, MAFB, and ID2, all of which are known to drive cells to malignancy when aberrantly overexpressed [30313233].
Also, nc886 depletion activates oncogenic NF-κB in a PKR-dependent manner [489] and in this case PKR acts as an oncogene in the few cells that have survived the nc886: PKR:eIF2α-mediated apoptotic branch.
In this context, nc886 is an inhibitor of the PKR oncogene and therefore is a tumor suppressor.
Fourth, nc886 is antiproliferative.
Ectopic expression of nc886 inhibits growth of several cancer cells [89].
This effect is specific, because such toxic effects are seen only in cancer cells that do not express endogenous nc886 but not in non-malignant cells that express endogenous nc886.
nc886&#39;s antiproliferative role is thought to be, at least partially, through suppressing NF-κB, as supported by our experimental data from cholangiocarcinoma [4].
nc886 is a newly discovered ncRNA that controls PKR activity and plays tumor sensing and suppressing roles.
nc886&#39;s expression pattern and function in tumors opens a possibility that its measurement can be a diagnostic or prognostic marker and its administration into cancer cells can be a therapeutic strategy.
Nonetheless, we expect that nc886&#39;s role is multifaceted in cancer and may even be oncogenic in certain contexts, because nc886 expression as well as PKR&#39;s role can be in two opposite directions in cancer.
Also, nc886 is a unique case of a Pol III gene that is epigenetically silenced in cancer and so provides an intriguing model for studying epigenetic regulation of Pol III.
Comparison among non-coding 886 (nc886), miRNAs, and nuclear non-coding RNAs (ncRNAs).
Non-coding 886 (nc886) genomic region and sequence.
The diagram on the top illustrates the nc886 genomic locus spanning TGFBI and SMAD5 (designated by thick arrows), with arrowheads indicating the direction of transcription.
Brown bars are CTCF binding sites.
The diagram is not drawn to actual scale.
The nc886 region containing a CpG island is magnified in the middle diagram, and the nc886 RNA portion is further magnified in the bottom with the sequence shown.
CpG sequences are highlighted in gray.
Pol III signatures are in red.
This cartoon has been modified from [89].
TGFBI, transforming growth factor β-induced; CTCF, CCCTC-binding factor.
Similarity between viral infection and non-coding 886 (nc886) silencing.
PKR, protein kinase R; eIF2α, eukaryotic initiation factor 2α subunit.
Title: Effect of Next-Generation Exome Sequencing Depth for Discovery of Diagnostic Variants

Sequencing depth, which is directly related to the cost and time required for the generation, processing, and maintenance of next-generation sequencing data, is an important factor in the practical utilization of such data in clinical fields.
Unfortunately, identifying an exome sequencing depth adequate for clinical use is a challenge that has not been addressed extensively.
Here, we investigate the effect of exome sequencing depth on the discovery of sequence variants for clinical use.
Toward this, we sequenced ten germ-line blood samples from breast cancer patients on the Illumina platform GAII(x) at a high depth of ~200×.
We observed that most function-related diverse variants in the human exonic regions could be detected at a sequencing depth of 120×.
Furthermore, investigation using a diagnostic gene set showed that the number of clinical variants identified using exome sequencing reached a plateau at an average sequencing depth of about 120×.
Moreover, the phenomena were consistent across the breast cancer samples.
Exome capture sequencing (simply referred to as &#34;exome sequencing&#34;) is a next generation sequencing (NGS)-based technique which targets the genomic sequences of protein-coding regions (&#34;exomes&#34;) of a species [1].
Although protein-coding regions constitute only 1% of the human genome, they harbor 85% of the mutations that have significant effects on disease-related traits [2].
Therefore, exome sequencing is a potential contributor to the understanding of diverse human diseases [2].
With a dramatic decrease in the cost and time required for the generation of sequences with high accuracy [3], exome sequencing is now widely used to understand many genetic diseases.
For example, in the Netherlands, exome sequencing of ten blood samples from patients with severe intellectual disabilities allowed the identification of five new candidate genes associated with such disabilities [4].
Further, Ng et al.
[1] sequenced the exomes of twelve human samples with or without Freeman-Sheldon syndrome (FSS), which is a rare dominantly inherited disorder, and observed an association between the MYH3 gene was responsible for FSS.
Furthermore, Huh et al.
[5] used exome sequencing to show that the c.234 G &gt; A and c.1150C &gt; T mutations in exon 18 of the HGSNAT gene were common in mucopolysaccharide patients.
Exome sequencing techniques have also been used to understand the risks of various cancers, including those of the breast [67], prostate [8], pancreas [9], and others [101112].
Therefore, exome sequencing techniques have become a new primary paradigm for research on genetic diseases and cancers.
One important issue that needs to be addressed for the clinical utilization of NGS-based sequencing data is the adequate depth of sequencing.
Sequencing depth is directly related to the cost and time required for the generation, processing, and maintenance of sequencing data [13].
In this vein, several studies have been performed to investigate the impact of sequencing depth on NGS data intended to identify genomic variants.
For example, Hou et al.
[14] investigated the discovery rates of single nucleotide polymorphisms (SNPs) and structural variants in healthy samples using whole genome sequencing (WGS) at an average sequencing depth of 180×.
They observed that most of the variations were identified at an average depth of 100×.
Meanwhile, Ajay et al.
[15] calculated the genome coverage and discovery rates of variants in healthy samples using WGS data at 100× average mapped depth.
The callable portion of the genome was 90% at a depth of 40×, and that of the protein-coding exome region was about 88% at a depth of 100×.
However, these studies analyzed sequencing data derived from healthy individuals; the adequate depth required to discover clinically significant variations still needs to be addressed.
In this study, we investigated for the first time the effect of exome sequencing depth on the discovery of genomic variations for clinical use.
Toward this, we performed exome sequencing in ten germ-line blood samples from breast cancer patients using the Illumina platform GAII(x) at a high depth of ~200×.
We also checked the discovery rates of diverse variations as a function of the sequencing depth, using total and diagnostic gene sets.
Title: Samples and sequencing

Total ten subjects were included in this study.
They were enrolled from the Seoul National University Hospital and Seoul National University Bundang Hospital in Korea.
They were all diagnosed with breast cancer, with a history of two or more affected family members and/or other risk factors, like bilateral breast cancer or young age of onset.
Exome capture was carried out with the blood samples of the ten subjects using Agilent exome capture kits (SureSelect V2) and sequencing was performed on the (Illumina, San Diego, CA, USA).
Title: Sequence alignment and variant calling

The raw reads in the prepared datasets were aligned to the hg19 reference genome, which was downloaded from the University of California Santa Cruz (UCSC) genome browser (http://genome.ucsc.edu/), using BWA (bwa-0.6.2) [16] with default parameters and a seed length of 45 bp.
The Sequence Alignment and Mapping (SAM) files were converted to Binary Alignment and Mapping (BAM) files using SAMtools [17].
Picard (http://picard.sourceforge.net/) was used to mark and remove the polymerase chain reaction duplicates detected from the BAM files.
The Genome Analysis Toolkit (GATK) [18] was then used for base quality recalibration and local realignment around the potential indel sites.
The UnifiedGenotyper [19] in the GATK was used in the final step for variant calling using a Bayesian model.
Variants were filtered by three types of filtering methods: (1) HARD_TO_VALIDATE: MQ0 ≥4 and [(MQ0/(1.0 × DP)] &gt;0.1; (2) QualFilter: QUAL &lt;10; and (3) Additional: QUAL &lt;30.0 || MQ &lt;20.0 || DP &lt;7.
Title: Annotation of genetic variants

We annotated variants using diverse tools and databases.
The region information of variants (such as coding or intron regions and splice sites) was annotated using SnpEff [20].
Further, we predicted the functional effects of variations on genes (such as silent, nonsense, or missense SNPs) using SnpEff [20].
We also predicted whether an amino acid substitution significantly affects protein function (such as deleterious or tolerated SNPs) using SIFT [21].
In addition, we checked previously known SNPs using the dbSNP database [22].
We also checked clinical SNPs using the ClinVar database [23].
Title: Analysis of depth of coverage in the diagnostic gene set

We extracted information for the positions of the 175 diagnostic genes from the hg19 reference genome, which was downloaded from the UCSC genome [24].
Based on this, we analyzed the depth of coverage and mapped mean depth according to increasing sequencing depths using Samtools &#34;mpileup&#34; with default parameters [17].
Title: Count analysis of diverse variants

We calculated counts of the number of diverse clinical variants including nonsense, missense, and deleterious SNPs in coding regions of the total genome and diagnostic genes using in-house scripts.
Title: Effect of exome sequencing depth on the discovery of variants for clinical use

To investigate the effect of exome sequencing depth on the discovery of clinically meaningful variants, we first sequenced the whole blood DNA samples from ten breast cancer patients using the Illumina platform GAII(x) at a high sequencing depth of ~200× (Fig.1A).
The platform covers 95% of the human exonic regions (http://www.gemomics.agilent.com).
We then extracted data at depths of 20× to 200× by random selection and shuffling (Fig.1B).
Next, we performed read mapping, local realignment, duplicate marking, and base quality recalibration for each sample (Fig.1C).
Diverse variations such as SNPs, and insertions and deletions (indels) were called by using the GATK [16].
We also called function-related variations after the annotation of functional and regional information using various open databases and tools (Methods).
Finally, we analyzed counts, positions, reading depths, and genomic coverage of the identified variations as a function of the sequencing depth using total or diagnostic gene sets (Fig.1D).
We first checked the numbers of called SNPs in protein-coding (denoted as &#34;coding SNPs&#34;) and total genomic regions (denoted as &#34;total SNPs&#34;) of the platform in order to assess the effect of sequencing depth on the discovery of SNPs (Fig.2A).
As expected, the numbers of the identified total SNPs increased at higher average sequencing depths.
For example, the median number of total SNPs for the ten samples gradually increased from 33,765 at 20× to 114,707 at 200× (an increase of 80,942 for the total SNPs).
However, the median number of SNPs in the coding regions increased more rapidly in the first half (an increase of 59,073 from 20× to 120×), and reached a plateau at an average sequencing depth of around 120×.
The increase in the number of coding SNPs was significantly smaller after that (an increase of 827 from 120× to 200×).
This also means that the number of non-coding SNPs detected, both intronic and intergenic, increased rapidly after a depth of 120× was achieved.
This phenomenon was more or less consistent across the sequencing data, from the first sample to the last.
This trend was also preserved in the indel variants (Supplementary Fig.1).
Further, the number of indels in the total region of the platform (denoted as &#34;total indels&#34;) increased steadily, but the increasing ratio of indels in the coding regions (&#34;coding indels&#34;) leveled off after a depth of 120× was achieved.
After functional annotation using the SnpEff database [20], we analyzed the numbers of functional SNPs (nonsense, missense, and silent) while increasing the sequencing depth.
The results indicate that the median number of missense SNPs detected in the ten samples increased from 6,015 to 9,731 (Fig.2B).
However, the rate of increase was sharp in the first half, slowing significantly after 120×.
We observed a similar trend for the nonsense SNPs as well.
In addition, this phenomenon was also observed for the deleterious SNPs using the SIFT [21] (Fig.2C).
In summary, the number of deleterious SNPs detected, such as missense and nonsense, increased with the sequencing depth, but the ratio of this increase reduced significantly after 120×.
Mutations in splice sites are associated with many diseases [252627].
Therefore, we next analyzed the number of SNPs detected near splice sites with respect to the sequence depth (Fig.3).
We found that the number of called SNPs increased with increasing depth.
However, many of the SNPs were detected in intronic regions around the splice sites, about twice the number of those detected in exonic regions, when increasing the sequencing depth.
The number of detected SNPs, though, was lowest at the splice sites.
This might imply that the sequences of the exonic regions surrounding the splice sites are well conserved than those of the intronic regions.
However, researchers who wish to detect susceptible SNPs in splice sites might have to sequence at depths of more than 120×.
Similarly, all the variations reported in the dbSNP database were also more common in the intronic, rather than the exonic regions (Supplementary Fig.2).
To summarize, the number of deleterious SNPs and indels detected in the coding regions (which are widely used in clinical diagnostics) was only weakly increased a depths more than 120×.
In other words, a sequencing depth of 120× can be considered reasonable when using the exome capture sequencing technique to identify significant variations in diagnostic studies.
Title: Exome sequencing depth for diagnostic genes

Next, we analyzed the genomic coverage and mean depths for 175 diagnostic genes (Supplementary Table 1) with respect to the diverse depths used.
These genes have been widely used for diagnostic, prognostic, and therapeutic purposes at the Seoul National University Hospital in Korea.
Specifically, these diagnostic genes are a subset of genes from the exome dataset.
The percentage of mapped sequences in the coding regions of the 175 genes was almost constant for each depth in the individual samples, although there were slight variations across the samples (Supplementary Table 2).
We further observed that the distributions of the average mapped depths between all human genes in the platform and the diagnostic genes were similar (Supplementary Fig.3).
Moreover, with increasing depths, as expected, the median average mean depth of the ten samples increased almost constantly from a depth of 13.3× to 125.1× (on average, an increase of 12.4× per depth) (bar charts in Fig.4).
In contrast, the coverage curves for the coding regions of the diagnostic genes followed logarithmic trends, regardless of the thresholds of the minimum read depths (line charts in Fig.4).
For example, based on the regions with more than 1× mapped depth, the coverage increased from 99.3% to 99.7% (0.4% increase) at 120× depth compared to that at 20×.
However, the coverage increased by only 0.1% at a depth of 80× (totally, 200×).
Similarly, at 25× minimum mapped depth, over 77.9% of the genomic regions were covered when 120× was used.
Moreover, a 64.6% increase in coverage was observed in the first half (from 13.3% at 20× to 77.9% at 120×), whereas a 7.6% increase was observed in the latter (from 77.9% at 120× to 85.5% to at 200×).
It is known that a 25× mapping depth is the minimum for detecting heterozygous alleles [2829].
In other words, the genomic coverage of the diagnostic genes was not increased significantly after 120×.
We next checked the number and positions of SNPs that have been detected in the early onset breast cancer 2 gene (BRCA2), one of the major risk factors in the development of this cancer [3031].
The results indicate that the read depths of SNPs in BRCA2 increased with increasing sequencing depths (Fig.5A).
However, most of the SNPs had been identified by the time 120× depth was reached, and no more were identified after 140× depth was used (Fig.5B).
Only mapped read depths increased with increasing sequence depths.
Similar phenomena were also observed in the early onset breast cancer 1 gene (BRCA1) (Supplementary Fig.4), another major risk factor in this cancer type [32].
Further, we validated our depth recommendation for diagnostic variant detection using exome sequencing.
To this end, we analyzed the numbers of non-synonymous (NS) SNPs, splice site acceptor or donor site (SS), and coding indel (I) variants in the diagnostic genes as a function of the diverse depths used (Fig.6).
The numbers of NS/SS/I variants of all human genes in the platform increased with increasing sequencing depths (Fig.6A).
However, the numbers of NS variants in the diagnostic genes converged sooner; there was least increase in the NS variants after 60× was used (Fig.6B).
We analyzed the numbers of NS variants using the variations in the ClinVar database [23], which is well known for clinical variants.
Although there was an increase in the numbers of NS clinical variants, the rate of increase was quite low after 120× was used (Fig.6C).
We also checked the variations related to breast cancer using only the clinical variants in the ClinVar database (Fig.6D).
The results indicate that all variations associated with breast cancer were detected at 140× depth, regardless of the sample.
In summary, with exome capture sequencing technique, the most significant clinical variations can be detected at an average depth of 120×.
Finally, we measured the discovery rates of variants at different sequencing depths using Sanger sequencing (Fig.7).
The results show that more than 90% variations were validated when 120×.
Hence, we conclude that, using the exome capture sequencing technique, the most reliable variants are detected at an average depth of 120×.
In this study, we determined the effects of exome sequencing depth on the discovery of function-related diverse variants of human genes and diagnostic genes, especially for clinical use.
For this purpose, we investigated the exome deep-sequencing data for whole blood DNA samples obtained from ten breast cancer patients using an Illumina platform GAII(x) as a function of sequencing depth.
The number of genomic variants identified using exome sequencing reached a plateau at an average sequencing depth of ~120×, and this depth allowed detection of most variations in the human genes.
The results were also consistent with a diagnostic gene set and were similar across samples.
Considering the diverse costs and time related to generation, processing, and maintenance of sequencing data, this suggests that a feasible depth for clinically relevant exome sequencing is about 120×.
These findings can be used to address important questions on the adequate depth for exome sequencing techniques for clinical use.
Study overview.
(A) Whole exome deep sequencing of whole blood DNA samples from ten breast cancer patients at ~200× depth using the Illumina GAII(x).
(B) Generation of ten datasets from 20× to 200× depths.
Each dataset was selected independent of the others.
(C) Read mapping and variant calling with quality control.
Variations such as indels and single nucleotide polymorphisms (SNPs) were annotated with related functional and regional information.
(D) Various analyses of identified variants, such as number of variants, read depth, and genomic coverage according to the mapped depths using both the whole genome and diagnostic genes.
Numbers of called functional single nucleotide polymorphisms of human genes according to sequencing depths.
(A) Numbers of called coding (blue) and total single nucleotide polymorphisms (SNPs) (red) with different sequencing depths: dashed lines are trend lines of means and their error ranges are represented.
Solid lines in the inner chart represents how numbers of SNPs are significantly different from each own preceding.
(B) The bar chart illustrates the numbers of called silent (blue), nonsense (orange), and missense (red) SNPs.
(C) The bar chart illustrates the numbers of called tolerated (blue) and deleterious (red) SNPs in the SIFT database.
(B) and (C) represent median values of the ten samples.
Numbers of single nucleotide polymorphisms (SNPs) near splice sites.
Median numbers of SNPs for the ten samples are depicted.
Different colors indicate increasing sequencing depths.
Mean mapped depth and coverage of diagnostic genomic regions according to sequencing depth.
Lines indicate the coverage proportion of the genomic regions of 196 diagnostic genes with specific minimum depths, according to the sequencing depths.
Red line: genomic coverage &gt;25×.
Bars indicate mean mapped depths (mean depth).
Mapped depths and positions of the called single nucleotide polymorphisms (SNPs) in BRCA2.
(A) Mapped depths (y-axis) of called coding SNPs in BRCA2 (x-axis) according to increasing sequencing depths.
The figure at the bottom indicates the exonic regions of BRCA2.
(B) Different views of panel (A) based on increasing sequencing depths.
The number of called SNPs is presented in parenthesis.
For example, 120× (16) indicates that 16 SNPs were successfully called in the 120× dataset.
Numbers of called non-synonymous (NS)/splice site acceptor or donor site (SS)/coding indel (I) in the diagnostic gene set and the ClinVar database according to different sequencing depths.
(A, B) Numbers of NS/SS/I per depth used in the human genome (A) or in the 175 diagnostic genes (B).
(C) Numbers of NS for the diagnostic genes in the ClinVar database per number of depth used.
(D) Numbers of variants in the genes associated with breast cancer among the diagnostic genes in the ClinVar database.
The x-axis represents increasing sequencing depths.
Fraction of 21 validated single nucleotide polymorphisms (SNPs) in the called SNP set according to different sequencing depths.
The median numbers of called SNPs are depicted for the ten samples.
Different colors indicate independent trials.
Title: Genetic Architecture of Transcription and Chromatin Regulation

DNA microarray and next-generation sequencing provide data that can be used for the genetic analysis of multiple quantitative traits such as gene expression levels, transcription factor binding profiles, and epigenetic signatures.
In particular, chromatin opening is tightly coupled with gene transcription.
To understand how these two processes are genetically regulated and associated with each other, we examined the changes of chromatin accessibility and gene expression in response to genetic variation by means of quantitative trait loci mapping.
Regulatory patterns commonly observed in yeast and human across different technical platforms and experimental designs suggest a higher genetic complexity of transcription regulation in contrast to a more robust genetic architecture of chromatin regulation.
Quantitative trait loci (QTL) mapping has been widely used to discover underlying genetic factors that can explain particular phenotypes of interest.
Thanks to DNA microarray technology, the expression phenotype of thousands of genes was associated with the genotypes across the whole genome in expression QTL mapping [1234567].
Recent advent of next-generation sequencing technology has enabled a genetic profiling of chromatin traits as well as more in-depth analyses of gene expression variation.
Especially, the mechanisms controlling chromatin accessibility have been of particular interest because of their importance in a wide spectrum of DNA regulation processes.
For example, Degner et al.
[8] utilized DNaseI hypersensitivity assay coupled with high-throughput sequencing (DNase-seq) to map genome-wide chromatin accessibility to the genotypes across 70 human individuals, for which a previous RNA-sequencing (RNA-seq)-based expression QTL study revealed new insights into the genetic regulation of transcription [5].
In yeast, expression QTL mapping has been performed for the genetic dissection of transcription regulation in a cross between two parental strains of Saccharomyces cerevisiae (BY4716 and RM11-1a) based on DNA microarrays [1291011].
In order to generate a matched dataset of chromatin accessibility for this set of yeast individuals, we carried out Formaldehyde-Assisted Isolation of Regulatory Elements followed by sequencing (FAIRE-seq) for a total of 96 segregants from the cross of BY4716 and RM11-1a [12].
In this study, we sought to dissect the genetic architecture of the regulation of gene expression and chromatin accessibility by analysing previous data generated in yeast and human based on different technical platforms and experimental designs.
Our main goal was to find differences in the overall regulatory structure between open chromatin and gene expression.
We were also interested to determine whether the two distant species, namely yeast and human, would be different in genetic regulatory architecture and to estimate the effect of the technical or experimental differences in genotyping and measuring the quantitative traits.
Title: Processing of human genotype data

Genotype data from the HapMap project [13] and 1000 Genomes Project [14] for 70 Yoruba (YRI) lymphoblastoid cell lines were used for DNase-seq analysis [8].
The genotype of each single nucleotide polymorphism (SNP) locus was estimated based on the Bayesian framework by means of the BIMBAM tool [15] and the genotype estimates were made available at http://eqtl.uchicago.edu/dsQTL_data/GENOTYPES/.
We first selected 2,157,286 genetic markers (SNPs) with the minor allele frequency greater than 30%.
To reduce complexity and ease interpretation, we focused on the genetic variants that can change the function of the protein (non-synonymous SNPs) or the abundance of the protein (SNPs associated with the expression level of a nearby gene).
The SIFT tool [16] was used to identify non-synonymous SNPs.
We performed expression QTL mapping as described below and identified SNPs that were associated (p &lt; 10-5) in cis (within 200 kb from the nearest gene).
Taken together, 7,211 SNPs were identified for QTL mapping.
Title: Processing of human gene expression data

RNA-seq data for 69 YRI lymphoblastoid cell lines [5] were downloaded from http://eqtl.uchicago.edu/RNA_Seq_data/results.
A total of 18,147 genes were used after normalization to zero mean and unit variance.
Title: Processing of human chromatin accessibility data

DNase-seq data for 70 YRI lymphoblastoid cell lines [8] were downloaded from http://eqtl.uchicago.edu/dsQTL_data/MAPPED_READS/.
Sequence reads from multiple replicates for each sample were combined and F-Seq [17] was run to identify the peaks of the reads from each sample.
Statistical significance of the peak was determined by fitting the data to a gamma distribution to obtain the p-value (script obtained from the F-Seq authors).
p &lt; 10-3 was used to identify significant peaks from each sample.
The overlapping peaks across the YRI individuals were merged into a single peak by using the mergeBED command of BEDTools [18], resulting in a total of 265,130 accessible chromatin regions.
For each sample, the number of the DNase-seq reads mapped to each region was counted and the read count was normalized as previously suggested [1920] to obtain normalized chromatin accessibility, which was then further normalized to zero mean and unit variance across the YRI samples.
Accessible regions falling on promoters or enhancers were identified based on chromatin annotation by Ernst et al.[21].
A total of 45,781 chromatin regions were found to reside in active promoters, weak promoters, poised promoters, strong enhancers, and weak enhancers annotated in the GM12878 lymphoblastoid cell line.
Title: Processing of yeast data

Genotype and gene expression microarray data [10] used in previous expression QTL studies [129] for &gt;100 segregants from a cross between two parental strains of yeast (BY4716 and RM11-1a) were obtained.
As previously suggested [22], adjacent genetic markers with less than three genotypic mismatches across the yeast strains were merged into the average genotype profile, resulting in 1,533 unique markers.
We employed the microarray dataset of normalized expression levels of 5,352 genes as previously used [10].
FAIRE experiments were performed based on the published protocol [23].
The FAIRE-seq data for the 96 yeast strains from our previous work [12] is available at the Gene Expression Omnibus (GEO) database with accession number GSE33466.
Briefly, we identified open chromatin regions in 96 yeast segregants by means of F-Seq [17].
The overlapping peaks across the 96 strains were merged into a single peak by using BEDTools [18], resulting in a total of 7,527 accessible chromatin regions.
For each sample, the number of the FAIRE-seq reads mapped to each region was counted and the read count was normalized as previously suggested [1920] to obtain normalized chromatin accessibility, which was then further normalized to zero mean and unit variance across the 96 segregants.
Title: Trans-QTL analysis

Sixty-three human samples were commonly present in the RNA-seq [5] and DNase-seq [8] data, and 96 yeast segregants in the microarray and FAIRE-seq data.
Therefore, we used the common samples for our QTL mapping.
Linear regression was carried out leading to 2,100,341 chromatin associations and 975,333 expression associations in human, and 110,802 chromatin linkages and 164,217 expression linkages in yeast at p &lt; 0.01.
Genetic markers farther than 200 kb from the nearest gene in human and 100 kb in yeast were identified to examine regulatory relationships acting in trans.
For each trait (gene or accessible chromatin region) and regulatory locus (genetic marker), the number of associations or linkages was obtained and divided by the total number of associations at a given p-value.
For a systematic comparative analysis of chromatin regulation and transcription regulation, we carried out QTL mapping in a unified analytical framework by applying the identical statistical methods and criteria for open chromatin and gene expression data in yeast and human (Methods and Fig.1).
From the human chromatin accessibility data [8], we selected chromatin sites at the promoter or enhancer (as defined by Ernst et al.[21]).
Yeast and human QTL mapping was different in the genetics setting (linkage vs. association) and technical platforms (DNA microarray vs. RNA-seq and FAIRE-seq vs. DNase-seq) (Fig.1).
We selected significant trans-associations based on the p-value of linear regression and then examined the distribution of linkages between regulatory loci (genetic markers) and quantitative traits (gene expression levels or chromatin accessibility).
First, there were particular regulatory loci that were associated with a large number of target chromatin traits but not with gene expression traits both in human and yeast (Fig.2).
Second, expression traits than chromatin traits were associated with a greater number of regulatory loci (Fig.3).
This trend was consistently found when varying p-values were used (Fig.4).
Intriguingly, the number of targets for chromatin regulatory loci is more dispersed resulting in outliers with extremely many targets while the average number is lower as compared to the pattern of expression regulatory loci (grey vs. white in the upper panel of Fig.4).
On the other hand, the number of associated regulatory loci was much greater for gene expression traits in terms of the outliers and average as well (grey vs. white in the lower panel of Fig.4).
These findings can be interpreted as follows.
There are a few regulatory hotspots that are responsible for a large number of chromatin sites.
Most regulatory loci, however, tend to influence fewer chromatin traits than gene expression traits.
Different biological functions are expected between the promiscuous and specific chromatin regulators.
On the other hand, gene expression seems to be more reactive to genetic variation when judged by the number of associated regulatory loci.
This may imply that transcription processes receive more regulatory inputs than chromatin regulation, in which case chromatin accessibility may not be a good predictor of precise gene expression levels.
There was a remarkable consistency of the data for two evolutionarily distant species across different technical platforms and experimental settings, thereby supporting the biological implications of the findings while ruling out the possibility of technical artifacts.
Based on these findings, we propose that chromatin regulation mechanisms have evolved a stable genetic architecture while the transcription regulatory network maintains high genetic complexity and connectivity and is more susceptible to mutations.
First, the lower number of associations per chromatin trait suggests that chromatin structure is more robust to genetic perturbation.
On the contrary, gene expression traits have more associations, increasing the probability of targets being associated with mutations.
Second, chromatin associations are biased to a handful of loci responsible for an extremely large number of chromatin sites.
As long as these hotspot loci are protected from mutations, the perturbation of chromatin regulation architecture could be minimized.
In contrast, the average number of associations is high for transcription regulatory loci, increasing the overall effect of single mutations.
Given the higher probability of expression traits being associated with mutations and the greater influence of individual mutations on transcription, it is conceivable that phenotypic diversity is more likely to be created in response to genetic variation at the level of transcription processes downstream of chromatin regulation steps.
Data analysis scheme.
Relationships between the genetic regulatory loci and the quantitative traits (chromatin accessibility or gene expression) were explored in yeast and human.
Data from different experimental settings and technical platforms were integrated into a unified analytical framework.
RNA-seq, RNA-sequencing; FAIRE-seq, Formaldehyde-Assisted Isolation of Regulatory Elements followed by sequencing; DNase-seq, DNaseI hypersensitivity assay coupled with high-throughput sequencing; QTL, quantitative trait loci.
Manhattan plots displaying the chromosomal distribution of regulatory loci and (x axis) and the number of associations for each locus divided by the total number of associations (i.e., percentage on the y axis) at a given p-value in human (p &lt; 0.01) and yeast (p &lt; 0.05).
QTL, quantitative trait loci.
Manhattan plots displaying the chromosomal distribution of target traits (x axis) and the number of associations for each trait divided by the total number of associations (i.e., percentage on the y axis) at a given p-value in human (p &lt; 0.01) and yeast (p &lt; 0.05).
QTL, quantitative trait loci.
Box plots showing the percentage of associations for each regulatory locus (upper) or quantitative trait (lower) from chromatin quantitative trait loci (QTL) and expression QTL mapping on a log scale in human (left) and yeast (right).
Title: Elucidation of the Molecular Interaction between miRNAs and the HOXA9 Gene, Involved in Acute Myeloid Leukemia, by the Assistance of Argonaute Protein through a Computational Approach

Acute myeloid leukemia is a well characterized blood cancer in which the unnatural growth of immature white blood cell takes place, where several genes transcription is regulated by the micro RNAs (miRNAs).
Argonaute (AGO) protein is a protein family that binds to the miRNAs and mRNA complex where a strong binding affinity is crucial for its RNA silencing function.
By understanding pattern recognition between the miRNAs-mRNA complex and its binding affinity with AGO protein, one can decipher the regulation of a particular gene and develop suitable siRNA for the same in disease condition.
In the current work, HOXA9 gene has been selected from literature, whose deregulation is well-established in acute myeloid leukemia.
Four miRNAs (mir-145, mir-126, let-7a, and mir-196b) have been selected to target mRNA of HOXA9 (NCBI accession No.
NM_152739.3).
The binding interaction between mRNAs and mRNA of HOXA9 gene was studied computationally.
From result, it was observed mir-145 has highest affinity for HOXA9 gene.
Furthermore, the interaction between miRNAs-mRNA duplex of all chosen miRNAs are docked with AGO protein (PDB ID: 3F73, chain A) to study their interaction at molecular level through an in silico approach.
The residual interaction and hydrogen bonding are inspected in Discovery Studio 3.5 suites.
The current investigation throws light on understanding of AGO-assisted miRNA based gene silencing mechanism in HOXA9 gene associated in acute myeloid leukemia computationally.
MicroRNAs (miRNAs) are small non-coding single stranded RNAs (~18-22 bp) found in most eukaryotes, playing a crucial role in a wide range of biological processes, such as host-viral interaction, apoptosis, stem cell maintenance, and metabolism, etc.
miRNAs are involved in gene silencing mechanisms by targeting un-translated regions of mRNA through sequence complementarity [12], and approximately 30% of protein-coding genes are regulated by miRNAs [3].
As many genes can be regulated by a single or multiple miRNAs, an immense interest has been created to understand miRNA-based gene regulation through the identification of miRNAs, as well as their target genes.
The Argonaute (AGO) class of proteins plays a crucial functional role in the molecular recognition of miRNA-mRNA hybrids during the process of gene regulation [45], and mutations in either the sequence of miRNAs or the miRNA target sites have a significant contribution in the manifestation of cancer due to the loss of gene regulation [56].
Leukemia is a cancer of white blood cells that is commonly classified as acute myeloid leukemia (AML), acute lymphoblastic leukemia, and chronic lymphoblastic leukemia.
AML is a well-characterized blood cancer in which the unnatural growth of immature white blood cells takes place, where several genes involved are either up-regulated or down regulated by miRNAs.
Though the importance of miRNAs in gene regulation [234578] and their regulation by RNA-induced silencing complexes [7] are well established, the mechanism of miRNA-induced gene silencing at the molecular level is still unclear [910].
Despite several detailed research studies, controlling abnormal cell proliferation at the molecular level through miRNA-mediated gene silencing is still a major challenge.
Therefore, with the current knowledge, attempts have been made to study the AGO protein-assisted molecular interaction between miRNAs and their target genes, and a computational approach [11] in this regard will be of great aid and value.
Here, in this study, the HOXA9 gene was selected from the literature as an important target in AML, and in silico approaches, like miRNA target site prediction, miRNA-mRNA duplex prediction, and molecular docking between AGO protein and four miRNAs, such as mir-145, mir-126, let7a, and mir-196b [81213], were conducted to elucidate the AGO protein-assisted molecular interaction between miRNAs and the HOXA9 gene in association with AML [1415] computationally.
Title: Selection of HOXA9 gene as a target in AML

The leukemia gene database (http://www.bioinformatics.org/legend/leuk_db.htm) documented the AF10, NUP98, CBFB, NUP214, HOXA9, CREBBP, ARHGEF12, CDX2, LCP1, CEBPA, DEK, FUS, and RUNX1 genes to be involved in AML.
Among them, the HOXA9 gene is reported to have major relevance to their regulatory functions and aberrant expression in both mice and humans [1415].
Furthermore, identification of miRNAs that target the HOXA9 gene would help us to understand its interaction at the molecular level in AML conditions.
Title: Selection of miRNAs targeting the HOXA9 gene

Firstly, from the literature, the miRNAs with affinity for the HOXA9 gene were identified [1213], which was further verified by prediction of miRNA target sites in the HOXA9 gene using the miRTarBase web server (http://mirtarbase.mbc.nctu.edu.tw/).
The search showed four miRNAs-namely, mir-145, mir-126, let-7a, and mir-196b-that have target sites within HOXA9 (Supplementary Tables 1 and 2).
Two different binding sites were identified for mir-196b in the HOXA9 gene, denoted mir-196b.1 and mir-196b.2, respectively.
Title: Prediction of duplex between miRNAs and mRNA of the HOXA9 gene

A computational approach was carried out to study the binding mode of selected miRNAs (mir-145, mir-126, let-7a, mir-196b.1, and mir-196b.2) with the mRNA of HOXA9 to confirm the selection of the best miRNA.
The properties, like predicted binding sites of HOXA9 for miRNAs (Supplementary Table 3) and minimum free energy (MFE) score of duplex interactions, were used for confirmation of binding.
Two different duplexes were formed for mir-196b (represented as mir-196b.1 and mir-196b.2) with HOXA9 due to the identification of two target sites in HOXA9 (Supplementary Table 3).
The MFE scores for folding of miRNAs and the mRNA duplex (Supplementary Table 4) were calculated using the mfold web server (http://mfold.rna.albany.edu/?q=mfold).
The appropriate secondary structures of miRNAs and the mRNA duplex (Supplementary Table 4) were predicted using the Centroid Fold web server (http://www.ncrna.org/centroidfold/).
The prediction of tertiary structures of the duplex were performed in 3dRNA web server (http://biophy.
hust.edu.cn/3dRNA/3dRNA.html) using dot bracket structures (Supplementary Table 4) that resulted from the Centroid Fold.
Title: Molecular interaction between miRNAs and AGO protein

To illustrate the assistance of catalytic AGO protein for the binding of miRNAs (mir-145, mir-126, let-7a, mir-196b.1, and mir-196b.2) with the HOXA9 gene, docking between AGO protein and miRNA-mRNA duplexes was performed in the PachDock online server [16].
The PachDock algorithm ranked the docked complexes on the basis of geometrical shape complementary scores.
Close amino acid residue interactions and participating atoms in hydrogen bonding were inspected and analyzed in the Discovery Studio 3.5 suite (http://accelrys.com/products/discovery-studio/visualization-download.php).
Title: Duplex interaction between miRNAs and mRNA of the HOXA9 gene

The leukemia gene database search result, supported by literature evidence [1415], signifies that HOXA9 (NCBI accession No.
NM_152739.3) has aberrant expression at the molecular level and was therefore selected as a target gene in AML.
Four miRNAs-namely, mir-145, mir-126, let-7a, and mir-196b (Supplementary Table 2)-were taken from the literature, and their target sites in the HOXA9 gene were verified through the miRTarBase web server.
The predicted binding regions in the HOXA9 gene (Supplementary Tables 1 and 3) for the miRNAs above were supported by good MFE scores calculated for the miRNAs and mRNA duplex (Supplementary Table 4) using the mfold web server, which implicates a strong affinity between them at the molecular level.
Three-dimensional binding modes of all miRNAs and mRNA duplexes (mir-145, mir-126, let-7a, mir-196b.1, and mir-196b.2) were constructed and deciphered (Fig.1A-1E) using the Discovery Studio 3.5 suite.
Title: Study of binding affinity between miRNAs and AGO protein through docking

The three-dimensional structure of AGO protein, with 685 amino acid residues, was retrieved from Protein Data Bank (PDB ID: 3F73, chain A).
The AGO protein is a silencing complex structure with a seed containing guide DNA and target RNA duplexes.
The structure preparation and correction of AGO protein were performed in Discovery Studio 3.5 suite before docking.
The docking between miRNA-mRNA duplexes (mir-145, mir-126, let-7a, mir-196b.1, and mir-196b.2) and AGO protein was carried out separately in the PachDock online server on the basis of geometrical shape complementary scores.
Out of 10 binding complexes generated for each miRNA-mRNA duplex docked with AGO protein, the highest-scoring complex was chosen as the best binding mode.
However, a good dock score in all cases of docking between the selected miRNAs (mir-145, mir-126, let-7a, and mir-196b) and AGO protein implicated a strong interaction between them at the molecular level (Supplementary Table 5).
Title: Study of interaction between miRNAs and AGO protein at the molecular level

Strong hydrophobic amino acid residues and amino acids with aromatic rings, which are relatively hydrophobic in nature, generally take part in hydrophobic interactions and contribute towards the stability in binding moieties while the protein interacts with macromolecules.
From the inspection of the close amino acid residues of AGO protein surrounding the miRNAs (mir-145, mir-126, let-7a, mir-196b.1, and mir-196b.2), some strongly hydrophobic amino acids, like LEU 652, LEU 267, ALA 644, and ALA 648 in mir-145; LEU 267, LEU 277, LEU 279, LEU 281, ALA 278, ALA 648, and VAL 606 in mir-126; LEU 267, LEU 279, LEU 281, and ALA 278in let-7a; ALA 111, VAL 58, and LEU 132 in mir-196b.1; and VAL 549, VAL 152, and ALA 170 in mir-196b.2, were observed within a distance of 3.5 Å (Table 1, Fig.2A-2E).
Similarly, amino acids, like TYR 642, PHE 649, and PHE 647 in mir-145; PHE 649 in mir-126; and TYR 171 and TYR 135 in mir-196b.2, with aromatic rings were also found in the binding pocket of miRNAs (Table 1, Fig.2A-2E) within a distance of 3.5 Å.
The study of hydrogen bonding patterns between atoms is another major parameter to study the stability of interactions between bio-molecules at their atomic level.
Some amino acids, like ARG 615, ARG 651, HIS 445, and PHE 649 in mir-145; SER 280, HIS 607, and THR 613 in mir-126; SER 280, ARG 608, GLY 612, LEU 281, and THR 613 in let-7a; ARG 51 and ARG 59 in mir-196b.1; and SER 576, GLY 577, ARG 580, and ARG 615 in mir-196b.2 (Tables 1 and 2, Fig.3A-3E), were observed to participate in hydrogen bonding with miRNAs within the binding pocket.
The current study focused on in silico verification of miRNAs for targeting the HOXA9 gene, associated in AML, and the molecular interaction of catalytic AGO protein with these miRNAs to study the gene silencing mechanism computationally.
As the literature has reported [1213], four mature miRNAs-namely, miR-145, mir-126, let-7a, and mir-196b-are found to have target sites in the HOXA9 gene and were hence taken for further study.
The binding sites in HOXA9 were predicted for these miRNAs using miRTarBase (Supplementary Tables 1 and 3), which was followed by the secondary structure prediction and miRNA-mRNA duplex formation.
Two binding sites were predicted for mir-196b in the HOXA9 gene (Supplementary Table 1), denoted mir-196b.1 and mir-196b.2, respectively; so, two duplexes are formed between mir-196b and mRNA of the HOXA9 gene (Supplementary Table 3).
The results suggested a strong and energetically favorable binding mode between all miRNA-mRNA duplexes (Supplementary Table 4), which is also supported by the reported interaction between them (Fig.1).
The good MFE score (-5.90 kcal/mol) also reflected that mir-145 had the highest binding affinity for the HOXA9 gene (Supplementary Table 4) among all miRNAs.
Furthermore, to study the molecular interaction between AGO protein and miRNA-mRNA duplexes (miR-145, mir-126, let-7a, mir-196b.1, and mir-196b.2), molecular docking was carried out.
The presence of strong hydrophobic amino acid residues (LEU 652, LEU 267, ALA 644, and ALA 648 in mir-145; LEU 267, LEU 277, LEU 279, LEU 281, ALA 278, ALA 648, and VAL 606 in mir-126; LEU 267, LEU 279, LEU 281, and ALA 278 in let-7a; ALA 111, VAL 58, and LEU 132 in mir-196b.1; and VAL 549, VAL 152, and ALA 170 in mir-196b.2) within a distance of 3.5 Å in the binding pocket of miRNAs implicates strong binding interactions at the molecular level between AGO protein and all chosen miRNAs (Table 1, Fig.2A-2E).
Also, amino acids with aromatic rings (TYR 642, PHE 649, and PHE 647 in mir-145; PHE 649 in mir-126; and TYR 171 and TYR 135 in mir-196b.2) were also observed within a distance of 3.5 Å, which supports the binding interaction (Table 1, Fig.2A-2E) between AGO protein and miRNAs.
The amino acid residues participated in hydrogen bonding (Table 2, Fig.3A-3E) with miRNAs (miR-145, mir-126, let-7a, mir-196b.1, and mir-196b.2) within a distance of 3.0 Å, showing strong evidence of binding and a molecular interaction between AGO protein and all chosen miRNAs.
The current investigation proved that mir-145, mir-126, let-7a, mir-196b.1, and mir-196b.2 have strong binding affinity for the HOXA9 gene and also throws light on AGO protein-assisted molecular interactions with miRNA-mRNA duplex structures.
This report would be helpful in understanding miRNA-based gene silencing mechanisms for the HOXA9 gene in AML disease.
Furthermore, a molecular dynamic simulation approach might be implemented to study the dynamic behavior of these interactions, which would help us better understand miRNA-based gene silencing mechanisms in AML computationally.
Once validated experimentally, this methodology would further help us to design suitable miRNAs against respective genes rationally.
Three dimensional binding modes of mir-145, mir-126, let-7a, mir-196b.1, and mir-196b.2 with mRNA of HOXA9 gene are deciphered in section (A), (B), (C), (D), and (E), respectively.
Amino acid residues of Argonaute protein participating in the interaction with mir-145, mir-126, let-7a, mir-196b.1, and mir-196b.2 within distance of 3.5 Å are reported in section (A), (B), (C), (D), and (E), respectively.
Amino acid residues of Argonaute protein participating in hydrogen bonding interaction with mir-145, mir-126, let-7a, mir-196b.1, and mir-196b.2 within distance of 3.0 Å are reported in section (A), (B), (C), (D), and (E), respectively.
Title: Amino acid residues of Argonaute protein participating in the interaction with mir-145, mir-126, let-7a, mir-196b.1, and mir-196b.2 within a distance of 3.5 Å

Title: Amino acid residues of Argonaute protein participating in hydrogen bonding with miRNAs within a distance of 3.0 Å
Title: In Silico Structural and Functional Annotation of Hypothetical Proteins of Vibrio cholerae O139

In developing countries threat of cholera is a significant health concern whenever water purification and sewage disposal systems are inadequate.
Vibrio cholerae is one of the responsible bacteria involved in cholera disease.
The complete genome sequence of V. cholerae deciphers the presence of various genes and hypothetical proteins whose function are not yet understood.
Hence analyzing and annotating the structure and function of hypothetical proteins is important for understanding the V. cholerae.
V. cholerae O139 is the most common and pathogenic bacterial strain among various V. cholerae strains.
In this study sequence of six hypothetical proteins of V. cholerae O139 has been annotated from NCBI.
Various computational tools and databases have been used to determine domain family, protein-protein interaction, solubility of protein, ligand binding sites etc.
The three dimensional structure of two proteins were modeled and their ligand binding sites were identified.
We have found domains and families of only one protein.
The analysis revealed that these proteins might have antibiotic resistance activity, DNA breaking-rejoining activity, integrase enzyme activity, restriction endonuclease, etc.
Structural prediction of these proteins and detection of binding sites from this study would indicate a potential target aiding docking studies for therapeutic designing against cholera.
Vibrio cholerae is a gram-negative, highly motile, curved or comma-shaped rod with a single polar flagellum [1].
V. cholerae is transmitted by the fecal-oral route, mainly found in unhygienic environment.
V. cholerae secretes enterotoxin that induces a life-threatening secretory diarrhea called cholera.
Cholera is a major epidemic disease.
The cholera toxin binds to the plasma membrane of intestinal epithelial cells and releases an enzymatically active subunit which causes a escalation in cyclic adenosine 5-monophosphate (cAMP) production.
The resulted high cAMP level inside the cell causes massive secretion of electrolytes and water into the intestinal lumen.
Other Vibrios may also be clinically significant for human and some are well-known to cause diseases in domestic animals as well.
Nonpathogenic Vibrios are widely dispersed in the environment, mostly in estuarine waters and seafood&#39;s [2].
V. cholerae comprises nearly 200 serogroups based on the O antigenic structures [3].
Among them two serogroups of V. cholerae O1 and O139 cause widespread cholera epidemics [4].
The emergence in 1992 of a V. cholerae non-O1 serovar, labeled V. cholerae synonym O139 Bengal, in Bangladesh and India and its subsequent appearance in Southeast Asia, displacing V. cholerae O1 El Tor, was well known causative agent in the history of cholera [5].
In the autumn of 1993, V. cholerae serogroup O139 (Bengal), was implicated in outbreaks of cholera in Bangladesh and India.
V. cholerae serogroup O139 (Bengal), causes characteristic severe cholera symptoms and has been implicated in a case of a traveler returning from India to the United States [6].
V. cholerae O139 serogroup strains showed susceptibility to 22 anti-bacterials in various regions of the world and an increase in resistant markers with resistance to fluoroquinolones [7].
During recent years, hundreds of bacterial genomes are available, while their annotation is of interest [8].
However, many of these protein functions are still unknown.
For this reason, there is an increasing demand for the annotation of the functions of uncharacterized proteins, called &#34;hypothetical proteins&#34; [9], but the structures of which are known though.
Structural genomics initiatives deliver plenty of structures of hypothetical proteins at a constantly growing rate.
However, without function annotation, this huge structural storage is of little use to biologists who are interested in particular molecular systems.
Additionally some of the proteins, which are known to be sound annotated, may have further functions beyond their listed archives.
About half of the proteins in genomes are candidates for hypothetical proteins (HPs) [10].
Many of the &#34;hypothetical proteins&#34; occur in fact in more than one bacterial species, which increases the probability that they are indeed protein coding genes and not the consequence of erroneous gene predictions.
Proteins that occur in diverse species can be combined into orthologous groups, which are known to be suitable for functional analyses and annotations of newly sequenced genomes [11].
Improving the functional annotation is of great importance for many follow up studies and we here apply computational tools for function prediction for one of the most devastating human pathogens V. cholerae O139, the causative agent of cholera especially in Southeast Asia.
Therefore an improved functional annotation of its proteome is of particular urgency.
The annotation of these HPs may be helpful as markers and pharmacological targets.
With the overall faith that the majority of hypothetical proteins are the product of pseudogenes, it is necessary to have a tool with the capability of analyzing the minority of hypothetical proteins with a high probability of being expressed [10].
So far, there is no classification of HPs and functioning terms are swapping definitions of hypothetical proteins.
Here, we combined physiochemical properties with protein-protein interaction (PPI) based function predictions.
Our present study is mainly aimed to predict the structure, function and binding sites of these HPs which are important for docking studies for drug designing.
Title: Sequence retrieval

Six randomly selected HPs which contain standard number of amino acids sequences of V. cholerae O139 were randomly retrieved from NCBI (http://www.ncbi.nlm.nih.gov/) for annotation.
Moreover these were supposed to find out interactions between these proteins as they are both from chromosomal and plasmid DNA.
The sequence IDs of those 6 HPs were gi|84095108, gi|163644906, gi|163644912, gi|163644916, gi|84468567, and gi|84468557.
Various computational tools and databases were used to analyze the different properties i.e., physicochemical, functional, and structural characteristics of HPs.
Title: Physicochemical and functional categorization

By using the Expasy&#39;s Protparam server (http://us.expasy.org/tools/protparam) physicochemical characterization, molecular weight, theoretical isoelectric point (pI), total number of positive and negative residues, extinction coefficient [12], instability index [13], aliphatic index (AI) [14] and grand average hydropathy (GRAVY) [15] of HPs were analyzed.
Title: Pfam

Pfam (http://pfam.sanger.ac.uk/) is designed as a comprehensive and accurate collection of protein domains and families [16,17].
Pfam families are typed as Pfam-A and Pfam-B.
Each Pfam-A family possess a curated seed alignment containing a small set of envoyed members of the family and an automatically created full alignment which contains all noticeable protein sequences belonging to the family, as defined by profile Hidden Markov Models searches of primary sequence databases.
On the other hand, Pfam-B entries are automatically created from the ProDom database and are shown by a single alignment [18].
Title: CDD-BLAST

CD-Search (http://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi/) was done to find out the conserved domain of these protein sequences.
This was performed with the use of RPSBLAST, a modified version of PSI-BLAST, to quickly scan a set of predetermined position-specific scoring matriceswith a protein query [19].
Title: PPI prediction

STRING (http://string.embl.de/) is a database of known and predicted protein interactions by using four sources: Genomic Context, (Conserved) Co-expression, High-throughput Experiments, and Previous Knowledge.
STRING currently contains the databases of 5,214,234 proteins from 1,133 organisms [20].
Title: Proteins location prediction

PSORTB (http://www.psort.org/psortb/) server was used to predict the cellular locations of HPs and then SOSUI server was used to find out whether the protein is soluble or trans-membrane in nature (http://bp.nuap.nagoya-u.ac.jp/sosui/sosui_submit.html).
Title: Detection of disulfide bridges

DISULFIND (http://disulfind.dsi.unifi.it/) server was used to predict the presence of any disulfide bond state between cysteine residues in the amino acid sequences of HPs.
Moreover, disulfide bridges play a key role in the stabilization of the folding process for many proteins.
We analyzed the data using this software.
The disulfide bridges are very important finding in the study of structural and functional properties of specific proteins [21].
Title: Protein structure prediction

(PS)2 (pronounced PS square) was used for the prediction of the tertiary structures of HPs (http://www.ps2.life.nctu.edu.tw/).
This method combined PSI-BLAST [22,23], IMPALA [24], and T-Coffee [25] by using an effective accord strategy in both target-template selection and target-template alignment.
Three dimensional structures were constructed further using the modeling package MODELLER [26,27,28].
The predicted structures obtained from the PS square were saved in the Protein Data Bank (PDB) formats.
Title: Active site prediction

Q-SiteFinder (http://www.modelling.leeds.ac.uk/qsitefinder/) was used to find out the ligand binding sites.
It works by finding clusters of probes and binding hydrophobic (CH3) probes to the protein with most favorable binding energy.
Q-SiteFinder requires uploading a PDB file or selecting one from the Protein Database.
Proteins are primarily scanned for ligands and it uses the interaction energy between the protein and a simple van der Waals probe to locate vigorously favorable binding sites [29].
We used this tool for evaluating these features including the active site in the desired sequence.
We analyzed the physiochemical properties of these HPs of cholera for the first time.
In Table 1 the physicochemical properties of HPs are tabulated.
Isoelectric point (pI) of the HP ranges from 4.62 to 9.78. pI is the pH at which the amino acid of protein tolerates no net charge and hence does not move in a direct current electrical field.
The determined pI will be handy as solubility is minimum and in an electro focusing system mobility is zero at pI.
Moreover proteins become stable and compact at isoelectric pH, for this reason computed pI will be helpful for developing a buffer system for purification by isoelectric focusing method.
At 280 nm, the extinction co-efficient of HPs ranges from 23295 to 62005 M cm computed by Expasy&#39;s Protparam instead of 276, 278, 279, and 282 nm.
The presence of high concentration of Cys, Trp, and Tyr indicates a higher extinction coefficient of HPs.
The quantitative study of protein-protein and protein-ligand interactions in solution can be done by using this computed extinction coefficients.
The instability index value of the HP was found to be ranging from 30.44 to 50.35.
It is predicted that a protein will be stable whose instability index is smaller than 40, a value above 40 predicts that the protein will be unstable [13].
Another parameter of structure identification of protein is instability index.
Proteins, gi|163644906, gi|163644912, and gi|163644916 were stable and others were unstable.
The instability index indicates an approximate stability of proteins in a test tube.
The AI is the relative volume of a protein occupied by aliphatic side chains (A, V, I, and L) and is considered as a positive factor for the raise of thermal stability of globular proteins.
The range of the AI for the HPs is from 64.14 to 82.92.
The proteins with very high AI may show stability in a wide temperature range where lower AI proteins are not thermal stable and show more flexibility.
The GRAVY of HPs is ranging from -0.304 to -0.633.
The better interaction of protein and water is occurring in low GRAVY.
The GRAVY value for a protein is calculated by adding the values of hydropathy of all the amino acids and dividing it by the number of residues in the sequence [14].
To study the functional analysis conserved domains were observed because conserved domains are functional units within a protein that act as building blocks in molecular evolution and recombine in various arrangements to make proteins with different functions.
The data are then used for putative functional annotation of protein query sequences based on matches to specific super-families history, identification of proteins with similar domain.
The proteins have been classified into particular families based on the presence of specific domains in the sequence [19].
In our study we used 6 HPs but found only 1 protein gi|84468567 possessing specific domains which were DNA_BRE_C super-family, Topoisomer_IB_N, DUF3946 domains and they were classified as super-families accordingly.
The presence of these domains in the HPs indicates that the protein might do the same function.
The domains of the HP gi|84468567 and their super-family is given by function in Tables 2 and 3.
Domains and families present in HPs were identified by the Pfam database research (Tables 4 and 5).
They are PhnA Zn ribbon, prokaryotic membrane lipoprotein lipid attachment site, Phage integrase family and integrase core domain.
To explain the protein functions involved in various cellular processes it is important to know the sub-cellular localization of that protein.
During the drug discovery process knowledge of the sub-cellular localization of a protein play a very significant role in target identification.
In our study, we have found two proteins gi|163644906, gi|184468567 are cytoplasmic as their best performing sites.
The remaining other protein localization was not found.
The server SOSUI differentiates whether the HPs are membranous or soluble.
No trans-membrane protein was found and all were soluble.
Moreover, DISULPHIND server revealed no disulphide bonds were present in any of those proteins which indicate that they were thermally unstable.
Moreover, disulfide bridges play a major role in stabilizing the folding process of many proteins.
Disulfide bridges are very important finding in the study of structural and functional properties of specific proteins [21].
For performing almost all the cellular functions the PPI are important.
Proteins often interact with one another in a mutually dependent way to perform a common function.
It is notable that translational factors interact among themselves to carry out the whole translation.
The function of protein is predictable from this based on their interaction with other proteins.
It is very rare that proteins bring out function with any interactions with other biomolecules.
For this reason, in this post genomic era PPI databases have turned as a most important resource for searching biological networks and pathways in cells [29].
The proteins gi|163644906 and gi|163644912 were found to have interaction with 2 proteins signal peptide peptidase SppA domain-containing protein and DSBA like thioredoxin domain containing protein.
gi|163644916 had interacted with 3 proteins such as IV conjugative transfersystem protein TraD &amp; TraI, putative type IV conjugative transfer system coupling factor.
gi|84468567 showed interaction with 6 proteins which were (1) ribosomal protein-alanine acetyl transferase, (2) recombination factor protein RarA, (3) ATP-dependent RNA helicase HrpA, (4) Zinc-binding domain-containing protein, (5) putative ATP-dependent helices, and (6) dihydroxy-acid dehydrates.
gi|84468557 protein interacted only one protein ISVch4 transposes.
Other HPs do not interact with any other proteins.
Fig.1 and Table 6 indicate the protein-protein interacting networks of HPs, which might have functions of their interacting proteins [30, 31].
PS square server (Fig.2) was used to determine the three dimensional structure of the HPs.
Out of 6 HPs, the PS square server could model only 2 proteins.
Due to low sequence identity, the other four proteins could not be modeled.
The server used templates to model those proteins which were tabulated in Table 7.
The location of ligand binding site identification on protein is important for a wide range of applications including structural identification, comparison of functional sites, molecular docking and de novo drug design.
Active site residues of the HPs are mentioned in Table 8.
This data of active binding site residues will give insight into identifying binding interactions and docking with specific ligand.
We have retrieved 6 HPs from NCBI database and determined their physicochemical properties and identified domains and families using various Bioinformatics tools and databases.
The three dimensional structure of those HPs were modeled (only 2) and their ligand binding sites were identified.
Among them we have found domains and families of only one HP, analysis showed that the domains and families are involved in DNA breaking-rejoining activities, integrase activity.
All of these features from our findings may be used to design new potential drugs against this infectious bacterium.
Protein-protein interaction of hypothetical proteins.
Three-dimensional structure of hypothetical protein by PS square.
Title: Physicochemical properties of hypothetical proteins of Protparam tool

Title: Identification of domains by CDD-BLAST

Title: Functional description of superfamilies of hypothetical proteins

Title: Families found in Pfam database

Title: Descriptions of Pfam families of hypothetical proteins

Title: Hypothetical proteins interacting with functionally important proteins

Title: Templates used by PS square server for modeling

Title: Residues involved in ligand binding sites predicted by QSITE finder
Title: In Silico Docking to Explicate Interface between Plant-Originated Inhibitors and E6 Oncogenic Protein of Highly Threatening Human Papillomavirus 18

The leading cause of cancer mortality globally amongst the women is due to human papillomavirus (HPV) infection.
There is need to explore anti-cancerous drugs against this life-threatening infection.
Traditionally, different natural compounds such as withaferin A, artemisinin, ursolic acid, ferulic acid, (-)-epigallocatechin-3-gallate, berberin, resveratrol, jaceosidin, curcumin, gingerol, indol-3-carbinol, and silymarin have been used as hopeful source of cancer treatment.
These natural inhibitors have been shown to block HPV infection by different researchers.
In the present study, we explored these natural compounds against E6 oncoprotein of high risk HPV18, which is known to inactivate tumor suppressor p53 protein.
E6, a high throughput protein model of HPV18, was predicted to anticipate the interaction mechanism of E6 oncoprotein with these natural inhibitors using structure-based drug designing approach.
Docking analysis showed the interaction of these natural inhibitors with p53 binding site of E6 protein residues 108-117 (CQKPLNPAEK) and help reinstatement of normal p53 functioning.
Further, docking analysis besides helping in silico validations of natural compounds also helped elucidating the molecular mechanism of inhibition of HPV oncoproteins.
Human papillomavirus (HPV) accounts for 5.2% of all cancers globally.
Besides cervical cancer HPV also causes a subset of anogenital, head and neck cancers [12].
Most of the cancer mortality in women worldwide is due to cervical cancer with an estimated 527,624 new cases and 265,653 annual mortality [3].
Among more than 200 HPV types, mostly HPV type 18 and 16 are the main cause of cervical cancer i.e., 15.7% and 62.6%, respectively [4].
Further, HPV 16 and 18 are also associated with penile cancer (63%-80%), vulva/vaginal cancers (80%-86%), anal cancer (93%), oropharyngeal cancers (89%-95%), etc.
[5].
Thus, these two HPV types are the main targets for designing anti cancer drugs.
The HPV onco-proteins E6 and E7, have been reported to interact with tumor suppressor proteins p53 and pRb respectively [6].
Both p53 and pRb act as negative regulator of cell cycle and inhibit G0-G1 and G1-S phase transitions.
These interactions apparently play important roles in the induction of cell immortality.
The importance of p53-mediated apoptosis has been recognized in terms of maintaining homeostasis and preventing neoplastic transformation.
E6 forms a ternary complex with p53 and E6 associated protein (E6AP) resulting in the degradation of p53 via ubiquitination pathway [7].
Although HPV has been known to be a causative agent for cervical cancer for more than three decades, the effective treatment against HPV infection is yet to be established [8].
Many natural plant origin compounds have been identified as promising sources of drugs for treatment and prevention of cancer in the recent years [910].
Indole-3-carbinol, an active ingredient of cabbage, broccoli, cauliflower, brussels sprouts, etc.
was reported with anti-estrogenic properties in cervical cells [11].
Ursolic acid having anti-mutagenic nature has the potential to induce apoptosis in tumor cells as well as to prevent malignant transformation of normal cells [12].
Resveratrol is a polyphenol isolated from the grapes skin, shown to alter both cell cycle progression and the cytotoxic response to ionizing radiation in cervical tumor cell lines [13].
Curcumin is cytotoxic to cervical cancer cells in a concentration dependent and time-dependent manner and the cytotoxicity was higher in HPV infected cells.
It down regulates both serine kinase AKT/nuclear factor κB pathway by sensitizing cancer cells [14].
It also down regulates the expression of HPV oncoproteins, resulting in loss of the transforming phenotype and the cessation of cellular growth.
Singh and Singh [15] explored other molecular mechanisms exerted by curcumin in cervical cancer cells, showing that it inhibits telomerase activity, RAS and ERK signaling pathway, cyclin D1, cyclooxygenase 2, and inducible nitric oxide synthase activity [15].
Ginger (active ingredient, gingerol) is a natural dietary component, which has antioxidant and anticarcinogenic properties and its supplementation has shown to suppress colon carcinogenesis [16].
Jaceosidin, a active ingredient of Artemisia argyi, reported to inhibit the binding between oncoproteins and tumor suppressors p53 [17].
Through the post-attachment heparan sulfate-independent effect, it is reported that carrageenan block HPV infection by preventing the binding of HPV virions to cells [18].
Due to the antiviral and antitumor properties of (-)-epigallocatechin-3-gallate (EGCG), it was found effective in HPV patients with infected cervical lesions [19].
EGCG also reported as an effective inhibitor of E6/E7 proteins due to its inhibitory effect on the growth of HeLa (HPV18 positive) and CaSki (HPV16 positive) cells in a time and concentration dependent manner [20].
Mahata et al.
[21] studied the effect of berberine on HPV16 and HPV18 positive cervical cancer cell line, and observed that it can effectively target both the host and viral factors responsible for development of cervical cancer through inhibition of viral oncoproteins E6 and E7 expression.
An in vitro study conducted by Karthikeyan et al.
[22] reported the radiosensitizing potential of ferulic acid (a natural phenolic acid) on human cervical cancer cell lines (HeLa and ME-180) [22].
Silymarin, an active ingredient contained in the seeds of the milk thistle plant, reported to inhibit cervical cancer cell [23].
Hu et al.
[24] evaluated the anti-tumor effect of dihydroartemisinin (DHA), an artemisinin derivative on HeLa and Caski cervical cancer cells and found that DHA treatment caused a considerable inhibition of tumor development [24].
The active compound of Withania somnifera i.e., withaferin A (WA) has reported to have antitumor, antiangiogenic and radiosensitizing activity [2526].
Through in vitro and in vivo study Munagala et al.
[27] demonstrated the effective inhibition of proliferation of cervical cancer cells by WA.
Further, they showed the down regulation of HPV E6 and restoration of p53 pathway by WA [27].
In the present study, we explicate the atomic interaction between plant-originated ligands and high risk HPV18 E6 oncogenic protein.
This study comprises of protein structure modeling of HPV18 E6 protein employing Phyre2 server followed by structural refinement and energy minimization by Yet Another Scientific Artificial Reality Application (YASARA) energy minimization server.
To analyze the molecular interaction between HPV18 E6 onco-protein with natural ligands, AutoDock4.2 tool was used in this study.
Title: Hardware and software

Dell Workstation with Windows operating system having 500 GB hard drive, 6 GB RAM and 2.26 GHz processor was employed in this study.
Different online resources and AutoDock 4.2 were used in this study.
Title: HPV18 E6 protein

As HPV18 E6 protein was selected as drug target in this study, its amino acid sequence (GenBank ID: NP_040310.1) was retrieved from NCBI (http://www.ncbi.nlm.nih.gov/).
Title: Prediction of protein structure and its validation

Phyre2 server [28] was employed for modeling of the tertiary structure of E6 protein followed by energy minimization using YASARA Energy Minimization Server [29].
Further the protein three dimension structure in pdb format was subjected to SCWRL4.0 software [30] for protein side chain modeling before docking.
Procheck [31], ProSA-web [32], and ProQ [33] server were used for assessing the model reliability which further verified by ERRAT server [34].
Title: Ligand preparation and protein-ligand docking

Chemical structures along with Chemical Abstracts Service (CAS) registry number of 12 natural compounds reported in literature (Fig.1) (artemisinin, WA, ursolic acid, ferulic acid, EGCG, berberin, resveratrol, jaceosidin, curcumin, gingerol, indol-3-carbinol, and silymarin), were retrieved from PubChem database (Table 1) [35].
Receptor molecules (HPV18 E6) was prepared in AutoDock 4.2 program [36] and docking studies were performed as per the standard methodology for protein-ligand docking used by Kumar et al.[37].
Title: Visualization

ADT tool and PyMol molecular graphics system (http://www.pymol.org) were used for visualizing the structure files.
Title: Protein structure prediction and validation

There are 158 amino acids in the protein sequence of HPV18 E6 protein.
As the experimentally determined tertiary structure of E6 was not available, Phyre2 server was used to predict its three dimensional structure.
During structure prediction, crystal structure of C chain of full-length HPV oncoprotein e62 in complex with lxxll peptide of ubiquitin ligase E6AP (PDB ID, 4GIZ) was taken as structural template by Phyre2 server.
There was query coverage of 89% and identities of 57% observed between target-template alignment.
Upon structural refinement of predicted structure by YASARA Energy Minimization Server, the total energy of refined structure was observed to be -19,732.64 kcal/mol (score, 0.07), whereas it was as 166,485.03 kcal/mol (score, -0.55) prior to energy minimization.
Upon Procheck analysis, 99.2% residues of the refined model (Fig.2A) were found in most favourable region and additional allowed regions and only 0.8% (Ser59) residue in situated generously allowed region, whereas not a single residue observed in the disallowed region of Ramachandran plot (Fig.2B).
The compatible Z score (Fig.2C) value was observed to be -4.46 by ProSA-web evaluation, which is quite well within the native conformations range of crystal structures [31].
The residue energy of the refined model were observed to be largely negative (Fig.2D).
The Protein Quality Predictor (ProQ) tool predict the LG score of 2.205 for HPV18 E6 protein model, implying high precision of the modeled structure as ProQ LG score of more than 1.5 is essential for signifying that a predicted structure is of fairly good quality [33].
The overall quality factor is predicted to be 91% by the ERRAT plot (Fig.2E) showing the well accuracy of predicted structure.
Overall, based on the above results, the reliability of the predicted model was suggested.
Title: Interaction study of HPV18 E6 with natural ligands through docking analysis

As all the natural compounds were observed to be interact with E6 protein in different conformations and binding energy, the conformation with lowest binding energy was selected for analysis (Table 2).
In case of HPV16 E6 protein, amino acid residues 113-122 (CQKPLCPEEK) were associated with p53 binding [38].
In our previous study we elucidated molecular interaction of HPV16 E6 proteins with natural inhibitors around these p53 binding site [37].
Upon BLAST [39] sequence comparison of E6 protein of HPV16 and HPV18, it was observed that 80% of the p53 binding residues, conserved in E6 of HPV18 (Fig.3).
Thus, on the basis of molecular interaction of p53 binding site residues with E6 protein, the active site residues i.e., 108-117 (CQKPLNPAEK) of HPV18 E6 protein were consider for docking analysis.
Fig.4 showed best docking conformation of E6 protein with twelve ligand molecules and the complete interactions were put in Table 2.
It was observed that all the ligands were interacting with E6 protein at p53 binding site residues.
WA was observed to bind with HPV18 E6 protein with lowest binding energy of -5.85 kcal/mol and the inhibition constant was found to be 51.35 µM.
It was found to interact with four amino acid residues of E6 (Glu116, Asn113, Asn122, and Ser140) by forming hydrogen bonds.
Next to WA, artemisinin bound with E6 protein with binding energy of -5.68 kcal/mol and inhibition constant of 68.22 µM.
It formed only one hydrogen bonds with the receptor at Leu112 residues.
Ursolic acid and ferulic acid also observed to inhibit E6 protein with good binding affinity i.e., binding energy of -5.31 and -5.18 kcal/mol, respectively.
Other six natural compounds indol-3-carbinol, resveratrol, jaceosidin, berberine, EGCG, and curcumin were observed to bind with the receptor (E6 protein) with binding energy range from -4.98 kcal/mol to -4.08 kcal/mol (Table 2).
Further, other two compounds silymarin and gingerol found to interact with the receptor with less binding affinity i.e., binding energy of -3.67 kcal/mol and -2.86 kcal/mol.
Few recent in silico studies are also been carried out on HPV showing the importance of computational approach in drug designing.
Muthukala et al.
[40] observed the inhibitory effect of quercetin compound against human cervical cancer cell line proteins through in silico docking analysis.
Kotadiya and Georrge [41] identified some putative drugs from natural products against HPV through in silico approach.
Samant et al.
[42] studied the molecular interactions of human immunodeficiency virus antiviral drugs against HPV18 E6 through in silico approach.
Mamgain et al.
[43] also observed the inhibitory effect of natural compounds such as colchine, curcumin, daphnoretin, ellipticine, epigallocatechin-3-gallate, etc.
against HPV16 E6 protein using molecular docking [43].
In our study, we have taken 12 natural compounds, reported to be used as an anti cancer agent against HPV [1112131415161718192021222324252627] for docking analysis and observed their molecular interaction against HPV18 E6.
All the natural compounds were found to interact with p53 binding site residues of HPV18 E6 onco-protein.
This interaction might disable E6 protein to bind with the host p53 protein helping correlate these natural compounds used as anti-cancerous agents to treat HPV infections.
For treatment of different cancers caused by HPV, various plant-originated compounds have been identified and tried as hopeful resources for cancer therapy.
Due to the encroachment in computational biology and bioinformatics, validation of those natural compounds is possible through computational approach.
The E6 protein of HPV6 and HPV11 (low risk HPVs) is not able to degrade p53 protein whereas in case of high risk HPVs (HPV16 and HPV18), the E6 protein able to inactivate p53 protein by inducing its degradation.
Thus, in order to design a novel drug against cervical cancer, the HPV18 E6 protein can be considered as a suitable target.
In this study, high throughput computational approach have been employed for modeling tertiary structure of E6 protein followed by docking using AutoDock 4.2.
This computational approach needs to be explored further in order to design novel drugs against cervical cancer from natural resources.
Chemical structure of natural compounds.
(A) Artemisinin.
(B) Withaferin A.
(C) Ursolic acid.
(D) Ferulic acid.
(E) (-)-Epigallocatechin-3-gallate.
(F) Berberin.
(G) Resveratrol.
(H) Jaceosidin.
(I) Curcumin.
(J) Gingerol.
(K) Indol-3-carbinol.
(L) Silymarin.
(A) 3D structure of predicted human papillomavirus (HPV) 18 E6 model.
(B) Ramachandran plot of predicted E6 model (The red, dark yellow, and light yellow regions represent the most favored, allowed, and generously allowed regions).
(C) ProSA-web Z-scores of all protein chains in Protein Data Bank (PDB) determined by X-ray crystallography (light blue) and nuclear magnetic resonance spectroscopy with respect to their length.
The Z-score of E6 was present in that range represented in black dot.
(D) Energy plot for the predicted E6 of HPV18.
(E) ERRAT plot for residue-wise analysis of homology model.
Sequence alignment results of E6 protein of human papillomavirus (HPV) 18 and HPV16 showing conserved p53 binding site residues.
Interaction profile of E6 with natural ligands.
(A) Withaferin A.
(B) Artemisinin.
(C) Ursolic acid.
(D) Ferulic acid.
(E) Indol-3-carbinol.
(F) Resveratrol.
(G) Jaceosidin.
(H) Berberin.
(I) (-)-Epigallocatechin-3-gallate.
(J) Curcumin.
(K) Silymarin.
(L) Gingerol showing interaction of ligands with the active site residues of E6 by forming hydrogen bonds.
Title: Natural compounds reported to use against HPV infection

Title: Polar contacts information from docking calculation between ligands and protein
Title: Editor&#39;s Introduction to This Issue

Title: Growing evidence has shown that miRNAs are involved in diverse biological processes, including the control of cell proliferation and differentiation.
Accordingly, a number of studies have reported that altered miRNA expression is related to tumor cell growth, invasion, and metastasis.
Recent studies have revealed that tumor-associated miRNAs are present in human plasma and serum, suggesting that plasma miRNAs can be useful non-invasive biomarkers.
In this issue, Dr. Yong Sung Kim&#39;s group (KRIBB) measured the plasma expression levels of three miRNAs to investigate the usefulness of miRNAs for gastric cancer detection.
DNA microarray techniques have been widely used in biological research to understand the pathobiology of diseases.
Microarray experiments also enable researchers to identify gene sets undergoing a response to a specific stimulus, as well as to analyze dynamic responses at the level of the gene regulatory network.
Prof. Sangsoo Kim&#39;s group (Soongsil University) has constructed an insulin-mediated regulatory network by using published gene expression microarray datasets of human skeletal muscle samples.
In this issue, based on a motif discovery method, they developed a regulatory network for insulin-mediated genes by integrating methods based on TF binding motifs and gene expression variations.
The development of transgenic biotechnologies has led to safety concerns regarding genetically modified organisms, particularly in food crops.
Ikyoung Choi&#39;s group (Seoul National University) demonstrated the success of NGS data in characterizing the genetically modified rice genome.
There are three interesting and informative application notes in this issue: &#39;A Database of Gene Expression Profiles of Korean Cancer Genome,&#39; &#39;VBioindex: A Visual Tool to Estimate Biodiversity,&#39; and &#39;A Frequency Profile of the ChromHMM Promoter States.&#39;
Title: Cell-Free miR-27a, a Potential Diagnostic and Prognostic Biomarker for Gastric Cancer

MicroRNAs (miRNAs) have been demonstrated to play an important role in carcinogenesis.
Previous studies revealed that miRNAs are present in human plasma in a remarkably stable form that is protected from endogenous RNase activity.
In this study, we measured the plasma expression levels of three miRNAs (miR-21, miR-27a, and miR-155) to investigate the usefulness of miRNAs for gastric cancer detection.
We initially examined plasma miRNA expression levels in a screening cohort consisting of 15 patients with gastric cancer and 15 healthy controls from Korean population, using TaqMan quantitative real-time polymerase chain reaction.
We observed that the expression level of miR-27a was significantly higher in patients with gastric cancer than in healthy controls, whereas the miR-21 and miR-155a expression levels were not significantly higher in the patients with gastric cancer.
Therefore, we further validated the miR-27a expression level in 73 paired gastric cancer tissues and in a validation plasma cohort from 35 patients with gastric cancer and 35 healthy controls.
In both the gastric cancer tissues and the validation plasma cohort, the miR-27a expression levels were significantly higher in patients with gastric cancer.
Receiver-operator characteristic (ROC) analysis of the validation cohort, revealed an area under the ROC curve value of 0.70 with 75% sensitivity and 56% specificity in discriminating gastric cancer.
Thus, the miR-27a expression level in plasma could be a useful biomarker for the diagnosis and/or prognosis of gastric cancer.
In the past decade, gastric cancer has become of the most common malignancies worldwide.
Despite significant improvements in screening and treatment technologies for gastric cancer, it remains the second most common cause of cancer-related deaths [1].
An accurate biomarker for detection of gastric cancer may reduce the cancer-related mortality.
Although conventional strategies for blood-based biomarker discovery have shown promise, the development of clinically validated cancer detection markers remains an unmet challenge for many common human cancers [2].
New approaches that can complement and improve on current strategies for cancer detection are urgently needed.
MicroRNAs (miRNAs) are a subset of non-coding RNA molecules (approximately 22 nucleotide in length) [3] that negatively regulate the protein expression of specific mRNAs by imperfectly base pairing together, resulting in protein translational repression of the target gene [4].
Growing evidence has shown that miRNAs are involved in a variety of biological processes, including cell proliferation, differentiation, and apoptosis [567].
Studies of miRNAs have been extended to many kinds of tumors [8] because altered expression of miRNAs has been demonstrated to play an important role in carcinogenesis, either by oncogenic [910] or tumor suppressor functions [1112].
For example, three miRNAs (miR-21, miR-27a, and miR-155) have been reported as oncogenic miRNAs.
MiR-21, the most commonly up-regulated miRNA in both solid and hematological tumor tissues [813], directly targets tumor suppressors such as PTEN phosphatase, actin-binding protein tropomyosin I and reversion-inducing-cysteine-rich protein with Kazal motif [91415].
MiR-27a is an oncogenic miRNA in gastric adenocarcinoma, targeting prohibition [16], and genetic variant of miR-27a is associated with its expression and increased risk of various cancer [171819].
MiR-155 is frequently overexpressed in many cancer tissues and targets tumor protein p53 inducible nuclear protein 1 and FOXO3a [2021].
Previously, studies revealed that tumor-associated miRNAs are present in human plasma and serum in a remarkably stable form that is protected from endogenous RNase activity [2223], and can be readily detected in these blood fluids [242526].
In addition, it was demonstrated that the oncogenic miRNA expression levels in plasma and saliva were decreased in cancer patients post-surgery [27282930].
These findings suggest that miRNAs have the potential to be useful diagnostic or prognostic biomarkers for cancer detection.
In this study, we evaluated the expression levels of these three miRNA variants in plasma samples from a screening cohort of 15 patients with gastric cancer and 15 healthy controls in Korean population.
The significant miR-27a expression levels found were further validated in an additional 73 paired gastric cancer tissues and 70 plasma samples from Korean population, using quantitative real-time polymerase chain reaction (qRT-PCR).
Title: Plasma and tissue samples

The plasma and tissue samples (including matched non-tumor tissues) from patients with gastric cancer were retrieved from the Chungnam National University Hospital, Republic of Korea.
All patients gave their written consent prior to sample collection.
The acquisition of clinical material was approved by the local institution review board.
These plasma samples were drawn from 35 patients (18 men, 17 women) with gastric neoplasm and 35 control individuals (18 men, 17 women) without neoplastic disease.
There was no significant difference in age between the neoplastic group (51.80 ± 11.88 years) and the control group (48.94 ± 11.86 years) (Supplementary Table 1).
The demographic characteristics of the subjects who provided the 73 paired tissue samples are summarized in Supplementary Table 2.
Title: Total RNA extraction from fresh tissues and plasma

Venous blood from all subjects was collected into sterile ethylenediamine tetracetic aci-coated vials.
Within 1 h, samples were centrifuged (2,000 ×g, 10 min), following which the plasma was removed, aliquoted, and stored at -80℃ until isolation of the circulating RNA.
Total RNA was isolated from 300 µL of plasma using the Ambion PARIS Kit (Ambion, Austin, TX, USA) according to the manufacturer&#39;s instruction.
Total RNA from tissue samples was extracted with the miRNeasy Midi Kit (Qiagen, Carlsbad, CA, USA), and finally eluted into 80 µL of elution solution according to the manufacturer&#39;s instructions.
The total RNAs isolated from plasma and tissues were stored at -80℃ until further use.
Title: miRNA quantification in plasma

The miR-21, miR-27a, and miR-155 expression levels were measured by using the TaqMan qRT-PCR analysis system (Applied Biosystem, Foster City, CA, USA).
The expression level of each miRNA was normalized to U6snRNA.
In brief, this assay entailed a 2-step qRT-PCR of miRNA of 19-25 nucleotides, priming with a stem-loop primer onto a longer cDNA that is amenable to amplification, and quantification by a TaqMan-based real-time PCR.
Ct was the number of cycles at which the fluorescence signal passed the appropriate threshold.
Expression levels were determined by using the delta Ct (ΔCt) method, where Ct represents the number of cycles in which the fluorescence signal passed the appropriate threshold.
Title: miRNA quantification in tissue

For the detection of miRNA, 2 µg of total RNA extracted from tissues was reverse transcribed into cDNA using M-MLV (Promega, Madison, WI, USA) with miRNA-specific stem-loop primers, as previously described [16].
The cDNA was used for the amplification of mature miR-27a and endogenous control (U6snRNA) by real-time PCR.
The real-time PCR was performed by using the SYBR Green PCR Mix (Bio-Rad, Hercules, CA, USA) with a CFX96 real-time PCR system (Bio-Rad).
The PCR cycles were as follows: initial denaturation at 95℃ for 0.5 min, followed by 40 cycles at 94℃ for 0.5 min, 56℃ for 1 min, 72℃ for 0.5 min.
The relative miRNA expression level was calculated using the delta Ct (ΔCt) method.
Title: Statistical analysis

The significance of miRNA expression level difference between patients with gastric cancer and cancer-free control subjects was analyzed using Student&#39;s t-test.
Determination of the receiver-operator characteristic (ROC), and the respective area under the ROC curve (AUC), and Kaplan-Meier survival analysis were performed using the R software (version 2.6.1).
Results with a p-value of less than 0.05 were considered significant.
Title: miR-21, miR-27a, and miR-155 expression levels in screening cohort

To select potential miRNAs for gastric cancer detection, we divided our samples into the 30 screening cohort and 40 validation cohort.
The expression levels of three well-known oncogenic miRNAs were evaluated in 30 plasma samples consisting of 15 patients with gastric cancer and 15 healthy controls.
Box plots for the quantitative fluorogenic real-time PCR values are shown in Fig.1.
The miR-21 and miR-155 expression levels were not significantly different between the neoplastic group and the control (p = 0.08 and p = 0.27) (Fig.1A and 1C).
However, the plasma miR-27a expression levels were significantly higher in the patients with gastric cancer (p = 0.002) (Fig.1B).
Although miR-21 expression level in plasma was tend to be increased in patient with gastric cancer compared to healthy control (p = 0.08), we selected miR-27a as a high priority for further analysis.
Title: miR-27a expression level in paired tissue samples

We examined the miR-27a expression level in 73 paired gastric cancer tissues to confirm the oncogenic expression of this miRNA variant.
The miR-27a expression level was elevated 2.8-fold in gastric cancer tissues compared with non-tumor tissues (p = 1.96e-06) (Fig.2A), and this miRNA was highly expressed in 61 out of 73 gastric cancer patient samples (80%).
The miR-27a expression levels were highly elevated in all stages compared with non-tumor tissues (Fig.2B).
We divided the patient samples into diffuse and intestinal types, based on the Lauren classification, because gastric cancer patients with diffuse-type tumors have a much poorer prognosis than those with the intestinal type.
We observed that the miR-27a expression levels were not significantly different between intestinal and diffuse types (p = 0.19; data not shown).
By ROC analysis, we observed that the optimal cut-off value for discriminating between patients and healthy individuals was 0.0067 for a miR-27a expression.
miR-27a had 73% sensitivity and 62% specificity in discriminating gastric cancer (Fig.2C).
Importantly, we observed a significant association between miR-27a overexpression and poor survival (p = 0.03) (Fig.2D), suggesting that miR-27a has potential to be used as a prognostic marker for patients with gastric cancer.
Taken together, the miR-27a expression pattern resembled that of an oncogenic microRNA and was associated with gastric cancer patient survival.
Title: MiR-27a expression level in validation cohort

To further validate the diagnostic value of miR-27a, its expression level was measured in additional plasma samples drawn from 20 patients with gastric cancer and 20 healthy controls.
The result of miR-27a expression data obtained from each 15 patients with gastric cancer and healthy control plasma samples used in screening cohort were also included in the validation cohort, because of limited sample size.
The qRT-PCR results revealed that the miR-27a expression level was highly elevated in the neoplastic group (p = 0.009) (Fig.3A).
In both early (I and II) and late stages (III and IV), the miR-27a expression levels were significantly higher in the group with gastric cancer (Fig.3B).
By ROC analysis, we observed the AUC value to be 0.70 with 75% sensitivity and 56% specificity in discriminating gastric cancer based on the cut-off value of 0.031 for miR-27a expression level in plasma (Fig.3C).
Our results suggest that the plasma miR-27a expression level may have potential as a non-invasive diagnostic biomarker for gastric cancer.
A diagnostic source that would provide high specificity and sensitivity may be of enormous benefit to cancer patients, particularly if the biosource could be obtained from non-invasive materials.
It was demonstrated that miRNAs are involved in multiple steps of carcinogenesis, either by oncogenic or tumor suppressor functions [31], and circulating miRNAs exist in the serum or plasma [293233].
More importantly, miRNAs have been detected in human serum and plasma in stable forms because of their short hairpin structures, encapsulation by microvesicles, and association with protein complexes [22233435].
Thus, the detection of any altered expression of tumor-specific miRNAs in saliva, serum, plasma, and urine may offer a promising approach for non-invasive gastric cancer detection.
In this study, the plasma expression levels of three miRNAs (miR-21, miR-27a, and miR-155) were measured in a screening cohort, and we observed that miR-27a expression was significantly elevated in gastric cancer samples compared with healthy control samples.
Our result is similar to previously reported results [36].
Next, we investigated whether plasma miR-27a could be released from gastric tumor tissues.
We observed that the miR-27a expression level was similar between gastric tumor tissues and plasma, suggesting that the plasma miR-27a expression level may reflect that of the gastric tumor tissues.
miR-27a is located on chromosome 19, and has oncogenic function in gastric adenocarcinoma, by targeting prohibition [16].
This miRNA variant is associated with lymph node metastasis and is overexpressed in gastric cancer [17].
In addition, miR-27a can regulate tumor suppress genes such as SPRY2, and FBW7 [3738].
SPRY2 can inhibit tumor growth and metastases by interfering with Ras/MAPK activation [37], and FBW7 acts as the substrate recognition component of a Skp1-Cul1-F-box-protein ubiquitin ligase that targets numerous oncoproteins for proteasomal degradation [38].
Growing evidence has suggested that miR-27a has an important role in gastric cancer development and progression.
To estimate its potential value for gastric cancer detection, we evaluated the plasma miR-27a expression level in a validation cohort consisting of 35 patients with gastric cancer and 35 healthy control samples.
The sensitivity for the validation cohort was estimated as 75% and the specificity as 56%.
Current diagnostic tools for gastric cancer, such as the serological markers carbohydrate antigen 19-9 and carcinoembryonic antigen have low specificity and sensitivity [36].
Our data indicates that the sensitivity and specificity of plasma miR-27a are better than that of the serological markers, although plasma miR-27a alone is not sufficient for accurate gastric cancer detection.
Thus, real improvement in the accuracy of gastric cancer detection still needs to be achieved by recruiting additional plasma miRNA markers.
In conclusion, plasma miR-27a could be a useful non-invasive biomarker for gastric cancer detection.
Additional studies using a large cohort of samples are needed to validate these results because our study had a small sample size.
Expression levels of three miRNAs in the screening cohort.
miR-21 (A), miR-27a (B), and miR-155 (C).
The expression level of each miRNA was normalized to U6snRNA.
miR-27a expression level in paired gastric cancer tissues.
(A) Box plot of miR-27a expression level (n= 73).
(B) MiR-27a expression level by TNM stage.
(C) The receiver-operator characteristic curve and area under the receiver-operator characteristic curve (AUC) value were estimated using R software.
The miR-27a expression level was normalized to U6snRNA.
(D) Kaplan-Meier analysis of patient survival based on miR-27a expression in gastric tumors.
The log-rank test was used to compare survival between the low- and high- miR-27a expression groups of patients on the basis of average miR-27a expression.
Plasma miR-27a expression levels in the validation cohort.
(A) Box plot of plasma miR-27a expression level.
(B) MiR-27a expression by TNM stage.
(C) Receiver-operator characteristic (ROC) curve analysis using the plasma miR-27a expression level.
The plasma miR-27a marker yielded an area under the ROC curve (AUC) of 0.70 with 75% sensitivity and 56% specificity in discriminating gastric cancer.
Title: The Construction of Regulatory Network for Insulin-Mediated Genes by Integrating Methods Based on Transcription Factor Binding Motifs and Gene Expression Variations

Type 2 diabetes mellitus is a complex metabolic disorder associated with multiple genetic, developmental and environmental factors.
The recent advances in gene expression microarray technologies as well as network-based analysis methodologies provide groundbreaking opportunities to study type 2 diabetes mellitus.
In the present study, we used previously published gene expression microarray datasets of human skeletal muscle samples collected from 20 insulin sensitive individuals before and after insulin treatment in order to construct insulin-mediated regulatory network.
Based on a motif discovery method implemented by iRegulon, a Cytoscape app, we identified 25 candidate regulons, motifs of which were enriched among the promoters of 478 up-regulated genes and 82 down-regulated genes.
We then looked for a hierarchical network of the candidate regulators, in such a way that the conditional combination of their expression changes may explain those of their target genes.
Using Genomica, a software tool for regulatory network construction, we obtained a hierarchical network of eight regulons that were used to map insulin downstream signaling network.
Taken together, the results illustrate the benefits of combining completely different methods such as motif-based regulatory factor discovery and expression level-based construction of regulatory network of their target genes in understanding insulin induced biological processes and signaling pathways.
Insulin resistance is a condition in which tissues, such as skeletal muscle, liver, and adipocytes, fail to respond to insulin, leading to type 2 diabetes mellitus.
Because skeletal muscle is the primary tissue of insulin-mediated glucose uptake [1], it would be important to study the insulin effects on gene expression in skeletal muscle and to identify the insulin-responsive target genes and regulatory genes.
DNA microarray techniques have been widely used in biological research by enabling comprehensive understanding of molecular and systematic mechanism [23].
In addition, microarray experiments enable researchers to identify gene sets undergoing a response to a specific stimulus as well as to analyze dynamic response at the level of gene regulatory network [4].
Especially, microarray experiments could be designed to study downstream effects of a specific stimulus or diverse stimuli on gene expression at the level of genomic scale, not a single genetic one [5].
Microarray techniques could be a power tool to discover regulatory factors as most of co-expressed genes are co-regulated by the same regulatory factors [6].
Moreover, because genes co-regulated by the same transcription factor (TF) commonly share binding sites for this TF [6], it would be worth to detect TF based on motif discovery method.
Although diverse motif discovery methods have been proposed and refined [78910], most of them are restricted to using human annotated position weight matrices (PWM).
On the other hand, iRegulon, a user-friendly Cytoscape plugin, provides approximately nine thousand PWMs that have been collected from various sources and from different species and enable to make regulatory network by linking them to candidate TF [11].
The other approach for pathway analysis is Genomica [12], which yields regulatory modules from gene expression based on probabilistic graphical models and Bayesian networks.
It has a benefit in directly mirroring expression of their target genes in a specific condition by comparing expressions between regulators and their target genes [12].
However, this approach requires looping over all regulators, taking too much time in big data.
Here, we tried to construct a regulatory network in order to understand the regulation of insulin-response genes.
We identified about 560 genes differentially expressed between before and after treatment in insulin sensitive individuals, and found that those genes act for insulin signaling biological process through Gene Ontology (GO) analysis.
Furthermore, we report a reliable regulatory network cont-rolling insulin-mediated genes by integrating motif-based approach using iRegulon and expression-based approach using Genomica; thereby present some benefits for the construction.
Title: Study samples

We downloaded and re-analyzed a previous published mRNA expression microarray data set measured on Affymetrix Human Genome U95A Array (Affymetrix, Santa Clara, CA, USA) from the Gene Expression Omnibus (GEO) database (GSE22309) [5].
This data was measured in human skeletal muscle cell of human individuals: 20 insulin sensitive, 20 insulin resistant, and 15 diabetic patients for before and after insulin treatment.
We used only the insulin sensitive group.
Total of 12,626 probes were monitored, and we considered genes which are protein-coding gene.
Title: Insulin-mediated genes

To select differential genes before and after insulin treatment in 20 insulin sensitive individuals, we carried out paired t test and calculated gene expression fold change.
Using local commands of R program, we selected insulin-responsive genes which were significantly dysregulated at p-value (paired t test) &lt; 0.01 (corresponding to FDR [13] ≤ 0.05) and fold change &gt; 1.2 (Fig.1).
We looked for enriched GO terms of the insulin responsive genes using DAVID Functional Annotation Tool [14].
Title: Search for candidate TFs based on regulatory motif and chromatin immunoprecipitation-sequencing

The regulatory factors for insulin-responsive genes were searched using the iRegulon app in Cytoscapev3.2.1.
iRegulon detects the TFs and their targets by scanning known TF-binding promoter motifs as well as the predicted motifs discovered from the Encyclopedia of DNA Elements (ENCODE) Project chromatin immunoprecipitation-sequencing data.
We set 20-kb upstream for the options &#34;Putative regulatory region,&#34; &#34;Motif rankings database,&#34; and &#34;Track rankings database.&#34;
Other options were taken as default.
We executed iRegulon and looked for TFs for each down and up regulated genes.
Title: Construction of regulatory networks based on expression value

With TFs searched by iRegulon, we tried to construct regulatory network modules which can explain situation of expressions of target genes based on expression level.
This construction was carried out using Genomica [12], which is an analysis and visualization tool for genomic data.
Among functions of Genomica, we used the &#34;Create a Module Network.&#34;
It identifies regulatory network modules from gene expression data based on probabilistic graphical models and Bayesian networks.
The options &#34;Max number of modules&#34; and &#34;Min experiments per context&#34; were set at 4.
Title: Insulin-targeted gene set

The expression data of 20 insulin sensitive individuals downloaded from GEO (GSE22309) were analyzed to find genes controlled by insulin.
In addition, the data contain expression values of transcripts before and after insulin treatment for each sample.
Among the total of 12,626 probes, we selected about 9,000 genes which are protein-coding gene.
With expression of each gene, we carried out paired t test between before and after insulin treatment and calculated fold change.
A total of 560 genes were identified by cutoff using t test p-value (padj &lt; 0.05) and fold change (FC &gt; 1.2) in insulin sensitive group (Fig.1).
Furthermore, we looked for enriched GO terms using DAVID functional annotation tool, and several biological process terms of the GO database were searched (padj &lt; 0.05).
As the resulting list of terms is redundant, the lists were prunedmanually (Table 1).
Most of the terms were compatible with insulin response and overlapped to GO terms which have been known from previous studies [5].
Title: TF discovery based on motif using iRegulon

We tried to discover the regulatory TFs for the insulin-treated gene set using iRegulon in Cytoscape.
For each down- and up-regulated gene set, we predicted 25 TFs of the target genes based on motif in upstream 20 kb of the target genes.
As shown in Fig.2, target genes are linked to multiple TFs.
A single TF targeted about 138 genes in average.
Title: Construction of regulatory network using Genomica

We tried to explain the expression variations of target genes with those of TFs identified by iRegulon.
Firstly, we checked correlation between expression of YY1, which is a predicted TF by iRegulon and known for an insulin signaling gene, and mean of those of target genes.
However, the correlation coefficient is rather low (Fig.3).
This prompted us to construct the TF network module to explain the expression variations of target genes using Genomica (Fig.4).
We found most TFs participated in regulatory network constructed by Genomica have been known to have relationships with insulin from previous studies [1516171819].
While iRegulon provides a list of potential regulatory TFs ignorant of their expression conditions, Genomica has the power to filter the list by constructing conditional hierarchical regulatory network where the expression variations of the target genes are explained by the expression conditions of the filtered TFs.
We describe a more reliable regulatory network controlling insulin-responsive genes by integrating completely different methods for constructing regulatory network.
Most genes have networks of interactions for appropriate biological functions.
Furthermore, regulators do not act alone.
Describing a combination of regulatory genes is important to more accurately explain their target genes and understand biological functions.
In this regard, construction of regulatory networks can give an insight in the biological interpretation of differential gene set between normal and complex diseases.
In order to understand insulin-responsive system, we tried to construct regulatory network with microarray mRNA expression data monitored before and after insulin treatment in skeletal muscle of insulin sensitive samples.
For the construction of a reliable network, we carried out integration of two different methods, one based on TF-binding motif and the other on expression variations.
Firstly, we defined insulin-mediated gene set using threshold of t test p-value and expression fold change, about 560 genes among total of 12,626 genes were identified.
GO enrichment analysis indicated that the gene set is enriched with the biological process terms related with insulin signaling pathway.
Especially, among those GO terms, &#34;response to hormone stimulus&#34; and &#34;trans membrane receptor protein tyrosine kinase signaling pathway&#34; have well known links directly to the insulin [8].
To understand regulation of those genes made up of the insulin signaling pathway, 25 TFs were predicted by iRegulon, based on the enrichment of TF-binding motif.
iRegulon can infer TFs of target genes, and visualize interactions between TFs and target genes with previously known evidences that what TFs bind to a target gene with sequence information.
In this case, expression values of target genes are not considered, and this regulatory network cannot explain how those regulators affect expression variation of target genes.
We tried to explain expression variation of target genes with those of TFs identified by iRegulon.
However, the expression variation of a TF did not resemble those of target genes.
It suggests that expression variations of multiple TFs should be considered collectively to understand those of target genes.
With the TFs predicted by iRegulon, we constructed a hierarchical regulatory network among them using Genomica, explaining the expression variation of the target genes.
As shown Fig.4, a combination of multiple TFs instead of a single TF explained the expression variations of target genes.
For example, MEF2A which is at the top of the regulatory TF network had the largest influence on the expression variations of the target genes, and YY1, the second highest ranking TF, did not influence as much as MEF2A.
However, YY1 affects to increase expression of target genes when MEF2A has low expression value.
In fact, MEF2A, YY1, SRF, PDX1, and NFATC4 are known for having relationships with insulin [1516171819].
Although expression of downstream TFs in the regulatory network is not perfectly fitted to that of their target genes, the construction based on expression data helps to understand more clearly regulatory network of target genes.
Altogether, in this paper, we presented a reliable regulatory network controlling insulin-mediated genes as well as the approach integrating two different methods, iRegulon and Genomica, may allow making a reliable regulatory network.
The volcano plot showing the transcriptional changes between before and after insulin treatment.
The -log10(p-value) of paired t test and log2 gene expression fold changes between before and after the treatment in insulin sensitive individuals were plotted on the x- and y-axis, respectively.
The genes colored by cyan are selected using the threshold of t test -log10(p-value) and log2(FC) of fold change [-log10(p-value) &lt; 2 (corresponding to FDR19 ≤ 0.05) and log2(FC) &gt; 0.2630344].
FC, fold change.
A representative regulatory network using iRegulon in Cytoscape.
The node with purple and green represent the up-regulated genes after insulin treatment and their predicted regulators (MEF2A, YY1, and SRF), respectively.
Genes linked to two or three regulators may be regulated by those multiple regulators.
The target genes for each transcription factor are portrayed by different edge colors.
The expression variations of a regulator YY1 and its target genes.
For each individuals, the expressions of the up-regulated target genes by insulin treatment were averaged and plotted on y-axis, and those of YY1 were plotted on x-axis.
The expressions of YY1 do not resemble those of their target genes (correlation coefficient=-0.5878721).
The regulatory network, upper section is a dendrogram for regulators&#39; expression and bottom section is a heat map for target genes&#39; expression, generated using Genomica, based on expression profiling.
The upper part of the figure represents a dendrogram for a combination of regulators and their expressions for each individual.
The lower part shows a heat map for the expression profiling of up-regulated target genes by insulin treatment (row and column indicate genes and individuals, respectively).
Title: The enriched GO terms of insulin-responsive genes
Title: Efficiency to Discovery Transgenic Loci in GM Rice Using Next Generation Sequencing Whole Genome Re-sequencing

Molecular characterization technology in genetically modified organisms, in addition to how transgenic biotechnologies are developed now require full transparency to assess the risk to living modified and non-modified organisms.
Next generation sequencing (NGS) methodology is suggested as an effective means in genome characterization and detection of transgenic insertion locations.
In the present study, we applied NGS to insert transgenic loci, specifically the epidermal growth factor (EGF) in genetically modified rice cells.
A total of 29.3 Gb (~72× coverage) was sequenced with a 2 × 150 bp paired end method by Illumina HiSeq2500, which was consecutively mapped to the rice genome and T-vector sequence.
The compatible pairs of reads were successfully mapped to 10 loci on the rice chromosome and vector sequences were validated to the insertion location by polymerase chain reaction (PCR) amplification.
The EGF transgenic site was confirmed only on chromosome 4 by PCR.
Results of this study demonstrated the success of NGS data to characterize the rice genome.
Bioinformatics analyses must be developed in association with NGS data to identify highly accurate transgenic sites.
Genetic engineering technology is widely used in the agricultural and plant biotechnology fields, ranging from the food and feed industries to bio-pharmaceuticals and cosmetics [12].
The history of genetically modified (GM) technology began with the discovery of plasmid DNA, where the plasmid could be transferred from one cell to another genome [3].
Scientists subsequently applied the basic plasmid vector system principle and developed recombinant DNA technology to create genetically engineered organisms.
Today, GM techniques have been applied to various research fields, including crop sciences, drug manufacturing, and animal husbandry.
The development of transgenic biotechnologies over the last 20 years has led to safety concerns regarding genetically modified organisms (GMOs), particularly in food crops and new pharmaceuticals, which are the most controversial issues.
Safety concerns regarding GMOs have resulted in research, debates, and ongoing public unease.
Therefore, the European Union (EU) and National Institutes of Health (NIH) in the United States proposed an authorization process in commercial GMO use; however, public apprehension for transgenic techniques remains uncertain and controversial [45678].
Generally, molecular characterization and identification of GMOs are performed using Southern blots and polymerase chain reaction (PCR) based detection followed by conventional sequencing methods [7].
However, these approaches are limited to evaluate whether the host genome has unintended sequence substitutions and indels [9].
Moreover, if sufficient genomic information is not available for the chosen comparative model species, it is difficult to detect the correct transgenic insert site location or sequence contamination of vector DNA [910].
Recent publications of GMO molecular characterizations reported the use of next generation sequencing (NGS) approaches as an effective means to detect the precise transgenic insert location [91112].
High-throughput DNA sequencing technologies and bioinformatics can be coupled with NGS to offer new possibilities in drawing genetic maps with feasible costs.
For these reasons, researchers have tested new approaches in the molecular characterization of GMOs using NGS technologies [91012].
Here, we examined transgenic insertion sites using paired-end whole genome re-sequencing data following Yang et al.with modifications [9].
Human epidermal growth factor (EGF) was inserted into GM rice cells, which could produce EGF safety without endotoxin derived from bacteria and was used as material for this study.
Deep sequencing was performed with the Illumina Hiseq2500 platforms (Illumina Inc., San Diego, CA, USA).
In this pilot study, we demonstrated the potential of NGS for examination of transgenic insertion loci and discuss some technical bottlenecks of this new method.
Title: GM rice samples

The GM rice event PJKS131-2 was transformed with the EGF inserted pJKS131 vector, produced by Natural Bio-Materials Inc. (Jeonju, Korea).
Taxonomically, the event PJKS131-2 was derived from Oryza sativa L. cv.
Dongjin.
The T-vector was transformed with rice callus as described by Chan et al.[13].
Transgenic rice calli were incubated with 50 mg/L of hygromycin B antibiotic (A.G. Scientific Inc., San Diego, CA, USA) for selection.
The GM rice callus samples were subjected to NGS and further validated by PCR amplification.
Title: DNA extraction and whole genome shotgun library and sequencing

The calli of GM rice event PJKS131-2 were collected and stored at -80℃.
Total genomic DNA was extracted using the CTAB method in liquid nitrogen.
Genomic DNA quality was evaluated by 0.5% agarose gel electrophoresis.
Following the quality check, genomic DNA was sheared with average 500 bp fragment sizes.
Truseq DNA PCR free Library Preparation Kit (Illumina Inc.) was used to construct the DNA library according to the manufacturer&#39;s protocol.
The quality of constructed DNA libraries was confirmed by the LabChip GX system (PerkinElmer, Waltham, MA, USA).
DNA libraries were sequenced with 150-bp paired-end sequencing using Illumina Hiseq2500.
Title: Transgenic insertion analysis

Initially, paired-end reads were filtered out by phred scores &lt; 20 and duplicate sequences were removed.
After filtration, DNA fragments were consecutively mapped against the rice reference genome (phytozome v9 [14]) and T-vector sequence (Supplementary Fig.1).
The transgene insertion types were classified by adaptation and modification of the analytical strategies reported in Yang et al.[9].
Fig.1 shows the workflow applied in this method.
Initially, all NGS reads were individually mapped to the rice reference genome and transgenic vector (types A and C in Yang et al.[9]).
Subsequently, these NGS reads were eliminated to conduct the following analyses.
NGS reads not classified as above were classified into the following two classes: one side of the NGS read matched the reference genome, (1) the other one matched to vector (type B in Yang et al.
[9]); or (2) one side of the NGS read exhibited both elements from the rice reference and transgenic vector (types D and E in Yang et al.[9]).
Title: Experimental validation of transgenic inserts

Each of the 13 combination primer sets was designed congruent with the transgenic insertion region orientation.
PCR was conducted using DNA polymerase (Solgent Co., Daejeon, Korea) following the manufacturer&#39;s instructions.
The reaction was performed under the following conditions: a pre-denaturation step at 95℃ for 5 min; denaturation at 95℃ for 60 s; 30 amplification cycles, including annealing at 60℃ for 45 s, and elongation at 72℃ for 120 s; and a final elongation at 72℃ for 5 min.
Title: Whole genome re-sequencing and mapping to discover the transgenic position

The transgenic GM rice site, PJKS131-2, was detected by performing whole genome re-sequencing using callus tissue.
Genomic DNA libraries were constructed with an average 500 bp and both ends were read with 150 bp paired-end sequencing methods.
A total length of raw sequencing reads were 29.3 Gb (~194.9 million reads), which showed ~72× coverage in the total read length (Table 1).
Following quality control processing, reads with average phred scores ≥ 30 were estimated at %71.5% (Table 1).
The types of mapped reads were classified by alignment of all NGS reads to the rice reference genome and transgenic vector sequences.
Fig.2 shows construction of the pJKS131 transgenic vector.
Reads were aligned on the cloning vector positions 8,500 bp to 10,500 bp, similar to transgenic insert locations.
Detailed mapping strategies were described in the Methods.
The transgene insertion site was identified by classifying reads where one end matched the host genome and the other end matched the vector sequences (i.e., types B, D, and E) mapped back to the rice chromosome and known vector sequences.
Eleven pairs of reads were identified on rice chromosome including chromosome 4.
The total mapped reads described above were compatible with the transgenic vector backbone sequences.
Title: PCR validation of mapping prediction

Thirteen PCR primers designed based on mapping direction validated the mapping results of 10 transgenic insert candidates.
PCR results confirmed the target EGF sequence was successfully inserted on rice chromosome 4 (Figs.3 and 4).
The remaining reads were concluded to be artifacts, because all matches were not detected with PCR.
Recent developments in NGS methods and accompanying bioinformatics tools have paved the way for ongoing genomics research widely used in the agricultural biotechnology field.
Consequently, several studies reported new approaches in GM crop safety assessment using NGS platforms [101112].
In our study, we investigated EGF inserted GM rice events using NGS technology and bioinformatics to test the potential uses of this new approach in molecular assessment of transgenic organisms.
Results were successful in differentiating NGS read types using in silico analyses from GM rice, PJKS131-2 and hypothetically, the outcome was acceptable in terms of read classification.
However, as a validation step, we experienced unexpected problems.
Consistent with mapping and aligning data, we considered all possible transgenic insertion directions on the rice chromosomes and designed PCR primers based on loci information.
Among the primers, except for locus specific primers on chromosome 4, results showed all matches were mismatches, which was caused by computational errors derived from analogous sequences between the rice genome and the transgenic vector.
Therefore, we concluded it is essential to develop more accurate algorithms based on the transformation vector.
In addition, it is important to note our experimental sample was collected from rice callus tissues, with Agrobacterium co-incubation and a plant cell suspension culture system.
Transgenic plant cell suspension culture system exhibits several advantages, including a low microorganism risk and chemical contamination, simple cell culture methods, economical facilities, and stable productivity.
However, it is difficult to obtain pure genomic DNA of the host plant without plasmid DNA mixing using the plant cell culture method.
We eliminated NGS raw reads mapped only against vector DNA (type C), however if raw reads contained too many vector backbone sequences, problems in further bioinformatics analyses would still occur.
Further studies are required with appropriate controls of GM plants in cell culture environments.
In the present study, we completed a proof-of-concept experiment to examine the molecular characterization of a recombinant-protein produced GM rice event using NGS methods.
New approaches have recently been reported to assess the development and release of GM crops, however these techniques are not popularized in the field of GM risk assessment.
However, previous studies in other disciplines have successfully established NGS, but for practical reasons, it has not been easy to apply this new method for testing GMOs.
NGS strategies largely depend on sample quality, amount of data, and subsequent bioinformatics analyses.
Therefore, it is critical proper guidelines to discovery transgenic site by NGS data matched and PCR test in the GMOs established and required.
Summary of the work-flow.
PCR, polymerase chain reaction.
Schematic drawing of the transgenic vector genome.
Red line represent the region of mapped reads.
MCS, multiple cloning site; RB, right border; LB, left border.
Polymerase chain reaction validation of transgenic site.
Transgenic position of epidermal growth factor (EGF) locus on the rice chromosome 4 and polymerase chain reaction (PCR) test to identify T-DNA junction sequence.
(A) The EGF is inserted on the position 31,104,341 of the chromosome 4.
(B) The bold with underline is T-DNA sequence of the vector 2,026-2,223 bp and the next bases is rice transgenic locus chromosome 4 (31107341-31107690) in the fragment amplified by PCR test primer1 (5&#39; TACCTGCATGCTGCGGTGAAG 3&#39;) and primer2 (5&#39;AGGGCTGTGTAGAAGTACTCGC 3&#39;).
Title: Whole genome sequencing summary
Title: A Database of Gene Expression Profiles of Korean Cancer Genome

Because there are clear molecular differences entailing different treatment effectiveness between Korean and non-Korean cancer patients, identifying distinct molecular characteristics of Korean cancers is profoundly important.
Here, we report a web-based data repository, namely Korean Cancer Genome Database (KCGD), for searching gene signatures associated with Korean cancer patients.
Currently, a total of 1,403 cancer genomics data were collected, processed and stored in our repository, an ever-growing database.
We incorporated most widely used statistical survival analysis methods including the Cox proportional hazard model, log-rank test and Kaplan-Meier plot to provide instant significance estimation for searched molecules.
As an initial repository with the aim of Korean-specific marker detection, KCGD would be a promising web application for users without bioinformatics expertise to identify significant factors associated with cancer in Korean.
Cancer is a complex disease with heterogeneous clinical behaviors developed by accumulation of multiple genetic or epigenetic alterations.
Several global research consortia have made great efforts to improve the understanding of cancer biology and the development of more effective cancer treatments [12].
Most treatment options for Korean cancer patients were established based on such western population investigations.
However, there are clear molecular differences showing different treatment effectiveness between Korean and non-Korean cancer patients [34].
Thus, it is very important to identify distinct molecular characteristics of Korean cancer patients.
Currently, numerous databases and analysis toolkits supporting cancer genomics studies have been reported [5678910].
These studies mostly support a database system for searching disease-associated genes or target drugs.
Although many researchers have tried to develop platforms to find molecular markers from genomics data, there are few suitable web-based resources that help researchers develop gene signatures associated with Korean cancer patients.
Collecting Korean cancer genomics data, comparing with other data obtained from non-Korean and estimating prognostic or predictive value of the genes or gene sets using proper statistical analyses may be a daunting task for many investigators, particularly clinicians and oncologists.
Here, we introduce a web-based initial repository, namely Korean Cancer Genome Database (KCGD), to help investigators in the efforts for searching prognostic signatures in Korean cancer patients.
The database contains the gene expression profile with clinical data obtained from more than 1,000 Korean cancer patients.
It is designed to be simple to search significant molecules, for which it is available for instant statistical survival analyses.
In addition, our database has gradually containing non-Korean datasets so that users can easily compare or validate newly identified molecules independently.
Title: Collection of datasets

We have collected and generated genomics data obtained from 1,304 Korean cancer patients collaborating with a number of hospitals in Korea, such as Seoul National University Hospital (liver), Chungbuk National University Hospital (bladder), Chonbuk National University Hospital (liver, bile duct, and colon), Yonsei University Severance Hospital (breast), Korea University Medical Center (stomach), and Kosin University Gospel Hospital (stomach).
The data were partially deposited to and freely available from the Gene Expression Omnibus of National Center for Biotechnology Information.
All datasets stored in the database were normalized using quantile normalization.
Detailed repository status was illustrated in Supplementary Table 1.
Title: Implementations

The system architecture consists of various software frameworks for robust activity.
Our system was mainly implemented with JAVA-based environment.
To provide user friendly and active interfaces, the ICEfaces (version 3.3.0, http://www.icesoft.org/) framework was used.
To store and handle the datasets, the MySQL database management system was used (version 5.5.11, http://dev.mysql.com).
Data queries on MySQL from JAVA are controlled by MyBatis, an XML-based SQL mapping framework (version 3.1.1, https://code.google.com/p/mybatis).
All statistical analysis methods were implemented using R (version 3.0.1, http://www.r-project.org) with Bioconductor plugins (version 2.12, http://www.bioconductor.org).
Calling R modules from JAVA is managed by the RCaller framework (version 2.1.1, https://code.google.com/p/rcaller).
All services are hosted on an Apache Tomcat web server (version 6.0.26, http://tomcat.apache.org).
A schematic diagram of the system architecture is shown in Supplementary Fig.1.
Title: Supported analysis methods

Our system currently contains statistical survival analysis methods for identifying a signature associated with cancer outcome and estimating its predictive value, i.e., the Cox proportional hazard model, log-rank test and Kaplan-Meier curves.
Detailed methodologies are available in the previous methodology paper [11].
In addition, the system determines the significance of a molecule using bar plots illustrating the landscape of intensities among patients and two or more group box plots with a p-value obtained from a two sample t test or ANOVA methods.
Title: Web-based bioinformatics tool

A platform to simply search and estimate statistical significance of molecules across various cancer types accessible to investigators without bioinformatics or statistics expertise are available at the KCGD website (Fig.1).
It supports dataset search, in which a user explores cancer genomics or epigenomics datasets in Korean stored in the database, and gene search, which determines statistical difference between cancer known subtypes or prognostic value of a molecule in which users are interested.
When accessing the gene search module, a number of accession identifiers including gene symbol, RefSeq (NM_* or XM_*), Entrez and protein accession number (NP_* or XP_*) can be selected, then the user input keyword(s) for search and click the search button.
Although the search criteria were arranged using small number of identifiers, most known molecule identifiers were incorporated in the database by full text indexing method, indicating any molecule identifiers were allowed to search by clicking &#34;anything...&#34; category.
Clicking the search button, the measurement data of expression or methylation are sought by the keyword and matched list with molecule information and thumb nail image of measurement intensities across cancers are displayed.
Next, by clicking a thumb nail image, the system try to estimate the significance of a molecule using the log-rank test with Kaplan-Meier curves and display bar plots to illustrate the landscape of intensities among patients and box plots with a p-value obtained from a two sample t test or ANOVA test (Fig.1).
With p-values by log-rank test or significance test with t test or ANOVA, the prognostic value of a molecule or its difference between cancer subtypes may be determined, respectively.
The entire analysis procedures of the system were described in Supplementary Fig.2.
Title: Statistics of datasets

Currently, we have created a database containing a total of 1,403 patient samples, among which 1,304 samples are Korean and the remained 99 samples are non-Korean patients for comparison.
Most data in the current database were created by gene expression profiling (n = 1,283, 91%) and the remained (n = 120, 9%) were by methylation profiling methods.
The vast majority of cancer tissues (n = 1,363, 97%) consists of liver (n = 431, 31%), breast (n = 365, 26%), stomach (n = 307, 22%), and bladder (n = 260, 19%), among which highly ranked three cancer types (i.e., liver, breast, and stomach) were known to be most frequently occurred in Korean.
In each cancer type, we also designated previously known sample subtypes to support assessment of difference of a molecule between cancer subtypes.
In addition, survival data of liver (n = 99) and bladder cancer patients (n = 165) were involved in the database to instantly estimate the prognostic value of a molecule in which users are interested (n = 264, 19%).
Supplementary Table 1 elucidated detailed statistics in KCGD database.
We have constructed the KCGD system, a web-based search platform, to assist investigators in estimating predictive value of interested genes and in identifying molecular signatures from Korean cancer genomics data.
As an initial repository with the aim of Korean-specific marker detection, KCGD always open to collaborate with any research investigators or clinicians in Korea and is ready to share Korean cancer genomics data with clinical information to solve the critical problems in the biomedical fields.
KCGD is a growing database: although current database mainly contains gene expression with clinical data, the database is ready to handle the RNA-seq data, another gene expression profiling method, or any continuous numeric intensity data (i.e., methylation or genomic variation).
In addition, for direct comparative genomic analysis with non-Korean data, we are trying to collect other genomic or epigenomic data from several public database including the Cancer Genome Atlas consortium and comparative browsing would be implemented in near future.
Therefore, we suggest that KCGD may be one of the best choice as a co-work partner when users try to discover significant novel factors associated with genomic studies of Korean cancer patients.
Snapshot of the Korean Cancer Genome Database (KCGD) webpage.
The system supports dataset search, in which a user explores cancer genomics or epigenomics datasets in Korean stored in the database (right upper panel), and gene search, which determines statistical difference between cancer known subtypes or prognostic value of a molecule (right lower panel).
Title: VBioindex: A Visual Tool to Estimate Biodiversity

Biological diversity, also known as biodiversity, is an important criterion for measuring the value of an ecosystem.
As biodiversity is closely related to human welfare and quality of life, many efforts to restore and maintain the biodiversity of species have been made by government agencies and non-governmental organizations, thereby drawing a substantial amount of international attention.
In the fields of biological research, biodiversity is widely measured using traditional statistical indices such as the Shannon-Wiener index, species richness, evenness, and relative dominance of species.
However, some biologists and ecologists have difficulty using these indices because they require advanced mathematical knowledge and computational techniques.
Therefore, we developed VBioindex, a user-friendly program that is capable of measuring the Shannon-Wiener index, species richness, evenness, and relative dominance.
VBioindex serves as an easy to use interface and visually represents the results in the form of a simple chart and in addition, VBioindex offers functions for long-term investigations of datasets using time-series analyses.
The Convention on Biological Diversity (CBD) defines biodiversity as &#34;the variety of life on Earth and the natural patterns&#34; [12].
CBD has underlined the importance of abundant biodiversity in the sustainable use of natural resources [3].
Moreover, maintaining biological coexistence for all organisms on earth, including animals, plants, and microbes, is imperative for human welfare and quality of life [4].
Thus, many researchers use quantitative measures of biodiversity in an attempt to sustain and restore ecosystems [5].
Biodiversity is generally measured by statistical indices based on the emergence and/or existence of various organisms such as plants, animals, and microbes; these indices can provide quantitative criteria with increased confidence for biodiversity [67].
Of the statistical indices, Shannon-Wiener index [8], species richness, and evenness [9] have been widely used as the gold standards, in spite of traditional methods.
The Shannon-Wiener index is based on information theory and is measured using the concept of entropy, and it is applied to estimate the degree of species diversity in a region [10].
Species richness and evenness measure the number of different species per sample and the relative abundance of the different species, respectively [11].
For measuring biodiversity, there are a few non-commercial tools, such as the Vegan package in R [12], EstimateS [13], and the web-based Shannon Entropy Calculator (http:// www.changbioscience.com/genetics/shannon.html).
Although they are highly useful, some of them are difficult and complicated for biologists to use, especially some programs involving R package.
While the web-based tools are easy to use, they are not suitable for personal and/or big data sets.
Furthermore, some programs do not serve any function for long-term investigations of data sets using time-series analyses for biodiversity.
We developed the VBioindex tool to aid scientists in evaluating the Shannon-Wiener index, species richness, and evenness.
Additionally, this program depicts relative dominance for each species.
As VBioindex supports biodiversity analyses for a given period (e.g., for a month, quarter, or year), users can easily analyze a variety of biodiversity data while applying a time-series analysis.
VBioindex supports four statistical indices-the Shannon-Wiener index, species richness, evenness, and the relative dominance of species-that are widely used as important criteria to evaluate ecosystems.
Title: The Shannon-Wiener index (H&#39;) was designed to predict the next letter of communication in a random set of data and to quantify entropy in terms of uncertainty or information content [10].
It has been widely applied in a variety of fields, such as biology and ecology.
H&#39; is measured using entropy with the proportion of individuals (n) for total emerged species (s) [10].
pi=ni∑i=1sni H′=-∑i=1spilogpi

Title: Species richness (DMg), known as Margalef&#39;s richness index, is a measure of the variety of species in a given area or sample [10], which is calculated based on emerged species (s), and the individuals (n) of total emerged species [14].
DMg=s-1log∑i=1sni

Title: Species evenness (E) is the degree of the distribution of individual organisms among species [15], which is measured using H&#39; and the log of emerged species (s) [16].
E=H′logs

Title: Relative dominance (RD) is the frequency of individuals of a species, which generally presents the degree of dominance for a species as a percentage [17].
RDi=ni∑i=1sni×100

VBioindex is implemented by clicking a few buttons, such as Open, Save, and some selectable radio buttons (Fig.1A).
An input data file is accepted as a tab-delimited file, labeled with the date, species name, and number of individuals (Fig.1B).
The results of the four methods are graphically represented charts with a tab-delimited text output file (Fig.1C).
Along with presenting these biodiversity indices, VBioindex allows time-series analyses at a monthly, quarterly, and yearly basis for all statistical methods, the functions of which are useful in analyzing long-term species diversity.
VBioindex is programmed in Java language (higher than version 1.7), and it can be used in all operating systems in which a Java Virtual Machine has been installed.
Statistical indices evaluating ecology, such as the Shannon-Wiener index, species richness, evenness, and relative dominance, are considered important criteria to sustain and restore ecosystems.
Although there are a few programs for these indices, some programs are inconvenient to use and do not support various indices.
We developed VBioindex, a user-friendly program that is capable of measuring the Shannon-Wiener index, species richness, evenness, and relative dominance.
In addition, VBioindex offers options with which a user can analyze a variety of biodiversity data while applying a time-series analysis.
VBioindex outputs a tab-delimited text file with visual charts and users can use the output text file with links to other statistical programs for a variety of ecological studies such as comparative analyses between many datasets.
VBioindex is publically accessible at http://sourceforge.net/projects/vbioindex/.
Implemented VBioindex Frames: Mainframe (A) of VBioindex, input sample data form (B), and visual output charts (C) for four indices.
Users can see the information such as contents and values, by waving the cursor over one of the randomly colored bars.
Title: Role of Chromosome Changes in Crocodylus Evolution and Diversity

The karyotypes of most species of crocodilians were studied using conventional and molecular cytogenetics.
These provided an important contribution of chromosomal rearrangements for the evolutionary processes of Crocodylia and Sauropsida (birds and reptiles).
The karyotypic features of crocodilians contain small diploid chromosome numbers (30~42), with little interspecific variation of the chromosome arm number (fundamental number) among crocodiles (56~60).
This suggested that centric fusion and/or fission events occurred in the lineage, leading to crocodilian evolution and diversity.
The chromosome numbers of Alligator, Caiman, Melanosuchus, Paleosuchus, Gavialis, Tomistoma, Mecistops, and Osteolaemus were stable within each genus, whereas those of Crocodylus (crocodylians) varied within the taxa.
This agreed with molecular phylogeny that suggested a highly recent radiation of Crocodylus species.
Karyotype analysis also suggests the direction of molecular phylogenetic placement among Crocodylus species and their migration from the Indo-Pacific to Africa and The New World.
Crocodylus species originated from an ancestor in the Indo-Pacific around 9~16 million years ago (MYA) in the mid-Miocene, with a rapid radiation and dispersion into Africa 8~12 MYA.
This was followed by a trans-Atlantic dispersion to the New World between 4~8 MYA in the Pliocene.
The chromosomes provided a better understanding of crocodilian evolution and diversity, which will be useful for further study of the genome evolution in Crocodylia.
Crocodilians (Crocodylia) are extant large-sized reptiles found throughout the tropics in freshwater lakes, rivers, and oceans.
Crocodilians diverged from birds, their closest living relatives as Archosauromorpha more than 240 million years ago (MYA).
Ancestral crocodilian species first appeared in the fossil record around 80 MYA during the Late Cretaceous [1].
Extant crocodilians contain 25 species, classified into two families: Alligatoridae (comprising genera: Alligator, Caiman, Paleosuchus, and Melanosuchus) and Crocodylidae (comprising genera: Gavialis, Tomistoma, Mecistops, Osteolaemus, and Crocodylus) [2].
These crocodilians share both ancient and recent morphology as well as ecological characters [1].
The understanding of the phylogenetic history and biogeography in this lineage has provided a fascinating study for evolutionary biologists.
A phylogenetic approach for crocodilians was originally conducted based on fossil records.
These suggested that the Indian gharial (Gavialis gangeticus) was a basal crocodilian [1]; however, molecular datasets indicated that Alligator diverged from the remaining crocodilians around 50~70 MYA in the Tertiary, of which Crocodylus are the most recent group (Fig.1) [3456789101112131415161718192021222324].
This suggested that Alligator was the most ancient crocodilian; however, relationships among living species of Crocodylus remain poorly resolved, since they radiated into various species around 9~13 MYA during the middle of the Miocene [16].
These issues have received high priority in modern systematics by combining many datasets to determine the true groupings and divergences among them.
One possibility is the level of different chromosome constitution, though crocodilians exhibit slow divergence of karyotype among living species.
However, various karyotypic forms in the lineage are probably the source of reproductive isolation, which eventually radiated into species diversity.
Remarkably, chromosomal investigation of 23 crocodilians revealed karyotypic features: small diploid chromosome numbers (30~42), the predominance of a few large chromosomes, and the absence of dot-shaped microchromosomes whose centromere positions are undetectable [1920].
This contradicts with the karyotypes of birds (the sister group of Archosauromorpha), and the relative lineage of turtles, which contain a small number of macrochromosomes and many indistinguishable microchromosomes [2025].
The increase of large bi-armed chromosomes and the presence of small chromosomes probably correlate with the reduction of medium acrocentric chromosomes and the absence of microchromosomes, respectively, in these lineages [2526].
The chromosome arm number or fundamental number (NF, bi-armed chromosome: metacentric or submetacentric = 2; uni-armed chromosome: acrocentric or subtelocentric = 1) of all crocodilian karyotypes shows little interspecific variation (56~60), and no sex chromosome heteromorphism has been recognized in any species [192027].
These characteristics of crocodilian karyotypes indicated that very few chromosomal rearrangements have occurred in Crocodylia, except for centric fusion and/or fission events.
To delineate the role of chromosomal rearrangement, which also plays an evolutionary force in crocodilian diversity and evolution, the karyotype data and phylogeny were summarized as (1) variation of chromosome constitution within crocodilians compared to other sauropsids (birds and reptiles), (2) current phylogenetic history of crocodilians from molecular data and paleontological perspective, and (3) correlation of chromosomal rearrangements in each lineage and phylogeny in crocodilians.
Sauropsids (birds and reptiles) number around 20,000 species, comprising large group diversity in birds (9,900 species) and squamate reptiles (9,700 species), but small diversity in turtles (340 species) and crocodilians (25 species) [2].
It is possible that the species richness of birds and squamate reptiles correlates with the large karyotype variability.
By contrast, turtles and crocodiles show low chromosome constitution variability, although the correlation between chromosome changing rate and species number is found in turtles, but not in crocodiles [20282930].
This probably results from the role of sex chromosomes as all crocodilians exhibit temperature sex determination, but both genetic sex determination (homomorphic and heteromorphic sex chromosomes) and temperature sex determination are found in other sauropsids (except for tuatara), leading to high speciation rate and species diversification [31].
The crocodilian karyotype contains few large and many small chromosomes, and there is an absence of dot-shaped microchromosomes [1920].
On the contrary, the karyotypes of most squamate reptiles, tuatara, turtles, and birds contain macro- and microchromosomes [20].
Many chromosomal rearrangements might, therefore, be the result of the appearance or disappearance of microchromosomes, which reduced whole-chromosome homology among sauropsid chromosomes.
The draft genome assemblies of chicken (Gallus gallus), Anolis lizard (Anolis carolinensis), Chinese alligator (Alligator sinensis), American alligator (Alligator mississippiensis), saltwater crocodile (Crocodylus porosus), Indian gharial (G. gangeticus), Burmese python (Python molurus bivittatus), and king cobra (Ophiophagus hannah) provided new perspectives on the comparative genomics of Sauropsida, which in turn facilitated extensive comparisons between genomic structures at a molecular level [323334353637].
Recent comparative gene mapping of several sauropsid species (Pelodiscus sinensis, Testudines; Crocodylus siamensis, Crocodilia; Gekko hokouenesis, Lacerta agilis, Elaphe quadrivirgata, Varanus salvator macromaculatus, Leiolepis reevesii rubritaeniata, Pogona vitticeps, and A. carolinensis, Squamata) with those of chicken revealed the extensive homology between avian and reptilian chromosomes, and suggested that the common ancestor of amniotes may have had many microchromosomes, whose linkages have been conserved among chickens and reptiles [252633, 38394041424344].
Comparison of the linkage homology between Archosauromorph and turtle chromosomes revealed that the macro- and microchromosomes of turtles are true counterparts of those of chicken [2526].
However, the comparative cytogenetic map of the Siamese crocodile (C. siamensis), i.e., the first cytogenetic map in the crocodilian lineage, revealed that genetic linkages of large chromosomes were also conserved in blocks on chromosome arms that corresponded to chicken macrochromosomal arms and/or entire macrochromosomes.
The five largest bi-armed chromosomes of crocodile are derived by combinations of chromosome arms that differ from the chicken karyotype, while chicken microchromosomal genes are all localized to small chromosomes of the crocodile.
This suggested that the Siamese crocodile karyotype resulted from two events that occurred in the crocodilian lineage: (1) centric fissions of bi-armed macrochromosomes in the ancestral Archosauromorph-turtle karyotype, followed by centric fusions between acrocentric macrochromosomes (new combination), leading to large chromosomes in crocodilians, and (2) repeated fusions of microchromosomes, which resulted in their disappearance and the appearance of a large number of small chromosomes [26].
Notably, chromosome size-dependent genomic compartmentalization is often found in birds and turtles but not in squamate reptiles, such as microchromosome-specific centromere repetitive sequences [4546474849505152].
This implied that the homogenization of centromeric repetitive sequences between macro- and microchromosomes did not predominantly occur in birds and turtles.
However, no chromosome sized-specific centromeric repetitive sequences are found in crocodiles [21].
This suggested that chromosomal size-dependent genomic compartmentalization which is supposedly unique to Archosauromorph and turtles was probably lost with the disappearance of microchromosomes, followed by the homogenization of centromeric repetitive sequences among chromosomes in the crocodilian lineage after their divergence from birds around 240 MYA.
Diploid chromosome numbers in crocodilians range from 30 to 42; most are 32 (Fig.1) [1920].
Most karyotypes in crocodilians contain 4~5 large bi-armed chromosomes, 20~22 small bi-armed chromosomes, and 4~8 uni-armed chromosomes.
However, genome rearrangements among members in the order rarely occurred, giving all crocodilians a fundamental number of 56~60, which suggested that the ancestral crocodilian karyotype was highly conserved [192021222326].
The chromosome number in Alligatoridae (Alligator 2n = 32, and Caiman, Melanosuchus, and Paleosuchus 2n = 42) is more varied than in Crocodylidae (Gavialis and Tomistoma 2n = 32, Mecistops 2n = 30, Osteolaemus 2n = 38, Crocodylus 2n = 30~34).
Karyotypes of Caiman, Melanosuchus, and Paleosuchus show a large number of chromosome arms (NF = 60) [1920], suggesting that multiple centric fissions occurred after they diverged from Alligator.
However, the largest chromosome variation is found in Crocodylus.
The diploid chromosome number of 4 from 12 species (C. porosus, C. siamensis, Crocodylus palustris, and Crocodylus rhombifer) differs from 32, despite having the same fundamental number (58) (Fig.1).
This suggests that centric fusion/fission played a crucial role in the process of chromosomal rearrangements, leading to the formation of ordinal radiation in the lineage of crocodilians.
Crocodilian systematics has been discussed at higher-level relationships based on morphological, paleobiogeography, and molecular data [1, 3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,53,54,55].
However, no study has completely explained crocodilian relationships with their distribution and ecology.
Most molecular crocodilian phylogeny contains 23 species (Fig.1).
The use of complete mitochondrial (mt) genome sequences and the mixture of several molecular loci (nuclear and mt genes) obviously provided the solution for at least three major conflicts of crocodilian clustering: (1) phylogenetic position of G. gangeticus and Tomistoma schlegelii; (2) relationship of Mecistops cataphractus and Crocodylus; and (3) position of C. siamensis, C. porosus, and C. palustris.
Firstly, the position of G. gangeticus and T. schlegelii was controversial, since G. gangeticus was considered to be Gavialidae at the basal position of Crocodylia based on morphological evidence [153].
The molecular phylogenetic study of complete mt genome sequences and the mixture of several molecular loci clearly showed that G. gangeticus was a sister-taxon with T. schlegelii, and formed the sister group with the traditional Crocodylidae (Fig.1) [8910111213141617].
This agreed with molecular sequence-based analyses of the CR1 retrotransposon which suggested that Alligatoridae (alligators and caimans) were sisters to all other crocodilians, and supported the G. gangeticus-T. schlegelii clade [56].
Therefore, only the two families (Alligatoridae and Crocodylidae) are now recognized to be within Crocodylia.
The most basal species within Crocodylus and relative species (Osteolaemus tetraspis) is the African sharp-nosed crocodile, M. cataphractus (formerly called Crocodylus cataphractus) [57].
Fossil evidence related to M. cataphractus appeared in the Miocene and Pliocene in Africa [585960], and implied that M. cataphractus was grouped within Crocodylus.
However, molecular datasets suggested two relative species of two African crocodiles, M. cataphractus and O. tetraspis, being a sister group to the true crocodiles (Crocodylus) [4891011121314151617].
The last conflict issue concerned the position of C. siamensis, C. porosus, and C. palustris.
In terms of molecular data, a complete mt genome of C. siamensis (DQ353946) showed it as a sister to C. porosus [24], whereas another mt genome sequence of C. siamensis (EF581859) related it to C. palustris [17].
Nonetheless, a molecular study based on the mixture of several molecular loci found in many C. siamensis individuals clearly showed that C. siamensis was sister to C. palustris and that C. porosus was sister to both species [16].
This agreed with karyological data that showed 2n = 30 for C. siamensis and C. palustris, and 2n = 34 for C. porosus (Fig.1) [1920212226].
The Siamese crocodile sample sequenced by Ji et al.
[24] was probably a hybrid between C. siamensis × C. porosus, frequently found in captivity in the crocodile industry [61].
The Siamese crocodile sample sequenced by Srikulnath et al.
[17] was confirmed by karyotyping, and showed a chromosome number 2n = 30, a true C. siamensis.
However, the relationship within Crocodylus remained unclear because of the position of Indo-Pacific crocodilians (Fig.1).
A molecular study based on complete mt genome sequences strongly indicated two major groups: (1) The Indo-Pacific species and (2) The African-New World (America) species.
A molecular study with the mixture of several molecular loci comprising nuclear and mitochondrial genes suggested that the African-New World species grouped with C. porosus, C. palustris, and C. siamensis and was a sister clade with the group of Crocodylus mindorensis, Crocodylus novaeguineae, and Crocodylus johnstoni (Fig.1) [16].
Multiple lines of evidence from biogeography and karyotypes have provided promising data to extensively discuss their evolutionary history.
According to the chromosome constitution of crocodilians, it would appear that chromosome changes evolved in parallel with the crocodilian relationship.
Alligatoridae, which appeared in the early Tertiary, are considered to be the most primitive group in extant crocodilians [1].
There are two major crocodilians in Alligatoridae (Alligator and Caiman + Melanosuchus + Paleosuchus) generally found in the New World (America).
This agreed with karyological data, which showed different karyotypic features in both chromosome number and fundamental number.
Two Alligator species (A. mississippiensis and A. sinensis) still have the same karyotype form (2n = 32), but the karyotype of Caiman, Melanosuchus, and Paleosuchus, a sister clade of Alligator contains a higher chromosome number (2n = 42): a lower number of bi-armed chromosomes and a higher number of uni-armed chromosomes.
This suggests that multiple centric fission occurred at the split of Alligator and Caiman + Melanosuchus + Paleosuchus, around 50~70 MYA before the middle of the Eocene (Fig.1).
This is more likely than an alternative explanation of the presence of centric fusion, because the karyotype of most crocodilians also exhibits 2n = 32.
Interestingly, the total chromosome numbers of Caiman, Melanosuchus, and Paleosuchus are the same, but Paleosuchus contains a lower number of bi-armed and a higher number of uni-armed chromosomes, resulting in different fundamental numbers (Fig.1).
This suggests that Caiman + Melanosuchus and Paleosuchus evolved independently through at least one pericentric inversion or centromere positioning event, leading to ordinal radiation in two different lineages.
Two Alligator species (A. mississippiensis and A. sinensis), which exhibit the same karyotype form of 2n = 32, spread over more than one biogeography.
A. mississippiensis is restricted to the Southeastern United States, whereas A. sinensis is restricted to China.
Both species shared a common ancestor around 20~60 MYA (Fig.1) [116].
This suggested the possibility of a geographic isolation process that played an important role in their evolutionary history.
An overland route (presumably through Beringia) is the shortest distance from North America to Asia.
Alligator probably moved to Asia during the warmer climate of the early Tertiary [16], based on the limitation of temperature and salt water tolerance [626364].
On the contrary, A. sinensis is the most cold-adapted of all living crocodilians, surviving the winter in burrows [65].
It might have moved into China during dramatic temperature changes.
However, there was no evidence to support this idea until alligator fossils were found in Alaska [1].
The ancestral karyotype of crocodile is still retained in Crocodylidae.
The karyotype of G. gangaticus (2n = 32) which resided in South-Asia, is also highly similar to that of Alligator [1920].
However, the karyotype of T. schlegelii (2n = 32), distributed around South-East Asia, contains lower numbers of bi-armed chromosomes and higher numbers of uni-armed chromosomes, and the fundamental numbers of these two species are different.
This suggests the presence of pericentric inversion or centromere positioning, which probably occurred in T. schlegelii after it diverged from G. gangaticus around 15~35 MYA in the late Tertiary.
In Crocodylus and relative species (M. cataphractus and O. tetraspis) lineage, the karyotype of M. cataphractus showed a lower total chromosome number with higher bi-armed chromosomes than those of T. schlegelii, despite having the same fundamental number (58), suggesting that the increase of bi-armed chromosomes is the result of centric fusion among the ancestral types of acrocentric chromosomes (Fig.1).
By contrast, the karyotype of O. tetraspis showed a higher total chromosome number with higher uni-armed chromosomes, but lower bi-armed chromosomes than those of G. gangaticus, T. schlegelii, and M. cataphractus [1920].
This suggests that at least multiple centric fissions occurred after O. tetraspis diverged from G. gangaticus, T. schlegelii, and M. cataphractus around 14~37 MYA.
Karyotypes resembling (2n = 32) T. schlegelii are also found in Crocodylus.
This suggests that both T. schlegelii and Crocodylus share the same karyotypic feature which is derived from G. gangaticus and Alligator as a common ancestor of Crocodylia.
By contrast, Crocodylus, only one genus in crocodilians, shows variation of chromosome number and chromosome morphology, in spite of the same chromosome number.
This suggests that karyotype evolution occurred in Crocodylus, and subsequently radiated karyotypic forms in different species, during the period of crocodilian extinction around 9~16 MYA in the middle of the Miocene (Fig.1) [16].
It is likely that after climate optimum in mid-Miocene, speciation process of Crocodylus was conducted along with global cooling and glaciation until the end of Pliocene, whereas other crocodilians underwent massive extinction [64].
The role of chromosome changes with species diversity is also found in other species during the same period, such as New World rodent (Calomys) [66] or African rodent (Tatera) [67].
Interestingly, the chromosome structure of Crocodylus species is likely conserved within the genus.
This agreed with the presence of hybridization in captivity, often found in Crocodylus species.
It would appear that all species from the genus have the potential to hybridize and produce fertile offspring.
Examples of such events have been reported in both sympatric [6869] and allopatric species [61].
C. siamensis with chromosome number 2n = 30 has also hybridized with C. porosus with chromosome number 2n = 34.
Chavananikul et al.
[70] found that interspecific breeding of C. siamensis and C. porosus produced F1 hybrids (2n = 32) and F2 offspring with 2n = 32.
This suggested hybrid viability, despite different chromosome numbers in the parent generation.
This probably results from genome features that are still similar and not diverse in a short period of time in Crocodylus.
Thus, the study of hybrid viability, homologous recombination, and meiotic segregation mechanisms may be critical to understand their genome evolution.
However, the relative stability of Crocodylus genomes contributes to the lack of postzygotic reproductive isolation mechanisms, which is a fascinating aspect of crocodilian evolutionary biology, and rejects the traditional biological species concept.
Other evidence of high conservation in chromosome structure is satellite DNA (stDNA) at the centromeric region on crocodilian chromosomes.
Two stDNAs (CSI-HindIII-S and CSI-HindIII-M) isolated from C. siamensis and localized to centromeric regions of all chromosomes, except for chromosome 2, were highly conserved throughout several crocodilian species (Crocodylus niloticus, T. schlegelii, G. gangeticus, A. mississippiensis, A. sinensis, Caiman crocodilus, and Caiman latirostris) [21].
This suggested that CSI-HindIII sequences were contained in the karyotype of a common ancestor of Crocodylia.
However, the evolutionary history of Crocodylus species recently diversified during a period of crocodilian extinction.
This tallied with the evidence of a centromeric stDNA (CSI-DraI), which was also isolated from C. siamensis and localized to chromosome 2, and four pairs of small-sized chromosomes, were recognized in the genus Crocodylus but not in other crocodilians (Lapbenjakul et al.unpublished data) [21].
This suggested that CSI-DraI differentiated as a very rapidly evolving molecule in the Crocodylus lineage when crocodilian extinction began, and the molecular structure of centromeric regions of Crocodylus evolved independently from other crocodilians.
Within the Crocodylus lineage, molecular sequence analyses suggested highly recent radiation in this crocodilian group (Fig.1) [16].
The relationship of A + T content (%) and AT-skew of the mtDNA genome also agreed with this indication as a high correlation between A + T content and AT-skew in many Crocodylus species but low correlation in other crocodilians (Fig.2), suggesting that the bias of base substitution that occurred in the Crocodylus lineage did not have time to eminently alter the base composition of mtDNA genome as in other crocodilian divergences [17].
Traditionally, Crocodylus was considered to have originated in Africa during the Cretaceous [71], and their current distribution has resulted from continental drift [72].
However, recent molecular data and fossil records in Asia suggested that the Indo-Pacific was the place of origin of Crocodylus species [16].
They then migrated to the land based on two hypotheses: (1) dispersion from Indo-Pacific to the New World, followed by dispersion to Africa; and (2) dispersion from the Indo-Pacific into Africa, followed by dispersion to the New World.
This hypothesis suggested the trans-Pacific dispersion of Crocodylus from the Indo-Pacific to the New World around 8~12 MYA, followed by moving to Africa from the New World around 4~8 MYA.
This hypothesis was supported by at least three pieces of evidence: the presence of fossil crocodilian records within Asia and the New World [173], the distribution of the saltwater crocodile C. porosus, and the extinct Crocodylus lineage in the Pacific islands.
Molecular phylogenetic placement by Oaks [16] showed the sister group of C. porosus, C. palustris, and C. siamensis clade and New World Crocodylus species.
However, it is hard to explain how C. porosus (2n = 34) and/or C. palustris and C. siamensis (2n = 30) migrated and evolved into New World species (2n = 32), whose karyotype retains an ancestral form.
It is likely that the C. porosus karyotype resulted from the proto-Crocodylus karyotype by centric fission, and the New World species karyotypes experienced centric fusion reversibly again from C. porosus.
The dispersion of Crocodylus from the Indo-Pacific into Africa occurred 8~12 MYA, followed by trans-Atlantic dispersion to the New World 4~8 MYA.
The crocodilian fossil record also showed that Crocodylus species were present in Southern Europe and Northern Africa during the late Miocene with warmer and wetter climate across this region [747576].
This might have allowed the Crocodylus species to move from Asia into Africa without requiring long-distance marine dispersion.
It is likely that the species group comprising C. mindorensis, C. novaeguineae, and C. johnstoni, positioned to the basal node of the phylogenetic tree as suggested by Oaks [16] might be the origin of the Indo-Pacific species.
This agreed with karyological data of Crocodylus, which showed a diploid chromosome of C. mindorensis, C. novaeguineae, C. johnstoni (Indo-Pacific species) C. niloticus (African species), Crocodylus moreletii, Crocodylus acutus, and Crocodylus intermedius (New World species) to be 32.
Crocodylus maintained the broad African and Indo-Pacific distribution, and then split between the African + New World species clade and the remaining Indo-Pacific species around 4~8 MYA.
The latter hypothesis is more likely, based on the most parsimonious explanation with karyological data.
This also agreed with the phylogenetic relationship of some Nile crocodiles (C. niloticus), complete mt genome sequences placed with those of New World species, and the exclusion of Nile crocodiles from other populations [15].
The two earliest fossils of C. niloticus and New World species appear during the Pliocene [73], and C. niloticus has lingual salt-excreting glands.
Moreover, some populations of C. niloticus occupy estuarine habitats [63].
This suggested that an African crocodilian crossed the Atlantic recently, when this oceanic barrier was hundreds of miles wide [1577].
The presence of trans-Atlantic dispersion is also found in other vertebrates [78].
The molecular placement of C. porosus, C. palustris, and C. siamensis is expected to position as the sister to African + New World species (Fig.1).
The ancestral karyotype feature is still retained in African and New World crocodiles.
Comparison of the data of chromosome painting in C. niloticus and chromosome maps with functional genes in C. siamensis, revealed that five large chromosomes of C. siamensis correspond to six large chromosomes of C. niloticus [2326].
C. siamensis chromosome 3 (CSI3) is metacentric, and C. niloticus chromosome 5 (CNI5) and CNI6 are acrocentrics.
CSI3p corresponds to CNI6, and CSI3q to CNI5.
This suggests that bi-armed chromosomes CSI3 are the results of centric fusion among the ancestral types of acrocentric chromosomes (Fig.3).
A centric fusion event is more likely because CNI5 and CNI6 are considered the prototypes in the lineages of Crocodylus, based on the evidence of karyotype and chromosome constitution with other crocodilians [1920].
The karyotype of C. palustris and C. siamensis (2n = 30) probably resulted from one centric fusion of two acrocentric chromosomes from ancestral Crocodylus species from the Indo-Pacific species, leading to the reduction of uni-armed chromosomes in both two species.
By contrast, karyotype of C. porosus (2n = 34) might result from one centric fission of large bi-armed chromosomes from ancestral Crocodylus species, leading to the increase of acrocentric chromosomes.
Simultaneously, the molecular placement of New World species is also controversial, considering complete mt genome sequences and the mixture of several molecular loci, in which C. rhombifer and C. moreletii are positioned at a basal or recent clade (Fig.1).
The karyotype data of C. niloticus and New World species (C. acutus and C. intermedius) suggest that the karyotype of C. moreletii and C. rhombifer is probably located at the recent clade of New World species.
The karyotype of C. moreletii contains a lower number of bi-armed chromosomes and a higher number of uni-armed chromosomes, compared to C. acutus and C. intermedius, making the fundamental numbers of these three species different.
This suggests that the presence of pericentric inversion or centromere positioning has probably occurred in the ancestral karyotype of C. moreletii after divergence from C. acutus and C. intermedius around 3~7 MYA during the Pliocene.
By contrast, the karyotype of C. rhombifer contains 2n = 30, and centric fusion might have occurred in the ancestral karyotype, leading to the reduction of acrocentric chromosomes in C. rhombifer.
The role of chromosomal rearrangement led us to understand the evolutionary history of crocodilians to sauropsids and within the crocodilian lineage.
It is likely that centric fusion/fission is a key evolutionary force of crocodilian diversity.
The Crocodylus group is considered to have variation of karyotypic features, comprising 2n = 30, 32, and 34, whereas the chromosome number of the remaining group was fixed in each lineage.
The Crocodylus species originated from an ancestor in the Indo-Pacific, and rapidly radiated and dispersed around the globe during a dire period in crocodilian extinction.
Moreover, Crocodylus species probably underwent multiple transoceanic dispersions from the Indo-Pacific to Africa and to the New World.
Karyotype analysis also supported the direction of molecular phylogenetic placement in crocodilians, which was previously controversial.
The study also provided insight into the phylogenetic hierarchy of genome evolution in crocodilians.
Conflict between most molecular studies based on complete mitochondrial genome sequences (concatenated twelve protein coding sequences: ND1, ND2, ND3, ND4, ND4L, ND5, COI, COII, COIII, Cytb, ATPase 6, and ATPase 8) (left) and the mixture of several molecular loci (nuclear and mitochondrial DNA) (right) with regard to crocodilian phylogeny, divergence time, and karyological data.
The majority-rule tree with interval node ages from the *BEAST posterior sample is conducted based on a conservative upper bound of 100 million year ago (MYA), placed on the root of Crocodylia (83.5~96.5 MYA) and on the divergence of Alligator-Caiman (64~71 MYA) [1, 18] (left image).
The phylogenetic tree within Crocodylia was obtained from Oaks [16] with slight modification (right image).
Karyological data were obtained from Cohen and Gans [19], Olmo and Signorio [20], Kawagoshi et al.
[21], Srikulnath et al.
[22], and Kasai et al.[23].
Estimated divergence times at individual nodes are shown with their interval ages.
Crocodylus siamensis (1), complete mitochondrial genome of C. siamensis (DQ353946) sequenced by Ji et al.
[24]; Crocodylus siamensis (2), complete mitochondrial genome of C. siamensis (EF581859) sequenced by Srikulnath et al.[17].
?, no data on chromosome constitution; NF, fundamental number.
In Crocodylus lineage, Indo-Pacific species is Crocodylus mindorensis, Crocodylus novaeguineae, Crocodylus johnstoni, Crocodylus siamensis, Crocodylus palustris, and Crocodylus porosus; New World species is Crocodylus moreletii, Crocodylus acutus, Crocodylus intermedius, and Crocodylus rhombifer; and African species is Crocodylus niloticus.
AT-skew versus A + T content (%) of complete mitochondrial (mt) DNA genomes in Crocodylia.
Values are calculated on heavy strands for full length of mtDNA genomes from 23 crocodilians [7911121314151724].
The x-axis indicates the skew values, the y-axis provides the A + T content (%).
The wide fine dashed line indicates the Crocodylus relation between AT-skew versus A + T content, whereas the solid line and the dashed line display the Alligatoridae and Crocodylidae relations, respectively.
Schematic representation for the process of chromosomal rearrangements that occurred among Crocodylus siamensis chromosomes (CSI) 3, and Crocodylus niloticus chromosomes (CNI) 5 and 6.
Chromosome homologies with the chicken (Gallus gallus) are obtained from the following sources: C. siamensis from Uno et al.
[26] and C. niloticus from Kasai et al.
[23], and are shown to the right of the chromosomes.
Homologous chromosomes and/or chromosome segments are shown using the same color.
Arrows indicate the directions of the chromosomal rearrangements.
NF, fundamental number.
Title: Introns: The Functional Benefits of Introns in Genomes

The intron has been a big biological mystery since it was first discovered in several aspects.
First, all of the completely sequenced eukaryotes harbor introns in the genomic structure, whereas no prokaryotes identified so far carry introns.
Second, the amount of total introns varies in different species.
Third, the length and number of introns vary in different genes, even within the same species genome.
Fourth, all introns are copied into RNAs by transcription and DNAs by replication processes, but intron sequences do not participate in protein-coding sequences.
The existence of introns in the genome should be a burden to some cells, because cells have to consume a great deal of energy to copy and excise them exactly at the correct positions with the help of complicated spliceosomal machineries.
The existence throughout the long evolutionary history is explained, only if selective advantages of carrying introns are assumed to be given to cells to overcome the negative effect of introns.
In that regard, we summarize previous research about the functional roles or benefits of introns.
Additionally, several other studies strongly suggesting that introns should not be junk will be introduced.
All eukaryotic genomes carry introns as parts of some gene structures and the introns are to be eliminated by a complex molecular machinery called the spliceosome comprising five snRNAs and more than 150 proteins [12].
Although the debate on the origin of introns, i.e., the intron-early versus intron-late hypothesis, has still not been completed, it is obvious that most spliceosomal introns have been gained after prokaryote-eukaryote divergence [345], and there has been no spliceosomal introns found in prokaryotic lineages so far.
The intron has still been propagating in some eukaryotic lineages [6], whereas other lineages have experienced extensive losses of introns during evolutionary life history [2].
Primates have a higher density of intronic sequences than primitive eukaryotes that diverged earlier in eukaryotic life history, such as yeast, Drosophila and Caenorhabditis elegans.
Intron sequences constitute approximately 25% of the human genome, which is 4~5 times the size of exons [7].
It has been noticed that the number of genes varies little between these eukaryotic species ranging less than 2-fold from 14,000 genes to 25,000 genes, whereas the size of introns greatly varies up to several fold, implicating that introns might have roles in determining species-specific characteristics and complexities [8].
Introns certainly impose a huge energetic burden to the cell, considering that the density of introns (i.e., the genic regions consuming large amounts of energy for nothing in terms of protein synthesis) is greater than that of exons in genomes.
The reasons that introns propagated in some eukaryotic genomes regardless of this energetic disadvantage have been issues yet to be explained.
According to Lynch [9], introns are just selfish DNAs that invade protein-coding genes in eukaryotic genomes, and the deleterious introns can be sustained due to severe population bottlenecks.
Many studies have discussed selective advantages that introns bring to the cell in eukaryotes, contributing to overcoming the energetic disadvantage [21011121314151617181920].
However, the results derived from different studies are still controversial so far [132122232425].
Recent multi-omics studies using a large-scale genome, transcriptome, and epigenome data produced by massively parallel sequencing techniques or next generation sequencing techniques provide an opportunity for us to investigate new territories in genomes and lead to novel functional insights into noncoding DNAs, intergenic regions, and introns.
In the present review, we first introduce some studies showing what molecular characteristics of introns cannot be explained by a simple random mutational process that real junk DNAs may have undergone.
Subsequently, we summarize the functional characteristics of introns that have been studied providing clues about the adaptive significance of introns in genomes.
We divide the functional roles of introns into two different categories, i.e., direct roles and indirect roles in Table 1 [15262728293031323334353637383940414243444546], and demonstrate the details in the Results section.
It is problematic, though, that the &#39;function&#39; of genes in molecular biology generally has been limited in the concept of &#39;protein function&#39;.
As recently intensively debated, &#39;biological function&#39; should be extended to expression regulation by cis-acting elements located at the outside regions of protein-coding parts in genes [47], and that is what we mean in discussing intron function.
Title: Direct roles of introns

Title: Regulation of alternative splicing

Introns are crucial because the protein repertoire or variety is greatly enhanced by alternative splicing in which introns take partly important roles.
Alternative splicing is a controlled molecular mechanism producing multiple variant proteins from a single gene in a eukaryotic cell.
One of the remarkable examples of the increasing protein repertoire by alternative splicing is the Drosophila Dscam gene, of which over 38000 isoforms can potentially be produced by alternative splicing.
Pan et al.
[27] have provided experimental evidence suggesting that approximately 95% of multiexon genes in the human genome may undergo alternative splicing.
Furthermore, very short introns are selected against because a minimal length of intron is required for the splicing reaction [28].
It has been noticed that the length of conservations in flanking introns of conserved alternative exons, i.e., exons that are alternative in several species, is greater than the length of conservations in flanking introns of conserved constitutive exons, i.e., exons that are constitutive in several species [26], suggesting that introns carry cis-acting elements that regulate alternative splicing.
In fact, short cis-acting motifs that are necessary for binding splicing factors have been recognized and named intronic splicing silencers and intronic splicing enhancers.
Title: Positive regulation of gene expression

The expression enhancing effect of introns was first recognized in the experiment using simian virus 40 constructs with or without introns, showing that their protein products were significantly diminished without their introns [15].
Subsequently, Buchman and Berg [48] showed that, in a certain condition, constructs with introns were expressed up to 400 times higher than constructs without introns, suggesting that introns can strongly enhance gene expression.
In fact, some introns are designed to be included to construct expression vectors for guaranteeing a higher level of expression [49].
A large-scale analysis performed in yeast also confirmed that genes with introns tend to have a higher level of gene expression compared to genes without introns [50].
A similar observation was made in mammals, as well [51].
Classically, enhancers mediate either direction of expression, up- and down-regulation of genes, and involve both spatial and temporal control of gene expression in a specific cell independent of genomic location [52].
On the contrary, intron-mediated enhancers (IMEs) mainly identified in plant generally act in the expression enhancement of genes and are primarily located in the first ordinary intron position within a gene.
In fact, in experiments performed in Arabidopsis, rice, and even mammals, the expression level of a gene with IMEs was increased up to 100-fold [29].
Genomic location and distance from transcription start site can influence the IME activity unlike the mode of expression regulation performed by the classical enhancers [53].
Transcription initiation and termination processes are cellular processes that involve introns, as well, which need some sequence elements in introns to be correctly completed.
For instance, some studies showed that specific sequence elements in introns, such as enhancers and silencers, regulate transcription initiation through modulating the function of the promoters of genes [3054].
Title: Regulation of nonsense-mediated decay

Nonsense-mediated decay (NMD) was originally known as a surveillance mechanism in eukaryotes that selectively removes mRNAs containing erroneously generated premature termination codons (PTCs).
However, several recent studies have suggested that NMD may be another normal mechanism of post-transcriptional gene expression regulation [343555].
Consistently, a recent study has shown that the levels of the expressions of genes important for plant development are regulated by NMD [36].
The question is how NMD recognizes the PTC-containing transcripts, i.e., what the molecular characteristics of the NMD target transcripts are.
Generally, NMD recognizes the transcript on which an exon-exon junction complex (EJC) resides more than 50~55 base-pairs downstream of an authentic termination codon as the premature transcripts, i.e., its target mRNAs, implicating that introns somehow play a role in recognizing the premature mRNA targets.
Kalyna et al.
[36] have shown that introns located in 5&#39; or 3&#39; untranslated regions (UTRs) play important roles in controlling NMD-sensitivity of transcripts.
Title: Introns may be associated with mRNA transport or chromatin assembly

It has been reported that spliced transcripts are exported faster from the nucleus to cytoplasm than their unspliced counterparts [5657] indicating the association between splicing machineries and nuclear export, although there are some contradictory studies [5859].
In fact, nuclear transport to the cytoplasm of transcripts containing introns in their 5&#39; UTRs was known to be regulated by the transcription export complex and the serine/arginine-rich (SR) proteins, whereas the transport of transcripts lacking introns in their 5&#39; UTRs was regulated by signal sequences located in the open reading frames (ORFs) of those genes [60].
A recent experiment using fluorescence in situ hybridization has investigated how intron-bearing and intronless constructs are distributed differently across the nucleus and cytoplasm and showed that intron-bearing transcripts are preferentially located in the cytoplasm [31].
There are some studies suggesting that introns may have a role in chromatin assembly as well.
Recent genome-wide mapping analyses of nucleosome positions have shown that nucleosomes are relatively depleted in intron regions compared to exonic regions [3233].
Schwartz et al.
[32] have suggested that sequence elements of intron ends may be responsible for nucleosome depletion in introns by pushing the nucleosomes away toward exons.
Title: Indirect roles of introns

Title: Different ordinal position of introns within the gene has a different functional role

The first intron among all introns within a gene has particularly been a research focus.
The first intron is the longest among all other downstream introns within a gene in most species including plants and animals [38].
Additionally, certain transcription factor binding motifs are enriched in first introns [61].
Different parts of genes have different average sizes of introns, e.g., the size of introns in the 5&#39; UTRs are twice as large as introns in coding regions [62].
In Drosophila, long introns evolve more slowly than shorter ones and first introns are the longest compared to other introns [3763].
In Tetrahymena, the introns located closer to the 5&#39; end of genes are more conserved than downstream introns.
Our team also proved in a previous study that first introns are the longest and the most conserved [39] compared to other downstream introns.
Furthermore, we showed that active histone marks, such as H3K4me1, and H3K4me3, are significantly enriched in the first introns, and the size of the first intron of a gene becomes bigger as the number of exons that gene carries increases.
Additionally, we showed in the same paper that the proportions of regulatory histone marks are positively associated with the levels of gene expressions in 12 normal human tissues including kidney, heart, liver, and ovary [39].
Additionally, a replacement of the second intron with other introns in the beta-globin gene in human led to a reduction of the efficiency of 3&#39;-end formation [64].
Introns, particularly first introns, have important roles in the correct cytoplasmic localization of some mRNAs, including the Drosophila oskar gene and mRNA export [6065] as well as in transcriptional and translational regulation [616667].
Taken together, first introns among all introns within genes have special functional characteristics, indicating that the existence of introns within genes is highly unlikely to be the product of a random process.
Title: Introns length matters in the efficiency of natural selection

According to Comeron et al.
[41], long introns are favored because they increase the efficiency of natural selection by releasing The Hill-Robertson (HR) interference.
The HR interference was basically described as genetic linkage between two sites under selection in finite populations, leading to decreasing effectiveness of natural selection [41].
The HR interference model predicts that selection efficiency should be different between genes that differ in exon-intron structures, so that genes with longer introns should be under weaker HR interference by increasing recombination between two sites in two neighboring exons.
In other words, introns may have a role in relaxing intragenic HR interference between sites under the influence of natural selection in finite populations.
Recombination gives the opportunity for two independently occurring favorable alleles at linked loci to be located together and thus enhances the efficiency of natural selection [40], which can be one of the plausible scenarios of how introns have been sustained through the evolutionary history of genes.
Title: Introns can provide a source of new genes

Recently, Carvunis et al.
[42] suggested a very interesting hypothesis about how novel genes arise from non-functional translated ORFs, named proto-genes, by showing that hundreds of short ORFs of proto-genes located in non-genic sequences were actually translated and might provide adaptive potential to cells in different physiological environments in Ascomycota phylogeny, including Saccharomyces cerevisiae.
According to their model, the short ORFs can evolve into real functional genes through a kind of continuous evolutionary process.
In that sense, long non-coding intron regions in higher eukaryotes can be a good reservoir of short and non-functional ORFs.
Title: Trait-associated single nucleotide polymorphisms are enriched in introns

Genome-wide association study (GWAS) has been a popular approach to identify trait associated genetic variants so-called single nucleotide polymorphisms (SNPs).
GWASs compare the allele frequencies of case groups (i.e., disease groups) and control groups (i.e., normal groups) of study participants to identify the SNPs that are significantly more enriched in case groups than control groups.
If an allele is significantly more frequent in case groups, the allele is said to be a disease-associated allele, or a trait-associated SNP (TAS).
In theory, TASs are considered to reside near sites of actual disease-causing mutations in genomes.
Interestingly, most of the TASs detected by GWASs have been mapped to intron regions rather than exonic or nonsysnonymous sites (Fig.1A) [4368].
The statistical significance of this finding was proven by a comparison of the proportion of intronic SNPs mapped by all SNPs obtained from &#39;v dbSNP 142&#39; after subtracting the TASs (i.e., all SNPs minus TASs) and the proportion of intronic TASs (p&lt;0.01) (Fig.1B).
Investigation of the functional implication of these intron-TASs will thus be an important research subject in the future.
Title: Introns harbor several kinds of noncoding functional RNA genes

Recent studies based on massively parallel sequencing techniques have contributed to identifying various types of noncoding RNAs (ncRNAs) in genomes including miRNAs, siRNAs, piwi-interacting RNAs (piRNAs), long noncoding RNAs (lncRNAs), and small nucleolar RNAs (snoRNAs), and they are known to be preferentially located in the intron regions within genes [46].
For instance, about half of the miRNAs in the human genome are located in introns, and they are usually co-expressed with their host genes regulated by the promoters of host genes [44].
Similar to miRNAs, some snoRNAs reside in introns, and they are also regulated by host transcriptional and splicing machineries [45].
Other ncRNAs, including lncRNAs and siRNAs, are also found in intron regions, though the proportion of lncRNAs and siRNAs in introns is lower than that of miRNAs and snoRNAs in introns [246].
Introns are classically degraded after the completion of splicing; however, these ncRNA genes embedded in intron regions are produced upon intron removal [246].
Furthermore, they can survive even longer than the intronic host genes [2].
Considering that the ncRNAs located in introns are co-expressed and co-regulated with their host genes by the promoters and splicing machineries of host genes, they are considered to be involved in auto-regulation of the expression of host genes [46].
The existence of introns in genome is a real mystery, given the expensive energy cost for a cell to pay for copying the entire length of several introns in a gene and excising them at the exact position, controlled by big RNA and protein complexes after transcription.
Nevertheless, most completely genomes of eukaryotic cells so far carry introns in their genomes [6970], and some studies even showed that introns had been propagated during eukaryotic lineage evolution [39717273].
The origin of spliceosomal introns in eukaryotic lineage has been attempted to be explained by the massive invasion of group II self-splicing introns from bacteria to eukaryotes [35].
It is very hard to understand how and why introns propagate in eukaryotic lineages and what the beneficial effect of introns on cell survival is.
We reviewed here putative functional roles of introns in various cellular processes such as splicing, mRNA transport, NMD, and expression regulation.
Besides, introns may give some advantages as a mutational buffer in eukaryotic genomes protecting coding sequences from being affected by randomly occurring deleterious mutations.
Introns occupy about 40% on average of the total length of genes, which means that most randomly occurring mutations will fall into intron regions, and do not affect protein sequences and functions.
However, it is not clear how extensively and strongly this buffering effect of intron regions might have evolutionary advantages for intron retention against the pressure of removing cellular burdens.
Taken together, introns are clearly not junk, and they provide selective advantages to cells to be evolutionarily maintained, nevertheless, it has expensive energetic costs.
New advanced molecular biology techniques will lead to the functional territories of introns in a more detailed scale in the near future.
Classification of trait-associated single nucleotide polymorphisms (TASs) across genic regions.
(A) A pie chart showing the proportion of each type of genetic variant obtained from GWASdb [68]; nearGene-3, SNPs within 500 bps from 3&#39; downstream of a transcript; nearGene-5, SNPs within 2,000 bps from 5&#39; upstream of a transcript.
(B) The proportions of intronic TASs and all intronic SNPs after subtracting TASs were compared.
The &#39;-&#39; in &#39;all SNPs – TASs&#39; means &#39;subtraction of&#39;.
The statistical significance was analyzed by chi-squared test.
Title: Summary of direct and indirect intron functions
Title: Analysis of Whole Transcriptome Sequencing Data: Workflow and Software

RNA is a polymeric molecule implicated in various biological processes, such as the coding, decoding, regulation, and expression of genes.
Numerous studies have examined RNA features using whole transcriptome sequencing (RNA-seq) approaches.
RNA-seq is a powerful technique for characterizing and quantifying the transcriptome and accelerates the development of bioinformatics software.
In this review, we introduce routine RNA-seq workflow together with related software, focusing particularly on transcriptome reconstruction and expression quantification.
The transcriptome is the entire set of RNA transcripts in a given cell for a specific developmental stage or physiological condition [1].
Understanding the transcriptome is necessary for interpreting the functional elements of the genome as well as for understanding the underlying mechanisms of development and disease.
Microarray technologies have been used for high-throughput large-scale RNA-level studies, such as to identify differentially expressed genes between developmental stages or between healthy and diseased groups [2].
However, its hybridization- based nature limits the ability to catalog and quantify RNA molecules expressed under various conditions.
Advances in massive parallel DNA sequencing technologies have enabled transcriptome sequencing (RNA-seq) by sequencing of cDNA.
RNA-seq has rapidly replaced microarray technology because of its better resolution and higher reproducibility; this method can be used to extend our knowledge of alternative splicing events [3], novel genes and transcripts [4], and fusion transcripts [5].
One concern regarding the application of RNA-seq is abundance estimation at the gene-level and transcript-level differential expression under distinct conditions.
Routine RNA-seq workflow may consist of the following five steps as shown in Fig.1: (1) preprocessing of raw data, (2) read alignment, (3) transcriptome reconstruction, (4) expression quantification, and (5) differential expression analysis.
As an initial step, RNA-seq data may be subjected to quality control (QC) of the raw data before data analysis.
Similar to whole genome or exome sequencing, read alignment can be performed to map the reads to the reference genome or transcriptome.
Clinical samples including formalin-fixed paraffin-embedded specimen and cancer tissue biopsies are often degraded or exist in limited amount [6].
Thus additional QC procedure can be performed to evaluate the performance of the RNA-seq experiment itself after read alignment.
Next, transcriptome reconstruction is carried out to identify all transcripts expressed in a specimen based on read mapping data.
If there is no available reference sequence, this procedure can be conducted directly using a de novo assembly approach.
Once all transcripts are identified, their abundances can be estimated using read mapping data.
Finally, differential expression analysis is conducted using currently available programs.
In this review, we discuss the RNA-seq workflow and its related bioinformatics tools in each step (Table 1), focusing on transcriptome reconstruction and abundance quantification.
Similarly to whole genome or exome sequencing, RNAseq data is formatted in FASTQ (sequence and base quality).
Numerous erroneous sequence variants can be introduced during the library preparation, sequencing, and imaging steps [7], which should be identified and filtered out in the data analysis step.
Thus, QC of raw data should be performed as the initial step of routine RNA-seq workflow.
Tools such as FastQC [8] and HTQC [9] can be applied in this step to assess the quality of raw data, enabling assessment of the overall and per-base quality for each read (i.e., read 1 and 2 in case of paired-end sequencing) in each sample.
Depending on the RNA-seq library construction strategy, some form of read trimming may be advisable prior to aligning the RNA-seq data.
Two common trimming strategies include &#34;adapter trimming&#34; and &#34;quality trimming.&#34;
Adapter trimming involves removal of the adapter sequence by masking specific sequences used during library construction.
Quality trimming generally removes the ends of reads where base quality scores have decreased to a level such that sequence errors and the resulting mismatches prevent reads from aligning.
The adapter trimming step is typically not necessary, as most recent sequencers provide raw data in which the adapters are already trimmed.
In contrast, quality trimming may be an essential step depending on the analysis strategy used.
The FASTX-Toolkit [10] and FLEXBAR [11] are useful for this purpose.
There are two strategies in which a genome or transcriptome is used as a reference for the read alignment step [12].
The transcriptome comprises all transcripts in a given specimen and in which splicing has been conducted by including the exons and excluding the introns.
If a transcriptome is used as a reference, unspliced aligners that do not allow large gaps may be the proper choice for accurate read mapping.
Stampy, Mapping and Assembly with Quality (MAQ) [13], Burrow-Wheeler Aligner (BWA) [14], and Bowtie [15] can be used in this case.
This alignment is limited to the identification of known exons and junctions because it does not identify splicing events involving novel exons.
However, if the genome is used as a reference, spliced aligners that allow a wide range of gaps should be employed because reads aligned at exon-exon junctions will be split into two fragments.
This approach may increase the probability of identifying novel transcripts generated by alternative splicing.
Various spliced aligners have been developed, including TopHat [16], MapSplice [17], STAR [18], and GSNAP [19].
Several intrinsic biases and limitations including nucleotide composition bias, GC bias and polymerase chain reaction bias can be introduced to RNA-seq data of clinical samples with low quality or quantity.
To evaluate the biases from RNA-seq data, several metrics may be examined as following: percentage of exonic or rRNA reads, accuracy and biases in gene expression measurements, GC bias, evenness of coverage, 5&#39;-to-3&#39; coverage bias, and coverage of 5&#39; and 3&#39; ends [6].
Some programs including RNA-SeQC [20], RSeQC [21], and Qualimap 2 [22] are currently available for the purposes, which take typically BAM file as input.
RNA-SeQC [20] provides three types of QC metrics based on read count (total, unique and duplicate reads, rRNA content, strand specificity, etc.
), coverage (mean coverage, 5&#39;/3&#39; coverage, GC bias, etc.
), and expression correlation (reads per kilobase per million mapped reads [RPKM]–based estimation of expression levels and correlation matrix by all pairwise comparison).
The software also provides multisample evaluation regarding library construction protocols, input materials and other experimental parameters.
RSeQC [21] is a Python-based package program that provides several metrics containing sequence quality, GC bias, polymerase chain reaction bias, nucleotide composition bias, sequencing depth, strand specificity, coverage uniformity, and read distribution over the genome structure.
Of the metrics, sequencing depth is importance, because it allows users to determine if current RNA-seq data is suitable for such application including expression profiling, alternative splicing analysis, novel isoform identification, and transcriptome reconstruction by checking whether the sequencing depth is saturated or not.
Qualimap 2 [22] is consisted of four analysis modes: BAM QC, Counts QC, RNA-seq QC, and Multi-sample BAM QC.
Compared to previous release, this version focuses on multi-sample QC for high-throughput sequencing data.
Multi-sample BAM QC mode allows combined QC for multiple alignment files, which takes the metrics from the single-sample BAM QC mode as input.
RNA-seq QC mode is added to compute the metrics specific to RNA-seq data, which contains per-transcript coverage, junction sequence distribution, genomic localization of reads, 5&#39;-3&#39; bias and consistency of the library protocol.
Counts QC mode enables to estimate the saturation of sequencing depth, read count densities, correlation of samples and distribution of counts among classes of selected features along with gene expression estimation based on NOIseq [23].
Transcriptome reconstruction is the identification of all transcripts expressed in a specimen.
There are two strategies used for transcriptome reconstruction, including the reference-guided approach and the reference-independent approach.
First, the reference-guided approach consists of two sequential steps: (1) alignment of raw reads to the reference as described in the previous section and (2) assembly of overlapping reads for reconstructing transcripts.
This approach is advantageous when reference annotation information is well-known, such as in human and mouse, which is employed in Cufflinks [24], Scripture [25], and StringTie [26].
Second, the reference-independent approach uses a de novo assembly algorithm to directly build consensus transcripts from short reads without reference, which is useful when there is no known reference genome or transcriptome.
Trinity [27], Oases [28], and transABySS [29] may be used for this purpose.
Two publications have described RNA-seq protocols: one is de novo transcriptome reconstruction without reference using the Trinity platform [30] and the other is differential expression analysis of a gene and transcript using a combination of TopHat and Cufflinks [31].
The latter protocol also includes a transcriptome reconstruction procedure (using Cufflinks) from read mapping data to a reference genome (using TopHat).
These protocols are good examples of different strategies that can be used for transcriptome reconstruction according to the presence or absence of a reference sequence.
Numerous methods have been developed for expression quantification using RNA-seq data.
The methods are grouped into two according to the target levels: gene- and isoform-level quantification.
Alternative expression analysis by sequencing (ALEXA-seq) [32], enhanced read analysis of gene expression (ERANGE) [33], and normalization by expected uniquely mappable area (NEUMA) [34] support gene-level quantification.
Isoform-level quantification methods are divided into three groups according to the reference type and requirement of alignment results.
The first group (e.g., RSEM [35]) requires the alignment result of reads using the transcriptome as a reference.
The second group (e.g., Cufflinks [24] and StringTie [26]) also requires alignment results of reads using whole genome sequences as a reference rather than the transcriptome.
The last group (e.g., Sailfish [36]) uses an alignment-free method.
We discuss each isoform-level quantification method in detail in the following sections.
RSEM is software that quantifies transcript-level abundance from RNA-seq data.
RSEM is operated in two steps: (1) generation and preprocessing of a set of reference transcript sequences and (2) alignment of reads to the reference transcripts followed by estimation of transcript abundances and their credibility intervals.
A FASTA formatted file of transcript sequences is used to generate the reference transcripts, which can be obtained from a reference genome database, a de novo transcriptome assembler, or an Expressed Sequence Tags (EST) database.
Alternatively, a gene annotation file in GTF format and the full genome sequence in FASTA format may be supplied.
RSEM uses the Bowtie alignment program [15].
A user-provided aligner can be used for mapping RNA-seq reads using reference transcripts.
RSEM provides gene-level and isoform-level estimates as the primary output by computing maximum likelihood abundance estimates based on the Expectation-Maximization (EM) algorithm after read mapping.
Abundance estimates are given in terms of two measures: an estimate of the number of fragments and the estimated fraction of transcripts comprising a given isoform or gene.
The latter estimates can be multiplied by 10-6 to obtain a measure of transcripts per million (TPM).
RSEM also supports the visualization of alignment and read depth using a genome browser such as the University of California Santa Cruz (UCSC) Genome Browser.
The Tuxedo package is the most widely used software for transcript assembly and quantification using RNA-seq and consists of a number of different programs, including TopHat, Cufflinks, and Cuffdiff [31].
In the initial step, TopHat is employed for mapping raw RNA-seq reads to a reference genome, where some reads can be spliced when they were aligned on the exon-exon junctions of transcripts.
These mapped reads are provided as input to Cufflinks for transcript assembly and abundance estimation.
Transcript assembly is achieved by building an overlap graph from the mapped reads followed by computing minimal path cover in the overlap graph, generating a minimum number of transcripts that will explain all reads in the graph.
Abundance estimation is performed by estimating the maximum likelihood abundance based on transcript coverage and compatibility together with the use of fragment length distribution.
Abundances are reported in fragments per kilobase per million mapped fragments (FPKM) for paired-end and RPKM for a single-end.
Cuffdiff, a part of the Cufflinks package, also uses the mapped reads to report genes and transcripts that are differentially expressed.
CummeRbund can produce figures and plots from the Cuffdiff outputs.
StringTie is software used for transcriptome reconstruction and abundance estimation.
Similarly to other tools, including Cufflinks, spliced aligners such as TopHat2 [37] or GSNAP [19] are used to directly align RNA-seq reads or subsequent alignment after generating pre-assembled contigs from the reads using a de novo assembler such as MaSurCa [38].
StringTie can perform transcriptome reconstruction and abundance estimation simultaneously by building a flow network for the path of the heaviest coverage and computing the maximum flow to estimate abundance.
StringTie reports estimated abundance in FPKM for paired-end and RPKM for single-end.
Sailfish is unique software adopting an alignment-free approach for isoform quantification.
An index is built from a set of reference transcripts and a specific choice of k-mer length, which consists of data structures that maps each k-mer in the reference transcripts to a unique integer identifier, enabling to count k-mers in a set of reads and to resolve their origin in the set of transcripts.
Until the set of reference transcripts or the k-mer length is changed, it is not necessary to rebuild the index.
Sailfish computes an estimate of the relative abundance of each transcript in the reference by employing an EM algorithm similar to that used in RSEM.
Because Sailfish avoids read alignment entirely, the running time for quantification is much lower than for other existing methods.
Sailfish reports terms of abundance measures, including (1) RPKM, (2) k-mers per kilobase per million mapped k-mers (KPKM), and (3) TPM.
We described four programs, RSEM, Cufflinks, StringTie, and Sailfish in detail.
In addition to the use of specific algorithm, a major difference between these programs may be the reference type used.
A set of transcript sequences is used as a reference in RSEM and Sailfish, indicating that the programs may be suitable for estimating the abundance of known transcripts.
In contrast, a reference genome is employed in Cufflinks and StringTie, making it possible to present the estimated abundance of novel transcripts as well as already known transcripts, as spliced read mapping data can reveal known and novel splice junction information simultaneously.
For differential expression analysis, a number of software packages and pipelines have been developed including edgeR [39], DESeq [40], NOIseq [23], SAMseq [41], Cuffdiff [24], and EBSeq [42].
Unlike edgeR and DESeq, which adopt negative binomial models, and NOIseq and SAMseq, which are non-parametric, Cuffdiff and EBSeq can be used to compare differentially expressed genes by employing transcript-based detection methods.
Many of the programs accept read count data as input, which can be produced by using HTSeq [43] or BEDTools [44].
Similarly to Cuffdiff, Ballgown program [45] is employed for differential expression analysis using read mapping data from StringTie [26] (https://ccb.jhu.edu/software/stringtie/index.shtml?t=manual).
The above programs adopt one or more of the several available normalization methods (total count, upper quartile, median, DESeq normalization, trimmed mean of M values, quantile and RPKM normalization) to correct biases that may appear between samples (sequencing depth [33]) or within sample (gene length [46] and GC contents [45]).
Although many programs have been developed, one research group reported that there may be large differences between these programs and that no single method may be optimal under all experimental conditions [48].
Thus, it may be difficult for most of users with no or weak statistical background to select a proper method.
However, because RNA-seq data sets are rapidly accumulating, we expect that new bioinformatics tools for differential expression will be developed, which will function robustly under a wide range of conditions.
Numerous bioinformatics programs have been developed for RNA-seq data analysis.
Even tools developed for a same purpose are based on distinct approaches using different algorithms and models.
The diversity of the methodology makes it possible to customize analysis protocols by choosing a program that provides the best fit to each specific goal.
In this review, we described the routine RNA-seq analysis workflow, focusing on transcriptome reconstruction and expression quantification, and also introduced its related bioinformatics programs.
Therefore, we expect that this review will be helpful for preparing a specific pipeline for RNA-seq data analysis, enabling to design new biological experiments.
Typical workflow for RNA sequencing (RNA-seq) data analysis.
This workflow shows an example for expression quantification and differential expression analysis at gene and/or transcript level using RNA-seq, which is typically consisted of five steps as following: preprocessing, read alignment, transcriptome reconstruction, expression quantification and differential expression analysis.
For each step, currently available programs are written in Table 1.
QC, quality control.
Title: Selected list of RNA-seq analysis programs
Title: Association of HLA Genotype and Fulminant Type 1 Diabetes in Koreans

Fulminant type 1 diabetes (T1DM) is a distinct subtype of T1DM that is characterized by rapid onset hyperglycemia, ketoacidosis, absolute insulin deficiency, and near normal levels of glycated hemoglobin at initial presentation.
Although it has been reported that class II human leukocyte antigen (HLA) genotype is associated with fulminant T1DM, the genetic predisposition is not fully understood.
In this study we investigated the HLA genotype and haplotype in 11 Korean cases of fulminant T1DM using imputation of whole exome sequencing data and compared its frequencies with 413 participants of the Korean Reference Panel.
The HLA-DRB1*04:05–HLA-DQB1*04:01 haplotype was significantly associated with increased risk of fulminant T1DM in Fisher&#39;s exact test (odds ratio [OR], 4.11; 95% confidence interval [CI], 1.56 to 10.86; p = 0.009).
A histidine residue at HLA-DRβ1 position 13 was marginally associated with increased risk of fulminant T1DM (OR, 2.45; 95% CI ,1.01 to 5.94; p = 0.054).
Although we had limited statistical power, we provide evidence that HLA haplotype and amino acid change can be a genetic risk factor of fulminant T1DM in Koreans.
Further large-scale research is required to confirm these findings.
Fulminant type 1 diabetes (T1DM) is a distinct subtype of T1DM that is characterized by rapid onset of hyperglycemia and ketoacidosis with absolute deficiency in insulin secretion, and near normal glycated hemoglobin (HbA1c) level at initial presentation [1].
It was first described in 2000 by Imagawa et al.
[2], and was defined as diabetes with remarkably rapid onset (within a few days) and absolute destruction of pancreatic β-cells.
The prevalence of fulminant T1DM is not uncommon and it is estimated to account for 7.1% of all newly diagnosed T1DM cases in Korea [3].
The detailed pathogenesis of fulminant T1DM is not fully understood.
Both genetic and environmental factors, and their interaction contribute to the development of fulminant T1DM.
Regarding genetic factors, it has been reported that human leukocyte antigen (HLA) genotype and the HLADR4─HLA-DQ4 haplotype play a crucial role in the predisposition to fulminant T1DM.
Specifically, the HLA-DRB1*04:05─HLA-DQB1*04:01 haplotype was associated with fulminant T1DM by an odds ratio (OR) of 2.9 in Japanese cases (n = 207) and controls (n = 325) [4].
In addition, the HLA-DRB1*09:01─HLA-DQB1*03:03 haplotype was also associated with fulminant T1DM by an OR of 2.1 [4].
However, these findings have not been replicated in other populations.
Recently, three amino acid positions (HLA-DRβ1 position 13, position 71, and HLA-DQβ1 position 57) in the HLA-DR and HLA-DQ molecules were shown to be significantly associated with T1DM in Europeans [5].
They explained 90% of the phenotypic variance in the HLA-DRB1─HLA-DQA1─HLA-DQB1 locus.
In this study, we investigated the HLA-DRB1─HLADQB1 haplotype and amino acid residues at HLA-DRβ1 position 13 and position 71, and HLA-DQβ1 position 57 in 11 Korean fulminant T1DM cases and compared their frequencies with that of the Korean Reference Panel obtained from 413 participants in the general population [6].
Title: Study participants

A total of 11 cases of fulminant T1DM were identified at Seoul National University Hospital between January 2005 and December 2011 and written informed consent was acquired from the participants.
The study protocol was approved by the Institutional Review Board (IRB) of the Clinical Research Institute of Seoul National University Hospital (IRB No.
0412-138-017 and 1205-130-411).
Peripheral blood DNA was used for massive parallel sequencing.
Title: Whole exome sequencing

Whole exome capture was performed with the SureSelect Human All Exon Kit V4+UTRs (Agilent Technologies, Santa Clara, CA, USA) and massive parallel sequencing was performed with the HiSeq 2000 Sequencing System (Illumina Inc., San Diego, CA, USA).
The sequence reads were mapped and aligned to a human reference genome using Burrow-Wheeler Alignment software version 0.7.12 [7].
The Genome Analysis Toolkit 3.0 HaplotypeCaller was used to identify single nucleotide variations, insertions, and deletions by multi-sample calling method [8].
Title: HLA imputation

For HLA genotype imputation, SNP2HLA software version 1.0.3 was used with default options [9].
Single nucleotide variations, amino acid residues, and four-digit classical HLA alleles of eight HLA genes in chromosome 6 were imputed.
HLA haplotype was inferred with Beagle version 3.0.4 [10].
Imputation was performed based on whole exome sequence genotype data after quality control and as reference, the pan-Asian (n = 530) pre-built reference panel provided by the SNP2HLA was used [11].
An imputation quality score of r2 &gt; 0.3 was used as the cut-off.
Among the 11 fulminant T1DM cases, we validated the HLA-DRB1 and HLA-DQB1 imputed genotype in two participants using the AlleleSEQR HLA Sequencing Kit (Atria Genetics, South San Francisco, CA, USA) and it revealed identical results.
For association testing we used the best-guessed genotype in 0/1/2 format.
Title: Statistical analysis

As we only had genotype data from a small number (n = 11) of fulminant T1DM cases, we tried to describe the HLA genotype and haplotype of each individual, as well as the recently reported amino acid residues at three major positions in the HLA-DR and HLA-DQ molecules [5].
For the case-control analysis, 11 fulminant T1DM cases and 413 Korean Reference Panel controls were compared [6].
Subject having two copies of a specific variant was classified as Present/Present, having only one copy was defined as Present/Absent, and having none of the variants was considered Absent/Absent.
Fisher&#39;s exact test was performed to compare the frequency of HLA haplotype or variant in each locus and Cochran-Armitage trend test was performed to examine the linear trend of three types, Present/Present, Present/Absent, and Absent/Absent between cases and controls.
A p-value of less than 0.05 was regarded to be statistically significant.
Assuming a type 1 error rate of 0.05 and a variant frequency of 0.27 in 11 cases and 0.08 in 413 controls, our study had a limited statistical power of 0.43 using Fisher&#39;s exact text.
Title: Clinical characteristics

The clinical characteristics of the fulminant T1DM cases are described in Table 1.
The mean age was 36.6 ± 9.8 y, and mean body mass index was 22.0 ± 3.4 kg/m2.
All participants had an HbA1c level of less than 7.0% except one patient who had HbA1c of 8.1% at initial presentation of diabetic ketoacidosis.
There was absolute deficiency in insulin secretion in all of the participants and none of the participants had anti-glutamic acid decarboxylase (GAD) autoantibody.
The HLA-DRB1─HLA-DQB1 diplotype of the four-digit classical allele of each individual is described in Table 1.
The amino acid residues at HLA-DRβ1 position 13, HLA-DRβ1 position 71, and HLA-DQβ1 position 57 are also shown in Table 1.
Title: HLA-DRB1─HLA-DQB1 haplotype analysis

The HLA-DRB1─HLA-DQB1 haplotype frequencies in 11 fulminant T1DM cases and 413 Korean Reference Panel subjects are shown in Table 2.
We focused on five haplotypes that were reported to be significantly associated with either increased (HLA-DRB1*04:05─HLA-DQB1*04:01, HLA-DRB1*09:01─HLA-DQB1*03:03) or decreased (HLA-DRB1*01:01─HLA-DQB1*05:01, HLA-DRB1*15:02─HLADQB1*06:01, and HLA-DRB1*08:03─HLA-DQB1*06:01) risk of fulminant T1DM in Japanese.
We found that the frequency of the HLA-DRB1*04:05─HLA-DQB1*04:01 haplotype was 0.27 in Korean fulminant T1DM cases and 0.08 in the Korean Reference Panel.
This haplotype was significantly associated with a risk of fulminant T1DM by Fisher&#39;s exact test (OR, 4.11; 95% confidence interval [CI], 1.56 to 10.86; p=0.009).
None of the other haplotypes showed a significant association presumably due to the limited statistical power.
Title: Three amino acid positions in the HLA-DR and HLA-DQ molecules

The frequency of amino acid residues at HLA-DRβ1 position 13 and position 71, and HLA-DQβ1 position 57 and their association with fulminant T1DM are shown in Table 3.
A histidine residue at HLA-DRβ1 position 13 showed marginal association with fulminant T1DM by Fisher&#39;s exact test (OR, 2.45; 95% CI, 1.01 to 5.94; p = 0.054) and Cochran-Armitage trend test (p = 0.034).
However, none of the other amino acid residues at the three positions showed a significant association with fulminant T1DM.
In this study, we have investigated the HLA genotype and haplotype in 11 fulminant T1DM cases and compared its frequency with that of the general population.
We have found that the HLA-DRB1*04:05─HLA-DQB1*04:01 haplotype is significantly associated with increased risk of fulminant T1DM in Koreans.
In addition, a histidine residue at HLA-DRβ1 position 13 showed marginal association with fulminant T1DM.
To the best of our knowledge, this is the first study to investigate the HLA-DR4─HLA-DQ4 haplotype in fulminant T1DM patients from a population other than Japanese.
We also investigated for the first time the association between amino acid residues in three positions of the HLA-DR and HLA-DQ molecules and the risk of fulminant T1DM.
It is well known that HLA genes account for a significant proportion of the susceptibility to T1DM [12].
The complex genetic structure of HLA loci arising from recombination, gene conversion, and natural selection has been a hurdle in identifying the precise genetic predisposition [13].
However, the linkage disequilibrium stretches for a long-range in the HLA loci and it is possible to determine accurate HLA genotype, and haplotype using common single nucleotide variations [14].
The SNP2HLA software has been shown to be a powerful tool in imputing HLA genotypes in large-scale genome-wide association studies [5].
However, it has not been applied to whole exome sequence genotype data.
We have applied SNP2HLA in 11 cases of fulminant T1DM cases and successfully determined HLA alleles, haplotypes, and amino acid residues.
The classic alleles of HLA-DRB1 and HLA-DQB1 were determined by direct genotyping of two participants of fulminant T1DM and the results were consistent with the imputed genotypes.
The most well-known HLA haplotype associated with fulminant T1DM is HLA-DRB1*04:05─HLA-DQB1*04:01.
In a large-scale association analysis in Japanese, it increased the risk of fulminant T1DM by 2.9 folds.
This haplotype was also associated with either acute onset or slowly progressive T1DM in Japanese [15].
In this study, we confirmed this finding and the haplotype was associated with a 4.11 fold increased risk in Koreans.
In the Korean Reference Panel, the HLA-DRB1*04:05 allele and HLA-DQB1*04:01 allele were in perfect linkage disequilibrium (r2 = 1.0).
Both the HLA-DRB1*04:05 allele and HLA-DQB1*04:01 allele were also significantly associated with the risk of fulminant T1DM (p = 0.011 by Fisher&#39;s exact test).
Although there are other haplotypes reported to be associated with fulminant T1DM, we had limited statistical power to obtain a sound conclusion.
In a recent report investigating the association of T1DM with amino acid residues at three positions of the HLA-DR and HLA-DQ molecules, HLA-DRβ1 position 13 and position 71, and HLA-DQβ1 position 57 explained a significant portion of phenotypic variation conferred by the HLA-DRB1─HLA-DQA1─HLA-DQB1 locus.
We have identified that a histidine residue at HLA-DRβ1 position 13 is marginally associated with a 2.45 fold increased risk of fulminant T1DM.
In Europeans this residue was associated with a 3.64 fold increased risk of T1DM [5].
This residue lines the P4 pocket of the HLA-DR molecule and was reported to be associated with rheumatoid arthritis [16].
It could be speculated that it plays a role in the recognition of autoantigens and autoimmunity [5].
However, it is difficult to have a direct comparison of our results with that of T1DM in Europeans, and it is not known whether these amino acid residues in the three positions are also major risk determinants of T1DM in East Asians.
In addition, the similarities in genetic predisposition between fulminant T1DM and other types of T1DM require further investigation.
It has been suggested that the genetic predisposition predisposition conferred by the HLA locus could be different between fulminant T1DM and other types of T1DM in Japanese [15].
In conclusion, we have investigated the HLA genotype and haplotype in 11 Korean fulminant T1DM cases and provide evidence that HLA-DRB1*04:05─HLA-DQB1*04:01 is an important genetic risk factor of fulminant T1DM.
There is suggestive evidence that a histidine residue at HLA-DRβ1 position 13 which lines the P4 pocket of the HLA-DR molecule, might increase the risk of fulminant T1DM in Koreans.
Further large-scale investigation is required to confirm our findings.
Title: Clinical characteristics and HLA diplotypes of study participants

Title: Haplotype frequency and association with fulminant T1DM

Title: Three amino acid positions at HLA-DR and HLA-DQ and association with fulminant T1DM
Title: Expressional Subpopulation of Cancers Determined by G64, a Co-regulated Module

Studies of cancer heterogeneity have received considerable attention recently, because the presence or absence of resistant sub-clones may determine whether or not certain therapeutic treatments are effective.
Previously, we have reported G64, a co-regulated gene module composed of 64 different genes, can differentiate tumor intra- or inter-subpopulations in lung adenocarcinomas (LADCs).
Here, we investigated whether the G64 module genes were also expressed distinctively in different subpopulations of other cancers.
RNA sequencing-based transcriptome data derived from 22 cancers, except LADC, were downloaded from The Cancer Genome Atlas (TCGA).
Interestingly, the 22 cancers also expressed the G64 genes in a correlated manner, as observed previously in an LADC study.
Considering that gene expression levels were continuous among different tumor samples, tumor subpopulations were investigated using extreme expressional ranges of G64—i.e., tumor subpopulation with the lowest 15% of G64 expression, tumor subpopulation with the highest 15% of G64 expression, and tumor subpopulation with intermediate expression.
In each of the 22 cancers, we examined whether patient survival was different among the three different subgroups and found that G64 could differentiate tumor subpopulations in six other cancers, including sarcoma, kidney, brain, liver, and esophageal cancers.
Different tumors or cancers in different patients have distinct genetic and cellular profiles, including kinds of genetic mutations, patterns of gene expression, and metastatic potential, which is often called cancer heterogeneity.
The heterogeneity occurs both within and between tumors and leads to intra-tumor heterogeneity and inter-tumor heterogeneity, respectively [123].
Understanding cancer heterogeneity has recently been one of the important research subjects, because it is the base of difficulty in developing effective cancer treatments.
Tumor heterogeneity is basically due to the different origins of tumor cells or tissues.
Various biological factors, such as smoking, gender, age, or hormonal status, can influence cancer initiation or progression, as well.
Fundamentally, there are genetic variations among the hosts, even for the same cancer [456].
Astonishingly rapid development of massively parallel sequencing, alternatively called nextgeneration sequencing technologies, has recently elucidated the extent of tumor heterogeneity, and several hundred somatic mutations and structural variants that drive cancers have been identified [789].
Previously, we reported the characterization of the heterogeneity of human lung adenocarcinomas (LADCs) from patient-derived xenografts via single-cell transcriptome sequencing [10].
These cells were categorized into two separate subpopulations based on their expression of a module gene named G64.
We found that G64 upregulation/ down-regulation was also present in patient tissue samples obtained from the Cancer Genome Atlas (TCGA), with G64 up-regulation corresponding to poor survival and associated with multiple clinical variables, such as smoking status (which exhibited the highest correlation) and tumor stage.
In the present study, we investigated if G64 can differentiate tumor subpopulations in other cancers, as well.
Title: Websites for downloading RNA sequencing (RNA-seq) data for 22 cancer datasets

The TCGA website (https://tcga-data.nci.nih.gov/tcga/tcgaHome2.jsp) was used for downloading RNA-seq-based transcriptome data and patient survival information for 22 cancers.
The RNA-seq data downloaded for each cancer are listed in Table 1 and Supplementary Table 1.
As described in the previous report [10], expression values of 0&lt;FPKM (fragments per kilobase of exon per million fragments mapped)&lt;0.1 were all converted to 0.1 to avoid the infinity problem.
Each FPKM value of each gene was divided by the average FPKM estimated for the total patient samples where the gene is expressed and was log2-transformed, for which heat map analysis combined with hierarchical clustering was performed with the &#39;hclust&#39; function of R package (https://stat.ethz.ch/R-manual/R-patched/library/stats/html/hclust.html).
Title: Statistical tests

We used R (v 3.1.3) for all statistical tests (https://cran.r-project.org/).
Using the factoMineR (http://factominer.free.fr) and rgl (https://r-forge.r-project.org/projects/rgl/) packages, principal component analysis (PCA) and visualization were performed [11].
The survival packages were used (http://r-forge.r-project.org) to compare patient survival rates and draw Kaplan-Meier plots.
The coxph function in the survival packages was used for performing the Cox regression analysis [1213].
Title: Selection of two extreme patient groups and one intermediate group determined by G64 expression

The G64 genes were originally identified by their coregulated expression pattern in single cells derived from a single LADC tumor region [10].
Interestingly, we have reported that 488 LADC patients samples downloaded from the TCGA were also separated into two distinct groups by the G64 genes.
In the present work, we examined whether G64 could be a classifier for other cancers, as well.
For this analysis, 22 RNA-seq-based transcriptome data with a sample size ≥150 were retrieved from the TCGA.
The threshold of 150 was selected, because it was considered to be the minimum number of samples for statistical tests between different groups (Table 1, Supplementary Table 1).
As done in the previous LADC study [10], heat map analysis was performed for each cancer type to see how patient samples were separated by G64 (Supplementary Fig.1).
As shown in Supplementary Fig.1, the co-regulated patterns of G64 expression were confirmed in all types of cancers we tested.
However, distinct two-group separations were not as evident as the case for LADC.
Therefore, we classifies patient samples further into three different categories by using a 15% extreme threshold of G64 expression (Supplementary Fig.2): the highest 15% group, the lowest 15% group, and an intermediate group.
For this purpose, the patients were sorted out by the intensity of the average FPKM values of the G64 genes expressed in each patient.
As a result, for instance, among a total of 531 kidney renal clear cell carcinoma (KIRC) samples, 160 samples were assigned into the two extreme groups—the highest 15% and the lowest 15%—whereas the remaining 371 samples were classified into the intermediate group.
Title: G64 up-regulating cancers tend to have a poor prognosis in six different cancers

We next investigated whether patient survival was significantly different in these three groups classified by the average intensity of G64 expression.
Patient survival rates were compared among the three patient groups for each of the 22 cancers.
As done in the LADC study, we performed Kaplan-Meier analysis using the patients&#39; survival data obtained from the TCGA in each type of cancer.
Survival rates were compared between the lowest 15% versus highest 15% groups (i.e., P1 comparison in Table 1), the lowest 15% versus intermediate groups (i.e., P2 comparison in Table 1), and the intermediate versus highest 15% groups (i.e., P3 comparison in Table 1).
PCA confirmed the groups&#39; separations by G64 expression (Fig.1, Supplementary Fig.3, 4, 5, 6, 7).
As summarized in Table 1, only six cancers were shown to have statistically significant differences between groups, including KIRC, brain lower grade glioma, liver hepatocellular carcinoma, kidney renal papilloma cell carcinoma, esophageal carcinoma, and sarcoma.
In all comparisons between groups in these 6 cancers, G64 up-regulation was consistently observed to be associated with worse patient prognosis in the Cox regression analysis (Fig.1, Supplementary Fig.3, 4, 5, 6, 7, 8).
The 22 cancers were subsequently divided into two batches: i.e., six survival-differentiating cancers and 16 survivalnon- differentiating cancers.
After the RNA-seq samples from the 22 cancers were collected as separate batches, we examined the aspect of survival difference by G64 expression in each batch.
Consequently, the survival-differentiating batch, mixing up six different cancer samples (531 + 514 + 351 + 289 + 184 + 257 = 2,126 samples), clearly maintained the survival differentiation among the lowest 15%, the highest 15%, and the intermediate groups (Fig.2), whereas the survival-non-differentiating batch, mixing up 16 different cancer samples (178 + 373 + 302 + ….
+ 179 + 150 = 5943 samples), showed no survival difference among the three different groups determined by G64 expression levels (Fig.2).
Taken together, the G64 genes can differentiate cancer samples by their survival rate differences in cancers other than LADC.
Basically, it was an interesting finding that genes, named the G64 module in our previous study (i.e., a highly coregulated gene group), can classify not only single cells derived from a single tumor but also tumors that have originated from several different cancers.
Various biomarkers have been identified and developed for molecular diagnostics in cancers.
In fact, numerous genes included in the list of the G64 module have already been identified as cancer diagnostic or prognostic markers by several independent researchers [141516].
For instance, CDCA5 and NCAPH have been characterized to be highly expressed in lung cancers [1718].
Our previous study showed that cell cycle genes were a main functional GO category in the G64 module.
Consistently, dysregulation of the cell cycle has long been proven to be a critical process causing cancers [19202122].
However, to our knowledge, it was the first finding ever that G64 genes were co-regulated in various cancers, and patient prognosis can be differentiated by these genes.
Here, we clearly showed that the G64 module can be a good biomarker, predicting cancer prognosis at least for six different cancers—i.e., seven different cancers if LADC is included.
It is unclear what common molecular characteristics the seven cancers (i.e., 6 cancers in the present study and LADC) share together other than co-regulated G64 expression.
The seven cancers must be initiated and progress by different genetic or environmental causes and pathways, and there is no proper explanation for the common subpopulation differentiation carried through G64 expression.
It is also hard to explain at this moment why the 64 genes are co-regulated on an inter-tumoral level and intra-tumoral level and why up-regulating patients tend to have a poor prognosis.
One possibility we can think of is that expression of these genes may be associated with drug metabolism, contributing to drug sensitivity or drug resistance.
It will be interesting to study further how expression of G64 is changed during the recurrence of cancers.
Taken together, G64 genes would be a good candidate of developing prognostic markers for multiple cancers.
Heat map, principal component analysis (PCA), and Kaplan Meier (KM) plot analysis of G64 in kidney renal clear cell carcinoma (KIRC).
(A) Heat map was plotted using the 160 samples corresponding to the highest 15% and lowest 15% of G64 expression among the total of 531 KIRC samples.
(B) A PCA plot of G64 expression was performed using the 531 KIRC samples.
Low, the lowest 15% group; INT, intermediate group; High, the highest 15% group.
(C) KM analysis of the 531 KIRC samples based on the average G64 expression.
Cox regression analysis was used to investigate whether the survival duration of the two different groups was significantly different (p&lt;0.001), as shown in Table 1.
KM analysis of combined samples of cancers on expression of G64.
(A) KM plot of the combined six samples of the survival-differentiating cancers by G64, including kidney renal clear cell carcinoma, brain lower-grade glioma, liver hepatocellular carcinoma, kidney renal papilloma cell carcinoma, esophageal carcinoma, and sarcoma (see Table 1, upper six cancers).
(B) KM plot of the 16 remaining samples of the survivalnon- differentiating cancers (see Table 1, lower 16 cancers).
Low, the lowest 15% group; INT, intermediate group; High, the highest 15% group; HR, hazard ratio; 95% CI, 95% confidence interval.
Title: List of the 22 cancers
Title: Prediction of Genes Related to Positive Selection Using Whole-Genome Resequencing in Three Commercial Pig Breeds

Selective sweep can cause genetic differentiation across populations, which allows for the identification of possible causative regions/genes underlying important traits.
The pig has experienced a long history of allele frequency changes through artificial selection in the domestication process.
We obtained an average of 329,482,871 sequence reads for 24 pigs from three pig breeds: Yorkshire (n = 5), Landrace (n = 13), and Duroc (n = 6).
An average read depth of 11.7 was obtained using whole-genome resequencing on an Illumina HiSeq2000 platform.
In this study, cross-population extended haplotype homozygosity and cross-population composite likelihood ratio tests were implemented to detect genes experiencing positive selection for the genome-wide resequencing data generated from three commercial pig breeds.
In our results, 26, 7, and 14 genes from Yorkshire, Landrace, and Duroc, respectively were detected by two kinds of statistical tests.
Significant evidence for positive selection was identified on genes ST6GALNAC2 and EPHX1 in Yorkshire, PARK2 in Landrace, and BMP6, SLA-DQA1, and PRKG1 in Duroc.
These genes are reportedly relevant to lactation, reproduction, meat quality, and growth traits.
To understand how these single nucleotide polymorphisms (SNPs) related positive selection affect protein function, we analyzed the effect of non-synonymous SNPs.
Three SNPs (rs324509622, rs80931851, and rs80937718) in the SLA-DQA1 gene were significant in the enrichment tests, indicating strong evidence for positive selection in Duroc.
Our analyses identified genes under positive selection for lactation, reproduction, and meat-quality and growth traits in Yorkshire, Landrace, and Duroc, respectively.
Artificial selection is an aspect of the process of domestication development [1] that is reflected in the evolution of domestic animals.
Positive selection increases the fitness of adaptive traits [2].
Previous research has demonstrated that the selection and mating of domestic animals is determined by human-generated pressures, and the genetic differences resulting from artificial selection have led to economic development [3].
Among domestic animals, the pig has experienced a particularly long history of haplotype changes through artificial selection in the process of domestication [24].
For most of their history, studies have found positive selection in Yorkshire, Landrace, and Duroc to be associated with specific genes related to lactation [56], reproduction [7], meat quality [8], and growth [3] trait.
Given the importance of these traits to the pig farming industry, it is necessary to detect the signals of positive selection in pigs.
Identifying genomic selection using high-density single nucleotide polymorphism (SNP) arrays or sequencing data is useful for the discovery of putative trait-related genes.
A variant under selection pressure between and/or within populations will show an increase of the frequency of particular alleles or linkage disequilibrium (LD) of SNPs.
Selective sweep can cause genetic differentiation across populations [9].
The cross-population extended haplotype homozygosity (XPEHH) [10] and cross-population composite likelihood ratio (XPCLR) [9] tests are widely used to search for signals of selection between the populations.
The XPEHH test detects the occurrence of selection based on measuring LD, while the XPCLR test considers the spatial patterns of allele frequencies of SNPs [9].
The XPCLR test is a likelihood method that uses differentiation of multi-locus allele frequencies between two populations to detect selective sweep.
Selection with significant signals can break up the haplotype structures more rapidly than mutation or recombination processes [11].
In this study, the XPEHH and XPCLR between-population methods were implemented to detect positive selection in three pig breeds through resequencing using the Illumina HiSeq2000 platform (Illumina, Inc., San Diego, CA, USA).
To further explain the biological implications of positively selected genes, functional enrichment analysis was performed.
Title: Sampling and whole-genome sequencing

We used genomic DNA samples gathered from 12 males and 12 females of three pig breeds: Yorkshire, 2 males and 3 females; Landrace, 7 males and 6 females; and Duroc, 3 males and 3 females.
Blood samples were collected for DNA extraction.
Sample collections and DNA quality check procedures were performed according to the manufacturer&#39;s instructions.
Next, we constructed genomic DNA libraries for each sample using TruSeq DNA Library kits (Illumina).
The paired-end library was sequenced on an Illumina HiSeq 2000 sequencing platform.
The pair-end sequence reads were aligned to the reference pig genome sequence from University of California, Santa Cruz (UCSC; http://genome.ucsc.edu/; susScr3) using Bowtie2 with the default setting.
We used the following open-source software: Bowtie2, Picard tools 1.94 (http://picard.sourceforge.net), Samtools 0.1.19 [12], Genome Analysis Toolkit (GATK) 2.6.4 [13], and VCFtools 4.0 [14] for resequencing data processing and SNP calling.
Substitution calling was performed using GATK Unified-Genotyper.
We phased the haplotypes for the entire pig populations using BEAGLE [15].
Picard tools was used for duplicate read removal and all mate-pair information confirmation.
Samtools was used for indexing the results from bam files and calculating the mapped reads using the flagstat option.
GATK was used for realignment and SNP calling from resequencing data, and VCFtools was used when VCF files were handled.
After filtering, non-biallelic SNPs were excluded.
Title: Detection of selective sweep

To detect selective sweep, we implemented two between-populations methods (XPEHH and XPCLR) between each pairwise breed contrast.
These statistics used the phased data based on the SNP genotypes obtained from the wholegenome resequencing.
For each contrast, XPEHH and XPCLR were implemented across all SNPs between the three pig breeds to identify potential positive selection.
We used a p-value of &lt;0.01 for XPEHH and the top 1,000 genes in XPCLR as the cutoff criteria.
These tests revealed that some SNPs might have been under positive selection in the Yorkshire, Landrace, and Duroc breeds, respectively.
Title: Bioinformatic analysis of genes under positive selection

Based on the findings of positive selection, enrichment analysis was performed to examine the biological functions of genes in detected regions.
In this study, regions with SNPs under positive selection were extended approximately 10 kb upstream and downstream [16].
We assembled genes located within the extended region using the RefGene from the UCSC Genome Browser (http://genome.ucsc.edu/; ver.hg19).
Gene enrichment analysis was performed GO terms [17], including biological process, molecular function, cellular component, and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway analysis [18] using the DAVID (http://david.abcc.ncifcrf.gov/) tool [19].
Title: Analysis of SNP effects

To identify non-synonymous coding SNPs (nsSNPs), we analyzed SNP effects using the SNPeffect database, which detects phenotypic effects of variation (http://snpeffect.vib.be/) [20].
For more exact results, a Fisher&#39;s exact test was performed to identify breed-specific amino acids in genes with nsSNPs.
The statistical test was used to analyze 2 × 2 contingency tables composed with two factors.
Considering our purpose, one factor was breed information, and the other was amino acid information.
From the 2 × 2 contingency tables, we expected to observe enrichment of specific amino acids in specific breeds at each position.
All data parsing and calculations were performed using Python (ver.
2.5) and R (ver.
3.1.2).
Title: Sequencing and alignments

Genome-wide positive selection in pigs was identified using whole-genome resequencing, in which we aligned whole genome reads from three pig breeds, Yorkshire, Landrace, and Duroc, to the pig reference sequence.
These breeds have undergone positive selection with strong signals for lactation, reproduction, meat quality, and growth.
On average, ~329,482,871 sequences were mapped to the pig reference genome from the University of California, Santa Cruz (UCSC) Genome Browser (assembly ID: susScr3) using Bowtie2.
Table 1 provides a summary of the alignments of 24 pig samples.
Over 92.8% of all reads were aligned to the pig reference genome.
Pig resequencing data with an average read depth of 11.7× were generated using the Illumina HiSeq2000 platform.
A total of 18,440,443 SNPs were identified using the GATK.
Title: Selective sweep detected by between-population methods

To detect signals of positive selection in the three pig breeds, two between-population methods (XPEHH and XPCLR) were implemented with phased haplotypes based on pairwise populations of Yorkshire-Landrace (Y-L), Yorkshire-Duroc (Y-D), Landrace-Yorkshire (L-Y), Landrace-Duroc (L-D), Duroc-Yorkshire (D-Y), and Duroc-Landrace (D-L).
Here, for Y-L, Yorkshire was regarded as the observed population and Landrace as the reference population.
XPEHH and XPCLR scores denote selection that happened in the observed population.
As a result, several regions of positive selection were identified at a significance level of p-value &lt; 0.01 [21] in XPEHH, and the top 1,000 genes in XPCLR.
Table 2 summarizes the numbers of genes under significant positive selection, which were identified in the six breed pairs (Y-L, Y-D, L-Y, L-D, D-L, and D-Y).
For example, for the Y-L breed pair, which detects selection that occurred in the Yorkshire population when compared to the Landrace population, 376 and 1,869 genes were under positive selection based on the XPEHH and XPCLR tests, respectively.
For the Y-D breed pair, when using the XPCLR test, 1,498 genes were under positive selection with 3,978 SNPs detected in Yorkshire, with Duroc as the reference population.
By the consensus set of genes under positive selection identified with both the XPEHH and XPCLR tests, 26, 7, and 14 genes were identified in Yorkshire, Landrace, and Duroc, respectively (Fig.1).
For example, the 26 Yorkshire genes identified by both the XPCLR and XPEHH tests in both the Y-L and Y-D comparisons, and so effectively detected four times.
Title: The biological functions of genes under positive selection

We performed functional enrichment analysis on the following genes under positive selection using DAVID ver.6.7: ST6GALNAC2 and EPHX1 in Yorkshire; PARK2 in Landrace; and BMP6, SLA-DQA1, and PRKG1 in Duroc.
The results revealed enrichment of Kyoto Encyclopedia of Genes and Genomes (KEGG) Pathway terms related to metabolism, diseases, genetic or environmental information processing, and organismal systems (Table 3).
The gene BMP6 acts as a hedgehog signaling pathway; the hedgehog protein is an important regulator of slow oxidative fiber clustering and is distributed throughout the tissue in pigs.
Previous research has reported that BMP6 plays an important role in meat quality [8].
Among the 26 positively selected genes in Yorkshire, the ST6GALNAC2 gene is expressed during lactation, contributing to immune-related functions [5].
Up-regulation of the EPHX1 (epoxide hydrolase 1) gene occurs during mammary gland involution.
Nakamura et al.
[6] suggested that EPHX1 is involved in the initial stage of involution.
The SEMA5A (semaphoring 5A) gene for Yorkshire reported strong candidate gene which affects the milk production traits for economic traits in cattle.
The estrogen receptor (ER) gene for Landrace reported that mice lacking ER α show severe reproduction [7].
This implies that the ER gene might be a positively selected lactation trait in pigs.
The Gene Ontology (GO) terms included immune system process (SLA-DQA1) and metabolic process (PRKG1) for Duroc (Supplementary Table 2).
Williams et al.
[3] reported that a low level of chronic immune system resulted in greater growth in pigs.
These results suggest that the 26, 7, and 14 population-specific genes with significant signals may result from potential positive selection for lactation in Yorkshire, reproduction in Landrace, and meat-quality and growth traits in Duroc.
Title: SNP effects for population-specific genes

To understand how SNPs related positive selection affect protein function, we analyzed SNP effects using the SNPeffect database.
We identified two genes, SLA-DQA1 and EPHX1, with nsSNPs showing evidence of positive selection in D-Y, respectively (Fig.2, Supplementary Table 1).
To obtain accurate results, Fisher&#39;s exact test was performed on these two genes.
The total numbers of statistical tests performed on the non-synonymous genes were 20 × 3 × 13 (number of cases of amino acids × number of breeds × number of non-synonymous amino acid positions) for SLA-DQA1, and 20 × 3 × 4 for EPHX1.
From the enrichment tests, we identified three positions (rs324509622, rs80931851, and rs80937718) that were significant under a Bonferroni-adjusted p-value of &lt;0.05 with an alternative hypothesis: the odds ratio is greater than 1 in the SLA-DQA1 gene.
The p-values of the three non-synonymous positions were 5.20e-05, 5.20e-05, and 7.43e-06, respectively.
As a note, the above three significant enrichment results were observed only in the Duroc breed.
The non-synonymous positions showed strong evidence for positive selection between D-Y based on the XPEHH test.
As the result, we validated the Duroc-specific enrichment of amino acids affected by non-synonymous SNPs using statistical tests.
This result will contribute to improving the meat quality of pigs.
Identifying positive selection assists in the exploration of the genetic mechanisms of phenotypic diversity [2].
Selective sweep causes changes in the allele frequencies of SNPs, and the resulting changes in genetic variation can be useful in detecting selective pressure [9].
Although several studies have reported selective sweeps identified using chips such as the Illumina Porcine 60K SNP BeadChip [22] in pigs, few have identified selective sweeps using whole-genome sequencing.
Therefore, detecting positive selection in the pig genome appears worthwhile.
Here we implemented two representative between-population methods, XPEHH and XPCLR, to explore positive selection.
XPEHH is designed to detect near fixation or fixed selective sweeps by comparing haplotypes between two populations [10].
XPCLR is more robust for SNP discovery bias than allele frequency-based methods [9].
XPEHH tests whether the SNP site is homozygous in one population and polymorphic in another by comparing the extended haplotype homozygosity score of two populations on a core SNP.
A negative XPEHH value indicates that selection happened in the reference population [15].
XPCLR was designed as a multi-locus composite likelihood ratio method to avoid the bias influence in SNP discovery.
XPEHH and XPCLR find one core SNP and grid window using multi-locus information, and thus these tests identify more significant selection regions than Fst or Tajima&#39;s D [23].
Identifying significant genome-wide selection regions with multiple loci using a single-marker Fst test is difficult.
Therefore, for multiple SNP methods, we carried out XPEHH and XPCLR and excluded Fst and Tajima&#39;s D. XPEHH assumes an approximately normal distribution.
Park et al.
[16] determined empirical p-values using the top 1% of the significance, and our results show that these significance cutoffs are more reasonable.
In addition, positive selection identified by multiple test methods is more convincing.
Bioinformatic analysis of genes under positive selection in this study revealed interesting biological information related to lactation, reproduction, meat quality, and growth.
We identified several GO terms using the DAVID functional annotation tool.
Immune system process (GO:0002376) was found to be significant in Duroc.
Williams et al.
[3] reported that a low level of chronic immune system resulted in greater growth in pigs.
This GO term suggests that genes with some growth traits may have been under selection in the domestication process.
Growth traits are referred to as important indicators of the immune system.
This suggests that SLA-DQA1 might play a role in the pig immune system.
In addition, BMP6 and PRKG1 in Duroc had biological pathways related to meat-quality traits.
Fonseca et al.
[8] reported that the BMP6 gene plays a role in increasing the quality of meat juiciness and tenderness.
Lee et al.
[22] identified the significant SNPs located in the BMP6 gene related to meat quality through a genome-wide association study using the Porcine 60K BeadChip.
Therefore, the SLA-DQA1 and BMP6 genes might be effects of positive selection for growth and meat quality, respectively, in Duroc.
In Yorkshire, three genes under significant positive selection were ST6GALNAC2, EPHX1, and SEMA5A; these genes were related to lactation traits in several previous studies.
The ST6GALNAC2 gene is expressed during lactation [5], EPHX1 is expressed in the early involution of mammary glands [6], and SEMA5A is reportedly related to milk production traits in cattle [21].
In Landrace, the ER gene was reported to be related to reproduction traits in previous studies.
Generally ER refers to either ER α or ER β and estrogens are involved in differentiation and maintenance of reproductive tissue.
Krege et al.
[7] reported that mice lacking ER α exhibited severe reproductive problems.
This implies that the ER gene could be an effect of positive selection for lactation traits in pigs as well as in mice.
Our findings show that several significant SNPs were present in genes found to be related to lactation, reproduction, meat quality, and growth traits in previous studies.
These results indirectly support the effects of positive selection.
Unfortunately, information on many of the genes with significant signals has not yet been analyzed.
Based on SNP analyses for positively selected genes, SLA-DQA1 and EPHX1 genes exhibited evidence of positive selection in D-Y, respectively.
However, an enrichment test revealed that the three significant amino-acid positions in SLA-DQA1 and EPHX1 were not significant.
Our analyses suggested that several genes–such as EPHX1, PARK2, BMP6, SLA-DQA1, PRKG1, and ST6GALNAC2–might have undergone positive selection.
These genes are associated with lactation, reproduction, meat quality, and growth traits.
All findings in this study provide useful data for the study of pigs, however our results are preliminary and need to validate these genes in another set of samples.
Visualization of genome-wide positive selection.
Manhattan plots of the degree of selection signals based on pairwise populations of three pig breeds using cross-population extended haplotype homozygosity (A) and cross-population composite likelihood ratio (B).
Red lines indicate the significance level and colored dots indicate the genes under significant positive selection.
Y-L, Yorkshire-Landrace; Y-D, Yorkshire-Duroc; L-D, Landrace-Duroc; L-Y, Landrace-Yorkshire; D-L, Duroc-Landrace; D-Y, Duroc-Yorkshire.
Visualization of single nucleotide polymorphism (SNP) effects on positively selected genes.
(A) Visualization of the sequence logo for the SLA-DQA1 gene with 13 non-synonymous SNPs (nsSNPs) showing evidence of positive selection in Duroc.
(B) Visualization of enrichment tests for the 13 nsSNPs.
The red box indicates the significant positive selection regions under the Bonferroni adjusted p-value &lt; 0.05.
Title: Summary of alignment rates in 24 pig samples

Title: Summary of genes under significant positive selection

Title: Enriched KEGG pathways associated with positively selected genes
Title: Heritability Estimated Using 50K SNPs Indicates Missing Heritability Problem in Holstein Breeding

Previous studies in Holstein have shown 35% to 51.8% heritability in milk production traits, such as milk yield, fat, and protein, using pedigree data.
Other studies in complex human traits could be captured by common single-nucleotide polymorphisms (SNPs), and their genetic variations, attributed to chromosomes, are in proportion to their length.
Using genome-wide estimation and partitioning approaches, we analyzed three quantitative Holstein traits relevant to milk production in Korean Holstein data harvested from 462 individuals genotyped for 54,609 SNPs.
For all three traits (milk yield, fat, and protein), we estimated a nominally significant (p = 0.1) proportion of variance explained by all SNPs on the Illumina BovineSNP50 Beadchip (h2G).
These common SNPs explained approximately most of the narrow-sense heritability.
Longer genomic regions tended to provide more phenotypic variation information, with a correlation of 0.46~0.53 between the estimate of variance explained by individual chromosomes and their physical length.
These results suggested that polygenicity was ubiquitous for Holstein milk production traits.
These results will expand our knowledge on recent animal breeding, such as genomic selection in Holstein.
New-generation sequencing technologies have substantially enhanced the development of genomic tools to assist in breeding decisions.
Among many techniques, genotyping technologies have enabled breeders or researchers to identify DNA regions or quantitative trait loci (QTL) associated with a particular phenotypic trait of domesticated animals.
Especially, several single-nucleotide polymorphisms (SNPs) and QTL associated with Holstein complex traits, including economic traits related to milk production, were investigated in the approximate 10-year wave of genome-wide association studies (GWASs) [123].
In the near past, researchers discovered genetic markers and created marker panels for use in performing marker-assisted selection (MAS).
Although MAS provided a first genomic approach to achieve breeding goals to animal breeders, the domesticated animal genome in MAS was taken lightly with narrow perspectives.
MAS is restricted when it comes to predicting animal capacity, as it depends on a small number of QTLs that are tagged by markers associated with each trait [4].
Additionally, the gap between the proportion of phenotypic variance accounted for by the top SNPs that reach genome-wide significance level in a GWAS and the heritability estimated from pedigree analyses remained unexplained for most complex traits of human.
This was called the missing heritability problem [5], explanations to which have been debated in the field [6].
Recent studies using whole-genome estimation approaches demonstrated that a large proportion of heritability for complex traits of humans can be captured by all common SNPs on current genotyping arrays.
It implies that there are a large number of variants with an effect that is too small to pass the stringent genome-wide significance level.
In animal breeding, genomics selection (GS) has been developed to overcome restriction factors of MAS; it has covered a large number of variants containing small effects.
The fundamental difference between MAS and GS as important animal selection tools is that scale GS embraces a much larger number of markers than MAS in predicting animal capacity.
GS uses a dense set of markers from across the entire genome to use all markers containing SNPs with small effects.
This feature gives GS a profound advantage over MAS.
For predictions based on genomic data for humans, a previous study has already estimated the proportion of phenotypic variance.
It was elucidated by the common SNPs all together on a genotyping array for a range of quantitative traits in a large homogenous human sample, using the whole-genome estimation and partitioning approaches [7].
The novelty of our research lies in animal capacity prediction based on genomic data due to the fact that animal evaluation is a core content in the breeding industry.
Finally, to conduct our animal breeding research on polygenic inheritance, which was likely to be ubiquitous for complex Holstein traits associated with milk production traits, the genomic selection technique was performed along with various different analyses.
Title: Korean Holstein data

This study used 462 Holsteins from Korea, and they had information on three phenotypes related to milk production for parity 1 (Fig.1).
Three traits used in this study were milk yield, milk fat, and milk protein; 339 of 462 Holsteins in records of parity 1 had records of parity 2.
The individuals produced milk in 63 farms and were born in 2005 to 2012.
All candidates were measured for a range of quantitative traits through public surveys for livestock improvement.
Title: Genotyped and imputed data

The genomic DNAs were isolated from snivel by using the nasal collection kit and were genotyped with 54,609 SNPs on the Illumina BovineSNP50 BeadChip.
We excluded the SNPs with a missingness rate of 0.05, minor allele frequency (MAF) 0.01, and Hardy-Weinberg equilibrium test p-value 10-6 using PLINK [8].
Then, we removed SNPs on the sex chromosome and retained 41,099 autosomal SNPs for further analysis.
After this quality control, 462 Holstein genomic data had been imputed using BEAGLE [9].
Title: Estimating and partitioning genetic variance using SNP data

Title: We estimated the genetic relationship matrix (GRM) between all pairs of individuals from all genotyped SNPs For each trait, we then estimated the variance that can be captured by all SNPs using the restricted maximum likelihood approach in a mixed linear model: y = Xb + gG + e

Title: where y is a vector of phenotypes, b is a vector of fixed effects with its incidence matrix X, and gG is a vector of aggregate effects of all SNPs.
Var(gG) = AGσ2G

Title: AG is the SNP-derived GRM, and σ2G is the additive genetic variance.
The proportion of variance explained by all SNPs is defined as h2G = σ2G σσ2p

σ2p is the phenotypic variance.
Details of the model and parameter estimation have been described elsewhere [1011].
When we estimated genetic variance using SNP data, we have to consider seasonal effects (winter, December–February; spring, March–May; summer, June–August; and autumn, September–November based on birth date) as environmental effects, because milk production traits are closely related to season.
Then, we performed ANOVA test to investigate the relationship between each trait (parity 1 and 2) and seasonal effect.
Milk fat had only a close relation to season in the ANOVA test of both parity 1 (p = 0.0267) and 2 (p = 0.000432).
We counted on seasonal effect in only the analysis including milk fat.
In addition, using the same method as above but allowing multiple genetic components to be fit simultaneously in the model, we partitioned h2G into the contributions of genic (h2Gg) and intergenic (h2Gi) regions of the whole genome across all traits.
The genic regions were defined as ±0 kb of the 3&#39; and 5&#39; untranslated regions (UTRs).
A total of 13,297 SNPs were located within the boundaries of 7001 protein-coding genes for this definition (±0 kb).
The total length of the 7001 protein-coding genes was approximately 585 Mb, which covered 21.91% of the genome.
We performed these estimating and partitioning genetic variance using SNP data through GCTA [11].
We used the data from the Korea Holstein population.
These data were collected from 462 cows recruited from 63 farms in South Korea, genotyped at 54,609 SNPs on the Illumina BovineSNP50 BeadChip.
There were 462 individuals and 41,099 autosomal SNPs after quality control (Methods section).
All individuals were measured for three traits (milk yield, milk fat, and milk protein), which were related to milk production.
The phenotypic correlations between pairwise traits are visualized in Supplementary Fig.1.
Correlations between milk yield and milk protein were much stronger than other pairwise comparisons.
We estimated the proportion of variance explained by fitting all SNPs in a mixed linear model for each of the three traits.
In general, there was a substantial amount of variance explained by all autosomal SNPs after quality control on the Illumina BovineSNP50 BeadChip (σ2G) for three traits of the records of two parities, with a mean of 43.51% (a range from 36.3 to 47%) across all three traits of the records of two parities (Table 1).
In the records, the estimate of h2G was non-zero and reached the nominal significance level (likelihood ratio test, p = 0.05).
We compared the estimates of h2G with the narrow-sense heritability h2, estimated from pedigree analyses in the literature based on Canada Holstein.
The value of h2G was similar to h2 from pedigree analysis in the literature, and all common SNPs explained most (average, 43.51%; range from 36.3% to 47%) of the narrow-sense heritability (average, 41.38%; range from 36.3% to 47%), despite the estimates of h2 being from different country populations (Table 1).
In contrast, when we performed a genome-wide association analysis in the same sample, we tried to identify genome-wide-significant (Bonfferoni correction p = 0.05) SNPs for three traits of parity 1.
Three SNPs and one SNP were significant in milk yield and fat, respectively, and there were no significant SNPs for milk protein (Supplementary Fig.2).
We hypothesized that there was a major reason for only a few significant large-effect SNPs associated with Holstein traits related to milk production.
First of all, many SNPs with small effects affected the three traits related to milk production.
Second, according to previous studies, there are many common variants associated with the traits in humans at a nominally significant level, but their effect sizes are too small to be genome-wide-significant [7].
Using the same method as above but allowing multiple genetic components to be fit simultaneously in the model, we then partitioned h2G into the contributions of individual chromosomes for each of the three traits of two parities and plotted the estimate of variance explained by each chromosome (h2C) against the chromosome length for each trait.
In a previous study, no linear correlation between h2C and chromosome length for any particular traits was observed for most traits in humans as strongly as shown in the other studies.
Then, the average value of estimates of h2G over all of tens traits to reduce the sampling error variance, and the average estimated h2C was strongly correlated with chromosome length.
However, we dealt with only three traits related to milk production, and we aimed to identify a linear correlation between h2C and chromosome length for each trait.
The squared correlation between h2C and chromosome length using trait information of parity 1 was 0.46, 0.532, and 0.509 for milk yield, fat, and protein, respectively (Fig.2).
The regression slope of the proportion of the genetic variance attributed to each chromosome on the proportion of the genome represented by each chromosome was significant (p&lt;0.1), suggesting a proportional relationship between genome length and genetic variance in Holstein.
Consistent results were achieved from myriad genetic variants, each possessing a small effect throughout the widespread whole genome in Holstein.
In addition, we partitioned h2G into the contributions of genic (h2Gg) and intergenic (h2Gi) regions of the whole genome (Methods section).
We estimated the variance explained by the genic (h2Cg) and intergenic (h2Ci) regions of each chromosome.
The numbers of genic and intergenic SNPs on each chromosome are presented in Supplementary Table 2.
We showed that the variance explained by the genic (intergenic) regions on each chromosome is also proportional to the total length of the genic (intergenic) regions (Fig.3).
Previous studies using the whole-genome estimation approach have shown that common SNPs explicate a large proportion of heritability for traits in human [1012].
The reason why GWASs in this field have not yet identified all common SNPs that contain information on the amount of variation is mainly that many peculiar effects are too small to pass the stringent genome-wide significance level.
Therefore, we hypothesized that inheritance of Korean Holstein milk production traits would consist of a lot of genetic variants with small effects.
Finally, we estimated and partitioned the genetic variance that met certain conditions, tagged by all common SNPs for three complex traits related to milk production and showing ubiquitous polygenic inheritance of Korean Holstein.
The estimates of h2G for three traits were different from 0 at the significance level (p&lt;0.1).
The estimate of h2G for milk yield of parity 1 was 43.0% (SE = 10.3%), which was smaller than the estimate from a study in Canada (h2G = 52%) [13].
There could be three possible reasons: (1) There is a difference in the reference population size between Korea and Canada.
In fact, Koreans import a lot of Canada Holstein semen, but the present reference population is smaller than Canada.
(2) The Korean environment is different from Canada.
(3) We can consider only one environmental effect (season) due to data size in this study.
The estimate for milk fat (h2G = 47.0%, SE = 10.1%) and protein (h2G = 44.3%, SE = 10.2%) in Korea was larger than in Canada (fat h2G = 36.9%, protein h2G = 42.3%).
It is demonstrated by the genome partitioning analysis that there was a significant linear relationship between the estimates of variance explained by individual chromosomes and chromosome length (Fig.2).
If we define the genic region as ±0 kb of the UTRs, the correlation between variance explained and chromosome length was stronger in the intergenic regions than in the genic regions (Fig.3).
A previous study showed by a number of analyses that the result in this was driven neither by the difference between the number of SNPs in genic regions and in intergenic regions nor by the difference in MAF distribution between genic and intergenic SNPs [7].
If trait-associated genetic variants are enriched in functional elements, such as introns and UTRs, and diluted in exons, the relationship between the explained variance and DNA length will be attenuated in the genic region, as in human.
However, it could possibly be just a sampling problem.
Because our data size was smaller than our expectancy, some chromosomes did not reach the significance level (p = 0.01) in the genome partitioning analysis.
In the genome partitioning analysis for milk yield using SNPs in the intergenic region and milk protein using SNPs in the genic region, the linear regression results were also insignificant, and we could identify some outliers.
Moreover, when these outliers were removed, the linear regression results were significant (E in Fig.3).
We hypothesize that some chromosomes could have a slightly larger effect than our expectation, based on the fitted line in special cases (chromosome 17 or 4 in milk protein).
We showed by whole-genome estimation and partitioning analyses that Holstein milk production traits appeared to be highly polygenic, which means that there were a large number of genetic variants segregating in the population with a small effect widely distributed across the whole genome.
All common SNPs expressed most of the heritability on average over all 3 traits analyzed in this study.
The slight, remaining unexplained heritability could be due to causal variants, including common and rare ones that are not well tagged by SNPs on this Illumina SNP chip, or possibly because the heritability was over-estimated in the pedigree study.
In conclusion, heritability consists of many variants with small effects for Holstein milk production traits in Korea.
Overall, this research will become a cornerstone for genomic selection applications in animal breeding.
Phenotype distribution of the three traits related to milk production.
Proportion of variance attributed to each chromosome of three traits of parity 1 against chromosome length.
We show proportion of variance of each chromosome, which is significant based on likelihood test (p-value&lt;0.1).
(A) Milk yield (slope = 8.05 E-04, S.E = 4.31 E-04, p-value = 8.46 E-02).
(B) Milk fat (slope = 5.79 E-04, S.E = 1.84 E-04, p-value = 4.26 E-03).
(C) Milk protein (slope = 5.75 E-04, S.E = 2.97 E-04, p-value = 5.24 E-02).
Estimates of the variance explained by all SNPs in genic (intergenic) regions for 3 traits (a, b, and c indicate milk yield, fat, and protein, respectively) against length of genic (intergenic) DNA.
Shown in panels (A), (C), and (E) are the results for the genic SNPs, and shown on panels (B), (D), and (F) are the results for intergenic SNPs, under the assumption that genic regions is Wor0 Kb of UTRs.
(A) Milk yield, genic region (slope = 7.89 E-04, S.E = 3.30 E-04, p-value = 3.56 E-02).
(B) Milk yield, intergenic region (slope = 9.11 E-04, S.E = 6.70 E-04, p-value = 2.07 E-01).
(C) Milk fat, genic region (slope = 4.04 E-04, S.E = 2.20 E-04, p-value = 8.01 E-02).
(D) Milk fat, intergenic region (slope = 5.23 E-04, S.E = 1.78 E-04, p-value = 7.02 E-03).
(E) Milk protein, genic region (slope = 3.66 E-04, S.E = 2.24 E-04, p-value = 1.34 E-01).
(F) Milk protein, intergenic region (slope = 7.53 E-04, S.E = 2.24 E-04, p-value = 7.20 E-03).
Title: Estimates of variance explained by all SNPs for the three traits related to milk production
Title: Building the Frequency Profile of the Core Promoter Element Patterns in the Three ChromHMM Promoter States at 200bp Intervals: A Statistical Perspective

Recently, the Encyclopedia of DNA Elements (ENCODE) Analysis Working Group converted data from ChIP-seq analyses from the Broad Histone track into 15 corresponding chromatic maps that label sequences with different kinds of histone modifications in promoter regions.
Here, we publish a frequency profile of the three ChromHMM promoter states, at 200-bp intervals, with particular reference to the existence of sequence patterns of promoter elements, GC-richness, and transcription starting sites.
Through detailed and diligent analysis of promoter regions, researchers will be able to uncover new and significant information about transcription initiation and gene function.
Both eukaryotic and prokaryotic cells contain important gene-regulating elements, located within promoter regions [12].
It is known that promoter sequences possess special signatures to distinguish them from the rest of the genome sequences.
A few core promoter elements have been detected, of which the most common elements are the CpG island, TATA box, initiator (Inr), downstream promoter element (DPE), and TFIIB recognition element (BRE) [2], but these promoter elements may not be universal.
Recently, a group from the Encyclopedia of DNA Elements (ENCODE) project published chromatic maps to mark histone modifications within nine cells of different types from the human genome, drawn from the results from chromatin immunoprecipitation sequencing (ChIP-seq) analyses of these same cells [12].
From each of these nine cells, Ernst et al.
[34] published 15 chromatin states of the human genome.
Among them, the first three states represent promoter regions: state 1 (Active_Promoter), state 2 (Weak_Promoter), and state 3 (Poised_Promoter).
The information gained from the exploration of promoter elements and transcription starting sites will enable researchers to produce more accurate promoter prediction algorithms, which in turn will yield deeper insight into the process of transcription initiation and gene functions.
Here, we publish frequency profiles of the ChromHMM promoter states of the human genome, at 200bp intervals, paying particular attention to the existence of promoter elements, such as CpG islands, TATA boxes, Inr, DPE, and recognition elements from the transcription factors that bind with polymerase II (BRE) (signal features), together with transcription starting sites, in terms of size, overlapping patterns, and locations.
Ernst et al.
[4] applied unsupervised learning methods to convert the Broad Histone track&#39;s results from ChIP-seq analyses into specific points that indicate promoter regions on chromatin maps for 15 chromatin elements within the genome.
These 15 chromatin states were chosen because of their rich biological content, as well their well-established presence in previous research [4].
The resulting files are hosted by the ENCODE Analysis Data Hub, which is available for public download under the ENCODE Data Release Policy (http://genome.ucsc.edu/ENCODE/downloads.html).
We downloaded the BED files for the chromatin signal tracks of the nine ENCODE cell types.
Within them, chromatin states 1, 2, and 3 represent Active, Weak, and Posed Promoters, respectively, which differ only in regard to the levels of expression in various biological assays.
These states were defined, based purely on 9 chromatin marks, and their input controls across 9 cell types.
Depending upon the cell type, slight variances were found, as seen in Table 1.
For example, the gm12878 cell line has an active promoter consisting of 14,674 blocks and occupies 0.80% of the whole genome, whereas k562 has 14,951 blocks and occupies 0.68% of the whole genome, for chromatin state 1.
To overcome this slight variance in cell lines, we divided the promoter regions uniformly at 200-bp resolution, regardless of the 9 cell line block boundaries.
Fig.1 shows our Java code, which relies on Java&#39;s Oracle interface (Oracle Database 12c Standard Edition) and implements regular expressions, based on predefined classes from the java.util.regex package, in order to pattern-match different promoter elements, such as CpG islands, TATA boxs, Inr, DPE, and BRE, for each 200-bp window.
The code also builds upon formal grammar rules based on the structural properties seen in promoter sequences, though these patterns usually apply only to the core promoters [56].
Our Java code produces an output profile consisting of the 18 required fields as follows (Fig.2).
The first three fields are: the name of the chromosome, the starting position of the feature in the chromosome, and the ending position.
The next four fields represent the frequencies of the 4 promoter elements.
For example, Fig.2 shows that the nucleotide sequences starting from 28337 to 23537 (200 bp) in chromosome 1 contains 1 occurrence of a TATA box, 6 occurrences of Inr, 0 occurrences of BRE, and 6 occurrences of DPE.
The next 9 fields indicate the ChromHMM annotation of 9 cell lines (K562, GM12878, H1HESC, HEPG2, HMEC, HSMM, HUVEC, NHEK, and NHLF).
The last two fields indicate the existence of DataBase of Human Transcription Start Sites (dbTSS) data [78], and the GC ratio, respectively, where the hg19promoter file for dbTSS data was used for indexing (ftp://ftp.hgc.jp/pub/hgc/db/dbtss/dbtss_ver8/Sanger_data/).
Table 2 shows the summary of our promoter frequency profile.
The data show significants difference from the known statistical factors of core promoters.
It is due to the fact that we divided the regions into 200 bp lengths whereas the core promoter elements are known to exist typically within 50 bp (base pairs) upstream to 50 bp downstream of the TSS [2], and that we are including the Poised_Promoter state of the ChromHMM.
From the ChromHMM human promoter blocks, the results of our statistical survey on 176,579 units show that DPE sequences have the highest frequency among the four chosen motifs in promoter elements, appearing in about 97.1% of the ChromHMM promoter regions, meanwhile Inr is seen in 92% of promoter regions, and BRE is seen in only 17.3% of the promoters (It is known that Inr usually occurs only about half of the human promoters [3]).
TATA boxes are found in 52.2% of the ChromHMM promoter blocks.
Some contextual patterns have been recognized for certain promoter elements.
For example, we can expect to find BREs within GC-rich areas, meanwhile areas with TATA boxes are typically GC-poor.
In total, there are 9,435 ChromHMM promoter blocks in the dbTSS locations (approximately 5.34%).
The number of 200-bp blocks that contain all four promoter elements was 176,579.
Among them, only 9,436 of the blocks contain dbTSS data [78].
Some of the blocks showed distinct associations with dbTSS data.
For example, the promoter blocks that do not contain any of the promoter elements tend to have a higher GC ratio (0.68) where a dbTSS location exists.
One of the strengths of our work versus previous studies is that we profiled the core elements separately, based on the three classes of promoter states (Active_Promoter, Weak_Promoter, and Poised_Promoter of the ChromHMM).
A more detailed analysis of frequency profiles in regard to these three states is beyond the scope of this paper.
However, the complete frequency profiles and Java code are published through GitHub (https://github.com/KyungEunLee/motif).
We believe that the frequency profiles presented of the three promoter states can be helpful for bioinformaticians to gain a deeper understanding of the characteristics of the various promoter regions and to improve upon old methods of core promoter analysis algorithms for their specific research needs.
Java code to recognize regular expressions of different promoter elements: TATA-box, Initiator, TFIIB recognition element, and downstream promoter element.
An output profile format consisting of the 18 required fields.
INI, initiator; BRE, TFIIB recognition element; DPE, downstream promoter element; TSS, transcription start site.
Title: Relative coverage of the ChromHMM promoter states in the GM12878 and K562 cell lines

Title: Summary of the promoter frequency profile
Title: ERRATUM: Acknowledgments Correction.
Cell-Free miR-27a, a Potential Diagnostic and Prognostic Biomarker for Gastric Cancer

Title: http://dx.doi.org/10.5808/GI.2015.13.3.70

Title: Genomics Inform 2015;13(3):70-75

Title: The funding acknowledgment in this article was partially omitted as published.
Additional acknowledgment is as follows:

This work was supported by the National Research Foundation of Korea (NRF), grants funded by the Korea government (MSIP; No.
2011-0030049 and 2013R1A1A2006621), and a KRIBB Research Initiative Grant.
Title: Editor&#39;s Introduction to This Issue

In this issue, diverse topics of genome biology and informatics, such as non-coding RNAs, biological implication of introns, transcriptome sequencing, disease-association of genetic variants, oncogenomics, genetic evolution, and analysis of promoter regions are published.
Drs.
Yong Sun Lee and Xiaoyong Bao&#39;s group (University of Texas) reviewed the biogenesis and functions of tRNA-derived RNA Fragments (tRFs).
tRFs is a newly introduced class of non-coding RNAs.
Considering that tRFs are not random degradation products but are functional non-coding RNAs through specific tRNA cleavage, their review must be helpful to understand this emerging class of non-coding RNA.
Dr. Sun Shim Choi&#39;s group (Kangwon National University) raised interesting questions about intron.
For examples, why only eukaryotes have introns or why the amount of total introns varies in different species and in different genes?
In their review, they summarized previous researches about the functional roles or benefits of introns and suggest their view.
Transcriptome sequencing is now one of the common and clinically applicable genome analyses tools.
In this issue, Dr. Sangwoo Kim&#39;s group (Yonsei University) introduced the routine workflow of transcriptome sequencing workflow together with related software, focusing particularly on transcriptome reconstruction and expression quantification.
There are three papers regarding genetic evolution in diverse species.
Dr. Kornsorn Srikulnath&#39;s group (Kasetsart University) reviewed the role of chromosome changes in crocodilian evolution and diversity.
Dr. Heebal Kim&#39;s group (Seoul National University) reported the genes related to positive selection in commercial pig breeds and also estimated the heritability in Holstein breeding.
For further details, please visit G&amp;I homepage (http://www.kogo.or.kr/webapp/kogo_publish/genomics_and_informatics/).
Title: Small Non-coding Transfer RNA-Derived RNA Fragments (tRFs): Their Biogenesis, Function and Implication in Human Diseases

tRNA-derived RNA fragments (tRFs) are an emerging class of non-coding RNAs (ncRNAs).
A growing number of reports have shown that tRFs are not random degradation products but are functional ncRNAs made of specific tRNA cleavage.
They play regulatory roles in several biological contexts such as cancer, innate immunity, stress responses, and neurological disorders.
In this review, we summarize the biogenesis and functions of tRFs.
It is now well known that mammalian genomes encode, in addition to protein-coding genes, RNA molecules that have no protein-coding capacity, and thus are referred to as non-coding RNAs (ncRNAs) [1].
Recent high throughput sequencing data suggest that there are about four times as many ncRNA transcripts as there are protein-coding RNA transcripts in human cells.
While some ncRNAs, such as ribosomal RNAs (rRNAs), transfer RNAs (tRNAs), and those well-studied ones including microRNAs (miRNAs) and small interfering RNAs (siRNAs), are recognized as functional molecules, the majority of ncRNAs and their biological roles in mammals are underappreciated, as evolution would have favored against such gratuitous and wasteful transcription [2].
However, accumulating evidence has exhibited a remarkable degree of their involvement in many biological events including cell proliferation, cellular stress responses, host immune responses and virus replication [3456].
Based on their size, ncRNAs are classified into long, medium, and small ones.
Those longer than 200 nucleotides (nt) are termed as long (or large) ncRNAs (lncRNAs) and include the majority of nuclear ncRNAs.
Those ranging 40-200 nt are medium-sized ncRNAs including many classic ncRNAs such as 5S rRNA, tRNAs, small nuclear RNAs (snRNAs), small nucleolar RNAs (snoRNAs), etc.
ncRNAs shorter than 40 nt are termed small ncRNAs (sncRNAs) that have been extensively studied in the last two decades and will be elaborated in the next section.
In this review, we will focus on a recently discovered class of sncRNAs called tRNA-derived RNA fragments (tRFs), as data from several groups including those from us suggested them to be emerging functional molecules.
The functionality of ncRNAs was initially bolstered by the discovery showing that sncRNAs are capable of mediating both transcriptional and post-transcriptional gene silencing [7].
The most extensively studied among them are miRNAs that regulate target mRNAs post-transcriptionally in animals and plants [8910].
Another class of sncRNAs is siRNAs that are closely related to miRNAs [11].
Although naturally occurring siRNAs have been reported to be present in some mammals in addition to plants and flies [1213], siRNAs are more renowned as an experimental tool in mammalian cells.
The gene silencing by synthetic double-stranded RNAs (dsRNAs) and their Dicer-cleavage products (that are siRNAs) is called RNA interference (RNAi) and has been being used as a popular gene knockdown method [3].
This technical breakthrough was well recognized and celebrated by the 2006 Nobel Prize, which was awarded to the two scientists who discovered RNAi [1415].
A number of other sncRNAs have also been identified in various organisms [1617], including piwi-interacting RNA (piRNA), repeat-associated siRNA (rasiRNA), tiny ncRNAs (tncRNA), heterochromatic small RNA (21U-RNA, hcRNA), scan RNA (scnRNA), transcription initiation RNA (tiRNA) [18], and promoter/termini-associated sRNA (PASR/TASR) [19].
Although some of these have been shown to play roles in several biological processes such as heterochromatin formation and transposon silencing, their functions are largely unknown and detailed description of them is beyond the scope of this review [10].
Mature tRNAs are characterized by their evolutionarily conserved secondary and tertiary structures, extensive post-transcriptional modifications (up to 100 nucleoside modifications have been described), extreme stability and resistance to nucleases [202122].
There are more than 600 tRNA genes in humans, which are far more than needed for at most 64 anticodons and 20 amino acids [232425], suggesting gene-specific functions of tRNAs other than its traditional function in linking a codon to the cognate amino acid during translation.
Indeed, tRFs were recently identified as a new class of sncRNAs fragmentized from tRNAs and shown to be important in regulating cell proliferation, stress-induced translational suppression, stress granule (SG) assembly, and virus replication.
Expression of some tRFs is commonly seen among different organisms, further supporting that tRFs have emerged as important players in diverse aspects of biology.
A mature tRNA has a 5&#39;-monophosphate group and 3&#39;-CCA charged with cognate amino acid.
All properly folded tRNAs, with so called a cloverleaf structure, have four base-paired stems (in 5&#39;–3&#39; direction: D arm, anticodon arm, T arm, and acceptor stem) bridging the conserved D-loop, tRNA-specific anticodon loop, variable loop, and T-loop.
Primary transcripts (pre-tRNAs) undergo several stages of maturation before becoming translation-competent tRNAs [26].
Compared to a mature tRNA, a pre-tRNA lacks a CCA tail but includes a 5&#39;-leader starting with triphosphate, a 3&#39;-trailer with an oligouridine stretch and, for intron-containing tRNA, a variably-sized intron.
The 5&#39;- and 3&#39;-end of a pre-tRNA are trimmed by the endonucleases RNase P and RNase Z respectively, and an intron is spliced if there is [2728].
The &#39;CCA&#39; acceptor stem must be added to the 3&#39;-end as this is not encoded in eukaryotes [29].
While mature, uncleaved tRNAs have a well-defined role in protein translation, multiple groups have cloned and sequenced shorter tRNA fragments [30].
These fragments (termed &#34;tRFs&#34;) identified so far can be classified into three groups; tRF-5 series whose 5&#39;-end is aligned to the 5&#39;-end of mature tRNA, tRF-3 series whose 3&#39;-end is aligned to the 3&#39;-end of mature tRNA, and tRF-1 series which is the 3&#39;-trailer sequence in a pre-tRNA and released during 3&#39;-end processing (Fig.1).
Some groups have also termed the tRF-5 or tRF-3 series as tRF halves or type I [3132] and tRF1 series as type II [32].
The tRF-5 series were the first identified class of tRFs and were usually formed by a cleavage in the D or anticodon loop of a mature tRNA.
They have been reported in mammalian cells, plants, and fission yeast [333343536].
As aforementioned, the tRF-3 series are those derived from the 3&#39;-end of a mature tRNA, usually by cleavage in the T loop [332373839].
As the 3&#39;-end of mature tRNAs is the acceptor trinucleotide CCA, the tRF-3 series also characteristically has an end CCA.
The tRF-1 series are undoubtedly the RNase Z cleavage product of a pre-tRNA, because their 5&#39;-ends begin immediately after the 3&#39;-terminus of the mature tRNA and their 3&#39;-ends are a series of U residues that are residual from RNA polymerase III transcription run-off [3324041].
Overall, the cleavage of tRFs is very much structure-dependent, suggesting they are not randomly degraded products.
Other reasons supporting that tRFs are biologically meaningful include their endonuclease-specific cleavage [343442].
The production of the tRF-5 series has been reported to be Dicer or angiogenin (ANG)-dependent in mammals [3334].
In one case, the tRF-5 series were identified to be produced after the hybridization of the 3&#39;-end of the human cellular tRNAlys with the HIV-1 primer binding site, raising the possibility that, at least in certain cases, tRFs may be produced when a tRNA base-pairs with another RNA molecule [38].
It has been reported that the tRF-3 series are generated by Dicer [3238].
In Tetrahymena, although the nuclease identity was not proven, the tRF-3 series have 5&#39;-monophosphates that are usually left after Dicer cleavage and are in physical association with Argonaute/Piwi proteins, suggesting Dicer-mediated cleavage [39].
The tRNA 3&#39;-trailer sequences are thought to be rapidly degraded in the nuclear compartment soon after being cleaved from pre-tRNAs.
Therefore, it is quite surprising that the tRF-1 series are localized mainly in the cytoplasm [341].
There are two explanations on how the tRF-1 series bypass nuclear cleavage: (1) the tRF-1 series are exported from the nucleus by an unknown mechanism upon processing, or (2) pre-tRNAs escape nuclear quality control and are processed by cytoplasmic RNase Z.
Biological evidence indeed supports the second possibility.
At least in one case of a tRF-1 molecule derived from tRNASer (termed tRF-1001 [3] and Cand45 [32]), the corresponding pre-tRNA is localized in the cytoplasm [332].
Also there are several reports supporting the cytoplasmic localization of tRNA endonucleases responsible for the tRF-1 series [343].
Are tRFs merely by-products of tRNA degradation or nuclease off-target effects?
Or do they have bona fide biological functions?
If they do have functions, how diverse are these functions?
Several lines of evidence have identified their regulatory targets, supporting that they are functional molecules [31343537].
Although the exact roles of tRFs are further to be elucidated, accumulating evidence suggests that they participate in two main types of biological processes as discussed in more detail below (Fig.2).
tRNAs are indispensable components of the translational machinery, hence one can surmise that tRNA cleavage affects protein synthesis.
However, the mode of translational regulation by tRNA cleavage is not simply a result of change in the mature tRNA pool available for protein synthesis.
Ivanov et al.showed that tRNA halves generated by ANG during stress play an active role in inhibiting protein synthesis and triggering the phospho-eIF2α-independent assembly of SGs [44].
In addition to tRNA halves, tRFs have also been reported to be involved in global translational inhibition in human cells.
In the archaeon Haloferax volcanii a 26-nt long tRF-5 originating from tRNAVal in a stress-dependent manner was shown to directly bind to the small ribosomal subunit and reduce protein synthesis by interfering with peptidyl transferase activity [45].
A similar mechanism of translation inhibition by a tRF-5 was also observed in human cells [46].
A 19-nt long tRF-5 derived from tRNAGln decreased the expression of a reporter gene that did not have a sequence complementary to that of the tRF-5, suggesting that non-specific translational repression is mediated by this tRF-5.
An interesting observation from this study was that the tRFs required a conserved &#34;GG&#34; dinucleotide for their activity in inhibiting translation.
Because the tRF-5 series were detected in the polysome fraction, interaction of the tRF-5 with the ribosome may contribute to translation [46].
In addition to above-mentioned translational regulation, it has become apparent that some tRFs mediate gene silencing similarly to miRNAs by binding to Argonaute family proteins [161747].
The first tRFs suggested to be involved in the gene silencing are some tRF-5 and -3 molecules [32].
Haussecker et al.observed that they are in association with Ago-3 and -4 and exhibit trans-gene silencing activity.
However, this activity is much lower than that of miRNAs.
On the other hand, some groups showed that the tRF-5 series have inefficient association with Argonautes [34] and do not exhibit detectable silencing effects [31].
Probably the most convincing result is from our recent studies on a tRF-5 molecule derived from mature tRNAGluCTC (tRF5-GluCTC).
Upon infection of respiratory syncytial virus (RSV), tRF5-GluCTC is induced robustly and plays a gene-silencing role on a target mRNA with a reverse complementary target sequence.
In this case, the trans-silencing mechanism used by this tRF was distinct from that of canonical miRNA/siRNA.
Mutagenesis studies demonstrated that the 3&#39;-portion of tRF5-GluCTC is critical for its gene silencing function, in contrast to miRNAs whose 5&#39;-portion is the key determinant in target recognition [5].
The expression of tRFs is altered in a number of pathological conditions and so those tRFs could be useful disease biomarkers.
The involvement of tRFs in cancer, neurodegeneration and infectious disease will be discussed in the following paragraphs (Fig.2) [548].
The role of tRFs in cancer was first reported by Lee et al3.
One of the tRF-1 series (namely tRF-1001), derived from pre-tRNASer, is highly expressed in several cancer cell lines relative to non-proliferating normal tissues, and is required for proliferation of prostate cancer cells [3].
The levels of tRF-1001 in cancer cells are directly related to the degree of cellular proliferation.
Knocking down this tRF causes a dramatic loss of cell viability and inhibition of cell proliferation with specific accumulation of cells in G2 phase and inhibition of DNA synthesis [3].
In addition, analysis of the small RNA transcriptome in prostate cancer revealed enrichment of tRFs in both non-metastatic and metastatic lymph node prostate cancer samples [3].
The molecular mechanism by which tRF-1001 affects cell physiology is not known.
There may be a differential processing of tRNA in prostate cancer because tRFs in non-metastatic samples were 18 nucleotides long whereas tRFs in metastatic tissues were 27 nucleotides long [49].
The important role of tRFs in controlling cancer is also supported by the observation showing enhanced ANG expression in almost all types of cancer [505152].
Recently, it has been shown that ANG produced tRFs that re-program translation and promote the assembly of stress granules.
As these events help cells survive under adverse conditions, it is possible that ANG-induced tRFs may contribute to ANG-mediated cancer cell proliferation [48].
tRFs have also been reported to help cancer cells to prevent apoptosome formation by binding to cytochrome c [53].
Although tRNA metabolism is essential for all human cells, a number of neurological disorders are caused by defects in tRNA metabolism and tRNA processing enzymes.
ANG mutants possessing reduced RNase activity were implicated in the pathogenesis of amyotrophic lateral sclerosis (ALS), a fatal neurodegenerative disease [54].
A subset of ALS-associated ANG mutants was also found in Parkinson&#39;s disease patients [55].
Administration of recombinant ANG to cultured neurons protects them from stress-induced apoptosis and promotes the lifespan and motor function of SOD1 (G93A) mice, a laboratory ALS model.
In neurons, excessive accumulation of some tRF-5 series members that are derived from a specific subset of tRNAs (Asp, Glu, Gly, His, Val, and Lys) triggers a sustained stress response leading to neuronal loss and links aberrant tRNA metabolism to the development of certain forms of intellectual disability [48].
tRFs derived from intron-containing tRNAs are also strongly implicated in neurodegeneration [565758].
A study on mutant mice carrying a catalytically inactive form of the RNA kinase CLP1, a component of the tRNA splicing endonuclease complex, revealed a neurological disorder characterized by progressive loss of lower spinal motor neurons.
Surprisingly, mutant mice accumulate a novel class of tRFs.
In addition, homozygous missense mutations in CLP1 (R140H) were found in patients with pontocerebellar hypoplasia, a heterogeneous group of inherited neurodegenerative disorders characterized by impaired development of various parts of the brain.
These patients were identified to suffer from severe motor-sensory defects, cortical dysgenesis and microcephaly, and exhibit alterations in tRNA splicing [59].
On the cellular level, the R140A mutation leads to the depletion of mature tRNAs, the accumulation of unspliced pre-tRNAs in patient-derived induced neurons [58] and the accumulation of linear tRNA introns [57].
Transfection of 5&#39;-unphosphorylated tRF corresponding to the 3&#39;-exon of pre-tRNATyr (the natural substrate of CLP1) into patient cells results in reduced neuron survival under oxidative stress compared to the transfection of the 5&#39;-exon, which had no effect on viability [58].
The kinase-dead CLP1 mice accumulate 5&#39;-leader-exon-tRFs that sensitize cells to oxidative stress and promote cell death in a p53-dependent fashion [56].
While the exact mechanistic details on the interplay between CLP1 activity, tRNA splicing and functions of pre-tRNA-derived tRFs are yet to be uncovered, these studies provide a basis for further investigations linking aberrant tRNA metabolism and development of neurodegeneration [48].
Viruses exploit a number of strategies to evade host immune recognition and to facilitate viral replication.
Some of these strategies involve viral-derived or endogenous cellular ncRNAs that interact with host immune responses [60].
We have demonstrated that RSV, the most common cause of bronchiolitis and pneumonia in babies, employs a novel mechanism to suppress antiviral responses via induction of some tRFs [561].
We have shown that RSV infection in airway epithelial cells leads to the activation of endonuclease ANG, subsequently inducing abundant production of tRFs, most of which belong to that tRF-5 series.
The cleavage sites by ANG are preferentially adjacent to the 5&#39;-end of the anticodon-loop, generating tRF-5 series.
These tRF-5s are localized exclusively in the cellular cytoplasm [5].
At least one of the tRFs, namely tRF5-GluCTC (tRF-5 derived from tRNAGlu), has a trans-gene silencing function.
Significantly, an antiviral target suppressed by tRF5-GluCTC downregulates RSV replication and significantly decreases the yield of infectious viral particles produced by infected cells, demonstrating a stimulatory role for tRF5-GluCTC in RSV replication [5].
Using an elegant biochemical approach, named HITS-ABt (hIgh-throughput sequencing of RNAs associated with biotinylated tRF), to capture tRF5-GluCTC-containing ribonucleoprotein complexes combined with bioinformatics analysis, we have identified a number of potential cellular mRNA targets of tRF5-GluCTC.
One of the candidate mRNAs was shown to be a direct target of tRF5-GluCTC.
Mechanistically, tRF5-GluCTC recognizes a target site in the 3&#39;-untranslated region of APOER2 mRNA and suppresses its expression by an unknown mechanism.
Further analysis identified a previously unappreciated role for APOER2 protein in antiviral defense against RSV infection [61].
Thus, we have illuminated the roles of tRFs in host–virus interactions and described a novel molecular mechanism for host response regulation via gene targeting at posttranscriptional level, proving a potential therapeutic target to control RSV replication by regulating tRFs induction.
As another example of tRFs with a positive effect on viral replication, a tRF-3 derived from tRNAPro (tRF-3019) has been suggested to function as a primer for reverse transcription of human T-cell leukemia virus-1 [62].
The induction of tRFs is virus specific, as human metapneumovirus, a negative-sense single stranded RNA virus belonging to the Paramyxoviridae family [63] and frequent cause for lower respiratory tract infections in young children, immunocompromised patients and older adults [64656667], did not induce tRFs but gradually enhanced the expression percentage of miRNAs in a time-dependent fashion [6].
The involvement of tRFs in infectious diseases suggests that tRFs may be targets for the development of new drugs to control viral replication.
The human genome encodes hundreds of tRNA genes and pseudogenes.
In the past two decades, an increasing number of reports have revealed abundant expression of functional tRFs; thus, it now appears highly plausible that cells use tRNAs as a source for functional sncRNAs to modulate biological processes beyond its translation function.
It is imperative to discriminate functional tRNA fragments from &#34;meaningless&#34; degradation products, and to biochemically elucidate the factors in their biogenesis as well as to determine their molecular functions.
Although tRFs, at least the tRF-5s induced by RSV, regulate gene expression at post-transcriptional stages in the cytoplasm, the precise mechanism is yet to be clarified.
Like miRNAs, tRFs can repress target mRNAs by inhibiting their translation or facilitating their decay.
These questions can be answered by analyzing their localization, identifying their interacting protein partners besides Argonaute proteins, and examining various biological phenomena in response to the changes in the expression of tRFs.
Bioinformatics studies with the support from biochemical assays have identified numerous novel tRNA-interacting proteins [68], implying that the biological functions of tRNAs and their fragments may be way beyond our expectations.
Further combination of computational and biochemical efforts will significantly advance our understanding of functional tRFs and expand our knowledge regarding the tRNA world.
The availability of genome sequencing data from patients with diverse conditions has already led to the identification of several mutations in tRNA genes and in genes involved in tRNA biogenesis [69].
The task now is to ascertain whether these mutations indeed affect the repertoire of tRFs and consequently the molecular pathology of the diseases.
Studies on tRFs are still at a very early stage.
Additional functions for tRFs are likely to be discovered continuously.
Improved understanding of the functions of tRFs will provide valuable insights into human physiology and pathophysiology.
Recognition of tRFs as an important class of functional molecules has not happened until recently.
The apparent presence of abundant tRFs in early studies with next generation sequencing data was often ignored.
However, the combined biochemical evidence from many years of tRNA biology studies has brought the field to a previously unexpected conceptual consensus: specific tRFs are not simply random degradation products but functional molecules based on the following observations: (1) in many cases, tRFs do not derive from abundant cellular tRNAs, and the numbers of tRFs do not correlate with the parental tRNA gene copy numbers; (2) their fragmentation patterns are tRNA structure-dependent; (3) the fragmentation patterns is determined by the developmental stage or cell conditions; and (4) it is reported that some tRFs are bound to Argonaute/Piwi proteins, well-known components of the RNA-induced silencing complex [213170717273].
The expression of tRFs does not usually affect mature tRNA pools.
Instead, it is involved in normal biological processes beyond translation and in diseases; thus, studies of tRFs have provided new insights into tRNA biology.
tRNA processing and classification of tRNA-derived RNA fragments (tRFs).
Biological roles of tRNA-derived RNA fragments (tRFs).
Title: Editor&#39;s Introduction to This Issue

In the post-genome era, understanding protein biomarkers is becoming more important.
Indeed, advances in mass spectrometry-based proteomics technologies have enabled the generation of global proteome data from tissue or body fluid samples collected from a broad spectrum of human diseases.
In this issue, there are two inspiring review articles on discovering protein biomarkers.
Dr. Daehee Hwang&#39;s group (DGIST, Korea) comprehensively reviews network-based protein biomarker mining platforms.
In this review, they summarize analytical platforms for network-based protein biomarker discovery and key components in the platforms.
Dr. Ji Eun Lee (KIST, Korea) introduces neuropeptidomics.
Neuropeptides produced from prohormones by selective action of endopeptidases are vital signaling molecules, playing a critical role in a variety of physiological processes, such as addiction, depression, pain, and circadian rhythms.
In this paper, they provide an overview of neuropeptides and mass spectrometry-based neuropeptidomic strategies for the identification and quantitation of neuropeptides.
Regarding neurobiology, Dr.
In Young Choi&#39;s group (The Catholic University of Korea) reviews protocols for web authorization.
Developing a database is important to understand the mechanism of neurobiological disorders, including behavioral addiction.
In particular, genetic databases require a high level of security and should be designed based on medical information standards.
In this respect, they propose the OAuth standard protocol for database access authorization.
Dr. Sangsoo Kim&#39;s group (Soongsil University, Korea) has developed a detection method for multicopy single-stranded DNA (msDNA), which is extrachromosomal DNA found in many bacterial species.
They applied it in scanning NCBI RefSeq bacterial genome sequences.
Their work can be a useful tool in studying the distribution, evolution, and physiological role of msDNA.
Dr. Seon-Young Kim&#39;s group (KRIBB, Korea) suggests that public datasets should not be expected to be error-free and, whenever possible, that we should check the consistency of the data.
For further details, please visit the G&amp;I homepage (http://www.kogo.or.kr/webapp/kogo_publish/genomics_and_informatics/).
Title: Neuropeptidomics: Mass Spectrometry-Based Identification and Quantitation of Neuropeptides

Neuropeptides produced from prohormones by selective action of endopeptidases are vital signaling molecules, playing a critical role in a variety of physiological processes, such as addiction, depression, pain, and circadian rhythms.
Neuropeptides bind to post-synaptic receptors and elicit cellular effects like classical neurotransmitters.
While each neuropeptide could have its own biological function, mass spectrometry (MS) allows for the identification of the precise molecular forms of each peptide without a priori knowledge of the peptide identity and for the quantitation of neuropeptides in different conditions of the samples.
MS-based neuropeptidomics approaches have been applied to various animal models and conditions to characterize and quantify novel neuropeptides, as well as known neuropeptides, advancing our understanding of nervous system function over the past decade.
Here, we will present an overview of neuropeptides and MS-based neuropeptidomic strategies for the identification and quantitation of neuropeptides.
Neuropeptides, which signal between neurons, are known to be involved in a variety of physiological processes, including addiction, depression, hunger, pain, fear, anxiety, and circadian rhythms [12].
Neuropeptides are 3–100 amino acid residues long and up to 50 times larger than classic neurotransmitters [3].
The term &#34;neuropeptides&#34; was first introduced by De Wied [4] in 1971 to describe an endogenous peptide synthesized in nerve cells.
While peptide hormones are also considered important intercellular signaling molecules, there are differences between neuropeptides and peptide hormones.
That is, neuropeptides are secreted from neuronal cells (primarily neurons but also glia for some peptides) and signal to neighboring cells (primarily neurons), whereas peptide hormones are secreted from neuroendocrine cells and act on distant tissues by traveling through the blood [25].
Approximately 100 different neuropeptides are currently known to function in cell-to-cell signaling (http://www.neuropeptides.nl).
Neuropeptides bind to post-synaptic receptors and elicit cellular responses like classical neurotransmitters.
Neuropeptides are produced from precursor proteins (or prohormones) by a series of enzymatic processing steps.
The neuropeptide precursors generally do not have functions on their own, requiring processing to generate the active forms.
The precursors are synthesized on ribosomes at the endoplasmic reticulum and processed through the Golgi [5].
When the NH2-terminal signal peptide of the neuropeptide precursors is cleaved by signal peptidase, the precursors are routed to the Golgi apparatus and packaged into dense-core secretory vesicles together with processing proteases, termed convertases.
Most neuropeptide precursors are first processed by endopeptidases, including prohormone convertase 1 (also known as prohormone convertase 3) and prohormone convertase 2 and, to a lesser extent, prohormone/proprotein convertase 5A (also known as prohormone/proprotein convertase 6A) [6].
The endopeptidases typically cleave at pairs of basic amino acids, usually Lys-Arg (KR) or Arg-Arg (RR) sites, but on occasion, the basic amino acids are separated by 2, 4, or 6 other amino acids (Fig.1).
The next stage of processing is the removal of C-terminal basic residues in the mature secretory vesicle.
Carboxypeptidase E is the exopeptidase primarily responsible for removing the C-terminal basic residues, and carboxypeptidase D has also been shown to act in this role.
After removal of the C-terminal basic residues, other enzymes perform additional post-translational modifications (PTMs), such as amidation, acetylation, phosphorylation, sulfation, and glycosylation [6].
For example, peptides with a C-terminal glycine are typically converted to the amide by peptidyl-α-amidating monooxygnenase [7], and tyrosine sulfation is known to be mediated by tyrosylprotein sulfotransferase [8].
These final bioactive forms of the peptides are stored within secretory vesicles that are accumulated within the cells.
When the cells are stimulated, the secretory vesicles are known to fuse with the cellular membrane to release the neuropeptides into the extracellular environment [2].
The secreted neuropeptides can interact with receptors containing specific binding sites.
The receptors are often G-protein-coupled receptors.
The interaction of the neuropeptides with the receptors causes a conformational change in the receptor, resulting in the production of a cellular response through a variety of mechanisms, depending on the type of receptor.
While a single neuropeptide precursor often produces multiple neuropeptides, the proteolytic processing of the precursor protein occurs in a tissue-specific and even region-specific manner.
A classic example is the processing of the precursor proopiomelanocortin in the pituitary.
In the anterior lobe of the pituitary, the precursor is converted into adrenocorticotropic hormone (ACTH), which binds to several subtypes of melanocortin receptors, such as MC2R, located in the adrenal gland; this receptor controls the production of glucocorticoids (cortisol) [910].
In the intermediate lobe of the pituitary and also in the brain, ACTH is further processed into alpha-melanocyte-stimulating hormone, which binds to melanocortin receptors with completely different affinities [11].
The PTMs can also regulate the bioactivity and stability of the neuropeptides and therefore affect the binding affinities of the neuropeptides for the receptors.
For example, acetylation of the N-terminus of β-endorphin removes the ability of the peptide to stimulate opioid receptors and to produce analgesia [6].
Conversely, octanoylation of the peptide ghrelin on a serine residue is essential for the binding of the peptide to its receptor [12].
Although the various aspects of peptide synthesis, including PTMs, lead to an increase in neuropeptide complexity, unveiling new peptides and unreported peptide properties is critical to advancing our understanding of nervous system function.
While neuropeptides are involved in a wide range of physiological processes, including hunger, food intake, body weight regulation, and circadian rhythms, it has been shown that dysregulation of neuropeptides often results in a variety of neurological disorders [131415].
Epilepsy is a common neurological disorder characterized by recurrent seizures.
Neuropeptides, such as arginine-vasopressin, corticotropin-releasing factor (CRF), enkephalin, β-endorphin, pituitary adenylate cyclase-activating polypeptide (PACAP), and tachykinins, were shown to have proconvulsive effects, whereas other neuropeptides, like ACTH, angiotensin, cholecystokinin, somatostatin, and thyrotropin-releasing hormone, were able to suppress seizures in the brain [16].
In addition, neuropeptides, such as dynorphin and substance P, were shown to be involved in the pathophysiology of Parkinson&#39;s disease, which is a neurodegenerative disease with motor and non-motor symptoms [17].
Progression to addiction is also known to be influenced by several peptidergic neuromodulators.
A few orexigenic neuropeptides, including galanin, enkephalin, and orexin/hypocretin, were shown to have a positive feedback relationship to alcohol, while other hypothalamic neuropeptides, such as dynorphin, CRF, and melanocortins, showed a negative feedback relationship to alcohol intake.
Dysregulation of these multiple neuropeptides is assumed to result in alcohol addiction [18].
There was also a report that neuropeptides, such as substance P, calcitonin gene-related peptides, vasoactive intestinal peptide/PACAP, and neuropeptide Y (NPY), are associated with regulation of cutaneous immune responses and tissue maintenance and repair.
Due to these diverse functions of neuropeptides, neuropeptides and neuropeptide receptors have been drug targets for the treatment of neurological disorders [319].
Title: Traditional methods of neuropeptide analysis

Traditionally, the analysis of neuropeptides was performed by Edman sequencing, in which the N-terminal amino acid is sequentially removed.
The method was developed by Edman in 1950 and is used for neuropeptide sequencing in many different species—for example, NPY in the porcine brain [20] and gonadotropin-releasing hormone in the dogfish brain [21].
However, analysis by this method is slow and does not allow for the sequencing of neuropeptides containing N-terminal PTMs [22].
Immunological techniques, such as radioimmunoassay (RIA) and immunohistochemistry (IHC), have been used for measuring relative neuropeptide levels and spatial localization.
RIA has been a popular tool to quantify neuropeptides in biological samples, such as NPY [23] and galanin in the rat brain [24].
Although RIA is a relatively sensitive technique capable of absolute quantification of peptide levels, it is not always specific for a single-peptide form because of its antibody-based detection method.
In addition, unless specific antibodies are available for the different isoforms, RIAs do not distinguish between these modifications.
IHC has provided essential information regarding the localization of neuropeptides within the complexity of the brain structure, improving our understanding of the distribution of peptides.
However, IHC does not report the actual size or full identity of the peptide, including PTMs.
In addition, these antibody-based methods require a priori knowledge of a potential neuropeptide in order to generate antibodies specific to the neuropeptides and only detect peptide sequences with known structures.
Title: Mass Spectrometry-Based Peptidomics

In contrast to immuno-based methods, mass spectrometry (MS) enables one to detect and identify the precise forms of neuropeptides without a priori knowledge of the peptide&#39;s identity, resulting in the identification of previously unknown neuropeptides.
MS has been used to identify and characterize hundreds of endogenous peptides in various animals [2526272829].
The high-throughput discovery of neuropeptides based on MS has made up the field of peptidomics (Fig.2) [630313233].
While typical proteomic analysis involves the digestion of proteins using proteases, such as trypsin, to generate peptides that can be readily sequenced by tandem mass spectrometry (MS/MS), peptidomics focuses on the analysis of the native peptide forms, including PTMs, without using digestive enzymes.
Therefore, it is important to identify and characterize as many neuropeptides as possible, because each peptide can have its own biological function.
While MS-based peptidomics can be utilized to characterize a large number of neuropeptides simultaneously, it can be also applied to the quantification of individual neuropeptides.
Title: Sample Preparation for Peptidomics Experiment

Although MS allows the identification of a large number of neuropeptides in a high-throughput manner, sample preparation prior to MS is a critical step that affects peptide identification coverage in peptidomics experiments.
When the animal is sacrificed and the brain tissue of interest is obtained, proteases rapidly degrade larger proteins into smaller peptides that fall into the mass range of neuropeptides.
The presence of degraded products derived from highly abundant proteins often complicates the MS analysis and interferes with the identification of less abundant neuropeptides.
Different techniques, including focused microwave irradiation before decapitation, microwave irradiation after decapitation, and boiling the tissue immediately after decapitation, have been used in order to inactivate the proteases responsible for the high background of protein degradation fragments [323435363738].
In addition to the procedures of protease deactivation, peptide extraction and subsequent treatment are other steps that affect the number of identified neuropeptides.
The use of different solvents and procedures can affect the peptide profiles detectable in brain tissue.
Che et al.
[31] compared different extraction conditions for the recovery of neuropeptides from mouse hypothalamus.
They found that sonication and heating in water (70℃ for 20 min), followed by cold acid and centrifugation, enabled the efficient extraction of many neuropeptides without the formation of the protein degradation fragments seen with hot acid extractions.
Bora et al.
[39] and Lee et al.
[37] also showed that multiple stages of peptide extraction, including boiling water, acidified acetone, and acetic acid, were able to maximize the number of identified neuropeptides from their respective analyses of supraoptic nucleus and suprachiasmatic nucleus (SCN) samples in the rat brain.
Title: MS Analysis

After sample processing, the extracted peptides are subjected to MS analysis.
A mass spectrometer consists of an ionization source, a mass analyzer separating the ions based on their mass-to-charge ratio (m/z), and a detector recording the number of ions at each m/z value.
Two commonly used types of ionization are electrospray ionization (ESI) and matrix-assisted laser desorption/ionization (MALDI).
While each approach has its own advantages, MALDI is more tolerant to the presence of salts.
Thus, MALDI is often applied to direct tissue profiling or molecular ion imaging from tissue sections.
Direct tissue sample analysis by MALDI-based MS is usually performed by a simple sequence of steps, which include placing the tissue of interest on the MALDI plate, applying a droplet of matrix, and irradiating the co-crystallized tissue to cause desorption/ionization of the peptides [30].
MALDI imaging provides valuable information pertaining to the spatial localization of neuropeptides [40].
Although MALDI-MS can be interfaced with separation strategies, such as liquid chromatography (LC) and capillary electrophoresis, ESI-MS can be coupled more easily with the separation methods, because the ions are produced as an aerosol from the liquid phase with a high voltage in ESI.
The introduction of a nano-electrospray, based on capillary LC, coupled to MS, significantly increases the sensitivity in LC-ESI-MS, and furthermore, the preferred formation of multiple-charge states in ESI allows for more comprehensive MS/MS fragmentation data.
Title: MS Data Analysis for Neuropeptide Identification

While neuropeptides can be identified by comparing experimental peptide masses obtained from MS data against a list of known neuropeptide masses, they are typically identified with both MS and MS/MS fragmentation data.
While collision-induced dissociation is widely used for peptide identification, other fragmentation methods, such as higher energy collision dissociation and electron transfer dissociation, have recently improved the identification capability of peptides and PTM localization [414243].
The automated identification of neuropeptides based on their MS and MS/MS data is pursued by using proteomic-based search engines, such as Mascot, SEQUEST, X!Tandem, Peaks Studio, and ProSightPC [3744454647].
Since peptidomics searches typically employ criteria, such as no enzymatic cleavage and various common modifications (e.g., N-terminal acetylation and C-terminal amidation), they often require a larger search space and longer search times for peptide identification.
Several neuropeptide prohormone databases, including SwePep, have been constructed to facilitate neuropeptide identification, and cleavage prediction programs, such as NeuroPred, were developed to predict cleavage sites in prohormones and to provide the masses of the resulting peptides [4849].
In addition to the database search strategies, neuropeptides can be also identified by de novo sequencing, especially when the species of interest does not have a sequenced genome.
While peptide spectra are compared with theoretical peptides in the database search strategy, de novo sequencing, which is the direct assignment of the amino acid sequence from the MS/MS spectrum, can be pursued when the database is not available.
Although de novo sequencing can be performed manually, the process is labor-intensive and time-consuming.
There are several software packages that can perform de novo sequencing directly on MS/MS data, including Lutefisk (http://www.hairyfatguy.com/lutefisk/), Mascot Distiller (Matrix Science), and PEAKS (Bioinformatics Solutions, http://www.bioinformaticssolutions.com/) [30].
The peptide sequences obtained from de novo sequencing are often compared with the nonredundant database of the National Center for Biotechnology Information to establish the peptide identities with the Basic Local Alignment Search Tool (http://www.ncbi.nlm.nih.gov/BLAST).
The BLAST search compares a partial neuropeptide sequence against the database of a closely related species.
Title: Quantitative Neuropeptidomics

MS-based peptidomic techniques allow for measuring relative levels of peptides in different conditions of the samples, as well as peptide identification.
Quantitation of peptides based on MS is generally achieved either by stable isotope labeling or label-free approaches (Fig.3) [5051].
In stable isotope labeling, the relative levels of peptides in two different samples are typically examined by labeling the peptides in one sample with a light stable isotope and those in the other sample with a heavy stable isotope; the two samples are then combined and analyzed together.
The relative levels of the peptides are calculated by the difference in the MS peaks (peak intensity or peak area) of the two samples.
While the stable isotopes can be introduced to the peptides either metabolically or through chemical reactions, isotope labeling through chemical reactions enables one to obtain quantitative information from biological samples, such as brain tissues, for which metabolic labeling is not available.
A number of different isotopic tags, including acetic anhydride, succinic anhydride, and trimethylammonium butyrate (TMAB), have been used for quantitative analyses of endogenous peptides in various samples and conditions [63452].
For example, the TMAB tag was used to evaluate the relative changes of endogenous peptides in response to food intake or drug treatment [535455] and to assess the role of protein convertase 1/3 and 2 in the processing of the peptides in mouse models [5657].
In label-free methods, each sample is prepared separately and then subjected directly to individual LC-MS or LC-MS/MS runs, followed by comparisons of either the mass spectral peak intensities of the detected peptides or the total number of MS/MS spectra identified for a peptide, called spectral counting [51].
Since there is no limit to the number of samples that can be compared, label-free quantitation allows for the comparison of multiple samples or conditions.
The approach has been used to measure the changes in the levels of neuropeptides present during the daytime and nighttime in the rat SCN [5859] and to examine the expression changes of endogenous peptides involved in the embryogenesis of Japanese quail brains [60] and the nucleus accumbens of morphine-dependent rats [61].
In addition, quantitative peptidomic analyses based on peak intensities has enabled us to discover endogenous peptides associated with repeated exposure to amphetamine and with individual variations in sensitivity to the behavioral effects of cocaine in rat models [6263].
Neuropeptides are important signaling molecules that are involved in many kinds of physiological processes, including addiction, depression, mood regulation, and circadian rhythms.
Because of their biological significance, neuropeptides have become key targets in drug discovery.
While immunological techniques, such as RIA and IHC, require a priori knowledge of neuropeptides and only detect peptide sequences with known structures, MS enables one to detect and identify the precise forms of neuropeptides without a priori knowledge of peptide identity, resulting in the identification of previously unknown neuropeptides.
In addition, MS-based quantitative analyses of neuropeptides based on either stable isotope labeling or label-free methods provide information on the relative levels of neuropeptides in different physiological conditions, which can be helpful in understanding the function of the neuropeptides.
Continuous efforts to implement current MS-based peptidomics techniques for neuropeptide identification and quantitation are expected to provide more profound insights into the neurochemical communication of neurons in various physiological processes.
Bioactive neuropeptide synthesis.
Adopted from Fricker et al.
Mass Spectrom Rev 2006;25:327-344, with permission of John Wiley and Sons [6].
A hypothetical precursor is shown to contain typical processing sites of Lys-Arg (KR), Arg-Arg (RR), or R-Xn-R, where n = 2, 4, or 6.
First, a signal peptidase in the endoplasmic reticulum removes the N-terminal signal peptide, and the precursor is routed to the Golgi apparatus and packaged into dense-core secretory vesicles.
The prohormone convertase 1, 2, or 5A cleaves the precursor to produce intermediates that contain C-terminal K and/or R residues.
Those basic residues are removed by carboxypeptidase E (or D).
After removal of the basic residues, peptides with a C-terminal glycine are typically converted to the amide by peptidyl-glycine α-amidating monooxygenase.
In addition, a number of other post-translational modifications (PTMs), including N-terminal acetylation, phosphorylation, and pyroglutamylation, have been reported.
Overall workflow of mass spectrometry (MS)-based peptidomics.
Adopted from Schrader et al.
EuPA Open Proteom 2014;3:171-182, according to Creative Commons License [33].
Peptidomics focuses on analyzing endogenous peptides that are present in biological samples, including brain tissue.
In contrast to bottom-up proteomics studies that involve proteolytic digestion, it is important to minimize the activity of proteases that may be present in the samples via heat or microwave irradiation prior to peptide extraction in the peptidomics experiments.
While the extracted peptides can be analyzed directly by matrix-assisted laser desorption/ionization (MALDI)-MS, they are often separated by reversed-phase chromatography or capillary electrophoresis in order to reduce sample complexity.
Then, the peptides are analyzed by MS in order to obtain amino acid sequence information of the peptides without a priori knowledge.
RP-HPLC, reverse phase-high performance liquid chromatography; LC-ESI-MS, liquid chromatography electrospray ionization mass spectrometry.
Workflow of mass spectrometry (MS)-based quantitation of endogenous peptides.
Romanova et al.
Curr Opin Chem Biol 2013;17:801-808, with permission of Elsevier [51].
Quantitation of endogenous peptides based on MS is generally achieved either by stable isotope labeling (A) or label-free approaches (B).
In stable isotope labeling, the endogenous peptides in two different samples are labeled with either a light stable isotope or a heavy stable isotope, and the two samples are then combined and analyzed together.
The relative levels of the peptides are calculated by the difference in the MS peaks (peak intensity or peak area) of the two samples.
In label-free quantitation, each sample is prepared separately and then subjected to individual liquid chromatography (LC)-MS or LC–tandem mass spectrometry (MS/MS) runs, followed by comparisons of either the mass spectral peak intensities of the detected peptides or the total number of MS/MS spectra identified for a peptide, called spectral counting.
Title: Network-Based Protein Biomarker Discovery Platforms

The advances in mass spectrometry-based proteomics technologies have enabled the generation of global proteome data from tissue or body fluid samples collected from a broad spectrum of human diseases.
Comparative proteomic analysis of global proteome data identifies and prioritizes the proteins showing altered abundances, called differentially expressed proteins (DEPs), in disease samples, compared to control samples.
Protein biomarker candidates that can serve as indicators of disease states are then selected as key molecules among these proteins.
Recently, it has been addressed that cellular pathways can provide better indications of disease states than individual molecules and also network analysis of the DEPs enables effective identification of cellular pathways altered in disease conditions and key molecules representing the altered cellular pathways.
Accordingly, a number of network-based approaches to identify disease-related pathways and representative molecules of such pathways have been developed.
In this review, we summarize analytical platforms for network-based protein biomarker discovery and key components in the platforms.
For a last decade, mass spectrometry (MS)-based proteomic technologies have emerged as a core technology to measure protein expression and the sites with post-translational modifications (PTMs) on large scales.
However, MS-based proteome analysis has been possible for only a limited number of proteins and also limited in accurately detecting low abundant proteins and the peptides with low abundant PTMs [1234].
Recent advances in high-resolution peptide separation, comprehensive fractionation, and high performance MS considerably improved the proteome size and depth (increased numbers of proteins and PTM sites measured) and also the accuracy in quantitative information of proteomic data (abundances of proteins and PTM sites) [5].
Moreover, MS-based proteome analysis required large amounts of samples to measure reliably proteins and PTM sites.
To resolve this problem, the methods for serial enrichments of different PTMs from the same sample have been developed to significantly reduce the sample amount required [6].
These advanced MS-based proteomic technologies have facilitated the generation of proteome and PTM profiles in tissue and body fluid (plasma/serum, urine, ascites, cerebrospinal fluid, synovial fluid, saliva, and tear) samples collected from the patients with a broad spectrum of human diseases.
Comparative proteomic analysis of samples from the patients and healthy control subjects is commonly applied to identify protein biomarker candidates [78910].
In this analysis, proteome profiles are first obtained for tissue or body fluid samples collected from the patients with a target disease and also healthy subjects, and the proteins showing altered abundances, called differentially expressed proteins (DEPs), in disease samples, compared to control samples are then identified (discovery phase) [10].
In the protein biomarker discovery platforms, about hundred DEPs are selected as an initial set of biomarker candidates, and their altered expression are then verified in a small cohort of the patients, which is independent to the patients used in the discovery phase (verification phase) [5].
However, the number of DEPs is often larger than 100, and how to select the initial set of biomarker candidates is not straightforward, although fold-changes and/or associations of the DEPs with pathophysiological processes in the target disease can be used as the criteria for selection of the initial set of biomarker candidates.
Next, for the initial candidates whose disease-related alterations in their abundances were confirmed in the small cohort during the verification phase, their validity as biomarkers is tested in a large cohort of the patients (validation phase) [5].
Finally, the biomarker candidates whose altered expression was confirmed in the large cohort are selected as the final set of biomarkers.
Recently, a number of studies have reported that cellular pathways can serve as better indicators of disease states than individual molecules [11121314151617].
Pathway enrichment analysis provides a list of cellular pathways enriched by the DEPs, and a set of cellular pathways related to pathophysiological processes can be then selected as altered cellular pathways in the target disease.
Next, we can focus on a subset of DEPs that are involved in the selected cellular pathways.
Furthermore, network analysis for this subset of DEPs enables effective identification of key molecules that represent the selected cellular pathways as protein biomarker candidates.
Using this approach, an initial set of biomarker candidates can be selected more effectively than the conventional criteria, such as fold-changes and associations of the DEPs with the target disease.
Thus, several network-based approaches to identify disease-related cellular pathways and representative proteins of such pathways have been developed.
In this review, we summarize bioinformatics methods for network-based protein biomarker discovery and also key components in these network-based methods.
Title: Peptide and protein identification

After obtaining proteomic data from tissue or body fluid samples using liquid chromatography–tandem mass spectrometry (LC-MS/MS) analysis, the tandem mass spectrometry (MS/MS) spectra are first searched against a protein sequence database (e.g., SWISS-Prot or UniProt) to identify the peptide sequences for individual MS/MS spectra (peptide/protein identification) (Fig.1).
The detected proteome size determines the depth to understand disease-related cellular networks based on the proteome data.
Thus, a sufficient size of the detected proteome is a prerequisite in effective discovery of protein biomarkers based on disease-related networks.
To ensure an adequate proteome size, accurate identification of the peptides for MS/MS spectra is important through database search using the engine, such as Mascot [18], SEQUEST [19], MS-GF+ [20], or Paragon (Table 1) [21].
During the database search, a score is assigned by the search engine to each peptide that can be generated from the sequence database, which reflects the degree of the agreement of the measured MS/MS spectrum to the theoretical spectrum of the peptide.
For example, SEQUEST assigns the scores of X-corr, deltaCorr, SPrank, and SP value to the peptide, whereas MS-GF+ assigns –log10(E-value) to the peptide.
For each measured MS/MS spectrum, the peptide with the largest score value is then selected, which defines a peptide-spectrum match (PSM).
Next, the statistical significance of the PSMs resulted from the database search is evaluated using a statistical method, such as PeptideProphet [22] in the Trans-Proteomic Pipeline [23] or the target-decoy method (Table 1) [24].
For example, for each PSM, PeptideProphet combines the scores (X-corr, deltaCorr, SPrank, and SP value) to an F score, and a mixture of Gaussian and Gamma distributions is fitted to the distribution of F scores for all PSMs.
Using the mixture distribution, the probability of a PSM being true (PeptideProphet p-value) is estimated [25].
Finally, the PSMs with PeptideProphet p-value &gt; 0.95 are selected as the correct PSMs.
In the target-decoy method, a reverse sequence database was generated by inversing the reference protein sequence in the database and then included in the database prior to the database search.
Using the score values for the PSMs in which the peptide sequence was obtained from the reverse sequence database, false discovery rates (FDRs) for all PSMs are estimated [24].
Finally, the PSMs with FDR &lt; 0.01 are selected as the correct PSMs.
Recent studies have shown that the best coverage of the detected proteome can be obtained by combining the outputs from multiple search engines.
For example, Compid enables the integration of Paragon and Mascot (Table 1) [24] by assigning the peptides with higher scores from the two search engines to the MS/MS spectra, which can leads to unreliable false positive rates.
Also, MSblender integrates the search scores from the search engines into a probability score for every possible PSM and then estimates FDRs for the PSMs in a reliable manner [26].
This method identifies more PSMs than any single search engine at the same FDR.
After identifying all the PSMs from the database search, a list of the detected proteins are further identified.
Using the SEQUEST search outputs, the probability that a protein is correctly identified (ProteinProphet p-value) is calculated by statistically combining the PeptideProphet p-values of the peptides derived from the protein, and the proteins with PeptideProphet p-value &gt; 0.99 are selected as the proteins correctly identified [22].
In the target decoy method, the proteins with more than two non-redundant peptides with FDR &lt; 0.01 can be considered as the reliable proteins [27].
Title: Protein quantitation

Several LC-MS/MS approaches provide quantitative information of the peptides identified from the database search (protein quantification) (Fig.1).
First, prior to LC-MS/MS analysis, the peptides can be labeled using the isotopic agents, such as isotope-coded affinity tag (iCAT) [28], isobaric tag for relative and absolute quantitation (iTRAQ) [29], or tandem mass tags (TMT) [30].
For the iCAT data, the abundance of an identified peptide is estimated as the area under the elusion curve of the peptide.
On the other hand, for the iTRAQ or TMT data, the peptide abundance is estimated as the intensities of the reporter ions in the MS/MS data [31].
Second, when no isotope labeling was done, called label-free LC-MS/MS analysis, the peptide abundance is estimated as the area under the elusion curve of the peptide as done for the iCAT data.
Next, the relative abundance of an identified peptide is calculated as the ratio of the peptide abundances between patient and control samples.
From the iCAT data, the ratios of the heavy and light peptides are calculated as the relative peptide abundance.
To estimate the relative abundances of a protein, a list of the peptides derived from the proteins are identified and the relative abundances of these peptides are then combined using a method for protein quantitation, such as the quantification tool in MaxQuant software [32] or a linear programming method [33].
Title: Identification of differentially expressed proteins

The biomarkers should reflect the alteration of disease-related pathophysiological processes in the patient samples, compared to controls.
Thus, the DEPs between control and patient samples are identified as the biomarker candidates (identification of DEPs) (Fig.1).
A number of statistical methods have been developed for identification of the DEPs, such as t-test [34] or integrative statistical methods [3536], with the multiple comparison correction.
These methods compare the relative protein abundances between control and patient samples and then estimate the significance (p-value or FDR) of the difference in the relative protein abundances between control and patient samples methods.
The proteins with p-value or FDR &lt; 0.01 or 0.05 are then selected as the DEPs.
Furthermore, additional constraints are used to select more reliable DEPs.
For example, for each DEP, the number of the patients showing the fold-changes larger than a cutoff (1.5- or 2-fold) is counted, and a subset of DEPs with the number of patients larger than a certain percentage (50% of the patients) can be selected as the reliable DEPs [37].
Using this criterion, we can focus on the DEPs that are likely to show their alterations in the abundance in at least more than half of the patients when they were used as biomarker candidates for newly collected patient samples.
Title: Functional enrichment analysis

The alterations in protein abundances under disease conditions reflect simple clinical symptoms that are commonly observed in many other diseases, such as inflammation or immune responses, or the alterations of the pathophysiological processes specific to the target disease, such as aggregation of toxic proteins in neurodegenerative diseases.
Thus, the DEPs that are involved in the disease-related pathophysiological processes can represent more effectively the alterations specific to the target disease (Fig.2).
To effectively understand the pathophysiological processes altered under disease conditions, functional enrichment analysis is often performed for the DEPs using diverse enrichment tools.
For example, the enrichment analysis of gene ontology biological processes (GOBPs) or Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways can be applied to the DEPs using DAVID [38] and PANTHER [39], and commercial tools, such as MetaCore [40] and Ingenuity Pathway Analysis (IPA, QIAGEN Redwood City, http://www.qiagen.com/ingenuity) (Table 1).
A list of GOBPs or KEGG pathways with enrichment p-value &lt; 0.05 are selected as the pathophysiological processes or cellular pathways altered under disease conditions.
Of these selected GOBPs and KEGG pathways, a subset of the GOBPs related to the target disease is further selected based on the knowledge of the target disease.
Finally, a subset of the DEPs that are involved in the selected GOBPs or KEGG pathways related to the target disease can be selected as an initial set of biomarker candidates, considering that these DEPs are likely to be specifically associated with the target disease.
No enrichment analysis mentioned above consider quantitatively the differences in fold-changes of the DEPs, but consider equivalently the DEPs that were selected using a fixed cutoff (p-value and/or fold-changes).
To remedy this problem, several functional class scoring methods have been developed, such as gene set enrichment analysis (GSEA) (Table 1, Fig.2) [41].
In the GSEA, all D-changes, and the enrichment of functional modules defined in the database, such as the molecular signatures database (MSigDB) (Table 1), by the top or bottom of the ranked protein list is statistically evaluated.
Several alternative module-level statistics to the conventional GSEA have been also used, including Kolmogorov-Smirnov statistic and the maxmean statistic [42].
Moreover, as an alternative enrichment analysis, several pathway topology approaches have been developed, such as signaling pathway impact analysis (SPIA) [43] and network perturbation amplitude (NPA) (Table 1, Fig.2) [26].
These approaches consider whether the proteins involved in functional modules defined by GOBP or GSEA database (MSigDB) interact with each other in cellular networks.
In the SPIA, a conventional overrepresentation measure, as in GOBP enrichment analysis and a topology measure of the pathway are combined to identify functional modules in which the DEPs are significantly connected.
In the NPA, the upper and lower tiers are defined as cellular pathways (e.g., mitogen-activated protein kinase pathway activation) and target genes/proteins regulated by the pathways in the upper tier, respectively, and positive and negative causal relationships are defined by the links between the upper and lower tiers.
Then, NPA evaluates the causal relationships by quantitatively summarizing whether the expression changes of the downstream nodes are consistent with pathway activation, inactivation, or no change.
These analyses provide a list of functional modules significantly altered under disease conditions.
Similar to the case of the GOBP enrichment analysis, a subset of function modules related to the target disease can be identified, and a subset of DEPs involved in the identified functional modules can be selected as an initial set of biomarker candidates.
Title: Network modeling and analysis

Functional molecules or cellular networks altered under disease conditions interact closely with each other to form a disease-perturbed cellular network.
The network model enables identification of hub-like molecules that can serve as core indicators of the activities of cellular processes described in the network model.
Thus, it is important to understand the disease-perturbed cellular network to identify the core indicators as the biomarker candidates.
The core molecules and their associated cellular pathways have been further suggested as therapeutic targets or the biomarker candidates that can be used to evaluate the efficacy of the treatments (e.g., drug efficacy) for the target disease.
Hence, the network-based approaches have been employed for protein biomarker discovery.
The network-based approaches first reconstruct a disease-perturbed cellular network model describing the interactions of the DEPs or a subset of the DEPs involved in the selected GOBPs related to the target disease using protein-protein (PPIs), protein-DNA (PDIs), and/or protein-metabolite interactions (PMIs) in the following interactome databases (Table 1, Fig.2): (1) PPI databases: human protein reference database (HPRD) [44], biological general repository for interaction datasets (BioGRID) [45], biomolecular interaction network database (BIND) [46], search tool for recurring instances of neighbouring genes (STRING) [47], and Molecular INTeraction database (MINT) [48]; (2) PDI databases: EdgeExpressDB (FANTOM4-EEDB) [49], transcriptional regulatory element database (TRED) [50], MSigDB [41], MultiNet [51], and MetaCore [40]; and (3) PMI database: KEGG pathway database [52].
Next, the network model are analyzed to identify network modules or clusters each of which includes a set of the nodes densely connected in the network (Fig.2).
For network clustering or modularization, a number of methods, such as jActiveModules [53], ResponseNet [54], NetWalker [55], and clusterMaker [56], have been developed (Table 1).
These methods can be categorized into two groups.
One group, such as clusterMaker, searches for the node clusters based on the network topology information such that the nodes in the same cluster have dense connections, but sparse connections between the different clusters [56].
The other group, such as jActiveModules, uses the quantitative fold-changes of the nodes between control and patient samples measured by LC-MS/MS together with the network topology information.
In this group, the node clusters are identified such that the nodes in the same clusters are densely connected and also show similar alteration patterns in protein abundance [53].
For functional interpretation of the node clusters resulted from network clustering, functional enrichment analyses mentioned above are then performed for the nodes in each network cluster.
Of the node clusters, prior to the enrichment analysis, we can focus on the major network clusters including large numbers of the DEPs.
The functional enrichment analysis then provide a subset of the major network clusters whose functions (e.g., GOBPs or KEGG pathways) are associated with the disease-related cellular processes.
This subset can be considered as potential determinants of the disease-perturbed cellular network.
Finally, the hub-like DEP in each major network cluster is selected as a biomarker candidate that can reflect the perturbation of the disease-related cellular process associated with the network cluster.
Mitra et al.
[57] demonstrated the power of the network-based biomarker discovery approach in prediction of metastatic cancers in human breast cancer.
They identified network clusters with dysregulated expression using SigArSearch and showed that the network clusters were more accurate in distinguishing metastatic cancers from non-metastatic cancers, compared with individual cancer-gene markers.
Title: Integrative analysis of proteomic data with other global data

Other global data than proteomic data, such as mRNA expression, mutation, DNA methylation, histone modification, can provide complementary information that enables to select reliable biomarker candidates when they are integrated with the proteomic data.
The integrative analysis of proteomic data with other types of global datasets can be categorized into two groups (Fig.3): (1) the molecular level and (2) the network-level integration methods [58].
The proteome data are contaminated with the noises coming from numerous sources of technical variability during LC-MS/MS experiments and biological variability in patient samples during sample collection and preparation [5960].
The integration of disparate global datasets reduces the artifacts of the noises in selection of the DEPs by enabling us to focus on the reliable DEPs showing consistent changes between control and patient samples in multiple types of global data [58].
In the molecular level integration method, the DEPs that show consistent alterations in the other global datasets (e.g., mRNA expression data) are searched, assuming that they are more reliable indicators of the perturbation of cellular processes under disease conditions (Fig.3).
Biomarker candidates can be then selected from these reliable DEPs.
For example, the DEPs selected from the tissue data can be supported by the consistency in the altered expression of their corresponding mRNAs.
Also, the DEPs can be further supported by the consistency in the changes of copy number variations or DNA and/or histone methylations of the corresponding genes.
Moreover, the reliability of the DEPs selected from the serum proteome data can be supported by the altered expression of their corresponding mRNAs or proteins in the tissues from the target organ.
Hyung et al.
[61] demonstrated the value of the integrative analysis of serum proteome data with mRNA expression and proteome data obtained from the tissues in human breast cancers.
They selected the DEPs between the serum samples collected from sensitive and resistant patients to a combinatorial chemotherapy using doxorubicin and docetaxol and then further selected the DEPs with altered expression of the corresponding mRNAs and proteins in the breast tissue samples.
During the validation of biomarker candidates, they showed that the DEPs with the consistent alteration in the tissues showed higher accuracy in their validation using western blotting for a validation cohort of independent serum samples.
Individual patients can show the variation in altered molecules of a cellular pathway though the same pathway is consistently perturbed under disease conditions.
Given the variation, these molecules cannot be selected as DEPs using the conventional statistical methods, thereby leading to the failure to identify biomarkers that reflect the alteration of such cellular pathways.
Thus, it has been addressed that cellular pathways can serve as a more reliable indicator of the altered cellular processes under disease conditions, compared to individual molecules [58].
Different types of molecules (mRNA/proteins, microRNAs, DNA methylations, and metabolites) can represent distinct layers of cellular networks.
Thus, a cellular network can be modeled in a multi-layered network where each layer can be delineated by a distinct type of the molecules.
For example, transcriptional and microRNA regulatory networks are defined by interactions of transcription factors and microRNAs, respectively, with their target mRNAs.
Also, cellular signaling networks are defined by protein-protein interactions (kinase-substrate interactions), while DNA methylation networks are defined by interactions of methyltransferases and demethylases with their target DNAs and mRNAs.
The integration of multiple global datasets for different types of molecules enables us to decode the multi-layered cellular networks associated with the target disease.
A number of the tools have been developed to understand the subnetworks (network clusters) of the multi-layered networks whose perturbations are collectively indicated by different types of global datasets, including IPA (QIAGEN Redwood City, http://www.qiagen.com/ingenuity), HotNet [62], or SteinerNet (Table 1) [63].
Although these tools can be applied for integration of proteomic data with other types of global data, they have been used mainly for integration of disparate genomic and transcriptomic data.
Recently, Shi et al.
[64] developed NetGestalt that can be used for integration of multi-dimensional global datasets including proteomic data.
For example, using NegGestalt, Zhu et al.
[65] identified KRAS and AKAP12 subnetworks that can play important roles in pathogenesis of colorectal cancers by integrating proteomic data with mutation, copy number variation, DNA methylation, and mRNA expression data generated from colon tissues and cells.
Finally, the hub-like molecules in the subnetworks are selected as key indicators of the alterations of the disease-related processes associated with the subnetworks.
For example, Iliopoulos et al.
[66] identified a set of the biomarker candidates by performing the network-based integration of the proteome and microRNA expression data.
Also, The Cancer Genome Atlas (TCGA) research network identified the notch signaling pathway as a key molecule representing the pathogenesis of ovarian cancers by integrating somatic mutation, copy number variation, and mRNA expression data using HotNet analysis [67].
In this review, we summarized a battery of bioinformatics analyses for network-based biomarker discovery using LC-MS/MS data.
These analyses include peptide/protein identification using database search engines, peptide/protein quantification from MS or MS/MS data, identification of DEPs using statistical methods, functional enrichment analysis of DEPs using diverse enrichment tools, and network modeling and analysis of DEPs, as well as integrative analysis of disparate global datasets with the LC-MS/MS data.
Peptide identification using database search has been known as a main challenge.
Of the MS/MS spectra measured, about half of them are commonly mapped to peptide sequences.
The inclusion of PTMs in database search can map the unidentified MS/MS spectra to peptide sequences.
However, the PTM inclusion exponentially increases the search time and thus it would be practically impossible to include the most common five PTMs, phosphorylation, glycosylation, ubiquitination, methylation, and acetylation in database search.
Recently, although several search tools, such as MODa [68], with improved search speeds have been developed, there is still significant needs for efficient search tools that can include the five common PTMs in the search.
Moreover, protein quantification has been also one of the challenges in MS-based proteomic analysis.
Relative protein abundances are estimated by combining the ratios of abundances of the peptides derived from the protein.
The same peptide sequences can be derived from multiple proteins due to the redundancy in their sequences, which can cause the discrepancy in the ratios of the peptides from the same protein between control and patient samples.
Also, technical and biological variability can add the discrepancy to the peptide ratios.
Because of these sources of the discrepancy in the peptide ratios, it has been difficult to estimate the relative ratios of protein abundances between control and patient samples.
Recently, although the optimization-based quantification tools, such as linear programming [33], have been developed, there have been significant needs for the quantification methods with improved accuracy.
Due to the issue with protein quantification, identification of DEPs can be also erroneous when inaccurate ratios of protein abundances exist.
Additional criteria, such as the number of patients showing fold-changes larger than a cutoff (e.g., 1.5-fold), are used to reduce the error (false-positives) in identification of DEPs.
The error in identification of DEPs can further propagate into functional enrichment analysis and network modeling and analysis for the DEPs.
Finally, the integrative analysis of disparate global datasets suffers from different scales of fold-changes and the size of detected molecules in the different types of global datasets.
For example, mRNA fold-changes between control and patient samples can be relatively larger than the fold-changes of protein abundances, which may cause the bias toward mRNA fold-changes during the integration of fold-changes of mRNA and protein abundances.
Also, differentially expressed microRNAs between control and patient samples are smaller than the DEPs or mRNAs, which can lead to the bias toward the DEPs or mRNAs during the integration.
Various integration strategies have been proposed to reduce the intrinsic bias from the differences in the size of disparate global datasets.
For example, in these methods, mRNA and protein abundance ratios are first integrated and the data with small sizes are separately integrated to the outputs from the initial integration of mRNA and protein data.
However, regarding the scale difference in fold-changes, there have been the needs for the algorithms that can effectively normalize the different scales of fold-changes in different datasets.
Basic data analysis pipeline for liquid chromatography–tandem mass spectrometry (LC-MS/MS) data.
This pipeline includes peptide/protein identification, peptide/protein quantification, and identification of differentially expressed proteins (DEPs).
Each of bioinformatics analyses in the pipeline is schematically shown together with the concepts and the tools.
PSMs, peptide-spectrum matches; FDRs, false discovery rates; iCAT, isotope-coded affinity tag; iTRAQ, isobaric tag for relative and absolute quantitation; TMT, tandem mass tags.
Schematics for functional enrichment and network analyses of differentially expressed proteins (DEPs).
Three types of functional enrichment analyses are shown.
The output from gene set enrichment analysis (GSEA) is displayed as an example.
Also, an example of network model is shown, which includes functional modules 1-3 (cell adhesion, vasculature development, and cell death, respectively).
Node colors in the network model represent up- (red) and down-regulation (green), the color gradient denotes log2-fold-changes of the proteins, and gray lines represent the connections between the nodes.
GOBPs, gene ontology biological processes; KEGG, Kyoto Encyclopedia of Genes and Genomes; TGF, tumor growth factor.
Integrative analysis of proteomic data with other types of global data.
The concepts for molecular level (A) and network based integration (B) are shown.
In the top panel, two example cases (one for tissue data and the other for serum data) are illustrated, and also one example case for integrative analysis of three types of global datasets (proteomic, mRNA expression, and DNA/histone methylation data) is shown.
In the bottom panel, network based integration of four types of global data using differentially expressed molecules from them are schematically displayed.
DEPs, differentially expressed proteins; IGF, insulin-like growth factor; LC-MS/MS, liquid chromatography–tandem mass spectrometry.
Title: Resources for network-based protein biomarker discovery
Title: The OAuth 2.0 Web Authorization Protocol for the Internet Addiction Bioinformatics (IABio) Database

Internet addiction (IA) has become a widespread and problematic phenomenon as smart devices pervade society.
Moreover, internet gaming disorder leads to increases in social expenditures for both individuals and nations alike.
Although the prevention and treatment of IA are getting more important, the diagnosis of IA remains problematic.
Understanding the neurobiological mechanism of behavioral addictions is essential for the development of specific and effective treatments.
Although there are many databases related to other addictions, a database for IA has not been developed yet.
In addition, bioinformatics databases, especially genetic databases, require a high level of security and should be designed based on medical information standards.
In this respect, our study proposes the OAuth standard protocol for database access authorization.
The proposed IA Bioinformatics (IABio) database system is based on internet user authentication, which is a guideline for medical information standards, and uses OAuth 2.0 for access control technology.
This study designed and developed the system requirements and configuration.
The OAuth 2.0 protocol is expected to establish the security of personal medical information and be applied to genomic research on IA.
Information technology has developed so rapidly all over the world that smart devices have become popularized and digital contents have been diversified.
As a consequence, internet addiction (IA) has become an important issue.
Although the prevention and treatment of IA are more important than ever due to pervasive digital media and although measures to establish a systematic research environment and overcome addiction problems are more necessary than ever, unfortunately, these steps not progressing fast enough.
It has been known that IA leads to diverse psychosomatic problems, such as nonaggressive or oppositional behavior and hostility.
In Korea, it was reported that 7% of internet users aged 5 to 54 years are at risk of IA, and the percentage keeps growing [1].
IA disorder, which has many similarities to psychopathological disorders, is on the rise and is considered a potential problem.
Therefore, a diagnostic standard is required considering DNA, protein, and the characteristics of the structure of the brain.
Many studies have been conducted on genomes that affect addictions to substances, such as alcohol, and a considerable number of databases have been built up.
However, a biological database of genes, proteins, and brain structures, which can be used directly for IA, is not sufficient.
To build a genome database related to IA, personal genome data, which are sensitive to handle, have to be examined, so security should be guaranteed.
In addition, a medical information standard-based environment is very important in terms of security.
It requires a medical information standard and access control technology for the service environment of genomic database [23].
Integration in the Healthcare Enterprise (IHE) [4] is providing various types of standard guidelines to control access to medical information.
IHE access control guidelines include cross-domain user authentication (XUA), enterprise user authentication (EUA) and internet user authentication (IUA) [5].
IUA provides the guidelines for access to OAuth 2.0-based medical information service.
OAuth 2.0 [6] is an access control method that manages several services in a group with one integrated ID.
It is used by Google and Facebook.
Therefore, it provides easy scalability into medical systems and easy control of authority for access [7].
In this paper, we propose the medical standard-based IA Bioinformatics (IABio) database platform service, which can integrate the data on genes, proteins, and brain structure image data in internet and game addiction into one database.
IABio follows medical standard-based guidelines and controls access to database using OAuth 2.0.
Therefore, using IABio can provide a medical standard-based service environment and will allow various integrated research studies with an access control method that enables interchange with other clinical data.
The present study is composed as follows.
Relevant studies and literature on this topic are described in section II.
In section III, IABio is designed.
In section IV, the designed IABio is explained.
Our study closes with conclusions in section V.
Title: Addiction genome database

A genome database has many merits, such as predicting the occurrence of disease and thus preventing and swiftly diagnosing it, which leads to early treatment; determining the prognosis accurately by selecting the treatment of a certain gene; and evaluating the effectiveness of treatment.
It has been difficult to verify the connection between addiction and genes in the past, but some attempts have been made to identify the DNA responsible for addiction in the last decade.
The findings say that when a genetic mutation occurs in a certain region of dopamine receptor in a human brain, dopamine receptor fails to function properly and thus can have superficial behavioral inclinations, such as addiction to smoking, alcohol, or drugs [8].
According to those studies, when genetic mutation occurs in a certain region of dopamine receptor in a human brain, dopamine receptor fails to function properly, and when the mutated genes make contact with a substance, they secrete acetylcholine, which stimulates pleasure, which leads to addiction.
The genes known to be related to addiction are DRD2 Tag 1A, DRD2 Tag 1B, and so on.
A German psychobiologic research team examined and compared 132 internet addicts and 711 normal people to find the cause of addiction and found that most of the addicts had a mutated gene called CHRNA4NA4 [9].
It is necessary that a database be built for addiction and that a service provision environment be established for studies regarding IA.
Therefore, an integrated database is required to support genome research and analysis.
Title: IUA/OAuth 2.0

Genes, proteins, and neuroimages require tight security, because they are very sensitive information in terms of personal information privacy.
To use such information, medical standards should be complied and proper guidelines should be used.
Our study uses the IUA method by IHE, which provides medical standard guidelines [45].
IUA is an IHE profile based on access control with token-based authentication and provides access authority to medical information services and guidelines for user authentication.
The IUA guideline is seen in Fig.1.
It can grant only necessary authorities to other users according to database policy and can also provide other services in a single server, which means that it has an excellent scalability.
In addition, when authorization is obtained from one authorization server, it allows the authority to use the information related to a certain user.
For an authorization client to use the resource, it is necessary to obtain an authorization token from the authorization server.
Authorization token holders need to check the terms of validity before using it.
In case the term expires, they need to get new authorization tokens.
A database environment is suitable for using other data, such as genes, proteins, and brain images, and can protect patients&#39; data systematically.
In this study, IUA is used to build a database applied with a medical information standard and OAuth 2.0 to be applied to standard IUA guidelines.
OAuth 2.0 is a protocol with authorization function to control and manage access to web services.
The HTTP RESTful method is used to perform access control using an authorization token and to ask an authorization server for resources [10].
HTTP RESTful is a method to request services through a URL.
Because it uses data messages in the form of JavaScript Object Notation (JSON), which is small in size, it is mainly used for mobile services at present [11].
OAuth 2.0 uses Transport Layer Security (TLS) to give and take authorization tokens and messages.
There are four authorization methods.
Because the authorization method differs by scenario, load in a server should be considered.
It should support scalability for a large service.
In addition, the service provider and authorization server are completely separate; so, server multiplexing and authorization server multiplexing are possible.
There are four authorization methods: authorization code grant (ACG), implicit grant (IG), resource owner password credentials grant (ROPCG), and client credential grant (CCG).
ACG and IG are re-direction methods and should interact with resource owners and receive requests.
ROPCG is an authorization method in which a client receives a resource owner&#39;s authorization information directly and relays it to the authorization server.
CCG can provide requests only with an authorization token.
Complicated methods, such as ACG and IG, are not needed in a database environment for analysis, because the level of trust between users and providers is high.
In addition, CCG can not be used, because resources have to be used.
Therefore, ROPCG is the most appropriate method for IABio.
The IABio platform service can be applied with medical information security guidelines; can provide a portal service; and can collect and analyze databases related to IA.
To design the proposed service model, our study explains the analysis of requirements, the structure of the platform service, the contents of the service, and the user authorization method.
Title: Analysis of requirements

It is necessary to analyze the complicated requirements of the system to design and develop the IABio platform service.
Because security is very important to a medical information system, it requires user identification.
In addition, the data exchange method, integrated data composition, and services should be specified.
Title: User identification

It is needed to analyze patients&#39; genomic data concerning IA for treatment and prevention and to provide them with customized service.
To do so, an integrated data analysis system is required and should work with all genomic information.
Users with access to IABio are classified into gene researcher, protein researcher, brain imaging researcher, senior researcher for integrated work, and server manager.
Each manager has access to a patient&#39;s genome information according to the Medical Care Act, but the researcher is limited to only &#39;concerned&#39; information and can not have access to other information.
However, all of the researchers can have access to the risk outcome that is yielded from the integrated analysis of all genomic information.
The integrated analysis risk results do not include individual patient information but are considered new information and thus are not against the Medical Care Act.
Therefore, each researcher can search, enter data, and modify genomic data he manages and can search out integrated analysis outcomes.
Title: Standard-based biological data exchange

To subdivide access from each researcher, access control is needed or else the server manager can not even read the genomic data.
Therefore, a server manager can only control information that the genome manager has access to.
To do so, the authorization server and IABio server have to be separated physically.
Medical information standard-based guidelines and access control are necessary for authorization and approval.
Title: Integrated biological data platform

Biological data (gene, protein, and brain structure image) have different identifier values.
However, if several pieces of genomic data for one patient have more than one identifier, it is against data integrity, because information for one patient is duplicated.
In addition, because genome information entered by each data manager is created in different systems, an identification system is needed.
Title: Web portal service

Access to an integrated database requires an accessible service and web portal service in which the authorization and approval procedure has been established, which is necessary to search data related to a genome.
Therefore, it is necessary to provide an integrated service using a security system suitable for medical standards.
Accordingly, a web portal is designed using OAuth 2.0.
Title: Provision of data analysis output

IA risk factors should be provided through integrated analysis based on data entered by each researcher and user.
Integrated information should be provided for single nucleotide polymorphisms (SNP) and protein markers related to IA.
Also, the accuracy of risk factors should advance enough to utilize a diagnostic index.
After risk factors that are highly relevant to IA are determined through the integrated analysis of several areas in the genome, a risk evaluation score for the respective factor should be provided.
To compose such a sophisticated system, our study uses medical standard guidelines in designing the genome database platform.
Title: IABio service architecture

Aiming to provide a genome database platform for IA, our study designs the IABio platform service structure in consideration of medical standard security and requirements as shown in Fig.2.
By relevance to the genome, the users are divided into gene researcher, protein researcher, brain imaging researcher, and senior researcher.
Each user can operate and monitor accessible data by his/her authority.
In addition, the users can connect to it through a smart device or web service.
For users to have access to IABio, they should be issued authorization tokens from the authorization server.
The authorization token functions to identify researchers using genome information and is physically separated from the IABio platform.
When using such an authorization token, it is possible to access IABio.
When an authorization token is not valid or when a user is not registered, the token should be issued or re-issued by the authorization server.
Our study complies with IUA medical standard guidelines for access, uses OAuth 2.0, and composes the security system.
IABio is accessible through a web portal.
Genome data entered by each researcher are entered in a database in an agreeable format to master patient index (MPI) structure.
Because gene, protein, and brain imaging data are all in different data formats, a method is necessary to integrate them.
Therefore, MPI is used.
It allows scalability in connection with different clinical data.
MPI configures the data format in a way to identify them, develops them into a hash-type algorithm, creates an identification system, and maintains data integrity.
In an MPI-based identification system, each manager does not know the password structure.
Therefore, researchers can not have access to different data.
The security system of the proposed IABio platform has pre-defined password keys with the authorization server and performs the authorization and approval procedure by the user.
The contents of MPI are not described here, because they are not part of the aim of this study.
Three analyses are performed to provide services from IABio.
The first is proteomics analysis.
Proteomics is a study concerned with the structure and analysis of proteins occurring in living organisms.
It aims to analyze proteins on a great scale; write a map of functional interrelations; and demonstrate a certain protein and the gene that creates it through structural analysis.
The second analysis is IA risk factor analysis.
SNPs and protein markers related to IA yield risk factors for IA.
In addition, it analyzes clinical data, such as survey, patient interview, functional magnetic resonance imaging, and brain imaging, in addition to a genome/protein index, in integrated manner, finally creating a diagnostic index.
The third analysis is risk evaluation analysis.
When risk factors highly relevant to IA are determined by integrated analysis of genome data, it generates an algorithm to provide a risk evaluation score for the respective factor.
In this study, the contents of genome analyses are not described, because they are separate areas of study.
Because IABio service model proposed in our study can ensure safe access to information entered by users and can provide integrated data analysis service.
It can be applied to several platforms as well as studies related to IA.
Title: IABio service architecture

Fig.3 shows the authorization processing procedure of the token method and the service contents that each user is provided in the use case diagram.
Four actors consist of a researcher, database, web portal, and authorization server.
A researcher accesses the authorization system to have access to the IABio platform and requests an access token.
The authorization server confirms the legitimacy of the access by the user and issues the right access token for his/her authority.
Then, the researcher acquires the access token.
The researcher proceeds to access the IABio platform through a portal.
The web portal and database identify the access token.
When the login is done properly, the user can input and output data.
Data can be entered according to information related to a concerned genome.
Genome input data are analyzed, and the analyzed information is saved.
The researcher requests data output to monitor the analysis results and related data.
Token authorization contents are analyzed for a second time to check if the researcher is legitimate for data confirmation before the analysis results and entered information are retrieved.
Title: OAuth 2.0-based authorization model

In this study, the IUA guideline [5] and OAuth 2.0 [6] are used to design and realize IABio.
Researchers should obtain access authorization to use IABio and the concerned data.
The server manager who controls the authorization server should not have access to the database in which patient information is saved.
Therefore, the IABio server should be physically separated from the authorization server and provide only authorization and authority.
This kind of environment is suitable for the IUA guideline.
In this study, the authorization method uses the OAuth 2.0-based ROPCG method and is designed in a communication diagram, as in Fig.4.
Our study proposes an authorization method based on the 2-step access authorization of IABio, along with resource owner server (web portal) authorization.
Each researcher (client) should be assigned an authorization token from the authorization server to get access to the portal.
After obtaining the authorization token, the researcher has access to the portal and proceeds with the first authorization.
Next, the researcher gets the second authorization in IABio to use the service.
Because security authorization is strictly designed in double walls, it can avoid weaknesses to phishing scams and impersonation attacks.
The first step of the authorization procedure is to check if it is possible to identify users&#39; personal information.
The second authorization step proceeds with authorization for database access one more time.
Our study uses this 2-step authorization in designing and developing IABio.
Title: System environment

Fig.5 shows the system environment in which the IABio system is composed with OAuth 2.0 of a medical information standard environment.
The system environment has the structure of a hardware and development environment.
Two servers are used: IABio server and authorization server.
HP ProLiant Micro Server Gen8 is used to design the IABio database, and it consists of 1TB SSD, 2TB HDD, and 12GB RAM.
To compose the development environment, Apache 2.2 is used as server software to configure the Tomcat Server.
For the web service, PHP 5.2 is used to supplement the functions of existing HTTP language.
MySQL 5.1 is used to compose the Tomcat server and PHP, which can interwork with the database.
APM (Apache, PHP, MySQL) is mutually complementary and flexible for interworking.
Furthermore, it is compatible with all operating systems, such as Unix, Linux, and Windows.
The authorization server uses Naver application programming interface (API), which supports OAuth 2.0 [12].
Since Naver API provides a security server in itself, it is cost-effective.
In addition, it is possible to customize so as to receive an authorization token by using Naver ID.
Currently, most domestic web service providers related to authorization use OAuth 2.0 of Naver API.
Therefore, it can ensure security.
JSON method is employed for the exchange of messages between a portal and database server and authorization server.
The JSON method enables message exchange with shorter codes than the existing Extensible Markup Language (XML) method.
Therefore, it is faster in speed and can reduce load.
For security protocol, IUA guidelines and OAuth 2.0 are used.
PHP 2.2 is used to realize OAuth 2.0.
The JSON method can transmit authorization tokens and protect users&#39; personal information.
The database module is structured to have access to genome data.
The authorization token method should be used to have access to a database module.
The Advanced Encryption Standard (AES) 256 algorithm is used to enhance the level of security.
AES 256 is a symmetric key algorithm that supports block passwords.
AES 256 is the most reliable symmetric key algorithm among any developed thus far.
Its method uses key values used between a user and service.
A user can access the web portal and database without concerned keys.
PHP 2.2 is used for the module to have access to and operate IABio.
In addition, MPI is used to ensure the integrity of the genome database.
It can support interworking between data.
Therefore, data integration is possible.
Furthermore, because MPI uses Hash 256, each user can confirm which data are interworked.
The Bio-Marker database is composed so as to receive the entry of gene data, protein data, and brain imaging data.
Title: Security evaluation

OAuth 2.0 has a weakness for vulnerable replay attacks.
In this study, a replay attack can solve problems by using the 2-step access authorization.
Replay attack and defense scenarios are shown in Fig.6.
An attacker is disguised as a client to obtain an EC1 key (with access token) when passed during the communication between the authorization server and client.
The attacker, disguised as a client, attempts to perform a replay attack obtained using the access token.
In this case, the attacker has access to the web portal using the EC1 key (with access token) shared with the authentication server.
However, the EC2 key is needed to access IABio database.
An attacker who does not have the EC2 key can not access the data, because the attacker can not analyze contents in the access token of EC2.
Thus, an attacker can not access the IABio database, because the attacker does not have the EC2 key.
EC1 and EC2 can be obtained only from the pre-consultation process.
Title: Implementation screen

The IABio platform uses the authorization server provided in Naver API [12].
Therefore, it can acquire an authorization token in Naver and can have access to the portal by using it.
To access the authorization server, only a pre-authorized Naver ID can be used.
Therefore, a token can be issued or re-issued only when having client information.
The left side of Fig.7 shows the page to access the web portal, and the right side is the page to receive authorization tokens through Naver.
As seen in Fig.8, login is done to the authorization server with client information.
Then, it is possible to have access to the web portal of IABio.
The right side of Fig.8 shows the contents of personal information and the authorization token received from Naver.
The authorization token is transmitted in the encryption of AES 256.
The contents are those of decoded authorization tokens.
Figs.9 and 10 are output results made by query from the data saved in the genome database.
It is a screenshot of sample data, not actual clinical data.
Actual data can not be printed out due to the Medical Care Act.
Fig.9 shows the screen that confirms the patient&#39;s basic information.
Only personnel who are authorized for access to it can read it.
Fig.10 is a screenshot that confirms protein data.
Only the personnel who are authorized for access to protein data can read it.
IA is a kind of illness in which the occurrence increases along with the development of IT technology.
Although it is urgent to have an analysis database of IA, it is not sufficient now.
Because the database needed to analyze medical information handles very sensitive personal data, security is very important.
Our study proposed the IABio platform service using OAuth in a medical information standardbased environment.
The proposed IABio uses IUA as a medical standard guideline and complies with medical information standards.
Also, it uses OAuth 2.0, which is an access control technology.
To realize IABio, it is necessary to analyze the complicated requirements of a medical information system and to design the system from the perspective of security.
In addition, our study developed a prototype IABio system and tested if it could be applied with a security system.
As a result, it was found that each user had a different environment in which the researcher needed to access a database.
Therefore, it is expected that the proposed IABio can be actively used for studies of IA.
As follow-up research, solutions to phishing scams and re-use attack, to which OAuth 2.0 is vulnerable, will be examined.
Also, an index searching method will be studied to quickly search genome information related to IA.
Internet user authentication actor diagram [6].
Internet Addiction Bioinformatics (IABio) service model.
User case diagram of Internet Addiction Bioinformatics (IABio) service.
Communication diagram of authentication method with resource owner password credentials grant [6].
IABio, Internet Addiction Bioinformatics.
Internet Addiction Bioinformatics (IABio) system environment.
MPI, master patient index; JSON, JavaScript Object Notation; API, application programming interface.
Replay attack scenario and defense.
IABioDB, Internet Addiction Bioinformatics database.
Authorization server page (left and right images indicate web portal login page and Naver login page for access token, respectively) [12].
IABio DB, Internet Addiction Bioinformatics database.
IABioDB web portal (left and right images indicate Naver login page and IABioDB web portal access, respectively) [12].
Patient information (sample).
Protein data (sample).
Title: Compiling Multicopy Single-Stranded DNA Sequences from Bacterial Genome Sequences

A retron is a bacterial retroelement that encodes an RNA gene and a reverse transcriptase (RT).
The former, once transcribed, works as a template primer for reverse transcription by the latter.
The resulting DNA is covalently linked to the upstream part of the RNA; this chimera is called multicopy single-stranded DNA (msDNA), which is extrachromosomal DNA found in many bacterial species.
Based on the conserved features in the eight known msDNA sequences, we developed a detection method and applied it to scan National Center for Biotechnology Information (NCBI) RefSeq bacterial genome sequences.
Among 16,844 bacterial sequences possessing a retron-type RT domain, we identified 48 unique types of msDNA.
Currently, the biological role of msDNA is not well understood.
Our work will be a useful tool in studying the distribution, evolution, and physiological role of msDNA.
A retron is a 2-kb-long bacterial retroelement that carries a promoter and three genes: msr, msd, and ret.
The msr/msd part of the transcript is processed by a reverse transcriptase encoded by ret, resulting in an RNA/DNA chimera called multicopy single-stranded DNA (msDNA) [1].
The RNA and DNA parts of msDNA are encoded by msr and msd, respectively.
The precursor RNA molecule possesses a palindrome between the 5&#39; and 3&#39; ends, forming a doublestranded region between both ends (Fig.1A).
The 5&#39; part of the palindrome is followed by a G residue, which is the branching point for covalent bonding to the cDNA (Fig.1B) [2].
The msr portion of the precursor RNA possesses two stem-loop structures, which are recognized by the reverse transcriptase [3].
The msd part of the resulting chimera, shown in black in Fig.1C, is palindromic, also forming a large hairpin structure.
While msDNA is found in many bacterial species, its biological role is not well understood.
Furthermore, msDNA is not well annotated in public sequence databases.
In order to facilitate the biological characterization of msDNA, we set out to compile msDNA sequences from known bacterial genomes.
Using a few known msDNA sequences, we extracted common sequence and structural features.
We screened National Center for Biotechnology Information (NCBI) RefSeq bacterial genome sequences using this rule to compile potential msDNA sequences.
Their authenticity as msDNA needs to be confirmed experimentally.
Title: Prototype msDNA sequences and their features

We used one of the published Escherichia coli msDNA sequences (GenBank accession No.
U02551) [2] as the query in a Basic Local Alignment Search Tool (BLASTN) search of NCBI GenBank High Throughput Genomic Sequence (HTGS) division.
From the BLASTN hits, we manually curated seven other known msDNA sequences (Table 1).
By visual inspection of these prototype msDNA sequences, several common features were recognized: (1) the length of palindrome between the 5&#39; and 3&#39; ends ranged from 5 bp to 10 bp; (2) the distance between both ends ranged from 140 bp to 200 bp; (3) the residues flanking the palindrome were conserved G residues in both strands; and (4) the distance between msr/msd and ret was less than 2 kb.
We developed a local python script that enforces these rules in the candidate retron sequences.
Title: Gene prediction and protein domain search

Title: From the downloaded DNA sequences, genes were predicted with the GeneMark suite [4].
The input sequences were split into 20-kb batches, and GeneMark was run for each batch with the default settings.
The predicted genes were translated into protein sequences, which were then scanned for the Clusters of Orthologous Group (COG) domain using Reverse Position Specific (RPS)-BLAST.
COG is a microbial domain database distributed by NCBI.
RPS-BLAST is one of the BLAST applications distributed by NCBI and uses position-specific scoring matrices (PSSMs) as the target database [5].
We looked for hits of the accession COG3344 (PSSM ID: 225881), which stands for &#34;Retron-type reverse transcriptase (mobilome: prophages, transposons).&#34;
Title: Palindrome motif search

One of the characteristic features of msDNA is a palindrome formed by its 5&#39; and 3&#39; ends.
In order to detect such a feature in the potential retron sequences, we used a local installation of the software tool palindrome in the European Molecular Biology Open Software Suite (EMBOSS) bioinformatics software package [6].
It requires several options to be set.
The minimum and maximum lengths of a palindrome and the maximum distance between a palindrome pair were set to 5, 10, and 200 bp, respectively, while the number of mismatches between a palindrome pair was set to 1.
Title: Profile search of msDNA

A multiple-sequence alignment of the eight prototype msDNA sequences was generated with ClustalW (Fig.2).
We then built a hidden Markov model (HMM) profile from the multiple-sequence alignment using the software tool HmmerBuild in the HMMER package [7].
The potential retron sequences were screened using HmmerSearch with the HMM profile [7].
The cutoff for a specific hit was determined by leave-one-out crossvalidation as follows: (1) among the eight seed sequences, one was set aside, and the remaining seven were used to build the profile; (2) what was set aside was then evaluated using the profile; (3) this process was repeated for each of the eight seeds, and the minimum score was defined as the cutoff (45.0).
Title: RNA secondary structure prediction

The precursor RNA molecule of msDNA forms a characteristic secondary structure.
We used a local installation of the software tool CentroidFold [8] to predict the secondary structure of the msr/msd portion of the candidate retron.
As the palindrome terminates with a G residue in both strands, the 5&#39; of which is the branching point for covalent bonding to the cDNA (Fig.1B), sequences without these G residues were removed.
The web service version of RNAfold [9] was also used to visually confirm the final prediction set.
Title: Compilation of candidate retron sequences from known genome sequences

Historically, msDNA has been discovered exclusively in bacteria.
A keyword search of NCBI RefSeq bacterial genomes with &#34;reverse transcriptase&#34; returned 34,637 hits, as of September 2, 2015.
Following the gene prediction of these genomes using GeneMark, a protein domain search for &#34;Retron-type reverse transcriptase (mobilome: prophages, transposons)&#34; using RPS-BLAST resulted in 16,844 candidate retron sequences.
Alternatively, filtering out nonretron-type reverse transcriptase sequences can be also achieved by extracting protein sequences from the RefSeq annotation, followed by the domain search.
Instead, we chose to predict genes from the genome sequences using GeneMark, followed by the domain search.
Our approach does not rely on annotation information and has the potential to be applied to newly sequenced genomes as an independent tool.
Title: Automatic filtration of candidate msDNA sequences

The candidate retron sequences identified above were filtered by the presence of palindrome motifs.
Using the local python script, we searched the 16,844 bacterial sequences for palindrome motifs 5–10-bp-long and separated by less than 200 bp within 2 kb upstream of ret, yielding 7,428,332 such hits.
These hits were then screened by the HMM models that were developed based on the prototype msDNA sequences.
There were 3,865 hits surpassing the crossvalidation threshold of 45.0.
Using a local installation of CentroidFold, the RNA secondary structures were predicted for the remaining hits, and those without the proper double-strand pairing between the 5&#39; and 3&#39; ends were removed.
Furthermore, those without the conserved flanking G residues were also filtered out.
The whole process, which was wrapped in a Linux shell script, resulted in 625 hits.
Title: Manual curation and clustering of msDNA sequences

The final set of 625 candidate sequences that were identified by an automatic process was highly redundant.
The exact copies were pruned, giving rising to a set of 88 unique sequence types.
As shown in Fig.1A, the true msDNA should possess two hairpins in msr and a long hairpin in the msd region.
In order to confirm this complex overall topology of msDNA, we examined the RNA secondary structure plots of the 88 candidates visually.
In fact, we removed 40 sequences that did not form the appropriate overall topology.
See Fig.3 for a few representative surviving examples.
Among the original 625 sequences, 525 can be mapped to these 48 unique sequence types (Supplementary Table 1).
In order to cluster the remaining 48 sequence types further, allowing small divergences within clusters, multiple-sequence alignment was performed with Multiple Alignment using Fast Fourier Transform (MAFFT) [10].
From the resulting dendrogram, we recognized 20 groups by splitting the tree at a branch length of between 0.12 and 0.20 (Fig.4).
The species distribution of the 525 msDNA sequences over the 20 groups is given in Table 2 and Supplementary Table 1.
We developed a rule-based protocol to detect msDNA in a given genomic DNA sequence.
The rule is based on the presence of a 5–10-bp-long palindrome motif in the precursor transcript that is less than 2 kb upstream of a retron-type reverse transcriptase gene.
The rule is augmented by enforcing a sequence similarity with known msDNA sequences.
This step is implemented with an HMM profile search method.
While this was a powerful filtration step, reducing the hits by about 1/20, it would inevitably miss some true msDNA sequences that are distantly related to the prototypes given in Table 1.
As more distinct msDNA sequences are discovered, the panel of the seed sequences must be updated.
The list of potential candidate msDNA sequences was filtered by conformation to the known topology of the RNA secondary structure.
The RNA secondary structure prediction programs usually output the result in terms of Vienna dot bracket notation [9].
As the manipulation of the string is not straightforward, some manual curation was involved.
In the future, we will explore the possibility of automatic implementation, which is critical for web-based service of the prediction tool.
To our knowledge, this is the first large-scale annotation of msDNA in all publically available bacterial genome sequences.
As our work is computational in nature, the list we compiled can include some false positives.
Its authenticity should be validated experimentally.
Nevertheless, our list of msDNA sequences and their species distribution profile constitute a useful resource for the future study of msDNA, as its biological role is still elusive.
Schematic diagram of multicopy single-stranded DNA synthesis.
(A) A precursor RNA molecule with the characteristic palindrome (green).
&#34;G&#34; residues in a circle or box (red) are conserved.
(B) Reverse transcription yielding cDNA (black).
(C) The final msDNA chimera.
Adapted from Wikipedia (https://en.wikipedia.org/wiki/Multicopy_single-stranded_DNA).
Multiple-sequence alignment of prototype multicopy single-stranded DNA sequences.
The msr/msd portion of each genomic DNA sequence in Table 1 was used to construct the alignment with ClustalW.
The corresponding secondary structures, predicted with CentroidFold, are shown in bracket notation.
The conserved positions are colored with Jalview.
Representative examples of secondary structures of msr/msd precursors of predicted multicopy single-stranded DNA sequences.
(A) NZ_AERV01000006.1 Salmonella enterica (J, 86).
(B) NZ_ADUL01000064.1 Escherichia coli (B, 3).
(C) NZ_AELI01000009.1 Vibrio cholerae (E, 48).
(D) NZ_CBWG010000185.1 Escherichia coli (N, 504).
The single letters and numbers in parentheses refer to the clustering group and sequence number in Supplementary Table 1.
Dendrogram of the final 48 multicopy single-stranded DNA sequence types.
The plot was generated with a multiple-sequence alignment program, Multiple Alignment using Fast Fourier Transform (MAFFT).
Distinct clusters are marked with symbols, A–T.
Title: Prototype msDNA sequences used for the development of detection rules

Title: Distribution of 525 msDNA sequences in each group
Title: Non-negligible Occurrence of Errors in Gender Description in Public Data Sets

Due to advances in omics technologies, numerous genome-wide studies on human samples have been published, and most of the omics data with the associated clinical information are available in public repositories, such as Gene Expression Omnibus and ArrayExpress.
While analyzing several public datasets, we observed that errors in gender information occur quite often in public datasets.
When we analyzed the gender description and the methylation patterns of gender-specific probes (glucose-6-phosphate dehydrogenase [G6PD], ephrin-B1 [EFNB1], and testis specific protein, Y-linked 2 [TSPY2]) in 5,611 samples produced using Infinium 450K HumanMethylation arrays, we found that 19 samples from 7 datasets were erroneously described.
We also analyzed 1,819 samples produced using the Affymetrix U133Plus2 array using several gender-specific genes (X (inactive)-specific transcript [XIST], eukaryotic translation initiation factor 1A, Y-linked [EIF1AY], and DEAD [Asp-Glu-Ala-Asp] box polypeptide 3, Y-linked [DDDX3Y]) and found that 40 samples from 3 datasets were erroneously described.
We suggest that the users of public datasets should not expect that the data are error-free and, whenever possible, that they should check the consistency of the data.
The completion of the human genome project has accelerated the development of many omics technologies that have been used extensively for the genomewide profiling of human samples [1].
Public repositories, such as Gene Expression Omnibus (GEO) and ArrayExpress, now hold data on hundreds of thousands of samples profiled by diverse technologies [23].
For many of the samples in the public repositories, various kinds of clinical information are also included so that interested researchers can use them for their own research interests.
While these public data have enormous potential for research use, it is inevitable that unknown errors may creep into public datasets without being noticed by depositors.
For example, Microsoft Excel is notorious for its automatic conversion function, which erroneously changes tens of human gene symbols [4].
But, the errors are not limited to gene symbols and may occur in the clinical information, as well.
Age and gender are some of the most basic clinical information associated with human samples and are most unlikely to be erroneous during data acquisition.
While they are basic clinical information, the importance of age and gender in human biology is not trivial.
There is strong evidence that men and women differ in terms of development and severity of many common diseases, including cardiovascular diseases, autoimmune diseases, and asthma [5].
Recent clinical studies have revealed an association between several genetic diseases and gender-specific genetic patterns [67].
While analyzing public datasets produced using Infinium 450K HumanMethylation arrays (Illumina Inc., San Diego, CA, USA) and Affymetrix Human Genome U133Plus 2.0 arrays (Affymetrix Inc., Santa Clara, CA, USA), we found many samples that were discordant between clinical gender information and the patterns of gender-specific markers.
Importantly, the errors were not limited to a few datasets but were prevalent in many datasets produced by many different laboratories.
We advise that the users of public datasets should not expect that these data are error-free and, whenever possible, that they check the consistency of data.
Title: Data collection and processing

Title: Infinium 450K HumanMethylation array

More than 5,600 samples from 11 datasets were collected from GEO [2].
Before data integration, we calculated the average beta value for measuring methylation levels at each CpG site, ranging from 0 (least methylated) to 1 (most methylated).
The individual beta value was dropped if the detection p-value was over 0.05.
For all datasets, raw signal intensity files were collected, and from them, methylation level was calculated as β = (max(Cy5, 0)) / (|Cy3| + |Cy5| + 100).
A constant of 100 was added to the denominator to regularize β values when both unmethylated and methylated intensities were small.
Title: Affymetrix U133Plus2 array

A total of 1,819 samples from 4 datasets for gene expression experiments using the Affymetrix U133Plus2 array were obtained from GEO [2].
For all datasets, CEL files were collected and normalized by Robust Multiarray Average method [8].
Title: Selection of gender-specific DNA methylation and gene expression markers

The gender-specific DNA methylation markers of the X chromosome were selected from reported X-linked housekeeping genes [9].
The gender-specific DNA methylation markers of the Y chromosome were estimated using differentially methylated CpG sites in Y chromosomes between males and females (Supplementary Fig.1).
Gender-specific markers were selected based on their beta value distribution in both males and females.
As a result, two markers (cg24139739 and cg02869694) were selected as X chromosome markers, and two markers (cg07851521 and cg10835413) were selected as Y chromosome markers.
The cutoff values for each marker were cg24139739 &lt; 0.25, cg02869694 &lt; 0.4, cg07851521 &gt; 0.5, and cg10835413 &gt; 0.45 (Supplementary Fig.1).
The gender-specific gene expression markers were selected from reported gender-specific gene expression patterns in human blood (Supplementary Fig.2) in a similar way [10].
Two markers (214218_s_at and 224588_at) were selected as X chromosome markers, and two markers (204409_s_at and 205000_at) were selected as Y chromosome markers.
The cutoff values for each marker were 214218_s_at &lt; 4.5, 224588_at &lt; 7.5, 204409_s_at &gt; 5, and 205000_at &gt; 5.
Title: Data analysis

Python version 2.7.6 and the Pandas python library version 0.15.2 were used for most data analyses.
R version 3.1.0 and ggplot2 version 1.0.0 were used for image production.
Title: Collection of DNA methylation array data of human whole blood with age and gender information

We obtained DNA methylation microarray data from the NCBI GEO database (Fig.1).
We collected datasets of normal human blood samples in which both age and gender data were available.
As a result, we collected a total of 4,862 samples for Infinium 450K HumanMethylation array data (Table 1).
Title: Determination of gender by gender-specific markers

The CpG sites of X chromosomes in females are hypermethylated for dosage compensation of female X chromosomes [11].
By analyzing female-specific hypermethylated genes in the X chromosome (Supplementary Table 1), we identified 19 samples (0.39%) in which the methylation patterns of gender-specific CpG markers did not match the given gender information (Fig.2A).
When we analyzed those samples with male-specific hypermethylated genes in the Y chromosome (Supplementary Table 2), we again observed the opposite methylation patterns (Fig.2B).
Importantly, the discordant patterns were observed in eight of 11 datasets from different depositors, suggesting that the errors were not limited to one laboratory (Supplementary Fig.3).
Interestingly, we found both types of discordant errors between DNA methylation patterns and given gender information.
That is, for some samples, DNA methylation patterns were found to be female-specific while they were designated as males (designated here as discordant-male), and for the other samples, DNA methylation patterns were male-specific while they were designated as female (designated here as discordant-female).
Title: Analysis of markers of chromosome abnormality syndromes

We next analyzed whether the observed discrepancy between methylation patterns of gender-specific genes and the given clinical information could be explained by rare sex chromosome abnormality syndromes, such as Turner and Klinefelter syndromes.
While a normal male inherits an X chromosome and a Y chromosome and a normal female inherits two X chromosomes, several abnormalities in the number of sex chromosomes occur gender-specifically.
For females, abnormalities are a result of variations in the number of X chromosomes.
For males, abnormalities are due to irregular numbers of the X or Y chromosome or both.
The most frequent sex chromosome abnormalities in females are Turner (one X; 1:2,000) and triple X (XXX; 1:1,000), while those in males are Klinefelter (XXY; 1:500) and XYY (1:1,000) syndromes [12131415].
For comparison, we collected Infinium 450K Human-Methylation array data of one Turner syndrome patient and five Klinefelter syndrome patients.
The characteristic methylation pattern of Turner syndrome (one X) is the hypomethylation of both X- and Y-specific markers (Fig.3A).
However, we found that most discordant-female samples in our dataset showed hypermethylation patterns in Y-specific markers, suggesting that they had XY chromosomes (a normal male) but not X (Turner syndrome) chromosome (Fig.3A).
Only one discordant-female sample (green circle in Fig.2A) showed a pattern similar to Turner syndrome.
For Klinefelter syndrome (XXY), the expected methylation patterns are the hypermethylation of both male- and female-specific markers.
We observed one discordant-male sample in which both male- and female-specific markers were hypermethylated (red circles in Figs.2B and 3A).
However, a close examination of three Klinefelter syndrome-specific markers revealed that the one discordant-male sample did not show the patterns of a Klinefelter syndrome patient (Fig.3B, Supplementary Fig.4) [16].
In conclusion, only one sample showed a methylation pattern associated with sex chromosome abnormalities.
Title: Significant difference between clinical age information and predicted age using a DNA methylation age calculator

Recently, several studies have shown that DNA methylation markers can be used to estimate the age of an individual to within 5 y [17].
Because the datasets we analyzed had age information, as well, we tested whether the given age information deviated much from the estimated age from age-specific DNA methylation markers.
The age prediction was performed by using the DNA methylation age calculator, a web-based tool that provides a predicted age based on the methylation values of DNA methylation markers of Illumina&#39;s Infinium platform [18].
When we compared the absolute age deviation between concordant and discordant samples, we found much larger deviations in age prediction among discordant samples compared with concordant samples (Table 1, Supplementary Fig.5).
This result suggests that the discordant samples are highly likely to have errors both in age and gender information, a scenario that occurs when two samples are mislabeled as each other.
Title: Identification of gender-discordant samples from the analysis of gender-specific gene expression

To see if the same problem occurs in other types of data (e.g., gene expression), we collected gene expression microarray data from the NCBI GEO database using a similar strategy for methylation microarray data (Fig.4).
By applying the same criteria as with the DNA methylation data, we collected a total of 1,683 samples from 4 datasets produced using the Affymetrix U133Plus2 array platform (Table 2).
We analyzed the 1,683 samples using several gender-specific genes (X (inactive)-specific transcript [XIST], eukaryotic translation initiation factor 1A, Y-linked [EIF1AY], and DEAD [Asp-Glu-Ala-Asp] box polypeptide 3, Y-linked [DDDX3Y]) [10] and found that 40 samples (2.3%) from 3 datasets were erroneously described (Fig.5).
In the case of Y chromosome markers, some discordant male samples showed overlapping distribution of methylation levels compared with normal male samples.
Those samples needed to be compared samples with male sex abnormality syndromes (e.g., Klinefelter syndrome), but unfortunately, we could not find datasets with male sex abnormality syndromes produced using the same platform.
However, as these samples showed the opposite pattern in X chromosome markers, we considered them to be putative discordant samples.
But, further validation will be needed to confirm these samples.
Unlike the previous methylation data, some of the collected gene expression datasets contained technical replicates of the same samples (Table 2).
Surprisingly, discordant gender-specific expression patterns were even observed from some of the technically replicated samples (Supplementary Fig.6).
Recently, genomewide omics data have accumulated very fast due to advances in omics technologies (e.g., Illumina&#39;s Infinium methylation assay and next-generation sequencing).
Now, researchers can exploit public data repositories (e.g., ArrayExpress [3] and GEO [2]) that store data of hundreds of thousands of samples with both genomewide profiling data and associated clinical information.
However, as we have shown here, datasets are not error-free, and any type of error can occur in public datasets.
Indeed, one paper reported that the aggregate mislabeled blood sample rate was 1.12% at various US institutions in 2013 [19].
Possibly, various kinds of errors (e.g., mislabeling of tubes, mixing of samples) may occur from sample preparation to various steps of the experimental procedures.
When we began the analysis of Infinium 450K Human-Methylation array data, we did not expect errors in gender description to occur so often in several datasets.
We thus analyzed if those erroneous samples showed the methylation patterns of gender-specific chromosome abnormality syndromes (e.g., Turner and Klinefelter syndromes) but found that most of the erroneous samples did not.
Thus, we concluded that most discordant samples were real human errors.
This finding led us to analyze expression datasets of Affymetrix U133Plus2 array data, as well, and again, we found that errors were not rare as well.
Fortunately, for transcriptomic and epigenomic datasets, gender-specific markers (Supplementary Tables 1, 2, 3, 4) are well known, so that researchers can check whether the given gender information is concordant with the expected patterns of gender-specific markers.
For DNA methylation, a user-friendly website (https://dnamage.genetics.ucla.edu/) [18] is also available for researchers.
In conclusion, we suggest that users of public data should not expect that the data are error-free, and whenever possible, they should check them carefully before use.
Work flow of the Infinium 450K HumanMethylation array data analysis.
Methylation pattern of gender-specific markers.
X (A) and Y (B) chromosome markers.
The red circles indicate the discordant male sample showing overlapping distribution of levels for normal male samples.
M, male; F, female; M_dis, male samples showing methylation patterns of females; F_dis, female samples showing methylation patterns of males.
Comparison of methylation status between normal and sex chromosome abnormalities.
(A) Scatterplot of methylation status of gender-specific methylation markers on X and Y chromosomes.
The red circle indicates the sample with suspected Klinefelter syndrome.
The green circle indicates the sample with suspected Turner syndrome.
(B) Box plot of Klinefelter syndrome-specific methylation markers of normal males, normal females, a sample with suspected Klinefelter syndrome, and Klinefelter syndrome samples.
M, male; F, female; M_dis, male samples showing methylation patterns of females; F_dis, female samples showing methylation patterns of males; T, Turner syndrome sample; KS, Klinefelter syndrome sample; KS_sus, Klinefelter syndrome-suspected sample.
Work flow of the Affymetrix Human Genome U133Plus 2.0 array analysis.
Patterns of gender-specific gene expression of X- and Y-specific genes.
X (A) and Y (B) chromosome markers.
The red circles indicate the discordant male samples showing overlapping distribution of levels for normal male samples.
M, male; F, female; M_dis, male samples showing methylation patterns of females; F_dis, female samples showing methylation patterns of males.
Title: Collected Infinium 450K HumanMethylation array samples and information on discordant samples

Title: Collected Affymetrix U133Plus2 array datasets and information on discordant samples
Title: Editor&#39;s Introduction to This Issue

Next generation sequencing (NGS) has been facilitating the discovery and functional validation of causal variants in diverse rare diseases.
However, for most of the 7,000 rare diseases, pathogenic mechanisms are still unknown.
In this issue, Dr. Murim Choi&#39;s group (Seoul National University, Korea) comprehensively reviewed ultra-rare diseases and genomics-driven precision medicine.
They presented the perspectives gained from recent studies on ultra-rare diseases which were diagnosed by NGS analysis and successfully treated.
They also discussed the potential obstacles to wider application of NGS-based precision medicine.
Dr. Young-Joon Kim&#39;s group (Yonsei University, Korea) reported classification of colorectal cancer patients based on promoter methylation patterns.
They used colon cancer data sets from the Cancer Genome Atlas to observe CpG islands that showed significant aberrations in DNA methylation and also analyzed changes in DNA methylation patterns to assess the relationships between epigenetic changes and tumorigenesis.
Dr. Jong Il Kim&#39;s group (Seoul National University, Korea) reported the genome-wide association study which evaluated the effects of genetic polymorphisms on the bone mineral density at the lumbar spine and femur in Korean men.
Investigating the etiology of osteoporosis in males of East Asian descent can have high diagnostic and clinical implications.
Dr. Thirumudi&#39;s group (Kamalnayan Bajaj Institute for Research in Vision and Ophthalmology, India) reported two potential inhibitors targeting the hydrophobic cleft of Toxoplasma gondii apical membrane antigen 1 (AMA1).
For further details, please visit G&amp;I homepage (http://www.kogo.or.kr/webapp/kogo_publish/genomics_and_inf ormatics/).
Title: Ultra-rare Disease and Genomics-Driven Precision Medicine

Since next-generation sequencing (NGS) technique was adopted into clinical practices, revolutionary advances in diagnosing rare genetic diseases have been achieved through translating genomic medicine into precision or personalized management.
Indeed, several successful cases of molecular diagnosis and treatment with personalized or targeted therapies of rare genetic diseases have been reported.
Still, there are several obstacles to be overcome for wider application of NGS-based precision medicine, including high sequencing cost, incomplete variant sensitivity and accuracy, practical complexities, and a shortage of available treatment options.
Next-generation sequencing (NGS) has achieved great advances in medicine and clinical practice, as well as in basic research field.
NGS-based precision medicine has mainly focused on cancer [12], given the prevalence, availability of drugs targeting major oncogenic factors, and impact on human health.
Another sector of focus through NGS-based precision medicine is ultra-rare diseases.
Ultra-rare diseases can be defined as disorders that occur with a prevalence of below 1 in 2,000,000, although this frequency cutoff can be flexible [3].
Usually, rare disease research is performed with large families for traits with dominant inheritance [45] or patient-parent trios for spontaneous cases [6].
Discovery of causative genetic variants using whole exome sequencing (WES) or whole genome sequencing (WGS) should be followed by in vitro or in vivo model functional studies to fully explain the molecular basis of the diseases and confirm the functionality of the variants [4567].
It is presumed that a large portion of ultra-rare diseases are caused by de novo mutations occurring at functionally important protein coding regions (Fig.1) [8 9].
Every human zygote is expected to have approximately 70–80 de novo mutations in its genome, and this poses a small chance to have ultra-rare diseases, depending on the position where the de novo mutations lie [810].
Since NGS was adopted into medical genetics, the number of rare disease-causing gene discoveries has been increasing rapidly, and the genetics of many idiopathic diseases have been elucidated [111213].
In this review, we present the perspectives gained from recent studies on ultra-rare diseases diagnosed by NGS and successful treatments and discuss the potential obstacles for wider application of NGS-based precision medicine.
Although cystic fibrosis is not an ultra-rare disease by definition, it presents a good example of precision medicine.
Typically inherited in an autosomal recessive manner, patients carry two non-functional alleles of cystic fibrosis transmembrane conductance regulator (CFTR).
Recently, an NGS-based diagnostic assay to detect mutations in CFTR was authorized for marketing by the United States Food and Drug Administration (FDA) [14].
In addition, ivacaftor (Kalydeco, Vertex Pharmaceuticals, Cambridge, MA, USA), a potentiator of CFTR, and lumacaftor (Orkambi, Vertex Pharmaceuticals; combination drug with ivacaftor), a corrector of CFTR, were approved by the FDA for the treatment of cystic fibrosis patients with nonsynonymous mutations (http://www.kalydeco.com) and the F508del mutation (http://www.orkambi.com), respectively [15].
NGS-based assays have made it possible to detect certain mutations that can be successfully treated by mutation-specific drugs (Table 1) [16].
The famous case of Nick Volker exemplifies the beneficial effect of NGS techniques on patients with atypical and undiagnosed disorders [17].
The cause of his fatal bowel inflammation and colocutaneous fistula turned out to be a mutation in the X-linked inhibitor of apoptosis protein (XIAP) gene by WES, which has important roles in inflammatory signaling and immunity.
His disease was almost cured by cord blood transplantation.
Following this, a few more cases of precision medicine supported by NGS have been reported.
WGS of 14-year-old twins with dopa (3,4-dihydroxyphenylalanine)-responsive dystonia revealed compound heterozygous mutations in the sepiapterin reductase (SPR) gene, encoding sepiapterin reductase.
Disruption of the enzyme caused a decrease in the neurotransmitters dopamine and serotonin, and administration of L-dopa with 5-hydroxytryptophan, a serotonin precursor, led to clinical improvements in both twins [18].
Another case was a cancer: the serine/threonine protein kinase B-raf (BRAF) V600E mutation in metastatic rhabdoid meningioma was found by targeted NGS in a 6-year-old girl [19].
She was successfully treated with dabrafenib, a BRAF inhibitor.
Most recently, a more dramatic case was of a 14-year-old girl who had suffered from autoimmune enteropathy since she was 3 months old [6].
A novel de novo mutation in the ligand binding domain of cytotoxic T lymphocyte antigen 4 (CTLA-4) was discovered by WES (Fig.2) [62021].
The mutation was presumed to be causative after several in vitro immunological experiments.
Abatacept, an FDA-approved fusion protein of the extracellular domain of CTLA-4 and the Fc region of immunoglobulin for rheumatoid arthritis, was administered to the patient and restored regulatory T cell function.
Clinically, treatment with abatacept without the need for other immunosuppressants reduced diarrhea output by about 67% (from 3 L/day to 1 L/day).
Markedly, blood transfusion was not necessary anymore, allowing the patient to be discharged from a 3-year-long hospitalization, merely needing partial parenteral nutrition.
Prenatal screening and prediction of ultra-rare disease with NGS are not widely used yet due to their cost and efficacy.
But, if the parents of a child with an ultra-rare disease who was diagnosed by NGS plan to have another child, prenatal testing with DNA extracted from chorionic villus sampling or amniocentesis is possible.
If the cost of NGS declines, screening of rare genetic diseases with NGS might be routinely performed.
At this moment, a non-invasive prenatal test using NGS is rapidly being introduced to the clinical field, which allows the detection of fetal aneuploidies, such as trisomy 21 (Down syndrome) [22].
This technique uses cell-free fetal DNA (cffDNA) present in maternal circulation and enables non-invasive liquid biopsy, in contrast to invasive procedures, including amniocentesis and chorionic villus sampling.
Although it is technically challenging to detect small nucleotide-level mutations from cffDNA at the present time, extensive research in this field has yielded potential in tackling the problem [23].
Despite these encouraging cases, there are wide gaps between the diagnosis and treatment of ultra-rare diseases.
Targeted and tailored therapies for ultra-rare diseases are usually hard, mainly because of scarce treatment options.
To fill these gaps, several conditions should be fulfilled.
First, the cost of NGS should decrease further.
As of this writing, the cost of WES has fallen to under USD $1,000, which is still a formidable price for a routine application in clinics, while the usage of WGS may still be impractical.
Second, the sensitivity and specificity of genetic diagnostics should be improved.
Current commercially available exome capture kits covers ~99% of the RefSeq exome, and over 95% of the targeted bases are covered at least eight times with a typical WES run [24], implying that we are achieving fair exome coverage technically.
However, the positive diagnostic rate of clinical WES for Mendelian phenotypes remains ~30% [25], suggesting that a substantial portion of the remainder (~70%) might be caused by variants located outside of exons or are not detectable by WES.
Although the majority of functionally critical and disease-causing mutations occurs in protein-coding regions, the vast majority of the genome is non-coding and may contain variants with functional significance that has been neglected.
Third, processes of NGS data generation should be more simplified.
Currently, the typical NGS pipeline consists of library preparation, sequencing, and multiple analysis steps, including read alignments, variant calling, and annotation.
For NGS to pervade in clinical practice, the processes from DNA samples to the final results should be more concise.
In this context, nanopore sequencing techniques, such as MinION, could function as a good alternative to current NGS technologies if they improve the high error rates and relatively small-scale data output [26].
Fourth, more treatment options, such as targeted drugs and gene therapy strategies, should be provided.
The metastatic rhabdoid meningioma with a BRAF mutation and CTLA-4 cases mentioned above are rather exceptional cases, since the genetic alterations were already targeted through commercially available drugs.
Nevertheless, the CTLA-4 case is a good example of drug repositioning, which can be an efficient option to accelerate treatments for ultra-rare diseases, given its shorter development time and lower cost [27].
Although they are given less clinical attention and researched less often, ultra-rare diseases have been occurring constantly, because every new baby will have de novo mutations in its genome, posing a small chance of the mutations causing an unseen phenotype.
Thus, it would be important to improve prompt and accurate diagnoses and suggest appropriate treatment options for ultra-rare disease patients.
Although we still face many technical difficulties to address—such as the high price of NGS, imperfect NGS technology, practical complexities, and a shortage of treatment options—the several cases illustrated here demonstrate successful application of NGS-based precision medicine for ultra-rare diseases, benefiting patient care in the long term.
In the near future, NGS will be an essential component in precision medicine regarding its high throughput and relatively low and further decreasing cost and the importance of individual genomic information for personalized strategies.
Mode of inheritance of rare genetic diseases that were definitively diagnosed in a cohort of developmental disorders.
Drawn with data adopted from Deciphering Developmental Disorders Study.
Nature 2015;519:223-228 [9].
De novo mutation in ligand binding motif of cytotoxic T lymphocyte antigen 4 (CTLA-4).
Modified from Lee et al.
J Allergy Clin Immunol 2016;137:327-330 [6].
(A) Proband-specific de novo mutation in CTLA-4 shown in the pedigree.
(B) Crystal structure of CTLA-4 homodimer (right) (Protein Data Bank [PDB] ID: 3OSK) [20] and close-up view of interaction between CTLA-4 and CD80 (left) (PDB ID: 1I8L) [21].
(C) Models of mutant CTLA-4 failing to bind to CD80/CD86 and regulate immune reactions and rescued by CTLA-4-Ig (abatacept).
APC, antigen-presenting cell; Th cell, T helper cell.
Title: Classes of CFTR mutations
Title: Classification of Colon Cancer Patients Based on the Methylation Patterns of Promoters

Diverse somatic mutations have been reported to serve as cancer drivers.
Recently, it has also been reported that epigenetic regulation is closely related to cancer development.
However, the effect of epigenetic changes on cancer is still elusive.
In this study, we analyzed DNA methylation data on colon cancer taken from The Caner Genome Atlas.
We found that several promoters were significantly hypermethylated in colon cancer patients.
Through clustering analysis of differentially methylated DNA regions, we were able to define subgroups of patients and observed clinical features associated with each subgroup.
In addition, we analyzed the functional ontology of aberrantly methylated genes and identified the G-protein-coupled receptor signaling pathway as one of the major pathways affected epigenetically.
In conclusion, our analysis shows the possibility of characterizing the clinical features of colon cancer subgroups based on DNA methylation patterns and provides lists of important genes and pathways possibly involved in colon cancer development.
Generally, it is known that cancer is a result of somatic mutations in DNA.
These mutations are located in genes that have important roles in regulating cell growth, cell differentiation, and DNA damage control [12].
Over the past decades, many cancer driver genes have been found by high-throughput sequencing technology, and thus, the number of cancer driver genes may have reached the limit [13].
Until now, many researchers have studied the mechanism of carcinogenesis and highlighted the biological role of driver mutations, such as TP53, PIK3CA, and KRAS [4].
However, in many types of cancers, the etiology of cancer cannot be explained only by DNA mutations.
Researchers have found that epigenetic factors, such as DNA methylation and histone modification, also contribute to cancer formation and development [5].
Epigenetic factors are dynamic modifications that can change the state of gene expression or regulate expression rates.
Some studies have shown that a large group of cancer patients have both globally low and high levels of DNA methylation (hypomethylation and hypermethylation, respectively) in specific promoter regions [6].
Based on analysis of DNA methylation data, they listed a few cancer-related genes that carry significant methylation changes as biomarkers [7].
However, the biological meaning of these markers is still not well known.
Hence, in this study, we used colon cancer (COAD) datasets taken from The Caner Genome Atlas (TCGA) to observe a CG dense region called CpG islands (CGIs) that showed significant aberrations in DNA methylation and also analyzed changes in DNA methylation patterns to further understand the relationship between epigenetic changes and cancer mechanism.
Title: TCGA COAD DNA methylation datasets and expression datasets

Both methylation and gene expression data were obtained from the TCGA Data Portal (https://tcga-data.nci.nih.gov/tcga/).
We collected COAD Level 3 (pre-processed) JHU-USC HumanMethylation450 data for methylation and UNC illuminaHiSeq_RNASeqV2 data for gene expression.
We neglected a normalization step for both datasets, since they were pre-processed and normalized by uploaded groups.
We matched methylation and gene expression data by patient header ID using the TCGA barcode.
Beta-value, a value of the ratio of the methylated probe intensity and the overall intensity, was used to represent the methylation percentage.
Gene expression fold-change was calculated by taking scaled estimate values, multiplying by 106 (transcripts per million, TPM), adding 1 to each normal and tumor TPM, and then taking the log2 value of tumor and normal per gene.
Title: Differential methylation and expression analysis and clustering

To get differential DNA methylation values between normal tissue and tumor, we averaged all normal samples using annotated probes.
For CGI analysis purposes, we intersected each beta-value for a total of 485,579 probes from the methylation data to the CGI location (provided by University of California Santa Cruz [UCSC]), averaged them using CGIs, and then subtracted the averaged DNA methylation value of normal samples from individual tumor samples.
To focus on the effect of promoter CGIs, we selected CGIs that fell only into our defined promoter region, which covers the transcription start site ± 1 kb.
Using this boundary, a total of 15,966 promoter CGIs were counted.
The methylation distribution pattern of promoter CGIs was plotted by taking the mean promoter CGI methylation from the entire tumor and normal sample datasets.
To get differentially methylated promoter CGIs, the averaged normal data were used as a reference, since there were no significant variations among normal samples.
Differential patient data were calculated by subtracting this reference from each patient methylation data point (n = 297).
In order to define the differential methylation cutoff, we referred to the methylation distribution pattern between normal samples and tumors.
Methylated CGI annotated genes varied in their methylation percentage throughout the patients.
Therefore, differentially methylated CGIs were identified as absolute difference of 0.3 in beta-values in at least 30% of total patients to obtain a broader range for gene selection (Fig.1).
We grouped COAD patients by clustering their CGI differential methylation values using Cluster3.0 (http://bonsai.hgc.jp/~mdehoon/software/cluster/software.htm).
We specifically used hierarchical clustering using the Euclidean distance similarity metric and the complete linkage clustering method, which grouped the patients the best.
To reveal the direct methylation effect on gene expression, we selected an expression dataset that only matched with the methylation dataset, as well as a normal sample-tumor paired dataset (n = 26).
We then aligned 26 paired patient data to the each group, divided by the CGI methylation clustering value.
Expression level of patients in each group were averaged by genes and then plotted with mean values and 95% confidence levels.
Title: Gene ontology analysis and pathway analysis

To gain a biological understanding from the selected genes, we carried out gene ontology analysis and pathway analysis using InnateDB [8].
A hypergeometric algorithm was selected, and Benjamini-Hochberg was used for the correction method.
All three ontology results, including molecular function, cellular components, and biological process, were considered.
Pathway results were also sourced from various databases, including Integrating Network Objects with Hierarchies (INOH), Reactome, Kyoto Encyclopedia of Genes and Genomes (KEGG), Pathway Interaction Database (PID) NIC, and PID BioCarta.
Significant Gene Ontology (GO) terms and pathways were selected based on p-value selection (p &lt; 0.05).
We combined functionally redundant GO terms, because genes that contained such ontologies were nearly identical.
Pathways were sorted and grouped by their functional similarity of each pathway.
From the many selected pathways, we focused on pathways that were previously found to be involved in tumorigenesis.
G protein-coupled receptor (GPCR) signaling pathway-related genes were gathered from the InnateDB and KEGG pathways, and we indicated the hypermethylated genes involved in GPCR-related signaling.
Title: Cluster pattern of DNA methylation of promoter CGI and COAD patients

From the TCGA data portal, we collected array-based DNA methylation data of 279 patients.
We then derived differential DNA methylation values between tumor and averaged normal samples.
Differential methylation values of single-base probes were averaged by individual CGIs, and the data were filtered to indicate CGIs with significant changes in methylation as described in the &#34;Methods&#34; section (Fig.1).
We also checked the distribution of DNA methylation within promoter CGIs between normal and tumor samples by taking the mean of each condition per CGI.
We observed that in general, promoter CGIs were hypermethylated in COAD patients in comparison to those in normal samples (Fig.2A).
We went on to group COAD patients based on their differential methylation status in promoter CGIs using clustering analysis.
Since it is rare to see promoter CGIs being differentially methylated across an entire group of patients, clustering was focused on selecting CGIs, as well as identifying patient groups, based on their differential methylation patterns.
From our results, we observed a greater number of hypermethylated CGIs than hypomethylated CGIs (Fig.2B).
As indicated by previous studies, COAD follows a common methylation pattern in cancer, which is hypermethylation of promoter CGIs [9].
We were also able to distinguish three distinctive clustered groups by their differential methylation patterns.
Group 2 (n = 76) showed a much higher level of methylation within selected CGIs than groups 1 and 3.
Group 1 showed intermediate differential methylation.
To find the meaning of each classified patient group, we investigated their respective clinical data (Table 1, Supplementary Table 1).
When we analyzed the groups by clinical category, such as race, age, gender, and tumor stage, we could not find any discriminating factor among the three groups.
However, we observed a distinct rate of metastasis in group 2, which showed the highest level of hypermethylation.
At the same time, group 2 showed a higher rate of microsatellite instability (MSI).
MSI is a hypermutable phenotype caused by impaired DNA mismatch repair.
It is already known that MSI is associated with hypermethylation in the promoter region of the MLH1 gene [10].
Although the MLH1 gene was not detected in our filtered data, we surmised that the MSI phenotype in COAD patients can be affected by both DNA hypermethylation and MLH1 gene activity itself.
Interestingly, the tumor sites differed among the three groups.
In group 2, the cecum was the most frequent tumor site, while the sigmoid colon was the least frequent site in comparison with groups 1 and 3.
These results provide a few insights about epigenetic mechanisms in COAD.
First, the varying methylation patterns across COAD patient groups imply distinctive epigenetic cancer mechanisms pertaining to different patient groups.
This could lead to subtyping COAD with varying promoter CGI methylation status, which can be utilized for treatment in patients.
Second, hypermethylated promoter CGIs were enriched in differentially methylated region, which implies that the mechanism for COAD in promoter CGIs might be driven predominantly by methylating factors, such as DNA methyltransferases.
Title: Downregulated gene expression in hypermethylated promoter CGI group

To see whether DNA methylation directly affects gene expression within selected CGIs, we annotated genes to promoter CGIs.
For the sake of accuracy, we selected only gene expression datasets that matched the methylation data.
We further selected datasets that had both normal and tumor samples from identical patients (n = 26).
Since we expected different gene expression levels within the divided groups, we averaged the expression level of patients by each group.
We then compared the mean values of promoter-overlapping genes between the groups to see the general expression level (Fig.3).
It is well known that the effect of promoter CGI methylation is repression of gene expression.
We observed that group 2 experienced the highest repression of gene expression levels, thus reflecting the role of promoter CGI methylation as a gene-repressive marker.
In comparison, we noted the highest overall gene expression levels in group 3.
COAD patients in less hypermethylated groups, however, were less affected by hypermethylation in their promoter region.
We noted that there were some patients in all groups whose gene expression was not affected by methylation.
We assumed that there were not only epigenetic factors but many other varying factors among cancer patients that could result in regulation at individual genes.
This DNA methylation change in promoter CGIs, which alters gene expression, is called epi-mutation [11].
Using this analogy, some patients were epi-mutated in aberrantly methylated promoter CGIs, while others were not affected significantly.
In general, we were able to see that promoter CGI hypermethylation is mostly linked to the overall repression of gene expression, confirming the epi-mutation effect.
Title: Epi-mutated genes and pathways in COAD

To see the biological function of hypermethylated promoter CGI genes, we performed gene ontology analysis [8].
Gene ontology analysis provides a functional interpretation of biological process, molecular function, and cellular component among selected genes.
The ontology results included mainly neural development-related terms, such as synaptic transmission and nervous system development (Table 2).
Interestingly, the gene ontology term of sequence-specific DNA-binding transcription factor activity was found.
This implies that a large group of downstream genes that are targeted by the affected transcription factors could have a potential role in cancer initiation and development.
Additionally, we carried out pathway analysis to further understand cancer-related biological processes affected by epi-mutated genes (Supplementary Table 2).
In fact, many recent cancer-related pathways are found with mutations among patients, and this information is useful for functional studies of cancer mechanisms [2].
We attempted to provide an extended network that involves hypermethylated promoter CGI genes to expand our understanding of epi-mutational pathways in cancer.
Selected pathways with p-value &lt;0.05 include Wnt signaling, the RAS pathway, migration and invasion, extracellular matrix organization, and cell adhesion.
These pathways are commonly related to cancer proliferation and metastasis [12].
In addition to the pathways previously mentioned, we found the following pathways to be distinctive to only hypermethylated promoter CGI genes in COAD: calcium signaling, ion channel, and GPCR signaling; especially interesting is the observation of hypermethylation in many ligands and receptors that are involved in GPCR signaling (Fig.4) [1314151617181920].
This implies that epigenetic mechanisms play a role upstream of the pathway, regulating many downstream signaling pathways that are involved in cancer growth.
Epi-mutated pathway analysis thus allows us to find additional cancer-related pathways that have not been previously highlighted by analyses based on somatic mutations.
Overall, these results suggest the function of hypermethylation in promoter CGI genes in colon cancer as a driver of transcriptional regulation and developmental events and indicate that epi-mutated genes are involved in regulating various cancer-related pathways, like GPCR signaling, in favor of colon cancer development.
In contrast to somatic mutations, epigenetic changes are reversible phenomena.
Using this property, epigenetic factors can be therapeutic targets of cancer treatments.
Until now, DNA methylation inhibitors, such as azacitidine, have been used as one of many cancer drugs.
However, this kind of drug is not able to specify the target molecule and shows differential drug effects in individual cancer patients.
In this paper, we observed impaired DNA methylation in colon cancer patients—so-called epi-mutation [21].
We can confirm that DNA methylation aberrations in specific promoter regions were widely distributed in cancer patients and that patient groups can be divided by the extent of DNA methylation change.
We expect that by introducing the epi-mutation concept, patients of a certain cancer type that is not explained by somatic mutations can be diagnosed more sensitively.
Moreover, the subgroups of patients will provide a clue for the different drug effects between individual cancer patients.
Further epi-mutation studies in each cancer type will define cancer-specific related biological pathways, and overall, these results will help us understand the cancer mechanisms and develop target-specific cancer drugs.
Workflow of the COAD data analysis.
CGI, CpG island; COAD, colon cancer; TCGA, The Caner Genome Atlas.
(A) Differential DNA methylation pattern in promoter CpG islands (CGIs) between colon cancer (COAD) patients and normal.
A distribution plot of mean beta-value between all tumor (red) and all normal (blue) samples.
X-axis indicates beta-value ranging from 0 to 1.
Y-axis indicates the density of accumulated beta-values.
(B) Differential DNA methylation pattern in promoter CGIs between COAD patients and normal.
A heatmap of clustered differentially methylated promoter CGIs.
We defined promoter CGI that has ±0.3 differential beta-value in at least 30% patients as differentially methylated promoter CGIs.
X-axis represents individual promoter CGIs (1,341 CGIs), and Y-axis represents each patient.
(n = 297).
Patients are divided based on hierarchical clustering using Euclidian distance similarity metric.
Scale bar ranging from -0.5 to +0.5 tumor-normal beta-value.
Expression comparison among the groups of clustered hypermethylated promoter CpG islands (CGIs).
Only the normal-tumor paired, and methylation-expression matched patients were selected for comparison of gene expression (n = 26).
Patients were divided into the previously clustered groups by promoter CGI methylation.
Expression fold change (Y-axis) was calculated by log2(T/N).
Genes that overlap with hypermethylated promoter CGI were averaged by each patient and each group.
Each bar represents standard error mean of each patient.
Right panel is DNA methylation changing pattern of corresponding individual patients.
T, tomor; N, normal.
Representation of G protein-coupled receptor related signaling pathway affected by the hypermethylated promoter CpG island (CGI) genes.
Hypermethylated promoter CGI genes are marked as red.
Title: Clinical data of three patient groups in colon cancer

Title: Ontology analysis results from hypermethylated pCGI genes
Title: Multilevel Precision-Based Rational Design of Chemical Inhibitors Targeting the Hydrophobic Cleft of Toxoplasma gondii Apical Membrane Antigen 1 (AMA1)

Toxoplasma gondii is an intracellular Apicomplexan parasite and a causative agent of toxoplasmosis in human.
It causes encephalitis, uveitis, chorioretinitis, and congenital infection.
T. gondii invades the host cell by forming a moving junction (MJ) complex.
This complex formation is initiated by intermolecular interactions between the two secretory parasitic proteins—namely, apical membrane antigen 1 (AMA1) and rhoptry neck protein 2 (RON2) and is critically essential for the host invasion process.
By this study, we propose two potential leads, NSC95522 and NSC179676 that can efficiently target the AMA1 hydrophobic cleft, which is a hotspot for targeting MJ complex formation.
The proposed leads are the result of an exhaustive conformational search-based virtual screen with multilevel precision scoring of the docking affinities.
These two compounds surpassed all the precision levels of docking and also the stringent post docking and cumulative molecular dynamics evaluations.
Moreover, the backbone flexibility of hotspot residues in the hydrophobic cleft, which has been previously reported to be essential for accommodative binding of RON2 to AMA1, was also highly perturbed by these compounds.
Furthermore, binding free energy calculations of these two compounds also revealed a significant affinity to AMA1.
Machine learning approaches also predicted these two compounds to possess more relevant activities.
Hence, these two leads, NSC95522 and NSC179676, may prove to be potential inhibitors targeting AMA1-RON2 complex formation towards combating toxoplasmosis.
Toxoplasma gondii is an Apicomplexan intracellular parasite and is a causative agent of toxoplasmosis in human.
It also plays a key role over a broad spectrum of clinical syndromes, like encephalitis, uveitis, chorioretinitis, and congenital infections [1].
It infects the host by various modes, such as intake of undercooked meat containing T. gondii cysts, consumption of contaminated water, and food defilement with feces from infected cats, and may also spread through blood transfusion [23].
The sexual reproduction of T. gondii occurs in domestic cats, which are considered a definite host [4].
The asexual reproduction of this parasite occurs in intermediate hosts—namely, humans, cattle, and birds [5].
The establishment of host-parasite interactions is crucial for parasite survival, as it depends on host nutritive resources.
Apicomplexan parasites implement a unique host cell invasion mechanism, wherein they anchor the host cell by forming multimeric protein machinery, called a moving junction (MJ) complex [6].
This complex is formed by interprotein interactions established between two types of proteins-namely, rhoptry neck proteins 2/4/5/8 (RON 2, 4, 5, and 8) proteins secreted by rhoptry organelle of the parasite onto the host membrane and apical membrane antigen 1 (AMA1), secreted by micronemes [789].
The molecular association of RON2 and AMA1 is essential for initiating the host invasion process.
Title: AMA1 hydrophobic cleft

The hydrophobic cleft region of AMA1 is documented to play a predominant role in facilitating the binding of RON2 to AMA1 and is also reinforced by recent crystallographic studies on AMA1-RON2 complex formation [10].
Moreover, this region is found to be highly conserved across the diverse family of AMA1 proteins and is surrounded by polymorphic flexible loops.
These loops are speculated to preclude the host antibody response and has also been found to guard the conserved sites in AMA1 [11].
The hydrophobic cleft is found to span within domain I of AMA1 and forms a binding pocket that receives the critical loop region of RON2 due to its accommodative shape and charge complementarity.
The hydrophobic cleft region of AMA1 includes 15 residues (Val142, Leu155, Ile161, Phe163, Ile171, Phe174, Leu179, Ile185, Phe197, Met203, Tyr230, Val231, Trp253, Trp353, and Trp354) [12].
Recent studies illustrate that RON2 displaces a loop in domain II of AMA1, thereby exposing its binding surface, enabling the RON2 loop to penetrate deep into the hydrophobic groove of AMA1 and hence forming a stable complex during host invasion [13].
A recent study also showed that an oligopeptide binds across the full length of the AMA1 hydrophobic cleft to prevent AMA1-RON2 complex formation and eventually blocks parasitic host invasion [1415].
Hence, the hydrophobic cleft and its surrounding loops, which include the cysteine loop region and coil connector, have been proposed to be a critical hotspot to block AMA1-RON2 complex formation.
To date, there is a paucity of data on small molecules targeting these interactions.
Hence, in this study, the coordinates of the crystal structure complex depicting these interactions (Protein Data Bank [PDB] ID 2Y8T, AMA1 interacting with a surface-exposed region of RON2) [16] were subjected to an exhaustive, multilevel precision high-throughput virtual screen (HTVS) of ligands from the National Cancer Institute (NCI) database with 250,000 compounds to identify leads that potentially disrupt these interactions.
We also implemented molecular dynamics (MD) simulation analysis to decipher the backbone rigidity changes of AMA1 hotspot residues, machine learning-based activity prediction, and binding free energy calculations for finalizing the potential lead compounds towards combating toxoplasmosis (Fig.1) [17].
Title: Protein preparation

As an initial step, the crystal coordinates of AMA1 in complex with a surface-exposed region of RON2 (PDB ID: 2Y8T) was pre-processed using Protein Preparation Wizard of the Schrödinger (New York, NY, USA) suite towards optimizing the stereochemistry by assigning proper bond order, removing steric clashes, adding hydrogen atoms, fixing the disulfide bonds and missing residues and loops.
Furthermore, optimization of the protein structure was also performed by adjusting the terminal chi rotation of asparagine, glutamine, and histidine residues.
Optimal protonation states for histidine residues were also assigned, followed by the removal of unwanted hetero groups.
Finally, energy minimization was performed with OPLS_2005 (Schrödinger) to obtain the optimal geometry.
Title: Receptor grid generation for exhaustive search

The receptor grid for docking was generated by assigning the entire hydrophobic cleft region of AMA1 to which the RON2 binds, thereby performing an exhaustive cavity search towards achieving an optimal ligand binding pose.
The residues spanning the RON2 binding cavity were noted from the Dimplot generated for the AMA1-RON2 complex [16].
The van der Waals radii for the receptor were assigned with a scaling factor of 1.0 and a partial cutoff of 0.25.
The grid-assigned residues were Leu99, Val105, Leu109, Tyr110, Arg111, Val142, Phe143, Thr144, Glu145, Leu155, Asn162, Thr165, Gln169, Arg170, Ile171, Asn182, Asn184, Leu200, Thr201, Val202, Ala203, Met204, Tyr213, Tyr215, Tyr230, Ser232, Met233, Met236, Tyr241, Thr252, Trp253, Gln338, Phe339, and Gln361.
Title: Ligand preparation

To expedite the virtual screening of ligands, the NCI ligand datasets (250,000 compounds) were screened for drug-likeness by implementing the Lipinski rule.
The filtered molecules were optimized by the LigPrep module (Schrödinger) by fixing the ring isomers, stereoisomers, and tautomeric forms.
The reactive functional groups were also removed, and the resultants were scaled for van der Waals radii with a scaling factor of 0.80 and a partial charge cutoff of 0.15.
Finally, the compounds passing all these filters were energy-minimized with OPLS_2005 as the force field.
Title: Multilevel precision-based virtual screening

The in silico virtual screening and docking of optimized NCI compounds against AMA1 were performed using the Glide HTVS option of the Schrodinger suite 2012 (Schrödinger, LLC).
As a first step, the entire RON2-interacting cavity was fixed as a grid box.
The van der Waals radius scaling was set to 1.0 to allow free scaling to soften the non-polar regions of the receptor and the rest of the other atoms.
Finally, the optimized small molecules were successively docked to AMA1, ensuring flexible sampling with no more than 300 atoms and 50 rotatable bonds.
A total of 10 energetically favorable conformations were selected among the 1,000 generated poses per docking, and the best docked complexes were finalized, based on the Glide docking score.
During the screening process, successive elimination of ligand hits was performed through three filtering modes using the Schrödinger suite: 100% of HTVS hits were passed to standard precision (SP), and 80% of the best hits from SP were passed to extra precision (XP) mode.
The top 10 hits in XP, based on Glide score, were shortlisted for further validation [18].
Title: Prediction of Activity Spectra of Substances prediction

The putative activities of the top 10 compounds were predicted using Prediction of Activity Spectra of Substances (PASS).
It implements a machine learning-based algorithm called biological activity spectrum (with 95% accuracy), and it works by describing the intrinsic properties of the compounds and by comparing the structure of the new compound with the structure of well-known biologically active substances to check whether the new compound exhibits any of the specific functional effects [19].
Among the 10 shortlisted compounds, four showed significant activity relevant towards suppressing toxoplasmosis, along with a favorable Glide docking score.
Hence, these four compounds were subjected to a rigorous validation process of calculating Molecular Mechanics Generalized Born Surface Area (MMGBSA) score and by performing an MD simulation.
Title: MMGBSA scoring

MMGBSA scoring was performed for the top 4 complexes, so as to calculate and identify binding free energies in an explicit-solvent exposed state by implementing the Prime module of the Schrodinger suite.
A more negative MMBGSA score indicates stronger binding [20].
Title: MD simulation

MD simulation for both apo and the docked complexes was performed using Desmond 3.6.
The simulation was initiated using the OPLS_2005 [21] force field with the system solvated using the simple point charge water model.
The system was neutralized by adding 4 Na+ for apo and 6 Na+ counter ions for complexes at a concentration of 4.751 mM.
Further, this system was energy-minimized with OPLS_2005.
The SHAKE algorithm was applied to restrain the geometry of water molecules and bond lengths and bond angles of heavy atoms [22].
Periodic boundary conditions were applied to stimulate a continuous system [23] and the particle mesh Ewald method for long-range electrostatics [24].
Further, the system was equilibrated with NPT ensemble by setting the temperature and pressure parameters to 300 K and 1.0 bar, respectively.
The Berendsen coupling algorithm was chosen for temperature-pressure coupling [25].
Further, the equilibrated system, with a total of 74,931 atoms, was exposed to a simulation period of 5,000 ps with a time step of 2 fs, and trajectories were recorded after every 1.0 ps.
The root mean square deviation (RMSD) was calculated for the backbone atoms and were graphically analyzed on a time point scale [2627].
Similarly, root mean square fluctuation (RMSF) for each residue was also calculated to compare the major conformational changes in the residues between apo and holo forms [28].
The radius of gyration (Rg) was also calculated to infer the compactness of the protein-ligand complex against the apo form [29].
Two-dimensional intermolecular interaction plots depicting the complex stability throughout the MD run were also generated to infer the stability of the protein-ligand complex formation.
Title: Multilevel precision virtual screening and docking analysis

Three different stages of the docking and scoring processes were used for this study, beginning with HTVS, followed by SP and final scoring with XP.
AMA1 protein was sequentially docked with compounds from the NCI database using Schrodinger Glide.
Initially, HTVS was performed, wherein 146,670 compounds were scored as hits, based on Glide score.
Subsequently, these compounds were passed to SP mode, which yielded 1,466 compounds.
Finally, within the SP resultant hits, 147 compounds were scored as potential hits in XP mode.
Further, these compounds were ranked in accordance with the XP docking score, and the top 10 ranking compounds proceeded to further analysis.
These top 10 compounds were filtered further, based on Glide docking score, significant intermolecular interactions, and PASS prediction.
Interestingly, all 10 of these compounds were found to occupy the hotspot region, which includes the hydrophobic cleft and cysteine loop region of AMA1, despite reference-guided docking procedures (Table 1).
Title: Receptor-ligand interaction studies

Scoring functions play a major role in the docking process for identifying plausible ligand binding poses and also for ranking the binding affinity.
Among the top 10 compounds, NSC95522, NSC88253, NSC13604, and NSC179676 (Table 1, Supplementary Fig.1) were found to be highly significant in collective terms of XP Glide score, PASS prediction, formation of intermolecular H-bonds, and π-cation interactions with the critical hydrophobic cleft residues, as well as in the cysteine loop region.
NSC95522 showed a Glide score of -10.564 kcal/mol and formed π-cation interactions with Tyr110, Tyr213, and Tyr215, stabilized by 3 hydrogen bonds formed with Tyr230, Ser232, and Tyr215 (Supplementary Fig.1A).
Tyr230 is a vital residue in the hydrophobic cleft, as it is the central tyrosine of the apical groove and is mandatory for RON2 binding and also in the formation of a contiguous surface, which is critical for MJ assembly [30].
Ser232 is also documented in Plasmodium falciparum as a key hotspot residue, as it occupies the hydrophobic cleft-spanning region [31].
Tyr213 and Tyr215 are also crucial, as these residues are key facilitators of conformational changes in AMA1 DII loop displacement towards exposing a clear pocket to which Pro1309 of RON2 docks to the triad of tyrosine residues (Tyr110/142, Tyr213/234, and Tyr215/236) during AMA1-RON2 interactions [32].
PASS predicted this compound to be an ophthalmic drug and thus might also be found to be effective in treating adverse ocular-related effects due to toxoplasmosis.
NSC88253 was bound to the hydrophobic cleft of AMA1 with a XP Glide score of -10.392 kcal/mol, forming almost similar interactions to that of the AMA1-NSC95522 complex with 3 hydrogen bonds: 2 with the backbone of Tyr230 and Ser232 and 1 with the side chain of Tyr215.
The π-cation interactions with Tyr110 and Tyr213 were also synonymously stabilized by hydrogen bonds formed by residues Tyr230, Tyr215, and Ser232 (Supplementary Fig.1B).
Tyr110 is a coil connector and is documented to play an important role in the displacement of the DII loop of AMA1, which helps in the binding of RON2 [32].
PASS predicted this compound to be an ophthalmic drug with anticataract and corneal wound healing-stimulating activities, thus suggesting it to be an efficient inhibitor targeting ocular toxoplasmosis.
NSC13604 also showed interactions at the hydrophobic cleft of AMA1, with a XP Glide score of -10.016 kcal/mol.
The docked complex showed 2 hydrogen bonds with the peptide backbone of Met204, which occupies the cysteine loop region.
It also formed π-cation interactions with Tyr213 and Tyr110, stabilized by the hydrogen bonds formed with Met204 (Supplementary Fig.1D).
Met204 is a key residue for AMA1 targeting, as it is present in the cysteine loop region, which is also a critical substructure for AMA1-RON2 complex formation.
As per previous studies, disruption of the cysteine loop is found to exhibit inefficiency in domain-II loop displacement [17].
PASS predicted this compound to be antiprotozoal and may prove it to be efficient in targeting toxoplasmosis and also other apicomplexans.
NSC179676 showed interactions with the hydrophobic cleft, with an XP score of -9.893 kcal/mol.
It formed a single hydrogen bond with the backbone Ser232, which is a critical residue, as discussed earlier.
It also formed π-cation interactions with Tyr213 and Tyr110 (Supplementary Fig.1I).
PASS predicted this molecule to be antiprotozoal with corneal wound healing-stimulating activities.
These four compounds exhibited significant features in terms of XP score, key molecular interactions, and relevant predicted activity in comparison to rest of the compounds.
Hence, these compounds in complex with AMA1 were further validated for stability of complex formation by implementing MD simulation.
Title: MD simulation of apo and holo forms

MD trajectory analyses were performed for apo and also for the 4 shortlisted complexes.
The backbone RMSD deviation of the apo form of AMA1 was 1.6 Å, which has started to equilibrate and converge after 3,000 ps with a mean of 1.432 Å and a standard deviation (SD) of 0.178 Å.
The receptor RMSD for all 4 holo forms showed convergence after the time frame ranging between ~3,000 and 4,000 ps, with a maximum mean value of 1.65 Å and SD of 0.27 Å (Fig.2A).
Hence, it does not mandate an extended production run.
In the case of holo forms, the ligand_RMSD for NSC95522, NSC13604, and NSC179676 was less than 2.5 Å (acceptable range of 1–3 Å) [33], while NSC88253 showed 3.6 Å of deviation, inferring complex instability (Fig.2B–F).
Title: RMSF, Rg, and intermolecular bond stability as a measure of shortlisting potential leads targeting AMA1

The RMSF graph inferred NSC95522 and NSC179676 to exhibit the least fluctuation of &lt;1 Å at the critical hotspot residues (Tyr230, Tyr213, Tyr15, and Tyr110) in comparison to the apo form, indicating a gain of backbone rigidity.
As discussed earlier, flexibility of these residues is crucial for homing RON2.
However, the other two compounds did not show this impact (Fig.3A and 3B).
Moreover, the Rg for both apo and all 4 holo forms was found to be within an admissible deviation of &lt;1 Å. NSC95522 and NSC179676 showed the lowest Rg deviation of 0.44 Å and 0.53 Å, while the apo form showed 0.57 Å; thus, this reinforces that these two compounds confer backbone rigidity and compactness, which strongly preclude RON2 binding (Fig.3C).
Secondary structure element (SSE) perturbations were also analyzed in a similar way as discussed above.
This inferred no major dynamism in the SSE during the entire production run; it also corroborated with the fewest changes observed in the intra-molecular hydrogen bonds formed for the apo and all other 4 holo forms (Fig.3D).
For all four of these compounds, the stability of protein-ligand contacts during the production run was analyzed and visualized as a 2D interaction map (Fig.4A–D).
This inferred π-cation interactions to be unanimously stable for all four compounds.
Recent studies propose that ligands exhibiting π-cation interactions with Phe, Tyr, and Trp of the receptor highly influence the conformational stability of the proteins, as they are strong non-covalent electrostatic interactions [3435].
A holistic and stringent comparative analysis was performed over the 4 holo forms against the apo form, which strongly reinforces that AMA1-NSC95522 and AMA1-NSC179676 are stable complexes.
Moreover, both these compounds exhibit prominent π-cation interactions with the proven hotspot residues Tyr213, Tyr230, Tyr110, and Tyr215 of AMA1 for ~80% of the entire production run, implying that these compounds are potential blockers of conformational changes of AMA1, which facilitates the binding of RON2 [35].
The MMGBSA scores of the AMA1-NSC95522 and AMA1-NSC179676 complexes were found to be -79 kcal/mol and -59 kcal/mol respectively, also ensuring significant binding and stable complex formation [20].
In conclusion, NSC95522 and NSC179676 shall be considered the most potential leads, with both showing significant binding free energies with strong intermolecular interactions to hydrophobic cleft residues.
Moreover, these compounds were also found to limit the torsion angle of the AMA1 tilt of 90°, while the other two compounds showed a wide range of deviation (~90° to 180°).
Hence, all of these inferences strongly suggest that NSC95522 and NSC179676, with antiprotozoal and ophthalmic activity, shall prove to be highly efficient leads for targeting AMA1-RON2 interactions (Fig.4E), thereby combating toxoplasmosis.
However, further experimental validation is required to study the efficacy of these molecules.
Schematic representation of methodology of performing an exhaustive search and stringent validations in identifying potential leads for combating toxoplasmosis.
Multilevel precision: screens the National Cancer Institute (NCI) ligands based on Glide docking score.
Molecular Docking: identifies the favorable interactions formed with the key residues.
Pass: helps in discovering additional effects of the top compounds that might favor in suppressing the broad spectrum of toxoplasmosis effects.
Molecular Mechanics Generalized Born Surface Area (MMGBSA): calculates the free energy binding.
Molecular dynamics: evaluates the backbone stability, fluctuation, and compactness of the complex.
HTVS, high-throughput virtual screen; PASS, Prediction of Activity Spectra of Substances; RMSD, root mean square deviation; RMSF, root mean square fluctuation; SP, standard precision; XP, extra precision.
(A) Root mean square deviation (RMSD) of apo against holo forms within 2.5 Å.
(B, D, E) Protein-ligand RMSD converged within 1~3 Å.
(C) RMSD of NSC88253 exhibited instability with deviation of 3.6 Å.
(F) The RMSD score of the apo and holo forms.
(A) The root mean square fluctuation (RMSF) fluctuations of apo against holo forms.
(B) RMSF&lt;1 Å within hotspot residues.
(C) Radius of gyration depicting compactness of the apo and holo forms.
(D) Intramolecular H-bonds showing secondary structure element stability.
(A) Interaction stability diagram for NSC95522 showing a stable π-cation interaction with Tyr213 and H-bond interaction with Ser232.
(B) NSC88253 did not form any stable interactions during the production run.
(C) NSC13604 exhibited a prominent π-cation interaction with Tyr110.
(D) NSC179676 displaced a stable π-cation interaction with Tyr230 and Tyr110 and water-mediated H-bond with Ser232.
(E) Three-dimensional representation of ligands NSC95522 (green) and NSC179676 (magenta) interacting with the hotspot residue region (blue) of apical membrane antigen 1.
Title: Docking molecular interactions of 10 best compounds with XP Glide score
Title: Genome-Wide Association Study of Bone Mineral Density in Korean Men

Osteoporosis is a medical condition of global concern, with increasing incidence in both sexes.
Bone mineral density (BMD), a highly heritable trait, has been proven a useful diagnostic factor in predicting fracture.
Because medical information is lacking about male osteoporotic genetics, we conducted a genome-wide association study of BMD in Korean men.
With 1,176 participants, we analyzed 4,414,664 single nucleotide polymorphisms (SNPs) after genomic imputation, and identified five SNPs and three loci correlated with bone density and strength.
Multivariate linear regression models were applied to adjust for age and body mass index interference.
Rs17124500 (p = 6.42 × 10-7), rs34594869 (p = 6.53 × 10-7) and rs17124504 (p = 6.53 × 10-7) in 14q31.3 and rs140155614 (p = 8.64 × 10-7) in 15q25.1 were significantly associated with lumbar spine BMD (LS-BMD), while rs111822233 (p = 6.35 × 10-7) was linked with the femur total BMD (FT-BMD).
Additionally, we analyzed the relationship between BMD and five genes previously identified in Korean men.
Rs61382873 (p = 0.0009) in LRP5, rs9567003 (p = 0.0033) in TNFSF11 and rs9935828 (p = 0.0248) in FOXL1 were observed for LS-BMD.
Furthermore, rs33997547 (p = 0.0057) in ZBTB and rs1664496 (p = 0.0012) in MEF2C were found to influence FT-BMD and rs61769193 (p = 0.0114) in ZBTB to influence femur neck BMD.
We identified five SNPs and three genomic regions, associated with BMD.
The significance of our results lies in the discovery of new loci, while also affirming a previously significant locus, as potential osteoporotic factors in the Korean male population.
Osteoporosis is defined as a chronic condition that is characterized by low bone mass and the micro-architectural deterioration of osseous tissue, leading to fragility.
Bone mineral density (BMD) has been established as the most significant diagnostic factor for osteoporotic fracture [12].
Although more widespread in women, about 1.5 million males over the age of 65 have osteoporosis and another 3.5 million are at risk in the United States [3].
Among Korean men, the prevalence of the condition has also increased by about 7.5%, according to national health survey data [4].
Major risk factors of male osteoporosis are aging and lifestyle choices, such as smoking, alcohol, and lack of exercise [567].
Genetic factors may also play a significant role, because BMD has been observed to have similar heritability patterns in both men and women [8].
Thought to be heritable, BMD is 78% in the lumbar spine and 84% in the femoral neck [9].
Up to 100 genes, associated with BMD, have been identified to date by various genetic studies [1011].
One current meta-analysis identified 56 loci including 32 new regions associated with BMD with highly significant level [12].
The study had demonstrated several genetic loci such as myocyte enhancer factor 2C (MEF2C), zinc finger- and BTB-domain containing 40 (ZBTB40), low-density lipoprotein receptor-related protein 5 (LRP5), and forkhead box L1 (FOXL1) which have been significantly replicated in Korean population in later studies [1314].
However, the influence of sex-specific genes remains unclear, because most of the research has been conducted on highly homogeneous population samples, mostly European post-menopausal women [11].
Several investigations of genetic components in male osteoporosis are available, but they rely on candidate data or produced no significant results [15].
Moreover, cases of East Asian populations as the participating demographic are largely absent from the literature or consist of replications and attempts to confirm results already derived from Caucasian samples [131416].
A recent study of a Korean group was able to newly identify several polymorphic factors, but unfortunately, it utilized BMD from the radius and tibia, which is less important in the development of bone deterioration than that of the lumbar spine and hip bone [17].
Investigating the etiology behind osteoporosis in males of East Asian descent is of high diagnostic and clinical importance.
Therefore, our study aimed to evaluate the role of genetic polymorphisms in forming the BMD at the lumbar spine and femur bone in Korean men.
Title: Study populations

We selected non-osteoporotic participants who were tested via dual-energy X-ray absorptiometry from the Center for Health Promotion and Disease Prevention of Seoul National University Hospital and the Gangnam Center for Health Promotion.
The subjects underwent regular health check-ups from May 2009 to December 2013.
Our sample consisted exclusively of Korean men aged between 20 and 60 years.
This initial step of the procedure was performed to detect genetic defects associated with obesity and metabolic syndrome [1819].
Those with conditions that could affect body weight were excluded from the study.
We also excluded individuals who had taken any medication that might influence BMD, such as bisphosphonate, within the last 3 months or had a history of surgery or foreign body insertion at the spine or femur.
After applying the above criteria, 1,192 subjects were selected.
After excluding 16 participants who were related to each other, our final sample consisted of 1,176 individuals.
Title: Phenotype measurement

Dual-energy X-ray absorptiometry (Prodigy Advance; General Electric Company, Fairfield, CT, USA) measurements at the lumbar spine and femur were obtained from all patients, following standard protocols.
We distinguished and analyzed three BMD phenotypes—lumbar spine (L1-L4) (LS-BMD), femur neck (FN-BMD), and femur total (FT-BMD)—identified here as the average value of mineral density at the neck and greater trochanter of the femur bone.
Title: Genotyping and quality control

All participants underwent a microarray assay, using the Illunina HumanCore BeadChip kit (Illumina, San Diego, CA, USA).
We filtered out poor quality single nucleotide polymorphisms (SNPs) and samples, which were defined as SNPs with a monographic genotype, a Hardy-Weinberg equilibrium p &lt; 10-6, a call rate &lt;95%, and a minor allele frequency &lt;1%.
After quality control, the remaining 264,283 SNPs were used in subsequent analysis.
Title: Genotype imputation

All missing genotypes were imputed in order to examine any potential effect of the unmeasured variants.
We used 1,000 Genomes ASN Phase I integrated variant set release (v3) in National Center for Biotechnology Information (NCBI) build 37 (hg19) as a reference panel, along with PLINK software (version 1.07) [20] for strand alignment, SHAPEIT2 [21] for phasing, and IMPUTE2 [22] for the imputation procedure.
Only variants with Info Score ≥0.9 were included in the final count, resulting in a total of 4,414,664 SNPs, which were either genotyped or imputed with high confidence.
Title: Statistical analyses

Multivariate linear regressions, adjusted for the effects of age and body mass index (BMI, weight/height2, kg/m2) [23] via an additive model, were used to further investigate the influence of SNPs on BMD.
Genome-wide association patterns were analyzed by the PLINK software package version 1.07 [20], while quantile-quantile and Manhattan plots were obtained by R software 3.2.2 (2015 The R Foundation for Statistical Computing).
Regional association plots of the significant SNPs were generated by LocusZoom software [24].
All results with p &lt; 1 × 10-6 were considered statistically significant.
Title: Association with previously identified loci

We attempted to determine similarities between our selected markers and five other genes, which have previously been identified as indicative of osteoporosis in Korean male populations.
Tumor necrosis factor receptor superfamily member 11 (TNFSF11) on 13q14, FOXL1 on 16q24, LRP5 on 11q13.4, ZBTB40 on 1q31.3, and MEF2C on 5q14 have been connected to BMD in several different genome-wide association studies (GWASs)—an association confirmed in Korean males by a number of replication studies [1314].
We examined the most significant SNP regions of these five genes by applying our markers and considering only results with a p &lt; 0.05.
Title: Ethics statement

The study protocol was approved by the Institutional Review Board at Seoul National University Hospital (IRB No.
H-0911-010-299) and all participants provided written informed consent.
The mean age of the study subjects was 49.57 years, with BMI values between 16.58 kg/m2 and 37.27 kg/m2.
We examined distribution patterns of three BMD phenotypes: LS-BMD, FN-BMD, and FT-BMD.
LS-BMD ranged between 0.78 g/cm2 and 1.84 g/cm2 (mean ± SD, 1.21 ± 0.38 g/cm2), FN-BMD ranged between 0.64 g/cm2 and 1.93 g/cm2 (mean ± SD, 0.98 ± 0.14 g/cm2), and FT-BMD ranged between 0.29 g/cm2 and 1.93 g/cm2 (mean ± SD, 1.04 ± 0.14 g/cm2) (Table 1).
We identified five SNPs with a significant influence on BMD.
Rs17124500 (p = 6.42 × 10-7), rs34594869 (p = 6.53 × 10-7), rs17124504 (p = 6.53 × 10-7), and rs140155614 (p = 8.64 × 10-7) were significantly associated with LS-BMD, and rs111822233 (p = 6.35 × 10-7) was associated with FT-BMD (Table 2).
Fig.1 illustrates regional plots of the five significant SNPs within 200 kilobytes.
No genomic association between FN-BMD and any polymorphic nucleotides was established.
Quantile–quantile and Manhattan plots of the p-values are shown in Supplementary Figs.1 and 2.
Examination of genomic location revealed that rs17124500, which was the polymorphic nucleotide most closely associated with LS-BMD, is generally detected in potassium channel, subfamily K, member 1 (KCNK10) in 14q31.3 (Fig.1A).
Rs34594869 and rs17124504, were observed upstream of regions with high linkage disequilibrium constants, defined here as D&#39; &gt;0.8.
Rs140155614 was most prominent in the aminoglycoside phosphotransferase domain containing 1 (AGPHD1) locus 15q25.1 (Fig.1B).
All four of these SNPs occurred as intron variants.
rs111822233, the only sequence to influence the mineral density of the femur total, was positioned around the nuclear receptor subfamily 3, group C, member 2 (NR3C2) in 4q31.1 (Fig.1C).
The function of this SNP is currently undetermined.
In addition to establishing links between genetic polymorphisms and BMD, our studies aimed to identify loci previously confirmed as osteoporosis factors in Korean male populations in order to verify our outcomes (Table 3).
Rs61382873 in LRP5 exhibited the highest association with LS-BMD (p = 0.0009).
Rs9567003 (p = 0.0033) in TNFSF11 and rs9935828 (p = 0.0248) in FOXL1 were also linked to the lumbar spine region, albeit with higher p-values.
Patterns of rs33997547 (p = 0.0057) in ZBTB and rs1664496 (p = 0.0012) in MEF2C were correlated with FT-BMD, while rs61769193 (p = 0.0114) in ZBTB was found to influence FN-BMD.
We identified five SNPs and three genomic regions, associated with BMD.
Four of these SNPs (rs17124500, rs34594869, rs17124504, and rs140155614) were linked with LS-BMD and dispersed between two loci (14q31.1 and 15q25.1).
One SNP (rs111822233) found in 4q31.1 was exclusively associated with FT-BMD.
We also re-evaluated three previously established genetic relationships.
First, we confirmed the connection between LS-BMD and the following loci: rs61382873 in LRP5, rs9567003 in TNFSF11, and rs9935828 in FOXL1.
Next, we linked rs33997547 in ZBTB40 and rs1664496 in MEF2C with FT-BMD and analyzed rs61769193 expression patterns in FN-BMD.
Three of our significant markers—rs17124500, rs34594869, and rs17124504—were located on the KCNK10 gene.
This channel protein, embedded in the cellular membrane, consists of two pore-forming P domains and regulates K+ concentration.
The role of potassium ions in maintaining the acid-base homeostasis of most organs and tissues, including bones, has been well established in the literature [25].
Additionally, KCNK10 is located on 14q31, a locus previously associated with LS-BMD in linkage analyses [26].
Although the relationship between the gene and osseous metabolism is not clear, it is highly possible that its translational mechanisms influence bone biological parameters.
rs140155614 was prominent in the AGPHD1 domain.
Alternatively known as hydroxylysine kinase (HYKK), this protein-coding gene and its associated pathway, catalyze the GTP-dependent phosphorylation of 5-hydroxy-L-lysine.
Located on 15q25.1, it has a well-known connection to the development of lung cancer.
CHRNA3-CHRNA5-CHRNB4, a gene complex near AGPHD1, has been linked to nicotine dependence in several genetic studies [27].
Current scientific evidence is insufficient in determining the role of this locus in BMD development &#39;s replication patterns is needed to validate or dispute a relationship with BMD.
rs111822233 was the only genetic marker with a direct relationship to FT-BMD levels.
Located on 4q31.1, it is near NR3C2, a nuclear receptor involved in steroid signaling and regulation.
Mineralocorticoid and various sex hormones, such as testosterone and estradiol, alternatively bind to the protein, which influences their secretion via a feedback loop [28].
Bone resorption has been reported to increase with the withdrawal of these sex hormones.
Although estrogen is the major steroid in regulating ossein metabolism, testosterone is also important in the resorption mechanism [29].
Moreover, prevalence of osteoporosis in the hipbone has been shown to increase in men whose testosterone levels are lower than the reference range [30].
As established in our introduction, research conducted with Korean participants tends to be inconclusive owing to extensive focus on replication of results obtained by the original GWASs.
Thus, we attempted to bridge this gap in information by validating previously established genetic linkages in Caucasian populations, whose significance has been confirmed in Korean men by replication studies [1314].
LRP5, MEF2C, and ZBTB40 belong to the Wnt signaling pathway, which is largely related to cellular development and growth.
Several studies have observed a link between this cascade and BMD [31].
We utilized a series of SNP markers from various Wnt-associated genes and were able to prove as association between rs61382873 in LRP5 and LS-BMD, between rs33997547 in ZBTB40 and rs1664496 in MEF2C and FT-BMD, and between rs61769193 in ZBTB40 and FN-BMD.
TNFSF11, also known as receptor activator of nuclear factor kappa-B ligand (RANKL) plays a critical role in adequate bone metabolism via activation of osteoclast cells to regulate resorption [31].
We identified a significant SNP marker in TNFSF11, rs9567003, which was related to LS-BMD.
Our results were consistent with those of a previous genetic research, which has established a connection between TNFSF11 and BMD of the lumbar spinal cord [31].
FOXL1 was originally identified by Genetic Factors for Osteoporosis (GEFOS) and has been associated with LS-BMD in European sample groups; this result has been replicated with high significance in Korean populations [13].
Our marker in FOXL1, rs9935828, was also correlated with LS-BMD levels.
In conclusion, we described a series of new loci and confirmed GWAS results in Korean male populations by working with skeletal-specific BMD and gene associations.
There were several limitations to our study.
First, no replications were performed to validate our results.
Because indicators for measuring BMD in men tend to be narrow, recruiting a sufficient replication sample proved challenging.
Thus, we aimed to use previously identified genetic markers, which have been established as valid in our selected ethnicity and gender.
Despite our efforts, BMD is a less meaningful factor of osteoporosis in men than in women.
Additionally, we could not consider all factors, including sex, age, weight, and menopausal status, owing to using only young men as our test subjects.
Finally, we needed a very large sample because phenotypic variations are hard to detect in homogenous populations.
However, our sample size was too small to employ a Bonferroni correction, and we were forced to set the significant threshold as p &lt; 1 × 10-6, thus increasing the chance of false-positives.
To minimize statistical error, we thoroughly analyzed gene functions and mechanisms.
Nevertheless, further research is necessary to validate our findings.
In conclusion, our study identified novel loci correlated with BMD at 14q31.3, 15q25.1, and 4q31.1 in Korean men.
Future studies should investigate these associations and the underlying genetic mechanisms.
Regional plots of genome wide significant associations for bone mineral density (BMD), adjusted for age and body mass index.
(A) Regional plot of significant associated single nucleotide polymorphisms (SNPs) with BMD at lumbar spine in chromosome 14.
(B) Regional plot of significant associated SNPs with BMD at lumbar spine in chromosome 15.
(C) Regional plot of significant associated SNPs with BMD at femur total in chromosome 4.
Title: Baseline characteristics of study subjects

Title: Summary of 5 SNPs significantly associated with bone mineral density at lumbar spine and femur

Title: Summary of the association with previously identified gene in Korean men
Title: Virtual Screening for Potential Inhibitors of NS3 Protein of Zika Virus

Zika virus (ZIKV) is a mosquito borne pathogen, belongs to Flaviviridae family having a positive-sense single-stranded RNA genome, currently known for causing large epidemics in Brazil.
</pre>
</div>
</div>

<div class="jp-OutputArea-child">

    
    <div class="jp-OutputPrompt jp-OutputArea-prompt"></div>


<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="application/vnd.jupyter.stderr">
<pre>IOPub data rate exceeded.
The notebook server will temporarily stop sending output
to the client in order to avoid crashing it.
To change this limit, set the config variable
`--NotebookApp.iopub_data_rate_limit`.

Current values:
NotebookApp.iopub_data_rate_limit=1000000.0 (bytes/sec)
NotebookApp.rate_limit_window=3.0 (secs)

</pre>
</div>
</div>

</div>

</div>

</div>
<div class="jp-Cell-inputWrapper"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput " data-mime-type="text/markdown">
<p>형태소를 분석하여 특정 품사의 단어만을 추출한다.<br>
형태소 분석은 다음의 세 단계로 이루어진다.</p>
<ul>
<li>어간 추출</li>
<li>원형 복원</li>
<li>품사 부착</li>
</ul>
<h3 id="3-1.-%EC%96%B4%EA%B0%84-%EC%B6%94%EC%B6%9C">3-1. &#50612;&#44036; &#52628;&#52636;<a class="anchor-link" href="#3-1.-%EC%96%B4%EA%B0%84-%EC%B6%94%EC%B6%9C">&#182;</a></h3>
</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell jp-mod-noOutputs  ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[62]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span><span class="kn">from</span> <span class="nn">nltk.stem</span> <span class="kn">import</span> <span class="n">PorterStemmer</span><span class="p">,</span> <span class="n">LancasterStemmer</span>
</pre></div>

     </div>
</div>
</div>
</div>

</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell jp-mod-noOutputs  ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[63]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span><span class="n">st1</span> <span class="o">=</span> <span class="n">PorterStemmer</span><span class="p">()</span>
<span class="n">st2</span> <span class="o">=</span>  <span class="n">LancasterStemmer</span><span class="p">()</span>
</pre></div>

     </div>
</div>
</div>
</div>

</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell   ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[65]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span><span class="n">words</span> <span class="o">=</span> <span class="p">[]</span>
<span class="k">for</span> <span class="n">i</span> <span class="ow">in</span> <span class="nb">range</span><span class="p">(</span> <span class="nb">len</span><span class="p">(</span><span class="n">raw</span><span class="p">)</span> <span class="p">)</span> <span class="p">:</span>
    <span class="n">words</span> <span class="o">+=</span> <span class="n">word_tokenize</span><span class="p">(</span><span class="n">raw</span><span class="p">[</span><span class="n">i</span><span class="p">]</span><span class="o">.</span><span class="n">strip</span><span class="p">())</span>
    
<span class="nb">print</span><span class="p">(</span><span class="s2">&quot;Porter Stemmer :&quot;</span><span class="p">,</span> <span class="p">[</span><span class="n">st1</span><span class="o">.</span><span class="n">stem</span><span class="p">(</span><span class="n">w</span><span class="p">)</span> <span class="k">for</span> <span class="n">w</span> <span class="ow">in</span> <span class="n">words</span><span class="p">])</span>
<span class="nb">print</span><span class="p">(</span><span class="s2">&quot;Lancaster Stemmer :&quot;</span><span class="p">,</span> <span class="p">[</span><span class="n">st2</span><span class="o">.</span><span class="n">stem</span><span class="p">(</span><span class="n">w</span><span class="p">)</span> <span class="k">for</span> <span class="n">w</span> <span class="ow">in</span> <span class="n">words</span><span class="p">])</span>
</pre></div>

     </div>
</div>
</div>
</div>

<div class="jp-Cell-outputWrapper">


<div class="jp-OutputArea jp-Cell-outputArea">

<div class="jp-OutputArea-child">

    
    <div class="jp-OutputPrompt jp-OutputArea-prompt"></div>


<div class="jp-RenderedText jp-OutputArea-output" data-mime-type="application/vnd.jupyter.stderr">
<pre>IOPub data rate exceeded.
The notebook server will temporarily stop sending output
to the client in order to avoid crashing it.
To change this limit, set the config variable
`--NotebookApp.iopub_data_rate_limit`.

Current values:
NotebookApp.iopub_data_rate_limit=1000000.0 (bytes/sec)
NotebookApp.rate_limit_window=3.0 (secs)

</pre>
</div>
</div>

</div>

</div>

</div>
<div class="jp-Cell-inputWrapper"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput " data-mime-type="text/markdown">
<h3 id="3-2.-%EC%9B%90%ED%98%95-%EB%B3%B5%EC%9B%90">3-2. &#50896;&#54805; &#48373;&#50896;<a class="anchor-link" href="#3-2.-%EC%9B%90%ED%98%95-%EB%B3%B5%EC%9B%90">&#182;</a></h3><p>같은 의미를 가지는 여러 단어를 사전형으로 통일하는 작업
품사(part of speech)를 지정하는 경우 좀 더 정확한 원형을 찾기 가능</p>

</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell   ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[66]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span><span class="kn">from</span> <span class="nn">nltk.stem</span> <span class="kn">import</span> <span class="n">WordNetLemmatizer</span>

<span class="n">WNL</span> <span class="o">=</span> <span class="n">WordNetLemmatizer</span><span class="p">()</span>
<span class="n">word</span> <span class="o">=</span> <span class="p">[</span><span class="n">WNL</span><span class="o">.</span><span class="n">lemmatize</span><span class="p">(</span><span class="n">w</span><span class="p">,</span> <span class="n">pos</span><span class="o">=</span><span class="s1">&#39;v&#39;</span><span class="p">)</span> <span class="k">for</span> <span class="n">w</span> <span class="ow">in</span> <span class="n">words</span><span class="p">]</span>
<span class="n">word</span>
</pre></div>

     </div>
</div>
</div>
</div>

<div class="jp-Cell-outputWrapper">


<div class="jp-OutputArea jp-Cell-outputArea">

<div class="jp-OutputArea-child">

    
    <div class="jp-OutputPrompt jp-OutputArea-prompt">Out[66]:</div>




<div class="jp-RenderedText jp-OutputArea-output jp-OutputArea-executeResult" data-mime-type="text/plain">
<pre>[&#39;Title&#39;,
 &#39;:&#39;,
 &#39;Survey&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;Applications&#39;,
 &#39;of&#39;,
 &#39;NGS&#39;,
 &#39;to&#39;,
 &#39;Whole-Genome&#39;,
 &#39;Sequencing&#39;,
 &#39;and&#39;,
 &#39;Expression&#39;,
 &#39;Profiling&#39;,
 &#39;Recently&#39;,
 &#39;,&#39;,
 &#39;the&#39;,
 &#39;technologies&#39;,
 &#39;of&#39;,
 &#39;DNA&#39;,
 &#39;sequence&#39;,
 &#39;variation&#39;,
 &#39;and&#39;,
 &#39;gene&#39;,
 &#39;expression&#39;,
 &#39;profile&#39;,
 &#39;have&#39;,
 &#39;be&#39;,
 &#39;use&#39;,
 &#39;widely&#39;,
 &#39;as&#39;,
 &#39;approach&#39;,
 &#39;in&#39;,
 &#39;the&#39;,
 &#39;expertise&#39;,
 &#39;of&#39;,
 &#39;genome&#39;,
 &#39;biology&#39;,
 &#39;and&#39;,
 &#39;genetics&#39;,
 &#39;.&#39;,
 &#39;The&#39;,
 &#39;application&#39;,
 &#39;to&#39;,
 &#39;genome&#39;,
 &#39;study&#39;,
 &#39;have&#39;,
 &#39;be&#39;,
 &#39;particularly&#39;,
 &#39;develop&#39;,
 &#39;with&#39;,
 &#39;the&#39;,
 &#39;introduction&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;next-generation&#39;,
 &#39;DNA&#39;,
 &#39;sequencer&#39;,
 &#39;(&#39;,
 &#39;NGS&#39;,
 &#39;)&#39;,
 &#39;Roche/454&#39;,
 &#39;and&#39;,
 &#39;Illumina/Solexa&#39;,
 &#39;systems&#39;,
 &#39;,&#39;,
 &#39;along&#39;,
 &#39;with&#39;,
 &#39;bioinformation&#39;,
 &#39;analysis&#39;,
 &#39;technologies&#39;,
 &#39;of&#39;,
 &#39;whole-genome&#39;,
 &#39;de&#39;,
 &#39;novo&#39;,
 &#39;assembly&#39;,
 &#39;,&#39;,
 &#39;expression&#39;,
 &#39;profile&#39;,
 &#39;,&#39;,
 &#39;DNA&#39;,
 &#39;variation&#39;,
 &#39;discovery&#39;,
 &#39;,&#39;,
 &#39;and&#39;,
 &#39;genotyping&#39;,
 &#39;.&#39;,
 &#39;Both&#39;,
 &#39;massive&#39;,
 &#39;whole-genome&#39;,
 &#39;shotgun&#39;,
 &#39;paired-end&#39;,
 &#39;sequence&#39;,
 &#39;and&#39;,
 &#39;mate&#39;,
 &#39;paired-end&#39;,
 &#39;sequence&#39;,
 &#39;data&#39;,
 &#39;be&#39;,
 &#39;important&#39;,
 &#39;step&#39;,
 &#39;for&#39;,
 &#39;construct&#39;,
 &#39;de&#39;,
 &#39;novo&#39;,
 &#39;assembly&#39;,
 &#39;of&#39;,
 &#39;novel&#39;,
 &#39;genome&#39;,
 &#39;sequence&#39;,
 &#39;data&#39;,
 &#39;.&#39;,
 &#39;It&#39;,
 &#39;be&#39;,
 &#39;necessary&#39;,
 &#39;to&#39;,
 &#39;have&#39;,
 &#39;DNA&#39;,
 &#39;sequence&#39;,
 &#39;information&#39;,
 &#39;from&#39;,
 &#39;a&#39;,
 &#39;multiplatform&#39;,
 &#39;NGS&#39;,
 &#39;with&#39;,
 &#39;at&#39;,
 &#39;least&#39;,
 &#39;2×&#39;,
 &#39;and&#39;,
 &#39;30×&#39;,
 &#39;depth&#39;,
 &#39;sequence&#39;,
 &#39;of&#39;,
 &#39;genome&#39;,
 &#39;coverage&#39;,
 &#39;use&#39;,
 &#39;Roche/454&#39;,
 &#39;and&#39;,
 &#39;Illumina/Solexa&#39;,
 &#39;,&#39;,
 &#39;respectively&#39;,
 &#39;,&#39;,
 &#39;for&#39;,
 &#39;effective&#39;,
 &#39;an&#39;,
 &#39;way&#39;,
 &#39;of&#39;,
 &#39;de&#39;,
 &#39;novo&#39;,
 &#39;assembly&#39;,
 &#39;.&#39;,
 &#39;Massive&#39;,
 &#39;short-length&#39;,
 &#39;read&#39;,
 &#39;data&#39;,
 &#39;from&#39;,
 &#39;the&#39;,
 &#39;Illumina/Solexa&#39;,
 &#39;system&#39;,
 &#39;be&#39;,
 &#39;enough&#39;,
 &#39;to&#39;,
 &#39;discover&#39;,
 &#39;DNA&#39;,
 &#39;variation&#39;,
 &#39;,&#39;,
 &#39;result&#39;,
 &#39;in&#39;,
 &#39;reduce&#39;,
 &#39;the&#39;,
 &#39;cost&#39;,
 &#39;of&#39;,
 &#39;DNA&#39;,
 &#39;sequence&#39;,
 &#39;.&#39;,
 &#39;Whole-genome&#39;,
 &#39;expression&#39;,
 &#39;profile&#39;,
 &#39;data&#39;,
 &#39;be&#39;,
 &#39;useful&#39;,
 &#39;to&#39;,
 &#39;approach&#39;,
 &#39;genome&#39;,
 &#39;system&#39;,
 &#39;biology&#39;,
 &#39;with&#39;,
 &#39;quantification&#39;,
 &#39;of&#39;,
 &#39;express&#39;,
 &#39;RNAs&#39;,
 &#39;from&#39;,
 &#39;a&#39;,
 &#39;whole-genome&#39;,
 &#39;transcriptome&#39;,
 &#39;,&#39;,
 &#39;depend&#39;,
 &#39;on&#39;,
 &#39;the&#39;,
 &#39;tissue&#39;,
 &#39;sample&#39;,
 &#39;.&#39;,
 &#39;The&#39;,
 &#39;hybrid&#39;,
 &#39;mRNA&#39;,
 &#39;sequence&#39;,
 &#39;from&#39;,
 &#39;Rohce/454&#39;,
 &#39;and&#39;,
 &#39;Illumina/Solexa&#39;,
 &#39;be&#39;,
 &#39;more&#39;,
 &#39;powerful&#39;,
 &#39;to&#39;,
 &#39;find&#39;,
 &#39;novel&#39;,
 &#39;genes&#39;,
 &#39;through&#39;,
 &#39;de&#39;,
 &#39;novo&#39;,
 &#39;assembly&#39;,
 &#39;in&#39;,
 &#39;any&#39;,
 &#39;whole-genome&#39;,
 &#39;sequence&#39;,
 &#39;species&#39;,
 &#39;.&#39;,
 &#39;The&#39;,
 &#39;20×&#39;,
 &#39;and&#39;,
 &#39;50×&#39;,
 &#39;coverage&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;estimate&#39;,
 &#39;transcriptome&#39;,
 &#39;sequence&#39;,
 &#39;use&#39;,
 &#39;Roche/454&#39;,
 &#39;and&#39;,
 &#39;Illumina/Solexa&#39;,
 &#39;,&#39;,
 &#39;respectively&#39;,
 &#39;,&#39;,
 &#39;be&#39;,
 &#39;effective&#39;,
 &#39;to&#39;,
 &#39;create&#39;,
 &#39;novel&#39;,
 &#39;express&#39;,
 &#39;reference&#39;,
 &#39;sequence&#39;,
 &#39;.&#39;,
 &#39;However&#39;,
 &#39;,&#39;,
 &#39;only&#39;,
 &#39;an&#39;,
 &#39;average&#39;,
 &#39;30×&#39;,
 &#39;coverage&#39;,
 &#39;of&#39;,
 &#39;a&#39;,
 &#39;transcriptome&#39;,
 &#39;with&#39;,
 &#39;short&#39;,
 &#39;read&#39;,
 &#39;sequence&#39;,
 &#39;of&#39;,
 &#39;Illumina/Solexa&#39;,
 &#39;be&#39;,
 &#39;enough&#39;,
 &#39;to&#39;,
 &#39;check&#39;,
 &#39;expression&#39;,
 &#39;quantification&#39;,
 &#39;,&#39;,
 &#39;compare&#39;,
 &#39;to&#39;,
 &#39;the&#39;,
 &#39;reference&#39;,
 &#39;express&#39;,
 &#39;sequence&#39;,
 &#39;tag&#39;,
 &#39;sequence&#39;,
 &#39;.&#39;,
 &#39;Scientists&#39;,
 &#39;have&#39;,
 &#39;try&#39;,
 &#39;to&#39;,
 &#39;understand&#39;,
 &#39;biology&#39;,
 &#39;through&#39;,
 &#39;DNA&#39;,
 &#39;sequence&#39;,
 &#39;information&#39;,
 &#39;,&#39;,
 &#39;since&#39;,
 &#39;the&#39;,
 &#39;DNA&#39;,
 &#39;be&#39;,
 &#39;verify&#39;,
 &#39;as&#39;,
 &#39;the&#39;,
 &#39;unit&#39;,
 &#39;of&#39;,
 &#39;genetic&#39;,
 &#39;heredity&#39;,
 &#39;.&#39;,
 &#39;Also&#39;,
 &#39;,&#39;,
 &#39;scientists&#39;,
 &#39;hope&#39;,
 &#39;to&#39;,
 &#39;dramatically&#39;,
 &#39;reduce&#39;,
 &#39;the&#39;,
 &#39;cost&#39;,
 &#39;of&#39;,
 &#39;read&#39;,
 &#39;genomic&#39;,
 &#39;DNA&#39;,
 &#39;and&#39;,
 &#39;to&#39;,
 &#39;obtain&#39;,
 &#39;the&#39;,
 &#39;high-throughput&#39;,
 &#39;DNA&#39;,
 &#39;sequence&#39;,
 &#39;information&#39;,
 &#39;.&#39;,
 &#39;Since&#39;,
 &#39;the&#39;,
 &#39;automate&#39;,
 &#39;DNA&#39;,
 &#39;sequencers&#39;,
 &#39;be&#39;,
 &#39;develop&#39;,
 &#39;with&#39;,
 &#39;fluorescent&#39;,
 &#39;dye&#39;,
 &#39;of&#39;,
 &#39;different&#39;,
 &#39;color&#39;,
 &#39;,&#39;,
 &#39;laser&#39;,
 &#39;,&#39;,
 &#39;and&#39;,
 &#39;computer&#39;,
 &#39;technology&#39;,
 &#39;in&#39;,
 &#39;the&#39;,
 &#39;1980s&#39;,
 &#39;,&#39;,
 &#39;the&#39;,
 &#39;human&#39;,
 &#39;genome&#39;,
 &#39;project&#39;,
 &#39;(&#39;,
 &#39;HGP&#39;,
 &#39;)&#39;,
 &#39;be&#39;,
 &#39;begin&#39;,
 &#39;in&#39;,
 &#39;1990&#39;,
 &#39;,&#39;,
 &#39;and&#39;,
 &#39;the&#39;,
 &#39;human&#39;,
 &#39;genome&#39;,
 &#39;be&#39;,
 &#39;completely&#39;,
 &#39;release&#39;,
 &#39;in&#39;,
 &#39;2003&#39;,
 &#39;,&#39;,
 &#39;while&#39;,
 &#39;further&#39;,
 &#39;analysis&#39;,
 &#39;be&#39;,
 &#39;still&#39;,
 &#39;be&#39;,
 &#39;publish&#39;,
 &#39;.&#39;,
 &#39;A&#39;,
 &#39;total&#39;,
 &#39;of&#39;,
 &#39;about&#39;,
 &#39;3&#39;,
 &#39;billion&#39;,
 &#39;dollars&#39;,
 &#39;be&#39;,
 &#39;invest&#39;,
 &#39;to&#39;,
 &#39;the&#39;,
 &#39;project&#39;,
 &#39;.&#39;,
 &#39;In&#39;,
 &#39;1991&#39;,
 &#39;,&#39;,
 &#39;the&#39;,
 &#39;National&#39;,
 &#39;Human&#39;,
 &#39;Genome&#39;,
 &#39;Research&#39;,
 &#39;Institute&#39;,
 &#39;(&#39;,
 &#39;NHGRI&#39;,
 &#39;)&#39;,
 &#39;fund&#39;,
 &#39;be&#39;,
 &#39;gear&#39;,
 &#39;toward&#39;,
 &#39;lower&#39;,
 &#39;the&#39;,
 &#39;cost&#39;,
 &#39;of&#39;,
 &#39;DNA&#39;,
 &#39;sequence&#39;,
 &#39;.&#39;,
 &#39;Some&#39;,
 &#39;of&#39;,
 &#39;technologies&#39;,
 &#39;invest&#39;,
 &#39;improve&#39;,
 &#39;the&#39;,
 &#39;DNA&#39;,
 &#39;sequence&#39;,
 &#39;.&#39;,
 &#39;To&#39;,
 &#39;date&#39;,
 &#39;,&#39;,
 &#39;the&#39;,
 &#39;Applied&#39;,
 &#39;Biosystems&#39;,
 &#39;,&#39;,
 &#39;Roche/454&#39;,
 &#39;,&#39;,
 &#39;and&#39;,
 &#39;Illumina/Solexa&#39;,
 &#39;have&#39;,
 &#39;successfully&#39;,
 &#39;develop&#39;,
 &#39;their&#39;,
 &#39;technology&#39;,
 &#39;and&#39;,
 &#39;apply&#39;,
 &#39;DNA&#39;,
 &#39;sequence&#39;,
 &#39;in&#39;,
 &#39;the&#39;,
 &#39;world&#39;,
 &#39;during&#39;,
 &#39;the&#39;,
 &#39;recent&#39;,
 &#39;6&#39;,
 &#39;years&#39;,
 &#39;.&#39;,
 &#39;Most&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;newest&#39;,
 &#39;technologies&#39;,
 &#39;currently&#39;,
 &#39;in&#39;,
 &#39;use&#39;,
 &#39;generate&#39;,
 &#39;sequence&#39;,
 &#39;from&#39;,
 &#39;36&#39;,
 &#39;to&#39;,
 &#39;1,000&#39;,
 &#39;base&#39;,
 &#39;pair&#39;,
 &#39;,&#39;,
 &#39;which&#39;,
 &#39;require&#39;,
 &#39;special&#39;,
 &#39;software&#39;,
 &#39;for&#39;,
 &#39;different&#39;,
 &#39;applications&#39;,
 &#39;,&#39;,
 &#39;include&#39;,
 &#39;whole-genome&#39;,
 &#39;sequence&#39;,
 &#39;,&#39;,
 &#39;transcriptome&#39;,
 &#39;analysis&#39;,
 &#39;,&#39;,
 &#39;and&#39;,
 &#39;regulatory&#39;,
 &#39;gene&#39;,
 &#39;analysis&#39;,
 &#39;.&#39;,
 &#39;In&#39;,
 &#39;particular&#39;,
 &#39;,&#39;,
 &#39;in&#39;,
 &#39;silico&#39;,
 &#39;method&#39;,
 &#39;development&#39;,
 &#39;use&#39;,
 &#39;bioinformation&#39;,
 &#39;software&#39;,
 &#39;for&#39;,
 &#39;next-generation&#39;,
 &#39;sequence&#39;,
 &#39;assembly&#39;,
 &#39;would&#39;,
 &#39;be&#39;,
 &#39;alert&#39;,
 &#39;for&#39;,
 &#39;many&#39;,
 &#39;genome&#39;,
 &#39;project&#39;,
 &#39;and&#39;,
 &#39;more&#39;,
 &#39;applications&#39;,
 &#39;in&#39;,
 &#39;genome&#39;,
 &#39;biology&#39;,
 &#39;.&#39;,
 &#39;Particularly&#39;,
 &#39;,&#39;,
 &#39;many&#39;,
 &#39;biologists&#39;,
 &#39;and&#39;,
 &#39;geneticists&#39;,
 &#39;use&#39;,
 &#39;massive&#39;,
 &#39;DNA&#39;,
 &#39;and&#39;,
 &#39;RNA&#39;,
 &#39;sequence&#39;,
 &#39;have&#39;,
 &#39;use&#39;,
 &#39;the&#39;,
 &#39;sequence&#39;,
 &#39;applications&#39;,
 &#39;focus&#39;,
 &#39;on&#39;,
 &#39;the&#39;,
 &#39;variable&#39;,
 &#39;research&#39;,
 &#39;field&#39;,
 &#39;in&#39;,
 &#39;medicine&#39;,
 &#39;,&#39;,
 &#39;such&#39;,
 &#39;as&#39;,
 &#39;improve&#39;,
 &#39;diagnosis&#39;,
 &#39;of&#39;,
 &#39;disease&#39;,
 &#39;;&#39;,
 &#39;gene&#39;,
 &#39;therapy&#39;,
 &#39;;&#39;,
 &#39;control&#39;,
 &#39;systems&#39;,
 &#39;for&#39;,
 &#39;drug&#39;,
 &#39;,&#39;,
 &#39;include&#39;,
 &#39;pharmacogenomics&#39;,
 &#39;``&#39;,
 &#39;custom&#39;,
 &#39;drug&#39;,
 &#39;;&#39;,
 &#34;&#39;&#39;&#34;,
 &#39;evolution&#39;,
 &#39;;&#39;,
 &#39;bioenergy&#39;,
 &#39;and&#39;,
 &#39;environmental&#39;,
 &#39;applications&#39;,
 &#39;of&#39;,
 &#39;creative&#39;,
 &#39;energy&#39;,
 &#39;source&#39;,
 &#39;(&#39;,
 &#39;biofuels&#39;,
 &#39;)&#39;,
 &#39;;&#39;,
 &#39;clean&#39;,
 &#39;up&#39;,
 &#39;of&#39;,
 &#39;toxic&#39;,
 &#39;waste&#39;,
 &#39;,&#39;,
 &#39;include&#39;,
 &#39;efficient&#39;,
 &#39;environmental&#39;,
 &#39;source&#39;,
 &#39;against&#39;,
 &#39;carbon&#39;,
 &#39;;&#39;,
 &#39;and&#39;,
 &#39;agriculture&#39;,
 &#39;project&#39;,
 &#39;,&#39;,
 &#39;include&#39;,
 &#39;livestock&#39;,
 &#39;of&#39;,
 &#39;healthier&#39;,
 &#39;,&#39;,
 &#39;more&#39;,
 &#39;productive&#39;,
 &#39;,&#39;,
 &#39;disease-resistant&#39;,
 &#39;farm&#39;,
 &#39;animals&#39;,
 &#39;and&#39;,
 &#39;breed&#39;,
 &#39;of&#39;,
 &#39;disease-&#39;,
 &#39;,&#39;,
 &#39;insect-&#39;,
 &#39;,&#39;,
 &#39;and&#39;,
 &#39;drought-resistant&#39;,
 &#39;crop&#39;,
 &#39;.&#39;,
 &#39;In&#39;,
 &#39;this&#39;,
 &#39;paper&#39;,
 &#39;,&#39;,
 &#39;we&#39;,
 &#39;report&#39;,
 &#39;several&#39;,
 &#39;ways&#39;,
 &#39;of&#39;,
 &#39;genome&#39;,
 &#39;sequence&#39;,
 &#39;and&#39;,
 &#39;expression&#39;,
 &#39;profile&#39;,
 &#39;in&#39;,
 &#39;genome&#39;,
 &#39;biology&#39;,
 &#39;.&#39;,
 &#39;Title&#39;,
 &#39;:&#39;,
 &#39;Roche/454&#39;,
 &#39;pyrosequencing&#39;,
 &#39;technology&#39;,
 &#39;The&#39;,
 &#39;first&#39;,
 &#39;high-parallel&#39;,
 &#39;sequence&#39;,
 &#39;system&#39;,
 &#39;be&#39;,
 &#39;develop&#39;,
 &#39;with&#39;,
 &#39;an&#39;,
 &#39;emulsion&#39;,
 &#39;PCR&#39;,
 &#39;method&#39;,
 &#39;for&#39;,
 &#39;DNA&#39;,
 &#39;amplification&#39;,
 &#39;and&#39;,
 &#39;an&#39;,
 &#39;instrument&#39;,
 &#39;for&#39;,
 &#39;sequence&#39;,
 &#39;by&#39;,
 &#39;synthesis&#39;,
 &#39;use&#39;,
 &#39;a&#39;,
 &#39;pyrosequencing&#39;,
 &#39;protocol&#39;,
 &#39;optimize&#39;,
 &#39;on&#39;,
 &#39;the&#39;,
 &#39;individual&#39;,
 &#39;well&#39;,
 &#39;of&#39;,
 &#39;a&#39;,
 &#39;PicoTiterPlate&#39;,
 &#39;(&#39;,
 &#39;PTP&#39;,
 &#39;)&#39;,
 &#39;[&#39;,
 &#39;1&#39;,
 &#39;]&#39;,
 &#39;.&#39;,
 &#39;The&#39;,
 &#39;DNA&#39;,
 &#39;sequence&#39;,
 &#39;protocol&#39;,
 &#39;,&#39;,
 &#39;include&#39;,
 &#39;sample&#39;,
 &#39;preparation&#39;,
 &#39;,&#39;,
 &#39;be&#39;,
 &#39;supply&#39;,
 &#39;for&#39;,
 &#39;a&#39;,
 &#39;user&#39;,
 &#39;to&#39;,
 &#39;follow&#39;,
 &#39;the&#39;,
 &#39;method&#39;,
 &#39;,&#39;,
 &#39;which&#39;,
 &#39;be&#39;,
 &#39;develop&#39;,
 &#39;by&#39;,
 &#39;454&#39;,
 &#39;Life&#39;,
 &#39;Science&#39;,
 &#39;(&#39;,
 &#39;now&#39;,
 &#39;part&#39;,
 &#39;of&#39;,
 &#39;Roche&#39;,
 &#39;Diagnostics&#39;,
 &#39;,&#39;,
 &#39;Mannheim&#39;,
 &#39;,&#39;,
 &#39;Germany&#39;,
 &#39;)&#39;,
 &#39;.&#39;,
 &#39;Even&#39;,
 &#39;follow&#39;,
 &#39;the&#39;,
 &#39;manufacturer&#39;,
 &#34;&#39;s&#34;,
 &#39;protocol&#39;,
 &#39;,&#39;,
 &#39;to&#39;,
 &#39;obtain&#39;,
 &#39;high-quality&#39;,
 &#39;DNA&#39;,
 &#39;sequence&#39;,
 &#39;with&#39;,
 &#39;maximal&#39;,
 &#39;total&#39;,
 &#39;DNA&#39;,
 &#39;sequence&#39;,
 &#39;length&#39;,
 &#39;as&#39;,
 &#39;possible&#39;,
 &#39;,&#39;,
 &#39;one&#39;,
 &#39;should&#39;,
 &#39;make&#39;,
 &#39;a&#39;,
 &#39;library&#39;,
 &#39;with&#39;,
 &#39;both&#39;,
 &#39;adapters&#39;,
 &#39;on&#39;,
 &#39;the&#39;,
 &#39;shear&#39;,
 &#39;DNA&#39;,
 &#39;fragment&#39;,
 &#39;and&#39;,
 &#39;mix&#39;,
 &#39;the&#39;,
 &#39;optical&#39;,
 &#39;ratio&#39;,
 &#39;of&#39;,
 &#39;library&#39;,
 &#39;DNA&#39;,
 &#39;versus&#39;,
 &#39;bead&#39;,
 &#39;for&#39;,
 &#39;emulsion&#39;,
 &#39;PCR&#39;,
 &#39;.&#39;,
 &#39;Now&#39;,
 &#39;,&#39;,
 &#39;the&#39;,
 &#39;system&#39;,
 &#39;be&#39;,
 &#39;upgrade&#39;,
 &#39;to&#39;,
 &#39;through-put&#39;,
 &#39;a&#39;,
 &#39;total&#39;,
 &#39;of&#39;,
 &#39;an&#39;,
 &#39;average&#39;,
 &#39;of&#39;,
 &#39;700&#39;,
 &#39;Mbp&#39;,
 &#39;with&#39;,
 &#39;an&#39;,
 &#39;average&#39;,
 &#39;600&#39;,
 &#39;bp/read&#39;,
 &#39;in&#39;,
 &#39;one&#39;,
 &#39;PTP&#39;,
 &#39;run&#39;,
 &#39;.&#39;,
 &#39;The&#39;,
 &#39;sequencer&#39;,
 &#39;be&#39;,
 &#39;available&#39;,
 &#39;for&#39;,
 &#39;single&#39;,
 &#39;read&#39;,
 &#39;and&#39;,
 &#39;paired-end&#39;,
 &#39;read&#39;,
 &#39;sequence&#39;,
 &#39;for&#39;,
 &#39;the&#39;,
 &#39;application&#39;,
 &#39;of&#39;,
 &#39;eukaryotic&#39;,
 &#39;and&#39;,
 &#39;prokaryotic&#39;,
 &#39;whole-genome&#39;,
 &#39;sequence&#39;,
 &#39;[&#39;,
 &#39;2&#39;,
 &#39;,&#39;,
 &#39;3&#39;,
 &#39;]&#39;,
 &#39;,&#39;,
 &#39;metagenomics&#39;,
 &#39;and&#39;,
 &#39;microbial&#39;,
 &#39;diversity&#39;,
 &#39;[&#39;,
 &#39;4&#39;,
 &#39;,&#39;,
 &#39;5&#39;,
 &#39;]&#39;,
 &#39;,&#39;,
 &#39;and&#39;,
 &#39;genetic&#39;,
 &#39;variation&#39;,
 &#39;detection&#39;,
 &#39;for&#39;,
 &#39;comparative&#39;,
 &#39;genomics&#39;,
 &#39;[&#39;,
 &#39;6&#39;,
 &#39;]&#39;,
 &#39;.&#39;,
 &#39;Title&#39;,
 &#39;:&#39;,
 &#39;Illumina/Solexa&#39;,
 &#39;technology&#39;,
 &#39;The&#39;,
 &#39;numerous&#39;,
 &#39;cost-effective&#39;,
 &#39;technologies&#39;,
 &#39;be&#39;,
 &#39;be&#39;,
 &#39;develop&#39;,
 &#39;for&#39;,
 &#39;human&#39;,
 &#39;genome&#39;,
 &#39;resequencing&#39;,
 &#39;that&#39;,
 &#39;can&#39;,
 &#39;be&#39;,
 &#39;align&#39;,
 &#39;to&#39;,
 &#39;the&#39;,
 &#39;reference&#39;,
 &#39;sequence&#39;,
 &#39;[&#39;,
 &#39;7&#39;,
 &#39;,&#39;,
 &#39;8&#39;,
 &#39;]&#39;,
 &#39;.&#39;,
 &#39;The&#39;,
 &#39;first&#39;,
 &#39;successful&#39;,
 &#39;technology&#39;,
 &#39;to&#39;,
 &#39;gain&#39;,
 &#39;massive&#39;,
 &#39;DNA&#39;,
 &#39;sequence&#39;,
 &#39;available&#39;,
 &#39;for&#39;,
 &#39;resequencing&#39;,
 &#39;be&#39;,
 &#39;develop&#39;,
 &#39;by&#39;,
 &#39;Solexa&#39;,
 &#39;(&#39;,
 &#39;now&#39;,
 &#39;part&#39;,
 &#39;of&#39;,
 &#39;Illumina&#39;,
 &#39;,&#39;,
 &#39;San&#39;,
 &#39;Diego&#39;,
 &#39;,&#39;,
 &#39;CA&#39;,
 &#39;,&#39;,
 &#39;USA&#39;,
 &#39;)&#39;,
 &#39;.&#39;,
 &#39;The&#39;,
 &#39;principle&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;system&#39;,
 &#39;be&#39;,
 &#39;the&#39;,
 &#39;method&#39;,
 &#39;of&#39;,
 &#39;base-by-base&#39;,
 &#39;sequence&#39;,
 &#39;by&#39;,
 &#39;synthesis&#39;,
 &#39;,&#39;,
 &#39;where&#39;,
 &#39;the&#39;,
 &#39;shear&#39;,
 &#39;template&#39;,
 &#39;DNA&#39;,
 &#39;be&#39;,
 &#39;amplify&#39;,
 &#39;on&#39;,
 &#39;the&#39;,
 &#39;flat&#39;,
 &#39;surface&#39;,
 &#39;slide&#39;,
 &#39;(&#39;,
 &#39;flow&#39;,
 &#39;cell&#39;,
 &#39;)&#39;,
 &#39;and&#39;,
 &#39;detect&#39;,
 &#39;one&#39;,
 &#39;base&#39;,
 &#39;on&#39;,
 &#39;each&#39;,
 &#39;template&#39;,
 &#39;per&#39;,
 &#39;cycle&#39;,
 &#39;with&#39;,
 &#39;four&#39;,
 &#39;base-specific&#39;,
 &#39;,&#39;,
 &#39;fluorescently&#39;,
 &#39;label&#39;,
 &#39;signal&#39;,
 &#39;.&#39;,
 &#39;Signals&#39;,
 &#39;for&#39;,
 &#39;all&#39;,
 &#39;four&#39;,
 &#39;fluorescent&#39;,
 &#39;channel&#39;,
 &#39;be&#39;,
 &#39;collect&#39;,
 &#39;and&#39;,
 &#39;plot&#39;,
 &#39;at&#39;,
 &#39;each&#39;,
 &#39;position&#39;,
 &#39;,&#39;,
 &#39;enable&#39;,
 &#39;quality&#39;,
 &#39;per&#39;,
 &#39;score&#39;,
 &#39;to&#39;,
 &#39;be&#39;,
 &#39;derive&#39;,
 &#39;use&#39;,
 &#39;four-color&#39;,
 &#39;information&#39;,
 &#39;if&#39;,
 &#39;desire&#39;,
 &#39;[&#39;,
 &#39;8&#39;,
 &#39;]&#39;,
 &#39;.&#39;,
 &#39;Now&#39;,
 &#39;,&#39;,
 &#39;a&#39;,
 &#39;maximum&#39;,
 &#39;of&#39;,
 &#39;300&#39;,
 &#39;Gb&#39;,
 &#39;for&#39;,
 &#39;read&#39;,
 &#39;be&#39;,
 &#39;available&#39;,
 &#39;with&#39;,
 &#39;101&#39;,
 &#39;bp&#39;,
 &#39;paired-end&#39;,
 &#39;read&#39;,
 &#39;per&#39;,
 &#39;fragment&#39;,
 ...]</pre>
</div>

</div>

</div>

</div>

</div>
<div class="jp-Cell-inputWrapper"><div class="jp-InputPrompt jp-InputArea-prompt">
</div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput " data-mime-type="text/markdown">
<h3 id="3-3.-%ED%92%88%EC%82%AC-%EB%B6%80%EC%B0%A9">3-3. &#54408;&#49324; &#48512;&#52265;<a class="anchor-link" href="#3-3.-%ED%92%88%EC%82%AC-%EB%B6%80%EC%B0%A9">&#182;</a></h3><p>낱말을 문법적인 기능이나 형태, 뜻에 따라 구분한 것
Penn Treebank Tagset in NLTK</p>
<ul>
<li>NNP: 단수 고유명사</li>
<li>VB: 동사</li>
<li>VBP: 동사 현재형</li>
<li>TO: to 전치사</li>
<li>NN: 명사(단수형 혹은 집합형)</li>
<li>DT: 관형사</li>
</ul>

</div>
</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell   ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[68]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># pos_tag : 단어 토큰에 품사 부착하여 튜플로 출력.</span>
<span class="kn">from</span> <span class="nn">nltk.tag</span> <span class="kn">import</span> <span class="n">pos_tag</span>
<span class="n">tagged_li</span> <span class="o">=</span> <span class="n">pos_tag</span><span class="p">(</span><span class="n">words</span><span class="p">)</span>
<span class="n">tagged_li</span><span class="p">[:</span><span class="mi">10</span><span class="p">]</span>
</pre></div>

     </div>
</div>
</div>
</div>

<div class="jp-Cell-outputWrapper">


<div class="jp-OutputArea jp-Cell-outputArea">

<div class="jp-OutputArea-child">

    
    <div class="jp-OutputPrompt jp-OutputArea-prompt">Out[68]:</div>




<div class="jp-RenderedText jp-OutputArea-output jp-OutputArea-executeResult" data-mime-type="text/plain">
<pre>[(&#39;Title&#39;, &#39;NN&#39;),
 (&#39;:&#39;, &#39;:&#39;),
 (&#39;Survey&#39;, &#39;NN&#39;),
 (&#39;of&#39;, &#39;IN&#39;),
 (&#39;the&#39;, &#39;DT&#39;),
 (&#39;Applications&#39;, &#39;NNP&#39;),
 (&#39;of&#39;, &#39;IN&#39;),
 (&#39;NGS&#39;, &#39;NNP&#39;),
 (&#39;to&#39;, &#39;TO&#39;),
 (&#39;Whole-Genome&#39;, &#39;NNP&#39;)]</pre>
</div>

</div>

</div>

</div>

</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell   ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[69]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># 품사 태그 정보 - 동사인 토큰만 선택.</span>
<span class="n">verbs_li</span> <span class="o">=</span> <span class="p">[</span><span class="n">t</span><span class="p">[</span><span class="mi">0</span><span class="p">]</span> <span class="k">for</span> <span class="n">t</span> <span class="ow">in</span> <span class="n">tagged_li</span> <span class="k">if</span> <span class="n">t</span><span class="p">[</span><span class="mi">1</span><span class="p">]</span> <span class="o">==</span> <span class="s2">&quot;VB&quot;</span><span class="p">]</span>
<span class="n">verbs_li</span><span class="p">[:</span><span class="mi">10</span><span class="p">]</span>
</pre></div>

     </div>
</div>
</div>
</div>

<div class="jp-Cell-outputWrapper">


<div class="jp-OutputArea jp-Cell-outputArea">

<div class="jp-OutputArea-child">

    
    <div class="jp-OutputPrompt jp-OutputArea-prompt">Out[69]:</div>




<div class="jp-RenderedText jp-OutputArea-output jp-OutputArea-executeResult" data-mime-type="text/plain">
<pre>[&#39;genome&#39;,
 &#39;mate&#39;,
 &#39;have&#39;,
 &#39;discover&#39;,
 &#39;approach&#39;,
 &#39;find&#39;,
 &#39;create&#39;,
 &#39;check&#39;,
 &#39;understand&#39;,
 &#39;reduce&#39;]</pre>
</div>

</div>

</div>

</div>

</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell jp-mod-noOutputs  ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[70]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># 같은 토큰이라도 품사가 다르면 다른 토큰으로 처리</span>
<span class="c1"># 원래의 토큰과 품사를 붙여서 새로운 토큰 이름을 만들어 사용</span>
<span class="c1"># 철자는 같고 품사가 다른 단어 구분</span>
</pre></div>

     </div>
</div>
</div>
</div>

</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell   ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[71]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span><span class="k">def</span> <span class="nf">tokenizer</span><span class="p">(</span><span class="n">doc</span><span class="p">):</span>
    <span class="k">return</span> <span class="p">[</span><span class="s2">&quot;/&quot;</span><span class="o">.</span><span class="n">join</span><span class="p">(</span><span class="n">p</span><span class="p">)</span> <span class="k">for</span> <span class="n">p</span> <span class="ow">in</span> <span class="n">tagged_li</span><span class="p">]</span>

<span class="n">tokenizer</span><span class="p">(</span><span class="n">sentences</span><span class="p">)</span>
</pre></div>

     </div>
</div>
</div>
</div>

<div class="jp-Cell-outputWrapper">


<div class="jp-OutputArea jp-Cell-outputArea">

<div class="jp-OutputArea-child">

    
    <div class="jp-OutputPrompt jp-OutputArea-prompt">Out[71]:</div>




<div class="jp-RenderedText jp-OutputArea-output jp-OutputArea-executeResult" data-mime-type="text/plain">
<pre>[&#39;Title/NN&#39;,
 &#39;:/:&#39;,
 &#39;Survey/NN&#39;,
 &#39;of/IN&#39;,
 &#39;the/DT&#39;,
 &#39;Applications/NNP&#39;,
 &#39;of/IN&#39;,
 &#39;NGS/NNP&#39;,
 &#39;to/TO&#39;,
 &#39;Whole-Genome/NNP&#39;,
 &#39;Sequencing/NNP&#39;,
 &#39;and/CC&#39;,
 &#39;Expression/NNP&#39;,
 &#39;Profiling/NNP&#39;,
 &#39;Recently/NNP&#39;,
 &#39;,/,&#39;,
 &#39;the/DT&#39;,
 &#39;technologies/NNS&#39;,
 &#39;of/IN&#39;,
 &#39;DNA/NNP&#39;,
 &#39;sequence/NN&#39;,
 &#39;variation/NN&#39;,
 &#39;and/CC&#39;,
 &#39;gene/NN&#39;,
 &#39;expression/NN&#39;,
 &#39;profiling/VBG&#39;,
 &#39;have/VBP&#39;,
 &#39;been/VBN&#39;,
 &#39;used/VBN&#39;,
 &#39;widely/RB&#39;,
 &#39;as/IN&#39;,
 &#39;approaches/NNS&#39;,
 &#39;in/IN&#39;,
 &#39;the/DT&#39;,
 &#39;expertise/NN&#39;,
 &#39;of/IN&#39;,
 &#39;genome/JJ&#39;,
 &#39;biology/NN&#39;,
 &#39;and/CC&#39;,
 &#39;genetics/NNS&#39;,
 &#39;./.&#39;,
 &#39;The/DT&#39;,
 &#39;application/NN&#39;,
 &#39;to/TO&#39;,
 &#39;genome/VB&#39;,
 &#39;study/NN&#39;,
 &#39;has/VBZ&#39;,
 &#39;been/VBN&#39;,
 &#39;particularly/RB&#39;,
 &#39;developed/VBN&#39;,
 &#39;with/IN&#39;,
 &#39;the/DT&#39;,
 &#39;introduction/NN&#39;,
 &#39;of/IN&#39;,
 &#39;the/DT&#39;,
 &#39;next-generation/JJ&#39;,
 &#39;DNA/NNP&#39;,
 &#39;sequencer/NN&#39;,
 &#39;(/(&#39;,
 &#39;NGS/NNP&#39;,
 &#39;)/)&#39;,
 &#39;Roche/454/NNP&#39;,
 &#39;and/CC&#39;,
 &#39;Illumina/Solexa/NNP&#39;,
 &#39;systems/NNS&#39;,
 &#39;,/,&#39;,
 &#39;along/IN&#39;,
 &#39;with/IN&#39;,
 &#39;bioinformation/NN&#39;,
 &#39;analysis/NN&#39;,
 &#39;technologies/NNS&#39;,
 &#39;of/IN&#39;,
 &#39;whole-genome/JJ&#39;,
 &#39;de/FW&#39;,
 &#39;novo/FW&#39;,
 &#39;assembly/RB&#39;,
 &#39;,/,&#39;,
 &#39;expression/NN&#39;,
 &#39;profiling/NN&#39;,
 &#39;,/,&#39;,
 &#39;DNA/NNP&#39;,
 &#39;variation/NN&#39;,
 &#39;discovery/NN&#39;,
 &#39;,/,&#39;,
 &#39;and/CC&#39;,
 &#39;genotyping/NN&#39;,
 &#39;./.&#39;,
 &#39;Both/DT&#39;,
 &#39;massive/JJ&#39;,
 &#39;whole-genome/JJ&#39;,
 &#39;shotgun/NN&#39;,
 &#39;paired-end/JJ&#39;,
 &#39;sequencing/NN&#39;,
 &#39;and/CC&#39;,
 &#39;mate/VB&#39;,
 &#39;paired-end/JJ&#39;,
 &#39;sequencing/NN&#39;,
 &#39;data/NNS&#39;,
 &#39;are/VBP&#39;,
 &#39;important/JJ&#39;,
 &#39;steps/NNS&#39;,
 &#39;for/IN&#39;,
 &#39;constructing/VBG&#39;,
 &#39;de/FW&#39;,
 &#39;novo/FW&#39;,
 &#39;assembly/NN&#39;,
 &#39;of/IN&#39;,
 &#39;novel/JJ&#39;,
 &#39;genome/JJ&#39;,
 &#39;sequencing/NN&#39;,
 &#39;data/NNS&#39;,
 &#39;./.&#39;,
 &#39;It/PRP&#39;,
 &#39;is/VBZ&#39;,
 &#39;necessary/JJ&#39;,
 &#39;to/TO&#39;,
 &#39;have/VB&#39;,
 &#39;DNA/NNP&#39;,
 &#39;sequence/NN&#39;,
 &#39;information/NN&#39;,
 &#39;from/IN&#39;,
 &#39;a/DT&#39;,
 &#39;multiplatform/NN&#39;,
 &#39;NGS/NN&#39;,
 &#39;with/IN&#39;,
 &#39;at/IN&#39;,
 &#39;least/JJS&#39;,
 &#39;2×/CD&#39;,
 &#39;and/CC&#39;,
 &#39;30×/CD&#39;,
 &#39;depth/JJ&#39;,
 &#39;sequence/NN&#39;,
 &#39;of/IN&#39;,
 &#39;genome/JJ&#39;,
 &#39;coverage/NN&#39;,
 &#39;using/VBG&#39;,
 &#39;Roche/454/NNP&#39;,
 &#39;and/CC&#39;,
 &#39;Illumina/Solexa/NNP&#39;,
 &#39;,/,&#39;,
 &#39;respectively/RB&#39;,
 &#39;,/,&#39;,
 &#39;for/IN&#39;,
 &#39;effective/JJ&#39;,
 &#39;an/DT&#39;,
 &#39;way/NN&#39;,
 &#39;of/IN&#39;,
 &#39;de/FW&#39;,
 &#39;novo/FW&#39;,
 &#39;assembly/RB&#39;,
 &#39;./.&#39;,
 &#39;Massive/JJ&#39;,
 &#39;short-length/JJ&#39;,
 &#39;reading/NN&#39;,
 &#39;data/NNS&#39;,
 &#39;from/IN&#39;,
 &#39;the/DT&#39;,
 &#39;Illumina/Solexa/NNP&#39;,
 &#39;system/NN&#39;,
 &#39;is/VBZ&#39;,
 &#39;enough/JJ&#39;,
 &#39;to/TO&#39;,
 &#39;discover/VB&#39;,
 &#39;DNA/NNP&#39;,
 &#39;variation/NN&#39;,
 &#39;,/,&#39;,
 &#39;resulting/VBG&#39;,
 &#39;in/IN&#39;,
 &#39;reducing/VBG&#39;,
 &#39;the/DT&#39;,
 &#39;cost/NN&#39;,
 &#39;of/IN&#39;,
 &#39;DNA/NNP&#39;,
 &#39;sequencing/VBG&#39;,
 &#39;./.&#39;,
 &#39;Whole-genome/JJ&#39;,
 &#39;expression/NN&#39;,
 &#39;profile/NN&#39;,
 &#39;data/NNS&#39;,
 &#39;are/VBP&#39;,
 &#39;useful/JJ&#39;,
 &#39;to/TO&#39;,
 &#39;approach/VB&#39;,
 &#39;genome/JJ&#39;,
 &#39;system/NN&#39;,
 &#39;biology/NN&#39;,
 &#39;with/IN&#39;,
 &#39;quantification/NN&#39;,
 &#39;of/IN&#39;,
 &#39;expressed/VBN&#39;,
 &#39;RNAs/NNP&#39;,
 &#39;from/IN&#39;,
 &#39;a/DT&#39;,
 &#39;whole-genome/JJ&#39;,
 &#39;transcriptome/NN&#39;,
 &#39;,/,&#39;,
 &#39;depending/VBG&#39;,
 &#39;on/IN&#39;,
 &#39;the/DT&#39;,
 &#39;tissue/NN&#39;,
 &#39;samples/NNS&#39;,
 &#39;./.&#39;,
 &#39;The/DT&#39;,
 &#39;hybrid/JJ&#39;,
 &#39;mRNA/NN&#39;,
 &#39;sequences/NNS&#39;,
 &#39;from/IN&#39;,
 &#39;Rohce/454/NNP&#39;,
 &#39;and/CC&#39;,
 &#39;Illumina/Solexa/NNP&#39;,
 &#39;are/VBP&#39;,
 &#39;more/RBR&#39;,
 &#39;powerful/JJ&#39;,
 &#39;to/TO&#39;,
 &#39;find/VB&#39;,
 &#39;novel/JJ&#39;,
 &#39;genes/NNS&#39;,
 &#39;through/IN&#39;,
 &#39;de/IN&#39;,
 &#39;novo/NN&#39;,
 &#39;assembly/RB&#39;,
 &#39;in/IN&#39;,
 &#39;any/DT&#39;,
 &#39;whole-genome/JJ&#39;,
 &#39;sequenced/JJ&#39;,
 &#39;species/NNS&#39;,
 &#39;./.&#39;,
 &#39;The/DT&#39;,
 &#39;20×/CD&#39;,
 &#39;and/CC&#39;,
 &#39;50×/CD&#39;,
 &#39;coverage/NN&#39;,
 &#39;of/IN&#39;,
 &#39;the/DT&#39;,
 &#39;estimated/VBN&#39;,
 &#39;transcriptome/NN&#39;,
 &#39;sequences/NNS&#39;,
 &#39;using/VBG&#39;,
 &#39;Roche/454/NNP&#39;,
 &#39;and/CC&#39;,
 &#39;Illumina/Solexa/NNP&#39;,
 &#39;,/,&#39;,
 &#39;respectively/RB&#39;,
 &#39;,/,&#39;,
 &#39;is/VBZ&#39;,
 &#39;effective/JJ&#39;,
 &#39;to/TO&#39;,
 &#39;create/VB&#39;,
 &#39;novel/NNS&#39;,
 &#39;expressed/VBN&#39;,
 &#39;reference/NN&#39;,
 &#39;sequences/NNS&#39;,
 &#39;./.&#39;,
 &#39;However/RB&#39;,
 &#39;,/,&#39;,
 &#39;only/RB&#39;,
 &#39;an/DT&#39;,
 &#39;average/JJ&#39;,
 &#39;30×/CD&#39;,
 &#39;coverage/NN&#39;,
 &#39;of/IN&#39;,
 &#39;a/DT&#39;,
 &#39;transcriptome/NN&#39;,
 &#39;with/IN&#39;,
 &#39;short/JJ&#39;,
 &#39;read/NN&#39;,
 &#39;sequences/NNS&#39;,
 &#39;of/IN&#39;,
 &#39;Illumina/Solexa/NNP&#39;,
 &#39;is/VBZ&#39;,
 &#39;enough/JJ&#39;,
 &#39;to/TO&#39;,
 &#39;check/VB&#39;,
 &#39;expression/NN&#39;,
 &#39;quantification/NN&#39;,
 &#39;,/,&#39;,
 &#39;compared/VBN&#39;,
 &#39;to/TO&#39;,
 &#39;the/DT&#39;,
 &#39;reference/NN&#39;,
 &#39;expressed/VBD&#39;,
 &#39;sequence/NN&#39;,
 &#39;tag/NN&#39;,
 &#39;sequence/NN&#39;,
 &#39;./.&#39;,
 &#39;Scientists/NNS&#39;,
 &#39;have/VBP&#39;,
 &#39;tried/VBN&#39;,
 &#39;to/TO&#39;,
 &#39;understand/VB&#39;,
 &#39;biology/NN&#39;,
 &#39;through/IN&#39;,
 &#39;DNA/NNP&#39;,
 &#39;sequence/NN&#39;,
 &#39;information/NN&#39;,
 &#39;,/,&#39;,
 &#39;since/IN&#39;,
 &#39;the/DT&#39;,
 &#39;DNA/NNP&#39;,
 &#39;is/VBZ&#39;,
 &#39;verified/VBN&#39;,
 &#39;as/IN&#39;,
 &#39;the/DT&#39;,
 &#39;unit/NN&#39;,
 &#39;of/IN&#39;,
 &#39;genetic/JJ&#39;,
 &#39;heredity/NN&#39;,
 &#39;./.&#39;,
 &#39;Also/RB&#39;,
 &#39;,/,&#39;,
 &#39;scientists/NNS&#39;,
 &#39;hope/VBP&#39;,
 &#39;to/TO&#39;,
 &#39;dramatically/RB&#39;,
 &#39;reduce/VB&#39;,
 &#39;the/DT&#39;,
 &#39;cost/NN&#39;,
 &#39;of/IN&#39;,
 &#39;reading/VBG&#39;,
 &#39;genomic/JJ&#39;,
 &#39;DNA/NN&#39;,
 &#39;and/CC&#39;,
 &#39;to/TO&#39;,
 &#39;obtain/VB&#39;,
 &#39;the/DT&#39;,
 &#39;high-throughput/JJ&#39;,
 &#39;DNA/NNP&#39;,
 &#39;sequence/NN&#39;,
 &#39;information/NN&#39;,
 &#39;./.&#39;,
 &#39;Since/IN&#39;,
 &#39;the/DT&#39;,
 &#39;automated/VBN&#39;,
 &#39;DNA/NN&#39;,
 &#39;sequencers/NNS&#39;,
 &#39;were/VBD&#39;,
 &#39;developed/VBN&#39;,
 &#39;with/IN&#39;,
 &#39;fluorescent/JJ&#39;,
 &#39;dyes/NNS&#39;,
 &#39;of/IN&#39;,
 &#39;different/JJ&#39;,
 &#39;colors/NNS&#39;,
 &#39;,/,&#39;,
 &#39;laser/NN&#39;,
 &#39;,/,&#39;,
 &#39;and/CC&#39;,
 &#39;computer/NN&#39;,
 &#39;technology/NN&#39;,
 &#39;in/IN&#39;,
 &#39;the/DT&#39;,
 &#39;1980s/CD&#39;,
 &#39;,/,&#39;,
 &#39;the/DT&#39;,
 &#39;human/JJ&#39;,
 &#39;genome/NN&#39;,
 &#39;project/NN&#39;,
 &#39;(/(&#39;,
 &#39;HGP/NNP&#39;,
 &#39;)/)&#39;,
 &#39;was/VBD&#39;,
 &#39;begun/VBN&#39;,
 &#39;in/IN&#39;,
 &#39;1990/CD&#39;,
 &#39;,/,&#39;,
 &#39;and/CC&#39;,
 &#39;the/DT&#39;,
 &#39;human/JJ&#39;,
 &#39;genome/NN&#39;,
 &#39;was/VBD&#39;,
 &#39;completely/RB&#39;,
 &#39;released/VBN&#39;,
 &#39;in/IN&#39;,
 &#39;2003/CD&#39;,
 &#39;,/,&#39;,
 &#39;while/IN&#39;,
 &#39;further/JJ&#39;,
 &#39;analysis/NN&#39;,
 &#39;is/VBZ&#39;,
 &#39;still/RB&#39;,
 &#39;being/VBG&#39;,
 &#39;published/VBN&#39;,
 &#39;./.&#39;,
 &#39;A/DT&#39;,
 &#39;total/NN&#39;,
 &#39;of/IN&#39;,
 &#39;about/IN&#39;,
 &#39;3/CD&#39;,
 &#39;billion/CD&#39;,
 &#39;dollars/NNS&#39;,
 &#39;was/VBD&#39;,
 &#39;invested/VBN&#39;,
 &#39;to/TO&#39;,
 &#39;the/DT&#39;,
 &#39;project/NN&#39;,
 &#39;./.&#39;,
 &#39;In/IN&#39;,
 &#39;1991/CD&#39;,
 &#39;,/,&#39;,
 &#39;the/DT&#39;,
 &#39;National/NNP&#39;,
 &#39;Human/NNP&#39;,
 &#39;Genome/NNP&#39;,
 &#39;Research/NNP&#39;,
 &#39;Institute/NNP&#39;,
 &#39;(/(&#39;,
 &#39;NHGRI/NNP&#39;,
 &#39;)/)&#39;,
 &#39;funds/NNS&#39;,
 &#39;were/VBD&#39;,
 &#39;geared/VBN&#39;,
 &#39;toward/IN&#39;,
 &#39;lowering/VBG&#39;,
 &#39;the/DT&#39;,
 &#39;cost/NN&#39;,
 &#39;of/IN&#39;,
 &#39;DNA/NNP&#39;,
 &#39;sequencing/VBG&#39;,
 &#39;./.&#39;,
 &#39;Some/DT&#39;,
 &#39;of/IN&#39;,
 &#39;technologies/NNS&#39;,
 &#39;invested/VBN&#39;,
 &#39;improved/VBD&#39;,
 &#39;the/DT&#39;,
 &#39;DNA/NN&#39;,
 &#39;sequencing/VBG&#39;,
 &#39;./.&#39;,
 &#39;To/TO&#39;,
 &#39;date/NN&#39;,
 &#39;,/,&#39;,
 &#39;the/DT&#39;,
 &#39;Applied/NNP&#39;,
 &#39;Biosystems/NNP&#39;,
 &#39;,/,&#39;,
 &#39;Roche/454/NNP&#39;,
 &#39;,/,&#39;,
 &#39;and/CC&#39;,
 &#39;Illumina/Solexa/NNP&#39;,
 &#39;have/VBP&#39;,
 &#39;successfully/RB&#39;,
 &#39;developed/VBN&#39;,
 &#39;their/PRP$&#39;,
 &#39;technology/NN&#39;,
 &#39;and/CC&#39;,
 &#39;applied/VBD&#39;,
 &#39;DNA/NNP&#39;,
 &#39;sequencing/VBG&#39;,
 &#39;in/IN&#39;,
 &#39;the/DT&#39;,
 &#39;world/NN&#39;,
 &#39;during/IN&#39;,
 &#39;the/DT&#39;,
 &#39;recent/JJ&#39;,
 &#39;6/CD&#39;,
 &#39;years/NNS&#39;,
 &#39;./.&#39;,
 &#39;Most/JJS&#39;,
 &#39;of/IN&#39;,
 &#39;the/DT&#39;,
 &#39;newest/JJS&#39;,
 &#39;technologies/NNS&#39;,
 &#39;currently/RB&#39;,
 &#39;in/IN&#39;,
 &#39;use/NN&#39;,
 &#39;generate/NN&#39;,
 &#39;sequences/NNS&#39;,
 &#39;from/IN&#39;,
 &#39;36/CD&#39;,
 &#39;to/TO&#39;,
 &#39;1,000/CD&#39;,
 &#39;base/NN&#39;,
 &#39;pairs/NN&#39;,
 &#39;,/,&#39;,
 &#39;which/WDT&#39;,
 &#39;requires/VBZ&#39;,
 &#39;special/JJ&#39;,
 &#39;software/NN&#39;,
 &#39;for/IN&#39;,
 &#39;different/JJ&#39;,
 &#39;applications/NNS&#39;,
 &#39;,/,&#39;,
 &#39;including/VBG&#39;,
 &#39;whole-genome/JJ&#39;,
 &#39;sequencing/NN&#39;,
 &#39;,/,&#39;,
 &#39;transcriptome/JJ&#39;,
 &#39;analysis/NN&#39;,
 &#39;,/,&#39;,
 &#39;and/CC&#39;,
 &#39;regulatory/JJ&#39;,
 &#39;gene/NN&#39;,
 &#39;analysis/NN&#39;,
 &#39;./.&#39;,
 &#39;In/IN&#39;,
 &#39;particular/JJ&#39;,
 &#39;,/,&#39;,
 &#39;in/IN&#39;,
 &#39;silico/JJ&#39;,
 &#39;method/NN&#39;,
 &#39;development/NN&#39;,
 &#39;using/VBG&#39;,
 &#39;bioinformation/NN&#39;,
 &#39;software/NN&#39;,
 &#39;for/IN&#39;,
 &#39;next-generation/JJ&#39;,
 &#39;sequence/NN&#39;,
 &#39;assembly/NN&#39;,
 &#39;would/MD&#39;,
 &#39;be/VB&#39;,
 &#39;alert/JJ&#39;,
 &#39;for/IN&#39;,
 &#39;many/JJ&#39;,
 &#39;genome/JJ&#39;,
 &#39;projects/NNS&#39;,
 &#39;and/CC&#39;,
 &#39;more/JJR&#39;,
 &#39;applications/NNS&#39;,
 &#39;in/IN&#39;,
 &#39;genome/JJ&#39;,
 &#39;biology/NN&#39;,
 &#39;./.&#39;,
 &#39;Particularly/NNP&#39;,
 &#39;,/,&#39;,
 &#39;many/JJ&#39;,
 &#39;biologists/NNS&#39;,
 &#39;and/CC&#39;,
 &#39;geneticists/NNS&#39;,
 &#39;using/VBG&#39;,
 &#39;massive/JJ&#39;,
 &#39;DNA/NN&#39;,
 &#39;and/CC&#39;,
 &#39;RNA/NNP&#39;,
 &#39;sequences/NNS&#39;,
 &#39;have/VBP&#39;,
 &#39;used/VBN&#39;,
 &#39;the/DT&#39;,
 &#39;sequencing/NN&#39;,
 &#39;applications/NNS&#39;,
 &#39;focused/VBD&#39;,
 &#39;on/IN&#39;,
 &#39;the/DT&#39;,
 &#39;variable/JJ&#39;,
 &#39;research/NN&#39;,
 &#39;fields/NNS&#39;,
 &#39;in/IN&#39;,
 &#39;medicine/NN&#39;,
 &#39;,/,&#39;,
 &#39;such/JJ&#39;,
 &#39;as/IN&#39;,
 &#39;improved/JJ&#39;,
 &#39;diagnosis/NN&#39;,
 &#39;of/IN&#39;,
 &#39;disease/NN&#39;,
 &#39;;/:&#39;,
 &#39;gene/NN&#39;,
 &#39;therapy/NN&#39;,
 &#39;;/:&#39;,
 &#39;control/NN&#39;,
 &#39;systems/NNS&#39;,
 &#39;for/IN&#39;,
 &#39;drugs/NNS&#39;,
 &#39;,/,&#39;,
 &#39;including/VBG&#39;,
 &#39;pharmacogenomics/NNS&#39;,
 &#39;``/``&#39;,
 &#39;custom/JJ&#39;,
 &#39;drugs/NNS&#39;,
 &#39;;/:&#39;,
 &#34;&#39;&#39;/&#39;&#39;&#34;,
 &#39;evolution/NN&#39;,
 &#39;;/:&#39;,
 &#39;bioenergy/NN&#39;,
 &#39;and/CC&#39;,
 &#39;environmental/JJ&#39;,
 &#39;applications/NNS&#39;,
 &#39;of/IN&#39;,
 &#39;creative/JJ&#39;,
 &#39;energy/NN&#39;,
 &#39;sources/NNS&#39;,
 &#39;(/(&#39;,
 &#39;biofuels/NNS&#39;,
 &#39;)/)&#39;,
 &#39;;/:&#39;,
 &#39;clean/VBZ&#39;,
 &#39;up/IN&#39;,
 &#39;of/IN&#39;,
 &#39;toxic/NN&#39;,
 &#39;wastes/NNS&#39;,
 &#39;,/,&#39;,
 &#39;including/VBG&#39;,
 &#39;efficient/JJ&#39;,
 &#39;environmental/JJ&#39;,
 &#39;sources/NNS&#39;,
 &#39;against/IN&#39;,
 &#39;carbon/NN&#39;,
 &#39;;/:&#39;,
 &#39;and/CC&#39;,
 &#39;agriculture/NN&#39;,
 &#39;projects/NNS&#39;,
 &#39;,/,&#39;,
 &#39;including/VBG&#39;,
 &#39;livestock/NN&#39;,
 &#39;of/IN&#39;,
 &#39;healthier/JJR&#39;,
 &#39;,/,&#39;,
 &#39;more/RBR&#39;,
 &#39;productive/JJ&#39;,
 &#39;,/,&#39;,
 &#39;disease-resistant/JJ&#39;,
 &#39;farm/NN&#39;,
 &#39;animals/NNS&#39;,
 &#39;and/CC&#39;,
 &#39;breeding/NN&#39;,
 &#39;of/IN&#39;,
 &#39;disease-/JJ&#39;,
 &#39;,/,&#39;,
 &#39;insect-/JJ&#39;,
 &#39;,/,&#39;,
 &#39;and/CC&#39;,
 &#39;drought-resistant/JJ&#39;,
 &#39;crops/NNS&#39;,
 &#39;./.&#39;,
 &#39;In/IN&#39;,
 &#39;this/DT&#39;,
 &#39;paper/NN&#39;,
 &#39;,/,&#39;,
 &#39;we/PRP&#39;,
 &#39;report/VBP&#39;,
 &#39;several/JJ&#39;,
 &#39;ways/NNS&#39;,
 &#39;of/IN&#39;,
 &#39;genome/NN&#39;,
 &#39;sequencing/NN&#39;,
 &#39;and/CC&#39;,
 &#39;expression/NN&#39;,
 &#39;profiling/NN&#39;,
 &#39;in/IN&#39;,
 &#39;genome/JJ&#39;,
 &#39;biology/NN&#39;,
 &#39;./.&#39;,
 &#39;Title/NN&#39;,
 &#39;:/:&#39;,
 &#39;Roche/454/NNP&#39;,
 &#39;pyrosequencing/VBG&#39;,
 &#39;technology/NN&#39;,
 &#39;The/DT&#39;,
 &#39;first/JJ&#39;,
 &#39;high-parallel/JJ&#39;,
 &#39;sequencing/NN&#39;,
 &#39;system/NN&#39;,
 &#39;was/VBD&#39;,
 &#39;developed/VBN&#39;,
 &#39;with/IN&#39;,
 &#39;an/DT&#39;,
 &#39;emulsion/NN&#39;,
 &#39;PCR/NNP&#39;,
 &#39;method/NN&#39;,
 &#39;for/IN&#39;,
 &#39;DNA/NNP&#39;,
 &#39;amplification/NN&#39;,
 &#39;and/CC&#39;,
 &#39;an/DT&#39;,
 &#39;instrument/NN&#39;,
 &#39;for/IN&#39;,
 &#39;sequencing/VBG&#39;,
 &#39;by/IN&#39;,
 &#39;synthesis/NN&#39;,
 &#39;using/VBG&#39;,
 &#39;a/DT&#39;,
 &#39;pyrosequencing/NN&#39;,
 &#39;protocol/NN&#39;,
 &#39;optimized/VBN&#39;,
 &#39;on/IN&#39;,
 &#39;the/DT&#39;,
 &#39;individual/JJ&#39;,
 &#39;well/NN&#39;,
 &#39;of/IN&#39;,
 &#39;a/DT&#39;,
 &#39;PicoTiterPlate/NNP&#39;,
 &#39;(/(&#39;,
 &#39;PTP/NNP&#39;,
 &#39;)/)&#39;,
 &#39;[/VBD&#39;,
 &#39;1/CD&#39;,
 &#39;]/NN&#39;,
 &#39;./.&#39;,
 &#39;The/DT&#39;,
 &#39;DNA/NN&#39;,
 &#39;sequencing/VBG&#39;,
 &#39;protocol/NN&#39;,
 &#39;,/,&#39;,
 &#39;including/VBG&#39;,
 &#39;sample/JJ&#39;,
 &#39;preparation/NN&#39;,
 &#39;,/,&#39;,
 &#39;is/VBZ&#39;,
 &#39;supplied/VBN&#39;,
 &#39;for/IN&#39;,
 &#39;a/DT&#39;,
 &#39;user/NN&#39;,
 &#39;to/TO&#39;,
 &#39;follow/VB&#39;,
 &#39;the/DT&#39;,
 &#39;method/NN&#39;,
 &#39;,/,&#39;,
 &#39;which/WDT&#39;,
 &#39;is/VBZ&#39;,
 &#39;developed/VBN&#39;,
 &#39;by/IN&#39;,
 &#39;454/CD&#39;,
 &#39;Life/NNP&#39;,
 &#39;Science/NNP&#39;,
 &#39;(/(&#39;,
 &#39;now/RB&#39;,
 &#39;part/NN&#39;,
 &#39;of/IN&#39;,
 &#39;Roche/NNP&#39;,
 &#39;Diagnostics/NNP&#39;,
 &#39;,/,&#39;,
 &#39;Mannheim/NNP&#39;,
 &#39;,/,&#39;,
 &#39;Germany/NNP&#39;,
 &#39;)/)&#39;,
 &#39;./.&#39;,
 &#39;Even/RB&#39;,
 &#39;following/VBG&#39;,
 &#39;the/DT&#39;,
 &#39;manufacturer/NN&#39;,
 &#34;&#39;s/POS&#34;,
 &#39;protocol/NN&#39;,
 &#39;,/,&#39;,
 &#39;to/TO&#39;,
 &#39;obtain/VB&#39;,
 &#39;high-quality/JJ&#39;,
 &#39;DNA/NNP&#39;,
 &#39;sequence/NN&#39;,
 &#39;with/IN&#39;,
 &#39;maximal/JJ&#39;,
 &#39;total/JJ&#39;,
 &#39;DNA/NNP&#39;,
 &#39;sequence/NN&#39;,
 &#39;length/NN&#39;,
 &#39;as/IN&#39;,
 &#39;possible/JJ&#39;,
 &#39;,/,&#39;,
 &#39;one/CD&#39;,
 &#39;should/MD&#39;,
 &#39;make/VB&#39;,
 &#39;a/DT&#39;,
 &#39;library/NN&#39;,
 &#39;with/IN&#39;,
 &#39;both/DT&#39;,
 &#39;adapters/NNS&#39;,
 &#39;on/IN&#39;,
 &#39;the/DT&#39;,
 &#39;sheared/JJ&#39;,
 &#39;DNA/NNP&#39;,
 &#39;fragment/NN&#39;,
 &#39;and/CC&#39;,
 &#39;mix/VB&#39;,
 &#39;the/DT&#39;,
 &#39;optical/JJ&#39;,
 &#39;ratio/NN&#39;,
 &#39;of/IN&#39;,
 &#39;library/JJ&#39;,
 &#39;DNA/NNP&#39;,
 &#39;versus/NN&#39;,
 &#39;bead/NN&#39;,
 &#39;for/IN&#39;,
 &#39;emulsion/NN&#39;,
 &#39;PCR/NNP&#39;,
 &#39;./.&#39;,
 &#39;Now/RB&#39;,
 &#39;,/,&#39;,
 &#39;the/DT&#39;,
 &#39;system/NN&#39;,
 &#39;is/VBZ&#39;,
 &#39;upgraded/VBN&#39;,
 &#39;to/TO&#39;,
 &#39;through-put/VB&#39;,
 &#39;a/DT&#39;,
 &#39;total/NN&#39;,
 &#39;of/IN&#39;,
 &#39;an/DT&#39;,
 &#39;average/NN&#39;,
 &#39;of/IN&#39;,
 &#39;700/CD&#39;,
 &#39;Mbp/NNP&#39;,
 &#39;with/IN&#39;,
 &#39;an/DT&#39;,
 &#39;average/JJ&#39;,
 &#39;600/CD&#39;,
 &#39;bp/read/NN&#39;,
 &#39;in/IN&#39;,
 &#39;one/CD&#39;,
 &#39;PTP/NNP&#39;,
 &#39;run/NN&#39;,
 &#39;./.&#39;,
 &#39;The/DT&#39;,
 &#39;sequencer/NN&#39;,
 &#39;is/VBZ&#39;,
 &#39;available/JJ&#39;,
 &#39;for/IN&#39;,
 &#39;single/JJ&#39;,
 &#39;reads/NNS&#39;,
 &#39;and/CC&#39;,
 &#39;paired-end/JJ&#39;,
 &#39;read/NN&#39;,
 &#39;sequencing/VBG&#39;,
 &#39;for/IN&#39;,
 &#39;the/DT&#39;,
 &#39;application/NN&#39;,
 &#39;of/IN&#39;,
 &#39;eukaryotic/JJ&#39;,
 &#39;and/CC&#39;,
 &#39;prokaryotic/JJ&#39;,
 &#39;whole-genome/JJ&#39;,
 &#39;sequencing/VBG&#39;,
 &#39;[/JJ&#39;,
 &#39;2/CD&#39;,
 &#39;,/,&#39;,
 &#39;3/CD&#39;,
 &#39;]/NN&#39;,
 &#39;,/,&#39;,
 &#39;metagenomics/NNS&#39;,
 &#39;and/CC&#39;,
 &#39;microbial/JJ&#39;,
 &#39;diversity/NN&#39;,
 &#39;[/NNP&#39;,
 &#39;4/CD&#39;,
 &#39;,/,&#39;,
 &#39;5/CD&#39;,
 &#39;]/NN&#39;,
 &#39;,/,&#39;,
 &#39;and/CC&#39;,
 &#39;genetic/JJ&#39;,
 &#39;variation/NN&#39;,
 &#39;detection/NN&#39;,
 &#39;for/IN&#39;,
 &#39;comparative/JJ&#39;,
 &#39;genomics/NNS&#39;,
 &#39;[/VBP&#39;,
 &#39;6/CD&#39;,
 &#39;]/NN&#39;,
 &#39;./.&#39;,
 &#39;Title/NN&#39;,
 &#39;:/:&#39;,
 &#39;Illumina/Solexa/NNP&#39;,
 &#39;technology/NN&#39;,
 &#39;The/DT&#39;,
 &#39;numerous/JJ&#39;,
 &#39;cost-effective/JJ&#39;,
 &#39;technologies/NNS&#39;,
 &#39;were/VBD&#39;,
 &#39;being/VBG&#39;,
 &#39;developed/VBN&#39;,
 &#39;for/IN&#39;,
 &#39;human/JJ&#39;,
 &#39;genome/NN&#39;,
 &#39;resequencing/NN&#39;,
 &#39;that/WDT&#39;,
 &#39;can/MD&#39;,
 &#39;be/VB&#39;,
 &#39;aligned/VBN&#39;,
 &#39;to/TO&#39;,
 &#39;the/DT&#39;,
 &#39;reference/NN&#39;,
 &#39;sequence/NN&#39;,
 &#39;[/VBD&#39;,
 &#39;7/CD&#39;,
 &#39;,/,&#39;,
 &#39;8/CD&#39;,
 &#39;]/NN&#39;,
 &#39;./.&#39;,
 &#39;The/DT&#39;,
 &#39;first/JJ&#39;,
 &#39;successful/JJ&#39;,
 &#39;technology/NN&#39;,
 &#39;to/TO&#39;,
 &#39;gain/VB&#39;,
 &#39;massive/JJ&#39;,
 &#39;DNA/NNP&#39;,
 &#39;sequencing/NN&#39;,
 &#39;available/JJ&#39;,
 &#39;for/IN&#39;,
 &#39;resequencing/VBG&#39;,
 &#39;was/VBD&#39;,
 &#39;developed/VBN&#39;,
 &#39;by/IN&#39;,
 &#39;Solexa/NNP&#39;,
 &#39;(/(&#39;,
 &#39;now/RB&#39;,
 &#39;part/NN&#39;,
 &#39;of/IN&#39;,
 &#39;Illumina/NNP&#39;,
 &#39;,/,&#39;,
 &#39;San/NNP&#39;,
 &#39;Diego/NNP&#39;,
 &#39;,/,&#39;,
 &#39;CA/NNP&#39;,
 &#39;,/,&#39;,
 &#39;USA/NNP&#39;,
 &#39;)/)&#39;,
 &#39;./.&#39;,
 &#39;The/DT&#39;,
 &#39;principle/NN&#39;,
 &#39;of/IN&#39;,
 &#39;the/DT&#39;,
 &#39;system/NN&#39;,
 &#39;is/VBZ&#39;,
 &#39;the/DT&#39;,
 &#39;method/NN&#39;,
 &#39;of/IN&#39;,
 &#39;base-by-base/NN&#39;,
 &#39;sequencing/NN&#39;,
 &#39;by/IN&#39;,
 &#39;synthesis/NN&#39;,
 &#39;,/,&#39;,
 &#39;where/WRB&#39;,
 &#39;the/DT&#39;,
 &#39;sheared/JJ&#39;,
 &#39;template/NN&#39;,
 &#39;DNA/NNP&#39;,
 &#39;is/VBZ&#39;,
 &#39;amplified/VBN&#39;,
 &#39;on/IN&#39;,
 &#39;the/DT&#39;,
 &#39;flat/JJ&#39;,
 &#39;surface/NN&#39;,
 &#39;slide/NN&#39;,
 &#39;(/(&#39;,
 &#39;flow/JJ&#39;,
 &#39;cell/NN&#39;,
 &#39;)/)&#39;,
 &#39;and/CC&#39;,
 &#39;detects/VBZ&#39;,
 &#39;one/CD&#39;,
 &#39;base/NN&#39;,
 &#39;on/IN&#39;,
 &#39;each/DT&#39;,
 &#39;template/NN&#39;,
 &#39;per/IN&#39;,
 &#39;cycle/NN&#39;,
 &#39;with/IN&#39;,
 &#39;four/CD&#39;,
 &#39;base-specific/NN&#39;,
 &#39;,/,&#39;,
 &#39;fluorescently/RB&#39;,
 &#39;labeled/VBN&#39;,
 &#39;signals/NNS&#39;,
 &#39;./.&#39;,
 &#39;Signals/NNS&#39;,
 &#39;for/IN&#39;,
 &#39;all/DT&#39;,
 &#39;four/CD&#39;,
 &#39;fluorescent/NN&#39;,
 &#39;channels/NNS&#39;,
 &#39;are/VBP&#39;,
 &#39;collected/VBN&#39;,
 &#39;and/CC&#39;,
 &#39;plotted/VBN&#39;,
 &#39;at/IN&#39;,
 &#39;each/DT&#39;,
 &#39;position/NN&#39;,
 &#39;,/,&#39;,
 &#39;enabling/VBG&#39;,
 &#39;quality/NN&#39;,
 &#39;per/IN&#39;,
 &#39;scores/NNS&#39;,
 &#39;to/TO&#39;,
 &#39;be/VB&#39;,
 &#39;derived/VBN&#39;,
 &#39;using/VBG&#39;,
 &#39;four-color/JJ&#39;,
 &#39;information/NN&#39;,
 &#39;if/IN&#39;,
 &#39;desired/VBN&#39;,
 &#39;[/VBP&#39;,
 &#39;8/CD&#39;,
 &#39;]/NNS&#39;,
 &#39;./.&#39;,
 &#39;Now/RB&#39;,
 &#39;,/,&#39;,
 &#39;a/DT&#39;,
 &#39;maximum/NN&#39;,
 &#39;of/IN&#39;,
 &#39;300/CD&#39;,
 &#39;Gb/NNP&#39;,
 &#39;for/IN&#39;,
 &#39;reading/NN&#39;,
 &#39;is/VBZ&#39;,
 &#39;available/JJ&#39;,
 &#39;with/IN&#39;,
 &#39;101/CD&#39;,
 &#39;bp/JJ&#39;,
 &#39;paired-end/JJ&#39;,
 &#39;reading/NN&#39;,
 &#39;per/IN&#39;,
 &#39;fragment/NN&#39;,
 ...]</pre>
</div>

</div>

</div>

</div>

</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell   ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[78]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># [], {}, (), &lt;&gt; 괄호와 괄호 안 문자 제거하기</span>
<span class="n">pattern</span> <span class="o">=</span> <span class="sa">r</span><span class="s1">&#39;\([^)]*\)&#39;</span>
<span class="n">text</span> <span class="o">=</span> <span class="s1">&#39;&#39;</span>
<span class="k">for</span> <span class="n">i</span> <span class="ow">in</span> <span class="nb">range</span><span class="p">(</span> <span class="nb">len</span><span class="p">(</span><span class="n">raw</span><span class="p">)</span> <span class="p">)</span> <span class="p">:</span>
    <span class="n">text</span> <span class="o">+=</span> <span class="n">re</span><span class="o">.</span><span class="n">sub</span><span class="p">(</span><span class="n">pattern</span><span class="o">=</span><span class="n">pattern</span><span class="p">,</span> <span class="n">repl</span><span class="o">=</span><span class="s1">&#39;&#39;</span><span class="p">,</span> <span class="n">string</span><span class="o">=</span> <span class="n">raw</span><span class="p">[</span><span class="n">i</span><span class="p">])</span>
<span class="n">text</span><span class="o">.</span><span class="n">split</span><span class="p">()</span>
</pre></div>

     </div>
</div>
</div>
</div>

<div class="jp-Cell-outputWrapper">


<div class="jp-OutputArea jp-Cell-outputArea">

<div class="jp-OutputArea-child">

    
    <div class="jp-OutputPrompt jp-OutputArea-prompt">Out[78]:</div>




<div class="jp-RenderedText jp-OutputArea-output jp-OutputArea-executeResult" data-mime-type="text/plain">
<pre>[&#39;Title:&#39;,
 &#39;Survey&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;Applications&#39;,
 &#39;of&#39;,
 &#39;NGS&#39;,
 &#39;to&#39;,
 &#39;Whole-Genome&#39;,
 &#39;Sequencing&#39;,
 &#39;and&#39;,
 &#39;Expression&#39;,
 &#39;Profiling&#39;,
 &#39;Recently,&#39;,
 &#39;the&#39;,
 &#39;technologies&#39;,
 &#39;of&#39;,
 &#39;DNA&#39;,
 &#39;sequence&#39;,
 &#39;variation&#39;,
 &#39;and&#39;,
 &#39;gene&#39;,
 &#39;expression&#39;,
 &#39;profiling&#39;,
 &#39;have&#39;,
 &#39;been&#39;,
 &#39;used&#39;,
 &#39;widely&#39;,
 &#39;as&#39;,
 &#39;approaches&#39;,
 &#39;in&#39;,
 &#39;the&#39;,
 &#39;expertise&#39;,
 &#39;of&#39;,
 &#39;genome&#39;,
 &#39;biology&#39;,
 &#39;and&#39;,
 &#39;genetics.&#39;,
 &#39;The&#39;,
 &#39;application&#39;,
 &#39;to&#39;,
 &#39;genome&#39;,
 &#39;study&#39;,
 &#39;has&#39;,
 &#39;been&#39;,
 &#39;particularly&#39;,
 &#39;developed&#39;,
 &#39;with&#39;,
 &#39;the&#39;,
 &#39;introduction&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;next-generation&#39;,
 &#39;DNA&#39;,
 &#39;sequencer&#39;,
 &#39;Roche/454&#39;,
 &#39;and&#39;,
 &#39;Illumina/Solexa&#39;,
 &#39;systems,&#39;,
 &#39;along&#39;,
 &#39;with&#39;,
 &#39;bioinformation&#39;,
 &#39;analysis&#39;,
 &#39;technologies&#39;,
 &#39;of&#39;,
 &#39;whole-genome&#39;,
 &#39;de&#39;,
 &#39;novo&#39;,
 &#39;assembly,&#39;,
 &#39;expression&#39;,
 &#39;profiling,&#39;,
 &#39;DNA&#39;,
 &#39;variation&#39;,
 &#39;discovery,&#39;,
 &#39;and&#39;,
 &#39;genotyping.&#39;,
 &#39;Both&#39;,
 &#39;massive&#39;,
 &#39;whole-genome&#39;,
 &#39;shotgun&#39;,
 &#39;paired-end&#39;,
 &#39;sequencing&#39;,
 &#39;and&#39;,
 &#39;mate&#39;,
 &#39;paired-end&#39;,
 &#39;sequencing&#39;,
 &#39;data&#39;,
 &#39;are&#39;,
 &#39;important&#39;,
 &#39;steps&#39;,
 &#39;for&#39;,
 &#39;constructing&#39;,
 &#39;de&#39;,
 &#39;novo&#39;,
 &#39;assembly&#39;,
 &#39;of&#39;,
 &#39;novel&#39;,
 &#39;genome&#39;,
 &#39;sequencing&#39;,
 &#39;data.&#39;,
 &#39;It&#39;,
 &#39;is&#39;,
 &#39;necessary&#39;,
 &#39;to&#39;,
 &#39;have&#39;,
 &#39;DNA&#39;,
 &#39;sequence&#39;,
 &#39;information&#39;,
 &#39;from&#39;,
 &#39;a&#39;,
 &#39;multiplatform&#39;,
 &#39;NGS&#39;,
 &#39;with&#39;,
 &#39;at&#39;,
 &#39;least&#39;,
 &#39;2×&#39;,
 &#39;and&#39;,
 &#39;30×&#39;,
 &#39;depth&#39;,
 &#39;sequence&#39;,
 &#39;of&#39;,
 &#39;genome&#39;,
 &#39;coverage&#39;,
 &#39;using&#39;,
 &#39;Roche/454&#39;,
 &#39;and&#39;,
 &#39;Illumina/Solexa,&#39;,
 &#39;respectively,&#39;,
 &#39;for&#39;,
 &#39;effective&#39;,
 &#39;an&#39;,
 &#39;way&#39;,
 &#39;of&#39;,
 &#39;de&#39;,
 &#39;novo&#39;,
 &#39;assembly.&#39;,
 &#39;Massive&#39;,
 &#39;short-length&#39;,
 &#39;reading&#39;,
 &#39;data&#39;,
 &#39;from&#39;,
 &#39;the&#39;,
 &#39;Illumina/Solexa&#39;,
 &#39;system&#39;,
 &#39;is&#39;,
 &#39;enough&#39;,
 &#39;to&#39;,
 &#39;discover&#39;,
 &#39;DNA&#39;,
 &#39;variation,&#39;,
 &#39;resulting&#39;,
 &#39;in&#39;,
 &#39;reducing&#39;,
 &#39;the&#39;,
 &#39;cost&#39;,
 &#39;of&#39;,
 &#39;DNA&#39;,
 &#39;sequencing.&#39;,
 &#39;Whole-genome&#39;,
 &#39;expression&#39;,
 &#39;profile&#39;,
 &#39;data&#39;,
 &#39;are&#39;,
 &#39;useful&#39;,
 &#39;to&#39;,
 &#39;approach&#39;,
 &#39;genome&#39;,
 &#39;system&#39;,
 &#39;biology&#39;,
 &#39;with&#39;,
 &#39;quantification&#39;,
 &#39;of&#39;,
 &#39;expressed&#39;,
 &#39;RNAs&#39;,
 &#39;from&#39;,
 &#39;a&#39;,
 &#39;whole-genome&#39;,
 &#39;transcriptome,&#39;,
 &#39;depending&#39;,
 &#39;on&#39;,
 &#39;the&#39;,
 &#39;tissue&#39;,
 &#39;samples.&#39;,
 &#39;The&#39;,
 &#39;hybrid&#39;,
 &#39;mRNA&#39;,
 &#39;sequences&#39;,
 &#39;from&#39;,
 &#39;Rohce/454&#39;,
 &#39;and&#39;,
 &#39;Illumina/Solexa&#39;,
 &#39;are&#39;,
 &#39;more&#39;,
 &#39;powerful&#39;,
 &#39;to&#39;,
 &#39;find&#39;,
 &#39;novel&#39;,
 &#39;genes&#39;,
 &#39;through&#39;,
 &#39;de&#39;,
 &#39;novo&#39;,
 &#39;assembly&#39;,
 &#39;in&#39;,
 &#39;any&#39;,
 &#39;whole-genome&#39;,
 &#39;sequenced&#39;,
 &#39;species.&#39;,
 &#39;The&#39;,
 &#39;20×&#39;,
 &#39;and&#39;,
 &#39;50×&#39;,
 &#39;coverage&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;estimated&#39;,
 &#39;transcriptome&#39;,
 &#39;sequences&#39;,
 &#39;using&#39;,
 &#39;Roche/454&#39;,
 &#39;and&#39;,
 &#39;Illumina/Solexa,&#39;,
 &#39;respectively,&#39;,
 &#39;is&#39;,
 &#39;effective&#39;,
 &#39;to&#39;,
 &#39;create&#39;,
 &#39;novel&#39;,
 &#39;expressed&#39;,
 &#39;reference&#39;,
 &#39;sequences.&#39;,
 &#39;However,&#39;,
 &#39;only&#39;,
 &#39;an&#39;,
 &#39;average&#39;,
 &#39;30×&#39;,
 &#39;coverage&#39;,
 &#39;of&#39;,
 &#39;a&#39;,
 &#39;transcriptome&#39;,
 &#39;with&#39;,
 &#39;short&#39;,
 &#39;read&#39;,
 &#39;sequences&#39;,
 &#39;of&#39;,
 &#39;Illumina/Solexa&#39;,
 &#39;is&#39;,
 &#39;enough&#39;,
 &#39;to&#39;,
 &#39;check&#39;,
 &#39;expression&#39;,
 &#39;quantification,&#39;,
 &#39;compared&#39;,
 &#39;to&#39;,
 &#39;the&#39;,
 &#39;reference&#39;,
 &#39;expressed&#39;,
 &#39;sequence&#39;,
 &#39;tag&#39;,
 &#39;sequence.&#39;,
 &#39;Scientists&#39;,
 &#39;have&#39;,
 &#39;tried&#39;,
 &#39;to&#39;,
 &#39;understand&#39;,
 &#39;biology&#39;,
 &#39;through&#39;,
 &#39;DNA&#39;,
 &#39;sequence&#39;,
 &#39;information,&#39;,
 &#39;since&#39;,
 &#39;the&#39;,
 &#39;DNA&#39;,
 &#39;is&#39;,
 &#39;verified&#39;,
 &#39;as&#39;,
 &#39;the&#39;,
 &#39;unit&#39;,
 &#39;of&#39;,
 &#39;genetic&#39;,
 &#39;heredity.&#39;,
 &#39;Also,&#39;,
 &#39;scientists&#39;,
 &#39;hope&#39;,
 &#39;to&#39;,
 &#39;dramatically&#39;,
 &#39;reduce&#39;,
 &#39;the&#39;,
 &#39;cost&#39;,
 &#39;of&#39;,
 &#39;reading&#39;,
 &#39;genomic&#39;,
 &#39;DNA&#39;,
 &#39;and&#39;,
 &#39;to&#39;,
 &#39;obtain&#39;,
 &#39;the&#39;,
 &#39;high-throughput&#39;,
 &#39;DNA&#39;,
 &#39;sequence&#39;,
 &#39;information.&#39;,
 &#39;Since&#39;,
 &#39;the&#39;,
 &#39;automated&#39;,
 &#39;DNA&#39;,
 &#39;sequencers&#39;,
 &#39;were&#39;,
 &#39;developed&#39;,
 &#39;with&#39;,
 &#39;fluorescent&#39;,
 &#39;dyes&#39;,
 &#39;of&#39;,
 &#39;different&#39;,
 &#39;colors,&#39;,
 &#39;laser,&#39;,
 &#39;and&#39;,
 &#39;computer&#39;,
 &#39;technology&#39;,
 &#39;in&#39;,
 &#39;the&#39;,
 &#39;1980s,&#39;,
 &#39;the&#39;,
 &#39;human&#39;,
 &#39;genome&#39;,
 &#39;project&#39;,
 &#39;was&#39;,
 &#39;begun&#39;,
 &#39;in&#39;,
 &#39;1990,&#39;,
 &#39;and&#39;,
 &#39;the&#39;,
 &#39;human&#39;,
 &#39;genome&#39;,
 &#39;was&#39;,
 &#39;completely&#39;,
 &#39;released&#39;,
 &#39;in&#39;,
 &#39;2003,&#39;,
 &#39;while&#39;,
 &#39;further&#39;,
 &#39;analysis&#39;,
 &#39;is&#39;,
 &#39;still&#39;,
 &#39;being&#39;,
 &#39;published.&#39;,
 &#39;A&#39;,
 &#39;total&#39;,
 &#39;of&#39;,
 &#39;about&#39;,
 &#39;3&#39;,
 &#39;billion&#39;,
 &#39;dollars&#39;,
 &#39;was&#39;,
 &#39;invested&#39;,
 &#39;to&#39;,
 &#39;the&#39;,
 &#39;project.&#39;,
 &#39;In&#39;,
 &#39;1991,&#39;,
 &#39;the&#39;,
 &#39;National&#39;,
 &#39;Human&#39;,
 &#39;Genome&#39;,
 &#39;Research&#39;,
 &#39;Institute&#39;,
 &#39;funds&#39;,
 &#39;were&#39;,
 &#39;geared&#39;,
 &#39;toward&#39;,
 &#39;lowering&#39;,
 &#39;the&#39;,
 &#39;cost&#39;,
 &#39;of&#39;,
 &#39;DNA&#39;,
 &#39;sequencing.&#39;,
 &#39;Some&#39;,
 &#39;of&#39;,
 &#39;technologies&#39;,
 &#39;invested&#39;,
 &#39;improved&#39;,
 &#39;the&#39;,
 &#39;DNA&#39;,
 &#39;sequencing.&#39;,
 &#39;To&#39;,
 &#39;date,&#39;,
 &#39;the&#39;,
 &#39;Applied&#39;,
 &#39;Biosystems,&#39;,
 &#39;Roche/454,&#39;,
 &#39;and&#39;,
 &#39;Illumina/Solexa&#39;,
 &#39;have&#39;,
 &#39;successfully&#39;,
 &#39;developed&#39;,
 &#39;their&#39;,
 &#39;technology&#39;,
 &#39;and&#39;,
 &#39;applied&#39;,
 &#39;DNA&#39;,
 &#39;sequencing&#39;,
 &#39;in&#39;,
 &#39;the&#39;,
 &#39;world&#39;,
 &#39;during&#39;,
 &#39;the&#39;,
 &#39;recent&#39;,
 &#39;6&#39;,
 &#39;years.&#39;,
 &#39;Most&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;newest&#39;,
 &#39;technologies&#39;,
 &#39;currently&#39;,
 &#39;in&#39;,
 &#39;use&#39;,
 &#39;generate&#39;,
 &#39;sequences&#39;,
 &#39;from&#39;,
 &#39;36&#39;,
 &#39;to&#39;,
 &#39;1,000&#39;,
 &#39;base&#39;,
 &#39;pairs,&#39;,
 &#39;which&#39;,
 &#39;requires&#39;,
 &#39;special&#39;,
 &#39;software&#39;,
 &#39;for&#39;,
 &#39;different&#39;,
 &#39;applications,&#39;,
 &#39;including&#39;,
 &#39;whole-genome&#39;,
 &#39;sequencing,&#39;,
 &#39;transcriptome&#39;,
 &#39;analysis,&#39;,
 &#39;and&#39;,
 &#39;regulatory&#39;,
 &#39;gene&#39;,
 &#39;analysis.&#39;,
 &#39;In&#39;,
 &#39;particular,&#39;,
 &#39;in&#39;,
 &#39;silico&#39;,
 &#39;method&#39;,
 &#39;development&#39;,
 &#39;using&#39;,
 &#39;bioinformation&#39;,
 &#39;software&#39;,
 &#39;for&#39;,
 &#39;next-generation&#39;,
 &#39;sequence&#39;,
 &#39;assembly&#39;,
 &#39;would&#39;,
 &#39;be&#39;,
 &#39;alert&#39;,
 &#39;for&#39;,
 &#39;many&#39;,
 &#39;genome&#39;,
 &#39;projects&#39;,
 &#39;and&#39;,
 &#39;more&#39;,
 &#39;applications&#39;,
 &#39;in&#39;,
 &#39;genome&#39;,
 &#39;biology.&#39;,
 &#39;Particularly,&#39;,
 &#39;many&#39;,
 &#39;biologists&#39;,
 &#39;and&#39;,
 &#39;geneticists&#39;,
 &#39;using&#39;,
 &#39;massive&#39;,
 &#39;DNA&#39;,
 &#39;and&#39;,
 &#39;RNA&#39;,
 &#39;sequences&#39;,
 &#39;have&#39;,
 &#39;used&#39;,
 &#39;the&#39;,
 &#39;sequencing&#39;,
 &#39;applications&#39;,
 &#39;focused&#39;,
 &#39;on&#39;,
 &#39;the&#39;,
 &#39;variable&#39;,
 &#39;research&#39;,
 &#39;fields&#39;,
 &#39;in&#39;,
 &#39;medicine,&#39;,
 &#39;such&#39;,
 &#39;as&#39;,
 &#39;improved&#39;,
 &#39;diagnosis&#39;,
 &#39;of&#39;,
 &#39;disease;&#39;,
 &#39;gene&#39;,
 &#39;therapy;&#39;,
 &#39;control&#39;,
 &#39;systems&#39;,
 &#39;for&#39;,
 &#39;drugs,&#39;,
 &#39;including&#39;,
 &#39;pharmacogenomics&#39;,
 &#39;&#34;custom&#39;,
 &#39;drugs;&#34;&#39;,
 &#39;evolution;&#39;,
 &#39;bioenergy&#39;,
 &#39;and&#39;,
 &#39;environmental&#39;,
 &#39;applications&#39;,
 &#39;of&#39;,
 &#39;creative&#39;,
 &#39;energy&#39;,
 &#39;sources&#39;,
 &#39;;&#39;,
 &#39;clean&#39;,
 &#39;up&#39;,
 &#39;of&#39;,
 &#39;toxic&#39;,
 &#39;wastes,&#39;,
 &#39;including&#39;,
 &#39;efficient&#39;,
 &#39;environmental&#39;,
 &#39;sources&#39;,
 &#39;against&#39;,
 &#39;carbon;&#39;,
 &#39;and&#39;,
 &#39;agriculture&#39;,
 &#39;projects,&#39;,
 &#39;including&#39;,
 &#39;livestock&#39;,
 &#39;of&#39;,
 &#39;healthier,&#39;,
 &#39;more&#39;,
 &#39;productive,&#39;,
 &#39;disease-resistant&#39;,
 &#39;farm&#39;,
 &#39;animals&#39;,
 &#39;and&#39;,
 &#39;breeding&#39;,
 &#39;of&#39;,
 &#39;disease-,&#39;,
 &#39;insect-,&#39;,
 &#39;and&#39;,
 &#39;drought-resistant&#39;,
 &#39;crops.&#39;,
 &#39;In&#39;,
 &#39;this&#39;,
 &#39;paper,&#39;,
 &#39;we&#39;,
 &#39;report&#39;,
 &#39;several&#39;,
 &#39;ways&#39;,
 &#39;of&#39;,
 &#39;genome&#39;,
 &#39;sequencing&#39;,
 &#39;and&#39;,
 &#39;expression&#39;,
 &#39;profiling&#39;,
 &#39;in&#39;,
 &#39;genome&#39;,
 &#39;biology.&#39;,
 &#39;Title:&#39;,
 &#39;Roche/454&#39;,
 &#39;pyrosequencing&#39;,
 &#39;technology&#39;,
 &#39;The&#39;,
 &#39;first&#39;,
 &#39;high-parallel&#39;,
 &#39;sequencing&#39;,
 &#39;system&#39;,
 &#39;was&#39;,
 &#39;developed&#39;,
 &#39;with&#39;,
 &#39;an&#39;,
 &#39;emulsion&#39;,
 &#39;PCR&#39;,
 &#39;method&#39;,
 &#39;for&#39;,
 &#39;DNA&#39;,
 &#39;amplification&#39;,
 &#39;and&#39;,
 &#39;an&#39;,
 &#39;instrument&#39;,
 &#39;for&#39;,
 &#39;sequencing&#39;,
 &#39;by&#39;,
 &#39;synthesis&#39;,
 &#39;using&#39;,
 &#39;a&#39;,
 &#39;pyrosequencing&#39;,
 &#39;protocol&#39;,
 &#39;optimized&#39;,
 &#39;on&#39;,
 &#39;the&#39;,
 &#39;individual&#39;,
 &#39;well&#39;,
 &#39;of&#39;,
 &#39;a&#39;,
 &#39;PicoTiterPlate&#39;,
 &#39;[1].&#39;,
 &#39;The&#39;,
 &#39;DNA&#39;,
 &#39;sequencing&#39;,
 &#39;protocol,&#39;,
 &#39;including&#39;,
 &#39;sample&#39;,
 &#39;preparation,&#39;,
 &#39;is&#39;,
 &#39;supplied&#39;,
 &#39;for&#39;,
 &#39;a&#39;,
 &#39;user&#39;,
 &#39;to&#39;,
 &#39;follow&#39;,
 &#39;the&#39;,
 &#39;method,&#39;,
 &#39;which&#39;,
 &#39;is&#39;,
 &#39;developed&#39;,
 &#39;by&#39;,
 &#39;454&#39;,
 &#39;Life&#39;,
 &#39;Science&#39;,
 &#39;.&#39;,
 &#39;Even&#39;,
 &#39;following&#39;,
 &#39;the&#39;,
 &#34;manufacturer&#39;s&#34;,
 &#39;protocol,&#39;,
 &#39;to&#39;,
 &#39;obtain&#39;,
 &#39;high-quality&#39;,
 &#39;DNA&#39;,
 &#39;sequence&#39;,
 &#39;with&#39;,
 &#39;maximal&#39;,
 &#39;total&#39;,
 &#39;DNA&#39;,
 &#39;sequence&#39;,
 &#39;length&#39;,
 &#39;as&#39;,
 &#39;possible,&#39;,
 &#39;one&#39;,
 &#39;should&#39;,
 &#39;make&#39;,
 &#39;a&#39;,
 &#39;library&#39;,
 &#39;with&#39;,
 &#39;both&#39;,
 &#39;adapters&#39;,
 &#39;on&#39;,
 &#39;the&#39;,
 &#39;sheared&#39;,
 &#39;DNA&#39;,
 &#39;fragment&#39;,
 &#39;and&#39;,
 &#39;mix&#39;,
 &#39;the&#39;,
 &#39;optical&#39;,
 &#39;ratio&#39;,
 &#39;of&#39;,
 &#39;library&#39;,
 &#39;DNA&#39;,
 &#39;versus&#39;,
 &#39;bead&#39;,
 &#39;for&#39;,
 &#39;emulsion&#39;,
 &#39;PCR.&#39;,
 &#39;Now,&#39;,
 &#39;the&#39;,
 &#39;system&#39;,
 &#39;is&#39;,
 &#39;upgraded&#39;,
 &#39;to&#39;,
 &#39;through-put&#39;,
 &#39;a&#39;,
 &#39;total&#39;,
 &#39;of&#39;,
 &#39;an&#39;,
 &#39;average&#39;,
 &#39;of&#39;,
 &#39;700&#39;,
 &#39;Mbp&#39;,
 &#39;with&#39;,
 &#39;an&#39;,
 &#39;average&#39;,
 &#39;600&#39;,
 &#39;bp/read&#39;,
 &#39;in&#39;,
 &#39;one&#39;,
 &#39;PTP&#39;,
 &#39;run.&#39;,
 &#39;The&#39;,
 &#39;sequencer&#39;,
 &#39;is&#39;,
 &#39;available&#39;,
 &#39;for&#39;,
 &#39;single&#39;,
 &#39;reads&#39;,
 &#39;and&#39;,
 &#39;paired-end&#39;,
 &#39;read&#39;,
 &#39;sequencing&#39;,
 &#39;for&#39;,
 &#39;the&#39;,
 &#39;application&#39;,
 &#39;of&#39;,
 &#39;eukaryotic&#39;,
 &#39;and&#39;,
 &#39;prokaryotic&#39;,
 &#39;whole-genome&#39;,
 &#39;sequencing&#39;,
 &#39;[2,&#39;,
 &#39;3],&#39;,
 &#39;metagenomics&#39;,
 &#39;and&#39;,
 &#39;microbial&#39;,
 &#39;diversity&#39;,
 &#39;[4,&#39;,
 &#39;5],&#39;,
 &#39;and&#39;,
 &#39;genetic&#39;,
 &#39;variation&#39;,
 &#39;detection&#39;,
 &#39;for&#39;,
 &#39;comparative&#39;,
 &#39;genomics&#39;,
 &#39;[6].&#39;,
 &#39;Title:&#39;,
 &#39;Illumina/Solexa&#39;,
 &#39;technology&#39;,
 &#39;The&#39;,
 &#39;numerous&#39;,
 &#39;cost-effective&#39;,
 &#39;technologies&#39;,
 &#39;were&#39;,
 &#39;being&#39;,
 &#39;developed&#39;,
 &#39;for&#39;,
 &#39;human&#39;,
 &#39;genome&#39;,
 &#39;resequencing&#39;,
 &#39;that&#39;,
 &#39;can&#39;,
 &#39;be&#39;,
 &#39;aligned&#39;,
 &#39;to&#39;,
 &#39;the&#39;,
 &#39;reference&#39;,
 &#39;sequence&#39;,
 &#39;[7,&#39;,
 &#39;8].&#39;,
 &#39;The&#39;,
 &#39;first&#39;,
 &#39;successful&#39;,
 &#39;technology&#39;,
 &#39;to&#39;,
 &#39;gain&#39;,
 &#39;massive&#39;,
 &#39;DNA&#39;,
 &#39;sequencing&#39;,
 &#39;available&#39;,
 &#39;for&#39;,
 &#39;resequencing&#39;,
 &#39;was&#39;,
 &#39;developed&#39;,
 &#39;by&#39;,
 &#39;Solexa&#39;,
 &#39;.&#39;,
 &#39;The&#39;,
 &#39;principle&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;system&#39;,
 &#39;is&#39;,
 &#39;the&#39;,
 &#39;method&#39;,
 &#39;of&#39;,
 &#39;base-by-base&#39;,
 &#39;sequencing&#39;,
 &#39;by&#39;,
 &#39;synthesis,&#39;,
 &#39;where&#39;,
 &#39;the&#39;,
 &#39;sheared&#39;,
 &#39;template&#39;,
 &#39;DNA&#39;,
 &#39;is&#39;,
 &#39;amplified&#39;,
 &#39;on&#39;,
 &#39;the&#39;,
 &#39;flat&#39;,
 &#39;surface&#39;,
 &#39;slide&#39;,
 &#39;and&#39;,
 &#39;detects&#39;,
 &#39;one&#39;,
 &#39;base&#39;,
 &#39;on&#39;,
 &#39;each&#39;,
 &#39;template&#39;,
 &#39;per&#39;,
 &#39;cycle&#39;,
 &#39;with&#39;,
 &#39;four&#39;,
 &#39;base-specific,&#39;,
 &#39;fluorescently&#39;,
 &#39;labeled&#39;,
 &#39;signals.&#39;,
 &#39;Signals&#39;,
 &#39;for&#39;,
 &#39;all&#39;,
 &#39;four&#39;,
 &#39;fluorescent&#39;,
 &#39;channels&#39;,
 &#39;are&#39;,
 &#39;collected&#39;,
 &#39;and&#39;,
 &#39;plotted&#39;,
 &#39;at&#39;,
 &#39;each&#39;,
 &#39;position,&#39;,
 &#39;enabling&#39;,
 &#39;quality&#39;,
 &#39;per&#39;,
 &#39;scores&#39;,
 &#39;to&#39;,
 &#39;be&#39;,
 &#39;derived&#39;,
 &#39;using&#39;,
 &#39;four-color&#39;,
 &#39;information&#39;,
 &#39;if&#39;,
 &#39;desired&#39;,
 &#39;[8].&#39;,
 &#39;Now,&#39;,
 &#39;a&#39;,
 &#39;maximum&#39;,
 &#39;of&#39;,
 &#39;300&#39;,
 &#39;Gb&#39;,
 &#39;for&#39;,
 &#39;reading&#39;,
 &#39;is&#39;,
 &#39;available&#39;,
 &#39;with&#39;,
 &#39;101&#39;,
 &#39;bp&#39;,
 &#39;paired-end&#39;,
 &#39;reading&#39;,
 &#39;per&#39;,
 &#39;fragment&#39;,
 &#39;on&#39;,
 &#39;a&#39;,
 &#39;1-flow&#39;,
 &#39;cell&#39;,
 &#39;run&#39;,
 &#39;in&#39;,
 &#39;the&#39;,
 &#39;upgraded&#39;,
 &#39;HiSeq&#39;,
 &#39;system.&#39;,
 &#39;Title:&#39;,
 &#39;Novel&#39;,
 &#39;whole&#39;,
 &#39;genome&#39;,
 &#39;de&#39;,
 &#39;novo&#39;,
 &#39;assembly&#39;,
 &#39;More&#39;,
 &#39;than&#39;,
 &#39;11,000&#39;,
 &#39;sequencing&#39;,
 &#39;projects,&#39;,
 &#39;including&#39;,
 &#39;targeted&#39;,
 &#39;projects,&#39;,
 &#39;were&#39;,
 &#39;reported&#39;,
 &#39;on&#39;,
 &#39;the&#39;,
 &#39;Genome&#39;,
 &#39;Online&#39;,
 &#39;Database&#39;,
 &#39;in&#39;,
 &#39;early&#39;,
 &#39;2012.&#39;,
 &#39;Now,&#39;,
 &#39;more&#39;,
 &#39;than&#39;,
 &#39;3,000&#39;,
 &#39;genome&#39;,
 &#39;projects&#39;,
 &#39;have&#39;,
 &#39;been&#39;,
 &#39;completed&#39;,
 &#39;on&#39;,
 &#39;the&#39;,
 &#39;diverse&#39;,
 &#39;genome&#39;,
 &#39;species,&#39;,
 &#39;and&#39;,
 &#39;more&#39;,
 &#39;than&#39;,
 &#39;90%&#39;,
 &#39;of&#39;,
 &#39;completed&#39;,
 &#39;projects&#39;,
 &#39;were&#39;,
 &#39;bacterial&#39;,
 &#39;genome&#39;,
 &#39;sequencing.&#39;,
 &#39;The&#39;,
 &#39;greatest&#39;,
 &#39;bacterial&#39;,
 &#39;genome&#39;,
 &#39;sequencing&#39;,
 &#39;was&#39;,
 &#39;performed&#39;,
 &#39;with&#39;,
 &#39;454&#39;,
 &#39;pyrosequencing&#39;,
 &#39;because&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;available&#39;,
 &#39;largest&#39;,
 &#39;long&#39;,
 &#39;read&#39;,
 &#39;sequencing,&#39;,
 &#39;useful&#39;,
 &#39;for&#39;,
 &#39;de&#39;,
 &#39;novo&#39;,
 &#39;assembly&#39;,
 &#39;of&#39;,
 &#39;novel&#39;,
 &#39;genome&#39;,
 &#39;sequencing.&#39;,
 &#39;The&#39;,
 &#39;official&#39;,
 &#39;depth&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;deep&#39;,
 &#39;sequencing&#39;,
 &#39;strategy&#39;,
 &#39;of&#39;,
 &#39;454&#39;,
 &#39;pyrosequencing&#39;,
 &#39;technology&#39;,
 &#39;for&#39;,
 &#39;whole&#39;,
 &#39;bacterial&#39;,
 &#39;genome&#39;,
 &#39;sequencing&#39;,
 &#39;for&#39;,
 &#39;de&#39;,
 &#39;novo&#39;,
 &#39;assembly&#39;,
 &#39;in&#39;,
 &#39;novel&#39;,
 &#39;genome&#39;,
 &#39;sequencing&#39;,
 &#39;is&#39;,
 &#39;at&#39;,
 &#39;least&#39;,
 &#39;15-20×&#39;,
 &#39;in&#39;,
 &#39;depth&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;estimated&#39;,
 &#39;genome&#39;,
 &#39;size&#39;,
 &#39;[9-13].&#39;,
 &#39;However,&#39;,
 &#39;Li&#39;,
 &#39;et&#39;,
 &#39;al.[3]&#39;,
 &#39;reported&#39;,
 &#39;that&#39;,
 &#39;6-10×&#39;,
 &#39;sequencing&#39;,
 &#39;in&#39;,
 &#39;qualified&#39;,
 &#39;runs&#39;,
 &#39;with&#39;,
 &#39;500-bp&#39;,
 &#39;reads&#39;,
 &#39;would&#39;,
 &#39;be&#39;,
 &#39;enough&#39;,
 &#39;for&#39;,
 &#39;de&#39;,
 &#39;novo&#39;,
 &#39;assemblies&#39;,
 &#39;from&#39;,
 &#39;1,480&#39;,
 ...]</pre>
</div>

</div>

</div>

</div>

</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell   ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[83]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># 특수문자, 숫자 제거</span>
<span class="n">new_raw</span> <span class="o">=</span> <span class="s1">&#39;&#39;</span>
<span class="k">for</span> <span class="n">i</span> <span class="ow">in</span> <span class="nb">range</span><span class="p">(</span> <span class="nb">len</span><span class="p">(</span><span class="n">raw</span><span class="p">)</span> <span class="p">)</span> <span class="p">:</span>
    <span class="n">new_raw</span> <span class="o">=</span> <span class="n">re</span><span class="o">.</span><span class="n">sub</span><span class="p">(</span><span class="sa">r</span><span class="s2">&quot;[^a-zA-Z\s]&quot;</span><span class="p">,</span> <span class="s2">&quot;&quot;</span><span class="p">,</span> <span class="n">raw</span><span class="p">[</span><span class="n">i</span><span class="p">])</span>
<span class="n">new_raw</span> <span class="o">=</span> <span class="n">new_raw</span><span class="o">.</span><span class="n">split</span><span class="p">()</span>
<span class="n">new_raw</span>
</pre></div>

     </div>
</div>
</div>
</div>

<div class="jp-Cell-outputWrapper">


<div class="jp-OutputArea jp-Cell-outputArea">

<div class="jp-OutputArea-child">

    
    <div class="jp-OutputPrompt jp-OutputArea-prompt">Out[83]:</div>




<div class="jp-RenderedText jp-OutputArea-output jp-OutputArea-executeResult" data-mime-type="text/plain">
<pre>[&#39;Title&#39;,
 &#39;Directtoconsumer&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;Directtoconsumer&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;is&#39;,
 &#39;a&#39;,
 &#39;controversial&#39;,
 &#39;issue&#39;,
 &#39;although&#39;,
 &#39;Korean&#39;,
 &#39;Government&#39;,
 &#39;is&#39;,
 &#39;considering&#39;,
 &#39;to&#39;,
 &#39;expand&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;Preventing&#39;,
 &#39;the&#39;,
 &#39;exaggeration&#39;,
 &#39;and&#39;,
 &#39;abusing&#39;,
 &#39;of&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;is&#39;,
 &#39;an&#39;,
 &#39;important&#39;,
 &#39;task&#39;,
 &#39;considering&#39;,
 &#39;the&#39;,
 &#39;early&#39;,
 &#39;history&#39;,
 &#39;of&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;in&#39;,
 &#39;Korea&#39;,
 &#39;And&#39;,
 &#39;the&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;performance&#39;,
 &#39;or&#39;,
 &#39;method&#39;,
 &#39;has&#39;,
 &#39;been&#39;,
 &#39;rarely&#39;,
 &#39;reported&#39;,
 &#39;to&#39;,
 &#39;the&#39;,
 &#39;scientific&#39;,
 &#39;andor&#39;,
 &#39;medical&#39;,
 &#39;community&#39;,
 &#39;and&#39;,
 &#39;reliability&#39;,
 &#39;of&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;needs&#39;,
 &#39;to&#39;,
 &#39;be&#39;,
 &#39;assessed&#39;,
 &#39;Law&#39;,
 &#39;enforcement&#39;,
 &#39;needs&#39;,
 &#39;to&#39;,
 &#39;improve&#39;,
 &#39;these&#39;,
 &#39;issues&#39;,
 &#39;Also&#39;,
 &#39;principle&#39;,
 &#39;of&#39;,
 &#39;transparency&#39;,
 &#39;needs&#39;,
 &#39;to&#39;,
 &#39;be&#39;,
 &#39;applied&#39;,
 &#39;Directtoconsumer&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;is&#39;,
 &#39;a&#39;,
 &#39;method&#39;,
 &#39;of&#39;,
 &#39;marketing&#39;,
 &#39;genetic&#39;,
 &#39;tests&#39;,
 &#39;to&#39;,
 &#39;consumers&#39;,
 &#39;without&#39;,
 &#39;the&#39;,
 &#39;direct&#39;,
 &#39;involvement&#39;,
 &#39;of&#39;,
 &#39;a&#39;,
 &#39;health&#39;,
 &#39;care&#39;,
 &#39;provider&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;in&#39;,
 &#39;Korea&#39;,
 &#39;has&#39;,
 &#39;been&#39;,
 &#39;introduced&#39;,
 &#39;since&#39;,
 &#39;by&#39;,
 &#39;the&#39;,
 &#39;amendment&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;Bioethics&#39;,
 &#39;and&#39;,
 &#39;safety&#39;,
 &#39;Act&#39;,
 &#39;Through&#39;,
 &#39;the&#39;,
 &#39;amendment&#39;,
 &#39;phenotypes&#39;,
 &#39;by&#39;,
 &#39;genes&#39;,
 &#39;have&#39;,
 &#39;been&#39;,
 &#39;allowed&#39;,
 &#39;for&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;Those&#39;,
 &#39;phenotypes&#39;,
 &#39;include&#39;,
 &#39;traits&#39;,
 &#39;of&#39;,
 &#39;body&#39;,
 &#39;mass&#39;,
 &#39;index&#39;,
 &#39;cholesterol&#39;,
 &#39;blood&#39;,
 &#39;pressure&#39;,
 &#39;and&#39;,
 &#39;so&#39;,
 &#39;on&#39;,
 &#39;Recently&#39;,
 &#39;the&#39;,
 &#39;Korean&#39;,
 &#39;government&#39;,
 &#39;introduced&#39;,
 &#39;a&#39;,
 &#39;regulatory&#39;,
 &#39;sandbox&#39;,
 &#39;that&#39;,
 &#39;includes&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;tests&#39;,
 &#39;Several&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;companies&#39;,
 &#39;received&#39;,
 &#39;approval&#39;,
 &#39;for&#39;,
 &#39;predictive&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;tests&#39;,
 &#39;for&#39;,
 &#39;conditions&#39;,
 &#39;ranging&#39;,
 &#39;from&#39;,
 &#39;cancer&#39;,
 &#39;to&#39;,
 &#39;chronic&#39;,
 &#39;diseases&#39;,
 &#39;And&#39;,
 &#39;also&#39;,
 &#39;the&#39;,
 &#39;ministry&#39;,
 &#39;of&#39;,
 &#39;Health&#39;,
 &#39;and&#39;,
 &#39;Welfare&#39;,
 &#39;is&#39;,
 &#39;doing&#39;,
 &#39;the&#39;,
 &#39;pilot&#39;,
 &#39;program&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;expanded&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;has&#39;,
 &#39;a&#39;,
 &#39;lot&#39;,
 &#39;of&#39;,
 &#39;issues&#39;,
 &#39;from&#39;,
 &#39;accuracies&#39;,
 &#39;to&#39;,
 &#39;ethical&#39;,
 &#39;legal&#39;,
 &#39;and&#39;,
 &#39;social&#39;,
 &#39;issues&#39;,
 &#39;ELSI&#39;,
 &#39;I&#39;,
 &#39;raise&#39;,
 &#39;the&#39;,
 &#39;immediate&#39;,
 &#39;issues&#39;,
 &#39;relevant&#39;,
 &#39;to&#39;,
 &#39;DTC&#39;,
 &#39;briefly&#39;,
 &#39;Most&#39;,
 &#39;important&#39;,
 &#39;concern&#39;,
 &#39;is&#39;,
 &#39;whether&#39;,
 &#39;commercial&#39;,
 &#39;genetic&#39;,
 &#39;services&#39;,
 &#39;predictive&#39;,
 &#39;value&#39;,
 &#39;is&#39;,
 &#39;sufficient&#39;,
 &#39;to&#39;,
 &#39;meet&#39;,
 &#39;the&#39;,
 &#39;standards&#39;,
 &#39;for&#39;,
 &#39;clinical&#39;,
 &#39;use&#39;,
 &#39;The&#39;,
 &#39;clinical&#39;,
 &#39;utility&#39;,
 &#39;of&#39;,
 &#39;a&#39;,
 &#39;genetic&#39;,
 &#39;test&#39;,
 &#39;should&#39;,
 &#39;be&#39;,
 &#39;an&#39;,
 &#39;essential&#39;,
 &#39;criterion&#39;,
 &#39;for&#39;,
 &#39;deciding&#39;,
 &#39;to&#39;,
 &#39;offer&#39;,
 &#39;this&#39;,
 &#39;test&#39;,
 &#39;to&#39;,
 &#39;a&#39;,
 &#39;person&#39;,
 &#39;or&#39;,
 &#39;a&#39;,
 &#39;group&#39;,
 &#39;of&#39;,
 &#39;persons&#39;,
 &#39;Reliability&#39;,
 &#39;has&#39;,
 &#39;been&#39;,
 &#39;one&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;major&#39;,
 &#39;issues&#39;,
 &#39;from&#39;,
 &#39;the&#39;,
 &#39;beginning&#39;,
 &#39;even&#39;,
 &#39;in&#39;,
 &#39;United&#39;,
 &#39;States&#39;,
 &#39;Dr&#39;,
 &#39;Craig&#39;,
 &#39;Ventor&#39;,
 &#39;reported&#39;,
 &#39;the&#39;,
 &#39;discrepancies&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;interpretation&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;same&#39;,
 &#39;individuals&#39;,
 &#39;between&#39;,
 &#39;andMe&#39;,
 &#39;and&#39;,
 &#39;Navigenics&#39;,
 &#39;GAO&#39;,
 &#39;Government&#39;,
 &#39;Accountability&#39;,
 &#39;Office&#39;,
 &#39;in&#39;,
 &#39;United&#39;,
 &#39;States&#39;,
 &#39;also&#39;,
 &#39;investigated&#39;,
 &#39;inconsistent&#39;,
 &#39;interpretations&#39;,
 &#39;among&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;companies&#39;,
 &#39;And&#39;,
 &#39;more&#39;,
 &#39;TandyConnor&#39;,
 &#39;et&#39;,
 &#39;al&#39;,
 &#39;reported&#39;,
 &#39;of&#39;,
 &#39;variants&#39;,
 &#39;in&#39;,
 &#39;a&#39;,
 &#39;variety&#39;,
 &#39;of&#39;,
 &#39;genes&#39;,
 &#39;reported&#39;,
 &#39;in&#39;,
 &#39;DTC&#39;,
 &#39;raw&#39;,
 &#39;data&#39;,
 &#39;were&#39;,
 &#39;false&#39;,
 &#39;positives&#39;,
 &#39;in&#39;,
 &#39;United&#39;,
 &#39;States&#39;,
 &#39;There&#39;,
 &#39;was&#39;,
 &#39;a&#39;,
 &#39;comparison&#39;,
 &#39;of&#39;,
 &#39;two&#39;,
 &#39;persons&#39;,
 &#39;between&#39;,
 &#39;two&#39;,
 &#39;Korean&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;companies&#39;,
 &#39;and&#39;,
 &#39;the&#39;,
 &#39;concordance&#39;,
 &#39;rate&#39;,
 &#39;is&#39;,
 &#39;low&#39;,
 &#39;It&#39;,
 &#39;is&#39;,
 &#39;very&#39;,
 &#39;difficult&#39;,
 &#39;to&#39;,
 &#39;judge&#39;,
 &#39;which&#39;,
 &#39;one&#39;,
 &#39;is&#39;,
 &#39;the&#39;,
 &#39;accurate&#39;,
 &#39;result&#39;,
 &#39;when&#39;,
 &#39;the&#39;,
 &#39;discrepancies&#39;,
 &#39;happen&#39;,
 &#39;Even&#39;,
 &#39;when&#39;,
 &#39;their&#39;,
 &#39;results&#39;,
 &#39;show&#39;,
 &#39;the&#39;,
 &#39;same&#39;,
 &#39;results&#39;,
 &#39;it&#39;,
 &#39;does&#39;,
 &#39;not&#39;,
 &#39;guarantee&#39;,
 &#39;the&#39;,
 &#39;results&#39;,
 &#39;are&#39;,
 &#39;true&#39;,
 &#39;Establishing&#39;,
 &#39;reliable&#39;,
 &#39;prediction&#39;,
 &#39;models&#39;,
 &#39;by&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;needs&#39;,
 &#39;advancements&#39;,
 &#39;under&#39;,
 &#39;current&#39;,
 &#39;situation&#39;,
 &#39;Currently&#39;,
 &#39;most&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;companies&#39;,
 &#39;use&#39;,
 &#39;single&#39;,
 &#39;nucleotide&#39;,
 &#39;polymorphisms&#39;,
 &#39;SNPs&#39;,
 &#39;for&#39;,
 &#39;predictive&#39;,
 &#39;models&#39;,
 &#39;among&#39;,
 &#39;genomewide&#39;,
 &#39;association&#39;,
 &#39;study&#39;,
 &#39;GWAS&#39;,
 &#39;results&#39;,
 &#39;from&#39;,
 &#39;the&#39;,
 &#39;relevant&#39;,
 &#39;literatures&#39;,
 &#39;In&#39;,
 &#39;case&#39;,
 &#39;of&#39;,
 &#39;single&#39;,
 &#39;SNPs&#39;,
 &#39;for&#39;,
 &#39;complex&#39;,
 &#39;disease&#39;,
 &#39;or&#39;,
 &#39;trait&#39;,
 &#39;its&#39;,
 &#39;predictive&#39;,
 &#39;performances&#39;,
 &#39;would&#39;,
 &#39;be&#39;,
 &#39;almost&#39;,
 &#39;meaningless&#39;,
 &#39;because&#39;,
 &#39;odds&#39;,
 &#39;ratio&#39;,
 &#39;of&#39;,
 &#39;most&#39;,
 &#39;validated&#39;,
 &#39;SNPs&#39;,
 &#39;are&#39;,
 &#39;below&#39;,
 &#39;except&#39;,
 &#39;few&#39;,
 &#39;SNPs&#39;,
 &#39;such&#39;,
 &#39;as&#39;,
 &#39;APOE&#39;,
 &#39;allele&#39;,
 &#39;to&#39;,
 &#39;Alzheimers&#39;,
 &#39;Diseases&#39;,
 &#39;That&#39;,
 &#39;is&#39;,
 &#39;without&#39;,
 &#39;counting&#39;,
 &#39;the&#39;,
 &#39;other&#39;,
 &#39;many&#39;,
 &#39;SNPs&#39;,
 &#39;counting&#39;,
 &#39;only&#39;,
 &#39;one&#39;,
 &#39;SNP&#39;,
 &#39;is&#39;,
 &#39;not&#39;,
 &#39;likely&#39;,
 &#39;to&#39;,
 &#39;show&#39;,
 &#39;good&#39;,
 &#39;performances&#39;,
 &#39;as&#39;,
 &#39;company&#39;,
 &#39;claims&#39;,
 &#39;Thus&#39;,
 &#39;it&#39;,
 &#39;comes&#39;,
 &#39;to&#39;,
 &#39;use&#39;,
 &#39;multiple&#39;,
 &#39;SNPs&#39;,
 &#39;for&#39;,
 &#39;predictive&#39;,
 &#39;models&#39;,
 &#39;In&#39;,
 &#39;case&#39;,
 &#39;of&#39;,
 &#39;using&#39;,
 &#39;multiple&#39;,
 &#39;SNPs&#39;,
 &#39;Some&#39;,
 &#39;SNPs&#39;,
 &#39;come&#39;,
 &#39;from&#39;,
 &#39;the&#39;,
 &#39;study&#39;,
 &#39;of&#39;,
 &#39;Korean&#39;,
 &#39;subjects&#39;,
 &#39;but&#39;,
 &#39;many&#39;,
 &#39;SNPs&#39;,
 &#39;depend&#39;,
 &#39;on&#39;,
 &#39;the&#39;,
 &#39;results&#39;,
 &#39;from&#39;,
 &#39;Caucasian&#39;,
 &#39;subjects&#39;,
 &#39;Considering&#39;,
 &#39;the&#39;,
 &#39;experiences&#39;,
 &#39;from&#39;,
 &#39;GWAS&#39;,
 &#39;studies&#39;,
 &#39;among&#39;,
 &#39;Korean&#39;,
 &#39;subjects&#39;,
 &#39;most&#39;,
 &#39;validated&#39;,
 &#39;SNPs&#39;,
 &#39;from&#39;,
 &#39;Caucasian&#39;,
 &#39;subjects&#39;,
 &#39;without&#39;,
 &#39;studying&#39;,
 &#39;Korean&#39;,
 &#39;subjects&#39;,
 &#39;are&#39;,
 &#39;likely&#39;,
 &#39;to&#39;,
 &#39;be&#39;,
 &#39;replicated&#39;,
 &#39;among&#39;,
 &#39;Korean&#39;,
 &#39;subjects&#39;,
 &#39;although&#39;,
 &#39;not&#39;,
 &#39;all&#39;,
 &#39;of&#39;,
 &#39;them&#39;,
 &#39;However&#39;,
 &#39;when&#39;,
 &#39;applying&#39;,
 &#39;to&#39;,
 &#39;Korean&#39;,
 &#39;population&#39;,
 &#39;the&#39;,
 &#39;odds&#39;,
 &#39;ratio&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;used&#39;,
 &#39;SNPs&#39;,
 &#39;in&#39;,
 &#39;the&#39;,
 &#39;predictive&#39;,
 &#39;model&#39;,
 &#39;is&#39;,
 &#39;critical&#39;,
 &#39;to&#39;,
 &#39;calculate&#39;,
 &#39;the&#39;,
 &#39;relative&#39;,
 &#39;risk&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;individual&#39;,
 &#39;consumer&#39;,
 &#39;But&#39;,
 &#39;the&#39;,
 &#39;magnitudes&#39;,
 &#39;of&#39;,
 &#39;relative&#39;,
 &#39;risks&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;each&#39;,
 &#39;validated&#39;,
 &#39;SNPs&#39;,
 &#39;between&#39;,
 &#39;Caucasian&#39;,
 &#39;and&#39;,
 &#39;Korean&#39;,
 &#39;subjects&#39;,
 &#39;do&#39;,
 &#39;not&#39;,
 &#39;show&#39;,
 &#39;same&#39;,
 &#39;values&#39;,
 &#39;Therefore&#39;,
 &#39;the&#39;,
 &#39;established&#39;,
 &#39;models&#39;,
 &#39;for&#39;,
 &#39;each&#39;,
 &#39;trait&#39;,
 &#39;need&#39;,
 &#39;to&#39;,
 &#39;be&#39;,
 &#39;validated&#39;,
 &#39;for&#39;,
 &#39;Korean&#39;,
 &#39;patients&#39;,
 &#39;prospectively&#39;,
 &#39;at&#39;,
 &#39;least&#39;,
 &#39;But&#39;,
 &#39;the&#39;,
 &#39;most&#39;,
 &#39;companyrun&#39;,
 &#39;or&#39;,
 &#39;companyplantorun&#39;,
 &#39;traits&#39;,
 &#39;show&#39;,
 &#39;lack&#39;,
 &#39;of&#39;,
 &#39;these&#39;,
 &#39;evidences&#39;,
 &#39;It&#39;,
 &#39;is&#39;,
 &#39;not&#39;,
 &#39;possible&#39;,
 &#39;to&#39;,
 &#39;judge&#39;,
 &#39;or&#39;,
 &#39;estimate&#39;,
 &#39;which&#39;,
 &#39;company&#39;,
 &#39;supports&#39;,
 &#39;better&#39;,
 &#39;predictive&#39;,
 &#39;models&#39;,
 &#39;or&#39;,
 &#39;result&#39;,
 &#39;This&#39;,
 &#39;situation&#39;,
 &#39;drives&#39;,
 &#39;to&#39;,
 &#39;go&#39;,
 &#39;to&#39;,
 &#39;marketing&#39;,
 &#39;with&#39;,
 &#39;exaggerations&#39;,
 &#39;and&#39;,
 &#39;abuse&#39;,
 &#39;of&#39;,
 &#39;tests&#39;,
 &#39;or&#39;,
 &#39;price&#39;,
 &#39;competition&#39;,
 &#39;rather&#39;,
 &#39;than&#39;,
 &#39;competing&#39;,
 &#39;to&#39;,
 &#39;improve&#39;,
 &#39;the&#39;,
 &#39;quality&#39;,
 &#39;of&#39;,
 &#39;prediction&#39;,
 &#39;The&#39;,
 &#39;current&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;needs&#39;,
 &#39;more&#39;,
 &#39;objective&#39;,
 &#39;evidences&#39;,
 &#39;If&#39;,
 &#39;they&#39;,
 &#39;have&#39;,
 &#39;a&#39;,
 &#39;scientific&#39;,
 &#39;basis&#39;,
 &#39;from&#39;,
 &#39;Korean&#39;,
 &#39;population&#39;,
 &#39;even&#39;,
 &#39;poor&#39;,
 &#39;performances&#39;,
 &#39;can&#39;,
 &#39;have&#39;,
 &#39;a&#39;,
 &#39;chance&#39;,
 &#39;to&#39;,
 &#39;improve&#39;,
 &#39;because&#39;,
 &#39;they&#39;,
 &#39;know&#39;,
 &#39;which&#39;,
 &#39;part&#39;,
 &#39;needs&#39;,
 &#39;to&#39;,
 &#39;be&#39;,
 &#39;updated&#39;,
 &#39;or&#39;,
 &#39;changed&#39;,
 &#39;But&#39;,
 &#39;companies&#39;,
 &#39;extremely&#39;,
 &#39;rarely&#39;,
 &#39;show&#39;,
 &#39;the&#39;,
 &#39;method&#39;,
 &#39;and&#39;,
 &#39;performance&#39;,
 &#39;of&#39;,
 &#39;their&#39;,
 &#39;predictive&#39;,
 &#39;model&#39;,
 &#39;from&#39;,
 &#39;Korean&#39;,
 &#39;population&#39;,
 &#39;They&#39;,
 &#39;demand&#39;,
 &#39;to&#39;,
 &#39;the&#39;,
 &#39;government&#39;,
 &#39;authorities&#39;,
 &#39;to&#39;,
 &#39;allow&#39;,
 &#39;disease&#39;,
 &#39;prediction&#39;,
 &#39;services&#39;,
 &#39;as&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;including&#39;,
 &#39;cancer&#39;,
 &#39;diabetes&#39;,
 &#39;and&#39;,
 &#39;heart&#39;,
 &#39;disease&#39;,
 &#39;without&#39;,
 &#39;disclosing&#39;,
 &#39;their&#39;,
 &#39;models&#39;,
 &#39;and&#39;,
 &#39;performances&#39;,
 &#39;to&#39;,
 &#39;the&#39;,
 &#39;scientific&#39;,
 &#39;or&#39;,
 &#39;medical&#39;,
 &#39;societies&#39;,
 &#39;If&#39;,
 &#39;it&#39;,
 &#39;is&#39;,
 &#39;allowed&#39;,
 &#39;it&#39;,
 &#39;will&#39;,
 &#39;cause&#39;,
 &#39;the&#39;,
 &#39;profound&#39;,
 &#39;confusion&#39;,
 &#39;or&#39;,
 &#39;chaos&#39;,
 &#39;to&#39;,
 &#39;the&#39;,
 &#39;consumers&#39;,
 &#39;medical&#39;,
 &#39;institutions&#39;,
 &#39;and&#39;,
 &#39;health&#39;,
 &#39;insurance&#39;,
 &#39;system&#39;,
 &#39;and&#39;,
 &#39;the&#39;,
 &#39;huge&#39;,
 &#39;waste&#39;,
 &#39;of&#39;,
 &#39;medical&#39;,
 &#39;resources&#39;,
 &#39;Currently&#39;,
 &#39;the&#39;,
 &#39;facility&#39;,
 &#39;and&#39;,
 &#39;human&#39;,
 &#39;resources&#39;,
 &#39;in&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;company&#39;,
 &#39;are&#39;,
 &#39;not&#39;,
 &#39;regulated&#39;,
 &#39;by&#39;,
 &#39;law&#39;,
 &#39;There&#39;,
 &#39;is&#39;,
 &#39;no&#39;,
 &#39;penalty&#39;,
 &#39;to&#39;,
 &#39;their&#39;,
 &#39;rejection&#39;,
 &#39;to&#39;,
 &#39;proficiency&#39;,
 &#39;test&#39;,
 &#39;by&#39;,
 &#39;law&#39;,
 &#39;For&#39;,
 &#39;improving&#39;,
 &#39;the&#39;,
 &#39;situation&#39;,
 &#39;with&#39;,
 &#39;the&#39;,
 &#39;above&#39;,
 &#39;mentioned&#39;,
 &#39;direction&#39;,
 &#39;the&#39;,
 &#39;legal&#39;,
 &#39;coercion&#39;,
 &#39;is&#39;,
 &#39;inevitable&#39;,
 &#39;Amendment&#39;,
 &#39;of&#39;,
 &#39;Bioethics&#39;,
 &#39;and&#39;,
 &#39;Safety&#39;,
 &#39;Act&#39;,
 &#39;or&#39;,
 &#39;making&#39;,
 &#39;a&#39;,
 &#39;law&#39;,
 &#39;equivalent&#39;,
 &#39;to&#39;,
 &#39;Clinical&#39;,
 &#39;Laboratory&#39;,
 &#39;Improvement&#39;,
 &#39;Act&#39;,
 &#39;CLIA&#39;,
 &#39;in&#39;,
 &#39;United&#39;,
 &#39;States&#39;,
 &#39;for&#39;,
 &#39;appropriate&#39;,
 &#39;regulation&#39;,
 &#39;is&#39;,
 &#39;a&#39;,
 &#39;prerequisite&#39;,
 &#39;step&#39;,
 &#39;The&#39;,
 &#39;predictive&#39;,
 &#39;models&#39;,
 &#39;of&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;need&#39;,
 &#39;the&#39;,
 &#39;validation&#39;,
 &#39;before&#39;,
 &#39;implementation&#39;,
 &#39;But&#39;,
 &#39;practically&#39;,
 &#39;not&#39;,
 &#39;all&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;tests&#39;,
 &#39;can&#39;,
 &#39;be&#39;,
 &#39;validated&#39;,
 &#39;or&#39;,
 &#39;achieved&#39;,
 &#39;to&#39;,
 &#39;the&#39;,
 &#39;certain&#39;,
 &#39;standard&#39;,
 &#39;If&#39;,
 &#39;so&#39;,
 &#39;disclosing&#39;,
 &#39;all&#39;,
 &#39;the&#39;,
 &#39;relevant&#39;,
 &#39;information&#39;,
 &#39;transparently&#39;,
 &#39;for&#39;,
 &#39;the&#39;,
 &#39;right&#39;,
 &#39;choices&#39;,
 &#39;by&#39;,
 &#39;the&#39;,
 &#39;consumers&#39;,
 &#39;instead&#39;,
 &#39;can&#39;,
 &#39;be&#39;,
 &#39;considered&#39;,
 &#39;The&#39;,
 &#39;relevant&#39;,
 &#39;information&#39;,
 &#39;includes&#39;,
 &#39;the&#39;,
 &#39;explanation&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;whole&#39;,
 &#39;process&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;tests&#39;,
 &#39;the&#39;,
 &#39;references&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;method&#39;,
 &#39;they&#39;,
 &#39;used&#39;,
 &#39;disclosing&#39;,
 &#39;the&#39;,
 &#39;implication&#39;,
 &#39;and&#39;,
 &#39;limitation&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;results&#39;,
 &#39;how&#39;,
 &#39;to&#39;,
 &#39;deliver&#39;,
 &#39;the&#39;,
 &#39;report&#39;,
 &#39;to&#39;,
 &#39;consumers&#39;,
 &#39;etc&#39;,
 &#39;And&#39;,
 &#39;the&#39;,
 &#39;company&#39;,
 &#39;should&#39;,
 &#39;consider&#39;,
 &#39;the&#39;,
 &#39;possible&#39;,
 &#39;outcomes&#39;,
 &#39;after&#39;,
 &#39;getting&#39;,
 &#39;traits&#39;,
 &#39;risk&#39;,
 &#39;and&#39;,
 &#39;provide&#39;,
 &#39;the&#39;,
 &#39;report&#39;,
 &#39;guidelines&#39;,
 &#39;not&#39;,
 &#39;to&#39;,
 &#39;cause&#39;,
 &#39;any&#39;,
 &#39;harm&#39;,
 &#39;or&#39;,
 &#39;negative&#39;,
 &#39;effect&#39;,
 &#39;to&#39;,
 &#39;the&#39;,
 &#39;consumers&#39;,
 &#39;including&#39;,
 &#39;psychological&#39;,
 &#39;distress&#39;,
 &#39;or&#39;,
 &#39;waste&#39;,
 &#39;of&#39;,
 &#39;medical&#39;,
 &#39;resources&#39;,
 &#39;New&#39;,
 &#39;genomic&#39;,
 &#39;technologies&#39;,
 &#39;and&#39;,
 &#39;knowledges&#39;,
 &#39;expand&#39;,
 &#39;our&#39;,
 &#39;view&#39;,
 &#39;and&#39;,
 &#39;their&#39;,
 &#39;applications&#39;,
 &#39;would&#39;,
 &#39;improve&#39;,
 &#39;human&#39;,
 &#39;health&#39;,
 ...]</pre>
</div>

</div>

</div>

</div>

</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell   ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[85]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span><span class="c1"># 글자 1개만 있으면 제외하기</span>
<span class="n">li_raw</span> <span class="o">=</span> <span class="p">[]</span>
<span class="k">for</span> <span class="n">word</span> <span class="ow">in</span> <span class="n">new_raw</span><span class="p">:</span>
    <span class="k">if</span> <span class="nb">len</span><span class="p">(</span><span class="n">word</span><span class="p">)</span> <span class="o">!=</span> <span class="mi">1</span><span class="p">:</span>
        <span class="n">li_raw</span><span class="o">.</span><span class="n">append</span><span class="p">(</span><span class="n">word</span><span class="p">)</span>
<span class="n">li_raw</span>
</pre></div>

     </div>
</div>
</div>
</div>

<div class="jp-Cell-outputWrapper">


<div class="jp-OutputArea jp-Cell-outputArea">

<div class="jp-OutputArea-child">

    
    <div class="jp-OutputPrompt jp-OutputArea-prompt">Out[85]:</div>




<div class="jp-RenderedText jp-OutputArea-output jp-OutputArea-executeResult" data-mime-type="text/plain">
<pre>[&#39;Title&#39;,
 &#39;Directtoconsumer&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;Directtoconsumer&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;is&#39;,
 &#39;controversial&#39;,
 &#39;issue&#39;,
 &#39;although&#39;,
 &#39;Korean&#39;,
 &#39;Government&#39;,
 &#39;is&#39;,
 &#39;considering&#39;,
 &#39;to&#39;,
 &#39;expand&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;Preventing&#39;,
 &#39;the&#39;,
 &#39;exaggeration&#39;,
 &#39;and&#39;,
 &#39;abusing&#39;,
 &#39;of&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;is&#39;,
 &#39;an&#39;,
 &#39;important&#39;,
 &#39;task&#39;,
 &#39;considering&#39;,
 &#39;the&#39;,
 &#39;early&#39;,
 &#39;history&#39;,
 &#39;of&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;in&#39;,
 &#39;Korea&#39;,
 &#39;And&#39;,
 &#39;the&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;performance&#39;,
 &#39;or&#39;,
 &#39;method&#39;,
 &#39;has&#39;,
 &#39;been&#39;,
 &#39;rarely&#39;,
 &#39;reported&#39;,
 &#39;to&#39;,
 &#39;the&#39;,
 &#39;scientific&#39;,
 &#39;andor&#39;,
 &#39;medical&#39;,
 &#39;community&#39;,
 &#39;and&#39;,
 &#39;reliability&#39;,
 &#39;of&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;needs&#39;,
 &#39;to&#39;,
 &#39;be&#39;,
 &#39;assessed&#39;,
 &#39;Law&#39;,
 &#39;enforcement&#39;,
 &#39;needs&#39;,
 &#39;to&#39;,
 &#39;improve&#39;,
 &#39;these&#39;,
 &#39;issues&#39;,
 &#39;Also&#39;,
 &#39;principle&#39;,
 &#39;of&#39;,
 &#39;transparency&#39;,
 &#39;needs&#39;,
 &#39;to&#39;,
 &#39;be&#39;,
 &#39;applied&#39;,
 &#39;Directtoconsumer&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;is&#39;,
 &#39;method&#39;,
 &#39;of&#39;,
 &#39;marketing&#39;,
 &#39;genetic&#39;,
 &#39;tests&#39;,
 &#39;to&#39;,
 &#39;consumers&#39;,
 &#39;without&#39;,
 &#39;the&#39;,
 &#39;direct&#39;,
 &#39;involvement&#39;,
 &#39;of&#39;,
 &#39;health&#39;,
 &#39;care&#39;,
 &#39;provider&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;in&#39;,
 &#39;Korea&#39;,
 &#39;has&#39;,
 &#39;been&#39;,
 &#39;introduced&#39;,
 &#39;since&#39;,
 &#39;by&#39;,
 &#39;the&#39;,
 &#39;amendment&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;Bioethics&#39;,
 &#39;and&#39;,
 &#39;safety&#39;,
 &#39;Act&#39;,
 &#39;Through&#39;,
 &#39;the&#39;,
 &#39;amendment&#39;,
 &#39;phenotypes&#39;,
 &#39;by&#39;,
 &#39;genes&#39;,
 &#39;have&#39;,
 &#39;been&#39;,
 &#39;allowed&#39;,
 &#39;for&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;Those&#39;,
 &#39;phenotypes&#39;,
 &#39;include&#39;,
 &#39;traits&#39;,
 &#39;of&#39;,
 &#39;body&#39;,
 &#39;mass&#39;,
 &#39;index&#39;,
 &#39;cholesterol&#39;,
 &#39;blood&#39;,
 &#39;pressure&#39;,
 &#39;and&#39;,
 &#39;so&#39;,
 &#39;on&#39;,
 &#39;Recently&#39;,
 &#39;the&#39;,
 &#39;Korean&#39;,
 &#39;government&#39;,
 &#39;introduced&#39;,
 &#39;regulatory&#39;,
 &#39;sandbox&#39;,
 &#39;that&#39;,
 &#39;includes&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;tests&#39;,
 &#39;Several&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;companies&#39;,
 &#39;received&#39;,
 &#39;approval&#39;,
 &#39;for&#39;,
 &#39;predictive&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;tests&#39;,
 &#39;for&#39;,
 &#39;conditions&#39;,
 &#39;ranging&#39;,
 &#39;from&#39;,
 &#39;cancer&#39;,
 &#39;to&#39;,
 &#39;chronic&#39;,
 &#39;diseases&#39;,
 &#39;And&#39;,
 &#39;also&#39;,
 &#39;the&#39;,
 &#39;ministry&#39;,
 &#39;of&#39;,
 &#39;Health&#39;,
 &#39;and&#39;,
 &#39;Welfare&#39;,
 &#39;is&#39;,
 &#39;doing&#39;,
 &#39;the&#39;,
 &#39;pilot&#39;,
 &#39;program&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;expanded&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;has&#39;,
 &#39;lot&#39;,
 &#39;of&#39;,
 &#39;issues&#39;,
 &#39;from&#39;,
 &#39;accuracies&#39;,
 &#39;to&#39;,
 &#39;ethical&#39;,
 &#39;legal&#39;,
 &#39;and&#39;,
 &#39;social&#39;,
 &#39;issues&#39;,
 &#39;ELSI&#39;,
 &#39;raise&#39;,
 &#39;the&#39;,
 &#39;immediate&#39;,
 &#39;issues&#39;,
 &#39;relevant&#39;,
 &#39;to&#39;,
 &#39;DTC&#39;,
 &#39;briefly&#39;,
 &#39;Most&#39;,
 &#39;important&#39;,
 &#39;concern&#39;,
 &#39;is&#39;,
 &#39;whether&#39;,
 &#39;commercial&#39;,
 &#39;genetic&#39;,
 &#39;services&#39;,
 &#39;predictive&#39;,
 &#39;value&#39;,
 &#39;is&#39;,
 &#39;sufficient&#39;,
 &#39;to&#39;,
 &#39;meet&#39;,
 &#39;the&#39;,
 &#39;standards&#39;,
 &#39;for&#39;,
 &#39;clinical&#39;,
 &#39;use&#39;,
 &#39;The&#39;,
 &#39;clinical&#39;,
 &#39;utility&#39;,
 &#39;of&#39;,
 &#39;genetic&#39;,
 &#39;test&#39;,
 &#39;should&#39;,
 &#39;be&#39;,
 &#39;an&#39;,
 &#39;essential&#39;,
 &#39;criterion&#39;,
 &#39;for&#39;,
 &#39;deciding&#39;,
 &#39;to&#39;,
 &#39;offer&#39;,
 &#39;this&#39;,
 &#39;test&#39;,
 &#39;to&#39;,
 &#39;person&#39;,
 &#39;or&#39;,
 &#39;group&#39;,
 &#39;of&#39;,
 &#39;persons&#39;,
 &#39;Reliability&#39;,
 &#39;has&#39;,
 &#39;been&#39;,
 &#39;one&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;major&#39;,
 &#39;issues&#39;,
 &#39;from&#39;,
 &#39;the&#39;,
 &#39;beginning&#39;,
 &#39;even&#39;,
 &#39;in&#39;,
 &#39;United&#39;,
 &#39;States&#39;,
 &#39;Dr&#39;,
 &#39;Craig&#39;,
 &#39;Ventor&#39;,
 &#39;reported&#39;,
 &#39;the&#39;,
 &#39;discrepancies&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;interpretation&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;same&#39;,
 &#39;individuals&#39;,
 &#39;between&#39;,
 &#39;andMe&#39;,
 &#39;and&#39;,
 &#39;Navigenics&#39;,
 &#39;GAO&#39;,
 &#39;Government&#39;,
 &#39;Accountability&#39;,
 &#39;Office&#39;,
 &#39;in&#39;,
 &#39;United&#39;,
 &#39;States&#39;,
 &#39;also&#39;,
 &#39;investigated&#39;,
 &#39;inconsistent&#39;,
 &#39;interpretations&#39;,
 &#39;among&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;companies&#39;,
 &#39;And&#39;,
 &#39;more&#39;,
 &#39;TandyConnor&#39;,
 &#39;et&#39;,
 &#39;al&#39;,
 &#39;reported&#39;,
 &#39;of&#39;,
 &#39;variants&#39;,
 &#39;in&#39;,
 &#39;variety&#39;,
 &#39;of&#39;,
 &#39;genes&#39;,
 &#39;reported&#39;,
 &#39;in&#39;,
 &#39;DTC&#39;,
 &#39;raw&#39;,
 &#39;data&#39;,
 &#39;were&#39;,
 &#39;false&#39;,
 &#39;positives&#39;,
 &#39;in&#39;,
 &#39;United&#39;,
 &#39;States&#39;,
 &#39;There&#39;,
 &#39;was&#39;,
 &#39;comparison&#39;,
 &#39;of&#39;,
 &#39;two&#39;,
 &#39;persons&#39;,
 &#39;between&#39;,
 &#39;two&#39;,
 &#39;Korean&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;companies&#39;,
 &#39;and&#39;,
 &#39;the&#39;,
 &#39;concordance&#39;,
 &#39;rate&#39;,
 &#39;is&#39;,
 &#39;low&#39;,
 &#39;It&#39;,
 &#39;is&#39;,
 &#39;very&#39;,
 &#39;difficult&#39;,
 &#39;to&#39;,
 &#39;judge&#39;,
 &#39;which&#39;,
 &#39;one&#39;,
 &#39;is&#39;,
 &#39;the&#39;,
 &#39;accurate&#39;,
 &#39;result&#39;,
 &#39;when&#39;,
 &#39;the&#39;,
 &#39;discrepancies&#39;,
 &#39;happen&#39;,
 &#39;Even&#39;,
 &#39;when&#39;,
 &#39;their&#39;,
 &#39;results&#39;,
 &#39;show&#39;,
 &#39;the&#39;,
 &#39;same&#39;,
 &#39;results&#39;,
 &#39;it&#39;,
 &#39;does&#39;,
 &#39;not&#39;,
 &#39;guarantee&#39;,
 &#39;the&#39;,
 &#39;results&#39;,
 &#39;are&#39;,
 &#39;true&#39;,
 &#39;Establishing&#39;,
 &#39;reliable&#39;,
 &#39;prediction&#39;,
 &#39;models&#39;,
 &#39;by&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;needs&#39;,
 &#39;advancements&#39;,
 &#39;under&#39;,
 &#39;current&#39;,
 &#39;situation&#39;,
 &#39;Currently&#39;,
 &#39;most&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;companies&#39;,
 &#39;use&#39;,
 &#39;single&#39;,
 &#39;nucleotide&#39;,
 &#39;polymorphisms&#39;,
 &#39;SNPs&#39;,
 &#39;for&#39;,
 &#39;predictive&#39;,
 &#39;models&#39;,
 &#39;among&#39;,
 &#39;genomewide&#39;,
 &#39;association&#39;,
 &#39;study&#39;,
 &#39;GWAS&#39;,
 &#39;results&#39;,
 &#39;from&#39;,
 &#39;the&#39;,
 &#39;relevant&#39;,
 &#39;literatures&#39;,
 &#39;In&#39;,
 &#39;case&#39;,
 &#39;of&#39;,
 &#39;single&#39;,
 &#39;SNPs&#39;,
 &#39;for&#39;,
 &#39;complex&#39;,
 &#39;disease&#39;,
 &#39;or&#39;,
 &#39;trait&#39;,
 &#39;its&#39;,
 &#39;predictive&#39;,
 &#39;performances&#39;,
 &#39;would&#39;,
 &#39;be&#39;,
 &#39;almost&#39;,
 &#39;meaningless&#39;,
 &#39;because&#39;,
 &#39;odds&#39;,
 &#39;ratio&#39;,
 &#39;of&#39;,
 &#39;most&#39;,
 &#39;validated&#39;,
 &#39;SNPs&#39;,
 &#39;are&#39;,
 &#39;below&#39;,
 &#39;except&#39;,
 &#39;few&#39;,
 &#39;SNPs&#39;,
 &#39;such&#39;,
 &#39;as&#39;,
 &#39;APOE&#39;,
 &#39;allele&#39;,
 &#39;to&#39;,
 &#39;Alzheimers&#39;,
 &#39;Diseases&#39;,
 &#39;That&#39;,
 &#39;is&#39;,
 &#39;without&#39;,
 &#39;counting&#39;,
 &#39;the&#39;,
 &#39;other&#39;,
 &#39;many&#39;,
 &#39;SNPs&#39;,
 &#39;counting&#39;,
 &#39;only&#39;,
 &#39;one&#39;,
 &#39;SNP&#39;,
 &#39;is&#39;,
 &#39;not&#39;,
 &#39;likely&#39;,
 &#39;to&#39;,
 &#39;show&#39;,
 &#39;good&#39;,
 &#39;performances&#39;,
 &#39;as&#39;,
 &#39;company&#39;,
 &#39;claims&#39;,
 &#39;Thus&#39;,
 &#39;it&#39;,
 &#39;comes&#39;,
 &#39;to&#39;,
 &#39;use&#39;,
 &#39;multiple&#39;,
 &#39;SNPs&#39;,
 &#39;for&#39;,
 &#39;predictive&#39;,
 &#39;models&#39;,
 &#39;In&#39;,
 &#39;case&#39;,
 &#39;of&#39;,
 &#39;using&#39;,
 &#39;multiple&#39;,
 &#39;SNPs&#39;,
 &#39;Some&#39;,
 &#39;SNPs&#39;,
 &#39;come&#39;,
 &#39;from&#39;,
 &#39;the&#39;,
 &#39;study&#39;,
 &#39;of&#39;,
 &#39;Korean&#39;,
 &#39;subjects&#39;,
 &#39;but&#39;,
 &#39;many&#39;,
 &#39;SNPs&#39;,
 &#39;depend&#39;,
 &#39;on&#39;,
 &#39;the&#39;,
 &#39;results&#39;,
 &#39;from&#39;,
 &#39;Caucasian&#39;,
 &#39;subjects&#39;,
 &#39;Considering&#39;,
 &#39;the&#39;,
 &#39;experiences&#39;,
 &#39;from&#39;,
 &#39;GWAS&#39;,
 &#39;studies&#39;,
 &#39;among&#39;,
 &#39;Korean&#39;,
 &#39;subjects&#39;,
 &#39;most&#39;,
 &#39;validated&#39;,
 &#39;SNPs&#39;,
 &#39;from&#39;,
 &#39;Caucasian&#39;,
 &#39;subjects&#39;,
 &#39;without&#39;,
 &#39;studying&#39;,
 &#39;Korean&#39;,
 &#39;subjects&#39;,
 &#39;are&#39;,
 &#39;likely&#39;,
 &#39;to&#39;,
 &#39;be&#39;,
 &#39;replicated&#39;,
 &#39;among&#39;,
 &#39;Korean&#39;,
 &#39;subjects&#39;,
 &#39;although&#39;,
 &#39;not&#39;,
 &#39;all&#39;,
 &#39;of&#39;,
 &#39;them&#39;,
 &#39;However&#39;,
 &#39;when&#39;,
 &#39;applying&#39;,
 &#39;to&#39;,
 &#39;Korean&#39;,
 &#39;population&#39;,
 &#39;the&#39;,
 &#39;odds&#39;,
 &#39;ratio&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;used&#39;,
 &#39;SNPs&#39;,
 &#39;in&#39;,
 &#39;the&#39;,
 &#39;predictive&#39;,
 &#39;model&#39;,
 &#39;is&#39;,
 &#39;critical&#39;,
 &#39;to&#39;,
 &#39;calculate&#39;,
 &#39;the&#39;,
 &#39;relative&#39;,
 &#39;risk&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;individual&#39;,
 &#39;consumer&#39;,
 &#39;But&#39;,
 &#39;the&#39;,
 &#39;magnitudes&#39;,
 &#39;of&#39;,
 &#39;relative&#39;,
 &#39;risks&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;each&#39;,
 &#39;validated&#39;,
 &#39;SNPs&#39;,
 &#39;between&#39;,
 &#39;Caucasian&#39;,
 &#39;and&#39;,
 &#39;Korean&#39;,
 &#39;subjects&#39;,
 &#39;do&#39;,
 &#39;not&#39;,
 &#39;show&#39;,
 &#39;same&#39;,
 &#39;values&#39;,
 &#39;Therefore&#39;,
 &#39;the&#39;,
 &#39;established&#39;,
 &#39;models&#39;,
 &#39;for&#39;,
 &#39;each&#39;,
 &#39;trait&#39;,
 &#39;need&#39;,
 &#39;to&#39;,
 &#39;be&#39;,
 &#39;validated&#39;,
 &#39;for&#39;,
 &#39;Korean&#39;,
 &#39;patients&#39;,
 &#39;prospectively&#39;,
 &#39;at&#39;,
 &#39;least&#39;,
 &#39;But&#39;,
 &#39;the&#39;,
 &#39;most&#39;,
 &#39;companyrun&#39;,
 &#39;or&#39;,
 &#39;companyplantorun&#39;,
 &#39;traits&#39;,
 &#39;show&#39;,
 &#39;lack&#39;,
 &#39;of&#39;,
 &#39;these&#39;,
 &#39;evidences&#39;,
 &#39;It&#39;,
 &#39;is&#39;,
 &#39;not&#39;,
 &#39;possible&#39;,
 &#39;to&#39;,
 &#39;judge&#39;,
 &#39;or&#39;,
 &#39;estimate&#39;,
 &#39;which&#39;,
 &#39;company&#39;,
 &#39;supports&#39;,
 &#39;better&#39;,
 &#39;predictive&#39;,
 &#39;models&#39;,
 &#39;or&#39;,
 &#39;result&#39;,
 &#39;This&#39;,
 &#39;situation&#39;,
 &#39;drives&#39;,
 &#39;to&#39;,
 &#39;go&#39;,
 &#39;to&#39;,
 &#39;marketing&#39;,
 &#39;with&#39;,
 &#39;exaggerations&#39;,
 &#39;and&#39;,
 &#39;abuse&#39;,
 &#39;of&#39;,
 &#39;tests&#39;,
 &#39;or&#39;,
 &#39;price&#39;,
 &#39;competition&#39;,
 &#39;rather&#39;,
 &#39;than&#39;,
 &#39;competing&#39;,
 &#39;to&#39;,
 &#39;improve&#39;,
 &#39;the&#39;,
 &#39;quality&#39;,
 &#39;of&#39;,
 &#39;prediction&#39;,
 &#39;The&#39;,
 &#39;current&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;needs&#39;,
 &#39;more&#39;,
 &#39;objective&#39;,
 &#39;evidences&#39;,
 &#39;If&#39;,
 &#39;they&#39;,
 &#39;have&#39;,
 &#39;scientific&#39;,
 &#39;basis&#39;,
 &#39;from&#39;,
 &#39;Korean&#39;,
 &#39;population&#39;,
 &#39;even&#39;,
 &#39;poor&#39;,
 &#39;performances&#39;,
 &#39;can&#39;,
 &#39;have&#39;,
 &#39;chance&#39;,
 &#39;to&#39;,
 &#39;improve&#39;,
 &#39;because&#39;,
 &#39;they&#39;,
 &#39;know&#39;,
 &#39;which&#39;,
 &#39;part&#39;,
 &#39;needs&#39;,
 &#39;to&#39;,
 &#39;be&#39;,
 &#39;updated&#39;,
 &#39;or&#39;,
 &#39;changed&#39;,
 &#39;But&#39;,
 &#39;companies&#39;,
 &#39;extremely&#39;,
 &#39;rarely&#39;,
 &#39;show&#39;,
 &#39;the&#39;,
 &#39;method&#39;,
 &#39;and&#39;,
 &#39;performance&#39;,
 &#39;of&#39;,
 &#39;their&#39;,
 &#39;predictive&#39;,
 &#39;model&#39;,
 &#39;from&#39;,
 &#39;Korean&#39;,
 &#39;population&#39;,
 &#39;They&#39;,
 &#39;demand&#39;,
 &#39;to&#39;,
 &#39;the&#39;,
 &#39;government&#39;,
 &#39;authorities&#39;,
 &#39;to&#39;,
 &#39;allow&#39;,
 &#39;disease&#39;,
 &#39;prediction&#39;,
 &#39;services&#39;,
 &#39;as&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;including&#39;,
 &#39;cancer&#39;,
 &#39;diabetes&#39;,
 &#39;and&#39;,
 &#39;heart&#39;,
 &#39;disease&#39;,
 &#39;without&#39;,
 &#39;disclosing&#39;,
 &#39;their&#39;,
 &#39;models&#39;,
 &#39;and&#39;,
 &#39;performances&#39;,
 &#39;to&#39;,
 &#39;the&#39;,
 &#39;scientific&#39;,
 &#39;or&#39;,
 &#39;medical&#39;,
 &#39;societies&#39;,
 &#39;If&#39;,
 &#39;it&#39;,
 &#39;is&#39;,
 &#39;allowed&#39;,
 &#39;it&#39;,
 &#39;will&#39;,
 &#39;cause&#39;,
 &#39;the&#39;,
 &#39;profound&#39;,
 &#39;confusion&#39;,
 &#39;or&#39;,
 &#39;chaos&#39;,
 &#39;to&#39;,
 &#39;the&#39;,
 &#39;consumers&#39;,
 &#39;medical&#39;,
 &#39;institutions&#39;,
 &#39;and&#39;,
 &#39;health&#39;,
 &#39;insurance&#39;,
 &#39;system&#39;,
 &#39;and&#39;,
 &#39;the&#39;,
 &#39;huge&#39;,
 &#39;waste&#39;,
 &#39;of&#39;,
 &#39;medical&#39;,
 &#39;resources&#39;,
 &#39;Currently&#39;,
 &#39;the&#39;,
 &#39;facility&#39;,
 &#39;and&#39;,
 &#39;human&#39;,
 &#39;resources&#39;,
 &#39;in&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;company&#39;,
 &#39;are&#39;,
 &#39;not&#39;,
 &#39;regulated&#39;,
 &#39;by&#39;,
 &#39;law&#39;,
 &#39;There&#39;,
 &#39;is&#39;,
 &#39;no&#39;,
 &#39;penalty&#39;,
 &#39;to&#39;,
 &#39;their&#39;,
 &#39;rejection&#39;,
 &#39;to&#39;,
 &#39;proficiency&#39;,
 &#39;test&#39;,
 &#39;by&#39;,
 &#39;law&#39;,
 &#39;For&#39;,
 &#39;improving&#39;,
 &#39;the&#39;,
 &#39;situation&#39;,
 &#39;with&#39;,
 &#39;the&#39;,
 &#39;above&#39;,
 &#39;mentioned&#39;,
 &#39;direction&#39;,
 &#39;the&#39;,
 &#39;legal&#39;,
 &#39;coercion&#39;,
 &#39;is&#39;,
 &#39;inevitable&#39;,
 &#39;Amendment&#39;,
 &#39;of&#39;,
 &#39;Bioethics&#39;,
 &#39;and&#39;,
 &#39;Safety&#39;,
 &#39;Act&#39;,
 &#39;or&#39;,
 &#39;making&#39;,
 &#39;law&#39;,
 &#39;equivalent&#39;,
 &#39;to&#39;,
 &#39;Clinical&#39;,
 &#39;Laboratory&#39;,
 &#39;Improvement&#39;,
 &#39;Act&#39;,
 &#39;CLIA&#39;,
 &#39;in&#39;,
 &#39;United&#39;,
 &#39;States&#39;,
 &#39;for&#39;,
 &#39;appropriate&#39;,
 &#39;regulation&#39;,
 &#39;is&#39;,
 &#39;prerequisite&#39;,
 &#39;step&#39;,
 &#39;The&#39;,
 &#39;predictive&#39;,
 &#39;models&#39;,
 &#39;of&#39;,
 &#39;DTC&#39;,
 &#39;genetic&#39;,
 &#39;testing&#39;,
 &#39;need&#39;,
 &#39;the&#39;,
 &#39;validation&#39;,
 &#39;before&#39;,
 &#39;implementation&#39;,
 &#39;But&#39;,
 &#39;practically&#39;,
 &#39;not&#39;,
 &#39;all&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;tests&#39;,
 &#39;can&#39;,
 &#39;be&#39;,
 &#39;validated&#39;,
 &#39;or&#39;,
 &#39;achieved&#39;,
 &#39;to&#39;,
 &#39;the&#39;,
 &#39;certain&#39;,
 &#39;standard&#39;,
 &#39;If&#39;,
 &#39;so&#39;,
 &#39;disclosing&#39;,
 &#39;all&#39;,
 &#39;the&#39;,
 &#39;relevant&#39;,
 &#39;information&#39;,
 &#39;transparently&#39;,
 &#39;for&#39;,
 &#39;the&#39;,
 &#39;right&#39;,
 &#39;choices&#39;,
 &#39;by&#39;,
 &#39;the&#39;,
 &#39;consumers&#39;,
 &#39;instead&#39;,
 &#39;can&#39;,
 &#39;be&#39;,
 &#39;considered&#39;,
 &#39;The&#39;,
 &#39;relevant&#39;,
 &#39;information&#39;,
 &#39;includes&#39;,
 &#39;the&#39;,
 &#39;explanation&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;whole&#39;,
 &#39;process&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;tests&#39;,
 &#39;the&#39;,
 &#39;references&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;method&#39;,
 &#39;they&#39;,
 &#39;used&#39;,
 &#39;disclosing&#39;,
 &#39;the&#39;,
 &#39;implication&#39;,
 &#39;and&#39;,
 &#39;limitation&#39;,
 &#39;of&#39;,
 &#39;the&#39;,
 &#39;results&#39;,
 &#39;how&#39;,
 &#39;to&#39;,
 &#39;deliver&#39;,
 &#39;the&#39;,
 &#39;report&#39;,
 &#39;to&#39;,
 &#39;consumers&#39;,
 &#39;etc&#39;,
 &#39;And&#39;,
 &#39;the&#39;,
 &#39;company&#39;,
 &#39;should&#39;,
 &#39;consider&#39;,
 &#39;the&#39;,
 &#39;possible&#39;,
 &#39;outcomes&#39;,
 &#39;after&#39;,
 &#39;getting&#39;,
 &#39;traits&#39;,
 &#39;risk&#39;,
 &#39;and&#39;,
 &#39;provide&#39;,
 &#39;the&#39;,
 &#39;report&#39;,
 &#39;guidelines&#39;,
 &#39;not&#39;,
 &#39;to&#39;,
 &#39;cause&#39;,
 &#39;any&#39;,
 &#39;harm&#39;,
 &#39;or&#39;,
 &#39;negative&#39;,
 &#39;effect&#39;,
 &#39;to&#39;,
 &#39;the&#39;,
 &#39;consumers&#39;,
 &#39;including&#39;,
 &#39;psychological&#39;,
 &#39;distress&#39;,
 &#39;or&#39;,
 &#39;waste&#39;,
 &#39;of&#39;,
 &#39;medical&#39;,
 &#39;resources&#39;,
 &#39;New&#39;,
 &#39;genomic&#39;,
 &#39;technologies&#39;,
 &#39;and&#39;,
 &#39;knowledges&#39;,
 &#39;expand&#39;,
 &#39;our&#39;,
 &#39;view&#39;,
 &#39;and&#39;,
 &#39;their&#39;,
 &#39;applications&#39;,
 &#39;would&#39;,
 &#39;improve&#39;,
 &#39;human&#39;,
 &#39;health&#39;,
 &#39;Medical&#39;,
 &#39;diagnosis&#39;,
 &#39;and&#39;,
 &#39;treatment&#39;,
 &#39;is&#39;,
 &#39;shifting&#39;,
 &#39;to&#39;,
 &#39;genetic&#39;,
 &#39;based&#39;,
 &#39;precision&#39;,
 &#39;medicine&#39;,
 &#39;Its&#39;,
 &#39;progress&#39;,
 &#39;is&#39;,
 &#39;strictly&#39;,
 ...]</pre>
</div>

</div>

</div>

</div>

</div><div class="jp-Cell jp-CodeCell jp-Notebook-cell jp-mod-noOutputs  ">
<div class="jp-Cell-inputWrapper">
<div class="jp-InputArea jp-Cell-inputArea">
<div class="jp-InputPrompt jp-InputArea-prompt">In&nbsp;[&nbsp;]:</div>
<div class="jp-CodeMirrorEditor jp-Editor jp-InputArea-editor" data-type="inline">
     <div class="CodeMirror cm-s-jupyter">
<div class=" highlight hl-ipython3"><pre><span></span> 
</pre></div>

     </div>
</div>
</div>
</div>

</div>
</body>







</html>
