{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "57053f14",
   "metadata": {},
   "outputs": [],
   "source": [
    "from urllib import request\n",
    "\n",
    "url = \"https://genominfo.org/journal/view.php?number=38\"\n",
    "response = request.urlopen(url)\n",
    "raw = response.read().decode('utf8')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "900daf09",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "str"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "type(raw)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "a41183b2",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "' (NCEP) Expert Panel on Detection, Evaluation, and Treatment of High Blood Cholesterol in Adults (Adult Treatment Panel III) final report; citation_journal_title=Circulation; citation_volume=106; citation_issue=106; citation_pages=3143-3421; citation_date=2002;\\'/>\\n<meta name=\\'citation_reference\\' content=\\'citation_title=Fasting compared with nonfasting triglycerides and risk of cardiovascular events in women; citation_author=S Bansal; citation_author=JE Buring; citation_author=N Rifai; citation_author=S Mora; citation_author=FM Sacks; citation_author=PM Ridker; citation_journal_title=JAMA; citation_volume=298; citation_issue=298; citation_pages=309-316; citation_date=2007;\\'/>\\n<meta name=\\'citation_reference\\' content=\\'citation_title=Implications of recent clinical trials for the national cholesterol education program adult treatment panel III guidelines; citation_author=SM Grundy; citation_author=JI Cleeman; citation_author=CN Merz; citation_author=HB Brewer; citation_author=LT Clark; citation_author=DB Hunninghake; citation_journal_title=Circulation; citation_volume=110; citation_issue=110; citation_pages=227-239; citation_date=2004;\\'/>\\n<meta name=\\'citation_reference\\' content=\\'citation_title=HDL as a target in the treatment of atherosclerotic cardiovascular disease; citation_author=P Linsel-Nitschke; citation_author=AR Tall; citation_journal_title=Nat Rev Drug Discov; citation_volume=4; citation_issue=4; citation_pages=193-205; citation_date=2005;\\'/>\\n<meta name=\\'citation_reference\\' content=\\'<html xmlns:mml=\"http://www.w3.org/1998/Math/MathML\">\\'/>\\n<meta name=\\'citation_reference\\' content=\\'citation_title=Polymorphisms associated with cholesterol and risk of cardiovascular events; citation_author=S Kathiresan; citation_author=O Melander; citation_author=D Anevski; citation_author=C Guiducci; citation_author=NP Burtt; citation_author=C Roos; citation_journal_title=N Engl J Med; citation_volume=358; citation_issue=358; citation_pages=1240-1249; citation_date=2008;\\'/>\\n<meta '"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "raw[8000:10000]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "6f5e93c4",
   "metadata": {},
   "outputs": [],
   "source": [
    "from bs4 import BeautifulSoup\n",
    "raw = BeautifulSoup(raw, 'html.parser').get_text()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "a287d00f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'\\n\\n\\n\\n\\n\\nInteraction Effects of Lipoprotein Lipase Polymorphisms with Lifestyle on Lipid Levels in a Korean Population: A Cross-sectional Study\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nHome\\nE-Submission\\nSitemap\\nContact Us\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nHOME\\nABOUT\\n\\nAims and Scope\\nJournal catalog\\nEditorial Board\\nBest Practice\\nOpen Access Policy\\nContact us\\n\\n\\nARTICLE CATEGORY\\xa0\\n\\n\\n\\nOriginal Article\\nReview Article\\nEditorial\\nApplication Note\\nBrief Communication\\nClinical Genomics\\nGenomic archive\\nResearch Communication\\nOpinion\\nOthers\\n\\nBrowse all articles >\\n\\n\\n\\nBROWSE ARTICLES\\xa0\\n\\n\\n\\nNew Issue\\nAll Issues\\nMost View\\nMost Download\\nMost Cited\\nFunded Articles\\nGenomics Inform Search\\nAuthor Index\\n\\n\\n\\n\\n\\nFOR CONTRIBUTORS\\xa0\\n\\nInstructions for Authors\\nGuideline for Reviewers\\nPublication Ethics\\nAuthor’s Checklist\\nE-Submission\\nArticle-processing Charge\\nCopyright Transfer Agreement\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nSearch\\n\\n\\n\\nGenomics Inform Search\\n\\n\\n\\n\\n\\nCLOSE\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nGenomics'"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "raw[:1000]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "4d116fd9",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "\"\\n\\n\\n\\n\\n\\nInteraction Effects of Lipoprotein Lipase Polymorphisms with Lifestyle on Lipid Levels in a Korean Population: A Cross-sectional Study\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nHome\\nE-Submission\\nSitemap\\nContact Us\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nHOME\\nABOUT\\n\\nAims and Scope\\nJournal catalog\\nEditorial Board\\nBest Practice\\nOpen Access Policy\\nContact us\\n\\n\\nARTICLE CATEGORY\\xa0\\n\\n\\n\\nOriginal Article\\nReview Article\\nEditorial\\nApplication Note\\nBrief Communication\\nClinical Genomics\\nGenomic archive\\nResearch Communication\\nOpinion\\nOthers\\n\\nBrowse all articles >\\n\\n\\n\\nBROWSE ARTICLES\\xa0\\n\\n\\n\\nNew Issue\\nAll Issues\\nMost View\\nMost Download\\nMost Cited\\nFunded Articles\\nGenomics Inform Search\\nAuthor Index\\n\\n\\n\\n\\n\\nFOR CONTRIBUTORS\\xa0\\n\\nInstructions for Authors\\nGuideline for Reviewers\\nPublication Ethics\\nAuthor’s Checklist\\nE-Submission\\nArticle-processing Charge\\nCopyright Transfer Agreement\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nSearch\\n\\n\\n\\nGenomics Inform Search\\n\\n\\n\\n\\n\\nCLOSE\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nGenomics Inform > Volume 10(2); 2012 > Article\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nPyun, Kim, Park, Baik, Cho, Koh, Lee, Cho, Kim, Go, Shim, Kwack, and Shin: Interaction Effects of Lipoprotein Lipase Polymorphisms with Lifestyle on Lipid Levels in a Korean Population: A Cross-sectional Study\\n\\n\\n\\nArticle \\n\\n\\nGenomics & Informatics 2012; 10(2): 88-98.\\n\\nPublished online: June 30, 2012 \\nDOI: https://doi.org/10.5808/GI.2012.10.2.88\\n\\n\\n\\n\\nInteraction Effects of Lipoprotein Lipase Polymorphisms with Lifestyle on Lipid Levels in a Korean Population: A Cross-sectional Study\\n\\nJung-A Pyun 1, Sunshin Kim 1, KyungChae Park 2, Inkyung Baik 3, Nam H Cho 4, InSong Koh 5, Jong-Young Lee 6, Yoon Shin Cho 6, Young Jin Kim 6, Min Jin Go 6, Eugene Shim 7, KyuBum Kwack 1, Chol Shin 8\\n\\n\\n1Department of Biomedical Science, College of Life Science, CHA University, Seongnam 463-836, Korea.\\n2Department of Family Medicine, CHA Bundang Medical Center, CHA University School of Medicine, Seongnam 463-712, Korea.\\n3Department of Foods and Nutrition, College of Natural Sciences, Kookmin University, Seoul 136-702, Korea.\\n4Department of Preventive Medicine, Ajou University School of Medicine, Suwon 443-749, Korea.\\n5Department of Physiology, Hanyang University College of Medicine, Seoul 133-791, Korea.\\n6Center for Genome Science, National Institute of Health, Osong 363-951, Korea.\\n7National Hypertension Center, Yonsei University Health System, Seoul 120-752, Korea.\\n8Division of Pulmonary and Critical Care Medicine, Department of Internal Medicine, Korea University Ansan Hospital, Korea University College of Medicine, Ansan 425-707, Korea.\\n\\n\\nCorresponding author: Tel: +82-31-725-8376, Fax: +82-31-725-8350, kbkwack@cha.ac.kr\\n \\n\\n Received April 15, 2012 \\xa0 \\xa0 \\xa0 Revised May 13, 2012 \\xa0 \\xa0 \\xa0 Accepted May 23, 2012\\n\\nCopyright © 2012 by The Korea Genome Organization \\n (open-access, http://creativecommons.org/licenses/by-nc/3.0):  \\nIt is identical to the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/).\\n\\n\\n\\n\\nAbstract\\n\\nLipoprotein lipase (LPL) plays an essential role in the regulation of high-density lipoprotein cholesterol (HDLC) and triglyceride levels, which have been closely associated with cardiovascular diseases. Genetic studies in European have shown that LPL single-nucleotide polymorphisms (SNPs) are strongly associated with lipid levels. However, studies about the influence of interactions between LPL SNPs and lifestyle factors have not been sufficiently performed. Here, we examine if LPL polymorphisms, as well as their interaction with lifestyle factors, influence lipid concentrations in a Korean population. A two-stage association study was performed using genotype data for SNPs on the LPL gene, including the 3' flanking region from 7,536 (stage 1) and 3,703 (stage 2) individuals. The association study showed that 15 SNPs and 4 haplotypes were strongly associated with HDLC (lowest p = 2.86 × 10-22) and triglyceride levels (lowest p = 3.0 × 10-15). Interactions between LPL polymorphisms and lifestyle factors (lowest p = 9.6 × 10-4) were also observed on lipid concentrations. These findings suggest that there are interaction effects of LPL polymorphisms with lifestyle variables, including energy intake, fat intake, smoking, and alcohol consumption, as well as effects of LPL polymorphisms themselves, on lipid concentrations in a Korean population.\\nKeywords: interaction, lipoprotein lipase, lipoproteins, lipids, single nucleotide polymorphism \\n\\n\\n\\n\\n\\nIntroduction\\nLipoprotein lipase (LPL) plays an important role in the metabolism and transport of lipids. It hydrolyzes triglycerides (TGs) in lipoproteins, such as chylomicrons (CMs) and very low-density lipoproteins (VLDLs), into free fatty acids, releasing CM remnants and intermediate-density lipoproteins that act on high-density lipoprotein cholesterol (HDLC) metabolism. Plasma LPL therefore plays an essential role in elevating HDLC and lowering TG [1-3].\\nWhile HDLC plays a role in the removal and transport of cholesterol from peripheral tissues to other tissues where it is required, such as the liver, low-density lipoprotein cholesterol (LDLC) is, conversely, transported from the liver to peripheral tissues. The concentration of lipids has been tightly associated with the risk of cardiovascular diseases [4-6]. Specifically, LDLC levels have been shown to have a positive log-linear association with the risk of coronary heart disease: a reduction of 1% in the serum concentration of LDLC has been shown to reduce the relative risk of coronary heart disease risk by approximately 1% [7], whereas an increase of 1 mg/dL in serum HDLC levels is associated with a decrease in the risk of coronary heart disease of approximately 2% to 3%, independent of LDLC levels [8]. It was also reported that TG levels are associated with risk of cardiovascular disease, independent of concentrations of other lipids and cardiac risk factors, although there has been controversy [6]. Recently, a report based on the Korea National Health and Nutrition Examination Surveys I, II, III, and IV has indicated that the prevalence of dyslipidemia and hypertriglyceridemia in this country has increased during the last decade [9]. Genetic association studies aimed at identifying the genetic factors associated with variations in lipid concentrations have been performed mostly for Europeans, and they have shown that the polymorphisms in the LPL gene, including the 3' flanking region, are strongly associated with HDLC and TG levels [10-13]. There is a report of a candidate gene approach for lipid concentrations in a Korean population, but the LPL gene was not included in the study [14].\\nThe concentration of lipids is also influenced by environmental factors, such as smoking, alcohol consumption, diet, and physical activity, as well as genetic factors [5, 15]. However, studies about the influence of interactions between LPL polymorphisms and lifestyle factors have not been sufficiently performed. In this study, we examined the effects of potential interactions between LPL polymorphisms and lifestyle, as well as genetic variations of the LPL gene, including the 3' flanking region, on the lipid profile of a Korean population.\\n\\n\\nMethods\\n\\n Subjects\\nData for this study were obtained from the Korean Genome Epidemiology Study (KoGES), conducted by the Korean National Institute of Health (KNIH). The study comprised two stages. In the first stage, a cross-sectional analysis was conducted of samples from urban and rural communities from South Korea, whereas in the second stage, the study was replicated in other urban regions in Korea. Subjects who were examined in the first stage, described in a previous report [16], were recruited from a rural (Ansung) and an urban (Ansan) community from South Korea, which have been studied since 2001 as a part of KoGES. In total, 5,018 and 5,020 subjects from the Ansung and Ansan communities, respectively, participated in the study. The age of the participants ranged from 40 to 69 years. Samples were scrutinized for quality control purposes, after which 8,842 subjects remained in our sample [16]. We further excluded 1,306 subjects who were on medication for hypertension, diabetes, myocardial infarction, and hyperlipidemia. Information about the lifestyle of the subjects was based on their responses to questionnaires.\\nFor the second stage of the study, 4,302 samples were obtained from both urban regions, Seoul and Busan, in South Korea that were established from 2006 to 2007 as part of KoGES. Samples satisfying any of the following criteria were excluded: i) gender inconsistencies, ii) genotype call rate lower than 95%, iii) excessive heterozygosity, iv) cryptic first degree relative, v) outliers on a multidimensional scaling plot, or vi) cancer history. A total of 3,703 subjects remained in our sample. These exclusion criteria for samples have been described [17]. Details of the sample characteristics are listed in Table 1.\\n\\n\\n Genotyping\\nWe obtained genotype data for single-nucleotide polymorphisms (SNPs) located in the LPL gene, including the 3' flanking region, from KoGES, which are accessible to the research community through the Korean Associated Resource project. The study protocol was approved by the Institutional Review Board of KNIH. For genotyping of the samples from the Ansung and Ansan cohorts, Affymetrix Genome-Wide Human SNP Array 5.0 (Affymetrix Inc., Santa Clara, CA, USA) was used. Affymetrix Genome-Wide Human SNP Array 6.0 (Affymetrix Inc.) was used for genotyping samples from the Seoul and Busan cohorts. The Bayesian Robust Linear Model with the Mahalanobis distance algorithm was used to determine the genotypes at each SNP of Affymetrix 5.0. The BirdSeed calling genotyping algorithm was used on Affymetrix 6.0. The SNPs were filtered if any of the following criteria was met: i) a call rate lower than 95%, ii) a minor allele frequency (MAF) lower than 0.05, or iii) significant deviation from the Hardy-Weinberg equilibrium (HWE). Significant deviation from HWE for 28 successfully genotyped SNPs was determined based on a threshold of p < 0.002 (0.05/28).\\nSNP imputation was performed using the IMPUTE program (https://mathgen.stats.ox.ac.uk/impute/impute.html). A reference panel from International HapMap (phase II/release 22/NCBI build 36 and dbSNP build 126) data, comprising SNPs from 90 Japanese (JPT) and Han Chinese (CHB) individuals, was used as a template for inferring untyped genotypes. Imputed SNPs that were located in the LPL gene, including the 3' flanking region, with low quality were excluded according to the following criteria: low genotype information content (information lower than 0.5), a posterior probability score lower than 0.90, a call rate lower than 0.95, and MAF lower than 0.05.\\n\\n\\n Biochemical measures\\nBiochemical data from KoGES were obtained through the Korean Association Resource (KARE) project. Blood samples were obtained from the participants with at least 8 hours of overnight fasting. Serum total cholesterol (TCHL), HDLC, and TG concentrations were measured using the Advia 1650 analyzer (Siemens, Tarrytown, NY, USA) for the Ansung and Ansan cohorts and the Hitachi 7600 analyzer (Hitachi, Tokyo, Japan) for the Seoul and Busan study cohorts [16, 17]. LDLC concentration was estimated using the equation by Friedewald et al. [18] for individuals with TG levels lower than 400 mg/dL.\\n\\n\\n Statistics\\nTo test the association between LPL SNPs and haplotypes (HTs) and lipid concentration, linear regression analyses, adjusted for age, gender, and body mass index (BMI), were performed. For stage 1 and imputed data, the residence area of each participant was additionally adjusted. The influence of the interaction between LPL SNPs and lifestyle variables on lipid levels was analyzed by using linear regression, including main and interaction effects, adjusted for residence area, age, gender, and BMI. The effect of energy and fat intake, physical activity, smoking, and drinking status on lipid profiles was examined. Energy intake, fat intake, and physical activity were measured as the total intake of kilocalories per day, daily fat intake, and metabolic equivalent scores per hour, respectively. Smoking status had two categories, non-smokers and smokers, and alcohol consumption was divided into non-drinkers and drinkers. These lifestyle factors were available for subjects from stage 1 and imputed data but not from stage 2.\\nAll analyses were performed by log-transforming HDLC, TG, and TCHL values to normalize their distribution. Untransformed LDLC values were used for all analyses. To determine the effect of SNPs, untransformed lipid concentrations were used.\\nLinkage disequilibrium (LD) block partition was performed by Gabriel's rule, applied in Haploview version 4.1 (Broad Institute of MIT and Harvard University, Boston, MA, USA), and HT inference was performed using the expectation maximization algorithm with PLINK version 1.07 (http://pngu.mgh.harvard.edu/~purcell/plink). All statistical analyses were performed in an additive model (AA vs. AB vs. BB or +/+ vs. +/- vs. -/-) using PLINK.\\n\\n\\n\\nResults\\n\\n Association of LPL SNPs with lipid concentrations\\nWe sequentially discarded 6 SNPs with MAF < 0.05, 2 SNPs with call rate < 95%, and 1 SNP with HWE p-value < 0.002, leaving 19 SNPs for the association study at stage 1. Our results show that SNPs located in the LPL gene, including the 3' flanking region, were strongly associated with HDLC and TG levels. The linear regression, adjusted for age, gender, and BMI, using combined data from the two study stages, revealed that 15 SNPs were associated with HDLC and TG levels (Table 2). The strongest associations were between rs10503669 and both HDLC (combined p = 3.6 × 10-22; 2.2-mg/dL increase per minor T allele) and TG (combined p = 3 × 10-15; 13.6-mg/dL decrease per minor T allele) in additive models. However, no strong association was detected between the SNP and either LDLC or TCHL levels (Supplementary Fig. 1).\\nFor LPL SNPs, we tested 2 intronic SNPs and 2 SNPs located in the 3' untranslated region (UTR) from stage 1. Of these, two SNPs in the 3' UTR (rs11570892 and rs3200218) were not significantly associated with any lipid concentration (Supplementary Table 1). Two intronic SNPs, rs263 and rs271, were significantly associated to higher HDLC levels (p = 1.2 × 10-10 and p = 2.5 × 10-11; 1.2- and 1.3-mg/dL increase per minor A and T allele, respectively) and lower TG levels (p = 1.7 × 10-6 and p = 7.2 × 10-7; 8.8- and 9-mg/dL decrease per minor A and T allele, respectively) (Table 2). In stage 2, 7 intronic SNPs and 2 SNPs that were located in the 3' UTR were tested. It was found that rs263 and rs271 were also strongly associated with increased HDLC (p = 5.5 × 10-6 and p = 5.1 × 10-6, respectively; both showed a 1.5-mg/dL increase per minor A and T allele) (Table 2) and decreased TG levels (p = 3.3 × 10-3 and p = 2.2 × 10-3; 8.6- and 8.7-mg/dL decrease per minor A and T allele, respectively). Additionally, 4 intronic SNPs (rs253, rs326, rs327, and rs12679834) were significantly associated with increased HDLC (p = 4.2 × 10-3, p = 1.5 × 10-9, p = 1.6 × 10-9 and p = 3.1 × 10-14; increase of 0.8, 2.1, 2.1, and 3.3 mg/dL per minor allele, respectively) and decreased TG levels (p = 3.6 × 10-3, p = 9.1 × 10-7, p = 5.9 × 10-7 and p = 1.3 × 10-6; increase of 7.4, 12.2, 12.3, and 13.7 mg/dL per minor allele, respectively) (Supplementary Table 2). The analysis using combined data from the two stages showed that rs263 and rs271 were also significantly associated with HDLC (p = 1.7 × 10-11 and p = 5.7 × 10-12, respectively; increase in both cases of 1.2 mg/dL per minor A and T allele) and TG levels (p = 6.8 × 10-7 and p = 2.7 × 10-7; decrease of 8.1, 8.3 mg/dL per minor A and T allele, respectively) (Table 2).\\n\\n\\n Association of LPL HTs with lipid concentration\\nTwo blocks, including SNPs located in the LPL gene, were identified (Supplementary Fig. 2), and the association of HT with lipid profiles for the two blocks was examined. The analysis using combined data from the two stages showed that block 2, which includes SNPs located in the LPL gene, including the 3' flanking region, was the region most strongly associated with lipid concentrations (Table 3). HT3 in block 2 was strongly associated with increased HDLC levels (p = 2.86 × 10-22; 2.25-mg/dL increase per HT3) and reduced TG levels (p = 9.09 × 10-15; 13.7-mg/dL decrease per HT3), whereas HT1 was strongly associated with reduced HDLC levels (p = 1.1 × 10-14; 1.2-mg/dL decrease per HT1) and higher TG levels (p = 3.05 × 10-12; 8.6-mg/dL increase per HT1). There was no significant association between HT and either LDLC or TCHL.\\nThe analysis using imputed data also showed a strong association between HTs and lipid levels in the 2 blocks, including LPL SNPs (Supplementary Table 3). HT1 in block 2 was significantly associated with a reduced HDLC level (p = 4.2 × 10-15; 1.3-mg/dL decrease per HT1) and an increased level of TG (p = 1.9 × 10-8; 8-mg/dL increase per HT1), whereas HT3 was significantly associated with increased HDLC (p = 7.8 × 10-16; 2-mg/dL increase per HT3) and LDLC levels (p = 5.1 × 10-3; 2.3-mg/dL increases per HT3) and decreased TG levels (p = 5.6 × 10-13; 15.1-mg/dL decrease per HT3).\\n\\n\\n Effect of the interaction between polymorphisms located in the LPL gene and lifestyle on lipid concentration\\nWe tested LPL SNPs and HTs from stage 1. All significant associations were found in the results of the dominant model for the minor allele. The association of minor alleles for 3 SNPs (rs263, rs271, rs328) with higher HDLC levels also depended on energy intake (Supplementary Fig. 3). Of these, the minor alleles of 2 SNPs (rs263, rs271) were associated with lower TG levels, also depending on energy intake. The association of the minor alleles of two SNPs (rs263, rs271) with higher HDLC levels also depended on fat intake (Fig. 1). The minor alleles of 2 SNPs (rs263, rs271) were also associated with higher HDLC levels and lower TG levels through an interaction with cigarette smoking and with lower TG levels through an interaction with alcohol consumption (Fig. 2, Supplementary Fig. 4). Interaction effects of HT1 and 2 in block 1, which were formed by two SNP (rs271, rs263) in strong LD (r2 = 0.99), with lifestyle factors on HDLC and TG levels were found to be almost same with those of rs271 and rs263 (data not shown). In block 2, HT3 was associated with higher HDLC levels and lower TG levels and depended on an interaction with energy intake (Fig. 3, Supplementary Fig. 3). HT4 in block 2 was associated with lower TG levels through an interaction with cigarette smoking (Supplementary Fig. 4). For the interaction between LPL polymorphisms and physical activity, there was no significant association with any lipid levels.\\n\\n\\n\\nDiscussion\\nThe human LPL gene is located on chromosome 8p22 and includes 10 exons encoding 475 amino acids, of which 448 amino acids are part of the mature protein and 27 of the signal peptide [19, 20]. LPL has an essential role in the metabolism of lipids, as it hydrolyzes a number of substrates, such as long- and short-chain glycerides and phospholipids in vitro, as well as major substrates, such as CMs and VLDL in vivo [21]. Dysfunction of LPL has been reported to be associated with chylomicronemia, atherosclerosis, obesity, Alzheimer's disease, and dyslipidemia [22].\\nGenetic association studies have shown that some SNPs located in the LPL gene, including the 3' flanking region, are strongly associated with HDLC and TG levels, mostly in Europeans. Specifically, by performing meta-analyses of genomewide association studies, Aulchenko et al. [13] found that rs2083637 and rs10096633 were associated with HDLC and TG levels, and it was also demonstrated that rs325 and rs10105606, reported by Waterworth et al. [11], and rs12678919, reported by Kathiresan et al. [23], were associated with HDLC and TG levels. Studies using the candidate gene approach, which examines the effects of genetic variants of a putative gene in an association, have reported that some LPL SNPs are associated with HDLC and TG levels. Tang et al. [24] recently revealed that rs326 and rs328 (which convert serine to a stop codon) and rs13702 are associated with HDLC and TG levels in African-Americans and European-American populations.\\nHere, rs263 and rs271, located, respectively, in introns 5 and 6, were strongly associated with increased HDLC and decreased TG levels (Table 2). The two significant SNPs formed an HT block with strong LD (r2 = 0.99) in stage 1, and rs3200218, located in the 3' UTR, formed another HT block with 15 SNPs located in the 3' flanking region. At stage 2, nine SNPs in the LPL gene remained after SNP filtering, out of which six were significantly associated with HDLC levels (Supplementary Table 2). SNPs rs343, rs253, rs263, and rs271, located, respectively, in introns 3, 4, 5, and 6, formed an HT block with moderate LD (r2 > 0.3). SNPs rs326, rs327, rs12679834, rs11570892n, and rs3200218 formed another HT block with 31 SNPs located in the 3' flanking region. In the two blocks at stages 1 and 2, some HTs were found to be strongly associated with HDLC and TG levels (Supplementary Tables 4 and 5). The analysis using the combined data from both stages showed that HTs were also significantly associated with HDLC and TG levels (Table 3). Using the imputation data, we found that more LPL SNPs were included in the two blocks. HTs that were significantly associated with lipid profiles were also identified (Supplementary Table 3).\\nThe strongest association of HT with HDLC and TG levels was observed in block 2 (from both stages). A previous study reported that one HT was associated with increased LPL activity [25]. The HT was found to be consistent with HT3 in block 2, found using stage 1 and imputed data, which was associated here with increased HDLC and decreased TG levels. In block 2, we found that one SNP, rs328, which converts serine to a stop codon, was included using imputed data. Interestingly, the minor allele G of rs328 was found only in HT3. It has been reported that this minor allele is associated with increased HDLC and decreased TG concentrations in Japanese [26] and European populations [10, 27], and our results support this finding. This truncated variant was also demonstrated to have higher catalytic activity [28-30].\\nSNP rs328 has been reported to be in strong LD with rs17482753 in European (r2 = 1) and African (r2 = 0.852) populations [31]. rs17482753 was genotyped in stage 1 and 2 of this study, and the SNP was found to be in strong LD with rs328, according to the imputed data (r2 = 0.993). We additionally identified strong LD (r2 = 0.93) between rs328 and rs17482753 by using genotype data of 830 Korean participants obtained from a previous study [32], in which samples in stage 1 of this study were included. Similar to the results of rs328 obtained using the imputed data in this study, the rs17482753 minor allele was found to be strongly associated with raised HDLC and reduced TG levels in a dominant manner at both stages.\\nIn a previous study, LPL SNPs located in the 3' UTR region were considered possible targets affecting miRNA binding sites, and 5 SNPs were predicted to be putative miRNA binding sites by polymiRTS [33]. The 3' UTR region of the LPL gene was included in block 2 in this study, and two of the predicted 5 SNPs were tested (Supplementary Table 1). Rs1059611 major allele T from the imputed data was significantly associated with reduced levels of HDLC and LDLC and with increased TG levels in an additive manner. In this SNP, the major T allele was predicted to create a new miRNA (miR-136) binding site, causing abnormal repression of this gene. The major allele of another SNP, rs11570892, was predicted to disrupt a conserved miRNA (miR-592) binding site, and this could cause abnormal control of normal repression. However, this SNP showed inconsistent association with LDLC levels in stages 1 and 2 of this study.\\nSome environmental factors have been suggested to be associated with lipid profiles. Physical activity was reported to be associated with higher HDLC and lower TG and TCHL concentrations [34], whereas smoking was reported to be associated with higher LDLC and TG and lower HDLC levels [35, 36]. Alcohol consumption was suggested to be associated with increased HDLC levels [37, 38], whereas excessive alcohol consumption was associated with a higher TG level [39]. A low fat intake, in turn, was associated with reduced TCHL, LDLC, and HDLC and increased TG levels [40]. Some LPL SNPs have also been shown to be associated with lipid profiles through their interaction with smoking, alcohol consumption, and physical activity. Specifically, rs320 has been suggested to interact with smoking [41, 42] and physical activity [42], and rs1801177 interacts with physical activity [43]. The SNP rs328, which is a nonsense SNP included in our imputed data, has been associated with HDL through its interaction with smoking and alcohol consumption in females but not in males [44]. In a recent study, however, the interaction of several LPL SNPs, including rs328, with alcohol consumption was suggested to have no influence on HDLC levels [45].\\nIn this study, we show that the effects of some LPL polymorphisms on lipid concentrations were regulated by several environmental factors. Carriers of each minor allele of 3 SNPs (rs263, rs271, and rs328) were associated with slightly higher HDLC levels, depending on an interaction with energy intake. This effect, however, was not observed among non-carriers (Supplementary Fig. 3). In two of these SNPs (rs263 and rs271), which were in strong LD (r2 = 0.99 in stages 1 and 2), each major genotype carrier led to higher levels of TG, whereas the TG levels of each minor allele carrier were not affected by a higher energy intake. Carriers of each minor allele, but not non-carriers, led to a higher HDLC level through an interaction with a higher intake of fat (Fig. 1). Each major genotype carrier of two SNPs (rs263 and rs271) showed lower HDLC levels and higher TG levels than minor allele carriers through an interaction with smoking (Fig. 2). In the major genotype carrier of these two SNPs, higher TG levels were also found by interacting with alcohol consumption (Supplementary Fig. 4), whereas each minor allele carrier was less affected than non-carriers by smoking or alcohol consumption.\\nFor interactions of HTs, blocks 1 and 2 were shown to interact with inconsistent lifestyle variables. Interaction effects of HT1 and 2 in block 1 with lifestyle factors, which were formed by the two SNPs, on lipid levels were almost same with those of rs263 and rs271 (data not shown). In the LPL gene, 61 missense mutations have been identified, and interestingly, 42 of them are located in exons 5 and 6, which may be contained in block 1 [46]. Carrier of HT3 in block 2, which was reported to be associated with increased LPL activity [25], showed higher HDLC levels and lower TG levels, whereas non-carriers were not affected by a higher energy intake (Fig. 3, Supplementary Fig. 3). In the non-carrier of HT4 in block 2, higher TG levels were found by interacting with cigarette smoking (Supplementary Fig. 4). Smoking and moderate alcohol consumption were reported to be related with reduced and increased LPL activity, respectively [47, 48], but the precise mechanisms by which the interaction of LPL polymorphisms with environmental factors affects lipid concentrations have not been demonstrated yet.\\nIn conclusion, we found that HTs are strongly associated with HDLC and TG levels in the 2 blocks in which LPL SNPs are included and that SNPs in these blocks are also strongly associated with HDLC and TG levels in stage 1. Interestingly, these results were replicated in stage 2, which consists of independent samples, and the combined analyses, including the samples of stages 1 and 2, showed the most significant results. Moreover, by analyzing the effect of the interaction of LPL polymorphisms with several lifestyle factors on plasma lipid levels, we have shown that the effects of some LPL SNPs and HTs on HDLC and TG concentrations are regulated by energy and fat intake, smoking, and alcohol consumption. Since these results have not been reported in any previous studies, they are novel findings. These interaction effects suggest that the appropriate lifestyle of each individual could be recommended according to the type of LPL polymorphism. These results should be confirmed in other populations and in a longitudinal study.\\n\\n\\n\\n\\nAcknowledgments\\nThis research was performed within the Consortium for Large Scale Genome Wide Association Study III (2011E7300400), which was supported by the genotyping data (the Korean Genome Analysis Project, 4845-301) and the phenotypic data (the Korean Genome Epidemiology Study, 4851-302) from the Korea Center for Disease Control. This study was also supported by grants from Korea Health 21 R&D Project, Ministry of Health and Welfare, Republic of Korea (A080734), and the Priority Centers Program of the National Research Foundation of Korea (NRF), which is funded by the Ministry of Education, Science, and Technology (2009-0093821).\\n\\n\\nSupplementary materials\\n\\nSupplementary data including 5 tables and 4 figures can be found with with this article online at http://www.genominfo.org/src/sm/gni-10-88-s001.pdf.\\ngni-10-88-s001.pdf\\n\\n\\n\\nReferences\\n\\n\\n1. Lusis AJ, Pajukanta P. A treasure trove for lipoprotein biology. Nat Genet 2008;40:129–130. PMID: 18227868.   \\n2. Eckel RH. Lipoprotein lipase: a multifunctional enzyme relevant to common metabolic diseases. N Engl J Med 1989;320:1060–1068. PMID: 2648155.   \\n3. Goldberg IJ, Merkel M. Lipoprotein lipase: physiology, biochemistry, and molecular biology. Front Biosci 2001;6:D388–D405. PMID: 11229871.  \\n4. Prospective Studies Collaboration. Lewington S, Whitlock G, Clarke R, Sherliker P, Emberson J, et al. Blood cholesterol and vascular mortality by age, sex, and blood pressure: a meta-analysis of individual data from 61 prospective studies with 55,000 vascular deaths. Lancet 2007;370:1829–1839. PMID: 18061058.   \\n5. National Cholesterol Education Program (NCEP) Expert panel on detection, evaluation, and treatment of high blood cholesterol in adults (adult treatment panel III). Third report of the National Cholesterol Education Program (NCEP) Expert Panel on Detection, Evaluation, and Treatment of High Blood Cholesterol in Adults (Adult Treatment Panel III) final report. Circulation 2002;106:3143–3421. PMID: 12485966.  \\n6. Bansal S, Buring JE, Rifai N, Mora S, Sacks FM, Ridker PM. Fasting compared with nonfasting triglycerides and risk of cardiovascular events in women. JAMA 2007;298:309–316. PMID: 17635891.   \\n7. Grundy SM, Cleeman JI, Merz CN, Brewer HB Jr, Clark LT, Hunninghake DB, et al. Implications of recent clinical trials for the national cholesterol education program adult treatment panel III guidelines. Circulation 2004;110:227–239. PMID: 15249516.   \\n8. Linsel-Nitschke P, Tall AR. HDL as a target in the treatment of atherosclerotic cardiovascular disease. Nat Rev Drug Discov 2005;4:193–205. PMID: 15738977.   \\n9. Korea Center for Disease Control. The Forth Korea National Health and Nutrition Examination Survey (KNHANES IV-1), 2007. 2008. Seoul: Korea Center for Disease Control and Prevention.\\n10. Kathiresan S, Melander O, Anevski D, Guiducci C, Burtt NP, Roos C, et al. Polymorphisms associated with cholesterol and risk of cardiovascular events. N Engl J Med 2008;358:1240–1249. PMID: 18354102.   \\n11. Waterworth DM, Ricketts SL, Song K, Chen L, Zhao JH, Ripatti S, et al. Genetic variants influencing circulating lipid levels and risk of coronary artery disease. Arterioscler Thromb Vasc Biol 2010;30:2264–2276. PMID: 20864672.    \\n12. Ridker PM, Paré G, Parker AN, Zee RY, Miletich JP, Chasman DI. Polymorphism in the CETP gene region, HDL cholesterol, and risk of future myocardial infarction: genomewide analysis among 18 245 initially healthy women from the Women's Genome Health Study. Circ Cardiovasc Genet 2009;2:26–33. PMID: 20031564.    \\n13. Aulchenko YS, Ripatti S, Lindqvist I, Boomsma D, Heid IM, Pramstaller PP, et al. Loci influencing lipid levels and coronary heart disease risk in 16 European population cohorts. Nat Genet 2009;41:47–55. PMID: 19060911.   \\n14. Park MH, Kim N, Lee JY, Park HY. Genetic loci associated with lipid concentrations and cardiovascular risk factors in the Korean population. J Med Genet 2011;48:10–15. PMID: 20972250.    \\n15. Pollin TI, Hsueh WC, Steinle NI, Snitker S, Shuldiner AR, Mitchell BD. A genome-wide scan of serum lipid levels in the Old Order Amish. Atherosclerosis 2004;173:89–96. PMID: 15177127.   \\n16. Cho YS, Go MJ, Kim YJ, Heo JY, Oh JH, Ban HJ, et al. A large-scale genome-wide association study of Asian populations uncovers genetic factors influencing eight quantitative traits. Nat Genet 2009;41:527–534. PMID: 19396169.   \\n17. Kim YJ, Go MJ, Hu C, Hong CB, Kim YK, Lee JY, et al. Large-scale genome-wide association studies in East Asians identify new genetic loci influencing metabolic traits. Nat Genet 2011;43:990–995. PMID: 21909109.   \\n18. Friedewald WT, Levy RI, Fredrickson DS. Estimation of the concentration of low-density lipoprotein cholesterol in plasma, without use of the preparative ultracentrifuge. Clin Chem 1972;18:499–502. PMID: 4337382.   \\n19. Sparkes RS, Zollman S, Klisak I, Kirchgessner TG, Komaromy MC, Mohandas T, et al. Human genes involved in lipolysis of plasma lipoproteins: mapping of loci for lipoprotein lipase to 8p22 and hepatic lipase to 15q21. Genomics 1987;1:138–144. PMID: 3692485.   \\n20. Deeb SS, Peng RL. Structure of the human lipoprotein lipase gene. Biochemistry 1989;28:4131–4135. PMID: 2765475.   \\n21. Murthy V, Julien P, Gagne C. Molecular pathobiology of the human lipoprotein lipase gene. Pharmacol Ther 1996;70:101–135. PMID: 8843465.   \\n22. Mead JR, Irvine SA, Ramji DP. Lipoprotein lipase: structure, function, regulation, and role in disease. J Mol Med (Berl) 2002;80:753–769. PMID: 12483461.   \\n23. Kathiresan S, Willer CJ, Peloso GM, Demissie S, Musunuru K, Schadt EE, et al. Common variants at 30 loci contribute to polygenic dyslipidemia. Nat Genet 2009;41:56–65. PMID: 19060906.    \\n24. Tang W, Apostol G, Schreiner PJ, Jacobs DR Jr, Boerwinkle E, Fornage M. Associations of lipoprotein lipase gene polymorphisms with longitudinal plasma lipid trends in young adults: The Coronary Artery Risk Development in Young Adults (CARDIA) Study. Circ Cardiovasc Genet 2010;3:179–186. PMID: 20150529.    \\n25. Goodarzi MO, Wong H, Quiñones MJ, Taylor KD, Guo X, Castellani LW, et al. The 3' untranslated region of the lipoprotein lipase gene: haplotype structure and association with post-heparin plasma lipase activity. J Clin Endocrinol Metab 2005;90:4816–4823. PMID: 15928243.   \\n26. Nakayama K, Bayasgalan T, Yamanaka K, Kumada M, Gotoh T, Utsumi N, et al. Large scale replication analysis of loci associated with lipid concentrations in a Japanese population. J Med Genet 2009;46:370–374. PMID: 19487539.   \\n27. Rip J, Nierman MC, Ross CJ, Jukema JW, Hayden MR, Kastelein JJ, et al. Lipoprotein lipase S447X: a naturally occurring gain-of-function mutation. Arterioscler Thromb Vasc Biol 2006;26:1236–1245. PMID: 16574898.   \\n28. Skoglund-Andersson C, Ehrenborg E, Fisher RM, Olivecrona G, Hamsten A, Karpe F. Influence of common variants in the CETP, LPL, HL and APO E genes on LDL heterogeneity in healthy, middle-aged men. Atherosclerosis 2003;167:311–317. PMID: 12818414.   \\n29. Groenemeijer BE, Hallman MD, Reymer PW, Gagné E, Kuivenhoven JA, Bruin T, et al. REGRESS Study Group. Genetic variant showing a positive interaction with beta-blocking agents with a beneficial influence on lipoprotein lipase activity, HDL cholesterol, and triglyceride levels in coronary artery disease patients. The Ser447-stop substitution in the lipoprotein lipase gene. Circulation 1997;95:2628–2635. PMID: 9193431.   \\n30. Henderson HE, Kastelein JJ, Zwinderman AH, Gagné E, Jukema JW, Reymer PW, et al. Lipoprotein lipase activity is decreased in a large cohort of patients with coronary artery disease and is associated with changes in lipids and lipoproteins. J Lipid Res 1999;40:735–743. PMID: 10191298.  \\n31. Deo RC, Reich D, Tandon A, Akylbekova E, Patterson N, Waliszewska A, et al. Genetic differences between the determinants of lipid profile phenotypes in African and European Americans: the Jackson Heart Study. PLoS Genet 2009;5:e1000342. PMID: 19148283.    \\n32. Cho YS, Go MJ, Han HR, Cha SH, Kim HT, Min H, et al. Association of lipoprotein lipase (LPL) single nucleotide polymorphisms with type 2 diabetes mellitus. Exp Mol Med 2008;40:523–532. PMID: 18985010.    \\n33. Heid IM, Boes E, Müller M, Kollerits B, Lamina C, Coassin S, et al. Genome-wide association analysis of high-density lipoprotein cholesterol in the population-based KORA study sheds new light on intergenic regions. Circ Cardiovasc Genet 2008;1:10–20. PMID: 20031538.   \\n34. Marrugat J, Elosua R, Covas MI, Molina L, Rubiés-Prat J. The MARATHOM Investigators. Amount and intensity of physical activity, physical fitness, and serum lipids in men. Am J Epidemiol 1996;143:562–569. PMID: 8610673.   \\n35. Freeman DJ, Griffin BA, Murray E, Lindsay GM, Gaffney D, Packard CJ, et al. Smoking and plasma lipoproteins in man: effects on low density lipoprotein cholesterol levels and high density lipoprotein subfraction distribution. Eur J Clin Invest 1993;23:630–640. PMID: 8281981.   \\n36. Craig WY, Palomaki GE, Haddow JE. Cigarette smoking and serum lipid and lipoprotein concentrations: an analysis of published data. BMJ 1989;298:784–788. PMID: 2496857.    \\n37. De Oliveira E Silva ER, Foster D, McGee Harper M, Seidman CE, Smith JD, Breslow JL, et al. Alcohol consumption raises HDL cholesterol levels by increasing the transport rate of apolipoproteins A-I and A-II. Circulation 2000;102:2347–2352. PMID: 11067787.   \\n38. Gaziano JM, Buring JE, Breslow JL, Goldhaber SZ, Rosner B, VanDenburgh M, et al. Moderate alcohol intake, increased levels of high-density lipoprotein and its subfractions, and decreased risk of myocardial infarction. N Engl J Med 1993;329:1829–1834. PMID: 8247033.   \\n39. Bessembinders K, Wielders J, van de Wiel A. Severe hypertriglyceridemia influenced by alcohol (SHIBA). Alcohol Alcohol 2011;46:113–116. PMID: 21245063.   \\n40. Brinton EA, Eisenberg S, Breslow JL. A low-fat diet decreases high density lipoprotein (HDL) cholesterol levels by decreasing HDL apolipoprotein transport rates. J Clin Invest 1990;85:144–151. PMID: 2104877.    \\n41. Peacock RE, Temple A, Gudnason V, Rosseneu M, Humphries SE. Variation at the lipoprotein lipase and apolipoprotein AI-CIII gene loci are associated with fasting lipid and lipoprotein traits in a population sample from Iceland: interaction between genotype, gender, and smoking status. Genet Epidemiol 1997;14:265–282. PMID: 9181356.   \\n42. Sentí M, Elosua R, Tomás M, Sala J, Masiá R, Ordovás JM, et al. Physical activity modulates the combined effect of a common variant of the lipoprotein lipase gene and smoking on serum triglyceride levels and high-density lipoprotein cholesterol in men. Hum Genet 2001;109:385–392. PMID: 11702219.   \\n43. Boer JM, Kuivenhoven JA, Feskens EJ, Schouten EG, Havekes LM, Seidell JC, et al. Physical activity modulates the effect of a lipoprotein lipase mutation (D9N) on plasma lipids and lipoproteins. Clin Genet 1999;56:158–163. PMID: 10517255.   \\n44. Lee J, Tan CS, Chia KS, Tan CE, Chew SK, Ordovas JM, et al. The lipoprotein lipase S447X polymorphism and plasma lipids: interactions with APOE polymorphisms, smoking, and alcohol consumption. J Lipid Res 2004;45:1132–1139. PMID: 15060087.   \\n45. Marques-Vidal P, Bochud M, Paccaud F, Waterworth D, Bergmann S, Preisig M, et al. No interaction between alcohol consumption and HDL-related genes on HDL cholesterol levels. Atherosclerosis 2010;211:551–557. PMID: 20430392.   \\n46. Merkel M, Eckel RH, Goldberg IJ. Lipoprotein lipase: genetics, lipid uptake, and regulation. J Lipid Res 2002;43:1997–2006. PMID: 12454259.   \\n47. Freeman DJ, Caslake MJ, Griffin BA, Hinnie J, Tan CE, Watson TD, et al. The effect of smoking on post-heparin lipoprotein and hepatic lipase, cholesteryl ester transfer protein and lecithin: cholesterol acyl transferase activities in human plasma. Eur J Clin Invest 1998;28:584–591. PMID: 9726040.   \\n48. Nishiwaki M, Ishikawa T, Ito T, Shige H, Tomiyasu K, Nakajima K, et al. Effects of alcohol on lipoprotein lipase, hepatic lipase, cholesteryl ester transfer protein, and lecithin: cholesterol acyltransferase in high-density lipoprotein cholesterol elevation. Atherosclerosis 1994;111:99–109. PMID: 7840818.   \\n\\n\\n\\n\\n\\n\\nFig.\\xa01\\nEffect of the interaction of rs271, which was in strong linkage disequilibrium with rs263 (r2 = 0.99), with daily fat intake on high-density lipoprotein cholesterol (HDLC) levels (p = 9.6 × 10-4). Subjects were divided into daily fat intake ≤ 20 g (TT or TC, 774; CC, 1,314), 21-30 g (TT or TC, 695; CC, 1,183), 31-40 g (TT or TC, 547; CC, 934), and ≥ 41 g (TT or TC, 702; CC, 1,144). Interaction p-value was calculated by linear regression analysis, which includes main and interaction effects, adjusted for residence area, age, gender, and body mass index.\\n\\n\\n\\nFig.\\xa02\\nEffect of the interaction of rs271, which was in strong linkage disequilibrium with rs263 (r2 = 0.99), with cigarette smoking on high-density lipoprotein cholesterol (HDLC) (A) and triglyceride (TG) (B) levels (p = 5.7 × 10-3 and p = 8.60 × 10-3, respectively). Subjects were divided into non-smokers (TT or TC, 1,629; CC, 2,689) and smokers (TT or TC, 700; CC, 1,289). Interaction p-values were calculated by linear regression analyses, which includes main and interaction effects, adjusted for residence area, age, gender, and body mass index.\\n\\n\\n\\nFig.\\xa03\\nEffect of the interaction of haplotype 3 in block 2 (TGTGTAGAGCTCCTCG) with daily energy intake on high-density lipoprotein cholesterol (HDLC) level (p = 5.6 × 10-3). Subjects were divided into daily energy intake ≤ 1,500 kcal (HT3/HT3 or HT3/-, 387; -/-, 1,353), 1,501-2,000 kcal (HT3/HT3 or HT3/-, 646; -/-, 2,297), 2001-2,500 kcal (HT3/HT3 or HT3/-, 358; -/-, 1,216), and ≥ 2,501 kcal (HT3/HT3 or HT3/-, 235; -/-, 801). Interaction p-values were calculated by linear regression analyses, which includes main and interaction effects, adjusted for residence area, age, gender, and body mass index. HT, haplotype.\\n\\n\\n\\nTable\\xa01.\\nDescription of subjects used in stages 1 and 2 of the study\\n\\n\\n\\n\\nParameter\\n\\nStage 1\\n\\n\\nStage 2\\n\\n\\n\\n\\nAnsung (n = 3,434)\\n\\nAnsan (n = 4,102)\\n\\nTotal (n = 7,536)\\n\\nTotal (n = 3,703)\\n\\n\\n\\n\\n\\nAgea\\n\\n\\n54.8 ± 8.8\\n\\n48.2 ± 7.4\\n\\n51.2 ± 8.7\\n\\n53.2 ± 8.3\\n\\n\\n\\nGender (%)\\n\\n\\n\\n\\n\\n\\n\\n\\u2003Malea\\n\\n\\n1,534 (44.7)\\n\\n2,104 (51.3)\\n\\n3,638 (48.3)\\n\\n1,651 (44.6)\\n\\n\\n\\n\\u2003Femalea\\n\\n\\n1,900 (55.3)\\n\\n1,998 (48.7)\\n\\n3,898 (51.7)\\n\\n2,052 (55.4)\\n\\n\\n\\nBMI (kg/m2)a\\n\\n\\n24.2 ± 3.2\\n\\n24.5 ± 2.9\\n\\n24.4 ± 3.1\\n\\n24.0 ± 2.9\\n\\n\\n\\nSystolic blood pressure (mm Hg)a\\n\\n\\n123.8 ± 17.6\\n\\n115.4 ± 16.3\\n\\n119.2 ± 17.4\\n\\n121.9 ± 14.8\\n\\n\\n\\nDiastolic blood pressure (mm Hg)a\\n\\n\\n81.3 ± 10.6\\n\\n77.4 ± 11.3\\n\\n79.2 ± 11.2\\n\\n77.2 ± 10.2\\n\\n\\n\\nLipid concentration (mg/dL)\\n\\n\\n\\n\\n\\n\\n\\n\\u2003HDLCa\\n\\n\\n45.1 ± 10.3\\n\\n45.0 ± 9.9\\n\\n45.0 ± 10.1\\n\\n54.6 ± 13.3\\n\\n\\n\\n\\u2003LDLCb\\n\\n\\n109.0 ± 31.0\\n\\n120.6 ± 31.4\\n\\n115.3 ± 31.8\\n\\n119.0 ± 31.4\\n\\n\\n\\n\\u2003TGa\\n\\n\\n158.8 ± 103.3\\n\\n156.8 ± 101.7\\n\\n157.7 ± 102.4\\n\\n123.3 ± 91.4\\n\\n\\n\\n\\u2003TCHLa\\n\\n\\n184.5 ± 34.5\\n\\n195.7 ± 35.5\\n\\n190.6 ± 35.5\\n\\n197.4 ± 35.1\\n\\n\\n\\nEnergy intake (kcal/day)c\\n\\n\\n1,977.6 ± 916.7\\n\\n1,895.9 ± 516.0\\n\\n1,932.2 ± 723.2\\n\\n-\\n\\n\\n\\nPhysical activity (MET/h)d\\n\\n\\n3.8 ± 1.3\\n\\n2.4 ± 0.6\\n\\n3.0 ± 0.7\\n\\n-\\n\\n\\n\\nAlcohol consumption (%)e\\n\\n\\n\\n\\n\\n\\n\\n\\n\\u2003Non-drinker\\n\\n1,628 (48.2)\\n\\n1,733 (42.3)\\n\\n3,361 (45.0)\\n\\n-\\n\\n\\n\\n\\u2003Ex-drinker\\n\\n255 (7.5)\\n\\n196 (4.8)\\n\\n451 (6.0)\\n\\n-\\n\\n\\n\\n\\u2003Current drinker\\n\\n1,495 (44.3)\\n\\n2,164 (52.9)\\n\\n3,659 (49.0)\\n\\n-\\n\\n\\n\\nSmoking status (%)f\\n\\n\\n\\n\\n\\n\\n\\n\\n\\u2003Non-smoker\\n\\n1,987 (59.3)\\n\\n2,331 (57.1)\\n\\n4,318 (58.1)\\n\\n-\\n\\n\\n\\n\\u2003Ex-smoker\\n\\n403 (12.0)\\n\\n724 (17.7)\\n\\n1,127 (15.2)\\n\\n-\\n\\n\\n\\n\\u2003Current smoker\\n\\n962 (28.7)\\n\\n1,027 (25.2)\\n\\n1,989 (26.8)\\n\\n-\\n\\n\\n\\n\\n\\n Quantitative parameters are shown as mean ± SD or as the number observed, followed by its corresponding percentage.\\nBMI, body mass index; HDLC, high-density lipoprotein cholesterol; LDLC, low-density lipoprotein cholesterol; TG, triglyceride; TCHL, total cholesterol; MET, metabolic equivalent.\\n\\n\\na Missing rate of the parameter is 0 in both stages I and II;\\n\\n\\nb Missing rates of the parameter are 0.027, 0.025, 0.026, and 0.015 in Ansung, Ansan, total of stage I, and total of stage II, respectively;\\n\\n\\nc Missing rates of the parameter are 0.056, 0.012, and 0.032 in Ansung, Ansan, and total of stage I, respectively;\\n\\n\\nd Missing rates of the parameter are 0.021, 0.004, and 0.012 in Ansung, Ansan, and total of stage I, respectively;\\n\\n\\ne Missing rates of the parameter are 0.016, 0.002, and 0.009 in Ansung, Ansan, and total of stage I, respectively;\\n\\n\\nf Missing rates of the parameter are 0.024, 0.005, and 0.014 in Ansung, Ansan, and total of stage I, respectively.\\n\\n\\n\\n\\nTable\\xa02.\\nSNPs associated with HDLC and TG concentrations at each stage of the study\\n\\n\\n\\n\\nSNP\\n\\nPosition\\n\\nAlleles\\n\\nAnsung\\n\\n\\nAnsan\\n\\n\\nStage 1 (Ansung ＋ Ansan)\\n\\n\\nStage 2\\n\\n\\nCombined (stage 1 ＋ 2)\\n\\n\\n\\n\\nEffecta\\n\\n\\np-valueb\\n\\n\\nEffecta\\n\\n\\np-valueb\\n\\n\\nEffecta\\n\\n\\np-valuec\\n\\n\\nEffecta\\n\\n\\np-valueb\\n\\n\\nEffecta\\n\\n\\np-valueb\\n\\n\\n\\n\\n\\n\\nHDLC-associated SNPs\\n\\n\\n\\n\\u2003rs263d\\n\\n\\n19812812\\n\\nG＞A\\n\\n1.25\\n\\n1.70 × 10-5\\n\\n\\n1.23\\n\\n1.20 × 10-6\\n\\n\\n1.23\\n\\n1.20 × 10-10\\n\\n\\n1.53\\n\\n5.50 × 10-6\\n\\n\\n1.22\\n\\n1.70 × 10-11\\n\\n\\n\\n\\n\\u2003rs271d\\n\\n\\n19813702\\n\\nC＞T\\n\\n1.32\\n\\n5.30 × 10-6\\n\\n\\n1.24\\n\\n8.30 × 10-7\\n\\n\\n1.27\\n\\n2.50 × 10-11\\n\\n\\n1.53\\n\\n5.10 × 10-6\\n\\n\\n1.24\\n\\n5.70 × 10-12\\n\\n\\n\\n\\n\\u2003rs17482753e\\n\\n\\n19832646\\n\\nG＞T\\n\\n2.27\\n\\n1.90 × 10-10\\n\\n\\n1.57\\n\\n4.60 × 10-7\\n\\n\\n1.88\\n\\n1.10 × 10-15\\n\\n\\n3.21\\n\\n1.20 × 10-13\\n\\n\\n2.19\\n\\n1.20 × 10-21\\n\\n\\n\\n\\n\\u2003rs10503669\\n\\n19847690\\n\\nG＞T\\n\\n2.31\\n\\n1.10 × 10-10\\n\\n\\n1.53\\n\\n1.30 × 10-6\\n\\n\\n1.89\\n\\n2.00 × 10-15\\n\\n\\n3.18\\n\\n1.90 × 10-13\\n\\n\\n2.24\\n\\n3.60 × 10-22\\n\\n\\n\\n\\n\\u2003rs17410962\\n\\n19848080\\n\\nG＞A\\n\\n2.29\\n\\n1.30 × 10-10\\n\\n\\n1.56\\n\\n5.50 × 10-7\\n\\n\\n1.88\\n\\n1.00 × 10-15\\n\\n\\n3.21\\n\\n1.20 × 10-13\\n\\n\\n2.18\\n\\n1.70 × 10-21\\n\\n\\n\\n\\n\\u2003rs17411031\\n\\n19852310\\n\\nC＞G\\n\\n1.59\\n\\n3.80 × 10-8\\n\\n\\n1.28\\n\\n8.80 × 10-7\\n\\n\\n1.42\\n\\n2.10 × 10-13\\n\\n\\n1.94\\n\\n9.40 × 10-9\\n\\n\\n1.47\\n\\n1.30 × 10-15\\n\\n\\n\\n\\n\\u2003rs17489282\\n\\n19852518\\n\\nG＞A\\n\\n1.6\\n\\n2.50 × 10-8\\n\\n\\n1.19\\n\\n4.70 × 10-6\\n\\n\\n1.38\\n\\n8.80 × 10-13\\n\\n\\n1.93\\n\\n1.10 × 10-8\\n\\n\\n1.45\\n\\n2.70 × 10-15\\n\\n\\n\\n\\n\\u2003rs4922117\\n\\n19852586\\n\\nA＞G\\n\\n1.58\\n\\n4.70 × 10-8\\n\\n\\n1.28\\n\\n8.60 × 10-7\\n\\n\\n1.42\\n\\n2.50 × 10-13\\n\\n\\n1.96\\n\\n7.20 × 10-9\\n\\n\\n1.5\\n\\n3.90 × 10-16\\n\\n\\n\\n\\n\\u2003rs17411126\\n\\n19855272\\n\\nT＞C\\n\\n1.57\\n\\n5.60 × 10-8\\n\\n\\n1.28\\n\\n8.50 × 10-7\\n\\n\\n1.41\\n\\n2.90 × 10-13\\n\\n\\n2\\n\\n4.00 × 10-9\\n\\n\\n1.47\\n\\n1.20 × 10-15\\n\\n\\n\\n\\n\\u2003rs765547\\n\\n19866274\\n\\nC＞T\\n\\n1.57\\n\\n5.50 × 10-8\\n\\n\\n1.27\\n\\n9.60 × 10-7\\n\\n\\n1.41\\n\\n3.20 × 10-13\\n\\n\\n1.92\\n\\n1.40 × 10-8\\n\\n\\n1.46\\n\\n1.40 × 10-15\\n\\n\\n\\n\\n\\u2003rs11986942\\n\\n19867445\\n\\nG＞C\\n\\n1.57\\n\\n6.30 × 10-8\\n\\n\\n1.24\\n\\n1.60 × 10-6\\n\\n\\n1.39\\n\\n6.20 × 10-13\\n\\n\\n1.94\\n\\n9.30 × 10-9\\n\\n\\n1.47\\n\\n1.40 × 10-15\\n\\n\\n\\n\\n\\u2003rs1837842\\n\\n19868290\\n\\nT＞C\\n\\n1.57\\n\\n5.50 × 10-8\\n\\n\\n1.24\\n\\n1.70 × 10-6\\n\\n\\n1.39\\n\\n5.40 × 10-13\\n\\n\\n1.92\\n\\n1.40 × 10-8\\n\\n\\n1.48\\n\\n9.60 × 10-16\\n\\n\\n\\n\\n\\u2003rs1919484\\n\\n19869676\\n\\nC＞T\\n\\n1.58\\n\\n5.20 × 10-8\\n\\n\\n1.23\\n\\n2.20 × 10-6\\n\\n\\n1.39\\n\\n6.90 × 10-13\\n\\n\\n1.92\\n\\n1.30 × 10-8\\n\\n\\n1.46\\n\\n1.90 × 10-15\\n\\n\\n\\n\\n\\u2003rs7461115\\n\\n19871540\\n\\nG＞C\\n\\n1.57\\n\\n4.90 × 10-8\\n\\n\\n1.23\\n\\n2.40 × 10-6\\n\\n\\n1.39\\n\\n7.20 × 10-13\\n\\n\\n2.01\\n\\n4.00 × 10-9\\n\\n\\n1.5\\n\\n4.80 × 10-16\\n\\n\\n\\n\\n\\u2003rs7013777\\n\\n19878356\\n\\nA＞G\\n\\n1.51\\n\\n1.40 × 10-7\\n\\n\\n1.28\\n\\n6.80 × 10-7\\n\\n\\n1.39\\n\\n4.80 × 10-13\\n\\n\\n2.05\\n\\n1.80 × 10-9\\n\\n\\n1.51\\n\\n2.50 × 10-16\\n\\n\\n\\n\\nTG-associated SNPs\\n\\n\\n\\n\\u2003rs263d\\n\\n\\n19812812\\n\\nG＞A\\n\\n−8.73\\n\\n5.10 × 10-4\\n\\n\\n−8.82\\n\\n9.30 × 10-4\\n\\n\\n−8.79\\n\\n1.70 × 10-6\\n\\n\\n−8.58\\n\\n3.30 × 10-3\\n\\n\\n−8.13\\n\\n6.80 × 10-7\\n\\n\\n\\n\\n\\u2003rs271d\\n\\n\\n19813702\\n\\nC＞T\\n\\n−8.81\\n\\n3.90 × 10-4\\n\\n\\n−9.14\\n\\n5.20 × 10-4\\n\\n\\n−9\\n\\n7.20 × 10-7\\n\\n\\n−8.71\\n\\n2.20 × 10-3\\n\\n\\n−8.31\\n\\n2.70 × 10-7\\n\\n\\n\\n\\n\\u2003rs17482753e\\n\\n\\n19832646\\n\\nG＞T\\n\\n−13.56\\n\\n1.60 × 10-6\\n\\n\\n−16.01\\n\\n6.60 × 10-8\\n\\n\\n−14.83\\n\\n6.00 × 10-13\\n\\n\\n−12.97\\n\\n3.30 × 10-6\\n\\n\\n−13.59\\n\\n4.60 × 10-15\\n\\n\\n\\n\\n\\u2003rs10503669\\n\\n19847690\\n\\nG＞T\\n\\n−14.04\\n\\n7.20 × 10-7\\n\\n\\n−15.79\\n\\n1.10 × 10-7\\n\\n\\n−14.94\\n\\n4.60 × 10-13\\n\\n\\n−12.44\\n\\n8.20 × 10-6\\n\\n\\n−13.64\\n\\n3.00 × 10-15\\n\\n\\n\\n\\n\\u2003rs17410962\\n\\n19848080\\n\\nG＞A\\n\\n−13.58\\n\\n1.60 × 10-6\\n\\n\\n−16.3\\n\\n2.90 × 10-8\\n\\n\\n−14.99\\n\\n2.80 × 10-13\\n\\n\\n−12.52\\n\\n6.90 × 10-6\\n\\n\\n−13.53\\n\\n6.40 × 10-15\\n\\n\\n\\n\\n\\u2003rs17411031\\n\\n19852310\\n\\nC＞G\\n\\n−10.26\\n\\n1.90 × 10-5\\n\\n\\n−10.19\\n\\n6.20 × 10-5\\n\\n\\n−10.26\\n\\n4.50 × 10-9\\n\\n\\n−12.18\\n\\n6.80 × 10-7\\n\\n\\n−10.27\\n\\n3.70 × 10-12\\n\\n\\n\\n\\n\\u2003rs17489282\\n\\n19852518\\n\\nG＞A\\n\\n−10.42\\n\\n1.80 × 10-5\\n\\n\\n−9.75\\n\\n1.30 × 10-4\\n\\n\\n−10.09\\n\\n9.60 × 10-9\\n\\n\\n−11.92\\n\\n1.40 × 10-6\\n\\n\\n−10.13\\n\\n9.40 × 10-12\\n\\n\\n\\n\\n\\u2003rs4922117\\n\\n19852586\\n\\nA＞G\\n\\n−10.25\\n\\n1.70 × 10-5\\n\\n\\n−9.94\\n\\n1.00 × 10-4\\n\\n\\n−10.12\\n\\n6.80 × 10-9\\n\\n\\n−12.22\\n\\n6.80 × 10-7\\n\\n\\n−10.28\\n\\n2.80 × 10-12\\n\\n\\n\\n\\n\\u2003rs17411126\\n\\n19855272\\n\\nT＞C\\n\\n−10.29\\n\\n1.80 × 10-5\\n\\n\\n−10.17\\n\\n7.30 × 10-5\\n\\n\\n−10.27\\n\\n5.10 × 10-9\\n\\n\\n−12.59\\n\\n3.10 × 10-7\\n\\n\\n−10.36\\n\\n2.90 × 10-12\\n\\n\\n\\n\\n\\u2003rs765547\\n\\n19866274\\n\\nC＞T\\n\\n−10.31\\n\\n1.70 × 10-5\\n\\n\\n−9.6\\n\\n6.20 × 10-5\\n\\n\\n−9.98\\n\\n4.10 × 10-9\\n\\n\\n−11.9\\n\\n2.10 × 10-6\\n\\n\\n−10.01\\n\\n6.70 × 10-12\\n\\n\\n\\n\\n\\u2003rs11986942\\n\\n19867445\\n\\nG＞C\\n\\n−10.28\\n\\n1.90 × 10-5\\n\\n\\n−9.43\\n\\n9.40 × 10-5\\n\\n\\n−9.87\\n\\n6.80 × 10-9\\n\\n\\n−11.95\\n\\n1.90 × 10-6\\n\\n\\n−9.99\\n\\n7.70 × 10-12\\n\\n\\n\\n\\n\\u2003rs1837842\\n\\n19868290\\n\\nT＞C\\n\\n−10.23\\n\\n2.00 × 10-5\\n\\n\\n−9.18\\n\\n1.10 × 10-4\\n\\n\\n−9.72\\n\\n8.10 × 10-9\\n\\n\\n−12.12\\n\\n1.60 × 10-6\\n\\n\\n−9.98\\n\\n5.50 × 10-12\\n\\n\\n\\n\\n\\u2003rs1919484\\n\\n19869676\\n\\nC＞T\\n\\n−10.36\\n\\n1.60 × 10-5\\n\\n\\n−9.34\\n\\n1.00 × 10-4\\n\\n\\n−9.86\\n\\n6.30 × 10-9\\n\\n\\n−11.96\\n\\n1.80 × 10-6\\n\\n\\n−9.98\\n\\n7.10 × 10-12\\n\\n\\n\\n\\n\\u2003rs7461115\\n\\n19871540\\n\\nG＞C\\n\\n−10.24\\n\\n3.10 × 10-5\\n\\n\\n−9.14\\n\\n1.20 × 10-4\\n\\n\\n−9.71\\n\\n1.40 × 10-8\\n\\n\\n−11.7\\n\\n3.70 × 10-6\\n\\n\\n−9.83\\n\\n2.10 × 10-11\\n\\n\\n\\n\\n\\u2003rs7013777\\n\\n19878356\\n\\nA＞G\\n\\n−10.37\\n\\n2.90 × 10-5\\n\\n\\n−9.56\\n\\n7.40 × 10-5\\n\\n\\n−9.99\\n\\n7.50 × 10-9\\n\\n\\n−12.02\\n\\n1.20 × 10-6\\n\\n\\n−10.12\\n\\n5.20 × 10-12\\n\\n\\n\\n\\n\\n\\n SNP, single-nucleotide polymorphism; HDLC, high-density lipoprotein cholesterol; TG, triglyceride; LPL, lipoprotein lipase; LD, linkage disequilibrium.\\n\\n\\na Unstandardized coefficients of untransformed values (effect). Effect sizes (mg/dL) of minor alleles assessed in an additive manner;\\n\\n\\nb p-values of the minor allele (assessed in an additive manner) adjusted for age, gender, and BMI;\\n\\n\\nc p-values of the minor allele (assessed in an additive manner) adjusted for residence area, age, gender, and body mass index;\\n\\n\\nd SNPs located in intronic region of LPL gene;\\n\\n\\ne This SNP was in strong LD (r2 = 0.93) with a nonsense SNP, rs328 in a Korean population.\\n\\n\\n\\n\\nTable\\xa03.\\nAssociation between haplotypes, including LPL SNPs and lipid concentrations, using combined data from stages 1 and 2\\n\\n\\n\\n\\nBlock\\n\\nHT\\n\\nHT frequency\\n\\n\\nHDLC\\n\\n\\nTG\\n\\n\\nLDLC\\n\\n\\nTCHL\\n\\n\\n\\n\\n＋＋\\n\\n＋－\\n\\n－－\\n\\nEffecta\\n\\n\\np-valueb\\n\\n\\nEffecta\\n\\n\\np-valueb\\n\\n\\nEffecta\\n\\n\\np-valueb\\n\\n\\nEffecta\\n\\n\\np-valueb\\n\\n\\n\\n\\n\\n\\nB1\\n\\nHT1: AT\\n\\n490 (4.4)\\n\\n3,645 (32.4)\\n\\n7,103 (63.2)\\n\\n1.24\\n\\n8.30 × 10−12\\n\\n\\n−8.27\\n\\n3.70 × 10−7\\n\\n\\n0.92\\n\\n0.08\\n\\n0.96\\n\\n0.11\\n\\n\\n\\n\\nHT2: GC\\n\\n7,065 (62.9)\\n\\n3,678 (32.7)\\n\\n495 (4.4)\\n\\n−1.23\\n\\n1.10 × 10−11\\n\\n\\n8.18\\n\\n5.00 × 10−7\\n\\n\\n−0.96\\n\\n0.06\\n\\n−1.03\\n\\n0.09\\n\\n\\n\\nB2\\n\\nHT1c\\n\\n\\n4,888 (43.5)\\n\\n5,064 (45.1)\\n\\n1,287 (11.5)\\n\\n−1.19\\n\\n1.05 × 10−14\\n\\n\\n8.6\\n\\n3.05 × 10−12\\n\\n\\n−0.66\\n\\n0.14\\n\\n−0.37\\n\\n0.34\\n\\n\\n\\n\\nHT2d\\n\\n\\n155 (1.4)\\n\\n2,427 (21.6)\\n\\n8,657 (77.0)\\n\\n0.07\\n\\n0.5\\n\\n−2.59\\n\\n0.21\\n\\n0.47\\n\\n0.47\\n\\n0.02\\n\\n0.87\\n\\n\\n\\n\\nHT3e\\n\\n\\n166 (1.5)\\n\\n2,295 (20.4)\\n\\n8,778 (78.1)\\n\\n2.25\\n\\n2.86 × 10−22\\n\\n\\n−13.7\\n\\n9.09 × 10−15\\n\\n\\n1.21\\n\\n0.06\\n\\n1.1\\n\\n0.08\\n\\n\\n\\n\\nHT4f\\n\\n\\n71 (0.6)\\n\\n1,670 (14.9)\\n\\n9,498 (84.5)\\n\\n0.08\\n\\n0.62\\n\\n−2.85\\n\\n0.39\\n\\n−0.68\\n\\n0.38\\n\\n−0.83\\n\\n0.33\\n\\n\\n\\n\\n\\n LPL, lipoprotein lipase; SNP, single-nucleotide polymorphism; HT, haplotype; HDLC, high-density lipoprotein cholesterol; TG, triglyceride; LDLC, low-density lipoprotein cholesterol; TCHL, total cholesterol; B, block.\\n\\n\\na Unstandardized coefficients of untransformed values (effect). Effect sizes (mg/dL) of haplotypes assessed in an additive manner;\\n\\n\\nb p-values of the haplotype (assessed in an additive manner) adjusted for age, gender, and body mass index. p ＜ 0.0083 is considered significant after Bonferroni correction;\\n\\n\\nc Haplotype 1 in block 2: TGGGGGCGATCGTCGA;\\n\\n\\nd Haplotype 2 in block 2: CGGGGGCGATCGTCGA;\\n\\n\\ne Haplotype 3 in block 2: TGTGTAGAGCTCCTCG;\\n\\n\\nf Haplotype 4 in block 2: TAGAGGGAGCTCCTCG.\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\nTOOLS\\n\\n\\n\\n PDF Links\\n\\n PubReaderXML Download \\n Full text via DOI\\n\\n Full text via PMC \\n\\n Download Citation\\n\\n\\n CrossRef TDM\\n\\n Supplement\\n Print\\n\\n\\nShare :\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\t\\t\\t\\t\\tMETRICS\\n\\t\\t\\t\\t\\t\\n\\n\\n\\n\\n\\n9 \\nCrossref\\n\\n\\n\\n\\t\\t\\t\\t\\t\\t\\t\\xa0\\t\\t\\t\\t\\t\\t\\t\\n\\xa0 \\n\\n4,243\\nView\\n\\n\\n\\n\\t\\t\\t\\t\\t\\t\\t48\\t\\t\\t\\t\\t\\t\\t\\nDownload\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\t\\t\\t\\t\\tRelated articles in GNI\\n\\n\\nAssociation of the Human IL-28RA Gene Polymorphisms in a Korean Population with Asthma.2006 September;4(3)Characterization of Single Nucleotide Polymorphisms in 55 Disease-Associated Genes in a Korean Population.2007 December;5(4)Lack of Association of BIRC5 Polymorphisms with Clearance of HBV Infection and HCC Occurrence in a Korean Population.2009 December;7(4)Association Analysis of SERPINB5 Polymorphisms with HBV Clearance and HCC Occurrence in a Korean Population.2010 March;8(1)Association Analysis of TEC Polymorphisms with Aspirin-Exacerbated Respiratory Disease in a Korean Population2014 June;12(2) \\n\\n\\n\\n\\n\\n\\n\\nABOUT\\n\\n\\n\\n\\nAims and Scope\\nJournal catalog\\nEditorial Board\\nBest Practice\\nOpen Access Policy\\nContact us\\n\\n\\n\\n\\n\\n\\nARTICLE CATEGORY\\n\\n\\nOriginal Article\\nReview Article\\nEditorial\\nApplication Note\\nBrief Communication\\nClinical Genomics\\nGenomic archive\\nResearch Communication\\nOpinion\\nOthers\\n\\nBrowse all articles >\\n\\n\\n\\nBROWSE ARTICLES\\n\\n\\nNew Issue\\nAll Issues\\nMost View\\nMost Download\\nMost Cited\\nFunded Articles\\nGenomics Inform Search\\nAuthor Index\\n\\n\\n\\n\\nFOR CONTRIBUTORS\\n\\n\\nInstructions for Authors\\nGuideline for Reviewers\\nPublication Ethics\\nAuthor’s Checklist\\nE-Submission\\nArticle-processing Charge\\nCopyright Transfer Agreement\\n\\n\\n\\n\\n\\n\\n\\nEditorial Office\\n\\n\\n\\t\\t\\t\\t\\tRoom No. 806, 193 Mallijae-ro, Jung-gu, Seoul 04501, Korea\\n\\n\\n\\t\\t\\t\\t\\tTel: +82-2-558-9394\\xa0\\xa0\\xa0\\xa0Fax: +82-2-558-9434\\xa0\\xa0\\xa0\\xa0E-mail: kogo3@kogo.or.kr \\t\\t\\t\\xa0\\xa0\\xa0\\xa0\\xa0\\xa0\\xa0\\xa0\\xa0\\xa0\\xa0\\xa0\\xa0\\xa0\\xa0\\n\\t\\t\\t\\t\\t\\t\\t\\t\\t\\n\\nCopyright © 2022 by Korea Genome Organization.  \\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\\n\""
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "raw[:100000]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "dbf2d953",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://genominfo.org/journal/view.php?number=500\n",
      "https://genominfo.org/journal/view.php?number=501\n",
      "https://genominfo.org/journal/view.php?number=502\n",
      "https://genominfo.org/journal/view.php?number=503\n",
      "https://genominfo.org/journal/view.php?number=504\n",
      "https://genominfo.org/journal/view.php?number=505\n",
      "https://genominfo.org/journal/view.php?number=506\n",
      "https://genominfo.org/journal/view.php?number=507\n",
      "https://genominfo.org/journal/view.php?number=508\n",
      "https://genominfo.org/journal/view.php?number=509\n",
      "https://genominfo.org/journal/view.php?number=510\n",
      "https://genominfo.org/journal/view.php?number=511\n",
      "https://genominfo.org/journal/view.php?number=512\n",
      "https://genominfo.org/journal/view.php?number=513\n",
      "https://genominfo.org/journal/view.php?number=514\n",
      "https://genominfo.org/journal/view.php?number=515\n",
      "https://genominfo.org/journal/view.php?number=516\n",
      "https://genominfo.org/journal/view.php?number=517\n",
      "https://genominfo.org/journal/view.php?number=518\n",
      "https://genominfo.org/journal/view.php?number=519\n",
      "https://genominfo.org/journal/view.php?number=520\n",
      "https://genominfo.org/journal/view.php?number=521\n",
      "https://genominfo.org/journal/view.php?number=522\n",
      "https://genominfo.org/journal/view.php?number=523\n",
      "https://genominfo.org/journal/view.php?number=524\n",
      "https://genominfo.org/journal/view.php?number=525\n",
      "https://genominfo.org/journal/view.php?number=526\n",
      "https://genominfo.org/journal/view.php?number=527\n",
      "https://genominfo.org/journal/view.php?number=528\n",
      "https://genominfo.org/journal/view.php?number=529\n",
      "https://genominfo.org/journal/view.php?number=530\n",
      "https://genominfo.org/journal/view.php?number=531\n",
      "https://genominfo.org/journal/view.php?number=532\n",
      "https://genominfo.org/journal/view.php?number=533\n",
      "https://genominfo.org/journal/view.php?number=534\n",
      "https://genominfo.org/journal/view.php?number=535\n",
      "https://genominfo.org/journal/view.php?number=536\n",
      "https://genominfo.org/journal/view.php?number=537\n",
      "https://genominfo.org/journal/view.php?number=538\n",
      "https://genominfo.org/journal/view.php?number=539\n",
      "https://genominfo.org/journal/view.php?number=540\n",
      "https://genominfo.org/journal/view.php?number=541\n",
      "https://genominfo.org/journal/view.php?number=542\n",
      "https://genominfo.org/journal/view.php?number=543\n",
      "https://genominfo.org/journal/view.php?number=544\n",
      "https://genominfo.org/journal/view.php?number=545\n",
      "https://genominfo.org/journal/view.php?number=546\n",
      "https://genominfo.org/journal/view.php?number=547\n",
      "https://genominfo.org/journal/view.php?number=548\n",
      "https://genominfo.org/journal/view.php?number=549\n",
      "https://genominfo.org/journal/view.php?number=550\n",
      "https://genominfo.org/journal/view.php?number=551\n",
      "https://genominfo.org/journal/view.php?number=552\n",
      "https://genominfo.org/journal/view.php?number=553\n",
      "https://genominfo.org/journal/view.php?number=554\n",
      "https://genominfo.org/journal/view.php?number=555\n",
      "https://genominfo.org/journal/view.php?number=556\n",
      "https://genominfo.org/journal/view.php?number=557\n",
      "https://genominfo.org/journal/view.php?number=558\n",
      "https://genominfo.org/journal/view.php?number=559\n",
      "https://genominfo.org/journal/view.php?number=560\n",
      "https://genominfo.org/journal/view.php?number=561\n",
      "https://genominfo.org/journal/view.php?number=562\n",
      "https://genominfo.org/journal/view.php?number=563\n",
      "https://genominfo.org/journal/view.php?number=564\n",
      "https://genominfo.org/journal/view.php?number=565\n",
      "https://genominfo.org/journal/view.php?number=566\n",
      "https://genominfo.org/journal/view.php?number=567\n",
      "https://genominfo.org/journal/view.php?number=568\n",
      "https://genominfo.org/journal/view.php?number=569\n",
      "https://genominfo.org/journal/view.php?number=570\n",
      "https://genominfo.org/journal/view.php?number=571\n",
      "https://genominfo.org/journal/view.php?number=572\n",
      "https://genominfo.org/journal/view.php?number=573\n",
      "https://genominfo.org/journal/view.php?number=574\n",
      "https://genominfo.org/journal/view.php?number=575\n",
      "https://genominfo.org/journal/view.php?number=576\n",
      "https://genominfo.org/journal/view.php?number=577\n",
      "https://genominfo.org/journal/view.php?number=578\n",
      "https://genominfo.org/journal/view.php?number=579\n",
      "https://genominfo.org/journal/view.php?number=580\n",
      "https://genominfo.org/journal/view.php?number=581\n",
      "https://genominfo.org/journal/view.php?number=582\n",
      "https://genominfo.org/journal/view.php?number=583\n",
      "https://genominfo.org/journal/view.php?number=584\n"
     ]
    }
   ],
   "source": [
    "from urllib import request\n",
    "from bs4 import BeautifulSoup\n",
    "\n",
    "for i in range(500,728):\n",
    "    try:\n",
    "        url = \"https://genominfo.org/journal/view.php?number=\" + str(i)\n",
    "        print(\"https://genominfo.org/journal/view.php?number=\" + str(i))\n",
    "        response = request.urlopen(url)\n",
    "        raw = response.read().decode('utf8')\n",
    "        raw = BeautifulSoup(raw, 'html.parser').get_text()\n",
    "        f = open(\"file\" + str(i) + \".txt\", 'w', encoding='UTF-8')\n",
    "        f.write(raw)\n",
    "        f.close()\n",
    "    except UnicodeDecodeError:\n",
    "        print(\"cannot process https://genominfo.org/journal/view.php?number=\" + str(i))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "70eee4de",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.7"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
